<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/3/197" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/3/197.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/3/197.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/3/197.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/3/197.atom" />
<meta name="HW.pisa" content="ggg;1/3/197" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.111.000216" />
<meta name="DC.Date" content="2011-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Chang et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In telomerase-deficient yeast cells, like equivalent mammalian cells, telomeres shorten over many generations until a period of senescence/crisis is reached. After this, a small fraction of cells can escape senescence, principally using recombination-dependent mechanisms. To investigate the pathways that affect entry into and recovery from telomere-driven senescence, we combined a gene deletion disrupting telomerase ( [est1Δ][1] ) with the systematic yeast deletion collection and measured senescence characteristics in high-throughput assays. As expected, the vast majority of gene deletions showed no strong effects on entry into/exit from senescence. However, around 200 gene deletions behaving similarly to a [rad52Δ][2] [est1Δ][1] archetype ( [rad52Δ][2] affects homologous recombination) accelerated entry into senescence, and such cells often could not recover growth. A smaller number of strains similar to a [rif1Δ][3] [est1Δ][1] archetype ( [rif1Δ][3] affects proteins that bind telomeres) accelerated entry into senescence but also accelerated recovery from senescence. Our genome-wide analysis identifies genes that affect entry into and/or exit from telomere-initiated senescence and will be of interest to those studying telomere biology, replicative senescence, cancer, and ageing. Our dataset is complementary to other high-throughput studies relevant to telomere biology, genetic stability, and DNA damage responses.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479" />
<meta name="DC.Contributor" content="Hsin-Yu Chang" />
<meta name="DC.Contributor" content="Conor Lawless" />
<meta name="DC.Contributor" content="Stephen G. Addinall" />
<meta name="DC.Contributor" content="Sarah Oexle" />
<meta name="DC.Contributor" content="Morgan Taschuk" />
<meta name="DC.Contributor" content="Anil Wipat" />
<meta name="DC.Contributor" content="Darren J. Wilkinson" />
<meta name="DC.Contributor" content="David Lydall" />
<meta name="article:published_time" content="2011-08-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In telomerase-deficient yeast cells, like equivalent mammalian cells, telomeres shorten over many generations until a period of senescence/crisis is reached. After this, a small fraction of cells can escape senescence, principally using recombination-dependent mechanisms. To investigate the pathways that affect entry into and recovery from telomere-driven senescence, we combined a gene deletion disrupting telomerase (&lt;i&gt;est1Δ&lt;/i&gt;) with the systematic yeast deletion collection and measured senescence characteristics in high-throughput assays. As expected, the vast majority of gene deletions showed no strong effects on entry into/exit from senescence. However, around 200 gene deletions behaving similarly to a &lt;i&gt;rad52Δest1Δ&lt;/i&gt; archetype (&lt;i&gt;rad52Δ&lt;/i&gt; affects homologous recombination) accelerated entry into senescence, and such cells often could not recover growth. A smaller number of strains similar to a &lt;i&gt;rif1Δest1Δ&lt;/i&gt; archetype (&lt;i&gt;rif1Δ&lt;/i&gt; affects proteins that bind telomeres) accelerated entry into senescence but also accelerated recovery from senescence. Our genome-wide analysis identifies genes that affect entry into and/or exit from telomere-initiated senescence and will be of interest to those studying telomere biology, replicative senescence, cancer, and ageing. Our dataset is complementary to other high-throughput studies relevant to telomere biology, genetic stability, and DNA damage responses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/08/01" />
<meta name="citation_mjid" content="ggg;1/3/197" />
<meta name="citation_id" content="1/3/197" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/3/197" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/3/197.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/3/197.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/3/197.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000216" />
<meta name="citation_pmid" content="22384331" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="197" />
<meta name="citation_lastpage" content="208" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hsin-Yu Chang" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author" content="Conor Lawless" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author" content="Stephen G. Addinall" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author" content="Sarah Oexle" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author" content="Morgan Taschuk" />
<meta name="citation_author_institution" content="School of Computing Science and ‡School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author" content="Anil Wipat" />
<meta name="citation_author_institution" content="School of Computing Science and ‡School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author" content="Darren J. Wilkinson" />
<meta name="citation_author_institution" content="School of Computing Science and ‡School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author" content="David Lydall" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author_institution" content="Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom" />
<meta name="citation_author_email" content="d.a.lydall@ncl.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=P. Abdallah;citation_author=P. Luciano;citation_author=K. W. Runge;citation_author=M. Lisby;citation_author=V. Geli;citation_title=A two-step model for senescence triggered by a single critically short telomere.;citation_pages=988-993;citation_volume=11;citation_year=2009;citation_issue=8;citation_pmid=19597486;citation_doi=10.1038/ncb1911" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. G. Addinall;citation_author=E.-M. Holstein;citation_author=C. Lawless;citation_author=M. Yu;citation_author=K. Chapman;citation_title=Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.;citation_pages=e1001362-e1001362;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21490951;citation_doi=10.1371/journal.pgen.1001362" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Anbalagan;citation_author=D. Bonetti;citation_author=G. Lucchini;citation_author=M. P. Longhese;citation_title=Rif1 supports the function of the CST complex in yeast telomere capping.;citation_pages=e1002024-e1002024;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437267;citation_doi=10.1371/journal.pgen.1002024" />
<meta name="citation_reference" content="citation_journal_title=Carcinogenesis;citation_author=S. E. Artandi;citation_author=R. A. DePinho;citation_title=Telomeres and telomerase in cancer.;citation_pages=9-18;citation_volume=31;citation_year=2009;citation_pmid=19887512" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=S. H. Askree;citation_author=T. Yehuda;citation_author=S. Smolikov;citation_author=R. Gurevich;citation_author=J. Hawk;citation_title=A genome-wide screen for Saccharomyces cerevisiae deletion mutants that affect telomere length.;citation_volume=101;citation_year=2004;citation_pmid=15161972;citation_doi=10.1073/pnas.0401263101" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Azam;citation_author=J. Y. Lee;citation_author=V. Abraham;citation_author=R. Chanoux;citation_author=K. A. Schoenly;citation_title=Evidence that the S.cerevisiae Sgs1 protein facilitates recombinational repair of telomeres during senescence;citation_pages=506-516;citation_volume=34;citation_year=2006;citation_issue=2;citation_pmid=16428246;citation_doi=10.1093/nar/gkj452" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. W. Chan;citation_author=J. Chang;citation_author=J. Prescott;citation_author=E. H. Blackburn;citation_title=Altering telomere structure allows telomerase to act in yeast lacking ATM kinases.;citation_pages=1240-1250;citation_volume=11;citation_year=2001;citation_issue=16;citation_pmid=11525738;citation_doi=10.1016/S0960-9822(01)00391-8" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=M. Chang;citation_author=J. C. Dittmar;citation_author=R. Rothstein;citation_title=Long telomeres are preferentially extended during recombination-mediated telomere maintenance.;citation_pages=451-456;citation_volume=18;citation_year=2011;citation_issue=4;citation_pmid=21441915;citation_doi=10.1038/nsmb.2034" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. R. Collins;citation_author=K. M. Miller;citation_author=N. L. Maas;citation_author=A. Roguev;citation_author=J. Fillingham;citation_title=Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.;citation_pages=806-810;citation_volume=446;citation_year=2007;citation_issue=7137;citation_pmid=17314980;citation_doi=10.1038/nature05649" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010a;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=S. J. Dixon;citation_author=Y. Fedyshyn;citation_author=J. L. Koh;citation_author=T. S. Prasad;citation_author=C. Chahwan;citation_title=Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.;citation_volume=105;citation_year=2008;citation_pmid=18931302;citation_doi=10.1073/pnas.0806261105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. L. DuBois;citation_author=Z. W. Haimberger;citation_author=M. W. McIntosh;citation_author=D. E. Gottschling;citation_title=A quantitative assay for telomere protection in Saccharomyces cerevisiae.;citation_pages=995-1013;citation_volume=161;citation_year=2002;citation_pmid=12136006" />
<meta name="citation_reference" content="citation_journal_title=DNA repair;citation_journal_abbrev=DNA Repair (Amst);citation_author=E. Duro;citation_author=J. A. Vaisica;citation_author=G. W. Brown;citation_author=J. Rouse;citation_title=Budding yeast Mms22 and Mms1 regulate homologous recombination induced by replisome blockage.;citation_pages=811-818;citation_volume=7;citation_year=2008;citation_issue=5;citation_pmid=18321796;citation_doi=10.1016/j.dnarep.2008.01.007" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=S. Enomoto;citation_author=L. Glowczewski;citation_author=J. Berman;citation_title=MEC3, MEC1, and DDC2 are essential components of a telomere checkpoint pathway required for cell cycle arrest during senescence in Saccharomyces cerevisiae.;citation_volume=13;citation_year=2002;citation_pmid=12181334;citation_doi=10.1091/mbc.02-02-0012" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Enomoto;citation_author=L. Glowczewski;citation_author=J. Lew-Smith;citation_author=J. G. Berman;citation_title=Telomere Cap Components Influence the Rate of Senescence in Telomerase-Deficient Yeast Cells;citation_pages=837-845;citation_volume=24;citation_year=2004;citation_issue=2;citation_pmid=14701754;citation_doi=10.1128/MCB.24.2.837-845.2004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. Gatbonton;citation_author=M. Imbesi;citation_author=M. Nelson;citation_author=J. M. Akey;citation_author=D. M. Ruderfer;citation_title=Telomere length as a quantitative trait: genome-wide survey and genetic mapping of telomere length-control genes in yeast.;citation_pages=e35-e35;citation_volume=2;citation_year=2006;citation_issue=3;citation_pmid=16552446;citation_doi=10.1371/journal.pgen.0020035" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=N. M. Gomes;citation_author=J. W. Shay;citation_author=W. E. Wright;citation_title=Telomere biology in Metazoa.;citation_pages=3741-3751;citation_volume=584;citation_year=2010;citation_issue=17;citation_pmid=20655915;citation_doi=10.1016/j.febslet.2010.07.031" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=N. Grandin;citation_author=M. Charbonneau;citation_title=Telomerase- and RAD52-independent immortalization of budding yeast by an inherited-long-telomere pathway of telomeric repeat amplification.;citation_volume=29;citation_year=2009;citation_pmid=19047370;citation_doi=10.1128/MCB.00817-08" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_journal_abbrev=Oncogene;citation_author=J. D. Henson;citation_author=A. A. Neumann;citation_author=T. R. Yeager;citation_author=R. R. Reddel;citation_title=Alternative lengthening of telomeres in mammalian cells.;citation_pages=598-610;citation_volume=21;citation_year=2002;citation_issue=4;citation_pmid=11850785;citation_doi=10.1038/sj.onc.1205058" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=S. Hiraga;citation_author=E. D. Robertson;citation_author=A. D. Donaldson;citation_title=The Ctf18 RFC-like complex positions yeast telomeres but does not specify their replication time.;citation_pages=1505-1514;citation_volume=25;citation_year=2006;citation_issue=7;citation_pmid=16525505;citation_doi=10.1038/sj.emboj.7601038" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. IJpma;citation_author=C. W. Greider;citation_title=Short telomeres induce a DNA damage response in Saccharomyces cerevisiae.;citation_volume=14;citation_year=2003;citation_pmid=12631718;citation_doi=10.1091/mbc.02-04-0057" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=D. Jain;citation_author=J. P. Cooper;citation_title=Telomeric strategies: means to an end.;citation_pages=243-269;citation_volume=44;citation_year=2010;citation_pmid=21047259;citation_doi=10.1146/annurev-genet-102108-134841" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Jain;citation_author=A. K. Hebden;citation_author=T. M. Nakamura;citation_author=K. M. Miller;citation_author=J. P. Cooper;citation_title=HAATI survivors replace canonical telomeres with blocks of generic heterochromatin.;citation_pages=223-227;citation_volume=467;citation_year=2010;citation_issue=7312;citation_pmid=20829796;citation_doi=10.1038/nature09374" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=L. Khair;citation_author=Y. T. Chang;citation_author=L. Subramanian;citation_author=P. Russell;citation_author=T. M. Nakamura;citation_title=ROLES OF THE CHECKPOINT SENSOR CLAMP RAD9-RAD1-HUS1 (911)-COMPLEX AND THE CLAMP LOADERS RAD17-RFC AND CTF18-RFC IN SCHIZOSACCHAROMYCES POMBE TELOMERE MAINTENANCE.;citation_pages=2237 2248;citation_volume=9;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Lawless;citation_author=D. J. Wilkinson;citation_author=A. Young;citation_author=S. G. Addinall;citation_author=D. A. Lydall;citation_title=Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.;citation_pages=287-287;citation_volume=11;citation_year=2010;citation_pmid=20509870;citation_doi=10.1186/1471-2105-11-287" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Le;citation_author=J. K. Moore;citation_author=J. E. Haber;citation_author=C. W. Greider;citation_title=RAD50 and RAD51 define two pathways that collaborate to maintain telomeres in the absence of telomerase.;citation_pages=143-152;citation_volume=152;citation_year=1999;citation_pmid=10224249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lebel;citation_author=E. Rosonina;citation_author=D. C. Sealey;citation_author=F. Pryde;citation_author=D. Lydall;citation_title=Telomere maintenance and survival in saccharomyces cerevisiae in the absence of telomerase and RAD52.;citation_pages=671-684;citation_volume=182;citation_year=2009;citation_pmid=19380905" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=J. Y. Lee;citation_author=M. Kozak;citation_author=J. D. Martin;citation_author=E. Pennock;citation_author=F. B. Johnson;citation_title=Evidence that a RecQ helicase slows senescence by resolving recombining telomeres.;citation_pages=1334-1344;citation_volume=5;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. Y. Lee;citation_author=J. L. Mogen;citation_author=A. Chavez;citation_author=F. B. Johnson;citation_title=Sgs1 RecQ helicase inhibits survival of Saccharomyces cerevisiae cells lacking telomerase and homologous recombination.;citation_volume=283;citation_year=2008;citation_pmid=18757364;citation_doi=10.1074/jbc.M804760200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. S. Lendvay;citation_author=D. K. Morris;citation_author=J. Sah;citation_author=B. Balasubramanian;citation_author=V. Lundblad;citation_title=Senescence mutants of Saccharomyces cerevisiae with a defect in telomere replication identify three additional EST genes.;citation_pages=1399-1412;citation_volume=144;citation_year=1996;citation_pmid=8978029" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=T. T. Lin;citation_author=B. T. Letsolo;citation_author=R. E. Jones;citation_author=J. Rowson;citation_author=G. Pratt;citation_title=Telomere dysfunction and fusion during the progression of chronic lymphocytic leukemia: evidence for a telomere crisis.;citation_volume=116;citation_year=2010;citation_pmid=20538793;citation_doi=10.1182/blood-2010-02-272104" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. Lundblad;citation_author=E. H. Blackburn;citation_title=An alternative pathway for yeast telomere maintenance rescues est1- senescence.;citation_pages=347-360;citation_volume=73;citation_year=1993;citation_issue=2;citation_pmid=8477448;citation_doi=10.1016/0092-8674(93)90234-H" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=D. Lydall;citation_title=Taming the tiger by the tail: modulation of DNA damage responses by telomeres.;citation_pages=2174-2187;citation_volume=28;citation_year=2009;citation_issue=15;citation_pmid=19629039;citation_doi=10.1038/emboj.2009.176" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. Lydeard;citation_author=S. Jain;citation_author=M. Yamaguchi;citation_author=J. E. Haber;citation_title=Break-induced replication and telomerase-independent telomere maintenance require Pol32.;citation_pages=820-823;citation_volume=448;citation_year=2007;citation_issue=7155;citation_pmid=17671506;citation_doi=10.1038/nature06047" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Maringele;citation_author=D. Lydall;citation_title=EXO1 plays a role in generating type I and type II survivors in budding yeast.;citation_pages=1641-1649;citation_volume=166;citation_year=2004a;citation_pmid=15126386;citation_doi=10.1534/genetics.166.4.1641" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=L. Maringele;citation_author=D. Lydall;citation_title=Telomerase- and recombination-independent immortalization of budding yeast.;citation_volume=18;citation_year=2004b;citation_pmid=15489288;citation_doi=10.1101/gad.316504" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=M. J. McEachern;citation_author=J. E. Haber;citation_title=Break-induced replication and recombinational telomere elongation in yeast.;citation_pages=111-135;citation_volume=75;citation_year=2006;citation_pmid=16756487;citation_doi=10.1146/annurev.biochem.74.082803.133234" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Nautiyal;citation_author=J. L. DeRisi;citation_author=E. H. Blackburn;citation_title=The genome-wide expression response to telomerase deletion in Saccharomyces cerevisiae.;citation_pages=9316-9321;citation_volume=14;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Pan;citation_author=P. Ye;citation_author=D. S. Yuan;citation_author=X. Wang;citation_author=J. S. Bader;citation_title=A DNA integrity network in the yeast Saccharomyces cerevisiae.;citation_pages=1069-1081;citation_volume=124;citation_year=2006;citation_issue=5;citation_pmid=16487579;citation_doi=10.1016/j.cell.2005.12.036" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_author=R Development Core Team;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. B. Ritchie;citation_author=T. D. Petes;citation_title=The Mre11p/Rad50p/Xrs2p Complex and the Tel1p Function in a Single Pathway for Telomere Maintenance in Yeast;citation_pages=475-479;citation_volume=155;citation_year=2000;citation_issue=1;citation_pmid=10790418" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Roguev;citation_author=M. Wiren;citation_author=J. S. Weissman;citation_author=N. J. Krogan;citation_title=High-throughput genetic interaction mapping in the fission yeast Schizosaccharomyces pombe.;citation_pages=861-866;citation_volume=4;citation_year=2007;citation_issue=10;citation_pmid=17893680;citation_doi=10.1038/nmeth1098" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=R. Shachar;citation_author=L. Ungar;citation_author=M. Kupiec;citation_author=E. Ruppin;citation_author=R. Sharan;citation_title=A systems-level approach to mapping the telomere length maintenance gene circuitry;citation_pages=172-172;citation_volume=4;citation_year=2008;citation_issue=1;citation_pmid=18319724;citation_doi=10.1038/msb.2008.13" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=S. Smolikov;citation_author=Y. Mazor;citation_author=A. Krauskopf;citation_title=ELG1, a regulator of genome stability, has a role in telomere length regulation and in silencing.;citation_volume=101;citation_year=2004;citation_pmid=14745004;citation_doi=10.1073/pnas.0307796100" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=S. A. Stewart;citation_author=R. A. Weinberg;citation_title=Telomeres: cancer to human aging.;citation_pages=531-557;citation_volume=22;citation_year=2006;citation_pmid=16824017;citation_doi=10.1146/annurev.cellbio.22.010305.104518" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. C. Teng;citation_author=V. A. Zakian;citation_title=Telomere-Telomere Recombination Is an Efficient Bypass Pathway for Telomere Maintenance in Saccharomyces cerevisiae;citation_pages=8083-8093;citation_volume=19;citation_year=1999;citation_issue=12;citation_pmid=10567534;citation_doi=10.1128/MCB.19.12.8083" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=H. Tsubouchi;citation_author=H. Ogawa;citation_title=Exo1 Roles for Repair of DNA Double-Strand Breaks and Meiotic Crossing Over in Saccharomyces cerevisiae;citation_pages=2221-2233;citation_volume=11;citation_year=2000;citation_issue=7;citation_pmid=10888664;citation_doi=10.1091/mbc.11.7.2221" />
<meta name="citation_reference" content="citation_journal_title=EMBO Reports;citation_journal_abbrev=EMBO Reports;citation_author=I. W. Zaidi;citation_author=G. Rabut;citation_author=A. Poveda;citation_author=H. Scheel;citation_author=J. Malmstrom;citation_title=Rtt101 and Mms1 in budding yeast form a CUL4(DDB1)-like ubiquitin ligase that promotes replication through damaged DNA.;citation_pages=1034-1040;citation_volume=9;citation_year=2008;citation_issue=10;citation_pmid=18704118;citation_doi=10.1038/embor.2008.155" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=M. K. Zubko;citation_author=D. Lydall;citation_title=Linear chromosome maintenance in the absence of essential telomere-capping proteins.;citation_pages=734-740;citation_volume=8;citation_year=2006;citation_issue=7;citation_pmid=16767084;citation_doi=10.1038/ncb1428" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
<meta name="twitter:description" content="In telomerase-deficient yeast cells, like equivalent mammalian cells, telomeres shorten over many generations until a period of senescence/crisis is reached. After this, a small fraction of cells can escape senescence, principally using recombination-dependent mechanisms. To investigate the pathways that affect entry into and recovery from telomere-driven senescence, we combined a gene deletion disrupting telomerase ( [est1Δ][1] ) with the systematic yeast deletion collection and measured senescence characteristics in high-throughput assays. As expected, the vast majority of gene deletions showed no strong effects on entry into/exit from senescence. However, around 200 gene deletions behaving similarly to a [rad52Δ][2] [est1Δ][1] archetype ( [rad52Δ][2] affects homologous recombination) accelerated entry into senescence, and such cells often could not recover growth. A smaller number of strains similar to a [rif1Δ][3] [est1Δ][1] archetype ( [rif1Δ][3] affects proteins that bind telomeres) accelerated entry into senescence but also accelerated recovery from senescence. Our genome-wide analysis identifies genes that affect entry into and/or exit from telomere-initiated senescence and will be of interest to those studying telomere biology, replicative senescence, cancer, and ageing. Our dataset is complementary to other high-throughput studies relevant to telomere biology, genetic stability, and DNA damage responses.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479" />
<meta name="og-title" property="og:title" content="Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/3/197" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In telomerase-deficient yeast cells, like equivalent mammalian cells, telomeres shorten over many generations until a period of senescence/crisis is reached. After this, a small fraction of cells can escape senescence, principally using recombination-dependent mechanisms. To investigate the pathways that affect entry into and recovery from telomere-driven senescence, we combined a gene deletion disrupting telomerase ( [est1Δ][1] ) with the systematic yeast deletion collection and measured senescence characteristics in high-throughput assays. As expected, the vast majority of gene deletions showed no strong effects on entry into/exit from senescence. However, around 200 gene deletions behaving similarly to a [rad52Δ][2] [est1Δ][1] archetype ( [rad52Δ][2] affects homologous recombination) accelerated entry into senescence, and such cells often could not recover growth. A smaller number of strains similar to a [rif1Δ][3] [est1Δ][1] archetype ( [rif1Δ][3] affects proteins that bind telomeres) accelerated entry into senescence but also accelerated recovery from senescence. Our genome-wide analysis identifies genes that affect entry into and/or exit from telomere-initiated senescence and will be of interest to those studying telomere biology, replicative senescence, cancer, and ageing. Our dataset is complementary to other high-throughput studies relevant to telomere biology, genetic stability, and DNA damage responses.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
    <title>Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8TQIhjQFY_d-YM_Gqh8JMQIXY3tt44EOTd2D-yEOLvw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459171","apath":"\/ggg\/1\/3\/197.atom","pisa":"ggg;1\/3\/197","ac":{"\/ggg\/1\/3\/197.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/3\/197.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/3\/197"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/3\/197"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/3\/197.figures-only":true,"\/content\/1\/3\/197":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+SHTemwmwL0cCZBi6KJCAFoBlR6519Lu+jwFCbAqVBZ7iIKSdKd25EClWzq+5KC9Mt2O7LmFjJlMgaydJY+c0XPxucSxduE60QKBG40uwJkx17DNg0NQaEMqtYL3Q0yg3w8uSKN0bfeyAWsoOBc2x5OIbqMhhRskYREF92tUQMsAneSuliv7SzQ+1BPR1gUyT1oXE7Yt7yahK+vnfbEgPFn2bAm2sTY3bBXIFZP1mCwUx9Xu9Laete5JU4Js723rXuzDGstXjuO7PE7\\\\\\\/R7xd7+Gw+fq7lIxpRtGk0he1+VSyl7q0RQlZ6U4kHWlcPdaQu2MfJ8quKeteK8ecpCEQxRGqYRO9ilxLVMpZCBUD76Ewi74mQ9CwKsgz+B44s80X7UaWSDhaBw6bL+MMVeysCKzPqeHdHpeWq80t8LZ4TIdQd47kMvYFCRyCTe\\\\\\\/mX8Vnt3IB41DGd60kSoSRK79deU2C5H3IEGa8SX1DXA88RD2QwtoHLSq1fJ0OPA1Tyat1cM+k33Qr3B4mWEjnKg3EVoCDQVejJJv3vjRxiYYs2ik4gxBvMDOb9KTpCEuWmsfjqHwNWBia3kxFSmEdKFreUQyNnjkPaeHOGafnsR6Wf\\\\\\\/PRWSpOUoL++W\\\\\\\/7uNBSA2DNQuZ1+q\\\\\\\/C6DA0pYgKxYOFCiZkbLSXoFOwmrhJbI07To7anrrMzKpp1tWMHnahkS+RZCVUiiqUlnEuzI+RhBxtRGiGj5CN4LR99JaL8xC8lQDSn+uabnoR7mLkwCVREPlvJcf7Q4m7NxpX45J7jarMRBiAJsqxlm7u3QiuLdwxYsUFqq5lkxn6rfUEz7\\\\\\\/7NosESFjuUd3lW69WI6yQzUFbN6WD0JpCIM8Hp7aNCpirRLjzWe2f3q1Jm9EPJdRkg5oPQq3aCKZLDZS734PJi7JeSV\\\\\\\/9mZn15ZFdvZEYYu8\\\\\\\/2sZkQbqkweHejhshYddjJULBumNWGe7fReZotKvSIxziSwIdDFIXOn5QYjIey+AIox6AhIfFSuGD7cmqILBWl4TILjPCjFonXqvQMLr5FdhhR+s9PVZz7wrtfboz\\\\\\\/0Hj9UIb0JjZ19QkLvxO+XKWwTJykxqItDX8b\\\\\\\/c7xNBV54\\\\\\\/Jp1HngxpqF6AwcEKMBF3bOKl47suB6Mym5tn12wCo6Zv\\\\\\\/qaUu2biuVJNx37C9Hsof8k4ZFIV\\\\\\\/HImvtxhrYcxsmu77rYDE2V6A08q4pUPx+n3eOmFsYM3WtFnmKJzIApipDPRGK\\\\\\\/LhPGHxS634APNT8uuWvvhEU4nSwuscrZ5P4ZQ4\\\u0022,\\\u0022iv\\\u0022:\\\u0022DKXD96UkGyGg6wR6dCuS0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022866ea628fba91a19803fd8656760e8dc6899f9134abf4534e89e53f63757b801\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lHeXBuRz5aCod\\\\\\\/uLeYUHEzk9Q2nbTI6GfkGaxU9\\\\\\\/9A4h1+oJ\\\\\\\/pK0+ugoJfJBn8JUg1mTkEOXiU10tPtliiF79uux8LjOhdp0mKpD7jvG2t2Z77JLN\\\\\\\/1fj\\\\\\\/+WKrB0pjwilCEqMzeCQcaxS7S8NyeMcodBy5TN3P+c6thHQGPLJUoEg6BAm4kBH\\\\\\\/flZX7IJBD+zD3OYWeDDp5FmNdop1U4OVG64AXEW4WYw5035LvUyx+U8tOhav5ZRFSfaY0R8JTpoIO7nEr6YFDZ3hFFVmdCKYsmZl5EwFkoAyP5zhuQWelNbfpq04vxv3JngAQTCd10cIJwcY\\\\\\\/D8eqmQ\\\\\\\/R9v8NI2S+5U0IhM95E+ye0w7HXzoeOdHfLR\\\\\\\/RWPTAHoM9L5KHtb6HPZkTSER9m5loThBSoMEQ2c+tiC0dMOBXevZb8Q6jgmpNQSzCBxz9\\\\\\\/cNokCKhoLf1hJ3Kr05Tb+8ZTQnpq7+Z0JHHpz55E7IbVFxLVu6X2mJNB9eHEQk9ZxlJ4gOwCJ2HdK8SFIhsDrBjAvbcT6hhraHEceLsLS2PodEkMv9ebOCPjB+WS+GFzD8fMWKE5jswa2r2EY+0\\\\\\\/KhQIUHlcEDUT15uPd3XRugP\\\\\\\/rd0NjgcjJake+h8N5RHgRJNJ27R8tlQBZ0nVMeZmf2ZaOuhCk6cQobbwPzCSxr4pxkNACyMR7kxNxkewx6YKZb0LWQv8vwlmBZ9ZdyHFzNN8hBbEzj+INAQwm9b4RG1lH2HCwM5cARen9SxwZZb4sEA\\\\\\\/CxKGfJZ2nAtMRY8Zc21m3YXk\\\\\\\/7lI9mM\\\\\\\/1v2KaCSWeXTj\\\\\\\/E+iAcVTKrgwj2O5HUJ+OrCuV93bL+50eMs9rflH+X1B4xHyhWx5ZlQEjZF6w54pEBCD8dKGNbXmvbJNG0bKM5lbJTpB0+RxeZCHsfvIc9WWX3I9OU2NZIgdLQVBPW28ZX8DI8WX7B+DI5TiecLqibzKMSjNOar8oX8DEnFUatko4kMXK1QbJu6h80baxjgsC78BRfwKKN7auW8vNPe2EQu+EYeeUsNNMxfPPugWOas3E92lnN+6DZXRdB+\\\\\\\/QhKEhQBvYXrmqpKeLkKl3K\\\\\\\/nHDobvNpcC3J7KDBk\\\\\\\/nrk0tebNUdPF0kdH6HVla8eXc5PdQdLomeLLLn6ijHGy6naihdi0pVY8uCPsDpQibNUl3Lgjr6ZShC0Hxx+TL+SQcqO0\\\\\\\/NQDLmeuQg2w9SIykoQiJgBTS4GQVSf2Te1l1IDm9E+h1BOoFYfHt4rwdAyBFr6XSJMnkmaFDYo5LvaTsjSbaJriG1ss8Zgna42i2rRmDIP8xMYdPqw0CcuMbilj\\\\\\\/BPjylV5YOHs13KGCwbqFWucoD3djXxd94SPw6OoptKdozEEdBPNlJkh27chjsprYtaopWkr0R52x9KVGg2Il\\\\\\\/5QnSbqAdG2qiBstt5C0dwzBXCNdjK+nFvyqR6gL2bI7aPVTBjsl9IRVuxeF2ie\\\\\\\/IPncfTIl7E5KGyccs\\\\\\\/XJ5XeaKw\\\\\\\/SQmX\\\\\\\/o3SyIILfNyM048fQXJMUBgSvG55qB6LUiJ\\\\\\\/++bByqD7JTs\\\\\\\/Z4mYSHsxCYqT6U5PRyJK8dCp4ug\\\\\\\/tjx2TFxrZyJLog4+Oh0TwgsR\\\\\\\/CRwkjqLSPe1BdNZCjB2LDzYAW4YgRbZc1uijmqQaELu98bwCOG5DwSDznLveqbMUjMCe32SJ6v3yoU4erBV1Je75w3WYnAJnVZBNzEYewT2f6Cfr8jIjnfpO3NNm1o7ac62EPsvO6o9hD2iuf4ciSaoP4fRGaajk0K5Ro6zvBP\\\\\\\/FvOmFVBtJWDIJeXn4Jt3m8VTRwh4KsGWMATwx5PkIxMRPpKbBYx5uj30iTZjf4f7Aei91MwfsZyndr3iy87h+rh3SlIS1VAmS++VNYTKSOkczatIdnAOemSdTVi1RwtnczUhoQMIzf7v+84q7CILpN6Nvf095GSYHSdjI9wK9q4ctl5elDAf1qJpAqjM7Q9UmWFnp3BdAXCu1z8dukcYALv4aUOKCXReQ2U9N5JCEreWtYC3dPaEHuSEPXxmWDAG3wkHiXeShGA7dd1DHTDy86tdCW8rGReclPS1n\\\\\\\/x3WTD5icGWCocvYUDLUrVq27WHkk5ItgfVKbiw25Qirf\\\\\\\/4v5Ll2Fdo\\\\\\\/Xc+9tW+C8Jm3uzgSN80Uwj958++qOSQxBPc+XfkMkOLj6Skb06szdDhzWqSXqgMHDbGRSZagPn8tNhvryIGHX\\\\\\\/A+wI5BclFkRuGtxi5lswlD1R2m+IBakKB2haNC7WB2yMUarcBrqlKvrlQE3LoV9nJjoeoO8Lj8bxNL8u1tnb4R+4lnN4zN1kikc7JEoBzV3e1xTzAj3NfIZ6d9yWobOFBSrmxuXKIxK9\\\\\\\/zMFxVRotiDH8VoWzrZAXXihkUIGnyTFGAuvsQPKXwfluDtD7uVR0RQb1oQDJvenT17z4NbCbdFViAEPA8estCyFBaNxeyWCYfV74GH4PT+FmDzyI38fwWRLqc++DqRlR5ZqdM7WAUOBU0cXEfqs3PL\\\\\\\/i2CyW1HCJCdzHjwGSaQt0X4vZnpSQfghCFBc2wMwVi2myl3E6qhpk8yLca2kHU0bgc3OF60pwKmW9pgxdHQiiBiWE\\\\\\\/tKZVkNKIs2hYUhOXt70187eK4w4+v2cqkRheMrLzdEBVZJDan2LFup0dFtyHk6PqMIt\\\\\\\/VPQjqXfwIdl54p75Z222mMbpdTg2c9ELhoWUOAGDfoPJOast+ffK6cr4HA7XM0nGZ\\\\\\\/hX+ixrdb6ZDmkTTNT\\\\\\\/X4lpHyNtVR0ya+UPNfBh1p\\\\\\\/L2zT\\\\\\\/LZjzlTVn3uw0SpQrwCqJNfdlTIwcEkfMizDnrzdTG569SkleUnFux3VuWPCIOkspLzHNfrsbJkWs\\\\\\\/t2mbFg3eu8qQvDaaWnr2GloOIG5jgl8mcSeEKhPkwfUw+3SpoFS51jnmTkeZ39oAadCakDHKdp8bYfn9Aouf82TiRV8Yjq2ZeNPJwc+cosvQNN6a8apyah6SnedZDu3Zi59ceHQ4e+ufKRBuhiJdS8sV7BF+ONcZOPOTGTM7bUONPrHjICcxTN9mpQ5RtWeWwUV5nh0uq1abrkUROvPCzCuuV+0HcCP1w1Fsp0VSz4rMTyJLBm3o6uzThzHrMFoUGQTik3IhWSpnFm1la1jQ37XT3Q0gJliydImWBDDJXiWjjTX8FnvMXzrV\\\\\\\/WE5Z4siqMyya0SH5T+evRMW3f+9JQB27Taqdh4Xqpf9GdibMpKfwvwHq4aRbFrP+Rr8TX6bbxbHPejwVlwAt7eEYpnBIQwewP8nfp14g25vllqlCjhiLuFkUnHj+JYCQFq5jBd71+VB13GsjY6NhaG8K3lwoUGNM2cCFRC3V2VRx\\\\\\\/YYAqzLpsvHbxbRqE83FPLG24VruOdDbnapCZvgzGISRGbxW51UsZsT5q2H6FzMJ4eMH4iZXFnm+82Oqngpkw\\\\\\\/C5Fw1kF2CJgXZXowK9UgG5brZeXHFywjR5FwKHqxnOOjUW1nI4ttVKrSgGGsejO4NbJqdYHu5CvV\\\\\\\/ogvvjq8TfUBjwtnszSp\\\\\\\/4BtKHUCtC61hGpOh18Si9\\\\\\\/yrRZI7dnxxyV4ftwKGxyLpu37idYGi5KVI9eAQJobUR7baYlAM4zNrHmZpkroHzmRJu+OgrnqiRadcwJU2j4Y2YUaKKdA1BmL6Z9XLYB0hInksc65tjjfgQvUJ97ZhaMgkSfr97GlOuana67NlKtYmd\\\\\\\/eI4oWuNbY8G97srDlsS7YyO\\\\\\\/q2l9oiJoR7Wacjb2Hsy9d4GQtymUb4X6C49NhQCiEgGIO4ein7S+JdKxXeF2fYKWgRg+lGYNOLWS3xmFZYA+ikiVPjmJIFGe34QJEa8BxqnYC78+qd0bPSqA9v7\\\\\\\/cJTzLnXVGqkeQHOVnHEVbRsSpMbN3e0xfkNbJf+kiAp89rCSWAYDFzc25PWblP5BQEnNcQPFYJ\\\\\\\/IIaz0sw7l0fqrlU8HWjFxjCVNmD\\\\\\\/cnFgJeZCR7XhshYAlbGjJFUf7jQ6wmYgNu1cPYF4PP54o8NeBwGNaBSgB4pHBMonDjMs0utrF8IVRako457\\\\\\\/gR3uNLGT+5aflClIsaENAqq6ed0nGUVdBpLh\\\\\\\/NssPtdppHwXWfxnJ3BORgZqnzrsWNk+LKltKbq1uSA3jaxsjIQoXU4UD0WpkB+pLosCLPaNBFj+WP3w3kFg24eV6qIf1Xf7vPm694FSxsO5ZUMJ5eFljKm9rbP41CGedNP+NPbGbrf7fxzKHOJCQlNkwF4nbJCNECRd6ge2UV\\\\\\\/4krwA\\\\\\\/6eviNFy++cePl+NmzGAL934N3sx2C3\\\\\\\/rx7ukOPlRy9T\\\\\\\/zaFjuk0w+Bl+FG8a5euSRy5uxdQgQezj6FQt7pqhYIcwSf8arZgBrhLJbFmdeEBAuO86Qr3QPIrf6PxdJuc4TMx3E8WMTOtHh2K90cqgI\\\\\\\/vLbFnu7eLKLA3Qi95l+wU7gK+9er\\\\\\\/udkZXucbObuDPKx4KZ0L7st+ItNHwuxGs8QQeI+xu9cW2bc20da91RANBVffp+uy24+DbexeGdnHlsD4TrI0l+m9LS9vZN1RXLhOgJHWUgc9aKcWLu2amKFNg02hj1n1+X7VpXKH0+t\\\\\\\/NVgZ2lr9faFVeuNVUFTqSGeBN1F\\\\\\\/aTMb+3rd2kVCXCk6f9hJkC\\\\\\\/IrHVYrnQqLRzzbBJolblLGUJrdfF7gBrguLdMxdYEkC+0Mc\\\\\\\/yzh\\\\\\\/myCnnPBmQtQZkkKOuGPuaDjVyDrImesEbYmt2AiJL9ZOz93AgeO1mXyG3dyRObEj4FseWkipwnMJ+LDZ6MrSBufs\\\\\\\/45mnOGq3GebcgVF\\\\\\\/bYYlHMOzwQ3vVVXoaYpXQFQAE+l6V4iEhSL+Xp\\\\\\\/nfFQ4UT3ArLL50LEUPyfX\\\\\\\/2xoc+GNkGUNWukC4Ojq6Mh5wRdXO3l8UAJddcQ78Odbtoj2RDjCBG0VLyA94yKSMbz0Ivjb8F3+vjBHKxO8O4YdavPjy5GTyvJ4vYLtjgNgwkF7oDRJWxAr3xl36kBtW3IY3a3u5xCoohFTdqEVMInkR2E92O1wdn5nb48vHfLTJ+OYjvR6FyZROvUQ\\\\\\\/RdQxx0KLtvstpqp9tu3TKw\\\\\\\/vTK9cZu2I\\\\\\\/TBleicnRqSZ67rqfoEWmtEa8mKrnW8RGUYfDGc8Kj+hk15oAiq6X0GF3Jkr2BxELQX9G3Njo0gOGv3qrM5m25XDknNf2By9YSqZNo3seSwRhvSuSMxGjA6o5DxXD2Eb1aHrhgVo2\\\\\\\/oxoA+7aAsYJ3vjDMc6h4DYU13mgovsSWdaVIY8ywCCNF3gWPvmgMzI7qSMrGXtUke6lcWXZxGHR+efGvxRaqh21GQQb0oaXwwIUXbbAkgut5VfLHo1l+1BlROedKtL8t9CiXG5VwUMDd13iqihDyqABRVOPYk2+3+i0iC0EVOpfGH8L+FuG7Um4zxMUPvqMPvKZimvms9LyD7WOLeyEiAKNEtkUVUNsnsQTmJN3FJYBMRw8ldmQecYrgfJn5UiULpKOlVSRb5pekYrbsi8geMT8bO+3co1b\\\\\\\/yc4d8EaFKy15ap6mcVPPbBdstCNhLntPQmEA6BESOgO9UkJtCZj9nr1bo+5LKVackajDJ5xlsFqep3sbtUTu1UFGNOJwQ3YbujTU9cPmQKaPAnyhhQ8sLpv6CludT0U3ooFXt\\\\\\\/BkkAKsrgbGmCQNoRWMGvQohm2DveXDie3XodppD34g9PJwFpy40eIa9UIPRV3lWRIoaSScDs637Ot+ShA5B0\\\\\\\/s2pVmiAoRYaWTvwRsKRrJzw8YllOAaBXWVpjI9q\\\\\\\/pXCC\\\\\\\/+VkO3V2g+kc21yM39aud200mnWo0H9o8Kt0SV3SfzKE\\\\\\\/Bb6vvWRHC1WS2z7U2Jsmb0TNDlDO1pPpJaO7prd43Bdul3XICa5bwBy99JTjYEt6kPzd+rsCmZi8OcXMWLmHwJpmfc30I2bqH7DVQx+lnLWK6bUGBOLk2CsXehToyWSi1119x7pPXaf\\\\\\\/ec\\\\\\\/XebXhU6Br1yMq5pmrzOY2JNeO+qMYNc7wUnGhwftaFPNHUe6thUpJBukdoigotUMUtXhGd9sVYz8arNUWps1SuFz3EUiVYFnlNoalGyNDS94BFl6sD\\\\\\\/UcLL3F5P2wKg4Mo2DXOSMOBPP0SElSqXnq2bG+oEZvytgA4gmtitaBqbsEpWd\\\\\\\/ynmaMgr6kzAA3fAA7IKHmwXYT1kB3nGi58gJaQ1X7nfN\\\\\\\/jDNto4O4X6qX9vo9RnOz1gSJ33YJg1VT+\\\\\\\/356J\\\\\\\/CGGaF4XRopFV04UiEiKJaY+1PDb9t7ddUxNs4xkC0ZX\\\\\\\/\\\\\\\/i0OGra8fRVCNYCZa+Qsz1I0BFZW5VaKwWTwMxjmkNy2BayPDjbJWYybMqYGhvIDEfXiDpCC2uAzggqnjWvmu5GCVqcxtXHk26Wpe8o96DFQPlRxlfopks\\\\\\\/C\\\\\\\/k3GI\\\\\\\/+naCJLq+NrFf86xSCAusnN\\\\\\\/bRhxBxq\\\\\\\/dYUGdwUryhl8zPh9pWIFFdGrV2eDy+kiABV9ZD\\\\\\\/Lcpjv6pUMSXNq3KD3LhFJ01vtUGNDq1ibN9wvogaobNoNPHP07we8O7icQaNENq2f7bTROznVAgMNtQOyROMw+iM9dk6P7dH8T2b+vuWrqugeb9pkX8EIq4h5ckxLIP3QEX1HQ6S8jaraV+EMXSQd9K8vR3Dj4OkiBNzIMbku5Q6GF\\\\\\\/RDYYf4JgQO1GWlQ3M2YyONA2rM1ptd8VikDcgDjIgKIulHoWe8ZChUoY\\\\\\\/NDkZX6ESKQjrcDMcyB3dswsL3srcWEK7p3V00qE437w2D7zTJpYIJ\\\\\\\/iHM3STZTw3lMtF6ChG9otsATTmEwYqCBah8pJwNrI9TEas0lt0YEKOuXgLblr8M7bMGd2QOB\\\\\\\/x2lDaL6Zb88VksSedyAxd\\\\\\\/qhatkmBThDDqy2Zzxy05gewTIWB6dqP2Lx7wvvMahzrg0mokyFqdm4yc3dkdkhCpYgTd12iZ5IA2usXp7WmKN41KcqrjSYCEOawBv9fco+bNqiK6FaLDd5kAI08Rh9T5UifidcwGKR\\\\\\\/\\\\\\\/7YJnFDcYk8Eyxv4DNlmzZU1IKKJRSiozKPklXN75Q7r72udKJ4gEbBGzL5g9sxp51S2f2kJorZvA8M885VLdbdgPI7XbMDqXoBgW4qyxk6SHl1sR0VCjdczJi7YPCAB2d816\\\\\\\/pdPOERA7vkJ8Bp8o8XGd9MbvVAS8yoosfYHs4eRf5J1QjabIzqAw3ua34cVAjt412UJqy74YYpWrvnQbSatnBo7vnAQANaigHhg28IV76BgFyAZXLCirK72NLwV4\\\\\\\/EcyRLSTt23v9+1jgR+5csRPTb4JtwWNqeG88axXoa0tbVBBZveRgZFBqXh4Ve1oi2dtJ2sIavoOLmlmLNBBf5kdvJO1Eayak07WKC6O\\\\\\\/pzDEuNqZyfQfoKfE3t+XZU18w1RHOXGRrSD2CKGQaPP4w7PB4vhMSaDr0CBCzEthhkgAUdhprRR4TlxDHkkFEjqWH6Izj4cDfcaCf8OZekJlA7TtzRgG2udhB2gEIICOoqkuwTSbE8ahOEHh01U0Eq0qSILiEBnUOhxWKxWtuTNb56ahmWDrPqzLfnfI08h8eLkywkPoGeSfWfvNGabOvyroc9FsdJFabLibuIYLqQxS6DuJduGsmNnU7X3heAEwhiAWFrfLmsz00lQt7lvpTNFiQMVsGiUpbSamuStKJqUq0qeg\\\\\\\/asepB3+8YHeKIU4eJFM6NSWYI4\\\\\\\/GSkKAcQHO37G4+egLHLHC44sIWtijxZwS+d1lcoCW6L5BTNlQakpYpA1l73wi2clNvJFlkz84Uf+HN3YtJg5QPLWlhVcD8zjbjJSO2XvWFf31aI8grUi3Ok2RlPlASa3Lk\\\\\\\/ZIO+KxNeqpr5UFEofBt3W4ZLXYO32p1xFQFdfPeAPp1UctCdeWFkxIKw20fAoqqGR1sYXmTwE73Q7Rvq+C90KdL+JWFrlmR+y1qEV\\\\\\\/r0XAiADsaV2aYl2bJf4Pf\\\\\\\/1S3MlESdJVKD46v\\\\\\\/4AKeAoiOKsrNsBM72KysNkX0PxAZwIdb2qZNwshkKM\\\\\\\/D5H4MjZHvV+YZHvo566nkdMAM3fSiCK9zWPWxbs9UuWjwrOnQrBOK0xlRHCFGHYl3KvoBaqEH56v0j1Wc+kdG47gsrJ8uVdArxXysbDYgPcHjWB2X2EVy8djUngOCo0ft+uyblwYnsy5t1OKniyxSDxfRgWDjMsJTB6OOwFKBj4XVpvP2YS1D+lMJneb0xygAYdeHDEH+\\\\\\\/8UFE2B4ut90tZY1V\\\\\\\/DUSSvoy3Tj6Z2Vm1l6+O1IW\\\\\\\/ITHDBDHMwDWb6XZdqf0ge4P46dFz8wY27p\\\\\\\/B7hbi2Z250O5IflbP8Q7awaCO9jbo8ha78WaIWY7fxlkyddfIaVFsxm65w8F\\\\\\\/exrZPdoaiVQDxwrtfUpLklX2zjz1KzNaQnD2C5mu6AzRXwZ5al\\\\\\\/kG4QOxPsEbRpOFQzO5YX+0QOpfo7QdX4vPu\\\\\\\/hcf8BTz6h5AgB1gKnW3AcgQXG\\\\\\\/NVE9SxfDoY7gG8gj5qoyh6NVJxUkZtAkqb4wR2vjKvzLKzC\\\\\\\/C7OuPir2NZlbX5tfLP07gWetszHjQlL+pudXVCMPCd2dtj0t33Zirwys493qRNmxANOTI0sgZA4017Z80gLdQV2sW5NdSQl9xY+B0a+vb00Z7RjBmL9IP1pp2fzbyUnJjRm39dj5fgHsLNAFb5LZLUWh4dDcJAX1XdqfTtcFPplWPfNk4Jv4rNfnqj1yesYHgwyY6gGM+1lx31lNTOfqmYKsW2fWIV8I9uIskW5c0Xa+XR94R9EWa6waZS4T7Jw2fBkiWAhB4A9YY3csDjeGfDgMzL28r\\\\\\\/QNAw2+juAs7HhBrWCC8udOU04x2Sr6pBQKPwfXwkHawj3W54VfKkT4JM6HJvw1sC\\\\\\\/NgfgHyNcF3a7xlJpwxmSdam23PEdfsem3zt+qr7xkHQEuG0FMRWocp8ubR2LZj7nCw1l3VJuDgWaMzERLNYAvL8X0LxtRg3mSah0gxMUvW0rwM4cUC0TWsillslbb9w2vjMSrM2vZSOvyoAIifQf0DO13Fv04vnzRxoJn+DEaH867fsQ1dq\\\\\\\/1xHyXdeYVtUpbksBAaDTsy9+FTv\\\\\\\/cKduSBDIae0hy1\\\\\\\/56Zkq79kn4VMna8JD0NkfD2X3EH6QqSPbw9WQPYPuuxFXDAvGhwjT+iVJLTSOQ0oGBWt61JpfclbPFGhULMl1sm6T3th9KzJgXfBGkckuxubDQ7l3RC9vPdDwYymaAv0YWLoK0LTPMJ7Gn3J2oFS3CDqKE+TxoFFLw8kIbdRbv0iSXa823Zpy5Zr\\\\\\\/2ZYiaGD+rp\\\\\\\/ABzuEzrqhUdYd7Y8cncqVEx3UEgPZoVEUcV8jDa5Yz4c6BM+9e848yUaT2h1ms9\\\\\\\/kwZ62wsmwAG4tbjrLAFydrNKHRUlmQ3r3gqRFbL6m0ZvJfhZc9n+5N7i6sTfJtbqo1GvrGo2aFk3aH2moKi6+PcQE21MLK1sR1AS\\\\\\\/W8CLj73EvyAcy9ome5ZmgBJbOIi3\\\\\\\/Ds4hrHnelP2ReI3VdAHPcBHpi8eztqmij3NBx+LhekX6KxFM6cY6vUaIB9+lMmxPWoNk5RyLsoIPWCvwY74U\\\\\\\/5ctSPU7ROAKElCWhoj9RV44AE8z1S+oOKPqkV4cgmvIsj4PLpQC3RExpUt89NfOL7EOUQGNnbKb2R9VvcyLkj\\\\\\\/dNjfNYJPpZQPUtGxrL1DqZ5YF2rzpP7kzfALtjb\\\\\\\/dpmRP749\\\\\\\/BivF+g3QjAuiLWGwkYuA\\\\\\\/PxZPiN1L9YapAeJikeh5OoolfIw4tEDpEZzOrE7mW3RwrAJdB1ok+5udH42dLdtGrldKe1eEXLWI4vNKbKCGpfe1xWtAQ6h94laey0VFIWL3j\\\\\\\/Eu44SCuWEDfuC4x1YEktwp1J8soP1zp\\\\\\\/GFJGSYoy14XKTpTuOb8zbm2w7tT1GoCSpXIzihk6tA8dRtB0GK7Y\\\\\\\/G8Xz2jKEZ7NUvK6gdJUpeJdD+jpafBz2wVDnfeftipIrsEoDcOnCVpD8pnbawJOG\\\\\\\/6s9VJ427lJoTm+GMKRg5FIFs5gqEiwlDMqOqYxDOhjMt5EMJFAFrLPDozVlKzWED98OvOxWtOp2bGj9wBXxZhTw2xQEn\\\\\\\/AZVx6EUgxLeW22ZV3nw7ScSjhlGxt9srT0Hl1nrHNYPTr0ogrRWfAzzwzpmoxfcli0H3WiaXWCK2W6Ks8QiIb+W83n2vzRMXrSJJmGKe2bofWCobYwVX1g5SgMR7flTnSvO+sfpq7WPzCHys9KLqbSUfLpzR4hfn+zs18+sjb+RJCMMv3P5cCEOMW0OLz+zXIUiNcwoqa6KcB1MuB6\\\\\\\/ktSqEUpwXWbAjdPwajc9PWHgUB2jzyGOHLVpGtFS7dUDAdbApG+N66W7Fr3uf8QmYgR\\\\\\\/aDaNMjoHR8\\\\\\\/rEmkS0tW6DSq2sZEngiz5SBYY1\\\\\\\/nS2nTj9ZMlYtR6vvWmGeLRfjfmAS14EmqVlcfiqTqycSi4cdW+y5WE3NukWRuI2uh1658VWplDyx9ESuVat4i1VCKensjYDFdRbl9dof9kMzYb+Hgy\\\\\\\/vJdPpWMs431ynt+bqOPn37xDi64xezWvg9sZ2Cy36gQnHXZ7cGr4VIrln4Ef+u7P64i\\\\\\\/\\\\\\\/UwuR8xsFOcJNyPyrzhgCQBDvv1gKNqsVgeVa6Pv2cOtkQpUpVOZ4LR0FXsEOv6cgpQdRxibNJkZ4UwMr7Etw\\\\\\\/O6E7Z+O3tpr\\\\\\\/+m+ouss86nKx+r\\\\\\\/tBrQDICsSHtGZSIQSJGs6gsGK3Abn4CAmpvQU0lNVoKkLoNsZMIFoMEqk59Aw8M50kz2TksxvvXjnNpg\\\\\\\/piayI9\\\\\\\/kfd30wFdV8c1KMwG4nPi+ROA89NhhdyCk3KzBncFOhsFWQrOLbZ4SITpYV3i7SKSZDwjRyhEGPZidY72kAzdItolWs8tqvB2Ev0EIOLUWTaO1mP9+axTwLipkrWn++JktO3aU2nQwjVJddo\\\\\\\/70WLY40FVvvFBFdUFGEKjrSFIGgd2mnJWAGzFTlTz3+UH24XQuYz0wFQnxoOFLeR1yxNctWyDfkhzI+Pnu\\\\\\\/6GUZFl25JFqKeqz1dqmZofjDO1adE1k10I6JvTYJcPFzWNZjPbvtfMvA9R52fuy\\\\\\\/DcLsbdAy+gVqi6wrvrhu\\\\\\\/8dLneptiB\\\\\\\/Jo9D36RsS2nfuyBKSXlT8EB1PWB5h41LON7\\\\\\\/m8sHGWIuIfkQV7pKXUJRKbOWQW\\\\\\\/KI4Nv7BtpJkbUw5mbuEY4XbL9J3IxyEMwEpfUnI0VaLvdEtl2R+9MBHWPENGbkh\\\\\\\/iVsw954AVHzE7jlibnhLZhnr623GBby4qofckCGy8FrGR0PddOisWQ2ISslE1G0jSovAgUXlFCPKhdsbzFHg2r7AsXSITEVOJg14EAlZVrwmzq9eB9LE2x\\\\\\\/o1AlMoVL29GH4iEQqsW5t6pG+bC9jq8gXIAN6OquiVWYL+IN7hdDVscj2bTWSU1\\\\\\\/45jnJB2jR9hnuzAbaZBxuw4MuRWY1CxGNO0RzKUwLAS5Q5duvYHT+31ELqbOd1qqoVJUyvN8RUDVGte7YxGzqvsnNGJlygtNWHnC6xLWwiQWRbAKJStnm5ANLYK65rRh1NGIioiNKALIXCcW1sgPnDF4z8+0OeS\\\\\\\/k5JGgXXpwJhL7qcH7pt1bPcGKI+wmFMUbdn7w9F0LLoSOAE5KdNkTtKUuocouZnjl1pmjRkK7zcHOeHzlfKrTYPfP2aXoT39qSS8NLwjRfdh9Du15Jj1ewv8bk5ASUosPaxR3QT+2zzVohbpzEwk2QYTH1jmvl8uGeB8apkbwzfS\\\\\\\/4yWL4qO3tToM0JBXyWBkqAKGXkrghR+lVlauO7XN5Ccn9VPmOxk0N5perbSQ1emC7nXVmu6nFHWnT01ve3hjtpONmZPt+oIEfHln4pW9\\\\\\\/cwog9GhPOhMfq3VXabJJdBZMtYFqaQ6W5+NnXhDeS9NgXQuk76N9k5UxBTc8xK9WFLV3GJAvivtpc5bjlxtN6aXQJtuDWB8OQNKPr\\\\\\\/U7q+mF7HvE2tZWQiLIDQbUzHHu\\\\\\\/2INXMRBUsBNDJpa6BhNsqSJnzppr6tK+B1pHGryAAqoiLR+NoCYKDKKEtbdRH9MV+Q67VIKrReudYPtJwKhVQxoV7KmdG7ZDksOoGSOOvzxI15uGHGwJWi7I7OxWkxVjj31hpj0n98+6uXAanuBm7lQAG6\\\\\\\/ZOPZj3JGEnrua7lQU7Dz7aCWRdZr4txxuQu7R3sW542Vc7ym\\\\\\\/cGQHzTLWAa3PDW5FlFs82XRaXN54HJU3QwT0jHTQ8tZGNtx8v+Msz36D1qwpYhdPC24OvL3KeaQtWXMlLTgdJ1lAaSVlFayvOQYx\\\\\\\/W41O3OWarY0XEngBuz89RWsBr\\\\\\\/7gcjKPBtT6GC92nO0CcSb6BbNNDwYX9BoE90x9xbt7RiqcNOmLM+jnYxtFKqOxW\\\\\\\/Z3169y\\\\\\\/Kgb3\\\\\\\/dDXjO6sd8jKO0drMJIL2XgX4UYy08ZDNeQzFu2e2zzrhMM1xamzcgXPJTSTSesm82XZaSyEslaFcRTImRUa1+MsSzQbKTCrATOakk\\\\\\\/pt4ZbYYposrAd+c1h+fJICwCFniowdWe9quxSLIby\\\\\\\/RRnsgSjBqHT0PclTHIoVk3hWqTuU\\\\\\\/vyGwKtIn\\\\\\\/ZDa7Zhh15lGV+ZvDvuVgsBGcy5tsS2hy9zUA3VJbYfulvhKroF1QLqDdvkAGpb0xSPsbJKsEjNRWSlZsWFWQTrgFahCnuImvkLA61bkjbjaCVwpyJE\\\\\\\/SrGcTMVSkR3j3SW7EqEol6zqYTAvRcWmUWqDdgoO5x89mPuzjNiOSiViFztk\\\\\\\/BiZhYJD\\\\\\\/qNT\\\\\\\/pCyjv61lJLkGMvyJCJscpr4VTNYkA+lrW5vkidOXOq9MCDnwA01BP\\\\\\\/7R7U15lAvJ8dawpcSXgdhNF\\\\\\\/6riZlRuW5ZFU2k8PUZNkSftYnqTmIpZFcFWiYYPQ2wqCE+ATIFBvetNjCo7Pda1\\\\\\\/EbWhC0tgiwllW3WhFl6CHHwM52B45J\\\\\\\/zp1UpxtI4vLTqJJQ\\\\\\\/SJMEbz1CallprDKoFWalwe1C44jkNN8fru65pIKC4b8UUfHuXbHZOwINx9DIPIpi6Q+MGgZxba4azePGp4mSfSdttnh5vQvYkW60IT\\\\\\\/IPlKS3mRmBx9NI7mA7o3kKOayEOIiTvQIbLoyTULmRS\\\\\\\/P\\\\\\\/yeMyj14p+p\\\\\\\/Ez7rTdAFbc6rbnD3CHHL2V9CYC4XvLy1bWcPDIiAI+VgPtq8U89yNJa0D5saz1L16Zta0Uz2NOQFd4CdWlAJGd0av3dvkZKx7tg18zNfNX0+dM6QT2NqRNE488OeL2ral7ArheSJU7dKQRGkj0JKVPCUYi3rXnbBy1\\\\\\\/o\\\\\\\/41v2cNh4N0KpClTIST9qxL7Or2fUCzlGF+j8ufeRfYjBNgzojRz0Kj2Yc0o27YWLk4uvXTl3vnXtND8E3E7TpKNsMJ6PaTMY89L+ueMWO2YhtG2R7uYYpP9HGk8U4Hz3XF9qwWUOFWfCUr0ISzY\\\\\\\/WPF+a8NXxd6PHjW4G+MTorcv4juKwcM1if+G3CKvUYu\\\\\\\/xtvtEGMGwp0BwXP2wuQQc76pSL+0HQXm+mysw1E0toyQFFd2s83ThycFj2F7dy5GUOD\\\\\\\/Zy98elese9TBpopoAg5CcSfxcRN7nk1wcHyEJlCCrXgMToRNQOs7fmw7HOFQ+HdBCTHTawnpR0e6k3Z43kkaSmEi23cL7nk0VbdMc2kOaGfod+tp018u7sZ+q6LyZyKWhlhin7XY0Lv7MqswlBAjB2D3ZaC0kXZybWqz6ZnsNqwJxEZVeo3qruNLh+0AGUSCpHy8NwKdBKXW\\\\\\\/gw7l6c98T2G\\\\\\\/0rdukc2evQkKn9334x5ijtDIgD5Vsu9SojoqebRin4xVw4eqiciO2L47k5XRUGctlBtEHPXTiVedJD54eJb63a64QlbrnkjkDCavUoysOnzR9YdKjtwAsl1wclOkSEX11COFb0W718gttO\\\\\\\/35ab2b+il33IvH6RTRb9YSExaSmcMKwqT4XaVDOLwgKezW3cTBuBbQ6KtUbDb5O0p+cWv\\\\\\\/KJyP9BrOzn1V\\\\\\\/7j2hI\\\\\\\/dkOrqcvMHm\\\\\\\/GxgdIu+1nIZwBFI2MonU45QIaI6RQl4qdE5CwVzZLgijF4FtGgCdLQi6nZqiQPIz8HCnenV4Dm1tj6VaAEOzUt6I79DoL9SW7W\\\\\\\/smll7XnDMTPMrvet1DespMF+NsxeLLSqecAVwNtSuI0ROmvc4GF9KPlUw59XpQrkxVnTX25Fwk2AJxMUbUcviwX9osluSxZ\\\\\\\/hOX\\\\\\\/+iP3\\\\\\\/\\\\\\\/L8gkNWTXw5VMG\\\\\\\/Ks\\\\\\\/4X7Q8iiyZT54xHqNioUMvQ2PpRJh9xQp4\\\\\\\/3Kg4zV8noDcKFnzxFz\\\\\\\/aMB\\\\\\\/LXDta9Tar\\\\\\\/zGLG0aWfrj0X3ecoi4XFVKwTujytJ5X71e34YDynA5K60Fc4qh+Kht62Z1\\\\\\\/Klc4rX6d0GFANltStRz1vnbc6fkbTIpxN6\\\\\\\/0QKuBk\\\\\\\/ihDFuNQEE5ZD0MEx\\\\\\\/JyAc8fVSk0mAoXzdJjEDL6gsTgmeSX7fxAeOyEs2ntrP2Y++9qlnM4gR4wTe6GsKWIyA8b3ZoxR+f9VyI\\\\\\\/QPVbdFXfAt7Gip1gdBXFX3rlRtHE0O+chtFVsHtR6MvCJmoWL220dMd+C+NoL9S0QoiRk4j1DUrTjGh3CpPgLHc\\\\\\\/pXpyzlUt34RvFueIZ\\\\\\\/EauiUzPuBLxVGMSfAlIkTqlvhbnnd8Ev65qx5QlXYYOhp+aNKuEPLrnvcVR5RrPeNn+zd2pJISDgHbtM6GXuxGS0oKvaZoDs3m68IM4Sw0qgPQz\\\\\\\/pnnBt4TNSUBYzQ5a3ya1NZgjU\\\\\\\/xMd2ucBHChojQdvx22UyHBZdsi82JbSA6wqo8xoDnX7Bb5bERh1QT37BiN497TW1+OBCAm1jcnIpNdgbLOhrDirFXTxzYMyUtVtjlpIlNkAtwKTs9gRxZOxiAbf+bBh31KrGgB9ZPyAUau71Zhtyb\\\\\\\/f7CWOfMFJPaoFjm15zG6fysXlTWhOLl+R2wJ3\\\\\\\/q6bZ6ettxB7G1DOnzXzM0ok8VqaM8lu6mHotjyEpLvQ6OSvZOnQg5sIDjy2KmOfiT1buRLYcFQDgyIgKDdM1Dco5r76bgmgY5VHLxZQ8xr0aZu8sZWpRod+qURSVEB3LwLFaDNHw5eb0KM6fwXZa74kv+6T1LzaMlovRLkbJvnQnu6839VXe3YY49LQXS9uA0J+SJSEeYx1+AfvNwGl8nQy5tJuuFjXyRYeSErPUqCvfDvNVbzcSJtj0yW22YTi8VHZ\\\\\\\/Kr2Z6PYE2RELUUSOOJN0YPfN0gQGrQWBUNKI9QucBCNqs6nLKUZHuOLEUCmNAyTI+dPKfDpPC9XEliwsFJds63Ous5kY4fKLvxshEg65\\\\\\\/EhYzQrTBz9yJlLjrrJ3TE8NxrJdQ6KPpjrP9+2hFXTARhifXXK5iHBtbG1zXV4u9lhwNpYWY0JaEr6rsuKiA0psWzU4pqEZe+S7nLzDU2Vqc3hg6tVykPdrxfjWfpCWZQbc6PYXDw0j4daiyoilLG3HTxHBEoBlL+Tc3y48m1m3fQ8f3laFmQ9prVICtzPtnJlva9iqmnrAIKhw37XFGOKawYpdemQd4xJqa\\\\\\\/ZHZ+UabWs4eo6AlGM84oJhK5wsdc228K+UQ3TI5zOPbiYP07NEdEvpYaOAwBH1pEze1ED3+z74WWaC4UVPqV5Hjq5Q39XNpCFCEq8D5u48AE5\\\\\\\/ONvkOV+fGWJoWj5OC4ckpwcWlP3czn4tQS3EYPul9m6VeoOSJmfWbGvdLkrSXjhwmqcNx5sQSTFNhlz7fzR89yK30EipPM3+9Sj5VPHZ0wCTYqhhMUbsHMCldg9H54es8KzOD5btVIwkwK3ffkp6h+i5iuuBi94VusGd+G9Pj2wGHu6pHfz8lW8Hlj0rKqPG4PkfopPgCQRW+VvL2huOyvcWOEhA7q7svupa8UCD83hpvF2ZjQs76bduGbSAsQMHuYxNBm4wwqoAR5gjXfYe7O9sbPSkcKwMDLS\\\\\\\/VpdXXpJyjnxunvkseJhZ17EEQqLGr7S2TPz2ikwiGruDylX69N3fLl7fRKr6myR1Vj560HopCyaAmSldV5szuViKNgURaWnKG2X6\\\\\\\/XV2VD5z+4PNWgTawqmKEwZ7KkrBuqqKorY5NtdZjbhlcq+t+7si5bCutgAkPXPNnNVNzG9LQBYXYf91\\\\\\\/5y+bhXZzg9WCpQyvAyfhMrMsXEM8hoNp3QHHy83daghLzYfaYU6YJDH2djPXI0UpEGWDsxM6JUkXfh2QVSCtv5d2LiXboROiR+sCDAncp2GYi3KYqbk1kiUOosXPwR+tNHzRHjDw87s+hzQ9gTp2eYUYmn\\\\\\\/imQmuR3xMtqKDReqaazLImPnHPvngRr7Hox1vIXteJBTVLv1ec5w7Mx0HXD0UnTx10j12vw4c5ZpETH0jwHaEPEwRrhugdknC4szmTcpsTDU6BAde2L96NxCHfj9R39ghKTGOCOqdi\\\\\\\/myAoeD7ew40Doz37PYqGHwynVvMsmKECAAIga+966I+7oFEeoUzaFIjG6ww\\\\\\\/8DCF3TtQ2JP+pLYPWWDXE\\\\\\\/fvypnThOD40bX3xztJHYyoz9FRlfcOKYT9RoKUiMj5YNuu\\\\\\\/ZkIti7LtOYT4gqV+DzX4xEmZupsio9dDIV7wwS\\\\\\\/\\\\\\\/gVLVYrScwTYvskIb5zduk9BksflTZjAXPf5BCth2DO\\\\\\\/qIGyIQX40sicMkYN4WMbP7k1U+MCS7wJHfThK+OrBu68gf8udd8dW0y1jHWJh5FkpttVHxhHpVjIfhnp9mBoo3N0NiXrvH47sgq1KtFQnUbBJtxPICM5Ry0lXsm63ebEKYqCVJzcPzoAh+MvT7F2lj1NrL8cDVhk9YWDGeDSktgeqaJRkrPvyBrPkp9DhQErfj25RjmvSRwa5tZdBjwCIhFI5s9DfO7KVGHJUnKNSZrG0CeGBbsnzyar85oAHo6kf4YLtQOddyw3fKElMLO0F0N3e9SWb7iABp5388r1\\\\\\\/+Tr288ynlHPE9VtbdOth1tJgI68EzyNhPv9THCuuMSjR46D6JZd\\\\\\\/vpV6XcX57bLeCQDTECvfk7fzT7uRZ5gv9egYLeIDHd9o93XicDMwZbtqsBJ37xocAp11qy4rJqXhfojQwfY9jxAp0i\\\\\\\/tj1dwWBLeyV35mS9AiatVmNk6OlVhHItoviwyZLmCNi66PHP3aorI4SNBqmL4GH+kthNey\\\\\\\/InoyKgpU82S24ivboSGw2SX6JYPHG2p85shdUiNN5bHB3Omhpc4qpwwJylTx5nIExqkxKjL7StXx+BIoxCoNQW2eRhsIfB6Euy3WVxt7qhDaZ5KX9gbzOf9qJUsQYWBz2+YhsJNE+RFwbjpsLVjXnFtcsWg\\\\\\\/xDz0WFhBN3UMrM7wC9nXnLGr7rt\\\\\\\/3m+P4Pk5tTm8J9hymj2URjW4g42CL5dHL+8fIjStvOZqoFc8euTgSBS+GnimnzMKFZd9VD3gRFP1WU6ADDzrAs2hMHyA0DuW\\\\\\\/h5G2rUuYjEsJXjxOsRJDkhvc5AYgYbKVlE6+2vr9OZOs8pEJnZ3skYTanl2sG1g5dMMJe798Nd5d9EIAPtw14PaUUqezz\\\\\\\/ks7JgYlf\\\\\\\/XMt8ev1xs909axD8AcMKtrN2nAfkr725JAkBAP2X\\\\\\\/sW6bHoRToawpki8qxGnU90dN6VUvwIUp3aZldrlChktuH6A+ZZ4q\\\\\\\/iE6Hc4zea2QH2hf7Z8dQ9i3Jcu69nHxT+eeQOSnf\\\\\\\/woi9X4a05JIlGJsEubHGTP6zjrfLRl0kpH8KqC1wMbC4F8P80jIVPjI+SHpopinhIsyxexb\\\\\\\/ZOO61DfdvDJ3mg9w8IE+Jw7mltFP0K5GRIaQppErcd1wIM42mgmRDpOEBOV1XMDKguEePu5qlSEZR34mj7DfNq+ZPjcfAVbF3HibFnRKHQZeiC8lCWSqcRnDU6mF6d7DTgA31fYKPg\\\\\\\/sk9p3Sj71gXxM7Uqm1oRDefqoLRWXVo4QXp+IPa0cWXSgLZBay02sSoJMscmgg+pmj2Dc+hnXVl2YAK1vKK\\\\\\\/Oo1H9lUYORGdvtZflo4rqzjLaTp9CCi+mlpOP1tO5Y7KwqFaGxtuleN6zY94Gfrd+PFlKwuQlf251pxnkBl3t\\\\\\\/KtN4GRQ1lnnmmRsdb6HZ9B\\\\\\\/dUQsjXRvkURZCf\\\\\\\/INP8hhTGHZFevhYLitHCVnTHznr4i7vrpo1D75Wgx4mjpr7u05dbQAl27Akzm4O1UtqDvuW0LDYSQmrfvuYzZb\\\\\\\/aBoAnpcjz3XWckI5HzJo9RwjoV05cDmSpOQNhp+CIccr3aLY6fe+YrDLh3YEhjNP4Bh1OlFQjd7REtNaGDSJF7HUWi82sI9Y153bFTOvVhc118KapP5gnmVP+afIE\\\\\\\/jf9\\\\\\\/EEEj\\\\\\\/4LkftcIEOIquLR099NH2RK2TqVqzVlwiLvxDwMd2myugNbePpfifPZAfCMFe4\\\\\\\/m8r01C7CjhW4y7LpWAQ1LBDuLm0txwkYkxhLx\\\\\\\/0eU\\\\\\\/coY4AtRcgCp2apasGlEkDj+4Oz+KhxXNhlnQt3x1bgy3dzjUBag4gsAz1h8fuMyIc6s+ooF2Yyej4ED8mIilKtTgvdBoiKbMxBmoxGbpUg5ExUxICh1GsEm\\\\\\\/DZkoJx13oJo7ZCfDwUAZwZg\\\\\\\/QnVK2KFS9isaifNa3XGa\\\\\\\/1bYbNya3acnLnRo9R+yHZBgnyfNd2vHxbTEcUgPhhetsZhmd\\\\\\\/2EEPAHJizlxMwd67n2hkm43BxS4cMLkNXC7q8QiSObX4D6gJZzYp2Evhw3aFhuyRUSqvVllUCF0ytAWI0ur5iqs42dBndm+AYm4oZeEI5+jkP4xFP\\\\\\\/0RFFw4O46Tw+WMCFhtZDxzKmR2V7RwDmqOoaG7mdoP8acMmlhzs+nz9cojOdOCbYSsKr9Iqhvwulq1Ow8+r2tKr8wncxJHdXkweEW5\\\\\\\/T713TZz+eKrPTAc24DDBXcXCbfCVAH8ZaskyrNFBT88XI+sIQiYyiNsztfyZpbewEYzqmAEplap\\\\\\\/v0gOdwaQZUGOftKPQxlXw5aHjg\\\\\\\/t1u81fQQgbHKdbFfNGHnpLnp0jzRXRQhDhQWEa87ClYPdsVe\\\\\\\/J7CuOdjKyd0Knte8nNmnaYEoTMW2e6crJZja9QqCYaQFM6HpjdxxZICuzAv+tMrXT0FNZngZMy7LWXojOOWU2GpKCJ\\\\\\\/tE76Bw73UonFaJhJFhxhjxDBCCblin3vyRpa\\\\\\\/5cmvb+6meayxqeTNW+Lmvs8e2HDiuCPd7o6sOa7tXxG0m1D0UrrpPtIao0i3ZhxUKdnR6FDirwGXcIvvgqXthGLxKCAY9YtWaV3AupxrxOgdEnE16iRXSE+I0T1+\\\\\\\/uAtN7V85EzYEWkEaE3nE\\\\\\\/wlBvo9F77Xyyg5v0huFo0bCOfdxTubkADRW6I8Qd5nYRHh7O4H\\\\\\\/RBZufBCeJI\\\\\\\/V6OE6UnmGqlcTVwtGicxyjBMomvzU2gU8bvzYshoAtvWwTTFtBB29ACR\\\\\\\/tsJlbSlnFKfrRdpVzNU+Mp2dT4D4+\\\\\\\/YWzevWKbSl5gfsO2Zy4OEOlGDg\\\\\\\/vgg7GSVA4pu\\\\\\\/xH3YVNXrwleKEjrV7gdYYiAtkDDO\\\\\\\/n1l7xGJu0V3TLu2dB+h3hapbpKbnGQNpx5uvnqJapRjxrVxk8qYZLZVbzCmpq0jDZZ6UbQkq8l6TO+45f0qaxGEgc6CbPJHsBJ4ME0EOqNxgeHMlUe3\\\\\\\/tgpoap5RE7nXEFE3kmZBxB0v\\\\\\\/jeeWaCWAg4Ad+V27yQ6ZQWOaqfTznbRzvyy7EU7CkJa4kseraYe3lcy6\\\\\\\/mOclXLC3RodMkB54OSjG2CFjh9tiHk3UXAu0ZHT7dbp6OdrilrmGWxw1t4Mq6bn8zJ38jc0eGxZWOZP1k252IuPjLMgMH2my3d9Tken3z1fjwJugfA5kpvFLEEHMN7DlCIR7seWvXyUcSNtqJVMgygMBfUEJZ6YROVuJxDWsHeAC5ujNcC58BVGaZ+64cRrJZ8OWuLcKJ7fQk1Ye2R8rrPO010YAAKUDENZzswmkeHLNzV8f\\\\\\\/EHCJSjWGXGRqSAKBGDhlAAv+dzQcYe8nVvxHDipC3t4nGW6O7uBFO5fR\\\\\\\/3HK7AdUJwNQ8ZcM7ma0pihNzUyHa1QLuSy5lHuzMwpxiTZa+4a\\\\\\\/1s\\\\\\\/fGtYL7fOW7a4vsUMElbGZyrnM\\\\\\\/TKI5H3WzrVirMSRFGmwR2MwuxkZsgnUvCexdSfHgLYQKl+Q1I3Hhkwld13pP8n2jHJ7rwfdnfTZrpfw5AUFW2JNpQKZsGn2Zx+h2zGm+ovXU06UmCgfF\\\\\\\/gLknjW+0Lpei2krKtxLnr5mfGHxHp3Y\\\\\\\/OJ7kje\\\\\\\/XWCKma2YXn13KlgXEeRUXwqlItt6hkn0MZj5fdZ3u+hr9EOaT8Apb5vL4E76oLYn0dWgz1D1gg1i2oeOT7EXWXLG4zWJd7faFQzpFxAirAKTIgOvPmRF7tuO3S0h6m4kWhyVhxfqV8mbCEuFEEfL3zlammzr2ulhs1Eeoo4EOls4ebasih7EOm9wCPCWDKvAEubF1WMCS2j3IWWjuAlSrIG7AjPB6bgPJCfCQS+o9LajkolGF3XSS837tT+QKq9F5nZR7x6nI7BdFnh7kMMBxiF2HlLV8xWVoHWksWO89x9PNJZMB\\\\\\\/xFKZQwOmVAYLXT2MHUJNThx2k+K78wYdd52yTWS8hi4PGL7bxlMJdoHKI2yyh0v8AgJQLAzLx+gn3Wtw2FzBe09f2eJ+sGmwtXEvm198Fm0Sh4UGGiSa4KBD\\\\\\\/x1gdpW0XUTRDxUJNfbLQtvR+vULN6ofX3ggYHS8N25VEdJAsFH0bXyRqYQO6x2aeBTGu3+jx1CEQIM25eJOzA3HVEvtLExRugvKLs\\\\\\\/qRrF++\\\\\\\/WEnYtq\\\\\\\/8H959XMq14Bb\\\\\\\/VgN3yL4z0I2F4siZ+0eAI9\\\\\\\/4YKc1ba1HvrewWx\\\\\\\/47PHk5nvSjmv37QmBHqBk116KMF4JMx1C+gfNs5PSY9LbOIGWPzvuTBPAdxFBaQ8af4fStjvTyveKl7PQJWAL9oTvj6AdO+H06NjpSMCOmhyUfOp7GSlCxgx4KUnJ7NED\\\\\\\/5lJ4+Yan9TO4uQU6aa3Tm+meFM3eeHnZR0fPwqRn4AnnplySTzvNh5sg7a+GwzsDXkpxas3DN4H0tWLMpOcV1x5V9ujDKppx\\\\\\\/3fcfRRdl5icFOR5TnMkSjMuMhBwpf8tGyvq\\\\\\\/57my3Nws235P3MvFP2EYii5lK6BqNfl+cpCvukq9HB1CxwiHfrYjfJBwDT3b7Q4ZFIcy9s6F7xDCRlYCZH097XZEZhsiOxlT1ItWqSyHNwqoAqUZniCGGf7taDBavUWkXZXlUXwlhEsT\\\\\\\/YOaBWj3cotQYGgjGk4N4bm\\\\\\\/ac0is31PDnKmNWzzVfAJxoFV5OziuhIWhyB+hdpj8RwV93vCV0NibIhTc6+c7ja7KHOeQGE0\\\\\\\/MudhUIfmj+I5lOoFNlSk8FZxNkSjjYPkpy6yzUWsZa1Nz242usvZKcPwtplCv5XculHKrnP47T1jgB3JA4syl3Fw+Wbx5YSkI\\\\\\\/Xpb+AFAleiQDj\\\\\\\/epnAbeW+vSSg+7chj2SVP86obPlwhWNCfMtI9f4xCTtWGzqnQl6hGc06t9BQGAA2zTewqYvzZ3oV\\\\\\\/s9QzCSMLmuPwNFAGEPrcrDKXPOX9OmMZyu6wuu9fnvO\\\\\\\/b\\\\\\\/xPM96b6SahnXjxgd8xhXcLv3HNQCbQUbaMvEOc4hnzM7+nkdwbvGaYnHuEC5qyuR2gukAKc2Ns5+u66qotS5xhP3KuJnoqa9s1WrMY9CT+RbqyWO83+\\\\\\\/pQEtuunmHPiat2cjlg3qpYsFYZc9cg2DxopmxGDuzcRcai9eO51ESctiALgdKsUZso\\\\\\\/sK0311j5Jw\\\\\\\/JjYG0LUP7VQmhLx5xRIaDfICURp7AaUtR1Gl4CUBROodxBGysk9F31H5BCFki0t+IwwmFxnrYKhasBz5WRx2BdlLXxyehIXP\\\\\\\/CjSoGSriOHjoubApQrpwz4rRD8jMbsnFETujQIo8sW0GR+cRms7XY764uTSMwY+NcEqpNonup09rYnBrvEsiganV3\\\\\\\/yuG9N06XOvabLQt1j59arDKKM9Y8M5Yi5U6dhqtEwFjTZs1dmpTheDGq6Ez7q9rw7y5rw2GgVXhzwe4xP6dTPW0ujjdXDs+DN4qhxBcZzldm7cMWhapOUyderHs8TdDzxHZKWYbcWXJivwXjgPVdifncXEkpyAk7ZKcK8cyymFJeSzh74ZKyA0sb8n0KL7LhuJv7A77PqhqJzVBUf50hIvAEa\\\\\\\/hwPimZDHVLlJ173lW2pIS+UlO4EGFoxkjW5yQ3lDgLmCdOe+H1txMK12MwLX2a2LhSXWOwedIXZ85+Q+JDOZcw5tuHsWhconIoQXHWKi57e5rhWkuSabELDmgI1irdt5w7bjmUlDdEQitnD5b8FIhFHGpuzsHbvRgKOxxklLh75Ipy3cxQ43p8VQF74fbhvNHm6CRhKg36BJpSZLe\\\\\\\/pB\\\\\\\/FnqOsE\\\\\\\/9uGGeF3mUDBrkYjpO48rbI+zNOwsX0aFiFohK9hsnQX9sMLYm0OH5Df39vE3GYAXFJltqoti\\\\\\\/dXzQzRKQwVvXBpflbcIYfNkAyAf+DnebmpqWghaZKmTadOsrhmaqsaCvaSUlog2Dv+SyXLzVvZxLJFZba86jU9gLwdigXFwRHukLjlUg3Du9uXgj5d0PLbGzLKp5OAAo3BrzwTAyJcSqCARpyaYqD2fB8Y\\\\\\\/E\\\\\\\/pvVtwG6E\\\\\\\/i\\\\\\\/iMQOJHQfYor6IqfyWzpg2A8UvM7qefdoFv2RGH2DXF1QfPosSo9rR\\\\\\\/Lujm7BEISFHJD\\\\\\\/1SG2vc3CGXrZ3LD\\\\\\\/WylBTIY45ZJIuHcb8Qki+i3bu6zlsQ+8hgoAlLOqwWHEzhcCjLkP5vqDp3YDlg5wgNQpmWrf1I666sKBasXGRmC9qQvBcNBIxgDTrYC30xvzAsm2rF0z1LV8roR5RrrkKDxZbqp9CoSQXa6yg1IPBBjE2oq+d22H8vIjXBKFBr4DADt3dt\\\\\\\/O9N+\\\\\\\/GKPUt8z7DrzypzfIUNPDzkxhGSvw4NnbvIozpTqgTcWjBWBJ9hYCC9lfwctedZFGGWhSeR7dZItIsXRQDzl1ODdlvOdszKEWT+TEvLgVi8Z+eDnmHMnWwJr+keXXd+1siwlDuYMVJ91aKrYvbScxpbF66zuXVDbWuxLOyj\\\\\\\/+JLTXtQe7sMKYI9HYtU2UDa6xYloHKthjFunhpzGnYkFukY55OCRLKJkDSl7JqiA\\\\\\\/\\\\\\\/zSQop5BEsXIgFDuJVWgRs8znPtLied7\\\\\\\/NywFwAjE4DCyNnxnXrbbPNM9m+3lXc\\\\\\\/SJll9BnXoElCsbcmpoOGTajEoE6XZ3tMX5Aqqxn7Fs9BISFuAoR1Bp1sEqnTHyVhQHfpmvNmmeR+1vijjAR7ebsdpVxyhEcSjVvSjGCwfbiKUiY+EyjMmWW1\\\\\\\/JZCfHBvXQw3Ha1poTgHhy40w2O5hA9ryj7C7c7U0pbtTB6vlxTDAwuDZ56ZvnC+jIW1eRv4GGn+djf+PuuW4pwMNkJcZTDUI1Ow6kO9K4nekGmyV9iqooR3WGMj5kiGuedQgrisIfaT+n3fZ9vVYnuXdvwBmzecADqY\\\\\\\/tk7BwNDol0VWEMgRSZNdxg+3Cyue\\\\\\\/S4GUNEn3QAZyW7jR7DlOpXEEe57vkkTUz+BHtbnUcTyrLTsI5ZJYJsyix3KsPiA1ZVFhWK0vglNPmnkHyL7FB8T1EHYOlJEL13MeZqFmpfkaH\\\\\\\/gjgl5P8Ys+joC7F6xedBZGfjRDqcT\\\\\\\/8pJrMTD1o\\\\\\\/\\\\\\\/HW\\\\\\\/x32VHbWyPnR+Ftj+eVxhCR3\\\\\\\/8xqWpsmdYyiQEP6ATI4uEvqUFAqAep6L4W5nQx1bzShpVTWFvaRFxAuD1dxob8uCBixgHnXGKS0dmbsNFrVk7CuDVu73Kvm+tmqjHqIdw7xz4G4\\\\\\\/yI9CtO0rTwT606xDbWuMpjSrWeREzvpP\\\\\\\/+WUgF1LgmalALJFwq8HyUXk\\\\\\\/\\\\\\\/Ljgq\\\\\\\/\\\\\\\/JokvqTl2OaA6oFSb5x1+zcesFzC5VaabXYDQiHgUOMML6pFPdzRzKHzzK+w7xMpT9sgteRrnqJ+SYf4Z12fBNI\\\\\\\/L1H+RSVwIwJkHi+kdfgF3XiRWEyLg8pEEL1ivAOJsQjDs+ue07O3GPK2xAz18x0yrRGjZUq0pywQm0zUaOS3ZXm0KfhnszrPQpDbRPAdV6zftuFJS42D7ipNbBat1OtOZKpIuAI9fNfiVt8KcEj0Uq\\\\\\\/wgKKu7JmqMfOXuqEUBjUYtdgH0aVv9ipSTkTNq+I96igHoirTg2gDjUZuG+hKsVn4ebQyi\\\\\\\/G5X6baL0kC8bA0PrlZ7ZWuKVrfwSfHG2T3qpLwCi595LNGG72HY531rb3BMhg5g5vbTIm0ccgaUzlWKwDv0R+yBK0vYQtIvIKS7vicgKoRRhsVSYYAX4wGOrqtq08iocyDAeL1t8Kjm25quxZu6Ds4HK4aQnXBWaT5Z0rQymM7NdFylnC8AjIytd53f17bDBWpysw3jnZcXkxdPRUAjRbSPHpgCgAOWUnnXTkgL3ux8FTvjRErwOGKQzMinxEPUs2epYQb8+6dgwK97J9IQuFrjSDgnLkB0DEl\\\\\\\/YCW59ufgqD+InqXywjX0ejKtnP2Yd7eVxHZCqCz18zfGYH8YpXpym+WWYabx\\\\\\\/Yb5g29nDAF2aDL+KCH5fLJDP0viKsDs5R3TO1HwpSv28XAEM6Dmkrq7TR+EKeiWlvsXeU08NXI3hI7ks3aaBDk2kibgzQOrReLudyicjiDt7FSTem6LuNANP5fSuhhX8S3R\\\\\\\/ZsyGGsfYsYZtkp4OJoGe9gR\\\\\\\/e5tCGYneH3uy3LgGs+hXFwIPV4LuL7Iepj5L9WnXZCGbpQHB+VPu2QSB3RGOoGUAp6bl1WFkqHaG4jmI\\\\\\\/OkGB47yG3gHipeDypX9KqfO\\\\\\\/uM9CIvrgTyJyo6DoTWYAhNeVmzQqxD3B+E00j6iJvk4a1EPZR8xYoMK\\\\\\\/ZbyDPW7Og7WZ4fdlKqn6WgSh3nSvrLWXPC2SHH1d7i3ls+\\\\\\\/M\\\\\\\/X8OiifjDpIQxTLLJlkIaZqxOvq3CWnh47BSDBSUUzoPVsImPNGUz2MxBNRyuVo5qJbYSeNXiC+saw8vTMZ7OOtgMHojHK\\\\\\\/vY3\\\\\\\/hH2+Sc7A\\\\\\\/uDLU\\\\\\\/gKe28IYr6VfYPbfxQgdim4o3XE\\\\\\\/EP7z64oCbK2\\\\\\\/jLWq0Xyi7G8vvDCgIC1RVNePTLRJq4xHFsJmjmUgjhkSVQDUmGZxrQWH0QoYuVz84H3PoCp0V\\\\\\\/k1rF8jlZFUxXQXjD7\\\\\\\/XP0b9Z71Bl83GoqYK6AmrRKSUoTl5gZXyvaLOKfIeGeURXm6wfKXJMxlaTzv1m\\\\\\\/e0jNO3F9tT\\\\\\\/bAwN6d0hUjmCszb3LVhOzLHDX2ITk+XNj\\\\\\\/WDLqc6Ogl5tEiRlrB8lCrUY+B0zAQPSMx4pDGTFwAZYqxKwhxMVmUjUo98t75PmFAKmgZxS8TGtrJDMdKeltoO3qI01oHyBoLR4gZGMls9oI+r8m9c8xx5ZrbuzTgM43aWRHDn1V61ZGoJ2AGmffraVkSYH2VtgpMcGl0GKx24ISr0wOMfsI6xkPPm\\\\\\\/NvchZwxdT9SDRKQmwGE\\\\\\\/+BReEjG28D+Ceb7Lk6KEPvabUcHZ14yJw4yRf2AGe\\\\\\\/qNeFJNhMN2cdl3IPQLwjdkukqHM\\\\\\\/2umlY0Lxsq2gxH6zLZbp3sWe8YGTTcF3lNwgC++bhacI2nUt1UqQIfzhN\\\\\\\/kmnaHph2JfnAxfWh6yuL0XCLrv3vXg0jQbh0+qpgTPrdQtaKDbUI2N0kIZ6omGnbVCaVaxbXwS\\\\\\\/7fIyoy3IWFnWLXHWlactbeHFF7Nw0XwV5VnqG5X59JnhsiEGxbRLvux++Tgt+mLNVUllWuxf7Tzz3Q1StmY7zU69DKJf8TzUvPzGKyhSQhigw7gxDoRkFi3wJ\\\\\\\/Vw8gfS2AsYM2zp1BL4ikH5jfjmk1LvM+O9BzvY2ZKV7VfVQ8riH7+7glg676ktwCrpsNsZx4q+yA947E4UE3CY2pBIlRRKOMbDq2V4oj73Bt1lm35op9lR\\\\\\\/3EZl1PiX1sZsIqupvzlFwOljGLYHPPNINVPEgZEtHK\\\\\\\/JqrYIdRYjZXkyn8bXXeLUB2IVxyNGxxtC2u6vFxWlo9p00pclxm9JT\\\\\\\/CnlwNfUuS7BoWODQtHVJqGF5+UcpHFoy71RLO+bme7\\\\\\\/j+uC6Gzuhcc0h4Q9mJ\\\\\\\/PwTAuIo6Tu9c+0tkSsOyypNvvPT2Ts\\\\\\\/7T3WsvuTWKKyxRSI9RaJG79y4X6TzJIwsMIi1lU3UszpIvzw17L6wOusQLJ\\\\\\\/pM\\\\\\\/I9jU08DnP\\\\\\\/LtWD0b8spf9o\\\\\\\/20Gprm8JMN6J+M6haUpgcyh8VdBXgCovTnV8ImXEEdnnGseCTQxxHDh9SjN9V0Cwg3D4WRTk9YXHrK8lbbSAolzrFJqrnvasizxT6vjQvVw6P+R7PL7Y2Mpz91MGE4rKYhPdD9jwqHM0K6YFH\\\\\\\/xcCO301747E7AFWMKKvP6WIL539qEJLjzRbyeB6cOkpvUt3YcbIUPzpku9zGFtFkprsnyAcbZC8IaJonVjzVlvKqMug4BNTWg\\\\\\\/yL9owtvZLX6G5\\\\\\\/RDgNnjVmOh0LWYoAMF9PjP\\\\\\\/G6g\\\\\\\/38E7BpXqBPaHpmAcBQCd0x1BLcBCIstZSARzxvFtCOXKDHXyA4xW24Qdbm7ICa9W7eLs0ojlEvWnuvOsHzi95FHb04AFl4kMK\\\\\\\/MImrmTn4B6xwZRvB+5vJSOem8+l5+B5pPduK44LWDuvsBR76VSQn12Grs79jYxD6nRn5lZSz255WX53zE8j0oakx1JBrM4Qtv2yUdb27wRLSDv\\\\\\\/J53N3KM51JGQHWp0mVyO8i1GkH7GWJUjTiwaFgXYFJ8AUy+8F9Ze3jXo4TORcPA4FdHTyQQ0DXqMK1M2dzS8bWM9gzh\\\\\\\/F2zAVw7022al4gBWGU62s1L18FbSQZju7bUp2qiM90CTmPUIsXcfrnNt1krOQ5GJo5HGDZYSdKYOvPPK7VT3LlCeMjyWpghZKUHcqTqi1TJoSJtqQVs6FG6s1KLiBgHNEXZwOlPJ7ao4a\\\\\\\/+mROOKOPr8BMr70AXFKF8JyXqi9LtBNjsTWKkwhMMwKIAINakBxssbbbJn0zO661veaOzOEFzK7l6I6W6HGj\\\\\\\/fw0qBAB\\\\\\\/1pzwiKAhpJoZ9S4hi8MySoSzhhSOlGNxovucfsLPtbRW9iKWgrF\\\\\\\/XSFlPNc3VnpEk5fWIaTfskJn3hP\\\\\\\/FlcY0VCLc4aprkmGlrHGFGAO4O1xaNVI9zOeoktltGHIDJGiVVvDtpemCTNZzEr2Cf1nuNBDseMtKmSRzoismdXIO3J\\\\\\\/Ipn8tcQwQreCaa5dDuUFRhisFH4hEXTOwF+JusCL2Yr9Ynq7xQyuDm\\\\\\\/pYCE9SlAWXGglDZsybH5PEizeIuK9O2KhxvO2yRaif39IC8bDbyVNTbwk+mF37toYYmFNFPye+IJYkSWXHfD73Sc0KLwrsiQuGoXmvaMzA1IPI5SGEHcgOqfBl\\\\\\\/So3No8hPGjj\\\\\\\/v88BEBqpjqm8R\\\\\\\/+SaqBtzzvxcR27NkpD\\\\\\\/Zy6OzRUUffGlVx\\\\\\\/uxXi25pETYZkf0AW4HC3rgk7K7do2Xyd8WqgeXmhElG8l7yOnQI4XHWjHeA6oO+IWTtbonaHoZi4TZMSiK0AYuUqqxUZKDnIlOK13S8Xxoil4AxV8yt0ZDNfTJqMmwva2zu1d9DD2RpMYHYu5BEcVIRqme59q1YOW+x8BRZcozG7kyfocHPuseR65n7UvitnfZADm9nlZ5lgFjXOdI+uq6LS8x7HBd+bFBg7saNmad+CYO+assUgWuAUnRzMnvEcr65rD5vnKGI5GxoKaoQl+zS42kCKCT2tb6sQiqsEPi5gQAdG0ehszB3fpK3YgNhOPdngdfn9T08njY13nSoNenEi4of6GXo0O7DmLGRnkPtILwfqEu2kDmqL5byHvbp4TL2DGHZL9SpMrbXtbvvEHScLgMp0rBg9+sddpYSCt\\\\\\\/ccvj9o4KJXRy8rVuu+PV26jcB6qbb9n8uM88tFq3S6+Ch4dSWAGUDknbHc5lswbW+NMk1Fo7dvEMNoXbKgmHaUbEwr01XSTZa0kaYF4tGgppjVDpOC4M9PDUqU91ZMp1\\\\\\\/K5Mx9FzQZeEVgE+VFoVZp4I6zCzBILz\\\\\\\/57B4rsIvAE4rd7KLJ9bncLszAXEicFuwK0xZbMBT5jByKhRTXbADDLU+lmMyIj4ls4h9deIX3j+Ik9Izu0SZJG9oZiI7juYPlDt9T3R4N8kqBy61nOi2WG5Swakknmr+FAfZ4SC8EhJiuINaOLgbM0v46YpkptNzl8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kgaU+CeRORNGiMkrPvi9YQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fd651eb58ca3e9509f361a3e097d48d5ab4a88aa3d28289d8d32fa3c3a17b0cf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221W1Kgn0hiKH8j0lVjjjwd966VD1WB7sIio5bfXCYTznfwNXAaiujAiysHIeGHp49n+i+rmnzRKooJ5C8W8BMXF1KG5fu2ypdhj1pDGMtyAiDLLFWBj\\\\\\\/aHJOV3f2JQOYoUGn5lWrVLr0MVZHPml7\\\\\\\/3qfkqS1zE4Y3tDx1EeDj7a1OF\\\\\\\/5sjsin0Xt\\\\\\\/a154CAuu9o2B9x+MwC5nD8TNRQKRA80Vyi4raRuYmjWYnRIQ0pfKDE8NnVWk0TlFioKuHYi322zlzCXFY\\\\\\\/VIPvCAprvr27ozP1gWIJwE7Aw+ZK2BP5M96UMwNql\\\\\\\/rrAP8JQ2+vVNugm2uqt0pL3I+rmsC2rIftFi\\\\\\\/6RM1KUie31HeWl7eScLSCutR5zBXvkghc+JAArPZLdfoA1hRsFw1OXP4XczFvtvgRRK4MzrJYnPuSqnzAxRtnacQiBrjgSq2vZKJ4MlY+40Ffl5FCrn5Px3ZbJgsxBX1kTOTGCsv8\\\\\\\/iEyZI38wU6lHC2\\\\\\\/arkTbwgrv+z6MrXsct79mu1b9h0ZLrxnu5fQlaMuNvv9Z\\\\\\\/rJJHs4NT4UhcPBasa5kty5qieiococVDLU3ntLiUXIgBojayOj8Vzwfc+dxRYwGOdShbpm806z6c7gUbre5tKsJTpUcbRKIMi4jpGQ9rgL5l8qZcZx2T71FZ69DoD1nIv9EanLBbm6fbpy2qbRCw91\\\\\\\/749qKt6sPWgrVXzNSaNs8U9cWHeQmIIjB47yFsGLiN23vvT9yRa59MHt\\\\\\\/LVj2VZAt6vigtCz1NGHb+rqggZ9zCAsHYLKx6x5+32qAJ6ElE2QSo0Kb8JU6KA4Lp4BnDB53OlOOLAezPjsfviX2RuwvLrk7wV7VL3wttgN3BLY7CoPUVPA6X0H11Rf5CPT+Qzu6cLItgEBdpQ8C3w+x7ZgDi4upkEf8HflUnO7bYuMTvV\\\\\\\/f2M7pEDxiUjp\\\\\\\/YVZ8xKcY\\\\\\\/s6iPE2et3PmeM3tKXzA+dVnqXNiWhkG0U5Va2mFoIjdzrCRjUwYpq5jrmcSmPsGUIkao+AOVi8ByZrqbiw+sSJncNbCDkNBfAofpr+v44Rr+W1kbaZdmWH7fT4duL5Yg5DrZhDrNge4jpTOuTFl5T0FcZBcpFY\\\\\\\/MqxniBh\\\\\\\/sRM9Az7LjR6hw7T+DcQYGTp63KOEJXo8hk6BLBzyKxvy7JQZB+Id\\\\\\\/6xf4DrsbUaiq+G7rH7JVb5W+GVtOHDPnnzwrPxOHnUSxKMPIFniC\\\\\\\/Eco83QHJVhF3CL92LTqcmVSfTnWxOFRbcmUQpTFvYp4905u2TTAixt+FeC9k8HdsUJqrocWREr2fSzlmoFgrOiNHK\\\\\\\/1GMP78vy5XTqazun64KuhZyfdGlUgbYiwX7qpSUgMskuaMfMPWdjrlF3RI1XzR27CpFzbLEXdLWmiSf6cUYBpQ4n\\\\\\\/HWt+URUYmc1zr7K1VrJaxAvTs8oI+Cu0sRJAydmcH2iBmFAg6LLiQsgaHlAGwfGWitgS7KF+Ix6dp2jTdzd65lGL+SEbXgvUi7Nt5PdLNPKP5pChYqSMulMP8TRQUwJzcF5FVbuT\\\\\\\/BjhzFao3KjN+4g1YrshapN0QnLdqwLroyi3HFFir5kv6ch1WebS09HstAbbvRJFJZI3\\\\\\\/orj\\\\\\\/zzMuGP319vuXhimgNDu2DOQ0\\\\\\\/+SOajS6fDHImdAygL5ILIlknwcWrLAgmja2FSsRqLoXUmC8e+ofK7WGFsCTyQa6j8d0Bkix4OtoDGMzrFy0ZNB4IfyRGE+DBn9tKYSe0sOeTXR+yqdENKgx4\\\\\\\/FnPEElgbVrB4y4bnYGK5efEW76Z72T5mxrLMx92racrLTuOIzfFwC83wkjd5ZnTgSN\\\\\\\/8nXXulHBdMVAFUjZlkpsvr0hrBYLUWGZ3EJ0njm2BWlOjPIIajrk5H\\\\\\\/UXqW3TJDemrWhK\\\\\\\/QWCNLQU3WLBI0NJsiw6PMVKOSiloPkt+bCF+ifjE095FeIBXh+PZmkxaAqELptXIJ2vPmkUHuinRMcgrF3g+VJxqSI4VNpHf+M5t6LLDSzzR2D7WpSn219Ex7Vhg+WqpVmWq2M6PQYnwJr8SU9fAdKl0F12bjE7QuEOxwF1ivaIH+tvsJtfVT\\\\\\\/oR\\\\\\\/p09gcPIVFgcCjp6ip+jzIUAMpat2awUI9ndumLlRxzG7h5uSnGoA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022gn9xKGj6POi++fmjQr+s7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a0376c2c13c1cea9e1bfb8116c02ed1744a1edcb1add0f85b59f483a46dd635d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459171 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/197" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aWqrshjRHOS_AwGUbQ-tumcFwO_EHKDV0SvF6S1wj7A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/197" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3DgtiKxUorXRcyQ-M3plE5SSZliwnR6Eclfv0kNeljU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/3/187" title="High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/3/209" title="Mixed Model Association Mapping for Fusarium Head Blight Resistance in Tunisian-Derived Durum Wheat Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459171" id="node459171" data-pisa="ggg;1/3/197" data-pisa-master="ggg;g3.111.000216" data-apath="/ggg/1/3/197.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hsin-Yu Chang</span>, <span class="highwire-citation-author" data-delta="1">Conor Lawless</span>, <span class="highwire-citation-author" data-delta="2">Stephen G. Addinall</span>, <span class="highwire-citation-author" data-delta="3">Sarah Oexle</span>, <span class="highwire-citation-author" data-delta="4">Morgan Taschuk</span>, <span class="highwire-citation-author" data-delta="5">Anil Wipat</span>, <span class="highwire-citation-author" data-delta="6">Darren J. Wilkinson</span> and <span class="highwire-citation-author" data-delta="7">David Lydall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">197-208;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000216 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459171" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hsin-Yu Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-Yu%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-Yu%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen G. Addinall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BG.%2BAddinall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addinall%20SG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BG.%2BAddinall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sarah Oexle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BOexle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oexle%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BOexle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Morgan Taschuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morgan%2BTaschuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taschuk%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorgan%2BTaschuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anil Wipat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anil%2BWipat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wipat%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnil%2BWipat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Darren J. Wilkinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BJ.%2BWilkinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilkinson%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BJ.%2BWilkinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.a.lydall@ncl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.a.lydall@ncl.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/3/197" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459171" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/197.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459171" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/197.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459171" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/197.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459171" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/3/197.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459171" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In telomerase-deficient yeast cells, like equivalent mammalian cells, telomeres shorten over many generations until a period of senescence/crisis is reached. After this, a small fraction of cells can escape senescence, principally using recombination-dependent mechanisms. To investigate the pathways that affect entry into and recovery from telomere-driven senescence, we combined a gene deletion disrupting telomerase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1Δ</a></em>) with the systematic yeast deletion collection and measured senescence characteristics in high-throughput assays. As expected, the vast majority of gene deletions showed no strong effects on entry into/exit from senescence. However, around 200 gene deletions behaving similarly to a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52Δ</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1Δ</a></em> archetype (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52Δ</a></em> affects homologous recombination) accelerated entry into senescence, and such cells often could not recover growth. A smaller number of strains similar to a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1Δ</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1Δ</a></em> archetype (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1Δ</a></em> affects proteins that bind telomeres) accelerated entry into senescence but also accelerated recovery from senescence. Our genome-wide analysis identifies genes that affect entry into and/or exit from telomere-initiated senescence and will be of interest to those studying telomere biology, replicative senescence, cancer, and ageing. Our dataset is complementary to other high-throughput studies relevant to telomere biology, genetic stability, and DNA damage responses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/telomere" class="hw-term hw-article-keyword hw-article-keyword-telomere" rel="nofollow">telomere</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li><li class="kwd"><a href="/keyword/senescence" class="hw-term hw-article-keyword hw-article-keyword-senescence" rel="nofollow">senescence</a></li><li class="kwd"><a href="/keyword/crisis" class="hw-term hw-article-keyword hw-article-keyword-crisis" rel="nofollow">crisis</a></li></ul><p id="p-3">In most eukaryotes, including mammals and yeasts, telomerase helps maintain telomeric DNA and solve the chromosome end-replication problem (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gomes <em>et al.</em> 2010</a>). Telomeric DNA recruits a number of different telomere-binding proteins to make functional telomeres. Dysfunctional telomeres can stimulate DNA damage responses, leading to genetic instability, cell senescence, or apoptosis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jain and Cooper 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lydall 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stewart and Weinberg 2006</a>).</p><p id="p-4">In most human somatic cells, there is insufficient telomerase to maintain telomere length indefinitely through cell divisions, and consequently, telomeres shorten and ultimately become nonfunctional. Telomere shortening is thought to be a double-edged sword in metazoan cells (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gomes <em>et al.</em> 2010</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Stewart and Weinberg 2006</a>). On one hand, the limited lifespan of functional telomeres limits cell divisions and therefore protects against uncontrolled cell division in cancer (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Artandi and Depinho 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lin <em>et al.</em> 2010</a>). On the other hand, limiting cell division prevents tissue renewal and repair and thus contributes to ageing.</p><p id="p-5">Telomere structure and function need to be maintained in cells with unlimited proliferative potential. In the majority of cancers, structure and function are preserved via the expression of high levels of telomerase, permitting unlimited and uncontrolled cell division. However, in around 15% of cancers, telomeres are maintained by alternative lengthening of telomeres (ALT) mechanisms (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Henson <em>et al.</em> 2002</a>). Therefore a fuller understanding of the pathways allowing eukaryotic cells to tolerate dysfunctional telomeres or to maintain functional telomeres, either in the presence or absence of telomerase, is critical for understanding processes underlying carcinogenesis and aging.</p><p id="p-6">In budding yeast, telomerase is constitutively expressed and telomere length is stable. However, telomerase can be deleted, giving yeast cells a life-cycle much like mammalian somatic cells; their telomeres shorten, they enter into a period of senescence/crisis, and at low rates, the cells recover from senescence using telomerase-independent mechanisms to maintain telomere length and function (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lundblad and Blackburn 1993</a>). Telomerase-deficient yeast cells are therefore an excellent model system to study the pathways that allow eukaryotic cells to tolerate dysfunctional telomeres or to maintain telomeres in the absence of telomerase, and are thus useful for studying pathways relevant to human carcinogenesis. Telomerase-deficient yeast cells typically survive by using recombination-dependent mechanisms to maintain telomeres and are a model for ALT pathways in human cells (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lendvay <em>et al.</em> 1996</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lundblad and Blackburn 1993</a>). However, recombination-independent mechanisms for chromosome end protection also occur in budding and fission yeasts (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jain <em>et al.</em> 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lee <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Maringele and Lydall 2004b</a>).</p><p id="p-7">Although a number of yeast genes, principally genes affecting homologous recombination, have been identified that affect entry/exit from telomere driven senescence (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdallah <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Azam <em>et al.</em> 2006</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Enomoto <em>et al.</em> 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Enomoto <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ijpma and Greider 2003</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Le <em>et al.</em> 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lee <em>et al.</em> 2007</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lydeard <em>et al.</em> 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Maringele and Lydall 2004a</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mceachern and Haber 2006</a>), so far no genome-wide analysis of these processes has been reported. Here we describe the first genome-wide survey to systematically determine the effects of budding yeast gene deletions on the growth of cells with telomere defects. Through these systematic approaches in budding yeast, we identify many yeast genes that affect telomere-driven senescence, orthologs of which may play important roles during carcinogenesis or aging.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Growth media, synthetic genetic array, and strains</h3><p id="p-8">An <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a>::natMX</em> strain (DLY5026) was combined with the genome-wide knockout collection using SGA as previously described (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). W303 genetic background strains were cultured in yeast extract, peptone, dextrose, and adenine (YEPD) media.</p></div><div id="sec-3" class="subsection"><h3>Culture passage</h3><div id="sec-4" class="subsection"><h4>Solid passage:</h4><p id="p-9">1536 colony solid agar-to-solid agar pinning was performed on a Biomatrix BM3-SC robot (S&amp;P Robotics Inc., Toronto, Canada) using an 0.8 mm diameter pin tool.</p></div><div id="sec-5" class="subsection"><h4>Liquid passage:</h4><p id="p-10">Selected <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a>::natMX yfg::G418</em> double mutants were cherry-picked after SGA. Inoculation from solid to liquid media was performed on the Biomatrix BM3-SC robot using a 96-pin (1 mm diameter) pintool. 96-well liquid culture plates were incubated at 30°C for 22 hr without shaking. A Biomek FX robot (Beckman Coulter Limited, High Wycombe, UK) was used to vortex the cultures to resuspend and then inoculated into new sets of 96-well plates containing 200 µl media in each well. A 96-pin (2 mm diameter) pin tool (V&amp;P Scientific, Inc., San Diego, CA, USA) was used, transferring approximately 3 µl of cell culture at each passage. In a similar procedure, resuspended cell cultures were diluted in 200 µl sterile H<sub>2</sub>O and vortexed before the diluted cultures were spotted onto solid agar plates. Agar plates were incubated for 48 hr at 30°C before being photographed.</p></div><div id="sec-6" class="subsection"><h4>Culture size estimation:</h4><p id="p-11">Solid agar plates were photographed on an spImager (S&amp;P Robotics Inc.) and analyzed using Colonyzer (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lawless <em>et al.</em> 2010</a>). Culture size variances were stabilized using a Box-Cox transformation with a power parameter of 0.6 as implemented in the R statistical package (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">R Development Core Team 2011</a>).</p></div><div id="sec-7" class="subsection"><h4>Mean density profile similarity:</h4><p id="p-12">The similarity of the MDP shape of each strain to that of three example archetypes was estimated using Pearson's correlation coefficient (<em>r</em>) between strain MDPs and the MDP of each archetype, using the <em>cor</em> function in the R statistical package (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">R Development Core Team 2011</a>). As well as calculating <em>r</em>, absolute differences in MDP were estimated using the root mean square (RMS) difference between profiles. The area under each MDP (representing total strain fitness across all passages) was also estimated as the sum of mean culture size across all 22 passages. Lists ranking the MDP for each strain according to its correlation coefficient were generated, with the strains having MDP shape most similar to the archetype (and the archetype itself) at the top of the list (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/000216.pdf">supporting information</a>, <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S5</a>, <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S6</a>, <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S7</a>, and <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S8</a>). Note that the absolute fitness of a comparator strain does not have to be similar to that of the archetype for their MDPs to correlate well. Two strains, one sick and one healthy, can have similarly shaped MDPs that correlate well. It is for this reason that some of the heat maps in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> are perceptibly stripy.</p></div><div id="sec-8" class="subsection"><h4>Fraction of recovered repeats:</h4><p id="p-13">Recovered fraction for each genotype was estimated by counting the number of repeated cultures for each strain and calculating the fraction of those repeats which had recovered to a size of 1000 or greater by passage 19 (typically in the range from 0/8 to 8/8). For classes similar to each of the three archetypes, the average recovered fraction of class members was calculated.</p></div></div><div id="sec-9" class="subsection"><h3>Telomere length detection</h3><p id="p-14">Southern hybridization to examine telomere length and structure was performed similarly to that previously described (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Maringele and Lydall 2004a</a>). Genomic DNA was extracted, digested with <em>Xho</em>I, then run overnight on a 1% agarose gel at 1 V/cm. Southern transfer and detection was performed using DIG-High Prime Labeling and Detection Kit (Roche) and visualized on a FUJI LAS4000. Telomeric Y′-TG probes (pHT128; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tsubouchi and Ogawa 2000</a>) were labelled using DIG-High Prime DNA Labeling and Detction Starter Kit II.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Measuring senescence by high-throughput, multipassage assays</h3><p id="p-15">To identify gene products that affect how telomerase-deficient cells enter into and recover from senescence, we used the synthetic genetic array (SGA) method to combine a systematic genome-wide collection of gene deletions with an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> mutation defective in telomerase (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Costanzo <em>et al.</em> 2010</a>). As recovery from senescence is a rare and stochastic process (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lundblad and Blackburn 1993</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nautiyal <em>et al.</em> 2002</a>), we compared two high-throughput methods of measuring senescence. These two methods were analogous to low-throughput liquid culture or serial tooth-picking approaches (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lundblad and Blackburn 1993</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Maringele and Lydall 2004a</a>). In a liquid procedure we passaged yeast strains in 96-well plates, while in a solid agar procedure we passaged 1536 colonies per plate by pinning directly onto agar (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The liquid procedure allowed greater dilution and therefore a greater range of divisions at each passage, but throughput was less efficient than in the pinning procedure. The liquid procedure had a higher risk of culture contamination.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/197/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two methods for genome-wide analysis of entry into and recovery from senescence. An est1::natMX strain (DLY5026) was crossed to the deletion mutant collection and double mutants were selected using SGA procedures. (A) Double mutants were passaged in 96-well plate liquid cultures, and fitness was monitored after spotting onto solid agar plates at 384 spot cell density for a total of 16 passages. (B) Cultures were serially pinned at 1536 colonies per plate and photographed for 22 passages." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-156552368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two methods for genome-wide analysis of entry into and recovery from senescence. An &lt;em&gt;est1::natMX&lt;/em&gt; strain (DLY5026) was crossed to the deletion mutant collection and double mutants were selected using SGA procedures. (A) Double mutants were passaged in 96-well plate liquid cultures, and fitness was monitored after spotting onto solid agar plates at 384 spot cell density for a total of 16 passages. (B) Cultures were serially pinned at 1536 colonies per plate and photographed for 22 passages.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/197/F1.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/197/F1.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/197/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/197/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460734" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-16" class="first-child">Two methods for genome-wide analysis of entry into and recovery from senescence. An <em>est1::natMX</em> strain (DLY5026) was crossed to the deletion mutant collection and double mutants were selected using SGA procedures. (A) Double mutants were passaged in 96-well plate liquid cultures, and fitness was monitored after spotting onto solid agar plates at 384 spot cell density for a total of 16 passages. (B) Cultures were serially pinned at 1536 colonies per plate and photographed for 22 passages.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>96-well liquid senescence assay</h3><p id="p-17">For the liquid senescence assay, 96-well plates that contained 60 different <em>yfg</em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ genotypes along with 36 independently generated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ “wild type control” strains were inoculated. Cultures in 16 such replicate plates, each with independently generated double mutants, were grown in parallel (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS1.pdf">Figure S1</a> shows the plate layout). In combination with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆, we tested the effects of deleting 27 genes previously reported as affecting telomere length (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gatbonton <em>et al.</em> 2006</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shachar <em>et al.</em> 2008</a>), 15 affecting DNA replication and repair (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Collins <em>et al.</em> 2007</a>), and 18 other genes that were randomly chosen controls or of some prior interest (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/TableS1.pdf">Table S1</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains are suitable “controls” because, although <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> is disrupted with the <em>KANMX</em> marker, the cells express sp<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001378">HIS5</a></em> from the <em>MATa</em>-specific promoter and are therefore histidine prototrophs and expected to show no growth defects (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Costanzo <em>et al.</em> 2010</a>). In total, 16 <em>yfg</em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ and 576 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ replicate cultures were examined in these liquid-based experiments.</p><p id="p-18">To follow growth in liquid, we used the procedure outlined in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>. Two-hundred microliter (200 μl) cultures were incubated for 22 hr at 30°C without shaking, and then shaken to resuspend the yeast cells. A sample from each culture was then diluted approximately 1:70 into fresh media at each passage using a pin tool. To assess fitness at each passage, a separate sample from each culture was also diluted approximately 1:70 in water and spotted onto agar plates, at 384 spots per plate. Each of four 384-spot plates was incubated for 48 hr and photographed to determine cell density (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). A 1:70 dilution was a pragmatic choice of dilution in the range used in low-throughput experiments. In low-throughput experiments where we carefully count cell number at each passage and dilute back to 5 × 10<sup>5</sup> cells per milliliter every day (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS3.pdf">Figure S3</a>), the dilution ranges from 1:600 for healthy cultures to 1:7 when cells are deeply senescent.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/197/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Medium throughput liquid culture based senescence assay. 96-well cultures were spotted at 384 spots per agar plate, and plates photographed and analyzed after 48 hr growth. (A) Example images showing growth at passages 1, 5, 10, and 15. Four independent cultures of five genotypes are illustrated by the colored boxes. (B) Mean density profile (MDP) analysis for six genotypes for 16 passages. Mean culture size and standard deviations are shown. All images can be browsed on our supplementary website (http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-156552368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Medium throughput liquid culture based senescence assay. 96-well cultures were spotted at 384 spots per agar plate, and plates photographed and analyzed after 48 hr growth. (A) Example images showing growth at passages 1, 5, 10, and 15. Four independent cultures of five genotypes are illustrated by the colored boxes. (B) Mean density profile (MDP) analysis for six genotypes for 16 passages. Mean culture size and standard deviations are shown. All images can be browsed on our supplementary website (http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/197/F2.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/197/F2.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/197/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/197/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460716" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-19" class="first-child">Medium throughput liquid culture based senescence assay. 96-well cultures were spotted at 384 spots per agar plate, and plates photographed and analyzed after 48 hr growth. (A) Example images showing growth at passages 1, 5, 10, and 15. Four independent cultures of five genotypes are illustrated by the colored boxes. (B) Mean density profile (MDP) analysis for six genotypes for 16 passages. Mean culture size and standard deviations are shown. All images can be browsed on our supplementary website (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images">http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">It is clear that the majority of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains, situated in the outer two rows/columns of each 384-spot plate, entered into and recovered from senescence over 16 passages (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The majority of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains grew well at passage 1, by passage 5 they grew very poorly, by passage 10 some had improved, and by passage 15 the majority had returned to a fitness similar to that at passage 1. At passage 15, some <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ cultures did not grow at all. This pattern of growth is consistent with the hypothesis that the SGA procedure worked efficiently and that most, but not all, telomerase-deficient <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ cultures entered into and recovered from telomere-initiated senescence. Recovery, in particular, was stochastic. To follow the behavior of individual cultures, it is useful to browse through senescent time courses, with each time point shown as a stack, in files (.tiff) from our supplementary data website (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images">http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images</a>).</p><p id="p-21">Because most gene deletions we chose to analyze affect DNA repair or telomere biology, most other strains (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆) grew poorly in comparison with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em> is required for recombination-dependent maintenance of telomeres in the absence of telomerase, and as expected, by the first passage the majority of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains were growing poorly and never recovered (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Mceachern and Haber 2006</a>). The slight increase in fitness at the end of the time course for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>) may reflect a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em>-independent mechanism of survivor generation that occurs in this and some other genetic backgrounds (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grandin and Charbonneau 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lebel <em>et al.</em> 2009</a>). Interestingly, the effect of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2</a></em>∆, which affects nonsense-mediated RNA decay, has previously been reported to delay entry into senescence (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Enomoto <em>et al.</em> 2004</a>), whereas our data suggest that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2</a></em>∆ accelerates senescence in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ background and that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains only weakly recover (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-22"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains behaved very similarly to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains, whereas, interestingly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains recovered from senescence more rapidly than the bulk of other strains and the control <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains. The effects of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1</a></em>∆ and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2</a></em>∆ can be seen from the high culture densities for these strains in passages 5 through 10, relative to neighboring cultures (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>). This is of interest because <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">Rif1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">Rif2</a> have well-known roles in telomere biology, encoding proteins which bind via <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005160">Rap1</a> to telomeric DNA, and are known to affect telomere capping (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anbalagan <em>et al.</em> 2011</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> 2011</a>). Interestingly, and consistent with our results, it was recently reported that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1</a></em>∆ and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a></em> strains enter and recover from senescence more rapidly than other cell types (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> 2011</a>). The positions of all other genes tested by this liquid senescence assay are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS1.pdf">Figure S1</a>, and one complete series of photographs are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS2.pdf">Figure S2</a>A. To utilize data from all replicate cultures and to allow quantitative comparison between strains, we constructed mean density profiles (MDP) for all genotypes. MDPs are curves showing the senescence characteristics of each genotype and are constructed after estimating yeast culture cell densities for repeat cultures of different genotypes at each passage from photographs using the image analysis tool Colonyzer (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lawless <em>et al.</em> 2010</a>). Six examples of this type of analysis showing the MDP of each are shown in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>, and all 61 genotypes are shown in <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S1</a>. Analysis of 576 independent <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ cultures shows that at passage 1 they grew well, with an average fitness of around 4000 units (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2b</a>). As expected, strains became less fit with each passage, reaching a nadir of fitness at around passage 6, before recovering reasonable fitness by the end of the experiment at passage 16. The shape of this curve, combined with the fact that some cultures became sterile, is further evidence that the SGA procedures were efficient and that little contamination by other genotypes (<em>e.g.</em>, diploids) or organisms (bacteria) was observed. By passage 16, at the end of the experiment, the mean fitness of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains was nearly as high as that at passage 1, which was as expected because of the emergence of recombination-dependent survivors; however, a large variance in cell densities was observed (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>). This variance at passage 16 was largely due to some cultures being essentially dead (see <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a>), presumably because in these particular cultures no senescent cells recovered. Sterile cultures were expected because the majority of cells do not recover from senescence and only a small volume of cell culture (&lt;3 µl) was transferred each time. Variance also began to increase earlier in the experiment, most likely due the stochastic nature of entry into senescence.</p><p id="p-23">As expected from the examples shown in <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains showed a similar MDP to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ strains, although interestingly, there was less variance between repeated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ cultures and less evidence of loss of vitality with time (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2B</a>, second panel from top). Quantitative analysis of 16 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2</a></em>∆, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1</a></em>∆, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a></em> strains was consistent with the images.</p><p id="p-24">We used MPDs to visually classify each of the 60 gene deletions into different classes (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S3</a>). These included those that entered senescence rapidly, including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004837">mre11∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002777">xrs2∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003131">rad54∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002483">rad55∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002411">rad57∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003583">rtt101∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001197">rtt107∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003925">rtt109∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003804">pol32∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004685">upf1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003304">upf3∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000521">dcc1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001234">ctf8∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004683">ctf18∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002171">slx5∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000918">slx8∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004802">sgs1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001596">rad27∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006368">mms1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004312">mms22∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004410">cdc73∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003213">rtf1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005805">rfm1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002718">sum1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003141">kem1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002490">rrp8∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003651">asf1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000184">tel1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005670">elg1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002581">hmo1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000143">ede1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002276">ydl118w∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001763">rho4∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004791">spt21∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004490">tsa1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006134">lea1∆</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000103">sla1∆</a></em>; a large number that behaved normally, including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005559">exo1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002625">rad9∆</a></em>; and a small group of just three genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006432">est3∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a></em>, consisting of mutants that appeared to enter senescence early and recover early. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/TableS1.pdf">Table S1</a> summarizes the behavior of individual genes and whether the results we obtained were consistent with previously published studies. It is clear that our results largely reproduced results previously reported from detailed, high-quality, low-throughput experiments giving us confidence in our data. However, there were some interesting surprises, such as the behavior of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004685">upf1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2∆</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003304">upf3∆</a></em> mutants that failed to recover from senescence here (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS5.pdf">Figure S5</a>) but had previously been reported to delay senescence (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Enomoto <em>et al.</em> 2004</a>).</p><p id="p-25">In summary, the 96-well senescence assay was capable of measuring entry of telomerase-deficient cells into senescence and recovery. However, this procedure was comparatively inefficient and not readily scalable to performing a genome-wide analysis of genes that affect entry into/exit from senescence. Therefore, we also tested a much more parallel and efficient solid pinning procedure, discussed next.</p></div><div id="sec-13" class="subsection"><h3>1536-pin solid-agar senescence assay</h3><p id="p-26">By pinning parallel colonies rather than growing liquid cultures, we increased throughput 36-fold. In pinning experiments, we grew a minimum of 8, sometimes 56 (for those on plate 15, which contained 24 extra copies of a small number of gene deletions), and in the case of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a></em>, 144 independently generated double mutant cultures for a total of 22 passages. Comparing cell density estimates immediately after inoculation with estimates in stationary phase for robustly growing, 1536-format, pinned cultures, we estimate that the maximum dilution achievable at each passage with this method is 16-fold. It is clear that, as cultures were passaged, the fraction of viable cultures decreased, suggesting cells entered into senescence (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). In some cultures, culture size decreased to a minimum and then increased (with increasing passage), suggesting that these cultures recovered from senescence. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> shows plate 15 that, in most cases, contains 24 independently generated <em>yfg1∆ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> double mutants and numerous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a></em> strains. Overlaid, colored boxes highlight the positions of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1∆</a></em> strains, allowing a visual assessment of reproducibility between cultures. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS2.pdf">Figure S2</a> compares the growth of cells in the 1536 colony format with the lower throughput, 384-spot format across several passages. All senescent time course photographs, with each time point shown as a stack in files (.tiff), are in our supplementary data website (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images">http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/197/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="High-throughput, solid-culture-based senescence assay. 1536 parallel colonies were transferred from agar plates, photographed, and analyzed. (A) Example images showing culture size at passages 1, 6, 11, and 22. Representative sets of four independent cultures of four genotypes are indicated by colored boxes. Plate number 15 is presented, which has 24 independently generated double mutant cultures per genotype, whereas plates 1 to 14 only have 4 independent cultures per strain. (B) Mean density profiles (MDP) for six genotypes for 22 passages. Mean culture size and standard deviations are shown. (C) Examples of rif1&#x2206; and rif2&#x2206; strain growth during passage. All images can be browsed on our supplementary website (http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-156552368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;High-throughput, solid-culture-based senescence assay. 1536 parallel colonies were transferred from agar plates, photographed, and analyzed. (A) Example images showing culture size at passages 1, 6, 11, and 22. Representative sets of four independent cultures of four genotypes are indicated by colored boxes. Plate number 15 is presented, which has 24 independently generated double mutant cultures per genotype, whereas plates 1 to 14 only have 4 independent cultures per strain. (B) Mean density profiles (MDP) for six genotypes for 22 passages. Mean culture size and standard deviations are shown. (C) Examples of &lt;em&gt;rif1&#x2206;&lt;/em&gt; and &lt;em&gt;rif2&#x2206;&lt;/em&gt; strain growth during passage. All images can be browsed on our supplementary website (http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/197/F3.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/197/F3.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/197/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/197/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460788" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-27" class="first-child">High-throughput, solid-culture-based senescence assay. 1536 parallel colonies were transferred from agar plates, photographed, and analyzed. (A) Example images showing culture size at passages 1, 6, 11, and 22. Representative sets of four independent cultures of four genotypes are indicated by colored boxes. Plate number 15 is presented, which has 24 independently generated double mutant cultures per genotype, whereas plates 1 to 14 only have 4 independent cultures per strain. (B) Mean density profiles (MDP) for six genotypes for 22 passages. Mean culture size and standard deviations are shown. (C) Examples of <em>rif1∆</em> and <em>rif2∆</em> strain growth during passage. All images can be browsed on our supplementary website (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images">http://research.ncl.ac.uk/colonyzer/ChangSenescence/#images</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">If strains had entered into and recovered from senescence, then telomere structure should alter, showing Type I– and Type II–type structures typical of cells using recombination-dependent ALT mechanisms to maintain telomeres (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Lundblad and Blackburn 1993</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Teng and Zakian 1999</a>). Importantly, Southern blots showed that the telomere structures of strains that had been passaged on solid plates had indeed altered, forming typical Type I or Type II survivors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS4.pdf">Figure S4</a>). In addition many of the DNA repair mutants known to be defective in recovery from senescence did not recover from senescence in these highly parallel experiments (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). These two observations strongly suggest that most, if not all, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ cells that were remaining after 22 passages had entered into and recovered from senescence and, therefore, that the high-throughput 1536 solid pinning approach is a useful method for following the senescence process.</p><p id="p-29">We generated MDPs of all 4299 genotypes over 22 passages, and plots of these profiles together with culture size standard deviations are available (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S2</a>). It is interesting to compare the solid pinning data (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S2</a>) with that for the 60 strains also analyzed in liquid culture, 6 of which are shown in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>, and the remainder of which are shown in <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S2</a>. In contrast to the pattern observed in liquid cultures, in pinning experiments evidence of a period of poor growth followed by recovery for many strains was less striking (<em>e.g.</em>, MDPs for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains, <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). However, it is notable that the variance in fitness increased with passage for each of these strains, more so for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> than for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains, and that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains showed an overall decline in fitness that plateaued around passage 15. We believe that the lack of clear decline and recovery in the solid experiments is because of the low dilution at each passage (16-fold in solid <em>vs.</em> 70-fold in liquid).</p><p id="p-30"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains entered senescence and failed to recover, as expected, but interestingly showed different growth patterns in solid <em>vs.</em> liquid plates. At passage 1, the fitness of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains (1400 units) was clearly less than the fitness of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">chk1∆</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a></em> strains (2000 units). This most likely is due the low fitness induced by deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em> itself. With time, the fitness of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> declined until, by passage 10, cultures were very small and did not really recover. The rate of entry of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> cultures into senescence was much slower than that observed in liquid cultures, consistent with cultures being diluted less on solid agar compared with liquid cultures and, therefore, having less potential to divide.</p><p id="p-31">Curiously, the density profiles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains on solid pins seemed to be more similar to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains than to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains. Conversely, in liquid cultures, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains are more similar to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains (compare the density profiles in <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figures 2B</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">3B</a>). We are unsure of the reason for this difference, but it seems to be reproducible as the effects of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004685">upf1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001119">upf2∆</a></em>, <em>and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003304">upf3∆</a></em> mirror each other in both solid and liquid cultures (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS5.pdf">Figure S5</a>). Disrupting the nonsense-mediated RNA decay pathway affects many different transcripts and, hence, cellular processes, including those relevant to telomere biology (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Enomoto <em>et al.</em> 2004</a>). Thus the explanation for the differences between solid and liquid cultures may be as trivial as the <em>upf∆</em> strains forming cultures more easily from dense inoculum after pinning on solid agar than from the dilute inoculum used in liquid cultures. Finally <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ mutants essentially behaved as expected given the liquid culture results and, if anything, improved fitness with time (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>).</p></div><div id="sec-14" class="subsection"><h3>Classifying senescence curves: three archetypes</h3><p id="p-32">To help classify the behavior of the thousands of different genotypes, we generated MDPs for each of the ∼4300 gene deletion strains (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S4</a>). We classified strains according to the similarity of their MDP to that of three archetypal gene disruptions: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, a neutral control strain (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A, B</a> and <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S5</a>); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, affecting a typical gene required for recovery from senescence (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C, D</a>, <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S6</a>, and <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S7</a>); and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, showing accelerated entry into and exit from senescence (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4E, F</a> and <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S8</a>). Similarity between MDPs was quantified using Pearson’s correlation coefficient (<em>r</em>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/197/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three senescence profile archetypes. Mean density profiles (MDP) were estimated for each of 4300 gene deletions combined with est1&#x2206;. Pearson&#x2019;s correlation coefficient (r) was calculated for each mutation compared with his3&#x2206;, rad52&#x2206;, and rif1&#x2206; mutations, representing three distinct senescence curve archetypes. (A) A heat map showing the MDP of the mutations having the 14 highest correlations with that of his3&#x2206; est1&#x2206;. (B) All 3035 MDPs (pink) correlating strongly (r &gt; 0.85) are shown along with the MDP of the archetype his3&#x2206; est1&#x2206; (red), and the mean curve of the 3036 strongly correlating MDPs (black). (C) A heat map showing the MDP of the mutations having the 20 highest correlations with that of rad52&#x2206; est1&#x2206;. To construct this group, we only included strains whose culture size at passage 2 was less than Cs = 1680, where Cs = Cave &#x2013; 0.5*CSD, Cave is the average of all double mutant culture sizes at passage 2, CSD is the culture size standard deviation at that passage. (D) All 208 MDPs (pink) correlating strongly (r &gt; 0.5, culture size &lt; Cs) with the MDP of rad52&#x2206; est1&#x2206; (red), together with the average of the 209 strongly correlating MDPs (black). (E) A heat map showing the MDP of the mutations having the 14 highest correlations with that of rif1&#x2206; est1&#x2206;. (F) Twenty gene disruptions whose MDPs (pink) correlate best with the MDP of rif1&#x2206; est1&#x2206; (red), together with the average of the 21 most strongly correlating MDPs (black). (G) Scale for heat maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-156552368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Three senescence profile archetypes. Mean density profiles (MDP) were estimated for each of 4300 gene deletions combined with &lt;em&gt;est1&#x2206;&lt;/em&gt;. Pearson&#x2019;s correlation coefficient (&lt;em&gt;r&lt;/em&gt;) was calculated for each mutation compared with &lt;em&gt;his3&#x2206;&lt;/em&gt;, &lt;em&gt;rad52&#x2206;&lt;/em&gt;, and &lt;em&gt;rif1&#x2206;&lt;/em&gt; mutations, representing three distinct senescence curve archetypes. (A) A heat map showing the MDP of the mutations having the 14 highest correlations with that of &lt;em&gt;his3&#x2206; est1&#x2206;&lt;/em&gt;. (B) All 3035 MDPs (pink) correlating strongly (&lt;em&gt;r&lt;/em&gt; &gt; 0.85) are shown along with the MDP of the archetype &lt;em&gt;his3&#x2206; est1&#x2206;&lt;/em&gt; (red), and the mean curve of the 3036 strongly correlating MDPs (black). (C) A heat map showing the MDP of the mutations having the 20 highest correlations with that of &lt;em&gt;rad52&#x2206; est1&#x2206;&lt;/em&gt;. To construct this group, we only included strains whose culture size at passage 2 was less than &lt;em&gt;C&lt;sup&gt;s&lt;/sup&gt; = 1680&lt;/em&gt;, where &lt;em&gt;C&lt;sup&gt;s&lt;/sup&gt; = C&lt;sup&gt;ave&lt;/sup&gt; &#x2013; 0.5*C&lt;sup&gt;SD&lt;/sup&gt;&lt;/em&gt;, &lt;em&gt;C&lt;sup&gt;ave&lt;/sup&gt;&lt;/em&gt; is the average of all double mutant culture sizes at passage 2, &lt;em&gt;C&lt;sup&gt;SD&lt;/sup&gt;&lt;/em&gt; is the culture size standard deviation at that passage. (D) All 208 MDPs (pink) correlating strongly (&lt;em&gt;r&lt;/em&gt; &gt; 0.5, culture size &lt; C&lt;sup&gt;s&lt;/sup&gt;) with the MDP of &lt;em&gt;rad52&#x2206; est1&#x2206;&lt;/em&gt; (red), together with the average of the 209 strongly correlating MDPs (black). (E) A heat map showing the MDP of the mutations having the 14 highest correlations with that of &lt;em&gt;rif1&#x2206; est1&#x2206;&lt;/em&gt;. (F) Twenty gene disruptions whose MDPs (pink) correlate best with the MDP of &lt;em&gt;rif1&#x2206; est1&#x2206;&lt;/em&gt; (red), together with the average of the 21 most strongly correlating MDPs (black). (G) Scale for heat maps.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/197/F4.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/197/F4.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/197/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/197/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460703" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-33" class="first-child">Three senescence profile archetypes. Mean density profiles (MDP) were estimated for each of 4300 gene deletions combined with <em>est1∆</em>. Pearson’s correlation coefficient (<em>r</em>) was calculated for each mutation compared with <em>his3∆</em>, <em>rad52∆</em>, and <em>rif1∆</em> mutations, representing three distinct senescence curve archetypes. (A) A heat map showing the MDP of the mutations having the 14 highest correlations with that of <em>his3∆ est1∆</em>. (B) All 3035 MDPs (pink) correlating strongly (<em>r</em> &gt; 0.85) are shown along with the MDP of the archetype <em>his3∆ est1∆</em> (red), and the mean curve of the 3036 strongly correlating MDPs (black). (C) A heat map showing the MDP of the mutations having the 20 highest correlations with that of <em>rad52∆ est1∆</em>. To construct this group, we only included strains whose culture size at passage 2 was less than <em>C<sup>s</sup> = 1680</em>, where <em>C<sup>s</sup> = C<sup>ave</sup> – 0.5*C<sup>SD</sup></em>, <em>C<sup>ave</sup></em> is the average of all double mutant culture sizes at passage 2, <em>C<sup>SD</sup></em> is the culture size standard deviation at that passage. (D) All 208 MDPs (pink) correlating strongly (<em>r</em> &gt; 0.5, culture size &lt; C<sup>s</sup>) with the MDP of <em>rad52∆ est1∆</em> (red), together with the average of the 209 strongly correlating MDPs (black). (E) A heat map showing the MDP of the mutations having the 14 highest correlations with that of <em>rif1∆ est1∆</em>. (F) Twenty gene disruptions whose MDPs (pink) correlate best with the MDP of <em>rif1∆ est1∆</em> (red), together with the average of the 21 most strongly correlating MDPs (black). (G) Scale for heat maps.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Normal senescence: <em>his3 est1</em> archetype</h3><p id="p-34">As we expected, the vast majority (more than 3000) of the double mutants can be classified in the category of “normal senescence” (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>), defined as mutants whose MDP correlates well with that of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains (<em>r</em> &gt; 0.85). There was a general trend of lower density across all passages for these strains (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>); at late passages, this is essentially due to mean density being estimated from a mixture of strongly growing cultures and sterile or poorly growing cultures (<em>e.g.</em>, <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A</a>, P22 Supp Fig S2). In this class, the mean density plateaued around passage 18. In normal senescence strains, on average 50% of cultures recovered growth by passage 19. Interestingly, the gene deletions whose MDP have the strongest anticorrelation with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> strains include many genes affecting telomere maintenance, DNA damage response, and cell-cycle control (bottom of <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S5</a>). In fact, the five strains whose MDP is least well correlated with those of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001596">rad27∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005670">elg1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006432">est3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em>) are the strongest examples of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> archetype, which we define below.</p></div><div id="sec-16" class="subsection"><h3>Accelerated entry into and exit from senescence: <em>rif1 est1</em> archetype</h3><p id="p-35">Using a strict correlation cut off (<em>r</em> &gt; 0.85) leaves only three members of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> class; however, we can see in <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4F</a> that the 20 most highly correlated genes display a distinctive MDP. Deletions in this class appear not only to speed up senescence but also to accelerate the appearance of survivors. For strains in this class, on average 81% of cultures recovered growth by passage 19. Interestingly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006432">EST3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">RIF1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">RIF2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000184">TEL1</a></em> are genes that are well known to affect telomere maintenance and <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> (2011)</a> reported that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1</a></em>∆ and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2</a></em>∆ mutants accelerated entry into and recovery from senescence in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004310">est2∆</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004445">rif2∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004310">est2∆</a></em> mutants. The effects of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006432">est3∆</a></em> on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ mutants that we have observed are not consistent with previous reports (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lendvay <em>et al.</em> 1996</a>). However, we have confirmed this phenotype in careful low throughput experiments in the different W303 genetic background (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS3.pdf">Figure S3</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000184">tel1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> results from previous studies are somewhat conflicting (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abdallah <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chan <em>et al.</em> 2001</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Enomoto <em>et al.</em> 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ritchie and Petes 2000</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/TableS1.pdf">Table S1</a>). Some previous data agree with our results whereas other data do not; it is not clear whether the differences are due to strain background or some other factor, such as experimental design. Other genes in this class, notably <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005670">ELG1</a></em> encoding a replication factor C (RFC)-type protein and with known telomere-related phenotypes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Smolikov <em>et al.</em> 2004</a>), warrant further study.</p></div><div id="sec-17" class="subsection"><h3>No recovery from senescence: <em>rad52 est1</em> archetype</h3><p id="p-36">There were 208 double mutants (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S7</a>) having an MDP that correlated well with that of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> (<em>r</em> &gt; 0.5, culture size at passage 2 &lt; 1680, <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4C, D</a>). For strains in this class, on average 29% of cultures recovered growth by passage 19. This subset is interesting because it contains many genes required, in principal, for recovery from telomere-initiated senescence, and orthologs of such genes could play critical roles during carcinogenesis, particularly in ALT tumors. Consistent with the role of this gene class at telomeres, there was a strong enrichment of gene deletions previously found to result in short telomeres but not in long telomeres (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gatbonton <em>et al.</em> 2006</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Shachar <em>et al.</em> 2008</a>; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). This enrichment is perhaps not surprising because cells with gene deletions inducing short telomeres are likely to enter senescence faster than those with gene deletions that result in normal length or long telomeres.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/197/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Properties of gene deletions that cause similar effects to rad52&#x2206; in an est1&#x2206; background. (A) Venn diagram showing gene deletions reported to result in significantly shorter (red) or longer (green) telomeres (Askree et al. 2004; Gatbonton et al. 2006; Shachar et al. 2008) and those we classified as similar to the rad52&#x2206; est1&#x2206; archetype (black). (B) Gene ontology (process) analysis demonstrating the P values for significance of over representation of terms (both GO terms and classifications from other screens) over represented in the class similar to the rad52&#x2206; est1&#x2206; archetype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-156552368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Properties of gene deletions that cause similar effects to &lt;em&gt;rad52&#x2206;&lt;/em&gt; in an &lt;em&gt;est1&#x2206;&lt;/em&gt; background. (A) Venn diagram showing gene deletions reported to result in significantly shorter (red) or longer (green) telomeres (Askree &lt;em&gt;et al.&lt;/em&gt; 2004; Gatbonton &lt;em&gt;et al.&lt;/em&gt; 2006; Shachar &lt;em&gt;et al.&lt;/em&gt; 2008) and those we classified as similar to the &lt;em&gt;rad52&#x2206; est1&#x2206;&lt;/em&gt; archetype (black). (B) Gene ontology (process) analysis demonstrating the &lt;em&gt;P&lt;/em&gt; values for significance of over representation of terms (both GO terms and classifications from other screens) over represented in the class similar to the &lt;em&gt;rad52&#x2206; est1&#x2206;&lt;/em&gt; archetype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/197/F5.medium.gif" width="269" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/197/F5.medium.gif" width="269" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/197/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/197/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460793" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-37" class="first-child">Properties of gene deletions that cause similar effects to <em>rad52∆</em> in an <em>est1∆</em> background. (A) Venn diagram showing gene deletions reported to result in significantly shorter (red) or longer (green) telomeres (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gatbonton <em>et al.</em> 2006</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Shachar <em>et al.</em> 2008</a>) and those we classified as similar to the <em>rad52∆ est1∆</em> archetype (black). (B) Gene ontology (process) analysis demonstrating the <em>P</em> values for significance of over representation of terms (both GO terms and classifications from other screens) over represented in the class similar to the <em>rad52∆ est1∆</em> archetype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">To further characterize this class, we subjected the genes with MDP similar to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> archetype to a “gene ontology (GO) yeast process” analysis. Interestingly, we found significant enrichment for genes involved in response to stress, chromosome organization, DNA metabolic processes, cellular membrane organization, vesicle-mediated transport, transcription DNA dependence, cell cycle, and protein modification (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Our results suggest that these genes share many properties and may play an important role in replicative senescence and might do so by affecting several cell processes. Moreover, when subjecting these genes to a GO analysis, we identified many components of known and relevant complexes, for instance, the MRX complex and the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004683">Ctf18</a> RFC–like complex (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The MRX complex has many known roles at telomeres, and when deletions are combined with deletions in the Ku complex, the double mutants enter senescence like telomerase-deficient strains (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dubois <em>et al.</em> 2002</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Maringele and Lydall 2004a</a>). However, we are unaware of reports of the MRX complex inhibiting senescence. The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004683">Ctf18</a> complex does have roles at telomeres (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hiraga <em>et al.</em> 2006</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Khair <em>et al.</em> 2010</a>); however, as far as we are aware, this is the first report showing that deletion of components of this complex accelerates senescence. The mechanisms by which these complexes affect replicative senescence need to be clarified in future studies.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460724/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460724/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">GO term analysis of <em>RAD52</em>-like genes in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Genes similar to the <em>rad52</em> archetype have negative genetic interactions with <em>yku70</em> but not <em>cdc13-1</em></h3><p id="p-40">The high-throughput senesence assays we report here are complementary to other high-throughput studies aimed at understanding telomere or chromosome biology. For example, <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> (2011)</a> performed systematic suppressor/enhancer analyses on yeast strains containing temperature-sensitive telomere capping defects caused by <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004897">yku70∆</a></em> mutations. All gene deletions were classified as suppressing, enhancing, or showing no strong interaction with each defect, and interaction strengths were quantified. The two datasets were compared in a single plot, and it is interesting to overlay gene deletions that behave like <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> in high-throughput senescence assays on this plot (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/197/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene deletions that cause similar effects to rad52&#x2206; in an est1&#x2206; background are enriched for deletions that negatively interact with yku70&#x2206; but not cdc13-1. (A) Gene deletions identified in this study as accelerating senescence overlaid on data showing genes that suppress or enhance temperature-sensitive growth defects caused by cdc13-1 or yku70&#x2206;. The data points are from Figure 4 in Addinall et al. 2011. Genes are highlighted in blue if their mean density profile (MDP) is classed as similar to the rad52&#x2206; est1&#x2206; archetype, in red if they additionally have a short telomere phenotype, or in green if they additionally have a long telomere phenotype as defined before (Askree et al. 2004; Gatbonton et al. 2006; Shachar et al. 2008) (B) Left: Fraction of strains similar to the rad52&#x2206; est1&#x2206; archetype found in indicated regions of the genetic interaction profile presented in panel A. Right: Fraction of total gene deletions tested found in indicated regions of the genetic interaction profile presented in panel A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-156552368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene deletions that cause similar effects to &lt;em&gt;rad52&#x2206;&lt;/em&gt; in an &lt;em&gt;est1&#x2206;&lt;/em&gt; background are enriched for deletions that negatively interact with &lt;em&gt;yku70&#x2206;&lt;/em&gt; but not &lt;em&gt;cdc13-1&lt;/em&gt;. (A) Gene deletions identified in this study as accelerating senescence overlaid on data showing genes that suppress or enhance temperature-sensitive growth defects caused by &lt;em&gt;cdc13-1&lt;/em&gt; or &lt;em&gt;yku70&#x2206;&lt;/em&gt;. The data points are from Figure 4 in Addinall &lt;em&gt;et al.&lt;/em&gt; 2011. Genes are highlighted in blue if their mean density profile (MDP) is classed as similar to the &lt;em&gt;rad52&#x2206; est1&#x2206;&lt;/em&gt; archetype, in red if they additionally have a short telomere phenotype, or in green if they additionally have a long telomere phenotype as defined before (Askree &lt;em&gt;et al.&lt;/em&gt; 2004; Gatbonton &lt;em&gt;et al.&lt;/em&gt; 2006; Shachar &lt;em&gt;et al.&lt;/em&gt; 2008) (B) Left: Fraction of strains similar to the &lt;em&gt;rad52&#x2206; est1&#x2206;&lt;/em&gt; archetype found in indicated regions of the genetic interaction profile presented in panel A. Right: Fraction of total gene deletions tested found in indicated regions of the genetic interaction profile presented in panel A.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/197/F6.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/197/F6.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/197/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/197/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460740" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-41" class="first-child">Gene deletions that cause similar effects to <em>rad52∆</em> in an <em>est1∆</em> background are enriched for deletions that negatively interact with <em>yku70∆</em> but not <em>cdc13-1</em>. (A) Gene deletions identified in this study as accelerating senescence overlaid on data showing genes that suppress or enhance temperature-sensitive growth defects caused by <em>cdc13-1</em> or <em>yku70∆</em>. The data points are from <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a> in <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>. Genes are highlighted in blue if their mean density profile (MDP) is classed as similar to the <em>rad52∆ est1∆</em> archetype, in red if they additionally have a short telomere phenotype, or in green if they additionally have a long telomere phenotype as defined before (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Gatbonton <em>et al.</em> 2006</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Shachar <em>et al.</em> 2008</a>) (B) Left: Fraction of strains similar to the <em>rad52∆ est1∆</em> archetype found in indicated regions of the genetic interaction profile presented in panel A. Right: Fraction of total gene deletions tested found in indicated regions of the genetic interaction profile presented in panel A.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Interestingly, most of the gene deletions that accelerated senescence also enhanced <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004897">yku70∆</a></em> temperature-sensitive growth defects but showed no significant bias toward either positive or negative effects on the fitness of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> mutants. This is shown by the clear left/right but lack of top/bottom bias of such strains in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004897">yku70∆</a></em> mutants have short telomeres, similar to those of telomerase-deficient, pre-senescent cells, whereas <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> mutants have telomeres of normal length or slightly longer than normal (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zubko and Lydall 2006</a>). Therefore, a simple explanation for this bias is that the gene deletions highlighted on the left of <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a> contribute to the ability of cells with short telomeres to continue through the cell cycle. These data also suggest that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004897">yku70∆</a></em> is perhaps a better model for telomere-initiated senescence than <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em>.</p></div><div id="sec-19" class="subsection"><h3>Genes required for efficient replication fork progression</h3><p id="p-43">Global genetic analyses in budding yeast <em>Saccharomyces cerevisiae</em> have identified a pathway, including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003651">ASF1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003925">RTT109</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006368">MMS1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004312">MMS22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003583">RTT101</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001197">RTT107</a></em>, instrumental to cell survival after replisome stalling (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Collins <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Duro <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pan <em>et al.</em> 2006</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zaidi <em>et al.</em> 2008</a>). Interestingly, it has been proposed that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006368">MMS1</a></em> are essential for pre-senescence growth (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Abdallah <em>et al.</em> 2009</a>). To further investigate this pathway in telomerase-deficient cells, we examined the behavior of these strains in both our high-throughput and low-throughput screens. We found that not only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006368">MMS1</a></em> but also <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003651">ASF1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003925">RTT109</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004312">MMS22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003583">RTT101</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001197">RTT107</a></em> showed an accelerated senescence phenotype when deleted in both screens (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS6.pdf">Figure S6</a>). These results strongly suggest that replication fork progression plays an important role in replicative senescence. However, our data also suggest that there are many additional pathways/complexes affecting replicative senescence (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). It will be interesting to investigate the roles of these pathways/complexes further.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-44">We report the first genome-wide analysis of the effects of gene deletions on entry into and exit from telomere-initiated senescence in <em>S. cerevisiae</em>. We have successfully carried out a medium-throughput, liquid-based senescence screen and a high-throughput, solid pin-based screen. Although the liquid-based screen is a valuable source of data describing the behavior of 60 gene deletions hand-picked for the relevance of their effect on entry into and recovery from senescence, we chose to perform a genome-wide analysis of 4299 genes by using a solid pinning approach because it was much more efficient (36 times the throughput). Inevitably, the high-throughput pinning approach suffers from a lack of resolution compared with the liquid-based approach. Nevertheless, it contains a great deal of valuable quantitative information. We have constructed a high-quality dataset demonstrating senescence profiles for all 4299 gene deletions in the knockout library combined with a telomerase defect that will complement other low- and high-throughput analyses of telomere biology and the DNA damage response.</p><p id="p-45">We chose to classify double deletion mutations based on the similarity of their MDPs to those of archetypal mutations: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> (normal), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> (accelerated entry into senescence and failure to recover), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> (accelerated entry into senescence and recovery). However, other clustering or analysis approaches may also prove informative. For example, we also performed unsupervised hierarchical quality threshold (QT) clustering to identify 23 different MDP classes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS8.pdf">Figure S8</a>). The MDPs which we constructed (<em>e.g.</em>, <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S2</a> and <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S4</a>), together with the images and raw quantifications from which they are derived, are published to allow others to analyze our dataset using different approaches, for example, by searching for strains whose deletions result in MDPs correlating strongly with those of archetypes of specific interest, which could be different to the three presented here.</p><p id="p-46">The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> archetype strain and mutants in this class show poor growth when generated by SGA immediately after germination. Therefore we tested whether additional information is gained from continuing to passage such poor-growing strains. To address this issue, we chose to compare our <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a></em>-like cluster with genes we identified as growing poorly with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> at passage 0 in our experiments, and with analogous datasets from large-scale, genome-wide synthetic lethal analysis experiments (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Costanzo <em>et al.</em> 2010</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS7.pdf">Figure S7</a>A shows that there is overlap among these three datasets but that they are clearly not identical. The group of strains with MDP similar to that of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a></em>∆ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> archetype is distinct from that which we identified as synthetic lethal interactions immediately after SGA (<a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S10</a> and <a href="http://research.ncl.ac.uk/colonyzer/ChangSenescence/">File S11</a>).</p><p id="p-47">To compare the relative merits of each dataset, the member strains of each are highlighted in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000216/-/DC1/FigureS7.pdf">Figure S7</a>C–F on plots similar to <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>. Interestingly, the left right/bias we observed for the strains in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> class (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a> and S7C) is not as pronounced in the other datasets. This strongly suggests that gene deletions we identified in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52∆</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1∆</a></em> class, based on their behavior with repeated passage, have different properties than those growing poorly at early passages identified by us or in the Costanzo <em>et al.</em> (2010) dataset. Therefore we conclude that, at the very least, our high-throughput senescence assays complement the existing datasets, identifying gene deletions with notably different properties.</p><p id="p-48">Our dataset is a rich resource for those working in telomere biology in budding yeast and other eukaryotic organisms. All our data are freely available, and there are numerous provocative and interesting observations worthy of further investigation. For example, the genes behaving like <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">RIF1</a></em> show very interesting effects on the senescence process in budding yeast. Although some genes in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">RAD52</a></em> class, such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002171">SLX5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000918">SLX8</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006368">MMS1</a></em>, were previously reported to affect the processes of recovery from senescence, we have identified 200 or more genes in this class. Our data also shed new light on the functions of proteins with known roles in telomere biology, such as <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005670">Elg1</a> or members of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004683">Ctf18</a> complex.</p><p id="p-49">We imagine that iterations of our genome-wide approach to understand senescence, perhaps utilizing genome-wide liquid passage approaches, like <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2</a>, or “bar code–based” approaches (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hillenmeyer <em>et al.</em> 2008</a>), may extend our systematic analysis of gene deletions affecting telomere-driven senescence. It may also be interesting to perform genome-wide screens analyzing the effects of deleting other protein or RNA subunits of telomerase, as we have shown that deleting the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006432">EST3</a></em> gene affects the senescence profile of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>∆ mutants in a manner analogous to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">rif1∆</a></em> mutants. It may also be interesting to extend this approach to <em>S. pombe</em>, where telomere biology is significantly different from <em>S. cerevisiae</em> and genome-wide knockout libraries and SGA approaches are now available (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dixon <em>et al.</em> 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Roguev <em>et al.</em> 2007</a>). It will be more technically challenging to perform analogous studies in mammalian cells.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We gratefully acknowledge all members of our laboratory and Centre for Integrated System Biology of Ageing and Nutrition (CISBAN) for support and helpful discussions, particularly Alan Leake for making media. This study was supported by Biotechnology and Biological Sciences Research Council (BBSRC) (BB/C008200/1) and the Wellcome Trust (075294).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-51"><em>Communicating editor: Brenda J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-04-08"><span class="received-label">Received </span>April 8, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-06-01"><span class="accepted-label">Accepted </span>June 1, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Chang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.1" data-doi="10.1038/ncb1911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdallah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luciano</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Runge</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisby</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geli</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A two-step model for senescence triggered by a single critically short telomere.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DAbdallah%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D988%26rft.epage%253D993%26rft.atitle%253DA%2Btwo-step%2Bmodel%2Bfor%2Bsenescence%2Btriggered%2Bby%2Ba%2Bsingle%2Bcritically%2Bshort%2Btelomere.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1911%26rft_id%253Dinfo%253Apmid%252F19597486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19597486&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268593200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.2" data-doi="10.1371/journal.pgen.1001362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holstein</span>  <span class="cit-name-given-names">E.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAddinall%26rft.auinit1%253DS.%2BG.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1001362%26rft.epage%253De1001362%26rft.atitle%253DQuantitative%2Bfitness%2Banalysis%2Bshows%2Bthat%2BNMD%2Bproteins%2Band%2Bmany%2Bother%2Bprotein%2Bcomplexes%2Bsuppress%2Bor%2Benhance%2Bdistinct%2Btelomere%2Bcap%2Bdefects.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001362%26rft_id%253Dinfo%253Apmid%252F21490951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21490951&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.3" data-doi="10.1371/journal.pgen.1002024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anbalagan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonetti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longhese</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rif1 supports the function of the CST complex in yeast telomere capping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAnbalagan%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1002024%26rft.epage%253De1002024%26rft.atitle%253DRif1%2Bsupports%2Bthe%2Bfunction%2Bof%2Bthe%2BCST%2Bcomplex%2Bin%2Byeast%2Btelomere%2Bcapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002024%26rft_id%253Dinfo%253Apmid%252F21437267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437267&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Artandi</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePinho</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Telomeres and telomerase in cancer.</span> <abbr class="cit-jnl-abbrev">Carcinogenesis</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCarcinogenesis%26rft.volume%253D31%26rft.spage%253D9%26rft_id%253Dinfo%253Apmid%252F19887512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19887512&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.5" data-doi="10.1073/pnas.0401263101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askree</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolikov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide screen for Saccharomyces cerevisiae deletion mutants that affect telomere length.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">8658</span>–<span class="cit-lpage">8663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401263101%26rft_id%253Dinfo%253Apmid%252F15161972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIzLzg2NTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.6" data-doi="10.1093/nar/gkj452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanoux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoenly</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evidence that the S.cerevisiae Sgs1 protein facilitates recombinational repair of telomeres during senescence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">506</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DAzam%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253D2%26rft.spage%253D506%26rft.epage%253D516%26rft.atitle%253DEvidence%2Bthat%2Bthe%2BS.cerevisiae%2BSgs1%2Bprotein%2Bfacilitates%2Brecombinational%2Brepair%2Bof%2Btelomeres%2Bduring%2Bsenescence%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj452%26rft_id%253Dinfo%253Apmid%252F16428246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16428246&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235291300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.7" data-doi="10.1016/S0960-9822(01)00391-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prescott</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackburn</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Altering telomere structure allows telomerase to act in yeast lacking ATM kinases.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1240</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DChan%26rft.auinit1%253DS.%2BW.%26rft.volume%253D11%26rft.issue%253D16%26rft.spage%253D1240%26rft.epage%253D1250%26rft.atitle%253DAltering%2Btelomere%2Bstructure%2Ballows%2Btelomerase%2Bto%2Bact%2Bin%2Byeast%2Blacking%2BATM%2Bkinases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252801%252900391-8%26rft_id%253Dinfo%253Apmid%252F11525738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(01)00391-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11525738&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170636100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.8" data-doi="10.1038/nsmb.2034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dittmar</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Long telomeres are preferentially extended during recombination-mediated telomere maintenance.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DChang%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D451%26rft.epage%253D456%26rft.atitle%253DLong%2Btelomeres%2Bare%2Bpreferentially%2Bextended%2Bduring%2Brecombination-mediated%2Btelomere%2Bmaintenance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2034%26rft_id%253Dinfo%253Apmid%252F21441915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21441915&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.9" data-doi="10.1038/nature05649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maas</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fillingham</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">446</span>: <span class="cit-fpage">806</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D446%26rft.issue%253D7137%26rft.spage%253D806%26rft.epage%253D810%26rft.atitle%253DFunctional%2Bdissection%2Bof%2Bprotein%2Bcomplexes%2Binvolved%2Bin%2Byeast%2Bchromosome%2Bbiology%2Busing%2Ba%2Bgenetic%2Binteraction%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05649%26rft_id%253Dinfo%253Apmid%252F17314980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17314980&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245626800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.10" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010a</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.11" data-doi="10.1073/pnas.0806261105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedyshyn</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chahwan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">16653</span>–<span class="cit-lpage">16658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806261105%26rft_id%253Dinfo%253Apmid%252F18931302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQzLzE2NjUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DuBois</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haimberger</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntosh</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A quantitative assay for telomere protection in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">995</span>–<span class="cit-lpage">1013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D161%26rft.spage%253D995%26rft_id%253Dinfo%253Apmid%252F12136006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12136006&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177192500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.13" data-doi="10.1016/j.dnarep.2008.01.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaisica</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Budding yeast Mms22 and Mms1 regulate homologous recombination induced by replisome blockage.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Brepair%26rft.stitle%253DDNA%2BRepair%2B%2528Amst%2529%26rft.aulast%253DDuro%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D811%26rft.epage%253D818%26rft.atitle%253DBudding%2Byeast%2BMms22%2Band%2BMms1%2Bregulate%2Bhomologous%2Brecombination%2Binduced%2Bby%2Breplisome%2Bblockage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2008.01.007%26rft_id%253Dinfo%253Apmid%252F18321796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2008.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18321796&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.14" data-doi="10.1091/mbc.02-02-0012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enomoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glowczewski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MEC3, MEC1, and DDC2 are essential components of a telomere checkpoint pathway required for cell cycle arrest during senescence in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2626</span>–<span class="cit-lpage">2638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.02-02-0012%26rft_id%253Dinfo%253Apmid%252F12181334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMjYyNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.15" data-doi="10.1128/MCB.24.2.837-845.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enomoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glowczewski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lew-Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Telomere cap components influence the rate of senescence in telomerase-deficient yeast cells.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DEnomoto%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D837%26rft.epage%253D845%26rft.atitle%253DTelomere%2BCap%2BComponents%2BInfluence%2Bthe%2BRate%2Bof%2BSenescence%2Bin%2BTelomerase-Deficient%2BYeast%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.2.837-845.2004%26rft_id%253Dinfo%253Apmid%252F14701754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvODM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.16" data-doi="10.1371/journal.pgen.0020035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatbonton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imbesi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Telomere length as a quantitative trait: genome-wide survey and genetic mapping of telomere length-control genes in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGatbonton%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253De35%26rft.epage%253De35%26rft.atitle%253DTelomere%2Blength%2Bas%2Ba%2Bquantitative%2Btrait%253A%2Bgenome-wide%2Bsurvey%2Band%2Bgenetic%2Bmapping%2Bof%2Btelomere%2Blength-control%2Bgenes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020035%26rft_id%253Dinfo%253Apmid%252F16552446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16552446&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.17" data-doi="10.1016/j.febslet.2010.07.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shay</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">W. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Telomere biology in Metazoa.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">584</span>: <span class="cit-fpage">3741</span>–<span class="cit-lpage">3751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DGomes%26rft.auinit1%253DN.%2BM.%26rft.volume%253D584%26rft.issue%253D17%26rft.spage%253D3741%26rft.epage%253D3751%26rft.atitle%253DTelomere%2Bbiology%2Bin%2BMetazoa.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2010.07.031%26rft_id%253Dinfo%253Apmid%252F20655915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2010.07.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20655915&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281161100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.18" data-doi="10.1128/MCB.00817-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charbonneau</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Telomerase- and RAD52-independent immortalization of budding yeast by an inherited-long-telomere pathway of telomeric repeat amplification.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00817-08%26rft_id%253Dinfo%253Apmid%252F19047370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI5LzQvOTY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.19" data-doi="10.1038/sj.onc.1205058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeager</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddel</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Alternative lengthening of telomeres in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">598</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DHenson%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D598%26rft.epage%253D610%26rft.atitle%253DAlternative%2Blengthening%2Bof%2Btelomeres%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1205058%26rft_id%253Dinfo%253Apmid%252F11850785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1205058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11850785&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173390500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.20" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.21" data-doi="10.1038/sj.emboj.7601038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiraga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The Ctf18 RFC-like complex positions yeast telomeres but does not specify their replication time.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1505</span>–<span class="cit-lpage">1514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DHiraga%26rft.auinit1%253DS.%26rft.volume%253D25%26rft.issue%253D7%26rft.spage%253D1505%26rft.epage%253D1514%26rft.atitle%253DThe%2BCtf18%2BRFC-like%2Bcomplex%2Bpositions%2Byeast%2Btelomeres%2Bbut%2Bdoes%2Bnot%2Bspecify%2Btheir%2Breplication%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601038%26rft_id%253Dinfo%253Apmid%252F16525505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7601038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16525505&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.22" data-doi="10.1091/mbc.02-04-0057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">IJpma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greider</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Short telomeres induce a DNA damage response in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">1001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.02-04-0057%26rft_id%253Dinfo%253Apmid%252F12631718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzMvOTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.23" data-doi="10.1146/annurev-genet-102108-134841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Telomeric strategies: means to an end.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DJain%26rft.auinit1%253DD.%26rft.volume%253D44%26rft.spage%253D243%26rft.epage%253D269%26rft.atitle%253DTelomeric%2Bstrategies%253A%2Bmeans%2Bto%2Ban%2Bend.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-102108-134841%26rft_id%253Dinfo%253Apmid%252F21047259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-102108-134841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21047259&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286042600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.24" data-doi="10.1038/nature09374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebden</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">HAATI survivors replace canonical telomeres with blocks of generic heterochromatin.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJain%26rft.auinit1%253DD.%26rft.volume%253D467%26rft.issue%253D7312%26rft.spage%253D223%26rft.epage%253D227%26rft.atitle%253DHAATI%2Bsurvivors%2Breplace%2Bcanonical%2Btelomeres%2Bwith%2Bblocks%2Bof%2Bgeneric%2Bheterochromatin.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09374%26rft_id%253Dinfo%253Apmid%252F20829796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20829796&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281616300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khair</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Roles of the checkpoint sensor clamp Rad9-Rad1-Hus1 (911)-complex and the clamp loaders Rad17-RFC and Ctf18-RFC in Schizosaccharomyces pombe telomere maintenance.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2237–2248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D9%26rft.spage%253D2237%2B2248%26rft.atitle%253DROLES%2BOF%2BTHE%2BCHECKPOINT%2BSENSOR%2BCLAMP%2BRAD9-RAD1-HUS1%2B%2528911%2529-COMPLEX%2BAND%2BTHE%2BCLAMP%2BLOADERS%2BRAD17-RFC%2BAND%2BCTF18-RFC%2BIN%2BSCHIZOSACCHAROMYCES%2BPOMBE%2BTELOMERE%2BMAINTENANCE.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.26" data-doi="10.1186/1471-2105-11-287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLawless%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.spage%253D287%26rft.epage%253D287%26rft.atitle%253DColonyzer%253A%2Bautomated%2Bquantification%2Bof%2Bmicro-organism%2Bgrowth%2Bcharacteristics%2Bon%2Bsolid%2Bagar.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-287%26rft_id%253Dinfo%253Apmid%252F20509870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20509870&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greider</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">RAD50 and RAD51 define two pathways that collaborate to maintain telomeres in the absence of telomerase.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D152%26rft.spage%253D143%26rft_id%253Dinfo%253Apmid%252F10224249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10224249&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080219100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lebel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosonina</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sealey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryde</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Telomere maintenance and survival in saccharomyces cerevisiae in the absence of telomerase and RAD52.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D671%26rft_id%253Dinfo%253Apmid%252F19380905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19380905&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennock</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">F. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence that a RecQ helicase slows senescence by resolving recombining telomeres.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1334</span>–<span class="cit-lpage">1344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D5%26rft.spage%253D1334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000247173200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.30" data-doi="10.1074/jbc.M804760200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mogen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">F. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sgs1 RecQ helicase inhibits survival of Saccharomyces cerevisiae cells lacking telomerase and homologous recombination.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">29847</span>–<span class="cit-lpage">29858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M804760200%26rft_id%253Dinfo%253Apmid%252F18757364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvNDQvMjk4NDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lendvay</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sah</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Senescence mutants of Saccharomyces cerevisiae with a defect in telomere replication identify three additional EST genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1399</span>–<span class="cit-lpage">1412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D144%26rft.spage%253D1399%26rft_id%253Dinfo%253Apmid%252F8978029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8978029&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VY93100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.32" data-doi="10.1182/blood-2010-02-272104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Letsolo</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Telomere dysfunction and fusion during the progression of chronic lymphocytic leukemia: evidence for a telomere crisis.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">1899</span>–<span class="cit-lpage">1907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2010-02-272104%26rft_id%253Dinfo%253Apmid%252F20538793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTE2LzExLzE4OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.33" data-doi="10.1016/0092-8674(93)90234-H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackburn</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">An alternative pathway for yeast telomere maintenance rescues est1<sup>−</sup> senescence.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLundblad%26rft.auinit1%253DV.%26rft.volume%253D73%26rft.issue%253D2%26rft.spage%253D347%26rft.epage%253D360%26rft.atitle%253DAn%2Balternative%2Bpathway%2Bfor%2Byeast%2Btelomere%2Bmaintenance%2Brescues%2Best1-%2Bsenescence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990234-H%26rft_id%253Dinfo%253Apmid%252F8477448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90234-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8477448&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KY50500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.34" data-doi="10.1038/emboj.2009.176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Taming the tiger by the tail: modulation of DNA damage responses by telomeres.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2174</span>–<span class="cit-lpage">2187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DLydall%26rft.auinit1%253DD.%26rft.volume%253D28%26rft.issue%253D15%26rft.spage%253D2174%26rft.epage%253D2187%26rft.atitle%253DTaming%2Bthe%2Btiger%2Bby%2Bthe%2Btail%253A%2Bmodulation%2Bof%2BDNA%2Bdamage%2Bresponses%2Bby%2Btelomeres.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.176%26rft_id%253Dinfo%253Apmid%252F19629039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2009.176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19629039&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.35" data-doi="10.1038/nature06047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lydeard</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Break-induced replication and telomerase-independent telomere maintenance require Pol32.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLydeard%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D448%26rft.issue%253D7155%26rft.spage%253D820%26rft.epage%253D823%26rft.atitle%253DBreak-induced%2Breplication%2Band%2Btelomerase-independent%2Btelomere%2Bmaintenance%2Brequire%2BPol32.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06047%26rft_id%253Dinfo%253Apmid%252F17671506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17671506&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248754200043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.36" data-doi="10.1534/genetics.166.4.1641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maringele</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004a</span> <span class="cit-article-title">EXO1 plays a role in generating type I and type II survivors in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1641</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D166%26rft.spage%253D1641%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.4.1641%26rft_id%253Dinfo%253Apmid%252F15126386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.166.4.1641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15126386&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.37" data-doi="10.1101/gad.316504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maringele</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004b</span> <span class="cit-article-title">Telomerase- and recombination-independent immortalization of budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2663</span>–<span class="cit-lpage">2675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.316504%26rft_id%253Dinfo%253Apmid%252F15489288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzIxLzI2NjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.38" data-doi="10.1146/annurev.biochem.74.082803.133234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEachern</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Break-induced replication and recombinational telomere elongation in yeast.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DMcEachern%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D75%26rft.spage%253D111%26rft.epage%253D135%26rft.atitle%253DBreak-induced%2Breplication%2Band%2Brecombinational%2Btelomere%2Belongation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.74.082803.133234%26rft_id%253Dinfo%253Apmid%252F16756487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.74.082803.133234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16756487&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239807600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nautiyal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackburn</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The genome-wide expression response to telomerase deletion in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">9316</span>–<span class="cit-lpage">9321</span>. <span class="cit-comment">Erratum in Proc. Natl. Acad. Sci. USA 35:13100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D14%26rft.spage%253D9316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.40" data-doi="10.1016/j.cell.2005.12.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A DNA integrity network in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPan%26rft.auinit1%253DX.%26rft.volume%253D124%26rft.issue%253D5%26rft.spage%253D1069%26rft.epage%253D1081%26rft.atitle%253DA%2BDNA%2Bintegrity%2Bnetwork%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.12.036%26rft_id%253Dinfo%253Apmid%252F16487579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.12.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16487579&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237241200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-1.3.197.41"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-pub-date">2011</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Mre11p/Rad50p/Xrs2p complex and the Tel1p function in a single pathway for telomere maintenance in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRitchie%26rft.auinit1%253DK.%2BB.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D475%26rft.epage%253D479%26rft.atitle%253DThe%2BMre11p%252FRad50p%252FXrs2p%2BComplex%2Band%2Bthe%2BTel1p%2BFunction%2Bin%2Ba%2BSingle%2BPathway%2Bfor%2BTelomere%2BMaintenance%2Bin%2BYeast%26rft_id%253Dinfo%253Apmid%252F10790418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzEvNDc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.43" data-doi="10.1038/nmeth1098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiren</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-throughput genetic interaction mapping in the fission yeast Schizosaccharomyces pombe.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DRoguev%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253D861%26rft.epage%253D866%26rft.atitle%253DHigh-throughput%2Bgenetic%2Binteraction%2Bmapping%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1098%26rft_id%253Dinfo%253Apmid%252F17893680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17893680&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249778200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.44" data-doi="10.1038/msb.2008.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shachar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharan</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A systems-level approach to mapping the telomere length maintenance gene circuitry.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DShachar%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DA%2Bsystems-level%2Bapproach%2Bto%2Bmapping%2Bthe%2Btelomere%2Blength%2Bmaintenance%2Bgene%2Bcircuitry%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2008.13%26rft_id%253Dinfo%253Apmid%252F18319724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjQvMS8xNzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.45" data-doi="10.1073/pnas.0307796100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smolikov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazor</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauskopf</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">ELG1, a regulator of genome stability, has a role in telomere length regulation and in silencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1656</span>–<span class="cit-lpage">1661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307796100%26rft_id%253Dinfo%253Apmid%252F14745004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAxLzYvMTY1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.46" data-doi="10.1146/annurev.cellbio.22.010305.104518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinberg</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Telomeres: cancer to human aging.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.aulast%253DBlanpain%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.spage%253D531%26rft.epage%253D557%26rft.atitle%253DTelomeres%253A%2Bcancer%2Bto%2Bhuman%2Baging.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.cellbio.22.010305.104518%26rft_id%253Dinfo%253Apmid%252F16824017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.cellbio.22.010305.104518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16824017&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242325100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.47" data-doi="10.1128/MCB.19.12.8083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakian</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Telomere-telomere recombination is an efficient bypass pathway for telomere maintenance in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">8083</span>–<span class="cit-lpage">8093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTeng%26rft.auinit1%253DS.-C.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D8083%26rft.epage%253D8093%26rft.atitle%253DTelomere-Telomere%2BRecombination%2BIs%2Ban%2BEfficient%2BBypass%2BPathway%2Bfor%2BTelomere%2BMaintenance%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.12.8083%26rft_id%253Dinfo%253Apmid%252F10567534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi84MDgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.48" data-doi="10.1091/mbc.11.7.2221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Exo1 roles for repair of DNA double-strand breaks and meiotic crossing over in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2221</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DTsubouchi%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D2221%26rft.epage%253D2233%26rft.atitle%253DExo1%2BRoles%2Bfor%2BRepair%2Bof%2BDNA%2BDouble-Strand%2BBreaks%2Band%2BMeiotic%2BCrossing%2BOver%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.7.2221%26rft_id%253Dinfo%253Apmid%252F10888664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjExLzcvMjIyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.49" data-doi="10.1038/embor.2008.155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zaidi</span>  <span class="cit-name-given-names">I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabut</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poveda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malmstrom</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rtt101 and Mms1 in budding yeast form a CUL4(DDB1)-like ubiquitin ligase that promotes replication through damaged DNA.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BReports%26rft.stitle%253DEMBO%2BReports%26rft.aulast%253DZaidi%26rft.auinit1%253DI.%2BW.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D1034%26rft.epage%253D1040%26rft.atitle%253DRtt101%2Band%2BMms1%2Bin%2Bbudding%2Byeast%2Bform%2Ba%2BCUL4%2528DDB1%2529-like%2Bubiquitin%2Bligase%2Bthat%2Bpromotes%2Breplication%2Bthrough%2Bdamaged%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2008.155%26rft_id%253Dinfo%253Apmid%252F18704118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/embor.2008.155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704118&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259682100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.197.50" data-doi="10.1038/ncb1428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zubko</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Linear chromosome maintenance in the absence of essential telomere-capping proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DZubko%26rft.auinit1%253DM.%2BK.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253D734%26rft.epage%253D740%26rft.atitle%253DLinear%2Bchromosome%2Bmaintenance%2Bin%2Bthe%2Babsence%2Bof%2Bessential%2Btelomere-capping%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1428%26rft_id%253Dinfo%253Apmid%252F16767084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16767084&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238837800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/3/197.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/3/187" title="High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/3/209" title="Mixed Model Association Mapping for Fusarium Head Blight Resistance in Tunisian-Derived Durum Wheat Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/3"><span>Volume 1 Issue 3, August 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/3.cover.gif?itok=eI_pz32_" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/197.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ksyalxXUhjYJIbsvIrBREIKhS6sHmvb_gfyXvhL2IOY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/3/197" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/3/197" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/3/197" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459171" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mNuMJoj5esJnr8URnmdw93j5nrYxm9SobIVPERoQ-jE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/3/197.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/3/197" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-G1TDdbLPt9aWEnB1yrPz6o0801kx2cpJnSdb4ZYbSog" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/3/197" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/3/197.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459171" id="node459171--2" data-pisa="ggg;1/3/197" data-pisa-master="ggg;g3.111.000216" data-apath="/ggg/1/3/197.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hsin-Yu Chang</span>, <span class="highwire-citation-author" data-delta="1">Conor Lawless</span>, <span class="highwire-citation-author" data-delta="2">Stephen G. Addinall</span>, <span class="highwire-citation-author" data-delta="3">Sarah Oexle</span>, <span class="highwire-citation-author" data-delta="4">Morgan Taschuk</span>, <span class="highwire-citation-author" data-delta="5">Anil Wipat</span>, <span class="highwire-citation-author" data-delta="6">Darren J. Wilkinson</span> and <span class="highwire-citation-author" data-delta="7">David Lydall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">197-208;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000216 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459171--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hsin-Yu Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-Yu%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-Yu%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen G. Addinall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BG.%2BAddinall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addinall%20SG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BG.%2BAddinall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sarah Oexle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BOexle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oexle%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BOexle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Morgan Taschuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morgan%2BTaschuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taschuk%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorgan%2BTaschuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anil Wipat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anil%2BWipat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wipat%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnil%2BWipat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Darren J. Wilkinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BJ.%2BWilkinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilkinson%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BJ.%2BWilkinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.a.lydall@ncl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.a.lydall@ncl.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/3/197" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/3/197&amp;title=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/3/197&amp;title=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/3/197&amp;title=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/3/197&amp;text=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/3/197&amp;title=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/3/197&amp;t=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/3/197&amp;title=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/3/197&amp;title=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459171/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459171" id="node459171--3" data-pisa="ggg;1/3/197" data-pisa-master="ggg;g3.111.000216" data-apath="/ggg/1/3/197.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hsin-Yu Chang</span>, <span class="highwire-citation-author" data-delta="1">Conor Lawless</span>, <span class="highwire-citation-author" data-delta="2">Stephen G. Addinall</span>, <span class="highwire-citation-author" data-delta="3">Sarah Oexle</span>, <span class="highwire-citation-author" data-delta="4">Morgan Taschuk</span>, <span class="highwire-citation-author" data-delta="5">Anil Wipat</span>, <span class="highwire-citation-author" data-delta="6">Darren J. Wilkinson</span> and <span class="highwire-citation-author" data-delta="7">David Lydall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">197-208;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000216 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459171--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hsin-Yu Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-Yu%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-Yu%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen G. Addinall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BG.%2BAddinall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addinall%20SG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BG.%2BAddinall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sarah Oexle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BOexle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oexle%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BOexle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Morgan Taschuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morgan%2BTaschuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taschuk%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorgan%2BTaschuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anil Wipat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anil%2BWipat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wipat%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnil%2BWipat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Darren J. Wilkinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computing Science and <span class='nlm-sup'>‡</span>School of Mathematics and Statistics, Newcastle University, Newcastle upon Tyne, NE1 7RU, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BJ.%2BWilkinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilkinson%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BJ.%2BWilkinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Medical School, Newcastle University, Framlington Place, Newcastle upon Tyne, NE2 4HH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Integrated Systems Biology of Ageing and Nutrition, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, NE4 5PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.a.lydall@ncl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.a.lydall@ncl.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=197&amp;rft.epage=208&amp;rft.atitle=Genome-Wide%20Analysis%20to%20Identify%20Pathways%20Affecting%20Telomere-Initiated%20Senescence%20in%20Budding%20Yeast&amp;rft.volume=1&amp;rft.issue=3&amp;rft.date=2011-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chang%2C+Hsin-Yu&amp;rft.au=Lawless%2C+Conor&amp;rft.au=Addinall%2C+Stephen+G.&amp;rft.au=Oexle%2C+Sarah&amp;rft.au=Taschuk%2C+Morgan&amp;rft.au=Wipat%2C+Anil&amp;rft.au=Wilkinson%2C+Darren+J.&amp;rft.au=Lydall%2C+David"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459171/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459171/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459171/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459171/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459171/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459171/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459171/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459171/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459171/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459171/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459171/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459171/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496043" id="node496043" data-pisa="ggg;g3.120.401101v1" data-pisa-master="ggg;g3.120.401101" data-apath="/ggg/early/2020/03/17/g3.120.401101.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.120.401101" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Adaptations of an Island Pit-Viper to a Unique Sedentary Life with Extreme Seasonal Food Availability</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/3/197" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/3/197.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/3/197.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/3/197.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/3/197.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
