<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/3/187" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/3/187.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/3/187.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/3/187.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/3/187.atom" />
<meta name="HW.pisa" content="ggg;1/3/187" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay" />
<meta name="DC.Identifier" content="10.1534/g3.111.000182" />
<meta name="DC.Date" content="2011-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Schmalenbach et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetically well-characterized mapping populations are a key tool for rapid and precise localization of quantitative trait loci (QTL) and subsequent identification of the underlying genes. In this study, a set of 73 introgression lines (S42ILs) originating from a cross between the spring barley cultivar Scarlett ( Hordeum vulgare ssp. vulgare ) and the wild barley accession ISR42-8 ( H. v . ssp. spontaneum ) was subjected to high-resolution genotyping with an Illumina 1536-SNP array. The array enabled a precise localization of the wild barley introgressions in the elite barley background. Based on 636 informative SNPs, the S42IL set represents 87.3% of the wild barley genome, where each line contains on average 3.3% of the donor genome. Furthermore, segregating high-resolution mapping populations (S42IL-HRs) were developed for 70 S42ILs in order to facilitate QTL fine-mapping and cloning. As a case study, we used the developed genetic resources to rapidly identify and fine-map the novel locus thresh-1 on chromosome 1H that controls grain threshability. Here, the recessive wild barley allele confers a difficult to thresh phenotype, suggesting that thresh-1 played an important role during barley domestication. Using a S42IL-HR population, thresh-1 was fine-mapped within a 4.3cM interval that was predicted to contain candidate genes involved in regulation of plant cell wall composition. The set of wild barley introgression lines and derived high-resolution populations are ideal tools to speed up the process of mapping and further dissecting QTL, which ultimately clears the way for isolating the genes behind QTL effects." />
<meta name="DC.Contributor" content="Inga Schmalenbach" />
<meta name="DC.Contributor" content="Timothy J. March" />
<meta name="DC.Contributor" content="Thomas Bringezu" />
<meta name="DC.Contributor" content="Robbie Waugh" />
<meta name="DC.Contributor" content="Klaus Pillen" />
<meta name="article:published_time" content="2011-08-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetically well-characterized mapping populations are a key tool for rapid and precise localization of quantitative trait loci (QTL) and subsequent identification of the underlying genes. In this study, a set of 73 introgression lines (S42ILs) originating from a cross between the spring barley cultivar Scarlett (&lt;i&gt;Hordeum vulgare&lt;/i&gt; ssp. &lt;i&gt;vulgare&lt;/i&gt;) and the wild barley accession ISR42-8 (&lt;i&gt;H. v&lt;/i&gt;. ssp. &lt;i&gt;spontaneum&lt;/i&gt;) was subjected to high-resolution genotyping with an Illumina 1536-SNP array. The array enabled a precise localization of the wild barley introgressions in the elite barley background. Based on 636 informative SNPs, the S42IL set represents 87.3% of the wild barley genome, where each line contains on average 3.3% of the donor genome. Furthermore, segregating high-resolution mapping populations (S42IL-HRs) were developed for 70 S42ILs in order to facilitate QTL fine-mapping and cloning. As a case study, we used the developed genetic resources to rapidly identify and fine-map the novel locus &lt;i&gt;thresh-1&lt;/i&gt; on chromosome 1H that controls grain threshability. Here, the recessive wild barley allele confers a difficult to thresh phenotype, suggesting that &lt;i&gt;thresh-1&lt;/i&gt; played an important role during barley domestication. Using a S42IL-HR population, &lt;i&gt;thresh-1&lt;/i&gt; was fine-mapped within a 4.3cM interval that was predicted to contain candidate genes involved in regulation of plant cell wall composition. The set of wild barley introgression lines and derived high-resolution populations are ideal tools to speed up the process of mapping and further dissecting QTL, which ultimately clears the way for isolating the genes behind QTL effects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/08/01" />
<meta name="citation_mjid" content="ggg;1/3/187" />
<meta name="citation_id" content="1/3/187" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/3/187" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/3/187.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/3/187.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/3/187.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000182" />
<meta name="citation_pmid" content="22384330" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="187" />
<meta name="citation_lastpage" content="196" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Inga Schmalenbach" />
<meta name="citation_author_institution" content="Max Planck Institute for Plant Breeding Research, Department of Plant Breeding and Genetics, 50829 Cologne, Germany," />
<meta name="citation_author" content="Timothy J. March" />
<meta name="citation_author_institution" content="Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and" />
<meta name="citation_author" content="Thomas Bringezu" />
<meta name="citation_author_institution" content="Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and" />
<meta name="citation_author" content="Robbie Waugh" />
<meta name="citation_author_institution" content="Scottish Crop Research Institute, Invergowrie, Dundee DD2 5DA, United Kingdom" />
<meta name="citation_author" content="Klaus Pillen" />
<meta name="citation_author_institution" content="Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and" />
<meta name="citation_author_email" content="klaus.pillen@landw.uni-halle.de" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=K. B. Alpert;citation_author=S. D. Tanksley;citation_title=High-resolution mapping and isolation of a yeast artificial chromosome contig containing fw2.2: A major fruit weight quantitative trait locus in tomato;citation_pages=15503-15507;citation_volume=93;citation_year=1996;citation_issue=26;citation_pmid=11038534;citation_doi=10.1073/pnas.93.26.15503" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=P. Azhacuvel;citation_author=D. Vidya-Saraswathi;citation_author=T. Komatsuda;citation_title=High-resolution linkage mapping for the non-brittle rachis locus btr1 in cultivated x wild barley (Hordeum vulgare).;citation_pages=1087-1094;citation_volume=170;citation_year=2006;citation_doi=10.1016/j.plantsci.2006.01.013" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. A. Burton;citation_author=G. Ma;citation_author=U. Baumann;citation_author=A. J. Harvey;citation_author=N. J. Shirley;citation_title=A customized gene expression microarray reveals that the brittle stem phenotype fs2 of barley is attributable to a retroelement in the HvCesA4 cellulose synthase gene.;citation_volume=153;citation_year=2010;citation_pmid=20530215;citation_doi=10.1104/pp.110.158329" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. H. Chin;citation_author=X. Lu;citation_author=S. M. Haefele;citation_author=R. Gamuyao;citation_author=A. Ismail;citation_title=Development and application of gene-based markers for the major rice QTL Phosphorus uptake 1.;citation_pages=1073-1086;citation_volume=120;citation_year=2010;citation_issue=6;citation_pmid=20035315;citation_doi=10.1007/s00122-009-1235-7" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Lonardi;citation_author=Y. H. Wu;citation_author=N. Rostoks;citation_title=Development and implementation of high-throughput SNP genotyping in barley.;citation_pages=582-582;citation_volume=10;citation_year=2009;citation_pmid=19961604;citation_doi=10.1186/1471-2164-10-582" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Cockram;citation_author=J. White;citation_author=D. L. Zuluaga;citation_author=D. Smith;citation_author=J. Comadran;citation_title=Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome.;citation_volume=107;citation_year=2010;citation_pmid=21115826;citation_doi=10.1073/pnas.1010179107" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. C. Cocuron;citation_author=O. Lerouxel;citation_author=G. Drakakaki;citation_author=A. P. Alonso;citation_author=A. H. Liepman;citation_title=A gene from the cellulose synthase-like C family encodes a beta-1,4 glucan synthase.;citation_volume=104;citation_year=2007;citation_pmid=17488821;citation_doi=10.1073/pnas.0703133104" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Comadran;citation_author=L. Ramsay;citation_author=K. Mackenzie;citation_author=P. Hayes;citation_author=T. J. Close;citation_title=Patterns of polymorphism and linkage disequilibrium in cultivated barley.;citation_pages=523-531;citation_volume=122;citation_year=2011;citation_issue=3;citation_pmid=21076812;citation_doi=10.1007/s00122-010-1466-7" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=W. Deleu;citation_author=C. Esteras;citation_author=C. Roig;citation_author=M. Gonzalez-To;citation_author=I. Fernandez-Silva;citation_title=A set of EST-SNPs for map saturation and cultivar identification in melon.;citation_pages=90-90;citation_volume=9;citation_year=2009;citation_pmid=19604363;citation_doi=10.1186/1471-2229-9-90" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Deng;citation_author=X. Wu;citation_author=Y. Wu;citation_author=R. Zhou;citation_author=H. Wang;citation_title=Characterization and precise mapping of a QTL increasing spike number with pleiotropic effects in wheat.;citation_pages=281-289;citation_volume=122;citation_year=2011;citation_issue=2;citation_pmid=20872211;citation_doi=10.1007/s00122-010-1443-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. Druka;citation_author=J. Franckowiak;citation_author=U. Lundqvist;citation_author=N. Bonara;citation_author=J. Alexander;citation_title=Genetic dissection of barley morphology and development.;citation_volume=155;citation_year=2011;citation_pmid=21088227;citation_doi=10.1104/pp.110.166249" />
<meta name="citation_reference" content="citation_journal_title=Molecular Plant;citation_author=F. M. Dwivany;citation_author=D. Yulia;citation_author=R. A. Burton;citation_author=N. J. Shirley;citation_author=S. M. Wilson;citation_title=The CELLULOSE-SYNTHASE LIKE C (CSLC) family of barley includes members that are integral membrane proteins targeted to the plasma membrane.;citation_volume=2;citation_year=2009;citation_pmid=19825677;citation_doi=10.1093/mp/ssp064" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Eshed;citation_author=D. Zamir;citation_title=An introgression line population of Lycopersicon pennellii in the cultivated tomato enables the identification and fine mapping of yield-associated QTL.;citation_pages=1147-1162;citation_volume=141;citation_year=1995;citation_pmid=8582620" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. D. Faris;citation_author=B. S. Gill;citation_title=Genomic targeting and high-resolution mapping of the domestication gene Q in wheat.;citation_pages=706-718;citation_volume=45;citation_year=2002;citation_issue=4;citation_pmid=12175074" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. C. Falke;citation_author=Z. Sušić;citation_author=B. Hackauf;citation_author=V. Korzun;citation_author=J. Schondelmaier;citation_title=Establishment of introgression libraries in hybrid rye (Secale cereale L.) from an Iranian primitive accession as a new tool for rye breeding and genomics.;citation_pages=641-652;citation_volume=117;citation_year=2008;citation_issue=4;citation_pmid=18535814;citation_doi=10.1007/s00122-008-0808-1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Frary;citation_author=T. C. Nesbitt;citation_author=S. Grandillo;citation_author=E. Knaap;citation_author=B. Cong;citation_title=fw2.2: A Quantitative Trait Locus Key to the Evolution of Tomato Fruit Size;citation_pages=85-88;citation_volume=289;citation_year=2000;citation_issue=5476;citation_pmid=10884229;citation_doi=10.1126/science.289.5476.85" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Fridman;citation_author=F. Carrari;citation_author=Y. S. Liu;citation_author=A. R. Fernie;citation_author=D. Zamir;citation_title=Zooming In on a Quantitative Trait for Tomato Yield Using Interspecific Introgressions;citation_pages=1786-1789;citation_volume=305;citation_year=2004;citation_issue=5691;citation_pmid=15375271;citation_doi=10.1126/science.1101666" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. L. Fu;citation_author=C. Uauy;citation_author=A. Distelfeld;citation_author=A. Blechl;citation_author=L. Epstein;citation_title=A Kinase-START Gene Confers Temperature-Dependent Resistance to Wheat Stripe Rust.;citation_volume=323;citation_year=2009;citation_pmid=19228999;citation_doi=10.1126/science.1166289" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Hackauf;citation_author=S. Rudd;citation_author=J. R. van der Voort;citation_author=T. Miedaner;citation_author=P. Wehling;citation_title=Comparative mapping of DNA sequences in rye (Secale cereale L.) in relation to the rice genome.;citation_pages=371-384;citation_volume=118;citation_year=2009;citation_issue=2;citation_pmid=18953524;citation_doi=10.1007/s00122-008-0906-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. Keurentjes;citation_author=L. Bentsink;citation_author=C. Alonso-Blanco;citation_author=C. J. Hanhart;citation_author=H. Blankestijn-De Vries;citation_title=Development of a near-isogenic line population of Arabidopsis thaliana and comparison of mapping power with a recombinant inbred line population.;citation_pages=891-905;citation_volume=175;citation_year=2007;citation_pmid=17179089;citation_doi=10.1534/genetics.106.066423" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Komatsuda;citation_author=M. Pourkheirandish;citation_author=C. F. He;citation_author=P. Azhaguvel;citation_author=H. Kanamori;citation_title=Six-rowed barley originated from a mutation in a homeodomain-leucine zipper I-class homeobox gene.;citation_volume=104;citation_year=2007;citation_pmid=17220272;citation_doi=10.1073/pnas.0608580104" />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=R. Kota;citation_author=R. K. Varshney;citation_author=M. Prasad;citation_author=H. Zhang;citation_author=N. Stein;citation_title=EST-derived single nucleotide polymorphism markers for assembling genetic and physical maps of the barley genome.;citation_pages=223-233;citation_volume=8;citation_year=2008;citation_issue=3;citation_pmid=17968603;citation_doi=10.1007/s10142-007-0060-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Law;citation_title=The location of genetic factors affecting a quantitative character in wheat.;citation_pages=487-498;citation_volume=53;citation_year=1966;citation_pmid=17248295" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. B. Liu;citation_author=R. G. Zhou;citation_author=Y. C. Dong;citation_author=P. Li;citation_author=J. Z. Jia;citation_title=Development, utilization of introgression lines using a synthetic wheat as donor.;citation_pages=1360-1373;citation_volume=112;citation_year=2006;citation_issue=7;citation_pmid=16550399;citation_doi=10.1007/s00122-006-0238-x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. J. Lorenz;citation_author=M. T. Hamblin;citation_author=J. L. Jannink;citation_title=Performance of single nucleotide polymorphisms versus haplotypes for genome-wide association analysis in barley.;citation_pages=e14079-e14079;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21124933;citation_doi=10.1371/journal.pone.0014079" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=T. C. Marcel;citation_author=R. Aghnoum;citation_author=J. Durand;citation_author=R. K. Varshney;citation_author=R. E. Niks;citation_title=Dissection of the barley 2L1.0 region carrying the 'Laevigatum' quantitative resistance gene to leaf rust using near-isogenic lines (NIL) and subNIL.;citation_pages=1604-1615;citation_volume=20;citation_year=2007;citation_issue=12;citation_pmid=17990968" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=K. F. X. Mayer;citation_author=S. Taudien;citation_author=M. Martis;citation_author=H. Simkova;citation_author=P. Suchankova;citation_title=Gene Content and Virtual Gene Order of Barley Chromosome 1H.;citation_volume=151;citation_year=2009;citation_pmid=19692534;citation_doi=10.1104/pp.109.142612" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. W. Mei;citation_author=J. L. Xu;citation_author=Z. K. Li;citation_author=X. Q. Yu;citation_author=L. B. Guo;citation_title=QTLs influencing panicle size detected in two reciprocal introgressive line (IL) populations in rice (Oryza sativa L.).;citation_pages=648-656;citation_volume=112;citation_year=2006;citation_issue=4;citation_pmid=16331475;citation_doi=10.1007/s00122-005-0167-0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Moragues;citation_author=J. Comadran;citation_author=R. Waugh;citation_author=I. Milne;citation_author=A. J. Flavell;citation_title=Effects of ascertainment bias and marker number on estimations of barley diversity from high-throughput SNP genotype data.;citation_pages=1525-1534;citation_volume=120;citation_year=2010;citation_issue=8;citation_pmid=20157694;citation_doi=10.1007/s00122-010-1273-1" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. Muchero;citation_author=J. D. Ehlers;citation_author=T. J. Close;citation_author=P. A. Roberts;citation_title=Genic SNP markers and legume synteny reveal candidate genes underlying QTL for Macrophomina phaseolina resistance and maturity in cowpea [Vigna unguiculata (L) Walp.].;citation_pages=8-8;citation_volume=12;citation_year=2011;citation_pmid=21208448;citation_doi=10.1186/1471-2164-12-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Ogawa;citation_author=P. Kay;citation_author=S. Wilson;citation_author=S. M. Swain;citation_title=Arabidopsis dehiscence zone polygalacturonase 1 (Adpg1), Adpg2, and Quartet2 are polygalacturonases required for cell separation during reproductive development in Arabidopsis.;citation_volume=21;citation_year=2009;citation_pmid=19168715;citation_doi=10.1105/tpc.108.063768" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. Pillen;citation_author=A. Binder;citation_author=B. Kreuzkam;citation_author=L. Ramsay;citation_author=R. Waugh;citation_title=Mapping new EMBL-derived barley microsatellites and their use in differentiating German barley cultivars.;citation_pages=652-660;citation_volume=101;citation_year=2000;citation_doi=10.1007/s001220051527" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Ramsay;citation_author=J. Comadran;citation_author=A. Druka;citation_author=D. F. Marshall;citation_author=W. T. B. Thomas;citation_title=INTERMEDIUM-C, a modifier of lateral spikelet fertility in barley, is an ortholog of the maize domestication gene TEOSINTE BRANCHED 1.;citation_pages=169-172;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217754;citation_doi=10.1038/ng.745" />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=M. Röder;citation_author=X.-Q. Huang;citation_author=A. Börner;citation_title=Fine mapping of the region on wheat chromosome 7D controlling grain weight.;citation_pages=79-86;citation_volume=8;citation_year=2008;citation_issue=1;citation_pmid=17554574;citation_doi=10.1007/s10142-007-0053-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Rostoks;citation_author=L. Ramsay;citation_author=K. MacKenzie;citation_author=L. Cardle;citation_author=P. R. Bhat;citation_title=Recent history of artificial outcrossing facilitates whole-genome association mapping in elite inbred crop varieties.;citation_volume=103;citation_year=2006;citation_pmid=17085595;citation_doi=10.1073/pnas.0606133103" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Sato;citation_author=K. Takeda;citation_title=An application of high-throughput SNP genotyping for barley genome mapping and characterization of recombinant chromosome substitution lines.;citation_pages=613-619;citation_volume=119;citation_year=2009;citation_issue=4;citation_pmid=19488734;citation_doi=10.1007/s00122-009-1071-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Schmalenbach;citation_author=N. Körber;citation_author=K. Pillen;citation_title=Selecting a set of wild barley introgression lines and verification of QTL effects for resistance to powdery mildew and leaf rust.;citation_pages=1093-1106;citation_volume=117;citation_year=2008;citation_issue=7;citation_pmid=18663425;citation_doi=10.1007/s00122-008-0847-7" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Schmalenbach;citation_author=J. Léon;citation_author=K. Pillen;citation_title=Identification and verification of QTLs for agronomic traits using wild barley introgression lines.;citation_pages=483-497;citation_volume=118;citation_year=2009;citation_issue=3;citation_pmid=18979081;citation_doi=10.1007/s00122-008-0915-z" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Schmalenbach;citation_author=K. Pillen;citation_title=Detection and verification of malting quality QTLs using wild barley introgression lines.;citation_pages=1411-1427;citation_volume=118;citation_year=2009;citation_issue=8;citation_pmid=19255740;citation_doi=10.1007/s00122-009-0991-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=D. Schulte;citation_author=T. J. Close;citation_author=A. Graner;citation_author=P. Langridge;citation_author=T. Matsumoto;citation_title=The International Barley Sequencing Consortium--At the Threshold of Efficient Access to the Barley Genome;citation_pages=142-147;citation_volume=149;citation_year=2009;citation_issue=1;citation_pmid=19126706;citation_doi=10.1104/pp.108.128967" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. J. Simons;citation_author=J. P. Fellers;citation_author=H. N. Trick;citation_author=Z. Zhang;citation_author=Y. S. Tai;citation_title=GENETICS;citation_pages=547-555;citation_volume=172;citation_year=2006;citation_pmid=16172507" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Sood;citation_author=V. Kuraparthy;citation_author=G. Bai;citation_author=B. S. Gill;citation_title=The major threshability genes soft glume (sog) and tenacious glume (Tg), of diploid and polyploid wheat, trace their origin to independent mutations at non-orthologous loci.;citation_pages=341-351;citation_volume=119;citation_year=2009;citation_issue=2;citation_pmid=19421730;citation_doi=10.1007/s00122-009-1043-0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Stein;citation_author=M. Prasad;citation_author=U. Scholz;citation_author=T. Thiel;citation_author=H. N. Zhang;citation_title=A 1,000-loci transcript map of the barley genome: new anchoring points for integrative grass genomics.;citation_pages=823-839;citation_volume=114;citation_year=2007;citation_issue=5;citation_pmid=17219208;citation_doi=10.1007/s00122-006-0480-2" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. J. Szalma;citation_author=B. M. Hostert;citation_author=J. R. LeDeaux;citation_author=C. W. Stuber;citation_author=J. B. Holland;citation_title=QTL mapping with near-isogenic lines in maize.;citation_pages=1211-1228;citation_volume=114;citation_year=2007;citation_issue=7;citation_pmid=17308934;citation_doi=10.1007/s00122-007-0512-6" />
<meta name="citation_reference" content="citation_journal_title=Okayama University;citation_author=R. Takahashi;citation_author=J. Hayashi;citation_title=Linkage study of two complementary genes for brittle rachis in barley. Ber Ohara Inst Landw Biol;citation_pages=99-105;citation_volume=12;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Turner;citation_author=J. Beales;citation_author=S. Faure;citation_author=R. P. Dunford;citation_author=D. A. Laurie;citation_title=The Pseudo-Response Regulator Ppd-H1 Provides Adaptation to Photoperiod in Barley;citation_pages=1031-1034;citation_volume=310;citation_year=2005;citation_issue=5750;citation_pmid=16284181;citation_doi=10.1126/science.1117619" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. Uauy;citation_author=A. Distelfeld;citation_author=T. Fahima;citation_author=A. Blechl;citation_author=J. Dubcovsky;citation_title=A NAC Gene regulating senescence improves grain protein, zinc, and iron content in wheat.;citation_pages=1298-1301;citation_volume=314;citation_year=2006;citation_issue=5803;citation_pmid=17124321;citation_doi=10.1126/science.1133649" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. van Berloo;citation_title=Computer note. GGT: software for the display of graphical genotypes;citation_pages=328-329;citation_volume=90;citation_year=1999;citation_issue=2;citation_doi=10.1093/jhered/90.2.328" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. K. Varshney;citation_author=P. J. Hiremath;citation_author=P. Lekha;citation_author=J. Kashiwagi;citation_author=J. Balaji;citation_title=A comprehensive resource of drought- and salinity- responsive ESTs for gene discovery and marker development in chickpea (Cicer arietinum L.).;citation_pages=523-523;citation_volume=10;citation_year=2009;citation_pmid=19912666;citation_doi=10.1186/1471-2164-10-523" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. von Korff;citation_author=H. Wang;citation_author=J. Léon;citation_author=K. Pillen;citation_title=Development of candidate introgression lines using an exotic barley accession ( Hordeum vulgare ssp. spontaneum) as donor.;citation_pages=1736-1745;citation_volume=109;citation_year=2004;citation_issue=8;citation_pmid=15502912;citation_doi=10.1007/s00122-004-1818-2" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Wang;citation_author=X. Z. Liao;citation_author=Y. L. Li;citation_author=R. H. Zhou;citation_author=X. J. Yang;citation_title=Fine mapping a domestication-related QTL for spike-related traits in a synthetic wheat.;citation_pages=798-804;citation_volume=53;citation_year=2010a;citation_issue=10;citation_pmid=20962886" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. W. Wang;citation_author=I. Schmalenbach;citation_author=M. von Korff;citation_author=J. Léon;citation_author=B. Kilian;citation_title=Association of barley photoperiod and vernalization genes with QTLs for flowering time and agronomic traits in a BC2DH population and a set of wild barley introgression lines.;citation_pages=1559-1574;citation_volume=120;citation_year=2010b;citation_issue=8;citation_pmid=20155245;citation_doi=10.1007/s00122-010-1276-y" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. L. Yan;citation_author=A. Loukoianov;citation_author=A. Blechl;citation_author=G. Tranquilli;citation_author=W. Ramakrishna;citation_title=The Wheat VRN2 Gene Is a Flowering Repressor Down-Regulated by Vernalization;citation_pages=1640-1644;citation_volume=303;citation_year=2004;citation_issue=5664;citation_pmid=15016992;citation_doi=10.1126/science.1094305" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=L. L. Yan;citation_author=J. von Zitzewitz;citation_author=J. S. Skinner;citation_author=P. M. Hayes;citation_author=J. Dubcovsky;citation_title=Molecular characterization of the duplicated meristem identity genes HvAP1a and HvAP1b in barley.;citation_pages=905-912;citation_volume=48;citation_year=2005;citation_issue=5;citation_pmid=16391696" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Yan;citation_author=D. Fu;citation_author=C. Li;citation_author=A. Blechl;citation_author=G. Tranquilli;citation_title=The wheat and barley vernalization gene VRN3 is an orthologue of FT.;citation_volume=103;citation_year=2006;citation_pmid=17158798;citation_doi=10.1073/pnas.0607142103" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Zamir;citation_title=Improving plant breeding with exotic genetic libraries.;citation_pages=983-989;citation_volume=2;citation_year=2001;citation_issue=12;citation_pmid=11733751;citation_doi=10.1038/35103589" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
<meta name="twitter:description" content="Genetically well-characterized mapping populations are a key tool for rapid and precise localization of quantitative trait loci (QTL) and subsequent identification of the underlying genes. In this study, a set of 73 introgression lines (S42ILs) originating from a cross between the spring barley cultivar Scarlett ( Hordeum vulgare ssp. vulgare ) and the wild barley accession ISR42-8 ( H. v . ssp. spontaneum ) was subjected to high-resolution genotyping with an Illumina 1536-SNP array. The array enabled a precise localization of the wild barley introgressions in the elite barley background. Based on 636 informative SNPs, the S42IL set represents 87.3% of the wild barley genome, where each line contains on average 3.3% of the donor genome. Furthermore, segregating high-resolution mapping populations (S42IL-HRs) were developed for 70 S42ILs in order to facilitate QTL fine-mapping and cloning. As a case study, we used the developed genetic resources to rapidly identify and fine-map the novel locus thresh-1 on chromosome 1H that controls grain threshability. Here, the recessive wild barley allele confers a difficult to thresh phenotype, suggesting that thresh-1 played an important role during barley domestication. Using a S42IL-HR population, thresh-1 was fine-mapped within a 4.3cM interval that was predicted to contain candidate genes involved in regulation of plant cell wall composition. The set of wild barley introgression lines and derived high-resolution populations are ideal tools to speed up the process of mapping and further dissecting QTL, which ultimately clears the way for isolating the genes behind QTL effects." />
<meta name="og-title" property="og:title" content="High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/3/187" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetically well-characterized mapping populations are a key tool for rapid and precise localization of quantitative trait loci (QTL) and subsequent identification of the underlying genes. In this study, a set of 73 introgression lines (S42ILs) originating from a cross between the spring barley cultivar Scarlett ( Hordeum vulgare ssp. vulgare ) and the wild barley accession ISR42-8 ( H. v . ssp. spontaneum ) was subjected to high-resolution genotyping with an Illumina 1536-SNP array. The array enabled a precise localization of the wild barley introgressions in the elite barley background. Based on 636 informative SNPs, the S42IL set represents 87.3% of the wild barley genome, where each line contains on average 3.3% of the donor genome. Furthermore, segregating high-resolution mapping populations (S42IL-HRs) were developed for 70 S42ILs in order to facilitate QTL fine-mapping and cloning. As a case study, we used the developed genetic resources to rapidly identify and fine-map the novel locus thresh-1 on chromosome 1H that controls grain threshability. Here, the recessive wild barley allele confers a difficult to thresh phenotype, suggesting that thresh-1 played an important role during barley domestication. Using a S42IL-HR population, thresh-1 was fine-mapped within a 4.3cM interval that was predicted to contain candidate genes involved in regulation of plant cell wall composition. The set of wild barley introgression lines and derived high-resolution populations are ideal tools to speed up the process of mapping and further dissecting QTL, which ultimately clears the way for isolating the genes behind QTL effects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
    <title>High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4AltWSSXPUfm-AJneeq7DXmsPrJHgHXn8DGMXl-AVW8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459170","apath":"\/ggg\/1\/3\/187.atom","pisa":"ggg;1\/3\/187","ac":{"\/ggg\/1\/3\/187.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/3\/187.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/3\/187"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/3\/187"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/3\/187":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022W4p6qSCjxjdKO97u96J2LZMYKsayJQEu5gLr3NUgx6nmo3M\\\\\\\/SLSN2rmETrRRQzP8TfFhfFjOXOxiFKafe9Fjg+2BISxap9k1VprVNGEig0Pg0UMuIT0YzEt983hnXCJUL7HN6xFgs\\\\\\\/OKzwhactYQaohhOdFD3EgqPx8ew\\\\\\\/yyn4tAbFv0ogRzn0PS+MyLKfoEYggap1IbRG8h\\\\\\\/sQh00EwgmMcumHBJtkZXklE1bOecjB+L6detrPZ4JSfJlUdo4q3YSQlvTolvXXzVgQzZR1p2VMTY8IOG85OdbdUV+hsxhXkx4RIt\\\\\\\/a0HnDEbdpXOjbaMD1D6hTmmlnicI7nkxeTZyrys7zwuZdxxJnLHgHlUD4k9cf9PSy1aUhtHcpF8+bZthrtuXbPRvc1CmEQcv6gPxFeU+RHLNfIJEA05go8KSUtZn5iIQeoUPkbTO2j5NcKe6LTTMpzJwOThE7F0x4MGpoFuKUc8CGAx8\\\\\\\/ysYszruSs7ciWFosDoCnYMPe\\\\\\\/1BCLR+IRuG9S4FMLdwaXXbABGvHbFg6sn63EB4AXxrCPgotAs2AnE3knnnsuuMqDnl3HyfeD1O71gnbJ+Tn4zl3kaYkFPkqxc4LXccChxFmCxa7F0gt0SDMAo9DCukTWwKn7v7hwowYSC\\\\\\\/GzwAXP3CbM45dlmCIkN4jG5QQq+72\\\\\\\/BPyaP7QmJAXKrdp3cTwbuxSB6kfcjSrP6BQPlSScbJPBBd9VqjTD1xWVg28eRmkhMpWeoDUbsESVuGe2OEzYaBgp2+QiZVBzDgLeATMOYkchl0nMYPVit\\\\\\\/PZ778fqrbAuvXxCMcE12iF7SIUy\\\\\\\/SZ1GdS+HxaGgREf6EEopDlCeaKwbkIxuM6B65HhWtDhx0TmvE9v02ctSansqJzV2DLOxolzT8lMEd3EODIyNBhJJZS61xxa13LoaRc8EKxaN0EQTzE1vliBqGYi1jSaeEI8A7Po7eQH90+QAUrWbfyuNazgXuBXwuwo0EBT1s0\\\\\\\/7QtajCvUs2mnpsHBI89XOfR6Y0ZMtrAB3jLqC\\\\\\\/LkOYdbp1MOk98O\\\\\\\/1lp4RWaWMMyWFF4qHptHM6lZoLrUA1np1e8kvriIfjK663kjOnbzxOg2CxUMZEtVP4RDjQ6GQLcqqcYkeAwYFHUQ964GamqjTgm0G4a+U7EkGUfqRoVHm9lmiar9+rB9XKX9O\\\\\\\/zRraNSECGNHH4Yn1peFpWxID0z3BscC1tOkUJl49J+JTbu13D\\\\\\\/JIl2c0yJ0zI3nGS+VhfA86ZWmkURKU\\\\\\\/tF84eqjKSSl1\\\\\\\/d9\\\u0022,\\\u0022iv\\\u0022:\\\u0022qt+aVWJdaQ6Vwp8KjpbA7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228f239b18fedd4f98b3df0f7400bae26c3d80156fa3e9c77c4bf60d35f99b3ed4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mjjQ\\\\\\\/cECLF+w8qgc9Qeh5ikAV8uiQgqqHx5lqnpSKXV\\\\\\\/prgyHa\\\\\\\/CFMt8+ZUFwZWDmzmUApwNFpUprO\\\\\\\/c1\\\\\\\/NanX6FbuXroChpAnC39fIQn\\\\\\\/yohJOJW+BJEMVZOSWKwoqunADSFUIrdpKVMKqqDtj1WTknxeArrmVzT4p4NnNcA31Wcu3APrhjsgDYyc1sJAiVNY0T+j4ss2UexQhGzAMsWfgXYrptf0R2EA\\\\\\\/UwZ6uI69OCT0TCkUs3vt3Wck74\\\\\\\/g3QAs7tE6pKlDrC8tkaorFSiuqLV3Z8DN8zfDVDUFTkyiclj+Te0T80BQKJjB\\\\\\\/z0tHZNKq3K8iY+f367eYa74uLxpq5jYNo9Iwby3ltwZO7ta+z1FeBovvr5Zf6qK\\\\\\\/Yo7Aur+yoCGwv9++2CjrE+IkD2pd4TGPLvfmEiEvgtOZSkc8ujagR\\\\\\\/TMxRhtQbc9iqsOrlMAdYXTZ2Ejd04M3oevLidseFt8RK1oBPVC485k6iiYadPCTnnvbr3Tbej37rBzd1spZfClilT2ohaAlgyMFVqQy6wSAzVP0yiv3gJ8GMpC6AinMJctPgkH2QNPHCluHeDP0arpjRTcotbEAYKgnfo05h8Mobqfgm9qJ7eZ4NLnLkj5c\\\\\\\/dzvCdv7SVd4rgAAd7pk96I2QecYCiBOfHeK7HFy6WbrRH7blxtt7lEFmZ6Cb7G6XGIrSloololFQSUemLTJDyTkqDRT8TOBtmCP+ufCi9PglE\\\\\\\/r7ua+kSlr+g+T31FvsucY9iVW4iKeokHCGYQiFPMgfrWNxSC0j5lShRGn4ifoNR2qARO0e7Qila0eQVKQrFv+F1jSwM2Ew1fiWHdRoYg0CHzuXcWFCfX\\\\\\\/G+koUQBpD7OOEXRH5hW9C1TX4GAUUvgVmnfAy8Fm8YzsPi2gay5XzcJK3DkxD5GyLEPg5\\\\\\\/uplQKMZeu1cJUsTGqgIktg97CWzulI6HEPSce+rFP+5xnC+hn1HxW380shhtK2tx9vAPbbowBY7QZrEnCJFVMQ2oMYvlHupWgPNNTFFW8szY+nhjnD34rbzJIOAgZeef7jUrAalcxZooYxodUD6IZas6EbE2JBkAeDiQpjQOvQ9VuUKvmx40LlYfQuuAFe\\\\\\\/2oaaPvNEPa+nvscEQ4pXC+hS0iLBzwMNj\\\\\\\/kSXNdX\\\\\\\/pdAUPNDYeZLIgspAY8u2IAYLUlBMxxYOGtZmD+VXRQOq6fY9DjMg\\\\\\\/97\\\\\\\/NIjWQpbXKBkRLZFtuwQmeZUGNSZwx2UgFxnJRyHmvgJPH11zkCNwaZM7VIPoae9dS0kls\\\\\\\/FndU7MjQ\\\\\\\/QSJLJ6dHwGtDo+lgdOQ7C7xQic87EsQCtG4HGs4vx7BTi41jipG675jsnBZWLwVLNfOJNo8HE8K5UDJdkuKTtMfUXhjSpSM1Xgu2Znfl7puhMN4+nOdAPry8cE9NXANx2FxtbpwvLbQqDkGH\\\\\\\/PUfjJHH+2jjkf5mpHArvUfW11vIrOl4yKapU8vKJyg\\\\\\\/2a5IPQbOqnAelkEo342KJ2WJycFiS4IKrwxvmh7qjm0ljlJhUBNgLbsmHZdd9RG\\\\\\\/j4VrszDqxmFD6rPsvOoy2XaJGbxJ0H+JSwPgx\\\\\\\/05NgTEcJYWUg4buxDeWwI51A5qQjzqTITLfhCQ8U\\\\\\\/IFDRban8uK2dB9qW38TBqWUG3I2i9AZElgo9DqsmFjM1qPs7SGc4LT3DYw9sW1eMS+hZGRWsZq4WgERYfC5Iun7jms3u+S7LHflNnnez1\\\\\\\/azj0\\\\\\\/r+7SDWBXEDEMoUm2YUP\\\\\\\/\\\\\\\/Ny5XocRqvx0RIkYjM\\\\\\\/EJ9KaXosY\\\\\\\/XOnlvw79FC9ActxfjRvVyw\\\\\\\/FxdLgAqHGr\\\\\\\/5rgAjwofcBJ7epuA1eB3K67gTz5Qz3Begh1Md8kndtFUbOuPwZbqk58AgRnA82oud8WFl\\\\\\\/Y9QRQa8pUrULf7l0vlYznHJ5MvnHPOhkbKQjI7BBkQaS4PWcOWxKM+hxmz27NAhq43j9dvqIDliahkooD3XawaC4EfZifbS27e8HjYxboQnOryX38oC\\\\\\\/thnsRdH2Zj9rvFckWaDop0\\\\\\\/TtBcfW1lJ+\\\\\\\/r1PDfbcbabGAqmTpo97iupVIwBh0hysfg0BNHON9uaYkPsUksevqdWnXGBETXSbZIU78y7tkhu61orK4++q7Io7qLdXqQMxbLE\\\\\\\/NRfT1XHarfyeT8UZX6VqF7kZMethWzevE\\\\\\\/5NhVJwlD5cdW7\\\\\\\/ydOc2ZsWcuIiPtU6Gdk70\\\\\\\/21xvgn7CcU4nmS\\\\\\\/LnE\\\\\\\/6aPnQfMSCElfzd3K9rXp4guD2o1S9ShCAYzaAGRNjurdIHZRtMeqKw3j7FYXqBR738U2+a2XutR+74YWtMcD27jnbNx+\\\\\\\/tSq57F8A9m\\\\\\\/D1vvG4zuzm6wizF7JB+yUMC7Mp\\\\\\\/\\\\\\\/GSwdSZ62b3mMOai3QGjDl5MnZKsxRnArqQFtZdLItoTmuiSG0nZDK7Mh85CelFPYyug18q4U8OITGCykII6EZylzOPSJ0f6fBkOPN7N8b+EXK6aamVJh3t52eq2HvshVinrHUg13nxdTczC9d\\\\\\\/Yzzj8OKAhduRby1bFXNzHwm7Zv0DcI8\\\\\\\/a6Q1i\\\\\\\/fQEB2N5FO8earJoWqe84TOtTGtmM\\\\\\\/UD\\\\\\\/RR2Hj3ABWTFMX8lb3V6gX\\\\\\\/M20ZxyQ\\\\\\\/mP4IftG+B2aYsVqWbBUGw2HvEACcBGz6iqtRJpefHTk\\\\\\\/v0t2xHtZQ8LuAEwVCogZnVlxiZ7\\\\\\\/iX72Y7VhcvmSEZk+N1oEgNsyJE7ibxTiRkxuEzWJmRuY2mV\\\\\\\/Bdie+\\\\\\\/4fk43vBJe3linwxyL9zBgjSv2S7XQGgcFHU6xbQnhV98sD90Dg7rpD2GGSFcXRSEE+A0Y1g+cPp2f1IV04\\\\\\\/dCaWxeTXGynP3dfTBeNsQ3\\\\\\\/Euhy2YNuQ1hTw4YoFF3t\\\\\\\/7SAf7uUs9S7Zc5nJou\\\\\\\/kvzoqK9jagOUCbvEJCKmXRCIivHPWlTNP\\\\\\\/NTJZiu8fvrzUeqd5RSg3h+54BQdPpJoe0dyOrROLyWdjEBx51QC1lUb9RnTs95h+fOwsUeFzpljfDBBrIXYblr8edhEPHzARVi\\\\\\\/7W6v9VN2sygbxARfXH9JalZ+yJaZ0Yuq8vvC1z71aZ8dbWT0EYTriG8Azw5HxrQpoBU476Y59sCEMEgwwCqYScX+L8DHyBOCaTRflO0Js5AAa4BkjmwoHBodfWu4MZIxWe6\\\\\\\/bpgWVkXGnfos\\\\\\\/NZSNWctY04\\\\\\\/D4xpt4BL4HHuRIK2QZx51CPXVQkWTfHmKq8athB5Gurk\\\\\\\/laPnnhE9WXMrmxxUSQCgXeOAK09Yu9hKboRmByvIAjU4wl3KAqmOfukkHjHQXoEBN2QdJRXHi2eujIYuZeKXmsJnZVMCdcGTWawOMBYf7gXv3QUrBE2WMk0ouKmdt32YVktQV3WB185VfUhP6NIcQtXGlO8iqzOEoiIJGdNAnVskWWgdiAN+fIlaybUNVoy6+GWeHMSdODLT7gGjTqjV+zHd7Mmzyfzy93Ohz\\\\\\\/mi4aQO6SYN0fjtZ79B3Ssv0zh\\\\\\\/Z7ERSfOFwD\\\\\\\/1KeSDAmvqUXITOFx2mcm4BhgfVAfyNU51\\\\\\\/AcdyZPzKe\\\\\\\/9tKKFy9vofbJNwFBBjm9qbAq5QMUX0vnP4Z59i82jDceaOwEpfFQ2nKrdsmoo\\\\\\\/MNBGdvO+GCqp+ytC2NSmw7bLyuKmFU0Ow9iXTiguAL0ku697cmnU4IxoiiaVmAOw7VCyr33wWLKr9P4Momv7YJ57XBLcmnluliQpAPrlj8BzKJ5F2YSX7q2zLVgUYg\\\\\\\/cbUyqrSOoXnpg61gf4QHbyGMFcVnnYJuLqUFw4pgWvWNKS5dk8nuD8wtzpEjKX6pQ\\\\\\\/rvUnoLFtAdrdOWKM1L3HDK9oGVmiF6DdiNS\\\\\\\/HZJfJPz\\\\\\\/7xOMs9oXF8NyX6lkEbYAxCjdBOAB8fg8+e0gW1I4GwezkbNukMnQac6TlAyTFRrz3CyMZYl2vntlDA1k+Ou5IPc3rnSUR4OE6AGJwLzkMcd96KUMjRyzQLLkdnEMLNYwf5ndHVGVNt8QoIbUUpxdgNmk259Lppi9uFG726ckwk4Fa27KFjWL56Pxle885tsKAZcDmTWKf\\\\\\\/dBurprvbfU9jpqDRg8hlkhptIvnzUaTpTho6iExmsQu05X+2r7U0CVEBlJ21ne+wahDD8Jq\\\\\\\/p\\\\\\\/aX95U85g1Ig1K53o3BZYX9Qj5\\\\\\\/7g+oHn36YjwS\\\\\\\/q91Fqjk2FHI3D5o+5PL6ROtAcEOEXmQd\\\\\\\/PqQ7xZh5YLZcB1OeoquNwiKXxTJbLzqIrftJ3LLurnQhIW614ePTe3sqfyPuZlJ7aUiKIFDAeyWIy\\\\\\\/pNQ\\\\\\\/0uFpzrsUXxhHenSQt4KWq1aXydSIK3dpPVBTl9zlQs4QBPt99qYdMfnjA+ziEMLaAOFdQXgoedgtf7RTYbMnOUPEJBUQFhYwYV5WIasZbnGz61fU2qc9+W0kPWCQtPbHaK5ht4lVhcTQ8ljcwNhj5xc5mIw1LkYlCZlVr+FYf9zh4UWd3CcYfl7c+PWurtGB5kU7oApu9llFJcZ\\\\\\\/vcKJvWhFqJaizxRD1vVnyVSy5j3gnRiDutU5sAe8O\\\\\\\/qN67EIqhFFxHNiXI\\\\\\\/WysAOWEwlqQHvjIALDmKMiKW\\\\\\\/ykl66LYxqloEotmLyjEPDwy8ACTQoRWMdgKZknrDcl5tq4wuPoCRC7eVzYqfk1a2n3FoCVjmK4beu4Eue\\\\\\\/4g5u2sRCLybQvWrUhJON4X5B43g1bYVi0QiFuXceRCXeJ+C89sLj7r4+69AMKfE57YTiD1TCwGksu2VgRuWsQKQE7r\\\\\\\/DXtBfUyX076fQotY0dHxVJHmdf6v467Cec6f0BzLR3TT9e7+8pVLX8X5Gu9YdtiSR7LcJaAeuDRG9HKO2QFP6ohWdCoe0mnTBFC+sZF20FZSvjFWFSVObjANZ4ZCAyP6tcA8hBBOXcgTqQi9RPbrqPqlymPweyxCGQSIsEHDOURdqqjvlM86yjqafBpslDm2XUAJvtyzQiWMKiUr1nisyvP5mKTwbB46aQ0aBHkNp5M0b5JoHw4RF0CTBJsqDpCRZqGKxcjExPftk6R5XNohuhAFRlax2cswHayl3cwTq2oT34mwB\\\\\\\/dBokTielciMfHZii2kQdQhr\\\\\\\/D5AQLE51nUVxM9Y4SiY8kUml957K2FzB3bd34epyP7upR2qW9DLIlQcXh1\\\\\\\/Af90CZp\\\\\\\/XsOP3XdyZHHKpY+WSTPpHCRuwCm1lVrk1dlniTfWcQE8x\\\\\\\/jbDL\\\\\\\/C9QLKIrhqyhKGR28v1T\\\\\\\/FlsclMyQbimik2hiV2lyqyn5JYdWeoURGx2V\\\\\\\/vMP4jcQsePnyMmh\\\\\\\/60U5GU81eDVNDW4ii1vGN7YMgSew3EeSd5U0nLSkediPTcVJW82wUHJhSVkJEEE2JMEnC3WxvqH81ZHGi1sorLJvB1vSA8zLIMTXVedE1SUOxL1XmlL6\\\\\\\/yIaTxM69SYoaXgZ9pvNMr68Dpfork2f6FTXMkvW+LJ8MjVLsH5Mq\\\\\\\/GHDw8RRDScXBq4OqzTBLYsKdTt7zhY82oeKrLgj+k4OVd5OYJx6iazLJ5RP2RFOdVW3S\\\\\\\/kGsmItOLX4+vrKPWT9ZFkJnliOyVu9aHCXBLNrXHypibvsmyi5YO3KFYagixRBnGZUX6\\\\\\\/b0lBBHU+dImg0sMrCcXXFretzM1LvjPDSPWbONVbZagrHZ2ajJ\\\\\\\/ZrMFbNuj1JQd3fIZ2lLhgwruiedbZeBRezgSVXsIaRTGWj2\\\\\\\/Mdlbp98vkTVzhVtTuswbmC+RUbiWhDwnfqF3FDCJ+GCN6jE7brdBRN+2sEJ7c0SCmKd\\\\\\\/V9hy992o6jmROI+A6CfaDrRvNiUm8mBSr4FLxTdNE\\\\\\\/hVkXiYNLy5KLKJnLhc88RQAi63NhYp+nV1E4ZjaGSdvj2cOJLZJJLRdL8el3GiDmCwxNqxJyvUjRn0EruFx2eurXu\\\\\\\/7CfqDrsbFacqsi9kJX82aGQDuJgPuxnhiEzl8iMOI4yeSChK5rPql5N5fKRJacvU7ihqc67cOAf\\\\\\\/NFoY4z1dOolqWHDaHFmkXUGFJUaVFwnEJR1nDiXWzSCL08xvqkG7IS7qgEZlKUHw6rG9+0WXb3MY5M+TY\\\\\\\/Ghei3vCHvciZi8F3Q7Udsd7CUg2Jc3+FQUkrcKe5GIO3mcy\\\\\\\/03LcBksrP+mTVipTm6FlQaX3h+GucENfBwX5WtlcQpw+nxUS\\\\\\\/h\\\\\\\/kLVezJrV3KQP47DNWTgC5bJk+y0drDMXiZ0XPQgFfdQDGeFIpsACRdjjqoBDHfmYh+Y48G9PK1psvB0pMbtPEF4XpPRLcr6X3MsIipV+0idXAO2MPB2pafyqTFsL1pUHPaNB+CQZC68zgg0MRNaKZ\\\\\\\/oOh2\\\\\\\/6PHhkzNbPw9u732GZ9se6oZZK9i\\\\\\\/NuMknK2ZMEiz7PWF0J5d6jiy2kg6bOQY1yvYC9QySTnBuC8NVphrk5U8BhQ1azHGXOHvYjkvMziWRYQBp9K75apUY6aVKWlJ13QZSS8B4bTs+NTsch3rr9FtBzj1pJ8A1+M6Twc9Cl7XWkwblJ2+49DG16gUJw2KQATQ1i3TycZd2tgTSDEQwdQnqo+Ztp0SFkwRoGpnXpCbTvO0fl8zSYqCvlJUiujrbaqkhx2lwgyDaWRLaZQUFcVB0qfHImCbgZlJg7MYIbZ51SLo8Svr4SG49c4rFoe+IjQyKlhIytJWyyoXIwRxJcf7UyUeUw3O6V58AByN+fenZDcF9sJgiYoWjlp8lMRN4viGuEVrQQqFv6jDybyrFhv8xWL\\\\\\\/n8RBatvvBXrcfsAVq+zZRXwhRF9P7QHVTYUJTLkxsLSeUaP80O4HXQyfHRyUBlV9b7Xc78xbAgSmWTuXfKD6tKjMwtFmdODYQoEfcVtqLyBgPeFnytyqxTlmVvbHe+NNP00\\\\\\\/t7OZZ+49Y31cb245pxuz\\\\\\\/VuzMyOWu3QUdDiKNR+viufFMbsLC2cIP+nONDshey2mENh+SzCOWzHoO9KScHbwibaa8QxM8r3Vpt8fGb9RzpgYUqVMhbRaTSQkXaYC9Mcg3wJ0I3ngCglonn+kT3qc1ejjKptWtzGhdK\\\\\\\/\\\\\\\/3ax1od7Jb5s6H1AMAXnxxB9R1qSxiQ6vP0gq9PnetKlFMEdQtagcIsqUsc2vWHzySXJcLSEqNhBh\\\\\\\/6bWsS2kdZuEl33dV7\\\\\\\/7dVvJXjrBE9sn0gYqo36xzsRWErvsz3EePlD1skPo22ne7S9PfBIoSwCc67H1VfVu2vIshS\\\\\\\/PMGbBLxzTIRtIM\\\\\\\/VkKSS5Q\\\\\\\/GPMYBBJksfJqrzg1sNQlAuRAlwShVuPF9mbJMSMrG8p9TCPl9YCK5AwfKLZC0o6AStS4KdwMSjSDXUTAshUHjL6PYRjXMSAxHkDWbGjrI4hcS6x0MKfhUCVLSAFeaucZ+D+qY+ZZhQ614gW2wazVShvNMJqCVD0z9lA1DBpHhcdLx3Jl7WwEWZVFCwuCnAFnM4eGz0W0ExSLUaFrNaUWdFlpSSJwcSlDmoKbW67Vg9a63oNvQEOo6f6\\\\\\\/lZcVpE0YnpxaenmPsEpyyNVL7SbvTXpS+V7fLv28nx1VMDqOD2sIMJz8uQkL\\\\\\\/W4+EjOvoBbeB51logverIDymTdxYc\\\\\\\/xX+W4DWxm5W19wCUH7L+9S455nR+hlzWiDfvF9lAgZaS2cIGClMttVkAm\\\\\\\/qrnUn0bY5dvLYAZur2fNGuqo9x+GcFkjFRI81cVS3iGFpMHbOLbdKK\\\\\\\/Jbo7H9eO5X6ymiEFpr85TQzfKXTvZAIEC3UoQCWyob6oI2QK6qY+6v+csQTkFbNhANfHfoyIwSkkReS5az131sAkMi1gkdNUoJxTPMgWNLrnT5J65e26+19N7ZcKVBG6Cqxp7vGr9TbiiLly7UAOxnoEthQ9SziB5kPW7E+ulf7g2NucMCVrABO1JInYFUmUw\\\\\\\/H+aZxWFV0Hl2Zclhu48\\\\\\\/6ImIm07NRsdaTt0YFfydIIHQaWHdfETkNlgdRrLIcghlS7SvB9q903BE4i3T\\\\\\\/h2uBh4qQ5uyZDe9k6dEH4HNZ6MBNbRzavm2I8LYAdriHhzNhxtgcp4OPnFOdStnC2n7OH\\\\\\\/Av9uaStKIMYP\\\\\\\/QLzyj99Od\\\\\\\/E7YQrpW+PHHcbEfbXVEtIwTJc3BRGutVOCPm9wOdzNgV23VFd84fJ\\\\\\\/En8I8dX5cRsVgp4eILX+9JdF\\\\\\\/2lMl1qDz\\\\\\\/B70gmmYEoxDhiTaGOF8rOPv4bl7GReMc17A3yzF80nwhYwDaJZ7a8yNH1+\\\\\\\/m567TMo1eJt8x6SZGvFHgkg0K7xZadP+gM7lDBL+O3GlM\\\\\\\/HhW+nmoERi2GeTfTyK+Pdsj6qBSfzGoD+T3mQ+cOsJOy3PeMAsrfDGhA585XCR7DWq06hyu+EcOVkh6gnoCMOfq6gR8SdaRjNyalu5eGnJztdkjU8VtAUNapNDsSF9jBZQlkxg2L7s4J1rJn0kMPl2M2GWNjXcvvWwPcu0sqnc9aUjcDLlEkz5h2PbYAcHi5AmKRIC8cfie1K72ZtHlNIwYJH4foTnoPv\\\\\\\/LOrZslPJvrOReP4qzgv+jd5\\\\\\\/sVxYO5igJkyY5yGxjxBGAJL4umKFpQU9OFwKPsdjA\\\\\\\/6LwRFHDWX5VTj7OxPe21jPto7Ve8tgpSuGYntXFwFsjD9VPMvDrb4y+yLk1sZSHDfOqicM7JE8OR5o7iIXnJjnXB3pgh1EGn5dvCRc7Pc3S5G7BCFJIFcSK7VD+zP49QoWKq8d85EcTllYVJcb+safz1FW5QsJk5Zqfn9tvlO2TXWEHg1EeLV0KrSxx7L11ZDKBjRP7SQiJ3evU61NYilN4nBbG6Ktt9uwslTBH5vFS9LM236whoqNB70rzCKGtA5c4PfH5wG\\\\\\\/abafEDnayYXYJMwO4r4BKnns+wJsCZrXqL8ECav7Fj4Fa6CdJiXdnVx4tG7RI10WDP4OrhvAjlLHWf6m5O+XHoX1XpuV0owmgFFzhC7GsvCnIAuk9\\\\\\\/EzpxV1KSUJfxgMIPE9vv9RlczITxp8EemeYvWASkxsqS5SjzqHNp6aSLe5vB5Tt4u4wh2xTKmrS5a1e+e1p0zjV0FYV+thSbjC549XF7dlLyleq6\\\\\\\/MNuSVkfCTZhZMxT7UxdJLI4gHaUH6hugMhVkVGtAUKeUHm7zYGKzV7WcKTaKHdjbmaveJsrA3Hbx\\\\\\\/Iz+qYuGZsiTa0UVtKeSr9RfemFN8AHQVWf1cBJ+Xl81YXqdLBwbx1Sxo0fpY1PmRHidwYQg6JBPDtcF8yu2vE4PvtgHqNzDmo+i\\\\\\\/U2x0qv734MChk8iWfnEUePUgtw+nxQPVowoxGe2RqECg732az+mFZkRyuSYzdE243qyVnuCx2iHdi1fu4K6wOL544GaUg432j0PDg+42TwgbUp\\\\\\\/5Na3e7inZjUeSsTrBTFOuWItFd0Hr\\\\\\\/7ZGngTOpX2+dtfW4qt0z1L0NFldGfgor\\\\\\\/Ya7egGj+HjpxMJR4hzrf\\\\\\\/fNKeeE4hFgri6f8smEVdFaa4N0IcKbKInTGJbXWwuhlSA6jgmGjNzpAZtFfig1ZXzIxunYL1GkTof15kDHjPeoejxGjqfb+VvQf2wOt8y0\\\\\\\/bh0sZlrXdAaSFhgtXW+Vw6WVI2JvqRoQKj8Lr5uULu2cCyoWPO\\\\\\\/iZLUYBtujCdLhM\\\\\\\/JpDcp8feqKVq9Pj7ly0+RDEgPJGkSp4aa5rQ4kGQ6WELhUejOeOB74KXAbdT2yRObXAGnzVZ9mP5bOYaG9OsVnZWtqr+u0maVJzxXTin0KFfmQCqpRE\\\\\\\/nL5CeisuDsWTS0W35eAZGR2Cj3035wv7SSIx8XdlPJgmMiG2h9riT1Zr6\\\\\\\/CCFbZletXMHxz5kGAU\\\\\\\/\\\\\\\/zaRBUDymeYVAv0bVnHWAepIDXdxw97XuMndtFlh7mEudy98oWFjCNOg8kaUMFuwWdg6Vnuebgh1mBVYbZ+WENVIhv56D1yg1Sr5EPovfWV6VJZwaEBI81REIku7HrookRV2L23dWyz3SZeWQFSeqc8hrDvJQQyMoMHtTBBGV0oJwFfbBmXoh0\\\\\\\/AqI8XgX\\\\\\\/wCQqhL4NjRvm8wrTzpWtSQQXH0xmUyD7WGkDcr+3yShhFpGJmiJXr8g9wfpRv7V3qxRYI\\\\\\\/5b\\\\\\\/LLSs8zlyI78JJI2SRYxebIBMkCjoChySWgg\\\\\\\/SEbvLkRVlwLOim6xCRIzlGmeiVV5Hflf7N9tRZgoK8EW2jRQIkXau8xL6dVDlyhQR6tgPzRyCh\\\\\\\/M1Wef7LeDz8z0vX0WWvmjVzq8gCkySDuyFBypaQUIWDFoTDT2jQCQR6HHu9DR+gvgjckkKsbnPf3xVq2Qf0b+caUkL00J\\\\\\\/VEB4uuB0Lut4Io4iXwYzLnrPf2d+QRboo3wo2p1XxeodlvY\\\\\\\/Posu4LEwq+qR5BeMwHD+rQu9Q8MCoaAxgWeJWkk9F7vl0oxVrsW+Wa3Q8RYQBVVbPBIP5w1JygubhNv+uWmz6bBQnTFQ6NSbTUyNS19\\\\\\\/8PPpEXvPzr3u8SRdg\\\\\\\/yxiifZQbn2fPagtt1RuSCYo99K711hp5vmJJfCVR2K3On\\\\\\\/HAFWG2QCtB\\\\\\\/fLjqjhiPnGjkzKJwGgmWrRly12uRveVzMipYTigwdH86hiMFxsFwvC5KcAtpmoAfohkroHOGRrKjdS8U0RP\\\\\\\/dH+5QyhTkTQS9pIXmK2EAgDFSZtaxJkd1yqo3uPzJmFpyOwFEGmoXhnKcdi3ONNe8MLeQ3MXwhmeQ+vaEiLXn2DnOBdrANO0oqupJf7E6YVga3Okjl\\\\\\\/CK2QzPOBAgD18+yOF4+JSD8a8B\\\\\\\/5kyN9bSUx4pdJhQOi66ikJKdz9bg23iLPiReDg0PcSE6QKDFgWiwcGmzYjR4JigXxk39gzLGCzJeTC52czik8Cxa+yi9ILFHA8UHhDVK6uV0Tb8yFodNPlLlpHs981giF2TtAMBJcRmSfOYHfwx6r3IcoSq1yxQHHLkZ1qh4t48kk\\\\\\\/JEkP2\\\\\\\/yA4i48fRJys0b0hsHtgLVg\\\\\\\/HE0qyVlLsGdf2n54MUG57N9DCL1wWr6KPa256oji5YJDYkXpCCcYWdDZq++Vg2EuE48HokGnVjw74FARtzAuf8EyEAbda8PDIrAnKjuE1oCVmE1p6lzHjOei5FCUQ\\\\\\\/WO+PsygNVBgR5TeQ3rXsu\\\\\\\/gX+diXp1ZfSdIVHWMm9rhyiW3TTK55Vg4uR3vALPG+vXytHW28nQJLrhaGPkdZBnkXyEG7zaQBN6FHP3R0I\\\\\\\/fPGGJs2zuJWcg9l4JBraAnFAycab+MEQkvR\\\\\\\/YEPyIqvQ4gHCnIOl5zWEO6OsEV198PSDKf8VqxtT4\\\\\\\/hD+ZrFCHiSUYXtK60Tl9OfmIV2ZMzKjv3RRsrh0mqpKw8+WsQAnhMULyPwmL1SwRu0323WAg3BzE6sw7NElnq\\\\\\\/c6+4MYuF0cQP5NVW+Ubi9bw7oKCWdkVvQ0OTP08DB+Oj7mtufqCTJrmbNlbflnoK85EobovjEJdjpgA\\\\\\\/ytXntDpMOtB+s9sPqKBKdkbzN++h\\\\\\\/WF70+jvKoaVSeDoOCSfMZfA4bQjur93W3DNefsHNW1kEEdMvkfMyxguw94vpTwfdPpbnVyQa2TY6Wy+bKGKF\\\\\\\/nCcO1cwfCZPGeDi3V7n9j4irLlYokD9MUJCx7k7R2V+s3RtZRz\\\\\\\/M4EYJUeFB9LZ9Wo4TWDcPP5NSeMf+bCmDLvDrSMuhWSFU21DW6dwUQpe7B0Jch\\\\\\\/EDPUuWlu5AulA3K4o+XzO1cRbGkGXyopVjYV2lXAcf4+Rqs3ToN6seTZXIPSC509Ru9dI9GyO49l3c\\\\\\\/6U+iO93SFJVmSiOnqjUeMN4BGEk2T9QP41X4tQmmonLj2BhmIvyL43RsvP+bmqmlXWLlpPwRWe3rMOluIKCWvNlbWCgl4ntKZUI3q8NMVvr7YnslOci0fEEFy+v+iijCDQECurr6iRlhPb4AWpVnB74G6jaURimXWtt16DjbWJ5UEa3erxg\\\\\\\/rBGgGJ4HvX7LFHwfqXGbCezUoweBjooa4vwGrKbiqypaHzGW5oy7\\\\\\\/LpJ26breXE+Vkvteefm9DMf550V3zGm1KlFYmWRh74eB4sHCN3xovdN71x21h1nr9xsQ0RwkyJE2fdLQgoyYc3\\\\\\\/mU3NiRUk+5i3qnlePQCLB8BQgnUlkFbVq\\\\\\\/RU6f3uc5BaQsq8kHgdsTooyQbYj8Ylkz\\\\\\\/g+A7c8+nm7xatb1TNFP6DDgXpC94hVe\\\\\\\/55KdZvvYukM\\\\\\\/p4jjrR\\\\\\\/cuQJWxr2QJyLsn6WGs+bauft97ijwwAy9qoFudLP0sE7ZM1SkeYrDjCXSMtXWWDq9kVNfJmliMa3txqWjdLJHZ14ZdF\\\\\\\/6CXGNLHCkVIGKhyKnB6VhdIcUT6+SzhhQpTuyQpeTLs0L3+Oa4GGjv8F23E91jQ\\\\\\\/wuCpIWIGGjbL6chSqeYr6j5CQPyAaMNOQugGiOq20sfkCacHQOtJJa5clSWDX1USk9IABOOdHlKSTCtzxINW4jFwotbGVxCPJ5EuEPCBDBbeU55kdCVTMUpn\\\\\\\/Qg5vU8vLGW7XYTaMg6hHnO4V4JYlkSgBKCCXZxZIVTYiVxMCErGfHJkXGZNNjUqptIK2JgWDnngtxvTsrDdy\\\\\\\/4pfT25eKzwYTsBsJLRoLpDE26adK0RzLWg+sTI2mxv3EXbc3TLm+VYSTjRZnnjnMb3PpdiPvX\\\\\\\/GklJtuTVozPPv+TFJfp+Ocm7aKU2FCMaYuUaMhdpRs0U2DjRZW7rCcrRL5mVsAtJWvFxGBZy5++TnOgq4hzA8Luc1iOVkmh3AsrlYKiLNo6MbCZc+ShtM3P06ZRCK5O5sJI7XSF4TV1D\\\\\\\/YjwUyl397JgNJhRFiRl79SUJvorqH8lPjEmEeZM15lt9\\\\\\\/Ev1cneIl3C6+b3r\\\\\\\/uc+YjMBRD7m0CmF2kV4bToSzyTCXQg\\\\\\\/31aOil9pcFaBS20qck68LuPr5iYhjZ\\\\\\\/xSWIwvXFmV+WTUGRg6FUwRllK7jqTzu7mLz0rmKm9ZMvn1YxvWXe5hlietQqqgnM+c\\\\\\\/CMLhy4NJcXgfiy7kGIhifivlA8mjrNoDLyJFfITaCVozb3MgBylJ27MPf9AGwYE3TsBYfjxmQnHL67ZNP5ASo+7cQgIwBVPkXM6uiOkq7Xk6cYDjCA6wSd9Y1kkK3Or2V3+xkndDw9rS4cZr3kqPZox9wJgCCVuOmYuXAUas8ooqqevMIw0gWCNwUL0uMJXSdA3SOqVxzJIMh0PbXCFUq2GvPRN4+4XqWC86WUWj0D\\\\\\\/plh\\\\\\\/nKpMw3dgoyvvOMIK0RDf+J9V0HlmQIqtWK9B4DkOCNupwraxKXTivOq5Yw\\\\\\\/eBnup+ywxAmgywAN6NAFfuldmrb++VA\\\\\\\/GcfC4A5znadgvjGyU+vNCMrg6nfw495I4XlSJEKaB62zNEBo3L+mPNPVY3MccMiU2Vq+7ukRLkzBbeSuVsMGwTiPTj3yjJRuxqbM+xWe+RP+jpdYgMXSIOQDYcQ7jb4HX0xMsdxaeB8OK81R83oKbPijFxM6jLIP1Gq2U8FijR3UHyl80plhtPLs7fyX+So\\\\\\\/7WWHS3flfD3Qkcv0yhoa28303k205vU9PyF6LVbqMQN43RHFChourag7YpKpeIVplkiOG6NyUPdcOGb4mkUWH4CqxICvgIcTcvkb2+ClC38wy7YcbxBfCjbTKyiIlIo1h7TuEXWSYOZdqFhtZDe5xT52n6xVvDVQBCqTYjsckOnSrd0oDPN+mTanodF4hooBG7iYQUiGuECNeQ3BWrWhiue5NSdNcNflHNYtblCoGnox6RTJnMVN32kgMCUEzJWEn7cClJUhOJmcvR7SR\\\\\\\/cAlS7f2blEoblvQakqTOmyfKD2CqUXf7+5UK0qfNLH+iTmKFZjjJ5UF\\\\\\\/TZqG7rZsm1dTxDwC262hf8ndZwau9+G\\\\\\\/dJFrwXD\\\\\\\/MV8LLuEim9lpIS3Po6m5B66p07nAzIM6cm5aO+8XNH8reY6t\\\\\\\/MqLstdZD\\\\\\\/reaDVyjTYr2VnRUv8UUFv+EZet9k4My3EuVRbMMzic15HRBgHUhY+d0fPrAqvFQ50p5dtUJ+TOZOtMfRTgCMKx7ZsI5VqQbPmDSumcQLeGHp3uObTtzBJ1EmXWXMKsf8YEIecnYuemIs4hX04XLCqIUIoHDU5rylIOvjGZEq6DVxFH0lKiis3+bLNGvEaeX7ERsPm\\\\\\\/bKyWUPg4\\\\\\\/fKXFdLahOGWzU6lV3VoMwxujjRW2qMdDZapLDnYIoKpQC7xaZefa0S8hnLFgSX0iORflj3LhrJAohe6wFcC9rWhY2eTDXj3WOnSK6SndNd2hHVb0IQaBM29eHXoaOdlN9QWkcy6EPWKoxb5GHvxcUgWVpElpFeCIampoBSs5Hw3\\\\\\\/z9afEFY8TBPRernAzYo1CGcjWHnuUfl+rQvs0mW6uPOf5K8KM26ec7KU4aVrtegW+Ra4Ee3pX+ebTy6AWAs4+3Cc2LQ5vrfM4fGR9oBUPTVc65apFfiGuahthRLbWch\\\\\\\/57qLYLxsFDImzctpzXzLx2jAiCNQcwjVEtKeSjpZ+saWvX9f83JHpbRAI5+ESV3dXaruWeVL8nnbDF6IWbMVwZd2wqN4Kll36f4THBExBeUcpWPAlLohRwTeSveM5TzrOBwuZJxbswb7qAWTjlHu34Ba1JPO9EK7ZjA9pO0x9JUgf5dV4S4zoQ+yG7Y6LZfad1U\\\\\\\/wAVuV1AmFyIV9NRgWcYmN+SNl3sEXlHpMPowLlMtSS4TGOIVmyGtguyZY+8+e6yJEyUMAp9tPznuyh0nqc8vWRTN7jf0yCjoM+oVepe8RvPUfDmkN3LY1Dq01K81i7QHowqF6MgVh+EBye2YZbEq4gnNRaEO\\\\\\\/gHzOvLZ8kY6UHHWKwMHRecuwsVllAoEwFGGIiFTFVbNgF62R9RpMtY1XApIvwVF1WlDRVGyD0tDbdJSlDbBl6EAGR0wsz+3cMaT\\\\\\\/Ts44cMc806suBKRCPzlttTsUS\\\\\\\/p4LY22YN0sKSAm6HHOMiHNk\\\\\\\/33iWPS6z7VXhrm\\\\\\\/RiDhtOX4aaeg0sV\\\\\\\/yIsdRxaimARpP4H6lPZnrLe+GzOLRNVs5C9BTU9e967fP69N7X9ruF9epKlWhFuDDfar4RlMI7mQsGoCML4bxwM2f03TiKck7eC78DPdH\\\\\\\/AMw57COgB8qddViogPBsRwPVTMrtP7ZIYps7fDyxEC14Hr3LVOroMZGUCC4ihFQlZY6evKx9G4fCbxatW2jMSN852Tkyyso3qISgcsnFgCRIfqwZjRk3FjkjuoQp0ca1QUBPsaD\\\\\\\/03H2ZaaVeR\\\\\\\/b6QF7Eecuj9ep3FFaoVp1Y03BACUDrPjPtlkvb7HF3c\\\\\\\/vbL37A2QSqs3jBA6Mz2xSic3He3BU\\\\\\\/EPk4qXZE0YdG3Q1ADB1AGeE93XmoQM9l\\\\\\\/0V3tOdhf7L7f1IEWeYVC2yYQQKRIHPRVkEfmcJH0lYC0Q0+Hrj+c64YEoBMrvPjf9Qg5Y1GaE9PIzezUa0x8WtSuDrqmEd0r6W62UlyUh5jhIeShtGP7lwV+d\\\\\\\/3uXJ9ahfnAdXRFHfn2nymKgLQwFSgIhZi+yLmiydLuIROHNJvK+nufO+EijuESSq2nq0F3CmDyCQ2XoiIpaezxtNoVjmcAXxhpocLmxUCfYv7U0rQF7QQuM6rIJHv5rY08\\\\\\\/8eUcemcyw3pRp61ciFWeBfyqScsKcpGrcmNiGclftiMcu+\\\\\\\/C9\\\\\\\/LfPlFW04zbZaumSss+p2Yx17NwpZMOJxPTjOWWOacl1WDklYa+E\\\\\\\/B4nQAd+EOKVqA7CEIaKR3FKDE5wgJD\\\\\\\/oyvM2NIG0ngT7QFCGR1XBJRcqsMW+yw\\\\\\\/ty1i6IABoo9A9YxIxusepY402zScVDEq9jE0cGEh7MUP1X6xA\\\\\\\/X4r+qTk0SH5VGMLPhuCwTiWVMQs+h34MMkYX4azoTUJTsgL5IYUxTymvl8wcUTYt14zsSY3mdQlCFhWB0Gy05vlNRHEbD4qKyzMI4lt9SaA87PxO3nzsvEFQlY99X84fpvTiK8tdQtfSz7\\\\\\\/dZTkjFbKOHuRC\\\\\\\/LTSYkqZXcGSvUu6mXtv2NkYBifkH\\\\\\\/G+DTxC7hVCcB4catdBE5nMIFjbTWdoJ2CkLQlKgpftCx5tgaNyyt9jOC3M\\\\\\\/IxYXc7rygoZrd3Gab62xhaZvlrzrVsHsWl1yRrPgJwIYY52DvbZ+xNbHZMDD3sRlO0vqZYNikF6RxbQdAwx3prZ0ffL3Rxbw\\\\\\\/AKAQmg06POX\\\\\\\/0gHhjWDhi9LEsxYyDDhlUXjd4QWfIVDaX22aitgO1u9iz1xBHK+NcK9lVbB3k3fPiIfRokw72Ac38KjqUb4mooNqj5beHYpj0S8zhkexdFAU2DzeoThgtomqVAr+ywjQKTaScSNTABlcDT8VhQcu0Mt9laVnO8z6HvGzwH+nBqNYBrat99db0S7abGobJI652sM1\\\\\\\/8hfBi1ljZOP8DUClMAtRzVUP6kp0+MURToCoL92\\\\\\\/dBV\\\\\\\/FMzXBEgN36YCmyEtGc3h64\\\\\\\/t+fnGLLIYifUL\\\\\\\/KCmcCRUhkX6RWVevaHg2fXQIYn0z5Fcjq9Ftrc1PpIprNm2MTIa64vLCxXJ\\\\\\\/Hs1zeTrMnEAcE7htRHL5WWCOJ733c9SzVKQ0VJFfOuiRza\\\\\\\/Q6unY4sr\\\\\\\/lJyciifgwhyVqgE7aCVHMhv5YmxkBGQtibAOFgfO749EIiJd0QRGI0nvmxN4T4ciEtZQjzG0fYasMPVda1icuSd\\\\\\\/hxk7FCGixWmp3NMmDahLBximW5KhX+RmTrJPRFAknvhTyEkI+J+G6xEBK8U7w7Yhx6JA+Q+NWFy+2oj\\\\\\\/eXRbYvU2jZNUHMr5SiVCSZbBvfHfP6ZWoMgb\\\\\\\/nRlWj9vxM\\\\\\\/aNQ4b8VVkfwqZaYBS\\\\\\\/JagPbAipuUnVLDju9VPn+PibWUfKiPXJwAT8dEndqUnDW2xOR9YnA3U26KteRXOUcM4am8CbDmNv8s5w+FaFm8fxcikQS1sWLKaCH2phMQ\\\\\\\/PvnPI8vbxXTZ7NHKW6Uajw7QFRfdQXky\\\\\\\/bj4EBDCEdsX1rKztDQRl098TeSgkKNQXPbwU3\\\\\\\/oXZD9RXW9iBw6YiW9V7CirCjr0B7J6FsQcEEq96ZVjrWkTPSh71l5c2XfmThjKTZFHssO7lxbWbZu1A7j08yy6yqo5PPfV2nUHmMNS\\\\\\\/CwQ4xc5OmXKQiCdO5J4+nbPmYOSwRAOZ3lTr7EpAENHtsmbYu293bMdra8y66byCRzLEcTIklHlUQBCGr1ezIwLKphFiosw57L3R2de9wWvVGFcKkds3xjjTTOUSsa7C0Ms2QlN4y8d\\\\\\\/\\\\\\\/gFJwd9ka7iICn8jUpAEHcSfaAUAk0C3k9POodZdFMiI67HpOBtGVmmmyyxFn59QQvcNgS3lXCqUXUZMsRlChqP3W0NRIVEWadf45c0+i06velcBcE\\\\\\\/+BEq3HhHwVY3GHKt\\\\\\\/oaZGhhU6slJbJmA2X1mouY7bxk+eDWPHU5yYmxgEdNuFl0EJ6Ud+5CdUIZSGI1Cp2IYAdfnZT+bZZHfRCs17ldE5zCZ4I47Cq\\\\\\\/VFwAHzuXVpTDHxs\\\\\\\/aWjY5D8t7wjsdDGapHcB+Xva+K5BxmBAzVj8KHRZJEcONuEKLkkb5l3LgQ2HlE+CWc9038imQZQ6+tsrNQw4q0o9hWxP2rhWNO+EMQMq24XoiM9ax6QGNzq9KGSia2JOiLkdTAigal5XolJqK2BqLe1cBGFLu1DEOqm66addep8CAEpIE3mqlML7JTmobSoZ2\\\\\\\/d65xGlhshr98jvL4YQGYR8hVXYAOQzK6NFgLB2TbZ2JCcF6+2CYf6JL1+YNUMPsOrCUQWht2njaNf7WT7wvm0f0F96ZigwV\\\\\\\/MvZV2DFUfFK5VPzNn1xz\\\\\\\/nJ0pRhPRYWp5UpUioUnITPowcQs6s837HC\\\\\\\/lv5IrKGOuzXciJ3R6XquWqorD7yzYpWXKwaI0crXzkkdS3B+hP88AwKIh0dnnnPeJcUmgNO6FrsAUXckhSApgiGTtyyzhbKI2g\\\\\\\/MJx0XwnOxP\\\\\\\/hJM9yZI021HYfkOmDLwip9WAeX3+0ubnqIooVTepv792T3Hf4vIGNKQtUbIBCZ2goWRiGfcFS7UnkQ+M0MKow1Mpod1loMQs0Rk6fUXqV\\\\\\\/7r7I4rCSn1vI3VDDQnfpE\\\\\\\/B7aAnqrf8mkyhjSsOXXaou7hEGrAGo2SI7oYHgRyfLqhhdPGp81VM\\\\\\\/Nv0NOKs6PhTKSKLeBMc1JFqjw1hIrTEzshGvc7zhdY+hq6YurCFiBXkZ9UGNkNLNuNhJwrlSuOaBkn07pzKcVYHj\\\\\\\/eumIhBCavPGAwvv2\\\\\\\/jshiWr16tCJyN+yb19wfqDSWBSw9MUulBOj08iJDZ54i\\\\\\\/6ZRgipIkBz3ToLCW9C6A+C2bzT3+5N\\\\\\\/npiV6L2xkJ62yuGuwfHlH3meUseswkoUrdMhi6g0NwJWihbh3IWV+PFac6\\\\\\\/4q\\\\\\\/9agZ1xcQlZZEuzAuCMQI35eE6wPS3Cb9sVCMpRG8NETDvRAHomFgRKUz3yxf4RjQYibHjt2PqRx1Ugjxa5o3vtNugNtyJ9\\\\\\\/ssVzM4QLmLrQ4Xeh3FNhZs45w\\\\\\\/FZHkkytRxCrBvxCqY6zFrMuyKZmeWWdiQo5Fd7wcB8LJFE0aEisRxoHlJrP0mvzBrRuKjEYIy7o\\\\\\\/eSVYOs95\\\\\\\/2Z2FgTRrzfHJzkgo1pqG51SWxFZsCom8Zml4aSr47u6U0RBVYuxH6Oi5N+6UjxcuwOvsmoQgYO6Hh28gyOsMkDlpaq6iLV7i6AXku\\\\\\\/he\\\\\\\/9OML1Cv\\\\\\\/ZCsmOljfOK\\\\\\\/xkVZDVbwWXJs0ccwBCIFeSJmWmjnkVE181WUNqIqHi93jqlf+Jlkkudtt6Zz7pmsRWCTwZsZECH41LpX3pjxe8mvua5HLHwesf+w1CNPt71bdkcH3yhEFBxhd7x7mALZyQv9ImxbY\\\\\\\/tWDDQAy83vpC+4tt27mH4sDsYHvnFHvKGYP50StXYWSTpZYOBWwcUhhOFi1JuIpShbB7EaS2Mss+2jvRyp6plH+Zss9kmML\\\\\\\/begkd0XzIGgPR4W4ZX9iQyHprWHDx4bFPQmI5U4LXDtxIFUZSddX\\\\\\\/fH1dhT\\\\\\\/p1aiAnZjBK\\\\\\\/\\\\\\\/aaTdd41n99ftUBDPlTZvSPtMGvos\\\\\\\/qFILSIsJJkmZjoyyP1zKgFNzcRZfLNIgGTGmlsPVQuygQwlTiTfM\\\\\\\/QKo\\\\\\\/KG\\\\\\\/BukraAdPtWTq6qURZY76WhffWEdtK5FYbQvf\\\\\\\/SqB\\\\\\\/RpLxI9QWJlnbfS8\\\\\\\/RGcOv20wYNqPFiKtW9GGqdudT+IMjmtfucX63fcyaJgrIzDtWK4VcuPqbWFwprsJzmKM8hTinQbmUlgz3AokLKAEGqy+hffAVVi1KfiEEHdtDZCjeMFw4Y5TbuQZ5uJRuRfvo3aiRBtylkczbk6b3S9fltUED5BUve29WxO141XXvJMcBQD1c2digh0LdHETpAUsBJwMqII4JvFcPsJfrDo7eeDPzNJyEXaMgoQbPwKbuXAyfYuOg0t7r8mTWC602uuAYRqxkcNn5jqNr9P4e8zczmRkHbn3CCG+yJ0M8OzEI4px5948X4Ct1eTlMumJvMF4Ah+S72xPWup\\\\\\\/G5WYlTVlkBJ37w8vFTHxKzkL7xSILG2a8ipuGa09otft9Ih+3HsD2StuuSxweNMW7apvz+Z6BGPyaluDvsifhD1\\\\\\\/v6FrKRd99pD4r7fYvAxlOQhaUFIM4XXBhCGUf0OuQ+YZZT4WNjpD7ygcBSLSSZUjF5wARHY6HB+aGtoXD7j21QB0B1wdLiAvSlDtbli\\\\\\\/8iE6M1RbFAWvsHAmuyXu5Uku5pldU4ZjWQQUWRFCOtS8kPfO\\\\\\\/GEfmGmaYlQUa9Wb+1ARZaEMzsMOq8L8tPKMhZuVUAEd0qZVDRiziWbu0LQFk4pF588dU+U19mosQc7f5jFh\\\\\\\/Bv4nR3ZTp\\\\\\\/I66aw7+hwCrBoGjkHWdAY7eblPcmPZzFutDE1iDs0exh18QJFsSlFp7W6YDMlY6+6WA9gzccto0cjAfhU9rnthG539YbSjCXl+a1KdcYE36TxMicJvYF6Pw7+dTTTgIVMIrMPZR1CkclbTIzyoWzZ6zVu60YPOXC5D74uz98gaxS\\\\\\\/QO7j00no9wBlGUbvozVE4p1oESN8N7td3LmrrtLfYiD5KMAcad1xrvgp2xsoKsptmUXTKCgmo\\\\\\\/kNYMYiADkty0LQVzgpaoWQM\\\\\\\/Fkf\\\\\\\/HdO1NLeWsDQ\\\\\\\/epVrnv35RDcSLgp6knXfEhqdrXHYte+8B\\\\\\\/SF7UZblTw2F9vJAQGutsdmgw57jJ8JqboYmsdzFSjxgCYPbjM85RNIIw12xvQn5eCr4sjtKvaftNYW6zP4Jl5PuBCJ+4egqNsCifyGzsBLVI9QHrewrRyEhO6orc1Klqfauw\\\\\\\/COZeDiBjbWaJc4gDKRoGYyKXFi7LrwPCpqYnAUD8NOesVGDvAWoswmTGtMtv\\\\\\\/2JXa2+vMTIZKTqmVkFZmcqnnCfVzxQJ+FSOl0M71O3yENvoc+9s8lYwnQChPzSFFVE8cjlDXibEl2YNCEwXfXRGswo27YV7kSSs6neBO8a5FK4SR8BsPeMuywXcYA4LsGFqxQgsGTxyngSFHSlH9FOFJRmF1Tsoir+8UXui9PWqwHvzRhm51BtkgbEpdrSmLY0CgSwbsAzETkj6zxhqqZybr+xkmMPBOUTmP9jFOsDl7FQD4ISey9ia8IpEEEsjQh\\\\\\\/QMdUW4H8KjXcD\\\\\\\/1Qvca\\\\\\\/3L6mz6MIXH1vSUMuJg2r8B2\\\\\\\/93e5+IUmZypJ3iwbIcZZ0xpjI9sqQnk0x8D028tuSOYBkMBl0QVXcKSQgN7luUHGmEe0FE1uiOxcJiPcV1JphPLaySCBG38zGmwfNy6lAMFVSj3dL+pGf58c\\\\\\\/qFCT7IAKoj9VaK4de2MXEbVjJg5S\\\\\\\/tIdY+nkvb051uQQHb7Rub25PAJx4zdmqz9Jvx5RSNIcb6RxsMhoNkK+e1O+2SkkKvEoY4dMiKtkbKU8L9EzSjO2i8RJj8AhXCQ1XwJCp09v4us8ksJ+MwNngTOraccooAQ4dt+3o2Xfrf8bwcOqblwYblV2Z7PxM57GOje8ssb8Cl14OQqlRUp03jsE03MBVEzNHmfutOdr7fVMtfjtNVAA6uN2n7fWdoSrCRMdr2pJal9m+WIrE3uUpHGQFMaXSkoJO3FV5Rb07X+eDLAlFYjadTtuYw0XE+aR231I0sFPj0ZlLB80HkG5WLOXawEYzT4OU\\\\\\\/M4tGfFjGV8JwA+3X1aLPYxf0JGqmy3pIzB39G8+CylBT5gGik5LtWrgDbAkEGvcilYbavzrdq1qbHC0xmhnkDksSUgWfPvBR7S6\\\\\\\/lx0CD81cbaQQgKgbrfzwbfE4UY5CV46QAVv0fsEDsHEU8ZXD5LRUYoQQnlEaS8ahDzIuuplag\\\\\\\/qc6Y+3T7WQGeGB5GlkDrNJ3hFO0JGjpKzbRiOkGV9hleTX1QM0U1X\\\\\\\/WSdBxiplZBLCOqtEo5MoOJgw2aHWC5l4C9tmeQUhtxwHwt5oJzI0mGEFsDgbKUtLQJLhXWh1P0uIZWo+Vfo4ThYUJb27kWNaowxU+MUL+KQyQhf69QjLukn2ODWWFSqYNbIRstTOCrLmWFXQMxgsFKQHH0nRDd1F5X8RPd\\\\\\\/1LRP5bD5cm5dZgKHzZwjlHOGjblCPWaHRCmZ1QdV\\\\\\\/Nl1xLZpn3P5v+eSFTQNq70cGb17FPeF3XDX9p2nVGsdPfACAtUwPwlNc8KrZP81pTwY\\\\\\\/dRDd\\\\\\\/IEb0aermQzVBxs716S\\\\\\\/Tfjem630pesE8dRSuime2La8zf45xjX89QQf6pQndgSdWe2QGVeFNJvWFt11+5+2HtPXGvzx1+oxRFLqFHjgdg5cTEomHB9IdQjXL2rw05UgMkXRF3ipBRTf+Nu5lqWeTGuvdQ+X\\\\\\\/Wdd4BRAcs41VcFlN\\\\\\\/pioFcK2y14K\\\\\\\/gIf85qEzOXqAmIOcJtEvQG+UTFc2FztzYsHnJFHP1Uvq35YiR1fHmmU4\\\\\\\/BS2c5zndKcfSZr0rVysPk+hezr3PKylI6+aMcj\\\\\\\/+1z6wNtAkeQAXR31S3p4NHEoo7f+V+\\\\\\\/Z5w9OCNUHGEXwpMlazX0uUIOj4V4qfJbakjCInx+F5UteYabmUnV\\\\\\\/USmqZihXwFS7RWZAb1+yN6ZzGDjavtgOwsYCqlOMBNGWQPFuGIspKCp7KtYZqCvO1oyY64MgFH+d9hcQWWODKiHTWK0RNu8Aqyoa6TBRZq2p\\\\\\\/hKR40gjX85Kz598MMD6CnRKsErfEXMk2utgUBwchEE9c1HjAG2F4Zj\\\\\\\/6IbjzTsQqhp36uoXMd87EAax5UfdScZlBDOQynGYHIWkBQ2k7jQBuTagjOo5V2dVguhc0iR1Gx41n6mj3ERPTK5vnFFD+9jePGEakvUg9jEEEnHPAL8AkwxhK+VBOL+hXTkNtc\\\\\\\/oC2LqLT\\\\\\\/U5aH+UOSJzfgAkC7CevaWo4EAfy43ywidtdqvF650xqYktHTMnlcmsS7QamVsgLzfSC9sVEYRLKqFbjReLFmhVmomMTw32HmnVZ0hyIiGezzvOTyyQnS8W3eJCiIRy2u9nMy8lU1mzCmuWmD407w0zh7XYgd2FlrqoZnOtlwG5pj\\\\\\\/6Ybzp4V9zXvQTILYnPlHq2+UPl8LeL+80YGVXkJ64y1HMWLuQCxfGFane99fpvm5sq\\\\\\\/KUpjj7oB7NdtBWlTPSQkouhlMEdX2u\\\\\\\/EWQ81xpzrpmVpKiZ82\\\\\\\/R2Mt0txyXisH0uXF3wTq5W7bG+Jy7e\\\\\\\/KSfCN0kTk\\\\\\\/gaEC+gFB+w\\\\\\\/AV1wnYVLvYX5064oOLP+1t8m+GaKJGWFt7i4GomAbdHJsSVcl8taw6cs4miM6kl8Gy+dU8IlvMLzmBVF7+LWg7zXB2vL2jh0Yk44vkjDOU3636kjPA1rRrc0F+uJcF913dElyhaCDkr3MPcNGLg5p3n716vS2kmALit0x2R8JPcqPQf5Zc1Rs2\\\\\\\/Bo1vTGPlxA\\\\\\\/vFKtlL9Ywz1S2jTeWgnDnSVI4hBXAERT1vLPeyrul6OG7tJ2sjBaRfb6duWKFvijQRiOPDitK6kmukz+WwZTGSTiZAHCDZCVGIEJpEjvEoIB3jGfLuFpyG\\\\\\\/c0BCjuowHzqo5TFlK6+riY56UeZWHw7kByNyJwTtvjK2dTXPff2LGX3XStil\\\\\\\/JxCqUye+ri1ZFr1FlohYdcCF467CeIn5A+fSpNqWCOeOgIC4OidvzIVKCR2ubm1yLG7CIVnX4drXnw\\\\\\\/1BHu4DUD3qGxtJdSXiWepnxFXisxVNPz4rw7x1XB60724P3Lz0V4UBesuhcbQhgh\\\\\\\/JesMeS1m08vWgJzUcSyrgagEJhAOgiLcZnnzi3UAjPR2yekrlRVEhoo+SGnVKuefrLGLUtNL9jVOL5Fw1uZiT+MSJ9EYlGqs7fEISBs+egLhL++Qy64bRxAboTgaQI1KozMgFwSBdpt5Ua5w86kXkW1IsnNpdOQC07nnW5LVunGMiIxxO\\\\\\\/qr6CIrTzJJdKtpcb3py5\\\\\\\/WXwv+nvzrmmzOcrZc11LeAYa47Hnp5\\\\\\\/ayejchrlPaQvsGLx0sH3uapbcdc7h5WeBflXYIIcKFoc0fp9vdmwRifbJvKMllNi7KyZAlyyMlL4dwIqgJo3PE6S2ltnt2N4f4qCqLQq8oMHEE06\\\\\\\/JuR8ofCX8A51aicpSqsE5dIRY+cp+HWM6S724VBPhAqZYvA3Pviq1RWDCjiBIAbr\\\\\\\/HZONG9MlYnl\\\\\\\/SQfh\\\\\\\/UW9Gol5J7YmeNUNXquPNdldHvMSuJZTV0dLxqTMPg7edTL\\\\\\\/3TxyEA6qitFyJjEEn\\\\\\\/9nzvnmr6VbhyINVdeHxN\\\\\\\/COqDgrl+Q93xzPMxEzV5NoekxLda+WwmnSb3ZpFSzCRJIhx8t7\\\\\\\/xT\\\\\\\/9PUZqJKRHlP8bC16n0fu8riyElIAJ0+VcBZcZy2JKEZ4SJXeki\\\\\\\/+vkvVmbAVfbYLGqO\\\\\\\/Nxm9r4mvFmhRQyvnIgpBtIwUcpAHODIRKBmLCdvpLn08Nt9FgV2GNmlWScVIEHVICJ5Hjp5inMw6TPIhVfkd9oZkzsiuBJ8ksz+R2Jffn8OkI63IzMX\\\\\\\/i4FgXkVH3OdXFj97xN9U0AstrVHF1boBwIh7k3It1mmL8QrG\\\\\\\/cyoBJ2JisFmfgZAE6sDvL0UxaKgggBJYEt7nd7t6oHzGjRUCs+x1QB+VUCi42egBeFJTb4kkPYmtc7IK4skH7u2MtG08zGlZpGWNUmntTsGDMubbNXDd2lChW5EdblrJsYyuEHytGsx7mLnvs3XV\\\\\\\/\\\\\\\/U4sUdFTiJ+rxmLnARcxEHli771VaUh3NoYQhiwyIjyrUydjN4mZZcBdr6\\\\\\\/ys8Hidem3\\\\\\\/kZBGpVCizCmA2B\\\\\\\/0XiJWg4R5SHyB5BDUxMP7YaeYPuKgHd9Ao7+3COMqoTq+hkEuMMYfzLNOViJOO7Fvu7mFvfUP3MKVqtNFz85BPfkbDtRa9jmmo3TdcUVB\\\\\\\/eakRekNJqxQuULlIbajGNysmnJhSs00WZl+yJb2Dd7ecadg8ClvZePbO+NZf+pdRXspsWlNJU1aDrQLUxXA4aUu9mBg8A48KMKs6sN9FoER+E9up20+danuBHsfMeHmUiqz9HC2t884CyzLhHvybupiOdipBeUP0asBRBDYzLdybR8mh6yYcU1GajZgfWA3LMdX5zTAEHMrsPxoPBl0eO2sjhKNy2WDYwBkg0pLW6cvyltEF6h8IUdV0oRBGfkSi9adpmrLaIxYdnF2O0Dl9S4+kuANos9r\\\\\\\/QEFJih0awY7ShcQUZvQoItWNJz5l\\\\\\\/SjBhIv6tzsTVNAGuAOnzvjZiqGNskzLE83QJUOjBMY9FaWn1X66GinAia7F1YjgjqCdLEnDdmFYG5XkXeJB8RKRAC3q0++oWZaUXAoZ81yGYZR+ZtIneiYELO9xLbqS2t5xCxzOZHbQNptOo4P99Vu9wXmufOBTl65z4DaFh52m1n4pGuTbPijBtqHPtPHehqaMDvqKoBaINS5MP1S32ArqTtylCK77og9S6X4VsNym5X7WrhSL2A4UK7dUKzUC83sn5eYQRdBAGsQYgLtCF5cGuEGcwCFPfR32WCn4txHRwRzjkjVyeia5u0bMfokW6vFUEGRsKcG6pSdh87F\\\\\\\/BLvJulpYbKoDvt7TCCOrgGKFJAz7vfzXKZVaMKTLqDecWfu7581w6N7tfqSQLY1C603bcSX1j002x+JCAgrqgQRc660pwkbEKkBZX6dg7fbFENUumODGy\\\\\\\/oV2nHBHDfbE8RPMUmXmx5vrWLIxPsNUjVd5f\\\\\\\/tEb4cQxERY4gdOKoNK\\\\\\\/AOUHlHhCLOS3DPaKMvTgycEuFwPPnLaj15yPI3fPrxmpXM7Y+xh3Ps31xsLVnssHsuTG5AXArevR8s6v7PpTMQJApoCPEj+YDna0upxt5MCOnp0F7bp5iBkzLuqx+2FWIeyd6cUHdVyX4G9asNKa17ktB9v4SOS1UmuoxlkYRvH3M+DpWZSekXlWHWEqxSiJEilcq\\\\\\\/CW3+6leYSWbADwbKKq7cSooZr0yfhAAazDamVlX2SNjS1bkNJQFoMv8PfZMTkKcwxfutsvbUXzwbDWWA1phCIOnzt\\\\\\\/W7M9ohhS\\\\\\\/5RIATLEYIrUru0+veFZ+R87gROOth9sd\\\\\\\/AsdDWCX7yvQYsVFEKmRC4OsBYYqO+p5FJCsT2TU6intdyYYtnfROIy0qML3LzKIUIunQMBryUpCz5h31LfmpyvfUqgltArjFhDEv7P4E+OM8IuEexTVPFoHiOijAgnT+m\\\\\\\/2gpSP14wrF5bDMibwpzLYGVbLM3VbkwYlcoIDwRG+oEsmcOhSI5wrscvnD\\\\\\\/QzFj++ST\\\\\\\/TsBaYSvWK8RdE590apaiCYwky2LCV47JeG5Qhy4NXltceK2hM15ZFzXzLs99nIFuVMsRhebaWlBXgKo1uVXL6kEkrrBU8g7jNzYeHbm+yWIUc4JY9bMb7\\\\\\\/6l2lD11\\\\\\\/vK\\\\\\\/pTJ3YdKIigosBlR93EifbDoa3dc+sS5RBhK\\\\\\\/S9PQncr\\\\\\\/a4UtO9nNIrY3deQB9rfdK86jYbguIYf79aUaVaTR2WWMS7z0dx6uoc7Mgd8tLMnTarwJ7gYUTuHO+JdxQWfRkWDZP7tmYOlOGE6c+7Z+pyEgoISTN9ZbVVpjeSQPAk1JHoWQrCyOdHLMOdToj9MmFSz+ACwiOdo+v6m8NmSxOF6scxH3f1JJ2Ufe2BZw8SyoZ4Rv9S7r5sctZHr7WSHaw4OLGPkEVyxX6UWcAirpw+Qyh0LAzhVfQmJY7cLBn48k9hhhXdLGTC0NIkJZ1ZZ2NkRt2NAbag\\\\\\\/O89x0scIrWKe0FLCKmsR3xzsdEt5aYZYmntWzOjb1XX1QqvE3MSG4GZMjCTdgaL8+8WIo2xyw8DevLO602C9kze2K6keD08NNEPtlH61lviOXBfzOjFSc+EAAv5x7vjb\\\\\\\/zYe\\\\\\\/RKgzLl2yLq+npxj4WkX0aNLvKcuqFBx3tmU8WOEt8raxJArJbFKXTjIAbZ11VB9QGxX17sDxA8+H5TFQNPhRxyMCT5dA5XQnVZyi5AFuk1BW3uLw7lC7Nyy5ZY8wlFIIDnI1DbOa3tD2YWyylqvhQb919iGUf6V0FnrXHTvXv4weBU+mCqA1ISOt6KQ2xREdVcMXn8\\\\\\\/wAyk5FUlWP+GJjf3hVfBN3Qi5RE7s\\\\\\\/DwvAcxjkFKJCUhSdX77Zqof8avnhi9oJorkN7fNn8a+cEon+o60uD7rwQV41SNm\\\\\\\/ibWKTnW6MccBt\\\\\\\/Hn6aOeMclW7iEbOqbOV1hncIXrQ24oNJ8BM\\\\\\\/bLipSrOmn7Lxfom\\\\\\\/ruwpn+Nvqs4IZCZP+twB8Dib439t+wuFnMWATMlt2pfPFWKeFOow88FjwfJ25yFSU+0zVBozOU8n\\\\\\\/xR+FzIlYXGq7S3mQ4F+qvYVuc1S6YqEfQv7wfUOknI8rHAAsSEsd8tJfK5s5k+RqITU\\\\\\\/gFfF5upLentTep4PhknagYs9fvEgjx+Szzdj0oWK\\\\\\\/ZEagvSTctAJ0UshwNkcLvqdb9bYxn3ySv9gkMsltz0UrKl\\\\\\\/t+2EkdlsiZhpmSxuxR6XMwfu\\\\\\\/fjD1YERYBcEKoZOdzzFE7daVIGlbSfsafoLXCWu85wOVosnkE2Xw72Svcmv6psKQpF2e8kSNP3Jz4SiMid3W1Fy6LdjjEydomLCxRBFDAhoL1y6u5z2vgq4bHvkWA88rMfTRT80W2Vur3D8wStED\\\\\\\/ak4C5tnfQzIhdGVSfnyhKECi5a7lvMqxT5Ph8zuhZCuM\\\\\\\/pxww\\\\\\\/Uzl3oXURUR2yGkldwcfSjJ+AmpoETGO+eKdhF3g+mUmaI10miUlDKd3jQ78eFVKUb7y5xWHTpyEAk8JL3fGlp0Yh7fZHthE6mhjd2F1MqLkFYv8h9jYuVD2lFve8oDZEmSqn7fvymB+QFsdaohEjSAdHd2TFOvH2ooMElQgj+2V6U6C9oJnWmSOv6vCxhE1xjhDKLxmSkxZt1tq+WGJN9dRyJCwulz4TVq7usYjStIEWlNA3bKXV6wxxHz6nzXuFgiBekbaAeRt1ChNpfTBrrBKgdZdIAB+Yo4UyPvG7bTarccxrQkV\\\\\\\/2b78lZf2Qp2h1Im\\\\\\\/JQd4It8QoJgP9WKW1aLslayp\\\\\\\/72KlksWQ1IDNeJD6aZ\\\\\\\/CnK87iq2pJYSXAPKGxTMFBW9ZRlV5wMdUDAGJe3QT7BtzYqJGOPyCHWgC8EESsJhQPq0JiaxzwMKGKKVxTOD0V3sQErHSL00jSDGVOej1L3RyzZx84uaNK0b0mUUwCbEMCGIiPdienBOpbX\\\\\\\/WUVJhUomE+cP2veZTReUtT32gk7\\\\\\\/qKILV\\\\\\\/Yjnw8MMRvAYKeCea6613WdB9zK\\\\\\\/jIMoDnuGqiuAXBXTHsZNQQF191wuR\\\\\\\/13oAwZjl1rw4mPMQzFvwGA\\\\\\\/MZ565L1Reku4bKhSnSIMDvoaN9n55iRngCi\\\\\\\/vn\\\\\\\/4oF67yf87X95mZ35pGie6XlCa8XKcIHLyV1sKJfYDobr2FdYPjfB9UzelRq0mVC4y8ecTzA3O0D6vU58mWWEw0vxuG96rtB1cb8zG47CQ7v644AT2+39JXtVn6iI59YW96OFGrGJgas+Y6ewthQHwULOw+ZKgm5q3ZcE3RVN8YrgEi2w+tA\\\\\\\/6ZQri48ticwlqhJhZuWSVxg3QkeQ4pJUkGGWkQZgyZ6eGjH7RGMlvnlJK3slryD4ZEVLIYeApOL9ftsDN8svkglU198vQv77foY5dr0+0cdYv0Fq655yAokO7pN4\\\\\\\/l\\\\\\\/zTG3HXC9+nJXdWpCBQ3oLttBu1wGNubHgIvlTHX7hmW9\\\\\\\/NoKH4Hzcy5g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LwBX9IKDojADfsnSAovIvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225e8a6a522695e6f6768cd5d5ea58981c4af1834899ab2d535012a2c52f252613\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ua4lOeetdRNhbxdmoNl68EyjPq4k5u3B69mD9T3OAP83Fqt8cgpFlTj23i2OIoSgZOS48a8LF5b5RWZzIIUTW78yTeiARXQ22VCn6mGSJgUlQl3duonwKUNhx00KvSxfDIm+BiWNBGBl3mL9QQYSKHf2cIT1YLPvP9sjKSyoRWV6j6dpNcA7K\\\\\\\/4nUNQBb79HnDPB9k5amLRgQ8zYyL\\\\\\\/1WF9cwjX9011fkb1kEidCOSI77+\\\\\\\/qKvN04lhsZTYs6jUWF2O+JJdG8qTD4spVQFeyewdCGy21QAOMAx\\\\\\\/b+hqLuicCuLZOeBc8pnqOsUF17qFzEOnc3BVAsKTanrL\\\\\\\/2QfWLnbveG7cqajeV7gKFa\\\\\\\/3I8HUcALlaAST9GAQyyJLbmO0x89pwCv8DDWqm0\\\\\\\/SLyQk1H2fDNGh0kDoy2q7qKXyudRUAFepvT8skJqUQGbbAdtfeTV1gCG\\\\\\\/RlOtQWR5ZA7TplixUTzrJcEupwvlEeKy0ArUmthSrJQmzIdNxZW1Cway82Nf3jh2W4Rg9h\\\\\\\/h+MWw4HI36iZsRm+TKDri9PuwYf5w5gtuHnDvpgc57k5jqU930sYvPPErnNlFRC8YBO2G+McZjJp4xWoHA8N\\\\\\\/qbIdXDZFGM3xU3A3OLmr8rBXA3LDGAbPRGCbqAYTGrHmNslWKRuydvOfrSZfTG8T1o8aT8ktSH4STuO6OYm7zPjNb+3T7C85ZYtatLEYaGGWz8LnyF0MFDkFTPUXXwhn\\\\\\\/ls65lRZpc2sJfpbSWrMeXdnqHyEZ2NbUguAIGa5KoMTbW5NvQTCKt\\\\\\\/\\\\\\\/rTYc9G6NxWtwgdZyNVIJ\\\\\\\/B8sFJXN9t23u1nImcOy5fil6ixkZIRKKpD9EHLFM\\\\\\\/2egCpTFq6vjkgkWo+MrOdZngU+rvEfEhOT5KqtGYyoKsOpvM1vc5hwoUVeRUiUTMCazc8lTDO0v6tM2YOA\\\\\\\/tMCrLqydCDyEVSWn2jNv5o6h451h96AtMOY\\\\\\\/EfVeXhtOTgi2DCOqdYrgwUFkp5vNVj95cYyGhuF9uck8bG6N7a7dBTEqr4ZyQn\\\\\\\/z+RrxkF27xveobp+GojdoLxOTTxMTSTwXOhTPBMWkm1OpxFqctOApICd20TroTMvHT38Q+096PWLkk8sZAPGWJRPgL3aDmCrlcUrp6HSZ4j6rxGZTna8C3jzH0k5e6sK\\\\\\\/2MaMKiHCNemMWJR4x9B8DOGCgGVpWGsH1xlZAPbMEsUGrzTsejsqLw0zD3J8mh9D0sIdWLK8eGjy6M\\\\\\\/qVbLPeUFssRwyk61qLvY+MNf\\\\\\\/MQXdJGFPsAlK97wYhRnpN0ZcCdQ3gSnIji5dRYfxZLH0yi2VVQtgz3QDw9G6a6DZIv6+ja9v3I5qwta7h5+Ew8Lhe+gbNGoMwcmSGWnf4BbWhuArFrtR1bC0aWCM9X3TwQHIJqZbxUEcdElraCg64eFNbChwb7piPGBUGUBA9A6IgyRvsoXR5VEM9raeG59alne50L2F\\\\\\\/ohRjB9JGkIbEGRjJgeyy+b4p6NWle1iV44XgmpHI3kxaBtDldIzEEzoqkpEjsbvdZddc8rQhNyuZ+UBd3Eqv4hbbOVcIBgOKjlenZdDn35kpfyQRfIt7PRRebo9vjtyFZE2+zS0\\\\\\\/6CzbwvO5LRfNVG+lWobpfJuO9jfyLLPDguuy2V06OE3B77iPUJDXaU8GBwz9BU8qKJ3EasGdVab1js5c\\\\\\\/tyKXr8UmVc+SFYWIY9xz7GQDwoJFQs1MCPQmhsaT7NqLMfgMfxLuhVrXUPNW++jcIZqj0tX3I+wULKLWtBmOlO1E1iL\\\\\\\/GjYKQ2QPV5uKzDAy5EdaDC57Vf1zMFnU5rMh\\\\\\\/dyvTa3ektl9Bus+TrBp9YXBW\\\\\\\/aRBDu6lh+NNd7csZpFkFl\\\\\\\/a7LlWQi8m\\\\\\\/ddQ2YuSylsgTqBkFXkyCQ7ougS29VHj+tbY0Po\\\\\\\/f9P3ihOsQ3sYhAgitDt8Oo5hm465vWmyXeG62j5IBYd7au1u5TVlVvULHevUb6dEZPdaozz34cSX6DWTapNUCLDh+w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TbU6mIr8Z6AvP36YiU2h2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022842320b53a13001216d39d26fdd5d6f3b7f81974e16147f5830e8453b197fa5b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459170 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/187" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i_1368x5ywGYXNDJR76YrKWlwNJQono0XnSeLKEBxwg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/187" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--uRCKV6NjC0hKY9YmH9xtO4h41VgKV9CDd9-8axfj0Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/3/183" title="Mutation Rate Inferred From Synonymous Substitutions in a Long-Term Evolution Experiment With Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/3/197" title="Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459170" id="node459170" data-pisa="ggg;1/3/187" data-pisa-master="ggg;g3.111.000182" data-apath="/ggg/1/3/187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus <em>thresh-1</em> Using the Illumina GoldenGate Assay</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Inga Schmalenbach</span>, <span class="highwire-citation-author" data-delta="1">Timothy J. March</span>, <span class="highwire-citation-author" data-delta="2">Thomas Bringezu</span>, <span class="highwire-citation-author" data-delta="3">Robbie Waugh</span> and <span class="highwire-citation-author" data-delta="4">Klaus Pillen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">187-196;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000182 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459170" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Inga Schmalenbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Max Planck Institute for Plant Breeding Research, Department of Plant Breeding and Genetics, 50829 Cologne, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Inga%2BSchmalenbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmalenbach%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInga%2BSchmalenbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy J. March </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ.%2BMarch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=March%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BJ.%2BMarch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas Bringezu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBringezu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bringezu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BBringezu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robbie Waugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Scottish Crop Research Institute, Invergowrie, Dundee DD2 5DA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robbie%2BWaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waugh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobbie%2BWaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Klaus Pillen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klaus%2BPillen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pillen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKlaus%2BPillen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:klaus.pillen@landw.uni-halle.de" class="" data-icon-position="" data-hide-link-title="0">klaus.pillen@landw.uni-halle.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/3/187" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459170" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459170/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/187.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459170" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459170/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/187.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459170" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459170/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/187.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459170" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459170/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/3/187.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459170" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459170/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genetically well-characterized mapping populations are a key tool for rapid and precise localization of quantitative trait loci (QTL) and subsequent identification of the underlying genes. In this study, a set of 73 introgression lines (S42ILs) originating from a cross between the spring barley cultivar Scarlett (<em>Hordeum vulgare</em> ssp. <em>vulgare</em>) and the wild barley accession ISR42-8 (<em>H. v</em>. ssp. <em>spontaneum</em>) was subjected to high-resolution genotyping with an Illumina 1536-SNP array. The array enabled a precise localization of the wild barley introgressions in the elite barley background. Based on 636 informative SNPs, the S42IL set represents 87.3% of the wild barley genome, where each line contains on average 3.3% of the donor genome. Furthermore, segregating high-resolution mapping populations (S42IL-HRs) were developed for 70 S42ILs in order to facilitate QTL fine-mapping and cloning. As a case study, we used the developed genetic resources to rapidly identify and fine-map the novel locus <em>thresh-1</em> on chromosome 1H that controls grain threshability. Here, the recessive wild barley allele confers a difficult to thresh phenotype, suggesting that <em>thresh-1</em> played an important role during barley domestication. Using a S42IL-HR population, <em>thresh-1</em> was fine-mapped within a 4.3cM interval that was predicted to contain candidate genes involved in regulation of plant cell wall composition. The set of wild barley introgression lines and derived high-resolution populations are ideal tools to speed up the process of mapping and further dissecting QTL, which ultimately clears the way for isolating the genes behind QTL effects.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/wild-barley-introgression-lines" class="hw-term hw-article-keyword hw-article-keyword-wild-barley-introgression-lines" rel="nofollow">wild barley introgression lines</a></li><li class="kwd"><a href="/keyword/illumina-goldengate-assay" class="hw-term hw-article-keyword hw-article-keyword-illumina-goldengate-assay" rel="nofollow">Illumina GoldenGate assay</a></li><li class="kwd"><a href="/keyword/high-resolution-mapping-populations" class="hw-term hw-article-keyword hw-article-keyword-high-resolution-mapping-populations" rel="nofollow">high-resolution mapping populations</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/threshability-locus-thresh-1" class="hw-term hw-article-keyword hw-article-keyword-threshability-locus-thresh-1" rel="nofollow">threshability locus <em xmlns:default="http://www.w3.org/1999/xhtml">thresh-1</em></a></li></ul><p id="p-5">As demonstrated in various crop species, introgression lines (ILs) are a valuable genetic resource for the identification of QTL for important agronomic traits (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eshed and Zamir 1995</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Faris and Gill 2002</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liu <em>et al.</em> 2006</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mei <em>et al.</em> 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Simons <em>et al.</em> 2006</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Uauy <em>et al.</em> 2006</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Szalma <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Falke <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fu <em>et al.</em> 2009</a>). An IL set represents the genome of a donor parent through single lines each carrying one or few introgressed donor segments in the same genetic background of the recurrent parent. This is achieved by several rounds of backcrossing to the recurrent parent followed by marker-assisted selection (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zamir 2001</a>). Compared with traditional mapping populations such as recombinant inbred lines (RILs), ILs offer increased statistical power to detect small QTL effects due to the removal of confounding segregating alleles at background genomic regions (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Law 1966</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Keurentjes <em>et al.</em> 2007</a>). As a result, complex traits, controlled by several unlinked genes, can be broken down into simple Mendelian factors (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Deng <em>et al.</em> 2011</a>). In addition, because only small genetic regions are introgressed, favorable alleles from exotic species can be incorporated, exhibiting no or only a limited number of additional unfavorable effects that might cosegregate as linkage drag. All in all, these factors make ILs a useful base to embark on fine-mapping and cloning of important QTL.</p><p id="p-6">ILs developed in tomato have been used extensively to map QTL controlling complex quantitative traits including fruit weight, sugar content, and plant size (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alpert and Tanksley 1996</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Eshed and Zamir 1995</a>). In barley, a subset of near isogenic lines were developed for fine-mapping a locus controlling leaf rust (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Marcel <em>et al.</em> 2007</a>). QTL controlling grain weight (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Röder <em>et al.</em> 2008</a>) and the number of grains per ear (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2010a</a>) were fine-mapped in wheat using ILs. Additionally in tomato, segregating populations derived from ILs containing QTL facilitated the cloning of genes controlling fruit size (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Frary <em>et al.</em> 2000</a>) and fruit sugar content (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fridman <em>et al.</em> 2004</a>).</p><p id="p-7">A dense genetic map which allows for localizing the introgressed segments with high resolution is crucial for the selection of ILs containing only a small portion of the introgressed genome. So far, the majority of IL sets has been developed and applied for QTL mapping based on a relatively low number of markers that have been generated from anonymous genomic regions (<em>e.g.</em>, simple sequence repeats, restriction fragment length polymorphisms). Currently, gene-based molecular markers originating from cDNAs/ESTs (expressed sequenced tags) have gained importance in plant genetics and genomics-assisted breeding (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kota <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deleu <em>et al.</em> 2009</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Varshney <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chin <em>et al.</em> 2010</a>). Here, EST–single nucleotide polymorphisms (SNPs) are especially useful due to their high abundance and adaptability to high-throughput, low-cost genotyping assays. Markers derived from ESTs allow for detecting polymorphisms within protein-coding transcribed genes and thus may facilitate gene isolation via map-based cloning or comparative genomics (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Stein <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hackauf <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Muchero <em>et al.</em> 2011</a>).</p><p id="p-8">In barley (<em>Hordeum vulgare</em>), <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Close <em>et al.</em> (2009)</a> identified a significant number of genic SNPs from ESTs and sequenced PCR amplicons and used them to develop two Illumina barley oligo pool assays (BOPA1 and BOPA2), each enabling the simultaneous genotyping of 1536 SNPs. Furthermore, a barley consensus genetic map comprising 2943 SNPs has been created from linkage maps of four reference populations (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Close <em>et al.</em> 2009</a>). The advantage of these resources has been proven by diverse association mapping studies (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rostoks <em>et al.</em> 2006</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cockram <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Comadran <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2010</a>). So far, two collections of backcross-derived barley lines have been developed and characterized using the Illumina genotyping arrays and the above described consensus map. <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Druka <em>et al.</em> (2011)</a> constructed a set of near isogenic lines carrying mutant alleles for most of the morphological and developmental variation in barley. <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sato and Takeda (2009)</a> reported on the characterization of a set of recombinant chromosome substitution lines, each containing a small portion of a wild barley accession introgressed into the genetic background of an elite parent. Both populations are proposed as valuable resources for identifying genes underlying simple and quantitative traits.</p><p id="p-9">Such populations might also be helpful to shed further light on genes which were selected during domestication. In barley domestication-related genes like brittleness, spike row-type, flowering time control under photoperiod and vernalization signals are already mapped or cloned (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Azhacuvel <em>et al.</em> 2006</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Komatsuda <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ramsay <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Turner <em>et al.</em> 2005</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yan <em>et al.</em> 2004</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">2005</a>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">2006</a>). Further genes that control important domestication traits like grain size, grain yield or threshability of grains are still awaiting molecular discovery.</p><p id="p-10">In the present study, we aimed to characterize the previously selected barley S42IL population, originating from the cross Scarlett (<em>H. vulgare ssp. vulgare</em>, hereafter abbreviated <em>Hv</em>) × ISR42-8 (<em>H. vulgare ssp. spontaneum</em>, hereafter abbreviated <em>Hsp</em>) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>) by genotyping with the Illumina BOPA1 array (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Close <em>et al.</em> 2009</a>). In addition, we intended to further develop the S42ILs into a resource for rapid identification, fine-mapping, and positional cloning of QTL. Besides a dense genetic map, the major requirement for these applications is the development of large populations with sufficient recombination events in the target QTL region. In order to increase the recombination events within the target QTL interval, we developed for most introgression lines a high-resolution mapping population (S42IL-HR) segregating within the introgressed region. As a case study, we describe the application of the developed resources to rapidly identify and fine-map a genomic region on chromosome 1H containing the novel locus <em>thresh-1</em>, derived from <em>Hsp</em>, which controls grain threshability.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-11">A set of 73 wild barley introgression lines (hereafter referred to as S42ILs) was subjected to high-throughput genotyping with the Illumina GoldenGate assay. The S42ILs are derived from a cross between the German malting barley cultivar Scarlett and the Israeli wild barley accession ISR 42-8. As reported in <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Von Korff <em>et al.</em> (2004)</a>, the initial cross was backcrossed twice to produce the advanced backcross doubled haploid population S42 consisting of 301 BC<sub>2</sub>DH lines. The S42ILs were generated based on 40 lines selected from that population (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Von Korff <em>et al.</em> 2004</a>). Here, a strategy combining one further round of backcrossing, two to four selfings and marker-assisted selection with simple sequence repeat (SSR) markers was applied (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The development of 59 S42ILs (S42ILs-101 to -157, -175, and -176) as well as their initial genetic characterization with altogether 98 SSR markers is described in detail in <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> (2008)</a>. Since then, 14 new lines (S42ILs-158 to -164, -166 to -171, and -173) were selected following the same strategy as described above (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Based on the initial SSR genotype data, each single line contained one or two chromosomal segments of the exotic parent ISR 42-8 (<em>Hsp</em>) within the genetic background of the elite parent Scarlett (<em>Hv</em>) (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>). The genotyping of the 73 S42ILs with the Illumina GoldenGate assay was carried out with BC<sub>3</sub>S<sub>6</sub> plants which originate from a single BC<sub>3</sub>S<sub>4</sub> line (S42IL).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/187/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strategy for developing introgressions lines and high-resolution mapping populations from the wild barley cross Scarlett &#xD7; ISR42-8. In order to select a set of introgression lines (S42ILs), a strategy combining backcrossing, repeated selfing, and marker-assisted selection (MAS) was applied. In the BC2S4:6 generation, the S42ILs have been characterized precisely on the genotype level using Illumina SNPs as described in the present study. Additionally, phenotype data were assessed for a subset of 39 S42ILs in order to validate QTL effects detected in the parental population S42 (Schmalenbach et al. 2008; Schmalenbach et al. 2009; Schmalenbach and Pillen 2009). High-resolution mapping populations (S42IL-HR) were finally developed by backcrossing the S42ILs with the recurrent parent followed by two selfings. They are available for future fine-mapping and cloning of interesting genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783670201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strategy for developing introgressions lines and high-resolution mapping populations from the wild barley cross Scarlett &#xD7; ISR42-8. In order to select a set of introgression lines (S42ILs), a strategy combining backcrossing, repeated selfing, and marker-assisted selection (MAS) was applied. In the BC&lt;sub&gt;2&lt;/sub&gt;S&lt;sub&gt;4:6&lt;/sub&gt; generation, the S42ILs have been characterized precisely on the genotype level using Illumina SNPs as described in the present study. Additionally, phenotype data were assessed for a subset of 39 S42ILs in order to validate QTL effects detected in the parental population S42 (Schmalenbach &lt;em&gt;et al.&lt;/em&gt; 2008; Schmalenbach &lt;em&gt;et al.&lt;/em&gt; 2009; Schmalenbach and Pillen 2009). High-resolution mapping populations (S42IL-HR) were finally developed by backcrossing the S42ILs with the recurrent parent followed by two selfings. They are available for future fine-mapping and cloning of interesting genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/187/F1.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/187/F1.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/187/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/187/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460756" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-12" class="first-child">Strategy for developing introgressions lines and high-resolution mapping populations from the wild barley cross Scarlett × ISR42-8. In order to select a set of introgression lines (S42ILs), a strategy combining backcrossing, repeated selfing, and marker-assisted selection (MAS) was applied. In the BC<sub>2</sub>S<sub>4:6</sub> generation, the S42ILs have been characterized precisely on the genotype level using Illumina SNPs as described in the present study. Additionally, phenotype data were assessed for a subset of 39 S42ILs in order to validate QTL effects detected in the parental population S42 (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schmalenbach <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schmalenbach and Pillen 2009</a>). High-resolution mapping populations (S42IL-HR) were finally developed by backcrossing the S42ILs with the recurrent parent followed by two selfings. They are available for future fine-mapping and cloning of interesting genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">To enable fine-mapping and map-based cloning of interesting phenotypes, high-resolution mapping populations derived from the S42ILs were generated through one further round of backcrossing with Scarlett and two consecutive rounds of selfing (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). For backcrossing, one to five S42IL plants in BC<sub>3</sub>S<sub>6</sub> were randomly chosen to produce BC<sub>4</sub>S<sub>0</sub> seeds. For selfings, at least 6 BC<sub>4</sub>S<sub>0</sub> seeds and, respectively, 100 BC<sub>4</sub>S<sub>1</sub> seeds were randomly chosen to reduce the risk of selection or drift effects present in the resulting BC<sub>4</sub>S<sub>2</sub> HR populations.</p></div><div id="sec-3" class="subsection"><h3>Extraction of genomic DNA</h3><p id="p-14">For DNA isolation, 30-50 mg leaf material of 2-week-old seedlings grown in the greenhouse was harvested for each S42IL. Per line, leaf material from on average 12 BC<sub>3</sub>S<sub>6</sub> plants was pooled. After adding 400 µl RLT buffer (Qiagen, Hilden, Germany), the material was homogenized using a TissueLyser bead mill (Qiagen) and extracted using the BioSprint DNA Plant Kit and the BioSprint 96 workstation from Qiagen. Isolated DNA was disolved in distilled water, and, based on agarose gel electrophoresis, DNA concentration of all samples was assessed. If required, samples were concentrated using a Savant SpeedVac concentrator (GMI) to achieve a final concentration of about 100 ng/µl. For Illumina SNP genotyping, a sample volume of 25 µl was provided.</p></div><div id="sec-4" class="subsection"><h3>Genotyping and characterization of the S42ILs with the Illumina GoldenGate assay</h3><p id="p-15">DNA samples were submitted to the Southern California Genotyping Consortium (SCGC), Illumina BeadLab at the University of California, Los Angeles (<a href="http://scgc.genetics.ucla.edu/">http://scgc.genetics.ucla.edu/</a>) and genotyped with the 1536-SNP barley BOPA1 set (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Close <em>et al.</em> 2009</a>). Forty-one S42ILs (S42IL-101 to -139, -149, and -150) were genotyped in two technical replications and the remaining 32 lines in one replication. In addition, the parents Scarlett and ISR42-8 were analyzed in four replicates each.</p><p id="p-16">Obtained raw data were transformed to genotype calls and subsequently manually supervised to correct for excessive emphasis on heterozygote calls using GenCall software (Illumina, San Diego, CA) at the Close lab (University of California, Riverside, CA). Only the most reliable calls were retained. All SNPs, which had no genotype or map data, were nonpolymorphic between the parents, or showed ambiguous and nonreproducible genotypes, were discarded. The informative SNPs used for characterizing the S42ILs are designated by BOPA1 numbers (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Close <em>et al.</em> 2009</a>). Detailed information such as the according HarvEST unigene assembly #32 numbers is given in supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS1.xlsx">Table S1</a>. The genetic order of all markers was taken from the <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Close <em>et al.</em> (2009)</a> consensus map which included 2943 SNP loci. Based on their graphical genotypes, obtained using the Graphical GenoTypes (GGT) software (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Van Berloo 1999</a>), the S42ILs where ordered according to the chromosomal positions of their overlapping target introgressions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS1.xlsx">Table S1</a>). The size of the target introgression was calculated for each line, where the half-intervals flanking a marker locus were assumed to be of the same genotype. For calculating the portion of <em>Hsp</em> genome per S42IL, a total genome size of 1576 cM was assumed, based on <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Close <em>et al.</em> (2009</a>) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS1.xlsx">Table S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/187/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical genotypes of 73 Hsp introgression lines (S42ILs). The 73 S42ILs, presented vertically, have been characterized with 636 Illumina SNPs, shown horizontally. The order of the S42ILs is in accordance with Table 1, and the marker order is taken from the consensus map of Close et al. (2009). Each S42IL carries one or several homozygous or heterozygous wild barley introgressions (depicted in black and dark gray, respectively) in the genetic background of the elite parent (in light gray). Missing marker data are indicated in white." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783670201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graphical genotypes of 73 &lt;em&gt;Hsp&lt;/em&gt; introgression lines (S42ILs). The 73 S42ILs, presented vertically, have been characterized with 636 Illumina SNPs, shown horizontally. The order of the S42ILs is in accordance with Table 1, and the marker order is taken from the consensus map of Close &lt;em&gt;et al.&lt;/em&gt; (2009). Each S42IL carries one or several homozygous or heterozygous wild barley introgressions (depicted in black and dark gray, respectively) in the genetic background of the elite parent (in light gray). Missing marker data are indicated in white.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/187/F2.medium.gif" width="321" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/187/F2.medium.gif" width="321" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/187/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/187/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460766" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-17" class="first-child">Graphical genotypes of 73 <em>Hsp</em> introgression lines (S42ILs). The 73 S42ILs, presented vertically, have been characterized with 636 Illumina SNPs, shown horizontally. The order of the S42ILs is in accordance with <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and the marker order is taken from the consensus map of <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Close <em>et al.</em> (2009)</a>. Each S42IL carries one or several homozygous or heterozygous wild barley introgressions (depicted in black and dark gray, respectively) in the genetic background of the elite parent (in light gray). Missing marker data are indicated in white.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Phenotyping for grain threshability</h3><p id="p-18">To evaluate grain threshability (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) a subset of 49 S42ILs (S42IL-101 to -144, -146 to -149, and -153) and Scarlett as the control genotype were evaluated in two independent glasshouse experiments. Each experiment consisted of three completely randomized blocks. Ten plants of each genotype were grown in 1.5 L pots filled with a cultivation substrate containing peat, clay, and NPK fertilizer with 250, 300, and 400 mg/L nitrogen, phosphate, and potassium, respectively, pH 5.8. Plants were grown under 14/10-hr day/night photoperiod with a daily temperature range of 15°C to 22°C. At maturity grain spikes were collected and threshed using a rotating home-made threshing drum. The rotational speed of the threshing drum and duration of threshing was adjusted so that grains from Scarlett were completely removed from the spike rachis and the awns were detached from the grain. Threshability was scored as a qualitative trait, where a genotype was considered difficult to thresh if parts of the awns and/or the rachis remained attached to the grain after the outlined threshing procedure (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Based on the results of this initial experiment, 91 individual seeds from the high resolution mapping population S42IL-143HR were selected depending on their threshability phenotype. The selected HR individuals were grown, along with S42IL-143 and Scarlett as controls, in a second experiment under the same glasshouse conditions as stated above and phenotyped for threshability to enable fine-mapping of the grain threshability locus.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/187/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypes of the thresh-1 gene after mechanical threshing. Left image: Difficult to thresh grains from line S42IL-143, homozygous for the recessive Hsp allele thresh-1 where parts of the awns and the rachis remain attached to the grain. Right image: Easy to thresh grains from cultivar Scarlett, homozygous for the dominant Hv allele Thresh-1 without remnants from awns and rachis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783670201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypes of the &lt;em&gt;thresh-1&lt;/em&gt; gene after mechanical threshing. Left image: Difficult to thresh grains from line S42IL-143, homozygous for the recessive &lt;em&gt;Hsp&lt;/em&gt; allele &lt;em&gt;thresh-1&lt;/em&gt; where parts of the awns and the rachis remain attached to the grain. Right image: Easy to thresh grains from cultivar Scarlett, homozygous for the dominant &lt;em&gt;Hv&lt;/em&gt; allele &lt;em&gt;Thresh-1&lt;/em&gt; without remnants from awns and rachis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/187/F3.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/187/F3.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/187/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/187/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460730" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-19" class="first-child">Phenotypes of the <em>thresh-1</em> gene after mechanical threshing. Left image: Difficult to thresh grains from line S42IL-143, homozygous for the recessive <em>Hsp</em> allele <em>thresh-1</em> where parts of the awns and the rachis remain attached to the grain. Right image: Easy to thresh grains from cultivar Scarlett, homozygous for the dominant <em>Hv</em> allele <em>Thresh-1</em> without remnants from awns and rachis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Genotyping the S42IL-143HR population</h3><p id="p-20">For genotyping of the S42IL-143HR population, the Illumina BOPA1 markers were converted to either cleaved amplified polymorphic sequence (CAPS) or pyrosequencing markers as described previously (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2010b</a>). Primer information is detailed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS2.xlsx">Table S2</a>. In total nine BOPA1 markers were converted to genotype <em>Hsp</em> introgressions on chromosomes 1H, 5H, 6H, and 7H. In addition, the SSR marker HvABAIP, which was originally used to select the line S42IL-143 (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>), along with the CAPS marker HvFT3 previously shown to map within the 1H introgression of S42IL-143 (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2010b</a>) were also used for genotyping. Genetic distances between the markers were calculated with the Kosambi mapping function in JoinMap v.3.0 (Kyazma B.V, Wageningen, The Netherlands) and graphical genotypes were visualized with the GGT software (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Van Berloo 1999</a>).</p></div><div id="sec-7" class="subsection"><h3>Identification of genes in the <em>thresh-1</em> region</h3><p id="p-21">The previously published virtual gene order of barley chromosome 1H (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mayer <em>et al.</em> 2009</a>) was used to predict the genes located in the <em>thresh-1</em> region. The SSR marker HvABAIP (GenBank accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=X13498&amp;atom=%2Fggg%2F1%2F3%2F187.atom">X13498</a>) was positioned into the chromosome 1H gene order scaffold by performing a blastx search against the rice genome sequence to identify the orthologous gene position in rice (<a href="http://blast.ncbi.nlm.nih.gov">http://blast.ncbi.nlm.nih.gov</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genetic characterization of the S42ILs</h3><p id="p-22">A set of 73 wild barley introgression lines was genotyped with high resolution using the Illumina GoldenGate assay. Out of 1536 BOPA1 SNPs, 1148 markers gave useful genotype information in the S42IL set. Of these, a total of 636 SNPs (55.4%) were polymorphic between Scarlett and ISR42-8 and were finally used for characterizing the S42ILs. The average marker density varied from 1.9 cM for chromosome 2H (228 cM/120 SNPs) to 3.1 cM for chromosome 7H (230 cM/75 SNPs). Overall, 87.3% of the <em>Hsp</em> genome (1376 out of 1576 cM) is represented by, in most cases overlapping, target <em>Hsp</em> introgressions in the <em>Hv</em> genetic background (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Wild barley chromosomes 1H, 4H, 6H, and 7H are completely represented, whereas the lowest genome coverage was estimated for 5H (63.3%). The biggest gaps where no lines are developed yet were detected on top of chromosomes 5H and 3H with 98.4 cM and 49.6 cM, respectively.</p><p id="p-23">A number of lines carry a second nontarget <em>Hsp</em> segment that covers a chromosomal region not represented by primary introgressions in the complete set of lines. An example is S42IL-114 possessing the target introgression on chromosome 3H plus an additional segment on chromosome 5H, which has a size of 14.4 cM (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS1.xlsx">Table S1</a>). Taking into account these additional introgressions, the total <em>Hsp</em> genome coverage of the S42IL set increases to 89.5%.</p><p id="p-24">As shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, 20 S42ILs carry a single introgression (<em>i.e.</em>, only the target one), 22 lines contain one additional homozygous or heterozygous <em>Hsp</em> segment, and 31 lines exhibit two to four secondary introgressions. Three target segments are represented by two lines simultaneously (<em>i.e.</em>, S42IL-118 and -120, and -119 and -120 on chromosome 4H, and -149 and -152 on chromosome 6H). The target introgressions possess an average size of 42.9 cM, ranging from 5.7 cM in S42IL-110 to 109.7 cM in S42IL-114 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Taking into account all homozygous introgressions, on average 3.3% of the wild barley genome is represented per line, varying from 0.8% in S42IL-170 to 8.4% represented by four independent introgressions in S42IL-156. Furthermore, 38 lines exhibited heterozygosity at one or more SNP loci. A core set of 32 S42ILs that represents the minimum number of lines required to cover the <em>Hsp</em> genome was selected (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Here, preferably lines with overlapping introgressions were picked to ensure a maximum coverage of the donor genome. The remaining 41 S42ILs are partial duplicates with shorter introgressions that can be useful for fine-mapping of markers and QTL.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460714/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460714/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Genetic characteristics of 73 <em>Hsp</em> introgression lines (S42ILs) based on genotyping with 636 Illumina SNPs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Development of high-resolution mapping populations</h3><p id="p-34">After backcrossing the S42ILs with Scarlett and two further rounds of selfing, a library of 70 high-resolution populations (S42IL-HR) in BC<sub>4</sub>S<sub>2</sub> generation was developed (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Each HR population includes between 17 and 2096 grams of seed, segregating for markers and genes that are located within the introgression of the original S42IL. Assuming an average thousand grain weight of 40 g (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Schmalenbach <em>et al.</em> 2009</a>), these values correspond to a range from 425 to 52,400 segregating BC<sub>4</sub>S<sub>2</sub> individuals per HR population. So far, 42 HR populations consist of at least 400 g of seed (<em>i.e.</em>, ≥10,000 individuals). Fourteen HR populations are currently multiplied again in the field, as indicated in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>. A first application of a HR population is shown by the following mapping of the <em>thresh-1</em> locus.</p></div><div id="sec-11" class="subsection"><h3>Genetic mapping of the thresh-1 locus</h3><p id="p-35">From 49 examined S42ILs, only line S42IL-143 was found to show reduced grain threshability compared with the control genotype Scarlett. After mechanical threshing, rachis and awns of line S42IL-143 were only partially removed from the grains. Line S42IL-143 was therefore classified as difficult to thresh (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). The BOPA1 genotype data showed that S42IL-143 possess a major <em>Hsp</em> introgression on the long arm of chromosome 1H (51.8 cM) plus four additional <em>Hsp</em> introgressions on chromosomes 5H (3.2 cM and 7.0 cM), 6H (28.9 cM), and 7H (4.7 cM). The additional introgressions on chromosomes 5H, 6H, and 7H were also represented by other S42ILs that did not show the difficult to thresh phenotype, thus it was assumed that the <em>thresh-1</em> gene is located within the chromosome 1H introgression.</p><p id="p-36">The location of <em>thresh-1</em> on chromosome 1H was further confirmed by phenotyping threshability of 91 plants of the high-resolution mapping population S42IL-143HR. Simultaneously, the same plants were genotyped with converted BOPA1 markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS2.xlsx">Table S2</a>) from all four introgressions of S42IL-143. The genotyping confirmed the presence of the additional <em>Hsp</em> introgressions on 5H, 6H, and 7H in S42IL-143. The phenotyping, however, proved that these loci were not genetically linked to the <em>thresh-1</em> locus. In contrast, recombinants identified within the 1H <em>Hsp</em> introgression allowed to map the <em>thresh-1</em> gene within a 4.3-cM interval between markers 1_0357 and HvABAIP (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). From 91 HR individuals phenotyped, 38 were easy to thresh and 53 were difficult to thresh. However, this segregation ratio was greatly distorted due to the individual selection of seeds, resulting in an overrepresentation of difficult to thresh genotypes. To clarify the mode of inheritance, four HR lines (#2, 19, 88, 89 in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) that proved to be heterozygous for the <em>thresh-1</em> interval between markers 1_0357 and HvABAIP were used for progeny validation. From a total of 88 progeny (<em>i.e.</em>, 22 offspring plants from each heterozygous HR plant), 61 were easy to thresh and 27 were difficult to thresh. This was not significantly different from a 3:1 segregation ratio (<em>x</em><sup>2</sup> = 1.151, <em>P</em> = 0.218) confirming that the <em>Hv</em> allele is dominant over the <em>Hsp</em> allele at the <em>thresh-1</em> locus.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/187/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fine mapping the thresh-1 locus using the high resolution mapping population S42IL-143HR. (A) Screening 49 S42ILs localized the thresh-1 gene to an Hsp introgression on chromosome 1H, which is present in line S42IL-143. (B) Graphical genotypes showing informative recombinants, identified among 91 BC4S2 individuals of the S42IL-143HR population, that were used to delimitate the thresh-1 interval based on their threshability phenotype. Lines with missing or ambiguous genotype data were excluded. The numbers below the graphical genotypes refer to the HR line depicted, while the numbers in brackets refer the total number of individuals identified with the same graphical genotype. Progeny tests from HR lines 2, 19, 88, and 89 proved that the elite (Hv) allele Thresh-1 is dominant over the exotic (Hsp) allele thresh-1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783670201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fine mapping the &lt;em&gt;thresh-1&lt;/em&gt; locus using the high resolution mapping population S42IL-143HR. (A) Screening 49 S42ILs localized the &lt;em&gt;thresh-1&lt;/em&gt; gene to an &lt;em&gt;Hsp&lt;/em&gt; introgression on chromosome 1H, which is present in line S42IL-143. (B) Graphical genotypes showing informative recombinants, identified among 91 BC&lt;sub&gt;4&lt;/sub&gt;S&lt;sub&gt;2&lt;/sub&gt; individuals of the S42IL-143HR population, that were used to delimitate the &lt;em&gt;thresh-1&lt;/em&gt; interval based on their threshability phenotype. Lines with missing or ambiguous genotype data were excluded. The numbers below the graphical genotypes refer to the HR line depicted, while the numbers in brackets refer the total number of individuals identified with the same graphical genotype. Progeny tests from HR lines 2, 19, 88, and 89 proved that the elite (&lt;em&gt;Hv&lt;/em&gt;) allele &lt;em&gt;Thresh-1&lt;/em&gt; is dominant over the exotic (&lt;em&gt;Hsp&lt;/em&gt;) allele &lt;em&gt;thresh-1&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/187/F4.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/187/F4.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/187/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/187/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460693" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-37" class="first-child">Fine mapping the <em>thresh-1</em> locus using the high resolution mapping population S42IL-143HR. (A) Screening 49 S42ILs localized the <em>thresh-1</em> gene to an <em>Hsp</em> introgression on chromosome 1H, which is present in line S42IL-143. (B) Graphical genotypes showing informative recombinants, identified among 91 BC<sub>4</sub>S<sub>2</sub> individuals of the S42IL-143HR population, that were used to delimitate the <em>thresh-1</em> interval based on their threshability phenotype. Lines with missing or ambiguous genotype data were excluded. The numbers below the graphical genotypes refer to the HR line depicted, while the numbers in brackets refer the total number of individuals identified with the same graphical genotype. Progeny tests from HR lines 2, 19, 88, and 89 proved that the elite (<em>Hv</em>) allele <em>Thresh-1</em> is dominant over the exotic (<em>Hsp</em>) allele <em>thresh-1</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Prediction of genes within the thresh-1 region</h3><p id="p-38">The previously published virtual gene order of barley chromosome 1H was used to predict the genes within the <em>thresh-1</em> region (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Mayer <em>et al.</em> 2009</a>). Reciprocal blast searches identified the SSR marker HvABAIP (ABA inducible protein; (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pillen <em>et al.</em> 2000</a>), which flanks the bottom of the <em>thresh-1</em> locus, to be a putative ortholog of the rice gene Os05g0542500. This allowed anchoring HvABAIP to the chromosome 1H virtual gene order between BOPA1 markers 2_0780 and 2_0921. The BOPA1 marker 2_0267, which flanks the top of the <em>thresh-1</em> locus, was already present in the virtual gene order, therefore enabling the <em>thresh-1</em> interval to be located. Based on the virtual gene order for this region, which was derived from synteny with the rice and sorghum genomes, the interval containing the <em>thresh-1</em> locus was predicted to contain 60 genes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS3.xlsx">Table S3</a>). Of particular interest regarding the threshability trait were the identification of cell wall–related genes that encode a cellulose synthase-like family C protein and polygalacturonase.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Genetic characterization of the S42ILs</h3><p id="p-39">The Illumina BOPA1 assay was applied for characterizing a set of 73 <em>Hsp</em> introgression lines with high resolution. In total, 636 SNPs out of 1148 (55.4%) revealed polymorphic alleles between the recurrent parent Scarlett and the donor parent ISR42-8. This polymorphism rate is similar to the amount of polymorphic BOPA1 SNPs between the elite barley cultivar Haruna Nijo and the wild barley strain H602 (51%), reported by <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sato and Takeda (2009)</a>. As described by <a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Close <em>et al.</em> (2009)</a>, SNP frequency was clearly increased when including H602 in the genotype panel used to develop the BOPA1 assay. Nevertheless, polymorphism rates around 50% between <em>Hv</em> and <em>Hsp</em> genotypes appear to be relatively low. This might be attributed in part to the SNPs’ origin from coding sequences. An ascertainment bias in favor of <em>Hv</em> alleles during the development of the ILLUMINA array might also contributed to the observed polymorphism rate (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Moragues <em>et al.</em> 2010)</a>.</p><p id="p-40">The S42IL set is an immortal genetic resource that represents most of the parental wild barley genome, where each single line contains only a small proportion of the <em>Hsp</em> donor genome. Initially, the set was developed and characterized with 98 SSR markers (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>). Based on the SSR data, each line was assumed to contain only a single homozygous <em>Hsp</em> introgression in the genetic background of the elite parent. Genotyping the S42-ILs with the BOPA1 SNPs resulted in an increased mapping precision of the introgressed segments. This, in turn, revealed the existence of additional nontarget introgressions in S42ILs, which had not been detected previously. So far, eight randomly chosen S42ILs were regenotyped with 25 informative pyrosequencing markers in order to verify these secondary introgressions. The latter were derived from original Illumina SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000182/-/DC1/TableS2.xlsx">Table S2</a>). The original Illumina assays of these markers had detected 28 <em>Hsp</em> introgressions among the eight S42ILs. Through pyrosequencing, the genotype of all 28 <em>Hsp</em> introgressions could be verified as homozygous <em>Hsp</em> or heterozygous <em>Hv/Hsp</em> (data not shown). The complete validation of all tested BOPA1 genotypes demonstrates the high accuracy of the Illumina GoldenGate assay.</p><p id="p-41">Pure S42ILs carrying only the target segments are advantageous as they enable the localization of a genetic effect (<em>e.g.</em>, QTL) to a specific chromosomal interval. For this reason, the selection of pure introgression lines from the existing S42IL-HR populations is currently in progress. Nevertheless, S42ILs with overlapping introgressions are also useful for validating and confining the genetic localization of a phenotypic effect as shown in previous S42IL studies on pathogen resistances, yield and its agronomic components, malting quality and flowering time control (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>; <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Schmalenbach <em>et al.</em> 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Schmalenbach and Pillen 2009</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2010b</a>). In addition, we are currently investigating variation in the S42ILs for traits like mineral nutrient content, nitrogen deficiency tolerance, drought tolerance and others. Based on the Illumina genotypes, we selected a core set of 32 S42ILs which represents the minimum number of lines required to achieve the <em>Hsp</em> genome coverage of the complete set. Such a core set is beneficial for an initial low-resolution genome-wide QTL screening, whereas additional lines carrying smaller introgressions can be used subsequently for QTL fine-mapping (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fridman <em>et al.</em> 2004</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Keurentjes <em>et al.</em> 2007</a>). The present set of S42ILs includes two large gaps of <em>Hsp</em> introgressions on chromosomes 5H and 3H. We assume that these gaps occurred by chance in our S42IL set due to the limited number of backcross lines we started with. We neither conducted a selection of phenotypes nor did we observe a reduced viability of plants during IL development. To achieve a complete donor genome coverage, we are currently selecting lines from BC<sub>3</sub> and BC<sub>4</sub> progenies of a pool of potential S42ILs.</p><p id="p-42">The present and other studies demonstrate the usefulness of the Illumina GoldenGate assay for developing precisely characterized advanced backcross populations (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Sato and Takeda 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Druka <em>et al.</em> 2011</a>). Because barley genome sequencing is still in progress (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schulte <em>et al.</em> 2009</a>), these populations are proposed as a key tool for high-resolution mapping of QTL and Mendelian loci and subsequent identification of the causal genes.</p></div><div id="sec-15" class="subsection"><h3>Development of high-resolution mapping populations</h3><p id="p-43">The development of a library of HR populations for most of the original S42ILs will foster both map-based cloning and the transfer of interesting exotic genes that exhibit strong and/or favorable effects in the elite barley background. By means of the new HR populations, map-based cloning of QTL seems feasible since the available SNP data of S42ILs and the corresponding QTL data for pathogen resistances, yield-related traits and malting quality traits (<a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Schmalenbach <em>et al.</em> 2008</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Schmalenbach <em>et al.</em> 2009</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Schmalenbach and Pillen 2009</a>) can be used to directly select the appropriate S42IL-HR population from the HR library. The available number of up to 52,000 seeds for each HR population can thus be used to rapidly select recombination events in close vicinity to the target gene, allowing the genetic separation of tightly linked markers/genes from the target gene.</p></div><div id="sec-16" class="subsection"><h3>Localization of the thresh-1 locus and identification of candidate genes</h3><p id="p-44">Within the S42IL population we identified the <em>thresh-1</em> locus on the long arm of chromosome 1H, where only line S42IL-143 carried the <em>Hsp</em> allele at the <em>thresh-1</em> locus. The <em>Hsp thresh-1</em> homozygous phenotype showed increased rachis and awn strength, which prevented them from breaking during normal mechanical threshing. To our knowledge, this is the first published report of such a phenotype in barley. The most well-studied grain threshing trait in barley is the brittle-rachis phenotype, where the grain spikes shatter upon applying slight mechanical force. The brittle-rachis phenotype is controlled by two tightly linked genes, <em>Btr1</em> and <em>Btr2</em>, located on chromosome 3H (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Takahashi and Hayashi 1964</a>). Therefore, based on the phenotypic and genetic differences, we concluded that <em>thresh-1</em> and <em>Btr1</em>/<em>Btr2</em> are unrelated.</p><p id="p-45">During domestication of barley, the <em>Btr1</em>/<em>Btr2</em> genes were selected against by early farmers to prevent seed loss before harvesting. Similar findings hold true for wheat domestication where, over time, early farmers selected at independent loci the alleles <em>br</em>, <em>tg</em>, <em>sog</em>, and <em>Q</em> to control brittleness, glume tenacity, glume softness, and threshability of wheat spikes (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sood <em>et al.</em> 2009</a>). It is likely that early farmers also selected against the <em>thresh-1</em> allele to facilitate easier harvesting. It remains open what evolutionary advantage the exotic <em>thresh-1</em> allele provided prior to domestication. The remaining of the awns or parts thereof with the grain might have increased the chances of long distance seed dispersal through animals. After map-based isolation of the <em>thresh-1</em> allele, further studies analyzing its allele frequency in wild barley populations might help to better understand the evolutionary history of this gene.</p><p id="p-46">We were rapidly able to reduce the <em>thresh-1</em> locus to an interval of 4.3 cM by utilizing the S42IL-143HR high resolution mapping population, in combination with genotyping of polymorphic BOPA1 SNP markers for the target region. Furthermore, we could take advantage of the recent low-coverage sequencing of barley chromosome 1H and subsequent gene order prediction (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Mayer <em>et al.</em> 2009</a>) to identify candidate genes located in the <em>thresh-1</em> interval. Based on the increased mechanical force required to remove the rachis and awns from the seeds we hypothesize that the <em>thresh-1</em> gene is involved in altering the cell wall composition of the spike. In support of this hypothesis, the genomic <em>thresh-1</em> region is predicted to contain at least two orthologous cell wall related genes from rice encoding for cellulose synthase-like family C (CSLC7) and polygalacturonase proteins. In barley, the CSLC gene family is known to comprise at least four members (CSLC1-4) with CSLC1 being orthologous to the rice CSLC7 gene (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dwivany <em>et al.</em> 2009</a>). The CSLC1 has been mapped in barley to the long arm of chromosome 1H in the vicinity of the <em>thresh-1</em> locus (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burton <em>et al.</em> 2010</a>). The proposed function of the CSLC gene family in plants is to synthesize xyloglucan (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cocuron <em>et al.</em> 2007</a>), which is a major component in strengthening cell walls. In contrast, polygalacturonase enzymes are involved in the degradation of pectin and subsequent weakening of the cell wall. In <em>Arabidopsis</em>, loss of function mutants for the polygalacturonase genes ADPG1 and ADPG2 caused the siliques not to shatter at maturity by preventing cells of the dehiscence zone from breaking apart (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ogawa <em>et al.</em> 2009</a>). Based on this evidence, both CSCL1 and polygalacturonase are potential candidates for <em>thresh-1</em>. Further work is now under way to screen several thousand lines from the S42IL-143HR population to positionally clone and then functionally analyze the <em>thresh-1</em> gene.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank Dr. Joseph DeYoung at the Southern California Genotyping Consortium, University of California, Los Angeles, CA, for carrying out the ILLUMINA GoldenGate genotyping assays; Dr. Timothy Close and Dr. Prasanna Bhat, University of California, Riverside, CA, for carrying out the SNP genotype calling; and Merle Noschinski, Helga Sängerlaub, and Astrid Hoffmann for technical assistance in preparing the DNA samples of the S42ILs. I.S. and T.J.M. were supported by the German Plant Genome Research Initiative (GABI) of the Federal Ministry of Education and Research (projects 0313125B and 0315041B). T.B. was supported by the Ministry of Education of the State of Saxony-Anhalt.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-12"><p id="p-48"><em>Communicating editor: Bikram S. Gill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-04-06"><span class="received-label">Received </span>April 6, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-05-25"><span class="accepted-label">Accepted </span>May 25, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Schmalenbach <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.1" data-doi="10.1073/pnas.93.26.15503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alpert</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">High-resolution mapping and isolation of a yeast artificial chromosome contig containing fw2.2: A major fruit weight quantitative trait locus in tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">15503</span>–<span class="cit-lpage">15507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAlpert%26rft.auinit1%253DK.%2BB.%26rft.volume%253D93%26rft.issue%253D26%26rft.spage%253D15503%26rft.epage%253D15507%26rft.atitle%253DHigh-resolution%2Bmapping%2Band%2Bisolation%2Bof%2Ba%2Byeast%2Bartificial%2Bchromosome%2Bcontig%2Bcontaining%2Bfw2.2%253A%2BA%2Bmajor%2Bfruit%2Bweight%2Bquantitative%2Btrait%2Blocus%2Bin%2Btomato%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.26.15503%26rft_id%253Dinfo%253Apmid%252F11038534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTMvMjYvMTU1MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.2" data-doi="10.1016/j.plantsci.2006.01.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azhacuvel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidya-Saraswathi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komatsuda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-resolution linkage mapping for the non-brittle rachis locus <em>btr1</em> in cultivated x wild barley (<em>Hordeum vulgare</em>).</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D170%26rft.spage%253D1087%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2006.01.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2006.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237498700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.3" data-doi="10.1104/pp.110.158329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumann</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirley</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A customized gene expression microarray reveals that the brittle stem phenotype <em>fs2</em> of barley is attributable to a retroelement in the <em>HvCesA4</em> cellulose synthase gene.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1716</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.158329%26rft_id%253Dinfo%253Apmid%252F20530215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTUzLzQvMTcxNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.4" data-doi="10.1007/s00122-009-1235-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haefele</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamuyao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ismail</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development and application of gene-based markers for the major rice QTL Phosphorus uptake 1.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DChin%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D120%26rft.issue%253D6%26rft.spage%253D1073%26rft.epage%253D1086%26rft.atitle%253DDevelopment%2Band%2Bapplication%2Bof%2Bgene-based%2Bmarkers%2Bfor%2Bthe%2Bmajor%2Brice%2BQTL%2BPhosphorus%2Buptake%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1235-7%26rft_id%253Dinfo%253Apmid%252F20035315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1235-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20035315&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275661300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.5" data-doi="10.1186/1471-2164-10-582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rostoks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and implementation of high-throughput SNP genotyping in barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DClose%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D10%26rft.spage%253D582%26rft.epage%253D582%26rft.atitle%253DDevelopment%2Band%2Bimplementation%2Bof%2Bhigh-throughput%2BSNP%2Bgenotyping%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-582%26rft_id%253Dinfo%253Apmid%252F19961604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19961604&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.6" data-doi="10.1073/pnas.1010179107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuluaga</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">21611</span>–<span class="cit-lpage">21616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010179107%26rft_id%253Dinfo%253Apmid%252F21115826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUwLzIxNjExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.7" data-doi="10.1073/pnas.0703133104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cocuron</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lerouxel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drakakaki</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liepman</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A gene from the cellulose synthase-like C family encodes a beta-1,4 glucan synthase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8550</span>–<span class="cit-lpage">8555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0703133104%26rft_id%253Dinfo%253Apmid%252F17488821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIwLzg1NTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.8" data-doi="10.1007/s00122-010-1466-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patterns of polymorphism and linkage disequilibrium in cultivated barley.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DComadran%26rft.auinit1%253DJ.%26rft.volume%253D122%26rft.issue%253D3%26rft.spage%253D523%26rft.epage%253D531%26rft.atitle%253DPatterns%2Bof%2Bpolymorphism%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bcultivated%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1466-7%26rft_id%253Dinfo%253Apmid%252F21076812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1466-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076812&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.9" data-doi="10.1186/1471-2229-9-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deleu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esteras</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roig</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-To</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Silva</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A set of EST-SNPs for map saturation and cultivar identification in melon.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DDeleu%26rft.auinit1%253DW.%26rft.volume%253D9%26rft.spage%253D90%26rft.epage%253D90%26rft.atitle%253DA%2Bset%2Bof%2BEST-SNPs%2Bfor%2Bmap%2Bsaturation%2Band%2Bcultivar%2Bidentification%2Bin%2Bmelon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-90%26rft_id%253Dinfo%253Apmid%252F19604363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19604363&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.10" data-doi="10.1007/s00122-010-1443-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Characterization and precise mapping of a QTL increasing spike number with pleiotropic effects in wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DDeng%26rft.auinit1%253DS.%26rft.volume%253D122%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D289%26rft.atitle%253DCharacterization%2Band%2Bprecise%2Bmapping%2Bof%2Ba%2BQTL%2Bincreasing%2Bspike%2Bnumber%2Bwith%2Bpleiotropic%2Beffects%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1443-1%26rft_id%253Dinfo%253Apmid%252F20872211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1443-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20872211&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.11" data-doi="10.1104/pp.110.166249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Druka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franckowiak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundqvist</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic dissection of barley morphology and development.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.166249%26rft_id%253Dinfo%253Apmid%252F21088227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTUvMi82MTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.12" data-doi="10.1093/mp/ssp064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dwivany</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yulia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirley</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The CELLULOSE-SYNTHASE LIKE C (CSLC) family of barley includes members that are integral membrane proteins targeted to the plasma membrane.</span> <abbr class="cit-jnl-abbrev">Molecular Plant</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1025</span>–<span class="cit-lpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssp064%26rft_id%253Dinfo%253Apmid%252F19825677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssp064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19825677&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270218900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">An introgression line population of <em>Lycopersicon pennellii</em> in the cultivated tomato enables the identification and fine mapping of yield-associated QTL.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D141%26rft.spage%253D1147%26rft_id%253Dinfo%253Apmid%252F8582620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8582620&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TB62900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genomic targeting and high-resolution mapping of the domestication gene <em>Q</em> in wheat.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DFaris%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D706%26rft.epage%253D718%26rft.atitle%253DGenomic%2Btargeting%2Band%2Bhigh-resolution%2Bmapping%2Bof%2Bthe%2Bdomestication%2Bgene%2BQ%2Bin%2Bwheat.%26rft_id%253Dinfo%253Apmid%252F12175074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12175074&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.15" data-doi="10.1007/s00122-008-0808-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falke</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sušić</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackauf</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schondelmaier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Establishment of introgression libraries in hybrid rye (<em>Secale cereale</em> L.) from an Iranian primitive accession as a new tool for rye breeding and genomics.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFalke%26rft.auinit1%253DK.%2BC.%26rft.volume%253D117%26rft.issue%253D4%26rft.spage%253D641%26rft.epage%253D652%26rft.atitle%253DEstablishment%2Bof%2Bintrogression%2Blibraries%2Bin%2Bhybrid%2Brye%2B%2528Secale%2Bcereale%2BL.%2529%2Bfrom%2Ban%2BIranian%2Bprimitive%2Baccession%2Bas%2Ba%2Bnew%2Btool%2Bfor%2Brye%2Bbreeding%2Band%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0808-1%26rft_id%253Dinfo%253Apmid%252F18535814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0808-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18535814&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.16" data-doi="10.1126/science.289.5476.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaap</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">fw2.2: a quantitative trait locus key to the evolution of tomato fruit size.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFrary%26rft.auinit1%253DA.%26rft.volume%253D289%26rft.issue%253D5476%26rft.spage%253D85%26rft.epage%253D88%26rft.atitle%253Dfw2.2%253A%2BA%2BQuantitative%2BTrait%2BLocus%2BKey%2Bto%2Bthe%2BEvolution%2Bof%2BTomato%2BFruit%2BSize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5476.85%26rft_id%253Dinfo%253Apmid%252F10884229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyODkvNTQ3Ni84NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.17" data-doi="10.1126/science.1101666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fridman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernie</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Zooming in on a quantitative trait for tomato yield using interspecific introgressions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">1786</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFridman%26rft.auinit1%253DE.%26rft.volume%253D305%26rft.issue%253D5691%26rft.spage%253D1786%26rft.epage%253D1789%26rft.atitle%253DZooming%2BIn%2Bon%2Ba%2BQuantitative%2BTrait%2Bfor%2BTomato%2BYield%2BUsing%2BInterspecific%2BIntrogressions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101666%26rft_id%253Dinfo%253Apmid%252F15375271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDUvNTY5MS8xNzg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.18" data-doi="10.1126/science.1166289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uauy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Distelfeld</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blechl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A Kinase-START Gene Confers Temperature-Dependent Resistance to Wheat Stripe Rust.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">1357</span>–<span class="cit-lpage">1360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166289%26rft_id%253Dinfo%253Apmid%252F19228999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjMvNTkxOS8xMzU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.19" data-doi="10.1007/s00122-008-0906-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackauf</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Voort</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miedaner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wehling</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative mapping of DNA sequences in rye (<em>Secale cereale</em> L.) in relation to the rice genome.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHackauf%26rft.auinit1%253DB.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D371%26rft.epage%253D384%26rft.atitle%253DComparative%2Bmapping%2Bof%2BDNA%2Bsequences%2Bin%2Brye%2B%2528Secale%2Bcereale%2BL.%2529%2Bin%2Brelation%2Bto%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0906-0%26rft_id%253Dinfo%253Apmid%252F18953524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0906-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18953524&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261656800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.20" data-doi="10.1534/genetics.106.066423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentsink</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanhart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankestijn-De Vries</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Development of a near-isogenic line population of Arabidopsis thaliana and comparison of mapping power with a recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.066423%26rft_id%253Dinfo%253Apmid%252F17179089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.066423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17179089&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244689600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.21" data-doi="10.1073/pnas.0608580104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komatsuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pourkheirandish</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azhaguvel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Six-rowed barley originated from a mutation in a homeodomain-leucine zipper I-class homeobox gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1424</span>–<span class="cit-lpage">1429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608580104%26rft_id%253Dinfo%253Apmid%252F17220272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzQvMTQyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.22" data-doi="10.1007/s10142-007-0060-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kota</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">EST-derived single nucleotide polymorphism markers for assembling genetic and physical maps of the barley genome.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DKota%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D223%26rft.epage%253D233%26rft.atitle%253DEST-derived%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers%2Bfor%2Bassembling%2Bgenetic%2Band%2Bphysical%2Bmaps%2Bof%2Bthe%2Bbarley%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-007-0060-9%26rft_id%253Dinfo%253Apmid%252F17968603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-007-0060-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17968603&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257131600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The location of genetic factors affecting a quantitative character in wheat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D53%26rft.spage%253D487%26rft_id%253Dinfo%253Apmid%252F17248295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17248295&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.24" data-doi="10.1007/s00122-006-0238-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">J. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Development, utilization of introgression lines using a synthetic wheat as donor.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1360</span>–<span class="cit-lpage">1373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DS.%26rft.volume%253D112%26rft.issue%253D7%26rft.spage%253D1360%26rft.epage%253D1373%26rft.atitle%253DDevelopment%252C%2Butilization%2Bof%2Bintrogression%2Blines%2Busing%2Ba%2Bsynthetic%2Bwheat%2Bas%2Bdonor.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0238-x%26rft_id%253Dinfo%253Apmid%252F16550399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0238-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16550399&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236585600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.25" data-doi="10.1371/journal.pone.0014079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Performance of single nucleotide polymorphisms <em>vs.</em> haplotypes for genome-wide association analysis in barley.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e14079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLorenz%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De14079%26rft.epage%253De14079%26rft.atitle%253DPerformance%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bversus%2Bhaplotypes%2Bfor%2Bgenome-wide%2Bassociation%2Banalysis%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014079%26rft_id%253Dinfo%253Apmid%252F21124933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124933&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcel</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aghnoum</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niks</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dissection of the barley 2L1.0 region carrying the ‘Laevigatum’ quantitative resistance gene to leaf rust using near-isogenic lines (NIL) and subNIL.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1604</span>–<span class="cit-lpage">1615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DMarcel%26rft.auinit1%253DT.%2BC.%26rft.volume%253D20%26rft.issue%253D12%26rft.spage%253D1604%26rft.epage%253D1615%26rft.atitle%253DDissection%2Bof%2Bthe%2Bbarley%2B2L1.0%2Bregion%2Bcarrying%2Bthe%2B%2527Laevigatum%2527%2Bquantitative%2Bresistance%2Bgene%2Bto%2Bleaf%2Brust%2Busing%2Bnear-isogenic%2Blines%2B%2528NIL%2529%2Band%2BsubNIL.%26rft_id%253Dinfo%253Apmid%252F17990968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17990968&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.27" data-doi="10.1104/pp.109.142612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K. F. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taudien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simkova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suchankova</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene Content and Virtual Gene Order of Barley Chromosome 1H.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.109.142612%26rft_id%253Dinfo%253Apmid%252F19692534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTEvMi80OTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.28" data-doi="10.1007/s00122-005-0167-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTLs influencing panicle size detected in two reciprocal introgressive line (IL) populations in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMei%26rft.auinit1%253DH.%2BW.%26rft.volume%253D112%26rft.issue%253D4%26rft.spage%253D648%26rft.epage%253D656%26rft.atitle%253DQTLs%2Binfluencing%2Bpanicle%2Bsize%2Bdetected%2Bin%2Btwo%2Breciprocal%2Bintrogressive%2Bline%2B%2528IL%2529%2Bpopulations%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0167-0%26rft_id%253Dinfo%253Apmid%252F16331475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0167-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16331475&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235309000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.29" data-doi="10.1007/s00122-010-1273-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moragues</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flavell</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Effects of ascertainment bias and marker number on estimations of barley diversity from high-throughput SNP genotype data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1525</span>–<span class="cit-lpage">1534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMoragues%26rft.auinit1%253DM.%26rft.volume%253D120%26rft.issue%253D8%26rft.spage%253D1525%26rft.epage%253D1534%26rft.atitle%253DEffects%2Bof%2Bascertainment%2Bbias%2Band%2Bmarker%2Bnumber%2Bon%2Bestimations%2Bof%2Bbarley%2Bdiversity%2Bfrom%2Bhigh-throughput%2BSNP%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1273-1%26rft_id%253Dinfo%253Apmid%252F20157694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1273-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20157694&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277023900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.30" data-doi="10.1186/1471-2164-12-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muchero</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehlers</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genic SNP markers and legume synteny reveal candidate genes underlying QTL for <em>Macrophomina phaseolina</em> resistance and maturity in cowpea [Vigna unguiculata (L) Walp.]</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMuchero%26rft.auinit1%253DW.%26rft.volume%253D12%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DGenic%2BSNP%2Bmarkers%2Band%2Blegume%2Bsynteny%2Breveal%2Bcandidate%2Bgenes%2Bunderlying%2BQTL%2Bfor%2BMacrophomina%2Bphaseolina%2Bresistance%2Band%2Bmaturity%2Bin%2Bcowpea%2B%255BVigna%2Bunguiculata%2B%2528L%2529%2BWalp.%255D.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-8%26rft_id%253Dinfo%253Apmid%252F21208448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208448&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.31" data-doi="10.1105/tpc.108.063768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Arabidopsis</em> dehiscence zone <em>polygalacturonase</em> 1 (<em>Adpg</em>1), <em>Adpg</em>2, and <em>Quartet</em>2 are polygalacturonases required for cell separation during reproductive development in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.063768%26rft_id%253Dinfo%253Apmid%252F19168715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIxLzEvMjE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.32" data-doi="10.1007/s001220051527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreuzkam</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mapping new EMBL-derived barley microsatellites and their use in differentiating German barley cultivars.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D101%26rft.spage%253D652%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.33" data-doi="10.1038/ng.745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. T. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">INTERMEDIUM-C, a modifier of lateral spikelet fertility in barley, is an ortholog of the maize domestication gene TEOSINTE BRANCHED 1.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRamsay%26rft.auinit1%253DL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D169%26rft.epage%253D172%26rft.atitle%253DINTERMEDIUM-C%252C%2Ba%2Bmodifier%2Bof%2Blateral%2Bspikelet%2Bfertility%2Bin%2Bbarley%252C%2Bis%2Ban%2Bortholog%2Bof%2Bthe%2Bmaize%2Bdomestication%2Bgene%2BTEOSINTE%2BBRANCHED%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.745%26rft_id%253Dinfo%253Apmid%252F21217754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217754&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.34" data-doi="10.1007/s10142-007-0053-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Börner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fine mapping of the region on wheat chromosome 7D controlling grain weight.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DRoder%26rft.auinit1%253DM.%2BS.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D86%26rft.atitle%253DFine%2Bmapping%2Bof%2Bthe%2Bregion%2Bon%2Bwheat%2Bchromosome%2B7D%2Bcontrolling%2Bgrain%2Bweight.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-007-0053-8%26rft_id%253Dinfo%253Apmid%252F17554574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-007-0053-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554574&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.35" data-doi="10.1073/pnas.0606133103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rostoks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardle</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Recent history of artificial outcrossing facilitates whole-genome association mapping in elite inbred crop varieties.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">18656</span>–<span class="cit-lpage">18661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0606133103%26rft_id%253Dinfo%253Apmid%252F17085595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ5LzE4NjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.36" data-doi="10.1007/s00122-009-1071-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An application of high-throughput SNP genotyping for barley genome mapping and characterization of recombinant chromosome substitution lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSato%26rft.auinit1%253DK.%26rft.volume%253D119%26rft.issue%253D4%26rft.spage%253D613%26rft.epage%253D619%26rft.atitle%253DAn%2Bapplication%2Bof%2Bhigh-throughput%2BSNP%2Bgenotyping%2Bfor%2Bbarley%2Bgenome%2Bmapping%2Band%2Bcharacterization%2Bof%2Brecombinant%2Bchromosome%2Bsubstitution%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1071-9%26rft_id%253Dinfo%253Apmid%252F19488734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1071-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19488734&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268747200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.37" data-doi="10.1007/s00122-008-0847-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmalenbach</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Körber</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selecting a set of wild barley introgression lines and verification of QTL effects for resistance to powdery mildew and leaf rust.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchmalenbach%26rft.auinit1%253DI.%26rft.volume%253D117%26rft.issue%253D7%26rft.spage%253D1093%26rft.epage%253D1106%26rft.atitle%253DSelecting%2Ba%2Bset%2Bof%2Bwild%2Bbarley%2Bintrogression%2Blines%2Band%2Bverification%2Bof%2BQTL%2Beffects%2Bfor%2Bresistance%2Bto%2Bpowdery%2Bmildew%2Band%2Bleaf%2Brust.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0847-7%26rft_id%253Dinfo%253Apmid%252F18663425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0847-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18663425&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260019800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.38" data-doi="10.1007/s00122-008-0915-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmalenbach</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification and verification of QTLs for agronomic traits using wild barley introgression lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchmalenbach%26rft.auinit1%253DI.%26rft.volume%253D118%26rft.issue%253D3%26rft.spage%253D483%26rft.epage%253D497%26rft.atitle%253DIdentification%2Band%2Bverification%2Bof%2BQTLs%2Bfor%2Bagronomic%2Btraits%2Busing%2Bwild%2Bbarley%2Bintrogression%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0915-z%26rft_id%253Dinfo%253Apmid%252F18979081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0915-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18979081&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262577300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.39" data-doi="10.1007/s00122-009-0991-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmalenbach</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection and verification of malting quality QTLs using wild barley introgression lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">1411</span>–<span class="cit-lpage">1427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchmalenbach%26rft.auinit1%253DI.%26rft.volume%253D118%26rft.issue%253D8%26rft.spage%253D1411%26rft.epage%253D1427%26rft.atitle%253DDetection%2Band%2Bverification%2Bof%2Bmalting%2Bquality%2BQTLs%2Busing%2Bwild%2Bbarley%2Bintrogression%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-0991-8%26rft_id%253Dinfo%253Apmid%252F19255740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-0991-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19255740&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265959300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.40" data-doi="10.1104/pp.108.128967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The international barley sequencing consortium - At the threshold of efficient access to the barley genome.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DSchulte%26rft.auinit1%253DD.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D142%26rft.epage%253D147%26rft.atitle%253DThe%2BInternational%2BBarley%2BSequencing%2BConsortium--At%2Bthe%2BThreshold%2Bof%2BEfficient%2BAccess%2Bto%2Bthe%2BBarley%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.128967%26rft_id%253Dinfo%253Apmid%252F19126706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDkvMS8xNDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simons</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellers</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular characterization of the major wheat domestication gene <em>Q</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D547%26rft.atitle%253DGENETICS%26rft_id%253Dinfo%253Apmid%252F16172507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16172507&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235197700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.42" data-doi="10.1007/s00122-009-1043-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sood</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuraparthy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The major threshability genes, soft glume (<em>sog</em>) and tenacious glume (<em>Tg</em>), of diploid and polyploid wheat, trace their origin to independent mutations at non-orthologous loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSood%26rft.auinit1%253DS.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D351%26rft.atitle%253DThe%2Bmajor%2Bthreshability%2Bgenes%2Bsoft%2Bglume%2B%2528sog%2529%2Band%2Btenacious%2Bglume%2B%2528Tg%2529%252C%2Bof%2Bdiploid%2Band%2Bpolyploid%2Bwheat%252C%2Btrace%2Btheir%2Borigin%2Bto%2Bindependent%2Bmutations%2Bat%2Bnon-orthologous%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1043-0%26rft_id%253Dinfo%253Apmid%252F19421730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1043-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19421730&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267687900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.43" data-doi="10.1007/s00122-006-0480-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A 1,000-loci transcript map of the barley genome: new anchoring points for integrative grass genomics</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DStein%26rft.auinit1%253DN.%26rft.volume%253D114%26rft.issue%253D5%26rft.spage%253D823%26rft.epage%253D839%26rft.atitle%253DA%2B1%252C000-loci%2Btranscript%2Bmap%2Bof%2Bthe%2Bbarley%2Bgenome%253A%2Bnew%2Banchoring%2Bpoints%2Bfor%2Bintegrative%2Bgrass%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0480-2%26rft_id%253Dinfo%253Apmid%252F17219208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0480-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17219208&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244486600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.44" data-doi="10.1007/s00122-007-0512-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szalma</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hostert</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeDeaux</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">QTL mapping with near-isogenic lines in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1211</span>–<span class="cit-lpage">1228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSzalma%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D114%26rft.issue%253D7%26rft.spage%253D1211%26rft.epage%253D1228%26rft.atitle%253DQTL%2Bmapping%2Bwith%2Bnear-isogenic%2Blines%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0512-6%26rft_id%253Dinfo%253Apmid%252F17308934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0512-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17308934&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246185900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Linkage study of two complementary genes for brittle rachis in barley. Ber Ohara Inst Landw Biol</span>. <abbr class="cit-jnl-abbrev">Okayama University</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOkayama%2BUniversity%26rft.volume%253D12%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.46" data-doi="10.1126/science.1117619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beales</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunford</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The pseudo-response regulator <em>Ppd-H1</em> provides adaptation to photoperiod in barley.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">1031</span>–<span class="cit-lpage">1034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTurner%26rft.auinit1%253DA.%26rft.volume%253D310%26rft.issue%253D5750%26rft.spage%253D1031%26rft.epage%253D1034%26rft.atitle%253DThe%2BPseudo-Response%2BRegulator%2BPpd-H1%2BProvides%2BAdaptation%2Bto%2BPhotoperiod%2Bin%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1117619%26rft_id%253Dinfo%253Apmid%252F16284181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTAvNTc1MC8xMDMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.47" data-doi="10.1126/science.1133649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uauy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Distelfeld</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blechl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A NAC gene regulating senescence improves grain protein, zinc, and iron content in wheat.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">1298</span>–<span class="cit-lpage">1301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DUauy%26rft.auinit1%253DC.%26rft.volume%253D314%26rft.issue%253D5803%26rft.spage%253D1298%26rft.epage%253D1301%26rft.atitle%253DA%2BNAC%2BGene%2Bregulating%2Bsenescence%2Bimproves%2Bgrain%2Bprotein%252C%2Bzinc%252C%2Band%2Biron%2Bcontent%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1133649%26rft_id%253Dinfo%253Apmid%252F17124321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTQvNTgwMy8xMjk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.48" data-doi="10.1093/jhered/90.2.328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Berloo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">GGT: software for the display of graphical genotypes.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253Dvan%2BBerloo%26rft.auinit1%253DR.%26rft.volume%253D90%26rft.issue%253D2%26rft.spage%253D328%26rft.epage%253D329%26rft.atitle%253DComputer%2Bnote.%2BGGT%253A%2Bsoftware%2Bfor%2Bthe%2Bdisplay%2Bof%2Bgraphical%2Bgenotypes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F90.2.328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/90.2.328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000078904100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.49" data-doi="10.1186/1471-2164-10-523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiremath</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lekha</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashiwagi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balaji</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comprehensive resource of drought- and salinity- responsive ESTs for gene discovery and marker development in chickpea (<em>Cicer arietinum</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DVarshney%26rft.auinit1%253DR.%2BK.%26rft.volume%253D10%26rft.spage%253D523%26rft.epage%253D523%26rft.atitle%253DA%2Bcomprehensive%2Bresource%2Bof%2Bdrought-%2Band%2Bsalinity-%2Bresponsive%2BESTs%2Bfor%2Bgene%2Bdiscovery%2Band%2Bmarker%2Bdevelopment%2Bin%2Bchickpea%2B%2528Cicer%2Barietinum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-523%26rft_id%253Dinfo%253Apmid%252F19912666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19912666&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.50" data-doi="10.1007/s00122-004-1818-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Development of candidate introgression lines using an exotic barley accession (<em>Hordeum vulgare</em> ssp. <em>spontaneum</em>) as donor.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1736</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253Dvon%2BKorff%26rft.auinit1%253DM.%26rft.volume%253D109%26rft.issue%253D8%26rft.spage%253D1736%26rft.epage%253D1745%26rft.atitle%253DDevelopment%2Bof%2Bcandidate%2Bintrogression%2Blines%2Busing%2Ban%2Bexotic%2Bbarley%2Baccession%2B%2528%2BHordeum%2Bvulgare%2Bssp.%2Bspontaneum%2529%2Bas%2Bdonor.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1818-2%26rft_id%253Dinfo%253Apmid%252F15502912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1818-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15502912&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225054800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">X. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010a</span> <span class="cit-article-title">Fine mapping a domestication-related QTL for spike-related traits in a synthetic wheat.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">798</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D53%26rft.issue%253D10%26rft.spage%253D798%26rft.epage%253D804%26rft.atitle%253DFine%2Bmapping%2Ba%2Bdomestication-related%2BQTL%2Bfor%2Bspike-related%2Btraits%2Bin%2Ba%2Bsynthetic%2Bwheat.%26rft_id%253Dinfo%253Apmid%252F20962886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20962886&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.52" data-doi="10.1007/s00122-010-1276-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmalenbach</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010b</span> <span class="cit-article-title">Association of barley photoperiod and vernalization genes with QTLs for flowering time and agronomic traits in a BC2DH population and a set of wild barley introgression lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1559</span>–<span class="cit-lpage">1574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DG.%26rft.volume%253D120%26rft.issue%253D8%26rft.spage%253D1559%26rft.epage%253D1574%26rft.atitle%253DAssociation%2Bof%2Bbarley%2Bphotoperiod%2Band%2Bvernalization%2Bgenes%2Bwith%2BQTLs%2Bfor%2Bflowering%2Btime%2Band%2Bagronomic%2Btraits%2Bin%2Ba%2BBC2DH%2Bpopulation%2Band%2Ba%2Bset%2Bof%2Bwild%2Bbarley%2Bintrogression%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1276-y%26rft_id%253Dinfo%253Apmid%252F20155245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1276-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20155245&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.53" data-doi="10.1126/science.1094305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loukoianov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blechl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tranquilli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishna</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The wheat <em>VRN2</em> gene is a flowering repressor down-regulated by vernalization.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">1640</span>–<span class="cit-lpage">1644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYan%26rft.auinit1%253DL.%26rft.volume%253D303%26rft.issue%253D5664%26rft.spage%253D1640%26rft.epage%253D1644%26rft.atitle%253DThe%2BWheat%2BVRN2%2BGene%2BIs%2Ba%2BFlowering%2BRepressor%2BDown-Regulated%2Bby%2BVernalization%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1094305%26rft_id%253Dinfo%253Apmid%252F15016992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Zitzewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skinner</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular characterization of the duplicated meristem identity genes <em>HvAP1a</em> and <em>HvAP1b</em> in barley.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DYan%26rft.auinit1%253DL.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D905%26rft.epage%253D912%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bthe%2Bduplicated%2Bmeristem%2Bidentity%2Bgenes%2BHvAP1a%2Band%2BHvAP1b%2Bin%2Bbarley.%26rft_id%253Dinfo%253Apmid%252F16391696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16391696&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.55" data-doi="10.1073/pnas.0607142103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blechl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tranquilli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The wheat and barley vernalization gene <em>VRN3</em> is an orthologue of <em>FT</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">19581</span>–<span class="cit-lpage">19586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0607142103%26rft_id%253Dinfo%253Apmid%252F17158798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzUxLzE5NTgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-1.3.187.56" data-doi="10.1038/35103589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Improving plant breeding with exotic genetic libraries.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">983</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DZamir%26rft.auinit1%253DD.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253D983%26rft.epage%253D989%26rft.atitle%253DImproving%2Bplant%2Bbreeding%2Bwith%2Bexotic%2Bgenetic%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35103589%26rft_id%253Dinfo%253Apmid%252F11733751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35103589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11733751&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172545800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/3/187.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/3/183" title="Mutation Rate Inferred From Synonymous Substitutions in a Long-Term Evolution Experiment With Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/3/197" title="Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/3"><span>Volume 1 Issue 3, August 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/3.cover.gif?itok=eI_pz32_" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/187" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EzGU0AWLtjJxI954eecyKbVnXVdtEdD223hZABuZpbE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/3/187" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/3/187" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/3/187" class="active" data-icon-position="" data-hide-link-title="0">High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus thresh-1 Using the Illumina GoldenGate Assay</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459170" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Kp61EkcUXU83svKP9sziDsUAjmlpz6oNMhEwv26HHO4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/3/187.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/3/187" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZsCwHOa0bK9gmZ3uDPBZWylDkyeotM7DeUoOU78-pII" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/3/187" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/3/187.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459170" id="node459170--2" data-pisa="ggg;1/3/187" data-pisa-master="ggg;g3.111.000182" data-apath="/ggg/1/3/187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus <em>thresh-1</em> Using the Illumina GoldenGate Assay</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Inga Schmalenbach</span>, <span class="highwire-citation-author" data-delta="1">Timothy J. March</span>, <span class="highwire-citation-author" data-delta="2">Thomas Bringezu</span>, <span class="highwire-citation-author" data-delta="3">Robbie Waugh</span> and <span class="highwire-citation-author" data-delta="4">Klaus Pillen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">187-196;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000182 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459170--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Inga Schmalenbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Max Planck Institute for Plant Breeding Research, Department of Plant Breeding and Genetics, 50829 Cologne, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Inga%2BSchmalenbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmalenbach%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInga%2BSchmalenbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy J. March </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ.%2BMarch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=March%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BJ.%2BMarch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas Bringezu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBringezu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bringezu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BBringezu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robbie Waugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Scottish Crop Research Institute, Invergowrie, Dundee DD2 5DA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robbie%2BWaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waugh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobbie%2BWaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Klaus Pillen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klaus%2BPillen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pillen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKlaus%2BPillen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:klaus.pillen@landw.uni-halle.de" class="" data-icon-position="" data-hide-link-title="0">klaus.pillen@landw.uni-halle.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/3/187" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/3/187&amp;title=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/3/187&amp;title=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/3/187&amp;title=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/3/187&amp;text=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/3/187&amp;title=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/3/187&amp;t=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/3/187&amp;title=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/3/187&amp;title=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459170/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459170" id="node459170--3" data-pisa="ggg;1/3/187" data-pisa-master="ggg;g3.111.000182" data-apath="/ggg/1/3/187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Genotyping of Wild Barley Introgression Lines and Fine-Mapping of the Threshability Locus <em>thresh-1</em> Using the Illumina GoldenGate Assay</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Inga Schmalenbach</span>, <span class="highwire-citation-author" data-delta="1">Timothy J. March</span>, <span class="highwire-citation-author" data-delta="2">Thomas Bringezu</span>, <span class="highwire-citation-author" data-delta="3">Robbie Waugh</span> and <span class="highwire-citation-author" data-delta="4">Klaus Pillen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">187-196;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000182 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459170--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Inga Schmalenbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Max Planck Institute for Plant Breeding Research, Department of Plant Breeding and Genetics, 50829 Cologne, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Inga%2BSchmalenbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmalenbach%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInga%2BSchmalenbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy J. March </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ.%2BMarch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=March%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BJ.%2BMarch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas Bringezu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBringezu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bringezu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BBringezu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robbie Waugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Scottish Crop Research Institute, Invergowrie, Dundee DD2 5DA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robbie%2BWaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waugh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobbie%2BWaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Klaus Pillen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Martin-Luther-University Halle-Wittenberg, Institute of Agricultural and Nutritional Sciences, Chair of Plant Breeding, 06120 Halle/Saale, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klaus%2BPillen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pillen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKlaus%2BPillen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:klaus.pillen@landw.uni-halle.de" class="" data-icon-position="" data-hide-link-title="0">klaus.pillen@landw.uni-halle.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=187&amp;rft.epage=196&amp;rft.atitle=High-Resolution%20Genotyping%20of%20Wild%20Barley%20Introgression%20Lines%20and%20Fine-Mapping%20of%20the%20Threshability%20Locus%20thresh-1%20Using%20the%20Illumina%20GoldenGate%20Assay&amp;rft.volume=1&amp;rft.issue=3&amp;rft.date=2011-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Schmalenbach%2C+Inga&amp;rft.au=March%2C+Timothy+J.&amp;rft.au=Bringezu%2C+Thomas&amp;rft.au=Waugh%2C+Robbie&amp;rft.au=Pillen%2C+Klaus"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459170/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459170/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459170/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459170/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459170/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459170/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459170/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459170/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459170/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459170/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459170/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459170/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496056" id="node496056" data-pisa="ggg;g3.120.401188v1" data-pisa-master="ggg;g3.120.401188" data-apath="/ggg/early/2020/03/27/g3.120.401188.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401188" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">MYCs and PIFs Act Independently in Arabidopsis Growth Regulation</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496055" id="node496055" data-pisa="ggg;g3.120.401106v1" data-pisa-master="ggg;g3.120.401106" data-apath="/ggg/early/2020/03/27/g3.120.401106.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401106" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identifying Small Molecules That Promote Quasipalindrome-Associated Template-Switch Mutations in <em>Escherichia coli</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496053" id="node496053" data-pisa="ggg;g3.120.401160v1" data-pisa-master="ggg;g3.120.401160" data-apath="/ggg/early/2020/03/27/g3.120.401160.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401160" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/3/187" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/3/187.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/3/187.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/3/187.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/3/187.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
