<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/3/171" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/3/171.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/3/171.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/3/171.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/3/171.atom" />
<meta name="HW.pisa" content="ggg;1/3/171" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences" />
<meta name="DC.Identifier" content="10.1534/g3.111.000240" />
<meta name="DC.Date" content="2011-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2011 Catchen et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Advances in sequencing technology provide special opportunities for genotyping individuals with speed and thrift, but the lack of software to automate the calling of tens of thousands of genotypes over hundreds of individuals has hindered progress. Stacks is a software system that uses short-read sequence data to identify and genotype loci in a set of individuals either de novo or by comparison to a reference genome. From reduced representation Illumina sequence data, such as RAD-tags, Stacks can recover thousands of single nucleotide polymorphism (SNP) markers useful for the genetic analysis of crosses or populations. Stacks can generate markers for ultra-dense genetic linkage maps, facilitate the examination of population phylogeography, and help in reference genome assembly. We report here the algorithms implemented in Stacks and demonstrate their efficacy by constructing loci from simulated RAD-tags taken from the stickleback reference genome and by recapitulating and improving a genetic map of the zebrafish, Danio rerio ." />
<meta name="DC.Contributor" content="Julian M. Catchen" />
<meta name="DC.Contributor" content="Angel Amores" />
<meta name="DC.Contributor" content="Paul Hohenlohe" />
<meta name="DC.Contributor" content="William Cresko" />
<meta name="DC.Contributor" content="John H. Postlethwait" />
<meta name="article:published_time" content="2011-08-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Advances in sequencing technology provide special opportunities for genotyping individuals with speed and thrift, but the lack of software to automate the calling of tens of thousands of genotypes over hundreds of individuals has hindered progress. &lt;i&gt;Stacks&lt;/i&gt; is a software system that uses short-read sequence data to identify and genotype loci in a set of individuals either &lt;i&gt;de novo&lt;/i&gt; or by comparison to a reference genome. From reduced representation Illumina sequence data, such as RAD-tags, &lt;i&gt;Stacks&lt;/i&gt; can recover thousands of single nucleotide polymorphism (SNP) markers useful for the genetic analysis of crosses or populations. &lt;i&gt;Stacks&lt;/i&gt; can generate markers for ultra-dense genetic linkage maps, facilitate the examination of population phylogeography, and help in reference genome assembly. We report here the algorithms implemented in &lt;i&gt;Stacks&lt;/i&gt; and demonstrate their efficacy by constructing loci from simulated RAD-tags taken from the stickleback reference genome and by recapitulating and improving a genetic map of the zebrafish, &lt;i&gt;Danio rerio&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/08/01" />
<meta name="citation_mjid" content="ggg;1/3/171" />
<meta name="citation_id" content="1/3/171" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/3/171" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/3/171.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/3/171.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/3/171.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000240" />
<meta name="citation_pmid" content="22384329" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="171" />
<meta name="citation_lastpage" content="182" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Julian M. Catchen" />
<meta name="citation_author_institution" content="Center for Ecology and Evolutionary Biology" />
<meta name="citation_author" content="Angel Amores" />
<meta name="citation_author_institution" content="Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403" />
<meta name="citation_author" content="Paul Hohenlohe" />
<meta name="citation_author_institution" content="Center for Ecology and Evolutionary Biology" />
<meta name="citation_author" content="William Cresko" />
<meta name="citation_author_institution" content="Center for Ecology and Evolutionary Biology" />
<meta name="citation_author" content="John H. Postlethwait" />
<meta name="citation_author_institution" content="Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403" />
<meta name="citation_author_email" content="jpostle@uoregon.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Allendorf;citation_author=R. G. Danzmann;citation_title=Secondary Tetrasomic Segregation of MDH-B and Preferential Pairing of Homeologues in Rainbow Trout;citation_pages=1083-1092;citation_volume=145;citation_year=1997;citation_issue=4;citation_pmid=9093860" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Amores;citation_author=A. Force;citation_author=Y. L. Yan;citation_author=L. Joly;citation_author=C. Amemiya;citation_title=Zebrafish hox Clusters and Vertebrate Genome Evolution;citation_pages=1711-1714;citation_volume=282;citation_year=1998;citation_issue=5394;citation_pmid=9831563;citation_doi=10.1126/science.282.5394.1711" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Amores;citation_author=J. M. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome evolution and meiotic maps by massively parallel DNA sequencing: spotted gar, an outgroup for the teleost genome duplication.;citation_pages=799-808;citation_volume=188;citation_year=2011;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. Arias;citation_author=M. Keehan;citation_author=P. Fisher;citation_author=W. Coppieters;citation_author=R. Spelman;citation_title=A high density linkage map of the bovine genome.;citation_pages=18-18;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19393043" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Å. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_title=The C Clustering Library for cDNA microarray data;citation_author=M. J. L. de Hoon;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=P. Dehal;citation_author=J. L. Boore;citation_title=Two rounds of whole genome duplication in the ancestral vertebrate.;citation_pages=e314-e314;citation_volume=3;citation_year=2005;citation_issue=10;citation_pmid=16128622;citation_doi=10.1371/journal.pbio.0030314" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Edgar;citation_title=Local homology recognition and distance measures in linear time using compressed amino acid alphabets;citation_pages=380-385;citation_volume=32;citation_year=2004;citation_issue=1;citation_pmid=14729922;citation_doi=10.1093/nar/gkh180" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=K. J. Emerson;citation_author=C. R. Merz;citation_author=J. M. Catchen;citation_author=P. A. Hohenlohe;citation_author=W. A. Cresko;citation_title=Resolving postglacial phylogeography using high-throughput sequencing.;citation_volume=107;citation_year=2010;citation_issue=37;citation_pmid=20798348;citation_doi=10.1073/pnas.1006538107" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. D. Etter;citation_author=J. L. Preston;citation_author=S. Bassham;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Local de novo assembly of RAD paired-end contigs using short sequencing reads.;citation_pages=e18561-e18561;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21541009;citation_doi=10.1371/journal.pone.0018561" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Ewing;citation_author=P. Green;citation_title=Base-calling of automated sequencer traces using Phred. II. Error probabilities.;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9521922;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources;citation_author=P. A. Hohenlohe;citation_author=S. J. Amish;citation_author=J. M. Catchen;citation_author=F. W. Allendorf;citation_author=G. Luikart;citation_title=Next-generation RAD sequencing identifies thousands of SNPs for assessing hybridization between rainbow and westslope cutthroat trout.;citation_pages=117-122;citation_volume=11;citation_year=2011;citation_pmid=21429168" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=P. D. Etter;citation_author=N. Stiffler;citation_author=E. A. Johnson;citation_title=Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.;citation_pages=e1000862-e1000862;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20195501;citation_doi=10.1371/journal.pgen.1000862" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Jiao;citation_author=N. J. Wickett;citation_author=S. Ayyampalayam;citation_author=A. S. Chanderbali;citation_author=L. Landherr;citation_title=Ancestral polyploidy in seed plants and angiosperms.;citation_pages=97-100;citation_volume=473;citation_year=2011;citation_issue=7345;citation_pmid=21478875;citation_doi=10.1038/nature09916" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Kelley;citation_author=M. Schatz;citation_author=S. Salzberg;citation_title=Quake: quality-aware detection and correction of sequencing errors.;citation_pages=R116-R116;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=21114842;citation_doi=10.1186/gb-2010-11-4-116" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. D. Kelly;citation_author=F. Chu;citation_author=I. G. Woods;citation_author=P. Ngo-Hazelett;citation_author=T. Cardozo;citation_title=Genetic Linkage Mapping of Zebrafish Genes and ESTs;citation_pages=558-567;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779498;citation_doi=10.1101/gr.10.4.558" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. F. Koop;citation_author=K. R. von Schalburg;citation_author=J. Leong;citation_author=N. Walker;citation_author=R. Lieph;citation_title=A salmonid EST genomic study: genes, duplications, phylogeny and microarrays.;citation_pages=545-545;citation_volume=9;citation_year=2008;citation_pmid=19014685;citation_doi=10.1186/1471-2164-9-545" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_issue=16;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. F. McDaniel;citation_author=J. H. Willis;citation_author=A. J. Shaw;citation_title=A linkage map reveals a complex basis for segregation distortion in an interpopulation cross in the moss Ceratodon purpureus.;citation_pages=2489-2500;citation_volume=176;citation_year=2007;citation_pmid=17603096;citation_doi=10.1534/genetics.107.075424" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_issue=2;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. Nechiporuk;citation_author=J. E. Finney;citation_author=M. T. Keating;citation_author=S. L. Johnson;citation_title=Assessment of Polymorphism in Zebrafish Mapping Strains;citation_pages=1231-1238;citation_volume=9;citation_year=1999;citation_issue=12;citation_pmid=10613846;citation_doi=10.1101/gr.9.12.1231" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=R. B. Phillips;citation_author=A. Amores;citation_author=M. R. Morascha;citation_author=C. Wilson;citation_author=J. H. Postlethwait;citation_title=Assignment of zebrafish genetic linkage groups to chromosomes.;citation_pages=155-162;citation_volume=114;citation_year=2006;citation_issue=2;citation_pmid=16825768;citation_doi=10.1159/000093332" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Postlethwait;citation_author=S. Johnson;citation_author=C. Midson;citation_author=W. Talbot;citation_author=M. Gates;citation_title=A genetic linkage map for the zebrafish;citation_pages=699-703;citation_volume=264;citation_year=1994;citation_issue=5159;citation_pmid=8171321;citation_doi=10.1126/science.8171321" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=N. Shimoda;citation_author=E. W. Knapik;citation_author=J. Ziniti;citation_author=C. Sim;citation_author=E. Yamada;citation_title=Zebrafish genetic map with 2000 microsatellite markers.;citation_pages=219-232;citation_volume=58;citation_year=1999;citation_issue=3;citation_pmid=10373319;citation_doi=10.1006/geno.1999.5824" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. Snyder;citation_author=J. Du;citation_author=M. Gerstein;citation_title=Personal genome sequencing: current approaches and challenges;citation_pages=423-431;citation_volume=24;citation_year=2010;citation_issue=5;citation_pmid=20194435;citation_doi=10.1101/gad.1864110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Streisinger;citation_author=F. Singer;citation_author=C. Walker;citation_author=D. Knauber;citation_author=N. Dower;citation_title=Segregation analyses and gene-centromere distances in zebrafish.;citation_pages=311-319;citation_volume=112;citation_year=1986;citation_issue=2;citation_pmid=3455686" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_journal_title=TAG Theoretical and Applied Genetics;citation_author=Z. Sun;citation_author=Z. Wang;citation_author=J. Tu;citation_author=J. Zhang;citation_author=F. Yu;citation_title=An ultradense genetic recombination map for Brassica napus, consisting of 13551 SRAP markers.;citation_pages=1305-1317;citation_volume=114;citation_year=2007" />
<meta name="citation_reference" content="citation_title=JoinMap 4.0: Software for the Calculation of Genetic Linkage Maps in Experimental Populations;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. van Os;citation_author=S. Andrzejewski;citation_author=E. Bakker;citation_author=I. Barrena;citation_author=G. J. Bryan;citation_title=Construction of a 10,000-marker ultradense genetic recombination map of potato: providing a framework for accelerated gene isolation and a genomewide physical map.;citation_pages=1075-1087;citation_volume=173;citation_year=2006;citation_issue=2;citation_pmid=16582432;citation_doi=10.1534/genetics.106.055871" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. Vinga;citation_author=J. Almeida;citation_title=Alignment-free sequence comparison--a review;citation_pages=513-523;citation_volume=19;citation_year=2003;citation_issue=4;citation_pmid=12611807;citation_doi=10.1093/bioinformatics/btg005" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=I. G. Woods;citation_author=P. D. Kelly;citation_author=F. Chu;citation_author=P. Ngo-Hazelett;citation_author=Y.-L. Yan;citation_title=A Comparative Map of the Zebrafish Genome;citation_pages=1903-1914;citation_volume=10;citation_year=2000;citation_issue=12;citation_pmid=11116086;citation_doi=10.1101/gr.10.12.1903" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. G. Woods;citation_author=C. Wilson;citation_author=B. Friedlander;citation_author=P. Chang;citation_author=D. K. Reyes;citation_title=The zebrafish gene map defines ancestral vertebrate chromosomes.;citation_volume=15;citation_year=2005;citation_issue=9;citation_pmid=16109975;citation_doi=10.1101/gr.4134305" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_issue=5;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
<meta name="twitter:description" content="Advances in sequencing technology provide special opportunities for genotyping individuals with speed and thrift, but the lack of software to automate the calling of tens of thousands of genotypes over hundreds of individuals has hindered progress. Stacks is a software system that uses short-read sequence data to identify and genotype loci in a set of individuals either de novo or by comparison to a reference genome. From reduced representation Illumina sequence data, such as RAD-tags, Stacks can recover thousands of single nucleotide polymorphism (SNP) markers useful for the genetic analysis of crosses or populations. Stacks can generate markers for ultra-dense genetic linkage maps, facilitate the examination of population phylogeography, and help in reference genome assembly. We report here the algorithms implemented in Stacks and demonstrate their efficacy by constructing loci from simulated RAD-tags taken from the stickleback reference genome and by recapitulating and improving a genetic map of the zebrafish, Danio rerio ." />
<meta name="og-title" property="og:title" content="Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/3/171" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Advances in sequencing technology provide special opportunities for genotyping individuals with speed and thrift, but the lack of software to automate the calling of tens of thousands of genotypes over hundreds of individuals has hindered progress. Stacks is a software system that uses short-read sequence data to identify and genotype loci in a set of individuals either de novo or by comparison to a reference genome. From reduced representation Illumina sequence data, such as RAD-tags, Stacks can recover thousands of single nucleotide polymorphism (SNP) markers useful for the genetic analysis of crosses or populations. Stacks can generate markers for ultra-dense genetic linkage maps, facilitate the examination of population phylogeography, and help in reference genome assembly. We report here the algorithms implemented in Stacks and demonstrate their efficacy by constructing loci from simulated RAD-tags taken from the stickleback reference genome and by recapitulating and improving a genetic map of the zebrafish, Danio rerio ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/3.cover.gif" />
    <title>Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ezFJQveWz2_u8CnXLRIE62Z29OVTeJLjY06UdCSDQjE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459169","apath":"\/ggg\/1\/3\/171.atom","pisa":"ggg;1\/3\/171","ac":{"\/ggg\/1\/3\/171.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/3\/171.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/3\/171"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/3\/171"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/3\/171.full":true,"\/content\/1\/3\/171":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rvBbmQVIQQB5Ht6PMCQJvi2ZMSbEF76rmLtkcllyBVUqr2SPxF8Kqq3sJRkh6SDuxWxbiJM+ksq2cfYjJayQKSe0R+Ei6VbEIeaZ2oXBA4Jz2hnyt9VCJopNOcv86jH5VNcVr3fwGUUPsPMTJ2rQlCRgpvK0UkDyg4OGb7eRsKen4UgCk7tdPxTrekC3t+oGgo+kiro+tSZO7JUUFbnetYJPNVVlc7QlNOnEBePYBu04JOFicLQxgZeOXCl+5XzcmKqHKq+q3wgSD8W4RHRCQds0tXdU8iJWrrPWE+UQFW2M5ptu63K08x8xQtk248aAnGVXlIeE8Yl0FDt93xO8zdZGqPUct8cri+Dd2IWQZC2fNKcNaALCEVJfWv2n84pZ8vRa0ubwczRZe60Pg0jZ3eLe8nPyoZ7ipMG3Qa14VB70b3yS6Ay2Vl3sD474M0JMPpnmK2LyRfWzwJZLDWk0V1zeza2MG+QoHbtKNceDcVEP31fe6I9o2HtaxWDZhXmMmOxEbUXYMmmXURNVWrL3E+STTB2jGIyrxqh5+O\\\\\\\/Vy3eviENOZ5QHTIBX9igwYuS8xXp5FT\\\\\\\/VVixhBBNfPBsFQzdkYLQ\\\\\\\/kiGL\\\\\\\/SV3QLL7P2NZF1t+6hLqHJLFclikfYklzQGiNM+CTrWrKtinI+Om05RRA3W5j2UXBouaqaRK7Y3eVX2yrWQ8TVBMoKrRK5fGy7gMNr+NonrN5bVBHFBLrvQ\\\\\\\/4Hiiqq0YdmryAHZ8YoeUE9xoMmP+Enef1WGkXlhHSKmJ1m\\\\\\\/\\\\\\\/dymjlAymZi\\\\\\\/ozKauc9vPwjzboxozEg30r\\\\\\\/QMdDetmr3LHCQrxJJjx9p4nLhY52GUW26V3KfUYaEzScJV3zXFNgRWA1rngHG3t4uz37rPKYVFOt\\\\\\\/avOk1Wnk5lGdUXQi5w\\\\\\\/BScXJ16uUkqjXJEYCFdmqtSSBare0eOEsiZ+25l0ar5AEyz2FavmRnC7A0OARS0Cew6jqTpaC52VfKe6XurMf0OF3\\\\\\\/1QPOUimSw02VWR+guigKzQc7n0I7aFyVeSJb7WiMI7yUvRvNOu7eyb4Lig0YeTAj5cGPx4uYg0ZSIUY2Pvgix1ENkk53aVZEQ+KS1m+hoYXQPEyQWSlU3iWGjqiyqQ77OijTgzyEtgXKC\\\\\\\/yJ84rD01aSWcridCoTj+m5icik+fyzBmaOD2g44CnNylLj7f0TeH0u9THkElaJxPtwX5p4A94J+Hk4B8Rra82ruINkqgUpyUQ58FWHqDflWnYR98B2RzviF9DTkNoAZlugSYEslpWdOW08\\\u0022,\\\u0022iv\\\u0022:\\\u0022XjPEn6gS8AuhLvQAU31DTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220bf6148eec033ba55f43e6522f4578e9c42e7c7a6d7ee419d4b439b9a204fa15\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022y50Mtk\\\\\\\/7C5h\\\\\\\/lM3lZfgP448bPO195UcwWiO2NYjyaYUMDdngawV9ySUSe0efxx6ZkvIelYUizQuyE1LnO+JynBKuQZzgHpUfSEZPotPGbH1k4C5ovt5\\\\\\\/oGVPGzDg9n9BebacXpjJEYHD0p1I11aPEhkHf6fpGvt6vOXdncSak5h7EUcOltXMfSlyju7r+QvFhJltxX\\\\\\\/qYkrefOtsEcWkryaZQeKHi+zZgghYvEdwiT21s2YD5AdZWVLgx+SEl2dyvxkmgTgltda4r2SWaL\\\\\\\/ryuE0QAHSA6xAnTBa4erFSdOQom+pzKVGBHrKdh3nwJSHc7bDsXhIrIxaK80kJzX4MyfZkGbVUYjH1UW6QRdLtpVBDVQQcjRF2IHObHFenKr6sxkTb5xelyGrtlhKtFGVXdTnlqIEq4\\\\\\\/1i5hLfBkSYKAuny+GloII59l3untW3Q7mCtM8Efeyw5c5n+i69JKrvywA5tQbZKyHxTqyhmptJBckSLxkFU3ZlyDmK3lnkGXlGIFGwi+p+zEPfpW\\\\\\\/L9i81aB+GOO40YCkBYwqkDF+Z6c2ze6tXe9snXN286NSBEHQ\\\\\\\/fWY8vVm3\\\\\\\/q2PWqrFrwgwyYnUZ15DWp0JFmMeEx+qkktK8AN6vLV8l4i+r6PgYoPbwT8fNUyonrmCSeg2gdP8bf+1WqG9gj5sUkf0+BaqDqvni2GOsvfGKvsRQR7c6IbDg8u\\\\\\\/dR8d4vTnBXa8A10hzkTKh2f8MwHwYUhIgPCV3TvINLYO1kLRmOAStqSTjneWd5itLaOBvm0f1Q6ayrDPmI2WUi0VAWEPmNk2GvaZXm8R23hmpd5wDJPCU\\\\\\\/X5iEBJjNad0oozU4ibBdyIG486CKSTqQZCa8PVEtw0W8XjrC9SGmT0fhSJU33VK7sYIhNWOGkQo+LBk9jRuPZR2oGAiRzaiVptSVjz4i8R9KibK0yf6brUxjY99rrH8S7E+Ko83KZ240QQKMnpQxuQnPO+pWHibbEalrm94kPssxE0l6raHYm5znwk7K\\\\\\\/Yo6fC+jqKjdC9421HjKabC1M4dnkVEYfpD8WGpaLe7tIZVmCPjcv\\\\\\\/4t6ZiTtXBF929GjBbECSlJhbzn6Lfj\\\\\\\/wSEJDwUvFhue0N+neQ6NgPwnlrjs2Z4VGSSvGUQ6rkKrsiP3TA\\\\\\\/HVOZRi6HS9Df93OXtmGgQ8VHEOxVGwYOGE4J4Z7yn571R3tFpNIXMi+uft02wk\\\\\\\/9YKcQvX0lu2TSOmAW74+hBJkmh8qm7ZFwYUibbxIBpiQMhnbdMTjkGG\\\\\\\/BbxHzyI3W1VuNboRZ\\\\\\\/StDQysCfBB4hPXLTg1nxwsnmAm8DKHxaqBCcXOPIM51\\\\\\\/MSowep2BcH83NixdjeAKV+EGlwTKU2gM+FSNtuJaS4ra4SlhAN3R6\\\\\\\/FKl2+Fu5KrAioZEolkB9ISlJ818jZMzHo0BkbdHtqaDbvOlwT9wKYMhqOzSNczw4uxyvdZ8WceDYgU+J\\\\\\\/Y8Jk4i61lud+m+jJwpEZCC0brwhWn93j1m3kYEpwWNLPoZ8MtKIcagmMI0Ve6a+UJQQwLnFVpmLInqlnZbClhqRYu+RZTALw83ul6rPZ5wPZgv3ca\\\\\\\/qwE9BhmEaXiDh6w0ZoHLp7rx6i3qfgrPp98D5OpxQUG5PkjyYWe93t1uZs9QJjT4qyjG0StoNQrkOIMBmnw1cq01ETvUVkuqME3VqoACwH1RMLfgpIpRrre1L5XJqAzPgwMsDDEujEO+MLSKUtIggjMv6J+8zVC\\\\\\\/YhnE1ZFZV7WMSa7vR7MaO6hsfKSqy\\\\\\\/gNC1Q1cqznq51lYhI3AHGaJ4DdlYnEKlwk\\\\\\\/iFnoyXP6gyVnyacxoEyNMBmgDZ7F95tYvY4XOEmHD+ODbM8v5HylOXQF9GZoL6ChKF27BtyQhlZGl1YYB4xzdGsUQ1Nzs1iW8tDYDZzRhifEdKHD5pi5M532Lgt9C3d1oF0iT43h9zgeNb0fD5iuhhsf75OQ+2GRVs4dq1iAIQQMl\\\\\\\/WsSnAYABVGrHJ\\\\\\\/ib+GuWlGQ7YOkAQcMl5FUZVwFP94j5E0aAte5znJ4nLB2pgKLdvpu7hM7uPtnPElxVEnDDymPUBfoXXxDTWRLNGyLUVINCNkpCjKOgDJiqMg1KYDxRzZXLt3jlHYFXOhb2LliFE1GHHmGnGJCiEbWLg2oRpJBriugh8ADZyDzzqGxg+06D5V5dXW+Va4YnytzZNmz65SVIjG9BJYlHFp4PQW1nrDBi9FHNtO1S+lVU51tXXbRdEPGf1aMreWXh3g5Wi5RssRy8nsYIqAJJKZ+U8kOr4wEGoYmB0nVUG1UjJDNX4MtvyMgwrgC8S\\\\\\\/k4qlD2OCquQX4qQjte9XDgVaVLcXnVXZLL9yOb\\\\\\\/f+8bvaF+aJazMBVlIQkKk5t7o3sIENVT2gh3U0DP8SUaPfiVn3Z49AYUKDgUOL9WAK6iasBiokgwH+Dh+VznzfFgrSLMMawsX8Ndu7gGZah+AOXEbqCmacmn5bbM8ujuoL3pfycj704UerjRB9Cx+Csbp1NfqqEpxMbbBelF0G9gYVwTwW9KWor7QISCvtPDy4y4FG+h9382Yp5vTVcSKA6OaRO+xW4OxTTeuUhx6KQaAJbjrt7WtmjG\\\\\\\/yczhNCpoZ336qUhQ0cm5e5bOdkgKydZlp6L\\\\\\\/CzU65AvPWQZ0i64bZ0rN0\\\\\\\/IirVe8GAHjrzUnsKDUbQmIIUTarr0JAQq7JQUm8qFUEvC\\\\\\\/reXip+TzIle2gL3+p\\\\\\\/j0vTzj1im4+0iRSgo5OOz6jFZbW5pCFuqQqtzZNJ5fbPDtnyYxKxp+zO6eV37heWprouXcAyZhcTtBcbkZGNfG0R6njepv5MfGlKguaMN785TDJmTZ9fytX8XnTmJPbu2VM2Bf4W3nNgrijnblmI8hoGh+IjWZ\\\\\\\/dxPWRI48a6IgsYgUsDsLzKl8Kgw5LfOnDXMgbkp6GuEAveh8JWrWwzGXe95EQPR7ITF5mmdCYcITnNXaWPkwPjXwwQBZVpj84pSq\\\\\\\/qKTMpLbTbtTFw4JNnrlBlQayRSIE1x9fiuXwwKNOgVXQUrWYSSCbInj70aR6agg\\\\\\\/uRiWMWQaUXL1KNHmZVRsyQ41jKeIxwT1fKoUBPKzNkGOU4fk45U++bRZ0tEmxad90lrOT7j8dcX4W950NxXPI5l+Y7cJK498WMtV\\\\\\\/ROorhI4CliKgm9c\\\\\\\/feseM11wiz+kblSWEtD9WdCOQFVbcpwTW5ot9qQ8acFidMEB3MBNjIOQmVdEeTVw0VxT35fQTHpG80iVCPsRgNR70k+MpihMGcvtWgnj7H\\\\\\\/PxZ03kaDVY3nO1ECAAqq5MdLapZdTNfRqrZoKOTWPVU34cDgDaZGabmnxpKFFvBHZpMgNrV1Gk0iYnmK8G99sh6WWoXwn7OqnRrgDrdN7IlMl\\\\\\\/4AqUof8pmBX\\\\\\\/z8hMk4h\\\\\\\/EBtBYW1yMVOQt8Jm7szgN5EtEY2n6iSa3kjCuhw6BpkB+XpKbZofBkcfDOwMM9J2Abtunmyzsl+00p956bzCsk6PR1Ama+50OIhboNKOKx0k\\\\\\\/KYx0b1YjYfnI9lAtmd2aKpY2SQYo+pF9AyzSEMQLD1\\\\\\\/OC8zhlFgUX0rQ5O17Cm3bTEtJYO2zCwqG5RqNVuEH+0wV1r9Y1SXa4ZNRyzlPgs20RP2KmfOaV2RU2oo4U7Ju7uMhA7imaoE8I4J8s6zzaf1A9zPHMH1w+v5BcQOF9p9b4FL7G7BjztO1dMzeK9adBPDJ4tZ5LJFMqVfn+E11CTQa9meB1dzbDINUV6svWBnB0jW+O5XEkGDkmcXpCI+bwpxzT0GgBaJfGk19NI1yN91t54UdTcQ5Oz0yGhkKBnfp5vGvbFvliSE\\\\\\\/WkhhwWdLcUp0JQjEDdIfz5eU1wNCw0fXpJPRWmgasWQbbPjCMbyuz38tyzz3S4zcf8FJxc9p3DnAn\\\\\\\/7V3rlGPK9eSBQlFUU\\\\\\\/8QQEauBP5jOL5Zf0B6x9S2oUKQG8080y4q\\\\\\\/UTphw4jiMiuCiKvEyLCWSLlnEoUvd9KZoHoU0YzuerGBDxy39vqcSgRjgNr9n4b5mn4XPwkoA3+DPxJyt113I4Jl+zGLE323Ovd1BQb2jMXIuwLo\\\\\\\/ZkSbbM96+2r\\\\\\\/VpGaVd8W28JbcZ63i2lZEdXGLetWFCp2dclDJXZXv9fbn99dzCpGwAX1Zxkm5jGxxlQAOV+HXjsEeddVJYiWnoQpCy8PvNYC+QcgYD\\\\\\\/1T+yY4xz+6egOOCUCXarXfbvArMP+uUpAJLTZj\\\\\\\/z6zsgKvSYcswXWav7I6bkQPoYRRW4yqfl2d+9ad0fqZrbMue5\\\\\\\/5FEEQSJ+3XAQwJk22dGgqOEyPoHSUS\\\\\\\/AWl653avbXPBheBJt5Jv9bTG\\\\\\\/vYS+OLcuck6Vm8NI3NsDk3tguFAAqCI4QAexEVaBJ6SfK7l8kcquk75EJgjIOaufMwe5H+V4eNcSNNmXZ+RSV5QwyYz7lY8x7mha6SJFRP8tOm8JWsjFdH3LVPumGV4CfXLnIZGsmkHMxKuWdhQVmmVSCrb1pPclrmTS8m\\\\\\\/PvB0tRtutZbA0dx\\\\\\\/rq1EuJjtGSUC2NWZoH4H661EVodCHEwJ7QzCkFHBK6g+TCc7TORQP10d2WjWn2xkQb12TNKMdfKWeu4lSyeLrXXnFKTcQSzYiiskja\\\\\\\/MUnfavkcy+ffMLHe8STaClyZKUqnCay\\\\\\\/jQIpV1jJ3dvjcMOtABsGAQPdnBCFrdvrWauBQJWSxF2jmEgxPo3FzNs89oKdBT+H\\\\\\\/ntLkB\\\\\\\/czG9JhdCslzSuDMZLTyosuoANJ2Db7qDNoUXpumC5dwj+Wf2h5NemxpdZijRsLaD93x31vj7dXc\\\\\\\/EXIfTeM6DXloE179dK2Noh+yZFyKLi9lWUAxVtmw3afIW83W4z\\\\\\\/S9ICO91HpiTjQm\\\\\\\/YuDJfkUPfPHNO1\\\\\\\/RbWBuMBrWBDJTYkQFk0jG7Oax81wKIGthbDOBNPM2Wm1mi\\\\\\\/Z2jmgs\\\\\\\/DvzZfpCi3kkPwHh5\\\\\\\/RSpNKd7JJC5JXitYOvuL+wtwe8sTCsM2i7N6lR3rPenRyr6dD1LBgRkSjDFhVw9RIcOt6pI1WFORF4KRDGS5yJrOOw8ylLAZJe2w\\\\\\\/\\\\\\\/nnbM2Bucbk837tWPuGnhopYjWnJFm1Fg4aX8VUioyIg9AW83KtShIX0r5Z9z3t\\\\\\\/Wr8kU+mrklgk3MyGZ7+AZXXbW0O21DER9w7EMYrYW1Q\\\\\\\/qW68Le4UPAmWLbLi1FsRraCgSscyoVDoHCpJskCtGBtTmb0b924QlhlvHDjB3iK67n2aNp6PJysKFA6THKRK2vHZJDT4ooqneuISbGY2TrRm6RkHYJsUHxSzrpbOd5Oze4t7GZxzbzQ8qWU\\\\\\\/kelIlzKo6D8Muwvq8nxFcc0DMJnWHdgtZF4Y\\\\\\\/BBs0WPQR1HBGDVb8UjpfbmuGUi2U5nzj5KslRemaZjyAiDFly1\\\\\\\/5GxnvXOuDPTrb0goE3ZBzMZbvCtL9mGMqkvRNpPfJUsFPqCLmlW4CDvs5HDqAro1owwQ6XbNdA2FsM9h0vzPKpT5pRdXSNfcPGxvwh2hmd\\\\\\\/gClZLI1v0D3tqF3L7QRRJY6dOmXRwAaiEUZTQkQC6u+azpoAEblZrwBaVHdfYbSa0h6WaUbASWD1AMtri5Z68+RgP\\\\\\\/Ei6WWsXUsc6YNJqgUiNvSBYhHb9hyfBYzN4NoWZifYBdB+6IBelz6w41TGo4b157rtebgUQiBlbBYkUt5uiziYMjgItD2IeDXV5sAWDq\\\\\\\/2Xn\\\\\\\/8sL39ocNS7ui9ItfJUuApjVhituFhZVn1S7+D5Hwe+lnZ9JhrtSxY53L\\\\\\\/UhTyeReA3knOBqvggwSccUkSP72ipTxJ7d97fEku8qn+YtZZXeOENMmaRBpfQRk1T0llfeQaHPx3lLOdGn7pkhfxdtMZVXqSpiYrBZn6qfpBJJu\\\\\\\/GczdktdAoa+1kUGmHMl26BxYR\\\\\\\/Cekq6j\\\\\\\/q0Ur4pxWMtnDGfEuKME0RzpDTznm7tEKWujapf+J6j3\\\\\\\/DkSSBxYQPhl6gK+cL4OmTSrgP8RpDPElAW+Ri\\\\\\\/E\\\\\\\/9Hapkj\\\\\\\/7J5+VtSqqaHJ7E+6+eXrlKrDPp1gCzCeYjX563l9wSzmD+W+qHkUyUQgFLVIVQ9yeYTD1krVTscFAL6X6vBIkLtmg6njK6L6hNLseAQVHVf3xUj3KejjBnYLOFvYdjD9CMMCvfw4HpzHU85ogXiQYne5mh\\\\\\\/sCzxpkZHp1TeGO9i2nZElYT9i4zJ7FU3qVNy5Fsrm6uhO4OF53fTXIG+LBg2D5z9WWUa2piLb6ee6Xcp+MkAw4XQp8+9qMfojQirjm7shEQjyEfOF7Es604439W3Kq8L\\\\\\\/r52rtXCfLNUqmxh\\\\\\\/RCg0aMFN+cuaZfLx2+Tn+LCvQ22ORHNfKLE88nsG+76tundGBTD0NvoqRucxDjnPv7R9yNNbqu0vvEiXZMbQKdytyDfuyl9NSgHbYi0C0\\\\\\\/FMynf24W3TdqW5NX4LiEkteyrVptaQQ1ITVWmi6xUSPBaQ0OPm27BMtS5TtUQEz2uMXhlxEtImg4KcH34Zw8O7E9BwQ8F\\\\\\\/74xkFnohqdZi6Lfqozb79o0NKID0zYcXXzTSe8GAmYIJ+pmdsq2qv27NzLCsunZPiNSgZQYvw2doiykU0ctuW4J8OekS1fF6tpL5OYOMOSBQvGrJ8N3U32UcTgn8BUk8B7btIodz9M2CKvV8SeW2nudP9gs0tSEQSghGC4mRhpc1R\\\\\\\/1VCeWZN\\\\\\\/HYxJ2WvrKsc4PLlQaTFUwLqw5WgXT+33Zx3ENPKhgrPdSr4WkUk+At6IZXbX1BBNOKabWnXJBPNMpAwTXpifQbQfTorGgWRZ+SbgwrsNgBebmdpOPFFgW8PtqTDwMZwIt\\\\\\\/NNrgjPYySKuLVqqwmy1gOObBa\\\\\\\/8Hh34R3d8nm0WzxzF\\\\\\\/iBa93X6L0b6AYRbWJqkyV4VHbzGjaJcaz5euYC8UN20m2tr8nyGEjSpzNPhyOObmVmmfeWHXRPPbC964HKJMIF5ToBxawkW9O4K2IqmZxUMuuKCsnH0xOlxItmSWSFwXLGiupZeubrqQ5N04ORh1Ag1oBe+zIxY4VwO+sVy2m\\\\\\\/BTj+eTkoedZbQ1kLvxPPA2P8Dv+Gn7jK0VWGBOWKvuiJbkeAcFs\\\\\\\/A6ZENBKfZQYj2kwD1fqEoR\\\\\\\/VxBzNwVOovvS8UalhTIA\\\\\\\/Es8HynPDIQIz+Ap1bLutzXXtCHWoFh4j9JIq7oqDsMUaWGqL8sqs\\\\\\\/UPRgfRc0SFGcObFUGn0mhweLG\\\\\\\/3H5CGLpvnuj8Eqqnfl6A9tAQr6iOpbXRrwFuPSXSydqz\\\\\\\/QrlqpMFBXzLrcrUThtXF\\\\\\\/4qodJga4uipbdyzrOkXRA8LUtfxvj+5IdZhcrc4TKoEeiGcN1WOIY71J+eZD+tfVHwl3H4IuRsLtn3VojVtEaKzZHmJ6UjbPnGDG81Z63kCV+gdctz6GAfKpUoTGgyL3Gt3kbgHKdQGeMatj65wZofedFg7RR+4ta8BwGSQ+bDv2I8bszKdNdQQk6BYP1QRyiKBgyDcTAvV\\\\\\\/JL99iBGSGb1JMgcJNBggtr5+vB7oG8zNJtNsiNBe8IObz3rd8AgYQw31Oz7VbU8cPXeGY0YN7dbuT10m7mhB48Q4lHxbLFThcuJ1wV5QTF3oiqhz4bhB2rt5aSjAJPdP2amtq4cxYsFiCRAydaNGnMaePdWzAiOSoyVZxPRbUZ1Nrs5t1uSmQJ4dq2DEnTW+gWE1C1VedewUykIrdwVp7M0kZ\\\\\\\/dlM1ifXaqFvEy7Yoz60oAGwuWcPwV7nWYjJtG5M0zXRCT6I3uU27V9mWsMNJWGs7T8hRhqgkwh\\\\\\\/PP0UJ6Fym5KJBXS9uX\\\\\\\/DneirNgg7rF4cEQACM7a0WQHeepc9lDinA2UVIBGonwbaN84M4bMKzoZYS8TzWozUTYHiOxI6k4BRdau\\\\\\\/fiTvOKd+rjf09pHATgC51MNtmVhQX2WWyV3g+Fvq0AExwOQr5m6wxgllJwNn4wJfwg407Z3+JQ2Wi8Bpd3AFNRJ9gcVtbAcnz1+YwQMwQLNncCv\\\\\\\/WAPUPMZYrNi1Jl7wmqkuTxDGBEn6yqQEuyYxz8hXpb4s4q8nzZ3c\\\\\\\/+fnq06T3hn26QpUQINkwiktJSR4jIC6wjNhWqEPnboR683rlfZJMYC+Z+gBbya0wH1pH\\\\\\\/DJoVDYcyYlL67Vdybf0q3XqJNq7GGLQwC1aK+OJRo6fAyoxZrNPBjiUfAPAS6MwOWrOrDtLMB4vE8pjvz3SmGnp+1\\\\\\\/lw8g2MSSeyAjhg7H8FHkJY3BS5QqmsiIg8M3SUQ7tZx1wVA7EveuU2510IWfmoHYvY2qOl59iD8oV4NIyQzHUxRWjAj+1OmaQnfUFZezW+jhATEPfxstdtWWccaHErbrwGdjJOV2UpWTV7xY4ucZbv4qzCy3iTww3manusQA1LpvfIRdpmxTfI\\\\\\\/upb7KBXJh3S\\\\\\\/0TOtbWJqCkXipOF11OMBdEdu\\\\\\\/3Fdgro7i2NYRjaMrt7ZwXjH1ovgaytB70HjEh4cjm22RBVTdq\\\\\\\/31LxR2ZQrR5TDzNJUqJ3V9rFAt9Ly57ohPZhy52DM87qsAhxMQ6ZjjFZGCkTpha1z6iyFgaf1zymCVxmWHIqV57I4cgU38El7eRzXCUSIOkdBE7kJ4ZUhClBW2cm6fy4z3jMtxV4hvnER0F6tNd2aec2zxMdMDwjPSuyWHlLkiZfH9IAqruA7I0jCHRGyREPel0vZdgEooJ\\\\\\\/DdM21izGtYChHzH0xkV4NGrBYmDJrzsE\\\\\\\/sxG\\\\\\\/Egr6BOTWPfgYjvFLaR4+n1Ubrmr9iA2GohXlxVV++3Xcc4ezLa+uuGL90S3+typtagnpG\\\\\\\/NMe0lN\\\\\\\/1W5\\\\\\\/LE6dQzSuVTVIwLydkNQhDieaEe0+WgSEXhvpWZmefAxAz3cqVW7RUHaM0Tbv8hDxbwjQReof8LS8N7jcuQA5T1wvuz36F+1NM6lRRnSyqqay+FLrxG34yQZNfNgQGSEiw\\\\\\\/f79858EeCv6rpl1pbeHkOuvSm1BKgBZi+A7gLWmEcrsWIX\\\\\\\/k6FH29gJCHqBaIozZIvI3vRL8O+0lN6otMZcohtC\\\\\\\/NgVj9PnIy7anhkY5R6rAgOBDJ8k9nu5FgWgYGI4dE0ptDWPhm4bJ6J\\\\\\\/3L2YjaGdNA6is+GCQYmlu3vWZakrydQxmCvtgkgrToPDrTI3CMcNi\\\\\\\/\\\\\\\/tr+tpowG00oYOuR7qt9dtg7XvPlma+rg1+1EHhLrTck\\\\\\\/oYMOTW2QWi9fwjR3s+l3l8VcW0NCQhdtpBHpdxqrMwUQLRS89juNtfpaDwXP8MeLJFeovnZRJHCeG9Sc9micgYZrRIsi4ZeMNIntsV4el2YOiXuIEim+b+iHSXAEQAxCz8ZO9oD86ScH+6GQDN2VwDoyzIj\\\\\\\/YeZVlrUVBZWVNTfVAH9MA3dHtdCZmTR96bNzc7WdRKQpIhDK2AzCXuWVmG\\\\\\\/Z3w2APlVrRMxsWyBpemV3mln7Hi1mEYDZrTM0B7nFGciN7xzCM3xCjaz9UXGx4ToDNEJxweSh5lwm3iHp\\\\\\\/n\\\\\\\/xWptCVkCF3tNHPmZS1dFSSEm8cRWXmLmr2kmiwdZ6Z81FngEuCHCFavTCOSjLKVnLJrumg3fJXj8x109LfWezpQ3FCQxxnJoRxqCDwh5QMbG63s3jYCVtPkOrKP7s9KwzdMRCfaLO8ruEap7OQF5bzOvLJVnl4I+OG3P+ZLMG5VbR0wnY3vR7cNDAvIm0IpC4YORraQUmKOFmnbgqlDzDcmk3MwjPuJnHX9OgrxMQO55ssuUBK1QtRNUTDmLZAVdP\\\\\\\/aWFId79K78SSiuT6fbfrCYkINbRCsndbtHwAODWswMnbLgo5ZQIecVEs7GzTLUfRfXPDeBy6r051bKp71WSzc+\\\\\\\/tZTE5E8yekh7p2oAiW9wf3K8gx\\\\\\\/0yMIk4pwI1f1xLj3PsBxVy1ZhPSUhYqTcmIijeCJvSnN3SgH2Rl7wjPMTK6Ba\\\\\\\/GmYxMGgDOrJ0UR6PWQBhKbxeG3xrfEji+j6YvWjz5IEGGLPIze\\\\\\\/r30rDyeEqTHg+M3VSf9s6QbsFNp8E6BziAFFiitMWx\\\\\\\/O85hE3kJ7rGES12xnwJ7s3Xl5\\\\\\\/1A6wNzsAnzh6rtt1Bml16Y12sTbXqkEOi3nU8rmpVDwfOKlq5zYvXcrjlgj8JT5VeFaowmhLOwH6AfE3XkbuOse99cBjIqJZpzVBgil9Jodo0Y6iPi7hMCPFgIIE3uD6PIsfVNz2sU2qWJXh\\\\\\\/+KKloBkAV0\\\\\\\/88FScfYbRELOohLKvDAliLqJBlJZZhZiiI7O+RrPGV2iJtRLXwkHPk24dZOFb1zbmaHK1mPwcbHFemM\\\\\\\/Yo66WYdS6d4rZ60+V2LJdtWf4VwropF5xm+6xmP8gBrsU\\\\\\\/ar3zjvTV+A7rOv7ZvYqhSZcKnBzXpA4KD94IWckOOF7SyNjvwTYFuSIeUBfSaHVu4a7ydlmpMcnAolSXZ2pqf1Yokdc69z9V+djOUnH8pmiURTaLBm8zJe6eJKIpDPTe1schw3iIwlSzPGW9NWgV4cu\\\\\\\/ACe2\\\\\\\/\\\\\\\/ydhkkH+y6gSKI3WVPOcseisOU+vFCQ7EF4mqQ6HRhbLyKXXaOOkVENa8r0yPDJ+wlfNm\\\\\\\/\\\\\\\/vzmtVIBGZuwv16YkJ1WGA1FuHQtIkc4+v3nZy44MdCe0f1UvqI7EUiXe4I49E9S7fJfqXgsEzQBEjh8xghPNkwybFLXfnmRL0jpd7VzrbWmaDj9xodVQRMlOi8VCN4FWkGIVUnnEpSxRY9QhZc5XkzTIpbeDfHN3XK\\\\\\\/ehCWvoNPdXjo\\\\\\\/onq1QvoCwmrT4gqntfetYwmXnbTszGsLE+FmKhAOnd7aMamnDOGHwyiNyY1SM1GibGZpr4aQwjlX8KemVS20QXr3BIRLc8kC0woD\\\\\\\/p5OtdqDpnoUcO\\\\\\\/oVSBhVL0L9FIxGWpop01W1WA9ZKeYgUeFIdsico0Om9XJTp2rxV2HoXubkUYMQOTLjsZqdNhYJVGLrAGaRLYdI8mcDaeDn+VLBMtu4FAr5TyZpoP9fSDR4xCe8ph0KTI1x4F28a2phOoxqRSBRWWb4RKla8GstuyOYXxdRoA9YsqPq1HyCZUo8qIL5hsd86x\\\\\\\/2d8CLUi8u8FsxAngZDCCs\\\\\\\/9VQCr2PXgaeOk6wfwt\\\\\\\/jRHjWrEKLW7qx52P016wPRkD5QEFUzZeffdlJbX\\\\\\\/aFEAOE9DrNlK23WWEqYoXtKzAx9HUNAgD9HZtAlWKilEgFq+kOs6DCXseqBFQ+TO79kP9biNnszu6mH5W8Ik4+N4VNqvb4mY8hNeG\\\\\\\/Ey32RY7mNHsa06NXGY\\\\\\\/hvh5uD2EV6W0F+iwMtezg22iZmmiDM8mqN\\\\\\\/Sijht4in4P6aTfDHQ3DpJlfuj4lPQG2+ofMtNrlArQPcI2l\\\\\\\/jbx5FVm+9cLfbITQI4mist6JCwnwy6tL0GLcgBwG\\\\\\\/qjs8troM2hMan2lteE0EoTShMkpBZYVQ3icmO6ECDnQPOr+FOqj5lBVeuWwapc4uwKmh8uRheUIDg7kGo7+GEc6z222BQqUWn1fnZSkF9E4HFM3YTmLUiaQ7P90eTIzLlgK3RaQzAutrMQhhhNFiOYcEwNBr1fLUy0wB7DhdtzrYpNp+QT3qS37NOcJUjWqGaMmwETZf51Gl7GN5uWOEbiRl6vk73ps3mu6BXE2G7Wolj3taDvgS1HeDNSAahW+cfGPwK2Gu2\\\\\\\/zdBX3hWbpMdUjltQrh\\\\\\\/w\\\\\\\/JJKlDuygefRoBsnJFyt4Dzhtsc\\\\\\\/Hj\\\\\\\/g1dnb0RJ5JegDnnADxWJua9BzFHHlsa3luJpakATMvjo\\\\\\\/NYjzpg1fkcc3HyxJEcmogAFOaUgtlv29yN0E\\\\\\\/reqllg8N549Oal97H6IZ\\\\\\\/YKmeoBZiFgi1gCSMfU7flCeSpVLEPCvkaDBion6Ed0YRaVrtqHRnz5mkDPAQAYQBNxCLRg0IEnbWdV+in6qMTvINGBcSRuqWHd8v0CeehxdtODM04UCUL1HhItg\\\\\\\/Jqzp+n3N7u6hbjhIYidv2V7gpIjobTd7iAK8xgrX6i+R1oINuwZnF4HE0h5SyhIndCa\\\\\\\/FNrUXrYL3qOB5uFp57ZDCXPhOCDqIC8Qu+KNqhowA2EdGT0R4ElRiQkuu2aIDHwmeDdQqtHf\\\\\\\/SAtqCA8I+Rpa4WnG9yEDRArfB1J23SKiW4wP\\\\\\\/pIBOQY9ffysweXmtzyhmlG34GymvufvZ92c\\\\\\\/EUHuNRvK7Auw8CD1nAQjvpcnLgf0YLIDJoDztZ7MysxdOwLgCDj8AjyucBn8tIAcSGxX\\\\\\\/HC8aKzTwqbfURb1OETEgJqAqE2Eyp+kpNabdRAGcfL236gpre0TFg3cnRCJ6kZLZdWLeekVf8z4BZ9zPUWz\\\\\\\/aNrvcgb99kTY\\\\\\\/8jYHxxBWuXieIKW8RRRWppJci0YhEPx5m037VAZhMGonHeEDI2LZ957R7wgX4YvytaO9M411VNdQANmJktzWHlkXjolznUPwxGFqXoY3IrcDuURjTEgiw3RNnoSFZ99NcVfx1qJ0e07t1VpeXcNu1w8MNUpscRwK\\\\\\\/DCK\\\\\\\/6D7DqMqXQ1HuJI5UNNiebK\\\\\\\/s0ebB\\\\\\\/McXNBDXP41bqWQ6QF3FUnJJ9lu5yHcbMZa9VtWQ9pJnzw3DQoZBLnY5Atq+YI8qP6AeogjtZDRwmUaF2mk7c66I6qhaEoRcRMVojo8XcClGC9xos5i3dU7ZvhAFXeH6o35KJLje2jWN4QOIEAEVRvaoGnkTQ98tm2o7GeriBJEVFpPyi+bkeiePrIbijNnEDt2tTyQLWlvMvpFZ+GbPvLrKV19I\\\\\\\/UX6deXXfz66qeHzQdnTPHN7c2mmJZL09A4IsMpovyaZMKK2YoAm0lYCJ6QN\\\\\\\/IxHQ9+Aa8Lq15Qxyy7a4QRrmKAhoGAyJhMIOK\\\\\\\/5C9As+v7+SECzSj\\\\\\\/KgX+izHAeybxaV7\\\\\\\/AoTluPvdJMm5TLZ8i+LOHMMWeiHNHf7XZ3SkPj9ClHVnjotKQvFxYuFvF78Pjj2Bmfh1guw7RYhBwJcTgcH9Od+kfk7xOML0RTfGQ1TE+LM2A19\\\\\\\/Ghfuzf3yIGdEzGxsMVC+6iRsSJdRE3SroRDK6Ole8HETlUsTjVOsDMxAJ8HHkWc5sSiewR2xOX5fcjZFdr9y66Ys\\\\\\\/CKTpB\\\\\\\/hDz5X+3dQtiPGi6nF+P5C+p+2CIw0+odVgnQRKziI1JUERO4+Qqt3Y1ctHv2am3wjd2LQNKwX2cipLEp+He4lvybsAnbBDlAT96G9cxj5GFhGkl6HiLw2KRcjm4yluSrUASX0ZiFShsg2I5xVjmXha3AGtZvFzOR\\\\\\\/LU8U7Euzdx7dFaWx21dzET94NzqD9RhHIY4hXZbaw7CfJ3gJrrKgCnO+HVHQuuoWtc9P9oiu+QFXIAcGG6r5Ecp8x0Lf7lHqs7AOrQqkesMh0MV710aWB6Bq+U01vqD4wMyvr5WZHffq7YB9lVM81t\\\\\\\/owM4PaPaVFBsCspD+4pTJSJdOXaiGclSgGt5h6UAlvZnLrqKybSYAiDTLhhT3djA3raRbYxXcGZ+Q41y37lo7wNf0HCB1QIFFL473vSa9vOzmcy9MBgbn2akMhRKY5eJDVTrbDmLBtoY6jHHn3b\\\\\\\/i2O1qkxjiS+pgUZBQo3vbcKz90QgtAIacd\\\\\\\/HSpfnShW04tY8fkFVZAkFaHlRmvvl4sbuyZWl\\\\\\\/5xd5nobFIL9AcIEQQ1yklDq64KCpi5yox1u0yu\\\\\\\/I02sbOjN1F59xW+Da9eefsgdnnRNlHUwuWcVuemCbRYLvYfDiZmTURe0NPs8xxOQH89WZfykNPVyrHLaB3ucwpQbPNdxO0VD12nabQbRZMY0B6r0nFTpJxX+SeYXpEWrVUXXBzoiqVZQvFGvC0F7CXP9GMRqKcAPrKx4DwYI9WGnzZABnkP8mqsijligEc8hojJ\\\\\\\/eFdVG6uU2OGAf2Kz8Q9Nl5Xs6jXZgZGQa3gm7g7+79Zk+Fkban9Pq2Ru6RPeY7WaqEMQSfaTevAtRTa4UproLLBDFTwPXNrBpzI0qSHoR8KjHBo0AkmgfJxdv6bb4fGJpnaDf097VzhiotjAUe5ZfQyZNg2ielFSS1+7rAWiAtLgef5ZtKHvoTc+Qn8+MrV0jFFtjAjRCeGX6sCMfqt1KJtBo1guKDE8oqnDOVnQgITJGMRHooe+h\\\\\\\/e927zxrS8wT4UjoStph4eTpUOx8R1RLS8W\\\\\\\/vwSnov8n6htL8z+6qLZWJwUnDTGOAKWIhfCygV969NIohOK3XpqMqmifFJZ+7UTDmrd8xbV++zAdwjW5esUcBui9bO3+5c6tF01N1Z1okmdUfmocsAyCuadwycSCrBCgXoLyy1Zl\\\\\\\/dK\\\\\\\/rpE7wXbMcq2MawEV3ETc9TXfnHTulNh29oY\\\\\\\/\\\\\\\/KIUfLbzSOL0wTcdRRemae6AWiZgRI3RI5GmXY2CXbgyY4Co6cntD+i6aMh3tbZr1cQt\\\\\\\/k+GVvXB8lxcE5VeNonXR8sJ51OXieYCN\\\\\\\/t6sit4klYF2dz2x5PEqZO9rGaR6fvHL0Dgilgn26kL3JvXhUA525dxRTx7+xT6ua3hz7DZtvQYP9\\\\\\\/QDRfEVe8fTW3XZmSEdCPxTN3CdAuMhR\\\\\\\/+i0wHHcJLXns9+5+TywXCoHfLUnFIaQ0mSgz\\\\\\\/T3QPfqz6wrASrlqt2dzyujh4CzYpSMm7T0v4UQBFLJJc\\\\\\\/s+1I2G\\\\\\\/n5cmToH01Uiah6a4NI5dfXC0CgIJTVsh2hYHvniHdMCwoIDLfC6LxsyhqxiHpKeVVN8lVlwDBiKKi05xuTCsxM2ffyV5JuLm5qSXr\\\\\\\/IPUbyX5bvBI9EcTVUV0v9k+QREqCoOOv+ES3UbKnCdoxlhp2aGwLD4oGidzYZobhC\\\\\\\/0rhkvSw6TzuTphBbIrE1KQ4iBpQbeW9y3Yc+KVRwiOkPHuVTXdaMJe99eEb9nMgT2xgXxifEABJ815F10o7LCy0bDftEFe8B4eWlSR1dNcRyEC6Z8PtuQX4pdfwAOrGS9CCYZ1wAmD1sXaAfPAkCseVbsJWBpaFSdnHuNV\\\\\\\/h2+2m+OSvfu\\\\\\\/ls6dGR1+U0QlvAA1hmSBKXi8qQSSdCl6IC033Kr6wo7AXWEUh8g2RtXG2\\\\\\\/18LpAhTthLQ5ef+PrvT36FT4915GVooEzOeyUWArRUUq2e7yYM3Pl\\\\\\\/I4CXo6i0wjPp8FHWCpFXMYTRPR5LK5xj3D\\\\\\\/3rXcsQiGxlo6NLAWl98l5Dl8R4GRYZXf1uFYtnlGxcRVvaek5XyIGbZ3TI9din\\\\\\\/Z4h653Htmw86X04YgjU\\\\\\\/u7P9GlioKhu2+P7JRyXp0aBRnJZy1feFuQPqDrxM1jTmA1MNSTmGxJOGMp0rAKTlQYxKhbMBe6EgfmpXRDflxmwyLIu9D+etjdl4IXSTBdWDf4J31FKXAcR4l7cm3y8ZDRIw+yI8fv34PKa+\\\\\\\/zxUd245AlxYKFdwvImzGetzXAVsdpJ9IbXJOyyv0RosCF7dNXdCAnkIIFWp\\\\\\\/bj6GBfxi3MnjL2nW54nu9ZQ8KXnjOso4RuONgVfFow4kFqyazJj8hk6zzq+fpo1Ka2fxFB5UIwYoVz5Y+KYcVVE5MI1Z0yj5RKCYiEFt9aRGRzIj\\\\\\\/hvR9Ih2Jfquowi1XQxscD5CX2IOdGNlwyCPRWHfWq6ryZ2AgFJeAwracKm\\\\\\\/sK7oSaVP1H8hD462wY37+miD7spuZvw+tGJQA7Z8ZEX23ByvBkbBpAV6aNfiHGf9YLH4dBzi9Z4Y1j9Zf2LZlpsj7+JSIkysxItnVvo60BuPN73L++NYF1hUc9NPKw9PEfPVnYHxHcKtvagwgcFjKmWndA78luXTCnSt2XkomXsBJkUUZBMuDjtZ+C6wAbvOhLzIlAp+1UHe6RZPOt7Iv2H6d7HbHzU8Oz9N\\\\\\\/24sxExt9W9EsSEHnxaDSfST3naZLlSixXgeBFWuBxteJrThj4pZmghsUXMVHYVQSxztr6yBhLyz6ucY23IODy8uuLSUV6OGoLLeYkfhTwCh4Gcfq0J1lD4xYFb2XLi+TfGoBRuzKpY5HYrzrdAR3QTzYLORiPLQQ5oGlyxYUA97RlFYo4HfS5Laz\\\\\\\/Mtkho\\\\\\\/drhXZhLSTPmHWxMIIT6jXldUpEsKwJLFxuqjmEUCF17X09DXaQo9WkojkXoCgS3fcOfFWMDVhZ9IgAIWmv5dx\\\\\\\/lKGzJrzeDQ6RGrvw3cV5BRscbjlusyNGA1amCBz\\\\\\\/OqnT7CBqD8i+PdQv302ED+OihAJkddrnyvU7vRlAvPRjEVGUttlU\\\\\\\/wEiVw4qvAXY6jo2YUtfzUV6gA4fw2Z3xQHtTgpTUMtRSDuUA+tGMoDwzeDKTeiZsHYnNYtLFnveybRWaksnqfjUDX105rA5LHXPKUnWUvLP8INDbPXARkAdubhQXL5+e2EXsyVQhfVfXsD8wYa2eSIn97IA4Hx9Famabsxve+AYMESBnHmatoZXMljFj57zvVmY1KmXX5uyhinH3TSxsTNLbqW7H0Flb2cFjXGj7Y2uxO64akIvZTUa2j6I\\\\\\\/KOx5Hymqg2Msgxa+XYgihz1CreriEUmVGSxUCc0i7Zu+JY2PjaDP0zYYhBI2j2Ukb\\\\\\\/YQeB+SCUQ64vpHYT5bPCgxpt8R3o58hm5v94AM4m\\\\\\\/6pHeRczgPLBIYKwnFKfbx1Auhqd6Fsf8yCW2pni5aGs8gTJCpldwNzwCs7ixkKmhCizPko\\\\\\\/OMyQjHEa8vFOyd9rok2+W1YrcQAgEk+tHYhah+0K5zk8LNBVbj23IKpgiYX7yzKU\\\\\\\/qBBvzaDccZBeFoXe5LLODArykHrVkplPrwkNOEfac9qI2LMMHKbbrSTgSKSZszgWZDGbPtkfx3QMTY+ZgJS6j2WtsmJC11a2DD0DF63IQqL\\\\\\\/q1uSgHAD\\\\\\\/0febl46lzp4pQNS4nIOvE0\\\\\\\/AEtlgf+Dop7kE+sx0Ome4NEAtGHSjgk6dCfVlV7FqF\\\\\\\/46RM\\\\\\\/KVXQsFF8lwbksX76qdDqHFrdi7BWLAfIIMZHfLR4xwMRqEo8TX3S8qRvejdJTxk7EqovF9sCKsFdFSEX1HCnlxTPJZxVP2nwA0O0drT0vNbJkwqly0t8gNL2JOnKjlyn7L4VHG1MhC91lOgjqGa6g3wwhos7tBfyTeeSGOG3jbvSUW4R0ohbKsTUT0q5vx3R7aTWHOew4\\\\\\\/QfHZPp9CxrSkYVfFzAY55VF1wfJElvSiDtHTgXfLRLblPh1F1tZ+ZMjNiLP37ukEOIqKbOENTgaoegH\\\\\\\/ljO+08y5mjpZtJ0PgJW9RADGPpIIswiwvTqhNv9Ohh\\\\\\\/AidwAynQzppJ0ZCel5Rj4teEO83coQdx8Iiz0cBq6Dp6kiol+SDTSdGiEzhOJVChojf7OoNUJPlvR4XSaigMIsW\\\\\\\/pY\\\\\\\/8cXnb7rXw3jpj2aFjv9EAY9le6rlFePrytIkDSJuWBdMgdRy8Mr+gJw7KxqaNpD1RYleLJluXxInfMdmvoTumeOPfh7f9UyMehr4zSpGF7vqHkDJFRSXX0jetNMtsl10sSo0FTfCo+jSh5Z1KKbkfZkzA9RbfkSYojnOh\\\\\\\/drT6yp9PJvHd+rXdhlheDkCJyj4s73PF3rUg6PmnQTiuS+TLUQZyFlpHPvmltaUlvhlRNe1HyNMA5Ft7s\\\\\\\/xXhRroqHallL1QGna1pmHG\\\\\\\/DGen9xHiYoCcVyesM2HojS3JtjEqYYgcQgbJQGIhiuB9LMxrdqpINPxLHUVGRu5nPCfigMe5XNDxveqBTn4jkGEmikHOEkf81poiGBvZsP0G2lJ9\\\\\\\/nk+ipROiIkOQaV2miugcEZ2daWlw7nhhYXvN4C9AMTrEDAOVMPibo2vnQRfNHkIx0fqoltYiqpYAYdDaBJyqMWO4TNmldDuePuK\\\\\\\/amKwstbULHTqTXE0dEuFKmPgBkt6hH1gVgnRjoVdL7+tOlk25Z7q2FWZAHcBgOnSvcK7Rhy5JsdHntSfmIlyB57dhmQ8LXo2IkOgkCiLRF6OrDwya\\\\\\\/Glb9eoFZDXeIMLRhdEtef6JM+Qxrp+ueQZfQS4rVE9p8SwfjN+CXANpiTTM982a+nIJOCEZOVaBlD78njRD8\\\\\\\/1e26rSFoXV5Vp3Uwb5QCxrE6A7qg0d+XihkUdzEv5XDAhPqwYjDpMa3U6zXquNiL78Rk2IwZ\\\\\\\/jTwRWHpndN08QOGTHfXFzashZvkEnSPbpqyqK7xfbEHrSedndtD18oEfA51NWMh65m2w7HuZ3LOtJSiOGvm\\\\\\\/fxMBKOLwyV4G2V6xbj9uYTyKD6CJICiXaWmZECl4w3+i0pBQVN3KPNZgFYTUMbWE4Sw1ayXzfKxwdGWlInzgdcQKUxw68Ep62UviooiEX4Ayg6Fwt2HPhU7uBo2SmmB\\\\\\\/YrRCvvslKYmPQk0gBvcQptyiCRb\\\\\\\/xJqnKt6m0O9PqSc4yGpJIQel98Tpic6fJDSwsVKbwQ8JbHTohoURvNJD0i3vSD0AxgDLimx9L7dm09jXwlzt0AbHOMSwI6sy3YGoPx+8Lmeh7WtI2FyA8ZPKIJZ9xV0YMrDPX8TVumyawEdJmkU9cR+pJkiWEqJOTwdv+N2aiV07phadwd1hImf7fC\\\\\\\/60Um10Md+DAkCwLUMjjtHUn\\\\\\\/xZM0Ue+LQhwAPQg34MRzkNRZdQy1qA+D2da4ZqtrHpV6yRaLBjHHvMexj7phse0EZyxYTqwSZYsbRWCHratIwIEJb8e8y3b\\\\\\\/yrZNxZRJp5qeSSH4DnfB\\\\\\\/E99Jo1Urs8Dn87TaEYD1mCXQbmJBJJ69qoWs03QDKlGtAmjmGOyR2tBBNCq2en3Vmgv93C922JkzjQRdqXP3tEFuVGg+u3jAhLR\\\\\\\/wHcy\\\\\\\/pPpHSEuNJSs1s0+NHSoq75GQ3cR3FdPyJOjS9XQYxr3g4u3uQ\\\\\\\/\\\\\\\/+6PCbvPMig2Y93L3zlAv++oFmkrKl8rKNZsde8dQqe0qgqFm7l6WifCsHyz3rA1MgInugpUpCp6YYQkyXlZinz+ApV3O001yTSAPIh3\\\\\\\/ATm+lLGMDRoi61NtPE7cgPfmfc4f7WJQzGjz0QYyXp3IVQ9rQr54EchvWpMTpmJgiOKF4a5hglSWunidZaTP8s7A++mir53BzSDUAEBhd5t4FDFu0udcCzwcdBkBIvIY5mxMtYJWdn77kQ\\\\\\\/jQZf1DlMRPc+4osPgLMRDAi1+PO+q\\\\\\\/Uty6U9IBKoGEjsiP07DdVpiXzCs+AHfjWkKNBsWJvu3VZ5brqCknG7wcB\\\\\\\/CNVuN0002lvn6j0ylsBbuLSWfxfDjZ+2EtV7OYIgdX0NuBHBdiRyD\\\\\\\/B+QX7wqPaeaGZJ7S9hr3rfTQr0Kf0VhxBAZe+cRLLQT98UmVWxb+wHFV4MQf1eVJvSiv2FWk8Un4cJWd8k\\\\\\\/ov47vx\\\\\\\/\\\\\\\/\\\\\\\/R7HMjHPbhZ0PeTSHCu\\\\\\\/oEaVM0iRxS6QHNSiYbDllKWY2BSt33AOoZbOLlzd2u5ki9I1z\\\\\\\/HIgkDcRo6x9etqyLl5p7FmJ7daqFYNtYkZPbrP8reqvyEZEgWLMAJcVc4El+LjE7LV+moeJcnyBwmka\\\\\\\/+zlx4GWTN4xFeW8YFXc1DkFgKhzhBqaf92vhDi4XJhe7VTrHxoHS7HeJXlS5tRMzMazncQzFhq6ZhV4mR69DCqHfJqtIW79IIC32M5MZZ7RaBZcB9rvj2hAWAO9UkZxr+YPVbRdja0Dhurh85NvqpE4GaepxmlUSlyGnzjp7RqK7MltqX6Xma+9Zg6nFDaxR\\\\\\\/VVToXBAhYBC8yRW\\\\\\\/VLxs6j24k6lMP9Yh3KUmKZogFaKdxttyCjucT03HbDHv71MkzBw\\\\\\\/ub3\\\\\\\/4Ezpvh6KOpOxGzH\\\\\\\/vnVQeOzzfpRlRotcgkDd6asmlfaKfNE2+CT22bvwAVnm\\\\\\\/EjOiJXYRCL\\\\\\\/U9DXEVGUe2wKs5sGj4cmLocssCSZRYznEiEzjJsqADqey6vn+wYrE\\\\\\\/snT\\\\\\\/hfBk5LBQ5YAeQOGD3vRzYuBESZJAUyk4zJONwlLKop0KJMfDKAgTGxdRE5xX5dWnKfA89aVeUdv8VZVtc4yEDSZ+RpE200KcMuqP+iac\\\\\\\/LLPFALWFwVYfRexFZODQJu\\\\\\\/ZkqqEMgp22nu\\\\\\\/Ys0R3Dd5sC4eZiNG9JjylCJNNT9mkleVfPCs8\\\\\\\/++XQLUpTe3R6lAXA+UlMGHRhK\\\\\\\/pi0jBmTukvw0WbrKCq9qLIIAmCFUhz1M5JAHykvOOmWBacpXe5nOxGTwliYDMC9qiXb+na+LbsOqcykmOxrqIl\\\\\\\/npw1ffI3WA+pQFb+LZ8gOETh\\\\\\\/+IgLerWx1pBn0vFcNjtF27M0\\\\\\\/baSM39Zr4Nl4+dAASSoXqhJFhIz6WonQYS+VerifFm+10QcOBIaYib1Ka80wfYW7wURSRVE39iO8ZOFMme\\\\\\\/4LqqZ2DHINjxM5JLqpjv62erlawTHpEP4k+To6nKgO3ZBG\\\\\\\/SxdjImj+mhLd6sKTMDKGC7rcusNB8e2M5b22v5ARL30RpO3C2z8ieKuqnAPFsUvgjVfbqZqzjsBUw413fekQ3txx\\\\\\\/eOKejq+IjuW8nPeCKGr7qcLPeflnLPw\\\\\\\/I0YkkTMejdUCOR4iuz4\\\\\\\/b5AjmggzGEYjhF0QeBqA5EOsUAOLGRFxd4I7GfbJeOoYx5AdE\\\\\\\/lejDoXMPzl5iQ\\\\\\\/ydckegNxEGQY7yJEG5EiunLgHvZOVFrcqfmWJ8fGpEQT7PQo7uq2R\\\\\\\/iGpSp0mnkNBnoFdXnby4cjcmfgPJby6BmiJ5K3eFX5rFHXD16hyg6AwX5WqrS5518qJxfMCcQ2ik7V2uAyj5GZsAhf1Ja\\\\\\\/jAD9Qz948zuYSt3zNNFu+Vb+Mpyf0aYJYdrqOcxggjuC6EeNsuVhq+eBoi2AXcJ6wP18iWNLRMhZ3qP+Ixlw8FmluZMrGS5gZknFO2DxysG0SNcp00GnT9FOGyHvxXWZQCdQlcYe\\\\\\\/nODBl7o8TJ9QeJATkuvR7lLsS0q6iO86vb0JBLEhQlT3MFiOyeYnLBOJUTc8hTIVY6rT21bAnthxuYkoTavnlTT8SLokKxvqiEQmkan0xxBtorbhP0xXY9N7NKHA1DBfcaqK22Qjzrra\\\\\\\/gWd6IGKQxuIIzfsUqV3Y4IfUs3E6eI7fqc6bJpGnoNUeuEROFeWxKqW1vhIBxt8JOJJyNdgbA6CrhS0BdC91i6ZL5izym1TFage2wvKwIBc13b6bnDqxTDVF9aug2o5XEkAuQs0Bf\\\\\\\/3CGq8lt+uuSqxiveu\\\\\\\/PmTmnt\\\\\\\/PEpzjhMroaz8eml+f5HDvlxUWzv9kjPYsqcmj2IAuFo9XdtFKtQMg7Lrwz8znNCvCoRL5L+pUdRIBNCWGfBsGkzZVUIAXA62nrHyQjZe9lwUZRNGgPdCWLDaLNLiayfMjQoQoqmUeVAxfK1Wd1eRwWt5p\\\\\\\/rPxcHHxBOmSCZ11QqTH0tpet1ZFTDkYLMH3mG3Fz6WEtHWIvYdY9j1tPrNlMxdmTQkmo7lhB4UpNuA+YckaFZimMqbVbsTwldBBejk4MdrtPVhsRqtjjkmDwZdFOi5Eh3D0ciKzH+HiP+ofzhjz2b6tDyDY0Be5Lt0HA+5dMzekY8qsMt4kpkGhlbcfK4i6hLb1iO73tJwmIgJVRvnJiur3qDSfYMJR81DZbR37YHh0h\\\\\\\/gEWr8uCD0gsPbeOrOlU8Kt1zvjD1V9uyfgBQlTc4ygb3lGmmnGXhLInXlpXnnptFOqjP60TQJPYIpjNqaxz2YfaTxuKQbFKHEV\\\\\\\/EM67QaX3puGwNlWrxYSHL+A5xR8Ngqh2vqkX9dyXcOKFYC8IFGkDq4qRsXqp5eV8M1am0MVcOzto1PtAHMqmSWwiphb6lA1lVucirTtjRPEj7E5njPPu5VyWUKD4LHjsNOg+HQ5a3UQy0IUrmxm1QhCxKNeG6V0Ld+vMKSZ44f1qwUUFgPO1inWPjn3caV0nwwWgwbp931KmxHb8I+l9zbC7Nvqu1ffp3FHnNBiAuPi\\\\\\\/PjNqKVAXgxaMOtt9UIOlfbXpt+yfWZ7jlSXCLhn+u\\\\\\\/Le5rX6E3JRg91uLxuPRoUVWK1+PRfWkeWQ0oj3G6eCfFzni6JUPk0UF31p0U2tMxMhB1YD3BSSxv1OWoqRDtGi9YIkPQuIs70PmiF4sTrE7xx8moXUz7+HFSGF7isHcLzZZMrKvwLNehBmdyl30Tz6zoPkIgPx8z+V4fig1Vr7HbFkFL\\\\\\\/T8QaDQYPMLHhwLq\\\\\\\/3AK8DLkOcUy8\\\\\\\/YeCvLiDAvQn6GzoKEbfWhabq2zYPiFaFiT88zbLsVHQ8mx7FSOjBm2kVStyO3rwJ9kHmelaRLv4dzrIPaIv7+G4s+eycL\\\\\\\/nKz1C6NlhryHhtrQGbKtzI3oYYwNmLJHALqMY85OChQrQInkZb53ms1TzmVH1o2zpe5Fcyae2Qq\\\\\\\/5+sep9ilItVtwMUpclVB+FPzbIzmUih2J\\\\\\\/znHEZ\\\\\\\/8R\\\\\\\/R2\\\\\\\/7mB16X2FrLN0sdgvfGo1\\\\\\\/pe6I29mTF7BDut6bDIx\\\\\\\/82MdlNA7KsYwUXZKKgPGxPziQxKPAGZ6HpmTX358QT7\\\\\\\/eec38Yxeb2PsPFFurMl+jrdAW5\\\\\\\/CJIzxe15SsGoDGjd3nPZr0k+xdvFr2wY7Fa2AwOGuMtcuCF3\\\\\\\/xSZz8ImiJ3k5MX\\\\\\\/CxTma0dafPDmMrEu3d6sTVizspuDIl5RKJYvn7V7DUJLPqfQm8SmT5ArQlLh1T3k7Qlh6sGNvUTPzmJgSu63RcHF7ZB9Aaw7w8U2iHQmsS7qL3DMqPcXDgHkKl1ExZF0s+2UpWmmANgqWQkT9lYrAvyuDXAocKrxvxDM+4vpGbZP5jEMxblttjO4CEMtYrMMxgA5qon9H7pf8Uy\\\\\\\/DFVAuK4kwnk6o+rAbLZkJEVIJGYvUVwUh2wv4VewMSKqVDTN6IRMf5YheJGtccIXb0dMaoCCBRjmRrnn6KBxSxSUJTRQ5xffPhaqMYnlxlqd2zJQkKoqma46FlJgGoVzfb9eJZJFk6rODIRFoAQDyXsVI++Si\\\\\\\/pnWLnCKcC62XXIVxfBDCLtUOwcDpiAehZ92Wsp5wV2Vl\\\\\\\/+yMJpX9wD\\\\\\\/+pOJS48PkYSp4UDBp4yHGOLi7sIsB1daz50JnpEKsPNxlduWT6u3AV4EVB0OCdSfggRo08Tt5Q5VAVuJkrNPNd\\\\\\\/OxmLolhSSmsBEPyKfxzecLeYhwRqqMyTk5\\\\\\\/32iV6MDMMdb45pCy0wD7ORInaj8chngW+W4uqtEWd7Hs3NhSrFwHuV+m0I5rem+oGMgiePt7XhX8AA6zibk0b5cjTrRweqCGyK5oZ4mxr\\\\\\\/LSZUV2F0SzwLBEYRKyk88rQFBsxuEGPPdLjGF0xBhyd7yiNweYwkQkRDLUyHWFw+ZK\\\\\\\/TMQJMWGFn5pFZJ1uWQZTsF52e3OyjlSf+eVPaQvEk0C97CD7vibASNVECLS\\\\\\\/4ozqRtWZf7DTJW46MKpnxPzs0a9+0kAli3iNZse2RmOJ1mRAB9ZbybX5QMfwewNpOSUWE0hGtZ5VWln2rBvXhts1RMSbz8Hbl9mFeqK0VPe8KEkqpAWzQiQtlpQuGnxTus1pEcicixTlPWA2\\\\\\\/XS8GjdsRNIhwBptYMCqq5GvkYQxTzCV67fOfTHeGKR8zX+wjOW\\\\\\\/Q94sQFV2IfCnD0tdLjgts7SZ87q1eqyipXyyOpvO1l\\\\\\\/1LbzL8PEjH9bsG8PxbYuuRoAP2Kw6pCrt0vlgfXQxZ4xvcQNT+5nuSR+H1hPp2\\\\\\\/S5FlkFVW3MwMOaKoJJc6m4EM8CoMuBCb5Ff1neXPp18SL\\\\\\\/jFK253PIVnXYwvwpRj45WFHU2Z6FVSwBto09OA0VQwsmtqEWuMc56OqlJJk8F1Rke4Uh\\\\\\\/d2rEOUepDk\\\\\\\/A+I1sVxaDyoxA5xFXSeCkE6N8xa22F+DDLVA4RtMYHE3kAIhDpgheOaqpdOI+VPUNJHuxOLVN3K\\\\\\\/IiAqhWq7Wj76g\\\\\\\/oYP7wdC8xeTIUcMkcBFKe8kZHn3x6aaWEpa1k7M4chvag9Ivh4vRrgCvpkiISKOPfBDk836XtMZ7x\\\\\\\/aYIg52\\\\\\\/oqihKcz4ejrrISzXaWo4UIHR5KIB8YBIa++cpBrdR8ak4GpTl+pS76tgjWhzcXSEFR\\\\\\\/z3qk6juWIrLq0Soyxl9XFIfzGbjj3APWqCuMSPryRj8J1tcd2DedfXIoJM3aKYOXhAKkCqM9VYSRvHtjSqKeDyS9nXfUumoNp2xPdX2ZreZnQiGedNzJjl9nQ7WxXoIn9QkJJPoNNDT4qZt4gL44yiv2gT2vdbfENjVmKv26TNGVctrlkZDkdGaY45C+mC8uq5z8gy6gkxKMK6Cq\\\\\\\/ADInFIZGW9LXVeFT1Zk1qp6sL7oofaLTNYh7l6BzyO8uIq0Rva7ki8yKVlJWIBYWq9nrUu5Yk5hBm9EzBggrdCZRNp2evilPEuiVbeDK2LJzdcmBMQIah\\\\\\\/6huS33fTghsaupOSv4wflGhYCsMRgWeRXlI5MlvZ1UsPY9CjdujjuUKPLO2j8FvfC6W8djVJ3dT1u+kuYcp6GH5os37CZw+m8f\\\\\\\/qfL1EITv5acf\\\\\\\/Egoj3Am1K\\\\\\\/9vBIbLp92IvPzZB+eIu47mjIA20zB4Zar4qEgVf82W6P+AUIdLqgZg4E6WW6c5mL1gJM+Saknp4KGgK2T4F8MTy3puvuZBFnajSsvEtsVmIkwrQe2hEscbuLH9Ke6\\\\\\\/En4B9OwNRxKHptzUfUDiS7zNTS8\\\\\\\/4ly4s7MG0fGBEnwz7bhFpuufcPZUepi71bFcU3u5CfnFAAXStk4UHYxtT6b65305TZOgprZ04QQ9atMifBlj\\\\\\\/awxJOBAqyWu6uYEgtzhawTRu\\\\\\\/54VgIJ24OFm53NC\\\\\\\/L7+MaxQLH1kFuaCV0ek2rVvTG2JVI0ZsC\\\\\\\/HXkwIrdzumCOlGXcdNDxGwB2qZwsXWcfzbTp3Q0R++v6gTJiFetiIheepcxa5ShPeUYTLHUGpxVMIc3t4X30qGN8rzPP407BPc6UlgnxU25fAWyMrx5Aj0sLMq8w91SAYhqMtxWiTjRZADq8S3ZMKef1rYomJTqwuD86301czT9L+MHX5bMvlpKSXwP2L5hULieJ5ofp2l3URlh7h0v36qyvgvvODPfdfEVTwHTwTWr2l8RMHEz4L\\\\\\\/TlkteFck3zBKkbC3Gk4zWdgEGAGUvan3bOnk3SbpZ1gy0y7GQVocdUKx1qhP\\\\\\\/iF9YVGbdUVMIEEP1GvW3lX14KeKkgJw1pBZj5oNUubjqq2CeDQ\\\\\\\/WDH3InTOEChSge0sFbo3+aFOJaIxu\\\\\\\/+ODt1UqbohVTxG\\\\\\\/Cxusqo0ixvESMOD4NCywPU7bGErDuZCRAYQ+\\\\\\\/8s0cGNC21EwiDDpOY5RWEpS2oq919OlQ1DDC0opi\\\\\\\/kUaCDXLEJJnmBPKy3jIEfqnAE+Gl+QKjU4CMzKSr80OoOsrwSqbDLYLXnMWHqyaQGx8ZPmXWtg1R2G3COo9\\\\\\\/2ets0LdNs0NIJ394+RFwuGsJovqcVy84JTt4ysabFLz2H+bnJltTGZsJAAsbBdMeS\\\\\\\/+yiI4XKDMCar\\\\\\\/Le+Yt+weHwdq7YbD5t8jmroXFLtnQCkGS2wqukO84HgJok82mVZRUloCKA741dv3P3CL5EvcTCl\\\\\\\/fCpQzi25AaN+tX9RiI0167jX8fjzxB6vxuVSXgb3RoZzMUY5nitUPwrYUsSowBiU5d+fAUF+a7GH28MYBHU9U3EEbQDP1zLzOB9CRL9bMN0LYTnBWlWKhrd8XqDfHzT4zZPqatrIvjvwAqBwCvKwXbsMOAI4UX1bjjK0goXAjx3kSGRwvK1x\\\\\\\/bRw+ofzZ2fg5i5wo+BIghLmbGBVIeoJ\\\\\\\/2WlsJnD32sMhQI8QKMTqrJ3bEs6zb1q\\\\\\\/lKczQDDWN1UaarjUnx+3TCLWObO1euId2ToBNmeTGZcKELKmI3S7+F1HwgKWmcXqwiaQQF6Hq2p+SCAZVbKvJnhtYnEU85FF5IP3jG+2cKwEdUuoxfcBRgaJbN87AirvUchmvu5PcoB2e1f6vjDpaTi87REfnuEkA1DVykhXVJ51iIeV3wf4hM2cEAuC0JgrE6CBvail0De1wxxVM6NlNiCOQZ\\\\\\\/GUeNNnStP5kvj6HQkWMqu5\\\\\\\/AVpfs4unE82qaw6yKv1DagSsoJms6jJ2YZt6N4CLxRPzoi4O8WcWyDwFwkHnBadI+4W6HNFVvFKAhZ2uxTSthC+DLKWNb3G\\\\\\\/5V2iwRyMwEa56BO8c60LjJdqdbgPpDV4oakBHe5641OyyCxXoVLcxQi5Cq+U0YD2Hpdjhdvc62L0qc\\\\\\\/2rfwi4NpDzJMFg128yDAOC6qcbClcw9GHsqTRzVJHmHY3FpGyu99ZgIi5u9peRuI9e8TRvg9G6f4tLRLqwLXw+f\\\\\\\/aGoDNUK8Jf3sqQtnzMuan9IGAF6b9QOSsbqszocTo25CanoNZzNq+5M6XM\\\\\\\/Xj149CZ1+pCv8DT2cfAiWxnxDlTAHqBcorT7G1r7WC0tQEl1J9\\\\\\\/dXvMtSl7IWU3Z4lgOU\\\\\\\/22Dgex7plw2\\\\\\\/013GlDCIbW8JyLnI0czwZ5f0jSntm1RULJUg+j9QS2gMfA5iEnLe2hMMTCBQvDh8qaSJv1FpIp7mvNFNPA1afasvHK6e0JddprNc4WIq0nhFw+cl4lHqgJHmrLMww9GDo70tkYQsZffTfcacwH+rxLr2WoqL3l5vDl6vddXHLvBA1TPN588syyYjf8zJTtDM6ifJTXmq8GO\\\\\\\/mperVfperuFan9HLOJ\\\\\\\/wGFETm02V6Z999iVBt7QkW1O+WK9mp4ATKsn2Wj24De5XJ\\\\\\\/X\\\\\\\/xD0dxZ3fIk+60sRJ7924MbjN3LsAJaPGiD4LOZQ8nr8TDcE2usL+k83LAvNARXNfj3d82khxQQofcCns2kofXUAqe3hlqEOHgDpvAHKHieRs+xQukWjBO6R8o0W6W5DwQUEWvvlTy97KSjOA7sD6xDU06VL5deAIHluzn8gZQd1\\\\\\\/JzKQz9PX0y0oACaubCLZ2yrNbFvWxxvtIVLsfKZ68NdZjH+WaaFcdgo5NOs+M4x5IhpH\\\\\\\/1HhChJ0iiWTgArpkqdRJ1DNDmpu3Is5o2wQ8rwvrJNSXJpq1ZQYTTxX6CpyxBYfdnIVuZViiLJGueyv70j1SNLM8M5+3PfXle1vdFjMl415CJt1JR4fTvDdvPGghbksP6t5ETKXoRROlglA0q\\\\\\\/NDQymv05iWKsWa0f5JHqsty0W9gIF5Nd8kw6raH0N9\\\\\\\/0kMzWij0Bsasxzb\\\\\\\/ZbSklVAbZt0ogbSfWWnTG6djDzEURmjQmFJ7i5VgAk9ezd8vD7tyno28qYwa8KYnPA0vMG84wEd5bNbWIE7HCdTzfQuvtwvWq3XlEPJUyB2din6bViL3lEu4vmRDcj+8nRGvVN2QtxxEh4BjqF\\\\\\\/EmJTAZCTeYt6gHp9h+H+FOVNs\\\\\\\/wIM8fNlIwUKD6tNbADgb4e1nPw6ZDdsRctJQfnxRr+6lA\\\\\\\/bdgIe3MIHqaRnjfFUaJde1EdyOZUnvlMIHlrUL16uGkmX0rgU4JJRqptfpto9XFzD+a8KxTf1cXuiECP6p2VdsR+hH9edrsZ\\\\\\\/OHHGBrN64LSvy6wRgNSRtu0PcldSrCcABmMKSvnfjhgbKdNH+1FHcskFupCwBeKz0lmUPao9rPbbr4dgbHW3rSC4DD2WOscDcx7l6ZOq7+F0\\\\\\\/Ah6XybN48kojNqH55fGaEqTMGyDa2R9RfwidOdKpoj6hkv4p+1EehcUTKg\\\\\\\/feWIYXgMKoIDWzHGVOKxRlZRtL8Cpv2hOv0J6OBcsjzj2UTPgjQlyAxR9Ix5rQAOQHayDX0\\\\\\\/oMWL5mLfS+0hjrm3zmvWcy25q9xEld\\\\\\\/OY+\\\\\\\/8Q8a8t5LQhCvXfajRaIIEN81lAabp6LpH5SNPiERy9ERaQiHofDvH\\\\\\\/zNV19b548OhPwrt9MvEF2fpec6z\\\\\\\/OMUD6XXR2R6hNPsosa7+Yt3D+i76gOYXvqQNYIppiV\\\\\\\/CZnf6xefe6bAPv+syb2Km00VxlYkI2etvI0dy4YcPkIwQ8pB4PV7AOWIZEogjJJIEDUwcTr87xrRWIjm83UFEDnUkvCucpK+6NGpzwY6cOQH4M2hDCoPUfRfJV1bOfyrSZCCy9d\\\\\\\/0u5N3vNkokb4GqvBzJmb4CMKYMfXtB\\\\\\\/0gVplbhdKFHhDNddQv453uK6KcqKX\\\\\\\/1jFjfq9NsX3BZzkZWsqYEMa9oqirQK1VGKd6Qvf3TSaiKwNvzc3uMfu5mcS10Gph5udZzUKRoFw0jdGLHievrd89uFlFojzDypGGCR9YgeT\\\\\\\/COTQMTfW3wH6j4izQoal6v8YTVoHI47\\\\\\\/MBI038AAWvDH\\\\\\\/+8yhMv\\\u0022,\\\u0022iv\\\u0022:\\\u0022yeqHG9\\\\\\\/NcugMHOhNZRwUGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221622f1ff8f7ef987ec4f24c98ae76d8e8125d43f45ce06ae95de7afeded76ce3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mkgXmXveXFlKvjayuqVRRJZdS9PkRdurUkAT\\\\\\\/vxWOtt0vbK9b2kgNtXESzHqDi6Nrw8jc\\\\\\\/OKCp2qW2RqCm6hONlzcniDN+G69tBP1OywyyQe3DrVDwyNCMHYyQ6GbgIs\\\\\\\/p4TAww48afJf5MtzjTiTuqjSJpCOhCyy6wFnkwoKP6I5xAiGWX8TreMZHn2SaWFCH2xOj8xOeK9pKSGxGv\\\\\\\/kmtd7JZtzsC4Q1VUJBA2th+LEbyYO0YQLRFH+P9bYmd506wZw9AVkomKP+rH5VXHCmEzkGk641lcwi1aooa35+N+Nzs5BO1AvFUX3QtstwOtEBORyllFmy2T+a60a7Pbs4ajjnMJIxLMybQ99vHH+zIh+fZUcn4qADrCryCCOmlHd6VeLz9vTxVIhEx2pwhjdiIR5L6izqmBaZZvRu29Niz0RdSdr0AZsuQZhNnmau8QWilTSTsEGsmHqfXfNG5dceqMGJJeIzrCv3pFSuU3GiUXQVaga4B3qARQvy\\\\\\\/Gicl3z\\\\\\\/asdIwPFAJ+UD2s+P\\\\\\\/PFnhPiqQgD80FqUusgr7\\\\\\\/JvJ\\\\\\\/6w1CRgLzjp1tcWO5lvrbHswrKcyZhzBifsxMumvDcZV1mSpgSNWoIoUZQBG\\\\\\\/0zIwVyWWk4ZbdazQwDK9kMJYZBGNHUQJXQt6FcGt9GuGIDIB4GrZ1NsSnkUzBOZ\\\\\\\/W+y48OXb5jI4BFmnuDMA9tFGPQLXfypYYsgg5JXmgQaObvXbBrGRy7kBngnCC2GYspd0fSeNLmx7tpNj4sheI3T17TDO+w1jdmXJrExO2nb22GLzS9ovjt1ilreL18GSEC\\\\\\\/gjY9F1oYOwjikqFFNVPJzxDPcn7Ov98Ld1WSU5XAwgLT8VS4U7fez3a0Kwf+Ug8OcuGo2GYNQt2KBHFl\\\\\\\/6iR0Pt00JI\\\\\\\/l0tc18+OGOgyXh08JyzF8GhgAKBQJwTecM3M1dCPF5rFl+pTqonemWINHlBZ9vEyNyPPC3YHwb7DXG6Vt5QG1bUChVqNDsvbt0Gd+fEfv3OGhku7JsYrq7NvlRurBPpctjJ338MlCH4oGNI6wjYWugTETOEwl9gg5G1Md0pVLZgweBDPmg4OpLtyRq1HpjXwaSmm+DqR2eDosyXSLFkLhweLvwgCsNCyjDOTWvT1c4T7wr3pMzMB95pb5KmOO96YBgcXGrVtJNVF+bN38XM3TsMUch5GRJtLHazztgZHCJHZh4+ZbTQqPgsoVrzfw1X5kpDyOtjJtJj9xmwfnlE6f2WzndQghbUbvDBj2HNMYbnaSIt1FnFC+sQyMoaV4Z5Ec8X1s7jR3df11zkO\\\\\\\/E2FDK+Vw5npZ99XpkxGGC+J6PyJ1lJiH0PxWpYwkBv9iFXDiXK+HUDk5RRMTlxhT\\\\\\\/jQ15z2N\\\\\\\/4NpFgyyB035alZ50Dr5jKBvgODekbTl9oMKu46tT+sO1fneZdwPOGVUNOfGyaQkjGAglz3FwCxqYmEuu6dIgsB\\\\\\\/qHs945R3Ffx7j7pyeJEW0o01HEd1AlqHAJnQPQ0UK34PLMOMQlcMyC3bWZUp5TFEVHPHfgClg7AXDHeKxpUYVYh3y6kwJ+0ReLA8qA5DyhL3FvTZdNfOReTUfZVhhvahlck6yZGw+50yrNfz\\\\\\\/FiLB8UDuaXxjw744EXTb6NiAS9VdpPmrPixSJu9LxNnrjqGPeVRU2OB+dVQQ03oPa+ioQoHrPz7hLz5yJED2ctBvMWI3sbAba9Zvj2d72EjL2+1nUU7l3K4ozxNh6P8rKQwoSJSckRXt75H+AFYPhKEVMVIbeh0qNVLHzwZ0Sd1fTHv73VdqqJIHhFIj7YnCoGOuNJdqwhs91SYxUM75ic1HA9rJPLb6foBTNyu+6UyRIkMtCYJ+Jxl\\\\\\\/F6jFZAP1lSvkNLYs+DeL\\\\\\\/GeqMN7oxXaDiZ\\\\\\\/1nj0cscJByh3O6XHfdWCT0dgUgrKjFMz8k9dc5NGrCm4oDh7y8Hn4ADE4ygz2EG2Jq9zzggK6lShYWziZ7kc4DwRuRIcJ0EHxu27Z4ZtjoSwKz2irtn3l9FsJlc2jH44tyxuWBk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kC+9y7rskfRFEa6zTlXfgg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220301388a918fa5165769d7e5585207ebb993c2476eba0fd3746ca4b80dc5df1a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459169 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/171" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1tqHAc5WESIeSVXk7Xnj3cwpL0KE5kq_xsuykhEtZXs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/171" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5oUvTFzM9LIu3yk7q31JSCZqNi4gBpReCpsPenJvzz4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/161" title="polo Is Identified as a Suppressor of bubR1 Nondisjunction in a Deficiency Screen of the Third Chromosome in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/3/183" title="Mutation Rate Inferred From Synonymous Substitutions in a Long-Term Evolution Experiment With Escherichia coli" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459169" id="node459169" data-pisa="ggg;1/3/171" data-pisa-master="ggg;g3.111.000240" data-apath="/ggg/1/3/171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Stacks</em>: Building and Genotyping Loci <em>De Novo</em> From Short-Read Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Julian M. Catchen</span>, <span class="highwire-citation-author" data-delta="1">Angel Amores</span>, <span class="highwire-citation-author" data-delta="2">Paul Hohenlohe</span>, <span class="highwire-citation-author" data-delta="3">William Cresko</span> and <span class="highwire-citation-author" data-delta="4">John H. Postlethwait</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">171-182;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000240 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459169" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Julian M. Catchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BM.%2BCatchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Catchen%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BM.%2BCatchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Angel Amores </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angel%2BAmores%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amores%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngel%2BAmores%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Paul Hohenlohe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BHohenlohe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hohenlohe%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BHohenlohe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William Cresko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BCresko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cresko%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BCresko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John H. Postlethwait </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BPostlethwait%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Postlethwait%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BH.%2BPostlethwait%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jpostle@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">jpostle@uoregon.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/3/171" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459169" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459169/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/171.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459169" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459169/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/171.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459169" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459169/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/3/171.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459169" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459169/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/3/171.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459169" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459169/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Advances in sequencing technology provide special opportunities for genotyping individuals with speed and thrift, but the lack of software to automate the calling of tens of thousands of genotypes over hundreds of individuals has hindered progress. <em>Stacks</em> is a software system that uses short-read sequence data to identify and genotype loci in a set of individuals either <em>de novo</em> or by comparison to a reference genome. From reduced representation Illumina sequence data, such as RAD-tags, <em>Stacks</em> can recover thousands of single nucleotide polymorphism (SNP) markers useful for the genetic analysis of crosses or populations. <em>Stacks</em> can generate markers for ultra-dense genetic linkage maps, facilitate the examination of population phylogeography, and help in reference genome assembly. We report here the algorithms implemented in <em>Stacks</em> and demonstrate their efficacy by constructing loci from simulated RAD-tags taken from the stickleback reference genome and by recapitulating and improving a genetic map of the zebrafish, <em>Danio rerio</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/illumina" class="hw-term hw-article-keyword hw-article-keyword-illumina" rel="nofollow">Illumina</a></li><li class="kwd"><a href="/keyword/meiotic-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-meiotic-linkage-map" rel="nofollow">meiotic linkage map</a></li><li class="kwd"><a href="/keyword/rad-seq" class="hw-term hw-article-keyword hw-article-keyword-rad-seq" rel="nofollow">RAD-seq</a></li><li class="kwd"><a href="/keyword/rad-tag" class="hw-term hw-article-keyword hw-article-keyword-rad-tag" rel="nofollow">RAD-tag</a></li><li class="kwd"><a href="/keyword/zebrafish" class="hw-term hw-article-keyword hw-article-keyword-zebrafish" rel="nofollow">zebrafish</a></li></ul><p id="p-4">DNA sequencing costs are dropping exponentially (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Snyder <em>et al</em>. 2010</a>). In addition, short-read sequencing technologies, such as the Illumina HiSeq 2000 that can sequence 100 gigabases of DNA in a few days (<a href="http://www.illumina.com/systems/hiseq_2000.ilmn">http://www.illumina.com/systems/hiseq_2000.ilmn</a>), are expanding experimental space, from fosmids (tens of kilobases), to bacterial artificial chromosomes (hundreds of kilobases), to entire genomes of bacteria (megabases), vertebrates (gigabases), and plants (tens of gigabases). Recent work genotyping 100 stickleback fish at 45,000 loci (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hohenlohe <em>et al.</em> 2010</a>) reveals the potential to address questions in population genomics that have not previously been tractable even in model organisms.</p><p id="p-5">Coupling restriction enzyme-based genetic markers, such as RAD-tags (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Miller <em>et al.</em> 2007</a>), with the Illumina platform (called RAD-seq, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baird <em>et al.</em> 2008</a>) allows the rapid and inexpensive construction of genetic linkage maps containing thousands of genetic markers (<em>e.g.</em>, 8406 in gar, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>), more than appear on the maps of any but a few intensely investigated species such as mouse (10,000 markers, <a href="http://www.informatics.jax.org/genes.shtml">www.informatics.jax.org/genes.shtml</a>) and economically valuable species such as cow (7063 markers, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arias <em>et al.</em> 2009</a>), potato (10,000 markers, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">van Os <em>et al.</em> 2006</a>), and oilseed rape (13,551 markers, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sun <em>et al.</em> 2007</a>). Because RAD-seq identifies an enormous number of polymorphisms, single individuals taken directly from the wild possess sufficient genetic diversity to generate high-density, high-quality genetic maps (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>), thus providing genomic information for little-studied species. Exploiting population genomic or genetic mapping datasets with tens of millions of raw reads and millions of genotype calls requires a robust, efficient, and easily useable set of software tools that, unfortunately, have not previously been available.</p><p id="p-6">To solve this problem, we developed <em>Stacks</em>, software that identifies loci, either <em>de novo</em> or from a reference genome, and calls genotypes using a maximum likelihood statistical model. <em>Stacks</em>, named because the restriction enzyme site that anchors each short sequence causes reads at a locus to pile up, is effective for genomic applications ranging from linkage mapping to population genomic and phylogeographic studies.</p><p id="p-7">Here, we report the algorithms implemented in <em>Stacks</em>, demonstrate their efficacy through simulation, and test their ability to reconstruct <em>de novo</em> a zebrafish genetic map using RAD-tag mapping from the doubled haploid mapping panel (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kelly <em>et al.</em> 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Postlethwait <em>et al.</em> 1994</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shimoda <em>et al.</em> 1999</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Woods <em>et al.</em> 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Woods <em>et al.</em> 2005</a>). Our results verify the efficacy and efficiency of <em>Stacks</em> for inferring genetic loci and automated calling of genotypes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8"><em>Stacks</em> is implemented by component programs written in C++ and Perl, with the core algorithms parallelized using OpenMP libraries. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> lists <em>Stacks</em> components along with a brief description of each. The <em>Stacks</em> web interface is implemented in PHP and, along with several component programs, stores and retrieves data from a MySQL database. The web interface interacts with the database using the MDB2 Pear module. <em>Stacks</em> is available as open source software under the GPL license and can be downloaded from <a href="http://creskolab.uoregon.edu/stacks/">http://creskolab.uoregon.edu/stacks/</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460785/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460785/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1â</span> <span class="caption-title"><em>Stacks</em> component programs</span><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Simulating RAD-tags to test performance</h3><p id="p-9">The <em>Stacks</em> core component program is ustacks, which identifies unique loci <em>de novo</em>. To test ustacks, we created simulated datasets from the stickleback reference genome (BROAD S1, Ensembl version 59) by extracting 45,547 reads each 60 bp long in both directions at each SbfI restriction enzyme cut site (CCTGCA<sup>v</sup>GG) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS1.pdf">Figure S1</a>A). We re-diplodized the genome <em>in silico</em> by creating alleles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS1.pdf">Figure S1</a>B) into which we uniformly introduced single nucleotide polymorphisms (SNP) at a rate of 0.5%. We âsequencedâ each allele to a depth determined by a draw from a Poisson distribution at three different mean sequencing depths (10Ã, 20Ã, and 40Ã) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS1.pdf">Figure S1</a>C). For each âsequencedâ read, we simulated sequencing errors at a rate that increased linearly along the sequence to mimic Illumina reads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS1.pdf">Figure S1</a>D). We investigated three mean error rates (0.5%, 1%, and 3%) to cover normal to high error rates. Each simulation run involved 10 replicates. For each dataset, ustacks was executed setting the <em>within-individual distance</em> parameter to two nucleotides and the <em>stack-depth</em> parameter to three identical reads.</p></div><div id="sec-3" class="subsection"><h3>Constructing a dense zebrafish map</h3><p id="p-10">DNAs from the gynogenetic doubled haploid zebrafish HS mapping panel (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kelly <em>et al.</em> 2000</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Woods <em>et al.</em> 2005</a>) were prepared for RAD-tags according to <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> (2011)</a> and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Etter <em>et al.</em> (2011)</a>. Progeny were sequenced with 60 bp reads in three Illumina GAII lanes, resulting in 70,921,725 raw reads, of which 57,451,403 were retained after cleaning. Because DNA of the original female parent was no longer available, we combined all reads from her gynogenetic progeny to create a synthetic maternal genome and processed the content through <em>Stacks</em>. We executed the <em>Stacks</em> pipeline with a <em>stack-depth</em> parameter of three and a <em>within-individual distance</em> parameter of two and constructed a linkage map using JoinMap (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Van Ooijen 2006</a>). While <em>Stacks</em> has no limit to the number of markers it can handle, JoinMap is limited to about 8000 markers. To work around this deficiency in JoinMap, we subdivided <em>Stacks</em> output into overlapping datasets small enough for JoinMap to handle and then ran JoinMap to construct linkage groups, using markers shared in overlapping datasets to identify corresponding linkage groups. Linkage groupâspecific datasets with fewer than 8000 markers each were finally loaded into JoinMap to identify locus order.</p><p id="p-11">Besides comparing the RAD-tag map to a previously published meiotic map, we also aligned RAD-tag markers to the physical genome (Zv9, Ensembl version 61) by BLASTn. These searches used an e-value cutoff of 1 Ã 10<sup>â17</sup> (to allow for sequencing errors and for polymorphisms between the reference genome and the HS panel) and required a unique best hit to the reference genome or a top hit with a raw BLAST score at least an order of magnitude greater than the second best hit with 70% of the query sequence aligned. Genotypes for markers present in at least 36 of the 42 HS map cross individuals were exported into JoinMap 4.0 (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Van Ooijen 2006</a>). Linkage between markers, recombination rate, and map distances were calculated using the Kosambi mapping function and the maximum likelihood function in JoinMap. Markers were grouped at an initial logarithm of the odds (LOD) threshold of 7.0, and small linkage groups were incorporated using the strong cross-link feature of JoinMap at a minimum LOD of 5.0. Markers with strong segregation distortion or that appeared unlinked at LOD &lt; 5.0 were excluded.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><p id="p-12">We designed <em>Stacks</em> as a modular pipeline to efficiently curate and assemble large numbers of short-read sequences from multiple samples. <em>Stacks</em> identifies loci in a set of individuals, either <em>de novo</em> or aligned to a reference genome, and then genotypes each locus. <em>Stacks</em> incorporates a maximum likelihood statistical model to identify sequence polymorphisms and distinguish them from sequencing errors. <em>Stacks</em> employs a Catalog to record all loci identified in a population and matches individuals to that Catalog to determine which haplotype alleles are present at every locus in each individual. <em>Stacks</em> stores results in a MySQL database and displays them through a web interface that facilitates marker annotation. The database also allows linking markers to other sequence information, such as RNA-seq data (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mortazavi <em>et al.</em> 2008</a>). <em>Stacks</em> can export data as genotypes for JoinMap (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Van Ooijen 2006</a>) or R/qtl (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broman <em>et al.</em> 2003</a>) or as a set of observed haplotypes for a general population.</p><p id="p-13">Because <em>Stacks</em> was originally designed to build meiotic maps (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>), some pipeline terminology pertains to genetic mapping, but <em>Stacks</em> can be used for nearly any analysis using genomically localized short-read sequences. We describe here how the pipeline functions to build a genetic map <em>de novo</em>, and then how it can use a reference genome. Finally, we describe the testing of <em>Stacks</em> by simulation and by reconstructing a zebrafish genetic map. The <em>Stacks</em> component programs are discussed below and described in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><div id="sec-5" class="subsection"><h3>Building markers for a genetic map <em>de novo</em></h3><div id="sec-6" class="subsection"><h4>Overview:</h4><p id="p-14">The discussion here assumes that input to <em>Stacks</em> is composed of RAD-seq data (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) from the parents and progeny of a genetic cross. <em>Stacks</em> builds map markers by identifying loci and their constituent alleles in each individual (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1AâF</a>) and by creating a Catalog of parental loci (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1G</a>). <em>Stacks</em> then matches progeny against the Catalog (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1H</a>), which defines alleles at each locus in each individual. At each stage, <em>Stacks</em> exports outputs into a MySQL database.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/171/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stacks schematic. (A) The ustacks program forms stacks in an individual from short sequencing reads (cleaned by process_radtags.pl) that match exactly. (B) The ustacks program breaks down the sequence of each stack into k-mers and loads them into a dictionary. The ustacks program breaks down each stack again into k-mers and queries the k-mer Dictionary to create a list of potentially matching stacks, which can be visualized as nodes in a graph connected by the nucleotide distance between them. (C) ustacks merges matched stacks to form putative loci. (D) ustacks matches secondary reads that were not initially placed in a stack against putative loci to increase stack depth. An SNP model in ustacks checks each locus at each nucleotide position for polymorphisms. (E) ustacks calls a consensus sequence and records SNP and haplotype data. (F) The cstacks program loads stacks from the parents of a genetic cross into a Catalog to create a set of all possible loci in a mapping cross. (G) sstacks matches map cross progeny against the Catalog to determine the haplotypes at each locus in every individual in the cross." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1971575796" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Stacks&lt;/em&gt; schematic. (A) The ustacks program forms stacks in an individual from short sequencing reads (cleaned by process_radtags.pl) that match exactly. (B) The ustacks program breaks down the sequence of each stack into k-mers and loads them into a dictionary. The ustacks program breaks down each stack again into k-mers and queries the k-mer Dictionary to create a list of potentially matching stacks, which can be visualized as nodes in a graph connected by the nucleotide distance between them. (C) ustacks merges matched stacks to form putative loci. (D) ustacks matches secondary reads that were not initially placed in a stack against putative loci to increase stack depth. An SNP model in ustacks checks each locus at each nucleotide position for polymorphisms. (E) ustacks calls a consensus sequence and records SNP and haplotype data. (F) The cstacks program loads stacks from the parents of a genetic cross into a Catalog to create a set of all possible loci in a mapping cross. (G) sstacks matches map cross progeny against the Catalog to determine the haplotypes at each locus in every individual in the cross.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/171/F1.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/171/F1.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/171/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/171/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-15" class="first-child"><em>Stacks</em> schematic. (A) The ustacks program forms stacks in an individual from short sequencing reads (cleaned by process_radtags.pl) that match exactly. (B) The ustacks program breaks down the sequence of each stack into k-mers and loads them into a dictionary. The ustacks program breaks down each stack again into k-mers and queries the k-mer Dictionary to create a list of potentially matching stacks, which can be visualized as nodes in a graph connected by the nucleotide distance between them. (C) ustacks merges matched stacks to form putative loci. (D) ustacks matches secondary reads that were not initially placed in a stack against putative loci to increase stack depth. An SNP model in ustacks checks each locus at each nucleotide position for polymorphisms. (E) ustacks calls a consensus sequence and records SNP and haplotype data. (F) The cstacks program loads stacks from the parents of a genetic cross into a Catalog to create a set of all possible loci in a mapping cross. (G) sstacks matches map cross progeny against the Catalog to determine the haplotypes at each locus in every individual in the cross.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16"><em>Stacks</em> requires clean sequence data in FASTA or FASTQ output files (<em>e.g.</em>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kelley <em>et al.</em> 2010</a>) using the program process_radtags.pl. The process_radtags.pl program examines each read using a sliding window: if the average quality score within a window drops below 90% confidence [a Phred score of 10, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ewing and Green (1998)</a>], <em>Stacks</em> discards the read. Thus, <em>Stacks</em> accepts reads with isolated errors but detects reads with prolonged drops in quality and discards them. Uncalled nucleotides, nonexistent barcodes, or deficient restriction enzyme cut sites can also cause <em>Stacks</em> to exclude reads. <em>Stacks</em> can correct isolated errors in the restriction cut site sequence or in the barcode if the barcode is two or more nucleotides distant in sequence space from other barcodes used in the same sequencing library.</p></div><div id="sec-7" class="subsection"><h4>Identifying stacks, inferring loci:</h4><p id="p-17">The ustacks (unique stacks) program reads cleaned sequences and distills data into unique, exactly matching stacks by loading reads into a hash table (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). Unique stacks that contain fewer reads than a configurable threshold (the <em>stack-depth</em> parameter) are disassembled, and the reads are set aside because these stacks are indistinguishable from stacks generated with sequencing error. Reads in a stack are <em>primary reads</em>, and reads that are set aside are <em>secondary reads</em>. The ustacks program calculates the average depth of coverage, then identifies stacks that are two standard deviations above the mean and excludes them, along with all stacks that are one nucleotide apart from these extremely deep (<em>lumberjack</em>) stacks, which usually represent repetitive elements.</p><p id="p-18">Polymorphic genetic loci produce stacks that differ in few nucleotides. A k-mer search algorithm defines loci based on a user-specified distance between stacks (the <em>within-individual distance</em> parameter). This configurable distance depends on the datasetâs genetic properties, such as polymorphism rate and read length, and usually allows just a few nucleotide differences. To implement this comparison, ustacks breaks the sequence of each stack into a set of overlapping fragments of equal length <em>k</em> (k-mers) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Edgar 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Vinga and Almeida 2003</a>) (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). The first k-mer spans nucleotides 1 to <em>k</em>, the second 2 to <em>k</em>Â +Â 1, the third 3 to <em>k</em>Â +Â 2, and so on. The ustacks program automatically maximizes k-mer length according to the allowed nucleotide difference (longer k-mer lengths produce less promiscuous k-mers that require fewer comparisons to other reads) and loads k-mers into the Dictionary (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-19">The ustacks program queries the k-mer Dictionary with each k-mer from each stack to identify other stacks with matching k-mers. For pairs of stacks with sufficient numbers of matching k-mers, ustacks aligns the pair, naively matching nucleotide by nucleotide to verify that each pair of stacks is within the allowable nucleotide distance, and if they are, it records a match.</p><p id="p-20">The k-mer search algorithm transitively relates pairs of stacks. For example, if stacks 4 and 5 match and stacks 5 and 6 match with an allowable distance of one nucleotide, ustacks records two matching pairs (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1C</a>). Then ustacks merges all matching pairs, in this case merging 4, 5, and 6, even though 4 and 6 are two nucleotides apart. Merged stacks represent putative loci displayed as a graph with nodes representing unique stacks and edges weighted by the nucleotide distance between them (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>). In a full graph containing all stacks in the dataset, each putative locus represents a disconnected subgraph (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-21">In a diploid genetic cross, homozygous and heterozygous loci should contain one and two stacks, respectively. Allowing for some error, if more than three unique stacks have been merged, or if the coverage of the merged stack is more than two standard deviations above the mean coverage, ustacks shunts the stack to the deleveraging algorithm to determine which subset of these large stacks is most likely to represent a locus (see <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix 1</a>).</p><p id="p-22">The process of merging stacks is iterative. With a user-specified distance of three nucleotides between stacks, ustacks first finds stacks that are a single nucleotide different and merges them, then continues at a distance of two, and finally at a distance of three. At the end of each round, ustacks excludes lumberjack stacks. Secondary reads (2Â° reads, <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1E</a>) that were set aside earlier are now matched against putative loci using the k-mer search algorithm but with greater nucleotide distance (two nucleotides larger than the <em>within-individual distance</em> parameter by default). Secondary reads that do not have a best match to a unique defined locus are discarded. At the end of this stage, <em>Stacks</em> has constructed a set of putative loci from high-confidence unique stacks and has buttressed locus depth by adding secondary reads.</p></div><div id="sec-8" class="subsection"><h4>Inferring alleles and haplotypes:</h4><p id="p-23">The next step is to identify polymorphisms within loci. To detect polymorphisms and infer alleles (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1E</a>), ustacks examines each putative locus one nucleotide position at a time using a maximum likelihood framework (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hohenlohe <em>et al.</em> 2010</a>) (see <a id="xref-app-1-2" class="xref-app" href="#app-1">Appendix 1</a>). Some loci have polymorphisms at more than one position, but rarely in a short-read locus would a recombination event occur between two polymorphisms; hence, the configuration of SNPs at a locus represents a haplotype.</p><p id="p-24">SNPs and haplotypes are visualized as a two-dimensional matrix containing stacked sequencing reads (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1E</a>). <em>Stacks</em> identifies SNPs by examining the matrix column-wise and calls haplotypes by examining the matrix row-wise. Haplotypes that define alleles in each locus become genetic markers for subsequent analyses. Finally, <em>Stacks</em> determines a consensus sequence for each locus (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1F</a>).</p></div><div id="sec-9" class="subsection"><h4>Aggregating loci into a Catalog:</h4><p id="p-25">At this point, <em>Stacks</em> has constructed loci for one individual (the large top box, <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1AâF</a>). After <em>Stacks</em> has accomplished this task for a number of individuals (<em>e.g.</em>, the two parents in a genetic cross), cstacks (Catalog stacks, <a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1G</a>) synthesizes a Catalog of loci that appear in members of the population.</p><p id="p-26">The cstacks program reads the output from ustacks and merges loci into the Catalog. The first individual (say, the female parent of the cross) initializes the Catalog. Each additional individual is then merged into the Catalog in turn. Individual loci are matched to those already in the Catalog using the same k-mer search algorithm used by ustacks, except that each locus is represented in the k-mer dictionary by the set of k-mers resulting from each haplotype at that locus. When two loci match, cstacks merges their SNPs in the Catalog. If, however, those SNPs have conflicting alleles (for example, a fixed A in the Catalog and a segregating G/C in the locus that is being merged in), the merge fails and cstacks issues a warning. The cstacks program adjusts its haplotype calls based on the newly merged SNPs.</p><p id="p-27">The <em>between-individual distance</em> parameter of cstacks allows for mismatches while merging loci into the Catalog. If each parent is fixed for a different allele at a particular locus, cstacks can detect the mismatch and properly merge the loci. This property is particularly useful when fixed differences occur, as in crosses between inbred populations or between divergent species.</p></div><div id="sec-10" class="subsection"><h4>Matching the population against the Catalog:</h4><p id="p-28">To identify which locus/haplotype combinations are present in each individual in the population, sstacks (search stacks) matches every individual in the cross, including the parents and the progeny, against the Catalog (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1H</a>). The sstacks program constructs a hash table from every haplotype in the Catalog, compares all haplotypes from an individual, and records matches. Loci that match more than one Catalog locus are excluded because their true matching locus in the Catalog is ambiguous; multiple loci, however, can still uniquely match the same Catalog tag (these could represent, for example, repetitive sequences in the progeny that are not in the parents); users can elect to exclude these in later analyses.</p></div><div id="sec-11" class="subsection"><h4>Calling mappable markers:</h4><p id="p-29">At this stage, <em>Stacks</em> has identified haplotypes segregating in each individual in the population. Next <em>Stacks</em> identifies informative markers. The markers.pl program identifies mappable markers in the parents by downloading Catalog matches from the MySQL database and tallying up all the matching parental haplotypes. The markers.pl program characterizes parental loci into 10 classes of mappable markers, including loci that are segregating in the family due to variation in a single parent (ab/â, two alleles, a and b, in one parent, and a missing restriction site in the second parent), loci homozygous within parents but heterozygous between parents (aa/bb), loci with two (ab/aa), three (ab/ac), or four (ab/cd) haplotypes, as well as other related types (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460704/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460704/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2â</span> <span class="caption-title"><em>Stacks</em> marker types</span><div class="sb-div caption-clear"></div></div></div><p id="p-30"><em>Stacks</em> has now processed enough data to genotype map cross progeny, but it first must build an index in the MySQL database to unify the outputs of the previous analyses. The index_radtags.pl program performs this task and provides results to the web interface. This business logic is implemented in the denovo_map.pl program, which executes ustacks, cstacks, sstacks, markers.pl, and index_radtags.pl, and then uploads data to the database.</p><p id="p-31">At this point, <em>Stacks</em> calls genotypes from map cross progeny using genotypes.pl after specifying a particular map type (F1, F2, doubled haploid, or back cross) and an export type (JoinMap or R/qtl). The genotypes.pl program maps haplotypes in the progeny to the marker types detected in the parents. First, genotypes.pl downloads from the database the set of loci containing mappable markers recorded by markers.pl. It then maps haplotypes: if the first parent has haplotypes GA and AC, and the second parent has the GA haplotype, <em>Stacks</em> declares an ab/aa marker for this locus. The genotypes.pl program maps GA to a, and AC to b in the parents and checks progeny to see which haplotypes each contains, recording the genotypes (either ab or aa, in this case). Finally, genotypes.pl formats genotypes for use with the mapping program and outputs a properly formatted file. Users can specify the minimum number of matching progeny required for locus export.</p></div><div id="sec-12" class="subsection"><h4>Automated corrections:</h4><p id="p-32">Users can tell the genotypes.pl program to perform automated corrections for certain errors, including checking homozygous tags in the progeny to ensure that a SNP is not present. As described in <a id="xref-app-1-3" class="xref-app" href="#app-1">Appendix 1</a>, if the SNP model cannot identify a site as heterozygous or homozygous, the site is tentatively labeled a homozygote to facilitate matching to the Catalog in sstacks. If a second allele identified in the Catalog (<em>i.e.</em>, in the parents) is present in a progeny individual at a low frequency (less than 10% of reads in the stack), genotypes.pl corrects the genotype. Likewise, genotypes.pl removes a homozygous genotype call for a particular individual if the locus contains fewer than five reads supporting the genotype. Users can adjust these thresholds.</p></div><div id="sec-13" class="subsection"><h4>Iterative corrections:</h4><p id="p-33">The genotypes.pl program can optionally output a file formatted for loading into the database. The web interface allows users to manually correct genotypes. For example, a stack for Locus 1 in one of the progeny (<a id="xref-fig-1-18" class="xref-fig" href="#F1">Figure 1E</a>) might have just one A allele but 19 C alleles. <em>Stacks</em> would call the genotype as homozygous C, not being able to distinguish the single A from a sequencing error. But if a homozygous C call results in a double cross-over involving this single locus, the genotype is more likely to be heterozygous C/A with the A allele undersequenced. Users can make this correction through the web interface, and the corrected genotype will be included on the next execution of genotypes.pl.</p></div></div><div id="sec-14" class="subsection"><h3>Utilizing a reference genome</h3><p id="p-34"><em>Stacks</em> can identify loci not only <em>de novo</em> as described above but also using a reference genome. The two processes differ: instead of building stacks and loci from similar sequence reads, <em>Stacks</em> first aligns sequence reads to the reference genome using Bowtie (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Langmead <em>et al.</em> 2009</a>). And instead of invoking ustacks, we use pstacks (population stacks), which reads either Bowtie or SAM (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2009</a>) files and builds stacks based on alignment positions. SNP calling proceeds as before, and parameters exist for both cstacks and sstacks to build Catalog loci and to match against those loci, respectively, based on reference genome alignment positions instead of sequence distance. The business logic of this pipeline is embodied in the ref_map.pl program, which executes each stage and loads the resulting data into the database. Because pstacks and ustacks output the same file formats, the web interface displays them as in a genetic map.</p></div><div id="sec-15" class="subsection"><h3>Generating paired-end mini-contigs and adding other sequence sets</h3><p id="p-35">Mini-contigs from Rad-seq paired-end reads can be assembled and added to <em>Stacks</em>, thereby providing several hundred additional genomic nucleotides downstream of each marker that increase hits to expressed sequence tags libraries and thus connect markers to protein coding genes in other organisms (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Etter <em>et al.</em> 2011</a>). The sort_read_pairs.pl program collates paired-end reads associated with each stack and outputs a FASTA file for each locus in the catalog. Users can execute a program such as Velvet (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zerbino and Birney 2008</a>), which assembles reads in each FASTA file, to form contigs that can then be loaded into the <em>Stacks</em> MySQL database using the <em>Stacks</em> load_sequences.pl program.</p><p id="p-36">The load_sequences.pl program assumes that the sequence definition line, which is preceded by a âgreater than signâ (&gt;) for each sequence in a FASTA file, is a Catalog locus ID, and will store that sequence in the MySQL database linked to the Catalog locus. Therefore, in addition to mini-contigs, if ESTs are available or were constructed <em>de novo</em> using RNA-seq (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Mortazavi <em>et al.</em> 2008</a>), they can also be loaded into the database after they are matched to catalog loci using a program such as Bowtie or BLAST (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>). Any sequence data loaded into the MySQL database can later be exported in association with their markers using export_catalog.pl.</p><p id="p-37">In summary, the <em>Stacks</em> importing and exporting capabilities can associate <em>Stacks</em> markers with additional sequences, including mini-contigs and ESTs. These sequence sets can associate mappable loci in protein coding genes to orthologs in other species by BLAST searches, or to genomic contigs in an emerging reference genome.</p></div><div id="sec-16" class="subsection"><h3>Web-based interface</h3><p id="p-38"><em>Stacks</em> provides a web-based interface for viewing, annotating and correcting loci in a population (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The web interface displays haplotypes present in every individual (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) and clicking on a haplotype returns the appropriate stack (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The web interface, coupled with the MySQL database backend, provides extensive filtering capabilities, which facilitate the separation of useful data from background error, and it can export observed haplotypes as a Microsoft Excel document. This modular design allows <em>Stacks</em>, the database, and the web-based user interface to be located on the same or remote servers.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/171/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stacks web interface. (A) The interface allows a researcher to view observed haplotypes at each locus in all individuals. (B) Researchers can click each haplotype to view the stack itself. The interface provides extensive filtering facilities as well as the ability to annotate and export results in a number of formats, including Excel, JoinMap, and R/qtl." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1971575796" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Stacks&lt;/em&gt; web interface. (A) The interface allows a researcher to view observed haplotypes at each locus in all individuals. (B) Researchers can click each haplotype to view the stack itself. The interface provides extensive filtering facilities as well as the ability to annotate and export results in a number of formats, including Excel, JoinMap, and R/qtl.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/171/F2.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/171/F2.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/171/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/171/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460706" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-39" class="first-child"><em>Stacks</em> web interface. (A) The interface allows a researcher to view observed haplotypes at each locus in all individuals. (B) Researchers can click each haplotype to view the stack itself. The interface provides extensive filtering facilities as well as the ability to annotate and export results in a number of formats, including Excel, JoinMap, and R/qtl.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Simulation results</h3><p id="p-40">To test the ability of ustacks to identify loci, we simulated the RAD-seq process from the well-assembled genome sequence of threespine stickleback. We generated data at a per-allele mean sequencing depth of 10Ã, 20Ã, and 40Ã, and we varied the sequencing error rate from 0.5 to 3%. In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Reference Loci (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) represents loci present in the stickleback reference genome (Ensembl version 59) after the RAD-seq simulation, whereas Observed Stacks (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>) represents data discovered by <em>Stacks</em>. Results showed that, at low and moderate error rates, ustacks correctly reconstructed nearly all (86%) known loci (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A, B</a>). A comparison of Reference Loci and Observed Stacks, however (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A, B</a>), shows that ustacks collapsed repetitive sequences. Apart from repetitive sequences, less than 1% of stacks assembled incorrectly. At the highest error rate (3%) and lowest coverage (10Ã), about 51% of the known loci disappeared from the results, but at 20Ã coverage, <em>Stacks</em> identified most loci (81% correctly assembled), and at 40Ã coverage, the error rate had little effect on the number of identified loci (86% correctly assembled, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A, B</a>). Loci disappeared from the dataset likely due to low depth of coverage, which occurs by chance, as well as reads confounded by error.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/171/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stacks simulation results. The stickleback reference genome was digested in silico by SbfI, and 60 bp reads were made from each direction from the 22,774 cut sites at several different sequencing depths with several different error rates. The left panel shows the number of (A) loci, (B) stacks, and (C) SNPs observed in the Stacks output. Loci that Stacks assembled incorrectly are displayed in a dark color, whereas loci containing repetitive sequences are shown in a crosshatch pattern. A comparison of the number of loci present in the dataset (A) vs. the number of stacks reconstructed (B) showed that ustacks collapsed repetitive loci but correctly reconstructed nearly all other loci at low and moderate error rates or at high coverage. The right panel shows the number of reads with a certain number of sequencing errors that were incorporated into correct stacks, incorrect stacks, and unused reads for 20&#xD7; coverage and error rates of (D) 0.5%, (E) 1%, and (F) 3%. As errors accumulated, Stacks excluded more reads, lowering the overall depth, whereas some reads accumulated enough errors to be incorporated into stacks that appeared to be correctly assembled but, in fact, joined stacks representing loci from which they did not originate (indicated by reads with more errors than allowed by the k-mer matching algorithm, four errors in the simulation)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1971575796" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Stacks&lt;/em&gt; simulation results. The stickleback reference genome was digested &lt;em&gt;in silico&lt;/em&gt; by &lt;em&gt;SbfI&lt;/em&gt;, and 60 bp reads were made from each direction from the 22,774 cut sites at several different sequencing depths with several different error rates. The left panel shows the number of (A) loci, (B) stacks, and (C) SNPs observed in the &lt;em&gt;Stacks&lt;/em&gt; output. Loci that &lt;em&gt;Stacks&lt;/em&gt; assembled incorrectly are displayed in a dark color, whereas loci containing repetitive sequences are shown in a crosshatch pattern. A comparison of the number of loci present in the dataset (A) &lt;em&gt;vs.&lt;/em&gt; the number of stacks reconstructed (B) showed that ustacks collapsed repetitive loci but correctly reconstructed nearly all other loci at low and moderate error rates or at high coverage. The right panel shows the number of reads with a certain number of sequencing errors that were incorporated into correct stacks, incorrect stacks, and unused reads for 20&#xD7; coverage and error rates of (D) 0.5%, (E) 1%, and (F) 3%. As errors accumulated, &lt;em&gt;Stacks&lt;/em&gt; excluded more reads, lowering the overall depth, whereas some reads accumulated enough errors to be incorporated into stacks that appeared to be correctly assembled but, in fact, joined stacks representing loci from which they did not originate (indicated by reads with more errors than allowed by the k-mer matching algorithm, four errors in the simulation).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/171/F3.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/171/F3.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/171/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/171/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460718" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-41" class="first-child"><em>Stacks</em> simulation results. The stickleback reference genome was digested <em>in silico</em> by <em>SbfI</em>, and 60 bp reads were made from each direction from the 22,774 cut sites at several different sequencing depths with several different error rates. The left panel shows the number of (A) loci, (B) stacks, and (C) SNPs observed in the <em>Stacks</em> output. Loci that <em>Stacks</em> assembled incorrectly are displayed in a dark color, whereas loci containing repetitive sequences are shown in a crosshatch pattern. A comparison of the number of loci present in the dataset (A) <em>vs.</em> the number of stacks reconstructed (B) showed that ustacks collapsed repetitive loci but correctly reconstructed nearly all other loci at low and moderate error rates or at high coverage. The right panel shows the number of reads with a certain number of sequencing errors that were incorporated into correct stacks, incorrect stacks, and unused reads for 20Ã coverage and error rates of (D) 0.5%, (E) 1%, and (F) 3%. As errors accumulated, <em>Stacks</em> excluded more reads, lowering the overall depth, whereas some reads accumulated enough errors to be incorporated into stacks that appeared to be correctly assembled but, in fact, joined stacks representing loci from which they did not originate (indicated by reads with more errors than allowed by the k-mer matching algorithm, four errors in the simulation).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The simulation further showed that ustacks robustly identified SNPs, except at a high error rate and low depth of coverage, or when confounded by repetitive sequences (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a>). These data show that excess frugality or oversequencing are both wasteful. At the highest error rate and under the parameters of this simulation, moving from a per-allele depth of 10Ã to 20Ã gains 14,000 additional loci, whereas moving from 20Ã to 40Ã, which also doubles sequencing cost, nets only an additional 2400 loci.</p><p id="p-43">To further study locus drop out and error rates, we examined the effect of error rate on the distribution of errors per read. Because our simulation allowed tracking the origin of each read, we could deduce that, at the lowest error rate, reads that ended up in either correct stacks or in incorrect stacks contained no more errors than are allowed by the k-mer matching algorithm (four errors, in the worst case) (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>). In contrast, reads that could not be assigned to a stack (unused reads) tended to have more errors even at the lowest error rate (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3D</a>). At higher error rates, the number of unused reads increased greatly, from approximately 6000 at 0.5% to about 200,000 at 3%, thus decreasing stack depth (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3E, F</a>). Reads with more errors than allowed by the matching algorithm (again, four errors) accumulated in all three categories of reads at a 3% error rate (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3F</a>). The accumulation of error-riddled reads in correctly assembled stacks indicates that some reads suffered enough error to make them more similar to a different locus than to their original, known locus. These results demonstrate that raw sequence quality has a strong effect on the ability of <em>Stacks</em> to successfully reconstruct loci.</p><p id="p-44">Simulation data revealed the interacting effects of SNPs, sequencing depth, and error rate on stack quality. First, consider the effect of introducing SNPs into simulated reads. The known distribution of stacks with a particular sequencing depth (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, dotted red line) showed a peak at 40Ã, twice the average sequencing depth, because a diploid has two alleles at each locus. Without the introduction of SNPs or error, ustacks produced a rather erratic distribution of stack depth (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, gray line) with peaks at 80Ã and 120Ã due to the erroneous collapsing of two or three loci known to be different because of their known origin in the stickleback genome. Furthermore, ustacks collapsed over 6000 repetitive <em>SbfI</em> RAD loci in the stickleback genome into a smaller number of loci with very high depths of coverage, as indicated by the long right tail of the distribution that stretches far beyond the truncated display in the figure. The introduction of SNPs into the simulated reads at a rate of 0.5% caused a shoulder to appear on the distribution at 20Ã, half the depth of the main peak (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>, green line). These erroneous stacks of approximately 20Ã depth appeared because ustacks failed to find and join the alternative alleles for these stacks.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/171/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stacks depth of coverage distribution. (A) Correctly reconstructed stacks have a depth of coverage equal to twice the mean sequencing coverage because the simulation assumes diploid individuals. With no polymorphism or error (gray line), the depth of coverage distribution nearly matched the known simulation distribution (dotted red line), with the exception of repetitive loci, which created the long tail of the distribution to the right, which was truncated at 200&#xD7; but extends to 17,000&#xD7;. After adding SNPs, ustacks failed to reconstruct a small number of loci (green arrow) as shown by the increase in stacks with a depth of coverage equal to the sequencing mean depth. (B&#x2013;C) With the addition of sequencing error and increasing mean sequencing depth, most stacks were still properly reconstructed. Results showed a repeating pattern of improperly reconstructed stacks occurring at multiples of the mean sequencing depth corresponding to the number of loci improperly merged together. The increasing error rate caused a general loss of depth in the stacks (green vs. violet lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1971575796" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Stacks&lt;/em&gt; depth of coverage distribution. (A) Correctly reconstructed stacks have a depth of coverage equal to twice the mean sequencing coverage because the simulation assumes diploid individuals. With no polymorphism or error (gray line), the depth of coverage distribution nearly matched the known simulation distribution (dotted red line), with the exception of repetitive loci, which created the long tail of the distribution to the right, which was truncated at 200&#xD7; but extends to 17,000&#xD7;. After adding SNPs, ustacks failed to reconstruct a small number of loci (green arrow) as shown by the increase in stacks with a depth of coverage equal to the sequencing mean depth. (B&#x2013;C) With the addition of sequencing error and increasing mean sequencing depth, most stacks were still properly reconstructed. Results showed a repeating pattern of improperly reconstructed stacks occurring at multiples of the mean sequencing depth corresponding to the number of loci improperly merged together. The increasing error rate caused a general loss of depth in the stacks (green &lt;em&gt;vs.&lt;/em&gt; violet lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/171/F4.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/171/F4.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/171/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/171/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460782" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-45" class="first-child"><em>Stacks</em> depth of coverage distribution. (A) Correctly reconstructed stacks have a depth of coverage equal to twice the mean sequencing coverage because the simulation assumes diploid individuals. With no polymorphism or error (gray line), the depth of coverage distribution nearly matched the known simulation distribution (dotted red line), with the exception of repetitive loci, which created the long tail of the distribution to the right, which was truncated at 200Ã but extends to 17,000Ã. After adding SNPs, ustacks failed to reconstruct a small number of loci (green arrow) as shown by the increase in stacks with a depth of coverage equal to the sequencing mean depth. (BâC) With the addition of sequencing error and increasing mean sequencing depth, most stacks were still properly reconstructed. Results showed a repeating pattern of improperly reconstructed stacks occurring at multiples of the mean sequencing depth corresponding to the number of loci improperly merged together. The increasing error rate caused a general loss of depth in the stacks (green <em>vs.</em> violet lines).</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">To explore the effects of error rate on locus quality, we studied, at three levels of mean coverage, the effects of a typical low error rate of 0.5% and an unusually high error rate of 3.0% (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B, C</a>). At 10Ã mean coverage and 0.5% error, the distribution of correctly formed loci matched closely that of the true distribution, differing only by having somewhat fewer loci (a 14% reduction) than actually exist (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). This decrease came from two types of incorrectly joined stacks: some incorrect stacks occupied a peak at 10Ã, representing single stacks for which ustacks could not identify their true alternative alleles due to errors, and other incorrect stacks fell in a peak at 40Ã, representing cases in which ustacks inappropriately joined four stacks coming from two independent diploid loci. And still other stacks in the long tail represented the fusing of repetitive loci. An error rate six times higher (3%) reduced the number of correctly joined stacks to 49% of the true number and resulted in the loss of the peak at 10Ã found with the lower error rate. We conclude that high error rates cause inappropriate joining of stacks more frequently than incorrect failure to fuse stacks. With the introduction of errors at sequencing depths of 20Ã and higher, the distribution of correctly joined stacks shifted slightly to the left due to the accumulation of unused reads (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C, D</a>, green <em>vs.</em> purple lines). In sum, these simulations demonstrate remarkable fidelity of locus identification, even in the face of mounting errors, when the sequencing depth is between 20Ã and 40Ã.</p></div><div id="sec-18" class="subsection"><h3>A zebrafish genetic map</h3><p id="p-47">If <em>Stacks</em> works well, it should reconstruct a known genome map. To test this prediction, we constructed for <em>Danio rerio</em> a genetic map (RADmap) by using RAD-seq and <em>Stacks</em> to re-genotype a previously published doubled haploid mapping panel (HSmap, <a href="http://zfin.org/cgi-bin/webdriver?MIval=aa-crossview.apg&amp;OID=ZDB-REFCROSS-000320-1">http://zfin.org/cgi-bin/webdriver?MIval=aa-crossview.apg&amp;OID=ZDB-REFCROSS-000320-1</a>) that consists of 42 progeny (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Kelly <em>et al.</em> 2000</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Woods <em>et al.</em> 2005</a>). <em>Stacks</em> reconstructed the 25 zebrafish linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS2.pdf">Figure S2</a>), each with a length nearly identical to the original (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, 3186 cM in the HSmap <em>vs.</em> 3160 cM in the RADmap). With 7861 markers, our RADmap has nearly twice as many markers as the original HSmap (4073 markers), but it required less than 1% of the cost and took less than 1% of the time to genotype and construct. The RADmap and HSmap had nearly identical marker order (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS3.pdf">Figure S3</a>); differences could represent errors in either map.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/171/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Danio rerio RAD-tag map compared to the doubled haploid map. We constructed a RAD-seq genetic map of zebrafish (RADmap) using DNA from 42 individuals of the doubled haploid mapping panel (HSmap) that had been previously genotyped by microsatellites or single strand conformation polymorphism (Kelly et al. 2000; Woods et al. 2000; Woods et al. 2005). Stacks recovered the 25 zebrafish linkage groups (Figure S2) with lengths nearly identical to published values (3186 cM in the HSmap vs. 3160 cM in the RADmap). With 7861 markers, our RADmap had nearly twice as many markers as appeared in the HSmap (4073 markers). The insert shows the scale for marker density." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1971575796" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Danio rerio&lt;/em&gt; RAD-tag map compared to the doubled haploid map. We constructed a RAD-seq genetic map of zebrafish (RADmap) using DNA from 42 individuals of the doubled haploid mapping panel (HSmap) that had been previously genotyped by microsatellites or single strand conformation polymorphism (Kelly &lt;em&gt;et al.&lt;/em&gt; 2000; Woods &lt;em&gt;et al.&lt;/em&gt; 2000; Woods &lt;em&gt;et al.&lt;/em&gt; 2005). &lt;em&gt;Stacks&lt;/em&gt; recovered the 25 zebrafish linkage groups (Figure S2) with lengths nearly identical to published values (3186 cM in the HSmap &lt;em&gt;vs.&lt;/em&gt; 3160 cM in the RADmap). With 7861 markers, our RADmap had nearly twice as many markers as appeared in the HSmap (4073 markers). The insert shows the scale for marker density.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/171/F5.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/171/F5.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/171/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/171/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460737" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-48" class="first-child"><em>Danio rerio</em> RAD-tag map compared to the doubled haploid map. We constructed a RAD-seq genetic map of zebrafish (RADmap) using DNA from 42 individuals of the doubled haploid mapping panel (HSmap) that had been previously genotyped by microsatellites or single strand conformation polymorphism (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Kelly <em>et al.</em> 2000</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Woods <em>et al.</em> 2000</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Woods <em>et al.</em> 2005</a>). <em>Stacks</em> recovered the 25 zebrafish linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000240/-/DC1/FigureS2.pdf">Figure S2</a>) with lengths nearly identical to published values (3186 cM in the HSmap <em>vs.</em> 3160 cM in the RADmap). With 7861 markers, our RADmap had nearly twice as many markers as appeared in the HSmap (4073 markers). The insert shows the scale for marker density.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">A comparison of the zebrafish RADmap to the sequenced reference genome showed alignment of 5787 RADmap markers and revealed that marker order for the RADmap and the physical assembly generally agreed (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). An additional 157 mapped RADmap markers aligned to genomic scaffolds that are currently unordered in the Zv9 reference genome, thus positioning these errant contigs into the reference genome.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/3/171/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RADmap marker order is consistent with the sequenced zebrafish genome. A specific region on LG20 with no recombination in the RADmap spanned almost 10 Mb in the physical genome (inset). This recombination suppression could be due to a heterozygous inversion present in the genome of the mother of the gynogenetic HS mapping panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1971575796" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;RADmap marker order is consistent with the sequenced zebrafish genome. A specific region on LG20 with no recombination in the RADmap spanned almost 10 Mb in the physical genome (inset). This recombination suppression could be due to a heterozygous inversion present in the genome of the mother of the gynogenetic HS mapping panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/3/171/F6.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/3/171/F6.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/3/171/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/3/171/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6â</span> <p id="p-50" class="first-child">RADmap marker order is consistent with the sequenced zebrafish genome. A specific region on LG20 with no recombination in the RADmap spanned almost 10 Mb in the physical genome (inset). This recombination suppression could be due to a heterozygous inversion present in the genome of the mother of the gynogenetic HS mapping panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">A plot of the RADmap <em>vs.</em> the reference genome identified several regions of low recombination rate per physical distance. One region in LG20 showed recombination suppression in the RADmap over a region of about 10 Mb (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, inset) that could be due to a heterozygous inversion in the mother of the HS mapping panel, who was a heterozygote of the clonal C32 line and the highly inbred SJD strains (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nechiporuk <em>et al.</em> 1999</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Streisinger <em>et al.</em> 1986</a>). This hypothesis, generated by the extraordinarily high density of RAD markers, warrants further investigation. LG4, which is chromosome 3 in the physical genome (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Phillips <em>et al.</em> 2006</a>), has a mostly heterochromatic long arm, whose repetitive elements would produce lumberjack stacks that would be excluded from analysis. Markers off the diagonal of <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> could be due to errors either in the RADmap, in the BLAST assignment of RADmap markers to the physical genome, or in the physical assembly. These results show that RAD-tag markers can recapitulate a known genetic map at greater density and with less time and expense than methodologies currently in use.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-52">Analyzing RAD-seq data with <em>Stacks</em> can recover hundreds to tens of thousands of informative markers that describe the genetics of a population. <em>Stacks</em> has been used to generate an ultradense genetic map using the F1 offspring of wild-caught spotted gar (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>), to examine the phylogeographic distribution of the mosquito, <em>Wyeomyia smithii</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Emerson <em>et al.</em> 2010</a>), and to generate informative SNPs in trout populations (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hohenlohe <em>et al.</em> 2011</a>). The zebrafish map constructed <em>de novo</em> here and compared with a well-assembled sequenced genome demonstrates the rapid nature of this approach that took a few weeks of part-time effort, whereas a previous map using the same DNAs required several years to construct, cost 100 times as much, and had half the number of markers.</p><p id="p-53">Having shown the biological precision of <em>Stacks</em>, we now discuss how to increase its informative value by iteratively improving data and associating loci to additional sequence data. <a id="xref-app-2-1" class="xref-app" href="#app-2">Appendix 2</a> presents methods to optimize <em>Stacks</em>, including alternative strategies to build the Catalog and to adjust important <em>Stacks</em> parameters.</p><div id="sec-20" class="subsection"><h3><em>Stacks</em> reveals loci <em>en mass</em></h3><p id="p-54">Not all RAD-seq loci appear in all individuals due to polymorphisms in restriction enzyme cut sites, stochastic events related to sequencing (as our simulations showed), PCR errors, or sequencing errors. Loci that appear in a large number of individuals in a population or in a large number of map cross progeny are the most reliable. Once <em>Stacks</em> has generated a set of markers, it is most effective to select markers supported in as many progeny as possible by using the set of filters provided in the web interface (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) or by specifying a minimum number of progeny when exporting genotypes.</p></div><div id="sec-21" class="subsection"><h3>Iterative corrections</h3><p id="p-55">One of the key attributes of <em>Stacks</em> is its convenient web interface, which supports manual corrections. Iterative corrections can make significant improvements in a genetic map based on the principle that double recombinants in a short genetic distance are unlikely events. Manual examination of markers that expand the map can identify, correct, or remove troublesome genotypes, followed by re-exporting data and reconstructing the map. Reiteration can provide a genetic map with strong statistical support on all linkage groups.</p></div><div id="sec-22" class="subsection"><h3><em>Stacks</em> and genome duplication</h3><p id="p-56">Genome duplication events, like those that occurred in the stems of vertebrate, teleost, salmonid, and flowering plant lineages (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allendorf and Danzmann 1997</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amores <em>et al.</em> 1998</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dehal and Boore 2005</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Koop <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jiao <em>et al.</em> 2011</a>), result in paralogs that are initially identical but diverge over time. In some cases, <em>Stacks</em> might erroneously confuse paralogs that have nearly identical sequences with alleles of the same locus. Fortunately, <em>Stacks</em> can detect âovermergingâ of paralogous stacks because all individuals homozygous for a specific sequence at one paralog and homozygous for a slightly different sequence in the other paralog would appear to be heterozygotes for the relevant SNP. In contrast, a meiotic mapping population that is segregating a SNP at one locus or a population in Hardy-Weinberg equilibrium would have, on average, only about half of the individuals being heterozygotes. In addition, a diploid individual will never have more than two alleles of a single locus, so if individuals are discovered with three or more alleles, paralogs are likely to blame. <em>Stacks</em> can detect markers in which observed heterozygosity is significantly different than expected and flag them. The problem of confusing paralogs with allelic variants is evolutionarily transitory. Identical stacks (as might be found for paralogs in recent tetraploids) are uninformative and donât cause a problem; furthermore, a few neutral mutations are sufficient for <em>Stacks</em> to identify paralogous loci, particularly if the user sets the <em>within-distance</em> parameter to a small value. In a recent study of trout populations, <em>Stacks</em> flagged loci that differed from Hardy-Weinberg expectations, thereby successfully removing the effects of the recent (25â100 million years ago) salmonid genome duplication (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hohenlohe <em>et al.</em> 2011</a>).</p><p id="p-57">The stringency of applied filters should depend on a number of factors that reflect both the biology of the species (<em>e.g.</em>, time since duplication) and the experimental goals (<em>e.g.</em>, trade-off between marker number and marker reliability). In some cases, however, the indiscriminate filtering of loci that do not appear to meet Hardy-Weinberg expectations can lead to erroneous conclusions. For example, in a recent moss linkage map, 45% of the loci exhibited segregation distortion, likely due to lethal interactions between distant loci (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mcdaniel <em>et al.</em> 2007</a>). Thus, while <em>Stacks</em> can flag markers that do not fit expectations, careful interpretation is required to understand the biology of the species.</p></div><div id="sec-23" class="subsection"><h3>Increasing the informative value of <em>Stacks</em></h3><p id="p-58">Given the high marker density of a RAD-seq genetic map and the fact that those markers consist of genomic sequence, BLAST searches can associate markers or mini-contigs to ESTs, such as those generated by RNA-seq, or to orthologous genes in other species (<a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>). These features make comparative genomics a natural extension of a <em>Stacks</em> analysis. The <em>Stacks</em> database contains several tables supporting the importation of paired-end mini-contigs or RNA-seq-assembled ESTs. A table also exists to store BLAST hits from markers, mini-contigs, or ESTs, and the web interface displays these data. Combined with programs in <em>Stacks</em> that import and export these sequences from the database, it becomes straightforward to perform conserved synteny analyses on genetic maps (see <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>). Mini-contigs can be exported to help design PCR primers for marker-assisted selection or to isolate genomic clones for specific markers in the genetic map. In addition, <em>Stacks</em> facilitates the alignment of genomic contigs from an emerging, often highly fragmented, reference genome assembly to the genetic map, thereby creating linkage groupâbased scaffolds from the physical contigs.</p><p id="p-59">Nearly a century after the first genetic maps (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sturtevant 1913</a>), <em>Stacks</em>, coupled with massively parallel DNA sequencing, makes the genetic map relevant again. Because <em>Stacks</em> and RAD-seq rapidly and inexpensively provide unprecedented numbers of genetic markers, fragmented genome assemblies can be ordered, and variation existing in single individuals taken directly from the wild can provide genetic maps with genome-wide comparative information. In addition, <em>Stacks</em> makes genome-wide association studies (GWAS) more tractable in nonmodel species because the enormous linkage map provides a framework for the analysis of population genomic data. <em>Stacks</em> is available for download, along with a set of example data, tutorials, and other documentation at <a href="http://creskolab.uoregon.edu/stacks/">http://creskolab.uoregon.edu/stacks/</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We thank S. Bassham for coining the term â<em>stacks</em>.â The <em>Stacks</em> software, and its description in this manuscript, was improved by discussions with other members of the Postlethwait and Cresko laboratories. This work was supported by National Institutes of Health/National Center for Research Resources grant R01RR020833 (J.H.P.), National Institutes of Health/Ruth L. Kirschstein National Research Service Award postdoctoral fellowship 1F32GM095213-01 (J.M.C.), National Science Foundation grant DEB-0919090 (W.C.), and National Institutes of Health grant R24GM079486 (W.C.).</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1</h2><div class="section other1" id="sec-24"><h3>ALGORITHMIC DETAILS</h3><div id="sec-25" class="subsection other2"><h4>The deleveraging algorithm</h4><p id="p-62">Given a connected graph of unique stacks, the deleveraging algorithm contained in ustacks determines which subset of stacks is most likely to represent a locus. This heuristic program assumes that stacks originating from the same locus have approximately the same depth of coverage; for example, a true stack should have greater depth of coverage than stacks resulting from sequencing errors but less depth than stacks derived from repetitive elements. To achieve the goal of identifying the true components of the locus, the deleveraging algorithm first scales edges in the graph (which reflect nucleotide distance) by the log of the difference in depth of coverage between nodes. Nodes with a small nucleotide distance and relatively equal depths of coverage will be connected by an edge weighted with a very small distance in the graph, whereas nodes separated by a large nucleotide distance and/or a large difference in depth of coverage will be connected by an edge weighted by a large distance. <em>Stacks</em> feeds these scaled distances into a hierarchical clustering algorithm (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Hoon 2010</a>) that arranges stacks in a tree according to distances that separate stacks from one another. This resulting tree is split into two groups, one representing the most closely related nodes in the tree, and the other containing the rest. For example, given a set of five stacks that have been grouped into a putative locus, where two stacks have coverage depth similar to that of the mean and where the other three stacks are shallow due to sequencing errors, the deleveraging algorithm is successful if it can separate the two real stacks from the stacks with sequencing errors. After the deleveraging algorithm executes, <em>Stacks</em> checks the maximum distance between the nodes in both clusters, and if this distance is greater than the user-specified nucleotide distance (the <em>within-individual distance</em> parameter), the lumberjack stack is blacklisted and excluded from use in the remainder of the pipeline; otherwise, the grouped loci persist in the dataset.</p></div><div id="sec-26" class="subsection other3"><h4>Maximum likelihood SNP model</h4><p id="p-63">At each nucleotide position in a locus, <em>Stacks</em> adapts the single-nucleotide, diploid genotyping method described by (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hohenlohe <em>etÂ al.</em> 2010</a>). This approach considers the counts of each of the four possible nucleotides in a multinomial sampling and uses a likelihood ratio test to assess the significance of the most likely genotype. The sequencing error rate is estimated implicitly by maximum likelihood at each position, and a significance level of Î± = 0.05 is used (without correction for multiple testing) to assign a diploid genotype (homozygote or heterozygote) at each position for each individual (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Hohenlohe <em>etÂ al.</em> 2010</a>). If this likelihood ratio test is not significant, due either to low coverage or to read counts that lie between expectations for a heterozygote and a homozygote with error, then the model considers the location to be a homozygote for the most commonly observed nucleotide. This procedure avoids uncalled bases in the subsequent merging of stacks. The genotypes.pl program later corrects these genotypes by using information across individuals in the dataset. Future versions of <em>Stacks</em> will allow prior distributions to be placed on the sequencing error parameter and the significance level Î± to accommodate known attributes of the dataset, and it will dynamically modify thresholds that determine when a particular genotype call is significant.</p></div></div></div><div class="section app" id="app-2"><h2 class="">Appendix 2</h2><div class="section other4" id="sec-27"><h3>IMPORTANT PARAMETERS; USAGE STRATEGIES</h3><div id="sec-28" class="subsection other5"><h4>Catalog construction</h4><p id="p-64">The cstacks program, which builds the Catalog, can accept data from an arbitrary number of individuals. The Catalog was designed to contain all possible loci that might appear in an analysis. The merging of each additional individual into the Catalog, however, brings with it a small number of erroneous stacks. In the case of a genetic map, the choice of what to load into the Catalog is simple: the parents of the cross contain all possible loci present in the progeny and thus together act as a natural limit to the number of loci that will be in their progeny; hence, the parents should appear in the Catalog. In a population genomic investigation, however, loading individuals into the Catalog is likely to increase the number of erroneous loci in the Catalog as a function of the number of individuals loaded. One strategy around this problem is to create a âsuperparent,â a virtual individual created by combining reads from many individuals. The ustacks program builds stacks from the superparent normally, and cstacks loads them into the Catalog. If the investigated individuals are members of different subpopulations (say, lake <em>vs.</em> marine fish or different plant ecotypes), then a superparent could be built from each subpopulation with several superparents loaded into the Catalog. This approach works well for correctly identifying loci with a moderate to high frequency ofÂ minor alleles in the population sample, which is likely to be a primary application of <em>Stacks</em>. Note, however, that the multinomial sampling model in the genotyping algorithm applies strictly to single diploid individuals, where alternative alleles in a single heterozygous individual are expected to appear in a stack in roughly equal frequencies. Rare alleles in a population might be erroneously excluded by this approach because the program treats their SNPs as sequencing errors in the superparent. In the future, we plan to eliminate stacks representing errors by comparing loci across a population and eliminating very low frequency haplotypes at a particular locus, which may obviate the need to construct superparents.</p></div><div id="sec-29" class="subsection other6"><h4>Important parameter values</h4><p id="p-65">Users can specify three major parameters when running the <em>Stacks</em> pipeline <em>de novo</em>.</p><p id="p-66">The <em>stack-depth</em> parameter controls theÂ minimum number of identical reads required to form a stack. In our empirical work, we have had success using aÂ minimum depth of three, although this number should scale along with the number of raw reads available.</p><p id="p-67">The second parameter, the <em>within-individual distance</em> parameter, is the maximum number of nucleotide mismatches allowed between stacks before fusing two or more stacks into a locus. If this parameter is set too low, loci containing multiple SNPs per haplotype will not be recovered. If it is set too high, <em>Stacks</em> will incorrectly combine distinct genetic loci that happen to be near each other in sequence space. The default value is two nucleotides, but we have had success using a value of up to four. This parameter must be sensitive to the biology (for example, duplication history) of the species.</p><p id="p-68">The third parameter, called the <em>between-individual distance</em> parameter, is the number of mismatches allowed between loci in the Catalog. The <em>between-individual distance</em> parameter allows <em>Stacks</em> to detect loci that are homozygous in individuals but polymorphic between individuals. By default, <em>Stacks</em> sets catalog-mismatch to zero; increasing the catalog-mismatch limit can have the same potentially negative effects as increasing the <em>within-individual distance</em> parameter. The most appropriate value for the <em>between-individual distance</em> parameter varies with the evolutionary distance of the parents of the cross or of the members of the population being examined. In a standard F2 mapping cross, an F1 pseudo-testcross [as in the gar map (<a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Amores <em>etÂ al.</em> 2011</a>)], or the zebrafish doubled haploid cross, zero is appropriate for the <em>between-individual distance</em> parameter. In contrast, with highly divergent populations, a higher value might be more appropriate because different populations might be fixed for different alleles of the same locus. A reasonable approach is to set the <em>between-individual distance</em> parameter to the same value as the <em>within-individual distance</em> parameter if one expects fixed alleles in the parents of a cross or in members of a population.</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">IACUC approval number #08-13RR</p></li><li class="fn" id="fn-2"><p id="p-61"><em>Communicating editor: D-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-04-11"><span class="received-label">Received </span>April 11, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-05-16"><span class="accepted-label">Accepted </span>May 16, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2011 Catchen <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Secondary tetrasomic segregation of MDH-B and preferential pairing of homeologues in rainbow trout.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1083</span>â<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1083%26rft.epage%253D1092%26rft.atitle%253DSecondary%2BTetrasomic%2BSegregation%2Bof%2BMDH-B%2Band%2BPreferential%2BPairing%2Bof%2BHomeologues%2Bin%2BRainbow%2BTrout%26rft_id%253Dinfo%253Apmid%252F9093860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NS80LzEwODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.2" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>â<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.3" data-doi="10.1126/science.282.5394.1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Force</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joly</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amemiya</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Zebrafish hox clusters and vertebrate genome evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">1711</span>â<span class="cit-lpage">1714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAmores%26rft.auinit1%253DA.%26rft.volume%253D282%26rft.issue%253D5394%26rft.spage%253D1711%26rft.epage%253D1714%26rft.atitle%253DZebrafish%2Bhox%2BClusters%2Band%2BVertebrate%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5394.1711%26rft_id%253Dinfo%253Apmid%252F9831563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5NC8xNzExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.4" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontenot</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome evolution and meiotic maps by massively parallel DNA sequencing: spotted gar, an outgroup for the teleost genome duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>â<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D188%26rft.spage%253D799%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.111.127324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21828280&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293700000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keehan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppieters</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A high density linkage map of the bovine genome.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DArias%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D10%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DA%2Bhigh%2Bdensity%2Blinkage%2Bmap%2Bof%2Bthe%2Bbovine%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F19393043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19393043&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.6" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.7" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Å.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">889</span>â<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-web" id="cit-1.3.171.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Hoon</span>  <span class="cit-name-given-names">M. J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The C Clustering Library for cDNA microarray data</span>. <span class="cit-comment">Available at: <a href="http://bonsai.hgc.jp/~mdehoon/software/cluster/software.htm#source">http://bonsai.hgc.jp/â¼mdehoon/software/cluster/software.htm#source</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.9" data-doi="10.1371/journal.pbio.0030314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boore</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Two rounds of whole genome duplication in the ancestral vertebrate.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDehal%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De314%26rft.epage%253De314%26rft.atitle%253DTwo%2Brounds%2Bof%2Bwhole%2Bgenome%2Bduplication%2Bin%2Bthe%2Bancestral%2Bvertebrate.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030314%26rft_id%253Dinfo%253Apmid%252F16128622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16128622&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.10" data-doi="10.1093/nar/gkh180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Local homology recognition and distance measures in linear time using compressed amino acid alphabets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">380</span>â<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D380%26rft.epage%253D385%26rft.atitle%253DLocal%2Bhomology%2Brecognition%2Band%2Bdistance%2Bmeasures%2Bin%2Blinear%2Btime%2Busing%2Bcompressed%2Bamino%2Bacid%2Balphabets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh180%26rft_id%253Dinfo%253Apmid%252F14729922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14729922&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188988700055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.11" data-doi="10.1073/pnas.1006538107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merz</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Resolving postglacial phylogeography using high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">107</span>(<span class="cit-issue">37</span>): <span class="cit-fpage">16196</span>â<span class="cit-lpage">16200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006538107%26rft_id%253Dinfo%253Apmid%252F20798348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzM3LzE2MTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.12" data-doi="10.1371/journal.pone.0018561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Local <em>de novo</em> assembly of RAD paired-end contigs using short sequencing reads.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e18561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEtter%26rft.auinit1%253DP.%2BD.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De18561%26rft.epage%253De18561%26rft.atitle%253DLocal%2Bde%2Bnovo%2Bassembly%2Bof%2BRAD%2Bpaired-end%2Bcontigs%2Busing%2Bshort%2Bsequencing%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018561%26rft_id%253Dinfo%253Apmid%252F21541009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21541009&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.13" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Base-calling of automated sequencer traces using Phred. II. Error probabilities.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">186</span>â<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft_id%253Dinfo%253Apmid%252F9521922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9521922&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amish</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Next-generation RAD sequencing identifies thousands of SNPs for assessing hybridization between rainbow and westslope cutthroat trout.</span> <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">117</span>â<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D11%26rft.spage%253D117%26rft_id%253Dinfo%253Apmid%252F21429168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21429168&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.15" data-doi="10.1371/journal.pgen.1000862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiffler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1000862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000862%26rft.epage%253De1000862%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bparallel%2Badaptation%2Bin%2Bthreespine%2Bstickleback%2Busing%2Bsequenced%2BRAD%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000862%26rft_id%253Dinfo%253Apmid%252F20195501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20195501&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.16" data-doi="10.1038/nature09916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickett</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyampalayam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanderbali</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landherr</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Ancestral polyploidy in seed plants and angiosperms.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">473</span>: <span class="cit-fpage">97</span>â<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJiao%26rft.auinit1%253DY.%26rft.volume%253D473%26rft.issue%253D7345%26rft.spage%253D97%26rft.epage%253D100%26rft.atitle%253DAncestral%2Bpolyploidy%2Bin%2Bseed%2Bplants%2Band%2Bangiosperms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09916%26rft_id%253Dinfo%253Apmid%252F21478875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478875&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290218300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.17" data-doi="10.1186/gb-2010-11-4-116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Quake: quality-aware detection and correction of sequencing errors.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">R116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKelley%26rft.auinit1%253DD.%2BR.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253DR116%26rft.epage%253DR116%26rft.atitle%253DQuake%253A%2Bquality-aware%2Bdetection%2Band%2Bcorrection%2Bof%2Bsequencing%2Berrors.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-4-116%26rft_id%253Dinfo%253Apmid%252F21114842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-4-116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21114842&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.18" data-doi="10.1101/gr.10.4.558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngo-Hazelett</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardozo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genetic linkage mapping of zebrafish genes and ESTs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">558</span>â<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKelly%26rft.auinit1%253DP.%2BD.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D558%26rft.epage%253D567%26rft.atitle%253DGenetic%2BLinkage%2BMapping%2Bof%2BZebrafish%2BGenes%2Band%2BESTs%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.558%26rft_id%253Dinfo%253Apmid%252F10779498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTU4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.19" data-doi="10.1186/1471-2164-9-545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Schalburg</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieph</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A salmonid EST genomic study: genes, duplications, phylogeny and microarrays.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKoop%26rft.auinit1%253DB.%2BF.%26rft.volume%253D9%26rft.spage%253D545%26rft.epage%253D545%26rft.atitle%253DA%2Bsalmonid%2BEST%2Bgenomic%2Bstudy%253A%2Bgenes%252C%2Bduplications%252C%2Bphylogeny%2Band%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-545%26rft_id%253Dinfo%253Apmid%252F19014685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19014685&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.20" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.21" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.22" data-doi="10.1534/genetics.107.075424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDaniel</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A linkage map reveals a complex basis for segregation distortion in an interpopulation cross in the moss Ceratodon purpureus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2489</span>â<span class="cit-lpage">2500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D2489%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075424%26rft_id%253Dinfo%253Apmid%252F17603096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.075424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17603096&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249530000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.23" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">240</span>â<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.24" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">621</span>â<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.25" data-doi="10.1101/gr.9.12.1231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nechiporuk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finney</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keating</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Assessment of polymorphism in zebrafish mapping strains.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1231</span>â<span class="cit-lpage">1238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNechiporuk%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D1231%26rft.epage%253D1238%26rft.atitle%253DAssessment%2Bof%2BPolymorphism%2Bin%2BZebrafish%2BMapping%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.12.1231%26rft_id%253Dinfo%253Apmid%252F10613846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjkvMTIvMTIzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.26" data-doi="10.1159/000093332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morascha</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Assignment of zebrafish genetic linkage groups to chromosomes.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">114</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">155</span>â<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%2BB.%26rft.volume%253D114%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D162%26rft.atitle%253DAssignment%2Bof%2Bzebrafish%2Bgenetic%2Blinkage%2Bgroups%2Bto%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000093332%26rft_id%253Dinfo%253Apmid%252F16825768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000093332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16825768&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.27" data-doi="10.1126/science.8171321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Midson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">A genetic linkage map for the zebrafish.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">264</span>(<span class="cit-issue">5159</span>): <span class="cit-fpage">699</span>â<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPostlethwait%26rft.auinit1%253DJ.%26rft.volume%253D264%26rft.issue%253D5159%26rft.spage%253D699%26rft.epage%253D703%26rft.atitle%253DA%2Bgenetic%2Blinkage%2Bmap%2Bfor%2Bthe%2Bzebrafish%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8171321%26rft_id%253Dinfo%253Apmid%252F8171321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjQvNTE1OS82OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.28" data-doi="10.1006/geno.1999.5824"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimoda</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapik</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziniti</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Zebrafish genetic map with 2000 microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">58</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">219</span>â<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DShimoda%26rft.auinit1%253DN.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D219%26rft.epage%253D232%26rft.atitle%253DZebrafish%2Bgenetic%2Bmap%2Bwith%2B2000%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.1999.5824%26rft_id%253Dinfo%253Apmid%252F10373319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.1999.5824&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10373319&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081149500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.29" data-doi="10.1101/gad.1864110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Personal genome sequencing: current approaches and challenges.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">423</span>â<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSnyder%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D5%26rft.spage%253D423%26rft.epage%253D431%26rft.atitle%253DPersonal%2Bgenome%2Bsequencing%253A%2Bcurrent%2Bapproaches%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1864110%26rft_id%253Dinfo%253Apmid%252F20194435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjQvNS80MjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzMvMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Streisinger</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knauber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dower</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">Segregation analyses and gene-centromere distances in zebrafish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">311</span>â<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D112%26rft.spage%253D311%26rft_id%253Dinfo%253Apmid%252F3455686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3455686&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986AYA8400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.31" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span>â<span class="cit-article-title">The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>â<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">An ultradense genetic recombination map for Brassica napus, consisting of 13551 SRAP markers.</span> <abbr class="cit-jnl-abbrev">TAG Theoretical and Applied Genetics</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1305</span>â<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D114%26rft.spage%253D1305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-book" id="cit-1.3.171.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-source">JoinMap 4.0: Software for the Calculation of Genetic Linkage Maps in Experimental Populations</span>. <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.34" data-doi="10.1534/genetics.106.055871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrzejewski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrena</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Construction of a 10,000-marker ultradense genetic recombination map of potato: providing a framework for accelerated gene isolation and a genomewide physical map.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1075</span>â<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D173%26rft.spage%253D1075%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.055871%26rft_id%253Dinfo%253Apmid%252F16582432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.055871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16582432&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238650900047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.35" data-doi="10.1093/bioinformatics/btg005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Alignment-free sequence comparison - a review.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">513</span>â<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DVinga%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D513%26rft.epage%253D523%26rft.atitle%253DAlignment-free%2Bsequence%2Bcomparison--a%2Breview%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg005%26rft_id%253Dinfo%253Apmid%252F12611807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12611807&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F3%2F171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181410700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.36" data-doi="10.1101/gr.10.12.1903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngo-Hazelett</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">A comparative map of the zebrafish genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1903</span>â<span class="cit-lpage">1914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWoods%26rft.auinit1%253DI.%2BG.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D1903%26rft.epage%253D1914%26rft.atitle%253DA%2BComparative%2BMap%2Bof%2Bthe%2BZebrafish%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.12.1903%26rft_id%253Dinfo%253Apmid%252F11116086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMi8xOTAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.37" data-doi="10.1101/gr.4134305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedlander</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The zebrafish gene map defines ancestral vertebrate chromosomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1307</span>â<span class="cit-lpage">1314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4134305%26rft_id%253Dinfo%253Apmid%252F16109975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzkvMTMwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMy8xNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.3.171.38" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">821</span>â<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8zLzE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/3/171.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/161" title="polo Is Identified as a Suppressor of bubR1 Nondisjunction in a Deficiency Screen of the Third Chromosome in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/3/183" title="Mutation Rate Inferred From Synonymous Substitutions in a Long-Term Evolution Experiment With Escherichia coli" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/3"><span>Volume 1 Issue 3, August 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/3.cover.gif?itok=eI_pz32_" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/3/171.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-45X__P5XciSIGVUCPP29doNCAvg8y3f6Epe5Jiw-ois" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/3/171" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/3/171" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/3/171" class="active" data-icon-position="" data-hide-link-title="0">Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459169" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ffsS4khMtqqM3mzErur9wvR8wYYFMSYRfc_D9ai4YeY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/3/171.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/3/171" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-mVJoJtM2toFue3fxdWc9rxLkQhnZwDay-bdpXL2Fzhg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/3/171" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/3/171.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459169" id="node459169--2" data-pisa="ggg;1/3/171" data-pisa-master="ggg;g3.111.000240" data-apath="/ggg/1/3/171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Stacks</em>: Building and Genotyping Loci <em>De Novo</em> From Short-Read Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Julian M. Catchen</span>, <span class="highwire-citation-author" data-delta="1">Angel Amores</span>, <span class="highwire-citation-author" data-delta="2">Paul Hohenlohe</span>, <span class="highwire-citation-author" data-delta="3">William Cresko</span> and <span class="highwire-citation-author" data-delta="4">John H. Postlethwait</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">171-182;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000240 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459169--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Julian M. Catchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BM.%2BCatchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Catchen%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BM.%2BCatchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Angel Amores </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angel%2BAmores%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amores%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngel%2BAmores%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Paul Hohenlohe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BHohenlohe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hohenlohe%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BHohenlohe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William Cresko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BCresko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cresko%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BCresko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John H. Postlethwait </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BPostlethwait%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Postlethwait%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BH.%2BPostlethwait%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jpostle@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">jpostle@uoregon.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/3/171" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/3/171&amp;title=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/3/171&amp;title=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/3/171&amp;title=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/3/171&amp;text=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/3/171&amp;title=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/3/171&amp;t=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/3/171&amp;title=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/3/171&amp;title=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459169/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459169" id="node459169--3" data-pisa="ggg;1/3/171" data-pisa-master="ggg;g3.111.000240" data-apath="/ggg/1/3/171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Stacks</em>: Building and Genotyping Loci <em>De Novo</em> From Short-Read Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Julian M. Catchen</span>, <span class="highwire-citation-author" data-delta="1">Angel Amores</span>, <span class="highwire-citation-author" data-delta="2">Paul Hohenlohe</span>, <span class="highwire-citation-author" data-delta="3">William Cresko</span> and <span class="highwire-citation-author" data-delta="4">John H. Postlethwait</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">171-182;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000240 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459169--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Julian M. Catchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BM.%2BCatchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Catchen%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BM.%2BCatchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Angel Amores </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angel%2BAmores%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amores%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngel%2BAmores%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Paul Hohenlohe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BHohenlohe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hohenlohe%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BHohenlohe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William Cresko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Ecology and Evolutionary Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BCresko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cresko%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BCresko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John H. Postlethwait </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Neuroscience, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BPostlethwait%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Postlethwait%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BH.%2BPostlethwait%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jpostle@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">jpostle@uoregon.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=171&amp;rft.epage=182&amp;rft.atitle=Stacks%3A%20Building%20and%20Genotyping%20Loci%20De%20Novo%20From%20Short-Read%20Sequences&amp;rft.volume=1&amp;rft.issue=3&amp;rft.date=2011-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Catchen%2C+Julian+M.&amp;rft.au=Amores%2C+Angel&amp;rft.au=Hohenlohe%2C+Paul&amp;rft.au=Cresko%2C+William&amp;rft.au=Postlethwait%2C+John+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459169/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459169/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459169/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459169/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459169/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459169/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459169/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459169/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459169/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459169/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459169/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459169/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/3/171" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 2</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/3/171.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/3/171.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/3/171.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/3/171.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
