<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/3/357" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/3/357.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/3/357.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/3/357.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/3.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/3/357.atom" />
<meta name="HW.pisa" content="ggg;2/3/357" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass" />
<meta name="DC.Identifier" content="10.1534/g3.111.001503" />
<meta name="DC.Date" content="2012-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Liu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Switchgrass ( Panicum virgatum ) has been identified as a promising cellulosic biofuel crop in the United States. Construction of a genetic linkage map is fundamental for switchgrass molecular breeding and the elucidation of its genetic mechanisms for economically important traits. In this study, a novel population consisting of 139 selfed progeny of a northern lowland genotype, NL 94 LYE 16X13, was used to construct a linkage map. A total of 2493 simple sequence repeat markers were screened for polymorphism. Of 506 polymorphic loci, 80.8% showed a goodness-of-fit of 1:2:1 segregation ratio. Among 469 linked loci on the framework map, 241 coupling vs. 228 repulsion phase linkages were detected that conformed to a 1:1 ratio, confirming disomic inheritance. A total of 499 loci were mapped to 18 linkage groups (LG), of which the cumulative length was 2085.2 cM, with an average marker interval of 4.2 cM. Nine homeologous LG pairs were identified based on multi-allele markers and comparative genomic analysis. Two clusters of segregation-distorted loci were identified on LG 5b and 9b, respectively. Comparative analysis indicated a one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet ( Setaria italica ) chromosomes, suggesting strong homology between the two species. The linkage map derived from selfing a heterozygous parent, instead of two separate maps usually constructed for a cross-fertilized species, provides a new genetic framework to facilitate genomics research, quantitative trait locus (QTL) mapping, and marker-assisted breeding." />
<meta name="DC.Contributor" content="Linglong Liu" />
<meta name="DC.Contributor" content="Yanqi Wu" />
<meta name="DC.Contributor" content="Yunwen Wang" />
<meta name="DC.Contributor" content="Tim Samuels" />
<meta name="article:published_time" content="2012-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Switchgrass (&lt;i&gt;Panicum virgatum&lt;/i&gt;) has been identified as a promising cellulosic biofuel crop in the United States. Construction of a genetic linkage map is fundamental for switchgrass molecular breeding and the elucidation of its genetic mechanisms for economically important traits. In this study, a novel population consisting of 139 selfed progeny of a northern lowland genotype, NL 94 LYE 16X13, was used to construct a linkage map. A total of 2493 simple sequence repeat markers were screened for polymorphism. Of 506 polymorphic loci, 80.8% showed a goodness-of-fit of 1:2:1 segregation ratio. Among 469 linked loci on the framework map, 241 coupling &lt;i&gt;vs.&lt;/i&gt; 228 repulsion phase linkages were detected that conformed to a 1:1 ratio, confirming disomic inheritance. A total of 499 loci were mapped to 18 linkage groups (LG), of which the cumulative length was 2085.2 cM, with an average marker interval of 4.2 cM. Nine homeologous LG pairs were identified based on multi-allele markers and comparative genomic analysis. Two clusters of segregation-distorted loci were identified on LG 5b and 9b, respectively. Comparative analysis indicated a one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet (&lt;i&gt;Setaria italica&lt;/i&gt;) chromosomes, suggesting strong homology between the two species. The linkage map derived from selfing a heterozygous parent, instead of two separate maps usually constructed for a cross-fertilized species, provides a new genetic framework to facilitate genomics research, quantitative trait locus (QTL) mapping, and marker-assisted breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/03/01" />
<meta name="citation_mjid" content="ggg;2/3/357" />
<meta name="citation_id" content="2/3/357" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/3/357" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/3/357.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/3/357.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/3/357.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001503" />
<meta name="citation_pmid" content="22413090" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="357" />
<meta name="citation_lastpage" content="370" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Linglong Liu" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078" />
<meta name="citation_author" content="Yanqi Wu" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078" />
<meta name="citation_author_email" content="yanqi.wu@okstate.edu" />
<meta name="citation_author" content="Yunwen Wang" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078" />
<meta name="citation_author" content="Tim Samuels" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Andru;citation_author=Y. B. Pan;citation_author=S. Thongthawee;citation_author=D. M. Burner;citation_author=C. A. Kimbeng;citation_title=Genetic analysis of the sugarcane (Saccharum spp.) cultivar 'LCP 85-384'. I. Linkage mapping using AFLP, SSR, and TRAP markers.;citation_pages=77-93;citation_volume=123;citation_year=2011;citation_issue=1;citation_pmid=21472411;citation_doi=10.1007/s00122-011-1568-x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=F. L. Barnett;citation_author=R. F. Carver;citation_title=Meiosis and Pollen Stainability in Switchgrass, Panicum virgatum L;citation_pages=301-304;citation_volume=7;citation_year=1967;citation_issue=4;citation_doi=10.2135/cropsci1967.0011183X000700040005x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. S. Beckmann;citation_author=M. Soller;citation_title=Restriction fragment length polymorphisms in genetic improvement: methodologies, mapping and costs;citation_pages=35-43;citation_volume=67;citation_year=1983;citation_pmid=24258478;citation_doi=10.1007/BF00303919" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. Beldade;citation_author=S. V. Saenko;citation_author=N. Pul;citation_author=A. D. Long;citation_title=A gene-based linkage map for Bicyclus anynana butterflies allows for a comprehensive analysis of synteny with the lepidopteran reference genome.;citation_pages=e1000366-e1000366;citation_volume=5;citation_year=2009;citation_issue=2;citation_pmid=19197358;citation_doi=10.1371/journal.pgen.1000366" />
<meta name="citation_reference" content="citation_journal_title=Thell. Cell Mol. Biol. Lett.;citation_author=A. Blanco;citation_author=R. Simeone;citation_author=A. Cenci;citation_author=A. Gadaleta;citation_author=O. A. Tanzarella;citation_title=Extension of the “Messapia x dicoccoides” linkage map of Triticum turgidum (L.);citation_pages=529-541;citation_volume=9;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=P. Blasi;citation_author=S. Blanc;citation_author=S. Wiedemann-Merdinoglu;citation_author=E. Prado;citation_author=E. H. Rühl;citation_title=Construction of a reference linkage map of Vitis amurensis and genetic mapping of Rpv8, a locus conferring resistance to grapevine downy mildew.;citation_pages=43-53;citation_volume=123;citation_year=2011;citation_issue=1;citation_pmid=21404060;citation_doi=10.1007/s00122-011-1565-0" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=T. Broquet;citation_author=E. Petit;citation_title=Quantifying genotyping errors in noninvasive population genetics.;citation_pages=3601-3608;citation_volume=13;citation_year=2004;citation_issue=11;citation_pmid=15488016;citation_doi=10.1111/j.1365-294X.2004.02352.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=T. P. Brutnell;citation_author=L. Wang;citation_author=K. Swartwood;citation_author=A. Goldschmidt;citation_author=D. Jackson;citation_title=Setaria viridis: a model for C4 photosynthesis;citation_volume=22;citation_year=2010;citation_pmid=20693355;citation_doi=10.1105/tpc.110.075309" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. W. Cai;citation_author=N. Yuyama;citation_author=H. Tamaki;citation_author=A. Yoshizawa;citation_title=Isolation and characterization of simple sequence repeat markers in the hexaploid forage grass timothy (Phleum pratense L.).;citation_pages=1337-1349;citation_volume=107;citation_year=2003;citation_issue=8;citation_pmid=12920514;citation_doi=10.1007/s00122-003-1386-x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. W. Cai;citation_author=M. Inoue;citation_author=N. Yuyama;citation_author=W. Takahashi;citation_author=M. Hirata;citation_title=Isolation, characterization and mapping of simple sequence repeat markers in zoysiagrass (Zoysia spp.).;citation_pages=158-166;citation_volume=112;citation_year=2005;citation_issue=1;citation_pmid=16235048;citation_doi=10.1007/s00122-005-0118-9" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. A. Carbonell;citation_author=M. J. Asins;citation_author=M. Baselga;citation_author=E. Balansard;citation_author=T. M. Gerig;citation_title=Power studies in the estimation of genetic parameters of quantitative trait loci for backcross and doubled haploid populations;citation_pages=411-416;citation_volume=86;citation_year=1993;citation_doi=10.1007/BF00838555" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. Causse;citation_author=T. M. Fulton;citation_author=Y. G. Cho;citation_author=S. N. Ahn;citation_author=J. Chunwongse;citation_title=Saturated molecular map of the rice genome based on an interspecific backcross population;citation_pages=1251-1274;citation_volume=138;citation_year=1994;citation_pmid=7896104" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Charlesworth;citation_author=J. H. Willis;citation_title=The genetics of inbreeding depression.;citation_pages=783-796;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19834483;citation_doi=10.1038/nrg2664" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=D. E. Costich;citation_author=B. Friebe;citation_author=M. J. Sheehan;citation_author=M. C. Casler;citation_author=E. S. Buckler;citation_title=Genome-size variation in switchgrass (Panicum virgatum): flow cytometry and cytology reveal rampant aneuploidy;citation_pages=130-141;citation_volume=3;citation_year=2010;citation_doi=10.3835/plantgenome2010.04.0010" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=P. Devaux;citation_author=A. Kilian;citation_author=A. Kleinhofs;citation_title=Comparative mapping of the barley genome with male and female recombination-derived, doubled haploid populations.;citation_pages=600-608;citation_volume=249;citation_year=1995;citation_issue=6;citation_pmid=8544825;citation_doi=10.1007/BF00418029" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=K. M. Devos;citation_author=M. D. Gale;citation_title=Comparative genetics in the grasses.;citation_pages=3-15;citation_volume=35;citation_year=1997;citation_issue=1-2;citation_pmid=9291955;citation_doi=10.1023/A:1005820229043" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=K. M. Devos;citation_author=M. D. Gale;citation_title=Genome Relationships: The Grass Model in Current Research;citation_pages=637-646;citation_volume=12;citation_year=2000;citation_issue=5;citation_pmid=10810140;citation_doi=10.1105/tpc.12.5.637" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. N. Doust;citation_author=E. A. Kellogg;citation_author=K. M. Devos;citation_author=J. F. Bennetzen;citation_title=Foxtail Millet: A Sequence-Driven Grass Model System;citation_pages=137-141;citation_volume=149;citation_year=2009;citation_issue=1;citation_pmid=19126705;citation_doi=10.1104/pp.108.129627" />
<meta name="citation_reference" content="citation_journal_title=Focus;citation_author=J. J. Doyle;citation_author=J. K. Doyle;citation_title=Isolation of plant DNA from fresh tissue;citation_pages=13-15;citation_volume=12;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Gale;citation_author=K. M. Devos;citation_title=Plant Comparative Genetics after 10 Years;citation_pages=656-659;citation_volume=282;citation_year=1998;citation_issue=5389;citation_pmid=9784118;citation_doi=10.1126/science.282.5389.656" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Graner;citation_author=A. Jahoor;citation_author=J. Schondelmaier;citation_author=H. Siedler;citation_author=K. Pillen;citation_title=Construction of an RFLP map of barley;citation_pages=250-256;citation_volume=83;citation_year=1991;citation_pmid=24202366" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. E. Gunter;citation_author=G. A. Tuskan;citation_author=S. D. Wullschleger;citation_title=Diversity among Populations of Switchgrass Based on RAPD Markers;citation_pages=1017-1022;citation_volume=36;citation_year=1996;citation_issue=4" />
<meta name="citation_reference" content="citation_author=A. S. Hitchcock;citation_author=A. Chase" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Hirata;citation_author=H. W. Cai;citation_author=M. Inoue;citation_author=N. Yuyama;citation_author=Y. Miura;citation_title=Development of simple sequence repeat (SSR) markers and construction of an SSR-based linkage map in Italian ryegrass (Lolium multiflorum Lam.).;citation_pages=270-279;citation_volume=113;citation_year=2006;citation_issue=2;citation_pmid=16791693;citation_doi=10.1007/s00122-006-0292-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Y. Hoarau;citation_author=B. Offmann;citation_author=A. D’Hont;citation_author=A. M. Risterucci;citation_author=D. Roques;citation_title=Genetic dissection of a modern sugarcane cultivar (Saccharum spp.). I. Genome mapping with AFLP markers;citation_pages=84-97;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220000390" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. A. Hopkins;citation_author=C. M. Taliaferro;citation_author=C. D. Murphy;citation_author=D. Christian;citation_title=Chromosome Number and Nuclear DNA Content of Several Switchgrass Populations;citation_pages=1192-1195;citation_volume=36;citation_year=1996;citation_issue=5;citation_doi=10.2135/cropsci1996.0011183X003600050021x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Jansen;citation_author=A. G. De Jong;citation_author=J. W. Van Ooijen;citation_title=Constructing dense genetic linkage maps;citation_pages=1113-1122;citation_volume=102;citation_year=2001;citation_doi=10.1007/s001220000489" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Jenczewski;citation_author=M. Gherandi;citation_author=I. Bonnin;citation_author=J. M. Prosperi;citation_author=I. Oliveri;citation_title=Insight on segregation distortions in two intraspecific crosses between annual species of Medicago (Leguminosae);citation_pages=682-691;citation_volume=94;citation_year=1997;citation_doi=10.1007/s001220050466" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Jia;citation_author=Z. Zhang;citation_author=Y. Liu;citation_author=C. Zhang;citation_author=Y. Shi;citation_title=Development and genetic mapping of SSR markers in foxtail millet [Setaria italica (L.) P. Beauv.].;citation_pages=821-829;citation_volume=118;citation_year=2009;citation_issue=4;citation_pmid=19139840;citation_doi=10.1007/s00122-008-0942-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. C. D. Johnson;citation_author=D. T. Haydon;citation_title=Maximum-likelihood estimation of allelic dropout and false allele error rates from microsatellite genotypes in the absence of reference data;citation_volume=175;citation_year=2007;citation_pmid=17179070;citation_doi=10.1534/genetics.106.064618" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. S. Jones;citation_author=M. P. Dupal;citation_author=R. Kolliker;citation_author=M. C. Drayton;citation_author=J. W. Forster;citation_title=Development and characterization of simple sequence repeat (SSR) markers for perennial ryegrass (Lolium perenne L.);citation_pages=405-415;citation_volume=102;citation_year=2001;citation_doi=10.1007/s001220051661" />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=S. Jung;citation_author=A. Abbott;citation_author=C. Jesudurai;citation_author=J. Tomkins;citation_author=D. Main;citation_title=Frequency, type, distribution and annotation of simple sequence repeats in Rosaceae ESTs.;citation_pages=136-143;citation_volume=5;citation_year=2005;citation_issue=3;citation_pmid=15761705;citation_doi=10.1007/s10142-005-0139-0" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=Y. Kashi;citation_author=D. King;citation_author=M. Soller;citation_title=Simple sequence repeats as a source of quantitative genetic variation.;citation_pages=74-78;citation_volume=13;citation_year=1997;citation_issue=2;citation_pmid=9055609;citation_doi=10.1016/S0168-9525(97)01008-1" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. D. Kosambi;citation_title=The estimation of map distances from recombination values;citation_pages=172-175;citation_volume=12;citation_year=1944;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. La Rota;citation_author=R. V. Kantety;citation_author=J. K. Yu;citation_author=M. E. Sorrells;citation_title=Nonrandom distribution and frequencies of genomic and EST-derived microsatellite markers in rice, wheat, and barley.;citation_pages=23-23;citation_volume=6;citation_year=2005;citation_pmid=15720707;citation_doi=10.1186/1471-2164-6-23" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=P. Li;citation_author=T. P. Brutnell;citation_title=Setaria viridis and Setaria italica, model genetic systems for the Panicoid grasses;citation_volume=62;citation_year=2011;citation_pmid=21459768;citation_doi=10.1093/jxb/err096" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. Lichten;citation_author=A. S. Goldman;citation_title=Meiotic Recombination Hotspots;citation_pages=423-444;citation_volume=29;citation_year=1995;citation_issue=1;citation_pmid=8825482;citation_doi=10.1146/annurev.ge.29.120195.002231" />
<meta name="citation_reference" content="citation_journal_title=Bioenergy Res.;citation_title=Identification of a selfing compatible genotype and mode of inheritance in switchgrass;citation_author=L. L. Liu;citation_author=Y. Q. Wu;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Lu;citation_author=J. Romero-Severson;citation_author=R. Bernardo;citation_title=Chromosomal regions associated with segregation distortion in maize.;citation_pages=622-628;citation_volume=105;citation_year=2002;citation_issue=4;citation_pmid=12582513;citation_doi=10.1007/s00122-002-0970-9" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=T. W. Lyttle;citation_title=Segregation Distorters;citation_pages=511-557;citation_volume=25;citation_year=1991;citation_issue=1;citation_pmid=1812815;citation_doi=10.1146/annurev.ge.25.120191.002455" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. M. Martinez-Reyna;citation_author=K. P. Vogel;citation_author=C. Caha;citation_author=D. J. Lee;citation_title=Meiotic Stability, Chloroplast DNA Polymorphisms, and Morphological Traits of Upland  Lowland Switchgrass Reciprocal Hybrids;citation_pages=1579-1583;citation_volume=41;citation_year=2001;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. M. Martinez-Reyna;citation_author=K. P. Vogel;citation_title=Incompatibility Systems in Switchgrass;citation_pages=1800-1805;citation_volume=42;citation_year=2002;citation_issue=6;citation_doi=10.2135/cropsci2002.1800" />
<meta name="citation_reference" content="citation_journal_title=Biomass Bioenergy;citation_author=S. B. McLaughlin;citation_author=L. A. Kszos;citation_title=Development of switchgrass (Panicum virgatum) as a bioenergy feedstock in the United States;citation_pages=515-535;citation_volume=28;citation_year=2005;citation_doi=10.1016/j.biombioe.2004.05.006" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. M. Missaoui;citation_author=A. H. Paterson;citation_author=J. H. Bouton;citation_title=Investigation of genomic organization in switchgrass (Panicum virgatum L.) using DNA markers.;citation_pages=1372-1383;citation_volume=110;citation_year=2005;citation_issue=8;citation_pmid=15841364;citation_doi=10.1007/s00122-005-1935-6" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=A. M. Missaoui;citation_author=A. H. Paterson;citation_author=J. H. Bouton;citation_title=Molecular markers for the classification of switchgrass (Panicum virgatum L.) germplasm and to assess genetic diversity in three synthetic switchgrass populations;citation_pages=1291-1302;citation_volume=53;citation_year=2006;citation_doi=10.1007/s10722-005-3878-9" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=M. Morgante;citation_author=A. M. Olivieri;citation_title=PCR-amplified microsatellites as markers in plant genetics.;citation_pages=175-182;citation_volume=3;citation_year=1993;citation_issue=1;citation_pmid=8401603;citation_doi=10.1046/j.1365-313X.1993.t01-9-00999.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Morgante;citation_author=M. Hanafey;citation_author=W. Powell;citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes.;citation_pages=194-200;citation_volume=30;citation_year=2002;citation_issue=2;citation_pmid=11799393;citation_doi=10.1038/ng822" />
<meta name="citation_reference" content="citation_journal_title=Bioenergy Res.;citation_author=B. Narasimhamoorthy;citation_author=M. C. Saha;citation_author=B. Swaller;citation_author=J. H. Bouton;citation_title=Genetic diversity in switchgrass collections assessed by EST-SSR markers;citation_pages=136-146;citation_volume=1;citation_year=2008;citation_doi=10.1007/s12155-008-9011-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Okada;citation_author=C. Lanzatella;citation_author=M. C. Saha;citation_author=J. Bouton;citation_author=R. Wu;citation_title=Complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions;citation_pages=745-760;citation_volume=185;citation_year=2010;citation_pmid=20407132;citation_doi=10.1534/genetics.110.113910" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. H. Paterson;citation_author=T. H. Lan;citation_author=K. P. Reischmann;citation_author=C. Chang;citation_author=Y. R. Lin;citation_title=Toward a unified genetic map of higher plants, transcending the monocot-dicot divergence.;citation_pages=380-382;citation_volume=14;citation_year=1996;citation_issue=4;citation_pmid=8944014;citation_doi=10.1038/ng1296-380" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=L. R. Pinto;citation_author=K. M. Oliveira;citation_author=T. Marconi;citation_author=A. A. F. Garcia;citation_author=E. C. Ulian;citation_title=Characterization of novel sugarcane expressed sequence tag microsatellites and their comparison with genomic SSRs;citation_pages=378-384;citation_volume=125;citation_year=2006;citation_doi=10.1111/j.1439-0523.2006.01227.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=F. Pompanon;citation_author=A. Bonin;citation_author=E. Bellemain;citation_author=P. Taberlet;citation_title=Genotyping errors: causes, consequences and solutions.;citation_pages=847-859;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304600;citation_doi=10.1038/nrg1707" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_author=C. L. Porter;citation_title=An analysis of variation between upland and lowland switchgrass, Panicum virgatum L., in central Oklahoma;citation_pages=980-992;citation_volume=47;citation_year=1966;citation_doi=10.2307/1935646" />
<meta name="citation_reference" content="citation_author=K. J. Preacher;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Qu;citation_author=J. F. Hancock;citation_title=Detecting and mapping repulsion-phase linkage in polyploids with polysomic inheritance;citation_pages=136-143;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100647" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. A. Quarrie;citation_author=A. Steed;citation_author=C. Calestani;citation_author=A. Semikhodskii;citation_author=C. Lebreton;citation_title=A high-density genetic map of hexaploid wheat (Triticum aestivum L.) from the cross Chinese Spring x SQ1 and its use to compare QTLs for grain yield across a range of environments.;citation_pages=865-880;citation_volume=110;citation_year=2005;citation_issue=5;citation_pmid=15719212;citation_doi=10.1007/s00122-004-1902-7" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. L. Remington;citation_author=R. W. Whetten;citation_author=B. H. Liu;citation_author=D. M. O’Malley;citation_title=Construction of an AFLP genetic map with nearly complete genome coverage in Pinus taeda.;citation_pages=1279-1292;citation_volume=98;citation_year=1999;citation_issue=8;citation_pmid=12238515;citation_doi=10.1007/s001220051194" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. S. Röder;citation_author=V. Korzun;citation_author=K. Wandehake;citation_author=J. Planschke;citation_author=M. H. Tixier;citation_title=A Microsatellite Map of Wheat;citation_pages=2007-2023;citation_volume=149;citation_year=1998;citation_issue=4;citation_pmid=9691054" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=M. S. Röder;citation_author=V. Korzun;citation_author=B. S. Gill;citation_author=M. W. Ganal;citation_title=The physical mapping of microsatellite markers in wheat;citation_pages=278-283;citation_volume=41;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. C. Saha;citation_author=R. Mian;citation_author=J. C. Zwonitzer;citation_author=K. Chekhovskiy;citation_author=A. A. Hopkins;citation_title=An SSR- and AFLP-based genetic linkage map of tall fescue (Festuca arundinacea Schreb.).;citation_pages=323-336;citation_volume=110;citation_year=2005;citation_issue=2;citation_pmid=15558229;citation_doi=10.1007/s00122-004-1843-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. C. Saha;citation_author=J. D. Cooper;citation_author=M. A. Rouf Mian;citation_author=K. Chekhovskiy;citation_author=G. D. May;citation_title=Tall fescue genomic SSR markers: development and transferability across multiple grass species.;citation_pages=1449-1458;citation_volume=113;citation_year=2006;citation_issue=8;citation_pmid=16947059;citation_doi=10.1007/s00122-006-0391-2" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. A. Saski;citation_author=Z. Li;citation_author=F. A. Feltus;citation_author=H. Luo;citation_title=New genomic resources for switchgrass: a BAC library and comparative analysis of homoeologous genomic regions harboring bioenergy traits.;citation_pages=369-369;citation_volume=12;citation_year=2011;citation_pmid=21767393;citation_doi=10.1186/1471-2164-12-369" />
<meta name="citation_reference" content="citation_journal_title=Proc. Nat. Acad. Sci. USA;citation_author=M. R. Schmer;citation_author=R. B. Mitchell;citation_author=R. K. Perrin;citation_title=Net energy of cellulosic ethanol from switchgrass;citation_pages=464-469;citation_volume=1050;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Biotechnol.;citation_author=K. Semagn;citation_author=Å. Bjørnstad;citation_author=Y. Xu;citation_title=The genetic dissection of quantitative traits in crops;citation_pages=1-45;citation_volume=13;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linkage maps by means of a new computer package: JOINMAP;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. E. Talbert;citation_author=D. H. Timothy;citation_author=J. C. Burns;citation_author=J. O. Rawlings;citation_author=R. H. Moll;citation_title=Estimates of Genetic Parameters in Switchgrass;citation_pages=725-728;citation_volume=23;citation_year=1983;citation_issue=4;citation_doi=10.2135/cropsci1983.0011183X002300040029x" />
<meta name="citation_reference" content="citation_author=C. M. Taliaferro;citation_author=K. P. Vogel;citation_author=J. H. Bouton;citation_author=S. B. McLaughlin;citation_author=G. A. Tuskan" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Tobias;citation_author=P. Twigg;citation_author=D. Hayden;citation_author=K. Vogel;citation_author=R. Mitchell;citation_title=Analysis of expressed sequence tags and the identification of associated short tandem repeats in switchgrass.;citation_pages=956-964;citation_volume=111;citation_year=2005;citation_issue=5;citation_pmid=16049706;citation_doi=10.1007/s00122-005-0030-3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=C. M. Tobias;citation_author=D. M. Hyden;citation_author=P. Twigg;citation_author=G. Sarath;citation_title=Genic microsatellite markers derived from EST sequences of switchgrass (Panicum virgatum L.);citation_pages=185-187;citation_volume=6;citation_year=2006;citation_doi=10.1111/j.1471-8286.2006.01187.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=C. M. Tobias;citation_author=G. Sarath;citation_author=P. Twigg;citation_author=E. Lindquist;citation_author=J. Pangilinan;citation_title=Comparative genomics in switchgrass using 61,585 high-quality expressed sequence tags;citation_pages=2;citation_volume=1;citation_year=2008" />
<meta name="citation_reference" content="citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=R. K. Varshney;citation_author=A. Graner;citation_author=M. E. Sorrells;citation_title=Genic microsatellite markers in plants: features and applications.;citation_pages=48-55;citation_volume=23;citation_year=2005;citation_issue=1;citation_pmid=15629858;citation_doi=10.1016/j.tibtech.2004.11.005" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=K. P. Vogel;citation_author=J. F. Pedersen;citation_title=Breeding systems for cross-pollinated perennial grasses;citation_pages=251-274;citation_volume=11;citation_year=1993" />
<meta name="citation_reference" content="citation_author=K. P. Vogel;citation_author=G. Sarath;citation_author=A. J. Saathoff;citation_author=R. B. Mitchell" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. W. Wang;citation_author=T. D. Samuels;citation_author=Y. Q. Wu;citation_title=Development of 1,030 genomic SSR markers in switchgrass.;citation_pages=677-686;citation_volume=122;citation_year=2011;citation_issue=4;citation_pmid=20978736;citation_doi=10.1007/s00122-010-1477-4" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. F. Wendel;citation_author=M. D. Edwards;citation_author=C. W. Stuber;citation_title=Evidence for multilocus genetic control of preferential fertilization in maize;citation_pages=297-302;citation_volume=58;citation_year=1987;citation_pmid=3471740;citation_doi=10.1038/hdy.1987.44" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. H. Wolfe;citation_title=Yesterday's polyploids and the mystery of diploidization.;citation_pages=333-341;citation_volume=2;citation_year=2001;citation_issue=5;citation_pmid=11331899;citation_doi=10.1038/35072009" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. K. Wu;citation_author=W. Burnquist;citation_author=M. E. Sorrells;citation_author=T. L. Tew;citation_author=P. H. Moore;citation_title=The detection and estimation of linkage in polyploids using single-dose restriction fragments;citation_pages=294-300;citation_volume=83;citation_year=1992;citation_pmid=24202510" />
<meta name="citation_reference" content="citation_journal_title=Popul Biol.;citation_author=R. Wu;citation_author=C. X. Ma;citation_author=I. Painter;citation_author=Z. B. Zeng;citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species;citation_pages=349-363;citation_volume=61;citation_year=2002;citation_doi=10.1006/tpbi.2002.1577" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=Y. Wu;citation_author=Y. Huang;citation_title=An SSR genetic map of Sorghum bicolor (L.) Moench and its comparison to a published genetic map;citation_pages=84-89;citation_volume=50;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=Y. Xu;citation_author=L. Zhu;citation_author=J. Xiao;citation_author=N. Huang;citation_author=S. R. McCouch;citation_title=Chromosomal regions associated with segregation distortion of molecular markers in F2, backcross, doubled haploid, and recombinant inbred populations in rice (Oryza sativa L.).;citation_pages=535-545;citation_volume=253;citation_year=1997;citation_issue=5;citation_pmid=9065686;citation_doi=10.1007/s004380050355" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Res.;citation_author=H. A. Young;citation_author=B. J. Hernlem;citation_author=A. L. Anderton;citation_author=C. Lanzatella-Craig;citation_author=C. M. Tobias;citation_title=Dihaploid stocks for switchgrass isolated by a screening approach;citation_pages=305-313;citation_volume=3;citation_year=2010;citation_doi=10.1007/s12155-010-9081-7" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. E. Zalapa;citation_author=D. L. Price;citation_author=S. M. Kaeppler;citation_author=C. M. Tobias;citation_author=M. Okada;citation_title=Hierarchical classification of switchgrass genotypes using SSR and chloroplast sequences: ecotypes, ploidies, gene pools, and cultivars.;citation_pages=805-817;citation_volume=122;citation_year=2011;citation_issue=4;citation_pmid=21104398;citation_doi=10.1007/s00122-010-1488-1" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=Y. Zhang;citation_author=J. E. Zalapa;citation_author=A. R. Jakubowski;citation_author=D. L. Price;citation_author=A. Acharya;citation_title=Post-glacial evolution of Panicum virgatum: centers of diversity and gene pools revealed by SSR markers and cpDNA sequences.;citation_pages=933-948;citation_volume=139;citation_year=2011;citation_issue=7;citation_pmid=21786028;citation_doi=10.1007/s10709-011-9597-6" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/3.cover.gif" />
<meta name="twitter:description" content="Switchgrass ( Panicum virgatum ) has been identified as a promising cellulosic biofuel crop in the United States. Construction of a genetic linkage map is fundamental for switchgrass molecular breeding and the elucidation of its genetic mechanisms for economically important traits. In this study, a novel population consisting of 139 selfed progeny of a northern lowland genotype, NL 94 LYE 16X13, was used to construct a linkage map. A total of 2493 simple sequence repeat markers were screened for polymorphism. Of 506 polymorphic loci, 80.8% showed a goodness-of-fit of 1:2:1 segregation ratio. Among 469 linked loci on the framework map, 241 coupling vs. 228 repulsion phase linkages were detected that conformed to a 1:1 ratio, confirming disomic inheritance. A total of 499 loci were mapped to 18 linkage groups (LG), of which the cumulative length was 2085.2 cM, with an average marker interval of 4.2 cM. Nine homeologous LG pairs were identified based on multi-allele markers and comparative genomic analysis. Two clusters of segregation-distorted loci were identified on LG 5b and 9b, respectively. Comparative analysis indicated a one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet ( Setaria italica ) chromosomes, suggesting strong homology between the two species. The linkage map derived from selfing a heterozygous parent, instead of two separate maps usually constructed for a cross-fertilized species, provides a new genetic framework to facilitate genomics research, quantitative trait locus (QTL) mapping, and marker-assisted breeding." />
<meta name="og-title" property="og:title" content="A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/3/357" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Switchgrass ( Panicum virgatum ) has been identified as a promising cellulosic biofuel crop in the United States. Construction of a genetic linkage map is fundamental for switchgrass molecular breeding and the elucidation of its genetic mechanisms for economically important traits. In this study, a novel population consisting of 139 selfed progeny of a northern lowland genotype, NL 94 LYE 16X13, was used to construct a linkage map. A total of 2493 simple sequence repeat markers were screened for polymorphism. Of 506 polymorphic loci, 80.8% showed a goodness-of-fit of 1:2:1 segregation ratio. Among 469 linked loci on the framework map, 241 coupling vs. 228 repulsion phase linkages were detected that conformed to a 1:1 ratio, confirming disomic inheritance. A total of 499 loci were mapped to 18 linkage groups (LG), of which the cumulative length was 2085.2 cM, with an average marker interval of 4.2 cM. Nine homeologous LG pairs were identified based on multi-allele markers and comparative genomic analysis. Two clusters of segregation-distorted loci were identified on LG 5b and 9b, respectively. Comparative analysis indicated a one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet ( Setaria italica ) chromosomes, suggesting strong homology between the two species. The linkage map derived from selfing a heterozygous parent, instead of two separate maps usually constructed for a cross-fertilized species, provides a new genetic framework to facilitate genomics research, quantitative trait locus (QTL) mapping, and marker-assisted breeding." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/3.cover.gif" />
    <title>A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Dplnt8WLQeTIVHqCzEvDe9H105dC9nX6l8-t7TS0a_g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459259","apath":"\/ggg\/2\/3\/357.atom","pisa":"ggg;2\/3\/357","ac":{"\/ggg\/2\/3\/357.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/3\/357.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/3\/357"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/3\/357"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/3\/357":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c9WZ9AnFVRRkqKowiTMDNYPf2dfzgqNmYETEnLiMRh2pFPjgHCkaGjJLdnfrpBvDH\\\\\\\/W4yAuCxFnTpOH1WR3lrBnOtBNbeNmaogvcgDy\\\\\\\/oouA8W+D\\\\\\\/PBK\\\\\\\/3oSS8ciA6IAC8v8ZhQeN7V73T1syYkgwoT7gH3yS82jbt8QUZzd8v2rEJl4NBjzkkO1sPlPDEcVixEZsceIEPru3B0mLKF+sja3hzNYKoN2il3rbPz1QZ6V9jEu5kjo6S1ZqRJIKCXjclHZkuTHsdEJFqZzpt1vmcUdOlPEU89XRxTrZlB\\\\\\\/hxwNOYJN3TcDZ7rSyAItdsKVWfgEQA7feuhOhJyAJAdURGJFYs\\\\\\\/nYx4VbIdh578txf5Uo7ngIsv27v2+lZD8etOyY+3eVMnBLEW3WvlI9JT4pqxHzcr3Rbyyax9MrZw8Es9x3bXeS6+VcG3ue8VTh5ipPZZv3Jr6sJt9ml41ZYB\\\\\\\/5fOHFI3tk0KICvGmWl5u4EHqA5GtWbialY7qI+LUJDlS+IjV1X4ggMh5qoiBV9Lr8apzZ1cFloQw7MiceXtkyxSxlcdXC+K6Il1mBLIFPtYgCrYC5b7bL2+NuPVwcChQ5kfc1xKV4TSDxbIq6Aqo9kPgAr15ButU9xhQhxs+i2xV5Zko9eHnk+gs7trP0iLRTsFgiszxg3+JQmgs4Gt4\\\\\\\/Pz7YhE4omnzyfAn0Vgrgk7wOdSVAWu8Pd4jWoPiQKq\\\\\\\/M0N0NwzyBYl9smgqnGBXNGESikGHHUxvw16p3dw0JzPTDgFoG\\\\\\\/R1x6u5m\\\\\\\/AdgjvWRZABHv2hQFeoO+XDcXJiSNxTwJuD9iAhRlLsCcNDlaSVcTKCla3TeNg+sY0RwcrQ8FULiI6q0F0vrDPMx5y6WEuSHVmQg5rytrhvKqpSjc\\\\\\\/N6l\\\\\\\/2EzP0KpBPHWZemwW9rz5KfuIxgNukcBWkKR+YkFdYr1\\\\\\\/OywTXebi\\\\\\\/yCb4rcp6hKEPcIAnG7DIQlE\\\\\\\/mkQtKszv5A0lmH88mf4\\\\\\\/u9AEI0wyjFM\\\\\\\/ALF3QpF1h1lYMfLKrkwYdQJA7ymsCk0\\\\\\\/10rtBoGeK4TWaW2yfVWSOWfiut8cBSl28vY+9Um5kYoXoUkaDdDUwBOpFKp9S1JrZ+qAlWitU46kLkb+yOQ+sV0tVUbh5ShgGTyoJSL\\\\\\\/oRE4ni7wNaqWmSd9dVHsASjhaKHQKCipSaArXGOCws93VxBtqbvzxG9rwMULuUq+OVgV6ao0cTjpDPXubDFZuYsPcQYfg5vcGMaBqLKW8s7JBFNCjLBBJExyogr9ed4RdMMKZQYU\\\u0022,\\\u0022iv\\\u0022:\\\u0022nwsxH454RANP1HvHaXQFAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022093d6fe88d1bce9816478b347040b5eebf35a690be95f8c0c6a78ce4a3fd6d9f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Tzt22GEYjMwiFknajrcqFK4yIfXIf6my9lzx9mtDecKqgrwNfKoPeuMux5OnJykLMWjbSmNNsoc6HcTGdLX13SPH3O4ZEhTjCAtvPVG2nyOY2H3Xwj7prS0Ej7WyDyqDDa1OIwLYTgiMM2Ho43QoECxI1tCS4AL3gNDIDgVv1GOW3O2j1KSglvTEGMHATx7Z2lC2lqJy6gi1u89zU509H\\\\\\\/K6kHoIqICTPEC2WHXVjA+ZCmZqKJfETAIopxCqtcdHpIOLmHAtVyVRJrXJRC2dgqe2MZKV2Sz8D7598kR9OEM9sJOVP0EW2LbvasBwqVqBpr+auflm5NNXnMMRq2jM57SEk5\\\\\\\/SwN9D1HwnpnOyn\\\\\\\/OecMTNkIiFh2qih+Idxmdi96Rx6WIxVTrlTrAIfjX5nwK3nGKVXFfn4QHURIVRTPa4IU0b03JP0NQ7m1NtEGah3UGa4LnG1P2V1bIUzEF37CnbJFv3A\\\\\\\/D95ULpm2SNgGz0R4euR05TF+o3yPdiyEaJQcYHu5WT3fI2gCAKPeCBOl8HkIpeXT1B7jHspm2eOjAv3fYdf+P6Cf8Hvauupp9mJNd+rtumdVM9OLMBO6DaUaub7BsX+4KTtJC06oLOj1cuTqSYmBl4t0jvu6qXsQKo\\\\\\\/q2+gO3ENWuDrm12pP7KpXrPJ3DZkUDuGmKEy67dnZYW52V3Vs3bKWSaSV5JcqzsqfUM7bolo1HEXXt7sGV5UBCTKNlcJ1DqfJEEiSjNylhJAEguuVm23UdsY664qf1kh4FE3SJ3Wn8Vy57kVD+miq5WlRWa7zq3B1sWX+iaykEHp7EQEAj2CiDPlxhZq3h9hVJYxwhBgprwcOIgnHtytV8dQptMwWNZmBMHdSO+0s0m+2MHIV8fM2gdRWkBbZmtuuJjurne7kfwyAkg+PoxlLbSvW8iEq0fQytJve08W3OQnVn1ChdGR\\\\\\\/VdbHTMHZ1KTIeueqxKh9gEreC7QLIwww6chFhSSFxmGVZjqzHADCsFXEwh7bYvCa\\\\\\\/Fh73bjZ3QjFsYNsxadf8sGbigxB3oIOZ9xnc1EkAF1IkFr83bsuWPAqTsb3fmOEtP2mqidYtYHqjlZt2DM4aQzYB7+ZQdfmMMjjbTNS6uR\\\\\\\/wqqBIJAiPH2lAkwPlGDXM3V1JKReEtsVCrL\\\\\\\/Wrjc\\\\\\\/27Se66BhUUQHQPjthdgwnQqGW7A51iG7XeaZ6W99ooNmrzi7NUqz6vSIEoeypQsrn0qKdqSdg\\\\\\\/6KzxuKG6Ykoh2IM6z5ulmN5aR0T2Jp+wEqVt4+8dhQqo7Rnd8j2HACkRLj2k7IRXZ0NURQDRBkep6SHRuVRs1qSlbjlagjMESXe+TtY4l0ozuMQOtgaBDl9one7Mym8zrjZ1qROCX54fnlz6vyvMJHIBanutGvTglcaS9Muw8M0qDUgTxcj8gU4wgwABKZlHjVwn+tdr8YHukbkyQVb83omJKf9h5mIPRs9P1iEo\\\\\\\/pCs8xss6DqOFeKUVjTtLpOTi9iswTRFNjVh7FFg2PGdBSGCha87pbkSF4dVSn9dxoE1Q0DM+IO4Ev1Hb4s2+BNz+sVEoZVSjy0Y0FGPR3JXBp1p4vZk56hvdnhAWB5lrUEaBCydE0iwP5CxXPx+fEGifyUOJFjwymXDaIch1x\\\\\\\/wwZhGlmD0Lhl4dczzcG5NSX\\\\\\\/sTLaMmzM5QLfCsXzFnk9sEY7AtINC84UBed0KK8E8wo1ZgxF6NdLJoTGfVzhN5C6S42uDJpto8+fpHbb\\\\\\\/BjkdLwslMsJB38budL5OkynmsRI3yNnBmYJbySu04rAQu\\\\\\\/h0hqf8UpQnaBB9fELYfBHARvPYglrVqglrwzqH0ptFtIW+z8\\\\\\\/l1vfC7HeN6wLKZAfb7x1gWbKXPatHBaGf+R+R7Z\\\\\\\/GbdGt4cZw+27E5WKXN0bbwGjFWf1t8MNMH\\\\\\\/6YubhFBO0p9yr\\\\\\\/Quv9eaB2rghFoxV8E5gOe6voIdsgueBsf5GapopMpJFWox79VSxzLzxxmgrux34lqkFZyTVqLsuW8YO8TsbC9g9nmg+uoabUMJ4L\\\\\\\/Rg1OSYOTXwpjCVTbxc7KeUWqgM\\\\\\\/i632Qe9WZQaIql3ZvmJvwZeNv\\\\\\\/qgpboLsHd8IHYKSxGaW9ruyAtTkMzpESoyYGyQaHgx\\\\\\\/xMZEbZbTuxHN4rZiqVh34v78lty72\\\\\\\/Pcig8dFfaEB1qJnrc8zPjNFpbj\\\\\\\/Gk8EteCJ1dbFsEJrmUB8nzkK4d0Q\\\\\\\/YTguNi0xiiML9nP2nhMtdMB58D9yQDVAsBbf2zUA8mxj8dc4ypIA8YMxXc4TF39f3c1cHgyWTRHPPs8bgm1IIzRtB7KZfMyAa8tbvoCVAgx2B+O9dREwZefYMuT9YbrH8exulpW\\\\\\\/RlxuHPv52GbIzwtiJBwp7tZH9MMvnIZ+zBUujvKwQC3GZUt3\\\\\\\/aBNMO0zd2a+BCf5oOWKVGowTR\\\\\\\/sd9T5U1V0OpshCZJMASfYrvmiCP+kcaeY\\\\\\\/HM3Yqz+RAswumEv120QjuaIKtIfP7IBxe85vFxgFb+Cdf0MiHCWVG2l5FCi22NWGo2mhftrBgn7S+MlZ+Mr3uPr93fup5Zhbh+addx4d+OV7i2an7Kda7e\\\\\\\/MEbPZiJTt7mLx1XzHl7OzdMsFy\\\\\\\/bcQRSkzoxGnpxZDOnk26nvWjEjBQP9kdsKWXr5Mtt89DE73yFHBQWkw6Iqp8Q\\\\\\\/7yC63ZYT50vWz872WONxAqj6FbGERtACLmEGIVH82wRui+rvf\\\\\\\/Qz9ezgbrR9XqvPV7GKEUaWdrhwj9iWPaGQkc0B+0DHv435Ybtnx4RqRdzn2HKAETV9b6OWyOnHi2Y9fIVVVZNbYSwvVWATczNirBRMT8dAAgK9Pd5nHWajMBIwm4r4NfAc0ZwPp3V3CfReXm1EMEsZacUMRHjN2tKlBgooSvJQ4KAvq1h7OlpPOWTLqcE4loWY92SR75PD9JkmpIkYYatKc9Mzw4Yzi99AVKU2n1hW7ckfyiwMfcrUKchOP0h1YhpsVYWdtOdtgLL4Ud\\\\\\\/blRPf3AJKI9RnCL4WxMuZCWQivQ1JrwkYts4UAPGEM9zAGL8hfBqEX4BjXzpp045PXmIL\\\\\\\/GY3Tr8apOWRlaTB1t3P7NClUqAB1dQKRsg\\\\\\\/jfI+NbZtnFwNQ0gezYv2Vov5dsOEYnAZiTkezjpepWFIDCEQKooZYzlvqPjOnSNff3A8bMyU1q9g8kV5moVySjfO5g7A0rPd9Dp577IS144XUqgGNwZdqSQcPSQEu2UvVwcnwwPmXGrp2EaqlqKQx5EEyEo4yt8osbNJWFfhZBRMPbBlqNr62AshT8eMpgta2E4wAVYjBtGx44rqnggkuFMPhK5YDkgUWOzgfszerKc5fW3WqC5D4WoR3+0l3d9uXOtGoTpEXd5PxqBhqTrwY0jA8fMuDNiPsns249v258Yt0GN2W4OyC2K+HEF3YDwc7pxZeV4lL1ublYs\\\\\\\/yGnUJnd2BGuO4yotgpwC0P\\\\\\\/PDu8ug0ybHUqaYAKxZJllxkP92mEuqPBsrBse5AnCPGL7WRzqj2ujcmifyBx6HJTZJ8AJL40+M0LEIXUzeAZVs7dDgosyEEPtkuFI7ytspolNa4aX1JL1ekcEse\\\\\\\/y7rmTJSa25Fi7p69Yng8PNyyjqwCdKlsgYElIBrFWwqITMyfM2Wbh3pJ5WsmfiOhPdGc\\\\\\\/JQKqvbEJNVQxNwnoHiSp3NME\\\\\\\/HqUcLimJZJmkSjASrF6MdKXoH2ymXdLCv79BeLNXdMeWJRpbOg5jZCCLKsTBlKAPcVniZ3YE3NswvlYyNv6bQ7dz4cxII\\\\\\\/G2ePVwovvCIIylZ5vg8A6phdF4fFW5GZvyuWVv7FOemG8NLhWQCoHGPIgUUv2Ws+wb3uA9tZ2+1n0n4+wV5PwBBn8w+PZ5hNZPSNPZyb3kHheGY7fyE+0rUKZjF0wrV2LtWGo6ZQvhnWhyOu8ifMSSRBBpLgHTs8453\\\\\\\/3iX34hTWpjFxEv\\\\\\\/3pwr9bcImqzEu0HvCZ99f4V9+WPRoxI\\\\\\\/K80PtDIKNTKChadPmlarB4v9tI4AiovCNzeR2V7f43DK5CnboSeFxsWgWASpSkMy6v5+xS2Okm\\\\\\\/YiVgj5p4Tn6d4pkXZ8ZOiSAAPY34rvkS0wJ2HKkU\\\\\\\/xa7fGyViGBf33\\\\\\\/bCbi9jpoLGCVSNxDowof1ozB9bXyVnpxQ7Hbsc3ftmthGG7FMtWouT8LAhI4Ej+MCZObylabiliI4X5J+QL0Exi2KrHGIQAdnh2C34ak8swOmRu341Rn\\\\\\\/ycbN4OaByGYk+mGLt20scAzwUgekUGBQ9N\\\\\\\/B4bheb4qqPOjD+vouqioeitKMPis2SW6xXQJbX\\\\\\\/d5ccR\\\\\\\/GhzfndxeKqaVs5cnyM8bG0ZpuS2gzJsodat1KNcaQPYBkZnmQqE3GKlK6V+GUk3amLGkZpJZqmxoB4o9nagDTl9gvP7PJk7DCYFZP6yKhufm6SchsLkDyJQTe1CHT4V0Grrelnr5CNb26uKR0X71DNoje76+nsUUN18ZiJsx07xo+qlzF6Lypyi5FNkaDWBBff9z3zsuVi+Mg\\\\\\\/fWZgjq+dUtT4yRaYV4YC9awD1E7OByi3G0DldFfe1+ithgGeTueK5n2O5xtl9Srz5LyeCnFENmchKfncHgGzN2TGjJKs1dtRqZjiiGc6Mivz9lIwlu7gg7eWDQki\\\\\\\/JckZA5bxBhMSuUTjZEHArPp+pJWPZgGPrV4rL\\\\\\\/HhzQkyrVIEzpyR\\\\\\\/V1\\\\\\\/5P01vKNZ94eBh3dOk0LLWU5EXdUa22to+TyEoQnvQbJRJ1kKT\\\\\\\/nuxwCaig8WBFOp4kY1CjM9fahiaaJGnbu82Ms+DOSf+LtpJYpvKCjcZ1u+8cpm14rfWXj5L+1Qldo8AMd\\\\\\\/vs2Qx2i8cIKRs5VuLamtAg+4YQ1RrAbrRJLkbU2m+q82FMAu7UY2NxFf6EHknmu1ZQFutpcWh2AlKhdu\\\\\\\/ICd28X\\\\\\\/bn7RTNAbxOnbrielSkAnH0wU5Z+Lj7RxYd\\\\\\\/4TH4kwHHlou3Bdfqxd24npYrJv6vTF14TnsRXqQRRFSz7TrtlcofW3IVTzH7pBpym4jIkzJSOSaXnPB\\\\\\\/xtKU2veNdrWYKUeO7A8\\\\\\\/pgOk702s6U\\\\\\\/CxxE6g09NkAhcdgA6OCUEvyaKqQUEMoU2SOf0zf0gBa67LIBjxQ\\\\\\\/OHwSHhdCi3gxITrPxDU3QwJ8whiBlS8M7ksb9Q62q6O+D507H+5IvPtQ50omB5yIx09+W36Z4kW54fO0upZrROMOwErDvJMBGomrIFus8TJnO1Psxtn1uoKlcIWBBjGNfYUHmSRyzel6f7el6sYii6MHvOEMcWGNJbWXkPUAujaWv8AWFfyfjU0+2xUNMjRhfVgVSvKLoRgyHDK1wrVbUr0enrG5mFVJVuEKM\\\\\\\/pkAwFPomniV3W+3YQOv8G5zyZDwHSXz\\\\\\\/vH9lcH4tADdygdEuOVEvcSLGDLodNNQLzp7HoeB2WAGEz1jU3ZsWlTY8h5laeSsClKTBi5TdYejektMKUK6VExTfq5zEJ0ODrPSBDT+Mtf\\\\\\\/pQ5WRQJEKXFIs8UTnfh6\\\\\\\/qViYvzU472oa4neXEn4Q+cjZsAxGOcS07n+99LI1W7dZuMM7xMmKcQB2\\\\\\\/VI+k5pQFsifOAv9Y0LrQQph5uqvj\\\\\\\/G+Czbp6ZsD9z7XiE4y8JC2YF9pF6TMRH\\\\\\\/6Q7zq2QKB18ufizu\\\\\\\/mCXMWCoCJ17e1f0XW1k9icOiMqEft3VXu7XT5BHUcTBLhEIqsHXSNmShkiJWJQpM0Z3w\\\\\\\/nrhmVQs+Oue20zpWhkJzPB8VAZYSm+cxVVxbw+s59rg20oGaXm+gsIh8122bWK8eKfifSwa5Cv2dZrbHbz2XeeFxtHevhtHREUoUfXFNHpotDetB5D\\\\\\\/C2rq3uPBN+rOcOB23hRnOAbufEHKgDTtC\\\\\\\/E9OIaZceICv7PIQC4jsaCx1bi1T9v491kGWIYFXsjCXRk8zceWkoKAiPtBi\\\\\\\/YFEx7wdyrtPAlLol3fv5R4enPITwXuAOVBxDCIU90pj0a+PLD2w7cNH9EoKStRxjItEiKCgUsvFQjFGtMNT48dkW\\\\\\\/M+nWZQcCmo9lgHnf3kOsnR9Lx0GMNrok8VRWTjVfXFUVJjtpggJcm50Ihu1N5fM9tUJtjGuktRmoLRCPxF0G7UzFos3X8uiO\\\\\\\/hLDajZNCEUMHmU858cejfdATn1rI\\\\\\\/MovxMY0kxgqUsnZbJFpo9lRGeG9FuIlaOXpm7k1\\\\\\\/DBGO+sJlKEHZ2EvzyrJsHelAhgiZ4o8D8ZVHWHoCVv4JzdIf0G2a\\\\\\\/GerGqo3z5LVBQKjuJKPvFrGNl9RLX5XJdLU0jpMy6OnnUNuPzNn5Xgd+ILOuP5f0PPK9vvQVM3qkj5dSyhfR1oVQxLL6CzN3\\\\\\\/Oi7nex0c28gMezDI9GAhESIFZVK6W+gUrC5Gplb67odKZU+QX0vvmdcOpORuMXI953qKwV9ouRYA0HC4xgGpQI5qgNmNTN4fzXw7EQR+8TbvHnHrfiZsaJ2Hwa5TjU660nyviUKnF1oRp11+yYF3o8+caNRE0FmwSIEMG8Sey0p+gZ\\\\\\\/YQEKdBPU4bkSBzIz1ytBhK1Wkoy7YCB7Iff2II2qEx5jpI4MqaC+fWtd9agIcLEkDTs38gm0CuaVSN2CDj+rMc4RL9v5opKUbymmNdDvMTfkG6vbjz6wg7o1rfLgb25F6HHCKjgvjnbtSseUvLPPwCN2BVkg\\\\\\\/kfLjbdSVoRyC5JIp756uZyU+EMe2vkhUPvwmPgcGevBNo2DfEHGlynhSQdV0mW+vMQrF84NgR+A+vy8PL6\\\\\\\/UlvnUvtSi2wMj003hJlAK1dVJjmN9BZgxSNNrQVhq2AdbIU4iWoC5yvvBc\\\\\\\/1krfNnHvb8SGl7tWzXF74tHOn7F1NXqnJ2ghV+zQCecr4ZwM68s+3FkQuOToe8jZo1+BB3rnNx97eYx1Vz\\\\\\\/kiYXSiHgrk+YIEBQDsmtYGdimPheSjyTuHHyr5Q1oilTp063Nn5fhfWT7A0xURdWpWPcvk5KZ3ZmcisVlc\\\\\\\/BZqtCxWcTedUipvnDwRRsa4zGYpLhzUzPFggZl++AkOt5Pru09EOoS9yb1mHHqFnTQ\\\\\\\/8lCKlqEp8iZHKJalPzRL8g1ph3ivg+daOT\\\\\\\/KAQLZ6T8Pls1sfYVjlV9q6I9nQjNsXWOxvBdPLZyjH97PSD8CAMkpaQOYcM0DzWprkWTgsbsvgdZQ7Ir4lij9skrvVz7dnrZZnZ2hv9U6VoGvXnjFdKMfJ+IicmTXv+ihHkCD7LoOebblK55wNsltewI4UQ7t+oSOiAHuQZ0S8mRkSKikPBKZbc8obRQn45rvS6KXl40M97qqkVtdcuYRX1QEdYX3WR8iD1UXc0V0si0pd+gOE656gmM4cRmZbiNruNtFRJ4bcCUYQc0j5xSGkAByuycgtYeyKZJ7y9No2\\\\\\\/AemqmmIkmkznVquMPEOyfNHWwIH52AUWC3esl8+RRrShRvihlY8P5awD48vsSIfGB+gbir+TqTSx8Kqk8sV2t71j48x9XqZgycTa9hP7vPtjSJ6AUz6qlISOaiznsVTYJIczs25x+y3+VoVv3bOmNONt\\\\\\\/0PyGFqZFFTQoWzTHfXNi41R8n7PEoxk6FIjfoIpdCXwQM2GVA\\\\\\\/o4bx8wAhVTiNlNQ6iMBsfKvHUaNuN+mS8joBl4hbuUoKUmx6B8WvvHKAbddod\\\\\\\/IJZmPUx8lsq9dMtVBSX+Pw8iSQMBXkqMy1yCNjX\\\\\\\/kI9SYE0V3VtYE2uiyo1OMYwIV7uaGnl1vA1YL8t9A8xaspUHQaCX\\\\\\\/BMEkTaVpDvSTHx0Zq5\\\\\\\/MMJ51tDxlVqC2VNHuk\\\\\\\/zdUdA+CT+k3fEV+Fpy8rIR3YjfQOJH3qnepDYRwb0klKrBIdqi\\\\\\\/doJRTlrRKDOqSx8OmgwymSd5ZxjAJ9dbinmdnV2bHsYjESLc8VOpjTPDgNHizaYXU6emfKexu3Ay2KxRPGEijczJme\\\\\\\/lhJwsf85ngl5wX1POxbEKf7IFRulc4T6zMreMCLGkbvuYJLHu8QUlS22CWgiSiXH+lQpefpJhmm2l6xCEAKnkf35+h5lmmCgJQfXsxZbLQQA6m8WQOPrFG2ykUEVCGBNHPep5cFPw\\\\\\\/t29iT5W3KUP3sfx2cpk0J8HQe5+uTMdPC\\\\\\\/4FRWlMyFx5BlbZaTD4VhTdxGFBfnarn+g\\\\\\\/lVV\\\\\\\/jO\\\\\\\/VMF8NQWoCVrBgDxnvzOnKh7kVaW4VMCRJ\\\\\\\/vCWRgvpmc64qW23psvTML5njLHUNXgXqAgh+dKDHKyf8zt+CwXh0bx823BRbNbZkps3QYglAsBa4diMaE1dnqokcNOadgV99KPPR6l7atiYVRoF0\\\\\\\/wHrIkfiD7Qrg1fQ\\\\\\\/WzxuOn4MY8471qNLe1d1IbwBswzKMgsY0c08EqAsc2Tf3IoFT5ACvXAHNCufuEOoeO8tNhc3EYrh56xtyCAxGoZjaWxY+OHgSPZLkJ6\\\\\\\/fnXzMqjaj\\\\\\\/VRoCfX6V9bjLZSw+jOpbQRPlOlbxRz0jZ+iWwgszvRoTj\\\\\\\/wHQSPwEmZkuOGD+z12F\\\\\\\/K9ofZUbvUpUPdVx+9PSczqxik\\\\\\\/gbQDbjJuDtLWt0tMLsNtoxKjQ0VFrMBzITM337ScRnssCUmiBHrKyx\\\\\\\/XDORr\\\\\\\/KEVkTey8IZJgNyEMpji+8n7CwNXsaAdWDhiOeIL2MP98BD9NgbQRfBelc9mVIiPb4mnF4T+8xgS8+xeDar0UnCUa\\\\\\\/ecVC0oNp9OPSUHE\\\\\\\/kbP+DtkOPbQO8nX\\\\\\\/VZSdTq\\\\\\\/batYto\\\\\\\/wjg3dl6d7sXdnTy8CUKBpYw334lyMFNi5PihawMgiKyMy0YgO4GZkOwONE+jGNRysBngcW2L\\\\\\\/nGlVEHqUqPmM179NsVPwTTpRpiqhHXKRYcK4FjJJl9nr5+zkK2LoW9Zyp2kLVk8GL0ye76paPuBXJh7GXXhaZ8KNBk+eEAG4O6bkS9gaB+Loiqfl1kIlr3KZ0r4TMltRenQTkW48v2SgxdnJZ9iFcT0p5ZfKZpFCV9FJQvH3x0NLWfXdXz7XsyQPrLkjiJ96Q5pTGQFiD6Cx5O8j5fRPANtLJj\\\\\\\/Cu\\\\\\\/V9sqr+iiP\\\\\\\/Z7ZJa2ZKQL0VLJLsUHlq6IRHky8QKNTI106mgS5RykZsB9GsXy4A92lTCcTISkuS+McF2GJDKzCmbq1IVAXYQIDbvGxw+GVbJwbPaCBofYMsYrggwQlHjG4VTw3ykbzSHPuTS0D4sBEfKEx1UlAiwR8lby6agZZdDO1ZOYDyEnFtd3PPAzkcDHlS6m9EC43xDi7TyGmaJGVRPDnppWASwvVwdz7Q1q17MM0H9IAVUma2zqwoIsr3YiTIS3txu54aG9uPuajF1e2bODjlgTJ51GMDTmBj1Lq0zClbAC0VmvWUNFPjaAASnNbUkqxxlUfXrq7fO6\\\\\\\/Dpe5dwEWU9jDTWLKjd6Uau1nfjkI5aoH\\\\\\\/vB1A1M9G2EO1FnvOtrzgzhgmfx\\\\\\\/aIvZiOjueDmswYYtTRwLLCBGhyPjMorgYnWtCI0fNVFv4TCCz0ToUeXrRcgQ\\\\\\\/g9dG3l00IcK9w0Tik3QoS00PgxcwusmhrAl0hQgEq8nC+mC0pDirUaiiX0kMP2uo\\\\\\\/nTC6LvFoxmoR7xjUnK72cYhYHHBIjduP1FC+e7\\\\\\\/3WLBIeckSgw1\\\\\\\/8LgfgKMYrwblsHlwZhm1mzG\\\\\\\/HkaCSZBibWOPDgly9PGg0fQ1QKbY0bqlGZJyyL6vKrOo0CfswZKHUp52dG52xu9r5aWo6wgUZVWTxzSiN3UhpKvV9L0NlepL3HMwLA2sCLjSgfSScvi48yiL\\\\\\\/5kH9dG\\\\\\\/UD+UqttmWMMU\\\\\\\/qWb1UrG0+OVS5P3AccRFfaSpXJZeJoR3rwq8FbnGPxtYxhI8VM9O8irjHThbE5YFPTdPMPkb8qFMDzlwMNGpDkTxlyUJB401dgkkaOduYAxRFj0F18Za33SO7MgLIpHAwakDG3HaVbwglRzUGXh4jtKb\\\\\\\/VEmB5CMRymDc6qoFSuubi+liWScWm9vYnoZxf3E\\\\\\\/SF70sEr3RhrAwQnI4tnf+cfCx50hG9xQv4JEz8ASRvr2jO2iNAF5NGpAkijELJApmMoXMHc9JvPS58g+kypv81pJ4orZJTc+nUpQ+1\\\\\\\/HGjkBLlSiugow9sKjYXSGJTc5G5HnySIlIm0RAU+6RVIf5a0P7nwtQ6eSgtnQSmoAZctHnSIpXAXRnj6pdzUJHvEO3u9Ny7nm4aRIXIcFahqXTjI3CGsrneMc2JTaBSNxvAaOFrF\\\\\\\/nODzqn71GCY1yz8b4d9oPHtiMUuNf2k6xm\\\\\\\/X+vOvTtqVjl9k2IiyrYjllr2Y1ksWaBKJn9xSfqnaQcxQI2zPoN8tlwM9ycymMTSBhp7vdxInmrnjtP+6jcLmdDyVCFcDoIUcwEu87+\\\\\\\/yrAUlLrpLk97mzd06MlQhIdSJeoJVHMWuc3j2giHi6vmeBMCaMH3fnXrOKfPlXy\\\\\\\/T3K1jt93xcDR7rqrAqcusoYYo+0waE5SHXeatbRDxfSLh7MVpeHve5i0g0zYr+2R5kHpZmfcsB4zqLDksWsUi6zbgI8hM8tqx1vqBHgCGCrebfaGv\\\\\\\/4PZ7NWtkErBsyInVMvrhOIhX5guO9kK5X0oN\\\\\\\/ixRTtlIDGoUT9xrQKRW\\\\\\\/PFQ3VBFPLfgtmW5oU65moLYvV9sqHHaZZa3bHkPADCgXfcPHSuXyD2\\\\\\\/fuJ1cmFU1SWoPs4wy7hQ2yxQi5GsI4\\\\\\\/mu3TD+EQDAN2ufmTyWgFcPux\\\\\\\/hGRvzU\\\\\\\/UwVcuxm277ekXWLI8AbrzWk5yjLTdKQIGJstsLqy0GoqiDMtWNCiTswKn9RGyoXoJrvTAJUE4KJ7BsgwcNeJwiIrHq49A6+pXDUrn2sl54RpA84tzxjocY+RykQsZC5xdtoJbTzqnybuUMN+atu83a1fEi12CJ50n5FE5c5YURLgGyFvf9lNco2cAV0sjQ69arGp1g4QMg87VpATweCSsj2h8p+n8nhOcpm+7DL65K\\\\\\\/he2QZR8joNyjraKWAJYUFkwyE71djzwMyyzQfr40S7kveQBvs96JPO9eVC+Ln8QGcmo9ES+kkeckjRFi9LkeFOk9q382DzSpEBBVllj2mrC43KsW\\\\\\\/0LesXBv4pXTgI51N4QDbmj0G7wfPBrPV3\\\\\\\/TCV7AANKIpo9bjv0ZohNl\\\\\\\/EpkEyBvGLxZHXfRSG2q6LLAsAFe0E6ht5QrnfrTi8Umr9BTMwkd0fM40\\\\\\\/q8b6cQhMUjDthSISvW8Eta8J5GMfqvWrCTAvAAPW32f7qsgBmHTkIQRJXD4rOhu3ofGb6MsFfGELe+DOc25XULJYCYiJuS7LMxeJSPUnFzSaJmRAU5cvylnK+\\\\\\\/Xi7ijNRDMpIYKHjF6pjWoaFU4K08\\\\\\\/2a6fXKMNmkF8adnfeYsV56E9DwSToY5wQdBTHCRnU3x7v4UkfCP3XlEV30FxRqGfHsQa7OIfbwEt++JjtA0yzjU+8FaD2IHPMHjyIurzEekaWIf8hPY11OT+IPKcaIqSoCyCTXPPA0PVEqUPpeObl4I7ZG0egFsi\\\\\\\/18g1oNZkPLSOdg8s6vL8eraq46jS7CmAgrtYuOgsqmh0tvdoyRjHPxFwqzmog1SLZ5NQVWrslw\\\\\\\/JIE0ZLbJ3mxar02B15RwPqJmSy9mtGZf3Jkw1cYKCtLDHV123tWgRN9rWjiQzJaMb07ylqDc37TOlukJIPufxUIchwJNasvBX6DXJGc+6nBiwERBQayo\\\\\\\/S9soLF1jDJEW4Esmhg7rIwoZxr+i0HLqhS64MxSy9YlJRYn4LKJ7PGjrzGGbhMvGOqSog4XaGYqpXhVSe68jvSLfteMT4hz3lo6d0bVxItjn14qNvFZY8LRvvd+AlOz5Cu+6wcfiVU8EIXLWQB1nl\\\\\\\/kLJt5ZSSBZrjqgP+NDvQiGIBciQdgqKB1M9hqJG7nqVyTE1l8Wwy9w6eR5F3J56gBJZAOeyO4AijGIuOSDfqjX7wjG3WKqvjZaFi0zovUORLH02Vyb6r+RLRD+H24uy1zZisbOluXzmO7nDlLQ\\\\\\\/3x7565m2J4ET7zZ524moneRovU0HnbSN3cDMi73BjgYgE4kpqZ443TNzICtqlZL2kx23s461iynT05M65MpF\\\\\\\/Mqe7VqpvseVLCiAeJKqK+dsVTPnD7hJF4MTVzr9\\\\\\\/DM0OpqONCEM\\\\\\\/da5phehanNRIxgt0rtefo0MDGu6i4Q41B8s8sxoqocDEkZyjvwQl3NwK7foQOdggFzegFkhe3yGveqs+sGBIeLdqc9AUCyDQqJ0iCt1EHy2EgHe4Myu+d0w67otp9eImGa8s3ZS6j\\\\\\\/9ExnCdXKa1D5rtki8VbgNc2kwc5tCj05aF16pErTO4QfQ2BKwOH1DuC1zWPBYfJ5lzUcZTjrmmy9qeMPVC62PUn28IO0UhFzYq+cPSB2BjsuPlWQe74p0Nj2d\\\\\\\/i6F4Z095hee9pie5VdQ91Z+OfuQ6+SrriTeCNe+1vQqKVM9UPdObK8FIlSqz0vcj1FyHxBeOk+79x3\\\\\\\/n2ghPKrCvqTe+UE5VNT36cRLbC+FMlQ5fmiDplj9G0vK9NLo0uwutu96qADdXy8lbWptdSi6p\\\\\\\/\\\\\\\/okXAVpwy0qHxMIBG+ZkB43lLGBAL4XOYxpnFzrYgOF2xU+zF+\\\\\\\/hzRzO5J9peVqb8E08RLSr9QUmNgsUkzYMVYYftJzsup1YL1HiLjotIAL9QYreqDB19y5eu+CGpWUAgpjgq3wdF4rRqlac1MreleyZv+r0MJSinr3Wsi9fMKAbn664\\\\\\\/oWdaCPQsuFGG\\\\\\\/k1BNIv2MZ7SA93umngN6dDk5E1+ZXojenXfHITLL+vlaOWeC2ooRt6XBKhdTk6dk0dfxeqYjLYyQOQ7OcVhnm1hDybb7GUMMcZiJYMEc5M83QFOEdHYwKik+LAT2dF7uB0iV4yeh+2WNXYrnubP6Ohf0ciNQk5n4TjCpnAQhQFCTK\\\\\\\/hnQ\\\\\\\/6TaA3rqZQZvpnoVZO\\\\\\\/9bM14m38lIb9UdaCvAfYgXQa6N8C1J\\\\\\\/IBiK7Fxr\\\\\\\/BRicmSzhapH6aMv4yEQOGGS9y3JCvcpGvqPx+RsehT7jmHitl2ZDN9LNVwUZ4LsjLfjDvhn1unjVPj\\\\\\\/X50RzhjrCaiFv13CyBTezxxkgPdJ5BkndngXUA+mzFsHVD+ahIoQUqI3lHzalbxbVtGFUeYBrbblIAgU+v+unAYrepfEmBZTOmT\\\\\\\/agKSYSeYfnZhu+Ii52vnQNj8Ionl1X8WrW5NXPaTWakayIICSLbGmyV9m+JKkd79J2MXMk0PRXhTSHtEkAZF3UcA2yeYgiTMxCK9giKKXEjR\\\\\\\/qHVpvORoNlWjlrE2zpwnDv4KB1WuK7JgTUUJH4JDXu99j7sgdvJvvGkaW46OZxolfJIDL7EtB6idrQsq4RIUp\\\\\\\/lVBjj2bTOgMFjRoPjhL6uuST\\\\\\\/fn06W3G4a\\\\\\\/NeQmrjBliOPcTJjSYYiuOyx3mq4qWZuXozgjbCSlH8w9rC+M479PJGjmZS2iE8X+958P6P0lkwoYkdmaQQ4E75XrKnizpP\\\\\\\/AJLw7SDaKGUOlnaB0RgehHac0gAC\\\\\\\/9lxJSXFehYXBN9XmR8\\\\\\\/wwbNTpSAnrz6BmHktpg2cfAzZ4MyAfTVHh1wgSsTKOLgT\\\\\\\/qo0JX+VtrfCtqL5sF0RKaYaqQmV70EU7iEF4JVvy0T9zjgznMq+F18Blwv398l9nAW\\\\\\\/MGdbjKfXFfQy41BndguRkXOR89XDhRt\\\\\\\/mA3wXBq+IAbHlgYfzfae3Zv2iCXvv1350xn43zgsbDilohFrGZ60nWNNi8IpAycHzipvfyy4Sp4mXLLBQUw1UCA2lYm\\\\\\\/zbgwjEvaOq5xHc7IGfMNdnwl4AcM4B9pcmxVlC9lHaIMMiuYk4AkSiQWXpIPu8G0Ba68jCku3JUE\\\\\\\/+o1bHILQQJbbr5f9bctkEPrg4t8fw8SraNOhsbhp4EE9+HRGTKMjM9gdVrOV7C1kL5PqBX0qOJZYtNeeVXShXttPrwuxXPy1GtF3pez0ookEFeOQcYMJazxRlxOG38pNkwedOZf5iMR5IIy7U2KnDfa3lTag36gONRiPDDPopXZDf8N4HrRq6uDF6TlQYlMNNNOEN9T5uX35vtUlzD2dwSxAURc9N2s2Dmk2tTlxre0J0KT0vDxUU7ZSoQcgaiFcUGXsE3OUAUYdzltI0GGQFmdRsynCPwHCI9p7xXAj8kWUrF4+50XLWykC9MoMFp5Kr\\\\\\\/23fm\\\\\\\/mOWYArmvf\\\\\\\/x6lbmRwhWGw0XgksE8isy2aj9KtNKPBkanjiybqX4xtds52Ewo0KPFIOVZmhcHgaJsZ\\\\\\\/7w\\\\\\\/\\\\\\\/OTjl3cbsn8rdpcPasKbdOUDDegv9PN4pBBzRj7kxIIDxpo6jP+fdIqHcC9Esjt66w2VkkwgBycPgc16SkPU2BJ8NwXEEzkZ5zAvLsiFgA4K5Xcv4QJtYMY64xmQbBk3b17quwdJ84Bkq+BBuBsVYrXR40d27sEKZUuAKGoOYLwCc3awYRirWnXZ74YSbhiz7dIX3ti5upc2XE9pqhFKHfGmYvhmV\\\\\\\/FlVbrldJb1Fee3AN5RfPZXQ+XS7rPMzEiuQ79giQzLWz0felh\\\\\\\/VPSgXu4dWFzCEqG27pFBTMhu88hvniEXmuuyUqjjgRkUQIDR8SGaD9qKHHZEXwlnIf1mhglKRO9CVDGKLcq+3msNKy\\\\\\\/QuDHegdUwLk5lSIWj8oPA5FV0Bo1iVeo85jQCyOTNl4\\\\\\\/cbp0+ekjjwxiD5vFRwFsmtiGgBH9KT9TDHajnm8rrOQBqgsoawHplArB+odColkeSQbcN5S+hGxB5TOoXoZe9IPcQaA3\\\\\\\/K5kPp4LAe0k6GYdQJotBCFgnhnf1MK2hCbuWRsUA397ZKjB3LLXC4VRRA8JIOsF4BNXVXDp8KAtDaaAtMbsDhSj6CFuyZRuuAkjuKhTC+NbYvF1I4y2NIs8Fdo0ALF8as9sN\\\\\\\/Benum8Ip27OVFMmXnKR0cDXHBSho4s2csVn2UTDr4x0OdGgU0sSbCCfLciEhS3JSZcFaa6UAdcHLEMtFZr\\\\\\\/OtWSfGYG+e4OC0XYgzi4tf6v+AgpxWmEfWi\\\\\\\/zxoi1Ip7r5mr9Oqk\\\\\\\/FMoa+oPePM\\\\\\\/l+SumJ1PH\\\\\\\/3nZdSdSO1\\\\\\\/VU\\\\\\\/Emjr6gJXoSaGjUkXb+xfOUZLclXBnqGlUIoublN5PQbJvhLcBk\\\\\\\/+Uylo0+bjBgv0VNhPxN5Jm0lv\\\\\\\/dYI6k3BH8V5Q0cCIfn6D3L0tqmAL1IkICx88+Ed2HULzFTMYtQo2U7AEZKSEZGUnduM6i5N8dG3OpvVhB012iAiYGpRjTaMlNGgYFYIOxuiuS3xoc1fhXOHxJC8GI\\\\\\\/QyH3NYQ2I1yiAMgyaQXYW98D3zLszmL5JVnMZOXOtHEDQIMVD63V2ZjlC2y8bhhljBXjPxqcUvQeykOoTmktGk0WJFJcVMJjqWOGGt7\\\\\\\/uymHEdsBazj0yBrt0BQBrpk0MuEEojA7G\\\\\\\/JidwmIL\\\\\\\/P4gXtv+ud8BCz2ann4A+5992iHrLzmfGbfvQ+Ft1sImkE1a0HBLKW6FpeceMP8k\\\\\\\/xcvk+HGg33QC6717ZWz9SmIPMNX8EY2fyGUIzZGPaBrWZ2NaHE8z2K3ZHJ4m82o6bw6eKnAt7nteN6qN+9ZSCzfJ55BrqisLoc0wZJLqCM2\\\\\\\/0Ny1jubZa5GP0QoMg2CMNei1aHyxrct8I+KfDPmQf3wAH1s4yWRNfPFBD18PSHvGGkwWVoAmXjK6WXzkH3zWo+K0eMkh7Is76KnhMBYtzbQoDz\\\\\\\/JnQpKJyAV08xdUSn4xtOzyOxD81RBKHU7NPTSycZFxOSnvcVwMF1uVInZ\\\\\\\/Z49+vjLPjoYWVI5jBxHKDpnMlkc7waq+IZ3bnBX0vhMrVqVO1KlGhRjSpDTutQR7TJwig2FfFgJKdFnlrP3tcnkDjxxPRFI6MKPmDe1VB84xvJXjL7Ud2c5NcplgGfN7hznLS0eyeAqtHmGOZGlbYYMI55d7p30KnPlSahOyE473IfkBs4+vkxTIHmXpQ3wRevEv8Hof+d6SpHh0vQXsEVViUb+YqR3j2Q3FnrfcRE604XFQP1hsoeFye5Nrg2XMlJD3HW7pvqBwC1ARCs96C9HPiLtwER9+hNZ2CiiCy9MlBlbxwXZ\\\\\\\/xb5RLkG4maamIlTzgtmxCJxaInY3yzf7MsIPdxWzNJsMOG37BUWUCZRWKCdcQwUacCwqQzvBu9aPrbt+3qy74TT2PwhU4dy75q9xMo3m4piEa4OF8ljAb00xA53\\\\\\\/hY4AHpLkPEzOBNlyc+QwdYAjmin0V3mezaeK4Ji2eqW6EhkGbUcO26HIx7BGqAylAXmNk\\\\\\\/vFxPsVTAZ4fUC5VINnVYclbZR\\\\\\\/PIQo1MAKZ6HZRbRu6PsFUKlM4zf0YYfNgMMsUGhhWLrV60E5AF3PnLlaSUJixyv2VSvgPMIHCERphH1Q3Lj3d9PjV9VIwH+0zlPDQ33CcZrs2qyN0IncmJsQOicNll2kQHXU15HiY3XfzuAUw2dK0\\\\\\\/z5a+\\\\\\\/oLe4hOR\\\\\\\/Bn9rDOtRWVSzk5MqRryWC6uQ+wjapHXSuc8K9ucu2R3jELnxMu9tFsb1I4Ym7uKDNIqennf2STiYFBIO4Dn1qugHk3FBjOu82M1OmZjQ4ZhnwvN9iQ6GWKTWt4hy5BesY48c5yZcf5kk66CgkcEbQKP8bqoiFLMMMv\\\\\\\/nGHRkgHRjzAisuEMeEQp+7UgoNbPCzyA4dGDx5UihGyDcQwXIbFXJIPLkdd1hdN2Rtdh1TTGvurJq92lcm5cCq+ru2Y7yq\\\\\\\/1CdsGrpJwAjFwgcl4N9sQwLP6A2fhN0cyDLnl\\\\\\\/2UZTszYFP2G62H+asu9fJHZarOlLofxZsFqKcsdES+J7FAcpSsGJEc3oWh63dHVtCQkQYv2wkaHkp5ZlQSoRfVOIbzod1T6hjcq6UFGrhwEJxz0VU4piLvjcEX+FKSUcjJBJ8G44iIJ5O0Gae0hVUwmpYD76vHcBG3LnpzFVmH5qdhVttEueB3555KdgAVyhZcyA+YgjexIiMAwLXJOsDUjNyLba8M613DHK6ULCjPNpdEPiDsZzHQgETzhq09y+4LQ++RT6Sv5t+e5uvZNg7DNLd16EhTYIFuBTmWqyMi8menaQXIgX+M9O40+Ykm29L69zlkyVtHOJwlKXUWIEdwDxpF\\\\\\\/h8\\\\\\\/DpXUYJTj8fCFQfOT6vljOPeSUtUN2kjhyHUvze\\\\\\\/Sw0htS+wuTPX45omdcKgZWTriqusArrpF8ihuLxfzWJ8nd7B5lMDtHPAqdQwhDc37UEnk0RXRmRFj9XE3+npKL1u+XM2R4ONPsaBqFpvJqq32UPwy2lhYTXL7BCd88JCyxu3icmOSdOrc6BaR6XqcPik6eMsDjXkBrt+gU\\\\\\\/IIM9YQt5VgAVoYgYMLZ0o8v\\\\\\\/gZ9w5RFLXUvOXVipCoJKV8nbUOGcWaUNNDxyYJEAI6PX92NRYNInUlISCfVFPzEug\\\\\\\/RP9ClNL8KEq61DFYFUWoqU\\\\\\\/bExVaNR8pNRhnA21Vf3YKWVeGzBUZPp2VVWHxsslP8v+MosZw8lGE2ulprbUpZwGoR6\\\\\\\/pde4mEbPEkQXuantyEwXRRTf+aoMrf20ohsu8WNGo5o\\\\\\\/W0t8WIyB2ILH1I9oVdoACK\\\\\\\/HPvz0A2Z5SdxwysBjrvgQ45TBw+WAdLkpjdh6Fwb1GUZxtOoVinBB+YnXWadZhy+MwkvkrzrRI8FZlJ6ieK0Sj9IG4kdxQ9z3M6S5bYJKf0UB6t3VBciBYSGJ5rzkx3JSbL7MTuwqLKrRwFgINhrhQ5CWb2N0fZRvWJxdwS3jNKT1ZgJRmEMntCzfy\\\\\\\/5PW9LZjwzGYJPlhJyA50DUQu62lOvUqvzefxKns\\\\\\\/X3wXqcfOL7Mazn\\\\\\\/mLS+ske3ib+sS0sbZCg\\\\\\\/hLL6AFws\\\\\\\/hPYi\\\\\\\/UVZYaxqYzOIfumHDbrcn3Ku8cYuTsG6Ej7MY4CBT6pH6Kh6ILY65FZdtS7gl704W6E90SNT+86c8msDNVvPZa82zdJamo7oaxkjPJqMhr8G8Xo3pUHjkNGh+l4xfZFT5sefvHG7fvYdpvtRxKdb8JYdhXZSpkv\\\\\\\/HmpirxvZsnjhwLnr0PjA2PjWsadYjAEwIplvQDSw\\\\\\\/PbVYfnoL0ZfTFk1h5LttsqCOrS0F3MZNoONUhWYuaXjUdGHNU4hYfkUa2R5oE48GJJgnYmcNUruOIBE9+0VF9aupt9pZFiKZbo61v1IaLvV\\\\\\\/jZf96q6JINwTf0jBq5P6AVQ4mnZ0cd9OoSMPUFoxhwSoLbPmUxzwIfjkxR1fivKmBi3gzsFRlt75ESJQo+vfQd3XtlTVcHKdtUgTPcWj5kqM906D\\\\\\\/Wh8CPf1YOx+MA6gXPWqu3q24UNFHZs1dJobGkIbApFJ+quosxlT8xJe8QsSD7\\\\\\\/KOuRpjS6X5S2CPChDiHURDhREXV8rfe7otaO93LU8hLSmuoVjbGDG6rOu9pzzqElXC44URFOjajk4jWkMmOEf7+H1ry\\\\\\\/OofFF6D4fkePX5TPz1ror9pnX97nmd9sucPHfrZOsBxgPWujgqrWTal1sWIYdVjHSbverzhVdSP7Se0tyCb3mOyaMTpPsePf+MHgVBff+Dos5+dw7K92dtUPKsqUhWuZuVET40AQRW5sZdUHCM9eYfAKSN2Qco579LEkE3LwpoUfa28lA1TnnifckEHrh1P5B5eT69tY0hHEsVYFTc+dkQG18FB3\\\\\\\/yA9YaMIdkAQvnEn6e0XS4HGmwfzcMz9sRo\\\\\\\/uUgielUUDLMo6uT\\\\\\\/Jx7qbo62mesPlzSnMKYkIgJYn370HZTdtHSZzSTUIl6y4Ro3kmU90do9CuQRhlDiCBfKTy2ZXmC2p7YpWyHUWWy92WJtV2ZP05evBICzjrHPXFvSf9noIx3X7oV23\\\\\\\/rEh7telBMKxocvBtBN9F57yTtN7qoF+gAZ1yICHZ6y3R6tKRd0e9fUSittbsUbuO0Nyi1WKO+IzKOX73EuJAa6Ky5m2JmcI+mywVBUENXjUy9mbSFWs0zMcnqfFecLadiCKZPAgnnO2OFFXEj79oQx\\\\\\\/BVU\\\\\\\/lHc+rhS\\\\\\\/3Y1JlJBTH7ksazP\\\\\\\/ByZVMHLDbegwSGwk42y89Y6OAUEs2R\\\\\\\/gDeFOdvrbRHHzuv9\\\\\\\/CiGjDHwJspb+eRYrscIZthd23ys1fp6OMIJsRL1VtRhTGirnsqhqKxFaZ3ZOzFG3qnpG0vcZStV+hEc\\\\\\\/XPsWJkavZAbF3bQT9Pz9HV4dxjX6VmHLJKldB+BfOLQefafRSlulW6pU0vauZ7PR6GHxNEeUUpiJx9jUXGqEp57wKRAapngZb\\\\\\\/CiElHkPRQcapaLCZhNcmlFPRCW2Gm+rnXn9\\\\\\\/4Dr7enF\\\\\\\/J\\\\\\\/hWCmj\\\\\\\/jOou8Hx\\\\\\\/mN2EqZHtoWKdeLP2bxr9nZcE4nNkeSjGxpZsJI7nxwdSI2qmoid9FRlgekVac5je+TjtL\\\\\\\/IMdHr\\\\\\\/zwSlr1UEQ4v3pIHKwulpgzR9D2INUVhr\\\\\\\/ei1HJgx2E3Qc8GKHVU3xZCgxw8epweW8g4+3lR7+XX8impTTNU2Czwi5l9L0hYQ5ViZb5fqXgzBg1IuPkRbgSuGi6MQakO1wxZVKEF3F2IcuEkqN7KirfwIauhzQHt2rScjFX5hP9GFg\\\\\\\/BKGHsLkdDSkhQ13yQ8V8Vb9QF3ja+YOb7yVWqbiYTzGqSYMtQ\\\\\\\/fZT+Zf5dFRAC8RqcuYDx1QkeFB4w4ulpOx+l8FvTqXHI0ixkZ85xt82WOh0AGIqqUBPLp6asdIwXVT2yFtIOF6\\\\\\\/Xe3WbtxWF2i4fahkYG0udVd68Puy\\\\\\\/t3DRM0JiliFQu3E\\\\\\\/vtG0T0YoXrH2wL\\\\\\\/pO2sVSlp9mbKAeiPLYCaaDYXB5lXmQoALhenSQKOQhiz0CgtbVN6XijmGGa5l8e5nJzpJuxs8XBA2Y1sBccUn5Jh3dgoNpDnjpKsFpCzQncPlftta2GxHCyeNOBG3x50avWixJQnZO5F\\\\\\\/qOT7ERRkZMG3629hB9Aa4BfCiSn8om\\\\\\\/K4p81KwO1u+BP5y6OtNe02VExrqZTkP3ERA5O1l286oDxmcfrbNYzCWLFhZo5zaV\\\\\\\/n+N5VGLBmeMAFPneJAD4cjikITPW4N+bFdc8rdl\\\\\\\/d36pUk8nkVpo49BrOugg3OqbHJ0fBWxeeZchKWq\\\\\\\/Bn6dPJJdH2t2OrBobmG3lhLddGuUxRpipkBbcSOY1Zic95f0CQnTn6Jh8PkPC4iXdHOzAZ9Gj8FWlO4VcEiY2\\\\\\\/m7TmPUrLsmDKuRjHORLWenRAN9xmkxiaVsVARO72a8acm8Kn3cDxXIE13v5OggmuBddetmNnQPX1GhUSsoEID1ZokqPt42z70bCsvsW5aI9n9o2Qbc62fMGus0pvZ3RIbS63MuE10uULNQ\\\\\\\/QxI9Rk7ujU5TukhqjWQJvmysIZZ+pcYucp64mMfnjo9ducylPhPRcDQEIJXUGbEq2RDTb2uQ1NUQvq+1h03NaEWN1cZx2ycHYQlDilv+0MPPPHqAaiGNBbr8PvRm\\\\\\\/zqsEWZykQrQFHRt45+g6CE6ZZRP6lRtIibD5Q3w0zqW8+2\\\\\\\/8JdjEp6zFK3aNysScm0zkUJiwE+Ag9gjc5MdeZp66jRGepLxr29gJEXeo7bqbfPkd3tzITBZp56QAlRBAxHCPI4hhR68DQP40wu4vHoD7vtU9068hWre6q+Y6pLkHA7uvbb0p5tC5HLM1PhY5xHgAF2LBrAPDZ32jygRMtljsD4KvWYrMpS1eusw8NcaxjDchrQbgKZOSVpJ1NPPhPtU\\\\\\\/CKFsQEUmWKYQtYg\\\\\\\/BXZNeLdNz+CGTAqwOyNw10jcI5FBTeA4tTo2tzyCRe7ODiJ5BDfXfb9MkHwPqgiMPqBx7NGPsGRy+bnUcUcexoqoqBo5fyBSdRt33Nw+H0NeLbFXuPKU4nhfchjb+bJAP3RFdDqEsslW9dXeBg1ku27AcrYNd7wx9g7iNG6zH3TM\\\\\\\/FuW\\\\\\\/LL8RoZ4YBDj19cPreOhfEeevwhbdiPrCZ3w0X3bxQmIixHXF5Nq\\\\\\\/YvRK0wYjHBZItfb5Bmrz8pMZURXuwrJjJ7XYX\\\\\\\/rNSxHmBobbqmu+DMmMXKDipPPzucI97FwdrCtF9a7uCo\\\\\\\/eLSao7gK3Uu1MbJyji3gUUKb1hHsHmyaF3qouGrdd+BMnVFZGdTqsG2QeRLO5sIS0xy8TEpvBoRVRI9v1rJsjgnhdJO9FfqTkE0HSZoesNAqC8su6qLjn9HpQwnIxKA6YWJs04mlJGxVy0550fB4q6SZZTsAcoX1jxQNZJ9pELe0oZRl0SMMgnTlsStvDCbvx6sgBJx2T2GXVJRoO7IfVYv\\\\\\\/NMZdeqWVlHkbqQa2HNiHs+tiXAU9+AWr83EqQsh5lXzLq5S5aOsqUrRuCshxSlBOOpj3oY6pg\\\\\\\/nD1VbI47Z6RSJ+0tGpJ82DgK5TkU6\\\\\\\/+CM1SKbFgMPzFea8BOY1yIz\\\\\\\/VIZ2hIAA57QZyfUQk64Kqp0NgTWC+qshNAQ0y68zu0O7bA7AdwV6nVHFThUl54avPAfyG6Zf73EebnFNCTSAR0VmsarGyPWRmVVe64m+oLMvwqLE15lv6jLG8RasRmHCE1P6UJ7lBDiZBFdasOrpObl+D7Us6PqrVMpIbSAiQLuKFZqYbWfU4gudFi8x\\\\\\\/jum7WSTNf\\\\\\\/ausEmhPiPE7KKMp2lvPNz0WfUM4\\\\\\\/I9KjayiBaFPYHgV\\\\\\\/qTNidLAM8D3HBmqOE5pmDSaIRHPabkUne6vMr2yXgPaFhp9e04ym7bibG1QQtTV685YFGy6tuVLvY9DXjX+u0pUDServpKG4LOUk3Bof7y2vUlAWrUgzpXwkREIZaNN9TILwPMXSmoj2bDUzUQNlpDT13utDWJytDvQ0XIhZZpP6PVr6VxDhRicDy1AX99sxe5+EzkzzmcHV1EVSk8AiSUR0X6BHSYHYkHIX1U6MFV6Y3jHVXZWoZ\\\\\\\/2zMV\\\\\\\/C7qJt9\\\\\\\/KTNmkbiGHr43YiITROW1Io1dSdYL9Zhl+4CFhRohJaXyVPs0RQGhZl+DrXcoESEkDOpBw12FC+9p1DwzKBI6oQqkPFRVevEw+CyA0qUN2dk8kdcI1cxkLM7P6j4urhkjw7\\\\\\\/IN40AS9apC2t3iXUqWTMaMp7GiGRpuG7pHEguCsg1hFzPBcTRbXq3FtQBOGUOMehzFATLjWmkHiVOVita8d0A6TjGWBSreCWptP3nEciKGWHqfNzpKf\\\\\\\/CZR68uQeND+JticVXbr4VLhmxN21Ej0HMCKc8zgrFwqxNuqBKnwbWljPzl7tte77VpIiiqiWvWlJKPn8vaWLSpArRTAvfe3nptFq8Py8uVDCvwSw+Eyctd2Pp+P1kq6nBOxY0LgjKH0+MH\\\\\\\/EDYH8DH\\\\\\\/\\\\\\\/KhdxfoDerzhE8ACQNyae7V0JHIv9ODxpXmV75sF3J3+S3FtVjFcVwOtuUgd014RKYXMZqZJSm3iTDPlI3uFk+znb0N12am+Dbc8vs151m0hsD7rdYBzAA0720a0DqKWEtp360ZjePhI8jwCMMZ+ne0xxBIGoSWcAyHe7SC\\\\\\\/yWnYGLyCE8uWgLXuBg+uGruSkul\\\\\\\/fRxOFpxd+Rj84Uou6mGMtNtsQN621+5CW5BIbBnLxEkMYISxUqUzkRLU3B3UYR7sORaqWjjBlcMzAxJ51jZ7Q0GKaABAzBIp2er8cHNacwgEyofIogDtoxRRHRpXAjPQghHGUFA1FXqEVqYuehHPRnwAADYRy127nAISgvRhJK2+MFD\\\\\\\/uZuL\\\\\\\/NlLu90HdozKa9TbRRyORFT8212XBfvKoN7KHshJ4eOkMDmYrLtyhk03mPeXbmqHZbyZIzgq2KBEbw+QR3POBiAAVoGA\\\\\\\/ErdKtofXyU7wM3C8dR4kNIUcQ7VSiBG\\\\\\\/6UdcJgJeh8ewakhronqBfnItBm1+ZZofGCSW+YGOZo9cxLgyLvpFKtdkjED7IARvrigpx4rijxJ76Gpk1LGXG3YeBRDFrurf91LrSRd0HXNPws2TuE9l6uhtcJw\\\\\\\/j\\\\\\\/9ACGJPM7m5P\\\\\\\/fbqFiy0RHwNqknULtrW\\\\\\\/t1Ih1cMBJxNlaTGl4bd7tkqZRW42Sj6j8gj7hB2Cmwvvf33gRHFHW2ZLgQi2a7nE0d\\\\\\\/+WSPToRx3Zzd3\\\\\\\/VzhK\\\\\\\/U0vtSunOuX72\\\\\\\/WVbjCAKWIYl3tZDBVVYQpmKNGcF7BCya1KqQ6JwzVtZOJ88jKRG+tcRAeyYnA9DcKkob7\\\\\\\/GrjiE9jnZWL9hkGOTCnVKHyR3TzvMleRD3TShJDPlj+g8LKaGZq27xVjKHQGQBTgalMWso6IiHqE3swdS0aoQok3CtA9jQSJyIgw7VaUGpV3fcpkyiKYgb04fIQuvMWlHBkFtQkrG+0kF8XNdBO8iu0g4G\\\\\\\/KqweJrldR1s\\\\\\\/mx\\\\\\\/G3RigWuhcWNSEFD+eKSfmsgi\\\\\\\/gEobeU8jXlnvMP8U5VQp5\\\\\\\/UX\\\\\\\/LbHyStB1SY\\\\\\\/5rzebOz2y\\\\\\\/KTarrPLSveFMdqv+Y6osjiuwJHm38m5r6YQob\\\\\\\/UDVoQejOTRdKr8CVOlv\\\\\\\/4cOmeZJrdAw3qqsC2SCNoRuyGTfWv\\\\\\\/DrQ0msce97+h\\\\\\\/WcFGninGQmwZGxa9Mc7CHzOCQpQJ86CFfhfo+X6h1GnAlWpt98g9ZvxhTh+r9yidL0bJebZUsAAtz+j1e+YwOg4WttcvhxVlrSpQA+bt5eC8CsBlE9xmPqw0j8M\\\\\\\/K+r2SB8ePnTtCWxlapKG\\\\\\\/We53Py6ESrIvoNNTXVF2Hed4SwQkWfOvZyQGOui0IHrCKRVFlG0JEDqnTdYrRIYuYP+GfvKpvLn2EymmWMJwZipPoRXMtIcYQJxvXaPYc1KECLYBa0aEoZhXPpntjQJBmkcw5SADTAkNt2TjoFQzmdz\\\\\\\/R8X1T8E3BDSuy\\\\\\\/QZy4p9rR9f8Y9fD7aOsNYJbt4m+DZYJD53i66lM1wTeov3RhEsCtVTYbuGOOhOiVfknUvoq6+BDqz+GUZkdxBzkx8\\\\\\\/PY1pBBt0Y9ahgyzSzdi+TBHn5afIGbMsTQ4YvBLe6o5wpBG2ZStdIiHG6oo8ftDvSAJrX\\\\\\\/EakJyFyccfrV5uhsP9uBMWtSMf6hmJUb5Qr0WXGaPdd\\\\\\\/z0wVubiZQqOC16pt7s3UHkHe3lGxipgJhChTpksHh80DbX3DLwuXTIZxRcJb6\\\\\\\/xilhR4Pmj3WdGtKt66fjKqd9c92WWpPz7kUFh3n9p82ahnPRl6fEXDxxZkOZBdaIqw46hqlLoLKVIG\\\\\\\/rfx\\\\\\\/hmNaX3j9fxUoIRSExNH+ZLsPCI+C1abjGk\\\\\\\/I7Oip6b4JWoiG0WyZnPcsUYt8IPe0e5IKnAZTdHxwdjjPx7E2eG+GwksIaLzrpiRVDfaHn3DVKTRxdnR9SRL5lTWk3XWdHBV5Ce6eE3j7lgd9qhAdceZIT9DeBlWYRQQOfZlKzZFT8zeSEVaxQWrkmgn5td4e1XS3rqpXZg48gMmZ79Pmp6vAhX0sM6HUCwCxziZwLMSwweVWgLhFfUNHiRQxFSx5KsonNdTVP\\\\\\\/zNkViExsdgneYjfz56hKtcogtdm5zw+vrPukLb39tgvp6vSciu4PiGzhjsUbLLMLn0kUQJa7NPfm5ugnwemhQ2\\\\\\\/B7rvEAizo2g8LBKxdFdHcBYvI2wcy2KZzt1++Q5K+0rdEvs4kqRTmHhA184zGDNkpLJu8O09IMK4+2fxJ4Ehw1vJHrQkFD8dVqX84MqiBZH8K1PH+I2V2iBJUZtfgNS0dueyipzJN8YvIa0PkFGZnMplY2yWBE0gFq0+FOb5VZ1ZU2anf5jQPNWSY9UwVbMvSn+ORzTBkbb1Tazis85pF0XZBjsBLpbKC4W+rWbrJN7isvAFCXmH6q0MJmFQ9l4ON2c1IGJuW9B5SQiI6\\\\\\\/axXk77LaTd59IhGzinUOMJEKxbrZb+s0MnsooBfM\\\\\\\/OpWbaW+N6kD5d+6GtOFLTkgucvEg\\\\\\\/oLGs9GvzmMmQrgCHi\\\\\\\/9oyfNAyy8RG47VKF72bmw8kYtVaN3GE7jJAI+0Qy0cw8lpbBg1SS3\\\\\\\/VUbx8TqteLVpQ1lmyVgLfSzQtSR01JgcQc\\\\\\\/Zx3iEAgQvp+UD7260BJja4kWc1N8zA1b7WAsOQyU0KCFR5iClbN4yqX26AVflNZIs8KvHechw5n5dExHIOM5yCab1vjK6nXmPAlF7gYLiHNkdn\\\\\\\/fl1wjlpNinAohw++SiV+P0eB5bfEKqDkyyiMieHYY52DfYBWQskZ1DyNpmV+ftDRIkP\\\\\\\/z31cBK2T7PeRkSuRyB88HLG46q6o+76xHOpuK6zTb6Uat5uqTSFDV+TzBRTiROhBBQ1kjOklFImlzQ31v4Zt0S9V4gQredudbhIqI4Dt4TfdtcodOadWnkYJuDf\\\\\\\/UvgpgNWfMi6\\\\\\\/dlgTB6h6hiyvHlaJq5K+by8N7JWDoQ3QXmCSJc4R2c4\\\\\\\/EDi+hxUtG1ncG926yfla3KfpOJ9ccoNpSUogEFijDaeC\\\\\\\/wtNDf+3PaS35JnpC1tyjvenFjo8OUDxS0fGFDyZu\\\\\\\/Hd7FrjHIfC\\\\\\\/q6+GD23JWi5\\\\\\\/9lfJrusQbbuD8GqrDrVgekkoY3v1QkN3QzfLAYeFU24FLEv92q9YdoZWsHR2suWhm\\\\\\\/mDGQZ7RCV9R5cCZ2IVV4TWsNxsNm9OIU0xgB5ldSre+IKNv3CXKQoz729ROOTuX6D6DbpzAEvUxPDRVzss9GO6cR+wYJtE6t5a9BgPTddGooDs+XTEikIH38gaLJpIXB0JmoSf+APPKoD7qWKWTbJf+4kMPN0mMRrLQKDgxWZx4Lo97u6ivE0p0zMq06M4\\\\\\\/vd1X7an3\\\\\\\/6PlsBMLcW7d0XSU5pUn3bPzk\\\\\\\/4lh5JEZrw0hGVQouoxQhlJn\\\\\\\/th1PyV9jef3IvSyBbesjvAC+CLOZZSjMNLzU01eZJgwuTynl\\\\\\\/slx6bSjJpdiSoK\\\\\\\/igz57Fe8quvS\\\\\\\/JjVn48zGcIcLUCAw9L2b1r5U+ZzFGWXSPZlcoWJYHl7ho+xwomGc2hsBCvmIKw+1yOOwblOykjw6mIyDy3\\\\\\\/19z9BDtccONIcbsbl62caETlLWnLXtT6Ly9ARBqcgwt5bX7TjidJ2SUyB2RP8dq8Mnn1ozfW8y4SegyDMUIsvY7SdXYMe97KsNjoSnjJjsMFEZvh8F3osf1mFrwT7on\\\\\\\/yTfISAnS\\\\\\\/3\\\\\\\/QpGil92dIAmT1XscHvP5KgGisYCIEOgTfPmVbYnFLD5VJgtp0kYw9se7tdYkhaH\\\\\\\/O+K7VC3apy7i1cxa67GXURot1ODHcHkKm47rz356h\\\\\\\/\\\\\\\/GjbQLRBSjNLRaqBc3muj\\\\\\\/R0PDbalHBB+wLGP6DuWLMY34PrAfJEc9R+znoBFI2peWV0P7iv\\\\\\\/3a3ms2pDQUGnWpVmoBLlJNO4QBS4htsQpsEsvcfRUPytp+JUwgSegn5PqgaaK6PeDEe1zyKFtud9QQevDce8M9LGES0yeA+Gy+wqWw8lanf40F1YKtdMPZMfSr5pdknNzZkjQyOOJo2\\\\\\\/WlncfDKGcdLZ5\\\\\\\/GCr2FF5EzcjQgqv4LuKaw\\\\\\\/BpD63oAAe9zY8jmkTM266C4Nd8J26SBhmvJq2p+Y4ZVynfAMLwa8OL3qmUg\\\\\\\/G0iDbPKO58JEXAIdmkF9587aNjvUTGhsWus3uHejFx7BczU0x7iHmG2v+lWkHVf7IGCYoPiHJZuj\\\\\\\/UQYH6S+NxZuFVpHCjtfovxT\\\\\\\/2F9xPipkhae7ChQYC3jUPzJkZjOUuwWw3rcMwDzPHkpSTzI+glpxP2w7mffpMKV8amNTf94mLhX8a7v73IGk9AjHq2vz7bR3x3WwFQI8vlv81DNjwmmp8\\\\\\\/ygTjVKNIcjFjlN1CNe3YMh2Njn5WfH7S7TpQJhrya70+Z8aV\\\\\\\/QPYJ1c9akYrow5m7ggvcIYRp+jCpsUZrF7fpYgYZfifNGAK6YDq\\\\\\\/cPSjp6cWrO+ovQZdl6P0Ii1fZyrUge5QFvHJKQWV1vuktaOZfEcOb0KgFPWcZ\\\\\\\/6JjS9\\\\\\\/FmLZ9sFce+Y\\\\\\\/bMh7JJbRjG5sM5kPjEG6nVEMWwoxvm6i+o31vtn6Bx9AZnBU3FygVsNNNs+TP75924cXY5oxjxdSX1ZtEzx12eJcstduXhzUMGOJLWn1KmgvwbUo2ydIN\\\\\\\/HABP\\\\\\\/TFc84xtkYulBkAijzuYMO1OYHbTRNt0w+y+HQ9ytrWwSIsGeyLUu9Qml0tCok3Fud75eGFkcB\\\\\\\/tniY\\\\\\\/wmjDRr+OPyzYrX+17Xfa5v2pHg8\\\\\\\/E\\\\\\\/fSncULAa98I0r0F1vCV6HyVGZAk9jmRJc2OyoM75CGyzserNqQRANl6oGgyVTkX7Ww+uCwez2bZLDG4rOxDMRnOJjGIbTCNi9QGmZd2iSNz6I9FvBoNeNYThNZnO6R2w0OHMupICRL\\\\\\\/bp9E9mLF5Dkm\\\\\\\/qHq748fgpgYiCMJCyh8ohbw7i+HWv62gIgq65NkKFzStiQ53P+Im+G0IG2WMwdAR8F2mlYa5BcvP0dt1xXVcOkHfOZzZGD2LbFNHRxxN6RC4\\\\\\\/Xta9DmMCCqMgZ6\\\\\\\/bwL4LNjbV9Tg2uEtYtVzA17CJNgmO7xpPZoAm\\\\\\\/x3XsfaTDg0uvJfjnmxrlIyqrughw8zobX04q1f8epkMQqnbbqe\\\\\\\/wJD5KMPWYyF1f04Uu2ekXdUa0zVxPjsA5q0Is6VS1aRrCJNmWwMT+q0R1GDXoLyiK3xzr86NWC9iZ5WEuci\\\\\\\/3m5fBDiq\\\\\\\/cF4dvbC7s3cZ7RNniiAE6ml9ZWaWCLyxezVuoDpphsAhn1prX4jjiXQivXqQ1en0VZzC9rmR547lhC12hRUhBA9CJPNQgcmEQ2HB7WAgKCLg1P1p+Nmjf8k0euvzJ9La3Smbwx2VfnW2pcTzZjSqOpxQ421Gx8V0pdKQR\\\\\\\/0QnXSbhgLPG6gPeAYWp04gW57YEWgXSVXRuXmOXT+QOmB+W5aTkArXdBa9RL4vgtbXB5peNuoeWrhsdYEHLT8p7Vg0+\\\\\\\/Bd761T2vpQqZ7L\\\\\\\/TFJf7y7PdT9fBsHzOKETjG1EE+U3MV9VQHbP6GrGpe3Yx2qignuOEMfK4otZahFIo36OYm7n0IWuvYhzN+W1X5SCeSEV2QL3Wm3Mo2K4sLPPloxnuahIE+XfXCSYyiVLyQokfOuJZt9H13o3zF6IbgSgUSWrfgO0Ay3E\\\\\\\/mNYRjFukbgybCNUA80FK2BRih6vzyiSJSrBkfDkz+efFV+TtA3ERAmj++N2Sy0CHJKzIGpePSLOeSuW2fL6fukImIXUmvYFkY6inCDYyqSdP95g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022agoY9NWY8cy\\\\\\\/mciF8aiERQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002229fbf0f90e9e8639f59f16549f14539392dd19422eb4d7681c8794fdc16277ec\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HEZMT30bNc5S45OB96LlTfbc3rqd5UurlJGevqGdPpaKgknPtzIksNsIC9Ums9m6+lx6gaiVYy9oJo8ZjwkQnpBxHXb+dcJ6C3ftzKfHjkTXD7oj3AwTimomyYB62SROyJX\\\\\\\/pd00imUjmgiOAN4rxo2PXI8SGPLDCgJEOu9VZBdo4f1QU3cMI8xBdZWPyBGnR6bzkDmMCspYlBkmrvgydonT5G4+OUnZukDJV+m57s5w4QIcZuQB\\\\\\\/ws+ERt5l\\\\\\\/iA2k0BflRTYkRmaKQ8LXRDGYH8p62AXqugnjxc1PNJmhws8mjkGcmysbXaDeZfapW14X0SJgeXQIGqet6G+G1lzPjLP6m9HX\\\\\\\/YyCAf2bn5Iu+JXAh4Zfeb8veOQ7sdyq5ySmzZ6j7HUbiTAA\\\\\\\/1BOiDrPeE2cF5PqqpNcmjPwct1s555q3n4VI3qYaZ\\\\\\\/Tl3TlO09KJ62f2TiEklYuflAjwPkE48YT2LxSsY0+4epKCSB\\\\\\\/RT\\\\\\\/djdejt9tREKNMQ5tVEwU6OfZBwMpAiL6VsWJBZJCFPnW8HrBfTIQ497zwBbuQbsfFyMflePfYPftq16MPflCouLEDGAtEY2imzHrg9Wtn6gO32d3rd1kqMe+iyFiCMMqGS8270Z9bhipqSf3Y7zHAm5z4xp0eyQ3biiTCS3Wa7Ffvoc0jUgcQclCHwCtQ\\\\\\\/1w3uq\\\\\\\/mjh9zm3DhFew8HF4O9eYbI4IAZUZv4IgJf9B5vA3JIAKeb5WIvK+5uRJ1tfU8O5FyWXIOSCRr+JGwPKoKt\\\\\\\/459+kb7zbXvvgygspYlcqIImX6hPipewDuhBK9qn5cl8gRxgqWhH\\\\\\\/BVJ8piGwcMbCubKzekyNqA4oN68ohKRxq1ajmYTWcdlhZLOMkDfVjVLjqmU2YMSfvXFWC7Z3x95hzX\\\\\\\/rDPL296IbSrILAgc0Wwatc+7ND+Qd54cLSFRki4p8VVr4txYYqHTRT3JK8eCdFH1GMRLp54R+KPPRs3FYZlYs4KdGmZSgAZkIzZJjI2+OuYuc\\\\\\\/X+BA1\\\\\\\/Z1+tSlZniO7tlsZgEJbjOprhIAXfpvPF39gNLUURTQS1A6JuXw0FHH33viw5R5nOedpzehq2VpsRu\\\\\\\/uHHRHR12utsJUKqCM+DHRefeNFUhbt9rdEjKuvE6rgndeWNcEiUIWH4G0pCZatvxpmG+pAq6Ftc1bjEzqtZpQ0lnC14x7fQJAi8pTdDDUSYwRJds9yuda10bw2SKmoB2MNfeIpkQp8MjWVwEq9JVUpaZWNzMUgUEZE0EMLYd5w5UAOmGPO1DJnpwJaBGLnGH7dGlq\\\\\\\/Gj1awHXQSOgwTjb5\\\\\\\/\\\\\\\/ZqyQIPwLFEgqgXg2yVsaGS0OyA7NN6g6HzWlXK40HKJNFxA0CtvK7+YBPWWXe2o3vdgmfgvjDCFDO\\\\\\\/DDdN5fpqGSMyzJun1PaFWklPb5ZcjrY+RtBxBgHSwMEYGyc\\\\\\\/lgcaX04CKYzUUlEr2FZfrXhm1Iovl8ZKhE\\\\\\\/x4Stf1brtWRXLaD4msTP\\\\\\\/QaqcM1Th42o7znVRIf1dFL5QUPkS5ONxU2MIZvmH9djB7NOsdXV2lGl+SvvferF8ANweUPkU+Szk8AEz7sbQbDBE3CJg8VoI20FiVq52xDUDVKQToKwUTf5\\\\\\\/H3sfeZbMKuMG7ek2T5L6rA7muD2EJPA5ZmpdCz59aFExubc8mNVIQJr7pAa+ybHQvqblSDDyYiA++rqIWD8pFOBjD0jJDuuP8ZttWHWUP4k6Ow8CxJq3\\\\\\\/sSAI2hKZe9zJrpOgJmPaLseBhKJxQ1AYhEwRu1zpjLl7gKA\\\\\\\/HCaDbiXyB+gsGKmCu21SZvU67TpnLqsblf248JgA7sigtQmbrkjGN\\\\\\\/DTX02Xb1WeGS4aw7y4+kbexA+6DhhPt+f+iXYMHP+L2An\\\\\\\/\\\\\\\/QoH8ZAhBBZFGnb6ZK2ICSCEQrG7luq4GxTLtWn3fi6aB0fF8I2OS85m89QolIdQ8jbRfuRxQjO4wQjfY5De2y62Egd2OCEhkw3AjZUMD7Wb+Tk9r9IytvNB480\\\\\\\/Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QMV2QuPCZV1q0YvrwbF4LA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a32f1c5c7c7d2645bb3f4eedc3db0731cc5785dc4bf192bb3c0bf0efbb4ade1f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459259 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/3/357" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nx3WI_U3Z-sTQBrc2i0vgLPhoEHquWLHJuhDqDsFIBY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/3/357" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VL_MLb9picZjMG_W2pSqv6KdJovPw81C27tlvacjUOY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/3/353" title="Production of a Natural Antibody to the Mouse Polyoma Virus Is a Multigenic Trait" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/3/371" title="A Screen for Round Egg Mutants in Drosophila Identifies Tricornered, Furry, and Misshapen as Regulators of Egg Chamber Elongation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459259" id="node459259" data-pisa="ggg;2/3/357" data-pisa-master="ggg;2/3/357" data-apath="/ggg/2/3/357.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Linglong Liu</span>, <span class="highwire-citation-author" data-delta="1">Yanqi Wu</span>, <span class="highwire-citation-author" data-delta="2">Yunwen Wang</span> and <span class="highwire-citation-author" data-delta="3">Tim Samuels</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">357-370;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001503 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459259" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Linglong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linglong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinglong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanqi Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanqi%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanqi%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yanqi.wu@okstate.edu" class="" data-icon-position="" data-hide-link-title="0">yanqi.wu@okstate.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yunwen Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunwen%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunwen%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tim Samuels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tim%2BSamuels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samuels%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATim%2BSamuels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/3/357" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459259" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/3/357.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459259" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/3/357.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459259" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/3/357.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459259" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/3/357.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459259" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Switchgrass (<em>Panicum virgatum</em>) has been identified as a promising cellulosic biofuel crop in the United States. Construction of a genetic linkage map is fundamental for switchgrass molecular breeding and the elucidation of its genetic mechanisms for economically important traits. In this study, a novel population consisting of 139 selfed progeny of a northern lowland genotype, NL 94 LYE 16X13, was used to construct a linkage map. A total of 2493 simple sequence repeat markers were screened for polymorphism. Of 506 polymorphic loci, 80.8% showed a goodness-of-fit of 1:2:1 segregation ratio. Among 469 linked loci on the framework map, 241 coupling <em>vs.</em> 228 repulsion phase linkages were detected that conformed to a 1:1 ratio, confirming disomic inheritance. A total of 499 loci were mapped to 18 linkage groups (LG), of which the cumulative length was 2085.2 cM, with an average marker interval of 4.2 cM. Nine homeologous LG pairs were identified based on multi-allele markers and comparative genomic analysis. Two clusters of segregation-distorted loci were identified on LG 5b and 9b, respectively. Comparative analysis indicated a one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet (<em>Setaria italica</em>) chromosomes, suggesting strong homology between the two species. The linkage map derived from selfing a heterozygous parent, instead of two separate maps usually constructed for a cross-fertilized species, provides a new genetic framework to facilitate genomics research, quantitative trait locus (QTL) mapping, and marker-assisted breeding.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/simple-sequence-repeat-ssr" class="hw-term hw-article-keyword hw-article-keyword-simple-sequence-repeat-ssr" rel="nofollow">simple sequence repeat (SSR)</a></li><li class="kwd"><a href="/keyword/selfed-progeny" class="hw-term hw-article-keyword hw-article-keyword-selfed-progeny" rel="nofollow">selfed progeny</a></li><li class="kwd"><a href="/keyword/switchgrass" class="hw-term hw-article-keyword hw-article-keyword-switchgrass" rel="nofollow">switchgrass</a></li></ul><p id="p-5">Switchgrass (<em>Panicum virgatum</em> L.) is one of the dominant C4 perennial species present in the North American tall grass prairies. Its natural habitat extends to a larger geographic span between about 15 and 55 degree north latitudes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hitchcock 1951</a>). According to gross morphology and habitat preference, switchgrass is classified mainly into lowland and upland ecotypes (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Porter 1966</a>). Lowland plants are tetraploid (2n = 4x = 36 chromosomes), whereas uplands include both tetraploid and octoploid plants (2n = 8x = 72) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hopkins <em>et al.</em> 1996</a>). Aneuploidy is common in both lowland and upland plants, although octoploid upland plants have more aneuploidy incidences than tetraploid accessions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Costich <em>et al.</em> 2010</a>). Molecular marker investigations have revealed enormous genetic diversity within the species (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gunter <em>et al.</em> 1996</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Narasimhamoorthy <em>et al.</em> 2008</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zalapa <em>et al.</em> 2011</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zhang <em>et al.</em> 2011</a>).</p><p id="p-6">Switchgrass is a tall growing and resilient species. Its genetic diversity has historically been used for soil conservation, forage production, game cover, and as an ornamental grass. More recently, it has been selected as the model herbaceous species for use as a dedicated bioenergy feedstock crop (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McLaughlin and Kszos 2005</a>). Switchgrass is listed as one of the major biomass energy crops in the Billion-Ton Update report (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">U.S. Department of Energy 2011</a>). In a farm-scale study of switchgrass grown as a biomass energy crop on marginal cropland, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schmer <em>et al.</em> (2008)</a> reported switchgrass produces 540% more energy than the energy used for producing its cellulosic feedstock. They estimated greenhouse gas emissions from converting switchgrass feedstock to ethanol were 94% lower than that from gasoline. Switchgrass has received substantial attention and has the potential to be genetically improved for higher biomass production along with other important agronomic traits that can add value to its use as a biofuel feedstock in breeding programs.</p><p id="p-7">Switchgrass is a wind pollinated and largely self-incompatible species (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Talbert <em>et al.</em> 1983</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Taliaferro <em>et al.</em> 1999</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Martinez-Reyna and Vogel 2002</a>). Because of this sexually out-crossing mode of reproduction, all released cultivars were populations composed of genetically heterozygous individuals. Recently released switchgrass cultivars were primarily developed using recurrent selection procedures (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Vogel <em>et al.</em> 2011</a>). Those breeding and selection protocols are effective but require a lengthy time period to develop new cultivars. Consequently, genetic gains per year are relatively low (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Vogel and Pedersen 1993</a>). Molecular tools and genomic information are limited in switchgrass and need to be developed. These new and quickly evolving technologies have extensive potential if incorporated into and coupled with conventional genetic improvement and breeding programs for developing superior cultivars.</p><p id="p-8">Molecular markers have been developed to investigate inheritance in the species and facilitate the construction of genetic linkage maps. These maps are fundamental for switchgrass breeding through marker-assisted selection and elucidation of the genetic mechanisms for economically important traits. The first linkage maps were constructed with 102 restriction fragment length polymorphism (RFLP) single dosage markers (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Missaoui <em>et al.</em> 2005</a>). The markers are distributed in eight homology groups covering over 400 cM. Developing microsatellites or simple sequence repeat (SSR) markers, which are tandem repeats of short (1 to 6 bp) DNA sequences, has gained substantial attention in switchgrass (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Tobias <em>et al.</em> 2005</a>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">2008</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>). The desirable features of SSR markers include their easy use, high information content, codominant inheritance pattern, even distribution along chromosomes, reproducibility, and locus specificity (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kashi <em>et al.</em> 1997</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Röder <em>et al.</em> 1998a</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">b</a>). A pair of genetic maps using SSRs scored as single dosage markers has been developed in switchgrass (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). These maps covered, respectively, 1376 and 1645 cM of 18 linkage groups that are expected to represent the full set for a tetraploid genome. <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a> reported that the two tetraploid switchgrass parents had complete or near-complete disomic inheritance.</p><p id="p-9">Marker-assisted selection is more efficient when molecular maps are well saturated, as high-density maps provide increased opportunities for detecting polymorphic markers in genomic regions of interest. Linkage maps developed using different genetic backgrounds are needed to better understand inheritance in the species. Linkage maps constructed from different populations will enhance the understanding of the genome structure and gene interaction (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Semagn <em>et al.</em> 2010</a>). Two full-sib F1 populations were used to construct previously published maps (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Missaoui <em>et al.</em> 2005</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). The F1 full-sib populations were made by crossing two selected heterozygous parental plants.</p><p id="p-10">NL94 LYE 16x13 (abbreviated as NL94) is a self-compatible genotype selected from a breeding nursery of the OSU northern lowland population formed using Kanlow and other germplasm (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Liu and Wu 2011</a>). Selfing NL94 and other self-compatible lowland plants can produce promising inbred lines that offer numerous advantages in breeding, especially for utilizing heterosis by crossing selected heterotic inbreds. We have developed an inbred population from NL94 (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Liu and Wu 2011</a>). Accordingly, the major objective of this study was to construct a more saturated SSR-based linkage map using the inbred progeny population. This map would provide a new genetic framework to study associations of molecular markers and agronomic traits of interest.</p><div class="section materials-methods" id="sec-1"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-2" class="subsection"><h3>Plant materials and the selfed mapping population</h3><p id="p-11">The mapping population was recently described by <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Liu and Wu (2011)</a>. Basically, the mapping population consisted of 139 individuals randomly selected from 279 inbreds derived by selfing NL94, which was identified as a typical tetraploid lowland ecotype (2n = 4X = 36) based on the detection of the 49 bp deletion in <em>trnL</em>-<em>UAA</em> intron (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/001503SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/FigureS1.pdf">Figure S1</a>), a special marker for switchgrass classification between lowland and upland (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Missaoui <em>et al.</em> 2006</a>). The decision to use 139 progeny was dictated by our genotype-detecting equipment, which enables the organization of the entire mapping population in two 66-well plates plus a small marker screening panel including seven individuals and the parent.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation and PCR amplification</h3><p id="p-12">The genomic DNA for NL94 and its progeny plants was respectively isolated from healthy leaf tissues using the CTAB method (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Doyle and Doyle 1990</a>), with minor modifications as described by <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Liu and Wu (2011)</a>. To avoid allele dropout due to poor DNA quality, DNA samples with smear bands of A260/A280 less than 1.8 were extracted <em>de novo</em>. The working solutions were diluted to 10 ng/μl as PCR templates.</p><p id="p-13">SSR markers were amplified using selected primer pairs (PP; described in the next section) on Biosystems 2720 thermal cyclers (Applied Biosystems, CA), using the PCR reaction conditions as described by <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> (2011)</a>. PCR products were separated using 6.5% KB plus polyacrylamide gel solution on a LI-COR 4300 DNA Analyzer (LI-COR Biosciences, Lincoln, NE). The band sizes of these amplified fragments of SSR markers were determined using Saga Generation 2 software, version 3.3 (LI-COR Biosciences).</p></div><div id="sec-4" class="subsection"><h3>SSR markers and genotyping analysis</h3><p id="p-14">A total of 2288 switchgrass SSR primer pairs were assembled from previous publications (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Tobias <em>et al.</em> 2006</a>, <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">2008</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>). They were compared to each other using a specialized blast program called bl2seq in NCBI (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">www.ncbi.nlm.nih.gov/BLAST/</a>) designed to exclude redundancy. Non-redundant markers were then selected for polymorphism. In addition, 354 sorghum (<em>Sorghum bicolor</em>) SSRs from <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wu and Huang (2006)</a> were tested for their transferability in switchgrass varieties “Cave-in-rock” and “Alamo.” Of 189 foxtail millet (<em>Setaria italica</em>) nonredundant SSRs, 80 were taken from <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> (2009)</a> (a primer pair “b255” was excluded from their primer list due to the same primer sequences with “b225”), and the remaining 109 were kindly provided by Dr. A. Doust (Botany Department, Oklahoma State University).</p><p id="p-15">The markers were initially screened for informative segregation using a small screening panel. Polymorphic SSRs were used to genotype the first DNA panel of 66 individuals, and then those having stable, heritable, and reproducible markers were genotyped on the second panel of other 66 individuals. At last, the information from the small screening panel and the two 66-well panels was collected to represent the entire mapping population. The markers with greater than 10% missing data were genotyped again from those samples that did not have data in the previous genotyping runs.</p></div><div id="sec-5" class="subsection"><h3>Marker scoring</h3><p id="p-16">All codominant markers were scored using the same segregation pattern (&lt;hkxhk&gt;: locus heterozygous in the parent, two alleles). SSR-amplified fragments were encoded as “hh” (only one upper band), “hk” (two bands), and “kk” (only one lower band). For dominant loci, “h-” was scored for presence, and “–” for absence. In both scenarios, “u” was recorded as missing data; this included unclear or ambiguous bands. If a marker produced multiple bands with the same segregation profile but with different sizes, only two main bands were recorded as segregating alleles and the other bands were omitted as redundant information. However, in addition to these main bands, if those markers produced stable secondary bands with different segregation profiles from the main bands, <em>i.e.</em> multi-allele markers, they were separately encoded by primer name, and the band size in the base pairs was used as a suffix for differentiating between them. All gel bands were manually scored by two independent people. Raw genotyping data are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-6" class="subsection"><h3>Segregation and linkage analysis</h3><p id="p-17">Several segregation ratios are possible in the selfed progeny derived from a tetraploid plant with two segregating bands (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/TableS1.pdf">Table S1</a>). The goodness-of-fit between observed and expected Mendelian ratios was analyzed for each marker locus using a χ<sup>2</sup> test built in JoinMap 4.0 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Van Ooijen 2006</a>). Markers that deviated from the theoretical expected ratios were considered distorted and were marked to indicate different significance levels (<sup>*</sup><em>P</em> &lt; 0.01, <sup>**</sup><em>P</em> &lt; 0.001, and <sup>***</sup><em>P</em> &lt; 0.0001).</p><p id="p-18">Linkage analysis was performed using JoinMap 4.0, and the outcross pollinated (CP) full-sib family was used as the population type, which enabled the analysis of a self-pollinated population derived from a heterozygous parent. The linkage map was constructed in two steps. Initially, loci were grouped into linkage groups using the following parameters: the independence test log-likelihood of the odds (LOD) score ≥ 8.0, maximum-likelihood (ML) mapping module (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stam 1993</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jansen <em>et al.</em> 2001</a>), Kosambi’s mapping function (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kosambi 1944</a>), maximum recombination (REC) frequency = 0.35, goodness-of-fit Jump threshold for removal loci = 5.0, ripple = 1, and third round = yes. After grouping, loci within linkage groups were ordered using the regression mapping algorithm (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Stam 1993</a>). The linkage groups established from the third round of analysis formed the initial framework map. Then four to six loci distributed evenly on the framework map were fixed and the calculation parameter was changed to a LOD score ≥ 3.0 and a maximum REC frequency = 0.40. This step allowed us to assign some of the otherwise ungrouped loci on the already established linkage groups. Two independent linkage groups were accepted as linked if a marker on the end of one group showed a cross linkage to another marker on a second group through the “<em>maximum linkages</em>” function of JoinMap 4.0. In addition, those unmapped markers that showed weak links with mapped loci at the maximum linkage parameter threshold of 2.0 were listed next to mapped loci as accessory loci and formed the final linkage map. Markers showing segregation distortion were included in the final map if their presence did not alter surrounding marker order in a given linkage group. For any markers with an estimated position of less than 0 cM, their position was set as 0 cM, and the positions of other markers on the same linkage group were adjusted accordingly (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beldade <em>et al.</em> 2009</a>). To compare the collinearity between the initial and final maps, the same markers and their individual distances on both maps were arrayed in Microsoft Excel 2007, and a function “correlate” was conducted to obtain a correlation coefficient.</p><p id="p-19">Linkage groups (LG) were identified to be homeologous if they shared common SSR markers. Linkage groups were numbered based on the comparison with published linkage maps (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). The designation of subgenome “a” or “b” for each LG in this study was given according to the named subgenome of a corresponding LG, which shared more markers than its alternate LG (<a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). For a LG (<em>i.e.</em> 7b), on which only four gSSR markers mapped and no bridge markers were found, the original clone sequences harboring the four gSSR markers (<a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>) were blasted against sorghum genome in GRAMENE (<a href="http://www.gramene.org/">http://www.gramene.org/</a>) with “near-exact matches” set as the search sensitivity parameter. Thus, using the sorghum genome sequence as a tool, LG 7b was identified and compared with the reference switchgrass maps of <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a>.</p><p id="p-20">The linkage phase of each locus on the final framework map was obtained from JoinMap 4.0, which automatically determined the coupling and repulsion phase types during the estimation of the recombination frequencies. Chi-square testing for the ratio of coupling to repulsion linkage phase was conducted with the online software of <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Preacher (2001)</a>.</p></div><div id="sec-7" class="subsection"><h3>Comparative mapping</h3><p id="p-21">To search for the locations of the mapped SSR loci on foxtail millet chromosomes, foxtail millet genome sequence Phytozome v7.0 (<a href="http://www.phytozome.net/foxtailmillet.php">http://www.phytozome.net/foxtailmillet.php</a>, accessed on March 31, 2011) was used for alignment with the switchgrass mapped marker sequences. The parameters for BlastN program were as follows: Expected (E) threshold = 10, comparison matrix = Blosum62, alignments to show = 100, allow gaps = yes, filter = yes. The output was parsed manually to identify those significant hits with the lowest e-value and the position of each query sequence. Only the final framework markers were evaluated, and all others were omitted.</p></div></div><div class="section results" id="sec-8"><h2 class="">RESULTS</h2><div id="sec-9" class="subsection"><h3>Determination of nonredundant PCR markers and polymorphism screening</h3><p id="p-22">Of the 2288 switchgrass gSSR and EST SSR (eSSR) markers from different sources (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Tobias <em>et al.</em> 2006</a>, <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">2008</a>; <a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-76-5" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>), 19 were determined to be redundant PPs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/TableS2.pdf">Table S2</a>), in addition to the 4 redundant PPs reported previously (<a id="xref-ref-76-6" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>). The resultant 2265 nonredundant SSR markers from switchgrass were screened for polymorphisms. Polymorphic markers were preliminarily identified if they showed segregation in the small panel of eight genotypes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Of 1105 switchgrass gSSRs, 377 were determined to be polymorphic. After amplifying them on a panel of 66 individuals, 7 gave monomorphic amplifications (no segregation) and 58 produced unclear amplifications, resulting in difficulties in band scoring and subsequently discarded. The remaining 312 (28.2%) were used for the linkage map construction. Of the 1160 switchgrass eSSRs, 210 showed polymorphisms in the small screening panel. Later, 48 were further discarded due to their unclear amplifications, and the remaining 162 (14%) eSSRs were used for genotyping the entire population.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/3/357/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A gel image of screening SSR primer pairs for polymorphism and reliability on a panel of NL94 (first lane from left side per panel) and seven selfed progeny. Polymorphic and segregated markers are indicated in boxes. The first and last lanes are DNA ladder 50&#x2013;350 size standards (LI-COR Biosciences, Lincoln, NE)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1153249929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A gel image of screening SSR primer pairs for polymorphism and reliability on a panel of NL94 (first lane from left side per panel) and seven selfed progeny. Polymorphic and segregated markers are indicated in boxes. The first and last lanes are DNA ladder 50&#x2013;350 size standards (LI-COR Biosciences, Lincoln, NE).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/3/357/F1.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/3/357/F1.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/3/357/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/3/357/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461319" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-23" class="first-child">A gel image of screening SSR primer pairs for polymorphism and reliability on a panel of NL94 (first lane from left side per panel) and seven selfed progeny. Polymorphic and segregated markers are indicated in boxes. The first and last lanes are DNA ladder 50–350 size standards (LI-COR Biosciences, Lincoln, NE).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To explore which marker types give more information for the linkage map construction, the relationship between polymorphism, repeat, and motif type were further analyzed. Of the 2265 nonredundant SSR markers, 1022 were of dinucleotide type, 924 were trinucleotide, 244 were compound, and the remaining 75 were tetranucleotide, pentanucleotide, hexanucleotide, or unknown repeat type. Their polymorphic rates were as high as 25.8% (63/244) for the compounds, followed by 25.1% (257/1022) for dinucleotide repeats, 16.7% (155/924) for trinucleotide, and 9.3% (7/75) belonging to other repeat types.</p><p id="p-25">Among all switchgrass SSRs tested, 2221 had known motif types. Of them, GA/AG/TC/CT occurred at the highest frequency of 23.9% (531/2221), followed by CA/AC/GT/TG with 17.5% (388/2221), CCG/GCC/CGC/CGG/GGC/GCG with 16.9% [376/2221, most of them from ESTs developed by <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Tobias <em>et al.</em> (2008)</a>], CAG/GCA/AGC/CTG/TGC/GCT with 15.1% (336/2221), and AAG/GAA/AGA/CTT/TTC/TCT with 10.5% (233/2221). The frequencies of motif type were almost consistent with marker polymorphisms. GA/TC was the most abundant motif type with the highest polymorphic frequency of 36.4%, followed by the motif CA/TG with 17.1% (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/3/357/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution frequency and polymorphism of switchgrass SSRs based on motif types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1153249929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution frequency and polymorphism of switchgrass SSRs based on motif types.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/3/357/F2.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/3/357/F2.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/3/357/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/3/357/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461424" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-26" class="first-child">Distribution frequency and polymorphism of switchgrass SSRs based on motif types.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Of the 354 sorghum SSRs, 39 (11.0%) amplified clear and scorable bands in switchgrass, indicating their transferability across the species. In the small screening panel, 6 sorghum SSRs showed polymorphisms but only 1 produced clear and heritable bands and was used for the map construction. The 189 nonredundant foxtail millet SSRs were screened, and 102 (54.0%) showed expected bands and were scorable. Of them, 8 markers were validated to be effective for genotyping the entire mapping population. Together, 483 unambiguous polymorphic SSR markers, including 312 gSSR, 162 eSSR, 1 sorghum SSR, and 8 foxtail millet SSR markers, were identified to have effective polymorphisms in the population. The average number of segregated alleles per polymorphic PP was 2.03, with a range of 1 to 6 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461302/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461302/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Amplification, polymorphism, and mean number of segregated alleles in NL94 switchgrass</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Inheritance of markers</h3><p id="p-29">The typical segregation of SSR markers was scored as “hh,” “hk,” or “kk” in the mapping population (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), and only three loci produced from sww-2097, sww-1678, and PVAAG-3053/3054 were scored as dominant. Of 503 codominant polymorphic loci, 81.3% (408/503) had a goodness-of-fit of 1:2:1 segregation ratio in the χ<sup>2</sup> test (<em>P</em> = 0.05); the remaining 94 (18.7%) loci demonstrated distorted segregation, <em>i.e.</em> deviating from the Mendelian ratio (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Of the 3 dominant loci, 2 (sww-1678 and PVAAG-3053/3054) showed a 3:1 ratio and 1 (sww-2097 with presence:absence ratio = 80:58) deviated from 3:1.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/3/357/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A gel image of genotyping an SSR marker PVGA-1963/1964 in the parent NL 94 (P) and 65 selfed progeny. Individual genotypes were scored as homozygous (hh or kk) or heterozygous (hk). The first and last lanes are DNA ladder 50&#x2013;350 size standards (LI-COR Biosciences, Lincoln, NE)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1153249929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A gel image of genotyping an SSR marker PVGA-1963/1964 in the parent NL 94 (P) and 65 selfed progeny. Individual genotypes were scored as homozygous (hh or kk) or heterozygous (hk). The first and last lanes are DNA ladder 50&#x2013;350 size standards (LI-COR Biosciences, Lincoln, NE).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/3/357/F3.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/3/357/F3.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/3/357/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/3/357/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461309" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-30" class="first-child">A gel image of genotyping an SSR marker PVGA-1963/1964 in the parent NL 94 (P) and 65 selfed progeny. Individual genotypes were scored as homozygous (hh or kk) or heterozygous (hk). The first and last lanes are DNA ladder 50–350 size standards (LI-COR Biosciences, Lincoln, NE).</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461348/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461348/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Segregation of 506 polymorphic loci in the switchgrass selfed mapping population by chi-square test</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Linkage map construction</h3><p id="p-32">Under the highly stringent conditions with a minimum LOD score of 8.0 and maximum REC value of 0.35, an initial framework map was constructed with 360 loci (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Then the framework order was fixed to allow the positioning of an additional 109 loci with the minimum LOD = 3.0 and maximum REC = 0.4. [This value is the maximum detectable recombination frequency for our population size of 139, according to the calculating equation of <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wu <em>et al.</em> (1992)</a>]. A lower LOD score of 2.0 was set for joining two separated groups in LG 6b because two of the markers resided in the end of each LG (sww-1969 and sww-1889) and showed a cross link with an REC frequency of 0.38. Thus, a total of 469 loci from 453 SSR PPs were ordered and placed on the final framework map with 18 LGs, the complete set expected for a tetraploid switchgrass genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In addition, 30 accessory loci were assigned to likely positions on the map (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Only 7 (1.4%) of 506 loci were not grouped or placed on the final map. Excellent correlation between the initial and final maps was observed by comparing their orders (r = 0.9873, <em>P</em> &lt; 0.01; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461311/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461311/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Loci composition and recombination distance of linkage groups</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (Van Ooijen 2006), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1&#x2013;9) follows the previously published linkage map (Okada et al. 2010). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum (Sorghum bicolor) and foxtail millet (Setaria italic) are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter &#x201C;d&#x201D; appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as &#x201C;sbi&#x201D;) or foxtail millet (abbreviated as &#x201C;Sit&#x201D;) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: *P &lt; 0.01, **P &lt; 0.001, and ***P &lt; 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1153249929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (Van Ooijen 2006), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1&#x2013;9) follows the previously published linkage map (Okada &lt;em&gt;et al.&lt;/em&gt; 2010). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum &lt;em&gt;(Sorghum bicolor)&lt;/em&gt; and foxtail millet &lt;em&gt;(Setaria italic)&lt;/em&gt; are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter &#x201C;d&#x201D; appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as &#x201C;sbi&#x201D;) or foxtail millet (abbreviated as &#x201C;Sit&#x201D;) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: &lt;sup&gt;*&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.01, &lt;sup&gt;**&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.001, and &lt;sup&gt;***&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-4.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-4.medium.gif" width="440" height="204"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (Van Ooijen 2006), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1&#x2013;9) follows the previously published linkage map (Okada et al. 2010). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum (Sorghum bicolor) and foxtail millet (Setaria italic) are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter &#x201C;d&#x201D; appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as &#x201C;sbi&#x201D;) or foxtail millet (abbreviated as &#x201C;Sit&#x201D;) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: *P &lt; 0.01, **P &lt; 0.001, and ***P &lt; 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1153249929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (Van Ooijen 2006), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1&#x2013;9) follows the previously published linkage map (Okada &lt;em&gt;et al.&lt;/em&gt; 2010). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum &lt;em&gt;(Sorghum bicolor)&lt;/em&gt; and foxtail millet &lt;em&gt;(Setaria italic)&lt;/em&gt; are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter &#x201C;d&#x201D; appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as &#x201C;sbi&#x201D;) or foxtail millet (abbreviated as &#x201C;Sit&#x201D;) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: &lt;sup&gt;*&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.01, &lt;sup&gt;**&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.001, and &lt;sup&gt;***&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-5.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-5.medium.gif" width="440" height="221"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (Van Ooijen 2006), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1&#x2013;9) follows the previously published linkage map (Okada et al. 2010). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum (Sorghum bicolor) and foxtail millet (Setaria italic) are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter &#x201C;d&#x201D; appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as &#x201C;sbi&#x201D;) or foxtail millet (abbreviated as &#x201C;Sit&#x201D;) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: *P &lt; 0.01, **P &lt; 0.001, and ***P &lt; 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1153249929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (Van Ooijen 2006), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1&#x2013;9) follows the previously published linkage map (Okada &lt;em&gt;et al.&lt;/em&gt; 2010). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum &lt;em&gt;(Sorghum bicolor)&lt;/em&gt; and foxtail millet &lt;em&gt;(Setaria italic)&lt;/em&gt; are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter &#x201C;d&#x201D; appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as &#x201C;sbi&#x201D;) or foxtail millet (abbreviated as &#x201C;Sit&#x201D;) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: &lt;sup&gt;*&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.01, &lt;sup&gt;**&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.001, and &lt;sup&gt;***&lt;/sup&gt;&lt;em&gt;P&lt;/em&gt; &lt; 0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-6.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-6.medium.gif" width="440" height="208"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/3/357/F4/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-36" class="first-child">A linkage map derived from 139 self-fertilized progeny of tetraploid switchgrass NL 94. Map distances in Kosambi map units (cM) of each linkage group (LG) are shown on the left, and marker names are shown on the right. The gray segment in LG 6b indicates linkage identified with maximum linkage function at a LOD value of 2.0. The accessory loci, which are nearly equivalent to the mapped loci based on the maximum linkage function in JoinMap 4 (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Van Ooijen 2006</a>), are listed next to mapped loci. The linkage groups were grouped into homeologous groups based on multi-allele markers (in bold) and connected by dashed lines. The Arabic numeral designation of each homeologous groups (1–9) follows the previously published linkage map (<a id="xref-ref-49-11" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). The bridge markers with good collinear relationships between two switchgrass maps are underlined, and those commonly used markers but cross-linked with other LGs are denoted behind the loci with square brackets and their corresponding LGs inside. Note some markers are labeled with both symbols. The markers from sorghum <em>(Sorghum bicolor)</em> and foxtail millet <em>(Setaria italic)</em> are indicated in italics and species names are added to distinguish them from switchgrass. The dominant markers are indicated with the letter “d” appended to the marker name. The common markers shared by this map and previous sorghum (abbreviated as “sbi”) or foxtail millet (abbreviated as “Sit”) maps are labeled behind the loci with curly brackets and their corresponding LGs inside. Segregation-distorted loci (SDL) indicate different significant levels: <sup>*</sup><em>P</em> &lt; 0.01, <sup>**</sup><em>P</em> &lt; 0.001, and <sup>***</sup><em>P</em> &lt; 0.0001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Including accessory loci, the percentage of polymorphic markers mapped was 98.6% (499/506). The number of loci per LG varied from 4 (LG 7b) to 52 (LG 9b). The total length of the map was 2085.2 cM, and the average distance between two adjacent markers was 4.2 cM (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The length of the LGs varied from 3.8 (LG 7b) to 162.5 cM (LG 6b), with an average of 115.8 cM. The marker loci were not evenly distributed across LGs, and consequently, some LGs (LG 2a, 2b, 3b, 5b, and 9b) were denser with the clustering of markers than others. Twenty-three gaps with each ≥ 15.0 cM, a distance suitable for QTL analysis and marker-assisted application (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beckmann and Soller 1983</a>), remained and collectively spanned 339.6 cM (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-38">There were 120 common markers shared by this and the previous maps of <a id="xref-ref-49-12" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a>. Except for three local rearrangements on LG 2b and four on 9a, good collinearity of marker orders along 18 LGs was observed through 102 bridge SSR markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/FigureS3.pdf">Figure S3</a>). The LGs, therefore, were named according to the previous maps (<a id="xref-ref-49-13" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>) for consistency (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). The maximum number of bridge markers within each pair of corresponding LGs ranged from 3 to 16 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/TableS3.pdf">Table S3</a>). The designations of subgenome (“a” or “b”) were assigned to each LG based on the identification of bridge markers with previous maps (<a id="xref-ref-49-14" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). Among the remaining 18 common markers, 15 (83.3%) resided on their corresponding homeologous LGs based on the reference map information (<a id="xref-ref-49-15" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/TableS3.pdf">Table S3</a>), and the other 3 were distributed in nonhomeologous LGs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). One short LG was formed with 4 marker loci, of which none were mapped in the <a id="xref-ref-49-16" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a> map. Blast analysis indicated 2 of the 4 mapped markers (<em>i.e.</em> PVCAG-2491/2492 and PVCAG-2163/2164) had hits on the bottom of sorghum chromosome 6 with the E-values of 2.2e-03 and 2.8e-05, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/FigureS3.pdf">Figure S3</a>). Thus, it was named “LG 7b” as the sorghum chromosome 6 corresponds to switchgrass LG 7, and LG 7a was identified based on the bridge markers with the previous maps (<a id="xref-ref-49-17" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>).</p><p id="p-39">A polymorphic sorghum SSR (Xtxp-46), which resided on the end of sorghum LG Sbi01 (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Wu and Huang 2006</a>), was mapped on LG 9a (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). Except for a foxtail millet SSR (p58) which resided on an accessory locus of LG 1b, the other seven polymorphic foxtail millet SSRs were distributed on six different LGs, <em>i.e.</em> MPGD25 on LG 2a, b255 on LG 3a, MPGD19 on LG 5b, MPGD17 on LG 7a, b159 on LG 8b, and both b171 and p44 on LG 9b (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>). A comparison of shared markers between this study and a published foxtail millet map (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> 2009</a>) indicated that, except for one mismatch where a marker (Millet-b159) was expected on LG 6a (or LG 6b) but was actually mapped on LG 8b, the other seven markers showed consistent correspondence of LGs between switchgrass and foxtail millet (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-40">A total of 20 multi-allele PPs were used to determine the homeologous LGs. The majority of these PPs (18 of 20) amplified one locus on each subgenome, and the remaining 2 PPs produced three different loci each. Six homeologous LG pairs (LG 1a to 1b, 2a to 2b, 4a to 4b, 6a to 6b, 8a to 8b, and 9a to 9b) were identified based on 12 shared PPs (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>). The result was consistent with the LG naming system described by <a id="xref-ref-49-18" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a>. The other three homeologous LGs were identified based on bridge markers with the previously published maps (<a id="xref-ref-49-19" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>).</p></div><div id="sec-12" class="subsection"><h3>Distribution for segregation-distorted loci and the ratio of linkage phase</h3><p id="p-41">Among 94 segregation-distorted loci (SDL), 84.0% (79/94) were placed on the final framework map, 13.8% (13/94) belonged to accessory loci, and only 2.1% (2/94) were unmapped. Except for LG 7b and 8a, the other 16 LGs possessed SDL ranging from 3.6% (1/28) in LG 3a to 38.5% (20/52) in LG 9b (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>). Clustering of SDL with greater than 4 loci was observed only in the middle part of two LGs, <em>i.e.</em> 5b and 9b. Remarkably, 10 consecutive SDL were found in LG 9b (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4</a>). The remaining SDL were randomly scattered over the LGs.</p><p id="p-42">The ratio of coupling to repulsion linkage phase is expected as 1:1 for allopolyploids, and 1:0.25 in autotetraploids (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Wu <em>et al.</em> 1992</a>). Among the 469 linked loci on this framework map, a ratio of 241:228 coupling to repulsion phase linkage were detected, which conformed to a 1:1 ratio (X<sup>2</sup> = 0.36 &lt; X<sup>2</sup><sub>(1, 0.05)</sub> = 5.99), completely confirming the disomic inheritance reported by <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Liu and Wu (2011)</a> and being highly congruent with the results of <a id="xref-ref-49-20" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a>.</p></div><div id="sec-13" class="subsection"><h3>Mapping SSR sequences to foxtail millet genome</h3><p id="p-43">Of the plant species with the genome sequenced, foxtail millet is currently the closest relative of switchgrass, as both species belong to the <em>Paniceae</em> tribe and share a common ancestor 13 (±3) million years ago (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Doust <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brutnell <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li and Brutnell 2011</a>). The blast analysis indicated that of the 453 tested markers, 389 had hits in foxtail genomes. There existed a one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet chromosomes (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). A total of 319 of 389 markers from the switchgrass LGs matched nine corresponding foxtail millet chromosomes (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/TableS4.pdf">Table S4</a>). Two large homeologous groups, 2 and 9 (including 71 and 62 markers, respectively), were anchored in foxtail millet chromosomes II and IX. In contrast, only 8 markers were on foxtail millet chromosome VII. The mismatched markers appeared to scatter at random on foxtail millet chromosomes.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461405/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461405/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4 </span> <span class="caption-title">Positioning of switchgrass mapped loci in the genome of foxtail millet (<em>Setaria italica</em>)</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">DISCUSSION</h2><div id="sec-15" class="subsection"><h3>The selfed progeny population for mapping the genome in switchgrass</h3><p id="p-47">Due to wind-facilitated cross-pollination and strong genetic self-incompatibility, switchgrass is an allogamous species and homozygous inbred lines are unavailable in nature (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Talbert <em>et al.</em> 1983</a>). For this reason, full-sib populations from two heterozygous parents were developed to construct switchgrass linkage maps (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Missaoui <em>et al.</em> 2005</a>; <a id="xref-ref-49-21" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). The population of <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Missaoui <em>et al.</em> (2005)</a> was composed of 85 full-sib progeny from a cross of ‘Alamo’ genotype AP13 (seed parent) and ‘Summer’ VS 16, whereas the population of <a id="xref-ref-49-22" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a> consisted of 238 full-sib plants derived from crossing one genotype (seed parent) of “Kanlow” with a selection of “Alamo.” Because male and female meioses in the full-sib populations were distinct and independent processes, two separate parental maps were constructed, one map for the male parent and another for the female parent (<a id="xref-ref-49-23" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). Recently we identified a self-compatible lowland switchgrass genotype NL94. A first (S1) generation inbred population from selfing NL94 was developed with the assistance of marker-based identification (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Liu and Wu 2011</a>). This S1 population is similar to an F2 population derived from selfing a F1 hybrid of a cross between two different inbred lines; therefore, only one map was constructed instead of two separate (male and female) maps. In other cross-pollinated species, such as loblolly pine (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Remington <em>et al.</em> 1999</a>), sugarcane (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hoarau <em>et al.</em> 2001</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andru <em>et al.</em> 2011</a>), and grapes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blasi <em>et al.</em> 2011</a>), S1 progeny have also been used for constructing linkage maps using SSR, amplified fragment length polymorphism (AFLP), resistance gene analog (RGA), and target region amplification polymorphism (TRAP) markers.</p><p id="p-48">Population size is the other important issue in constructing a linkage map, because it is highly associated with the accuracy of detecting recombination events. However, the size of the mapping population is often limited by the genotyping and phenotyping costs. The population size used in this study was 139 S1 progeny resulting in 278 gametes that were used in the calculation of the recombination frequencies. Consequently, its mapping accuracy should be much higher than that used in the reference maps, as each map was constructed using 238 gametes (238 full-sib F1 individuals, <a id="xref-ref-49-24" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>).</p></div><div id="sec-16" class="subsection"><h3>SSR marker polymorphisms</h3><p id="p-49">SSRs used as a DNA marker system are advantageous over many other marker systems and have been widely utilized in many plant genomic studies (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Morgante and Olivieri 1993</a>). In this study, we exhaustively screened all available switchgrass SSR markers from various sources (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Tobias <em>et al.</em> 2006</a>, <a id="xref-ref-70-6" class="xref-bibr" href="#ref-70">2008</a>; <a id="xref-ref-49-25" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-76-8" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>) and found the polymorphic ratio of gSSRs is nearly 2-fold higher than that of eSSRs (<em>i.e.</em> 28.2% <em>vs.</em> 14.0%). This result is consistent with previous studies revealing higher polymorphism levels in gSSR than in eSSR markers (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">La Rota <em>et al.</em> 2005</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Saha <em>et al.</em> 2005</a>). This is because gSSRs mostly reside in nongenic regions (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Varshney <em>et al.</em> 2005</a>), so more variations can be tolerated than eSSRs. The majority of gSSRs used here were inherited in a codominant manner, and in most cases, they are chromosome-specific because only a single locus is amplified from one of the two homologous chromosomes. This unique marker-chromosome relationship is a very useful feature in a polyploidy genome. Similar phenomena were observed in wheat gSSRs (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Röder <em>et al.</em> 1998a</a>). Although previous studies indicated eSSRs are prone to accumulate in gene-rich regions and affect the coverage of linkage maps (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pinto <em>et al.</em> 2006</a>), in this study, mapped eSSRs were interspersed along the whole linkage map, mostly between the gSSRs, and no obvious clusters were observed. It is further noteworthy that 80.6% of SSRs (2010 of 2493 tested markers) amplified monomorphic bands in the parent NL94 of this mapping population, indicating the parent has a high level of homozygosity. Higher polymorphisms of the same gSSR and eSSRs markers were reported between switchgrass varieties (<a id="xref-ref-76-9" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-70-7" class="xref-bibr" href="#ref-70">Tobias <em>et al.</em> 2008</a>). We observed a higher frequency of polymorphism in dinucleotide microsatellites (GA- and CA-motifs) than in trinucleotide microsatellites (CCG-, CAG-, and AAG-). This is in agreement with the previous results obtained in switchgrass (<a id="xref-ref-76-10" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>) and other grasses, such as tall fescue (<em>Festuca arundinacea</em>) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hirata <em>et al.</em> 2006</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Saha <em>et al.</em> 2006</a>), timothy (<em>Phleum pretense</em>) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cai <em>et al.</em> 2003</a>), perennial ryegrass (<em>Lolium perenne</em>) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jones <em>et al.</em> 2001</a>), and zoysiagrass (<em>Zoysia</em> spp.) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cai <em>et al.</em> 2005</a>). Trinucleotide motifs could stably reside in the target regions and suppress frameshift mutations and variations (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Varshney <em>et al.</em> 2005</a>), and dimeric repeats have been found in the untranslated regions of many species (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Morgante <em>et al.</em> 2002</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jung <em>et al.</em> 2005</a>). On the basis of the higher occurrence and polymorphism, dinucleotide SSRs could be preferentially selected for linkage map development in the future.</p><p id="p-50">We observed that sorghum and foxtail millet gSSR markers had a marker transferability of 11% and 54%, respectively, which mirrored the respective divergence distance of the two species from switchgrass. Because the whole-genome sequence (although not 100%) of foxtail millet is publicly available now and has a higher transferability to switchgrass than sorghum, it provided a valuable resource to develop new molecular markers for constructing LGs of switchgrass, a plant with limited sequence information.</p></div><div id="sec-17" class="subsection"><h3>Genetic linkage map</h3><p id="p-51">Estimates of the genome length and map distance between markers are important for the characterization of gene effects, integration of genetic and physical maps, and evaluation of map coverage. Several factors can affect the accuracy of LG construction. Among them, genotyping errors are a common factor that inflates the map length, especially when multiple bands (alleles) of the markers are scored in a polyploidy species (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pompanon <em>et al.</em> 2005</a>). Errors can be divided into two distinct types: type I is allelic dropout, in which one allele of a heterozygote randomly fails in PCR amplification, and type II is false alleles, in which the true allele is mistakenly genotyped due to low quality of DNA, resulting in PCR or electrophoresis artifacts or scoring errors in reading and recording data (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broquet and Petit 2004</a>). To ensure the quality of the linkage map, a number of measures were taken as follows. First, high-quality DNA samples were extracted using a stringent CTAB method (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Doyle and Doyle 1990</a>), and each sample was measured for A260/280 value to be ∼1.8 and analyzed in agarose gel electrophoresis to show only one bright band in a high molecular weight area. Second, the scoring system of codominant markers was utilized rather than the single dose dominant markers previously used by <a id="xref-ref-49-26" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a> and <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Missaoui <em>et al.</em> (2005)</a>. We found most polymorphic markers showed mainly two reproducible alleles in parent NL94. This result is similar with previous studies where the mean number of amplicons per individual was 2.18 (<a id="xref-ref-70-8" class="xref-bibr" href="#ref-70">Tobias <em>et al.</em> 2008</a>). The segregation of all markers in progeny was scored as only one pattern (&lt;hkxhk&gt;), instead of five types (&lt;abxcd&gt;, &lt;efxeg&gt;, &lt;lmxll&gt;, &lt;nnxnp&gt;, and &lt;hkxhk&gt;) ranging from two to four alleles in a full-sib population. The fewer alleles recorded, the fewer chances that an allele dropout could occur, thus increasing the accuracy of genotyping. The codominant markers used here provided more information for linkage analysis than do dominant ones (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wu <em>et al.</em> 2002</a>). Third, all marker data were scored independently by two people to reduce errors. Those markers with spurious, unstable, or redundant bands were either repeatedly tested or excluded (accounting for about 10% of total data) from the final map calculations. Four, we initially started linkage calculations using the markers with high fidelity to construct a framework map with LOD to 8.0. Those markers were fixed to allow more markers to be added on the linkage map by reducing the LOD threshold to 3.0 (except for 2.0 to link two segments of LG 6b). This strategy has been successfully used by previous studies (<a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Wu and Huang 2006</a>; <a id="xref-ref-49-27" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>) and guaranteed both the accuracy and high coverage of the final full map.</p><p id="p-52">A high collinearity between our map and the previously published switchgrass maps of crossed populations (<a id="xref-ref-49-28" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>) was observed, excepting the following small discrepancies: seven inversions in marker order between our map and the reference maps (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1/FigureS3.pdf">Figure S3</a>). Except for 1 mismatched foxtail millet marker, the other 7 markers from foxtail millet (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> 2009</a>) and 1 marker from sorghum (<a id="xref-ref-81-4" class="xref-bibr" href="#ref-81">Wu and Huang 2006</a>) matched their corresponding foxtail millet and sorghum linkage maps, respectively (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4</a>). The local rearrangement in some LGs could be real due to their universality in plant genomes (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Paterson <em>et al.</em> 1996</a>) or to genotyping errors in either of the populations. After all, errors can never be completely avoided in linkage mapping (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Johnson and Haydon 2007</a>). When a genetic map is constructed, it is assumed that marker crossovers during meiosis occur at random intervals along chromosomes (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lichten and Goldman 1995</a>). In this study, although we used all available markers, only 4 markers were mapped on LG 7b. In contrast, LG 7b is one of the linkage groups with the highest marker density in a previous study (<a id="xref-ref-49-29" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). Of the 63 markers mapped on LG 7b by <a id="xref-ref-49-30" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a>, 57 markers were monomorphic and 6 markers produced unscorable bands in our population, indicating the very high homozygosity level of the NL 94 chromosome corresponding to LG 7b.</p></div><div id="sec-18" class="subsection"><h3>Genome structure of switchgrass</h3><p id="p-53">The genome structure of switchgrass was described in previous studies, but no consistent conclusions were drawn (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Missaoui <em>et al.</em> 2005</a>; <a id="xref-ref-49-31" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Saski <em>et al.</em> 2011</a>), <em>i.e.</em> allotetraploid <em>vs.</em> autotetraploid. Allotetraploids have two diverged subgenomes and show the same inheritance as diploids. In this study, we speculate NL 94 switchgrass is an allotetraploid genotype because of the evidence described as follows. First, all 503 codominant markers had disomic inheritance, although 18.7% showed segregation distortion. Second, the ratio of coupling to repulsion linkage was 1:1, which is consistent with the disomic inheritance mode in polyploidy map construction (<a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Wu <em>et al.</em> 1992</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Qu and Hancock 2001</a>). Third, among 476 mapped SSR markers, only 20 (4.2%) amplified more than two bands and mapped on homeologous LGs, indicating that the two subgenomes are significantly different from each other. The allotetraploid-like genome structure of the tetraploid switchgrass uncovered by this study and the two recent molecular marker investigations (<a id="xref-ref-49-32" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Liu and Wu 2011</a>) agree well with the published cytological observations of the bivalent pairing behavior of meiotic chromosomes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barnett and Carver 1967</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Martinez-Reyna <em>et al.</em> 2001</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Young <em>et al.</em> 2010</a>). However, because diploid wild ancestors of switchgrass are unknown and there are many examples of autotetraploids that through time have undergone “diploidization” leading to disomic inheritance (see review by <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wolfe 2001</a>), much more sequence data are needed to address this issue of allopolyploidy confidently.</p><p id="p-54">In this study, we successfully identified six homeologous LGs using the 20 multi-allele markers. Attempts were made to find more multi-allele markers, but most of those multi-allele primer pairs were error-prone during band scoring or unstable for the entire population. To avoid eroding the quality of the linkage map, we were cautious when selecting these markers. Approximately 20% the eSSR markers amplified alleles in two subgenomes of switchgrass in previous maps and identified nine complete homeologous LGs (<a id="xref-ref-49-33" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). Although higher numbers of multi-allele SSRs were reported in their study, <a id="xref-ref-49-34" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> (2010)</a> strongly believed that the subgenomes of switchgrass were distantly related. Through the bridge markers and comparative mapping between ours and published maps (<a id="xref-ref-49-35" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>), nine complete homeologous LGs were identified in this study.</p></div><div id="sec-19" class="subsection"><h3>Segregation distortion</h3><p id="p-55">Numerous examples of segregation distortion have been reported in many crop species, including barley (<em>Hordeum vulgare</em>) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Graner <em>et al.</em> 1991</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Devaux <em>et al.</em> 1995</a>), rice (<em>Oryza sativa</em>)(<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Causse <em>et al.</em> 1994</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Xu <em>et al.</em> 1997</a>), maize (<em>Zea mays</em>) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wendel <em>et al.</em> 1987</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lu <em>et al.</em> 2002</a>), and wheat (<em>Triticum aestivum</em>) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blanco <em>et al.</em> 2004</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Quarrie <em>et al.</em> 2005</a>). In this study, segregation distortion was observed for 18.7% of the total marker loci analyzed. This is slightly higher than that of the full-sib switchgrass population with a value of up to 14% (<a id="xref-ref-49-36" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). Because a selfed population was used here, the result is consistent with previous reports that showed selfed populations had a higher tendency for segregation distortions due to inbreeding depression (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Andru <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Charlesworth and Willis 2009</a>).</p><p id="p-56">Two SDL clusters were identified on LG 5b and 9b in this study (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4</a>). Previous studies indicated that segregation distortion was an indication of the linkage between molecular markers and distorting factors (such as recessive lethal genes and incompatible alleles) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lyttle 1991</a>). If the linkage was tight, they usually had similar segregation patterns and the skewed markers would appear to be clustered (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jenczewski <em>et al.</em> 1997</a>). These two SDL clusters were not found in other studies, suggesting they are population-specific. Similarly, in sugarcane and grape S1 maps (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hoarau <em>et al.</em> 2001</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blasi <em>et al.</em> 2011</a>), some distorted markers were also clustered together.</p></div><div id="sec-20" class="subsection"><h3>Comparative mapping</h3><p id="p-57">Comparative genome relationships have been established among rice, foxtail millet, sugar cane, sorghum, pearl millet, maize, and the <em>Triticeae</em> cereals (wheat, barley, rye, and oats) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gale and Devos 1998</a>). As an emerging biofuel crop with a relatively short breeding history, switchgrass had the only reported relationship with the sorghum genome, in which nine homeologous groups of switchgrass corresponded to 10 sorghum chromosomes (<a id="xref-ref-49-37" class="xref-bibr" href="#ref-49">Okada <em>et al.</em> 2010</a>). Here we utilized the newly released foxtail millet sequence and revealed that about 80% of switchgrass markers were located in the foxtail millet chromosomes. This result directly indicated the strong one-to-one relationship between the chromosomes of switchgrass and foxtail millet. The corresponding genomic relationship among switchgrass, foxtail millet, and sorghum is consistent with a previously integrated grass comparative map (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Devos and Gale 1997</a>). The high genetic similarity between switchgrass and foxtail millet is not surprising when considering their close relationship, with both belonging to the same <em>Paniceae</em> tribe and the high collinearity among grasses reported previously (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Devos and Gale 1997</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2000</a>). However, mismatched markers still existed. These might have been derived from chromosomal rearrangements occurring after the respective speciation of both switchgrass and foxtail millet. Due to its small stature, rapid life cycle, inbred pollination, and prolific seed production, foxtail millet is a desirable candidate to be used as a reference model for switchgrass genetic and functional genomics investigation.</p></div><div id="sec-21" class="subsection"><h3>Genomics and breeding implications</h3><p id="p-58">Switchgrass has large, highly heterozygous polyploidy genomes that hinder the effort of whole-genome sequencing. Dihaploid lines were identified in switchgrass (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Young <em>et al.</em> 2010</a>) and seem to be helpful for simplifying sequence assembly but are unsuitable for entire genome sequencing due to their high sterility and instability (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Young <em>et al.</em> 2010</a>). A diploid relative of switchgrass (<em>i.e.</em> <em>Panicum hallii</em>) (2n = 2x = 18) is being sequenced at the Joint Genome Institute (JGI, <a href="http://www.jgi.doe.gov/genome-projects/">http://www.jgi.doe.gov/genome-projects/</a>); however, it has only one set of chromosomes, and therefore, it may not be effective for defining two subgenomes of switchgrass. Here we constructed a high-density genetic map derived from selfing a single genotype NL94. A total of 476 SSR markers were mapped in more than 2000 cM genome length with average marker distance of 4.2 cM. The present map provides an excellent tool for distinguishing switchgrass subgenomes. Through continuing to self switchgrass plants like NL94, inbred lines are being produced in our lab. These could be used as the starting material for whole-genome sequencing.</p><p id="p-59">Based on our preliminary observations in the field, the selfed progeny of NL94 showed significant segregation for biomass and its associated traits, such as plant height, circumference of plant base, tiller number, and spring growth vigor. It would be much easier to perform a QTL analysis using a single map than two maps from a full-sib population. Our S1 mapping population should have some features of F2 populations, including the estimation of both additive and dominant effects (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carbonell <em>et al.</em> 1993</a>).</p><p id="p-60">In conclusion, this study developed a complete genetic map of 18 linkage groups in an inbred lowland switchgrass population using mostly codominant SSR markers. This map is the longest and most dense one constructed to date. Two clusters of segregation-distorted loci were found in the switchgrass genome. A one-to-one relationship between nine switchgrass homeologous groups and nine foxtail millet chromosomes was established. This linkage map should provide a new genetic framework to facilitate genomics research, quantitative trait locus (QTL) mapping, and marker-assisted selection.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-61">We would like to thank the following funding sources and individuals for sponsoring and helping in this research: National Science Foundation Award EPS 0814361; Oklahoma Agricultural Experiment Station; Andrew Doust (Botany Department, Oklahoma State University) for sharing foxtail millet primer and sequence information; Christian M. Tobias (Western Regional Research Center, USDA–ARS) for his helpful discussion in the construction of the linkage map; Mrs. Janelle Malone for editorial input; and lab members, including Yiwen Xiang, Candace Pitts, Elke Grether, Gary Williams, Sharon Williams, Pu Feng, Chengcheng Tan, James Todd, Shiva Makaju, and Shuiyi Lu. The genome nucleotide sequence of foxtail millet for blast analysis was generated by J.G.I. and has not yet been published.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001503/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-62"><em>Communicating editor: K. Dawe</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-10-30"><span class="received-label">Received </span>October 30, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-01-16"><span class="accepted-label">Accepted </span>January 16, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Liu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">LITERATURE CITED</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.1" data-doi="10.1007/s00122-011-1568-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andru</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thongthawee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimbeng</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of the sugarcane (<em>Saccharum</em> spp.) cultivar ’LCP 85–384’. I. Linkage mapping using AFLP, SSR, and TRAP markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAndru%26rft.auinit1%253DS.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D93%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2Bsugarcane%2B%2528Saccharum%2Bspp.%2529%2Bcultivar%2B%2527LCP%2B85-384%2527.%2BI.%2BLinkage%2Bmapping%2Busing%2BAFLP%252C%2BSSR%252C%2Band%2BTRAP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1568-x%26rft_id%253Dinfo%253Apmid%252F21472411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1568-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21472411&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.2" data-doi="10.2135/cropsci1967.0011183X000700040005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnett</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Meiosis and pollen stainability in switchgrass, <em>Panicum virgatum</em> L</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBarnett%26rft.auinit1%253DF.%2BL.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D301%26rft.epage%253D304%26rft.atitle%253DMeiosis%2Band%2BPollen%2BStainability%2Bin%2BSwitchgrass%252C%2BPanicum%2Bvirgatum%2BL%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1967.0011183X000700040005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1967.0011183X000700040005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.3" data-doi="10.1007/BF00303919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beckmann</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Restriction fragment length polymorphisms in genetic improvement: methodologies, mapping and costs</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D67%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00303919%26rft_id%253Dinfo%253Apmid%252F24258478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00303919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24258478&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RV54200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.4" data-doi="10.1371/journal.pgen.1000366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beldade</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saenko</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pul</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A gene-based linkage map for <em>Bicyclus anynana</em> butterflies allows for a comprehensive analysis of synteny with the lepidopteran reference genome</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBeldade%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253De1000366%26rft.epage%253De1000366%26rft.atitle%253DA%2Bgene-based%2Blinkage%2Bmap%2Bfor%2BBicyclus%2Banynana%2Bbutterflies%2Ballows%2Bfor%2Ba%2Bcomprehensive%2Banalysis%2Bof%2Bsynteny%2Bwith%2Bthe%2Blepidopteran%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000366%26rft_id%253Dinfo%253Apmid%252F19197358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19197358&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simeone</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cenci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadaleta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanzarella</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2004</span> <span class="cit-article-title">Extension of the “Messapia x dicoccoides” linkage map of <em>Triticum turgidum</em> (L.)</span>. <abbr class="cit-jnl-abbrev">Thell. Cell Mol. Biol. Lett.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThell.%2BCell%2BMol.%2BBiol.%2BLett.%26rft.volume%253D9%26rft.spage%253D529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.6" data-doi="10.1007/s00122-011-1565-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blasi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiedemann-Merdinoglu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prado</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rühl</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Construction of a reference linkage map of <em>Vitis amurensis</em> and genetic mapping of <em>Rpv8</em>, a locus conferring resistance to grapevine downy mildew</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBlasi%26rft.auinit1%253DP.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DConstruction%2Bof%2Ba%2Breference%2Blinkage%2Bmap%2Bof%2BVitis%2Bamurensis%2Band%2Bgenetic%2Bmapping%2Bof%2BRpv8%252C%2Ba%2Blocus%2Bconferring%2Bresistance%2Bto%2Bgrapevine%2Bdowny%2Bmildew.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1565-0%26rft_id%253Dinfo%253Apmid%252F21404060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1565-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21404060&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.7" data-doi="10.1111/j.1365-294X.2004.02352.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broquet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantifying genotyping errors in noninvasive population genetics</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3601</span>–<span class="cit-lpage">3608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBroquet%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D3601%26rft.epage%253D3608%26rft.atitle%253DQuantifying%2Bgenotyping%2Berrors%2Bin%2Bnoninvasive%2Bpopulation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02352.x%26rft_id%253Dinfo%253Apmid%252F15488016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02352.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15488016&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224524000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.8" data-doi="10.1105/tpc.110.075309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brutnell</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swartwood</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldschmidt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Setaria viridis</em>: a model for C4 photosynthesis</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2537</span>–<span class="cit-lpage">2544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.075309%26rft_id%253Dinfo%253Apmid%252F20693355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMjUzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMy8zNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.9" data-doi="10.1007/s00122-003-1386-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuyama</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshizawa</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Isolation and characterization of simple sequence repeat markers in the hexaploid forage grass timothy (<em>Phleum pretense</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1337</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCai%26rft.auinit1%253DH.%2BW.%26rft.volume%253D107%26rft.issue%253D8%26rft.spage%253D1337%26rft.epage%253D1349%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bsimple%2Bsequence%2Brepeat%2Bmarkers%2Bin%2Bthe%2Bhexaploid%2Bforage%2Bgrass%2Btimothy%2B%2528Phleum%2Bpratense%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1386-x%26rft_id%253Dinfo%253Apmid%252F12920514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1386-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12920514&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.10" data-doi="10.1007/s00122-005-0118-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuyama</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">Isolation, characterization and mapping of simple sequence repeat markers in zoysiagrass (<em>Zoysia</em> spp.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCai%26rft.auinit1%253DH.%2BW.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D158%26rft.epage%253D166%26rft.atitle%253DIsolation%252C%2Bcharacterization%2Band%2Bmapping%2Bof%2Bsimple%2Bsequence%2Brepeat%2Bmarkers%2Bin%2Bzoysiagrass%2B%2528Zoysia%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0118-9%26rft_id%253Dinfo%253Apmid%252F16235048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16235048&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233726000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.11" data-doi="10.1007/BF00838555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbonell</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baselga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balansard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerig</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Power studies in the estimation of genetic parameters of quantitative trait loci for backcross and doubled haploid populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D86%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00838555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00838555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chunwongse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1994</span> <span class="cit-article-title">Saturated molecular map of the rice genome based on an interspecific backcross population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D138%26rft.spage%253D1251%26rft_id%253Dinfo%253Apmid%252F7896104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7896104&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU54800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.13" data-doi="10.1038/nrg2664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of inbreeding depression</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bgenetics%2Bof%2Binbreeding%2Bdepression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2664%26rft_id%253Dinfo%253Apmid%252F19834483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834483&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270915400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.14" data-doi="10.3835/plantgenome2010.04.0010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-size variation in switchgrass (<em>Panicum virgatum</em>): flow cytometry and cytology reveal rampant aneuploidy</span>. <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D130%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.15" data-doi="10.1007/BF00418029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devaux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinhofs</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Comparative mapping of the barley genome with male and female recombination-derived, doubled haploid populations</span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">249</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DDevaux%26rft.auinit1%253DP.%26rft.volume%253D249%26rft.issue%253D6%26rft.spage%253D600%26rft.epage%253D608%26rft.atitle%253DComparative%2Bmapping%2Bof%2Bthe%2Bbarley%2Bgenome%2Bwith%2Bmale%2Band%2Bfemale%2Brecombination-derived%252C%2Bdoubled%2Bhaploid%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00418029%26rft_id%253Dinfo%253Apmid%252F8544825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00418029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8544825&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.16" data-doi="10.1023/A:1005820229043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Comparative genetics in the grasses</span>. <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DDevos%26rft.auinit1%253DK.%2BM.%26rft.volume%253D35%26rft.issue%253D1-2%26rft.spage%253D3%26rft.epage%253D15%26rft.atitle%253DComparative%2Bgenetics%2Bin%2Bthe%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1005820229043%26rft_id%253Dinfo%253Apmid%252F9291955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1005820229043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9291955&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU45800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.17" data-doi="10.1105/tpc.12.5.637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome relationships: the grass model in current research</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DDevos%26rft.auinit1%253DK.%2BM.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D637%26rft.epage%253D646%26rft.atitle%253DGenome%2BRelationships%253A%2BThe%2BGrass%2BModel%2Bin%2BCurrent%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.12.5.637%26rft_id%253Dinfo%253Apmid%252F10810140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjEyLzUvNjM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8zLzM1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.18" data-doi="10.1104/pp.108.129627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doust</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellogg</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Foxtail millet, a sequence driven grass model system</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DDoust%26rft.auinit1%253DA.%2BN.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D137%26rft.epage%253D141%26rft.atitle%253DFoxtail%2BMillet%253A%2BA%2BSequence-Driven%2BGrass%2BModel%2BSystem%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.129627%26rft_id%253Dinfo%253Apmid%252F19126705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDkvMS8xMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzMvMzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Isolation of plant DNA from fresh tissue</span>. <abbr class="cit-jnl-abbrev">Focus</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFocus%26rft.volume%253D12%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.20" data-doi="10.1126/science.282.5389.656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Plant comparative genetics after 10 years</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGale%26rft.auinit1%253DM.%2BD.%26rft.volume%253D282%26rft.issue%253D5389%26rft.spage%253D656%26rft.epage%253D659%26rft.atitle%253DPlant%2BComparative%2BGenetics%2Bafter%2B10%2BYears%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5389.656%26rft_id%253Dinfo%253Apmid%252F9784118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTM4OS82NTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzMvMzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahoor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schondelmaier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siedler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1991</span> <span class="cit-article-title">Construction of an RFLP map of barley</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D250%26rft_id%253Dinfo%253Apmid%252F24202366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24202366&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GU45600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunter</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Diversity among populations of switchgrass based on RAPD markers</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGunter%26rft.auinit1%253DL.%2BE.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D1017%26rft.epage%253D1022%26rft.atitle%253DDiversity%2Bamong%2BPopulations%2Bof%2BSwitchgrass%2BBased%2Bon%2BRAPD%2BMarkers%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1996UV54900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-other" id="cit-2.3.357.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hitchcock</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chase</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-comment">1951 <em>Manual of the Grasses of the United States</em>. United States Department of Agriculture. Miscellaneous Publication No. 200. U.S. Government Printing Office, Washington</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.24" data-doi="10.1007/s00122-006-0292-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuyama</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2006</span> <span class="cit-article-title">Development of simple sequence repeat (SSR) markers and construction of an SSR-based linkage map in Italian ryegrass (<em>Lolium multiflorum</em> Lam.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">270</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHirata%26rft.auinit1%253DM.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D270%26rft.epage%253D279%26rft.atitle%253DDevelopment%2Bof%2Bsimple%2Bsequence%2Brepeat%2B%2528SSR%2529%2Bmarkers%2Band%2Bconstruction%2Bof%2Ban%2BSSR-based%2Blinkage%2Bmap%2Bin%2BItalian%2Bryegrass%2B%2528Lolium%2Bmultiflorum%2BLam.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0292-4%26rft_id%253Dinfo%253Apmid%252F16791693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0292-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16791693&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.25" data-doi="10.1007/s001220000390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoarau</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Offmann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Hont</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risterucci</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roques</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic dissection of a modern sugarcane cultivar (<em>Saccharum</em> spp.). I. Genome mapping with AFLP markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220000390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220000390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170129000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.26" data-doi="10.2135/cropsci1996.0011183X003600050021x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chromosome number and nuclear DNA content of several switchgrass populations</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1192</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHopkins%26rft.auinit1%253DA.%2BA.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D1192%26rft.epage%253D1195%26rft.atitle%253DChromosome%2BNumber%2Band%2BNuclear%2BDNA%2BContent%2Bof%2BSeveral%2BSwitchgrass%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600050021x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600050021x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VK30900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.27" data-doi="10.1007/s001220000489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Constructing dense genetic linkage maps</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D1113%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220000489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220000489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169278600044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.28" data-doi="10.1007/s001220050466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenczewski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gherandi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prosperi</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveri</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1997</span> <span class="cit-article-title">Insight on segregation distortions in two intraspecific crosses between annual species of <em>Medicago</em> (<em>Leguminosae</em>)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">682</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D94%26rft.spage%253D682%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WX38800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.29" data-doi="10.1007/s00122-008-0942-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and genetic mapping of SSR markers in foxtail millet</span>. <span class="cit-comment">[<em>Setaria italica</em> (L.) P. Beauv.]</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DJia%26rft.auinit1%253DX.%26rft.volume%253D118%26rft.issue%253D4%26rft.spage%253D821%26rft.epage%253D829%26rft.atitle%253DDevelopment%2Band%2Bgenetic%2Bmapping%2Bof%2BSSR%2Bmarkers%2Bin%2Bfoxtail%2Bmillet%2B%255BSetaria%2Bitalica%2B%2528L.%2529%2BP.%2BBeauv.%255D.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0942-9%26rft_id%253Dinfo%253Apmid%252F19139840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0942-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19139840&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.30" data-doi="10.1534/genetics.106.064618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">P. C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haydon</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Maximum-likelihood estimation of allelic dropout and false allele error rates from microsatellite genotypes in the absence of reference data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064618%26rft_id%253Dinfo%253Apmid%252F17179070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvODI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8zLzM1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.31" data-doi="10.1007/s001220051661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupal</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolliker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drayton</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Development and characterization of simple sequence repeat (SSR) markers for perennial ryegrass (<em>Lolium perenne</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000167232800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.32" data-doi="10.1007/s10142-005-0139-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jesudurai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Main</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Frequency, type, distribution and annotation of simple sequence repeats in <em>Rosaceae</em> ESTs</span>. <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DJung%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D136%26rft.epage%253D143%26rft.atitle%253DFrequency%252C%2Btype%252C%2Bdistribution%2Band%2Bannotation%2Bof%2Bsimple%2Bsequence%2Brepeats%2Bin%2BRosaceae%2BESTs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-005-0139-0%26rft_id%253Dinfo%253Apmid%252F15761705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-005-0139-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15761705&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.33" data-doi="10.1016/S0168-9525(97)01008-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Simple sequence repeats as a source of quantitative genetic variation</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKashi%26rft.auinit1%253DY.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D74%26rft.epage%253D78%26rft.atitle%253DSimple%2Bsequence%2Brepeats%2Bas%2Ba%2Bsource%2Bof%2Bquantitative%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252897%252901008-1%26rft_id%253Dinfo%253Apmid%252F9055609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(97)01008-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9055609&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WG90300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.34" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span> <span class="cit-article-title">The estimation of map distances from recombination values</span>. <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.35" data-doi="10.1186/1471-2164-6-23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">La Rota</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kantety</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nonrandom distribution and frequencies of genomic and EST-derived microsatellite markers in rice, wheat, and barley</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLa%2BRota%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.spage%253D23%26rft.epage%253D23%26rft.atitle%253DNonrandom%2Bdistribution%2Band%2Bfrequencies%2Bof%2Bgenomic%2Band%2BEST-derived%2Bmicrosatellite%2Bmarkers%2Bin%2Brice%252C%2Bwheat%252C%2Band%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-6-23%26rft_id%253Dinfo%253Apmid%252F15720707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-6-23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15720707&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.36" data-doi="10.1093/jxb/err096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brutnell</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Setaria viridis</em> and <em>Setaria italica</em>, model genetic systems for the <em>Panicoid</em> grasses</span>. <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">3031</span>–<span class="cit-lpage">3037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferr096%26rft_id%253Dinfo%253Apmid%252F21459768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjYyLzkvMzAzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMy8zNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.37" data-doi="10.1146/annurev.ge.29.120195.002231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Meiotic recombination hotspots</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DLichten%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D423%26rft.epage%253D444%26rft.atitle%253DMeiotic%2BRecombination%2BHotspots%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.29.120195.002231%26rft_id%253Dinfo%253Apmid%252F8825482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.29.120195.002231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8825482&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TL71900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of a selfing compatible genotype and mode of inheritance in switchgrass</span>. <abbr class="cit-jnl-abbrev">Bioenergy Res.</abbr> <span class="cit-comment">(in press) DOI: 10.1007/S12155-011-9173-z</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.39" data-doi="10.1007/s00122-002-0970-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero-Severson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Chromosomal regions associated with segregation distortion in maize</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLu%26rft.auinit1%253DH.%26rft.volume%253D105%26rft.issue%253D4%26rft.spage%253D622%26rft.epage%253D628%26rft.atitle%253DChromosomal%2Bregions%2Bassociated%2Bwith%2Bsegregation%2Bdistortion%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-0970-9%26rft_id%253Dinfo%253Apmid%252F12582513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-0970-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582513&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178256200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.40" data-doi="10.1146/annurev.ge.25.120191.002455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyttle</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Segregation distorters</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DLyttle%26rft.auinit1%253DT.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D511%26rft.epage%253D557%26rft.atitle%253DSegregation%2BDistorters%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.25.120191.002455%26rft_id%253Dinfo%253Apmid%252F1812815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.25.120191.002455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1812815&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GX17400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Reyna</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caha</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Meiotic stability, chloroplast DNA polymorphisms, and morphological traits of upland × lowland switchgrass reciprocal hybrids</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1579</span>–<span class="cit-lpage">1583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMartinez-Reyna%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D41%26rft.issue%253D5%26rft.spage%253D1579%26rft.epage%253D1583%26rft.atitle%253DMeiotic%2BStability%252C%2BChloroplast%2BDNA%2BPolymorphisms%252C%2Band%2BMorphological%2BTraits%2Bof%2BUpland%2B%2BLowland%2BSwitchgrass%2BReciprocal%2BHybrids%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000176236800032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.42" data-doi="10.2135/cropsci2002.1800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Reyna</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Incompatibility systems in switchgrass</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1800</span>–<span class="cit-lpage">1805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMartinez-Reyna%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D1800%26rft.epage%253D1805%26rft.atitle%253DIncompatibility%2BSystems%2Bin%2BSwitchgrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.1800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.1800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000181430200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.43" data-doi="10.1016/j.biombioe.2004.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kszos</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Development of switchgrass (<em>Panicum virgatum</em>) as a bioenergy feedstock in the United States</span>. <abbr class="cit-jnl-abbrev">Biomass Bioenergy</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomass%2BBioenergy%26rft.volume%253D28%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biombioe.2004.05.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biombioe.2004.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229348000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.44" data-doi="10.1007/s00122-005-1935-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Missaoui</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Investigation of genomic organization in switchgrass (<em>Panicum virgatum</em> L.) using DNA markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1372</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMissaoui%26rft.auinit1%253DA.%2BM.%26rft.volume%253D110%26rft.issue%253D8%26rft.spage%253D1372%26rft.epage%253D1383%26rft.atitle%253DInvestigation%2Bof%2Bgenomic%2Borganization%2Bin%2Bswitchgrass%2B%2528Panicum%2Bvirgatum%2BL.%2529%2Busing%2BDNA%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-1935-6%26rft_id%253Dinfo%253Apmid%252F15841364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-1935-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15841364&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229718000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.45" data-doi="10.1007/s10722-005-3878-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Missaoui</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular markers for the classification of switchgrass (<em>Panicum virgatum</em> L.) germplasm and to assess genetic diversity in three synthetic switchgrass populations</span>. <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D53%26rft.spage%253D1291%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-005-3878-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-005-3878-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.46" data-doi="10.1046/j.1365-313X.1993.t01-9-00999.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivieri</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">PCR-amplified microsatellites as markers in plant genetics</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMorgante%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DPCR-amplified%2Bmicrosatellites%2Bas%2Bmarkers%2Bin%2Bplant%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1993.t01-9-00999.x%26rft_id%253Dinfo%253Apmid%252F8401603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1993.t01-9-00999.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8401603&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KJ19500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.47" data-doi="10.1038/ng822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanafey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Microsatellites are preferentially associated with non-repetitive DNA in plant genomes</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMorgante%26rft.auinit1%253DM.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D194%26rft.epage%253D200%26rft.atitle%253DMicrosatellites%2Bare%2Bpreferentially%2Bassociated%2Bwith%2Bnonrepetitive%2BDNA%2Bin%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng822%26rft_id%253Dinfo%253Apmid%252F11799393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11799393&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173708700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.48" data-doi="10.1007/s12155-008-9011-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narasimhamoorthy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swaller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic diversity in switchgrass collections assessed by EST-SSR markers</span>. <abbr class="cit-jnl-abbrev">Bioenergy Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioenergy%2BRes.%26rft.volume%253D1%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-008-9011-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-008-9011-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000207576200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.49" data-doi="10.1534/genetics.110.113910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzatella</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2010</span> <span class="cit-article-title">Complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D745%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.113910%26rft_id%253Dinfo%253Apmid%252F20407132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.113910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20407132&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281906800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.50" data-doi="10.1038/ng1296-380"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reischmann</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1996</span> <span class="cit-article-title">Toward a unified genetic map of higher plants, transcending the monocot-dicot divergence</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D380%26rft.epage%253D382%26rft.atitle%253DToward%2Ba%2Bunified%2Bgenetic%2Bmap%2Bof%2Bhigher%2Bplants%252C%2Btranscending%2Bthe%2Bmonocot-dicot%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1296-380%26rft_id%253Dinfo%253Apmid%252F8944014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1296-380&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8944014&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VV73000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.51" data-doi="10.1111/j.1439-0523.2006.01227.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marconi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulian</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2006</span> <span class="cit-article-title">Characterization of novel sugarcane expressed sequence tag microsatellites and their comparison with genomic SSRs</span>. <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">378</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D125%26rft.spage%253D378%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2006.01227.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2006.01227.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.52" data-doi="10.1038/nrg1707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pompanon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellemain</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taberlet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genotyping errors: causes, consequences and solutions</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPompanon%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D847%26rft.epage%253D859%26rft.atitle%253DGenotyping%2Berrors%253A%2Bcauses%252C%2Bconsequences%2Band%2Bsolutions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1707%26rft_id%253Dinfo%253Apmid%252F16304600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304600&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.53" data-doi="10.2307/1935646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">An analysis of variation between upland and lowland switchgrass, <em>Panicum virgatum</em> L., in central Oklahoma</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D47%26rft.spage%253D980%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1935646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1935646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19668861300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-web" id="cit-2.3.357.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preacher</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-comment">Calculation for the chi-square test: an interactive calculation tool for chi-square tests of goodness of fit and independence [Computer software]. Available at <a href="http://quantpsy.org">http://quantpsy.org</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.55" data-doi="10.1007/s001220100647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Detecting and mapping repulsion-phase linkage in polyploids with polysomic inheritance</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.56" data-doi="10.1007/s00122-004-1902-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quarrie</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calestani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semikhodskii</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebreton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">A high-density genetic map of hexaploid wheat (<em>Triticum aestivum</em> L.) from the cross Chinese Spring × SQ1 and its use to compare QTLs for grain yield across a range of environments</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DQuarrie%26rft.auinit1%253DS.%2BA.%26rft.volume%253D110%26rft.issue%253D5%26rft.spage%253D865%26rft.epage%253D880%26rft.atitle%253DA%2Bhigh-density%2Bgenetic%2Bmap%2Bof%2Bhexaploid%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529%2Bfrom%2Bthe%2Bcross%2BChinese%2BSpring%2Bx%2BSQ1%2Band%2Bits%2Buse%2Bto%2Bcompare%2BQTLs%2Bfor%2Bgrain%2Byield%2Bacross%2Ba%2Brange%2Bof%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1902-7%26rft_id%253Dinfo%253Apmid%252F15719212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1902-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15719212&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228150300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.57" data-doi="10.1007/s001220051194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whetten</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Construction of an AFLP genetic map with nearly complete genome coverage in Pinus taeda</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1279</span>–<span class="cit-lpage">1292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRemington%26rft.auinit1%253DD.%2BL.%26rft.volume%253D98%26rft.issue%253D8%26rft.spage%253D1279%26rft.epage%253D1292%26rft.atitle%253DConstruction%2Bof%2Ban%2BAFLP%2Bgenetic%2Bmap%2Bwith%2Bnearly%2Bcomplete%2Bgenome%2Bcoverage%2Bin%2BPinus%2Btaeda.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051194%26rft_id%253Dinfo%253Apmid%252F12238515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12238515&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081124700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wandehake</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Planschke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tixier</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1998</span><span class="cit-comment">a</span> <span class="cit-article-title">A microsatellite map of wheat</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">2007</span>–<span class="cit-lpage">2023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRoder%26rft.auinit1%253DM.%2BS.%26rft.volume%253D149%26rft.issue%253D4%26rft.spage%253D2007%26rft.epage%253D2023%26rft.atitle%253DA%2BMicrosatellite%2BMap%2Bof%2BWheat%26rft_id%253Dinfo%253Apmid%252F9691054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OS80LzIwMDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzMvMzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">b</span> <span class="cit-article-title">The physical mapping of microsatellite markers in wheat</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D41%26rft.spage%253D278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.60" data-doi="10.1007/s00122-004-1843-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mian</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zwonitzer</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chekhovskiy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An SSR- and AFLP-based genetic linkage map of tall fescue (<em>Festuca arundinacea</em> Schreb.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSaha%26rft.auinit1%253DM.%2BC.%26rft.volume%253D110%26rft.issue%253D2%26rft.spage%253D323%26rft.epage%253D336%26rft.atitle%253DAn%2BSSR-%2Band%2BAFLP-based%2Bgenetic%2Blinkage%2Bmap%2Bof%2Btall%2Bfescue%2B%2528Festuca%2Barundinacea%2BSchreb.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1843-1%26rft_id%253Dinfo%253Apmid%252F15558229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1843-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15558229&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226553900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.61" data-doi="10.1007/s00122-006-0391-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouf Mian</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chekhovskiy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Tall fescue genomic SSR markers: development and transferability across multiple grass species</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1449</span>–<span class="cit-lpage">1458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSaha%26rft.auinit1%253DM.%2BC.%26rft.volume%253D113%26rft.issue%253D8%26rft.spage%253D1449%26rft.epage%253D1458%26rft.atitle%253DTall%2Bfescue%2Bgenomic%2BSSR%2Bmarkers%253A%2Bdevelopment%2Band%2Btransferability%2Bacross%2Bmultiple%2Bgrass%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0391-2%26rft_id%253Dinfo%253Apmid%252F16947059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0391-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16947059&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241798000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.62" data-doi="10.1186/1471-2164-12-369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feltus</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">New genomic resources for switchgrass: a BAC library and comparative analysis of because genomic regions harboring bioenergy traits</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSaski%26rft.auinit1%253DC.%2BA.%26rft.volume%253D12%26rft.spage%253D369%26rft.epage%253D369%26rft.atitle%253DNew%2Bgenomic%2Bresources%2Bfor%2Bswitchgrass%253A%2Ba%2BBAC%2Blibrary%2Band%2Bcomparative%2Banalysis%2Bof%2Bhomoeologous%2Bgenomic%2Bregions%2Bharboring%2Bbioenergy%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-369%26rft_id%253Dinfo%253Apmid%252F21767393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21767393&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrin</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Net energy of cellulosic ethanol from switchgrass</span>. <abbr class="cit-jnl-abbrev">Proc. Nat. Acad. Sci. USA</abbr> <span class="cit-vol">1050</span>: <span class="cit-fpage">464</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNat.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D1050%26rft.spage%253D464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjørnstad</span>  <span class="cit-name-given-names">Å.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic dissection of quantitative traits in crops</span>. <abbr class="cit-jnl-abbrev">Electron. J. Biotechnol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BBiotechnol.%26rft.volume%253D13%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.65" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linkage maps by means of a new computer package: JOINMAP</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.66" data-doi="10.2135/cropsci1983.0011183X002300040029x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timothy</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawlings</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Estimates of genetic parameters in switchgrass</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTalbert%26rft.auinit1%253DL.%2BE.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D728%26rft.atitle%253DEstimates%2Bof%2BGenetic%2BParameters%2Bin%2BSwitchgrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1983.0011183X002300040029x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1983.0011183X002300040029x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RE73500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-other" id="cit-2.3.357.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-comment">1999 Reproductive characteristics and breeding improvement potential of switchgrass, pp. 147–153 in <em>Biomass</em>, <em>a Growth Opportunity in Green Energy and Value-added Products—Proceedings of the 4th Biomass Conference of the Americas</em>, edited by R. Overend and E. Chornet. Elsevier Sciences, Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.68" data-doi="10.1007/s00122-005-0030-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twigg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">Analysis of expressed sequence tags and the identification of associated short tandem repeats in switchgrass</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTobias%26rft.auinit1%253DC.%2BM.%26rft.volume%253D111%26rft.issue%253D5%26rft.spage%253D956%26rft.epage%253D964%26rft.atitle%253DAnalysis%2Bof%2Bexpressed%2Bsequence%2Btags%2Band%2Bthe%2Bidentification%2Bof%2Bassociated%2Bshort%2Btandem%2Brepeats%2Bin%2Bswitchgrass.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0030-3%26rft_id%253Dinfo%253Apmid%252F16049706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0030-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16049706&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.69" data-doi="10.1111/j.1471-8286.2006.01187.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyden</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twigg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarath</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genic microsatellite markers derived from EST sequences of switchgrass (<em>Panicum virgatum</em> L.)</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D6%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2006.01187.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2006.01187.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarath</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twigg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pangilinan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative genomics in switchgrass using 61,585 high-quality expressed sequence tags</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D1%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-other" id="cit-2.3.357.71"><div class="cit-metadata"><cite><span class="cit-comment">U.S. Department of Energy, 2011 <em>U.S. Billion-Ton Update: Biomass Supply for a Bioenergy and Bioproducts Industry</em>, led by R. D. Perlack and B. J. Stokes. ORNL/TM-2011/224. Oak Ridge National Laboratory, Oak Ridge, TN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-other" id="cit-2.3.357.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-comment"><em>JoinMap 4</em>, <em>Software for the Calculation of Genetic Linkage Maps in Experimental Populations</em>. Kyazma BV, Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.73" data-doi="10.1016/j.tibtech.2004.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genic microsatellite markers in plants: features and applications</span>. <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DVarshney%26rft.auinit1%253DR.%2BK.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D55%26rft.atitle%253DGenic%2Bmicrosatellite%2Bmarkers%2Bin%2Bplants%253A%2Bfeatures%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2004.11.005%26rft_id%253Dinfo%253Apmid%252F15629858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2004.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15629858&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226565700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Breeding systems for cross-pollinated perennial grasses</span>. <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D11%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-other" id="cit-2.3.357.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarath</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saathoff</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-comment">2011 Switchgrass, pp. 341–380 in <em>Energy Crops</em>, edited by N. Halford and A. Karp. The Royal Society of Chemistry, Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.76" data-doi="10.1007/s00122-010-1477-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samuels</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development of 1,030 genomic SSR markers in switchgrass</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DY.%2BW.%26rft.volume%253D122%26rft.issue%253D4%26rft.spage%253D677%26rft.epage%253D686%26rft.atitle%253DDevelopment%2Bof%2B1%252C030%2Bgenomic%2BSSR%2Bmarkers%2Bin%2Bswitchgrass.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1477-4%26rft_id%253Dinfo%253Apmid%252F20978736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1477-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20978736&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.77" data-doi="10.1038/hdy.1987.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Evidence for multilocus genetic control of preferential fertilization in maize</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D58%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1987.44%26rft_id%253Dinfo%253Apmid%252F3471740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1987.44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3471740&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.78" data-doi="10.1038/35072009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Yesterday’s polyploidization and mystery of diploidization</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D333%26rft.epage%253D341%26rft.atitle%253DYesterday%2527s%2Bpolyploids%2Band%2Bthe%2Bmystery%2Bof%2Bdiploidization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35072009%26rft_id%253Dinfo%253Apmid%252F11331899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35072009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11331899&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168467800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnquist</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tew</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1992</span> <span class="cit-article-title">The detection and estimation of linkage in polyploids using single-dose restriction fragments</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D294%26rft_id%253Dinfo%253Apmid%252F24202510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24202510&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HB49800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.80" data-doi="10.1006/tpbi.2002.1577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species</span>. <abbr class="cit-jnl-abbrev">Popul Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPopul%2BBiol.%26rft.volume%253D61%26rft.spage%253D349%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An SSR genetic map of <em>Sorghum bicolor</em> (L.) Moench and its comparison to a published genetic map</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D50%26rft.spage%253D84%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.82" data-doi="10.1007/s004380050355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Chromosomal regions associated with segregation distortion of molecular markers in F2, backcross, doubled haploid and recombinant inbred populations of rice (<em>Oryza sativa</em> L.)</span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DXu%26rft.auinit1%253DY.%26rft.volume%253D253%26rft.issue%253D5%26rft.spage%253D535%26rft.epage%253D545%26rft.atitle%253DChromosomal%2Bregions%2Bassociated%2Bwith%2Bsegregation%2Bdistortion%2Bof%2Bmolecular%2Bmarkers%2Bin%2BF2%252C%2Bbackcross%252C%2Bdoubled%2Bhaploid%252C%2Band%2Brecombinant%2Binbred%2Bpopulations%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004380050355%26rft_id%253Dinfo%253Apmid%252F9065686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004380050355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9065686&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WM03600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.83" data-doi="10.1007/s12155-010-9081-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernlem</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzatella-Craig</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dihaploid stocks for switchgrass isolated by a screening approach</span>. <abbr class="cit-jnl-abbrev">BioEnergy Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BRes.%26rft.volume%253D3%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-010-9081-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-010-9081-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.84" data-doi="10.1007/s00122-010-1488-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hierarchical classification of switchgrass genotypes using SSR and chloroplast sequences: ecotypes, ploidies, gene pools, and cultivars</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">805</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZalapa%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D122%26rft.issue%253D4%26rft.spage%253D805%26rft.epage%253D817%26rft.atitle%253DHierarchical%2Bclassification%2Bof%2Bswitchgrass%2Bgenotypes%2Busing%2BSSR%2Band%2Bchloroplast%2Bsequences%253A%2Becotypes%252C%2Bploidies%252C%2Bgene%2Bpools%252C%2Band%2Bcultivars.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1488-1%26rft_id%253Dinfo%253Apmid%252F21104398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1488-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21104398&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2.3.357.85" data-doi="10.1007/s10709-011-9597-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Post-glacial evolution of <em>Panicum virgatum</em>: centers of diversity and gene pools revealed by SSR markers and cpDNA sequences</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D139%26rft.issue%253D7%26rft.spage%253D933%26rft.epage%253D948%26rft.atitle%253DPost-glacial%2Bevolution%2Bof%2BPanicum%2Bvirgatum%253A%2Bcenters%2Bof%2Bdiversity%2Band%2Bgene%2Bpools%2Brevealed%2Bby%2BSSR%2Bmarkers%2Band%2BcpDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-011-9597-6%26rft_id%253Dinfo%253Apmid%252F21786028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-011-9597-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21786028&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F3%2F357.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/3/357.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/3/353" title="Production of a Natural Antibody to the Mouse Polyoma Virus Is a Multigenic Trait" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/3/371" title="A Screen for Round Egg Mutants in Drosophila Identifies Tricornered, Furry, and Misshapen as Regulators of Egg Chamber Elongation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/3"><span>Volume 2 Issue 3, March 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/3.cover.gif?itok=hlyjI6G3" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/3/357" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Jimv01zv6bh7SiD-i4VUro2P9K-9g0w6fwBqyOQVKag" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/3/357" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/3/357" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/3/357" class="active" data-icon-position="" data-hide-link-title="0">A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459259" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-M7GrNJmRwd_6HgojFCCeLn5uwxAZ_36k5zmbMJi8HJ0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/3/357.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/3/357" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Wl5Z2RS4ySybn6--QU2_9-Kae2mhsfS4qRiQpBxEDLg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/3/357" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/3/357.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459259" id="node459259--2" data-pisa="ggg;2/3/357" data-pisa-master="ggg;2/3/357" data-apath="/ggg/2/3/357.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Linglong Liu</span>, <span class="highwire-citation-author" data-delta="1">Yanqi Wu</span>, <span class="highwire-citation-author" data-delta="2">Yunwen Wang</span> and <span class="highwire-citation-author" data-delta="3">Tim Samuels</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">357-370;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001503 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459259--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Linglong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linglong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinglong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanqi Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanqi%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanqi%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yanqi.wu@okstate.edu" class="" data-icon-position="" data-hide-link-title="0">yanqi.wu@okstate.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yunwen Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunwen%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunwen%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tim Samuels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tim%2BSamuels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samuels%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATim%2BSamuels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/3/357" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/3/357&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/3/357&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/3/357&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/3/357&amp;text=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/3/357&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/3/357&amp;t=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/3/357&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/3/357&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459259/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459259" id="node459259--3" data-pisa="ggg;2/3/357" data-pisa-master="ggg;2/3/357" data-apath="/ggg/2/3/357.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Simple Sequence Repeat-Based Genetic Linkage Map of Switchgrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Linglong Liu</span>, <span class="highwire-citation-author" data-delta="1">Yanqi Wu</span>, <span class="highwire-citation-author" data-delta="2">Yunwen Wang</span> and <span class="highwire-citation-author" data-delta="3">Tim Samuels</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">357-370;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001503 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459259--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Linglong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linglong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinglong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanqi Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanqi%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanqi%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yanqi.wu@okstate.edu" class="" data-icon-position="" data-hide-link-title="0">yanqi.wu@okstate.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yunwen Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunwen%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunwen%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tim Samuels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant and Soil Sciences, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tim%2BSamuels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samuels%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATim%2BSamuels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=357&amp;rft.epage=370&amp;rft.atitle=A%20High-Density%20Simple%20Sequence%20Repeat-Based%20Genetic%20Linkage%20Map%20of%20Switchgrass&amp;rft.volume=2&amp;rft.issue=3&amp;rft.date=2012-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Liu%2C+Linglong&amp;rft.au=Wu%2C+Yanqi&amp;rft.au=Wang%2C+Yunwen&amp;rft.au=Samuels%2C+Tim"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459259/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459259/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459259/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459259/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459259/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459259/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459259/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459259/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459259/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459259/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459259/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459259/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/3/357" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MATERIALS AND METHODS</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">LITERATURE CITED</a></li></ul></div></li><li class="odd"><a href="/content/2/3/357.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/3/357.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/3/357.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/3/357.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
