<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/5/909" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/5/909.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/5/909.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/5/909.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/5/909/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/5/909.atom" />
<meta name="HW.pisa" content="ggg;3/5/909" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine" />
<meta name="DC.Identifier" content="10.1534/g3.113.005975" />
<meta name="DC.Date" content="2013-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Zapata-Valenzuela et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Replacement of the average numerator relationship matrix derived from the pedigree with the realized genomic relationship matrix based on DNA markers might be an attractive strategy in forest tree breeding for predictions of genetic merit. We used genotypes from 3461 single-nucleotide polymorphism loci to estimate genomic relationships for a population of 165 loblolly pine ( Pinus taeda L.) individuals. Phenotypes of the 165 individuals were obtained from clonally replicated field trials and were used to estimate breeding values for growth (stem volume). Two alternative methods, based on allele frequencies or regression, were used to generate the genomic relationship matrices. The accuracies of genomic estimated breeding values based on the genomic relationship matrices and breeding values estimated based on the average numerator relationship matrix were compared. On average, the accuracy of predictions based on genomic relationships ranged between 0.37 and 0.74 depending on the validation method. We did not detect differences in the accuracy of predictions based on genomic relationship matrices estimated by two different methods. Using genomic relationship matrices allowed modeling of Mendelian segregation within full-sib families, an important advantage over a traditional genetic evaluation system based on pedigree. We conclude that estimation of genomic relationships could be a powerful tool in forest tree breeding because it accurately accounts both for genetic relationships among individuals and for nuisance effects such as location and replicate effects, and makes more accurate selection possible within full-sib crosses." />
<meta name="DC.Contributor" content="Jaime Zapata-Valenzuela" />
<meta name="DC.Contributor" content="Ross W. Whetten" />
<meta name="DC.Contributor" content="David Neale" />
<meta name="DC.Contributor" content="Steve McKeand" />
<meta name="DC.Contributor" content="Fikret Isik" />
<meta name="article:published_time" content="2013-05-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Replacement of the average numerator relationship matrix derived from the pedigree with the realized genomic relationship matrix based on DNA markers might be an attractive strategy in forest tree breeding for predictions of genetic merit. We used genotypes from 3461 single-nucleotide polymorphism loci to estimate genomic relationships for a population of 165 loblolly pine (&lt;i&gt;Pinus taeda&lt;/i&gt; L.) individuals. Phenotypes of the 165 individuals were obtained from clonally replicated field trials and were used to estimate breeding values for growth (stem volume). Two alternative methods, based on allele frequencies or regression, were used to generate the genomic relationship matrices. The accuracies of genomic estimated breeding values based on the genomic relationship matrices and breeding values estimated based on the average numerator relationship matrix were compared. On average, the accuracy of predictions based on genomic relationships ranged between 0.37 and 0.74 depending on the validation method. We did not detect differences in the accuracy of predictions based on genomic relationship matrices estimated by two different methods. Using genomic relationship matrices allowed modeling of Mendelian segregation within full-sib families, an important advantage over a traditional genetic evaluation system based on pedigree. We conclude that estimation of genomic relationships could be a powerful tool in forest tree breeding because it accurately accounts both for genetic relationships among individuals and for nuisance effects such as location and replicate effects, and makes more accurate selection possible within full-sib crosses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/05/01" />
<meta name="citation_mjid" content="ggg;3/5/909" />
<meta name="citation_id" content="3/5/909" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/5/909" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/5/909.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/5/909.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/5/909.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005975" />
<meta name="citation_pmid" content="23585458" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="909" />
<meta name="citation_lastpage" content="916" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jaime Zapata-Valenzuela" />
<meta name="citation_author_institution" content="Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author" content="Ross W. Whetten" />
<meta name="citation_author_institution" content="Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author" content="David Neale" />
<meta name="citation_author_institution" content="Department of Plant Science, University of California-Davis, Davis, California 95616" />
<meta name="citation_author" content="Steve McKeand" />
<meta name="citation_author_institution" content="Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author" content="Fikret Isik" />
<meta name="citation_author_institution" content="Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author_email" content="fisik@ncsu.edu" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=S. Avendano;citation_author=J. A. Woolliams;citation_author=B. Villanueva;citation_title=Prediction of accuracy of estimated Mendelian sampling terms.;citation_pages=302-308;citation_volume=122;citation_year=2005;citation_issue=5;citation_pmid=16191038;citation_doi=10.1111/j.1439-0388.2005.00532.x" />
<meta name="citation_reference" content="citation_journal_title=Int. Rev.;citation_author=P. Bettinger;citation_author=M. Clutter;citation_author=J. Siry;citation_author=M. Kane;citation_author=J. Pait;citation_title=Broad implications of southern United States pine clonal forestry on planning and management of forests.;citation_pages=331-345;citation_volume=11;citation_year=2009;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. R. Brown;citation_author=G. P. Gill;citation_author=R. J. Kuntz;citation_author=C. H. Langley;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in loblolly pine.;citation_volume=101;citation_year=2004;citation_pmid=15477602;citation_doi=10.1073/pnas.0404231101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=G. R. Wiggans;citation_author=B. J. Hayes;citation_author=J. A. Woolliams;citation_author=M. E. Goddard;citation_title=Imputation of missing genotypes from sparse to high density using long-range phasing.;citation_volume=189;citation_year=2011;citation_pmid=21705746;citation_doi=10.1534/genetics.111.128082" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=J. van Heerwaarden;citation_author=J. L. Wegrzyn;citation_author=C. D. Nelson;citation_author=J. Ross-Ibarra;citation_title=Patterns of population structure and environmental associations to aridity across the range of loblolly pine (Pinus taeda L., Pinaceae).;citation_volume=185;citation_year=2010;citation_issue=3;citation_pmid=20439779;citation_doi=10.1534/genetics.110.115543" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Y. A. El-Kassaby;citation_author=M. Lstiburek;citation_title=Breeding without breeding.;citation_pages=111-120;citation_volume=91;citation_year=2009;citation_issue=2;citation_pmid=19393127;citation_doi=10.1017/S001667230900007X" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. A. El-Kassaby;citation_author=E. P. Cappa;citation_author=C. Liewlaksaneeyanawin;citation_author=J. Klápště;citation_author=M. Lstibůrek;citation_title=Breeding without breeding: is a complete pedigree necessary for efficient breeding?;citation_pages=e25737-e25737;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=21991342;citation_doi=10.1371/journal.pone.0025737" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=N. Gengler;citation_author=P. Mayeres;citation_author=M. Szydlowski;citation_title=A simple method to approximate gene content in large pedigree populations: application to the myostatin gene in dual-purpose Belgian Blue cattle.;citation_pages=21-28;citation_volume=1;citation_year=2007;citation_issue=1;citation_pmid=22444206;citation_doi=10.1017/S1751731107392628" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0.;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_title=Total and Bark Volume Tables for Small Diameter Loblolly, Shortleaf, and Virginia Pine in the Upper South Carolina Piedmont. Forest Research Series No. 9.;citation_author=N. B. Goebel;citation_author=J. R. Warner;citation_year=1966" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Grattapaglia;citation_author=M. D. V. Resende;citation_title=Genomic selection in forest tree breeding.;citation_pages=241-255;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0328-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=C. R. Henderson;citation_title=Use of Relationships Among Sires to Increase Accuracy of Sire Evaluation;citation_pages=1731-1738;citation_volume=58;citation_year=1975;citation_issue=11" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Lambeth;citation_author=B. C. Lee;citation_author=D. O’Malley;citation_author=N. Wheeler;citation_title=Polymix breeding with parental analysis of progeny: an alternative to full-sib breeding and testing.;citation_pages=930-943;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100627" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Misztal;citation_title=Technical Note: Computing Strategies in Genome-Wide Selection;citation_pages=360-366;citation_volume=91;citation_year=2008;citation_issue=1;citation_pmid=18096959;citation_doi=10.3168/jds.2007-0403" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granie;citation_author=E. Manfredi;citation_author=J. M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_title=Silvae Genet.;citation_author=S. E. McKeand;citation_author=F. E. Bridgwater;citation_title=A strategy for the third breeding cycle of loblolly pine in the Southeastern U.S.;citation_pages=223-234;citation_volume=47;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. H. E. Meuwissen;citation_author=M. E. Goddard;citation_title=Accurate prediction of genetic values for complex traits by whole-genome resequencing.;citation_volume=185;citation_year=2010;citation_pmid=20308278;citation_doi=10.1534/genetics.110.116590" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_title=Linear Models for the Prediction of Animal Breeding Values;citation_author=R. A. Mrode;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. M. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_title=SAS 9.2 User’s Guide Statistics;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Stranden;citation_author=D. J. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. A. M. van Arendonk;citation_author=B. Tier;citation_author=B. P. Kinghorn;citation_title=Use of multiple genetic markers in prediction of breeding values.;citation_pages=319-329;citation_volume=137;citation_year=1994;citation_issue=1;citation_pmid=8056319" />
<meta name="citation_reference" content="citation_journal_title=Interbull. Bull.;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=B. Villanueva;citation_author=R. Pong-Wong;citation_author=J. Fernandez;citation_author=M. A. Toro;citation_title=Benefits from marker-assisted selection under an additive polygenic genetic model;citation_pages=1747-1752;citation_volume=83;citation_year=2005;citation_issue=8;citation_pmid=16024693" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. Zapata-Valenzuela;citation_author=F. Isik;citation_author=C. Maltecca;citation_author=J. Wegrzyn;citation_author=D. Neale;citation_title=SNP markers trace familial linkages in a cloned population of Pinus taeda-prospects for genomic selection.;citation_pages=1307-1318;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-012-0516-5" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="twitter:description" content="Replacement of the average numerator relationship matrix derived from the pedigree with the realized genomic relationship matrix based on DNA markers might be an attractive strategy in forest tree breeding for predictions of genetic merit. We used genotypes from 3461 single-nucleotide polymorphism loci to estimate genomic relationships for a population of 165 loblolly pine ( Pinus taeda L.) individuals. Phenotypes of the 165 individuals were obtained from clonally replicated field trials and were used to estimate breeding values for growth (stem volume). Two alternative methods, based on allele frequencies or regression, were used to generate the genomic relationship matrices. The accuracies of genomic estimated breeding values based on the genomic relationship matrices and breeding values estimated based on the average numerator relationship matrix were compared. On average, the accuracy of predictions based on genomic relationships ranged between 0.37 and 0.74 depending on the validation method. We did not detect differences in the accuracy of predictions based on genomic relationship matrices estimated by two different methods. Using genomic relationship matrices allowed modeling of Mendelian segregation within full-sib families, an important advantage over a traditional genetic evaluation system based on pedigree. We conclude that estimation of genomic relationships could be a powerful tool in forest tree breeding because it accurately accounts both for genetic relationships among individuals and for nuisance effects such as location and replicate effects, and makes more accurate selection possible within full-sib crosses." />
<meta name="og-title" property="og:title" content="Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/5/909" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Replacement of the average numerator relationship matrix derived from the pedigree with the realized genomic relationship matrix based on DNA markers might be an attractive strategy in forest tree breeding for predictions of genetic merit. We used genotypes from 3461 single-nucleotide polymorphism loci to estimate genomic relationships for a population of 165 loblolly pine ( Pinus taeda L.) individuals. Phenotypes of the 165 individuals were obtained from clonally replicated field trials and were used to estimate breeding values for growth (stem volume). Two alternative methods, based on allele frequencies or regression, were used to generate the genomic relationship matrices. The accuracies of genomic estimated breeding values based on the genomic relationship matrices and breeding values estimated based on the average numerator relationship matrix were compared. On average, the accuracy of predictions based on genomic relationships ranged between 0.37 and 0.74 depending on the validation method. We did not detect differences in the accuracy of predictions based on genomic relationship matrices estimated by two different methods. Using genomic relationship matrices allowed modeling of Mendelian segregation within full-sib families, an important advantage over a traditional genetic evaluation system based on pedigree. We conclude that estimation of genomic relationships could be a powerful tool in forest tree breeding because it accurately accounts both for genetic relationships among individuals and for nuisance effects such as location and replicate effects, and makes more accurate selection possible within full-sib crosses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
    <title>Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"5kpdUZ6UE7qTrY-ebNPAKEWUM8gsCdQpDAYl-D58NBM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459462","apath":"\/ggg\/3\/5\/909.atom","pisa":"ggg;3\/5\/909","ac":{"\/ggg\/3\/5\/909.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/5\/909.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/5\/909"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/5\/909"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/5\/909":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uQTET48WT6ISde46zeCsQH1Ba9GVkA6IHtdp4JgOmqzQEW26UENTTvpEpzCm5rZFbfXiLMrfy5I79XCw15\\\\\\\/sSfE2zOn4MV23Pv3xDB\\\\\\\/Uyr1ezw7WvTwoVQWnazRuR11H2LmdGuFXHwwTqqYnXmDNsnA8xA0o9aCZH6YaZ7Db0yuBDCx+NGy7iboRX8kxIHlMsYUIRYmX9M\\\\\\\/Ouh155afRtYrzQBaRy+OkogKAjcQmarjO2YEyO7K\\\\\\\/lR+EKnlIVWdY+nI6kDeZ9BD8bTW2k0lohMKXDr4\\\\\\\/4tIauz8DwzPG4Uwk3nBdcsPBd1OrsE97+Lp8iv4A6HgSAy2ZcNYLAs0X1PsQb\\\\\\\/P+psz8yzG+9ymOH8ZES8Pk60ycxLUWNVQ2wT\\\\\\\/kdTyRaqR3qw8IYaexmrVsB3UFdsuTSzJBhet2adjngmFRFW83DyC6FDtypi13S9\\\\\\\/zbIl\\\\\\\/ch3TSVzZwbr1fQuSSIv8tnV+Rwmm\\\\\\\/JXVSVMwhcoJJpLj2jRwAU54tidnwwmEjZ+8cmVprs0R4HE2riYGpJ+S6+6IdKGHPbttv1i\\\\\\\/OEIKGVTURYv4DROEvGYKT\\\\\\\/HpMZc\\\\\\\/qqy38zMOyPFttu+QB+9JjxE4iLUQRn+mmwNX3dBbnIbCapuu1kYUEeC7qtRNA5\\\\\\\/2bPnUsNNHxLCB8fjHmJIVph5pPKJ2CFO2Necy1odvMp\\\\\\\/XxFe4uN3YdcPXfreyhOkTB\\\\\\\/w580OEZpZgkiy8I+COuwurW6w7L96qVZAjuBeTCRZ30nbVSCl2dVrw\\\\\\\/Bg38qdulIfvnUpmNY8gfXKg93a0h+YIfKkWp+ISMlNewnDq2NV0RtZ+jLTEYzeXmGIlcLWZT+XNkVw0DAqsXGQLyG3LBuUtMNYDx1QIyNOHJoofUd7hq28O21w4MfE7KBU420yBUMwrH5xjS4zNgVoRVZVwUrtIm0WOllCWgYyYwGs3LKdygBm2PtgrUfaJjTXnoTGleHhzbNTUDyilaeNZfUvO0Z5wiUbtPCK\\\\\\\/6WHgSi\\\\\\\/YYzYU78zYNI7phk9JqnuPDmmfWWX9lGewM2\\\\\\\/lsedxBAvJGdMDZOAcKWwOS12Q51WPnoi2\\\\\\\/tXEBOzmGIMII\\\\\\\/EvsCRM\\\\\\\/my52Zpjmovi\\\\\\\/3cK5ov2V0k7ZhJxMHry4wYVcphgytY2rq6bXS4fh+Pc2Ybqi083TOKUoeR\\\\\\\/5si2MGsV8wnEzfWnm\\\\\\\/pnbhKBdMbgE+0ywcMaE0FZ\\\\\\\/4OltE7uaQWoa29BDjsmd6mIDXmjXa+Ri0yLaguPOMeinJgBw8Ptwq4b\\\\\\\/Fys33OVasUaFE8f6eFl\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZXgqrhEpVMqPQlhEkZPi2g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222096703ecabf244b067d3a84a87ec99fe30b41ca5ebb1a3d4c3096640bcfc487\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MydDSK0g5yunO3LCiO7mgDxL4yX0Sfg\\\\\\\/iX20HiUVIx+JCZmaPn9Rkfc1f30GKCLOdg6TZLzL0N110fi\\\\\\\/dPQS5fBudvvIhHh9y+yF9PqD+XsjJte\\\\\\\/y5GF0EByWyxpNxLLfjgBK9aZ2EMw90hivTUHzd2YPrLLq++hCtdsNIlj5+m82lejUrCl6r+gAbNEgwEnMaSU5zu4XpgM4QCeN9CC6g8T\\\\\\\/JfuoI7XUDrPCrNhBeJ+SPEJTKcLwPmxktVF4w8FDfwQ+K49PCA9f+X9GgOssv2ozLwQJ0uqhOU1pAnZP18iH6OeQ0ZNG3XOdbmX88YW4oMUsGODTyKWWuYr9gGEK6SMrQ+MRQhKtfppzuHjYk2HWrnrUKhfYk4aYk485r0q5eLbXsElB5jw7jZjDmkFOIvWCMklZmJYnWIRSu39WaPa\\\\\\\/sCYz\\\\\\\/r1Z0qNLn+mTXS2iBzLFMwQVLiuqdhD2rltGkfNF5sxP6JcBhnoRtXhVuK6TVutRlmJm4515fusEislAAl+pvd9Fkc3pxF2zYvQGpbp2weZWSlYef\\\\\\\/XzaY3DlYhCzyZdR4vm2LNvaIKYVFTTWl4CM4JjqS8WoWlxZuhtp4Y2MUwpvDrPGRRAVQLh9Kod6OUuYnzirJnaJvf7Oc06Wb7v5XH9zkpdmudn0ynv2IxWZ229+sdJymunfIoaMWStLckPBqQ+g0RZ6wTFzyKUiTx6HEm7HwLkxVocsCIxAV0WvLImBZYh60mjoIVPmMg2FhQmUPa+H4lQVYdfUAo6bT0iPcFJJP2XC3IOpz+7Ff5FNqNyIBZfgy44RUvXjvIJY1jco0T5\\\\\\\/aP9MWImshiUhR50KbsCjaMNB5ynexlBxKIi\\\\\\\/qqIOq3WDT2ggYZ6Xbk6q1RbP2K8QxF8L4UgPWNuXxtvFbomDSjBYlzxcBqlBiFd+WFR\\\\\\\/UsGk9falJGynDWC8jS86I9Flq9gPwwFFpYGEL10HBdDrEZODqvEa1MJiYWsMtkCGhNi7B9rqOPRH5wqtp1VXE5Iqzz7A8NOcwu2fme036VIdzmcd5BilohxSoWyXA+W2+QAu8chrTsaVJIhr8pEyk+rHwHX3KTz8f6SjKLwXTob4scsRElM7MUbGSsZ4U7dr9dzbggBxcqTMOCqtHT3K3oSf25qZTmSbtOZcbvw7azAsIrOKL1AHS28E+khFI\\\\\\\/WQHpyC1X0Yh9rn98y+J8zZFrHM1BkE3H6fNgXw8XxmOYMHfw19BI0lnS7zViOk9Q14qKQ\\\\\\\/oUw\\\\\\\/Rfj\\\\\\\/4d9Pm1TbplSmmOkBz6XUZEJDFC3qhkUlic1EqcTTfWEcZZZzTgbmI9gcT8qcao9NcpyUk72Np7ROvX4WW56YF+xCuqUFPYcRPX\\\\\\\/bpoeoAeRZOGzFg56twaw7S\\\\\\\/WL2dWChSd53pTxmXobHBug6OGV\\\\\\\/mi8Rg2Ww\\\\\\\/TCX7EhoK8mUcLxLdorZvMXPia5HSXsRKxpt1f8fT2a\\\\\\\/h9R\\\\\\\/UzFOYPNFluDY8SQKPqWufO3au8dwJyH\\\\\\\/OvKWR3Tqh2zndbugqPYt12KtckcssZpzSgL1wuokETc0jf7wxpFtoofPTT87kzxNxK\\\\\\\/8zwOumZX5yJlnIWj0LvDV5a\\\\\\\/XCqkk9RPgXxJm68n6tJH15MX8NyMYVnSflBHH1rbTDmjJCRrWygSVhd9egEfsdfQafBcMl0UjxCWiRkPZ4t8oTg\\\\\\\/HLDD1BHXgIMMew3vKyEnu7i7FbIhuUGjAV+EX7x3pMew5XnE0jL+gvkNKya3ZioOxsOiLnwcczp8q4DAdxTiJ\\\\\\\/s3onZHWnSebNP5LI6ZjqVU5m5lUMs7Pj5yC8\\\\\\\/tm+3WPdY2fTtKOKMElWgV\\\\\\\/neMlkT5mqw84EHi9sc1dpjSO15+GOL43AFvHG8oCbOHx7pXIwN+GaSJot+14uOBG\\\\\\\/S2dLUN5vAt81pb6Z90hE\\\\\\\/loFs99UXhCZZGzZJyyT6FAa5fT3heoEIjs\\\\\\\/YUedj877nzyflbIO9KQyLB5pPvFkWnC0lrFCG+VnOSKwa5mBLaGyemBl4+A\\\\\\\/AyFKLfCtbx4odKVUBfmzptP9ZZyb3Ocsu3ZxOG9UPcPAoFJPzfbxUYM0dKOiQLEwpfFq8balhAecx5WIq65A4\\\\\\\/PGgZ2mIPS9L3mbbjh\\\\\\\/UJMNkXe3WWMrpFjPPpQQHIBtn5zsvBeODtBJG64C8HeFqNKh8mQA1BJhu8ivk+i65i+QpR0topRi1rTzKRUj+piVShQe4qFismCARVxEQOq2KGpUmm34VE0BkQXZStvKz\\\\\\\/D405ZvR9QFVJc55i1qd+6RE9p0mFYA\\\\\\\/VuK9CaCu0PmI5vmXl09JWkYkYWwDwthEO9ZD3jy3N236++hksgRXkE3bHNmHrpumPrpkx1JmbGpR+7Hta\\\\\\\/UZ22cOQ00NTxbkJY35eMZKFn3xLmgvxIxI0Pc4E3jYy75kdM+d8Nvot6j48GUN\\\\\\\/Xa+q+7xgOM3hOup3Lqx8hIMQVNwqtYuztUF74NtNwYZxN6vnAjKQ4ymhgRC\\\\\\\/mKAQ2ZK2m8V7+vj0bF9diF0bKnWI1Ia\\\\\\\/29ZiTWghHQ9xdo8nZH60\\\\\\\/WXSQYHaA2Wu3iwM0gwrb6TJLfOsnAJxSbIDhSQ95ZH3vjSk8h7p1QTObCDlPbs+GEHH3Yyqs+kGbbA0RZ\\\\\\\/L2WJDazEOXE9fVoHlqgSsKOwVu1EzVaKl8hymcg\\\\\\\/GYwU4tn1X\\\\\\\/P\\\\\\\/vXfEYeFGlQAGYQwd1ImTgLghzplcJNv0zyv3P55w+XL2uerOongrw7Qd7dTdjvB09ZNjJ1A1rSSARLTDDCkur2fg4iJMBMy1ufKYnX1PLfkF0DK7CHlnrqp1YOa7+jGAbTb\\\\\\\/WJw\\\\\\\/CHlbTGbeHZAWwyTbszPIekp+9YKSAN\\\\\\\/uNSuRceNFVH9jcUzeww0pM9rB1OUXhMR5rQVgDg7TH4RAPYNcV4dAQKSiNwbpLSfR8O0bBih6QTfMghr2\\\\\\\/ko5zqi5HuKxLVBEl70cn6yC4CaFyKuQr7OTfJFAf1\\\\\\\/kuMV9Hq\\\\\\\/+iTC8JRmDt+C5HPlNXHzRNg8qM3eK4Wf15dWXktWk+Jm5XfU3d\\\\\\\/blpH1t6L64t9R2iHsgR9q17C0MbVzXkeLiJ0KRLYeoN5eAop+9fpUK0dZe13ab\\\\\\\/58XRR0RrOU6fbSB\\\\\\\/V7aK+c1Hv9IjdXYsaeP5PyCer638AKAmW9piwlkv3sj+ccqW9Px3cdr2wny\\\\\\\/9dOIKRh8WGFBR8l6giZsXFZo2hOBdlRnAE8yWO5anX6BuDPCTeO9sU+YYDI4gR1xFH0vHJkU\\\\\\\/EN6zHTxz7OYszigmekPzqaCdmIyH5H1AhiFttDYcqRzwc\\\\\\\/RLrTbMhZirktsCKzWe+Z4jTCKYk\\\\\\\/MjnDOTu6cV8nxnqeKGsv5GXGVBwJ7So0RhUEnh8aVzi\\\\\\\/rjxswGIaQCUm+MZScTlwxZLi7ezItX+Fa2p67K3edoXg0cTyUDqZoZmenbBZAs8xS\\\\\\\/Wtv+GnDIlKIAIbxPJ1djFJ\\\\\\\/kF1x2q9J+FbIzAP5JXCPL7tJLS8PtXFibn+vP9jC0IP1W3pKYSzMrG0n0qvaUTb\\\\\\\/s3kFQkgQIrFizDvrcW1UkjTUGUELB9BYKbmPQJDwUVn+lyHs2L1M2hgVHK6eETnlzd9hVABH6bMgZSv4854mJJI3mHKojBfkt4COXnK5JIVdEUYrpcn2jWjyghRy4v7ZzmErPN4pZ6XJ4x7CJLWvHKlnzMuzLcW0VzOJ1iumsLipAyNG1zZR\\\\\\\/cbgsmJVDpzgW4jAIncPa0fNdzVJE4pyl4CpVtUSmc4ZhK7QIt5rfE+5VvP2F2Nk+N9nqs9pQQeVGtty9LxlXYpHETroNwPqs\\\\\\\/qi4d7iU6NPLJfUHtvQikk7aRttgCFg\\\\\\\/eqbzTfT7SyFvMPNF1RADBgS1BbdaN5B9Yuwe22W8eZLYd0arSRbwctknVKsXj830t3MenkSP0waRpig3Y8EohHCDVBK0du8CTFMzZRDoDPBjiK7RkfYfLiD9EXnX2dTAtv6r2FFxFKMoo7065zWZTFlCVQIhc8j8WNbW++VoqHHfY8h5QaDATqvE\\\\\\\/uP5jT2QiQmO3SQC0JnkuN3whdzviFsm8qfrv09lrBZvyEpuvqV3uiMp5\\\\\\\/AVoFkJhhtVlLSHgkULgnzHnBCKku+utssBz+4zScf3s218G\\\\\\\/n0UtGo7VeB0gBQWXxekMXb6nOV2q06ns7JxWUa0qFmITPPCNWBFo74496rnftMbHR8EOVjeSpaki6Ugt8U58PWuLjRAgsKB5CQw11Vcs9T4UewuGOUCigBt1V2gaUtDTFOVfuKIhjWPBHE2ks7\\\\\\\/DdvhgKq+j7jscbF8HcAPf719NJAyDjyboEG4l\\\\\\\/PWT6Shaoyd\\\\\\\/KrVwEm\\\\\\\/HUWtF9hWp\\\\\\\/uPlsIXzrBwzfFZWHzxZWzjNKG2qQ6SkEc9l7I+8CK22\\\\\\\/Zt5bBhKtu6dnhNCRi1vAQWKIzb10Mx7Qnj5UA97l2Y4rVXKlCBYWYfVUerzc2vZZL6BBO5GfJmB8Clh2YMa35HUr0dMdoh4SsTaSJE\\\\\\\/D+Izokd98uVD3gMRoRtXkdh\\\\\\\/d9AMoA\\\\\\\/xqgsEA3mgJXArNTphbcDLZ0mYBzjofb0n+BpgH0GCd0Vutbzhu48NBa9s5SEiJdG1LvnXmtnOC3CYQVzQnh925e5SZZGNSKzvBrXj4ROmVBEubSXpWxjRjzkNnjBX\\\\\\\/oLmHiqhE7fp+3N9SF1gAP6KUmOy+abT98KmCF811hQ6sXwwT8GY04u8HZxhA0xCcCtNDWqS0vs23e1U\\\\\\\/oBJHdlgSU2MxSFKpCmfWlQqvYAdbYFQ\\\\\\\/iU6+QGFCmTQFCxtVMPS1+z63EBk1wFRzPmVQhDo1Jp74zRUr4hK67o5ubVkP2bueBlHFzjvH1kiEM6KY+La45Ob6sp+pjyL6v+LuleDIlstO9erOuFWepB8pH2rUMHYTWIiNbNtJIfdtsbFhbUtdzpYL7LoWOwEQzxL+0a9CcjTZAteCsFIofxAvK76zSoX67LfqjA5NbijNZRY\\\\\\\/pk1Kxq7dLWb9NNIJVpL5ltKdqJM63pP1dyFffkg+mC79ZHqaJXpMC7QEqUeB2C7j0wyyD7Euozji36qyGM2ZwWzAj1QqBAkMFhkVNizsrKdTdrhrA+xcJR6d+2aaJEPqhd8pm2e3ixV1IkUoFKoZCVR4G1VVldYFVaidtKQ3scsrW\\\\\\\/fdI2klAdCH1PQ+nd30+zPKa11oPdwM7cMnl46wKOovEfL7skK\\\\\\\/W\\\\\\\/Gng8ZOoWIneW+6pWG3nNUTgVU4UeEJ9fkTnXEOrWNztrrURP6+NyNIbMVvpUIa8UJ5uvJZR8nR4UFEtYkNepNoIJA4nqN675pxDn2bRbs1vvTNmpHd8bv+o\\\\\\\/NWfrItQyX4SyTruVaykpSpGjMhzBnDY1PUNXewewCHxP+Lemv8W93Fr+vAmgDKD310S8Iq3RDqAjga+WCwiH+hoPS\\\\\\\/ZIW3AFyXgzzeGjqh4ywoHeviZoIynA6MU66FKA3zZiqP68gRq716VGsfZQjzXPtCTVgzZ2dPtSPAIDblRdnRELk5gQErXhQC6\\\\\\\/jd61iG9JznrRzlBXDSb7U9Ai9HuRY1cHhLsvyCHcT2G9va3dhD0bZIzDgfQ7Pon+zCyrKCpBhf3+e+1MM+QgnfV78gGcWG1rtIaZtkbowP2CNMXsFVDXUO9AMyTphMcG5U4akA1CHgi4JFPWVe9Ue2gCkNYsgugY8MmNM441u+uAQ2S+4i++164t7N1krLtS\\\\\\\/gjlE1itUyTVmqfgZ7KxYx6kDCOOONVrmYgcwoCLP7VI5Hh0BZLsWkt31qCoXjdFwqe7Bbj+4FF222HpXy4gtTdo2mR5Y6jrezAvHjWIhb9AnB8ACStH1qNKK31ZR5MorZirsooqcKn7XA7o9u0vHpVX7u+40fl6Uq9ZtPOSqm255uOWw69H7QO6JS19CEOX6sKoBHUu1DjSyjXMrqiV6LPUj9oCSCFzi1ePAdbYagHrR3+KtUEa4HSueGZoagbDA39MB2W49D84tkewvSTMrV5zB399wUJgRqqHAXwPrO1a9I2cT7+9\\\\\\\/AgpF9mgEBGndVI\\\\\\\/DgSskuPXsHkwCxwp0+4e9Ok\\\\\\\/tY6IS5rNToe2HwBl\\\\\\\/FySiWMWYhO6itMSaptmNRJIxIZSaSMz9NdwtfjbtN87PKfa+WZ4I47\\\\\\\/WjqgWE+5x5whoJrG\\\\\\\/WtsoxyoApInUARlAD4MhCWNBiavrO5E4CxNXu+7JiR\\\\\\\/DcpOrro6XUd3WxBHkNKD8hiXa5JcNFngO4VH5IAo8h7AKJpnEFlkj6k6kv7b8Q5vy+ng62ZZFMWHPGNwiH4cavTvOVFIN6o\\\\\\\/IcHphXuE9CzLuvsq46slMTG+s7j\\\\\\\/jmAhgoigqqgc\\\\\\\/YVZI7nq2d7UbBUEXRJld45IWg2adfpqqkdYxcRwm42qT4l+T1mDEZ+WMtDGqA5feUFZ7VpiPRi8QRyeqiBim5TYiZWOINCU5CxWWOT+DRM242RFptd7DhNytFiVMTe8BLBHAcFHDKF6f7CK4oeG\\\\\\\/ZUtHOTUIFru0umVmFzGH0c8A1g7XAogpzP9k4jJprIdWiX\\\\\\\/ZwSUEPUgv4ek5QRSZxmYvu1ljrK5W7Q3MO6Y1ZT6aEVVUJYNtY\\\\\\\/9woDjjfTLB5vptdhcMAf6lemqnWk9NThl7BTu5UnoPxUFlAd5T6d4XX15Evxm0DaaEc8y1SJEMMFUDxgaw5W6F0UIUXc2kADaPybo0fevOqfoLjoxqa80nCNz6VgmVylnnaXN7oVJ3nkgeOvCkCYhjh35fNlMITUnQC9712oHS4KjBR7EhloeBN+f4g3HA1gTUhHct3A6kh16oXHNCBCjINg8B4B9QpgEdmQY9Pvlj9pv18+ER8jyzdOWcVLyrG+MBfPHbsgfhDW\\\\\\\/4QuZmA\\\\\\\/E4rJSYLp1Lg8Nob9kA+6N8Uff7QoUZPqFhQ\\\\\\\/z8jIFtAyq2xoZRNlp2649kLmprJSh\\\\\\\/4XBgRGEwv7ZcBrayzvuLfQld2NzK3kXw33B81Gkblk12h9UT2a1iL2K9qdzaN5H88A+QA8hwCSruQLVJjzPeDHiJgATXwSQtGw8HQbkfUIDLAMgcn4LUm8TcRTIJveQ0CcP7RkF5xXp2p589Q1KUaA2Rpn8SF0Q6GIRc3EZsnO32CYnacFb0XyiJazifAED+XJGdOofLJCdxhAyOkkl9lhUcJjEAV4Gm4Rs4ISuUb6axuZCSukePKDpuzN+jGAFtb1NGgErbYGrKIh7pvbaV8iLKq5DX2LGnmkuF+1O+Wo87IWjymtFmDl\\\\\\\/7tyN55vpB9pEw+K\\\\\\\/TmW29vmTMuhcvdF4oKezIyaPYmT\\\\\\\/hurlZv\\\\\\\/CgAZ8K6pvjp1SrMYLKeEp\\\\\\\/nVlIRFJtpqINTeIUu7899KUO44ETEYAIRdH1SjDgeevoDlaSpDfEwwBu9gdVE0vkZKTne39LxwMLzEpgvvSzw38tx6rxPAQZ+9vY7AfP\\\\\\\/U+BLFwU2iclGf4INX4Xa+HGN9H6KMAB5BaufZDZGcNdfvdYiHXgcmEicu+XZyF6oU3Ou9aicjLkpZCzStHwZQJBMFTMDtIMGPwczUPUiuHZTHt41uJ4qWB88udBm6Tnux1uNwikJnx6OOZkfsDp7HoaP2JCbpxvReAAkZ9POMcixwjWFaVOPNpiFBLOFTf0JnQcUKrFh2a3P4UWROatXE3nuWq0Mrdx0QTQgsWbJx9PG4U1HmlQjTEqsE+qhwsNFsgfo+yFxXdw2T+QQxhM3UVV6iURL7ZFjhFV7it+PQABl52Kgp\\\\\\\/BsSytWp0ayQ5ya5L8Pn5j9OkH6eN\\\\\\\/fHc9BbFIoweNPIdWEV5LT8zJQ2JgEVuTqtiWRxl\\\\\\\/KQzfgoGx923lq6\\\\\\\/miD0O8e6hSSth3BjocX2RIc\\\\\\\/27M3bYa9IVGMsYJLXyzbPo0RND09r\\\\\\\/3sQzVUoblcJDy6tWO6hc\\\\\\\/VeCU8l6nDIS+x0TrRgLqg2Z5Sx0u7L4RMRLlDwqwW4AAHYf7K\\\\\\\/TSyG4UkKOPJlPGl\\\\\\\/aTC6S87JUPbLBIb9ZQ57R2BNUGKzcDpK3LY8gp+1Mil8WuRhDuDVCcL\\\\\\\/JoRFDD2slC2uKFLxsZC\\\\\\\/ABKSTn9V7MyatTRDBq2yclCCwfcLMw2bITDbvg5tdYYpeMwUdlRFTowGkbqqkFE4YMRqucOvf+jMeHps2LBZG432U+t4AjaIkhw+PuD9wby3yd5a4AY0BbUkCeDA0erj5BRBprKRru7nOFs4D693UicXf\\\\\\\/1RFumiLJ56Fx8xkBhAFAYDJs8AN9CRnrvZmMIXgWFX\\\\\\\/1ED8PIOhkXnRmjVWFqfVMtxg8MKh1twBwkXnmntGgySNrAYVG4wA6UuhwTEn9VOTyV\\\\\\\/N3uaOK02Cg+byuh1JM\\\\\\\/e3Cmw+0GOuKNoUKFym8jH1u0YReY7+Ohc3LJPt6q5PROPdwrvyNE60uyIRS+tC8hJXfO8\\\\\\\/PthNAf3mZVGtADDf6RL0iDD8j5QTqGaI37M2\\\\\\\/PzRifZRtcxXK9nkbGXXBr90oYEyxDUWvh4s1MasaoJUOxnL111ddT6t6p\\\\\\\/pNJH7HDsi2gLKVCa5hz3xrZFh0bO7bpgF\\\\\\\/Q5faBojUt5ZgrEBfzs1MSU6G9QK\\\\\\\/\\\\\\\/jMz+Ks2kxAF46nH5sD1a7pGk+JOgRmtYqgRJyDVtuIaqAYFzvgYCQZ\\\\\\\/EO8pyZu9jcJMZh+0wPyy9RvsOKQAsLRH\\\\\\\/3TwvEeyxiiZqYMGQ4XmvkBNRPSXoZsoVJ3uu5Gr7rqcvOWROqXmCjgUlBapYwwYYeR4flJvG62D+bWcbCglYqNsL0gHX34Gv8CHLSY03SQ2ecWlMdEGS2hW+XyE3qc9hJKQ2p1xdKj5z25IwciImGiWD7+HBlIjEeXW3eJAbpqV6JiNLVKxaT1FNfW8vxBn4nEkB0xEp1RnOwfXdItL7FaYazcr4cmSei8DCROxBYhWSCFbBUlscET68V5VuJVYswn8K60CMRmL0MJZvyWD+5Ku4ZO8SAr+CMUMuN4xmMzaaGT\\\\\\\/wDcF4as2vXNiMCLVZ95c6bISZp5GVIw6pk\\\\\\\/JNhM6xdzAHnyaJfUweXHnmf7Ol2q+v1upi5AbjwC2GUSaLn37AjniPaaiLy6IZkV9pPp3TtejkRzvi0TUfQTAI+UDNwCaFYjg+CjB6JK3HhBTGcRlwwAcOEufBJda7NChiw0OHbcVREs+KLF+cJo2kug5DPa0P2ZZ3P5NPuP8ZvBNzHDUX2A3pYaM0gPTcUQY8uzaOUF+Zd27m0E54ct+TfZqM78K+xd54OCUzrlxttoliF8UsAkqh8yE8f+YPaY63u7NgtRDs2ng1ZSNwOSCkQ6jnlXfojfy9O\\\\\\\/6o7UD\\\\\\\/B3IRTRYiX+WZDrflQySklk9gl8hN3QZrn1c7mdwCuioosG7dzEV\\\\\\\/dlpRFY06cXbaBeBiukHTmIWgDX\\\\\\\/AW6hkqHwrI7oG1TmiPse5ijjmui4Nz0xuWE8hLQkQUpqoY8KgNQuhA6S51n+rG3rG5eUSVbq7V2Kv0mTb2ZSReOlYjypWSnHdHxVfTRwZSt0rnwiA9Bp8rA9lwCzeZi\\\\\\\/8XeU9WIrrhpi5xG129+yAS7z6bTKQ2v6uEWZqgOGwjs8IyULwZW9h2\\\\\\\/Gadqpmhk2fWKhydpHwO\\\\\\\/0betjv3xTX54pOO0d0ocp6iBoIfkM3Cd+hcpi1w10U17jRiBPV6FL\\\\\\\/2ZRUuH8pdFqduRBzg1Kc0fryMuGPjnblWUdFldaejiSq3MpAYrdYZWB7QtPjxJZG0agHAhpYUPZPDr0QqNm3Gql+Ro+O2e2IYBBmQu5jMUjHFaBFgMkqcAwzmU5YAe+Q0VlRLwuJsPirLTxj\\\\\\\/Lb8HojLnZWgCac\\\\\\\/gGtX20\\\\\\\/4CsfV5qQLIpuWeomqFs2J3AnlB2+A8oD5YPyQ27UuPklt5\\\\\\\/FKwjaXviDgkpaZ+wlrc44tvQpIFkM7OJO7NT4\\\\\\\/a4B68aiG6EEtXO743hS5Ujuf8APmEpQhZjudjnTwRQoKVe7jXwMHcww8eq+YcUcoHfUss56sPiZjCgMgtEr9FC+CgyoU5V4TJmIphzK8dY3OKVYSvgsWj6Qi81UUD0nUaOjT1Sj9tU5k7L2\\\\\\\/FAOH1jwItyI6OnfZnG9\\\\\\\/HsgInnE0ZlYCFWwx45GWUEILYu\\\\\\\/KCqRc22c9t8dnvVON8oztM7sCB\\\\\\\/\\\\\\\/bw5rxIrEqImyQhOcSOpzluDcDyOSTpDtADo5fjRMpTxWs7V5bHpj4mozEHrJdAZgUA316wALg\\\\\\\/M8p3xk28Zimm5ERQ9cCDvxyL0I+JCDqIgOY2Dl32ecPgZ1vmLTmBEMAdoqH5gjyElk5lAqRmLTiRQLRCzf+ljlWbYyWotdvNY7cOsGtphod7uOEUbCS6IzU4ZdFx5vFyO0xGDiYjzvyl+GDerDc5gBGDuMW1wonTSxMuhDDHijTXSQaZJN5O5z3ANfRoarzNfBqFpNKNd35IlP4sK1go1RXJr8PX7bU8nUCD7EqQNl3yudIgxSp+eBIzchSFMldRkwFRE3dSEXPYCpU2q6qhVVXu5TCoYyZp+qa270BZtoTVskGssBK5ncVWus0J\\\\\\\/zUI8MbwgysKOwbHPJYesX3EWBld5wCG94ujSelYBNPvyI92X\\\\\\\/vYGM5IGN8eM+hXt4ocNPaK6DcNZsOB9GMORXOTea1QvjnTV2c2VXGzD4lsdAEQPs\\\\\\\/s8lhn75KkN1RNJdWySe2BTUF45cM2dIBQ\\\\\\\/5V3QE793KsA9nIYOVtKxev2mdy\\\\\\\/KRdU5RTf\\\\\\\/Y\\\\\\\/s7YBiisbzgbgywassLz5ReFwv9tt4Hpqj2pGv2B3Hyg\\\\\\\/+BgTlqVCOW+LXJjVp5P7kLbJ8Ttv\\\\\\\/5ccDEUHkShIuwg8kB9TZldIqGdG2NN505MQuOprGAzSqSwbu+v2u3eRQwwN9yDRhjsa27b+JtD2M17gsdPO+TG5KcVMZ2Eh83TTv3M3xlLmVAIfugH0FpsPKKyHY01Iut0YYIFScfVB5FjNC9mq46Ch015XTtv8\\\\\\\/Q2S73P+ZsTOJe2dy+ivI23FBZ29H3GpBiKBR0lmnBmQp+nbO2KvgK94WrdtH9N93bBTn\\\\\\\/5yKAoKXenykOBH0bCIwH1H0CwNCQyCqS2h3jMGFK8NkdPCp5jeOdH\\\\\\\/YOGp5S\\\\\\\/I8SS8pt3s++iKSgwzOo81DWWWiPiGTMjMeTIBrkIk\\\\\\\/r\\\\\\\/hHzEQ2IWW9p3xrE\\\\\\\/zTJsFB9\\\\\\\/BH5FG4esErcy7KA0NVLcNbjzHzohkHfO4wer\\\\\\\/aL7mUF7jlHE7hUmYPaECdvwmGBlhignHOjKYHoUVYbE1gbFFqOZVpzUZyLQDIXcFca234JaW3DheKdwcCXRvRVtfqa+LfOsYEYgtuUTWzr0sGDbsw2dGFPxhW4zW+nKEmSqO37PHqBRYGNb+Dlt2+\\\\\\\/8e9BvTaBxeY8qY3pP1wWqXxcSjhBckVtgRzygj47sz3D1QRofOHrHKWL\\\\\\\/oIcpFY\\\\\\\/mZMLf9Y0hKzfnS7zyEhHS8s8K0dcMxRG97z86H\\\\\\\/CxJ8ZYvYV7az42AI8Z0lZp+Bxy9xKESgnhW5oKaJLdJvTNs48DaaQMviIexkejgyIHJNfz3qaTPLanQVgcWihIHw47NJKtiv42pW3aIuLTTMwEEh3Ie+nhUWbyuCXEUa4WZWJcMBwQYflFNDZvXm7H\\\\\\\/UhKZsNnA7HVoSdhpi9+0X44dgHogUj1fba1Yzbzu0nHhRw8bO5e0dcyVbPQ3gpuGIO91xcOvEuuxcPP\\\\\\\/q9H0Jn0M4f\\\\\\\/ua0svufP7yNDS4vA9yY53PeSkIRYu84Mfh54chyzO3XSGrNiE4cg9VXOX+QqswN0StF0VoIZgo3cMnQ4NdDPYQb3B7IIEnqdMWMZShgf1eMHjusoo9Ekn\\\\\\\/LAlHZjyyWq8sa3lBjfZvjqRJpbS5TeaU86Ou8voOhbn1s+tz\\\\\\\/ZO3ksA3okpIVksGhEeXtExhUd6zf3apB9OOqv7SYc2e92ySSmC8c3p5AH\\\\\\\/10dMqLIsZIKYxo05vwnZeNP2vGq18WIcQIL54CX+EZM5SCWyhcERCEK3w5tr8QCnP3ZY83MFi5fMQzn+Q+UmYX\\\\\\\/cm2DynObd9bOWydokJ7vceP\\\\\\\/Cw7w+Qg3vjjM6W89i5OhMjrxm\\\\\\\/UTJt761rkebc00dQR3XolzIWzcg3o1Z5FDsZOPvFr7Yk2x4VqMVpTMdB\\\\\\\/J2kqel6Djgmm20lMPe4iP\\\\\\\/jVzbGJMLh942j\\\\\\\/b\\\\\\\/7FTEkQnL+jYncG0OLSJHWPAljNhiQmWR\\\\\\\/rSGLHercK7FMEaWepuBxIPRtEPBnpP911Kna4XZZmJccPGKof3UNwUnFv6ORen0aUwXo6nZ\\\\\\\/o4FqiGP0oOlF7pYi5r9YF90ZDHtO2HSQUuC0XCAePbSWDbSzrWGQ9T3ZAKwL4da32y+8To1rL\\\\\\\/9uMqTMlm5auRvZVnp4paKdsC6Odb0kBYv8FFfyocCLFWI4iPLwa3SsGp48BMWmTgTZj2FgP94qeDkHpI\\\\\\\/jboCdnMDa7R7URhmMgurwKVvR0mPCprgcBQkKRHEdyk7na1Lv7e0SvqhmjSFFPOrVDhTjV2ClM3OakNQKvZjSkJzwrsQ73\\\\\\\/IGnX47u9htq\\\\\\\/LvX5y2GCI6qqTjeXLEWh5vtEEtmmng1H5h351v9Gan+I1arieNaMUGYDVCjKXXDvA1Cmz80dgDbqatnySZ8tfzl3IfSE3X530yQq0WCM7LKTmX3ALtV3i935wYFg6R4c9fgb8CIiOBbk6N5kQVqSDsb2QOmV341O5jybdfg2qHzGB9eCML+VA4ooFgQ9YyeOCDQ+E1CmTMnjNc75rHLyfCC0wTECwe3ZKVcmRCaZ4OWTrrCPBSc08e93JDhdnRLQShUCbmgL4RhiT6QL4LaGg4bEid5ui4mQQWdM+e87t94\\\\\\\/+sT8mAejbx4ej+1qv8ltIOO+xGiA1Lq7mr26xgLj9rTePC9gynMjN0fYz7HeExGw1mRkLQSdQ07DbIT1zFTECS+dxbxZ9MKTmTKh6PcbQoyI6BlqECQzqSGsbzcZM0r+zWv8Tis07A+7j2xpVfhL9xfaExMpByNHmf3aBfoQbLceTSYFiRNUbGX15hNmoXpEkmXRAY5jcEXpfRYvb+25eYJ6844bXQFWNPBgyXqAd1mcOC3b85BwBOxbNjPu6LN1L5+26s31sOurS1Hn0zcex3hqUrSVbr+HcA79iaFQLIBXWwRNlYqrhj8psakQyfLdrZo3WZox1TCtRXNH7+X0ulK5TpH8CpWcmWbJd9UJGImNDc0wpoaGL0rhCp3jCFmNZV6qV+h9ToyYflFUJ8lOLhcj4SfSGRXlP8z1HTMHrGID3cP+bg6ERS1m8Lj0uPbEvZI+wCdUj7N+9GA5UC7HTIwKUhQKWZIpuzSFgl+ncm3KcPHmTxHusYbVbONpa67v7QVPA\\\\\\\/J2+zxC\\\\\\\/ZCI+DlfsfcweFZ45lT2tLBjGCGiTCBLt1H2mesohIu8a\\\\\\\/Ot1sBGBCdrV57+mwuJQvBOUWjP6P0IEA5Z8SLUfBw62n4fozftvu6f1ty32sT3d6Qf7JGwzbTYOSG\\\\\\\/FMcr4AWwrTywaz9QGKBqk37UbEfa1I8mQNsVgsa210twadIXlP\\\\\\\/jnN6aLPH2w5H8YpqL6FOvB5GWKj\\\\\\\/eTUnunZ4LE7pc1yPpEZrVqkNOiIwd9wPYSoeccWrNpejKJX+\\\\\\\/6SphpGuXhqiRNt7y1YMxntx4Hy2Z3mx\\\\\\\/lPpS2VuBQU5sxrY9EhptSV7YPWSE6sdlk4FSaTOqnQ5F0FEgYruvtSimhuoNBOdcCamlrqBuXHL8TLn+HFQRwMXvGohAEmjqu\\\\\\\/+J+Is3d7dzE\\\\\\\/8UBoyr+P1aCvvJcTQ1TPVL8FeJERWLbEMx3csoPzdsyOLDJX0xkUycw0rJ\\\\\\\/MUoi7vO9YVtT6tuA\\\\\\\/8oHRIojKX+uVPS2\\\\\\\/uFqKWdS1NNNrgZB9dkIVykcK7caVQM\\\\\\\/Nn3AQ55YAHpJ3MQ5DDkNubX0awc80J\\\\\\\/rf9q6ZcGL1Vv+ynyZ9LiS5wi9lo9QFsZCUK9YtqToPyaXpudR390QYAjD+peikTOYT4reZujymbrez7G6Nl2zBgi7lFYDdYyrfMYow8Cb8v748hqCVeR8Ort4+m9QwnTs81N0uQDqQ08lB+Md2+7I90NqxDdFnYL0xfSWTxJSNCrL2VDzXxuI4h2NByAyvjLo+vDzp3AHRi42gSfWI+srAmYoEmnrkJUjTNESDYzRPJyDV8\\\\\\\/Shf0G6u7S2Fv6RidOU7MJMBUppVS+e6X2CbMXWZqqX9qqTdv8OmSC3ozxF4btnOhvQxQdI6hp+vL1PS5aP3nIspd1sn6Yq7cWmo9E+K+UG830cKEgC46q9oj\\\\\\\/0ILkBXx+dtrlDmxItsLnhj4iOSeuPgHKuHAucToy+ret9gRiw1gGUU7YhWo4lKCRTEvucJ6C3Df+h8ijZXLaOeiaV0Yt4hppZ7raw4B5HJB0LOlWcqm6rCwhNEthjy5q6FGF2C8iHXDdEJhDT9pO3gQHjhkUTnyA83eT+UCbyq5ikzuiHkF1KhcW8Y3CivDkDZeYcmohf6xZURdnagbtXW+YScfNT6h8Bknqp+m7gvMf2A4BRk3DLNT6jii6+AlEXqt3optOxOg3YymArv3Wv7t6Tjk3FGlk0gHrPd20hnPYIlGL3xzPCEZmFYDnOzO1HRcOCILHN\\\\\\\/PakvFOPJbBjBcKYk+brB2HiljIQFbkVO61lIYKNuukV\\\\\\\/9nFTzmS9Zu6CdNbbbN3I6WXTFAs\\\\\\\/ZMYF02JG2lk81t\\\\\\\/bY\\\\\\\/WaO6af5tvQ+x3DZLEfFJXfhZkxQlNyX+mbIvDqNYfZTGRPX+4PPR042WJiWT3jJel0EZiSmT0PBi1TOB0Qc3GGpveKaWfA1Peje7BAKAAUIMbOv1l7W59rt7qaFOh3o60pgQpTBM6KMUc\\\\\\\/cgZEj3op7rZ69ukpZ2fr40YPoYbpffKVzlzCjLS3kLUiK2KSEL5Q4C2CVLnrS4MQXMkay3fGPlvSNDPtjuIo\\\\\\\/OALrXqAIrQoHIyqPPAVoCUO2QSK0GaDgr08y70ZHjwvyeG+crvxfBuY\\\\\\\/Fc9lKezbq2QQIb+axxGt5j1znX14vc9GaDovpviycX85yEN+ELy069LPvtSzdG5hMigU4nnRRgb4sP\\\\\\\/UIOMsfQKe1e5Y73qzNF1lCJMErEM1zg0OyYmRY+9iC9NIk9HKpU\\\\\\\/xglue4PvUo3PQJePTQF1LYLQx1DdPxluJOxYwXwNmGrWYnCj41I9WuuAhsVbtU2A8sD23J7NPP+EibZBa8D4opV8fd2lM249hwKjfFQ7hWHBLNrt2tvZupmOOLOX7dwyt38qwcdFryjM4NPxmOmVDs5\\\\\\\/n2fvI8sHNwGrbZUCd3XRNAdQ839hnZuE8GrqXsTBlYk7c50mJ3fLpd21gEiptDHDcMmq2hpTVkNCXaTyDbOEYhywSpGTiGLrL9wLHGF\\\\\\\/pikHQ+b6koWvnAr199M4oCN2QTILqWV7wS58Zwq9YZ9Tff1pL1jQq3o3dr8hLpQFstGAUxFsP9zhGGvr1CGsTtPLcMODKiokhW+D44t5+2XOGBBbhECV\\\\\\\/tqVNXiaOlNtLKxhzWjrrwqjBaYVjcG3UswyeF27\\\\\\\/aAl1h+QIcRbSVCrg4ShqSyeh4ZScMILi+2ot4mDXY4cmhQx9TIk\\\\\\\/srVbH8jqZg9GLX7M4\\\\\\\/zOPAQLxuHqSOit\\\\\\\/DM7dFiav2bbNTRZYf5H5Fw0GCXskIr+WzDwrJj5CxaQewGqdx5+\\\\\\\/pc4\\\\\\\/ZWoMP3oE5DsW+prb\\\\\\\/2fhhiBOMm0EMREz1PXXGhHgHXf1oawCYL9enwlrX+qiN0mI5WoJySuoLw+p+CfjP+muCxwJevZQPaqKjRglK0dCG3jUucRle3Mb3TCu3A8B7mY2bx1sKPzDVXM0hDScYnJ55wunDoJyRW6snWkOs19pakxwUEUIvsvW1T5MDJx7bYj8dSLgFLPITLbf+MuPkVasus88F3vL01DMz1wD+fpMluwCF4gNSCb6y4wMz\\\\\\\/3taOEQfFd\\\\\\\/T+rgc7JjprMSwpSoFvy5T4haeLn1Xglhacc4QNF0bsUB2ndiu9iP4jKG2SgrUY1DCnD0g6sVCpd0Fi7QpNFhm8j1oIWMRgA\\\\\\\/LU7wQ3iho4dt9Rmk91onxP+ZwKV6S8136LgaHtS+F\\\\\\\/JJr6DKH+oHdnKa+ufZ8oMSVRbrjpqSXcjsOMgHN5RnQoTLkGsY5aCm1MItNAGxikH2PyhWuPrcpRkMSRPfv0igrxMb0vXoBp5rMzr1VK\\\\\\\/L0mfKMaoUFE4BMb6hEQSfseyn4uaKxOkSdDepthppX0C0vstCTvL7voaQmKVAhCp6B9CWJZWo1+3PgHBqTmq5EMeOHxWXn1vh8p+fksf2mKj8tmV01cdBtrOZIUJKcOqtPprqmB1bPejCV6oUtHr2SjwLJDW4u8ThB2UHgJmCXRsytCnHxnOlfZnMcq1U\\\\\\\/8gValj3dJwsjEtBx60lk5FUFlL3Iddk2fCBOIVM4npJufurhr3kYb6Ub863Zbj4WaIdFas\\\\\\\/NXx7jq092omfQ1SOyoOFH\\\\\\\/3i+\\\\\\\/ndBoX\\\\\\\/\\\\\\\/SGDptlcwutxpV2qm9Hz2fla\\\\\\\/szCVWqN8WflNB9vwcYci798P3BJdRMja500KOY+2rpFPAQV2edBt6H4eYDJ9gffUmllbYe1Xvoj6b1fLo2WhDVD9MFl\\\\\\\/isvp+ldBXyN7svO0ny4yYyxFZNzSk5jwZw3eCnr9ZbSx4nZ8C4JwwtPZRRwCbeqNXvRH6Qpn4ln\\\\\\\/G98zhp3dIKzeLnJobdeznmca1OG1o9BdjG28r7yM\\\\\\\/3hKJhrO9RSNKmNBrswS65tZlnO7WqKF\\\\\\\/Hfcg+mf0UYWUVME8hpdZzdXJhecZ5xJNHa06GtKGT58wlO6Z3TCENSAxrZFpuN+\\\\\\\/lwiQOV3RZT99o++xeWaQxXJRS5M3h9ZK2etOg\\\\\\\/kaZn3+f3+VWEPzz9VXS3mxHAa6Ds4z5Qw2LqQdV5P4tSBafBc2ogHao4YMs\\\\\\\/NdzlGhyPPalwt5nEZtoqFxzBCLI8XuN2UZef9churRhn2kJGNkPbb3Si9aiLkBC3vwQuR5xM9BEdb4c6Kng+fzIPjT2hSDQHOubmq7Om6zd7U5IlcNTxTA6kVUF4lvNrTQ7TucUtLnJQte0skHr1F2bwzyvOzwPf0z\\\\\\\/S2bt8cS5eJuXaOr8AbMnxWhuOT7MP\\\\\\\/8Qz3caJXq6LvvxvVUTYJ5Jr4VLO9z7CgkulUUYHeiqHRxUsj3PXnMK8quPKAF\\\\\\\/vf9JjVTYwIziK0sQLbnA1ZpvIe3xTy1OctDebhn3NRpPJ0\\\\\\\/PwLjmPRNff2qdwX3Rp32MN4lgVzx2RItSy3Xgtus8j6Ttfelc8IzErCoQbE\\\\\\\/uxKPhdt8TLsroJeZQyW6NWpxahx7Pjeptmz6JXz83MF6a8xe3gCBIYYUbGZufvfSnaeilAq49DX98V6i+GEPwf1Z6utIdJztmYiJ\\\\\\\/WLuIlgJirymTJqR5xTPM7+cxcQAi6GJPmVaD\\\\\\\/xMAYkFlm7n\\\\\\\/yLb3tOVzClwrDwa9IM6e\\\\\\\/uAVIWN5d18az7lb\\\\\\\/BAHdELC1AjIBCwT4m+dw0y2wSdy4bNjNdNrhZ\\\\\\\/ThulBunSErKExANNizfrYYthkObC8mzLsb2tIv\\\\\\\/puUwz6RAu3DkJ9qB1iQvikMQHoZeDc7vsqY0lmbtIvvifMAxX1U9AUQIRTurG5L1Y71SIIqKt8wkZ8SaARpGOgM9EcyhYyORXGCZ7f\\\\\\\/R6r5ec\\\\\\\/6DvINytQRo3HFSi6lG3\\\\\\\/WLkxhC6K\\\\\\\/YeDJ71a5c0aqUnMUnfPHit4Fda0y3OmKihzPjHDveiMiywFCbk9tYQmP5S0hZv0qkAp6lc7ScsSj5+7YJw9T6yq\\\\\\\/BFJV7b6bFC1VVGZLaXJun\\\\\\\/CRkGhJVmpxQhIZhZHpbI\\\\\\\/Tvfq6cVmF0TfAKeJme5Y8XuJVUlnarVKUS56sgRXkbzE9J5RgER4Be9OpywCmLevENty03SIveXa\\\\\\\/OXyR6JGJrmquG0N+sNxQqdryERv4tYTeiU7Kj59\\\\\\\/kSXO0S+ZdaDulVIw5glIUfYE9l4VjHWbtD4AOQ7NcY7kW\\\\\\\/7Mc6IkMy5h+\\\\\\\/+hHYkWleXL0vewDooZ0N6iAN6SZ1oCxpVRG6pZSNSAEox7VddRv10os9ZHGNXa2EzRz8zEebd7EqVN5Fjpyz1K2FUIs5KR\\\\\\\/CmDUvdfsQTYMkgrW+ZBPHyM2kzP8ed27hJ0ROYK5JPwHOihqvLx7Y9jZ2UbYyaPRbiOyLZkqlWQftcYHMTB0BouL3dUJi\\\\\\\/ifBzsqpyHHr0STSDpEZfXLVcw7WWhLbwMhII2O5kGU6ef0k\\\\\\\/ACp725QG5ULTZtEYyNesvHX0MnJAqPg5ydnenTagupqBemr7AKR\\\\\\\/6UCG0eZ1ZzXX8sRBxttVxDQXQifAYr5spCIsohLew48jnmnchFuz1qw9yjv\\\\\\\/0yStCeGp2z5pRoxSSw+1z6I4eeIPuK10k88m7JF8llfihek9MSnG+x\\\\\\\/6W\\\\\\\/UMZMZf18AvI\\\\\\\/MwHZzQGho3wNRPB8n5tUd5z+kbf9JrKf6KFmed9RzI73W9u\\\\\\\/o1dMdYGFSxrTE2VUGi6+mHZG8GIz7\\\\\\\/ZiQXwsGdZ3arpR6RejeLiF+kfRZBEsNCk8I0ZAB5EbauK3E5elcFSSHOA2oXhTsqX2qKj07D\\\\\\\/+BEZ3xLbHzWavVgIgGoqf\\\\\\\/DdWDwqOr6SAaPh3VuNOywY7pzvyYZP1IvZQjjXKAuFmiTdK52viFhj\\\\\\\/yEoZwtD4uCGhOMF0orZ3CzmloUNr6SvaiR1YUPbp2UxFprb+o+VcJlEISZg2jRP5IaVCic85xSUCv1RzAKXuQxrljx6wiisU8GXwdyBoEfGlOeqFD4monG3EBWSoz4L0ajO\\\\\\\/5aX3HiYcb4YEiK\\\\\\\/9lVeQr8SXKpWrkI+umU2E6+gwlZK7kF9aH9E7AFGYrUTfj8OjN83q9bqL4JF1C6GvqfZJu6uf\\\\\\\/4aLqPlWTvFIUkdN5DsRXHnFkKxH3OvQstcpoPrASb3SFPqUOvXo1WYEfnVYeO3UJs4+eoTKVNb5pME2W5qdzd7QLsfiyR9lorqx+W83pkm3Lzzs7AhngWzCW5wP2S7qzC9yXW8kNXhQGHNeTjaN\\\\\\\/CO\\\\\\\/zX56WE6YrD2CK99kBKxD2sVwNDpglwTC7Ad0Dwsl4DYWXgE+DJfa69LOKGL3gA9HoBCJKpi8BA69K7tEbHikQO9PCHKOkeAZr6I1Dd9N8qhmXQ7aU\\\\\\\/KMpCYjxw3gVBuecJkn6WRcUPNgEC4x+twZeof3yhGpKPXkgcqCLtJU1iCwb7Gp1cOvcz1TBqtSMP7U5wDHSveubsS1vWAI1R4A43twWMsWjE+Xfcfjr\\\\\\\/AUjZY84tuukYL62\\\\\\\/s\\\\\\\/mXDm+AxhjSzbWJluGtFOQt6JRqVmzQeLJ5SaYEYU0gtEsBG3jYtKY2\\\\\\\/bgOitlecvBdXApji7O7ZKfKFb7aaepBLKjXKGsA4V7vxNmR3kgCPQ2ZhZ8loej8jJNVGo3Ddbsfyaacpqq1vcfjqhswMD\\\\\\\/V+oTvpW41dGfbXjx+m8ttLzYtmJOVSpZ1MVFUHjHAkMfu7+1pF11k0IuDykyCJEcqcDjVknko9\\\\\\\/rVLRgGBw8Er+FS36\\\\\\\/NjFsiHIKPulpjdoWnjGdMYHOc9x+H3rPM\\\\\\\/jjaSdql0S2u5NuLnEiHMfIQ1JVTerVLEVpCZqhfmnWCVl6oGZYwAkTaXUcl3BGlW77yA37\\\\\\\/xSP9Bd+8g9lleRCr40ZvLA+dGGqk64qSM2fXkfMWNvS5jNbzDvSnnw4Fi1DdhSXwviY4FvmMZG2AAEiSebHR8\\\\\\\/5lfwB5\\\\\\\/WkBuPkdHISOWEAKui6TRxgJneBwq1mKv\\\\\\\/g8Gu5TrIu\\\\\\\/+jYKr8uBoG0\\\\\\\/agPQBTdlhocHLPF9XitQiSxPH0KvfpO4EXxvk6A\\\\\\\/EqzyuTvFpm\\\\\\\/ToBEXkWQl5+B2XXflcjs8bnkIg6eouvw0a9KeS4P1dUupt6\\\\\\\/uJSB4ltLfTdkD9OxipFJg18WMl7OEkRkhT6oxKU+2ZbOSP7EAbMACcFRNuxS6uCW+gq42kyc1PfS8XzTTgUkN\\\\\\\/NkKtcK4Paumy3F9u+mX8BPMBldm6T1NnOlk3Hcyyt5qqmEVjm3yzCtfBaBZNcP4pdIVE\\\\\\\/VYImC4GxGq1Z11ooB6uTwauq3WjZLV7gh1QbD5lOBi9lA0kiYFDo25YljmlknOzUJeoV0Mk0EsTMPgDJw8S7tHzDLNa1ksJZKJcbFlB+c6iQ0msSYimyoT68QOGbZUkRNUMeQF8DDujP114OCJOQuxsx7F51Cvl9VpIKptibQ24tAGoBUJngKiDB0KIBGA8iQnPVots5wJRWmK6w4KZhP66KQDjypU6Eq6iC7grPYHJ+v6qTgo++aJEJnpRA7ZLq9q43lI1m8H15UGfnxUFOOe33wRhAVtO0wSUMjs33zkuTXRTTuCUglzl5j9gHOBCR4CYSofY6VsdR+T6ZzQ0SOFGnq8DwKN6PEc+TANw1IEoj01J5uuzIfVoHmXqovJcoGzQppqdaL4LHp8E3imEuWgbprBbLTD6KpCJjI4e7AbhFWlC3JTPoopHSrlFOTNrnomveMMatIbK8P0Z7+gnvD0uTWONYYLyg+z8micJeKeXoyEnDVOJifI0lShSLjyad3V5GSG1\\\\\\\/CIbV+hZphPjUf75vKQHr\\\\\\\/km6DqxXXjlTncl0j02XKTdgPQ1uze8KnCRiOhGAjQ5H4jNrQVcVTG3oPip9CNF0GTozy11fg+rwV+KBVDzkHoExbyNtlW2u5zhhbTthB1CwP16DxPRn6dd0V+QeIe3MV+sguWilICc8x9QV5fKGRlMUXJ1xjFsBh\\\\\\\/1D\\\\\\\/8xEx6Th5yo\\\\\\\/cSGM8Xg4iMtbp43TDhpQXZJNHlx+VWO8L6U+ZcjgeaDvuRw2R0yEXe+OlP1dfxGhzq8RbFvzOXPifuiogl9rRQAJezC9gMXZmznQxUCnPr4ruuLSuvr8zGiEWleqJxyO5ei1xBDl7hnNIxhTUcyi5Mg2JefEcTYwk\\\\\\\/1drOkbQsjEx90b2FCJUCSvVIYq3IIB8PoP6NRsflD9t\\\\\\\/+AoPCl7\\\\\\\/XiGd6vHR53Ln8GssJXjSJPUswztZPdiu3qe9pofeuusH7TjB5oRep\\\\\\\/ZTnEvkAQ1nxHUtddfe3XPtWNlx6gQz6wwDqHY0k3CZePXPAq5AuiVXrQCB1QsCAY6QrQm5QZOvlop+G1BhYteqfD6r2ojNfhVvzrqEhTZV8zIxcz3HnmQWIPu62+kWBuFP0ard0q7bGyxrzflb9D0xPfxhBvXHW\\\\\\\/xqTV5rqT5bjcTsHv7\\\\\\\/U88tQkw8nvy4gFsZdBehBp59pzkyD1y6oYWgekmq\\\\\\\/IZ6xrlyO7qHhXQvL4kChMAf4y9VOaSuCKY2TFaW3smRXATdjpMS2o9wDhgpS+Tz4iwTGzn2ozRPwqQwwD1Pgy1YZ4qXBqDyTdkfxJYZsDlbL8+YBRjHvh\\\\\\\/doqs2Dzl8FyIcnJyb\\\\\\\/\\\\\\\/L8grrt5nskEp1UoT1JjpQiDnzTfEcWluPQ2oegVHUaaW\\\\\\\/Ko0QyuE1B7p2BuThXTkqwOVFQFHzN+t62LFXZApXw0deHbebXhLsSh3u52e9tuXGGB9lW8KQyOPyPn1Z4PS+oKzKQH1weUvLnHdtt5\\\\\\\/+rKSbkISkdty19JCqPaFvmOKk67jdGOPv\\\\\\\/SSOFPosbs4qL9YrSfjwov0l5eCk1Y0O4jDozVbYSt1ai3T0vY1K\\\\\\\/I+zN6\\\\\\\/8MA7IOO\\\\\\\/7N9JYQG\\\\\\\/OiMIQYWHLoJCZ7AsoOehuBD3gDWbPWi4CVXz6kH4du9uzysVwYfqo95qjJuaGt7u8+yk6yTSkkOrTbBrR4I9WxlGXtuPWTUKDbE2B2bnQUUOFGtd3Nd\\\\\\\/e8QSEzizmIwZmwMBZn\\\\\\\/fQQU5x7+JE3VT2rUTBXsmb\\\\\\\/8aE2YPGIfJJuGPB4uPDR\\\\\\\/xgDROKuqdlHrCdVc5p2AXETuhGITXV43iAnyYvBgB4xs6WzSMZtPgUSgsVx5GFqvS5kA4jF\\\\\\\/exHJgcK6bJh09l9x5IagBOEE\\\\\\\/33j\\\\\\\/7lQWgm+lM4GygIAvSgtI7l6UtsEwwtM34x1qqGPODvdmStckgr2BzVuGJ5iAv1iiYBzQGnkdxj\\\\\\\/5lYEqTsL2F9JBnSOO\\\\\\\/YD5\\\\\\\/GfqdmME0bw8Zzl1nsznCP4FoGE0BFtidOiHdp3g9xDVNK5\\\\\\\/N3NRox2Ag5shTMYW12+3pps5bC+Ag6jjCLX\\\\\\\/GW27B+eOkru++6KxQJixEGMt6xkWn1DZK9Oi1pUQNPVw7U8GCY1UHWSA1rKhccZmld9TOqu5TFkra84v\\\\\\\/yr3PHjtOzcDUnQYGpFiO0x9ZWfH\\\\\\\/L\\\\\\\/sBMTgI94\\\\\\\/vNb0y4aMK5WYNtw992GcZXIlXk1QjaTeX0lU9ik6bB9GeBQTjkWyPigcJi62IVZNpEY8+6bF0R7OhXxsZGRutq4l2RVyLWZyFnfNMfRhMM1OLkYkdySPoAVvgPFrqOW3Xkm7+zaqQ9h4hF2KTuF6jQFP6Ox0\\\\\\\/hF6n9eZqoqDPP9\\\\\\\/D1E8rA9U2s+NJ7ig81b7oeAg+PX83VdJWlKeuHn+EE2Oh3gXVHBJT0h\\\\\\\/\\\\\\\/Zvfzg4zm6ITP3wEIppH8wmNYV9Y79SGxcl\\\\\\\/yruW\\\\\\\/NbGhCU9B6THbIuTXl4Q6c8kGVmjutOaGAeJS8B34kGQFzGk9hjUyjWSwFujW4U7ORbQ++w3oKuSDiudbqzOnVfoZ\\\\\\\/PWLpsW\\\\\\\/npHJxEe9zMBxrzY+PQ2paMA6+V0jjzDSy2BicOIPYpwzM\\\\\\\/1+JbvKrlZKZlRVWubrFm4PTo62aTC+ndOC9oraKl\\\\\\\/3LrYVxoGO45pz96NUW80Jd2soNVCeqXxD3ea3DT3bkTRbgIhepup2iKTNZQwEi3oKEHunSLjFR0+YB9qVOBGYm6wz2ZEXrMS6Tm2pjH\\\\\\\/NJpsN9EP5LbSqjGkh3HFwSiO34ALQDs4OR7wDvVnuAA9+vaRyQcpP+ySa5YiKoRQ28jyqY0eQlPUkZYYDtqJm3hVDGpyupcinFT9nbjRVwKt3b\\\\\\\/Ez1JHy2iL1OdsY7A4ofRKekVBr6e9WnXT3wsitq\\\\\\\/VrTkOa6hvc7SIMYFWL51a042nKrbOBOEqwFV+3yUSdkQFI2ah1lpsBP+31dcUPp5N251dr3Joc0+xTv7scdrxsXdssn+4s3WxHUdOWeGKhz0X7CNAlP8cAoFdoxLPU35EgpsFwsx12EevJuoJZT+v6qy9aiitnLgNrc6nxRfS33cINzTH\\\\\\\/OXN8BKuQXG6kOOjkn\\\\\\\/9ffsT6F5U+BjiRMeBH2xOx1ZWV6fur0wJ6nxw1u\\\\\\\/DclGla2GZQ3QUf1GBjK0GIj5apWwTWEeo+9Vx2t7zDXDKfhYTeQVLsiiOdRGCkhI3lTM+cVzo69vRMk2VihTrrWdRfsVKCS5qd\\\\\\\/OfwR\\\\\\\/2V8uBOYRU4ukU4wI74lRV7I4LeV1jpdklFjjxopkMsUZac7SrXcGS\\\\\\\/C\\\\\\\/ZtpLeC7c3KNeS6jkeB0mPPQH4h6rZ+dvF3RP0bU651Hk+bhUOImS+88mETLiaNQeV5tm2onOumtkHL3W75t3ppUSc2zAajVZ0Vki\\\\\\\/dIoZ1kK++6PjDCBwM2pYBvWp1NWxBhuK4cvVm3VbGvK7TO3s8AIjXVxozJY3ZtCujQaG47VRhRwHbjyDGM6OQ91dkPcWH5xZLwqnW\\\\\\\/643UVIIqwFjMSD2fMegY75epTyeECKhzkrgY0yR4ndqJP2TY940pWiPsgNqAuz2f91UcRwPrTBQP0M1YCMUJ4NAGAuouO3bkFMUQBlCecPNPfBOh9DA8RNLEPRIWzc5eDGmAd24kjrJZISO12o4b9wEPwxNJUgRn3+Bqen\\\\\\\/LyDvfC7JkrIe5hWlRTymOBgvmCxy1S3h48JmFXAGuHpsmFtNkd8bKrYcyvoPU01axyTXY4yFlK23R40RWKO68CvC8Ng+NfgRuYpSCDzWP81mS8FP1BI2oMNr5gh5KPhoRoqSMlMqQxl5d6s9NOMGR10l2vRtIMREMXFMuZ4r2BrDYYgaTzezNhAKMba4dULapHu0fY401byFnxvNP+u7NYpSVhhElf061vjkEDOgUxXEpQFYDpmXHG2ZpFsNfBg16A6vsiqLdYtNFcXPzQ4GeAhfEW29zln9\\\\\\\/g6tis8iQb6cAmirdq919CNLNeEWgVAdGa7PKhtz\\\\\\\/WBuL+tAUN08oH76XdJLU+7k3Q\\\\\\\/9B+xu4aQxXdH9eTeFDfmykTW24USfRK2aH6UVfUpwlj9aJvUf\\\\\\\/OKP1B8lS5n8ULwmx+5pACEErKWcqJoZRcvHNbjXOdkebE58sOwDOm1amD4uWrHG1IdV3Fe379miMArX\\\\\\\/q5rONXC+9TYCpHT0111WdhE+zHpgRojrFLDi8nV39OjmuqHwYjtpL2g\\\\\\\/agWQFOHt\\\\\\\/Xd4RMUFeAxcPxphlnaqi9Wgj9Wz\\\\\\\/5EBpmyiHKWrV3oN21ioeCPBrrgLnWLuYxsnF5bCZ2wM1xGs7tLFgjobdYjcbzEO6GVwNpB6Ce1DQxlQstJZTc2vLZiuX8ED6UtqKuyWRlqN8lFSW4WGxz2uRsE0Jzo59w3DZwJDX3mLUN1gdNTABCqhEDsfcd5SchJkim2eHcz04fjJtrB3OJYZFN3D9P\\\\\\\/tBCYvA1DchITBJ1SWFOdjGNC9gAKim4OwBvOu2Yz\\\\\\\/OJ9awafn+\\\\\\\/wSCgiaVHXew4xSIPBrPSSBXNUCBhYtRyn8B+P8PXsDc3GiNT6uw7kC8pCmn+7meRKyRw+6j+4OSGwsSqHkAVPqRb55G\\\\\\\/w0da4jfmqEK6wtEK+JtehlbhbsBDqfcoJlPiNDbGJ2txU0Xz7sDeX61MxppkwNjiQzSa8ekUeF1lL\\\\\\\/9mrEPp55W0xy7zS+VUVUaLcLmrr+AotSH+5qUzc2zIL\\\\\\\/ekx\\\\\\\/vj0MyJ3ENAai7PVn+da9N4Rfh2qkUMLAXTVa8iQe85qNgpfyK2xfkoaZHOQMPWnhkgIVEaufdsWd9dRVpznWrR6IY9brbwb0TrkTbeHtnF5aFiwTYgSodgjQTbNba+JQi8VM6ACTOyU5wWcvAtLZKGUSTOiQTCo5ev5KxuULiO0ql0N4\\\\\\\/JwpJ7a+wItO7pIo9pBfmGXaVWGclVBPil8djUSQCwvPPIpXAZzpYdk09L7tSMcTKWiJPAx5OG31+2fFQ3b0wkMcE68eGKKII6bWGuy0l8WD1sPSetb5Axl3Ts4iuPPHOVGlar1ClE6Y9CIMPmpfx\\\\\\\/yBixZNfXN8jLKsEKGietgrHmtUwQ2TTxV0w9hOUMBkZFF87\\\\\\\/PeQ0ACZ\\\\\\\/cBxszwdMTBqdQVg6gF+sD4hqXCe2bHy1nbEefX\\\\\\\/j6nwKUw5o538owTDyf04xIY9M\\\\\\\/yl79Uvv45Ue\\\\\\\/iNp+ev8IIQra9HbCASnmW9H9Pi\\\\\\\/oqM9XhOLwKkPZx5YR+rKmd0aDLZm+iXwM1XNZbeAfrpewvHYdYPK5MklM+zBnfLGLqQl3upZNafq\\\\\\\/2xHJiXljv7T9jmLzfMB\\\\\\\/I2CYIhKjg+kcWZJ9NMYTUq5WCmkR21SeGZ6KoXYB\\\\\\\/tBkaW+L9sxdfykI2xqrJAN0pfpXI3SaP0QoJ+gujw\\\\\\\/3p6boXXiiqEXHU2AhJTOml\\\\\\\/DdL68KEJrUp3I27TlUxxuWRvu\\\\\\\/EfXnVcOooHPdAbUoUhzUrOYqrUOAnnv6lvV94zQr2hDNYulXQEg1+cEvGvAvXqgM0PapiyQHek22vPeggN4+FB0LZddwXwGBR3Z8GhYH\\\\\\\/eeexy7u+GhOC9P53IKaVuTgro5DGSu9Bce+OdmdRQYRseWpvjdVtmHVcxIVyXWSF6ZT5EoYu9EDi3+RaX41OXnQmEjUa4fE8tyWc1Der2O6ANXAgi3RG95TKDB2W7eg\\\\\\\/OVx5882DowtG2IwGfoTM9Wej3o8UF3BNRRYJccKYoF3PLrgFSZl+Ly3eIUUd+afYsaY2GAtD3sw1sefSgv+xyc4VB2EDkpf\\\\\\\/Z9MiM4M8vHemR2\\\\\\\/ZUBNsqImIfY07uXNNfT5Fw8vKR+VVgIuGhN4f9hrSOb9CksbsKcEBEmdmnhDgFQfrTpPwvkCH1+Wj0DJ85DrqO0YG2M9eOdWNIjVXXDp+Lc5MBMS8KMDugWmux3I4HsLMpDPrwP+hc45v\\\\\\\/071V+9+XRFs9SyFgE+LeRwOdxx6RNpyxvUFq+nN4idMVPtj1\\\\\\\/oc3HJUIKszqmDzuNz0QUJK7XY8eXWoac24ap79zvnk4UiIg1T5R0uNPOa4wkG2dUGIt\\\\\\\/uDHDCNcIUguSYGAEfBRJ3kPuBuqmjO+S+9bSKZ1Hda4kjZlSSw54qfvX6lISYGSdVEG33LNjiedqQtVFxHoKaL4M5jntRBDgJZ7qWd3iY3+srIGXeNjGBPdHZKS\\\\\\\/1QtM6+AiVzUN6MJK\\\\\\\/7XailObA5iyLy3f4SSE9c9UvODien3gSgi5CoQnt4TefWHwfrj9R7k7BUBQ9NqH+kPbK47ecDk5SPqYVsbBqO\\\\\\\/k8SFB+epneYYzW6+9Zrfiyazv4yw4I5AsCDnr5FbM4L5\\\\\\\/LI0RsarTrjF5E5sevMwKfJL69E0UCpxCElzWPmxBhmt9o2Crdcc+iMG\\\\\\\/oXWHFm1+kgrHU3Z41MfL\\\\\\\/XE0yelTr3zZylkDr00nZFmo3zL9vNiITeI53MBq3XPcuFUrxeiES3lJYWBigJzBcrmE99gYoaCd7EPgq\\\\\\\/jRl090+ItiwTMdnKEDs+QhN8q+2qD\\\\\\\/Tf+5Yiir1S5ewGYxTk+lrQu8vbFVSiFMzUJGwXqHIGQ1Fm8Dh5aQUW5Yx0SoYNlpCBc7Ipde3KGchNPscH52Hx2oVPtKdgeeTNAFLV4ZIHyatdnF+ZdAVJfvl\\\\\\\/WJnJ5rt9HjbiSjY19xvKbzP+4dKongs2Mqq1dogRw9CO4fGjvjezhNAkd6QMhvsGR740r86YcCePQg2Bo24FdDy9R6LX6QgKi9b1xx+CodNCxu\\\\\\\/KTyf1amjpAblLRzVV+3M0mfiwY3PfVjwi8Pg+kqZe4VsPZdnWOuypkoCKj82HODoWrzvIOWQPy8fLxJpdOt5+728y3b9Z7cXEB\\\\\\\/A56vBIPW+m+UlxizPfC4ihU+ghKG2i6oO3+FPqzJFrsh2pJ8+n0brXMiOrCO6FKZwwu7NjYBlaLHSxUq1q08+7xn+yFoZwP\\\\\\\/xMCJuSPNsLNd\\\\\\\/wCXozavEpGpb1BmrM5BS0ZYUgC\\\\\\\/YqAxyEAVptdEnI+UbGrg3J6+fXoVmAW5yOTbgYxArx9uHTXDetQIKvvdABX7sUTEPw97RxYvGMkLv0Az+XEBcGZGtzfwgbVVc02Qoi94XkpjYSrOZxB0dywCYlcAuVo1k+ZPoa5mUEHp831JTJ6c26LlnJM3c8AvbIwTi0Q1r2Hb4oXvAl3VSvDzt\\\\\\\/KiPXBpxaSAN6InBA4y6pT9YorSJCBf9vuSPjIJlEncd90FuqkJ8o\\\\\\\/j4+f9KOzDxPBVFCiAeP6OGFW3nkww5JPe0WTlRfn0luT\\\\\\\/kLlA9mup2Kdl1a7NWMQFaRgA+u07Jn7gXA2a5R84P8\\\\\\\/SIH32uentqpqPTWZgla6GX0+MpiUNHwmTGgb6o8Du7ASNI5GrYWcaesKIZBXEviAERKB+x822hT57q+3Ps9PROLPGxTpzHc2F3j\\\\\\\/iLF78rddWFOUaiFwt8eoRzH9Nx3jaS9MvZIm3Ei6tSTEYmv7I\\\\\\\/+9Wtk7YZWxg2eG61kYhuN0v66w2\\\\\\\/YbWamjGAtf14h\\\\\\\/FjiJYUsMGhkKdYe4u8zxqnlFC0bwy0uY6QNGpjEYmEFwPllpjgMLfXBQCr\\\\\\\/pHWtyUT\\\\\\\/P0C94bEVKZ\\\\\\\/\\\\\\\/1ffslAiZQcRerLMLmburST\\\\\\\/eUfgbNbaNhe9OgR3Om+9ghondcira6jCJ\\\\\\\/Ay1JB0iNPOdul82MO48F4+cZahaGDfGzl6pRv5Q9JgWUVTnSwTX0djllzckPWMhQEqueGAQac5hl1B0nzlzxlbfD+lQdnMddH+RsZYRFH5doyb8rmxL67nIO\\\\\\\/RJ0Vhu6HLneSbkjjcPIvnOH51G8woUQD8MxwmuBpXi8Ik6wW79LRpL58LjUROUU3FAXgQEJeSAHx06yXZaDhGO7MsJlZDd0iXuRENyjA8uxZrTMTObugeWpv41+PwhygKzjsIJ\\\\\\\/bTml0gDiXMvP+heuv2t8Xn2hBRygaryuo8di6Ey0HV6WrWyiLoFGU3kijJl4cToe0hDxFrPzCFDVQRPQG6gu9KvrsHNWOareVyJjQMKLe+HpmsRJdnwOGpAUVLra5hsU07ofYf90tDoD2FWXoGPHD2dac6yA\\\\\\\/fZOJ6zSg9o8nNlRV1PEfBn3JqisuXOLHcYiaTo4o4xf+EFwa\\\\\\\/CcEzs\\\\\\\/EPvZoY\\\\\\\/UmaLuA5FVsQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022erRf5hKOefsoiFHIAZhUww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a3873f6fda58deee1380ac165ae735ed49c6fde9b1869f9e3a0be7d63cc9c70e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FfLjAyBavJh6Bc4ow1SIXvfGuK1nPBGBWP\\\\\\\/i\\\\\\\/JQ5VDy8b3jQSGCAV8O8uicVw3sYFr2bE8gmD95xBqoAEZme3KJ7dS3Dp9uDSoRNqI2Dhk7vyZKLzDGI3S\\\\\\\/hg7DpwRRsrBDkX0b993z\\\\\\\/a4CVnNhXCVuUqwkdnEgxEogQ7Et0yOCP8Y+zyRb\\\\\\\/ts+xVrmMv5GPHdTXTJKeL5Pf72+Z\\\\\\\/n05OkeDw9AQKfpi8MIiCwR5I8F4md0Oxv5OVVc5RLJfe+7VXDKlVdQRSWGhVhMRBH5\\\\\\\/Kmfonvv0Pxr21D6AFSewVmiXLYmyHM+QIr+6Nvtmx0E+qXGRcpcx2H+52oXz7RNJ84y8HWwj\\\\\\\/rvQdHlImdWiBygVrSDmBEnKJ3NaUNEXfbxrsE4JFRCVKZcsFWpg49AWDmwUHtwMjlgTz7jwwRe511iuQCSav7H19RCN6Uq3I6+5Bak4mHSgOsQ0IcmyykzVo1HAOyGA2TT07KcYXzWvRHP4G9uoksHnT26mO2ieJj0V4hTgyvMddDsSdLobokbJF4JNVftjD9ihbTZwCVrW3eJuLEMiygdZN7IOcKJv64L+9thtvx32Dwmv3ROkMN4IqEjmH\\\\\\\/ppD1oMvcoZEVywIiOAlVpwV38IIQGiFBeoJttM2g0lnDdFSU4\\\\\\\/1+57Bcocy48GxH0HRvp7xsVxqfGh2QvWfPbGTkFM+O421tTHieXYoULiLW\\\\\\\/s2v3Qi0FSU3xGe\\\\\\\/oYnihqdatO8x+gZoLtdxbiltBKgKuTWWfyXgKtvvP8+9U7v0tOEfCzdlJOm27F5c+4vfRY26wtzSpnwGKxvGoZVZH93P0ZaNusHr8FekNEHLzZzoshGuRT941P1Ei\\\\\\\/bE5MwjFbQGySMnLfkd064gW3Ho3N8IcS0JT2vKCu9CZeeD6AEpd306DJCypHZ1bm2L4+yqLpj2a7RS7pe3ZwTLt6rO94N+KdgxW6zcvpB5OJuwkHmJBKLsW8\\\\\\\/WZhJM\\\\\\\/WSmCm5rvV7OH3xKqW4hFLHn4eUw7KJU2b+dNZKKPugEa+5bYFYZJvrQCUKKtzxQcUnI4VyZNRbmcasNKcnkNatNeeB5cG4FG0WgzG1vhBxa0nZ4kkWMnPKAs2COqxUzTFpU\\\\\\\/yVgHNkQVTEewiHhur0QoXLQ06FxcOEa0F2ZFHATnvLSARVBIOvfmrxc2wEuFPafoIpKzzocI4XbAr3mvvj0agt+EUyrhvwYmIZk9T9wfSXsW0CPw5zLop51ChlzTK6R\\\\\\\/9\\\\\\\/4rI5y9We+XJ26YweCZb1uho9YHRYiTSU4I3UxS96+CbGMwOgvaarnT38bigSXf7jMTTIiEAYoTEB0PkLQvACRuwJciJGyaZ\\\\\\\/Eo9NKmfSwqFUyECEWZJdgd0llJHCCMN+gfMuBjK+Agzlf8u4NKqyoBFsnXHrCX2Yqm2LfpvRqmdfYIy8G4aSXJ8M9XcaGZD+l\\\\\\\/p6Vk\\\\\\\/tQeGRs3QMkPfswBfc8NLaHpQWgoXBRMsl4St30QwdCcc9Yjf+SeuisTLDNh+2BvZeupLJLkZtrAXTFjMZV9moxbkQ7vQKGtDNV+N6ZmTkOq25mmHfvmxjBvBfgDcZe3v84C3fksto61ELn6qTRNlIavjhYu0TEDDr3xVcslKHlnA2M69b3vVk7Z\\\\\\\/xACEcLPNZkFCZR33wFnr5pu1i9dm\\\\\\\/r7I8qsNhuw34Xokj\\\\\\\/S2SezPK9iGDRJ28UqCiKkiymDvnlodLhwMGKBrt35p4LiCdS6vseE\\\\\\\/84AIJI0wUxeqBoUt17RgVLhzuDWAQlgnM6h1LFzCFNmnt+VOeff+M0bNr27CMguWlMtKG7AV5biClxnpVZFTE0rFM8eQszUGnVCHpN1K\\\\\\\/EEigKP8AGjy54qB9Tz+YzWwN7k3IMG6GTM+4Fvw3AMT6rbJeVEQ+Y2SzXm13SAofr2uW3i01y8N3qIm6BYtqJXcevaeGYmcL74Mflmd+AnmoQeMuFvP4IZomi+5DTmIZUavdswvTrDx3YsrKj0zU\\\\\\\/lTBofu3B9+7fRAsPG9ygVQCc+oxQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hqCKrM03XIyTOoocw\\\\\\\/MPKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222fcecdb503df24b3f0cf6f421999ea04f86afc1bbe733c96bd1c716d336b4188\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459462 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/909" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-R8vWMSWkxN9Avp-kbScbWnXQ-eedBTv4e0bMbLwD_DU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/909" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-01TUa674AWv_aXO0aLeIK-sXubHL8xT03vPr1Jp5PG0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/891" title="Population Structure in a Comprehensive Genomic Data Set on Human Microsatellite Variation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/917" title="A Genetic Screen for High Copy Number Suppressors of the Synthetic Lethality Between elg1Δ and srs2Δ in Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459462" id="node459462" data-pisa="ggg;3/5/909" data-pisa-master="ggg;g3.113.005975" data-apath="/ggg/3/5/909.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jaime Zapata-Valenzuela</span>, <span class="highwire-citation-author" data-delta="1">Ross W. Whetten</span>, <span class="highwire-citation-author" data-delta="2">David Neale</span>, <span class="highwire-citation-author" data-delta="3">Steve McKeand</span> and <span class="highwire-citation-author" data-delta="4">Fikret Isik</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">909-916;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005975 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459462" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Zapata-Valenzuela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BZapata-Valenzuela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zapata-Valenzuela%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BZapata-Valenzuela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ross W. Whetten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BW.%2BWhetten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whetten%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoss%2BW.%2BWhetten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Science, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steve McKeand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BMcKeand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKeand%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BMcKeand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fikret Isik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fikret%2BIsik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Isik%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFikret%2BIsik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fisik@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">fisik@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/5/909" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459462" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/909.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459462" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/909.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459462" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/909.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459462" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/5/909.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459462" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Replacement of the average numerator relationship matrix derived from the pedigree with the realized genomic relationship matrix based on DNA markers might be an attractive strategy in forest tree breeding for predictions of genetic merit. We used genotypes from 3461 single-nucleotide polymorphism loci to estimate genomic relationships for a population of 165 loblolly pine (<em>Pinus taeda</em> L.) individuals. Phenotypes of the 165 individuals were obtained from clonally replicated field trials and were used to estimate breeding values for growth (stem volume). Two alternative methods, based on allele frequencies or regression, were used to generate the genomic relationship matrices. The accuracies of genomic estimated breeding values based on the genomic relationship matrices and breeding values estimated based on the average numerator relationship matrix were compared. On average, the accuracy of predictions based on genomic relationships ranged between 0.37 and 0.74 depending on the validation method. We did not detect differences in the accuracy of predictions based on genomic relationship matrices estimated by two different methods. Using genomic relationship matrices allowed modeling of Mendelian segregation within full-sib families, an important advantage over a traditional genetic evaluation system based on pedigree. We conclude that estimation of genomic relationships could be a powerful tool in forest tree breeding because it accurately accounts both for genetic relationships among individuals and for nuisance effects such as location and replicate effects, and makes more accurate selection possible within full-sib crosses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">Shared data resource</a></li><li class="kwd"><a href="/keyword/pinus-taeda" class="hw-term hw-article-keyword hw-article-keyword-pinus-taeda" rel="nofollow">Pinus taeda</a></li><li class="kwd"><a href="/keyword/quantitative-genetics" class="hw-term hw-article-keyword hw-article-keyword-quantitative-genetics" rel="nofollow">quantitative genetics</a></li><li class="kwd"><a href="/keyword/best-linear-unbiased-prediction" class="hw-term hw-article-keyword hw-article-keyword-best-linear-unbiased-prediction" rel="nofollow">best linear unbiased prediction</a></li></ul><p id="p-5">The expected additive genetic relationships (genetic covariances) derived from a pedigree are based on probabilities that gene pairs are identical by descent (IBD) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lynch and Walsh 1998</a>). For example, the expected average genetic covariance between full-sibs is 0.5 because two individuals with the same parents are expected to share 50% of their alleles IBD. Pedigree-based additive genetic relationships have been widely used to estimate genetic covariances and genetic merit of individuals in animal- and plant-improvement programs (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Henderson 1975</a>). However, genetic relationships derived from pedigrees ignore the random sampling of the two possible alleles from each parent at each locus during meiosis; this variation is defined as the Mendelian sampling term (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Avendano <em>et al.</em> 2005</a>). Traditional genetic evaluations based on pedigree do not trace individual alleles (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">VanRaden 2008</a>).</p><p id="p-6"><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">van Arendonk <em>et al.</em> (1994)</a> suggested that large numbers of DNA markers covering the genome could measure genetic similarity more accurately than a pedigree-based relationship matrix because the realized genetic covariances would be based on the actual proportion of the genome that is IBD between any two individuals. DNA markers can trace alleles that are IBD as well as alleles that are identical in state (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">VanRaden 2008</a>). Genomic covariances are based on the fraction of total DNA that two individuals share (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Stranden and Garrick 2009</a>). In the classic “infinitesimal model” of quantitative genetics, genetic merit is considered to be the sum of thousands of allelic effects. In real genomes, those alleles are physically located at individual locations on the genome whose transmission can be traced through genetic markers. Dense markers can be used to trace IBD at each locus, and these IBD probabilities can then be used to construct relationship matrices (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Forni <em>et al.</em> 2011</a>).</p><p id="p-7">With advancement in high-throughput genotyping technologies, realized genomic relationships are being used in cattle-breeding programs to increase selection efficiency (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goddard and Hayes 2007</a>). Selection based on realized genomic relationships can produce more accurate predictions than the pedigree-based method because genomic selection can exploit variation created by Mendelian segregation during gamete formation. In simulation studies, the accuracy of prediction of net merit for young bulls was 63% compared with 32% when traditional pedigree-based relationships were used (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">VanRaden 2008</a>). In another simulation by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Villanueva <em>et al.</em> (2005)</a>, accuracies of predictions based on a realized genomic relationship matrix were greater than accuracies of predictions based on pedigree and phenotype. Such methods do not require known location of markers in the genome or estimation of relative effects of individual QTL on the trait.</p><p id="p-8">Forest tree breeding programs are logistically difficult and expensive to carry out. It takes many years (typically 15 years or more for conifers) to complete a breeding cycle. For example, one loblolly pine (<em>Pinus taeda</em> L.) breeding program in the southern United States has completed its third cycle of breeding in 55 years (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McKeand and Bridgwater 1998</a>). One of the challenges tree breeding has experienced is tracking pedigrees created via controlled pollination, due to the cost of making large numbers of single-pair matings to maximize the likelihood of crossing high-value individuals to each other. Pollinating a group of female trees with mixed pollen and then constructing full pedigrees using DNA markers was suggested as an alternative way to reduce the cost but also track full pedigrees (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lambeth <em>et al.</em> 2001</a>). <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">El-Kassaby and Lstiburek (2009)</a> extended the idea as a low-cost breeding option. Paternal parents of offspring were delineated for a <em>Larix occidentalis</em> population to reconstruct full pedigree information using SSR markers (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">El-Kassaby <em>et al.</em> 2011</a>). However, fingerprinting and parentage analysis using DNA markers is conceptually not different from using a numerator relationship matrix generated from a known pedigree. The genetic relations between relatives are still categorical, not continuous, as are the expected additive genetic covariances derived from the reconstructed pedigree. Using DNA markers to construct realized genomic relationships for genetic evaluations in conifers is conceptually different. The major difference between realized genomic relationships and relationships based on pedigree reconstruction is that the realized genomic relationships predict the actual fraction of total DNA that two individuals share and utilize this information in predictions of genetic merit of individuals without phenotype.</p><p id="p-9">In this study, using a small empirical data set, we tested the utility of genetic evaluation using genomic relationship matrices in a small population of loblolly pine and compared the reliability of estimates of the genetic merit of cloned pine individuals with those obtained by traditional pedigree-based genetic evaluation. The objectives of this study were to test two specific hypotheses: (1) the use of genomic relationships based on single-nucleotide polymorphism (SNP) markers to predict genetic merit is as efficient as traditional pedigree-based and phenotype genetic evaluation; and (2) genomic relationship matrices calculated by two different methods, the allele frequency and regression approaches suggested by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Forni <em>et al.</em> (2011)</a>, do not differ in accuracies of predictions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Phenotypic and genotypic data</h3><p id="p-10">Thirteen loblolly pine parents were used as females and males to generate nine full-sib families (hereafter family). The majority of full-sib families were obtained by single-pair mating design. Several full-sib families were genetically related by a common male or female parent (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zapata-Valenzuela <em>et al.</em> 2012</a>). The number of progeny per family ranged from 3 to 37, and they were cloned via somatic embryogenesis (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bettinger <em>et al.</em> 2009</a>). A total of 165 cloned full-sib progeny from nine crosses were field tested on 16 sites planted between 2000 and 2002. An alpha-lattice incomplete block design with single tree plots was used as the field layout. The test sites were located across varying edaphic conditions and productivity classes in the Coastal Plains of South Carolina, Georgia, and in the Gulf Coast of Mississippi in the southern United States. A total of 6253 pine trees were measured 5 years after they were planted. Height and diameter at 1.4 m above ground were assessed and used to calculate volume of trees according to <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goebel and Warner (1966)</a>.</p><p id="p-11">For the genotypic information, SNP markers based on 7535 resequenced amplicons were developed for loblolly pine by the Conifer Translational Genomics Network by integrating the results of the Allele Discovery of Economic Pine Traits project (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Eckert <em>et al.</em> 2010</a>). For marker analysis, we collected needles from 165 pine clones during the growing season in 2008. Samples were dried and shipped to University of California Davis for DNA extraction. Genotypes were obtained for a total of 5379 SNP markers using an Infinium SNP array, through a service provided by Illumina. We carried out exploratory data analysis on SNP markers using the ALLELE procedure of SAS GENETICS software (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">SAS Institute Inc. 2010</a>). Of 5379 SNP markers analyzed, 1700 SNP markers were monomorphic (homozygous in all individuals), and 218 contained missing genotypes for more than 15% of the clones. Thus, a total of 3461 SNP markers were informative and used for calculations of realized genomic relationships. Phenotypic and genotypic data are provided in Supporting Information, <a href="http://www.g3journal.org/content/suppl/2013/03/29/g3.113.005975.DC1/FileS1.csv">File S1</a> and <a href="http://www.g3journal.org/content/suppl/2013/03/29/g3.113.005975.DC1/FileS2.csv">File S2</a>.</p></div><div id="sec-3" class="subsection"><h3>Imputation of missing genotypes</h3><p id="p-12">Matrix-based analytical methods require datasets with no missing values, but genotypes were missing for 1.2% of the total data points in a 165 pine trees by 3461 genotypes data matrix. We tested the effect of different imputation approaches by comparing the accuracy of breeding value predictions based on datasets with missing values imputed by different methods. Imputation methods were (1) allele frequencies, (2) replacing missing genotype by the major homozygote of the loci, and (3) using allele content as suggested by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gengler <em>et al.</em> (2007)</a>. The allele frequency method is a stochastic approach that imputes a categorical genotype (0, 1, or 2 copies of the minor allele), based on the frequency of all genotypes observed at the same locus, across all families. Across all missing values and loci, this approach should on average impute values that do not change the genotype frequencies observed in the nonmissing data. The method of <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gengler <em>et al.</em> (2007)</a> for imputation of missing genotypes is based on the idea that the covariance between genotypes is proportional to the additive genetic relationship between individuals. Genetic covariance (covariance of genetic values) arises because two related individuals have alleles that are IBD. The solutions of mixed model equations are predicted SNP genotypes for individuals. The solutions were continuous numeric values, centered on 1, because the gene content number could be 0, 1, or 2. Gene content for a biallelic locus cannot be less than 0 or greater than 2, so we scaled the values, adjusted by mean and SE, and truncated data to a range between 0 and 2 to keep gene content predictions in a realistic range of values. For the comparison of imputation methods, the accuracy (<em>r</em>) of the genomic estimated breeding value (GEBV) was calculated as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where SE is the standard error reported with the predicted breeding value of a clone. The term σ<sup>2</sup><sub>a</sub> is the estimated genetic variance, <em>f</em> is the inbreeding level, and 1+<em>f</em> is derived from the diagonal of numerator relationship matrix. For simplicity, the inbreeding coefficient was assumed to be zero for unrelated individuals.</p></div><div id="sec-4" class="subsection"><h3>Traditional genetic evaluation using pedigree (best linear unbiased prediction based on pedigree, or ABLUP)</h3><p id="p-13">Traditional genetic evaluation relies on the inverse of genetic relationship matrix (<strong>A<sup>−</sup></strong><sup>1</sup>) derived from pedigree and phenotype. The model in matrix form is as follows<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <strong>y</strong> is the vector of observations representing the trait of interest (dependent variable), <strong>X</strong> and <strong>Z</strong> are the design or incidence matrices for the vectors of parameters <strong>b</strong> and <strong>u</strong>, respectively, which are the fixed covariates (<em>e.g.</em>, location effect) and random effects (tree) to be estimated, respectively. The term <strong>u</strong> is the vector of breeding values of trees with Var (<strong>u</strong>) = <strong>A</strong>σ<sup>2</sup><sub>a</sub>, where <strong>A</strong> is the numerator relationship matrix and σ<sup>2</sup><sub>a</sub> is the additive genetic variance. The variance-covariance matrix of random effects (trees) in the linear mixed model is replaced by the <strong>A</strong> matrix to predict breeding values. The term <strong>e</strong> is the residual component or vector of residuals with Var (<strong>e</strong>) = <strong>I</strong>σ<sup>2</sup><sub>e</sub> = <strong>R</strong>, where <strong>I</strong> is the identity matrix and σ<sup>2</sup><sub>e</sub> is the error variance (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lynch and Walsh 1998</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mrode 2005</a>). The predicted breeding values for all 165 cloned pine trees, using all available phenotypic data in the model, were denoted as estimated breeding value 1 (EBV1). The estimated breeding values for the cloned pine trees used in cross-validation, estimated without using the corresponding phenotypic data, were denoted as EBV2, following the same model of Eq. 2.</p></div><div id="sec-5" class="subsection"><h3>Estimation of realized genomic relationships</h3><p id="p-14">We used the allele frequency and the regression methods to obtain genomic relationship matrices, as described by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Legarra <em>et al.</em> (2009)</a> and <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Forni <em>et al.</em> (2011)</a>. For all genotyped trees, in the allele frequency method we first standardized the variance of <strong>u</strong> as Var (<strong>u</strong>) = <strong>ZZ</strong><em><sup>T</sup></em> σ<sup>2</sup><sub>m</sub> = (<strong>ZZ</strong><em><sup>T</sup></em>/ [2 Σ<sub>i</sub>p<sub>i</sub> (1 − p<sub>i</sub>)]) σ<sup>2</sup><sub>a</sub> = <strong>G</strong>σ<sup>2</sup><sub>a</sub>, where the expression [2 Σ<sub>i</sub>p<sub>i</sub> (1 − p<sub>i</sub>)] is twice the sum of heterozygosity of the markers, <em>p<sub>i</sub></em> is allele frequency at locus <em>i</em>, <strong>ZZ</strong><em><sup>T</sup></em> represents the number of shared SNP alleles among two individuals, <strong>Z</strong><em><sup>T</sup></em> is the transpose of the <strong>Z</strong> matrix, σ<sup>2</sup><sub>m</sub> is the variance explained by markers, and σ<sup>2</sup><sub>a</sub> is the additive genetic variance. The division of <strong>ZZ</strong><em><sup>T</sup></em> by 2Σ<sub>i</sub>p<sub>i</sub> (1 − p<sub>i</sub>) scales <strong>G</strong> to be analogous to the <strong>A</strong> matrix, which enables computation of the genomic relationships (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">VanRaden 2008</a>). The genomic inbreeding coefficient for individual <em>j</em> is simply G<sub>jj</sub> − 1, and genomic relationships between individuals <em>j</em> and <em>k</em> are obtained by dividing elements G<sub>jk</sub> by square roots of diagonals G<sub>jj</sub> and G<sub>kk</sub> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">VanRaden 2007</a>, <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">2008</a>).</p><p id="p-15">The regression method does not require allele frequencies to obtain <strong>G</strong>, the realized genomic relationship matrix. The <strong>G</strong> matrix was produced by adjustment of mean homozygosity by regressing <strong>MM</strong><em><sup>T</sup></em> as a dependent variable on <strong>A</strong> as an independent variable to obtain <strong>G</strong>, as <strong>MM</strong><em><sup>T</sup></em> = g<sub>o</sub><strong>11</strong><em><sup>T</sup></em> + g<sub>1</sub><strong>A</strong> + <strong>e</strong>, where g<sub>o</sub> is the intercept, g<sub>1</sub> is the slope of the regression model, and <strong>e</strong> includes differences between the true value and the expected fraction of alleles shared in common. Thus, the expected value of <strong>G</strong> would be the numerator relationship matrix derived from pedigree (<strong>A</strong>) plus a constant matrix (<em>E</em> (<strong>G</strong>) = <strong>A</strong> + constant). It is simpler to do an extension of single marker association analysis to multiple marker regression to account for all genes (markers) simultaneously (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Stranden and Garrick 2009</a>). The regression system was written with summation notations, as in <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">VanRaden (2008)</a>:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>In general, this method did not give singularities because the number of markers <em>m</em> was much larger than <em>n</em>, the number of individuals, which was required to obtain <strong>G</strong><sup>-1</sup>. The <strong>G</strong> matrix may be singular, for example, if the number of markers <em>m</em> does not exceed the number of <em>n</em> individuals genotyped (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">VanRaden 2008</a>).</p></div><div id="sec-6" class="subsection"><h3>Genomic estimated breeding values (genomic best linear unbiased prediction, or GBLUP)</h3><p id="p-16">Genomic estimated breeding values were obtained using the following mixed model and substituting the <strong>A</strong> matrix by the two <strong>G</strong> matrices estimated using the alternative methods.<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where <strong>Z</strong> is the incidence matrix of markers, <strong>a</strong> is the vector of marker effects, and <strong>X</strong>, <strong>b</strong>, and <strong>e</strong> are as explained in Eq. 2. The expected variance of vector <strong>a</strong> is Var(<strong>a</strong>) = <strong>I</strong>σ<sup>2</sup><sub>m</sub>, where σ<sup>2</sup><sub>m</sub> is the variance explained by markers, and <strong>I</strong> is the identity matrix. The important difference from model 1 is that we let the sum <strong>Za</strong> across all marker loci (<em>m</em>) to be equal to the vector of breeding values. In other words, <strong>u</strong> = <strong>Za</strong>. The <strong>A</strong> matrix was replaced by the genomic relationship matrix <strong>G</strong> derived from allele frequencies as suggested by <a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">VanRaden (2008)</a>. The selection index equations were used to predict genetic merit <strong>u</strong> as follows:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span> where λ = σ<sup>2</sup><sub>e</sub> /σ<sup>2</sup><sub>a</sub>, the shrinkage factor in BLUP. The predicted value of <strong>u</strong> is the sum <strong>Za</strong> over all alleles that the individual inherited. We denoted the genomic estimated breeding values based on allele frequency method as GEBVa.</p><p id="p-17">Using the <strong>G</strong> matrix derived from markers by the regression method, the estimated genetic merits of trees were obtained as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/909/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span> We denoted the genomic estimated breeding values using this method as GEBVb. This method was more efficient than the method given in Eq. 5 because <strong>G</strong> could be inverted and <strong>R</strong> could be processed by iteration.</p></div><div id="sec-7" class="subsection"><h3>Cross-validation methods used for predictions of breeding values</h3><p id="p-18">We tested two different cross-validation scenarios, as follows: 165 cloned trees were divided into a training data set and a validation data set. In the first scenario, approximately 90% of the cloned trees (148) were sampled for the training set, either within each of the nine families or at random from the whole population without consideration of family origin. The remaining cloned trees were used for the validation (17 clones). In the second scenario, approximately 50% of cloned trees (84) were sampled either within family or randomly from the whole population for training, and the remaining cloned trees were used for validation (81 clones). For each scenario, six independent samples were analyzed. All the analyses were conducted with ASReml software (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gilmour <em>et al.</em> 2009</a>). The SEs of breeding value predictions for each type of relationship matrix were averaged across the six independent validation samples for purposes of comparing the accuracy of predictions.</p><p id="p-19">We examined the correlation between predicted breeding values of 165 clones (EBV1) from the ABLUP and GBLUP regression methods. To avoid bias in the EBV1 for the cloned trees, the pedigree data were omitted, and EBV1 were calculated based only on the phenotypes of the 16 to 50 ramets (genetically identical copies) of each cloned tree. We report the accuracy of a prediction as the correlation between the predicted genetic values (GEBV) from the cross-validation and the nonpedigree-based estimated breeding value (EBV1), which will determine the potential gain using markers (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meuwissen and Goddard 2010</a>). We produced scatter plots and product-moment correlation coefficients between GEBVb and EBV1 values for all the validation populations (50% and 10% of cloned trees sampled within family). Also, to compare the predictive power of GBLUP and ABLUP on the validation sets, the correlation and scatter plots between GEBV and EBV2 values for the same replicates were produced.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><p id="p-20">We used different methods to impute missing genotypes for calculation of genomic relationships among trees to use in genomic estimated breeding values. The results suggest that different methods of imputing missing genotypes did not have a noticeable effect on the accuracy of predictions in this study (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). All four methods of imputing missing genotypes produced similar (0.71) accuracy values.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462775/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462775/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Accuracy of genomic estimated breeding values based on different imputation methods</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">We compared predictions (EBV1) based on all the phenotypic data without the pedigree with GEBVs for all 165 cloned trees, using marker data to model realized relationships (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The correlation between EBV1 and GEBV was almost perfect (0.997). This result indicates that family relationships are not affecting the estimates of breeding value, in contrast to the result reported for unbalanced datasets with little or no clonal replication of progeny genotypes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Garrick <em>et al.</em> 2009</a>). The EBV1 values were therefore used as the true breeding values, the standard against which other estimated breeding values from the cross-validation studies were compared.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/909/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatter plot between predicted breeding values from ABLUP and GBLUP (regression method) for all 165 cloned trees. Predictions based on genomic relationships are highly correlated (r = 0.997) with the predictions based on nonpedigree-based analysis. Phenotypic data for all the genotyped trees (165) were included in both analyses (no subsampling for training and validation)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-369191679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scatter plot between predicted breeding values from ABLUP and GBLUP (regression method) for all 165 cloned trees. Predictions based on genomic relationships are highly correlated (&lt;em&gt;r&lt;/em&gt; = 0.997) with the predictions based on nonpedigree-based analysis. Phenotypic data for all the genotyped trees (165) were included in both analyses (no subsampling for training and validation).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/909/F1.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/5/909/F1.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/909/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/909/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462835" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Scatter plot between predicted breeding values from ABLUP and GBLUP (regression method) for all 165 cloned trees. Predictions based on genomic relationships are highly correlated (<em>r</em> = 0.997) with the predictions based on nonpedigree-based analysis. Phenotypic data for all the genotyped trees (165) were included in both analyses (no subsampling for training and validation).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-9" class="subsection"><h3>Validation</h3><p id="p-24">Efficiency of markers in general is evaluated by a correlation between true and estimated breeding values (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Meuwissen <em>et al.</em> 2001</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2011</a>). When 50% of individuals within family were sampled for validation, the correlations between the true breeding values coming from all the 165 clones (EBV1) and GEBV were 0.37 and 0.38 for allele frequency or regression method, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). However, when a larger number of individuals (sampling 90% of trees within family) were used for training, we observed greater correlations of GEBV with the EBV1; <em>i.e.</em>, 0.52−0.55. The last two rows in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> represent the correlations between EBV2 and GEBV. The EBV2 are predictions obtained for the validation set (no phenotypic data) from a traditional BLUP approach using the numerator relationship matrix. The correlation between GEBVa and estimated breeding values coming from only the cloned trees included in the validation set (EBV2) was 0.74 for 10% of sampled clones, and 0.69 for 50% of sampled clones. Similar high correlations were obtained between regression-based GBLUP (GEBVb) and EBV2 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). After comparing the four cross-validation methods used in this study, we found that predicting on 10% of the clones either sampled within family or at random was more accurate than predictions on 50% of the clones sampled for validation. This result could be due to a larger training model, where more trees were included to estimate the relationships between individuals.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462672/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462672/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Results of cross-validation</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Breeding values based on realized genomic relationships obtained from SNP markers were as accurate as predictions based on the traditional pedigree-based genetic evaluation, based on comparison of SE estimates. The average SE for all 165 cloned trees using traditional genetic evaluation based on pedigree was 1.14, whereas the average SE for breeding values estimated using genomic relationships was about 0.71. This finding suggests that the marker genotypes used in construction of genomic relationship matrices, by either the allele-frequency method or the regression method, effectively reconstruct the family relationships in this structured population. The accuracy of prediction is higher for cross-validation scenarios that use larger training populations (90% of clones used for training <em>vs.</em> 50%), suggesting that a much larger training population (perhaps 1500−2000 trees) would result in even more accurate predictions of breeding value. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Muir (2007)</a> reported that the accuracy of GEBV for traits of low heritability can exceed that of the traditional BLUP approach if the training population is large enough.</p></div><div id="sec-10" class="subsection"><h3>Mendelian sampling effect</h3><p id="p-27">Scatter plots between GEBVa and EBV2 demonstrate the ability of markers (GBLUP) to capture Mendelian segregation within families (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In the figure, we see full-sibs grouped for nine crosses. The traditional pedigree-based method for prediction of genetic merit of full-sibs with no phenotype (EBV2) produces the same prediction for every offspring within a full-sib family, the mid-parent breeding value (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Falconer and Mackay 1996</a>). In other words, when prediction is not based on phenotypic data, all full-sibs coming from a single cross have the same mid-parent breeding value, because Mendelian segregation is not modeled by the <strong>A</strong> matrix. However, when marker-derived genomic relationships are used for predictions, Mendelian segregation within full-sib families is modeled (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Capturing the Mendelian sampling effect will have a significant impact on the accuracy of forward selection in tree breeding programs. Seedling genetic tests with a single observation per progeny genotype have been more common in conifer breeding programs than clonally replicated trials, and forward selection based on phenotypes of low heritability is a significant challenge. The use of a realized genomic relationship matrix could prove to be a powerful combination for identification of the individual with the best breeding value within each full-sib family.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/909/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatter plots of GEBVa vs. EBV2 of individuals from different full-sib families used for cross-validation. The vertical axis is the genomic estimated breeding values based on allele frequency (GEBVa), and the horizontal axis is the breeding values based on pedigree derived A matrix (EBV2). As expected, without phenotype, the predicted breeding values (EBV2) of full-sibs are the same (mid-parent values). On the other hand we see segregation of full-sibs when GBLUP is used." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-369191679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scatter plots of GEBVa &lt;em&gt;vs.&lt;/em&gt; EBV2 of individuals from different full-sib families used for cross-validation. The vertical axis is the genomic estimated breeding values based on allele frequency (GEBVa), and the horizontal axis is the breeding values based on pedigree derived A matrix (EBV2). As expected, without phenotype, the predicted breeding values (EBV2) of full-sibs are the same (mid-parent values). On the other hand we see segregation of full-sibs when GBLUP is used.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/909/F2.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/5/909/F2.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/909/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/909/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462841" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Scatter plots of GEBVa <em>vs.</em> EBV2 of individuals from different full-sib families used for cross-validation. The vertical axis is the genomic estimated breeding values based on allele frequency (GEBVa), and the horizontal axis is the breeding values based on pedigree derived A matrix (EBV2). As expected, without phenotype, the predicted breeding values (EBV2) of full-sibs are the same (mid-parent values). On the other hand we see segregation of full-sibs when GBLUP is used.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>A <em>vs.</em> G matrices</h3><p id="p-29">When pedigree-based expected additive genetic covariances are used as in traditional genetic evaluation, the model assumes constant and categorical covariances between relatives. For example, full-sibs are expected to share 50% of alleles that are IBD, and thus the additive genetic covariance between full-sibs is assumed to always be 0.5 (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Falconer and Mackay 1996</a>). On the other hand, the realized genomic relationships among the same full-sibs can vary around a mean of 0.5. In this study the range of genetic covariance between full-sibs coming from one single full-sib family was 0.44−0.65. The realized relationship matrix indicates the specific relationship between each pair of individuals, estimated by marker tracking of the alleles they share that are IBD. Genomic relationships estimated from marker data provide more accurate estimates of genetic covariance between relatives, which in turn leads to more accurate predictions. We presume that the more accurate genetic covariances among relatives are essentially modeling which allele at a QTL was transmitted from the parent to the offspring, even though QTL are not explicitly modeled.</p><p id="p-30">Our results are in agreement with the suggestions of <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> (2007)</a>, in that we find molecular markers can capture additive genetic relationships even when linkage disequilibrium (LD) is low, and can generate GEBV with accuracy different from zero. Using simulations, they demonstrated that with a significant level of LD in the population, the accuracy would be expected to be greater than with low levels of LD. Furthermore, they showed that the simulated accuracies over generations declined as the LD decays with additional recombination. LD has been reported to be low in pine populations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brown <em>et al.</em> 2004</a>), suggesting that any LD contributing to the accuracy of GBLUP in our analysis is likely to have arisen due to sampling effects related to the small number of founding parents of our experimental population (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grattapaglia and Resende 2011</a>). This finding suggests that the potential advantage of GEBV needs to be estimated for each breeding population based on the contribution from LD to the accuracy of the predictions, and that phenotypic measurements and genotyping to model the genetic control of phenotypic variation may be required in every generation to maintain the predictive accuracy of genomic breeding models. Use of GBLUP methods to select individuals from the same generation as the training population, as modeled by our cross-validation studies, is likely to be more accurate than using GBLUP to predict the value of progeny of the training population. If LD arises through the sampling effect of small founding population size, recombination will decrease the LD from one generation to the next.</p></div><div id="sec-12" class="subsection"><h3>G matrices from regression <em>vs.</em> allele frequency</h3><p id="p-31">We used two methods for calculating genomic relationships (allele frequencies and regression). In general, they were comparable in the accuracy of the predicted genomic breeding values. The main advantage of the method based on allele frequencies is that the observed minor allele frequencies were included to scale the <strong>G</strong> matrix, with no consideration of inbreeding or selection. The markers used in this study represent a very small fraction of the genome, and probably explained a very small fraction of phenotypic variation. It would be possible to use the average allele frequencies from the base population, as provided by a larger set of families and parents included in previous mating designs. The genomic relationships should be estimated using the allele frequencies from the unselected base population. The regression based method employs the marker effects as dependent variable, and the mean homozygosity is adjusted by regressing the markers on <strong>A</strong> (dependent variable). This method was efficient for control of bias in the predictions, because it includes a matrix of residuals of the differences between true and expected fraction of alleles shared in common, as well as a measurement residual to account for the fact that the genotyped markers represent only a subset of the entire individual tree genome. When allele frequencies in the base population are different from 0.5, rare alleles contribute more to the genetic resemblance between individuals than common alleles (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Forni <em>et al.</em> 2011</a>). Those authors compared different methods to calculate <strong>G</strong> matrices and reported similar high correlations (~0.98) between methods, and small differences in the ranking of livestock individuals with different genomic matrices. They suggested that modifications to the <strong>G</strong> matrix could be important for different species or different populations. For example, the <strong>G</strong> matrix can be normalized to have average diagonal coefficients equal to 1 to assure compatibility with an <strong>A</strong> matrix, when either the average inbreeding or the number of generations are low, as is the case in most forest tree breeding populations.</p><p id="p-32">Based on minor allele frequency, a subset of markers can be explored to estimate the realized genomic relationship matrix. For example, marker loci should be chosen so that all parents are heterozygous for at least a few rare minor alleles. Selecting loci with low minor allele frequency can create problems with singularities in the genotype matrix, if there are many homozygous genotypes across the individuals. <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Forni <em>et al.</em> (2011)</a> gave a possible solution, using weighting of the genomic relationship matrix by the A matrix to eliminate singular matrices, if the number of loci is limited or two individuals have identical genotypes across all markers.</p><p id="p-33">If markers are assumed to contribute differently to the genetic variance, an alternative approach is the use of non-linear methods for prediction of marker effects (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">VanRaden 2008</a>). Bayesian methods are a common alternative, due to the fact that small estimated effects can be regressed toward zero; larger estimated effects can be regressed less, to account for a non-normal prior distribution. The different methods for generating <strong>G</strong> matrices, compared with Bayesian methods to estimate marker effects, resulted in a slightly greater accuracy of nonlinear models in some simulations (<a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">VanRaden 2008</a>). Other authors (<em>e.g.</em>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> 2007</a>) have reported that accuracies from Bayesian methods were comparable with the accuracy of Ridge regression if many different loci contribute equally to the simulated phenotypes, while Bayesian methods give better performance if a few loci account for much of the phenotypic variation.</p></div><div id="sec-13" class="subsection"><h3>Application of GBLUP in forest tree breeding</h3><p id="p-34">Genomic BLUP has some advantages over genome-wide predictions of breeding values using specialty software such as GS3, developed by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Legarra and Misztal (2008)</a>. The GBLUP procedure simply requires replacing the numerator relationship matrix with the realized genomic relationship matrix, so it is straightforward to model complex variance-covariance structures, such as genotype by environment interactions in plant breeding. There are established procedures and software, such as ASReml (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gilmour <em>et al.</em> 2009</a>), to run such models. Genomic BLUP is simply an analog of traditional genetic evaluation based on pedigree and phenotype with possibly a few additional factors in mixed models, and is simpler than solving large numbers of equations to model marker additive and dominance effects simultaneously (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Zapata-Valenzuela <em>et al.</em> 2012</a>).</p><p id="p-35">Another advantage of using a genomic relationship matrix over fitting thousands of markers simultaneously to estimate breeding values is that GBLUP requires a much smaller number of markers to construct realized genomic relationships. <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Grattapaglia and Resende (2011)</a> suggested that for forest trees a density of 10−20 markers per centimorgan would be necessary for genomic selection, depending on the degree of LD in the training and selection populations. Given the large genome size and relatively low population-wide LD of most forest trees, particularly conifers such as loblolly pine, hundreds of thousands of markers might be needed to produce accuracies similar to those obtained from classical evaluation. High density genotyping would require cost effective, repeatable genotyping platforms for routine application of genomic selection in forest trees. GBLUP is an appealing approach for forest trees to overcome high-density genotyping costs, and is an alternative to pedigree construction as suggested by <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">El-Kassaby <em>et al.</em> (2011)</a>.</p><p id="p-36">GBLUP is expected to play a major role in forest trees compared with pedigree reconstruction or genomic selection because it allows modeling the Mendelian segregation effect, it allows straightforward modeling of experimental design factors, and it requires a fraction of the number of markers required for genomic selection. The computation required is simple to implement. Predictions are less biased than those based on average relationship matrices (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Legarra and Misztal 2008</a>). It is also a way to generalize to complex models such as random regression or multi-trait analyses.</p><p id="p-37">To our knowledge there are no studies in forest trees on using genomic relationship matrices for predictions. The accuracy of genomic estimated breeding values based on genomic relationship matrices would be lower in cases of strong genotype by environment interactions, but the same would be true of pedigree-based BLUP also. Forest trees are typically progeny tested across diverse environments. The linkage phase of markers and beneficial alleles at trait loci could be different in environments with contrasting climatic and edaphic conditions, and prediction models developed in one environment might not have the same predictive power in a different environment. <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Resende <em>et al.</em> (2012)</a> suggested that a prediction model based on a training population grown in one environment had considerably lower accuracy in other environments, for a cloned loblolly pine experimental population.</p><p id="p-38">Two hypotheses were tested in this experiment: (1) using genomic relationships derived from SNP marker genotypes could be an efficient approach to increase the reliability of breeding value predictions of forest trees, and (2) two different methods of estimating realized genomic relationships from marker genotypes yield equivalent accuracy of breeding value predictions. We found that the predictions based on genomic relationships derived from markers (GBLUP) were as accurate as traditional genetic evaluation based on an average relationship matrix derived from the pedigree. Second, our results showed that genomic relationships generated either by the allele frequency method or the regression method did not differ in the accuracy of genomic estimated breeding values. Realized genomic relationships captured Mendelian segregation among full-sib individuals, even in the absence of phenotypic data, which was not the case with the traditional genetic evaluation. In practical terms, using a genomic relationship matrix is no more difficult than using the traditional numerator relationship matrix derived from a pedigree. The GBLUP model could use raw measurement data rather than deregressed breeding values as phenotypes; it incorporates fixed effects of common environments and genotype by environment (G×E) interactions; and it allows spatial analysis and other more complex variance−covariance structures. These characteristics will allow easy implementation of future applications of GBLUP using standard software available for linear mixed model analyses.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">We thank Plum Creek Timber Company, Inc. and CellFor, Inc. for the field measurement data. This work was supported by the Conifer Translational Genomics Network Coordinated Agricultural Project (USDA NRI #2007-02781 and AFRI #2009-01879), the North Carolina State University (NCSU) Cooperative Tree Improvement Program, and the Department of Forestry and Environmental Resources at NCSU.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005975/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005975/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-40"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-03-01"><span class="received-label">Received </span>March 1, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-03-27"><span class="accepted-label">Accepted </span>March 27, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Zapata-Valenzuela <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.1" data-doi="10.1111/j.1439-0388.2005.00532.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avendano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Prediction of accuracy of estimated Mendelian sampling terms.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DAvenda%25C3%25B1o%26rft.auinit1%253DS.%26rft.volume%253D122%26rft.issue%253D5%26rft.spage%253D302%26rft.epage%253D308%26rft.atitle%253DPrediction%2Bof%2Baccuracy%2Bof%2Bestimated%2BMendelian%2Bsampling%2Bterms.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2005.00532.x%26rft_id%253Dinfo%253Apmid%252F16191038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2005.00532.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16191038&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bettinger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clutter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pait</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Broad implications of southern United States pine clonal forestry on planning and management of forests.</span> <abbr class="cit-jnl-abbrev">Int. Rev.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">331</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BRev.%26rft.volume%253D11%26rft.spage%253D331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.3" data-doi="10.1073/pnas.0404231101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuntz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in loblolly pine.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15255</span>–<span class="cit-lpage">15260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404231101%26rft_id%253Dinfo%253Apmid%252F15477602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQyLzE1MjU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzkwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.4" data-doi="10.1534/genetics.111.128082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Imputation of missing genotypes from sparse to high density using long-range phasing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128082%26rft_id%253Dinfo%253Apmid%252F21705746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzkwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.5" data-doi="10.1534/genetics.110.115543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda</em> L., Pinaceae).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">969</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115543%26rft_id%253Dinfo%253Apmid%252F20439779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzkwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.6" data-doi="10.1017/S001667230900007X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstiburek</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Breeding without breeding.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DEl-Kassaby%26rft.auinit1%253DY.%2BA.%26rft.volume%253D91%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D120%26rft.atitle%253DBreeding%2Bwithout%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667230900007X%26rft_id%253Dinfo%253Apmid%252F19393127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667230900007X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19393127&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266659200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.7" data-doi="10.1371/journal.pone.0025737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cappa</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liewlaksaneeyanawin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibůrek</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Breeding without breeding: is a complete pedigree necessary for efficient breeding?</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e25737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEl-Kassaby%26rft.auinit1%253DY.%2BA.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25737%26rft.epage%253De25737%26rft.atitle%253DBreeding%2Bwithout%2Bbreeding%253A%2Bis%2Ba%2Bcomplete%2Bpedigree%2Bnecessary%2Bfor%2Befficient%2Bbreeding%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025737%26rft_id%253Dinfo%253Apmid%252F21991342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21991342&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.909.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-edition">Ed. 4</span>., <span class="cit-publ-name">Longman Group, Ltd.</span>, <span class="cit-publ-loc">Essex, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.9" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>:<span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.10" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.11" data-doi="10.1017/S1751731107392628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gengler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayeres</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szydlowski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A simple method to approximate gene content in large pedigree populations: application to the myostatin gene in dual-purpose Belgian Blue cattle.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DGengler%26rft.auinit1%253DN.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D28%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bto%2Bapproximate%2Bgene%2Bcontent%2Bin%2Blarge%2Bpedigree%2Bpopulations%253A%2Bapplication%2Bto%2Bthe%2Bmyostatin%2Bgene%2Bin%2Bdual-purpose%2BBelgian%2BBlue%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731107392628%26rft_id%253Dinfo%253Apmid%252F22444206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731107392628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22444206&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.909.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ASReml User Guide Release 3.0</em>.</span> <span class="cit-publ-name">VSN International Ltd.</span>, <span class="cit-publ-loc">Hemel Hempstead, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.13" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.909.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goebel</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-source"><em>Total and Bark Volume Tables for Small Diameter Loblolly</em>, <em>Shortleaf</em>, <em>and Virginia Pine in the Upper South Carolina Piedmont. Forest Research Series No. 9</em>.</span> <span class="cit-publ-name">Clemson University</span>, <span class="cit-publ-name">Clemson</span>, <span class="cit-publ-loc">SC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.15" data-doi="10.1007/s11295-010-0328-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in forest tree breeding.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0328-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0328-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000288656800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.16" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvOTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Use of relationships among sires to increase accuracy of sire evaluation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1731</span>–<span class="cit-lpage">1738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D58%26rft.issue%253D11%26rft.spage%253D1731%26rft.epage%253D1738%26rft.atitle%253DUse%2Bof%2BRelationships%2BAmong%2BSires%2Bto%2BIncrease%2BAccuracy%2Bof%2BSire%2BEvaluation%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1975AX14500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.18" data-doi="10.1007/s001220100627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambeth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Polymix breeding with parental analysis of progeny: an alternative to full-sib breeding and testing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">930</span>–<span class="cit-lpage">943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D930%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000172530400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.19" data-doi="10.3168/jds.2007-0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Technical note: computing strategies in genome-wide selection.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D360%26rft.epage%253D366%26rft.atitle%253DTechnical%2BNote%253A%2BComputing%2BStrategies%2Bin%2BGenome-Wide%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0403%26rft_id%253Dinfo%253Apmid%252F18096959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18096959&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251800200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.20" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.5.909.21" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzkwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.909.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Simauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.909.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKeand</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bridgwater</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A strategy for the third breeding cycle of loblolly pine in the Southeastern U.S.</span> <span class="cit-source">Silvae Genet.</span> <span class="cit-vol">47</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D47%26rft.spage%253D223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.24" data-doi="10.1534/genetics.110.116590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accurate prediction of genetic values for complex traits by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116590%26rft_id%253Dinfo%253Apmid%252F20308278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzkwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvOTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.909.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Linear Models for the Prediction of Animal Breeding Values</span>, <span class="cit-edition">Ed 2</span>. <span class="cit-publ-name">CAB International</span>, <span class="cit-publ-loc">Wallingford, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.27" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-3.5.909.28"><div class="cit-metadata unstructured">Resende, Jr., M. F. R., P. Munoz, J. J. Acosta, G. F. Peter, J. M. Davis <em>et al.</em>, 2012 Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments. New Phytologist. Available at: <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2011.03895.x/full">http://onlinelibrary.wiley.com/doi/10.1111/j.1469–8137.2011.03895.x/full</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.909.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">SAS Institute Inc</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">SAS 9.2 User’s Guide Statistics</span>. <span class="cit-publ-name">SAS Institute Inc.</span>, <span class="cit-publ-loc">Cary, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.30" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical note: derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Use of multiple genetic markers in prediction of breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVan%2BArendonk%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D137%26rft.issue%253D1%26rft.spage%253D319%26rft.epage%253D329%26rft.atitle%253DUse%2Bof%2Bmultiple%2Bgenetic%2Bmarkers%2Bin%2Bprediction%2Bof%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Apmid%252F8056319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzEvMzE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzkwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull. Bull.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull.%2BBull.%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.33" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Benefits from marker-assisted selection under an additive polygenic genetic model.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1747</span>–<span class="cit-lpage">1752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DVillanueva%26rft.auinit1%253DB.%26rft.volume%253D83%26rft.issue%253D8%26rft.spage%253D1747%26rft.epage%253D1752%26rft.atitle%253DBenefits%2Bfrom%2Bmarker-assisted%2Bselection%2Bunder%2Ban%2Badditive%2Bpolygenic%2Bgenetic%2Bmodel%26rft_id%253Dinfo%253Apmid%252F16024693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16024693&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235847700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.909.35" data-doi="10.1007/s11295-012-0516-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zapata-Valenzuela</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isik</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maltecca</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP markers trace familial linkages in a cloned population of <em>Pinus taeda</em>-prospects for genomic selection.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D1307%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-012-0516-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-012-0516-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/5/909.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/891" title="Population Structure in a Comprehensive Genomic Data Set on Human Microsatellite Variation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/917" title="A Genetic Screen for High Copy Number Suppressors of the Synthetic Lethality Between elg1Δ and srs2Δ in Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/5"><span>Volume 3 Issue 5, May 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/5.cover-source.jpg?itok=NJpwBPbV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/909" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xs8EelJQMY_QAx4ALe6SkwNWeJNeZZWIsqDQYhIb4Xs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/5/909" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/5/909" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/5/909" class="active" data-icon-position="" data-hide-link-title="0">Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459462" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HAaPEdWdhylaGblo1g2kT56mWdcVHy23nSTbBGAZ57c" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/909.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/5/909" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KYo9q4a4R0aiHH95vtCdkpbY2HOZSojfEwhJ8Bj1PKw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/5/909" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/909.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459462" id="node459462--2" data-pisa="ggg;3/5/909" data-pisa-master="ggg;g3.113.005975" data-apath="/ggg/3/5/909.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jaime Zapata-Valenzuela</span>, <span class="highwire-citation-author" data-delta="1">Ross W. Whetten</span>, <span class="highwire-citation-author" data-delta="2">David Neale</span>, <span class="highwire-citation-author" data-delta="3">Steve McKeand</span> and <span class="highwire-citation-author" data-delta="4">Fikret Isik</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">909-916;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005975 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459462--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Zapata-Valenzuela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BZapata-Valenzuela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zapata-Valenzuela%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BZapata-Valenzuela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ross W. Whetten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BW.%2BWhetten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whetten%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoss%2BW.%2BWhetten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Science, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steve McKeand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BMcKeand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKeand%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BMcKeand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fikret Isik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fikret%2BIsik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Isik%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFikret%2BIsik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fisik@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">fisik@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/5/909" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/5/909&amp;title=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/5/909&amp;title=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/5/909&amp;title=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/5/909&amp;text=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/5/909&amp;title=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/5/909&amp;t=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/5/909&amp;title=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/5/909&amp;title=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459462/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459462" id="node459462--3" data-pisa="ggg;3/5/909" data-pisa-master="ggg;g3.113.005975" data-apath="/ggg/3/5/909.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Estimated Breeding Values Using Genomic Relationship Matrices in a Cloned Population of Loblolly Pine</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jaime Zapata-Valenzuela</span>, <span class="highwire-citation-author" data-delta="1">Ross W. Whetten</span>, <span class="highwire-citation-author" data-delta="2">David Neale</span>, <span class="highwire-citation-author" data-delta="3">Steve McKeand</span> and <span class="highwire-citation-author" data-delta="4">Fikret Isik</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">909-916;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005975 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459462--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Zapata-Valenzuela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BZapata-Valenzuela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zapata-Valenzuela%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BZapata-Valenzuela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ross W. Whetten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BW.%2BWhetten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whetten%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoss%2BW.%2BWhetten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Science, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steve McKeand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BMcKeand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKeand%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BMcKeand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fikret Isik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry and Environmental Resources, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fikret%2BIsik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Isik%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFikret%2BIsik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fisik@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">fisik@ncsu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=909&amp;rft.epage=916&amp;rft.atitle=Genomic%20Estimated%20Breeding%20Values%20Using%20Genomic%20Relationship%20Matrices%20in%20a%20Cloned%20Population%20of%20Loblolly%20Pine&amp;rft.volume=3&amp;rft.issue=5&amp;rft.date=2013-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zapata-Valenzuela%2C+Jaime&amp;rft.au=Whetten%2C+Ross+W.&amp;rft.au=Neale%2C+David&amp;rft.au=McKeand%2C+Steve&amp;rft.au=Isik%2C+Fikret"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459462/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459462/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459462/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459462/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459462/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459462/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459462/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459462/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459462/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459462/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459462/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459462/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/5/909" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/5/909.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/5/909.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/5/909.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/5/909.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
