<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/5/851" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/5/851.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/5/851.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/5/851.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/5/851/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/5/851.atom" />
<meta name="HW.pisa" content="ggg;3/5/851" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1534/g3.113.005918" />
<meta name="DC.Date" content="2013-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Mace et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Advances in microscopy and fluorescent reporters have allowed us to detect the onset of gene expression on a cell-by-cell basis in a systematic fashion. This information, however, is often encoded in large repositories of images, and developing ways to extract this spatiotemporal expression data is a difficult problem that often uses complex domain-specific methods for each individual data set. We present a more unified approach that incorporates general previous information into a hierarchical probabilistic model to extract spatiotemporal gene expression from 4D confocal microscopy images of developing Caenorhabditis elegans embryos. This approach reduces the overall error rate of our automated lineage tracing pipeline by 3.8-fold, allowing us to routinely follow the C. elegans lineage to later stages of development, where individual neuronal subspecification becomes apparent. Unlike previous methods that often use custom approaches that are organism specific, our method uses generalized linear models and extensions of standard reversible jump Markov chain Monte Carlo methods that can be readily extended to other organisms for a variety of biological inference problems relating to cell fate specification. This modeling approach is flexible and provides tractable avenues for incorporating additional previous information into the model for similar difficult high-fidelity/low error tolerance image analysis problems for systematically applied genomic experiments." />
<meta name="DC.Contributor" content="Daniel L Mace" />
<meta name="DC.Contributor" content="Peter Weisdepp" />
<meta name="DC.Contributor" content="Louis Gevirtzman" />
<meta name="DC.Contributor" content="Thomas Boyle" />
<meta name="DC.Contributor" content="Robert H Waterston" />
<meta name="article:published_time" content="2013-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Advances in microscopy and fluorescent reporters have allowed us to detect the onset of gene expression on a cell-by-cell basis in a systematic fashion. This information, however, is often encoded in large repositories of images, and developing ways to extract this spatiotemporal expression data is a difficult problem that often uses complex domain-specific methods for each individual data set. We present a more unified approach that incorporates general previous information into a hierarchical probabilistic model to extract spatiotemporal gene expression from 4D confocal microscopy images of developing &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; embryos. This approach reduces the overall error rate of our automated lineage tracing pipeline by 3.8-fold, allowing us to routinely follow the &lt;i&gt;C. elegans&lt;/i&gt; lineage to later stages of development, where individual neuronal subspecification becomes apparent. Unlike previous methods that often use custom approaches that are organism specific, our method uses generalized linear models and extensions of standard reversible jump Markov chain Monte Carlo methods that can be readily extended to other organisms for a variety of biological inference problems relating to cell fate specification. This modeling approach is flexible and provides tractable avenues for incorporating additional previous information into the model for similar difficult high-fidelity/low error tolerance image analysis problems for systematically applied genomic experiments.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/05/01" />
<meta name="citation_mjid" content="ggg;3/5/851" />
<meta name="citation_id" content="3/5/851" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/5/851" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/5/851.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/5/851.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/5/851.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005918" />
<meta name="citation_pmid" content="23550142" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="851" />
<meta name="citation_lastpage" content="863" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel L Mace" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98103" />
<meta name="citation_author_email" content="dlm19@uw.edu" />
<meta name="citation_author" content="Peter Weisdepp" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98103" />
<meta name="citation_author" content="Louis Gevirtzman" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98103" />
<meta name="citation_author" content="Thomas Boyle" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98103" />
<meta name="citation_author" content="Robert H Waterston" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98103" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Stat.;citation_author=F. Al-Awadhi;citation_author=M. Hurn;citation_author=C. Jennison;citation_title=Three-dimensional Bayesian image analysis and confocal microscopy;citation_pages=29-46;citation_volume=38;citation_year=2011;citation_doi=10.1080/02664760903117747" />
<meta name="citation_reference" content="citation_title=NeuroImage;citation_journal_abbrev=Neuroimage;citation_author=J. Ashburner;citation_author=K. Friston;citation_title=Unified segmentation.;citation_pages=839-851;citation_volume=26;citation_year=2005;citation_issue=3;citation_pmid=15955494;citation_doi=10.1016/j.neuroimage.2005.02.018" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Bao;citation_author=J. I. Murray;citation_author=T. Boyle;citation_author=S. L. L. Ooi;citation_author=M. J. Sandel;citation_title=Automated cell lineage tracing in Caenorhabditis elegans;citation_volume=103;citation_year=2006;citation_pmid=16477039;citation_doi=10.1073/pnas.0511111103" />
<meta name="citation_reference" content="citation_journal_title=IEEE Comput. Graph. Appl.;citation_author=A. H. Barr;citation_title=Superquadrics and angle-preserving transformations;citation_pages=11-23;citation_volume=1;citation_year=1981;citation_doi=10.1109/MCG.1981.1673799" />
<meta name="citation_reference" content="citation_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=V. Bertrand;citation_author=O. Hobert;citation_title=Linking asymmetric cell division to the terminal differentiation program of postmitotic neurons in C. elegans.;citation_pages=563-575;citation_volume=16;citation_year=2009;citation_issue=4;citation_pmid=19386265;citation_doi=10.1016/j.devcel.2009.02.011" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=T. Boyle;citation_author=Z. Bao;citation_author=J. Murray;citation_author=C. Araya;citation_author=R. Waterston;citation_title=AceTree: a tool for visual analysis of Caenorhabditis elegans embryogenesis.;citation_pages=275-275;citation_volume=7;citation_year=2006;citation_pmid=16740163;citation_doi=10.1186/1471-2105-7-275" />
<meta name="citation_reference" content="citation_journal_title=Journal of Physical Chemistry. B, Condensed Matter, Materials, Surfaces, Interfaces and Biophysical;citation_journal_abbrev=Journal of Physical Chemistry. B, Condensed Matter, Materials, Surfaces, Interfaces and Biophysical;citation_author=X. Cheng;citation_author=G. Cui;citation_author=V. Hornak;citation_author=C. Simmerling;citation_title=Modified replica exchange simulation methods for local structure refinement.;citation_pages=8220-8230;citation_volume=109;citation_year=2005;citation_issue=16;citation_pmid=16851961" />
<meta name="citation_reference" content="citation_journal_abbrev=Brain Struct Funct;citation_author=C. Ewald;citation_author=C. Li;citation_title=Understanding the molecular basis of Alzheimer's disease using a Caenorhabditis elegans model system.;citation_pages=263-283;citation_volume=214;citation_year=2010;citation_issue=2-3;citation_pmid=20012092;citation_doi=10.1007/s00429-009-0235-3" />
<meta name="citation_reference" content="citation_journal_title=Computer Vision Pattern Recogni.;citation_author=L. Fei-Fei;citation_author=P. Perona;citation_title=A Bayesian hierarchical model for learning natural scene categories;citation_pages=524-531;citation_volume=2;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=C. A. Giurumescu;citation_author=S. Kang;citation_author=T. A. Planchon;citation_author=E. Betzig;citation_author=J. Bloomekatz;citation_title=Quantitative semi-automated analysis of morphogenesis with single-cell resolution in complex embryos;citation_volume=139;citation_year=2012;citation_pmid=23052905;citation_doi=10.1242/dev.086256" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_author=P. J. Green;citation_title=Reversible jump Markov chain Monte Carlo computation and Bayesian model determination;citation_volume=82;citation_year=1995;citation_doi=10.1093/biomet/82.4.711" />
<meta name="citation_reference" content="citation_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. L. Hendriks;citation_author=S. V. E. Keranen;citation_author=C. C. Fowlkes;citation_author=L. Simirenko;citation_author=G. H. Weber;citation_title=Three-dimensional morphology and gene expression in the Drosophila blastoderm at cellular resolution I: data acquisition pipeline.;citation_pages=R123-R123;citation_volume=7;citation_year=2006;citation_issue=12;citation_pmid=17184546;citation_doi=10.1186/gb-2006-7-12-r123" />
<meta name="citation_reference" content="citation_journal_title=Trends in neurosciences;citation_journal_abbrev=Trends Neurosci;citation_author=O. Hobert;citation_author=I. Carrera;citation_author=N. Stefanakis;citation_title=The molecular and gene regulatory signature of a neuron.;citation_pages=435-445;citation_volume=33;citation_year=2010;citation_issue=10;citation_pmid=20663572;citation_doi=10.1016/j.tins.2010.05.006" />
<meta name="citation_reference" content="citation_title=Applied Survival Analysis: Regression Modeling of Time to Event Data;citation_author=D. W. Hosmer;citation_author=S. Lemeshow;citation_author=S. May;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Syst. Man Cybern. C;citation_author=W. Hu;citation_author=T. Tan;citation_author=L. Wang;citation_author=S. Maybank;citation_title=A survey on visual surveillance of object motion and behaviors;citation_pages=334-352;citation_volume=34;citation_year=2004" />
<meta name="citation_reference" content="citation_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=R. Hunt-Newbury;citation_author=R. Viveiros;citation_author=R. Johnsen;citation_author=A. Mah;citation_author=D. Anastas;citation_title=High-throughput in vivo analysis of gene expression in Caenorhabditis elegans.;citation_pages=e237-e237;citation_volume=5;citation_year=2007;citation_issue=9;citation_pmid=17850180;citation_doi=10.1371/journal.pbio.0050237" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. J. Keller;citation_author=A. D. Schmidt;citation_author=J. Wittbrodt;citation_author=E. H. K. Stelzer;citation_title=Reconstruction of Zebrafish early embryonic development by scanned light sheet microscopy;citation_volume=322;citation_year=2008;citation_pmid=18845710;citation_doi=10.1126/science.1162493" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Kumar;citation_author=K. Jayaraman;citation_author=S. Panchanathan;citation_author=R. Gurunathan;citation_author=A. Marti-Subirana;citation_title=BEST: A Novel Computational Approach for Comparing Gene Expression Patterns From Early Stages of Drosophila melanogaster Development;citation_pages=2037-2047;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524369" />
<meta name="citation_reference" content="citation_journal_title=Pattern Recognit.;citation_author=C. Lacoste;citation_author=X. Descombes;citation_author=J. Zerubia;citation_title=Unsupervised line network extraction in remote sensing using a polyline process;citation_pages=1631-1641;citation_volume=43;citation_year=2010;citation_doi=10.1016/j.patcog.2009.11.003" />
<meta name="citation_reference" content="citation_title=Proceedings of the Eighteenth International Conference on Machine Learning, ICML ’01;citation_title=Conditional random fields: probabilistic models for segmenting and labeling sequence data,;citation_author=J. D. Lafferty;citation_author=A. McCallum;citation_author=F. C. N. Pereira;citation_pages=282-289;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Journal of neurochemistry;citation_journal_abbrev=J Neurochem;citation_author=M. Lakso;citation_author=S. Vartiainen;citation_author=A.-M. Moilanen;citation_author=J. Sirvio;citation_author=J. H. Thomas;citation_title=Dopaminergic neuronal loss and motor deficits in Caenorhabditis elegans overexpressing human {alpha}-synuclein;citation_pages=165-172;citation_volume=86;citation_year=2003;citation_issue=1;citation_pmid=12807436;citation_doi=10.1046/j.1471-4159.2003.01809.x" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Comput. Vis.;citation_author=B. Leibe;citation_author=A. Leonardis;citation_author=B. Schiele;citation_title=Robust object detection with interleaved categorization and segmentation;citation_pages=259-289;citation_volume=77;citation_year=2008;citation_doi=10.1007/s11263-007-0095-3" />
<meta name="citation_reference" content="citation_title=Computer Vision – ECCV 2006, volume 3954 of Lecture Notes in Computer Science;citation_title=Learning to combine bottom-up and top-down segmentation;citation_author=A. Levin;citation_author=Y. Weiss;citation_pages=581-594;citation_year=2006" />
<meta name="citation_reference" content="citation_title=PLoS Computational Biol.;citation_author=F. Long;citation_author=J. Zhou;citation_author=H. Peng;citation_title=Visualization and analysis of 3D microscopic images;citation_pages=1002519e;citation_volume=8;citation_year=2012;citation_doi=10.1371/journal.pcbi.1002519" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. L. Mace;citation_author=J.-Y. Lee;citation_author=R. W. Twigg;citation_author=J. Colinas;citation_author=P. N. Benfey;citation_title=Quantification of transcription factor expression from Arabidopsis images;citation_pages=e323-e331;citation_volume=22;citation_year=2006;citation_issue=14;citation_pmid=16873489;citation_doi=10.1093/bioinformatics/btl228" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. L. Mace;citation_author=N. Varnado;citation_author=W. Zhang;citation_author=E. Frise;citation_author=U. Ohler;citation_title=Extraction and comparison of gene expression patterns from 2D RNA in situ hybridization images;citation_volume=26;citation_year=2010;citation_pmid=19942587;citation_doi=10.1093/bioinformatics/btp658" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=D. Marr;citation_author=E. Hildreth;citation_title=Theory of Edge Detection;citation_pages=187-217;citation_volume=207;citation_year=1980;citation_issue=1167;citation_pmid=6102765;citation_doi=10.1098/rspb.1980.0020" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=R. McKay;citation_author=J. McKay;citation_author=L. Avery;citation_author=J. Graff;citation_title=C elegans: a model for exploring the genetics of fat storage.;citation_pages=131-142;citation_volume=4;citation_year=2003;citation_issue=1;citation_pmid=12530969;citation_doi=10.1016/S1534-5807(02)00411-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. I. Murray;citation_author=Z. Bao;citation_author=T. J. Boyle;citation_author=M. E. Boeck;citation_author=B. L. Mericle;citation_title=Automated analysis of embryonic gene expression with cellular resolution in C. elegans.;citation_pages=703-709;citation_volume=5;citation_year=2008;citation_issue=8;citation_pmid=18587405;citation_doi=10.1038/nmeth.1228" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. I. Murray;citation_author=T. J. Boyle;citation_author=E. Preston;citation_author=D. Vafeados;citation_author=B. Mericle;citation_title=Multidimensional regulation of gene expression in the C. elegans embryo;citation_volume=22;citation_year=2012;citation_pmid=22508763;citation_doi=10.1101/gr.131920.111" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=R. M. Neal;citation_title=Slice sampling;citation_pages=705-741;citation_volume=31;citation_year=2003;citation_doi=10.1214/aos/1056562461" />
<meta name="citation_reference" content="citation_journal_title=Aging cell;citation_journal_abbrev=Aging Cell;citation_author=S.-K. K. Park;citation_author=P. M. Tedesco;citation_author=T. E. Johnson;citation_title=Oxidative stress and longevity in Caenorhabditis elegans as mediated by SKN-1.;citation_pages=258-269;citation_volume=8;citation_year=2009;citation_issue=3;citation_pmid=19627265;citation_doi=10.1111/j.1474-9726.2009.00473.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Cell Biol.;citation_author=H. Peng;citation_author=F. Long;citation_author=J. Zhou;citation_author=G. Leung;citation_author=M. Eisen;citation_title=Automatic image analysis for gene expression patterns of fly embryos;citation_pages=S7;citation_volume=8;citation_year=2007;citation_pmid=17634097;citation_doi=10.1186/1471-2121-8-S1-S7" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Comput. Vis.;citation_author=P. Peursum;citation_author=S. Venkatesh;citation_author=G. West;citation_title=A study on smoothing for particle-filtered 3D human body tracking;citation_pages=53-74;citation_volume=87;citation_year=2010;citation_doi=10.1007/s11263-009-0205-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular neurobiology;citation_journal_abbrev=Mol Neurobiol;citation_author=D. Rebolledo;citation_author=A. Minniti;citation_author=P. Grez;citation_author=R. Fadic;citation_author=R. Kohn;citation_title=Inclusion body myositis: a view from the Caenorhabditis elegans muscle.;citation_pages=178-198;citation_volume=38;citation_year=2008;citation_issue=2;citation_pmid=18773311;citation_doi=10.1007/s12035-008-8041-0" />
<meta name="citation_reference" content="citation_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=A. Santella;citation_author=Z. Du;citation_author=S. Nowotschin;citation_author=A. K. Hadjantonakis;citation_author=Z. Bao;citation_title=A hybrid blob-slice model for accurate and efficient detection of fluorescence labeled nuclei in 3D.;citation_pages=580-580;citation_volume=11;citation_year=2010;citation_pmid=21114815;citation_doi=10.1186/1471-2105-11-580" />
<meta name="citation_reference" content="citation_journal_title=Mon. Weather Rev.;citation_author=C. Snyder;citation_author=T. Bengtsson;citation_author=P. Bickel;citation_author=J. Anderson;citation_title=Obstacles to high-dimensional particle filtering;citation_pages=4629-4640;citation_volume=136;citation_year=2008;citation_doi=10.1175/2008MWR2529.1" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W.C. Spencer;citation_author=G. Zeller;citation_author=J. D. Watson;citation_author=S. R. Henz;citation_author=K. L. Watkins;citation_title=A spatial and temporal map of c. elegans gene expression;citation_volume=21;citation_year=2011;citation_pmid=21177967;citation_doi=10.1101/gr.114595.110" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Grap. Stat.;citation_author=M. A. Suchard;citation_author=Q. Wang;citation_author=C. Chan;citation_author=J. Frelinger;citation_author=A. Cron;citation_title=Understanding GPU programming for statistical computation: studies in massively parallel massive mixtures;citation_pages=419-438;citation_volume=19;citation_year=2010;citation_doi=10.1198/jcgs.2010.10016" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. Sulston;citation_title=The embryonic cell lineage of the nematode Caenorhabditis elegans.;citation_pages=64-119;citation_volume=100;citation_year=1983;citation_issue=1;citation_pmid=6684600;citation_doi=10.1016/0012-1606(83)90201-4" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.;citation_author=D. Terzopoulos;citation_author=D. Metaxas;citation_title=Dynamic 3D models with local and global deformations: deformable superquadrics;citation_pages=703-714;citation_volume=13;citation_year=1991;citation_doi=10.1109/34.85659" />
<meta name="citation_reference" content="citation_title=Genome Biol.;citation_author=P. Tomancak;citation_author=A. Beaton;citation_author=R. Weiszmann;citation_author=E. Kwan;citation_author=S. Q. Shu;citation_title=SYSTEMATIC DETERMINATION OF PATTERNS OF GENE EXPRESSION DURING DROSOPHILA EMBRYOGENESIS;citation_pages=research0088.1 0088.14;citation_volume=3;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Comput. Vis.;citation_author=Z. Tu;citation_author=X. Chen;citation_author=A. L. Yuille;citation_author=S.-C. Zhu;citation_title=Image parsing: unifying segmentation, detection, and recognition;citation_pages=113-140;citation_volume=63;citation_year=2005;citation_doi=10.1007/s11263-005-6642-x" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=J. R. Vanfleteren;citation_author=Y. Van De Peer;citation_author=M. L. Blaxter;citation_author=S. A. R. Tweedie;citation_author=C. Trotman;citation_title=Molecular genealogy of some nematode taxa as based on cytochrome c and globin amino acid sequences.;citation_pages=92-101;citation_volume=3;citation_year=1994;citation_issue=2;citation_pmid=8075836;citation_doi=10.1006/mpev.1994.1012" />
<meta name="citation_reference" content="citation_journal_title=IEEE transactions on image processing :  a publication of the IEEE Signal Processing Society;citation_journal_abbrev=IEEE Trans Image Process;citation_author=C. Zanella;citation_author=M. Campana;citation_author=B. Rizzi;citation_author=C. Melani;citation_author=G. Sanguinetti;citation_title=Cells segmentation from 3-D confocal images of early zebrafish embryogenesis.;citation_pages=770-781;citation_volume=19;citation_year=2010;citation_issue=3;citation_pmid=19955038;citation_doi=10.1109/TIP.2009.2033629" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.;citation_author=T. Zhao;citation_author=R. Nevatia;citation_author=B. Wu;citation_title=Segmentation and tracking of multiple humans in crowded environments;citation_pages=1198-1211;citation_volume=30;citation_year=2008;citation_pmid=18550903;citation_doi=10.1109/TPAMI.2007.70770" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in Caenorhabditis elegans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="twitter:description" content="Advances in microscopy and fluorescent reporters have allowed us to detect the onset of gene expression on a cell-by-cell basis in a systematic fashion. This information, however, is often encoded in large repositories of images, and developing ways to extract this spatiotemporal expression data is a difficult problem that often uses complex domain-specific methods for each individual data set. We present a more unified approach that incorporates general previous information into a hierarchical probabilistic model to extract spatiotemporal gene expression from 4D confocal microscopy images of developing Caenorhabditis elegans embryos. This approach reduces the overall error rate of our automated lineage tracing pipeline by 3.8-fold, allowing us to routinely follow the C. elegans lineage to later stages of development, where individual neuronal subspecification becomes apparent. Unlike previous methods that often use custom approaches that are organism specific, our method uses generalized linear models and extensions of standard reversible jump Markov chain Monte Carlo methods that can be readily extended to other organisms for a variety of biological inference problems relating to cell fate specification. This modeling approach is flexible and provides tractable avenues for incorporating additional previous information into the model for similar difficult high-fidelity/low error tolerance image analysis problems for systematically applied genomic experiments." />
<meta name="og-title" property="og:title" content="A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/5/851" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Advances in microscopy and fluorescent reporters have allowed us to detect the onset of gene expression on a cell-by-cell basis in a systematic fashion. This information, however, is often encoded in large repositories of images, and developing ways to extract this spatiotemporal expression data is a difficult problem that often uses complex domain-specific methods for each individual data set. We present a more unified approach that incorporates general previous information into a hierarchical probabilistic model to extract spatiotemporal gene expression from 4D confocal microscopy images of developing Caenorhabditis elegans embryos. This approach reduces the overall error rate of our automated lineage tracing pipeline by 3.8-fold, allowing us to routinely follow the C. elegans lineage to later stages of development, where individual neuronal subspecification becomes apparent. Unlike previous methods that often use custom approaches that are organism specific, our method uses generalized linear models and extensions of standard reversible jump Markov chain Monte Carlo methods that can be readily extended to other organisms for a variety of biological inference problems relating to cell fate specification. This modeling approach is flexible and provides tractable avenues for incorporating additional previous information into the model for similar difficult high-fidelity/low error tolerance image analysis problems for systematically applied genomic experiments." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
    <title>A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in Caenorhabditis elegans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"REbNiXZZuSG8uqkXbdPfUTilaqSLJUCTXsuJNprPx-o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459460","apath":"\/ggg\/3\/5\/851.atom","pisa":"ggg;3\/5\/851","ac":{"\/ggg\/3\/5\/851.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/5\/851.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/5\/851"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/5\/851"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/5\/851":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JE6E20D680\\\\\\\/5mteHTiW+IuDHZATYs9x6yRZYp3NJn4uQ5+LT+R0FGTXhRzqQpUP9XXZo0QbrzXhhMhD1Ztt6EpvAHG1YIMyEldz+MBCiCDqbadzItWtYd5Mf8wjo7vjlf1uIZ2xCqbqV0i7N90ZIAccxlMVYRbmdAaKUiNIui9GbKShOj9ZFwaNhyCxMgd0yU0kxtggHT\\\\\\\/i9RDo3JOLErVldMM6iI2QLaMxnJzH7YcA1QRkpuLgHERCMmjoYbN3xco56Ojme5FYe6W3GSDfYmzKWBjX5+d0pGwKJa00LHSewa3ZL5AoNPf2n9HEoM1H+hHDEckUzHOCcKFhRP5c\\\\\\\/a+wjnmvjAT41J9JWA7l8VtT3PemjLVRVp66HOzN+QroRXxDyUh9JoEDjir5QvBJhaQ6JybNLKXlPM8o1uP4kOCq5VJH77Qwglmk8E7hvHCneLT8QQ3vczccWJOPNpnQsu51YTilNaZvX360zvTNUQ3AeJMHwXNj3gqPTCOffNLrhoqK\\\\\\\/TWTJcJowLobRti\\\\\\\/q78cNYC1Ae+nXfSUYl\\\\\\\/7AMn27PTK5iXRJF38AwI1DBXtlCDQpXu8UtoOo6br\\\\\\\/G5L6Lud\\\\\\\/f0Pn00UzS91mFNtQLzmy9IBZ2Y0KONNGFKJFcGpfkQkcWpIfBO+I5xUO7NwbWeCWQF406zRZ7ZhXJGP+q28TjsjTY26K4UO8IOGO6M3Pd4AB+w5dcGT1TVGlFDU+nEn2lhyQW4qRV5TCMBENwGNe32aWqIKSPERGJQvwanmGIdPFdVaFccWBrVA94HDVa8tQwiZnTWd+kntZcr0Z+blqSOJCu5IfShNXPWfPfKgbHXqchrlCSaC7QDWhtp4HXvQHiQMapAS\\\\\\\/iANGF9HgQL\\\\\\\/qo9+X7w85i9WBXjEnJ+IkhTxhqwzyqNYCFp+4hux3Hxe2eo32qAYFy292IQ1G1lUHPagaGBp9Goajh9owAzgHCmdz7fl2\\\\\\\/UKNBeU67zxIl5ORlsUyLLN12mKjE4uFnyZrIwsWX9A5BdfYy5SHjYa0GyRU3y8hLp53lp+pJxlY4yioYyEK8a0rPCTVIvNmZetYPiOmm0lTTxesxAc5\\\\\\\/ibyg20l2x5AN11\\\\\\\/QcReL83UPhW5BekqujisSz7XGIKVxu9A8UtEqSTyam7PUveRTbMdy+Ll70vMs2MvgUFFG3S+9Fmyxkut5caOJN5a5Pn9N1Gx9kI46\\\\\\\/\\\\\\\/Pu+oly\\\\\\\/rEfTr3MfQsyvjdMPsLoTSHRS1E1Jd0XxuFPm+cODRB0X\\\\\\\/22SKiByDW1dlj4R7ZdrIUcKF5\\\u0022,\\\u0022iv\\\u0022:\\\u0022yXYXyXl+akZIq\\\\\\\/aO9JPjKg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fa74736b6cad5e288627b1c1f0afb84fb6fee97b24579fcb3b5ea90d13a3984a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/CB3R26E8diIEqIMjIi4LcmoT0AjQ+zcVR\\\\\\\/g5zXmetxUU1DMFgkHSlbRoHYuXhUzFeRh95jj1QzH5EOEITxZ\\\\\\\/5PidPSskJ7o2ea2kLXEkhefFkUZG61zzgDgp9+uTv19CCtqMyMu5DnXNUqAXpUnGNTaAn2+aqouSotr1QePO7ugs0\\\\\\\/woY+r9bwq3k1KKRtxJIGRfQ5nmutrn1rwp1uO7Ppct4wqo+3NtKLcmnmTKzjmmEtPuG4YMuO8HsvbBnBYo7EzIxhqLlDvoa2qQs5wMiDFV2CajtdhRotgzsCCl6x1uSmDsWcXQxZttPC6r38ibgfHF7JJbLAP8a9V4cDRueVWxBoPZdIVhNjq1O19NtsFmN0Z5v5+MxJGHZAU\\\\\\\/kWSeX2uau3dnsegPQbHnWJfzs0UbSmaPgN3mJAJDuzCCoodwvjrYdYflfOcj1rL9syfmXXTDGI37ESJDDPE\\\\\\\/2+NQggRrt601vxEFUUMlwsHUS9zZ4\\\\\\\/WKvwMpTU08eTe3gNPjau6njnIHfT9thUODCjj7TuVNsWMs3Wn0\\\\\\\/RBBOzRTiHtyqD1vzQv89eYdaevSZoBsnaFXEIqbmlGXSzm7rx0Q13nrnGlSnPXFf4uThNnN\\\\\\\/vBaT82yxPaUmQHW9YUuoaRmWhkiURAVB8e110HZrSm872pYjGpg1Oq9SkBfHdsdDpI7fVojVLNc6UA+K5ZTvw2823Zo+lVxRh1vo1tZ+L+xIC\\\\\\\/GTDzs8OMrBWPWT6XFrgQASwmFIdx5lMwrgTfqdr5mtuSSirNMOsyGS\\\\\\\/J\\\\\\\/CFf4nYG8S9TCAlSiT7KGofZDVbfV1PnnDKV\\\\\\\/hnRjjg6\\\\\\\/v+3b7SLwXBn91BdMRc\\\\\\\/zuBcvuM35wB+7MZgzZC9bjTgqpjeLlj1JaR6m7LkKzyob2asnk2A3foeRlaA2f\\\\\\\/+8zpE0+uebq7KeUIfyDKpj1OdUw\\\\\\\/D3LS+iIspj8sX4Ajx8jTWatOJzxnVyR1GgF5PL1FfoVt4JQ0hTAf4BzsZri\\\\\\\/hs0V\\\\\\\/wg1b7p+r6INVpUsBaBNP7cjGXj5ZK8SmZumfK\\\\\\\/9SQ5ePOqRUv1RwxcwuYJtfzh4eyHCnkclD4PczGNP9Iud5aQjhJSZIbMBn8X\\\\\\\/fqzXLHXKfIs+89OW1sqrjlH2k6uy4ri8tLZ4decEIhsty0C0rhTOnPpV+A4\\\\\\\/ZvND1ebIUu8CFs3aoEXRptMciHuxwjNumKtgU3PT0WvJvZuHQzXiX8Y8XFRZSONppp72xuU937PwK339rRkjvlSUd\\\\\\\/sWBtiwInj6HcUp6nFz4VxxG2rjbDf9St05PnrS6JCqRvs1rzI2+3qZSthB5i76jZNucPHh3734PlviyBWh8QJpt1UvJpuYSm9P9BOzlaH2htdN27Z6Z8si\\\\\\\/jddv+BGwdKCz0YZNlmEbnam1Z0AsGVh6co6NecUJ9gzX48yVFuxev\\\\\\\/qssHO4EzZqvCcJ+gzX72TZBv5mhyS67h1wzAU9Uwj4ldXuFAX+EYa7xobyGMghXM9+jHhq\\\\\\\/R5eZyrAsiBBRntKGNol283VQ6DZbo0Bfna5AH4p2FSKpaWCi7SAzovh6iLK81S3QEBzAkhI0u+ErMXydH3P027ZTpHkTgHuYlFw\\\\\\\/\\\\\\\/Zqxiq0UJlpyOt+BiU\\\\\\\/HHJ+A3QQtlEFQxxd0XcM9GLvmgrow6O9ptTOH1STeLPcL+pXrBvbAPh8OiLFd9YSk7igv0qPDLc846+beForBtJlEj+m\\\\\\\/cT6aBUmLocAEyIV2wJE2FsfhCR72F5DVOHt+bAR14oP7\\\\\\\/lyFAbNmNHYDZJYWKt4lgk2mZuF0FwnQFzYh0SntFCuMn37mVVcFNz1KNrt26yVLVDJu1R\\\\\\\/sI6HZ6StyNeArToUuwIahyI+WtTxEKs2xBdYdpxFnpc8mM7zq78\\\\\\\/FZJnvI+hiCntgvizyKCXzikbQomc\\\\\\\/yCRfbfLQJzh7DiE63zMv5xWvrFxnxpFncTMUMjNe0DNPWlv2Ap9U36Jwx0PSYl4b3ChwgPK69J9+P++vuUIQPqM+dVTL2jRxgkk981Ej4UtKdS7P8zYsQ92Bi1BOH7QBEpHOKrhqxB8lNZXHZioJ3qg3Dc055ulobFVddZpELh0RPvDH7hM2ObZhE2KHSsJ6WK2NmhuqQZpXjH+1AP9YvzkfJeWvDgwnB3mt9DXWtJqX2xJY0MhqAdknN+NK2im1RfEP8mRIdqaoxTWNbaudEI8jSJ2otyzcLotzkrttCJnSKaTwyVgmJyqwObBw2U+An9A8ryiXWLO2bxajC3qbMJQw0e8vkbsxT2cBYhncqsMXF6do5LLMOR\\\\\\\/6kPyeh\\\\\\\/jQ0AwKnCkihiXfspnbGHlOD\\\\\\\/rkhF6Wn7g5iZEWGB4Ef\\\\\\\/BL\\\\\\\/hEwZiYtvLPllrKndY4DGO+N4R9BlDTQet2OUSmWZ6nagH7BeyECEApB\\\\\\\/IZpXoH56YLLy1DpV30rSD\\\\\\\/BQC\\\\\\\/SGMrHjRrReCWndxHErZ29\\\\\\\/kARg3\\\\\\\/OEBHBtWQF5u5zvKltYTB+zNcouXuYMDKMQGfo1+81yCMQdvMbLu\\\\\\\/DfEuOy67WBHk3CTUtOwLZUYHYNVN1I7bbrVLkkgXDh+F0xvmZqsvN6zbx4tMP2KIiOmI5i46jjhDx5QbvJgrZ\\\\\\\/DQr2bjbdJiassmIUCbuVQ+nbtCENucSqYSiufkwwr+44z2eDtog3u1NhxuU9HEV0k3bz6eypE9vo0XzlLsg7Rx3TCLJbhL6eJyvOpi6Q\\\\\\\/bePqC842tjyaiwNnO6IrjKfR9AAEebaJoPa40FixIcMa8N7rDSnP3iIhcF+ZD+R6QxeOXSynZ5sr7eO8JV90fddwQ6cwUIypXqkmjllodXDa6D6KIakIbciI8B3rKZzBc77c7NC4Mm4hCOn3CoVMgh61ztfsFDNvhF71Naw77IbTLf9jVpeE\\\\\\\/DTqixY+EcnjgCGim157aGTA1uZDbYQWITjxILI0EnMpQX7\\\\\\\/Rpe1\\\\\\\/CAfqgOFGOMxigU0vr3EnSljzqjatAHPKuR9tCjxA+ApRYBWZPJkl\\\\\\\/GVPhLgCMOYBzRRhN4T7TsfXzodJOCiU7F6Ht683+I14FwEnMh4+OseBQhX5ZXepjL5RBq6qXNHgClm+lvqH6f\\\\\\\/YysyY5DNYVcYy4U7Evq1ecpBP29yI5x3e9frJ8v+3Xoo9Hod8oplIJQUOneu73ei3nWQ0x7wTSxOibkufEQm9vjo+ow6p268REgR78NQZYE9A9r3YhiOKpN1jc0IZrcrDXaUSxab7ju1JWRrdKpfIlBp2M\\\\\\\/BsVhpvBFtozchhm+vYZsFrqqUa+pdOHWZTssq1e53qhnGeuyfErw\\\\\\\/D1oRJ7jdQa+YvKR0REppHvGP7\\\\\\\/hpvuCVv4UO289PWAu69HujbYXTm3il9NgU3pAUXWPGGlqxmHMy61l\\\\\\\/VG7mpfvf94MMieEx+vGBhQnEggM3Qr7NQe4Xm6ci2P2sCoB7t3f2AFD\\\\\\\/njN97EVG4yQNuy4YslIwBa3J0cDWHthDvzNKTEtVXPGJYToadRlfEKXgKL3Cg1tUYFg6uYY6QWBmlLMmb6mczBqt5Sjs4r2AwG17TGsQXU5DxkoOqkrsBRqE9stHM9euJvPmXSW+K9r9GChvFtNlnom5\\\\\\\/aR3P6d1K3bxRJVRAJ6XkCzSS0PRmybtPZ+gIfSuIGMBRBIAouftx4WB4hhO0pu85sK0jxxHT1QjY5uIYBT1srNojFuBD7IvalhbkC61dUmwuSzN4C6VZQ\\\\\\\/tLB7\\\\\\\/oHEYGJZxxf6LmUYstNEZNUBqgjeobIPjz8LTGHsVqay3pJBL0kUf9tOzpMPHSddLZgPG4\\\\\\\/YNQ2KTLBddD7k63Sbi9gLv5TkrPQs8TsBaPzpK8Ik+jkPS4WGA++l1R82OgJmaL95ObFvPko0HobuP2pMh1PTqVIbVJYMDQUfV0m+xMtcWZIOh1rYrj\\\\\\\/KeBNBd6nX\\\\\\\/gzCX9Iac5CVgkb5XCpf8ghWNS09xWbFyrUl3BXYQsC+chpoREXONFDBuAAV\\\\\\\/CjhyW1Qg8v1SDmGSCPugqHRHQVwiBI+9b\\\\\\\/g6OcwcqgNee4xu1OaH5VrVGxZvPRSnoNo0E+3ts8XVXYDleDfM6FNBgRmLieiU9H86ii02LK92iwj1tW+zeuFjw8vlbHpga7M16ism1w14wG3b454NvOV4LsF9fHGH\\\\\\\/CmSnsafedcoc0T6wb8oV3kGVJVs2aP5CLeGXxoC3RPFQZ8Aa4a806ISZ4vKuvxkQuinVmbWdxVQs4DXj6UWaEotRBOZZr6A9s0TUvlg\\\\\\\/WQGiALcXAkI\\\\\\\/Gh9C8C1GSTs01dn9p2L4CXUoLbpr6NNZIp6Odu5dLjtTLp+LWRdHulqURtsegfDHUlnRx3jSXmeCY09UPBsPHMsk+c1I\\\\\\\/WDwq1wsmUpxWtOh6IeK9Oqv1L8tq5z+ZcxZpuDErhWdrfjsxOrMPev0JXmrPEnp7GyRHdSsRvs3W3qurff4t3x+6hXgc\\\\\\\/9vyApgSgriA2\\\\\\\/NAYKp1spUb7RDef2d4trGUZ94EMHgalnqpvcHr10KMnUFHbvi9K81Xxrb8ANrOYo1pFVIlagudc30e2DoyEIvwHyuiDrUcNuMJOmIyOKBhfPD1YRQ2wZfTGdYl03wpMIo3\\\\\\\/KeMdrsiMmmyIeHeCakyTQ0xo1rySP4XkU4vcKEA3SlK4pJvmQVv7xRmRQdUFyum5MB1JiodtIZmyFCJcqkXAP8ybp8QpuBdUUEBDXsd9islK+BHCIwHeqUKmSgipL\\\\\\\/\\\\\\\/ZXs+NoH\\\\\\\/3ijt6j7TACRupHGuEp6PK7+7ET8+UACNIn9+3w21wpui8JbLZIPrvasv6Uw6EEmMPrlUZAEjW3CF9AdhshQSK0UhI5xq7yxpN9a66CnnRlKoK2oLnbS3Fg85PuYUXSzyRh9xwIQgkbkwZjaCQ8LXbRVshixC5vT96csXHCz6pEt2CMHND5dQFA6DRsZXl7PdZe3XpIZahvHt2I3CwWwL1+7KlOC4AHNa6PZbxwusdwa6iZ6oySoUdPPhzPE0QBTxy1ySpneuvSuLnPe2WbHn4CQ0ONuvn+QH9zbWnxc2+YhN1YB9px8ySw+cvGftSEqb5Ub\\\\\\\/UrpRjIxfsg1lb8\\\\\\\/j+cMag\\\\\\\/mRWfQUE\\\\\\\/IrhQ6rBGNfTkr+9Dud5F06mjDc89DQ7wkCPNOFDwLbfh9yAivxiQGTVd5b8Kh4AWRzaYV3l7grYpDg431Tgg\\\\\\\/ECF9GE\\\\\\\/tGWW7M8qkWHiBRGB9Y3hwPyXe52quRgTPyoHxarPNrWgbi26fyUqaVEUmLYZdJpNrsM5DO6LoLpFWNuCaHxHF2Wfe7jBj0IEXiSDAADCH\\\\\\\/niUUAWWHKTXzzAcf5W895S8ouPFnfEqcBDQbW4Hcn7R0oguamc0099hdgKFf8\\\\\\\/H84SwiOUxE2kHIVlTSASqq2hO9s8p5ozxOS+OHcLsEQ50WnZg3M1R7Q7mbwWOiTCU7cuik7uDIfiFaogvL6hDpaX3czT7HkrlNteyou0y9mZ6RDpwrh30EZnfYhHZpd6EwOGHsj7dU5hiGhLRUoDA2\\\\\\\/fMhMlfp0gIx8lzIJixZw0Z4aNtBxGKYXFBZksQEoHfrJxKfCfpkIV+JZU\\\\\\\/0NyFIXE3UR1jzXjQI5WsppkuvGCVcSW+As61\\\\\\\/TZni2xsg6skpUzfVbTGv8DEZfZ\\\\\\\/oRYMJy6wfoYDpq1xwO9ReqocD\\\\\\\/nBByJuseNkkbA9hTSL3krlurFKbSTbV1a0jxe3ZLT8zvI07jNdo2nc2u+DY2TkA\\\\\\\/FUUAtakmPMyTmf1XIgqH+D+bnsOgdLtPyD0PPNeVegWdVYyiweC\\\\\\\/o\\\\\\\/DKhuiQ6DuAouh0Z4VaUCC7uZB3bRZsgFEQOGbEmX53XIo02K15pEkq9qzy\\\\\\\/Eh7Jg791IkTUSxSCoH+AaBL4A+sIUwl5mjE9sGB59uKsBJjPzJX6\\\\\\\/R4ARaElwoD9uiJG1cCRqqbfvtQ0OJ+Tj64hLd2Yo22Wj3iLP9rXKFKRqkXcRdFKnOeUN\\\\\\\/6vCmBUyUj5cHcc7\\\\\\\/ZDhcHhkjD8unLKsXdVS1R\\\\\\\/BRecJy83W0\\\\\\\/ndR9z+c1ZJBLgZWKo\\\\\\\/yyXBOrvmSbznYma\\\\\\\/rN30zmy6b3\\\\\\\/55PsIf+Yuep7pXh5kPKwpx88EKZD1pyqMvNmcSAzLCHTja8TdATqrMz6osTMppD9P9+hE3CTlJCeyz3p+MeMgaV5es3DpbiVsvvkv6fmp3tA\\\\\\\/G+K92ZIJ8b\\\\\\\/\\\\\\\/AkT6tZvA\\\\\\\/5U\\\\\\\/qQermOIgNbYFtiy6oxeVa5ohILPy++UttLlsHFYCsM\\\\\\\/xTR4PZrY27kVFG2IXNXH1Lc8Hk4T4jgRgG4h1RDq7DmFlS7NERfrVFRIe5lJMijiCCGH1EZPQIyST\\\\\\\/nYnckk8viHb9vFTR10ilo7iBu\\\\\\\/QTxOK6xavy+mXGkwB81\\\\\\\/wm6FlbuA6Ki1z1C2leAjaDDemIn6pnv\\\\\\\/xA7rWNHq91VHTES1S7qXV1dsN6LrQlkpaHwpKOQ7O96zaDR\\\\\\\/FYxLD1yIEXdXXry31BDLaXHSUIQf0tf9zXOu3657V+KsX0i5XmoX6pyIifk7W0Nca3JVSP9xTwWRDUa\\\\\\\/Q\\\\\\\/o4\\\\\\\/zvCfKEgeqE12cFlfy5OS1eu23POLMcusRO5mbkwfub\\\\\\\/UM1cSqxuIyV\\\\\\\/Fn3AO+j8zIh0CPSjMTqtfw1KAd\\\\\\\/Qn7EMYLJxPNeTjrlBZymTANGptTYxO02exNihCnOIUH4x0aREwkTARqk7WcZ8OXmfoF7dkI6UBqZ2d7lLmOGFfktTjtqD2e5e+l0WcEQ1IJb1yxiqWEeTF93nPSQfb3fPsutM1\\\\\\\/tSutbfFjtYX4e6kpOdhI6dvwTQASgqxhSOGKnSZbntiho0bEY5aMQP6Jh0hoQQrDwA2A\\\\\\\/e0YBslPl+5MSXX7atvz53qwPQG+Bz+xVCgsYBPg2KlI5mIacoQtThmYmhJzWVPWpZ6y1TGa\\\\\\\/k8GROg0nx5r0gaAkO7xywgstU6yIYn1sDFrnywSU\\\\\\\/iOI5biOFJp4y4o8LS1SdWXPUSycEb4x3DDu3UAyXpPiGO\\\\\\\/QcvJzCd8OY52ptEySVh4LqMdQGaESDaIBaGdod3xfCIHE859mpspwunl9LXu9zAj03DaI7jCCdRowLx0V6miRZ2HamJfsCEadtaR7MPnBh2hk8hLOwz3oo1+gD8O2TPb7BUCDTlouEJmM1J0QjWyW\\\\\\\/KQhdvUM\\\\\\\/pn97tcAadaKDHxBoxP6NJc1Qee+gR+zv5gJgsQUKVV86NLXP3QHCVcARnInT0eZKq2D+Th8KOJZ6aJaAOpzxsfwsR1EQ0DQR9PKZyvbqnO6sHIyEyLYRDKObWobjFOp\\\\\\\/qqlZDpJLR6WHpdCbfqgozz84qaHEuv6URdHiZ764PcRb6MNbgVxSREkTKoSnpxlmmk7iioE2BVhRp\\\\\\\/HzaIlfkbTXYYMHscfgu+SKbcfrQKqn20cNt\\\\\\\/Avo4dTNdeoCFO3NgP+5jWwiR7DvhMuJeMpW23F5Uz1BMVQTfdZK2X8ZkwMmoe5cl3hZ6UvJsIgcO4XGvWdvkIKCEpfLyPn7OAhhNNZ2p321K+ynaxxM559NcRy+BBKEbZ3SAxeS0rAiErpaeJ6HfEVV8h6Z7+pHG3w0OyTrsti2ynT3bPxS8deJQprzV6OUKOTVDVRt6Kwjy+o\\\\\\\/8waXImJogCHHhI6yAxoQxjVhQNcwO7IuGalvX1RdVPwa3hodgdgM+jnKZbCPCZX7LhHo8O6fXYQz6EysdvCe9CzzpReJK71GpZETBJkogA3pzvTkXf2CPWNt5blhtMq0tLmlfYTwJwbA8cg70UnxbCt0y4yzL9nnmd9WBDeDinuhZap\\\\\\\/zmjz+wvb9fhNaLPLQ6kB7D65NoIGAzAPeHCEmXog5L5myxI9d7Lpuz4rP\\\\\\\/Fz01oCUptAVS6K1GSpCLRpa1Lp5L2RhU+cV2Zz48GPzCGIFbY0XVMYnYtVfp+eMZahYsA8PtPjfdPh2y2ETsYNkOdkP58x5CAMxqP0a0ofUCpypj3Du4KOFrcBZdiP32Wu9qm656o4JYxDpJqJMaEPNOtacBKwwqHaDk1\\\\\\\/VThSTbf9xjucre1xRFutgxEUeVrON1n66gOix6eApLAL3pgh0z9X4E714a46CT1RwL1pt37pG9j73Awh45Etrmkvr6\\\\\\\/bMDF\\\\\\\/fHGMsuwY7kt5KUphKwStJcEA5pRj8DuK7KtHCHIh8TqQ+H0wl\\\\\\\/Q7CrSuLMteaHbSk35WHbwIH4JCVgwCU6amBx6S9ti8EoClg1ImX4aF0jL7cuKQb4sxxfkck7yUijp\\\\\\\/1TpKr7NkgAzf8sfQ8ygnICpcpBXziEIHgi0H2BIFbwqyy0N8BiRRdmlgM7fCJ5rLaRqW+zbZkvszZI+MIV7ejTksN40zVYZdCSHkzcwOReR6TQdFGKfIo6WcFnF8mniMHdbHPhH8o1HTHcNj36UGtBhLr5OGMY5xaDjmDxY\\\\\\\/yZhVQkY8r1g1XCPW5solzZNPhWpX6YcyQ\\\\\\\/re5uor8C9+4IwL0maYLqec5MaImyDTxip3jyAvyJhaemSV72r\\\\\\\/1O1wU01Ti0MXV5Sdn3m7EpoIkB4s6+YLmE4vZYpsFBFIRp+tsKXvMpD\\\\\\\/TiuGIEXv27CUJrYYD95rKOs6jm2otNdWpzr72qSPrI5Xtt\\\\\\\/gf\\\\\\\/PH0EvZ1i1umrThtJBaLZRrUXDYSEMonwMYhVIZU+8BbavDW0jS1Q0QS1IF9T+lUxwo\\\\\\\/Hn3Tve6zZ3ui2JCaddW9TiyvGddQMHK5nanbJ5jJ8869FEHec+N+B5x\\\\\\\/mQXqiCSMNaZTxMjeFj0OlegMDM71pAyNOYDsUXhRtFd4MJA3sHXj2x3OoQ9f8JaM4pH47eNG1ayo2iIqNZhtzxyzLXqtleD0J03h37FZQeqalj9QuqMmH4hzKCMk338J\\\\\\\/tRgK2ARyJK+4jqYINN1mtGY56YsVvBhLKH32ev6Cqztrd5Y041f+8UQ3SkxeylmLQpj7H6Co1wTLcNDBMUm9Cff5iXoKvpRfE6OT\\\\\\\/q\\\\\\\/RMddZDZ9mjPUzXdhEwCmjet6fap3saLeV8Xv6IcOZ3wD\\\\\\\/nLojyh+IKb3y5HYFSf61YifP7qFxFXB6H0NfCJu2me2b\\\\\\\/VYGW0wl6RK0\\\\\\\/7vp3gdhB8he3sm8Awa1aXjcmtNUA3bvg7D7BFF7ZAAY4slNMuWeHOfTiGBF4UeNY6ZFUpOg1NFsbjWMVMIsCa8u5MjFfIUSgdcPm6AXJ7lD1ppwYN30mXiQ92gfjaH5LOGcypk2Vsr2hhbYVbplNll\\\\\\\/M+KjwPYt6GFn1C6t31x60zF2wVF+RSMUG3zn0nxpmn8YYVEO8o6NrRXRqgBJi7OE3IKRoIHQDtoh+gf2LAg1FFrH9EqD3Gxwmzmv5C8oPIcyDytXelPkDADctfTLvY7mriANn4dGbHrip4YqX1G54PqO0g\\\\\\\/FtAAWNRBupIDEUWHO0dxcV6xRsNBZ7OdbgXWowO9lI53l1a1Tr3FABDLQePsPCX5WsN2BdAyg5L+lUfqZhU8eE2BYXeyh2SiBzgiN5viROYs877NkSitog2ucfDzvF98MJ6j5QnWgTmFw8d1\\\\\\\/F2A\\\\\\\/ltkcw4jz7iSJo+5gHZsSkaJ5b9FDPor38IFN2I7rZmsfEuPi5XX8Bm5SfT99VYyrCUKcOq0kebH2ciCGon+VkBmTcXAaIxu6+v5cfsEIvFEBoC4gu9YdSm415L6vSjTvOLS9QqI6+diRJt1ZpXTVmJbJJeLixrBBW\\\\\\\/vO+iSRk9w5T12r6hqEhwBNdHRFB3Z\\\\\\\/eAEb\\\\\\\/uMQAxO2cQuFuPgN\\\\\\\/\\\\\\\/\\\\\\\/sYPXewLQ2g8M6vSUJhkLatIyFFXKzE+ZTVBdKbR4Cocd58vv0pmQtxsPxwWhwMDd0h\\\\\\\/3dtwbpspGtB8qvF+W5rKTouIKSRM1KVOpG7bG0TJtT++R+xWCSwoWlkba8cdWnh83LYJP5utw6ImyB3DejCcCC6zxwAl2tTGeeYkgeI7MDKN5kyppESDLTv2QoYni9y760tsab9eIfAlhjgrNvLRXa7z4CoiDraAZaoB1AVWrLuEEFySPu9HkmOHxosfUaTfTYowqpZE\\\\\\\/tSfprlVLKhR7nnnJn5GOePspbB2Zv4TsBzluVAgaOhRs2mo5BhbZz14di6c141h0QOA0IEQA6MSVa4QYvxFy45jqV5iQVqC8RQIYHGEwOqRRgZF5WX4P\\\\\\\/X1mTUFWR48WcN\\\\\\\/u+9ZKKS88tTeQlZuiYb5O5s8QMPP8W4QmovG7wStW2glpnimUzUxBEYoFnsJHw\\\\\\\/IW\\\\\\\/VCJUR5wvw\\\\\\\/8PvkWGtjrMOJfIGcTwyoZDg7YmtdbYofhZraPYh8qQJ2rI7TQBrFEpzEXRgCNY9+AiabpSBWKSzo+PPi2H6xisk6ItrNFQoih\\\\\\\/moemf0pJrDhHy6WBC6SBGgacZGYG67D1PSzyFQTkaqyL0DRpiXVL4smNdg9vrgairNAMSeBs4OwKyh1NAxWumJmmu3MM2rYx9tsB1j8xeiZlvxNXHEqME94\\\\\\\/6r2vVjOjFJFmABqLTPUPQ8H0fFUv0KehcIj8bQ9SyLzBq1EkLIXMyPKPE8dN+A4YMMleISzgNy5IuEKNW4vb5c3WBWbOxqG7euM7jG1P8RJY4mprq8SuytdNHNA81i9N4zMfbZhvVLdgaQ+z2ADh6Fc0CVpmaacmiJfqL09kjRlTok\\\\\\\/gCdk4Q+Dteo0Fpl515ljdKWziDpXybL3UoMfKOhrMLWHtZmY4GpKfo9zpX65A9feRPaEjjGyPCBPsmyDKJ0t8FoJmOS1siud+X7YgUI4h\\\\\\\/oKpYH\\\\\\\/QTMTx1kqaLOQwAvhYRBM2Pm5zEBa\\\\\\\/meeQeNdzfsSRsCTwQSsUl\\\\\\\/veBD6UO1Kh5gDystjgB4W0HbzcR+XO\\\\\\\/y5xPhmLyu75IRyXhSsIbtXkEpXyeLWA7ZNf6rl9wz7ER9khHqIu8fsdkdHl5Iv1bY810zZOmUx4JH5BykMOLTz9dzoUmYflcCf09YbxpYyIwORuTq+VpTygtjciLkkDHPphWbPZKjZf+o5tp0Mn+tnSP8MqdjQQTLvr\\\\\\\/sW5YBPqrbyOTIKyNul97PsqJxEVpXXaAgMlk926rbJjoo\\\\\\\/QPjSz0LdJg+ibuISuGP8QJpSF7BbO6f0S8e7ypHL\\\\\\\/vCmePydqSV6nDEe3q4goURyEEJI+V34U+aWJjf7dOTiuyzaikz39en1OFbYAS3MtSUdEDSPJuj76HyzMD6dsBK0xW6YskfaQDI9AWxJXAuNkFg39ol4EWAgw21YJb+5EwOU+hMCNEuv0ghA6ULRJ3WoS7NNxpWDPs++vViXuSjWEY1axrV3a9jLwojV5YLIOE3VzXJc860Kt3S\\\\\\\/ZdRR0Ibr3pl4ze8dbzX8YoDCz\\\\\\\/4NS4vTw1PzEkp4ELQtnSMf4sL+bG9XjEZJRgnjEWVdzpTJFL5gCzVW5XBq8DNm\\\\\\\/4afiHFh\\\\\\\/aVKe9SHFQ66hM5kcudHo7PntirHDOBi4bf0Y+1elpv8tdSc8kZ0QC7G1VHSqIKZiIy74DmY8PhDg0DaWEfWTrR+qwLJsqeuKsQ8Z53sA+BLvs7kuhN2WyTOt+r6o7aGurXlHBgRNHas+mOqawbwq4XcHhH\\\\\\\/kDQsOvyfqHXmpfgxn+4W6Q3TAHcoLI8rOmZdCAt2WIKabkU3k6HPYNuGlazniX\\\\\\\/Kume7A58HI9Lk6MWOevZNEomZRLEW3lcuIVKVGDBT7LiOkZ7S4dIKF78I+VTqcksoS0NgBM9PGUU3fLesFWiwsIwB8+kSoKv6Kfpixk+mQbitDt+Ebwx+YrcRXbvjADpceXosVF\\\\\\\/Ctg6vgpvaRaU5AHpk7JUuwW+jSMhtTNEiZYmJQLNaaTN4Bbu0qnXcdmOkLBZn5U41FujQXR4V9QI6HfR58+EDcdcHFYJpdy732bD3FXqLeX+Fr8qyZiPMeDEs7zVOIFIvs1NoEyvO5i9uIwJBpfB8JTwdcu5SzAJ4jn2ZD8Sp6Zb7Dhz4wC+Kxzpeqabs5j\\\\\\\/jfaXDRE+zb+IiHk8JlycNYzoi+B7nd3hZShxOM5szSkKxdR8mXej28XTnXlSFwfh1Ri6wXO29TcdFlr4z0KLAqk3bpdS2jgM0DKTwB3ONqyNdlMzQbzvq1ghlo\\\\\\\/gY4fPbCUggw1JelhutAvlMxL2EEd52a6iTPMouRLmSL5DkXYkfFfUqfHA0AJb3sn34V8VhF4LCSqh4CqtCgEDM+Z76eHODdYT9pFf17kSACunw4K4fMgTXhuAoOElIN3aZcPHmMc2fIDZDMh5TFZQaiQ1q2WFChsZrdigpbXO3uQkBf47HDkKqhavj8kEjpC7D9b8mnZbhjcQ7MwNHeRqp5Cg7HvvcUiPqI\\\\\\\/aAUBHASLlJE9QgVGpGzaqS+bv1TO1CB3oMEY6oJAj0USDONGKAuTYqOb6qnwV2beA6qHskBz9op4aDGJdGgupBcEgvhftGRsThDJcQVT1s3+zWDgl\\\\\\\/RUpdyH7JxwQaaBvTuo8m+b8GHXSDJIF5SWFE42qL433HC6kXy50KikZ3LOznI59sSWQZatNz6Vm6s8UELwi\\\\\\\/T\\\\\\\/9UNBds5N2ErX32Ied5ws+KjTMjJBZiBDuIA8T+PlYbWJSgjOI0qwN634gijn6x0eaKpCBIw7mz5teEhDN9k3EKS3NSYzaLTFDPWgOkmQ05IYT1hhQk1FkKRr+mrvaYLBtYue\\\\\\\/BPSs2x4CRCOdQid+5eWPp96XB9Ai7RiaH7HHHT3AH0NpuYunUCEt0rIoLfJFRF2N0547oJPlAhQiYRZ4wNwuglTWdBDEdVRjb7kziODs3xRccmjoqomwCUwcXbeVMiLyrk5x+XEhTYBx7fhd31FiESRVR7eRLWln2KR0MBcm3ip+8OnipIXbov5cwh7xBQn0oNVAmDF6MSw5QY45ICheoxXRkcAO9ysxoTpuavykmDi271zuOHh\\\\\\\/o79UiTZlQhbkfTfhFNXp+Y9lhMa9BKQ40gE9Fz4GkOHcAHOCderi8z6qlQhkHIz463cePQQ+Va7pa6BovHlBL09c\\\\\\\/GODIv4VRxn+eQj9qRKvp4Qf16gEts2mo1re6jMiqE8By4vqNFwD7bjpXQS5KdP6MLLbJdyrAOgHaR9+H8PCaTCWsPeiqF45geeKOHuNg8PVw6\\\\\\\/Uqu75FTzwJzefj6nRKMf9aQ2y9bQjBi\\\\\\\/Iaj7\\\\\\\/MC1ELB1sbAW8dOUEwDna\\\\\\\/7lRjF7o3V4c4XYiUSTmJyZjW8JpQ9Nf3oz3DkupDuAxutNcmhr74MIN8DYofMifeXEM56Q7M9lhK1LLkwhWPQfuTObVQCje+l3ulMwj84nl2hMfdSZy4Ppr7hS2MiGJxlux6APJAxpZxwcszeUNJg+rLEuraaY+cZ23z9yeH\\\\\\\/CPjLnbf9KlZJU0GzlozO\\\\\\\/MCzU4BiqmQBPMqom1mHPngYpSTSc\\\\\\\/TYfnzOvkEVlJ78GJXVTej+F+fX2T5j8xrH1Hp1Mb5Kz8S3s4zPepXpTlK7x3pt13FHpeSMhIP7m1obfzMC7+1LLE8YR5Gg6TIBnxMoxKTQZWzpXHohQ5xmAwW1ZdhUZexP2ff6EgmJJ4JVqLVfEIbjJxZvOFd+gGZ5FwUhWcKPzYumxrZLVaTfTGU7+3KPWgDsWG+Ui\\\\\\\/hKRKTYeup4VxAmgkBrtzbGHx7ty8RT3GEpoM3i9n8kPonQnqH+874+osW5SliLMfa\\\\\\\/U+0V99E6h2WTE8CLtrCzZDCbhQnEnfqedGm2MJeSjFvUmJmEqnWwgZUmYNRabuXYiDRv\\\\\\\/Uyzx0whJ9s6x3pjHLSc29UfnfxqqgYzbInAZTniFbrKMA0yMagMFa+7wqiUBR\\\\\\\/uAokw4ZCpFvwQuUlGsLzbfc3b60Tq6KB73bPoQOWlDbaoz3QMcQHrYxYXhrvRapzH+PFnvuOvUHNNjrTOsQS59mhQX9+x3wGdtlZ6uQ1rSFv2gZvuMQnCTULZlV4rNiZA+6VzZlIYvrtFvPrGSIBwh\\\\\\\/lBFaPbzgV09E0MRbKravoSIjcDklX09FklWZ7zVX3XWLVT55AnTh6lTE7bXIH2ifj3MOnn2B\\\\\\\/27F7mEqXNBnvF63Gfisx+IXOkH9BZaLFW7tk2wVo29V+Qo+rRfdviv8WraSxNkh65c7DAafJSRTi0aukNE40FWhoajHfML0hvNVflOdLJ3bcXGU7Eu2H+PQZ6OlxXpU6rCUmFCAWBN9o6LAoSONRTmyzEX49UvpZ\\\\\\\/O\\\\\\\/yCKJAZoDflnNdN+FUHKch9SDLA11WRONEo\\\\\\\/JMaquqJcuaDpIKrvpk13JiWfJz61AKIi\\\\\\\/B9xh6YPZNfj7\\\\\\\/rWrUr4HV15yb+9buVTDyIstrcvRUZzKjfo0JOxsBJcKKyiV7m6id4ASoGArQAETemJwLGmsNTAeV+gC\\\\\\\/mH4nI5RVZpS9Gi6FYPk5IJyi0AKsQfC+nlWjOvlPt3ZMZJdddvRh6vRjUQOv2Iq6cyoheDYAzH1sjU2SRwnujgmzpiAtptgOB5g5MwfUV3e2gcmfmqwopqPiei7awB8+mOA\\\\\\\/IeGGQ0e2NqZzmSH9+4bSrjY66EU93S3QPBX15Xr4bf4Q+rtdNA6LSqCrvYBJCtWiCRWXiMSGIY6DprLB\\\\\\\/ggXHJPqrzFEsqCMxZc9tDWA5W2VbIPVmpVWXYLtoMdHFyhuJDnC68gvDOfkc517lSKAbR+VRT2kuTotQQ93GKqXsPuhrvKE4F4KFL2dOlFt2Q5BLmamaCehhw0cJDXscKtCKY8p1v5R63CKlg56E9tcwoNy3Z5YeDvxfxU1V8L1xyckY7afzDU3+34aRlm56oxrs8OYE2I\\\\\\\/mCnh+IjgFZN1vptg+vOoIJ+4WlBWyRysCYFrJMmWxwd7XgBwDXBwrICwYBp5MVnDBOnlLNZ9\\\\\\\/pretOtDU0hXOlj4Qah\\\\\\\/Agmw3nFEXoa\\\\\\\/zHu0ys03Gt5Mcf+Re9BegQgDiJTvPiJ4wojwysfkBKU2UuU7k6yZ9Vqrul0l7tS1o7OUxHTHhqsd7yhgW35H88D4bLKVSQ9DCvobPW\\\\\\\/RlKbhb8b\\\\\\\/gEmp31AqPjDO\\\\\\\/IaacCgAhiFScCpCF0CoPQ5U3oudYEbpbazAOGhEu6rKKUAOED9MJyulKTqKSGH7a7ZYiAfdsm5Kg0rLtQl7IXAbUesMqTMrA\\\\\\\/FX09iNxza9ZA1RPpDbuMtonXKcuAe5UzyVsmhY9jhMaXp1hyMt\\\\\\\/WKyuXnhekmA7Q7iF325pzCp4Nzf7C8M2\\\\\\\/k4ziOHmLGis8XCQTysM8BIfxIuEF0jcKWpvU2mgcIMGjMdrO8AHpcSg9Sl3WtpAjGKzwqzwhLl56n+APZxNk21F8Q37cj2hETPzGIHjDFxl+ugD0lTa2RFkzCc+dzWhr1CVWmhMfSa6OS\\\\\\\/vm15\\\\\\\/qdwCEyS9msVXeRxI7rN8O853\\\\\\\/MmP6f3cRYJ5JucY476y\\\\\\\/imWQbaincnnd411AVf0ii5JOAqYbsmGQD0YTErHoAOsF7kiRvNFLNzlQ89i1Xft8fGL7cC1IL0vgfGgNR5NsFe8xt\\\\\\\/JCzULv77PnhkUALTSyeRqZwC7zAvo6Fzp9vYVL3cNC\\\\\\\/ikH2Rxb4WQ+tNhwk1WDs2qEqE+QgJOduzZ7PsHcbFXvMFe8ZaW0gdy02seD4IlAQeoDhr1MI5LpjDJIb5e\\\\\\\/QbxgomazBXgLLQ7BY\\\\\\\/3RjcETxg9sz6iV\\\\\\\/kUnf3iWDiqFhzwJEJpGAMhy7VcXPmSQhFIkOBJvIWdFJMiMw++zUUaXjkC+pahQGTpoKkboxPUQhKlh1g7Jm05uIOV27+OOdGDwPn4Y0AFbQ69NCotnzOpYVPUP2uGJ+wyDSeSKWjOl5ysubfIBWmoPfXzH9DsMEGGl+KYDO9v0mB22Ypb0bq1hHs2uEwb7fnFbsCN+X4jHpr4OlIKNFPjFlHS7PtExnlqdANb2dChAJca\\\\\\\/ryKoz0Mr+O5DZuATbYybv1ZbUup13MqzOLiFq2VlVTnNFZPFWNuAMczzhVqDTNxBZb7rcyIqk6IECp4VCt3YPxhYZRpxDcoEoZ1aHj+XT\\\\\\\/4ups5uXcmoI3\\\\\\\/w0+O2eDzpGUk7tUNXE\\\\\\\/VAjmd\\\\\\\/3hFJNwgIaps7BQO2ERMXyi6NTjNw2qX+ZvuTiTSXwGNSzaU2O8KYWDRrev1EGPe2DbG21NY5SERLaqIZkzdm9gT5hRYhhmxMqasj1Pgyi\\\\\\\/rkJ0981NZ2gmxiM91RbjeQKQ2+Ny3t5WPoEfRxLwkc9Q5+LYjPcLjCkHHuDR1uIwu7mety\\\\\\\/4BMmaqJ42sWak5ZFLX4gxiFQ44CxVF2sSSCrN2xdHlplBxoWEduv5sTvvJloRYLDrSRSxejV71Ujiy0U2w6Zqv5G4jb97jSwj0RYa1RR6KFW2J4Aa33fxNKc8AXYUu8nB8WRc\\\\\\\/xPTmhdbtVWJwfo7NcLDHZmEjdICLEo35L14Zf7yovPAY0p8QPcrx4NbMUMOysOeHDQse\\\\\\\/hndExehn6yXb9TZ7wTbeAjenq4lsBE5h92L8i5tz6HtUeFvyPmQWbKh5R1SI0AweIB6\\\\\\\/ivnfAXp9z2uCPD90gSCqXYpD\\\\\\\/eo6PF4udxgHFww2zBt5eCMC2QThy+qUyf7mTNstt578FyHzeNoDIWqgHR2kxf4iQif62wcH8lBtIhgjMNj97UsZnmBg+F4hQKsbw+DChIn7AXvZGV9N3Fub23pOJb\\\\\\\/iS4e1j3qODDoeKGx3hysvAmmp7+DwpM005foHR0YtMmXZInVM2OH\\\\\\\/jBxBv42zPPSbSYxzS0Fm2w4+dYQ3d56X13q9OBo8SzyCTBrREwrafXJuZN2KEiXOVOm5UZz8Zg\\\\\\\/bemIPJFeRAHcrEuFlGqFEry+97GMtDdO7xHT0a8sHz7Ws8cyICxg1NBWSOkhIPC0viMWmSz4rZNJzYKW4IQ7wuIOPzesjLDNZTa4cik0HNzSMwNjf+TS7z1+TxeJDp6\\\\\\\/6PaSQSqUSuGPzsjsgNwDwl2\\\\\\\/+2NHHesVxGctgIhjwfxYUW1ZjymBVMGCiWYwppZKrFDk48h5PJHqQeuvFp0CYDeSRCl3bvNmYdYLLBdPloOgHCEc0gC+EvJuNGUy1hUAP6BuAcL0ctR1bq\\\\\\\/XeQhTMjUBI8vwWAS2qRrhyDOVFGQVsJCGv9W0z6TczHCOK3e0wAyiMf3oTldGTp\\\\\\\/6ppdoU8a03lhd3fFKdhx\\\\\\\/CrYcwl2CB3ZjKH5dxduN58HuX5WFqqaa1SyzV7cDrnUI2kD3ctN2byz5qU\\\\\\\/UfDOcgzbmkXq6BRFvZ1TdHDXAws5nMtl1LYtFgKyUIxLH0Tuz1GWQQUYfuLcb\\\\\\\/cTALYAYpRwmT5H\\\\\\\/AdH3pVTvo8pzHH3jHw9yVCExP1UfBAqwO\\\\\\\/TDFgdXBWsPCopQbeJJQ75FBJ9\\\\\\\/xliGOnC3ID1+0d1xR+UqFlnFGFjPf+3WouuqHmenSHNsYCWPtZmDObWWXJXpP0LcuNopYxG5uRz2pqXAt9tf+6a\\\\\\\/Tb7zga7DF+Asj4WknH4RIqNdwld1kzYIWMj6SOO9HRiUQKFbzHgICPg+D0Cz5CvTHu4AE7FlEk05pYQOsjQGiydWkELElsLqDsIOdVKLGVoFy03NAJSyoIzZhdm8dJgVXc3pJE1btA8Xvs4Cpv98L9V9S0DRukZ2qoG3Jt6K6EjDsNRJAhp6esI1bhfvFi7wZWfUUYc9ZNKHJi+vyPmJwURR1XlDREMqTR2eA\\\\\\\/dV3ApxMK2hFFX77NdP08ONg6mf1eFzyJqIlaTUveD2C06gRWT6MKotyJJQwNnY2Ov9B5\\\\\\\/tBzx+tILB2civ4+XlIhFiqXmBzrww8zRS\\\\\\\/gkjDS+nrIyI6uQVfj+1iijessotq7Y2ISr1eqDW4EjBW1E1\\\\\\\/CyaaXAlMyfaDYItCGs8XzYaiKZc9n4lrUTQPaOHA1HAwoNSCncBT2G57DhAglLquBEmJhG3v37RhrAPFXr6pIpo1jclakJK46akKUttu28SU7TAvtpwPSWpPcEeTBvKW\\\\\\\/OY0Gaeqg0PUyfQKcIvhRnreRPKis3jHcprWD9+U2YCaSq+8Y8jX6qadsdWEPFp\\\\\\\/BgCDiSlEhYHB98J6FHLoVZhQNCd+5XDfrGKDKPJ7r3Y+vDMs+mujS+jjl\\\\\\\/yhkwD2wCNOFv1ib3TCHJ78484aF511mxwphZgzIF6ZueFq45eLTlq670Wi3UjvXVVPCWSWtGBJuk5zz1WQnu9Vwzck71eh4I8gyQeFEDLxq4HciMeWnvnZL8GZInFOASZPrbgNkXywD0C15Qs8+vTuon9XerqwEFE1QgFEwyMvZIgFCsNBZyb51JOh7MmCZ1TSK5bhScE03CcrGui246tsNN+VB+jXo8Loh5ttTAdDKYCuFUyq9lLFBUUhyMEcGZGDFG6VLYhEXVCM9hMjCpJ0qGu\\\\\\\/t09kNcfowRw6SeBLjuH47m52sIfG83fulDyIM77TDphPoNR+ZiE1VFhcYsMgT\\\\\\\/KHk9uQ4eRUJlXDaHgFz8Nqi0zUQ6RLkKkATV9wSLBFJrxrdWwF7Dan+SxSbUdN+hMiDXOm35EeaQealE2IJcvy+v5Di6N3A+WCsyfqf7uZYM07rfZpM3a95C3IosFGuQMrAiiH7Lm1NzdWt+WeVpXk7HIutApNrhSuSsktHDwfzl4znG2wlW4cRk7aNDc\\\\\\\/ZZNpGc5+FS7Qt9mecB56W14v\\\\\\\/MLs1XHPCoTGmtzD1BT+W0moMcA7LKQnfjr6oucb1ZgRquTnz0TyqnHZfNyVHrt+KVfYRX4KJ6nYA8\\\\\\\/YHRT+99yLCuYFGBW\\\\\\\/678\\\\\\\/AMxXY5I0XLEzhk1BeE1Y99dkUv3L5SlGlys3z9aIhH8o4TmYHBzgk03k59pKgLYZjxXszToN82cXFNVWD3kcZKHKJrglduIW0pMi\\\\\\\/J\\\\\\\/xyjJcY7PuLIiwITa0Ee11NBVo\\\\\\\/403fP6Wa4REmL9\\\\\\\/xIwn42HsMXxj3cwrGAjBJWiPgOmKjTGBBhVFf+XtmCD4sKUoUB1a54rjLs9MCS+FkzpkhALM8rRXdE3suN2z8ClhuOdwMycleaCLHm9+ttcKfldRErzRlufXY8wezRcCXLO62rnLo8TqiwsfUvq\\\\\\\/2LI8neZ0wAtCerHP\\\\\\\/UsCVdSrWVCJdDQQ9noJkc\\\\\\\/YM04EOaqt3UQXs\\\\\\\/MYqYbjj97FNRaPxt2ej86ThDCYnkA02F4Cby9YFILTxAygqWplG3IA3sAwht3UByBs5tivcTLHIvvjB30twTc2H36UgAQdOOcKGC7YDZ+ZcHMtYyZ4f+vdrhgdaOfYyoUSwN77lXYuIWE0V\\\\\\\/Y4rj71BFWPu2DtWNAL9QHoIW45HlE7NW9pPF86BSLq7y8hu61BCmItYJFVypAMALYvTKK03CvBa6fDQO\\\\\\\/IP56CFeBv1NzG9vZeh10UARSb25EVv94QH7TCv+fuP5L7C5pJXumnGYsn7apjW11a9wK\\\\\\\/fFn9mSp50EbZikKlto+iKQmNrfh7Mjw+of1C\\\\\\\/OJyE\\\\\\\/oQU93WNMWy9K2LtF1jBJ\\\\\\\/kvM9nnQYo3qA+K4ZUeuXM9OGQI+1Km4avTbwnrYFtKaao+0tRHbWZn59W\\\\\\\/fFh6WTXqQtn3nOeaywui6BnNZ3lg7gog+ACxPAXLJ5KiL+JGNfdFgCZN7y5l6qmD8SLsURkTnld4Rfe2lghGJDSFJ8Pm0clnMRoW+vmvZ4LTCETSc8wYO9mUBk3XpRs2DqsPUUi0vLgG3QItst4sODnKY1UXNbPmVMR\\\\\\\/q91Jp+NRuBmjxYLKNtNV9uKFMvKwR8VRw+TBwQb+DdJ2dwgS3mlc7OdIWfDxEZiwWl5\\\\\\\/kTllTQp0Ga8Lhr4mQcYs7V46BzBWLwhvFdKT\\\\\\\/ouhWZx6nofXoNsd4I2cETiIiVm+X4BEYTpM4u9\\\\\\\/aEKDhGga3hfOiRv97sMgBQg6wB+eyMXdYyy6HbZT6RivMmvtVNzU716phh6TRjQ3oesG8FxeMbVA94nUGWGI2vvOtTcOTHgRLcxWCUCJyHzQvPvonYqdIFqqSqu+o\\\\\\\/745Mt0PEOWdqaBrUX\\\\\\\/W0MEV199U7HS2S4XKTQ4ZLCNHVQtwlCJpG51miZVitBd7rYIFH4bD8jpc675Hotdqn\\\\\\\/QafP0TDKCKWda+sUdpBJYZk8LP9Xks0zaBxmrn0gITn5xksqjRnfkUQP+0fF7iOKVFbpsU6QataLn+XHCLBvt8AfSVm0Rzf0Uv6hbi1PC8qoXYj8fDfIjHF0HG1gfV8kXQ8aLVCNwjuI\\\\\\\/UYnADwINOo9HTc8xvrDz44sap4pftVKd7JlFn6KoFchG1DA7J4EOhS7zkgWQCuBCrCGfS2Ate1DKio9ynVk61EjMoHf\\\\\\\/4JHdshxssx61ZmkKCufDi4f2S5kerVC14wQktLlxzfZSwRMwQ4fGDySDz8CQ2nNkcxOzpBpGnGgEEO3LS05YQnb94E5jSHPKBbRVvqa5fWGY9+52cXsKomh8sKIt7NqQvfeztmocDge2BE++uzygdzNo0epoXQVlS5f0UVB4ArmeodOcAAkq5IWxZha\\\\\\\/2VuYMUqUYf4VtYhAww2UrxwOLAsPVWn0JQZKFlqJaz1AIPO2FqiQi6Xg6yVzIHOnZf\\\\\\\/gRSQv5hiqXInTGFmDSkxoQBLNTGJQhNTXyjb29C3xXGLrbJB5GR6d478DdgBKEQ4bg6wBj0oAV+9yepwuT1SKhHXdN3Ms364j5ZuFaErIfieKFyTX+euAznisxkK8H6haAnfk4mRsxi7++sB2xCEO9fqRTpfxdM1PU7bxSgenhmZ++jn\\\\\\\/NTrPbZVoMm2ksSf+dzZVGt5gVZS3gvVy9JWaj4XJPoCNYJn4vZ5nC0nDTyvOCdHyE\\\\\\\/KSKPZvo3hWTrCjiKgL7A1gii3xSqH2sBGiAf2P2dCtgcmtuj1a1zTGDofTdgRh39\\\\\\\/oUI91jv+WWAKbW79Gk9qp1QvFZq9FL2CRI2rUriCQpma0kvJhXkOGtghOYo4hHTmC6IdUGl3qvLgaU+K8qA7jGVL2z7tWk2UjYR0RoyeEU\\\\\\\/A\\\\\\\/uKTHAJtrXYEESgD\\\\\\\/ushFZqO1zeskTd1qloh+hYHYGUCPbSUueVj+CkWc\\\\\\\/PoT4X2l8HSptzB+V85u8DwaSgxtGsExfPzgFviSfazXxHWKyT+cWduG4\\\\\\\/zqE+L01GLAIMb3AHp+g2fqBkzpekpPveGO\\\\\\\/PooWKTS+qjFO5s22LBds9refpjLOdIQJYszoDzwtcD1cSsw7Epsuj1UwUhNWSwFgHPUUmIdxhGhA6DdCrJuqrjCLXmXAiebZlOTbx\\\\\\\/uX4K0zBupzrX17GMKU\\\\\\\/nRc\\\\\\\/XIt5yW8o1E6alOr4ZsvGRjIU35iIRnmTg9al4+EG3lf1NVAL+yvuWaB0bdtVNj57rx4C\\\\\\\/uTd\\\\\\\/WGEkmyS1I3xxWSYFkB3+ZIH\\\\\\\/LuA3ocsqy1OT1oJDZl+AlDEOkcAI7MiGnb5wKdN3sJarY+MPs1XzIyYcE5UcoAOU+3IZZ\\\\\\\/zy0znaLoAIQNR5P200phYZ6ZMpWczh0+tv8SO4fotj0IQ6KtL2HaUEhUaXx8\\\\\\\/IVyBQ+80eJgpgSkvwEEK6BXTyIs3RVQASZcnSEII\\\\\\\/zpHxY1cwDBFb8qyEoGnO5zziOr3fLvKlMKsW75YIsinTqfs6UM7sQR3\\\\\\\/ygpt0azQlau0W9l6OJ8DssWtlQS8BnUnILL3zIIuxRWncxiFoU+EwGea0RBFzDSyJN21yGUiUN\\\\\\\/9HkbFIdc61pFQbO7wXcVeyIH2v1NwsInbE05U9D6uo4TrA\\\\\\\/+1JfzypPwqqL4KGi8Wgr\\\\\\\/fhowsymFKTaYCt85VEXLmm6qpot5E87W9NzaI+DLijjtu4IM7ISwO\\\\\\\/vzNeL9s3I\\\\\\\/P1CUdLjDv0\\\\\\\/MFe1wmmAiE655SRJlHxLzUD1US6xnGV624mhaOELhKv2TisvAdIuJVB2t1wnuH1Gfdghly2QuJmw+AaENQTNedU\\\\\\\/XULvkM5bK5QxsYlu8Nrjf2cPhVU2gsIIJ7in\\\\\\\/uS4xt4hWps7MZaqr6xJoatKa0VpAY3GMNHOLBqOzFOthMMsxB8MSCkdO\\\\\\\/8RaCkJj2GEUHwwZRbBi84M0rWD8j+V1NAYHoQzH1GzK+QdbFT+G4IPDAUvSVRXcYze6YUoVNvG\\\\\\\/6NQVqR+1fGK18c8kOe78l+NBXKBwKYCqNkIY0AXeikPbWFJlwpZdRA+9MZ23zxtyviyjQtA1agDAoRcpwZUTCZ+spJaJCblnljN6Wk9drShrzkafcQ2u9SfjJWqkkwlaTdZdsuTdZmYplQ6t5mJPRbKb+y7CDP\\\\\\\/iWBfquM5bI6LkSbIbo4t40T62dCw17Z+UK\\\\\\\/ER8QRoRLGWeo3KGmRlSnxyH4Lx\\\\\\\/r9mfkj6nW60bC3\\\\\\\/c+9sD3YB44sRrU\\\\\\\/MoGCWu2j011K2KwF0aGLTYzpdQVn3Kc1Z4yyTrhQd2EiWbLFk1wG3Uvi6ZmCgAv8cVPwPFLgWmAleYVw7Kd0NjjaXotSAU7obN\\\\\\\/ZNe1iiU5Xr9U3MI9SWuXxi5R7E0zschBG\\\\\\\/my4jKn8N89Gb4qOG1G68ne9smY6mgvucKlLfemVVGZK6sb8anSM13IIskJfjoUIUPivoB2OqPiJsY9PowoMkqBpR60SwMQgkRO1PZUHWdZtl1zV9pniR0bNb9sNPtcpI4LX451yxGN9hCupykojhoHGfETbZBFbt+2WNsaFv+jcx+bfhf+ZhmKow72X+RFF4GFAoSS3UXz8uhHiZRkTefGKe9cgK4IQVTqo9Cm4Sy4s4lYla5+CPltYN5jA6W3aIoGw9aV4Podc+L4Weh4reLukhWyZtyJjc39XJfB5lOCFhXl\\\\\\\/chVxvpGrE25nxJTySYtRywe\\\\\\\/lPzh7XRkTjRiZNsuzE6QteXhtFqhgPS8JxXzCCsLZjeBOCjmQU9VM1P9k94vvQOztw6h6uO772S8xXRvTf5Gs16KMC++sa9YuwmZOYOVGxG+HGHoV7e2cfGfExgm2JmUE+KlZcehE4nAbEQfLDjoilcZj2KOg8B3Y6Gp3aLg0\\\\\\\/G+rYs+DGMsfFvGphlY8DGRNqRfHG3OeJyqrP7BXVy\\\\\\\/xGsPhCmIXS7U4NAQ0eMti\\\\\\\/nXgpRA0MGZ4m93u3VRDMtrNonSf6ByrU1p5E8z6KiJUKhgLxszCenOq1gi7FxV\\\\\\\/lNqKkvXJ9\\\\\\\/yNvkmk0+hscphgu04OQOzGTwfK8zkVYAPp5ZpyD\\\\\\\/jJG\\\\\\\/PW2a33yE21iJRMiahK6k7Pjkwz7+OxS6xoU1DQVKfRQNpUk\\\\\\\/5v0UzVJgdpvBJARlpa7qBwZlZ5dYDHRY4nOf53iCogpqgVSqiiovQuwxw5KkSIHs5g+6BywXahR+8MXtGAdsLZ2Y0clV9Z8OKzArLI87XSh6zApJ9iGL2FhEgjtAf11cyZJOZESg5\\\\\\\/njITmGt1s3FRO\\\\\\\/eUmTozp8S2UhxphqFSctEn4FAoPjwCp2qo0INwH4bKdwZx\\\\\\\/XTItVFwHI+yEOhKDcI8a63Ahsn2tiy6Wkd4QhSKZwQirj0822OVgX\\\\\\\/7ZV1WkOYhUpR8JNlF+caD7czT36ixf+4kg9ZFNiGKF636nYpwkMywEI9kXnWAsKBlfjM8sRy93zNtwYboJX61N+r6ujmFAPAWZFhRIwQL5W8gAbtblb3Ggb4i1kdmcwEKGAopL46rfVoz8xqkaeqaP4AJS3n4\\\\\\\/Tm2m+ZaqVVUsfTJumLb0XxGBtjTIPEe\\\\\\\/9Q+2mMtUWAu48U5+IrmoMHm94BgAvmEpqG+9jO1s9sUOAbAuKYOffc9NYC3McpkvCFGEAxbcGA2uBTXK+5TUvCDLlmkyd3pgrhCKaShw2DZiB7sdaJ\\\\\\\/u1Sp6XiKwSoojP12pJemKcVNF8427ivb+r7T2wFKbKbTZhD1Nf2KF0NKQ33Xo+2EpyfjRgsxZ5l2fDOgaZCoM2NFPgeqOPr6OMAq3P7u05nuo9GHyCI8b+nA2gzk42oPeGNvg2vD1j8svJ9qVKf40f0CNoahRaOuz78yYK0MYTxcBH+dyjcxJBhB5QDmppvL8sZ2lTwCJBaYqlf8\\\\\\\/KcnAZfJIujsdx\\\\\\\/aJwOD9bGY3A+JRDioRSX6ARsZj0\\\\\\\/D9K7SDM+YsjmmFCfu+qUcTynzwMnuM8YrH8c1l2k3DkUJnRz7cmU1xCjwMH\\\\\\\/LjAaTT8h8OobPUnVC61C9YJvJ1Pth9ioO6WDf0gSZSJ27J1WaI2vNhjXyyA+q1HDzMk6ZTPvkXtM2UwIOhFvZR9vDIOF+WYYGYgRZ8AS9zZVSn12HmjESXWDrreM4Ptj+Y6EXzK8oE8BYnCIsuc+kpQUHhXMu\\\\\\\/FbtW13eDFhOOnomBYScYy6lm9PjJaGDNP5BZsrjAgrxFymHhoDC04guOqTMZumtfGSoira03\\\\\\\/0PnRNQBEJAhxJsrZev6OE1zD6KZalOfGcOFJFwXRWm8hwKfgHIn01YkrHykKGKaDzNbaJWZEqF77ZUGjmZOMQNT2vBsKLGXScVpaf5K1Fblq7tKgtJtU1wVaDnRJrC\\\\\\\/H5uJPuhdhM4HyAAjxf4g+mkdMGUIcegqB3aUPR6EwImJHICPCIdHIwSkvh8FtoQUkzJfYIpore8Ofn6k5UcC671MXrgu+BY6YlMXPiwxKT3tTVPTwCJOWzhH1c16BBEXCDlStKrlYLUVP2UMqjCoJw5byhTg15mTnkpk8YDrKiZDBxoJgv++E\\\\\\\/yuCLLhC+SjyIGswkZeSIV3TBN9k+86YZ1HwyeyY\\\\\\\/8qzq2oy\\\\\\\/0CEqig5lVtlERuIVeEB1hrp7vU+cC4VP27pUD2uJw8QjCUxcI2r43Vo8pxJMBEPjcnM1zW96NeOyydtdK7Jd2DE70y4+fEld9RxAU8cEU0aUKY2zb6XhHR45f1mW6hLGGxQTTeqB9vnDlDfUC5hduYtzPtp+OP8iFq0eBBrC58BrEwCZ0xnrfnlXJgO2D5RZBOAiGNhNRScHJkSwuQlibkyie47PnGinIXE9khv50JUJFRespyFLmqUYH0875qzTPOyXw9tzB\\\\\\\/9v0WiPpAC4+bUbueyXcqKK5DyIoTuL0ZuYcivX4YgJ\\\\\\\/OA7qN3PrEvevBQxUzw5kL+mz0p2M5dvnQ05nsLZVD2TjKpD2Bq0BiBy9PvalDzARB+sti5EmQDGf9kjWroRxAQeE+85RYrNdDnZn7f8rlS7WB+8JQE4Z4QXLgZgYPTtDHL\\\\\\\/oQNtgw+B0HvrPgEqviS\\\\\\\/JMAvop6pvJVND+TqoWCn14RWUAy3nCnh3D+k23nW1DIuT0Zw\\\\\\\/tbuaDrMKu1JGmHd9oVHY3fwiqDh4ljKL9PV+UekrlcHZ6BWLb5ePu2BvcsxavOq4GEg0Jsfmq\\\\\\\/zz9S9C1sNp6TQ8+FKNoIDXQhgIGBY4MntyE9WLEpHQl5QzAEoHzNKvnWLhcOszuSuzTOy1WBqdVJwfCogy16MbuH+eO9ZWrMqw2LXMcKKG75ui65M5wpJS3Gdy8u4ZvC1BMI\\\\\\\/rkysav2AAW5gr\\\\\\\/hhcG3TjzbCuzQteLXG6ZorQpBYXNoyHxEivwCvTjFGvfgnSnXEtbMjEv3vVSKiL\\\\\\\/4suBl\\\\\\\/pc+ivktzE4jhssO\\\\\\\/uDyGi3Zek6f3FiU8eh0dPzsnRGqwfrMQWezmJcQgQjadtKm72JqcUyh0+HdiQQqDRP8LNwEyrmi6HpHPsb4ajtM562GDiLa24yantMGvBqy6D7wmH2ITPHBL3N\\\\\\\/qRAR7Kk5OwLn4RjvJS+gfgIgGZuCxoq8vRXi4D6SoihDjpi1dN\\\\\\\/VAPciQ8GSKJyvcaG+8zK9bE26j2l5PG2X2HW9y\\\\\\\/wQ0GV1ljMLm+frmdrIm\\\\\\\/BCe1pGB+MchkZ83LS0N\\\\\\\/YeikV8c+FgHxK\\\\\\\/At\\\\\\\/3W2cnAq3TsT9HxwxD0l1cBUpOBXb2mtTC32C5thBB54Dcj8rFEyeD1007W9mwTtWioHJcklWY5yGzBebSuW4Hc0FksauK3jMyu1lITNorI5oP1ijh8vKfsby3D0fZSjxgKkJkjCMGoeRshxrqdqivHTkjw30O5PPdPLI6nfhxghclGWI09NL\\\\\\\/ZajPOrEYfjj9srddBzXH7UOlLzYA8ng3ebRVjF1riFlTuWPz1orWHHH5a3VenxqnnLfVK6B24eMBbS+L6kRXZmhGslXYzvcC\\\\\\\/hcGszxcm+ws0B2k7j0B0ULy7tqeacjqCNJuiUzJIWAKEPPldT6oohp2LetGehaYaPaU\\\\\\\/WPVpOj4y0QgqX1hWgsvPYcsMSioMwZ8Lhiuc8ftzqH1dzr7bgFA04tPVr3v\\\\\\\/aWVF2P\\\\\\\/hg3jDHJH82fKow1vCjfJtscS63lnAuszhNbhmN5wVlkEhIFckjsKgXMG0DazvJEFuxY5nDhHfeC2GldJLD37fSI9Lah3n+pux04PSP1YSf8qOophOOFxFpKsHngIrWPESop7Zb\\\\\\\/3DOCkWNyfWS\\\\\\\/2su87LG150Q\\\\\\\/Q+I3JD5fMK\\\\\\\/Cyjmy2CtwWVYAS643tnIZ+oJb7HxF7UVjGSqk5ow7J9pNQMZ28SfN0bQKlul3GlpfYIrIvU8kaevF+7UpL0nZHhWsOlWPj+00ymxRBOaHj9M881z+FA5+MKOYezU61L\\\\\\\/jnYn+WnfgdmpmZ2bsgRS9MUhrSyhSPiUP+OIYSs+9hUjzZC4YTrOiZPAs3ASiDSyIx7Jw2LxMEl8Qqt0QuijTK+tX8vRVLRcmBO9jJuTh3L4L2SVpvWyeR8pFLltOzb1wq5NkhmCNPGgzTm2IiNQA1CJ9NwHCRaXdnibnD64Jp+rHfYo39gzur4gyzX3KHOKgFu398OvROfq7bR8XVaYjifb4UWaOCaMrK+1p0oUU0OH1KGPMU3dWyLveu+riyJL0wm40rWDPg9UMNxppUSV5xb1BMzpUYudDmu2EiRCUCtgUEs5YHacYug9KvlHMHeLIDaKH4ytDpKYsGYTAsNCQAXpH5g15iYD483YHsChIl0kEXKUAH2k91k0C0mPuVCc+qgyYmNyms9+sfSjN7kMHpSU150K\\\\\\\/S2Dg+MSnY64O15VslksTt9Fq3VS+kYA6IWgfghaRspmpFvdxD8KMBiqPzOaAioBxoFEgKyfJAmJbH7t8XS\\\\\\\/tJpCyiutRe2JnEkZIuORXcXnjDBy+6NeLScuTcLdwE36VAPMkY48PY8MTgrQMnso3CyutChqh2vjpEKOK3+iW\\\\\\\/EZ1ceXT3\\\\\\\/f2ASKH4lPuiDlvRPz8aSEtr2ACby68CHE+K\\\\\\\/2ev+c7A4PVwsbi3G+O2wt17P08WTKAil82KD\\\\\\\/brtwBphciWKMzMfx9bUuFkh3DtdDTkt7bFQOtnbWP0rjAV6PmB\\\\\\\/o3QeexUwzbU3PhJzvxfp3Rusi0yu7gyjWBkSEwSh3KDHnpaNTMnN0vT\\\\\\\/ZPw4399m3NLuCCrUZJ05WgaaHf\\\\\\\/A9c1Z0S\\\\\\\/SKv4A6GSkVJOlhI5pTzVABvc\\\\\\\/Gv5QjpZlbz8rw19elpatzJes0VZwNJE9kfyhSFPzxqU4kf\\\\\\\/meSEU1ya6lRV\\\\\\\/nh9nORNR88bWLiaYKqEVAiW2iqR5d7PGRzIbUBeLRR9xt5iOe9BnJ5V8U0sgvH4VvecVw7zZR6dwo4ghRIHzmfQGyCTvXgi+dGgiAz7UhS+Fwvika\\\\\\\/ce97ikwLj+eGZ5FurSpV2Lkzu07VSXY9oVJsaqiu7Q0ox7VJzN7dxikcGnxznsCOkjwY4SlT9UBRm7WLXl9R6S5WlBmEBjXGvhK+RuhuCJsDxMQ34Uyy\\\\\\\/0BngT6QEz1muIQyKE+SKrBibkjV5nG0GdQq3xxixAehSaiq1BgS9WHfcQF2TNxWh97L\\\\\\\/SrQnbs5OtQ8+Pf+sWho+ntglxQydKo+yScpi1SjUSoZjEWH5HjwsqKDMqYkjGdSODj5xux1UFA1pYL7TXah+aGLtdq9585UOg0\\\\\\\/CBQ+b+fuVijzU3+ROImL\\\\\\\/dsuP8ypVrk8+gAI98DJ8TNgNax4FIWOx4BjhHv09jSxGQgkjiL02ACj0xWhq25migQukZMSirrz7\\\\\\\/lx0EEgEUSzXVWQ6anhB5ZC6Plts55XMkl6owwAX480bAJevULPLdxLElO0\\\\\\\/rT4pcP6WgdDJHDz1qhyBmZf8t+47bdbKCoe7DxT+nuVWVYIvvUFZcyjtajvwqOOXN4so7Lzd2\\\\\\\/R9K2yzKG8KD+JhhBf2Bueu\\\\\\\/ONBe+d7BbxpiOjJ5pxzy3phd3\\\\\\\/Ft7F1\\\\\\\/vakTMKUozziLqIWi4x+nplccSmvXpzRJcloxSjc7Gfqin6+VVy71ljvVdPslzeYH3gQjvk\\\\\\\/JucJD+pPPkp1aiPutFwPzV+3EgCYl0LX68opcODGeyfB4ekJwFT+ZcFvisUgLTaxCZMXZBMyvPa2Clz1A\\\\\\\/8Fs2\\\\\\\/qHXK0OrVmnWj\\\\\\\/hsJMu\\\\\\\/FF0f8WYwmCJrJDIGn3EY4tlwsqSflJzucTPMbzUGHlZmkTGuQHd964sHkqUQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Vl56ZYlY5DgjEtdmspUFKg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229533ba9f696cc34d0f8661400cf964b5856177738c210ada5e7743c2bec590b1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KVxjIk78NMTOlBnSRsUrb5bHNg4stvgHqKB5wsfmPD\\\\\\\/kld0mM\\\\\\\/1zwcJSyneWm3goa06NkHQJTAccji46rjoRY0MFjM4K37IH3svqU8ZzOTEGCGzutIjIleqocemC7T8DH+CSaJaZuF3sVPiPI02dtUwbeBoUBS5LvCmKFni41okd7zgv9AIGW9x9tW4NHdUJLWtzRHI5GJ6Dg9t\\\\\\\/9kXCEHpkZAAlHpckoPR11HrALDQwXFoAP00l1UoqZ1eEgkcWPkagHt4THzNPubJEb6GCbRHYk10eQ2tTAIBJiJKB5EUQecXZNsCEJMVudCHbl6xlnvnBlxXXVcBRORDU\\\\\\\/3YpoKxhdiOvYP1FG6VEN7kvwgrlI4gKWEywACsP2v34YBHpbIm9Q01clXOf\\\\\\\/iOyuQtOuAqkqyGjv1pe4arjFvqk6BigVHBrj9MdtyQ3j7G9gQKTAq8spS8xPSzUWObXAxvGjLJEYwAkst8ZIim81gOTVsblXYh+ABwj1JCIE0GZITebxFjnw5Q4+qgdqafJ1MB5kWpCdHFnA\\\\\\\/Vw8pcOmSVfN2PrjUl\\\\\\\/mMLS7ZqudpS1qqknmFJELVU8rxObiHc5N8i0QfzY6El\\\\\\\/s8f7a2bTXziemoliNeBbj\\\\\\\/fvd5sc034ZmqK2KA6e++9rxqLlbYIU5J5z3BFuY++fpe9Hh4FWNKViLK2fdyeuFYPc4ch\\\\\\\/A4Qj12vxLg80wVyzNwwvLw5BF7jZCo\\\\\\\/HaqKvuFi87Kvorkv0AYJTB7+2TnFz5Pj3j6XNjUTq\\\\\\\/rRW3TEHFTO2ER+5Hpt0TjvcPBQa\\\\\\\/XyR1JyqV8AbUVJ2fN79HowAUVXGVnNyKalXeMJhZUGs6DsP15eis5d7lro0xbuDqD52gT5\\\\\\\/6y3+BjBrcnL69luHKaUEi+Z3zfd1wdR3i+mNamzdM680YpheXRQNW1PGdUldX2Oe97iNurmuQqFOYTmubYsLrJGcVx4OLtvVdYQzIyz7P7PDk2mfBnYpoXFUGe5nEB6+JSOs8aemVIbhL23SFW0gr54+TQ36+orJ6ghXIHWT7Gop0ll5Vu9Hq2YgByxCPmQvlKqc\\\\\\\/iWy5eLlcOVIAD9T9RpZEyrgsR+wilbSC6dEy1TymKweUzJ7cPsmJvsLSD9nkPEETvsnr2QU\\\\\\\/CqjaLxZfsoqyxdnT8yQMW3tSpDihLAHjNBIDaqsDiSgLjJs3GpqqAo0+KXQKCpEls4YSYItdibdcrlR1+QD1v1NZMChC6OUVII+Xqx7jA4AP8NCctP6Xx62WYYWG8H5MUNkm2xXRsNcy\\\\\\\/c9OE+s1dCDBjow8LL7g0OFo4bI4UReoyX+uS3zovEuJcEsc5Tv3RL0VJFWSAEsX+41VE4r1ngXCD\\\\\\\/Ph1fZr\\\\\\\/DzF1IzEQh7dElwBjmiO4LKB1IHAfqHcyiqRsnMI8GnMtouGinyZ\\\\\\\/9hsFOVPyI0gMRQVovplFHlvBwWrZvkftBEnBcOja6FuwLvCDQzUBzv57+FU5KUvaOqiuStxStp0LCpZbbtJfvRXr\\\\\\\/JeKMixTiW7Fc0ry8kjA6VAJL6BDgvtMLAKtHPVL6tpY1NGNL64cdvkCS99OJ8shR9\\\\\\\/8k6XCSscjMiJcyzrGh\\\\\\\/qbL6p9l82RBPYVFR75QR7igKJkVgM9Z6ES0zJIPFdCdoLX0+Esv6THWV+wX8krk\\\\\\\/50QjJfCiTcxqGZtu+l8Ai6ftfmR2ZSYhGGEZV8PnGHPoSuW59mStDtF2gCyRAZ4u\\\\\\\/ITmRFVxEm\\\\\\\/BcIWzfqGK5fXYil9BkFPwBZ9s4oKW\\\\\\\/6WpGR2hRxKHMuDCmBlde9KHjcJ+CIK8jJf0izIpqQjA3Vf2hwmqTU\\\\\\\/VdizSTqR2yYTaQaCE9dEaNWYLs4m0HDYaEoCXhLYjTCwWCH4k+e3sAGdcZQdtH4ZgiaNH9t64iRqUdWhYpPkFzD0wklIwgmdszD\\\\\\\/HxUptoW\\\\\\\/5JW25nKrI1zfaWdtQ0QlE0KyAyd0uhFCRrDkQUgLpLPWaLwt\\\\\\\/JBFY3w7QPzHk5znUihYmm18FlQz4horBBherPw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jHpGJ3rC20HqpHTrnMv05A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c38e363f0c711635985a41a8eb069c517c2d2d642545fa77b40bd5cb38fca255\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459460 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/851" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NStgie9cWGrJ98TNr0Sztum-JgA_lKDbrcBO85fSBng" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/851" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ekwg3IzwWjiTbR8mM9W6ZpoBTAPmJkgT20H6T3X9JB8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/841" title="Influence of Steroid Hormone Signaling on Life Span Control by Caenorhabditis elegans Insulin-Like Signaling" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/865" title="PRICE: Software for the Targeted Assembly of Components of (Meta) Genomic Sequence Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459460" id="node459460" data-pisa="ggg;3/5/851" data-pisa-master="ggg;g3.113.005918" data-apath="/ggg/3/5/851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel L Mace</span>, <span class="highwire-citation-author" data-delta="1">Peter Weisdepp</span>, <span class="highwire-citation-author" data-delta="2">Louis Gevirtzman</span>, <span class="highwire-citation-author" data-delta="3">Thomas Boyle</span> and <span class="highwire-citation-author" data-delta="4">Robert H Waterston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">851-863;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005918 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459460" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel L Mace </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BL%2BMace%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mace%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BL%2BMace%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dlm19@uw.edu" class="" data-icon-position="" data-hide-link-title="0">dlm19@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Weisdepp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BWeisdepp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weisdepp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BWeisdepp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Louis Gevirtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BGevirtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gevirtzman%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis%2BGevirtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thomas Boyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyle%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BBoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robert H Waterston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH%2BWaterston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waterston%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH%2BWaterston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/5/851" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459460" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/851.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459460" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/851.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459460" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/851.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459460" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/5/851.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459460" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Advances in microscopy and fluorescent reporters have allowed us to detect the onset of gene expression on a cell-by-cell basis in a systematic fashion. This information, however, is often encoded in large repositories of images, and developing ways to extract this spatiotemporal expression data is a difficult problem that often uses complex domain-specific methods for each individual data set. We present a more unified approach that incorporates general previous information into a hierarchical probabilistic model to extract spatiotemporal gene expression from 4D confocal microscopy images of developing <em>Caenorhabditis elegans</em> embryos. This approach reduces the overall error rate of our automated lineage tracing pipeline by 3.8-fold, allowing us to routinely follow the <em>C. elegans</em> lineage to later stages of development, where individual neuronal subspecification becomes apparent. Unlike previous methods that often use custom approaches that are organism specific, our method uses generalized linear models and extensions of standard reversible jump Markov chain Monte Carlo methods that can be readily extended to other organisms for a variety of biological inference problems relating to cell fate specification. This modeling approach is flexible and provides tractable avenues for incorporating additional previous information into the model for similar difficult high-fidelity/low error tolerance image analysis problems for systematically applied genomic experiments.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/c-elegans" class="hw-term hw-article-keyword hw-article-keyword-c-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">C. elegans</em></a></li><li class="kwd"><a href="/keyword/cell-fate" class="hw-term hw-article-keyword hw-article-keyword-cell-fate" rel="nofollow">cell fate</a></li><li class="kwd"><a href="/keyword/gene-expression" class="hw-term hw-article-keyword hw-article-keyword-gene-expression" rel="nofollow">gene expression</a></li><li class="kwd"><a href="/keyword/image-analysis" class="hw-term hw-article-keyword hw-article-keyword-image-analysis" rel="nofollow">image analysis</a></li><li class="kwd"><a href="/keyword/lineage" class="hw-term hw-article-keyword hw-article-keyword-lineage" rel="nofollow">lineage</a></li></ul><p id="p-4">A fundamental interest in developmental biology is to understand the complex cascade of gene interactions that are responsible for allowing an organism to develop from its early stages as an embryo into a fully functional adult. A key component in this understanding lies in knowing when and where genes are being expressed throughout development. Although high-throughput technologies based on cDNAs (<em>e.g.</em>, RNA-seq) allow the measurement of expression for many genes across development in a single experiment, they do not allow the practical capture of high-resolution spatiotemporal gene expression data. Recent advancements in fluorescence reporter constructs and microscopy, combined with high-throughput imaging, have allowed the measurement of high-resolution (spatial and temporal) expression data for individual genes across development—resulting in large repositories of image data (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hendriks <em>et al.</em> 2006</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hunt-Newbury <em>et al.</em> 2007</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tomancak <em>et al.</em> 2002</a>). The data encoded in these images complements the data obtained from cDNA expression data sets: where cDNA expression data gives high coverage in the number of genes at the cost of lower temporal and spatial resolution, image data allows for greater-resolution data at the cost of lower coverage in the number of genes.</p><p id="p-5">Akin to other high-throughput technologies, the creation of these image data sets requires multidisciplinary expertise in both biological and computational methods. From a biological perspective, work must be done in the creation, preparation, and imaging for each individual gene. Similarly, significant computational hurdles exist in analyzing the data: the spatiotemporal patterns are encoded in images, and methods must be developed to extract meaningful data (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Long <em>et al.</em> 2012</a>). These methods are often domain specific and highly customized and have been developed for many model organisms, including <em>Drosophila</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> 2002</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mace <em>et al.</em> 2010</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Peng <em>et al.</em> 2007</a>), zebrafish (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zanella <em>et al.</em> 2010</a>), <em>Arabidopsis</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mace <em>et al.</em> 2006</a>), and <em>Caenorhabditis elegans</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2006</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Santella <em>et al.</em> 2010</a>). In this article, we focus on the computational problem of developing high-fidelity methods for tracking individual cell fates across development. Although the output of our method can be used to address a large variety of genomic problems (<em>e.g.</em>, natural and phenotypic cell fate variation), we seek to address particularly the problem of obtaining systematic gene expression data across the embryonic development of <em>C. elegans</em>.</p><p id="p-6"><em>C. elegans</em> is a nematode with a transparent body and a cell lineage that is invariant across wild-type worms (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sulston 1983</a>). By the end of embryonic development the worm produces 671 terminal cells, 113 of which undergo cell death, resulting in a total of 558 cells at hatching for the wild-type hermaphrodite. Upon hatching, some cells undergo another series of divisions before the worm reaches its final adult total of 959 somatic cells. As illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, the lineage can be broken down into several well-defined sublineages (often referred to through their founder cells, <em>e.g.</em>, AB, C, D), where each individual sublineage yields one or more cell types (<em>e.g.</em>, muscle, hypodermal, neuron, intestinal). Although the worm is 700 million years diverged from humans (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vanfleteren <em>et al.</em> 1994</a>), many gene regulatory pathways are conserved between <em>C. elegans</em> and humans, which, when combined with the aforementioned attributes, makes <em>C. elegans</em> an ideal organism for translational research into human diseases and conditions such as Parkinson disease (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lakso <em>et al.</em> 2003</a>), Alzheimer disease (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ewald and Li 2010</a>), diabetes (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McKay <em>et al.</em> 2003</a>), myopathy (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rebolledo <em>et al.</em> 2008</a>), and aging (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Park <em>et al.</em> 2009</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/851/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="C. elegans lineage and tracing: (A) The C. elegans lineage follows an invariant division pattern, giving rise to the same 671 terminal leaf cells during embryonic development (early embryonic development to 51 cells shown here). (B) The founder cells are responsible for establishing individual sublineages, which produce one or more primary cell types. (C) Using confocal microscopy, we are able to trace this lineage by using fluorescent promoter constructs to label the individual nuclei from the original images (left). The nuclei are then algorithmically decoded and traced across time (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1288434914" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;C. elegans&lt;/em&gt; lineage and tracing: (A) The &lt;em&gt;C. elegans&lt;/em&gt; lineage follows an invariant division pattern, giving rise to the same 671 terminal leaf cells during embryonic development (early embryonic development to 51 cells shown here). (B) The founder cells are responsible for establishing individual sublineages, which produce one or more primary cell types. (C) Using confocal microscopy, we are able to trace this lineage by using fluorescent promoter constructs to label the individual nuclei from the original images (left). The nuclei are then algorithmically decoded and traced across time (right).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/851/F1.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/5/851/F1.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/851/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/851/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462734" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child"><em>C. elegans</em> lineage and tracing: (A) The <em>C. elegans</em> lineage follows an invariant division pattern, giving rise to the same 671 terminal leaf cells during embryonic development (early embryonic development to 51 cells shown here). (B) The founder cells are responsible for establishing individual sublineages, which produce one or more primary cell types. (C) Using confocal microscopy, we are able to trace this lineage by using fluorescent promoter constructs to label the individual nuclei from the original images (left). The nuclei are then algorithmically decoded and traced across time (right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Gene expression in <em>C. elegans</em> can be captured <em>in vivo</em> by the use of confocal microscopy in conjunction with fluorescent reporter constructs (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Murray <em>et al.</em> 2008</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2012</a>). In this approach, a ubiquitous histone localized green fluorescent protein construct is used to mark the nuclei, whereas a gene specific promoter driving a red fluorescent protein marks the expression pattern of the endogenous gene. An automated confocal microscopy setup is then used to capture 4D (3D space and time) images across embryonic development. To extract the expression data from these images, we need to develop robust methods to identify and follow the individual cells throughout their development while undergoing a variety of developmental events (<em>e.g.</em>, migration, division, death). These methods, often referred to as lineage tracing, provide the full cell identity for each individual cell, allowing the capture of full spatiotemporal gene expression patterns.</p><p id="p-9">Tracing a lineage involves both identifying and linking individual objects (cell nuclei) across time. Lineage tracing shares many similarities to the natural image processing problem of object tracking, which has been intensively studied (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hu <em>et al.</em> 2004</a>). Although both problems must confront the issue of identifying and linking objects over time, they each present their own challenges. In the identification process, lineage tracing has to deal with limitations of optics in later stages of development that often result in closely spaced, small nuclei and attenuation effects deeper into the specimen that can lead to difficulty in identifying the individual nuclei. Similarly, in object tracking, in addition to having to deal with irregular backgrounds (<em>e.g.</em>, shading), the objects are often 2D projections of an actual 3D scene, which can lead to difficult occlusions. The linking process has several differences as well: in object tracking, objects are often assumed to be independent (or weakly dependent), and have movement characteristics that are modeled using standard physical motion laws. Conversely, lineage tracing deals with objects that follow a stereotypical biological process: cells divide and die off in a very constrained pattern throughout development. These differences in detection and linking behavior limits our ability to use traditional object tracking methods for decoding and extracting data from the images.</p><p id="p-10">Previous methods have been developed for lineage tracing and have been used for decoding high-throughput <em>C. elegans</em> image data. An early attempt at lineage tracing, StarryNite, used a separate detection algorithm based on a modified watershed algorithm and a custom distance based algorithm to link the nuclei across time (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2006</a>). Improvements were made for detecting the nuclei by using a blob-slice detection algorithm based on the difference of Gaussians (DoG) feature transform of the image with a slightly modified version of the linking algorithm used in StarryNite (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Santella <em>et al.</em> 2010</a>). Although the blob-slice method decreased the false-positive and false-negative rates, its independent linking and detection algorithms still led to many errors in later stages of development—where the nuclei become closely packed and become difficult to properly ascertain parent/child relationships in divisions/deaths.</p><p id="p-11">We introduce a method for lineage tracing that allows the joint detection and linkage of the nuclei through a hierarchical probabilistic model. This hierarchical framework is able to incorporate additional previous information and constraints in a tractable form. Probabilistic modeling approaches to computer visions problems have become increasingly popular, largely due to the flexibility for incorporating previous information (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashburner and Friston 2005</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Borenstein <em>et al.</em> 2004</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Levin and Weiss 2006</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tu <em>et al.</em> 2005</a>). Much of this work has been done in natural image processing, for whole-scene classification or segmentation problems (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fei-Fei and Perona 2005</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Leibe <em>et al.</em> 2008</a>). Although enjoying increased attention in the natural image processing community, the biological/medical computer vision communities have been slow to adopt these approaches. Historically, this delay is primarily attributed to the large computational and memory intensive requirements of such models, which does not lend itself well to the already computationally and memory-intensive data sets present in the biological/medical field.</p><p id="p-12">This article distinguishes itself from previous work on lineage tracing in three major aspects. First, we design a hierarchical probabilistic model that allows us to incorporate both a more robust bottom-up detection method with top-down prior information on the biological development. We refer to this model as the Statistically Applied Lineage Tracing model, or SALT. Second, we develop empirical reversible-jump Markov Chain Monte Carlo (RJMCMC) methods that allow us to address detection and linkage simultaneously to decode the <em>C. elegans</em> lineage in a more unified fashion. Finally, we validate the model and compare it with previous <em>C. elegans</em> lineage methods, demonstrating that the improved detection and reliability have allowed us to extend our lineaging capability to later stages of embryonic development. This increase in resolution and throughput has been paramount for our approach of obtaining systematic genomic spatiotemporal expression data.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-13">Although the <em>C. elegans</em> lineage is often displayed in a tree-like fashion (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>), it is also possible to represent this in a more formal fashion as described here and shown graphically in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Let <em>c<sub>j,t</sub></em> be the <em>j<sup>th</sup></em> nucleus at time <em>t</em>. We refer to the collection of all <em>m<sub>t</sub></em> nuclei at time <em>t</em> as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-1.gif"/></noscript></span></span>, and the collection of all nuclei across all times as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-2.gif"/></noscript></span></span>. Because the lineage forms a developmental tree, we can represent this time dependence by establishing parent/children relationships; let <em>e<sub>p</sub></em> (<em>c<sub>j,t</sub></em> ) and <em>e<sub>c</sub></em> (<em>c<sub>j,t</sub></em> ) be the parent set and children set functions for <em>c<sub>j,t</sub></em>. An edge exists between two nuclei if they exist within their respective children and parent sets <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-3.gif"/></noscript></span></span>. The number of children define the developmental event (zero children represents a death, one child is a continuation (same cell, new observed nucleus), and two children represents a division with new daughter nuclei). Because divisions and deaths are important developmental events, we further define <em>N</em> to be the set of all nuclei that are the result of a division or birth in a tree <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-4.gif"/></noscript></span></span>, and <em>V</em> the set of all nuclei that die or divide <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-5.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-7.gif"/></noscript></span></span> represent the start and end of a branch <em>B<sub>i</sub></em>, where we define a branch to be the collection of all continuous children from <em>N<sub>i</sub></em> to <em>V<sub>i</sub></em>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-8.gif"/></noscript></span></span>. The branch identifier <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-9.gif"/></noscript></span></span> is used to define the relationship between a nucleus and its containing branch: <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-10.gif"/></noscript></span></span>. Completing this formalization, we set ĉ to be the collection of all nuclei <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-11.gif"/></noscript></span></span>, edges <em>E</em>, branches <em>B</em>, and development times <em>o</em>, <em>s</em>, <em>ĉ</em> = {<em>c</em><sub>.,.</sub>, <em>E</em>, <em>B</em>, <em>o</em>(<em>e</em><sub>[</sub><em><sub>p/c</sub></em><sub>]</sub>), <em>s<sub>t</sub></em>}. <em>s<sub>t</sub></em> is the continuous normalized development time at image capture observation <em>t</em>, and <em>o</em>(<em>e</em><sub>[</sub><em><sub>p/c</sub></em><sub>]</sub>) is the occurrence time of the parent and children edges respectively (described in the <em>Methods</em> section).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/851/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SALT model: a condensed graphical representation of the model. Variables are described using circles, where gray circles represent observed data. Boxes represent multiple occurring values and arrows describe dependencies between variables. In this representation, we describe the lineage of the tree using a variable number of superquadrics cj,t, where j and t are indices of the nuclei and time. The superquadrics are linked through time as shown using the arrows, and the aggregation over all the time points forms a tree, represented as &#x109;. We represent two types of information in our model, bottom-up information and top-down information. The bottom-up information contains dependencies between the image data and the lineage, and consists of time series of 3D confocal microscopy images, denoted xt; a feature-transformed image obtained from the original images, denoted dt; and an induced mean field &#x3BC;t. The top-down information consists of various constraints trained on prior lineage data/experiments, and include: measures on spatial distance, size, number, denoted s; dependencies describing how cells link through time l; and a time to event model on the lifespan of the nuclei, e." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1288434914" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SALT model: a condensed graphical representation of the model. Variables are described using circles, where gray circles represent observed data. Boxes represent multiple occurring values and arrows describe dependencies between variables. In this representation, we describe the lineage of the tree using a variable number of superquadrics &lt;em&gt;c&lt;sub&gt;j,t&lt;/sub&gt;&lt;/em&gt;, where &lt;em&gt;j&lt;/em&gt; and &lt;em&gt;t&lt;/em&gt; are indices of the nuclei and time. The superquadrics are linked through time as shown using the arrows, and the aggregation over all the time points forms a tree, represented as &#x109;. We represent two types of information in our model, bottom-up information and top-down information. The bottom-up information contains dependencies between the image data and the lineage, and consists of time series of 3D confocal microscopy images, denoted &lt;em&gt;x&lt;sub&gt;t&lt;/sub&gt;&lt;/em&gt;; a feature-transformed image obtained from the original images, denoted &lt;em&gt;d&lt;sub&gt;t&lt;/sub&gt;&lt;/em&gt;; and an induced mean field &#x3BC;&lt;em&gt;&lt;sub&gt;t&lt;/sub&gt;&lt;/em&gt;. The top-down information consists of various constraints trained on prior lineage data/experiments, and include: measures on spatial distance, size, number, denoted &lt;em&gt;s&lt;/em&gt;; dependencies describing how cells link through time &lt;em&gt;l&lt;/em&gt;; and a time to event model on the lifespan of the nuclei, &lt;em&gt;e&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/851/F2.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/5/851/F2.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/851/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/851/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462769" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">SALT model: a condensed graphical representation of the model. Variables are described using circles, where gray circles represent observed data. Boxes represent multiple occurring values and arrows describe dependencies between variables. In this representation, we describe the lineage of the tree using a variable number of superquadrics <em>c<sub>j,t</sub></em>, where <em>j</em> and <em>t</em> are indices of the nuclei and time. The superquadrics are linked through time as shown using the arrows, and the aggregation over all the time points forms a tree, represented as ĉ. We represent two types of information in our model, bottom-up information and top-down information. The bottom-up information contains dependencies between the image data and the lineage, and consists of time series of 3D confocal microscopy images, denoted <em>x<sub>t</sub></em>; a feature-transformed image obtained from the original images, denoted <em>d<sub>t</sub></em>; and an induced mean field μ<em><sub>t</sub></em>. The top-down information consists of various constraints trained on prior lineage data/experiments, and include: measures on spatial distance, size, number, denoted <em>s</em>; dependencies describing how cells link through time <em>l</em>; and a time to event model on the lifespan of the nuclei, <em>e</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Although the previous approaches to lineage tracing had used spheres to represent the nuclei, we use a superquadric representation (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barr 1981</a>). A superquadric is an extension of a sphere that includes additional parameters that allow us to more properly model the anisotropic shape characteristics that are inherent to dividing nuclei. Superquadrics have been used in the computer vision community before, often to describe nonparametric shape models (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Terzopoulos and Metaxas 1991</a>). A full superquadric representation would require 10 parameters: three location parameters <em>x</em>, <em>y</em>, <em>z</em>, three axis parameters <em>a</em><sub>1</sub>, <em>a</em><sub>2</sub>, <em>a</em><sub>3</sub>, two exponent parameters <em>e</em><sub>1</sub>, <em>e</em><sub>2</sub>, and two rotation parameters <em>r</em><sub>1</sub>,<em>r</em><sub>2</sub>. We instead reduce the dimension of our superquadric to six superquadric parameters and one additional mean parameter <em>μ</em> (described in the <em>Methods</em> section): <em>c</em> = (<em>x</em>, <em>y</em>, <em>z</em>, <em>a</em><sub>1</sub>, <em>a</em><sub>2</sub>, <em>r</em>, <em>μ</em>). We assume the first axis and the third axis are linked <em>a</em><sub>3</sub> = <em>a</em><sub>1</sub>, the second rotation is fixed <em>r<sub>2</sub></em> = 0, and that the minor exponent and minor axis are ratio linked <em>e</em><sub>2</sub> = <em>a</em><sub>2</sub><em>/a</em><sub>1</sub>. The first exponent parameter is fit during the training of the model. To account for differences in physical voxel size, all representations of coordinates and calculations are done in physical space. Conversion between the physical coordinate space and image space can be done using fast offset and scaling methods.</p><p id="p-16">With the lineage and shape of the nuclei defined, we can introduce a hierarchical probabilistic model that will include previous information about the development of <em>C. elegans</em> for high-accuracy decoding of the lineage. Our hierarchical statistical model is composed of two types of information: bottom-up information (information obtained from the 4D image data sets), and top-down information (previous constraints about the development of the organism). The bottom-up information is composed of two different components: a generative distribution based on the raw observed image and a discriminative distribution based on the DoG feature transform space. The top-down information is composed of three different components: a general component consisting of various topological distributions that constrain nuclei within a time point (major-axis size, spatial interaction, nuclei number, fluorescence intensity); a linkage component to describe the dynamics of nuclei between time points; and a time to event component that describes the occurrence of developmental events (divisions, deaths). Combined, this model allows us to evaluate a proportional probability of any given lineage trace.</p><p id="p-17">Throughout this section, we use simulated annealing methods wrapped around standard Bayesian sampling techniques (in most cases, a standard slice sampling method) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Neal 2003</a>) to train the model. The full model specification, including all previous distributions and the trained parameters, are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/005918SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>.</p></div><div class="section other1" id="sec-2"><h2 class="">Bottom-Up Information</h2><p id="p-18">The bottom-up component deals with two sources of information: the raw observed image <em>x</em>, and a feature transformed DoG image <em>d</em>. For the raw observed image we will describe a shared dependent generative model for all nuclei. The feature transformed image will model the individual nuclei independently using a discriminative model. The combination of a generative and discriminative model on two sources of information provides a more reliable and accurate method for nuclei detection: although each individual distribution is prone to its own false-positives and false-negatives, by combining the two approaches we can provide a more accurate method of detection.</p><div id="sec-3" class="subsection"><h3>Generative component</h3><p id="p-19">The generative component is used to model the relationship between the individual nuclei and the observed image. This relationship is captured by allowing the nuclei to compose a shared dependent mean field <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-12.gif"/></noscript></span></span> that induces a generative distribution on the observed image. This shared dependency from the mean field is important, as in later stages of development, the compactness of the nuclei exceeds our optical resolution, resulting in the appearance of overlapping and shared fluorescence signal between nuclei.</p><p id="p-20">Intuitively, the generative component describes a relationship between the decoded nuclei and the observed image. During the decoding process, the model will iteratively propose new additional nuclei to the model and evaluate the proposal of the new addition. As part of this process, an expected mean image is generated based on the current and newly proposed nucleus (taking into account information such as the position and intensity of nuclei). The closer the new expected image is to the actual observed image taken with the microscope, the more likely the model is to accept the new configuration.</p><p id="p-21">We define the intensity at location <em>i</em> at time <em>t</em> of the mean field image to be a linear combination of a background mean, and a weighted value from every individual nucleus. <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-13.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-14.gif"/></noscript></span></span> is a location function that returns the x, y, z location of an object (<em>e.g.</em>, a voxel location). The summation occurs over the fluorescence mean parameter <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-15.gif"/></noscript></span></span> for all nuclei within this time point (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-16.gif"/></noscript></span></span>), where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-17.gif"/></noscript></span></span> is the normalized anisotropic kernel function. This is a piecewise function that overall decreases in value the further the pixel is from the center of the nucleus. The anisotropic nature of the function leads to a slower decrease in the <em>z</em> plane than the <em>x/y</em> planes to account for attenuation effects that are consistent with confocal microscopy. A graphical representation of this distance function is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. 4 and is described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/851/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bottom-up components: (A) The generative component of the model. An expected background model for the observed image is created by using the set of nuclei (2) c to induce a mean field by allowing each nucleus to contribute a localized mean to the surrounding voxels. The contribution to each voxel is shown in (4), where the x-axis shows the percentage of the mean, and the y axis shows the decay based on the normalized distance (distance/radius). The blue line shows the decay in the x/y plane, and the red dotted line shows the decay in the z plane. The resulting mean field image (5), can be used to form a generative distribution on the observed image xt (6), by using a generative Laplace distribution (7). (B) The discriminative component of the model. The original image is transformed into a DoG filter bank, an example single element/z-plane of the bank is shown in (4). For each individual nuclei, a discriminative function is created by discretizing the points along equidistant rays extending out from the center of the nuclei (6). A generalized linear model is then fit to model the normalized decay of the DoG image as shown in (8). The y-axis is the normalized DoG value, and the x-axis is the normalized distance from the center of the nucleus (blue/red lines are the x/y plane and z plane, respectively). By the use of a series of transformations, a probability distribution (9) can be used to calculate a 0&#x2212;1 probability on the existence of any point with a specific shape and location configuration." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1288434914" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bottom-up components: (A) The generative component of the model. An expected background model for the observed image is created by using the set of nuclei (2) &lt;em&gt;c&lt;/em&gt; to induce a mean field by allowing each nucleus to contribute a localized mean to the surrounding voxels. The contribution to each voxel is shown in (4), where the &lt;em&gt;x&lt;/em&gt;-axis shows the percentage of the mean, and the &lt;em&gt;y&lt;/em&gt; axis shows the decay based on the normalized distance (distance/radius). The blue line shows the decay in the &lt;em&gt;x/y&lt;/em&gt; plane, and the red dotted line shows the decay in the &lt;em&gt;z&lt;/em&gt; plane. The resulting mean field image (5), can be used to form a generative distribution on the observed image &lt;em&gt;x&lt;sub&gt;t&lt;/sub&gt;&lt;/em&gt; (6), by using a generative Laplace distribution (7). (B) The discriminative component of the model. The original image is transformed into a DoG filter bank, an example single element/z-plane of the bank is shown in (4). For each individual nuclei, a discriminative function is created by discretizing the points along equidistant rays extending out from the center of the nuclei (6). A generalized linear model is then fit to model the normalized decay of the DoG image as shown in (8). The &lt;em&gt;y&lt;/em&gt;-axis is the normalized DoG value, and the &lt;em&gt;x&lt;/em&gt;-axis is the normalized distance from the center of the nucleus (blue/red lines are the &lt;em&gt;x/y&lt;/em&gt; plane and &lt;em&gt;z&lt;/em&gt; plane, respectively). By the use of a series of transformations, a probability distribution (9) can be used to calculate a 0&#x2212;1 probability on the existence of any point with a specific shape and location configuration.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/851/F3.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/5/851/F3.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/851/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/851/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462755" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Bottom-up components: (A) The generative component of the model. An expected background model for the observed image is created by using the set of nuclei (2) <em>c</em> to induce a mean field by allowing each nucleus to contribute a localized mean to the surrounding voxels. The contribution to each voxel is shown in (4), where the <em>x</em>-axis shows the percentage of the mean, and the <em>y</em> axis shows the decay based on the normalized distance (distance/radius). The blue line shows the decay in the <em>x/y</em> plane, and the red dotted line shows the decay in the <em>z</em> plane. The resulting mean field image (5), can be used to form a generative distribution on the observed image <em>x<sub>t</sub></em> (6), by using a generative Laplace distribution (7). (B) The discriminative component of the model. The original image is transformed into a DoG filter bank, an example single element/z-plane of the bank is shown in (4). For each individual nuclei, a discriminative function is created by discretizing the points along equidistant rays extending out from the center of the nuclei (6). A generalized linear model is then fit to model the normalized decay of the DoG image as shown in (8). The <em>y</em>-axis is the normalized DoG value, and the <em>x</em>-axis is the normalized distance from the center of the nucleus (blue/red lines are the <em>x/y</em> plane and <em>z</em> plane, respectively). By the use of a series of transformations, a probability distribution (9) can be used to calculate a 0−1 probability on the existence of any point with a specific shape and location configuration.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The mean field induces a generative distribution on the observed image:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-18.gif"/></noscript></span></span>where <em>L</em><sub>1</sub> and <em>L</em><sub>2</sub> and are modified Laplace distributions based on the actual <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-19.gif"/></noscript></span></span> and first derivative<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-20.gif"/></noscript></span></span> respectively of the mean and observed values. The combination of both distributions is necessary for both sampling and accuracy reasons: <em>L</em><sub>1</sub> provides a convex surface that is easy to sample individual nuclei with but is often prone to errors in determining the minor axis and rotation, whereas <em>L</em><sub>2</sub> provides a nonconvex surface that is difficult to sample from, but is more accurate with determining the minor axis and rotation. The Laplace distributions are normalized to adjust the weight of the generative model compared to all other components in the model: because the area of the superquadric increases at the cube of the major axis, the largest nuclei would have an ≈125-fold increase in voxel contributions to the likelihood—significantly adjusting the relative weight of the generative component to the rest of the model. To adjust for this, additional weighting fields <em>π</em><sup>1</sup> and <em>π</em><sup>2</sup> are constructed similar to the mean field construction which addresses this imbalance by reweighting the nuclei such that the largest nuclei would have only ≈3-fold increase in voxel contributions than the smallest nuclei.</p></div><div id="sec-4" class="subsection"><h3>Discriminative component</h3><p id="p-24">Although the generative component describes a relationship between all nuclei and the raw observed image, the discriminative component describes an independent relationship between each nucleus and the feature transformed image. The discriminative component is more akin to classical feature detection, as it is often used to describe particular regions that “appear” to be a nucleus. Instead of the traditional “bag of features” approaches that are often used, we use a more formal probabilistic approach for the discriminative model. For any proposed nucleus, a generalized linear model is used to assign a 0−1 probability value on the existence of a nucleus at a given location for any given superquadric parameters. Because the discriminative component is often looking for subtle features of the image space, it is often able to pick out small changes in image intensity that would otherwise be missed. As a result, it is generally more sensitive than the generative model, but less specific.</p><p id="p-25">To fit the discriminative model, we transform the original space into a bank of DoG images (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Marr and Hildreth 1980</a>), where <em>d</em><sub>b</sub> is the <em>b</em><sup>th</sup> filter bank image, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-21.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-22.gif"/></noscript></span></span> is the starting bank, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-23.gif"/></noscript></span></span> is the spacing of the banks, and <em>n</em> is the number of banks. As can be seen in step 4 of <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, the feature space has peaked values at the center of the nucleus that decrease in value as it extends away from the nucleus. This decay differs from the generative model because it is a strictly decreasing function based on the normalized distance of the superquadric (compared with the piecewise disjoint decreasing function). To model this decay, we define a set of points for nucleus <em>c<sub>j</sub></em>, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-24.gif"/></noscript></span></span> that are constructed by selecting evenly spaced points along a set of evenly angular spaced rays from the center of nucleus <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-25.gif"/></noscript></span></span>, <em>y</em> is a unit vector normalized to the length of the major axis, and <em>A</em> is an evenly angled spaced transformation matrix described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a> and shown in step 6 in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>. Because the location of the sampled points do not lie on a standard lattice, a second-order cubic spline interpolation method is used to calculate the values.</p><p id="p-26">Let <em>r</em><sub>i</sub> be the <em>i</em><sup>th</sup> element in set <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-26.gif"/></noscript></span></span>. We set <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-27.gif"/></noscript></span></span> to be the DoG value of the discretized point normalized at the filter bank set at the nucleus major axis <em>a</em><sub>1</sub>, by the value at the center of the nucleus <em>r</em><sub>0</sub>: <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-28.gif"/></noscript></span></span>. This decrease of values from the center of the nucleus is modeled using a multivariate normal distribution:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-29.gif"/></noscript></span></span></p><p id="p-27">An anisotropic angular distance function is used to model the expected value of the discretized values<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-30.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-31.gif"/></noscript></span></span> is an anisotropic distance function and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-32.gif"/></noscript></span></span> is a parameter that describes the rate at which the DoG value decreases and is show in step 8 of <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>. In later stages of development, this mean level is often distorted by closely spaced neighboring nuclei. To account for this variability, we model a full covariance structure that captures the dependency between closely spaced points:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-33.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-34.gif"/></noscript></span></span> is an individual error that is on the diagonal only <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-35.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-36.gif"/></noscript></span></span> are correlation functions described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>.</p><p id="p-28">We transform this generative multivariate normal distribution into a discriminative model by converting the probability to log space and using a generalized linear model<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-37.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-38.gif"/></noscript></span></span> are parameters that affect the scaling and offset of the log probability, whereas <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-39.gif"/></noscript></span></span> is a penalty function that ensures that the DoG value at the center of the nucleus is above a certain baseline value. <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-40.gif"/></noscript></span></span> is wrapped around a logistic regression model that re-normalizes the value between [0−1], where any value greater than 0.5 would be indicative of the presence of a cell nucleus. As the model normalizes the DoG value to the center baseline pixel, the sensitivity/specificity trade-off comes from <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-41.gif"/></noscript></span></span>. Small values would pick up many of the dimly lit cells (<em>e.g.</em>, Z2, Z3) but would result in many false positives from image artifacts (general intensity, cover slip). This parameter is learned while training the remainder of the parameters.</p></div></div><div class="section other2" id="sec-5"><h2 class="">Top-Down Model</h2><p id="p-29">Although the bottom-up components deal with information from the observed and transformed images, the top-down components deal with information pertaining to the development of the embryo and is obtained from previously semiautomated methods for tracing the <em>C. elegans</em> lineage (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2006</a>). The models are trained from a collection of 4D images (large collection of 2D TIFF images) as well as comma delimited files describing the location and parent/child relationships between nuclei. Taken together, we believe in combination with appropriate data input transformations, the method could be readily extended to similar lineage tracing problems in other organisms. A fundamental aspect of the top-down model is a series of components for modeling the relationships between nuclei across time, allowing us to simultaneously predict and link the cell nuclei during the decoding phase. The remaining top-down information consist of previous constraints on either the detection or on relationships between nuclei that allow us to improve the detection and linking accuracy of the model.</p><p id="p-30">Throughout development, many of the topological constraints vary depending on the time of development (<em>e.g.</em>, as developmental time progresses, cells become smaller, times between divisions become longer). To more appropriately model this, we use kernel functions to induce time varying parameters as described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>. Although this provides more accurate models for the topological constraints, it also introduces an additional complexity during the decoding process as the normalized development time becomes an additional parameter to be sampled. Although the true dynamics of development can be more properly modeled using the cell generation rather than the development time, this approach was avoided as small errors in linking early in development would have a cumulative effect on all remaining daughter nuclei. An intermediate approach of using mixture models to account for heterogeneity in nuclei generation had a similar effect, where decoding the lineage was difficult as daughter cells would often inherit the mistakes of its parent lineage. The coarser approach of using development time and aggregate measures over all nuclei provided a higher and more accurate decoding.</p><div id="sec-6" class="subsection"><h3>General top-down components</h3><p id="p-31">The general top-down components consist of four independent distributions on various topological constraints: nucleus size, number of nuclei, individual fluorescence, and spatial interaction. The nucleus size consists of two distributions based on the major and minor axis of the nucleus. For the major axis, we use a time varying normal distribution</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-42.gif"/></noscript></span></span><p id="p-32">As can be seen in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, the size of the major axis becomes smaller as time progresses. For the minor axis, we use a fixed normal distribution<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-43.gif"/></noscript></span></span></p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/851/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Top-down components: Trained parameters for various top-down components, x-axis refers to the normalized time, mean levels are shown with points, and error bars represent one standard deviation. (A) Number of nuclei as a function of normalized time. (B) Size of the nucleus as a function of normalized time. (C) Time to event model as a function of normalized time: (top) the proportional probabilities of an event to occur; (middle) the timing between divisions; and (bottom) the death event timing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1288434914" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Top-down components: Trained parameters for various top-down components, x-axis refers to the normalized time, mean levels are shown with points, and error bars represent one standard deviation. (A) Number of nuclei as a function of normalized time. (B) Size of the nucleus as a function of normalized time. (C) Time to event model as a function of normalized time: (top) the proportional probabilities of an event to occur; (middle) the timing between divisions; and (bottom) the death event timing.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/851/F4.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/5/851/F4.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/851/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/851/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462821" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Top-down components: Trained parameters for various top-down components, x-axis refers to the normalized time, mean levels are shown with points, and error bars represent one standard deviation. (A) Number of nuclei as a function of normalized time. (B) Size of the nucleus as a function of normalized time. (C) Time to event model as a function of normalized time: (top) the proportional probabilities of an event to occur; (middle) the timing between divisions; and (bottom) the death event timing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Like the major axis distribution, the distribution on the number of nuclei is also a time varying normal distribution:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-44.gif"/></noscript></span></span></p><p id="p-35">The individual fluorescence of each nuclei is defined as follows:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-45.gif"/></noscript></span></span></p><p id="p-36">The last general top-down distribution is the spatial interaction distribution. To prevent nuclei from directly overlapping, we use a mixed soft/hard core penalization distribution. This distribution is defined as:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-46.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-47.gif"/></noscript></span></span> is a penalty function from point <em>c</em><sub>i</sub> to <em>c</em><sub>j</sub>:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-48.gif"/></noscript></span></span></p><p id="p-37">This penalty function prevents cells from becoming too close through stepwise constraints: as points enter a soft core distance <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-49.gif"/></noscript></span></span>, they become slowly penalized based on their proximity to each other until they reach a hard core distance <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-50.gif"/></noscript></span></span>, where the penalty goes to infinity and the probability of the configuration goes to 0.</p><p id="p-38">The constraints described allow the model to eliminate strong outliers or unexpected nuclei from the decoding. A large 4-μm radius nucleus would be appropriate in early stages of development but would not be expected in later stages of development; rather, the model would be more likely to find two separate nuclei of 2 microns each. Additionally, as the model is often accurately searching for the most optimal solution, the previous constraints prevent the model from searching in parameter spaces that are outside the range expected from the developing embryo, which improves runtime performance.</p></div><div id="sec-7" class="subsection"><h3>Time-dependent top-down component</h3><p id="p-39">The previously described components have dealt with the dependence of nuclei within a time point, but not across time points, and can be loosely thought of as the “detection” part of the model, but not the “linking” part, of the model. To establish a linking relationship across time, we model both the dynamics of the cell movement, as well as the timing and fate of the development. For the dynamics, we model the relationship between time points <em>t</em> and <em>t</em> + 1 using a conditional random field to capture parent child movement constraints (<em>e.g.</em>, cell movement, major axis change, division angle). Our linkage component is closely related to the time dependence of a particle filter model, and is merely a modification with additional constraints. Conversely, the timing and fate of the development component deals with the eventual (sometimes unobserved) probability of a developmental event occurring. When a cell arises from the fusion of the pronuclei or from a division, it is “predestined” to undergo one of three developmental fates: it will divide, it will die off, or it will reach its final differentiated cell type. A time-to-event model is used to describe the probability and timing of these events. It is important to note that the scope of each of these components differ. Where the dynamic submodel considers the relationship between nuclei in adjacent time points, the time-to-event submodel considers the aggregate relationship of all cells within a branch across multiple time points.</p></div><div id="sec-8" class="subsection"><h3>Time-to-event component</h3><p id="p-40">As described, the time-to-event model deals with the timing and occurrence of developmental events: divisions, deaths, final differentiation. Time-to-event models are often used in clinical studies for survival analysis of patients with disease (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hosmer <em>et al.</em> 2008</a>). This component can be abstractly thought of as a distribution on the development lineage tree itself. Placing restrictions on the tree allows the model to prevent unexpected occurrences (<em>e.g.</em>, early cell deaths, fast divisions, missed divisions). These constraints are crucial in later stages of development, where the close proximity and attenuation effects make cell divisions and cell deaths difficult to properly follow individual nuclei across time.</p><p id="p-41">A crucial requirement of the time to event model is the ability to account for the changing characteristics of the developing embryo. Early development is characterized by rapid divisions that quickly establish the main founder cells responsible for the individual sublineages. With the exception of the germline cells Z2 and Z3, none of these cells have entered their final differentiated state, and there are no deaths. Later in development, the organism enters organogensis, and the time between divisions increases. Cell deaths are common in this stage, and many cells are nearing their final embryonic cell state. We model this using a time varying time to event model.</p><p id="p-42">Let <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-51.gif"/></noscript></span></span> be the indicator function of the ending development event for branch <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-52.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-53.gif"/></noscript></span></span>. We describe the probability of this branch as<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-54.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-55.gif"/></noscript></span></span> is the base probability of the event, and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-56.gif"/></noscript></span></span> is the probability that a branch exists with a duration length of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-57.gif"/></noscript></span></span>. We introduce <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-58.gif"/></noscript></span></span> as the developmental timing of the start or end of the branch. These are additional parameters of the model and are sampled during the decoding phase, bound by their respective normalized times. The base probability of an event at time <em>t</em> is:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-59.gif"/></noscript></span></span></p><p id="p-43">The duration between events is modeled using a time varying Gaussian distribution based on the square root of the branch duration:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-60.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-61.gif"/></noscript></span></span> are the mean and variance for the event. The timing of what constitutes a cell death often varies based on the literature, and can mean many things. In our representation, the timing of cell death is the time from when a cell divides until it essentially disappears and becomes “untrackable.” Because this timing is directly obtained from previous lineage experiments, we expect it to vary from other definitions of cell death. The time-varying parameters of the trained model are shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>.</p></div><div id="sec-9" class="subsection"><h3>Linkage component</h3><p id="p-44">The linkage component describes the dynamics of the relationships of nuclei between neighboring time points <em>t</em> and <em>t</em> + 1. For the dynamics, two possible configurations exist between parent and children: cells continue on (one child), or cells divide (two children). The third possible configuration, cell death, is not modeled as there is no cell at time <em>t</em> + 1 to model the dynamics; however, the occurrence of timing and cell death is accounted for in the time to event component.</p><p id="p-45">The linkage component describes the fundamental element responsible for establishing relationships between nuclei across two time points. In other models, this relationship is often established through custom heuristics based on the Euclidean distance between nuclei across time, or in the case of particle filters, previous constraints on the dynamics of the cells movement. As we describe, our method extends upon these approaches by developing models that describe specific constraints on how the nuclei change over time (<em>e.g.</em>, size, location), and how they respond during division events (<em>e.g.</em>, angle of division, division distance).</p><p id="p-46">The dynamics between a parent and its children varies based on whether a nucleus is continuing on or dividing. We model individual state specific dynamics by imposing time varying constraints on the parent child relationships between nuclei. In the single child case (continuation), constraints are placed on the movement of the nucleus (expected to be small), and the change in nuclear size (also expected to be small). For divisions, the case is more complex: the cell often undergoes a stereotypical pattern of division in which the dynamics of each daughter cell are dependent. To account for this dependency, we use five features to model nuclear divisions: the movement of each daughter nucleus (2), the change between the radius of the parent and each child (2), the difference of the change between the radius of the parent and each child, the difference in the movement of both daughter nuclei, and the angle of the division. We use a conditional random field (CRF) to link these constraints (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lafferty <em>et al.</em> 2001</a>). The CRF provides a natural way of describing links by incorporating each individual constraint into a feature, and then fitting a feature weight:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-62.gif"/></noscript></span></span><em>ψ</em> is the summation over all possible feature functions <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-63.gif"/></noscript></span></span>, whereas <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-64.gif"/></noscript></span></span> are the individual weights for the features. <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-65.gif"/></noscript></span></span> is an interaction term that accounts for some of the covariability of the division features. For the features we use time varying squared error functions: <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-66.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-67.gif"/></noscript></span></span> is an observed feature. Because the feature functions are unnormalized log normal probabilities, and the CRF is an exponentiated summation over these functions, the CRF essentially becomes a weighted unnormalized product of Gaussian distributions. We make extensions to the mean and variance to allow it to become more generalized:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-68.gif"/></noscript></span></span></p><p id="p-47">This extends previous time varying parameter specifications to include an additional parameter <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-69.gif"/></noscript></span></span>, which is the elapsed development time for the link: <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-70.gif"/></noscript></span></span>. By allowing the model to change based on the duration between time points we provide a dynamic relationship for the model: as the spacing between time points increases, the mean and variance of the feature space changes (<em>e.g.</em>, the variance on the cell movement increases while the mean division angle decreases). This extension allows us to decode images taken from varying time spacings of image capture. Time varying parameter representations are shown in <a href="http://www.g3journal.org/content/suppl/2013/03/23/g3.113.005918.DC1/FigureS1.pdf">Figure S1</a>.</p></div></div><div class="section other3" id="sec-10"><h2 class="">Decoding the Lineage</h2><p id="p-48">The hierarchical probabilistic representation allows us to evaluate the probability of any given lineage as the proportional product of the generative, discriminative, general top-down, time to event, and linkage submodels:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-71.gif"/></noscript></span></span></p><p id="p-49">This proportional product can be used to determine the most probable configuration for any given 4D image series by employing hybrid simulated annealing and RJMCMC to decode the lineage. RJMCMC is an extension to standard MCMC methods that are often used <em>in trans</em>-dimensional model selection problems such as variable selection, mixture models and factor analysis (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Green 1995</a>). In our particular case, the number of nuclei and their link through time is an unknown. As a result, determining the most probable lineage becomes a <em>trans</em>-dimensional model selection problem. Similar work using RJMCMC methods have been applied in computer vision for identifying cells (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Al-Awadhi <em>et al.</em> 2011</a>), detecting trees (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Perrin <em>et al.</em> 2006</a>), plant branching (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schlecht <em>et al.</em> 2007</a>), road extraction (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lacoste <em>et al.</em> 2010</a>), and human tracking (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2008</a>). Although particle filters can also be used to address the uncertainty in nuclei in the lineage, they are known to have difficulties in high dimensional problems (558 cells at the end of embryonic development), where the number of particles needed to accurately represent the underlying model is both computationally and memory prohibitive (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Peursum <em>et al.</em> 2010</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Snyder <em>et al.</em> 2008</a>).</p><p id="p-50">We decompose the RJMCMC methods into two distinct set of moves: single time point RJMCMC moves, and multi-time point RJMCMC moves. The single time point RJMCMC moves behave similar to traditional decoding approaches such as particle filters, by addressing each individual time point in a sequential fashion <em>t</em> = 1, 2, 3,… through a series of reversible jumps moves such as adding a point, or removing a point within a given time point. The multi time RJMCMC methods are akin to back filtering methods of particle filters in that they can occur over multiple time points and at any current or previous time and are generally defined by moves that affect the tree of the lineage, such as: adding a branch to the lineage, or removing a branch from the lineage. By combining both of these approaches, we are able to address issues that are difficult to account for using standard sequential RJMCMC methods, such as false deaths or lack of detecting a daughter cell during a division. The full algorithmic details on the selection of moves, updates, and all details on empirical proposals are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>.</p><div id="sec-11" class="subsection"><h3>Single-time RJMCMC</h3><p id="p-51">The single-time steps deal with moves on a single individual time point. These consist of six moves, four RJMCMC moves, and two standard MCMC moves. The four RJMCMC moves are: add a point, remove a point, split a point into two points, and merge two points into one point. Adding a point <em>k</em> = <em>m</em><sub>t</sub> + 1 to time point <em>t</em> consists of a transition from <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-72.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-73.gif"/></noscript></span></span>. To simulate an accurate draw from the posterior of <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-74.gif"/></noscript></span></span>, we draw a new set of parameters for <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-75.gif"/></noscript></span></span>, where the shape parameters are drawn from their time varying prior distributions (uniform if no previous exists (rotation)), and the location is drawn from an empirical proposal distribution. Standard MCMC methods are used to update the parameters of the new point using a slice sampler (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Neal 2003</a>) and are intermixed with a linking step to establish parent/child relationships from the previous time point <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-76.gif"/></noscript></span></span> as described in the following linking step. After updating the parameters and linking the parents for the point, the addition of the new point is then accepted or rejected using the acceptance ratio described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>. The remove move performs the opposite, drawing a point <em>j</em> from time <em>t</em> from an empirical proposal and choosing either to accept or reject the move. The merge move operates by taking two individual points, <em>c<sub>j,t</sub></em> and <em>c<sub>k,t</sub></em>, and merging them into one new point, <em>c<sub>l,t</sub></em>, whose parameters are set to the mean parameters of the previous two points. The merged point then undergoes an updating and linking step similar to the add move and is either accepted or rejected. The split move is the reverse of the merge move, and splits one point <em>c<sub>l,t</sub></em> into two points, <em>c<sub>j,t</sub></em> and <em>c<sub>k,t</sub></em>, where the new parameters are then drawn from the previous distributions, and the location is drawn from an orthogonal split along the location of previous points. The parameters and parents of each new point are then sampled similarly and either accepted or rejected. In addition to the reversible jump moves, there are also two standard MCMC moves. The first is an update point move, which involves updating the parameters and parents of each individual point. The second move, is a relink area move, which uniformly samples a point <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-77.gif"/></noscript></span></span> from all points at time <em>t</em>, selects all points within a radius <em>r</em> of <em>c<sub>j,t</sub></em>, and updates their parent cell relationship.</p></div><div id="sec-12" class="subsection"><h3>Multi-time RJMCMC</h3><p id="p-52">The multi-time steps are a combination of individual single time add and remove steps that span over multiple time points. These consists of five moves, four RJMCMC moves, and one standard MCMC move. The four RJMCMC moves are: add a branch, remove a branch, adjust start of branch, and adjust end of branch. For the add a branch move, a time point is empirically sampled and proposed to extend backward a length of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-78.gif"/></noscript></span></span> in time using standard MCMC methods to update the parameters for each individual time point. A new configuration is then selected by empirically choosing a branch length from all possible branches <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-79.gif"/></noscript></span></span> based on their individual probabilities. To maintain balance with the deletion move, the selected branch then undergoes a relinking step, where the end of the new branch is adjusted as described below. The new adjusted branch <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-80.gif"/></noscript></span></span> is then accepted or rejected based on the standard acceptance ratio. The remove branch proceeds in the opposite fashion: a uniformly selected branch <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-81.gif"/></noscript></span></span> is selected from the lineage, and undergoes a relinking of the right side of the branch. All points within this branch are then removed from the lineage <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-82.gif"/></noscript></span></span>. The new configuration, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-83.gif"/></noscript></span></span> then undergoes a similar acceptance ratio.</p><p id="p-53">The adjust start and end branch moves involve extending or contracting the start or end of a particular branch of the lineage tree. Let <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-84.gif"/></noscript></span></span> be a randomly selected branch of the lineage. The start relinking of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-85.gif"/></noscript></span></span> involves proposing to add or remove a length of <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-86.gif"/></noscript></span></span> elements of the branch. The move is considered in both directions for both relinking steps, which results in <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-87.gif"/></noscript></span></span> possible moves (the relinking of the left, right, or the current state). The new state of the model is then selected from the empirical probabilities of all possible moves.</p></div><div id="sec-13" class="subsection"><h3>Parent/child linking step</h3><p id="p-54">The parent child relinking step is a sampling step for linking parent/child relationships. For a cell <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-88.gif"/></noscript></span></span>, we propose to link it to a parent cell at time point <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-89.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-90.gif"/></noscript></span></span> be the set of all points at time <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-91.gif"/></noscript></span></span> that are within a radius <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-92.gif"/></noscript></span></span> of point <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-93.gif"/></noscript></span></span>, where the mean and variance are the expected mean and variance from the distance feature of the division link CRF. For each point in the proposed set, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-94.gif"/></noscript></span></span>, with configuration <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-95.gif"/></noscript></span></span>, we let<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-96.gif"/></noscript></span></span>be the normalized proposed probability of a new parent. A parent is empirically selected from these normalized probabilities. Additional empirical moves are also used to allow for switching moves and for relinking past/future events, and are discussed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-14" class="subsection"><h3>General parameters</h3><p id="p-55">In addition to the parameters for the nuclei and their links, we sample the normalized development time for each time point in the series. Let <em>s<sub>t</sub></em> be the observed normalized time for time point <em>t</em>. A linear model is then fit to <em>s<sub>t</sub></em>: <em>s<sub>t</sub></em> = β<sub>0</sub> + β<sub>1</sub><em>t</em>, where β<sub>0</sub> is the starting normalized offset time, and β<sub>1</sub> is the spacing between time points. The parameters are fit using a weighted posterior that is proportional to the prior cell number distribution of the initially seeded data described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a> as well as the cell number distribution of the decoded data and the generative time varying components of the full model (time to event, major axis, and cell numbers). These parameters are updated throughout the decoding of the model.</p></div><div id="sec-15" class="subsection"><h3>Replica exchange model</h3><p id="p-56">To improve performance and increase convergence, a local replica exchange model is used (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2005</a>). Throughout the decoding process, a series of <em>l</em> possible chains are run, where each individual chain has separate parameters <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-97.gif"/></noscript></span></span>, with their own temperature <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-98.gif"/></noscript></span></span>. For each decoding move (single and multi time), standard updates are mixed with replica exchange moves, where the temperature of each individual chain are proposed to exchange. Let <em>q</em>, <em>r</em> be two individual replicas, with individual temperatures <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-99.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-100.gif"/></noscript></span></span>. A proposal is made to swap the temperatures of particles <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-101.gif"/></noscript></span></span> with acceptance probability<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-102.gif"/></noscript></span></span></p><p id="p-57">The acceptance or rejection of each decoding move is performed on the lowest temperature replica:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-103.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-104.gif"/></noscript></span></span> is the replica indicator of the lowest temperature element (the <em>k<sup>th</sup></em> temperature). After the acceptance or rejection, the individual replicas are all synchronized to the lowest temperature replica: <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/851/embed/mml-math-105.gif"/></noscript></span></span>. The local replica exchange model allows us to more thoroughly explore the configuration space of the lineage without getting stuck in local optima.</p></div><div id="sec-16" class="subsection"><h3>Postanalysis curation and expression determination</h3><p id="p-58">After the image series has been processed as described, a final manual curation step takes place where individual errors are corrected as described in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boyle <em>et al.</em> (2006)</a>. As part of this curation step, automated methods are run that provide the final cell lineage name using a rule-based system and quantification of the expression across the whole lineage.</p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><p id="p-59">We have developed a method to identify nuclei and trace the cell lineage of the developing <em>C. elegans</em> embryo from 4D (3D space and time) confocal microscopy images. Our method differs from previous cell lineage tracing approaches by incorporating prior constraints about <em>C. elegans</em> embryonic development into the tracing problem. These constraints are designed to correct many of the errors prevalent in earlier approaches by more explicitly taking into account the natural patterns inherit to development (<em>e.g.</em>, expected angle of division, timing between divisions). We modeled these constraints using common statistical approaches and use general aggregate measures across the population of cells to provide a more extensible model. In addition, we also integrate the often two-step approaches for cell lineage tracing (identifying nuclei in a first pass, and then linking nuclei through time in a second pass) into a single unified method. This unified approach, when combined with the previous constraints, allows us to correct for many of the errors that were common with previous methods in later stages of development.</p><div id="sec-18" class="subsection"><h3>Validation and accuracy</h3><p id="p-60">To assess the performance of this unified approach, we compared our model (SALT) to the two previous full embryonic methods for <em>C. elegans</em> lineage tracing of StarryNite referred to as WaterShed-StarryNite (WS-SN) (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2006</a>), and BlobSlice-StarryNite (BS-SN) (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Santella <em>et al.</em> 2010</a>). Although other methods have also attempted to address this lineage problem (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carranza <em>et al.</em> 2011</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Giurumescu <em>et al.</em> 2012</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kang <em>et al.</em> 2012</a>) all have error rates on par with the original WS-SN, with none approaching the BS-SN method. Additionally, none of these methods have been evaluated beyond early/mid embryonic development. As we demonstrate, our method is a significant improvement over both the WS-SN and BS-SN methods at both early and later stages of embryonic development.</p><p id="p-61">We established a quantifiable measure for the accuracy of each method by creating a test set of 12 image series evaluated at two different time stages: the 350-cell stage (245 normalized time, right after the 8th, but before the 9th AB lineage division), and the 550 (550 total visible cells, some of which are undergoing cell death)-cell stage (335 normalized time, right after the 9th, but before the 10th AB lineage division). The 550-cell stage was chosen as an easily identifiable embryonic development time needed to address the varying biological experiment ranges of the individual image series. The error rate for each individual series was assessed on two types of errors: detection errors (expected cells that are missing, or new unexpected cells) and linking errors (<em>e.g.</em>, incorrect assignment of daughter cells during division, change of naming between time points). To provide a standardized method for evaluation, each individual method is automatically compared to fully curated data sets as described in detail in Supplemental Methods. Full results of all three methods are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and a detailed breakdown and analysis on the types of linking errors are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1/FileS1.pdf">File S1</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/851/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Error rate evaluation. (A) The evaluation was performed at the 245 Sulston time (350 visible nuclei) and 335 Sulston time (550 visible nuclei). 3D representation of the nuclei are shown, colors are identical to colors used in Figure 1. (B) Comparison of three methods: our method (SALT), Blob Slice-StarryNite (BS-SN), and WaterShed-StarryNite (WS-SN). The method was evaluated for two types of errors, detection and linking. Lower bars represent fewer errors and greater accuracy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1288434914" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Error rate evaluation. (A) The evaluation was performed at the 245 Sulston time (350 visible nuclei) and 335 Sulston time (550 visible nuclei). 3D representation of the nuclei are shown, colors are identical to colors used in Figure 1. (B) Comparison of three methods: our method (SALT), Blob Slice-StarryNite (BS-SN), and WaterShed-StarryNite (WS-SN). The method was evaluated for two types of errors, detection and linking. Lower bars represent fewer errors and greater accuracy.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/851/F5.medium.gif" width="440" height="270"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/5/851/F5.medium.gif" width="440" height="270"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/851/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/851/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462726" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-62" class="first-child">Error rate evaluation. (A) The evaluation was performed at the 245 Sulston time (350 visible nuclei) and 335 Sulston time (550 visible nuclei). 3D representation of the nuclei are shown, colors are identical to colors used in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>. (B) Comparison of three methods: our method (SALT), Blob Slice-StarryNite (BS-SN), and WaterShed-StarryNite (WS-SN). The method was evaluated for two types of errors, detection and linking. Lower bars represent fewer errors and greater accuracy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-63">Our approach outperformed the previous methods in both detection and linking errors. In earlier stages of development (350 cells), both SALT and BS-SN performed similarly in detection abilities, with a near-identical number of detection errors (6.8 <em>vs.</em> 7.3). However, in linking the nuclei, SALT performed better, having a 4.9-fold reduction in errors (10.8 <em>vs.</em> 52.8). In later stages of development (550 cells), both the detection and linking of SALT were significantly improved compared with the other methods. Our approach had a 2.2-fold reduction in detection errors (49.6 <em>vs.</em> 106.8), as well as a 4.5-fold reduction in linking errors (119.0 <em>vs.</em> 536.7) for an overall 3.8-fold reduction compared with BS-SN.</p><p id="p-64">The error rate is an important measure of quality because these errors must be corrected through manual curation to obtain an accurate lineage tracing and hence expression pattern. This step is critical since even a single error early in development can cause a cascade effect throughout the remainder of the lineage. A decrease in the number of errors translates into a decrease in editing time for a given developmental stage and also allows us to curate our data sets to later stages of embryonic development. Our evaluation of the editing effort in this paper is based on the number of curation steps required to correct mis-specifications in the lineage. The previously reported edits from WS-SN at the 350-cell stage required a combined 225 curation steps to establish the correct lineage. By contrast, our method required 18 curations at the 350 cell stage, and only 168 curations at the 550-cell stage. Thus, our method effectively performs better at the 550-cell stage than WS-SN at the 350-cell stage, requiring 25% fewer corrections. This significant improvement allows us to follow spatiotemporal gene expression data out to later stages of development than previously possible.</p></div><div id="sec-19" class="subsection"><h3>Detecting expression in final differentiated cells</h3><p id="p-65">Extending our lineage capabilities to the 550-cell (335 time) point allows us to address biological questions that would not be possible with earlier approaches. At the 350-cell (245 time) point, many of the cells still require one or more divisions before they reach their terminal embryonic cell fate. The majority of this final differentiation occurs in the AB lineage, where the last remaining division often signifies the specialization of a cell into one of many different subclasses of neurons. Proper specialization of the neuronal sublineage is an intricate process and involves many different genes acting in a complex cascade of gene transcription (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hobert <em>et al.</em> 2010</a>). With the extension of lineaging to the 550-cell point, nearly all the individual cells have reached their final state of differentiation in embryonic development.</p><p id="p-66">To illustrate the importance of this increase in fidelity, we show the expression profile of an individual gene, <em><a href="http://www.wormbase.org/db/get?name=ttx-3;class=Gene">ttx-3</a></em>, at both the previous limit of 350 cells as well as at 550 cells obtained with our new method. <em><a href="http://www.wormbase.org/db/get?name=ttx-3;class=Gene">ttx-3</a></em> is a transcription factor required for proper functioning of the thermosensory and olifactory response, and is responsible for cell fate specification for a subset of the neurons (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bertrand and Hobert 2009</a>). As can be seen in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, data at 350 cells fails to capture any of the mid/late onset expression for <em><a href="http://www.wormbase.org/db/get?name=ttx-3;class=Gene">ttx-3</a></em>. In contrast, our method is capable of detecting the onset of expression that occurs in 12 cells near or after the final cell fate decision. These 12 cells are composed of the six previously detected cells: AINL, AINR, AIYL, AIYR, SMDDL, SMDDR (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bertrand and Hobert 2009</a>), as well as the ASKR and ASKL cells and four cell deaths. As can be seen in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, ASKL and ASKR appear in both technical replicates and we expect this may relate to differences in detection sensitivity, differences in constructs, or possibly differences in integration sites.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/851/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ttx-3 Evaluation. ttx-3 Expression: (A) 3D nuclei representation at 335 Sulston time. Individual nucleus color is identical to colors used in Figure 1. (B) Representation with expressing no death cells highlighted. Colors represent intensity. (C) Comparison of two replicates (1 and 2) of ttx-3. Green line represents previous limitations of gene expression (245 Sulston). Blue asterisks represent an individual cell that still has one division cycle remaining." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1288434914" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;ttx-3&lt;/em&gt; Evaluation. &lt;em&gt;ttx-3&lt;/em&gt; Expression: (A) 3D nuclei representation at 335 Sulston time. Individual nucleus color is identical to colors used in Figure 1. (B) Representation with expressing no death cells highlighted. Colors represent intensity. (C) Comparison of two replicates (1 and 2) of ttx-3. Green line represents previous limitations of gene expression (245 Sulston). Blue asterisks represent an individual cell that still has one division cycle remaining.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/851/F6.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/5/851/F6.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/851/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/851/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462759" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-67" class="first-child"><em>ttx-3</em> Evaluation. <em>ttx-3</em> Expression: (A) 3D nuclei representation at 335 Sulston time. Individual nucleus color is identical to colors used in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>. (B) Representation with expressing no death cells highlighted. Colors represent intensity. (C) Comparison of two replicates (1 and 2) of ttx-3. Green line represents previous limitations of gene expression (245 Sulston). Blue asterisks represent an individual cell that still has one division cycle remaining.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">To provide an estimate of how many genes are differentially expressed in late stages of development that our method would be able to detect, we looked at the next closest representation of spatiotemporal data in <em>C. elegans</em>: tissue-specific expression data obtained from fluorescence-activated cell-sorted data (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Spencer <em>et al.</em> 2011</a>). Using a precompiled list of genes with significant expression, we filtered for genes that have no expression in early embryonic expression, but have differential expression in later embryonic stages. Although this filtering provides a stringent criteria for detecting differential late gene expression, we were still left with 4456 potential genes that are candidates for late-onset differential expression. This list, encompassing nearly 25% of the genome, illustrates the importance of having high-fidelity robust methods for our systematic approach to obtain genomic level expression data throughout embryonic development.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-69">By incorporating general prior constraints into the model and simultaneously addressing detection and linking, we have developed a robust method that has significantly reduced the error rate for lineage tracing applied to 4D confocal microscopy images. Our model is more accurate than previous approaches, with an overall reduction in errors of 10.4-fold and 3.8-fold compared with the WS-SN and BS-SN versions. This increase in accuracy has allowed us to follow spatiotemporal gene expression to later stages of development, providing greater resolution data for differentially expressed genes.</p><p id="p-70">Although our method has increased performance in both detection and linking at later stages of development, we expect the increased performance in detection is not the result of an improved detector, but rather from the combined integration of prior data with a unified detection and linking method. Many of the detection errors in later stages can be corrected by accounting for prior constraints and dependency between time: false-positive results are prevented by the general constraints on division appearances and timings, and false negatives are prevented from expectations on deaths. The importance of this integration becomes even more apparent when dealing with assigning relationships to nuclei between time points further on in development. In these later time stages, the nuclei are small and tightly spaced, creating irregular division patterns that often cannot be explained by simple proximity relationships. Many of these issues are often corrected at later sampling times, when constraints on division timing and the presence of new information allows the model to correct previous mis-assignments.</p><p id="p-71">Full 4D imaging of developmental organisms has become increasingly important, and methods to trace the lineage are paramount for biological discovery. Although our current application of cell lineage tracing has been for the direct purposes of obtaining high-resolution systematic gene expression data, robust methods for following cell identity are applicable to a variety of problems and are critical for understanding phenotypic variation through detailed quantifiable measures (cell movement, cell−cell interaction, cell cycle). The benefits of high-fidelity lineage tracing become more apparent when considering organisms or processes with variant cell fate, such as cancer progression, where these quantifiable measures would provide us with a more fine grained analysis of the underlying mechanisms that is not possible with other high throughput methods.</p><p id="p-72">The methods used in our model are generalized linear models that are extensions of standard hierarchical statistical models, with slight modifications to the RJMCMC to allow for empirical Bayesian proposal distributions. With modifications to account for increased variance from more complicated lineages, as well as performance modifications to account for increased number of nuclei (discussed below), all of the bottom-up and top-down distributions used in this model have a direct application to other model organism. Additionally, we expect the more formal probabilistic structure to be amendable to incorporating additional previous information on the development of the organism (<em>e.g.</em>, joint lineage tracing of the nuclei with segmentation of the cell wall/membrane), expanding our ability to address new biological problems.</p><p id="p-73">We note that although our method outperformed both WS-SN and BS-SN in accuracy, it does have a drawback in its runtime performance. The total computational and runtime demands of our model are ≈22 hours and 12 GB of ram for the 550 cell (335 time) stage. This is in contrast to WS-SN’s requirement of 30 minutes and 200 MB of ram for the same stage. This runtime performance requirement would be prohibitive for more complicated model organisms such as zebrafish, which contains tens of thousands of cells (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Keller <em>et al.</em> 2008</a>). Extensive profiling and runtime simulations suggest, however, that the majority of this performance requirement comes from the overhead of accessing the memory of the observed and featured transformed image from the bottom up components. Recent work has shown that the increased memory bandwidth speeds and massive parallel capabilities of graphical processing units would be a more practical solution for these methods (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Suchard <em>et al.</em> 2010</a>). To explore this possibility, these methods were rewritten in OpenCL and run separately— resulting in a 25- to 100-fold increase in throughput when executed on the graphical processing unit. These results are encouraging and suggest that when combined with other modifications (<em>e.g.</em>, finely tuned proposal mechanisms), such improvements would be capable of addressing the increased computational demands for more complicated organisms.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005918/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-74"><em>Communicating editor: D. S. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-02-19"><span class="received-label">Received </span>February 19, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-03-16"><span class="accepted-label">Accepted </span>March 16, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Mace <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.1" data-doi="10.1080/02664760903117747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Al-Awadhi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jennison</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Three-dimensional Bayesian image analysis and confocal microscopy</span>. <abbr class="cit-jnl-abbrev">J. Appl. Stat.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BStat.%26rft.volume%253D38%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1080%252F02664760903117747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/02664760903117747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.2" data-doi="10.1016/j.neuroimage.2005.02.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friston</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Unified segmentation</span>. <span class="cit-source">NeuroImage</span> <span class="cit-vol">26</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroImage%26rft.stitle%253DNeuroimage%26rft.aulast%253DAshburner%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D839%26rft.epage%253D851%26rft.atitle%253DUnified%2Bsegmentation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2005.02.018%26rft_id%253Dinfo%253Apmid%252F15955494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2005.02.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15955494&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230211100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.3" data-doi="10.1073/pnas.0511111103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ooi</span>  <span class="cit-name-given-names">S. L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandel</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Automated cell lineage tracing in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">2707</span>–<span class="cit-lpage">2712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0511111103%26rft_id%253Dinfo%253Apmid%252F16477039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAzLzgvMjcwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS84NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.4" data-doi="10.1109/MCG.1981.1673799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barr</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Superquadrics and angle-preserving transformations</span>. <abbr class="cit-jnl-abbrev">IEEE Comput. Graph. Appl.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BComput.%2BGraph.%2BAppl.%26rft.volume%253D1%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCG.1981.1673799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCG.1981.1673799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.5" data-doi="10.1016/j.devcel.2009.02.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Linking asymmetric cell division to the terminal differentiation program of postmitotic neurons in <em>C. elegans</em></span>. <span class="cit-source">Dev. Cell.</span> <span class="cit-vol">16</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DBertrand%26rft.auinit1%253DV.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D563%26rft.epage%253D575%26rft.atitle%253DLinking%2Basymmetric%2Bcell%2Bdivision%2Bto%2Bthe%2Bterminal%2Bdifferentiation%2Bprogram%2Bof%2Bpostmitotic%2Bneurons%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2009.02.011%26rft_id%253Dinfo%253Apmid%252F19386265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2009.02.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19386265&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265470400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.5.851.6"><div class="cit-metadata unstructured">Borenstein, E., E. Sharon, and S. Ullman, 2004 Combining top-down and bottom-up segmentation. <em>Computer Vision and Pattern Recognition Workshop</em> <strong>4</strong>: 46, 2004.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.7" data-doi="10.1186/1471-2105-7-275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araya</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Acetree: a tool for visual analysis of caenorhabditis elegans embryogenesis</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBoyle%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D7%26rft.spage%253D275%26rft.epage%253D275%26rft.atitle%253DAceTree%253A%2Ba%2Btool%2Bfor%2Bvisual%2Banalysis%2Bof%2BCaenorhabditis%2Belegans%2Bembryogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-275%26rft_id%253Dinfo%253Apmid%252F16740163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16740163&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.5.851.8"><div class="cit-metadata unstructured">Carranza, N., I. Smal, O. Dzyubachyk, W. Niessen, and E. Meijering, 2011 Automated lineage tree reconstruction from caenorhabditis elegans image data using particle filtering based cell tracking, pp. 1921 –1924 in <em>2011 IEEE International Symposium on Biomedical Imaging: From Nano to Macro</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmerling</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Modified replica exchange simulation methods for local structure refinement</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem. B</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">8220</span>–<span class="cit-lpage">8230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPhysical%2BChemistry.%2BB%252C%2BCondensed%2BMatter%252C%2BMaterials%252C%2BSurfaces%252C%2BInterfaces%2Band%2BBiophysical%26rft.stitle%253DJournal%2Bof%2BPhysical%2BChemistry.%2BB%252C%2BCondensed%2BMatter%252C%2BMaterials%252C%2BSurfaces%252C%2BInterfaces%2Band%2BBiophysical%26rft.aulast%253DCheng%26rft.auinit1%253DX.%26rft.volume%253D109%26rft.issue%253D16%26rft.spage%253D8220%26rft.epage%253D8230%26rft.atitle%253DModified%2Breplica%2Bexchange%2Bsimulation%2Bmethods%2Bfor%2Blocal%2Bstructure%2Brefinement.%26rft_id%253Dinfo%253Apmid%252F16851961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16851961&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.10" data-doi="10.1007/s00429-009-0235-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewald</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding the molecular basis of Alzheimer’s disease using a <em>Caenorhabditis elegans</em> model system</span>. <abbr class="cit-jnl-abbrev">Brain Struct. Funct.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBrain%2BStruct%2BFunct%26rft.aulast%253DEwald%26rft.auinit1%253DC.%2BY.%26rft.volume%253D214%26rft.issue%253D2-3%26rft.spage%253D263%26rft.epage%253D283%26rft.atitle%253DUnderstanding%2Bthe%2Bmolecular%2Bbasis%2Bof%2BAlzheimer%2527s%2Bdisease%2Busing%2Ba%2BCaenorhabditis%2Belegans%2Bmodel%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00429-009-0235-3%26rft_id%253Dinfo%253Apmid%252F20012092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00429-009-0235-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20012092&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fei-Fei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perona</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A Bayesian hierarchical model for learning natural scene categories</span>. <abbr class="cit-jnl-abbrev">Computer Vision Pattern Recogni.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputer%2BVision%2BPattern%2BRecogni.%26rft.volume%253D2%26rft.spage%253D524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.12" data-doi="10.1242/dev.086256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giurumescu</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Planchon</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betzig</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloomekatz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative semi-automated analysis of morphogenesis with single-cell resolution in complex embryos</span>. <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">4271</span>–<span class="cit-lpage">4279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.086256%26rft_id%253Dinfo%253Apmid%252F23052905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM5LzIyLzQyNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.13" data-doi="10.1093/biomet/82.4.711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Reversible jump Markov chain Monte Carlo computation and Bayesian model determination</span>. <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F82.4.711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjgyLzQvNzExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.14" data-doi="10.1186/gb-2006-7-12-r123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendriks</span>  <span class="cit-name-given-names">C. L</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keranen</span>  <span class="cit-name-given-names">S. V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fowlkes</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simirenko</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">3D morphology and gene expression in the Drosophila blastoderm at cellular resolution I: data acquisition pipeline</span>. <span class="cit-source">Genome Biol.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">R123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLuengo%2BHendriks%26rft.auinit1%253DC.%2BL.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253DR123%26rft.epage%253DR123%26rft.atitle%253DThree-dimensional%2Bmorphology%2Band%2Bgene%2Bexpression%2Bin%2Bthe%2BDrosophila%2Bblastoderm%2Bat%2Bcellular%2Bresolution%2BI%253A%2Bdata%2Bacquisition%2Bpipeline.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-12-r123%26rft_id%253Dinfo%253Apmid%252F17184546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-12-r123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17184546&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.15" data-doi="10.1016/j.tins.2010.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrera</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefanakis</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The molecular and gene regulatory signature of a neuron</span>. <abbr class="cit-jnl-abbrev">Trends Neurosci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bneurosciences%26rft.stitle%253DTrends%2BNeurosci%26rft.aulast%253DHobert%26rft.auinit1%253DO.%26rft.volume%253D33%26rft.issue%253D10%26rft.spage%253D435%26rft.epage%253D445%26rft.atitle%253DThe%2Bmolecular%2Band%2Bgene%2Bregulatory%2Bsignature%2Bof%2Ba%2Bneuron.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tins.2010.05.006%26rft_id%253Dinfo%253Apmid%252F20663572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tins.2010.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20663572&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283398700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.851.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosmer</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemeshow</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Applied Survival Analysis: Regression Modeling of Time to Event Data</span>, <span class="cit-edition">Ed. 2.</span> <span class="cit-publ-name">Wiley-Interscience</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maybank</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A survey on visual surveillance of object motion and behaviors</span>. <abbr class="cit-jnl-abbrev">IEEE Trans. Syst. Man Cybern. C</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BSyst.%2BMan%2BCybern.%2BC%26rft.volume%253D34%26rft.spage%253D334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.18" data-doi="10.1371/journal.pbio.0050237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt-Newbury</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viveiros</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mah</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anastas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-throughput in vivo analysis of gene expression in <em>Caenorhabditis elegans</em></span>. <span class="cit-source">PLoS Biol.</span> <span class="cit-vol">5</span>: <span class="cit-fpage">e237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHunt-Newbury%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De237%26rft.epage%253De237%26rft.atitle%253DHigh-throughput%2Bin%2Bvivo%2Banalysis%2Bof%2Bgene%2Bexpression%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050237%26rft_id%253Dinfo%253Apmid%252F17850180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17850180&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.5.851.19"><div class="cit-metadata unstructured">Kang, S., C. A. Giurumescu, A. D. Chisholm, and P. Cosman, 2012 Automated nuclei tracking in c. elegans based on spherical model fitting with multiple target tracking, pp. 17–20 in <em>2012 IEEE Southwest Symposium on Image Analysis and Interpretation (SSIAI)</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.20" data-doi="10.1126/science.1162493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittbrodt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelzer</span>  <span class="cit-name-given-names">E. H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reconstruction of Zebrafish early embryonic development by scanned light sheet microscopy</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162493%26rft_id%253Dinfo%253Apmid%252F18845710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjIvNTkwNC8xMDY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayaraman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panchanathan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurunathan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marti-Subirana</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BEST: a novel computational approach for comparing gene expression patterns from early stages of drosophila melanogaster development</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2037</span>–<span class="cit-lpage">2047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2037%26rft.epage%253D2047%26rft.atitle%253DBEST%253A%2BA%2BNovel%2BComputational%2BApproach%2Bfor%2BComparing%2BGene%2BExpression%2BPatterns%2BFrom%2BEarly%2BStages%2Bof%2BDrosophila%2Bmelanogaster%2BDevelopment%26rft_id%253Dinfo%253Apmid%252F12524369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.22" data-doi="10.1016/j.patcog.2009.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacoste</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Descombes</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerubia</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Unsupervised line network extraction in remote sensing using a polyline process</span>. <abbr class="cit-jnl-abbrev">Pattern Recognit.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPattern%2BRecognit.%26rft.volume%253D43%26rft.spage%253D1631%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.patcog.2009.11.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.patcog.2009.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.5.851.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lafferty</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">F. C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Conditional random fields: probabilistic models for segmenting and labeling sequence data,</span> pp. <span class="cit-fpage">282</span>–<span class="cit-lpage">289</span> in <span class="cit-source"><em>Proceedings of the Eighteenth International Conference on Machine Learning</em>, <em>ICML ’01</em></span>. <span class="cit-publ-name">Morgan Kaufmann Publishers Inc.</span>, <span class="cit-publ-loc">San Francisco</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.24" data-doi="10.1046/j.1471-4159.2003.01809.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lakso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vartiainen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moilanen</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirvio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dopaminergic neuronal loss and motor deficits in <em>Caenorhabditis elegans</em> overexpressing human alpha-synuclein</span>. <abbr class="cit-jnl-abbrev">J. Neurochem.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurochemistry%26rft.stitle%253DJ%2BNeurochem%26rft.aulast%253DLakso%26rft.auinit1%253DM.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D172%26rft.atitle%253DDopaminergic%2Bneuronal%2Bloss%2Band%2Bmotor%2Bdeficits%2Bin%2BCaenorhabditis%2Belegans%2Boverexpressing%2Bhuman%2B%257Balpha%257D-synuclein%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-4159.2003.01809.x%26rft_id%253Dinfo%253Apmid%252F12807436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-4159.2003.01809.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12807436&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183526200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.25" data-doi="10.1007/s11263-007-0095-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leibe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonardis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiele</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Robust object detection with interleaved categorization and segmentation</span>. <abbr class="cit-jnl-abbrev">Int. J. Comput. Vis.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BComput.%2BVis.%26rft.volume%253D77%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11263-007-0095-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11263-007-0095-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-3.5.851.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Learning to combine bottom-up and top-down segmentation</span>, pp. <span class="cit-fpage">581</span>–<span class="cit-lpage">594</span> in <span class="cit-source"><em>Computer Vision – ECCV 2006</em>, <em>volume 3954 of Lecture Notes in Computer Science</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Leonardis</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bischof</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Pinz</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin/Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.27" data-doi="10.1371/journal.pcbi.1002519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Visualization and analysis of 3D microscopic images</span>. <span class="cit-source">PLoS Computational Biol.</span> <span class="cit-vol">8</span>:<span class="cit-fpage">e1002519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiol.%26rft.volume%253D8%26rft.spage%253D1002519e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.28" data-doi="10.1093/bioinformatics/btl228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twigg</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colinas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benfey</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Quantification of transcription factor expression from Arabidopsis images</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">e323</span>–<span class="cit-lpage">e331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DMace%26rft.auinit1%253DD.%2BL.%26rft.volume%253D22%26rft.issue%253D14%26rft.spage%253De323%26rft.epage%253De331%26rft.atitle%253DQuantification%2Bof%2Btranscription%2Bfactor%2Bexpression%2Bfrom%2BArabidopsis%2Bimages%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl228%26rft_id%253Dinfo%253Apmid%252F16873489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTQvZTMyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS84NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.29" data-doi="10.1093/bioinformatics/btp658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varnado</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frise</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohler</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Extraction and comparison of gene expression patterns from 2D RNA in situ hybridization images</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp658%26rft_id%253Dinfo%253Apmid%252F19942587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzc2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS84NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.30" data-doi="10.1098/rspb.1980.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marr</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hildreth</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Theory of edge detection</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DMarr%26rft.auinit1%253DD.%26rft.volume%253D207%26rft.issue%253D1167%26rft.spage%253D187%26rft.epage%253D217%26rft.atitle%253DTheory%2Bof%2BEdge%2BDetection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1980.0020%26rft_id%253Dinfo%253Apmid%252F6102765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjA3LzExNjcvMTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.31" data-doi="10.1016/S1534-5807(02)00411-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avery</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graff</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title"><em>C. elegans</em>: a model for exploring the genetics of fat storage</span>. <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DMcKay%26rft.auinit1%253DR.%2BM.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D131%26rft.epage%253D142%26rft.atitle%253DC%2Belegans%253A%2Ba%2Bmodel%2Bfor%2Bexploring%2Bthe%2Bgenetics%2Bof%2Bfat%2Bstorage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1534-5807%252802%252900411-2%26rft_id%253Dinfo%253Apmid%252F12530969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1534-5807(02)00411-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12530969&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180862000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.32" data-doi="10.1038/nmeth.1228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeck</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mericle</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Automated analysis of embryonic gene expression with cellular resolution in <em>C. elegans</em></span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMurray%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D703%26rft.epage%253D709%26rft.atitle%253DAutomated%2Banalysis%2Bof%2Bembryonic%2Bgene%2Bexpression%2Bwith%2Bcellular%2Bresolution%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1228%26rft_id%253Dinfo%253Apmid%252F18587405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18587405&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258077400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.33" data-doi="10.1101/gr.131920.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vafeados</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mericle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multidimensional regulation of gene expression in the <em>C. elegans</em> embryo</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1282</span>–<span class="cit-lpage">1294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131920.111%26rft_id%253Dinfo%253Apmid%252F22508763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzcvMTI4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS84NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.34" data-doi="10.1214/aos/1056562461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neal</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Slice sampling</span>. <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D31%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1056562461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1056562461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000184078200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.35" data-doi="10.1111/j.1474-9726.2009.00473.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.-K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedesco</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Oxidative stress and longevity in <em>Caenorhabditis elegans</em> as mediated by SKN-1</span>. <abbr class="cit-jnl-abbrev">Aging Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAging%2Bcell%26rft.stitle%253DAging%2BCell%26rft.aulast%253DPark%26rft.auinit1%253DS.%2BK.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D258%26rft.epage%253D269%26rft.atitle%253DOxidative%2Bstress%2Band%2Blongevity%2Bin%2BCaenorhabditis%2Belegans%2Bas%2Bmediated%2Bby%2BSKN-1.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1474-9726.2009.00473.x%26rft_id%253Dinfo%253Apmid%252F19627265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1474-9726.2009.00473.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19627265&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266417700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.36" data-doi="10.1186/1471-2121-8-S1-S7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Automatic image analysis for gene expression patterns of fly embryos</span>. <abbr class="cit-jnl-abbrev">BMC Cell Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-supplement">Suppl. 1</span>): <span class="cit-fpage">S7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BCell%2BBiol.%26rft.volume%253D8%26rft.spage%253DS7%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2121-8-S1-S7%26rft_id%253Dinfo%253Apmid%252F17634097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2121-8-S1-S7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17634097&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.37"><div class="cit-metadata unstructured">Perrin, G., X. Descombes, and J. Zerubia, 2006 Point processes in forestry: an application to tree crown detection. Technical Report RR-5544, INRIA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.38" data-doi="10.1007/s11263-009-0205-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peursum</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A study on smoothing for particle-filtered 3D human body tracking</span>. <abbr class="cit-jnl-abbrev">Int. J. Comput. Vis.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BComput.%2BVis.%26rft.volume%253D87%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11263-009-0205-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11263-009-0205-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.39" data-doi="10.1007/s12035-008-8041-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebolledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minniti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fadic</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Inclusion body myositis: a view from the <em>Caenorhabditis elegans</em> muscle</span>. <abbr class="cit-jnl-abbrev">Mol. Neurobiol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bneurobiology%26rft.stitle%253DMol%2BNeurobiol%26rft.aulast%253DRebolledo%26rft.auinit1%253DD.%2BL.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D178%26rft.epage%253D198%26rft.atitle%253DInclusion%2Bbody%2Bmyositis%253A%2Ba%2Bview%2Bfrom%2Bthe%2BCaenorhabditis%2Belegans%2Bmuscle.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12035-008-8041-0%26rft_id%253Dinfo%253Apmid%252F18773311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12035-008-8041-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18773311&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259997700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.40" data-doi="10.1186/1471-2105-11-580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowotschin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadjantonakis</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A hybrid blob-slice model for accurate and efficient detection of fluorescence labeled nuclei in 3D</span>. <span class="cit-source">BMC Bioinformatics</span> <span class="cit-vol">11</span>:<span class="cit-fpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSantella%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.spage%253D580%26rft.epage%253D580%26rft.atitle%253DA%2Bhybrid%2Bblob-slice%2Bmodel%2Bfor%2Baccurate%2Band%2Befficient%2Bdetection%2Bof%2Bfluorescence%2Blabeled%2Bnuclei%2Bin%2B3D.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-580%26rft_id%253Dinfo%253Apmid%252F21114815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21114815&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.5.851.41"><div class="cit-metadata unstructured">Schlecht, J., K. Barnard, E. Spriggs, and B. Pryor, 2007. Inferring grammar-based structure models from 3d microscopy data, pp. 1–8, in <em>IEEE Conference on Computer Vision and Pattern Recognition CVPR ’07</em>. IEEE Computer Society, Los Alamitos, CA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.42" data-doi="10.1175/2008MWR2529.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengtsson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Obstacles to high-dimensional particle filtering</span>. <abbr class="cit-jnl-abbrev">Mon. Weather Rev.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">4629</span>–<span class="cit-lpage">4640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMon.%2BWeather%2BRev.%26rft.volume%253D136%26rft.spage%253D4629%26rft_id%253Dinfo%253Adoi%252F10.1175%252F2008MWR2529.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/2008MWR2529.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000262467000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.43" data-doi="10.1101/gr.114595.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">W.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henz</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watkins</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A spatial and temporal map of c. elegans gene expression</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114595.110%26rft_id%253Dinfo%253Apmid%252F21177967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMzI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.44" data-doi="10.1198/jcgs.2010.10016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frelinger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cron</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding GPU programming for statistical computation: studies in massively parallel massive mixtures</span>. <abbr class="cit-jnl-abbrev">J. Comput. Grap. Stat.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BGrap.%2BStat.%26rft.volume%253D19%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjcgs.2010.10016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jcgs.2010.10016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.45" data-doi="10.1016/0012-1606(83)90201-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulston</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The embryonic cell lineage of the nematode <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSulston%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D100%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D119%26rft.atitle%253DThe%2Bembryonic%2Bcell%2Blineage%2Bof%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252883%252990201-4%26rft_id%253Dinfo%253Apmid%252F6684600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(83)90201-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6684600&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RN28500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.46" data-doi="10.1109/34.85659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terzopoulos</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metaxas</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Dynamic 3D models with local and global deformations: deformable superquadrics</span>. <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BPattern%2BAnal.%2BMach.%2BIntell.%26rft.volume%253D13%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1109%252F34.85659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/34.85659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991GA13900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.851.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tomancak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiszmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Systematic determination of patterns of gene expression during Drosophila embryogenesis</span>. <span class="cit-source">Genome Biol.</span> <span class="cit-vol">3</span>: <span class="cit-fpage">research0088.1–0088.14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D3%26rft.spage%253Dresearch0088.1%2B0088.14%26rft.atitle%253DSYSTEMATIC%2BDETERMINATION%2BOF%2BPATTERNS%2BOF%2BGENE%2BEXPRESSION%2BDURING%2BDROSOPHILA%2BEMBRYOGENESIS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.48" data-doi="10.1007/s11263-005-6642-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuille</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">S.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Image parsing: unifying segmentation, detection, and recognition</span>. <abbr class="cit-jnl-abbrev">Int. J. Comput. Vis.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BComput.%2BVis.%26rft.volume%253D63%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11263-005-6642-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11263-005-6642-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229049200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.49" data-doi="10.1006/mpev.1994.1012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vanfleteren</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van De Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tweedie</span>  <span class="cit-name-given-names">S. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trotman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Molecular genealogy of some nematode taxa as based on cytochrome c and globin amino acid sequences</span>. <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DVanfleteren%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D92%26rft.epage%253D101%26rft.atitle%253DMolecular%2Bgenealogy%2Bof%2Bsome%2Bnematode%2Btaxa%2Bas%2Bbased%2Bon%2Bcytochrome%2Bc%2Band%2Bglobin%2Bamino%2Bacid%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmpev.1994.1012%26rft_id%253Dinfo%253Apmid%252F8075836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/mpev.1994.1012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8075836&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.50" data-doi="10.1109/TIP.2009.2033629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanella</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campana</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanguinetti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cells Segmentation from 3-D confocal images of early Zebrafish embryogenesis</span>. <abbr class="cit-jnl-abbrev">IEEE Trans. Image Process.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">770</span>–<span class="cit-lpage">781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2Btransactions%2Bon%2Bimage%2Bprocessing%2B%253A%2B%2Ba%2Bpublication%2Bof%2Bthe%2BIEEE%2BSignal%2BProcessing%2BSociety%26rft.stitle%253DIEEE%2BTrans%2BImage%2BProcess%26rft.aulast%253DZanella%26rft.auinit1%253DC.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D770%26rft.epage%253D781%26rft.atitle%253DCells%2Bsegmentation%2Bfrom%2B3-D%2Bconfocal%2Bimages%2Bof%2Bearly%2Bzebrafish%2Bembryogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTIP.2009.2033629%26rft_id%253Dinfo%253Apmid%252F19955038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TIP.2009.2033629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19955038&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274732000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.851.51" data-doi="10.1109/TPAMI.2007.70770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevatia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segmentation and tracking of multiple humans in crowded environments</span>. <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1198</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BPattern%2BAnal.%2BMach.%2BIntell.%26rft.volume%253D30%26rft.spage%253D1198%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTPAMI.2007.70770%26rft_id%253Dinfo%253Apmid%252F18550903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TPAMI.2007.70770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18550903&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/5/851.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/841" title="Influence of Steroid Hormone Signaling on Life Span Control by Caenorhabditis elegans Insulin-Like Signaling" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/865" title="PRICE: Software for the Targeted Assembly of Components of (Meta) Genomic Sequence Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/5"><span>Volume 3 Issue 5, May 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/5.cover-source.jpg?itok=NJpwBPbV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/851" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hinHFwPy4POkiWotAt9jE5L0tySMCL1zPN3849IOn20" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/5/851" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/5/851" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/5/851" class="active" data-icon-position="" data-hide-link-title="0">A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459460" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LuYsTVXNQyul-RVBeeav24tAqfCjNljE025qCYQmmjo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/851.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/5/851" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qX74-45H1pvgpmP16zaplw6zM0yCsYOT4rhugklYHiA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/5/851" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/851.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459460" id="node459460--2" data-pisa="ggg;3/5/851" data-pisa-master="ggg;g3.113.005918" data-apath="/ggg/3/5/851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel L Mace</span>, <span class="highwire-citation-author" data-delta="1">Peter Weisdepp</span>, <span class="highwire-citation-author" data-delta="2">Louis Gevirtzman</span>, <span class="highwire-citation-author" data-delta="3">Thomas Boyle</span> and <span class="highwire-citation-author" data-delta="4">Robert H Waterston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">851-863;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005918 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459460--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel L Mace </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BL%2BMace%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mace%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BL%2BMace%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dlm19@uw.edu" class="" data-icon-position="" data-hide-link-title="0">dlm19@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Weisdepp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BWeisdepp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weisdepp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BWeisdepp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Louis Gevirtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BGevirtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gevirtzman%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis%2BGevirtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thomas Boyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyle%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BBoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robert H Waterston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH%2BWaterston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waterston%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH%2BWaterston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/5/851" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/5/851&amp;title=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/5/851&amp;title=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/5/851&amp;title=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/5/851&amp;text=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/5/851&amp;title=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/5/851&amp;t=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/5/851&amp;title=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/5/851&amp;title=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459460/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459460" id="node459460--3" data-pisa="ggg;3/5/851" data-pisa-master="ggg;g3.113.005918" data-apath="/ggg/3/5/851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Fidelity Cell Lineage Tracing Method for Obtaining Systematic Spatiotemporal Gene Expression Patterns in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel L Mace</span>, <span class="highwire-citation-author" data-delta="1">Peter Weisdepp</span>, <span class="highwire-citation-author" data-delta="2">Louis Gevirtzman</span>, <span class="highwire-citation-author" data-delta="3">Thomas Boyle</span> and <span class="highwire-citation-author" data-delta="4">Robert H Waterston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">851-863;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005918 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459460--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel L Mace </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BL%2BMace%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mace%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BL%2BMace%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dlm19@uw.edu" class="" data-icon-position="" data-hide-link-title="0">dlm19@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Weisdepp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BWeisdepp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weisdepp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BWeisdepp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Louis Gevirtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BGevirtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gevirtzman%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis%2BGevirtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thomas Boyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyle%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BBoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robert H Waterston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH%2BWaterston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waterston%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH%2BWaterston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=851&amp;rft.epage=863&amp;rft.atitle=A%20High-Fidelity%20Cell%20Lineage%20Tracing%20Method%20for%20Obtaining%20Systematic%20Spatiotemporal%20Gene%20Expression%20Patterns%20in%20Caenorhabditis%20elegans&amp;rft.volume=3&amp;rft.issue=5&amp;rft.date=2013-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mace%2C+Daniel+L&amp;rft.au=Weisdepp%2C+Peter&amp;rft.au=Gevirtzman%2C+Louis&amp;rft.au=Boyle%2C+Thomas&amp;rft.au=Waterston%2C+Robert+H"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459460/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459460/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459460/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459460/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459460/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459460/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459460/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459460/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459460/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459460/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459460/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459460/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/5/851" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Bottom-Up Information</a></li><li class="even"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Top-Down Model</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Decoding the Lineage</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/5/851.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/5/851.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/5/851.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/5/851.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
