<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/5/807" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/5/807.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/5/807.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/5/807.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/5/807/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/5/807.atom" />
<meta name="HW.pisa" content="ggg;3/5/807" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis" />
<meta name="DC.Identifier" content="10.1534/g3.113.005710" />
<meta name="DC.Date" content="2013-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Carter. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-throughput genetic interaction screens have enabled functional genomics on a network scale. Groups of cofunctional genes commonly exhibit similar interaction patterns across a large network, leading to novel functional inferences for a minority of previously uncharacterized genes within a group. However, such analyses are often unsuited to cases with a few relevant gene variants or sparse annotation. Here we describe an alternative analysis of cell growth signaling using a computational strategy that integrates patterns of pleiotropy and epistasis to infer how gene knockdowns enhance or suppress the effects of other knockdowns. We analyzed the interaction network for RNAi knockdowns of a set of 93 incompletely annotated genes in a Drosophila melanogaster model of cellular signaling. We inferred novel functional relationships between genes by modeling genetic interactions in terms of knockdown-to-knockdown influences. The method simultaneously analyzes the effects of partially pleiotropic genes on multiple quantitative phenotypes to infer a consistent model of each genetic interaction. From these models we proposed novel candidate Ras inhibitors and their Ras signaling interaction partners, and each of these hypotheses can be inferred independent of network-wide patterns. At the same time, the network-scale interaction patterns consistently mapped pathway organization. The analysis therefore assigns functional relevance to individual genetic interactions while also revealing global genetic architecture." />
<meta name="DC.Contributor" content="Gregory W. Carter" />
<meta name="article:published_time" content="2013-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput genetic interaction screens have enabled functional genomics on a network scale. Groups of cofunctional genes commonly exhibit similar interaction patterns across a large network, leading to novel functional inferences for a minority of previously uncharacterized genes within a group. However, such analyses are often unsuited to cases with a few relevant gene variants or sparse annotation. Here we describe an alternative analysis of cell growth signaling using a computational strategy that integrates patterns of pleiotropy and epistasis to infer how gene knockdowns enhance or suppress the effects of other knockdowns. We analyzed the interaction network for RNAi knockdowns of a set of 93 incompletely annotated genes in a &lt;i&gt;Drosophila melanogaster&lt;/i&gt; model of cellular signaling. We inferred novel functional relationships between genes by modeling genetic interactions in terms of knockdown-to-knockdown influences. The method simultaneously analyzes the effects of partially pleiotropic genes on multiple quantitative phenotypes to infer a consistent model of each genetic interaction. From these models we proposed novel candidate Ras inhibitors and their Ras signaling interaction partners, and each of these hypotheses can be inferred independent of network-wide patterns. At the same time, the network-scale interaction patterns consistently mapped pathway organization. The analysis therefore assigns functional relevance to individual genetic interactions while also revealing global genetic architecture.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/05/01" />
<meta name="citation_mjid" content="ggg;3/5/807" />
<meta name="citation_id" content="3/5/807" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/5/807" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/5/807.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/5/807.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/5/807.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005710" />
<meta name="citation_pmid" content="23550134" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="807" />
<meta name="citation_lastpage" content="814" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gregory W. Carter" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_email" content="greg.carter@jax.org" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_title=Systems genetics of metabolism: the use of the BXD murine reference panel for multiscalar integration of traits.;citation_author=P. A. Andreux;citation_author=E. G. Williams;citation_author=H. Koutnikova;citation_author=R. H. Houtkooper;citation_author=M. F. Champy;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. Avery;citation_author=S. Wasserman;citation_title=Ordering gene function: the interpretation of epistasis in regulatory hierarchies.;citation_pages=312-316;citation_volume=8;citation_year=1992;citation_issue=9;citation_pmid=1365397;citation_doi=10.1016/0168-9525(92)90263-4" />
<meta name="citation_reference" content="citation_title=Data Reduction and Error Analysis for the Physical Sciences;citation_author=P. R. Bevington;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Boone;citation_author=H. Bussey;citation_author=B. J. Andrews;citation_title=Exploring genetic interactions and networks with yeast.;citation_pages=437-449;citation_volume=8;citation_year=2007;citation_issue=6;citation_pmid=17510664;citation_doi=10.1038/nrg2085" />
<meta name="citation_reference" content="citation_journal_title=Journal of biology (Online);citation_journal_abbrev=J Biol;citation_author=A. B. Byrne;citation_author=M. T. Weirauch;citation_author=V. Wong;citation_author=M. Koeva;citation_author=S. J. Dixon;citation_title=A global analysis of genetic interactions in Caenorhabditis elegans.;citation_pages=8-8;citation_volume=6;citation_year=2007;citation_issue=3;citation_pmid=17897480;citation_doi=10.1186/jbiol58" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=G. W. Carter;citation_author=S. Prinz;citation_author=C. Neou;citation_author=J. P. Shelby;citation_author=B. Marzolf;citation_title=Prediction of phenotype and gene expression for combinations of mutations;citation_pages=96-96;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17389876;citation_doi=10.1038/msb4100137" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=G. W. Carter;citation_author=D. J. Galas;citation_author=T. Galitski;citation_title=Maximal extraction of biological information from genetic interaction data.;citation_pages=e1000347-e1000347;citation_volume=5;citation_year=2009;citation_issue=4;citation_pmid=19343223;citation_doi=10.1371/journal.pcbi.1000347" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. W. Carter;citation_author=M. Hays;citation_author=A. Sherman;citation_author=T. Galitski;citation_title=Use of pleiotropy to model genetic interactions in a population.;citation_pages=e1003010-e1003010;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071457;citation_doi=10.1371/journal.pgen.1003010" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Chen;citation_author=G. I. Mias;citation_author=J. Li-Pook-Than;citation_author=L. Jiang;citation_author=H. Y. Lam;citation_title=Personal omics profiling reveals dynamic molecular and medical phenotypes.;citation_pages=1293-1307;citation_volume=148;citation_year=2012;citation_issue=6;citation_pmid=22424236;citation_doi=10.1016/j.cell.2012.02.009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. R. Collins;citation_author=K. M. Miller;citation_author=N. L. Maas;citation_author=A. Roguev;citation_author=J. Fillingham;citation_title=Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.;citation_pages=806-810;citation_volume=446;citation_year=2007;citation_issue=7137;citation_pmid=17314980;citation_doi=10.1038/nature05649" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. L. Drees;citation_author=V. Thorsson;citation_author=G. W. Carter;citation_author=A. W. Rives;citation_author=M. Z. Raymond;citation_title=Derivation of genetic interaction networks from quantitative phenotype data.;citation_pages=R38-R38;citation_volume=6;citation_year=2005;citation_issue=4;citation_pmid=15833125;citation_doi=10.1186/gb-2005-6-4-r38" />
<meta name="citation_reference" content="citation_title=Gene ontology annotation in FlyBase through association of InterPro records with GO terms;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=J. Gillis;citation_author=P. Pavlidis;citation_title=&quot;Guilt by association&quot; is the exception rather than the rule in gene networks.;citation_pages=e1002444-e1002444;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22479173;citation_doi=10.1371/journal.pcbi.1002444" />
<meta name="citation_reference" content="citation_journal_title=Differentiation;citation_author=N. Harden;citation_title=SIGNALING PATHWAYS DIRECTING THE MOVEMENT AND FUSION OF EPITHELIAL SHEETS: LESSONS FROM DORSAL CLOSURE IN DROSOPHILA;citation_pages=181-203;citation_volume=70:;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. Stat.;citation_author=S. Holm;citation_title=A SIMPLE SEQUENTIALLY REJECTIVE MULTIPLE TEST PROCEDURE.;citation_pages=65-70;citation_volume=6;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. Horn;citation_author=T. Sandmann;citation_author=B. Fischer;citation_author=E. Axelsson;citation_author=W. Huber;citation_title=Mapping of signaling networks through synthetic genetic interaction analysis by RNAi.;citation_pages=341-346;citation_volume=8;citation_year=2011;citation_issue=4;citation_pmid=21378980;citation_doi=10.1038/nmeth.1581" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=F. D. Karim;citation_author=G. M. Rubin;citation_title=PTP-ER, a novel tyrosine phosphatase, functions downstream of Ras1 to downregulate MAP kinase during Drosophila eye development.;citation_pages=741-750;citation_volume=3;citation_year=1999;citation_issue=6;citation_pmid=10394362;citation_doi=10.1016/S1097-2765(01)80006-X" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Lehner;citation_author=C. Crombie;citation_author=J. Tischler;citation_author=A. Fortunato;citation_author=A. G. Fraser;citation_title=Systematic mapping of genetic interactions in Caenorhabditis elegans identifies common modifiers of diverse signaling pathways.;citation_pages=896-903;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16845399;citation_doi=10.1038/ng1844" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=R. H. Li;citation_author=G. A. Churchill;citation_title=Epistasis contributes to the genetic buffering of plasma HDL cholesterol in mice.;citation_volume=42A;citation_year=2010;citation_pmid=20858711;citation_doi=10.1152/physiolgenomics.00044.2010" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=B. A. McKinney;citation_author=N. M. Pajewski;citation_title=Six Degrees of Epistasis: Statistical Network Models for GWAS.;citation_pages=109-109;citation_volume=2;citation_year=2011;citation_pmid=22303403" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=M. Michaut;citation_author=G. D. Bader;citation_title=Multiple genetic interaction experiments provide complementary information useful for gene function prediction.;citation_pages=e1002559-e1002559;citation_volume=8;citation_year=2012;citation_issue=6;citation_pmid=22737063;citation_doi=10.1371/journal.pcbi.1002559" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=S. Noselli;citation_author=F. Agnes;citation_title=Roles of the JNK signaling pathway in Drosophila morphogenesis.;citation_pages=466-472;citation_volume=9;citation_year=1999;citation_issue=4;citation_pmid=10449347;citation_doi=10.1016/S0959-437X(99)80071-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. C. Reifsnyder;citation_author=G. Churchill;citation_author=E. H. Leiter;citation_title=Maternal Environment and Genotype Interact to Establish Diabesity in Mice;citation_pages=1568-1578;citation_volume=10;citation_year=2000;citation_issue=10;citation_pmid=11042154;citation_doi=10.1101/gr.147000" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=M. D. Ritchie;citation_title=Using biological knowledge to uncover the mystery in the search for epistasis in genome-wide association studies.;citation_pages=172-182;citation_volume=75;citation_year=2011;citation_issue=1;citation_pmid=21158748;citation_doi=10.1111/j.1469-1809.2010.00630.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Segre;citation_author=A. Deluna;citation_author=G. M. Church;citation_author=R. Kishony;citation_title=Modular epistasis in yeast metabolism.;citation_pages=77-83;citation_volume=37;citation_year=2005;citation_issue=1;citation_pmid=15592468;citation_doi=10.1038/nm1161" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Shao;citation_author=L. C. Burrage;citation_author=D. S. Sinasac;citation_author=A. E. Hill;citation_author=S. R. Ernest;citation_title=Genetic architecture of complex traits: large phenotypic effects and pervasive epistasis.;citation_volume=105;citation_year=2008;citation_pmid=19066216;citation_doi=10.1073/pnas.0810388105" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. P. St Onge;citation_author=R. Mani;citation_author=J. Oh;citation_author=M. Proctor;citation_author=E. Fung;citation_title=Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.;citation_pages=199-206;citation_volume=39;citation_year=2007;citation_issue=2;citation_pmid=17206143;citation_doi=10.1038/ng1948" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=A. Yamamoto;citation_author=R. R. Anholt;citation_author=T. F. MacKay;citation_title=Epistatic interactions attenuate mutations affecting startle behaviour in Drosophila melanogaster.;citation_pages=373-382;citation_volume=91;citation_year=2009;citation_issue=6;citation_pmid=19968911;citation_doi=10.1017/S0016672309990279" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput genetic interaction screens have enabled functional genomics on a network scale. Groups of cofunctional genes commonly exhibit similar interaction patterns across a large network, leading to novel functional inferences for a minority of previously uncharacterized genes within a group. However, such analyses are often unsuited to cases with a few relevant gene variants or sparse annotation. Here we describe an alternative analysis of cell growth signaling using a computational strategy that integrates patterns of pleiotropy and epistasis to infer how gene knockdowns enhance or suppress the effects of other knockdowns. We analyzed the interaction network for RNAi knockdowns of a set of 93 incompletely annotated genes in a Drosophila melanogaster model of cellular signaling. We inferred novel functional relationships between genes by modeling genetic interactions in terms of knockdown-to-knockdown influences. The method simultaneously analyzes the effects of partially pleiotropic genes on multiple quantitative phenotypes to infer a consistent model of each genetic interaction. From these models we proposed novel candidate Ras inhibitors and their Ras signaling interaction partners, and each of these hypotheses can be inferred independent of network-wide patterns. At the same time, the network-scale interaction patterns consistently mapped pathway organization. The analysis therefore assigns functional relevance to individual genetic interactions while also revealing global genetic architecture." />
<meta name="og-title" property="og:title" content="Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/5/807" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput genetic interaction screens have enabled functional genomics on a network scale. Groups of cofunctional genes commonly exhibit similar interaction patterns across a large network, leading to novel functional inferences for a minority of previously uncharacterized genes within a group. However, such analyses are often unsuited to cases with a few relevant gene variants or sparse annotation. Here we describe an alternative analysis of cell growth signaling using a computational strategy that integrates patterns of pleiotropy and epistasis to infer how gene knockdowns enhance or suppress the effects of other knockdowns. We analyzed the interaction network for RNAi knockdowns of a set of 93 incompletely annotated genes in a Drosophila melanogaster model of cellular signaling. We inferred novel functional relationships between genes by modeling genetic interactions in terms of knockdown-to-knockdown influences. The method simultaneously analyzes the effects of partially pleiotropic genes on multiple quantitative phenotypes to infer a consistent model of each genetic interaction. From these models we proposed novel candidate Ras inhibitors and their Ras signaling interaction partners, and each of these hypotheses can be inferred independent of network-wide patterns. At the same time, the network-scale interaction patterns consistently mapped pathway organization. The analysis therefore assigns functional relevance to individual genetic interactions while also revealing global genetic architecture." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
    <title>Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"THWzx1FK6hg5hP8nWZW6xYdAMPdvIri_6Aa321DxHt8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459468","apath":"\/ggg\/3\/5\/807.atom","pisa":"ggg;3\/5\/807","ac":{"\/ggg\/3\/5\/807.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/5\/807.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/5\/807"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/5\/807"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/5\/807":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qK+fE9tl7hJjjIJydRpxS06\\\\\\\/\\\\\\\/moh687wAJfc11w8BeWZuPS4c3IHSz8b7Z6ht0uUHnwrD7nJsQMbwvMlsMwmo+vHjesamQEBkCrWUAFVfPnIvL6RHBc+9D5pClmKbndBUOWhpZhvHLkhPxp+ecrzXT0R6dhe707CtR8nKT4fPPVD5hAkEaCQqt5VWb90YbCEfYMyUc89IHjvv\\\\\\\/O7w6cf+u\\\\\\\/3DBezbG7ssnSVeOrD1YkqRMdI6QKTtTpMz8uMYwgPNnSztD8diE\\\\\\\/41PGhwYCc4lhy2nrRGK6fB8UnDnd4t3ge19TD8eCEd2SAffLIBOsCdJWVD54hcXPWI7QcA5KbYboRStGWM1RXJQs1ZxBHZzEbnm5+Vh7aNkjMZzKTkIqo1A6zV22QNqzPNU0xjATJK6UlfRRFbw+0sA3skk+Qbrt4HXylHv5TgtsZTYK0AzgNPhiDI2UbKRts6vRf4RxrICk2jQKoC02iPVT4wk0MYWbz8FzTMwhWRpM1tf5K9E2j40mulF6NEEPudU5RGYXUnBi2xBZ40iZr84P9FwUFMXZ9o\\\\\\\/dY\\\\\\\/V0NWuV6EH\\\\\\\/XBqJl183VghF+iiCVtdCtPew8YKjSQb2oqfMWr6cHanY0dJ4pVnsASsn\\\\\\\/mDAllCf\\\\\\\/yXKd7K\\\\\\\/jX6KqDW14rnFs2dTAw+omTAzYEfg2prQMVa\\\\\\\/zdgDKwly+zQQCavtSM57H+9JOtc7FBvpAokBibvQgGjB\\\\\\\/AmbJvEeQ9zWzq0pP2+Z\\\\\\\/akTpX8RAFSSIBOGgbOrmhJ7BsP8OofTv5nVGNo7pEqsjk8yyaT0Aqw3obKcpLqjtVN6y+8ZxHddt9fWWyyfPV0lEZttDfcRyoIKlq35\\\\\\\/Sx7wfWth0vU2VNZ2mbgxdrX6xaXS3kHWDpqvsNHltBZtvDUC36xkxGNQxXKc7h9Bn0UoWWcxS+iaivEngmN2iupco18He+iBeylpiU2p9piIhq5MA7d3kPow+WkWVj3Va0nFKiDGaCTnlk3qOPjOrBA71tdQL+N9oeMOGoqX6h0kUMhC+ssQ5\\\\\\\/f+KPxQt9HesTM1YrhfGwmy10EHfwdbyu1YWK+spzaxemfcHHXwIj4xdLJ7mKmhu36XO+v7sd\\\\\\\/0W6FFMgwaJSUP3bLnCNzHzEFs9QWEONbwVnGUtpAEf7pmU0JRJFoAA4WNRKfUk3B5jaaF+YPXYFQ5i+iGZK\\\\\\\/4fPcD7Nd3DqK2fz2Jev6Tpqu41w2mBYqn9cNoKLFcfFp1DH+mgsZFNvC1f8Le+nJCH9BjKjWQviN2dsrw6mMYLYLXREc2\\\u0022,\\\u0022iv\\\u0022:\\\u0022bhmCWsPGGew18w9y462SNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294d3fcbb577e2a1a2b09d56e521f13e097956e0942b0823d88925e56738581a1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LGmC6qfZEKx5F8FnfA5gVZwSFXQwMN\\\\\\\/JFgErzunCR89tuMkXyliSIC8iuo3QZwkxrT8jv\\\\\\\/sRzmDRtPMY\\\\\\\/Xou+iuHN5U6A1V7nJ0ov0cT8PbnVMIgomzCs1YScB2aW4ssPxFZBW9Xr0cX5M9h1ERHbbOTmmO3GDaxwbAJ3hZQpI3nNEJC18I6cpcbVa\\\\\\\/KVZE8lVDQrY0cJQMXHQF7lTy6lvUwrOpKKxXbYbeFYdB\\\\\\\/sDpFKkIi7HmK9PoLChO29tatjg0WG250gBo3mM2uTss2HFweaANZ3wbVl5LtnK8Ve832kSP1j2UJvDGS0nEUz5defhSQgC1PQvVJmcTxCQWTVhId0Mm3DjbXo7+a7wimxLtVjAQzurjBI7w+GqbgiL9R9BRKBU3fpCs1XwbOhcgF\\\\\\\/12llK74PhtRMLS2n1Gyq7xZhb0lxZbiMzCUiBnZYRkFIaujDE7u9Vt+HN7YnqFV4pLZqcOjGVFZhXzL1riZn+pRJFVnDc1oRE90FQGnSwfppXvN6AvQs7inj3SuWaREsKvXe+OG2f2+M8VHJSbqS\\\\\\\/jMzppVpqRwyJWqBKU3H+ENgfxACZkaNE6tpL+EIA\\\\\\\/mZVSzE+ebR3cjru3b8j\\\\\\\/aL5DmNc+AV5rxIpkJN6YIeBQRLh+CfdsIGzQl98XvFf1yisl4OvyklKY5IMG3kd5TVB0xH1B+qzKPN3wmfXvhAvuydePNxc2efvG7\\\\\\\/RZb0mEy0\\\\\\\/6WcVePJbl37RoP8JxMGUfJqyX4IgbZCQt5vIOmnIvVbQZVJSE8x0CEendhKi4lE2OCPa2fb+UigxH8BrBwmV9n1h2EASnwKLAf81uzhaoSZ0UxmOfcCjlSTwMYkIp7RICi\\\\\\\/4n443qXfk7ipf0LZYRTNSHQldu1XN1STfuUu7ktOeKA20Yc+IU93bi9P4mmLyyKBbKUyjPIjaresERkQq43QTJMBWbcbZg16iysMo4T6lvab+rK0g62rkQ\\\\\\\/e7GoSJJ\\\\\\\/\\\\\\\/ISj5J6E+H3+zjPDYeArdZld++677lj1Cw2A605Emi8s8WxxzZUp82BD9GlFG\\\\\\\/CdwxLcIHQbQMv2XiQ4trabE8dUPDvkf5WfphoG5N7yicIXnSWgMTU0EAGOGX2rB43PtS58cZSHum4Hqz54Rh223J8DIxr8CnkHfT4wuvv1Sr0ITQwc8\\\\\\\/6+r6gdz7PKKJEhBbZDzZB5WfUqCfX0GI5Zy8EdqwukhoSVCZZCjxjHf3NK\\\\\\\/Y40svogb6ItqWPyeN+e+Ol6pMX71MuzFZCkJZCEpJuski95WCfCahV2EOM91ngKLw6iRBHxT29uYyrU7dD\\\\\\\/ZTe9hRjVLFcQP+QNObNpMgvGZOIZbojaTzQXpkmOxTYZAXQLSz\\\\\\\/9ytpGZ2flTY7dM4ElvINixtEYwdIvfQ+QQOtNIggblGWIueGB97FpKNi5PTZmAUpsxfnDhQcw97ZR5gPa7DySBLXzeCCC1qBKX3BSHGDKGDDBt4qkPM3jv2cPVLllbUXCqoMe7lnRe3A3qlRkNP616eRNPmRXc6N\\\\\\\/Ft6u3KEw\\\\\\\/T4RizG\\\\\\\/DXM5W+tmig2pNG98kByzpd3jDbbl7UPLRl73GVjdX4+KZYWfH2emfegnUFLGz42SyCHnDriXV9hZ1\\\\\\\/KGsEtFW4WnffZYELfcG7QEkSIXHHwvMvTssM6DFic1dDoj3l1nYISSC71MxYZLKaUaU+dbn37Yv4XNFtW9zwRmBJ9BZYVcI8cqK9qIamM6AEP0xzQxgDd7svKhZvGmXPw5R8LNFsEaF1dNcBCFlHFKjkTfGOTcxa4oD1jxRj8BSr3lGTp5RVyq8lkUbK2Zv\\\\\\\/b6HiuQfvcMVSRLvA5Qq+F12Y8yCxshLRSwbplxMYAYonXnDxp3aFOzX4wE1j\\\\\\\/OR8+bWx3\\\\\\\/c6olmtuhBA9CyQIyUPtJ2GNuNNDsErgiDphjkF6vLdSTIFXSzC5wg0DsMJm2twVFn+vcIxidMPNN+T2eNZQTU\\\\\\\/aTNtqwFvxqWkdrIRywVe1iLzl7kg6QRGyWOCDPImb144mb4FcHSCz3iX\\\\\\\/\\\\\\\/mNvJB6H2FTaZttbgbkqDIkf\\\\\\\/Z39pwK2ytSu3rLjzlRoJ6kF0qmhLgi4Pmc9moenn5dXeAFLGRt2NY9pGzvOpmX3mT0rPnMf+LvBZToHREDoJ1utz9+ke6NQRZ1SdRntysbM7IIe01AfQJHa+BixDO5H9p9xNM2ksHV8ulD1uHEcFb+qRKSpxsunI3Qoo8ZSwSusaAhiHbabNgCbdfu0q3HolWuSH+hkBchZJYaMtxc+1XlDi0hK3mzpvC9kY8hDv1W8pUb3UcY9lbrozJgEyppzb+muRFrRDPV6WCB11AGQl9SS56nFT+\\\\\\\/FBiFq8nFpdDtgzu9AtRe7KchxLLi1k\\\\\\\/LmjJxLL5AputAAWL2osHEVcAD77ws6eFBmzkeNIHgnmtr7i6Kd1u7aPOxTIFGFG0uc+SWcPIWkGGexjLE9daTb+8VtY7CgRzlHA0OELcqkgDr0zfiGLi7toeTls76FUAKObPuU2XQBFMWVqs62kRWqsEUyyaH4LdadqlUQIzT2SbVUQJrYCcir33ta0lhvrdnfYMjMbdUAu\\\\\\\/nIMaMefnSNgA3g5hsaa5I2ipr8c+GYfiA+jrYuWFwcc\\\\\\\/URg27YUrjHx+HT83Dwl6GSx1uRwHIV+LVNMC8cz6hbLhG6Oj1Oxhlessy9KI8LurF8tohm\\\\\\\/J83DyJ1RnyjTNYz8GKkGyIfTqOu\\\\\\\/cV1Y2UbDj7WAx5Rq4jsKxE8PCGDB9VS9C4PcBPXl3JZtP2LzVapqQU2Txep29vWpmTwGJ3OAScVMbc+\\\\\\\/YBZMU1\\\\\\\/BcCv0+A+sovjecNMMOtA93LsScGBeYn4ahOr8aTpCc0PCvJ2I4ObKsvvFaHm+C\\\\\\\/Bkd+82UE86c+pJ6hoDiUELUAJpFhh02cGa1sIuJv75LlraAcpAA2+TLXFU8ZLZ9JorHPE\\\\\\\/byImCGu2EtXJQAujGDD45uArnT0V3TMpbCI84KOwUV1kE6NkQsozekVONLrphLhHLfz2WVQ7ureoigLjQ9cAK8\\\\\\\/wMu+Tlv9AcZBwVrx6OL\\\\\\\/QMoHuFaHJX3AUsg0Zl+8VNOqTvO62AMo4mpZYESeJI1\\\\\\\/YIuyotieOVVcjJqMRINv1vWAE\\\\\\\/H8DyOh7fcbie3IoOo+NGCAWoW4RljpjYzY2M5aNBUBum3EpxE6jFlN8\\\\\\\/Cnx+wrGj66XaT+FMcfnGUGORtJswS+uPMa3MXGpjqMZxbWFNEJ3NcNnmFxFAqCm+4fYmlKNckhOqSUoJkt\\\\\\\/B4ICOFgT\\\\\\\/CN4QjKp9j75AeNIxD38xnuO62A8nl9sm8HytifpDLXHof+C3fCA\\\\\\\/\\\\\\\/uxxZ7waKpmtaz\\\\\\\/A9NrinGVO0Q82UfYJMo6wFjJIfPPHguu0+v66lQxMAWYYvOqOHA9BooPyl2uhW4U1Q+vJgyE79jBKz74BqYvZZ56ODLWcVh\\\\\\\/XFM9oF8PoxQsaSVyRFX+oniUAzgMXZjIv6QHVmqgplxYkXoTyp0wOgcKUHdhpqgbyPUb04wNcWW5WctJBDQ+GxdEJ7j+9YWOY8lOcsWwTCXE58DAzudtEkSpkAnvaDtQy4mo2aVAqb7dKcmnPEpqRvH2K3Pb+NUupBR\\\\\\\/2rMNTLZwrqDP9mRoYbxw6AMh2Y74p2uY1zdFanED16wc6nBP\\\\\\\/+9zvSgXWK0aEXbSJV3Zyw0zieWkaMsiYQ527q1QKYKDigeB3EZzJkveaPAJmnFJvx8mbl5Do91tQVzCSe27IIrCrtJozOodmFbpp2tqlHP5xi+DRzmNbNEVX+0gY8TJDD0VayQGJvImdTpl32ZJD1zBxy0hlKHPqu37+LobSpK53ZkXqkRHNRMx64YZwdHOe3CIC\\\\\\\/XbnVGHaVmg7hVOBiPFmR4uuYFgTvpRJhSb7\\\\\\\/wzfHl0Z8E9M9HGhU+nhWimVMzxoa1iH8HiYpkoy8fPzUmmaYttAnQ2Zhvnrbs9kr5tIvptkJk2kajEG1mw7X4COvAKlBMBDMFTnzBBUaeNKqik6x8H1gCvwANx7y9rPKvzkhJwdetbf4yTQQZOTvMKn9noroOl1whbRUHaZkoOkOsk0z\\\\\\\/NKTJsAc5mEKyfSEad2sjCnkSN7Dnbl+FPJ5KBZCAMrNym1EcYtZ5p904pqIDeBQDc0KmSmLVipp2c4ObXuHkHXEdOPknUIwka1IhTxEi\\\\\\\/ywvP\\\\\\\/52\\\\\\\/pZo0xxewf0NEBdZD6FnekrPQJc86Bewj9K\\\\\\\/ZY07Kpvhhpvuin4\\\\\\\/qsJ1mZoX1fKSYeybkXvme\\\\\\\/PRchPGAavQGBVJTb8IgTphbps+VRQFxMEfLZLgYRnUhjJNNk8m5+Kylba2UjVFALPhwPkvpmy3EU\\\\\\\/lUpb6YcHzQsPeC8Xcpf8uN6NiHMXfwm7h5V6QvR+WTPCy7xTKWZDfpMsiysSbs1w4GVHqm9QiPxUV+Yh96Y18JEylVKCkbe\\\\\\\/zX9EA+V2hBuHbs\\\\\\\/y\\\\\\\/f6X4BKAwLqF9GFH6OBqtc2i9k8vfpK1BU04Gi\\\\\\\/0iRUqLMtq7pOtyGGM0+CJvzCO7N9NTDofBmfcqTwigoWbKfR342dLfOra16fj2Ffzy8ADSqs6s\\\\\\\/RcWHF6y6oNkH\\\\\\\/oMCGEhxxx7\\\\\\\/5ttfh3YqAjRzydcSmavGHbGyyCZ4ubse06D4zy4ck0zAC4IjhbSnLVEv7Acb2K\\\\\\\/TdB5YhODERvcmaLi4\\\\\\\/DgQZsKh0Q9rIMzZZveOKbs3cujZAqKv6502vwxSO5E5uN4HBfaxKimuzCX9wg8t9nLcCosCo2\\\\\\\/RqsB2CYzgRK3\\\\\\\/8Aa84MU1fgPZYSNDbAk7M\\\\\\\/d\\\\\\\/uHrav\\\\\\\/3+2zL\\\\\\\/iuKv\\\\\\\/dG4nE8l0NRCMrZeTpUvyUJQtJd+Ll2jCCFaaDErQmzqGc7gzGByPtec3ZBRgnbx\\\\\\\/MtjJ6Y\\\\\\\/zxZRvaYDIdPqb\\\\\\\/M0fcUQU0xfkzex6SJxqG5YNQYhzokBf2EYCQyIoHDypt9QfUTm2vTM75ui8a+gdBvm9IC6I2IoyCCfbiHcnOBIIuD78PP8q6DICZ5TuOEl8um7jFc9Oo\\\\\\\/l0vMQXPnMFaKPg3jqpGHP0sYBmcgfd1k5EdFDp\\\\\\\/FZ\\\\\\\/H+BBmmDknk1V0mowZse5tKeT79q9s2msFZKKjg1l9B4iZygz14os70t0JZnBfqFbLcwz02ZF1mEwWu7xLb06T4OY+pvyRzmDl79Gumvy+wWtgMlaCrT+FV+HPSL1E\\\\\\\/i5+37KawL9zLJ9iTdDXD+ZwextX+nEYT6MGQvZMvW+gcfIJcNlQcfsN4OC6J7uw1FtpKY0Dm6OqTP8hxFmS3z49jx7EAq5nUPbIxgSwVjXvcbOkhc+NWo2bx1TLQpqDsLNcOmRvOS4kjifOX9JOwl7aCvQq1umGH4R3X37yv\\\\\\\/J5hb82buGPkYLglyrfyOLuIz13OHLPVYkb+yjlFSzU1IM\\\\\\\/NotdfhwCK9O4b51cU\\\\\\\/Sl0TKCJw5z26KW1qkxkIQ+t1BNkkgHCbZ2K0a0G0\\\\\\\/\\\\\\\/SFix3ncy8cVQzKlN0CJVmspsDsuiZvxy5bxVeluK0GtVGASvD2g6H5\\\\\\\/S2xcKe3R5BPhM2T7WUVv2rwsEgBpxEjWBiFVxSyRHQzZd2CQqS5jerSgJDqHAlm1egj3mGrgpk6Le6bRxX2XsIAJEaUPGc3lWlo5WA0zD5TxoFqXFb4fz90sRq5F95euWkSM5To3lzXgsLPMZ3laqP5Vz15Ko0iPyPU0TfLqp4sA6SJMW3FgW5x\\\\\\\/GFZtoU8JmxhtLeqypNC1H7g4bRseAoX+KhQEI4d1kC\\\\\\\/VO11BW1an9DzDRfyJYbzEfkNCr26cpGYJ0Tl5UcrXZv9w7GtfsVBpzidAvxrFmY9EavdC42aVMhdOn8sj9XgyJfHdZiazfpVRDc3T57ZfDWtSjkGbwUA6Bi6Fg8fNH7kZ6MaJ9TzumYGtqeFkdCQp9APglt8MnF0mg+8c9uuvVN31JnjvKlcWAu8t7RJhRDBGmfgW8JfCnFX4NjFZw\\\\\\\/HtpIBJ2gRd+ZsWsJcxE7xhQrg\\\\\\\/OvUqZpGHuLOf0bqogkysrog401qLGLoQkmGNtAC4PBO0G0ILZ4CV9iTy4YW7nSTQXO0ax294YAVY8mnHIwU6lVCNBuhat4NogG0O4co+1sb4c+9IgIVP5gKCie4TA1VLArZstnamZqioh9sk5QOM5lfyrdIW9xTPEhDRroMwQJR4QWWSCX+D3WCoxOkI\\\\\\\/n7QOegqwQlZBDctb+Qcz34bD5++VZFhSx1aC5zer12gwBIAzHavf89D4ccxMB1\\\\\\\/MvKrqN8Lr58zyz0nVjkzTW3FHjkE0rIvfds2t\\\\\\\/QQt1vE8o4xqhJ8zM3gBnPtWzxPND9LiKHxDxq5Clfdlb1bZdAu3e9Y\\\\\\\/UXfwvEgVHv09cDazV8bPvXQBV6WDQCEusenG6B1r9O2rDEMg5rawA5pOtYnBlmyXTsdIbhy92UEcOimvwTwaJatze+yYALCka57IObtIXcH6QFgSPXMq9U3f+mvhj\\\\\\\/8ZjZGKFMEea8n3bmRKFhew4Ffs5rdIkO72eYtLRlJ2ey+RTJ+7dVfXFTtpf4WUE9nkZbJMd5LpmsA2sYUkV3vZwSZvg+8z\\\\\\\/W31jgKxHMoYR58f38XXhigZQW1i2vnsbLOAAFY0XQQvyhHsE\\\\\\\/SN+5TWF+0L8FgS4R+lHQCz7rnLaVerpAY6+7E7tvknXPlR1RGx5Q0mm25jkX\\\\\\\/xk+x2O0KDP5\\\\\\\/mL7oBV1zkYRZgb91Q7Rhmc0iqX+Lql6jfmjxH+aigIDqQqurnIEG\\\\\\\/+YdJ+HvrPn3sgAnWNTOG\\\\\\\/c32A\\\\\\\/CEOmnddBfp3g6\\\\\\\/xcSMV6fOS83zTNF\\\\\\\/mVLU+gmqrUXBTgDRj+lTAglcpAzNuXHqqwLujmju+\\\\\\\/ZhtqlQbTf3Z4bjVDm6371WoD5yxE7CUYuMaIS0pkSmujBklEcb0lx88KEdQDMqnSTwiDsuOq7xTJka3PBoOhF4j0IIlifVVXLkI1nCJJjJJdvrGUqzFWtikvAbx\\\\\\\/io2n9Z7Wi7w0cW1FAKIwSSyee2aZ870aUMwvVIBrvRYr\\\\\\\/d4Tn9GL\\\\\\\/Uk5r7LwycdVi4rhVgvyLxt7zXGKBNeI7tVMF7H\\\\\\\/a0\\\\\\\/5rDbTXC0YXyx\\\\\\\/oVyuzt7Bmnu+HKZeZx8N\\\\\\\/O\\\\\\\/hYRwlVm3fJ3r9q8gNSmtYgDGCx2JMDhmF04p2o6svgR9mh1Rfs5Z6z2zfQ4GRYA6P4uMNyGlj38ensi40Em7oU1lln68X5qcBvBfjeKKt11x7ZeSCl9cUNYCkV1oaQL33F\\\\\\\/f43hNESjoYxKO9xv2I0Jkj4SFQjo9C4Y\\\\\\\/H7oHyluia3XCPaJ8X1E+pS8w2j9H594+5k\\\\\\\/fcukNekACot8BnWUt8xUv9zVxeRZi4bliBRjBM8joIKYm3yslTtTABcykkD6ANBe3KSNHh8s+SjRnFj4\\\\\\\/DzBOqWyAJLmxC\\\\\\\/qTztPM3K4VT2egXU\\\\\\\/MoO0J4YyrFJKVyV42RWjgH86Iklo9MWbaKIfaiJhqmrOC3flKumEYmh2Kuo5\\\\\\\/anJOV\\\\\\\/BlWPXmZN809DeHKDW8Y5UBuXPpTtQiR+t9wyatJ4QCs57l+ctILeGpXRAQb\\\\\\\/aXei3HJn9W17mOcN0obAV5jBB\\\\\\\/CLytgZSCVnm\\\\\\\/5tmu+589wLhzNpKGS\\\\\\\/WcpCmkhzLmoCi4v7Vyyde1zYj8Whd72x7LkTXAP9jBR7Y7ry+OGbsstvEsmAKSFgnSVz5WZKdu3fiXQKlph2zHCCP+eImRdRM\\\\\\\/rf7asKtZgvhrZHJACaVnbL9i1Pt3y\\\\\\\/RNsBWYrSkta0SMTjHndKoujNSaMGb71ZawyClCRVetrfSfC4LnhkwSuUkxTbULH11l0UjJBQzjqGbdCMg2AZ+V5gjm8EjJp9zIc4sRDNUEmWKMcgr43\\\\\\\/G5lzYceYFXRc9K6q9dlIwip8OIqrQncqm8P6CFshzQg\\\\\\\/GjJGOokQtD8z7DbLYvWIUwsB+ux21OjUUE159mwYQ3uiB1wBfwsnD\\\\\\\/YFE2NHJhS6K9Yzvb1j2Zc7OeKSGJRzOxMdIYDKNtIAvun07yJzM4L74ipK4PzACK4pxZ80bAJtxDJzZpqbIierQj12wS5QqCBh+s0OH2oZKBE8mxX0EUrRhgEnxvRwQ2q\\\\\\\/YhKXj7VJaNdMxYT0hjwBuF4o3LYo6hWIhDENax52eTAC3dFccauWIaRTCGGcVYROLxBstzkdDSDD2qxxbg5eQ9ssrO8Pj2ISYCCrVjoD+WSVmdun\\\\\\\/tTzuF+H\\\\\\\/7xfu8cx9bh6rqLbIk3BBTkRo9mF+tee5g46uNEPUq75R3mO9FnSsR2XNboWMtTe8eDSDwtBFQ9\\\\\\\/bItkp+o8uLfeGeb2csVrNHR0v39n447ha+IpmA4t5tAnATSma62T6ZZFWONkeiT0E8tWhxWx7KBtqzF\\\\\\\/mhAs4ciO9BrI6USbT5yM4lSrnLPdBWJmwOpdDl+wvRwPAwj8E19uJjVNDNurzpGejUcBz9sPEAZQibjj0RpLt7J4Vb5ST1iyn7rKfABJA8we1AUbwjHPEFbochmtrhJ5EbmYk2jb2wImyNTIf500n2ct\\\\\\\/2LUYKiZqpTpM4rkIUN2ZEJwqoNOPkY7ZDcqy4QyOPK8qLn6VhW41dfsguhAoZRqFhdrKTa3iIIVBZN7pFcHqp4DBeNapimDFgBzoD8rJbwGjhWrERO0MTb4DzAKA\\\\\\\/1JISco9tncSVppt6DDlxHnhNZGHGU\\\\\\\/H6xXEbwbJHv1z0QZEa3huERPb4sfcllelfuT\\\\\\\/efqio+chcT6HEiK04C0e22O1EewvrNAd75cAmp1mo3nv2xrHLvsFDqx5ScF4kBX20pxcKAWZkunnrEblqhFhaROMBZh97jW2OGF78ifvNcZNOOKm2BjWTGJo8s97nzynySREnBqfwCZBRKcZQZEraDxa4IP1hUeRdvLPwIynLuMHrdPdnaCrBSRlWbnN\\\\\\\/r\\\\\\\/nPzS+EEky1aV11LN7NvX9U7Dwkv4dISf+Y1XDxikchdqsTKJxKKP7lzYNVHnyZKgNAU6AViGlnsJg+zIHvCbEltoZAzgx6YckW1FjEIgNpj5Usg\\\\\\\/4jkLAhZ6hRkXjWGhNXntKRIZ\\\\\\\/MGV14pWabw4aPNKpsoYNy1FSg4arGc9DHFW28Hvl\\\\\\\/3SNKBSALBS2SSb4DE4\\\\\\\/TXZzR0sbutL8hdGzH7Z0Dw\\\\\\\/F35TN4hHocauTFKYXZxvzNthGtDOjC\\\\\\\/k3E6H1\\\\\\\/Wd+ia\\\\\\\/ZgREFpN49jYX1EJOPjPorooqgUDhAXFiRHd9JKZPq1dEE6384fGmy\\\\\\\/Im6mmxKsKQkvtZLNtSz7cBH9xPxUTDrCWmMyE4H7J\\\\\\\/I6tsheqQUz00gLrA0B9umNJ2ZUl1sPrS8vFcmsKoxZhaSmT2PJ4u7ZBXJSQoOzqPNIHQnOFpg2Mlgwcf7mo5qOX7YNHAF9A3N7Ycpoq7UMhb7NRlYO8wJIOm3SkU3JlBruvDTfFjP537faF0FI44zYgb4rOMPRWu+DitRM3Ag1v0QkBhbVJLLqWsPZe+YRHSFKJDk7lfOnkiHlCttvVqd+cFBnRFqn6+tKURjdGYBkA8NK3sr\\\\\\\/PsndKHLAalrc33JikcUFVveahig0jueX6YP0IIm27OME1tQkMDLfMD8Ev4F6BfKeGcAg2HvseSNJNfOl5FXpnqD5504+JTZz8G5qjzQFAhJ43uiq+oGt3wATuEnghpZzwPKySH2PKnvQ+1Wz2qbE0nSenRA1\\\\\\\/71ZMuKQrWY8j5hP6BCRIvTXXVMF27X1f2eOTBiNhvesQt1hEHueLC10Xm44lqSex3BTIE5HWw7AOgaLwbkbzrI56aO\\\\\\\/ckzb1RTHXiq7kWzXNtBNCWwjPZ2NJJOJb+JxcztoNgSyajdAyDQlwGCeJMzpWdv+STMmardENETJ37H4qy3XVPVv3ojc\\\\\\\/KNeobiTgNE2qpyA4Jnsb2DGZXMNvHA5vti3AX8FoQdH0KoQtzgd6z445nTeqww5GQQQQ3tKSnTKo48O5ZT7wwRXds3a52IktWEnw+bRbz+0EGNhXwZ6miN+r\\\\\\\/24N9uZiJ9xyPkDFSPQPcTmektyXE5u591cXPeuiHS9SL4c3VkJvhuuVyqcfIF5GRyIU2S1cSuX\\\\\\\/04TjBY7w6moNbK3o2Sryz\\\\\\\/lFzdws9yR1E7rbDZaKX2UeVrwJFw7hTsj163GjHqmFEtpZAV03Q7o4PclcesS7VTuHcM9bE8GAVt6wo6DxKV2xIdPvpvrGVHj8z95CMlbngY3QKJ5ZwTStAKWPHcREkgDi89e3i5pRWO\\\\\\\/hW2r5KW1GNdp58N69L+k0YIPk5QSNQM2hbrhi2jlUKVxCyX6Y1zUVqls8JbAz72+4bfTkGGgjay75bkbT47p8mqktmAAtCjubwkPg7Xd8NWWspKgOYpN4Px8\\\\\\\/hyd195m867JIortb4kIJlVVYNB1kDYvw7evb85Od4XkR6DtgvbvCqA\\\\\\\/fR3LKjhwPlVP1qbol0TUJswfkWpi4lNKRq7MYvNFMiSQGQpmWvC8qZx1mQdWj9UozThon16DcP11rIvSKVUhKDcid2xQF25I5+92EPi8+sOV\\\\\\\/xx41tiYR6LpmTzrbMPmHth5OTz9E8LU+FjqUmc4iJ\\\\\\\/ClijHyecYtD0BIfSns49kqKm9vmLOHlLSul11nbT4FgY4\\\\\\\/dYdgP078SFnfXTAFwmgAgKFyJBUH+s2NyuliTnfGkWSAX\\\\\\\/VjBaM9TsYAWuhpXRZroQpdA2w+U6hiNkVwmlYOxycWxEPkdIrCRW0CI9lnEy4RL9LaEveWHdu4bZlFVMcVcJIMq3DWxmLupVIK3hS7YXq070dJu91aTsY2hPARDQTeFz9kxrncAD0H0gvNCS5tV2S\\\\\\\/Fcc7r5rcnNs\\\\\\\/6\\\\\\\/3\\\\\\\/uylhzz2Mn4RMhNNmpVENxlSE0jystF1DdntTBdSWdo7Tn4Wjq001+7Qon5niAuO6LTp8ITLkaSYfdnzugefQDVQzXVrQCfme7NxDfAcc9J0yhFfjijZv8E80wkrXnpYdiEF87FJpH5G9NZGTOH4RF3xn9zDxXBa7K9knnkzXFIDjifbG2UwsnAlQmBIQ6P4s\\\\\\\/BsjuOqyxn8jPv3fwn6z0V8m5QJnXrA8uccQAWzFdS58DY2UEwWLDgfVRIbOaqLf6toSaDHP\\\\\\\/Edh\\\\\\\/nFYHvxff2IfMmRett33vdXyUZ+y7oMiT+vxe9403ssjDNH6EbUFaRyVO4KbUssTQOtSH8gEwZfCa8UuBccEksfgCHP+wRYvuWeJ0p+ks9+AOycUqZEZmSXNyBmTJQvEueXrReGP0eMsANhURSUPqTCNRJe0sKn0coniBpyhQhykvUa\\\\\\\/Dz8XqPxd05i2sKDuAn1f3VrV1DDXV5WjNj8+EulCNtAck02NCB\\\\\\\/HamPkNJ9GtxxKwlTU1eX62NLWP30x3VEq5MPzv6edA6G22p5gigmW3n9cPAfx7nTL2gKN4MDx0cM+rDaYpMWmRbTRu0tRXbdo3394xjLtdd4qC2MeJFM2nw2AIL1LA00CrKVeNONMA+Z8h3dWfN\\\\\\\/WCVSxWzD\\\\\\\/EZg2pon79AX8GNSw\\\\\\\/pEnnoCycibfJYXiNjD+ajWXyIerDW5qhTMHu8SfHEQCirgV6a\\\\\\\/ZQZf23I5oJKn\\\\\\\/kRdeDo7oYwT4o6a0IHtiqRWrY\\\\\\\/TD8d0XAqO3CSQ0YIZjOSKPyijUDH7kmfvhEJoaQY7Huh106Kv32ITAiqNW83uMrd79XgSipZmaOw8siht1Y+Q+yV+oLRMt1\\\\\\\/D0lio\\\\\\\/pgOUDWq5x+oZ5RLkYWasf0DJXLxfckMAx4Xookc3ys0vQAgMMCQxJJ5BRKuqlxKfx6Ev2XQhzDneJ+hY3vhFGPsRahPqAA+mhinvihNgkoX4VEgtvPtIerjmxRbRCUcwegmixqwf2mZ\\\\\\\/J3NAj22iVV5mufcqdCRLa4RiIkQmLsw6FOrw1S7+z\\\\\\\/yjV+fW8NpvX1+bKEC+LF0+IDhOg2ZHSvP79pId3CiQ+itPMv0xzjrop1aDVPBuXYggZlFOM8mNFb1VrPFNquxKBGO1NuDm3aOSjPPhdttodgdzzJ9uKH\\\\\\\/VoskefkfnotlDnQc\\\\\\\/cs04c3N1wksdArqRgxmTGgHP2chQoJbf5PtbcQtFzqWfj6spxUEPdB3dUYYDkT59BGGQ54HF0vezNnnluaQobEzJI26GYWmJwtHpScXB4Z3m1HYdvItosbtUH1GBcVFMCgkg6M3ThSqP3BiONbH3RBr2FbdQODJ\\\\\\\/bwXW91dS2\\\\\\\/XHfNYVe9Stwe3Cvy\\\\\\\/UIZjxHjBGM9mT2EstfucXx2zmW41aCazxycqJwlk16xnpDY0oVLuqPEcsERLs7eLoqGEjpYoHmaPHEthftJVM0RMJ\\\\\\\/raqGe9y4qAown8Y3qqJhTaEMgpuMLuTanK3xdYowFu\\\\\\\/wkkEl0ItTeb9S++RWswQLwrzQSLDUqyfHNLdmbhCJH1rxycZ0AG0iGydYLrRBmsWf1jvxd2OdcKf5JEp4s4fF5ZoS8cxGkrleXZy9q7Q72XSBUQEHQ\\\\\\\/\\\\\\\/hUrIh3WWHa5renZv0lBYHLcA7eQBOZhLCjZnt+g77I6JAlJrtUB5Ca2QuA3NPBLl00GNoeyRHO3HBYf5WfntT4o\\\\\\\/15yxutsYEeChoXNR9vVOKwHQbNfl99T2SvaJU0qkY\\\\\\\/QiUQWNpgZw5FDTlSyVNxqfFXY7h5gJmuL34efPxzspkAd7faCEBvyfYiN\\\\\\\/jUjrNYWc+oX9pGKtZ0Puat2nrxQ+6qZBQ32kmn0pQ2EALKt8f1PUPScqTmjUtQH3dsynT6okIVH2m1HoUQORgKi8uDnil0coE3Kfw0NoBJD4pf0FR6AOWGYiHsEOYsXmaW7wR6zpV33uILpHnhtwu3Ga+Ju9P\\\\\\\/TttQ13LbyqEGiLHDWDsqq8i6BQ8DEGrho+z0JNb3+2Dg5EEVPywnnI72LsyKWjM4fnvZqcV7zc3DkPALOPCBlVF57PkMjZ+kPCJ+yH+QZsy9hrpdwISncgcWc6D7+fGaKvHQeF3jITicqYyQCozwe12hcuyrz0d2QFa1LoPKfXdLKTa7YDn5iJad\\\\\\\/AMg4+WMnEBqxvMMpzmFuJEQBWDNorIYB2Psbw9NDznzZEvfvWUA3DhD7cJE76tYtnfU9d69\\\\\\\/eAzfT5ai3Be\\\\\\\/tjGDVwSZJoLDZxtrzHgK2326F+PGdRJlCRuQGdD5VnJ\\\\\\\/nW4LmC3QvXK5mEGvVevDOH5jVCSwoE1\\\\\\\/HziNLUbCCrjb5AanBWHcymeeAmgzRshe01zUBkJObWiLUWBweyExnSGBDBu31eH\\\\\\\/EqCh84vks2PzQnEbRWxHSjWOggYIowzYC6JZ6Vdt\\\\\\\/8Q6j7X+zEVbM8SAKavGIDf2jL5O7+X+U86nA+oGCOPUsJLeutiS7i0ct5CzfcDy+IadPJTgwwKiTLnso\\\\\\\/7xdqF+iRyF3UzXjK5rgiMDCg4byuJwFZEphJEf62Wp0i3de3aUKLpOPZQESvjAmtOdDHZE6auOSjz1VyqcmJDav00wmpdyzPDwlKEJWEjY+NsQdzFGSu\\\\\\\/nRXJnjgnylmcqB6irQp2rsmm4XZqD5jObnpEHObFGiGjKt4Vg1N7MifhTA0w+vEVkEBgGMZ8pI95\\\\\\\/jTcINVPpDqO6hafgWxQGfYGBnjXJQzhaKInJchZ9Zl24gHysxwMlZ8nyNUy7nDwzBoRLvgkpXgIl3mFAgYTUeuvt1HTKAVQE5ebbiA1tZDOU65jvYv8Vg09z++aZLGfYTzIqfvSznzET5TSYdeEj+Af6nBwiZovvDwQvOgpp\\\\\\\/aIc5Jiel3F\\\\\\\/WkbcyLUy47zYP9Wp\\\\\\\/ucam1NMlc7r9ZVwBVTzFV+1W3fbKWX7yAdl47I0Wsu1MxLnph8vKqkj3U7Bc09kIzeOCQ6rqepVcgtcch+6PRIl3ErGH2iVKUBpaIsqu0MYzp8i17uFYYTIfH\\\\\\\/9vPAuwp2zbGKKbOdPRNGBHOdA0eGZ+SAqsjrxmb3N9h1zW4X3E8SiSZ52odbeZcCS161Y7qkTPnUGfeGPp8Vssqcr8rc7L2he3Buk24apEaJ0XQz4xWLWxVIfl+2R2x0bUpX+IJJyheKQAfv41mDbYicWu1ujxEN0xDQ6w+1x757peKPiHZpehuamlzyUedLyMKJZrEI4yT2suDB7VGgPQZo2J\\\\\\\/syZBWDNOM\\\\\\\/lU6LLbIM\\\\\\\/cQNXbfHCQJr2JeYDtCAu7nGE3\\\\\\\/GwD3jdCqIBIrJyjUj1Xcz3fIDfWdDBV8yJs\\\\\\\/pmKPNpqBgyU4k7eEbnty2hXmub0EInHQAcfYcBaG\\\\\\\/\\\\\\\/CVaVhAiaHXhj6i5gtSdy9+BNXktGvP\\\\\\\/+OQFHI\\\\\\\/j7PpEh8zk9c9pCrGsHhRo35NwFFKrmVRHzfEGXpDRaDM+9bue31SkFLKmr9xBR8kRUekn217QeE5oQ5UShbwGivyVZQYrbE+Z50R0HTkQm728y3cV49v1V1AcITrBY4qOYvnsl9k4Xx35AwsFG5YLHpBmBN4aKsvYmSIvv4khIey3UPGVpuuq+FcNKDl1XNZwJdDjqfjlMwA7mm6EpQC00yS7qga+ZFAZkMtTMUJ9ITHOTnSq+KHL5fWN4AJ7B0thQzAwZ13AAznGaDzVznU5aUOz4uwWVaxLdgdqNWfP4pFn52WROBFSZmMf8iZEtlLCUJgb40rgVCGd85x8uIRvtpAomqtZx7lTMsk\\\\\\\/k\\\\\\\/sVX7nRYnckCQceYBZ9PE3d6pE\\\\\\\/itlXC4FwpDPcbKjmhvdMidErrFNQu5VWv05y+xjzCoplimP2Fvcmv8mtv73MNXmZ6VD2MpuDy3HPTbTRFypaKEMqSaGQJThUm7tkDggKZFIKhyHP59MAY3Ct8SK6iP0aP4\\\\\\\/wwROWACCcaKCzNYtxcdVKRWKFtFi3qAndC6QTxLqSGParQTYAD1EfJJfyONAQyamjR1CKrlB6aIKPxt5D7WWDsJ11X2qyYcmyoOdadhCXA8F4rtLAS\\\\\\\/wNMG50xjSLU9toSu4Nwc3qUk0+ThompVNcJmak4QqbfRkIIFfFHiEpk+PYfVBrz2TvhUmtJ65p6KDmkvTFeOjWKrQ\\\\\\\/P6uLSafBa5aQHUEKcvLlbEBTagkOE7zKRb5n1zrPbUByHaH90YEtjycx2PuzhUxMVoSyjd50J6QtEBBkPlWrq3JLUSN2Wj3dFNmncZiHQzYxTXoWTE0QIHIkewy8B8TrDw9wEjKeFqjN5ePFufAZpkoKfklZWb8BgxL8TnuOAyc\\\\\\\/pdZPd9HQGUA2TfA2h6ElWZtJ3vTewJS32N5e2UaJygw0GEnDUegY1imN0AC13wV8xD2s5GskKrm9ySIj3ZuJu304teEyiFNXCXj5LuaFRY7jj2YqB7p+TWPsNnrlYTvVIwXWitLKSH2cIngjou+3dU74I8MjIWj8shzyyYzHPltAENnC1AegIyRyqWN4LrofASvDp1Cuyhxfsprv\\\\\\\/EmxZaBmCSOViyXwSOIdjiHj3LJyB+1Gx6hDGy5KNsFMb05PTCRR5ANCNEyxiMpC4LJNEJFC\\\\\\\/khKjPQmhuJwLd4tb32eM7GKNcIm+pcmybXUgPUdi3o5gI3Lr7FdzCCE3dMLC\\\\\\\/2+iXdwJgi\\\\\\\/oVsZ9m68Bj5BZAbV76gGeLEf7MUub5fZHMd4Yg8qJwcRG\\\\\\\/yb2NF6fyu15yJolxsw9um4lYg80RFEa\\\\\\\/P+JzkRNKIT6woZguK4Wn64a5g0oyX7qcUWzMSrk36KRUbm6BlhqMBDPC8z3En4SK8kJYdnnH2XHDWLd8f93HOyFxHMULTDmDQYAKicrFN0XOyb2UtKawL4MbuPul48OCclhW3wxJyJgfUrFCAl6kxDLRYJJaOVRIQCiux6\\\\\\\/eyhIpTChIaGTazmbhkVCtqLNCG3P6A3lilsyNBM3Lz0Eb59p74\\\\\\\/Z+3boX0gJfKWHAlBUfP4pzziSL7k54AuGchHAb08MjjkJxHzl6noL7IvrUtJfL3XBifsYv60fJJJGnZDHMpoMpQ77qtiKhTXQU0yuPjsevEGnRisEeBaSDaQQsRJLY56ILxN3TmPsAEgznicTd9pGD\\\\\\\/PDpC78A+wHidU\\\\\\\/utztk0kYmX8rxomGh83IkvT8IiZL2WrQXxuQc1\\\\\\\/dMNTG4BWcvBKBt1+L90waAZl0ZZWe3rmz0hqPzAOKg\\\\\\\/6ZjJPfD58ly1Vb1WghVMryZBD8Rskhrh1LScs31xOKpqYTbqGwciZKaVRuUHAHruePjXnJSZnbaGlx6C\\\\\\\/v87tZLYZjmskU5+uBfQ8wEQtdb47mKwb3el3NQSb\\\\\\\/HHMhHnDTpTElbNHlPTE0ldAKYr8pgButYeaEnCNfexZutciwGtC3nxhtQC9ZpRySecL0cNiyjCPxbrQW7+Fy\\\\\\\/9D6znfvtV28LHVgcaoV6IrOYFENKPDIb2OfqHHfbwjk56XhFHyKW9IFTSlqTvHfJxFwLP6la0CZVkpJzTSQMId8fCnY0KXv0F77Qw8WlE4c6y4qV9WMHc+q1SX1B+3strveRXX1rcd68Tg1EeRzGhOjlDNg4NwkSmK1yeWhtavr\\\\\\\/4qGvixA89DxGfS0TTrjKMbK45ng\\\\\\\/o\\\\\\\/OLBDNMgb7AAtsJZPYIkF6lJI0AkBpmkxlB\\\\\\\/VDapwH1hsHUHX9qAVMRFNF3LPzNrLKa5QxZ6mPgPkDe29XWOQa\\\\\\\/13vk+KGV\\\\\\\/jdo4jGUL\\\\\\\/\\\\\\\/vkYR5V+X8QateKZE1MsUyR2y0tL+PyKnDx391rqWJqnDjgpjUgvH2LaJ8KjoYlYpysEZYRe4bxGoos7Dy9xBR2Mp2EmwdpWaDjQKMkxwThmqlj2\\\\\\\/EpwoUI+Rp8h\\\\\\\/TEJxLVObrg4by4dTOB1LH8rfwAKhwouMvJoAu\\\\\\\/\\\\\\\/DsZsUUqR5hlwKq81kQ0PbDqBLLfjNxVmbNIPMMzrridBl9Dj\\\\\\\/6M6s99zwpADB8egOGlu4VbEm7wElnoPCbiK3xyvSx7TKcQhJNVkzB280aKUTaH1K1Q6TNC1\\\\\\\/SGGJwUv\\\\\\\/1d0k4sY8Lc5KVB5C2+H6cJ5PWMqtxfhZkhpgu38PD+eI4SRF7rdkpI5p1cTSpVhgDKRwttXVf4ZNFqOWPo2mz2Pw317SENMaN4bAYstjdu2hH1gFPG6h9MFjNfCRj+eqFRcMHmx8fB95ZqkJwqV9JwCpWsfb5jiOn0LUzreRBZ\\\\\\\/Ykh0YNam+O1Zq5346vTRJvQcIhM557WPs64M24XlFHD9SnPtu+ldJjyegB3DBiXNkQ9mWERgWpT2LM6HGxzK095nDSROtzFQwqLqyqdW\\\\\\\/LC4I7ETbsLvbMcy9lc8nCSWc0kdPWTmo4HrZrI4XSryMByLWerrupHTM0B2e6GZL0PrAy78MTCDAz7g8SFLZ7pbySPlM6Gy0A8gFkO5+MaGkINkOtNLGluiuEgDbT5uGl2xuXvfDAvEiLUUAID0kUlKs2HCimUkIlnQS6E89uMOhsdShRiwnV8NWsyHQsooWU\\\\\\\/g+NWCcUgKkQClcTtAXoy1YHXLFJqTbuspUpUSfeVFZXe5BQ7ueK1xqMT\\\\\\\/6PJLGUR0oIwdDP96CqMomldwD1yazmecg1PKTlVXPsdUg8DrBiByWnAhbaEf+Bajp3NnI41B5W\\\\\\\/cX4rqkE4cDHXp5xk8JTsYfPnH2lEsZCypEZGMC+42U3D0Sqs+1qadGpAUXV+fZ2ctUkmF+9O1ulmqE6AgpZnvCUfsRqG3BsAGnyeJ3g0zDUO8KXeC7jSympO2xR2SW4QoTQMfP5byIozqGkkwCBjqHO7B4aitZA3j4G51s+PlbsQo6IhHHkp87nvHcFuN5Njyp7wsYEZ8cXeLbKP1UZwZAYzgwtCwtJtmdUjhM3f4+mqt26hl5JaHrqiJ2TAcPkQl\\\\\\\/\\\\\\\/13N3Uuh5JnY0iyL+yyKt3\\\\\\\/6D6bvesRWh+24hQSHs+ghuOcj9QG3OVOUXbE+\\\\\\\/9jFAVAtLmr9HHnFZMFy9Rbq9nvlWO+9fzXdABB5COCYLA1BDQYOQ7d6X931MxvJ2nKzpDQ6Y2pAw3KTPhlMwg24PQUns7PZ9+dKEhYkgconsVgLU7VkrB\\\\\\\/aRzY+IrgFhQn3fUFcpnIAiypPhFG3kxWo\\\\\\\/AJwnX1ERy9NdOMByfeSMHwBgzoy1wRdQoz8XvO0PxZD2PgT6Zh1WxDzYsn5xRZuCaa0epIlBYVXSvL+VLaRZ2cN4idQmvRDZysponVbpcMZaMVNu8KzJD8Re1N5wPE4OG73rzxi3ZL8DbpIUNpwX+1LLBqmqxQteIPEPvTRohu+9q\\\\\\\/eaQZ7lAiQAtATCMh1EkHmPcjAXOnd7ywrF+ZJvfPD\\\\\\\/xJH7oiy8rbgMG2DJccM7UtxV57\\\\\\\/kwueG9OVssT83G03AEoM7ano7JjyWt74RBkV2XDXFhYnxv0QqcU+BcygCSRYq3teGj6BScgNmlQiZH\\\\\\\/PjkBijGv4fFUCI1L2k64E2S8hotRk3hs3Xg79DL3qTJqVYLwgTYTQ3XAYOrLzI9vqNVhTi13L1xkdbj0kbJmVYlwks6Vwb+9xaL\\\\\\\/atRh3ttNS+ukjnzuvrB+1P9fH+GFgRMT6rhsVchzM1dU7X3cERY5dEynXETPxb1C2U9qPdWu1evJ7Sz8RxrnWS1rAguKNIV1Cee58vD7ycKFJizC8E+8YB+62rZCJxvG7M3vWZ5tSJLK\\\\\\\/RheJkqrFviq+n1FNqxpIUJNANa0osFSGPt8zsvZILtx1X4ucNBRZMHBe9uvhPFbW\\\\\\\/3umTEFpujm+EDiNEtn77ulKZ4rJSNUxzrVBgmHhhGW8rx3Wm4C3qZEXFs3fBw9ly3pIK7wr1n2hT8I4bg9ibc7++cePWfiPykirCbOtEreErdgTHeUxnzDVAbBpuekh5LKCSsrNJ\\\\\\\/mxyl\\\\\\\/hWkLYjBdhAdG8bSWX\\\\\\\/y2qv9bcDaER4xUwTa4lGcmpDnKVDbI5b2Nrkm1zm6pAOB9MR6ead2D5G6cy+RLzDg0vDP2TwFwD+043EbTBZj7eoc5DHlTCzS2E1z\\\\\\\/u28DrNWQVAcRSg95JH6PY480jois06Nhd2Kt5+zW\\\\\\\/hM4xxO8g5Mw702+fXvr1v7TMWGcbDdxZPq6q6\\\\\\\/VUPINHJ4pZyyON+PH2rb+jA2mJH151GBwUXaUIXYCtz0lVqYr95\\\\\\\/e+ALGEsdTCvlPvUT6RVjITDS4T9SLbGWWlUXcmBgu0\\\\\\\/ijgj8d3g5FmA8bf5RA4upKtyOsrdylw2c\\\\\\\/6v3qPq6BCeiDet50zrJUMcq\\\\\\\/RSqMnU3Lpiw6BpFcMIfGO7CxBRHxGZb6xtyGfDZdlUakONuhpVgweDYk5dEbf7XHlgN\\\\\\\/IwVYtF9JcnrrAqsAtGMbRjsQs0vrvaLO0qOKgN90PejFN7xhMRFBzHuOIoQwi38JQrntgQaRAMVQ7BEmRWKHTaUkyqAJUb3wIzT6t+\\\\\\\/X9rd5y7weRZyM3ihrJ0gT0sMjgmnYtHBPTunN8D9gQizKSu4JYX4zMGINi0QN5APySOib3cgtqftN1yQskRpO\\\\\\\/xTgaYgAtb0Lh5w2q0wNzJjVjdrhbqMebmNw4fqYhvG1RszqJmhun+oaRZ15JLazlwNIn0QIrYGm9NkvGmzYH8kL7xEPa0wQbWx3xlf79zsk\\\\\\\/xICMkHMyDPzmyarThjeOIwNavsUZNjD9kdnE0H9iWOZGROoPQ6IcD1akgyfGxNk\\\\\\\/UWQ51tBzzJakWQ07GKGMSsREVFPZLB5BCUZ+ojJhyVfd\\\\\\\/qo+fEHN2cZdQK2DBWJodB2ySAXsWm0eF5rBv+RAvem1uVdofKqiymGPanQVOapRzIAq+4wEhyPDe0GEnHAAhiXRux9wrPNtI3N1xIU7ht52C0NRE9LQPgLan81WfTJ89l+DP6p7DQRiQkXIu9TeuVVgpJseQU50BsxcqNloAZu00KPbUMWp9rrpMPOHR3S6SJO0rKv5Xs6gegqODYnJhmYU5oHpxo4Q6rr\\\\\\\/V21ifUyMvlfbKXtzk7W40A5am4Ra8GpfcuRut0vVB6DJAl54FERGZjC0oDo7DAdL2khMF7HYSrKcYVYkNmxFtXN\\\\\\\/NDYSHR2Aq581G58yXW\\\\\\\/L7Gxbkn\\\\\\\/7ziz2nyDzh+DKlHnH0arK8lvI5HKr\\\\\\\/cF6fJbkFEgw91Bs6PLNjVPFntJHYSMrpMfBqs3Ml1+75fMrltdbBFj4XWgGaDwVKkfmvjQUoqyBUfxst2Yhpqjqpe4f5GAXI0OieErhPfEhU0tqDdPeSJumoWOwwsToFgzURLFuapQvbYKqI2bKLRaBu2j0fAVWW0MyX6xeTq3OdgBXBIarqXM8ikXsh7cKY0A81CfG4f0ENaa9Ybt3mpTRsHl\\\\\\\/GehXl0UdXEj0\\\\\\\/RV+I4vMrkHRI3h9Qo18VD4IjXxGgbxn1MpXkcI8umwWC5sxrOOb8mjpllujejplQrTJMeVVmDpo0273JUWpoBrtjsQd+guXv73IJEWV2E7GjS5zh8mtStGfMOHO0oB0m\\\\\\\/x+LiQMKUMWeeH50AK9ilhoRl5cETsKKfBNsj\\\\\\\/CvqbvIv7fTZhjzT5mfC1Av0gA0o6sMVteau7fNrnU\\\\\\\/N8aHpad+Hx0ZM5yxetXv4KHnBSuM1sUzQ9X\\\\\\\/K1R69B7r8ZulhuG7ZsU096L1JVNh0b9XdQrDyk362JiKQBPbUS+EFw0TCaWWuHve6Ab0SafqjiwOeZh+TpUcSx0lLpcS1GV+t6y9hREeQlYcG8zAE3+Gr7SAPsi8w6HZ\\\\\\\/gzhnGvtfr74orqdgUM\\\\\\\/HF3vekQ5IBhZ54eqNRWiilyO0iTAK3DdXNdmzvQQThgsUZvU2hKU0HNh3Z0ufZZqHM\\\\\\\/KoZuSCzkANixibkskivvqu3b0ZESW12V3\\\\\\\/sfvMkkgiCDGLuhV9+U55CpCuh0MmkbN9ZknTFSG+DXzSPQyiDIm0RjXGwFmwN0LLlV\\\\\\\/L0plIrpXzaomtIODuf4PiffkAIOc8SxsOw1wiFkBmhfvQMu5iUCIyurRzAZnbFeHDGd04V9VzJmQU58dPHBeuRWCkNwvfQrgnDesGPjJ2oaonP1GGxs2oG3pQvFiXQpVIyQluTw6nIuN0yJ1dKZn00R8vdTSMRr83pgurJ\\\\\\\/q08RLtxdGuMtU9lutcFvJyVlzfjt5rRETtdpSDr+0Z75wBJ6G3LBdcdc3aOeMJsfutAH+5P0PyV1WIfrKumCsqKAV2kF0+2Pd4wG8aEICna1PRKThhdEp0Ka1uTlBKWtruVICC7j\\\\\\\/mkGEaORudeiTYGPmoa1ScWtMy29hPbj3t8Q6DM7gBOpHwe2aWgHy8gPkv6jgxuRIw9hj2pz9FyqnD6Ly\\\\\\\/luUyadvavUsOTgqqmM4hcukYuPIzXAbBZYGwQ4ATcc34kk3qKe234fYHayFdd6XOeVytcQvVgxnUsdnxkpxtAELRcvXZTKB1cs8h9RlIenhQxIiCLgA+TL7X3q5wn6ZRsVJS0dlLuS0OIoWq4BtF5gnNSNOWOgNTw4uei4uAu7hldf0zRjZsNgl9CMTC0FapgkNNDlsCnfVtx4xovNgbXLOa5AqT4B6l2F0yiAXUXe\\\\\\\/mHMAS75pFn+19wtZ7Lj40v+\\\\\\\/neVmEoVcWPtMjFc5\\\\\\\/wSDrNHm8+MXgthU+SGnGsOe+\\\\\\\/V5n3EyInkThljJtpE1gtArQ+iTK0pvBfL8MSBiq+td8JmERbomk0am32LNVctO8iYddEggVWaJy3i1iGDF+Iyd4nBI\\\\\\\/FC9PDGFNOXZ6DDVsXex8Vtk+vdm3AhJpsGvAiIB5Gap8QEFZ\\\\\\\/lzy1EwtJa6lhrrsQjuTUINE157oZNAPNZ1lc5TDZsifOSKHotBUEecXkRXArFlC9FGSdH7mL4\\\\\\\/irBkcThkPEhyJ5MKgPFJEOFF0+ooOkHeXu229G7eK5qBraBVXJhEXSR0EAktxxWQIKsNdOEbfsfshsZ6UesAYWLV\\\\\\\/pnwcMH1CzQg4HNZjn4W3GHlvbQ6y9PdGSrUOU8MVgRFbgorWLO5juvzzbPW4gD25eEJj7Rh9gC3ujlXURLiuyMH78IgY0q+I8c02FG9L5rfzWEnJrNp6epxh1UYvbV6ioM5CgWPso1L4wIDkAN0Eps0bwlZobCFzMGYmRefYHhcxpfjl\\\\\\\/AagGItCwq+aQtla0W1Lw8slXH\\\\\\\/+jprxclE+fxka5eVIhRcTMLD9xgrbbafF2xyY\\\\\\\/dd6IXFTFKuqPPXiHdXzMyf5p6yyjtWn4PyVviCO8Lp5ODDqgtG+qf3s\\\\\\\/rx61Ok6Twsw0Vu6S65Ni09n\\\\\\\/nwF0RfIChx0viaEySC5xTudEeCssG1c8e1IpsRi77sjEo3mQTQ5Lh+ZHmobdJHPtSlUo5JZajmKog9B1uYeKb3VOHvxCHk48r0wqt05KwBCnFYTNROTFsUQlv7APqo9W+sxsido+\\\\\\\/ZjxwnfFPUtOlQcf4oufF5QlWIOFgM9M5RUruZ1hjAzJwz+Qyi1Q7W72Ad\\\\\\\/Lx529GjM4NP\\\\\\\/C4mvGpM9i2k4v0a49jZmfiQAnm19SYYbCfC535OEYGnEU2UdzL1Adg+sLleNslt59nJv\\\\\\\/Rf47+aGxngYwXeq6Sk\\\\\\\/v1UkjE8XN1AnG6USDmRmOMvJLm7X3aDmo3Vrtr4Pp0XOY4YZdSt3BurANF54jDaPbLgiuSJ55YgeU8R5wBX5Ps+BF3Xmpyx0U4bp35f+Hjvl6RdMfNOqjGlrfKsvusCOzVDpkBHjdbNoJ4SxdZFzzkiAP1pAu9GrScDwuroQTI4OD3uQ8Jb5wEq36calgtXPyGca3RJefnpiYx0aDlQ1M8OCrbfmVIhUNsooaiVKN67KfU+9P+UK06M5hNWx\\\\\\\/IKKGEuqH+GZ1WUItvkNduYR21CKBJlSyvNLCdZXIyTnemhtqweu07wKOBwLIQmSsj7oiIFhUboblX82ZuYF2yoDXutQ7kG8jTaa0\\\\\\\/I2KR5yFpozx6i8FS2sVy4+KNOBYAJUUCkoA9kcHUZ5d7m+tgRu32FylUN6PTDR5aFwM1SmuvGVdOVyZ3rJYQmIwci\\\\\\\/Y7UIWAHxSgGqWJUe6\\\\\\\/uIEC9F5hEJNrp0H7h+8N5ZDRPz2XWIG7GIIRk6ZrOo\\\\\\\/ucIlfl8aOdmqfadR0BoYwrYqB41gLVQqnnk+\\\\\\\/NwUap1iXDDkgq1iCh9YIJ8nt29xOiMQ7gggD4IO6aaw05MzvEBR6diMItE4Zuh8SY9\\\\\\\/HQIm0J6eARto418LEW+RajS5pr6Hsc1\\\\\\\/LX3UV5Bx\\\\\\\/+0RI9h4TF+kUfNeBzZUTtPDv6\\\\\\\/KMVI36c8kMYFWCcVwCg5EUxl0EK4jAVV0nfCtSMga94wFbd\\\\\\\/esTUzHZSlEW85q9KkQUbRnhoAlJed43LQjGDqP78ifB8WN9o9S5erZEDOXcvySqu+ujvxLXSCz9LhH+jTrIK1G+8MCx1V8\\\\\\\/vfiIVmwTomczdhypzZ4SJNWK+6Xip7RVdvPPTFc\\\\\\\/AbSXdgp62dLlrd8DnfRTj4IYQTt9tdtHvnSEsbLgZEqtXPhjn4qiUt6hOJp+Iaapv5gDQ3UH2d3HFH3KB73Arh3sVqLekHU0KNiAwjSHjZue5fRgjTN1arbD6y\\\\\\\/cXa4s4xSr62g2ScJfw0BNO9VWToCTMXTEAf6xOgQiw5xVkUb4cu93k2gKK1d\\\\\\\/H+Ani8+506PMhLCYGnd6Qsm2ygfMgXp3AiLLUndmJOR4DKb\\\\\\\/oAKEDCdddeHwoha6P7Xm0sa3k\\\\\\\/B4sRKdAqlh7X7RVlYLWnyvGLk+GNMTa0wTDK93x7HFvJ9pJkgv++4pexmsXYjwC59iBwSMqnxga3SHClugCzF\\\\\\\/VDGxM3KJSi0MbNjOfoj6iY1upRD9BeW3CwA+g9G57Ct7QXnsZ5+r9onLS+Tp0kI3nDbDvrqQbuw4sZIcGQPdNHU4wuNrJds+CtlxkTLa2+FyqYjnwbVkea18uOUK98u3I7FXC2eCiQW9Qc+Ray1OCjlM2NXLcEmHK3M404BEZtxG5pTQ2\\\\\\\/IlR+JCsk5AKacy5pf4\\\\\\\/gMww01yDRhtMQiMoljnqrRB19rQZX1czcFUlWk\\\\\\\/2LhnLY94Ts3DgAe040djwqdg1QvTF+eS9pEj+Vajrf8Qh8r4rNjyVl\\\\\\\/kwDbszFR0kstuG1a4jwt1nAtRetfTPCaYJoIQJIbJuUGDgjcktCHphYmBIjeoDn\\\\\\\/gFztW\\\\\\\/7JKxxLCQe\\\\\\\/u2J4jAAnnhXmPukl\\\\\\\/4wokC30m8toaOK50DZAg7p61MVJ5X2mcZYVaSHFfVFtPVPPtwfx3wwK\\\\\\\/\\\\\\\/WhYxoNBiM6mnhuyphg9U\\\\\\\/aUU1ZACpF08PzZsHWQnRApZWjKsHORB+FvQYdnWgbulJ+NDH72AkO2D7Yfm2D9xfFXrR55hOjynuWwem72s5gqgzFWEjauCHnb+SA7npn51LAeQ35QF8t9QjFh5+Qc7OvxtCUU1n7xkB4ZehLA9ZU1xRPuEOfZ3BPxnJ1q0SWYIAu8QTMs1VoVfUpAoKnvRCm8OU2Vt+xxu7A6v5IVYyqbqWNAa8QQX1LJDFJumxsTKdI2qeCP9fhVxwlniS5EOFB6N4LqQHu3gY7o93GKcv5bOeAn7peuq2gswO00GP19bv7aMTIFtp902n08TZHPPeGOHeldACEXhA\\\\\\\/bOGy4HmQALcrfzilT9hYPkSZNJXfxgT3SlWiMzmcm8ucXNHYZ9nxWdzwulrXLm1n+wNOISCWHnq5pzcUYUy6pXMR2dL+bhzBCCv8FMc77o7S9TmqI2l+9tRBBb2pjgyLVfYWSn2kUZrt\\\\\\\/Vs\\\\\\\/2v4n5K3oafy0QhX2O7HF3V9QevUvcctysT7k\\\\\\\/YIHGxWUiCA4KC227i8PDJ4ETePXYuegv+afT3reqtRwC746M9ovAW23\\\\\\\/vPHo4zNuDBYShqsGo2JuhDPeRAR32kMfQP\\\\\\\/pHZEgx1G9xfv4qIMedmTAPSxJ4WAtbt2UntdSg0QIYJnquZvXFaQwQCs\\\\\\\/f2VBGbD7AdvM8dT50qG+PjfGJSge+tiym1GrOTV9z2E9RUJuJcDLQ9G7wjdoXxZNkASALdvsCZDthhUm8xTdH3JtXUFBjRw1bTaoC8\\\\\\\/7EArjzl6Asl+07fIMQ6ZVKhrTDpyXG85bcPfEkcdDlz\\\\\\\/HxTd7GQ7eVbkGT5flypxJOxZvlJAgunoki22PcPrmKt+4iBFmXBPn7TfaaM5J8Ov5WShl7GVZs4SaaZIqOyn55raykbON5edYJbHKj7ZdA4J8nYEWGtMQlOUlDg1zJaaxNKSfcw9u2h2f1emV8E3\\\\\\\/+EL\\\\\\\/lhh64cXa2J16BAaJjvlM90vkIZ41iA19qTXcEyMCWNEwQnEBrmhj6JB9e4DlmbSjPKD5Gm2VUOfxVLl\\\\\\\/r0j8q8\\\\\\\/8x0CLe9ZFAvMW1rJtsFXNfTTTHz\\\\\\\/DN\\\\\\\/+KFZJyk8Jhi1eDEx5+fybdqrCU1GH+6kVuihL0jx1CZqMyViR3wWjzNJwANOMsUZVnuZb8MEz6e\\\\\\\/XaRNQmY2bJg8T4DmguH0rIYVTgwt3EU5\\\\\\\/waHvu+Ved5CFcV01tmnSHFNugVpp+2XW4yjbI7n5uAvbh\\\\\\\/LcOBCG\\\\\\\/C74SRMjvxdZa9cvZ9LSfbm7eiMQak9NV4ZYMo6LILN6dAu83NhtJILeupBWeVxxUs6wMufiuKEKbLGK7nd+DvGuppmVYpnF4QBJC7qVEpUIK7xJP4h4YxTvNApu6vr4ynl9vSPOEuHdwM42NQOLPrJAPDBPYQStBuXmCXNDDgtfh5cf5HgTPaFvpywAZ2H9OEtQ4eXok9FHEYAhlxx+T4INGWCZyOONBG7O3ezqPNKiIY51DxEqSLEB9sGQMbydJgqQTeZRx2mefWqn\\\\\\\/FVdy3r9cjcVULD70tUWyn28\\\\\\\/Uul62WWLw2bXp7EGEEyzRMJFUcb9go+KoeyNxLToFwGUnXN27NAVYUgVAzVpVkC0DdiEJcbCwJS09Of+zN9OirFL9TTalPOo+vx1lD5HJ3xfkq+AMNtOXt5gIJaubDlmqK4oQhMzJ+2j6V5oKs7vH8gQilhsH4WmULhMA5eQEgVB95b55urRr9LJ5Btpbp8XrxdeUr2DV0ftfnCOk9IdyKYnwOzTAvqh8oh0EWgRBEX2RWarIrLHnKa6LAORZ0MOAFRDz4PpLwaa1nWnSHbkgW4IFTwYi\\\\\\\/G2vUQHOsaTfE85TbYEQRVaffFSG9kXJh+OHetYzRxCV8jBub6C9fP2gy4vC4QJBYWo4heRbbxLpEfEgDbc9ZjNpShALEV0LqmFHq5zYim4qQ78m78gUIws7116rItYA+uxFR1GeGj5RK+kH+aPHwW5vSOtB0G8Vp5+2iabkbtj4WwqbS\\\\\\\/QXzy3hEwDNfBYAaL9o5ZCeomoOqWo5e8C7WhZ4wJzHb9eN4QRTXdVRBrpb5vVqVjx2z6M2pPj8AjNjKZCfDWhrWlHy18VoQKGwYsyO4HdL3x+44Lq+oEKJ7cUS77XOhsVgtVuouTgLQS608Koh50pjR8vIg8wH\\\\\\\/P7euYvhCgSB153ze6VvJxzrod8HzpTXBeRcBBKn5fremRZT3IhaHAtfYAIzCgox1G36n+jDOZi3ULjsDw3XeqxUVjCbVFpIVhtuBVez7r9NwVUJ\\\\\\\/or7iyWdujmRkw+HSQ4dHgsNWiaX2ThuVD+yU4fWAJ5ELMY14o3nTIU02Z2zEYoRaHGNJyS4j+QkBYhrNmcLlLcQ+HfEISpUbZcY40bQS1opNNrV1DobO7fl\\\\\\\/SGBBjfrAOFYRRG8bFrcLfCHyutfQqPidUBNN\\\\\\\/1ts5yX2hIw8Ufnaane9Jw9V9vmT9NFD5whaxu+q\\\\\\\/k2rAflAn2+pRQaTjTUjBLzwdbt8Sa3Pd7NnDdgoToZ2tjsRqQpAdm0CHpMFnZ8lWFWIzv8xFj9deE+sQSOoK2V7G+seG+8PEU8j1Tbn964thrKuf9580Fo8yzF05URMqHAr1JOQQUghhbXRCVtBLNjbSSHz9NnFcxseGBMtsG\\\\\\\/c1900xzPQiyBvQUBg5plR2JyKiXWPA5cpGTTCnTSokGb7Dra2lpdza7KNqPcXawdza7qNYNH4YvJUAK130AZ+eZKoEcGj2Ggf+RCIwYGuPI2AdkGDn8B2VcnLPsL4yeCojLNXOCjmcxfuQ9w3owdHmygaUW+ikVZbwQznlCPg+eWxBNZ3lU3+2DWYE2wrdvEtI6ryhjA+tt2cKxkCoOOu1mNP4X7ZJiOwtBPpzD3gcRdNktDpNF626fkR1WnQqn9AtzNarbzW3yrYVXqorjwd0RAN5OOvdA8RGLH0VpZA026856YlLIcdt6a7WYFYFG90hK\\\\\\\/kiiXKeiD8tyTnV2FPjOAEg5W2IaKnQzXWCBVTvt8JPmW3paGTqA\\\\\\\/kbXzW47mmJTHOFspRdkXbS7SBPgT+C7fyEYMAmOsegKX7aLTGMyvdp0XYWHFsnEJOYVTaJQSq2i95KD2PlE8C8z+EuWYd0KXs6ZxLImddlybeaYieNNXpmN6EcPWx7LCJRalrEnT2KidWO\\\\\\\/W9QOwUixr+u+6AejLX1\\\\\\\/cslMZVUVTBKLm1RiaHw0P+FZFlLg2kMEIZTca728lBdjqNy5CYOQCTwJuvAV6wu2eHcwBDbokmkvVdpIjKgvLcIj01kOFenlVU3itrLUarYpY3L4ILD2Mn6MjXkjogu8gjNfYc0qxdwcAz423TPQuXd9TcTJhzeZHmwL8\\\\\\\/58rnV5rYoS9LcMRfWweYy3gY9aMovjGBZVj+CtzCOjIzXWf\\\\\\\/anjAhvb1cji3DiRIN5JVegKv0bblD2SgoQQqHfdrD+xVisJWYiA\\\\\\\/WdapF9BtdqxPWmGDYcE5vwLYIHS+Y8prQSnfmo498zw26Q10bA\\\\\\\/5rAEZh3lPgSsevZE4hpQ0pXuigiQ40rXEpFZGXKii5jyZJSFSqUIcfNlfV2IomFVdu\\\\\\\/iywKaCF+AunX5gqD\\\\\\\/aCeoAQb2k5kILCV+AShjX\\\\\\\/V+ijGc1AdTiiYQ8Tbn3Sp2iaZz3sO0RRKp7tvnbBjbltlOcmsPffflu2Pr+yFHYGhFpRztAXVmu9d6Uq2lz8GUgPNgYuXWALoUQlqwcEsu+q5zAU1FlipHWaS2MW4KYSIleeXXQWC9oXNW0fkFeZUNgZAcJ7VcmcqsMEnbd8RLcinYtuoVdH429lCZItexDsV\\\\\\\/6NDhp4QHoLuOGu3thUg+HYYMoBj0leupFNkSWdOwJSej+gb2euLPpgH9I7YkkGY5MQZxRdTy1jL75UoGJtj5vqC+S5RtGaNyzMaTXfVf6294J7IhqPZeH44\\\\\\\/lyJznXweEnTb\\\\\\\/48hAtqsfBIBavHN7ecEqmwjBh315ygZ8y88lC7z+IfSS7UKkZOd9Uw\\\\\\\/ogP6du4nRVn4ih9nJYEOihA3e4PV5S1SetPdESFVjyJPA0359UOuY+tXps1ZKoXqQaj6kPZuLIPEVlKYF32nGC88gXqJ4e5wx22t1ozc22YNoIljLTWA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Me5Wt8yrZIkKiqYdgyd73Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a3bb2527aaa77a0860af7fcb31e0482e82de2924e1fef8222ddc0cfc72aa624\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JTQMedbhIZzdy8sxNMjVTKFC9AaitMxxv4zgEG33PJRwXQQ8CGvOH4FfrXIQA0wj4LtyAYx0bV88ABOizrIGYmigCnXZNmDRyP2BuMktmsh6abxK0mFICMAgBQKKnm0mwWsAUUCcQ+oD5qjsigcNUBs3CIxalsmLYHbCxwYkCH\\\\\\\/a3Xwftn59KfWtLsPnBPJDYet0apeO4FEap64WL3Bdf2xKV4jIZhns7M8Su7cahA8YvF4V+DbVatcfLsh6kv2Mtc9F2onOnkfLhvEAG\\\\\\\/MS8tlnbxdvX3E5+zaIBkSPbB3+c4M2TQJAXy5mONc1TowjceEZfVaiDLc4Fansz8J+FrDc6BciwiQ644PGWqGeYWnShSziJG48tKabW0n+QZV0j+riuIxFJkUtrKskj3PAE2LhvGDagA\\\\\\\/mmew5sBancGd3\\\\\\\/TQtE9wtFc9t3VgWH\\\\\\\/XaIIH99iwhH273DrlKFMjzJp48vc10noH6XxYFpkcbDw64F91XLvCgf7gDJz\\\\\\\/a0TgGUjMNIVwh7NXueU\\\\\\\/xy\\\\\\\/evNV\\\\\\\/FhyG\\\\\\\/18+nmDk\\\\\\\/TGpaGktFHEFDuENH6fWgrLGgcOflip6ADPUOngT35zvNJCRxwhO4Lp0\\\\\\\/kRgtEzCRwcbMcoggXc2KxWMicePqHlj3zVvXw6rfL1L6b\\\\\\\/k0S7JYUwUvMsY89ZKm+AraWxQNPe3kCdrnkzUcXel1tTbjnBjartlCLgmBrcyWojiTgwpuWGttwIFtMNeMBISsVp1kOzhqqBre7X2+Mxmk0BVfncSbZarRsZb9OF0OvGHsV\\\\\\\/uVABfxhxUoO3XAjGtnMJJu7txQc5vHKKuooIdTZlvMSjVpMpGlXkGananU2sWKnHluMyZ3xigeAD8S4hHPJZACXQWwMAmQbxK2Kit7B0z0e7kI4WWnXwIQsmmxe3zZl\\\\\\\/A9ypFHStKB2J3mSiC8oYk9Y5T8aW4yaO2LvjtdgOoaGuSRNIwJOkJjIcZfNGADcLfgLVQv40XQiXmAAqfsXRQlD0Z1rJwtle6\\\\\\\/LsuhLwMhfC5WYJT3WAA7xwm4oTG3ax7HkIOrm29+bTntP4KSYY92xY+YoTewIL\\\\\\\/NX0Y1Y0i0Z3\\\\\\\/xOcZ2SPv\\\\\\\/fJJQAzJgqCUqZ2EILYsfWDdLWZ07H1IYGB4K7oQ+E\\\\\\\/QfPPuOjr9Oh1UewZk5BGL\\\\\\\/KDU1omgNOOR++CJoc9lKutiB0E6SdVgDWNgTBeX0hmPGmkgBv+biNUjq3yFojQ0XX+YSTpPoquN1PFlyZiHrrqvMfjCBmxWG5Rhl3O0NKXP0YF9NhxAr\\\\\\\/eQtjXNCNl6NV0\\\\\\\/LDu05ZNC9MUnLrKSut4P0F+QPt4kfS\\\\\\\/gcx3Uexl1ovzo9KkZ7G5GZWNAkbFbDUOuys3jfJGAU5h2nZd5\\\\\\\/rwsbX0TQjqzgMd2ZtQEjPUqgRrdH9xC3RwQWPaDlPSJkwBtMTcuh4qYqCM889VaYTy7gJk3GSe1li3F8DdmTiNubZ24gUnsraGSKRmKDfgr0aKkEIxCWYC7dY1\\\\\\\/hswK\\\\\\\/kDzXj9yPEJmjgl\\\\\\\/Uv+NrL2JdC+Wr3AIvsRkU2+Ll8lqQVDl6593JzHkNmWP+4XTKn\\\\\\\/xHSic4My3dGEZ46xbgkZw0d2yaG\\\\\\\/1UIFhR6ycucZswZ82oAQP6IFsqxYWsr4dJdRuTY151Kykr57nTef+LaFk\\\\\\\/ZSSZntPoUzSwHQGto+o4n\\\\\\\/koESCOw0k3hkITT8CEqYFgFXB6g88Ft11ThapZenhiFwoT8gLsivoZw1qJd9s1j13vQeJXfhoRx7XmbgxQpfCY9AN6NVM3RPOl3OD\\\\\\\/vjvtpxEsfeQd0yMWGEir2W0clhPl2r2V4wrztj6wPUOB1npcyBs6p7z\\\\\\\/IGIeBNbmmv5UiPhEhFHfARk6mmhC+5Bvzgz7mpp+RaLQDt1EnSjIw27usm03vsUW2+FjpWqpjpXimRZijbXl6akPtIPzQVPdlcBt6\\\\\\\/P8BzRlvYK6e1FeGq+w4Q2Lq+wbK2eNJ+nMgKsV5LqKL8pOUjHRyZBVKPtu2IuTjF1Ajox1Mdx\\\\\\\/TQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Qlvi4sGAjrfoByC3VuyIKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a77e24060648e3e3580d5ed912c5e196983283267ed43ea0f3f2b925925da03\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459468 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/807" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-47Oa59N-T4eZb2wUrR1kXs-Fr0KMtifOjKCfPLvRjRc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/807" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QZHkX0Ats4_FSfs9letHJECK2eknRonc7zGnRIN4C2A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/795" title="Distribution of Long-Range Linkage Disequilibrium and Tajimaâs D Values in Scandinavian Populations of Norway Spruce (Picea abies)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/815" title="A Targeted Genetic Modifier Screen Links the SWI2/SNF2 Protein Domino to Growth and Autophagy Genes in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459468" id="node459468" data-pisa="ggg;3/5/807" data-pisa-master="ggg;g3.113.005710" data-apath="/ggg/3/5/807.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Gene Function and Network Organization in <em>Drosophila</em> Signaling by Combined Analysis of Pleiotropy and Epistasis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory W. Carter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">807-814;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005710 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459468" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory W. Carter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BCarter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carter%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BW.%2BCarter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greg.carter@jax.org" class="" data-icon-position="" data-hide-link-title="0">greg.carter@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/5/807" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459468" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/807.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459468" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/807.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459468" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/807.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459468" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/5/807.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459468" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">High-throughput genetic interaction screens have enabled functional genomics on a network scale. Groups of cofunctional genes commonly exhibit similar interaction patterns across a large network, leading to novel functional inferences for a minority of previously uncharacterized genes within a group. However, such analyses are often unsuited to cases with a few relevant gene variants or sparse annotation. Here we describe an alternative analysis of cell growth signaling using a computational strategy that integrates patterns of pleiotropy and epistasis to infer how gene knockdowns enhance or suppress the effects of other knockdowns. We analyzed the interaction network for RNAi knockdowns of a set of 93 incompletely annotated genes in a <em>Drosophila melanogaster</em> model of cellular signaling. We inferred novel functional relationships between genes by modeling genetic interactions in terms of knockdown-to-knockdown influences. The method simultaneously analyzes the effects of partially pleiotropic genes on multiple quantitative phenotypes to infer a consistent model of each genetic interaction. From these models we proposed novel candidate Ras inhibitors and their Ras signaling interaction partners, and each of these hypotheses can be inferred independent of network-wide patterns. At the same time, the network-scale interaction patterns consistently mapped pathway organization. The analysis therefore assigns functional relevance to individual genetic interactions while also revealing global genetic architecture.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-interaction" class="hw-term hw-article-keyword hw-article-keyword-genetic-interaction" rel="nofollow">genetic interaction</a></li><li class="kwd"><a href="/keyword/pleiotropy" class="hw-term hw-article-keyword hw-article-keyword-pleiotropy" rel="nofollow">pleiotropy</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/genetic-network" class="hw-term hw-article-keyword hw-article-keyword-genetic-network" rel="nofollow">genetic network</a></li><li class="kwd"><a href="/keyword/signaling-network" class="hw-term hw-article-keyword hw-article-keyword-signaling-network" rel="nofollow">signaling network</a></li></ul><p id="p-4">The systematic study of genetic interactions has proven a powerful means to map genetic networks (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boone <em>et al.</em> 2007</a>). Genome-scale interaction analysis has provided a global view of gene function in yeast (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>), and studies focused on specific processes have mapped large-scale networks in yeast (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Drees <em>et al.</em> 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Segre <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">St Onge <em>et al.</em> 2007</a>), worm (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Byrne <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lehner <em>et al.</em> 2006</a>), and fly (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Yamamoto <em>et al.</em> 2009</a>). Analyses of statistical epistasis, the population-level manifestation of genetic interaction, have identified important effects in mouse (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li and Churchill 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Reifsnyder <em>et al.</em> 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shao <em>et al.</em> 2008</a>) and human (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">McKinney and Pajewski 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ritchie 2011</a>) genetics. These studies indicate that genetic interactions reveal underlying structure in biological networks and map complex genetic architecture. Advances in study design and the characterization of genetic populations have been accompanied by parallel progress in quantitative phenotyping. Multidimensional phenotypic characterization is becoming increasingly common, often including multiple physiological traits coupled with thousands of molecular measures such as protein and transcript abundances (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andreux <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2012</a>). Such research ultimately aims to provide a genetically precise and phenotypically predictive approach to medicine. Success of this approach is contingent on the development of analytical methods to extract quantitative models from genetic interactions across multiple phenotypes. These methods will increase the power to formulate precise biological hypotheses to potentially address the complex genetics that underlie human health and disease.</p><p id="p-5">To date, studies have primarily used statistical concordance of interaction patterns across multiple genes to infer the role of previously uncharacterized genes. This strategy is often referred to as guilt-by-association (GBA). Advanced GBA approaches, such as clustering genes based on correlated interaction spectra across multiple interaction partners (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carter <em>et al.</em> 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2007</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Drees <em>et al.</em> 2005</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Segre <em>et al.</em> 2005</a>), have successfully mapped genetic architecture on a large scale. In these networks genes often form highly connected communities, or gene modules, that are enriched in one or more functional annotations. The principle of GBA dictates that a minority of uncharacterized genes within a module can be assigned the dominant function of the module.</p><p id="p-6">While successful on a large scale, GBA-based methods have multiple limitations. First, they require large data sets to generate adequate statistical power to resolve modules, and can therefore be limited in populations with a small number of relevant mutations such as studies of specific developmental or signaling processes, drivers of cancer evolution, or interacting candidates in genome-wide associations. Second, GBA relies on the availability of functional annotations for the vast majority of interacting genes. Third, GBA approaches often generate implicit predictions of gene function without providing explicit predictions of the effects of a mutation or combination of mutations, thereby limiting the power to generate directly testable hypotheses. Fourth, large-scale GBA approaches rarely take advantage of the complementary information in multiple phenotypes. In cases when multiple phenotypes are considered, the analysis is usually based on coincidence of interactions derived independently for each phenotype (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Michaut and Bader 2012</a>). Finally, it has been proposed that GBA results may be driven by a small number of critical interactions and therefore network associations are not generally reliable (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gillis and Pavlidis 2012</a>).</p><p id="p-7">Here we use an approach based on the combined analysis of pleiotropy and epistasis to infer the genetic architecture of growth-related signaling in <em>Drosophila melanogaster</em>. This strategy infers and interprets genetic interaction data in terms of quantitative variant-to-variant and variant-to-phenotype influences, rather than non-directional epistasis. This results in a network model that maps how each specific variant affects each other and, in turn, multiple related phenotypes. This method is applicable to a range of genetic diversity from a few genes to genome-scale screens. It integrates information from multiple phenotype measures to generate specific hypotheses for genetic interactions. We recently demonstrated the utility of this method in a population of yeast strains by mapping relationships of the yeast mating pathway (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carter <em>et al.</em> 2012</a>). Here, we extend the method to a large set of double knockdowns of genes involved in signal transduction on a common genetic background (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>). This represents a much larger network of potential interacting genes and involves multiple signaling pathways. The analysis exploits the subtle differences between the regulation of cell proliferation and nuclear size across 93 mutations. We obtain an interaction network of knockdown-to-knockdown influences that identifies novel suppressors and enhancers of Ras signaling, demonstrating the methodâs applicability to complex networks and large-scale genetic screens of multiple phenotypes. At a local level, the network provides specific hypotheses of how each gene knockdown modifies the activity of other gene knockdowns in the same pathway and across antagonistic pathways, narrowing the possible molecular mechanisms that underlie the observed genetic interactions.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Data source</h3><p id="p-8">Data were obtained from a study of double-stranded RNA (dsRNA) gene knockdowns (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>) that is briefly summarized as follows. Schneider S2 cells were exposed to two independent dsRNA molecules to knock down 93 genes in all 4278 pair-wise combinations. Genes were chosen based on prior annotation for mitogen-activated protein kinase (MAPK) signaling or prior annotation as a protein or lipid phosphatase expressed in S2 cells. High-throughput fluorescent imaging and automated image analysis were used to quantify three non-redundant features: total cell number, mean nuclear area per cell, and nuclear fluorescence intensity. Data for each knockdown pair were averaged and log-transformed, and each phenotype distribution was mean-centered and normalized to a standard deviation of 1. At a population level, each phenotype resembled the combination of a normal distribution and a long tail below the mean that was populated by lines with knockdowns of a few especially strong perturbations such as <em>Pvr</em>, <em>pnt</em>, and <em>drk</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/005710SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462812/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462812/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genes with significant main effects used as covariates for all pair-wise scans, with prior pathway annotation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">FlyBase 2004</a>)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Interaction model</h3><p id="p-10">Our data analysis technique is described at length in a previous publication (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Carter <em>et al.</em> 2012</a>) and is summarized here. We first perform singular value decomposition (SVD) on the phenotype matrix to maximize orthogonality. In this case, we analyzed the first two left singular vectors, referred to hereafter as eigentraits, since the third singular value represented less than 2% of the global variance and was therefore unlikely to encode significant biology (<em>Results</em>). With these eigentraits denoted <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-1.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-2.gif"/></noscript></span></span> for sample <em>i</em>, we performed linear regression for each of the 93 perturbed genes in isolation to identify strong-effect knockdowns to be treated as additive covariates in subsequent pair-wise regressions. For each locus we considered the model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The index <em>i</em> is from 1 to <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-4.gif"/></noscript></span></span> and j is 1 or 2. The variable <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-5.gif"/></noscript></span></span> is the probability of the gene variant at the locus in the strain <em>i</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-6.gif"/></noscript></span></span> is the effect on the eigentrait <em>i</em>, and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-7.gif"/></noscript></span></span> is the residual error. For our data all <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-8.gif"/></noscript></span></span> were binary, corresponding to the presence (1) or absence (0) of dsRNA that target the locus. We define strong-effect knockdowns as those with a significant effect (see below) and condition each subsequent knockdown-pair scan by including strong-effect knockdowns as covariates for the associated phenotype. We next model every possible knockdown pair with main effects and an interaction term. For two knockdowns labeled 1 and 2 we have:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/inline-graphic-1.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>The variables are defined as in Equation 1 with the additional interaction coefficient <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-9.gif"/></noscript></span></span> and sum over strong-effect knockdowns as covariates (excluding knockdowns 1 and 2). This step is conceptually similar to the original analysis of the data (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>), which identified pair-wise interactions for each phenotype independently.</p><p id="p-11">To derive a model in terms of knockdown-to-knockdown influences we consider how each knockdown affects the activity of each other knockdown, first in terms of modified knockdown activity and then in terms of the knockdown-to-knockdown influences that account for the modified activity level. We first recast the <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-10.gif"/></noscript></span></span> interaction coefficients <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-11.gif"/></noscript></span></span> in terms of modified knockdown activity parameters <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-13.gif"/></noscript></span></span> that are independent of the eigentrait <em>j</em>. The activity variables are computed by matrix inversion:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span> The variables <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-16.gif"/></noscript></span></span> are an exact reparametrization of the interaction coefficients. We next compute the directional influence coefficients that generate the modified activities. These knockdown-to-knockdown influences,<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-18.gif"/></noscript></span></span>, are:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span> By substituting the solution of Equation 3 into Equation 4 we obtain the influence coefficients <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-20.gif"/></noscript></span></span> as a function of the regression parameters. This defines a model in terms of knockdown-to-knockdown influence coefficients (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-21.gif"/></noscript></span></span>) and main-effect, knockdown-to-eigentrait coefficients (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-22.gif"/></noscript></span></span>). Without loss of information, we multiply the eigentrait coefficient matrix by the other center and left singular value matrices to recompose knockdown-to-phenotype coefficients for the original phenotypes.</p></div><div id="sec-4" class="subsection"><h3>Calculation of significance</h3><p id="p-12">We assess the significance of the influence coefficients, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-23.gif"/></noscript></span></span>, and knockdown-to-phenotype coefficients using standard error analysis methods on the regression parameters (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bevington 1969</a>). For example, the variance of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-24.gif"/></noscript></span></span> is estimated by differentiating with respect to all model parameters:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span> The indices i and k run over main effect and interaction coefficients, and j and l run from 1 to <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-26.gif"/></noscript></span></span>. The first sum is over individual parameters and the second double sum is the cross terms. Variances and covariances are estimated from the least-squares regression using standard methods (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bevington 1969</a>).</p><p id="p-13">After computing models for all knockdown pairs, we obtained two knockdown-to-knockdown influence coefficients between each knockdown pair (one in each direction) and knockdown-to-phenotype coefficients. To determine significance we used effect size divided by estimated standard error as our test statistic because it could be computed for both regression coefficients and variables computed from them (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5/807/embed/mml-math-27.gif"/></noscript></span></span>). We selected the coefficient of median standard effect size to represent overall knockdown-to-phenotype coefficients. We first determined significance for single-locus scans. We performed 2000 permutations of the genotype data and fit an extreme value distribution (EVD) for the maximum test statistic from each permutation. This accounted for multiple tests and empirically estimated the likelihood of chance association. We determined that a test statistic of 5.18 or greater corresponds to <em>P</em> &lt; 0.001. We repeated the procedure for pair-wise scans, permuting the genotypes of the two knockdowns being tested in tandem. This procedure retained the structure between all other knockdowns (including covariates) and all three phenotypes, and thereby randomized only the marginal effects after conditioning on covariates. We collected test statistics for 700 permutations to obtain null distributions for knockdown-to-phenotype and knockdown-to-knockdown influence coefficients. We computed an empirical <em>P</em> value for each coefficient. The family wise error rate was controlled by adjusting <em>P</em> values using a step-down procedure (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Holm 1979</a>). Step-down EVDs were not used because the empirical distributions had slightly greater support at higher values than fitted EVDs and thus the EVDs would artificially inflate significance. A significance cutoff of adjusted <em>P</em> &lt; 0.01 was used in our network (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) and all estimated <em>P</em> values are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/TableS1.xlsx">Table S1</a>.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><p id="p-14">The assayed phenotypes shared many genetic components and therefore exhibited substantial pleiotropy. All three phenotypes were significantly correlated, with Pearson coefficients as follows: cell number and nuclear area, 0.65; cell number and nuclear intensity, 0.90; and nuclear area and nuclear intensity, 0.85. However, there was variation across the tested knockdowns. For example, the <em>Rho1</em> defect in cytokinesis caused decreased cell numbers and increased nuclear area, whereas Ras signaling knockdowns such as <em>drk</em> decreased both numbers and area (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>). To maximize complementarity and dimensionally reduce the phenotype data, we performed singular value decomposition (SVD) to define two composite eigentraits, each an orthogonal combination of the phenotypes (Methods). The eigentraits were orthogonal, normalized combinations of the phenotypes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) and represented 87%, 11.5%, and 1.5% of the variance in the data. Since the bulk of the variance was present in the first two eigentraits we discarded the third. We refer to these composite phenotypes as eigentrait 1 (ET1) and eigentrait 2 (ET2). ET1 is a common signal to all three phenotypes, whereas ET2 primarily differentiates nuclear area from cell number and nuclear intensity. Thus, the inferences made for genetic interactions will be mostly based on combining an overall common signal with a signal that separates nuclear area from the other two, highly correlated phenotypes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/807/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Eigentrait compositions in terms of measured phenotypes. Eigentrait 1 (ET1) is the signal common to all three phenotypes, whereas eigentrait 2 (ET2) encodes the differences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1818384973" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Eigentrait compositions in terms of measured phenotypes. Eigentrait 1 (ET1) is the signal common to all three phenotypes, whereas eigentrait 2 (ET2) encodes the differences.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/807/F1.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/5/807/F1.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/807/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/807/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Eigentrait compositions in terms of measured phenotypes. Eigentrait 1 (ET1) is the signal common to all three phenotypes, whereas eigentrait 2 (ET2) encodes the differences.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">For each eigentrait we performed linear regression for each knockdown individually, similar to the method in the original study (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>). This identified significant knockdowns to be used as covariates in pair-wise scans (Methods). We identified 21 knockdowns as covariates for ET1 and 17 knockdowns for ET2 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/FigureS2.pdf">Figure S2</a>). We next computed a pair-wise model for each of the 4278 knockdown pairs to obtain quantitative interactions between knockdowns (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/TableS1.xlsx">Table S1</a>). We reparametrized the two interaction coefficients from pair-wise linear regression on the eigentraits (Methods, Equation 2) in terms of two knockdown-to-knockdown influences between perturbations (Methods, Equation 4) (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Carter <em>et al.</em> 2012</a>). To determine the allelic effects on the original cellular phenotypes, we recomposed the phenotype SVD for each pair-wise model and averaged all models to obtain knockdown-to-phenotype influence coefficients (<em>Methods</em>). Errors were estimated for all coefficients using standard least-squares regression and error propagation formulas for quantities computed from regression coefficients (Methods, Equation 5).</p><p id="p-17">A representative interaction is illustrated in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The <em>drk</em> and <em>Rho1</em> knockdowns have opposite effects on the first eigentrait (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) and a negative regression interaction coefficient (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). For the second eigentrait, both knockdowns have positive effects and the regression interaction coefficient is negative and larger than that for ET1. Reparametrizing these results in terms of our genetic influence coefficients inferred a significant negative influence from <em>drk</em> to <em>Rho1</em> (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>); that is, the <em>drk</em> knockdown suppressed the effects of the <em>Rho1</em> knockdown. The complementary information in the two eigentraits ruled out alternative hypotheses such as a <em>Rho1</em> enhancement of the <em>drk</em> effect that would have been consistent with ET1 but not ET2. Finally, we recomposed the eigentraits into the original phenotypes (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>), an operation that does not modify the inferred genetic interaction. This analysis recapitulates the interpretation in <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> (2011)</a> with the advantage of a mathematically defined procedure that has been systematically applied to all pairs of knockdowns in the data set. In total, we obtained a network of 290 significant knockdown-to-knockdown edges and 146 significant knockdown-to-phenotype edges. Interactions for 61 genes with existing or novel (see below) functional annotations are shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/807/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic interactions between drk and Rho1 knockdowns. (A) Inferred main and interaction effects of drk and Rho1 mutations on eigentraits ET1 and ET2. (B) Significant main and interaction effects shown in terms of positive (green) and negative (red) influences. Edge width represents interaction strength. (C) Interaction model consistent with both ET1 and ET2, in which drk knockdown suppresses Rho1 knockdown. (D) The same model expressed in terms of the original phenotypes of total cell number and nuclear area (total fluorescent intensity is similar to cell number and not shown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1818384973" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic interactions between &lt;em&gt;drk&lt;/em&gt; and &lt;em&gt;Rho1&lt;/em&gt; knockdowns. (A) Inferred main and interaction effects of &lt;em&gt;drk&lt;/em&gt; and &lt;em&gt;Rho1&lt;/em&gt; mutations on eigentraits ET1 and ET2. (B) Significant main and interaction effects shown in terms of positive (green) and negative (red) influences. Edge width represents interaction strength. (C) Interaction model consistent with both ET1 and ET2, in which &lt;em&gt;drk&lt;/em&gt; knockdown suppresses &lt;em&gt;Rho1&lt;/em&gt; knockdown. (D) The same model expressed in terms of the original phenotypes of total cell number and nuclear area (total fluorescent intensity is similar to cell number and not shown).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/807/F2.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/5/807/F2.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/807/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/807/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Genetic interactions between <em>drk</em> and <em>Rho1</em> knockdowns. (A) Inferred main and interaction effects of <em>drk</em> and <em>Rho1</em> mutations on eigentraits ET1 and ET2. (B) Significant main and interaction effects shown in terms of positive (green) and negative (red) influences. Edge width represents interaction strength. (C) Interaction model consistent with both ET1 and ET2, in which <em>drk</em> knockdown suppresses <em>Rho1</em> knockdown. (D) The same model expressed in terms of the original phenotypes of total cell number and nuclear area (total fluorescent intensity is similar to cell number and not shown).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/807/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adjacency matrix of significant interactions for knockdowns of Ras (blue labels), Ras inhibitors (red labels), and JNK (green labels) genes. Previously uncharacterized gene names are in italics. Color denotes interaction sign and intensity of directional interactions. Overall, within-pathway interactions are overwhelmingly suppressive (blue squares), whereas interactions across antagonistic pathways are enhancing (yellow squares). Ras knockdowns generally reduce cellular phenotypes, while Ras inhibitors and JNK knockdowns increase phenotypes. Empty squares did not meet the significance threshold of P &lt; 0.01 (see text)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1818384973" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Adjacency matrix of significant interactions for knockdowns of Ras (blue labels), Ras inhibitors (red labels), and JNK (green labels) genes. Previously uncharacterized gene names are in italics. Color denotes interaction sign and intensity of directional interactions. Overall, within-pathway interactions are overwhelmingly suppressive (blue squares), whereas interactions across antagonistic pathways are enhancing (yellow squares). Ras knockdowns generally reduce cellular phenotypes, while Ras inhibitors and JNK knockdowns increase phenotypes. Empty squares did not meet the significance threshold of &lt;em&gt;P&lt;/em&gt; &lt; 0.01 (see text).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/807/F3.medium.gif" width="290" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/5/807/F3.medium.gif" width="290" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/807/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/807/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462753" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Adjacency matrix of significant interactions for knockdowns of Ras (blue labels), Ras inhibitors (red labels), and JNK (green labels) genes. Previously uncharacterized gene names are in italics. Color denotes interaction sign and intensity of directional interactions. Overall, within-pathway interactions are overwhelmingly suppressive (blue squares), whereas interactions across antagonistic pathways are enhancing (yellow squares). Ras knockdowns generally reduce cellular phenotypes, while Ras inhibitors and JNK knockdowns increase phenotypes. Empty squares did not meet the significance threshold of <em>P</em> &lt; 0.01 (see text).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Sixty-three knockdowns had a significant main effect (<em>P</em> &lt; 0.01; Methods) on at least one of the three phenotypes, and 29 had significant influences on all three phenotypes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Knockdowns of the 18 genes previously annotated for Ras signaling had 48 negative effects, compared to only one positive effect [<em>CG3573</em> (<em>Ocrl</em>) knockdown on nuclear area]. In contrast, knockdowns of the 29 genes previously annotated for JNK cascade and Ras inhibition had 48 positive main effects, compared to 13 negative effects. These findings are consistent with Ras signaling activation of cell proliferation and growth and the generally antagonistic relationship between Ras and JNK signaling. For each knockdown, main effects were generally consistent across all partners in each pair-wise scan; no specific combination of knockdowns revealed a uniquely significant effect for one of the genes.</p><p id="p-21">Our total of 290 significant knockdown-to-knockdown interactions out of 8556 possible (two for each pair) was numerically comparable to the 531 significant interactions out of 12,834 possible (three phenotypes for each pair) obtained with the same threshold in the previous study (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>). Gene pairs with at least one significant interaction in both analyses strongly overlapped, as 55% (126 of 229 pairs) of our significant pairs were also significant in at least one phenotype in Horn, <em>et al.</em> (8 Ã 10<sup>â88</sup>, Fisherâs exact test). Pairs that did not overlap nevertheless tended to have low <em>q</em> values in Horn, <em>et al.</em> For each of our pairs we identified the lowest <em>q</em> value across all three phenotypes, and found a median of 0.006 as compared to a global median lowest <em>q</em> value of 0.38. This demonstrates that our analysis generally agreed with more standard measures of epistasis and, in some cases, improved significance by reformulation in terms of directional model parameters.</p><p id="p-22">Global genetic interaction patterns revealed large-scale gene function and module/pathway organization (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/FigureS3.pdf">Figure S3</a>). Knockdown-to-knockdown influences between knockdowns of documented Ras signaling genes were fairly common (63% of possible interactions were significant) and always negative. We interpret this as genes with redundant signaling functions. Although it is suggestive that downstream knockdowns should suppress upstream knockdowns (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Avery and Wasserman 1992</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Carter <em>et al.</em> 2012</a>), the majority of pair-wise knockdowns of known Ras genes exhibited significant negative interactions in both directions. In cases of unidirectional suppression, some known downstream knockdowns (<em>e.g.</em>, <em>pnt</em>) tended to suppress upstream proteins (<em>e.g.</em>, <em>Ras85D</em> and <em>Sos</em>) but others (<em>e.g.</em>, <em>rl</em>) were suppression targets more often than not. Overall, there is a clear pattern of negative interactions between within-pathway genes. In contrast, Ras inhibitor knockdowns tended to enhance the effect of knockdowns of Ras signaling genes. This is consistent with standard interpretations of genetic interactions in signaling. For example, <em>PTP-ER</em> is named for its mutant alleles that enhance Ras signaling in eye development (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Karim and Rubin 1999</a>). Furthermore, knockdowns of genes documented or predicted to be involved in JNK signaling generally enhanced Ras mutations, consistent with the antagonistic relationship between JNK and Ras signaling.</p><p id="p-23">Specific edges in the interaction network allowed us to infer the function of many uncharacterized genes, based on the rule that knockdowns functioning in the same signaling process will exhibit suppressive interactions whereas knockdowns functioning in antagonistic processes will exhibit enhancing interactions (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Avery and Wasserman 1992</a>). We assessed the interactions for 45 genes not previously associated with Ras or JNK signaling and one gene (<em>Rho1</em>) which had been annotated for both (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Flybase 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Noselli and Agnes 1999</a>). Novel pathway assignments were made when an unannotated knockdown exhibited interactions with Ras knockdowns annotated in Horn, <em>et al.</em> (<a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>) or curated data (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Flybase 2004</a>). Negative interaction partners were candidate Ras signaling genes while positive interaction partners were designated as Ras inhibitors. We assigned Ras signaling function to three knockdowns that suppressed the effect of one or more Ras signaling knockdowns and/or had its effect enhanced by one or more Ras inhibitor knockdowns (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These novel Ras signaling knockdowns had generally negative influences on cell number and nuclear area (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), consistent with other Ras signaling knockdowns. The multiannotated gene <em>Rho1</em> was in this group, suggesting that for the measured phenotypes it operates as a Ras signaling component. We assigned Ras inhibitor function to 13 knockdowns that enhanced the effect of one or more Ras signaling knockdowns and/or had its effect enhanced by one or more Ras signaling knockdowns (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). In most cases these novel Ras inhibitor knockdowns suppressed the knockdown of a fellow Ras inhibitor knockdown and the vast majority had a positive influence on cell number and nuclear area (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>), consistent with other Ras inhibitors. Ten of these genes are annotated as phosphatases and, in many cases, we identified specific Ras signaling kinases for which knockdown effects are unidirectionally enriched by one or more phosphatase knockdowns (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). These provide specific hypotheses for the prioritization of phosphatase-kinase targeting experiments. The inferred Ras-inhibitor phosphatase <em>PpD3</em> exhibited mutual enhancing interactions with Ras knockdowns <em>Sos</em>, <em>Ras85D</em>, and <em>rl</em> (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>), which suggests a more complex role in Ras inhibition. Taken in sum, these 16 new functional hypotheses can be added to the set of approximately 20 novel pathway assignments inferred in the original, classifier-based analysis of the interaction data (<a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462700/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462700/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">New candidate Ras signaling genes and Ras inhibitors, with relevant GO annotations for the knockdown (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashburner <em>et al.</em> 2000</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">FlyBase 2004</a>)</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/807/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Network of novel candidate Ras inhibitor phosphatases and Ras kinases inferred from interaction patterns. Knockdowns of phosphatase genes (orange nodes) enhance the effects of Ras kinase knockdowns (blue nodes), similar to known Ras inhibitors and JNK signaling genes (Figure 3). Edge width denotes relative strength of enhancement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1818384973" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Network of novel candidate Ras inhibitor phosphatases and Ras kinases inferred from interaction patterns. Knockdowns of phosphatase genes (orange nodes) enhance the effects of Ras kinase knockdowns (blue nodes), similar to known Ras inhibitors and JNK signaling genes (Figure 3). Edge width denotes relative strength of enhancement.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/807/F4.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/5/807/F4.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/807/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/807/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462674" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Network of novel candidate Ras inhibitor phosphatases and Ras kinases inferred from interaction patterns. Knockdowns of phosphatase genes (orange nodes) enhance the effects of Ras kinase knockdowns (blue nodes), similar to known Ras inhibitors and JNK signaling genes (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>). Edge width denotes relative strength of enhancement.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-6"><h2 class="">Discussion</h2><p id="p-26">We have used a strategy that combines patterns of epistasis and pleiotropy to derive a large-scale network mapping how mutations in signaling genes interact to affect distinct phenotypes. The approach integrates information in multiple phenotypes to resolve ambiguities in model interpretation that arise when individual phenotypes are analyzed in isolation. We have demonstrated the applicability of this strategy to large-scale data, in this case involving 4278 cell lines that mix 93 genetic perturbations. Significant knockdown-to-knockdown interactions were obtained among 68 knockdowns even though each was fairly rare in the population (present in 2.2% of lines). While the original analysis of these data (<a id="xref-ref-18-12" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>) revealed patterns of genetic interactions separately for each phenotype, this analysis combines the multiphenotype patterns of epistasis to derive directional models of genetic interactions that are consistent across all three phenotypes. This additional mathematical analysis recovered previous findings and was used to propose additional functional relationships between gene pairs.</p><p id="p-27">Many of the interactions in the network (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>) recapitulated known relationships. Genes that function in the same pathways show consistent, suppressive genetic interactions across a spectrum of interaction partners. This is consistent with the idea that mutations with redundant effects suppress each other, particularly in signaling pathways. The spectral consistency for interactions, which has been previously referred to as âgenetic monochromaticityâ (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Segre <em>et al.</em> 2005</a>), is maintained after their reparametrization in terms of directional influences. A similar spectral consistency is demonstrated for enhancing interactions from pairs spanning multiple pathways. We interpret this enhancement as a signature of an antagonistic relationship. For instance, a knockdown of a Ras inhibitor will increase Ras signaling activity, and therefore a subsequent Ras knockdown will have an enhanced effect since the signaling baseline is increased (<em>cf</em>. <em>PTP-ER</em>, Results). Following this interpretation, in addition to spectral consistency, these interactions can map activation and suppression across the Ras and JNK pathways. The specificity represents <em>de novo</em> derivation of key functional relationships that are more difficult to detect when analyzing each phenotype independently.</p><p id="p-28">We note that classical synthetic effects, in which two perturbations with insignificant main effects generate a strong effect in combination, would also produce enhancing interactions. We did not observe any such cases in our data. When we broaden the definition to include knockdowns with main effects and greater than additive combinatorial effects, all with the same sign (positive or negative), we find only two candidate interactions. <em>Gap1</em> knockdown enhances <em>Rho1</em> knockdown activity and both positively affect nuclear area and nuclear intensity; however they show opposite effects on cell number and therefore the effect is not consistently synthetic (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>). A less ambiguous candidate is the strong enhancement of <em>kay</em> knockdown effects by <em>Src42A</em> knockdown. Both perturbations significantly decrease nuclear intensity, and this effect becomes significantly greater when the knockdowns are combined. These genes, both annotated as JNK pathway elements, provide the only unambiguous instance of synthesis in the data.</p><p id="p-29">Our analysis provides new and refined information on gene-gene relationships and places this information in a network context. At a single-interaction level, we identified novel candidate Ras inhibitors and coactivators (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). At a network-wide level, these genes were spectrally consistent with other genes of similar known function (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1/FigureS3.pdf">Figure S3</a>). Thus, although prior information is usually helpful in placing new candidate mutations in a functional context, the interactions we inferred represent a <em>de novo</em> model of enhancement or suppression with functional implications. Furthermore, some of the network inferences propose refinements of current knowledge, such as the hypothesis that <em>Rho1</em> is primarily a Ras signaling gene in the context of the measured phenotypes. Likewise, the interaction patterns and phenotype influences of the <em>kayak</em> (<em>kay</em>) knockdown suggest that its functional role in cell proliferation is more similar to Ras signaling genes than JNK cascade genes, in contrast to its similarity to JNK in dorsal closure (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harden 2002</a>). Specifically, the <em>kay</em> knockdown resembled Ras signaling genes by suppressing the knockdown effect of multiple Ras signaling genes, being enhanced by the knockdown of multiple JNK cascade genes, and having a negative main effect on cell number (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-30">In particular, our analysis suggested novel Ras inhibitors not detected the previous study using the same data (<a id="xref-ref-18-13" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>). In most cases, inhibitor knockdowns redundantly enhanced multiple kinases (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). This could be due to non-specific phosphatase activity, but is potentially a result of functional redundancy among the kinases that leads to similar genetic interactions between a phosphatase and both its direct target and genes that cofunction with that target in a signaling cascade. One possible avenue to resolve this ambiguous redundancy would be to extend the interaction analysis beyond independent pair-wise analysis to models that find the most parsimonious model for three or more genes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carter <em>et al.</em> 2007</a>). We note that the majority of these phosphatase knockdowns significantly suppressed the <em>Gap1</em> knockdown, providing evidence of redundant function with a known <em>Ras85D</em> inhibitor. We also note that a number of additional enhancing interactions from phosphatases to kinases that did not meet our significance threshold were also detected, providing additional evidence of functional coherence but target non-specificity. However, there does seem to be limited specificity in that <em>CG3632</em> and <em>Ptp99A</em> enhance <em>drk</em> and <em>Sos</em>, genes that fall relatively upstream in the KEGG-annotated Ras pathway, while <em>CG7115</em> and <em>Pdp</em> knockdowns most strongly enhance the relatively downstream pathway knockdown <em>Ras85D</em>. In contrast, phosphatase knockdowns <em>l(1)G0232</em> and <em>CG3530</em> enhance the <em>Pvr</em> knockdown, which is not placed in the KEGG-annotated Ras cascade. The additional information that the <em>Pvr</em> knockdown tends to unidirectionally suppress the KEGG pathway genes suggests that <em>Pvr</em> potentially acts downstream of the KEGG-annotated pathway and is deactivated by the phosphatases <em>l(1)G0232</em> and <em>CG3530</em>. Thus, even with the ambiguities arising from pair-wise models, we obtained a degree of specificity in hypotheses for Ras inhibitor activity.</p><p id="p-31">The network of directional influences suggests hypotheses that are often amenable to experimental testing (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Carter <em>et al.</em> 2007</a>). In contrast with non-directional associations such as coexpression, statistical epistasis, or synthetic lethality, a directional influence identifies a driver mutation that affects the activity of its target, a second gene that, in turn, affects the phenotype. With this in mind, we can predict that removing the influence with an experimental perturbation, such as an exogenous inhibitor molecule, would nullify the effect of mutations in the target gene. Furthermore, the effect type (enhancement or suppression) specifies the nature of the effect on the target gene. We can therefore predict that overexpression of a novel Ras inhibitor will not only affect, but specifically reduce the relevant activity of its enhanced targets, whereas perturbations of the targets are not expected to affect the Ras inhibitor. For example, overexpression of <em>Ptp99A</em> will specifically inhibit the activity of Ras signaling proteins <em>Cka</em>, <em>drk</em>, <em>mts</em>, and <em>Sos</em>. In this way, a network of directional genetic interactions could prove useful in the selection of candidate genes for validation and, in disease-related networks, the prioritization of therapeutic targets.</p><p id="p-32">Our analysis was limited by the genotypes and specific phenotypes selected in the original study. Although the three phenotypes used exhibit differential interaction patterns (<a id="xref-ref-18-14" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2011</a>), their high correlation limited the complementary information that could better resolve directional interactions. By probing additional phenotypes, new interactions might be detected and existing interactions refined. For instance, we speculate that mutual cosuppression among Ras signal knockdowns is potentially due to measuring insufficiently distinct outputs. Combining detailed assays of eye development and terminal patterns might have revealed more specific roles and pathway order for canonical Ras signaling proteins. Furthermore, although the genetic perturbations are limited to MAPK signaling genes, the methods could be applied to a wider screen or population-based analysis to identify additional pathway relationships. As data collection in experimental systems and clinical studies becomes more diverse and quantitative, our analytical method has the potential to derive more refined and precise network models.</p><p id="p-33">The approach we have applied derives quantitative, directional models of how each gene knockdown modifies the effects of each other knockdown. It represents an alternative to guilt-by-association methods that infer function by scoring correlations across large-scale data. Taken together as a network, the interactions form consistent patterns between and within gene modules. Large-scale data and prior information is helpful but not necessary because guilt precedes association in our analysis. This allowed us, for instance, to propose new candidate Ras inhibitors that are consistent with known Ras inhibitors, but do not necessarily depend on this prior knowledge for functional inference.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">I thank Chad Meyers and Gary Churchill for fruitful discussions and Tonnya Norwood for a critical reading of the manuscript. This work was supported by National Institute of General Medical Sciences (NIGMS) grants K25 GM079404 and P50 GM076468. The content is the sole responsibility of the author and does not necessarily represent the official views of NIGMS or the National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005710/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-35"><em>Communicating editor: F. P. Roth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-01-21"><span class="received-label">Received </span>January 21, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-02-24"><span class="accepted-label">Accepted </span>February 24, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Carter</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andreux</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koutnikova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houtkooper</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Champy</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Systems genetics of metabolism: the use of the BXD murine reference panel for multiscalar integration of traits.</span> <abbr class="cit-jnl-abbrev">Cell</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.2" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>â<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.3" data-doi="10.1016/0168-9525(92)90263-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avery</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Ordering gene function: the interpretation of epistasis in regulatory hierarchies.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">312</span>â<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DAvery%26rft.auinit1%253DL.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D312%26rft.epage%253D316%26rft.atitle%253DOrdering%2Bgene%2Bfunction%253A%2Bthe%2Binterpretation%2Bof%2Bepistasis%2Bin%2Bregulatory%2Bhierarchies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9525%252892%252990263-4%26rft_id%253Dinfo%253Apmid%252F1365397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9525(92)90263-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1365397&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JK24800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-book" id="cit-3.5.807.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bevington</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span>â<span class="cit-source">Data Reduction and Error Analysis for the Physical Sciences</span>. <span class="cit-publ-name">McGraw-Hill</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.5" data-doi="10.1038/nrg2085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Exploring genetic interactions and networks with yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBoone%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D437%26rft.epage%253D449%26rft.atitle%253DExploring%2Bgenetic%2Binteractions%2Band%2Bnetworks%2Bwith%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2085%26rft_id%253Dinfo%253Apmid%252F17510664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17510664&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.6" data-doi="10.1186/jbiol58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koeva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A global analysis of genetic interactions in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiology%2B%2528Online%2529%26rft.stitle%253DJ%2BBiol%26rft.aulast%253DByrne%26rft.auinit1%253DA.%2BB.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DA%2Bglobal%2Banalysis%2Bof%2Bgenetic%2Binteractions%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fjbiol58%26rft_id%253Dinfo%253Apmid%252F17897480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/jbiol58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17897480&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.7" data-doi="10.1038/msb4100137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prinz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelby</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzolf</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Prediction of phenotype and gene expression for combinations of mutations.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DCarter%26rft.auinit1%253DG.%2BW.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D96%26rft.epage%253D96%26rft.atitle%253DPrediction%2Bof%2Bphenotype%2Band%2Bgene%2Bexpression%2Bfor%2Bcombinations%2Bof%2Bmutations%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100137%26rft_id%253Dinfo%253Apmid%252F17389876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjY6IjMvMS85NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS84MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.8" data-doi="10.1371/journal.pcbi.1000347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galas</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galitski</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Maximal extraction of biological information from genetic interaction data.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DCarter%26rft.auinit1%253DG.%2BW.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De1000347%26rft.epage%253De1000347%26rft.atitle%253DMaximal%2Bextraction%2Bof%2Bbiological%2Binformation%2Bfrom%2Bgenetic%2Binteraction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000347%26rft_id%253Dinfo%253Apmid%252F19343223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19343223&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.9" data-doi="10.1371/journal.pgen.1003010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galitski</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Use of pleiotropy to model genetic interactions in a population.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCarter%26rft.auinit1%253DG.%2BW.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1003010%26rft.epage%253De1003010%26rft.atitle%253DUse%2Bof%2Bpleiotropy%2Bto%2Bmodel%2Bgenetic%2Binteractions%2Bin%2Ba%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003010%26rft_id%253Dinfo%253Apmid%252F23071457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071457&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.10" data-doi="10.1016/j.cell.2012.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mias</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li-Pook-Than</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Personal omics profiling reveals dynamic molecular and medical phenotypes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1293</span>â<span class="cit-lpage">1307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChen%26rft.auinit1%253DR.%26rft.volume%253D148%26rft.issue%253D6%26rft.spage%253D1293%26rft.epage%253D1307%26rft.atitle%253DPersonal%2Bomics%2Bprofiling%2Breveals%2Bdynamic%2Bmolecular%2Band%2Bmedical%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.02.009%26rft_id%253Dinfo%253Apmid%252F22424236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22424236&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301889500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.11" data-doi="10.1038/nature05649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maas</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fillingham</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">446</span>: <span class="cit-fpage">806</span>â<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D446%26rft.issue%253D7137%26rft.spage%253D806%26rft.epage%253D810%26rft.atitle%253DFunctional%2Bdissection%2Bof%2Bprotein%2Bcomplexes%2Binvolved%2Bin%2Byeast%2Bchromosome%2Bbiology%2Busing%2Ba%2Bgenetic%2Binteraction%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05649%26rft_id%253Dinfo%253Apmid%252F17314980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17314980&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245626800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.12" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>â<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvODA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.13" data-doi="10.1186/gb-2005-6-4-r38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drees</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorsson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rives</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Derivation of genetic interaction networks from quantitative phenotype data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DDrees%26rft.auinit1%253DB.%2BL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253DR38%26rft.epage%253DR38%26rft.atitle%253DDerivation%2Bof%2Bgenetic%2Binteraction%2Bnetworks%2Bfrom%2Bquantitative%2Bphenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-4-r38%26rft_id%253Dinfo%253Apmid%252F15833125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-4-r38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15833125&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-web" id="cit-3.5.807.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FlyBase</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Gene ontology annotation in FlyBase through association of InterPro records with GO terms</span>, <a href="http://www.flybase.org">http://www.flybase.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.15" data-doi="10.1371/journal.pcbi.1002444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">âGuilt by associationâ is the exception rather than the rule in gene networks.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGillis%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002444%26rft.epage%253De1002444%26rft.atitle%253D%2522Guilt%2Bby%2Bassociation%2522%2Bis%2Bthe%2Bexception%2Brather%2Bthan%2Bthe%2Brule%2Bin%2Bgene%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002444%26rft_id%253Dinfo%253Apmid%252F22479173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22479173&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harden</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Signaling pathways directing the movement and fusion of epithelial sheets: lessons from dorsal closure in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Differentiation</abbr> <span class="cit-vol">70:</span> <span class="cit-fpage">181</span>â<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDifferentiation%26rft.volume%253D70%253A%26rft.spage%253D181%26rft.atitle%253DSIGNALING%2BPATHWAYS%2BDIRECTING%2BTHE%2BMOVEMENT%2BAND%2BFUSION%2BOF%2BEPITHELIAL%2BSHEETS%253A%2BLESSONS%2BFROM%2BDORSAL%2BCLOSURE%2BIN%2BDROSOPHILA%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-article-title">A simple sequentially rejective multiple test procedure.</span> <abbr class="cit-jnl-abbrev">Scand. J. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">65</span>â<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BStat.%26rft.volume%253D6%26rft.spage%253D65%26rft.atitle%253DA%2BSIMPLE%2BSEQUENTIALLY%2BREJECTIVE%2BMULTIPLE%2BTEST%2BPROCEDURE.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1979JY78700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.18" data-doi="10.1038/nmeth.1581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Mapping of signaling networks through synthetic genetic interaction analysis by RNAi.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">341</span>â<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DHorn%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D341%26rft.epage%253D346%26rft.atitle%253DMapping%2Bof%2Bsignaling%2Bnetworks%2Bthrough%2Bsynthetic%2Bgenetic%2Binteraction%2Banalysis%2Bby%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1581%26rft_id%253Dinfo%253Apmid%252F21378980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21378980&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288940300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.19" data-doi="10.1016/S1097-2765(01)80006-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karim</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">PTP-ER, a novel tyrosine phosphatase, functions downstream of Ras1 to downregulate MAP kinase during <em>Drosophila</em> eye development.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">741</span>â<span class="cit-lpage">750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DKarim%26rft.auinit1%253DF.%2BD.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D741%26rft.epage%253D750%26rft.atitle%253DPTP-ER%252C%2Ba%2Bnovel%2Btyrosine%2Bphosphatase%252C%2Bfunctions%2Bdownstream%2Bof%2BRas1%2Bto%2Bdownregulate%2BMAP%2Bkinase%2Bduring%2BDrosophila%2Beye%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252980006-X%26rft_id%253Dinfo%253Apmid%252F10394362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)80006-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10394362&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081123200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.20" data-doi="10.1038/ng1844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crombie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tischler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Systematic mapping of genetic interactions in <em>Caenorhabditis elegans</em> identifies common modifiers of diverse signaling pathways.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">896</span>â<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLehner%26rft.auinit1%253DB.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D896%26rft.epage%253D903%26rft.atitle%253DSystematic%2Bmapping%2Bof%2Bgenetic%2Binteractions%2Bin%2BCaenorhabditis%2Belegans%2Bidentifies%2Bcommon%2Bmodifiers%2Bof%2Bdiverse%2Bsignaling%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1844%26rft_id%253Dinfo%253Apmid%252F16845399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845399&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.21" data-doi="10.1152/physiolgenomics.00044.2010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Epistasis contributes to the genetic buffering of plasma HDL cholesterol in mice.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">42A</span>: <span class="cit-fpage">228</span>â<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00044.2010%26rft_id%253Dinfo%253Apmid%252F20858711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo5OiI0MkEvNC8yMjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvODA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pajewski</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Six degrees of epistasis: statistical network models for GWAS</span>. <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DMcKinney%26rft.auinit1%253DB.%2BA.%26rft.volume%253D2%26rft.spage%253D109%26rft.epage%253D109%26rft.atitle%253DSix%2BDegrees%2Bof%2BEpistasis%253A%2BStatistical%2BNetwork%2BModels%2Bfor%2BGWAS.%26rft_id%253Dinfo%253Apmid%252F22303403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22303403&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.23" data-doi="10.1371/journal.pcbi.1002559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaut</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Multiple genetic interaction experiments provide complementary information useful for gene function prediction.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMichaut%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002559%26rft.epage%253De1002559%26rft.atitle%253DMultiple%2Bgenetic%2Binteraction%2Bexperiments%2Bprovide%2Bcomplementary%2Binformation%2Buseful%2Bfor%2Bgene%2Bfunction%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002559%26rft_id%253Dinfo%253Apmid%252F22737063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22737063&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.24" data-doi="10.1016/S0959-437X(99)80071-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noselli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agnes</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Roles of the JNK signaling pathway in <em>Drosophila</em> morphogenesis.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">466</span>â<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNoselli%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D466%26rft.epage%253D472%26rft.atitle%253DRoles%2Bof%2Bthe%2BJNK%2Bsignaling%2Bpathway%2Bin%2BDrosophila%2Bmorphogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252899%252980071-9%26rft_id%253Dinfo%253Apmid%252F10449347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(99)80071-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10449347&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082025900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.25" data-doi="10.1101/gr.147000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reifsnyder</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Maternal environment and genotype interact to establish diabesity in mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1568</span>â<span class="cit-lpage">1578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DReifsnyder%26rft.auinit1%253DP.%2BC.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D1568%26rft.epage%253D1578%26rft.atitle%253DMaternal%2BEnvironment%2Band%2BGenotype%2BInteract%2Bto%2BEstablish%2BDiabesity%2Bin%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.147000%26rft_id%253Dinfo%253Apmid%252F11042154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMC8xNTY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzgwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.26" data-doi="10.1111/j.1469-1809.2010.00630.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Using biological knowledge to uncover the mystery in the search for epistasis in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">172</span>â<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DRitchie%26rft.auinit1%253DM.%2BD.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D172%26rft.epage%253D182%26rft.atitle%253DUsing%2Bbiological%2Bknowledge%2Bto%2Buncover%2Bthe%2Bmystery%2Bin%2Bthe%2Bsearch%2Bfor%2Bepistasis%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2010.00630.x%26rft_id%253Dinfo%253Apmid%252F21158748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2010.00630.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21158748&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.27" data-doi="10.1038/nm1161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deluna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishony</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Modular epistasis in yeast metabolism.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">77</span>â<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegr%25C3%25A8%26rft.auinit1%253DD.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D83%26rft.atitle%253DModular%2Bepistasis%2Bin%2Byeast%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1161%26rft_id%253Dinfo%253Apmid%252F15592468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592468&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225997500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.28" data-doi="10.1073/pnas.0810388105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burrage</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinasac</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernest</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetic architecture of complex traits: large phenotypic effects and pervasive epistasis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">19910</span>â<span class="cit-lpage">19914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810388105%26rft_id%253Dinfo%253Apmid%252F19066216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUwLzE5OTEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81LzgwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.29" data-doi="10.1038/ng1948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">199</span>â<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOnge%26rft.auinit1%253DR.%2BP.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D206%26rft.atitle%253DSystematic%2Bpathway%2Banalysis%2Busing%2Bhigh-resolution%2Bfitness%2Bprofiling%2Bof%2Bcombinatorial%2Bgene%2Bdeletions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1948%26rft_id%253Dinfo%253Apmid%252F17206143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17206143&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.5.807.30" data-doi="10.1017/S0016672309990279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Epistatic interactions attenuate mutations affecting startle behaviour in <em>Drosophila</em> melanogaster.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">373</span>â<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DYamamoto%26rft.auinit1%253DA.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D373%26rft.epage%253D382%26rft.atitle%253DEpistatic%2Binteractions%2Battenuate%2Bmutations%2Baffecting%2Bstartle%2Bbehaviour%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990279%26rft_id%253Dinfo%253Apmid%252F19968911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19968911&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F807.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/5/807.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/795" title="Distribution of Long-Range Linkage Disequilibrium and Tajimaâs D Values in Scandinavian Populations of Norway Spruce (Picea abies)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/815" title="A Targeted Genetic Modifier Screen Links the SWI2/SNF2 Protein Domino to Growth and Autophagy Genes in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/5"><span>Volume 3 Issue 5, May 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/5.cover-source.jpg?itok=NJpwBPbV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/807" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j_KCz107zmJ3-dqWnuhx7ZKnupmRn75gPQcUhKLhVcw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/5/807" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/5/807" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/5/807" class="active" data-icon-position="" data-hide-link-title="0">Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459468" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8iUwGWJzbG8lqxCcI4mWpMPmzb2UhTnqJQ0qBCKscOo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/807.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/5/807" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wW1URvOQ_3aPD59MNul8OWcTsF4UpkhsScBu67WLNAE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/5/807" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/807.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459468" id="node459468--2" data-pisa="ggg;3/5/807" data-pisa-master="ggg;g3.113.005710" data-apath="/ggg/3/5/807.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Gene Function and Network Organization in <em>Drosophila</em> Signaling by Combined Analysis of Pleiotropy and Epistasis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory W. Carter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">807-814;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005710 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459468--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory W. Carter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BCarter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carter%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BW.%2BCarter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greg.carter@jax.org" class="" data-icon-position="" data-hide-link-title="0">greg.carter@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/5/807" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/5/807&amp;title=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/5/807&amp;title=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/5/807&amp;title=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/5/807&amp;text=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/5/807&amp;title=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/5/807&amp;t=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/5/807&amp;title=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/5/807&amp;title=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459468/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459468" id="node459468--3" data-pisa="ggg;3/5/807" data-pisa-master="ggg;g3.113.005710" data-apath="/ggg/3/5/807.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Gene Function and Network Organization in <em>Drosophila</em> Signaling by Combined Analysis of Pleiotropy and Epistasis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory W. Carter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">807-814;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005710 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459468--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory W. Carter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BCarter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carter%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BW.%2BCarter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greg.carter@jax.org" class="" data-icon-position="" data-hide-link-title="0">greg.carter@jax.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=807&amp;rft.epage=814&amp;rft.atitle=Inferring%20Gene%20Function%20and%20Network%20Organization%20in%20Drosophila%20Signaling%20by%20Combined%20Analysis%20of%20Pleiotropy%20and%20Epistasis&amp;rft.volume=3&amp;rft.issue=5&amp;rft.date=2013-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Carter%2C+Gregory+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459468/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459468/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459468/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459468/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459468/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459468/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459468/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459468/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459468/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459468/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459468/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459468/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/5/807" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/5/807.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/5/807.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/5/807.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/5/807.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
