<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/5/795" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/5/795.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/5/795.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/5/795.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/5/795.atom" />
<meta name="HW.pisa" content="ggg;3/5/795" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (Picea abies)" />
<meta name="DC.Identifier" content="10.1534/g3.112.005462" />
<meta name="DC.Date" content="2013-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Larsson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The site frequency spectrum of mutations (SFS) and linkage disequilibrium (LD) are the two major sources of information in population genetics studies. In this study we focus on the levels of LD and the SFS and on the effect of sample size on summary statistics in 10 Scandinavian populations of Norway spruce. We found that previous estimates of a low level of LD were highly influenced by both sampling strategy and the fact that data from multiple loci were analyzed jointly. Estimates of LD were in fact heterogeneous across loci and increased within individual populations compared with the estimate from the total data. The variation in levels of LD among populations most likely reflects different demographic histories, although we were unable to detect population structure by using standard approaches. As in previous studies, we also found that the SFS-based test Tajima’s D was highly sensitive to sample size, revealing that care should be taken to draw strong conclusions from this test when sample size is small. In conclusion, the results from this study are in line with recent studies in other conifers that have revealed a more complex and variable pattern of LD than earlier studies suggested and with studies in trees and humans that suggest that Tajima’s D is sensitive to sample size. This has large consequences for the design of future association and population genetic studies in Norway spruce." />
<meta name="DC.Contributor" content="Hanna Larsson" />
<meta name="DC.Contributor" content="Thomas Källman" />
<meta name="DC.Contributor" content="Niclas Gyllenstrand" />
<meta name="DC.Contributor" content="Martin Lascoux" />
<meta name="article:published_time" content="2013-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (Picea abies)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The site frequency spectrum of mutations (SFS) and linkage disequilibrium (LD) are the two major sources of information in population genetics studies. In this study we focus on the levels of LD and the SFS and on the effect of sample size on summary statistics in 10 Scandinavian populations of Norway spruce. We found that previous estimates of a low level of LD were highly influenced by both sampling strategy and the fact that data from multiple loci were analyzed jointly. Estimates of LD were in fact heterogeneous across loci and increased within individual populations compared with the estimate from the total data. The variation in levels of LD among populations most likely reflects different demographic histories, although we were unable to detect population structure by using standard approaches. As in previous studies, we also found that the SFS-based test Tajima’s D was highly sensitive to sample size, revealing that care should be taken to draw strong conclusions from this test when sample size is small. In conclusion, the results from this study are in line with recent studies in other conifers that have revealed a more complex and variable pattern of LD than earlier studies suggested and with studies in trees and humans that suggest that Tajima’s D is sensitive to sample size. This has large consequences for the design of future association and population genetic studies in Norway spruce.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/05/01" />
<meta name="citation_mjid" content="ggg;3/5/795" />
<meta name="citation_id" content="3/5/795" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/5/795" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/5/795.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/5/795.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/5/795.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005462" />
<meta name="citation_pmid" content="23550126" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="795" />
<meta name="citation_lastpage" content="806" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hanna Larsson" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden" />
<meta name="citation_author_email" content="Hanna.Larsson@ebc.uu.se" />
<meta name="citation_author_email" content="larsson.h77@gmail.com" />
<meta name="citation_author" content="Thomas Källman" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden" />
<meta name="citation_author" content="Niclas Gyllenstrand" />
<meta name="citation_author_institution" content="Department of Plant Biology and Forest Genetics, Swedish University of Agricultural Sciences, Uppsala, 750 07 Uppsala, Sweden" />
<meta name="citation_author" content="Martin Lascoux" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=S. Berlin;citation_author=J. Fogelqvist;citation_author=M. Lascoux;citation_author=U. Lagercrantz;citation_author=A. C. Rönnberg-Wästljung;citation_title=Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf;citation_pages=387-400;citation_volume=1;citation_year=2011;citation_issue=5;citation_pmid=22384349;citation_doi=10.1534/g3.111.000539" />
<meta name="citation_reference" content="citation_journal_title=Quat. Sci. Rev.;citation_author=H. A. Binney;citation_author=K. J. Willis;citation_author=M. E. Edwards;citation_author=S. A. Bhagwat;citation_author=P. M. Anderson;citation_title=The distribution of late-Quaternary woody taxa in northern Eurasia: evidence from a new macrofossil database;citation_pages=2445-2464;citation_volume=28;citation_year=2009;citation_doi=10.1016/j.quascirev.2009.04.016" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. R. Brown;citation_author=G. P. Gill;citation_author=R. J. Kuntz;citation_author=C. H. Langley;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in loblolly pine;citation_volume=101;citation_year=2004;citation_pmid=15477602;citation_doi=10.1073/pnas.0404231101" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. Chen;citation_author=T. Källman;citation_author=N. Gyllenstrand;citation_author=M. Lascoux;citation_title=New insights on the speciation history and nucleotide diversity of three boreal spruce species and a Tertiary relict.;citation_pages=3-14;citation_volume=104;citation_year=2010;citation_issue=1;citation_pmid=19639012;citation_doi=10.1038/hdy.2009.88" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Chen;citation_author=T. Källman;citation_author=X. Ma;citation_author=N. Gyllenstrand;citation_author=G. Zaina;citation_title=Disentangling the roles of history and local selection in shaping clinal variation of allele frequencies and gene expression in Norway spruce (Picea abies);citation_volume=191;citation_year=2012;citation_pmid=22542968;citation_doi=10.1534/genetics.112.140749" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. D. Cutter;citation_author=G. X. Wang;citation_author=H. Ai;citation_author=Y. Peng;citation_title=Influence of finite-sites mutation, population subdivision and sampling schemes on patterns of nucleotide polymorphism for species with molecular hyperdiversity.;citation_pages=1345-1359;citation_volume=21;citation_year=2012;citation_issue=6;citation_pmid=22320847;citation_doi=10.1111/j.1365-294X.2012.05475.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Ewing;citation_author=P. Green;citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities;citation_volume=8;citation_year=1998;citation_pmid=9521922;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=B. Ewing;citation_author=L. Hillier;citation_author=M. C. Wendl;citation_author=P. Green;citation_title=Base-Calling of Automated Sequencer Traces Using Phred.  I. Accuracy Assessment;citation_pages=175-185;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9521921;citation_doi=10.1101/gr.8.3.175" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=L. Excoffier;citation_author=H. E. L. Lischer;citation_title=Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows;citation_pages=564-567;citation_volume=10;citation_year=2010;citation_doi=10.1111/j.1755-0998.2010.02847.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. S. Gaut;citation_author=S. I. Wright;citation_author=C. Rizzon;citation_author=J. Dvorak;citation_author=L. K. Andersson;citation_title=Recombination: an underappreciated factor in the evolution of plant genomes.;citation_pages=77-84;citation_volume=8;citation_year=2007;citation_issue=1;citation_pmid=17173059;citation_doi=10.1038/nrg1970" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. González-Martínez;citation_author=E. Ersoz;citation_author=G. R. Brown;citation_author=N. C. Wheeler;citation_author=D. B. Neale;citation_title=DNA sequence variation and selection of tag single-nucleotide polymorphisms at candidate genes for dought-stress response in Pinus taeda L;citation_volume=172;citation_year=2006;citation_pmid=16387885;citation_doi=10.1534/genetics.105.047126" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=D. Gordon;citation_author=C. Abajian;citation_author=P. Green;citation_title=Consed: A Graphical Tool for Sequence Finishing;citation_pages=195-202;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9521923;citation_doi=10.1101/gr.8.3.195" />
<meta name="citation_reference" content="citation_journal_title=Yi chuan = Hereditas / Zhongguo yi chuan xue hui bian ji;citation_journal_abbrev=Yi Chuan;citation_author=A. Y. Guo;citation_author=Q. H. Zhu;citation_author=X. Chen;citation_author=J. C. Luo;citation_title=[GSDS: a gene structure display server];citation_pages=1023-1026;citation_volume=29;citation_year=2007;citation_issue=8;citation_pmid=17681935;citation_doi=10.1360/yc-007-1023" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=P. W. Hedrick;citation_author=S. Kumar;citation_title=Mutation and linkage disequilibrium in human mtDNA.;citation_pages=969-972;citation_volume=9;citation_year=2001;citation_issue=12;citation_pmid=11840186;citation_doi=10.1038/sj.ejhg.5200735" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Heuertz;citation_author=E. De Paoli;citation_author=T. Källman;citation_author=H. Larsson;citation_author=I. Jurman;citation_title=Multilocus patterns of nucleotide diversity, linkage disequilibrium and demographic history of Norway spruce [Picea abies (L.) Karst];citation_volume=174;citation_year=2006;citation_pmid=17057229;citation_doi=10.1534/genetics.106.065102" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variances and covariances of squared linkage disequilibria in finite populations.;citation_pages=54-78;citation_volume=33;citation_year=1988;citation_issue=1;citation_pmid=3376052;citation_doi=10.1016/0040-5809(88)90004-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. A. Holliday;citation_author=M. Yuen;citation_author=K. Ritland;citation_author=S. N. Aitken;citation_title=Postglacial history of a widespread conifer produces inverse clines in selective neutrality tests.;citation_pages=3857-3864;citation_volume=19;citation_year=2010;citation_issue=18;citation_pmid=20738783;citation_doi=10.1111/j.1365-294X.2010.04767.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=J. P. Jaramillo-Correa;citation_author=M. Verdú;citation_author=S. C. González-Martìnez;citation_title=The contribution of recombination to heterozygosity differs among plant evolutionary lineages and life-forms.;citation_pages=22-22;citation_volume=10;citation_year=2010;citation_pmid=20100325;citation_doi=10.1186/1471-2148-10-22" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. Karlgren;citation_author=N. Gyllenstrand;citation_author=T. Källman;citation_author=J. F. Sundström;citation_author=D. Moore;citation_title=Evolution of the PEBP gene family in plants: functional diversification in seed plant evolution;citation_volume=156;citation_year=2011;citation_pmid=21642442;citation_doi=10.1104/pp.111.176206" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. V. Krutovsky;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in cold-hardiness- and wood quality-related candidate genes in Douglas fir;citation_volume=171;citation_year=2005;citation_pmid=16157674;citation_doi=10.1534/genetics.105.044420" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=C. Lepoittevin;citation_author=L. Harvengt;citation_author=C. Plomion;citation_author=P. Garnier-Gere;citation_title=Association mapping for growth, straightness and wood chemistry traits in the Pinus pinaster Aquitaine breeding population;citation_pages=113-126;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-011-0426-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_title=THE INTERACTION OF SELECTION AND LINKAGE. I. GENERAL CONSIDERATIONS; HETEROTIC MODELS;citation_pages=49-67;citation_volume=49;citation_year=1964;citation_issue=1;citation_pmid=17248194" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: A software for comprehensive analysis of DNA polymorphism data;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=F. Marroni;citation_author=S. Pinosio;citation_author=G. Zaina;citation_author=F. Fogolari;citation_author=N. Felice;citation_title=Nucleotide diversity and linkage disequilibrium in Populus nigra cinnamyl alcohol dehydrogenase (CAD4) gene;citation_pages=1011-1023;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-011-0391-5" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=G. A. McVean;citation_title=What do patterns of genetic variability reveal about mitochondrial recombination?;citation_pages=613-620;citation_volume=87;citation_year=2001;citation_issue=Pt 6;citation_pmid=11903556;citation_doi=10.1046/j.1365-2540.2001.00965.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. McVean;citation_author=P. Awadalla;citation_author=P. Fearnhead;citation_title=A Coalescent-Based Method for Detecting and Estimating Recombination From Gene Sequences;citation_pages=1231-1241;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901136" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Moritsuka;citation_author=Y. Hisataka;citation_author=M. Tamura;citation_author=K. Uchiyama;citation_author=A. Watanabe;citation_title=Extended linkage disequilibrium in noncoding regions in a conifer, Cryptomeria japonica;citation_volume=190;citation_year=2012;citation_pmid=22209904;citation_doi=10.1534/genetics.111.136697" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M.-C. Namroud;citation_author=C. Guillet-Claude;citation_author=J. Mackay;citation_author=N. Isabel;citation_author=J. Bousquet;citation_title=Molecular evolution of regulatory genes in spruces from different species and continents: heterogeneous patterns of linkage disequilibrium and selection but correlated recent demographic changes.;citation_pages=371-386;citation_volume=70;citation_year=2010;citation_issue=4;citation_pmid=20354847;citation_doi=10.1007/s00239-010-9335-1" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=D. B. Neale;citation_author=O. Savolainen;citation_title=Association genetics of complex traits in conifers.;citation_pages=325-330;citation_volume=9;citation_year=2004;citation_issue=7;citation_pmid=15231277;citation_doi=10.1016/j.tplants.2004.05.006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. R. Nelson;citation_author=D. Wegmann;citation_author=M. G. Ehm;citation_author=D. Kessner;citation_author=P. St Jean;citation_title=An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people;citation_volume=337;citation_year=2012;citation_pmid=22604722;citation_doi=10.1126/science.1217876" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=N. Pavy;citation_author=M. C. Namroud;citation_author=F. Gagnon;citation_author=N. Isabel;citation_author=J. Bousquet;citation_title=The heterogeneous levels of linkage disequilibrium in white spruce genes and comparative analysis with other conifers.;citation_pages=273-284;citation_volume=108;citation_year=2012;citation_issue=3;citation_pmid=21897435;citation_doi=10.1038/hdy.2011.72" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Pyhäjärvi;citation_author=M. R. García-Gil;citation_author=T. Knürr;citation_author=M. Mikkonen;citation_author=W. Wachowiak;citation_title=Demographic History Has Influenced Nucleotide Diversity in European Pinus sylvestris Populations;citation_pages=1713-1724;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=18039881;citation_doi=10.1534/genetics.107.077099" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=T. Pyhäjärvi;citation_author=S. T. Kujala;citation_author=O. Savolainen;citation_title=Revisiting protein heterozygosity in plants - nucleotide diversity in allozyme coding genes of conifer Pinus sylvestris;citation_pages=385-397;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0340-8" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Städler;citation_author=B. Haubold;citation_author=C. Merino;citation_author=W. Stephan;citation_author=P. Pfaffelhuber;citation_title=The impact of sampling schemes on the site frequency spectrum in nonequilibrium subdivided populations;citation_volume=182;citation_year=2009;citation_pmid=19237689;citation_doi=10.1534/genetics.108.094904" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=K. St. Onge;citation_author=A. E. Palmé;citation_author=S. I. Wright;citation_author=M. Lascoux;citation_title=Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories;citation_pages=803-814;citation_volume=2;citation_year=2012;citation_issue=7;citation_pmid=22870403;citation_doi=10.1534/g3.112.002410" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=H. Tachida;citation_title=Decay of linkage disequilibrium in a finite island model.;citation_pages=137-144;citation_volume=64;citation_year=1994;citation_issue=2;citation_pmid=7813904" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; genetic systems;citation_journal_abbrev=Genes Genet Syst;citation_author=H. Tachida;citation_title=Linkage disequilibrium in a population undergoing periodic fragmentation and admixture.;citation_pages=125-135;citation_volume=87;citation_year=2012;citation_issue=2;citation_pmid=22820386;citation_doi=10.1266/ggs.87.125" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. M. Tollefsrud;citation_author=J. H. Sonstebo;citation_author=C. Brochmann;citation_author=O. Johnsen;citation_author=T. Skroppa;citation_title=Combined analysis of nuclear and mitochondrial markers provide new insight into the genetic structure of North European Picea abies.;citation_pages=549-562;citation_volume=102;citation_year=2009;citation_issue=6;citation_pmid=19259114;citation_doi=10.1038/hdy.2009.16" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=M. Väliranta;citation_author=A. Kaakinen;citation_author=P. Kuhry;citation_author=S. Kultti;citation_author=J. S. Salonen;citation_title=Scattered late-glacial and early Holocene tree populations as dispersal nuclei for forest development in north-eastern European Russia;citation_pages=922-932;citation_volume=38;citation_year=2011;citation_doi=10.1111/j.1365-2699.2010.02448.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. F. Voight;citation_author=A. M. Adams;citation_author=L. A. Frisse;citation_author=Y. Qian;citation_author=R. R. Hudson;citation_title=Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes;citation_volume=102;citation_year=2005;citation_pmid=16352722;citation_doi=10.1073/pnas.0507325102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Zeng;citation_author=Y. X. Fu;citation_author=S. Shi;citation_author=C. I. Wu;citation_title=Statistical tests for detecting positive selection by utilizing high-frequency variants;citation_volume=174;citation_year=2006;citation_pmid=16951063;citation_doi=10.1534/genetics.106.061432" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (Picea abies)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
<meta name="twitter:description" content="The site frequency spectrum of mutations (SFS) and linkage disequilibrium (LD) are the two major sources of information in population genetics studies. In this study we focus on the levels of LD and the SFS and on the effect of sample size on summary statistics in 10 Scandinavian populations of Norway spruce. We found that previous estimates of a low level of LD were highly influenced by both sampling strategy and the fact that data from multiple loci were analyzed jointly. Estimates of LD were in fact heterogeneous across loci and increased within individual populations compared with the estimate from the total data. The variation in levels of LD among populations most likely reflects different demographic histories, although we were unable to detect population structure by using standard approaches. As in previous studies, we also found that the SFS-based test Tajima’s D was highly sensitive to sample size, revealing that care should be taken to draw strong conclusions from this test when sample size is small. In conclusion, the results from this study are in line with recent studies in other conifers that have revealed a more complex and variable pattern of LD than earlier studies suggested and with studies in trees and humans that suggest that Tajima’s D is sensitive to sample size. This has large consequences for the design of future association and population genetic studies in Norway spruce." />
<meta name="og-title" property="og:title" content="Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (Picea abies)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/5/795" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The site frequency spectrum of mutations (SFS) and linkage disequilibrium (LD) are the two major sources of information in population genetics studies. In this study we focus on the levels of LD and the SFS and on the effect of sample size on summary statistics in 10 Scandinavian populations of Norway spruce. We found that previous estimates of a low level of LD were highly influenced by both sampling strategy and the fact that data from multiple loci were analyzed jointly. Estimates of LD were in fact heterogeneous across loci and increased within individual populations compared with the estimate from the total data. The variation in levels of LD among populations most likely reflects different demographic histories, although we were unable to detect population structure by using standard approaches. As in previous studies, we also found that the SFS-based test Tajima’s D was highly sensitive to sample size, revealing that care should be taken to draw strong conclusions from this test when sample size is small. In conclusion, the results from this study are in line with recent studies in other conifers that have revealed a more complex and variable pattern of LD than earlier studies suggested and with studies in trees and humans that suggest that Tajima’s D is sensitive to sample size. This has large consequences for the design of future association and population genetic studies in Norway spruce." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/5.cover-source.jpg" />
    <title>Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (Picea abies) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"b0qiAZ0yOUjGIbWExMJyZHDBO_WrKww7wnp-PmZkbcI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459469","apath":"\/ggg\/3\/5\/795.atom","pisa":"ggg;3\/5\/795","ac":{"\/ggg\/3\/5\/795.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/5\/795.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/5\/795"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/5\/795"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/5\/795.abstract":true,"\/content\/3\/5\/795":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ROTMWapuKzYgy\\\\\\\/3g+geQs6C+ixOUGZw74K67TqGYcRKI47ZNNN9wehOFJQ69aMuqdou97x\\\\\\\/xFUEYgDsmvnJT70GsrwPYEvxzs6EmeciTtVfA2jH\\\\\\\/4r\\\\\\\/LoRqWwRxMzflY0OAgDegziQeva5kr2OgRHD1GjZEYHaon6ywqWpPss44\\\\\\\/rdJH4ZfYxxzqpecZwNmaEaP6YVFdSFIipxqP4yLA\\\\\\\/\\\\\\\/x08Bptv21XFCwh7130d6cHm4K5x35FILOzp\\\\\\\/dk\\\\\\\/FMipxMdI\\\\\\\/x0kLVA8D9+7M1xHFFujclJotQz586axLZ7MY7ovdtS7LWRgkWfRaQRjshrIU1oLW4rm4pLRDnAn8Km8srOfN9wXl19EXsE0vKmteP7e+kAwGBkVBHwlTuqUaZS\\\\\\\/x+ECmw1cja5KDS9arGTaW7B7x4JzYVn\\\\\\\/eBDEubn7TIhKX4JTJ7IRDIP8eNFJX\\\\\\\/\\\\\\\/5wMfawvZzKXabQTLyU7OsdmerXQMUAJEJJjBx23tW3BySQ8g3FCBe6UhBz4mw9UsVaP46SnRy0+aiVb26LYf+Lf0xje83BeB3GXOqPhHTyld02FBzN3zFj8o3z2DQtuzcMSwz4NJ\\\\\\\/YDbgHEnVrJSdgtLhhwJtSowMqbWkQifXpoTYzz8ucK9V8A3oNleWuArfAKkczvS42DweLfgl72UTL+tAOdOUjvkEyjFqJ\\\\\\\/qqZCpTXzBAiWym4D1ShUjiuRG7V2jIUEBOwt8s0KkJS6ZNQqo3sTcHXft25muywyu4EOP0nCeKZQhxAfWbE560qrlYQMQoQ0kji3TZziGCq11wudCFf0RJ55rQzv4XMbqUOdSTXePW\\\\\\\/3RpchQ6Y9z4k5T8AFpK\\\\\\\/T8pl7jtXI\\\\\\\/RhL2IVqR3xxDet9Jy4ltpY5UluPoDDcrWVlSGaf8ZCbe3oeLBgAfcIxMxyl8sdAW3DNufiQjkMlpi2TXl3B7gRzPUZxo8Ys5ainsdcqGIky71lofW6pu6sKnPuktGEzQLYhoHSPQ3aIRQOgJYfdRJpSV0U1EmJef8S3WT6cjsaVNXV6s56xHoz0M5PARU0u0uKStd2Bq7j7OIIOtPf07pwRdAzHFbpa\\\\\\\/QYwyxw\\\\\\\/zPgN9\\\\\\\/tZwV88H+4aqqO2mVk\\\\\\\/CXip1yLIGH+aJZbU+9UNZIA9G6wHnUGX1jxZNk3motMwXQl+YnAe43rj2Q3jGN+jSyxAV+pMXbeDBozFt3uB6S7JUCrXNPWyVVsQ5vyXbFOGqeqUpjo212mjNui2lTxkyv37QOwMk2lCfPs\\\\\\\/EYQ+ucRC5SEExNqEoYt2J\\\\\\\/hOYduLbJSFq\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q6l+8fRHU7iFTxtYohZLZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022790190459923a95f30f3d672d86738064c473b645814c5bc8052b22578599c98\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Dmcg8k8A+dzPodhyqorGbFA7OEMphfc3orFjFadIc2sQYIPnSbq\\\\\\\/qEf0e7UF78vsYiIlszYeHHq8xnVAiV1zwfqQSlcb9fOjzfBYoj35LkirayK2xlhvKiwtp9rvuwtPy40isKqrhnRIfaPd4RzCz0N5eWx5Q+PnvWYBOnRucplTvTqn7yNlneYWl5TmP6cGBDCBRgELAOkJqcAKkBuCMUr7JgYiEKfSQWVEkp3qCb9qxlFi5Z7Z9KMxJ0WIDPlueaAKK\\\\\\\/cf4KwmYuQ0\\\\\\\/MBgTy53FlVar6XqYht2nsNtWkfI+2Moq30wDUylD3SRylxAVPpTzUJEXqHbyqqk9O3LB0FMmETRhONLCzYcbYjktb\\\\\\\/mVQyRIOnRffbiyQXJHZ5Gd71ZU92\\\\\\\/\\\\\\\/zImUXidKyy6dUp+aqLeO7S1a1OaNQP8um\\\\\\\/hUEek5KUXWMf4ZD1DNhFxgY1vgNk\\\\\\\/cSoYENHC67W2ATj1wKAykE2z4hSJUsQQoC\\\\\\\/Sy7J+eUuVcqVfWrY9l5K+Q3NPz1034gI9PzpVZ+L712qgqC+ZCty5xdZH95ZAQ2XxJc25Gwitu8Obw4gVbIjRq7TnkZCGikW+KZ9s2b9ZgohIuvuyYlUWjvelNe+huzzTv0EbvVvdd7JfhdRS8IXBDQlKFfEBvU0K4HPEZA8mCGuwUew+bcYIYQAKiUkQxdlLagpYB2vDr6KRRFDvMwvO0NA8zqislPbq5V96XQipsMXmXYUQOsCo7iweoDCBx0fBh6DoHJH1pPGn4wmKJ8XzD54eo\\\\\\\/7VBhHCTFPhgfSNvRH2MP2SWthIKV2Wk2DoIR6GCc2DlC9pgf8vyikzbANJIqvZ08zvnWPAsrlONKTcWQrshQ9GVRXfQOV6tzAPo\\\\\\\/AiQhgTRZUtE\\\\\\\/IIBnsFilTu+3dtjUzSq24ULal35AaWh79qeB7+GXfveXzfJdaeg5Z993P\\\\\\\/PZcql8IhDLNZPLnR0KzLtfoD3WrahA9GXDE+7Ndd8d4Ar0YEqtvOwMk96TRoFMyO9DyUcaRG+4uPsM7H121s8HISnf1trrQoK1Z1\\\\\\\/EKBW046eBJeLz7+VQ3V69qL2GIvWyz3ftSAgJT84GaQgQsCn7VTGFo6mbWyJpnNN\\\\\\\/uEQsDDjSlNIWqWG5nYDL9L559WPcwL7K8wK8mFazdr+Vxs6OyuhNvnwqqnDfzau1O3x2UEE2qslBqsn7gCmC8BJJyRKZ1Me\\\\\\\/UF5UazxsFGfkSMvoDbMIRiB5Dddiq9QRupRnzSgawxD8T\\\\\\\/W39FljD99ufcYqVkl9jberMWk0sqLm5bet6K\\\\\\\/9yfe+op1Hs3PhuhHH5Ttm\\\\\\\/DT5Kb3vKcu0WsJSgQc\\\\\\\/sEuUtcQZMtDv18i3nXrTtv60brb\\\\\\\/KTZsIF1+BsvHBdBkwnpDxF19RlD2WwVIGYBZF5l7qrsjK1ZWrH1Nd8MmBWeChcBkzBEQyB2zEDmRm0i7xRWclmE1JERTzVOXuiFWqKTGyobMog6ByDyux+mt13Q0Mjl\\\\\\\/uU0cxaYDkdlunY6CZPTnDj+GCUYCw8SZzswnUMf6yHP6kQC9A7zim2i9NJgg4azw9djop7hdXnuG9RMcMM2jyEphKZ3jE9lo970RL6hNI9MG3FbthtH+OT71yk00rDHgjCQ6gRjt8pXYiZRYvbFlBY9kjO6e44VRye88STL\\\\\\\/3Ic8LBOSHRWp7fLt2FXW3ZZDtrIjkQ1BePmohMR2DfM1\\\\\\\/lUsP2LrfjSQEg6J9qgtPejHqOq0NutiQld9Rl7cNBdHmaELfGoJEyB+7laBj47XHVbo1lXgpiZEt+C5v0ppTvzLtrREfFHBGPO9xJjzlvnGdho3UjH2wDcx7D2WdcYwVwbHiyVOfUOQKRpa6LhVQ94jYUJsw8Hn5myZZ4vg9XrhXaepNVtQC20oMvs\\\\\\\/6D0XxFzl6YHqTBkY3WbVWgqSS7o5y8AwYhN2T8AauHY9VVVMHqLVSKCqrQZUijNLl2XjXpd8rMJt+leRAfDOg7uYMLblLwM8vVjnBVtxbqM8cPtZlanX3e5kB6JOq2vikKgbOv\\\\\\\/rQoeCvGOOzbPeeQDJ5l0zYkQZCvcYbi+fT1Ry3plARVttwlhpwoYqw8XYe8yRMi900FIAoBuWcQ\\\\\\\/FPsg\\\\\\\/oAM8Vmf9Ob57rzoXlyBDOLzTyADJDdOj48XEAUR\\\\\\\/ucQODA2Kq8pcnOX4LsUx+6WYNd832Ocw2Swir60yopdapitqpfu1wE6mLPD7q5PRCniRXZfE+cYKZJwSkxD4OLzhvQtUVh8Pfaho6otUhlAG7q+DBuTFlyPpkgAX8j5G4juqR2+rVxkExB6jKX6JjUD0XXkxDSVRtWGxJy3\\\\\\\/f8S69IDfMaJJI8u5ZfHVYYniZt5dbXCv\\\\\\\/cepflj2COwbsjlnE13e7ZiaKpo9ghrkImcCn6hxtkQLfUOK3hUMQVYiWpd20SUUjZ\\\\\\\/g7TUMqqgdC0jIocSc4ZTz0AAvxLk1AvY6G2PR6HFzR\\\\\\\/WgmKItb20L5ahCsIpaBqTHLF0jy7Go3IuN1FxDrgh1d+9pZg3W\\\\\\\/xmgmy\\\\\\\/lH1hsYZQ+nm7bA95PhaCPkcnDKps5GNzhPRoxLTN\\\\\\\/RASAYpTGZhz9FiUYVC2odRnqo85TI46D+KOQO\\\\\\\/Tme5MefiTlM5\\\\\\\/W9n\\\\\\\/lg83iIxL2ZQHexGbx\\\\\\\/UQHL\\\\\\\/HIztkWccRbO2pzmY7JNFUfpgjJHsXbyAh6ZDiGTZLQpCDz8kKyJYwhMkrDNne8erQfUmBG5e3XfTbz\\\\\\\/kypq4cPT6aQJUM0lefZZFfpdz2ZttO8IpAIz3AYQYlb7bzJNW4ifZYa82DoBZmObANdcpz5+6Y9F+TFeeY8tn3sx+WvHjNVrQTucJeM\\\\\\\/RNbqfgJqsPnvem6qSPv+A3h7ijGAwNrVNv982Z7+LbEhKVIVRAikpd0PHkYqPcmZYn4Pty0bHxoy9SwLIimSFXVVcUZ3ARdLqTLWHvIwYxZFJ5iEGwcGEGQpGyDS7Mcz2vYaKBsTWfKsuJveYaHbJNTxHzQkKj73j8EZr01cPrCjfjHicyFeiZG86Gj5ivWWE20ztYIs7NJE8FWI5K0SGYnJuRdlFmBuSvMVfF5+jaN41nWen8oo9h6fUVZ+ZLmTMbIXGZlP0gzMBILCGsl3LlBH\\\\\\\/HMWNmALpG\\\\\\\/k2vjawHhBkHbCQHUTnGJGOCPf8Ums7Qk7iDRa5QbC+kVcx2+3FYpgiJSYW5BDoJzkBOOvUAHksna+sH0GqPbnW2fItKHbKkD2KMInoPf0ssCinYDa+mBHDbf\\\\\\\/toLWS\\\\\\\/wht0MYlG3BtzGnHNqPuWb5dhNkTSWc53TlZ3Ih0r6VoBJFPYoUvoalOFkW8gTe9FeU3VxXwr1viXcmJmiEUPG50b8DoMIWjssMPi8dEoohQekRAYzP2RRzk7k2LQjfLZVCy4Euxm+E1aKxN4MTMK52eGvh0PbrP5DAmKRGD8G2eW\\\\\\\/EV5d9x289IglrXDLCi89IDegf6jN6uermQboit1AFhf9xRyS6mqqDsy8NJR5SXUYhiG3j+2yN\\\\\\\/wZ6jXv1FQXQi6ZQL3EQUnHE62\\\\\\\/qxm4ZhQShUJQFkTWBGY\\\\\\\/GJkLQjwvCafpTT8y\\\\\\\/woOsvNvPoZiawqKueVAzNRyRDejKuQlddtcWjv1s64PiWs+WFKcEnH0ITaGhwWYfTdaODG\\\\\\\/eAyNMmWS2bnM4At5Cxx8z8onSexCT6R4SD7bJDv6N0+SJy5DoLi1eHj6bgBDyN4nNbEsIs4vbgOFLomQgkNm2O6wg8dYijqUc61gBQ1qudzQCQFR9v9wMrc+pCpIPk+u9vbqkkVvLq76LR7vemG1ssmOXj+x8i0PMld8L+Gnu9j+75ldP\\\\\\\/qi470jTdU3TGjCWJjZuEaWLymB3zvmQGikID1KiGUYn\\\\\\\/XFgBZ46oxp5NqaFNbYe6ZaIRfcXAEmQ4H8H5Oo+Dvr93CsXWD73G8g4rvMYAXDsMtdyPmGESZ+Ct1m3FieK7nBJgG1zYEoLn3DBDA7W7zhDZSTLu4dgfYuZGO\\\\\\\/G6lsbukXJ7ayS5sgY\\\\\\\/YkuoCxW0Qn+OCXzX0REDhDsAfZV+BUqp\\\\\\\/USuEdJxuu5R1rAp4632CI+IMFRVEmyG29Jhdd\\\\\\\/V5oOSzk8qH4qeLqILkBmpqaCjA5bL3RjcyQnyNJMm\\\\\\\/Xfj9zQRpQKBkAt5tYVTf1JiWkNZL9Lqnf95p9mBNyD7PT4veOQfREJ8n50ukqaZDxdlZ7EtllqpN2Q0vHB90xJQFkDPl1ksIw0DUr7TFc3k9hnixw8tt0SgE78IqH6aegSgUlHkP2GkVMnjlMUS9\\\\\\\/4hx0OJidU4+4s1sv7Uw5+9OqLQn0Y5Yo3\\\\\\\/6mcWZ\\\\\\\/PBu66m\\\\\\\/DKUoVEF6GlXEpDkTZX\\\\\\\/Ja87mpZEvWBMgjg1WaL7MGBSbLAzAT7tM\\\\\\\/hp+YE8CPrVzry8a3Rl2bYTaWaPzlR7yGbRL1gx+cmo\\\\\\\/g7OiNPWG5EkGwS5JrtAozDfXlo0PjHWT0POywApQaOBSAoEst1OWqFAV0OpqTfZmKD3ZYyj1B+0JfDHF\\\\\\\/OZjcjo859of2xmuj0clYeI9tMPsKwLz5g1PM\\\\\\\/NuMLsLaiKHM2\\\\\\\/RN4l7YhnvpcG8HZPXfudIERguVYNeSRPEcPTiiiwoB+JEWlcK1+tqsSWO68+unW51xUp5cAcBNPVq4CNlQ+IhgJniX7WC9nF4e6DR8omREikPK8dvQFkpnezIv3ObwkfmNXquVxt++NfrVq7oXU3uPM\\\\\\\/OT5rmuUP47i05ZdZ\\\\\\\/quKdgMNNSi6TnSg2P6TvFSysS4AJCQSGEPvhDrFiE\\\\\\\/JmIwJ7r8YHZCFH3u6\\\\\\\/AJeDfU\\\\\\\/VRZHsioMbPghEvvUHTl6t0Hmr\\\\\\\/FdHE0kyI56bUtjVb76s6344SjK+GR0vQbjDOXZfp\\\\\\\/RQLcKxKa8vshblcJ+yZjuxl7fXvcdUCQO66pkKmUKVWNyZ7\\\\\\\/H2VguoFgVnh+z\\\\\\\/7ZEISX7wgY2c7UzuhqJHiSm0Iw+0qWphOYkRmOfQLvWnldRLbh5dy7QepvCA9\\\\\\\/cwkD44mP1Rk9pGB+G6x1a5+1WfBMi\\\\\\\/3uXgY4BwzYbm6RnOQ2CEF7h1r5TMAKzfWWtxWthiXD9DPLkxw2PRK0FTt7jNGerSn1HufhTE+pmcGKGOidohXMDhztLBIzNNXXnsPeI\\\\\\\/B4Lq+wJF\\\\\\\/1\\\\\\\/ETtghD40dZtvOwsLnyFkL3KPlZ6trJbrsH1dVjd4uUXKv97frJSff5gNUE9JqP5G8cOWHKzQMBZ8XMjaa6UOApxsaqIfzjcWcMIXdXJpD7OChAgwXlUd\\\\\\\/yVwtZuUu8pEr4KG5R3d3W25N8IlRFnivswHo84P7+whl65amPG3jqJdZ2xR0oDfaCixrhXXrmNd790ZeEtNmslhYsirt37GeAyCIKnaBxboGE922YOjL6jDBAzCAR7\\\\\\\/SE0rw3YSUWRECGHZ6B1VJhgOKSJbFN9d3MlOuBV7gkEBYUvLdk7k4XtnyLwRM48WOAaK8NnNFWcdZtYlXp5tKQC9EDrXq9AvSgQ7+stfpS8JGDLp+TV1JsS8UEyUUNTXBfV+fZNAb00RdxDiHV949b0lyKCo79Tz80izCBdCMuPojOyYvTbwZ2GvQhei+tYs04Lke+r7t+OV151f\\\\\\\/zFo6Ljpo0OSt5bR5+HKjZFyVYy4w26W+3peHySzn2aj8UJkV4L1cfaidhcNrVGmH7KtQOFrcJOPHNadVMj1cW+JmKb2pFBnYc1lCMLJaZoMgGPJGfD8OZNSG3vkJOdS65eCeQFz03J5vGh7kCMsHxnQfrPPBMHy4VKpU3Te8Cu3EJZtDdy6hUKY7krQwctyWkzhQlZ692eD9BgtHuo1bTaX0npBxIUtQtRNiPiHrYpo6QkOOEmHOAwe3+I81+12xPNh4qKpcGEfOgNtUQ4bA\\\\\\\/wvevDvCpvDdRy0lGntG\\\\\\\/Dh0+TR+YIBgNnNUvHCONZ91uUXooViTy4VstqD8jonn3dJJjfcz+k2PgLkhh9abbYsnpB7Kj+hYWsrNQGTYaNU4JWaf1RIJh7BWp0uYoAYHxgxOH96pa1cF\\\\\\\/6NzdOsO2+IsZVcNB2hn5OM2W41kXwBG4xdJeOWgeZNLc34wSpMDVFab9r4gw9k2Ew1Nu1bBb5zqSP3N3zI7N03Q2xeqUUDLU9zU+m39RMJ\\\\\\\/vDkrNW7yKIv5cIuibMtBZrLBtzWzi6ARyll0kCW+Az9qfOUrMye2WhyubxS8Sg1UVRRGMeEViUZ7i0LWMgnK5P\\\\\\\/JKB9KvK2D7iJCvdB7WxNBoE8y32TcT4vyfAp\\\\\\\/xD7\\\\\\\/lqXWox4xMpt+iOuyZFr6Mi63+oCldAyzoP6oAqj4QXqaCnXakgRdr1TZunR\\\\\\\/rK7PXnKDyNW8mMC5UkQ0jW5qcO+KsvedzH3Cr7xNX+jVFwMBZeIWrEHxF\\\\\\\/QX4AyNV\\\\\\\/upGEP+L\\\\\\\/y4\\\\\\\/HsnuTFq\\\\\\\/Iobr31547pnuCq4zguoaGJcvhv1Pt5Wjl4VISScL+fIEB9MznJsfo2Vt5KnxW7L6AuIN8mJwl6WFf\\\\\\\/3mhjlfH+o9wg+VpoqSsBSrr9dYpBYGiYArYtjdSyasZJ+AM1QPq67YE9CywtHNYv3+gQL6A2BC6KF3WUE87tpeGng0gjuh\\\\\\\/fKqVPxS5JPstDO5yiNyEJ5QckqvPY8lh0F054hGv3DF2Fw5STxYpePRpTEHQBx2XXwyjs\\\\\\\/exd0Yy9pK8HUW4PEICQXtyVPEFxocbPlpXq4a93KxN5dFrEQ2k6LSDOAKt1im\\\\\\\/owOmoFq5zKE7TzAqpgAfNTKi2ECol50imdyqSLmhhrF1Y+0VXrAbZJj17XCOjct59K0xqEJqznFouuTgsn0STGG2TJQlwDgocd5iFDLvnOdRDX3Z3jcDqpyNpDkyFtqcYEPhpMHMbIc\\\\\\\/wWVP3imCnIyFKwvf6ZNW8ZT+Jv2G5Wtx8sN\\\\\\\/slB0EtDMlO624nBNDJeTYKScm6RYz9KvhoF9Of7vPowgLsqStavN71gYM\\\\\\\/ytur1ffNnQVdv++7a9\\\\\\\/fGyciIfMvuImXpUdXWSJBKC45pI008jpOa+DhIAWAp6CPyvoBfZormCNbDcT0lUypHKvGW2gIQAkR1fIEMH7XelSqeoN23WYALC+NcNEhCLt3Oolj6U0QxF6p3WGINl2SN5dtjItMZZUVu5QtP1B3TzbvK0rnF5ce5mC16q9ZftQI60dtZh4oWI0lSQEXdxBAnPuFK46u062tnCYc2T9izxt2OFGRXU1+KmW6DGmFHV9+P12Ryu6imYNKm6u57vQVlz9+1CWZOP6m0iy+R2cOpeZpXDGO22RM++woWLLg9cdn+ayLbIbdAWPrHFKqUumAcGZrdgRHOhO18bwFkc\\\\\\\/gK9L5c+2hROXhtK9NSM7ak46DESrSxr0iW5oFKQWXn\\\\\\\/D97BYQvS9NEwWCOO+VXocYR3m9O1MtQPDoIBkhEsH9CXuxH8jP3BYwMWZKdDLTfnlwC2hpPaToLD5Ms0u6llsMofSe9UoEnB3v8IYNWzGjNItVNhT8\\\\\\\/BBv7qp64NzoqmmTpx9xgX5GOzRY6Y+r77Z4LLrJpMbxgF7QNQ8JTAVA4yd347P1BLZECaVOOtPHBwojp8UOzZl36rtrazBZeef79X01+T\\\\\\\/MQXYbdR8df1yTyypYJq3\\\\\\\/KtAnyPnzVq5qfb6L4KS4uf\\\\\\\/HYYemPdvreaUfaVEVoT0kQ3aAFiZjWCkXWBluKZ9Rupv2MRLSUQ90MVaTCUMoOf4hSQxzdk9iXU91mXtYUYe7iiDShsdWjKDK\\\\\\\/H9ncxgPWyd4\\\\\\\/EOKHMSmAKN7u95xjSE1a4GEZPVa8SYLYJtWLhNiINMwypTmTSo4pCwqNGMVfIVlS68w2ApOqz\\\\\\\/KhgVp2EmguTrB2YKPla8lhy1vSs2En2lH1L4UnRAwwcI6xxNZaHqViwQkp+BqJRzmuHvO7LcNpvGS7M1V8HSGDC7LSgJpl\\\\\\\/zjqa8+QUAjAE5aOAv1mn66Sx7o32ZXofBCaC7pOQV67Gkkg7XS9JmeLKooqtKz2vDY6wPAAcqWKg0hrC8r9riVuXzcZwMe33QnGK++nlhsJTGONyBZcyd1wbK6uA1PPdFK3RNwNLO+Phe0uKZunKbQbf2VBukA+kYb6nI8fUBo1X4kblJmlgRSwBfU1wS9Y97tQqfF5txqfKbyhDpDg0kTiM\\\\\\\/E2xP9JOfXPMr2MMOtsrulNQDTc7pCxn4D4\\\\\\\/NK0ysfiW7k2kv25C\\\\\\\/kjJHIFUAa1VL5Nx1uGobfrvtV7OdlhjjR6lh2YNnyWqfR+M4cRJuM133StmgnlDL2Q\\\\\\\/mP3K5GdF7I6lnYnXeZ1\\\\\\\/K6IMxn\\\\\\\/+nsEBZKbscfTK7VE3bbiKOD6BS1paieAIIPuKU5NuiwWKo+jo\\\\\\\/wp5dyUXnuP5Yam3DZya4uHbmIJGfoLz2r19xqt3ENi3ujQbdDNBt4f+b4iX\\\\\\\/79DSciiAtKSCOUV+zzXOlD9q7vjfEn18urNvnL9JMaz4z9Qz+QqzXSYsbBtVNjaxR3RcnLbU\\\\\\\/+uV3b6LKggMb0ZK8G7K+4OSR7EH9iQGvOCf5TGfVYh6CAaPaJbohK7lSiUlOi5ZrrJ034Cf175YHZXECQi77oqE+JM3GfPBzwJlhAsnVbbYxW2h7gUg+Aj74l\\\\\\\/7NXjxQgrPycYF3B5EtIXYtmGxCXTV1hSFfLrbvyTvD28EgyQIsV07obcYhjSjHvvzEg6s+8E1D27rK4i8MknmSDnIwhWj6ZQHJJlHZHRHIOh59zAnUJf8e2yXaeFV9pSdJVEFQUL\\\\\\\/alq8oXCC4VY5fPSb1749\\\\\\\/n3sPlMh7bvedQgOEtxCVxWuXilrROOAW+\\\\\\\/VQZlS7k+wf3svmV9O84wzb47waOQte4ktVrNgFktBtsz1Yb07DI7UoWUuzaOLftM0DW+asJJ5k4YC2xcdQ45XNpUoAdVlAYA9Z8TikO\\\\\\\/W9q2Y9BB2K3rKhA63z9obSOd66T+zacTXqq+tZRz5nOMTIh5UKX\\\\\\\/ZKJlh7GNjjbqAnezlWJfCLqUShNCku+cOKDhuS6JDL+c0aWk1LBHVEr8IBTNV5pvM\\\\\\\/QxtOVsHc6928J6pP1aW5T7eqeAOofw8tW0Z\\\\\\\/0tWCFJkKiPpv0vFole6hkI\\\\\\\/6XbLpHDp0U\\\\\\\/2cZhr4v74KNMxwPTeol\\\\\\\/EHmtAJF7SvZ\\\\\\\/8T9JL1P155+5Gx43kapY1CmZOT8DYuEh2OU3li+v\\\\\\\/VEbIsJ88h\\\\\\\/fdwlr\\\\\\\/fwSe\\\\\\\/X1NNig4cj28E13aLJqkB7FcPoJkG\\\\\\\/JNCL7nihaXX2uK7IeHuP5y1njKeKDi6sUkTDnmfBoMfoo9sNYSyJ7mNUe4iBmPcqJH8C3TBFErTnjFKpbzthfu25s5eJsMOpF6urhTQFTDZPnED29Sqn5OmxXxThNoMsoTz9v5vgNuxMzcBc3N\\\\\\\/c+oNKaJEjT6xmGYu2rEoxvAlOIoyP6VojGjZHjXk9KiZh0g+\\\\\\\/DHmJ4zQmmfap6oT6qoku1LnYVE7LQJPdDt1aqSWFXilFli6lvw0pHi5\\\\\\\/FiG+BNxrfpjWzEIr2aG6hmnAuRHkLp3VK+sswFOgguVcIPCDFuna13qLPA4Yc86kAkPQv669bfI2O8bylDwnI879l8rersVCZy7VMafPTZ9TK95uyXpoA09w7EgVr1y+MyckcS1n+91VdXvPVTaGsfBjzAtGpiYZkjs8EXQ1zvgN0WWf1rlXjfod14kEyJ5QtmHuaXwb9hl5XFpdL1z\\\\\\\/t4a4bcX\\\\\\\/UP3VVHB85u5fiwaiChr0XFdjCmEJH3ZBx7yU3HLP25tlOVDNmzGeNDMZbS40z1asv98UGWLFWk\\\\\\\/wApl7aBaoZtMPo\\\\\\\/94OyymuKkMck1s3wFw0VujPa4HZWZHZkImszHTWB9LPHl7xPHyFiblHlOGemOHNASXQmKPhV+XYIgJ\\\\\\\/1x23J20TjT3Cd4xO8ApVpC0t\\\\\\\/6tC83tT7ciTGdPoG4AsK3GP9mrxeUXg6YK6SOvtaZrR\\\\\\\/9AUZ7kGQ06qDbVxkHdgzoEYeNqot+bYg1cWiWuo3FfdycEd51KeUtsHtNIVQ5kY\\\\\\\/tUYjY23niJv7Pl46gMgBCqRytYjraeVs6kLHM4fT0P1ln0N5aFUlOPqAk0bZhDCGr6J7QdXyUl5tYzRSBVifD3f\\\\\\\/rtNwTRF3M88Sv0gFnhtKe0vhPzNbXWFrNZnH8A7Q2I0JHdjXctcDeJMK9GlEslLvo16YX+crwdp9qU0sC7GHBDBW3z8a2VLKx80W\\\\\\\/k3OoPpnyt8MmrMiYBvGMeGW\\\\\\\/WDFngCbI3TBbPUpFqqJEoLxKQIUamB1pigU1fSFZbQdQIn6FTgHWRTTz5u7lonkTJNl+3eZ9ZYZ\\\\\\\/KrWS+EwvD9jJNBFxSh9XNG+TW1UDTC5QgbCmaG6fOJBb2FpIkUodQyZVAUh40PMgustjAMbahJ48Zyt1qW1ibW3gvEL\\\\\\\/JP3NyzP06M8zX0f947lROJXj\\\\\\\/6cqm5xGM+FfZgszt9+pitlic1PD2Y5Sr\\\\\\\/0psg1WSVK\\\\\\\/Wa9WphTTOol\\\\\\\/Z7VwtVk2GYAfXoRuMeXw9OLeLa3R+GFWvv\\\\\\\/DInTdXZZNDL0SqCb0PjUr+IHlw3ERwVPJy4xeNZL1Agn9YsgzMCY2QFPywE2gc7BiRho0EErTfonzThqk3laT9kx+fej2B5Qsf1DY5Cw8IDJgzITsGeGpir\\\\\\\/qprvUQ\\\\\\\/3jZrIUtSn\\\\\\\/uCht0OxYWSA57NkZsUFh5EOOCQ+B5fLUFDrQ\\\\\\\/urX6mWH6xwuq7X\\\\\\\/p8NM9uT+RsO505eoyLiap3BfVOgwptLXKg5r5C9h9kL+ngRNK2cPWueoqd09hSoaLRpNv37hiOFMytEH0LaWPDRvLPlbm4CxPF2ArjbiyNRuQDtHRBgGh86EC20pgQRZp37FJnoJN5i9uwgexvmSRW20ID79aGdy7M1KMqHu8Cxe4d\\\\\\\/WcIuHH7LDabFINx4nQqFChetLmRLJZ1bfjRbRyK2S01gsZ4oictXvIGfkOllssAF1MmZsRtFjH+oNX9tAMrZnxqGpVt90REiO8Epnlcscwvk\\\\\\\/vNzt1SOKcf9OS\\\\\\\/geQf+MGKVTe3CXwErNYtnAzhkY\\\\\\\/jqMu4RM0OxinSWkWomWkh9kgCZcbrs5BJqWRh5oxSqBnvm8wjdqHN8jMxZDgSsVweSAnAKcQpOo16G0nVzQ7SghZqpVG1EClemaf8NXbb\\\\\\\/GQG8xKiIwUQXixlwo3YcNOM5x9anMrv06oEP90EUvXNZ6FHWzz9V+pwbsr3tG13QLVgNTeIGxpMSxsgdS+BNViZ8CVhQsr6ck3i\\\\\\\/ZlUNjbJUUGuvPq6y0xhBmzVW9VkiM7r1SNKDMe8IT11RSa0wCPEXV5sk8yGqlWaWCJuf2LZUa4CghphV17Pp\\\\\\\/K43YVGPXM67zOrLgi96hrl0oLMOMg9E+HTUjxHEw9VVZtAPa8A14YmBOjVx6SxtsdL0UPxXpZyh2yfKtStj\\\\\\\/YD6C8lV5mUQ4Wu9kOLJMYzReZ\\\\\\\/l\\\\\\\/HSE8qplV2h2Cy+uV5nOaRW1ZLrYwKSr0oFs0B+fHn9CBCR+NoYc6h13fs8GXBs4NkZ0mvbblSqpKyHr8CfVisrmN8kqZejq4qyP4YE1QWoMVJnnwjooribvSnICow7Y5ArZDglYFnzFvBF1lNsHpEWP1kU8QGRIEfiwqFaKpxYU+igts2iKaLCAsqYm4Z0Qap7ysQmbptqlc8h6kNkVCuDd2706jSbx1ECTaWnImFAe73XW8QhuxwvL61AOGeZStthaIIdgeAv2vl6MYxuKV94lfSi56tAeiBi9i80PUf9cjVnUXxx+Vu7n8G8rrOhSDar3aM1\\\\\\\/IKBfiIKCrxi4o\\\\\\\/Qrc4KCBFyjPX5tR45\\\\\\\/GnNShRorVzq5t2yQSctqJmOAW\\\\\\\/hSI12mnxOaFVmBDc9RfzILXf6iiTTRyItt2dTBmJplKq7fZkfqu9uDf5V4pUaqM1XqeVmAU5V+YLKBTRHe9v0MBxglKYe3iNkD12wApln8iimSf3YCXyYps3wAI0Wzrj3twVxgscn5tkY9u9Xawlm7YH0mDp7m4BZypW+KzttS+dIoNYMmTw0B7msVzpeFgQdD3D2vccn4nfbL17L+ipfzd0Iey4clG7baY4nVom\\\\\\\/tbyHGXpjkWOtM3rg4ahle9tmD6r3J7FK41wzlBzDbmMRRurBZdH9MYogw+6tRI5YbleNUA+OKmB\\\\\\\/fR6ls+1kV3Aqf+NHJV7mwGmS4u8WOQZu91PmEGF8L8LDwTND2c7FW5iIa8x0Pm0MFx6sFwle8KCP3GnsKaY\\\\\\\/Tsf1BAcFnxIzJFlxxerrD43F\\\\\\\/vVUoEUTKcMnirw1L1TYeUdMrEhzu0OjhjKfGeoFr7UO265SsJFJwy015AR6SbHCHbZ3CvkkcoAr2VCoxOz9QxJ627tLC63fGKrmrankCQyyz50aUaPftaQIAtZ0KJ9QPHCTOiLO\\\\\\\/C6BUqOaQQ2SzSafIMuuw4u1NSvwJvd+KnQPpDPUbCDz2iMUjBMeiyWW4jVhVDXVu31dLxcOlYCItBLGcpKAYEYDvgvQ9y8s1KCBrIx15p4poMVRNLnGdJLe8QH79ju224QVX7XoSUbdO4UZAlnMPoQz8FHz7Bljg+FQWKtAP4pfl8U37mIEnVEipyhjuQeIJ7NuXYw0pUeG9QV\\\\\\\/AtleA6VBD+TudxWAF2b74b6lah8\\\\\\\/0OXfB6Ii+ibPWHAoku0OS5OKkOnxHVyEnFpj23WAvrAc8VgmgwywP\\\\\\\/DePrSGNDpL3UWNGvB+K6ioQ+\\\\\\\/HG6Kw5maMDl4eyAlICngWc2x2Sm3oGHPFZpks6vUcUTyQfG82dqxYRhIKaVp9NkQpxg0vBidZQK1DQwjP3kQZrW9YZDcFusIp6w5dEGXNqzoIKnvJ6cSY02WnDTxNqmJvdB3j85Sl39jjs+CPs2cWpB8kVEGxHEJ+Js1fI7x7oIcqe4gkNU12y7qIIyjxnRducZ3R4yHbxi9vpH1Jfo0SEdaSK4A+jm5Of0q8DD8J2hdhPGz21+OmzXFWOQSHzytCXGKHHMazXpjqQpIls+sJFgDSq+B85EwtRTeHcKboYw19okPFJKQAxbFjgoYoPDeH96Q+5bc82Kj2grfGXQqVjEYGRk\\\\\\\/x1p0AfhLhT94Pb32mxZZkz0CwZe6q\\\\\\\/8VmlsniR6HQRRARYKUs9xOWz61viJkFkoIENBhpA1di+ECQMdxDPPBV\\\\\\\/KLBzyPqc7dqrSr89mPtVRxSskjNFjI+nXtylPgWbfaoOJj+IaBSYIbkzMsR0D\\\\\\\/ss\\\\\\\/ADbU\\\\\\\/QXG+qfouqzpZnyzrWqg5dxx4OUJOn\\\\\\\/jk+DExPobuyLRqK8HKPF4x96wMOnT9KCIk51FJcW29rD7MMHDqRsvvlGXiurwsdqPFjtTHHl0dBldvrBzLd7WacyO2Bo\\\\\\\/rB3S3CUkleYhRsmhiAhevN2lINRMo8M0oyOmoBRJfk5hGu7dHFfgCD+uSfa8Fhed4uuFl04t7bBQfkUTGl4X+Bw2XovgeHP12hZV0kysLzSoiCzLCKI6pOQ4AvZuu6XQ83FA37VxqC63mgLtrgRaPsXCGycSNXIFBTpTRDpYOkBekaZRHqfuXKNaiahF0yWLRTpliqqTh0xMVNc32e8BET56NzYPGdje9cLTTGzSwG4GMq6QuRlNCSmew4f+YHPgfeelFykgEufPS6L3LZrDSrbwUqWL0lH21EXXtKwb56fNG4XgMo5HLOpdT7kwL3+7X1SDGdNc0XWKl9nyDPRm+46i6fYHfyc29cHbKGcAbARIVj7Nb4KTriEvDY3TZRZi\\\\\\\/i\\\\\\\/BGhl1S7cT8\\\\\\\/sfDIn4dv8j1J5yzUXfX1YMmP\\\\\\\/5\\\\\\\/H3raTmfEEii\\\\\\\/CSsLqKLdVrVL4yUICgPjmvI1SkUhFd538eGo2SNeZQaBU7hV2RS3vaykjhVoKBA\\\\\\\/0fqVnT9PPREIwhdQcX3LtkamRM0teZE1D9aCfGmNV8\\\\\\\/ELVyFPbMurnowIJ9uG9ncYevwoxWb3NqpxfWh5iiqooZZwV2TOooE+gfxddvrS0j68Sl0ZLweySbJnc8phNeaf2fslArVjgoPXNC2a773EF7z16miJzBhbvHapNAxjRIQe9GyB1SpugEvPdB90GS++eCWvCZggIQYXk9bHgcdrACiAKCZGfTMRw\\\\\\\/YbrwC04RHxeCvMHhK7OaCBTqFddSYQBtRZaA4gVLZSlS3XS+gFlayFzVAXiqjZNhbsvAZciRgHctQpVSXByzRvuxbXYRBTYrJJN7AV+Dnda1NJdoMinSauRseKjRA0LsTPfuQGRE4kCFpYslK0iN\\\\\\\/bWwLH9cRfq4xUNQsT8CZvcR6hsLkA4O+VHAZDstboAcLXRGWhCVd88uajio5L8eqkG0Qi3YHLI6dE0Ay9i998Ccv7sk4uznEzJi7VsZL4M2hS2TsW66lptgKD4TbWmHig6ICDmg8Fi93MD4XfrJ2eTLgIZXi5GtR6Z3FLb58dlRnWkMKWxTDkugB6XMrLp5pfXGoFa3rqgUCNknQWgkM8sRMXR77AX0lUNJUKJBejDQxQVE8j4CB3od54KnAHAkCiAMcwtNTHzppr3lAUafbWX+0jY6RTOg4l\\\\\\\/xyLoqBwD0+DKzKCV70cNmj3jPJRZC5UOs889x4pkrCI4ur5bCQzh61OFQuSeaCaVitQwuMoX4aN4fYeOY3+ZpMdXX+fOiCF6dEtUq4SPybwQv2zQmzqimIDvLzIGJrhsJAEBylmdQRqKN7CSVjF8yUYQ0dVtUCL5eWHbrKwFEHisPf0aWDOD3JARoaLIyd5VHOgcnyQTqwWY+w61fibkFVrP32E2TX0Y0O4dP\\\\\\\/s8+HMQRqCUUfUYjeQKUgH5tDf4l2q13aIYNWiRpY8lfderAF3GzUF0ku4ad7Fi3Hh1Jm\\\\\\\/t1qs05vlutka21MD0N0pQ\\\\\\\/sTJHkBBAfjyYwyCHWGpqbMKagi6Rga1I5Ic5zn0odZJd5snlDKM6kGsAp5ufZHnsnrhGEtG8v9wpoUowomTueh4MxaMkC5kb9D3AUjiER46mcDyshUfKrO1WQJg6NQBkyMFzYYlX4e9KLJVH6tpEyGyDwhgJAXl\\\\\\\/ajciNOOichbURURVgk6DhpGOmJLJ5t5dAsScmCXhysu7Flou3HdeMgvTfOdUPhrwPIjBTPeHsE4Ynpqw7znQizcD87M5xjLs3xeoKS0az1\\\\\\\/Tu47rXYAG7PFlr2DlXHqi0FhpZi77+bCbuSgpPZlD6rRM0Tb0sLru2fmsyAfLCCMKqSrCYePA0IndKX1ipI9UjW7FEyUlVpL\\\\\\\/fo2V1kcp4DstFU7SxmKx3RZYOxizy3mltC9PCSIab7N91fhJJgPb5QcTOxAjsVjR3P8nmzxRhR0ECd0Vsn5ujjjQ6Jx2FiB2bkpSgbzcEa6ct4LB1q2HwcRgEdLMeasKide6xVLAutKfoRVdwGsj0sR1wuE9HtpcK\\\\\\\/0Urj05z7RYS7Jojo3FqPkXMz\\\\\\\/vty94cBt7ilZyXaNWFA5+Egc3SZ+11jilXNf71\\\\\\\/Cyl9E+orfZ\\\\\\\/9gzuP\\\\\\\/y7ftCrV+H6KxR5kW+Dqn7oJKNK5mV9Hm7BVywj8b+gfCYHO+R48cYHfRAgoWMwB0L2rofvawVIoJDESLY4gpKFq\\\\\\\/GG63MLDcHlui1J+7udttT50O32wc6iBqVtcLM6SBvfwl1HdGlNgDo93Kyb1U+\\\\\\\/+rKcTLzfh\\\\\\\/9jAYZeg4nO0YlkyslrWOIKeomHdU2VoE68o4wec2leOCeiwa8uzXOKqxZpXow7Eb09fY9Y8Y9za\\\\\\\/ecGhchsrgf8yo9DyE2CWzQqeGdLGawhpdyZvskC7xv5wTf+WBFNKFJMtQ9zOFnnEYSh4P3Yp\\\\\\\/uMb5aCoxF5SxZP2kmSByuFzdVVNTZx4nWywjltt62wEwzPWRvwtbiopBtIsEIVE14TN2r\\\\\\\/jO9Bvk1iw9VWSIingn7YeGOLpbz4vHq+eE9ZZiIyCKLi+bUPLoB9WTA57rpTuWa1JSFUhnlvGP6qX5OD5MSSO+v3hu1Qrz9Lxi1FPYF7KEbhWRdNCJ9mAedyMXheiMLbv8d2zmzv5kJyQKop5IYLyN8BUPT+iiEFQcffiawtkFyIe8lcZc0UhBzpP8WEULtovvQC5j194ZMRUIax4AiWwTuIhujL4\\\\\\\/dbBbRGyEUzayu\\\\\\\/k6bYVZr10qeBxgkGuTD4Iiz6JfNMoiFEbCVD4ONMyH0GZcDzZvQtiyDW4GBIsLIPhaMH7SuBc5ewV+zB0qKlHhQJA8F9G1ri3b7jss4QAKONbhuTGFIm8zIxGdWKhTQ8RcWdo2F0YCsql2YFfDttcmGaRoil8g5lBdM3KjWOULTVY7vKX1HKoTb5bW1WoB5B17nQde\\\\\\\/i4qHDBeNfiCvESP87rJRjMoA3G8l5\\\\\\\/s\\\\\\\/QDmqTp+nGtTKMFa5K4940cpFPYESpVh0XZVwgPulaD0i5jjcAVjzSA+Gi\\\\\\\/HVuu0\\\\\\\/u\\\\\\\/2gYdRm3dEg0o07+u5sDmmCHvIOinibCnWhiB9EHlXLw0zwXSBfA9wh6KvAX8LaioGYb5W3eNk1JaIDHkcfEgR\\\\\\\/4tBaVPmWKi0awJrq4o0fUNnt382FAraVJPSeOKmGBjWH+N0yBuBOsQ1hy60KUpfgalsxDwy7R96iU95C1oW4w0asynqqPhilO7HYpnQHC3ft26lC\\\\\\\/E88DZJt18JpBgIx8l4OM6OAZlEc26IlXM2GxcUaEeqFuLJxeHTBd1uZD2LeWgBM8x9AkOn3PS8mk1kzsCLyyWnSNFww5A66oZafrC\\\\\\\/dRO9YF9Ro8w1+BQkcTltyPWIXe8GukbqAZ1DgOuK2DbYLaFphApo81mfOcRL9QQJ6din0eF1oACSayWWTg6LtCYPQ4IsiF38AcrzML7lur4kGm9rJL+l1GNNMYRnAvVGgeo\\\\\\\/p0feR92Wquh7qCsOtVjggxsxkOB\\\\\\\/MIcdxkU4N1k3BwQ+4cNGOrpo0pcWQntoWC+5jLWjfDOn5HmEvymir5HWDSlcHEG94mDnwfK71FKxHi1p3nVLn44sKHJM4vYVaMXHpRwjdG+Gm+F33Vcg89EB4FB9I2RA5D5DC1poi\\\\\\\/wi1NkyznVnNYUl7desE10bswmIK9e4pGSSh5yIqSKrcV\\\\\\\/p8u2z437RN4b+v39NaLTD90qQv5BjgyyojTd0pi7LuYk6+2M+YTotT6GW0t3m8YBex82B06L\\\\\\\/prIStiUwBfJoPPRqfKgW6CGlKt40ZVbWC6TkJLfaXUfSGJrUjCHdYdPEo4JCwDyHD47vMJor1oExlSBEmco1td1ZRs4p+wktjn+S8waj5mBfSMosqxZUNU+yYMQehKaWpFm+unrBRe9XVhAVA171c6GPcgdJAhWdqwmfs4ZCAkYyuowkkd7JRAvSsojoSw6hNVdo2V3hWCU51lDbhDBuvZDsL1VZBhvluNbzZnmU02qn+R8RuHbhi7rUAsDZ2gmAKsogLMaD\\\\\\\/oLcQPyGHIdDyRxvWkkzsRco7SHr8eO3B2ZDse6Ibdox3XBiGU2T6MsILldyNGhKgm6NqB3tlH2Eqd2neRNTzEXz+UOeZVv5mCFl3y65a\\\\\\\/BHPwU6Mkv2lNNcnERrEHMoHXUEGR7mHuDCxeKslOp2Y7Lp3ekhWGfdpaJWj4JWG\\\\\\\/8J4FWGvgrUqwRjrUesZgXP5K2f0oRnH8wqlJpV0QP8r0C1l+vfAlP4lgxYD\\\\\\\/esprqZXPgbg8OABhe0ycDExIGgD9563kJBbt2qQPcVkQ2YH2kPiKRRfkWhvX8jPNvM87GYtYmWKOSOLjV1YtOkZ0RLZNY00ATSEM\\\\\\\/z1iU1jJP30meQnI3bARv7y0IfBlChncErviqG81c0gxdiwrxiStbuUhXR3mDL3eb4klFY2a0vm1PK414oIZjtqxQOWcP3L2AkNbZlKX\\\\\\\/0VMJgiXXmkU8hSgGYC3Zd2ZRJ45w7FNyu7XfI9tcs9zAY3F4ISU86Y9agOyWbemruW1ZQJUx3VgmrUxVi9Cuo3TnrmvODkWhVJznBQ4ME53RJLrO7HlZwXk+HcRyq8AK7WIHkiIfE2thGfoNFK0ejcTvlQ5gg00lG+wflLYUmyDr5QELGmzxC6Kw\\\\\\\/MQp7wyK3d4DA2rSsvRRD+W31hPisyhTdsU7Xyql5iK\\\\\\\/4HD1BIrJ2LksNJQGF9pV7\\\\\\\/HLcMxKk\\\\\\\/hJouAvnHeoEllSN+ZBVjbJXGfNse8HhUm5fPAgKfCjFk\\\\\\\/68pDXfeKixzYeo+rTp\\\\\\\/qasJ6dvOReOAfqkHYpN74PubTedSGUVgBg3CCQOjyWNZu1leu94+WGWiruXlI2TxpqPVY0ID6rhu0e2Y76ufbshBMMYfjXi8PklbTSkHBlUZxbK8emHyQyynwenT94iRbkgnt5VlMTAhgKBI5rdvYRg8VekqEaFgCCT7TdhvIc6gWjkX+Rl1p9+y1bEuCDAAetoGxRlZzONAYwoOQ083Me1a24b32qccyNnUSWArHv5pw7VrMIQw12GMJFzj2l39QCn9HmN74KNmr\\\\\\\/wv4LjBHrwgMHuEr2fR+tzlatWFoNdr5AIN8tUxbOCERW8BAsqdu7db5JqBrCR7yVVFP4lMQIAAPpmo4+OS0Ant9dUrEUX6xuHvoGCQEcEroln\\\\\\\/86e91T6znuqYDYA5ilo0qtyDgammfzi2U3tE9GmI8cpyBvfDYEOb9JZQ0fWySwNfA2RwbTeRuLCBcSZFogLGST8d8usfH4cRpedGI5nrwVZFr0VBln7swAl2PtlgNOxl0MJM7OCpNUWFi8gw6SF66y46KOz6k66lIdcvYuNFdqy2QbKUvdd3KdlBYhMrv3dDr2hfU1HjL\\\\\\\/9EgoZ+RNTcG3o9h7TfCS52ORhk7KYlYBV9eTHVYC1N9qyTb3ICuJqBPo4rlb\\\\\\\/pJJ3uHHfxhfMA76IktU9NPbofgQ5+JWvvwITKe3aJeD27fSJrNRGQ3eP0jqn4bR4sfRypy6+3lQoFpik5q11Vvxtq3rLJkLnit9xOWotGO7Nerrhl7mYCu2GOe76j38a7yPjW8L8YQlFWZeZFhRMz3XNTqQB1vdAfDaA1v21pWZ7yXfT+cIShfp+ZChVfqfzlCdWdiMd83qoucOlyAoaLNhM0D29Ujl+WPwlYAsA1HnTEhZvJtvogFbhq2QYBUwEU2HuI2yXBQ1o4N1qZR1scRNuIEG1T3+Mr3vomDdcmwIWUqlNp7hhi2YM2SySeMDP5\\\\\\\/l16o\\\\\\\/71ksjcWL6dKYl+pK1a2YTZj56UjQ7fK1nUFPqY1HuwBQ09r43jLyr79ipr2OFuTFdoM2aNzJ6z1Vk7z2ynWXTee8GIT33OPcLPSa9wkIJBEnPBHDM43nDbWHpQTSwG4RPtMAG8THlh\\\\\\\/OjxJVYp26JEODhcONL7Iel+KL7aRx1zUcOqWvkRoAcIh8+VFoEBb7Kgv5\\\\\\\/aKSvVtyMOn0vSDCZIshLaPE\\\\\\\/94CU\\\\\\\/czh0Mw5EWG7rbwK3\\\\\\\/+Pw6fZpNIS359+m2vCWIFhGUg9FYN4AM6oLmMC6sHTianXTmUdCpgfYbFrGPetIUsDIPLyf8OEcQRuwqnRVW1bRMyCBf67WzxCNuFq5iPX+kQhnaLLTGnTvr2HPfydfGNZb9G7DVEC0\\\\\\\/yMWnCg9mZhq4d0kFSXXPhPomi24R22Kao2E9NW2hhE70NCgQgSCx1f0A9ib5EHLCeZ87YqYiy5V5UNjVXq9GXvgKP6U5sVIGzNTo8WfM1oQ8CPjvoJs+PcHWZUkBlE2qub+H9413vFKTcbdjStzwzJpv+zIijOrsNfzv5GJuYZ\\\\\\\/BXwW\\\\\\\/HEs+1S2x2C10thFq1G3Jb6llWFHt5iRDQP+wPtujr2969p69USjqsqMmw6LERhVq74NMSN+PDcHli0r3lXeNLveknhWmY+SIQU\\\\\\\/eakyB2XSsbdVkW068HWAPLGRlJH7S1DqrXp6gXycksdvEHCRjAKeMMQS1oU0mupsMIZDqNLyshtfwi\\\\\\\/FtkYEVhayfE6NGkEoadh4N9r8bMCd6clkZaMF4gDX21Chn1gMVwaPf9OrCClU3bMuUAHdrH2Ly5eo8+s+kae5+HHi\\\\\\\/dbRJNhVuniGIBYyEPCCrEV7t+FpQViDsC+FNMFFZh1bw6SOc0kJsVPlZdPH4KYfNvdMlg9v2u72Jt0KyTqka0T0N3pmv9lAFT499BzgahjY\\\\\\\/9qr9RjiJ\\\\\\\/ANbEqW5WSBXEA1C4LGNELjsrtXfcU7OdmGegmiBQhLD8mpx5JRKHYxIovYazTVxb3W1oe\\\\\\\/73Pvj8UM82ykSUHY0kDdJ259cgmtaeZnvymDWBkBeXGpMFDiyraZnGoiTAE4U8QawIHCmeEZnBCBCgkjNu03IF0ukIaptNGtrKYzqorS8IqS\\\\\\\/wMaqZ62nTupVuSLsFWm11D2ge\\\\\\\/4w\\\\\\\/33MDMk4vF8KSA9+GsvOV+L0HTfL2LuWlSyTJ9lsZanr+WYzwfOzW3vpp5Tk2aYc4KZC7IhJs98igx5pDpWVfIt5CT1saAX6kXB\\\\\\\/kSpzWiSnKePMo\\\\\\\/ACtY5zUcqoVu7xwJHtGFGy8AfpN+6F5+ycN0Le46MqW5OmI70cKXpXvtaiBuVUbOMzE0JONGC+35rrdr3t\\\\\\\/IDSeQfhqBUq2J7cZX6ne38q6qDHTRpRE2papiJclbVDLrVtrdze27ZEirUOhU28PegswOAYeCkxSJ1KEaqW74FHukeUJVzwgPMwISVSsSReJl92viT1BPZn6rrVQB8oPbNvFvzK\\\\\\\/\\\\\\\/RwQnIB\\\\\\\/lLLMh0urmimvKEG7fWIuWfin9DrrVvcDuEkNdBEH6LbPT+FG0pHvfA8egJvBPAuymCxgTgasN\\\\\\\/2haticiQhlMV6rRSl5daItZ6liN2p9g1QgQrEP+fIkyo6E0zJ7iiwdmryNL9+U\\\\\\\/NkDEOvdDnEKnOia4BG1MsP7rrOVRsoGleQ4DEOaR0VF3flvVDPjrr0VN6fQkx8MefnZ2L2NVHUBnKXNNa2rdeRomtkQ\\\\\\\/JgLyh+4Z\\\\\\\/Sdiq3OanS7kcaVWMVGvBcwjPKX64qbXrxd4f0w\\\\\\\/348p9NtB\\\\\\\/PxGzWdSitAJ0TMFeLndNEX2VHvDYDN94jCQyQA\\\\\\\/jhZgwhWOVK+cci7twCsKUXvq6pQSQnhf0pLGFduSfh2GJhZZpAn74DpVaX5HBuEHRipYct3X9BSIx56aRtt7jWmNN5edvHTB5lFr79YiYldd+HryjxOkmbgwun5LEKrJ1PaNcx6Ew6A2ND1mEk+q+iNP1YjhSRFnZr5qJAkaeu5pTqaB4O3TVpCc+7\\\\\\\/BEbc9HACoZPqD+XeiASDNPzkDUeMPzq3hasTmuPu6Mb3FuMFHczopghjd9Rgco669spcUrjZuWGESG7gdCPB2hk5HV8TSl9\\\\\\\/fM1rgWpSmzyUBY+FhQXlYFbM9L7Fice7mdBypMTFc1Q2p\\\\\\\/WtOV2Qrq\\\\\\\/8X5lW7honlNodDKDPJ78h\\\\\\\/u6+HUk2J7Im\\\\\\\/5N5oVJJZXZrA4z+Ri9bP2MPi8MH+N1sf4GYIUJfKDu99X4o7RtiGpbOY\\\\\\\/VCPsWVJr8NnPyBFpIbWIGzQNPbWZKHFw2TBlakYf4CIvtaIAa7sJ3L+AE3qVcXbCkw\\\\\\\/xpnuCl7s6xzsx+WIZ2SpHP7rc7w5KJ2FNTftoZAj\\\\\\\/BI6cqkmghKwSjGq6Le5WWcBTUX9jyNCbQsh8pVDmy5q0EFrTQnBkFjMfuKGZ\\\\\\\/XzjWNfZKt8HpkQ4og9NyIU1G61KCEmDSrakFUaAHHsfutig7H2skl3\\\\\\\/teQ70IPErUvD3upOXdfEOUpguIH0wHsbYLLL6cOnibPtgUjeRyOJqhhzAYi++eJaikym1VTd5JA8cqN\\\\\\\/xzwhNCfHySDdY2g2JmdHKOiyeAAsi6C5CB1ZFAa6afclWoT4uuXLLeqaDkZHTaBngdhK438HTcp7nbDuyHOthEq8gdpexFMNZLm1H8kG\\\\\\\/o9VSP7IxNPPbXbPjOKEtTmo6rvDkop71fT09zTh9IN5y9n0z\\\\\\\/364gmMZPKfNgZLQPIvQ2alWnLVwAoex7VoT8N\\\\\\\/gFPh7uuDfAZmk41Fxh+8Y\\\\\\\/zcQjppG9Sho1m8elsPO3ceSYwya4wOPS9ynSiicGUcyqT\\\\\\\/ZyJrcDW2lyuArL5NvJceYn1AFW4tK8cSwheR9d+ns0zBRnahdnm3T3VJF5ClJmkz86tRKrlMHZRuZiy7Bi5kwHh5sQay7vPfPGH29tebHoR1YTATXRR0vzuV+z5Ui\\\\\\\/ZK53OupTc64oFNAb6\\\\\\\/baU1233Rt7smrHN3KYDqJVpPHfrn9Z66weak0SNjeMPyLGwBIN20gsEAPryT91KlhoN7cIWz1bf3FTy1FbgZpPbR439wdpI3cO9dHr0FDknCxTTMmuHvFEhtM4xn6YcmWXkKL28TuXdHBb8khAE98YQMLOGzIwdaYBI3g1+s3xssB8XplNWwJGYuawd+z5Ki1Yd4PuZV\\\\\\\/JeD9hotyndcT2qX5fc7vk34cJPiAfcktTEHCg0KTKRttFBvqAAcZw0HjWUWSCvhfZXrDpJptieou54F1q9w8vs6vxZY3larrw1B4\\\\\\\/qzDZFul0FWt\\\\\\\/CVyd34b0WhePDfHQkCZDtsX462Ec\\\\\\\/71vRkpNTNMl14ovFRi2RMLa8eS8JaW6+wp7HSmdFApVM\\\\\\\/eccF2FzNSBIs4179ChFiLuaqKjG6Lko59giJ5ZoropO\\\\\\\/AZ\\\\\\\/l\\\\\\\/8dDsQyPw1OYylM4SY4LyUck3NarPMxCicgt8F2+Q6uhkfjG3GcHZhLbXD1hK7T4bCaRlWxYSE7S7u8F\\\\\\\/\\\\\\\/GmjWS6zLueAn4PAGGidgBuTrwBMUOFWDWYezm0fNq+l4xJnDrkXjHOYynewTru+nAqEe4f2KKPQWX2Z9ycEIWphg9Ki\\\\\\\/UIFS7W6wJemLqLR9IWdr9i6fVw1sX6Tsi0nTlS10xWrr+Wa0Xw\\\\\\\/GnZ261dL4jiArXHeb980pG22\\\\\\\/Gv+PY3X7iZwTxeqtHKCXrDA4VxeqjjliIFQEQRI9JY3GN1XEb91VK20GfalZsGYJQSxGtX7UoZLG4Z0LLJ1idGGoqykDVj7DSen8EngQoQVDnSLqlEiB9QjOhKi4aEQ+zTuT0w+RKLp88sq39robXOmCMu2KK+d3vy5bthpgqgP2qLgzO8J0+B4bUxy7WkivN5dojYMyqp9iLiks5PxGSgGBnP5WaiJVzUh7lrnkYxF8FKRPE845nMzSmie\\\\\\\/vPt2kVYSlJnulgCb++PFqwNauF+FT9AoD1seb3NtsFEElDaudU08PnmEqmsxwWFnRHYXzXJsGVC+edFtq1WGiL32ddQZ5N6APJSHbPgNiVnY7yCUezbwMa2EYjg+zvUwAI60Z501jngjRfiX9oLAJCH0ETDDhs2Je96KUu8Vq2Hge1KTxGM8sQKufSWBo8W6bonv0Q\\\\\\\/FP7Tm\\\\\\\/aNo2G7\\\\\\\/LINBvyRznFi1klW7SBC3W7nOqvYHkOg6OBP1id0IeVH7v0qhADG\\\\\\\/Z3Nrod0lpx2TWvrJFROQPq9CXa+mHI\\\\\\\/fgKybS78y0W1D\\\\\\\/FPyPip1UUnkdZi79Oy0KGgxSCtnnWiBuzJ4cNCwyolwCzMaP40w1fgl+1f085klatTt3CG7sOpnOvXPJRYwvCX7VlrK\\\\\\\/N2ziyv1AaYKpP85wQkIZS4SPWUZe35ORtSp26hI4iuHqYwV+Cn3dJn2x3h8hp76UlpK1CrAxetGLc4lIgPggEDnV3P+qo8LIpv7VA7\\\\\\\/1Ne+HkmDxybauITdmA8+v0BspW530t1I\\\\\\\/WOMaCHmlR935sTWYES+9\\\\\\\/3koePyA293++6mhR08esQ9dfC9uN8w2H6NBc3ll\\\\\\\/Jvm3j4ivyQuFMFoTFDNREh2F5V3KvB0MLrJ2FephZkI\\\\\\\/d8DT+LW\\\\\\\/UZltaWhK\\\\\\\/XEIH4NL5\\\\\\\/AayHAx1DGEDGlZHcBAjPKezLQTYxbOc7X7YhgVJjQRdImE3hACqNaXx1f+x554eq6ahzSFy4B\\\\\\\/zpzeYUBQyCNtjfqxK\\\\\\\/ACCIWDxM0XLD43ZtNVKyPmVbO2J5X4U31Pd2IITdfmBRwGfhQV03IqrP19KpFcUh01Bm6IQOfiunW1OTw1O0mVlktyNdc6yvH0GMl0lov4Qbc7SvXmaHKipT7p+6bB4RFMfWB8SD1gOAaW9Kn5qiTRD2sWBTOu8sK7Ap\\\\\\\/B3TaZPmdSZ8IiYiyUnz70PCqWFde4BxpVQOKjMDIvC5RsmFMLJVrljWTRdcaIpvGnrTlpvzv7FY22p2GtsD8HCRMTQkhZnlpVBTmEuL4zIuC9tyPU0kK6AgiyFRmoqfxXD+pZI3XSjdy+TIwhwQpkCrhf\\\\\\\/DUDg+ej70DV+ipVb9q1TQhBxntIH7v8oyiLI7Pb00UbO9STDY+tETWzLHluJL5zxO7Q81gUYwNNvYmlEW0InpHiwCzmHQc4cSnPdKdGTgqecQMtd7YNcijadNLQm7sF6zrAvt4yVErHE1eoHMq4Rfz8T8LM4Y8O0okpKpDq25azegIvZiI0bhDWGYMO7c1M3iT\\\\\\\/M\\\\\\\/ysSCzfod\\\\\\\/ZSNGn9WTe93qkkormxrJbG5LmPEhadAyivwx03DObyhI0oyf9l+Yz2wqqtLu9N0WoaFmXYju5sN5cNK\\\\\\\/MKobKeSOaq5n69y5+L120dMYkYdETcGjPIU03OVIROEFYET1+0GYr0JX4UWR2gHVd+M5tV4Oz5Ib0QwDG8D0+TRXIu+jrGcu4QtDu5+pJGitjLi5eLzJrzjHujkEqiF93s9N66pIV3m9Tu7nXhR8GKV853f8HjCrj+x\\\\\\\/BvjQlFZsji8bx+Q+knwu1TzU4iPE\\\\\\\/\\\\\\\/VfLGa1EZ4r+QVsex3fX3gQM+4bsiJepyIiQ9M5bGxVdX8HRmqjKaGmIhljXw1bFIMGhpMOA1I4N9wihsiiSWTjf0dCzVKHsHhTiko6oApPMPVUtJw1jTN3kYgXyjI2DM14Uau3YZDP7POSApwh7HhtfoWuM4fYqn6JfLpOiwUtuMaUcS3+iH8KiClm4xw4gDksM8ZnPFk9CyCU117mlLLb\\\\\\\/glzJ+T+13FYPVsFQ\\\\\\\/llQZV7vq7ApWrgzhhikd0Ujr94fVkLQAc048AeCO4R2eYxMWrLvHBjncr3HQ0PGM1GW2Xc4TpcoaRUp+ZqWdMzna+\\\\\\\/HOI0+wTFobXOQya1nBuhQVQJmQP2TvhIrBTyWmypf4h2kTbkmEbG8kO2rivngvXCe0PYJhYsXi8E5RWjfL8odyl4KblmGfMkL7ffd5F5kscKDWXhLJT9kj0yv2312EpEZMTo24JgC0AZ3i0QBt9s1kRJF\\\\\\\/9gLSEeMsADeWQR9faMiadmLSqE4\\\\\\\/tpTZOCcmx5goKBYYoIit+kQgdaPtcPiGguhHmft\\\\\\\/uSkNR24Dgv8fv65iWb7nBlqnWyEXSbqM40YNXxw3\\\\\\\/EyHLo9S1Hzo1jfeogxlBasMwCEMF4XqOnYOuxKvA9j7bkfi\\\\\\\/WQSUmLfGDvmUgNt0CHDDw\\\\\\\/7\\\\\\\/P+X7VT8cf5dZoE5623c3Kd4NmliTsY1yLmfwCNJ2Q7trX3Heohq2D0u3EIPZJAwhCPVmE3jG\\\\\\\/NoCGWM2hBDePKvCpJui\\\\\\\/Cks+Igw+6+8Z4G6KVOXa0AywQ+Rucm\\\\\\\/XUEuWgR6QJu3zzkoUIVKueEj5HOV7M5LgEzSCTGbZf0HsEY\\\\\\\/5W7CC\\\\\\\/Ik3iaqs3U4SoXskmYGyk4B6Ntb\\\\\\\/BjHz2G5MdX9xuiXCqdwAJyXVKMRO0DV\\\\\\\/v3+6MUe6yWhpvxqCG12BpUfvzLnQ4CG15gT5rhlO3A8uy\\\\\\\/Ghf4+nHTgbvKHaKwMrhvMjQFCx7Nf6gbn+3M19N+iUH3dmLmbQAcqSqQbP1EK3bHw+FWpyrXKb89egzMnyxe5TsjrlaFo3sxIqFvf8Cid5VKFXjaGSCskhudg0\\\\\\\/3a\\\\\\\/VUPGBMeToken48lRTRlcHLzd35GFm7r\\\\\\\/tzmHdFjOaBGrWn+E7w4Js4k\\\\\\\/KABlU7RR0dSiNKoPKDKWZqar68n+9DmeAcbnyNzHEwT4Tkab6\\\\\\\/BRQGvQ9CO3RIOMpLQQhE0itqBumhm32YqNB80gTFSBNFSagAagBoXn087+fgwN9yrlaeAlGyj6C\\\\\\\/kdEV8E\\\\\\\/xaBp\\\\\\\/I5tB1fCtk\\\\\\\/Hs+9wW6bSDeGmZszjATOrKtluD9BV9Y\\\\\\\/PaZYO6L2PZnxGdxVXfGds59IukyL5MQYA7go\\\\\\\/olQaPDvNpCYp1j9MeS6Ypg0uifFUzzsY1wq5mvhzw7fSJR0k4wjvcRgDKEFyOBGdiW64P2UikHO1Cd8eceoULROzs9+ch3P90xMYMJMsYalsgbSBdBpVu2IV1IMrTPPY\\\\\\\/U6NGvH0fT6x2d\\\\\\\/0bMo9apV4cbQ2r7zKg10G4YhojVAKq5mWctN1AvmK7sU1BZA3E2A0X8AMSIHGNZeG+9W6jYbDf\\\\\\\/702rPO8wHkBlqCBv3KXsq+gFbQOEY9ArFDKwu8hrXhqjEwuz\\\\\\\/hw7JwtdMLGav95jUuPwGPYr99F3F\\\\\\\/9GyZ\\\\\\\/21iJUGE5SK3HomJSdUIXT1JrgeTmVuX5sCpQIYGNqXlwkGZC2VehXWbyTwz5R92+8CH37TK5tgTpoFuBTzcVF5CGg+Ajww\\\\\\\/YvUYHKwUm0dAohZKvqEtAFZGAl+0KpBSfjGggQHByDcAvmEM2RCa9CeQI+sgBGxtiR4YZPkJdZgEcQ8Qvh5tD94Hp32Uj+sJvaMm7OmFFy21DpwuNkpdzGO3juG5dRwG9Xbwx4XBKxs2aRSY\\\\\\\/vb3MOAujRVMXrznd2SyeKQV+f7cZUi+jsxCXrFQt\\\\\\\/54jflQUvV+zxYUdA0rlaSfaHgXZgUoqyeueKJijtXcJAEgogBb2sZ26xImmYAihBWjEKqLPcYChbb1Kb5rtO87\\\\\\\/7bIYmUzVPoquEG3nf2wAUazKksYqFK\\\\\\\/XzgaFHz12thnV7qPag8\\\\\\\/GgisCc3ZRdrYg7je8LHUnKZONAYz6x191o2+LNDHgKnHrdma\\\\\\\/iaVmL8XM9s0TRWsRonk4CqRb4aGwRSdDV2m6KBcuNtqh7GdfvLyVCAqfkIuaDGRCOG3dO5E9XjZiSeUO+yYQvTriiK+GxdLUL0R6GqOJLmg5A3H\\\\\\\/LMcQw7p6GMZSFAxRivIqL929FdF5aO\\\\\\\/Mbp3QqQlcwgQ\\\\\\\/YNYfYcRdmEJfO7ZVUwmudh4Vvg59rd\\\\\\\/5AHkSsLIu7gQtihPt2Xq7PBt1Jnwd\\\\\\\/AfLFpsFLHfurcLCAUPSIasQcxgo3P1\\\\\\\/kw9hiJsScNqPa2VAC1\\\\\\\/D7OsrDPZA7LKt0o8MbNsBh\\\\\\\/+mLmsNb142bjhpfhRIW8O8bS6gI3ghG2BIJKJqFr\\\\\\\/\\\\\\\/Yp\\\\\\\/aUk5a8kLKuo04guT2RSqVkOlmJOhYiF8A0oUMXkwpbABFXJWrjS1jpNdObGBVKFmvyj82Wca1Xu7psbaR+aeA06b7Igh6tbAAZjv+JWhHreeq3z7zeuhe+JoxmIVu027KwXrOZC4hQe2APP8DngRZdfx\\\\\\\/HfkHhEE1LBfI+Ap7KTLtdvhZ3r8e2982PRYoQq0VotfJ\\\\\\\/xRYo3Zd9BUYPItHPKR3xZoxsdqPUSa3uaaEUAoXlXYzAgpxmXv19g1604fEcY9tuVa66PERqaTqNdGHE9jsFr3XFI4SqfDbxQnv+a6TQ5su8ToMKBG+beDAZ2\\\\\\\/KFBZ1BqFfNWcYrJVB+a+iVrpXhKEbFEasEOghEuGWrz0zeBxVgRQ7e+1jGU6TE6G\\\\\\\/i5BbhGLw5b5l\\\\\\\/Rwj8CVzwtKJoMqzXeLxtmVmYJcm8swdZqhb2Nf3+8hXQr+DZIBser7urOYShhpCjyVIHVJuGhp0vjhrjn+kjG+DwRLbrPrFrhGhNaeXj9PZn\\\\\\\/oLdJhHeNA21WllUm7vHjkpDCZ3ndUjKJR09s9T5oZIYCOTIt68EpdLsuBWbVn8Sud8DOOxk2150J3VFGB\\\\\\\/GWMw3WLFageVXt1NRUgRcyrLC849\\\\\\\/W6Zs+q1bQ645gg4ESpQLHjSCGTVBueYXgJTfdwdcAkdNMwOYWKgGYTiaDt6TbUCtpt6iDnYyloPTzRUMvwWC4lBbkd0qKwH6hm3g3HQxeOqSGIdsJJtjpdntjiHRXBQxPUBoJwhHl0tbMVaUK+VSpnUsMlXLqcEMxVfx+JKYLh33Q3QmQcRUvqZWKEbIh39SFZIWRiI06rBYmdNVs7BI3E4aJyjXUt36RQV7iappNP\\\\\\\/tq3QV1mY6qMMyOggvQZPTZgFZ4EikqeYbODwnMPG9eOBkb1CnUsQdX9ukedZ97AvavHsH0fLMwiiHotPkTtiQ2TUbnhogRgYNS+2GXhiPLgWyywCBhumzuIF3ko0COMez2r4NEr\\\\\\\/WIAtzb+ScHPM7INDMqZeGWITbe1pgQ3IOx3pQLNiRv4U7i9TMz\\\\\\\/LS9GidsU3JaBoFnmzlbi51xU\\\\\\\/TieJ8u05RwN4DpuzqppkOW6G9tgm7c2oI+LfYMn5ijCMWmie03d6q19ZDLGXm8hNJctfftGg9\\\u0022,\\\u0022iv\\\u0022:\\\u00220z8c2tQGCLrNSv2FDS1ayQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227950d73f1877b2292fe69119910f0b9bc572129529d8e7480f8b5adda1784047\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JmVY5ugoK6Ag1hRk4nV105T5yrdtC6Clm3Rg5\\\\\\\/SCshQCg2zBk6ShqfVP9NuATasEqnFcJP+jcvwAjYKod6IFppcBkUoHOJqhnqIfaMomn4yJ9jcUxrp0DRmAycdWSedb9yjN2gnAUS2IipY6LXCmIAZB7Sdu7p78yt6P+nqjtWHywkX4fWomrkuhb0Pt\\\\\\\/DlgckWR9RhVyJxL7hIRR3fY9Mp58MuSHF0cAKnHrHKRf7StU+PcvVAXhQCpf214F4OQwIhd\\\\\\\/eWH4tNTp8LCOcLrXtZUuJvTyLeJZfVvZ7SxG8PEz2upR4Rqelh1qR1kZMQ9nMq4B8+ms38lZXMn8c\\\\\\\/o1biKUpNc5KMdDw+NfXWE+MuD933ZMLtX63XbieLvbrmxYfelCp0rp3k2kQjG2ai8o5Jwhsx+tMhVV9sw\\\\\\\/gJfx2zQBJbbgAQav285FLXSK4WnSyQ1BxdEtABHkdCBT2Uydherx4vOp\\\\\\\/CSxF6IZmI8xXUfc8jfUYjXDAT4Z0XdxlGH3FpjEjRNUYGT41RpAxeMKSH3\\\\\\\/SHrhR0cIw5aD9x6zr6CV0qYs+sd4e6UDRb1LhFHR1r\\\\\\\/TT93ud\\\\\\\/7IJc05ezdYsc4BlE4YJEGKcQXzXn6ESzgZjB7bs3TXEaYvjVP8yUmMaDEPFlD1EVyp7rMG4FLKRc8qmvpIWeFXhjfNm7YAX8jj3aGnFzPadT\\\\\\\/bq4rSkq1nYf6UfwUHJwa3S0UK\\\\\\\/Ic7qrgXjLAL5anPNpXCZ7+W1moUlsOWEFzxIiqqOBoOFoOLa3fwZgQl9Mdp1iQRaohIRM6Wvp+dXQZmpW0RLucoO\\\\\\\/rHG4ncOrDvQR0Bed0kmN+4CiGf+QWRU\\\\\\\/8VJWovNsKIlPCtqY4TUfxIBcN0PdUTabmJN6US\\\\\\\/ooTF2p+twDskj15xKzup3Sx4nE3tcyUo4BxBveP9b\\\\\\\/aPj9yjmsVzRJ1sBsjGl+gMYVuO2SPQ3jcvRn0DPWQYyhO5IdCBqMrScpnYNFRciKWHgXZczU3aklnG4vbE96g38jEy8vksn0O6l\\\\\\\/GsaEieIVYZtzpVOegxWkwbtOIFc7GZdiRatrilcBb63DRPTLaj6t8BKM36H9vhEei7XbQJGNJ7nSB1NiqUnM5QEkKA0TKLQk5z4GSrkAdqOvunPa3xuq5h4znOZB0dSoJ2CS2K7vU91uNa7VtweA207dLEnogzPTC9fZoHlrM2rG2oLlr43+VzaKTKRX9sRm3f3HKM5y\\\\\\\/qTbV2lc2qVn\\\\\\\/Nz+NmIeK7X7iX60X+KHr1CDjCj6BPgh8eztQr5Q56d2T5gKusPmnyWfTrKKgMOlRD\\\\\\\/xCIg1tIYvS1imAR8f+olS\\\\\\\/KsIGdfD2aJveV0jfSBHKVduTRlm125i3QyVc7PSfqEC8R3tlUr33vTKhQPTg6Sd\\\\\\\/bazpfrOe7f5iiCU9P9tvaiN1SMwv4jbAh7Vgor5rQpj6sQHcwvNvnTpO9XEN1Q55foRS30IRqkOwQRBOI4bjjN9HNxzxrvyHcUFzVHOSArbWe0kP2xbb3YKMPSovijx+\\\\\\\/CDqxM6JgzePq2HN1V5g930pfgyl4SlIGdwN2dVjdTvQx5UqXVMB0RKSZESn5A5lPvWMKq\\\\\\\/LE0Ok723la4\\\\\\\/15ah2E+I9r3bSQRq3r+oPm6cJievh4jOWRqX1scwV45z4O5mDBNocXASdYXed07nVUNKxeKtGFJoyjId5tDndiS97U6ZRAVEyHrO2NN2Z15cfkXfNt\\\\\\\/vTMjOM70NAOKBQvxLG2ZoegzulrzTaKzzbsghGA2Mktd5AkKEO5TLKgoby+u6aLrOTwjxfAYRD4l5Ef90TGc9XMBV+0Oyp5abceW4sZu+EPtFXwCm8kmThvt7Caab886YFwILt4G9jxtQmR2gTbZL1NJ5\\\\\\\/uHGf80Aw5CLvGPJH8oh2y4Bqf+IfLMArmw4wJv9flh0IfS3iz5zjAX5OGM4IDbvKgDF7cQlw2A3mJ7UwDGXF1BvmFquinUB6zjpv8ZoSsDDUdIqJc78zR4MUJCYXoWzwwVB0eIyvXHMAHxVzsyFtSp84bbvO58=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kh9CLjG0Pah65\\\\\\\/9\\\\\\\/kniVWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022871308c7e7ca5f64d917e85a0260b1cff4ad9f147f33a4ea13023409ccbf244d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459469 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/795" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lF0gYhvNged2Nxk9JEn32BhAd8AkXXdnzlpRVZ9ccDM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/795" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Kl3NH9rQ6Yx1vcTyn1otF3PGzI0Fd_iO9ug_UMo8Zjc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/783" title="Genetic Structure and Linkage Disequilibrium in a Diverse, Representative Collection of the C4 Model Plant, Sorghum bicolor" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/807" title="Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459469" id="node459469" data-pisa="ggg;3/5/795" data-pisa-master="ggg;g3.112.005462" data-apath="/ggg/3/5/795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (<em>Picea abies</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hanna Larsson</span>, <span class="highwire-citation-author" data-delta="1">Thomas Källman</span>, <span class="highwire-citation-author" data-delta="2">Niclas Gyllenstrand</span> and <span class="highwire-citation-author" data-delta="3">Martin Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">795-806;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005462 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459469" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hanna Larsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanna%2BLarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Larsson%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHanna%2BLarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Hanna.Larsson@ebc.uu.se" class="" data-icon-position="" data-hide-link-title="0">Hanna.Larsson@ebc.uu.se</a>
<a href="mailto:larsson.h77@gmail.com" class="" data-icon-position="" data-hide-link-title="0">larsson.h77@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Källman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BK%C3%A4llman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4llman%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BK%25C3%25A4llman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Niclas Gyllenstrand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology and Forest Genetics, Swedish University of Agricultural Sciences, Uppsala, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niclas%2BGyllenstrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gyllenstrand%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANiclas%2BGyllenstrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/5/795" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459469" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459469/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/795.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459469" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459469/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/795.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459469" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459469/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/5/795.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459469" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459469/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/5/795.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459469" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459469/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The site frequency spectrum of mutations (SFS) and linkage disequilibrium (LD) are the two major sources of information in population genetics studies. In this study we focus on the levels of LD and the SFS and on the effect of sample size on summary statistics in 10 Scandinavian populations of Norway spruce. We found that previous estimates of a low level of LD were highly influenced by both sampling strategy and the fact that data from multiple loci were analyzed jointly. Estimates of LD were in fact heterogeneous across loci and increased within individual populations compared with the estimate from the total data. The variation in levels of LD among populations most likely reflects different demographic histories, although we were unable to detect population structure by using standard approaches. As in previous studies, we also found that the SFS-based test Tajima’s D was highly sensitive to sample size, revealing that care should be taken to draw strong conclusions from this test when sample size is small. In conclusion, the results from this study are in line with recent studies in other conifers that have revealed a more complex and variable pattern of LD than earlier studies suggested and with studies in trees and humans that suggest that Tajima’s D is sensitive to sample size. This has large consequences for the design of future association and population genetic studies in Norway spruce.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/conifer" class="hw-term hw-article-keyword hw-article-keyword-conifer" rel="nofollow">conifer</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/tajima%E2%80%99s-d" class="hw-term hw-article-keyword hw-article-keyword-tajima&#x2019;s-d" rel="nofollow">Tajima’s D</a></li><li class="kwd"><a href="/keyword/resampling" class="hw-term hw-article-keyword hw-article-keyword-resampling" rel="nofollow">resampling</a></li></ul><p id="p-4">Population genetics inferences are primarily based on two sources of information: the site frequency spectrum of mutations (SFS) and the statistical association among those, that is, linkage disequilibrium (LD). The SFS has been the main source of information for demographic inferences and tests of selection, whereas LD is the key parameter in association mapping studies, impacting their feasibility, cost, and resolution. In the presence of long-range LD, fewer markers are required for mapping purposes, but mapping precision will be low. Conversely, if LD decays quickly, a large number of markers will be needed to cover the genome leading to a high cost but also to a high precision. Both the SFS and LD are influenced by the joint effect of biological factors, such as the rates of recombination and mutation, the demographic history of the population, and natural selection. They are therefore both highly dependent on which species, which part of the range within that species, and which parts of the genome are considered. In the present study we will focus on LD and the SFS estimated in Scandinavian populations of Norway spruce (<em>Picea abies</em>).</p><p id="p-5">In general, conifers are expected to have a low level of LD or, conversely, a high population recombination rate, because their representatives are mostly outcrossing species with large effective population sizes (<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2010</a>). Consistent with this notion, several multilocus studies have found LD decaying rapidly with distance between segregating sites in conifers (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brown <em>et al.</em> 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Krutovsky and Neale 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">González-Martínez <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pyhäjärvi <em>et al.</em> 2007</a>). This has led to the conclusion that too many markers would be required in conifers to make genome-wide association studies cost-effective (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Neale and Savolainen 2004</a>). These earlier studies were, however, based on relatively short loci (generally around 500 bp and up to 2.5 kb), and a recent review found that genome-wide recombination rates inferred from genetic mapping efforts were in fact lower in conifers than angiosperms (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jaramillo-Correa <em>et al.</em> 2010</a>). In model plants, genome regions rich in repetitive elements have reduced levels of recombination (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gaut <em>et al.</em> 2007</a>) and the low genome-wide estimate in <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jaramillo-Correa <em>et al.</em> (2010)</a> is hypothesized to be, at least in part, the result of averaging recombination rates across conifer genomes rich in repetitive and transposable elements.</p><p id="p-6">Indeed, in a recent study in the conifer <em>Cryptomeria japonica</em> the authors measured extensive LD at a distance of &gt;100 kb in mostly noncoding DNA consisting of ~50% repetitive sequences (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Moritsuka <em>et al.</em> 2012</a>). The approach of using several loci and sampling individuals from a large distribution range to estimate a common level of LD may also have masked the variability of LD among genes that is now emerging as whole gene studies report high levels of LD in some of the genes examined (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lepoittevin <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Namroud <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pavy <em>et al.</em> 2012</a>). Notably, in <em>Pinus sylvestris</em>, in which previous studies suggested a rapid breakdown of LD, the sequencing of a group of allozyme coding loci revealed reduced recombination rates and complete linkage extending over several kilobases, possibly reflecting the presence of selection at some of those genes (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pyhäjärvi <em>et al.</em> 2011</a>). Clearly, the genomes of conifers are less than homogenous with regards to patterns of LD, and more data on the variability of LD across species and the genome are needed to fully evaluate the appropriate method for association mapping studies.</p><p id="p-7">Tajima’s D (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Tajima 1989</a>) is one of the most popular summary statistics of the SFS. It belongs to a large family of “neutrality” tests that compare different estimates of the population mutation rate, θ = 4N<sub>e</sub>μ (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zeng <em>et al.</em> 2006</a>), where N<sub>e</sub> is the effective population size and μ is the mutation rate. The key idea behind these tests is that some estimators of θ = 4N<sub>e</sub>μ will be more sensitive than others to specific departures from the standard neutral model. Comparing two estimators of θ should then indicate the type of departure experienced by the population. For example, Tajima’s D is calculated by taking the difference between the average pairwise nucleotide diversity (θ<sub>π</sub>), and the number of segregating sites as measured by θ<sub>W</sub>, the latter being more sensitive than the former to an excess of rare variants (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Tajima 1989</a>). Hence, Tajima’s D values are negative when there is an excess of rare variants and positive values are obtained when there is an excess of intermediate frequency variants. Tajima’s D, as well as other test statistics of the same family, is inherently sensitive to sample size, as θ<sub>W</sub> is defined as a function of the number of segregating sites (S) and the number of individuals (n). The impact of difference in sample size on estimates of Tajima’s D is difficult to track analytically because both sampling strategy and species demographic history can have impact on the estimate (see, for example, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Städler <em>et al.</em> 2009</a>). In a study of climate-related candidate genes in six populations of Sitka spruce, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holliday <em>et al.</em> (2010)</a> showed that estimates of Tajima’s D became more positive toward the northern leading edge of the species distribution. They suggested that sequential population bottlenecks during postglacial recolonization created this pattern with rare variants more common in the south and medium-frequency variants more common in the north.</p><p id="p-8">Here, we investigate variation in estimated Tajima’s D values and the level of LD at eleven loci in 10 Scandinavian populations of Norway spruce. In a previous study of nucleotide diversity, we concluded that the species has low-to-moderate level of nucleotide diversity and a low level of LD (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>). However, these data included only two loci longer than 1 kb and results were therefore mostly based on loci of short length (~500 bp). In light of the variability of LD recently revealed in other conifer species, our first aim was to examine levels of LD within longer fragments. The 10 populations used in this study are located along a north to south gradient on both sides of the Baltic Sea. Earlier studies of spruce populations from this geographical area have revealed a weak, though biologically significant, population structure, likely reflecting recolonization routes within Scandinavia following the last glacial maximum (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2012</a>). Norway spruce recolonized Scandinavia after the last glacial maximum from Russian populations located south of the ice sheet and, possibly, from cryptic refugia in the northern part of European Russia too (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Binney <em>et al.</em> 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Väliranta <em>et al.</em> 2011</a>). The spread westwards followed two main recolonization routes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tollefsrud <em>et al.</em> 2009</a>), today reflected in a complex population genetic structure despite the generally high levels of gene flow (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2012</a>).</p><p id="p-9">The second aim of our study was to investigate if estimated values of Tajima’s D from different populations varied and whether the variation varied in a clinal fashion as reported from Sitka spruce. Because Tajima’s D estimates depend on sample size (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Marroni <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nelson <em>et al.</em> 2012</a>), three populations were sampled more densely and we used a resampling approach within these populations to check the reliability of Tajima’s D values obtained from small sample sizes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><p id="p-10">The 11 genomic loci for this study were chosen from previous studies of sequence variation and gene expression in <em>P. abies</em> and the selection was based on length of locus, available primer sequences, and ease of amplification. <em>PaCOL1</em> (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>), <em>PaMFT1</em>, <em>PaFTL1</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Karlgren <em>et al.</em> 2011</a>), <em>PaPRR1</em>, and <em>PaPRR7</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Källman 2009</a>) are hypothesized to be involved in the regulation of bud set due to sequence similarity to known genes involved in the regulation of flowering time in the model species <em>Arabidopsis thaliana</em>. The remaining loci correspond to genes exhibiting a significant differential expression under a bud set experiment in which plants were transferred from constant light to short day conditions (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Källman 2009</a>). Of these loci, <em>PaCCA1</em>, <em>PaCDF1</em>, and <em>PaAP2L3</em> also show similarity to flowering time genes. The length of the genes ranges from 1.5 kb to 7.2 kb, with a mean of 3.7 kb.</p><p id="p-11">Ten natural populations from the Scandinavian distribution of <em>P. abies</em> were chosen along a latitudinal gradient on both sides of the Baltic Sea, ranging from 58°N to 67°N (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Seeds from eight trees were sampled from each population except for two Swedish populations, Fulufjället (SE-61) and Höglunda (SE-64), and the most northern Finnish population of Sodankylä (FI-67), where 24 trees were sampled (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Total genomic DNA was extracted from the haploid megagametophyte tissue of one seed per individual using DNeasy plant mini kit (QIAGEN). Because the aim was to obtain long-range estimates of LD, we chose to amplify genes of considerable length in full, or as overlapping fragments when total length exceeded successful amplification, and to sequence only the ends of amplified fragments to reduce cost (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Voight <em>et al.</em> 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Berlin <em>et al.</em> 2011</a>). A graphical representation of the genes with exon and intron structure was produced using the GSDS server (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guo <em>et al</em>. 2007</a>). Amplifications were performed with primer sequences and conditions as described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/005462SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/TableS1.pdf">Table S1</a>. This paired-end sequencing approach was performed using an ABI3730XL sequencer (Macrogen Inc., Seoul, Korea). Sequences were base-called, assembled, and visualized for manual inspection and editing using the PHRED/PHRAP/CONSED program suite (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ewing <em>et al.</em> 1998</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ewing and Green 1998</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gordon <em>et al.</em> 1998</a>). Bases with a Phred score &gt;20 were retained for analyses. Sequenced fragments were aligned to existing full-length sequences of all genes to determine the length between sequenced ends. An alignment file containing sequence data for all loci and individuals is available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FileS1.zip">File S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/795/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of Scandinavia with the locations of sampled populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-869417027" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Map of Scandinavia with the locations of sampled populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/795/F1.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/5/795/F1.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/795/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/795/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462742" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Map of Scandinavia with the locations of sampled populations.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462711/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462711/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Location of the populations used in this study and sample size</span><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Population structure</h3><p id="p-13">For each locus, Arlequin v 3.5 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Excoffier and Lischer 2010</a>) and R (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Development Core Team 2012</a>) were used to identify linked SNPs with a significant <em>r<sup>2</sup></em> of &gt; 0.2 after Bonferroni correction. These SNPs were subsequently removed from a multilocus data set of all haplotypes and the population genetic structure was investigated using STRUCTURE v 2.3.2 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pritchard <em>et al.</em> 2000</a>). All 11 genes and all individuals (in total 255 SNPs) were used in one analysis whereas a more stringent approach, excluding 15 individuals for which only a few genes were successfully sequenced and filtering loci with more than 15% missing data, left loci from seven genes represented by 112 SNPs to be analyzed. For both data sets, we used the LOCPRIOR model with correlated allele frequencies among populations to perform the analysis for a number of clusters ranging from K = 2 to K = 10 with 10 independent runs for each K. A burn-in period of 100,000 iterations and a run length of 1,000,000 iterations were used. We merged data from the 10 runs using the software CLUMP (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jakobsson and Rosenberg 2007</a>). Global F<sub>ST</sub> was calculated between the 10 populations for each locus in DnaSP v 5.10 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Librado and Rozas 2009</a>).</p></div><div id="sec-3" class="subsection"><h3>Nucleotide diversity and sampling effect on Tajima’s D</h3><p id="p-14">Standard population genetics parameters were calculated using DnaSP v 5.10 (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Librado and Rozas 2009</a>) for the combined data set as well as for each of the three populations with larger sample size. Indels were excluded from all analyses. Tajima’s D was computed for all 10 populations to evaluate deviations from the standard neutral model. To examine the effect of sample size on estimates of neutrality in <em>P. abies</em>, eight individuals were resampled randomly from each of the three populations with larger sample size. The resampling was repeated 100 times in each population and COMPUTE (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Thornton 2003</a>) was used to calculate Tajima’s D for each of the resampled data sets, excluding missing data to get comparable results to those obtained with DnaSP v 5.10 (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Librado and Rozas 2009</a>).</p></div><div id="sec-4" class="subsection"><h3>Levels of LD</h3><p id="p-15">The squared correlation of allele frequencies (<em>r<sup>2</sup></em>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hill and Robertson 1968</a>) and D′ (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lewontin 1964</a>) were used to estimate the degree of LD per locus and calculated based on pairwise comparisons between informative sites only in DnaSP v 5.10 (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Librado and Rozas 2009</a>). The two measures of LD provide different information about the association between alleles: <em>D′</em> reflects recombination between the two loci whereas <em>r</em><sup>2</sup> is more informative on the power to predict allele identities at one locus given allelic states at another locus. Fisher’s exact test was used to determine the statistical significance of each pairwise test at a level of <em>P</em> &lt; 0.05 after Bonferroni’s correction. The decay of LD over distance was investigated by plotting <em>r<sup>2</sup></em> values between all informative sites against the distance between sites and fitting the expectation of <em>r<sup>2</sup></em> to the observed data, applying the formula from <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hill and Weir (1988)</a> in R (R core team 2012). The same was done with D′. In this case the decay of D′ was fitted to the function D′(t) = (1-r)<sup>t</sup> where r is the recombination fraction between pairs of SNP and t, the parameter to be estimated, is the number of generations since D′ = 1. In order to compare patterns over populations we assumed 1 cM = 1 Mb. However, because there are no data available on the relationship between genetic distance (centiMorgan) and physical distance (base pairs) for spruce species we also performed the analysis assuming 1 cM = 10 Mb, 1 cM = 20 Mb and 1 cM = 0.5 Mb, but because the comparisons among populations were not altered we only report results for 1 cM = 1 Mb. R code written by F. Marroni and available at <a href="http://fabiomarroni.wordpress.com/">http://fabiomarroni.wordpress.com/</a> was used to estimate t. The results were computed for each gene separately and for the complete data set. Finally, all genes were used to investigate the decay of LD within each of the populations with a larger sample size.</p></div><div id="sec-5" class="subsection"><h3>Recombination</h3><p id="p-16">The composite likelihood method implemented in LDhat (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McVean <em>et al.</em> 2002</a>) was used to estimate the population recombination parameter ρ = 4N<sub>e</sub>r for each locus, where N<sub>e</sub> is the effective population size and <em>r</em> is the per locus recombination rate per generation. Because our data set contained missing data, we took advantage of the precalculated likelihood files that are included with the software and assumed a fixed θ = 4N<sub>e</sub>μ οf 0.001 for all loci. The per-locus estimate was calculated using the data set including all population, and using the three populations of larger sample size separately.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-17">Based on the individuals with shortest read length in every locus, a total of ~1.11 Mbp (1,118,584 bp) of sequence was used for the estimation of population genetics parameters (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). A schematic representation of the genes and the regions screened for variation can be found in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. On average, 97 individuals per gene were successfully sequenced (ranging from 74 to 114 individuals) and 1062 bp were sequenced per gene (ranging from 457 to 2449 nucleotides), yielding ~103,400 bp of sequence data per gene. For analyses allowing for missing data the full range of read lengths among individuals was used, increasing the amount of sequence data analyzed. In the restricted dataset, we identified a total of 356 segregating sites, of which 142 were singletons and 209 were parsimony informative sites. Five sites were found to be multi allelic and were not included in the analyses.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462752/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462752/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Nucleotide diversity and summary statistics for the 11 loci used to estimate long-range LD and structure in populations of <em>P. abies</em></span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/795/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic representation of the eleven genes amplified in this study. The regions sequenced and analyzed are indicated underneath each gene (see Legend)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-869417027" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A schematic representation of the eleven genes amplified in this study. The regions sequenced and analyzed are indicated underneath each gene (see Legend).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/795/F2.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/5/795/F2.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/795/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/795/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462748" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">A schematic representation of the eleven genes amplified in this study. The regions sequenced and analyzed are indicated underneath each gene (see Legend).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In the three populations of larger sample size, the average number of sequenced individuals was 20 in SE-61 and 16 in both SE-64 and FI-67. The average length of sequence analyzed per gene was 1224 bp in SE-61, 1251 bp in SE-64, and 1393 bp in SE-67, with a total of 218, 187 and 208 segregating sites identified respectively, for details see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462705/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462705/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary sequence statistics for the 11 loci within the populations SE-61, SE-64, and FI-67</span><div class="sb-div caption-clear"></div></div></div><div id="sec-7" class="subsection"><h3>Population structure</h3><p id="p-23">There was no clear population structure among the populations sampled in this study. The ΔK method (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Evanno <em>et al.</em> 2005</a>) suggests that the data set most likely consists of two clusters, whereas the likelihood values suggest that the data set is made up of five, seven, or eight clusters with generally small differences in likelihood between different K values. Plotting individual cluster assignments for both K = 2 and K = 3 (the number of possible clusters suggested by <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2012</a>) further vindicate the lack of meaningful structure as most individuals look admixed and there is no clear geographic pattern emerging (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS1.pdf">Figure S1</a>). There was no difference between using all loci (including those with missing data) and the more conservative approach where less missing data were allowed.</p><p id="p-24">In line with these results, observed global F<sub>ST</sub> values at individual genes between all populations varied between virtually 0 and 0.049. Using only the three populations with larger sample size led to a slight increase in F<sub>ST</sub> values and for <em>PaFTL1</em> an F<sub>ST</sub> of close to 0.2 was obtained (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/TableS2.pdf">Table S2</a>).</p></div><div id="sec-8" class="subsection"><h3>Nucleotide diversity and Tajima’s D</h3><p id="p-25">Polymorphisms from both coding and noncoding regions of the genes were used to estimate nucleotide diversity. We chose to analyze them jointly because the amount of coding sequence was limited and treating synonymous and nonsynonymous sites separately would have led to estimates based on only a small number of sites. For the total data set, the mean nucleotide diversity (π) was 0.0047 (SD = 0.0034), the mean value of Watterson’s estimator (θ) was 0.0067 (SD = 0.0046), and the average haplotype diversity was 0.86 (SD = 0.16). Overall, <em>PaAP2L3</em> was the least variable gene, with only five polymorphic sites. <em>PaWS02746</em> and <em>PaWS02749</em> both had greater π and θ values than all other genes but similar values of haplotype diversity, whereas <em>PaZIP</em> had lower haplotype diversity with average values of nucleotide diversity. The mean Tajima’s D was −0.88 (SD = 0.59), with <em>PaPRR1</em> being the only gene with a positive value (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The only significant Tajima’s D, −1.81, was found in <em>PaMFT1</em>. Mean Tajima’s D over all genes was plotted for each population against latitude of origin, revealing a weak trend toward more positive values as one moves north. The pattern was clear when looking at the three populations with larger sample size, but for the populations with fewer individuals there was no such pattern (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Reducing the data set to include only candidate genes for bud set did not alter the outcome (data not shown).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/795/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Within population estimates of mean Tajima&#x2019;s D across eleven loci plotted against latitude of origin. Boxes denote estimates from the number of individuals sampled from the population. The mean across eleven loci within resampled populations SE-61, SE-64 and FI-67 is plotted with circles, triangles and crosses respectively (see legend)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-869417027" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Within population estimates of mean Tajima&#x2019;s D across eleven loci plotted against latitude of origin. Boxes denote estimates from the number of individuals sampled from the population. The mean across eleven loci within resampled populations SE-61, SE-64 and FI-67 is plotted with circles, triangles and crosses respectively (see legend).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/795/F3.medium.gif" width="308" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/5/795/F3.medium.gif" width="308" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/795/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/795/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462732" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Within population estimates of mean Tajima’s D across eleven loci plotted against latitude of origin. Boxes denote estimates from the number of individuals sampled from the population. The mean across eleven loci within resampled populations SE-61, SE-64 and FI-67 is plotted with circles, triangles and crosses respectively (see legend).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Despite an added amount of sequence data and reduced sample size, the mean π in populations FUL, HOG, and SOD was very similar to the mean obtained over the total data set and single gene estimates varied only slightly between populations (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In contrast, values of Tajima’s D obtained in the three populations differed from estimates obtained from the total data set and were also quite variable among populations. Several genes had a positive Tajima’s D, with SOD being the population with the most positive values.</p></div><div id="sec-9" class="subsection"><h3>Effect of sample size</h3><p id="p-28">As expected, resampling showed that estimates of π were fairly robust to variation in sample size. In contrast, Tajima’s D showed large fluctuations across the resampled data sets. In some instances, the mean of the 100 resampled estimates was close to the estimate based on all individuals in the same population, but in many instances the estimated Tajima’s D value was even reversed, shifting from negative to positive or vice versa. The largest difference in values of Tajima’s D was for <em>PaZIP</em> in HOG, where resampled population values ranged from −1.74 to 1.43 (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-10" class="subsection"><h3>Levels of LD and recombination</h3><p id="p-29">The informative sites of all 11 genes generated 2378 pairwise comparisons, 272 (11.4%) of which remained significant after Fisher’s exact test with Bonferroni’s correction. The average <em>r<sup>2</sup></em> over all pairwise comparisons was 0.11 (SD = 0.05) and LD decreased rapidly with distance between pairs of SNPs, reaching an <em>r<sup>2</sup></em> value of less than 0.2 within 80 bp (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The decrease was slower when D′ was considered (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/795/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the squared correlation of allele frequencies (r2) vs. distance in base pairs across 11 loci for different subsets of populations. (Top left) all ten populations n=97, (top right) SE-61 n=20, (bottom left) SE-64 n=16, and (bottom right) FI-67 n=16." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-869417027" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the squared correlation of allele frequencies (r&lt;sup&gt;2&lt;/sup&gt;) &lt;em&gt;vs.&lt;/em&gt; distance in base pairs across 11 loci for different subsets of populations. (Top left) all ten populations n=97, (top right) SE-61 n=20, (bottom left) SE-64 n=16, and (bottom right) FI-67 n=16.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/795/F4.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/5/795/F4.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/795/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/795/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462786" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Plot of the squared correlation of allele frequencies (r<sup>2</sup>) <em>vs.</em> distance in base pairs across 11 loci for different subsets of populations. (Top left) all ten populations n=97, (top right) SE-61 n=20, (bottom left) SE-64 n=16, and (bottom right) FI-67 n=16.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31"><em>PaAP2L3</em>, with only six pairwise comparisons between informative sites, was excluded from the per gene estimates of <em>r<sup>2</sup></em>. The remaining 10 genes, with 45 pairwise comparisons or more, had mean <em>r<sup>2</sup></em> values from 0.04 in <em>PaCDF1</em> to 0.2 in <em>PaZIP</em>, with five genes having a mean <em>r<sup>2</sup></em> &gt; 0.11 (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The mean <em>r<sup>2</sup></em> value was 0.15 for the above-average group of genes and 0.08 for the below-average group, with <em>r<sup>2</sup></em> values being differently distributed in the two groups (Wilcoxon’s rank sum test: <em>P</em> &lt; 0.01). There was no significant correlation between π and mean <em>r<sup>2</sup></em> (Spearman’s rank correlation: r = 0.56, <em>P</em> = 0.07). The decay of LD with distance varied between genes, again with <em>PaCDF1</em> in the low extreme, where the fitted curve never reached an <em>r<sup>2</sup></em> value of 0.2, and <em>PaZIP</em> standing out as LD extended for 353 bp before declining to values less than 0.20. Roughly, the genes can also be divided into two groups based on their pattern of LD. The first consists of <em>PaCCA1</em>, <em>PaCDF1</em>, <em>PaCOL1</em> and <em>PaWS02746</em>, where most SNPs are in weak linkage and few SNPs in high linkage appear at distances greater than 1kb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS3.pdf">Figure S3</a>). The other group, including the rest of the genes, also displays a great number of SNPs in weak LD but, in addition, displays SNPs in complete LD at distances of several thousand base pairs and SNPs in high LD throughout the length of the genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462728/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462728/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Mean linkage disequilibrium and recombination rate parameters estimated per locus for the merged data set of 10 populations with a mean of 97 individuals</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">The recombination estimate ρ per site ranged from 0 in <em>PaZIP</em> to 0.0071 in <em>PaFTL1</em>, with a mean value for all loci of 0.0033 (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The likelihood curves indicate good estimates of ρ in all genes except <em>PaCOL1</em>, <em>PaFTL1</em>, <em>PaMFT1</em>, and <em>PaWS02746</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS4.pdf">Figure S4</a>). There was no significant correlation between ρ per site and mean <em>r<sup>2</sup></em> (Spearman’s rank correlation: r = −0.3, <em>P</em> = 0.34). The estimate of ρ over θ was &gt;1 for all loci except <em>PaPRR1</em> and <em>PaZIP</em> (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-36">Mean <em>r<sup>2</sup></em> was elevated for all genes in all within population estimates compared to the merged data set (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). A multilocus estimate of the decay of LD within populations was strikingly different from the merged data set, with LD extending from 705 bp in SE-61 to 4580 bp in FI-67 before falling under <em>r<sup>2</sup></em> values of 0.2 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B</a>−D). The difference was not as striking when the decay of D’ was considered (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS2.pdf">Figure S2</a>). The population recombination rates varied greatly, but likelihood curves for the smaller sample sizes were not showing distinct peaks at estimated values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1/FigureS4.pdf">Figure S4</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462707/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462707/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Mean linkage disequilibrium and recombination rate parameters estimated per locus for each of the three populations SE-61, SE-64, and FI-67</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><div id="sec-12" class="subsection"><h3>Population structure</h3><p id="p-40">Despite of the fact that the populations covered up to 9º of latitude (from 58° 36′N to 67° 41′N) we were unable to detect a clear genetic structure among the populations sampled within the Scandinavian distribution of <em>P. abies</em>. This confirms the lack of structure generally observed in central Scandinavia in Norway spruce, but is slightly different from the results of <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (2012)</a> where populations above 67°N tended to form a separate cluster. The lack of population structure here could simply reflect a lower power in the present study than in <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (2012)</a> and the presence of a smaller number of populations at high latitudes in this study.</p></div><div id="sec-13" class="subsection"><h3>Nucleotide diversity and Tajima’s D</h3><p id="p-41">The overall level of nucleotide diversity was for the most part comparable with previous results in <em>P. abies</em> (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2010</a>), with greater values for <em>PaWS02746</em> and <em>PaWS02749</em> and its estimate was not sensitive to sampling. The mean Tajima’s D was negative, again consistent with previously reported results and an expanding population size (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2010</a>). Because we did not detect any meaningful population structure, the overall estimate of Tajima’s D is unlikely to have been strongly affected by pooling of samples from different populations (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Städler <em>et al.</em> 2009</a>) and given the size of the total sample the estimate of Tajima’s D should be close to its true value (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Marroni <em>et al.</em> 2011</a>). In line with the results of <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Marroni <em>et al.</em> (2011)</a> and in contrast to estimates of π, estimates of Tajima’s D value and its significance were quite sensitive to the reduction in sample size and, apart from the general trend of an increase in positive values associated with smaller sample sizes, resampling displayed a wide range of values depending on the randomly sampled representatives of the population. The trend of more negative values of Tajima’s D with increasing number of individuals is contrary to neutral expectations under a fixed number of segregating sites, where adding individuals should cause Tajima’s D to become more positive. However, as Norway spruce reveals signs of population expansion throughout the distribution range (<a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2010</a>), increasing the sample size in this species, in effect, means introducing new mutations occurring in low frequencies or even as singletons, leading to more negative values of Tajima’s D and a more accurate reflection of the demographic history of Norway spruce. It seems, therefore, that estimates of Tajima’s D based on small samples (&lt;50 according to <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> in <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Marroni <em>et al.</em> 2011</a>) should be considered with caution, especially in populations that putatively depart from the standard neutral model and harbor a large number of rare variants. A striking example of the increase of the number of segregating sites with increased sample size is provided by the recent study of <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nelson <em>et al.</em> (2012)</a> in humans, where θ<sub>W</sub>, which is based on S, increased fivefold when sample size went from 500 to 10,000 whereas θ<sub>π</sub> remained stable. Finally, as we would expect sampling to affect Tajima’s D more strongly when populations are structured and depart from the standard neutral model, resampling can in itself be a new source of information on past demographics that warrants further investigation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cutter <em>et al.</em> 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">St. Onge <em>et al.</em> 2012</a>).</p><p id="p-42">We were unable to detect a clear latitudinal gradient of Tajima’s D values among the 10 populations, but as resampling clearly showed, a sample size of eight individuals was not sufficient for this data set to yield robust estimates of Tajima’s D within populations. The three populations with a larger sample size did reflect a pattern of more positive values of Tajima’s D with increased latitude similar to the pattern observed in <em>P. sitchensis</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Holliday <em>et al.</em> 2010</a>), but with only three populations of sufficient sample size it is difficult to accurately determine if the pattern is true. Obviously, to assess reliably the presence of a latitudinal cline in Tajima’s D, a larger number of individuals per population as well as a larger number of loci would be required.</p></div><div id="sec-14" class="subsection"><h3>Level of LD and recombination</h3><p id="p-43">The within gene level of LD in this study was generally low. However, <em>P. abies</em> does not seem to be an exception to the pattern of LD varying between genes shown in other conifers. Estimating the decay of LD with distance using the pooled <em>r<sup>2</sup></em> estimates for all genes failed to reflect the heterogeneous levels of LD among the genes, evident in the variable estimates of linkage and recombination across genes. <em>PaZIP</em> and <em>PaPRR1</em> in particular displayed high mean <em>r<sup>2</sup></em> values. Plotting the squared correlation of allele frequencies against distance in <em>PaZIP</em> reveals that SNPs in complete LD extend throughout the length of the gene, a total of 4 kb (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), resembling the pattern found for allozyme coding loci in pine (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Pyhäjärvi <em>et al.</em> 2011</a>). More sequence data may reveal a consistent pattern of LD along the whole length of <em>PaZIP</em>. In <em>Pinus pinaster</em>, a gene coding for transcription factor from the HD-ZIP family, <em>HD-ZIPIII</em>, showed high levels of LD across more than 3.2 kb (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lepoittevin <em>et al.</em> 2012</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/5/795/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the squared correlation of allele frequencies (r2) vs. distance in base pairs in the gene PaZIP using all populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-869417027" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the squared correlation of allele frequencies (r&lt;sup&gt;2&lt;/sup&gt;) &lt;em&gt;vs.&lt;/em&gt; distance in base pairs in the gene &lt;em&gt;PaZIP&lt;/em&gt; using all populations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/5/795/F5.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/5/795/F5.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/5/795/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/5/795/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462676" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Plot of the squared correlation of allele frequencies (r<sup>2</sup>) <em>vs.</em> distance in base pairs in the gene <em>PaZIP</em> using all populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">For <em>PaZIP</em>, the high mean <em>r<sup>2</sup></em> value of 0.2 was matched by a population recombination rate of 0, but contrary to expectations this was not a general occurrence because we could not find a significant correlation between mean <em>r<sup>2</sup></em> and ρ per site across genes. The flat likelihood curves for four of the genes, indicating some uncertainty in the estimate of ρ, may explain the lack of correlation. Less surprising was the lack of correlation between mean <em>r<sup>2</sup></em> and π, as previous studies have obtained the same result in both <em>P. glauca</em> and <em>P. abies</em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Pavy <em>et al.</em> 2012</a>). The ratio of ρ to θ revealed that recombination was the major force behind the variation in all loci except <em>PaPRR1</em> and <em>PaZIP</em>, which is consistent with their high mean <em>r<sup>2</sup></em> values and relatively slow decay of LD, maintaining mean <em>r<sup>2</sup></em> values &gt; 0.2 over more than 100 bp.</p><p id="p-46">Despite the lack of population structure between the Scandinavian populations, LD and recombination were quite variable between the three populations more intensely sampled. In part, the inflated values of LD can be explained by the reduced sample size in within population estimates because <em>r<sup>2</sup></em> depends on allele frequencies and an absence of rare alleles will tend to inflate LD estimates slightly (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hedrick and Kumar 2001</a>). However, it seems unlikely that the increased distances of LD decay evident in individual populations should only be a result of a reduction in sample size. In a study on the effect of minor allele frequency thresholds on estimates of LD in <em>Populus nigra</em> the decay of LD estimated with <em>r<sup>2</sup></em> increased by a distance of ~400 bp when increasing the threshold from 0.05 to 0.1 (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Marroni <em>et al.</em> 2011</a>).</p><p id="p-47">Because we only used parsimony informative sites in the calculation of <em>r<sup>2</sup></em>, the minor allele frequency would increase from 0.02 in the merged data set with a mean sample size of 97 individuals to 0.125 in the mean sample size of 16 individuals in FI-67. This does not seem sufficient to explain the increase in distance of LD decay from 46 bp in the merged data set to 4580 bp in FI-67. Perhaps a more plausible explanation to the difference in the extent of LD between the pooled sample and the individual populations is that, despite the apparent lack of population structure, the three subpopulations have had different histories. Boreal forest tree species, and Norway spruce in particular, have had complex histories and have undergone periodic fragmentation and admixture due to glaciation cycles in some parts of their current range. Simulations have shown that LD created through postglaciation admixture can be elevated and will be maintained in species with long generation times (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tachida 2012</a>) and the difference in LD levels between populations may be the result of subtle differences in past fragmentation-admixture events. Partial admixture, or population structure, could also be an explanation as <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tachida (1994)</a> in a finite island model showed that the initial rate of decay of LD is increased by finiteness of the population but that the ultimate rate of decay is decreased. Hence, LD created in the past may persist longer in smaller subdivided populations (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tachida 1994</a>). If the populations have a different past then we would expect different levels of LD. We note that there is a cline in LD with the lowest value found in SE-61 and the highest in FI-67 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, B</a>−D and <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Although we did not detect a clear differentiation between FI-67 and the other populations in the present study, other studies (X.-R. Wang, personal communication) (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2012</a>) have shown that populations above 65°N are genetically different from more southerly Scandinavian populations. The exact cause of this genetic difference is not yet well understood. It could reflect differences in origins and/or differences in selection pressure and reproduction. In any case the extensive LD observed in FI-67 could reflect this difference. A demographic explanation seems also supported by the fact that differences in the decay of D′ was much less striking than for the decay of <em>r<sup>2</sup></em> with distances. As was noted previously D′ reflects recombination between the two loci, whereas <em>r</em><sup>2</sup> reflects more the gene genealogies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McVean 2001</a>).</p><p id="p-48">To successfully design an association mapping experiment one needs to have detailed knowledge about many basic population genetic parameters of the species of interest. These parameters are for the majority of species still missing. In Norway spruce, despite the relatively low levels of population structure it is evident that there are differences in both pattern of LD and allele frequencies between populations, calling for caution in estimating parameters on species-wide sampling and highlighting the importance of larger sample size within populations to obtain meaningful results. As the pattern of LD largely determines the power of an association study and the existence of population structure can yield an excess of false positives there are strong incentives to obtain better estimates of these fundamental parameters to be able to optimize the design of association experiments. Together with data from other conifers the data put forward here indicate a more heterogeneous pattern in LD than earlier studies have suggested. Hence the classic view of a lack of LD within genes in conifers needs to be reconsidered. Even though gene space is going to remain specifically interesting for association mapping, especially in species with large genomes, with the arrival of full genome data from a single individual we should now focus our efforts on estimating LD and factors that influence LD at full genome level.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We thank Kerstin Jeppsson for help in the laboratory and the staff at the Swedish Forest Research Institute (Skogforsk) and at the Finnish Forest Research Institute (METLA) for seed samples. The research was supported by the European Community’s Sixth Framework Program under the Network of Excellence Evoltree and the Seventh Framework Program (FP7/2007-2013) under the grant agreement no. 211868 (Project Noveltree), BiodivERsA projects LinkTree and TipTree, and by the Swedish Research Council FORMAS grant dnr 217-2007-433 to M.L. and by a Nilsson-Ehle foundation to H.L.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005462/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-50"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-12-20"><span class="received-label">Received </span>December 20, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-03-05"><span class="accepted-label">Accepted </span>March 5, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Larsson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.795.1" data-doi="10.1534/g3.111.000539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogelqvist</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnberg-Wästljung</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Polymorphism and eivergence in two willow species, Salix viminalis L. and Salix schwerinii E. Wolf</span>. <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">1</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DBerlin%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D387%26rft.epage%253D400%26rft.atitle%253DPolymorphism%2Band%2BDivergence%2Bin%2BTwo%2BWillow%2BSpecies%252C%2BSalix%2Bviminalis%2BL.%2Band%2BSalix%2Bschwerinii%2BE.%2BWolf%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000539%26rft_id%253Dinfo%253Apmid%252F22384349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvNS8zODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.2" data-doi="10.1016/j.quascirev.2009.04.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Binney</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwat</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The distribution of late-Quaternary woody taxa in northern Eurasia: evidence from a new macrofossil database</span>. <abbr class="cit-jnl-abbrev">Quat. Sci. Rev.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2445</span>–<span class="cit-lpage">2464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat.%2BSci.%2BRev.%26rft.volume%253D28%26rft.spage%253D2445%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2009.04.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2009.04.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000271335400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.3" data-doi="10.1073/pnas.0404231101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuntz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in loblolly pine</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15255</span>–<span class="cit-lpage">15260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404231101%26rft_id%253Dinfo%253Apmid%252F15477602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQyLzE1MjU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.4" data-doi="10.1038/hdy.2009.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New insights on the speciation history and nucleotide diversity of three boreal spruce species and a tertiary relict</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D104%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D14%26rft.atitle%253DNew%2Binsights%2Bon%2Bthe%2Bspeciation%2Bhistory%2Band%2Bnucleotide%2Bdiversity%2Bof%2Bthree%2Bboreal%2Bspruce%2Bspecies%2Band%2Ba%2BTertiary%2Brelict.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.88%26rft_id%253Dinfo%253Apmid%252F19639012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19639012&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272794600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.5" data-doi="10.1534/genetics.112.140749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Disentangling the roles of history and local selection in shaping clinal variation of allele frequencies and gene expression in Norway spruce (<em>Picea abies</em>)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140749%26rft_id%253Dinfo%253Apmid%252F22542968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvODY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.6" data-doi="10.1111/j.1365-294X.2012.05475.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Influence of finite-sites mutation, population subdivision and sampling schemes on patterns of nucleotide polymorphism for species with molecular hyperdiversity</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCutter%26rft.auinit1%253DA.%2BD.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D1345%26rft.epage%253D1359%26rft.atitle%253DInfluence%2Bof%2Bfinite-sites%2Bmutation%252C%2Bpopulation%2Bsubdivision%2Band%2Bsampling%2Bschemes%2Bon%2Bpatterns%2Bof%2Bnucleotide%2Bpolymorphism%2Bfor%2Bspecies%2Bwith%2Bmolecular%2Bhyperdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05475.x%26rft_id%253Dinfo%253Apmid%252F22320847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05475.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22320847&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301116000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.7" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.8" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft_id%253Dinfo%253Apmid%252F9521922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9521922&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.9" data-doi="10.1101/gr.8.3.175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendl</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Base-calling of automated sequencer traces using phred. I. Accuracy assessment</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DEwing%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D185%26rft.atitle%253DBase-Calling%2Bof%2BAutomated%2BSequencer%2BTraces%2BUsing%2BPhred.%2B%2BI.%2BAccuracy%2BAssessment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.175%26rft_id%253Dinfo%253Apmid%252F9521921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.10" data-doi="10.1111/j.1755-0998.2010.02847.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lischer</span>  <span class="cit-name-given-names">H. E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D10%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02847.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02847.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.11" data-doi="10.1038/nrg1970"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination: an underappreciated factor in the evolution of plant genomes</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGaut%26rft.auinit1%253DB.%2BS.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D84%26rft.atitle%253DRecombination%253A%2Ban%2Bunderappreciated%2Bfactor%2Bin%2Bthe%2Bevolution%2Bof%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1970%26rft_id%253Dinfo%253Apmid%252F17173059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1970&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17173059&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242934300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.12" data-doi="10.1534/genetics.105.047126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">DNA sequence variation and selection of tag single-nucleotide polymorphisms at candidate genes for dought-stress response in <em>Pinus taeda</em> L</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047126%26rft_id%253Dinfo%253Apmid%252F16387885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE5MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.13" data-doi="10.1101/gr.8.3.195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Consed: a graphical tool for sequence finishing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DGordon%26rft.auinit1%253DD.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D202%26rft.atitle%253DConsed%253A%2BA%2BGraphical%2BTool%2Bfor%2BSequence%2BFinishing%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.195%26rft_id%253Dinfo%253Apmid%252F9521923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.14" data-doi="10.1360/yc-007-1023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GSDS: a gene structure display server</span>. <abbr class="cit-jnl-abbrev">Yi Chuan</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1023</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYi%2Bchuan%2B%253D%2BHereditas%2B%252F%2BZhongguo%2Byi%2Bchuan%2Bxue%2Bhui%2Bbian%2Bji%26rft.stitle%253DYi%2BChuan%26rft.aulast%253DGuo%26rft.auinit1%253DA.%2BY.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D1023%26rft.epage%253D1026%26rft.atitle%253D%255BGSDS%253A%2Ba%2Bgene%2Bstructure%2Bdisplay%2Bserver%255D%26rft_id%253Dinfo%253Adoi%252F10.1360%252Fyc-007-1023%26rft_id%253Dinfo%253Apmid%252F17681935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1360/yc-007-1023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17681935&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.15" data-doi="10.1038/sj.ejhg.5200735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mutation and linkage disequilibrium in human mtDNA</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DHedrick%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D969%26rft.epage%253D972%26rft.atitle%253DMutation%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bhuman%2BmtDNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.ejhg.5200735%26rft_id%253Dinfo%253Apmid%252F11840186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.ejhg.5200735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11840186&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173907200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.16" data-doi="10.1534/genetics.106.065102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heuertz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Paoli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Multilocus patterns of nucleotide diversity, linkage disequilibrium and demographic history of Norway spruce [Picea abies (L.) Karst]</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">2095</span>–<span class="cit-lpage">2105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065102%26rft_id%253Dinfo%253Apmid%252F17057229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzIwOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.17" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.18" data-doi="10.1016/0040-5809(88)90004-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Variances and covariances of squared linkage disequilibria in finite populations</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D78%26rft.atitle%253DVariances%2Band%2Bcovariances%2Bof%2Bsquared%2Blinkage%2Bdisequilibria%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252888%252990004-4%26rft_id%253Dinfo%253Apmid%252F3376052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(88)90004-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3376052&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988M750300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.19" data-doi="10.1111/j.1365-294X.2010.04767.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Postglacial history of a widespread conifer produces inverse clines in selective neutrality tests</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3857</span>–<span class="cit-lpage">3864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHolliday%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D3857%26rft.epage%253D3864%26rft.atitle%253DPostglacial%2Bhistory%2Bof%2Ba%2Bwidespread%2Bconifer%2Bproduces%2Binverse%2Bclines%2Bin%2Bselective%2Bneutrality%2Btests.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04767.x%26rft_id%253Dinfo%253Apmid%252F20738783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04767.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738783&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281797400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.20" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTQvMTgwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS83OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.21" data-doi="10.1186/1471-2148-10-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo-Correa</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdú</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Martìnez</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The contribution of recombination to heterozygosity differs among plant evolutionary lineages and life-forms</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DJaramillo-Correa%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D10%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DThe%2Bcontribution%2Bof%2Brecombination%2Bto%2Bheterozygosity%2Bdiffers%2Bamong%2Bplant%2Bevolutionary%2Blineages%2Band%2Blife-forms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-22%26rft_id%253Dinfo%253Apmid%252F20100325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20100325&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-thesis" id="cit-3.5.795.22"><div class="cit-metadata unstructured">Källman, T., 2009 Adaptive evolution and demographic history of Norway spruce (<em>Picea abies</em>). Ph.D. Thesis, Uppsala University.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.23" data-doi="10.1104/pp.111.176206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlgren</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundström</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of the PEBP gene family in plants: functional diversification in seed plant evolution</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1967</span>–<span class="cit-lpage">1977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.176206%26rft_id%253Dinfo%253Apmid%252F21642442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzQvMTk2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS83OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.24" data-doi="10.1534/genetics.105.044420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in cold-hardiness- and wood quality-related candidate genes in Douglas fir</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">2029</span>–<span class="cit-lpage">2041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044420%26rft_id%253Dinfo%253Apmid%252F16157674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzIwMjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.25" data-doi="10.1007/s11295-011-0426-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lepoittevin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvengt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garnier-Gere</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association mapping for growth, straightness and wood chemistry traits in the <em>Pinus pinaster</em> Aquitaine breeding population</span>. <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0426-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0426-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The interaction of selection and linkage. I. General considerations; heterotic models</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D67%26rft.atitle%253DTHE%2BINTERACTION%2BOF%2BSELECTION%2BAND%2BLINKAGE.%2BI.%2BGENERAL%2BCONSIDERATIONS%253B%2BHETEROTIC%2BMODELS%26rft_id%253Dinfo%253Apmid%252F17248194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI0OS8xLzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.27" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: A software for comprehensive analysis of DNA polymorphism data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS83OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.28" data-doi="10.1007/s11295-011-0391-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marroni</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinosio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogolari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felice</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in Populus nigra cinnamyl alcohol dehydrogenase (CAD4) gene</span>. <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D1011%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0391-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0391-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.29" data-doi="10.1046/j.1365-2540.2001.00965.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">What do patterns of genetic variability reveal about mitochondrial recombination?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%26rft.volume%253D87%26rft.issue%253DPt%2B6%26rft.spage%253D613%26rft.epage%253D620%26rft.atitle%253DWhat%2Bdo%2Bpatterns%2Bof%2Bgenetic%2Bvariability%2Breveal%2Babout%2Bmitochondrial%2Brecombination%253F%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2001.00965.x%26rft_id%253Dinfo%253Apmid%252F11903556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2001.00965.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11903556&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173587200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fearnhead</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A coalescent-based method for detecting and estimating recombination from gene sequences</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1231%26rft.epage%253D1241%26rft.atitle%253DA%2BCoalescent-Based%2BMethod%2Bfor%2BDetecting%2Band%2BEstimating%2BRecombination%2BFrom%2BGene%2BSequences%26rft_id%253Dinfo%253Apmid%252F11901136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyMzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.31" data-doi="10.1534/genetics.111.136697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moritsuka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hisataka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uchiyama</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extended linkage disequilibrium in noncoding regions in a conifer, <em>Cryptomeria japonica</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1145</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136697%26rft_id%253Dinfo%253Apmid%252F22209904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC8zLzExNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.32" data-doi="10.1007/s00239-010-9335-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Namroud</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillet-Claude</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular evolution of regulatory genes in spruces from different species and continents: heterogeneous patterns of linkage disequilibrium and selection but correlated recent demographic changes</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DNamroud%26rft.auinit1%253DM.%2BC.%26rft.volume%253D70%26rft.issue%253D4%26rft.spage%253D371%26rft.epage%253D386%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bregulatory%2Bgenes%2Bin%2Bspruces%2Bfrom%2Bdifferent%2Bspecies%2Band%2Bcontinents%253A%2Bheterogeneous%2Bpatterns%2Bof%2Blinkage%2Bdisequilibrium%2Band%2Bselection%2Bbut%2Bcorrelated%2Brecent%2Bdemographic%2Bchanges.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9335-1%26rft_id%253Dinfo%253Apmid%252F20354847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9335-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20354847&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277938300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.33" data-doi="10.1016/j.tplants.2004.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Association genetics of complex traits in conifers</span>. <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D325%26rft.epage%253D330%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bcomplex%2Btraits%2Bin%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2004.05.006%26rft_id%253Dinfo%253Apmid%252F15231277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2004.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15231277&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223010600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.34" data-doi="10.1126/science.1217876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217876%26rft_id%253Dinfo%253Apmid%252F22604722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5MC8xMDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.35" data-doi="10.1038/hdy.2011.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namroud</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The heterogeneous levels of linkage disequilibrium in white spruce genes and comparative analysis with other conifers</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DPavy%26rft.auinit1%253DN.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D273%26rft.epage%253D284%26rft.atitle%253DThe%2Bheterogeneous%2Blevels%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bwhite%2Bspruce%2Bgenes%2Band%2Bcomparative%2Banalysis%2Bwith%2Bother%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.72%26rft_id%253Dinfo%253Apmid%252F21897435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.72&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21897435&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300596100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.36" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.37" data-doi="10.1534/genetics.107.077099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gil</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knürr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikkonen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachowiak</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Demographic history has influenced nucleotide diversity in European <em>Pinus sylvestris</em> populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1713</span>–<span class="cit-lpage">1724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPyhajarvi%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D1713%26rft.epage%253D1724%26rft.atitle%253DDemographic%2BHistory%2BHas%2BInfluenced%2BNucleotide%2BDiversity%2Bin%2BEuropean%2BPinus%2Bsylvestris%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.077099%26rft_id%253Dinfo%253Apmid%252F18039881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE3MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.38" data-doi="10.1007/s11295-010-0340-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kujala</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revisiting protein heterozygosity in plants - nucleotide diversity in allozyme coding genes of conifer Pinus sylvestris</span>. <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0340-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0340-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-3.5.795.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2012</span>  <span class="cit-article-title">R: A Language and Environment for Statistical Computing.</span> <span class="cit-comment">Available at:</span> <a href="http://www.R-project.org">http://www.R-project.org</a>. <span class="cit-comment">Accessed: March 18, 2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.40" data-doi="10.1534/genetics.108.094904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Städler</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaffelhuber</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The impact of sampling schemes on the site frequency spectrum in nonequilibrium subdivided populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094904%26rft_id%253Dinfo%253Apmid%252F19237689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMjA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-other" id="cit-3.5.795.41" data-doi="10.1534/g3.112.002410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Onge</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmé</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Impact of sampling schemes on demographic inference: an empirical study in two species with different mating systems and demographic histories</span>. <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DSt.%2BOnge%26rft.auinit1%253DK.%2BR.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D814%26rft.atitle%253DImpact%2Bof%2BSampling%2BSchemes%2Bon%2BDemographic%2BInference%253A%2BAn%2BEmpirical%2BStudy%2Bin%2BTwo%2BSpecies%2Bwith%2BDifferent%2BMating%2BSystems%2Band%2BDemographic%2BHistories%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002410%26rft_id%253Dinfo%253Apmid%252F22870403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNy84MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tachida</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Decay of linkage disequilibrium in a finite island model</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DTachida%26rft.auinit1%253DH.%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D144%26rft.atitle%253DDecay%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Ba%2Bfinite%2Bisland%2Bmodel.%26rft_id%253Dinfo%253Apmid%252F7813904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7813904&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.43" data-doi="10.1266/ggs.87.125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tachida</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Linkage disequilibrium in a population undergoing periodic fragmentation and admixture</span>. <abbr class="cit-jnl-abbrev">Genes Genet. Syst.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2Bgenetic%2Bsystems%26rft.stitle%253DGenes%2BGenet%2BSyst%26rft.aulast%253DTachida%26rft.auinit1%253DH.%26rft.volume%253D87%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D135%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Ba%2Bpopulation%2Bundergoing%2Bperiodic%2Bfragmentation%2Band%2Badmixture.%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fggs.87.125%26rft_id%253Dinfo%253Apmid%252F22820386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/ggs.87.125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820386&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA Polymorphism</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.45" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">libsequence: a C++ class library for evolutionary genetic analysis</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTcvMjMyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNS83OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.46" data-doi="10.1038/hdy.2009.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tollefsrud</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstebo</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skroppa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Combined analysis of nuclear and mitochondrial markers provide new insight into the genetic structure of North European (<em>Picea abies</em>)</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DTollefsrud%26rft.auinit1%253DM.%2BM.%26rft.volume%253D102%26rft.issue%253D6%26rft.spage%253D549%26rft.epage%253D562%26rft.atitle%253DCombined%2Banalysis%2Bof%2Bnuclear%2Band%2Bmitochondrial%2Bmarkers%2Bprovide%2Bnew%2Binsight%2Binto%2Bthe%2Bgenetic%2Bstructure%2Bof%2BNorth%2BEuropean%2BPicea%2Babies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.16%26rft_id%253Dinfo%253Apmid%252F19259114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19259114&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F5%2F795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266199100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.47" data-doi="10.1111/j.1365-2699.2010.02448.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Väliranta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaakinen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kultti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salonen</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scattered late-glacial and early Holocene tree populations as dispersal nuclei for forest development in north-eastern European Russia</span>. <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">922</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D38%26rft.spage%253D922%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2699.2010.02448.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2699.2010.02448.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000289630200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.48" data-doi="10.1073/pnas.0507325102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisse</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">18508</span>–<span class="cit-lpage">18513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0507325102%26rft_id%253Dinfo%253Apmid%252F16352722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzUxLzE4NTA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy81Lzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.5.795.49" data-doi="10.1534/genetics.106.061432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Statistical tests for detecting positive selection by utilizing high-frequency variants</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061432%26rft_id%253Dinfo%253Apmid%252F16951063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE0MzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzUvNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/5/795.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/5/783" title="Genetic Structure and Linkage Disequilibrium in a Diverse, Representative Collection of the C4 Model Plant, Sorghum bicolor" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/5/807" title="Inferring Gene Function and Network Organization in Drosophila Signaling by Combined Analysis of Pleiotropy and Epistasis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/5"><span>Volume 3 Issue 5, May 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/5.cover-source.jpg?itok=NJpwBPbV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/5/795.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sUtZzlKoC0gpvDkNY7SzU-Yf1QCLbGHfxQiL4yMH9Xg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/5/795" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/5/795" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/5/795" class="active" data-icon-position="" data-hide-link-title="0">Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (Picea abies)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459469" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Re-0vWs_DxfOeErHOTx1tsw2_0h2k0NeP018t2J90Yk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/795.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/5/795" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZzI4yrNfaaKczwiwS0_ESSXchl3gs_Kn8Ze26H1iBsM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/5/795" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/5/795.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459469" id="node459469--2" data-pisa="ggg;3/5/795" data-pisa-master="ggg;g3.112.005462" data-apath="/ggg/3/5/795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (<em>Picea abies</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hanna Larsson</span>, <span class="highwire-citation-author" data-delta="1">Thomas Källman</span>, <span class="highwire-citation-author" data-delta="2">Niclas Gyllenstrand</span> and <span class="highwire-citation-author" data-delta="3">Martin Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">795-806;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005462 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459469--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hanna Larsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanna%2BLarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Larsson%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHanna%2BLarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Hanna.Larsson@ebc.uu.se" class="" data-icon-position="" data-hide-link-title="0">Hanna.Larsson@ebc.uu.se</a>
<a href="mailto:larsson.h77@gmail.com" class="" data-icon-position="" data-hide-link-title="0">larsson.h77@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Källman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BK%C3%A4llman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4llman%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BK%25C3%25A4llman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Niclas Gyllenstrand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology and Forest Genetics, Swedish University of Agricultural Sciences, Uppsala, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niclas%2BGyllenstrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gyllenstrand%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANiclas%2BGyllenstrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/5/795" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/5/795&amp;title=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/5/795&amp;title=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/5/795&amp;title=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/5/795&amp;text=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/5/795&amp;title=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/5/795&amp;t=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/5/795&amp;title=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/5/795&amp;title=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459469/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459469" id="node459469--3" data-pisa="ggg;3/5/795" data-pisa-master="ggg;g3.112.005462" data-apath="/ggg/3/5/795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distribution of Long-Range Linkage Disequilibrium and Tajima’s D Values in Scandinavian Populations of Norway Spruce (<em>Picea abies</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hanna Larsson</span>, <span class="highwire-citation-author" data-delta="1">Thomas Källman</span>, <span class="highwire-citation-author" data-delta="2">Niclas Gyllenstrand</span> and <span class="highwire-citation-author" data-delta="3">Martin Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">795-806;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005462 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459469--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hanna Larsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanna%2BLarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Larsson%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHanna%2BLarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Hanna.Larsson@ebc.uu.se" class="" data-icon-position="" data-hide-link-title="0">Hanna.Larsson@ebc.uu.se</a>
<a href="mailto:larsson.h77@gmail.com" class="" data-icon-position="" data-hide-link-title="0">larsson.h77@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Källman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BK%C3%A4llman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4llman%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BK%25C3%25A4llman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Niclas Gyllenstrand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology and Forest Genetics, Swedish University of Agricultural Sciences, Uppsala, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niclas%2BGyllenstrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gyllenstrand%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANiclas%2BGyllenstrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, EBC, Uppsala University, 752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=795&amp;rft.epage=806&amp;rft.atitle=Distribution%20of%20Long-Range%20Linkage%20Disequilibrium%20and%20Tajima%E2%80%99s%20D%20Values%20in%20Scandinavian%20Populations%20of%20Norway%20Spruce%20%28Picea%20abies%29&amp;rft.volume=3&amp;rft.issue=5&amp;rft.date=2013-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Larsson%2C+Hanna&amp;rft.au=K%C3%A4llman%2C+Thomas&amp;rft.au=Gyllenstrand%2C+Niclas&amp;rft.au=Lascoux%2C+Martin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459469/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459469/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459469/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459469/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459469/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459469/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459469/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459469/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459469/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459469/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459469/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459469/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/5/795" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/5/795.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/5/795.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/5/795.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/5/795.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
