<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/6/1525" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/6/1525.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/6/1525.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/6/1525.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/6/1525/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/6/1525.atom" />
<meta name="HW.pisa" content="ggg;6/6/1525" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)" />
<meta name="DC.Identifier" content="10.1534/g3.116.028779" />
<meta name="DC.Date" content="2016-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Browning et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We estimated local ancestry on the autosomes and X chromosome in a large US-based study of 12,793 Hispanic/Latino individuals using the RFMix method, and we compared different reference panels and approaches to local ancestry estimation on the X chromosome by means of Mendelian inconsistency rates as a proxy for accuracy. We developed a novel and straightforward approach to performing ancestry-specific PCA after finding artifactual behavior in the results from an existing approach. Using the ancestry-specific PCA, we found significant population structure within African, European, and Amerindian ancestries in the Hispanic/Latino individuals in our study. In the African ancestral component of the admixed individuals, individuals whose grandparents were from Central America clustered separately from individuals whose grandparents were from the Caribbean, and also from reference Yoruba and Mandenka West African individuals. In the European component, individuals whose grandparents were from Puerto Rico diverged partially from other background groups. In the Amerindian ancestral component, individuals clustered into multiple different groups depending on the grandparental country of origin. Therefore, local ancestry estimation provides further insight into the complex genetic structure of US Hispanic/Latino populations, which must be properly accounted for in genotype-phenotype association studies. It also provides a basis for admixture mapping and ancestry-specific allele frequency estimation, which are useful in the identification of risk factors for disease." />
<meta name="DC.Contributor" content="Sharon R. Browning" />
<meta name="DC.Contributor" content="Kelsey Grinde" />
<meta name="DC.Contributor" content="Anna Plantinga" />
<meta name="DC.Contributor" content="Stephanie M. Gogarten" />
<meta name="DC.Contributor" content="Adrienne M. Stilp" />
<meta name="DC.Contributor" content="Robert C. Kaplan" />
<meta name="DC.Contributor" content="M. Larissa Avilés-Santa" />
<meta name="DC.Contributor" content="Brian L. Browning" />
<meta name="DC.Contributor" content="Cathy C. Laurie" />
<meta name="article:published_time" content="2016-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We estimated local ancestry on the autosomes and X chromosome in a large US-based study of 12,793 Hispanic/Latino individuals using the RFMix method, and we compared different reference panels and approaches to local ancestry estimation on the X chromosome by means of Mendelian inconsistency rates as a proxy for accuracy. We developed a novel and straightforward approach to performing ancestry-specific PCA after finding artifactual behavior in the results from an existing approach. Using the ancestry-specific PCA, we found significant population structure within African, European, and Amerindian ancestries in the Hispanic/Latino individuals in our study. In the African ancestral component of the admixed individuals, individuals whose grandparents were from Central America clustered separately from individuals whose grandparents were from the Caribbean, and also from reference Yoruba and Mandenka West African individuals. In the European component, individuals whose grandparents were from Puerto Rico diverged partially from other background groups. In the Amerindian ancestral component, individuals clustered into multiple different groups depending on the grandparental country of origin. Therefore, local ancestry estimation provides further insight into the complex genetic structure of US Hispanic/Latino populations, which must be properly accounted for in genotype-phenotype association studies. It also provides a basis for admixture mapping and ancestry-specific allele frequency estimation, which are useful in the identification of risk factors for disease.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/06/01" />
<meta name="citation_mjid" content="ggg;6/6/1525" />
<meta name="citation_id" content="6/6/1525" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/6/1525" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/6/1525.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/6/1525.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/6/1525.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028779" />
<meta name="citation_pmid" content="27172203" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1525" />
<meta name="citation_lastpage" content="1534" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sharon R. Browning" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_email" content="sguy@uw.edu" />
<meta name="citation_author" content="Kelsey Grinde" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Anna Plantinga" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Stephanie M. Gogarten" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Adrienne M. Stilp" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Robert C. Kaplan" />
<meta name="citation_author_institution" content="Department of Epidemiology and Population Health, Albert Einstein College of Medicine, Bronx, New York 10461" />
<meta name="citation_author" content="M. Larissa Avilés-Santa" />
<meta name="citation_author_institution" content="Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Brian L. Browning" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_institution" content="Division of Medical Genetics, Department of Medicine, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Cathy C. Laurie" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_issue=9;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Baran;citation_author=B. Pasaniuc;citation_author=S. Sankararaman;citation_author=D. G. Torgerson;citation_author=C. Gignoux;citation_title=Fast and accurate inference of local ancestry in Latino populations.;citation_volume=28;citation_year=2012;citation_issue=10;citation_pmid=22495753;citation_doi=10.1093/bioinformatics/bts144" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=K. Bryc;citation_author=E. Y. Durand;citation_author=J. M. Macpherson;citation_author=D. Reich;citation_author=J. L. Mountain;citation_title=The genetic ancestry of African Americans, Latinos, and European Americans across the United States.;citation_pages=37-53;citation_volume=96;citation_year=2015;citation_issue=1;citation_pmid=25529636;citation_doi=10.1016/j.ajhg.2014.11.010" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=M. P. Conomos;citation_author=C. A. Laurie;citation_author=A. M. Stilp;citation_author=S. M. Gogarten;citation_author=C. P. McHugh;citation_title=Genetic Diversity and Association Studies in U.S. Hispanic/Latino Populations: Applications in the Hispanic Community Health Study/Study of Latinos.;citation_pages=165-184;citation_volume=98;citation_year=2016;citation_pmid=26748518;citation_doi=10.1016/j.ajhg.2015.12.001" />
<meta name="citation_reference" content="citation_title=Multidimensional Scaling;citation_author=T. F. Cox;citation_author=M. A. A. Cox;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=J. C. Gower;citation_title=Some distance properties of latent root and vector methods used in multivariate analysis;citation_pages=325-338;citation_volume=53;citation_year=1966;citation_issue=3-4;citation_doi=10.1093/biomet/53.3-4.325" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=A. Moreno-Estrada;citation_author=J. K. Byrnes;citation_author=M. Muzzio;citation_title=Reconstructing Native American migrations from whole-genome and whole-exome data.;citation_pages=e1004023;citation_volume=9;citation_year=2013;citation_issue=12;citation_pmid=24385924;citation_doi=10.1371/journal.pgen.1004023" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. J. Hoggart;citation_author=M. D. Shriver;citation_author=R. A. Kittles;citation_author=D. G. Clayton;citation_author=P. M. McKeigue;citation_title=Design and analysis of admixture mapping studies.;citation_pages=965-978;citation_volume=74;citation_year=2004;citation_issue=5;citation_pmid=15088268;citation_doi=10.1086/420855" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. R. Homburger;citation_author=A. Moreno-Estrada;citation_author=C. R. Gignoux;citation_author=D. Nelson;citation_author=E. Sanchez;citation_title=Genomic Insights into the Ancestry and Demographic History of South America.;citation_pages=e1005602;citation_volume=11;citation_year=2015;citation_issue=12;citation_pmid=26636962;citation_doi=10.1371/journal.pgen.1005602" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. A. Johnson;citation_author=M. A. Coram;citation_author=M. D. Shriver;citation_author=I. Romieu;citation_author=G. S. Barsh;citation_title=Ancestral components of admixed genomes in a Mexican cohort.;citation_pages=e1002410-e1002410;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22194699;citation_doi=10.1371/journal.pgen.1002410" />
<meta name="citation_reference" content="citation_journal_title=Annals of epidemiology;citation_journal_abbrev=Ann Epidemiol;citation_author=L. M. Lavange;citation_author=W. D. Kalsbeek;citation_author=P. D. Sorlie;citation_author=L. M. Aviles-Santa;citation_author=R. C. Kaplan;citation_title=Sample design and cohort selection in the Hispanic Community Health Study/Study of Latinos.;citation_pages=642-649;citation_volume=20;citation_year=2010;citation_issue=8;citation_pmid=20609344;citation_doi=10.1016/j.annepidem.2010.05.006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Z. Li;citation_author=D. M. Absher;citation_author=H. Tang;citation_author=A. M. Southwick;citation_author=A. M. Casto;citation_title=Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation;citation_pages=1100-1104;citation_volume=319;citation_year=2008;citation_issue=5866;citation_pmid=18292342;citation_doi=10.1126/science.1153717" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. K. Maples;citation_author=S. Gravel;citation_author=E. E. Kenny;citation_author=C. D. Bustamante;citation_title=RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference.;citation_pages=278-288;citation_volume=93;citation_year=2013;citation_issue=2;citation_pmid=23910464;citation_doi=10.1016/j.ajhg.2013.06.020" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Moreno-Estrada;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=J. L. McCauley;citation_author=J. K. Byrnes;citation_title=Reconstructing the population genetic history of the Caribbean.;citation_pages=e1003925;citation_volume=9;citation_year=2013;citation_issue=11;citation_pmid=24244192;citation_doi=10.1371/journal.pgen.1003925" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Moreno-Estrada;citation_author=C. R. Gignoux;citation_author=J. C. Fernandez-Lopez;citation_author=F. Zakharia;citation_author=M. Sikora;citation_title=The genetics of Mexico recapitulates Native American substructure and affects biomedical traits;citation_pages=1280-1285;citation_volume=344;citation_year=2014;citation_issue=6189;citation_pmid=24926019;citation_doi=10.1126/science.1251688" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Pasaniuc;citation_author=S. Sankararaman;citation_author=D. G. Torgerson;citation_author=C. Gignoux;citation_author=N. Zaitlen;citation_title=Analysis of Latino populations from GALA and MEC studies reveals genomic loci with biased local ancestry estimation.;citation_volume=29;citation_year=2013;citation_issue=11;citation_pmid=23572411;citation_doi=10.1093/bioinformatics/btt166" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Patterson;citation_author=N. Hattangadi;citation_author=B. Lane;citation_author=K. E. Lohmueller;citation_author=D. A. Hafler;citation_title=Methods for high-density admixture mapping of disease genes.;citation_pages=979-1000;citation_volume=74;citation_year=2004;citation_issue=5;citation_pmid=15088269;citation_doi=10.1086/420871" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. L. Price;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=K. C. Barnes;citation_author=N. Rafaels;citation_title=Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.;citation_pages=e1000519-e1000519;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543370;citation_doi=10.1371/journal.pgen.1000519" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Reich;citation_author=N. Patterson;citation_author=D. Campbell;citation_author=A. Tandon;citation_author=S. Mazieres;citation_title=Reconstructing Native American population history.;citation_pages=370-374;citation_volume=488;citation_year=2012;citation_issue=7411;citation_pmid=22801491;citation_doi=10.1038/nature11258" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Physical Anthropology;citation_journal_abbrev=American Journal of Physical Anthropology;citation_author=A. Salas;citation_author=M. Richards;citation_author=M. V. Lareu;citation_author=B. Sobrino;citation_author=S. Silva;citation_title=Shipwrecks and founder effects: divergent demographic histories reflected in Caribbean mtDNA.;citation_pages=855-860;citation_volume=128;citation_year=2005;citation_issue=4;citation_pmid=16047324;citation_doi=10.1002/ajpa.20117" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=U. M. Schick;citation_author=D. Jain;citation_author=C. J. Hodonsky;citation_author=J. V. Morrison;citation_author=J. P. Davis;citation_title=Genome-wide Association Study of Platelet Count Identifies Ancestry-Specific Loci in Hispanic/Latino Americans.;citation_pages=229-242;citation_volume=98;citation_year=2016;citation_issue=2;citation_pmid=26805783;citation_doi=10.1016/j.ajhg.2015.12.003" />
<meta name="citation_reference" content="citation_journal_title=Annals of epidemiology;citation_journal_abbrev=Ann Epidemiol;citation_author=P. D. Sorlie;citation_author=L. M. Aviles-Santa;citation_author=S. Wassertheil-Smoller;citation_author=R. C. Kaplan;citation_author=M. L. Daviglus;citation_title=Design and implementation of the Hispanic Community Health Study/Study of Latinos.;citation_pages=629-641;citation_volume=20;citation_year=2010;citation_issue=8;citation_pmid=20609343;citation_doi=10.1016/j.annepidem.2010.03.015" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="twitter:description" content="We estimated local ancestry on the autosomes and X chromosome in a large US-based study of 12,793 Hispanic/Latino individuals using the RFMix method, and we compared different reference panels and approaches to local ancestry estimation on the X chromosome by means of Mendelian inconsistency rates as a proxy for accuracy. We developed a novel and straightforward approach to performing ancestry-specific PCA after finding artifactual behavior in the results from an existing approach. Using the ancestry-specific PCA, we found significant population structure within African, European, and Amerindian ancestries in the Hispanic/Latino individuals in our study. In the African ancestral component of the admixed individuals, individuals whose grandparents were from Central America clustered separately from individuals whose grandparents were from the Caribbean, and also from reference Yoruba and Mandenka West African individuals. In the European component, individuals whose grandparents were from Puerto Rico diverged partially from other background groups. In the Amerindian ancestral component, individuals clustered into multiple different groups depending on the grandparental country of origin. Therefore, local ancestry estimation provides further insight into the complex genetic structure of US Hispanic/Latino populations, which must be properly accounted for in genotype-phenotype association studies. It also provides a basis for admixture mapping and ancestry-specific allele frequency estimation, which are useful in the identification of risk factors for disease." />
<meta name="og-title" property="og:title" content="Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/6/1525" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We estimated local ancestry on the autosomes and X chromosome in a large US-based study of 12,793 Hispanic/Latino individuals using the RFMix method, and we compared different reference panels and approaches to local ancestry estimation on the X chromosome by means of Mendelian inconsistency rates as a proxy for accuracy. We developed a novel and straightforward approach to performing ancestry-specific PCA after finding artifactual behavior in the results from an existing approach. Using the ancestry-specific PCA, we found significant population structure within African, European, and Amerindian ancestries in the Hispanic/Latino individuals in our study. In the African ancestral component of the admixed individuals, individuals whose grandparents were from Central America clustered separately from individuals whose grandparents were from the Caribbean, and also from reference Yoruba and Mandenka West African individuals. In the European component, individuals whose grandparents were from Puerto Rico diverged partially from other background groups. In the Amerindian ancestral component, individuals clustered into multiple different groups depending on the grandparental country of origin. Therefore, local ancestry estimation provides further insight into the complex genetic structure of US Hispanic/Latino populations, which must be properly accounted for in genotype-phenotype association studies. It also provides a basis for admixture mapping and ancestry-specific allele frequency estimation, which are useful in the identification of risk factors for disease." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
    <title>Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"QiduRO-d3dTsThHIIHa2Pxrk-LLsbPbj7GouonuV3T4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460374","apath":"\/ggg\/6\/6\/1525.atom","pisa":"ggg;6\/6\/1525","ac":{"\/ggg\/6\/6\/1525.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/6\/1525.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1525"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1525"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/6\/1525":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nNR5nebOf44q0o1GsQxqg3eG3x03ISZ0TjQFlPBSJOPHIUErRdcCf5hjN3120Rf5AVqaIuvbyEEw+KoeTo9Xzx3jqvVb5CNbdJKFF4mdh\\\\\\\/RyBw78V53xEOycMdSCLg59TmdQG649+2lRa9i93BfBga1NR\\\\\\\/xIkeZaLoAz63bk4ZRvv+2KJ3BrD5Mc\\\\\\\/gnZgYMvlnhhxzz6XBsKO89q+dThrQHWCyR1VJix4Y5Mj7qxyboH\\\\\\\/rhGin1qzfYirZSLZXadK5IE8UE\\\\\\\/3lcGqs2Iv4yOGdc90bkzSZPVukPUqVcsA6m+qqF4cuq0SUhVrvlBiPum7GpSAGfd\\\\\\\/PNhF5LBQlpEjYkFOMTICP\\\\\\\/3B6ebasYdELGOOiDtMv1iM0TD6cAGXBnse8kIL9b6E9xAtK+7JWrt0NPpcSZDIJsUP1OmzVLYm0uDF7WYiDgYTFhLCfF6YmXRayzX289BEQjz8aB4YumVWvQxEzXm\\\\\\\/59cIBpJWHBOmh+idPyEXILEl71MpUEPP3\\\\\\\/aonkZ\\\\\\\/iSEybqt4\\\\\\\/cCDFFNZ1ehU2JuVoDGmMxfcwinwTurHg8blszMWIDAgcb7wErtsyU2oglsyRW\\\\\\\/RR9\\\\\\\/7HvoYiLn+CoN2dTsZVOObdNifgt9kuGItott1acTmo\\\\\\\/d4oSdDyKRWKo1BK1sWz\\\\\\\/paEGK0PE80KgqxNHn0NKzr0BGsI8l+okRSjBcTpuXNtm3EZHV8txnsis6YcZBXduASQO6v7DkZ5tiWaAYjw\\\\\\\/Vy1GNB5IBBEC2q3RtYYGCKbftGo4\\\\\\\/e73Z1C+IiMqFnzO3CDtXITNzaDZSvFFkMQkjMybRheVlT7cPlpF1+xQNR5eawFuyOsTIXq0nSQzJNWFYo3PF+keOF0Se66Pn\\\\\\\/PVITsJYhbbRWDZm8im8fVN4s1qmfXtQ4T4+adAyqtEuuFhKVueYJvlZjENu3q9VHGIL3dzoFiH4BFm3ylVw382BtSbA3FhWAyMRdqdqarbzXHrx+cGaqWueN3\\\\\\\/afbqraYtugcahv3yRtFgxOSAX1X8RUgAdblKSiO\\\\\\\/V9fm2BMXJtT\\\\\\\/1g0P\\\\\\\/jU2qIevtip80D4D6FqEMUn6t8JJM2U1uGkm2kSH+hffvMv\\\\\\\/eztHWGm83HccO5eJ4+JhPYJNYQCOHQqcRLIT508AxziKVJqBJIMhuiUdUAu5IJmZEdW0WzPtw02nV1MQ0mfYmE5cMNgHUyFAt0DUQJNJi7fw\\\\\\\/sHJZQW0PABJkyLN42dIxpf3\\\\\\\/FRKvSCArGSIWHtCHc59itDoOI0mTnbLC28eFlTsuQnDXEMEYXr0B5CLK\\\u0022,\\\u0022iv\\\u0022:\\\u0022GCuXeUIPKDuWbNLsNWRHUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f304f0ef5000a1c6cec713db9be79eba6d4856e99c818aa7749951ddaaa961dc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sFZLfYuRvBU4SpH0NUUxggprrJI7NlpAMJWFmwu9WbCgUUSybCETZ2vepdNIltChnusrF2JSu5ofo+uEaRRK9acuebzkY5jtIiqoAXurENU1e94BVoDjdT5OBnjx1V0wvY2hVZ5WQFDEN59s81i9KuAUm6cK5f+XaiOLRkF2S0QHpJU8ISezaolzeK1xMZwVma5boefGke3BMYlfptFf3g4+fyMARAkF\\\\\\\/UX0zyzV\\\\\\\/7g9QXkoxB8vyamRXz+Nb9IoJ2WTB8BnkNfVCBy32iXivVSnBSJvQ+s\\\\\\\/jq2FrOeA6zz62w5KEK7CO+uqWzOQbWMKa6+8r27yGnNp3Af8irYsUlxhDtIccC3HMR6AGShmvmL2JbxhP3jWUK0F6dtxFixha1ymxbSt\\\\\\\/EylEBscAX72hDf2enUioUvvB\\\\\\\/6Q8Eq2XQxCWxWCiBZePwNBZoIuWo0DCMII8bavGPi7GCPPBQsf2WEZths9KJHi1FmUCjqtGPMMOwcl2pFLBj7iztgR633dVL\\\\\\\/of4ocL0s+5Dz1gjQHdtsY1Nuk2JRAfCjvKGXy\\\\\\\/q1bVMYMwc+3asqTH\\\\\\\/hmPT2jP501UOfiXz\\\\\\\/qJPwDTEZZBj2PPz7rL09NqVOtEtOS3RFyMTM4gRr3iecvXNFUkByL6youSjXDu9uKOQd6h56CaZryHfyHXuFzL7\\\\\\\/qSWv83UBh\\\\\\\/O905Fc9AKaxBbDY82KgEtv75nsXyibD0ss6vaEFpTnetDkf95Kf8qPcO0\\\\\\\/ynwdj\\\\\\\/C29iJofVChZ5lHg0YwYuGKT2n\\\\\\\/0secSUopt8InZNo7Azy5BJjr+MDo8i+8ccN6lK8GwyYHfM\\\\\\\/IFAZCgvDspzi7WmpPGivgAqG2DByaELcpoKARm4mxxi8vdeSQA2rRFV6QKg4S5n4R89UJKf9krwg9QDrjv5YQA7ZXnw+2Z5HzeOwE9GyL7wVb9RDbyjV0ug2kqkLy8Rf87FPAWNTNBq1coLQZZU8Cf7J9x9uB0PYAZzbybHUc\\\\\\\/j4lMQ8fqKgbscWafo6s\\\\\\\/z\\\\\\\/VMdFfDFMKdM36CQCTYjEytmWE0nsmhZRRYwj9GIB1EyLdUj3fjIZ+RX3FQo5LUqD\\\\\\\/RGZ57NjEKRz5rmLOEkm3LTV+V9bF2iJyE1SdsoHAFAa16IdYYQDQyLax2AdcsHj+f4YCNOWu\\\\\\\/UxfcOmkFsJwku6iDbaGMQqrQmA77NuItfnazadutD4VPQrPz4ln1rivFut2AaT\\\\\\\/40CKkbemS\\\\\\\/PA2EBo+lSI+3mKXSwNAIGHCS1FQYmyKulLK5rct5DvpDNA4LnOMBqNmnXWu0Cc1oNzPCnbW9lHxxjEzSaxpWtYtCxKXu30D58OH8n+gaaNEjLJCo5seOcefHcr+ctA8xzFXPboylwdnFQYHOjAEpJULrXgKhedwbUu4f8jnhQil2d68WAVH\\\\\\\/I5qWxHbkmrSI9g7VqjuvLTyFBDtLAVqwkYt2AgNkp6pClmcNE5iDVsL93gDOBzGEZrx+4H9RaKAlTa7fODxTYaJsKpiQLseFkhT06NKvoYLiMT7mGS+SVx8l7qm4\\\\\\\/Jxj3njMgrp7bA64o2KZR9ZySw+vw0jIN5L57LIkcsABcSEUyh44pOpibmYG9tiCIh3ieo93uZIOw2juSy11xXlbNnj5yYUuwk8WFyfxVh9nJ+JMqrf4UjD1jyDA0y+Np9VM1dGQr2TFIVLMbfUms4xSPDEqFzjq\\\\\\\/x3xbUZxBMXLFPXa000vHDQJksorJqAuMTz\\\\\\\/ZL6sLg8TsEy48jkByy2XOsawHdQ166ZQZv3fsnNj4QXwSzaHvAkV0EvZoMpXpn62vMqsoYPL7gmBUyjwLLh\\\\\\\/qEHngbA98HrBUxwh4s32\\\\\\\/PqwYGJgNVzA6IxSiGeOFRkPVUv+EPumzqZdNlQVlLLq49QCZ6gNV2iojILEOchC5HCxKSlqJv6CSUw92LGbXf3WE7WypqIODYpv8zJmyb9OSCPs88aK67s78NA\\\\\\\/FQMIMl9cF2l3h\\\\\\\/SEp78Ery2fHS1JZidlxq8UgjAsUBv1y2jE7oLYYEtwKH4zIA\\\\\\\/hFqjNk\\\\\\\/NBdrk\\\\\\\/3AYzPMOpSDIc91CP3g1g9RU7+t3LsXdjEEcQagg4AzCe3IQii70y5m+ta3k3tmtHXuRvk07QuMbz\\\\\\\/wm\\\\\\\/Js1OGwfOYV8cscsEJMKZXJFpnz+t7bGIGPW2s1EjlxMBwXlKg0EOyGEgtr6zhsKAeIpLnb7XgkN\\\\\\\/V0+SC0uRMjY8YzFFe5SgBI0tNUmgBECPUYueAEbdhwWF3zWBzT2wcEIcI\\\\\\\/Z7mP36qRH2FViFvSpAmky531L0xXskW1cT+06z\\\\\\\/0ahOHqg4PDSXjZN+HI+fgYKSNatkWriem5rCwjl0JG7fxoFHsi6ikfvdNe0JbIlLHfRN+xjH2dsvTh59JV2RNjDdtGJjgn7tpKzh6u8Z4WsjlJBPMlN54504d9uE5FeyyjEfYHNCISmH5JpKneBTrinwO1BNqxpLCiOYnDQDOoNtC1tI7PtOt0cilTeV\\\\\\\/ONv\\\\\\\/D0uLwekgAbNSUpY97tNbjAdp2NmQRl7MeO3pY75S5vA+XvAPbWtQTbZtnBPP9IzbRGRMvAl2Xb4BP+gj3x4mMk6h0DltWOA7FWpRDtSB0XToQqmvpztaGYxR6Tyij68uPXRyIq+VCAgmg08Om9rJ3i5IdVpHd3EPsz5GT897e6xNUrxM0A54KnFvAk0Ejfl9J43F5epRjASrl4NMlZLb+WAGRGSBErW7bW84Dn9deAR7JhTg5gj7y7wNEhxM+5WNzAMc\\\\\\\/sOG3cFg99rxT2pNf5ESNjnsKnlV0\\\\\\\/\\\\\\\/PMiVwWFiJArzD6ZTlSwD5qdJ\\\\\\\/iYD\\\\\\\/KmpZLHxvbZZylJeVPeLgJegvKDhdxtR17y2P1VgfBZkoHNheSMAimpwhRImRwSoWA1mlsDa0syYFZGyjh5SKb+45v4FISf\\\\\\\/Nh5IWaFW96jg2\\\\\\\/HH9mI+kvA9nkYrgUeVpqIqzi9DtlqvikITyWPteDJQJd5vn9ddzYsSG07DdVbFZmC5lX1s9AHPk7wHPE783HFZfiA1SIiO7R3dKDq6a5y8l3\\\\\\\/lEDgMvQiGIKRzUapvXAzyUIm1yed5igMbw2vMQHRCKs82QPACR8QyEqSbQfR0fGr9BpFn4YzBEf\\\\\\\/CRA0iYEBbNqN5t6QnpvxIDJfEsm0z4JCPT4k19Vle9ooXtRK39nkXrxCoejZcUWZEYw7Dvnw8QdgwwVwHWLr0C2wDx6UCNx7pI2a5wH+8U1IY00IRyaZJLOCTHlF47DRHs3Tc0xYSBHRUBRJDzsITm25IPuFG1kh63AQFR9NIcrojkjQzLuJhWlTHNvm7rhk+Nh9UWwVPgnMiCt9sjRm2L0n5AQ+I+z8bh4Hd7QsWCBhk3LUAEw2qqIDYHVM06HiT+0eBt9DhicVDJrI57yE4MYWOE0PoF\\\\\\\/CStaFCWzvImYdE2e\\\\\\\/n45M1j+UrJ\\\\\\\/OM+pPY3KGrVLoS1yUz2z0XfTvx7rYxnGZSbUyl3kIMezjQxeAmqngz8QxoS\\\\\\\/FFqx2UOjB7e6x39HrHvJDG4UUFHd+s6sU\\\\\\\/3gV1tyqGsGkohMbCb38\\\\\\\/4eskI8Vub7jXFUQilLhxyTpPLw3AuchCO9mSBolPjMNyi3tJqr+pqpmTifREj1kM+MJhgVuGCuEEIrAuJKd+aXVOloVaPokF+fGl\\\\\\\/Bn1hikAGAUhcLSYjTDRm5edGdnQ9+L5eGZSnZgzNug0u6TXx\\\\\\\/bJCpVoj9HUOpjMXmK2+tlTE+KzsRtdeAIDa7k8MVdFAg91M6kfLIQjydYXjnq6G0roP6i5MdKUT9k8P1x70OxB\\\\\\\/nCu\\\\\\\/dMYeBYEQ4QPG0EnXHwtMmeCt9Ay\\\\\\\/mB4NpsVsafYTY7b0JzxykUN6RrDasvFc\\\\\\\/nEGPFMVBdk5MMVC\\\\\\\/YEV8tMyWGtXDqxsZmmAza5pj6W7ZBa9f9E6DYshFUenVFAHueb3AbMwCVSy+NIa6OO+B5jgwKHeFBBJQ3Lofcyiw2EoT45m7dJXoQCGFztzuv6EhLlhF4mdl43j8CDY6gvMIryGiyQe9H4WB+jQXPiE18igFVyW4nBO4FSVOr9QlJF\\\\\\\/XYzkNbtdFQwCSDGlsEf07+1PjxOlrxiVUEpK0+CdZ\\\\\\\/5BCpe2Z\\\\\\\/+70xSX8gkZsS5oC7xlfjQaG62FB0p80Z\\\\\\\/pr2cOmKEl\\\\\\\/30UuNghVO88K1uNOtGsB8FpwtlqRPeRTl\\\\\\\/dTxdQ1+kJ0CBhi8rYA6KP1zvWNMb4u+qzpcOwnRpx1aRWl2ODlwEzT95XXRTBfnWMg5ys5\\\\\\\/uNhm7PC+ZUoXuujC+XvSpEUhkrN4lUUElUdiRgHZD1CuOu6+DT1hSWBBkGXQ88FtDQ95Zr9zipaX\\\\\\\/nbYBozuNr14ZiDJ6wFHzK1Cj4MwXxfqjvEVvhJck9xpdjTQ13lLVBS0Gp125bIQmu96vSxe8FJWZHQA6VjPjvwslVqr7gPIt6YBzAmLUFLlnueEl3pSjqU8c4j1Vdmm6VUb9W1sDZYz89H8Nh0gRIbC+cAIZOI45RivBF1DDydl2hjkHjzRprQOc4XO3IDMS9IM5sYA9PFAbyobIwaAbEBhmXN9rd\\\\\\\/+jWWFwayW9+oDvI\\\\\\\/vTuvZq6CC4TTJz78VCYYeAGFhOPcMDx7fYTZhyAr1qaidI2wau6xrt+uvtskXpU1RJb4tEoWvFdCkgeZuEB50n4+tegx\\\\\\\/97u+YsP84iqnWGTZf8\\\\\\\/rT47ekja+XJDHUptu8OeVNyWvJ\\\\\\\/tY4feriOeexhEzQar1JTyFCKLjeLFLOSmjM+HGjABKyZVPDKoa9mniry\\\\\\\/p4MAjPLauLulFYlpAz0uOsBTfUFl1YF6Ap1fuoktYLw\\\\\\\/aDRaomDLlyWT2HhtovAWJdU+bUO9KLr2ye7mXl023ZImDxit\\\\\\\/CkCWin4I4pLoAYUxL2ZajcZ0044PGEVrahZW+YLNwtJGBEQB+6Dwfa30pesXtTWKFOBZ+qS+CZPTHkoix44fVsSwMAxW4gG10PPZdSvXae86lDOaR3tWsnDJQh5lMdgFc7aWyDawhSVxRYjMy3\\\\\\\/JGF8OPhUtMUNh3bm88QnYDNA2jaYnTc3cOS1BC4FxKqi29apiNIgTWR0ojKnHI+q8fcWGbqTmfoUrckLQjhY36zhVAlMQrxTTraDt4gdp7HygbNFbvPRMRGl\\\\\\\/1QyBb4sF0UElwrK\\\\\\\/gnfq\\\\\\\/ILz1LeDTNOK2pY0iHJ2LjmMRUVYgDtbnrHCmUq4PD66kACE6rwL7It7\\\\\\\/DWzKd5Ez4XU\\\\\\\/St6FVVD9WxU0T0Un7A+5hZNE6Zwf+JuEH14GjHJM2zQ5EjPAml7j9YpE+7YXJKB4I0mYWp41sXi9tDkL1LdhsC2Ny4fHSF2TDDOVw1byjxPS3t9C8AtIAq428IMVOrDaPl9CTjalwjI7GURcWedHGkdaYurNlYqQDKuwSk8xoMWNopR8Z5BSme8TPwaCXauo1f9byo6NMuIsajf1DY+0MJoVEcRwzbha1D0b3O+BjRk2E2gQwBGwM86gpsfttVo3dAFLWMSsawA33DlFlvZcRNaATQv+wnKNdrrJNLva7s4e3dqY2szJ3XvV6ZXf5Cp8c3dB1LQYHS6+yC8ks\\\\\\\/xEGxVJRJ0ObJMU79bIcDjtx9Uvo6dgd4Mzt7FRFPs8KU\\\\\\\/cmjHhRD370nW1ImhktxwzqH5a4g6MIBnPzJ\\\\\\\/V4f+hmEHB6qK8om71xrw3+jiOoshAd0yP9ZwiJJHLpwuaiATlIkddCz+CtQ2Iwt1XpaA4qylzEDs0Vi82onwLSwN7qU0uOxgUiqGgCxswIsq161DZBmuyjEfh479nx70ui2sjhweHG0tESWhIj3LIsMg8D6se2C4qZl6JwsOfHocHMZVpm\\\\\\\/qODpUvbnr4BrUjJ2WoqbEbf9KhRn7YmDyP+hzHfpYPdBlDI+jq7FVYO6JzA7ZXD7Jbd5bbXLkB2712OIMDYpC5C5+vnUVsnS6BEdSMMLtRN2GI+RJ6YKD88F\\\\\\\/Ul4250GX9yxRSTy\\\\\\\/2NVsVtFvfI6WlUIgEzLa06j6CnGpilRV64VJJlbAXINNIJnZoo0vOE4\\\\\\\/TYbWl6F4jmsQrWGDcuzr5fAuhHxejQrWzy4U\\\\\\\/mqKXOzXipdin0u7o5fS4kog77zwXvTvx8wU73el6a+IOalyBUzl6FxMZx4W0GmFG4rfXvtsLenS71jyuNjoEFb54PdKQ5AA0vRfgWJdBCFk3DzKQA+p1k1zkZFUQSsRrXV9YR3GuSieb3mdstTHS768CIvMJydd\\\\\\\/6OsabM2diq+ebw7SAIoAVy2k6CL6a4Cw9TsAATstL+6Rl7555EZ0MceZGaaMB28SmjhniSc2nonxOayR4anP2bwZA1kch7ucZca2NkpMYfLXocMW+OLQdubmlpEQJKg69EhcDnDwfF74A3rc+XHltO191DRDq1Igem3448lpx\\\\\\\/3PWgk1HXbgR4tORo0vNaL+VxvS9DoY03a0NNl5SLp0dkaDmMFgYF9\\\\\\\/IKePsWeDHEf7vo+bd949doTCWek3Xf67agrqAghpbHU2DWDrxjbN\\\\\\\/kyvu5BcMmQH7kaEqVbOSWMG5X42fvdRv8Qy7GHqE6KjDIKoi\\\\\\\/jiRC\\\\\\\/45hlUh0Cwi5SOAz9m6IS8uQDGhuqG3t2pkWMsU\\\\\\\/J22Ac6VfMXg9qY9esoRszc6ai7ODes29kHt3eqfkOqZWhpTyvshz\\\\\\\/KDZU25yN6dieHk5uMUwZlt6+t5eWKaD2BgFvbYHn0MSn\\\\\\\/OZKIYfU2IhlMFiUY\\\\\\\/AXdOzgbmw4swewIJUM0L9u7vHR\\\\\\\/XUUjulAU7z8CTYL75XVGpOg+c23CFK0u324MhASfjjaj1WpUP8DPviHuuXikSG\\\\\\\/4Jh0NhsZm\\\\\\\/CM6YLzipBtJCm7\\\\\\\/oJgtLpdmXj6XaC1hviGIoTz\\\\\\\/mi86L+ib8BW9qmKTDivmlYXRAUefpDqX2\\\\\\\/nQr14UughMqPIloItn2GH6g7qqEgY9VVbCYV7MhNyUu0qFrxvcGrVqzjSb6YdxndwPbcjlNy6uStJ97dUuMp+rirFyv5o+1sFenkY5Z4jI22RxTIj\\\\\\\/QtqaEriTFZMSDl7YDFKfYKxzsnSzulcGk\\\\\\\/1DWuffHDMUO6b\\\\\\\/VQqvbRyvMnhL7z\\\\\\\/Wa+H9e7ui33IuZsPhJTom7wcQerH11u92DcaKJj+cv5IWxJOYpX5C6dt30H2PHSGrMfZCWytIOlrt7zLYsB1sCrJrq\\\\\\\/2qUp\\\\\\\/oOTC1NUqH\\\\\\\/j1tE23hi0Dv2V1Yhi47MroTTdQDTJR7DvdUEBzvPlR3LyTjkhlZJV+cAHXdqk+s\\\\\\\/KbebyLzIG6c0tGZIA1+XzUBnqdshxsvOUWwg7Kn8cQUmhioCLRG\\\\\\\/zkU6TfJIpOKKYe9tPrtCJjxmO+T4TDkXHyWn89UpJQzldyk0n7W3nf3EBm2q4t3yNIkqM75ero6omUbYWp9QCm50PUqJIM155hl8UdN4W5UnoTQVZR6Ds0VLX4wdEYnBP1XtPScsgHi9o3kyUDD1hMNzuwETEbPNPYwBZqp+BUwFwoIPDuvw3i1+3CVkNo8K6bA8Q4dQe7Nv5rRVTIoqJM72tL7dqiClSqDagEoyQhs6pGliscDeKpwZlqX5o+7K2nVyXaILrfQnJ4RbfadkwryqzJdaw2TiyQMviNDZNnu3dnQ6\\\\\\\/7WhLT2Cm409afb51EygH1u8xBKT64Z3TO218oyf\\\\\\\/gx+dPVMO8H69sctJ7ECZEU8RHtY3OT6v3e2rMRT6qWluXXrMtnE3aaLT1y\\\\\\\/3Jq\\\\\\\/\\\\\\\/dDgsyx4gbe71QucrVwecpsKl0M8GIsflOh5zonNcUl\\\\\\\/GTpzs\\\\\\\/w1A\\\\\\\/E83U4cQ8DhSfVIAmXiy2rla7HbEHRf85\\\\\\\/OZ\\\\\\\/1GHb\\\\\\\/rZWtChdjXqwzWVNdZsRI7kKT9M9p0yx01l08GTztBnhmwsowBNy7s\\\\\\\/eg9FffDYZm3SoLp8wEqN90QX+Mtcg6PYdhtTtn4B46hiE87fW5Dlqkv\\\\\\\/s03EADeEfFMPtbt0TYfFYj2OTC76moYLOIGlSwl39ZHGdY4b3wSnS4k2VvOb5MZbveM7Z5VGinFIbEzsDzlvZFQg+Lx7IdXiRwbcmrouUNW1DdQRjgHYtf6Ddm8XmwOjLBNmS2vZ\\\\\\\/0XAZuKkwnf3ec41UG6\\\\\\\/HVIYMehAAFAHfFQBbZy8FFwfuJx81XUSwB4nv\\\\\\\/\\\\\\\/a2NJnHRZmZyGr5ESKPzOf7ZDxpki6TAHcb9uL5lLlXStedlXgr5329xmNFoPkkJIRN5xLDBHcGNausft+bukj6Z2mTwaa6+einXFQitaSk8qoXZQAs4vKYUytKcLx5C5pcOoQ6JEPFyAFmO3\\\\\\\/S3uNU2wtGev4k5SKIEOPbKK0sdWOS7\\\\\\\/m\\\\\\\/7Zo\\\\\\\/\\\\\\\/yuhU5G4IffFxhSMPiOtuKoKWmHxNORq+iEFI60AG3Jnvi8SnqAILr8VQFIf7wJkLDALIQstNdYwPyL2LGaj6OeXEeNwqv9voww1lxvfIC0P67xsw\\\\\\\/mNLGzPwzbc72zRMg5k7Rczg6n7M6e5oyudJ80NZf4vJV\\\\\\\/Am+GWOqZSsrj\\\\\\\/34\\\\\\\/6nEdUQBSF8ry\\\\\\\/LqzQqYv4blDTOmpCq7SwpjKrgWVmU0rBrS6cVCyqVA15fvF\\\\\\\/ZSHW5xw69vGXyF65rUBEl7R+zA072sqmnepc5422SbrUPts03xbtIIKX3socxNXLwXdkc1FM9+jnhBId3pxEIcWfoPVfmgEIgN7yYx1CnDeL8MYcmuUF8WIClbd2q+f0brY5ukPcoVDifIg9GxdPPPaTzZ6RHhygsMwmZiNB4mZ4UW\\\\\\\/qoZZYqpTREHq\\\\\\\/zqF7kehL4DdRUiVFL+k2NInsxuSAs1VAE878sFhDeM8zW5sFSA4XrUdumYAHEoMHvbCLI6isfmZ+mExviZ6jIUAHnaI+pyELdGfKwomBZ1EtBC76i\\\\\\\/9eIHHAo1tV+CQAWJtQqTciY8cCtkDmwNF9WtU2IVucgBNnzW2cQ6RZV8LFoPbWzUYEi+v1vsgZ2iq9aLQ3xFlrTFp02UmQ8UXuDV+IZe4+m8duwCyHlk5fnfVmVXiVYlv1RU\\\\\\\/+TxulccgP5Ub0mkeuxqtReFlagnSqWC1l\\\\\\\/jWarQE1ZSyJqUwwyTGuD0hKh36iHHFo7nByn8KXGdf0a6zLDBk4iWNelYLbCQoLEFUthbAAeebTIyh1GLjPCfDQvGcV9HmOt8a+UBEA1m2ojYtZDBh2xl\\\\\\\/PSszCKnQzHCkmNQx4s\\\\\\\/cNqYvnb4LAPcxfsvMpVY7\\\\\\\/cWh39WnorOFfuxdknMRB4A\\\\\\\/iBvLSLf1kgC2uJpRfe7Ghe0A1oC8+yhBbQFeksOnAn\\\\\\\/bVdYkY3XmSxPMqkO4WOH\\\\\\\/qDQTUZm4f0frpFTgZ+zzs+HbDeobpzSu1vNH5sErQF4nreHqwGK78iLaHhh8i4AiTzG8cF50l\\\\\\\/imZTWM9jTXI7qoKPei+SHdrpstKkLaQEEfV3n3LtXaFcKPbFUGYv+SWFcX2d2SkizPaRq2F03WY1BXyFSQWVKD1JvPyfruwKEtaVkUy2vDazGAgyfhy1PDT6WDPAtortm2W5zf6WA7EoEM076z5ZxbdS1+6TaQKZhhIookI6T+vo0Z93eqWPBITtIgYc52Qv2HMe0d\\\\\\\/AMzR6PMOmsTjJ2wYk124\\\\\\\/VtAejz0Ao2Tvjktmu\\\\\\\/5NcGM37jjO8F3FY1uHUWCBmT9qhhjTP8kkmiSylpib4bXkdsy13g3ql8gQBxtoPBoJQJiOG95CuvsydlXoXCJgXzL2b3nXVgiNy2dmFrfQS2MKzqIQi96kiQXPTf0K387G2I+f9U46GeW9mHufEN6zU485c0kZLNd6OKf7vxMMsZ4IFD7wXHsCJUZbgts8xAFpVzAPcwhyqoFU3xqwBkAoVAmxjroGq5sDEdF1HXz\\\\\\\/sF6cY+LxoOtOExs\\\\\\\/+sWIA2WdkrwSEj0wH7OVrlTMxJLguVjjuQCTmLWafFZoViKAXoL6HdUXG7NGybqwspggDqlkhd\\\\\\\/SC\\\\\\\/wRdjn+NowgYfPbjZvTdSiRovkt5JrgJkGuuUZ32R2TanDTm7H5lOeCo12+k1EKKc7oEnVosGkoHTXpm4L+y5LadNF+27XGapFJZ9hhAyxmBnyqG5ExJzcNddUj\\\\\\\/0SE9LzkMdGhmWIxBT+Vud4gnkoZRFuGgdV6p8pr0NVDjcEPlBv3q1uJ7bIT+e1Z9r2C49IoNcKFJKrywg8tXNYNJRpWmvxWp4AdwmgWd\\\\\\\/cxsKwkI7fJ+5K52Mybm+6ves1okvxoJnWyAnajCOSsINivKKjSK4eX+VpA2DAGtza1jwhiIMB7f9ujptmurkYaui7611wjnfHmoJjwQGfwkfheUfkSseHF1\\\\\\\/W1YVQYMV1Oo+hZSCKglK6gB+CpR+ZP\\\\\\\/lBdsTkg+fyg6lmRyfxw59puiPE6Qn6FiXi6CeYYXHEzENDcnVe5o\\\\\\\/uMKngSuFac1gsMRCW0Vk2SEHkvI1f3f5ZkeQZuPsglQLJPnN1pcCIW052WJqs5gLchtlRGpwzrj71jAgxWSlCfWglzHB+OEY9nvdFkqIpp2IzSwGB10yYX\\\\\\\/5BD4Z2p4l7dkhcDJS2dieqtWhx4cuMNZPCZtja5RcGrG+eVgG0weT\\\\\\\/UyixUDzrWzYe1Kyb0JJgDlptuicCV0owzBSOvThtCQCU35qlGeh8BGQEbQgGxZ05tCKkbWuxmtRHyakI0XWQwpdXw4fgaLDt7tolQJlTUb\\\\\\\/oVE42Kp7GNNL9zZleEjbbfNSGCQW6PU0Qa5VVZi+IZ2VxQzbD+16PZxbqu99kC5Gu2DoZ1lNNH68B81WHnhKiFyK8R8p0Ex89EsfLRU8qILKgMDIPBiRRg3zCA1j0UOOWaN35OVhZcrDoNd15hsvVM91viz1ZxwbryKLXD\\\\\\\/u+Q5ch3p5euml4sOxNWpMX1mxFBR0vXb9DAeDkQSVHbKWWjd0kyjXQVKjD69RxjYJOKydAU38311SXiLdJO+LdIYXbv1V8cAFYOtx8y4hEdGcLSh1VOTlqtvq4BTryZKJaVCff2Ruka4tV9k73AR\\\\\\\/Hn5VRvalz5TCmPXD0A\\\\\\\/2OKMbCWDQ2ReLKTpB5tG5gFDdtKTravNokcuSSXqaOGZGMrVDakeZmCirWUMKh31d3yS7VkvJE3WmrRixEw9ZTCT8Jcl04Tq8xQM3MCU3Ev07PygrDijJFIdz8r4bVrz\\\\\\\/dcg0bXnJCtqjrtN9n13\\\\\\\/ndLVZkCAO4rzsXavpeCATF5irxG8\\\\\\\/NS+afcBLTpWCkylyr4DJd\\\\\\\/rjGYzyMRA4fmZCXmOQRKZkCuLwhGhWcwc7Eh4FBopXFAPRiaESW2NkKiAKXccWsWAjMDYHA2gYYtAZLCdkR0gkXDYzs\\\\\\\/4cftsJfWqXDLTc00PELU7zQHVWufOD9UTBtCTWVZm\\\\\\\/uMWQlcYG7TOlTrE32sDxgUR\\\\\\\/1fv4QK8PE6a7SmNEntKrUcVzkAvWbkdmiG+VVfUl8hJSvgldu3P9eieEKmmgjsAvSF4f19LHs87EQiX1idaMt0IjJLv166l2ix6kahlApaox5AIG79QN3FYF\\\\\\\/TT9s4h\\\\\\\/lZ06jWAptetc1SdlP5JD8AMZ3mzKrfAc6+zIF563bloJJp6fUDoCItc8XxCirIcyZpY0euuOKH3VXluhWA4vwkZdW0++pzbs4yQ\\\\\\\/A2EPMIqYxIJntk7lgf3p8LhjgyQrGJNIX6f2HjDkA4lGXmM2hcapaQPKafhLeZTbvE\\\\\\\/M1PmNrfthAMBeuYFN+FbUQGhS\\\\\\\/2\\\\\\\/h5mlwHCplysWJvu8JChP3Is1+qTAWbZuF2qLAV5LtgPCmxN57Er27VIYrVJSKED1zuzeSsoffAGzIYfPct3\\\\\\\/OVQwdu0UB0+ZQ1e8TxHpVyKl18cEhGwODI13V3rnqNeKYKTMf8a8djYUS1ukDVsY00\\\\\\\/AfswwLs+uevK9ZuaMzjK5ndChZve4Grq27DASXFI3DFS2xgeJaYXgz2EeBhrGz2q5dmf7yNF+s5yVQDB9HXZuaBE1gIOW2NEltO3OmxwQSwrclu5i8gFGiyRv\\\\\\\/MCI8LBuvw\\\\\\\/os\\\\\\\/lrdVvKC8OyqnopZkQ43tQix7che8kauJLdHAEg9EgCpJ5wSVGbcH1z5Sq8HuP2gBCwV\\\\\\\/AqpJ9T14iUGXV57nmpnBBfrFV08lIeSQgqcrvqdydIPdKc3Sooi360xIrXhjT9aF8FBjEEheJGR2kCtAhTwnW4IKNu0KScOW5dgIW7Wy1xkX5ZcqpTUOEeQAvVJahHA82KM4MJU7yMy2DmhiawTo8b6NrZ3XIQOXFrO56XaCHuSRgMBwHIdmx0vxFZbCSUx1Jvj0fqarikMskAlL4wOSvXDEC3n4YmQyWmbkMYgQlzMs0aXrfYqCN\\\\\\\/vrCuxI2pMHKMHx4+5KLBrto9QZNSKY6Zo\\\\\\\/cwbTl7qm8pH8lCO8uad+z7f+5t35bkTJBnQwNC5fPIwPep0OSkyTI2DSoeqJdcFJA39ExCjlOKEE35Tx0lz1PJzFFMcL\\\\\\\/o\\\\\\\/yD+0+paEXgHH3G2fOMh4zUjgIzhGvPF1tbgmPJFzc5c\\\\\\\/I7oaFqq8\\\\\\\/HXvG9kSymGBeC6x7\\\\\\\/KzOZLrPQOtCxORKll2i3Ox+93s+CaHtJSW8tLtNSu3f6KKG2Vy8I8hduQhCUoSEHYrW4Srf6+SiYJHAEzWbTXByo089EswBKY6iwDSCxY+1p1gHlSKj1GZaq1zMB1PKm2nlnt5PjUfMHAJEm\\\\\\\/Lqh4ITrH2cZlKe21M8vJYGkimRNpGJHl2pLY3uIeg38vTkcQupSdbCFsleY9Qe1VdkEKhNF1BXEJ7n2wz0LcTuwuZbQ0Y\\\\\\\/GdDHU1czLi7t6SwMACcR2EZ6H5xzP32hWOgxAqQ64V+4jMLRi\\\\\\\/DsoYvcOucsIZc3xrC\\\\\\\/l1B4it7yt4xD9OuiSxs3ctNOdMg58ipeIjGWTn9+qnI9pWb77hSZ\\\\\\\/n8mD2Z1QOuRWpAsGZrNaUScatFJkERefUfAMIbAqxqy5IfJR60So2FoYT6wyw9x3\\\\\\\/hJkIjHvKE47AZzk7ha7tUQ9CwG6uNvORKcMm\\\\\\\/GYItK5ohsP0ZPi3Y5JBghZ+fzAg5h6s0+7B+qu4TBHKCpSta8skiTiCa\\\\\\\/Cn9xOVwWwFVgKGc7ixflrBOYGfDhB235+Dar68YubWBDfo4TcmL6\\\\\\\/jbBcW8aHInaE5NSmMzFpiQFzY8EIVg8u4uExQmkU0IXnNJLRm8bsWjzrhCfs0Mm+vZN8WHSgfC3C7pnRZ3PxduauHPkq65RFpzzQlkMYVoqzxHU5V\\\\\\\/VB0d0FIthwfVPdcGkZb2tFsh3ehyziWSo3mQvPUQwTMOznZZcT5tBDBGRfyLc82RlKNH5EPGD1wKQ5d\\\\\\\/hkHf5mM9joI1MUYsTWOdsAILqdHTbIIasv0zkDBrE4XFpcDBGXkEiOLfj5v53bn\\\\\\\/FVI8lrrYXmUK3oY24gLBf4JzBrj51p56Ljhec4Z39M8P\\\\\\\/e5v820YJqujwNUfXctHqsC4lsQ50PwrXWNsUMR1Iq5dz8A8UZQS9W4two0xwAIzQ4yv9pUwuX0dMN832HmHQ5NzE6HKe1s5Q0GeYXw5\\\\\\\/qTESzV77o5QY\\\\\\\/bDlErgL6nBTI47dhz2JTiQAat1mwHlroYSOApZwEJo8m1kr3Ga0HZHkXg6VTqPkD1s53GszXrcvQ9qNs3nm4R0WHIvHZzO8btRmI\\\\\\\/HIWovyp1QpsEc3k9vS7jzYWrLZom0YcY7WxSveJE89gk+ZN8ENILLOSuUmqjM\\\\\\\/EpVn69xnKPbI0TeG+1G5hUWGXFLofffYeax3Ff9xOUaO53Tnl9e+gpOXCGdAEb58W3dNuhxJs+1F4maxy3BEQXNsDuxIOyI07TnCo5GFVOcojnYeiZAXvv2IVpZ53uWSmETLFkH4s9UMbVypDbA+vkxQ5y\\\\\\\/gurHFPAQRchX1rHfZ1dzB9utEUJyj91Q4ko12z+FRYPEZ84B7plzAgjwhMiVJRfxZB87LhiZoxFEc4eftKj+meOFfk\\\\\\\/xii5xZpG1QBjceswTMEPFd3OGkQHRJASdsoTg\\\\\\\/V5JKPkgqt89w552GnQ\\\\\\\/BJWx337J58pJ3WmQZrwxOGfGvLFGMi0djM6G\\\\\\\/Y2C9iEaGbUNDT3QGglRArtq2l2xQmT5iaTjDWkSTtKeznIGCABW1VTU4lIXsviPux+Q\\\\\\\/0Qj0Dvsuyk87R9Y5XSqUEgHUQpqF\\\\\\\/t6RNfCN3v8bZ80Gi52gDpH16asxuJCIrpAlCHl7hrTrNh2GJiw+8rCN8YtiJLHHTFnducFuhRL2TM6m8CxCnqx70WZipz7DVrPNT0c2fIMe1HRxkpw90I9ULnf0KmCArJ2zFSBHcqiJPzXa\\\\\\\/nmrMWGasZoAuBrWNOXIlcP6T9n4WpOp7nbkuAT5a4ZgnbfP2e9Bbq2ajMgFGZ2A4qrK3vqrMoLc6uou2PpLERvQXQ3cGZOpz3eATiMJwgbEz\\\\\\\/58RLokMl5ZIMU+XhINdWDjkkuZgVnm9VK\\\\\\\/G59+\\\\\\\/hBFfNjyDP9KbVCHNimtmIiR1RF6zlsbZUA7plWAuc5KyhNvyoyyd\\\\\\\/bT94xntZckOYSGbWLg0KaUSlwADbf\\\\\\\/JHi9uSQjYUTBRgKYoLVfM+QNrirH8y6LLhGhU+Erl41KwOf6axDe4VDuU9fYSFVeUdkVAHngnNUX21CqaykQBBvYrYMOi35fh8mhNmhMk3ic0ZxeR44DxkJyInmDzLYQk2xfiMzWzDA7y8p4MU8R3BHSxCBQ20ZKBkULPdRvQ3h3Zx2K5PMN7JtkNrWSTzOx4dV+BW3+69aFbdDh1akmBlw8CGZAtB2zgWL4\\\\\\\/dsqikNO6vWEFXUh9il1r2Y8vHFAJ8x1D5R9YSvUJzlck3asMYjnxeERKvmcJOA6dBSTW\\\\\\\/16e5c7rpU4+U4jilem4gBFARJS9KxVm81Z+IA2H1bk7kk4U1KpEmpupmDPpREv2wvbEDdGU8pva7k\\\\\\\/WHrfwfkb3oPDY\\\\\\\/hzuMr+pakz5SQSDXFyyRFLWc2qRc6VXobKPq1nklZ7OCF0hX76YS2emB1DlwPbBVDy0Z2TRVV9JCwWA+agdj\\\\\\\/mXxA3E19ftNi9MgKaphRlzLNAFHBJMpEwvMaH+wYZcNjk5O1n9uES+6alP3rNebDPGlXivQUITSdYhsxQ36Z\\\\\\\/nomVZYDiScLKLr\\\\\\\/QnrQIERPScmKj5jwCPo8Do69aiK5sRiuJIs5TPN8l\\\\\\\/a+T5mQAlrLYHypqSCgmQJL99LWYiO1Jqc+dsqZyotQYq17w+cHedxQry1R7Q\\\\\\\/mEC7qB\\\\\\\/n8Q6hQdU8Mruaj\\\\\\\/TxWGGOa58lUxv2BBFYBkswb6G4yqz4LTC89pQ\\\\\\\/LqNez9kc\\\\\\\/bSuhjJXR\\\\\\\/360WJJqGYRHnxbuaTLLlHUyT4dzMRLmSo7c6qQ5oACAed1MrcQUO5Tt9eSJ92SX+LtQzzH89gpH431zMMxHSC9t3auwt961ykOtqRpMsUiwfhmLIbapE3sHYG436SuKo1FW+NSOQy6\\\\\\\/FKs64XkqFTEzmuWnwEbHQLsWjM+8kDiFsqCT3AfEfGho3ntLIvGdgVM2LPeFji2ERw6AuqtF1WeHp32Gn5S9rq1IkU0RHmuI9JRVky3hWNjIUwxqJZmrOFGwc2Hpfj1oS7DESPzXOtcedLa\\\\\\\/wZIDRQzf0Y5MPIhl70Lx3JfExfH5\\\\\\\/z3Y7GPEbN4c5GTB1ipdCVZOOtj1dS8ojJy1SXW1nw\\\\\\\/fg5qUJY+luOkAAu7SFUiy1N12uP15TI2ElvlVZkyIjMG9\\\\\\\/S5VsB\\\\\\\/L2PfG3QcSMT5GWqVVBEoI\\\\\\\/FUjV2076w3fuvcZHYib+tAmMVnB\\\\\\\/YCwWrGS+ht3keF98F\\\\\\\/Pn8ElLASHx2Hm5BlebwHiqIW+a9SmfMtzaqKJcB\\\\\\\/I7ZWm7Oj2eaVS\\\\\\\/netrQSGlb2uPpkc7AWPaDOF7+PYJou3FD9nc441zfiLphOtZxVtlXyhCIln3FRNO\\\\\\\/stQmfIvbZ83PBiCS0qh\\\\\\\/9cv3gUmLi1Q6lZzfLzZRuf3HEumfE0I0h\\\\\\\/fXMEDhIREPVjP1petXj52NSe7qnMpLnyEVvL+x9Yh2AOy7uxp+qn6oKgrtHnAIqCj839RpyFaFtEMiMPJDo5AMhDVvGZM5rArLYlbqc+NQ1wrINsHUU7H+HedQJSnw59g8dI8lZZMGxDkjsehCTpz\\\\\\\/WLdh79IpAMJZvA3WMPON41lYdpnwLl\\\\\\\/0HJGxbmq7SmbxTNluZeviuqBKV5jxubF0xFhxvw3PckhtPTV4QPyBTQV0yHtZcyLjlC+vsbQ5udwGL82YcuGcsAZlu3dTi8iYveITULUH5DDPcDJe3gTIveEWi0ymALBsJzXeONzQ0NWSHn54MrscH\\\\\\\/4hylNHSPn0XGq7u044rJIdOyMBLPd4ZQFla7dDtseqJUsxdP79RKZ1akCr5ycS0MEoUs7bFdTBX\\\\\\\/yosJYX0z4mF3Lm9xqj3wQhDHJAjPHZfU+2iinUTua1Uq6vrfKp18EVnR2jB+gCqohlPKFfak5i39jIgVek0\\\\\\\/khf6Z15UN0bhFH\\\\\\\/E+i0s5W1RwqKjxno7t3oGN4AUhDTm5cR3wN269vG\\\\\\\/PLp81mU\\\\\\\/GUFS3EjiJy4Z+punpyA7wROLwEV\\\\\\\/h3ikul1w9yzLOqM0TILY2tfSUhaRsVx9ozvKXwK8pqO8u6xsP1GHfdgWEI\\\\\\\/VGKY3qgGP78LjeVftzzaLRvZ\\\\\\\/JQluQmJxT1UbPn2hDur9BZE133KiO2kifVlzx9YOLMspnlWMr61HM06\\\\\\\/u1wMOrUQ\\\\\\\/\\\\\\\/8tslnbVHDBv0CNhoZv5w80CXamllYpumxrBYi\\\\\\\/sofEexwmiFDO9mbCnjCeQR9ZO7e8DwtlW7tREhFl8f5OcELSW6dr+an+CWu8XngjpfkBovfPk8Ef6Wh0B6WQmVIhHIQgOmTP6g627ZomdkensyatEwNmOM2EX0Q1k6HUzGzPJ02EZh9WbmagKV8T1QyMksePc70cfMXZcuwsQ0e5oJ8VPVdb5txIHbEGD4wPrYDW9cmLS0cuYNJbtLwsoJOdp\\\\\\\/zYsBmK7LzWPFw\\\\\\\/VzEEZf\\\\\\\/PqhnqghXpr1aa+bFRmSQngxKzSyV576kZm48Hr+A1a1lO2kSc51ATW3qj2IAU+wx2qpjMPVSKD9ZJZUSAM4Ne1CEQauIpWvcj9ExaAG5U8GjdEqmJigXDFvWVgvC4mHsJTb+6DotFpP8b0F\\\\\\\/C9RIh\\\\\\\/\\\\\\\/TdXaZjX2vq537FpH3AuRyT2GKcmar8QiLSJrWNGbyMxdopCSevMnStICS5yM7BoQ7nNFYmt1dUL7e41aFIS+yqVl6jnLE0+ujPmuWphmUKlDMWgmmC3eUAPMHx4XalMwn6QU4DUesNAaLn4xMbf5546IRBzSPEm266zH5zlOImBGRh2khToDiSw4fM6ZPaF5yQlmLA08ZhwxIOxRA2rTICO10Kre7QUNMUo3IIVApzkIb6b\\\\\\\/7j84WiiepRz604H33mfLFlWO9iMlzKyg32HA8H3Mg8I4DVSZOr\\\\\\\/TxJPhwwFQX69LjK1WsBeudNmCjFrwCtHd6RaSBY7BRh\\\\\\\/1sMklVtyPl0a3\\\\\\\/CRAGr4cX1JTw3A3oFEWvMO3p3nf2WXVRwJfYGnfJksCbrdHeQ+6jH3RG1rVu00Eyd\\\\\\\/Crywhtipng\\\\\\\/VLfwSqYR5YI\\\\\\\/eX1A4pDJemttxm0PUvlnpwLbIevT4+aX2sKaqOurviVAFZ00F\\\\\\\/midou1uObgvMDuAmAFy87gt9plCFTxr\\\\\\\/23c\\\\\\\/usljQygpVaSFftNqkTie4vfHEQ0D0J4Y4q9266NXdJrvYK5sq8lkVbnGL2k0TGuWYkvP6Jj9JfXwoZDJEbSGc9SqcIwZ0RHr2hxZiYC5bgthEzecjR+1EYcZP+\\\\\\\/nYQZEYAMzCwpb24HgXUztmTtetavVy9OzUQZGErYALZaMRT1kNUjJyjr+Le\\\\\\\/JVV0mu+DOw0JU+kMgmGU1hY72Gyh81Hn4wbTob0ciBlVDcVq8LQIcMqPT3Pmzmxjcb7ypUhDSWsXJRTQymneQlzlZP4TMyuXHyWsTbT0\\\\\\\/+OmC4F\\\\\\\/rZ6xTFCMHcgpcJoGnjWF\\\\\\\/ysqRCc8+wa+CSLO97AT5I0srNKqQvYczpx0l1T\\\\\\\/6CIofRAbPoR9Rni\\\\\\\/guNIeGfkC9ls3RHgPx3ucf4HZ8GCdUBg0VEhA6KI6XCHeS5kJK5dP0RpGzpvLqlVlMN3lPNEbKR\\\\\\\/XKlnZOkru9eJr3Kri+dfkw24j0eDtbsGkdHdKLN77S+9DsniVZifHvektuLi6Ol0ii7eOsoPB39xr0m3bnxqMSi6D\\\\\\\/se+M3wCdoEzQPUqGfUVfJPDRc7vK1pxMhngITD9A9rjCeDRX7iYGyknIp9lGQ4MRdW7jK9vk2gqz76MyYSwkukIsrzeIp4+83P1iUIyfCoMNeJh4ZTNUWQ1bSOxhLthFHDnMnjsBPlqjceS2BS0N9kDsyc2n7cm5lQf6ViLbHr1j52uChBVimSUXqp1VNZdyTHGK9K8yiunUvcXOscg8dEUdMhWnTvm9V6HBBWuqKuqAHIqSRL0hanZEtG5GcEbGompyDE2eN2cKSBHvOS+Dsg6\\\\\\\/7I7CkDAAzwju1MGU\\\\\\\/VNGy80OwhOBTj4ZL\\\\\\\/+vqrdtAxYquyM8mUIFeF3RSXBhxVDvA6\\\\\\\/sLyJBwrzTz6Ki4ZwUqm9c34M7cNYAKPchZLADZwB2tLmE3WqaNKL+Dh5Do1rKSSNFpYY7NMUtUxCVjnNAQ6qK0nSf13Xm66tVLdy1hctSkCUeFJJyeDIvy\\\\\\\/09FZiWetXRoMJG5sQvZRw3PvFKTpeOkiUW\\\\\\\/fdDNbTBb6WjJ03io2Mp3T\\\\\\\/83PivTtfnOhLFLDO3ajAMyTN1KF640iiPN0F2ivZU4cczZFXpEYbUOhb9l3Jfky4AYvpDApHHDpMy27xrisP18JJrI2rMiyXIoQC6bJeJ1Nn94bj\\\\\\\/3IGyOTq3ccAo9qTsm28Hr2seewew0wWOh90PCRSxxli+TQul3RynK5FyBU2Mf1KDV\\\\\\\/HTt22uOkzjLmmG0yW3+Jp1s0thd4nL40+LgIV06rfZdEWw2t9\\\\\\\/nII7ZAo0iAlfoN6kUGcRcO3yCkaStGSAQTHkgSJFXg3I1KkyeJFG+Z1ZwMwV1yqt9pwqZxHfZPB0L6MGFDhAa5r+T34WoQyV7vsYJX6VXORl8Wp2DqCQwSEYC2T1j61fRhbr1DoZoIZaWncZXi+JhjZQw0Gmq4+foeZw8tQUrsaQrRmdfQOnA+GF7C36hVxhQq7GaEl15RyvoSFaT6HEqJXUspBIFj8FWtbPQ9C\\\\\\\/wmdzderbhyCOLuFf4HnsduE\\\\\\\/QiuvALOFp0Z5Oo0+sVJQbz1iQy2DXOmbgptDLnM3\\\\\\\/FaEwQMvIyPfGBuKKzrQjgAmhrcLO2jE1ltrlsTY1VY3xhEMfnX6Aq2l6lt+cs5JEIvEmpWEaKgEcdufuY7L6OM9JtJWNxemT622IuLI4ubMnIMlrWj8pwR4jz5WrPFRjqYK6cZ4kPCcF+fQCiw9BF6GHmh32iUCszEk+QjH52Rr37iBVVTIxbzVjaXDMr2SGC\\\\\\\/1FBCQZXC57xg6oWFOtD7KUkzZSsfj2P1GegAvVArcMBzs07sEMNkWnykIvRIVK9Gx8tLimWupZMDsJSvn86QoFaA21ybmEURlfFdhgiUtPXw3yfE5c5Jcs49huhlW\\\\\\\/rrLuUSqk6LAbkgjkKhu2R1T\\\\\\\/W6v68gHMZknqhWq3RE8pxYCY2yaVJ0uwsTL8+mg4iRoT5T8SMn2rYfXd4RmB3f5W7eYJgXOTlYjYRyfTsz2LbPBD9TZ4q4DGBQWJwxVA1Jn+fEsJWTNcj+9zXAiBtt2nn9kf9lVoxaqiSABfBN0NekOacPSVtllGC+WYVaW7kIuEOnIvYgFFWImxBX0SagRwqHN4vpTKC5gOOphjQOpM2dUHPERGXSePfu5YhoFYYiUPiL8qHRAj4RtiQoUBJPx48hkyV8f+Jhta\\\\\\\/Fktbh0HAXTDkaqUsQUOATaRbGm8Oi3Yg7J9FiwBd6b32sdVeRLAHrm0fTDOKsqPgxqFm4fvc9A3Rth0v9j4wWWMFiwP8bqHGe0moBln2xC8dAZ\\\\\\\/80RQOiQouoLJXa5Y\\\\\\\/\\\\\\\/G1NjAI53yeoEyDHAQbnu8yevRIS1P6+QsFFahqlulwqpac8Loq+xc3\\\\\\\/sWRVbEBYmoE432v+ML0XD0UvvVdVTuu0Q6ZYhPes9BVYzljvDzpiBkFL+lAB4Ne7QCSX0X3e+zxJ3sNP+gSdkYVvorcfKO+R0\\\\\\\/WMxEje8S6okJ0tc5DZ\\\\\\\/8VHdUnn1IVnDL68ESnwACb3FDZISFbEwd1bVkl9IVL6in1ZqpnKS\\\\\\\/imlv7j632Mio8bK0hCBgF1u+r45AqXiWWJzDYDEffxPDNmyxZvfCyaQxyGkOkTX6JrclU369Cs+PUfhCLr6MyHW7SSJ54QC6JciXUFWo3z7w+HwPlHugWrw9mhpAmGHMPhcaRkmSpZpwto\\\\\\\/xOCpqdPeOT6lFLbfLZY\\\\\\\/Spppn\\\\\\\/1hERw0jbtZJ90FQckHb3M7EokcUU+K7g0B1hdxMPTBX9hO4yn2UaLyR1mYKdSatU9fB0GiBqaykOSeqg+P\\\\\\\/\\\\\\\/1Sp7zdH+JClXdV8Z6rT8AeAiGhlECULzcDJ9DajD1koCRX5iBVcwCvkoSjjypMF6jphDDmL\\\\\\\/HeJ4JgPpasgADstYJYF+KAi1r9qkUyyFfAOZG2332BulR2ReIQ4lrZ8gnTXuYjfLA1oLAmEqjmk1l8j6OH\\\\\\\/i5k7HQBob90c3hQxHKQMU2wZtmxiJEvihsJN7at2ZPlMImrrZAc+f\\\\\\\/boWqdBGcGQuAINsuU6aWFpLo16RCBR3yR7uJsUfEXZ8uEeRb66slHEV58gPSRzuEkiJoBiaeeBlAv72fXDvHjHzmVL5lRsSWMo37e7Zr981DDOBG+MTByX2rlWE7sc2JWuHgEiQ1vBoCJGUqz76dMQIDS+0CEfVnIoV3xbfHbl6H41ytFK5zJvRcXHpqdxBclzQlFjvIChXAQ4VjKKkvkriI9HS1FB21LN7QRbUv583VwJ4ZucOCf62OjoY2Yo5113F4xTcduIoUsaRbm+sj5mJDhitr0xDftBjHR3Phw7gCLaNLqSr8\\\\\\\/ualliQACptX07VbaJJVkaR03X3W4Sl6b2rDKx0IjqHFLTJCAzfzQJ+SqxEnmxf116QF\\\\\\\/ipJvnl+DVWiHwb9sPPZC9bkNXP4cXbv0p1wNC4rKNtS+9UEPOCmFoQOmzqMYAVVHY\\\\\\\/5C82tIxq7z+QyQMQyCcctnBCTVQf8gLxD3VZodmhR9N6hxqNbVUBxcadWLuDZuXTRiqUjH\\\\\\\/j5epNCAsLdtspk\\\\\\\/UIwm1RGakargyhx\\\\\\\/882DXYcjaZejlfBBWT4j0xN3c+A2uSzYl3VmeUVwtE864UYO9HsM4ZbyyI2Ay15dEIABSSHgkIcKL66f0is7hmqQikKjo2M8MzJnsKHTVkMqIttpAHIlkVDPaAV8N6CycQMHM\\\\\\\/g0WrwLrGn1eAIBOEKfFjEIYOH0jWK7B68olKkc5aP0FIyz9R0fs31xsFZ6s+PQDbBz47lXWkyNri7mgFbfDUhrDKSbm\\\\\\\/1aZ789QKUiY4jyhF5BbZPhK9lqj4ku7wY\\\\\\\/2JlfW7pdG00XWb353MfVRr4tTvp1cEXkAxwwS\\\\\\\/Jog+TAcB6pgVDP4IIA2AQ7\\\\\\\/+7EjLr+69oOmTqiZY1XBFReZP6Naw6w0dD8egjczTHoQdN3p8bKtjpM+lkoOUl+ycAe5FibZeBRCc75eLfMUNcX\\\\\\\/nvdYHbTBJhEF\\\\\\\/UDHN0wEQyCq2J6JfihBnqzrFhgb3wMCdg0T5h01+c6XTR6aXxp8Oq3XLsWoKAryRwrrP\\\\\\\/lEBKGekm2Oa5TtWE3JSdGCMz8HZ\\\\\\\/EpkDV4582iZzhuNradSvmJoD02mxzOvKLx+N3aPrCb9afo2J1vPKR2tMJTtBxUCvYSBv+EfqdUywvRCYk6DzEg3D5z8EUheJn8wKS3FPOo3YtRP60hXqPfplyocuB8+ibrlRhJw12z6YKL9hxui0JH+ozZKyaxOwJk9+pnrO8s6+lKEeMhPWgqi7A0TdKD7jty5oKKNGd2eCP0+IyWpJTkmO1UAtr2d5bu1JBjbugkKFyOriar1PWJGiN7wI68qjYGy4tt9ZC8PJvuo8cKiKeWZ72uNbdw9b3TLer5llGNSIgrGv1XIk+C\\\\\\\/feu3\\\\\\\/KOpOhwkZE5TcDdj2pUKBEYB7qs7XYMcQhVGicMs3k4iw5jKAsy8266eJSzBbmMmNuzOzEIFco9rZAsI\\\\\\\/Vr9iTHVORKyGFloJiVSTnLK0KcRZC5vsfUMZ\\\\\\\/DIYAXP3uz\\\\\\\/4vR0reIpf6DsIzmTBDkYnjQlhpNolwRA8nlGEBeIf+htm5HMf9y77D9CEeDmqXEcltW1dQPQuHYQTR77NHxPmaXPi1LioKtERYspAKTQFwanQ0Gi5nzcx\\\\\\\/p+ZdDvpiYQmPJLpBinjS3yWKw9pdSXTUUPDoil1QjE2niyGdi1qf\\\\\\\/1iIdqG++q7HvTaEFzZvjuu53fz\\\\\\\/suUjM777GJDdNWKl1lBDUCtwwkrgD68afFT8gdd+NfiIeOvhRS5u+ngCeFZOnaKWqWDbGo8bZNbfUK1QUcGVtVLmvQ3zYIkTuOVB016LOvDdQWTEwWg+9y5z3OWULrQAKUgwDagWaiPUaqnAhzROZP9CaTUGLK2hFCBP9xV3StVMOB7iuCX89dn1ap66vK2ZddiokIYqR4rFszNuPAkAgBptQYx5PlRv1kLYH0b+N+9NhL3kSFWN1RvpFBtIxkxxkDhJfIcwG4l\\\\\\\/xLNP9NLXHWbfB8GF8jA3fNoFR8YHCH6CkzfGWMLrHNyNwA7CjcOu8iP6SPGuzwa6jd0GaVrUeEtp0rndg5NomNyc66S4K5ZRLSYpO16zHA3hAJtBe\\\\\\\/YySWXKDUVfSEITqaRokDxUP1CPTKhboCx5MRHp130ok+IdaGmennTV36qHUZuKOvH+ixRefNW6tq2Ve5eJ0LZue4p92\\\\\\\/oT4QuuRIXgPK\\\\\\\/GSQx0CqWoFE+HXc4dNVtCg9BcdqJprtVcuV7P9pwhQyCmlv+qprcQ3DqmewPQrr30vYmjqFlVaODwb+P6RlDU6Qgh13AKITJkQDB6nKp8no9f3eoN2AM4QKgORqVmpfzAzNI4gH9ICw0qOmHu74Xqafx8Lr7fAAfppmb3w1E+qJQqS\\\\\\\/oKG9lzd5wFbB4BnDTMDNd0Y5rXcbG2ZRI10Z93wY86zalEt382UFbHwK3OxkWZ5JKRKletzRlJDXpzaCpuKuzEj2aRflV+lxftrpr0GOqymGPcvnrhIxANk8U0+Ukx3EVMbBRz+0Q9mE3YbzDFJ9iYxFN5LZCCFRChTpnImfO5obpKlvtDltGNCdoBf\\\\\\\/yLudLHOld9LxkgUnOwPgXE61rti2pmEEAa5D3cQVkspnBpzkcROB26PLueWiLO6Mr989OBB39vFphkoEw66ELfpXWS8OhqMo7v\\\\\\\/e6Khc60g5x9JnQc2Xv3TRki8XmyHPjEj1beyjwRgmtDixiamZm3zBOX2rI+bALYGeu1HcbpM\\\\\\\/T4wo9DxEYH5ATIs2D0LWswDq\\\\\\\/6X1oFtHOWMYHbop7Q\\\\\\\/6quzki8+0C7eosCB2dGVjEsByNAEqjuJjOLDyRGA8ZAgmJ4NqCg25IFG1iCzG1S4NnYZPVZnO+2KQ3yJQA4dHpaLsj3vV7tYWQGLKYjIywA3Tw8if9zpfSPRp04kOuvYsLj7fHC1NCj0w7JlQDpfLBrLIZ9S9jLObkMqkaujKrYE19TGg\\\\\\\/C\\\\\\\/lxAh7kz3nXqzwNgOpxyQ4yMXr2z6t1Vd6w+MpCcNd65b2oZlEfZdifUgP1\\\\\\\/b7slNm57O2F1LIdZSA3zXcTGOmz8CSorxybPPB+A\\\\\\\/VxyIJpTFXCAksPVMjJ0sqtKGXIF\\\\\\\/r8C5DQGjFMbOFkjqpU3tiAAS+eeE4X169HHJ6eTYGl3tPBkZ1i5WZXm6OU7C7hC5RvYbrfiMP5TKFPLYpGOVj1LagwaDtAPlqVPfFLC35BRRPjrHzFwqM1Fcr+6jK5NsSt1OpRYk2rDwiQuqv4UoXV1VyG7jqSJcQ6baSMEkxw+8elkxHLvH566LiIWKdcddz9A9YeNNBIkT2\\\\\\\/PUXj6A8rfEZ+DQ0AUIq+XSiaPfI8K2q5yH7j2KCRtx0ZQ3Yn4zCOnLuUUz9sQVvNWm1i92aDxG4XAZx+RkPzQZO4Ux0Aj7cs8xKfzsrRhSzliEn696RquEZLFjrKqxOvR5kG7pfHb2Dck74TtKtZld94PmIqIfJq\\\\\\\/VJ2J5KG23kZySTCuEwrf8fWLb+CP0E+UPDyLfFvI7TD5v9CAbWlDdyxzOiRgdyepNqqWhzi9GTiMQns3w9cXvjZVbC0oPI4TWs8PVHWke5Ki0+dG8TNMjg35wa4EZEif5HqWzBagrsusIQx0\\\\\\\/ok4mc6IgeNdG7Kq4C5jTV4BQb8+6oavb1Hv7UqgysLjv0mIEDnPkKF12eZWNpVPQ8g7QDTW0M6qVhwSLixHTGGD5GcxxSCSmyL9aZ9yXBEsDSUR\\\\\\\/FbJW8ghca2gKrY7LaL8UzFnPdhH2QEUtOs2WmK21CEo9Bfr\\\\\\\/CAJdH+DO9fGqKWuJ5XSKsvnkmwz5OLpmMv6eq4uth3WSMGjYf63VxDPWOC0kNi+h4YOvqUu6gkZbl+uhgvaVtmBbSSdfCXQHOYj5iInczGPzlUsnv2+qtdQN8iz5hi9Qc7wTr2pbPKaVXw1MviTZ5mwqaOp7qRDzmOnkLBn7BT7BYmnrj95ynuSHD1vjJ653RN6vyxwfXaPgTEsPI+mKBVQn5Y2RBa3wWupjZ5aNzHFpA40xAmJB+x\\\\\\\/xt6VOAWLNeKjlpQNnlMCB3oZbzjUBPH52MeP\\\\\\\/+N6x3KZFkyeceXUJeNbIurg0Cb91SZFMwA7Yjc\\\\\\\/zbCs37E3JVqhc+AW83lLZCjd0Tp+HrsCLaa\\\\\\\/KM6R9KfGqPCd0e0dl6xGzvdJK5XO9MO41TuKUAJoWPQVQFyA8We8AH09IYpK4oo0soMBNYWsAwAKhtnHZwpbCg582vaH\\\\\\\/tJ2SiWTI3qiElUAn\\\\\\\/nPLW8z9u\\\\\\\/5VotdY9uBHV535FMLm\\\\\\\/ofTy3BdP+HJQlT5SXIEzdS7xpfMIc5lpKgOZzZGyEEdZpKwgJFhJpCx8sGhEy6kxZU2yRLFKM0u\\\\\\\/76IPrsdXUrJGdod1V8XvNLWFsiSluPem29L6urpTl54NLVCi8+Q3FwCiu0KmGrWA6DUEgsXHcJm9NnI1SDNkWWYmRCTbaRLD2pwSfCDVStigOisIoxuyDVefO19kZ\\\\\\\/V1iXVmMeQ+Xa1\\\\\\\/Q7GYIOj32qB8TlYL5AmAkVPRfMsh8AuENbUkPCQ10TYHGflCy9o003Ku\\\\\\\/eXWVkdjK\\\\\\\/aI0UQDvXzrqZfzxgxojWT51D2AG4MBuFjRT2qROKD0rZ6nxKX4PnU5G3KcDnhP0Dx8CcmQe4K3Ebq4Lp7juXjRs+Fi7y6x4YjGhOKqtCXbcjMEL8vbQvykbF8ceRZlOFqOGtqXtHsFXzdEyeaASNLg0aFDRxiVG2Fs6jnrlmd\\\\\\\/9cbPHmYNL0szGoN24ydrFQ6lmQUERcwdQisgQ7xcyXu\\\\\\\/sWlok29xVG7Vk+iEwLOucZwkIYnn8uDd0FT933oKLBJ7nk0xOgGwFWNMqLpDC9fVPKAX6H23xh09rZgKIINEZtZJ+gjpLRnSNPrr7tq8\\\\\\\/UWlBi5ZYJoSRZ3NMPCuRwIgLzN3LSAOFsak4u+\\\\\\\/TgxBV1e+8JWW+yS9akHk4FC7UlV4gM1s4CIVzdj\\\\\\\/oCQgKNpw7+oaCOmkBjo9gSUi\\\\\\\/uzKZoPWkzUWUyJ6oZtduf+eQT5lNUGqyik9VnUgIia0UDd1ipsXqAeMnQip0tSdjGBcM8bediNT3AJTzRVEMUR+I8tmeq+s5FiLtJ6+rtHsXCx3rwbbUzmGiEa3\\\\\\\/I1zwqtOGqY9bnd9Ly+S2cuhnE+YwHR3qHIOi6Bf7hv8UHFhsQvwvXrke6kuOTLGtLdWobTGoFh71709brdKPDfFg0fvxl7aDT\\\\\\\/2BI\\\\\\\/NJgCPpCxAekHv43nOshEvI7wlYgSX3i\\\\\\\/Wu2atK0jHnsz9Vwuc1tMyuG6bB7EY17cZ+KsPPSWDvIEE20\\\\\\\/3AgMC9sJKkPrMfKTzBksMCSP\\\\\\\/b5PSTNPMCEOew8qJDiKFKSYlFjRSO0kpDcEId+bllOm9y16GyxvHz2egPIYsppt8F50zCUkdb2DxTUByxn8jDqWqp0UoGgw3K8G2zW61dGnKugum584Y6WXpmHHJCAcEQY7xx0Zs6CLat9xX24ON1gvSUCBPypO3zV0Sx86shPbYPfTq99CxHN4b06dAAgLUAKdNM\\\\\\\/Pp\\\\\\\/ymBUNrOfZgwYq18IBAMwU0q\\\\\\\/boh8t61cLS+VRsgml0d62lR3zNnGQ\\\\\\\/2Z1zaedhx1ge0RYCQD0Kq6b8a7ZqnYgX+ge+Ahogy7z2Bp2hE3JqKNl8IHp9h9Ns01SC8n9T4m9fyJQPk8Wdhlc0PEjwhWgbZknpY71Cm5z3qgK0rJAkB1btlwSe6a2ns5A\\\\\\\/nwicZyEbRkdp2GugbtFTVrzvIiXuKFSC0CDV\\\\\\\/BLKxjsgpETvT1CsoVFPUw9A4Edrzy31a33XMrzhPIzCCc5L6Yw7DqriFMPhpErsJM3FS0KaEUWWQnVwZ0EsJBvzhGCXoatRZq619V843GisoNkc0botrCIrw84cpfb7a2T0wkKWktZMrlYCbZ9\\\\\\\/9czn\\\\\\\/oFPYmvMeqW+jB4G9UJ2+uQTnXhUjv4tW4gFisybmdkUx4TLZgf5P1HrEw+RDhN7N5xZihyVupXoQruL\\\\\\\/vFZDO9aLTSbXGJfJrD7IXQRn4rdEw\\\\\\\/D941UvU9yv4KTj5fQkTahjxWxkHF2FujxtuzHfilbvKbquAxG0PZfakJDN4ke0VELLvd99xz3weptYoGVNqzgzFrOY7hHG0dwrkZjkTHxLC\\\\\\\/ajW3TkQ6RipSvecCLDYLbu+3UK6ON5D4lJe5Twq6brKVM\\\\\\\/QSltNIa1THupk9g0qaCGV0X\\\\\\\/bHOHtz2UGh5dX9Q7+1OBhYSuydWjcKsreNwl4vIdJkQrVoTNYGhZz10TunYSjQ9jHE9qnshwosZCUXSZvnlKSJ4eN6jB8GmnQpnH2m99yghEJn1LecfGiO\\\\\\\/eLxsqHOX67znoSi78uoIZhGQEcWHIY2r9Kh5I61yvLLl9toOJdP2E8cET3qaxZVPWeDXo3QIQ5xXQ404CwTLmuuCLpkf7HHuglaFl6ZBZNvuFwgWwf5sgXDEkkquq0yaiRnDl\\\\\\\/QPfggMEyqsOpuoSmTJ43BcZ9wMx380Vj2zZFS08r9VYJgpIm2ItyYWOvb9GNE9ph7zwzUuYa00VqGT9uQ1ctLK16xYm\\\\\\\/ly4d4PgQXbcYmHzV2MJ2s71QZSrfvFZNIEapdRBglRBxaxfyRTViTiZzlrEVj5PFgyehGk1gnZ9fpDl5gjrWC6nNHxVeGOqai\\\\\\\/xJxwh9lr8JEN5IcGIsYOCoKlFUUjezz5s8MREhNiUjzLtDGv7vEfh\\\\\\\/wz\\\\\\\/Do8Uz8EHkK96\\\\\\\/ZAnl9xPPk+FM7r7Ip9iaBRWC+CVQiJ+RSJsDaZvb3MKCEeSDjgjXwvcsXDKvnTmNjXS5z2ib6ESOpi6CVLG0MM5wjgIsRd5yzcl\\\\\\\/qvAr9HfoZM+BdMsvX4C9LO12d50UcWvUK2f34wTp13PNJFAgP+Cqzjj\\\\\\\/GjVoc3sM7BlactPSNmcZ5Y25Qdn7zkNOb+s\\\\\\\/m1qbtH7TjnP2rMjQ1lLJHqvKxNyOB4SkYlO7atCIZItqEoE1za3m6dfl0BXlr2Ffy6hPlDfqr04qCKk4gL2VHrUIHxNAhNFmpSUGW1MEVL\\\\\\\/KhUMs9OHX6snmUeHOGomL+uJXtTpbrMiIWHr\\\\\\\/wbIg3lRdAVQ71m0jtQke6zSOH5Nql3S5c8mNh0QJd5lLAy1Rgm\\\\\\\/QeBaVDe3gRkhDfhBMs98UrEcSLqgTFu+PlIsM7dcG2LWzCmhlkpsTWRbJXPCxvsBjPHQ0beW67ekIwurdtyrVHhC+MW65UsSE3GTQ135cW94oBAl6qsskAMQqr4Z24nHxRyWNmNDasqejSbDWmYoTiuPBZirrfhYmE6NceOZMrkrV9B+ScCFERWwq41xRl3B07Zfqc+O2KWsU9XlJUELaOoX30ttVNBKl+iAvQGm\\\\\\\/ZxHn11qv5TeVzIsEgzY\\\\\\\/QhvwWltzTL4RZC576pH+vQDt0Kd3G9HLzxlXD8RWPibcHCsIHUrH00oBel1NUrQjR8iiy5O5oXSbJBlXB9u4bNZHyH3Ti0VfrtO5NpbEg7PAN1MdaUTnd3+7me8Uk2rkhHuTi9F2WAh0cuJhbWvcySHCUf1191Z2QaWY+6bw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022bjNg6YUKyIOq2V51lPO2SQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226eeb802f9c8955bdfb7fd0ee69280c4545295c9c235fa8aacf8eaba42a722367\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fv9IWplc7ZVFRr0Tq\\\\\\\/Pc5tzS50d\\\\\\\/fI1eTUUoGez6MN4nK9vtkcQqvPKjk2iET6ptDcxNTtXz1VPLYzDou2gzjoxJq9DRcTBebjmTVDsFK8J0A+N8gWPN+U9BFOEW8FCn9r6vEWCZT\\\\\\\/DQYWV98NByMK\\\\\\\/M6I71rje\\\\\\\/ukST1fSyGHUZEQTiATuNNDAeK5fccxcGi4z6h4Cw6GR1VRgSyTRWjlFG44p1JJOS9X19PVwRE4PeNv3zVLHXy62T0+IQFS59BL62ZtWBpHCCgxItUSyq7woGmmzb09dT9ZmkF8yA5CXzZmieKMvQnAN+OrBdfSeo4P\\\\\\\/Urk4xEQjv9Wy33gMZPwzrFdsPKZbldUFbyPAKNr5DmfXJBdbFm4ZfAhtPiS6KxX\\\\\\\/QqD32sW1nu+NguHS1sM5xp8jUuARmKiivpl+bcIBj8Z1AfBNEycrwv7wdtzECNOwJchRNxO2pcRB+hCo+qarnbDAkIeccheUEJ9ykusAZYGc3+nucgNnUHKoDICBnoGbEgVz737tHDQ\\\\\\\/jCzW2f+wbbVVoaloD1A4pPbUxaFoYoMABOU+aYdSjTTnC9HR4uEec+LuM5VJDKzMbig2XVoJ\\\\\\\/PMjEgfMNS6iID2AHhp9YGv6gsOBKf9wxBC9dDrTBaPQilsAN0Zuom2fyz7PVaJUaz88bAyZq7kh4u3ScLD5Q0UudDHWRwqC3PXw4VwBaCmp6wlEePt50Onk5GKtRFhxpZlwtRGOPVz+Q7nhuAuI5HPcF4rwyGAFyc+4+5e0rJJp8Q42CWSmBwoWvyt2LZtd47ZNNd6exqeInd65KBQN1GWjRnCT0nlOw9CwLW\\\\\\\/O\\\\\\\/PSoMRlWoZ7l\\\\\\\/YD\\\\\\\/SmsK0CYB\\\\\\\/8et4ymfYOfuiODvN6QdaEK5z6UKZzDGTpb8+Px1aFQkyrl88iklUwk8Zv0mMC1uFBeckmRz\\\\\\\/ZDj1FDjXGljcYdwTQ2kDePtOk\\\\\\\/sUOQGPeXaZUvzIvqxWHQFC\\\\\\\/xrnV0tVA43nQuSSJa7Ph2y2AsquQ32xGw7WGe3M9Gqh1bQfDzqNlIlsZXKOa2XfLA7q1zNsOlElOXAo76CtR1HORnudNA50WYKP\\\\\\\/9I2tFL9HsBE\\\\\\\/ZTAyI39GWKRCAIlkwjjU6yvPvpDepZ5mexzQSfnkMoYEMzta6b7G4MbIUoOh+Jhy5qgBBKsUeXUQWa3\\\\\\\/kQRxEuWuP6sts4B+jrSxEr4f+Tg\\\\\\\/IdjB7XkvCIRB9pn2O6MTHFM6H0ZPOIRX8tYY0JTo3ojRRPlmHDbyN80KJZEt6q1FLjbmIygOZAqyMxhVQjp\\\\\\\/zBqNmUSCMTKCNhVbXwXjj5qsCN5hOuahDjcp0wk\\\\\\\/1luNxMDEdNjinGBwXMOJ7XXI2pc8bBpYP6wX9EHiI8W\\\\\\\/FkVf46Tbhl69SlmSVmEnoeGxyN3gldSG6jc2Bn2pSiMs20ibU6UhbXF7yMWeNleqgkB7Ij0VHNppov2kxR72iUUOjnv9F1tJ4D\\\\\\\/FqsAIoEOGhIrA4du4qGsIZyKirqBqexf6BEkxqoJX9Ymw7WJa6SLuv+FUOobhFvHGSk4r4WUXVSgl9MbX14KgxzswiR5BsbylyHKKT9EK9srnj3x0euXdpXdepfO4OOvHb1qYCvfRqSZ2gdGqkdOYpuf4nO7HOJX+rogQqYSk3OcLxe9wrFiJ1p9pitDlaaulQ0lcf8GaQEfEVPz1LWN2w2TZztmOSu2PD3Sthxw3dePs29Br8qvt6eqZSQblP7ZVQsLkMbrelT4QzphvH4NKi00LQd0HE2U214pz6LdlECICtc5eUgd5F5W0X9ndwxeemhUz32mL+hnUYQd40+EG7AUxY+DE9fonaGV\\\\\\\/5DbzWvftt7tsA7RzagD93sNsCNPQvtW8YGc1hpP2\\\\\\\/R6CFLC5rJ7rTIoDH5VocZqmvOnbHL1ESEzLima0UVFE7QbM2JESibxo0\\\\\\\/iokxEKEoM6IL0e8VXlWK04sqlYe8gQJPA9W0ZMNq0+soLGzPem5oIGXnd7RqJP4XCnl5qWHbwuh9L1wVlVS5dVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EOCf\\\\\\\/54LzE80O1G64gxq7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ad3d739935e58f97030f3610179cb9213dec507dd3dc10471ce16df75a5b7e3e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460374 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1525" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C75Sy7SKDwJXVlMlOVgmZEKZhGAuTtIDF_0JjkLtEm8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1525" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nedqA-M1AblsPyDUF0uPHdrmcDnluSt-iatoMaGNd-c" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1513" title="Interconnections Between RNA-Processing Pathways Revealed by a Sequencing-Based Genetic Screen for Pre-mRNA Splicing Mutants in Fission Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1535" title="Genomic Signatures of Experimental Adaptation to Antimicrobial Peptides in Staphylococcus aureus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460374" id="node460374" data-pisa="ggg;6/6/1525" data-pisa-master="ggg;g3.116.028779" data-apath="/ggg/6/6/1525.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sharon R. Browning</span>, <span class="highwire-citation-author" data-delta="1">Kelsey Grinde</span>, <span class="highwire-citation-author" data-delta="2">Anna Plantinga</span>, <span class="highwire-citation-author" data-delta="3">Stephanie M. Gogarten</span>, <span class="highwire-citation-author" data-delta="4">Adrienne M. Stilp</span>, <span class="highwire-citation-author" data-delta="5">Robert C. Kaplan</span>, <span class="highwire-citation-author" data-delta="6">M. Larissa Avilés-Santa</span>, <span class="highwire-citation-author" data-delta="7">Brian L. Browning</span> and <span class="highwire-citation-author" data-delta="8">Cathy C. Laurie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1525-1534;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028779 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460374" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sharon R. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BR.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BR.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sguy@uw.edu" class="" data-icon-position="" data-hide-link-title="0">sguy@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kelsey Grinde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BGrinde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grinde%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelsey%2BGrinde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Anna Plantinga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BPlantinga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plantinga%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BPlantinga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephanie M. Gogarten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BM.%2BGogarten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogarten%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BM.%2BGogarten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adrienne M. Stilp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrienne%2BM.%2BStilp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stilp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrienne%2BM.%2BStilp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert C. Kaplan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Epidemiology and Population Health, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BC.%2BKaplan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaplan%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BC.%2BKaplan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. Larissa Avilés-Santa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BLarissa%2BAvil%C3%A9s-Santa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avil%C3%A9s-Santa%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BLarissa%2BAvil%25C3%25A9s-Santa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brian L. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Division of Medical Genetics, Department of Medicine, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BL.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BL.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cathy C. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cathy%2BC.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACathy%2BC.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/6/1525" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460374" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460374/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1525.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460374" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460374/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1525.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460374" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460374/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/6/1525.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460374" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460374/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We estimated local ancestry on the autosomes and X chromosome in a large US-based study of 12,793 Hispanic/Latino individuals using the RFMix method, and we compared different reference panels and approaches to local ancestry estimation on the X chromosome by means of Mendelian inconsistency rates as a proxy for accuracy. We developed a novel and straightforward approach to performing ancestry-specific PCA after finding artifactual behavior in the results from an existing approach. Using the ancestry-specific PCA, we found significant population structure within African, European, and Amerindian ancestries in the Hispanic/Latino individuals in our study. In the African ancestral component of the admixed individuals, individuals whose grandparents were from Central America clustered separately from individuals whose grandparents were from the Caribbean, and also from reference Yoruba and Mandenka West African individuals. In the European component, individuals whose grandparents were from Puerto Rico diverged partially from other background groups. In the Amerindian ancestral component, individuals clustered into multiple different groups depending on the grandparental country of origin. Therefore, local ancestry estimation provides further insight into the complex genetic structure of US Hispanic/Latino populations, which must be properly accounted for in genotype-phenotype association studies. It also provides a basis for admixture mapping and ancestry-specific allele frequency estimation, which are useful in the identification of risk factors for disease.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/hispaniclatino" class="hw-term hw-article-keyword hw-article-keyword-hispanic-latino" rel="nofollow">Hispanic/Latino</a></li><li class="kwd"><a href="/keyword/local-ancestry" class="hw-term hw-article-keyword hw-article-keyword-local-ancestry" rel="nofollow">local ancestry</a></li><li class="kwd"><a href="/keyword/principal-components-analysis" class="hw-term hw-article-keyword hw-article-keyword-principal-components-analysis" rel="nofollow">principal components analysis</a></li></ul><p id="p-3">Hispanic Americans are a significant population group in the US, and comprised 16% of the US population in 2010. Due to the colonial history of the Americas, Hispanics derive their ancestry mostly from European, African, and indigenous American (Amerindian) ancestors, with many Hispanic/Latinos having admixed genomes that are mosaics of ancestry from two or more of these continental sources. Statistical methods have been developed to infer the local ancestry at each point of the genome (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Price <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Maples <em>et al.</em> 2013</a>). The local ancestry at a genomic position describes the continental origin of the individual’s two chromosomes (one from each parent) at that position. For example, each of the two alleles at each variant site in the genome of a Hispanic/Latino individual can be designated as deriving from African, European, or Amerindian ancestors through local ancestry estimation.</p><p id="p-4">Inferred local ancestry can be used for genetic association analysis via admixture mapping, in which one tests for association between the number of copies of an ancestry at a genomic location and the trait of interest (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Patterson <em>et al.</em> 2004</a>). Admixture mapping can be more powerful than SNP association mapping when the causal SNP is not genotyped and is difficult to impute, but has very different frequencies between continental populations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hoggart <em>et al.</em> 2004</a>). Inferred local ancestry can also be used to help interpret results from SNP association mapping, as one can infer the continental origin, or continent-specific allele frequencies, of variants that are significantly associated with the trait of interest (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gravel <em>et al.</em> 2013</a>). An additional application of inferred local ancestry is to better understand the underlying within-continental structure in admixed populations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Moreno-Estrada <em>et al.</em> 2013</a>), which can be used to inform decisions about association analysis, such as the use of population grouping variables.</p><p id="p-5">Previous studies have analyzed within-ancestry population structure using local ancestry of individuals from Mexico, Central and South America, and the Caribbean islands. <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Moreno-Estrada <em>et al.</em> (2013)</a> analyzed local ancestry of 330 individuals from the Caribbean basin (individuals from Cuba, Puerto Rico, Hispaniola, Honduras, and Colombia, and individuals from the Yukpa, Bari, and Warao Amerindian populations). <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gravel <em>et al.</em> (2013)</a> analyzed local ancestry of individuals from the 1000 Genomes Colombian, Mexican-American, and Puerto-Rican populations. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Moreno-Estrada <em>et al.</em> (2014)</a> analyzed local ancestry in over 1000 Mexicans. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Homburger <em>et al.</em> (2015)</a> analyzed local ancestry of 437 individuals from five South American countries (Colombia, Ecuador, Peru, Chile, and Argentina).</p><p id="p-6">A previous large study of Latinos in the US, derived from the 23andMe database, calculated local ancestry for over 8000 Latinos (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bryc <em>et al.</em> 2015</a>). The local ancestry estimates were used to calculate global ancestry proportions and to study ancestry tract lengths, but were not used to investigate within-ancestry population structure.</p><p id="p-7">The study reported here differs from the previous studies of admixed Hispanics/Latinos in that it focuses on within-ancestry structure of individuals living in the US and is much larger than previous studies. It includes genetic data on over 12,000 self-identified Hispanic/Latino individuals, living in four major cities across the US, who are participants in the Hispanic Community Health Study/Study of Latinos (HCHS/SOL). We previously analyzed the ‘global’ ancestry of individuals in this cohort, estimating overall proportions of continental ancestries and detecting variation among six self-identified background groups (Cuban, Dominican, Puerto Rican, Mexican, Central, and South American) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Conomos <em>et al.</em> 2016</a>). Here, we extend these analyses by utilizing local ancestry estimates to investigate within-continental ancestries of Hispanic/Latino individuals in the US. We also provide evaluation of methodological issues in local ancestry estimation, such as the selection of reference individuals, treatment of the X chromosome, and ancestry-specific principal components analysis (PCA).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">The HCHS/SOL (<a href="http://www.cscc.unc.edu/hchs/">http://www.cscc.unc.edu/hchs/</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lavange <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Sorlie <em>et al.</em> 2010</a>) is a community-based cohort study of 16,415 self-identified Hispanic/Latino persons aged 18–74 years old, from randomly selected households in four US field centers (Chicago, IL; Miami, FL; Bronx, NY; San Diego, CA) with baseline examination (2008–2011) and yearly telephone follow-up assessment for at least 3 yr. The HCHS/SOL cohort includes participants who self-identified as having Hispanic/Latino background, the largest groups being Central American (n = 1730), Cuban (n = 2348), Dominican (n = 1460), Mexican (n = 6471), Puerto-Rican (n = 2728), and South American (n = 1068). The goals of the HCHS/SOL are to describe the prevalence of risk and protective factors for chronic conditions (<em>e.g.</em>, cardiovascular disease (CVD), diabetes, and pulmonary disease), and to quantify all-cause mortality, fatal and nonfatal CVD and pulmonary disease, and pulmonary disease exacerbation over time. The HCHS/SOL study was approved by institutional review boards at participating institutions, and written informed consent was obtained from all participants. The individuals were genotyped on an Illumina Omni 2.5M array with additional custom content, and the genotype and phenotype data are posted on dbGaP (accession numbers phs000880.v1.p1 and phs000810.v1.p1). We chose not to investigate Asian ancestry in this study, because the overall proportion of Asian ancestry in the sample is very low. After quality control and removal of 19 outlier individuals with significant Asian ancestry (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Conomos <em>et al.</em> 2016</a>), our analyses included 12,774 individuals who consented to genetic studies.</p><p id="p-9">Inference of local ancestry requires the use of reference panels representing the ancestral populations. We considered two reference panels. The first was derived from the Human Genome Diversity Project (HGDP) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2008</a>) and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">The 1000 Genomes Project Consortium (2012)</a> (henceforth referred to as the HGDP reference). The second was derived from data published in <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Reich <em>et al.</em> (2012)</a> combined with data from the 1000 Genomes Project (henceforth referred to as the Reich2012 reference).</p><p id="p-10">For the HGDP reference, we first selected individuals from populations in the HGDP and 1000 Genomes data with European, West African, Amerindian, or East Asian ancestry. These collections are broadly consented for research and publicly available. We ran an unsupervised ADMIXTURE analysis with K = 4 populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>), and retained individuals with at least 90% estimated ancestry from one of the inferred ancestral populations, excluding East Asia. This resulted in 195 West Africans (from Nigeria, Senegal, and Barbados), 63 Amerindians (from Peru, Mexico, Ecuador, Brazil, and Colombia), and 527 Europeans (from Spain, Italy, the British Isles, France, and Utah). The HGDP individuals were genotyped using the Illumina HumanHap650Y array, and the 1000 Genomes individuals were genotyped on the Illumina Omni2.5M array. After intersection with the HCHS/SOL SNPs, 419,645 SNPs remained.</p><p id="p-11">The Reich2012 reference included samples from HGDP, HapMap3, 1000 Genomes, and additional Amerindians. We included only individuals consented for both population genetics and health-related research. The genetic data were processed as for the HGDP reference, resulting in 198 West Africans, 154 Amerindians, and 516 Europeans. This reference set has a larger number of Amerindian individuals than the HGDP reference, but a smaller number of SNPs (236,736) after intersection with HCHS/SOL SNPs, because many of the reference individuals were genotyped with an Affymetrix array that had lower overlap with the HCHS/SOL Illumina array.</p><p id="p-12">For local ancestry inference on the autosomes, we phased the HCHS/SOL data combined with the reference panels for the intersection SNPs using Beagle 4.0 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Browning and Browning 2007</a>, <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a>). Although the HCHS/SOL data include individuals of known relationship, we did not use pedigree information in the phasing. We inferred local ancestry using RFMix (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Maples <em>et al.</em> 2013</a>, <a href="https://sites.google.com/site/rfmixlocalancestryinference/">https://sites.google.com/site/rfmixlocalancestryinference/</a>) version 1.5.4 with the PopPhased option and a minimum node size of 5, as recommended in the documentation. We did not use the RFMix EM option as it was not computationally feasible on a data set of this size.</p><p id="p-13">RFMix does not provide a specific chromosome X setting. There are several factors to consider when choosing appropriate settings for X. Males are haploid for the nonpseudoautosomal regions of X, and thus their data on these regions of X has no phase uncertainty, whereas females are diploid for X and phase can be statistically inferred but not without error. It is common to code males as homozygous diploid on X; this is undesirable in the reference panel as it makes the observed male haplotypes look more frequent than they really are. In contrast, coding admixed males as homozygous should provide good results when using RFMix without the EM option because inference on one admixed haplotype does not affect inference on another admixed haplotype. In order to account for these factors with the available RFMix options, we artificially paired reference males into phased pseudodiploid individuals within each continental reference group, discarding the final male if the group had an odd number of males. RFMix does not attempt to rephase the reference individuals, so phase information is not lost by the pairing. We then either (Option 1) performed separate analyses for admixed males and admixed females, with admixed haploid males treated as perfectly phased and accounting for phase uncertainty in the admixed females, or (Option 2) analyzed all admixed individuals together, coding males as homozygous diploid and accounting for phase uncertainty in all the individuals. Option 1 allows analysis of the two sexes in parallel, which speeds computation, whereas Option 2 is easier to implement. We also investigated (Option 3) coding all males (reference and admixed) as homozygous. In Option 1 we used the TrioPhased option in RFMix to analyze the males, which makes use of the known phase of these individuals. For all other analyses we used the PopPhased option in RFMix, which accounts for phase uncertainty. Before running RFMix with any of these options, we phased chromosome X for all the admixed and reference individuals together using Beagle 4.0 (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Browning and Browning 2007</a>) with males coded as homozygous, which imputes missing data in the males as well as phasing the females.</p><p id="p-14">It can be helpful to plot principal components (PCs) corresponding to the parts of individuals’ genomes that are derived from a specific continental ancestry (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Moreno-Estrada <em>et al.</em> 2013</a>). In order to do this, one can mask (set to missing) alleles corresponding to haplotypes inferred not to be of the specified ancestry. Because the missing pattern applies to haplotypes rather than genotypes, it is natural to apply PCs on a per-haplotype basis. Multidimensional scaling (MDS) with Euclidean distance produces PCs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cox and Cox 1994</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gower 1966</a>). We thus applied MDS with Euclidean distance (treating alleles as 0’s and 1’s) to the masked haplotype data using the dist() and cmdscale() functions in R (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">R Core Team, 2014</a>), with masked positions treated as missing data. Missing data induced by the masking are accounted for when the distances between two haplotypes are calculated. Only positions at which both haplotypes are nonmissing are used in the distance calculation. If two haplotypes share no nonmasked positions, we set their distance to the mean distance between all pairs. Our MDS-based approach to ancestry-specific PCA is relatively straightforward. We provide an outline of the code used to generate results in this paper, including running RFMIX, masking haplotypes, and calculating ancestry-specific PCs (<a href="http://faculty.washington.edu/sguy/local_ancestry_pipeline/">http://faculty.washington.edu/sguy/local_ancestry_pipeline/</a>).</p><p id="p-15">We removed 2168 close relatives (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Conomos <em>et al.</em> 2016</a>) from the ancestry-specific PC analyses (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). For a given continental ancestry, we used only individuals with at least 50% ancestry derived from that continent (calculated from the local ancestry calls), because individuals with low levels of ancestry will have little genetic data after masking and thus have noisy PC values. To avoid spurious PCs arising from genomic regions with high linkage disequilibrium, and to reduce computation time in calculating interhaplotype distances, we thinned markers to 10,000 SNPs across the autosomes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Processes used to obtain ancestry-specific PCA for Figure 5 and Figure 6. Data analyses and filtering are shown in circles/ovals, while data are shown in rectangles. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; MDS, multidimensional scaling; PCA, PC analysis; PCs, principal components; Reich2012, reference panel derived from data published in Reich et al. (2012) combined with data from the 1000 Genomes Project." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-169889752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Processes used to obtain ancestry-specific PCA for Figure 5 and Figure 6. Data analyses and filtering are shown in circles/ovals, while data are shown in rectangles. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; MDS, multidimensional scaling; PCA, PC analysis; PCs, principal components; Reich2012, reference panel derived from data published in Reich &lt;em&gt;et al.&lt;/em&gt; (2012) combined with data from the 1000 Genomes Project.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1525/F1.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/6/1525/F1.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Processes used to obtain ancestry-specific PCA for <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. Data analyses and filtering are shown in circles/ovals, while data are shown in rectangles. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; MDS, multidimensional scaling; PCA, PC analysis; PCs, principal components; Reich2012, reference panel derived from data published in <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Reich <em>et al.</em> (2012)</a> combined with data from the 1000 Genomes Project.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">We ran two sets of ancestry-specific PC analyses. We performed a single worldwide analysis that includes all continental ancestries (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The main purpose of this analysis is to check that the results from the local ancestry calling are reasonable. The African part of the admixed individuals’ genomes should cluster with the African reference individuals, for example. In order to investigate fine-scale structure more closely, we performed a second set of analyses comprised of three within-continent analyses (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). These analyses use reference individuals from a single continental group (African, Amerindian, or European), along with the parts of the admixed individuals’ genomes that are called as having ancestry from that continental group. In this fine-scale analysis, we only include an admixed individual if his/her four grandparents are known to have come from the same country, and we label the individual by that country in the results.</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-18">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><p id="p-19"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> compares the autosomal calls from using the HGDP reference with those from using the Reich2012 reference. There is very little difference in the calls from the two analyses, with 98.7% of the calls being identical. Calls made from the HGDP reference are slightly more likely to be European and slightly less likely to be Amerindian than those from the Reich2012 reference, which may be due to the lower number of Amerindian reference individuals in the HGDP reference.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468373/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468373/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Proportion of autosomal local ancestry calls</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">We looked at Mendelian inconsistency rates (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baran <em>et al.</em> 2012</a>) to compare the accuracy of the inferred local ancestry from analysis with the HGDP reference and the Reich2012 reference. Mendelian inconsistency rates have previously been used to compare methods (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baran <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Pasaniuc <em>et al.</em> 2013</a>) and to find regions of the genome with high rates of error in local ancestry calls (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Pasaniuc <em>et al.</em> 2013</a>). We say that a parent–offspring pair or mother–father–child trio are Mendelian inconsistent at a position if their ancestry calls at the locus are not consistent with the laws of Mendelian inheritance. For example, if a mother is called as having a chromosome of African origin and a chromosome of European origin at a locus, while the child is called as having two Amerindian chromosomes, then the inferred local ancestry is Mendelian inconsistent. Due to the household-based sampling design of the HCHS/SOL study, there are large numbers of parent–offspring pairs and trios. We analyzed 174 pairs and 203 trios. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows Mendelian inconsistency rates for the autosomal data. The rates of Mendelian inconsistency are lower in pairs than in trios because errors in ancestry calls are more difficult to detect when one parent’s data are missing. Overall, the rates of inconsistency are low, and the rates are lower with calls from the HGDP reference than with calls from the Reich2012 reference.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468360/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468360/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mendelian inconsistency rates on the autosomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">The main output from RFMix is most-likely ancestry calls, however, the program can also produce posterior probabilities that indicate how certain the algorithm is about its call (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Maples <em>et al.</em> 2013</a>). For each position and each haplotype in the individual there are three posterior probabilities for the three possible local ancestry calls. For example, the probabilities might be 0.99 for African, 0.001 for European, and 0.009 for Amerindian, indicating that the haplotype is almost certainly of African ancestry at this position. We recorded the highest posterior probability for each haplotype at each position (0.99 in the example), and report the autosomal averages in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Regardless of which ancestry has the highest posterior probability, the average highest posterior probability is close to 1, but is slightly higher for the HGDP reference than for the Reich2012 reference. Thus, both Mendelian inconsistency rates and average posterior probabilities indicate that the two reference sets are giving high accuracy but that the HGDP reference gives slightly better results. This is interesting as it indicates that, in this particular setting, the advantage of having a higher number of SNPs in the RFMix analysis (419,645 <em>vs.</em> 236,736 genome-wide) outweighs the disadvantage of having a reduced number of reference individuals (63 <em>vs.</em> 154 Amerindian reference individuals).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468417/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468417/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Average highest posterior probabilities</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">We used the HGDP reference in calling local ancestry on chromosome X. The Mendelian inconsistency rates for chromosome X in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> demonstrate that Options 1 (separate analysis of haploid admixed males and diploid admixed females) and 2 (admixed males coded as homozygous) have similar accuracy, while Option 3 (admixed and reference males coded as homozygous) is only slightly inferior. The chromosome X mother–daughter pair rates (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>) are similar to the autosomal pair rates (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), indicating that accuracy on X with these options is similar to autosomal accuracy. Local ancestry calls from Options 1 and 2 were 99.8% identical, while results from Option 3 were 99.6% identical to those from Options 1 or 2. We also calculated average highest posterior probabilities for the X chromosome. On this chromosome, results on males are not comparable across options due to the different diploid and haploid coding schemes, so we only present results from females (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Options 1 and 2 are almost identical in terms of average highest posterior probabilities for females, which is to be expected since these two options treat females identically. Option 3 has higher posterior probabilities, which may be because the RFMix program thinks it has a larger number of reference haplotypes due to duplication of male haplotypes and thus is, erroneously, more confident in its results. This illustrates one of the pitfalls of using posterior probabilities to assess accuracy, which is that they may be miscalibrated. We chose to proceed with Option 1 for future analyses of these data.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468347/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468347/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Mendelian inconsistency rates on chromosome X</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468392/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468392/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Average highest posterior probabilities in females on X</span><div class="sb-div caption-clear"></div></div></div><p id="p-28"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows average autosome-wide local ancestry call proportions for each unrelated admixed individual against global autosomal ancestry estimated using a supervised ADMIXTURE analysis (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>). ADMIXTURE was run with K = 3 populations, using reference individuals from the 1000 Genomes project and the HGDP. We see that RFMix is tending to call higher proportions of European ancestry and lower proportions of African and Amerindian ancestry. This may be because some of the African and Amerindian reference individuals have small amounts of European admixture, which biases the estimates of continental allele frequencies. As a result, ADMIXTURE may be assigning some of the European ancestry to the African and Amerindian components. RFMix and other local ancestry calling methods should be less susceptible to this issue because they look for long segments of continuous continental ancestry. Although a small portion of a European segment in an HCHS/SOL individual may match a haplotype seen in an Amerindian reference individual due to European ancestry in that individual, the RFMix algorithm will tend to call the segment as European because the overall evidence from the larger region is for European ancestry.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Autosome-wide average ancestry proportions from RFMIX calls (&#x201C;local ancestry&#x201D;) vs. ADMIXTURE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-169889752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Autosome-wide average ancestry proportions from RFMIX calls (&#x201C;local ancestry&#x201D;) &lt;em&gt;vs.&lt;/em&gt; ADMIXTURE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1525/F2.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/6/1525/F2.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468495" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Autosome-wide average ancestry proportions from RFMIX calls (“local ancestry”) <em>vs.</em> ADMIXTURE.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">A previous approach to obtaining ancestry-specific PCs was to compute the subspace spanned by the first <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1525/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1525/embed/mml-math-1.gif"/></noscript></span></span> PCs by finding a matrix decomposition that minimizes the reconstruction error (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Johnson <em>et al.</em> 2011</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Moreno-Estrada <em>et al.</em> 2013</a>). However, we found that when using the PCAmask program implementing this approach (<a href="https://sites.google.com/site/pcamask/home">https://sites.google.com/site/pcamask/home</a>), we obtained artifactual separation of clusters between reference and admixed individuals. In order to investigate this behavior, for each continental ancestry, we ran PCAmask analyses using the HGDP reference data and genotypes of 100 unrelated HCHS/SOL individuals with at least 50% estimated ancestry from the specified continent, and masked genotypes at positions not called to be of the specified continental ancestry. In a second PCAmask analysis of the same individuals, we chose three reference individuals to move from the reference file to the admixed file, and added corresponding entries to the masking file (with no masking of these individuals). Thus, the input data of the two analyses was identical in content. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows that the ancestry-specific components of the admixed individuals cluster separately from the corresponding reference individuals in the PC plots, and that moving individuals from the reference file into the admixed file causes these individuals to move from the reference cluster to the admixed cluster, despite no change in the analyzed genotypes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example illustrating artifactual results from PCAmask. We performed separate analyses with PCAmask for each continent (column in the figure), including the appropriate HGDP reference individuals and 100 admixed individuals with at least 50% estimated ancestry from the specified continent. Alleles not estimated to be from the specified continent were masked out. The top row shows the original PCAmask analysis, while in the bottom row we reran PCAmask after moving three reference individuals (six haplotypes) from the reference file to the admixed file. The six selected reference haplotypes are shown as large black dots in both rows. PC, principal component." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-169889752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example illustrating artifactual results from PCAmask. We performed separate analyses with PCAmask for each continent (column in the figure), including the appropriate HGDP reference individuals and 100 admixed individuals with at least 50% estimated ancestry from the specified continent. Alleles not estimated to be from the specified continent were masked out. The top row shows the original PCAmask analysis, while in the bottom row we reran PCAmask after moving three reference individuals (six haplotypes) from the reference file to the admixed file. The six selected reference haplotypes are shown as large black dots in both rows. PC, principal component.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1525/F3.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/6/1525/F3.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468370" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Example illustrating artifactual results from PCAmask. We performed separate analyses with PCAmask for each continent (column in the figure), including the appropriate HGDP reference individuals and 100 admixed individuals with at least 50% estimated ancestry from the specified continent. Alleles not estimated to be from the specified continent were masked out. The top row shows the original PCAmask analysis, while in the bottom row we reran PCAmask after moving three reference individuals (six haplotypes) from the reference file to the admixed file. The six selected reference haplotypes are shown as large black dots in both rows. PC, principal component.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We investigated the effect of masking in our MDS approach by setting to missing a proportion of the genotype data in reference individuals. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the result of masking 50% of the genotype data in selected individuals. These individuals cluster with the nonmasked individuals, demonstrating that the masking does not lead to artifactual cluster separation. Not surprisingly, because of the loss of information by masking, the masked individuals cluster slightly less tightly than the nonmasked individuals. Similarly, in the full data, we see that the ancestry-specific masked haplotypes of the admixed individuals are clustering with the corresponding reference haplotypes in all but a couple of cases, which are discussed in more detail below (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of masking in the MDS-based PCA. We used MDS to calculate principal components for the reference individuals in the intersection of the Reich et al. (2012) and HGDP panels. Half the individuals (shown in cyan) had no missing data. In the other individuals (shown in magenta), we masked (set to missing) a random 50% of the alleles on each of the two haplotypes. HGDP, Human Genome Diversity Project; MDS, multidimensional scaling; PCA, PC analysis; PCs, principal components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-169889752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of masking in the MDS-based PCA. We used MDS to calculate principal components for the reference individuals in the intersection of the Reich &lt;em&gt;et al.&lt;/em&gt; (2012) and HGDP panels. Half the individuals (shown in cyan) had no missing data. In the other individuals (shown in magenta), we masked (set to missing) a random 50% of the alleles on each of the two haplotypes. HGDP, Human Genome Diversity Project; MDS, multidimensional scaling; PCA, PC analysis; PCs, principal components.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1525/F4.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/6/1525/F4.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468252" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Effect of masking in the MDS-based PCA. We used MDS to calculate principal components for the reference individuals in the intersection of the <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Reich <em>et al.</em> (2012)</a> and HGDP panels. Half the individuals (shown in cyan) had no missing data. In the other individuals (shown in magenta), we masked (set to missing) a random 50% of the alleles on each of the two haplotypes. HGDP, Human Genome Diversity Project; MDS, multidimensional scaling; PCA, PC analysis; PCs, principal components.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Worldwide ancestry-specific PCs. The top row shows PCs 1 and 2, while the bottom row shows PCs 3 and 4, as outlined in Figure 1B. The first three columns show the three ancestral components of the admixed individuals, while the fourth column shows the reference individuals. The values in the four columns were calculated together and may be compared, but are shown in separate plots for clarity. The two haplotypes from each individual are connected by a line. The legend in the first three columns indicates color-coding for the self-identified background groups of HCHS/SOL participants, while that in the fourth column indicates regional origin of reference individuals. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; PCs, principal components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-169889752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Worldwide ancestry-specific PCs. The top row shows PCs 1 and 2, while the bottom row shows PCs 3 and 4, as outlined in Figure 1B. The first three columns show the three ancestral components of the admixed individuals, while the fourth column shows the reference individuals. The values in the four columns were calculated together and may be compared, but are shown in separate plots for clarity. The two haplotypes from each individual are connected by a line. The legend in the first three columns indicates color-coding for the self-identified background groups of HCHS/SOL participants, while that in the fourth column indicates regional origin of reference individuals. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; PCs, principal components.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1525/F5.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/6/1525/F5.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468238" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Worldwide ancestry-specific PCs. The top row shows PCs 1 and 2, while the bottom row shows PCs 3 and 4, as outlined in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>. The first three columns show the three ancestral components of the admixed individuals, while the fourth column shows the reference individuals. The values in the four columns were calculated together and may be compared, but are shown in separate plots for clarity. The two haplotypes from each individual are connected by a line. The legend in the first three columns indicates color-coding for the self-identified background groups of HCHS/SOL participants, while that in the fourth column indicates regional origin of reference individuals. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; PCs, principal components.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ancestry-specific PCs for single ancestral continents. Each column of the top two rows of the figure clusters ancestry from one ancestral continental group (African, Amerindian, or European), with PCs calculated as indicated in Figure 1C. The top row highlights the HCHS/SOL individuals labeled by the country of origin of their grandparents (with reference individuals in the background in gray), while the middle row highlights the reference individuals (with HCHS/SOL individuals in the background in gray). The bottom row of the figure gives maps showing the locations of the populations. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; PCs, principal components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-169889752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Ancestry-specific PCs for single ancestral continents. Each column of the top two rows of the figure clusters ancestry from one ancestral continental group (African, Amerindian, or European), with PCs calculated as indicated in Figure 1C. The top row highlights the HCHS/SOL individuals labeled by the country of origin of their grandparents (with reference individuals in the background in gray), while the middle row highlights the reference individuals (with HCHS/SOL individuals in the background in gray). The bottom row of the figure gives maps showing the locations of the populations. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; PCs, principal components.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1525/F6.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/6/1525/F6.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1525/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468325" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">Ancestry-specific PCs for single ancestral continents. Each column of the top two rows of the figure clusters ancestry from one ancestral continental group (African, Amerindian, or European), with PCs calculated as indicated in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>. The top row highlights the HCHS/SOL individuals labeled by the country of origin of their grandparents (with reference individuals in the background in gray), while the middle row highlights the reference individuals (with HCHS/SOL individuals in the background in gray). The bottom row of the figure gives maps showing the locations of the populations. HCHS/SOL, Hispanic Community Health Study/Study of Latinos; PCs, principal components.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">For the ancestry-specific PC analyses shown in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>, we used the reference individuals from the Reich2012 data set, because this data set covers a broader range of reference populations. We used the local ancestry calls derived from the RFMix run using the Reich2012 reference, which are almost identical to those from the RFMix run using the HGDP reference (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). There were 513 unrelated admixed individuals with at least 50% African ancestry, 1997 with at least 50% Amerindian ancestry, and 5735 with at least 50% European ancestry.</p><p id="p-37">In <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, we plot ancestry-specific PCs from a worldwide analysis. The ancestry-specific parts of admixed individuals’ genomes cluster closely with the reference individuals from the same ancestry on PCs 1 and 2. The tight, well separated clusters for genomic segments identified with each of the three continental groups, and the lack of intermediates, indicates that the procedures used here are accurately delineating the boundaries of ancestral segments in multi-way admixed individuals. On PC 3, the Amerindian parts of admixed individuals’ genomes cluster with reference individuals from the American geographic region (Mexico, Central America, or South America) that matches the admixed individuals’ self-identified backgrounds. PC 4 shows that the European component partially differentiates individuals of Puerto Rican background from those of the other background groups. This PC 4 axis is not related to north–south geographic variation within the European reference population, and thus may primarily represent drift due to a founder effect.</p><p id="p-38">To look more closely for within-continent ancestry structure, we calculated PCs for each continental ancestry separately, without the reference groups from the other two continents. We calculated four PCs per continental ancestry, but only show the first two in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> as the third and fourth components did not further illuminate population structure in the HCHS/SOL individuals.</p><p id="p-39">The left panels of <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a> show PCs for African ancestry. The individuals from the Caribbean (primarily Dominican Republic and Cuba as country of grandparental origin) cluster with the reference Yoruba in Ibadan, Nigeria and the African Caribbeans in Barbados, but less so with the Mandenka. In contrast, the individuals from Central America (Honduras and Guatemala as country of grandparental origin) do not cluster closely with the African ancestry from the Caribbean individuals, or with the Yoruba or Mandenka. The variation is on the same axis (PC1) that separates Yoruba from Mandenka, so may represent an ancestry contribution from other populations within Africa. The populations of Honduras and Guatemala include the Garífuna people. Previous mitochondrial analysis of the Garífuna population suggests primarily West African ancestry, with possibly some Mozambiquan ancestry, and a population bottleneck (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Salas <em>et al.</em> 2005</a>). Thus, the separation of the Central American African component may be due to ancestry from African populations that are distinct from the Yoruba and Mandenka, or genetic drift due to a population bottleneck, or a combination of these two factors.</p><p id="p-40">The center panels of <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a> show PCs for Amerindian ancestry. The individuals with grandparents from Central America tend to cluster with the reference group closest to Central America (the Maya). The individuals with grandparents from Mexico cluster with the other Mexican reference individuals (Pima, Tepehuano, and Zapotec) and the Maya. The individuals with grandparents from a South American country cluster with the South American reference individuals, with additional structure present. The individuals with grandparents from Peru cluster with the reference individuals from Peru and the Quechua individuals. The individuals with grandparents from Colombia cluster with the reference individuals from Colombia and Brazil. The individuals with grandparents from Ecuador tend to cluster with the Peruvian and Colombian reference individuals. Previous studies have also found population structure within the Amerindian ancestral component of admixed individuals from the Caribbean, Mexico, and South America (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Moreno-Estrada <em>et al.</em> 2014</a>, <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">2013</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Gravel <em>et al.</em> 2013</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Homburger <em>et al.</em> 2015</a>).</p><p id="p-41">The right panels of <a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6</a> show PCs for European ancestry. As in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, Puerto Rican European ancestry diverges from that of the other admixed groups, and does so on an axis of variation (here PC1) that does not reflect the north–south geographical gradient in the reference European samples.</p></div><div class="section discussion" id="sec-4"><h2 class="">Discussion</h2><p id="p-42">We used RFMix to infer three-way (African, European, and Amerindian) ancestry in the HCHS/SOL individuals. Our local ancestry calls are being used for admixture mapping with the HCHS/SOL data (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Schick <em>et al.</em> 2016</a>). We will also use the local ancestry calls to calculate ancestry-specific allele frequency estimates, which will be used to investigate the ancestral origins of variants associated with disease-related phenotypes in the HCHS/SOL data in future work.</p><p id="p-43">The unprecedented size of our study, with over 12,000 admixed Hispanic/Latino individuals, allowed us to stress-test RFMix and to investigate within-ancestry population structure in finer detail than previous studies. Due to the large numbers of parent–offspring relationships in our sample, we were able to calculate Mendelian inconsistency rates, which allowed us to compare approaches and to confirm that RFMix is giving consistent results.</p><p id="p-44">Due to the colonial history of the Americas, it is challenging to obtain reference individuals of nonadmixed Amerindian ancestry. We compared two reference panels, one with more markers but fewer Amerindian reference individuals, and the other with more Amerindian reference individuals but fewer markers overlapping with those in the HCHS/SOL data. We found that the former gave slightly superior results in our data, as measured by Mendelian inconsistency rates and posterior local ancestry probabilities. Further studies will be needed to see the extent to which this result generalizes to other data sets and to the use of other local ancestry calling methods. Computational constraints prevented us from using RFMix’s EM option, which augments the reference panel with the admixed individuals after an initial round of local ancestry calling; the use of an EM-type option in other studies might further favor higher marker density over higher numbers of reference individuals.</p><p id="p-45">Like many genetics software tools, RFMix was designed only for autosomal data. We investigated three approaches to analyzing the X chromosome with RFMix. We assessed the results using Mendelian inconsistency rates. The two best approaches involved pairing haploid males into pseudodiploid individuals in the reference panel in order to avoid the double-counting that would occur if these individuals were coded as homozygous diploid. However, coding reference males as homozygous resulted in only a small loss of accuracy.</p><p id="p-46">We calculated global ancestry proportions from the autosomal local ancestry calls, and compared these to global ancestry calls obtained directly from the genotypes using ADMIXTURE. We found that the ADMIXTURE results tended to have lower proportions of European ancestry and higher proportions of African and Amerindian ancestry. It seems likely that local ancestry calls, which are based on large chromosomal segments rather than individuals’ SNPs, are less susceptible to bias due to low levels of admixture in the reference individuals. Thus, global ancestry proportions based on local ancestry calls are likely to be superior to direct estimates of global ancestry using methods such as ADMIXTURE.</p><p id="p-47">Ancestry-specific PCA, calculated using inferred local ancestry, can be used to examine the fine-scale structure of the data without confounding by differing ancestry proportions between individuals. For example, in the usual ‘global’ PCA of autosomal SNP genotypes, we found a cluster of HCHS/SOL individuals with high African ancestry and Central American background, which separated from individuals with high African ancestry and Caribbean background [the small yellow cluster in the lower left corner of Figure 3a of <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Conomos <em>et al.</em> (2016)</a>]. In this previous global analysis, it was not clear whether the Central American cluster separated from the Caribbean cluster because of their different Amerindian ancestries, or because their African origins are different. The local ancestry analysis presented here strongly favors the latter interpretation. Similarly, the global PCA showed that individuals of Puerto Rican background are well-separated from other background groups. The continental ancestry-specific analysis presented here suggests that differences in European ancestry, likely due to a founder effect, contribute to this separation, which was not apparent from the global analysis.</p><p id="p-48">We first attempted to use PCAmask (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Moreno-Estrada <em>et al.</em> 2013</a>) to perform ancestry-specific PCA, but found that it produced artifactual separation of reference and ancestry-specific admixed individuals. <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Moreno-Estrada <em>et al.</em> (2013)</a> infer divergence between the European ancestry component of Caribbean individuals and current day Iberian individuals based on results from PCAmask. Our study shows that separation between reference and admixed individuals in results from PCAmask can be artifactual. We then developed a straight-forward approach for ancestry-specific PCA using existing statistical software tools. Based on the results obtained on our data, this approach does not seem to be susceptible to artifactual separation of reference and admixed individuals.</p><p id="p-49">Our ancestry-specific PC analysis of the HCHS/SOL data revealed clear population structure within each continental ancestry across the individuals in the HCHS/SOL study. These results provide support for the HCHS/SOL analysis plan to separate individuals into genetic analysis groups (Cuban, Dominican, Puerto Rican, Mexican, Central, and South American) based on self-identified background and PCA, and to use these groups in association analyses (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Conomos <em>et al.</em> 2016</a>). Due to the within-continental structure, some causal genetic variants may be specific to a certain subgroup, and appropriate use of the group variables may increase power to find associations with such variants. Furthermore, allele frequencies calculated within ancestral genomic segments can identify the origins of risk alleles and guide the selection of replication cohorts. In interpreting the results of our analyses, it is important to note that the study individuals came from just four centers within the US (Chicago, IL; Miami, FL; Bronx, NY; San Diego, CA), and are thus not necessarily representative of the US Hispanic/Latino population as a whole.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">The authors thank the staff and participants of the Hispanic Community Health Study/Study of Latinos (HCHS/SOL) for their important contributions, as well as Andrés Ruiz-Linares, David Reich, and other authors of <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Reich <em>et al.</em> (2012)</a> for providing reference data. The HCHS/SOL was carried out as a collaborative study supported by contracts from the National Heart, Lung, and Blood Institute (NHLBI) to the University of North Carolina (N01-HC65233), University of Miami (N01-HC65234), Albert Einstein College of Medicine (N01-HC65235), Northwestern University (N01-HC65236), and San Diego State University (N01-HC65237). The following Institutes/Centers/Offices contribute to the HCHS/SOL through a transfer of funds to the NHLBI: National Institute on Minority Health and Health Disparities, National Institute on Deafness and Other Communication Disorders, National Institute of Dental and Craniofacial Research (NIDCR), National Institute of Diabetes and Digestive and Kidney Diseases, National Institute of Neurological Disorders and Stroke, and the National Institutes of Health (NIH) Institution-Office of Dietary Supplements. The Genetic Analysis Center at the University of Washington was supported by NHLBI and NIDCR contracts (HHSN268201300005C AM03 and MOD03). S.R.B. and B.L.B. were supported in part by NIH grant GM099568. K.G. was supported in part by NIH training grant T32GM081062. A.P. was supported by the National Science Foundation Graduate Research Fellowship Program grant DGE-1256082. This manuscript has been reviewed by the HCHS/SOL Publications Committee for scientific content and consistency of data interpretation with previous HCHS/SOL publications.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-51"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-11"><span class="received-label">Received </span>January 11, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-17"><span class="accepted-label">Accepted </span>March 17, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Browning <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.2" data-doi="10.1093/bioinformatics/bts144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate inference of local ancestry in Latino populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1359</span>–<span class="cit-lpage">1367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts144%26rft_id%253Dinfo%253Apmid%252F22495753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTAvMTM1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.3" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.4" data-doi="10.1016/j.ajhg.2014.11.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macpherson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mountain</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genetic ancestry of African Americans, Latinos, and European Americans across the United States.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">37</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.11.010%26rft_id%253Dinfo%253Apmid%252F25529636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25529636&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.5" data-doi="10.1016/j.ajhg.2015.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conomos</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stilp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHugh</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic Diversity and Association Studies in U.S. Hispanic/Latino Populations: Applications in the Hispanic Community Health Study/Study of Latinos.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.12.001%26rft_id%253Dinfo%253Apmid%252F26748518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748518&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1525.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">M. A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source"><em>Multidimensional Scaling</em></span>, <span class="cit-publ-name">Chapman and Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.7" data-doi="10.1093/biomet/53.3-4.325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gower</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Some Distance Properties of Latent Root and Vector Methods Used in Multivariate Analysis.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DGOWER%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D53%26rft.issue%253D3-4%26rft.spage%253D325%26rft.epage%253D338%26rft.atitle%253DSome%2Bdistance%2Bproperties%2Bof%2Blatent%2Broot%2Band%2Bvector%2Bmethods%2Bused%2Bin%2Bmultivariate%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F53.3-4.325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjEwOiI1My8zLTQvMzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.8" data-doi="10.1371/journal.pgen.1004023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muzzio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing Native American migrations from whole-genome and whole-exome data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1004023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1004023%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004023%26rft_id%253Dinfo%253Apmid%252F24385924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24385924&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.9" data-doi="10.1086/420855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoggart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriver</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittles</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKeigue</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Design and analysis of admixture mapping studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">965</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHoggart%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D74%26rft.issue%253D5%26rft.spage%253D965%26rft.epage%253D978%26rft.atitle%253DDesign%2Band%2Banalysis%2Bof%2Badmixture%2Bmapping%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F420855%26rft_id%253Dinfo%253Apmid%252F15088268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/420855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15088268&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220926100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.10" data-doi="10.1371/journal.pgen.1005602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Homburger</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic Insights into the Ancestry and Demographic History of South America.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1005602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005602%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005602%26rft_id%253Dinfo%253Apmid%252F26636962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26636962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.11" data-doi="10.1371/journal.pgen.1002410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coram</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriver</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romieu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barsh</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ancestral components of admixed genomes in a Mexican cohort.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1002410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJohnson%26rft.auinit1%253DN.%2BA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002410%26rft.epage%253De1002410%26rft.atitle%253DAncestral%2Bcomponents%2Bof%2Badmixed%2Bgenomes%2Bin%2Ba%2BMexican%2Bcohort.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002410%26rft_id%253Dinfo%253Apmid%252F22194699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22194699&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.12" data-doi="10.1016/j.annepidem.2010.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavange</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalsbeek</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorlie</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aviles-Santa</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sample design and cohort selection in the Hispanic Community Health Study/Study of Latinos.</span> <abbr class="cit-jnl-abbrev">Ann. Epidemiol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">642</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bepidemiology%26rft.stitle%253DAnn%2BEpidemiol%26rft.aulast%253DLavange%26rft.auinit1%253DL.%2BM.%26rft.volume%253D20%26rft.issue%253D8%26rft.spage%253D642%26rft.epage%253D649%26rft.atitle%253DSample%2Bdesign%2Band%2Bcohort%2Bselection%2Bin%2Bthe%2BHispanic%2BCommunity%2BHealth%2BStudy%252FStudy%2Bof%2BLatinos.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.annepidem.2010.05.006%26rft_id%253Dinfo%253Apmid%252F20609344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.annepidem.2010.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20609344&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.13" data-doi="10.1126/science.1153717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southwick</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casto</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Worldwide human relationships inferred from genome-wide patterns of variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>(<span class="cit-issue">5866</span>): <span class="cit-fpage">1100</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D319%26rft.issue%253D5866%26rft.spage%253D1100%26rft.epage%253D1104%26rft.atitle%253DWorldwide%2BHuman%2BRelationships%2BInferred%2Bfrom%2BGenome-Wide%2BPatterns%2Bof%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153717%26rft_id%253Dinfo%253Apmid%252F18292342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2Ni8xMTAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.14" data-doi="10.1016/j.ajhg.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maples</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">278</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23910464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23910464&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.15" data-doi="10.1371/journal.pgen.1003925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCauley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing the population genetic history of the Caribbean.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1003925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003925%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003925%26rft_id%253Dinfo%253Apmid%252F24244192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244192&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.16" data-doi="10.1126/science.1251688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Lopez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikora</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetics of Mexico recapitulates Native American substructure and affects biomedical traits.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>(<span class="cit-issue">6189</span>): <span class="cit-fpage">1280</span>–<span class="cit-lpage">1285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMoreno-Estrada%26rft.auinit1%253DA.%26rft.volume%253D344%26rft.issue%253D6189%26rft.spage%253D1280%26rft.epage%253D1285%26rft.atitle%253DThe%2Bgenetics%2Bof%2BMexico%2Brecapitulates%2BNative%2BAmerican%2Bsubstructure%2Band%2Baffects%2Bbiomedical%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251688%26rft_id%253Dinfo%253Apmid%252F24926019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDQvNjE4OS8xMjgwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.17" data-doi="10.1093/bioinformatics/btt166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of Latino populations from GALA and MEC studies reveals genomic loci with biased local ancestry estimation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1407</span>–<span class="cit-lpage">1415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt166%26rft_id%253Dinfo%253Apmid%252F23572411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTEvMTQwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.18" data-doi="10.1086/420871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattangadi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Methods for high-density admixture mapping of disease genes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">979</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D74%26rft.issue%253D5%26rft.spage%253D979%26rft.epage%253D1000%26rft.atitle%253DMethods%2Bfor%2Bhigh-density%2Badmixture%2Bmapping%2Bof%2Bdisease%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F420871%26rft_id%253Dinfo%253Apmid%252F15088269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/420871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15088269&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220926100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.19" data-doi="10.1371/journal.pgen.1000519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafaels</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1000519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000519%26rft.epage%253De1000519%26rft.atitle%253DSensitive%2Bdetection%2Bof%2Bchromosomal%2Bsegments%2Bof%2Bdistinct%2Bancestry%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000519%26rft_id%253Dinfo%253Apmid%252F19543370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543370&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-other" id="cit-6.6.1525.20"><div class="cit-metadata"><cite><span class="cit-comment">R Core Team, 2014 R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="http://www.R-project.org">http://www.R-project.org</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.21" data-doi="10.1038/nature11258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazieres</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reconstructing Native American population history.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>(<span class="cit-issue">7411</span>): <span class="cit-fpage">370</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReich%26rft.auinit1%253DD.%26rft.volume%253D488%26rft.issue%253D7411%26rft.spage%253D370%26rft.epage%253D374%26rft.atitle%253DReconstructing%2BNative%2BAmerican%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11258%26rft_id%253Dinfo%253Apmid%252F22801491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22801491&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307501000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.22" data-doi="10.1002/ajpa.20117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lareu</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobrino</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Shipwrecks and founder effects: divergent demographic histories reflected in Caribbean mtDNA.</span> <abbr class="cit-jnl-abbrev">Am. J. Phys. Anthropol.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">855</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPhysical%2BAnthropology%26rft.stitle%253DAmerican%2BJournal%2Bof%2BPhysical%2BAnthropology%26rft.aulast%253DSalas%26rft.auinit1%253DA.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D855%26rft.epage%253D860%26rft.atitle%253DShipwrecks%2Band%2Bfounder%2Beffects%253A%2Bdivergent%2Bdemographic%2Bhistories%2Breflected%2Bin%2BCaribbean%2BmtDNA.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajpa.20117%26rft_id%253Dinfo%253Apmid%252F16047324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ajpa.20117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16047324&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233647100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.23" data-doi="10.1016/j.ajhg.2015.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schick</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodonsky</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide Association Study of Platelet Count Identifies Ancestry-Specific Loci in Hispanic/Latino Americans.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">229</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.12.003%26rft_id%253Dinfo%253Apmid%252F26805783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26805783&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.24" data-doi="10.1016/j.annepidem.2010.03.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorlie</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aviles-Santa</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wassertheil-Smoller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daviglus</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Design and implementation of the Hispanic Community Health Study/Study of Latinos.</span> <abbr class="cit-jnl-abbrev">Ann. Epidemiol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">629</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bepidemiology%26rft.stitle%253DAnn%2BEpidemiol%26rft.aulast%253DSorlie%26rft.auinit1%253DP.%2BD.%26rft.volume%253D20%26rft.issue%253D8%26rft.spage%253D629%26rft.epage%253D641%26rft.atitle%253DDesign%2Band%2Bimplementation%2Bof%2Bthe%2BHispanic%2BCommunity%2BHealth%2BStudy%252FStudy%2Bof%2BLatinos.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.annepidem.2010.03.015%26rft_id%253Dinfo%253Apmid%252F20609343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.annepidem.2010.03.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20609343&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280131600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1525.25"><div class="cit-metadata"><cite>The 1000 Genomes Project Consortium, <span class="cit-pub-date">2012</span> An integrated map of genetic variation from 1,092 human genomes. Nature 491(7422): 56–65.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/6/1525.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1513" title="Interconnections Between RNA-Processing Pathways Revealed by a Sequencing-Based Genetic Screen for Pre-mRNA Splicing Mutants in Fission Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1535" title="Genomic Signatures of Experimental Adaptation to Antimicrobial Peptides in Staphylococcus aureus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/6"><span>Volume 6 Issue 6, June 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/6.cover-source.jpg?itok=WAtQozJg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1525" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-POKMNBWCvMNS-AgDxohkiQxVzmQ8LNTtSddQfqryJvM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/6/1525" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/6/1525" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/6/1525" class="active" data-icon-position="" data-hide-link-title="0">Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460374" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0JFywmyO2FvX9E-rOXwD1S2yyqzG1sEqHQSm0spHyus" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1525.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/6/1525" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JmGt-rYB-gElDsBA_r0I7N4BuFUONm9lA2hcpQELBMo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/6/1525" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1525.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460374" id="node460374--2" data-pisa="ggg;6/6/1525" data-pisa-master="ggg;g3.116.028779" data-apath="/ggg/6/6/1525.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sharon R. Browning</span>, <span class="highwire-citation-author" data-delta="1">Kelsey Grinde</span>, <span class="highwire-citation-author" data-delta="2">Anna Plantinga</span>, <span class="highwire-citation-author" data-delta="3">Stephanie M. Gogarten</span>, <span class="highwire-citation-author" data-delta="4">Adrienne M. Stilp</span>, <span class="highwire-citation-author" data-delta="5">Robert C. Kaplan</span>, <span class="highwire-citation-author" data-delta="6">M. Larissa Avilés-Santa</span>, <span class="highwire-citation-author" data-delta="7">Brian L. Browning</span> and <span class="highwire-citation-author" data-delta="8">Cathy C. Laurie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1525-1534;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028779 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460374--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sharon R. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BR.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BR.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sguy@uw.edu" class="" data-icon-position="" data-hide-link-title="0">sguy@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kelsey Grinde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BGrinde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grinde%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelsey%2BGrinde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Anna Plantinga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BPlantinga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plantinga%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BPlantinga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephanie M. Gogarten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BM.%2BGogarten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogarten%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BM.%2BGogarten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adrienne M. Stilp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrienne%2BM.%2BStilp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stilp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrienne%2BM.%2BStilp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert C. Kaplan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Epidemiology and Population Health, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BC.%2BKaplan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaplan%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BC.%2BKaplan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. Larissa Avilés-Santa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BLarissa%2BAvil%C3%A9s-Santa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avil%C3%A9s-Santa%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BLarissa%2BAvil%25C3%25A9s-Santa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brian L. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Division of Medical Genetics, Department of Medicine, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BL.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BL.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cathy C. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cathy%2BC.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACathy%2BC.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/6/1525" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/6/1525&amp;title=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/6/1525&amp;title=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/6/1525&amp;title=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/6/1525&amp;text=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/6/1525&amp;title=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/6/1525&amp;t=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/6/1525&amp;title=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/6/1525&amp;title=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460374/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460374" id="node460374--3" data-pisa="ggg;6/6/1525" data-pisa-master="ggg;g3.116.028779" data-apath="/ggg/6/6/1525.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Local Ancestry Inference in a Large US-Based Hispanic/Latino Study: Hispanic Community Health Study/Study of Latinos (HCHS/SOL)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sharon R. Browning</span>, <span class="highwire-citation-author" data-delta="1">Kelsey Grinde</span>, <span class="highwire-citation-author" data-delta="2">Anna Plantinga</span>, <span class="highwire-citation-author" data-delta="3">Stephanie M. Gogarten</span>, <span class="highwire-citation-author" data-delta="4">Adrienne M. Stilp</span>, <span class="highwire-citation-author" data-delta="5">Robert C. Kaplan</span>, <span class="highwire-citation-author" data-delta="6">M. Larissa Avilés-Santa</span>, <span class="highwire-citation-author" data-delta="7">Brian L. Browning</span> and <span class="highwire-citation-author" data-delta="8">Cathy C. Laurie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1525-1534;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028779 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460374--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sharon R. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BR.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BR.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sguy@uw.edu" class="" data-icon-position="" data-hide-link-title="0">sguy@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kelsey Grinde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BGrinde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grinde%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelsey%2BGrinde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Anna Plantinga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BPlantinga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plantinga%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BPlantinga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephanie M. Gogarten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BM.%2BGogarten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogarten%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BM.%2BGogarten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adrienne M. Stilp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrienne%2BM.%2BStilp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stilp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrienne%2BM.%2BStilp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert C. Kaplan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Epidemiology and Population Health, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BC.%2BKaplan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaplan%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BC.%2BKaplan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. Larissa Avilés-Santa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BLarissa%2BAvil%C3%A9s-Santa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avil%C3%A9s-Santa%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BLarissa%2BAvil%25C3%25A9s-Santa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brian L. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Division of Medical Genetics, Department of Medicine, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BL.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BL.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cathy C. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cathy%2BC.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACathy%2BC.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1525&amp;rft.epage=1534&amp;rft.atitle=Local%20Ancestry%20Inference%20in%20a%20Large%20US-Based%20Hispanic/Latino%20Study%3A%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20%28HCHS/SOL%29&amp;rft.volume=6&amp;rft.issue=6&amp;rft.date=2016-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Browning%2C+Sharon+R.&amp;rft.au=Grinde%2C+Kelsey&amp;rft.au=Plantinga%2C+Anna&amp;rft.au=Gogarten%2C+Stephanie+M.&amp;rft.au=Stilp%2C+Adrienne+M.&amp;rft.au=Kaplan%2C+Robert+C.&amp;rft.au=Avil%C3%A9s-Santa%2C+M.+Larissa&amp;rft.au=Browning%2C+Brian+L.&amp;rft.au=Laurie%2C+Cathy+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460374/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460374/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460374/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460374/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460374/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460374/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460374/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460374/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460374/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460374/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460374/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460374/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/6/1525" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/6/1525.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/6/1525.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/6/6/1525.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
