<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/6/1673" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/6/1673.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/6/1673.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/6/1673.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/6/1673.atom" />
<meta name="HW.pisa" content="ggg;6/6/1673" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genetic Map Between Gossypium hirsutum and the Brazilian Endemic G. mustelinum and Its Application to QTL Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.116.029116" />
<meta name="DC.Date" content="2016-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Among the seven tetraploid cotton species, little is known about transmission genetics and genome organization in Gossypium mustelinum , the species most distant from the source of most cultivated cotton, G. hirsutum . In this research, an F2 population was developed from an interspecific cross between G. hirsutum and G. mustelinum (HM). A genetic linkage map was constructed mainly using simple sequence repeat (SSRs) and restriction fragment length polymorphism (RFLP) DNA markers. The arrangements of most genetic loci along the HM chromosomes were identical to those of other tetraploid cotton species. However, both major and minor structural rearrangements were also observed, for which we propose a parsimony-based model for structural divergence of tetraploid cottons from common ancestors. Sequences of mapped markers were used for alignment with the 26 scaffolds of the G. hirsutum draft genome, and showed high consistency. Quantitative trait locus (QTL) mapping of fiber elongation in advanced backcross populations derived from the same parents demonstrated the value of the HM map. The HM map will serve as a valuable resource for QTL mapping and introgression of G. mustelinum alleles into G. hirsutum , and help clarify evolutionary relationships between the tetraploid cotton genomes." />
<meta name="DC.Contributor" content="Baohua Wang" />
<meta name="DC.Contributor" content="Limei Liu" />
<meta name="DC.Contributor" content="Dong Zhang" />
<meta name="DC.Contributor" content="Zhimin Zhuang" />
<meta name="DC.Contributor" content="Hui Guo" />
<meta name="DC.Contributor" content="Xin Qiao" />
<meta name="DC.Contributor" content="Lijuan Wei" />
<meta name="DC.Contributor" content="Junkang Rong" />
<meta name="DC.Contributor" content="O. Lloyd May" />
<meta name="DC.Contributor" content="Andrew H. Paterson" />
<meta name="DC.Contributor" content="Peng W. Chee" />
<meta name="article:published_time" content="2016-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Genetic Map Between Gossypium hirsutum and the Brazilian Endemic G. mustelinum and Its Application to QTL Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Among the seven tetraploid cotton species, little is known about transmission genetics and genome organization in &lt;i&gt;Gossypium mustelinum&lt;/i&gt;, the species most distant from the source of most cultivated cotton, &lt;i&gt;G. hirsutum&lt;/i&gt;. In this research, an F&lt;sub&gt;2&lt;/sub&gt; population was developed from an interspecific cross between &lt;i&gt;G. hirsutum&lt;/i&gt; and &lt;i&gt;G. mustelinum&lt;/i&gt; (HM). A genetic linkage map was constructed mainly using simple sequence repeat (SSRs) and restriction fragment length polymorphism (RFLP) DNA markers. The arrangements of most genetic loci along the HM chromosomes were identical to those of other tetraploid cotton species. However, both major and minor structural rearrangements were also observed, for which we propose a parsimony-based model for structural divergence of tetraploid cottons from common ancestors. Sequences of mapped markers were used for alignment with the 26 scaffolds of the &lt;i&gt;G. hirsutum&lt;/i&gt; draft genome, and showed high consistency. Quantitative trait locus (QTL) mapping of fiber elongation in advanced backcross populations derived from the same parents demonstrated the value of the HM map. The HM map will serve as a valuable resource for QTL mapping and introgression of &lt;i&gt;G. mustelinum&lt;/i&gt; alleles into &lt;i&gt;G. hirsutum&lt;/i&gt;, and help clarify evolutionary relationships between the tetraploid cotton genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/06/01" />
<meta name="citation_mjid" content="ggg;6/6/1673" />
<meta name="citation_id" content="6/6/1673" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/6/1673" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/6/1673.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/6/1673.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/6/1673.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029116" />
<meta name="citation_pmid" content="27172208" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1673" />
<meta name="citation_lastpage" content="1685" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Baohua Wang" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_institution" content="School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7449-8292" />
<meta name="citation_author" content="Limei Liu" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author" content="Dong Zhang" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Zhimin Zhuang" />
<meta name="citation_author_institution" content="School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author" content="Hui Guo" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Xin Qiao" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Lijuan Wei" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Junkang Rong" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_institution" content="School of Agriculture and Food Science, Zhejiang A&amp;F University, Lin’an, Hangzhou, Zhejiang 311300, China" />
<meta name="citation_author" content="O. Lloyd May" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author" content="Andrew H. Paterson" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_email" content="paterson@uga.edu" />
<meta name="citation_author_email" content="pwchee@uga.edu" />
<meta name="citation_author" content="Peng W. Chee" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author_email" content="paterson@uga.edu" />
<meta name="citation_author_email" content="pwchee@uga.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of genetics;citation_journal_abbrev=J Genet;citation_author=S. L. Ahuja;citation_author=L. S. Dhayal;citation_title=Comparative characteristics and gene action in three petal-spotted mutants of Gossypium hirsutum.;citation_pages=81-84;citation_volume=86;citation_year=2007;citation_issue=1;citation_pmid=17656854;citation_doi=10.1007/s12041-007-0012-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Blenda;citation_author=D. D. Fang;citation_author=J. F. Rami;citation_author=O. Garsmeur;citation_author=F. Luo;citation_title=A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.;citation_pages=e45739-e45739;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029214;citation_doi=10.1371/journal.pone.0045739" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=C. L. Brubaker;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_title=Comparative genetic mapping of allotetraploid cotton and its diploid progenitors.;citation_pages=184-203;citation_volume=42;citation_year=1999;citation_doi=10.1139/g98-118" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=P. W. Chee;citation_author=J. K. Rong;citation_author=D. Williams-Coplin;citation_author=S. R. Schulze;citation_author=A. H. Paterson;citation_title=EST derived PCR-based markers for functional gene homologues in cotton.;citation_pages=449-462;citation_volume=47;citation_year=2004;citation_issue=3;citation_pmid=15190362" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=H. Chen;citation_author=M. K. R. Khan;citation_author=Z. Zhou;citation_author=X. Wang;citation_author=X. Cai;citation_title=A high-density SSR genetic map constructed from a F2 population of Gossypium hirsutum and Gossypium darwinii.;citation_pages=273-286;citation_volume=574;citation_year=2015;citation_pmid=26275937;citation_doi=10.1016/j.gene.2015.08.022" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=D. R. DeJoode;citation_author=J. F. Wendel;citation_title=Genetic diversity and origin of the Hawaiian-Islands cotton, Gossypium tomentosum.;citation_pages=1311-1319;citation_volume=79;citation_year=1992;citation_doi=10.2307/2445059" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=J. E. Endrizzi;citation_author=D. T. Ray;citation_title=Linkage analysis of open bud (ob2) and yellow petal (Y1) in cotton.;citation_pages=461-463;citation_volume=34;citation_year=1991;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Adv. Genet.;citation_author=J. E. Endrizzi;citation_author=E. L. Turcotte;citation_author=R. J. Kohel;citation_title=Genetics cytology and evolution of Gossypium.;citation_pages=271-375;citation_volume=23;citation_year=1985;citation_doi=10.1016/S0065-2660(08)60515-5" />
<meta name="citation_reference" content="citation_title=Cotton,;citation_title=Taxonomy and germplasm resources,;citation_author=P. A. Fryxell;citation_pages=27-57;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Rheedea;citation_author=P. A. Fryxell;citation_title=A revised taxonomic interpretation of Gossypium L., (Malvacea).;citation_pages=108-165;citation_volume=2;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=C. E. Grover;citation_author=J. P. Gallagher;citation_author=J. J. Jareczek;citation_author=J. T. Page;citation_author=J. A. Udall;citation_title=Re-evaluating the phylogeny of allopolyploid Gossypium L.;citation_pages=45-52;citation_volume=92;citation_year=2015;citation_pmid=26049043;citation_doi=10.1016/j.ympev.2015.05.023" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=C. E. Grover;citation_author=X. Zhu;citation_author=K. K. Grupp;citation_author=J. J. Jareczek;citation_author=J. P. Gallagher;citation_title=Molecular confirmation of species status for the allopolyploid cotton species, Gossypium ekmanianum Wittmack.;citation_pages=103-114;citation_volume=62;citation_year=2015;citation_doi=10.1007/s10722-014-0138-x" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=W. Z. Guo;citation_author=G. J. Ma;citation_author=Y. C. Zhu;citation_author=C. X. Yi;citation_author=T. Z. Zhang;citation_title=Molecular tagging and mapping of quantitative trait loci for lint percentage and morphological marker genes in Upland cotton.;citation_pages=320-326;citation_volume=48;citation_year=2006;citation_issue=3;citation_doi=10.1111/j.1744-7909.2006.00174.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Z. Guo;citation_author=C. P. Cai;citation_author=C. B. Wang;citation_author=Z. G. Han;citation_author=X. L. Song;citation_title=A microsatellite-based, generic linkage map reveals genome structure, function and evolution in Gossypium.;citation_volume=176;citation_year=2007;citation_issue=1;citation_pmid=17409069;citation_doi=10.1534/genetics.107.070375" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. Guo;citation_author=C. Cai;citation_author=C. Wang;citation_author=L. Zhao;citation_author=L. Wang;citation_title=A preliminary analysis of genome structure and composition in Gossypium hirsutum.;citation_pages=314-314;citation_volume=9;citation_year=2008;citation_pmid=18590573;citation_doi=10.1186/1471-2164-9-314" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Z. G. Han;citation_author=W. Z. Guo;citation_author=X. L. Song;citation_author=T. Z. Zhang;citation_title=Genetic mapping of EST-derived microsatellites from the diploid Gossypium arboreum in allotetraploid cotton.;citation_pages=308-327;citation_volume=272;citation_year=2004;citation_issue=3;citation_pmid=15368122;citation_doi=10.1007/s00438-004-1059-8" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. G. Han;citation_author=C. B. Wang;citation_author=X. L. Song;citation_author=W. Z. Guo;citation_author=J. Y. Gou;citation_title=Characteristics, development and mapping of Gossypium hirsutum derived EST-SSRs in allotetraploid cotton.;citation_pages=430-439;citation_volume=112;citation_year=2006;citation_issue=3;citation_pmid=16341684;citation_doi=10.1007/s00122-005-0142-9" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=S. C. Harland;citation_title=The genetics of cotton. Part II the inheritance of pollen colour in New World cottons.;citation_pages=387-399;citation_volume=20;citation_year=1929;citation_doi=10.1007/BF02983156" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=D. H. He;citation_author=Z. X. Lin;citation_author=X. L. Zhang;citation_author=Y. C. Nie;citation_author=X. P. Guo;citation_title=QTL mapping for economic traits based on a dense genetic map of cotton with PCR-based markers using the interspecific cross of Gossypium hirsutum × Gossypium barbadense.;citation_pages=181-197;citation_volume=153;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=M. Y. Hou;citation_author=C. P. Cai;citation_author=S. W. Zhang;citation_author=W. Z. Guo;citation_author=T. Z. Zhang;citation_title=Construction of microsatellite-based linkage map and mapping of nectarilessness and hairiness genes in Gossypium tomentosum.;citation_pages=445-459;citation_volume=92;citation_year=2013;citation_issue=3;citation_pmid=24371166;citation_doi=10.1007/s12041-013-0286-3" />
<meta name="citation_reference" content="citation_journal_title=Genes Genom. Genet.;citation_author=A. M. Hulse-Kemp;citation_author=J. Lemm;citation_author=J. Plieske;citation_author=H. Ashra;citation_author=R. Buyyarapu;citation_title=Development of a 63K SNP array for cotton and high-density mapping of intraspecific and interspecific populations of Gossypium spp. G3;citation_pages=1187-1209;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Res.;citation_author=T. H. Kbarnsy;citation_title=Inheritance of petal spot in pima cotton.;citation_pages=491-512;citation_volume=27;citation_year=1924;citation_issue=7" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. J. Kohel;citation_author=T. R. Richmond;citation_title=Isolines in Cotton: Effects of Nine Dominant Genes;citation_pages=287-289;citation_volume=11;citation_year=1971;citation_issue=2;citation_doi=10.2135/cropsci1971.0011183X001100020036x" />
<meta name="citation_reference" content="citation_journal_title=J. Cotton Sci.;citation_author=R. J. Kohel;citation_author=L. S. Bird;citation_title=Inheritance and linkage analysis of the yellow pulvinus mutant of cotton.;citation_pages=115-118;citation_volume=6;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. Konieczny;citation_author=F. M. A. Ausubel;citation_title=A procedure for mapping Arabidopsis mutations using co-dominant ecotype-specific PCR-based markers.;citation_pages=403-410;citation_volume=4;citation_year=1993;citation_issue=2;citation_pmid=8106085;citation_doi=10.1046/j.1365-313X.1993.04020403.x" />
<meta name="citation_reference" content="citation_journal_title=Bonplandia;citation_author=A. Krapovickas;citation_author=G. Seijo;citation_title=Gossypium ekmanianum (Malvaceae), algodón silvestre de la República Dominicana.;citation_pages=55-63;citation_volume=17;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. M. Lacape;citation_author=T. B. Nguyen;citation_author=S. Thibivilliers;citation_author=B. Bojinov;citation_author=B. Courtois;citation_title=A combined RFLP-SSR-AFLP map of tetraploid cotton based on a Gossypium hirsutum x Gossypium barbadense backcross population.;citation_pages=612-626;citation_volume=46;citation_year=2003;citation_issue=4;citation_pmid=12897870;citation_doi=10.1139/g03&amp;ndash;050" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. Lacape;citation_author=J. Jacobs;citation_author=T. Arioli;citation_author=R. Derijcker;citation_author=N. Forestier-Chiron;citation_title=A new interspecific, Gossypium hirsutum x G. barbadense, RIL population: towards a unified consensus linkage map of tetraploid cotton.;citation_pages=281-292;citation_volume=119;citation_year=2009;citation_issue=2;citation_pmid=19387609;citation_doi=10.1007/s00122-009-1037-y" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. J. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. Li;citation_author=G. Fan;citation_author=K. Wang;citation_author=F. Sun;citation_author=Y. Yuan;citation_title=Genome sequence of the cultivated cotton Gossypium arboreum.;citation_pages=567-572;citation_volume=46;citation_year=2014;citation_pmid=24836287;citation_doi=10.1038/ng.2987" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=F. Li;citation_author=G. Fan;citation_author=C. Lu;citation_author=G. Xiao;citation_author=C. Zou;citation_title=Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution.;citation_pages=524-530;citation_volume=33;citation_year=2015;citation_pmid=25893780;citation_doi=10.1038/nbt.3208" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=Z. X. Lin;citation_author=D. H. He;citation_author=X. L. Zhang;citation_author=Y. C. Nie;citation_author=X. P. Guo;citation_title=Linkage map construction and mapping QTL for cotton fibre quality using SRAP, SSR and RAPD.;citation_pages=180-187;citation_volume=124;citation_year=2005;citation_doi=10.1111/j.1439-0523.2004.01039.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=R. Z. Liu;citation_author=B. H. Wang;citation_author=W. Z. Guo;citation_author=Y. S. Qin;citation_author=L. G. Wang;citation_title=Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in Gossypium hirsutum L.;citation_pages=297-311;citation_volume=29;citation_year=2012;citation_issue=2;citation_doi=10.1007/s11032-011-9547-0" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=O. L. May;citation_title=Registration of PD 94042 Germplasm Line of Upland Cotton with High Yield and Fiber Maturity;citation_pages=597-598;citation_volume=39;citation_year=1999;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. B. Nguyen;citation_author=M. Giband;citation_author=P. Brottier;citation_author=A. M. Risterucci;citation_author=J. M. Lacape;citation_title=Wide coverage of the tetraploid cotton genome using newly developed microsatellite markers.;citation_pages=167-175;citation_volume=109;citation_year=2004;citation_issue=1;citation_pmid=14997301;citation_doi=10.1007/s00122-004-1612-1" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Y. H. Park;citation_author=M. S. Alabady;citation_author=M. Ulloa;citation_author=B. Sickler;citation_author=T. A. Wilkins;citation_title=Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line cotton population.;citation_pages=428-441;citation_volume=274;citation_year=2005;citation_issue=4;citation_pmid=16187061;citation_doi=10.1007/s00438-005-0037-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=A. H. Paterson;citation_author=C. L. Brubaker;citation_author=J. F. Wendel;citation_title=A rapid method for extraction of cotton (Gossypium spp.) genomic DNA suitable for RFLP or PCR analysis.;citation_pages=122-127;citation_volume=11;citation_year=1993;citation_issue=2;citation_doi=10.1007/BF02670470" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_title=Cotton: Origin, History, Technology, and Production;citation_title=Qualitative genetics,;citation_author=R. G. Percy;citation_author=R. J. Kohel;citation_pages=319-360;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. J. Reinisch;citation_author=J. M. Dong;citation_author=C. L. Brubaker;citation_author=D. M. Stelly;citation_author=J. F. Wendelt;citation_title=A detailed RFLP map of cotton, Gossypium hirsutum x Gossypium barbadense: chromosome organization and evolution in a disomic polyploid genome.;citation_pages=829-847;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851778" />
<meta name="citation_reference" content="citation_journal_title=Beltwide Cotton Production Research Conferences;citation_author=C. L. Rhyne;citation_author=J. C. Carter;citation_title=Research efforts with genes of linkage group V. II. Loci distal the centromere.;citation_pages=596-598;citation_volume=2;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Rong;citation_author=C. Abbey;citation_author=J. E. Bowers;citation_author=C. L. Brubaker;citation_author=C. Chang;citation_title=A 3347-Locus Genetic Recombination Map of Sequence-Tagged Sites Reveals Features of Genome Organization, Transmission and Evolution of Cotton (Gossypium);citation_pages=389-417;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020432;citation_doi=10.1534/genetics.166.1.389" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=Y. Shi;citation_author=W. Li;citation_author=A. Li;citation_author=R. Ge;citation_author=B. Zhang;citation_title=Constructing a high-density linkage map for Gossypium hirsutum × Gossypium barbadense and identifying QTLs for lint percentage.;citation_pages=450-467;citation_volume=57;citation_year=2015;citation_issue=5;citation_pmid=25263268;citation_doi=10.1111/jipb.12288" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=X. L. Song;citation_author=K. Wang;citation_author=W. Z. Guo;citation_author=J. Zhang;citation_author=T. Z. Zhang;citation_title=A comparison of genetic maps constructed from haploid and BC1 mapping populations from the same crossing between Gossypium hirsutum L. and Gossypium barbadense L.;citation_pages=378-390;citation_volume=48;citation_year=2005;citation_issue=3;citation_pmid=16121235" />
<meta name="citation_reference" content="citation_title=Pollen–Biology, Biochemistry, Management;citation_author=R. G. Stanley;citation_author=H. F. Linskens;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. G. Stephens;citation_title=INTERSPECIFIC HOMOLOGIES BETWEEN GENE LOCI IN GOSSYPIUM. I. POLLEN COLOR;citation_pages=701-711;citation_volume=39;citation_year=1954;citation_issue=5;citation_pmid=17247514" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Turcotte;citation_author=C. V. Feaster;citation_title=A Second Locus for Pollen Color in Pima Cotton, Gossypium barbadense L.;citation_pages=117-119;citation_volume=6;citation_year=1966;citation_issue=2;citation_doi=10.2135/cropsci1966.0011183X000600020003x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=V. N. Waghmare;citation_author=J. K. Rong;citation_author=C. J. Rogers;citation_author=G. J. Pierce;citation_author=J. F. Wendel;citation_title=Genetic mapping of a cross between Gossypium hirsutum (cotton) and the Hawaiian endemic, Gossypium tomentosum.;citation_pages=665-676;citation_volume=111;citation_year=2005;citation_issue=4;citation_pmid=16044266;citation_doi=10.1007/s00122-005-2032-6" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Wang;citation_author=Y. Nie;citation_author=Z. Lin;citation_author=X. Zhang;citation_author=J. Liu;citation_title=Molecular diversity, genomic constitution, and QTL mapping of fiber quality by mapped SSRs in introgression lines derived from Gossypium hirsutum Ã G. darwinii Watt.;citation_pages=1263-1274;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733444;citation_doi=10.1007/s00122-012-1911-x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res.;citation_author=B. Wang;citation_author=P. Rong;citation_author=X. Cai;citation_author=W. Wang;citation_author=X. Zhu;citation_title=Development of EST-SSR markers related to disease resistance and their application in genetic diversity and evolution analysis in Gossypium.;citation_pages=10630-10644;citation_volume=14;citation_year=2015;citation_issue=3;citation_pmid=26400294;citation_doi=10.4238/2015.September.9.4" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. Wang;citation_author=X. Draye;citation_author=Z. Zhang;citation_author=Z. Zhuang;citation_author=O. L. May;citation_title=Advanced backcross QTL analysis of fiber elongation in a cross between Gossypium hirsutum and G. mustelinum.;citation_year=2016;citation_doi=10.2135/cropsci2015.12.0753" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Wang;citation_author=Z. Wang;citation_author=F. Li;citation_author=W. Ye;citation_author=J. Wang;citation_title=The draft genome of a diploid cotton Gossypium raimondii.;citation_pages=1098-1103;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22922876;citation_doi=10.1038/ng.2371" />
<meta name="citation_reference" content="citation_journal_title=Genes Genom. Genet.;citation_author=Z. Wang;citation_author=D. Zhang;citation_author=X. Wang;citation_author=X. Tan;citation_author=H. Guo;citation_title=A whole-genome DNA marker map for cotton based on the D-genome sequence of Gossypium raimondii L. G3;citation_pages=1759-1767;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Cotton;citation_title=Taxonomy and evolution of the cotton genus, Gossypium;citation_author=J. F. Wendel;citation_author=C. E. Grover;citation_pages=25-44;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=J. F. Wendel;citation_author=R. C. Cronn;citation_title=Polyploidy and the evolutionary history of cotton.;citation_pages=139-186;citation_volume=78;citation_year=2003;citation_doi=10.1016/S0065-2113(02)78004-8" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Syst. Ecol.;citation_author=J. F. Wendel;citation_author=R. G. Percy;citation_title=Allozyme diversity and introgression in the Galapagos-Islands endemic Gossypium darwinii and its relationship to continental Gossypium barbadense.;citation_pages=517-528;citation_volume=18;citation_year=1990;citation_doi=10.1016/0305-1978(90)90123-W" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=J. F. Wendel;citation_author=R. Rowley;citation_author=J. M. Stewart;citation_title=Genetic diversity in and phylogenetic relationships of the Brazilian endemic cotton, Gossypium mustelinum (Malvaceae).;citation_pages=49-59;citation_volume=192;citation_year=1994;citation_doi=10.1007/BF00985907" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Yang;citation_author=C. C. Hu;citation_author=H. Hu;citation_author=R. D. Yu;citation_author=Z. Xia;citation_title=QTLNetwork: mapping and visualizing genetic architecture of complex traits in experimental populations;citation_volume=24;citation_year=2008;citation_pmid=18202029;citation_doi=10.1093/bioinformatics/btm494" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=J. W. Yu;citation_author=S. X. Yu;citation_author=C. R. Lu;citation_author=W. Wang;citation_author=S. L. Fan;citation_title=High-density linkage map of cultivated allotetraploid cotton based on SSR, TRAP, SRAP and AFLP markers.;citation_pages=716-724;citation_volume=49;citation_year=2007;citation_issue=5;citation_doi=10.1111/j.1744-7909.2007.00459.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Yu;citation_author=D. Yuan;citation_author=S. Liang;citation_author=X. Li;citation_author=X. Wang;citation_title=Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between gossypium hirsutum and G. barbadense.;citation_pages=15-15;citation_volume=12;citation_year=2011;citation_pmid=21214949;citation_doi=10.1186/1471-2164-12-15" />
<meta name="citation_reference" content="citation_journal_title=Genes Genom. Genet.;citation_author=J. Yu;citation_author=R. Kohel;citation_author=D. Fang;citation_author=J. Cho;citation_author=A. Van Deynze;citation_title=A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome. G3;citation_pages=43-58;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Zhang;citation_author=W. Guo;citation_author=T. Zhang;citation_title=Molecular linkage map of allotetraploid cotton ( Gossypium hirsutum L. x Gossypium barbadense L.) with a haploid population.;citation_pages=1166-1174;citation_volume=105;citation_year=2002;citation_issue=8;citation_pmid=12582895;citation_doi=10.1007/s00122-002-1100-4" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=T. Z. Zhang;citation_author=Y. Hu;citation_author=W. K. Jiang;citation_author=L. Fang;citation_author=X. Y. Guan;citation_title=Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement.;citation_pages=531-537;citation_volume=33;citation_year=2015;citation_pmid=25893781;citation_doi=10.1038/nbt.3207" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Zhang;citation_author=J. Rong;citation_author=V. N. Waghmare;citation_author=P. W. Chee;citation_author=O. L. May;citation_title=QTL alleles for improved fiber quality from a wild Hawaiian cotton, Gossypium tomentosum.;citation_pages=1075-1088;citation_volume=123;citation_year=2011;citation_issue=7;citation_pmid=21735234;citation_doi=10.1007/s00122-011-1649-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genetic Map Between Gossypium hirsutum and the Brazilian Endemic G. mustelinum and Its Application to QTL Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="twitter:description" content="Among the seven tetraploid cotton species, little is known about transmission genetics and genome organization in Gossypium mustelinum , the species most distant from the source of most cultivated cotton, G. hirsutum . In this research, an F2 population was developed from an interspecific cross between G. hirsutum and G. mustelinum (HM). A genetic linkage map was constructed mainly using simple sequence repeat (SSRs) and restriction fragment length polymorphism (RFLP) DNA markers. The arrangements of most genetic loci along the HM chromosomes were identical to those of other tetraploid cotton species. However, both major and minor structural rearrangements were also observed, for which we propose a parsimony-based model for structural divergence of tetraploid cottons from common ancestors. Sequences of mapped markers were used for alignment with the 26 scaffolds of the G. hirsutum draft genome, and showed high consistency. Quantitative trait locus (QTL) mapping of fiber elongation in advanced backcross populations derived from the same parents demonstrated the value of the HM map. The HM map will serve as a valuable resource for QTL mapping and introgression of G. mustelinum alleles into G. hirsutum , and help clarify evolutionary relationships between the tetraploid cotton genomes." />
<meta name="og-title" property="og:title" content="A Genetic Map Between Gossypium hirsutum and the Brazilian Endemic G. mustelinum and Its Application to QTL Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/6/1673" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Among the seven tetraploid cotton species, little is known about transmission genetics and genome organization in Gossypium mustelinum , the species most distant from the source of most cultivated cotton, G. hirsutum . In this research, an F2 population was developed from an interspecific cross between G. hirsutum and G. mustelinum (HM). A genetic linkage map was constructed mainly using simple sequence repeat (SSRs) and restriction fragment length polymorphism (RFLP) DNA markers. The arrangements of most genetic loci along the HM chromosomes were identical to those of other tetraploid cotton species. However, both major and minor structural rearrangements were also observed, for which we propose a parsimony-based model for structural divergence of tetraploid cottons from common ancestors. Sequences of mapped markers were used for alignment with the 26 scaffolds of the G. hirsutum draft genome, and showed high consistency. Quantitative trait locus (QTL) mapping of fiber elongation in advanced backcross populations derived from the same parents demonstrated the value of the HM map. The HM map will serve as a valuable resource for QTL mapping and introgression of G. mustelinum alleles into G. hirsutum , and help clarify evolutionary relationships between the tetraploid cotton genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
    <title>A Genetic Map Between Gossypium hirsutum and the Brazilian Endemic G. mustelinum and Its Application to QTL Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DllyTEfG4QQFis3Lg4Xozl7b3ikPeiLq47G2kcAKO40","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460391","apath":"\/ggg\/6\/6\/1673.atom","pisa":"ggg;6\/6\/1673","ac":{"\/ggg\/6\/6\/1673.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/6\/1673.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1673"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1673"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/6\/1673":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228yGPP6wJsJAtiVOlr1gz6JY8bbuu3NUEnpRFm1uvqXl+uZth3EcklRh6CAsUf0dk\\\\\\\/1cpLbCqhVjX3BXzUSkkIZKH+3fxZWSPrxwD+2vtgGwMvW2mYBGIKbmccLWtQqP5\\\\\\\/sSyxZpkIXKm0TEF79QIiZyoAQHmjPPJL397LwI6jL1q9RYqYmdMxFnsoOf3TOQK0pjwaDHqeva7iwYhZlCDqdY\\\\\\\/aCblipFUFRz6Itylp\\\\\\\/i7JninnxhZuvO+MLZjZ+dKkbDUzPkK7C4oZDgmmOpAfMmnC91jcSaAKbEZRhayy4vmTSBDspUTd7tO5EpfotgnnM6Xk86R4IglBG4j52CW5aOVAXBeHaauOm2nKeI8RCmE6p+Rq446sUL9Iwr\\\\\\\/lTuCtu4CbeKsZbzAaiGBgY0rKQIexVteI2gQSM8dYKB7TIE7NNLVoKoxOi8M9mytSJfRgszdesXvPHq1h2iuEw2Mtv1NbvK04Sj9MvCrKR9Xih+bcejv5TlSSv3beiZiebk9EMvimloleIpCRiIXV7rdfeFNEAL\\\\\\\/QzSoDqpVl4XmM8Ol3\\\\\\\/GtUScSt29rvlH7UCuOd1p9eCVPv7B4mKoE\\\\\\\/qofILAp0YcGPcYx6KhHDMhD9ndsJR\\\\\\\/+cQZ2XBiD2zruHENIiF+6pFmbJEymHstBxtNz6hWYtqJq8YU+Rm1\\\\\\\/6510Us+CUCYy4oKnq2Kw3ee5TdTq0g4CiAdD9T9aoRx2sB+qu13b0L1PKxqTvVLH9Pq4DxPIQBrVwlqKnzwEMUPDVICCsBImVr7uTZvAICEqqNXFbOkHYCuK37hTtMrpPmhhu5K6pSOncBat95GOcI\\\\\\\/EKvk9Wn7BIkrIrTLSvrj2EH1c5gpXBIxkBJDVVwacEPd8lNNqVOJSBDptD5nNWOGFQQEWCWXaN18sPdv8r5M4mp+Nnit8X3ESqBooeUld58AzECX2VEa2OoJ6pDU8F\\\\\\\/uz2dairOrtx\\\\\\\/j0+Rf3qLnYK3FBeyveaGNOGFhJJ0FxYILKNs\\\\\\\/\\\\\\\/zObyCqnDLHxVO\\\\\\\/NVKbmewrNIZBzqh77yWIaJNBLDl3GUTaMskdIH\\\\\\\/ztDFczZb16wsxGCfQk+BZ8A5xYEG9N3\\\\\\\/zQxmC1spr+9ftqZuQhKz4TbAjvRgyRJPnifS6ZK149mxnKoyrFZccs8JtMmvfbbkA8gA6yr4e2E502kc8nsERiEu1BcMwuGgpxM467ylS9hI\\\\\\\/QafF1RKs2E39XVMRiTmuyHCk9WEDtt9KQgOHTByoea6rGw9CK7SugoY1ETtvMtgLoMEZPSIYgmFHNPyoco\\\u0022,\\\u0022iv\\\u0022:\\\u00225+\\\\\\\/tuic+5vynzfVN5ByWHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ffbe6ec05c5014d16a8b6e70f595df43189ad1f8eaa308a45cba27b534e5a0dc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qiB1eOTZd5dyj2zqEUSD77e\\\\\\\/c2URTokY+W4mWL+dJkeWeCmF69B7l6MyJvMaQdNSI8ilpS1FvJPD6ZZ1Ta6Q\\\\\\\/gvpGoQ0j0RabqDeqgMMDFM9W893oV\\\\\\\/rH6Frx7p523DEBKd6YC3mwdS\\\\\\\/39QulFJ90jjAhMxAM8ZaF7kDSkYhveO6mO1JphU5w4FFv2YHsUjDDIIF9FHQ265RG8wxTxHtrAE2dwbW+kkHzhlsuCLTU65IFZTI9uJn7Mu1K8Li3TdYl7e+TnkXsuyRq5JibmX\\\\\\\/0ieNtlcPgD5xLhm3qEw3FHfhx1F3be\\\\\\\/cyb55\\\\\\\/FT3mlqN5iQp2yHUPAzlP3VbJ6lpJi786RrEo77MbwhnPbEqVM3iuhYJfhFMOkQxisNI+LDn8z8qrrP1F6t+UiHvpJ\\\\\\\/lTyyH7OKHO2hwNBSd7SDGvOpwm7CI11kGrARSYnzHZ5g1G1aScd24BM68errvS5CPVXLXrH7U8NZP9HIsfU5gLGZGHySznszFSbUXFETHJqScgEb8U3UQKplNu4X9JRxKkV+U9PmRKSuqjxoTM3ChWYW4yO\\\\\\\/Zf1Omdd+tb2mivdPiP9FsidFSzM3PdI7SzArlcMyhSTvhyixrlwKBuy382Z1hHaNuc8l6XSij0mONjfODnhMg8NO8njqFxoorRiNqWd3WY8ofnWDOwPjDgxJrDMaWindBcxdaJiBd7ddzfON81biZtZJ211SfGye\\\\\\\/iARHZtnk82DBxhIPs67Qzm\\\\\\\/3HtqTsJjszBDtJpciNw7Jb5RTymmClgGVBL324K0Vi\\\\\\\/1NnuhRzBpAGjDymlz65KrWJfdtXteIXF0sjAEcmjLjJ\\\\\\\/DcxdJZiAZEanD6O\\\\\\\/z0yzcc6ZP7zVbKhRBtxs5sT70T2baQUgOmm0gYJ7Qf9z6ge1VmBFe9Pldm8NKgsPUDKkanf2Z+mjqU+xHxK1XrTa\\\\\\\/pj9Xb0WW94vC+OFH+0qVb2OjKUK+w3uipWuOEwquMDqZjBP8g9+wjYEYJYYDFplYQQrUeGIAhuzn0U9NMn3iIrzv8k6HAjZsYzwrOfiuagMXHC5E5DkjnD4+Ea5tiZXrrr7Ww8y4B9C91+6PcN6V4dnsuXDQVvp5grps2+DzZb88QcoKhE\\\\\\\/Mts\\\\\\\/OIK\\\\\\\/SLeAIzIniEmvP+TuFRy4ffxzQAA4HZzjaHoHoBNqLRJRnCypDZQEN\\\\\\\/DW4FhdIBK+i5xNVcPuVZ\\\\\\\/l1IvyWCA2hnkwFNaM2AYwNRx9KPbeEe2mJ5VHsdZzijwVmggz7blUXBtYLA7o82onrs4fazC7j\\\\\\\/9sjbbLwP1GR0q+uSp5pcUBTRIiV2EvDaYY1gwEEdJzse2hpXbvDsct36ZMwSCJsu2+fPtLSTa+Zpu9WFXAti4lG9lDiZ\\\\\\\/7EKE2YvPODfLgiayflDDWhP85wRY8zWdsmqeVfNtgRfZh+CkioIiDWXj5WTqqW2Kwhb7+tR0xQgofidFy9qJnFtCZIAhMEimBGVmzVV5mBnG3TwAhWTteQSQT9k2eFGjKuHrXVj56BnNu8yOXjnIQNz3jzlHyOuAbwzZJKnTswhXR1d2NsczO6Se0qWw\\\\\\\/FTBzM5HDsn5uyjuIavGBngQdffN+EFvg3gJWkbhnKNXNb8Uq9rL0QvbZRPPG937khi4iqdfhGDTcjjfAkvLumc9Tz\\\\\\\/b4mh9SYo8MXok7mtf9Y06\\\\\\\/8ijHQ8jXgjRIp6yexAlImRA+8pWlXX4MI73osbNhjS8aW2YYKu\\\\\\\/uexVUSKUU2VtJAgwWOky9xaNTYzmYQ5dCqIYGZXiiOZMGzoYhj31sPm0s+Ve+nkm2EqTBN2W5INwLKIhYYb4EbCaBtSc0P+bdbgAwEbyKgFibeysTo+PunPidI1irPFkmtgXTCgd1bNmFy2BHS\\\\\\\/c9G2pWM\\\\\\\/ZH04cnKWxfYG9qV0a9yc5ywhlnBU+029\\\\\\\/fFqFJw0p1XYZJgpo12h+HaA1vJHMh5tpAvKzpjWVqQ4WDPDCFTM\\\\\\\/bvBV8SzvH+hUKDcVHurg4X2RrHLt+UAhS9f7KbiMM4d0Vq5zXy6kpJ0vb5Gdplkyir4aXTnWqPdi\\\\\\\/aSfFWApB5KsWxg3cGU7ou08IMjk1qc7p9app0q5c7W3IvyRAdAjtOOjjlCT5SXTI1RwvEblHVZGXfT6jOAs2hj+wMo6pfk0ZLEWnDFZO6zH9z6Pa9uQrFUCpp4nZzhhDC59uXYZVHf0vgOBo6gjMM43bgQZtvXH5OhNn2qpIu8\\\\\\\/Cb7y5heUQjq+2+m8EpYO5jpyejq38e\\\\\\\/CbgoJ86+TO2FkaxgEzgUjzg\\\\\\\/TSLcAvP8lbpbLOGrAaJwZ4Zqa9WtKyEdc6HAH0fHbgbkLwtlV6V88mKek15Hx4TwNKrNny3aHc3LHCGwxiGQSU\\\\\\\/xT\\\\\\\/1Re0pHA2c\\\\\\\/qwwahEc\\\\\\\/Xnnh\\\\\\\/K\\\\\\\/spMvEOQmpDk4Uiar\\\\\\\/Bdgi062uFQW5IM5LIBLCrf9ne0ZrXZygVEW5LjlYoNybr36VTtQI2evMEUU9jjHgnJPk420swQUI4fd4EEZAnKeHWjIcpAaNRcJSOqBVRzA5Fr+AvmRcEXQeAIlLv\\\\\\\/3E2SgUmFhsid5WtuRstRdqI6+rSZrUwyD5DunAWOiHtUsOJOOYmD0vHeOEwYRC5BZRD3n2FykdVu\\\\\\\/B2RkYI0faB9g18fh06PY6DfW+tel4m3YcPpTAWsmNlHGTWOwa82dE\\\\\\\/+cvgWvgd3URZ7mYS+jDbC+p5gh\\\\\\\/Y\\\\\\\/bWwNVu2yeyu3vSVMjQyGN6kWSzvQ6N08plk5W7XY1\\\\\\\/ygGBH5iwHLryq59lF1NhSbNLMWUWgT1WM83dAIZpyPpoErqDWtOM3TXSP+JD\\\\\\\/sRjWWESsyaJJ+C5ygEujOAN+0A+Zg4rcysUIc2qczK6CZHT01CmjGGtM5+LvM2ozavHUOFKzfj4lkbm5MzVeHNB6qP0BADV5qA8gD1XlikjLRvrub6pBUfQ4pc5B3TR5i4ZREEXQmE6dII\\\\\\\/rikGbUc6XW7vBdXCiUFj+iDIwQwRGyb9Cim3Q8CSM6k3GCutIiPYtSdb6lRGX4bghI5dVYCbMeZiemV2p84gvOj0YkCjIc0D3ff+zTZB1tZvkrY7+PXfcdCKrsvhwYcWq9XEC++4YgqQ2\\\\\\\/4Jy0EW3ZeE8O8rXTqIx2iO80eM7UAuowBQqd3cCZ8ksTUhgahSWAScP2Ruuz3JZiUg61VkpfKyGyjMY8+ER71MUXLFBwO9hjJNZswkXV6NC18fhJS0JdTx\\\\\\\/AapxQluTvqV8FjvvQ6aa+sYkNlXXGON7PdHAZ9SYB649LYgFZYazLMFXcXCmFVWF9F2Vm541E3jKJAxMHB\\\\\\\/L4t8JxAeFfDJyVQ7LOR9kD0PFfRb7XDrmgTc5EWeoB+KpKVdjB7\\\\\\\/x2HxSuzyhTwGQTvRuLKubYpsW4qFJIHeNW2yU98nZ0Sm5YK8j2txFJXTWF\\\\\\\/z8C2EKWsu8AJVm2VjfiWAtEM35azXRIinvrXvIrKewyAPbUoelYxuBeN1oJ1CNwGzq2FzwldsBkrAoWLRR3JSpPeH6ypuPoZMfrbbwiblB9yfGFHNPisu+E9ayuNiWe7Z9UH58zmlT\\\\\\\/s5uC0OSblZifssz\\\\\\\/7NltmMcZSmhmzu26kbcuQ2rSQi4b7PqfaH5LKDuELcc6KvdAAFcAX4eUA0\\\\\\\/AbAKUk0f3qp9paOgM0bTqjxVoP+VshJmgQnEqeBzH5rmsv6\\\\\\\/Cb2pGK4XU\\\\\\\/40nomzUKbb2RtMztQ8AFjwqZxoZyuCnvGW59p3zSt90JkZS3qVEcPUar9agbVowPlRr2bdkQWl5lw3hRBtqIqYB1NQ7PXgnGIcuIr6TpiWRszaM5kYVPKc1IKhAzRkh6C8pYFm71c1Yuj+9892Y0U7XDOvMD4PQm7SsfiZteH3WtWRuaE\\\\\\\/FArtgZyTOXs1H7+fGkoPMuYGuEZnKJHhxTrQYECPZ8JVuR5HTOb5d5Mz8atgvsOMoUOT0xPKQxUcKjVUr8DTSv91+LR7k01R8Lhth+Rxs3kv9Wi9O+CGxCU0cv5tMljPW46ipAGDoNrn2yGq6OPPkMiv7eXIV2bUEzaS54M09KmMLs\\\\\\\/jdfCQipTKK5lHFaBSD2slENoeFxZY4nyGgbxXDSP8sUOdfbmcH7FrCY0Rk2esFusVakQzEhE3M1mEiGegLlGd+n9RTvynXYYoArF6KE4z\\\\\\\/cJWpkSShtBz755jc78Z4pS\\\\\\\/8rOIxGY9rKgWtRDitUk6ep5wy\\\\\\\/4OEarTt\\\\\\\/cZ\\\\\\\/rnL6on9qm7CJXcoOdC8tiyqblmRC2EoMKuFpvmdp9GK\\\\\\\/YJFDphImH7fQzft\\\\\\\/mWbYtXChwUaajfmGQAUrquN+KdTBm0qfy+6RqScu\\\\\\\/ZhR99FvSy3D6+6g4ikTNBISt6N09+BFgs24jWJqMtNZrpbpLhA349Wzcxx6pX7HNo3awgKxn9iNKb8MYueSxLo6G+QM5mXgKpxrKtdcpAKA54x3mjGv7UqFzUBiP1f\\\\\\\/R+tsTISyH4qJ0SP3DnCaUI2qeZpQwjnH9H9MGom\\\\\\\/ETZfW1Er5Yr2HXs+vsgkhlchhISppHTz9Go0NYK0YScRdJcGiKG7+hdQ6DlKxjq3DXhzYfLNAzmtt2Z5k+Vjf7hywydwU3Ehi\\\\\\\/uGtyIvF+Vklptz0L4hSONFFPYmGcJzS7LJ+5upL6+uBPTKy2cYudJJMELPdRM3qYTgt4Ryg8KUEgrWWSjDciLvMySkN3v77NkJ+In2px13Cu92uvZz8\\\\\\\/bOXzS8y+paiCKvBqiARd50OmaRXxh1vVU3DxJw53hz+z58GwM1OoDL7Q0d2VPP\\\\\\\/qLyEcBSjTSQUPQTP9HMJQ5+zy22L48ogq5NoXsFJdvyksRS43DphfwFS8BLMol2E03I5ZPmGrf6sznmecQPozcJRaOCzAJDuY1ZACAiGH3gg6siZpjAiYyn5L5jvW7DMy6slKXiJjFcnyhLsKEe4KQCOAbHTZpibLgtJhIYFConcMd27\\\\\\\/VctkVki7N\\\\\\\/pwG4GVPRzIiIgxKPj7FdkX4OVrIrKarxCNgKHHz8UtyLbwemoypH8IFeSzhS9nBVakZIMf07EG0udnUZX9uiT9cQsmtQlpf5hh+pU3jv98tFC4fwzvKJvJjsUa90\\\\\\\/VjYDL35U6xmsbeS1jcJRTm10TknlYfKnRQvCLqzdOQthZzV29sz898nqtGsjeaKN6MqyN5WXOrr0mN3zVL6DTMZGjYTVT5\\\\\\\/m2P0fOIcnRzK+gNPzdehNUFjmp+RV17W\\\\\\\/NOIfrqpzTJdldG6KbaZM\\\\\\\/5Grvex1zxhJ17aM0wFEGR7WUDQ3L+X6fPtIY5ky3Hg6YJZr2ItC\\\\\\\/b\\\\\\\/rPbdiTdOz1BlcxYTclVbX8T2xm9elD21ItWva4ddHSBdXejdFCwDYlmh09LyqPR6lgls6B1fCjIRi\\\\\\\/PtlJKVuI8dcwxIJyiRTjqs4SHZ++qrCk6\\\\\\\/DqF3POjQuy7PupRDLGnkD\\\\\\\/K2VgtneCQBGtvopQ4hu1kZhdZc9O1RMTUobeBq1XB742YPKYMr5A\\\\\\\/kZMJteMD4q+hfN8zoGuQ5Zv+suieBWLKxYRxtXnvMIyzBDKIt8Il82WTGj7rzvayoewbPf8TNi4TKv2qlQVt+uxmzW4dYULS9lgMKEBYGxofmxJG8c+pI39uFnOzu0SzIdSTMrBTKKjpbMrXihI0tTcqL7vhU7Qra\\\\\\\/GTHlL9pRCrp8XjsRSVjdO0HdY2dzP1LD+zjm\\\\\\\/JhEz24BQ67Lr\\\\\\\/xV0CB9VJLMuZj7ZG\\\\\\\/z25DukXfwXpel2ENTYo+7xaREPyAi6tvQjHFw7SxxHUXC6GTzza4+X0Ka1FzfT7vxUvcIQOYtQUzlqpmFQfjPIyBaw82LppYS\\\\\\\/bMObwrVih0qorUPHzb+xOG46MV3JLKHdeZFCMAcNocVeohAC3gR+a33wfH\\\\\\\/blDg2orCOyad4YYsS1oIiJOdk3Vzte\\\\\\\/UPtQjrsZisRv\\\\\\\/nUeFqjiDgMW+kN6vMy0T1HYO6aX3Zp25Og60vcmQpx7Z40f4Ka6tadYqlisl1XHsO8mRqQtZQbjavOKOwEAg4nKmXwiU7eYcgcG01Hg6m8XVZAXOYUMKFVxesm8TR3\\\\\\\/SRSBr1VKqvKidHsn4JgNH9oQpyoX2dOa4uli3ZOAaNxBRBb6SB\\\\\\\/LbeSQ52OWBtu5n33Wdg0xDKZD1NxlYivB4vefYI05IIJRyF4zW9oNAF+DluREbxi+i4zl+VzHPDQXmEZaS5rRBKvajKCCoMC+RYH97g4uj2fYsqk2uhTufBQFFjfAyZM1wrisaPe7jGjnYdnB4ZBbwvSRLttZcO59TE7sLP1mw+1cuaq2\\\\\\\/M34dku+dkT1Pd+ywGgd3df7uO+OnMnsgYhhFvqGkJK+Z3+GBpSdiB+z2U4qNuhuyZvKFNPihEa+6BPsEgC7q5LxVlRcpiH2c02ukZBAWFYFobSA7ByUnXpFmMY0ok\\\\\\\/Omf1bdzUU09mInSW1jF58zwmVrdzOxtClMmO+k4aZyCeSyXfXbrRxYvJzfjmx0NXQEBzy1hER11\\\\\\\/BT1DMNQV7U6l0lb+57Xgg4tC6oh9FsBX5QpqWOGlPD88hAesDhNE8YHs+fzytGe26ZNcw\\\\\\\/yllJUy69FS9ObQJ5IemvBEVVy8Ow\\\\\\\/ECtGUtFoozJUvSAw055j62ZnYIMA4HMc5YWwmgXLkAkCAsxty2hWhO1h36igi8IG1o6ryBv8mXG7KYTky01jwx51BoFqgoiCxUkZiP8knUiL\\\\\\\/RlicPEyLvXDzXOQ97pAZ8KwQnFIH+eIb1Qyh74KMCAyDuO1OOtn5qWZO96s\\\\\\\/ZpjOQJQirmVR32lAKJdaOjNadVSxoWaLk51MKsw5iQ2RhMlu4\\\\\\\/Dc+lT4kKVRS5RN9vM1VfUC7BC+5s9GMklNDAfE5cAHDlK+bqRsKT81er1deKz\\\\\\\/thKMZfjb6cnzzcStE\\\\\\\/HiHy+AkaCYP2sHfPJAUSL3EfODTPfDuMG2YCc60FNNuF9a7qXDkoIjGCQCyxRNXJjJ9r1QSjCdf6e+vqnuV09d0CrpCH2jlNJSoT9GoBezfhQQyOB1F3LaJIUJksMkdwrAVIgdLcOWmcrrAITG4dbODVctGTs6DHYrK4H5aisNJ3BMhbYT4tox6KfB+Y9N\\\\\\\/Ry+2f2bHRpMj7jEKkrPmMpxT5Te7UgkzorIv5UDg1LQJ+Kd6BSsz75q8yKgcMsGw4dFdTo1k4cYisZNMUG66\\\\\\\/ga\\\\\\\/s63k6MZvxYbG0L6i+KVNXr9IMGlbyJQ7QbTQVL\\\\\\\/5tVxaZOtufO2dpDjoQaBFOxKgVfdV9HCPxjlaGWqN9I7mHOFC104lFOKMEy3CpIoNqI1ab8WEnPfC0GeHy9nuCKUuE3fzojDjSGcW3xzIb\\\\\\\/EFoUCZ0YMZji3JIO5JccNRGiKqkwEAlwKMQWgFIRvqB0giLQfebdqMZ1oSbgcaT9M21n7VH7ew\\\\\\\/K8jJaAXkIb5F46JpfMg0eMwzPm+IrGtunRIfNgE1xkxR9VwfpTSV44tkcuuk4JnZOxqL96gJSwmwMvFUQ6bVGT0uTT7qqlHv6rlsZOXjeiSMXp9uLdaYymwR+er7pO8QLfskPPFJZTYue7q8XKljzL2OjE4uWdddwxTbq2FiV9Oj9J64Y\\\\\\\/x3IGibela6rxui2UDAzHgxt4I97BCVYVwTuA\\\\\\\/LDtmJTUOCkCbkUJSOkZ9+5d1yF+1X1dIV3WiRRTXUk1Cqxmwc5bn+Mz7OvExONmceu0Oj82v5IxCsbqKywQqF9N3PKDsw5zHoTTX8v21LUShAdHvQj7vv3u3++tF0yCXzIHHOrMbQG43YjGKbXP2b1jREc+Epw6GzL3mE9KFEHoLYvawXRkUd6ucXjoBKPu45zd8Vcrsy4AB+A3la43TSSswNScTIppNGnJ28X9Cv02+BZLgQ81kV2DOBo91fqqrwetwZr2vp6JUsIzxbxYZbJID2QAjTnMLJ7r0IBAnWnF6PBVcVMZaSKDQkTeAiEJW3bzdz2VUW39wKjphH4m+17kH48mAOvz9wB9BCwSwDOkUPNneHNAvBjWTE4zjF44jHXnq2fKo8aKXfe4kD+6UR4erGxmP8uEbRoyB8nC3Zy0tGF+oHl586lFyuSeadSz3\\\\\\\/kLtkRui\\\\\\\/kicficgEFXehY80rIj82+v5JPHcU3E0+xnykf\\\\\\\/i6cJe7aj5RBwFNQVEEDyboWxhyHz0XgrA+C+X036EfFI9V9GRhL5GV2+p8swYBc1Afuc5+3X4bAxSQ2HQ65FY9uSmG+pdOiAgLYx4Gp4CyiU6etn+u4VFjdEJrK1z2bLpiaeJ+L+18M4YQdl+IDUw8VrYyvvKEW\\\\\\\/myLSKKNFVJAf5ktkCjEAPyMcZdBJ5XIG0ehYkxxWFVMi4u49Scee6y\\\\\\\/LXd16odOODRnIwZ3m+EdLGfObDRiJu4T+8tRTdtLM6ZWnd5O0Aeg6GS9\\\\\\\/ZDMc9Cd3WBwbwZS5ahItj90+8QwgSptuzPo2Z1F6RLiPka6\\\\\\\/Q3r1BikUEtWTn5MeBU92Z106+tFI3TQAXtBNBBq55T7l7usZ5Na1uB30fMfEbnpa8fvM7lrMa874iADZ9kzgm4orvdEePVFU1VkLOYEkm7OK0ijaEjPUUWJ6WYhNM25E\\\\\\\/4jU\\\\\\\/KyDKfLH2+OTamsyQEZPOoGgOAlgpJFsd5O1r7OWFMVQtdh9GoH3H5CQTA5TjPQGPySiDKAg23+pkEF83oUbCI7ABDJaHU6EJ0BVt61YFJBpsyRwIat2VbJD+28v88NhjVVPmiSdD+Mrmsgpri\\\\\\\/V7hUPlIUvt5DmVp0LanS81G9pPYut2t6ZlAxwJOxZq\\\\\\\/FJOloOCkRKDrJakv3dNogxedUhODXmfAzJXLHiXgswHiTnTcuS0XnZZFgRkEZC7MSWe2Isd0VQonZdxK+QYo4QfvRTALdletTpYvViaCc3Ox39WNRLXR263zVWIFyARBqTcHweI\\\\\\\/Ce8pIUmbJj3FkeEcdYS4n85eYcjBO5fJu\\\\\\\/8MQgtrUIRFxG\\\\\\\/Az+ksLQRlcBseITcdqP26M0wLyZrMApe3kzv2cCwKs2JC3WrtFh1ls8rpXI5a\\\\\\\/FAJOrE1ZY8M+tPCnzCU7+OsHo407RQQE5FgWjhII6jdrOMq50AbxxzI2vl7I3dza1\\\\\\\/FuHKYE3iqZnqySYeEVGu+MJSGx9yd64icmerjAjCkjibaAFxVYzMu+RGaWuKZi6Xwzv\\\\\\\/CBjmSr3CYZpV9vOhfoM9ZyEd8ftcb2sXTHjiK1Y0cjJWBR0UkRwk49+icN\\\\\\\/p\\\\\\\/pQW7K8LSPE6X6ttp+5+Ln8JoOfYyLKgI6m8cesr7nqyhyXyPb2tlI0Qi32U3GyBN4RNh7AxEbdAAXC\\\\\\\/Ckqmpi\\\\\\\/XsgtS8Gf91RNtcsCE0Uoa9bL2GttPrwyD1AqpRIkIkhY42Ij66riIQBoG\\\\\\\/+uCt00eSeYmI6gIN5yg+5FPdLCaXtprW1Og4HvOZJ\\\\\\\/Vcw26LiUIqy2I9NuG6XAEAqz9YeVLq63KTLo8V9lN0D62aytFgoePiDRtIA7xFdimImZ1EG7NA+JA2t00urR3wMS8TwUGRRzi83UdFAGJNbjCRBF8uvLR6wsa9n2sjW3eX3S3UezWR+WWKpWskMDjGBDs6qvExNZ6SFydr7\\\\\\\/eYUbpncd+F9Ci\\\\\\\/hXzIXFZK21mphvECFiFPw9j7mO46mANJxdKGl8lvk6h9x2eEQ6srJ\\\\\\\/2b0d+YmI25Ptm8ZM7uFD\\\\\\\/rg6SMhE7jDTlni3LDE9E4bxszPAbxcMFw3V8SN+WCq9FTeVuVwEkB0gtk7LSL8hkppGkwRwYri98OGVVVKzrl\\\\\\\/LJmgWcwU3+\\\\\\\/iyEk9lC2UmtaY4sNPhbZcUbWs+MlgRry7DoImTLyh39gWbu6pqSCEiCwlZE6Mw8iNXl90oYVrC0co0ZR1w6vWrjQipx5u1FiK3IWFeCBzUVJ\\\\\\\/ueJuuHgLP14KO3eqdLf2Y3mbQ0J\\\\\\\/3zoekGfieEuquZwrM6lVU66Vf7wVGfiV\\\\\\\/dQzrE7yvOsaRlk0+pczjzEy\\\\\\\/f8yGpoKCGl5MJGGecntDbchk1f5hFAuqRHxp\\\\\\\/PsUE2TBwEeRhioyPCTpgGtsZ3QuERBH20J6fw9dcpiOXT5Var1Bqp4QcqTEjLtM8d9ZId8aj0rElAhbNgptzshcMqv6ZOu34jV5WMB+x1BCpUVxybOTKOrKB1p0u5Jjf0vODxkdMRMZznW\\\\\\\/TMYZti98bv+YhxSvgUtfhEdSC8IbBoo+B28ccE6kuicRGeF5KnZUMooLtRE4hREflTpucnkg3rk7uPW8qm225yu3AUVtsGj6sgnYg1PinaXOW4AuujV9BWRCiCbEQlXEq5w6IorkTzQCiPlu+2JsUN\\\\\\\/YS1Qjpjl37Pg\\\\\\\/6gwdva6+eyReWkdbKhyNJcUQpUxBMRGy8VsE74gsRQD1gU03GfK6LSMEqQnp5aOqcAHOvTfCe1l6GEmigivkmnlrStgkNRt\\\\\\\/9dPFdA2ybJPLeEFObZ8ewcUy0hO6xeonRCQPJcXNSWUXCGBztObklT49YNndEfXfhPVABOIwMa0OOp7TOjyvPEAXXvoPLWA1PlTzL7XyWXVcs3j5kGGxQSvWEs6xqNU5R3uVL\\\\\\\/yYDZFEN3C3rO7PqDExTN2hY1xUjwbMg9s4HF+1qNM8g06bLE7Gm1CWrclYnyhllY9bo6Tg\\\\\\\/gY9OgN7SThLm3YFFC4ec8RBfjgMopE8UVMqo53+I8jdfUwm70exawYB9Po0wZGnMX7SX9R6wBZzDP2uiEE8VtXkc1VNsn\\\\\\\/jQL7KhIDlvbrfTCty4PdpyFfRl1sjOg7qpM2ZQxXF0VkZHMn2XMe1iMN9T1ZOftIKdSN7iKUNsa761gDwis6O\\\\\\\/BWzlCpFcpx8RPx41y8\\\\\\\/pqRjkWV2s0svJ6zDyWIKPuXSnw\\\\\\\/3whTtmZIDYFQIyueV3lzCStHdkjp00y2lZr17axyTgmrl4BnQL4P0QSeOhXAc5TBgX2\\\\\\\/Xv6kkUGlu+56RuLqUAHSszMIKkqlYdiHDcPPrW7z2SzY5syZsAUoBKAQM40hQSqziWWY+0FsZqb77XAKh2nfp49PB2cY1\\\\\\\/6DgSXdOFZd6yv\\\\\\\/DsrNa4ObHdM2uE61JomcdhfNk+tJQSRDCXTcU\\\\\\\/W2PFVBJ1aAC0Czx4R5N0YsQmG50fHq7fz94Cuvr4\\\\\\\/YjBKkCSgoYsAjUfXR7ojhs1dHkdPO748tw3b4nldjVf5B46NMaBBiqplokRfLpzFZvm7+rQLdoe0fliQxPF4\\\\\\\/amZtDLUzDxo4AtSRyYT4Yd6gPnEc+WWQmnNfBQFRKmqps0BeimSm4jYOnA1eTPpW0p3uL7g8PvY9Zwdydlw8xoEpuhOrZrGHehnJyvK9ycqS+uw\\\\\\\/y9PKyRGlwOL+s650pd6eJIzEbsIOQafsl0qRk8\\\\\\\/Ha1gDFkYTGRi7uk06g0dzmS8KBDweFKX2WY0s1aV1j3uwnJycRn52B6quZzeSKq0+wvtaF9C9kC9yC1ibvE9ah8JQZ1xp8ri7AiqKiHAfrFx3E5Exrx8HkCD10xdwhsmwOnlGTQWewd9So0gDqaC7zLf9aChJCnIg0BmDCISDghqMeSeO4w28y5C8+6HIHrRutKAYSR7saP3HorlRR48EJ7w\\\\\\\/oeZsFRPrsIRDctZO4z+Yj8eL7UAQTMzLhq94LqwIWCl2R0rxTDJDMsx8F\\\\\\\/was4xjVtYsR5plZtzff1uqSVf0vqcbjC9ZlVW00rTYKkAD5w+lTa29SRcMlILBZ\\\\\\\/b82q44HE29rwDYJ23B2ucKg5b5rtc9jNOVvyI56p27uy7JBcy2vPqoTKCBlZ+FLNzoxpNucOkkwjNBKIKQ7QHFOSwc3xestMK8S8znkQK\\\\\\\/CvESO2Jahgf5jaD+QdUR2RI3IL7J\\\\\\\/It2nVUMvit9ytiDrDlT5FKiU+YRpF8duGMrZ+FtVCP4v9HzMWCaNST33YeYiaaUiaqsA1TvevAOCOJN2pDcvh4Sx6GAR1v5CoT\\\\\\\/B\\\\\\\/KA\\\\\\\/ORerndTE9gdNKPXHzlHIvEwGWDrvR5uWbzCAfUO9urInCYIlTsYw5GV+jARjpkpKkKybPHm9H3dVgpxIUk4zxRrCyQ4XxXRGPzlIohXNyu2X2EG0HlqyOle0wOFlfMyMF7t5QFncGHHCv8YHOJbe5pwv7uEuhzU27HEZGFh\\\\\\\/6weV6v7gnEs4qrjjuaJppvgZVmNI8mSpHAE6FJZNP3T\\\\\\\/NBRW7T3z+qUW4WbJENUmQObs4qUrEGFysOOUwfGe3GOTGgKHwKCXwetxn68OQWjFp6wyOYVdw75pXWbPG25kiDzbQY2tL0fmGR0cKnTol6fQ0GTgIwRa8OQKYDMJO7kSbLkLbvUmQ\\\\\\\/6xulwBITCa1PLpva6nLEleRB7Vn1cC4hOD\\\\\\\/67fY8qVcr7QaWQNEnOIG7FAhVktBABTtJJVKTa7ccIuV90h9hNGjKR1gg8\\\\\\\/7d35DS2qd2hm4uukdFzZGUstM7vuNLD2N2wLLAV6UhcbUtdz8THSsjPbCyUISaiG8GhIpjN49SOwecUzJ0E4Fm22yZ4tgOUI6RxLOSXgqy4BCxZo5nt9J9LvaQct45bYcHmA2fyKkP5SF2ymWomH083RqdRu5pMUOtFZqvjuJvmqFhcn5\\\\\\\/9kYtVUaB1xYKtS0VXOoP09sxZ9BDSWzXESniuA48eh0j+F8ksWfANrXVSNsL6RPNHxXdaWhhaLorcSkGf7cyF4v+1Lc68OlzCLy45P84cNIzHQGsJxvN3rA1yUAuBQclJwhTiKFG0pT0nsKduYESWHrHqJcPbVacov6YUdMTZeq6Tsq3MIh52N8hd4M5YDTNMq4WYX2TNIkpvUPNc2P7nfdY1nUtVWfavooE5IfV5wb2Qzy33+6DRaVK\\\\\\\/j\\\\\\\/oaYSok6sT9nKqLeMOp0jDOE0GEY3CPoVK7R7oWDT7Jbhm\\\\\\\/fyaK8U10biON+dpxzGyjFaNZLk2ob64zwuIFyR6eLWxoTaiUMEXpPJ18qr6C2JgxDfNd8dzUjp5sDPmAEGF8qHArfrEQBkFbLFL1A2imnIkDYiXdtQ33JfPznCeuCDMo1OlezE0Zkf8kj8n26mBMaV1tli\\\\\\\/XgZRPTDrGZnV5P1O76KXiK1D+hqsMmiRbtCt9KQuEYlLaBuQFHx6EaldTr3MgapWK5CDnBxEuuR+9Z\\\\\\\/s5zrsyOGVvPVf03tsHr1uZRT308NC1UgFhjMST0Mwr71VMNr+Ym5ApuVsQ7DGAuAjBLrJ63dee\\\\\\\/q8v13wQwnXfmYNOSUV9c5RL8mznenkO0iTL5GHB5jZUTMgz3S2gcnuns3y8qaPlwgoMuwYAZwi7Ofu7vXML0DpI0EEnL1sKSAw8aFdy3bXD0pjSIwuwNkeWncM3lt\\\\\\\/1HOncaNhnWyYcOqgNGAoVm0KYnIBCuwb9BJLVsaeV6XFrz6wSdT\\\\\\\/DWGXyIEi7Fy8R334Sj938b19PNbzL9hvseaLicNQF4STvynyzegX8255Gb8GdVsWiSKGazdAtO2xCWrsayNtw28oi429pzcwck2lC+o\\\\\\\/99A05cJqKhfkM7SK8t5xz9HVQU0FQ1EF9tHHgfTry1BKgiFPAJBu5QmxXb6X31tC8bUAz40RQyehDjsGnHdNyyf5A57ErXUd9xO9JFDr9cLOp32Y9U4UkivxvZe7U1qroGWrR\\\\\\\/P1463YmQEmf0enqiNvOKnf7RWhiiyDQsHhLQSZQAV0x9N4MbUDKx9NBg+392XV0hyGH8r+EEjfJ1r6p1mNKrWWNB5UDaaGhuiqz0ZyktpVIw5enECtSWeIpe9WcN9ZcyMUKrfKe8FOHvhOlAmcl5G0ZHR6njWLqYddUvzxLOczYEwg+EMDTIud9NBR4YQXhG2ydb69CQp9HUx35RN1+sObxextdJq74GVU2uEUlH1L+yfZ9DWV6+2tpnXSt0T8Av8hgXsJhma4g8qXC\\\\\\\/TIT3A7kCQEh+FxZBu++XtSyhvoctzmXzieuDx7O7G\\\\\\\/PYLFFZozOQ7uM1y+T4nLDyW5jVbXE08vLKvrsdMFxFaJCaXcXVreklbq5YnxHkParbroKtwX0KYSBwd6zPNvaovL24gI3IHtY2dkDy124KMXmEM+68VsH\\\\\\\/NvWNX5yWShX81cZ8CzPAU+IkAmU5fl97JwDwyFr2p7H9nE6NTO\\\\\\\/34pMvYBAI7ArweWkC0n3l+TUOyhDjU4RhRjjSWcRmoDLJrXgD9SBcnYL1RmiNRM7guPY8Rns6y36GuCHHE5JqRm5nIWowzV2dkTfn4SEotpOi5+l\\\\\\\/LaJYgzFtgXrt2CsmBKt4cUM9yk+nSJogrNTY+bO01UTmJYbnjIP8JXS2nOazThpY2gICqJFyrAcgce6Tx+C95c7f+0UAJyoSdRT8aYo5urCgmKBerREEoF3sys8nwHoDNQhfY\\\\\\\/YEQ4ZcmfVl\\\\\\\/7RrvLcwLiLGDnjQdVKr7QFHQ68Qw\\\\\\\/g05CGZ30gTOxAokMwVrRrPZEtU\\\\\\\/5wlVNH7LnKcKeKiVmFwDgnd6B9Au8EwCT0O9npqliSjO7wdlGRYqxSgY3Sv\\\\\\\/1MrjAi7ErARpya+i84AVSr+2QW5mE1PbIC\\\\\\\/zQqVL42su0BHWoBAHwHivDfFXHoS51beVCBGQrn0taCq0xad+HmV7qIYAL1kgfZ4iB3Up3ROKv8dBDpGE97nLgEPCaXcOmQmH0KyHgyumhu6KltetrOR5DGzv8BUjDG1R+lHVSX7\\\\\\\/Y4PSyDyPWv4FWF\\\\\\\/iNlyq04EdNkuVd3+pCJm87UzexekXFCO26hvQZThA4PkCi7fZp7SUyV+LczB7UST470Q1flLlSgpNgg8DQ2fPjxbhqwfJ+2lHTnv+Rpc0ozxMOHmFwBY8yqzkhjt9VaXVZRRQRt\\\\\\\/7uWTz9vMme4gm0XFOV3JtxW3iVazuWN0gt71eKHlOOGrgFdcawxYvtEKLMZr+n6tEiL5MjwmVmxHVuqukOhoqNnw9xeSFfFA4mkWA4PzcT3EzvD5V+oc3yiPGHOlEDpV+FL9BjlZRKWnWr7DjHSzAdapBcEc7kdRHWtLs6+EcKBo9jaIv9jfxOhfmx9T+ANgwNdmLdWKWPt1AqjSR0m9IbtM42Au1R2Zi+9JFN84gVVjePRa6daFi5hElWQgHQTZgXiOhmS1dacXRo5K6TuZqGcEUaYyNJzNmQb5EpVB553OLkRwSqi+jJ\\\\\\\/woWHfb6c8D7YFqDWphSGaiGbHcNgCCmdMJ\\\\\\\/3yUb8q2qReWx7FK15EWgxSNuEP0wTDKVgTQCRdylLRy6k4FHtE\\\\\\\/I3uzIF5VfOym\\\\\\\/I9KeJBApdnMglKLxPEpTl8tXSa\\\\\\\/IZ+mDjGlW3CoWBa4U2NLcB25AWdxHoeWGoFA\\\\\\\/14CIWWNs5H2MgwQT3TU7\\\\\\\/6SJblArRWqKWDNWnFAejLATQg9ws965bhcdrCjCjXFwIOmtdWenQY9Uoh4xQSAA3B9QlaVruf+fvSq79tBvTWAvAcDG0PTALiLZogAyNhkIotqqIwgacEJ+tL3ltMhABpxXFH1aK8uPnTtrbbIMX0Ztm8y18fCAxyYA1yTpPm40EBfKXhOANxE3jLufUljjLW0owfrAYLvjHd53ExR0Am0dhWJsyxDjCqch4uJ8bl7UgaDdu83bO3w1aeC78Syxf4OU\\\\\\\/8iFflSUKweREIY4Vl7UZKBAbRYDCFGx6ZSiYx85aMd1IVrWLXIkVLVgcHOQuL\\\\\\\/PNLpwcOklv9iMHORXcCBbAQA15UQdEszrFNwPNDZeUeC9PwAhb3CsdPzqG1+im6Ii90glQWpHVnJjW9bqef2UmI2AgIlipRLbFUg1e7zzUeAzep3n6NAL0LnVsd4Xn+W\\\\\\\/PEi6BHCnFUrL4IwGVjrFSYibNDtNdtfhkP6mhpa6FR3U7m8JDQgwCmTKOsemem6O8um7kM1YIbLARsekBwd4jfM9gjKKWikplPzPz7CEXYftQGmhWJbPlF3DE5MOvSKAkY1eVHqtdGMf7c7e5ywJdGXAORTCFR6a5+ib8zs3z3dKHCHjyjXqZBKY5cJwnEkdx6Zs1tW90ZdHEg7WHaxDP\\\\\\\/rL+mXbt3VHdFcnKHcl+9F43ZTGpmZZXoad88zjqjdQVagcKA6nCW\\\\\\\/H3+LhM8yCz79cn9H+T79qSOF+Lh5UHfCQn2qFWagsuTz+J+OTaZhHomhW6LhaSx3SWgmIdJfsruNZljW3AUiUeFM2JX5ANTebJEiGiChVYsGBjOxPevi11f8r2jRc887qjTeKeb+NHRGwZahhS7aI2nqJ+Vm3i1zAtB9qNQW0llL\\\\\\\/JS0Tox7ro2FdV4++F\\\\\\\/xK\\\\\\\/lmjbMg9S\\\\\\\/bq7KX3f0oCcPYO6pqSK1AXV8I2z8Co5ePthRZWGe4AuLxC4u+lgnZS3L9Z\\\\\\\/u8G9hAx3QxXPSjFNB7xk\\\\\\\/AzsKHKjFuKtNT6d5wv8NFro8WTChlzzygAvqNNtx6Ycr1eWjRMiPMFhBfBEemAyKIQdwVsq5ECchTahT1B9QLeIeTUENGWpCD6fbykrtX3RTGqxjDLGPUgC5Bwx5ck7wbNlWmWxtlOvOLT06D6lzwdkrtc\\\\\\\/9rJT0G8EjIzydaWpuovx9c+Oa5SWM4Jz+2kHXsWplSpavFl20TSfDBnh8\\\\\\\/vYLJCiynBFaKJ3B5SzKNv6fTnIT6A+49fuecqZyl1CrChwGHnivn+GmFNDEx1i3RlA5zrYPgAH23acFKhXtN+buv5IojM3BCDTebGkQJ7TLEmA9yMiTyAqCaxr\\\\\\\/4qyb5oxYbbUu+iErEyODXE8DjLvXAs4E1nxubRr+eML+SYJfhwfxcMxEiDf9jap2XUrR4YnXCtLojBDR402KlmDOlYzzmJSQvKKBNUKNBkgfl2oREYqxRlpin9QN4Gp4NKk\\\\\\\/Mo2usqhBnxQlP4ycRGSaB3uAWPvdF1CjM0VMokHe2BwN\\\\\\\/C4Tsw9tgBYzZet9aLEn8NPtJq\\\\\\\/IN77ngKbze800Dlcm6dDxIQWyLbPSEm1OSqpazbH0XUO3yBaj6iR0WWbETRUfZVRqsWoerllS0lqZ3g6+xh4B6iF72xjl4g93fJyXU51N0T2xjG8\\\\\\\/KsS34Oyc5SHDgG0Fyyp1i+f+OLqVxTnJBnHOFkS5h7Lizogct6zVTKa3uiPq4dP2j8pvEavRJV3em9Ceota9YXW\\\\\\\/JtacJ7LGElQykEaHZeX2GMnW+U3umlqfOPjRw5dUu0lM1zYALUVU+oVD+UaovHysdshBpEpuVYrWS2QT4j\\\\\\\/VDtBu8WrC6mo2POp28qlwjiBBpeg0A94jyalPP6Db4dqLt0eYvHjHG075CyEzIN\\\\\\\/GPXGmgzW5Id3FB7Ylt61MAEkiEn\\\\\\\/DshDfIB8if+JupFKiZjEwfvNWxtaoNBBCRB4CKcmF\\\\\\\/BxyRtziFM7kgH+rZA8+C41\\\\\\\/BzTe0Fe2BCiOgXPIjPpOdq9hCV1lFlLJQluvRj6IxGGOdp22iFwKH0zd2ERZ29ItQJMrWIC1szUQ09MKxhlTqnTpcJ1CluIUT9KU7GwtD76uZFWvoBF4A\\\\\\\/G01m6UPhgslW\\\\\\\/JUs6DTJglTrrrFIMH9UI5do52INNoJgnKYMeTdSQjJ\\\\\\\/Qpa+bGw8VY0Z08Gj2BYFm8iXbqK\\\\\\\/4IPYUL4WMkVgwnP3TrOyxgbAgk1569BgJe8V3np+UiU0dUst03+xzmYlUe7BVYOU7XZ2kryjdyos5zm0HO\\\\\\\/3cJmOi9i41J6JdM\\\\\\\/l4bxvc6qf+IyctxBMepWgu5DN0d8r7TF4J+VoSqIMdPK+xUdpgsXCTOPYMnq6zEXaazdRtOxmdWHsbHoc5zsLf4Uy3ZRDhbx6Qo1XVqo\\\\\\\/9i9RIaZZWP22NOCOqnPCeh659BZ4dl\\\\\\\/qj27PAttZ7G4qltvM75N4s9nvWycjJm7TMbqE4Fj\\\\\\\/xVfJMLLvxLbY0Zz95qojchQfDgq0YDbkNoC5fJsCWpnsJ2mvNx2DL5tEkN2oV9xEJQgEdDx\\\\\\\/8qIz8GxT1INJa3JUcTb7fMPyYHxH3M1lF5Dy0i6clLFIYYCO8H3sJrAmXEW+2GmOR52lmgkN1P0MrFv51bwFYgfb2AjO8U+ZvI+Q\\\\\\\/gisa4ylQ4oFIztT0jO\\\\\\\/Wu48FatC1\\\\\\\/YhEjtLxS5okD3Z9PPvbhQw46vivZd3TpO8oUqQNHJSkPeDN6GAfsOWvAmivo8xuG+feQH7BC7kL828TlBJ282ilBSyqModAygOZlYsIjaAVuaaX71OylKaR7RTv4zDBn72EcglWptrHtcPW0VU+XQsJEV+q8B3h8Wef5bRjiOTP2JQc4\\\\\\\/TSkJVCQeQAuQf+m+vQCZoKvVMrUaGjWFgv\\\\\\\/cneWxY7ePSdp3qQnn5eqVTV8dtGWC3u9QKT1FXaIJ3M+JfpgN11i8Vr8XgW5cROSNfYShpRqOuD6LemnXypa3\\\\\\\/sobb0bRNClTIOtxEsqmFbs7jfR\\\\\\\/XuhK5yZ1jQh5wKnV9Bau2+W8B5zcuRQBb+vQf1NF1JHFX+ZC5iK8hS9OwVAYD9XYcib2X1+O2EBlW2qSXnxIVHfHS3UcxIFscUFtAxfVhjhXosJitoOMuYX5OVQFcyxeov6c033GaDozmBNjs2gDbcCnFUZiEj7TZWcosnzzixUYP5xPQHDYWBZBJixzeNPxj0zqOI0Iw93Ui4AJwep8Ba5Jv+he1TK+SxYew91J4AdSHczjPaMS9BWfcVC2QpOUeIDTXV9qt+uygBM0OIrqAmeXOaxvEW1pgrtBlwH8j0DWsauZiUbFtY\\\\\\\/qAq+on+3p7awpMWDpmTVk1HNOBOPlhy21mylvMs8PdZ8xw+RD4XlSeTLtAoVe0syFm2XA+bVvZDUB5xz2eZ63exGtu\\\\\\\/tt5YBKXfOyLakqYd1ivFZCJSll2LFfwrYuM3OfiZFyhtZvh+kEa284d9rSVgD6DEX\\\\\\\/xpE86A\\\\\\\/KH+mPSb\\\\\\\/ABLovD2P09\\\\\\\/IN5HzTRnSLfZuAQTQUb7TzxfZ6u52BFa6dueW7FPI38lOKLKBXhQJWxz\\\\\\\/zsNqS7hMpjC8cLB5g21Quh7sxgihCOHA7xdn989VUDzaHloA\\\\\\\/8IwPcTwajSkv1KxLIDxgmkfFtsgb8fpuz5bN34chWRDtrN8uXo03m0753bVOTT5I0kDxqVFZ+CxeG0WQqD44518zRHvQKJvcxxwdVxkR9EvUb8UJhgTczlLzujDS2CcWj1p\\\\\\\/S3dGdHvcZYUAgcZrEn\\\\\\\/62eaXfXEt04lpkVLq8clUe999hxGxUm+omgXXlq9U6+GOMD0rkOjFOEsLAgrvJ01ntKYvnTtqAZ7jOHpLXFvVqo5NE7zfY2uyL9iErS82aXLjm8rF74qHxGA0djkt06niTxNHRXOl8ey9feri1MTrYMD6S7gVN\\\\\\\/ObqtRwTGM+iOH0LIVsdYBqEE2X4bbm7f2VazhDIUuptkgSv2V62MvHYhVktWO\\\\\\\/uOE4JsJf4yRi4EV7tXVd2mOzLc1uFvoRmVxusgTw6vn0tuJ6436RTtc4G4\\\\\\\/vxuh2Ci2wpDNPqhBd6v6ffQR0qF+eY0H173Wubtqed1TA4DPw2KoAhbEwKIOK2yGc7QI75vkSE2yJPgp1bJfPXfuKcU5\\\\\\\/3HuREToSUcZFUYNkiPXWXjUgk+YRslqueNukx4r+542tq1j7InPjG27jjEEaZHp6R8eO+h31Ul3eyZ5mwGgS4w6dpP5l4RP1fRXUKPI7DE8gemO68ywf9RGBC7Vp\\\\\\\/8d00N6tey5yr4gypgzlJYlr+BMgyWVD3F2RSS79uKfISDH2TBlUj072QfZ6tGKcUdts+7di\\\\\\\/lda76IDVxv\\\\\\\/GV4lpAk\\\\\\\/tUEFvQXrn9iZBCxSddfIOUh9x5B6dih2HfpJpRE8CcFz5IbOXARgZe77myjlhAojBKYb6lNrNsjmHxqbQ1zhWUReQ\\\\\\\/TQJNUI1xltK7rQToftpm\\\\\\\/rHKOtuH9vK8ISelNyc\\\\\\\/uOr\\\\\\\/TVysG+3FbSQQkcizzsdwzuZwMVmIPLP9zH7BLFyUnNR8mQgELLd1uNrOM\\\\\\\/n4wy3E0AH\\\\\\\/76gz3w+vmsstBU+dPmWoznvlLKRKthVMLssFODo7dvqtJ4F2RHwYNpTwqEQfuuumTHAOnxIIm6rvs9lzuYfojjsbG+3utN30o5z\\\\\\\/GpdfHed6dnNWXWEr1Vr0ozz9m1R78b7XkNLil5N\\\\\\\/JaOeeZVr+XHICAgyerWcVCJeQW276A9CfRzN8rlwOxxwl1D76si8YQNpA2716s+mjSqRMdSHRayLOEeWBWxJiLfErJaNg3SEHnK\\\\\\\/H2jaUYeIsWm6dWejQ+j3M0mZIlxdL3ifzx6OkDh6mTL5aWrvHYsLnBXJ0lMpO3yozCOTrFAjAawire0cewhSCZs0bfaxiqLOfSWd2PUi2H5qfmZGxp5rWv1PMUCRgLGdSKGgzhZ0TSCSlGbWyQeVd6XG6rw1BchkEB60uS89xc5YeZtZmAqsoO6DD4WJZzplc7RcSNLOLAgK9d7O\\\\\\\/1SApxZMY7ZolBu71noqp7URzS6HQxuwHBE2cFgvbxdl+lg4elH0VPB\\\\\\\/kdATJEJd\\\\\\\/HJaiA66qUBsnE1DpajSRlOTw\\\\\\\/YDpoUEZDZw7ZZpr8QYtXArL03ZFjFDmsTcQINdAXgAPya\\\\\\\/D0OdGNqDi20ptkpx4\\\\\\\/sQDTcM\\\\\\\/WIjb0gufz+\\\\\\\/7Y0QSiBb9mdbudoAEMBaOrhawt1cD+xvc1QI8PhvNTYBt4ZSTdAhh8JAofxwKCSk0ekXfxZZP3oPebwtbw5VyQU2l6NryV9y5zPzSA2t4KHdhCJZ8kD6HkD\\\\\\\/K3k0vcYs32kKuQia1x0tFgT0+Pcy8J\\\\\\\/fl5ROI2sUcgclOdN1knxYvnhTO7IEktty7s9AlnYGlB0hw3EXA0947748pgwI508BPauV3U3vvfYA\\\\\\\/AZrcQun3UWvp+VTQi3Her2SSmfcPZ7M1CI8gLKNyZV\\\\\\\/2uCVKVItd6Ft+KynnkeMyLhqEpSjxmEczeYvyWR\\\\\\\/O9F4NH7EkcDWyQFt1MiPEmK0vhTw625ZjwfOLRMcB5+SDuC1Gd7zIHVlFdugVDvURioPO\\\\\\\/BWSDlJxMDfFhWCuci3wJ2ydTv5mI\\\\\\\/F5y2Nv2Ss1ztlNgKrmncT+9yynwZlsyplx\\\\\\\/YSySXZapTBp0Qyamx+KZH\\\\\\\/NEoGn3L5TaZenOrtmSAJ1ha8BvkIvM4uvI7qVaKcXMD8vf7Wtdd5hvxSoAXqE9fz9thQxI8j2D8znoFibxz+RJSdsWUZW7VLfNTLFFtMuY4N8+WR5h+QCWCU3fKqRm4USzZJgdaoFBRVrJarmrSUBqy\\\\\\\/\\\\\\\/OfcS7r0Mhimq9uz7kSRt7BXqC1aULHADAv8ussshE3+mwpnb\\\\\\\/2\\\\\\\/RI7HNlW5fOI++sgb5r823WD8kYnmwzKTUPz6D5dE0T90iXN+26\\\\\\\/JWy3aDQFwYVIKlcz7272wE6XoisWmTFgOaLN\\\\\\\/5vYxECnUSQI2BJUSskPyAkOMbK+d\\\\\\\/5CPrqGyqVlrgNmvA+Kw50THAu4I68g7ZPws8\\\\\\\/Y8cXUJiMAmT6ogmNAjdSmPFQY0KAGgowS0tG+ssTOx7d7SgNnew\\\\\\\/Gm70aAo4\\\\\\\/PKoPFynbtfWXqMB\\\\\\\/c6lBHIiEJVRk+ZLKOyyfaBG39pmkOgBMjHntQQxlGz+03\\\\\\\/c07SSj3XtLnA+F8xVe59OQ7AoWKPYimIUXnASDDQLJrA2WkYboX9f3tHDBKpm3L3g4Yc5kAvNgI4jNkU9biY5jUoJz\\\\\\\/rH8xo5Us5vSOeuVPdiOHmfyvRiD1TU2IUEAkQP5adB3DYUKtCBnhZe5nSkK8HaG0RHxnKuZdivteBv9hv3SehaoxIYviFeGyBpu3gnRJpiwMzWkIFnCpkdCbZ867pSHJCQ4\\\\\\\/fAtfVlWIIP+NR1QGlILnbPaBqkVhxklqU\\\\\\\/n16u3BUiEL5Tjb10Kw353VDoVcSqvj4BVgPB1LSW4kE\\\\\\\/wInOOrmYpYGhiM+5FK23Tg+mPU3CnwYF9xh0W3teuBg8kaCT5dWHc5vgkKPAu8nJom8wCtio2yUydHmGQskT\\\\\\\/tne559hOjCiDZX+krB4MfBtjfQA+KaZnHxFcfhpIgfju2s9tSoeGP1JxFBWJGeFFlef9EVfB7EZSdj8DDduJonvpCWUhSQIvBaBgsgzza6wj4SG1hrM8eqlwvt7h4LrDYAYqxcvLzR4zbr6kPpGE26D3M53S4P9T1pZajb9wwzIi1mygtWye40g3cMErYQ9wxWrXesHvBFouyLeBlPkjn3j6GFisrYnxLObzyTEZwBtvwh5LQ1wJhefNiPgx4UfPeAtpnVeY3WaUlb1smLjKBVJIJCIEeieA\\\\\\\/+gR2\\\\\\\/JqJtovfYaZqn\\\\\\\/hUJHURBUU3ZooqI11RlciuPxvBcKhJs6M9OK2I22pMKbgYIjwWva6FmfS8oo7\\\\\\\/bdomEhgfr9I6kwnkwtpcCD9tFild9dt0O25PB8fL6YC3zr0kUv8RfvZfBgUM2ccDp\\\\\\\/POIlZKWqoNb2S9x7XztgI4HtYkoBk1BwI113EzYW6o4p6cnNwl1veF99RD09tSqZV0VsUGfMjK78hDIIOR63G+Nt6JR8XHZDTxwMLIIzAISzojZo20gbxwnpX8G15WUudv2hHCIOCclnMoOBjgraEB7Kb2eYkCa9urm+Uu\\\\\\\/VIBvEtQRKXSNd1TOnyL6X6uqz5s1lqfFvd99GcU9hHiiJxPswNkqGw\\\\\\\/tBCuSgTN\\\\\\\/o7M\\\\\\\/mPtPJsY8EFLX5NQ8s279DxoU2nm3Rkn3PhH\\\\\\\/OvLONFVkeLhAeaxELab+ezVNRJcjc9OVIAYrGY7FdB5mjEvSFMmCq3bagOHtmbp7wUdMYY7RFRgztY\\\\\\\/DrRNVcApPSyUwL6AzJaf3tzzncDqKBCFnLg1kXCqf45diTKFlrLHwY7+Yw2q1lGci6tgd+03rQYZGuob0I3CKsXZe+vktNbT+SS04xvgypx45Xapu+kE0AGRmvyA2xqmPRihfW6UEtp\\\\\\\/ScTQ+Isssp7OVGbfLVb7xgeXLoB90M2yVUHqNPnjMTJmgVTWbyHZGtOdrzv5Q\\\\\\\/uWa0wY8gGFXiTryz+g97vJdYN9O\\\\\\\/kgAhbppIZKyYcg+BYy5rmW3Gw3JtdNElPfVtaGcrqpmagglrzn487mqLJDikmGtDoMtmJ9WYGfL6wUuELa\\\\\\\/g7qJWvlYiAPeH4xggaMd4dUImfIUVeodMoIxGJy0AieP47z78FE\\\\\\\/+fAXFvHij0J1SHN+BToLTDtgt\\\\\\\/29QO1Rx12cPdHqV+y7B+uaBzEZ0PO5cQ17yCJURNSwuliUmBSTDb8t6QP58x125EJIhu+CmSIn3JKPsVdgSsc9B4hZpTSJ0GyOTKkbxBKB1Y6t+ib10tf1gBBPghcC+zp91JwNdCko5uaIK2ZyUrdV24tam0NOg3K8U27MyYji8Avo3158ASlByuxQvM48ioEED+PgNCGsCiwNN85mg+AQTYXevqedfJpJd8H1q4kdWyK\\\\\\\/cuNiTDYkFDfdEQVyGFLaFks7V+gM5KtamfihfXq7owfBKquzPJAe9JXUD2O46hRxB\\\\\\\/m\\\\\\\/P0GsLxcHGptsu9FYjQXPp7WhyOudMNuS9zzSUzJ6i8afiwpbiqTtOj5DM+7dvSYAHp6iHgisncQ5BBVshHPybKNNOvGTeOIN0CqMBC\\\\\\\/VBOy8TVe6uWAdHFnh7q6RywGKXTB\\\\\\\/vo5jLe7uKDsFvyUXKXGKO09pzFccvu54bbeW1nlGcgAan03jl0MgQaASU+z7IR0Z\\\\\\\/15Gp84J61hl8QiQcHC3lLCcLHZtqXbgB9LbxZ2qJzm3D7jxGrF+HQxxwq+QkfncHiW8m\\\\\\\/iAV3dYwICt3z+yWtFd69EhT1NXqDv2VbZlprInVDYEw70YKxdWBESUBZPPa27OPty74\\\\\\\/VI1n7K\\\\\\\/4o\\\\\\\/jvgvj9blqfJNdJrj0TxnosamnnqJorUxIBF33EgkfzB7jPv4zztEBSbYZ\\\\\\\/7O43x\\\\\\\/hDWB+ukBPQl8W3h5\\\\\\\/xZ5LCp08LyDdmVc981wMpWuJoC45lCSgnuVANGRXA7uabFRy\\\\\\\/wMH8IK6j6x7n1jYRYbwFxys4NoQvoXj4qHYG95LU0EVT80JWth\\\\\\\/bW7I4udiJGCNwnfrgtGF3ywcBoSI1dH6CkkLHgmhDbX9wI9DBaB1LnWsoGt7U0Lz2Uiwesi8wtiO+JZHuRaJ1bqFFQoJ+T469abFu6i4XmgQbAwUIE6be27BICst+DjGGiJyoUXfy0T3ymwhaMovQsP0xc3Vg5w4fI4en4rSlaQ9odkxpJnCpPLaLSJHDAlte9EyQ\\\\\\\/Ghh4iSVAqTb8c2hqSZFisuulnQ7dVeUYfePONJa4BViKF5Oa5qZV2G+1CiN44ilkJndHVq0FzH2\\\\\\\/VkSbg738BQ8kXQ8afBbOsNklk+637KXLVP9duCkfRM211+qs5fKwsn6DhPnM+2pxrgl5LvzljFGRx76U8TKWSP9Sc8kGHVXSgkAltMHwOcrQAoGvRWE+TPd8rO5fkpRVuKO0Cqobxhz+3CxJET1e1\\\\\\\/mrvaT8R5Sq3R8N3NF7hNPKHsKCVmvZLZU\\\\\\\/oUiwUpNYFKpmXYN7VpJMytDpBvEPTh4MeDui5+pfPDFAsVqYrbGPMwyi9AxB\\\\\\\/F3JDw1zG8olBez9TXRxLzqhXnpNQtgeKdWwQFrhkj1yBJ8wv8ZX+HdirXSxWtBQu8aE7gPRgpg40XCaCGOo9gL5FUznMWRRd6lsoHno6j9foYYwEC+PxpUmj3PH8KTjKjhJcqobKLPnoqLUfUhAKLbnuzmL9a1T6r1IxQjjcVwxVpc+m4zYRGA8Hp1Hf6kXGAzAScUAiewBAbZFfUtzZfWulAT+WPnncl48I3soJKYf\\\\\\\/Ck+6MXFFTyebTPDorzClX27KGCX8TGui59PHQt4DFGn0tggVB+4Mgyt9A\\\\\\\/rO1SLG+QnnXWGM\\\\\\\/Tk6cYQQvpLz6x4OGkpEsw2LZW9B8TDmCdfUH3JWvyVngvGYzcRapSq0SjtvCi1SInsXHZlMEob0YoCYhMhLSZ8j8iPd2UlHJsH0nxkRq9OsuIhREyLMW2eRn8hlxXnlMAZe0bOaObko+5s7qw6aVPobw6ZDf5zNBcGVihWRp9AKbla0H3D2W+Qk78bZoP0jes\\\\\\\/D3tUK78nXw7CjL+Fp80lW\\\\\\\/2gdx0wKec45AH2EA5rKp0UDg0Lk2Wy+qhm0IQOP8+QVXEjdmxKNC042givBsT0Wio90Bmkku1mdv\\\\\\\/DPM3s\\\\\\\/Hzz+v+ywVSQeEHtx9wTXZZ85hYed\\\\\\\/bzb6Et6xUiwDqiCQ3zOGWkKWZyXi+yekb0Lg1kc63y+lfBHha1ujlIRn1KXovP+iDsCFU6LkYce5RtfXN\\\\\\\/hG+oMdYyg0MSj57B6QF11U424S8mU4exfd6n6KoVLSjXHX7+iBYro4ecid19QjuB9tJ+7YgoTCi5trYaCnKg+axEU867C8qYULomY0xmW2XybuDwUHT0cskvn2jJEEDqPvgLlOLIrus8TDzovGHAam5YtuuO36ElEZNmW5nnUIPE4Ms7qwfZ5WDEm8yGgnsrxCaL4kP1tNOyo141eiCh4GK+PMz\\\\\\\/aUMAMgLSK9UWvVxrh7CkxC8jhbqL45qO\\\\\\\/cnPYXJapA43uDB1BAA7oLZ63YJhDf0x4MAG1dU8NmPnXSHNGKapWwYATEwKGiLY6rFbAD1a5Ob6x+Z07BbEyOVcPVp8tAq21WUlCZr7fLfIVvHJwzxrywwwar6JMiyYJrc8fS8ux0\\\\\\\/8Qfq8FJW8CTXvUj6us6qHgk5EpR\\\\\\\/1jpIjNmmSaZKRnDSCSy4u1zbClsPZw3ym6ZrhMj4FA6iQAxRjZtKM5ncT7GU8zUey5yr0xJeQjLNxI1AcQc3eaeEJhcRGv5UNruBkPqw3CCMjenKBdg5P12u4qYBOW\\\\\\\/DXRZKd3A6JEc1CQAjl9xF\\\\\\\/0yqdDXPbzUhfOXSQY9uG6uM3i3uW39woEdVQCI6jo74INsnkYp5GCmay8C2EdXLd5eC3LD4slYNeCPmYDluHox4QOfsi+3DB8SbuLBaDSdNYGma7wZ0BipptpQN2Zvw90cs8QHi9nzi68dHeg54INAjAg1Xi2aU9Xp\\\\\\\/vxBX3LT4vDLa3P8lTjoWOpbCNtSYD3UswIhECEHAyouRsikCl4LYugXWSMcXEnyRhzjgGZNZ6M+UtIBqGrt0wAxr2rMMPPv\\\\\\\/3LfB5RuttJxjuLtbHp5f0+IeHgaEvxtDQswPJ4K5ytijzrJEzgenISs0WKD06Xoy2yI5THJz2SovyCuwqMNMD6AwTGGS1rA6XrZxVymy+r8brqlkW\\\\\\\/KfEgGG7dCZSCwpiCR3VxS95NqmkBriB\\\\\\\/CCfz\\\\\\\/pTR6Y\\\\\\\/GKPyvInOs7fVY4BumEj14bpgMy7hM2p6pybq1Ifd3CgQvYNRthy+EihLKIUQC08jfIeSXMDS7B3axvBcocJatpyd1XLUm4UVFdq7boZlwuDkCduYKTepqNRdWDTdUKvz1aM3qmYLkfWMHUsQA3MNR1jcFIKmH4PgCc6KtXhNcokkObxfpYslxYh0kELDg\\\\\\\/XiTVOZSzieB7PNk4OauNUm1CjVNIHzKEOiNpH0ryWpnwxADUUXv3LKemRLwtPpcAAeAws\\\\\\\/jxQXL6Uy4sbWEt6TmAhngmqsL7NY\\\\\\\/14pKqn+RfQqMFKQLoZwQNOkkBMLN1WKMSFD1ieSLIHJW5DeUsJV16s44mlSPsa92Rq\\\\\\\/g5q4Iczh+VV45sai6EYbgzm9Dt6cdER\\\\\\\/kIfN1PlbXDCqxGSRqmxl\\\\\\\/Fns75YC9X+UK9M21g6Dc1uECrqflhTAwos4TpmveaTATLPwKkhvwuiankIrrCTU8P7hDe+\\\\\\\/qFh2QbsjlbRyEr\\\\\\\/Q9h\\\\\\\/p+ZyKgveKTpY3P57IyJmJd\\\\\\\/WWTB3304Y4ZwEDLHdi8P7h3NrkWZ8ETHw1cu503UCajnFveJShYSnnflzMVIH8EJ5khnMaXZ2pkP5aCAepbGSLQR0E6o5tcnPZwzM0mW\\\\\\\/fPdv35Tpqh++gINuS31lBH7ml+sMFuE7R4nugkOQlw5BdATYQ3P3XFDqB\\\\\\\/q1joywsudYigrnnoGUB6Vtpj\\\\\\\/nXOwIj0cKRM1jtsnYpamBo0g4TAnBiRRKmzcLUhojZWNhhSlZX2VDxbj0CPUo8QuA\\\\\\\/mKqhjyvb0IjFP9g8RfafJ8VG4yNiOjpX3ZW3bCCnWcaYnsLhdbG8ehf0sGTbIS9hGgCGf2lF83ftOAx866fE\\\\\\\/mB+o2k9WNhcfiONB8cOLGCIrqnXXusma1rWaq84Lgh9uiiVqDtR0DGp1YcWygwSB\\\\\\\/3PNqsb4sGVcvrAfdGCHk\\\\\\\/+sJQcFv2Tj\\\\\\\/Wnqo2SXNJ7g8AOX8wudYzSkB8ICEvLlcdWapsSDeIgQCfNmmJFRuExJKlJsm8zwmy9ag\\\\\\\/9nMe34+dmgJOyocDUOAT+2LkU5Co4DQiOJHxb7xd0ywZtQu8Z7EgpqYeQ0aD7PZLuVFsYIfqd1Lev7mskdWbGYjs\\\\\\\/vVn+GMErHWaE\\\\\\\/WqMmruhc95PBAO31nY1BEfYU73PZ8o9wPLDSVVI9ZHuYhF\\\\\\\/qA\\\\\\\/uizxaDsiClhP+i690Wv6W3BFrvN9TAR72yFdIZoSL8wgAsBBd8B54E4Yx5gmlo22fl4w1rzROXTimrYSePgvFJWiamGT0lGApfq4KYF\\\\\\\/n7zf8H2SIqlDH1rsZWzstg9X71nfJsXGVWPm5KMdf6rpbGT3vrlcnAn2Wy2q+RyNKy9fEL965WrjECKnxxDaw37mYmw5TCzqliOMHdGgXpvLH\\\\\\\/4GXx4HeDZ\\\\\\\/qmiSNLUGMfs40CsBkEHuFfL5J\\\\\\\/nzPKRyVKssi0cwEZ23qmDTHVXW11vFm497rhxBF6MHdKGMS8lLlj+N2HxrFjMn5JXFXzGOe\\\\\\\/WaqxL4oAXP3753EF\\\\\\\/Pqkk7OCNh+leG\\\\\\\/Zn4NsGjgC63NfIT1kR0MUZINu8jJUV6KfRi1m2kWIntl4DPMQ7H2vpQBGvvpMjXr8DWKAdlSuZc8f0y58s5LBitK3NbOpbCZfpv2OL+9oUhy1y+NeQmoIvPDRiWQCN4rKD02Q96VjY2IRpjb2myi666h19e1Ec6j4LdOypSwXYAO+9KAZ+O\\\\\\\/rTV0tBxgHBS\\\\\\\/HYfyfvpJ7SY2GYQ0VSK+ryi0SxKo8amFi19fqmEDIx30PWsya\\\\\\\/rBVvk6Ppq5ubbXkXkgDQlrF3fdSYtGW0B7hHFqxZQfXNrEVbyRRWtnJpKdsKjz2Jv9ikU\\\\\\\/OhlEyfB0BFnI0TucG9q6cV1CIcwhPJ4Pev3u9p5v5op095sbuccAMLtZslvWBTj5+cORdzY7w8sMXAb7hZ\\\\\\\/K1WtzsDZvH4wLkiJKDguCByi6pMu\\\\\\\/diWeWsDc\\\\\\\/Auo9Z6pFBLl7rzJMfOxSt19\\\\\\\/mMwnpgQxbPx95atMDaElRQdmWkN9j6Ek+EuZUKh9NwF+2igJKqi4NRamD+rPeuJfU1EYmaK0A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/V7GCeh3C4ii4xXEr9B1MA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d2b03db6cf094e4df60a2b1e94cadc40a3846f5e1065c1665b11d471a421899\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mz3wYm3XPn\\\\\\\/oAcYDqO4SMlxUiqWk8LIrIeA9rzqLSiiAJKIJxmrPtvoQPr0vYE8uC5i3iDmrHPLonbJUVRAFw8vBGBwf0zhKSyRIr7i4dpN7eAv+0pyLnMPWHhDt5lRrBedYYVnQoIk6F37c2zjPZ7qjGat30YQwsWb+xEM77kwMd9NYKcc4gGM5\\\\\\\/22JfcQEMr5H3oGeLqWbbNyemKIvlb8uvcbSwWcFZEjhqBl1nZue9VODgabtCGDyUPs1+RnuBJavpndZn9jpDJS+Vu\\\\\\\/97NII3SwaI+cKTqplYm84UpKR3KeH9WaJwa2dVoKTvieG4m6XOtUJjcdgcgVKLOy96b9Bp1U\\\\\\\/1WEYMsz+HQnLoW6ZrtvPVVQQQwF5GyC\\\\\\\/LrcDyu6\\\\\\\/DPxyCs6SdjWK\\\\\\\/I4oACaJoIbO6K7wexDVPfelNmURd4hViHOdRiLNR5Z6tq1UKkntcpxzSGjDArNr0PquJsxlTa9StndtuzsPLq834lfCFEWo5rtMe5EiwLx3zF1jDR06q3YN3CCefQXtKE0LYYWn\\\\\\\/dvaE4zFaeyNKnENKGXuEiXvUfqMyVqmLBVfnqRwDkH4Yd3KOh\\\\\\\/axofOzKiwBDbo7pIPd2H4fuwITfxKzWZjALqRVUGUJiDbWBztJJQ6u26+Zv1fIxBcy3w9f+mptKqVFxK6bOCiPNGnDoyAZvW11SQBj7chW2NEVpep48\\\\\\\/F4MSGDrrnhgWGgTPIrzUifrhWru1S48g0jLA2BBemewGNMbprQ16frTu4Tx89KA9TPh4Ft2iGngIFQ1n0gJplzCA03x+ed4x2VSyVKjVrd6GgiUQ421TgXvFlA2IZ55qv4ByDp702x+aoswVu6cozm2b1MDJKOGx0eVU7aebJM5QdPMJ5X8LJ89VzmOGCYNl32COZK8aOsahylzKy+mPwD9huRbBlCew9tzN17IiakCjRoUsIlbd07\\\\\\\/5NFMNsTNDGkSklANNe8dCqJuMdEF53omryXxNvRRUq8RFF0r+ii7hSNKPpP7IhOYVkhhcQEb0C4YrJZeThN7bpiryC5DuFfRKE9YprUPEdsUXUVguh9M\\\\\\\/NFvrsZhlJRZjmgPwI5liU4\\\\\\\/mOjOBoo5lpdQqHCRay1ce3J++tTfFknjtgp0QHT7Rg2Sv6cx6Wu8c7lYLFg+F7tnzqMCVVaiuyX1i5reYlUoWBIi\\\\\\\/9jApIJ6+rzwEKdS6nHvMVwIyw61SVaw96G3q9l9YjRV0kL4lRm1y0HoVMwAvklaig+NNMK19m1dnlyFg8JLEjbIEJWvbOqj77t7mJrhW4Nyz8B1LZ1gE4CUxG6z0wRwGsJKWFTGfFLyd0vqWLvT98BhvRK1CNV+sTtNoCLNmJO+MADFeaQx\\\\\\\/TB2Rp9d0obw5hkLfWef2vCGLfwpMzLB41LQ0eWg5Gv10691GUBkxfk\\\\\\\/LMsdRgdLdkMSpGVDA1vt\\\\\\\/psYyi63bv\\\\\\\/fYpR\\\\\\\/C5W6vTuGufhdZGW3U9lbrzPbjf+BPWeiC64Zgq7oNzcr+F2auUFH8Eq3Z8O\\\\\\\/BMw8Q1fSGZclpJh4h8c\\\\\\\/pj67bqueB3S4X3odngGcNxA1ACXpmrD+xZCccXxwFrK35t53Lj1gp2TWk9OGejNdmXVUGSomVe1shH\\\\\\\/OZevrAHC5dU01afQeZ5ChOV0CQYw24a1pKnhLYtNuiX73UVXYG2g+ZLzvaabi87Ma4uoHlpVPto8BBOGSFy4\\\\\\\/BySc1Aq5ahNn4d0MtRW6NB5qO+BVjxLRQSsHvrIYo64rexnuEmbKiTXJtTSUTcPC1e\\\\\\\/QtW2FSgZ3lLuIGUD\\\\\\\/F0QuMQeYjRkSx9zjqSc+nnO3qvh7Tr0Q8kNt8gWsoQhYpFcIIsMM18aTUujg\\\\\\\/69QZst\\\\\\\/vS9VS+nAi5Egck5ufjJXTmzIcuxC1hXfnFcK6YM7\\\\\\\/hxg2EM8Hl4\\\\\\\/5pEtkDI\\\\\\\/K5HozLvqsaFaLFY55p88o06AoIM3LJ2kTCsdQ36\\\\\\\/alA4ai+bL4NY3xlw1xjaJUFrGzQH0UKYaNmt1xgbrZBoD5Ixwtv46zn9tTZg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HCVQ\\\\\\\/UgCnIoKhPKP61psHw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002248db237eb500182e9df8a74cd111d7bc48a5ff0c3891ce718a54119399571e37\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460391 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1673" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pgkJuB6fdmE6MQEqFSXgcjGZt8cRVhvkPHIibaXhc38" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1673" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vg6wLEbgATyqsKWhB24Jg_HquMNNjj6Su3YFbE26RoI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1661" title="The Histone Variant H3.3 Is Enriched at Drosophila Amplicon Origins but Does Not Mark Them for Activation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1687" title="The Drosophila prage Gene, Required for Maternal Transcript Destabilization in Embryos, Encodes a Predicted RNA Exonuclease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460391" id="node460391" data-pisa="ggg;6/6/1673" data-pisa-master="ggg;g3.116.029116" data-apath="/ggg/6/6/1673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genetic Map Between <em>Gossypium hirsutum</em> and the Brazilian Endemic <em>G. mustelinum</em> and Its Application to QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7449-8292" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Baohua Wang</span>, <span class="highwire-citation-author" data-delta="1">Limei Liu</span>, <span class="highwire-citation-author" data-delta="2">Dong Zhang</span>, <span class="highwire-citation-author" data-delta="3">Zhimin Zhuang</span>, <span class="highwire-citation-author" data-delta="4">Hui Guo</span>, <span class="highwire-citation-author" data-delta="5">Xin Qiao</span>, <span class="highwire-citation-author" data-delta="6">Lijuan Wei</span>, <span class="highwire-citation-author" data-delta="7">Junkang Rong</span>, <span class="highwire-citation-author" data-delta="8">O. Lloyd May</span>, <span class="highwire-citation-author" data-delta="9">Andrew H. Paterson</span> and <span class="highwire-citation-author" data-delta="10">Peng W. Chee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1673-1685;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029116 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460391" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Baohua Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baohua%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABaohua%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7449-8292" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Baohua Wang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Limei Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Limei%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALimei%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhimin Zhuang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhimin%2BZhuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhuang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhimin%2BZhuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xin Qiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BQiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qiao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BQiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lijuan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lijuan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALijuan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Junkang Rong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Agriculture and Food Science, Zhejiang A&F University, Lin’an, Hangzhou, Zhejiang 311300, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junkang%2BRong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunkang%2BRong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">O. Lloyd May </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=O.%2BLloyd%2BMay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=May%20OL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AO.%2BLloyd%2BMay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrew H. Paterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BH.%2BPaterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paterson%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BH.%2BPaterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@uga.edu</a>
<a href="mailto:pwchee@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pwchee@uga.edu</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Peng W. Chee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BW.%2BChee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chee%20PW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BW.%2BChee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@uga.edu</a>
<a href="mailto:pwchee@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pwchee@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/6/1673" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460391" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460391/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1673.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460391" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460391/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1673.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460391" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460391/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1673.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460391" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460391/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/6/1673.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460391" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460391/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Among the seven tetraploid cotton species, little is known about transmission genetics and genome organization in <em>Gossypium mustelinum</em>, the species most distant from the source of most cultivated cotton, <em>G. hirsutum</em>. In this research, an F<sub>2</sub> population was developed from an interspecific cross between <em>G. hirsutum</em> and <em>G. mustelinum</em> (HM). A genetic linkage map was constructed mainly using simple sequence repeat (SSRs) and restriction fragment length polymorphism (RFLP) DNA markers. The arrangements of most genetic loci along the HM chromosomes were identical to those of other tetraploid cotton species. However, both major and minor structural rearrangements were also observed, for which we propose a parsimony-based model for structural divergence of tetraploid cottons from common ancestors. Sequences of mapped markers were used for alignment with the 26 scaffolds of the <em>G. hirsutum</em> draft genome, and showed high consistency. Quantitative trait locus (QTL) mapping of fiber elongation in advanced backcross populations derived from the same parents demonstrated the value of the HM map. The HM map will serve as a valuable resource for QTL mapping and introgression of <em>G. mustelinum</em> alleles into <em>G. hirsutum</em>, and help clarify evolutionary relationships between the tetraploid cotton genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/map-comparison" class="hw-term hw-article-keyword hw-article-keyword-map-comparison" rel="nofollow">map comparison</a></li><li class="kwd"><a href="/keyword/chromosome-structural-changes" class="hw-term hw-article-keyword hw-article-keyword-chromosome-structural-changes" rel="nofollow">chromosome structural changes</a></li><li class="kwd"><a href="/keyword/phylogenetic-context" class="hw-term hw-article-keyword hw-article-keyword-phylogenetic-context" rel="nofollow">phylogenetic context</a></li><li class="kwd"><a href="/keyword/colinearity" class="hw-term hw-article-keyword hw-article-keyword-colinearity" rel="nofollow">colinearity</a></li></ul><p id="p-4">The cotton genus <em>Gossypium</em> L. comprises more than 50 species, including eight diploid groups and seven tetraploid species (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fryxell 1992</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wendel and Cronn 2003</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wendel and Grover 2015</a>). Two tetraploid species—<em>G. hirsutum</em> L. (AD1) and <em>G. barbadense</em> L. (AD2)—have been domesticated, whereas five—<em>G. tomentosum</em> Nutall ex Seemann (AD3; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">DeJoode and Wendel 1992</a>), <em>G. mustelinum</em> Miers ex Watt (AD4; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wendel <em>et al.</em> 1994</a>), <em>G. darwinii</em> Watt (AD5; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wendel and Percy 1990</a>), <em>G. ekmanianum</em> Wittmack (AD6; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Krapovickas and Seijo 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Grover <em>et al.</em> 2015b</a>), and <em>Gossypium</em> sp. nov. (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wendel and Grover 2015</a>)—are wild. In the most recent phylogenetic relationships among tetraploid cotton species, <em>G. mustelinum</em> comprises one branch of the earliest split following allopolyploid formation; <em>G. hirsutum</em> was sister to the recently recognized tetraploid <em>G. ekmanianum</em>, and they formed a clade sister to <em>G. tomentosum</em>; <em>G. barbadense</em> was sister to <em>G. darwinii</em>, and these two species formed a clade that was sister to the <em>G. ekmanianum</em>–<em>G. hirsutum</em>–<em>G. tomentosum</em> clade (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Grover <em>et al.</em> 2015b</a>).</p><p id="p-5">The transmission genetics of crosses between the two cultivated tetraploids, <em>G. hirsutum</em> and <em>G. barbadense</em>, have been investigated in detail, and many interspecific <em>G. hirsutum</em> × <em>G. barbadense</em> genetic maps have been developed based on different molecular marker types including restriction fragment length polymorphisms (RFLPs), amplified fragment length polymorphisms (AFLPs), random amplified polymorphic DNA (RAPDs), simple sequence repeats (SSRs), and others (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Reinisch <em>et al.</em> 1994</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lacape <em>et al.</em> 2003</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nguyen <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Song <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Han <em>et al.</em> 2004</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lin <em>et al.</em> 2005</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yu <em>et al.</em> 2007</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">2011</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">He <em>et al.</em> 2007</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blenda <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shi <em>et al.</em> 2015</a>).</p><p id="p-6">Recent cotton genome sequencing, including the A-genome of <em>G. arboreum</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2014</a>), the D-genome of <em>G. raimondii</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">K. Wang <em>et al.</em> 2012</a>), and the tetraploid AD-genome of <em>G. hirsutum</em> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2015</a>), facilitates the construction of high-density maps and comparison between genetic maps and genome sequences. <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> (2013)</a> constructed a cotton map comprising 48,958 loci that were aligned to both a consensus genetic map and a reference genome sequence; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hulse-Kemp <em>et al.</em> (2015)</a> constructed two high-density genetic maps containing 22,829 SNPs for two F<sub>2</sub> mapping populations, and 3533 SNP markers co-occurred in both maps.</p><p id="p-7">It would be a valuable research tool to have maps among the tetraploid cottons crossed in all possible combinations as well as maps of intraspecific crosses for each tetraploid. For <em>G. tomentosum</em>, a wild tetraploid species that is closely related to <em>G. hirsutum</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Grover <em>et al.</em> 2015a</a>), <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Waghmare <em>et al.</em> (2005)</a> described the first <em>G. hirsutum</em> by <em>G. tomentosum</em> (HT) map comprising 589 loci based on RFLP markers. <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> (2011)</a> further exploited QTL alleles for improved fiber quality from <em>G. tomentosum</em> based on advanced-backcross populations derived from the same cross. <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hou <em>et al.</em> (2013)</a> constructed an SSR-based HT genetic map consisting of 1204 loci, with a mean density of 2.76 cM per locus. For <em>G. darwinii</em>, the tetraploid cotton species most closely related to <em>G. barbadense</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wendel and Percy 1990</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Grover <em>et al.</em> 2015a</a>), <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">B. Wang <em>et al.</em> (2012)</a> performed QTL mapping of fiber quality in introgression lines derived from <em>G. hirsutum</em> × <em>G. darwinii</em>. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (2015)</a> constructed an interspecific high-density linkage map using an F<sub>2</sub> population of <em>G. hirsutum</em> × <em>G. darwinii</em>, which consists of 2763 SSR markers with an average interlocus distance of 1.5 cM.</p><p id="p-8">Little is known about transmission genetics and genome organization in <em>G. mustelinum</em>, which is isolated as one branch of the earliest split following allopolyploid formation, and is genetically farthest from <em>G. hirsutum</em> (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Grover <em>et al.</em> 2015a</a>). To reveal the basic transmission genetics in crosses between <em>G. mustelinum</em> and cultivated cotton, and build information useful to extract agriculturally valuable alleles from <em>G. mustelinum</em>, a primary <em>G. hirsutum</em> × <em>G. mustelinum</em> (HM) genetic map was constructed and compared with those involving <em>G. hirsutum</em> crossed with <em>G. barbadense</em>, <em>G. tomentosum</em>, and <em>G. darwinii</em>, respectively. Colinearity between our HM map and the tetraploid cotton genome was also investigated. To provide an important demonstration of the usefulness of the map, QTL mapping of fiber elongation that differentiates the parental lines was performed in advanced backcross populations derived from the same parents.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-9">An interspecific F<sub>2</sub> population comprising 92 plants was developed from a cross between <em>G. hirsutum</em> (PD94042) and <em>G. mustelinum</em> (AD4-8). PD94042 is a public cotton (<em>G. hirsutum</em>) germplasm line that combines high yield potential and improved fiber maturity developed at the Pee Dee Research and Education Center situated in Florence and Darlington counties, SC (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">May 1999</a>). Plants of the F<sub>2</sub> population were grown in a greenhouse in Tifton, GA.</p><p id="p-10">Advanced backcross populations were developed by first crossing <em>G. hirsutum</em> acc. PD94042 and <em>G. mustelinum</em> (AD4-8), then independently backcrossing F<sub>1</sub> plants to the <em>G. hirsutum</em> parent for three cycles. BC<sub>3</sub>F<sub>1</sub> plants were selfed to generate BC<sub>3</sub>F<sub>2</sub> families (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2016</a>), and 12 BC<sub>3</sub>F<sub>2:3</sub> and BC<sub>3</sub>F<sub>2:4</sub> families of 130–160 lines per family (totally 1826 lines with average population size of 152) were planted in 2008 and 2009 in Tifton, GA. Fiber elongation data were collected from two random replicate plots and genetically mapped. All cultural practices followed standard recommendations for Georgia cotton production as described in <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> (2016)</a>. Fiber elongation was tested by using a High-Volume Precision Instrument (HVI; Zellweger-Uster, Knoxville, TN) in the Cotton Incorporated Textile Services Laboratory (Cary, NC).</p></div><div id="sec-3" class="subsection"><h3>Molecular and morphological markers</h3><p id="p-11">The majority of molecular markers utilized here were SSR markers. Most of them were selected from an interspecific <em>G. hirsutum</em> by <em>G. barbadense</em> map (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>), and the marker sequences were downloaded from the Cotton Marker Database (CMD, <a href="http://www.cottonmarker.org">http://www.cottonmarker.org</a>). RFLP and sequence-tagged sites (STS), which were largely sampled from published HB (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>) and HT maps (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Waghmare <em>et al.</em> 2005</a>), were also used in this experiment. In addition, root-related genes in <em>Arabidopsis</em> were identified from The <em>Arabidopsis</em> Information Resource (TAIR, <a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>), and their homologous cotton expressed sequence tags (ESTs) were obtained through “basic local alignment search tool” (BLAST) searches of an online cotton EST database (<a href="http://www.agcol.arizona.edu/cgi-bin/pave/Cotton/index.cgi">http://www.agcol.arizona.edu/cgi-bin/pave/Cotton/index.cgi</a>). Eighteen pairs of EST-SSR primers were designed as described by <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Han <em>et al.</em> (2006)</a>, and designated here using UGT (the University of Georgia, Tifton Campus) as a prefix (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Enzyme digestion was utilized on the PCR products of UGT primer without polymorphism, and used as cleaved amplified length polymorphisms (CAPs) described by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Konieczny and Ausubel (1993)</a>. Three CAPs were analyzed, namely CAPs0005, amplification products of UGT0005 digested by <em>Hin</em>fΙ; CAPs0010, amplification products of UGT0010 digested by <em>Hae</em>III; and CAPs0011, amplification products of UGT0011 digested by <em>Hha</em>Ι. Three morphological markers, namely anther color (yellow or cream, <em>P1</em> gene), petal color (yellow or cream, <em>Y1</em> gene), and petal spot (presence or absence, <em>R2</em> gene; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kohel and Richmond 1971</a>) were also investigated.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468326/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468326/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Targeted <em>Arabidopsis</em> root-related gene homologs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>DNA extraction, RFLP protocol, PCR amplification, and electrophoresis</h3><p id="p-12">DNA extractions followed the protocols established by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paterson <em>et al.</em> (1993)</a>. RFLP analysis was performed as described by <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Reinisch <em>et al.</em> (1994)</a>. SSR and STS-PCR amplifications were performed using a Peltier Thermal Cycler-225 (MJ Research) as described by <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zhang <em>et al.</em> (2002)</a> with modifications: predenaturation at 94° for 3 min; 30 cycles of 40 sec denaturation at 94°, 45 sec annealing at 57°, and 1 min extension at 72°; 7 min extension at 72°; and finally a 10° hold. Enzyme digestion of CAPs was performed as described by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chee <em>et al.</em> (2004)</a> after the UGT primers were amplified. Polyacrylamide gel electrophoresis of SSR, STS, and CAPs was performed as described by <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Zhang <em>et al.</em> (2002)</a>.</p></div><div id="sec-5" class="subsection"><h3>Construction of a genetic linkage map</h3><p id="p-13">A genetic linkage map was constructed using MAPMAKER/Exp Version 3.0 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lander <em>et al.</em> 1987</a>) software, in which the Kosambi centiMorgan function and threshold LOD = 5.0 were used. Linkages at distances of greater than 35 Kosambi cM were considered to be nonsignificant. Assignment of linkage groups to subgenomes and chromosomes was made based on the information from framework markers on the published maps (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Reinisch <em>et al.</em> 1994</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nguyen <em>et al.</em> 2004</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>).</p></div><div id="sec-6" class="subsection"><h3>Sequence homology between markers and G. hirsutum genome</h3><p id="p-14">The reference genome of the tetraploid species <em>G. hirsutum</em> (<a href="http://mascotton.njau.edu.cn">http://mascotton.njau.edu.cn</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Zhang <em>et al.</em> 2015</a>) was used in this study. Markers mapped on the HM map with available sequences were aligned to the scaffold sequences using the BLASTN algorithm with an e-value cutoff of 1e-5 and an identity percentage cutoff of 90%. All hits separated by distances of 5 kb or less were assembled into single loci, and we retained up to the top 10 matching loci with 95% length coverage of the original marker sequence for each marker.</p></div><div id="sec-7" class="subsection"><h3>Application of the HM map in QTL mapping of fiber elongation</h3><p id="p-15">The genome structure of the BC<sub>3</sub>F<sub>1</sub> individuals was evaluated based on genotyping the DNA of BC<sub>3</sub>F<sub>1</sub> plants with 218 SSR markers, which were approximately evenly distributed on the HM map. The markers with introgression from <em>G. mustelinum</em> in the BC<sub>3</sub>F<sub>1</sub> were then used to screen the entire BC<sub>3</sub>F<sub>2</sub> family (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2016</a>).</p><p id="p-16">The mixed model-based composite interval mapping (MCIM) of QTL Network V2.1 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yang <em>et al.</em> 2008</a>) was used to analyze main-effect additive QTL with the critical F value of MCIM being calculated based on 1000 permutation tests. QTL effects were estimated using the Monte Carlo Markov Chain method with 20,000 Gibbs sampler iterations, and candidate interval selection. Gibbs sample size was set to the default value. Window size and walk speed were set at 10 cM and 1 cM, respectively. The threshold for significance was set at <em>P</em> = 0.001 to claim a putative QTL. Since QTL for fiber elongation were mapped in 21 BC<sub>3</sub>F<sub>2</sub> populations in our previous report (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2016</a>), here QTL mapping was performed in two additional generations/environments, namely BC<sub>3</sub>F<sub>2:3</sub> and BC<sub>3</sub>F<sub>2:4</sub>, and also in joint analysis of the three generations/environments (BC<sub>3</sub>F<sub>2</sub>, BC<sub>3</sub>F<sub>2:3</sub>, and BC<sub>3</sub>F<sub>2:4</sub>) considering the environmental effects.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-17">The sequences of microsatellite markers for this project are available at both CottonGen (<a href="https://www.cottongen.org/">https://www.cottongen.org/</a>) and Cotton Marker Database (CMD, <a href="http://www.cottonmarker.org">http://www.cottonmarker.org</a>); the sequences of RFLP probes for this project are available at CottonGen (<a href="https://www.cottongen.org/">https://www.cottongen.org/</a>). Detailed comparison between HM map with HT, HB, and HD maps are available in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>. The raw marker data used to construct the HM map are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FileS1.xls">File S1</a>, and the genotype data and phenotype data used to map QTL of fiber elongation are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FileS2.xlsx">File S2</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>General information about the genetic map</h3><p id="p-18">A total of 690 polymorphic SSR primers, 201 cDNA and genomic DNA probes, 29 STS primers, three CAPs and three morphological markers were analyzed in the F<sub>2</sub> progeny (the raw marker data are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FileS1.xls">File S1</a>). Among the 926 markers mapped, a total of 163 detected multiple loci, namely 129 detected two, 25 detected three, seven detected four, and two detected five loci. In total, 1134 loci were generated, of which 988 (87.1%) did not deviate significantly from—and 146 (12.9%) deviated from—Mendelian 1:2:1 (for codominant) and 3:1 (for dominant) inheritance ratios. A total of 674 loci were codominant, whereas 228 were dominant for the <em>G. hirsutum</em> allele, and 232 were dominant for the <em>G. mustelinum</em> allele. The HM map constructed here comprises 1055 loci on 26 chromosomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), with total recombinational length of 5595 cM, within the 3500–5700 cM range of <em>G. hirsutum</em> by <em>G. barbadense</em> (HB) maps (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lacape <em>et al.</em> 2009</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gossypium hirsutum&#xA0;&#xD7;&#xA0;G. mustelinum (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, i.e., Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Gossypium hirsutum&lt;/em&gt;&#xA0;&#xD7;&#xA0;&lt;em&gt;G. mustelinum&lt;/em&gt; (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, &lt;em&gt;i.e.&lt;/em&gt;, Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in &lt;em&gt;Materials and Methods&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-1.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-1.medium.gif" width="440" height="264"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gossypium hirsutum&#xA0;&#xD7;&#xA0;G. mustelinum (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, i.e., Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Gossypium hirsutum&lt;/em&gt;&#xA0;&#xD7;&#xA0;&lt;em&gt;G. mustelinum&lt;/em&gt; (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, &lt;em&gt;i.e.&lt;/em&gt;, Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in &lt;em&gt;Materials and Methods&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-2.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-2.medium.gif" width="440" height="260"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gossypium hirsutum&#xA0;&#xD7;&#xA0;G. mustelinum (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, i.e., Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Gossypium hirsutum&lt;/em&gt;&#xA0;&#xD7;&#xA0;&lt;em&gt;G. mustelinum&lt;/em&gt; (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, &lt;em&gt;i.e.&lt;/em&gt;, Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in &lt;em&gt;Materials and Methods&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-3.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-3.medium.gif" width="440" height="275"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gossypium hirsutum&#xA0;&#xD7;&#xA0;G. mustelinum (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, i.e., Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Gossypium hirsutum&lt;/em&gt;&#xA0;&#xD7;&#xA0;&lt;em&gt;G. mustelinum&lt;/em&gt; (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1&#x2013;Chr.13) and D (Chr.14&#x2013;Chr.26) chromosomes, &lt;em&gt;i.e.&lt;/em&gt;, Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in &lt;em&gt;Materials and Methods&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-4.medium.gif" width="251" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-4.medium.gif" width="251" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F1/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child"><em>Gossypium hirsutum</em> × <em>G. mustelinum</em> (HM) genetic map. The 26 chromosomes are arranged according to the 13 homeologous pairs of A (Chr.1–Chr.13) and D (Chr.14–Chr.26) chromosomes, <em>i.e.</em>, Chr.1 (At genome) is followed by its homeologous Chr.15 (Dt genome), etc. Underlined loci are duplicated loci. Map distances are given in centiMorgans (cM). Locus names are as described in <em>Materials and Methods</em>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468307/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468307/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">General information of chromosomes in the genetic map of <em>G. hirsutum</em> and <em>G. mustelinum</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Characteristics of the HM map</h3><p id="p-20">Among the 1055 loci mapped on chromosomes, about 6.9% more markers were detected in the D than the A subgenome (545 <em>vs.</em> 510), and the recombinational length was about 4.6% larger in the D than the A subgenome (2860.7 <em>vs.</em> 2734.3 cM). The average recombination distance between consecutive loci was 5.3 cM, whereas the density of markers along chromosomes ranged from 3.2 cM (Chr.2) to 8.5 cM (Chr.13). The largest gap between two adjacent loci was 33.5 cM (Chr.24). The overall average recombination distance in the two subgenomes was similar (5.4 cM in At <em>vs.</em> 5.2 cM in Dt). Significant variation in chromosome length was observed, ranging from 90.4 cM (Chr.2) to 304.6 cM (Chr.9). Seven At chromosomes were longer than homeologous Dt chromosomes, whereas six At chromosomes were shorter than Dt chromosomes (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-21">Tetraploid cotton containing At and Dt subgenomes was derived from a naturally occurring cross between two diploids with A and D genomes about 1–2 million yr ago (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Wendel and Cronn 2003</a>). The distributions of duplicate loci (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) were generally consistent with the homeologous relationships among chromosomes that are well established in the HB, HT, and HD maps (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Waghmare <em>et al.</em> 2005</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2015</a>). Many loci were also duplicated on chromosomes that were nonhomeologous (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), perhaps reflecting single-gene duplication, or earlier genome duplication events (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2012</a>).</p><p id="p-22">Nonrandom patterns of DNA marker distribution provided clues regarding important features of cotton genome organization. Unlike the duplicate loci on homeologous chromosomes, nonhomeologous duplicate loci were scattered over many chromosomes. For example, 13 markers on Chr.5 detected duplicate loci, with four markers having duplicate loci on homeologous Chr.19, three on Chr.5 itself, and the other six scattered on nine other chromosomes. The intrasubgenomic duplications have been proposed as supporting the ancient chromosomal duplication hypothesis (paleo-polyploidization) predating divergence of modern <em>Gossypium</em> diploid genomes, that has now been shown to have been a 5–6 × multiplication of ancestral chromosomes (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2012</a>).</p></div><div id="sec-12" class="subsection"><h3>Chromosome structural differences between HM, HB, HT, and HD maps</h3><p id="p-23">Based on alignment of common DNA markers, our HM map was compared to HT (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Waghmare <em>et al.</em> 2005</a>), HB (HBr, <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>; HBg, <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>), and HD (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2015</a>) maps, respectively. The inversions found between HM and HT, HM and HB, HM and HD maps are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>. A summary of the detailed comparisons of HM with maps of different species follows.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inversions found in HT (A), HB (B), and HD (C) maps compared to the HM map. D indicates when each inversion occurred during tetraploid cotton diversification based on the phylogeny proposed by Grover et al. (2015a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inversions found in HT (A), HB (B), and HD (C) maps compared to the HM map. D indicates when each inversion occurred during tetraploid cotton diversification based on the phylogeny proposed by Grover &lt;em&gt;et al.&lt;/em&gt; (2015a).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-5.medium.gif" width="319" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-5.medium.gif" width="319" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468490" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inversions found in HT (A), HB (B), and HD (C) maps compared to the HM map. D indicates when each inversion occurred during tetraploid cotton diversification based on the phylogeny proposed by Grover et al. (2015a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inversions found in HT (A), HB (B), and HD (C) maps compared to the HM map. D indicates when each inversion occurred during tetraploid cotton diversification based on the phylogeny proposed by Grover &lt;em&gt;et al.&lt;/em&gt; (2015a).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-6.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-6.medium.gif" width="440" height="140"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F2/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468490" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Inversions found in HT (A), HB (B), and HD (C) maps compared to the HM map. D indicates when each inversion occurred during tetraploid cotton diversification based on the phylogeny proposed by <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Grover <em>et al.</em> (2015a)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">A possible inversion was found on Chr.15 when comparing HM with HT, HB, and HD (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>). In HM and HT, the inversion was detected based on seven anchor markers, namely pAR957, A1738, A1553, pGH468, A1340, A1593, and pAR906, with the affected regions spanning 80.8 cM in HM and 75.6 cM in HT; two of these anchor markers, pAR957 and A1738 cosegregated in HM, but were 13.8 cM apart in HT, consistent with a possible inversion. In HM and HB, the inversion was found based on six anchor markers pAR957, A1738, A1553, pGH468, A1340, and BNL3090, with the affected regions spanning 58.4 cM in HM and 101.8 cM in HBr; the two anchor markers pAR957 and A1738 cosegregated in HM, but they were 8.6 cM apart in HBr, consistent with a possible inversion. In HM and HD, the inversion was divided into two sections, the first based on four anchor markers BNL1667a, JESPR243, NAU4045, and BNL786, with the affected regions spanning 89.1 cM in HM and 24.9 cM in HD, and the second based on another four anchor markers, namely BNL2440, MUCS422, MUCS410, and MUSS563 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-26">On Chr.3, two anchor markers G1164 and pXP3_28 cosegregated in HT, but were 23.8 cM apart in HM, consistent with a possible inversion; a second possible inversion was found based on five anchor markers, namely pGH279, P9_53, pGH619, A1171, and A1834, with the affected regions spanning 55.1 cM in HM <em>vs.</em> 44.4 cM in HT. Based on three anchor markers (pAR764, pXP2_60, and G1164), an inversion distinguished HM Chr.3 from HBr; the affected regions spanned 25.5 cM in HM and 19.4 cM in HBr. HM Chr.3 was differentiated from HD by an inversion based on five anchor markers (NAU1070, NAU5027, BNL3989, BNL226b, and NAU1068c), with the affected regions spanning 20.5 cM in HM compared to 14.8 cM in HD (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-27">Based on two anchor markers A1583 and A1536, an inversion appears to distinguish HM Chr.17 from HT; the affected regions spanned 36.5 cM in HM compared to only 8.5 cM in HT. An overlapped inversion was found on Chr.17 between HM and HBr based on three anchor markers BNL3408, A1536, and pAR172; the affected regions spanned 64.7 cM in HM whereas only 3.6 cM in HBr. Based on five anchor markers BNL3408, DPL0529, DPL0279, BNL4003b, and NAU1028, an inversion distinguished HM Chr.17 from HD, with the affected regions spanning 96.9 cM in HM compared to 31.2 cM in HD (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-28">An inversion was indicated on Chr.5 based on eight anchor markers between HM and HT, namely A1159, A1459, A1535, pAR482, pAR597, pAR825, pAR112, and pAR2_42; the affected regions spanned 217.2 cM in HM <em>vs.</em> only 109.7 cM in HT. Two inversions were detected on Chr.5 between HM and HD. The first inversion was based on two anchor markers, DPL0368b and DPL0177; the second inversion was based on two anchor markers BNL3241 and JESPR42, with the affected regions spanning 18.9 cM in HM comparing to only 14.4 cM in HD (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-29">Based on two anchor markers, G1276 and pAR953, an inversion distinguished HM Chr.8 from HT, and the affected regions spanned 57.3 cM in HM and 33.7 cM in HT. An inversion was also found on Chr.8 between HM and HBr based on two anchor markers A1412 and A1108, with the affected regions spanning 6.7 cM in HM comparing to 3.6 cM in HBr (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-30">Three overlapped inversions were found when comparing HM with both HBr and HBg (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>). On Chr.9, <em>G. mustelinum</em> was differentiated from <em>G. barbadense</em> by an inversion based on four anchor markers BNL2590, BNL1030, A1270b, and G1267b; the affected regions spanned 110.1 cM in HM comparing to 54.5 cM in HBr. When compared to HBg, this inversion was also found on Chr.9 based on three anchor markers BNL2590, BNL1414, and BNL1030; the affected regions spanned 25.1 cM in HM compared to 9.9 cM in HBg. On Chr.23, based on four anchor markers A1270, pGH783b, BNL3031, and pAR474, an inversion distinguished HM Chr.23 from HBr, with affected regions spanning 110.7 cM in HM, whereas only 29.2 cM in HBr; when compared to HBg, an overlapped inversion was detected based on three anchor markers JESPR110, BNL1414, and NAU2954, with the affected regions spanning 96.2 cM in HM, whereas only 8.7 cM in HBg. The third overlapped inversion was found on Chr.13. Based on three anchor markers pAR958, BNL2652, and pAR274, an inversion distinguished HM Chr.13 from HBr, with the affected regions spanning 91.9 cM in HM <em>vs.</em> 27.1 cM in HBr; an overlapped inversion was found on Chr.13 of HBg based on three anchor markers BNL1438, BNL1421, and BNL2652; the affected regions spanned 70.5 cM in HM compared to 9.7 cM in HBg. This third inversion was also found between HM and HD on Chr.13 based on four anchor markers, namely BNL4029, BNL1438, BNL1421, and BNL2652a, with the affected regions spanning 88 cM in HM but only 22.5 cM in HD.</p><p id="p-31">A terminal inversion was found on Chr.12 based on six anchor markers NAU4047, NAU1274, BNL3261, NAU3812, BNL1673, and NAU943, with the affected regions spanning 132.4 cM in HM and 44.4 cM in HBg. An overlapped terminal inversion was found on Chr.12 between HM and HD based on two anchor markers BNL3261 and BNL4041, with the affected regions spanning only 0.1 cM in HM but 2.4 cM in HD (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-32">Based on five anchor markers BNL1079, BNL1721, pGH576, pAR046, and BNL2652, HM Chr.18 differed from HBr Chr.18 by an inversion; the affected regions spanned 12.5 cM in HM and 42.1 cM in HBr. An overlapped inversion was found on Chr.18 between HM and HD based on two anchor markers BNL1079 and BNL1721, with the affected regions spanning only 0.1 cM in HM but 10.6 cM in HD; another inversion was found on Chr.18 between HM and HD based on two anchor markers, JESPR246 and MUSB1135, with the affected regions spanning 36.8 cM in HM but only 5.8 cM in HD (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-33">Some cases of inversions between HM and only one other mapped tetraploid species (HT, HB, and HD) were also detected (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>). Two inversions were detected on Chr.2 between HM and HD; the first inversion was based on four anchor markers, namely NAU3427a, BNL3971, AU3875, and MUSS599, with the affected regions spanning 1.9 cM in HM and 16 cM in HD; the second inversion was based on two anchor markers, BNL1897 and BNL3512a, with similar distance between these two loci in both maps. Based on three anchor markers BNL2572, G1045, and pAR903, an inversion was found on Chr.4 between HM and HBr, with the affected regions spanning 27.3 cM in HM and 29 cM in HBr. An inversion was found based on two anchor markers A1378 and pAR610 on Chr.19 between HM and HT; the affected regions spanned 41.4 cM in HM but 19.9 cM in HT. Based on three anchor markers NAU2679c, NAU2565 and DPL0519, an inversion appears to distinguish HM Chr.25 from HD, with the affected regions spanning 1.6 cM in HM <em>vs.</em> 5 cM in HD. An inversion was found on Chr.7 between HM and HBr based on two anchor markers pAR040 and G1045; the affected regions spanned 41.3 cM in HM, whereas only 0.5 cM in HBr. A terminal inversion was found on Chr.20 between HM and HBg based on three anchor markers CIR094, NAU1066, and NAU3574 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>), with the affected regions spanning 91.7 cM in HM and 20.6 cM in HBg. Based on nine anchor markers, namely NAU3862, NAU1039, BNL3867, NAU3920, BNL840, BNL3510, DPL0183b, MUSB0846b, and BNL1227b, a terminal inversion distinguished HM Chr.26 from HD, with the affected regions spanning 54.3 cM in HM and 42.1 cM in HD.</p></div><div id="sec-13" class="subsection"><h3>Homology of the HM genetic map with the tetraploid cotton genome</h3><p id="p-34">The result of colinearity analysis between the linkage map and genome sequences of <em>G. hirsutum</em> (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Zhang <em>et al.</em> 2015</a>) is shown with dotplots in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Most marker positions correspond closely between the two maps, and translocations involving Chr. 2 with Chr.3 and Chr.4 with Chr.5 were identified (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), whereas some discrepancies existed.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dotplots of the syntenic positions of molecular markers in the HM genetic linkage map vs. the G. hirsutum reference genome. The G. hirsutum reference genome chromosomes are shown on the y-axis and the HM genetic linkage map chromosomes are shown on the x-axis. Red arrows indicate translocation events relative to G. hirsutum reference genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dotplots of the syntenic positions of molecular markers in the HM genetic linkage map &lt;em&gt;vs.&lt;/em&gt; the &lt;em&gt;G. hirsutum&lt;/em&gt; reference genome. The &lt;em&gt;G. hirsutum&lt;/em&gt; reference genome chromosomes are shown on the &lt;em&gt;y&lt;/em&gt;-axis and the HM genetic linkage map chromosomes are shown on the &lt;em&gt;x&lt;/em&gt;-axis. Red arrows indicate translocation events relative to &lt;em&gt;G. hirsutum&lt;/em&gt; reference genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F3.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/6/1673/F3.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468388" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Dotplots of the syntenic positions of molecular markers in the HM genetic linkage map <em>vs.</em> the <em>G. hirsutum</em> reference genome. The <em>G. hirsutum</em> reference genome chromosomes are shown on the <em>y</em>-axis and the HM genetic linkage map chromosomes are shown on the <em>x</em>-axis. Red arrows indicate translocation events relative to <em>G. hirsutum</em> reference genome.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Assignment of morphological markers and root-related cotton ESTs to chromosomes</h3><p id="p-36">Three morphological traits were investigated in the F<sub>2</sub> population of <em>G. hirsutum</em> × <em>G. mustelinum</em>, namely anther color (<em>P1</em>), petal color (<em>Y1</em>), and petal spot (<em>R2</em>). The chi-squared values for yellow <em>vs.</em> cream anther; yellow <em>vs.</em> cream petal; and presence <em>vs.</em> absence of petal spot were 0.90, 0.44, and 0.73, respectively; each does not deviate significantly from the Mendelian segregation ratio for a single gene (3:1) (χ<sub>c</sub><sup>2</sup> <em>&lt;</em> χ<sub>0.05,1</sub><sup>2</sup> = 3.84). The three traits were used as morphological markers to construct the genetic map. Anther color was associated with the interval between A1459 and A1535b on Chr.5; petal color was associated with the interval between BNL2652a and PAR274 on Chr.13; petal spot was associated with the interval between A1625 and pAR040 on Chr.7 in our research (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-37">UGT primers were synthesized according to cotton ESTs homologous to root-related <em>Arabidopsis</em> genes, of which UGT0009 was mapped on Chr.5. Three CAPs were also mapped, namely CAPs0005 on Chr.17, CAPs0010 on Chr.13, and CAPs0011 on Chr.24. According to TAIR, the <em>Arabidopsis</em> genes homologous to these cotton ESTs are involved in root development; the inclusion of these genes may help detect cotton root QTL and build on the connections between cotton and <em>Arabidopsis</em> genes.</p></div><div id="sec-15" class="subsection"><h3>Phenotypic performance and QTL mapping of fiber elongation in advanced-backcross populations</h3><p id="p-38">The distribution of fiber elongation in the BC<sub>3</sub>F<sub>2:3</sub> and BC<sub>3</sub>F<sub>2:4</sub> generations is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The 12 families as a whole were normally distributed for fiber elongation in both generations. Although <em>G. mustelinum</em> does not produce spinnable fiber, many BC<sub>3</sub>F<sub>2:3</sub> and BC<sub>3</sub>F<sub>2:4</sub> lines have higher fiber elongation than the <em>G. hirsutum</em> parent.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution of fiber elongation in the BC3F2:3/BC3F2:4 progeny lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1386533814" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency distribution of fiber elongation in the BC&lt;sub&gt;3&lt;/sub&gt;F&lt;sub&gt;2:3&lt;/sub&gt;/BC&lt;sub&gt;3&lt;/sub&gt;F&lt;sub&gt;2:4&lt;/sub&gt; progeny lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1673/F4.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/6/1673/F4.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1673/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468336" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Frequency distribution of fiber elongation in the BC<sub>3</sub>F<sub>2:3</sub>/BC<sub>3</sub>F<sub>2:4</sub> progeny lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">By assuming that QTL with common flanking marker(s) and additive effects from the same direction represent single QTL, a total of 14 fiber elongation QTL (<em>P</em> &lt; 0.001) were detected (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), and six of them were also detected in BC<sub>3</sub>F<sub>2</sub> generations (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2016</a>). These QTL were mapped to 12 chromosomes; eight on six chromosomes from the A-subgenome, and six on six D-subgenome chromosomes. QTL were detected in 10 of the 12 families, with one to six per family, and a maximum of six QTL detected in family POP16. The phenotypic variation explained per QTL ranged from 1.24 to 19.41%, with an average of 10.14%. Eight QTL could be detected at least twice in different families or across different environments; notably, the QTL <em>qELO-19-1</em> could be detected in three environments and also the joint analysis. Ten of the 14 QTL had positive additive effects, where alleles from <em>G. hirsutum</em> increased fiber elongation, which was consistent with the parental phenotypes. The remaining four of the 14 QTL had negative additive effects, where alleles from <em>G. mustelinum</em> increased fiber elongation (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468419/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468419/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimates of additive effects of quantitative trait loci for fiber elongation across different environments by mixed linear model</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Chromosome structural changes</h3><p id="p-45">Two post-polyploidization reciprocal translocations of Chr.4/Chr.5 and Chr.2/Chr.3 were further confirmed by many homologous loci. At-genome chromosomes 4 and 5 have homeologous relationships with two Dt-genome chromosomes (Chr.22 and Chr.19). Nonoverlapping sets of loci on Chr.4 and Chr.5 have counterparts on different regions of Chr.19 and Chr.22 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), consistent with the finding (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brubaker <em>et al.</em> 1999</a>) that Chr.4 and Chr.5 have undergone a reciprocal translocation, which is also consistent with results from <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> (2004)</a>. At-genome chromosomes 2 and 3 have homeologous relationships with Dt-genome chromosomes 14 and 17. Nonoverlapping sets of loci on Chr.2 and Chr.3 have counterparts on different regions of Chr.14 and Chr.17 (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>), consistent with the results from <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> (2007)</a>.</p><p id="p-46">Across most pairs of homeologous chromosomes, the linear order of loci was substantially conserved (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>), although some inversions existed. Most locus order differences were due to reversals of neighboring markers explicable by inversions. Many apparent inversions involved only two neighboring loci, and, in our comparatively small mapping population (92 plants), may be due to occasional scoring errors or missing data. However, some putatively orthologous loci mapped to significant different locations and cannot be easily explained by inversion. For example, A1270 mapped to locations of 212.2 cM on Chr.9 and 35 cM on Chr.23, whereas the next common marker NAU3888 was mapped to location of 7.7 cM on Chr.9 but cosegregated with A1270 on Chr.23. It may be ancient duplication together with structural changes, or proximal duplication accompanied by failure to find the true ortholog due to either lack of polymorphism or its deletion, that lead to such differences (<a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>).</p></div><div id="sec-18" class="subsection"><h3>Mapping of morphological markers</h3><p id="p-47">Cotton morphological mutants have been used widely in genetic mapping, and have proven useful in efforts of agronomic improvement in some cases (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kohel and Bird 2002</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahuja and Dhayal 2007</a>).</p><p id="p-48">Early reports (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Harland 1929</a>) proposed that cotton anther color was conditioned by one pair of alleles, <em>P</em> and <em>p</em>, with yellow dominant to cream. <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stephens (1954)</a> proposed that the anther color locus in amphidiploid cotton was located in the A genome by means of a tri-species hybrid. <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Turcotte and Feaster (1966)</a> pointed out that two pairs of genes controlled anther color, and amphidiploid cottons that breed true for yellow pollen would have the genotype <em>P1P1P2P2</em>, while true-breeding cream-pollen strains would be <em>p1p1P2P2</em> or <em>p1p1p2p2</em>, and an orange-pollen mutant would have the genotype <em>P1P1p2p2</em>. <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rhyne and Carter (1992)</a> found a third locus, <em>P3</em>, and a true orange phenotype was conditioned by the genotype <em>P1P1p2p2p3p3</em>, but by then only the <em>P1</em> locus was associated with linkage groups (Chr.5; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Percy and Kohel 1999</a>). <em>Gossypium mustelinum</em> has yellow anthers and <em>G. hirsutum</em> has cream anthers. The segregation ratio for yellow: cream anther does not deviate significantly from the Mendelian ratio for a single gene (3:1) in the F<sub>2</sub> population. The gene was further mapped on Chr.5, as reported in some previous research (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Endrizzi <em>et al.</em> 1985</a>; <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Yu <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2012</a>); in addition, the anther color gene shared a common linked marker (A1535b) with <a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> (2004)</a>. The yellow color of most plant pollen is due to the presence of flavonoid and carotenoid pigments (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stanley and Linskens 1974</a>). Further fine-mapping work could be performed to locate the gene controlling cotton anther color with the help of genome sequence data released recently.</p><p id="p-49">In allotetraploid <em>Gossypium</em> species, yellow petal is controlled by duplicate dominant genes <em>Y1</em> (At subgenome) and <em>Y2</em> (Dt subgenome). <em>Y2</em> had been shown to be on Chr.18 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Endrizzi and Ray 1991</a>), but <em>Y</em>1 had not been mapped to chromosome (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Endrizzi <em>et al.</em> 1985</a>). <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Guo <em>et al.</em> (2006)</a> proposed that the <em>Y</em>1 gene might be anchored to one of several possible chromosomes, including the short arm of Chr.5, the long arms of Chr.11, Chr.8, Chr.13, or possibly Chr.10. <a id="xref-ref-42-10" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> (2004)</a> assigned petal color to LGA01 (Chr.13). In our research, petal color was also mapped on Chr.13, associated with the interval between BNL2652a and PAR274.</p><p id="p-50">The five petals of <em>G. hirsutum</em> have an area of anthocyanin pigmentation at the base, called a petal spot, which is conditioned by a gene called <em>R2</em> (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Percy and Kohel 1999</a>). Cultivated Upland cotton lacks a petal spot, whereas such spots are not uncommon in primitive cottons or race stocks (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fryxell 1984</a>). Spotless petal was once used as the “hall-mark” of a commercially and agriculturally valuable stock of Pima cotton. For strains possessing this character, the fact that spotless is recessive would make it easy to recognize first generation hybrids resulting from accidental cross-pollination with normal Pima, as they would have a well-developed spot (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kbarnsy 1924</a>). <a id="xref-ref-42-11" class="xref-bibr" href="#ref-42">Rong <em>et al.</em> (2004)</a> located the petal spot gene to two regions on Chr.1 and Chr.7. In our research, it was associated with the interval between A1625 and pAR040 on Chr.7, which is consistent with previous reports (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Endrizzi <em>et al.</em> 1985</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Yu <em>et al.</em> 2007</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Lacape <em>et al.</em> 2009</a>). Mapping of the three morphological markers, namely anther color, petal color, and petal spot in our research helps reveal or confirm their positons in cotton genome, which will benefit further research such as exploring the related genes controlling these traits.</p></div><div id="sec-19" class="subsection"><h3>Colinearity of the HM map with the tetraploid cotton genome</h3><p id="p-51">To study colinearity and genome variations, dotplots were performed between the HM genetic map and the <em>G. hirsutum</em> reference genome (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). The overall marker order on the HM map agreed well with the corresponding sequences on the 26 major scaffolds of the <em>G. hirsutum</em> genome (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Discrepancies in the orders along linkage groups and scaffolds were located in a few regions, which may be interpreted either as errors in genome assembly or in the construction of the genetic linkage map, or may be indicative of some structural rearrangements between different cotton species.</p></div><div id="sec-20" class="subsection"><h3>Inversions between HM and other maps and their phylogenetic context</h3><p id="p-52">Our HM map consisted mainly of SSR and RFLP markers, making it comparable with other published maps, and suitable for a wide range of investigations in structural, functional, and evolutionary genomics. In most cases, the arrangements of genetic loci along the chromosomes of the HM map were the same as in HB, HT, and HD maps. Some locus order differences were likely due to reversals of neighboring markers explicable by occasional missing data in either population; meanwhile, some significant structural rearrangements were also observed (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>). There is evidence in some genomic regions of inversions that differentiate among HM, HT, HB, and HD maps (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2D</a>). The tetraploid phylogeny proposed by <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Grover <em>et al.</em> (2015a)</a> facilitates phylogenetic inference regarding the origins of inversions between HM and other maps. No inversions were found to distinguish HM, HT, HB, and HD on Chr.1, Chr.6, Chr.10, Chr.11 Chr.14, Chr.16, Chr.21, Chr.22, and Chr.24 (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>), indicating that these nine chromosomes of <em>G. mustelinum</em>, <em>G. tomentosum</em>, <em>G. barbadense</em>, and <em>G. darwinii</em> have experienced little or no structural change since their divergence from a common ancestor.</p><p id="p-53">On Chr.12, Chr.13, and Chr.18, HB and HD share common DNA marker orders, but each is inverted with respect to HM and HT (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>). The most parsimonious interpretation of this is that inversion occurred in a <em>G. barbadense–G. darwinii</em> common ancestor after its divergence from the <em>G. ekmanianum</em>–<em>G. hirsutum</em>–<em>G. tomentosum</em> clade. Less simply explained is a region of Chr.5 with common arrangement between HT and HD, but with each being inverted relative to HM and HB (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>). If some artifactual reason for this is not found, then it seems to suggest that <em>G. darwinii</em> and <em>G. tomentosum</em> have undergone similar changes, despite belonging to different tetraploid cotton clades. These species are both island endemics, and typically reside near coastlines with similar environments (<em>G. darwinii</em> is endemic to the Galapagos Islands, <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Wendel and Percy 1990</a>; <em>G. tomentosum</em> is endemic to the Hawaiian Islands, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">DeJoode and Wendel 1992</a>). Close relationships between these two island-endemic species has also been revealed in previous research (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> 2015</a>).</p><p id="p-54">Several inversions appear to be lineage-specific, presumably occurring more recently than the divergence of the affected tetraploid species from its nearest relative. Inversions limited to HM were found on Chr.3, Chr.15 and Chr.17; to HD on Chr.2, Chr.25 and Chr.26; to HT on Chr. 8 and Chr.19; and to HB on Chr.4, Chr.7, Chr.8, Chr.9, Chr.20, and Chr.23 (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-21" class="subsection"><h3>Application of the HM map in QTL mapping</h3><p id="p-55">The <em>G. hirsutum</em> by <em>G. mustelinum</em> map was applied in mapping of fiber elongation QTL in advanced-backcross populations derived from the same parents. A total of 14 QTL was detected, with good reproducibility. Eight of the 14 QTL were detected simultaneously across different families or environments; six of the eight QTL were also mapped in the BC<sub>3</sub>F<sub>2</sub> generation (<a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2016</a>); and the QTL <em>qELO-19-1</em> was detected in three generations, and also by joint analysis across environments. While <em>G. mustelinum</em> produces little fiber, <em>G. mustelinum</em> alleles for four (28.6%) of the 14 QTL increased fiber elongation (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), indicating the potential benefit of introgressing them into Upland cotton.</p><p id="p-56">The identification of four transgressive QTL from this wild species provided one example of the value of our HM map. The HM map reported here is an important tool to elucidate cotton genome structure, and it may also aid in future genome sequencing of <em>G. mustelinum</em>. In addition, it will be beneficial to QTL analysis and facilitate cotton breeding with molecular marker technology.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank Edward L. Lubbers and Jennifer McCurdy for help with field work. We appreciate financial help from the National Science Foundation (PFI award IIP 0917856; AIR award IIP 1127755), Cotton Incorporated, and the China Scholarship Council.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-58">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029116/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-59"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-05"><span class="received-label">Received </span>January 5, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-29"><span class="accepted-label">Accepted </span>March 29, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.1" data-doi="10.1007/s12041-007-0012-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahuja</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhayal</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparative characteristics and gene action in three petal-spotted mutants of <em>Gossypium hirsutum</em>.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">86</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">81</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgenetics%26rft.stitle%253DJ%2BGenet%26rft.aulast%253DAhuja%26rft.auinit1%253DS.%2BL.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D84%26rft.atitle%253DComparative%2Bcharacteristics%2Band%2Bgene%2Baction%2Bin%2Bthree%2Bpetal-spotted%2Bmutants%2Bof%2BGossypium%2Bhirsutum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12041-007-0012-0%26rft_id%253Dinfo%253Apmid%252F17656854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12041-007-0012-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17656854&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.2" data-doi="10.1371/journal.pone.0045739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blenda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garsmeur</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">e45739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlenda%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45739%26rft.epage%253De45739%26rft.atitle%253DA%2Bhigh%2Bdensity%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Bthrough%2Bmolecular%2Bmarker%2Bredundancy%2Bcheck.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045739%26rft_id%253Dinfo%253Apmid%252F23029214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029214&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.3" data-doi="10.1139/g98-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Comparative genetic mapping of allotetraploid cotton and its diploid progenitors.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D42%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg98-118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g98-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams-Coplin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">EST derived PCR-based markers for functional gene homologues in cotton.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DChee%26rft.auinit1%253DP.%2BW.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D462%26rft.atitle%253DEST%2Bderived%2BPCR-based%2Bmarkers%2Bfor%2Bfunctional%2Bgene%2Bhomologues%2Bin%2Bcotton.%26rft_id%253Dinfo%253Apmid%252F15190362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15190362&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.5" data-doi="10.1016/j.gene.2015.08.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A high-density SSR genetic map constructed from a F<sub>2</sub> population of <em>Gossypium hirsutum</em> and <em>Gossypium darwinii</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">574</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D574%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2015.08.022%26rft_id%253Dinfo%253Apmid%252F26275937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2015.08.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26275937&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.6" data-doi="10.2307/2445059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeJoode</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Genetic diversity and origin of the Hawaiian-Islands cotton, <em>Gossypium tomentosum</em>.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D79%26rft.spage%253D1311%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2445059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2445059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992JY42900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Linkage analysis of open bud (<em>ob2</em>) and yellow petal (<em>Y1</em>) in cotton.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">461</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D34%26rft.spage%253D461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.8" data-doi="10.1016/S0065-2660(08)60515-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turcotte</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Genetics cytology and evolution of <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BGenet.%26rft.volume%253D23%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960515-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60515-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1673.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fryxell</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Taxonomy and germplasm resources,</span> pp. <span class="cit-fpage">27</span>–<span class="cit-lpage">57</span> in <span class="cit-source"><em>Cotton</em>,</span> edited by R. J. <span class="cit-ed"><span class="cit-name-surname">Kohel</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. F.</span></span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fryxell</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A revised taxonomic interpretation of <em>Gossypium</em> L., (Malvacea).</span> <abbr class="cit-jnl-abbrev">Rheedea</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">108</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRheedea%26rft.volume%253D2%26rft.spage%253D108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.11" data-doi="10.1016/j.ympev.2015.05.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jareczek</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Re-evaluating the phylogeny of allopolyploid <em>Gossypium</em> L.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D92%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.05.023%26rft_id%253Dinfo%253Apmid%252F26049043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.05.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26049043&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.12" data-doi="10.1007/s10722-014-0138-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jareczek</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Molecular confirmation of species status for the allopolyploid cotton species, <em>Gossypium ekmanianum</em> Wittmack.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D62%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-014-0138-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-014-0138-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.13" data-doi="10.1111/j.1744-7909.2006.00174.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular tagging and mapping of quantitative trait loci for lint percentage and morphological marker genes in Upland cotton.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">320</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D48%26rft.spage%253D320%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2006.00174.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2006.00174.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.14" data-doi="10.1534/genetics.107.070375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A microsatellite-based, generic linkage map reveals genome structure, function and evolution in <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">527</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070375%26rft_id%253Dinfo%253Apmid%252F17409069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNTI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.15" data-doi="10.1186/1471-2164-9-314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A preliminary analysis of genome structure and composition in <em>Gossypium hirsutum</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGuo%26rft.auinit1%253DW.%26rft.volume%253D9%26rft.spage%253D314%26rft.epage%253D314%26rft.atitle%253DA%2Bpreliminary%2Banalysis%2Bof%2Bgenome%2Bstructure%2Band%2Bcomposition%2Bin%2BGossypium%2Bhirsutum.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-314%26rft_id%253Dinfo%253Apmid%252F18590573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18590573&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.16" data-doi="10.1007/s00438-004-1059-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic mapping of EST-derived microsatellites from the diploid <em>Gossypium arboreum</em> in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">308</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DHan%26rft.auinit1%253DZ.%2BG.%26rft.volume%253D272%26rft.issue%253D3%26rft.spage%253D308%26rft.epage%253D327%26rft.atitle%253DGenetic%2Bmapping%2Bof%2BEST-derived%2Bmicrosatellites%2Bfrom%2Bthe%2Bdiploid%2BGossypium%2Barboreum%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1059-8%26rft_id%253Dinfo%253Apmid%252F15368122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1059-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15368122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.17" data-doi="10.1007/s00122-005-0142-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gou</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Characteristics, development and mapping of <em>Gossypium hirsutum</em> derived EST-SSRs in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHan%26rft.auinit1%253DZ.%26rft.volume%253D112%26rft.issue%253D3%26rft.spage%253D430%26rft.epage%253D439%26rft.atitle%253DCharacteristics%252C%2Bdevelopment%2Band%2Bmapping%2Bof%2BGossypium%2Bhirsutum%2Bderived%2BEST-SSRs%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0142-9%26rft_id%253Dinfo%253Apmid%252F16341684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0142-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16341684&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234552400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.18" data-doi="10.1007/BF02983156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harland</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1929</span> <span class="cit-article-title">The genetics of cotton. Part II the inheritance of pollen colour in New World cottons.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D20%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02983156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02983156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">QTL mapping for economic traits based on a dense genetic map of cotton with PCR-based markers using the interspecific cross of <em>Gossypium hirsutum</em> × <em>Gossypium barbadense</em>.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D153%26rft.spage%253D181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.20" data-doi="10.1007/s12041-013-0286-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Construction of microsatellite-based linkage map and mapping of nectarilessness and hairiness genes in <em>Gossypium tomentosum</em>.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">445</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D92%26rft.spage%253D445%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12041-013-0286-3%26rft_id%253Dinfo%253Apmid%252F24371166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12041-013-0286-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24371166&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashra</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buyyarapu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development of a 63K SNP array for cotton and high-density mapping of intraspecific and interspecific populations of <em>Gossypium</em> spp. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genom. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1187</span>–<span class="cit-lpage">1209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenom.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kbarnsy</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1924</span> <span class="cit-article-title">Inheritance of petal spot in pima cotton.</span> <abbr class="cit-jnl-abbrev">J. Agric. Res.</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">491</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BRes.%26rft.volume%253D27%26rft.spage%253D491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.23" data-doi="10.2135/cropsci1971.0011183X001100020036x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Isolines in cotton: effects of nine dominant genes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DKohel%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D287%26rft.epage%253D289%26rft.atitle%253DIsolines%2Bin%2BCotton%253A%2BEffects%2Bof%2BNine%2BDominant%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1971.0011183X001100020036x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1971.0011183X001100020036x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Inheritance and linkage analysis of the yellow pulvinus mutant of cotton.</span> <abbr class="cit-jnl-abbrev">J. Cotton Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCotton%2BSci.%26rft.volume%253D6%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.25" data-doi="10.1046/j.1365-313X.1993.04020403.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konieczny</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Procedure for mapping <em>Arabidopsis</em> mutations using co-dominant ecotype-specific PCR-based markers.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKonieczny%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DA%2Bprocedure%2Bfor%2Bmapping%2BArabidopsis%2Bmutations%2Busing%2Bco-dominant%2Becotype-specific%2BPCR-based%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1993.04020403.x%26rft_id%253Dinfo%253Apmid%252F8106085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1993.04020403.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8106085&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LT72100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Gossypium ekmanianum</em> (Malvaceae), algodón silvestre de la República Dominicana.</span> <abbr class="cit-jnl-abbrev">Bonplandia</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBonplandia%26rft.volume%253D17%26rft.spage%253D55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.27" data-doi="10.1139/g03&amp;ndash;050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibivilliers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bojinov</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A combined RFLP-SSR-AFLP map of tetraploid cotton based on a <em>Gossypium hirsutum</em> × <em>Gossypium barbadense</em> backcross population.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D612%26rft.epage%253D626%26rft.atitle%253DA%2Bcombined%2BRFLP-SSR-AFLP%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bbased%2Bon%2Ba%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%2Bbackcross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg03%2526ndash%253B050%26rft_id%253Dinfo%253Apmid%252F12897870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g03&amp;ndash;050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12897870&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.28" data-doi="10.1007/s00122-009-1037-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arioli</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derijcker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forestier-Chiron</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A new interspecific, <em>Gossypium hirsutum</em> × <em>G. barbadense</em>, RIL population: towards a unified consensus linkage map of tetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D292%26rft.atitle%253DA%2Bnew%2Binterspecific%252C%2BGossypium%2Bhirsutum%2Bx%2BG.%2Bbarbadense%252C%2BRIL%2Bpopulation%253A%2Btowards%2Ba%2Bunified%2Bconsensus%2Blinkage%2Bmap%2Bof%2Btetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1037-y%26rft_id%253Dinfo%253Apmid%252F19387609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1037-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19387609&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.29" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.30" data-doi="10.1038/ng.2987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2987%26rft_id%253Dinfo%253Apmid%252F24836287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836287&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.31" data-doi="10.1038/nbt.3208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome sequence of cultivated Upland cotton (<em>Gossypium hirsutum</em> TM-1) provides insights into genome evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3208%26rft_id%253Dinfo%253Apmid%252F25893780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25893780&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.32" data-doi="10.1111/j.1439-0523.2004.01039.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Linkage map construction and mapping QTL for cotton fibre quality using SRAP, SSR and RAPD.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D124%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2004.01039.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2004.01039.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.33" data-doi="10.1007/s11032-011-9547-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">297</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D29%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9547-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9547-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">O. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Registration of PD94042 germplasm line of Upland cotton with high yield and fiber maturity.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMay%26rft.auinit1%253DO.%2BL.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D597%26rft.epage%253D598%26rft.atitle%253DRegistration%2Bof%2BPD%2B94042%2BGermplasm%2BLine%2Bof%2BUpland%2BCotton%2Bwith%2BHigh%2BYield%2Band%2BFiber%2BMaturity%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.35" data-doi="10.1007/s00122-004-1612-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giband</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brottier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risterucci</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Wide coverage of the tetraploid cotton genome using newly developed microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DNguyen%26rft.auinit1%253DT.%2BB.%26rft.volume%253D109%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D175%26rft.atitle%253DWide%2Bcoverage%2Bof%2Bthe%2Btetraploid%2Bcotton%2Bgenome%2Busing%2Bnewly%2Bdeveloped%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1612-1%26rft_id%253Dinfo%253Apmid%252F14997301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1612-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14997301&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221936300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.36" data-doi="10.1007/s00438-005-0037-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alabady</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sickler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line cotton population.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">428</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DPark%26rft.auinit1%253DY.%2BH.%26rft.volume%253D274%26rft.issue%253D4%26rft.spage%253D428%26rft.epage%253D441%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Bnew%2Bcotton%2Bfiber%2Bloci%2Busing%2BEST-derived%2Bmicrosatellites%2Bin%2Ban%2Binterspecific%2Brecombinant%2Binbred%2Bline%2Bcotton%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-005-0037-0%26rft_id%253Dinfo%253Apmid%252F16187061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-005-0037-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16187061&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233380500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.37" data-doi="10.1007/BF02670470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A rapid method for extraction of cotton (<em>Gossypium</em> spp.) genomic DNA suitable for RFLP or PCR analysis.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">122</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D11%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02670470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02670470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.38" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1673.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Qualitative genetics,</span> pp. <span class="cit-fpage">319</span>–<span class="cit-lpage">360</span> in <span class="cit-source"><em>Cotton: Origin</em>, <em>History</em>, <em>Technology</em>, <em>and Production</em></span>, edited by W. C. <span class="cit-ed"><span class="cit-name-surname">Smith</span></span>, <span class="cit-ed"><span class="cit-name-surname">Cothren</span>  <span class="cit-name-given-names">J. T.</span></span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinisch</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendelt</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A detailed RFLP map of cotton, <em>Gossypium hirsutum</em> × <em>Gossypium barbadense</em>: chromosome organization and evolution in a disomic polyploidy genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DReinisch%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D829%26rft.epage%253D847%26rft.atitle%253DA%2Bdetailed%2BRFLP%2Bmap%2Bof%2Bcotton%252C%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%253A%2Bchromosome%2Borganization%2Band%2Bevolution%2Bin%2Ba%2Bdisomic%2Bpolyploid%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F7851778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvODI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhyne</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Research efforts with genes of linkage group V. II. Loci distal the centromere.</span> <abbr class="cit-jnl-abbrev">Beltwide Cotton Production Research Conferences</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">596</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBeltwide%2BCotton%2BProduction%2BResearch%2BConferences%26rft.volume%253D2%26rft.spage%253D596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.42" data-doi="10.1534/genetics.166.1.389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals types of genome organization, transmission and evolution of cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRong%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D417%26rft.atitle%253DA%2B3347-Locus%2BGenetic%2BRecombination%2BMap%2Bof%2BSequence-Tagged%2BSites%2BReveals%2BFeatures%2Bof%2BGenome%2BOrganization%252C%2BTransmission%2Band%2BEvolution%2Bof%2BCotton%2B%2528Gossypium%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.389%26rft_id%253Dinfo%253Apmid%252F15020432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.43" data-doi="10.1111/jipb.12288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Constructing a high-density linkage map for <em>Gossypium hirsutum</em> × <em>Gossypium barbadense</em> and identifying QTLs for lint percentage.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">450</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D57%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjipb.12288%26rft_id%253Dinfo%253Apmid%252F25263268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jipb.12288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25263268&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A comparison of genetic maps constructed from haploid and BC mapping populations from the same crossing between <em>Gossypium hirsutum</em> L. and <em>Gossypium barbadense</em> L.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">378</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSong%26rft.auinit1%253DX.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D378%26rft.epage%253D390%26rft.atitle%253DA%2Bcomparison%2Bof%2Bgenetic%2Bmaps%2Bconstructed%2Bfrom%2Bhaploid%2Band%2BBC1%2Bmapping%2Bpopulations%2Bfrom%2Bthe%2Bsame%2Bcrossing%2Bbetween%2BGossypium%2Bhirsutum%2BL.%2Band%2BGossypium%2Bbarbadense%2BL.%26rft_id%253Dinfo%253Apmid%252F16121235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16121235&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1673.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linskens</span>  <span class="cit-name-given-names">H. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-source"><em>Pollen–Biology</em>, <em>Biochemistry</em>, <em>Management</em></span>, <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Interspecific homologies between gene loci in <em>Gossypium</em>. I pollen color.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStephens%26rft.auinit1%253DS.%2BG.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D701%26rft.epage%253D711%26rft.atitle%253DINTERSPECIFIC%2BHOMOLOGIES%2BBETWEEN%2BGENE%2BLOCI%2BIN%2BGOSSYPIUM.%2BI.%2BPOLLEN%2BCOLOR%26rft_id%253Dinfo%253Apmid%252F17247514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzOS81LzcwMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.47" data-doi="10.2135/cropsci1966.0011183X000600020003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turcotte</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feaster</span>  <span class="cit-name-given-names">C. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">A second locus for pollen color in Pima cotton, <em>Gossypium barbadense L</em>.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTurcotte%26rft.auinit1%253DE.%2BL.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D117%26rft.epage%253D119%26rft.atitle%253DA%2BSecond%2BLocus%2Bfor%2BPollen%2BColor%2Bin%2BPima%2BCotton%252C%2BGossypium%2Bbarbadense%2BL.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1966.0011183X000600020003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1966.0011183X000600020003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.48" data-doi="10.1007/s00122-005-2032-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waghmare</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic mapping of a cross between <em>Gossypium hirsutum</em> (cotton) and the Hawaiian endemic, <em>Gossypium tomentosum</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWaghmare%26rft.auinit1%253DV.%2BN.%26rft.volume%253D111%26rft.issue%253D4%26rft.spage%253D665%26rft.epage%253D676%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Ba%2Bcross%2Bbetween%2BGossypium%2Bhirsutum%2B%2528cotton%2529%2Band%2Bthe%2BHawaiian%2Bendemic%252C%2BGossypium%2Btomentosum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-2032-6%26rft_id%253Dinfo%253Apmid%252F16044266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-2032-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16044266&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231313100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.49" data-doi="10.1007/s00122-012-1911-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular diversity, genomic constitution, and QTL mapping of fiber quality by mapped SSRs in introgression lines derived from <em>Gossypium hirsutum</em> × <em>G. darwinii Watt</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DB.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1263%26rft.epage%253D1274%26rft.atitle%253DMolecular%2Bdiversity%252C%2Bgenomic%2Bconstitution%252C%2Band%2BQTL%2Bmapping%2Bof%2Bfiber%2Bquality%2Bby%2Bmapped%2BSSRs%2Bin%2Bintrogression%2Blines%2Bderived%2Bfrom%2BGossypium%2Bhirsutum%2B%25C3%2597%2BG.%2Bdarwinii%2BWatt.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1911-x%26rft_id%253Dinfo%253Apmid%252F22733444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1911-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733444&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.50" data-doi="10.4238/2015.September.9.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development of EST-SSR markers related to disease resistance and their application in genetic diversity and evolution analysis in <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">10630</span>–<span class="cit-lpage">10644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%26rft.volume%253D14%26rft.spage%253D10630%26rft_id%253Dinfo%253Adoi%252F10.4238%252F2015.September.9.4%26rft_id%253Dinfo%253Apmid%252F26400294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4238/2015.September.9.4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26400294&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.51" data-doi="10.2135/cropsci2015.12.0753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Draye</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Advanced backcross QTL analysis of fiber elongation in a cross between <em>Gossypium hirsutum</em> and <em>G. mustelinum</em>.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.12.0753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.12.0753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.12.0753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.6.1673.52" data-doi="10.1038/ng.2371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1098</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1098%26rft.epage%253D1103%26rft.atitle%253DThe%2Bdraft%2Bgenome%2Bof%2Ba%2Bdiploid%2Bcotton%2BGossypium%2Braimondii.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2371%26rft_id%253Dinfo%253Apmid%252F22922876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922876&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A whole-genome DNA marker map for cotton based on the D-genome sequence of <em>Gossypium raimondii</em> L. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genom. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenom.%2BGenet.%26rft.volume%253D3%26rft.spage%253D1759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1673.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Taxonomy and evolution of the cotton genus, <em>Gossypium</em></span>, pp.<em> </em><span class="cit-fpage">25</span><em>–</em><span class="cit-lpage">44</span> <em>in</em> <span class="cit-source">Cotton</span>, <span class="cit-edition">Ed. 2</span>, edited by D. D. <span class="cit-ed"><span class="cit-name-surname">Fang</span></span>, <span class="cit-ed"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R. G.</span></span>. <span class="cit-publ-name">ASA, CSSA, and SSSA</span>, <span class="cit-publ-loc">Madison</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.55" data-doi="10.1016/S0065-2113(02)78004-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polyploidy and the evolutionary history of cotton.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D78%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2113%252802%252978004-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2113(02)78004-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.56" data-doi="10.1016/0305-1978(90)90123-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Allozyme diversity and introgression in the Galapagos-Islands endemic <em>Gossypium darwinii</em> and its relationship to continental <em>Gossypium barbadense</em>.</span> <abbr class="cit-jnl-abbrev">Biochem. Syst. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSyst.%2BEcol.%26rft.volume%253D18%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0305-1978%252890%252990123-W%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0305-1978(90)90123-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990EW56100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.57" data-doi="10.1007/BF00985907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic diversity in and phylogenetic relationships of the Brazilian endemic cotton, <em>Gossypium mustelinum</em> (Malvaceae).</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D192%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00985907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00985907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.58" data-doi="10.1093/bioinformatics/btm494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">QTLNetwork: mapping and visualizing genetic architecture of complex traits in experimental populations</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm494%26rft_id%253Dinfo%253Apmid%252F18202029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNC81LzcyMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.59" data-doi="10.1111/j.1744-7909.2007.00459.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-density linkage map of cultivated allotetraploid cotton based on SSR, TRAP, SRAP and AFLP markers.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">716</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D49%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2007.00459.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2007.00459.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.60" data-doi="10.1186/1471-2164-12-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between <em>Gossypium hirsutum</em> and <em>G. barbadense</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYu%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DGenome%2Bstructure%2Bof%2Bcotton%2Brevealed%2Bby%2Ba%2Bgenome-wide%2BSSR%2Bgenetic%2Bmap%2Bconstructed%2Bfrom%2Ba%2BBC1%2Bpopulation%2Bbetween%2Bgossypium%2Bhirsutum%2Band%2BG.%2Bbarbadense.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-15%26rft_id%253Dinfo%253Apmid%252F21214949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21214949&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genom. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenom.%2BGenet.%26rft.volume%253D2%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.62" data-doi="10.1007/s00122-002-1100-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular linkage map of allotetraploid cotton (Gossypium hirsutum L. × Gossypium barbadense L.) with a haploid population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D105%26rft.issue%253D8%26rft.spage%253D1166%26rft.epage%253D1174%26rft.atitle%253DMolecular%2Blinkage%2Bmap%2Bof%2Ballotetraploid%2Bcotton%2B%2528%2BGossypium%2Bhirsutum%2BL.%2Bx%2BGossypium%2Bbarbadense%2BL.%2529%2Bwith%2Ba%2Bhaploid%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-1100-4%26rft_id%253Dinfo%253Apmid%252F12582895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-1100-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179982700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.63" data-doi="10.1038/nbt.3207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequencing of allotetraploid cotton (<em>Gossypium hirsutum</em> L. acc. TM-1) provides a resource for fiber improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D531%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3207%26rft_id%253Dinfo%253Apmid%252F25893781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25893781&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1673.64" data-doi="10.1007/s00122-011-1649-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waghmare</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">QTL alleles for improved fiber quality from a wild Hawaiian cotton, <em>Gossypium tomentosum</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D123%26rft.issue%253D7%26rft.spage%253D1075%26rft.epage%253D1088%26rft.atitle%253DQTL%2Balleles%2Bfor%2Bimproved%2Bfiber%2Bquality%2Bfrom%2Ba%2Bwild%2BHawaiian%2Bcotton%252C%2BGossypium%2Btomentosum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1649-x%26rft_id%253Dinfo%253Apmid%252F21735234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1649-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21735234&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/6/1673.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1661" title="The Histone Variant H3.3 Is Enriched at Drosophila Amplicon Origins but Does Not Mark Them for Activation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1687" title="The Drosophila prage Gene, Required for Maternal Transcript Destabilization in Embryos, Encodes a Predicted RNA Exonuclease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/6"><span>Volume 6 Issue 6, June 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/6.cover-source.jpg?itok=WAtQozJg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1673" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qDvFyZbogxrx-dxRTl-0YHgh9z_6TA9dnz9z8o6awU0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/6/1673" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/6/1673" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/6/1673" class="active" data-icon-position="" data-hide-link-title="0">A Genetic Map Between Gossypium hirsutum and the Brazilian Endemic G. mustelinum and Its Application to QTL Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460391" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gJZ_PIVMXA02zb1CpjBhmSMIWAcPWMJOVrkRXXEWjig" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1673.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/6/1673" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2chR4zgOUzClS2t4ZLXeF2duD2SAU4a_G8YNTewBFAk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/6/1673" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1673.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460391" id="node460391--2" data-pisa="ggg;6/6/1673" data-pisa-master="ggg;g3.116.029116" data-apath="/ggg/6/6/1673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genetic Map Between <em>Gossypium hirsutum</em> and the Brazilian Endemic <em>G. mustelinum</em> and Its Application to QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7449-8292" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Baohua Wang</span>, <span class="highwire-citation-author" data-delta="1">Limei Liu</span>, <span class="highwire-citation-author" data-delta="2">Dong Zhang</span>, <span class="highwire-citation-author" data-delta="3">Zhimin Zhuang</span>, <span class="highwire-citation-author" data-delta="4">Hui Guo</span>, <span class="highwire-citation-author" data-delta="5">Xin Qiao</span>, <span class="highwire-citation-author" data-delta="6">Lijuan Wei</span>, <span class="highwire-citation-author" data-delta="7">Junkang Rong</span>, <span class="highwire-citation-author" data-delta="8">O. Lloyd May</span>, <span class="highwire-citation-author" data-delta="9">Andrew H. Paterson</span> and <span class="highwire-citation-author" data-delta="10">Peng W. Chee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1673-1685;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029116 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460391--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Baohua Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baohua%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABaohua%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7449-8292" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Baohua Wang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Limei Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Limei%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALimei%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhimin Zhuang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhimin%2BZhuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhuang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhimin%2BZhuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xin Qiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BQiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qiao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BQiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lijuan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lijuan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALijuan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Junkang Rong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Agriculture and Food Science, Zhejiang A&F University, Lin’an, Hangzhou, Zhejiang 311300, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junkang%2BRong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunkang%2BRong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">O. Lloyd May </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=O.%2BLloyd%2BMay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=May%20OL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AO.%2BLloyd%2BMay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrew H. Paterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BH.%2BPaterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paterson%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BH.%2BPaterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@uga.edu</a>
<a href="mailto:pwchee@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pwchee@uga.edu</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Peng W. Chee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BW.%2BChee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chee%20PW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BW.%2BChee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@uga.edu</a>
<a href="mailto:pwchee@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pwchee@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/6/1673" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/6/1673&amp;title=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/6/1673&amp;title=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/6/1673&amp;title=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/6/1673&amp;text=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/6/1673&amp;title=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/6/1673&amp;t=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/6/1673&amp;title=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/6/1673&amp;title=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460391/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460391" id="node460391--3" data-pisa="ggg;6/6/1673" data-pisa-master="ggg;g3.116.029116" data-apath="/ggg/6/6/1673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genetic Map Between <em>Gossypium hirsutum</em> and the Brazilian Endemic <em>G. mustelinum</em> and Its Application to QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7449-8292" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Baohua Wang</span>, <span class="highwire-citation-author" data-delta="1">Limei Liu</span>, <span class="highwire-citation-author" data-delta="2">Dong Zhang</span>, <span class="highwire-citation-author" data-delta="3">Zhimin Zhuang</span>, <span class="highwire-citation-author" data-delta="4">Hui Guo</span>, <span class="highwire-citation-author" data-delta="5">Xin Qiao</span>, <span class="highwire-citation-author" data-delta="6">Lijuan Wei</span>, <span class="highwire-citation-author" data-delta="7">Junkang Rong</span>, <span class="highwire-citation-author" data-delta="8">O. Lloyd May</span>, <span class="highwire-citation-author" data-delta="9">Andrew H. Paterson</span> and <span class="highwire-citation-author" data-delta="10">Peng W. Chee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1673-1685;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029116 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460391--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Baohua Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baohua%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABaohua%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7449-8292" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Baohua Wang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Limei Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Limei%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALimei%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhimin Zhuang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Life Sciences, Nantong University, Nantong, Jiangsu 226019, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhimin%2BZhuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhuang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhimin%2BZhuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xin Qiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BQiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qiao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BQiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lijuan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lijuan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALijuan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Junkang Rong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Agriculture and Food Science, Zhejiang A&F University, Lin’an, Hangzhou, Zhejiang 311300, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junkang%2BRong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunkang%2BRong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">O. Lloyd May </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=O.%2BLloyd%2BMay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=May%20OL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AO.%2BLloyd%2BMay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrew H. Paterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BH.%2BPaterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paterson%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BH.%2BPaterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@uga.edu</a>
<a href="mailto:pwchee@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pwchee@uga.edu</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Peng W. Chee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BW.%2BChee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chee%20PW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BW.%2BChee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@uga.edu</a>
<a href="mailto:pwchee@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pwchee@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1673&amp;rft.epage=1685&amp;rft.atitle=A%20Genetic%20Map%20Between%20Gossypium%20hirsutum%20and%20the%20Brazilian%20Endemic%20G.%20mustelinum%20and%20Its%20Application%20to%20QTL%20Mapping&amp;rft.volume=6&amp;rft.issue=6&amp;rft.date=2016-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Baohua&amp;rft.au=Liu%2C+Limei&amp;rft.au=Zhang%2C+Dong&amp;rft.au=Zhuang%2C+Zhimin&amp;rft.au=Guo%2C+Hui&amp;rft.au=Qiao%2C+Xin&amp;rft.au=Wei%2C+Lijuan&amp;rft.au=Rong%2C+Junkang&amp;rft.au=May%2C+O.+Lloyd&amp;rft.au=Paterson%2C+Andrew+H.&amp;rft.au=Chee%2C+Peng+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460391/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460391/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460391/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460391/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460391/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460391/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460391/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460391/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460391/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460391/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460391/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460391/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/6/1673" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/6/1673.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/6/1673.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/6/1673.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/6/1673.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
