<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/6/1597" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/6/1597.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/6/1597.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/6/1597.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/6/1597.atom" />
<meta name="HW.pisa" content="ggg;6/6/1597" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?" />
<meta name="DC.Identifier" content="10.1534/g3.116.028274" />
<meta name="DC.Date" content="2016-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Khedkar and Seshasayee. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomes evolve not only in base sequence but also in terms of their architecture, defined by gene organization and chromosome topology. Whereas genome sequence data inform us about the changes in base sequences for a large variety of organisms, the study of chromosome topology is restricted to a few model organisms studied using microscopy and chromosome conformation capture techniques. Here, we exploit whole genome sequence data to study the link between gene organization and chromosome topology in bacteria. Using comparative genomics across ∼250 pairs of closely related bacteria we show that: (a) many organisms show a high degree of interreplichore translocations throughout the chromosome and not limited to the inversion-prone terminus ( ter ) or the origin of replication ( oriC ); (b) translocation maps may reflect chromosome topologies; and (c) symmetric interreplichore translocations do not disrupt the distance of a gene from oriC or affect gene expression states or strand biases in gene densities. In summary, we suggest that translocation maps might be a first line in defining a gross chromosome topology given a pair of closely related genome sequences." />
<meta name="DC.Contributor" content="Supriya Khedkar" />
<meta name="DC.Contributor" content="Aswin Sai Narain Seshasayee" />
<meta name="article:published_time" content="2016-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomes evolve not only in base sequence but also in terms of their architecture, defined by gene organization and chromosome topology. Whereas genome sequence data inform us about the changes in base sequences for a large variety of organisms, the study of chromosome topology is restricted to a few model organisms studied using microscopy and chromosome conformation capture techniques. Here, we exploit whole genome sequence data to study the link between gene organization and chromosome topology in bacteria. Using comparative genomics across ∼250 pairs of closely related bacteria we show that: (a) many organisms show a high degree of interreplichore translocations throughout the chromosome and not limited to the inversion-prone terminus (&lt;i&gt;ter&lt;/i&gt;) or the origin of replication (&lt;i&gt;oriC&lt;/i&gt;); (b) translocation maps may reflect chromosome topologies; and (c) symmetric interreplichore translocations do not disrupt the distance of a gene from &lt;i&gt;oriC&lt;/i&gt; or affect gene expression states or strand biases in gene densities. In summary, we suggest that translocation maps might be a first line in defining a gross chromosome topology given a pair of closely related genome sequences.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/06/01" />
<meta name="citation_mjid" content="ggg;6/6/1597" />
<meta name="citation_id" content="6/6/1597" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/6/1597" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/6/1597.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/6/1597.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/6/1597.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028274" />
<meta name="citation_pmid" content="27172194" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1597" />
<meta name="citation_lastpage" content="1606" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Supriya Khedkar" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_institution" content="SASTRA University, Thanjavur 613401, Tamil Nadu, India" />
<meta name="citation_author_email" content="aswin@ncbs.res.in" />
<meta name="citation_author_email" content="supriyak@ncbs.res.in" />
<meta name="citation_author" content="Aswin Sai Narain Seshasayee" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_email" content="aswin@ncbs.res.in" />
<meta name="citation_author_email" content="supriyak@ncbs.res.in" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Bernander;citation_author=S. Dasgupta;citation_author=K. Nordström;citation_title=The E. coli cell cycle and the plasmid R1 replication cycle in the absence of the DnaA protein.;citation_pages=1145-1153;citation_volume=64;citation_year=1991;citation_issue=6;citation_pmid=1848482;citation_doi=10.1016/0092-8674(91)90269-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. H. S. Block;citation_author=R. Hussein;citation_author=L. W. Liang;citation_author=H. N. Lim;citation_title=Regulatory consequences of gene translocation in bacteria.;citation_volume=40;citation_year=2012;citation_pmid=22833608;citation_doi=10.1093/nar/gks694" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in microbiology;citation_journal_abbrev=Curr Opin Microbiol;citation_author=D. F. Browning;citation_author=D. C. Grainger;citation_author=S. J. Busby;citation_title=Effects of nucleoid-associated proteins on bacterial chromosome structure and gene expression.;citation_pages=773-780;citation_volume=13;citation_year=2010;citation_issue=6;citation_pmid=20951079;citation_doi=10.1016/j.mib.2010.09.013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. A. Bryant;citation_author=L. E. Sellars;citation_author=S. J. W. Busby;citation_author=D. J. Lee;citation_title=Chromosome position effects on gene expression in Escherichia coli K-12.;citation_volume=42;citation_year=2014;citation_pmid=25209233;citation_doi=10.1093/nar/gku828" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. E. Cabrera;citation_author=D. J. Jin;citation_title=The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues.;citation_pages=1493-1505;citation_volume=50;citation_year=2003;citation_issue=5;citation_pmid=14651633;citation_doi=10.1046/j.1365-2958.2003.03805.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Cagliero;citation_author=R. S. Grand;citation_author=M. B. Jones;citation_author=D. J. Jin;citation_author=J. M. O’Sullivan;citation_title=Genome conformation capture reveals that the Escherichia coli chromosome is organized by replication and transcription.;citation_volume=41;citation_year=2013;citation_pmid=23632166;citation_doi=10.1093/nar/gkt325" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=C. Condon;citation_author=D. Liveris;citation_author=C. Squires;citation_author=I. Schwartz;citation_author=C. L. Squires;citation_title=rRNA operon multiplicity in Escherichia coli and the physiological implications of rrn inactivation.;citation_pages=4152-4156;citation_volume=177;citation_year=1995;citation_issue=14;citation_pmid=7608093;citation_doi=10.1128/jb.177.14.4152-4156.1995" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. Cooper;citation_author=C. E. Helmstetter;citation_title=Chromosome replication and the division cycle of Escherichia coli B/r.;citation_pages=519-540;citation_volume=31;citation_year=1968;citation_issue=3;citation_pmid=4866337;citation_doi=10.1016/0022-2836(68)90425-7" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=E. Couturier;citation_author=E. P. C. Rocha;citation_title=Replication-associated gene dosage effects shape the genomes of fast-growing bacteria but only for transcription and translation genes.;citation_pages=1506-1518;citation_volume=59;citation_year=2006;citation_issue=5;citation_pmid=16468991;citation_doi=10.1111/j.1365-2958.2006.05046.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. T. Dame;citation_author=M. C. Noom;citation_author=G. J. L. Wuite;citation_title=Bacterial chromatin organization by H-NS protein unravelled using dual DNA manipulation.;citation_pages=387-390;citation_volume=444;citation_year=2006;citation_issue=7117;citation_pmid=17108966;citation_doi=10.1038/nature05283" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Dekker;citation_author=K. Rippe;citation_author=M. Dekker;citation_author=N. Kleckner;citation_title=Capturing Chromosome Conformation;citation_pages=1306-1311;citation_volume=295;citation_year=2002;citation_issue=5558;citation_pmid=11847345;citation_doi=10.1126/science.1067799" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. L. Delcher;citation_author=D. Harmon;citation_author=S. Kasif;citation_author=O. White;citation_author=S. L. Salzberg;citation_title=Improved microbial gene identification with GLIMMER.;citation_volume=27;citation_year=1999;citation_pmid=10556321;citation_doi=10.1093/nar/27.23.4636" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=S. C. Dillon;citation_author=C. J. Dorman;citation_title=Bacterial nucleoid-associated proteins, nucleoid structure and gene expression.;citation_pages=185-195;citation_volume=8;citation_year=2010;citation_issue=3;citation_pmid=20140026;citation_doi=10.1038/nrmicro2261" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=J. U. Dimude;citation_author=A. Stockum;citation_author=S. L. Midgley-Smith;citation_author=A. L. Upton;citation_author=H. A. Foster;citation_title=The consequences of replicating in the wrong orientation: bacterial chromosome duplication without an active replication origin.;citation_pages=e01294-e15;citation_volume=6;citation_year=2015;citation_pmid=26530381" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. DiNardo;citation_author=K. A. Voelkel;citation_author=R. Sternglanz;citation_author=A. E. Reynolds;citation_author=A. Wright;citation_title=Escherichia coli DNA topoisomerase I mutants have compensatory mutations in DNA gyrase genes.;citation_pages=43-51;citation_volume=31;citation_year=1982;citation_issue=1;citation_pmid=6297752;citation_doi=10.1016/0092-8674(82)90403-2" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.;citation_author=C. J. Dorman;citation_title=Genome architecture and global gene regulation in bacteria: making progress towards a unified model?;citation_pages=349-355;citation_volume=11;citation_year=2013;citation_pmid=23549066;citation_doi=10.1038/nrmicro3007" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=P. Dupaigne;citation_author=N. K. Tonthat;citation_author=O. Espéli;citation_author=T. Whitfill;citation_author=F. Boccard;citation_title=Molecular Basis for a Protein-Mediated DNA-Bridging Mechanism that Functions in Condensation of the E. coli Chromosome.;citation_pages=560-571;citation_volume=48;citation_year=2012;citation_issue=4;citation_pmid=23084832;citation_doi=10.1016/j.molcel.2012.09.009" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol;citation_title=Evidence for symmetric chromosomal inversions around the replication origin in bacteria.;citation_author=J.A. Eisen;citation_author=J.F. Heidelberg;citation_author=O. White;citation_author=S.L. Salzberg;citation_volume=1;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. M. Engreitz;citation_author=V. Agarwala;citation_author=L. A. Mirny;citation_title=Three-dimensional genome architecture influences partner selection for chromosomal translocations in human disease.;citation_pages=e44196-e44196;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23028501;citation_doi=10.1371/journal.pone.0044196" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Esnault;citation_author=M. Valens;citation_author=O. Espéli;citation_author=F. Boccard;citation_title=Chromosome structuring limits genome plasticity in Escherichia coli.;citation_pages=e226-e226;citation_volume=3;citation_year=2007;citation_issue=12;citation_pmid=18085828;citation_doi=10.1371/journal.pgen.0030226" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=D. Esposito;citation_author=G. F. Gerard;citation_title=The Escherichia coli Fis Protein Stimulates Bacteriophage {lambda} Integrative Recombination In Vitro;citation_pages=3076-3080;citation_volume=185;citation_year=2003;citation_issue=10;citation_pmid=12730167;citation_doi=10.1128/JB.185.10.3076-3080.2003" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=J. Clements;citation_author=S. R. Eddy;citation_title=HMMER web server: interactive sequence similarity searching.;citation_volume=39;citation_year=2011;citation_pmid=21593126;citation_doi=10.1093/nar/gkr367" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Freilich;citation_author=A. Kreimer;citation_author=E. Borenstein;citation_author=N. Yosef;citation_author=R. Sharan;citation_title=Metabolic-network-driven analysis of bacterial ecological strategies.;citation_pages=R61-R61;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19500338;citation_doi=10.1186/gb-2009-10-6-r61" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Gao;citation_author=C.-T. Zhang;citation_title=DoriC: a database of oriC regions in bacterial genomes.;citation_volume=23;citation_year=2007;citation_pmid=17496319;citation_doi=10.1093/bioinformatics/btm255" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Gowrishankar;citation_title=End of the beginning: elongation and termination features of alternative modes of chromosomal replication initiation in bacteria.;citation_pages=e1004909;citation_volume=11;citation_year=2015;citation_pmid=25569209;citation_doi=10.1371/journal.pgen.1004909" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=S. Gruber;citation_author=J.-W. Veening;citation_author=J. Bach;citation_author=M. Blettinger;citation_author=M. Bramkamp;citation_title=Interlinked sister chromosomes arise in the absence of Condensin during fast replication in B. subtilis.;citation_pages=293-298;citation_volume=24;citation_year=2014;citation_pmid=24440399;citation_doi=10.1016/j.cub.2013.12.049" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Hawkins;citation_author=S. Malla;citation_author=M. J. Blythe;citation_author=C. A. Nieduszynski;citation_author=T. Allers;citation_title=Accelerated growth in the absence of DNA replication origins.;citation_pages=544-547;citation_volume=503;citation_year=2013;citation_pmid=24185008;citation_doi=10.1038/nature12650" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=I. Junier;citation_author=J. Hérisson;citation_author=F. Képès;citation_title=Genomic organization of evolutionarily correlated genes in bacteria: limits and strategies.;citation_pages=369-386;citation_volume=419;citation_year=2012;citation_issue=5;citation_pmid=22446685;citation_doi=10.1016/j.jmb.2012.03.009" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Junier;citation_author=F. Boccard;citation_author=O. Espeli;citation_title=Polymer modeling of the E. coli genome reveals the involvement of locus positioning and macrodomain structuring for the control of chromosome conformation and segregation.;citation_volume=42;citation_year=2014;citation_pmid=24194594;citation_doi=10.1093/nar/gkt1005" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=J. A. Klappenbach;citation_author=J. M. Dunbar;citation_author=T. M. Schmidt;citation_title=rRNA Operon Copy Number Reflects Ecological Strategies of Bacteria;citation_pages=1328-1333;citation_volume=66;citation_year=2000;citation_issue=4;citation_pmid=10742207;citation_doi=10.1128/AEM.66.4.1328-1333.2000" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. V. Koonin;citation_author=Y. I. Wolf;citation_title=Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world.;citation_volume=36;citation_year=2008;citation_pmid=18948295;citation_doi=10.1093/nar/gkn668" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=L. J. Koppes;citation_title=Nonrandom F-plasmid replication in Escherichia coli K-12.;citation_pages=2121-2123;citation_volume=174;citation_year=1992;citation_issue=7;citation_pmid=1551835;citation_doi=10.1128/jb.174.7.2121-2123.1992" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun;citation_author=A. Lal;citation_author=A. Dhar;citation_author=A. Trostel;citation_author=F. Kouzine;citation_author=A. S. N. Seshasayee;citation_title=Genome scale patterns of supercoiling in a bacterial chromosome;citation_pages=11055;citation_volume=7;citation_year=2016;citation_pmid=27025941;citation_doi=10.1038/ncomms11055" />
<meta name="citation_reference" content="citation_journal_title=FEBS Lett.;citation_author=M. C. Lagomarsino;citation_author=O. Espéli;citation_author=I. Junier;citation_title=From structure to function of bacterial chromosomes: Evolutionary perspectives and ideas for new experiments.;citation_pages=2996-3004;citation_volume=589;citation_year=2015;citation_pmid=26171924;citation_doi=10.1016/j.febslet.2015.07.002" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=T. B. Le;citation_author=M. T. Laub;citation_title=New approaches to understanding the spatial organization of bacterial genomes.;citation_pages=15-21;citation_volume=22;citation_year=2014;citation_pmid=25305533;citation_doi=10.1016/j.mib.2014.09.014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=T. B. K. Le;citation_author=M. V. Imakaev;citation_author=L. A. Mirny;citation_author=M. T. Laub;citation_title=High-resolution mapping of the spatial organization of a bacterial chromosome.;citation_volume=342;citation_year=2013;citation_pmid=24158908;citation_doi=10.1126/science.1242059" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Letunic;citation_author=P. Bork;citation_title=Interactive Tree Of Life v2: online annotation and display of phylogenetic trees made easy.;citation_volume=39;citation_year=2011;citation_pmid=21470960;citation_doi=10.1093/nar/gkr201" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=Y. Liu;citation_author=H. Chen;citation_author=L. J. Kenney;citation_author=J. Yan;citation_title=A divalent switch drives H-NS/DNA-binding conformations between stiffening and bridging modes;citation_pages=339-344;citation_volume=24;citation_year=2010;citation_issue=4;citation_pmid=20159954;citation_doi=10.1101/gad.1883510" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=M. Marbouty;citation_author=A. Cournac;citation_author=J.-F. Flot;citation_author=H. Marie-Nelly;citation_author=J. Mozziconacci;citation_title=Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms.;citation_volume=3;citation_year=2014;citation_pmid=25517076;citation_doi=10.7554/eLife.03318" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=M. Marbouty;citation_author=A. Le Gall;citation_author=D. I. Cattoni;citation_author=A. Cournac;citation_author=A. Koh;citation_title=Condensin- and replication-mediated bacterial chromosome folding and origin condensation revealed by Hi-C and super-resolution imaging.;citation_pages=588-602;citation_volume=59;citation_year=2015;citation_pmid=26295962;citation_doi=10.1016/j.molcel.2015.07.020" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. Maurer;citation_author=J. Fritz;citation_author=G. Muskhelishvili;citation_title=A systematic in vitro study of nucleoprotein complexes formed by bacterial nucleoid-associated proteins revealing novel types of DNA organization.;citation_pages=1261-1276;citation_volume=387;citation_year=2009;citation_issue=5;citation_pmid=19254726;citation_doi=10.1016/j.jmb.2009.02.050" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=A. T. McGeoch;citation_author=S. D. Bell;citation_title=Extra-chromosomal elements and the evolution of cellular DNA replication machineries.;citation_pages=569-574;citation_volume=9;citation_year=2008;citation_issue=7;citation_pmid=18523437;citation_doi=10.1038/nrm2426" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Mercier;citation_author=M.-A. Petit;citation_author=S. Schbath;citation_author=S. Robin;citation_author=M. El Karoui;citation_title=The MatP/matS site-specific system organizes the terminus region of the E. coli chromosome into a macrodomain.;citation_pages=475-485;citation_volume=135;citation_year=2008;citation_issue=3;citation_pmid=18984159;citation_doi=10.1016/j.cell.2008.08.031" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in microbiology;citation_journal_abbrev=Curr Opin Microbiol;citation_author=A. Mira;citation_author=L. Klasson;citation_author=S. G. E. Andersson;citation_title=Microbial genome evolution: sources of variability.;citation_pages=506-512;citation_volume=5;citation_year=2002;citation_issue=5;citation_pmid=12354559;citation_doi=10.1016/S1369-5274(02)00358-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. V. Mirkin;citation_author=S. M. Mirkin;citation_title=Mechanisms of Transcription-Replication Collisions in Bacteria;citation_pages=888-895;citation_volume=25;citation_year=2005;citation_issue=3;citation_pmid=15657418;citation_doi=10.1128/MCB.25.3.888-895.2005" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=J. Narula;citation_author=A. Kuchina;citation_author=D. D. Lee;citation_author=M. Fujita;citation_author=G. M. Süel;citation_title=Chromosomal arrangement of phosphorelay genes couples sporulation and DNA replication.;citation_pages=328-337;citation_volume=162;citation_year=2015;citation_pmid=26165942;citation_doi=10.1016/j.cell.2015.06.012" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. B. Needleman;citation_author=C. D. Wunsch;citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins.;citation_pages=443-453;citation_volume=48;citation_year=1970;citation_issue=3;citation_pmid=5420325;citation_doi=10.1016/0022-2836(70)90057-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=H. J. Nielsen;citation_author=J. R. Ottesen;citation_author=B. Youngren;citation_author=S. J. Austin;citation_author=F. G. Hansen;citation_title=The Escherichia coli chromosome is organized with the left and right chromosome arms in separate cell halves.;citation_pages=331-338;citation_volume=62;citation_year=2006;citation_issue=2;citation_pmid=17020576;citation_doi=10.1111/j.1365-2958.2006.05346.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. J. Pruss;citation_author=S. H. Manes;citation_author=K. Drlica;citation_title=Escherichia coli DNA topoisomerase I mutants: increased supercoiling is corrected by mutations near gyrase genes.;citation_pages=35-42;citation_volume=31;citation_year=1982;citation_issue=1;citation_pmid=6297751;citation_doi=10.1016/0092-8674(82)90402-0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Qian;citation_author=E. K. Dimitriadis;citation_author=R. Edgar;citation_author=P. Eswaramoorthy;citation_author=S. Adhya;citation_title=Galactose repressor mediated intersegmental chromosomal connections in Escherichia coli.;citation_volume=109;citation_year=2012;citation_pmid=22733746;citation_doi=10.1073/pnas.1208595109" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=Z. Qian;citation_author=M. Macvanin;citation_author=E. K. Dimitriadis;citation_author=X. He;citation_author=V. Zhurkin;citation_title=A new noncoding RNA arranges bacterial chromosome organization.;citation_volume=6;citation_year=2015;citation_pmid=26307168;citation_doi=10.1128/mBio.00998-15" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Reyes-Lamothe;citation_author=C. Possoz;citation_author=O. Danilova;citation_author=D. J. Sherratt;citation_title=Independent positioning and action of Escherichia coli replisomes in live cells.;citation_pages=90-102;citation_volume=133;citation_year=2008;citation_issue=1;citation_pmid=18394992;citation_doi=10.1016/j.cell.2008.01.044" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet. TIG;citation_author=P. Rice;citation_author=I. Longden;citation_author=A. Bleasby;citation_title=EMBOSS: the European Molecular Biology Open Software Suite.;citation_pages=276-277;citation_volume=16;citation_year=2000;citation_pmid=10827456;citation_doi=10.1016/S0168-9525(00)02024-2" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=E. P. C. Rocha;citation_title=The replication-related organization of bacterial genomes;citation_pages=1609-1627;citation_volume=150;citation_year=2004;citation_issue=6;citation_pmid=15184548;citation_doi=10.1099/mic.0.26974-0" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. P. C. Rocha;citation_title=The organization of the bacterial genome.;citation_pages=211-233;citation_volume=42;citation_year=2008;citation_pmid=18605898;citation_doi=10.1146/annurev.genet.42.110807.091653" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. P. C. Rocha;citation_author=A. Danchin;citation_title=Gene essentiality determines chromosome organisation in bacteria;citation_pages=6570-6577;citation_volume=31;citation_year=2003;citation_issue=22;citation_pmid=14602916;citation_doi=10.1093/nar/gkg859" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=I. Santi;citation_author=J. D. McKinney;citation_title=Chromosome organization and replisome dynamics in Mycobacterium smegmatis.;citation_pages=e01999-e14;citation_volume=6;citation_year=2015;citation_pmid=25691587;citation_doi=10.1128/mBio.01999-14" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. B. Schmid;citation_author=J. R. Roth;citation_title=Gene location affects expression level in Salmonella typhimurium.;citation_pages=2872-2875;citation_volume=169;citation_year=1987;citation_issue=6;citation_pmid=3294809;citation_doi=10.1128/jb.169.6.2872-2875.1987" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Y. Shanado;citation_author=K. Hanada;citation_author=H. Ikeda;citation_title=Suppression of gamma ray-induced illegitimate recombination in Escherichia coli by the DNA-binding protein H-NS.;citation_pages=242-248;citation_volume=265;citation_year=2001;citation_issue=2;citation_pmid=11361334;citation_doi=10.1007/s004380000399" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Sobetzko;citation_author=A. Travers;citation_author=G. Muskhelishvili;citation_title=Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle.;citation_volume=109;citation_year=2012;citation_pmid=22184251;citation_doi=10.1073/pnas.1108229109" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Stella;citation_author=D. Cascio;citation_author=R. C. Johnson;citation_title=The shape of the DNA minor groove directs binding by the DNA-bending protein Fis;citation_pages=814-826;citation_volume=24;citation_year=2010;citation_issue=8;citation_pmid=20395367;citation_doi=10.1101/gad.1900610" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet. TIG;citation_author=M. Suyama;citation_author=P. Bork;citation_title=Evolution of prokaryotic gene order: genome rearrangements in closely related species.;citation_pages=10-13;citation_volume=17;citation_year=2001;citation_pmid=11163906;citation_doi=10.1016/S0168-9525(00)02159-4" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol;citation_title=Evolution of gene order conservation in prokaryotes;citation_author=J. Tamames;citation_volume=2;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Tamas;citation_title=50 Million Years of Genomic Stasis in Endosymbiotic Bacteria;citation_pages=2376-2379;citation_volume=296;citation_year=2002;citation_issue=5577;citation_pmid=12089438;citation_doi=10.1126/science.1071278" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=K. Tamura;citation_author=M. Nei;citation_title=Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees;citation_pages=512-526;citation_volume=10;citation_year=1993;citation_issue=3;citation_pmid=8336541" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=G. Stecher;citation_author=D. Peterson;citation_author=A. Filipski;citation_author=S. Kumar;citation_title=MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0.;citation_volume=30;citation_year=2013;citation_pmid=24132122;citation_doi=10.1093/molbev/mst197" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. R. Tillier;citation_author=R. A. Collins;citation_title=Genome rearrangement by replication-directed translocation.;citation_pages=195-197;citation_volume=26;citation_year=2000;citation_issue=2;citation_pmid=11017076;citation_doi=10.1038/79918" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=A. E. Tupper;citation_author=T. A. Owen-Hughes;citation_author=D. W. Ussery;citation_author=D. S. Santos;citation_author=D. J. Ferguson;citation_title=The chromatin-associated protein H-NS alters DNA topology in vitro.;citation_pages=258-268;citation_volume=13;citation_year=1994;citation_pmid=8306968" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. A. Umbarger;citation_author=E. Toro;citation_author=M. A. Wright;citation_author=G. J. Porreca;citation_author=D. Baù;citation_title=The three-dimensional architecture of a bacterial genome and its alteration by genetic perturbation.;citation_pages=252-264;citation_volume=44;citation_year=2011;citation_issue=2;citation_pmid=22017872;citation_doi=10.1016/j.molcel.2011.09.010" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. Valens;citation_author=S. Penaud;citation_author=M. Rossignol;citation_author=F. Cornet;citation_author=F. Boccard;citation_title=Macrodomain organization of the Escherichia coli chromosome;citation_pages=4330-4341;citation_volume=23;citation_year=2004;citation_issue=21;citation_pmid=15470498;citation_doi=10.1038/sj.emboj.7600434" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. S. Vernikos;citation_author=J. Parkhill;citation_title=Interpolated variable order motifs for identification of horizontally acquired DNA: revisiting the Salmonella pathogenicity islands.;citation_volume=22;citation_year=2006;citation_pmid=16837528;citation_doi=10.1093/bioinformatics/btl369" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Vieira-Silva;citation_author=E. P. C. Rocha;citation_title=The systemic imprint of growth and its uses in ecological (meta)genomics.;citation_pages=e1000808-e1000808;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20090831;citation_doi=10.1371/journal.pgen.1000808" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_title=Exploring the mechanisms of genome-wide long-range interactions: interpreting chromosome organization.;citation_author=J. Wang;citation_author=X. Meng;citation_author=H. Chen;citation_author=C. Yuan;citation_author=X. Li;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Wang;citation_author=J. R. Moffitt;citation_author=G. T. Dempsey;citation_author=X. S. Xie;citation_author=X. Zhuang;citation_title=Characterization and development of photoactivatable fluorescent proteins for single-molecule-based superresolution imaging.;citation_volume=111;citation_year=2014;citation_pmid=24912163;citation_doi=10.1073/pnas.1406593111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Wang;citation_author=G.-W. Li;citation_author=C. Chen;citation_author=X. S. Xie;citation_author=X. Zhuang;citation_title=Chromosome Organization by a Nucleoid-Associated Protein in Live Bacteria;citation_pages=1445-1449;citation_volume=333;citation_year=2011;citation_issue=6048;citation_pmid=21903814;citation_doi=10.1126/science.1204697" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=X. Wang;citation_author=X. Liu;citation_author=C. Possoz;citation_author=D. J. Sherratt;citation_title=The two Escherichia coli chromosome arms locate to separate cell halves.;citation_pages=1727-1731;citation_volume=20;citation_year=2006;citation_issue=13;citation_pmid=16818605;citation_doi=10.1101/gad.388406" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=X. Wang;citation_author=O. W. Tang;citation_author=E. P. Riley;citation_author=D. Z. Rudner;citation_title=The SMC condensin complex is required for origin segregation in Bacillus subtilis.;citation_pages=287-292;citation_volume=24;citation_year=2014;citation_pmid=24440393;citation_doi=10.1016/j.cub.2013.11.050" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Wang;citation_author=P. Montero Llopis;citation_author=D. Z. Rudner;citation_title=Bacillus subtilis chromosome organization oscillates between two distinct patterns.;citation_volume=111;citation_year=2014;citation_pmid=25071173;citation_doi=10.1073/pnas.1407461111" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet. TIG;citation_author=X. Weng;citation_author=J. Xiao;citation_title=Spatial organization of transcription in bacterial cells.;citation_pages=287-297;citation_volume=30;citation_year=2014;citation_pmid=24862529;citation_doi=10.1016/j.tig.2014.04.008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biosyst.;citation_author=M. Zarei;citation_author=B. Sclavi;citation_author=M. Cosentino Lagomarsino;citation_title=Gene silencing and large-scale domain structure of the E. coli genome.;citation_pages=758;citation_volume=9;citation_year=2013;citation_pmid=23412141;citation_doi=10.1039/c3mb25364c" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="twitter:description" content="Genomes evolve not only in base sequence but also in terms of their architecture, defined by gene organization and chromosome topology. Whereas genome sequence data inform us about the changes in base sequences for a large variety of organisms, the study of chromosome topology is restricted to a few model organisms studied using microscopy and chromosome conformation capture techniques. Here, we exploit whole genome sequence data to study the link between gene organization and chromosome topology in bacteria. Using comparative genomics across ∼250 pairs of closely related bacteria we show that: (a) many organisms show a high degree of interreplichore translocations throughout the chromosome and not limited to the inversion-prone terminus ( ter ) or the origin of replication ( oriC ); (b) translocation maps may reflect chromosome topologies; and (c) symmetric interreplichore translocations do not disrupt the distance of a gene from oriC or affect gene expression states or strand biases in gene densities. In summary, we suggest that translocation maps might be a first line in defining a gross chromosome topology given a pair of closely related genome sequences." />
<meta name="og-title" property="og:title" content="Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/6/1597" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomes evolve not only in base sequence but also in terms of their architecture, defined by gene organization and chromosome topology. Whereas genome sequence data inform us about the changes in base sequences for a large variety of organisms, the study of chromosome topology is restricted to a few model organisms studied using microscopy and chromosome conformation capture techniques. Here, we exploit whole genome sequence data to study the link between gene organization and chromosome topology in bacteria. Using comparative genomics across ∼250 pairs of closely related bacteria we show that: (a) many organisms show a high degree of interreplichore translocations throughout the chromosome and not limited to the inversion-prone terminus ( ter ) or the origin of replication ( oriC ); (b) translocation maps may reflect chromosome topologies; and (c) symmetric interreplichore translocations do not disrupt the distance of a gene from oriC or affect gene expression states or strand biases in gene densities. In summary, we suggest that translocation maps might be a first line in defining a gross chromosome topology given a pair of closely related genome sequences." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
    <title>Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology? | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NsnblO_ty7WQGyKVSyZWwX_FmWOqeKyuWhYLOBT8hoA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460387","apath":"\/ggg\/6\/6\/1597.atom","pisa":"ggg;6\/6\/1597","ac":{"\/ggg\/6\/6\/1597.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/6\/1597.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1597"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1597"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/6\/1597.full":true,"\/content\/6\/6\/1597":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UMxUy4oXCffMoZxagoolcfju4EdQtJd1ZlAQopp+41iTKO0yBJuZccIRl6R5FzuvPgh2vgMk7a9YYVcO6OItiQfJ63W9GLLuL2P2+GxA09lz7xTV+K9w01ov9s\\\\\\\/P5K3rRj5sUGEAtjplZjh59eq3bG\\\\\\\/LOfFaswOziFsxsSBQF7rqit8awot8rStv4K2dr\\\\\\\/XZ3EGVl1dcsnp6qAPHtytRK23igwrdaz9L0tTAbJmgewYcR4cN6LKlIwuJAHmzWSTJzUZw7slwKauS1iAZ6wLaHGe5kAldvKIzKxZPqt42NI9MAEFs9ncjskUYohQPXYZ503rOup9NyzrCv86\\\\\\\/UbIUqLIHzQvNB64FBPdSKAQ1\\\\\\\/7TRnBn8l0mAPGVXE\\\\\\\/rAOGEZ1K5HvMQMHAz4syzsv81J2SdjhzYYnTr8N8lf+REHHrZBClngO5ZHp85qgEBoAdYym3pj23WFM1jOsRuVRfQtC6+yOuILTep5VOPaIosY5af45HhdNSSF\\\\\\\/l9qo0QHtmeTYoqqzLrwcli2iA1Hrqd1CNnisJZ5HuLipK9Nxqh7oJfF5jOYubhQ3IYcIl+OGF5JUl8TFDrqKVZ9H2SSaZeKNh15GuJM8vQ4ra6UG2yoJOPglg2idc3zN0U26r8BbXgzH+glpr7F8zWA8I7ktt0\\\\\\\/JsZ2saskZrVbACIgpKvAku3VgN6LHpit87tF2npHwJR6iFA+j5TjWfrwRebflfUdW8H6zQD0Pr6vqgxb6jw9k8GLsbUBQqwpxKvtPvqmwOQ7cy781+Caxyt2fM5HJN8Do87tdGPxrt9d1QurwNfCrSt6at1c\\\\\\\/qHMhvYaN+IQYHWW9QdGKi4CGz4EFABsE9Vs9euFEJMzHVyPUeX+jmwiRaCX+0d6UyO+pQAQFxembD+Km\\\\\\\/\\\\\\\/G6MduUwJH3VVn99kdV1cVWZ7meF2CdO8W1efxORQdyzv5\\\\\\\/mcqSmu9vTjdHg0HLzBtbT+0Wp3yChC9ZonpZrf3Kxk0+zz+jBOVUqJMImD94Dxz2DbjqfaydoafRfHTYFqg3ulyy410g7nmJlEkFfThMFKeUY4j8PNXfsjm9zpHBDSFsFZk33jrD0w9X+jS6bBCge9agjmq1WRE7yi6ubN\\\\\\\/OgyD7A0HPtUWnyy8+C1cYQlKjbAPNw9TjBbZXvVXGXi3Nj4OAnGJCuzoo6QWc8lK9UXJQS7VHIrUD7hygDYl3PbFOF4am2055Zd9QSaJ2Z1h5v0PHvH9Gmj0qYnOrdOd+hdxfpVithmL3a1ipKy93zge\\\\\\\/LNpLsHXbn3qGEG6\\\u0022,\\\u0022iv\\\u0022:\\\u002214ohMKfMz4GcUWxbxMA+vw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9916f291fe16756d422b5e54b7dd3288c6f1f1a9927a3f53071db9611bf25e7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yi\\\\\\\/dB4q7eNM5fciAjrRREPdxfeGL1goDhzYbF3+oOvg3oJpDVDrsg9e67kUScmBRqOYxOibz8X3oIg+tZAsymm5Fk8\\\\\\\/XmPF93mXVaNJ2vkTmABTgUQeOiiPy7rDBOId6iDY3u+7bHdHbZJcO83\\\\\\\/bIeWFCfeh1YST9GYRzJdFLiDYH8DD09rvORXylrZx+wirh5a8OuNx4my2xPWoVbxH9LFZXfoBrF8C0FWsuBAsEdSPcCpZ9x82IfyUqQGHBZB\\\\\\\/adqZ7gWyuCBgNkTuOdK1qWKSkDYPXuzVXVbEB7xUUoHc25qEIEIMKl8mKmwZ9gMy969lCVNYbKUlRj\\\\\\\/6QvSJPSm5cOxqSahcqRLp69IRzI1eOZb8NAEGdynumRO8B8fw84LX+JoGQGRryfsbn\\\\\\\/vSzxrf2eaiOwNSbYYCfLp8vi8VWE2YYwwrfFhHb1KPI0XFK9sD6PAkBBMhDNf755AeNWCqyUSLpyi44XNZPxCcv7Ri04XtyNS8Px3IDJ\\\\\\\/sDkPNUyPHRzmGeAFvfQIIcsNpCJe9qauVppDXYYfqyhsdWeE3O01Ga2avboDZZCjzcu0TDXi0IY2IaeEmZr0tdC1uTHKCMBAcKgRevkVUFJJawy4Q0YV0lcWzQHV4fo8oPA6OU3xmKfd+NnI5ZqlloVwoA7DELxbuU4Kt8vJUB5yNzuEvLRKnBTRcEKcdyii47LqHYZsZOjuE\\\\\\\/gd6dTDukFhUqYBa+Rkuvt\\\\\\\/5i5P7UsBfXt8g\\\\\\\/zOaVxw2rUtYGYYmfyYEjDug0iBFG2gz4\\\\\\\/sYq8DkYOlVsJn2VfQbuEpM+FU2VQsrh49TejPHJUnAk20WYlwxCTOk5ztX9mxttweXWd69n9LgBK7H2YOC7UbCg2fB+ndKFeskr52QWTBQJG0RHK9rBk\\\\\\\/ZgU\\\\\\\/BlDDVQGDgRTlteEXAKmEV+\\\\\\\/2QewnE3ofj3az+khcxuUUmqAJlVKLWIm82TRYBq6+Pp84cUjeB+7KydJMawsKZpG3M3qNjIFQABc+8Kn4KRibtpu2MJ9PFsePIOZ\\\\\\\/UvaHWynB2AOKD108NEpvF2p1PbnR6xg8vvFDP5ehRVMOqLJvbEXCiZMQNzT1X+I29mNv9\\\\\\\/JZ4Yb9OKwaof+roVv\\\\\\\/RNZtYRq\\\\\\\/6art8Dhd+UxHnT+C79xPhk+RM0WXr\\\\\\\/2d1+XdtqSNLstCB\\\\\\\/sZC1MxJhWq7je\\\\\\\/Ein690uGFv\\\\\\\/xvKea0fXWXpfRR+24ALHmtHtZCtiUdYGAVX1VJry3fDCJB+yTHCELV4W\\\\\\\/fk08k5ReDvvrl3C+J40L\\\\\\\/V2pRvdt97tikb2ev26AF0ku6LC4s9yrWqgEO4bmsvbHmVHthu1dYA2QHqVVrr2ASP6rdcECPyebdTOUvQA2M7vnlPj7koc3J69N7985pIiEZ2vll6\\\\\\\/LQ3sECiVt1+gf2iIDdWFl4nbAKQLFXvhS+VZ3n0u2c0BarsYchrZefY\\\\\\\/03HoCXKq5UsgYeN2Ca0z8JDt7D6ORSaJuW4hzWAF7yV8g5uzu1huKDv3VPLp\\\\\\\/P6h4dEC620dWDmsN7t93ew17+dYSBpmIdCabRvUrYueCqBBRNbDXIWRFnTvI0tzIaeYYmz3YIgkVc\\\\\\\/KVf+KFBTuLcI7sWO2rPDRhjL5ZmmqgmIxh6tStkBtE3e1hTCmaH+Dhg6YgUrmdgDpKXOLv2bdcfkb2aP6jNNRtPEgiZ7nNH8QMa9aEV7NjdgHcgQ5lkC+zD2yzy2MorPLGmXdQ\\\\\\\/JlMiir9X3WVCR62E0z6VUNrpqoSS3NzFyF1nRKUfUNR8OifU8NKTmWVF7XYuIxRT4vIjC9n1GjpECrRquT00kf3QHmQoKByaPcF\\\\\\\/Z0Z04gC5TuLvYAywXBQwDxmp1cuSS56nYML49FZJjFCVCG3SABclxUryJwymXS8RvxJ+5jc+5xJWkbvagSDOK9LUxCtA9kogNLMt9sIgUn3NRAlW1OzSkdutBi80fz9pxi6TeAYmPXqvqHkT6aPg5fkPMD+K6E84MABQQDOvmO0AdRLANt7StXSRyu5DrRYgUGiUJTZYmCiNLep00rtqoBJfijqkOizcOreO+oHqAjvilmwsvZKGnicBVDclUNzQ6GtgIS2lm2d\\\\\\\/mfGnTSnRDIJX8GTWiFC7+2Ozx7RQxTRHgWzkW0z8i63QcNaGbX+cdYp1vPBsYE2HK5lJmWSrzd0oZtqJ4hTzhrg+suBJ97jPSZKWqVSQ3UoMz\\\\\\\/AYy83bhMh7ZfeesYccYgMG1FPR0io9VSJZB1U3YKWe6gChZ\\\\\\\/PMCPxOiggjGjmlL5kGSQecCYowLbNSiaDSgGqIXeLzAfvNL9EX5XnPOunodW\\\\\\\/MaA5JCvm6kf8bLdzJhJwEKg8dYn\\\\\\\/vnzkGtas8Igw9C\\\\\\\/63CcWwoLDOyV4Wiwp70pbrnoYX\\\\\\\/l5k2gsF7fZRUOQUdnXYzkSFQpvdf1OjYF17EzGiHcyszObL58lxMbT2fTTVeumPnZMgf9N57jOkJUq+7lVrTTqkrRZ8VHVGlQ3RZI1VwJ9vDpT5SUiSd1tT5DRlQvCLuwgY0dTKMWjD8ftMMpsFKo3VM\\\\\\\/xgeIY7GhsvIIci8KO+FtJJQ2M4\\\\\\\/lpHmjB5HpkBBFqLVlYP18b+finWgiekiDRL5U2XpP1gTNwZo9moQGdVb79D2NS4+QsCRRCnx4\\\\\\\/1uXMjS+2k7opD5oXCqZCqoqdKC0F6029cAN8U6gs+g+xOLk0XtYGrgUQnW1xgMv3xnqfgX6GqJ5+AB85IcG0\\\\\\\/nwgYHOdfjBDYBcoFgotTG4aApd3URE+u5q23F8sYjx6SNCGxbRmd1KO+BpDk5OKVJWRFRZko4iRoK3Ds\\\\\\\/eKSkSs2bO9sfP4bTwPlrIOaEk3UDxnrrlbj6aDQs4Rr\\\\\\\/Z03VuSrL08EzBxAhOAkdKWZz4VY\\\\\\\/Qz8sIvYdNCsmzLfe3E2EJTQzbSnQVFz6YXHxxtnK6+iaRqGMMFhF2Az2t0PlEOKejkJe0UNqrhbYrrPuLh01ayJL8OruVd7tL9BLiG20nXQsPBRs8XwPoYAUYclUArfZdCi1fuhP8Cff51iZ6nnB46M6B\\\\\\\/gY8KyJinjnt8HxHZSbJqiUtN\\\\\\\/KCAxlH9nB5Q8J8PRXH1VSdNtcanMvaEnc92fi0mf9v+ASs7TWvS5yqtZ00uOA8poQSG9\\\\\\\/0ZywgIP0U8yuje3jUhoIY4Flrzmr5HOVXSJcddiq9EBYe5HE3AHvNWn4RxAbh+\\\\\\\/7psmGzEmHZcUqbt6TLkpb\\\\\\\/+SS8qZt3F\\\\\\\/ZqMsW5MImUMW9IVZv6+OVR+AWt4KQ6ipLsg+uygBjcVp+18mvCYCt0Ghq0scG+uT7WKeiheA5W1VAbXvqPsRP\\\\\\\/N+tJW5P+FEl1B38ViXnP0xzfEB7FASTtXzU6h7nxc\\\\\\\/UuYj\\\\\\\/lx+F9eL1GHs1wcspBO0h0wKelk7TNyVHYrBMWBedYTy4OPB80CGumZZ3GSmfOckiP1WZKb4bnWkquts39xSw4yoCkxvQBQAoFCExKfheL0Y6g8cUNv5uTGJn8ElaSsvcIFOFdHdsIkp9LHdZmkAopAASfEAPE4WPSFjrKfuyWFhKqdSpPRxzgcE7UpwNiL2B63fA\\\\\\\/v7EHcH7+zIwRssH0iANX1BwFjLuXon6OJYeY7n+q6TFek44GrOtjdcXoTWUjMwT7vvXdsLFZkgpHErdwth9gl72tdf37AMVRO76nLKJIlHB+vnoj9Y+NETUZISEVsig4OWbJtpf5qL+TvjkTnTZn1rFtkI3I1QlD9dGCpk+sWD5nHNWFYHicL45vbi29MLVYz9tefyfjJW2KEiNP+4IDKsEjJnb2H5HIgWI5Tt7l1FkKiSf1OK4reZ61qbFjjtduvZi+Fm7izuvj3nYSHapFom1\\\\\\\/T1TZwozFVEV7eJJ4\\\\\\\/8nUkQBPqjjK4vuHNIU\\\\\\\/LmEKTqg3diaoAR+\\\\\\\/9jvZ1BTGcIVDf2ReseWqSq1XZfyQjSSrwmpvhlo5SaZ3sA0mVI23dE9Go+mrqImw9MOY+kp9j1I0+mN8XkFL7o5xxj1wWkNM8lwItorvvIfMyocFf3WsZPIrPODE8Jwd7n3kMUEVVgodW8rWinTZwt7xBwtTzQtLvs8AC+Pse486\\\\\\\/EoirKKwMH9SZzTNdKXDlcnBPgefMJTJ\\\\\\\/hVimcJ1pFqa8S6x8ED9UBgRSYg8qLL65dJmzeCCG2zIHBmE60D1DmXgqr7BR351hnsxozZ+ZC4cUZ8X5KzYmORwkatcylz4gfpX07+X5\\\\\\\/\\\\\\\/\\\\\\\/vhkinHT\\\\\\\/iu5OwfqHAPBr5Y9LjSU4KZJjiNJClIvPh\\\\\\\/b284wgvERQpnqRxA9v125DxivZNBrF9+fL27Jvvs7LMjHKhUAInWcyzVGr7VjvqP\\\\\\\/77eNXw41hwn1ACjMVzb8aZe\\\\\\\/ZI1TjMIhnXUG459DXjC\\\\\\\/1UlilxxiF57T+fqjdIvJeNpfvPRMSaeQzCeObJPw3MPmFi3ezXMTTMJSWiAkad4VxcL+pTt7n9AI8q1MHAJ+qgpDbtSIqk0ZDdRmlU3HT7+g\\\\\\\/bMYtDeGXlm0cw4oAw5gUtYpPp6BuUa5O\\\\\\\/kEJDNv7\\\\\\\/LJ0yxROLyrHCTuAqUNiobUUdQTskQJWMmnKqstkL7xqgz+FoA8UkmnUgLhVQLV5wvYoijptJjDdu8N\\\\\\\/02Hmu8JX9ey84eGyc924l7AbpUydmpRHgbSeGl2ctmQDoOQ0prL8bWE4RHwRWLUIsh6p+kXBzwIEsn0kaXacbyDxy1UMu8tgOL3TLdBrkE9PGUvqG9DIgUJTFNAR43Eh4Q8Ct5ootxladm7y31JjBNB+wbrA0pXYnK9qYtFWzIvoYWsi5Xx\\\\\\\/SxVKktHCxmgZB3SX1FOap7pb6KsoNzqkZZR5Ku1wMvmLsPrITwRIGJL4zwBQ5FK7IMVkKKEkt8k5xq9piUXFHHxckayPzdu0o3k1Vc96+Q4qUFxt1R6E4LCeHOwibCWpURNMNTweiXDq1X2BLCinhGAYbaaxAlqz9t6xRRhEPVznD2SBe\\\\\\\/mP73VBQrNBoevSsl5ppyQ\\\\\\\/S2RZLRMlXut\\\\\\\/\\\\\\\/7NWqRtlOgTe5rH\\\\\\\/WTzqtvtJLIShs\\\\\\\/3Td5wxolbGFrIKphR5VCfbodT6WOPLRd85bMEysJb8edVFPGiykSm0KgDaLD84uNy8i3pmo0sOzlIY9KCAqLPICLXeT1I4xW3XMcKGNBs55zbslHAGWZgu2ovSjKkJn2JkHtoNmLnH49GnlQBa0nDUsHCxTFg5ngJVuYJyQ4kQUqs75R6JijXdLBy8TJgTUJjljM4jfxOQ+X+LNjx\\\\\\\/fyygh5PiE3XnClubfBC1vaC5\\\\\\\/\\\\\\\/1X4BhlSs60UDlktlj215LsPatKh3W3klvCbCPXc399KGoiMub3m7EiZJsayYD6qf708vBtFTC\\\\\\\/tgBXZYM9P98cBAwlCGCSWw1Q5KQBoNs8a8Jl7zySx3diQGbQZQj\\\\\\\/LClK0EabjzgIG5fHHB6Epr31ysG5jqzfwyC2GM1IuXeSFJWBKFjtcM5wVVZrFrqZub8fU0emEqbs11upmdIkV\\\\\\\/nT3JdhChcC5v3dRHwKxnoCk2jkty0O+gIDaLJXdViX8PWK3GHVDlk0OVu\\\\\\\/gjYlWOhNzxsswChwEcJt7UVKQce3Jq5baQ4gSegZ\\\\\\\/nqEoPiMjWvZhg81Rlkhb3PrFc5kaaut0RVwWAl2cYOidgRaFBCXatvdWWc4raxZkoZS7zYxYxp4vfWZsj\\\\\\\/fSUL1B\\\\\\\/v2DNOAl6LwdPU+2e+V9FF3bcFdFbapGE6qIy2rNGkSZPcEzxmQ9A\\\\\\\/XdZ2ClNo19t3s7hDIeXnPRGx0nYcAt2adTeWQvTpzZuZZ7fe6R2Jm5gBNYyphXVmdQhiclEexY7q7ibvORb9qka09ihZPNMDg7pt3QBV7IhxS9zfBAKomd19vkfMK6oySooPQlzCO4JDBq5HkikukRLNCj7+72RmajeaIQViVcB2zHyPoIOCvf13+4+bHUSNG52HRiRq6cyS2ZHO7Gn6GlDYUN5pkQ4c62HrAiK7aSCrW0dPssop3lFGAlzXErhD3U0T55\\\\\\\/o0RL+MEdUNHLvMg12v\\\\\\\/3AaDxbPdhBu4cZFgEsmL3xPjqz4A3hglZbzMbgLW5HQ\\\\\\\/HFfNldB+N4MLihkxu5tRG9XB+t9nRlvmbe2GuOzzMkgoRx+aDG4Q3nDUb1u5UQzpXTCsmW8iVEr+p0gpcfBQ6SW31vC\\\\\\\/H6y9nWPTF0\\\\\\\/4Nz5RUgJSbQkaw38p4N1tsnR5x7mDpKjj0hJHja7w14ArdCKdhtyZeHebwUnU1Hkh5Jplh9xC8HmwzN02MU3zJ628Fcp7Unv2O2JUD\\\\\\\/D4\\\\\\\/lli14oMesw\\\\\\\/Lr2u3ThoS2q3jbZJW\\\\\\\/b6rkDxUjRZBM+K770GbvyTAqcvm6lOZO2Iw\\\\\\\/4RNvLalxlKOJeEQ7Frd9nSYS2eG+oExLfzysll9W1SfxhNogOpFfzLzX2FNhFCcO85ygsziOF4Hjz5yyP7g8Gcppw2o99EvWl1nf7VLbkZrdUH\\\\\\\/sHJFKXk8fOWVCB8i4ZNSpSZF\\\\\\\/XiYQFWD1tcyBkZadfXylinSCdXGaf5K3Rfe1YObjJnHFPPPv+wC8VkHKFNjTC7CVKCRpKWLfFbwbB4u9s4n7NKWHtnpDxi09zYwDEV0cbb74Zkrr1TF2rlrjbpMtFaUoy3dZvJS9rMQJ7PUMJlnB8XM0XWa5oRLax1ofz3zP3etBSvGXuy7uTjdj\\\\\\\/U1BPXiuJcQZWva722Tx+3BvUo\\\\\\\/hXesyHMWYSvrn+HWv337qpuJ4KxRU0eLy\\\\\\\/n4tfNml\\\\\\\/iz2ya36zAwt23wzttahydC9z8kLKkIxrv2x8GAFc+VlH6Tr7Uwv6u2f4KV+9zGCemA4GyEITvNkBIn\\\\\\\/\\\\\\\/Uhwt6toDd+jYiOobNME0iJv7kwjZvXNXWmLnJ3yYUBN56CoL2J3IDDHDGMDBbDO1d6ukIadpmBpikGzUgIzA2oSPEWCpRt75OsiEUuvdpQpM6lEZCYbEbWEIj+2LBj1smqVQy56NYxwNRWEo4qry+dFfl0fWrkWkmFL1Fxxs\\\\\\\/kCbTJ5+P6WDkbae0gslcb0\\\\\\\/fYhnoIum4OSN6TlzICJR6KKiQrVhkrCJ+5mqyAu1UeG9Bn1u\\\\\\\/8U3MTQDD9t4\\\\\\\/4Tq37LarF7Ll6OzkCyJnNyXqdJ9PRRuaQIOWl9Wu21g4MjeJCRVvLHtTXmMVUP86f8sX5aARoHFeuyioUbswXlvh2bgk6tmErEUUHzex2Ym27itik5QBb4CsX6iF8T0KfP5UTuoENSAoD6yUArKiZOk4SMzJWlaUhB6fZmnFF92\\\\\\\/f9gM77SwSNwtxROA2eaSfSJRn4rshFowjutzI\\\\\\\/pesNgFSy2yzcK3TRXid2Gb+KVf42Mm2\\\\\\\/MnPgrdLQlf0NcdWPvMQljEhwmRQ0sJB3qLn6+ZqWus0XsGw6aHX3+EJe1PJRugLqk+ko6DRnz7TyXfzOvYyh1negEjXIRWJI6nfcGOvXAYFQ8mPBzBWnqAqsKF3191I17nHtRhiTEa6EPtYq8egN3T1wzBi2e04pVO087d9kfFwAKuwVroMs5PdHBrbkF+CQdd0jSGVsL\\\\\\\/6dHqBXt12Z0TC+GeJM\\\\\\\/JOvM82PYxIzhmi4ZbhBxN7FR7mdZct2BF7kaC16jR12q1cK+zWsEf3\\\\\\\/seVcBeW6d8fFdg5ikBDimUHkmwgbORwqQaNAMSW36dqu7eeLMcycbUyhB4+\\\\\\\/drcVaFqgrbyFOPuTKjfdlQkimhPCP2Uwt8QDWhoXG5AN2tFJ0bNhXlr4fa+N\\\\\\\/UEuou04ATVb4lqhxqjuRQZ22V7ox\\\\\\\/7HPCnFyYH2vaOCOJHSftmpDKmgOofIrAOeQYisaFh04EzTSzwNaGoLLTEfT82xS5GzBPOr50\\\\\\\/Ho700jACZVGKUtZr6ZZ76XexN\\\\\\\/tOezGOF0IrudLPGhlJhJWyjeqOryUizfwuVETgNJ6jX+\\\\\\\/dH34quHg1a4l\\\\\\\/tX7lh2GDAUDmIcPBUU5oQ+QbBs8YAalaWbV+px+LgDPkcVndglNXMYSS9kcFP6KyH3rKy0XqBcc4XxRbcHIG\\\\\\\/8aur0nm8lyRMWS87ePc+qCHF3I1Y7RmZbF8i4lEDPaJnK4YjGiB9npXlU\\\\\\\/pNsUBWavsHVjs9upWxKKpUzlK1yJLsbFXV5Jfxu9n1p3NSUGqdC\\\\\\\/aF222fEVUxxROC2oe+Fx322IRh6u2DNF26pmMBSbC4voWR+ZzeLMicQv1TuAjT5oHm7NQ8aREJMo0NPhZ2Iy0MswwUxNaOQkVZIUHOMqVO2iQKaLDDhAlWitZ04dC7jLAxD+1elNxhb+5W5RK3DQmwE83+0Fm0qAiEnhcxXqUrDE1QM0G1NIv5nWIUBSOXzh7nEDJHT4w4aYOGWvtTOy4sFr6\\\\\\\/BdW2P3AI70BZmmAx7IfQQH4wBMPXUZAYeupeWby5hpz+\\\\\\\/tYtJdBjxJqrF0FHqSdr7oH1lXru8rPBI\\\\\\\/fX7PgPRR8ZFOxJw\\\\\\\/uIISr+NFilGyg8kExHD4073AQNaQLWM\\\\\\\/5eCQGp1vNXC9EX\\\\\\\/Q8beuezx5c2wRiMdUKepOMPtWOKpsEaN3K63OoLu45qg+ohIgAnUXQtgj5A6vC+ICfDIlW6WFUd1viWNaiyv3b9BeiGEQffSgzSPjemIgjvxHOlJDkTDR0AOVE006KT+rbPJRRL9xsOtXtE1seAl6EVruXlqwxt9rP6Ahf7mZz7Be9t5+K6kNVwgRnortyjYm4b41ZNymUQ3QnnqR99Vdjgw80u\\\\\\\/xy5xW4siljfjdmTT72xEbcMGzWCHH9Yq1lUk+h3\\\\\\\/jJm\\\\\\\/N31Om2k6g1oXe+wW\\\\\\\/SJUi46xfYAJ6mfcguWMnwKIzcSv+CQrLw5XAhMChn1uo+bWsz5MpEynszZGOYQVpOJilrpNvJiMUdaCR4ogs2FXbnBgzoJF3UjqzE\\\\\\\/YXoKIXUtoExDvuGsTYTqM0FiG4uthrWDf0JC8Tplo4YLkMxgCtwZc2T6NFay+tRheC11EF8ufeg4LHGZzAkDsAl\\\\\\\/e32AlmLi4EvZqAO\\\\\\\/PnTdzm43RfAv\\\\\\\/63JFcRId8z9R8oLCEyCkhM35hYbIWinykk0PCVqgGKbmBbJnZ\\\\\\\/WJUK2NV1SZFO5dV7UuEbLMeS3oin639+OMClAbxty090mlvRam+WVRLqL0iJauk+pwtjFte\\\\\\\/Z1oy6iVaYBLCE7pOUBunEbLauoDYx5mh7XMUm0w4Sw6SgPlcbCoeGrYlLKYgxCUmDThHa+QSjKLVLdX3RerNoKZ\\\\\\\/6XB8Mw2nij9HuMKxin8hOPw0uA3k+XapHuJiZSi7jJi\\\\\\\/Lh3SzAn2TNJPZUZS5uYKq4wxfKoe+khQhJG74\\\\\\\/yzc7FRsEQOdWrNA7R9+emOydTrDC0+A0vVb0cMFbOx7JlC33DIhBV\\\\\\\/CBADu25eqV\\\\\\\/L6o6eqM43Zo2l7YPREhbmXyZBXZ10944A3jNpWFc6uR8b241ldAaFge\\\\\\\/10+rgtvlMuAfWJ4HzFOzzj5xT0ycIoWoxYIZMzMcloJdUH0xtC0PP\\\\\\\/s1bc5B43Pz5n4APnmo\\\\\\\/a9+ws6h0xTGZUBisPdmELLPrkZ4bJjgpaTQcYm0z3k+dH\\\\\\\/rGmxYSUwz3AP+sJUvanl8qJAfxM5wAr3slKu8zP+yVDvOU5p+SlU4cvUjO5+keN0ng+thEikFvI0amlU+seTe1tx9dNo06uS5wYoRXEgoXuS+qVcfrM2OoUaH2\\\\\\\/58VdKTKMJxiAvvzGpRFxRGUpJuuDgDnlYcclUx6dm1G1pBMLZooMRfp1F41MxeNawNLEkTvtlgB4xX\\\\\\\/\\\\\\\/PAHh04RZlJak98DS1FbiEI0GLVcJDPRP\\\\\\\/dYziTASF5RKBVU3bPDt5CKJ51PUEPpO8bSp5bEQkKGr34Nkp8sor97ek90WuJxgvG39l4TEJYVRVyMobhilsY1oMWL+ymN0S\\\\\\\/IS1bQquYoHCfzKpvPMZpu5SijYPfZyMWD5A5OuMryTf7qXT\\\\\\\/o6cwFisssrmepmHGj+PSfgKtEYjoh8d97uFSiSwBS5cByv8JMdUnvYAQeNZh7U7S0LS+ogcqIBVhiiCu9X63jOGwrUTuRixCVxwP3aGBxcpck7sJsK+NUgXjGsMaRFSzBUErEcDGSco727SReaNQDsN727rTht9LIRhe953ekGqxe\\\\\\\/I2x6ONNeRsUQWy\\\\\\\/+1TmGZ8Ttw6oK6JiAQXvLpkDTD58HvdiacFENE9OIiVHQIcg4jgtPSpAGopopZLEZBD\\\\\\\/R5wJJZyaxGeQHZeqRM6ZeW1+jxzOZUXVmy+8j6x7Odc3AvXuVcJCjWDc2uDDrBG+w24bDs+8ic47Id9c+lfODSi2Wbvk9ImlXu8AdLkOTa+KTnNYB+CvJ6N+0pvKkcJT5XTqab157aG3LFHbrs9CXx0ey4Jsi\\\\\\\/Lxug7jFcBLX9w4YrKyaLtR+7JOgnk6NqtppvSQNSY8UKTGYvefTV3DwNa3QVlRFTPmhFGgQQ3d5+CTyXZpKGrcypeeYd486rFH1U1Xktbuhf6vHmfb1REzYgZaH6VRdq+NCmTtAQerY49VKM0HEaG9r2YBWI6upuh4UO5MxGLCeEuFRWsKntUOY5Ge88NW3lrLQdlsW2m9eUIz\\\\\\\/XmPh\\\\\\\/yKy42ZUl4tbw\\\\\\\/9K\\\\\\\/bhap+xZ9hnnyFFTzx69GCvZXqOTU9ZshSLXXGhf0a6Y0udaquJ7v2BF\\\\\\\/jl75+G8X3vl+2bGeetqmw9+AVx1QUlZ6nXz0hCafvdiuoZWj5jJTSEKcvGNd+mc2Efc7iBOQyIXn2eN6PWFJKJ7JwgILdU4x8Ru1I4da51zObxp3x\\\\\\\/L8AhT4w3CeQWaqlqvKCTc6jYxc98a\\\\\\\/+OrU+mTTdbd9BJR7dO61RDj2NzADeKOsd61DRyc7e6MtTTSn\\\\\\\/idRk4GHELXHzo0BvDknoE3H3TBIIuf9o213PAFZ2plR6EBUUeSXmeLUfc9iVJW7TjZsSq1IE9aLME4JwjXfBNpszwkhUttrv1xv91bW9+EnjuH6kg6Q1mp5pOcZqcwm9ZwjpgBZU1jgMVYfifbbeT1kQADP\\\\\\\/MPfYjStWaMl84YQ7z4Efa6dDdaaCHvTv4KVcFZluBRjst\\\\\\\/aZqNZe8t24VpgDbaP\\\\\\\/852OtK0YU6IFG2YZSkmXanruSn+y9Rpe8P0qS+3BDHCSe9lumdzbH1pRh1HloxeDxX+gQrEHj5EV7H3ZlDwNxLtLM7XdAdDGl0rYZj8DHwiIeQ0Tklzp2Etje9MJned0Befbj1tjN2lU\\\\\\\/TQ9byx0JdL9B3sjvwUiTqEC7Rc6m+Jctq066ADp\\\\\\\/rUm42+X7sgz7d15U2yBjR8+uRvfSFL\\\\\\\/BgTVL0tEgMYZR+J+4AgMVFns2993UmOAGvBgWYULO4n0CuMcM0m0pbEPp5VFvbWiQSrOkADnMgTVasyXKF41qBHuAIHwa2h2NczCgQRNlZsy8ombu1U4G3rgK3hGBkXJFuXX4hxnAW5IK8lo83bRur01pS7gN0rWYnqMBXRWokWcKkmkCnU445cQvyhsLL8F2XXPlnC5OPikVzUS9BBO2\\\\\\\/ribpsWmDqGIlonA+81ByNKQ2R8VNUTBIxP\\\\\\\/IrPSyE4ISxfT+wIebG7IEibbHTIW4PBCkPgTtN0Bj0taelpHM+PB09DVYoWIDQuP307kojyOqZEiQPl9TnmHx9mINin1U8M35eOgnNCuD635LfBiSZmY5BcvvkMbLfoKcla4oX4E2g1lmKkPHBHtI0yYEPCIfkun68uUguOcLEewiA1eowu8E9LftsJW5kIPJfz0fGy7ZBIv19GPT8JbOqr74z1elVDqQw62iAe6jFeB+\\\\\\\/Gvw1YpKZYadk+KeNOS96G3wuX8vZkHgnMdpk8YKRWsuQPFYS24dyDzjcxw+fMIiG6g3w\\\\\\\/G+W2PAOgqgEnU8S9Y4OKiv0BVRhkaCRBA8NpEiFveeYV\\\\\\\/hBtlemQplu3NMh4\\\\\\\/REqJuhQwXBui6e7JOyuxStMpdfE84WpUJ0nE89VHZfJyGNVeVBPwHC+qlvVXRdM7terwWythlt2eshihesoxek1xbfU8vGr6Qy8XilvplCcMm4GnJviqf5k3XOO52lRBoH9ntwc9LRtBiUF1NRoFLlW0XivzXmJfG5ZmCTCgVCGgQszQ3uu\\\\\\\/27ur039mTn0DiPBXmbt5Ky034CV2Ly6RXKZ4fvIkniomYvKKs4IEp2Cjy7TtqMo943FNfKyKwdkQS2zEvxkseGS\\\\\\\/xintglWCN3ZR86x3eyNC7wCW45K6Uir6rnUtmy01oLo4s5JQPg\\\\\\\/cr16DOAKWR7vIsK4SGas48VI69n2tbwiOsZoYLfh5QJbmmWhKI3oU085ieHwOQYDwNM9rKHdhTtk9TJrFzjHpjFpX9m2rL6h2TSsfgk+UsswGQgWmHA\\\\\\\/nSLIbjlEHU\\\\\\\/y8yqSYcmevorpof4h44Gicy0BbJsGqReE2WsCa+2\\\\\\\/KO34XsA+Og5d24AG7KEHO+NOrCBMq0+xZsg0lOyqGXCJFwT2mrAi\\\\\\\/Scy2jIlNJbfy3r5YzdK9kGiCvs4LhscgXu9oe4JtNwWTuSXj17WxWW6kCIFEXPa16cIWvn9wdzFVy50a34KnMl0p+J9rQFBHyQRnfsJgrsOzZnQyvprwuJkLiwL0pcnbbdstc1Xs1jpB\\\\\\\/w70fiqT1At2SbbLiKkm\\\\\\\/QaGKuf27+2DGxCWJKondZwWUpAWr4Z\\\\\\\/XG9kLfRTi6UCtNQ\\\\\\\/ybqx42OTjDdMsFZO+rYocO27OJowvXGb7swNDlBg4hnpPxb+TXjF0T927EAIi9Lo0SykjXsx6MNbOu4h1Nv\\\\\\\/86TWMQit\\\\\\\/ZamGNBHzyZ0SnKGU+IerXMWiKsKMfmwbGHAulxz6m2vEhUgRyJ+jm5Iayn49iWDikNDL6z14fYZXZJk37expvSODNCXyLco9sq6vXlukVS1qeOw8N8WA1jyQ7+V6nRUHXC6xAZS9Hf1UR1dKPbt8aYvXqs3v++ZbgGXwvYpC\\\\\\\/uYl60O2Wp1fPl63OKpaKD3pUIAGiMkQt5v4l5Im5IDjJQwBbL7h1r6ipe2jdIB7TGhYFT9C+nbrJTqkz5+ZuuRgl1TyqzkdSpv9rouYUuLpiHz7dMFUiPKYqZj\\\\\\\/vn5yQBKH880Z9BtgfydEhNjUpVdnfcpkI94zcNIYDvGohEFytA0WhL57ezc3RzPGIv+BYU7+jeVOuekIRKcoRvxmLRlx\\\\\\\/chmc5LA86L55zUptsxCc5PoSYXhAi2KawwR31YRpK4lOz9c2fiX4MR7JbTSEX7IOnGfn0psF3LmKpHVioJIOvLZQEZf1KwtqRK8fFiTHwEd15O4n0zRdkK4icmthbx77CYa+SHjcn1VB3s51nn9ZyDNUeC6uLvEGthwvpB+HP6Z4i\\\\\\\/zpS2ckQTpBy07TZR908MZwRnsOfwec5hrDCAL\\\\\\\/qG2E1lk1vRuF4wxf9SFg3dI2958DyLzqzkC8+MvHt7P\\\\\\\/xso2dQLBag4ZcS7AATho0txsrsbt9KF3gfcXqp17jOvtqXrwfaScVDWEwu9VOmWnuc53ELNNK4Ucqpffp8AnViUanrG+7sFCMMuO22aNCYhKLSHUNYCzmCBa07EjXGVTMVa8+0nM583mzkCHcfd59kXj\\\\\\\/q2t7m17wn3itVfvURntKH8gtlgx84MlYjwxs4YMy7hOX\\\\\\\/+4VSV\\\\\\\/+u2KooH6mJ1r2wnWiQyqqiIyUk6OCI+A9C2wDUEPiBXF4gME+gBORuKgGe1DUKdOL5AVJIkZvlz7FZfhQuRNbqC5aimL1sjrA81XQ91QzNhegGAmZspoy4plcgBNrRv\\\\\\\/R1r6hD1DQkTHCzGhhZskfchg3AMKUO5mVJgHZMZpPaaPiOeJBaZTOui8qgTjjs+Gevd3\\\\\\\/20CA9j13jFPjLOrefFYyCZlOlrEAsNbpjjpKoOk0PPND5a9kPDmzQMY3BomleYbY2rsQU0H5cLLiFWHY9achchA+og1smqq80r4Ssr8RF2lb4GQ2nokS0QiDnar8sdY4Y6s9okotR+x69UwnAXzML3kYTgFqdnq5LPJ+sdYy\\\\\\\/5875XvCFd8jzcBsAwuixIsPd\\\\\\\/AY74b3BKdckh1Xx+zI9qdA8oH1dsjCBhBBSgwZ3KaroETn9mePh8ueHv7AYVoA2nN\\\\\\\/kuEQj6vPGcaYBK7sBpnihY2tesaJZR\\\\\\\/737JwjwNYxj0L9oht79UZrDNvZD9897peCNJQ0VqyBTJnFvi1I+N4sfxA2axsFFs1HN0yjG8MM6t93CReJPqxhQgzzM6hPj5ah4cDkWcB\\\\\\\/kd7+wTz03gja2ip\\\\\\\/1LmjjLFh5DYJuGZBouKoB053H+bZRj1YTjxOwWbXYfc5AiTXMFDFqNzrahn\\\\\\\/NpxPf\\\\\\\/v69ad8jMklx4Cr+TshAHpRg3wEvXoe7+DGVzHVC+vihQkLbBwyCUYA+ldDVhhi6sHkGXJLZhB19U04JkqrnX8BpwXfZJdpQisFWv8cerspss8S99\\\\\\\/gZ0c9N\\\\\\\/Xa3xLe+C1ocesF9c4RG1y5pYbUiBGjmVNBV+WHknUAikQLipDSC1HXQIFNNwdyoF4mGml4uTnv2LwbG27nHOVWXi0439pe32\\\\\\\/IFMPS0bNojKeZu2wBypJieZs\\\\\\\/irC69xLtjQNK77tgNG5M5l5b4pZ+EqcRYO2zeKuJo3iaTuXU3PMdbXjZ7tGVrO8hdccw\\\\\\\/hxkMOH+J9vtLtsUjJyycwBb6guCgzvjpU6opA7r6wfLUOfsyQrX0+3wq5uGNz7pSvwZmUjos7J3jBCB9xhmzcvnhQxZGdo2s+pC5FPjv+q8\\\\\\\/eAInpUUoggx1wNsi56lmcLLO5ADUcwy2pBbDdBc2aHRs2OjyDWYv\\\\\\\/j\\\\\\\/klFlQ65OJ4E07ffbgll3XfqH3\\\\\\\/eksIBor0KoqGgYmwPO1AtTinVLxr+u3Ni6SeX836v+U0UXoz+irA\\\\\\\/YZjRMaVnjISiVhS9ZDd3nLiEpIOE31YUbGiTO3DBOdN8wG65KPKLEXmI\\\\\\\/v1A1AscccdakVeEik2tBzhT9RWB0209ZRLInLe7yPZbLE3UDfB0X1ZaTheIwSszr3N1yK63o3xz+4cbxXN5hOiiAcw5xopPgiRnxZ0LKy56GYMemJhEjtD2qiJIttLfS\\\\\\\/VaKFD2g8AIrijpZgS7Y4TUdwBZ3SWSvSj5ahG67La\\\\\\\/xV643qzfp7y0P24zbSmQNXKq6d+WV78xIwE44v+9CtNh4KQmz1Mzz\\\\\\\/GAqQJ\\\\\\\/vexfYGxzgf4pM+VegtGWxl1KUwdVBlx2ZzyAybvOD4cYRAno4wqcan2T\\\\\\\/xbkG0ajNeRnwwRBXWLCDpS9M\\\\\\\/9ca8HJD46dgjrzeMyYpfsk68iYigFMN8x3cs2OIZox6I0YVFGtEYWhRtFFPXS52LWWUEixzyuNAF\\\\\\\/kDAiAniKokHsIX4IAQP7NFaalxZ3q7Dn\\\\\\\/kj11Qi6cmbIHDs6xlYXfyR\\\\\\\/YxE9QdcN3rwMjcyRbVZu1g2pfDqy6c7K7nfn2mOoeCI+emeKCGJXj9Fe+kqxQ0NtIym2Fa1aZO8J0jEcQh91kYVV3p2a6sG24MLrF7j2m\\\\\\\/3Wb7IJFLmk4fuSuO47za82o4sSVbsGJrA2fFwFLn0evcwWUHVDkLbJ4l91Vd00JZJRks+1rvm7QcQ5D2ED4s+7xhpwdkEeHC5WwjZuakDGKpp9Xa8OvcP5988NOIPKcbxUJapiQLzrDOagdFi6eortTNZsUEoEojHxpfXtRRXU2bF7n9ntzinPGVew\\\\\\\/RMplRc7Wat0HvCAUEQtFleIp7XlLQvpxW4RtzTlEEF2fTyQ1yNc7ragAQMTW6JTDUFHkCPdQUwbEvpY4\\\\\\\/74Jkga+ftkxsxupPNfQ5CBLDhBFlRPYXJsiSQXEMKg0ZPgZswYWy+zpkctmsEcGNQ3CSexcAcNgM5oDeMoYZIqcxCC+nqK3cdftqEL5osHq65UAm5bMGcVGGgj9m8MKEP63+PUp41ApKG8vtUwReLrTxkBt2eFS1MA3Wa+qq8Inp+r2Dm6Y6AIkrRphL9+JtIJHHioBM6pbNEi2RvQLPtLJiW49kh7vIveUAqfjJS7LZ4vUIyIwVsWdY\\\\\\\/7XdhUikmfXxpdzwFhYDGSUfaazkN9otpkrPVGiwgjX1K6zurprvf8u+fMsXoJPmxSZomtkTweSGri1U3addZg6\\\\\\\/AeBIyuwLVuXeB9fJAtd2n63Y2I4hHrvCrATtMrL1eOBCnOazl5H+zWvre9txpiKdhHZk8C8Ih0OF+9x37\\\\\\\/5hgh9FNEzApoBSPHbIRBMEaa8F5S9tRuomXYG3Q+cgkFA0vJAqEQGj1tkZaOjrYT6sCtcbye\\\\\\\/X+mS5UWqbCQldb3k4XNGN0vpidUmQk4FtZY5o066OairnnA4oAyMbAf+dGR1KNuQMC3QSpsXfLXnqGqkUzWQKBsytEQdo\\\\\\\/6ayv5ZduB0haDMMQdmzK4K9q8S\\\\\\\/9hauRE1ss3vu8k0j7MRCfdO+26WYm0LNpTXpu9Cr3QGozo+ruQ+QEJPuhpoLqNOEKU5rV1eOhwrkfAuaOM31BAzTR9w9HHhLwAz3cdDuePXVknBFQCvDB9rBL6gMBCHmdjZx1Ji6zE+XKL6YdESINiQ\\\\\\\/B7k699bgQu3pzCdr3fOpIbpaDzoynlDVpDh7LMC\\\\\\\/9d+HhY4M+MDgqtD\\\\\\\/Var03dgiiLG5fSltx3eHJmha9\\\\\\\/PqNvSdkaXA+twPY2gTtiO5r9al6yONL5x5XG2LivwlcYopkI3d+7DVY\\\\\\\/pCuCMKQJv0QddSSc9OT4GLJapyCnWFaSFkXI7N45d5b+O4q7Xf1w7sjGiAgOxoRkM+nXAAcq+LIJSPTikK5JoYe1Bm\\\\\\\/r02SABJ3eu+t2+QOE53DIPZGPOW8ph5EW2E4H3f6x7ot5RUIbwZtwnIxfykcYYB6U2PL3XQiuxV8nPKLOPMtBG92lodyfpTNISPLhjTyV2Qz4U1aMhYBZxZDn8ecD3q+I20Yx3i4HfRG2p5xu\\\\\\\/ewTfvefpNLD9gCv08inDBXcqlYTH7PNQFliixCwJk\\\\\\\/MUFXUefiLeVx6AwUlXOG63OnYLYykuyZUNSBt\\\\\\\/d+qDB2DnTK6aENxI3CfOUJUN++9jF9\\\\\\\/gAq9v4FkbwYjegpG5mEu0jZ0IPGZ1MmP\\\\\\\/AVUMNBqbaVeQ0GAAdXEfPLFJ2EHPP4135EE159zA4n6YmA+zYYBgs6oFD9gh6EPQKkOPThP8lxohjVV+h1lPULfPJDiPSfhcKmXx6f1v76opbYDu8g15ZjKco8se6EPNBzqDMTVH5eGNb6syvp766pBelKu+qkg6I7lj2oY0EITbAboOQt6cdYOi0Hg8SsBIe+DTTA39TGtkUQlMP52L4dXdRO5I2dp0Y3iksWTh9MgJ8RXxp4dLEavWSKWhKVD5hkf7MoLEzaFfFWcav1OZKWzcFZjqg+w8ecCuAb3odlJTLm3Mh5ZgyrfdTwewzHNYyg\\\\\\\/Gk+gTFW2GoeVeGE4Z6kPMJleKRXutpzOIh3PCP\\\\\\\/doKw4rJR2suS0QuRHLtfs1H3oI1ijS1H2hx6bym1FTK3lZlOkDqOWIgCzqSyNPfxeIlWhCWY88VGTHVtqF7On54m+6ybcynTGbV5Jhg7Zqz9bt4OZF5qYyEFB+TpazvvM03eidOaz3\\\\\\\/jPmqXnPRVbnQE5\\\\\\\/v7J6XvW5oG565pFO1Oe3ihs1xZmyJKOiX87FNCsg2mwos1AUyVTTpGOpx6AncRS9DV65wLituzVZY50ZjjDSYioY29okaLv2rYBW1POdH1FWOgj7LqtfsTUBPimGwWOIlXuqSqUekNuk3e6qv7VoaAOnuLH+FemCPigcbjYdilWvAXq2KsCbbwRumKClO8XzvVro5+D04GFJmeEpOLTr3DvaNzH5Prn5KpiCSX6+FlP\\\\\\\/hzMRfk24QajPPkLiV4Yh4ZsuNpQ5GnAHzPauZ+NTQDwR67k\\\\\\\/icvXY6nuYSL7eA7ia9ifUA\\\\\\\/OrM3kDSaUbRVYIdTBZO2Vf0i9DuaAQUwIBdi2dX2ZvZsTuGWxm+cnig7m2fHzo5tTbWOA\\\\\\\/84G71Ikg3cJuzkt3CNPgIZw7uV7+AwHpVCgNXcHNAT1AMZ3KT3vcuycvovDSullgTcPj\\\\\\\/GYkJHqibkOipt6tlxnIMV\\\\\\\/KE2+zpcHT6tE+Go+1zLtm+iPDqwtHMqbCdrGSosogYyVgQhtMuLa+h7aXuTalQP+c4mXmCkpZ+Uekup8QWT0bzn7QMDLWM58tJU18RtBr43\\\\\\\/izuPIp254\\\\\\\/KsTHgeCkiSsX0VUDUqLtmPpMOBCAD+JzYSeGny8Vf8JpPUCAOLKSHwsm7ijgY6EacS\\\\\\\/k0e5thXj\\\\\\\/0BCf1rcM\\\\\\\/x2CbzDJBmbd00pZ2JXPoDhVaVV0jTT+mvkJyO2tCuTFzYVOxEfJHhOSbk8i8nipXTYxtc+91Sf6A1Q1ZmMyZ9KVhkRreqfadphseP4UOvhbUuwtRZ7OPLM\\\\\\\/5AbvTzcN+sg3WrMqBH+KwZm+lzvAbDb6YeJD\\\\\\\/SFLEExld5Tsl87W+0ANWXWm4VR3HAJJehP6y7Ck1yJFdnPc39OOE2gIniAipT7A0HLoXeIm7ZqNYd\\\\\\\/lWN9hpOH7gmfghFzD9cyVIqrWyv8Zzul8khaeo1gP1rOWip6VUWFiC\\\\\\\/4z7OmDuPLIvbPL3KwS3hFXi5cTzS2AyI3L3NpRQd1i9\\\\\\\/AYabI\\\\\\\/K2nnAXKWHfIi5VpTlHVAB9g76zjowb5wWM6+RbJYLsKYi96\\\\\\\/l9UgkisKa3OrWbRo\\\\\\\/306QFxpkvwbey\\\\\\\/+2iCDwYvKfWKkPZhCWP+jylPtl6fJDOmc3SAWbMCuWmAlMBa9ei2IE0QOE8yEm+DrXfWveMn0OK7KyvKoGglz4jl0GeDAK3XAmMC6OBghNkqy9OTwdtFFaNZ3DgBQiKWTbA3AYVg2mPs2uPLFT7vVfcjvNIsGNvA\\\\\\\/kNMfq7mkTUIuCT6DVLjXdnvPIsUjfJRsH0bGGUPnA6x4GzjbVyfye5bGIqzosnfMx2DeK3O3Esp9ZSBbP5NbeNFXPFxH8RlfQg+q5S7Hfhwt8\\\\\\\/rCBmFwQ9TyZdqOLvj7MYe9YIJ\\\\\\\/zqD5IL9JQeckHKUo0nWXY3UoB\\\\\\\/1e52UMb5b79euL3UheYMVu9FJ8pCu906H6qn\\\\\\\/a93hX06pB1vv7NjeBS9YiegDPvi4z3xAEFplH5Y4mL8o6ogE5zF4J7czwPevci\\\\\\\/Z62fVePMRASLvYFpXVEkSTOFfxt1h2LfIL6YomZRHAh9uOEXeihUkrX0F3nx9\\\\\\\/fMN7INvA0jE6naAEka5PvedgZ2kKMHLfoj3iQLBsQS\\\\\\\/bjruO5LS6Xekkbu67h8nQg\\\\\\\/StpLzZA+byFsX\\\\\\\/1S4C3QaFHQc8p7tP4MUEeE78B\\\\\\\/+b2XsZetbPWg23vkCRzeAurU4jR4AOnvWDJu9HigT0BIZBQFtUBzkGgwS5nsF+bPJ\\\\\\\/rQdWrjfjWQZMrKjw7idsOMOiNzM2SIG9d+zXaRa9M9KqC+qU8D8Jx+C1FDOfvUEgvh37AOLeJd2AnEAilEaOYUETXHTcFBvWCI7jswFEP3PXvMwcK1nzopjxlwrw9MiFlfEJM1d8KBIRej2XfM9Ux7RLbensVCAj2NAgQ3Q1Bj3EdyPKsxVb0YcuJ4B5oxm4f12U59QoMkAD0KD\\\\\\\/ggPq5mnecVSeOTkYb9uFjMdkd6exTzvAalwuCwSnriZkB0h23+NbEFzF4iixrXDvdVBZDj51HEzQIj+NGit1rbGIKnR1ncOQKW3jpVu5G5ABAYccAjdfRdyt+40EfqU5TDPjHZRBDGcnmeZNoLaZU\\\\\\\/\\\\\\\/8K7y55anXcT17lVdAdqlzzroV4v\\\\\\\/xwYWavqhODwj9UeQdBhwtXtqu0VxWRE5i\\\\\\\/q\\\\\\\/gJ7kbUIVGNuvAzpZx6ebJ+64CM0wauyxvadTCg\\\\\\\/9zGaKQEkNbB6vyK217E5xeTbd0MIg+xeYSpu939ybJNVD8i+ooZAWcj9qoeL8wu0oU\\\\\\\/LTUZ4w1l2O57z2Jz2T+xETZu+HjeBnn6vEq9ytHX57I0ubnFeXxDsdMCNxVoKXsKnMU8dGV\\\\\\\/tyb8uh1EYtse\\\\\\\/eShdPS8nYjxwATl0tlJMe\\\\\\\/cfEIlnHn7wmk5uSmYcH+\\\\\\\/cv9D8mZk0hd+\\\\\\\/fo6T8PGAEj\\\\\\\/YSK2MFE1P5p3+Tcc+GijdSrssoHD0DMgLXlife5jTV4PdUqk85MCIMyHyCdl1oTivKirR+2G2LZjYKITP+Cuu\\\\\\\/UwZEZlfWOfI56pnHx\\\\\\\/ftYnahVmePZNnP9fbqI8cGDMs5Z23E4gMwptUfNLlFeTIrSBtvfR6VGyEYABVmvn8BC\\\\\\\/GPHcWlx0Aeb3Ww46XCAFEGQCzODG0YWl9+99RRRHVfIhRxJhqtIXSDK3Z\\\\\\\/mvexxUeAUbK5XBW4OhES6CwUW8rHjg75qtI6fiX5AFJVIzFzW5L5RCgp6mU9PwqxSdEQtKnrQsobjR6igwgQ9ksgW00zWySgmRayB95OsjVdpx+AinZ\\\\\\\/fXumFfl9uHTkT4vVJBuvaHvvYusLrdhUVQaaVfAmDh4vFrHdkL0AfvdGlaJW4EGvUP5QPbtLEJDh19skchVDF9NYt0WLcInFpop3Xo\\\\\\\/vUoG84MfHW9CYDHxWWTx1WQ4UP\\\\\\\/f0VNK1cKEOK9g4QbhCCDzf+IdC1K\\\\\\\/9XVg\\\\\\\/inorFk02CF\\\\\\\/r2W+cWi3bfJ81OQwTZRr\\\\\\\/uT9MUzP\\\\\\\/YNmJK3Cn08rng3fHjZr7LrByFT5SGroLbUpfruX+CGcTIE\\\\\\\/RAihjpTCWMRVFph9MLgOWixsowhPwwFt2ZBRve9Grq8BGjvaRvIOPP37Mde0013HwQTIq86K\\\\\\\/S3X5dyZKasZ4iDtx8DVlvfUgG14VRKYwU4cyEC6KCcWWJby3hDYs2NlGVsY4ge3jQ0TRuxPdG3CrxNadTOQoiu\\\\\\\/jrJU3OnEap+2K1n+WjA\\\\\\\/eXCSAMWJ4MLqEPmYBMtPhpDYZALZxuy\\\\\\\/T+wIEOsGSPpQY1E3flQoM4QMxJ8WfDOiuSxsu5ZKRwkXHzeSLPsneakfoBQTbaRx\\\\\\\/vcSPwm2iJvp5IzR5bDxZawQDXE\\\\\\\/qAW0sBgq9UX5JDKUnZbeZDw7kRFLmQHCg3nfzQMjBRbDyO+dtBR4EPfOtfM\\\\\\\/iGd795cXQkzZMeVLraFgIECMrQ6JYGPZBMwXz26BABXK0rFn6WrlSWygyNECVGriWIYPgpi0ovIkIVM\\\\\\\/ACIbzW8nAy1PABKxO+xdd6NnEK9rH9oQLN3G6McK4lbl+a0SaJMT3+2nJYafrrI9BeTLBysGPwAjoep0OtgwpBwPmIzCzDY8rU9dHihZoeEziFyaWxWRcdNUFXROhwFB8TwD5Kw6TITs1yeSGXXC3MRhFcg+1DxFQjDi4tD8khnKtvo4vaxLtxtTubWbPVPZ8jWm3TEdbmfKiG0XlOvBk1h2AaIiu5xz5A3B53rASO5u8NOlURHbqB3OFtXOQs5yDw5AO7kN0G+NRe\\\\\\\/YQRBecP5xtcu+9hu74t1RxPlWpfz4aimOo2FlYqimfl7rE45Viiwcfpfcs8kccQdxzjKYrYlUyG+o0fgjlj\\\\\\\/aOm2XCXxXa09pdzfmoZv0aLANcV+PZj9dxMKwBZ1lw5f1vg\\\\\\\/Vz9hFmyIyiyEANX1BnpN2oRHTuDdkOPCd+iBDPkEGLIWqfj5lYn3JizutmH0YnIwhlETV5IYpGpyGmgLmYNrPt7xwZS1W8L0iJ9vPGleXvI0bqBHZPwyRmntJeoG0vpxYGde6FhEl8I8js6mW2hCXLjM5L\\\\\\\/om0FkymZ5q8N5OuZcO9wwCTGs1Amq1caeE0Cf+NhIezVV0atAEOGcTwQ7SRtAzdpHg3Cb9OOuzpqZknpd+Eb+IyKJd8\\\\\\\/TPkOFKGM6v1XX5vH+Dw3Ygey9qYKX2IeNgmRaL8bJgDfP+2nthRHLpBPTdPISF13JShgdFvcTSYlxqGp0hOU8zW6nmvLMME7dkV6Hdh2BDFTr1JsQA4oLsJh78yDT9\\\\\\\/yOK7fglgnHPQeKNBxvzT5oFHVyXmE7At8U0JLmomGl7KzAQ5vzpwWBQWB7BxnP3jtSgmCdJuomx\\\\\\\/RnWZtLiI97ddn5UcjceQqqDLOenmV+AwSWKru9tnqA+UbLZSidKkJJ3tbOORtr5Ud0gAlannZdz1uK4z3D+KsCzXIefbrPvUJzYxOUXz2m4xZOcEWSi0A1HmmHUJzUQVL4erlVUyJ5lFluTVM3U0\\\\\\\/HZ3uKJUxQYlTcCZ\\\\\\\/1Zb0bE2IbsBcd7l7bnqvv8KdEFa1iv260nT3eEyuMhSp8VQT82LnLNVVt5gjlfTfW2+K0mdXzlIETFhg9nF\\\\\\\/RVGhYW1CDbrPtNqT64Vuuc3mTDet3mvnAZdYbJDB6Go+F4BxyNGd\\\\\\\/a+X3BVYpcCW2ZYtVTGjc7XCzZCW4m3CsNX3ZHR+zs+pGqnpFeV2nDscbvTfdxyiQ4jgDqEU1KMnMsAveUYoVyADDHqAkaek3KiNJAaZqQQNFzXztyiTDs6jZWrMGlHrACY2S3YFgMUCYOD7xa8y+4AtsC4UdD0Q6ml2UdSSlGI5mnNBwpt7L8b0Sq+Qq6K\\\\\\\/hcoGfBXuwf5\\\\\\\/npeYMFGJdyqyIgyt+QkdCoA6Ez\\\\\\\/LaI0irXivlUQHgc1D\\\\\\\/SCUWfK8uuZaZleNY9uJByii4ly\\\\\\\/38e2SUbO6rwLWFRP\\\\\\\/o91jdcTjMecjNSfcxa7iUhTE7h2wX9LBRasaF0mkaWjoPUMWPRVJzrzeGdPhPGCXd1+Omcgtqc8itlolAIXjpUmNm5CMH7pQ1rmAd4iVVF2R0Pjy4FO9Z6j6tupYy3aDRa0bFE2BsX84sBAlSwvI3TmKlup0VTu95KQEPhL\\\\\\\/ij73DZ\\\\\\\/2g\\\\\\\/NRPEfTmOq9mwfNa4diybxPUi7M2Y2Tr8fyLcnBia3ACZx5t2Z\\\\\\\/AX2\\\\\\\/\\\\\\\/0fJ8IMZAPhQ9YJ+7IFaw1CfBcqJFDKZ3nrbGiocTJHea4lYWZVvqj28Tqvv4ZV1SDsR+DovlwYWlBnAxWcr3vtyEjq+ivLmIpI2fX7BRhFOutYzmTDT3GltbCODrfdtdSsDpgEDmdg5QyOkLYm3WZITO6JbZ3R9GwnBSLx3Ka9mR4a4hsF6br7vPqmEUt7jDh7HqRL3uFfo5dnGvrXQmMvhTK9Hz1QhXjc74XTFLtPALGLM5+X\\\\\\\/AHLeXYHhCm3NWy964v5TEysTGh2653\\\\\\\/LIDNAM6BpMWD4JcQm3Hplu01jSCjCnpuxGY8F3M4d7lmqnHExFudgkjJQFhl1NBk8wIVGlwxBZGtgwz+FY+kP9JEacxsvOLbu8vjz0BijyaQbd9xW21kHK8wuEJwaMO1qYMDtW9jYJFee\\\\\\\/OazUcK5m6ZB0Uw3GSxahmMMkmjdpN6JmhM22MZz8o7VmCODwRS6M0VfVxm+HUsPNoff6Hx7iWDWB+wsxNz2TGrNGRTmEtOKZOlDBGVyYu+QeYo+p7KKrsy5CS7MFiiTzVwjHBzul71TCFlL1Ta+E3JKto+N9ibihUsYw4QrPD7KBBVNS0qq3Q6Vk3fpybcHVfe17udRfI0z9DZ4V0IiVTb7rZNGHlupGnKSgXBv4yg6jgcdIob2Yazmf1E1S8zJa8p4O7x+AzPVlKnI6+sjWrJ9Zr4MG9R22Ft+shRI9RcO+8M2enBgsiBTpWw0YAf56zdtOZv+z\\\\\\\/o5mtgZ5Lu1eikFMOyFC3IjzJ4bPHg71U5UkNH+0\\\\\\\/fqCmjvfQEAb57R2hKXCrrophD9gmKq+IDo9b14aNXyN1CYT0NFvz8acGI\\\\\\\/OAphjEbK+JPA2rFmEkPYYsNlwm\\\\\\\/g4V1nmApWwJgRrzOicPJ9LWHUSRY1v0RGfiWPByQUXbkXRqd+wWVxKbBBNY6Tg4tlE4L4tHXzlQ3MMAZnt4XE+\\\\\\\/YRI3iTJEyeCKQnC3zIYE5ziGl3o1QA09hldrSCB+kSaF8ghznD6eYrsKljk4YF0XNeE5t7rFA8Ao\\\\\\\/73ctzVPEdVvgh4HpJsggNxCW7qXfto4+6xF77Kl\\\\\\\/W3HynxXEpWUkWI1a2SJ6cknAOTMrZTYKHyQtQvefLKQjX5Hr0xxLMb6igqBquqM4R39kj51cmEkTUs2+V5PC+6WWQDg2Z5XJltKvdBYAXGa4jNjaZX8fmyLPAtnSzdp+AYPKQP3i+LyN+esrf9bFI8g+o6e3Jp+66tEuLYp+xTd67PrX1AfuuK7Uc+aNwBfMySqdhQ7irYCCwLQn1oV1kzhUNfl0zpTGG0nVfkSYzSLFvyPPnKf8IYids9fyEG7Nmf\\\\\\\/wJYoysS3G5hDkQwOmwZsK7tunw4tE3WgXCHV5zZWHgzfppjvwN1KPUL8U1nXF9jcuOf48AeBsGa+ixNAfSGPSqPzpV6UJLVu\\\\\\\/zRmohIuxp8Yw6pJBakCKGNZz6+uvy8u0Lzh4yHctUku5lUkhp9xqgWVoe184S5yQnxvRp1a8PBo73giXnvDGRubcQtDesypLKQe9RZPBml59GRqnYMWQHNvQIPt19IikACPF2Hl2mgsG92xI2XyvO6PWUTQUxXK2dOrNO\\\\\\\/kGAMtMTCugOOsEgv1j9sAazA1IrqwGftNPdQNics+kQfmTi2p6I7qWMcIblNn635O9DBxu1AFtSMs2ZeG15NxBY3qfmhSGZNZZhQsPwgr5aUgNrA9kNz00zr70CEP55KTSBlhh2NKOQ\\\\\\\/sb4KalcetAb9H\\\\\\\/h5r\\\\\\\/Qpw8Cd6d1ZcA9W\\\\\\\/6w2vsJoTMelnb0yX7Ztr+ECOUThfeEPeM3n1fNbonWdzqlp2ZhW1tp7lTVmnCCnVroXoI6O\\\\\\\/KlPVG6ZmbhWHTKjeed+W57Y7zYC6DIT4GBff9wzr0CbzDV18B5SJvztaG5nf\\\\\\\/e04rfArIcKNo0JoM54NA16b3uoetRSF6V\\\\\\\/1dR1wR2EIzSsJw3HlcfywpK32N0TQMVgh8oerdXiqGKshou8EZm8yPR3aD419WY0F36yypu3WtLuQsSZaPMHeAEVuW1j4n6Gq0OyubugDaCQcuDiBBI4OBEddNRTkE9A9fWeaQgt14Apk6+dBEZctMYQLRFaHEiF3WfW5KtUeMjdW5dorGSxX\\\\\\\/c6CqpLqhxr41HDNKZRloSH7xQzcKCGr6bNvIH9DVNyBBaDQa6rcFzhqdvoi5gex+tNLRQg4qgahvels7\\\\\\\/AZmhwlUOkX88RRjkeKUFKhdHBzkhp\\\\\\\/i7RT9BPrm34K4qoP7s62je2eOFoPLIZ\\\\\\\/0tBXcb9+DXHJoiwfAT\\\\\\\/qYN2RoKxYt18hbdKVOscbFTe1Qu\\\\\\\/9DGBcX\\\\\\\/HiAkqqaPRZc\\\\\\\/lWc1alHKIgxcecuClSNmvzSUhiAFj+9Flbhkn4exZ3pHo+bWoSp5n\\\\\\\/hFOCb1AdqMBEHwbStUFB4ByjH5gIzhFepNnBObWRPp52mSJL6nEqbRmEsyjnFakWYPsjvZFrd9u5RQRfgtq93ylc9kQ3095fuUj5ElTse+oe\\\\\\\/81sEGrTe7DuIlmczFLvj47q53pnevezL0ATL58dme82p+ig5W\\\\\\\/Tz+\\\\\\\/Ux2K+LfJy5bsaMgfxDOzFYqtsVnQ5fi+7cOgbLPAN4F+IKr7epG1CUVU+rYjclPIlKghMg5l+3NxoAh7I20nF6CQhLCZLker7LnqzusR1RUKjH9TG35kYg4do50mQKQvWGE\\\\\\\/SZXNtrUAlt5sf25dMebd3sAE8lFBd57F7dq9XipKXkFi9k4BGE8Fg+oajIO+jqTMtMxpRfocOG3D6tcjxznjFelBGC2+3LHh562ZrtoUZ16Zt9LBEvJUsHC2vXosiDADdOF6dulNRgBNd+Cbda6RbuI\\\\\\\/RonKEU7xaEDjqovE7OxnbuHbLgMy5AsVqC+tWYnJxDXVyFiZiJjQ\\\\\\\/sECBqIAdX90QUdLw4ADj\\\\\\\/+NQVvXtNvg3rduCRbqhbUTgfRMXNJ+B9Y3LlmrNgLkCbu2e5wJhtLx8R6FNLiJibSBhWkdy9QKhwOu5DXFAPk+xYA1stPHstLfLMLyaHLaH2jJsWomXHVgW\\\\\\\/kVxD1dtmyPuv1H5\\\\\\\/aII9JrKKedraLTgk9vEPUeG+s+6+3IkAPWnruAncDXV2ogc7a8ouExkzww6aF3p3e4hMl+9qhw0JWGhnU\\\\\\\/EcdRKkdX+BR1IfsaN5dAw97F80y9sf0\\\\\\\/w9\\\\\\\/M5d9DbJIwXAXRtakv1Dehk0Vh7J+xz5OKVvmzi0zWt8H3Vl57T56NYlz6SvNvjhdeDQcCU3V1a\\\\\\\/ePW6vfe\\\\\\\/4sXPN8ceXvRY5w4hngyMtxrwa9AlZOPyhRAu\\\\\\\/o6y0YvKUeBXTEXNZVnZq0F9rAcenuprJ02\\\\\\\/1oy27SwG2G8e\\\\\\\/jHXJt0Eqk8zav4ZBjosW1bPIodCjetjp\\\\\\\/krZP8YluiPBDUTXV0+VVAGL9i+KCk+QpehJoAS2ig3S5TPmlXHxJWrGkN3ggbZ6eDa0w6XoMCJG3gOqKHgLAiKBEAsZHQXjjPjwNqKgsZnJJHKwlUOhMar3KMuLLg4EdVEGmdjfN5SvxDaPuFBRYhfvHQoIIFZUVBEBH2vTdM422sejg6GYkve38oT2k7sf37mKvsYxROjA7LAdDMwKByZ3W78Nu7Gj4gLAL6P1ZaG8yDuCQnSXL5mHkWOrW9kovQbRMtyf8tuBPDsrpeQwfMpHE3YPLV7ggh+ZsxGhO\\\\\\\/KR+9SlkdrW3b6MHNlmE6rNepHwncbnIDc3iC2N2xuBnlQGYct2i3nUsirVs4iocJ++8dZ03jG9T0wHBdDencvclVTBEORO3kkf7ZUcVhGSzHpIwekonv0xQg8M\\\\\\\/pW0QQLbR3ks7MpSbR53hVmmWvD913wvsjhxAQmDmtyLbhfkuuIHtti9\\\\\\\/O5ZZMhOHsx8hOdeUcLdR3FfUMWPWWCKgKkKS3QUGJHAK35t7eONiN9cID00cTBCC+REArv8OmmHdzIeCRjN+VfZTpj7Y7zT3bfTsux691XDb4UkqmfqaKj8rgizhJI2EG57gKc9Fm1O3mF3vGRGeqTFeCRAacpzL03tkppmVEQbiMR3TYpCA47hWxLeaA2Ceokf8WE0MxrQfyxUG86o5yFbWqskD2fJC+B4biTSZK2LWFO\\\\\\\/J5h7gnSo90fT6mRWyH9SON\\\\\\\/F0NcahUBYEfWm3IMRkd1heemGF1nIi2Ihy9Ts+snOInn3vyL2yWQCd04az7\\\\\\\/TmF9zs8bXrDg3G0L+CSKnr0Mox\\\\\\\/LhY7yHf2pz0K3wlD+pnzr0xnIwBBGsMcFNbsL8NW3hVgaoTIVeia1yUhSG2Pn3XIp3Pw8t+9OBTl949cJeqrJjnS1HxOHKuWU+YWVrJjUiKwQ5Iv7\\\\\\\/6i1nZF0wSH6Ej3AVIT+7UlbVYYknTofSNuGYdzYtxlbT0dikQcb4TWLJI8oTrtCioXYyRfmmyyiCI5oelCRZ+0B\\\\\\\/XSh8CmPFpVGPyuktC+98VMy+79HZ7yOgZ5urP4G2Ew96p8s53fsdSVJopiL9OPCZzV3m10HQS9jEjxSS9WEIxRXimn\\\\\\\/skIlf1xc5vvufIDoetPHPfy+KWHKgg5EM+skWRAKacJW0zMbq+iyQN1\\\\\\\/z+Jxq5Ff\\\\\\\/JulP6St8ougmNsvMUbrrHJd3xm1xKRIBYDWKbf3LLjYgBZfXqqzFTjYDYQflveUKYDYXc0ea45XFGmA2dUNSU1S5zx09neo6aDDJumnXpfpsXPA+qHgJk9fNL\\\\\\\/brNx0fX1sCoL\\\\\\\/SnTqa1o\\\\\\\/188xDkH8ZGIQSnI6r54hqIchpcQRHyDpc90zKxdEAPSkP5j+2b+9PjI3McWXZKIRCv+lYD8hOqXLpJm33go1ZON1\\\\\\\/TRxpGSL0X0+oCfPkBnr+yEaGh\\\\\\\/kWqv0qbGIu7nbLV7WTi69dcD1fb+1EQT6JAWfwdyGY5TWWkOBunNgIfiR9MoZQ5+gzg9ucwcFAcU6i4BH6sU0uIGp4yLFJdIWebCz+Drqicpw6eCZaLp32cxcNXQ3pPRg7cFnVSTj3T1vpCxpUDvsI+pKPuD9vUEfWJIM2Kts7DfAHJecpIxpnZkzLjGP0ih1YtH0lZ+JNmB0s0uUsO6V8V+puevaVQZLNNxLjtOt+1pjZSesvY62jIR7Cwi3OAOaHhgw4CWRCdvl\\\\\\\/zHQFULS52+C1d1RXaT3qzLF5mVt3pA+I3pQDXKBVfQRsX+Xe4ISHVSHIPEXyYchO7oJNKzM1Jl+ETEWHzrhhnp8vJJsWxPyCh9i\\\\\\\/Yeb7HwWNnUqLnPMKelFt8Fm5WQGl3WuBXyGWnrOGa1Pe13t9RKKjqxB4HfYTvIcJzvHDUrv\\\\\\\/NfFk6XakoVR0avk0RmmMP5hm7WCjyzbRK1ZUZiBSzxA8yJOFiDF31LFveCJx\\\\\\\/06AwLxdhIlW3voqsVXO5IH\\\\\\\/mjLwIrtVe6B5fyujpWgNSXjErZAS6PaNoeUe2DxTpTk+FagC80uI45CYNG1ZDQb0ASptTV21Enp7DSyDxlLDuSTnTEvL1fJGvXoqhTCLmedcOyo37L\\\u0022,\\\u0022iv\\\u0022:\\\u0022KnAT\\\\\\\/9qxcaCStdQgAZA31Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bab5e9ac5a411c1d9933ecf158d8b496adba8126203765b67dbae427a52bb789\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Asc\\\\\\\/VsBSXfuY7IbqhnByHZsrQvTIMEo8Lc9lDEHaiiZA3XrLs98n2tVIeJH3Txax3+T4TMkY++fvQ4KB12qUW5KFRXh3t2XhiXBJtl0W0xnko8olXCcs22FY4+7h3XhftCdJO2cj\\\\\\\/JI7lz6Ayj0jAMRl60CjBq5dA+3Lu6s5vM53c4GVxuOiu5Fg9Kqa5NMcke3mC83\\\\\\\/gUbqeM6XPhXq308koeAecle29g+IbiigDv0LdXa0zKxK4c2s3XQJXUjO1yLTrF1hPryGtfs5EFp2CHWb9LJvvapgObD83x+0qilIQ884m1+zuASWGK0FCurWWCm43MmMv8YVbd92A\\\\\\\/Q8JYAf9ab4JhW9sreM3ASg5Dwop3SExYvCXeqjb6Q5ITRBgUgkx83c6EXuC2MGRwepob72y\\\\\\\/9y2VBDXlUj22W53y1q5oVkhjuX2YvMMd10DH910iEQBzu0y2eIf+NGgcY4i25vIhUmtW3z1iLMZv\\\\\\\/KX\\\\\\\/dHwujyew\\\\\\\/BMOvnz7+Ft1FJZKazr4\\\\\\\/gaHg5OF6eloOhmc+XBIqvAstPh5K3z7HG7MnpG5mVM6cdcg8\\\\\\\/nfbySUbiCkf+4S3VJWIoiUoeiPHXNJpq5xrmLB1QnHKPYH7SU2I3ICTbekgEzhTXKv2bEYkmhL5P5\\\\\\\/kaAq7P6bmt8QpEtLObz9IbMDo3Zz9dfPOGk8pUjrb87MEeaV6aUqSL9lqa3zp9mFkLJq8LHwFn8YqCYS2FJrftUrIvbkQKmmIltB2ABxVWvv1pY4QcYOW2zfB\\\\\\\/zU4naGUpo8\\\\\\\/2DnJFy0DaHPZthuYvfUk9rDhycL2HXxa8366lQeSEWnQ4eO8MxxY0r4iP86i23+9MXxXk7bwymPR5PziMeZgyIMBM3babI0UqHQ0SGl40VK9UQ\\\\\\\/E6fvmWez3gPsrZ9wZBlIZAejMKPGDBoI+gPgZkclIIXpTod7WfsazKvsOD2o8HqVoYP3PWBvgXT5lALPUbJ6j0JoUAtVfFUydqJNjT1PWn8lj+bkoIjVevSYRXGS5jQP54tgIjHc7XXWn8fc8htt3fiXmim4v24BeOi4NVjFfLBPzzGao\\\\\\\/LIbXy4s3vTaTsPvyM351kb7ciJdPVw\\\\\\\/V3eXR24SrRTnJFiAl9PN0ZxYAHxFifZn+N+aEICKohyl+EgiuLAXyYkTwP4csXy3x6hwdmOaBCOOw895\\\\\\\/+rY2LMKdr4foRtES2Wzrfp6bKUT9Ro9BdKzJ3ImVpGFmw+lOTh67a43wgpOYsrU9G5A2OgrKyB0D\\\\\\\/p1V+AUEkL7ivF+\\\\\\\/mb+bDhGaZOeH4MoxrQIDoPOTIOtgsi3IHxWj\\\\\\\/YF6CrL7TRkurEms8rtAJzKqYlZDrjYxpPH3uxnZi4YE0eoSd4cT7QIadq7QW4e7kS8Np32ZGVDfWDRprqPD\\\\\\\/1P5K6nzg81qeb8ExB3DKqMa1LgE5mu1\\\\\\\/c91r7uivtPDDHPo1Xntah7n41IDDBiNrDxaGIJvtRtyCweIsMlqkS3emuybmQ88I8aCoUUlQNz6UfpHOBRvozadYG\\\\\\\/ZSfjGQ8wqvY2ttbgvSBaspdK92\\\\\\\/Bc+bAgyGyWgpaNSY5xXKXt8n6PjeLAccjmc2TZ\\\\\\\/M2vO0LTV9ElO9ynbiLgNe\\\\\\\/Smbp+G2GpMcVhwv29H7z+Ds8Jyqyf\\\\\\\/3BzXikUu\\\\\\\/nhVaD1WyZyfLVJIXfZm8qM5juv+P6XP+L7N6XT489q+KL0ETYwSYOua5sJqk9iFlZRwN55dgahgRC6FUxpK3Iz7FfZgJAt4c2deM5hUHROOmRM1Xw2Kjr9hU395JlF0n1M3RGRJB5PoUGP4ofhQMVdzvigpVUYIRwg57zXC\\\\\\\/xCVEiA1RUtznNWfpixLW0C8yCKCRTW8doVAQqJX\\\\\\\/aWC92DttVqKM6FjriNL\\\\\\\/ak5t97YdbaOrRHstMiahffYIBiyr0CqFEFbEKadNBty864TZDwNGdDK56MHCcrbbXBCNj2QKNe6UauABcwr6jOAqlGx70XBD9jylnsHdJr0G2HQVU25VbIxEJJudxXJ7cD0voVPXGnXraR9aKxNeBRUr8tykE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022dI4\\\\\\\/IfCNYZ67PwWsdRXBfA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022180b55cabd0606c12cbda89490358f0f7a356e9c65f6482a0dd51f149e423c1c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460387 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1597" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Z6Ny_Ym_ygig1QHRbaryjXw0JjSGXgwUv2jhktC4NX8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1597" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j_Q85Hwek6UlYLuo3brEGIU8cWYXcRX5kbz282xDWcc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1585" title="Fungal Innate Immunity Induced by Bacterial Microbe-Associated Molecular Patterns (MAMPs)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1607" title="A High-Resolution SNP Array-Based Linkage Map Anchors a New Domestic Cat Draft Genome Assembly and Provides Detailed Patterns of Recombination" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460387" id="node460387" data-pisa="ggg;6/6/1597" data-pisa-master="ggg;g3.116.028274" data-apath="/ggg/6/6/1597.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Supriya Khedkar</span> and <span class="highwire-citation-author" data-delta="1">Aswin Sai Narain Seshasayee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1597-1606;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460387" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Supriya Khedkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Supriya%2BKhedkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khedkar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASupriya%2BKhedkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aswin@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">aswin@ncbs.res.in</a>
<a href="mailto:supriyak@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">supriyak@ncbs.res.in</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aswin Sai Narain Seshasayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aswin%2BSai%2BNarain%2BSeshasayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seshasayee%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAswin%2BSai%2BNarain%2BSeshasayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aswin@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">aswin@ncbs.res.in</a>
<a href="mailto:supriyak@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">supriyak@ncbs.res.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/6/1597" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460387" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460387/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1597.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460387" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460387/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1597.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460387" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460387/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1597.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460387" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460387/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/6/1597.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460387" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460387/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomes evolve not only in base sequence but also in terms of their architecture, defined by gene organization and chromosome topology. Whereas genome sequence data inform us about the changes in base sequences for a large variety of organisms, the study of chromosome topology is restricted to a few model organisms studied using microscopy and chromosome conformation capture techniques. Here, we exploit whole genome sequence data to study the link between gene organization and chromosome topology in bacteria. Using comparative genomics across ∼250 pairs of closely related bacteria we show that: (a) many organisms show a high degree of interreplichore translocations throughout the chromosome and not limited to the inversion-prone terminus (<em>ter</em>) or the origin of replication (<em>oriC</em>); (b) translocation maps may reflect chromosome topologies; and (c) symmetric interreplichore translocations do not disrupt the distance of a gene from <em>oriC</em> or affect gene expression states or strand biases in gene densities. In summary, we suggest that translocation maps might be a first line in defining a gross chromosome topology given a pair of closely related genome sequences.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-evolution" class="hw-term hw-article-keyword hw-article-keyword-genome-evolution" rel="nofollow">genome evolution</a></li><li class="kwd"><a href="/keyword/genome-architecture" class="hw-term hw-article-keyword hw-article-keyword-genome-architecture" rel="nofollow">genome architecture</a></li><li class="kwd"><a href="/keyword/chromosome-topology" class="hw-term hw-article-keyword hw-article-keyword-chromosome-topology" rel="nofollow">chromosome topology</a></li><li class="kwd"><a href="/keyword/gene-order-conservation" class="hw-term hw-article-keyword hw-article-keyword-gene-order-conservation" rel="nofollow">gene order conservation</a></li></ul><p id="p-3">Chromosome compaction is a necessary feature of all living cells including bacteria. It is intricately linked to chromosome topology and the selective pressures introduced by gene organization along the genome sequence.</p><p id="p-4">In many bacteria, nucleoid-associated proteins (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dillon and Dorman 2010</a>) and topoisomerases (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">DiNardo <em>et al.</em> 1982</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pruss <em>et al.</em> 1982</a>) establish local DNA geometries, which include bending (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Stella <em>et al.</em> 2010</a>), bridging (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dame <em>et al.</em> 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dupaigne <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Maurer <em>et al.</em> 2009</a>), wrapping, and relative over-/underwinding of DNA. These not only help compact DNA, but also play a role in constraining replication (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gruber <em>et al.</em> 2014</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">X. Wang <em>et al.</em> 2014a</a>), recombination (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Esposito and Gerard 2003</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Shanado <em>et al.</em> 2001</a>), and transcription (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Browning <em>et al.</em> 2010</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dillon and Dorman 2010</a>).</p><p id="p-5">Zooming out, two broad chromosome topologies have been discerned across model bacteria. First, in <em>Escherichia coli</em>, genetic, recombination-based contact maps of the chromosome showed the presence of four macrodomains, defined by the preferential occurrence of recombination events within rather than across these macrodomains (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Valens <em>et al.</em> 2004</a>). Chromosome topology in <em>E. coli</em> is further characterized by the localization of the origin of replication (<em>oriC</em>) at midcell and the transverse spread of the left and right replichores in opposite directions toward the cell poles (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nielsen <em>et al.</em> 2006</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wang <em>et al.</em> 2006</a>). This is referred to as the “<em>transverse</em>” chromosome topology.</p><p id="p-6">On the other hand, high-resolution 5C and Hi-C chromosome contact maps of <em>Caulobacter crescentus</em> present a different picture by revealing a “<em>longitudinal</em>” topology (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Umbarger <em>et al.</em> 2011</a>). In the <em>C. crescentus</em> chromosome, symmetric contacts (defined as contacts between loci equidistant from <em>oriC</em>) between the two replichores (two arms of the chromosome extending from <em>oriC</em> at one pole to the terminus at the other pole) could be discerned, possibly resulting in a helical structure (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Le and Laub 2014</a>); such contacts are rare in the <em>E. coli</em> chromosome (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nielsen <em>et al.</em> 2006</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Wang <em>et al.</em> 2006</a>). Similarly, recent microscopy experiments showed that the two replichores of <em>Mycobacterium smegmatis</em> colocalize throughout replication (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Santi and McKinney 2015</a>). For both the topologies, distinguished by the presence or absence of close interreplichore contacts, it is apparent that <em>oriC</em> and the terminus (<em>ter)</em> rarely contact each other. Finally, the chromosome of <em>Bacillus subtilis</em> has been shown to oscillate between these two conformations (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">X. Wang <em>et al.</em> 2014b</a>).</p><p id="p-7">The molecular mechanisms and players underlying the establishment of a defined chromosome topology are only beginning to be understood. These include DNA-binding proteins such as MatP (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mercier <em>et al.</em> 2008</a>), the SMC complex (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>), nucleoid-associated proteins (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Umbarger <em>et al.</em> 2011</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>), and noncoding RNAs (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Qian <em>et al.</em> 2015</a>). Chromosome topologies might also be influenced by active cellular processes such as DNA replication and transcription. In <em>E. coli</em>, replication has been implicated in shaping the topology of the chromosome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cagliero <em>et al.</em> 2013</a>). Modeling of the bacterial chromosome using tools from polymer science has indicated that large switches in chromosome topology can occur by the mere repositioning of <em>oriC</em> or <em>ter</em> sites within the cell (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Junier <em>et al.</em> 2014</a>). In <em>C. crescentus</em>, highly expressed genes act as topological domain boundaries on the chromosome (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>).</p><p id="p-8">An important question then, is whether chromosome topology has any functional relevance. In other words, is chromosome topology under evolutionary selection? Whereas local DNA geometries, including the writhe of DNA around a nucleoid-associated protein, has definite effects on transcription and DNA repair (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Dillon and Dorman 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dorman 2013</a>), whether the overall shape of the chromosome has any function, beyond being a solution for compacting DNA within the confines of a cell, is unclear. Long-range interactions between chromosomal segments might enable coexpression of genes encoded in these segments (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wang <em>et al.</em> 2016</a>). The collapse of transcriptionally silent genes by the global repressor H-NS into a few foci in <em>E. coli</em> (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>) – akin to the spatial clustering of distinct heterochromatin sequences in the eukaryotic nucleus − indicates conservation of this topological characteristic across phyla; whether this is a requirement for transcriptional silencing remains an open question. This point, however, must be tempered by the fact that fluorescent tags used for observing the foci may actually be the cause of some of the clustering effects (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">S. Wang <em>et al.</em> 2014</a>). Further, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Esnault <em>et al.</em> (2007)</a> have revealed that the effects of DNA inversions on growth are dependent on their impact on chromosome topology; <em>i.e.</em>, inversions that span multiple topological macrodomains of the chromosome are more detrimental than those that are limited to a single domain, thus suggesting that chromosome topology might be under evolutionary selection.</p><p id="p-9">While the study of aspects of bacterial chromosome topology and of its evolution is still nascent (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lagomarsino <em>et al.</em> 2015</a>), we have a substantially deeper understanding of chromosome architecture in terms of how genes are organized on the bacterial chromosome (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rocha 2008</a>). Depending on the time spent per cell cycle by a bacterial cell in replicating the genome, there is a clear difference in gene dosage between <em>oriC</em> and <em>ter</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Block <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Schmid and Roth 1987</a>). This is particularly apparent in fast-growing bacteria like <em>E. coli</em>, where <em>oriC</em> fires multiple times per cell cycle (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cooper and Helmstetter 1968</a>). As a potential consequence of this critical difference in gene dosage between <em>oriC</em> and <em>ter</em>, essential and highly expressed genes are encoded close to <em>oriC</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rocha 2004</a>). In contrast, many stress-responsive (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sobetzko <em>et al.</em> 2012</a>) and horizontally acquired genes are encoded around <em>ter</em> (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Zarei <em>et al.</em> 2013</a>). Finally, a majority of bacterial genes – particularly those that are essential – are encoded on the leading strand (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rocha and Danchin 2003</a>), consistent with the idea that head-on collision between the two polymerases is mutagenic and detrimental (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mirkin and Mirkin 2005</a>).</p><p id="p-10">Gene organization changes as species diverge (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tamames 2001</a>). For example, comparative genomics has shown that loci immediately adjacent to <em>oriC</em> and <em>ter</em> are prone to interreplichore inversions (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eisen <em>et al.</em> 2000</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Suyama and Bork 2001</a>). Previous anecdotal genome alignment analyses have also revealed the presence of interreplichore translocations in certain pairs of genomes; more so for distantly related organisms. These result in the classical X-shaped pattern in sequence comparison dot plots (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Eisen <em>et al.</em> 2000</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Koonin and Wolf 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mira <em>et al.</em> 2002</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Suyama and Bork 2001</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Tillier and Collins 2000</a>).</p><p id="p-11">Here we interrogate the relationship between chromosome topology and gene organization across bacteria. Specifically, we ask whether we can exploit the wealth of genome sequence data to explore the utility of gene organization as a tool to infer the topology of a bacterial chromosome, thus accounting in part for the paucity of chromosome conformation data across bacteria.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-12">We obtained genomic DNA [.fna files], RNA [.frn;.rnt files], and protein [.faa] sequence data for ∼3000 completely sequenced bacteria from the National Center for Biotechnology Information (NCBI). We obtained gene coordinates, protein IDs, and Cluster of Orthologous genes (COG) annotations from .ptt files (<a href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_refseq/Bacteria/">ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_refseq/Bacteria/</a>).</p></div><div id="sec-3" class="subsection"><h3>R-factor calculation</h3><p id="p-13">We calculated R-factor (<em>R</em><sub>f</sub>) as a physiologically relevant measure of replication-dependent gene dosage gradient down the <em>oriC−ter</em> axis. This considers both replication time and minimum doubling time. It was first described by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Couturier and Rocha (2006</a>).</p><p id="p-14">The R-factor (<em>R</em><sub>f</sub>) was defined as <strong><em>R</em><sub>f</sub></strong> = <strong><em>T</em><sub>R</sub></strong>/<strong><em>T</em><sub>D,</sub></strong> where <strong><em>T</em><sub>R</sub></strong> stands for the estimated time taken for a full round of chromosomal replication, and <strong><em>T</em><sub>D</sub></strong> represents the minimum doubling time. <strong><em>T</em><sub>R</sub></strong> was defined as the ratio of half the genome size and an average speed of DNA replication <em>in vivo</em>, which was taken to be 600 nt/sec (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Reyes-Lamothe <em>et al.</em> 2008</a>) for all bacteria. This is an oversimplification: DNA replication speeds vary across bacteria, but such data are not readily available, and the extent of its variability is unknown. <strong><em>T</em><sub>D</sub></strong> was obtained for ∼100 bacteria from <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Freilich <em>et al.</em> (2009</a>).</p></div><div id="sec-4" class="subsection"><h3>Distance calculation</h3><p id="p-15">We determined the location of a gene in the genome as its distance from <em>oriC</em> down the replichore. We first obtained <em>oriC</em> and <em>ter</em> coordinates for 1528 completely sequenced bacteria from DoriC (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gao and Zhang 2007</a>). We calculated the shortest distance of all genes from the origin of replication and divided it by half the genome size. The normalized distances of genes from <em>oriC</em> varied from 0 to 1, 0 being close to <em>oriC</em> and 1 being close to <em>ter</em>.</p></div><div id="sec-5" class="subsection"><h3>Dataset for pairwise comparisons</h3><p id="p-16">We extracted a 16S rRNA sequence for 1528 completely sequenced bacteria. For organisms with multiple 16S rRNAs, we obtained the sequence of the copy closest to <em>oriC</em>. We did a 16S rRNA sequence similarity search using the Needleman−Wunsch global alignment algorithm (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Needleman and Wunsch 1970</a>), implemented in EMBOSS (EMBOSS:6.4.0.0) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rice <em>et al.</em> 2000</a>) using default parameters. We then considered all pairs of bacteria with greater than 97% 16S rRNA sequence identity. For further analysis we included only those pairs of genomes encoding the same number of 16S rRNA genes, as determined from the .rnt files in NCBI. This ensures that pairs of genomes used for comparative genomics would have similar growth rates. As described in <em>Results</em>, we removed pairs involving: (a) strains of the same species; (b) redundancy; and (c) obligate parasites. Our final dataset consisted of 232 individual bacterial species forming 262 pairs of bacteria with the same 16S rRNA copy number and ≥97% 16S rRNA sequence identity.</p></div><div id="sec-6" class="subsection"><h3>Determination of orthologs</h3><p id="p-17">For the 262 pairs of bacteria, using the protein sequences (data from .faa files) we did an all <em>vs.</em> all bidirectional best-hit <em>phmmer</em> (HMMER 3.0, March 2010) search (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Finn <em>et al.</em> 2011</a>) with default parameters and an E-value cut off of 10<sup>−10</sup>. The data pertaining to all the identified orthologs used in this study for 262 pairs of bacteria can be accessed here: <a href="http://bugbears.ncbs.res.in/genome_architecture/">http://bugbears.ncbs.res.in/genome_architecture/</a>.</p></div><div id="sec-7" class="subsection"><h3>Correction for the effect of phylogenetic distance</h3><p id="p-18">The correction for the dependence of various measures of gene organization on phylogenetic distance (mentioned in the main text) was implemented as follows. We fitted the dependent variable (<em>y</em>) against 16S sequence similarity (<em>x</em>) using local polynomial regression (LOESS). The residual of this fit was calculated by subtracting individual values of <em>y</em> from the predicted values of <em>y</em> obtained after the LOESS regression. These residual values represented the component not explained by phylogenetic distance.</p></div><div id="sec-8" class="subsection"><h3>Assignment of chromosomal bins</h3><p id="p-19">We defined 25% of the chromosome centered around <em>oriC</em> as the O or origin bin. Twenty-five percent of the chromosome immediately to the right (along the text of the genome sequence) and to the left of the O bin was termed R and L, respectively. The remaining 25% of the chromosome, which includes <em>ter</em>, was termed the T bin.</p></div><div id="sec-9" class="subsection"><h3>Determination of functional enrichment</h3><p id="p-20">Functional annotations for individual orthologs were extracted from NCBI [.ptt files]. These COG annotations were used to determine enrichment of specific functions in each bin. Twenty-three COG classes (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.028274/-/DC1/FileS1.pdf">File S1</a>) belonging to three broad functions − information storage and processing, cellular process, and signaling and metabolism – were considered. For each genome, Fisher’s exact test was used to determine statistical significance for the enrichment of each COG class in any given bin. The obtained <em>P</em>-values were then corrected for multiple comparisons using Bonferroni correction and were negative log base 10 transformed.</p></div><div id="sec-10" class="subsection"><h3>Prediction of horizontally acquired genes</h3><p id="p-21">We used Alien Hunter (Release 1.7) (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Vernikos and Parkhill 2006</a>) with default parameter values to predict horizontally acquired genes. For this analysis, bacteria with guanine and cytosine (GC) content in the range of 40–60% were considered. The coordinates of predicted horizontally acquired genes obtained from Alien Hunter were further used to assign them to individual chromosomal bins. Depletion of predicted horizontally acquired genes in an individual bin was determined using Fisher’s exact test adjusted for multiple testing by Bonferroni.</p></div><div id="sec-11" class="subsection"><h3>Analysis of C. crescentus 5C data</h3><p id="p-22">We obtained chromosome conformation capture data for <em>C. crescentus</em> NC_011916 from <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Le <em>et al.</em> (2013</a>)(Gene Expression Omnibus − accession no. GSE45966; sample GSM1120445). From this dataset we extracted normalized contact frequency information for all bins showing interreplichore contacts.</p><p id="p-23">Ortholog positions that showed interreplichore translocations in <em>C. crescentus</em> NC_011916 and its closely related bacterial pairs were pooled together. For all these translocated loci we extracted the contact frequency information from the above dataset. This formed the test dataset consisting of contact frequencies for interreplichore translocated loci. The control set included all other pairs of interreplichore loci that did not show translocations. The two distributions were compared using the Wilcoxon test.</p></div><div id="sec-12" class="subsection"><h3>Phylogenetic tree</h3><p id="p-24">The 16S rRNA sequence of the rRNA copy closest to origin of replication was obtained for all 232 bacterial species used in this study. 16S rRNA multiple sequence alignment was generated using MUSCLE (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Edgar 2004</a>), as implemented in MEGA version 6 (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Tamura <em>et al.</em> 2013</a>) using default parameters. A maximum likelihood (ML) tree was generated in MEGA version 6 using all sites in the 16S rRNA alignment. The Tamura and Nei (1993) nucleotide substitution model was used to infer the tree. The phylogenetic tree was visualized using iTOL (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Letunic and Bork 2011</a>), and growth rate and interreplichore translocation frequency were overlaid on the phylogenetic tree.</p></div><div id="sec-13" class="subsection"><h3>Randomization protocol</h3><p id="p-25">For each pair of bacteria showing interreplichore translocations, the assignment of an ortholog to a gene was randomized, thus creating random pairing between genes. Then <em>D</em><sub>inter</sub> was calculated for each gene pair in this randomized dataset. These randomized datasets were used to compute a null distribution for <em>D</em><sub>inter</sub> in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>, and as the null expectation for the frequency of interreplichore strand flips in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>.</p><p id="p-26">All the statistical tests and data visualizations were performed using <em>R</em> (unless specified otherwise).</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-27">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>A comparative genomic analysis of gene position conservation</h3><p id="p-28">In order to infer chromosome topologies and their association with gene dosage gradients down the <em>oriC</em>−<em>ter</em> axis, we established a comparative genomic framework. Our approach is based on the frequencies of translocations between distal segments of the chromosome, as defined by sequence comparisons between pairs of closely related organisms. This is based on the supposition that spatially proximal regions of the chromosome would recombine more frequently, and recombination combined with selection would result in the establishment of translocation events between such loci. This approach is similar to the experimental mapping of the topology of the <em>E. coli</em> chromosome using recombination frequencies between distal loci (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Valens <em>et al.</em> 2004</a>), with the difference that the translocations we identify are products of recombination followed by selection and probably drift over long timescales.</p><p id="p-29">We obtained complete genome sequences and gene annotations for ∼3000 fully sequenced bacterial genomes from the NCBI. Because gene dosage gradients down the <em>oriC</em>−<em>ter</em> axis are dependent on growth rates, we sought to annotate each organism in our collection of genomes with growth rate information. To do so, we assembled a dataset of growth rates in the form of minimum doubling times for over 100 bacteria from Freilich <em>et al.</em> (2009).</p><p id="p-30">Although growth rate information is sparse relative to genome sequence data, we know that the number of rRNA genes encoded by a bacterium is a predictor of its growth rate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Condon <em>et al.</em> 1995</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Klappenbach <em>et al.</em> 2000</a>), with fast-growing bacteria encoding more rRNA operons than slow-growing ones. Consistent with this knowledge, we observed a negative correlation (ρ<sub>Spearman</sub>= −0.76) between minimum doubling times and rRNA copy numbers for these ∼100 bacteria (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparative Genomics Framework. (A) Plot representing the relationship between minimum doubling time (hr) and 16S rRNA copy number (&#x3C1;Spearman = &#x2212;0.76, P-value 50 hr are not represented in the plot. (B) Flowchart showing the selection of 262 pairs of bacteria used in this study. (C) Schematic showing the conservation of gene position (solid lines) and gene translocations (dashed lines) between pairs of bacteria. (D) Plot representing the correlation between residual &#x3C1;GOC and Rf (&#x3C1;Spearman = 0.31, P-value = 2.2 &#xD7; 10&#x2212;7). Inner panel shows a statistically significant difference between residual &#x3C1;GOC for slow (Rf &#x2264;1) and fast (Rf &gt; 1) growing bacteria (P-value = 3.3 &#xD7; 10&#x2212;6, Wilcoxon test). Asterisks indicate P-value &lt; 10&#x2212;3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1825183234" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparative Genomics Framework. (A) Plot representing the relationship between minimum doubling time (hr) and 16S rRNA copy number (&#x3C1;&lt;sub&gt;Spearman&lt;/sub&gt; = &#x2212;0.76, &lt;em&gt;P&lt;/em&gt;-value &lt;10&lt;sup&gt;&#x2212;10&lt;/sup&gt;). Minimum doubling time values &gt;50 hr are not represented in the plot. (B) Flowchart showing the selection of 262 pairs of bacteria used in this study. (C) Schematic showing the conservation of gene position (solid lines) and gene translocations (dashed lines) between pairs of bacteria. (D) Plot representing the correlation between residual &#x3C1;&lt;sub&gt;GOC&lt;/sub&gt; and &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;f&lt;/sub&gt; (&#x3C1;&lt;sub&gt;Spearman&lt;/sub&gt; = 0.31, &lt;em&gt;P&lt;/em&gt;-value = 2.2 &#xD7; 10&lt;sup&gt;&#x2212;7&lt;/sup&gt;). Inner panel shows a statistically significant difference between residual &#x3C1;&lt;sub&gt;GOC&lt;/sub&gt; for slow (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;f&lt;/sub&gt; &#x2264;1) and fast (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;f&lt;/sub&gt; &gt; 1) growing bacteria (&lt;em&gt;P&lt;/em&gt;-value = 3.3 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;, Wilcoxon test). Asterisks indicate &lt;em&gt;P&lt;/em&gt;-value &lt; 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1597/F1.medium.gif" width="411" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/6/1597/F1.medium.gif" width="411" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468492" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Comparative Genomics Framework. (A) Plot representing the relationship between minimum doubling time (hr) and 16S rRNA copy number (ρ<sub>Spearman</sub> = −0.76, <em>P</em>-value &lt;10<sup>−10</sup>). Minimum doubling time values &gt;50 hr are not represented in the plot. (B) Flowchart showing the selection of 262 pairs of bacteria used in this study. (C) Schematic showing the conservation of gene position (solid lines) and gene translocations (dashed lines) between pairs of bacteria. (D) Plot representing the correlation between residual ρ<sub>GOC</sub> and <em>R</em><sub>f</sub> (ρ<sub>Spearman</sub> = 0.31, <em>P</em>-value = 2.2 × 10<sup>−7</sup>). Inner panel shows a statistically significant difference between residual ρ<sub>GOC</sub> for slow (<em>R</em><sub>f</sub> ≤1) and fast (<em>R</em><sub>f</sub> &gt; 1) growing bacteria (<em>P</em>-value = 3.3 × 10<sup>−6</sup>, Wilcoxon test). Asterisks indicate <em>P</em>-value &lt; 10<sup>−3</sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Based on the fit between minimum doubling times and rRNA copy numbers, we estimated the growth rate expected for each rRNA copy number observed in our dataset. However, growth rate itself is not necessarily a strong predictor of differences in gene dosage between <em>oriC</em>-proximal and <em>ter</em>-proximal loci. Instead, a better correlate is the <em>R</em>-factor (<em>R</em><sub>f</sub>), defined as the number of replication cycles per cell division. Using a previous study from the Rocha group as a guide (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Couturier and Rocha 2006</a>), we calculated the <em>R</em><sub>f</sub> for each bacterium as described in <em>Materials and Methods</em>. We then defined organisms that are expected to initiate replication more than once per cell cycle on average (<em>R</em><sub>f</sub> &gt; 1) as fast-growing, and the rest as slow-growing. We also estimated <em>R</em>-factor independently using a larger, more recent dataset (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Vieira-Silva and Rocha 2010</a>) of minimum doubling times for ∼200 bacteria, and found estimates of R-factor derived from this dataset to be consistent with those used in this study (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/1">Figure S1</a>).</p><p id="p-33">We organized the ∼3000 genomes into 262 genome pairs on the basis of phylogenetic distance and growth rate. We considered pairs of bacteria with ≥97% 16S rRNA sequence identity (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/2">Figure S2</a>) and selected those with similar growth rates (same 16S rRNA copy numbers) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). We then removed: (a) pairs of strains of the same species, which typically showed very few long-range translocations (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/3">Figure S3</a>); (b) redundant pairs to minimize overrepresentation of certain phylogenetic groups: for example, there are several hundred comparisons involving <em>E. coli</em> and <em>Salmonella</em>, and of these one was picked at random; and (c) pairs of obligate parasites with genomes smaller than 1.3Mb (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Koonin and Wolf 2008</a>), whose reduced genomes are known to be stable with exceptionally high gene order conservation (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tamas 2002</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Mira <em>et al.</em> 2002</a>). Thus, our final selection consisted of 232 individual bacterial species forming 262 pairs of bacteria with the same 16S rRNA copy number and ≥97% 16S rRNA sequence identity. A phylogenetic tree of these bacteria, based on their 16S rRNA sequences, is shown in <a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/4">Figure S4</a>.</p><p id="p-34">For all the 262 pairs of bacteria, we identified gene orthologs by bidirectional best-hit phmmer (E-value &lt;10<sup>−10</sup>). We found that ∼72% (median) of genes in one organism had an ortholog in its partner genome. The percentage of genes conserved in a pair of organisms decreased with phylogenetic distance between the compared genomes and showed little correlation with growth rate (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/5">Figure S5</a>).</p><p id="p-35">For all these genes, we computed the position of each gene on its chromosome. The position of a gene was defined by its distance from <em>oriC</em> along the path of the replication fork and normalized to genome size. Positions equidistant from <em>oriC</em> on either replichore were considered equivalent in these calculations. Thus, a difference in gene position would indicate a difference in dosage, depending on the organism’s R-factor. For each genome and its partner, we defined gene position vectors, in which each gene with an identifiable ortholog in its partner genome was represented by its distance from <em>oriC</em>. Then, for each genome pair, we computed the correlation coefficient between their gene position vectors (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>) and treated this correlation as an estimate of gene order conservation (ρ<sub>GOC</sub>).</p><p id="p-36">As expected for closely related organisms, the average gene order conservation across the 262 pairs of organisms studied here was high (median ρ<sub>GOC</sub> ∼0.9). Loss of gene order conservation is dependent on phylogenetic distance (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Suyama and Bork 2001</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Tamames 2001</a>). Hence, we calculated residual ρ<sub>GOC</sub> after correcting for the dependence of ρ<sub>GOC</sub> on phylogenetic distance (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/6">Figure S6</a>). We found that the residual ρ<sub>GOC</sub> increases with increasing R-factor (<em>R</em><sub>f</sub>) weakly but in a statistically significant fashion (<em>P</em>-value= 3.3 × 10<sup>−6</sup>, Wilcoxon test; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>), consistent with earlier calculations by Rocha and colleagues (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Couturier and Rocha 2006</a>).</p><p id="p-37">To identify patterns of long-range translocations, which could inform us about chromosome topologies, we took the following coarse-grained approach. We binned each chromosome into four equally sized segments. The first segment, termed ’O’ for the presence of <em>oriC</em>, comprised 25% of the chromosome centered around <em>oriC</em>. The ’L’ (Left) and the ’R’ (Right) segments each covered 25% of the chromosome on either side of the O segment. The remaining 25%, including the terminus of replication, was termed the ’T’ (Terminus) segment (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Each gene with an ortholog in a partner genome was assigned to one of these four bins.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal bin-based conservation of function. (A) Schematic representing equal sized bacterial chromosomal segments. Red: Origin bin (O); green: Right bin (R); blue: Terminus bin (T); black: Left bin (L). (B) Boxplot showing the enrichment of translation and ribosome biogenesis genes (COG category J) in O, R, T, and L chromosomal bins. (The dashed gray line at y = 1.3 corresponds to a P-value of 0.05. Values &#x2265;1.3 are statistically significant.). Inner panel shows the enrichment of COG J in the O bin of fast-growing bacteria (Rf &gt; 1). (C) Boxplot showing the depletion of horizontally acquired genes in different chromosomal bins. Inner panel showing the depletion of horizontally acquired genes in the O bin of both slow- (Rf &#x2264; 1) and fast (Rf &gt; 1) growing bacteria." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1825183234" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosomal bin-based conservation of function. (A) Schematic representing equal sized bacterial chromosomal segments. Red: Origin bin (O); green: Right bin (R); blue: Terminus bin (T); black: Left bin (L). (B) Boxplot showing the enrichment of translation and ribosome biogenesis genes (COG category J) in O, R, T, and L chromosomal bins. (The dashed gray line at &lt;em&gt;y&lt;/em&gt; = 1.3 corresponds to a &lt;em&gt;P&lt;/em&gt;-value of 0.05. Values &#x2265;1.3 are statistically significant.). Inner panel shows the enrichment of COG J in the O bin of fast-growing bacteria (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;f&lt;/sub&gt; &gt; 1). (C) Boxplot showing the depletion of horizontally acquired genes in different chromosomal bins. Inner panel showing the depletion of horizontally acquired genes in the O bin of both slow- (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;f&lt;/sub&gt; &#x2264; 1) and fast (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;f&lt;/sub&gt; &gt; 1) growing bacteria.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1597/F2.medium.gif" width="174" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/6/1597/F2.medium.gif" width="174" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468330" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">Chromosomal bin-based conservation of function. (A) Schematic representing equal sized bacterial chromosomal segments. Red: Origin bin (O); green: Right bin (R); blue: Terminus bin (T); black: Left bin (L). (B) Boxplot showing the enrichment of translation and ribosome biogenesis genes (COG category J) in O, R, T, and L chromosomal bins. (The dashed gray line at <em>y</em> = 1.3 corresponds to a <em>P</em>-value of 0.05. Values ≥1.3 are statistically significant.). Inner panel shows the enrichment of COG J in the O bin of fast-growing bacteria (<em>R</em><sub>f</sub> &gt; 1). (C) Boxplot showing the depletion of horizontally acquired genes in different chromosomal bins. Inner panel showing the depletion of horizontally acquired genes in the O bin of both slow- (<em>R</em><sub>f</sub> ≤ 1) and fast (<em>R</em><sub>f</sub> &gt; 1) growing bacteria.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Analysis of gene functions, for each bin, at a broad level using the COG annotations revealed that the only statistically significant association between gene function and gene positioning was the enrichment of genes involved in translation and ribosome biogenesis in the O (origin) bin (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/7">Figure S7</a>). In line with previous literature (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Couturier and Rocha 2006</a>), we observed that the average distance of translation and ribosome biogenesis genes from <em>oriC</em> decreases with increase in <em>R</em><sub>f</sub> (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/8">Figure S8</a>). The enrichment of ribosome biogenesis and translation genes in the O segment was particularly apparent in fast-growing bacteria (<em>R</em><sub>f</sub> &gt; 1) but not in slower growing ones (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). In contrast, there was a significant depletion of predicted horizontally acquired genes in the O bin, in both fast- and slow-growing bacteria, and more so in the former (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). This is consistent with the increased chance of novel gene integration disrupting essential gene functions in this region of the chromosome. These results are also in agreement with observations made in <em>E. coli</em> (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Zarei <em>et al.</em> 2013</a>) wherein horizontally acquired genes were defined using methods different from what we have used here.</p></div><div id="sec-17" class="subsection"><h3>Long-range interreplichore translocations</h3><p id="p-40">The degree of conservation – defined as the proportion of genes with an ortholog in a closely related partner genome – decreased from the O bin toward the T bin for fast-growing organisms, with slow-growing organisms showing little effect (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/9">Figure S9</a>).</p><p id="p-41">We then asked whether genes present in a given segment of a chromosome would be conserved within the same segment in its closely related partner genome and whether this property would differ across the four segments defined here. We define <em>P</em><sub>i</sub> – the <em>gene position conservation</em> for segment i (where i stands for the four chromosomal segments O, T, R, or L) − as the proportion of genes in that segment, which are conserved in the corresponding segment in its partner genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The median <em>P</em><sub>O</sub> and <em>P</em><sub>T</sub>, for the O and the T bins, across our dataset of 262 genome pairs, was ∼80%, and the distributions of <em>P</em><sub>O</sub> and <em>P</em><sub>T</sub> were tight. In contrast, <em>P</em><sub>L</sub> and <em>P</em><sub>R</sub> showed a wide distribution, with the median around 50% (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamics of gene organization. (A) Schematic and boxplot showing the conservation of gene position (Pi) in O bin (O&#x2212;O), T bin (T&#x2212;T), R bin (R&#x2212;R), and L bin (L&#x2212;L). (B) Schematic representing O&#x2212;T and T&#x2212;O translocations (black arrows) and L&#x2212;R and R&#x2212;L translocations (gray arrows) and boxplot showing the proportion of these translocations (T(i,j))." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1825183234" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dynamics of gene organization. (A) Schematic and boxplot showing the conservation of gene position (&lt;em&gt;P&lt;/em&gt;&lt;sub&gt;i&lt;/sub&gt;) in O bin (O&#x2212;O), T bin (T&#x2212;T), R bin (R&#x2212;R), and L bin (L&#x2212;L). (B) Schematic representing O&#x2212;T and T&#x2212;O translocations (black arrows) and L&#x2212;R and R&#x2212;L translocations (gray arrows) and boxplot showing the proportion of these translocations (&lt;em&gt;T&lt;/em&gt;&lt;sub&gt;(i,j)&lt;/sub&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1597/F3.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/6/1597/F3.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468324" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Dynamics of gene organization. (A) Schematic and boxplot showing the conservation of gene position (<em>P</em><sub>i</sub>) in O bin (O−O), T bin (T−T), R bin (R−R), and L bin (L−L). (B) Schematic representing O−T and T−O translocations (black arrows) and L−R and R−L translocations (gray arrows) and boxplot showing the proportion of these translocations (<em>T</em><sub>(i,j)</sub>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Next, we asked whether a gene present in a particular bin in one organism translocates to the diametrically opposite bin (or a neighboring bin − <a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/10">Figure S10</a>) in the genome of a closely related organism. We called this <em>translocation rate</em> (<em>T</em><sub>i-j</sub>, where i and j indicate the segments between which the translocation rate is measured) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Consistent with our observation of high <em>P</em><sub>O</sub> and <em>P</em><sub>T</sub>, we found that <em>T</em><sub>O</sub><em><sub>−</sub></em><sub>T</sub> and <em>T</em><sub>T</sub><em><sub>−</sub></em><sub>O</sub> are very low with a median of ∼1% (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). This could be a possible consequence of selection acting against translocations that would drastically disrupt gene dosage, and is consistent with the observation of few contacts between <em>oriC</em> and <em>ter</em>.</p><p id="p-44">We next looked at translocations between the L and the R segments (<em>T</em><sub>L−R</sub> and <em>T</em><sub>R−L</sub>), which represent interreplichore translocations (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). For ∼42% of genome pairs, <em>T</em><sub>L−R/R−L</sub> &lt;0.1; for the remaining, <em>T</em><sub>L−R/R−L</sub> ∼0.45 (median). Therefore, in the majority of genome pairs compared here, genes present in one replichore in one genome are often present in the opposite replichore in its partner genome.</p><p id="p-45">Note that in identifying interreplichore translocations, by considering only the L and the R bins, we neglected 25% of each replichore immediately adjacent to <em>oriC</em> and <em>ter</em>. This was done to ensure that our observations were not skewed by previously reported high inversion rates immediately around <em>oriC</em> and <em>ter</em> (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Eisen <em>et al.</em> 2000</a>; <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Suyama and Bork 2001</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Tillier and Collins 2000</a>). Note, however, that the proportion of interreplichore translocations within the O and the T segments correlate well with that between the R and the L segments. Translocations appear to be more frequent in the <em>ter</em>-proximal than in the <em>oriC</em>-proximal half of the chromosome (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/11">Figure S11</a>). The difference in translocation rates between the <em>oriC</em>-proximal and the <em>ter</em>-proximal halves of a chromosome not only suggests a complementary strategy for studying long-range translocations, but also further corroborates the interplay of chromosome topology and gene organization with the process of replication.</p><p id="p-46">To ensure that the interreplichore translocations we describe are not merely a result of discrepancies in gene annotations, we predicted genes using Glimmer (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Delcher <em>et al.</em> 1999</a>) for all genomes included in this study. We compared predicted gene start positions with those reported in NCBI and found that the two datasets are in agreement with each other (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/12">Figure S12</a>).</p><p id="p-47">In summary, we show that interreplichore translocations between the L and the R bins are probably more common than could be perceived from the literature (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Eisen <em>et al.</em> 2000</a>; <a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Tillier and Collins 2000</a>). Further, the variation in the distribution of L−R translocation rates is only weakly explained by phylogenetic distance (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/13">Figure S13</a> and <a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/14">Figure S14</a>).</p></div><div id="sec-18" class="subsection"><h3>Interreplichore translocations as a possible readout of chromosome topology</h3><p id="p-48">Rates of recombination have been used as a proxy for measuring contact frequencies between distant regions of a chromosome. Together with the higher resolution analysis of contact frequencies by chromosome conformation capture methods and microscopy, these have so far underscored two broad chromosome topologies. First, in <em>E. coli</em> it has been shown that <em>oriC</em> is localized to the midcell and the left and the right arms of the chromosome stretch toward opposite poles with few interreplichore contacts (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Nielsen <em>et al.</em> 2006</a>; <a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Wang <em>et al.</em> 2006</a>). In line with this, we observed low interreplichore translocations (<em>T</em><sub>L−R/R−L</sub> ∼0.03) between <em>E. coli</em> and <em>Salmonella</em> (two closely related bacteria).</p><p id="p-49">In contrast, in <em>C. crescentus</em> (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>; <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Umbarger <em>et al.</em> 2011</a>) and <em>M. smegmatis</em> (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Santi and McKinney 2015</a>), <em>oriC</em> and <em>ter</em> are present at the two poles of the cell and the left and the right arms of the chromosome are in close contact with each other. Both these genomes showed high interreplichore translocations (<em>T</em><sub>L−R/R−L</sub> ∼0.50 and ∼0.70, respectively), consistent with their chromosome topology.</p><p id="p-50">Next, we analyzed the chromosome contact map from a previously published Hi-C experiment for <em>C. crescentus</em> (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>). Pairs of loci translocated between the two replichores displayed significantly higher contact frequencies in Hi-C experiments when compared to pairs of interreplichore loci not showing translocations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/15">Figure S15</a>). These indicate that high rates of translocation are localized to spatially proximal regions of the chromosome.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interreplichore translocations and chromosomal contact points. (A) Boxplot representing the normalized interreplichore contact frequencies as derived from Le et al. (2013) for C. crescentus NA1000 (NC_011916); for all the probed gene positions with or without interreplichore translocations (P-value &lt; 10&#x2212;10, Wilcoxon test). (B) Scatterplot representing 386 interreplichore (R &#x2192; L) translocations between C. crescentus NA1000 (NC_011916) and C. segnis (NC_014100) in red and randomized dataset (as described in Materials and Methods) in blue. Inner panel shows the distribution of Dinter = | dRi/Lj &#x2013; dLi/Rj | (solid line) for interreplichore translocations between C. crescentus NA1000 (NC_011916) and C. segnis (NC_014100). Dashed line indicates distribution of Dinter for randomized data (as described in Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1825183234" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interreplichore translocations and chromosomal contact points. (A) Boxplot representing the normalized interreplichore contact frequencies as derived from Le &lt;em&gt;et al.&lt;/em&gt; (2013) for &lt;em&gt;C. crescentus&lt;/em&gt; NA1000 (NC_011916); for all the probed gene positions with or without interreplichore translocations (&lt;em&gt;P&lt;/em&gt;-value &lt; 10&lt;sup&gt;&#x2212;10&lt;/sup&gt;, Wilcoxon test). (B) Scatterplot representing 386 interreplichore (R &#x2192; L) translocations between &lt;em&gt;C. crescentus&lt;/em&gt; NA1000 (NC_011916) and &lt;em&gt;C. segnis&lt;/em&gt; (NC_014100) in red and randomized dataset (as described in &lt;em&gt;Materials and Methods&lt;/em&gt;) in blue. Inner panel shows the distribution of &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;inter&lt;/sub&gt; = | &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;Ri/Lj&lt;/sub&gt; &#x2013; &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;Li/Rj&lt;/sub&gt; | (solid line) for interreplichore translocations between &lt;em&gt;C. crescentus&lt;/em&gt; NA1000 (NC_011916) and &lt;em&gt;C. segnis&lt;/em&gt; (NC_014100). Dashed line indicates distribution of &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;inter&lt;/sub&gt; for randomized data (as described in &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1597/F4.medium.gif" width="216" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/6/1597/F4.medium.gif" width="216" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468426" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-51" class="first-child">Interreplichore translocations and chromosomal contact points. (A) Boxplot representing the normalized interreplichore contact frequencies as derived from <a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Le <em>et al.</em> (2013</a>) for <em>C. crescentus</em> NA1000 (NC_011916); for all the probed gene positions with or without interreplichore translocations (<em>P</em>-value &lt; 10<sup>−10</sup>, Wilcoxon test). (B) Scatterplot representing 386 interreplichore (R → L) translocations between <em>C. crescentus</em> NA1000 (NC_011916) and <em>C. segnis</em> (NC_014100) in red and randomized dataset (as described in <em>Materials and Methods</em>) in blue. Inner panel shows the distribution of <em>D</em><sub>inter</sub> = | <em>d</em><sub>Ri/Lj</sub> – <em>d</em><sub>Li/Rj</sub> | (solid line) for interreplichore translocations between <em>C. crescentus</em> NA1000 (NC_011916) and <em>C. segnis</em> (NC_014100). Dashed line indicates distribution of <em>D</em><sub>inter</sub> for randomized data (as described in <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Many interreplichore translocations in <em>C. crescentus</em> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>) and in <em>M. smegmatis</em> (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/16">Figure S16A</a>) are symmetric, <em>i.e.</em>, a gene at a certain distance <em>d</em><sub>R</sub> from <em>oriC</em> in one replichore translocates to a position <em>d</em><sub>L</sub> in the other replichore such that <em>D</em><sub>inter</sub> <em>= |d</em><sub>R</sub> <em>− d</em><sub>L</sub><em>|</em> is close to zero. This is again consistent with the topology of these chromosomes, and also with the idea that a pair of concurrent replication forks, which are typically equidistant from <em>oriC</em>, are hotspots for recombination. Such symmetric translocations are true, not only for <em>C. crescentus</em> and <em>M. smegmatis</em>, but are a general feature of genomes with high <em>T</em><sub>L−R/R−</sub><em><sub>L</sub></em> values (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>), but less so for genome pairs with low <em>T</em><sub>L−R/R−L</sub> (<em>T</em><sub>L−R/R−L</sub> &lt; 0.1) (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/16">Figure S16B</a>). These are in line with the high ρ<sub>GOC</sub> values across most genome pairs tested here, indicating that interreplichore translocations rarely disrupt the relative distance of genes from <em>oriC</em>, and thereby maintain their dosage.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Selective value of chromosome topology. (A) Plot representing the distribution of Dinter = |dR/L &#x2013; dL/R| for all bacteria showing high interreplichore translocations (TL&#x2212;R/R&#x2212;L &gt; 0.1). Dashed line indicates the distribution of Dinter for randomized datasets (as described in Materials and Methods), similar to the inner panel of Figure 4B. Inner panel shows the percentage of gene pairs with Dinter &lt; 0.1 for all bacteria showing high interreplichore translocations, alongside random expectation (determined from randomized datasets as described in Materials and Methods). (B) Plot representing the proportion of leading-lagging strand flips post interreplichore translocations as a function of the proportion of interreplichore translocations (&#x3C1;Spearman = 0.17, P-value &lt; 10&#x2212;10) in cyan. A randomized dataset (as described in Materials and Methods) is shown in gray. Adjacent bloxplot represents the distribution of leading-lagging strand flips in bacteria with high and low interreplichore translocations alongside random expectation (Rm) for each dataset (D) (low TL&#x2212;R/R&#x2212;L, P-value = 2.9 &#xD7; 10&#x2212;6, Wilcoxon test and high TL&#x2212;R/R&#x2212;L, P-value &lt; 10&#x2212;10,Wilcoxon test). Asterisks indicate P-value &lt; 10&#x2212;3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1825183234" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Selective value of chromosome topology. (A) Plot representing the distribution of &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;inter&lt;/sub&gt; = |&lt;em&gt;d&lt;/em&gt;&lt;sub&gt;R/L&lt;/sub&gt; &#x2013; &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;L/R&lt;/sub&gt;| for all bacteria showing high interreplichore translocations (&lt;em&gt;T&lt;/em&gt;&lt;sub&gt;L&#x2212;R/R&#x2212;L&lt;/sub&gt; &gt; 0.1). Dashed line indicates the distribution of &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;inter&lt;/sub&gt; for randomized datasets (as described in &lt;em&gt;Materials and Methods&lt;/em&gt;), similar to the inner panel of Figure 4B. Inner panel shows the percentage of gene pairs with &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;inter&lt;/sub&gt; &lt; 0.1 for all bacteria showing high interreplichore translocations, alongside random expectation (determined from randomized datasets as described in &lt;em&gt;Materials and Methods&lt;/em&gt;). (B) Plot representing the proportion of leading-lagging strand flips post interreplichore translocations as a function of the proportion of interreplichore translocations (&#x3C1;&lt;sub&gt;Spearman&lt;/sub&gt; = 0.17, &lt;em&gt;P&lt;/em&gt;-value &lt; 10&lt;sup&gt;&#x2212;10&lt;/sup&gt;) in cyan. A randomized dataset (as described in &lt;em&gt;Materials and Methods&lt;/em&gt;) is shown in gray. Adjacent bloxplot represents the distribution of leading-lagging strand flips in bacteria with high and low interreplichore translocations alongside random expectation (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;m&lt;/sub&gt;) for each dataset (D) (low &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;L&#x2212;R/R&#x2212;L&lt;/sub&gt;, &lt;em&gt;P&lt;/em&gt;-value = 2.9 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;, Wilcoxon test and high &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;L&#x2212;R/R&#x2212;L&lt;/sub&gt;, &lt;em&gt;P&lt;/em&gt;-value &lt; 10&lt;sup&gt;&#x2212;10&lt;/sup&gt;,Wilcoxon test). Asterisks indicate &lt;em&gt;P&lt;/em&gt;-value &lt; 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1597/F5.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/6/1597/F5.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1597/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468493" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-53" class="first-child">Selective value of chromosome topology. (A) Plot representing the distribution of <em>D</em><sub>inter</sub> = |<em>d</em><sub>R/L</sub> – <em>d</em><sub>L/R</sub>| for all bacteria showing high interreplichore translocations (<em>T</em><sub>L−R/R−L</sub> &gt; 0.1). Dashed line indicates the distribution of <em>D</em><sub>inter</sub> for randomized datasets (as described in <em>Materials and Methods</em>), similar to the inner panel of <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>. Inner panel shows the percentage of gene pairs with <em>D</em><sub>inter</sub> &lt; 0.1 for all bacteria showing high interreplichore translocations, alongside random expectation (determined from randomized datasets as described in <em>Materials and Methods</em>). (B) Plot representing the proportion of leading-lagging strand flips post interreplichore translocations as a function of the proportion of interreplichore translocations (ρ<sub>Spearman</sub> = 0.17, <em>P</em>-value &lt; 10<sup>−10</sup>) in cyan. A randomized dataset (as described in <em>Materials and Methods</em>) is shown in gray. Adjacent bloxplot represents the distribution of leading-lagging strand flips in bacteria with high and low interreplichore translocations alongside random expectation (<em>R</em><sub>m</sub>) for each dataset (D) (low <em>T</em><sub>L−R/R−L</sub>, <em>P</em>-value = 2.9 × 10<sup>−6</sup>, Wilcoxon test and high <em>T</em><sub>L−R/R−L</sub>, <em>P</em>-value &lt; 10<sup>−10</sup>,Wilcoxon test). Asterisks indicate <em>P</em>-value &lt; 10<sup>−3</sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">We also found that interreplichore translocations do not impact strand bias in gene densities. We noticed that the translocated partners of leading strand genes tend to be encoded on the leading strand of the opposite replichore. This is more prominent for organisms showing high <em>T</em><sub>L−R/R−L</sub> as compared to low <em>T</em><sub>L−R/R−L</sub> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). This difference in strand maintenance between genomes with high and low <em>T</em><sub>L−R/R−L</sub> values might point to mechanistic differences between the two groups of organisms in the way these translocations occur.</p><p id="p-55">Finally, we observed that interreplichore translocations rarely break operons. However, the contiguity of genes belonging to the same functional category is often broken by these translocations (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/17">Figure S17</a>). In order to explicitly test the impact of these interreplichore translocations on gene expression states, we obtained publicly available microarray data for <em>Shewanella oneidensis MR1</em> (high <em>T</em><sub>L−R/R−L</sub>; <em>T</em><sub>L−R/R−L</sub> ∼0.56) across many conditions from the M3D database (<a href="http://m3d.mssm.edu">http://m3d.mssm.edu</a>). Analysis of these microarray data showed that genes present on different replichores show a higher correlation in expression (ρ<sub>Spearman</sub> &gt; 0.5) when they are equidistant from <em>oriC</em>. In contrast, gene pairs showing little or negative correlations between their gene expression profiles tend to show greater differences in their distance from <em>oriC</em>. Although the effect is very weak, the trend is statistically significant in terms of the difference between medians (<em>P</em>-value &lt; 10<sup>−10</sup>, Wilcoxon test; <a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/18">Figure S18</a>). This suggests that interreplichore translocations between genes equidistant from <em>oriC</em> may not disrupt their gene expression states.</p><p id="p-56">Thus, interreplichore translocations − potentially encouraged by the spatial proximity of the two replichores − neither disrupt gene dosage gradients down the <em>oriC</em>−<em>ter</em> axis nor do they impact strand biases in gene densities (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/22">Figure S22</a>); they may have only a small effect on gene expression states.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-57">Genome architecture broadly refers to gene organization and chromosome topology. While the wealth of genome sequence data provides valuable insights into gene organization, interrogation of chromosome topologies at high resolution requires additional experimental techniques involving chromosome conformation capture (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dekker <em>et al.</em> 2002</a>). These techniques were deployed on a genome-wide scale initially for eukaryotes and are now being used to study prokaryotic genome structures (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cagliero <em>et al.</em> 2013</a>; <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Marbouty <em>et al.</em> 2014</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2015</a>; <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Umbarger <em>et al.</em> 2011</a>). These studies have been limited so far to studying the three-dimensional chromosome architectures of a few model organisms. This currently precludes further analysis of associations between gene order and chromosome topologies.</p><p id="p-58">In this study, we have used a comparative genomic framework to indicate that large-scale translocations over short phylogenetic distances could be a proxy for chromosome topologies. This is consistent with the observation, in eukaryotes, that various cancer-causing translocations in the human genome involve loci with high contact frequencies in normal tissue (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Engreitz <em>et al.</em> 2012</a>). We notice that genomes with high interreplichore translocations may also experience high rates of intrareplichore recombination. Whether this implicates symmetric interreplichore chromosomal contacts in the maintenance of gene order is hard to address with confidence (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/19">Figure S19A</a>). It is however not clear as to what extent chromosome topology reflects phylogeny, as we observe a wide variation in the rates of interreplichore translocations within phylogenetic classes (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/20">Figure S20</a>). Additionally, we do not observe any association between growth rates and interreplichore translocation rates. And in light of our suggestion that high rates of interreplichore translocations may be predictive of a longitudinal chromosome topology, the growth rate of an organism may not determine its global chromosome topology (<a href="http://www.g3journal.org/cgi/data/g3.116.028274/DC1/21">Figure S21</a>).</p><p id="p-59">At this resolution, our analysis might predict whether the two replichores of a bacterial chromosome are spatially colocalized, but does not say anything about finer aspects of its structuring, such as macro- and microdomains. For example, there are only a few long-range translocations in the genome of <em>E. coli</em>; many of these appear to be located around <em>ter</em>, which is closely organized by the binding of the protein MatP (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dupaigne <em>et al.</em> 2012</a>) and where many targets of the nucleoid-associated protein H-NS are localized. Even the <em>C. crescentus</em> chromosome has local domain structures in the 20−400 kb size range (<a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Le <em>et al.</em> 2013</a>). However, these are not predicted by our translocation analysis.</p><p id="p-60">Another caveat of our study is that the dynamics of chromosome topology may not be predictable. For example, recent studies in <em>B. subtilis</em> have shown that its chromosome oscillates between the longitudinal and the transverse topologies (X. <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Wang <em>et al.</em> 2014b</a>). During replication <em>B. subtilis</em> is shown to have the <em>E. coli</em>-like transverse topology and the <em>C. crescentus</em>-like longitudinal topology for the rest of the cell cycle. However, <em>B. subtilis</em> shows interreplichore translocation frequency (∼0.02) similar to that of <em>E. coli</em>. This is consistent with the notion that concurrent replication forks are hotspots for recombination. Therefore, interreplichore translocations might occur during replication, and may be facilitated by close spatial contacts between the two replichores during replication. These observations hint toward the existence of a strong interplay between replication and chromosome topology. Hence, we suggest that the association between chromosome topologies and growth phenotypes, and its mechanistic and evolutionary underpinnings, should be explored in greater detail.</p><p id="p-61">Recent studies have shown that genome architecture and gene expression are linked in the following ways: (a) through differential positioning of genes with respect to <em>oriC</em>, popularly referred to as the gene dosage effect (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Block <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bryant <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Narula <em>et al.</em> 2015</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Schmid and Roth 1987</a>); (b) through spatial clustering of nonconsecutive genes, which might enforce transcriptional coherence (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cabrera and Jin 2003</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Junier <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Qian <em>et al.</em> 2012</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2011</a>); (c) through preferential binding of bacterial nucleoid-associated proteins like H-NS, which show position and gene feature-specific binding to the chromosome (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Tupper <em>et al.</em> 1994</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Weng and Xiao 2014</a>); and (d) through the effect of proposed gradients of DNA superhelicity on the chromosome on gene expression (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lal <em>et al.</em> 2016</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Sobetzko <em>et al.</em> 2012</a>). However, despite our attempts to interrogate this using a compendium of microarray data for one organism, the impact of interreplichore translocations on gene expression states remains open. Along this direction, it remains to be seen whether genomes showing high interreplichore translocations would also display symmetry in nucleoid organization between the two replichores; this could also serve to minimize the impact of such translocations on gene expression. Detailed analyses of the impact of replication on gene expression and chromosome topologies are also essential, in the light of evidence that <em>oriC</em> might itself be a selfish element (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hawkins <em>et al.</em> 2013</a>), and the ability of bacteria to undergo dispersed replication events independent of <em>oriC</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bernander <em>et al.</em> 1991</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dimude <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gowrishankar 2015</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koppes 1992</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McGeoch and Bell 2008</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We thank Sreenivasulu Kurukuti (University of Hyderabad, India), Sunish Radhakrishnan (IISER, Trivandrum, India), Marco Cosentino Lagomarsino (Universite Pierre et Marie Curie, France), Romain Koszul (Institut Pasteur, France), Axel Cournac (Institut Pasteur, France), and Vittore Scolari (Institut Pasteur, France) for in-depth discussions. We appreciate and thank the two anonymous reviewers for their valuable suggestions and comments on the manuscript. S.K. is supported by a fellowship from the Council for Scientific and Industrial Research, India [(09/860(0122)/2011-EMR-I)]. A.S.N.S. is funded by a Ramanujan Fellowship (SR/S2/RJN-49/2010) from the Department of Science and Technology, Government of India; Indo–French Centre for the Promotion of Advanced Research (IFCPAR/CEFIPRA) grant (5103-3) and National Centre for Biological Sciences (NCBS) core budget.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-63">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028274/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028274/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-64"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-02-19"><span class="received-label">Received </span>February 19, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-23"><span class="accepted-label">Accepted </span>March 23, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Khedkar and Seshasayee</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.1" data-doi="10.1016/0092-8674(91)90269-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernander</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasgupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordström</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The E. coli cell cycle and the plasmid R1 replication cycle in the absence of the DnaA protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1145</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBernander%26rft.auinit1%253DR.%26rft.volume%253D64%26rft.issue%253D6%26rft.spage%253D1145%26rft.epage%253D1153%26rft.atitle%253DThe%2BE.%2Bcoli%2Bcell%2Bcycle%2Band%2Bthe%2Bplasmid%2BR1%2Breplication%2Bcycle%2Bin%2Bthe%2Babsence%2Bof%2Bthe%2BDnaA%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990269-5%26rft_id%253Dinfo%253Apmid%252F1848482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90269-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1848482&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FD55800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.2" data-doi="10.1093/nar/gks694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">D. H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">H. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulatory consequences of gene translocation in bacteria.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">8979</span>–<span class="cit-lpage">8992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks694%26rft_id%253Dinfo%253Apmid%252F22833608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC8xOC84OTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.3" data-doi="10.1016/j.mib.2010.09.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grainger</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busby</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Effects of nucleoid-associated proteins on bacterial chromosome structure and gene expression.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DBrowning%26rft.auinit1%253DD.%2BF.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D773%26rft.epage%253D780%26rft.atitle%253DEffects%2Bof%2Bnucleoid-associated%2Bproteins%2Bon%2Bbacterial%2Bchromosome%2Bstructure%2Band%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2010.09.013%26rft_id%253Dinfo%253Apmid%252F20951079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2010.09.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20951079&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285657400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.4" data-doi="10.1093/nar/gku828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sellars</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busby</span>  <span class="cit-name-given-names">S. J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Chromosome position effects on gene expression in Escherichia coli K-12.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">11383</span>–<span class="cit-lpage">11392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku828%26rft_id%253Dinfo%253Apmid%252F25209233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi8xOC8xMTM4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.5" data-doi="10.1046/j.1365-2958.2003.03805.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DCabrera%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D50%26rft.issue%253D5%26rft.spage%253D1493%26rft.epage%253D1505%26rft.atitle%253DThe%2Bdistribution%2Bof%2BRNA%2Bpolymerase%2Bin%2BEscherichia%2Bcoli%2Bis%2Bdynamic%2Band%2Bsensitive%2Bto%2Benvironmental%2Bcues.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03805.x%26rft_id%253Dinfo%253Apmid%252F14651633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03805.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14651633&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186698600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.6" data-doi="10.1093/nar/gkt325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cagliero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grand</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome conformation capture reveals that the Escherichia coli chromosome is organized by replication and transcription.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">6058</span>–<span class="cit-lpage">6071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt325%26rft_id%253Dinfo%253Apmid%252F23632166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8xMi82MDU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.7" data-doi="10.1128/jb.177.14.4152-4156.1995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Condon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liveris</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squires</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squires</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">rRNA operon multiplicity in Escherichia coli and the physiological implications of rrn inactivation.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">4152</span>–<span class="cit-lpage">4156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DCondon%26rft.auinit1%253DC.%26rft.volume%253D177%26rft.issue%253D14%26rft.spage%253D4152%26rft.epage%253D4156%26rft.atitle%253DrRNA%2Boperon%2Bmultiplicity%2Bin%2BEscherichia%2Bcoli%2Band%2Bthe%2Bphysiological%2Bimplications%2Bof%2Brrn%2Binactivation.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.177.14.4152-4156.1995%26rft_id%253Dinfo%253Apmid%252F7608093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ny8xNC80MTUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.8" data-doi="10.1016/0022-2836(68)90425-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmstetter</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Chromosome replication and the division cycle of Escherichia coli.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCooper%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D519%26rft.epage%253D540%26rft.atitle%253DChromosome%2Breplication%2Band%2Bthe%2Bdivision%2Bcycle%2Bof%2BEscherichia%2Bcoli%2BB%252Fr.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252868%252990425-7%26rft_id%253Dinfo%253Apmid%252F4866337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(68)90425-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4866337&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1968A763700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.9" data-doi="10.1111/j.1365-2958.2006.05046.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Couturier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Replication-associated gene dosage effects shape the genomes of fast-growing bacteria but only for transcription and translation genes.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">1506</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DCouturier%26rft.auinit1%253DE.%26rft.volume%253D59%26rft.issue%253D5%26rft.spage%253D1506%26rft.epage%253D1518%26rft.atitle%253DReplication-associated%2Bgene%2Bdosage%2Beffects%2Bshape%2Bthe%2Bgenomes%2Bof%2Bfast-growing%2Bbacteria%2Bbut%2Bonly%2Bfor%2Btranscription%2Band%2Btranslation%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05046.x%26rft_id%253Dinfo%253Apmid%252F16468991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05046.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16468991&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235220200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.10" data-doi="10.1038/nature05283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dame</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noom</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wuite</span>  <span class="cit-name-given-names">G. J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bacterial chromatin organization by H-NS protein unravelled using dual DNA manipulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDame%26rft.auinit1%253DR.%2BT.%26rft.volume%253D444%26rft.issue%253D7117%26rft.spage%253D387%26rft.epage%253D390%26rft.atitle%253DBacterial%2Bchromatin%2Borganization%2Bby%2BH-NS%2Bprotein%2Bunravelled%2Busing%2Bdual%2BDNA%2Bmanipulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05283%26rft_id%253Dinfo%253Apmid%252F17108966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17108966&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242018300053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.11" data-doi="10.1126/science.1067799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rippe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Capturing chromosome conformation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">295</span>: <span class="cit-fpage">1306</span>–<span class="cit-lpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDekker%26rft.auinit1%253DJ.%26rft.volume%253D295%26rft.issue%253D5558%26rft.spage%253D1306%26rft.epage%253D1311%26rft.atitle%253DCapturing%2BChromosome%2BConformation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1067799%26rft_id%253Dinfo%253Apmid%252F11847345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTUvNTU1OC8xMzA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.12" data-doi="10.1093/nar/27.23.4636"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harmon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasif</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Improved microbial gene identification with GLIMMER.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">4636</span>–<span class="cit-lpage">4641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.23.4636%26rft_id%253Dinfo%253Apmid%252F10556321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8yMy80NjM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.13" data-doi="10.1038/nrmicro2261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bacterial nucleoid-associated proteins, nucleoid structure and gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDillon%26rft.auinit1%253DS.%2BC.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D185%26rft.epage%253D195%26rft.atitle%253DBacterial%2Bnucleoid-associated%2Bproteins%252C%2Bnucleoid%2Bstructure%2Band%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2261%26rft_id%253Dinfo%253Apmid%252F20140026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20140026&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274565400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dimude</span>  <span class="cit-name-given-names">J. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockum</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Midgley-Smith</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The consequences of replicating in the wrong orientation: bacterial chromosome duplication without an active replication origin.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e01294</span>–<span class="cit-lpage">e15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D6%26rft.spage%253De01294%26rft_id%253Dinfo%253Apmid%252F26530381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26530381&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.15" data-doi="10.1016/0092-8674(82)90403-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DiNardo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voelkel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternglanz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Escherichia coli DNA topoisomerase I mutants have compensatory mutations in DNA gyrase genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDiNardo%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D51%26rft.atitle%253DEscherichia%2Bcoli%2BDNA%2Btopoisomerase%2BI%2Bmutants%2Bhave%2Bcompensatory%2Bmutations%2Bin%2BDNA%2Bgyrase%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252882%252990403-2%26rft_id%253Dinfo%253Apmid%252F6297752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(82)90403-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6297752&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PQ46100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.16" data-doi="10.1038/nrmicro3007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome architecture and global gene regulation in bacteria: making progress towards a unified model?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D11%26rft.spage%253D349%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3007%26rft_id%253Dinfo%253Apmid%252F23549066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23549066&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.17" data-doi="10.1016/j.molcel.2012.09.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dupaigne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonthat</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espéli</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitfill</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boccard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular basis for a protein-mediated DNA-bridging mechanism that functions in condensation of the E. coli chromosome.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">560</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDupaigne%26rft.auinit1%253DP.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D560%26rft.epage%253D571%26rft.atitle%253DMolecular%2BBasis%2Bfor%2Ba%2BProtein-Mediated%2BDNA-Bridging%2BMechanism%2Bthat%2BFunctions%2Bin%2BCondensation%2Bof%2Bthe%2BE.%2Bcoli%2BChromosome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.09.009%26rft_id%253Dinfo%253Apmid%252F23084832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.09.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23084832&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311919500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.18" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">J.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heidelberg</span>  <span class="cit-name-given-names">J.F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.L.</span></span></li></ol><cite> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Evidence for symmetric chromosomal inversions around the replication origin in bacteria.</span> <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-vol">1</span>: RESEARCH0011.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.20" data-doi="10.1371/journal.pone.0044196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engreitz</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirny</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Three-dimensional genome architecture influences partner selection for chromosomal translocations in human disease.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e44196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEngreitz%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De44196%26rft.epage%253De44196%26rft.atitle%253DThree-dimensional%2Bgenome%2Barchitecture%2Binfluences%2Bpartner%2Bselection%2Bfor%2Bchromosomal%2Btranslocations%2Bin%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0044196%26rft_id%253Dinfo%253Apmid%252F23028501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0044196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028501&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.21" data-doi="10.1371/journal.pgen.0030226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esnault</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espéli</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boccard</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Chromosome Structuring Limits Genome Plasticity in Escherichia coli.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DEsnault%26rft.auinit1%253DE.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De226%26rft.epage%253De226%26rft.atitle%253DChromosome%2Bstructuring%2Blimits%2Bgenome%2Bplasticity%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030226%26rft_id%253Dinfo%253Apmid%252F18085828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18085828&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.22" data-doi="10.1128/JB.185.10.3076-3080.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esposito</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Escherichia coli Fis protein stimulates bacteriophage integrative recombination in vitro.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">3076</span>–<span class="cit-lpage">3080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DEsposito%26rft.auinit1%253DD.%26rft.volume%253D185%26rft.issue%253D10%26rft.spage%253D3076%26rft.epage%253D3080%26rft.atitle%253DThe%2BEscherichia%2Bcoli%2BFis%2BProtein%2BStimulates%2BBacteriophage%2B%257Blambda%257D%2BIntegrative%2BRecombination%2BIn%2BVitro%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.185.10.3076-3080.2003%26rft_id%253Dinfo%253Apmid%252F12730167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NS8xMC8zMDc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.23" data-doi="10.1093/nar/gkr367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">HMMER web server: interactive sequence similarity searching.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">W29</span>–<span class="cit-lpage">W37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr367%26rft_id%253Dinfo%253Apmid%252F21593126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzOS9zdXBwbF8yL1cyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.24" data-doi="10.1186/gb-2009-10-6-r61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freilich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreimer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borenstein</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yosef</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Metabolic-network-driven analysis of bacterial ecological strategies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFreilich%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253DR61%26rft.epage%253DR61%26rft.atitle%253DMetabolic-network-driven%2Banalysis%2Bof%2Bbacterial%2Becological%2Bstrategies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-6-r61%26rft_id%253Dinfo%253Apmid%252F19500338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-6-r61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19500338&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.25" data-doi="10.1093/bioinformatics/btm255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.-T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">DoriC: a database of oriC regions in bacterial genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1866</span>–<span class="cit-lpage">1867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm255%26rft_id%253Dinfo%253Apmid%252F17496319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTQvMTg2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.26" data-doi="10.1371/journal.pgen.1004909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gowrishankar</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">End of the beginning: elongation and termination features of alternative modes of chromosomal replication initiation in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004909%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004909%26rft_id%253Dinfo%253Apmid%252F25569209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25569209&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.27" data-doi="10.1016/j.cub.2013.12.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veening</span>  <span class="cit-name-given-names">J.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blettinger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bramkamp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Interlinked sister chromosomes arise in the absence of Condensin during fast replication in B. subtilis.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.12.049%26rft_id%253Dinfo%253Apmid%252F24440399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.12.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24440399&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.28" data-doi="10.1038/nature12650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blythe</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accelerated growth in the absence of DNA replication origins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>: <span class="cit-fpage">544</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D544%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12650%26rft_id%253Dinfo%253Apmid%252F24185008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24185008&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.29" data-doi="10.1016/j.jmb.2012.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Junier</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hérisson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Képès</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic organization of evolutionarily correlated genes in bacteria: limits and strategies.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DJunier%26rft.auinit1%253DI.%26rft.volume%253D419%26rft.issue%253D5%26rft.spage%253D369%26rft.epage%253D386%26rft.atitle%253DGenomic%2Borganization%2Bof%2Bevolutionarily%2Bcorrelated%2Bgenes%2Bin%2Bbacteria%253A%2Blimits%2Band%2Bstrategies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2012.03.009%26rft_id%253Dinfo%253Apmid%252F22446685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2012.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22446685&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.30" data-doi="10.1093/nar/gkt1005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Junier</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boccard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espeli</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Polymer modeling of the E. coli genome reveals the involvement of locus positioning and macrodomain structuring for the control of chromosome conformation and segregation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1461</span>–<span class="cit-lpage">1473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1005%26rft_id%253Dinfo%253Apmid%252F24194594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzMvMTQ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.31" data-doi="10.1128/AEM.66.4.1328-1333.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klappenbach</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunbar</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">rRNA operon copy number reflects ecological strategies of bacteria.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1328</span>–<span class="cit-lpage">1333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DKlappenbach%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D66%26rft.issue%253D4%26rft.spage%253D1328%26rft.epage%253D1333%26rft.atitle%253DrRNA%2BOperon%2BCopy%2BNumber%2BReflects%2BEcological%2BStrategies%2Bof%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.66.4.1328-1333.2000%26rft_id%253Dinfo%253Apmid%252F10742207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY2LzQvMTMyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.32" data-doi="10.1093/nar/gkn668"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">6688</span>–<span class="cit-lpage">6719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn668%26rft_id%253Dinfo%253Apmid%252F18948295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8yMS82Njg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.33" data-doi="10.1128/jb.174.7.2121-2123.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koppes</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Nonrandom F-plasmid replication in Escherichia coli K-12.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">2121</span>–<span class="cit-lpage">2123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKoppes%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D174%26rft.issue%253D7%26rft.spage%253D2121%26rft.epage%253D2123%26rft.atitle%253DNonrandom%2BF-plasmid%2Breplication%2Bin%2BEscherichia%2Bcoli%2BK-12.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.7.2121-2123.1992%26rft_id%253Dinfo%253Apmid%252F1551835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE3NC83LzIxMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.34" data-doi="10.1038/ncomms11055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trostel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kouzine</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seshasayee</span>  <span class="cit-name-given-names">A. S. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome scale patterns of supercoiling in a bacterial chromosome</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">7</span>: <span class="cit-fpage">11055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D7%26rft.spage%253D11055%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11055%26rft_id%253Dinfo%253Apmid%252F27025941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27025941&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.35" data-doi="10.1016/j.febslet.2015.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagomarsino</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espéli</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junier</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">From structure to function of bacterial chromosomes: Evolutionary perspectives and ideas for new experiments.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">589</span>: <span class="cit-fpage">2996</span>–<span class="cit-lpage">3004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLett.%26rft.volume%253D589%26rft.spage%253D2996%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2015.07.002%26rft_id%253Dinfo%253Apmid%252F26171924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2015.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26171924&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.36" data-doi="10.1016/j.mib.2014.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laub</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">New approaches to understanding the spatial organization of bacterial genomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D22%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2014.09.014%26rft_id%253Dinfo%253Apmid%252F25305533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2014.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25305533&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.37" data-doi="10.1126/science.1242059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">T. B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imakaev</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirny</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laub</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of the spatial organization of a bacterial chromosome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1242059%26rft_id%253Dinfo%253Apmid%252F24158908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1OS83MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.38" data-doi="10.1093/nar/gkr201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letunic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interactive Tree Of Life v2: online annotation and display of phylogenetic trees made easy.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">W475</span>–<span class="cit-lpage">W478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr201%26rft_id%253Dinfo%253Apmid%252F21470960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8yL1c0NzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.39" data-doi="10.1101/gad.1883510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenney</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A divalent switch drives H-NS/DNA-binding conformations between stiffening and bridging modes.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D339%26rft.epage%253D344%26rft.atitle%253DA%2Bdivalent%2Bswitch%2Bdrives%2BH-NS%252FDNA-binding%2Bconformations%2Bbetween%2Bstiffening%2Band%2Bbridging%2Bmodes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1883510%26rft_id%253Dinfo%253Apmid%252F20159954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjQvNC8zMzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.40" data-doi="10.7554/eLife.03318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marbouty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cournac</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flot</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marie-Nelly</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mozziconacci</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e03318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03318%26rft_id%253Dinfo%253Apmid%252F25517076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjMvMC9lMDMzMTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.41" data-doi="10.1016/j.molcel.2015.07.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marbouty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattoni</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cournac</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Condensin- and replication-mediated bacterial chromosome folding and origin condensation revealed by Hi-C and super-resolution imaging.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D59%26rft.spage%253D588%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.07.020%26rft_id%253Dinfo%253Apmid%252F26295962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.07.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26295962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.42" data-doi="10.1016/j.jmb.2009.02.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muskhelishvili</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A systematic in vitro study of nucleoprotein complexes formed by bacterial nucleoid-associated proteins revealing novel types of DNA organization.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMaurer%26rft.auinit1%253DS.%26rft.volume%253D387%26rft.issue%253D5%26rft.spage%253D1261%26rft.epage%253D1276%26rft.atitle%253DA%2Bsystematic%2Bin%2Bvitro%2Bstudy%2Bof%2Bnucleoprotein%2Bcomplexes%2Bformed%2Bby%2Bbacterial%2Bnucleoid-associated%2Bproteins%2Brevealing%2Bnovel%2Btypes%2Bof%2BDNA%2Borganization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.02.050%26rft_id%253Dinfo%253Apmid%252F19254726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.02.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19254726&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265501000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.43" data-doi="10.1038/nrm2426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGeoch</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Extra-chromosomal elements and the evolution of cellular DNA replication machineries.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DMcGeoch%26rft.auinit1%253DA.%2BT.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D569%26rft.epage%253D574%26rft.atitle%253DExtra-chromosomal%2Belements%2Band%2Bthe%2Bevolution%2Bof%2Bcellular%2BDNA%2Breplication%2Bmachineries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2426%26rft_id%253Dinfo%253Apmid%252F18523437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18523437&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257073300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.44" data-doi="10.1016/j.cell.2008.08.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schbath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El Karoui</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The MatP/matS site-specific system organizes the terminus region of the E. coli chromosome into a macrodomain.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMercier%26rft.auinit1%253DR.%26rft.volume%253D135%26rft.issue%253D3%26rft.spage%253D475%26rft.epage%253D485%26rft.atitle%253DThe%2BMatP%252FmatS%2Bsite-specific%2Bsystem%2Borganizes%2Bthe%2Bterminus%2Bregion%2Bof%2Bthe%2BE.%2Bcoli%2Bchromosome%2Binto%2Ba%2Bmacrodomain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.08.031%26rft_id%253Dinfo%253Apmid%252F18984159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.08.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18984159&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260536300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.45" data-doi="10.1016/S1369-5274(02)00358-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mira</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klasson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S. G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Microbial genome evolution: sources of variability.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">506</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DMira%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D506%26rft.epage%253D512%26rft.atitle%253DMicrobial%2Bgenome%2Bevolution%253A%2Bsources%2Bof%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5274%252802%252900358-2%26rft_id%253Dinfo%253Apmid%252F12354559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5274(02)00358-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12354559&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178901700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.46" data-doi="10.1128/MCB.25.3.888-895.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of transcription-replication collisions in bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">888</span>–<span class="cit-lpage">895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMirkin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D888%26rft.epage%253D895%26rft.atitle%253DMechanisms%2Bof%2BTranscription-Replication%2BCollisions%2Bin%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.3.888-895.2005%26rft_id%253Dinfo%253Apmid%252F15657418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvODg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.47" data-doi="10.1016/j.cell.2015.06.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narula</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuchina</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Süel</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromosomal arrangement of phosphorelay genes couples sporulation and DNA replication.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D162%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.06.012%26rft_id%253Dinfo%253Apmid%252F26165942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.06.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26165942&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.48" data-doi="10.1016/0022-2836(70)90057-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Needleman</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wunsch</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">A general method applicable to the search for similarities in the amino acid sequence of two proteins.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNeedleman%26rft.auinit1%253DS.%2BB.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D443%26rft.epage%253D453%26rft.atitle%253DA%2Bgeneral%2Bmethod%2Bapplicable%2Bto%2Bthe%2Bsearch%2Bfor%2Bsimilarities%2Bin%2Bthe%2Bamino%2Bacid%2Bsequence%2Bof%2Btwo%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252870%252990057-4%26rft_id%253Dinfo%253Apmid%252F5420325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(70)90057-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5420325&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970F961100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.49" data-doi="10.1111/j.1365-2958.2006.05346.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ottesen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Youngren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">F. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The Escherichia coli chromosome is organized with the left and right chromosome arms in separate cell halves.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DNielsen%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D331%26rft.epage%253D338%26rft.atitle%253DThe%2BEscherichia%2Bcoli%2Bchromosome%2Bis%2Borganized%2Bwith%2Bthe%2Bleft%2Band%2Bright%2Bchromosome%2Barms%2Bin%2Bseparate%2Bcell%2Bhalves.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05346.x%26rft_id%253Dinfo%253Apmid%252F17020576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05346.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17020576&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240834000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.50" data-doi="10.1016/0092-8674(82)90402-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruss</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drlica</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Escherichia coli DNA topoisomerase I mutants: increased supercoiling is corrected by mutations near gyrase genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPruss%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D42%26rft.atitle%253DEscherichia%2Bcoli%2BDNA%2Btopoisomerase%2BI%2Bmutants%253A%2Bincreased%2Bsupercoiling%2Bis%2Bcorrected%2Bby%2Bmutations%2Bnear%2Bgyrase%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252882%252990402-0%26rft_id%253Dinfo%253Apmid%252F6297751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(82)90402-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6297751&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PQ46100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.51" data-doi="10.1073/pnas.1208595109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimitriadis</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eswaramoorthy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adhya</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Galactose repressor mediated intersegmental chromosomal connections in Escherichia coli.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">11336</span>–<span class="cit-lpage">11341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1208595109%26rft_id%253Dinfo%253Apmid%252F22733746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzI4LzExMzM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.52" data-doi="10.1128/mBio.00998-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macvanin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimitriadis</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhurkin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A new noncoding RNA arranges bacterial chromosome organization.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00998-15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00998-15%26rft_id%253Dinfo%253Apmid%252F26307168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiNi80L2UwMDk5OC0xNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.53" data-doi="10.1016/j.cell.2008.01.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Lamothe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Possoz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danilova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherratt</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Independent positioning and action of Escherichia coli replisomes in live cells.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DReyes-Lamothe%26rft.auinit1%253DR.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D90%26rft.epage%253D102%26rft.atitle%253DIndependent%2Bpositioning%2Band%2Baction%2Bof%2BEscherichia%2Bcoli%2Breplisomes%2Bin%2Blive%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.01.044%26rft_id%253Dinfo%253Apmid%252F18394992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.01.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18394992&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254641300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.54" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleasby</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">EMBOSS: the European Molecular Biology Open Software Suite.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%2BTIG%26rft.volume%253D16%26rft.spage%253D276%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.55" data-doi="10.1099/mic.0.26974-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The replication-related organization of bacterial genomes.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DRocha%26rft.auinit1%253DE.%2BP.%2BC.%26rft.volume%253D150%26rft.issue%253D6%26rft.spage%253D1609%26rft.epage%253D1627%26rft.atitle%253DThe%2Breplication-related%2Borganization%2Bof%2Bbacterial%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.26974-0%26rft_id%253Dinfo%253Apmid%252F15184548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.26974-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15184548&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222437300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.56" data-doi="10.1146/annurev.genet.42.110807.091653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The organization of the bacterial genome.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DRocha%26rft.auinit1%253DE.%2BP.%26rft.volume%253D42%26rft.spage%253D211%26rft.epage%253D233%26rft.atitle%253DThe%2Borganization%2Bof%2Bthe%2Bbacterial%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091653%26rft_id%253Dinfo%253Apmid%252F18605898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18605898&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.57" data-doi="10.1093/nar/gkg859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gene essentiality determines chromosome organisation in bacteria.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">6570</span>–<span class="cit-lpage">6577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DRocha%26rft.auinit1%253DE.%2BP.%2BC.%26rft.volume%253D31%26rft.issue%253D22%26rft.spage%253D6570%26rft.epage%253D6577%26rft.atitle%253DGene%2Bessentiality%2Bdetermines%2Bchromosome%2Borganisation%2Bin%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg859%26rft_id%253Dinfo%253Apmid%252F14602916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8yMi82NTcwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.58" data-doi="10.1128/mBio.01999-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromosome organization and replisome dynamics in Mycobacterium smegmatis.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e01999</span>–<span class="cit-lpage">e14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D6%26rft.spage%253De01999%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01999-14%26rft_id%253Dinfo%253Apmid%252F25691587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01999-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25691587&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.59" data-doi="10.1128/jb.169.6.2872-2875.1987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Gene location affects expression level in Salmonella typhimurium.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2872</span>–<span class="cit-lpage">2875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSchmid%26rft.auinit1%253DM.%2BB.%26rft.volume%253D169%26rft.issue%253D6%26rft.spage%253D2872%26rft.epage%253D2875%26rft.atitle%253DGene%2Blocation%2Baffects%2Bexpression%2Blevel%2Bin%2BSalmonella%2Btyphimurium.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.169.6.2872-2875.1987%26rft_id%253Dinfo%253Apmid%252F3294809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE2OS82LzI4NzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.60" data-doi="10.1007/s004380000399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shanado</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Suppression of gamma ray-induced illegitimate recombination in Escherichia coli by the DNA-binding protein H-NS.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DShanado%26rft.auinit1%253DY.%26rft.volume%253D265%26rft.issue%253D2%26rft.spage%253D242%26rft.epage%253D248%26rft.atitle%253DSuppression%2Bof%2Bgamma%2Bray-induced%2Billegitimate%2Brecombination%2Bin%2BEscherichia%2Bcoli%2Bby%2Bthe%2BDNA-binding%2Bprotein%2BH-NS.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004380000399%26rft_id%253Dinfo%253Apmid%252F11361334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004380000399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11361334&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169677300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.61" data-doi="10.1073/pnas.1108229109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sobetzko</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muskhelishvili</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E42</span>–<span class="cit-lpage">E50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1108229109%26rft_id%253Dinfo%253Apmid%252F22184251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDkvMi9FNDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.62" data-doi="10.1101/gad.1900610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cascio</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The shape of the DNA minor groove directs binding by the DNA-bending protein Fis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">814</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DStella%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D814%26rft.epage%253D826%26rft.atitle%253DThe%2Bshape%2Bof%2Bthe%2BDNA%2Bminor%2Bgroove%2Bdirects%2Bbinding%2Bby%2Bthe%2BDNA-bending%2Bprotein%2BFis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1900610%26rft_id%253Dinfo%253Apmid%252F20395367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjQvOC84MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.63" data-doi="10.1016/S0168-9525(00)02159-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evolution of prokaryotic gene order: genome rearrangements in closely related species.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%2BTIG%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902159-4%26rft_id%253Dinfo%253Apmid%252F11163906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02159-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11163906&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamames</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Evolution of gene order conservation in prokaryotes</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-vol">2</span>: RESEARCH0020.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.65" data-doi="10.1126/science.1071278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamas</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">50 Million Years of Genomic Stasis in Endosymbiotic Bacteria</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">2376</span>–<span class="cit-lpage">2379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTamas%26rft.auinit1%253DI.%26rft.volume%253D296%26rft.issue%253D5577%26rft.spage%253D2376%26rft.epage%253D2379%26rft.atitle%253D50%2BMillion%2BYears%2Bof%2BGenomic%2BStasis%2Bin%2BEndosymbiotic%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1071278%26rft_id%253Dinfo%253Apmid%252F12089438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTYvNTU3Ny8yMzc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.6.1597.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DTamura%26rft.auinit1%253DK%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D512%26rft.epage%253D526%26rft.atitle%253DEstimation%2Bof%2Bthe%2Bnumber%2Bof%2Bnucleotide%2Bsubstitutions%2Bin%2Bthe%2Bcontrol%2Bregion%2Bof%2Bmitochondrial%2BDNA%2Bin%2Bhumans%2Band%2Bchimpanzees%26rft_id%253Dinfo%253Apmid%252F8336541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEwLzMvNTEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.67" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMi8yNzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.68" data-doi="10.1038/79918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tillier</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome rearrangement by replication-directed translocation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTillier%26rft.auinit1%253DE.%2BR.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D195%26rft.epage%253D197%26rft.atitle%253DGenome%2Brearrangement%2Bby%2Breplication-directed%2Btranslocation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F79918%26rft_id%253Dinfo%253Apmid%252F11017076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/79918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11017076&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089638800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tupper</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owen-Hughes</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ussery</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The chromatin-associated protein H-NS alters DNA topology in vitro.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D13%26rft.spage%253D258%26rft_id%253Dinfo%253Apmid%252F8306968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8306968&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MV81000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.70" data-doi="10.1016/j.molcel.2011.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umbarger</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porreca</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baù</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The three-dimensional architecture of a bacterial genome and its alteration by genetic perturbation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DUmbarger%26rft.auinit1%253DM.%2BA.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D252%26rft.epage%253D264%26rft.atitle%253DThe%2Bthree-dimensional%2Barchitecture%2Bof%2Ba%2Bbacterial%2Bgenome%2Band%2Bits%2Balteration%2Bby%2Bgenetic%2Bperturbation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.09.010%26rft_id%253Dinfo%253Apmid%252F22017872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22017872&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296212100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.71" data-doi="10.1038/sj.emboj.7600434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boccard</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Macrodomain organization of the Escherichia coli chromosome.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4330</span>–<span class="cit-lpage">4341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DValens%26rft.auinit1%253DM.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D4330%26rft.epage%253D4341%26rft.atitle%253DMacrodomain%2Borganization%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bchromosome%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600434%26rft_id%253Dinfo%253Apmid%252F15470498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvNDMzMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.72" data-doi="10.1093/bioinformatics/btl369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vernikos</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhill</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Interpolated variable order motifs for identification of horizontally acquired DNA: revisiting the Salmonella pathogenicity islands.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2196</span>–<span class="cit-lpage">2203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl369%26rft_id%253Dinfo%253Apmid%252F16837528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTgvMjE5NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.73" data-doi="10.1371/journal.pgen.1000808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vieira-Silva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The systemic imprint of growth and its uses in ecological (meta)genomics.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVieira-Silva%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000808%26rft.epage%253De1000808%26rft.atitle%253DThe%2Bsystemic%2Bimprint%2Bof%2Bgrowth%2Band%2Bits%2Buses%2Bin%2Becological%2B%2528meta%2529genomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000808%26rft_id%253Dinfo%253Apmid%252F20090831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20090831&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Exploring the mechanisms of genome-wide long-range interactions: interpreting chromosome organization.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr>. DOI: 10.1093/bfgp/elv062.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.75" data-doi="10.1073/pnas.1406593111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moffitt</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dempsey</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization and development of photoactivatable fluorescent proteins for single-molecule-based superresolution imaging.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">8452</span>–<span class="cit-lpage">8457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1406593111%26rft_id%253Dinfo%253Apmid%252F24912163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIzLzg0NTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.76" data-doi="10.1126/science.1204697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chromosome Organization by a Nucleoid-Associated Protein in Live Bacteria.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">1445</span>–<span class="cit-lpage">1449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWang%26rft.auinit1%253DW.%26rft.volume%253D333%26rft.issue%253D6048%26rft.spage%253D1445%26rft.epage%253D1449%26rft.atitle%253DChromosome%2BOrganization%2Bby%2Ba%2BNucleoid-Associated%2BProtein%2Bin%2BLive%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1204697%26rft_id%253Dinfo%253Apmid%252F21903814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0OC8xNDQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.77" data-doi="10.1101/gad.388406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Possoz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherratt</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The two Escherichia coli chromosome arms locate to separate cell halves.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1727</span>–<span class="cit-lpage">1731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D20%26rft.issue%253D13%26rft.spage%253D1727%26rft.epage%253D1731%26rft.atitle%253DThe%2Btwo%2BEscherichia%2Bcoli%2Bchromosome%2Barms%2Blocate%2Bto%2Bseparate%2Bcell%2Bhalves.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.388406%26rft_id%253Dinfo%253Apmid%252F16818605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIwLzEzLzE3MjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.78" data-doi="10.1016/j.cub.2013.11.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">O. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudner</span>  <span class="cit-name-given-names">D. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">The SMC condensin complex is required for origin segregation in Bacillus subtilis.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.11.050%26rft_id%253Dinfo%253Apmid%252F24440393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.11.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24440393&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.79" data-doi="10.1073/pnas.1407461111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montero Llopis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudner</span>  <span class="cit-name-given-names">D. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Bacillus subtilis chromosome organization oscillates between two distinct patterns.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">12877</span>–<span class="cit-lpage">12882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1407461111%26rft_id%253Dinfo%253Apmid%252F25071173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM1LzEyODc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE1OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.80" data-doi="10.1016/j.tig.2014.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Spatial organization of transcription in bacterial cells.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%2BTIG%26rft.volume%253D30%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.04.008%26rft_id%253Dinfo%253Apmid%252F24862529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24862529&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1597.81" data-doi="10.1039/c3mb25364c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zarei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclavi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosentino Lagomarsino</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene silencing and large-scale domain structure of the E. coli genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiosyst.%26rft.volume%253D9%26rft.spage%253D758%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc3mb25364c%26rft_id%253Dinfo%253Apmid%252F23412141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c3mb25364c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23412141&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/6/1597.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1585" title="Fungal Innate Immunity Induced by Bacterial Microbe-Associated Molecular Patterns (MAMPs)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1607" title="A High-Resolution SNP Array-Based Linkage Map Anchors a New Domestic Cat Draft Genome Assembly and Provides Detailed Patterns of Recombination" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/6"><span>Volume 6 Issue 6, June 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/6.cover-source.jpg?itok=WAtQozJg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1597.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8tjI5LNdS5GoVdr8bZ_WJqQ5dEB1b0_7ELCaL4KnOHo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/6/1597" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/6/1597" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/6/1597" class="active" data-icon-position="" data-hide-link-title="0">Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460387" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2aVvDX6XHn4A98JDdH-p5lw90RAC5_WgCXxV1VDRjW0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1597.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/6/1597" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-4ukCpkItJEZjzR1q5mtPDqERxjZ-3rCFMB8o1GCJmL8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/6/1597" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1597.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460387" id="node460387--2" data-pisa="ggg;6/6/1597" data-pisa-master="ggg;g3.116.028274" data-apath="/ggg/6/6/1597.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Supriya Khedkar</span> and <span class="highwire-citation-author" data-delta="1">Aswin Sai Narain Seshasayee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1597-1606;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460387--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Supriya Khedkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Supriya%2BKhedkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khedkar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASupriya%2BKhedkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aswin@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">aswin@ncbs.res.in</a>
<a href="mailto:supriyak@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">supriyak@ncbs.res.in</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aswin Sai Narain Seshasayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aswin%2BSai%2BNarain%2BSeshasayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seshasayee%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAswin%2BSai%2BNarain%2BSeshasayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aswin@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">aswin@ncbs.res.in</a>
<a href="mailto:supriyak@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">supriyak@ncbs.res.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/6/1597" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/6/1597&amp;title=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/6/1597&amp;title=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/6/1597&amp;title=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/6/1597&amp;text=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/6/1597&amp;title=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/6/1597&amp;t=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/6/1597&amp;title=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/6/1597&amp;title=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460387/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460387" id="node460387--3" data-pisa="ggg;6/6/1597" data-pisa-master="ggg;g3.116.028274" data-apath="/ggg/6/6/1597.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of Interreplichore Translocations in Bacteria: A Measure of Chromosome Topology?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Supriya Khedkar</span> and <span class="highwire-citation-author" data-delta="1">Aswin Sai Narain Seshasayee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1597-1606;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460387--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Supriya Khedkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Supriya%2BKhedkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khedkar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASupriya%2BKhedkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aswin@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">aswin@ncbs.res.in</a>
<a href="mailto:supriyak@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">supriyak@ncbs.res.in</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aswin Sai Narain Seshasayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Centre for Biological Sciences, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aswin%2BSai%2BNarain%2BSeshasayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seshasayee%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAswin%2BSai%2BNarain%2BSeshasayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aswin@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">aswin@ncbs.res.in</a>
<a href="mailto:supriyak@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">supriyak@ncbs.res.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1597&amp;rft.epage=1606&amp;rft.atitle=Comparative%20Genomics%20of%20Interreplichore%20Translocations%20in%20Bacteria%3A%20A%20Measure%20of%20Chromosome%20Topology%3F&amp;rft.volume=6&amp;rft.issue=6&amp;rft.date=2016-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Khedkar%2C+Supriya&amp;rft.au=Seshasayee%2C+Aswin+Sai+Narain"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460387/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460387/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460387/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460387/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460387/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460387/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460387/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460387/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460387/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460387/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460387/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460387/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/6/1597" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/6/1597.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/6/1597.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/6/1597.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/6/1597.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
