<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/6/1635" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/6/1635.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/6/1635.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/6/1635.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/6/1635/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/6/1635.atom" />
<meta name="HW.pisa" content="ggg;6/6/1635" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multi-Population Selective Genotyping to Identify Soybean [Glycine max (L.) Merr.] Seed Protein and Oil QTLs" />
<meta name="DC.Identifier" content="10.1534/g3.116.027656" />
<meta name="DC.Date" content="2016-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Phansak et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Plant breeders continually generate ever-higher yielding cultivars, but also want to improve seed constituent value, which is mainly protein and oil, in soybean [ Glycine max (L.) Merr.]. Identification of genetic loci governing those two traits would facilitate that effort. Though genome-wide association offers one such approach, selective genotyping of multiple biparental populations offers a complementary alternative, and was evaluated here, using 48 F2:3 populations ( n  = ∼224 plants) created by mating 48 high protein germplasm accessions to cultivars of similar maturity, but with normal seed protein content. All F2:3 progeny were phenotyped for seed protein and oil, but only 22 high and 22 low extreme progeny in each F2:3 phenotypic distribution were genotyped with a 1536-SNP chip ( ca . 450 bimorphic SNPs detected per mating). A significant quantitative trait locus (QTL) on one or more chromosomes was detected for protein in 35 (73%), and for oil in 25 (52%), of the 48 matings, and these QTL exhibited additive effects of ≥ 4 g kg–1 and R 2 values of 0.07 or more. These results demonstrated that a multiple-population selective genotyping strategy, when focused on matings between parental phenotype extremes, can be used successfully to identify germplasm accessions possessing large-effect QTL alleles. Such accessions would be of interest to breeders to serve as parental donors of those alleles in cultivar development programs, though 17 of the 48 accessions were not unique in terms of SNP genotype, indicating that diversity among high protein accessions in the germplasm collection is less than what might ordinarily be assumed." />
<meta name="DC.Contributor" content="Piyaporn Phansak" />
<meta name="DC.Contributor" content="Watcharin Soonsuwon" />
<meta name="DC.Contributor" content="David L. Hyten" />
<meta name="DC.Contributor" content="Qijian Song" />
<meta name="DC.Contributor" content="Perry B. Cregan" />
<meta name="DC.Contributor" content="George L. Graef" />
<meta name="DC.Contributor" content="James E. Specht" />
<meta name="article:published_time" content="2016-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Multi-Population Selective Genotyping to Identify Soybean [Glycine max (L.) Merr.] Seed Protein and Oil QTLs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Plant breeders continually generate ever-higher yielding cultivars, but also want to improve seed constituent value, which is mainly protein and oil, in soybean [&lt;i&gt;Glycine max&lt;/i&gt; (L.) Merr.]. Identification of genetic loci governing those two traits would facilitate that effort. Though genome-wide association offers one such approach, selective genotyping of multiple biparental populations offers a complementary alternative, and was evaluated here, using 48 F&lt;sub&gt;2:3&lt;/sub&gt; populations (&lt;i&gt;n&lt;/i&gt;&amp;nbsp;=&amp;nbsp;∼224 plants) created by mating 48 high protein germplasm accessions to cultivars of similar maturity, but with normal seed protein content. All F&lt;sub&gt;2:3&lt;/sub&gt; progeny were phenotyped for seed protein and oil, but only 22 high and 22 low extreme progeny in each F&lt;sub&gt;2:3&lt;/sub&gt; phenotypic distribution were genotyped with a 1536-SNP chip (&lt;i&gt;ca&lt;/i&gt;. 450 bimorphic SNPs detected per mating). A significant quantitative trait locus (QTL) on one or more chromosomes was detected for protein in 35 (73%), and for oil in 25 (52%), of the 48 matings, and these QTL exhibited additive effects of ≥&amp;nbsp;4&amp;nbsp;g&amp;nbsp;kg&lt;sup&gt;–1&lt;/sup&gt; and &lt;i&gt;R&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; values of 0.07 or more. These results demonstrated that a multiple-population selective genotyping strategy, when focused on matings between parental phenotype extremes, can be used successfully to identify germplasm accessions possessing large-effect QTL alleles. Such accessions would be of interest to breeders to serve as parental donors of those alleles in cultivar development programs, though 17 of the 48 accessions were not unique in terms of SNP genotype, indicating that diversity among high protein accessions in the germplasm collection is less than what might ordinarily be assumed.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/06/01" />
<meta name="citation_mjid" content="ggg;6/6/1635" />
<meta name="citation_id" content="6/6/1635" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/6/1635" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/6/1635.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/6/1635.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/6/1635.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.027656" />
<meta name="citation_pmid" content="27172185" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1635" />
<meta name="citation_lastpage" content="1648" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Piyaporn Phansak" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="Watcharin Soonsuwon" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="David L. Hyten" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="Qijian Song" />
<meta name="citation_author_institution" content="Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325" />
<meta name="citation_author" content="Perry B. Cregan" />
<meta name="citation_author_institution" content="Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325" />
<meta name="citation_author" content="George L. Graef" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="James E. Specht" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author_email" content="jspecht1@unl.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. Ayoub;citation_author=D. E. Mather;citation_title=Effectiveness of selective genotyping for detection of quantitative trait loci: an analysis of grain and malt quality traits in three barley populations.;citation_pages=1116-1124;citation_volume=45;citation_year=2002;citation_issue=6;citation_pmid=12502257" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=N. Bandillo;citation_author=D. Jarquin;citation_author=Q. Song;citation_author=R. Nelson;citation_author=P. Cregan;citation_title=A population structure and genome-wide association analysis on the USDA soybean germplasm collection.;citation_pages=1-13;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_title=Molecular Dissection of Complex Traits;citation_title=QTL analyses: power, precision, and accuracy;citation_author=W. D. Beavis;citation_pages=145-162;citation_year=1998" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. C. Brummer;citation_author=G. L. Graef;citation_author=J. Orf;citation_author=J. R. Wilcox;citation_author=R. C. Shoemaker;citation_title=Mapping QTL for Seed Protein and Oil Content in Eight Soybean Populations;citation_pages=370-378;citation_volume=37;citation_year=1997;citation_issue=2;citation_doi=10.2135/cropsci1997.0011183X003700020011x" />
<meta name="citation_reference" content="citation_title=Soybeans: Improvement, Production, and Uses;citation_title=Quantitative genetics: results relevant to soybean breeding;citation_author=J. W. Burton;citation_pages=211-242;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Chung;citation_author=H. L. Babka;citation_author=G. L. Graef;citation_author=P. E. Staswick;citation_author=D. J. Lee;citation_title=The Seed Protein, Oil, and Yield QTL on Soybean Linkage Group I;citation_pages=1053-1067;citation_volume=43;citation_year=2003;citation_issue=3;citation_doi=10.2135/cropsci2003.1053" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. R. Cober;citation_author=H. D. Voldeng;citation_title=Developing High-Protein, High-Yield Soybean Populations and Lines;citation_pages=39-42;citation_volume=40;citation_year=2000;citation_issue=1;citation_doi=10.2135/cropsci2000.40139x" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Darvasi;citation_title=The effect of selective genotyping on QTL mapping accuracy.;citation_pages=67-68;citation_volume=8;citation_year=1997;citation_issue=1;citation_pmid=9021156;citation_doi=10.1007/s003359900353" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Selective genotyping for determination of linkage between a marker locus and a quantitative trait locus.;citation_pages=353-359;citation_volume=85;citation_year=1992;citation_pmid=24197326" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. W. Diers;citation_author=P. Keim;citation_author=W. R. Fehr;citation_author=R. C. Shoemaker;citation_title=RFLP analysis of soybean seed protein and oil content.;citation_pages=608-612;citation_volume=83;citation_year=1992;citation_pmid=24202678" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Grant;citation_author=R. T. Nelson;citation_author=S. B. Cannon;citation_author=R. C. Shoemaker;citation_title=Soybean, the USDA-ARS soybean genetics and genomics database.;citation_volume=38;citation_year=2010;citation_pmid=20008513;citation_doi=10.1093/nar/gkp798" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=T. Hymowitz;citation_author=J. W. Dudley;citation_author=F. I. Collins;citation_author=C. M. Brown;citation_title=Estimations of Protein and Oil Concentration in Corn, Soybean, and Oat Seed by Near-Infrared Light Reflectance;citation_pages=713-715;citation_volume=14;citation_year=1974;citation_issue=5;citation_doi=10.2135/cropsci1974.0011183X001400050031x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. L. Hyten;citation_author=I.-Y. Choi;citation_author=Q. Song;citation_author=J. E. Specht;citation_author=T. E. Carter;citation_title=A High Density Integrated Genetic Linkage Map of Soybean and the Development of a 1536 Universal Soy Linkage Panel for Quantitative Trait Locus Mapping;citation_pages=960-968;citation_volume=50;citation_year=2010;citation_issue=3;citation_doi=10.2135/cropsci2009.06.0360" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E.-Y. Hwang;citation_author=Q. Song;citation_author=G. Jia;citation_author=J. E. Specht;citation_author=D. L. Hyten;citation_title=A genome-wide association study of seed protein and oil content in soybean.;citation_pages=1;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Keim;citation_author=B. W. Diers;citation_author=T. C. Olson;citation_author=R. C. Shoemaker;citation_title=RFLP mapping in soybean: association between marker loci and variation in quantitative traits.;citation_pages=735-742;citation_volume=126;citation_year=1990;citation_issue=3;citation_pmid=1979039" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Korte;citation_author=A. Farlow;citation_title=The advantages and limitations of trait analysis with GWAS: a review.;citation_pages=29;citation_volume=9;citation_year=2013;citation_pmid=23876160;citation_doi=10.1186/1746-4811-9-29" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. Ladouceur;citation_author=Z. Dastani;citation_author=Y. S. Aulchenko;citation_author=C. M. T. Greenwood;citation_author=J. B. Richards;citation_title=The empirical power of rare variant association methods: results from sanger sequencing in 1998 individuals.;citation_pages=1002496;citation_volume=8;citation_year=2012;citation_doi=10.1371/journal.pgen.1002496" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. J. Lebowitz;citation_author=M. Soller;citation_author=J. S. Beckmann;citation_title=Trait-based analyses for the detection of linkage between marker loci and quantitative trait loci in crosses between inbred lines.;citation_pages=556-562;citation_volume=73;citation_year=1987;citation_pmid=24241113;citation_doi=10.1007/BF00289194" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=A. A. Abraham;citation_author=S. Sen;citation_author=K. W. Broman;citation_title=Significance thresholds for quantitative trait mapping under selective genotyping.;citation_volume=177;citation_year=2007;citation_pmid=17720902;citation_doi=10.1534/genetics.107.080093" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=H. L. N. Muranty;citation_author=B. Goffinet;citation_title=Selective genotyping for location and estimation of the effect of a quantitative trait locus.;citation_pages=629-643;citation_volume=53;citation_year=1997;citation_doi=10.2307/2533963" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Myles;citation_author=J. Peiffer;citation_author=P. J. Brown;citation_author=E. S. Ersoz;citation_author=Z. Zhang;citation_title=Association mapping: critical considerations shift from genotyping to experimental design.;citation_volume=21;citation_year=2009;citation_pmid=19654263;citation_doi=10.1105/tpc.109.068437" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Navabi;citation_author=D. E. Mather;citation_author=J. Bernier;citation_author=D. M. Spanner;citation_author=G. N. Atlin;citation_title=QTL detection with bidirectional and unidirectional selective genotyping: marker-based and trait-based analyses.;citation_pages=347-358;citation_volume=118;citation_year=2009;citation_issue=2;citation_pmid=18854970;citation_doi=10.1007/s00122-008-0904-2" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Raychaudhuri;citation_title=Mapping rare and common causal alleles for complex human diseases.;citation_pages=57-69;citation_volume=147;citation_year=2011;citation_issue=1;citation_pmid=21962507;citation_doi=10.1016/j.cell.2011.09.011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=K. Rincker;citation_author=R. Nelson;citation_author=J. Specht;citation_author=D. Sleper;citation_author=T. Cary;citation_title=Genetic improvement of U.S. soybean in maturity groups II, III, and IV.;citation_pages=1-14;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2012.12.0710" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Biotechnol.;citation_author=K. Semagn;citation_author=A. Bjørnstad;citation_author=Y. Yu;citation_title=The genetic dissection of quantitative traits in crops.;citation_pages=16-17;citation_volume=13;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Sen;citation_author=J. M. Satagopan;citation_author=G. A. Churchill;citation_title=Quantitative trait locus study design from an information perspective.;citation_volume=170;citation_year=2005;citation_pmid=15781700;citation_doi=10.1534/genetics.104.038612" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=S. Sen;citation_author=J. M. Satagopan;citation_author=K. W. Broman;citation_author=G. A. Churchill;citation_title=R/qtlDesign: inbred line cross experimental design.;citation_pages=87-93;citation_volume=18;citation_year=2007;citation_issue=2;citation_pmid=17347894;citation_doi=10.1007/s00335-006-0090-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Sen;citation_author=F. J. Johannes;citation_author=K. W. Broman;citation_title=Selective genotyping and phenotyping strategies in a complex trait context.;citation_volume=181;citation_year=2009;citation_pmid=19153260;citation_doi=10.1534/genetics.108.094607" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Sonah;citation_author=L. O’Donoughue;citation_author=E. Cober;citation_author=I. Rajcan;citation_author=F. Belzile;citation_title=Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean.;citation_pages=211-221;citation_volume=13;citation_year=2014;citation_pmid=25213593" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.;citation_pages=1-12;citation_volume=8;citation_year=2013;citation_pmid=23620825;citation_doi=10.1371/journal.pone.0054583" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Fingerprinting soybean germplasm and its utility in genomic research.;citation_volume=5;citation_year=2015;citation_pmid=26224783;citation_doi=10.1534/g3.115.019000" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Strange;citation_author=H. F. Utz;citation_author=T. A. Scharag;citation_author=A. E. Melchinger;citation_author=T. Würschum;citation_title=High-density genotyping: and overkill for QTL mapping? Lessons learned from a case study in maize and simulations.;citation_pages=2563-2574;citation_volume=126;citation_year=2013;citation_pmid=23860723;citation_doi=10.1007/s00122-013-2155-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. W. Stuber;citation_author=R. H. Moll;citation_author=M. M. Goodman;citation_author=H. E. Schaffer;citation_author=B. S. Weir;citation_title=ALLOZYME FREQUENCY CHANGES ASSOCIATED WITH SELECTION FOR INCREASED GRAIN YIELD IN MAIZE (ZEA MAYS L.);citation_pages=225-236;citation_volume=95;citation_year=1980;citation_issue=1;citation_pmid=17249034" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. W. Stuber;citation_author=M. M. Goodman;citation_author=R. H. Moll;citation_title=Improvement of Yield and Ear Number Resulting from Selection at Allozyme Loci in a Maize Population;citation_pages=737-740;citation_volume=22;citation_year=1982;citation_issue=4;citation_doi=10.2135/cropsci1982.0011183X002200040010x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Y. Sun;citation_author=J. Wang;citation_author=J. H. Crouch;citation_author=Y. Xu;citation_title=Efficiency of selective genotyping for genetic analysis of complex traits and potential application in crop improvement.;citation_pages=493-511;citation_volume=26;citation_year=2010;citation_doi=10.1007/s11032-010-9390-8" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Thornsberry;citation_author=M. M. Goodman;citation_author=J. Doebley;citation_author=S. Kresovich;citation_author=D. Nelson;citation_title=Dwarf8 polymorphisms associate with variation in flowering time.;citation_pages=286-289;citation_volume=28;citation_year=2001;citation_issue=3;citation_pmid=11431702;citation_doi=10.1038/90135" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. N. Vaughn;citation_author=R. L. Nelson;citation_author=Q. Song;citation_author=P. B. Cregan;citation_author=Z. Li;citation_title=The genetic architecture of seed composition in soybean is refined by genome-wide association scans across multiple populations.;citation_volume=4;citation_year=2014;citation_pmid=25246241;citation_doi=10.1534/g3.114.013433" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Wray;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Sci.;citation_author=H. D. Voldeng;citation_author=R. J. D. Guillemette;citation_author=D. A. Leonard;citation_author=E. R. Cober;citation_title=AC Proteus soybean.;citation_pages=153-154;citation_volume=76;citation_year=1996;citation_doi=10.4141/cjps96-031" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=V. K. Wehrmann;citation_author=W. R. Fehr;citation_author=S. R. Cianzio;citation_author=J. F. Cavins;citation_title=Transfer of High Seed Protein to High-Yielding Soybean Cultivars;citation_pages=927-931;citation_volume=27;citation_year=1987;citation_issue=5;citation_doi=10.2135/cropsci1987.0011183X002700050020x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Z. Wen;citation_author=J. F. Boyse;citation_author=Q. Song;citation_author=P. B. Cregan;citation_author=D. Wang;citation_title=Genomic consequences of selection and genome-wide association mapping in soybean.;citation_pages=671;citation_volume=16;citation_year=2015;citation_pmid=26334313;citation_doi=10.1186/s12864-015-1872-y" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. R. Wilcox;citation_title=Increasing Seed Protein in Soybean with Eight Cycles of Recurrent Selection;citation_pages=1536-1540;citation_volume=38;citation_year=1998;citation_issue=6;citation_doi=10.2135/cropsci1998.0011183X003800060021x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Würschum;citation_title=Mapping QTL for agronomic traits in breeding populations.;citation_pages=201-210;citation_volume=125;citation_year=2012;citation_issue=2;citation_pmid=22614179;citation_doi=10.1007/s00122-012-1887-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Theoretical Basis of the Beavis Effect;citation_pages=2259-2268;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704201" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=S. Xu;citation_author=C. Vogl;citation_title=Maximum likelihood analysis of quantitative loci under selective genotyping.;citation_pages=525-537;citation_volume=84;citation_year=2000;citation_pmid=10849077;citation_doi=10.1046/j.1365-2540.2000.00653.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multi-Population Selective Genotyping to Identify Soybean [Glycine max (L.) Merr.] Seed Protein and Oil QTLs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="twitter:description" content="Plant breeders continually generate ever-higher yielding cultivars, but also want to improve seed constituent value, which is mainly protein and oil, in soybean [ Glycine max (L.) Merr.]. Identification of genetic loci governing those two traits would facilitate that effort. Though genome-wide association offers one such approach, selective genotyping of multiple biparental populations offers a complementary alternative, and was evaluated here, using 48 F2:3 populations ( n  = ∼224 plants) created by mating 48 high protein germplasm accessions to cultivars of similar maturity, but with normal seed protein content. All F2:3 progeny were phenotyped for seed protein and oil, but only 22 high and 22 low extreme progeny in each F2:3 phenotypic distribution were genotyped with a 1536-SNP chip ( ca . 450 bimorphic SNPs detected per mating). A significant quantitative trait locus (QTL) on one or more chromosomes was detected for protein in 35 (73%), and for oil in 25 (52%), of the 48 matings, and these QTL exhibited additive effects of ≥ 4 g kg–1 and R 2 values of 0.07 or more. These results demonstrated that a multiple-population selective genotyping strategy, when focused on matings between parental phenotype extremes, can be used successfully to identify germplasm accessions possessing large-effect QTL alleles. Such accessions would be of interest to breeders to serve as parental donors of those alleles in cultivar development programs, though 17 of the 48 accessions were not unique in terms of SNP genotype, indicating that diversity among high protein accessions in the germplasm collection is less than what might ordinarily be assumed." />
<meta name="og-title" property="og:title" content="Multi-Population Selective Genotyping to Identify Soybean [Glycine max (L.) Merr.] Seed Protein and Oil QTLs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/6/1635" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Plant breeders continually generate ever-higher yielding cultivars, but also want to improve seed constituent value, which is mainly protein and oil, in soybean [ Glycine max (L.) Merr.]. Identification of genetic loci governing those two traits would facilitate that effort. Though genome-wide association offers one such approach, selective genotyping of multiple biparental populations offers a complementary alternative, and was evaluated here, using 48 F2:3 populations ( n  = ∼224 plants) created by mating 48 high protein germplasm accessions to cultivars of similar maturity, but with normal seed protein content. All F2:3 progeny were phenotyped for seed protein and oil, but only 22 high and 22 low extreme progeny in each F2:3 phenotypic distribution were genotyped with a 1536-SNP chip ( ca . 450 bimorphic SNPs detected per mating). A significant quantitative trait locus (QTL) on one or more chromosomes was detected for protein in 35 (73%), and for oil in 25 (52%), of the 48 matings, and these QTL exhibited additive effects of ≥ 4 g kg–1 and R 2 values of 0.07 or more. These results demonstrated that a multiple-population selective genotyping strategy, when focused on matings between parental phenotype extremes, can be used successfully to identify germplasm accessions possessing large-effect QTL alleles. Such accessions would be of interest to breeders to serve as parental donors of those alleles in cultivar development programs, though 17 of the 48 accessions were not unique in terms of SNP genotype, indicating that diversity among high protein accessions in the germplasm collection is less than what might ordinarily be assumed." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
    <title>Multi-Population Selective Genotyping to Identify Soybean [Glycine max (L.) Merr.] Seed Protein and Oil QTLs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"hJAJsronqniFHJDOeMElCSw6sg9PNj9wQ5ezdinilzk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460366","apath":"\/ggg\/6\/6\/1635.atom","pisa":"ggg;6\/6\/1635","ac":{"\/ggg\/6\/6\/1635.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/6\/1635.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1635"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1635"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/6\/1635":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NpjMvCAUttK7F8xQ9Td5iA2\\\\\\\/wTlZA30kt2EE3qCURq\\\\\\\/0sgCltB30q\\\\\\\/zTxVjh\\\\\\\/7ZBkiuU8UNc3RIOuEVcAiPhyrdQ4dnQePX4ytg+5+urWt1afl6EizDx7Y7DKpxZG6+7MXb7veA1QItVP9GVq+ELbCCahhhEIoG2\\\\\\\/BewowxcN0q3+IE+CAoe4cPlVjXaUthd8o4OqDP8rKNaKZqsxwwtabSK3na\\\\\\\/GgJnpVd5i\\\\\\\/ZSUhNx8ENEGXCG+dtIFK\\\\\\\/pK0jhqZJN1oU1\\\\\\\/F19fXVywP+NFdzUj724bXkFKolfXu3RLuJoYCgmNGXHGq+w5pstdoj1q497Z4RfrSvDbA28AzDfXoJGF7+cXBRGoEEfI2oc086BTDSZoW9Oot1pJPPJ+Wa8fcwxJhSf3sqmGY3PE7peFc1KRKBTAjjp9Av3dxR7oyyfhNnsXaoQu3p0x5l9v6ZQCjAokzHselXGPJrfohU4+RdJUXyvhMxjOHqcBHoeDv8XKK4ilVy5uQOrXZ8G2tbVOY+7\\\\\\\/TCMXCX5xaIiks7LXJd36kzxBE5gGRjYo7vjN7Mx8V8GXfSlyv3GnZxNpUHNvUEcNzCYL2NyGktu8hLbm7ohEsuibnm9ONrTkG8R6wZOffBfe\\\\\\\/cBMYtSMExnn8j2BypYNC2IIIx74WYsNf9kWuPj0uG0TUeBwOegpS1CWUfkyTtFE24S0CvpQmEyJPp2XHLRXrX6Em7XNwU1HFkIERHbZWrcpuTaO8C\\\\\\\/pniIQnDOJqxbqIKoW8wxmGjDOIt8DjiTyQmsGjOZHc8ohs1Yb1Xatii+zpGYkhtq6x3\\\\\\\/TJ4dIXlU1gdKpEmxNr754BAsbKrstEvh+53CEXicBbTYsBUn3YLHDZnx9nBt0+1wMojdVm40nE5cZNDf7G\\\\\\\/oVm+sxSwYgebr16OQalIvaz7P+OgfAWK0yBeD+V66vPAQXSZ3LIxyAgqy0yx9bVt6KypE+syU7IXvix5kpSC8x3kIpiQp2+K2ljm1+nqQk8Z+ktbVGOUB2C5rXZp1mWQAFHuORF4jKW7nOpvbU0a1QqYcqxnJWDYkALixJuYAR45CNP2hGitQLEloADRxaFmf6HBcSMetbnEjxZg7incl9ekLhbFcGTVUfH4u8RGfcRnxueHk+M0LViF+EXMnvshUld40kpEApJ8UWyelDafrsO2zN2+os5UeU\\\\\\\/16sozkWPz\\\\\\\/vWSj2suZOQ\\\\\\\/ivJMPKy3Eg0ottfodtb1RnA3iNb0oqJ2qWRBVTAbihXHfScJixvefDEoqIhVpkUBuViI2ZGAUykM3\\\u0022,\\\u0022iv\\\u0022:\\\u0022JI9BNNbKkHXEWUrAEqRklQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002265545dd682a05856a5627925eeeeb127bffc327a9f1b5d37268d3f2538f486c9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rEUzVnFvpWCDGhXJvlFzpOCEW+CYaW\\\\\\\/kOAetqHLVEyqyAZpGDIdPxACz4I+aBrZRXQEjzDuHPAuf09LAM5hDwi6fp84NgR5m9w4gdMlENRtVgngjNwjGxtr6hA2xzeiIKdE3aGjzZ372ODkL5KqGd5RnUqHEMbTx6wA2TU6HOpnG1JVQR3PqkyFV0DU6d0hEru6Iya8se7N+TU1zfmsO5wWPqJBZoUYaJiJjcKyApAtG6kPaFgwnWOWquQ6ro97ylscJUlplcNCg6mKZ7dCazPZATbCYrOLlP2prOZQuSoMZJTISwEgHSaih\\\\\\\/AMbXlEjFTlQEjM6PdIGMuPWRVM6jiZ2N7KmXYBtBzcPLuDyaCY\\\\\\\/luml2pclA4LZKL9mQP7MSXZEF75HqfFY8QanH47TCtquuerM8bLggItPR34XW4c7bIFujs+XNAhe53rkiSyRg+Oshq7HcC\\\\\\\/lE0NBfuo\\\\\\\/bpUNR8fmZIy8RG5dpBruG5JLhkKXkcA\\\\\\\/hdbQTgXiXnJ8eYHQ7\\\\\\\/C3AUzowKuTBTB0gHc4U1X5Z0Nz5Byd4tfOe6S4cSkCtBFzGK1kbpMsqG4lw3\\\\\\\/6nzHKdoFvvDIpvNP2v1OKdi0Gfwfe8vbgkbT6laOdmx0L9F1oZUF56EgOgzcrmzYbraaV9BdaXEWHmbTrl8RW47S+k94KygnnAOHpO0Ii+CJMpfn0gFy1p7RoUfEkHOV3UgmpXUbwM2MTlgoNmWx+OJCbJK8FO8FHjs1hwdyw773B3y\\\\\\\/tB01SZrwzqWnlHPM0S1yQEQtQrQ8fmKggjXkUZpxKpGrk+CkQtZektnTpj8Y4qiGmoayg+L16g1DREwvKmgJouyGtCBB+ztTy1iUZhOIyqufuwmFnYUoBWilxIbpS3BTOEUA29dHT9tDhJfTJ5CQMPSieJYOm\\\\\\\/di4xPlv5hAxlOlaNfxIafoRzwCKnEyB1WdB8yzyXuCm\\\\\\\/GNqexoa+bhUeooZYZTSWGdEBMWRZRo0eTicRVIwjoHZbJVdTJd7Mtsf83R+ryqBPJmT+M0VpqZ9ewDr1LL5aVBjVBSivSrB6Qual71+8C5s314r5T26FwD\\\\\\\/18P7mJC3cIGR+KGCEwmD+0x+JI3nRZ3nGf\\\\\\\/IOBSLwc04KJKscAn5NFEa3dhRrIhJn+jjaweFaUtfZwVtUrdmBs0QAKGaSb+QbFlB4DuzbfNcyNCvMi78q2Pko+LdO7QvMf7RT6IIopRnQ7BWq7C5IV\\\\\\\/gtU1GWyeyZ1YKecTn9Ru7dFQoRgjnjnnnLT5CB56beU1P3juV9Oy31M5jmwu9i5BNxS1h2jn8nNrYo552W3aUWQlIaAfk7cQ32BlBfSJOd0ymbQ88tdeZgTA8ZfYp5lIMsKitv6f3BYx2Op61xCOXRaDTFNz2uWP0wNbEwlKrIL1d1MDhlEHVZYAlem3PfmEHU\\\\\\\/yBn7SbPQWff4kiVQbOeRz+7zZ\\\\\\\/1gME5v\\\\\\\/T02GdQIfOf5jByPLThXWNiIQAM3lafDsN8EoMo4mWSFE42C6V90rj1eBnT+GK4YopjR878s\\\\\\\/a1+gld3VYXOimmV8Mp1JEhBOc7PYdg7+a9WBcq3b5jD\\\\\\\/B89QIlTubArvnPazNVg1FURyr1jrkM6Ti4ZrKEpg5ExD7Z7HuJwwopf+DdO8CfufwnvUK6fxoPmUdLeMzQXDYjDGuj\\\\\\\/y3L30ZJ3rxNjP4vhXrPsTjB+rxJT5j1x0Kq398Thz9xje1KLXtlgnI+8sdCYDI91qCpcd6VuCVBL4mEIiZZ3fTzztDF13VdVy6+WF54b3JMO5+yLLqbue0WreXezCtSus0fPJM+LfWE6SIxH+EX7F0vD2j15jPgKPyoRsv4e3ukezrqe6pAk1fYCsj\\\\\\\/w8oBuzj1EVhyvDuyLa+34yhBBL4f9oAPld9jnkZQDxs5dsYAUoLgNWkBFlHMD3soO5T9oEQ\\\\\\\/XdnK+xq7qrFIRYNYtrDMBnHwCtpWwZ+oktTVn6Kndw1O\\\\\\\/TuBoRlfMQh9So1CaT0e3VCyBAm0gl1w+pBp258WbdvRPO5UPGbIHHiYB05P8yS5XcyUdhzclgHEWPxiritH7vGA+W24sHA3HWLRJEnqjnTJN77KiHY1ktrOGgKZ+9iLqZF2AJhnBu+wP+ijZdqRRVu\\\\\\\/fEaFS5K9Ifbju1+D2caGB0zazgCnUCNbiTRf77Ij9cOkVsMKZmPHyRrXa5K9FgA8HTeaEsSrIV5QRBDP4jcwFgb5x4JjIXquqiylWD\\\\\\\/wskG2NhU\\\\\\\/krmlQnMgD8HNUMdgWmgNeTnWJgvvC2OlWOaTnt12LEvEs0l4cZ2DljvftNKZ8+ifNFbJH1gB7JmeVi5ivnBxfyCWQ1Zbmx5XEPOl9uHw96F7h7C2Zqdu1O8vQN0Ac+2mkhYQ84m0thp19qKK\\\\\\\/abGeYytjuVi52TRLVdqoN5TJyitrom8Vk6f1icxgxVaZwk3+GzLn634KUsIjNK\\\\\\\/GXcl4uBrDcRvcBu3\\\\\\\/dYB9BG3GGt9tgw6H99EXOk3DhZ5emKcVuvJsnPME+JrzDnQcTf+bnnzMdrh6h8NBWOu8FOGLRrJfiB6in8LnAmOCxc0z1N4k8GB4DdThhUvGy7rqd2mrNdLvb\\\\\\\/B6T0YWfYZfYHtLW07B2CCnKtQ0ow8ETLKL3ogbQH9iFJtRVIUZzTYu4vAWwCIUPzTy8sfgw+sRaPq0+mulcPTh6Ndn9Ls1e6Qwnsy1oFxZ+Ndo7o391V+sPjWQL0DMgbnB\\\\\\\/vuociyv61W\\\\\\\/VAD337cgTqH8gKwDcMMp+4XS8k8u9w10hud1s4ucDMf6ZnijYfZzSBubyrh8LsCgi31vxyuY1qgYAQ4uQTAu7s8CutklwTQV0pupj98Kt7sJ6cch8bSAY+Z5SBXc2bPFVLejzCyddMYY7kRT+sAGh8z\\\\\\\/8HQ4OK5GHRloDwMCWY2jvQBx+kNkeXfGnC\\\\\\\/WGtUROlhA4luMeBLKztpeQ9qh31pPW5zRvmbg4rILnsJatuic6QJuJQs3xSqederBOGJaqxU33bHeInvkHs2n5WIkhr+BPpjuZX4bdS6eZru8g2tfrPQqDdGz8Y7WbDJ4gSTzLy8ybLor+f5pgjPnXI11to\\\\\\\/TXANfWOYwdNl2kRtVDIjcbrXLmkSkRl9UVitFnXzhIpWu41zna4vcGv0VkY4JwwzmdB8ZyQ8UGhfMg3A7F5zZVjFv3oO0pyNWoKxHzhl8m+pS3OWn\\\\\\\/6NhpFMWOVCXVgsonS5kOPS\\\\\\\/cDl47Z\\\\\\\/FFrnWdd6hSQ0AVd3umrtT5Eds5Q772ig1pN34ga+31ATNnpgQpXiCQKmn8+3kK2D\\\\\\\/fKv32qq9SUq33d4q2hXZ2WTxy6MkllVgj3pOKVA\\\\\\\/U2L4pyynxJYJUaYl+Ucna9YiB5WzOQJbKz+fxMgVz5Zz85GQ7B66vP5wl48ODocf5uD6\\\\\\\/kKFxqDOlIxjMF6PnUvirljTIpXGeQPcgeHZAHmBAla7oXWx8QDb2XazklKLbCe4HDtJdPi1V3okOGdcvawe8LFwsoTsu5QIKo39oAjhJ06kQEiJWXE4duoYfhgYrCM538TRMzxG2K8JhYONFWYmzKMN+tJ6V7K61ZVyuHRG3FFg9KmXU1oTAc1oqd385MsvJ5P0GAL1Tv6mMDsyZ9jh5dBeTwURa6+WKGdHw0mGsYl+SeFgs3EszZZONWXchVmvmAsiOBCWcVvDWy0m\\\\\\\/s0jHeMf6Px3f7QHIkJvf8yHXRWw6jDui2tAC5UnnoKTXn9U+x+Io6wCFXki8r3q08rbKkAY5XVHGGIN+kG+Qz2Qh66v5Doi+vk1fKY6HeV4xT8vJJnvgpQrwuTjPD3D+GlGCRMXi1dWF9WtPo+dwDaKYPlX98rYELqVJonFPu97AjZLhOh2org10aSBt\\\\\\\/F\\\\\\\/fROT+CUUr2FLU6iGLqHHTE2zvGM4nK1bx5L5hchnqEQyep8LnA9er2SZOtNTffh7FbhyI219jjWNXUv10g6zcmD1uxImTsACFJkkoL\\\\\\\/s1F46sIAxn1R8ZOR2kNkrsueTACtCWw5jDUjkeHEH+o\\\\\\\/g5H9PwSKvF0GmysNPHEwnvnH214woHNb4nOtUVdexCgWGBc1myqrzar30V8EAzHNIKOJK1pIX27hmEdHuPsNqlVyiLGerE0i0m1iZlMQtc\\\\\\\/tYj0DVgWtzFJE3SxIRvMQSjvqSQlimla4eAnbcxsYMp\\\\\\\/VFIm60mXubqelwEkFsfieFoxsVYwaHWoU\\\\\\\/TuXfTUxCUaPnPNyJMOoctI0zrwELM2kQfWQYZJUU8wzjaKHx\\\\\\\/9likB9cC+lKorbh2foLlZ0d2tsa25H0CzgoZw+98xUfMfn12MWZ41LPdkkyRNHwvmrZ0fshjcIGv+ohELuILEV5w0clhClRt+Tf4q9hzbstt18U5IG8p+dIzwYqm\\\\\\\/vxmbQLv5YE\\\\\\\/UUFnyioak6ShLdvEdR1jpTnn3Is\\\\\\\/4\\\\\\\/dFXxZ1vCxq9jxYSxu54GiLMGTeZxrDpiNVgI6oMaYFQPBm+WdfF0Mh7HfPNF+aFuHsjXso4S5XT1pD5JHzFDaf22f5AKiZmyRRne3k3BKvmSY7NsoWMnddx3C27bu2x1mAuate\\\\\\\/mN7DYyIJ\\\\\\\/qHuKevFrgmf3Mcki09nL881OTiokmkA19yFgHmllU34H3jyJLGUAHd\\\\\\\/AJ1E+xFyI113PeE2vZ3YNjgx6p105oP8izlq5o8AveP7ss3Ila8urXFE\\\\\\\/CVx8Tb1cqHqS86uxmpDszY09tRqB2NjYE1bI6uwFiZ0uFAmUHzd3JoTrBJ3GChALKL3X4FHl\\\\\\\/FWhoye5PZOD5ALIPy18zSl9cmXjc2rhWpy0qzHvh3HLvxynfn8v2Ve3nxQzM18qVbDZ4gXxdYGmYt51upNT9eovNqv8n+Jv1Y6cr\\\\\\\/D0idm3Kzf4oNvmWm+iyg\\\\\\\/JLHK8F1ksv0j8guPB980kBN0+mz9yomWcut85914eTDdY1cbBEzmqbfgVqywnr5ifVHFL30OV9+QE\\\\\\\/8e\\\\\\\/CYtrjuYK1KE7pNqTxiYd1AyYURaiV2\\\\\\\/UC2jR853tplo0TZKlQMVBpXLqoNh1PWqdurBHsCcL6tBndYxMyitnlPzcPgFIl3SinsnuMUYJH+xlOGBhN05YSaRkKj1uTHHQK8wfdMax5P\\\\\\\/m+QRko6UY\\\\\\\/GgcgX1i8kS9xjco4iqpg6h\\\\\\\/RRzibdeKESMKDscwnjpGhM8YOrGMSdCLQJ2D4HLSfh4Pmr3NCMyA\\\\\\\/a1mXdriIiv6wGlZ\\\\\\\/NYQ8QzR0kTjrAtj5ZFMsvoCPdVHhNGMgdNqN8htCIXVGX1965bOqNx2FGJRPTM0Z8yOEEKkc56WIGI1pqsTGdeAJ6g\\\\\\\/J34JLV4xtQQMXan2RLnykcsk55xAqpbee1FOkKjPU03bcXoJEOt1HSr22I5GkkSwKuprNH6Jhhxh9QTkzRjYB+2+7pxHlhoxh\\\\\\\/\\\\\\\/qIvc7RK4owXVj3JA4b5oQVv5WDddyjnAwyGyo03Fvm9dNK62RxySyLyXVu\\\\\\\/\\\\\\\/OPqjIHOE98zWuF\\\\\\\/ciHG46mG\\\\\\\/TzTlNLINurdwmiQ1P8e54SdTnkgCTZClJZez5nYu9E6\\\\\\\/Y\\\\\\\/tKwyrxkGq+Js3Re7OZnYd5povixKd4Vkv5fjtAIF0PDAy6fqpI9RDfUHHClKB2K7UQro9hAVDEuFGPEC8SW7AhRQxCoVR5SWLglTwJ5xtZ5FMeHU5\\\\\\\/0QsOlSYaysG\\\\\\\/o4gyUqUCkaTwWLvbqTwQM7Wkq\\\\\\\/0Cri48TBU6DOXGCNoarKhEW5QB++DaBCpy+98QEPkajGuXrCugOfM7QmclWNc2F4QQNQ4rw48S6jnhhZ5KsQJ7zQQ7kH+KUgFElbcZBahurbhqf6xKxCiSa0juyYbZKzgh10hbKlcd7oLv5axzhsPaOc1jPkd84eUTADhcWZgkjnb1RfnHMv5RXqkkFD83X0n0j977P2pYEJaSSJUuZV+5E8Yo3j8WStylrwzZ+sNtYNjg0MZUOeicT9LuFaxDjmK06g\\\\\\\/rJULzUvyrcNrC0SHxzJbqqkWWHrR59tEuQidN1D3MSykpwvV6ZZjcD5Edvx2SMdw\\\\\\\/xJDQYA4GO02aSKqTnl690Tm9dJx+6BmGbWCDIr0nxyOKQXzW5OowiFbXkOy987M\\\\\\\/i1EEZxH190oHwJSMqQD0aSPDhtovxWXtLLpRZjlOwZZ0fCe+y15lCfINaBgX2sU4yN9FlpYEj3JZc7F0hmHaH10F7rp3Plh2Jce5fIa4SrFjStY8mdsIzg+\\\\\\\/N0NROKGXyEFq6dZrjg9ChMeq+lWH\\\\\\\/mNKOh8iVK+gMqkZf6nJDApsZQtMUTX2SgyKDJ3EE5jww38FXzsXtSAE4PrAIcDughjqYMO\\\\\\\/23dTQBdKF+ZLi7k36WDF2eVcnL6J6wdyEX8VS8R+Cc+WK1XRWQcfEZ+OUrsyROxZyrEkfo1t90TUuAPqZ5eGBWVRgFgOnPHeUl6NYE6Y21jqKUg4co0cC67kcG1w2f0MgF6b7cVlfgV\\\\\\\/2oBJXdrkc6wgxY6DYS0Qv3haaMHqGkVASJzEfixrGgZWr7PIhbAFs6YQqGo+keDth8bP+\\\\\\\/wFYkfKCohM0IigLWvXtBH3CZABMi+0lEEjAB7axVCDTjpAzef5\\\\\\\/BA5PDNzwpB1XBQH4T2vcVrq\\\\\\\/zbwQLcC2tBF0Xek92SlNM5UI2WXy8hP3FKAz57MmNfdWlb8FA9u7V2Yo+MNC9XqT4j9fuLpD4ZhO7jN2WkU19S5N1ERkJG2SN3Rqa7vc+tmsqytWdjaYT\\\\\\\/FOIYsrr91juyC+z1jf+ymu1Y+fhioiKBY54hWKYxBApIQNwtwGK6OtQHtPHby2Og\\\\\\\/uYHLJlM4XXOgsddkVJ4EdKtNyQNY2rl9F5JseaDokA0EhGAA5iQL4Ejo1vSaNT75LBtZzu2zsTg11Ssk7UMf21y55jTejGE3Hwhi9S+KfN8jkTjEF0vy7Gm0xifqw6gweUmK\\\\\\\/m5sYJlfaZs5qpJ3mOi26ODkRCgJN85qvPRRiMvD5PZdyQvq6sRqg0V7wOYdDS+WFtKw49rFlcNSZMRpbZNJOktuivh5I4qWDP1DVQxI1VCh3p7PH3Fvxzp0oYU11NF2l83nMw2I3\\\\\\\/+ZKfYYhwddRvQRJ4hqJuCV\\\\\\\/+TVAkApm1vSeUWKQhAQIzUMaUwzx5wgcKptU6QMpEMAB1xxmGtB2y0ET1sIPdyUEm13y\\\\\\\/sIoup5+oDoKqWP1y2Asz6wNsVwnsMLnuTVgztIYYr0JCFW9krcVgLyGHKN0m9TncvKbYY4\\\\\\\/36gnWuX0v5esK2ftCgDDJtjqDTvJMroUSw7H9R+zbK4en81\\\\\\\/7tYFRhncZqMrWIDThLwmBSp0ThGGm8sOJUbrp9q9sZe+nKntBuJc2ds2HX+kZEeIReFjXaW1x\\\\\\\/\\\\\\\/5NBxi+2F2+3r8FzK2GVg+6E9BoMhcLopqoQC4wGUobvLoHe8cFfqz6eOpfRZcaJ56ZL2gW0OSJSVAECXQ4vL69G0ZiXKiUiNXLG7ip5LMFj\\\\\\\/bAmRApEzYDuR4099TuH59KvwLmyJUvg8cpG2pEhyEGlopm50576Q2dwI79bNlfzmqYFJIVZDMutii4mzNuyeBQrH72QZBP09pKFGBzVYkrPCQV\\\\\\\/JZwot\\\\\\\/OvAIrOBIMhOzWlPK8sG\\\\\\\/\\\\\\\/HouZde6AEiEe\\\\\\\/lwIQYfghRP6iEvcNsUFLXLq23SlS7Fucw4xGRFTg\\\\\\\/7qMeK0Esuj3RnuvXkp4JY9ikmpWy\\\\\\\/X4o+AiZ8HRnbqyOecd25lKLe1xXfUc9wASO3EohKwTvSxuf96ckZlWNCGR6q5ECPIsY9pF6wx6bZib0aDQAWhJzXCBgkJ5h4BAdfOR6ytl08Ak+7TxWWcVGTJWwTUw7yx3V8HyKmZhmY3T7ZjjPYlRkmBkl6M5b1PkUB9icWP8vcWQLiDsvMbMqjOfBU\\\\\\\/5MBvvwUXghbgq4k8eEYKTyjEUZbIXuXM5q3FwqiNMnAQ7wxv8XQY39Tc9I5ZOfnIQlSnfCggpGBX2ft+Su6kTqqjZVxeQABI6tgizxY+SfiY0vJ9Tn2w81pgZgSJO9gXRmlq8301zlmjKDVWh31\\\\\\\/VgH9ygRauPP+RO3t5wC6a01dF0iS931wM5DH\\\\\\\/iFX2DEG2wlaqH1pc6zX0WnO04PCcksnrDLpWrap7tuDAcUajMKHcU\\\\\\\/qPRbtcfksS51iY9RCdicEflRmKxHRSJy61Ttr+5A6WIeLUpFLcGqkfiCH4uxaqLRRdYPV3Bnr\\\\\\\/KMZSIAJb9l7PpeKAdB3Cv6+6+XIfrCHUDJGtVc5I6krU+k3vMlfgV7H\\\\\\\/DSTVA4W8TTALkwUd7ZJ+sZPdpMUjVUlx5o8\\\\\\\/HEpV8UOk8aZI948VMJEfdPqXPvdSlhNN3fp0Obsp6BcMU0Yg79bKOFmt8jiDVDAsp6eYEBEASCoJ+VKY9pq7jFvwIh5zqYPEkgfcfgWgcXT8i8I9fnjhhUhO8IqUUaZh5aQ+0lYBVXKChrPWzLMf9MhMmATjeA7a3l8R+x11JR0FL\\\\\\\/78t8XXyV8rWKbf4lh7l6vm6j0gBMWDqsIHnAYhn4Gg8iVTmVw++mwaikfePLfpc45PwmsBwQ4Wc1EalIO0gUyq\\\\\\\/HNilNFZBxp0HCOySIA\\\\\\\/321xt8lGEH6tDH5lkX1MDTOPVi532vtZ0\\\\\\\/UBC3oMYip8M7tIBEq1\\\\\\\/rn1MWVTsEwx85G1mplrdUMz8mV6qjOvAzR7Hzin+3664+Y8J5HGdlNsCXvAAOE\\\\\\\/+ZWkbd\\\\\\\/qgBVpmBLhmTUSsnYfi0cuspOB\\\\\\\/LSc6d3gRVT+tEOMf\\\\\\\/U9Yg1DZxheFT882U7U5djk49VcJ+yFbA+pZGvERVGK9CWWwe4spzW\\\\\\\/1lv7u3gOQqkO\\\\\\\/oG5voaxoR6+wK9B++HGgWSVe2XSOHdrPB6iXNFiuwjSz9G3BVlWZ8XcNhqVIExAw\\\\\\\/XGNqio7OoWOAxHhU6vokd5x9bnc2owOdeRrQq0E1f3S6GVePTbK35LYGDiLh1TiYZuRm7VjVAE4XqfV8N0yVrWqg3N8WSyxt30b1rECDmyqlOujN+Keba6LZsH5pBe2x8qtGLEMzPK2ZhRwAAh1KKglri9JAMfeexIdtVRJ\\\\\\\/vondG64j\\\\\\\/C7vxy4Dhj0r6ywUX2bssq28QyCvk1hzQl\\\\\\\/ItmTk8rpzyiNuWJqYn8hvFPB8GFX9uqKORnVwF+ehegfNj6Dp6iSJtw5YjV2UV4arVZLStyqpdpCzo\\\\\\\/lsm0KKrwLdi6RfRLXmsokU0jivxO35jZnLIQ64YTchTZ9GkqObjaeW5ZNIZp580Q3wpBcVc4Ryg2fljAdHAbSaxLZVT+IwoW3pLhRlvCk5Qq3Raal94igLd0G7eeci6V8gdop2y0ZcJ\\\\\\\/AkEJsSkISQFl6zYGnVBhSmM05Hy11penxr4aPzmeXVXSMZFFHEJU6CqHxnDxSF9uGlnNDvqxpElWMokACszlOXCe7fN6qshFSCOK5M+8mJIcGoT\\\\\\\/56OMsXgACt2FIIxgiQBsH1UOCd6qJmoDIlJZGt1Vjo6mpG1JJVhHJwDqD1h\\\\\\\/4pjkWS61Ma\\\\\\\/yv6Fk5nUeKWi31J28hr63PiT3hT2bSeDRzsCAzAmGDVQrhWE9a0CB16z3Q2FUqpYLL5Iy4ETxqOaipPSp3PCSfHu9qVO6FApioT7n2\\\\\\\/691za8oBau\\\\\\\/R9MW+z4dXEFfV\\\\\\\/xmOQeZ6qfVFLx5Y5vjYW0cphTlFAGhpCT2RxI9cDqaI\\\\\\\/\\\\\\\/oMIu6L9w1hp+692UiJStjRfEhEmNLhGChEFdIOqaP+K8+OsqsOVpl4EgwBUw7jfuDDTa2HDDFniPzDeEz3SeCb9RtRKETmBWi95QheMsR8MoCIXi\\\\\\\/\\\\\\\/L3O3R\\\\\\\/Mg\\\\\\\/SZUWtsg5KX\\\\\\\/8AyrIfifR3J+iDNci97m4SMaCqUpRkAeL4vpCXqnvE4m2s690O4NiDcSw4OXBdIDnMxCEfn0r6dPGEP5qJPdm9cNFllZeYKXVdpRlzjfsAj\\\\\\\/v1XW3G8Z8Ts0CupshXeSdCvX3zrhvlR6Sp0xpcNnGaqnpcvWm\\\\\\\/qNFaoVAbgZmAwSY9QG+TLPfnOZMpSxGBfM80lxDbnXhB3NmLK5j+2ob88r+OYV1cTws1zjBjtnX4YA3xDJ\\\\\\\/MQTTeBEf4p7RKx+ibplIcklDh6LFD9YSwj0zE1ijzu8wzjQiVxIcr\\\\\\\/uwIKx+BFcmHqrmkt5T64+2OkSy3mgA6r6jOei6RNvy+HgmbqzigZPlJSCvpROe6jZ5lEPh6zdBr1tKKVmlwcnHt3UQp3r+6CMj9\\\\\\\/hXdjyZLsW\\\\\\\/SN\\\\\\\/6HFXyiW0F\\\\\\\/HaovqsWwXVPiOFQIlqZhZgK4TB6+Xy5lKVKtqEkj1QcQ83wdXJ9HxGjqzoLokSEt9sgjy7mBNXav6st8vBzpmh1JC5g2JDc0kSYKFcjaz43gpxEwyWnWwOQ+Cs2+KgYEfij\\\\\\\/+P5tARPwrfpsipe23oundiMHU\\\\\\\/xFBdFHV3pvGY+Jl4oje5fecI9pnSYhbED6hBp8O8TtPBGvPNKgf9Y8uhmQmU2mbJoIUJmMM4un0K5+L7baS0T5T1fwAvvW6OuTV3PzQrwWZjK+StSKJg9use7Y1zR2+mk+8k67lWdyZcmtBH\\\\\\\/LqkfP54fZfxy23ezY1g4enQEZak4tfBzFTCLm9RBFpp8rITyzOGBn5hncc5a9BOOo72MbtCr+pFqhZRtLvcaTXzSX3mLtNPcioF+FX9iYWxhXUN8raniYpv1uwSg96BmFvYka5eCKgsQ3fBxOxPpsCWpScwmuKHOgMJwRYnrk2y7BOt84fv7fc8GHC9nKpdaG\\\\\\\/sKb\\\\\\\/wTg5yx45hx1rXp0IEwx+KFKYtbmYauGuvVBJIVmNdjKeaT3RpmChggmS+0G1I2h5lNendTYIBBHRmq2jOYwXHHAsx6s7IhXS2ZPW6Z4aS4yQ7VEG9BUTazmNZoo\\\\\\\/2AczDdwErt1bUGPM8dECkp\\\\\\\/qtULZkpez4aBqiuwaF7RVpQdD1\\\\\\\/Vmbj+K0tuY4cd6Gi9AOP4+MjQcEwrUIlZkdVt+ipYVooA\\\\\\\/70k3iG8RjkJRP2ga3XwZfBHpBCuv6i2lFWO52J5faXnpENhE7352nzEpMQHRpCYXxNAITIn8+QgrTv7VWkdXB\\\\\\\/anDgz\\\\\\\/N4df2w2MrtPqTAMheACcWOm\\\\\\\/RwtD5+0XqVUcCmbITDauwITVwnc412j1FohojtYzD78ImlsvV6cTODfxQbhWRYS6sMgsOmzAfuOAkVZWOnArSoc9bqfhnVOju4iE9AEeiIVEb\\\\\\\/GKJs4N\\\\\\\/IB6LcG0j33xMCh+ePM\\\\\\\/yFQcg6xt1PO9zn7qR6oO+zu+\\\\\\\/c0UiPv3ByBxnhYcVjJcsYI0CEz64OtexEaJyr6x1ftqXhc5ra24ylXqqa5jc3EUGGtsk0L4A9tjB8zKd2nMWygkGy+LJ73BXVRnTqfWTUvLvunp4IDjp4luKDkYlj4+Ik6wFeAXXTXjvnaMIPlyGCf\\\\\\\/iv4VCT0JSNFGk4qeHaXv2La0iNbiVCYs86YaWTdZD8Awo4mOmpmRbGt6TjlKCQfXxQHdXATNS+D3hvoHvn7HZPO+PhLJCYcA+4kSmp8Wj1ff+LPXtxialSIRQXxKlRoX2ph5dRWQT46ml1dsSwDEd17vHB71DJgqJWx6cNH53MnuGUePZRK6g096dzFe5kyzcFtO920emPdZURG+zKGYm+54O+hnG3DiIKkh9YbuIqlO5o+LjYE7h5Fk9C96WiWZ7ssio1mJpJr0cE5JtlX\\\\\\\/GyD1NznI4J6k21Ak9D0Ve\\\\\\\/TiXScJ+WNzs9GDURFQYnKDgiqiekBuzejAg+A3xXrQRDwn+xQIXB52twuI8r6oSdgIG0L6m\\\\\\\/a57\\\\\\\/XWWba97mQwrHMd5UZTPRKBUR2ZXSn9VZYXvTMDzj1mqxGoa9rDV+hsALRALpdzTpRCWrqAb9oFLi\\\\\\\/lLMuukuL15DWBxPPlH7QXJhiwdY5K+dqNaRUbWFsD1VgJYDPubArDsTrzCgKVcg9EIKMEwv30IJVD5y49sryT5dxlh83qXDApOZfebjyHYHlKdgaHoZ8fRzZxomvLs6EGJIDAl5A3cT1HEnu5nswqFRC198nroTcbfdcfLDr6NYt3ms8F8EXQ37eNS1eH3dvOx8a4OOVHgZBmuAsjPmvK6j9ziTfxMRobk1TvUIkfCp3VyzDaEgwOIFI9dW38A3pU21O4jJiOGx5EfWXbHlyFAwhGrEtXPHXhzg+NKCRbChyGP52q0Ij\\\\\\\/71maXZYsao1OTTuVpGxY2UQVyW764Y2hrKJIWMSBsDTs2Ki0vSH8WbmRJB+hsfesEuoKDYELf03zaytWNjFDpxTdEogXPCVnjZQIUib+FEnv\\\\\\\/uOhaUUHsWIfeE7vlgGA5w03W92MwbKagYvwxYnwAnq6bhYeo+cY2fwbxPg1XCeB20oUsms7QCLRuVBaglWETv4ikaXKFa8PjsLzKjfpxat8QDICAOTzNPpD8lTuMU7zmwCqkcCSBx7XPKnFefaLOQ8CWyEC5zkY1OgPpvZjHy+pIGcq3Ikg8KvOfIFZ+hULxA9mEogI6tjpRzBR3lPUr5unD4GYp+yaV4ltD+mQjGGlg9utKmo1uTAzgX+fyu9vtqFZobgMy7ZON4BNL3otI1PffAqukMtnqzdXE6Lnskvq4M9D5N1ioARhL7AP0pw7yWFeMfc+v4EPzSzXfE+wteK2EX74HYneqIQJmesvqWcmTLzKQCKct8+vuBgMscD3tBx5v6dJ+auT86zkznFla8c5gfPPWyNBPJSOAAVW9kNbyrQzF35UyZrxvnAdeRYSevo8FuD9i4E8DqIgH2IPLGx8n6MoDB6WJOr73ZNmcFuviTdGh5g1UeG18RpnhhU\\\\\\\/2Mvai2LIg\\\\\\\/vGhLG8zRzWzzhjp8Hn1KG48kLN2Q+8dqN0+eBgUH9lWxujOU6t0FqE3Nb3XtgNR8lg4m5bnWJ74uI7Ry1Cu1OA9saHUu3vd6w7RPxDMgFciZwrFdSolkYZEWG9iHLZdoKUG9O1HX49Kqcxh5AUELcyuERmNOOC8KgDITgeONbD2TDIsXDbM99d4Qo2KwB2hzs85W4fHCKKHAdw32dl6Ux7puAZmeZs9YS\\\\\\\/ZwKE5S+Apz42MiqQ252IehobC4ud40Ezyl9MFNewQuzUhd6AXkMyBnAg4XjYFAzvJ8SNfpyaSx2tbAQwSM+YbaW8c\\\\\\\/NKXdvXYWUFsPOwibFnD96c5Uw2Gc3ZKZ7XfyhklYH59HEVRUAodzJSL2Tv+Mo+tWBUhoPJUoHK0937fWXGaZ4kevIWPFOAyUCTmCMQkddF17OUmz8D1nwO\\\\\\\/qVFXyPSGLCFljOULW8CLxEXF6eTBRwFDA0Zy0BTnNOqlSrta2hV+AO1NG8uOV1RJEUzHdt4lAIxZk9xIoLMCbrBKWlfjoMFVc7LFVvgn195QUG9bF1kX8PkSKdfmjpqIe4Ny2JskBIZm7EmkPQ4JvXeA+3eKsukeZqBT2SdbRAAGFbvVYpl54SIohsr5daEl3K8yRIut0L\\\\\\\/NJET7OWKQuOaxX1LfVqpObsSadhsxoGLsNIl\\\\\\\/D6g10Xiq3f9Chg0oa0SFdPpq2PEv34nrIPws2V95USI5ishr07MtBR2CBOIZ8nUzhnVWrsNitHoRXYkzNVsLFMZqybka7ibCr2CCAcj3Rtjc+\\\\\\\/XkfulX6Mrl\\\\\\\/7xCgksBoPPhx\\\\\\\/YyqW8egh1PGHYULYxZSlWVVBjsqKuki\\\\\\\/Smrukp7tvoBMaBhs1q\\\\\\\/r17P9z0z3lomG9IfGjtinOhyroPOgbWF\\\\\\\/bbReJ0EPfOuJda892R7xFLEZm2AqfaWhcoABu\\\\\\\/fo52MZ1RuRLGogil9yrqW8AhvBlirYYl6wgpwuvh6FY77RO21FuL+Fjsp2WvtwIEnh31vrgYPziiilQWaXe0kVcb+bLJIwYM6SHCt8xSjAju5yL0FMIrTJ4WyS2P70pPJDytCBiZNq75UtrDy85tpkSMz8NOE+C7rUjrTye\\\\\\\/qgbsKQQUtBhpqeNwGFzpuC5xKVck0JvmhGB94nR31Y\\\\\\\/qkJ59AKoCx4qTL01goMZssMxy+AmlnDxlDE5y4b7GlqIhuN4eroKxcWDRrDqukmfGrS7ntLJxBSAISQI7qUjQJHl5j+0GOSiHrd9wnaOuN2JdGDg\\\\\\\/jVc7cLXqi+4DSK8faqCU28d1kJdNaz047MZ4tdDbZqq3WgpzsRqY4O2vTpgXXFc+9qmDNLkZEFwqYYkyfBuIr8sw9Cc0bKsiaZ4lYtt3eZQVwO59Xr0XLLB7cPf+pivw0\\\\\\\/zlwRRLjyWCw\\\\\\\/ikeYXnV18xmyc2drXe+k\\\\\\\/s6E+3rm0lc4csPKWkq4ZLzaSaZIKxUFnlO6FsYRqxhZwrYwuPSkMwuH5qD+DlJlIe6nQ53wBIWVLFcnFAkJpklvrJFT\\\\\\\/Eiuyed4j9+pJe5NjdaexkuauAoir1QC9uCIsn8nwgGomWU8gND1SF9txNuStInihDpV8FqguRkHUIivyeXjPcfgl5Nb2Uw7Xtxhv3xEbjJ4E58BG\\\\\\\/MtFQxSkGI7+jZMt2ZQW23Na\\\\\\\/MTQ71U0mbkoZ1Szq1BpMLXkvtCKz+klBka30L1l5YDzx+\\\\\\\/F+pcqn8mOpYSch0mQgGmU4An4V4vf65+DDhKwuRhF4tDtzvVXGS0IfrpQchxcO3B1ss4mIzT2qToM5l+SmHkmPzYqi79v1TrpxCSXL1ELW2PD6pSHpvgHz2FsihlffG9NGwfx+8gijkAyKzyC9oegFOdDBmHxzWfnJ91depYPfB+GLTmHipBnnsgc4zDFCfcUvv9OPwV9ZZXZ0aHjWhIt\\\\\\\/oeaTWZzTGEoC4ChvqINHJqiFK9JguF9n80bkbE6MDDs6JQ1q4OVqGD0onS1oRc\\\\\\\/pd2ms5yVHCWUgZ38inTJY7ciHrVX3QatisJOZCRLds0oZGV6OfkK7n1SlnnsY8uupHwMzA3LSaBXeLgIP\\\\\\\/BOP4j5EaLj2gwpTvRhNkh8VTKql14DeiRayWNTM74ymlquo6nkc5CdnHHWLp5kaO4k57lJ2FHK+rpS6RnLXlH9RNFU+dDpCLYUaNmoRsD+f0rDErqf6rNqJwsYkWFkQDl6BBJceSMNSEluELERAbz9dwaDO6g6eRBV6P2\\\\\\\/fQ5\\\\\\\/2cmF1l+LMndZdesSvaAdVO2x5lMh2G\\\\\\\/OSChAJIhpOx4ugQXLgGLAFWdAKF0929cpRHEcipMcDoIhn5yniRJmWg4V3ZgbweTj+HyG7owwZo3WHZQTM4VpalWUmxbFuojOCpUMwyfFLbDN07dFG+vS+NtPOIYsL1uEiy2pQlsp4Cj2jR1sL0dHVnfqACQrMJ\\\\\\\/VGbofHiyyDSA9AFzdenjiRQvCFJLvOZZIvmRbJBkanzGX\\\\\\\/SRkuixResnZOwDOriZ0rf7GEiJS2Z00TdOA2iWfdrMHqEsCW7paWSZGO9C8WOlx384klZaGfHG0rDzd2r\\\\\\\/BhgVbfkdUXCv98l7HaRYSdhK8RwJRlZkgPFb5O6Ur6UIrcG0dlTdh0Gb7ExL8Fl76LCE8Lh6G422zgVWTcl1PcRh2gafIgvz\\\\\\\/53LVccnKr9RSsqzn2duwUOkK1T0u5FAViKPuZmZ5ZQ2wEf4+oYkPmJFQlmeoZiegwvQ+a9bNRDMX7dxqNGdFgyJqMH3FhIzUP8xkTQsHXWxJ10In6OsZrbYt9Ax+aRYFXDZ1FMVi0JhIuZPxkEYzYa1hTddhahUQirkZDAKLGMa12VtQV3aOa\\\\\\\/Ip1wB1fNfAax5w61yeq8bSkR5msU\\\\\\\/NRI4q8F6kitYvttvSVyDkfAf4R3o382OPTPu6BU99XQ8dMVLpkZA0wT4CG+wXjpwbczyHEfL3wYRpklWig\\\\\\\/6b6pZ2hh8eI9CZc\\\\\\\/GKODgw10C9O7n6thy\\\\\\\/+BRuZxbcoFzbXgSQmS+D\\\\\\\/9ctDZAQvixY4mUBD21w55OmLrYp5POHduh\\\\\\\/3\\\\\\\/M+wR9z4VgICBGbhn9JXh8YrD+jxzjpSXKcH4bPV7H5L0q4L3Vg5Vmu\\\\\\\/Fb5YUiiPt1w08u7zx7HJOeUqdDt\\\\\\\/qyFa3dJsFGNdefXmYEQOtxF7k58DWvjvt7qjRqH38NHCJA7Ti7db80ajQqwJHaGitMvivEi09vFDzFALCKGaYBi5z8eXTiRJum8MSCz4gBSQCvaueJhJNe+bbFtGed\\\\\\\/pDEX6ctjhaENWjfo7lYnpACJKb3Vl22CzCWzjVMgvnLlvem6YvjElHi0D7\\\\\\\/ILO\\\\\\\/iLv2lisTmiIyHFkeJFaWRcq8zxLb93c0blEGVBXUGRlM5jnWss9cIwa010O9Z9s\\\\\\\/waxfFO7\\\\\\\/hmN3gMsp4ZgVDS5Qr5il2+1IcgFEmOU++7PnEiCG8K8KrsduvOoFE04hTzQJzM\\\\\\\/jAeZzhBt7OShqabuHrqsNRCS8tZZI1xz3x06rXVAxtjccq5KRzqSyqlgOiXiLdJYvO3CffGQVAkZHnbRrrnyLY5\\\\\\\/D5C76KeMVESbuLljs5GvBjv1DYYLM1l\\\\\\\/YxBqnTD44ghh1bJn4QquADYhQmpWlyvv6Xl7mtGEfCGE+HhfLzRtXQSSf0hhM5T+wT0k2jKwdDHSAl08Tr2b1WHw+I5D43bpC8WJiq7tgqlUVnGOTo\\\\\\\/IKXaMt303s3oSOleTux1\\\\\\\/DyPPZzCBKIwfBK12QOg6aigjfslcvmeDp1dBPlk6KR5532AjkxubKOuKFdI9c7H40C4vv64YBngIk8xnu\\\\\\\/k8PYYhHtV\\\\\\\/k1Th3vkCM32pb49Hi2JNMT1MorBDmsY730QIt8Kf\\\\\\\/K5yhFcZW5keOhRZqn0U0Y2RhaIWVhS3jqIinYp2n4hoKsWTV+zL2EJeWVkEXfSiR4P1FgDCCKv\\\\\\\/NRSfN25tPNXm+oXx9Expfn0R9nK1\\\\\\\/+YUs+8HistTPMECPNuTipFaBVcF6zZOs+0GF\\\\\\\/iU0hcC6gFWK7QwUxbknRNgb\\\\\\\/UjkBryMA+INel9JqSwAa18q8oM3gfuq0GMf1n6lXNNAC3FJnBedY30iQ68EqJ3\\\\\\\/GgWlrNDEATtEh8jBCeS\\\\\\\/6SM3SzfY7n539wEGl7cMyDdGNq5TA5HLQwngrzu\\\\\\\/x7dVeV6yysSEWdQqEeeTNgvqdZjK7l67zlIkmT1HpkDlvkobfCOa8q1P9b+6v\\\\\\\/CRGznrAx2LiH55rEzcVT\\\\\\\/XBNfxzMCKsQW2gmDsIa7JsTMHUoWmoT9rpGfsC6PAhg7+o53ozeYnMTqD5FkB3I2tkgyaUwZL1kt21IMqhrlU0\\\\\\\/aYUR6N0gLXTI\\\\\\\/+GmH1l+h8OTkywKcXIxr5YOGLhIJ+VZMs9jCaAP1kPK6saKnyWLzSeGZGbX88bmkoY5LZnP\\\\\\\/JBLVRVABYUsUG\\\\\\\/1dO7HWIRNps+Gs\\\\\\\/f7dXwpItuJ2jzl8MBKW7Rz7w9eZN+MqWfXOE50xOzrWrdfd9udgKfHV6rBmM9TowbDX8I\\\\\\\/PZaQjp2l8DwQ0Q001nlCy95x0iv7S1HQFJJnEK4emAcnBm6johPXty4gWRV1QqbAtCZx0ar34bXOPctbgC2HXAelgJEAtdvFDo+Z1INIYNkVl2X3w5InoKlYMtRsadBXaxEWTesNmn+fGB3fcS+c1Fh73FCy2v48zaLDKEtaFybNG0B+2p\\\\\\\/Ry5I6e2aZsQoBoVSvgrZfvJJd5RSdrUGD1c\\\\\\\/BBAqo1GHz1dJvh6ZKusRBZ8Y+rv8OtJ+mm52XAFb+SbZHtoiq4O0VcpoPE2CMGfQ5UJqR6OyVaOAq0enxsUiU94LFRD7x5yOZZK\\\\\\\/njEDeUWlMmOb5SVN0XkHzpzoFzTmmL2VFW5ZTk4oM+9WY5sVQ0GDY\\\\\\\/ao4AYCCN13IJa+rFaChz5WVnc\\\\\\\/CCtmefjs7O0\\\\\\\/rIK1I6X2O38iOdyuyImWN6m4jqlBCo1KUyx20X3JGBEIBKnBlxRRx1aCU1HPbmbSksZSSKEevhK7Fl3GDYD5rZGYDXPtDo+cSPrg2HPjhbaVlIV\\\\\\\/7BOIBFPWz6hyP+NctpTKwRl4YjxmDO1UP+UZKdNBkcAP\\\\\\\/i9bsDtK0PMqJR1CZmHXBHZp1qWZX8n5965t7Y+DZw\\\\\\\/h9AlGsAnVTD0Z\\\\\\\/e\\\\\\\/i\\\\\\\/omtA5ZNSFMqR4ZBbvXioJYvmX3V4gofIKSod8v21P+j3W+5ja+DcT+iaaGFE84WDnDU5fr1cE7NO70sxa24jy1N3pjeJw6MmZY6Zxu29SV0riFT\\\\\\\/c96GW2NjnuJq2U\\\\\\\/SaW7wx6rCC8enzqLMl5HincA\\\\\\\/RDNCkBWlzpZyvYmfFon\\\\\\\/HOcW5s5ImMSmqvleF4OhB44J5KJ\\\\\\\/P3rj1Rl9EdHjCHiauyVnkTcceh6Zh8OqfB7a3cBqA3pdrbCdxkK3gQR\\\\\\\/TxIf2t6mC5au1zoK5iy8\\\\\\\/Dn8jiIuYKBysu6EO+79v4ytwv1TpQKa7ftHSohCZKgOKtudRYyVgqRHkIwxG8YMUOOBIB0FHB2Ra12jpHtsOmwxr6EShGpm7pbtkQZKNxkanHshXlti6+w6vmsue33aj9j8+Q\\\\\\\/twku+kyzCF4tJfjbrp5TZFB1y4QFI2f4oRydUv2hoOOBAUAvV7NtSitIO+RLVnUMDpjuWaBdWyh+DL8PO2e7+pRYJULN9+ArSIUUY9S3a4v\\\\\\\/tQeUdg31NLORGQvXjVuudz4XTqby7PAXdK0Q+\\\\\\\/jpNzoK+oztx7LWEOrCueU6y5UW6PuIaFVvTv5NlHp4GNLN3FtgmA+KfbvaVc484M3Q6yBV013+2J1biVDrqjl+o1XGN2\\\\\\\/PKQhVLAMk08\\\\\\\/SXtg+y1u4TmgmgQBsDBbsQ+8At0zHAQ6MfDL6FFJVGAO118db0Kh8qLkNbERdTvgpn+97YEm8uWU5DtkluWrXehwqSKJfWk1ry2TTd3JBwVrxzBxUHWP3vD9zFiF+83Lo2WuXyNr\\\\\\\/TTb4PkKeYvsy7vN5PdVmVvGiyXELllXeqhDVPON1T38W8bIS8R4HZ9FeSnNG2jhI9v8E18tVkyEvNq85BibrSY2wGQbSBg7gquNOyHUNCpD3VmMZEmSUKUcTbU9J9mAS4ltV+XcbeGmmOJfyHvJL7myFNOqm6Rz5ibSjMG\\\\\\\/8WizBmPld4+TWL9EbYisf97AFcPjpNEJZwOCWSrWx0A6r1l2WiK1xOS+IaNOSO8z3KSvHzG1t5abuFcLzN47PkA42Omg2kBApGhsDU0lmc1iI0GAgQ9RyUW9rawUqRtbYlFncJAHWIW5liNroJuhiB\\\\\\\/lI9zOSgnpnOO6oLjiHeUlsJzxJZkVE3\\\\\\\/EPFtuUhLVZzfVQME\\\\\\\/+s0kUEdugumuyxNwTOwzCl30r1ZDglPDWnBjVMM\\\\\\\/6m\\\\\\\/N8Rmm7WEM6w9UTPy150Elewp5rZaH+bESCIWl\\\\\\\/WrmNitHFfu7has9H9F67bicI\\\\\\\/lhzPS+uEK6TTp5KugolG9rEqRe5vtUuOestlordzaZtSoCVudpO7Sv+HBNgj1vXLbOxGrNg3NqhMTgCRbUUJKBhYjq\\\\\\\/cgLiQ2Ag61Qg\\\\\\\/2Ouq6VWlAxJsM4ZuV0pxZcOBwbA+sF284pJSf3X8OQTVvAyJExUXRhS03Y6XnjMmJx4r\\\\\\\/1HUSTx89l19MCXU05UwOzoxqm22b2gbxGVP+pWtPtGG81vvbQGwOc3DUddGnxnJLUjpkr+OvY6rIPLBw4mexU+0wfUR74loi5q7gBZby1re7T+yfM9HOJ+CVzm1vQ3rKOX5ENUZ0ldeVQWAtPHUzWNqRUkE+wT9+RRL3PlMUUMlntBoff9qfJk6wqGGxT5se2EZjud0iSen\\\\\\\/IwgeD11\\\\\\\/6RwtyPlz3TLBYcITBBAI9c2AVlZRSeuwFTNZAwu5uYS51puHpNkKH+cQy6lYVHvSpd6aeHCzz7RAX072L61W43egIJsK5tomuEh2YGp7nJm3PztTgtVq4ZAlj599ONfzjKVILMoAUVFYrtkkEs3asF1BGsBMY\\\\\\\/rJlZXxFTYatPw\\\\\\\/jAzt9qApWHgEUd0T0DggECql4uu2lF1Z+lv8jlNcHf15\\\\\\\/bNrqMyR\\\\\\\/Eg1RYXhSDeaZAQXPVUQ0ftU9Eaf6Ref+xnQZguxwFWdNfT9y37kLtqq4stMo+X2IEBsQr2mGxuSIzpM63OUsRCLSFoy\\\\\\\/ul0pHO+rCaY4xm\\\\\\\/MqMpGF+qlrWFx1qMvbxMXZwVrrhdsrAZRtL07w31gpWI37aifOANhAUcjD8Bl1vMq3vWdL1voTLyDjmZ5bgqptj9in6t1U+kaHCCd0X7VvaHvafO7DDY\\\\\\\/7lVf2Uo8pKns0Lt1zblNeuVq9dNeQsskonVncbviFFfJ2ovbQfuNjP+Jr2WEaE04+1FUrMFTW3D9D3tYN\\\\\\\/IG\\\\\\\/el08Jpq+M5nlJEs8QVPFTKzg4U03V0QCm701pWumTqFC\\\\\\\/gNw1sB4CIbOQGWvH9ssLdEQfZJn\\\\\\\/RsKX\\\\\\\/Tbk4cuz+9vDgr0J1kf+z8gTPqJ+Bz6eU4quWTFmP7BEdEMed2uyaEEkXL+Iss5iV75wmTOGtPPvZLBYxnbYw6rsHjUIzL28IE5s\\\\\\\/Dl4L6C79qVSg8qZc8TuLUPlo7\\\\\\\/7RGPYs+m73ylQWsbLab74ma9SDazn3XG1OnyD7TOrmvXy3k3aYuQI+x4lhY1Y\\\\\\\/Thb3efhxJC6fKvHHPPEoYMFVi9lgAVgqR0Op72P8vXYi8TTsdCLXRaFRp4n\\\\\\\/UyJ3bZk3\\\\\\\/0W4qgX9k2dJAHuwIy6XOneWpHI83Vgw4c15C6bzsM5WjwsLhgqFuGug6oiWYJpzSDlPO6B0uWEsiZwrP169vcH1urOxfaVdE0GOPY6hqYIZnTw3yiDC1SXUSWL1UjnM1B2X7XL3ZmnoSmJHMADa1Qr0ElG4jxgd0OoCEREoNyJzI8WxDzrGSM\\\\\\\/VSPKq63ZqSeQFB1T53e1efv3pH\\\\\\\/\\\\\\\/uiRI0WCgM2Rd5NrUwpOFPJ79xiwXKKeFSd1x4Ae+4E8cuFvck6pNDnD8kzURSsTZD7FUEXPDa3v5tktsGkkNUjHJE7iV\\\\\\\/qJ1KUDatj2rWBkibexlo+EnxDslRlxwkxjLRtEyhGfkt6nAgEAmwSTfuQEoOdairv2eCUYRk1Na0HTRs5rBcyMngXcvSE6vB8pUwTOOi6FylYhBMxZ7OfiXuvb9ObkFI86lnwzmiGlLupRyOdwjlPv83le6B8OmyL+XE5DZVP62GS+UnAbbUz17llT9ZxF5PDgP4bSI9gVfBvfF68R2oDxSkjucStakPjV1OrSiCk7nQhao4tvgbJK2VurWzTO8SjWmKBqgYgiDF7ypipjh6GzABExSxrCL94mUU5S9d7Bkn\\\\\\\/DF9h+0kQsJULFtLcimnE2HgRRIkAtymqxkAxiUvbEXBHceVzJO6yVhJeRbZ4uq115wZ3mA+BTd4uHOztKEjiA+uzJUUIaZNGh2WLJr+YkAXhFajj53Hr3c70mBSA7z08AekuJhI4oUKLfuzL5rsjDAMky29tzxXXiE\\\\\\\/zgsdtyphncC++V+ZlVdlv\\\\\\\/\\\\\\\/KceQTzjTCwQ9BMokaDq\\\\\\\/Li9kKgcrUe4v1QFmVf4lAq6OY6fD0Sy\\\\\\\/04Wnuk1NQjG8WHwUEVroL31vEjgTSM8JeMZAq0xkVoI2cLTd2FUxVqvCMQBGwL2nzwKInuzPOniGr5alCbNIVth0gca4eF19wqgmDaemOIWe2aYA5kUkNnYmDcf2XQjLx1gv6hogDXfWst+A\\\\\\\/hWKdxtHJMyHUVVDcp4sdvl6AvPuJht0JSTw2NxkMsSN4EViBwhoE0RshYAL1bs7JdaBfR3rF0Edy5VfxvVYEP0eSjBWIjNbHAc1uvH55DhQKlrP6MCUIFAw1q6YZhchht\\\\\\\/ieHCUplIdH8rV2wxJb67rRjTl0Vbktfq5vfGywrFst1VwoJwJg1aIOGXvN073I8IP9QWgVvUJPQ0rlIOl0TvaSWwb4FvRRC3mT5Y9MFIu1cs2jmkO3d8rv0c+jnPtuZgfZuA4Mdp+1Ng6xDMcRODzqTwuoc8536G1Udy\\\\\\\/tuIrwmxn4MjkiK48lqyM8SoUTLgz2xnYq\\\\\\\/W3TdeR5Ni3xVnPfdsq6yt22GcAL\\\\\\\/soM2ulQTksWmDdtyEWtSvKwpEzfebx59PxXgsaGDJhciIugRLi+TZoYz5d449fy5LzYgyFC7OOHS2sCct+F0tcuoMcbsfEmcPS3O27cx3Wz4B11OSTeTqFrT3eP8gX3SVe0GDALFecVFP5eqW3ZIvLM1CbQ6LVB1Ga3CPP43JcedTn253nK4EGlYHoW8DwnqWfLb4yxbGI298\\\\\\\/4l4adKnKHcMWZRXRhBXKrKuFKnKMp7zUK3BhMeMxK30KfIBDIZDdJEhxUbj31Dr25EFmSw+a2zWKfyCoCOushidiHSAt40CzjhF4vjEN4qMgNDMqKmGVsVBW+mJPF2UjPZipF36E5ASTZU\\\\\\\/6kOafHvQJ5yKjagkmx2g5qa9Hes+BQp6wpzojoFBo8qtLXiCBgr+sYCfNh6YVouBYvOdHaHB3il9+ylV+sKJalXZkrnPAD46U1P7TqmEovziszqyyMLMPk1WAdBF2EUAi\\\\\\\/qMlQsGzMAHlH2sR+i3kycGTdjT\\\\\\\/IMnjbR4Wx07nKIC+99TVOY\\\\\\\/c1otGZBNXT4tcRof5W7GHp7jQHFHhPLABTNqXx91p0TNgMFoVMYDRIDwEv7HOkwg5Aid6jJEFISg\\\\\\\/cHXStydqyX9ZrlZ+ufERwPkyxmXy9lw0HH9WQnPRiiQledOIGKQVu4d6+F4MRTyrxplFSn\\\\\\\/d4dGfedfnjjAzXrx7mKwsLN76bJPgWKeO\\\\\\\/Rnj\\\\\\\/EJWbY7smyIGPQpGH101Fs57bv7sPJlM9lnsPKnRRSEot3QWCFaMPAhqJiYj02CyFe99QNkDN8rNdyIi+u7ikqqC4vJszuA1YEiwbPD+CbpVULmftE7oYaz8brvolKjxVLf3YyYXfYjMYi0056PWOiFWJtcCGJfzQTkj7r98U4K+EpWsFHkq1nkzQyc2hbOhgMG57sC70IKwJQ19j\\\\\\\/fxVZiMIg6VuAoVLDRrRGWC1WSm+Su5eF9ylKavAllIZkbZk3w6QGoBQoxNZu44hUEwusho8A\\\\\\\/bjzUwkZvVyhXr6VsnInpBCtuaqE0Gxodc3LsW4\\\\\\\/OOsWzO4gNJK7lKFTxsOInyW6\\\\\\\/lVAzUDsAMhbiqxOwDIMLbpCT7NEjvx4KeA4JiV0pTs8RwWsO0Hsq5A5JEy9Jrm7X3QNuM2z2Qamm5QkNB5NEuAZP2UhA5G\\\\\\\/ucUC6LWhehzmvdPmEeM6I0Hfy+Ytgp367yIGg9uqUZK2R0LKXnHnmtf550KUTVxdkymy+gvfZCoYYi9kbzsQr1Ib1eRvwOiXUji38TgjSeh6x0z1yTsUUGBXFioKaK723+OfR7zPOrWaNvT4y5TBJrtacCZVbG31nLfefVwxtSnrV3niqxh6p0fxL6rUVZl+6a1vQxmamwrpR9lzCxrfj2IjHSUkiupJAwNMYh+4+I6CGKnRZDFqFt44I3EoiPJr9tHWt67EpNk1J1n\\\\\\\/3QcypONvomSW7P6tHOs8alrKfwvno00Yitipgj74iEPMyA6E\\\\\\\/Oqrt9mbFszuc2JcLfhW1RYpP7IQdSNh5tyRQyZt2eA7Xd4OKDo5YE\\\\\\\/TTAuV9dnnEGBG1zvq7740WqqHPcZw4xykljxZk7hW64csbz\\\\\\\/shiEEPv2CGC3iXVaCGZ0qMxZqXzkRqgAZoO9MXGu\\\\\\\/JuN1RxTiHzTHvJ7wdpdJ28R+h4ehyo5XiZMgno3fa\\\\\\\/0A9KCkOmjMs2XNL2yK81GBno58i+29wCfNT\\\\\\\/sCskZep5h2e9XnsHOpNp9+8FTZgyqjZuUVUwNarXCLUt8zDg57wnwpizYYmiquHY0iA+WE0p963s6QUi5PoWivmRjtAElE77a9SeNmVvs+qxcPtvMKNkeARnsA7UzFVrK6fj9pevb78nBQW+xCH4rJCT8GymoV\\\\\\\/I8WgE4d+4M\\\\\\\/5Cc9Z6qc1YUr7xD7LWjDD78cl5ItXOcsqndbDRfdCrhIgCQW\\\\\\\/xVJMXhQtObiK7cOCQpdB1U5vKyiGDt1SVVDVgLygfZ3gNjfzpF87KihbWffY+joHyY3Qi1myz\\\\\\\/re0IwLyTJOkok8CHuUpgQ0htBFqnbApaRnsjmaeFLMr5WbbbhuT57g63DvddbgmsbKHM\\\\\\\/UxG\\\\\\\/AbLb4P7w5A5atRJ2ApHSa8IB5OKRjJMcpY3VFibkMSmg64XJp9cKV3EHFIX4rqe\\\\\\\/dsjKRii+FYEOCTv5MztqDikYzSPENpYoiflhAFakHjrH7VkAbONZD1xKSLIfpwUA3aHWEYSWjqRLtp7g9XSy16LUuu+Gpn6qAYkFjc3vgzgivvvzKEth5f7A3pa4ihUtHo6oycDH0ZBuGSsVFrsJxhkV2dAjAXsmu7TW19zbUvX3\\\\\\\/9LSswYZnNaSvIKu79M5iJe3N\\\\\\\/NJGZy7OQOS\\\\\\\/UdSy4pQutYc59VdpO3E2VTFoXqQbPMLje06V1y\\\\\\\/lL2NF8GK46MSIIAdS8RCgFaf7h6WhOZUndu5EW4YEfJO6JWw+gxxAJ4dHnuNbu40N\\\\\\\/GRlRWzsBI4fL3+jGBdGt62HSQl9rvJNiuYf1Euq+igh+PRqkoER7Kj0+MUFwsZFKMh\\\\\\\/sodwOlyFdfMrA1yNqRHKWz5NZYiikf3\\\\\\\/o3v2YpnkOe6DmN4OS1EhNVRZ9Sr9EOtHGTcudPntBzl4i47NuHBAbMA+dA5c+OASK6ryIoyxOGAfzxaCSvHYZ8VjCp31iywV2qMcANWJEac++Z4nWgH+GY8wIqK7xeYryJyUbV3mLbgsDIT2ir39uaA1+aQXhTePQQYIwCSo+h39\\\\\\\/nRUULOQeoCT\\\\\\\/5SgZ7FQTrnl5ZfJwnvPrwF7mVsRyO34Zn4LfSR+bcV+8LQyPmaxhn+F6tZJ4kmIcOUDUddQ+3H\\\\\\\/sYI0Hh4nwRWK2XKrz737ujOVmWquMFZzcvsEEien1RFeuhNiyjwrGvGggdjLVuTqKjYVgBYkNBKBiwTD4YEcu2dcOmN\\\\\\\/D2WIYYePbR\\\\\\\/L8exLWxB3+FtgPLN6kfQfy8ro\\\\\\\/TE8pA3fhs4hVLehSEAiD6WRk05bXBGLcaQqKHrDZm\\\\\\\/YSSTpid0o\\\\\\\/zZN09nnCapcT5anV5+GeCcmaDrMm1uMoaWW6v0fjuF7bL3LbZnqOKSGesjL9GinZ9EnMj4zkpDlSOd3qeldp27wJNyOMhUnwg9pT3QzP\\\\\\\/lvFx2uh+vu9QTKA3y26C9gOoOi6A4HAAa60p2Onk0vB5WSpfB6wb4xIe\\\\\\\/Lb6D11+jqyGhMzUJZH9A5sJdF+vka7qGt55qfib\\\\\\\/Cp6tge9+iDetbHtN2NNcz1v4zsFZ5t9oJokCQ0CsD2vIq6etVssaa\\\\\\\/CwIuIMwVw9JbPxC3CXuMusHIbQjVR1+xHsU1sHthvMh9RH99lcsow9it5b+erqyKuY+y0wFjYakT2BDtq73i0FxNgkmcPna0nilSclhiOIHKhaDRVwvfb72HCGJKPfbHWBCUyJDtLR5r\\\\\\\/lCdQE61H82YCyCXnn4zhMsFHWAGh70\\\\\\\/3ucWOZvCPQhSO8oQrcmmK2FXcFERnnJTWPCHOLNHulhP7l6KBVZ3jKLB243sXeXKU6z7m\\\\\\\/tfOsme7nn08VOxGCcueHq79xwTThnMvsAt9I3gc71cwm05kYK8h1+43G\\\\\\\/peDnXhSlxdzedguGnCYTkUJNYEkskJ05DGIBsohfKF4v8JhhsAB3iSFesM6PQ+Yj71MMtev7RfRRFyhKUml3qS4NL5ZnuktjzJXOYK3Q+BwZFcvZJgun7CyINo64L85zpzUV9a1\\\\\\\/4S2\\\\\\\/oOOq6whHusoaQX2K0upas\\\\\\\/lrflirxXfNteSyEMXX1NPvL3u9vyTpsZQAfysuyG6ecXfB36Xmq8ZqcaDzMVEKz5A2jJ3eHjlSMbzigBFWjBT+VNCPkj7r7XMvveevpe1cr6srtLt649w1IpYW0NCKj2dnh1aYVKElmtC8p3ny7XrZ3BkrSiHlpXde0uWql9VCS8CT3CDUwz8HN\\\\\\\/5OotpAadTgodc8s5ZpiPs3OX5TPHBYvKkFA1\\\\\\\/h\\\\\\\/kDWRLqoyHuWGxFjgc5qVeHNAcQT5kR0ixWWBbumJ47kCnhWrN6xwCGCqmFrxZhfSVogTw3Df7AUoILf+pTy0qRxe2SazndU9EoFThc7snUZ7hfuI9e66wsnVFLSGXIYe8KWUNaeTHoIeY0reKCX\\\\\\\/\\\\\\\/TepME6oQetyv4vM4oNz\\\\\\\/wVw0JWX2i84U0ykhFzeEQRY9QzzJ+aiAvdgcPCIQsyk4gmreQpQb\\\\\\\/DWq6gMraH29nIThtS+kvVYA7R7HLIrNcgqfo78OQjF93Q\\\\\\\/vKJld6fRlc6Ey2\\\\\\\/84gHuUVKtmIuo0t\\\\\\\/8Lz2wv9p02g412ztD81mdu0rQkRzjuK1hOkKo6ULZE3Of0LrKIxitsXTPaT76P4TXc2okChEm3807g6yq9jPYeR8XUfg4mEnTjEQ1bSiH066pCzIavklSdw8KxnmQN69otiqlCpwxRHLAosj2zllDRTvxNk32Qi8HobHLqcGMmJsD2f0J++i8lzUK\\\\\\\/OKH13LXgTDycW8Hbo+iO5cH8PRbz2Y\\\\\\\/nOT3ADKaSYutwExonrJkCr\\\\\\\/\\\\\\\/i+UW5L5ZKtai5MS8RBBiRrr7hZyYYh317lLF6venJGe3m9XTt5fdc2jRxOxL\\\\\\\/yCV2MS7zTC7+MiS85gZv1t+D6telCntKH5IiiokMgkQa4MYNmFZeKeq7UFe70zQu\\\\\\\/6Urkdb7eCn7ycozEOBP0oi8Uemps\\\\\\\/Fj6fSCOubY4FNtK8rIuQPMLyhIbAXF9d2KQylDxENw00sR0WfHk480s7MB85Yr7Uj7zeAglCsKDqhXj\\\\\\\/hHGO30WmhagwM4EcZqIhKMDq2Z7hgQwdoM8J\\\\\\\/CJX2erDHn6acPBBX6yT\\\\\\\/8ZJlQwuzKHehWhmW+pJQGueVlD16+kqakHHDKYrGsDMuh9\\\\\\\/6LjDqumYyhgPwhxLPRk3\\\\\\\/uk1eKQB5Xh0S9IPlRNR3\\\\\\\/U9wGQFaOEm+tXHLbBHZHRSCxaa6xxiIRGAvPtqXp+9Dc\\\\\\\/01iHBEwDVWI9B2WJ0YvtZulCfv\\\\\\\/gTdeyXimmneDXJqk7uM6mQQ5tarWq9xp+DDWxNdv1s2jMKl2UwKK\\\\\\\/Vfm09p3RBXi\\\\\\\/dKqnFQpBrqYR6vkb+dqGaBdDoJ07pRyvzxg+V+pDTjpZjyv\\\\\\\/tUQf7UjIlyWS\\\\\\\/3RJXtpUMxddW1YvHEvd7j6KmJWWDZ0ogQmslCzrtzCgYJ7+7DYwEo\\\\\\\/wHLI\\\\\\\/+P1\\\\\\\/kSIMKwaE4XY51i8\\\\\\\/jIKhmx7eyeYmusA3IqlbP6jFcTzXC6ekrvglZUJshOvAA6Zn9sWzO0DJ4E239ViV47KZ\\\\\\\/6dk95\\\\\\\/RfwIjy5g6ZIXXqfH5gdlM3NKx6rq3Od7PBbxi2YX4pc4nVHX3Dpq4GvwTiaTCfo2wYPay+jpL6X1Nhy+++0Si1yL3pGU5xAA6VKkbwNu3I5BB\\\\\\\/Zb4VJSt9woIxB23k4LM\\\\\\\/IzjsASEkUCPEZHPE+c0McMuG8r\\\\\\\/FdupeoCM0+I\\\\\\\/8oQmeBKmgGhireb\\\\\\\/K9S7XqCTiBTN0MzVhtSlAMlNdOOVuX3J+A4AqjdJMYiAeusUaszFnEVrtVYst5vdMe\\\\\\\/B0dc3LgYJicWF6j3P01jDjIggLYOWqFkwVt0Zh3mSG8r899w7IqPKuQbPnRu7UsAZ\\\\\\\/EvwuP60b2gf+qBeHKzLjEOPdehEnCZx3D+Ppv1oVuv97FKVV0teLlq5CKtzd5IuwASknxKx64YgmSnjs7LvCBJNnODaLt+oApe1srnfXBc2FgrYFpqSrEqArv73iC\\\\\\\/NDV4HzbukFRb1S6deXjsXn5dpUeN8342cJY+0vd32\\\\\\\/RxV8u0ljTGR5kkoKPtcRIveqkQXapU+YQiw1Iofh6nUA5\\\\\\\/uM1yaUXPpFnuiaV8MkiTMyqMcjb\\\\\\\/frpoaS3Fbf0VpJZJjQJ9T7eLjRfhrEFBfqWLoeaA4PuyJ+y1LYK1AD0PR+G27AlkJLGzvv5UeXAb3c2Lqn0b2tZ7FYaYXks+izCYhJdvGRlYWkXCikSl4uW19MKClfNw77p9bsr+DUciJ249Ep+VZcJn6f9cVk9EJ95kHYdIxsTvR23Fe10tMnETwCOtpfDhMRhOgki7EclvwUfJguMPvZewDT1OmQ581mo+ypQuh3rIXVwzQIxpGQwVjsozhEG7C6nHPxlRDqQwQN9gnThnOzrre6onVywg9MWEmTRAhaW\\\\\\\/hFzyj5zH1SL8o+LJF3O429XiqdAgc5s67FnP9kt28ORAUJjUbW59QTrxnFGpvuzK\\\\\\\/i78r2m3p3nNa6O0DO34bpbdasrrzbbtUND1lfmi1qq5GHli8y\\\\\\\/qMz0aMlY+Vlx45AighhOKC3EVOyjprSAUc5GJvG3iz\\\\\\\/tyCsvl35d8nWa92V6WsoA87gLxoVewZD2Kip4xl\\\\\\\/FRExKQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iip03L6hiOiBEHIZIIAMKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220bb8857451ef46dd0588e13eb8ef3d1c98b5a9a8dae4fbb1ca099230c3999c1e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZY5dxvti+Id+nNo2TRIlUoR4RMrwvjeEt530SUyFjleCqwm9m7D+2AiSL8oiQhpEsDouVrt3nQyNJzUDLl7SF4sb3RlE\\\\\\\/Hf5MLsQvYWxycFGa4VbgwuDSRimrl3nk3ubvqHprzbcuefuCR7ulBMAegGRDFaZvN9BUzP6qFRRtz6MplBt+VU1Gi+hmw9tbKSwQKRcuJH7AD4RRBL79MY0j7YYatJpJi\\\\\\\/BLtM\\\\\\\/o4z7IDbGp\\\\\\\/yWbLL3msG\\\\\\\/u6xn29gSHRi0QEo6T4K+XBdoeuFKKjdL5rdxM2V2\\\\\\\/rEWdoQr\\\\\\\/XCus0b8z3Q00BXYQJiYTStvLf6XDYv3ASt8cyFjo5M4\\\\\\\/LSpnq1Ol++qR2cdlBCl24OfAy6sAUzNGCe1eG6vzNG9i6cKd8Aq\\\\\\\/2UMk\\\\\\\/o7y\\\\\\\/jw9zYt8dU2Yjcp+bYyzKZB88ZhQa7erU0wlrUUN2uwEofV4+kKHuPgNTcFp0CL4Osljm2KUbYnL+5izeWUp71bR1jtVo2gLF\\\\\\\/MtVsL9yeOPQkalSh27uupq7JgBU7rSMkOuQZZ0OUQFWkrvquW\\\\\\\/5e6sXVvA7vB0rqZzQ4jvotzyylw4PC438OdfN7nKHtVUsSDIVSrZG880LROKE\\\\\\\/6cQkrqtq\\\\\\\/tjPCDiU6\\\\\\\/TiL4X0Qu9sD1njQauWgcVTU8hyiAvGdx0vTVfi0hjs4hcscooHjRdXw\\\\\\\/HxVQ2RBoTlmrrWt\\\\\\\/8Av5cYCVXBXzfnC4UzJHrg311CC9m5RS+zIo2pRMqiGBAwGDtUnxO3CbunokCUAED3YVO4ovjlk2kScT2Lh8mVc9lWYo28K9gVniG6S0jo8iZdWUVw45b7oZrGmXes\\\\\\\/MJMp6\\\\\\\/fjwt+Y5ofnkoh86h5gfdkQg+yTb9Tjwg3GzNUK6bpLXb7fYOa4pifxFANFTpiWoiS7T\\\\\\\/Iea6Sd7SfyIWUu1F1qbDtrFxgNz2Xbbzm8nweQtFEncN+1YkpaSpt3xuB80WIM6D3yLoCcgkjBHWLd1loFxmEVQFDcYl7phoaQ8QPMCeZVqVzhAMzSihKwpmym9l9EnN6r1JpeBNmP3izgDeeKpG8IocHZFrq0Wvz\\\\\\\/2FPL9BkZsg2DEKOkXmXFPyWTT3exTnk5jz2+Q+9rEOUJ6q8A9Ci5X1sTImV8wO5Wnv5rCZM7mD5yuGQgzOdPbr6VTIEjQYxa7NPY9bZjGKzco9FwyQfZ7Fx6ZYTOIrHEyvJ82167ZTZ7wSKUrj+Ka7yocBhOjRH8D4ueNqs23+qaI\\\\\\\/lixdIuMH96sJ6JxBLELnYXysCRt0L\\\\\\\/fVA5YR4LQvXmk5fwKcVD1s8ApYm\\\\\\\/NUcF0LNoo7P4Rg1zDpj5KRNLaojvggH3R0BlC6bLoLKg5d+WtWlE0MnLWCyVuQDSJCzN3HFjtdkeJCCVxAXI+9Wrz8VrHs1i6xvDGIJF8yDYpNRFxPa26q9OpngDHK9TD6nKoM76NLrLGWCl12qMcYhj4b0fEH6xjjj7AG6za4SHtlD2oDCzWuot9a0wluf1viwbyJccsO1jfvaJyWvwW5JCufoRiplpnG7CkoQzU+\\\\\\\/f8b3RKUTQDgsqkohopD2mnwaRG5tahM2rR31hrzMXB\\\\\\\/Ek0BlgW\\\\\\\/Xn5OJ0da5xC8zO4qBppp3rzb0JLvl2HrbcegDvxuDFSMwmkneF\\\\\\\/ZCopiaQgTY2kvjjHDXmYAbEU1HypGY1Fb94d\\\\\\\/IqDo9xIK1vEjvnAaVrPXQW70R23mveik7rv38aQ8BpKyTH9q9oCoNBOSxlwodmKCLxuzxu7BmvZhXfniuSte71pej7IksKTbywfuNnOBqHfRc\\\\\\\/rDJbzF69mfV55+3XZOz+0ajETpzn\\\\\\\/4Mkg6lMsJQecd5KbaSgOBEAUDqTp7p4bWGctMc6kW7y78qMbfnIJXkQVRs3UlPkxitLHtMwZ\\\\\\\/JQUhVJoEReLTgZX0TGluZXZ2eHcyivMYn78uwMOW8MdVemDkI\\\\\\\/beOZ9\\\\\\\/LZZa61dcRF7mb2ij9b0HPwQsjSzIA+onq3QiAR\\\\\\\/+tNdvrnBFLacIhDNE7XYjyAkbizfg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022dbyCaUk\\\\\\\/A051ZNXg8Syuuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002264688dcca4afd9529c732d36bcfff307dce246f0673d46eeb5f305b853a78df6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460366 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1635" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Lo9CUg9YMdVy7uSL1fFdpOo6ioHC4ZxZD4fqx9lISvc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1635" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Q8P6QEarQP3wOQtH_f0s_38TXuwimtPdcEgCkc8GAFA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1627" title="The Immature Fiber Mutant Phenotype of Cotton (Gossypium hirsutum) Is Linked to a 22-bp Frame-Shift Deletion in a Mitochondria Targeted Pentatricopeptide Repeat Gene" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1649" title="Saccharomyces cerevisiae Tti2 Regulates PIKK Proteins and Stress Response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460366" id="node460366" data-pisa="ggg;6/6/1635" data-pisa-master="ggg;g3.116.027656" data-apath="/ggg/6/6/1635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-Population Selective Genotyping to Identify Soybean [<em>Glycine max</em> (L.) Merr.] Seed Protein and Oil QTLs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Piyaporn Phansak</span>, <span class="highwire-citation-author" data-delta="1">Watcharin Soonsuwon</span>, <span class="highwire-citation-author" data-delta="2">David L. Hyten</span>, <span class="highwire-citation-author" data-delta="3">Qijian Song</span>, <span class="highwire-citation-author" data-delta="4">Perry B. Cregan</span>, <span class="highwire-citation-author" data-delta="5">George L. Graef</span> and <span class="highwire-citation-author" data-delta="6">James E. Specht</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1635-1648;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027656 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460366" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Piyaporn Phansak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piyaporn%2BPhansak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phansak%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiyaporn%2BPhansak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Watcharin Soonsuwon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Watcharin%2BSoonsuwon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soonsuwon%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWatcharin%2BSoonsuwon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David L. Hyten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BHyten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyten%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BL.%2BHyten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qijian Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qijian%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQijian%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Perry B. Cregan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Perry%2BB.%2BCregan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cregan%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APerry%2BB.%2BCregan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">George L. Graef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BL.%2BGraef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graef%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BL.%2BGraef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James E. Specht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSpecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Specht%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BE.%2BSpecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jspecht1@unl.edu" class="" data-icon-position="" data-hide-link-title="0">jspecht1@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/6/1635" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460366" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460366/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1635.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460366" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460366/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1635.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460366" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460366/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1635.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460366" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460366/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/6/1635.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460366" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460366/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Plant breeders continually generate ever-higher yielding cultivars, but also want to improve seed constituent value, which is mainly protein and oil, in soybean [<em>Glycine max</em> (L.) Merr.]. Identification of genetic loci governing those two traits would facilitate that effort. Though genome-wide association offers one such approach, selective genotyping of multiple biparental populations offers a complementary alternative, and was evaluated here, using 48 F<sub>2:3</sub> populations (<em>n</em> = ∼224 plants) created by mating 48 high protein germplasm accessions to cultivars of similar maturity, but with normal seed protein content. All F<sub>2:3</sub> progeny were phenotyped for seed protein and oil, but only 22 high and 22 low extreme progeny in each F<sub>2:3</sub> phenotypic distribution were genotyped with a 1536-SNP chip (<em>ca</em>. 450 bimorphic SNPs detected per mating). A significant quantitative trait locus (QTL) on one or more chromosomes was detected for protein in 35 (73%), and for oil in 25 (52%), of the 48 matings, and these QTL exhibited additive effects of ≥ 4 g kg<sup>–1</sup> and <em>R</em><sup>2</sup> values of 0.07 or more. These results demonstrated that a multiple-population selective genotyping strategy, when focused on matings between parental phenotype extremes, can be used successfully to identify germplasm accessions possessing large-effect QTL alleles. Such accessions would be of interest to breeders to serve as parental donors of those alleles in cultivar development programs, though 17 of the 48 accessions were not unique in terms of SNP genotype, indicating that diversity among high protein accessions in the germplasm collection is less than what might ordinarily be assumed.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/germplasm-survey-tool" class="hw-term hw-article-keyword hw-article-keyword-germplasm-survey-tool" rel="nofollow">germplasm survey tool</a></li><li class="kwd"><a href="/keyword/qtls-pleiotropy-or-linkage" class="hw-term hw-article-keyword hw-article-keyword-qtls-pleiotropy-or-linkage" rel="nofollow">QTLs: pleiotropy or linkage</a></li><li class="kwd"><a href="/keyword/rare-alleles" class="hw-term hw-article-keyword hw-article-keyword-rare-alleles" rel="nofollow">rare alleles</a></li><li class="kwd"><a href="/keyword/nonunique-snp-accessions" class="hw-term hw-article-keyword hw-article-keyword-nonunique-snp-accessions" rel="nofollow">nonunique SNP accessions</a></li><li class="kwd"><a href="/keyword/selection-bias" class="hw-term hw-article-keyword hw-article-keyword-selection-bias" rel="nofollow">selection bias</a></li></ul><p id="p-5">Soybean [<em>Glycine max</em> (L.) Merr.], produced mainly in North and South America and Asia, is high in seed protein (40%) and oil (20%). These two seed constituents are consumed worldwide by domestic livestock, poultry, and fish (<em>i.e.</em>, soybean meal), and by humans (<em>i.e.</em>, cooking oil and Asian-style soybean food products). Soybean seed protein is inherited quantitatively, though more in an oligenic than a polygenic fashion, and is highly heritable (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burton 1987</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wehrmann <em>et al.</em> 1987</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wilcox 1998</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cober and Voldeng 2000</a>). However, highly negative phenotypic and genotypic correlations of seed protein with seed yield and oil content have been routinely detected in biparental breeding populations (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Burton 1987</a>). Long-term selection for greater yield has also depressed protein and elevated oil (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rincker <em>et al.</em> 2014</a>).</p><p id="p-6">When soybean molecular markers became available in the 1990s (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Keim <em>et al.</em> 1990</a>), the detection and mapping of soybean quantitative trait loci (QTL) soon began. <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Diers <em>et al.</em> (1992)</a> was the first to detect a major seed protein and oil QTL on soybean chromosome 20. Many seed protein and oil QTL have since been reported, and a listing of these, as well as QTL for other traits, can be found in SoyBase (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Grant <em>et al.</em> 2010</a>; <a href="http://www.soybase.org">http://www.soybase.org</a>). However, nearly all of the protein and oil QTL reported to date have not been confirmed, and not one has yet been cloned. The additive effect values for these QTL are likely inflated due to the use of small population sizes in the published reports, because of an intrinsic QTL detection problem known as selection bias (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beavis 1998</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Xu 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>).</p><p id="p-7">The parental sources of most high protein genes (<em>i.e.</em>, QTL alleles) used by soybean breeders are typically the high protein accessions acquired from the USDA Soybean Germplasm Collection. Trait data are documented in the Germplasm Resources Information Network (GRIN) for the 21,728 <em>G. max</em> accessions present in the collection as of December 31, 2015 (<a href="http://ars-grin.gov/npgs">http://ars-grin.gov/npgs</a>). For just the 12,141 <em>G. max</em> accessions in maturity groups (MGs) 0–IV, substantive variation clearly exists for each trait (Supplemental Material, <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/2">Figure S1</a>), though it is also evident in this large set of accessions that seed protein exhibits a negative relationship with seed oil and yield. Knowing the allelic status of the seed protein QTL in these accessions would help breeders select donor parents, and also allow a focus on those QTL that have an allele that exerts a large positive additive effect on seed protein, coupled with a smaller negative pleiotropic effect on seed oil and yield.</p><p id="p-8">The allelic diversity of soybean seed protein QTL in germplasm collections would seem to be best addressed using an association analysis method (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Thornsberry <em>et al.</em> 2001</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Semagn <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Korte and Farlow 2013</a>). <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> (2015)</a> recently conducted a genome-wide association study (GWAS) involving all of the <em>G. max</em> accessions in the collection with protein and oil phenotypic data (<em>i.e.</em>, <em>n</em> = 12,116) that had been genotyped with a 50K single nucleotide polymorphism (SNP) chip. Strong signals were detected on chromosomes 20 and 15, plus weaker signals on chromosomes 13, 6, and 5. The authors of the latter study noted that the use of large numbers of accessions for GWAS greatly improved statistical power, and provided exceptional map resolution for the ultimate identification and cloning of the causal genes underpinning the major QTL on chromosomes 20 and 15. However, the rarity of a high protein allele at a QTL (and the coincident rarity of linkage-coupled alleles at QTL-flanking SNPs) can be an issue when selecting accession samples for GWAS. This was evident when <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> (2015)</a> stratified the 12K accessions into smaller subsets by sorting them into seven different countries of origin, or alternatively, into eight different MG classes. In some country subsets, and some MG subsets, the frequency of the protein-enhancing alleles of the QTL was lower than the GWAS minimum allele frequency (MAF) cutoff value, thereby resulting in no detection of one or two or all of the above-listed QTL. Thus, despite the power offered by GWAS in QTL detection, this “rare allele” problem can result in QTL not being detected in GWAS that were previously identified and confirmed to be present in biparental QTL mapping populations—wherein the frequency of the two parental alleles at any <em>segregating</em> high protein QTL is always expected to be near 0.5.</p><p id="p-9">Selective genotyping (SG) was a term first used by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lander and Botstein (1989)</a> to describe those cases of QTL mapping in which only the most informative individuals—those occupying the lowest and highest tails of a phenotyped trait distribution—were genotyped. A trait-based QTL detection approach had previously been conducted in plants (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Stuber <em>et al.</em> 1980</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">1982</a>). <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lebowitz <em>et al.</em> (1987)</a> and <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darvasi and Soller (1992)</a> subsequently formulated and discussed the statistical issues relevant to SG. When using SG, one must still phenotype the entire population to conduct an unbiased QTL analysis (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Darvasi 1997</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Darvasi and Soller 1992</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Muranty and Goffinet 1997</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sen <em>et al.</em> 2005</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2009</a>). Optimal efficiency is usually achieved with SG if one does not genotype more than the upper (and lower) 20–25% of the mapping population for a given trait. <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sun <em>et al.</em> (2010)</a> noted that the optimum size of the tail proportion of a population was governed by a balance between QTL detection power and total cost, which was reflective of the ratio between genotyping and phenotyping costs.</p><p id="p-10">Soybean breeders typically rely on near-infrared reflection (NIR) instrumentation to estimate the seed protein and oil content of germplasm lines (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hymowitz <em>et al.</em> 1974</a>). About 100–200 soybean seed samples can be nondestructively phenotyped per hour of effort. Seed protein and oil phenotyping is relatively inexpensive, though it is labor-intensive. Thus, SG would seem to offer a cost-effective means of conducting a QTL analysis of multiple biparental mapping populations segregating for major-effect high and low alleles at seed protein QTL.</p><p id="p-11"><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ayoub and Mather (2002)</a> demonstrated that if SG had been applied to just the lowest 10% and highest 10% of each trait in a North American barley mapping population, the resultant SG-based QTL analyses would have been sufficient to detect <em>all</em> of the grain and malt quality QTL that had been identified based on a genotyping of the entire population of about 140–150 lines. This publication triggered our interest in using a multiple mapping population SG approach as a means of surveying a large sample of high protein soybean germplasm accessions for the presence of high protein alleles at known and unknown QTL. The availability of a 1536-SNP marker assay—the Universal Soybean Linkage Panel 1.0 (USLP 1.0) developed by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hyten <em>et al.</em> (2010)</a>—in a 96-well genotypic sample format, was another contributing factor leading us to examine the utility of a multi-population SG strategy to identify alleles of QTL that condition high seed protein, but which may have a low frequency in germplasm collection accessions. Rare QTL alleles are difficult to detect in a GWAS, both in theory (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Raychaudhuri 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ladouceur <em>et al.</em> 2012</a>), and in practice (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2015</a>). In that regard, we hypothesized that a SG strategy might mitigate the traditional GWAS rare-allele problem (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Korte and Farlow 2013</a>).</p><p id="p-12">We thus report here on the use of a SG-based QTL analysis to survey 48 soybean populations, averaging about 224 F<sub>2</sub> plants, derived from the mating of 48 high-seed-protein soybean germplasm accessions in seven MGs (spanning 000 to IV) to one of seven high-yielding lower protein cultivars with a matching MG. The ultimate objective of this study was to discern whether a multiple-population SG approach could be used to identify and map both known and unknown protein QTL in these high protein accessions that might serve as donor parents.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Parents and population development</h3><p id="p-13">To minimize the segregation of major genes controlling date of flowering/maturity in the F<sub>2</sub> generation, the high seed protein accessions of a given MG were mated to a high-yield cultivar of ordinary seed protein content of the same MG. The parents are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, with each M-code-designated male parent listed just below the respective set of female parents to which that male parent was mated (except for MG V P1183, which was reciprocally mated to MG IV P1181M). The phenotypic data in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> (except as footnoted) were extracted from the GRIN website (<a href="https://npgsweb.ars-grin.gov/gringlobal/descriptors.aspx">https://npgsweb.ars-grin.gov/gringlobal/descriptors.aspx</a>). The 48 female parents had a GRIN-based seed protein content that ranged from 473 to 529 g kg<sup>–1</sup> (<em>i.e.</em>, zero seed moisture, dry weight basis), whereas the range for the seven male parents was 382 to 430 g kg<sup>–1</sup> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>); the latter range is typical for cultivars currently being grown in the North Central United States soybean production area.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468273/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468273/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The 48 high seed protein accession female parents, and the seven ordinary seed protein cultivar male parents (M-suffixed codes), ordered by soybean maturity group (MG), and then by mating and parent code. The seed protein and oil values listed for the female and male parents are those available in the Germplasm Resources Information Network (GRIN) website (but see footnote for exceptions).</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">Pollinations for all 48 matings were made in the summer growing season, and were successful in terms of generating putative F<sub>1</sub> seeds that were individually hand-harvested in the fall and packaged by pod. The F<sub>1</sub> to F<sub>2</sub> generation advance was conducted in a greenhouse. To ensure the authenticity of putative F<sub>1</sub> plants, a known parentally polymorphic SSR marker was used to genotype each F<sub>1</sub> to confirm F<sub>1</sub> hybridity in the 48 matings. Marker-confirmed F<sub>1</sub> plants from each mating were individually harvested at maturity to obtain F<sub>2</sub> seeds. Population-specific F<sub>1:2</sub> seed progeny were planted the following summer into 48 single rows (30 m long; 76.2 cm row spacing). About 300 F<sub>2</sub> seeds of each mating were planted in a row, with a goal of obtaining about 250 F<sub>2</sub> plants bearing F<sub>3</sub> seed. Parental seed and confirmed F<sub>1</sub> seed also were planted in repetitive sections of the same row. All F<sub>2</sub> plants were numerically tagged after emergence (during leaf tissue collection), and surviving tagged plants were gathered at maturity to be individually threshed to obtain F<sub>3</sub> seed.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic trait measurement</h3><p id="p-22">The F<sub>2:3</sub> seed progenies, the F<sub>1:2</sub> seed progenies, and parental seed of a given mating, plus seed of four checks (<em>i.e.</em>, breeding lines known to be low or high in seed protein), were evaluated for seed protein, oil, and moisture content using a near-infrared reflectance (NIR) analyzer (Infratec model 1255 NIR Food and Feed Grain Analyzer, Ultra Tec Manufacturing Inc. Santa Ana, CA). The four check samples were used at the beginning and end of each day to confirm that the NIR instrument was operating during the day within its performance standards. Seed protein and oil values were output on a zero per cent seed moisture basis.</p><p id="p-23">One complete replicate of the NIR-measured protein (and oil) data was obtained for all available F<sub>2:3</sub> progenies in each of the 48 populations. Though each population required about 2 hr of assay time, only two (and on occasion, three) 2-hr assays could be conducted on a given day due to worker availability, instrument warm-up and prep time, etc. Thus, this 48-population NIR assay effort required about five contiguous weeks of workdays to complete. The F<sub>2:3</sub> seed progeny in each mating were then ranked from lowest to highest based on their measured seed protein value. After completing a second replicate of NIR-assays of all progenies in just two populations (<em>i.e.</em>, matings 43 and 44; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), it was determined that the F<sub>2:3</sub> seed progenies present in highest and lowest 10% fractions of the first and second replicate assays were essentially the same progenies (data not shown). Thus, to reduce the phenotyping effort and time required to identify the F<sub>2:3</sub> progenies occupying just the lowest and highest 10% fractions, a second replicate of NIR measurement was performed only on the highest and lowest 20% fractions in each of the remaining 46 populations. In each low and high 20% of 2-rep means, those F<sub>2:3</sub> seed progenies ranking at the extreme ends of those 20% fractions were selected to become the corresponding 10% tail fractions of the seed protein distribution. Leaf tissue samples of the F<sub>2</sub> plant progenitors of just these extreme progenies (<em>i.e.</em>, 22 high and 22 low protein) were subsequently used for SG.</p></div><div id="sec-4" class="subsection"><h3>SNP marker genotyping</h3><p id="p-24">Standard methods for leaf collection and DNA extraction methods were used (for details, see <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/1">File S1</a>). All steps in the SNP genotyping assays of the parental, F<sub>1</sub>, and F<sub>2</sub> DNA samples of the 48 SG populations (<em>i.e.</em>, a total of 24 plates) were conducted by personnel at the Soybean Genomics and Improvement Laboratory, USDA-ARS, BARC-West, Beltsville, MD, using the Illumina GoldenGate assay and an Illumina Beadstation 500 (Illumina Inc., San Diego, CA). A soybean-specific USLP 1.0 GoldenGate assay had been developed by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hyten <em>et al.</em> (2010)</a> for 1536 SNP markers that were distributed (relatively) uniformly across the 20 chromosomes of the soybean genome. Automatic genotype calling for each SNP locus in each DNA sample in the first 10 two-population plates was conducted using Illumina GeneCall software, but the newer BeadStudio software was used for the 14 remaining two-population plates. All automated genotype call output was manually examined and adjusted as needed. Illumina base-pair allele calls were phase-translated into two-character genotype codes of <strong>AA</strong> for the high yield (normal protein) elite male parent, <strong>BB</strong> for the high protein accession female parent, and <strong>AB</strong> for the F<sub>1</sub> progenitor of the F<sub>2</sub> population, but were subsequently converted to single character codes of <strong>A H B –</strong> (<em>i.e.</em>, dash was assigned to missing genotypes) for use with linkage and analysis software.</p></div><div id="sec-5" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-25">The distributional statistics of the F<sub>2:3</sub> phenotypic data collected for seed protein and oil content, and their phenotypic correlation (in each population), were examined using the statistical and graphics R software (<a href="http://cran.r-project.org/">http://cran.r-project.org/</a>; version 3.1.3; 2015-03-9). A Shapiro-Wilk test of normality (Type I error criterion set to α=0.01) was performed on each of the 48 seed protein and oil phenotypic distributions. A Pearson correlation coefficient for protein and oil was also computed for each population.</p><p id="p-26">Individual F<sub>2</sub> plants (and the F<sub>3</sub> seed progeny each produced) were the experimental units in this experiment. Because F<sub>2</sub> plants cannot be naturally replicated to obtain an estimate of environmental variance, NIR assays were performed on the seed progenies harvested from the multiple homozygous female and male parent plants that had been grown in interspersed sections of the same nursery row containing F<sub>2</sub> plants. Parental assay data were used to obtain an indirect estimate of the environmental variance using the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-3.gif"/></noscript></span></span>were the respective phenotypic variances in the seed protein for the seed produced by the high protein female parent, and by the high yield (but ordinary protein) male parent, respectively. The genetic variance component of the F<sub>2:3</sub> progeny phenotypic variance was then estimated by subtraction, using this formula:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-4.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-5.gif"/></noscript></span></span> was the F<sub>2:3</sub> progeny phenotypic variance.</p><p id="p-27">A broad sense heritability (H<sup>2</sup>) estimate was then obtained in the usual manner for each of the 48 populations (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo 2010</a>):</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-6.gif"/></noscript></span></span></div><div id="sec-6" class="subsection"><h3>QTL analysis</h3><p id="p-28">The R/qtl software package (<a href="http://www.rqtl.org/">http://www.rqtl.org/</a>) was used in this study. A *.csv file containing phenotypic and genotypic data in a R/qtl csvr format was prepared for each of the 48 populations, and then error-checked prior to the QTL analysis (for details, see <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/1">File S1</a>). The maximum likelihood method of interval mapping, using the Expectation-Maximization (EM) algorithm, as implemented in R/qtl, was used for QTL detection (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Xu and Vogl 2000</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sen <em>et al.</em> 2009</a>). Estimates of chromosomal QTL map positions in each of the 48 populations were obtained not only for the SG trait of seed protein, but also for the non-SG trait of seed oil, primarily because of the well-known coinheritance of these two negatively correlated traits. With SG, stratified permutation testing was necessary (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Manichaikul <em>et al.</em> 2007</a>), and was applied to just the 44 genotyped F<sub>2</sub> progenitors of the selected F<sub>2:3</sub> progeny (<em>i.e.</em>, 22 low/22 high protein phenotypes) to obtain a (QTL peak) LOD score significance criterion for a <em>genome-wise</em> Type I error α of 0.05 +/− SE of 0.005. To attain this degree of precision (see p. 106 in <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>), 1900-replicate permutation tests were conducted for each trait in each population. The protein (or oil) additive (<em>a</em>) and dominance (<em>d</em>) effects conditioned by each marker on each chromosome were first examined graphically, but subsequently, these two effects were numerically estimated for just the putative QTL exhibiting the largest peak LOD score on each chromosome. This estimation used the phenotype means for each of the A, H, and B genotypes of the SNP marker, or a pseudo-marker nearest to the putative QTL. The heritability of each presumptive single QTL on a chromosome is the fraction of the phenotypic variance (<em>i.e.</em>, <em>R</em><sup>2</sup>) explained by that QTL, which was estimated with the following equation (p. 77 in <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6/1635/embed/mml-math-7.gif"/></noscript></span></span>where <em>n</em> is the number of phenotyped F<sub>2:3</sub> progenies in each population, and LOD is the log<sub>10</sub> likelihood ratio (LR) attained by that QTL at its peak map position in the Rqtl scanone output.</p><p id="p-29">The QTL detected in this study were declared statistically significant only if the observed peak LOD score exceeded a population-specific, permutation-generated LOD score computed for a genome-wise Type I error of α = 0.05. The chromosomal locations of these QTL were compared with the locations of QTL detected using GWAS in the recent reports, and also the QTL detected in older publications listed in SoyBase. In the latter reports, the authors often used a lower significance threshold for QTL declaration (<em>i.e.</em>, LOD ≥ 3.0), which in most cases was also a comparison-wise threshold that was not adjusted for multiple testing.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-30">Phenotype and genotype data for the 48 F<sub>2</sub> populations and three combined sets of F<sub>2</sub> populations (*.csv files) will be available on SoyBase (<a href="http://www.soybase.com">www.soybase.com</a>), along with the R/qtl command code (*.txt files). Supplemental files include: <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/1">File S1</a> contains additional <em>Materials and Methods</em> details; <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/2">Figure S1</a> illustrates genetic diversity for seed protein/oil in the Soybean Germplasm Collection; <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/3">Figure S2</a> shows the chromosomal map positions of the 1536 SNPs, and the 452 SNPs in the (example) SG mating 1; <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/4">Figure S3</a> depicts the chromosomal map positions of SoyBase-listed QTL reported to date; <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS1.xlsx">Table S1</a> documents the original identification codes for the 1536 SNPs aligned with the shorter five-digit Snnnnn names we used to reduce computer memory usage, and to lessen printed table space in this report; <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a> and <a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/6">Table S3</a> contain population-specific data for the respective phenotypic and genotypic data after R/qtl error-checking; <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS4.xlsx">Table S4</a> contains the parameter data derived from the population-specific QTL analyses, ordered by either mating number or by chromosome number; <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS5.xlsx">Table S5</a>, <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS6.xlsx">Table S6</a>, and <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS7.xlsx">Table S7</a> tabulate the QTL analysis information generated in the combined sets of parental matings of MG 000, 00, and 0 in which the high protein accessions were not uniquely different from each other in terms of SNP genotype.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-31">A total of 48 high seed protein soybean accessions were used as female parents in this research (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Additional high protein accessions have since been added to the germplasm collection, though the 48 used here remain representative of the current group of such accessions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The male parent accessions (<em>i.e.</em>, high yielding cultivars of ordinary seed protein content) have a seed oil content that is characteristically higher than that of most of the female parents. Accessions with a maturity greater than MG IV (except for one very early maturing MG V) were not used in this study because the normal fall frost date in Lincoln, NE precludes completion of their normal seed maturation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Seed protein values plotted against corresponding seed oil values. These are the GRIN values for 10,762 of the 17,711 Glycine max (L.) Merr. accessions in the USDA Soybean Germplasm Collection (as of December 31, 2015) in the seven maturity groups (MGs) of 000 (130), 00 (491), 0 (1179), I (1600), II (1831), III (1731), and IV (3800). Also shown are graph coordinates for 47 of the 48 high protein female parents (+), and seven agronomic male parents (&#xD7;) used in this study (see Table 1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066784401" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Seed protein values plotted against corresponding seed oil values. These are the GRIN values for 10,762 of the 17,711 &lt;em&gt;Glycine max&lt;/em&gt; (L.) Merr. accessions in the USDA Soybean Germplasm Collection (as of December 31, 2015) in the seven maturity groups (MGs) of 000 (130), 00 (491), 0 (1179), I (1600), II (1831), III (1731), and IV (3800). Also shown are graph coordinates for 47 of the 48 high protein female parents (+), and seven agronomic male parents (&#xD7;) used in this study (see Table 1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1635/F1.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/6/1635/F1.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468335" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Seed protein values plotted against corresponding seed oil values. These are the GRIN values for 10,762 of the 17,711 <em>Glycine max</em> (L.) Merr. accessions in the USDA Soybean Germplasm Collection (as of December 31, 2015) in the seven maturity groups (MGs) of 000 (130), 00 (491), 0 (1179), I (1600), II (1831), III (1731), and IV (3800). Also shown are graph coordinates for 47 of the 48 high protein female parents (+), and seven agronomic male parents (×) used in this study (see <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Our initial goal was to generate at least 250 F<sub>2:3</sub> seed progenies in each mating, which was reached in most matings (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>), but not in some later MGs, though sufficient F<sub>2</sub> plant numbers were raised per mating. Progeny numbers averaged 224 over the 48 matings, but ranged from 278 to 115. In later MG matings, many F<sub>2</sub> plants produced too few F<sub>3</sub> seed (due to pod shattering) to meet the minimum seed sample requirement of the NIR instrument.</p><div id="sec-9" class="subsection"><h3>Phenotype data</h3><p id="p-34">The F<sub>2:3</sub> seed protein distributions (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>), only three of the 48 seed protein distributions had <em>P</em>-values for the Shapiro-Wilkes normality test that were less than the prechosen criterion of <em>P</em> = 0.01 (<em>i.e.</em>, mating 31, <em>P</em> = 0.008; 38, <em>P</em> = 0.00005; 47, <em>P</em> = 0.003), primarily because of a rightward skew (perceptibly slight in matings 31 and 47, but notably more so in 38). Seven other distributions had <em>P</em>-values of less than <em>P</em> = 0.05, but these seven were still greater than <em>P</em> = 0.01 (<em>i.e.</em>, matings 3, 6, 9, 10, 12, 21, and 37).</p><p id="p-35">In the one replicate F<sub>2:3</sub> seed protein distributions, the minimum and maximum values among the 48 matings ranged from 371 to 402 g kg<sup>–1</sup>, and from 446 to 497 g kg<sup>–1</sup> (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>). The F<sub>2.3</sub> progeny seed protein means in those 48 matings ranged from 411 to 439 g kg<sup>–1</sup>.</p></div><div id="sec-10" class="subsection"><h3>Heritability</h3><p id="p-36">The seed protein phenotypic variance in the 48 matings ranged from 13 to 53, with a mean of 27 (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>)—typical magnitudes when protein content is NIR-measured using F<sub>3</sub> seed (<em>i.e.</em>, F<sub>3</sub> embryos with F<sub>2</sub> seed coats) produced by F<sub>2</sub> plants derived from matings of high protein parents with ordinary protein parents. The F<sub>2</sub> plant phenotypic variance, when divided by the summed parental plant phenotypic variances, led to moderately sized heritability estimates that averaged 66%, but ranged by mating from 30% to 87%. The seed oil phenotypic variance ranged from 12 to 58, with a mean of 26 (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>), and the heritability estimates (except for zero in mating 48) averaged 68%, and ranged from 18% to 93% in the other 47 matings. These estimates are based on just one (complete) replicate assay, one location, and one year, and thus do not have the accuracy of multi-environment-based heritability estimates (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Visscher <em>et al.</em> 2008</a>).</p></div><div id="sec-11" class="subsection"><h3>Population SNP genotyping numbers</h3><p id="p-37">The SG percentage of the 44 genotyped population individuals was actually a function of the number of phenotyped individuals which, in any given mating, deviated from a 48-mating average of <em>n</em> = 224. The SG two-tail percentage averaged 20.5% (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>), though that percentage by mating varied from 15.8% (<em>i.e.</em>, mating 16) to 38.3% (<em>i.e.</em>, mating 45).</p><p id="p-38">A majority of the SNPs (<em>ca</em>. 60%) in the 1536-SNP chip developed by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hyten <em>et al.</em> (2010)</a> were not bimorphic in each of the 48 matings (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/6">Table S3</a>). The 48-mating average for parental SNP bimorphism was 29.3% of the 1536, but, on an individual mating basis, ranged from 16.9% (mating 31) to 36.5% (mating 40). On a chromosome basis, the range was 24% (chromosomes 6 and 7) to 38% (chromosome 16). In a few matings, some chromosomes had fewer than 10 bimorphic SNPs, primarily because of the removal, during error-checking, of several problematic SNPs that, when paired with other nearby SNPs, generated recombination fraction values far above the expected 0.50 maximum.</p><p id="p-39">Version 4.0 of the soybean genetic map spans 2296.4 cM (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Hyten <em>et al.</em> 2010</a>), but, if restricted to just the 1536 SNPs, the map is shorter (<em>i.e.</em>, 2156.2 cM). A 5-cM SNP spacing is considered to be sufficiently dense for optimizing QTL detection power in populations of size 200 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Strange <em>et al.</em> 2013</a>), implying that 440–460 evenly spaced SNPs would thus be adequate for a 2150–2300 cM map. The mean number of SNP markers segregating per population in this SG study was, in fact, 450 (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/6">Table S3</a>), but ranged from a maximum of 560 (mating 40) to a minimum of 259 (mating 31). The number of genotyped SNPs was low in two other cases (317 in mating 7; 305 in mating 8), but 396 or more SNPs did segregate in 40 of the 48 matings, with 348 SNPs or more segregating in five of the remaining eight matings. The 1536-SNP chip was designed to position SNPs as uniformly possible over the chromosomes (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/3">Figure S2A</a>), but less than <em>ca</em>. one-third of those SNPs segregated in any given mating. An example is mating 1, in which only 452 SNPs were bimorphic (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/3">Figure S2B</a>). Marker monomorphism did result in SNP-coverage gaps of 30 cM or more in some matings (in the mating 1 example, chromosomes 1, 6, 7, 12, and 20), but marker gaps are not <em>a priori</em> predictable when using a SNP chip for genotyping in a multi-mating SG strategy.</p></div><div id="sec-12" class="subsection"><h3>QTL identified for seed protein and oil</h3><p id="p-40">The QTL analysis data obtained for soybean seed protein and seed oil in each of the 48 matings (<a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS4.xlsx">Table S4</a>) were translated into a heat map (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) to display the QTL peak LOD scores observed for seed protein (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) or oil (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) on any given soybean chromosome in each mating. The permutation-derived LOD score significance criterion (<em>i.e.</em>, genome-wise α of 0.05) for evaluating those observed QTL peak scores varied by population from 3.2 to 4.6 for protein, averaging <em>ca</em>. 4.0, and, for oil, varied from 3.2 to 5.6, also averaging <em>ca</em>. 4.0 (<a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS4.xlsx">Table S4</a>). Using the stratified permutation-based significance criterion, a QTL was detected on at least one chromosome for the SG trait of seed protein in 35 (73% of the 48) matings, and detected for the non-SG trait of seed oil in 25 (52% of the 48) matings (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, A and B</a>, red-center bubbles). In two of the 48 matings (<em>i.e.</em>, 22 and 45), LOD score values on all 20 chromosomes were &lt; 3.0, indicating the absence of any protein or oil QTL.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A heat map depicting parameter estimates for the SG-detected QTL for protein (A) and oil (B). The 48 MG-class matings are listed on the left axis, and 20 soybean chromosomes on the horizontal axis. The LOD score peak magnitudes are denoted by bubble size; those exceeding a genome-wise &#x3B1; =&#xA0;0.05 significance threshold derived from trait- and population-specific SG-stratified permutation tests (n&#xA0;=&#xA0;1900) have red dot centers. Additive effect magnitude is denoted by bubble color intensity; green denoting a positive and orange a negative directional effect of the female parent B allele. The magnitude of the R2 values is denoted by square tile color (light blue to deep black). See Table S4 for numerical values of QTL analysis parameters and permutation values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066784401" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A heat map depicting parameter estimates for the SG-detected QTL for protein (A) and oil (B). The 48 MG-class matings are listed on the left axis, and 20 soybean chromosomes on the horizontal axis. The LOD score peak magnitudes are denoted by bubble size; those exceeding a genome-wise &#x3B1; =&#xA0;0.05 significance threshold derived from trait- and population-specific SG-stratified permutation tests (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1900) have red dot centers. Additive effect magnitude is denoted by bubble color intensity; green denoting a positive and orange a negative directional effect of the female parent B allele. The magnitude of the &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; values is denoted by square tile color (light blue to deep black). See Table S4 for numerical values of QTL analysis parameters and permutation values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1635/F2.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/6/1635/F2.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468408" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">A heat map depicting parameter estimates for the SG-detected QTL for protein (A) and oil (B). The 48 MG-class matings are listed on the left axis, and 20 soybean chromosomes on the horizontal axis. The LOD score peak magnitudes are denoted by bubble size; those exceeding a genome-wise α = 0.05 significance threshold derived from trait- and population-specific SG-stratified permutation tests (<em>n</em> = 1900) have red dot centers. Additive effect magnitude is denoted by bubble color intensity; green denoting a positive and orange a negative directional effect of the female parent B allele. The magnitude of the <em>R</em><sup>2</sup> values is denoted by square tile color (light blue to deep black). See <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS4.xlsx">Table S4</a> for numerical values of QTL analysis parameters and permutation values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The LOD score heat map makes evident the near-ubiquitous segregation of the well-known chromosome 20 QTL for protein and/or oil in many SG matings. The protein QTL was significant in 27 (77%) of the above-noted 35 matings (<em>i.e.</em>, 56% of all 48) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>), with the oil QTL being significant in 20 (80%) of the above-noted 25 matings (<em>i.e.</em>, 42% of all 48) (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>). Significant protein QTL were also detected on chromosome 10 (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>) in five matings (<em>i.e.</em>, 30 of MG II; 35, 37, 38, and 39 of MG III), but only in one mating (30), was a significant colocalized oil QTL detected (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>). The QTL region on chromosome 20 is known to be highly homologous with the long arm of chromosome 10 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schmutz <em>et al.</em> 2010</a>). <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Diers <em>et al.</em> (1992)</a> reported that protein-oil QTL existed on chromosome 20 and 15. A protein and oil QTL was SG-detected on 15 in two MG 00 matings (13 and 17), but only for protein in MG IV mating 44 (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>). Other less common SG-detected QTL were on chromosome 6 for protein (matings 18, 30, 33, and 38) and oil (30, 38, and 46), on 7 for both protein and oil (matings 2 and 34) and 18 (33), but just oil on 14 (27 and 46), and 18 (12, 33, and 42). Significant QTL were detected on chromosomes 2, 4, 12, 16, and 18 for protein, and on chromosomes 2, 8, 9 and 13 for oil, but only in single (separate) matings.</p><p id="p-43">With respect to the significant seed protein QTL on chromosomes 20, 10, and 15, plus the protein QTL on chromosome 7 (matings 2 and 34), the QTL allele contributed by the high protein parent <em>enhanced</em> protein content, but coordinately <em>decreased</em> oil (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2, A and B</a>; +/− additive effects are denoted by a green/orange bubble color). Conversely, for the protein QTL repeatedly detected on chromosome 6 (matings 18, 30, 33, and 38), plus the protein QTL on chromosomes 2 and 18 (matings 31 and 18), the high protein parent allele <em>decreased</em> protein but <em>enhanced</em> oil.</p><p id="p-44">For those significant protein and oil QTL that had coincident map positions, the protein and oil additive effects were directionally inversed (cf. <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2, A and B</a>, and <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS4.xlsx">Table S4</a>). Fewer oil QTL than protein QTL were detected, but this was expected, due to a SG focus only on protein, and a protein-oil correlation in the SG matings that, while strong, was clearly not unity, ranging from –0.66 to –0.88, averaging –0.78 (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5">Table S2</a>). <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chung <em>et al.</em> (2003)</a> noted the chromosome 20 segment had opposite effects on protein and oil contents, perhaps due to pleiotropy. A single-QTL pleiotropy hypothesis is easily falsifiable upon detection of a recombinant with a coupling- (instead of a repulsion-) phased phenotype, but no recombinant individuals with a <em>high protein</em>–<em>high oil</em> seed content were detected in this study.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-45">Selective genotyping was a term first defined and used by <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lander and Botstein (1989)</a>, though the method had been essentially described earlier by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lebowitz <em>et al.</em> (1987)</a> as a “trait-based QTL analysis”, in which genotyping resources could be more efficiently allocated, with minimal loss of information, to just a fraction of progeny in a given mating. Indeed, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Navabi <em>et al.</em> (2009)</a> used simulation to document that, if 30 to 50 of 200 phenotyped progeny of a mating were genotyped in a bidirectional SG, Type I error would not exceed 0.02. With 20% genotyping, QTL detection power was still nearly 0.8 (<em>i.e.</em>, a Type II error of 0.20), though detection of QTL of moderate to large effect size would require a marker spacing of at least 5 cM. These results led <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Navabi <em>et al.</em> (2009)</a> to conclude that SG would be a very effective tool for screening large numbers of potential donors for large-effect QTL alleles governing a particular trait of interest.</p><p id="p-46">That strategy was evaluated here by genotyping <em>ca</em>. 20% of <em>ca</em>. 224 phenotypes in each of the 48 F<sub>2</sub> populations created by using 48 high protein donor parents. We calculated, using the R program qtlDesign (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sen <em>et al.</em> 2007</a>), that with a 5-cM SNP spacing, a Type I error (α) set to 0.05, and a Type II error (β) set to 0.2 to achieve a power (1 – β) of 0.8, QTL with an additive effect size of 5 g kg<sup>–1</sup> (accounting for <em>ca</em>. 15% of the phenotypic variance) could be detected in such populations. In our 48-mating SG study, for which the significance threshold (genome-wise Type I error of 0.05) in each population was obtained by permutation (<em>n</em> – 1900), significant QTL with additive effects of ≥ 4 g kg<sup>–1</sup>, and <em>R</em><sup>2</sup> values of 0.07 or more, were detected for protein on 10 chromosomes (<em>i.e.</em>, 1, 4, 6, 7, 10, 12, 15, 16, 18, and 20; <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2A</a>), and for oil on 11 chromosomes (<em>i.e.</em>, 2, 6, 7, 8, 9, 10, 13, 14, 15, 18, and 20; <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2B</a>, and see <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS4.xlsx">Table S4</a> for QTL summary data), confirming that multiple donor parents can be successfully surveyed for QTL presence using a SG strategy.</p><p id="p-47">Seed protein and oil QTL detected in biparental matings in older publications are summarized in SoyBase (<a href="http://www.soybase.com">www.soybase.com</a>). The QTL ANOVA F-statistics in old reports are not convertible into LOD scores, but the LOD scores in more recent reports are convertible into an F-statistic (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>), so we graphed the ANOVA F-statistic <em>P</em>-value (<em>y</em>-axis) and map position (<em>x</em>-axis) of each SoyBase-reported QTL (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/4">Figure S3</a>). The evidence for a SoyBase-reported QTL using these comparison-wise <em>P</em>-values ranged from “merely suggestive” (<em>i.e.</em>, <em>P</em> &lt; 0.01 = 10<sup>−2</sup>)—a significance criterion leading to a naïve supposition that a SoyBase-listed QTL exists on every soybean chromosome (except 16 for protein), to “highly likely” (<em>i.e.</em>, <em>P</em> &lt; 0.0001 = 10<sup>−4</sup>)—a stringent significance criterion that offsets an intrinsic multiple marker comparison-wise test problem in the older reports. Using the latter criterion, we filtered the SoyBase-reported QTL to just the “most likely” protein QTL on the eight chromosomes of <span class="underline">1</span>, <span class="underline">4</span>, <span class="underline">6</span>, <span class="underline">7</span>, 11, 13, <span class="underline">15</span>, and <span class="underline">20</span>, and the oil QTL on the 10 chromosomes of <span class="underline">2</span>, 5, <span class="underline">6</span>, <span class="underline">8</span>, 11, <span class="underline">14</span>, <span class="underline">15</span>, 16, <span class="underline">19</span>, and <span class="underline">20</span> (<a href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/4">Figure S3</a>), wherein the underscores denote chromosomes in common with those having SG-significant protein or oil QTL (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a>). Comparatively, the SG study did reidentify some prior reported QTL; however, none of the 48 high protein SG donor parents were used in any of the 35 to 38 matings listed in SoyBase QTL reports, so this multi-mating SG strategy effectively doubled the number of biparental mapping populations used to date for detection of protein and oil QTL.</p><p id="p-48"><a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Korte and Farlow (2013)</a> noted that GWAS surmounts two key limitations of biparental mapping: a QTL allele in a large-accession GWAS is not restricted to a 0.5 or zero frequency, as might be the case in any given biparental mating, and the QTL mapping resolution is greatly limited by the low number of potential recombination events in a F<sub>2</sub> or RIL population, even if the latter were to be increased to <em>n</em> &gt; 1000 individuals to boost the number of recombinant events. Though GWAS does require marker-dense genotyping (<em>i.e.</em>, thousands of SNPs) to achieve its signal resolution potential, those SNP numbers are nowadays more easily obtainable in soybean, given the availability of a 50K SNP chip (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Song <em>et al.</em> 2013</a>), or using genotyping-by-sequencing to generate, <em>de novo</em>, several thousands of SNPs (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sonah <em>et al.</em> 2014</a>).</p><p id="p-49">The question then is whether a SG strategy is a worthy alternative to just using GWAS. Notably, biparental mapping and GWAS are still considered complementary approaches (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Myles <em>et al.</em> 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Würschum 2012</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sonah <em>et al.</em> 2014</a>). In fact, we considered our multi-mating SG strategy, wherein <em>ca</em>. 450 SNPs were used to genotype just the highest 22 (10%) and lowest 22 (10%) protein phenotypes in <em>ca</em>. 224 progeny derived from 48 high protein × low protein parental matings of MG 000 to IV to be contextually analogous to a phenotypic contrast type of GWAS [like the one recently conducted by <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Song <em>et al.</em> (2015)</a> on soybean 100-seed weight]. The GWAS of <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hwang <em>et al.</em> (2014)</a> involved 31,954 SNP genotypes of 298 accessions of MG II, III, and IV, of which 151 had a high GRIN-based protein values, and 147 had a GRIN-based low protein values [though the contrasting GRIN values were only modestly (<em>r</em> = 0.6) correlated their field-based trial estimated values]. They detected significant QTL (using –logP ≥ 3) on <em>ca</em>. half of the chromosomes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>), and some of those QTL had chromosomal map positions coincident with some of our SG-detected significant QTL. Their two groups did include four SG high-protein female parents (32, 34, and 39 of MG III, plus 40 of MG IV), and two SG low-protein male parents (MGII Dwight and MG III Pana) (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Still, the comparative QTL results demonstrated that a SG survey strategy with <em>ca</em>. 48 accessions identified significant QTL with about the same degree of success achievable in a GWAS with <em>ca</em>. 300 accessions.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A graph of LOD score magnitudes of SG-detected QTL in 48 F2 populations for seed protein (A) and oil (B). The bottom axis is scaled in terms of the Version 4.0 cumulative genetic map positions in the 20-chromosome soybean genome. The blue-box symbols with centered blue dots denote SG QTL exceeding a genome-wise &#x3B1; =&#xA0;0.05 significance threshold derived from trait- and population-specific SG-stratified permutation tests (n&#xA0;=&#xA0;1900). Those thresholds varied from 3.6 to 4.6 for protein, and from 3.2 to 4.8 for oil, but averaged ca. 4.0 (horizontal black line). For comparative purposes, QTL detected in five recent GWAS publications are depicted relative to a &#x2013;logP scaled right axis, though some Bandillo et al. (2015) values (box-enclosed at graph top) exceeded the scale limit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066784401" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A graph of LOD score magnitudes of SG-detected QTL in 48 F&lt;sub&gt;2&lt;/sub&gt; populations for seed protein (A) and oil (B). The bottom axis is scaled in terms of the Version 4.0 cumulative genetic map positions in the 20-chromosome soybean genome. The blue-box symbols with centered blue dots denote SG QTL exceeding a genome-wise &#x3B1; =&#xA0;0.05 significance threshold derived from trait- and population-specific SG-stratified permutation tests (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1900). Those thresholds varied from 3.6 to 4.6 for protein, and from 3.2 to 4.8 for oil, but averaged &lt;em&gt;ca&lt;/em&gt;. 4.0 (horizontal black line). For comparative purposes, QTL detected in five recent GWAS publications are depicted relative to a &#x2013;logP scaled right axis, though some Bandillo &lt;em&gt;et al.&lt;/em&gt; (2015) values (box-enclosed at graph top) exceeded the scale limit.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1635/F3.medium.gif" width="293" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/6/1635/F3.medium.gif" width="293" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468406" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-50" class="first-child">A graph of LOD score magnitudes of SG-detected QTL in 48 F<sub>2</sub> populations for seed protein (A) and oil (B). The bottom axis is scaled in terms of the Version 4.0 cumulative genetic map positions in the 20-chromosome soybean genome. The blue-box symbols with centered blue dots denote SG QTL exceeding a genome-wise α = 0.05 significance threshold derived from trait- and population-specific SG-stratified permutation tests (<em>n</em> = 1900). Those thresholds varied from 3.6 to 4.6 for protein, and from 3.2 to 4.8 for oil, but averaged <em>ca</em>. 4.0 (horizontal black line). For comparative purposes, QTL detected in five recent GWAS publications are depicted relative to a –logP scaled right axis, though some <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> (2015)</a> values (box-enclosed at graph top) exceeded the scale limit.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Recently, GWAS was used to detect seed protein and oil QTL (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Sonah <em>et al.</em> 2014</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vaughn <em>et al.</em> 2014</a>; <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wen <em>et al.</em> 2015</a>). The signal strength and map position of the significant QTL detected in our 48-mating SG can be compared to these GWAS-detected QTL (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The –logP significance criterion / MG accession numbers / SNP numbers varied (<em>i.e.</em>, Sonah: 4/139 MG 0/17.2K SNPs; Vaughn: 4/619 MG I-II and 977 MG III–IV/∼32K SNPs; Bandillo: 5.7/12K MG 000 – X/36.5K SNPs; Wen: 5/1.4K MG I–III/3.75K SNPs), but in all cases, a minor allele frequency (MAF) cut-off of 0.05 was used. Accession number maximization is often sought in GWAS, because doing so increases historical recombinant event numbers, thus enhancing statistical power, and QTL signal resolution. However, if only a few accessions possess an allele of notable effect at a given QTL, nondetection of that QTL will occur in GWAS if those few accessions comprise less than a 0.05 fraction of all of the evaluated accessions. In fact, the routine use of MAF ≥ 0.05 in GWAS will, <em>a priori</em>, remove SNP locus alleles that are in complete linkage disequilibrium with a rare QTL allele that has an <em>in situ</em> frequency of &lt; 0.05. <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> (2015)</a> documented this by showing that the high protein–low oil allele (of large effect) at the well-known chromosome 20 QTL was present in just over 1% of the 12K accessions they examined. But, when they parsed the 12K accessions into smaller groups, based on seven countries of origin, or on eight MG classes, the high protein allele on chromosome 20 had an MAF &lt; 0.05 in all but the Korean accession subset (and in all but the MG V to X subsets). <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Sonah <em>et al.</em> (2014)</a> and <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Wen <em>et al.</em> (2015)</a> did detect the chromosome 20 QTL allele in their respective sets of MG 0 and MG I–III accessions, but <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Vaughn <em>et al.</em> (2014)</a> did not in two large sets of MG I–II or MG III–IV accessions (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Myles <em>et al.</em> (2009)</a> commented on the ineffectiveness of GWAS relative to the detection of rare alleles, and noted that controlled crosses and family-based mapping would be needed to artificially inflate the infrequency of rare functional alleles to improve the power needed for their detection, and to thus better understand the role that rare alleles play with regard to heritability of a given trait of interest.</p><p id="p-52">Despite its rarity, the chromosome 20 QTL was obviously detected in many of the 48 MG 000 to IV donor parent accessions surveyed in this SG study (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2</a>). Accessions chosen for a SG-based QTL survey are actually quite likely to possess rare QTL alleles of a large-to-moderate effect in heritable traits, given the use of an “extreme” phenotype criterion to select SG donor accessions. If at least one chosen donor parent accession possesses a rare allele, its frequency will obviously be 0.5 in the progeny of the corresponding SG biparental high × low mating, thus empowering its detection as noted by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Myles <em>et al.</em> (2009)</a>.</p><p id="p-53">A multi-mating F<sub>2</sub> population SG strategy can provide multiple estimates of the additive (and dominance) effects for the significant QTL detected in more than one mating. However, our population sizes were <em>ca</em>. 224 in size, and thus the effect estimates are likely overly optimistic, and selectively biased (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beavis 1998</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Xu 2003</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>). For greater precision and accuracy in effect estimation, a fivefold (or greater) population size is needed, which, along with a more marker-dense SNP chip for genotyping (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Strange <em>et al.</em> 2013</a>), potentially mitigates SNP-to-SNP marker linkage map gaps. Despite that problem, our foremost objective in this SG study was evaluating an economical means for <em>per se</em> detection of significant protein and oil QTL in a large potential donor accession set. Using GWAS instead of SG offers no panacea for better estimation, given that effect estimates are always specific for the reference population used in either approach, as noted by <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Würschum (2012)</a>. Breeders must obviously conduct follow-up research to precisely estimate the QTL allele effect size in the genetic backgrounds of their particular high-yielding cultivar sets, and to determine the worthiness of launching any marker-assisted high protein allele introgression program.</p><p id="p-54"><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Song <em>et al.</em> (2015)</a> found, after conducting a pairwise genetic similarity analysis using the 50K SNP chip, that 9% of the 18,480 accessions in the soybean germplasm collection had SNP genotypes that were not unique. They also reported that, using a 99.9% similarity criterion, 23% could be considered to be not unique. That discovery prompted us to review the 50K SNP genotypes of our 48 accessions. Unfortunately, eight of our MG 000 high protein accession parents (matings 1 to 8 in <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>), and five of our MG 00 high protein parental accessions (matings 9–12 and 14) were not unique. Four MG 0 parental accessions (matings 18–20 and 22) also were not unique, but these four did differ from the former 13 accessions. Thus, only 33 of our 48 accessions were truly unique. Soybean breeders have used these MG 000, 00, and 0 accessions as a source of high protein alleles (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>), generally presuming that their differing GRIN passport data implied source diversity, but the germplasm SNP genotyping data reveals this presumption was mistaken.</p><p id="p-55">The nonuniqueness of 13 MG 000 and 00 accessions, and the four MG 0 accessions, was a disappointing discovery, but it did offer a serendipitous opportunity to conduct a QTL analysis on three large biparental F<sub>2</sub> populations obtainable by pooling the <em>ca</em>. <em>n</em> = 224 F<sub>2</sub> populations of the three parental mating sets (<em>i.e.</em>, eight in MG 000, five in MG 00, and four in MG 0) based on the mating of those three sets of female parents to differing MG 000, 00, and 0 male parents. The pooled F<sub>2</sub> phenotype numbers were respectively 2052, 1249, and 986. Soybean populations of this size have not, to our knowledge, been reported for biparental QTL mapping studies, and thus could be used to obtain more precise estimates of QTL peak map positions and, because of the large population sizes, the inflationary impact of selection bias on allele effect estimates would be mitigated (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>).</p><p id="p-56">The well-known major QTL located at the proximal end of chromosome 20 was detected in each pooled MG set (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>,), and also in the <em>ca</em>. <em>n</em> = 224 populations, except mating 9 in MG 00 and mating 18 and 22 in MG 0 (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2</a>; for details see <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS5.xlsx">Table S5</a>, <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS6.xlsx">Table S6</a>, and <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS7.xlsx">Table S7</a>). In contrast, QTL were detected on chromosome 19 in the MG 000 and 00 sets (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), but were not detected in any small population comprising those two sets (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2</a>). Similarly, the chromosome 4, 6, 7, and 15 QTL detected in MG 000 were not detected in small populations (except on chromosome 7 in mating 2). Finally, the QTL on chromosomes 2, 16, and 18 that were detected in MG 00 were not detected in small populations (except for mating 12 on 16 and 18 – oil only). These QTL likely had modest allelic effects that did not exceed the QTL detection limit in the small populations (<em>i.e.</em>, equivalent to false negatives), but did exceed it in the 5- to 10-fold larger populations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal LOD score scans for protein (top panels) and oil (bottom panels). Selectively genotyped F2 populations derived from parental matings in which the high protein accessions were not unique in terms of SNP genotype were pooled into three MG sets of 000 (left panels), 00 (middle panels), and 0 (right panels). The SG percentages were a respective 16.4, 16.8, and 17.3%, relative to the numbers of phenotypes (P), genotypes (G), and bimorphic SNPs shown for each MG set. Genome-wise &#x3B1; = 0.05 significance thresholds, derived from trait- and population-specific SG-stratified permutation tests (n&#xA0;=&#xA0;1900), were nearly-identical (i.e., the 3.80 to 3.88 threshold horizontal lines shown in each panel)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066784401" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosomal LOD score scans for protein (top panels) and oil (bottom panels). Selectively genotyped F&lt;sub&gt;2&lt;/sub&gt; populations derived from parental matings in which the high protein accessions were not unique in terms of SNP genotype were pooled into three MG sets of 000 (left panels), 00 (middle panels), and 0 (right panels). The SG percentages were a respective 16.4, 16.8, and 17.3%, relative to the numbers of phenotypes (P), genotypes (G), and bimorphic SNPs shown for each MG set. Genome-wise &#x3B1; = 0.05 significance thresholds, derived from trait- and population-specific SG-stratified permutation tests (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1900), were nearly-identical (&lt;em&gt;i.e.&lt;/em&gt;, the 3.80 to 3.88 threshold horizontal lines shown in each panel).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1635/F4.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/6/1635/F4.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468416" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-57" class="first-child">Chromosomal LOD score scans for protein (top panels) and oil (bottom panels). Selectively genotyped F<sub>2</sub> populations derived from parental matings in which the high protein accessions were not unique in terms of SNP genotype were pooled into three MG sets of 000 (left panels), 00 (middle panels), and 0 (right panels). The SG percentages were a respective 16.4, 16.8, and 17.3%, relative to the numbers of phenotypes (P), genotypes (G), and bimorphic SNPs shown for each MG set. Genome-wise α = 0.05 significance thresholds, derived from trait- and population-specific SG-stratified permutation tests (<em>n</em> = 1900), were nearly-identical (<em>i.e.</em>, the 3.80 to 3.88 threshold horizontal lines shown in each panel).</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">By using chromosome-specific R/qtl additive and dominance effect scans (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), one can graphically view the impact of substituting a female parent B allele for the male parent A allele at each successive SNP on a chromosome. Coincident map positions were evident for most of the same-chromosome protein and oil QTL peaks, with such numbers being more concordant with a 1-locus pleiotropy than a 2-locus linkage model (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chung <em>et al.</em> 2003</a>). The allele contributed by the high protein parents for the chromosomes 2, 4, 7, 16, and 20 QTL enhanced protein but decreased oil, whereas the allele contributed by the same parents for the chromosome 6 and 19 QTL decreased protein but enhanced seed oil. One peculiarity in these scans was the differential additive effect scan patterns for the chromosome 15 QTL detected in MG 000 <em>vs.</em> MG 0 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The eight high protein female parents in the MG 000 set contributed an allele that decreased seed protein, whereas the four MG 0 high protein female parents contributed an allele that enhanced seed protein. The <em>trans</em>-phased phenotypic effect at these two linked QTL (<em>i.e.</em>, located at 20 cM in MG 000, but at 17 cM in MG 0) is notable, even if different male parents were used in these MG sets. We are not aware of any soybean linkage mapping study or GWAS documenting a similar <em>trans</em>-phased QTL pair.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal scans of the estimated additive and dominance effects for protein (top panels) and oil (bottom panels). The scans were limited to just the chromosomes exhibiting LOD score peaks shown in Figure 4. Effect magnitude and direction (+/&#x2212;) reflect the substitution of a female parent B allele for a male parent A allele at any given bimorphic SNP position, with shading denoting the SE of the effect mean at each SNP. The chromosomal cM positions of the positive and negative additive effect maxima are italicized." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066784401" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosomal scans of the estimated additive and dominance effects for protein (top panels) and oil (bottom panels). The scans were limited to just the chromosomes exhibiting LOD score peaks shown in Figure 4. Effect magnitude and direction (+/&#x2212;) reflect the substitution of a female parent B allele for a male parent A allele at any given bimorphic SNP position, with shading denoting the SE of the effect mean at each SNP. The chromosomal cM positions of the positive and negative additive effect maxima are italicized.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1635/F5.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/6/1635/F5.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1635/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-59" class="first-child">Chromosomal scans of the estimated additive and dominance effects for protein (top panels) and oil (bottom panels). The scans were limited to just the chromosomes exhibiting LOD score peaks shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. Effect magnitude and direction (+/−) reflect the substitution of a female parent B allele for a male parent A allele at any given bimorphic SNP position, with shading denoting the SE of the effect mean at each SNP. The chromosomal cM positions of the positive and negative additive effect maxima are italicized.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">The tracking of QTL dominance and additive affects in the three MG sets revealed that, at each chromosomal QTL (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS5.xlsx">Table S5</a>, <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS6.xlsx">Table S6</a>, and <a href="http://www.g3journal.org/content/vol0/issue2016/images/data/g3.116.027656/DC1/TableS7.xlsx">Table S7</a>), when the additive effect was positive, the dominance effect was typically (though not always) negative, and <em>vice versa</em>. However the SE boundary for the additive effect was narrower than that bounding the dominance effect—an indication that the latter was less precisely estimated, likely due to heterozygote infrequency in SG phenotypic extremes. Ordinarily, additive, plus additive × additive epistasis, accounts for most of the total trait genetic variance in soybean (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Burton 1987</a>). Only inbred cultivars are used in commercial production, and the creation of F<sub>1</sub> hybrids is not likely anytime soon. Dominance effects would more likely be of breeder interest if made available for yield rather than seed protein and oil.</p><p id="p-61">The discovery, or confirmatory rediscovery, of protein and oil QTL and map positions in this SG-based survey of high protein donor accessions will likely to be of relevance to soybean breeders. The SG survey strategy did identify “major” protein and oil QTL in the 48 donor accessions examined here, suggesting that it could be used detect to major QTL alleles (and potentially rare ones) in traits other than seed composition, assuming that such traits can be reliably quantified using individual F<sub>2</sub> plants. A major drawback to the SG strategy is the need to apply it to phenotyping populations much larger than the <em>n</em> = 224 size used in this study, if the goal is to detect QTL of more modest additive effect.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">The authors thank Dr. Karl W. Broman (Department of Biostatistics and Medical Informatics, School of Medicine and Public Health, University of Wisconsin, Madison, WI) for providing much assistance and advice in the use of Rqtl for the QTL linkage mapping in this multipopulation SG study. Piyaporn Phansak’s Ph.D. program was funded by a scholarship from the Committee Staff Development Project on Higher Education, Royal Thai Government. Watcharin Soonsuwon’s Ph.D. program was funded by a scholarship from the Faculty of Natural Resources, Prince of Songkla University, Thailand. Initial funding for the phenotyping phase of this research project was provided by the Nebraska Soybean Board. Subsequent funding for the genotyping phase was provided by the United Soybean Board.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-10"><p id="p-63">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027656/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027656/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-64"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-28"><span class="received-label">Received </span>January 28, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-25"><span class="accepted-label">Accepted </span>March 25, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Phansak <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayoub</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Effectiveness of selective genotyping for detection of quantitative trait loci: an analysis of grain and malt quality traits in three barley populations.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1116</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DAyoub%26rft.auinit1%253DM.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D1116%26rft.epage%253D1124%26rft.atitle%253DEffectiveness%2Bof%2Bselective%2Bgenotyping%2Bfor%2Bdetection%2Bof%2Bquantitative%2Btrait%2Bloci%253A%2Ban%2Banalysis%2Bof%2Bgrain%2Band%2Bmalt%2Bquality%2Btraits%2Bin%2Bthree%2Bbarley%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F12502257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12502257&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.2" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A population structure and genome-wide association analysis on the USDA soybean germplasm collection.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1635.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">QTL analyses: power, precision, and accuracy</span>, pp. <span class="cit-fpage">145</span>–<span class="cit-lpage">162</span> in <span class="cit-source"><em>Molecular Dissection of Complex Traits</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, <span class="cit-publ-name">CRC Press LLC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1635.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Breeding for Quantitative Traits in Plants</em></span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1635.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>A Guide to QTL Mapping with R/qtl</em></span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.6.1635.6" data-doi="10.2135/cropsci1997.0011183X003700020011x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graef</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mapping QTL for seed protein and oil content in eight soybean populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">370</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrummer%26rft.auinit1%253DE.%2BC.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D370%26rft.epage%253D378%26rft.atitle%253DMapping%2BQTL%2Bfor%2BSeed%2BProtein%2Band%2BOil%2BContent%2Bin%2BEight%2BSoybean%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1997.0011183X003700020011x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1997.0011183X003700020011x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WU90500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-6.6.1635.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Quantitative genetics: results relevant to soybean breeding</span>, pp. <span class="cit-fpage">211</span>–<span class="cit-lpage">242</span> in <span class="cit-source"><em>Soybeans: Improvement</em>, <em>Production</em>, <em>and Uses</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-publ-name">ASA, CSSA, and SSSA</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.8" data-doi="10.2135/cropsci2003.1053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babka</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graef</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staswick</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The seed protein, oil, and yield QTL on soybean linkage group I.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DChung%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D1053%26rft.epage%253D1067%26rft.atitle%253DThe%2BSeed%2BProtein%252C%2BOil%252C%2Band%2BYield%2BQTL%2Bon%2BSoybean%2BLinkage%2BGroup%2BI%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000182540300040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.9" data-doi="10.2135/cropsci2000.40139x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cober</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voldeng</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Developing high-protein, high-yield soybean populations and lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCober%26rft.auinit1%253DE.R.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D42%26rft.atitle%253DDeveloping%2BHigh-Protein%252C%2BHigh-Yield%2BSoybean%2BPopulations%2Band%2BLines%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.40139x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.40139x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000085505300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.10" data-doi="10.1007/s003359900353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The effect of selective genotyping on QTL mapping accuracy.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D68%26rft.atitle%253DThe%2Beffect%2Bof%2Bselective%2Bgenotyping%2Bon%2BQTL%2Bmapping%2Baccuracy.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003359900353%26rft_id%253Dinfo%253Apmid%252F9021156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003359900353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9021156&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WA74600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Selective genotyping for determination of linkage between a marker locus and a quantitative trait locus.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D353%26rft_id%253Dinfo%253Apmid%252F24197326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24197326&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JY22200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diers</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keim</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehr</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">RFLP analysis of soybean seed protein and oil content.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D608%26rft_id%253Dinfo%253Apmid%252F24202678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24202678&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HK21600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.13" data-doi="10.1093/nar/gkp798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Soybean, the USDA-ARS soybean genetics and genomics database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D843</span>–<span class="cit-lpage">D846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp798%26rft_id%253Dinfo%253Apmid%252F20008513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q4NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-6.6.1635.14"><div class="cit-metadata unstructured">Hill, J. L., E. K. Peregrine, G. L. Sprau, C. R. Cemeens, R. L. Nelson <em>et al.</em>, 2008 Evaluation of the USDA soybean germplasm collection: Maturity groups 000-IV (PI 578371-PI 612761). USDA ARS Technical Bulletin 1919.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.15" data-doi="10.2135/cropsci1974.0011183X001400050031x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hymowitz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Estimations of protein and oil concentration in corn, soybean, and oat seed by near-infrared light reflectance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHymowitz%26rft.auinit1%253DT.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D713%26rft.epage%253D715%26rft.atitle%253DEstimations%2Bof%2BProtein%2Band%2BOil%2BConcentration%2Bin%2BCorn%252C%2BSoybean%252C%2Band%2BOat%2BSeed%2Bby%2BNear-Infrared%2BLight%2BReflectance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1974.0011183X001400050031x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1974.0011183X001400050031x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974U548700031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.16" data-doi="10.2135/cropsci2009.06.0360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A high density integrated genetic linkage map of soybean and the development of a 1536 Universal Soy Linkage Panel for quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHyten%26rft.auinit1%253DD.%2BL.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D960%26rft.epage%253D968%26rft.atitle%253DA%2BHigh%2BDensity%2BIntegrated%2BGenetic%2BLinkage%2BMap%2Bof%2BSoybean%2Band%2Bthe%2BDevelopment%2Bof%2Ba%2B1536%2BUniversal%2BSoy%2BLinkage%2BPanel%2Bfor%2BQuantitative%2BTrait%2BLocus%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.06.0360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.06.0360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000276961900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">E.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide association study of seed protein and oil content in soybean.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D15%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keim</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diers</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">RFLP mapping in soybean: association between marker loci and variation in quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKeim%26rft.auinit1%253DP.%26rft.volume%253D126%26rft.issue%253D3%26rft.spage%253D735%26rft.epage%253D742%26rft.atitle%253DRFLP%2Bmapping%2Bin%2Bsoybean%253A%2Bassociation%2Bbetween%2Bmarker%2Bloci%2Band%2Bvariation%2Bin%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Apmid%252F1979039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI2LzMvNzM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.19" data-doi="10.1186/1746-4811-9-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The advantages and limitations of trait analysis with GWAS: a review.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D9%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-9-29%26rft_id%253Dinfo%253Apmid%252F23876160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-9-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23876160&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.20" data-doi="10.1371/journal.pgen.1002496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ladouceur</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dastani</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenwood</span>  <span class="cit-name-given-names">C. M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The empirical power of rare variant association methods: results from sanger sequencing in 1998 individuals.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1002496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253D1002496%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002496&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.22" data-doi="10.1007/BF00289194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lebowitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckmann</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Trait-based analyses for the detection of linkage between marker loci and quantitative trait loci in crosses between inbred lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D73%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00289194%26rft_id%253Dinfo%253Apmid%252F24241113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00289194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24241113&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G129000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.23" data-doi="10.1534/genetics.107.080093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Significance thresholds for quantitative trait mapping under selective genotyping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080093%26rft_id%253Dinfo%253Apmid%252F17720902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE5NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.24" data-doi="10.2307/2533963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muranty</span>  <span class="cit-name-given-names">H. L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Selective genotyping for location and estimation of the effect of a quantitative trait locus.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D53%26rft.spage%253D629%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.25" data-doi="10.1105/tpc.109.068437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association mapping: critical considerations shift from genotyping to experimental design.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2194</span>–<span class="cit-lpage">2202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.068437%26rft_id%253Dinfo%253Apmid%252F19654263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMjE5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.26" data-doi="10.1007/s00122-008-0904-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navabi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spanner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">QTL detection with bidirectional and unidirectional selective genotyping: marker-based and trait-based analyses.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DNavabi%26rft.auinit1%253DA.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D347%26rft.epage%253D358%26rft.atitle%253DQTL%2Bdetection%2Bwith%2Bbidirectional%2Band%2Bunidirectional%2Bselective%2Bgenotyping%253A%2Bmarker-based%2Band%2Btrait-based%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0904-2%26rft_id%253Dinfo%253Apmid%252F18854970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0904-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854970&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.27" data-doi="10.1016/j.cell.2011.09.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raychaudhuri</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping rare and common causal alleles for complex human diseases.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRaychaudhuri%26rft.auinit1%253DS.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D69%26rft.atitle%253DMapping%2Brare%2Band%2Bcommon%2Bcausal%2Balleles%2Bfor%2Bcomplex%2Bhuman%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.09.011%26rft_id%253Dinfo%253Apmid%252F21962507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.09.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21962507&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295396700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.28" data-doi="10.2135/cropsci2012.12.0710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincker</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sleper</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cary</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic improvement of U.S. soybean in maturity groups II, III, and IV.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.12.0710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.12.0710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000336745400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.29" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjørnstad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic dissection of quantitative traits in crops.</span> <abbr class="cit-jnl-abbrev">Electron. J. Biotechnol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BBiotechnol.%26rft.volume%253D13%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.31" data-doi="10.1534/genetics.104.038612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satagopan</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait locus study design from an information perspective.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038612%26rft_id%253Dinfo%253Apmid%252F15781700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzEvNDQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.32" data-doi="10.1007/s00335-006-0090-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satagopan</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">R/qtlDesign: inbred line cross experimental design.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D87%26rft.epage%253D93%26rft.atitle%253DR%252FqtlDesign%253A%2Binbred%2Bline%2Bcross%2Bexperimental%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-006-0090-y%26rft_id%253Dinfo%253Apmid%252F17347894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-006-0090-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17347894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245627200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.33" data-doi="10.1534/genetics.108.094607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannes</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Selective genotyping and phenotyping strategies in a complex trait context.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1613</span>–<span class="cit-lpage">1626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094607%26rft_id%253Dinfo%253Apmid%252F19153260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS80LzE2MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">S</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donoughue</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cober</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajcan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D211%26rft_id%253Dinfo%253Apmid%252F25213593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25213593&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.35" data-doi="10.1371/journal.pone.0054583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054583%26rft_id%253Dinfo%253Apmid%252F23620825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23620825&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.36" data-doi="10.1534/g3.115.019000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fingerprinting soybean germplasm and its utility in genomic research.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1999</span>–<span class="cit-lpage">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019000%26rft_id%253Dinfo%253Apmid%252F26224783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMTk5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.37" data-doi="10.1007/s00122-013-2155-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-density genotyping: and overkill for QTL mapping? Lessons learned from a case study in maize and simulations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2563</span>–<span class="cit-lpage">2574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2563%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2155-0%26rft_id%253Dinfo%253Apmid%252F23860723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2155-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23860723&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Allozyme frequency changes associated with selection for increased grain yield in maize (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D236%26rft.atitle%253DALLOZYME%2BFREQUENCY%2BCHANGES%2BASSOCIATED%2BWITH%2BSELECTION%2BFOR%2BINCREASED%2BGRAIN%2BYIELD%2BIN%2BMAIZE%2B%2528ZEA%2BMAYS%2BL.%2529%26rft_id%253Dinfo%253Apmid%252F17249034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTUvMS8yMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.39" data-doi="10.2135/cropsci1982.0011183X002200040010x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Improvement of yield and ear number resulting from selection at allozyme loci in a maize population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D22%26rft.issue%253D4%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DImprovement%2Bof%2BYield%2Band%2BEar%2BNumber%2BResulting%2Bfrom%2BSelection%2Bat%2BAllozyme%2BLoci%2Bin%2Ba%2BMaize%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1982.0011183X002200040010x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1982.0011183X002200040010x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982PG45700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.40" data-doi="10.1007/s11032-010-9390-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crouch</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Efficiency of selective genotyping for genetic analysis of complex traits and potential application in crop improvement.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D26%26rft.spage%253D493%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-010-9390-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-010-9390-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000282426600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.41" data-doi="10.1038/90135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Dwarf8 polymorphisms associate with variation in flowering time.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DThornsberry%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D286%26rft.epage%253D289%26rft.atitle%253DDwarf8%2Bpolymorphisms%2Bassociate%2Bwith%2Bvariation%2Bin%2Bflowering%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F90135%26rft_id%253Dinfo%253Apmid%252F11431702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/90135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11431702&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169656400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.42" data-doi="10.1534/g3.114.013433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic architecture of seed composition in soybean is refined by genome-wide association scans across multiple populations.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013433%26rft_id%253Dinfo%253Apmid%252F25246241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTEvMjI4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.43" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability in the genomics era—concepts and mis-conceptions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.6.1635.44" data-doi="10.4141/cjps96-031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voldeng</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillemette</span>  <span class="cit-name-given-names">R. J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cober</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">AC Proteus soybean.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Sci.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BSci.%26rft.volume%253D76%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.4141%252Fcjps96-031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4141/cjps96-031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.45" data-doi="10.2135/cropsci1987.0011183X002700050020x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wehrmann</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehr</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cianzio</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavins</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Transfer of high seed protein to high-yielding soybean cultivars.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWehrmann%26rft.auinit1%253DV.%2BK.%26rft.volume%253D27%26rft.issue%253D5%26rft.spage%253D927%26rft.epage%253D931%26rft.atitle%253DTransfer%2Bof%2BHigh%2BSeed%2BProtein%2Bto%2BHigh-Yielding%2BSoybean%2BCultivars%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1987.0011183X002700050020x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1987.0011183X002700050020x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987J978600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.46" data-doi="10.1186/s12864-015-1872-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyse</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic consequences of selection and genome-wide association mapping in soybean.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D671%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1872-y%26rft_id%253Dinfo%253Apmid%252F26334313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1872-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26334313&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.47" data-doi="10.2135/cropsci1998.0011183X003800060021x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Increasing seed protein in soybean with eight cycles of recurrent selection.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1536</span>–<span class="cit-lpage">1540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWilcox%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D1536%26rft.epage%253D1540%26rft.atitle%253DIncreasing%2BSeed%2BProtein%2Bin%2BSoybean%2Bwith%2BEight%2BCycles%2Bof%2BRecurrent%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1998.0011183X003800060021x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1998.0011183X003800060021x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077500100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.48" data-doi="10.1007/s00122-012-1887-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping QTL for agronomic traits in breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWurschum%26rft.auinit1%253DT.%26rft.volume%253D125%26rft.issue%253D2%26rft.spage%253D201%26rft.epage%253D210%26rft.atitle%253DMapping%2BQTL%2Bfor%2Bagronomic%2Btraits%2Bin%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1887-6%26rft_id%253Dinfo%253Apmid%252F22614179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1887-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22614179&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Theoretical basis of the Beavis effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2259%26rft.epage%253D2268%26rft.atitle%253DTheoretical%2BBasis%2Bof%2Bthe%2BBeavis%2BEffect%26rft_id%253Dinfo%253Apmid%252F14704201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyNTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1635.50" data-doi="10.1046/j.1365-2540.2000.00653.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogl</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Maximum likelihood analysis of quantitative loci under selective genotyping.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D84%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00653.x%26rft_id%253Dinfo%253Apmid%252F10849077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00653.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10849077&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/6/1635.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1627" title="The Immature Fiber Mutant Phenotype of Cotton (Gossypium hirsutum) Is Linked to a 22-bp Frame-Shift Deletion in a Mitochondria Targeted Pentatricopeptide Repeat Gene" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1649" title="Saccharomyces cerevisiae Tti2 Regulates PIKK Proteins and Stress Response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/6"><span>Volume 6 Issue 6, June 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/6.cover-source.jpg?itok=WAtQozJg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1635" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EQgysBjWyF9HU-MVRoS81XjogyFfeL4VBW3-YmAVOTw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/6/1635" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/6/1635" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/6/1635" class="active" data-icon-position="" data-hide-link-title="0">Multi-Population Selective Genotyping to Identify Soybean [Glycine max (L.) Merr.] Seed Protein and Oil QTLs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460366" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7GbfSSdB4DFSCrgUB54Ls6FAn5Y_z9pUu2YDn2YTJR8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1635.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/6/1635" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KRteat-KCbtYTzW5lDhPTdMh2pRVuEsH3lPtMp9lGFc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/6/1635" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1635.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460366" id="node460366--2" data-pisa="ggg;6/6/1635" data-pisa-master="ggg;g3.116.027656" data-apath="/ggg/6/6/1635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-Population Selective Genotyping to Identify Soybean [<em>Glycine max</em> (L.) Merr.] Seed Protein and Oil QTLs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Piyaporn Phansak</span>, <span class="highwire-citation-author" data-delta="1">Watcharin Soonsuwon</span>, <span class="highwire-citation-author" data-delta="2">David L. Hyten</span>, <span class="highwire-citation-author" data-delta="3">Qijian Song</span>, <span class="highwire-citation-author" data-delta="4">Perry B. Cregan</span>, <span class="highwire-citation-author" data-delta="5">George L. Graef</span> and <span class="highwire-citation-author" data-delta="6">James E. Specht</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1635-1648;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027656 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460366--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Piyaporn Phansak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piyaporn%2BPhansak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phansak%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiyaporn%2BPhansak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Watcharin Soonsuwon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Watcharin%2BSoonsuwon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soonsuwon%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWatcharin%2BSoonsuwon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David L. Hyten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BHyten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyten%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BL.%2BHyten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qijian Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qijian%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQijian%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Perry B. Cregan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Perry%2BB.%2BCregan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cregan%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APerry%2BB.%2BCregan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">George L. Graef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BL.%2BGraef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graef%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BL.%2BGraef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James E. Specht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSpecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Specht%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BE.%2BSpecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jspecht1@unl.edu" class="" data-icon-position="" data-hide-link-title="0">jspecht1@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/6/1635" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/6/1635&amp;title=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/6/1635&amp;title=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/6/1635&amp;title=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/6/1635&amp;text=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/6/1635&amp;title=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/6/1635&amp;t=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/6/1635&amp;title=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/6/1635&amp;title=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460366/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460366" id="node460366--3" data-pisa="ggg;6/6/1635" data-pisa-master="ggg;g3.116.027656" data-apath="/ggg/6/6/1635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-Population Selective Genotyping to Identify Soybean [<em>Glycine max</em> (L.) Merr.] Seed Protein and Oil QTLs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Piyaporn Phansak</span>, <span class="highwire-citation-author" data-delta="1">Watcharin Soonsuwon</span>, <span class="highwire-citation-author" data-delta="2">David L. Hyten</span>, <span class="highwire-citation-author" data-delta="3">Qijian Song</span>, <span class="highwire-citation-author" data-delta="4">Perry B. Cregan</span>, <span class="highwire-citation-author" data-delta="5">George L. Graef</span> and <span class="highwire-citation-author" data-delta="6">James E. Specht</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1635-1648;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027656 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460366--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Piyaporn Phansak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piyaporn%2BPhansak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phansak%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiyaporn%2BPhansak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Watcharin Soonsuwon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Watcharin%2BSoonsuwon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soonsuwon%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWatcharin%2BSoonsuwon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David L. Hyten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BHyten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyten%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BL.%2BHyten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qijian Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qijian%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQijian%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Perry B. Cregan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Perry%2BB.%2BCregan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cregan%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APerry%2BB.%2BCregan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">George L. Graef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BL.%2BGraef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graef%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BL.%2BGraef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James E. Specht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSpecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Specht%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BE.%2BSpecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jspecht1@unl.edu" class="" data-icon-position="" data-hide-link-title="0">jspecht1@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1635&amp;rft.epage=1648&amp;rft.atitle=Multi-Population%20Selective%20Genotyping%20to%20Identify%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D%20Seed%20Protein%20and%20Oil%20QTLs&amp;rft.volume=6&amp;rft.issue=6&amp;rft.date=2016-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Phansak%2C+Piyaporn&amp;rft.au=Soonsuwon%2C+Watcharin&amp;rft.au=Hyten%2C+David+L.&amp;rft.au=Song%2C+Qijian&amp;rft.au=Cregan%2C+Perry+B.&amp;rft.au=Graef%2C+George+L.&amp;rft.au=Specht%2C+James+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460366/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460366/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460366/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460366/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460366/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460366/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460366/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460366/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460366/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460366/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460366/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460366/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/6/1635" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/6/1635.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/6/1635.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/6/1635.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/6/1635.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
