<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/6/1777" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/6/1777.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/6/1777.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/6/1777.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/6/1777.atom" />
<meta name="HW.pisa" content="ggg;6/6/1777" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells" />
<meta name="DC.Identifier" content="10.1534/g3.116.028241" />
<meta name="DC.Date" content="2016-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Kunzelmann et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Custom genome editing has become an essential element of molecular biology. In particular, the generation of fusion constructs with epitope tags or fluorescent proteins at the genomic locus facilitates the analysis of protein expression, localization, and interaction partners at physiologic levels. Following up on our initial publication, we now describe a considerably simplified, more efficient, and readily scalable experimental workflow for PCR-based genome editing in cultured Drosophila melanogaster cells. Our analysis at the act5C locus suggests that PCR-based homology arms of 60 bp are sufficient to reach targeting efficiencies of up to 80% after selection; extension to 80 bp (PCR) or 500 bp (targeting vector) did not further improve the yield. We have expanded our targeting system to N-terminal epitope tags; this also allows the generation of cell populations with heterologous expression control of the tagged locus via the copper-inducible mtnDE promoter. We present detailed, quantitative data on editing efficiencies for several genomic loci that may serve as positive controls or benchmarks in other laboratories. While our first PCR-based editing approach offered only blasticidin-resistance for selection, we now introduce puromycin-resistance as a second, independent selection marker; it is thus possible to edit two loci ( e.g. , for coimmunoprecipitation) without marker removal. Finally, we describe a modified FLP recombinase expression plasmid that improves the efficiency of marker cassette FLP-out. In summary, our technique and reagents enable a flexible, robust, and cloning-free genome editing approach that can be parallelized for scale-up." />
<meta name="DC.Contributor" content="Stefan Kunzelmann" />
<meta name="DC.Contributor" content="Romy Böttcher" />
<meta name="DC.Contributor" content="Ines Schmidts" />
<meta name="DC.Contributor" content="Klaus Förstemann" />
<meta name="article:published_time" content="2016-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Custom genome editing has become an essential element of molecular biology. In particular, the generation of fusion constructs with epitope tags or fluorescent proteins at the genomic locus facilitates the analysis of protein expression, localization, and interaction partners at physiologic levels. Following up on our initial publication, we now describe a considerably simplified, more efficient, and readily scalable experimental workflow for PCR-based genome editing in cultured &lt;i&gt;Drosophila melanogaster&lt;/i&gt; cells. Our analysis at the &lt;i&gt;act5C&lt;/i&gt; locus suggests that PCR-based homology arms of 60 bp are sufficient to reach targeting efficiencies of up to 80% after selection; extension to 80 bp (PCR) or 500 bp (targeting vector) did not further improve the yield. We have expanded our targeting system to N-terminal epitope tags; this also allows the generation of cell populations with heterologous expression control of the tagged locus via the copper-inducible &lt;i&gt;mtnDE&lt;/i&gt; promoter. We present detailed, quantitative data on editing efficiencies for several genomic loci that may serve as positive controls or benchmarks in other laboratories. While our first PCR-based editing approach offered only blasticidin-resistance for selection, we now introduce puromycin-resistance as a second, independent selection marker; it is thus possible to edit two loci (&lt;i&gt;e.g.&lt;/i&gt;, for coimmunoprecipitation) without marker removal. Finally, we describe a modified FLP recombinase expression plasmid that improves the efficiency of marker cassette FLP-out. In summary, our technique and reagents enable a flexible, robust, and cloning-free genome editing approach that can be parallelized for scale-up.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/06/01" />
<meta name="citation_mjid" content="ggg;6/6/1777" />
<meta name="citation_id" content="6/6/1777" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/6/1777" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/6/1777.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/6/1777.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/6/1777.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028241" />
<meta name="citation_pmid" content="27172193" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1777" />
<meta name="citation_lastpage" content="1785" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Stefan Kunzelmann" />
<meta name="citation_author_institution" content="Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany" />
<meta name="citation_author" content="Romy Böttcher" />
<meta name="citation_author_institution" content="Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany" />
<meta name="citation_author" content="Ines Schmidts" />
<meta name="citation_author_institution" content="Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany" />
<meta name="citation_author" content="Klaus Förstemann" />
<meta name="citation_author_institution" content="Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany" />
<meta name="citation_author_email" content="foerstemann@lmb.uni-muenchen.de" />
<meta name="citation_reference" content="citation_journal_title=Biol. Open;citation_author=A. R. Bassett;citation_author=C. Tibbit;citation_author=C. P. Ponting;citation_author=J. L. Liu;citation_title=Mutagenesis and homologous recombination in Drosophila cell lines using CRISPR/Cas9.;citation_volume=3;citation_year=2014;citation_pmid=24326186;citation_doi=10.1242/bio.20137120" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Bottcher;citation_author=M. Hollmann;citation_author=K. Merk;citation_author=V. Nitschko;citation_author=C. Obermaier;citation_title=Efficient chromosomal gene modification with CRISPR/cas9 and PCR-based homologous recombination donors in cultured Drosophila cells.;citation_volume=42;citation_year=2014;citation_pmid=24748663;citation_doi=10.1093/nar/gku289" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=F. Buchholz;citation_author=P. O. Angrand;citation_author=A. F. Stewart;citation_title=Improved properties of FLP recombinase evolved by cycling mutagenesis.;citation_pages=657-662;citation_volume=16;citation_year=1998;citation_issue=7;citation_pmid=9661200;citation_doi=10.1038/nbt0798-657" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=S. M. Byrne;citation_author=P. Mali;citation_author=G. M. Church;citation_title=Genome editing in human stem cells.;citation_pages=119-138;citation_volume=546;citation_year=2014;citation_pmid=25398338;citation_doi=10.1016/B978-0-12-801185-0.00006-4" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=D. Carroll;citation_author=K. J. Beumer;citation_author=J. J. Morton;citation_author=A. Bozas;citation_author=J. K. Trautman;citation_title=Gene targeting in Drosophila and Caenorhabditis elegans with zinc-finger nucleases.;citation_pages=63-77;citation_volume=435;citation_year=2008;citation_pmid=18370068;citation_doi=10.1007/978-1-59745-232-8_5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. Chen;citation_author=L. A. Gilbert;citation_author=B. A. Cimini;citation_author=J. Schnitzbauer;citation_author=W. Zhang;citation_title=Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system.;citation_pages=1479-1491;citation_volume=155;citation_year=2013;citation_pmid=24360272;citation_doi=10.1016/j.cell.2013.12.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. Doudna;citation_author=E. Charpentier;citation_title=The new frontier of genome engineering with CRISPR-Cas9;citation_pages=1258096-1258096;citation_volume=346;citation_year=2014;citation_issue=6213;citation_pmid=25430774;citation_doi=10.1126/science.1258096" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=J. Fetter;citation_author=A. Samsonov;citation_author=N. Zenser;citation_author=F. Zhang;citation_author=H. Zhang;citation_title=Endogenous gene tagging with fluorescent proteins.;citation_pages=231-240;citation_volume=1239;citation_year=2015;citation_pmid=25408409;citation_doi=10.1007/978-1-4939-1862-1_12" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=Y. Fu;citation_author=D. Reyon;citation_author=J. K. Joung;citation_title=Targeted genome editing in human cells using CRISPR/Cas nucleases and truncated guide RNAs.;citation_pages=21-45;citation_volume=546;citation_year=2014;citation_pmid=25398334;citation_doi=10.1016/B978-0-12-801185-0.00002-7" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Y. Fu;citation_author=J. D. Sander;citation_author=D. Reyon;citation_author=V. M. Cascio;citation_author=J. K. Joung;citation_title=Improving CRISPR-Cas nuclease specificity using truncated guide RNAs.;citation_pages=279-284;citation_volume=32;citation_year=2014;citation_pmid=24463574;citation_doi=10.1038/nbt.2808" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=B. E. Housden;citation_author=A. J. Valvezan;citation_author=C. Kelley;citation_author=R. Sopko;citation_author=Y. Hu;citation_title=Identification of potential drug targets for tuberous sclerosis complex by synthetic screens combining CRISPR-based knockouts with RNAi;citation_pages=rs9-rs9;citation_volume=8;citation_year=2015;citation_issue=393;citation_pmid=26350902;citation_doi=10.1126/scisignal.aab3729" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Li;citation_author=G. Wang;citation_author=T. Andersen;citation_author=P. Zhou;citation_author=W. T. Pu;citation_title=Optimization of genome engineering approaches with the CRISPR/Cas9 system.;citation_pages=e105779;citation_volume=9;citation_year=2014;citation_pmid=25166277;citation_doi=10.1371/journal.pone.0105779" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. A. Moreno-Mateos;citation_author=C. E. Vejnar;citation_author=J. D. Beaudoin;citation_author=J. P. Fernandez;citation_author=E. K. Mis;citation_title=CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo.;citation_pages=982-988;citation_volume=12;citation_year=2015;citation_pmid=26322839;citation_doi=10.1038/nmeth.3543" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=X. Ren;citation_author=Z. Yang;citation_author=J. Xu;citation_author=J. Sun;citation_author=D. Mao;citation_title=Enhanced specificity and efficiency of the CRISPR/Cas9 system with optimized sgRNA parameters in Drosophila.;citation_pages=1151-1162;citation_volume=9;citation_year=2014;citation_pmid=25437567;citation_doi=10.1016/j.celrep.2014.09.044" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=C. Shah;citation_author=K. Forstemann;citation_title=Monitoring miRNA-mediated silencing in Drosophila melanogaster S2-cells.;citation_pages=766-772;citation_volume=1779;citation_year=2008;citation_issue=11;citation_pmid=18634912" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=J. Stewart-Ornstein;citation_author=G. Lahav;citation_title=Dynamics of CDKN1A in single cells defined by an endogenous fluorescent tagging toolkit.;citation_pages=1800-1811;citation_volume=14;citation_year=2016;citation_pmid=26876176;citation_doi=10.1016/j.celrep.2016.01.045" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Mol. Biol.;citation_author=N. Wyvekens;citation_author=S. Q. Tsai;citation_author=J. K. Joung;citation_title=Genome editing in human cells using CRISPR/Cas nucleases.;citation_pages=3131-31.3.18;citation_volume=112;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="twitter:description" content="Custom genome editing has become an essential element of molecular biology. In particular, the generation of fusion constructs with epitope tags or fluorescent proteins at the genomic locus facilitates the analysis of protein expression, localization, and interaction partners at physiologic levels. Following up on our initial publication, we now describe a considerably simplified, more efficient, and readily scalable experimental workflow for PCR-based genome editing in cultured Drosophila melanogaster cells. Our analysis at the act5C locus suggests that PCR-based homology arms of 60 bp are sufficient to reach targeting efficiencies of up to 80% after selection; extension to 80 bp (PCR) or 500 bp (targeting vector) did not further improve the yield. We have expanded our targeting system to N-terminal epitope tags; this also allows the generation of cell populations with heterologous expression control of the tagged locus via the copper-inducible mtnDE promoter. We present detailed, quantitative data on editing efficiencies for several genomic loci that may serve as positive controls or benchmarks in other laboratories. While our first PCR-based editing approach offered only blasticidin-resistance for selection, we now introduce puromycin-resistance as a second, independent selection marker; it is thus possible to edit two loci ( e.g. , for coimmunoprecipitation) without marker removal. Finally, we describe a modified FLP recombinase expression plasmid that improves the efficiency of marker cassette FLP-out. In summary, our technique and reagents enable a flexible, robust, and cloning-free genome editing approach that can be parallelized for scale-up." />
<meta name="og-title" property="og:title" content="A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/6/1777" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Custom genome editing has become an essential element of molecular biology. In particular, the generation of fusion constructs with epitope tags or fluorescent proteins at the genomic locus facilitates the analysis of protein expression, localization, and interaction partners at physiologic levels. Following up on our initial publication, we now describe a considerably simplified, more efficient, and readily scalable experimental workflow for PCR-based genome editing in cultured Drosophila melanogaster cells. Our analysis at the act5C locus suggests that PCR-based homology arms of 60 bp are sufficient to reach targeting efficiencies of up to 80% after selection; extension to 80 bp (PCR) or 500 bp (targeting vector) did not further improve the yield. We have expanded our targeting system to N-terminal epitope tags; this also allows the generation of cell populations with heterologous expression control of the tagged locus via the copper-inducible mtnDE promoter. We present detailed, quantitative data on editing efficiencies for several genomic loci that may serve as positive controls or benchmarks in other laboratories. While our first PCR-based editing approach offered only blasticidin-resistance for selection, we now introduce puromycin-resistance as a second, independent selection marker; it is thus possible to edit two loci ( e.g. , for coimmunoprecipitation) without marker removal. Finally, we describe a modified FLP recombinase expression plasmid that improves the efficiency of marker cassette FLP-out. In summary, our technique and reagents enable a flexible, robust, and cloning-free genome editing approach that can be parallelized for scale-up." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
    <title>A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4AOYkVRjRu3qvfh00Aq8LPvhFZNR0M6JD6BCPxiDywA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460384","apath":"\/ggg\/6\/6\/1777.atom","pisa":"ggg;6\/6\/1777","ac":{"\/ggg\/6\/6\/1777.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/6\/1777.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1777"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1777"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/6\/1777.supplemental":true,"\/content\/6\/6\/1777":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hjcsDCl9XH3fPmwGzNz2s1OaH\\\\\\\/\\\\\\\/XM\\\\\\\/lZ7UrqIA3QXHgnw9tsymJdwJFqTXH4ATW7QdhZTWIBF45cVCMr716V3mA9L+uXWysp\\\\\\\/8QbDBUqfFChccaXLjydciIa\\\\\\\/CIMQWneM+\\\\\\\/0Jnt3MgydbRBdussydAtFTwe+YUENoE3Mf+VV\\\\\\\/cgON7+YREa\\\\\\\/Ydv0EVbEOSaOYAoyjXOWoEzudPvagmkuX3WjcF9IGC7vqhws5tWunNZnRrG07\\\\\\\/2QjAfyUS3rZIBvoqxhCUL23tIt2Z7USa4TNGc2JyOJUNAQNQ++xLisfs9DgpIl9vzvpqn6n6463hrKexeW\\\\\\\/sF4eobTZW42G+GJlXXEWf+12RcyqucY6KFkmEo1udN+NLCBGn0tedzu62IHJ7zxdym2U5rxoqZrG4ArkYuGXPBjgiMTKiWS4S5cMGpfVpLedoTCN08y4+Fbui9Pj7O1VouEqQ+abwzr9ddro58Pti+qLZQSgyHZvG0HJ5LmZUmKPaOpWQNXdGhKgpdp3q7VvvKbknu2xbxhffDGy5lEAYSEVS32FZ6IJ5JceyVhuEE9W1XTBQYDYF4GVTD\\\\\\\/LoL2Lg5gINKfhjWGBKgQf6CqCUtWHBhR7SyV9K45Wxl2A4hhl0zN7RtAUS9vRblnUjz0KrSMXN3SkuJq1Q+zWt4PwUyN6CJtuKhEswcxJIb90rHEsbcfE\\\\\\\/kumncDAD5ZJEeijJZBViPP8YpOon9B6Xfs4D6xG8K3GJXs+dgWS52+s1Vo8aMlcxbNyP2Da7BOgYBfzaew0yXBC399zwxy4eZqvgzpN6S\\\\\\\/hhYWXBDwTkfBw5mAeqYKjxPkIFaCI0Fh6gLsWdFzc9zFcdAAJP14T9ZomBt+utJHS0CQXUBT0SG6uS1SjOYmhDBvuLEySIhwuWcGRymRSCXakpRWL9G6Re73xpoBnVbErUBYvoYKNnQxch8m80JV6rGgsGB84BUpFYSiBRjvGQOSniAontN+A9Ztm0UaParrOzFtkAIOiurttxtggEt\\\\\\\/UKZgAn9YB9BhxLOkl8wegE4Iwxa8GYJ07yxDmdtMNTzulSqbGVYvrv6zAmWCbqpIvNMCuybDJMV2veatT5J\\\\\\\/njwjc1+MH99Xyibw3EkxmoHeNT1d+egtrrgX+GTJ1dkjmlMTMyv5nqQsiTXoLEteSLvdK0bB7HhHL6M38HBBzrcUAOWuhSYrsL0McK+8hEPxI9Iya23iv9uXhDKgcOF\\\\\\\/ilode2U6BSzU\\\\\\\/RYZ+VWxNeGrtOQCDMivm\\\\\\\/Lv\\\\\\\/7zb34aJxfFrBeulK1RP\\\u0022,\\\u0022iv\\\u0022:\\\u00229uoZWDEulJ5q7RZE2WHf\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022635598d20d2af153da5e4458376b76e809105d9746a95a86df66591e5ac5ec27\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nUtk0DfUzRQAx72V39wQgU9yPBHvkJ2rkTDJm5SzMZtRUNMWzznCuu0gr6gfr2u2VcsSSru+aMZYuq\\\\\\\/NlqkFZgvy5Bey46z4J6WURYgOW4mgrVNaTYYSLNrepw2o\\\\\\\/M3cwfd51FM6kF5ru1O2FKOUtMcwNXgVYbldiXsijSvajJYPq2FLTYCOVpZwGBv4q7CDlWPtuqRWYO7C8\\\\\\\/Wo+1tXIImhRViU1vU33grskrnqn9j7HJ5ZpCVHL1sxD\\\\\\\/Pv+m+T+X5uk6gFMGBUQMM3wcrdeKygWqy34nVaCu8QH8t\\\\\\\/pI3ZB5L67pZLZVNWkqCpuKVL7VITl0AfE8VD05bxh46vGpC3oR17mujQDhq9qnjt55IwcKSiQCo6UGGjGP2EdovuGBBAWaMSNfcZ0BuPlBDNxZIZjVbb+OQ2SmaY+sWM3Fx0tZZX6PofELcZfYZ+FyniGZ8NOvZfhi3SJdmhhqNQCOzdAT7YO0Y4bvShh4aoXfN5AhHXECcJ3Oip9GvLmTgk1oCercheCIYFNIY5dl51hAh+S0Grtoteq\\\\\\\/9pOSlqiUocSBgecgzxwhN+LYqo\\\\\\\/bDoy55jMVSRGg8TQj0A4oJ6rH\\\\\\\/NEbNDQf8fxFaoDYFShfL2fg\\\\\\\/BslZGRUR1QdncXMLFTOJ2JCPQ6LVhHzuVbtCMyMmzO0WLPAHBu6FMW8Rdjhche+gR3NGFy\\\\\\\/eUjKjPOFQIKyLNT5dBucsiNqIhaFOwVh9Zp6as5A81WXfYTe7I3LZoVojBN0zi7AY1X\\\\\\\/j6BXhvBiv0SlirZhCiw4DcEGOVebtoLbNzu1O78t+7GCSRPqqVcz\\\\\\\/sDl9whxHwu27hOctlm3PuHvIkpeo5RMzZEaX1P\\\\\\\/\\\\\\\/0YFhzRwx7h1HWs0yHZ1yWECSxLYWyzVmoleNUwpbS4IXmGtBwdXVWd9rmf\\\\\\\/EftGZ1t9kG4tIqjREIl77bTNpvgMOk3tKBf0zaKMLXnAcr9iHZyNZVZcFTsFvhL9xuPAjvoh0Is9Qr7THAHASz+zo8qiaq9CQa5CIu65\\\\\\\/zViTzF43eYEYLRu09UxLPeInItfcTM0Wm9cLwlAx6pQXcU3R3egfUK7WWwP\\\\\\\/sBNTSf+x4S1T+Dq5kFIPT1R44MKoPMgkTql4ZxY8+3JuLHnSNyp1LViekbOxt6hSsA48lMHb71MlI8BrJvze3VF6u7Du833\\\\\\\/9IXQd01YlEmrwiLJjJrLzqQPTg8G6Uen7llzDj6ETqidnYjEbH5h5tlyjYCJqud4B35aJYOxDqxxRjoa682WgzZ\\\\\\\/SQ2tOjL6P4ZOR1wWBUfYpK5q462KCvie7MEAIJFaz+O1poCyHjSYpeMYeWcNe0IedZ1MfqInIBajrRVIBdF+ds450rAsTB3zNn6UHMTn624TnCsaj9fAl+8qabuFfYzxfAsgqasFH497DQUmAnz5S+3Togzh7xVvdhlfuDzCj0jxD7XNQNsB68ngzmZdFQz2rIfX9KnK8RA7GyOKtwHcBPwcVfjhmzG5cLbFaDRfpEnm726tAwgNVkvCLJXDqo+IF\\\\\\\/d0L1QrvWYXhi2JmqR+U956DRybvqGjFLFLUW+Yg3\\\\\\\/WiV2h6bs2b3NxqoMq6ifQeTqR\\\\\\\/hl1KnPMi2XsEX09ERNAqBJleCUSPLlyMeqDm4K3l5LqMPA3yx1iTiKfWmhhpwk1gY58GWu82LQKWbbRGA5NNezVENFzwByUqF9Il64icrHpkIIWDXbuKhNCQ4+k00EImCxsoq77N\\\\\\\/pKLkvIA78C0UmYKCemreQoKw\\\\\\\/uzjFtA539C\\\\\\\/pqIKaGbFIboZjiapPMm\\\\\\\/f5Icib9mUQFUnCdH90tIHlN41cu8EzCS+tJfykHkmO5lRmdqAde9J332nD359xUyY6rOmrliZ5uLA8uSRIuKGF9GPv2MGdR68X1eamIPtoHxU7YcUQMo1ojuCLZ4p8n9Jgj+ne8qNE54U6YJqrCnZzyasTKr9BK8qJll3Wy3J4qH5RYpuT6FHt0\\\\\\\/Js8Le9bOmSa6ovK\\\\\\\/SCJksnpTgNgDodZyxLPsy0vh3x8hUGWkvaB\\\\\\\/bYGBsQfAJfabBaZ4JibQ6l8TkWd7+WAMoIfaEwu9+4dZMO0SYL8wCjEPAJabMlPG7FyAFVsN0nHTGGUhMxSsbzYnFwuBiFe+RBhIZqkxWdyIL318j9ByzOyu90O09P+0yNS4qsP+tz17Od9EB1Gc0EQ0N9ZZHtFdfTtU8TcApeRXILVPTlMpj23bI\\\\\\\/4C2tQeqLpm9x4OJM8NM5y7KS48TgmCQK7zF1\\\\\\\/blbBjmVykxrhb68rCpCz+zcS8oyS51sdiU\\\\\\\/e7\\\\\\\/xo9Zgeg\\\\\\\/02sKvGUpcfvxPcOmVKcvkKm2nK7vyhNtOgcs6O8P704wyZkSS3HKd3kObCjYABD5XlJWzoUSrkqllI8OV05Pz\\\\\\\/WjR\\\\\\\/etTIzJXTrlq\\\\\\\/vGJuLF2jduJfrBjVIMqZy0Dpsiyrdkh7iBQK2HfoUCNqHclUUgYZTuxtB9HpuhShtIXoOnGEByDybQQMF1qqcAC+JWtVHuRUayAN4IbQAHcpoJ3Egklv3L\\\\\\\/y5AMiKvRWwgp35zTT15R56gn4nYWXBioNFyU5G5Iem3N6aONZpsu7eFsAgOY6M7ho\\\\\\\/sBkNa4knJtQeFA5QPGb1nqcYdIIrI+FJB2xKlJ9QoI8Y621X7UvOw65+s2icXGamOlu75FbfKffAkMyv+vQAQYMDu98gI\\\\\\\/brnYF4dGZzeGn1tO6HvEpv6CkRIRyCqwhSFhSoZ7V383Y24kXhGsEZ9EPxuao4qGg+mw+F9Gm7XR8dQ1tuAyOmu0SMgiuefRDEJuOClwP3ZKzO66B2jDiZxxfJkTZBziUXk5q49IMQBe+vkdK9hGltzeQlyYB\\\\\\\/vZA7d3sXzH7AXdrOvPoSgmElKLWsIgbt4OVfAFzHP\\\\\\\/lxoKZjXlJooCxYKSh5qpqmq3PzHYfiJ9TEEvUUFHoCq0eKCGBvsw2TZw67k6alzjRD\\\\\\\/8VRtc8MbqWmrRGF9YjefLboRZh7o9qf9F9v72eWUYOphOTmMpT4iUlGt6wRW5f6vZr10Za3ZHUfHoQ1r1km1Gh426D5epC1Zh3Nn2IEQ5mKTeXw7lR7ln+HG+mrVvFarr\\\\\\\/LS6Ck8U1MCZauLEjOLYsF4TSSD5BcjHP493TkOFg6K+3TP4E44C\\\\\\\/y\\\\\\\/w9anZ8zLHZgU8GKMLxtoxVSY2FDJA+QrJLwJlmfN7f\\\\\\\/ZKSEHWsYSegy1zxEsQsA7ywgAdUZPp2PjpOk8rWcIVvO+J26oSqtjkFl6QZNGCs2lBaYBlGDyLKmX7XX9qjF0bQ\\\\\\\/hZP0f42MPE6BbYs2Ae8aycN6YahLkSBhndN+V7mihrqn\\\\\\\/VUrrHn+eW2CeGMCGBjYT98gQaTxImMuMnxsmJPSDMtRHbt4dU72tSqqzvylaoXObluzbg0ddMwAL4LcR7PaR6x2PmSZQ\\\\\\\/ynqbTlhLEwADCA7E02twAqvQvIfAuCB7WtawSdiMdterRebZCHaZMnteOAWN5GYEMWmJgLUZPgrUK5g1fa8n14r32s2wX\\\\\\\/qEYRe+HYXRlsiEZUD5ElZFykJwPJiYBVXVAhBtM0qLF\\\\\\\/4IspAUaG2GpsGVjoHG78ebwo3mrpelUWHKbaDRwgYFjbkRgzL9OJPCM9sO+ZwGLk+53S\\\\\\\/IMRp+QNnP\\\\\\\/MakkKojF+hT022t1UnTMNYGclrBpmTE44LTxZS2xgLghs6NIe4DwTunleSjg+ttE7NAz\\\\\\\/CLQMJ8Kq2p8izHVF68pdiAkCgXPHQFtMdbmgdtQQg\\\\\\\/Ttk2mxPYzLW+t4Vitm+Tr90181jSMZr7hrMCnU5x41iRkQbJDRqDUz8zboDeJZR3IFoNKDlb65UjCEkWNq7gcLfLrGq5hYlXTEvpdOJJcXCHE2HwzvzxdEkutStGgparCLG1vUFKR2eJ82dAxHOS\\\\\\\/n2\\\\\\\/hM4sPemiSjrec\\\\\\\/ngnNbyR70bbIJKSl5CGreZ0oPTXxoQCz\\\\\\\/i4rp8YTjsIBVdDn2JaSSXdvNc6hzm4oqfozM5TZXndvTv2VEXM4deR6mh5hArT6skipZnMbKsD\\\\\\\/\\\\\\\/Ui\\\\\\\/cilmmwf5w+A7YLWst4juRcEpG+dxKaUHmvFe28kcNe9uG7+NR8eaJbinUUS8E0wbt4YiBQByAvcsJi33dM7SEp9mG3HQ70iVnXaPDwAA3edk+wlDnL3XXbn2UMdqdN+uZ8uTWXxkoaMPTj\\\\\\\/IxOUzoNB+zWM1cBiVbHQ6kJtGM2Lkf+1lV9A2PY51SpF3vipcha4Yn31KsM\\\\\\\/Y4v\\\\\\\/zg+ktIYSYNTJDILdecowN1Gs27wSI3ngBNfRbYMX5E+HuOE2ZpLdPsa+zjBJbgZ6g0K4qMH6Z+BHTQE3CUybvzfKYr2g5swErlZmW88LXe6gtUcCq7t\\\\\\\/b9vUdU0z4gARnpo7wPB2VAkFUWCvHf1LLdjCiAnYj0QlLaydRyLXL7gn4pgeseX7YlJuVCKkDegVlo3R1jhyCfp5TOditRyhWxczyRe5wb3uh\\\\\\\/7ljVbuLDLxeAgdlYJT98R0KlIxfFSL4KkS1bc+1SYEPIMGahipmDd8Y1+0GrRhXUSPONdc3Z1ED4h+c2C7K43+0bIjGw0sIfYTT0wFhqm+198D41e62mxqF0QZQYsv8+xvLRU3PByVTOu2wHWTar1xn5REKQSbpfWaJCAAywm\\\\\\\/WCXC7+Mw7lCJwsVNq4HsMBWwMs51jfY137CrSqMQAAhKEcvuMhvCPtfGl27m7TZK3C+nd3QaVlD0CCj1QZbRrtuqOHp5fdUsINxE7slsni4OmafefBaUNMrHzWW5BqRaT8GhKHvmJmuljii6gl6yBwug\\\\\\\/JSkStW89xitr0T6wniFvEINE6oREGMoPpyaSlZ4HRDiWTtYVejcYzYck4c1MscKAoO7qVE41sy0Dw3jCPj9lxHK40RtMxBrI8OATJryeTt9t2qFDzSMlQbCbBXGUDlM59zFWsl52LfbrGwciSXyxldZYRrfpReML2Vot6ewygbMtoTMvrxFb+YmqVyqJmeuzKdQ21AHZQcoAMQKRQuTM6CfttWPtfxeXJ\\\\\\\/mHgL8zS4lMI6kCa83M+qNI0VOm9Oyj4P1JVNytc04qPjucuQWKGhbTM8xdkJ+ZS3clB82NwcBsR3MV79qRsWbcnswnwRZwaVney\\\\\\\/0sxrtRdQmztNPd4G8+kkxVu6W3a9ogQB2iOC98THqEdJf4oVgbhCjJwy+UmToUSt8MviWjPWHgCW7TlEKTKx5rJOfPtdl8ryrXUIPsRD0vRRs9hofBD47Vknnuqd6TQDwRG2tFe8Op5cmM\\\\\\\/hMAELLeX6zU2WzJiI8LOLkPZtViUPweqcJHBeWE9nMwt1pd5NbJIR4vud4XIjHkZ3bOAnjk384nYK6uB4ePxh3\\\\\\\/Ba7gb2b7SsuS+f+H\\\\\\\/PasZyzVQ9azmENZHmh2ie3KwjxIDvCQ9j429rWkcWSp2v0suy0hDC8KzLeLIq26kwOt32EZ0rFfQvqEFFdP7ICKwSyvkF98skU6EmldQUsLs\\\\\\\/RkKmKJMwd6ugaAAj8l4Q0+Iv6gxdMQ6vlIplQkWItT7S8FwHgTsubHz1CjkwBe4B3l\\\\\\\/9e8IsrPsHl8As7groZKGQW1oYDdqnWngLEZq9UVu4bzzohHFI+I8W+M9cvcZQUZpGLMJy0Zri5ZAUO8nQiqD7xXNlYuvEbJmG\\\\\\\/wv4pZrougLsiFyZ\\\\\\\/U6GdX9p49LAgHqQKKCTzS9fSl6IJM5XTjlnnedu5mK5lSZsrMx7NKxWdEyxgMHLbkEVlHGTxbxnVXCvuxxfnrXuiJBY9Ts6avSq\\\\\\\/QU8+\\\\\\\/\\\\\\\/aD3jLUgxi0zv9R83\\\\\\\/aFug37Izurhi9D228cvsP5AOUNMALnjCKIB4Cm5PB\\\\\\\/iMtl1HoJ560+fva5gBIWX1+NI4ibanxKbIc7YhVuEmc6E96v8KwVaMYlytrcc4mBA57aX15\\\\\\\/Ksift2f6vGeasPr2yJbip1SxxASYTd+ljbmKJs6y4PIz+qar2JoEAZv4N66zCdkTZ8LOJ1dcrZGMXuuzJib6kayqhNf7PkvaxGkTQf3aLTCp3+jma1TL9x4v\\\\\\\/oXS5KFGllocSYYXk7+DADpThgfQzxA8Jq0DidjFGpxYJWfXisHJqRf+B1q+CojVPo9UZm1LOX6yZcP6X4kGyyCxbhmTNVotd7+WUgtL+nLVuU9d1+OsqJCwHJZTbPKdSkvUe6pDI\\\\\\\/jRkzF3yXl722BBk5vbGkMA9KBX9wr68r27cYp1NOSGR4AiI0h7RaOz7U695x\\\\\\\/YhKCIjz1SBAvwLEWrVmiyOUkiG\\\\\\\/0nH5jvhui++Pt00j70rK8yFOfV4usm5ciRic\\\\\\\/kUMQrhFtumkIXn06mor1P4BrIgnmnGKOhvPvPozIxuoBVXUqAwIZNMKgoivfjALaTSOaIv7AiW0FgN6GmKTNm+eDbUPZdtg5VVFmS\\\\\\\/24\\\\\\\/IJpffMdG38bUYXSgsWqcoMdJIUc3jTywm8Lg1Jyxc9NmhzLQIvmuGe2kobyOmWSTYkkFPYhIOqTG70lVWv2QX15Lh74LJdBoCE+JVKlZ939B2KIJJ+p3Fz2aJrK8J3IOpouo2U5JA02L0\\\\\\\/IO+7M5bXBRMzPVlgEHL5saEPswqA\\\\\\\/UcijomvBSk5lFFcdUP7kALSGAfA21ITiiBS7WrbdGkbG3R1839pcmK7ItpI0CBzgDYT4L\\\\\\\/0epvQDEUDoxEdfZpHfKIMgp2OJuSBnLWF2sUGkCYGeBj\\\\\\\/b0XSVNS63bxkIyrayadxmykC+0Tb6flFfTe3rUHeKUJVilwiYQHCrTPHDoC86Kz+xr0zlNc98\\\\\\\/CIuK9SrbuEMr157l4xDQ0xFUWN0WCn10175GXd2I+su5Vdy5S4Qll\\\\\\\/UCpSXpx2qYmkM9d9q\\\\\\\/prre197689btHBgQMlsRnsLVL7Cu2B7mxeze+DHa61I8XpETYrflZ6kNWk132AgA0ljDLD7LL\\\\\\\/2zyaOdSvh6WhZt33RDdmegLSCNIU4zu8y7cCJzClisnGl91xr7UtMpLjYZbmdaatLU4S0QUmhG9bfNyJoGx95A26LUCK6rHM0qGZhwVMFlrCHdixN9tZOBTbm6WabzYlVzhW0FGI1yz0Y8uHAAU7ruZ4FDuHbrJoUH4X6wQUebClrvyBfoHskPYC\\\\\\\/abaXfseq26GCS9mUYsiARrPf0wiFnMXnp6vqFS+9SroPxY9VRHvQ4mNo87sSrWFKaSOes\\\\\\\/z6rFxYBWzVM0QoibtFuoh6j97dKn+c\\\\\\\/YfpH4olysx7EKfeUwp3+XHYGNkIWwaZiTtj8E0pzJ8aKkPONArhmyRVtx4kchicCtS7uAyxc3gWD67GtccyC3xf3K+tE7Zh05K0iLXCafawYn+lFwR9aymdfR2A4yszm0m0aBjG\\\\\\\/lmCQSx1e5pIvKXHcW\\\\\\\/nKkGTMq9ErR8MV4EcY+EIBbOU9pUj0czGCp+vZQnMu7JUNZAbh1x+YA8JuBdD4Jr+ACcDWyI2dl\\\\\\\/T9z1\\\\\\\/kRqRJGSew6uY8bWFc26lf+5OpWsIg2\\\\\\\/Wg\\\\\\\/kuoFxV6T9zjjYTYi3FYZJiHvNNyk7VWvwzI8DU15xkPuMASKEdtPWAxD9ske1h0KT3aKgDGMNNt3ePL50ETpp71JXPZY1j5ezIl1qsb5\\\\\\\/\\\\\\\/hqAJSQnoZSObbQHzPqX9paUoYRN23LQYZWLFFoceDV4LaI4j3cFvCCcEQuO4Vyf32A43lbIgmjHCGN58y8DpDO9uxBgTBA4L9WnuSRwMdnEl2tsrBcR731uEtDTOqSC3w9DA9hD91kr\\\\\\\/SmtuBf7RI0c5V8o8YZk9Xzv+QMzT+OQz3Lm7YQr1TdVvQcgLVHnrirqmXr6d4UM1MfdD9A7lJkI5gloGb3glgndjImwA6EPnRYU8ZUVf+aq8batw5fzRwnMg8boQObJcrgmp8O\\\\\\\/06z0VwRQ5PvnMkDwE8fq6sWFYYKBTUXUaMZ1PyEy2JqZQdhXnq93zxZjwxV2mEg418qW1NyKiOwLHW7rwuf\\\\\\\/LTvWaMe5lid5gAhKiH71GhBrk0RbnaYS77Uhg+VRU\\\\\\\/1SPe5SHOmhSmyJho2QwcCdq\\\\\\\/G5wogNOo9J6T3rqlKzRbzB5XXui5QJVGZjPvG8nKsFtMqrZJXrrzgWyMimAgLhuLEY08Fumu+3GKQjiP2hrTvSpE4uF0vFsgnpqAkGQaNkYHKy7Iz9I9jbwsedQBIn5glK1BJFbAJjMsJoG8\\\\\\\/+a2CLfI6jKZP4qGKswlg24wT2GkqLGxf5Q7nSypqYJXP8sbWes64tNfyc69zOe4VQTDMU9iAVxs2wxBRDiRYCLFTSO9qHLEqy0JawEQKqJsv3PSWdv8n4gRKKJg3vrBnqOVfi2NM8D2rzlot81n7NBHd8QjRVTS652rrQrnuJahsZF4X6TNAX0xFyrrAPYuEFoyUCht\\\\\\\/zIKrf8yGioAcvH6SuK6E1pZJ08s6LHE4sFS0Vpj5bWC+2PEpNRPjH6NAHjmPqZga0oKrRS6RcCfARyDAksGuxjE8ngq\\\\\\\/NMPnHjqzDLK8zfbA+z67hWiOU9l7MEK19sJMxlqS+NlKAbW8QA53pj\\\\\\\/IVKRlteVOdqDxIrBEHOsSVHt9iAe+sW+5E9T7d+zDir82n8KjHRdDxjClNcKKe3YbT6FC8NgFj5YFROYl02hjFzRMinXdO7hQ417dxgaYpHeLVWITLdaFHdDmJ5WhRZmsE8X1gReO\\\\\\\/k7+\\\\\\\/arlqKwA1fWqtOsqm\\\\\\\/NX4snvCt4Y3GldQiIZ+ewGYSNZPldBkUqpRU5Xgvn8HrHawtxjqZERzxOjdZH5DVzhgw8tapoDi11fHUg2VpR4doFLgbX4bkDF5IubaOVSySkLLfYzqfQLikKB\\\\\\\/suMpzlqtDu13gmYqKsPK0HdTOO2zH5DuyEcldTSKENVUyVATD9MpZDQ9s+w+PznVfz05IeiR1J0QRqglCXU4hnTWprBldxMh99+QbvOchRSegwcrM9NqmRArYDFELYgxXhpI0VK\\\\\\\/p7aUD2SovEObVhTwSaWzKmg1pUcsiQAFEx1koGkaM\\\\\\\/d1mkTfpIUf5oJahOesiLTKjM8htTIXyKzycco49A7RFQtQEpUJzxBwrigXoyU8YYl8KOP1cayXPCuJExbwdU3Q8Q0pHxxEI+hJAtfHnMrxAfx1aZQAmaNairQkoL96AA5PYFYNz3Uam4enNahw6sG6TVlGmSktdhk7E8twGs57eGCudTVtWcEMV6zWQcMQAv4ZDLvgiJhC90AgC\\\\\\\/8aNtrgf\\\\\\\/OoJfkAgBVj04OHDJUQCgr+MKlvNuf4r6gF5RvP45j\\\\\\\/zfWv0JK6d58+CECT1SJ00Hn4OmOTgdSS8liLaIkPExePAnGnzktu9vU90eiqvFbolzljV5X0YMUAyeNZ5QNQj2on4um3v7op63l7f+jrrB9eYZNWsStgKpxGSrY4JuZm4\\\\\\\/RaqDTaKOXECGmwolgg1xdo2AwFL9Ir3r8N6An0P7D+e60vje6PUhcsoB3alsiqQVp9YWwtMoIb0TBecvNMkf6BQkXelwi4C+mTOfpR60wV21upHe1tIC+FDZ+CcwEOq1g2xMJO8AO6+N\\\\\\\/mRO+jCHaviRgea3Cwmf0b+CE956kc5wJjqp9N9mODxbLVpb6fgLWyDrYr+X7lLD\\\\\\\/rjpHHBxAqTR5g13f545mTrtsBfz45fH4JzzF5TRR6GOa0pjWjoG+LRyVgecQaO1OH5PwYS5fksAKeaf3GnThSsy+ugL5q\\\\\\\/z+Os6QsQfuZgXNVhLkbyNjr1Mb1um5lORfB52d9fZUKMBRym2PQJF5SDnzPzRIuix8oT\\\\\\\/9KaMKhoQ0AmzJp\\\\\\\/a2Q642ub4hbBlBirQbRiDQunjAvLHmR6Xw8IGz7Bk3azbCFKIKK4hVUMBe+SGQnwyedqkb4EQuHbEcKdIpQjUWxZ6WmWZ0w20NWLv4bS+eJqjdJxPWouiM9ooSjmTEaBeI\\\\\\\/tDfxcqwAyCQzUlGr7hETD1TzHxVvOztqJklN6Vuf8qj23Cf9\\\\\\\/G5U7sopFzPT3iW3+GMx14cmA4iWagKUREt3l8nEzzdLSrRtoFwTxg1TCNCEM0Wg\\\\\\\/3vB3wtIdrO+w9ItjZKbD0WIQhw8p2MGc333WIR8Ehx8bNI+0zEmLLWCrhXCdGYsLUdISk6vKGDm6IJl8gTgvS1lHNkVOHXlznZUKVmO6vn1N9wEP6WeR4yi0qyjseFw7mUCNMUC0I7rqcwor2EMlmKU4XkofSDUS2vnoYWxXdIbj79rVe0t8+wwkNWyMLts7Yi4xKix+vmZPff7WLRQnVUVMLzeR+5vC\\\\\\\/1VeDVXczBkTi2RmGzfwFFb0wEitegQGW\\\\\\\/nE4NufpUtBZunjER94zP3T694vj5Xd7L0xBoHAuyVsNnWf9xtDFyJAyyyeae5KuFT+AkanUOWK1tS\\\\\\\/yfe6h4s87ZsMmNLrb3ptbqUV80uxDUGgHFtowc+mfdzYnVDQsO66WN53EwJXginRLtSi3+FR1vjUvJohIHM0mQrnPl5IvuHDc8EsDaDapCw62d3oPokljonC3Whuy9BR\\\\\\\/lR7p0yaaJ4o+K+tYLR4nndrnwflJcohBrutRJvzQ7Vtdr6CcyckcQEbOs0fCdHr9m7obaLeMpJTYcWQFUiGJUpru6B\\\\\\\/hwRFfcLeLIA2Ux\\\\\\\/BseD63Mrf1W0lDJeCPbMl4gNb+s5AOeRWmkmLcabqBR86+oxxGWLVCI5RmWcYrWxpRESbaWgqRH\\\\\\\/qL8kM89YlKJdHXEM9eaXOv7WAWR7z+bh\\\\\\\/6nA\\\\\\\/Yo\\\\\\\/iTKo1VzBZVqvpt9amkYtCf\\\\\\\/bwswbJQdXwYeN4OgfhUDqL+lrUfrMiJrUirzZLOd4iaqn\\\\\\\/z0zczRwqDOJ0Iwtr6mV+lcntPtskzfsreJuL3Votmag8SDdTsE+r0kISVnWBjYJ6iRmnypyz\\\\\\\/5b91DwgGZlJP1uOZjszYFvNvvMq4C9ZMQeRkm0dE+6rJlEbgGM+usIw1ezM1NsLWX91R3SgQPICWbsVICpbXuZZy2B7rbUAIwpHhzLr\\\\\\\/o7Bs8CMB7gHDtQJjLM3GReqsExbIdBVbACFD4hbFCe4jbfxjl83vr0HUGA2w2z2oQT4HyR4uJB4piaSm02aONOTfV6ybIuojNA1LuqPawTc3W89h9UeTwd8YGnd16ECVQQ6SyCsiP07ZBxKHihllVmEVMWgbbG1PsOZCDPAe8G4UblmnVdwEPAagmiYPLU0+2xfGZq9dc7xNxwzkZMdwwb98hok67PgNcJqMSiWBAPhKJ+A\\\\\\\/\\\\\\\/qbKZ3q0Jy0wE8FqyHLtvuEJnUXqrUU\\\\\\\/nYyaXwnbUE7\\\\\\\/+Mg5yRv\\\\\\\/SWrlzuzyG+x0\\\\\\\/6nMuD68w+ThrOptysNXi7l8i07+HeOKqUibINLj3NeP98ziiyiyos0MXuMl0n2Rzt2w\\\\\\\/Upor+HM93fcGI+bEr0GeZnl2475fOJ2sD5SeLi8\\\\\\\/zLzfivwLJUAL9JEMQwdKu0RKJ3lyNmlCCpskR3+xrw0rsfFj6zMdsahw7bNMFtPRvMfcr8Bgm24xlem\\\\\\\/eddfIiRH\\\\\\\/U+elJqdtE6aIKFvtMYQ8JcKpq8EPEmYnx9B0qnnrf5ZW\\\\\\\/2CzsiZQ3Aj4bDhtM5ulQseoGFI3KhojhTDpP0cvRbtKh17T2M1QA8D8xysfGc8bpn10UCfI+vDjO0MPelESdCoxkNGAFhNnL3XOIAhTGuLv+4duiAmYIqpn31RyWKqCmytUFkKHriNaIWM5JNxCTXt4CeRQ0y9EoBwz2sGUOtFudzGJokmubrbuWTLF7rz9Tig9SEfKg0LPZ5vz8IP3Y0oVPlBA6AodRt9hOJcS78UeaB3v03jmgNIblARWisZC1aPqNUy1\\\\\\\/Pq+g3NQbSa2mdW2KyTkqxsnPNXnbkxt3lLAqYuT1wbNuXQNLiy5QBDJRfmjbie\\\\\\\/MuYXP4sD05uRJYzhQgjnWgHuClBQbW6sH2eHRsEzV73j76L5O3V9jiwnE9quPjoybIo9PRJC7FtcX6i58KMzySyGIR4Gzl9aDjyJgJ\\\\\\\/CeelHgM4\\\\\\\/5ckmkwOvM\\\\\\\/SDnl4364Sz4b7CD9mxyAhUi\\\\\\\/ybR7WdgJfot+vw2cyGQCFYYYS65Oxz2AZjZmTVma\\\\\\\/yC7leIHzjAWwMgfhOIiyiI6fpGrgK0NoxKLdyje0rZxKq0bTFLopaEUkHG6HH9IH8m\\\\\\\/nNpiLXgozNoPMJOpIv5WV\\\\\\\/4wCY0X6E6E3jYJCRTPp3cQKT7qxV1PHATqkSqnimXwk667k0wdwtO5z58XzpRbj3yDuRJjS0HPPwKgk+iEfckcSbvlCmtg0\\\\\\\/Z+gcISZCNl4dyy69zCTZ0gI+2NLZuvA4Om\\\\\\\/YfrX9rmwsisRFpHo7OoCFacG0n+J3wBjB3txkC4MUqjwD2wtkg+dU7CUVaYst8Tgb3Z0x0r+d6EBvQGSao2Fk8xoUsXNZB6D11J4YGJS5Z0mqQsNF2cNz4zQlKl4q8uSK6rE8N+vzNcwAghEezMpGrX93ZEspWz1cN4ckvd+2ylX04fqjtXb8RE7YzyPc3zT3jsMbjS0T6JaaWoEJowa7KCjtCe\\\\\\\/RnkN4v0OmTZj1j9zjP8Xu3fBMRBKZ5dynQQgcC2LX0VyLsktiARKGKgKul7wA8PTTAgv7xwi3VrlC5wEuQ09AT58SZUQvUXPSdD85k2Km40t8L+7U3VltxpyFrwsRDxURoJ+zSOtRKXz9KFMjUuWE3LTnjUlc38oeXJqhdZyckFImhL\\\\\\\/V5eoFgOrgayGQ1mAINdPG5ZgMpI9L2ycaWISwzhYsatGFDlF3wigU7lXpi\\\\\\\/8tOtR4R\\\\\\\/e4L0QUcnpdVPWR0A+OY+rP6FyRceCeFGJ1u3VgredZph1gLgrqEw5ut2kylaBc4H6SWG0KooP9R5xHvTbGJRcWOz8DM\\\\\\\/U7oJPI+63HmWpMKwf1pC5Hey0Hfa9z4Rvo9k\\\\\\\/3pOeOgVmTX6xbwjp6tencBYusr+Hcmg641gBJLxnPg2IicDSIgbQMbM4+ALU5y2aFyiqYyTQQGdJ0FTe\\\\\\\/CzoobNjJMJtWPx+09KiAUY6c84xIOMI3\\\\\\\/zk3Qq2ZASHEaUhoEs\\\\\\\/fz7OmlFrINIv3upOuhhKfN5jsadXPblR26v5t\\\\\\\/SiPkYXakYf8MEJdEKEVrrsd19Mif10SVmk+Bb9I2MJCZF0Ukj3pe4EQmiP2lM0IcX7upmxfYSOcCi2CQJt78MxiS1yLvrkZJVKd\\\\\\\/bKg7zW+LLYS+++\\\\\\\/G6VIs5JZYn9+jyt9sTZPhT7G85fUrs2dqdOPj4jqjCcDTZ37muQfbJsrvo\\\\\\\/wrUbGlUUgmpfUQR2xKeQ\\\\\\\/9wElNWFrepadDSQzG6386qufMEyhbgS+qoEvCZbTSrm75NGrO7\\\\\\\/Opms0\\\\\\\/8Bl9v4iPmkNujscfMN6D+H+i04JuUSXaKlbrRedCu\\\\\\\/0nTbFuJyTNeBnmbSINhTCvVfYkjfZmQdP9+AY+nWjXS\\\\\\\/Mqr\\\\\\\/DqpBsz2w88yYZ5FrPKCkzjPtX9BmtZ2RANKnw6+OgPONk98lU8431bmu073T+NQV18a66prCl7VbVj\\\\\\\/59HwqH6naFC+d52bZJR7heYLo3tE7eGIRitqMKXUOW\\\\\\\/Idv5KIYos6hDAWJ8MCr7CmQa09vqKU8GgJhf5K72Cf2MhZhaFhHXMZHUA2ZCeaMUVHGNmqeAqPkD52ab1bx0gRA7sZtB+sKV4INQngDyDKM571mwZImQw6pTlxR+bgHGCGei5OKmD9c1uSMCQzD4WsZ3Iiktb8b80mLsRTelknFlT64dEVpIyTWQSkYocSZJr+3aXTeGY+d7LgIdBFhKBtvHybcGEOSq5GUxPTg+8BG5OLI48G5Qk3wYDIW7jZP8pu6keYXBi62wexnMA2Yd3exsMjnecVG9flx\\\\\\\/PQAiDebchEFQU9+jzKENxjCID0Kn3VK+jQTF10DbIBlckWiuNZCjJFPLasUpiKhjLNVjdkLp\\\\\\\/T4ZDLewdZ98nJJKJ+3XuWqJeabewVyiN91UK+eCgtJUbgUB1KZS5tZqx7Wprt\\\\\\\/sKZAQAsdjpdTDfz8Tf3p\\\\\\\/A6nz+dir7a5Pb2Dyuu3akc2AOh2ZDfPpSCNYgTrtGgXwc2WgQsSKAizsS2BreyqZ26OZRsYwjGyswbw+mPpcQMi4nfXYxUcRj7xhW+tWjUJOadz6a2+2F19cqdIhCm9j7jvXEgoI6mXeby7UVPtxWdI3kG0+g8Cu0gb8D88FS7B5vf4goM9dOwa+G9klznDXDbWCPBd306jJaaExWkxLAcghFsE1Nc6\\\\\\\/Q3SpO1SlhNnEYMAf0qXeaENjLzG0SewrNYoF8OErtGF3hIDq8nn78IVHpT+UinxC5k\\\\\\\/fGG9T4+50vVvpYNyOkipFeV\\\\\\\/9i24jqLsD\\\\\\\/0m0Psn5CYDnmZDOk\\\\\\\/lGrg37o0TnQH2H0x1fDnYuB4xFgwUs2EGhky4yhRpOAzWBEXXWwsth5B2fgWn4K9oSZ3VzFgeT9BVYuuo1DV3oWBdOl1VyVwOB1V8EAhxtDL0727SZKPWGLH\\\\\\\/9zV880u+H868lYmzB3dvAtGtdNc7oZ6J5mEC8FBPXmA2WJQ9cDPMVinhFPlQlxziqhJ6boLIYr0R6Z9t8YJtFcY+MT5B2My8CqbI1JtSk+M975JWnIZAfMtVdBiY9GT76RsgbSnNqIjvmc9dzBVvF4bWtBNFxoG8iHXNAEDTrpDTzVwl0WKFM6ERl2sCTU5k8q3nRbEJxPN0e+l5BDL\\\\\\\/8hWLrLz5ZbrqukAGMUic6WVTVgTrTmWA\\\\\\\/sD6Quo8d7f18RejP9Y1F1pLIXnM3TpDv26D0dbQ77Wa5tAMpujfEXI\\\\\\\/k0yru4HKkdST8f0gGG+AZRvAYFJZLNohPpgiB+aPj7qaF46ZoX\\\\\\\/gZ+KsRwZokOWW9jbFViGK\\\\\\\/UAVOQpfTx8to3AGQ0ZmhLymXqOLVFH5Df90cXlKd0Be8inKsMPj2qiiNarWjbz5Pwnrq57ZvjbHhscs+WxJ7A5FuU8vbx1l+pNuHKOhEyGuh94x4Dgw8deYgZMZyFPOm2QTtuLikC7EO97iuxQJfMAXSS5sZftn7HnbXW5EOnGN7oTnwh0SARGNcdyooLWfcomCEXKDxRAGRJThY1OwbaDNGjFYOEoMSERnUxiOlfRD2pox89NUG05Xir\\\\\\\/gEoJfx8rItyEvhaf7z+MwTWCE0bS5JKAW4+Nf41aqYzPgkJLWctUKj2UB1o5XQn6Cxvq2cEfTUQDVHZkcktEA4Syp5USj8Ts7eyf\\\\\\\/fE31Bt0uYv0sJl6v00t9X4w4Ck00dbva3nKR4F\\\\\\\/SnaJjPvqKefCitOlhBem\\\\\\\/ONSiRXS3h7b7F+Aw571U4IjxpESAYnsiUy0vLy1ho2Qxk+3NQTeVqq+QpIzd0faULQYT7jmPSV6yY6Fw4w3OQIz8uwRv4jcbXjooUU8cXtCD0qtmzwKHGeojCQrIP8gjLnleEGos9gUL2VRJ8UJStykBkCvPv4WjWNLzAOYgmoM1XKfE6moRhGyd5tS26E0meWxIunjedEj+WgN8HLrqWVVCpN2IxjN9YErWyfm+cAy3fBYmFCNIXmLAnS5Y8oSFdcnttnMyTHcDpFRzWllkBz8O0rD\\\\\\\/BqZqY61sMOD\\\\\\\/3FR6yTSUb\\\\\\\/d1N6OKVF1A0JzKmG6mcBc+VQzUPEbDHaEDM2G9a4eB175ItX\\\\\\\/TbDMAmlotyADwreVtcF5CJoxeYAfQ2CkYw9AD3QqMPaNI2cQgadq7PrJcmDkSIigjgdZ5Lc\\\\\\\/Gx1hKi4mLeagm6cr1R84xZHCFQsL5iEa2oUNJfv2AOtUVgNW5q\\\\\\\/at7DJoesD143WA2OC9ds1RmGT1Vm4lHKc6sY609IGFH6L3BFmM1E0AuZHgH0dg4Li5Gjucz1GEtosJvKYusg5NKWf6BhcjYpKhrCWIqoATV81vgDOIWRbNN0M5Z+5JSybyAHCV2fQ+8BKKf59XepCDBeshFXqCxbIw+ua2NvLAt\\\\\\\/Syb2j2oFGxQyTY7jvGW\\\\\\\/9dRkG3o4XjUV6F8CoBK2ASm31Rxg5aElFFYbLJbrkSRCYTxA5gtAV7kHCLdRgenTzD\\\\\\\/vXh4yVPV28q0G5lHZDntQOywPTGahYPfYb7LSbv1aulSGlie8yZoJs1VNRvbK2GZA4SYZTYx6bxgEaP6dlfLiX6cjhGFl55lb6n9QpRQ892yrXGktMZFwzVJoQYhFVoeSPxhri\\\\\\\/lwW9xrJ5bInXdMldPM5T5kRwITOXiXonb\\\\\\\/KIhWSsfn4Bj\\\\\\\/AFIWoEAiO7XJoFbqc33oioyrgrXu9QCLBk9LrUItspeV6kcmklXnL7DnA1OjIXO1cLkPx0RP9CU\\\\\\\/CqSUHJ3Ty6iJ\\\\\\\/exrb38NCz2vn+0VXfP0EE0ngqZ2oMonObXvrEtijMPmYpM3Dz3xyPCaci3qLzG9fCBEH4ZwKl\\\\\\\/Qa5y0IH7npdqdl\\\\\\\/5Z3EZ05dc+TB7GwfEOfsbPqHLuoCuTYigtT6eAF\\\\\\\/LyFbMthwFbo2mBusV2w0PM6h1\\\\\\\/xMJ+6RCI88U6ThCylWlAAn39zGqL9oa\\\\\\\/v68bKPhL8AFqtRVYIytOP7rvVKrJrTDqhaXq6k0YtToJ4jWKh+6y0XpVOGVR8d4ROMveni9cQDRe3BdjdaN13tMw5BJJEc6SiUmNZhjTBYptvnr2W1fb4Zh06Y8DslAjMkF5OyuJbFWoDLa7+5uEKSdHDq8gDcfgEVy1A0wE8ysA2A\\\\\\\/znhx8IyFKYvsozwpe+Q3yQ7evzXqXchzvXRpqZCOliQF+MGeoABsBhGynOAVD7NYEZR1eDv7iHi+pxKM4uMBoVdYVnCb+mT0WTtOxiQs4BfI5ueTXO17NRxY3jtASbRZ0Pfqm3NMoXRp+HGqAHfSjgd1FSRj0JE3Ww3P8PFemQR\\\\\\\/lQ1FFPnQyzklCWOKY3IG26OpXvJpfXVKt+q1ObYZyHiVNtVJt\\\\\\\/wFT4cVpsQnZTEROYK3OnSMpYfphhRlRYkDnslTCKw\\\\\\\/V2hbBMPX2Jj3OBpYpOUeW0+8+wFg2DdgEpxBdqqTLRTX4kbZVidja+3AP+Zz5RFq\\\\\\\/Gw95gSzVbTvPm1uLQ1Fd6uDXrYjZYgoLLCj3LaddYg19lJBsS+IQuS+78hhfqm7sgHZwoCQHQ6gRJkLRqtxLCXKR6sdDmDXbnxr7py8j7Hj\\\\\\\/jiE\\\\\\\/TzNG57qLz+X9KRkyCsUcRDtPXi87MQOODbdRf0yDSESBvvbBcT4vmvrv5cWxjQVXQ9elvSzHxqbxI9flR3oIGOe1orUmMB78E5flfaleW5BPZzC+\\\\\\\/Mbg0PoDqYL7mc7Ah1ybBTNWNjpXurRIXZcpx74clgrXSuNxLMcRJUTD8\\\\\\\/3JCyZeio1B\\\\\\\/ehs+baExOb5kK2r8fooYws+Qz+KZBvUJQl81NpLzCFbfxiP7YkDDdE+GCBgEpXvvrUWnyL3wc74PjZ687tQZafxEMTvC17oNM3OORJ7JDMPrPGK\\\\\\\/gn72cLvC9kZHjMTsBzkCD\\\\\\\/xzLCH+dBaYHnlpLh5xdYvyLsUsrDzXhZcYTYyTMhXfib1V9SWiy+KIv32IDVJGPjf39lLAtsKoBlm4QG8\\\\\\\/ATMPVvXpfFnkrwci4Gyvxm+9owAMGiHXCxY7Y7J\\\\\\\/\\\\\\\/Do\\\\\\\/ijR4A+EA6nxEwXCALZ94Jy\\\\\\\/Grwh0916G2svHzdhKubzVBGOakjn3lnXcD92060gy7BIdwY\\\\\\\/arWi7hHSfA5UtJOBd5uXd4r3FCzlq5THj+N9EzVoT8gvyjK+16NNRwk8ksh6piKyesmQLFgi09stZmHvDcGa+K9ZGy3VNt2WHCiZEbSz68SZSVkVmCn1AoAKnZanf9t1UUXw4XBwZsWTS6IHqI8EYLeP6dhadbOgF1x0rjVttZLJGaTr2rD9zHQvXtjrKYPYwc46q1DyHJ78Uv\\\\\\\/xZR3qnWkCD3kk8milNoKinwgclnsnVjVg6xH+Q2+66e+NCzOfrBOAr5didq4nZu0fbwZdP3M7KQbVUZCBaYclRoT0KJC2n+S0aA2Fbdavyu0bClUGS8VAzoupZZTqz0ppiT47TNr04mB4FBFvIMFYnXY0iIa9VgqUTohHkGLDXpIYBjQv1Dm3AoIxaklN7H3HdR+LnlBGCpv9Q7TBApCa6c+sh439URmxdtkSmDnetWnL2KW\\\\\\\/6lxvN5Frg6GKSbyX9i3uOXgUeTJmA66RO5FfeG5K5WwAzMOuVYYwPyIHiQxW\\\\\\\/GBlRsCOvnCZBCf3Tw85KUZY1WzVPN+a58\\\\\\\/JZbGvTgvvCu1ltKQYN8lWy7QChqr7LiO00712kDVKRCDBpl5Ee3tU8u6bjlfDMf6f1586e59NQ1nL5bzjaTUVJ1shOAiyZIizceWX9BSQhJF4lyxdnSq0CHhK5Cruoa8Zpa\\\\\\\/eIf7eC0KbhR+hDt2nGZwdVZvZW9FjBLEwDfMS1NKB0T3vxgVJV36neinBhUIMp8IQQc1XtUaFCN3fRptxHk9oddWQy7G\\\\\\\/O7Y8ueEhP8PetKF4YuJQ5of44xT5MiR7n7yLs1Z+4hbmIY\\\\\\\/sbJcPoHRVFGT89OdpXS6vw54DwVDj4dPrrPI0EsHB4AHdD0CzC3c\\\\\\\/UWo3fS7a4nHNqs9qmXOzHLBty9plWPbbGn3nJenHQVDq2RQg47jSyk\\\\\\\/oUBpodUnyrw6mKth8Czgs5lfHbPvCURElfYuAnsfS+iM8QenA8zqU\\\\\\\/3LKI6x2drTMVsu9BoYKw8hwy6qh8o9CUsmxb2kS4PHZjto0FAHwNzure3fZao8SXgowWKr1iYjqiSQv4Yrip0GnvjlaLbsa4NkfStVqNvmS3Y0oB1k6NpZCzarwGtNSx+xKc0E6HBFnFtWyO3V0oIgr8BKsARyXh6tBwy36yCrAUFQ0tPtnllO76m\\\\\\\/mY4S5TnejalRa\\\\\\\/NfQR1QJp1qDlTMAypOTiI840eJ1Usk5SA6jsXX5xtmnjKLMcBLiCjx3HZ2DiJ+Y7ETHEY0KyfaG8QamHrw9BTWvt1xTZsHXQw4G78J+E0TGuOdS+NbnDkKsj7SiLhiLsQPbG\\\\\\\/+INtMr2UoFrGL5s73kZM\\\\\\\/7aDK6xuDZVe9Qyin1vKajRlY77wSK3Irqzx4BwZP1c31qtjLu4C8+1Ea8iQTFUn2CviWhR7PgvgQrEM3YU64j0BIy5SNh0WMZOiS2hD1sW2IrgpZLjsKqOQgPSwH0ryU\\\\\\\/VanCp0BPdYgowBPL\\\\\\\/JV2pqD9QKoaz6Agmlav3BVtLYJ90i\\\\\\\/UK2ELO44omBZoUpd7cd31DiesdgToGEx2YIC7SKKdQTHHt90du1u1ggi5JjvMPhdnKGFx+BX4j89MPXeZ8ggdEaz2FpPPcaX4QxFmGL97W5cEAqcoli+bOPHv+7rX3hDyZC5UBzz5ysS0l5WcIzSjyk3UOk4mvYiZj6XmfyUNe08xFXdPVtQRMmYSN5ckEeX1kh6p9uL+Gd86ecFGaPu75Iz2vEpfGVVAKq6enrywGOhaIBpLtF2hCHJu7iQasdvT\\\\\\\/GXTxDuSASzN\\\\\\\/YC\\\\\\\/x2ztaxzcmbHTQIGud5Pw+\\\\\\\/nXAB7aV+Z0Dhny87DWxUxwugHcIitxgLSUnFMC25s5N2amxYVYPIvwM9aFL2KdhI773BFt3PTEQoEij7vdmzpWRt+QAJYgfMqavHQMmS+HaU0CtZaZ2p3bsYa4jJhs\\\\\\\/MuWAUXo18MUNOo2c8SAGU9UyM8n00OfhPHwZl\\\\\\\/xSgveMTfx7P0PY8xVTrvu7rXYFeSUDQ690yyXNMR1k9XJwUknWTc1u9HCTRYvewsyH+kyZQR9UNWhoJ5se4g8GRuLgUjeksVoONCq5z22g5xZ+lJ9omYtxqFhi0GayWmwDkThVI5ty5FFagu9xupIk+cjQwY8qMHvvraVU+doZ34ezyKDYKXufE7vp0w24xXaRqWyiQ\\\\\\\/rvv7wU0UGvf2nPmQF1YzOj1lqaucdweFmIDvJaGblAjK2xWsrtvrSw8uozJD+FPkD9q398kpkdTIrtL5\\\\\\\/cHwhA2JJXp76WZbSpHAQBWGItioQf+V6w6Nxk9VnTLf80+nZvY4\\\\\\\/+hTy8Eqzew6VJ18fnHcZnAWeujT\\\\\\\/o70fNYqp32ya\\\\\\\/s04Dmh3rOb6KAFkkecpE8j4Z9dTRSYtbdDZXk90zR4DWSCflxtb8FskX17DiOrpxH+5PxJ2NjG+VrxfLm8NKlgu8hQlfa5eBbnRmDTsV1bnoa6I+UXWCadu1pZCLCEHWYXtzat1dQV20KTIYHXsphNrnPzK0bykoKc7ZTf4SANkuZUtv7FmM4dqcqNVUiAIfsDzE8bG7v3VL4oHTnl1fHJOv1p8YYtaFrqiz7MrPxwJ6\\\\\\\/x7X6xUVn4T6DQbZOR3\\\\\\\/qnSRacCxJLW0QK91fCadhjEsaCKrWNY+D\\\\\\\/7cd5z2A3HBluUy9QWdXnhPYL13ZGF3kcc2LgXcnLMYZR46B0YwreXW9gqkS3FTUB6jYKXxK1l6YgoRh3KwbC2yhcYVSZe22hiH0VVpD35bJnt7FoTkeB07LNGCcGBMenDwGMfq4U4VbqSpgN9HOP7DWhXepTyToStRnN7qgbqDLf6mhSoWcWkaQ+piNrwyiMBpTNcatSy7njsrvskfj6eooBp0mtbetNtegsTJJYrj\\\\\\\/+5zIKzEL\\\\\\\/Ch2FffSkV5EgigG5lqpL2XcXijbrzdf9YOWY6vpDBp2GJTgik1HSZ6jZRS5SbVapedN5sm8pBODveaNxSVrxl5PNQyVv3ZliEbBCyPWhYxu9lO8neH8iIm9ViBKTTkxDDrTeifGe23aLRnzUmct\\\\\\\/scmzyTbwsadaN7ENqD32ypYRFgsz5VV8GDLVqvM2yo8RL4OZKvFZcjLbVVkK5JCrrzZpZR4RmNMMRaSJemUg4TbPza2W4rQKhefW8oK9J4LozuozGHiGJbHSRbWZ6j40+bb9iXIG+bSuAwU2FRNbPg94hePrFM4Fhqppib2mbrwC+4F+VEfM2IKTV6QKoF+9cvYEn12yDgcWzXN0MLwdZt3jrK+IKccrivRc2i2RQdMNT3htQZ2IAwQgv6lKPLCXYmtlaHftMJKHA8hG49Y9GkeXi4yrbCTI+BmOzpvwr7SOIx53LeX3Ugq0E2mDNxNUAa43OM78cgi6WsbzYsTzNxoyOsUYpBtuF4oCAGHAiiWLaoi\\\\\\\/hIUfLBLwbIgeJG2JHLJTfUTXB\\\\\\\/ogUA2JHzWrXvUA1b9CZX2hzOfkZ8E8IJ7jVCtUNTMcqJRgYYe1m77b\\\\\\\/0+MfLX\\\\\\\/OxLHXz9fhLtPwOurPZlSofyjD4t9dMViCCJLBxlZMJyCKJGPdR9dVVtbCwjdJAtsR+DWy2c+CSuFoA9bNyiHi9yzk0wfEoxfid0BRQnbwuu9YvC\\\\\\\/ljO1BIhIshydfG2aWHvgpzHyiLfMmfmHeUSDoxAA9BaeB54mw7BouFmYS0qTZmPeW2NztVxcSAYiclRRflnQPSQWorvGIDLRux0hqB3Y9wGfEl0JzYSPYYO5kknWhjsxNGX5moIKwGsD3FuvC2VxGBHjMs7z3S9vnZZJK94G3d+72bQPcvcTQBjnT\\\\\\\/o3EabumIxm4qlYB6cdlMnsKjD0rmjjf+UgkSiSEzKwyLATLj0wAlKN18ysWeyGP+q+BtOi3peZ1+ceU2RAFAXibQnkF6NSKuOocfID1VCVNNp7WYygeioADdrw5PaEPZjSrYRuK2OaktGXyMnmvbuBYsM1zDI0tOYls6arxoOjK6+sM0FqjJB253kyBwfpvvH1P\\\\\\\/NyVAIGOQsOI3QwzJqSGXNqmueJ3zKHVVFYBZmKEZ1TJEZs4PevXV4v69zPJ+RmsO1lMbX8+zBpE55exQ1bGkA3DI8mZ8vwGNg6u\\\\\\\/m\\\\\\\/FR+rpaAA3jWH6O4cSIZ4FDVZsid6cYvs2Ii6yL7f2QR4m9lsn+ojeI1BslhA\\\\\\\/LGjsoC5yv26w4myVDwRhBv1buKQBTLNIL0ONBeIs2dwLrL5NZil34Rz\\\\\\\/qJNKzYI6SW3hEhngSlV+McCUTwbdza2kmxs4Dj+chHJyqZVSWTpEg5FIHyE\\\\\\\/QnUNNyBEbbhMLXy1lNrE6LHb81+m3ZJ5knkRlayFZOC7y\\\\\\\/uTrXKg3NKjOhGliddJpcqt1NlK1BB\\\\\\\/y6U2\\\\\\\/JfSCDM3AxGlCdYFh1Ou+HnEzGW9jfFhxUhin4INP\\\\\\\/gQs3D5wzogFF5czFyxICOecXjlFMwF\\\\\\\/YrBL8hOmk8T8BI1p8JZqDgOYSZAKfIjAWgJtbP4oGsyePKZTTihsaqsbg2RNQKCDkOlICmnjVz6gV9xfcKFWKYlcO1rpwQRQpcxbKRg+6hzFHlASxfol1ZDWKgWqPoAhmXUoyvM\\\\\\\/KJRTQ5KvfqG0aCSbJ4Lt96ZmsyxwSV2zKaGH\\\\\\\/NwRn\\\\\\\/t+yCxEYbjtWO\\\\\\\/zjP1TS9+NS\\\\\\\/6b1Ol9kqy9uRyjXjV\\\\\\\/4Ioc+0D+i4IvogrhpBbKh7gaLcPJCopmCcqTd9BnGgHDRAZGpkVLzivkgTi0wdZLZ1dxEmXPYSXnIybGwjI+zBUmxsJfNnEyGbz7i\\\\\\\/mVsjjzUqcxd3Pd+ct7tvOztJECPs0TNPQNHd1fFSvPCZ8g0Mu1+IwJp1yQ03ha5wViLyB9Op0+UN0PZSJuzMlP2eA9OamVuRTviQuy39vfdYvRXOkcxPSamVY+vjY1vBoPF\\\\\\\/vCn5JpRuHxcE4NpUY3ZvgEcdm6zr5hi\\\\\\\/fsL4S\\\\\\\/QE6bilUkGqOxgfmwiTQU8u9UPofYTx8GrYzVcj1jKfFESblUvATb1ucWoGwhqEYd+kkuV37LypmMzGgdwlHn2m\\\\\\\/mgMW56vf7b2vIIXeOUNNe8IJDrlAndlnwftDRGMEgpkAb50Lu+hdxxfxTd1vFQLRPQjy0zoIHylqANu9DA23tW3r5h8d8dOvut7OlDQ2uZgibjfX+hrhwbskYzrAWJfEbd38XoZIOrsGgGDc+b+oC+bXmNUKa3G7tdCuhHUNkXThVj5OPXLc5rjOTJN9XlCJ9iTaAMlRAF\\\\\\\/tepFeFbQnPIRr4ZdPL7aSWIfLAhqDpzKSWfshcfdAyBJd051u7vl1UwGlaScRRJcFX\\\\\\\/K5a9VO08A0GmjzPgPd4oLOkjH6BTqktAgF6NT86Cm9J0X\\\\\\\/p2SOfvwBMEXjEipJgY\\\\\\\/geSAfpe7HLs6pE5BwM043iEjQHj1sSuhWH5zmImXZc3J52+ryy50xE6EDBpAkUVm3QyRU3UXde61Y9py6G9pjEj6gbrLGgafVbE8Zxd3aL6mlCELWRktQCzoRAMqbU2FGZ+uuXR4KYSvGNRf0JBv+Mh+1sxBkI67FDHvMMMDi7nOcvhH05xS04pHoIEJr934r7ZpJSgtPX9r7CkLz8Dfp8KVkQLxgJbAuwC9PgS1sgV26m2vWDhURW6xrXUksj\\\\\\\/Lj5K1k6+kC5iKNwIenRkPBNgzrxxm5925d7FXBDBy9sX2Gk\\\\\\\/FHjnYBC9SbXyV+zSH4I+zKaaf+3hBawg0vpZOtnwAOZd+gWIEcArEGHI017UfZcOf7UT3Uqt0gC\\\\\\\/ttEDhwnUEV9X6Zu2cckaRwXdp69spH3CV5oouF+IRPwSOXHVtlJgFnZwUfrh5B77jmi9Zrp2NAqlnGcY4Eau26p9yWrjfftbyTuRlK7uHaE1CGd9iZ3JTcgOPoN278T7Kp13DrtCFOx9H\\\\\\\/TuhzOf2E2N8z+z04Pv3w5MsolrfNkw7dTdF36K8kV0JGoFjS8eSL0ILN9mCNOMMcEhIObIexI4sDqM34IYRzsAwThoXJurroiB8EkGCQXzOJzvBVTFDX3bEVHPJi1Pju5qAJOU6\\\\\\\/xsnLs+J5I7NS\\\\\\\/zomG\\\\\\\/Fn4PC5xK\\\\\\\/b5Vv\\\\\\\/spBIppES8Bc7JhgeE0Yc3ZM3\\\\\\\/ibjqFqAFRj5RD7svbpenahm\\\\\\\/c798W5sM5CyMfPzQbr8BqIisZNas3fcwRIP1oVG86SRpffQSIyuWG\\\\\\\/mpcKmetBv04sX\\\\\\\/WTyPfnqzbwhPe7OR01Kf0RtrYUhlhgDNJZ\\\\\\\/wvwvnIrgYIR8k23i0VHy5AMqMsYhrw+cPLGv3YqvgcrjuYFJmn67FduwqKWfHkSYpsqOee4ouv6f84tPi+us4NC+koQQNZ7wkTo1YAMwAHzLQmM+q6nyP906mD7m7Qp1TqHSsAHok14JQMnI9AYcUuWAIcOQYfrvVaGCeRwIskfzeARHldNwtq9+y55LRfUtnGyrDbfue9VEfSCnE3WaPeJ1gPtShhe3gQWhURHB1deL\\\\\\\/WNx4831574JZudfSAl49nqqbGjonKzluAAAG8Svz0FkQ9kL1Gd2wpDB\\\\\\\/i3a0mQgzORfCqJTRMTbo3CYj75G3ORTx78jJXLZSdSu78Q6lS4EdpZVa6w16RFPKJWfENC0KbFpyofTBDdaB9MYFqviXtxdLb8epC6BjNw3rIhCIcZ6zJbqLYL39h\\\\\\\/udlqaTfuFy9goQtuj9r6cNpfQ0+ZaLyRdOQLhI\\\\\\\/eoKS6AC3G3blCKQ9PdmjtyhR3lvAZ2\\\\\\\/8TCEd3ShMgDshGoj4\\\\\\\/hWrywa8\\\\\\\/8vb7u6wYmS5OCCf2fiMtP2\\\\\\\/fBldhOkSr75LpRlEbvgu84762X2LJiqz+0EvceNOQYr0NgQJGUNSJRJgAiooTWagA8FlaPVA9fhQZey0okqb2ZJi1N\\\\\\\/N6ExVFsyujpW3L3pcM92P4cDiVqk3vsk44WI3R7XIlgd89I9g1AeqDS5N7X+1Zyl54nVnbygjRvUrmM7P9NSSCI1+BL+pRkihj5UYtTCPN5pdl0l4msk88a6m7wJCfqjHo5u5vYrc4rN4VDX\\\\\\\/OPvSj62pWE7aGyr1pDbwPo825CztoCBiIqp+wA7Xou2BzFdyIBPVCxuEpdTB6L8GfaKzUhpc8+yNv6E9X7uoDg6HNpOHTgJy1g476XiFehcSqzo8uy39vpp927SDdtiGBGJgz+7pExgaWVVvE0Ez55ZX4z52l7mgEBx4EiypgpFsN92AkrAGjjWUZTMOzZFBXVPteJpqrCS0yqfu\\\\\\\/t4OuiFNVIjNpDiU7\\\\\\\/c4QsAGDQYJSYuNBPlgifkcbFB1teebXWvdilGtBmzVv6NQE0IkR9ooKs6ikaVAZs9pWwyoRwL6P6oqYRCmLhaEh4d4QkgmGCPfM80BH3kuN9s1BC4zrxNfMppNBga6WSxKj4LI5LLkwZSoNH1RDHx00o81oBavu8tlF6EldlSWmxLTn2OL8b8QJRYEin3m3Ow\\\\\\\/Vtda29PEB2X594KFKd5qlZ+gEXrxbOd2ouqxGgYvlbvs4IRWhOjHLZZwto+POEjN3v5a215AzuvtijfQKRMinPAoocjBAsJImWG7PXy6EnskJw48xE+vLNz2hGA99So6m9Qd6oyrPvOM7KU0yh2eJiSVP+cDgO+fmTdjMC11g\\\\\\\/DJzlEWiOn080Q0KCiFVevn9g5QZPS2pu566jLDoAvnDmEnbUf1QVIwwMxptcDm\\\\\\\/tUXSxi4SzEKdN9sfIIiVpdAqm6it6zUWyie0Wdr7ZcmSrprFnHqIzPAkmnX19mAT9ja7M1wGm3qDx\\\\\\\/ih1QkLEBoX8h8exAifQ2mwiVYinERL9GizLOLkHhi0pgeTXJZcdsSCTxDPFtWU9nd1zMpF+dBzpfRyJVxDcf4P\\\\\\\/9RHXBXXIyuDty5FMmjJt09InDyu74dNjFlQ+QDh\\\\\\\/onHd\\\\\\\/anrlSsDjDPorBrCKg2UcY2mck8yopFyKTnh4V0v4Izga7syAWuM73cZNTD8DbUZff6R\\\\\\\/uG3gY9M\\\\\\\/6PT15lXHnATmfRNnxcVZY9403JilURoLVXrSjmyZsVfCkSCHggql2fpL1xMqBe9iWKfx7yGaWpQJBKAk61onU6gK694ZOhxLW9axnxpu2miynKBIrdkuRWc\\\\\\\/16ej6nupaagNX6DS0ND0pqfmrgLinldlBma7hBNyju3WJ5tr7BRedsm1OxJ2P17+ot3JAt5xcgl7yJOgxnhnadbyOcDS\\\\\\\/39KzxEQhD\\\\\\\/342SZJHkIfPC4M7RdlNmtZJKQIQGeWgLlHxxgFMMZEGikwMyrVwXQ0MgTz69\\\\\\\/12ZZyc1kY4v8ZgWA+yPJFHG82yMGFcdkk9p5VXjW9tweXoU99aVigMbebMPqNbQ7EwOIUAhAMOL8YozWMLFsO1r1zBjxEozG2yQ8H15sgJu582SPLeNV82etQX+Qv5gq3sI2IjUYuZ4t3HND2ELIxno5agVXeEIOXPX1fdGouL+k\\\\\\\/6TCJ34eI6lFAbaI1g5+KV1MmBIKThOGkQx51S6mU65DNonRdMhZiZiwqu7peN6xg6FKSOJ4M1V8YAEX4Ork55+6Vffsj5Sb\\\\\\\/YNpKFy6X93wjxHaDWgQVWI5PMhaQWXIfbfOm0a1MfkpRP3aEHaxBCcJq27\\\\\\\/fIS0AR0Vi9CZC+l6ZwlRULmHQzmBDiqiHLy+iNfGZtTk2qgszG1nz6B3R8ASDM3dVMcG+4bVryEsCqlgID4arOYqkAD07djVmFFExpI3L68ZHKSkpr6v2ohNV2Y6KZVvhPdCZHPnjt2S2gUDod+e2a5CJSeTXTPJBILjqEKBl00eeoYBl6JXgoDa1IsevHOjvrdzbGLxgHQDp1D+ThqUkvSzjOXZfVMSNKHVd29TCiU0jnY9nsfelTWXFonMBRL\\\\\\\/wMPn6vzYB3qY5cqie9DdO4UbuesU8JejSYhhnV319H7+QxrHPrFZpEiZud\\\\\\\/mClfxw0uSBADvFFWA3wU+QhMS8wbP2VqPAPy0SXwrCunKmpC10nQz9mgvdmdJ0y58vk2BKuiuXrBnNAzBr+jUryamkwKG5hzuSip\\\\\\\/agmEEdEkVWr5c\\\\\\\/YkUZajRl+drffo4Dinhfho7H5GP+dtoOFWNVeL0UG5SW1QtmflH2HuULr71AvZJM3nn01uXWVQFhZ38nsFq6CjAwfACc2l3YMy9kFsLjiV+J8j70CabJ9cZ9PK5WUHVkF8kVh4awixoS8e72nGiDkqXkpZkbci1u5K3qLeSatGhl2RTvUTABNoZZYYXLaBP2gJi7V5duE0zmnyh9NF4KxlOcj7cwEc0xAF8rNp3lCODfwQcjKt\\\\\\\/6qK3J4LWy2yv9GQrK8w11HZQw4ccGI8zbjaRqmUUaF\\\\\\\/8mWtLE1THQMDxYjuzrFO78eFT9Z8WWRjp7\\\\\\\/8Qw62aaUWw0m6OQginzETHwYu8FBjhjSlQysW8rilTaPznD2oWJtFoa7xTTZgyqDrwK4rbh64oejRvsifFsokjerFBfxphXjDDtkxiElpon+dOJpu7d9hhJ9x3H+9W6HChj8R+Jw4OusTI5AgjUQ12boTvAAuFnVSj40+3JWO0oJ0ROdj2y3l6miNMX4QIUe4GrMOM2vE9DlIb+OL3gI+0UWjspMkREKhGps05820i\\\\\\\/uV9ZovrztnybrLoNJdtVS3r2+DH4kjD6CEDH46T\\\\\\\/ARzDwgsypG8IgRWlasxkDeTEOyhTbPgoYpJabwc4ccA9DL5giK3VKUTpYn+\\\\\\\/Ai+reITPkCdnkcnxMjoHgsuCyI5K46J+IxBuyhUxdr\\\\\\\/yXF2BJSHTiVWF3uWrYBzl452Nc5P3UZQgy1zbQtt74kn2LGxV4AcVmMAFTg0CR84tadmi\\\\\\\/Xt\\\\\\\/ns0wGxNtKPOZA5EJ5dIMZ5Emu\\\\\\\/fQuD8vfYFrcLYzb+zsXaxdXqL0AwQOFMw9mQRo4kJb6pBdlku8R2QLGmPsU2BVDG\\\\\\\/Lv\\\\\\\/kNEC\\\\\\\/G8mszsYqggBP2ZnVGz7q7kyi2ufuQ3bbKMJo6LYgpBcDD75LlZy7iP382Pr6JY2owyvh7e7ZoYCw1FX9JedvVcMTNj2kdDeaS00\\\\\\\/swTYegZz4QJY\\\\\\\/u8uXYKps+8aAesrq2mlWXYC\\\\\\\/yJrJoeXHWlCOTSoB1AxXIMhF+LIsKlWE8jqx+\\\\\\\/kyuCkKg0XakNC\\\\\\\/3NjtfbqYLCJkQsOEqDCYeLkl5q6lWFiGpDqt+TwuvbedXQd310U3rzXb3WNBHbR282oKjHoNdylqWcr5xqCXeKyTNyPc09MVBhOPOtSs2JFYwrJVeglTaCAQGqBodA4\\\\\\\/uvDetfM8YrqPYNWAQxOJkrzoJFi4JzWeEjNjiHheyS7AXALBIi1JpDDUTpEGkcv33EHPRHeqzfvvr1f08waqSlvFVW1UIJd8aRj1en6p\\\\\\\/O4MLJXlktLuPCWbxGqdqwivYORVD01Br6WpQ4CzsTGj5NRI2X8LHe+nQI7y+Hq852M8q1xU9f44TlC5DkJPo\\\\\\\/9CQLi5itsTRmglazQYgEnB\\\\\\\/ILnFxhhiLEm30RsQdvOUq9CMNH1+cTxSwqwXgfiki\\\\\\\/XV7TIVKXYc7Besud5MG+pnjt2C1QcLlXjeU4LLP20WjN6\\\\\\\/7XGSxcB\\\\\\\/vBILBRK3mKW+aD5tUWYOgRg8kvA3yZxgbfRH8FEFXGC89wskozO4oxCqRzXRiGLI\\\\\\\/cMLwMChwYYFNK7\\\\\\\/JTcP7j8IUGislxh13M51s91SNNxcDfi8IXvdA\\\\\\\/CYPKaeImKWe\\\\\\\/inKlzM6RmoXS694hl+YacwPmU\\\\\\\/RkqsKyNAQc1OCX5UGAS01DeBhEExKVUnr6jucK1wUh3VCmXSqOFY159YyeUINXCho7CanusQXylpRZpPN03tmW8BYY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022dnoqWC2W5r4Tm9wJzKwg7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f9c72f72faf6529f3f021fd9daa5c9294ca832822fad093545be30b3de8290a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yk2k2jhB2e+BNG2QxMZCc51RzvXLN\\\\\\\/BbZSVJ0qC+2kKXGhsB8+\\\\\\\/tOjjoPebcV9eylxoFRPjjo3Ha0vMt9Z91Oh9TmGcXZ2BV\\\\\\\/f8RNr5IY41pSJ6omFrwh3uESXNZ2ZAqca0IGNQ8Zt\\\\\\\/0s8rkQ6P6JJ5TLXX+g7cAonzF4VFhGvOCMOvfFUHL26eFLVrICuTH3YJyPBVbVeXAoHy1wikjx4+RJpzka6lLt2WXzuh5ipnDYCsLnp0bFjwtgsjbbsTbFQ9FM9jpIUU1tBLLVOpeI6RXFyu868rFfVeJYm9B0rYWS7YOi7H774yKQlVn1cDR+5wD4K88gr0sDAjnC5ANqPEXZdA1n3vlOGkRCqBJ9sr2SVSCOXjaACMsJ4nCQGsiEhxHKJW+dYsIKyMQFQC\\\\\\\/dpq1jYEZ3TlofdqR8B3dMc72F0HkVzD+HSzOwBf1sJJKiUuGx8sQaP0IxPuZS+AHkrLwTLwNNCscu+JBUjZvd4xFs8sDqxcINO7PjJ7gh\\\\\\\/\\\\\\\/jGLKBVwv8H\\\\\\\/gKoHd8oByw9+gyZP6ZkgM4Hh6NOXzsH1PNqoDEg+ypJ5h6zjT8aNHMX+9H0wiuuCZyZslo8RthPNUqATnpLO8i\\\\\\\/8+YBUhxvtfiFU+YfH902tgxMYfdcZ4P5g20L1MrUIwmJtK\\\\\\\/A7wPAv+hkzGtfNCvQqOVzIJ+8GsWQlPgcNEDsSlSviYaY8HJJs87DmjTQIvM0RXu5TuX8HymjDdXAAmeb9tsyUxBkL5lWQykxYyTrC9tbYWEHk4l+x08jmgI2guMwJ6kqFk1MjbFMna8QPie2jY6lfOAkCYy7wEH\\\\\\\/n4A6QcFQlGOqlnLL8KgDLqCw\\\\\\\/NhyisYLSKruqV214jxX1IjFTKqEXJup3UfPZciXhSrxo2PoHPEGMPeCUm3U2zX55JRgXc\\\\\\\/0rnV0GsbtuNDBOYCcIDu\\\\\\\/vo\\\\\\\/bpREpTJ7s9KyAzWKObLjeakAQHG9rw3h0HwWx\\\\\\\/S7CMPdM0qx4dvoRfqJY9U4L\\\\\\\/TAcQRCktUx5bp1O9Ya8KsK5\\\\\\\/lcFM+shf8upCLuJR045qUViXQAHmB4wLcAQfPqneR0HHhos3+KvUfnLi6lXQ2HLuhGq3txLnR+Jse\\\\\\\/N+h7vOWHtof8IZDzRWUe876ogmuxn\\\\\\\/B0zMoq7czAklg+OYoPexf\\\\\\\/iwbgAvY2Eka46bbXkbZBZWbK3qyS4EFA4a86jOizrJmFUjkyZ5HwNmSdfa0Zaa9gvXwbPxyg3PxsF+7wpOeOB\\\\\\\/D2BhIWzdzrBvr4VocaEXsFqeaFIyxtaoKhziFAL3VfQbakL\\\\\\\/7CeWMbliWAUKi+ufL4aMU+wHEvKCuO+mr1HIVo\\\\\\\/EsR7l6reIeHLdU+mckEdw\\\\\\\/aJQY4XzvXBBqCbRvNheaKMzkURmOBqHp02mMAcihxE1EzpPm3JYXkUIs0+ijHB+m5seJdwivDaLOAAvejhPPOIydmktWo54RiAIneG\\\\\\\/cksf33Dc1TqvfQVQfDsc2uiecfwroOML+nzI1LQs29L3YTlGAPJMPMm1qM+5khWQHdnK0GVqafTAcHZzxsdiBZTHnkDyQm9LPkqKsw\\\\\\\/C4oQnxeUIp6A7qkr+kStORu4q63h+fkvqcrKGwtOjBJk\\\\\\\/BBuD\\\\\\\/zeRQaHNP6nV9uCrmQjBxJEmvaLUKU9rNpe\\\\\\\/SyR7rXC8iJeyeSkc+5RN\\\\\\\/RkeUb95j76Fupn6RrGdu5nRfD5aUjpMkEgwS+AFgrQkJyJ3gQyaGCFBOQmi0rsnhvVdejAxlBaWm18A2vgPjCynO+yfQK3\\\\\\\/6+4IzqGW84Z+xmZANDoTa7ousTcaVJ0kBxmHBFeti+t2a\\\\\\\/UNBq6TP3Hz7K+yEX4ggoOVxUisQQ\\\\\\\/3NnwLo7qe1Tz8aMrwgoGjrI6C7s8thfoKT\\\\\\\/sk6A6pz23z5+GX4ghZbgkvXvdQXPiQr+Hup+A9OTMbkxIchcveKDxCenltNcXY8EeQHqV7QDRU6SvIteo9zS4\\\\\\\/oD5JIC46yt7gRX42zrwmPqmGoa3E01rcH7tVuLkIZuJFqKpWDOhVQkH153VJErwByriT4QET8XpHwOolYmJfEMw2Hpqj0aO1Yy9GJa+q2ntTeFRf1UClje1FPA5hOxppRCS+KF0bqP8AcAaaBoSAu7yYLziNEehUPwlgIjgOZua3AYCxj\\\\\\\/34lH6g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+U2BX2CqBRTdLw52QUClBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022df43d7ac646d5ded4391f6bd1a7ba9ce1fe7526dd0a993f67d37877ccc70b082\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460384 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1777" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CkZYIlzUbxoVVBANcJW2Aozh_R-zHnLV20U9QsF62N0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1777" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1vnylUNFAL6pdhqrtGhwxo2oYfIsyE0jE_32DYaw0Oo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1767" title="Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1787" title="Site-Directed Genome Knockout in Chicken Cell Line and Embryos Can Use CRISPR/Cas Gene Editing Technology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460384" id="node460384" data-pisa="ggg;6/6/1777" data-pisa-master="ggg;g3.116.028241" data-apath="/ggg/6/6/1777.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Comprehensive Toolbox for Genome Editing in Cultured <em>Drosophila melanogaster</em> Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Stefan Kunzelmann</span>, <span class="highwire-citation-author" data-delta="1">Romy Böttcher</span>, <span class="highwire-citation-author" data-delta="2">Ines Schmidts</span> and <span class="highwire-citation-author" data-delta="3">Klaus Förstemann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1777-1785;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028241 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460384" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stefan Kunzelmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BKunzelmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kunzelmann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BKunzelmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Romy Böttcher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romy%2BB%C3%B6ttcher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%B6ttcher%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomy%2BB%25C3%25B6ttcher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ines Schmidts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BSchmidts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidts%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BSchmidts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Klaus Förstemann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klaus%2BF%C3%B6rstemann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=F%C3%B6rstemann%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKlaus%2BF%25C3%25B6rstemann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:foerstemann@lmb.uni-muenchen.de" class="" data-icon-position="" data-hide-link-title="0">foerstemann@lmb.uni-muenchen.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/6/1777" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460384" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460384/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1777.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460384" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460384/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1777.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460384" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460384/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1777.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460384" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460384/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/6/1777.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460384" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460384/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Custom genome editing has become an essential element of molecular biology. In particular, the generation of fusion constructs with epitope tags or fluorescent proteins at the genomic locus facilitates the analysis of protein expression, localization, and interaction partners at physiologic levels. Following up on our initial publication, we now describe a considerably simplified, more efficient, and readily scalable experimental workflow for PCR-based genome editing in cultured <em>Drosophila melanogaster</em> cells. Our analysis at the <em>act5C</em> locus suggests that PCR-based homology arms of 60 bp are sufficient to reach targeting efficiencies of up to 80% after selection; extension to 80 bp (PCR) or 500 bp (targeting vector) did not further improve the yield. We have expanded our targeting system to N-terminal epitope tags; this also allows the generation of cell populations with heterologous expression control of the tagged locus via the copper-inducible <em>mtnDE</em> promoter. We present detailed, quantitative data on editing efficiencies for several genomic loci that may serve as positive controls or benchmarks in other laboratories. While our first PCR-based editing approach offered only blasticidin-resistance for selection, we now introduce puromycin-resistance as a second, independent selection marker; it is thus possible to edit two loci (<em>e.g.</em>, for coimmunoprecipitation) without marker removal. Finally, we describe a modified FLP recombinase expression plasmid that improves the efficiency of marker cassette FLP-out. In summary, our technique and reagents enable a flexible, robust, and cloning-free genome editing approach that can be parallelized for scale-up.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cas9-crispr" class="hw-term hw-article-keyword hw-article-keyword-cas9-crispr" rel="nofollow">cas9-CRISPR</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-melanogaster" class="hw-term hw-article-keyword hw-article-keyword-drosophila-melanogaster" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila melanogaster</em></a></li><li class="kwd"><a href="/keyword/homology-directed-editing" class="hw-term hw-article-keyword hw-article-keyword-homology-directed-editing" rel="nofollow">homology-directed editing</a></li><li class="kwd"><a href="/keyword/epitope-tag" class="hw-term hw-article-keyword hw-article-keyword-epitope-tag" rel="nofollow">epitope tag</a></li><li class="kwd"><a href="/keyword/genome-functionalization" class="hw-term hw-article-keyword hw-article-keyword-genome-functionalization" rel="nofollow">genome functionalization</a></li></ul><p id="p-3">Genome editing with the help of the CRISPR/<em>cas</em> system has become an indispensable part of our toolbox for molecular biology. The CRISPR “revolution” has shifted our attention from cloning work in the context of plasmids or bacterial artificial chromosomes to the modification of genes in their chromosomal context. The creation of a defined DNA double-strand break can either be followed by erroneous repair (end-joining activities), leading to targeted mutagenesis, or by homology-directed repair. The latter offers the possibility to introduce an experimentally provided, custom-modified homologous recombination (HR) donor into a desired locus (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Doudna and Charpentier 2014</a>). Various methods that combine programmable cleavage of DNA by CRISPR/<em>cas9</em> with introduced HR donors in cell culture have been described (<em>e.g.</em>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bassett <em>et al.</em> 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Byrne <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fetter <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fu <em>et al.</em> 2014a</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Li <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Wyvekens <em>et al.</em> 2015</a>). These strategies can be broadly grouped according to the particular type of HR donor material employed: Cloned homology arms, single-stranded synthetic oligonucleotides, or PCR-products with flanking homology regions. We have previously presented a protocol for cultured <em>Drosophila</em> <em>melanogaster</em> cells that employs PCR to generate both an expression cassette for the <em>cas9</em>-programming sgRNA and HR donors for selectable genome modification. In this publication, we describe several technical improvements to our original protocol, two experimental extensions (N-terminal epitope tags and an independent selection marker), and an improved FLP recombinase expression vector. In addition, we provided detailed protocols for use in the laboratory as Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS2.pdf">File S2</a>.</p><p id="p-4">We present the <em>act5C</em>-gene as an easily detectable and quantifiable positive control (via a C-terminal GFP-tag, see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) that may help other researchers set up the technology in their own labs. With an inverted microscope equipped with fluorescence illumination and a GFP filter set, the first Act5C-GFP positive cells can already be seen about 3 d after the initial transfection. Their enrichment during the selection procedure can also be observed. This model gene has allowed us to optimize a series of parameters (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) in our approach to make the method highly efficient yet cost-effective.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Optimization of technical aspects for the genomic tagging approach. (A) The model locus act5C can be efficiently tagged with e.g., GFP. This allows for easy quantification by fluorescence microscopy or flow cytometry. (B) PCR is used to now directly fuse the optimized sgRNA sequence with the U6 promoter; this has allowed a simpler PCR protocol and eliminates T7 in vitro transcription. For completeness, the HR donor PCR is also depicted schematically on the right. (C) Commercial PCR buffers contain detergents that interfere with transfection (average &#xB1; SD, n = 3 biological replicates). If PCR is performed in detergent-free buffer, no purification step is necessary. Both the U6-sgRNA expression fragment and the HR donor were amplified in the respective buffer system. The experiment was performed with knockdown of lig4 only. (D) The experimental workflow could be simplified with direct U6 fusion and detergent-free buffer. Essentially, it suffices to perform two PCR amplifications, then mix and transfect the products. If the recipient cells do not express cas9, a cas9-encoding plasmid can be transfected along with the PCR products. GFP, green fluorescent protein; HR, homologous recombination; PCR, polymerase chain reaction; SD, standard deviation; sgRNA, single guide RNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901039782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Optimization of technical aspects for the genomic tagging approach. (A) The model locus &lt;em&gt;act5C&lt;/em&gt; can be efficiently tagged with &lt;em&gt;e.g.&lt;/em&gt;, GFP. This allows for easy quantification by fluorescence microscopy or flow cytometry. (B) PCR is used to now directly fuse the optimized sgRNA sequence with the U6 promoter; this has allowed a simpler PCR protocol and eliminates T7 &lt;em&gt;in vitro&lt;/em&gt; transcription. For completeness, the HR donor PCR is also depicted schematically on the right. (C) Commercial PCR buffers contain detergents that interfere with transfection (average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 3 biological replicates). If PCR is performed in detergent-free buffer, no purification step is necessary. Both the U6-sgRNA expression fragment and the HR donor were amplified in the respective buffer system. The experiment was performed with knockdown of &lt;em&gt;lig4&lt;/em&gt; only. (D) The experimental workflow could be simplified with direct U6 fusion and detergent-free buffer. Essentially, it suffices to perform two PCR amplifications, then mix and transfect the products. If the recipient cells do not express &lt;em&gt;cas9&lt;/em&gt;, a &lt;em&gt;cas9&lt;/em&gt;-encoding plasmid can be transfected along with the PCR products. GFP, green fluorescent protein; HR, homologous recombination; PCR, polymerase chain reaction; SD, standard deviation; sgRNA, single guide RNA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1777/F1.medium.gif" width="422" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/6/1777/F1.medium.gif" width="422" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468521" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Optimization of technical aspects for the genomic tagging approach. (A) The model locus <em>act5C</em> can be efficiently tagged with <em>e.g.</em>, GFP. This allows for easy quantification by fluorescence microscopy or flow cytometry. (B) PCR is used to now directly fuse the optimized sgRNA sequence with the U6 promoter; this has allowed a simpler PCR protocol and eliminates T7 <em>in vitro</em> transcription. For completeness, the HR donor PCR is also depicted schematically on the right. (C) Commercial PCR buffers contain detergents that interfere with transfection (average ± SD, <em>n</em> = 3 biological replicates). If PCR is performed in detergent-free buffer, no purification step is necessary. Both the U6-sgRNA expression fragment and the HR donor were amplified in the respective buffer system. The experiment was performed with knockdown of <em>lig4</em> only. (D) The experimental workflow could be simplified with direct U6 fusion and detergent-free buffer. Essentially, it suffices to perform two PCR amplifications, then mix and transfect the products. If the recipient cells do not express <em>cas9</em>, a <em>cas9</em>-encoding plasmid can be transfected along with the PCR products. GFP, green fluorescent protein; HR, homologous recombination; PCR, polymerase chain reaction; SD, standard deviation; sgRNA, single guide RNA.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In vivo optimization of the tagging approach using the Act5C model locus and flow cytometry-based quantification of the Act5C-GFP tagging success. (A) The optimal length of the target site complementary region of the sgRNA appears to be 18&#x2013;19 nt (not counting the first G provided by the U6-promoter, see detailed protocol); values depicted are the average &#xB1; SD, n = 4 biological replicates, p-values: t-test. This is consistent with the observations made by others (see text for references). (B) We extended our efforts to provide quantitative information about the tagging success rates to five other loci. Shown are the GFP-positive cells detected one week after transfection, upon one round of Blasticidin-selection and after the second round of Blasticidin-selection; depicted are the average values &#xB1; SD, n = 4 biological replicates. The expression levels of the tagged target loci can be estimated according to the mean intensity of the GFP-signal among the positive cell population (arb. units, see also Figure S1): Act5C-GFP = 293 &#xB1; 42, H2Av-GFP = 47 &#xB1; 2, Nop56-GFP =19 &#xB1; 2, Sec61-GFP = 18 &#xB1; 2, ATPsyn-GFP = 16 &#xB1; 1, Nup133-GFP = 5 &#xB1; 0.3; the corresponding sgRNA sequences are indicated above the respective bars. (C) If an optimally placed GG dinucleotide PAM (i.e., a CRISPR target site that will be destroyed upon integration of the HR donor) cannot be found, the cas9 cleavage would also occur within the affected homology arm. A single point mutation is sufficient to recover much of the tagging efficiency (left panel, average &#xB1; SD, n = 3 biological replicates, p-values: t-test). If cleavage occurs at the optimal position, one or two point mutations do not alter tagging efficiency (right panel, average &#xB1; SD, n = 3 biological replicates, p-values: t-test). Note that the values shown represent the initial targeting efficiency. Efficiencies were normalized to the average number of GFP-positive cells obtained when cleavage occurred at the standard site and the homology region did not contain any point mutations (right panel, &#x201C;wt&#x201D; bar). The error bars thus reflect both the biological variation for Act5C-GFP tagging and the technical variation of transfection efficiency between replicates. This experiment was performed without depletion of lig4 and mus308 because repair via end-joining activities may be required for mutagenesis after cleavage by cas9. (D) Left panel: To determine the optimal length of the homology region contained in the PCR primers, we performed the Act5C-GFP tagging experiment with 40, 60, and 80 bp homology regions on either side. While 60 bp of homology length did enhance the efficiency relative to 40 bp, further extension to 80 bp did not increase the efficiency (average &#xB1; SD, n = 4 biological replicates, p-values: t-test). We thus recommend using 60 bp of target homology on either side of the HR donor PCR product. Right panel: We compared PCR-based HR donors with longer homology regions contained in a targeting vector (obtained by modification of the PCR template vector). Note that for this panel, a concomitant lig4/mus308 knockdown was performed (average &#xB1; SD, n = 3 biological replicates, p-values: t-test; see also Figure S2). (E) Concomitant knockdown of lig4 and mus308 improves the recovery of correctly genome-modified cells. The control knockdown was dsRNA directed against Renilla reniformis luciferase. The efficiency in this experiment was 82 &#xB1; 3% of Act5C-GFP-positive cells for the lig4/mus308 knockdown (average &#xB1; SD, n = 3 biological replicates, p-values: t-test). See also the left and right panels of Figure 2D. CRISPR, Clustered regularly-interspaced short palindromic repeat; GFP, green fluorescent protein; HR, homologous recombination; PAM, protospacer adjacent motif; PCR, polymerase chain reaction; RNAi, RNA interference; SD, standard deviation; sgRNA, single guide RNA; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901039782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;In vivo&lt;/em&gt; optimization of the tagging approach using the Act5C model locus and flow cytometry-based quantification of the Act5C-GFP tagging success. (A) The optimal length of the target site complementary region of the sgRNA appears to be 18&#x2013;19 nt (not counting the first G provided by the U6-promoter, see detailed protocol); values depicted are the average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 4 biological replicates, &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test. This is consistent with the observations made by others (see text for references). (B) We extended our efforts to provide quantitative information about the tagging success rates to five other loci. Shown are the GFP-positive cells detected one week after transfection, upon one round of Blasticidin-selection and after the second round of Blasticidin-selection; depicted are the average values &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 4 biological replicates. The expression levels of the tagged target loci can be estimated according to the mean intensity of the GFP-signal among the positive cell population (arb. units, see also Figure S1): Act5C-GFP = 293 &#xB1; 42, H2Av-GFP = 47 &#xB1; 2, Nop56-GFP =19 &#xB1; 2, Sec61-GFP = 18 &#xB1; 2, ATPsyn-GFP = 16 &#xB1; 1, Nup133-GFP = 5 &#xB1; 0.3; the corresponding sgRNA sequences are indicated above the respective bars. (C) If an optimally placed GG dinucleotide PAM (&lt;em&gt;i.e.&lt;/em&gt;, a CRISPR target site that will be destroyed upon integration of the HR donor) cannot be found, the &lt;em&gt;cas9&lt;/em&gt; cleavage would also occur within the affected homology arm. A single point mutation is sufficient to recover much of the tagging efficiency (left panel, average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 3 biological replicates, &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test). If cleavage occurs at the optimal position, one or two point mutations do not alter tagging efficiency (right panel, average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 3 biological replicates, &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test). Note that the values shown represent the initial targeting efficiency. Efficiencies were normalized to the average number of GFP-positive cells obtained when cleavage occurred at the standard site and the homology region did not contain any point mutations (right panel, &#x201C;wt&#x201D; bar). The error bars thus reflect both the biological variation for Act5C-GFP tagging and the technical variation of transfection efficiency between replicates. This experiment was performed without depletion of &lt;em&gt;lig4&lt;/em&gt; and &lt;em&gt;mus308&lt;/em&gt; because repair via end-joining activities may be required for mutagenesis after cleavage by &lt;em&gt;cas9&lt;/em&gt;. (D) Left panel: To determine the optimal length of the homology region contained in the PCR primers, we performed the Act5C-GFP tagging experiment with 40, 60, and 80 bp homology regions on either side. While 60 bp of homology length did enhance the efficiency relative to 40 bp, further extension to 80 bp did not increase the efficiency (average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 4 biological replicates, &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test). We thus recommend using 60 bp of target homology on either side of the HR donor PCR product. Right panel: We compared PCR-based HR donors with longer homology regions contained in a targeting vector (obtained by modification of the PCR template vector). Note that for this panel, a concomitant &lt;em&gt;lig4/mus308&lt;/em&gt; knockdown was performed (average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 3 biological replicates, &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test; see also Figure S2). (E) Concomitant knockdown of &lt;em&gt;lig4&lt;/em&gt; and &lt;em&gt;mus308&lt;/em&gt; improves the recovery of correctly genome-modified cells. The control knockdown was dsRNA directed against &lt;em&gt;Renilla&lt;/em&gt; &lt;em&gt;reniformis&lt;/em&gt; luciferase. The efficiency in this experiment was 82 &#xB1; 3% of Act5C-GFP-positive cells for the &lt;em&gt;lig4/mus308&lt;/em&gt; knockdown (average &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 3 biological replicates, &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test). See also the left and right panels of Figure 2D. CRISPR, Clustered regularly-interspaced short palindromic repeat; GFP, green fluorescent protein; HR, homologous recombination; PAM, protospacer adjacent motif; PCR, polymerase chain reaction; RNAi, RNA interference; SD, standard deviation; sgRNA, single guide RNA; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1777/F2.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/6/1777/F2.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468369" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-6" class="first-child"><em>In vivo</em> optimization of the tagging approach using the Act5C model locus and flow cytometry-based quantification of the Act5C-GFP tagging success. (A) The optimal length of the target site complementary region of the sgRNA appears to be 18–19 nt (not counting the first G provided by the U6-promoter, see detailed protocol); values depicted are the average ± SD, <em>n</em> = 4 biological replicates, <em>p</em>-values: <em>t</em>-test. This is consistent with the observations made by others (see text for references). (B) We extended our efforts to provide quantitative information about the tagging success rates to five other loci. Shown are the GFP-positive cells detected one week after transfection, upon one round of Blasticidin-selection and after the second round of Blasticidin-selection; depicted are the average values ± SD, <em>n</em> = 4 biological replicates. The expression levels of the tagged target loci can be estimated according to the mean intensity of the GFP-signal among the positive cell population (arb. units, see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FigureS1.pdf">Figure S1</a>): Act5C-GFP = 293 ± 42, H2Av-GFP = 47 ± 2, Nop56-GFP =19 ± 2, Sec61-GFP = 18 ± 2, ATPsyn-GFP = 16 ± 1, Nup133-GFP = 5 ± 0.3; the corresponding sgRNA sequences are indicated above the respective bars. (C) If an optimally placed GG dinucleotide PAM (<em>i.e.</em>, a CRISPR target site that will be destroyed upon integration of the HR donor) cannot be found, the <em>cas9</em> cleavage would also occur within the affected homology arm. A single point mutation is sufficient to recover much of the tagging efficiency (left panel, average ± SD, <em>n</em> = 3 biological replicates, <em>p</em>-values: <em>t</em>-test). If cleavage occurs at the optimal position, one or two point mutations do not alter tagging efficiency (right panel, average ± SD, <em>n</em> = 3 biological replicates, <em>p</em>-values: <em>t</em>-test). Note that the values shown represent the initial targeting efficiency. Efficiencies were normalized to the average number of GFP-positive cells obtained when cleavage occurred at the standard site and the homology region did not contain any point mutations (right panel, “wt” bar). The error bars thus reflect both the biological variation for Act5C-GFP tagging and the technical variation of transfection efficiency between replicates. This experiment was performed without depletion of <em>lig4</em> and <em>mus308</em> because repair via end-joining activities may be required for mutagenesis after cleavage by <em>cas9</em>. (D) Left panel: To determine the optimal length of the homology region contained in the PCR primers, we performed the Act5C-GFP tagging experiment with 40, 60, and 80 bp homology regions on either side. While 60 bp of homology length did enhance the efficiency relative to 40 bp, further extension to 80 bp did not increase the efficiency (average ± SD, <em>n</em> = 4 biological replicates, <em>p</em>-values: <em>t</em>-test). We thus recommend using 60 bp of target homology on either side of the HR donor PCR product. Right panel: We compared PCR-based HR donors with longer homology regions contained in a targeting vector (obtained by modification of the PCR template vector). Note that for this panel, a concomitant <em>lig4/mus308</em> knockdown was performed (average ± SD, <em>n</em> = 3 biological replicates, <em>p</em>-values: <em>t</em>-test; see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FigureS2.pdf">Figure S2</a>). (E) Concomitant knockdown of <em>lig4</em> and <em>mus308</em> improves the recovery of correctly genome-modified cells. The control knockdown was dsRNA directed against <em>Renilla</em> <em>reniformis</em> luciferase. The efficiency in this experiment was 82 ± 3% of Act5C-GFP-positive cells for the <em>lig4/mus308</em> knockdown (average ± SD, <em>n</em> = 3 biological replicates, <em>p</em>-values: <em>t</em>-test). See also the left and right panels of <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>. CRISPR, Clustered regularly-interspaced short palindromic repeat; GFP, green fluorescent protein; HR, homologous recombination; PAM, protospacer adjacent motif; PCR, polymerase chain reaction; RNAi, RNA interference; SD, standard deviation; sgRNA, single guide RNA; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Although cell culture is clearly a reduced system compared to <em>in vivo</em> work, it can complement <em>Drosophila</em> experimental strategies with respect to cell-autonomous phenomena. In fact, <em>Drosophila</em> S2-cells can be cultured essentially without dedicated equipment; while sterile working techniques are of course aided significantly by laminar flow hoods, the cells can also be handled with proper care on a clean laboratory bench. Since the popular Schneider’s medium contains pH-buffering substances, cell culture incubators with CO<sub>2</sub>-control are not required. If necessary, the cells can even be grown in a 25° incubator together with fly stocks (<em>e.g.</em>, inside a plastic box). The technique described in this manuscript is thus inexpensive to set up (only requiring standard molecular biology equipment) and can be employed in essentially any fly lab. Our stable <em>cas9</em>-expressing cell line is a convenient simplification, but cotransfection of a <em>cas9</em> expression plasmid is also possible (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>). All plasmids described in this manuscript have been deposited at Addgene (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> for a full list of plasmid reagents).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468367/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468367/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Currently available plasmids for the genomic tagging procedure</span><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Molecular biology methods</h3><p id="p-9">All PCR primer sequences are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/TableS1.pdf">Table S1</a>. The Act5C-GFP targeting vector was constructed by inserting PCR-amplified homology regions of 500 bp (upstream) and 420 bp (downstream) into the vector pMH3 using the <em>Not</em>I/<em>Xho</em>I (upstream) and <em>Cla</em>I/<em>Hin</em>dIII (downstream) restriction sites. Note that, for the downstream homology region, an internal <em>Hin</em>dIII site in the homologous sequence served as a cloning site. The improved FLP-recombinase expression plasmid was constructed by amplifying the FLP sequence using primers Bam_NLS_FLPe_sense and Not_FLP_as, then digesting the PCR product with <em>Bam</em>HI/<em>Not</em>I followed by ligation into correspondingly cleaved pMH5. Candidate clones were sequenced and pKF295 was retained as it carries the desired P(2)-&gt;S mutation and the NLS. In addition, the corresponding FLP protein carries a I(377)-&gt;L substitution likely due to PCR mutagenesis, which we estimate to be without major consequences. Plasmids for N-terminal HR donor templates were generated through a combination of oligonucleotide synthesis (details available on request), PCR, and restriction enzyme cloning. All oligonucleotides for this manuscript were ordered from Eurofins Genomics (Ebersberg, Germany). The <em>copia</em>-puromycin resistance cassette was ordered as a synthetic gBlock fragment (IDT, Coralville).</p></div><div id="sec-3" class="subsection"><h3>Cell culture</h3><p id="p-10">Culture of <em>Drosophila</em> cells in Schneider’s medium (Bio &amp; Sell, Nürnberg, Germany) supplemented with 10% FBS and penicillin/streptomycin (Life Technologies) and transfection conditions were as previously described (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Shah and Forstemann 2008</a>). Details on the transfection of PCR products are given in the detailed protocol provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS2.pdf">File S2</a>. GFP expression was quantified using a Becton-Dickinson FACScalibur flow cytometer; data analysis was performed with Flowing Software 2.0 (<a href="http://www.flowingsoftware.com">http://www.flowingsoftware.com</a>) using a two-dimensional plot (SSC and GFP fluorescence) to separate nonfluorescent from GFP-positive cells. Fluorescence microscopy images were acquired on a Leica DM IRE2 confocal microscope. Induction of the <em>mtnDE</em> promoter was achieved by adding CuSO<sub>4</sub> from a sterile 100 mM stock solution directly into the culture medium (final concentrations between 30 µM and 1 mM). The effective concentrations are likely somewhat lower since amino group-containing components from the medium form complexes with copper ions (visible as a change to a slight blue color with increasing concentrations). The conditions for induction should therefore be verified if serum-free culture medium is employed.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-11">This article is accompanied by detailed experimental protocols provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS2.pdf">File S2</a>. We maintain these protocols and the potentially updated versions can be downloaded from our web-page: <a href="http://www.foerstemann.genzentrum.lmu.de/protocols">http://www.foerstemann.genzentrum.lmu.de/protocols</a>.</p><p id="p-12">The described plasmids are available at Addgene (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for accession numbers) and we will provide the S2 cell line with stable <em>cas9</em> expression upon request. This cell line is based on our laboratory stock of S2 cells. The entire procedure described here can easily be transferred to other cell lines or engineered <em>cas9</em> protein variants by cotransfection of a <em>cas9</em> expression plasmid (<em>e.g.</em>, pRB14). This strategy is slightly less convenient but, with PCR product-based sgRNA delivery, reaches nearly identical efficiencies (see Figure 5B in <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>).</p><p id="p-13">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><div id="sec-6" class="subsection"><h3>Act5C-GFP as a positive control locus and model gene for optimization</h3><p id="p-14">Since our original publication, we have strived for an improved and simplified genome editing workflow. An important tool to assess beneficial changes in the protocol is a model locus that allows for single cell-based detection of the desired homology-directed modification even before antibiotic selection. We have made extensive use of the C-terminal addition of GFP to the <em>act5C</em> gene for this purpose. The GFP modification appears to be well tolerated in cells and can be scored by fluorescence microscopy or flow cytometry as early as 3 d after the transfection (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). We also found this to be a very helpful positive control to establish the workflow and to routinely confirm that cell growth and transfection conditions were appropriate.</p></div><div id="sec-7" class="subsection"><h3>Improved sgRNA primer design and direct U6 promoter-sgRNA fusion</h3><p id="p-15">The use of DNA-based sgRNA expression constructs, rather than <em>in vitro</em> transcribed RNA, has greatly simplified the experiment and work in our laboratory now relies exclusively on this approach. In our first publication (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>), we had used the T7-promoter present on the <em>in vitro</em> transcription template to generate a U6-promoter fusion product by overlap extension PCR. Since we essentially abandoned <em>in vitro</em> transcription of the sgRNA, we designed a new primer template that results in a direct fusion of the sgRNA sequence to the U6:96Ac promoter contained on plasmid pRB17 (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>). Furthermore, we exchanged the sgRNA scaffold oligonucleotide for an optimized version developed by the Weissman lab (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2013</a>). Generation of an sgRNA template by PCR is now a highly reliable, single-step PCR (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Since the gene-specific sequence in the oligonucleotide that bridges the U6-promoter with the sgRNA scaffold does not participate in any annealing step during PCR, multiple different sgRNA amplifications can be performed in parallel under identical conditions. We recommend including a control amplification without addition of the “programming” gene-specific oligonucleotide; this will reveal if cross-contaminations have occurred.</p></div><div id="sec-8" class="subsection"><h3>Use of detergent-free PCR buffer eliminates the need for purification</h3><p id="p-16">PCR products generated in commercial reaction buffers need to be purified in order to reach acceptable transfection efficiencies. We hypothesized that the detergent contained in the commercial buffers might be the reason. Thus, we performed our PCR in self-made reaction buffer that lacks detergent and transfected the products directly, <em>i.e.</em>, without any purification, enrichment, or precipitation step. As shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>, the success rate was slightly higher than the one obtained with column-purified material. Perhaps more importantly, this improvement was reached with a greatly simplified experimental workflow (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>).</p></div><div id="sec-9" class="subsection"><h3>An optimal sgRNA length of 18–19 nucleotides</h3><p id="p-17">We previously demonstrated that sgRNA length can be varied (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>), and others (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fu <em>et al.</em> 2014b</a>) have demonstrated that truncated sgRNAs provide improved efficacy and specificity for genome editing. Because there is only a narrow window for optimal DNA cleavage sites in our genomic tagging strategy, few (if any) sgRNA alternatives exist and strategies that improve <em>cas9</em> cleavage specificity and/or efficiency are thus beneficial. We optimized the length of the target-matching region in the sgRNA and tested this using our Act5C-GFP system. Consistent with previous observations (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fu <em>et al.</em> 2014b</a>), we found that a length of 18–19 nucleotides (nt) appears to give optimal efficiency. We did not test whether cleavage specificity was concomitantly improved (as has been reported by others).</p><p id="p-18">It is conceivable, if not obvious, that genome editing efficiencies may vary among targeted sites. We extended our flow cytometry-based, quantitative analysis of C-terminal GFP-tagging efficiency from Act5C (<a href="http://flybase.org/reports/FBgn0000042.html">CG4027</a>, X Chr.) to five additional loci: Nup-133 (<a href="http://flybase.org/reports/FBgn0039004.html">CG6958</a>, third Chr.), Sec-61 α (<a href="http://flybase.org/reports/FBgn0086357.html">CG9539</a>, second Chr.), ATP synthase β (<a href="http://flybase.org/reports/FBgn0010217.html">CG11154</a>, fourth Chr.), Nop56 (<a href="http://flybase.org/reports/FBgn0038964.html">CG13849</a>, third Chr.), and H2Av (<a href="http://flybase.org/reports/FBgn0001197.html">CG5499</a>, third Chr.). These loci were chosen because of their specific localization pattern, allowing for microscopic verification (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FigureS1.pdf">Figure S1</a>), and because their expression levels allowed us to quantify the tagging success via flow cytometry. We did not observe any anomalies in cell morphology or grossly altered proliferation rates caused by the fusion proteins. We could obtain efficiencies between 55–89% after two rounds of Blasticidin selection (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). This demonstrates that high targeting success rates are not limited to the Act5C locus. However, the number of loci we now present is still limited and their selection was not random. We suggest considering the observed success rates of 55–89% as the range that can be expected for modification of “permissive” loci. In addition to potential chromatin structure-dependent hindrances for DNA cleavage and repair, appending an epitope tag to cellular proteins may interfere with their function. Our approach modifies the genomic allele(s); dominant-negative or haploinsufficiency effects may thus also reduce the yield of cells if the desired modification leads to protein malfunction.</p><p id="p-19">Are there any particular sequence features that determine the efficiency of sgRNA cleavage? This is currently a matter of intense debate and several publications have addressed this issue with diverging conclusions (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Housden <em>et al.</em> 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Moreno-Mateos <em>et al.</em> 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ren <em>et al.</em> 2014</a>). We did not find any particular sequence feature that most or all of the six sgRNAs we tested in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> have in common. This is not too surprising, given the small number of loci, but it suggests that sequence optimization of the sgRNA may only have limited benefit. For the purpose of introducing an epitope-tag at the N- or C-terminus of a protein, the position where <em>cas9</em>-dependent cleavage must occur is rather narrowly defined; cleavage further away from the start- or stop-codon rapidly reduces overall efficiency (see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). Therefore, if sequence-based sgRNA selection rules are in clear conflict with the need to cleave close to the desired integration site, we suggest choosing the sgRNA sequence in favor of the best cleavage position. Obviously, repetitive sequences that are present in several instances throughout the genome should nonetheless be avoided.</p></div><div id="sec-10" class="subsection"><h3>Single point mutations suffice to prevent cleavage at target sites within homology regions</h3><p id="p-20">Sometimes the ideal situation, where the <em>cas9</em> target site is disrupted upon insertion of the HR donor, cannot be reached because an appropriately placed GG PAM is absent. In this case, the <em>cas</em>9 cleavage site will be present, both in the genomic sequence close to the desired integration site and also in one of the flanking homology regions in the HR donor. To prevent cleavage of the latter prior to integration, or subsequent mutagenesis of the edited locus, a silent point mutation should be introduced in the HR donor. We simulated this situation by deliberately cleaving upstream of the <em>act5C</em> stop codon. Perhaps not surprisingly, moving the cleavage site 20 bp away from the desired integration site reduced the efficiency of our genome editing procedure; most likely, this is because the effective length of one homology arm is correspondingly shortened. This efficiency reduction was particularly obvious if HR donors with wild-type sequence were used, presumably because the homology arm was cleaved off prior to integration or because of secondary mutagenesis after integration. A single point mutation within the corresponding HR donor primer was sufficient to restore efficiency; increasing the number of mismatches to the sgRNA sequence did not lead to further improvements (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>).</p></div><div id="sec-11" class="subsection"><h3>Homology arm lengths of 60 nt suffice for optimal efficiency</h3><p id="p-21">Our approach relies on the incorporation of homology arms into PCR primers during oligonucleotide synthesis. This is very convenient but also represents the most expensive reagent in our protocol. Therefore, it is imperative to minimize the length of these homology arms. We had previously demonstrated that 26 bp can suffice for sequence-specific integration, but the efficiency was clearly lower than with 80 bp. We have now compared homology arm lengths of 40, 60, and 80 bp (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>, left panel). According to our results, homology arm lengths of 60 bp are sufficient for optimal efficiency. While a shortening of the homology to 40 bp clearly reduced the efficiency, an extension to 80 bp did not result in further improvement. To apply this analysis to even longer homology arms, we generated an Act5C-GFP targeting vector (pIS1) by sequentially cloning homology arms of 500 bp upstream and 420 bp downstream homology into our pMH3 template vector. We compared this targeting vector in both circular and linearized form to PCR products with 60 and 80 bp of flanking homology (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2D</a>, right panel). Increasing the homology arm length from 60/60 to 500/420 did not result in an increased overall efficiency (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2D</a>, right panel) or initial recombination rate (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FigureS2.pdf">Figure S2</a>). Note that in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2D</a> only the right panel was generated with a concomitant <em>lig4/mus308</em> knockdown (see below), whereas the left panel was generated with only a <em>lig4</em> knockdown.</p><p id="p-22">Taken together, we recommend a homology arm length of 60 bp on each side for maximal efficiency; nonetheless, shorter homology arms (<em>e.g.</em>, 40 bp each) may be an acceptable compromise to reduce the costs in large-scale efforts if a somewhat lower efficiency can be tolerated. It is possible that extending the homology regions beyond 500 bp will result in even higher targeting rates. Our interpretation is, however, that beyond a size of 60 bp, the homology arm length is not the limiting factor in this genome editing protocol.</p></div><div id="sec-12" class="subsection"><h3>Improved efficiency due to combined knockdown of lig4 and mus308</h3><p id="p-23">The selection of genome-modified cells by resistance to the antibiotic blasticidin-S has allowed us to significantly enrich the desired outcome in the final cell population. However, cells may also become resistant if the HR donor construct integrates spontaneously at a random position within the genome. We refer to these events as “off-site” integrations, to distinguish them from the “off-target” cleavage that may occur with the <em>cas9</em> nuclease. As originally demonstrated with mutant flies (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carroll <em>et al.</em> 2008</a>), transient depletion of <a href="http://flybase.org/reports/FBgn0030506.html">DNA ligase 4</a> by RNAi reduces these off-site integrations and increases the yield of desired cells (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Bottcher <em>et al.</em> 2014</a>). This beneficial effect can be further enhanced by a concomitant depletion of the DNA polymerase θ ortholog <em><a href="http://flybase.org/reports/FBgn0002905.html">mus308</a></em>, an enzyme that is important during microhomology-mediated end joining (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2E</a>). In experiments where a concomitant knockdown of <a href="http://flybase.org/reports/FBgn0030506.html"><em>lig4</em></a> and <em><a href="http://flybase.org/reports/FBgn0002905.html">mus308</a></em> was combined with a 19 nt optimal-length sgRNA and 60 nt long homology arms, we could reproducibly achieve an Act5C-GFP targeting rate of &gt; 65% among the selected population (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2, B and E</a> and unpublished results). This is likely sufficient for many applications that rely on a gain-of-function approach, such as the introduction of epitope tags or fluorescent markers. Note that some of the experiments in this manuscript (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a> and <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2, A and D</a>, left pane) were performed using only a <a href="http://flybase.org/reports/FBgn0030506.html"><em>lig4</em></a> knockdown. The parameters that we optimized in these tests are unlikely to be affected by the additional <em><a href="http://flybase.org/reports/FBgn0002905.html">mus308</a></em> knockdown. In principle, a combination of <a href="http://flybase.org/reports/FBgn0030506.html"><em>lig4</em></a> and <em><a href="http://flybase.org/reports/FBgn0002905.html">mus308</a></em> alleles should be beneficial for homology-directed genome editing in injected fly embryos or other organisms as well; however, we observed that the two genes display synthetic lethality at higher dsRNA doses.</p></div><div id="sec-13" class="subsection"><h3>Extension of the approach to N-terminal tags</h3><p id="p-24">We extended our approach to the introduction of N-terminal protein tags. To this end, a new template vector design had to be developed that also comprises a promoter for heterologous expression of the tagged gene. This is necessary because, in the case of an N-terminal tag, the selection cassette separates the endogenous promoter from the gene body. We chose the inducible, bidirectional <em>mtnDE</em> promoter for this purpose. Induction is possible by adding <em>e.g.</em>, CuSO<sub>4</sub> to the growth medium and the promoter can drive expression of the selection marker and the tagged protein concomitantly. In addition, we combined the <em>mtnDE</em>-driven expression of the tagged gene with constitutively active marker cassettes, to select modified cells without the need for induction of the tagged gene. This may be helpful if expression of the resulting fusion gene is toxic (<em>e.g.</em>, due to overexpression, generation of N-terminal truncations, misexpression etc.). As for the C-terminal approach, our vector templates bear constant regions for annealing of the homology-containing targeting primers during PCR. All N-terminal tags and selection constructs can thus be amplified with a single set of HR-containing primers (summarized in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). We have developed N-terminal vector templates for GFP, the 3 × Flag epitope, a single Strep-tag, and a combined Strep-His<sub>8</sub>-Strep-TEV tag; the latter one can be removed from the purified protein by TEV-cleavage (see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> for an overview of all template vectors currently available at Addgene). The His<sub>8</sub>-sequence allows removal of the cleaved tag from the reaction along with the usually His-tagged TEV protease protein. The tag may also be used for tandem affinity purification (we recommend to start with Ni-NTA beads and use Streptactin beads as the second step). Since the selection cassette is flanked by FRT-sites in all vectors, it can be removed with FLP recombinase to restore expression control by the endogenous promoter. Note that both the N- and C-terminal template vectors can easily be modified, <em>e.g.</em>, using restriction enzyme-based cloning to harbor other tags, fluorescent proteins, or elements for genome functionalization. For example, we generated template vectors that allow the introduction of an <em>attP</em> target site based on the C-terminal epitope tag template series (see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Template vectors for N-terminal tagging. Top row: schematic representation of the three different vector template series used for N-terminal tagging. In the first series (left), the inducible and bidirectional mtnDE promoter drives the expression of the Blasticidin-resistance gene. We recommend adding CuSO4 to a concentration of 100 &#xB5;M or higher 2 d prior to transfer into selective medium. Constitutive expression (driven by the copia promoter) of the Blasticidin or Puromycin resistance genes occurs, respectively, when the second (center) or third (right) series of templates is employed. Note that a single HR-containing primer set can be used with any of these templates. Bottom: Sequence details of the different N-terminal tags for which we developed template vectors. AmpR, ampicillin resistance; BlastR, blasticidin resistance; FRT, Flp recombinase target; GFP, green fluorescent protein; His, histidine; HR, homologous recombination; TEV, tobacco etch virus protease cleavage site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901039782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Template vectors for N-terminal tagging. Top row: schematic representation of the three different vector template series used for N-terminal tagging. In the first series (left), the inducible and bidirectional mtnDE promoter drives the expression of the Blasticidin-resistance gene. We recommend adding CuSO&lt;sub&gt;4&lt;/sub&gt; to a concentration of 100 &#xB5;M or higher 2 d prior to transfer into selective medium. Constitutive expression (driven by the &lt;em&gt;copia&lt;/em&gt; promoter) of the Blasticidin or Puromycin resistance genes occurs, respectively, when the second (center) or third (right) series of templates is employed. Note that a single HR-containing primer set can be used with any of these templates. Bottom: Sequence details of the different N-terminal tags for which we developed template vectors. Amp&lt;sup&gt;R&lt;/sup&gt;, ampicillin resistance; Blast&lt;sup&gt;R&lt;/sup&gt;, blasticidin resistance; FRT, Flp recombinase target; GFP, green fluorescent protein; His, histidine; HR, homologous recombination; TEV, tobacco etch virus protease cleavage site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1777/F3.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/6/1777/F3.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468352" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Template vectors for N-terminal tagging. Top row: schematic representation of the three different vector template series used for N-terminal tagging. In the first series (left), the inducible and bidirectional mtnDE promoter drives the expression of the Blasticidin-resistance gene. We recommend adding CuSO<sub>4</sub> to a concentration of 100 µM or higher 2 d prior to transfer into selective medium. Constitutive expression (driven by the <em>copia</em> promoter) of the Blasticidin or Puromycin resistance genes occurs, respectively, when the second (center) or third (right) series of templates is employed. Note that a single HR-containing primer set can be used with any of these templates. Bottom: Sequence details of the different N-terminal tags for which we developed template vectors. Amp<sup>R</sup>, ampicillin resistance; Blast<sup>R</sup>, blasticidin resistance; FRT, Flp recombinase target; GFP, green fluorescent protein; His, histidine; HR, homologous recombination; TEV, tobacco etch virus protease cleavage site.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Heterologous control of gene expression by the mtnDE promoter</h3><p id="p-26">If all alleles for a given gene in the S2-cell genome have been modified (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FileS2.pdf">File S2</a> for clonal selection of cells and PCR genotyping strategies), the introduced mtnDE-promoter allows heterologous control over the expression of the targeted gene. As an example, we derived cell lines with N-terminally GFP-tagged Dcr-2 protein. After clonal selection, we could readily identify cells that carried only modified <em>dcr-2</em> alleles. These lines allowed us to tune the expression level of Dcr-2; analysis by FACS and western blotting demonstrated that both transcriptional shut-off and overexpression situations can be obtained (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We have not extensively characterized the leakiness of the <em>mtnDE</em> promoter in the noninduced state, but preliminary results indicate that the transcript levels are comparable to those observed after RNAi (data not shown; this likely also varies according to the genomic position). For simple loss-of-function approaches, RNAi is far easier to apply; however, the mtnDE promoter “alleles” may present an interesting tool to study genetic interaction in combination with RNAi of a second factor. In particular, they may be convenient to create hypomorphic expression levels of essential genes in order to make them amenable for synthetic genetic screens.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterologous control of gene expression with the mtnDE promoter and quantification of FLP-out efficiency. (A) We could generate clonal cell populations where all dcr-2 alleles had been modified to include an N-terminal GFP tag. Prior to marker excision, this allows for heterologous and tunable control of GFP-Dcr-2 expression levels. We quantified the amount of modified Dcr-2 protein per cell by comparing the N-terminal fusions (bottom left) with a C-terminal fusion (top left) that presumably reflects the endogenous level of Dcr-2 expression. We estimate that, in this particular case, a CuSO4 concentration of 30&#x2013;50 &#xB5;M leads to wild-type expression levels. Western blotting experiments with a monoclonal antibody directed against the endogenous protein confirmed the absence of untagged protein and that CuSO4 concentrations between 30&#x2013;50 &#xB5;M led to wild-type equivalent expression. The mtnDE promoter allows the tuning of expression levels within a roughly tenfold range (see western blot on top right and flow cytometry quantification on bottom right), but absolute levels may vary between genes. (B) An N-terminal GFP-tag at the twinstar (tsr) locus served as a test-platform to determine the FLP-out efficiency. The left panel depicts the experimental approach: In the absence of CuSO4, no GFP-expression is detectable in cells that harbor one or more edited tsr alleles. Upon FLP-mediated recombination between the FRT sites flanking the marker gene and mtnDE-promoter, transcription of the fusion gene is again controlled by the endogenous tsr promoter and GFP-tsr fluorescence can be measured and quantified by flow cytometry. This is depicted in the right panel for three clonal cell populations carrying modified tsr alleles. Untransfected cells and cells transfected with a luciferase-expression plasmid served as controls; we compared our previously published FLP expression vector (pMH5) with an improved version (pKF295). The depicted values are the mean &#xB1; SD, n = 4 biological replicates (clone 21: 5 biological replicates), p-values: t-test. FRT, Flp recombinase target; GFP, green fluorescent protein; SD, standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901039782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heterologous control of gene expression with the &lt;em&gt;mtnDE&lt;/em&gt; promoter and quantification of FLP-out efficiency. (A) We could generate clonal cell populations where all &lt;em&gt;dcr-2&lt;/em&gt; alleles had been modified to include an N-terminal GFP tag. Prior to marker excision, this allows for heterologous and tunable control of GFP-Dcr-2 expression levels. We quantified the amount of modified Dcr-2 protein per cell by comparing the N-terminal fusions (bottom left) with a C-terminal fusion (top left) that presumably reflects the endogenous level of Dcr-2 expression. We estimate that, in this particular case, a CuSO&lt;sub&gt;4&lt;/sub&gt; concentration of 30&#x2013;50 &#xB5;M leads to wild-type expression levels. Western blotting experiments with a monoclonal antibody directed against the endogenous protein confirmed the absence of untagged protein and that CuSO&lt;sub&gt;4&lt;/sub&gt; concentrations between 30&#x2013;50 &#xB5;M led to wild-type equivalent expression. The &lt;em&gt;mtnDE&lt;/em&gt; promoter allows the tuning of expression levels within a roughly tenfold range (see western blot on top right and flow cytometry quantification on bottom right), but absolute levels may vary between genes. (B) An N-terminal GFP-tag at the &lt;em&gt;twinstar (tsr)&lt;/em&gt; locus served as a test-platform to determine the FLP-out efficiency. The left panel depicts the experimental approach: In the absence of CuSO&lt;sub&gt;4&lt;/sub&gt;, no GFP-expression is detectable in cells that harbor one or more edited &lt;em&gt;tsr&lt;/em&gt; alleles. Upon FLP-mediated recombination between the FRT sites flanking the marker gene and &lt;em&gt;mtnDE&lt;/em&gt;-promoter, transcription of the fusion gene is again controlled by the endogenous &lt;em&gt;tsr&lt;/em&gt; promoter and GFP-tsr fluorescence can be measured and quantified by flow cytometry. This is depicted in the right panel for three clonal cell populations carrying modified &lt;em&gt;tsr&lt;/em&gt; alleles. Untransfected cells and cells transfected with a luciferase-expression plasmid served as controls; we compared our previously published FLP expression vector (pMH5) with an improved version (pKF295). The depicted values are the mean &#xB1; SD, &lt;em&gt;n&lt;/em&gt; = 4 biological replicates (clone 21: 5 biological replicates), &lt;em&gt;p&lt;/em&gt;-values: &lt;em&gt;t&lt;/em&gt;-test. FRT, Flp recombinase target; GFP, green fluorescent protein; SD, standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1777/F4.medium.gif" width="367" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/6/1777/F4.medium.gif" width="367" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1777/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468327" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Heterologous control of gene expression with the <em>mtnDE</em> promoter and quantification of FLP-out efficiency. (A) We could generate clonal cell populations where all <em>dcr-2</em> alleles had been modified to include an N-terminal GFP tag. Prior to marker excision, this allows for heterologous and tunable control of GFP-Dcr-2 expression levels. We quantified the amount of modified Dcr-2 protein per cell by comparing the N-terminal fusions (bottom left) with a C-terminal fusion (top left) that presumably reflects the endogenous level of Dcr-2 expression. We estimate that, in this particular case, a CuSO<sub>4</sub> concentration of 30–50 µM leads to wild-type expression levels. Western blotting experiments with a monoclonal antibody directed against the endogenous protein confirmed the absence of untagged protein and that CuSO<sub>4</sub> concentrations between 30–50 µM led to wild-type equivalent expression. The <em>mtnDE</em> promoter allows the tuning of expression levels within a roughly tenfold range (see western blot on top right and flow cytometry quantification on bottom right), but absolute levels may vary between genes. (B) An N-terminal GFP-tag at the <em>twinstar (tsr)</em> locus served as a test-platform to determine the FLP-out efficiency. The left panel depicts the experimental approach: In the absence of CuSO<sub>4</sub>, no GFP-expression is detectable in cells that harbor one or more edited <em>tsr</em> alleles. Upon FLP-mediated recombination between the FRT sites flanking the marker gene and <em>mtnDE</em>-promoter, transcription of the fusion gene is again controlled by the endogenous <em>tsr</em> promoter and GFP-tsr fluorescence can be measured and quantified by flow cytometry. This is depicted in the right panel for three clonal cell populations carrying modified <em>tsr</em> alleles. Untransfected cells and cells transfected with a luciferase-expression plasmid served as controls; we compared our previously published FLP expression vector (pMH5) with an improved version (pKF295). The depicted values are the mean ± SD, <em>n</em> = 4 biological replicates (clone 21: 5 biological replicates), <em>p</em>-values: <em>t</em>-test. FRT, Flp recombinase target; GFP, green fluorescent protein; SD, standard deviation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Generation of an improved FLP-recombinase expression vector</h3><p id="p-28">We reasoned that if a robustly expressed gene is N-terminally GFP-tagged with our system, then the efficiency of marker-cassette FLP-out can be quantified by measuring the proportion of GFP-positive cells in the absence of CuSO<sub>4</sub>, once expression is again driven by the endogenous promoter. We designed oligonucleotides targeting the <em>twinstar (tsr)</em> gene, a highly expressed <em>Drosophila</em> homolog of cofilin. Stable and clonal cell populations were selected and flow cytometry was used to verify that all cells in the populations became GFP-positive after CuSO<sub>4</sub> induction (data not shown). Successful cassette FLP-out could then be measured as the amount of GFP-positive cells following transfection of a FLP-recombinase expression plasmid. The measured GFP-fluorescence intensities were lower than what we had expected based on published <em><a href="http://flybase.org/reports/FBgn0011726.html">tsr</a></em> expression levels; nonetheless, the system allows reliable quantification of the successful FLP-out events. We had previously described an expression vector for wild-type <em>Saccharomyces cerevisiae</em> FLP recombinase (pMH5) and noticed that the efficiency of FLP-out was somewhat limited. Improved versions of the FLP recombinase have been described (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Buchholz <em>et al.</em> 1998</a>) and we appended an N-terminal nuclear localization signal in combination with the proline(2) -&gt; serine substitution to generate plasmid pKF295. Using the GFP-<em>tsr</em> test system in three separate clonal cell lines, we found that transfection of pKF295 approximately doubled the amount of GFP-positive cells compared to pMH5 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). In absolute terms, a FLP-out efficiency of 5–10% (pMH5) and 10–20% (pKF295) could be reached. Considering the transfection efficiency of ∼60%, the higher values correspond to about one third (pKF295) and one fifth (pMH5) of the transfected cells.</p></div><div id="sec-16" class="subsection"><h3>Introduction of a second selection cassette</h3><p id="p-29">To enable straightforward introduction of a second epitope tag (<em>e.g.</em>, for coimmunoprecipitation studies), we developed an independent selection cassette based on the puromycin acetyl transferase gene. Since the commonly used coding sequence of this gene proved to be rather refractory to PCR amplification (likely due to a high GC content), we created a <em>copia</em>-Puro resistance cassette as a synthetic gene. This element could be readily amplified by PCR and we have generated alternative versions of most template vectors by exchanging the <em>copia</em>-Blast cassette with the <em>copia</em>-Puro marker (see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). As a general proof-of-principle, we show the introduction of a C-terminal twin-Strep tag (pSK24-based, see <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) at the Dcr-2 locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1/FigureS3.pdf">Figure S3</a>). We observed that our S2-cell line is quite sensitive to puromycin; selection works well at a concentration of 0.5 µg/ml of puromycin in Schneider’s medium. Integration of a puromycin construct in a blasticidin-resistant cell line (or vice versa) is efficient and does not require any changes to the protocol (data not shown). We have not quantified the tagging efficiencies with puromycin resistance-based constructs in a manner analogous to the experiments shown, <em>e.g.</em>, in <a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a>. The tagging success rates clearly depend on optimal sgRNA length and the extent of homology arms in the HR donor PCR product; since these elements are independent of the marker chosen, we do not expect major quantitative differences between puromycin and blasticidin-based selections.</p><p id="p-30">In principle, the two constructs could also be integrated in parallel rather than sequentially. Indeed, we have been able to recover cells that underwent corresponding multiplex genome editing. However, these cells are very rare in the initial population due to the rather low initial targeting frequency (3–10%). As a consequence, the multiplexed editing approach is not very robust and more time is required until sufficient cells have grown for downstream analysis. We therefore recommend sequential introduction if multiple epitope tags need to be combined within the same cell. If one begins with an inducible blasticidin resistance construct for an N-terminal tag (pKF290-294 or pRB30, see <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>) and then continues with constitutive puromycin and blasticidin resistance cassettes, up to three epitope tags can be combined without the need to FLP out the marker in between.</p></div><div id="sec-17" class="subsection"><h3>Conclusions</h3><p id="p-31">In this publication, we describe a simplified genome editing protocol with increased efficiency, an extension to N-terminal epitope tags and the application of a second, independent selection cassette for our PCR-based genome editing of cultured <em>Drosophila</em> cells. <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a> summarizes all of the template vectors that we have developed to date. The genomic tagging system now offers significantly increased functionality, including the possibility to verify protein–protein interactions via coimmunoprecipitation. The improved primer design to generate U6-sgRNA expression cassettes has made the system much more robust and enables the reliable generation of reagents for many genes in parallel. Due to the use of PCR rather than cloning, our experimental approach is flexible, scalable, and can easily accommodate future changes (<em>e.g.</em>, different sgRNA designs, novel epitope tags). We estimate that it should be straightforward to extend our strategy to other <em>Drosophila</em> cell culture systems, potentially even to cultured cells from other insect species. Related PCR-based approaches have been described for use in cultured vertebrate cells (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Li <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Stewart-Ornstein and Lahav 2016</a>). We expect that our vector templates can be modified for use beyond insect cells by exchanging the <em>copia</em>-promoter used for selection and/or the inducible <em>mtnDE</em> promoter with sequences of corresponding functionality in, <em>e.g.</em>, vertebrate cells. Perhaps more importantly, it may be possible to transfer the conclusions from our optimization efforts to other cell culture systems as well.</p></div></div><div class="section" id="sec-18"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">The Dcr-2 antibody was a generous gift of Mikiko Siomi. This work was supported by the Deutsche Forschungsgemeinschaft (grant SFB646).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-33">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028241/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-34"><em>Communicating editor: K. S. McKim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-02-26"><span class="received-label">Received </span>February 26, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-12"><span class="accepted-label">Accepted </span>April 12, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Kunzelmann <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.1" data-doi="10.1242/bio.20137120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassett</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibbit</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mutagenesis and homologous recombination in <em>Drosophila</em> cell lines using CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Biol. Open</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BOpen%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fbio.20137120%26rft_id%253Dinfo%253Apmid%252F24326186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbG9wZW4iO3M6NToicmVzaWQiO3M6NjoiMy8xLzQyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.2" data-doi="10.1093/nar/gku289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bottcher</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merk</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nitschko</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obermaier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient chromosomal gene modification with CRISPR/<em>cas9</em> and PCR-based homologous recombination donors in cultured <em>Drosophila</em> cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku289%26rft_id%253Dinfo%253Apmid%252F24748663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzExL2U4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNzc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.3" data-doi="10.1038/nbt0798-657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchholz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angrand</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Improved properties of FLP recombinase evolved by cycling mutagenesis.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBuchholz%26rft.auinit1%253DF.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DImproved%2Bproperties%2Bof%2BFLP%2Brecombinase%2Bevolved%2Bby%2Bcycling%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0798-657%26rft_id%253Dinfo%253Apmid%252F9661200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0798-657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9661200&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074521800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.4" data-doi="10.1016/B978-0-12-801185-0.00006-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome editing in human stem cells.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D546%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-801185-0.00006-4%26rft_id%253Dinfo%253Apmid%252F25398338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-801185-0.00006-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398338&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.5" data-doi="10.1007/978-1-59745-232-8_5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beumer</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trautman</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene targeting in <em>Drosophila</em> and <em>Caenorhabditis elegans</em> with zinc-finger nucleases.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">435</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DCarroll%26rft.auinit1%253DD.%26rft.volume%253D435%26rft.spage%253D63%26rft.epage%253D77%26rft.atitle%253DGene%2Btargeting%2Bin%2BDrosophila%2Band%2BCaenorhabditis%2Belegans%2Bwith%2Bzinc-finger%2Bnucleases.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-232-8_5%26rft_id%253Dinfo%253Apmid%252F18370068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-232-8_5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18370068&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.6" data-doi="10.1016/j.cell.2013.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cimini</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnitzbauer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1479</span>–<span class="cit-lpage">1491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D155%26rft.spage%253D1479%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.12.001%26rft_id%253Dinfo%253Apmid%252F24360272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24360272&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328693300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.7" data-doi="10.1126/science.1258096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome editing. The new frontier of genome engineering with CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1258096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDoudna%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D346%26rft.issue%253D6213%26rft.spage%253D1258096%26rft.epage%253D1258096%26rft.atitle%253DThe%2Bnew%2Bfrontier%2Bof%2Bgenome%2Bengineering%2Bwith%2BCRISPR-Cas9%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1258096%26rft_id%253Dinfo%253Apmid%252F25430774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDYvNjIxMy8xMjU4MDk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.8" data-doi="10.1007/978-1-4939-1862-1_12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fetter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samsonov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenser</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Endogenous gene tagging with fluorescent proteins.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1239</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1239%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-1862-1_12%26rft_id%253Dinfo%253Apmid%252F25408409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-1862-1_12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25408409&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.9" data-doi="10.1016/B978-0-12-801185-0.00002-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Targeted genome editing in human cells using CRISPR/Cas nucleases and truncated guide RNAs.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D546%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-801185-0.00002-7%26rft_id%253Dinfo%253Apmid%252F25398334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-801185-0.00002-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398334&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.10" data-doi="10.1038/nbt.2808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sander</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cascio</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Improving CRISPR-Cas nuclease specificity using truncated guide RNAs.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2808%26rft_id%253Dinfo%253Apmid%252F24463574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24463574&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.11" data-doi="10.1126/scisignal.aab3729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Housden</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valvezan</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of potential drug targets for tuberous sclerosis complex by synthetic screens combining CRISPR-based knockouts with RNAi.</span> <abbr class="cit-jnl-abbrev">Sci. Signal.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">rs9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DHousden%26rft.auinit1%253DB.%2BE.%26rft.volume%253D8%26rft.issue%253D393%26rft.spage%253Drs9%26rft.epage%253Drs9%26rft.atitle%253DIdentification%2Bof%2Bpotential%2Bdrug%2Btargets%2Bfor%2Btuberous%2Bsclerosis%2Bcomplex%2Bby%2Bsynthetic%2Bscreens%2Bcombining%2BCRISPR-based%2Bknockouts%2Bwith%2BRNAi%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.aab3729%26rft_id%253Dinfo%253Apmid%252F26350902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6OToiOC8zOTMvcnM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.12" data-doi="10.1371/journal.pone.0105779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pu</span>  <span class="cit-name-given-names">W. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Optimization of genome engineering approaches with the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e105779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De105779%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0105779%26rft_id%253Dinfo%253Apmid%252F25166277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0105779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25166277&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.13" data-doi="10.1038/nmeth.3543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Mateos</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vejnar</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaudoin</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mis</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting <em>in vivo</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D982%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3543%26rft_id%253Dinfo%253Apmid%252F26322839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26322839&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.14" data-doi="10.1016/j.celrep.2014.09.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Enhanced specificity and efficiency of the CRISPR/Cas9 system with optimized sgRNA parameters in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D9%26rft.spage%253D1151%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2014.09.044%26rft_id%253Dinfo%253Apmid%252F25437567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2014.09.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25437567&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forstemann</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Monitoring miRNA-mediated silencing in <em>Drosophila melanogaster</em> S2-cells.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1779</span>: <span class="cit-fpage">766</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DShah%26rft.auinit1%253DC.%26rft.volume%253D1779%26rft.issue%253D11%26rft.spage%253D766%26rft.epage%253D772%26rft.atitle%253DMonitoring%2BmiRNA-mediated%2Bsilencing%2Bin%2BDrosophila%2Bmelanogaster%2BS2-cells.%26rft_id%253Dinfo%253Apmid%252F18634912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18634912&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.16" data-doi="10.1016/j.celrep.2016.01.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart-Ornstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahav</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Dynamics of CDKN1A in single cells defined by an endogenous fluorescent tagging toolkit.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1800</span>–<span class="cit-lpage">1811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D14%26rft.spage%253D1800%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2016.01.045%26rft_id%253Dinfo%253Apmid%252F26876176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2016.01.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26876176&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1777.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1777.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wyvekens</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome editing in human cells using CRISPR/Cas nucleases.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Mol. Biol.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">31.3.1</span>–<span class="cit-lpage">31.3.18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMol.%2BBiol.%26rft.volume%253D112%26rft.spage%253D3131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/6/1777.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1767" title="Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1787" title="Site-Directed Genome Knockout in Chicken Cell Line and Embryos Can Use CRISPR/Cas Gene Editing Technology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/6"><span>Volume 6 Issue 6, June 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/6.cover-source.jpg?itok=WAtQozJg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1777.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-A7V3EK6-AecHNPdT8S6xalwKajrx-6DC_qIIbgBmaiE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/6/1777" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/6/1777" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/6/1777" class="active" data-icon-position="" data-hide-link-title="0">A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460384" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--hJUZZSln12u_yBRrA6ar999NFgv_Cij1NccF6yh2jI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1777.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/6/1777" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3YJi1L9vpJi5lWkf8Hkq5XP81akEarfl911UQBVyInQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/6/1777" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1777.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460384" id="node460384--2" data-pisa="ggg;6/6/1777" data-pisa-master="ggg;g3.116.028241" data-apath="/ggg/6/6/1777.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Comprehensive Toolbox for Genome Editing in Cultured <em>Drosophila melanogaster</em> Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Stefan Kunzelmann</span>, <span class="highwire-citation-author" data-delta="1">Romy Böttcher</span>, <span class="highwire-citation-author" data-delta="2">Ines Schmidts</span> and <span class="highwire-citation-author" data-delta="3">Klaus Förstemann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1777-1785;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028241 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460384--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stefan Kunzelmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BKunzelmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kunzelmann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BKunzelmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Romy Böttcher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romy%2BB%C3%B6ttcher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%B6ttcher%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomy%2BB%25C3%25B6ttcher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ines Schmidts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BSchmidts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidts%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BSchmidts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Klaus Förstemann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klaus%2BF%C3%B6rstemann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=F%C3%B6rstemann%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKlaus%2BF%25C3%25B6rstemann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:foerstemann@lmb.uni-muenchen.de" class="" data-icon-position="" data-hide-link-title="0">foerstemann@lmb.uni-muenchen.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/6/1777" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/6/1777&amp;title=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/6/1777&amp;title=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/6/1777&amp;title=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/6/1777&amp;text=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/6/1777&amp;title=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/6/1777&amp;t=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/6/1777&amp;title=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/6/1777&amp;title=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460384/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460384" id="node460384--3" data-pisa="ggg;6/6/1777" data-pisa-master="ggg;g3.116.028241" data-apath="/ggg/6/6/1777.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Comprehensive Toolbox for Genome Editing in Cultured <em>Drosophila melanogaster</em> Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Stefan Kunzelmann</span>, <span class="highwire-citation-author" data-delta="1">Romy Böttcher</span>, <span class="highwire-citation-author" data-delta="2">Ines Schmidts</span> and <span class="highwire-citation-author" data-delta="3">Klaus Förstemann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1777-1785;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028241 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460384--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stefan Kunzelmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BKunzelmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kunzelmann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BKunzelmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Romy Böttcher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romy%2BB%C3%B6ttcher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%B6ttcher%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomy%2BB%25C3%25B6ttcher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ines Schmidts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BSchmidts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidts%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BSchmidts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Klaus Förstemann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry, Gene Center, Ludwig-Maximilians-Universität München, D-81377, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klaus%2BF%C3%B6rstemann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=F%C3%B6rstemann%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKlaus%2BF%25C3%25B6rstemann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:foerstemann@lmb.uni-muenchen.de" class="" data-icon-position="" data-hide-link-title="0">foerstemann@lmb.uni-muenchen.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1777&amp;rft.epage=1785&amp;rft.atitle=A%20Comprehensive%20Toolbox%20for%20Genome%20Editing%20in%20Cultured%20Drosophila%20melanogaster%20Cells&amp;rft.volume=6&amp;rft.issue=6&amp;rft.date=2016-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kunzelmann%2C+Stefan&amp;rft.au=B%C3%B6ttcher%2C+Romy&amp;rft.au=Schmidts%2C+Ines&amp;rft.au=F%C3%B6rstemann%2C+Klaus"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460384/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460384/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460384/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460384/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460384/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460384/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460384/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460384/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460384/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460384/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460384/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460384/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/6/1777" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/6/1777.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/6/1777.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/6/1777.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/6/1777.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
