<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/6/1767" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/6/1767.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/6/1767.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/6/1767.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/6/1767.atom" />
<meta name="HW.pisa" content="ggg;6/6/1767" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome" />
<meta name="DC.Identifier" content="10.1534/g3.116.028001" />
<meta name="DC.Date" content="2016-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Bernstein and Rockman. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Meiotic recombination creates genotypic diversity within species. Recombination rates vary substantially across taxa, and the distribution of crossovers can differ significantly among populations and between sexes. Crossover locations within species have been found to vary by chromosome and by position within chromosomes, where most crossover events occur in small regions known as recombination hotspots. However, several species appear to lack hotspots despite significant crossover heterogeneity. The nematode Caenorhabditis elegans was previously found to have the least fine-scale variation in crossover distribution among organisms studied to date. It is unclear whether this pattern extends to the X chromosome given its unique compaction through the pachytene stage of meiotic prophase in hermaphrodites. We generated 798 recombinant nested near-isogenic lines (NILs) with crossovers in a 1.41 Mb region on the left arm of the X chromosome to determine if its recombination landscape is similar to that of the autosomes. We find that the fine-scale variation in crossover rate is lower than that of other model species, and is inconsistent with hotspots. The relationship of genomic features to crossover rate is dependent on scale, with GC content, histone modifications, and nucleosome occupancy being negatively associated with crossovers. We also find that the abundances of 4- to 6-bp DNA motifs significantly explain crossover density. These results are consistent with recombination occurring at unevenly distributed sites of open chromatin." />
<meta name="DC.Contributor" content="Max R. Bernstein" />
<meta name="DC.Contributor" content="Matthew V. Rockman" />
<meta name="article:published_time" content="2016-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Meiotic recombination creates genotypic diversity within species. Recombination rates vary substantially across taxa, and the distribution of crossovers can differ significantly among populations and between sexes. Crossover locations within species have been found to vary by chromosome and by position within chromosomes, where most crossover events occur in small regions known as recombination hotspots. However, several species appear to lack hotspots despite significant crossover heterogeneity. The nematode &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; was previously found to have the least fine-scale variation in crossover distribution among organisms studied to date. It is unclear whether this pattern extends to the X chromosome given its unique compaction through the pachytene stage of meiotic prophase in hermaphrodites. We generated 798 recombinant nested near-isogenic lines (NILs) with crossovers in a 1.41&amp;nbsp;Mb region on the left arm of the X chromosome to determine if its recombination landscape is similar to that of the autosomes. We find that the fine-scale variation in crossover rate is lower than that of other model species, and is inconsistent with hotspots. The relationship of genomic features to crossover rate is dependent on scale, with GC content, histone modifications, and nucleosome occupancy being negatively associated with crossovers. We also find that the abundances of 4- to 6-bp DNA motifs significantly explain crossover density. These results are consistent with recombination occurring at unevenly distributed sites of open chromatin.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/06/01" />
<meta name="citation_mjid" content="ggg;6/6/1767" />
<meta name="citation_id" content="6/6/1767" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/6/1767" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/6/1767.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/6/1767.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/6/1767.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028001" />
<meta name="citation_pmid" content="27172189" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1767" />
<meta name="citation_lastpage" content="1776" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Max R. Bernstein" />
<meta name="citation_author_institution" content="Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6468-3953" />
<meta name="citation_author" content="Matthew V. Rockman" />
<meta name="citation_author_institution" content="Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003" />
<meta name="citation_author_email" content="mrockman@nyu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6492-8906" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. D. Akhunov;citation_author=A. W. Goodyear;citation_author=S. Geng;citation_author=L.–L. Qi;citation_author=B. Echalier;citation_title=The organization and rate of evolution of wheat genomes are correlated with recombination rates along chromosome arms.;citation_volume=13;citation_year=2003;citation_pmid=12695326;citation_doi=10.1101/gr.808603" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=D. G. Albertson;citation_author=J. N. Thomson;citation_title=The kinetochores of Caenorhabditis elegans.;citation_pages=409-428;citation_volume=86;citation_year=1982;citation_issue=3;citation_pmid=7172865;citation_doi=10.1007/BF00292267" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. C. Andersen;citation_author=J. P. Gerke;citation_author=J. A. Shapiro;citation_author=J. R. Crissman;citation_author=R. Ghosh;citation_title=Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity.;citation_pages=285-290;citation_volume=44;citation_year=2012;citation_issue=3;citation_pmid=22286215;citation_doi=10.1038/ng.1050" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. C. Andersen;citation_author=J. S. Bloom;citation_author=J. P. Gerke;citation_author=L. Kruglyak;citation_title=A variant in the neuropeptide receptor npr-1 is a major determinant of Caenorhabditis elegans growth and physiology.;citation_pages=e1004156;citation_volume=10;citation_year=2014;citation_pmid=24586193;citation_doi=10.1371/journal.pgen.1004156" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. K. Arico;citation_author=D. J. Katz;citation_author=J. van der Vlag;citation_author=W. G. Kelly;citation_title=Epigenetic patterns maintained in early Caenorhabditis elegans embryos can be established by gene activity in the parental germ cells.;citation_pages=e1001391-e1001391;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21695223;citation_doi=10.1371/journal.pgen.1001391" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Auton;citation_author=Y. Rui Li;citation_author=J. Kidd;citation_author=K. Oliveira;citation_author=J. Nadel;citation_title=Genetic recombination is targeted towards gene promoter regions in dogs.;citation_pages=e1003984;citation_volume=9;citation_year=2013;citation_pmid=24348265;citation_doi=10.1371/journal.pgen.1003984" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. L. Bailey;citation_author=M. Boden;citation_author=F. A. Buske;citation_author=M. Frith;citation_author=C. E. Grant;citation_title=MEME SUITE: tools for motif discovery and searching.;citation_pages=W202W208;citation_volume=37;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Barnes;citation_author=Y. Kohara;citation_author=A. Coulson;citation_author=S. Hekimi;citation_title=Meiotic recombination, noncoding DNA and genomic organization in Caenorhabditis elegans.;citation_pages=159-179;citation_volume=141;citation_year=1995;citation_issue=1;citation_pmid=8536965;citation_doi=10.1038/msb.2012.19" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Barrière;citation_author=M.-A. Félix;citation_title=Temporal dynamics and linkage disequilibrium in natural Caenorhabditis elegans populations.;citation_volume=176;citation_year=2007;citation_pmid=17409084;citation_doi=10.1534/genetics.106.067223" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. B. Barton;citation_author=M. R. Pekosz;citation_author=R. S. Kurvathi;citation_author=D. B. Kaback;citation_title=Meiotic recombination at the ends of chromosomes in Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18562657;citation_doi=10.1534/genetics.107.083493" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. Baudat;citation_author=Y. Imai;citation_author=B. de Massy;citation_title=Meiotic recombination in mammals: localization and regulation.;citation_pages=794-806;citation_volume=14;citation_year=2013;citation_pmid=24136506;citation_doi=10.1038/nrg3573" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. J. Bean;citation_author=C. E. Schaner;citation_author=W. G. Kelly;citation_title=Meiotic pairing and imprinted X chromatin assembly in Caenorhabditis elegans.;citation_pages=100-105;citation_volume=36;citation_year=2004;citation_issue=1;citation_pmid=14702046;citation_doi=10.1038/ng1283" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Brenner;citation_title=THE GENETICS OF CAENORHABDITIS ELEGANS;citation_pages=71-94;citation_volume=77;citation_year=1974;citation_issue=1;citation_pmid=4366476;citation_doi=10.1051/medsci/20031967725" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Brick;citation_author=F. Smagulova;citation_author=P. Khil;citation_author=R. D. Camerini-Otero;citation_author=G. V. Petukhova;citation_title=Genetic recombination is directed away from functional genomic elements in mice.;citation_pages=642-645;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660327;citation_doi=10.1038/nature11089" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Chowdhury;citation_author=P. R. J. Bois;citation_author=E. Feingold;citation_author=S. L. Sherman;citation_author=V. G. Cheung;citation_title=Genetic analysis of variation in human meiotic recombination.;citation_pages=e1000648-e1000648;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19763160;citation_doi=10.1371/journal.pgen.1000648" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. T. Cirulli;citation_author=R. M. Kliman;citation_author=M. A. F. Noor;citation_title=Fine-scale crossover rate heterogeneity in Drosophila pseudoobscura.;citation_pages=129-135;citation_volume=64;citation_year=2007;citation_issue=1;citation_pmid=17160365;citation_doi=10.1007/s00239-006-0142-7" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. M. Comeron;citation_author=R. Ratnappan;citation_author=S. Bailin;citation_title=The Many Landscapes of Recombination in Drosophila melanogaster.;citation_pages=e1002905-e1002905;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071443;citation_doi=10.1371/journal.pgen.1002905" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Cutter;citation_author=B. A. Payseur;citation_title=Selection at linked sites in the partial selfer Caenorhabditis elegans.;citation_volume=20;citation_year=2003;citation_pmid=12679551;citation_doi=10.1093/molbev/msg072" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. D. Cutter;citation_author=B. A. Payseur;citation_title=Genomic signatures of selection at linked sites: unifying the disparity among species.;citation_pages=262-274;citation_volume=14;citation_year=2013;citation_pmid=23478346;citation_doi=10.1038/nrg3425" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. de Bono;citation_author=C. I. Bargmann;citation_title=Natural variation in a neuropeptide Y receptor homolog modifies social behavior and food response in C. elegans.;citation_pages=679-689;citation_volume=94;citation_year=1998;citation_issue=5;citation_pmid=9741632;citation_doi=10.1016/S0092-8674(00)81609-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Drouaud;citation_author=C. Camilleri;citation_author=P. Bourguignon;citation_author=A. Canaguier;citation_author=A. Bérard;citation_title=Variation in crossing-over rates across chromosome 4 of Arabidopsis thaliana reveals the presence of meiotic recombination “hot spots.”;citation_volume=16;citation_year=2006;citation_pmid=16344568;citation_doi=10.1101/gr.4319006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Ercan;citation_author=Y. Lubling;citation_author=E. Segal;citation_author=J. D. Lieb;citation_title=High nucleosome occupancy is encoded at X-linked gene promoters in C. elegans.;citation_volume=21;citation_year=2011;citation_pmid=21177966;citation_doi=10.1101/gr.115931.110" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Gao;citation_author=H.-M. Kim;citation_author=A. E. Elia;citation_author=S. J. Elledge;citation_author=M. P. Colaiácovo;citation_title=NatB domain-containing CRA-1 antagonizes hydrolase ACER-1 linking acetyl-CoA metabolism to the initiation of recombination during C. elegans meiosis.;citation_pages=e1005029;citation_volume=11;citation_year=2015;citation_pmid=25768301;citation_doi=10.1371/journal.pgen.1005029" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. Gendrel;citation_author=A. Boulet;citation_author=M. Dutreix;citation_title=(CA/GT)n microsatellites affect homologous recombination during yeast meiosis;citation_pages=1261-1268;citation_volume=14;citation_year=2000;citation_issue=10;citation_pmid=10817760" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Gottlieb;citation_author=R. E. Esposito;citation_title=A new role for a yeast transcriptional silencer gene, SIR2, in regulation of recombination in ribosomal DNA.;citation_pages=771-776;citation_volume=56;citation_year=1989;citation_issue=5;citation_pmid=2647300;citation_doi=10.1016/0092-8674(89)90681-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Grishkevich;citation_author=T. Hashimshony;citation_author=I. Yanai;citation_title=Core promoter T-blocks correlate with gene expression levels in C. elegans.;citation_volume=21;citation_year=2011;citation_pmid=21367940;citation_doi=10.1101/gr.113381.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=C. S. S. Heil;citation_author=C. Ellison;citation_author=M. Dubin;citation_author=M. A. F. Noor;citation_title=Recombining without hotspots: a comprehensive evolutionary portrait of recombination in two closely related species of Drosophila.;citation_volume=7;citation_year=2015;citation_pmid=26430062;citation_doi=10.1093/gbe/evv182" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=U. Hellsten;citation_author=K. M. Wright;citation_author=J. Jenkins;citation_author=S. Shu;citation_author=Y. Yuan;citation_title=Fine-scale variation in meiotic recombination in Mimulus inferred from population shotgun sequencing.;citation_volume=110;citation_year=2013;citation_pmid=24225854;citation_doi=10.1073/pnas.1319032110" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Ikegami;citation_author=T. A. Egelhofer;citation_author=S. Strome;citation_author=J. D. Lieb;citation_title=Caenorhabditis elegans chromosome arms are anchored to the nuclear membrane via discontinuous association with LEM-2.;citation_pages=R120-R120;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21176223;citation_doi=10.1186/gb-2010-11-12-r120" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Kaur;citation_author=M. V. Rockman;citation_title=Crossover heterogeneity in the absence of hotspots in Caenorhabditis elegans.;citation_volume=196;citation_year=2014;citation_pmid=24172135;citation_doi=10.1534/genetics.113.158857" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=W. G. Kelly;citation_author=C. E. Schaner;citation_author=A. F. Dernburg;citation_author=M. Lee;citation_author=S. K. Kim;citation_title=X-chromosome silencing in the germline of C. elegans.;citation_pages=479-492;citation_volume=129;citation_year=2002;citation_pmid=11807039" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The Human Genome Browser at UCSC.;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=D. F. Gudbjartsson;citation_author=G. Masson;citation_author=A. Sigurdsson;citation_title=Fine-scale recombination rate differences between sexes, populations and individuals.;citation_pages=1099-1103;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981099;citation_doi=10.1038/nature09525" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. J. Lercher;citation_author=L. D. Hurst;citation_title=Human SNP variability and mutation rate are higher in regions of high recombination.;citation_pages=337-340;citation_volume=18;citation_year=2002;citation_issue=7;citation_pmid=12127766;citation_doi=10.1016/S0168-9525(02)02669-0" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Lunter;citation_author=M. Goodson;citation_title=Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.;citation_volume=21;citation_year=2011;citation_pmid=20980556;citation_doi=10.1101/gr.111120.110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=P. T. McGrath;citation_author=M. V. Rockman;citation_author=M. Zimmer;citation_author=H. Jang;citation_author=E. Z. Macosko;citation_title=Quantitative mapping of a digenic behavioral trait implicates globin variation in C. elegans sensory behaviors.;citation_pages=692-699;citation_volume=61;citation_year=2009;citation_pmid=19285466;citation_doi=10.1016/j.neuron.2009.02.012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Meneely;citation_author=A. F. Farago;citation_author=T. M. Kauffman;citation_title=Crossover Distribution and High Interference for Both the X Chromosome and an Autosome During Oogenesis and Spermatogenesis in Caenorhabditis elegans;citation_pages=1169-1177;citation_volume=162;citation_year=2002;citation_issue=3;citation_pmid=12454064" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. M. Meneely;citation_author=O. L. McGovern;citation_author=F. I. Heinis;citation_author=J. L. Yanowitz;citation_title=Crossover distribution and frequency are regulated by him-5 in Caenorhabditis elegans.;citation_volume=190;citation_year=2012;citation_pmid=22267496;citation_doi=10.1534/genetics.111.137463" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. R. Meyer;citation_author=A. S. Zweig;citation_author=A. S. Hinrichs;citation_author=D. Karolchik;citation_author=R. M. Kuhn;citation_title=The UCSC Genome Browser database: extensions and updates 2013.;citation_volume=41;citation_year=2013;citation_pmid=23155063;citation_doi=10.1093/nar/gks1048" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. A. Mieczkowski;citation_author=M. Dominska;citation_author=M. J. Buck;citation_author=J. D. Lieb;citation_author=T. D. Petes;citation_title=Loss of a histone deacetylase dramatically alters the genomic distribution of Spo11p-catalyzed DNA breaks in Saccharomyces cerevisiae.;citation_volume=104;citation_year=2007;citation_pmid=17360459;citation_doi=10.1073/pnas.0700412104" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=J. W. Much;citation_author=D. J. Slade;citation_author=K. Klampert;citation_author=G. Garriga;citation_author=B. Wightman;citation_title=The fax-1 nuclear hormone receptor regulates axon pathfinding and neurotransmitter expression;citation_pages=703-712;citation_volume=127;citation_year=2000;citation_issue=4;citation_pmid=10648229" />
<meta name="citation_reference" content="citation_journal_title=Statistical Modelling;citation_journal_abbrev=Statistical Modelling;citation_author=P. Ng;citation_author=M. Maechler;citation_title=A fast and efficient implementation of qualitatively constrained quantile         smoothing splines;citation_pages=315-328;citation_volume=7;citation_year=2007;citation_issue=4;citation_doi=10.1177/1471082X0700700403" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=L. M. Noble;citation_author=A. S. Chang;citation_author=D. McNelis;citation_author=M. Kramer;citation_author=M. Yen;citation_title=Natural variation in plep-1 causes male-male copulatory behavior in C. elegans.;citation_pages=2730-2737;citation_volume=25;citation_year=2015;citation_pmid=26455306;citation_doi=10.1016/j.cub.2015.09.019" />
<meta name="citation_reference" content="citation_journal_title=Nat. Cell Biol.;citation_author=J. C. Peng;citation_author=G. H. Karpen;citation_title=H3K9 methylation and RNA interference regulate nucleolar organization and repeated DNA stability.;citation_pages=25-35;citation_volume=9;citation_year=2006;citation_pmid=17159999" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Recombinational landscape and population genomics of Caenorhabditis elegans.;citation_pages=e1000419-e1000419;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19283065;citation_doi=10.1371/journal.pgen.1000419" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. V. Rockman;citation_author=S. S. Skrovanek;citation_author=L. Kruglyak;citation_title=Selection at Linked Sites Shapes Heritable Phenotypic Variation in C. elegans;citation_pages=372-376;citation_volume=330;citation_year=2010;citation_issue=6002;citation_pmid=20947766;citation_doi=10.1126/science.1194208" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Roesti;citation_author=D. Moser;citation_author=D. Berner;citation_title=Recombination in the threespine stickleback genome–patterns and consequences.;citation_pages=3014-3027;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12322" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=O. B. Silva-Junior;citation_author=D. Grattapaglia;citation_title=Genome-wide patterns of recombination, linkage disequilibrium and nucleotide diversity from pooled resequencing and single nucleotide polymorphism genotyping unlock the evolutionary history of Eucalyptus grandis.;citation_pages=830-845;citation_volume=208;citation_year=2015;citation_pmid=26079595;citation_doi=10.1111/nph.13505" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Singhal;citation_author=E. M. Leffler;citation_author=K. Sannareddy;citation_author=I. Turner;citation_author=O. Venn;citation_title=Stable recombination hotspots in birds;citation_pages=928-932;citation_volume=350;citation_year=2015;citation_issue=6263;citation_pmid=26586757;citation_doi=10.1126/science.aad0843" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Smukowski;citation_author=M. A. F. Noor;citation_title=Recombination rate variation in closely related species.;citation_pages=496-508;citation_volume=107;citation_year=2011;citation_issue=6;citation_pmid=21673743;citation_doi=10.1038/hdy.2011.44" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. S. Stevison;citation_author=A. E. Woerner;citation_author=J. M. Kidd;citation_author=J. L. Kelley;citation_author=K. R. Veeramah;citation_title=The time-scale of recombination rate evolution in great apes.;citation_volume=33;citation_year=2016;citation_issue=4;citation_pmid=26671457;citation_doi=10.1093/molbev/msv331" />
<meta name="citation_reference" content="citation_doi=10.1895/wormbook.1.101.1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. A. Thompson;citation_author=L. B. Snoek;citation_author=H. Nijveen;citation_author=M. G. Sterken;citation_author=R. J. M. Volkers;citation_title=Remarkably divergent regions punctuate the genome assembly of the Caenorhabditis elegans Hawaiian strain CB4856.;citation_volume=200;citation_year=2015;citation_pmid=25995208;citation_doi=10.1534/genetics.115.175950" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. J. Tsai;citation_author=A. Burt;citation_author=V. Koufopanou;citation_title=Conservation of recombination hotspots in yeast.;citation_volume=107;citation_year=2010;citation_pmid=20385822;citation_doi=10.1073/pnas.0908774107" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. R. Wagner;citation_author=L. Kuervers;citation_author=D. L. Baillie;citation_author=J. L. Yanowitz;citation_title=xnd-1 regulates the global recombination landscape in Caenorhabditis elegans.;citation_pages=839-843;citation_volume=467;citation_year=2010;citation_issue=7317;citation_pmid=20944745;citation_doi=10.1038/nature09429" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Yook;citation_author=T. W. Harris;citation_author=T. Bieri;citation_author=A. Cabunoc;citation_author=J. Chan;citation_title=WormBase 2012: more genomes, more data, new website.;citation_volume=40;citation_year=2012;citation_pmid=22067452;citation_doi=10.1093/nar/gkr954" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
<meta name="twitter:description" content="Meiotic recombination creates genotypic diversity within species. Recombination rates vary substantially across taxa, and the distribution of crossovers can differ significantly among populations and between sexes. Crossover locations within species have been found to vary by chromosome and by position within chromosomes, where most crossover events occur in small regions known as recombination hotspots. However, several species appear to lack hotspots despite significant crossover heterogeneity. The nematode Caenorhabditis elegans was previously found to have the least fine-scale variation in crossover distribution among organisms studied to date. It is unclear whether this pattern extends to the X chromosome given its unique compaction through the pachytene stage of meiotic prophase in hermaphrodites. We generated 798 recombinant nested near-isogenic lines (NILs) with crossovers in a 1.41 Mb region on the left arm of the X chromosome to determine if its recombination landscape is similar to that of the autosomes. We find that the fine-scale variation in crossover rate is lower than that of other model species, and is inconsistent with hotspots. The relationship of genomic features to crossover rate is dependent on scale, with GC content, histone modifications, and nucleosome occupancy being negatively associated with crossovers. We also find that the abundances of 4- to 6-bp DNA motifs significantly explain crossover density. These results are consistent with recombination occurring at unevenly distributed sites of open chromatin." />
<meta name="og-title" property="og:title" content="Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/6/1767" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Meiotic recombination creates genotypic diversity within species. Recombination rates vary substantially across taxa, and the distribution of crossovers can differ significantly among populations and between sexes. Crossover locations within species have been found to vary by chromosome and by position within chromosomes, where most crossover events occur in small regions known as recombination hotspots. However, several species appear to lack hotspots despite significant crossover heterogeneity. The nematode Caenorhabditis elegans was previously found to have the least fine-scale variation in crossover distribution among organisms studied to date. It is unclear whether this pattern extends to the X chromosome given its unique compaction through the pachytene stage of meiotic prophase in hermaphrodites. We generated 798 recombinant nested near-isogenic lines (NILs) with crossovers in a 1.41 Mb region on the left arm of the X chromosome to determine if its recombination landscape is similar to that of the autosomes. We find that the fine-scale variation in crossover rate is lower than that of other model species, and is inconsistent with hotspots. The relationship of genomic features to crossover rate is dependent on scale, with GC content, histone modifications, and nucleosome occupancy being negatively associated with crossovers. We also find that the abundances of 4- to 6-bp DNA motifs significantly explain crossover density. These results are consistent with recombination occurring at unevenly distributed sites of open chromatin." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/6.cover-source.jpg" />
    <title>Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_lzAdnIqxdiSkhG5-xdW-oiRFsDv2jyeIJ-8jbRmXFU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460388","apath":"\/ggg\/6\/6\/1767.atom","pisa":"ggg;6\/6\/1767","ac":{"\/ggg\/6\/6\/1767.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/6\/1767.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1767"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/6\/1767"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/6\/1767.full":true,"\/content\/6\/6\/1767":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002213szBaw5NmGJR3N1Lt33l77Xz43VNgspk7CT\\\\\\\/TPc8fr2WzLImR4QApS46qP6oBM44HUz5j7+ibFY\\\\\\\/mZZfEecOhIwm2FTw\\\\\\\/7p0IkyfRoWG9a+a4KYr2vvejEeNPJuik4CZz5QlOSJQp7f\\\\\\\/VGpmikf2FlX1HhTLZTxOUsFa+8piT1ktpbexqOky9hi3nR7A86uFaPCgQKlMshUrHivickNELHgM8GJXKxCRPS99K6dukMHkrjYus2oZBhhSQnpe+20vsnodPN5Oovq3JsqROnD1GSS7QXT8SZ+3cqBzIK5dNJlFpdJW6eCZ6sEWK0qTqKA0tIRz4GP99Y3hYzxP66mNB6Zl\\\\\\\/v2XZRXHUNc1SUIrPuw\\\\\\\/DS3nwkucGpsuNdrxaf229+1Nd8nE\\\\\\\/KpAAPb7zZp4PIfE\\\\\\\/yJAwpnICGtV2MultMYNUwbMAlwZBLuUxHIzuuxX0crWzEk7Glv1oGf+EYI83uOguD0ZIU2z01yo937\\\\\\\/HWPCJ1VEbbGdxp03D\\\\\\\/lnHLrHTxRpH+xMNl0jHogpEtfHpG2WJjy0cFQx0N5rdXY5RbyXkcriB13IwFsZlJfHq1JrvBLjpQ4FVGiZCz8HMcAlCt0F5l44JUGnwimDjI2s6Lc\\\\\\\/D2ecTMXhEKrEDrd8ZWhisPB0atu2trxO5WTl2aAXUETEo5H2uRh3r9jL2L5HINv70JHcxvvN6+9VAjweI4iO3j10+Ai1tRL2\\\\\\\/LOXQyjifaZ81R9hI1aJq3OLX1lREFTU+oKVg1DozKTxHu84ukjXsIF9kfkA8Y6H+SWvHBg\\\\\\\/1mkG04qxFCBt\\\\\\\/2oZq2e\\\\\\\/AXuOQeQfBtWp\\\\\\\/G6m+zX6j8yu9YX8pUuxW1pWwaIJehV+7oU5qGfEaFDQ8ptDq\\\\\\\/HCVKZoEHEx+XIf6NDXfuOZIFjYsP\\\\\\\/u99XTJaSftDoiLgi7XfYtbRlCggNYPhi9knUfo86o\\\\\\\/jZCa9+Tc2y5vnjRMRrw80XlnYH6vSPIHGGMQ8SSedUO+xbEq4qvVZBA0AYo0bny6Ewg3gku7jpZXun8kbok6TxZnOSeQ+3LubFD9+ZXVPrqGg1fSZAXXw4DjUuL1tkgSnDVGhOLlR9zC2sBOD\\\\\\\/C6Zdt6flDtMTJrXsjhkeGiIVvdS6HNoCtPTFZEoWUftGQuF3A+KLO\\\\\\\/3AxsUa9msE4JoMVT3vkcnvImPFrrfw4T1hWoG9uovSziY0SKNhLbkcO5XCiF7qZhqADd04sVxB\\\\\\\/xJA1Z3HpI5SaNoN6RrggjYssPYA\\\\\\\/q6DmJ3WQ531IQv\\\\\\\/tOcZ6kQ040WZwS0enKe5\\\u0022,\\\u0022iv\\\u0022:\\\u0022suc18rmpu9Wf3SyE9kGoDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eb7c9004d7b7cfc09222422f1da935042c02191a4c7ecd766c9a3221a828319c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kKnxhtL+HJ9lccuPDeA83Rg5hGD5V13CigOFnKDPMDeGPWh5likEeiqcHfLuel4PT6ANoL+21Jhfm3Um9zgJ9YBniWyMPs8rScrG\\\\\\\/GstO0LxJH99CvKfM88iqVnjRqDK8Ma\\\\\\\/CU\\\\\\\/5QTGXrOxWi\\\\\\\/klkC1QxM6pS8GO++OYCiqUxhWy3MkHIMEzKeT+l49g5uGgx3bHlm\\\\\\\/A25yPLDDBIu4sA\\\\\\\/7Gec\\\\\\\/ceQXPciS3FXCDD1QXM94TwranlJlmgwgXAu2HxEA06PD5eRcuZfbIlW8XPYl2KpMHB1TtV4YGCgJjM724AEPcjGZP0tl3I3Q53IF4J5pQF0iK3lX5wnkHLrHYZq\\\\\\\/z85+dRGgT3Pq0wcQNWLYYQwEYubR7B7tcq6tDX8IBfvbgOrWYYYttiHNFK0kC8Kmj84EvQ4daEz6QoVCM8kGuCbR\\\\\\\/Fy\\\\\\\/6GVu6yi+NM5TEBwT29uowcjxR+HjYbWIabVWu4CrjiCPw0LMM8RDj\\\\\\\/tqe6mVYn97gc8eMQi\\\\\\\/3gvESpstAC35GEe+N4hHS8QkoMarvbIL0JRq56xri4dLC376TDe93+0BDc8SjgerZJRbYrggAV3AAD93BkXiOkbHCUI12HZKN0Y9zunDEnunQVyf68cUa7q8fkDZeFfKFcBev1uPsYiKn8lPCkpYsyqX5xCHQMVIKETHGpftyTPDCka71Gz8tR06JTJBGvTMfo\\\\\\\/u6Z\\\\\\\/l7U6NTnEv0jFr\\\\\\\/6NZSbAQTneq0wNvAFWHbnqBEm4SkoavnQ5\\\\\\\/G+PAj9H95HU1TuZaydmvHNTfnFis7Hx2+UJrGIZMvkYMcCyWdBCnArgDytQKuOMEJlsepJxJbYGK2\\\\\\\/HrsqDclLKr7vwTWPiXw9mkH+URLHtsYuiFx+SUmtEatCj0fbT3NVIXww++u2w4pBnB6DCq2Vw5kVHN0lIma8a8i0G5krrjU28mJkvTRAQHJdQqxTE\\\\\\\/AhyRFcw\\\\\\\/AZZjS5aVopS2kldMSYvsWx16p1G5NtAlDiwwzWnu8ud\\\\\\\/tjrteQghD2JOw8zNzCDl52WPYpbvRVfv251OUIlhiHeo1r5MBVVJCpxFtC0P5QaLBLw8Muo8XHSJ2XvJHtZdYiBBiariS2Q0uhXty43K6ffaB3nXCNEHoL8BFs6TPm975Dpah1Mg3XnMiApgOVtCB1kwScjuPZ5zmABm0JfjGS330hCSgNE6OwPXszQKWo7LKx6xpPa9FMGXDvssBG78KrBOzbX8MGPIY1eZU08YOyddEzuoWLdT6ykhYoln0llfXER9ouemRR2JFoylSyLU1HN6LMg5toH\\\\\\\/CMsu\\\\\\\/CZAMAkjdC\\\\\\\/3JxUbU35CPrUiYDocMqbg9tjI06GBGnnnucM16+5x55kSG37MqDPh5jCmMofTybigagvNwUD32wJZOkwtAI24dUwzs73j0F\\\\\\\/WDEdtii57tfKoGxpXjaSmCZbT78afHk9O08afT8LLmozyhwfxc2Qe6r\\\\\\\/l5iyK6UrR5UF4iOz+lWKtBGSexqVI+mNT+1IlvsKREYqAVt+ogEPprCyvnagMDWmMt0pWhAt3X9CzsmxrKiZzlEV4u5Xdxaoqa8du3SB+DdkEd1tDaVfPrbUYw0z33zU1mzHZMu+A3y5mVwJGGoRHDLjCPH4k08dM2FuAR2xkcbcZaIqx4MM0T05OnY5cMnpybw3ftLIBDNcknUzeR2DlF0dgZcMThF61cCLdLAAkexE3+1rice9UmgCnBdAmZ5sdXDPO0U2fq1XdLs7BMKDS1ILrtn8VEQJMZjrojYqQ3Lpy5Xx5TVO6CQrwcyh5O2YDU0fqUUhU82LHVYourlL1hk0r8\\\\\\\/zna5SiRsWXygXztpzakAKY7ucXI333VDZrajV+YoV1\\\\\\\/oeOxrlqxAfGpbllr6XPguZ7hOpZrv9Eu0YAOP0K5Xiqnd4N8w+Z8iGnFRpqUGykD3uwCcOzR3fLsEFWv40mMuo7WMqC\\\\\\\/tUkhorq3W2ZOLZoG4QAv6DUFZ5Xc2XViYjZsKbBuwN5jX9x54i+jQ6Gqbb6VWHxFTbXvE9+QR9crD0L0TC\\\\\\\/pG6FGvRwpH1DGpuiobCPJQfa7HZQGzJ\\\\\\\/88lO+A6BeybsdDX5wMbg\\\\\\\/WyAuZ0PWrwvAcc0udaK4Vpit6d432v13cKqYRwOhQuCc9Zu+vDDT9TZ\\\\\\\/QyeUHzjUDNUlCln+CdgFekQSeJ\\\\\\\/HK2lyPVNKhErIeBTkyDnegJxZLKryGH9WJLagfoqJtPjh58koxfHHUAkfys8EF\\\\\\\/13aKIBpQk+rByzTwKwZc6EdlZoA4zSl0rPA\\\\\\\/d\\\\\\\/lDgaGErueA2lOKkPNe3+5lbRgmkTkLdTyYfNpMihjeZIYD8DF3S+ZfeC54+vWxxO6vvF2Ut+OolooKTZ7u7FQz3x8+Hn4MfJhXhiawceu2GWEZq\\\\\\\/6QtGj8\\\\\\\/6wjQ4CHyCA2QW5V2hptdjnfkHIi\\\\\\\/P0LpdMnwZggvCUfnb2ET3SwFOf95YcYMAzXYEf\\\\\\\/xcHnd3v9SFE8gIjuXJvV135AjNzQD7rrZCqmxPjXxt17VMv\\\\\\\/eh9y2MFUKzjGqORua6bmbD\\\\\\\/lvrrnvJNiGZJnzLuOnBFW86c1URVZvVrr2+arqctPRflFiWXmcBDpz70Szham8\\\\\\\/XnIU635HQ3G2NRH56V8k22nLL+\\\\\\\/3OqbKYPWm2DTHY1YfJYic43HslNf33MpVLFUBUOD28MCcwZAZ2aCzs7e+OzGUuCLkzsKV2HziTti2sitv1zfgOyjzs90BmuvNo\\\\\\\/duNbQijQNY8zA1kHJnbQUcr+jDKK99G+AxvY6ZbpSz3JTpwD8hMckBp5c\\\\\\\/h6vPhYPOZm2LluTjfidE\\\\\\\/ClnQ7bSf+C4DJXha1XSlvYgFNnJpSp6zrc5ic+oR5aNdlFHMO4\\\\\\\/53FQK8WYfAzmZX9khyzIfrdsg8nXJiYd691EzLPRL0O\\\\\\\/nTXXd1LcfJbSUZTBbqpSHUXsAWdkOxufAHCJPdAQNOdmKVhCOn7bqhHf59nxDuxmZKtD2qBBWkano77gE+3J8idOkk\\\\\\\/zkT\\\\\\\/VKbQ+oQFtKfB3TzrkxzJzFI8Z4eNRPT\\\\\\\/G59kGlN7rpryAh5cNPkCN7vvptNid0NvqKIgaulM3fg7YrWR3A1Aql3K1wMPraJQ+\\\\\\\/vmXgMIDGpWr3UK6eNNqIP4VPF0YDROjrD3gQpjEYGOudgpx240u7KSAxlMAQ+jevMegxrDb2N1THjd9IwxX+I6jwmrCCSTi+iZpjbY0yxYRK6BzCWkkk\\\\\\\/hE2w95R\\\\\\\/ZKSEfuiqK5uk+MJaplNNsIe9ioVePZTActdh3CuPZdGshBzFgdtKNpHLp\\\\\\\/eKMSRA511BO7\\\\\\\/7ziDSkPltkcGXV5kDTBf1DEi2yJVB\\\\\\\/ZE90WX\\\\\\\/gmkm3hT8dFd4y3gb56Iv\\\\\\\/3nfjeyzasVVVNRpx7ORnKsiY2tnI8SF5HL+n2x5WAkmHkbsjX1eavEXF83Kb8M6TqqiTnLzwoTCpqxCuOl2uC5Ss3j8vtiJ\\\\\\\/fJgjgrXyvvWmNAX6BPNSBi77CeQqLUwFN4nZ377XRI+eZehSR52niED5Uw3LOGuTQujjWTSND1kZmgSBSTUun+H9DJhM\\\\\\\/+z89GX1A5VOTCw47bMepBJXmIorOU4MmrhBzfInKIM7dAQV29+CS2EYO8hw1Ff7aFwD+riL6g0QRZhqG\\\\\\\/T7Y0XlDkPlZeJ9LC1ybknjv0BKA9jhlfnbRta8uSd\\\\\\\/ygwPsojw6dKKrfLEawVYfBZ6jaZ8k2hj8CgK74cKuvF\\\\\\\/sd2cc0pfTEXt+RSuZoV6JvuG07uj0OEkL2DaR6AYkjJtfUz8yv1qYCeibalx+1mAZOfaeMC5\\\\\\\/NG8ZmEZgI20olHzoKG9K+5PEZnZax6fq9oWXsj3PSJuu7fCpRXUSYz8vUd9h2sBDRcFSaJ3uCfRwgWgWrsZAZez7z5vFUWEdhOeMdK76mLx\\\\\\\/\\\\\\\/giATbDZDdVSG4ewGz\\\\\\\/7ZSJdowOsHoRaDTmtmTyazRRIuZAplG59rqwgiuJPwlz7zLZg0Yxq+K2fatF9iw9sNbETsJpHWctLq2yDNKKRjD8pZScUQxGwD\\\\\\\/NQ23GrWPAu35ODjilncELLoO8DBFydnbLq8n2S3CE+XQ5RyaEsgXuk5khkULYBA0uMKvtk1Wny8eY+s4FLAiWz+kPq\\\\\\\/2r+QE2XlHzh8KEUaW26czSjWX3jk8wUs7kW6ZfMKPyz6SrBWLRDhARBpoSRQpRQL2snYH5lThWSjtgkHWz51Bsfbc4DtweiVd7ADWrNUTREvBZ\\\\\\\/mLD6e6svcMXekxaW7cRkiKg3Bc\\\\\\\/5gR\\\\\\\/BzMcY5sBL19l6N+QyV7HP4kyf2QunCsc1+qKqplJahyR4jxImQtdOMHR1m7VEWWZaNAx\\\\\\\/sroMX7G3jpmtZfPanMFd5h+pBX5UcBdZLFp66ldcOEqVnwjXBo7MotKAtXDrM8IjXJK2+hb7Hem3+J0yWqLyJSNbzoHMIIBGI5D05EKnXHzZl6GcpY1FczWraWAPEaImXIdQ0BhFHyi8JmyJEU46b2fRdXa9Ij1U5YMPJuYHj4Bq9DhEyKjLXoMawcofXrXyXFW0zisdjJJvu3f7FndcZkDYyXTlZuL2i4M82Y7V0l3vUkKwzFfk\\\\\\\/EZZvmr\\\\\\\/wAvdkcnqJ5Dx+b6313JaMa7P59CTbdRuVsQB+6Onjd5pB6p4SIzGLWmH3c2AT1OwB\\\\\\\/\\\\\\\/CUDNwWwK4e\\\\\\\/Zno1+QbaxMO5tIW+bh+sKRfO6BJBySk0uPrbtQNsVkcI+dch789TL0ozBedlbWMFHcL9\\\\\\\/3pf6N\\\\\\\/O\\\\\\\/CNLrZAoAtvbUwlFARdc1iKWNHpz21itApAffcKleFlR0LwWy\\\\\\\/M2moEItJgPt\\\\\\\/R9\\\\\\\/utmTDx5Se8zLkiz+RBd\\\\\\\/VFngtPlRinJkmA+rp+j4bjKE5nWW9Rl+qvI6v7xGqUTFTHzY4TLLz0UlfsnVDpIP+\\\\\\\/ZQ+3kAt\\\\\\\/\\\\\\\/TvxR0sjAo1mlfsfna+l4d10xXWaXB3oQ6VGMzPaUUaJwt+TQr7+2JbTQv3rBxwlywQjJzwt5QbcKRd2VWCpGgwiQzMwCR6m\\\\\\\/ZH0LmKCbaY1oKGGc9\\\\\\\/+vI3yJpRk2V4D\\\\\\\/kyz9H7bZr6oamHD8TsTORwSrkuIizztYcMEeG44LC4BczL1SwWQD22ywfW9eyTNBvUaEBhyjboJeTcHYckFNrkv\\\\\\\/nWNaWhV\\\\\\\/4eKB6\\\\\\\/5gSwGGvMiQSAvGVywGi1Ud0TZtdnKoyDu1bVEgjgPacsUJbaaIc8gJF6bqzZXa6cGbzKl0y\\\\\\\/ZoN0Lsyk0dQgsFkmuUGeJur92iXX3rpcMEbJdN\\\\\\\/ntp+U7v0UPjhgNC1+\\\\\\\/DIDxDmMhQ9KPB2npldQX2NyfhQRO2xMOWEKcaG3KGPFP9ZH3KzXb88qYt8H\\\\\\\/8E+IiAB1OU+WcvgACLT3+sr+nqlSHNFLniHbGra6o4xozcA+imrQ67KGfENScy+SfQwWtFZykmVdhUO6lLuCgy+AF8lfHuWISkANmpsnqKDjphC73W3lSot0uyLcOwHxYbred0aKd9+RI7A6UV3GuL61nzXesb9yFnI7MaX3wKyOk4zt2wcZrEB64GUvWUpMgiVlcqiT0gLU4K+aITJA3EpduF3FXFIYhjDOwT8SkblrZAjJPkvYrw+m42g0UrUBKXB8QBusf3hUWIgNm8jUALfmD+CyvlUQJfXj2KvxOys0vCPlfZP3HLEcRTNGXLi2oLQEmFZxZlsCjnDqdUB0muLohDPkM5nFliowB3Ohm2RLPLpypCi9JgAmLfg5bP3TNPa8\\\\\\\/\\\\\\\/H6TWChbudypcfsigmiy6CPk6gTKq\\\\\\\/sZ3IrvhYru0uzNQX4ILFBrrd\\\\\\\/L06+Co75QbPJS\\\\\\\/RCiK84r8XGdj+QiJNhgH6mVOEt1S2nSfeWHGasQ0Ho+\\\\\\\/V0chV7VZRMCEjUYd9YnJYNpB5sgq42FZyuwC3wP3w0FO9ewvBbJ3yJ1hTAEDHwFQD74f4C7eM3c5\\\\\\\/skT2VKtkGkkpvSt1mnVMWeS\\\\\\\/0OEabLMzr2KeBF2qOaESP38LNKvmN2liNZ454CSz2XAZCqtD758J2eMTjn61vyzWOAhNNVlRjKyvPoNWlTPJ3yt\\\\\\\/yCnXVaHKNlHnFbpXrGr5J\\\\\\\/cLNw84ERVcMXx5C5reftvtMaXRuTGk5EBoSrUZA13ecfKb2T+z0a2kOcsgHxVjl\\\\\\\/h367R8e3fB4wMBrDl\\\\\\\/yxBsytShwyWyBbHW8qHkq1i\\\\\\\/HQwQUwc64rHhvR1TewNrgJlpsY8fXEKECPBAxxdHy0QPg0Pq4YXgnnP5GDlFOnzq8t524M58tFTac2Sx6C2GnlfKlWOWhaqmeq7unVb9qq3sT7nwufHGKcIvV8ebAgJl11vfe51sI4s4c72K4GQ\\\\\\\/BgV3IfUq8U9D9PdGfBJVzI\\\\\\\/0Xz+4i3W2ECx\\\\\\\/f0DSQ2OdWDZoEZNipuPpwWDJWkGi6T2rpHGISbKfLa7t9Nv7VfDNZAfRjRshI6IAxhBXDifxiZQJSyG+Zq4pi+o8OIz4sV8amBottRQeIxUhDzw3A4bc5rvY+MliEb15nCICDVRsWs0THiosfvR0mGL5QB3RDJbR+YvA5UlM8b8AI04TexXXAMXA8dKueXqN7+D98ZdmnW2G8IeXupUr82AT2zhvdHF1PX9eoLP7Kecd3ssq4rKeBEv8k5nz8GZkZjbaWC9ZC4uUzVSkQ4KDxRulasPmbM6pY2V7hsab7zfJMFuu0E7DxuRkIIr3NmhnHJYmrjo0vq6c2Mqa6ZrUmiQLfCdaqX2\\\\\\\/qZxAKWqc+zy5OeAD5UtPiipRAq50jpRHYQtXjcQD8ktliUIN\\\\\\\/DKWLiD9gRGYUxWR3RwKbxu0GClmug3c58NIJ37x1b5HhWHx0NZG6L\\\\\\\/UBdFiqVtJrXiRpDbxR7nhSA+pVdIc13ZZwslcddhlB2XiOCG6S8LFofcVzFezSiDZ8sT6zlnAt\\\\\\\/Op\\\\\\\/dO32sh+kxZjyR3hCjCKGlnwh8odL1XN\\\\\\\/\\\\\\\/KIXo4wmS1FRtI6E0SO5YJN\\\\\\\/bIiEXmo2YZkRhnUeoVlgbjETLlbL1Y8ZdhSUR4Rrgkn0sggKj3Mx2YmKjR5F594BW9sopJw5vDiczhz0oWX0LEPZi+mjwDQHNAhJkjDAfMyMbxfwg3EYSybjq4K3JxMsAYQWKpevXbrAp+HxVuUFkfflTQCyBawVz0fzbhEn4BiNKayf0WxvKGqFHezaRdwe55xE\\\\\\\/sIMKDNl6pOH6oFAPaOZWH4LK9CfBBhIdmcvzxcFPWhhkUoxT7C9EcRvSuj\\\\\\\/Xvt6KIl\\\\\\\/u8gh7wA\\\\\\\/N9dexvlP308aOE\\\\\\\/vqrpqxD6QSJIb3wyYF7ro8+yPavviBkiB6QTCUNGPjXAx7rB4Vrba76RHuz6JeAVZorteW9ez0hs7yvvJMQkzUNZVdA+TmlIjkw2Z\\\\\\\/0QH42tlae6T2wJg0+Zya0aZSzypbdVJ0BNkOxukVcn0yBSNAok95\\\\\\\/LemwDaUlD2LZhmEAOgiN49EKqtQixj+fqINLWYL9P+hVDcWHR\\\\\\\/WWU6rV\\\\\\\/poHNqRbJH+s16xOwpVO\\\\\\\/Xa+LFvNbWNiZhNuKrXGB8uSCmDGwRjm0GJSm3lHhbZQFO6KhtJm0Y8kUae5IUGkjmh1T00pRgg4DuFAPyvY\\\\\\\/vS8B3Qe1lORQ4dhqfvX2pKRwEGmsjTX4vAuNLjCciuMlObJuZ40WZvSx8QJXgyWdVzU7VK8aIqXtP1zG4NmNVRV4NUFayrx\\\\\\\/LnrXHBYr36ka+qfTpDC\\\\\\\/lmZ2lPFABM87ivLtramay\\\\\\\/0M7PTUxVzpAwz+HrjMNgy0Qu93wliPDry0\\\\\\\/gpU5o\\\\\\\/azEgUyX7280kgnvY5jyPFEs4IAijDJxWlXgELucILJVSvGxbYo5lNnkq6dpnk9p8GKaW7\\\\\\\/FW5K3FRMldsniKEhB111KpcHkN8UgNlHE5etPVupq6j907E6Pe4a\\\\\\\/+wyytzpS4Efhji8sEOu0RXxJmfk5b5U59a+4\\\\\\\/1+Eyu\\\\\\\/ALjKveo4mJbucxaAINcfmxQ9Bb7XuzkEIHUu3NfHwPfJk2iCVqWM2KA8to09Sg+UVNW9OcXKfZ1PF\\\\\\\/Vm8AGmMuYxciJHaIjS2A44FJ6sIWR7sWwuj1WoU+oQ1aKJDbQzup0Lo40+z5bupqj3qFg72Hp36CFCilR7EzEwD+i7UnI0+Rl6DTNfWjSv6dSz52vJfST00iRwWWU\\\\\\\/T4Co6OBjUmLvclWpKSfYfPAHx5jqbioOIceAR1yWOMyyUa6HBFJr8tuXkVxXOMOTHx9gQlewGQGQZJmh2MSxMqfwkyQfYs\\\\\\\/RS4MB1ZPs\\\\\\\/4UiXLwS+WOGNbMlvXux5Xr6BWd+EC7WSksNkNN27xFWAUjebA26hn0Fqr6e8ytMxJS6XAdP7U0M7FBnqm7\\\\\\\/\\\\\\\/L12iS2r6qcQjjBfOc20GYNpL1uU07tzEzyVOX6rGihcPnb75y7bHWAFFOQxOPlfKOFhDK9liFJJbRL3JjtZxOlFyCEX+264+VeQvy3verukUWsb7XT4jRI2zYYJrVAZGisDFd2SAMbPdRLiE09SDwXJmrYHYDObhky\\\\\\\/haTQIGdECJmba2jhHas2UmAWpZ6RU7I92UbhP9m1EFoO61LoVgPoYqTtaYogLmX4ApsjoIOk7WVgfXzHBINswOISkSqT0cHiNFTFLpCKiqM8vJiIZ4HCT6wrZWt40fDMmsOnlc+ZQPEAmrUuBKGNB+FOKma\\\\\\\/7ONvqogE3EUKvvGT6x2wPAfpl9KRcdpGv6PGU11G07I8x5gHCqLq3ygElXF0gQuP6CQ8dymhXzBLbAubLBkwkRr+YSMlx66pJQLVcwYoXQYdp7SpMZe17BhCGnwNmua4ydWxReJ8qRSeLiXfpl7Zz9\\\\\\\/k43EHyhB4XFEA7wEIUx10ZxLP24LubixCH72j7\\\\\\\/2Rtipq+DJcebKArLom3VFzsM50Kt10nqEWdpEpyeZ\\\\\\\/NnCPx2+ZARhUz\\\\\\\/+Gw7rm4+zQ7FCnI4+tgQYUJKbPb4KrfqtOUoJVN0oS5NSkT5FdkVAXm7t4OAlp93WCXTGzobF3XDDJX4+Z8f6ZKQ\\\\\\\/9EvBdzVvyC6c0oNgPRgk24ZlRE\\\\\\\/04HdXhWTovJmIKZ3KymDQIjC+toj8zgAIKzmEWtH+bIHi8qMQftj\\\\\\\/4VvAn24XbY9NRVDoWLNTYhgM5W8mnBbXmQUKa5Hw60ANQuu+NNxCkDKT4x0DwaJVC4mXp6MVfVu3kKULedjWjz169+keg9BcnlPbRAW33jzgVro3fFL8MMriv5SC9aT0d9RzRrAxIoxRRNIrqSG2LVJRJ1\\\\\\\/pDiISBCoWwFZH\\\\\\\/5SUWoGEgkrYQu+Ayg4gJd\\\\\\\/NEKhqNOQdVGLUq07iMrpHnVxQ0VZUSyoHHUvZiOiud3epchmRo9DlxciCQ76gQlPy4TdgrdrQRrUryPIxpUMlZT6nq\\\\\\\/R1gQrtZOOIC6Cjfg\\\\\\\/TfxaO8xdXHbcibHYYlk9JkpgHks46G9EhgaHbsp\\\\\\\/8v29oOIDdlESaBfgmzFCvy+76Es1B53rFHSNmbHOLMBV8tcWMH4td5tOvp+pf9GYZXFhTMmYaGIDtbgQ+UHJkVy7faD7Z\\\\\\\/sj4j67IUnxQE1HvIOyEvkxGGIK+nvjr3phz634jc9088mql7DG7IRFgphsNHAeZQ5WH7K1Wc1iv2zewx2svF5g5ptKz3wtdEwBhokAtXfdhxPilSfMuTxWnQbELHaWN5BVTX98LyFSfkgWI4XwOt59CzmL6AC2YxZ9bXMTbe+yU2PP6+jw3a6p8jIVTrOla6hfv3UfSuPdwnyAkqU4ojHExFZuWHUgdr+DO4Z0a5iZxQcOYV2FNs0zuicg9VycO8b5wparZ3Pv1hmCT1tpFioEHa78LV80IgEIOoIOxtdxOM5nY40p2tTB8fJojNiexrQnpmfHUi+LJ\\\\\\\/a7Bdfv75aTFGNBBMcGWbHyPjramFYBnvgtRoqqjKFwqLIsd07S2XlWl\\\\\\\/qW343ScerVlP85TR1wBj33cLHuc8h6NChAoKJUrul7jHwJd\\\\\\\/nPDCb7usiZAWPE96MZJ5tRDBi3LYkMzaSPpOzOGvgh3QGOkK2Mbmpds1NscyLhUo5MGkfITHfAoNi\\\\\\\/w4uihcB8enJfuA4WFoKzspl+gaj39RgZPtISUrBjf8AAr4aLcg1cI12pgFm9HIJMexGAeqrLFD7Z6A8GfabJYGzJIKvMCPpJYLQlymEBvZ54XRKroQNE+5MU38YjFbDe4syTP2WBsFomJHWtj0kkpP56JaAR\\\\\\\/rEthqohm+iUL7dCgLV+dzo5jo6MMkwuLWhHma8yMm1ZgQ5O9HSmx7usjqz8KkrhlsOiyGJQiWg\\\\\\\/O6VAAsVFJ2fJpy1jAUasPwKuz7ET6WCkrN1symNpYVTwdSdPJjiwy7YQIV1y6f6jeXIWjCmeJ\\\\\\\/r979RWZMRN9ngMD27e8xZODTmoddLkJM2MXdzrZ+xIkRMGTAPXmnudrXmUoW2NEjRVkFYyHjITPFYISlh5eNwPsEiVLly1ThAiuZHUfFI+rxjoQdzbeyppEzaa+iJONYaxgocnIVJKyNO7b4\\\\\\\/zVp6vh1CUIMeBabTpo2xkk1fM2kE7EAyWsZwX7a825VWqhx3CnmYDJTdRGVxlky2gT\\\\\\\/fOoWXyiK4EhwAm4Qi4tzAZI\\\\\\\/9Mdbg5kv\\\\\\\/wDEn56qYvp8L6g8LZ4caq9o1ad3bIqgI4DcqfJa6PyWsAgcHEI3QfRyviQn8ufjDfS4ACizC1Rg+\\\\\\\/qcjtEaU0zPiNLtH+10bGMSkm4zfJtVYmIBYrAVpf4Nd0Ubs3zG+5JuYpmXMu4mgvvCO11zdJe7NRQW3wn3sfxrJtvyAS5Ucry6cTM7I6YPwNzdEjd3bBwbYZARvnhOy5HKr21\\\\\\\/kaUci5t4utAjctcQD9v8SXS86giVsKq48n9GP+nivZZ8t67dLSSdTeBtOsZ5Z+wqXoah23oe0653445NJHknfdG9ZX+l5O2xnVxmVjPH0T9QRqn4z4GoPKQSgT7n26k+qiFsBbFwIABJQDfKP29Y2HyV57p29J8wA6JEoT0ULMszxVvepzgVwwFJ74PHIu\\\\\\\/N3jz\\\\\\\/Hi6LowFPhTdpS8rgvQUZv\\\\\\\/+I8QZXU4Po2MnhVatigFY\\\\\\\/qq7suyu4dYxgyNAwypcBMPYzg5wH9nLDGCBmMTSh+kVoGlB77Js3YZZCCIveqh19JH8JD9S4yC8rCRwgAuXeR+HEvbvMV4hEMzKV5C5q33kDzkDX\\\\\\\/9TOAJJ3a0TfJFomI8z2oR3I\\\\\\\/8DOtZRYpla1lnHxnr5wHlZGE4U+9v9VHczBMIj8y88u164WMT9gkwZbM01QUKKmODoDtI0FxjY931qu4W+nh4mMnaKgdg1xNJafyNiDU\\\\\\\/eyt0sW\\\\\\\/W3WoYqdDTfbEP7h8cIrUV5jj2AIhQ1EVPpBSjB4TYcZhuINzz9VH6p+Q63urfZd0wbmy4RocIBAkjOWOnKUMUBMQniGb6RDuMpGQJobOKUUzRgwnZoJKYBqM+w+PcYOHFQIhy467MG73bWIO4JWtDYwbfr70xVy1yebTw2T72itUOdwLG5oYTRWOWT9xZ539EJt+wmQDWUIGaJhMQ2Z34\\\\\\\/ac8pQVwiI3Ooluy3nKcs9w5mXr1lAemFhpDBvgE0Xed0uhZd1iPHOvey0Wjyc9qTdTCcQVm221W1a3KrJKIsoy\\\\\\\/VinIw2U2JBnsgkkTiHis2wzBNy+uIk6vNCjSy3JY0cLxJrPmvoOs09vB3su+a7v8Nrl8PCbCjeAJ0aQb\\\\\\\/bbt7+iHBsP3LxkeHDXwtQttjwVeqOiBKHrCTshilXV27TB4lUhB7fAUVi7hAhjjM0+ReizvFgUJkonYhzfjoDfbOl5VL5eEC0kCa9Z60BJRU3cFBHncd1C9hyFS0TCoyIRZ44v4njeQzjsP39lacbuMiGgrKbGWTID+\\\\\\\/hKwBskmIkrSCCsaCWz33x44LCIAT9FMsW5LGf7NXyPBLsbEQ25rEIw2H+j3j1mR73xTCP4RQ13Na0meLxs5deRAtp6DQV9LmjA4NM42A+siRZtG7+AA3yrGT8llqK8qgYe0qpyuHOIdPK6L+A5YvCJNKKtxLV+2QbHiM6+U0n5HGFNbJI7zPYeNjpSIDnY+fBNyloV1QSSUgmdSdvN5fRmnvHHRVdpx6bk6tR2ncuOKOFusVCaZNb9swQ\\\\\\\/lSEYdNQh6VIjrbrTmvafrrDDAur2krWHaUFR6lUk53gRnFn\\\\\\\/mhyabK7PwjFna6PLfVNU83+HfQmUojLSJ\\\\\\\/3bDZxojMzrvjiQu6gorRGRgW12sevTRTIGrW\\\\\\\/Y5WXW1lJmQQr\\\\\\\/gJryxy6kAl5294XcubYGnT4lTxJXGfCkYT9DE5SeNbvzV4eY2wM3WZQtWWxrEJvTnwL7ikhYY\\\\\\\/HmXr1XS4qOnQ\\\\\\\/PbNEBvaqwjIN4m5T8eCTbsVFEFK52BYcWJ9dV5\\\\\\\/dL6w\\\\\\\/Zb7VwxAjVjp21tuPrZu\\\\\\\/BKBvJpbT4pIsn+G6haiTtNvNOZ4cPduOHOyOolSOPqhB9YcOmkxyTUG6nhYEXqR3IlXusW7Bv93eUK2iwiMKKo9rERohgOJF13nRq+2SlIakJYKx1S6ZRmjIPuiFHXoxmE0Z5oVYCZTC6XKwpz+M1Ek8lyfcTyxWSCeK\\\\\\\/YJ6lagUdl+g08mlN3Xw0u7Yq27q4VJK67gP3Nm8dq65bHs3GCh3tgaFIJv7Mjv21i907Zb8S+I6OzpUH1v0hFmzhunx+cWKbxvAivapCg2tWAwIVFkoP+mXdpvKm3nIZzBTwnSmuo\\\\\\\/Gga9xRAqumCNgWpDrn0GkL8knEJBw8ZHrVUhBwc6TiCpD9vQ\\\\\\\/zJsNFiKuCFRpMNvqxGpDzingU46\\\\\\\/EkM1ufeTeqs4j3fdtjpyq56Sop+\\\\\\\/2f4c3wCZ1Wq\\\\\\\/eRQirw6BiHtRudSxpVk7wciYArmogTLe8\\\\\\\/uQkYH5LkbtQS\\\\\\\/sb09YdTPD5d0AtTAVc186lrXRd5qU7YxnidBKRr3B9kxdWEEh8Sy+Nz0\\\\\\\/3nFY6BuUe4XeUbfQUBIZci1P\\\\\\\/zRVnS0\\\\\\\/DbrWMNSSWyltYVFiLmbDzjsdoMuLs1GVWuvfWdQ5szKVyf04uoHIOAHGTvKd9lgsRsAkrm3m0wIBvNJ8Q\\\\\\\/aWMWUZ0L9Zd\\\\\\\/0BnYQPN5+Km0\\\\\\\/g9vujU5xwGB1i06G5MYi9HARohOPBVB23nC1KW2GmocSMHf14rgnPIXKskzrT3nNqcNHmi2AYrFQRccuAg7nf9o9iLjCP0CMzzausc6UbsdtX7KK2Jp4+Y2jd81ztNwmg1ClqPrHAnl0lqxHAud69PQ3SOBHJuU57+S9c7cbxqD0wRkFUHtfjlxStH1gd00B1CRaubPICo4TN6LRCu+\\\\\\\/3esfIJ59WuVFN8YFX73Pu1Gs78B\\\\\\\/UaZOMjTlfnvru9EqYBw2EnanB3g1dqVATB\\\\\\\/C57ZXco7eEKfCWyeRGxy5D2YRPx7hjTwHjFU6ScuTbALk\\\\\\\/PtWGC3cNQptTbksbmLOi1t6XmjsvXMWFhGf+G\\\\\\\/2hI2fHo7fP2b3ac5g+CU1S5gGIfnimqZley4KVLTMgVmzm6t5lWez4lUd8tBF5k\\\\\\\/ZVaRwXXV2xwbqyP8qZKcuUaLdfO5ZUyq2UzeOLHt8E1bbruzVnlRGJXK71l8+ll+8\\\\\\\/O4OcAozMULyRKhiGuL4c1LX7yrUQQS\\\\\\\/98CzIaNXvDBWvfgetTtUJR+YUwnTbG2a6cqGYD5ALntMBzR8ofce6Dbeq+OxEKa9dFUwegqztl5loxJ\\\\\\\/4xCu\\\\\\\/Ua9STmcW28NrneS19rtcy6Ygeuf+t7THI1KBlm\\\\\\\/bWh5nVswktseQCb2Hey1KgyzmyhzZSYtC6adGbYFSWxRNdEn8\\\\\\\/iIw8mAmldmlCfDKMNZkZkyVziWwjC06u5Z0RyQuWNJDfSg09znZEaqm1zsUEt4B6XRQuHj1D8N2AtmNm3T\\\\\\\/79lqg9QS0DzAJK7+KtooIl8S+siwTTvHWl7DtJ7aOAUKKkKFoEUT+gqz5FY8irl4ln6CXICHGTjRRDTBTmj9zEYKgxcfJ4bj9b3RBeIFgKNJ0As8tbT1V+c+DTlTuZa4mR1V+FE6ziPoGEUeNBefwUL9WyKg8zz\\\\\\\/3SBCyOjVUXhooHT+6TQ\\\\\\\/IHwuLD5dVUzI6WatcaIvIOUSq32P5wQRDTEqJRAtHFRHfWrDo1oz+fIkV+m5BbT3zomNXXMGT2kVYdcIPaMcc2206o7JnFHfUUsIYYG5wWZFWuzxAUObMv9qCscwho9+M9Svp+nnZsh0qc+\\\\\\\/0jiDsxVkyombBS2ICdB+dd3lc6y3759S4F2sUCTQkXvATj025GGCrSDaDhnNjwjGqOxBjuBI2hNxj7zvX3dB4kIU0Pa\\\\\\\/Nv27dsxNVEI7R6yStB205Qfp5unuDF8epY\\\\\\\/8JigQjas3ur8C8K6fP+uQ6IAkJOTBIdJsd2D8527qM0mOGu0TAkduxA2sZszPo8+T23XTGU7qFV5uEwGVIUrKBIoxjqQ4bbdFpxLEIBjf6ThtXatfTVg9HRzd3WiSGIphgEZmrzaaECy5zX5nTGVoHLeKaClm\\\\\\\/vK6Q\\\\\\\/cLHp\\\\\\\/clZhqxYaVYyP43nFXxmBs6xb\\\\\\\/NlnB8UW0RmScJJ+XrO3FZAGFPRVtjeIYeDe6n9eqQzJjFpcpHHeSJMFncwqjzzBxhOmgSK89VdxxN2VgyijxTUNaJCCrRb6w1jOoo+3NwwHJKUrSMJmsSYB89HUqjZtwRTlSZABEw08M8OlP9KhHxfeVSfTNUxT9mTTgqinQI+xEixPaB3bv9sdXDzJlcNSJ71RgvGAUWrIPPK9KSvrNMRD780Yb3UvselphqZEXZ2blPoBp5XUvmSNRgksTHhQJMHc2LPunY+auwtc4eoJ4o5dy\\\\\\\/gQ7M5lsvkxF73J9eckVyK8eIb0izw\\\\\\\/qMX5JTW8P8lT1XwER4o5zb06mW1gTG+CNMg2fQl\\\\\\\/RUQiMBfc8loi+0K4I4I5+VfJCqFXSsS33XEbCdBJ+kj\\\\\\\/5Ba5\\\\\\\/wx3S2uSIvCdYHF\\\\\\\/FybO6QAfWS\\\\\\\/PPyjt7mitLd7\\\\\\\/J8GG5qu8eEqd5FlGj3rhDy2ddc3Ch982xbeWONgqh5PCucV\\\\\\\/ZlTvFt3OD0KG5KpULOgPLfMTLEilZjl3rs7AuEUTx0pdXAS2Kqg0080dPc+7TtqxMpYeNKCD+nejgrmzLmOgoc3ZlQEhtzk7zM2k\\\\\\\/RzYmCjfrN7AKQSdHAjxmRlyJxwkZE41ne97hmlC7WYkj9PmvhWvT+g8b+5Hb11I1mlpBGzuG4QXaATEY8U+EVFnJsMp+fMyqNQN8Gp2Pr4QSQp\\\\\\\/vJI83Ot6tBgxtlhJe+riMnrzWCOxQ4GyB1k4R6WgkGPQy8cgC+ZM9\\\\\\\/y5zFDeplmLRh3NW9+DkOLjLtPDpiHn34EMM80BsDR\\\\\\\/dwgCfT8GdSpAIvv8jLtWF5tyS89ZhKjL\\\\\\\/s6tF64y7NsV2b5LsZMl14uePMX4\\\\\\\/GIUrqOc+qxHs+7av0nXtawQqIhgGlbEayL4w5Oao3UWJydZhAxcIskWwypTGauxLh4RhRh\\\\\\\/vrOfjkuLfhNlxs9ICS4e0gwmwgLn6SnkiW+QKOYRWKZ684ZdZljUBIx+b7Fo1eY2P2RM142qua6TYOGosagX8OqBptV3eJDWaRoXruHtIthCoBjojSTLhUpLkQ7wM6EURo\\\\\\\/\\\\\\\/kznzpVqY6sogmTDOy\\\\\\\/oCecjpT5svx2gBJgb3Fqt5LEXCwvWwkMPILWVhqpI3BPaBfe+9thW+CgYfS+aMxcUjRpXwUsdFr\\\\\\\/bnmTGNP5jFa4kYn+ZDbpqVMfGdkxl11a9yQ8QVLOfSK9PhW7Sqj33PZ2k\\\\\\\/mlDNMu56W+RERJV7JYVEcZL29TahA06IsbSdP9WbW5sLEuuYXVh7e0an70l8YlPOzZXZZOesujZAjVJzBRr0vyZN4hGgcdl6qpXOFNfxLsQ8oVlQ5bsrprbIPnY0s9RqNsspYIVaxQasNkCE166cu5Ad8J7u1SeyXU1lczgfoo44xTYge\\\\\\\/hcOp0wVIu5WkjwMN9YSEAS6HRiCoAsnz3pGaofzZUgd8ICvWvd34nIP1Jp3k+qXJT3nAnrWBuAYD71Md1BW26CreA\\\\\\\/+UEkK524DpbA22di4W1V+tCih4f\\\\\\\/SOoQ3EudObZ456ZlDngAWmkmTT07wp1k4AYTQSvY07V5LucnEH5V03Bvxn7WjVX12PfyYPHJiFUUKHupgRX5hywIqHWsu6ss\\\\\\\/1yX\\\\\\\/CkgKqjWF\\\\\\\/PYNq2NDFdMpbLFhIOrUh4AeqxuF8t+nr2arJGb\\\\\\\/iuZd\\\\\\\/t3vems6mdeOYejNbKUjWI3bqiT2FdpV9fE3SczR5gXoqv9kja4g+83wa4rS3EpsLo5yIDr\\\\\\\/LYmucZXqG+Ue3t8yT+SbI5ycwHFDH+O3tNZDmUJhvCvSTyhyDRbMB1he0fXDVMqCYyzGp0KuOcoYmZNotCWSBU6YtcfK8c4Phs8Mi\\\\\\\/zIl5WQE4Zl+\\\\\\\/3g6lxngLIaH7X21AGyIremi+SfRlYHZNXTX\\\\\\\/JowEjI1Pps+yhyIl\\\\\\\/LC+NfsBwrNE\\\\\\\/DtmoAGQ8qyZ9Ejw3FonP+mVnpeaNIWlqGNW273UQcztMmHiRjOnADdK6yNkjb+ByHIaeyU5jrU0TQMdfTPZoizjegztcIpToGNDtcUZA9LOc3hdBZQKCGNKLl5P\\\\\\\/yJxu38I6o0dmcp+gV4SGgOpgYWuh7tBSJhRUVCk8AcuVW2LtzA0RBWAd0WTwzxvqLOo6BEpnPEdIqS1vUgNPFSj4f8ZCl+9b8RFwEApDOiNJlQHiYI+MyyloHYSfEcir2ME0AoPc29po6ahjIlgF72FBAjAFzEeGfsY3d8\\\\\\\/CHeMQhXbPH1eqNClD7EqxBn8h0iAAWH6Xp82nTJSjryVcVW4e8dvCpdoZuVbdudV+dxnerZj2yCtw4ya3zYxe8xxNQkSXAQkpN41s2J2hw\\\\\\\/H3mEA4T67VU3VmnzBfJOb8JUt446ZBTwIOvGncloVZziORH7RUflmRogRm7lxpd9WSwovS2Q07SeeLXBJmGapXsQnlGb+s9ankCjm\\\\\\\/iuELuFlnE+ARPQdt3L7RSGG6T1KtBEJx5x82DSDz\\\\\\\/4uWiE705372j7jTCQzss31Qo7DuQ+c650ABjCdzCOh1MEunPXDizNWrQGT2nzAe5wvRrPPPRluo5WMbmb0oW6irKsf3Lqu\\\\\\\/Yps6G1br1QTBZF9H4Ta2Vtv7UjwBE3588eqVJSQBD4ZTKJ0Xj\\\\\\\/LzWaHtlFip1+tC2UAhDug5fq3lSAJevI42lMd2E4la7j9NxxD80VXxix6bqGiZTdScYIy7Bwj2cZIysIljCD1cqOZABIXHBnkB32qcTauoiYxsgsXHGafMV\\\\\\\/xpptSQJE2yAYOu2cILDOPsX9BquP3gr+Z6Thjw5PaUOi9Yd1+yxHahRyIWp18l87VYOTomBjGUcW542bZ3ZUkJAQHQ8rveCAPofXeXlDSLhO1yIG7gLv\\\\\\\/Z2Gu44z+xbPEAZ6NBfZ+8Pf6XyQ2iz4mDNskqR9R224zMurShPLyAH78lIQcPRWDcOAyHCKTeoorParvz6dozXuwk3QD+BMZLXupUWvfYFozdUUYTyQCmwwjwEVYPlyJx6rt3DD\\\\\\\/QCAJhxDRgcJOEnvyLgi8SVcrtJ4uAtKSeJMzsB17dNTkq9+hdSRRjlVBcHlyqLi\\\\\\\/8bpzMKgsSm9Vm\\\\\\\/dobVKx\\\\\\\/7psEe1gfl\\\\\\\/cHhnUr0FEKARe+vqzxwTvDNiPpPagL77ZldWNVG1qB8DXmLGSbf4BC6S5rdkml49yd+HPU8sgc1DMFbbpaRjrJy9EFh3X\\\\\\\/usKnbuPsNbAR0SX\\\\\\\/TVNwnYfcazhpmWK6zqfKYC0HiBmt5AsMy+SqFlPpCWAhQErNFqD1SoyNWECO3t+Xg3hy7r5y9ofL2E\\\\\\\/bBinDcywrEfKtUuCIUAtANpIXi+8RZH9QlWSdwJBLV\\\\\\\/Ah9+YojfbyiAiEDHZb+A\\\\\\\/Q0CbkZb\\\\\\\/+W2xEujF+Z6gKU69nfKWG8d\\\\\\\/RzT9owB\\\\\\\/\\\\\\\/j92HZMclx4x4omb14XAHOnQhmjmT7fnOjh4OwcIbVqVAyywfuLFARp+q7dP6+T7gH4kCBz480OcXp+rH6S7wCYyRIaMLAgM2ZnlQqJjdy4bnA9MKBtQtsNqAoe4JaH8Ei0AS6cPxBdtr+uv\\\\\\\/fUNjmHsBbPHZ32SNEh0QY6PVtFA9V9N4WmX\\\\\\\/Dq4pDtrS1iqx7CD4diJb6QLnhTCA816BDmoKWG5e9x9gduKy65uY8jv49ny01sQ6gqvlk7xX9mne4HR7Q+cCLCs9L7Mb0wDVXujGc6JsCzG3BxzGV7Jo77QfEHepVdsc713JYWXdj1zFmPU8EYCEmJFdNGIef6GrYB1MVDtTLNPSKkIkB473378yzIweFbe6GBLeoThi\\\\\\\/En1XCCvF21DLrMP4uZP5HoWEP7xLVZDJJd9mn0oPjeI2L7plzr8htTJJ2G4YC3NogqEYsR6UIxb2ABDUhgrrNrr6Iq9c2skcy73bTH8D8vPdDcuAHSnMwvEGg1uDUp2qk1FwA1jpik6Pvj3dRTkIlbrqyEj\\\\\\\/Cs2I3knqWKraTZ7C8Jxsukz4R1GgYQOQ13D1KMCWDKNnsf62GwSzhq1rvVBLJGM7I9U25gIaV9dZ25oZSkvVWdjfgyvxesQXMq1u6eMBhwB1BTa2MiyjSwxicf+uVEMlgDJWGMb1x65K\\\\\\\/bNScWtMEMY8bBgMtlKIPO3OY0JCm66ilr1UaTS5sm\\\\\\\/XyvJ068N\\\\\\\/G\\\\\\\/Avnq8hP0HN0gerTFRbLR7i+P475s6sHhTSuFVXxJbA3eWuJSV89SOIVOoTP0E6lEhbxpwydjL\\\\\\\/tbF58iy33Jo\\\\\\\/tvuMF8XVA5cD6WaTr19d9gI0BnDqGd4iN8n7GbWXKJFuoXMK\\\\\\\/MJh34VIIgcdfN3gFV6\\\\\\\/B3rujkIvMLCVcVDvx+2JVvxRlVA4mys4IYas6qCcfEIbLD+iX0Lt\\\\\\\/UEVqJrPEynbf7xeCkt2p4m2BbsOP1h5J1VKFOSRseKRs+xIZwyJEL\\\\\\\/KosMHUsi1ymr4fHxMwYZlOw6\\\\\\\/S8p+X+PDs0LrMTseWxR2DzwWSosRBmYO8Oi\\\\\\\/iR3Gho3aBfw2JuqNj3xUS8Gq5HBSDltNvq2RlU5fE+MfLy0V5CIqo1zEW\\\\\\\/aW8bdOraKDTI+tJva4fCFMwnxlcvU11bznH1xHg98du77r5jjHN6SAh09EzE87QYSbGBWoMBXtHACVXF+pRQME59CaqbSEKQZtsApiosdFcvBqls1EUHfjDHzg\\\\\\\/sT7MIcNxVjpmUCLOSwoXnkmSym0k7WCN\\\\\\\/EhsyUoNrWQ2PdFYzJnp6\\\\\\\/tGSj9rAvRLIU\\\\\\\/C912P26Z8mcTkg8PJiuy5Fa8GuRLDWc08xgfomCUeGkA9FDRC\\\\\\\/1xKJEgmldLGu84iPrHw6\\\\\\\/00CMVUIpLRXD4LVF8S7V4caCPjUrYhuzfkNrtFOYCNBEAILy4pTiqJC15S321mUjZtI4hKS2OeAy7ltqPH0nblFbDjV4AKxK2VulrtlGXplY6EjqY4RQDvjKv5qr\\\\\\\/8NzJmtlfKsg8yikA0cleRP1KrpKiYJn9MJJhKOQMzuJFax3E\\\\\\\/5qyv2TGA1nndQmI26RfvaNl+9pd\\\\\\\/7xAMGbQeNpcSSQxJ9\\\\\\\/xPcWvLGlzPvUxJepoiIbHzEqwOPcFeQyNPlz7XQMleuCXnCQlU6aLCs2tiGApy59FinxAxZP5Ns\\\\\\\/9IF\\\\\\\/u+AudnzIX3AmO5UvK0g02HuIE02pcZlpnGBFLybMKr\\\\\\\/NBqP8euMXY4D7s+7hiCyoT4U6yelPpXODpuos5hJJvp2ZYbEopEBtDCk8wh0ikvh3rP7L5minVlEaqqxFme8qQOBxz8WHIHMm1o2hER4Sw846zCRW6vxzilbKQoUFhMo7BlQ5AXmzHlZi6PNJ2efBkE2J3Bvi928VgMvYLCTNcPbxvL2ftWPQc15X4JJEVArA0S3AzM6tAMReha+uYFhOTQXQ0f7eKYyjj09TxQoRcl9R6Xee8T\\\\\\\/O5srSKSaK0qhcAOU4MTs1A\\\\\\\/yxIqB\\\\\\\/t5XTZGzH04mWn2fk8IZxvEMIyX1uFOPv\\\\\\\/N6FLj4L00YLjhCopgxhOASLNehr8Us\\\\\\\/pfax9ofxP0ohAqI1maBuPGeCXEGVmRjsNDGkLT8S+e7Glv80VU1e36X7uUQTAr20\\\\\\\/T7Mq+nZciapodzeW\\\\\\\/D9Jow0nzffghDCmH\\\\\\\/TLYe512E1T4eLcre00pNB9E36FukjVGecLIixOLJNn3DXOZApTu3dulwvEF5A14OQXxEIHAXNTL4Qsw2FLfxAX48AO8KHi9Q\\\\\\\/eWy6BLKbH3FxisBA8WfNdpidBLYkJEUB+IUazo+OaNXidjDnJtUwmUjfd4F5ZBrmzP5fTwdGgXuTb0wLUjlFltz4Ab3Y3GovwpVBrkEAn4eSX5eXgZFkhOhvEcd5WFK5KJgg8nI+wp945Mp8feojJlbgg9OFn0v36qRvfRxLhrTnLikLdS5vqwGdhpQLBNO6koToz+tQDrz8gPR2l313KoQPaypgCL82icSkJQsL6ldeSq\\\\\\\/Hr25DHIeyNUtvIHVWNA9fqloroYAJcO\\\\\\\/vKMulxgMk88iZeW\\\\\\\/LpG0wryRa9xPkXFripSXZ\\\\\\\/LbOcwYH9ArGFZzl6QhXtztKWh4DCIWEyFf9+ay3vwY3P8RT0BQgsBxEi024y9YfdJnpVqZkVEbdwWdL3EOCIiWBCAlByxugo3NY24N8wHTDJTalyDEliyVAh+\\\\\\\/MKxGpR47+C9OibGnbfst1aY97JI6+Z1fovScMoCo0rL+3\\\\\\\/Mz0dPHxf5QzvduDFYNYsjDLctoW4V0jpXeK37NRpaXDQ+jTL\\\\\\\/Vj8qaqBNfhtaSDVNLP5worWkhLmQu\\\\\\\/96DCY3ZrMqTpUlc0BJFu3WDz3aXWucHEYpa9oT\\\\\\\/i86aAaYji8gV1p5eLYUwB1oPAvHmUSHvNgC0Iy2m+cyESjcQ2xAFRWKLDVJ1RiYPutXlgMPJsbDGJ7CRA4fxpoUa4EGwvMbIg9dA\\\\\\\/ho\\\\\\\/jv14e\\\\\\\/1VuBH2F6T3DVbcpcrha0hGNTTc8B8g14NAkeBrRLv\\\\\\\/0YTnsDJyBjm5vaObHnOIWr9fdT4exX5Ey9BSApU6AYH0vDb1SR4x\\\\\\\/CSd8BbJ43CXtolsX4vxFyXAYmnYVzQGGPDQddIMnZ2qKwvmnGPOdty+CJweBZIRxkV7Ot+eeah75+TZLfkQqMO664F6nH5SNu9jQb7Epmd\\\\\\\/GcJVGY3wl5+zYTbt1U2cMI9RdikjpApnwKMW3nP3qlKv3TUXWV0ru3NgL7ZvBpmXXjLt1PdjCstYYf8vVcJfmxVps+n6naNZ92fqShz\\\\\\\/gRyHsCNm\\\\\\\/t\\\\\\\/c9Kfz\\\\\\\/+xj+O1PN2yg95dcK01bZhoSkADLsxNFupJyGp\\\\\\\/fTR9FOFjp690DvzTFc4ksefiII1FCqyIlkeFi2o0bDWwvYTVWtlOW2hlUWvwIeOMqs9Ah59JEOuBUivaT8vdp+kgsIX4pJIQQ9fls6caDPZ5fWwdWxpZBVwj8fi2IDMH5Y1ZMEiaPHRsz\\\\\\\/27UVLFKxfEgrqZqQxaMFOZ9p\\\\\\\/mud2DRG9HgbqLYjyQCJv+OkONBtbV3V7V52i0+PR\\\\\\\/s1kMnOvYR56PcNlKGQHISyJkfXR+A400\\\\\\\/kmajyLi9gIqvgiOz5nE+g6rNFwVVVhdSlDAtvLumR0mph4S9jS6Kv3xmAZJwN4b6AtFPR\\\\\\\/ITeJgVlEJvk2KvJxYqinypCA6PI6rtL1s8U+mTeG\\\\\\\/bnY9KMNBgS\\\\\\\/EsbtTToOF748bl0bo2NKDgbba4d7WNX7U3lrcKkj5vo9iTtdeKAlCbBSd5wWjvswSLbRUZvKe9awK0YVB7YLtWKWSaYKcR\\\\\\\/atw6aHSA1v9uGQ4RNXId+01dTkUAjwGExuQEZXMncQU8zZbci0jAS590pggMXdYB9xtYfU27e\\\\\\\/ZbCndEvDj7+htxNN0YpHWRA7sMAilskEHS+rp2HTIZTNGKSeuqC7HZiwdnM8lBv0KxmJhXbZZbae4EdR0ED8ux2qfNFsMukcPwrjJcMLVhz8iOd5Uy9abfH44Ky1VLxE1kRz\\\\\\\/Z624yJjh8PHBEMdNNTrLe\\\\\\\/uvnd1T7J4dIRpfqYEDXuHM\\\\\\\/\\\\\\\/Yu8aVyeoRXrBZxP4nXn9m7PiG9aiRdLLje+G+jRGXPDTr+EKYi7pKKx6owcqyPQQnfhJIy2YErIb8CzRZdSwVc71CvT0BcjYeaw3Sr2mtFHlMGjwUmd8MrCz8cXjlXdDSheqPBM547wq5K7e1BaAuRnACgemPhRVdb9DL1KqXYOWEB+d9XvBOf1kWYm8ZsE9WvIpbuYRXBeMVaCow2TpXkGaaGFUf8AiJLpTWlfjC7zZwg8s6pF8YfTN\\\\\\\/r36YLawwfmgpHBywkDSjoXzIdnGrKbWrGjrHG8zM8EL9CeZX0RscmysMGOZcI3OS8KKHpF6XuOQLaho36AMg4p\\\\\\\/i0+wlJikBC51LoQhnVg5b37BOxqF0SCAkmvabFRQeANjZ52mgmUFpMRmudAUUa3kyZdbvzWlw2xPaW48UA98UwSvxU+9Y6KfoTPgqY\\\\\\\/yy+A0Rx2s2X8nVqX1LCJEC1lWMTobfj0jkSHnERpFTR+nQA1yFuBG0nBiZ0s7rxPq+W2oZd0ptphm8Lc\\\\\\\/9kozUc2X3YlLzrJyzOxiTOCoOLrOLlAYqc1tGN8kgWIHqDSY7\\\\\\\/UP\\\\\\\/RaSBW5+NS0wZTVvo3xrdYMUwKvVpgPCFGpoEL8nVym7XeY9yGPpGBPrsuCDuMx2ijX\\\\\\\/LV9myfMbYEtWu6d4TMn\\\\\\\/UrA+VkFxXZ3ZNhT0Fi23SSVoIxFPyjKMr2XrrbCkR6umDt6zpTspFRhnxvS805JwxFSb\\\\\\\/Mf8kC9kzC7RS5xNC6tY5m1uG3Vxq4ougCwcm93pD9c7r+qGxMtdezi5JcLxXy4BVmnW2U9bLC+A0CrjV6LRd6VOAP\\\\\\\/vo4MuQrkkXkVx9kSlwU9nXWhOt3ZDjtbmASoj8R1e6ojofYd5tsyg\\\\\\\/lmWtVWD9F4bNiL4brFQezPujxPufow6FLKHVUGZv\\\\\\\/sW8lBISR1oI522+3R+MIdRY1U8pNY2s5ChQwWYFsNbjDQ4NlWvOw92GegADv2q2xnaOD7JsYwFULUAQEd2BbWFSvovsSScKmrFnQYRrPtLEynyeylzI1yEkOmKq8gc48h9RXD\\\\\\\/\\\\\\\/dZaLe9pQsyYGMxX2cCg245LXiBNr1Kw5lB3sKwKfXlxqdgXndcTrmFqPTzoViH0LbsVmibBoMnbPs6ryH4AVf\\\\\\\/AJkS2QlQ3bD\\\\\\\/ViwwHaxU3692yA10cOChFDbGjuqZBpXeVrJCt0FcEcCEU3WZPsBCJxj3AZKM6zOl8B7z+SPYCv+HbUes5trCT0IlcAqJo8WrFoJqeMEVhgRe8\\\\\\\/nSEzYnrxycU7ggcCTaN3qBkoq5OopWH6KHaDr5\\\\\\\/o2RBqwwQUVfaProP3I7ZtEHnD\\\\\\\/QrQ6AFVUGm3vv9OTlz9CiemyxgG17Jf2PPVJQDccboLMP2bF7grlblbtGZcCDrjPQV6HwuKP0eIsCU9ki3Cl\\\\\\\/9960BXJJxhlWY2bV9AeKh5MgRejh8JfjQJThhjVOp\\\\\\\/7S9K3CLVG4vK18N3jAq8Lx173PvCp7b\\\\\\\/R5nreVzJggQjy38BWEXcTcp\\\\\\\/OCq6gJOxAwes2ARtcUmwdpX+PlgPclzRpX7xnRU5k1w4m6NSRO1jMrb8MhBDxsUWGISGwBBOyy9SW+MJlF2sZdDQFQewSUR448K\\\\\\\/IssaXK7k8XJ1YjDAlCzLJybmt1Pm1lCyjxFVyuuwjVFfyeFwpFdyGilKgsskuiVBv+Ur\\\\\\\/zUrlGrYa19Lnj\\\\\\\/M8YqDd4cJWM8EJ561XW8Bh9Zc02tTLn0xbNJMMWk\\\\\\\/TrQONam5\\\\\\\/vHnb5mkTtzhfhf92obGRSSJ1PrRNiPWpEVI9yWRP8aor9XKRxEYo0GaLFiBNmtwjLia1b\\\\\\\/nYXPLwsMAZ\\\\\\\/HdJXEY1nnYkVqbrjFqb0wNt+kzVjGOC\\\\\\\/2pY6eS8hRAQgFA9WhonW6La0tyedpEn4ndFtshIH6YWvj\\\\\\\/c69ESk0AElcv+FD+W3sMuY59aOHFA9uRdESH+FIHq6qH6uHDZZ+ncn5t\\\\\\\/sIVqymzQOYEAp2FihOvYhPYdu5cDXp1RsTkqCJhHx6CiupkpWQpPibPLnG41TAA\\\\\\\/dvV5j+hwAWYPh1xaGBfRqiT7nJar4aRldSFskLgvuoBw5MIEOgneIPEx5oEF9fdXR3wTpMttCsQcIX\\\\\\\/MnTmZTpWujy9Q3E60fRL5VJGHhucl5WTJZaxSTBIUlPxSy4DlYiPaN0Ex3XQtTEPBR\\\\\\\/IYowz\\\\\\\/GFhHSe0CVdEGQXc61ApB1RmqlTqFNekt3x9FZN9X1BALXsp1JIzrDFmID8DJX24SKr6ZauHej17l1H83wywPVLDgeP1\\\\\\\/cbNKY9aOaHBSn6PfdaLzJrLblPJCysMBcSnNiq5zq9\\\\\\\/1RC8g5AAx5g9Yeaig4XI9pi90ktrjEhYYZZG1C2cs88ecuhxN2jOCh5Wb2VHz1udOR6juTv2hZJ73fw+9UuYdVJa4cC+kzc5GM6\\\\\\\/OO4XDReO4Xkx2AXTXL0T0YJWyOLq3oeCPlbGLR9flPE4hotWDOvSCSZotuzZE\\\\\\\/qsg5HfhqHjaiA7v+1piccCkgzfutzh4dEZPajYopk9DNOid88Ka+c8t+ofLaxMY2f95LSYtTEw9+s9petKvOskRryv0vifV+gyu0MolU26LagbAXFytCsRrb2oHDNLpDNhCI6IeA0iO\\\\\\\/WkvrnRJcPIy4iNf4tqV3kIkuuzFaBgrvg7NcuK+pYSn\\\\\\\/3iAzsKGvL8+a9irurFubevCzcMRGhtjlLkHWDrT5mwLTgKHwGtkQGsBg8gm\\\\\\\/z5FMNgWilPCYGfTkScTVmHK3Ys9npcVxyOcLDlGUUDiovvk7vWM1+ApTqHEDEGBEnIvLr5NAIfrdiIYsBcuXX9Savok31Kdu9Q8Bnbwk+L5Ou8ppCG8RLvpK6tKAK0jY8TH+jM2YceG012k1QT72g3oZFJQA3MCLuEz8k+u6xkxBOCsVjlOc8mbzqwL7RkzgAE1MNJ+DipzCrqXszbOD2CUIhugjfp6A01CH3nYe4si66CBHSzp8OlFErP41p3EsEPL3rWw\\\\\\\/\\\\\\\/7hRAbOPic7azQHLgUXCTB8QUqpf6GbPjfBMwIjgK79Vy8X+xw1uFX2dFaYyyvJOsj0AgLs83OkR3hYHVX1QEVIS2HZiovIro0RTXJ2OrvF2wJtgYgdmihIJrnOb5lQvh5U1jePlfitxlfQJfs9y1fSI6SMu\\\\\\\/K0gkX3kD1t6cMo2kbs6Zl7XyhVFBe7RwEg7\\\\\\\/ZQlNUJfOLv\\\\\\\/vGtibiYct3rryyRiVTtNOyxT2qNJ32ZTkLhi2h3WfPor4nxD7+K1CQAxg2iiXpWKy\\\\\\\/ngQNNYlbxmcrNRzRBmrTA3FdWn3OxjjQQpvN7TotOw\\\\\\\/rYAmiGmy69xLCai83X5Zh5v7yLgvfntc3ckVTf78+V8LZvHuSONzNlUXgBUzaZTgKV3BkK7TDRcDTeKN4QgDdRGywjCpp87AW\\\\\\\/zfb94dPI0zhTvZvq+O2iyf0fZ2SY256UvgmehF9CC5j+f3ksoq9cvfDjq\\\\\\\/XfaspfZ8CoPl0MyS2gEnhqsWe+FOrGWjwQ61WBeMubSX4GcUaMuACFoNsr2sCMAEoRJfj9yjejFyYsr5SfWxgbg3abRP39Z8mwJJzzXiwXdQ3f562RKBtoQDIBJpVVlhdE\\\\\\\/Vc12fe9MXkfDfMXWYK5Svx53e85wJ6K7w40MONEJqRRX97z4kZGhcBD8wSJCF3CoP7KFYno3sXjjw98FTOcUmZv7CFOoFl\\\\\\\/v2KhQ\\\\\\\/mnGnVfnw3D1Z2DFPN31415IlFsw2c0DF6zZA6GZhBbHEtqitiPNpgMH3mQkrOnbrasjHz4jc9cii8kJdDc0s7\\\\\\\/brLWasPy0Upirj\\\\\\\/KGvnPRRUy1KkzQDvYUa\\\\\\\/ws0CRdPEZNhyjsItuM71uJKMRYWD7F6S6oa7m4gx4M3KVuzlzFadKQj0E\\\\\\\/L1nDRIINRRpGyUv0Gtg9J+79uadhu64p7iz2QQGTXGiKL7IFXr1\\\\\\\/FffdEcEYUOARXEEPLKlZ+O0fx4yPlSOvIhLYbTEv32N38dlf1kU3N8u8feqozy2bpecfT4Mx5YB75oHpHqRwEK72RpDtWUv+IuxKA\\\\\\\/WtsY3GOTHWAdnV\\\\\\\/to5E7BrAn+q57aDy40o5IQXCmB3WwyEEiZB2D\\\\\\\/cO5nEz12GV7wxZ7sE6eVs+vSKzHDtxLz3f2o2HyUnlVjIyBwvYLGK1xku1DydUIoglmWe4bF7oNFIS2oG24k+BD4GxJ7JeJak8BXD+68zD++Gustl9MoQvHZRnvNyJXWfGtzBwPM1QMt6qg2AOOf52bjjPjFa1xwh1Zc+UyuwkeDFgTHx4YIubRBvk1BObwTwlN22qRSSVQjBzThihhZhVmVbUNFFXU2mNhYlBjPeEb9RBoUKiZjfhFo6\\\\\\\/UkechI1PgrP6a7O45w0531SK7eFIR0eAiEcuEhjDsPbfqqyKRkmAgJTbi\\\\\\\/OqSTzmZwOdjP95Ya7nHpjUTrU6aJdnFJpv58ZrIxCYDlG1nZ\\\\\\\/1NprQgIVdIDVXmVZPtY5hPkTCygDQynM3VSxH8UCtehs3yRI8hnjdK8o49kHChQZHj4oCB4\\\\\\\/0cHKRA6Ev20euJ3zC6UwGGxQsFoBwwRnL3UTVCymWxmF94vUoc894qHSxuYB8ymH3iXJ0Ap+UgUng2kIUgM3o+bWLSdklDMBsa1ZUWvtZc9eobqAypXWezNK23SLq\\\\\\\/FWLyFzNyRf5DqmPICTXXoNE6ruvATqBtaiIPttD4vS+bHRJbQwlgC4G4fsb8iDR0SS8bij2JJCnjPLPfNska08TjefQtCtqQ6gMU0yjgUFdrtQL1BMyhUJQtIQh83wc\\\\\\\/bjPrnl1B8UegegIN4hVF1Gf2hh6DTyFMecNN7oJB\\\\\\\/2VOAPS8oyWmFIvfM553mLvJYcVag03CPLi5sUU3A7jT\\\\\\\/soL5MSMBuDDz+e9xJwUUEooIYKcH8oIw6oryP82alkCcHGTzPqNFJC1J7D0YVfXotWBFqKusN3dCHhucpP+fDorOp9PylFWnYc8TeYYTW4MsXICibGuZ3VTzTO2l0sOcVGcv4zJWZ7f38i\\\\\\\/adiTt2YWxFLLEMTPIYrHz9ktxyu0xezDFzuxG09mnM3O\\\\\\\/R25x57c87Tke8OMAaBRt9yemUZO7m1jWPS2m5ArbsOGtyd+LtKf40WxwV6wqoEVRADuK3yApsm2D6PAi5GS9EQDelgVOQyaRS9GWSrg329sIIiiVjQcbl+X5kj8WzZufpbE8f\\\\\\\/zVKEfvyItFB2NPfEPsgEF5g7bivBs0GaRE5bv+dPxuX1Bv3gWcjPaJ6IIfyZvmqWdvzTVK\\\\\\\/XheVScdmZY7NgW6ULj1uelUeMZm2UC6VCKDaQVyL9U6\\\\\\\/MOeLg7slfDgqYk16DfD9PUEEb\\\\\\\/L0iShayBcYYq8wqt9roqu\\\\\\\/uEKNT6m1ZxnhIR1lq8UuuYHvfOvOTElZItc9tJeJHF6UfFVAEOS8WJhskQKvglJTH0V31ndTAmdX+75raWsY9NvatHR+nQl7WQiNI+qU7tOACVmGj0k3ekumlPn4BeLgkloK7QAYZt+iJ1plTa68qZVoCWVoWAsXPefyVfGkD\\\\\\\/HhwBg+D0xvA+b8XtetuvNLytBpfOHbM1DcoAsLG521tjtiMUApC5ooSaSIOFNr8jJWJMzXYMTL63d1NpAre+asrWvYFunUJQxoI0+b2UCdE+OjRWg8hnKoERCGLqd9Uhmtlllo6EVfA3sNncyeJToa7zPRFmCbC9afjnwu+eHuB9X+S5u7yFENzmg7bmX7uZs3R0csGGTiHsASCTSPpgxrOi480WzJa+HpnNnh33vZxXG06u4tBdwfp96az\\\\\\\/ilYFPs\\\\\\\/4b9dJXEDZevs08FnmKDaFgPvLVqoR7WutefgrpSBB85pmX7YTUIUxBLmKn\\\\\\\/xPtEOLAyd6EM1aXSttLQsazH7+SnZk9i0fqlz4CXIgybx4xnj9d4r8vs0UaTOegyR+RaeW8fNE3bpeEiwRKryLSl5gc2jR\\\\\\\/rlRfk578ne6QRXay901Bj0GZisp+WqKDS4xsSGo7o6MwtCX3\\\\\\\/NMs5Ra2WPU2U8Z3H7xKAuIvWoGy2PtfQGvpcuwFZEpCj0NaAIV+CAglw4HqIQ4\\\\\\\/FO3hcFuzm9ZyUwNTUrPZHptRFEQ3iSsPvpLuQWr7NiHk6KqHmuMo0h+1R5VA7teKF24b7LzvHKVBkb70GWVbBbQb2oGomzTYHPr3N1WlcAiOtdAOi9kqHtNhIQ+RHqxxKQBrKKfoV2gDs7s95fnyXy\\\\\\\/valukoJDm0a18vb86QxI\\\\\\\/nDg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wqLEtra6cG5jrYbPPQ+exA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002297fb7e897033f068af03d4f6be494e308bf7422f6cce04af6b9d0adffa0192d3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QQnA8poo\\\\\\\/PaSTkiB0\\\\\\\/Z0lIhbxWStLokLntYYeTjp\\\\\\\/ym\\\\\\\/v4GB++sPDU3KtMOT1b8mEuKB+6BcsjT2gFI0u3iuk398dgk+hRw62xiBzL7Nwmt6N8CDfaOjW\\\\\\\/7XKhIhB8O7y8wCnBwVXICmdXUtnf4cH1x7YUXsAruV1hdI5HO5y1r2quPQz+ykOppT6KAHhF+RKY+EsofYb93fFyaxiMt2GvuoeG2\\\\\\\/xHeRWzhlJhBl7EmN48iTOwFcrxitvyyd75JAW2Oa9JQMxDJabXiFhG7Ny2j0YIq1czaqpu1PCjj0Xf6297CIbPBA5ZzU6L2SZtPikO+c6knJRVC0PlaYM64tU8TPtRdeEgdkgNUOBnCjDcWM0D7a1FX420EbQYNjYBsgt7ZWKnB9u6nr5xBV1pH\\\\\\\/CZCxxkNObsQ2EuyP9zMbb90jLjHOC0cw6BHapaE6NsFDuxAWopiD\\\\\\\/q6pFplwfmLmMq33X4+snCBZ9F9h46Hf6aSLbcXrtFuboZIsqA2m2IteDS\\\\\\\/iuISOFP9JKh7BjfvPUVVfqR+JL\\\\\\\/ts5qWZejZctP6glmlKknvM5KL9cDmgkPcSM5XT1wIyTMvXAiyfeMG+fxseprxGZ9KFKxnjFNqSA0Z8owfznavbRTlAAxxybZh3eQ6juGPHP1YwzdjQEvZsYTHG50Jx8BDlXjgN7EJUlTD+S1tR5srDeJBFOVSbnMa4cImkwPW8tt\\\\\\\/F7Tr89JpmfpxtUxegJ2v1I6d8d2MzK9BHYapVwEka\\\\\\\/0GINwA5UXDDSeGeWsMDerNt38fG0OXIgHzDqaR\\\\\\\/JfE2H7UUq2iRyHKISR8bf5mA3vIVi3eedEB+RYaU8mvYXOXw7BfRYRUXVohb6B6lEBROS3t7NxA8yCvQbZZjVmAtcz8ls\\\\\\\/DJ4gNABsAn2MIbNdZhial+0Te8m2VLbW1uRwmNOqWtuyQcoqCxutsbqpb6DW+n4GtWoK37c1U33LDPAQGSJ15CoLa1ULOGHvkBlTjSNKpx8iqaaxGkak\\\\\\\/c33M\\\\\\\/YExw2ApmpEj8y5eJiPt2Z0DdlXiM0H4ANVyHpJ+ZsWwLdF8DMbqNPo50NE1QnSni3glPsPJlWTkL8Y7B7zr\\\\\\\/vbKw5LAUFVN6SHrgdmtxfPiQlNgoLclEgfO0RN\\\\\\\/2XJLALLcdBuhf1H4tnHgf8g8J2IGSj4Q8qSdh27Mt7e8dPOCxYZ1PuTA8HTfbRUsct30nqIqLQPikLn2IvOoszSQLtgmMi5ST2GlqvvhHbUYXbaAOC4BnULYggrzUKQoE6dIIzjI0cZ\\\\\\\/YchHkeNGJozy\\\\\\\/K7tBfV+PvHiuPTFMTX7PHDJDakMtd8nbEoal4tbVNWeF9lwOEc++Ln4GwynoBQnUTzeuR1dSzEke9Ixc\\\\\\\/OUtwODPkR0FvwETqYPRWgKMkVjg3Yc1MJQ2ZaA9WD4IGrhFHKcax77pKZVRchpbQBaezTd11\\\\\\\/Eyk72Zl8K06A18+ZPCtYfJ4E2sjy5KiQfTgYkAserXPsps8o9QcRZQxR7r+uAJGJkwq55pNvE\\\\\\\/uTmmS8IEPdwGxrreit25duaMTgxfA8haEvfkIitpvMGSaT\\\\\\\/EWBx1V20fjkoShT7xBPJtH20jqaOyLbP0WddK96Afyg9Xq8dUKVJr8k81tEhlqZegRETZo1SsqFd3UsNN7pnI6XQ+vnZE52HiUKvMtdJ5TPtWdhTgkb0MDBWfCDJ6wVu+unhVpnLJozyfERF5zaWma3WhUI+3g0Oqu1ZxV38EM0USco8ZNnnAAeqZYiXiFbF6iJdFAqJ3Ih1mMCusXRBZYZxsnhA1F98RG03QWXZyBCx9F9WuTetWAxYDzOTYubm5t4z0OeImj613G4U078fT89Tavs4r\\\\\\\/YcHoc3a6dUTKM\\\\\\\/uInY0utbmTHrV97EGxwEw0WeI57vq5C63MbJjI096ZbCgLb2WGyQQdfuCi94veZA05Sn5MgG0SSTfD8XfLlke2g8NvssnqoIj\\\\\\\/kO4R6ELfMECuC1EwZYzBfsGQQP9wCPGKw5R316+Z7mwow==\\\u0022,\\\u0022iv\\\u0022:\\\u00227U8KsaOQ85vx\\\\\\\/6\\\\\\\/musVvDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f7c35c0c1bcbf93e603e4c57a414645ef3a72beb8c39b9bce20c6b91b36c358\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460388 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1767" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UJSl4eoJcUkqitXxefXqA8olVFD5aiF1gvzPJh46nm4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1767" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6CHe83BdE9g9oznw3ksNzO-8GcjIvUM6nqIiikDWAEY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1757" title="Genome Sequence and Analysis of a Stress-Tolerant, Wild-Derived Strain of Saccharomyces cerevisiae Used in Biofuels Research" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1777" title="A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460388" id="node460388" data-pisa="ggg;6/6/1767" data-pisa-master="ggg;g3.116.028001" data-apath="/ggg/6/6/1767.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Crossover Rate Variation on the <em>Caenorhabditis elegans</em> X Chromosome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6468-3953" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Max R. Bernstein</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6492-8906" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew V. Rockman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1767-1776;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028001 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460388" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Max R. Bernstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Max%2BR.%2BBernstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernstein%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMax%2BR.%2BBernstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6468-3953" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Max R. Bernstein</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew V. Rockman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BV.%2BRockman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rockman%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BV.%2BRockman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6492-8906" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew V. Rockman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mrockman@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">mrockman@nyu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/6/1767" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460388" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1767.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460388" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1767.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460388" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/6/1767.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460388" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/6/1767.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460388" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Meiotic recombination creates genotypic diversity within species. Recombination rates vary substantially across taxa, and the distribution of crossovers can differ significantly among populations and between sexes. Crossover locations within species have been found to vary by chromosome and by position within chromosomes, where most crossover events occur in small regions known as recombination hotspots. However, several species appear to lack hotspots despite significant crossover heterogeneity. The nematode <em>Caenorhabditis elegans</em> was previously found to have the least fine-scale variation in crossover distribution among organisms studied to date. It is unclear whether this pattern extends to the X chromosome given its unique compaction through the pachytene stage of meiotic prophase in hermaphrodites. We generated 798 recombinant nested near-isogenic lines (NILs) with crossovers in a 1.41 Mb region on the left arm of the X chromosome to determine if its recombination landscape is similar to that of the autosomes. We find that the fine-scale variation in crossover rate is lower than that of other model species, and is inconsistent with hotspots. The relationship of genomic features to crossover rate is dependent on scale, with GC content, histone modifications, and nucleosome occupancy being negatively associated with crossovers. We also find that the abundances of 4- to 6-bp DNA motifs significantly explain crossover density. These results are consistent with recombination occurring at unevenly distributed sites of open chromatin.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crossovers" class="hw-term hw-article-keyword hw-article-keyword-crossovers" rel="nofollow">crossovers</a></li><li class="kwd"><a href="/keyword/recombination-rate-variation" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate-variation" rel="nofollow">recombination rate variation</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/c-elegans" class="hw-term hw-article-keyword hw-article-keyword-c-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">C. elegans</em></a></li></ul><p id="p-3">Meiotic crossovers are required for proper segregation of chromosomes, and, in all organisms studied to date, there is considerable variation in where these crossovers tend to occur. This variation impacts the relative efficacy of selection within the genomes of sexually reproducing species (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hill and Robertson 1966</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Smukowski and Noor 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cutter and Payseur 2013</a>). Most studied eukaryotic species have higher crossover rates at peripheral regions of their chromosomes compared to the centers (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akhunov <em>et al.</em> 2003</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Barton <em>et al.</em> 2008</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rockman and Kruglyak 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chowdhury <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Roesti <em>et al.</em> 2013</a>). At this broad scale, a number of genomic features have been found to be positively correlated with recombination rate, including nucleotide diversity (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lercher and Hurst 2002</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cutter and Payseur 2003</a>), GC content, and gene density (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akhunov <em>et al.</em> 2003</a>). It should be noted that negative relationships have also been observed for GC content (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Drouaud <em>et al.</em> 2006</a>) and gene density (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Barnes <em>et al.</em> 1995</a>). These associations are not applicable to all species, and may not necessarily remain at scales finer than several hundred kilobases. Further, the inferred causal directions differ among these factors, with recombination rate variation thought to cause variation in nucleotide diversity and GC content, via selection and biased gene conversion, but variation in gene density thought to cause variation in recombination rate, at least at the mechanistic within-meiosis level, via chromatin state.</p><p id="p-4">At the kilobase level, recombination rate heterogeneity can be much more dramatic, and additional factors may affect crossover location. In most mammals, the H3K4-methyltransferase <em>PRDM9</em> plays a major role in determining the genome-wide distribution of recombination hotspots, which are 1- to 2-kb regions where crossover events occur at greatly elevated rates. However, the <em>PRDM9</em> zinc-finger domains have evolved rapidly in primates and rodents, which have different preferred binding sites that contribute to different hotspot landscapes within and among these species (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Baudat <em>et al.</em> 2013</a>). In dogs, which have lost <em>PRDM9</em> function, hotspots are located preferentially near gene promoter regions, particularly those with CpG islands (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Auton <em>et al.</em> 2013</a>). Hotspots near transcription start sites have also been documented in several plant species (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Drouaud <em>et al.</em> 2006</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hellsten <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Silva-Junior and Grattapaglia 2015</a>), birds (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Singhal <em>et al.</em> 2015</a>), and budding yeast (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tsai <em>et al.</em> 2010</a>). The nematode <em>Caenorhabditis elegans</em>, however, does not appear to have such extreme fine-scale recombination rate heterogeneity (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>).</p><p id="p-5">In <em>C. elegans</em>, recombination rate broadly varies according to physical position in all six of its chromosomes. Each chromosome is comprised of three large domains: a low-recombining, gene-dense center, and two high-recombining arms (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Barnes <em>et al.</em> 1995</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Rockman and Kruglyak 2009</a>). In addition, crossovers are absent from smaller domains adjacent to the telomeres. Unlike most other species with low recombination at chromosome centers, <em>C. elegans</em> chromosomes are holocentric and lack defined centromeres (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Albertson and Thomson 1982</a>). Furthermore, <em>C. elegans</em> exhibits near complete crossover interference, such that each chromosome has only one crossover per meiosis (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meneely <em>et al.</em> 2002</a>). This chromosome-level recombination rate variation, restricted number of crossovers, and low rates of outcrossing have implications for selection at linked sites within the <em>C. elegans</em> genome (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Cutter and Payseur 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rockman <em>et al.</em> 2010</a>).</p><p id="p-6">The <em>C. elegans</em> X chromosome experiences a considerably different meiotic environment than the autosomes. The X chromosome is unpaired in males and therefore does not recombine in the male germline, though males are very rare in wild populations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Barrière and Félix 2007</a>). In both males and hermaphrodites, the X is condensed and transcriptionally silent through pachytene in meiotic prophase (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kelly <em>et al.</em> 2002</a>); in fact X chromosomes that experience spermatogenesis (in both males and hermaphrodites) are silenced throughout sperm maturation and into early embryogenesis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bean <em>et al.</em> 2004</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arico <em>et al.</em> 2011</a>). There are several genes that uniquely impact the X chromosome during early meiosis, including <em><a href="http://www.wormbase.org/db/get?name=WBGene00001514;class=Gene">xnd-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene">him-5</a></em>, which have effects on double-strand break formation (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wagner <em>et al.</em> 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Meneely <em>et al.</em> 2012</a>). Furthermore, the X chromosome receives comparatively fewer double-strand breaks than the autosomes, which is thought to be a consequence of its condensed state (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gao <em>et al.</em> 2015</a>). Of the six <em>C. elegans</em> chromosomes, the X has the least differentiation in recombination rates between the arms and center (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Barnes <em>et al.</em> 1995</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Rockman and Kruglyak 2009</a>). Whether fine-scale variation on the X chromosome is similar to that of the autosomes is unknown (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>).</p><p id="p-7">Here, we provide the highest resolution <em>C. elegans</em> crossover information to date, focused on a 1.41-Mb region on the left arm of the <em>C. elegans</em> X chromosome. The interval covers 7.9% of the chromosome’s physical length, and has an expected genetic map length of 3.7 cM (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yook <em>et al.</em> 2012</a>). We performed genetic crosses to generate a large panel of nested near-isogenic lines (NILs). In the animals whose meiosis we studied, the targeted genomic interval is heterozygous for DNA from the Hawaiian wild isolate <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> and from the lab strain <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>, and the rest of the genome is entirely <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>. <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> is highly diverged from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andersen <em>et al.</em> 2012</a>), which enabled us to densely genotype recombinant lines. In total, we have crossover location data for 870 NILs. The data reveal that the crossover distribution in the focal region of the X chromosome exhibits a nonuniform density with low heterogeneity, similar to the pattern observed on an autosome. Chromatin modifications and DNA sequence features explain some of the residual variation observed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Overview</h3><p id="p-8">We generated strains with visible marker mutations on each side of a <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>-derived interval in the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background. In crosses between NIL and <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> strains carrying the visible markers in repulsion, we selected recombinants. Finally, we genotyped <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>/<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> SNPs within the interval to localize crossovers.</p></div><div id="sec-3" class="subsection"><h3>Strains</h3><p id="p-9">We constructed the sub-NIL panel using four starting strains. QX1321 is a NIL with <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> genomic DNA introgressed into the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background. The left boundary of the introgressed region lies between X:2,552,391 and X:3,157,209 and the right boundary is between the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em> locus and X:4,892,211. QG7 contains the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00146327;class=Variation">gm83</a></em>) mutation (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Much <em>et al.</em> 2000</a>), and was derived from strain <a href="http://www.wormbase.org/db/get?name=MU1080;class=Strain">MU1080</a>, originally acquired from the <em>Caenorhabditis</em> Genetic Center (CGC) with six generations of backcrossing to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>. Fax worms fail to make omega turns. QG144 contains the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143398;class=Variation">e678</a></em>) mutation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brenner 1974</a>) and was derived by crossing strain <a href="http://www.wormbase.org/db/get?name=CB3273;class=Strain">CB3273</a>, also originally acquired from the CGC, to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and selecting non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000315;class=Phenotype">Mec</a> recombinants. <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000022;class=Phenotype">Lon</a> worms are long and thin. <a href="http://www.wormbase.org/db/get?name=QG1;class=Strain">QG1</a> is a NIL (<em>qgIR1</em>), where X:4,754,307–4,864,273 is of <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> origin in an otherwise <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genomic background. This region includes the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> locus, which has a lab-derived gain-of-function mutation in <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> that has pleiotropic effects on phenotypes and gene expression (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">de Bono and Bargmann 1998</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McGrath <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2014</a>).</p></div><div id="sec-4" class="subsection"><h3>Cross design and experimental conditions</h3><p id="p-10">We thawed the starting worm strains from frozen stocks and cleaned them by bleaching (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stiernagle 2006</a>). We maintained worms at 20° on NGM agar plates seeded with <em>Escherichia coli</em> <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a> for several generations before we started experiments, which all took place at 20°.</p><p id="p-11">To generate strains with marker mutations linked to the NIL introgressed interval, we first crossed QG7 hermaphrodites with QG144 males to yield a <em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a> <a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em> double-mutant line, QG569. QX1321 (a wild-type NIL covering the focal interval) males were crossed to QG569 hermaphrodites to eventually produce mutant NILs. We used marker segregation along with indel and SNP genotyping to construct a strain, QG596, with <em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em> tightly linked to <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> DNA in an otherwise <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genomic background.</p><p id="p-12">As <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> illustrates, QG596 was crossed to <a href="http://www.wormbase.org/db/get?name=QG1;class=Strain">QG1</a> to produce F<sub>3</sub> homozygous recombinant single-mutant individuals, where recombinants possessing the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a></em> allele would also have <em>qgIR1</em>. The strains possessing the most <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> genome based on the same genotyping methods mentioned previously are the NIL parental strains QG613 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00146327;class=Variation">gm83</a>)</em>; <em>qgIR1</em>; <em>qgIR3</em>] and QG614 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143398;class=Variation">e678</a>)</em>; <em>qgIR4</em>].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the sub-NIL cross design focused on the variable X chromosome region. Colors represent regions of the chromosome originating from N2 (orange) and CB4856 (blue). (A) To generate the parental single-mutant NILs, the double-mutant NIL QG596 was crossed with QG1, an introgression line (qgIR1) containing &#x223C;110&#xA0;kb of the CB4856 X chromosome, including the npr-1 locus. F1 heterozygotes were allowed to self, and give rise to F2 progeny. Single-mutant (Fax or Lon) recombinant F2 hermaphrodites were picked and allowed to self. F3 homozygous recombinant progeny yielded F4 animals for genotyping. Strains QG613 and QG614 had the most residual CB4856 genome, and were chosen as the parental mutant NILs. (B) One of the two NIL crosses (FaxNIL). The fax-1 mutant NIL was crossed to a lon-2 mutant in the N2 background. The crossovers that were scored occurred in F1 hermaphrodites. Wild-type F2 animals were picked to individual plates to determine whether they were recombinants. Recombinant F2s were identified based on whether one of the mutant phenotypes was not observed in their F3 progeny. Homozygous sub-NILs were obtained in the F4 generation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055600917" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of the sub-NIL cross design focused on the variable X chromosome region. Colors represent regions of the chromosome originating from N2 (orange) and CB4856 (blue). (A) To generate the parental single-mutant NILs, the double-mutant NIL QG596 was crossed with QG1, an introgression line (&lt;em&gt;qgIR1&lt;/em&gt;) containing &#x223C;110&#xA0;kb of the CB4856 X chromosome, including the &lt;em&gt;npr-1&lt;/em&gt; locus. F&lt;sub&gt;1&lt;/sub&gt; heterozygotes were allowed to self, and give rise to F&lt;sub&gt;2&lt;/sub&gt; progeny. Single-mutant (Fax or Lon) recombinant F&lt;sub&gt;2&lt;/sub&gt; hermaphrodites were picked and allowed to self. F&lt;sub&gt;3&lt;/sub&gt; homozygous recombinant progeny yielded F&lt;sub&gt;4&lt;/sub&gt; animals for genotyping. Strains QG613 and QG614 had the most residual CB4856 genome, and were chosen as the parental mutant NILs. (B) One of the two NIL crosses (FaxNIL). The &lt;em&gt;fax-1&lt;/em&gt; mutant NIL was crossed to a &lt;em&gt;lon-2&lt;/em&gt; mutant in the N2 background. The crossovers that were scored occurred in F&lt;sub&gt;1&lt;/sub&gt; hermaphrodites. Wild-type F&lt;sub&gt;2&lt;/sub&gt; animals were picked to individual plates to determine whether they were recombinants. Recombinant F&lt;sub&gt;2&lt;/sub&gt;s were identified based on whether one of the mutant phenotypes was not observed in their F&lt;sub&gt;3&lt;/sub&gt; progeny. Homozygous sub-NILs were obtained in the F&lt;sub&gt;4&lt;/sub&gt; generation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1767/F1.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/6/1767/F1.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468402" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Overview of the sub-NIL cross design focused on the variable X chromosome region. Colors represent regions of the chromosome originating from N2 (orange) and CB4856 (blue). (A) To generate the parental single-mutant NILs, the double-mutant NIL QG596 was crossed with QG1, an introgression line (<em>qgIR1</em>) containing ∼110 kb of the CB4856 X chromosome, including the <em>npr-1</em> locus. F<sub>1</sub> heterozygotes were allowed to self, and give rise to F<sub>2</sub> progeny. Single-mutant (Fax or Lon) recombinant F<sub>2</sub> hermaphrodites were picked and allowed to self. F<sub>3</sub> homozygous recombinant progeny yielded F<sub>4</sub> animals for genotyping. Strains QG613 and QG614 had the most residual CB4856 genome, and were chosen as the parental mutant NILs. (B) One of the two NIL crosses (FaxNIL). The <em>fax-1</em> mutant NIL was crossed to a <em>lon-2</em> mutant in the N2 background. The crossovers that were scored occurred in F<sub>1</sub> hermaphrodites. Wild-type F<sub>2</sub> animals were picked to individual plates to determine whether they were recombinants. Recombinant F<sub>2</sub>s were identified based on whether one of the mutant phenotypes was not observed in their F<sub>3</sub> progeny. Homozygous sub-NILs were obtained in the F<sub>4</sub> generation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">To create the recombinant sub-NIL panel in which we mapped crossover positions, QG613 and QG614 were crossed to QG590 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143398;class=Variation">e678</a>)</em>; <em>qgIR1</em>] and QG591 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00146327;class=Variation">gm83</a>)</em>; <em>qgIR1</em>], respectively. At the F<sub>2</sub> generation, wild-type L4 hermaphrodites were picked to individual plates. Recombinant NILs were established from F<sub>3</sub> plates lacking one of the two mutant phenotypes (<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000022;class=Phenotype">Lon</a> or Fax). Plates founded by an F<sub>3</sub> recombinant hermaphrodite that yielded only wild-type progeny were frozen and given unique strain names.</p><p id="p-15">In approximately 1% of F<sub>3</sub> plates, all animals were wild-type, indicating that the F<sub>2</sub> parent had two recombinant chromosomes within the focal interval. These plates were discarded and did not contribute to the final total of NILs. Sub-NILs generated for subsequent analyses represent a powerful fine-mapping resource and are available from the authors.</p></div><div id="sec-5" class="subsection"><h3>Whole-genome sequencing of parental strains</h3><p id="p-16">To determine the exact breakpoints of the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> introgressed regions in the parental NILs, we sequenced the QG613 and QG614 genomes by Illumina HiSeq. We generated 100 bp paired-end reads, which were mapped to the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> reference genome using <em>stampy</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lunter and Goodson 2011</a>). Variant calling was performed using <em>samtools</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2009</a>). We identified a G &gt; A nonsense mutation (R128Opal) at X:3198393 (ws250) in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001400;class=Gene">fax-1</a></em> gene as <em><a href="http://www.wormbase.org/db/get?name=WBVar00146327;class=Variation">gm83</a></em>. The sequence of QG614 did not reveal any candidate SNPs for <em><a href="http://www.wormbase.org/db/get?name=WBVar00143398;class=Variation">e678</a></em> in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em> locus. Visualizing the BAM file for QG614 in the UCSC Genome Browser (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kent <em>et al.</em> 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Meyer <em>et al.</em> 2013</a>), we found two large regions in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em> locus to which no reads mapped. Based on sequence data for reads spanning regions that mapped and failed to map, we infer two deletions: X:4740894-4749007 and X:4749218-4749987 (ws250). The first deletion comprises almost the entire first intron through most of the ninth intron, and the second deletion covers 135 bp to 905 bp upstream of the coding region. Sequencing of QG590 also yielded these deletions in <em><a href="http://www.wormbase.org/db/get?name=WBGene00003056;class=Gene">lon-2</a></em>.</p></div><div id="sec-6" class="subsection"><h3>Genomic DNA preparation and genotyping</h3><p id="p-17">We isolated genomic DNA from worms using a salting-out protocol (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Rockman and Kruglyak 2009</a>). Strains were genotyped at a 191-bp indel polymorphism in the interval (X:4,103,162-4,103,352) to sort them into two groups (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FigureS1.pdf">Figure S1</a>). The indel genotype implies that the crossover location is either to the left or to the right of the indel, and strains were then genotyped at 139 SNPs on the side of the indel inferred to carry the crossover, plus an additional five SNPs on the other side of the indel as a control for genotyping error. We genotyped equal numbers of strains with breakpoints on the left and right sides, though the numbers of crossovers observed on each side were not equal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FigureS1.pdf">Figure S1</a>). Consequently, our downstream statistical analyses incorporate interval side as a covariate to control for our unrepresentative sampling of crossovers. Illumina GoldenGate genotyping was performed at the DNA Sequencing and Genomics Core Facility of the University of Utah, Salt Lake City.</p></div><div id="sec-7" class="subsection"><h3>Quality control of genotyping data</h3><p id="p-18">Scatterplots of fluorescence intensities were manually inspected using a previously created pipeline (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>). One probe yielded ambiguous genotypes and was excluded from analyses. The final data set includes 139 SNPs in the left interval, and 138 SNPs in the right interval. Strains that had multiple SNP genotyping failures, ambiguous calls in the region of recombination, and/or stretches of heterozygous calls were excluded from this analysis.</p></div><div id="sec-8" class="subsection"><h3>Statistical analyses</h3><p id="p-19">Statistical tests and analyses were performed in R (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">R Core Team 2015</a>). The data were analyzed at two resolutions: <strong>full</strong>, which includes all regions flanked by SNP probes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS1.csv">File S1</a>), and <strong>25kb</strong>, which comprises a more uniform distribution of SNP probe-bounded regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS2.csv">File S2</a>). Both of these files contain the physical location of each genotyped SNP, the number of crossover events that occurred between each pair of consecutive SNPs for each cross, and the identity of the interval side (left or right) that carries the SNP. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS3.csv">File S3</a> includes three regions that were omitted from analyses (see <em>Results</em>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS4.csv">File S4</a> contains an alternative <strong>25kb</strong> data set that includes the omitted intervals.</p></div><div id="sec-9" class="subsection"><h3>Analysis of recombination rate variation</h3><p id="p-20">To determine whether the distributions of recombination events between the two parental crosses differed, we performed two-sample, two-sided Kolmogorov-Smirnov tests, as described previously (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>).</p><p id="p-21">Modeling of discrete constant-rate domains within the two interval sides defined by GoldenGate array identity was performed using a previously designed program (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>). In brief, the R package <em>cobs</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ng and Maechler 2007</a>) was used to fit increasing numbers of constant-rate domains within the two interval halves defined by GoldenGate genotyping. Likelihood-ratio test statistics were tested against null distributions generated by simulation. As done previously, we examined up to a maximum of 40 knots, where a knot defines the boundary of a constant-rate domain.</p><p id="p-22">The level of recombination rate heterogeneity was estimated by calculating the Gini coefficient, which is a measure of inequality within a cumulative frequency distribution. The Gini coefficient can range from 0 (recombination is equally likely at every location) to 1 (all recombination occurs in one location). Gini coefficients were calculated using a previously designed program (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>). This program was also used to simulate crossover data, with the rate in each 1-kb bin drawn from a gamma distribution. Modifying the shape parameter for the gamma distribution allowed for examination of distributions of recombination rates ranging from highly heterogeneous to nearly uniform. Within a bin, the recombination rate is constant and crossovers are randomly positioned. For a given shape parameter value, we ran 5000 simulations. Crossovers produced by the simulation were then placed within intervals defined by our genetic markers. Gini coefficients were then calculated for each set of simulated recombination observations.</p></div><div id="sec-10" class="subsection"><h3>Genomic correlates of recombination rates</h3><p id="p-23">Genome sequence data used in all analyses were obtained from the WS225 release of WormBase (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Yook <em>et al.</em> 2012</a>). Polymorphism data were obtained from <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> resequencing (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Thompson <em>et al.</em> 2015</a>). Chromatin modification data were obtained from modENCODE (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/TableS1.pdf">Table S1</a>; <a href="http://modencode.org">http://modencode.org</a>). Annotations for other sequence information within the <em>C. elegans</em> genome were obtained from WormBase. We wrote an R script to identify nonoverlapping occurrences of all 4- to 8-bp dsDNA motifs within the focal genomic interval (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS5.zip">File S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS6.txt">File S6</a>).</p><p id="p-24">We used the generalized linear modeling function in R, <em>glm</em>, to model crossover count per marker interval as a Poisson-distributed random variable. We incorporated an offset term to account for variation in the distance between consecutive markers. The identity of the interval side (left or right) was included as a covariate in all models to account for our crossovers sampling design (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FigureS1.pdf">Figure S1</a>). To determine the significance of the crossover rate variation observed, we generated a null distribution by simulation of crossover events governed only by the physical distance (in <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) between SNP markers, creating 10,000 data sets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS7.zip">File S7</a>). The residual deviance from the observed data was compared to the distribution from the simulated datasets to determine a <em>P</em>-value. To determine the significance of the small DNA motifs, we generated a null distribution by sequence permutation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FileS8.zip">File S8</a>). We created 1000 datasets by permuting the DNA sequence within each marker interval, preserving each interval’s GC content, and we tested each motif as an explanatory variable in an analysis of each permuted dataset. For each dataset, the minimum residual deviance across all motifs was used as a summary statistic. For pairs of motifs, a similar procedure was done where all possible combinations were examined in the permuted DNA sequences. Additionally, we used MEME (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bailey <em>et al.</em> 2009</a>) to search for larger DNA motifs. We allowed for any number of repetitions of a given motif within the 10 highest recombining regions at 25 kb resolution, and searched both the given strand and its reverse complement. We searched for motifs ranging from 7 to 13 nt in length and used a first-order background model that incorporated mono- and dinucleotide composition. Only motifs with E values less than 10<sup>−10</sup> were examined.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-25">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><p id="p-26">We used visible markers to generate more than 1000 strains, each containing one crossover in a 1.48-Mb region on the X chromosome. As shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/FigureS1.pdf">Figure S1</a>, we obtained these recombinants from reciprocal crosses, which we refer to as the FaxNIL and LonNIL crosses. After genotyping the strains at an indel within the NIL interval (191-bp indel at X:4,103,162-4,103,352), we observed a striking difference between the two parental crosses. Sub-NILs that arose from the FaxNIL cross had many more recombination events to the left of the indel than the right, whereas the proportion was roughly equal for the sub-NILs arising from the LonNIL cross (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468424/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468424/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Crossover distribution on left or right of indel at X:4.103 Mb</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">We obtained crossover locations for 870 lines across 277 SNPs along the 1.48-Mb interval (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The median distance between consecutive markers is approximately 4.4 kb, with a mean distance of 5.4 kb. Of the 870 lines, 56 had crossovers that occurred between one of the visible marker mutations, and either the first or last of the 277 marker SNPs, making the resulting sub-NILs either completely <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> or completely <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> within this interval. Another 16 lines had crossovers in regions that are unshared between the two parental crosses (left and right edges in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Less than 9% (22/258) of the intervals between consecutive probes in the shared 1.41 Mb had no observed crossovers, and 92% of the crossover events mapped to intervals of less than 10 kb. The largest distance between observed recombination breakpoints among strains is 41.5 kb (X:4,020,535-4,062,018), of which 30.5 kb is contiguous tandem repetitive DNA in <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (X:4,025,997-4,056,509). This is the third-longest stretch of tandem repetitive DNA (other than the rDNA repeats) in the <em>C. elegans</em> genome (WormBase). The crossover distribution on the left side of the region (3.28–4.10 Mb) significantly differs between the FaxNIL cross and LonNIL cross (<em>P</em> = 0.03), but does not differ on the right side (4.10–4.68 Mb; <em>P</em> = 0.26; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotypes within the focal interval in sub-NILs. Each sub-NIL is illustrated as a horizontal line, where the transition from N2 (orange) to CB4856 (blue) genome, or vice versa, marks the region where the crossover occurred. The &#x201C;blue triangle&#x201D; on top encompasses all sub-NILs derived from the LonNIL cross, while the one on the bottom encompasses those derived from the FaxNIL cross. Sub-NILs that are either all N2 or all CB4856 (56 in total) within the focal interval are not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055600917" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotypes within the focal interval in sub-NILs. Each sub-NIL is illustrated as a horizontal line, where the transition from N2 (orange) to CB4856 (blue) genome, or &lt;em&gt;vice versa&lt;/em&gt;, marks the region where the crossover occurred. The &#x201C;blue triangle&#x201D; on top encompasses all sub-NILs derived from the LonNIL cross, while the one on the bottom encompasses those derived from the FaxNIL cross. Sub-NILs that are either all N2 or all CB4856 (56 in total) within the focal interval are not shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1767/F2.medium.gif" width="440" height="289"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/6/1767/F2.medium.gif" width="440" height="289"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468268" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Genotypes within the focal interval in sub-NILs. Each sub-NIL is illustrated as a horizontal line, where the transition from N2 (orange) to CB4856 (blue) genome, or <em>vice versa</em>, marks the region where the crossover occurred. The “blue triangle” on top encompasses all sub-NILs derived from the LonNIL cross, while the one on the bottom encompasses those derived from the FaxNIL cross. Sub-NILs that are either all N2 or all CB4856 (56 in total) within the focal interval are not shown.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossover distributions between parental crosses on the left and right sides of the focal interval. The Marey map representation relates cumulative genetic distance to physical distance along the chromosome. On the left side of the interval, there is a 30.5&#xA0;kb tandem repetitive element that did not recombine at approximately X:4.02&#xA0;Mb. Notches at the bottom of each plot denote Illumina GoldenGate SNP probes. The two parental crosses significantly differ in their crossover distributions on the left side (P&#xA0;=&#xA0;0.03, Kolmogorov-Smirnov test), but not on the right side (P&#xA0;=&#xA0;0.26)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055600917" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossover distributions between parental crosses on the left and right sides of the focal interval. The Marey map representation relates cumulative genetic distance to physical distance along the chromosome. On the left side of the interval, there is a 30.5&#xA0;kb tandem repetitive element that did not recombine at approximately X:4.02&#xA0;Mb. Notches at the bottom of each plot denote Illumina GoldenGate SNP probes. The two parental crosses significantly differ in their crossover distributions on the left side (&lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.03, Kolmogorov-Smirnov test), but not on the right side (&lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.26).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1767/F3.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/6/1767/F3.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468333" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Crossover distributions between parental crosses on the left and right sides of the focal interval. The Marey map representation relates cumulative genetic distance to physical distance along the chromosome. On the left side of the interval, there is a 30.5 kb tandem repetitive element that did not recombine at approximately X:4.02 Mb. Notches at the bottom of each plot denote Illumina GoldenGate SNP probes. The two parental crosses significantly differ in their crossover distributions on the left side (<em>P</em> = 0.03, Kolmogorov-Smirnov test), but not on the right side (<em>P</em> = 0.26).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-13" class="subsection"><h3>Variation in recombination rate across the interval</h3><p id="p-31">We assessed the minimal number of constant recombination rate domains required to explain the observed data, if such domains existed. The simplest piecewise linear regression models that could not be rejected at <em>P</em> = 0.05 for the observed data suggest a minimum of 23 subdomains of constant recombination rate, with 20 in the 820 kb to the left of the indel at 4.10 Mb, and three in the 590 kb to the right (based on 100,000 simulations per model).</p><p id="p-32">Three regions within the focal interval harbor large length polymorphisms or repeat sequence, and these require special consideration. An interval described above includes a 30.5 kb tandem repetitive DNA tract (at least in the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genome) that is very highly enriched for histone modifications indicative of heterochromatin, particularly H3K9 methylation. Prior work in <em>Drosophila</em> found that satellite DNA organization within the genome is regulated in a similar manner to that of ribosomal DNA, where both are similarly enriched with H3K9me2 modifications. It is thought that recombination in these regions can lead to genome instability (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Peng and Karpen 2006</a>). For two other marker intervals, recent resequencing of <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> identified the absence of two duplications that are present in <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>, both of which are more than 6 kb in length (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Noble <em>et al.</em> 2015</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Thompson <em>et al.</em> 2015</a>). As each of these indels represent &gt; 40% of the physical distance bound by the markers defining their respective bins, we decided to remove these bins, along with the bin containing the tandem repeats, from the analyses below.</p><p id="p-33">We evaluated the data at two resolutions, as prior work has found that the explanatory power of genomic features depends on the scale of analysis (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cirulli <em>et al.</em> 2007</a>); <strong>full</strong> incorporates all of the intervals between genotyped informative SNPs (<em>n</em> = 256 bins, median distance = 4.4 kb, range: 3.3–18.2 kb; median of three breakpoints per interval), and <strong>25kb</strong> has a more uniform distribution of bin sizes, and all bins have at least three recombination events (<em>n</em> = 53 bins, median distance = 25.4 kb, range: 21.3–29.9 kb; median of 14 breakpoints per interval). We first tested the simplest model of crossover distribution, a single constant rate across the interval. We fitted the observed crossover data to a generalized linear model incorporating only physical distance between markers as an explanatory variable and interval side (left <em>vs.</em> right) as a covariate that accounts for our sampling design. The residual deviance of this simple model is significantly higher in both the <strong>25kb</strong> and <strong>full</strong> datasets than in datasets simulated with uniform recombination (<em>P</em> = 0.014 and <em>P</em> = 0.004, respectively), indicating that additional factors contribute to the observed heterogeneity (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant recombination rate heterogeneity. The histograms show the residual deviance under a model of uniform recombination rate for data simulated according to our sampling design. Red vertical line represents the deviance of the observed data. (Left) Full resolution, in which bin sizes range from 3.2 to 18.2&#xA0;kb. P&#xA0;=&#xA0;0.004. (Right) 25&#xA0;kb resolution, in which bin sizes range from 21.3 to 29.9&#xA0;kb. P&#xA0;=&#xA0;0.014." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055600917" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Significant recombination rate heterogeneity. The histograms show the residual deviance under a model of uniform recombination rate for data simulated according to our sampling design. Red vertical line represents the deviance of the observed data. (Left) Full resolution, in which bin sizes range from 3.2 to 18.2&#xA0;kb. &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.004. (Right) 25&#xA0;kb resolution, in which bin sizes range from 21.3 to 29.9&#xA0;kb. &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.014.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1767/F4.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/6/1767/F4.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468474" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Significant recombination rate heterogeneity. The histograms show the residual deviance under a model of uniform recombination rate for data simulated according to our sampling design. Red vertical line represents the deviance of the observed data. (Left) Full resolution, in which bin sizes range from 3.2 to 18.2 kb. <em>P</em> = 0.004. (Right) 25 kb resolution, in which bin sizes range from 21.3 to 29.9 kb. <em>P</em> = 0.014.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">To assess recombination rate heterogeneity, we calculated a Gini coefficient for the <strong>full</strong> data set. The Gini coefficient serves as an inequality measure for the observed crossover distribution. A value of 0 indicates perfect equality (<em>i.e.</em>, all locations have the same recombination rate), and a value of 1 indicates perfect inequality (<em>i.e.</em>, all recombination occurs in the same location). The left and right interval halves have Gini coefficients of 0.368 and 0.332, respectively (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). These values are considerably lower than what has been found in humans and yeast, with coefficients averaging 0.8 and 0.65, respectively (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kong <em>et al.</em> 2010</a>; <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>). Estimates of human and primate Gini coefficients from linkage disequilibrium (LD) data are closer to 0.7, lower than the estimates based on direct measurements of crossovers, perhaps because of the influence of population size variation on LD-based estimates (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stevison <em>et al.</em> 2016</a>). LD-based estimates in <em>Drosophila pseudoobscura</em> found a more modest Gini coefficient of 0.50 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Heil <em>et al.</em> 2015</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination rate heterogeneity along the C. elegans X chromosome left arm is modest compared to that observed in other model species. The Gini coefficient is the proportion of area above the curve and below the diagonal in a plot comparing cumulative physical distance, where the intervals between consecutive SNP probes (points on curves) are ordered by recombination rate, to cumulative genetic distance. The left (orange) and right (black) sides of the focal region exhibit similar low levels of rate heterogeneity, with Gini coefficients of 0.368 and 0.332, respectively. Representative simulated distributions of recombination rates with Gini coefficients of 0.65 (red) and 0.8 (blue) reflect the heterogeneity found in budding yeast and mammals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055600917" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Recombination rate heterogeneity along the &lt;em&gt;C. elegans&lt;/em&gt; X chromosome left arm is modest compared to that observed in other model species. The Gini coefficient is the proportion of area above the curve and below the diagonal in a plot comparing cumulative physical distance, where the intervals between consecutive SNP probes (points on curves) are ordered by recombination rate, to cumulative genetic distance. The left (orange) and right (black) sides of the focal region exhibit similar low levels of rate heterogeneity, with Gini coefficients of 0.368 and 0.332, respectively. Representative simulated distributions of recombination rates with Gini coefficients of 0.65 (red) and 0.8 (blue) reflect the heterogeneity found in budding yeast and mammals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1767/F5.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/6/1767/F5.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468320" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Recombination rate heterogeneity along the <em>C. elegans</em> X chromosome left arm is modest compared to that observed in other model species. The Gini coefficient is the proportion of area above the curve and below the diagonal in a plot comparing cumulative physical distance, where the intervals between consecutive SNP probes (points on curves) are ordered by recombination rate, to cumulative genetic distance. The left (orange) and right (black) sides of the focal region exhibit similar low levels of rate heterogeneity, with Gini coefficients of 0.368 and 0.332, respectively. Representative simulated distributions of recombination rates with Gini coefficients of 0.65 (red) and 0.8 (blue) reflect the heterogeneity found in budding yeast and mammals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The Gini coefficients for both sides of the studied interval are higher than the 0.28 observed for the 214 kb to the right of the center-right arm boundary on Chromosome II (<a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>). One difference between the Chromosome II and Chromosome X studies is simply a matter of scale; the Chromosome X interval examined here is approximately seven times larger than the 214-kb Chromosome II right-arm interval. Due to the number of strains analyzed over this interval, we actually have fewer crossover events in 210–220 kb windows (115 on average for the left side, and 150 on average for the right side) than the 218 obtained on Chromosome II. Upon down-sampling the Chromosome II data, and calculating Gini coefficients for 100,000 permutations, we find that our observed Gini coefficients are not significantly different than 0.28 (<em>P</em> &gt; 0.14 for left and right sides).</p><p id="p-38">To determine whether the Gini coefficients are compatible with the presence of hotspots, we simulated crossovers by drawing recombination rates from a gamma distribution. By altering the shape parameter of the gamma, we examined recombination rate landscapes ranging from hotspot-rich to nearly constant. For reference, a shape parameter of 1 produces an exponential distribution, and, as this parameter increases, heterogeneity in the distribution decreases. From these simulations, we calculated Gini coefficients and compared the results to our observed crossover data. For both the left and right sides of the studied genomic interval, our data do not conform to a highly heterogeneous recombination rate landscape, as their 95% confidence intervals do not include models whose most recombinant kilobase is expected to have a recombination rate ≥ 10 times the mean rate. The models that best approximate the left and right side of our interval have shape parameters of 1.70 and 3.57, respectively. These shape parameters correspond to distributions where the most recombinant kilobase is expected to have a crossover rate approximately 5.3 and 3.4 times the mean rate, respectively.</p></div><div id="sec-14" class="subsection"><h3>Genomic correlates of recombination rate</h3><p id="p-39">GC content was previously found to be associated with the shift in recombination rate across the Chromosome II center-right arm boundary region (<a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>), with higher GC content in the lower-recombination center. Within the 1.41 Mb common region among the sub-NILs, GC content was not associated with variation in recombination rate for the <strong>full</strong> dataset, but it had a significant negative effect at <strong>25kb</strong> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The same effect was observed with ChIP-chip data for mononucleosome occupancy (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) and histone modifications, many of which are highly correlated with each other (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/TableS2.pdf">Table S2</a>). Several other sequence-level parameters, which have been found to influence recombination rate in other organisms, had no observed effect in this region at both scales, including gene density (<em>i.e.</em>, the number of transcription start sites per basepair; a proxy for assessing promoters), amount of intergenic DNA between nonconvergently transcribed genes (a second proxy for assessing promoters), and amount of intronic DNA (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468418/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468418/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genomic correlates for recombination rate variation at 25 kb resolution</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histone modification heat map. The correlation matrix of modifications and the IgG binding control are hierarchically clustered. Asterisks below the plot denote significance (using raw P-values) of modifications in individually explaining deviance in the observed crossover distribution. All nominally significant associations are negative. *&#xA0;P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055600917" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histone modification heat map. The correlation matrix of modifications and the IgG binding control are hierarchically clustered. Asterisks below the plot denote significance (using raw &lt;em&gt;P&lt;/em&gt;-values) of modifications in individually explaining deviance in the observed crossover distribution. All nominally significant associations are negative. *&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.05, **&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/6/1767/F6.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/6/1767/F6.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/6/1767/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468264" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Histone modification heat map. The correlation matrix of modifications and the IgG binding control are hierarchically clustered. Asterisks below the plot denote significance (using raw <em>P</em>-values) of modifications in individually explaining deviance in the observed crossover distribution. All nominally significant associations are negative. * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">We next investigated more specific DNA sequence associations with recombination rate variation. Poly(A) repeats were previously found to be positively correlated with recombination rate in budding yeast (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Mancera <em>et al.</em> 2008</a>) and <em>Drosophila melanogaster</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Comeron <em>et al.</em> 2012</a>). Poly(A) ≥ 4 exhibits a significant positive association with crossover rate at <strong>25kb</strong> (<em>P</em> &lt; 0.001). However, (CA)<sub>n</sub> repeats found to be highly associated with crossovers in those same species (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gendrel <em>et al.</em> 2000</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Comeron <em>et al.</em> 2012</a>) do not explain the variation in crossover rate we observed; at best, (CA) ≥ 5 is marginally significant (<em>P</em> = 0.06, <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-43">We also performed an exhaustive scan of all DNA motifs ranging from 4 to 8 bp in length, controlling for the large number of tests by a permutation procedure. Significant motifs are described in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. For the <strong>full</strong> data set, AACA explains a significant portion of the deviance in a generalized linear model, and is positively associated with crossover rate. For the <strong>25kb</strong> data set, GGATAG significantly explains the crossover rate variation observed, and is negatively associated. Removal of deletion polymorphisms present in <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> from the examined sequence does not affect the significance of the relationships (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1/TableS3.pdf">Table S3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468393/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468393/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Small DNA motifs with significant effects on crossover rate</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">Additionally, we conducted a MEME search for longer motifs at 25 kb resolution. We examined motifs that passed the E-value &lt; 10<sup>−10</sup> threshold by counting occurrences within all 53 bins, and incorporating the occurrence rate vector as a variable in a generalized linear model. The most explanatory motif we found is AAAA[AT]AT[TC]AT[AT]T, which is positively associated with crossover rate (nominal <em>P</em>-value = 1.32 × 10<sup>−4</sup>). However, this <em>P</em>-value is similar to the <em>P</em>-value observed for poly(A) ≥ 4, which occurs once within the discovered motif, and is almost three orders of magnitude larger than the nominal <em>P</em>-value of GGATAG at this resolution. Lowering the E-value threshold to 10<sup>−5</sup> did not yield a more explanatory motif.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-46">We have generated a large, permanent mapping resource encompassing 798 recombination events within a 1.41-Mb span on the left arm of the <em>C. elegans</em> X chromosome. At a median resolution of about 4.4 kb, less than 10% of intervals lack a crossover event. We observed significant heterogeneity in recombination rate across the interval, though consistent with prior findings (<a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">Kaur and Rockman 2014</a>), there is no evidence for local recombination hotspots. Interestingly, simple DNA sequence features, including GC content and small sequence motifs, explain a significant portion of the observed heterogeneity.</p><p id="p-47">One of the intervals lacking a crossover in the NIL panel contains a very large tandem repetitive region where recombination may be suppressed. In budding yeast, the repetitive rDNA array is shielded from nonallelic homologous recombination by the histone deacetylase Sir2, and is maintained in a heterochromatic state during meiosis (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gottlieb and Esposito 1989</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mieczkowski <em>et al.</em> 2007</a>). ChIP-chip data for nucleosome occupancy and H3K9me2/3 in <em>C. elegans</em> embryos and adults show high enrichment in this repetitive region, which may support a similar function in recombination suppression.</p><p id="p-48">Within this X chromosome region, we found two large deletions in the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> genome relative to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>. In both cases, the genes missing in <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> appear to be tandem duplicates of neighboring genes. The <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allele of the indel at X:3.56 Mb is present in at least three other wild-isolated strains (<a href="http://www.wormbase.org/db/get?name=JU319;class=Strain">JU319</a>, <a href="http://www.wormbase.org/db/get?name=MY16;class=Strain">MY16</a>, and <a href="http://www.wormbase.org/db/get?name=AB2;class=Strain">AB2</a>), and the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allele of the indel at X:4.33 Mb is present in at least three different wild-isolated strains (<a href="http://www.wormbase.org/db/get?name=JU345;class=Strain">JU345</a>, <a href="http://www.wormbase.org/db/get?name=JU1171;class=Strain">JU1171</a>, and <a href="http://www.wormbase.org/db/get?name=QX1211;class=Strain">QX1211</a>; sequence data from <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Noble <em>et al.</em> 2015</a>). Both of these indels may have had an impact on local recombination rate due to the physical distances being markedly different between the two strains. At the finest-scale resolution we have, the distances between consecutive probes encompassing these large deletions in the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genome are both over 14 kb, and only one recombination event was observed in each. Assuming a uniform recombination rate using <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> interval sizes, we would have expected 6.7 and 10.0 crossovers for those regions on the left and right, respectively. The effect of indel heterozygosity on the crossover rate of nearby flanking sequence is unknown.</p><p id="p-49">Interestingly, we found that the FaxNIL cross produced significantly more crossovers on the left side of the interval than did the LonNIL cross. Furthermore, the distribution of crossovers on the left differs significantly between the crosses. However, we found that the correlates of recombination rate variation are concordant between the crosses, when examining each one individually. Because the size and location of the starting <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> genomic fragment differs slightly in the two crosses, we checked the unshared 75 kb for candidate genes that are involved in meiosis and/or chromatin remodeling; variants in these regions could act <em>in trans</em> to yield different crossover distributions in the two crosses. None of the genes in the variable region have a known function in either of those two processes. Another possible explanation for this cross effect is a deleterious epistatic interaction between two or more alleles that only occurs in one direction, either on the left of the LonNIL cross or on the right of the FaxNIL cross. For example, in the former scenario, the interaction could involve a <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allele on the left side of a crossover interacting with an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allele on its right side. The converse would apply to the latter scenario. Since we did not observe any obvious lethality in either cross, the effect may have been a slower development time, which could have been selected against based on the timing of picking F<sub>2</sub>s.</p><p id="p-50">We found multiple correlated sequence-level features that significantly explain the crossover distribution we observed, though, in all but one instance, this was at ∼25 kb resolution. The present dataset is likely underpowered to detect many associations at a finer scale. Additionally, due to unique features of our focal interval and our experimental design, we are unable to fully test the relationship of nuclear envelope attachment via <a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene">LEM-2</a> or nucleotide diversity with recombination rate variation. In the former case, <a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene">LEM-2</a> binding to the X chromosome drops off to the right of the large tandem repetitive sequence at 4.02 Mb (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ikegami <em>et al.</em> 2010</a>), which is near the indel used for preliminary genotyping. <a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene">LEM-2</a> binding ends up being near-perfectly confounded with the interval side covariate. We see no relationship between <a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene">LEM-2</a> and crossover location when analyzing the left and right sides separately. A similar situation exists for nucleotide diversity, where marker intervals on the right have increased diversity compared to the left (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Thompson <em>et al.</em> 2015</a>) and we do not observe an effect when analyzing the sides separately.</p><p id="p-51">Genome sequence features influence many activities within nuclei, including transcriptional state via histone modifications and nucleosome occupancy. These features also influence overall chromatin conformation, which may affect where crossovers occur during meiosis. At the 20–30 kb scale, we found several correlated associations of genome features with recombination rate variation. Notably, GC content, a suite of histone modifications, and nucleosome occupancy are negatively associated with recombination rate, while poly(A) is positively associated with recombination rate within our focal region. These associations may share an underlying mechanism. Prior work in <em>C. elegans</em> found that nucleosome occupancy in both early embryos and L3 larvae correlates strongly with GC content on the X chromosome, particularly at gene promoters (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ercan <em>et al.</em> 2011</a>). Furthermore, nucleosomes are depleted in core promoters that have many interspersed blocks of consecutive As or Ts (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Grishkevich <em>et al.</em> 2011</a>).</p><p id="p-52">In mice lacking functional <em>Prdm9</em>, and in budding yeast, crossovers preferentially occur at promoters (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Brick <em>et al.</em> 2012</a>). We found no significant association between recombination rate and either gene number or intergenic distance between nonconvergently transcribed genes. Furthermore, the H3K4me3 distribution in the focal interval (measured in early embryos) does not show the positive association with recombination rate that it does in mice and yeast. These results suggest that the targets for crossovers on the X chromosome are regions lacking markers of heterochromatin and nucleosomes. Importantly, there does not appear to be a necessity for histone marks associated with active transcription (<em>e.g.</em>, H3K4me3) to mediate where crossovers occur. It is unclear whether this is a unique feature of the X chromosome or applicable to all <em>C. elegans</em> chromosomes.</p><p id="p-53">Neither of the small motifs we discovered has been previously described as affecting recombination rate in any organism. As the only longer motif identified through MEME is less explanatory than these sequences, it is possible that the mechanisms underlying crossover distribution in <em>C. elegans</em> differ markedly from those in other studied species.</p><p id="p-54">In summary, our findings suggest significant heterogeneity exists within the high-recombination-rate left arm of the X chromosome in spite of a lack of crossover hotspots. As in other fine-scale recombination studies, the scale at which data are analyzed can affect the significance of explanatory variables; in this case the fine-scale data we have may be underpowered to detect some associations. Despite its unique regulation during early meiotic prophase, the X chromosome has a crossover landscape similar to that of Chromosome II. The crossover distribution is shaped in part by reduced recombination in heterochromatic regions and sites bound by nucleosomes. We do not find evidence of large DNA motifs that greatly explain the variation we observe, which may be a result of a recombination rate architecture that lacks well-defined hotspots.</p></div><div class="section" id="sec-16"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank Christina Dai for assistance in generating sub-NILs. We thank David Riccardi, Jasmine Nicodemus, and Jia Shen for assistance in the lab, and we thank David Riccardi and the New York University (NYU) Center for Genomics &amp; Systems Biology Genome Core facility for assistance in sequencing the parental worm strains. We thank members of the Rockman lab for helpful suggestions on manuscript drafts. We thank the <em>Caenorhabditis</em> Genetics Center, which is funded by the National Institutes of Health (NIH) Office of Research Infrastructure Programs (P40OD010440), for strains, and WormBase for maintaining and curating databases. This work was supported by the NIH (R01GM089972).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028001/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-57"><em>Communicating editor: M. C. Zetka</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-02-08"><span class="received-label">Received </span>February 8, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-12"><span class="accepted-label">Accepted </span>April 12, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Bernstein and Rockman</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.1" data-doi="10.1101/gr.808603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodyear</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">L.–L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echalier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The organization and rate of evolution of wheat genomes are correlated with recombination rates along chromosome arms.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.808603%26rft_id%253Dinfo%253Apmid%252F12695326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzUvNzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.2" data-doi="10.1007/BF00292267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albertson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The kinetochores of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DAlbertson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D409%26rft.epage%253D428%26rft.atitle%253DThe%2Bkinetochores%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00292267%26rft_id%253Dinfo%253Apmid%252F7172865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00292267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7172865&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PP41500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.3" data-doi="10.1038/ng.1050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crissman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.4" data-doi="10.1371/journal.pgen.1004156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A variant in the neuropeptide receptor <em>npr-1</em> is a major determinant of <em>Caenorhabditis elegans</em> growth and physiology.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004156%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004156%26rft_id%253Dinfo%253Apmid%252F24586193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586193&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.5" data-doi="10.1371/journal.pgen.1001391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arico</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Vlag</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Epigenetic patterns maintained in early <em>Caenorhabditis elegans</em> embryos can be established by gene activity in the parental germ cells.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DArico%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1001391%26rft.epage%253De1001391%26rft.atitle%253DEpigenetic%2Bpatterns%2Bmaintained%2Bin%2Bearly%2BCaenorhabditis%2Belegans%2Bembryos%2Bcan%2Bbe%2Bestablished%2Bby%2Bgene%2Bactivity%2Bin%2Bthe%2Bparental%2Bgerm%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001391%26rft_id%253Dinfo%253Apmid%252F21695223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695223&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.6" data-doi="10.1371/journal.pgen.1003984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rui Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic recombination is targeted towards gene promoter regions in dogs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003984%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003984%26rft_id%253Dinfo%253Apmid%252F24348265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003984&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24348265&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buske</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MEME SUITE: tools for motif discovery and searching.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">W202–W208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D37%26rft.spage%253DW202W208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.8" data-doi="10.1038/msb.2012.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hekimi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Meiotic recombination, noncoding DNA and genomic organization in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBarnes%26rft.auinit1%253DT.%2BM.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D159%26rft.epage%253D179%26rft.atitle%253DMeiotic%2Brecombination%252C%2Bnoncoding%2BDNA%2Band%2Bgenomic%2Borganization%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2012.19%26rft_id%253Dinfo%253Apmid%252F8536965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzEvMTU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.9" data-doi="10.1534/genetics.106.067223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Félix</span>  <span class="cit-name-given-names">M.-A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Temporal dynamics and linkage disequilibrium in natural <em>Caenorhabditis elegans</em> populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067223%26rft_id%253Dinfo%253Apmid%252F17409084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzIvOTk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.10" data-doi="10.1534/genetics.107.083493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pekosz</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurvathi</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Meiotic recombination at the ends of chromosomes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083493%26rft_id%253Dinfo%253Apmid%252F18562657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzEyMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.11" data-doi="10.1038/nrg3573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Massy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Meiotic recombination in mammals: localization and regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">794</span>–<span class="cit-lpage">806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D794%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3573%26rft_id%253Dinfo%253Apmid%252F24136506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24136506&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.12" data-doi="10.1038/ng1283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Meiotic pairing and imprinted X chromatin assembly in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBean%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D100%26rft.epage%253D105%26rft.atitle%253DMeiotic%2Bpairing%2Band%2Bimprinted%2BX%2Bchromatin%2Bassembly%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1283%26rft_id%253Dinfo%253Apmid%252F14702046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14702046&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187666800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.13" data-doi="10.1051/medsci/20031967725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrenner%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D94%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fmedsci%252F20031967725%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzcvMS83MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.14" data-doi="10.1038/nature11089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brick</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smagulova</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khil</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camerini-Otero</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petukhova</span>  <span class="cit-name-given-names">G. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic recombination is directed away from functional genomic elements in mice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBrick%26rft.auinit1%253DK.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D642%26rft.epage%253D645%26rft.atitle%253DGenetic%2Brecombination%2Bis%2Bdirected%2Baway%2Bfrom%2Bfunctional%2Bgenomic%2Belements%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11089%26rft_id%253Dinfo%253Apmid%252F22660327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660327&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.15" data-doi="10.1371/journal.pgen.1000648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chowdhury</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bois</span>  <span class="cit-name-given-names">P. R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feingold</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">V. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic analysis of variation in human meiotic recombination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChowdhury%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000648%26rft.epage%253De1000648%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bvariation%2Bin%2Bhuman%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000648%26rft_id%253Dinfo%253Apmid%252F19763160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19763160&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.16" data-doi="10.1007/s00239-006-0142-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cirulli</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fine-scale crossover rate heterogeneity in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCirulli%26rft.auinit1%253DE.%2BT.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D135%26rft.atitle%253DFine-scale%2Bcrossover%2Brate%2Bheterogeneity%2Bin%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0142-7%26rft_id%253Dinfo%253Apmid%252F17160365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0142-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17160365&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243047800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.17" data-doi="10.1371/journal.pgen.1002905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnappan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The many landscapes of recombination in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002905%26rft.epage%253De1002905%26rft.atitle%253DThe%2BMany%2BLandscapes%2Bof%2BRecombination%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002905%26rft_id%253Dinfo%253Apmid%252F23071443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071443&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.18" data-doi="10.1093/molbev/msg072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Selection at linked sites in the partial selfer <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg072%26rft_id%253Dinfo%253Apmid%252F12679551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNjY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.19" data-doi="10.1038/nrg3425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic signatures of selection at linked sites: unifying the disparity among species.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3425%26rft_id%253Dinfo%253Apmid%252F23478346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23478346&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.20" data-doi="10.1016/S0092-8674(00)81609-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Bono</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Natural variation in a neuropeptide Y receptor homolog modifies social behavior and food response in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253Dde%2BBono%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D5%26rft.spage%253D679%26rft.epage%253D689%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bneuropeptide%2BY%2Breceptor%2Bhomolog%2Bmodifies%2Bsocial%2Bbehavior%2Band%2Bfood%2Bresponse%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981609-8%26rft_id%253Dinfo%253Apmid%252F9741632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81609-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9741632&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075851900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.21" data-doi="10.1101/gr.4319006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drouaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camilleri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourguignon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canaguier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Variation in crossing-over rates across chromosome 4 of <em>Arabidopsis thaliana</em> reveals the presence of meiotic recombination “hot spots.”</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4319006%26rft_id%253Dinfo%253Apmid%252F16344568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvMTA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.22" data-doi="10.1101/gr.115931.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ercan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubling</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High nucleosome occupancy is encoded at X-linked gene promoters in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115931.110%26rft_id%253Dinfo%253Apmid%252F21177966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMjM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.23" data-doi="10.1371/journal.pgen.1005029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elia</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiácovo</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">NatB domain-containing CRA-1 antagonizes hydrolase ACER-1 linking acetyl-CoA metabolism to the initiation of recombination during <em>C. elegans</em> meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005029%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005029%26rft_id%253Dinfo%253Apmid%252F25768301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25768301&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gendrel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutreix</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">(CA/GT)n microsatellites affect homologous recombination during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGendrel%26rft.auinit1%253DC.-G.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D1261%26rft.epage%253D1268%26rft.atitle%253D%2528CA%252FGT%2529n%2Bmicrosatellites%2Baffect%2Bhomologous%2Brecombination%2Bduring%2Byeast%2Bmeiosis%26rft_id%253Dinfo%253Apmid%252F10817760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE0LzEwLzEyNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.25" data-doi="10.1016/0092-8674(89)90681-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gottlieb</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esposito</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A new role for a yeast transcriptional silencer gene, SIR2, in regulation of recombination in ribosomal DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGottlieb%26rft.auinit1%253DS.%26rft.volume%253D56%26rft.issue%253D5%26rft.spage%253D771%26rft.epage%253D776%26rft.atitle%253DA%2Bnew%2Brole%2Bfor%2Ba%2Byeast%2Btranscriptional%2Bsilencer%2Bgene%252C%2BSIR2%252C%2Bin%2Bregulation%2Bof%2Brecombination%2Bin%2Bribosomal%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990681-8%26rft_id%253Dinfo%253Apmid%252F2647300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90681-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2647300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T686200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.26" data-doi="10.1101/gr.113381.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grishkevich</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashimshony</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanai</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Core promoter T-blocks correlate with gene expression levels in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113381.110%26rft_id%253Dinfo%253Apmid%252F21367940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNzA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.27" data-doi="10.1093/gbe/evv182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heil</span>  <span class="cit-name-given-names">C. S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombining without hotspots: a comprehensive evolutionary portrait of recombination in two closely related species of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2829</span>–<span class="cit-lpage">2842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv182%26rft_id%253Dinfo%253Apmid%252F26430062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjcvMTAvMjgyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.28" data-doi="10.1073/pnas.1319032110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellsten</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fine-scale variation in meiotic recombination in <em>Mimulus</em> inferred from population shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">19478</span>–<span class="cit-lpage">19482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1319032110%26rft_id%253Dinfo%253Apmid%252F24225854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ4LzE5NDc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.29" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.30" data-doi="10.1186/gb-2010-11-12-r120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikegami</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egelhofer</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strome</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> chromosome arms are anchored to the nuclear membrane via discontinuous association with LEM-2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DIkegami%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253DR120%26rft.epage%253DR120%26rft.atitle%253DCaenorhabditis%2Belegans%2Bchromosome%2Barms%2Bare%2Banchored%2Bto%2Bthe%2Bnuclear%2Bmembrane%2Bvia%2Bdiscontinuous%2Bassociation%2Bwith%2BLEM-2.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-12-r120%26rft_id%253Dinfo%253Apmid%252F21176223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-12-r120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21176223&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.31" data-doi="10.1534/genetics.113.158857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Crossover heterogeneity in the absence of hotspots in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158857%26rft_id%253Dinfo%253Apmid%252F24172135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dernburg</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">X-chromosome silencing in the germline of <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.volume%253D129%26rft.spage%253D479%26rft_id%253Dinfo%253Apmid%252F11807039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11807039&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173759100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.33" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Human Genome Browser at UCSC.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.34" data-doi="10.1038/nature09525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigurdsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine-scale recombination rate differences between sexes, populations and individuals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1099%26rft.epage%253D1103%26rft.atitle%253DFine-scale%2Brecombination%2Brate%2Bdifferences%2Bbetween%2Bsexes%252C%2Bpopulations%2Band%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09525%26rft_id%253Dinfo%253Apmid%252F20981099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981099&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.35" data-doi="10.1016/S0168-9525(02)02669-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Human SNP variability and mutation rate are higher in regions of high recombination.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLercher%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D337%26rft.epage%253D340%26rft.atitle%253DHuman%2BSNP%2Bvariability%2Band%2Bmutation%2Brate%2Bare%2Bhigher%2Bin%2Bregions%2Bof%2Bhigh%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902669-0%26rft_id%253Dinfo%253Apmid%252F12127766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02669-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12127766&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176560800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.36" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.37" data-doi="10.1101/gr.111120.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111120.110%26rft_id%253Dinfo%253Apmid%252F20980556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.38" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.39" data-doi="10.1016/j.neuron.2009.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macosko</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative mapping of a digenic behavioral trait implicates globin variation in <em>C. elegans</em> sensory behaviors.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D61%26rft.spage%253D692%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.02.012%26rft_id%253Dinfo%253Apmid%252F19285466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285466&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264366900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meneely</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farago</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauffman</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Crossover distribution and high interference for both the X chromosome and an autosome during oogenesis and spermatogenesis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeneely%26rft.auinit1%253DP.%2BM.%26rft.volume%253D162%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1177%26rft.atitle%253DCrossover%2BDistribution%2Band%2BHigh%2BInterference%2Bfor%2BBoth%2Bthe%2BX%2BChromosome%2Band%2Ban%2BAutosome%2BDuring%2BOogenesis%2Band%2BSpermatogenesis%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F12454064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi8zLzExNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.41" data-doi="10.1534/genetics.111.137463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meneely</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGovern</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinis</span>  <span class="cit-name-given-names">F. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanowitz</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Crossover distribution and frequency are regulated by <em>him-5</em> in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137463%26rft_id%253Dinfo%253Apmid%252F22267496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzEyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.42" data-doi="10.1093/nar/gks1048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zweig</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karolchik</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The UCSC Genome Browser database: extensions and updates 2013.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D64</span>–<span class="cit-lpage">D69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1048%26rft_id%253Dinfo%253Apmid%252F23155063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQxL0QxL0Q2NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.43" data-doi="10.1073/pnas.0700412104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Loss of a histone deacetylase dramatically alters the genomic distribution of Spo11p-catalyzed DNA breaks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3955</span>–<span class="cit-lpage">3960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0700412104%26rft_id%253Dinfo%253Apmid%252F17360459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEwLzM5NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Much</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klampert</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garriga</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wightman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The <em>fax-1</em> nuclear hormone receptor regulates axon pathfinding and neurotransmitter expression.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DMuch%26rft.auinit1%253DJ.%26rft.volume%253D127%26rft.issue%253D4%26rft.spage%253D703%26rft.epage%253D712%26rft.atitle%253DThe%2Bfax-1%2Bnuclear%2Bhormone%2Breceptor%2Bregulates%2Baxon%2Bpathfinding%2Band%2Bneurotransmitter%2Bexpression%26rft_id%253Dinfo%253Apmid%252F10648229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMjcvNC83MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.45" data-doi="10.1177/1471082X0700700403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maechler</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A fast and efficient implementation of qualitatively constrained quantile smoothing splines.</span> <abbr class="cit-jnl-abbrev">Stat. Model.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistical%2BModelling%26rft.stitle%253DStatistical%2BModelling%26rft.aulast%253DNg%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D328%26rft.atitle%253DA%2Bfast%2Band%2Befficient%2Bimplementation%2Bof%2Bqualitatively%2Bconstrained%2Bquantile%2B%2B%2B%2B%2B%2B%2B%2B%2Bsmoothing%2Bsplines%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1471082X0700700403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3BzbWoiO3M6NToicmVzaWQiO3M6NzoiNy80LzMxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNi8xNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.46" data-doi="10.1016/j.cub.2015.09.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNelis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Natural variation in <em>plep-1</em> causes male-male copulatory behavior in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2730</span>–<span class="cit-lpage">2737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D25%26rft.spage%253D2730%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.09.019%26rft_id%253Dinfo%253Apmid%252F26455306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.09.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26455306&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpen</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">H3K9 methylation and RNA interference regulate nucleolar organization and repeated DNA stability.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCell%2BBiol.%26rft.volume%253D9%26rft.spage%253D25%26rft_id%253Dinfo%253Apmid%252F17159999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17159999&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-6.6.1767.48"><div class="cit-metadata unstructured">R Core Team 2015 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.49" data-doi="10.1371/journal.pgen.1000419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombinational landscape and population genomics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000419%26rft.epage%253De1000419%26rft.atitle%253DRecombinational%2Blandscape%2Band%2Bpopulation%2Bgenomics%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000419%26rft_id%253Dinfo%253Apmid%252F19283065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283065&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.50" data-doi="10.1126/science.1194208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skrovanek</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Selection at linked sites shapes heritable phenotypic variation in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D372%26rft.epage%253D376%26rft.atitle%253DSelection%2Bat%2BLinked%2BSites%2BShapes%2BHeritable%2BPhenotypic%2BVariation%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194208%26rft_id%253Dinfo%253Apmid%252F20947766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.51" data-doi="10.1111/mec.12322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recombination in the threespine stickleback genome–patterns and consequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3014</span>–<span class="cit-lpage">3027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3014%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.52" data-doi="10.1111/nph.13505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva-Junior</span>  <span class="cit-name-given-names">O. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide patterns of recombination, linkage disequilibrium and nucleotide diversity from pooled resequencing and single nucleotide polymorphism genotyping unlock the evolutionary history of <em>Eucalyptus grandis</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">830</span>–<span class="cit-lpage">845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D830%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13505%26rft_id%253Dinfo%253Apmid%252F26079595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26079595&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.53" data-doi="10.1126/science.aad0843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singhal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sannareddy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venn</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Stable recombination hotspots in birds.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSinghal%26rft.auinit1%253DS.%26rft.volume%253D350%26rft.issue%253D6263%26rft.spage%253D928%26rft.epage%253D932%26rft.atitle%253DStable%2Brecombination%2Bhotspots%2Bin%2Bbirds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad0843%26rft_id%253Dinfo%253Apmid%252F26586757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTAvNjI2My85MjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.54" data-doi="10.1038/hdy.2011.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smukowski</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination rate variation in closely related species.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSmukowski%26rft.auinit1%253DC.%2BS.%26rft.volume%253D107%26rft.issue%253D6%26rft.spage%253D496%26rft.epage%253D508%26rft.atitle%253DRecombination%2Brate%2Bvariation%2Bin%2Bclosely%2Brelated%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.44%26rft_id%253Dinfo%253Apmid%252F21673743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21673743&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297214900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.55" data-doi="10.1093/molbev/msv331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevison</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woerner</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veeramah</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The time-scale of recombination rate evolution in great apes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">928</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv331%26rft_id%253Dinfo%253Apmid%252F26671457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMzLzQvOTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-web" id="cit-6.6.1767.56" data-doi="10.1895/wormbook.1.101.1"><div class="cit-metadata"><cite>Stiernagle, T., 2006 Maintenance of <em>C. elegans</em> (February 11, 2006). <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.101.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.<span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1895/wormbook.1.101.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1895%252Fwormbook.1.101.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1895/wormbook.1.101.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.57" data-doi="10.1534/genetics.115.175950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijveen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterken</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R. J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Remarkably divergent regions punctuate the genome assembly of the <em>Caenorhabditis elegans</em> Hawaiian strain CB4856.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175950%26rft_id%253Dinfo%253Apmid%252F25995208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvOTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.58" data-doi="10.1073/pnas.0908774107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koufopanou</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Conservation of recombination hotspots in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">7847</span>–<span class="cit-lpage">7852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0908774107%26rft_id%253Dinfo%253Apmid%252F20385822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzE3Lzc4NDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzYvMTc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.59" data-doi="10.1038/nature09429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuervers</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanowitz</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">xnd-1 regulates the global recombination landscape in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWagner%26rft.auinit1%253DC.%2BR.%26rft.volume%253D467%26rft.issue%253D7317%26rft.spage%253D839%26rft.epage%253D843%26rft.atitle%253Dxnd-1%2Bregulates%2Bthe%2Bglobal%2Brecombination%2Blandscape%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09429%26rft_id%253Dinfo%253Apmid%252F20944745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20944745&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F6%2F1767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282898700066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.6.1767.60" data-doi="10.1093/nar/gkr954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yook</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabunoc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">WormBase 2012: more genomes, more data, new website.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D735</span>–<span class="cit-lpage">D741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr954%26rft_id%253Dinfo%253Apmid%252F22067452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENzM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi82LzE3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/6/1767.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/6/1757" title="Genome Sequence and Analysis of a Stress-Tolerant, Wild-Derived Strain of Saccharomyces cerevisiae Used in Biofuels Research" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/6/1777" title="A Comprehensive Toolbox for Genome Editing in Cultured Drosophila melanogaster Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/6"><span>Volume 6 Issue 6, June 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/6.cover-source.jpg?itok=WAtQozJg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/6/1767.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-davYYtE6x6ApMAXmygV0bQ8y7v37Uyik8h2rJN-3AE0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/6/1767" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/6/1767" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/6/1767" class="active" data-icon-position="" data-hide-link-title="0">Fine-Scale Crossover Rate Variation on the Caenorhabditis elegans X Chromosome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460388" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ch9YQXkgMyhjOPolnlL0Jalr3qMYU7A6eDj_k-v12qU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1767.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/6/1767" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-E8aJS9pbRaUZOv4qNHn6mMWcEvmOWdVLb-aB1Advw5w" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/6/1767" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/6/1767.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460388" id="node460388--2" data-pisa="ggg;6/6/1767" data-pisa-master="ggg;g3.116.028001" data-apath="/ggg/6/6/1767.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Crossover Rate Variation on the <em>Caenorhabditis elegans</em> X Chromosome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6468-3953" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Max R. Bernstein</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6492-8906" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew V. Rockman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1767-1776;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028001 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460388--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Max R. Bernstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Max%2BR.%2BBernstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernstein%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMax%2BR.%2BBernstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6468-3953" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Max R. Bernstein</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew V. Rockman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BV.%2BRockman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rockman%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BV.%2BRockman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6492-8906" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew V. Rockman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mrockman@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">mrockman@nyu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/6/1767" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/6/1767&amp;title=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/6/1767&amp;title=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/6/1767&amp;title=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/6/1767&amp;text=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/6/1767&amp;title=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/6/1767&amp;t=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/6/1767&amp;title=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/6/1767&amp;title=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460388/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460388" id="node460388--3" data-pisa="ggg;6/6/1767" data-pisa-master="ggg;g3.116.028001" data-apath="/ggg/6/6/1767.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Crossover Rate Variation on the <em>Caenorhabditis elegans</em> X Chromosome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6468-3953" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Max R. Bernstein</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6492-8906" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew V. Rockman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1767-1776;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028001 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460388--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Max R. Bernstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Max%2BR.%2BBernstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernstein%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMax%2BR.%2BBernstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6468-3953" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Max R. Bernstein</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew V. Rockman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology and Center for Genomics and Systems Biology, New York University, NY 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BV.%2BRockman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rockman%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BV.%2BRockman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6492-8906" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew V. Rockman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mrockman@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">mrockman@nyu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1767&amp;rft.epage=1776&amp;rft.atitle=Fine-Scale%20Crossover%20Rate%20Variation%20on%20the%20Caenorhabditis%20elegans%20X%20Chromosome&amp;rft.volume=6&amp;rft.issue=6&amp;rft.date=2016-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bernstein%2C+Max+R.&amp;rft.au=Rockman%2C+Matthew+V."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460388/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460388/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460388/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460388/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460388/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460388/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460388/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460388/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460388/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460388/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460388/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460388/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/6/1767" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/6/1767.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/6/1767.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/6/1767.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/6/1767.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
