<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/1/119" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/1/119.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/1/119.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/1/119.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/1/119.atom" />
<meta name="HW.pisa" content="ggg;3/1/119" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Systems-Level Analysis of Genome-Wide Association Data" />
<meta name="DC.Identifier" content="10.1534/g3.112.004788" />
<meta name="DC.Date" content="2013-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Farber. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association studies (GWAS) have emerged as the method of choice for identifying common variants affecting complex disease. In a GWAS, particular attention is placed, for obvious reasons, on single-nucleotide polymorphisms (SNPs) that exceed stringent genome-wide significance thresholds. However, it is expected that many SNPs with only nominal evidence of association ( e.g. , P &lt; 0.05) truly influence disease. Efforts to extract additional biological information from entire GWAS datasets have primarily focused on pathway-enrichment analyses. However, these methods suffer from a number of limitations and typically fail to lead to testable hypotheses. To evaluate alternative approaches, we performed a systems-level analysis of GWAS data using weighted gene coexpression network analysis. A weighted gene coexpression network was generated for 1918 genes harboring SNPs that displayed nominal evidence of association ( P ≤ 0.05) from a GWAS of bone mineral density (BMD) using microarray data on circulating monocytes isolated from individuals with extremely low or high BMD. Thirteen distinct gene modules were identified, each comprising coexpressed and highly interconnected GWAS genes. Through the characterization of module content and topology, we illustrate how network analysis can be used to discover disease-associated subnetworks and characterize novel interactions for genes with a known role in the regulation of BMD. In addition, we provide evidence that network metrics can be used as a prioritizing tool when selecting genes and SNPs for replication studies. Our results highlight the advantages of using systems-level strategies to add value to and inform GWAS." />
<meta name="DC.Contributor" content="Charles R. Farber" />
<meta name="article:published_time" content="2013-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Systems-Level Analysis of Genome-Wide Association Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies (GWAS) have emerged as the method of choice for identifying common variants affecting complex disease. In a GWAS, particular attention is placed, for obvious reasons, on single-nucleotide polymorphisms (SNPs) that exceed stringent genome-wide significance thresholds. However, it is expected that many SNPs with only nominal evidence of association (&lt;i&gt;e.g.&lt;/i&gt;, &lt;i&gt;P&lt;/i&gt; &amp;lt; 0.05) truly influence disease. Efforts to extract additional biological information from entire GWAS datasets have primarily focused on pathway-enrichment analyses. However, these methods suffer from a number of limitations and typically fail to lead to testable hypotheses. To evaluate alternative approaches, we performed a systems-level analysis of GWAS data using weighted gene coexpression network analysis. A weighted gene coexpression network was generated for 1918 genes harboring SNPs that displayed nominal evidence of association (&lt;i&gt;P&lt;/i&gt; ≤ 0.05) from a GWAS of bone mineral density (BMD) using microarray data on circulating monocytes isolated from individuals with extremely low or high BMD. Thirteen distinct gene modules were identified, each comprising coexpressed and highly interconnected GWAS genes. Through the characterization of module content and topology, we illustrate how network analysis can be used to discover disease-associated subnetworks and characterize novel interactions for genes with a known role in the regulation of BMD. In addition, we provide evidence that network metrics can be used as a prioritizing tool when selecting genes and SNPs for replication studies. Our results highlight the advantages of using systems-level strategies to add value to and inform GWAS.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/01/01" />
<meta name="citation_mjid" content="ggg;3/1/119" />
<meta name="citation_id" content="3/1/119" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/1/119" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/1/119.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/1/119.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/1/119.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004788" />
<meta name="citation_pmid" content="23316444" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="119" />
<meta name="citation_lastpage" content="129" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Charles R. Farber" />
<meta name="citation_author_institution" content="Center for Public Health Genomics, Departments of Medicine (Division of Cardiology) and Biochemistry and Molecular Genetics, University of Virginia, Charlottesville, Virginia 22908" />
<meta name="citation_author_email" content="crf2s@virginia.edu" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=O. Alter;citation_author=P. O. Brown;citation_author=D. Botstein;citation_title=Singular value decomposition for genome-wide expression data processing and modeling;citation_pages=10101-10106;citation_volume=97;citation_year=2000;citation_issue=18;citation_pmid=10963673;citation_doi=10.1073/pnas.97.18.10101" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Altshuler;citation_author=M. J. Daly;citation_author=E. S. Lander;citation_title=Genetic Mapping in Human Disease;citation_pages=881-888;citation_volume=322;citation_year=2008;citation_issue=5903;citation_pmid=18988837;citation_doi=10.1126/science.1156409" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=K. Askland;citation_author=C. Read;citation_author=J. Moore;citation_title=Pathways-based analyses of whole-genome association study data in bipolar disorder reveal genes mediating ion channel activity and synaptic neurotransmission.;citation_pages=63-79;citation_volume=125;citation_year=2009;citation_issue=1;citation_pmid=19052778;citation_doi=10.1007/s00439-008-0600-y" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=S. E. Baranzini;citation_author=N. W. Galwey;citation_author=J. Wang;citation_author=P. Khankhanian;citation_author=R. Lindberg;citation_title=Pathway and network-based analysis of genome-wide association studies in multiple sclerosis;citation_volume=18;citation_year=2009;citation_pmid=19286671;citation_doi=10.1093/hmg/ddp120" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=H. C. Blair;citation_title=How the osteoclast degrades bone.;citation_pages=837-846;citation_volume=20;citation_year=1998;citation_issue=10;citation_pmid=9819571;citation_doi=10.1002/(SICI)1521-1878(199810)20:10&lt;837::AID-BIES9&gt;3.0.CO;2-D" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=I. Bratic;citation_author=A. Trifunovic;citation_title=Mitochondrial energy metabolism and ageing.;citation_pages=961-967;citation_volume=1797;citation_year=2010;citation_issue=6-7;citation_pmid=20064485;citation_doi=10.1016/j.bbabio.2010.01.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Chen;citation_author=J. Zhu;citation_author=P. Y. Lum;citation_author=X. Yang;citation_author=S. Pinto;citation_title=Variations in DNA elucidate molecular networks that cause disease.;citation_pages=429-435;citation_volume=452;citation_year=2008;citation_issue=7186;citation_pmid=18344982;citation_doi=10.1038/nature06757" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=DAVID: Database for Annotation, Visualization, and Integrated Discovery;citation_author=G. Dennis;citation_author=B. T. Sherman;citation_author=D. A. Hosack;citation_author=J. Yang;citation_author=W. Gao;citation_volume=4;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=J. Dong;citation_author=S. Horvath;citation_title=Understanding network concepts in modules.;citation_pages=24-24;citation_volume=1;citation_year=2007;citation_pmid=17547772;citation_doi=10.1186/1752-0509-1-24" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=C. C. Elbers;citation_author=K. R. van Eijk;citation_author=L. Franke;citation_author=F. Mulder;citation_author=Y. T. van der Schouw;citation_title=Using genome-wide pathway analysis to unravel the etiology of complex diseases.;citation_pages=419-431;citation_volume=33;citation_year=2009;citation_issue=5;citation_pmid=19235186;citation_doi=10.1002/gepi.20395" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=C. C. Elbers;citation_author=K. R. van Eijk;citation_author=L. Franke;citation_author=F. Mulder;citation_author=Y. T. van der Schouw;citation_title=Using genome-wide pathway analysis to unravel the etiology of complex diseases.;citation_pages=419-431;citation_volume=33;citation_year=2009;citation_issue=5;citation_pmid=19235186;citation_doi=10.1002/gepi.20395" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Estrada;citation_author=U. Styrkarsdottir;citation_author=E. Evangelou;citation_author=Y. H. Hsu;citation_author=E. L. Duncan;citation_title=Genome-wide meta-analysis identifies 56 bone mineral density loci and reveals 14 loci associated with risk of fracture.;citation_pages=491-501;citation_volume=44;citation_year=2012;citation_issue=5;citation_pmid=22504420;citation_doi=10.1038/ng.2249" />
<meta name="citation_reference" content="citation_journal_title=Journal of bone and mineral research :  the official journal of the American Society for Bone and Mineral Research;citation_journal_abbrev=J Bone Miner Res;citation_author=C. R. Farber;citation_title=Identification of a gene module associated with BMD through the integration of network analysis and genome-wide association data.;citation_pages=2359-2367;citation_volume=25;citation_year=2010;citation_issue=11;citation_pmid=20499364;citation_doi=10.1002/jbmr.138" />
<meta name="citation_reference" content="citation_journal_title=Journal of endocrinological investigation;citation_journal_abbrev=J Endocrinol Invest;citation_author=R. Fontova;citation_author=C. Gutierrez;citation_author=J. Vendrell;citation_author=M. Broch;citation_author=I. Vendrell;citation_title=Bone mineral mass is associated with interleukin 1 receptor autoantigen and TNF-alpha gene polymorphisms in post-menopausal Mediterranean women.;citation_pages=684-690;citation_volume=25;citation_year=2002;citation_issue=8;citation_pmid=12240899" />
<meta name="citation_reference" content="citation_journal_title=Endocrinology;citation_journal_abbrev=Endocrinology;citation_author=Y. Fujikawa;citation_author=J. M. Quinn;citation_author=A. Sabokbar;citation_author=J. O. McGee;citation_author=N. A. Athanasou;citation_title=The human osteoclast precursor circulates in the monocyte fraction;citation_pages=4058-4060;citation_volume=137;citation_year=1996;citation_issue=9;citation_pmid=8756585;citation_doi=10.1210/en.137.9.4058" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. S. Gargalovic;citation_author=M. Imura;citation_author=B. Zhang;citation_author=N. M. Gharavi;citation_author=M. J. Clark;citation_title=Identification of inflammatory gene modules based on variations of human endothelial cell responses to oxidized lipids;citation_volume=103;citation_year=2006;citation_pmid=16912112;citation_doi=10.1073/pnas.0605457103" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=L. Gautier;citation_author=L. Cope;citation_author=B. M. Bolstad;citation_author=R. A. Irizarry;citation_title=affy--analysis of Affymetrix GeneChip data at the probe level;citation_pages=307-315;citation_volume=20;citation_year=2004;citation_issue=3;citation_pmid=14960456;citation_doi=10.1093/bioinformatics/btg405" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Ghazalpour;citation_author=S. Doss;citation_author=B. Zhang;citation_author=S. Wang;citation_author=C. Plaisier;citation_title=Integrating genetic and network analysis to characterize genes related to mouse weight.;citation_pages=e130-e130;citation_volume=2;citation_year=2006;citation_issue=8;citation_pmid=16934000;citation_doi=10.1371/journal.pgen.0020130" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. W. Gong;citation_author=W. Zhao;citation_author=N. Li;citation_author=B. Barajas;citation_author=M. Kleinman;citation_title=Air-pollutant chemicals and oxidized lipids exhibit genome-wide synergistic effects on endothelial cells.;citation_pages=R149-R149;citation_volume=8;citation_year=2007;citation_issue=7;citation_pmid=17655762;citation_doi=10.1186/gb-2007-8-7-r149" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=M. G. Hong;citation_author=Y. Pawitan;citation_author=P. K. Magnusson;citation_author=J. A. Prince;citation_title=Strategies and issues in the detection of pathway enrichment in genome-wide association studies.;citation_pages=289-301;citation_volume=126;citation_year=2009;citation_issue=2;citation_pmid=19408013;citation_doi=10.1007/s00439-009-0676-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Horvath;citation_author=B. Zhang;citation_author=M. Carlson;citation_author=K. V. Lu;citation_author=S. Zhu;citation_title=Analysis of oncogenic signaling networks in glioblastoma identifies ASPM as a molecular target;citation_volume=103;citation_year=2006;citation_pmid=17090670;citation_doi=10.1073/pnas.0608396103" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=W. Huang da;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Graph. Statist.;citation_author=R. Ihaka;citation_author=R. Gentleman;citation_title=R: a language for data analysis and graphics;citation_pages=299-314;citation_volume=5;citation_year=1996;citation_doi=10.2307/1390807" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_journal_abbrev=Biostatistics;citation_author=R. A. Irizarry;citation_author=B. Hobbs;citation_author=F. Collin;citation_author=Y. D. Beazer-Barclay;citation_author=K. J. Antonellis;citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data;citation_pages=249-264;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12925520;citation_doi=10.1093/biostatistics/4.2.249" />
<meta name="citation_reference" content="citation_journal_title=BMC Med. Genet.;citation_author=D. P. Kiel;citation_author=S. Demissie;citation_author=J. Dupuis;citation_author=K. L. Lunetta;citation_author=J. M. Murabito;citation_title=Genome-wide association with bone mass and geometry in the Framingham Heart Study;citation_pages=S14;citation_volume=8;citation_year=2007;citation_issue=Suppl 1;citation_pmid=17903296;citation_doi=10.1186/1471-2350-8-S1-S14" />
<meta name="citation_reference" content="citation_journal_title=Cellular physiology and biochemistry :  international journal of experimental cellular physiology, biochemistry, and pharmacology;citation_journal_abbrev=Cell Physiol Biochem;citation_author=J. M. Kim;citation_author=D. Jeong;citation_author=H. K. Kang;citation_author=S. Y. Jung;citation_author=S. S. Kang;citation_title=Osteoclast precursors display dynamic metabolic shifts toward accelerated glucose metabolism at an early stage of RANKL-stimulated osteoclast differentiation.;citation_pages=935-946;citation_volume=20;citation_year=2007;citation_issue=6;citation_pmid=17982276;citation_doi=10.1159/000110454" />
<meta name="citation_reference" content="citation_journal_title=Menopause (New York, N.Y.);citation_journal_abbrev=Menopause;citation_author=H. Kim;citation_author=S. Chun;citation_author=S. Y. Ku;citation_author=C. S. Suh;citation_author=Y. M. Choi;citation_title=Association between polymorphisms in tumor necrosis factor (TNF) and TNF receptor genes and circulating TNF, soluble TNF receptor levels, and bone mineral density in postmenopausal Korean women.;citation_pages=1014-1020;citation_volume=16;citation_year=2009;citation_issue=5;citation_pmid=19369902;citation_doi=10.1097/gme.0b013e3181a039c8" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Osteoporos Rep;citation_author=S. Kousteni;citation_title=FoxOs: Unifying links between oxidative stress and skeletal homeostasis.;citation_pages=60-66;citation_volume=9;citation_year=2011;citation_issue=2;citation_pmid=21404001;citation_doi=10.1007/s11914-011-0054-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Investigation;citation_journal_abbrev=JCI;citation_author=J. Lam;citation_author=S. Takeshita;citation_author=J. E. Barker;citation_author=O. Kanagawa;citation_author=F. P. Ross;citation_title=TNF-{{alpha}} induces osteoclastogenesis by direct stimulation of macrophages exposed to permissive levels of RANK ligand;citation_pages=1481-1488;citation_volume=106;citation_year=2000;citation_issue=12;citation_pmid=11120755;citation_doi=10.1172/JCI11176" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=WGCNA: an R package for weighted correlation network analysis.;citation_pages=559-559;citation_volume=9;citation_year=2008;citation_pmid=19114008;citation_doi=10.1186/1471-2105-9-559" />
<meta name="citation_reference" content="citation_journal_abbrev=Immunome Res;citation_author=J. E. Lattin;citation_author=K. Schroder;citation_author=A. I. Su;citation_author=J. R. Walker;citation_author=J. Zhang;citation_title=Expression analysis of G Protein-Coupled Receptors in mouse macrophages.;citation_pages=5-5;citation_volume=4;citation_year=2008;citation_pmid=18442421;citation_doi=10.1186/1745-7580-4-5" />
<meta name="citation_reference" content="citation_journal_abbrev=Bone;citation_author=S. F. Lei;citation_author=S. Wu;citation_author=L. M. Li;citation_author=F. Y. Deng;citation_author=S. M. Xiao;citation_title=An in vivo genome wide gene expression study of circulating monocytes suggested GBP1, STAT1 and CXCL10 as novel risk genes for the differentiation of peak bone mass.;citation_pages=1010-1014;citation_volume=44;citation_year=2009;citation_issue=5;citation_pmid=19223260;citation_doi=10.1016/j.bone.2008.05.016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. O’Dushlaine;citation_author=E. Kenny;citation_author=E. A. Heron;citation_author=R. Segurado;citation_author=M. Gill;citation_title=The SNP ratio test: pathway analysis of genome-wide association datasets;citation_volume=25;citation_year=2009;citation_pmid=19620097;citation_doi=10.1093/bioinformatics/btp448" />
<meta name="citation_reference" content="citation_journal_title=Nature neuroscience;citation_journal_abbrev=Nat Neurosci;citation_author=M. C. Oldham;citation_author=G. Konopka;citation_author=K. Iwamoto;citation_author=P. Langfelder;citation_author=T. Kato;citation_title=Functional organization of the transcriptome in human brain.;citation_pages=1271-1282;citation_volume=11;citation_year=2008;citation_issue=11;citation_pmid=18849986;citation_doi=10.1038/nn.2207" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=G. Peng;citation_author=L. Luo;citation_author=H. Siu;citation_author=Y. Zhu;citation_author=P. Hu;citation_title=Gene and pathway-based second-wave analysis of genome-wide association studies.;citation_pages=111-117;citation_volume=18;citation_year=2010;citation_issue=1;citation_pmid=19584899;citation_doi=10.1038/ejhg.2009.115" />
<meta name="citation_reference" content="citation_journal_abbrev=Genome Med;citation_author=M. D. Ritchie;citation_title=Using prior knowledge and genome-wide association to identify pathways involved in multiple sclerosis.;citation_pages=65-65;citation_volume=1;citation_year=2009;citation_issue=6;citation_pmid=19566917;citation_doi=10.1186/gm65" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=U. Styrkarsdottir;citation_author=B. V. Halldorsson;citation_author=S. Gretarsdottir;citation_author=D. F. Gudbjartsson;citation_author=G. B. Walters;citation_title=Multiple genetic loci for bone mineral density and fractures.;citation_pages=2355-2365;citation_volume=358;citation_year=2008;citation_issue=22;citation_pmid=18445777;citation_doi=10.1056/NEJMoa0801197" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. I. Su;citation_author=M. P. Cooke;citation_author=K. A. Ching;citation_author=Y. Hakak;citation_author=J. R. Walker;citation_title=Large-scale analysis of the human and mouse transcriptomes;citation_volume=99;citation_year=2002;citation_pmid=11904358;citation_doi=10.1073/pnas.012025199" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. I. Su;citation_author=T. Wiltshire;citation_author=S. Batalov;citation_author=H. Lapp;citation_author=K. A. Ching;citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes;citation_volume=101;citation_year=2004;citation_pmid=15075390;citation_doi=10.1073/pnas.0400782101" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=A. Torkamani;citation_author=N. J. Schork;citation_title=Pathway and network analysis with high-density allelic association data.;citation_pages=289-301;citation_volume=563;citation_year=2009;citation_pmid=19597792;citation_doi=10.1007/978-1-60761-175-2_16" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=A. Torkamani;citation_author=E. J. Topol;citation_author=N. J. Schork;citation_title=Pathway analysis of seven common diseases assessed by genome-wide association.;citation_pages=265-272;citation_volume=92;citation_year=2008;citation_issue=5;citation_pmid=18722519;citation_doi=10.1016/j.ygeno.2008.07.011" />
<meta name="citation_reference" content="citation_journal_title=Endocrinology;citation_author=A. van Nas;citation_author=D. Guhathakurta;citation_author=S. S. Wang;citation_author=N. Yehya;citation_author=S. Horvath;citation_title=Elucidating the role of gonadal hormones in sexually dimorphic gene coexpression networks;citation_volume=150;citation_year=2009;citation_pmid=18974276;citation_doi=10.1210/en.2008-0563" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Wang;citation_author=M. Li;citation_author=M. Bucan;citation_title=Pathway-based approaches for analysis of genomewide association studies.;citation_pages=1278-1283;citation_volume=81;citation_year=2007;citation_issue=6;citation_pmid=17966091;citation_doi=10.1086/522374" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=J. P. Williams;citation_author=H. C. Blair;citation_author=J. M. McDonald;citation_author=M. A. McKenna;citation_author=S. E. Jordan;citation_title=Regulation of osteoclastic bone resorption by glucose.;citation_pages=646-651;citation_volume=235;citation_year=1997;citation_issue=3;citation_pmid=9207213;citation_doi=10.1006/bbrc.1997.6795" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=K. D. Winden;citation_author=M. C. Oldham;citation_author=K. Mirnics;citation_author=P. J. Ebert;citation_author=C. H. Swan;citation_title=The organization of the transcriptional network in specific neuronal classes;citation_pages=291-291;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19638972;citation_doi=10.1038/msb.2009.46" />
<meta name="citation_reference" content="citation_title=Stat. Appl. Genet. Mol. Biol.;citation_title=A general framework for weighted gene co-expression network analysis;citation_author=B. Zhang;citation_author=S. Horvath;citation_volume=4;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Zhang;citation_author=S. Kirov;citation_author=J. Snoddy;citation_title=WEBGESTALT: AN INTEGRATED SYSTEM FOR EXPLORING GENE SETS IN VARIOUS BIOLOGICAL CONTEXTS;citation_pages=W741 748;citation_volume=33;citation_year=2005" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Systems-Level Analysis of Genome-Wide Association Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/1.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies (GWAS) have emerged as the method of choice for identifying common variants affecting complex disease. In a GWAS, particular attention is placed, for obvious reasons, on single-nucleotide polymorphisms (SNPs) that exceed stringent genome-wide significance thresholds. However, it is expected that many SNPs with only nominal evidence of association ( e.g. , P &lt; 0.05) truly influence disease. Efforts to extract additional biological information from entire GWAS datasets have primarily focused on pathway-enrichment analyses. However, these methods suffer from a number of limitations and typically fail to lead to testable hypotheses. To evaluate alternative approaches, we performed a systems-level analysis of GWAS data using weighted gene coexpression network analysis. A weighted gene coexpression network was generated for 1918 genes harboring SNPs that displayed nominal evidence of association ( P ≤ 0.05) from a GWAS of bone mineral density (BMD) using microarray data on circulating monocytes isolated from individuals with extremely low or high BMD. Thirteen distinct gene modules were identified, each comprising coexpressed and highly interconnected GWAS genes. Through the characterization of module content and topology, we illustrate how network analysis can be used to discover disease-associated subnetworks and characterize novel interactions for genes with a known role in the regulation of BMD. In addition, we provide evidence that network metrics can be used as a prioritizing tool when selecting genes and SNPs for replication studies. Our results highlight the advantages of using systems-level strategies to add value to and inform GWAS." />
<meta name="og-title" property="og:title" content="Systems-Level Analysis of Genome-Wide Association Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/1/119" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies (GWAS) have emerged as the method of choice for identifying common variants affecting complex disease. In a GWAS, particular attention is placed, for obvious reasons, on single-nucleotide polymorphisms (SNPs) that exceed stringent genome-wide significance thresholds. However, it is expected that many SNPs with only nominal evidence of association ( e.g. , P &lt; 0.05) truly influence disease. Efforts to extract additional biological information from entire GWAS datasets have primarily focused on pathway-enrichment analyses. However, these methods suffer from a number of limitations and typically fail to lead to testable hypotheses. To evaluate alternative approaches, we performed a systems-level analysis of GWAS data using weighted gene coexpression network analysis. A weighted gene coexpression network was generated for 1918 genes harboring SNPs that displayed nominal evidence of association ( P ≤ 0.05) from a GWAS of bone mineral density (BMD) using microarray data on circulating monocytes isolated from individuals with extremely low or high BMD. Thirteen distinct gene modules were identified, each comprising coexpressed and highly interconnected GWAS genes. Through the characterization of module content and topology, we illustrate how network analysis can be used to discover disease-associated subnetworks and characterize novel interactions for genes with a known role in the regulation of BMD. In addition, we provide evidence that network metrics can be used as a prioritizing tool when selecting genes and SNPs for replication studies. Our results highlight the advantages of using systems-level strategies to add value to and inform GWAS." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/1.cover-source.jpg" />
    <title>Systems-Level Analysis of Genome-Wide Association Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fy-YiSKTEz9vgOY8cR7UFPPMk33CyMMmFqLlCLGqCVs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459397","apath":"\/ggg\/3\/1\/119.atom","pisa":"ggg;3\/1\/119","ac":{"\/ggg\/3\/1\/119.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/1\/119.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/1\/119"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/1\/119"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/1\/119":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zv7+C2+1F+ZfGTtrr037tWz3hO8GHeHekd4KDgbNrOchN2rZsJfT0+XxsU4ThIXx7IA9JqVwTAPspSLlXBxMyKLqYq4YXQqX94gKNITkrw8PhMGTQxz3ZLPhedsdWJMLv32hLKEI9DVHjflA2WuVyM2NqS\\\\\\\/vnO+4yIDndaWI2ZvhIubeCerNFHaCV\\\\\\\/WHhuZvUT0lMhYy0SpRNJmjtBXdFEqOTWyfSuuRRLjfhOG\\\\\\\/Wd0GzaTd5FOl7VrpVZ3BjKEJgIj4crp5FJPIDYJbdELyNM3VrPmCPBqgE7MjrKYR5YZ3XCdtmtyTYuBzdxu9wBa0ldmrftq5YKJ0SfroI66wjGOTMPNXsyIgFG9bOSAlptckVciLWiww45CMevb5TF76LNw7mpmpskmuR0aX5LJU1Fdu85faB6Ia4r2QXf36pYkbcKy+I6qIUYJcWijKeYaKvTEs3unywEIAwOAKu3Rh0Ffj7IhCkgIRsqeiyMXhcKxge13qQFHdWYr6vhPO7Q4A1cMUuCfO14vDVPwkjdoBJpfqJGKXIgwEK+2n5bYtKqL93AjjqCdo+XCT0STQXCknQ59hS01owqkfDUT0u6Z5j+ezHwO8NE6lNM3atFeEXmEoKhUMTWg4eHD3t06d0lJ7QqPceJFgw9xeDvRnFevSF00zEaIUjq4ky\\\\\\\/yqmuHSPR8xFXbaB\\\\\\\/+AOVI8E1AmDbfECbuxQPeIMMs1A+XgigaP+GPH74Jgj\\\\\\\/z7vdSxmb1ZyOZP1CtYFVuywGAuYLoU6LtVDQ0Vha3n12aGOvy1ALBGd4Q2BYYyRBDCKyzdE04dgFG9rm7a+3sKbeGDP\\\\\\\/yzmQcj2bZqLKQRnrQoSAP2ZYLIj+x12B6dQDvrbwAeCq+B5wZXFdGF9P2XZQqZa\\\\\\\/f1aR8zLQZUASXKbN3otb86VFDg0\\\\\\\/7dF4cFHo56KffC1PFNNMnpLo6FX+avFVzKtrJORY\\\\\\\/EOoA1hZMi99LYhblvx3z9TwW35jNUxifFKTOqdDvyPV7APen4chYAHgpLaqhZLMFb4NjMeBcEL4qjurIfIpB14t\\\\\\\/4yls83Oui7py1mg3vadoDonyGKyWiUC4L0QkXwolvdvlLl5WHOMOdnjTQKJqEswsN\\\\\\\/UQNWt204wGAXPaY0jPNiEPF58xDfKvTPpVaGpNsH07LuiFEiHiwRpkM3qy22YxeRvonZ\\\\\\\/kJm6ogHR0uLmsCHwOlz\\\\\\\/LFC4vp4MPWtrqUWNJvaoeYVfC9UxP6iKpEGMisd5saooRQnLBGsYCrp50pt2+c+9rLqOBdTdzTiRME\\\u0022,\\\u0022iv\\\u0022:\\\u0022vUemgXDtYEC5L4OlthF5yQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022911f291decd4bfa8e4218b7dc522627de43e7d0c0a520d8c24c56aab9a7a7042\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PxRL+jzngy4AuVrKRQsAHncoMwMSX1ItwGwCiD5W5gKO4s83nqwjJqhJ1a9E74EuoInCm\\\\\\\/QqJL\\\\\\\/3zjlFdm1jWMBnn3N6NtkXuk2rZUG\\\\\\\/oabjo26dUAGDHd\\\\\\\/h7KDsEIqM68De18fvRlBC6m89PHFjyEMEQIwU0b9bsOasu3OHNfGXX0zzyo87YPZdv908z3+QCU0BtDWOR1V9oMzvYltJgYoMZfzWrrc2ibhccPB8l2HLo0Zi6V9Np75jj71SjcYe0LYwlf3fcAXpgDCTUKwjvGswEMBkuJUqBEZm2cmJNw+nShJQ3vb4Ip53wjMNLzdVcRDLhb26BCGvuLX6dC\\\\\\\/WjsS+uO2Y8i\\\\\\\/41vdIwdVpwthO6hKdFDkJJnGDvF\\\\\\\/XH3DhrD4K4Fp9xL2e8BrSoSJ33B3OqSVmjJEw8tNea7CYW2jCNgLdqACuqafIKeHXSPVmTkJgwzF0arXoj+JDv1vJGLkamRO0QYcSBi2x8e7\\\\\\\/K3ZB\\\\\\\/zv1SQjD08WX5NJuWy3PZpG6x+rSZ9oQxineDUzX64JQ7NU4WbO88jk3JORGM7twITfXA8vS9sk8JcyvJRNpHL+ejMFWcFivcwWfqIaFxLRjBj5kvv+QhrjQVt2SJEUONzFfSHqdCbfqJk6lCyjRQl5mB9wps6CSBI6KCAA\\\\\\\/pNhKBbDgnZPW+BlNMR4Oi+gNeYTEHgn55A5XwyRrN\\\\\\\/yiYtVlvLkKYO+eZqurxDcwRjkip1Md3ZksXEv7wHXWs6x8T+PjWYvnpFbgHUgy8ORbEvFVOx6EeN0y6\\\\\\\/xQK3G1\\\\\\\/XNRlV4ka0jD4Q9hRz1cMmvQ5Lk2SWPlGXHmEIseYQfr8mkRFhJpux8AZFteKK5PDtJGUHmI4OLBEhRjja5dV6Te1kbrud3E4ZZOvOIcLRB9\\\\\\\/E2toHZftcWswbuKZUdNCdsuR6Gk7Ke+tQcZaKehDVqEZFTZMbi\\\\\\\/bjrSiEQ2IYJSveJe0W1xH8nb75etWDngH+XRaltu0\\\\\\\/+mwcsFaXRDIEA91V8KLfmYdIh2fEQ96Rmj577SLxadyprK+8KnR7h576TcnkdPAH0OrODcHMjCDvF1hER8E719dJBsUKUWEfVPtvrmzV7Yu7ueYfY6bwLeqHTnTt+0uCjhqekxyBEHX6m8y75T69311F3rub7sJQbKXOcP3GLxAW2FB2J2eV2BZrYIBJY5\\\\\\\/b6JW0kSaEbw2V7QjOgTsuCh3mLLUQUHMT39\\\\\\\/HV4fZdVyXvD4E8T2c2znamf2nlP5FKRsk\\\\\\\/qtWYHn1ndX9Kq+M1Mq1L3MHT2anH32hwxybAQnk1TOOJwoVSPxHwaxMT8t+JkNv54o+sJVdYaJKKH9bc9vnVCnr8tPBRzVLiJ+YpPlpExpFGPP+uZYjlflVgohvUTqtBo5sWmPuJc\\\\\\\/oyFXt8WtdbNLhbBGWtatIDYF5tOTG3AtHq5y1JZeIabJ1IG65khsqkiK3eA0YERNKQp642krOpJroogzGO4th1T1QDOAyenOyYYQ2ylolMo7id3cT6cVldvB7suk20G4NR8oP7ahL9viSMLxYSpAdVBksolxW58UfQ31tqYncL8MzRvFWIUsRnLW2HMWK1Xo\\\\\\\/13kUxkr0SbxL5PLxkIJmtCJMuyyFx5tHLBlWSfj\\\\\\\/hR26dSntnCJrP+HNb9fBtD6\\\\\\\/AS0x84ITd3M54V43J7k\\\\\\\/L4GjmA6kBxrEGLOK\\\\\\\/wsp0uPZ6Hk66JEEf+i36d43rR7HZNosVMYN4b6mZ2MlH+ahqSeKhN4G+bdszl9Or242EBRAukA7s8TldkzGy8YLpiY48ENK9UqVlG\\\\\\\/upMD99N2RI43QMiDA8MTy7em+3KoAeJYkXf07SAlanJKCV3D+qmgtaCRVTEaxEup5cscKb5oiQoEdLG46odXMYMSnGsongjomDIzlyhJ0qwWfIuKEJz0PdaddYgBnwblLAqu1iBFYJR8I8Q6GGocQsBgQEAqAlXdr4Yw5ANSJ+p9kD7l7YuMYCcc8+dz6ORGSV9b0LF3gD6B0pKRZvrE9uOsiOzLJ0EZb0ee5COQGchUB2Zz0eLgETY4SbyY+57k80ZMBm1REePOmmEPgIAKvov0KpT5uboJEx7PR9C7NIJHaFeUSa2ZDyzlX3nMmqAMQrBrNOCIti3iPe9KgS6+Sw4aC7iPIMebISsoAtP9pRIPv+1sb7+mR9HSEb0JsQcwzONzNcVX3cOKn4T6tqui1U2xSG\\\\\\\/PscL5JwsCeW4kA3BCDVZ0pSt6XJ7+mnP0JEa39RTZylhO1k3TtPU8w9CKVzNpdddDvxPpUEqikRI3nDpS0c8YvqxPovbXqApltLJf3uHNimaL\\\\\\\/ZG481odpELxaYQ9Sr\\\\\\\/QBMdyZ64IqIE1Y7\\\\\\\/6ozkEL8Zz8JPt6dcug7KV3KuaLnafmCKwtkL+FfO6yFR5NZBOaaVpQqKrU4dYsl5NVDuWEnEeNRcBq8KLBrJH6SOFWhMchCQsZAa+d60NKvAtfR6CwaXy3PaJbRGNKa9cvzDBOkX3ZyonIJ0yx8Msarp+6g2fE\\\\\\\/HOJSLqJI2VXFS7sAkzgQERSJmSyfrb\\\\\\\/D7qUzJa6v2NW2rUYj5Jy1kv6aayrI22eCgooTydK8Cf1G5VMwSQ6jgre7wEW3gHYlWZ4GuBeW+GlKX8s5iPJfXpz0L9yZs3RdzxcrdRpJlQcFwtSq0c6rsXgoAC2qWui2y7Ubha7Aoba5iz53zrVRPZK\\\\\\\/Y+xOyGaFl5YHUwVpI1MqahDiseU1dzZ4OKNcePVbMV2zYM5M5VNfUUBT\\\\\\\/Co5idRR5vG5Ao839Xop9JJjpE8SXVbmZoPWYTuUzJ\\\\\\\/It2jHlY1GEd\\\\\\\/7GGUP3HIT42ha4OoZHsh6lUShljGEXxCUuQjT\\\\\\\/YWBoRLFKzUFBV1xAHqi993PyP8P8tK3r9ICO0BnbDhL6d5l5LKtUAg74ORZ3jJJUz22a7OuQGzbIG2HdO0tj0KpWXe+srzAJBUPr5eQAaaWdVSgz9jmVzf1o5Rpa7QYrI3cRWlfs0hafz6+hKqY+gQs0CGmH5orgNR173peAAs\\\\\\\/GgvC3or9Hjf6CVbzgPOFkdKFVp0C3goKfBHY2vyAgYUHe+GYwFSrkO+tJ3EB9Ba1vKJ0zx\\\\\\\/SKX3Vgqsn94+kV2FT0wLF54XFLNEdKqW3c5EiETAWoMBicZUaaYiYlB8AE2PKIqv8oud9yDDp4v7EzEPYVc0cdbDG2soITYioMwPlcGE3I64EMufgWA7BONOONs4p84Q1F0tPB8LoZKpsz\\\\\\\/P3KZxjskv5Jzu8SWYOulPrEfQ2j1a1ubNjd528+YIrToR1fidyUVpVsqUXXmRFNHzFdt\\\\\\\/rMWYKL2H3\\\\\\\/EaVOaLlb\\\\\\\/DpAb0rvz1ElvO7B1UrQXW8XZ8P+A\\\\\\\/1dVlj\\\\\\\/qbXlnItfX04\\\\\\\/FU23b3IrIdqhcDicRw1VxCKSDBqsSNETkLbCAUHW\\\\\\\/OZwbt6F16hXLcV8QryyZYsFyXZ29UsUvelS2RRN+0WEUsdQQLkTs5Urfx\\\\\\\/IPrRmxMfXpmWsMcRP8Zgq3vzt+3Msa8RVRz4XkEHbciHovHPhgL0oTBw3vtEEWnCFZ43xIx9vz4TsTAEUBJI\\\\\\\/ajit0ChLL8coqOo4UWTSSnzpEsazBpn2nuUvsdeclZfkTPrbn8x1TTpNFVB5U1K7HR4EbnH15xrYeL1ZQFm6zZCIiT4wQ7fnS\\\\\\\/u0Dp\\\\\\\/SvTq8p1O1KVP8rK2RqIJKA4jVRsEYh1Cjj8J58cgJhUt5v6R31x\\\\\\\/Q+Y2\\\\\\\/e4y7Z2OhxXc4YDJJSHkSNa4lQGEUTcqboQrHnAzGH3aAMYf+U05LnYx\\\\\\\/GbtBfKz+URoIRsgFXJXHAS5VsN2eeOcOaNTC4S+OaZ8TrmByVT4Z4GmIi6f\\\\\\\/45U6BaZQ9tfR3N7ZheGgyAobIo89lF+XMWda8la4rSWwxijJLmGnP\\\\\\\/pNsyQvEI28DS8haiLI8NIJOu9aH1Aqq+Cah+mN3mxyyYKI6JB7hQbFGs6eDfoaoRY0riVkS6EfyEGG+bgDaYE8hdFWLXGBksGfEO54HawO7dCnCs2BMgnO\\\\\\\/c+DOXo0LaolhqfQyPVey85jyhvyGBREImJGYoNb3LZHmOwNyALlA6CqfkVkTOG7Red5P6ymtQTM+CH1cxesPzMu+WnAv8jIaLMmRgTB0Kmmnt6\\\\\\\/EKGpbsQLDGdZq3mOrGXpKXW1jUT7fkw4nS3y4ggHog7L4e0KQDVDBYytTSKKi1AQv55YQZewymGdTZQn0\\\\\\\/2XAfa\\\\\\\/Ebe9adjFbq0HO\\\\\\\/6VFCrUX91RXXyD7OQbPDHhBLFAxu3ttBT+ucYmhHQMRQgOZ+4aLSvhN7CFni0LAS0dVfpNdDw5rI2J5QYqs9Mq+IDnXAOIJowC3KWmClRfaos0A2N+VnZ6Ex8JBOzOwlrAFFWVMXAdlKvJKSCVKxHJPsdN5GOBb239kbZ7++xwGgcTWBk0\\\\\\\/awLLDrOSt4g4VrXaUU0893Uh5CywDCE3o+p0lm3WCDQOzGvm5P\\\\\\\/Xry8Zxgx8BGCmj\\\\\\\/0uJGHiYhoPfBQ1nFgnipam\\\\\\\/7XxA\\\\\\\/iQdbtcfjnM6drt9iMQkg0tB4FBz45psWPpRzRZjxD82V+FNCrqLj5jlyJdBzzKbaFOolY1By0RdwBvYcCLmaR6T9bjzuUQ\\\\\\\/RlG5V2LM69KQ2DmfJfSluYbXar0OOprp8Vg1IbdqfQqSwMMn9G4tqcbeYGxauAu8o1pYWzknwBnYfAYLP8afgGd7WRLDrkXKbpmcFLConcx2dOx6tDlV81gf791lB+LdVIuCQC2bBre7VELLkya9GKhOOiVWyS\\\\\\\/2JHm0W+5V82yx5+j+AgCi0JI5QFJvdEDxEvZgg6vLupJuVKYudCVyl6J1WK2CHOO8BLQvtaeKfi0OYNpxb5iZwlgUAJI5g\\\\\\\/CLNiVxu8K3Regr2zOT3rnNWKUo8jTpRQnr9XVzHQ65JYmxQwPwXIajypv8G\\\\\\\/Ro6Yq\\\\\\\/cjmkQQpi8tcslslrnlsWd8aaASonpDcIov4Oz4jsviKjd5mjxEUHcuk2nW9aqY370c4E9xCwHLxMqNMr5KfBv6wOnChE\\\\\\\/EvJzSgU7NWT6HOeE0vOzOw44WtjuSfo2OeQFJmu8faU67JcoR4do9Y2WUTxxgVo2Gpli1l8ZHSCNMNYKfPZls+QT8VPf+MJWGR31f2MjPvYjzWf0HsxvF2NNLRciVwYglObP18rhVxpweRQARagFR1AtPZMfAtp7orH2f6qz3KEZuniwG\\\\\\\/x8jKl7MUq0xT\\\\\\\/fMnTLy7OD76K85dyeTzLoLmyTtpygWQm4wp+0dn5sz8WkQBs7\\\\\\\/stCkk7vWmmOuqgcBkEn4wfc+IZf4n2zfkzDGSFmpTksEaGnPfOCv45fCu0aNINfLEeRf265Pj8gwdOinc8U1Oip\\\\\\\/6eoELs0ivo8LZKoeehBehW2\\\\\\\/CwQvJHrYfM68snBi4n6GqNrgLTk3By0bVFgfDOG2QEf\\\\\\\/4L6QaGNAJAs45kBfYH8GQzz\\\\\\\/i7pcLjKqjD8xpd9nqoVzBEFOl4Y4SU0akQklfEm1s7XmgJ66GflvHaw8fb3J2s\\\\\\\/LMFMwYeAvrZTwLq271u6GZIOLnb33dvx5wiWYqBXonQ4LOjM9WUhezxpizPtk+XrFMwU1bz0y22YAo1H7UnbblggYT6vQamTmuTNk7rKcb6OQq7UKj8t94VU1b1PafYMNqom27zcuXNRKazwYw\\\\\\\/Vc9A9w1UDoBbe1Skd3az+7fLyGq2jAxF0TJDD147fW3rCxQreZlt7OwNvLPX6PPXZae7R1vBeohHk4XAxG9ASLeXVWZEN0Qrt5GIj+w1L+QdKvcM27cuDCTSJqfZIrHgbn\\\\\\\/apRCV7TU6ZHVAsN9EjaxYvUOH5AYQtk5G35om2LD+x230vN8knYhujgqq\\\\\\\/H70kWx692j3smnAva7li8WlThUJDIoWzllvWPfubNQgNXNsqOUUt8MWC3zKfxlgSNkRQEjhAuEx1BCbSXK\\\\\\\/CQGuqvptMfIm\\\\\\\/rxGbORp3CI3AIl4mdn6CuJ3p2wtjlJMNqP5ETm8DMNy31jOyH7V7mis4fxss6lRjKriQ4gfc\\\\\\\/KaXfYvm6NBR\\\\\\\/2vFUsRu2JNkvFHFvCRhVeADBK11AoTNqTTKEGEG8Yug2wmo8NvmTRg4XzvvFZglY1CAR74cwHq21He5yeS+tN\\\\\\\/zS3o81oVgB5Rhoy6cpLRyEUDYPlosMMMcUQ5IKoTL63\\\\\\\/AD4P3HbaPVS\\\\\\\/eiAzOo6XwKdNeQsAGBAzcCjxm282ETpl+CSrcons\\\\\\\/e9q\\\\\\\/RlPpyGtWZFPxNLCAHSfM75859TTMP+PotStLyzr5b+2seXDflphLmuhb68TUpNRK9O5SG3AhwqU9EIcAAEqKDSvM9gSrzjA\\\\\\\/cOwTYl7ylojfMbb2+ryHKrqbf7hB1GzFuV1BDtyKd9WK4jAxHANWIPBeMpsmyxjk9UYJQaG59MlI2xMpt6BBylcwIDkCrHuTCkCvF\\\\\\\/xzKWIIv+4irZ4fcAtz7YaCpBUQo3TcSDhE90qKP9d4WMNLAcGO2E7EyRcxbHTOu2mZJKVwb9UQXgWEFvcyuil1zbRMBeYeATSo2k6Oldn5C5w1VfBdSkCZrR7eFT6GXk19wPD2vXEkvIkmHh+ZTxSdMk+V5YNp\\\\\\\/f8XiO5eh39w4G9EqepdDtbgV7QhLR8ZguIzNFw3vck3C5oWyu6LdBhZG8Zj1bqFJDcvZYaRZuF1\\\\\\\/O3+qCgdu5ng2dk+HDK129V78l3Y39OFE6I6g0AxoHmW\\\\\\\/1MuzEBsH7v2\\\\\\\/DZHwtFBfS9unM91wLHvrarVdqi5edXbiVEAfZJcni6+fSvMeyGrzc6+TCguLudM3c98TBJmwZubDKa37IlW4HewNPrgYTeDw+RpwylFXIz0sFPMxmo4DOnU6KZieiOSW+N06MHpDic4w6t78kQ\\\\\\\/oO1HPeQn6e25Xxph3CZpCmcVYRAeW27gPBn9Kvd3mJedU+r068LhZG0Vyq3\\\\\\\/XFOTjz7LkYSrMR6M0eWGBNdu4xQh1Y5N+4p6r5chgMW0F91rFMG6mq8\\\\\\\/Rhs3nILrQJYa+FND6fPCIwYuRjM\\\\\\\/PgqZijKslWxAw4KO1n6HPGKNdkHEqBU651GPNEaqRk5ipnJwoMqlufgN08A+t0er95Uy1+OsCHWL99NArFx0ebh0+kdN66TDWPlVnp4rqw4vcv+On1aBic\\\\\\\/LjdsUOBoXxHS7kW2\\\\\\\/c4IF1xDgG6tuOL97Oj\\\\\\\/vNaCVId0aGNMe+wj7DPqnVtaMsKsh3Yejc3yTApmO9g0kvtQ+1N9zz4sLv0o40gpp41yLT4IYr6DkLq\\\\\\\/HD1XNvsG0yPFtx9OgTTDrmrCbJy0nHvX2Fa7nuAzmuJ0Vf5RwFrRDkeuzHgiXfCoobkAZHULJgVLVqOK9epTtUIkbEdv7pqOr7LOhJkGjyz+YakjQUv46akEF4sb7js7QNl6lf2q8rcTSa0ygUaON2tedck0AlWH3eNH7T5rN5az0WRJkpd0M4pH5zG6GC34exO9vUGnNRX0VANJlXfr979CVIz5DBGNMjPeYWz6Ly0o6jBMX2cZeM4qpk3iB\\\\\\\/wardN19swQ\\\\\\\/S\\\\\\\/PS0mUGGjHnlTtZQoja87UN01uNaWi1DikmsVp2lNXxluC1FipHhQynlmMN3diCvTUc1+0Oh+U5sqHeOXJUzmR2T7A6GxYA8v7ERMFP28Hv6VJYobObtQl3rllUexlYexDfdZ0H8UxYL5PO\\\\\\\/HXI6gAJVCrd261i5vDG3TiL0poLdCEGzwsNptxRy74+g\\\\\\\/Qc0vFBs9xUOObcIF8JK0hCqsWDmceKiWLNC6YpEE3CRAD8bx0zYJJ4c5gYHZsIBtnsedRo\\\\\\\/4UoVPsay6fdRsAwjNsI89XqHvErHwzXCuQJL3HhtY65nJDCt3CKWNgSFiJo1BADOUWJJ+iB3P\\\\\\\/tSv0BJ8pBwFTmsvJb06K9aiWBihIvm\\\\\\\/DW6hL\\\\\\\/A9caqsLI+O0Z+nt1PPQE+t7lB9NgeCnbM5H5wnqAU5rccU02hjnlUooXr6AFmmkuecHDURaOXDp6JATZtGXaNEZktl3061atOdfWBKeifYCxO4qi7NyooCY+a99ZKFTP3gz0bv7zCBWnTRYP4mUdBU9Mym3J6dWvhZ108tveObjSlD6VBUpa33rtqSD7tq4RoLpkbZH0lv\\\\\\\/YurCFlJfk4PVUyvxF\\\\\\\/j0lJ\\\\\\\/pljaScisDAPD8jt+xrSpawO\\\\\\\/YYgS73ix9tDjcZ0vzlhgKUVn0VqeGdPUuXbJkeyYzLGRKKPXm92YGBcCP8PKNlx91FD6tycHXFeMZwrTL0\\\\\\\/ct7uVdN1a4Ly56Fx19O6YMf91WA\\\\\\\/qqTGhwvlUTZSda9CH0GHlfR0np5mkBumhmCs1ZvMS8IXqa2WdSsDdv2xgsxkPhvX9zl7dWMMLZT5Ycnc55dGttuoFkcVM61DBeOiASIvLAQ4yY79JP8ulb7X9nmt59jtlbxBVei4S7vJx3yVDqo4vZJq\\\\\\\/k8lAuRotOQrHHxa2oiiQb5\\\\\\\/8Su8tNN9rpdrIw1mQ4zZ4kxjQnXv1McFCtWi58rW8rY6DbfrbbzDjHRFLVvwC\\\\\\\/MyoAY2a1yqAzdxPx9vI2fKLc78NISpH3jNymkJEMGcTGfYHz0mfe6POZrT0FdQbr1quButAWqCaMlinHMgWbW\\\\\\\/mB\\\\\\\/ofaA+Vuu3YopS8uAzynnp0utf0i4svVegdHkeUJ33lS3nVze3\\\\\\\/2smgjfcGSYbhan\\\\\\\/uRsWFz\\\\\\\/LJORpOyk7jLhizKqu+b0ASVnW79C6Nj5kPgIjN464BmPYFoKaehFyqD9qb92waDHzvIqjUM\\\\\\\/cD8SeNInkStMeb+R+DxsXhrkf0PQRi8oyzCayg3dHl9mxnYjc+3199yv\\\\\\\/DyjxcMb8cArjjOBlhbjFfOPVqwEKtfrU2SQv9f0gmx2wcH2WXo5KcwFpZvz5EbWfy3aUxs6k+IQaWi9RYL+IT\\\\\\\/KgTMpuxCm6DiOtzpiC\\\\\\\/LyxNe6lPMuSR+F98DaV6VtetQCXgzWrRFMmwbyj33bhYhFiTVX4T\\\\\\\/HtksczRSinVlSvIrYUdk1X0Y\\\\\\\/QPS2iMXsji9+p\\\\\\\/7kpFADzu2YdbmA+G5BwYklt+csoKn+3Pf6\\\\\\\/PA780ukxbiiZyzi7H4ijA06Q5hroxYV9u20yZiXvkrhfidAMWyix00jJXfThocerpCpt1Xf3GKjRCiQuM1hYWjhKMINqx8fHUNAripB9innmVEPDs2L29nhv3lEabkJIf6P5Q6pRD013ViC7fZNXuW3Rw2g2r8w+oJNchhBxi7HIKJ\\\\\\\/jFn8gW18h9qFuzgC0H\\\\\\\/XO\\\\\\\/Xz4GGtzf3QIX9ka9sbrM3nttASK\\\\\\\/HxlRm5GTGM4r43CtoUgKzMq25KqZWvkJlfYMa1ToB4c3vLVGSvY8gVt9+7kH9hY9ekg6ZmtN8CjdKmJDR41AjpDQnaBQwdPdIvTddqi\\\\\\\/inX2mWLMN6e50BwkPUrU3BpnBwNb6Cgp0YUtcO7zEM4rC\\\\\\\/WNQ9vNfnNIxNPBIFxyGfnE5sui4iUgORcNDkaV9BwJsHz7kYr39rpnqad\\\\\\\/LjrFrV5Hcd3VLVdO7F+Eafi3gZ8GQdjk1io7r25eqbaP6ek2I4+DvaB67h1NmSIW11\\\\\\\/MtLwdo+DRrqQmLf6bDNFQVceMG1ELtHigzwbp5DzpQmZjXANphpQlNzJpU+mvC5ac0tJW4Ph3FVOzx+NWBNoNHOycDkrmCgVmD9LtRia9JRmuRrLVwq\\\\\\\/MFXPFlrLkpjjF\\\\\\\/pw6MEia35sTNAVfbYgt9Zyx3ogkuMrJBq3O3e76oBrst12\\\\\\\/XRxgcpKvIjYZ4ufvXlLwDEQM3BhfbiPdnufZDsEteNPAXDELZUjtwkjqxmd0yLIhsYLnoYCDV15kU75LqJvpe9vdzoFjKtyu7bN+\\\\\\\/VqrgWHESb31Gzwvy00DMCDQwFuWvsqKQJWuQSVjox1mI5ZSCwK01MOo8BUeDPuMfSJcvEwDUmsJLhiQC4RKlv\\\\\\\/F0c+ETVEiMxwH0WCtuxGSbnHSa+IYuUH9Mw\\\\\\\/KdTGszLJBK9ezQhTwRNCun3+f+vw3xjhrPoLLysnW\\\\\\\/WN0l7AyHmVidRCs4wK1VuH1kmz7txh8wC3fOTNb6NOC6NQFa4xIHAyGl1c6cwE\\\\\\\/agBBuI97wntzhjFf9sGfvaV2UKSnF5fnpV7846Xxdk8gYFzTz\\\\\\\/WSiO\\\\\\\/JtPHg6F4zCyconfGXbhb+KscGZleQ+9230TkXqU7Hjr1kVfnxCuMJA3CgQTnj7Wqx2bpEy6rQ\\\\\\\/\\\\\\\/+\\\\\\\/C3KWRRumjXVDJ0RJVdc29q9GIpvI9xp9pj7cIyASWIgJl+5EflYT44mlR16P0IyBpOA9V24QPS5RqqTQWYRNOrLQ1TTI88v5VzVY5ayZ\\\\\\\/YnG7jXCcX\\\\\\\/y3cvZWD2GPoiRvggNFW5Ozv\\\\\\\/DF0acPLWXM03wH\\\\\\\/kVh7\\\\\\\/TMeOcKqnAVkqh7EFOU0TEbNdPKN8Pc\\\\\\\/0JVcAW\\\\\\\/s4aH3T3Y0WP2sbiYb4ko5orkTr0ItctUvJj\\\\\\\/YTpikKinpKEsc44nOHMsKe98C4ghxuzB4indjLuvzW++JIkHykOtNgUPIo3n3uudHnZ+s9XL0IhkDIkWHZKVBsnDDUDgowfho4PlnrNOeu2gy9XlrElM5hacRCy2LH01E9tvdmn0gK1tkmWiACMyd\\\\\\\/6EH9I4rDXusXKN7wFfn4Pdrw8dJzaslcBKbljqdCalhXKzVBQYHIRgERl9+4yqyEPA1U0O2VJ3SgonyNp7YJN4U6XkpAi5N9trEevUXvc3D31o84DwCh+fY2+SkgXNK09UDz1fs01DJ48+sBUJH4hf\\\\\\\/AnkJw1m1AN1yo1lOs7wX5EB9reX5hr4tUraayoyt01RuItKnK\\\\\\\/\\\\\\\/jdXOWPJg2oqjf75sByStb1ck+w18VAcuFEDKpQE2BYlppj8sagA+TcyrS3Wh2kNKPnq\\\\\\\/Q+7aPlxpGqOICHI7xMQnjhSQr\\\\\\\/H3POm6\\\\\\\/\\\\\\\/sbN6Ue4nj9nD3Z4D8GzG2UKKJ2YSWmlee10Jk1O4d1TOE8C8bDcn45eR7Ec4Qqpk74vZu7rmo72dhoDIqp1vuMfZi145cfgc1OaX9YBuZ5x\\\\\\\/IqnAPt2wTZrMh2IFJEFNIT8e\\\\\\\/fsbQVsUZNumtUaJfCgltxo20VRozxVixZsud7HMymBL16ruI2Dae4lf9ypjX99WvyLgbBKPmpBWrTyjkrhfiZd0PHBp+i9eGn4CFinoi1KeonwtnbOPAcRhpf+mgn3EekrgP3EhFGVq2E7bVZjK2UyE23JNLB+8Cmp\\\\\\\/yKuG0Op3AqCLfdcC7K1IbmNHIdI96Lgcn670MAlJjnTGpkC4kA4u0\\\\\\\/EGT\\\\\\\/lX+zEgGiQhV7JBC6F8vZHFifQoVgDIwv4hdISEWv+wntx2rQIh0rGl9x1lFo1UyPI6uzDA9NjXSXkYYspM1NXSD7t0kNSeDi61S1hA3w9agYltZOY\\\\\\\/14Jwypn9MeuDLuk2ihRFWA6PlPnj687LvCXCPPUh6tLHIROMyCawLXYX+Tl8u07ugbQVjFHPr2WY+qyuYhBhmIjysCBs4wYU99EuomVcRtOSiMtatg+\\\\\\\/ENKGWg3SaPINuaFxWupoBSCHpTw\\\\\\\/J6oJbjTlvTDB5ycd7pTLdAQrsEXdRarnQoiU5SQ\\\\\\\/9TctXpRIPWGja2zTzYfz36XAG3bbcbhtgnFyS0XitwkiDTvl4tcxnJ469bFatoYzVcFdIP4eMyO45Eox2Z8D7D7GU5UTY5CaG6A1i66yBILr\\\\\\\/lSxSaVwETpVc3mnDlXi119o8X6ftu\\\\\\\/7AnVnfNlxFR0WyywqGWTG91O0M4WNhzicp0znbvED8nhAj95ZYy+Ib69IGVqv9RB8KCa\\\\\\\/wFDDJJ6t0ZJpUXBzD\\\\\\\/viQmfxzbNMxx6d2je7zKu4SuX6pffRFjqMY62h1hn1pm+yVJJPmqK02\\\\\\\/IWTw38OOquDoYTpdSRtdSNeJfaJTAk8Mi40nQd0I0ykiPnBth+LYKu6vaTr4CkP\\\\\\\/8hWSCSELS8pa8e07l1Yl8d\\\\\\\/ZwyQTHYhlZg0xsP1Ph+XiuA4Bga2ixEvYPD0M8tOGvP6u8lhuvgjAfRvJF+ieJj52JUvkNEBWOPuKB98J7SMEBmi\\\\\\\/CLPBrLU2REIXlInYxxdsG2CcJB+Q8xWXwk9iMSYeQJew1tn1ZJjm2H7W7W3Wqi7RG0WVQB7mhQs0us2BxIUTvmfCoLicgAEcD1azmlvlgMO3xvQgOAu9hGewblELOCWWQrV\\\\\\\/3C61q3t6pk27lTFl+f\\\\\\\/xmWODrxJB\\\\\\\/+ncZ3PIKL7y36bjRH3S106d8tNMafH\\\\\\\/aO53XVf2bRCb\\\\\\\/k5TLIDgit0Lcr\\\\\\\/WUBg7qKRfpLgvB+nqa04l3BUjhnXHTQgmaR7Vdf0nEwMXyjub3wo3RT6PvCrwaAASJp2MG9orXxJiGfO6g64bofGK2nnrhQ7DSa0oB9U5mLfFRhu3EJxCoror6GmuIbA95JuETrk8\\\\\\\/NLKcmFBbfsLbLUr8PSNXUWTPgyP6RWZZmvj59M1gC6CkPser6uvLKmkHP+e2vwEm5w9q\\\\\\\/hwW2Oq+06bW6xyK30QdTzFKTQbs8T9MrQkrCIfrdvoW+CV+eh902weeVGzHjWu9+5Q5JJFYf5yJyTDF69o2ESFR8SzmFYiS99QCPLlauKWenCx0qIRSo6OVfZ2ZKWB9mUhwt\\\\\\\/miu+oHbG+gX+z9z3FxX3WFNlZ0sKDWl6mRqDIuB\\\\\\\/9cbXXCSA2f3kNG9a1X9ivyVRc\\\\\\\/x+sN0MdT+84MpSRuvsaMiYYytTrz1a90v+J+dj4IUsUU\\\\\\\/ZSPzYeIr06qo+eSeiYrRDBirZ1QqDIvhMHIPDzY4YMFS\\\\\\\/d2\\\\\\\/T6H8F4pY7c25vxd6udFHhwsFqMOuO4ED19+bTVvw3XZ90FF5uY85AT4EDdNAx8nQsJ2bYtaYfWvotcS\\\\\\\/PAaCg\\\\\\\/zY2CaIYSsb9ghuIXECjzxDwWfrOKyEgVrlgGx7qPM4NuxBpJO4omHLNDRSJGFMzwnjIOHQxb91FtStUsfNQe\\\\\\\/Ayo21rwv8LKoKHDJuzHmu0dfC1u9hAtJLcyvibnED2hRFcjs4WND4u8cYlEHfUiZKVhchjbzFRkaAqZQxnijR+SHWcM2W6R3Lq0IjL5zqQaiC\\\\\\\/N1Ck8alUhqLGVeBYYKrwXExaVReFQsiMBFONUBVFlSKjzLEh+DcdJwlFqMMJRB4jkgaJImq1k8ni2AcHe4K8zOOw\\\\\\\/mWBeW8ZRPZqKObZOdLZ2VSZ2CAHS\\\\\\\/+YUTqBhhs\\\\\\\/OxbzuRAuBGOS9h3RLSyCJ9He2KsP\\\\\\\/QQwGMG1xcx9I98XEQ9TAC+GEdf8grIQxbfxrjcS98sEMGkeKfmdUvrYZzkqNpZkJqUX6UywLO6HM5CsjFpvvx4kemSy7MAI7dFwF\\\\\\\/3wWk99oSkrnbE119ypwH\\\\\\\/0tZJYCoIgky9BAUDJ3zm6daNL9RI2JENQgWY13Ehq6CvMRPffvTn9Bp+CMhhCL83hdMSUvzaxWSs5O\\\\\\\/DIApI4TZ5DQDzp+IBcSSNd1aB932H\\\\\\\/Qv8tQT+hj7FIAtD\\\\\\\/td+RA3dqvCNCEkTUY7UiWEFI\\\\\\\/NW81jJxlvriVNZRZj+9BjxXAN0ONlF12BjwHu1BbLpab3bCZyTHxUrHDFabBM9E+rWDys9xWFT5\\\\\\\/kZapM+szDWk54IU+zVcPOMZ9q\\\\\\\/LOj+1FHaC+qvyGAlfJBKltkY7PTFANUx1cSIVzSmpHGm+6ebbpmPggoYnHV29TyQfP9\\\\\\\/Ngr5jKr2xqW84peVTvZ1fiwpXmjbgaSnVmee5waxBsgR9CIEnbFRzQBMpNh19alLxaqsTvA934\\\\\\\/vzYK4vIUyUOiTNefklXr23HAbQ92j0LGSzBoTHPJ\\\\\\\/csOmiNSsGrU018T\\\\\\\/QtftDASnFeczHpMbTO7ye\\\\\\\/FCMz4E+LI48FtIahiliz09AIKrGDqeFyd7Q1\\\\\\\/fR+0Fsi1pSs0nbWsQI9KAF6lWoG4ZJdgmIrP4xhzRR8FRlTjnIvB8aTjxaCXCEVZP+iLQpYzItbJlGXabfL2mAAMWC5zzNTHKDf1k1Ml8F56GEaUXMjWzas895GjUEGAQ5BuKz5ldWNNutrep7\\\\\\\/kOeoBjgAM9aSy8RhK5AeIiI2jA3cCedh01FS8TB9woKxnh546LFlJs4MLaJKGY1ZiuGWq0l5GfMiJnfAHmmR67d1ZT0HyIg5m7eVbEavU+KywJRFyuLdOD6oPtKvItviqd3xGvPwwfloxItjqBpF5CAcacS2pgUh9piLyzh6STL6\\\\\\\/ia2LIBCX0Ty28TvBdzMldxkAXh5GZTiRjrLzp3SsTk4okzGhs6nLp6Teesbi0XGTcZgLvASTLQ33Ve9QYtZIzZMtxY6VvdvAUKvFgwoXXUanQEDbDg+p0Nv+J6HPbcgFghEP9yIFvKPWPqlplaIut2QDEMI6dPUrMM2LQRRkKFNnaiHDJEstfisjXIcHJHPsAwuklzsm1U\\\\\\\/Zk5gzaA4UrTDCTA0BJhyu2zsr+E6bOG7avt05OogLHTGLJtImDgMjVNTwRW0B6Yn510+yL06jB8hCbBlgliKmAHym+KtdUZYwogyqWerQV4FQ5LAnn4Y5mPmtC1D9lHfNydu1IEBVpmYhiMdCRuEnVpQ1J6eak\\\\\\\/lSIJ+H1VwsDhKq2\\\\\\\/PCX6ljRHQomMYJrKn6kAtjKMlJZ0x9KGpDMS8Zk3\\\\\\\/Qv+2E9aEV3G9Ld3iuE0ok\\\\\\\/E1TbsAe1uxU2guJnvPTMKuGS4PAt7W5Vc2PJzIjVX7Lw6EID78LeHJd2TKSofM5ytgUSZPUqrS7OjXVzznw8owOhXWAAu6\\\\\\\/aX09VDWpcGjb5NbDgm7jA\\\\\\\/cYu3aUrZ5qixkR65MQeHXupiPUwjjU3r1lEPb3PICdodX6KzH8EU2tZIDMmXN\\\\\\\/6Ewy4rg7ePVUgnS7Zk6dkwmA2SY0ySKOGVEoI2IcMGPxrLN1wdf\\\\\\\/nu96whFTg+j19fUoQnc8LAUqsyfRyb+fGW09oiX8XJSwWGPeT\\\\\\\/gMVh9fAK3aDJSCFlMpJIxkUJu+pe4OL34XlePumDWW9T7ybql5c2kjVk9CRkMX9OdXJBMHPTM4cXtyahtgkEuvmwVFJ7\\\\\\\/D9gMsob23ZW+IiVG2w1lRtx1gm5oCwnW9zbL3GD6zuHeLD7G9L0UZO1bLZddycPK4tX3ARKdksxE28uI2uTRARS8NKRGgJymff3W9h13urw20kTNM7FJmGZh02Ugjls4t7w0EhVStWP8DkZzyxbsuEJdDTBJYn\\\\\\\/CV72JDX5+mbVA7fXhtLErTo2+5+oGrMD2PI4UP2lMkigSLKm2ruHeoAXKVbVY5g1KTj2LbTeRVmsn3JNTWqu59kwKcmeehBQNSkeQw5Nx4zhGBkxCv36RAtAm0dGmWURrbj9IDe5DIQaMUlBkLepPye1O2vfXz1+j\\\\\\\/e+8DyzCbLoxYPValsez3iz8Qvoh4P5LHU3xlF23IkEFTjPnYJ\\\\\\\/TxB8+SNgyGgOGbBbwWh5v3O7U763IHm46CmDkVgm3LsWuXu0OhBA\\\\\\\/IMW7yUHD+sEoU8ingDKxgyfBPfacDVplUYSLLg5RD3uWF0p9Gajhgc66Y5Ux3LNPpmm4am3\\\\\\\/s35h4411AIh5XiaFxRpD1pVa1nBhmaljZ\\\\\\\/9NhVRXpBmS+4buNrbG4TPAJAKSIEwjfLkg4PZmCu9FL6a7FqtMuNCeAz4Nl6fa+r70y1APOPGz0Wqi34YNS8HVZpEBQePu\\\\\\\/AumbrqUyNdQtI80v\\\\\\\/FYHN338OL+5qJxGHTcejGBmm3+nGqtgSo5lJsF0PidoUym+U+xVBiGqle4q3USLajS2D0oT1nxp5OeiCN0pLTqEqFZK\\\\\\\/4UZkX8OdSW5JNvPZYtnVo+KpieNF77qHYxZl4OV4jHmoOSGV9cCFuMAav\\\\\\\/wvE35+ThQtSx9u1e4pY5fbgmj7BgNTmM49\\\\\\\/6XDUDVJchNuvH4QTn3k6tamiyxheSK2mu6skWALFvv742eRq8RIjz4AmhNr4SS27eYiwPyjIqzhGL1G7bdD7UsG8QILsHs9L5Vyle35Ed5228QihFu\\\\\\\/MNuV7MlAsR6Z4Im99OojL8RvSKOnaC5RFntAw0I3KiEKIMukzf9EmN8vevA8J03X9TpRK5MjvZMak3gpU1p0xIVLxIrjF2UBnduticS+7smZ+Rd9eDr+013zYDGQ7BdozOQLnALOM1wDCK3Hofj425a3PdSBJuEtgrNsEZ3BEH6KnSBVgCW2oyqmy28ZHr1OphWkZLHoCI6fm2W838H8V6HX6vFktMTNpIXRG05Gajj4gXJK9PfhrL\\\\\\\/pin1JC3qSm4UJBevpVMACealh1Ngk8YadvjEN2lBMnFEU92NnR5YmRkDXEyfNQ0S0aZtr7WqgDCoRDQiklz9IcswkUyPucdUDKLXJOuqdMF1PGjTY6ODJmKFL4f0p0Mte9\\\\\\\/7HbT82lnYviEq1hQRdD1amC6nIZ3CcW1zEL57YGHcvHTkSRGP5zwgzGnnF7yo4QPh4SukwVEqAbjylRMG6iEPSCSY7137zNZzLjCiE3vnb0IdYMFAo0KwxctcIq4RB76D3ROyMbuL8QgJOMM9TtPr1x7tbLkbg2tmrDyizs7MBomGPF4\\\\\\\/+Or37cp\\\\\\\/TGvkrGDnbgqKHO\\\\\\\/Xy7MDUg2cVj4MfiatlaT3f1xvIJ9wsuBtsTx12sQ3HAhJLw\\\\\\\/9xV40\\\\\\\/7KqD+qmYqBc4kfJuRkzscZvR3rgYwNr3OnrXtTYbIGakB1j2L1nvKLHSPDsVROEXB9SbEZVV6uDUr5fLQpEpRnFwHArdQUnFIgnyiCkgTjQZOj3lF0wSxle4G8VB4OZ6fTB47CRO3o+99iAJ7IBmAtwUJvSBppHlizCE6m17rpVewGyKEtIEHjSCZxMDjVZtMB68098DIzVO12Zk0sbr4FeJOZ40eQpVYx2NR6VGolPr5HIS+yUZsgrDdQ32H41tbMAeSYnBY4PdF7KhlDMFwrytcf5MHK\\\\\\\/ZCDPT4mFTWslswyJHX3OyPL3sD7GhLBOYRIaoHpfHC21xpCYlxUUVFWvZVpH1h4dKDVqvzZRYLW7YqFGUlZiu3INCWSxtn0F2CinHvWpp2v7S9oCccSr+XyqTyOGdRioWQyL8GJy6JLs7gr2Us9dJwwbt3tqo834LLh0NJgemF1qq\\\\\\\/o1HQYchuHA566B7YIwX8PLD4AX1VgQwPTybVlrwwyNN0Si4\\\\\\\/VpG6Z0HQNoEe9+RwI9tUv7hOxc7WxHEUEEwK1Kv5sJ86eTy+APmV5nqQNumzNLwssWlW6WAlUVJvjHuz0A6YhjzISDrVNb2Ys3TSIvxDfQ8tvWOBTvcG9a9rFiCc0x4kBV9o\\\\\\\/MrFSsLJqTx7x3Wz4Azm8IYwMQWIvncqzpTynrqvFNnqnfY9Ebv\\\\\\\/aH8LL\\\\\\\/Odd1O7EslOFJ3BDeIW1fIbR+AOnccwRwfb8S3HThx4Kqq4lVbV6GBJZC6HJD2UUOT4gd+gp37oNHoXlxmP\\\\\\\/uEyBbkrosYOy9LwXFRKFZ5k0Al0imFIvOaLUxvmYL4bHYQvwuc8nNTRjFoLrwtbRsKxRXJ6EKZcrF+goF9uXxuG5mprOL5RZ+ORodYWxaE5M7u+6qfhFYsC8689d+Z5WiV7edGYmWs54DVcDkgAuAe3QjKj2v4hcAgk0K5AzcjdDLI0c\\\\\\\/aT8AMVvTpQVyGOGiDe01plIJ4TZppu78Q1ddvaaxo7\\\\\\\/MWs0ovq6cappLmkuuEL2hlSmwrQCwybKwhhclCzA5RUo3stnyVHHAe+fzZlGMsCtfAmeab3tMQHQpESCWAL08yGQZX2yorxjymLsRucLzbGFEAdOLGtoikXax0R5TumzWqSMxWYrdFwsnCzyV\\\\\\\/\\\\\\\/rFrOx9p3NpJhoUICEI+y7jtszGjBWl8BbqGe39aZaPOcdKStK3fy9z3PJAWEO6PnQ6pYObuU9ML4zPY0FM8iNmxEoKw7f0IgYjGrimhdcU\\\\\\\/0J\\\\\\\/\\\\\\\/MAh3LlyllCHEwZTgC+2tRcR7K7HSI9u9VFTzFEnPFCDZDg1FqUn+ddRgXbi\\\\\\\/U1anl8E2QQFOM2tmPo0gQw74y9NQXj3gdxGXolNKRdc0506sRtrCDHuPPtm04lLpHVzWeGuKsV+QakF3fVDx5ZNPdnk+ZJ7+I2YPBQkuJ4Hu8eI3Py0Ac\\\\\\\/Rh+sdBnteYKxHOX5exG6Uo6\\\\\\\/K5kE2Sr\\\\\\\/sBvmL8\\\\\\\/xAEiXQOWTZWHBJJoR190ce3GpPGwf3YhJszozvD9S4SjfYVz85jODynyZ66EDqU9J2B\\\\\\\/7kz37y3aYMUrCfBOIAqtM5qiGoiihrMxIWrvBRVUm27DW7bjWCxafK1AHXOq5xfQD4\\\\\\\/v2+bvKxzx36NLt6qp0nGd2qkxpv4ChYmt\\\\\\\/2U79BnY\\\\\\\/kbDoqAF\\\\\\\/1R5wiap3HKq7oNlpnYKmNKaVoIME9csMMfQdWT7ld3n50JjVsxTNkBybGtCxIqydz\\\\\\\/E3LCOcPiqwSW30cvsPFbMQUxCQGBjx8TnZZv0sDEHhd3gTbav1yzgXGpy\\\\\\\/L3P59P1Lhick2igDFEWa3gA0h9aF6z5oe46LjxK2qjxTiSnQiW9WHnv9GzXe10i7cxnZaTB8z0yxly6d\\\\\\\/cvvS3Q99qgbVrLUo464+WaMAq+dnkVEpdUv24QGM\\\\\\\/EgAVvvzGeZPFA4aSYkBYjKYpLob8dF9dnuwifI09is\\\\\\\/Z1jxBpMzU40lhhmlidMjyWZMxgukLQFxGTN5hL7fIF05Eys3G6ug\\\\\\\/bXPt7JZzoYbYXxUT5pTVTNXuANWGOW8J7Vzb76sCvIMPHGf\\\\\\\/dsh7zDL6hxsTWIF3MBnp3LegeRsaGyrIP6aq8QLtAWjIm3pccRM0JHLN1yDgxufPEaYEuJxGJEz0eNZJtKLxdd8Pbgz4X5MSpCbEj09okINkFr96YjbCLdCJCadOaTXZbXY6vjnmgVLirgg59Zdn8b5E+miC8z+ys\\\\\\\/ARk7+7cxQf4JCsiXVZIK688+9OyzEGw0iafLgUAWdEIC2iedj+gJhcDfws10phZTtTEqoGdeppgY5vMJvklE956Ay4VU5HLxn4I7pg5jK4U7XT1TU0NOJy7vVgJCucSj6o0XpQwDEyXAwumWEq33dKIgLZZ5XNmQYwpNfwL7lUA\\\\\\\/bWDjRmAjPDK+C9bmcfVALoZ13DS\\\\\\\/Q4cf2inL6GU030LxcHVrjzUYB6eGFz+3crDOS9yDNrksZ9MsHNmvcMN0ZZ+YKytnxBvn5g7lWLv\\\\\\\/VlTCDrJO7bwwkgIMBle7iGRo5xyl2P9AScYkVy4XJaHMyOta0yzc98HnPt09FzQWxp0ro9wtqoXSg8fBEtjA+4D27Z8sCjjR2JrWwZyXyTY2JHPPww0mkPfdTUKkTjA4\\\\\\\/ovoTJStefQgow3PXqyeW\\\\\\\/x9sIT6E39D0hQXdoE4zApOyZVLb5oizXbHjVRy2Ql0POUda7+B1N5xOj2ARD3EWLWnV3fyt9ePcWYDf34Td\\\\\\\/O4c1KHJLHgoMckIqmbZV8ajkcD2ghN46vkTaBCUL9BsH2eI1bRGOhln3WuMSl1FbOnGLOTfH1NXiVXccagEBF3wEgvOYyBLN71zuD7+L9\\\\\\\/XkRCJ\\\\\\\/paG\\\\\\\/ftzerdeDXn6Py\\\\\\\/XbMsJa+K0hvYboJTdzX79McqaKJ8qZRW7X2rzGJKpuBMXrPoDO1sbjEyPEMI4ez7f+I9WYfnNp\\\\\\\/q5g+eKjNYN9uO4UacWR0AqcJc2Emx5rA0fiVQRy4i1wvoNWjT6OgSMybWEMNmnXFggbFuSTxTKMxncoIm6v1kclxikoi4ChuR9IEvkhKpACAKiXnJIOlO8S3EJoYlSVg4dF9Blaj07+nmV5j2dZnZXxibrkD9N4da5Y3DLuOmdHgOjRGFe9dCjimueD9eOhbz44deK5rwrZOhx6U0Pajh5iGKmb4P2y2TQ1OGhqYD4328TospqsSjmcSfUXz3l3mvcwqCHktpHCTKVVX9jdjKEu0F\\\\\\\/BapsKTmVzE5ykKydoWKug3xRRm71LLwA3+44XspRDqdqBz\\\\\\\/cnCKeorg2ZaBT3BLnS2+8fx7vtcyzD55vI1pMTDKLI0KR5wCDXHJm+V3f8vX6OTndkPiLi7XpyOLkuqKGalHUjWcH9+wm6fd1Cm47AJ7kI4awRX6dEHh9v60xG73mnQtSIVjPUNsr5I2lF0LqFIgpn0U8gkWVO9C8ha6qnARS4CIPUirO33OPGwQF9ZGfzMmyIR2Lu6J5ZgNBL4qjWwRbOvR6wxUVstF9O8p+98KJI\\\\\\\/burv6xiTgaYCzdIv1Z\\\\\\\/t75t8vuza3bMv\\\\\\\/ONYsKiq\\\\\\\/z\\\\\\\/5iJgF36o0+EsS7OjJ5DsFV+\\\\\\\/ZgTmSNmcSs4tMPssQB7kmLY0spzpWGM0Vj7C0HZ9yIx32Jyr1tKFzRkfujFVyKFjd72RMALLOu5bkTAa\\\\\\\/bV34mpfpJjQ7I5PHeC7Y1nMQMVFsoG7mIJ1UKdcEq2zxsT4JwiD3yxi1rOQbFZghzVGm\\\\\\\/QiHc3NJhmZWxh9Qxzi\\\\\\\/+R47anffCCHdu\\\\\\\/HwLdNLtJZu7QeUaO9VOunLmFWSl3Plq\\\\\\\/XniDPMzJ7Xz8XKWmU1fL\\\\\\\/dnXbzhN+UxtSKrxoZpC3kBTXe6i7vGpBXoIisZmOdFbdjD2DvDlxs76yPXi\\\\\\\/23iB89YroGQP0nRaipO5kjNg543zzZ4s9PknIhrxfkAni5aq8wB0taMvrxE\\\\\\\/5pur4uytAC1ocEgJ\\\\\\\/Li4D9TYhGghsFF2Azr4NsEv0mbIlV6yDGTf4Ib0VMR1+0H1gGqB8e0nMG4fU+lDEx5aOWZPXc8wdWtq6pv9S\\\\\\\/AiE3Vsa9HLZ8uW9r\\\\\\\/+wK2jM2T9SXlTDsD6mEAT44JUP8QP7q\\\\\\\/nLqBtKehYo6dEdZyiYSRGsiupKaqG5nZSUhGSfrTRJENfNJ0cxYKcwSNLExmky44rwOk43NMw8Eih\\\\\\\/geRleWlKsoDbzpgJYiv1bzDP5OS1ziJ7CBDCS0MzR46iI+vQUadYqqUkgx3H7a3DyhMBH0cjLPNxk8b9vaHeCoHmbUoMCsOmwpi2igQMUrqM7qof\\\\\\\/G5XmdxmS5nBevBNYAz4VohD5M8yYbI4qhCzpn6WxHselhNxnrCCWY4sdh1xc03lp\\\\\\\/DM48SDHcW6ff5M4l9drx4\\\\\\\/wL4prUDAYz6LiM5XgW62Z7itjZfR2wtAtzfxkBDynM2U6JWl0278e1drEixt1g\\\\\\\/I9bOG88SqL269zQqiwrI1+wRZ9N5AiZyBFeiqXcp4+QzZxb5Kk2CCQdK4cfSDq9exFTX4HaP2ofBPs5wOAVkscGpuymzau8Ep3YkBd6Tyq\\\\\\\/5hA2VH2sF7yateyUUCeIpr\\\\\\\/UFLafpJDvUobEayBulCofmE9GjwSQvi0+pRjrT5X2v\\\\\\\/5xLqa210FZzurQaG5BLBGN5jvFGfxbvderbUg6fDEnO34tA8EehxmXZLdbAT3aWAu4Sx29rX8+pr7m4WclpRuGCaUwk+w7SKo5b+y1zD43PqGA8WWkGWrhAIEpyi07O18Z4BKSD15cg3rzyo6KGuDs3PwTtxxpfOoxidCQpNj0JwZQ\\\\\\\/JwVJsEl\\\\\\\/7ZLR70OisgLjSHkrMQatUdJWs3qIiS1P+2xU4xUI8ahxNwqvXtmRrokPDaTojVkKSTA3+waLKkgmNhoWSnmsxhMyyScDUOIfVqjYKwqTf9kw4GKKHa\\\\\\\/N55B8ckI0rzaczjzfwGnKGO9iThsQ0KCJNyBtnJrmGxeYw9rLFZdya0ORoJt\\\\\\\/50SkWTNawPn1TdED0XS+D2KTmX1KxXOppFxmTAOw3ZhHVGQ+GTjRiJhQtJ\\\\\\\/Ddw4Tzyd+fhCl+n7zxwNWOdCRsf2bSengmjZR+1c0vZ5PhEGigh0xAPMpyTarf+FM0mg+31OYPfABzaBcedzfgydTJ4bDwb+tNDA+n+xl5dH22nJARb\\\\\\\/i3chWstD+RrAO8F8JajTJmofvk0WRSTMk3HVij36gnXLozePCFYBGAxaVhDpump9Bfyzxowp1s6KBjDFYjI2BiBENtMgSSear45mmGQUumN\\\\\\\/h+PbJf0PxgNOOTHFEq+n\\\\\\\/3PGBd47XQySOoCmv5D\\\\\\\/540Yyvmy4DvRl8i8eEwoEV4CBEDDSWvi17JJvpkOdXPQcaNM5UB5EraZEoL1so2rIVWHLvA59pd0A1pEmZ6QTDLcGgHpCioDWGdMT8Fm\\\\\\\/+ozNn\\\\\\\/w4\\\\\\\/6RTroLnaL\\\\\\\/BjqHyjpTzPtOmliJRqHLCTVJHkyWsSmVwavd3Pp+SHpZ+scsrwIn4np7JLMr3ecbxDl6zTnz7wW2QP7Wxhvpd202YvHAU6L0KL+Qn1mqTDjuFctcZXo+QahZIfgrkxPnUACjN4eeEdkbW4r6RKvo2yY5QeN0uGVQI04\\\\\\\/30J6vpBo31hO86U\\\\\\\/auWDJWQ+I1pif+rMo3WAo\\\\\\\/H5tIldTEUnIrJdtgO7s1bKEeMFOUbx3ax+EKhu1xrK8JKi3ziNxUnXQvm1hyfsfZe6T\\\\\\\/z7ksf9lDSKzygIBq5Hslgcu7TkU+xBoL\\\\\\\/\\\\\\\/3xQy2UWzq8CuirEbR1KqC40vL9gpIC9BsXbPPS4+YTrE3MCiWvSzOwH+4rWj8g7u+rI5ooHCR2iIVG3ZnoihdYr2VfzEiszLTmkQYiB7FyAW9137MNWMr+F\\\\\\\/LER008MTL\\\\\\\/CTmEHjtxqGA7rH0au8cFP5aMZy7U+gr4633GAO9zNuZeLNQXfkmQPxEB1IppyucdHkKmYz4B44B1t0odb0iAsWcYuUC6a0uNp6YXcagFAAbBBANBpxZRo2hWSOl0IOt33v8MLsdYdiXhyPMznmZi7nFwhQpTY\\\\\\\/YjrEwXTbgW6KaU8mo3JwVYDxnm5DlVVwmtZuV45Zr8mK1appDIAGpL2ESdbQ2pVhWOd2PTgzVMM+SL0Ikyx5sSBR5YTcOQLWtNHHLFvWBtus9aDrLCE3KY9yMAnvPAHZnJ7GC0LLDyABOV8VivutSMZb7bpb4cj4ZlYvH8l5S+WFEDFP+tNqXJ3rE3hauv5OucsvDJgrkPf24JDNq5AxMmOqEtLiy1e2r4cMjG46boH5WOJyUGeo970Otq0X\\\\\\\/AcpPd6DkloqKsPAMIYd0YFr+UdKW8LxHpEPkRHSXzhURGXjwD3YG\\\\\\\/9D7hCYq9yr0p8B3CaiM4DJLdLXCcBpwQdTPZ5kJ1Bmy\\\\\\\/osNm4pdqJ8tVZ95RKmvpy6BSjSf9fczvbV6h\\\\\\\/Nsrpl9xL\\\\\\\/FSfDYcb3JonUSRX4smRfQ502y1jEKMYkPDCu6fa61ajaMJEBs0iMlOm05XP9Ky\\\\\\\/5uTCCsnKu0t1\\\\\\\/Rkim8cLU8Z\\\\\\\/AkonPxm1FcicdjY4ld77BXH67M59hfELeGTOtl\\\\\\\/7tZforhJO1U5aZeVu5L8Wo8toJht\\\\\\\/WIt1HcAOCbmR\\\\\\\/TBCslQB+b5zyz3qI8bwb6SeZXfxL+W+xMLCse3acjkCvp81nY5qFRZWINKU1xxeTfrCFJhHeI1uW7yEQGWgGqBnHXnm54fnj\\\\\\\/vVqfnoO1ON7W9ZM75wwmCZTtaEpdG+\\\\\\\/TpDU0\\\\\\\/GNiS2KWguS0DPEfpUIfGCpjEJ11+oH7XX1p3hEAHwPgK6KaUTNohF3eMjXOpRr39eO9g1Yh1bEc9VH10VzE3XyJAfZrzerWAps9bNmVc4SQD+u3WCnb9oJUhKiI4DGqC1Z3+kYQ+e\\\\\\\/gw5\\\\\\\/IcXBLDddWctbMCkTrM\\\\\\\/DqunsoS31yotxb+wG2Y9JmI6aSmCmxzkGuK5SQHW4t2O5C73gjKMWkSGRPUid+A0jXDgS2U4bBBLxBa7iiFzg8YVHVRyimrIwpOtOSyitbqJxCZ9SjX1JQoLSZKpRSjI4mURMISJQPABUxebZXQUWpe4ijRPDUPpTaT+\\\\\\\/3gAmDDDa33CGygdT+H6gnd7glPxPlCciREo7MEkLc09DmK96XkmdgM1tn8Pq3F5azO7dtuqvXpun1eOLITqN0FvHEGOG0K8wXs2dV25AGmG2eVJJaas6kBUXjPlW739DpsYYpaMPh\\\\\\\/LHKhlz1QL5YC073\\\\\\\/2Jmo54a++o+Gu0uZ\\\\\\\/+mKks7xudstfjFjAASlmIH6E99c3GIV0+eSCUO1aFY+kxZKUa9\\\\\\\/arqjy0WkHJQh5SoDnuDGBr2u+fYFgzzaFE3MdWy1ICoiHq4Qbs7xTox4eW8gbYX7FFlyUlYaiONfLketlx16G4QMt32fluMhi\\\\\\\/nDiNJNGymWkgpw7\\\\\\\/6NOCxQpCsk7qKHMjqUyBglkb8AHqWmwmCfKgMQw8rzL1EexO2WNgpXdSpaLiln0jwc60PDGTk50IT8VKCHuBqEyize6tXuh\\\\\\\/tuBoRhRTPyPAck+JMkbJI\\\\\\\/coAKF+3vpdEWKRQE1tihntF9qzUXKz2rQA86DYCl7KQoENsAq4DFxkH1e3uqx13nfs5Ex34G3N1zlJgkqHygJ8j9qBBh4ZhXpiulrQM8\\\\\\\/ALlDkQL7fSTo7Mwydyxy+VoIRd\\\\\\\/BTGPLG3ruOcxTezw5VakE1JKPpANZUtUZ+tjsS6G9HyWElr1zHw6KM51eJsGF97mvzVlth6u1hI4Uz8DLw0\\\\\\\/Ih8A\\\\\\\/3LCFgAkX+8ssvqTzXGgM9vyJ2fSrVRzJ0o2pUWwTLCNrPCuawG49JZqKg+z6OOXkvfImkHA+POWwK9mfiDm3pahqPpixPjBgsyZ0KVzkB+geS89iAw1OBPnDuAHCP7ff+YFv5Ig5IWgXeuotj7wGP1y2df2F++ilPSNbmOlo+dUzO5Y7308H1XNO+SNnoMKT+2q71CIW7Seax9geK1V8HVIocR6oGzK44KulQbFWx9MxzK82Db4cCuQ4yfI+Uz4n8XUkHjFp0gaPpKepnS0vTJjd3hmrVBpzWb2OfH\\\\\\\/1nPM7MXSlLj3b0MElXbSAF+78M5VvSZqsVWhr4Y0wUlAuiCt6Z78yiUV6kVkYXK++tq50\\\\\\\/1TGaluqT\\\\\\\/YuRKjNm2FKX2W\\\\\\\/bF+51\\\\\\\/YQ38I\\\\\\\/\\\\\\\/YH5ZKpFUxssjwDBoGLGMHQeV5QJa7ff8RzHwMMJIfWoJ4KmQTApOEFLFvljYNCgNz0SMI4ejj2C1zkPjLcmnHqiCkKp6uCPZJYJKSTytUHo662jZT\\\\\\\/8GPAoaj42MQOGIoM4wvaAY12N5XcmJSAg3pNBfszBi+3NSyE9rSWcLoLtlNAJ+gSijBc\\\\\\\/liLCSbRx0H+KpFU7awGChDfWiqremOFnVV9lM4jBbNE2AgBFsjOcUHl1xA0+irbBgYRI4TjFu81lsQcFMV0p7OnimWEzmuFiqZK5Pb0kZnPnK3HZtbTrLj9g2BqliS9eiYeidhXi2VOr\\\\\\\/AgRMVrvJ31jzUd4wmaC2FkEcZdy\\\\\\\/ekl9mSl9FW2XDMh9GRye8ID8+a9l4PBJ\\\\\\\/sdvt1Fku8+7tUhk837n5tLki8bV68UBoDKsx1sF64InoA0BOTt4eOrt04JzmT1DLAr6STtXs7qm+fFcay2g3U5Wi9O9wG7GXssLKecFeObPiBXJ\\\\\\\/JZU62VU4j+O7tLNJw5ajDK6phy\\\\\\\/rlWLQrmwDfq1H7s55PdH6teq\\\\\\\/RAhMax9u8mBgFgCcI2YHbWZ39gXd6WmvDYDqlPklGghG51rZpG\\\\\\\/OvnuYMmalIhagKEOwbgrR1vcl3fbEHAFVuD6rJ0auTSdU5cqStA8VhJdXXF6fJ3mCTV1hv8fXcLXjpC1dUcjQyVpYU8lNaMGfSEr52977nNL012AxJfsLPTKCDdsyrRFN5RZkPjn3elZegqyST\\\\\\\/RjEklZFM82yzaLX1HbqZTWKhgDWaWjghrzPTzg1glZTTXG\\\\\\\/364qgC0kSAM86eMY2Tt4zCzcTYCcSnLpN1w1XhXpxnURcRiex9pmvWWDFSX8I5dLYelqV6fYoy8T3ySCdGXNKs6yTwr\\\\\\\/acnyqGYAWcwmEcwAqT36xIOhjQVSw4T1ReeIqRJANvGE\\\\\\\/pA1T0E3AYT4XZErjuYCBujFKg1IFR7QBYkF+RDQT6bneyqEuX+KHx8gG6gYYPhNnM0RJ5LTsAoz8z04dycJo+oos7cKDLZ3ldrBeiESZ2spH+fmgXyHc0g8z36sCiRazPJQvdud4fg0HYZasuzIToL\\\\\\\/ziWTHCsE7DsGphOgpIIdrmBSo5qed2oGYjn4kDqMeZJqEvPNy7riEW95HjKO8Xl25QxoACVVXdfIlWVF+0bHdF66+8RUq0sYLrGNxqe1kDlcKh\\\\\\\/R2LnECjOiJuMsT0EV0iin2xz92axCiVV+g5hTZZEmlTeuCWWGi9Ad6YXHJUF7Ddm08GzFL9miG0VuPOHQGpsF89hDBbVKFUHQkjN7GaPyInM17J3w5hoGUqj4\\\\\\\/kpn1Pnv3ViSqS25518KHHXDDgHQXDXMcVtzOaHuK7bANmc43WedBH\\\\\\\/Ryb1XifjojThyskEeJiHAHEueC7ZS+GgzXttvq65Y2pBpElAaIfOXXjaP\\\\\\\/+TZOqj0uN\\\\\\\/ldi0FJI8X8ylfbDWfCUzm+EaljrNPWDMfqcXxvyMmBcNS4swod8X7upTZ7brrDi6HNWw2oJkJc3q5Wf8aaLwRMbcphxqEsPfH1JDOF\\\\\\\/Z1AOjYZQrwWP0ThbZ3gB8XZqrLcCbGIpMkwTvM9eH2swkoqKEdOWgN+bQdQRB7BgtMU3Dw5KcJuMQL+COPC5qbyO2SXd8uRfY+ShIwOAStDksBGo9tR4mwkBLT8O0GoaOgARm\\\\\\\/sRfMt4pM5M+T42021rjCB0epWizCLkIBSEr0dmNbj+v0oHMe5kle4iXq7eANMNombutfPY3bZUd0BaPpRK58N4NPpWAAXoRO9x4WkGu0gsApNBhHKKSoP2RKC\\\\\\\/F0y0fCk2YnXIDnpbeyAmlnqv8kHau7ZU6OzJfGGigxV3zapaac2tSZddLtSQmR+fItavMdCR6sT5+wXFB2RppuXNOpyASy7l289Z74Zj8XmKCF9CzRpAZMZPMjk6rS4qQritjM\\\\\\\/0LV8d5LabGQDOuIjUkBTMuxj+u+ZRXmFsCGTqbvxqRhwaaoIqdYm8hQgrhUyPfSW5NbnBwjpAUEDCC8+tVF6luH8Lw6Aet56aQuIWKVPBGxiALBuWR4B3S4XkujrGQ7VJJKCsdlmGyj34gj0H69iRSwaDcvPjCdTPzfVHhF5QqOVDQLDfx4ZtQfzbwH4WaV43ROHa0iQtreUyHFTNf8Vr4tcHooD1QXGkbqWJfwe9CGmCm5ys1POFqpfvaUu75\\\\\\\/G7Zhsnb+BVXeBNL0GyqHo4WXBP00c2rUEuyu\\\\\\\/y4BQFlhgAfzx+95ssNwYRPiJFBsA5U141s9j5nPqdGUpwtVVlQqRQmTvyv7Q+9Q6qwV7ywztrV5fGzOcBA8hxmTNyupP+8mrzxdJJRsopiIYjiEpYTGQiDoQFEK99mdI2jAL6X+NcfDGdm9H6rgEaNN\\\\\\\/9uOL6KdH65+bC55S2tvFUN8Tt20+4QTf7Z+OD1nWPRT11NHfKuJlcFzE2BDpba9oIm8+jY9LEk5+Ih6R24l57u3oEuzU+1gdtpX9QhUlm3pO41r7rOIuspqTqJLrKV7N\\\\\\\/u0NeAq4C93p71Dc9CDamwZIjJ6eAwB5UVWgGtDyCo\\\\\\\/+MrDw+f62G9OC4\\\\\\\/Jlv7+04Lfc2XEBcHZvImaw1mn+Z553bE2PG7iccJ5+z5BaeN2PwoKl3FAla98RKZ4d2znS+Y1r7GMEQFCggY+gGorPF51bogBZDNT8booehaUdbQKrbDlwaqSsGhJcj1ldPZMYHFtZLfPiJUaDlzjrKKwbdAriS93yHCtbICbiCxGEGQYneIcdY+1CTG4QQXwL7yvqxLuXDHcY51ZJASHRZeIdla7lbgnJLpFhA49L1eaf\\\\\\\/6wKi2EEvpNgn2KbB6yneYd+MxzYp8oPDSPejojrVJOVAZDdtJLwIPq9skPMZUo0OHX7qIssact8C+sqA6u5Nu3celiKY9fN9FpNB7OGut2B5MqG6dUfOT2rX6qJWpIYeCB0q0odRVikYimDexpxkTcHfQMh3X9sMqd2hgQv9S7Pnu6v6oVJZSz+izJ8z99+XjljpQxtl\\\\\\\/1gqbUL4ptC2xuHWNeG2i2yB3svOBEzlL5rqdiCq860Mn7zHPYEC2mnER06sQ8y0mwTtAUMQ2vA\\\\\\\/9ck09tEjjr7w5hpHs+qjnNMvJpR0gYCFG2smzNmia\\\\\\\/iO\\\\\\\/9KUDyRNSbsH0GsnMo2wBti6kSWqkWzjJMcBKNSH\\\\\\\/e37Kc08Le1OM1UOtBkHQiL1QuXssBXAOma61zO9ei\\\\\\\/0esgJFDsQM4pGbDal8uzwxdDIJxgByD1m3EJD4z0TmmH85QatnuUWdcpQiBH0lt70YnMSQBEg00mTS7e0C9dsNWjXFywVo1m3dT4uo845u3plFC5KYVjMmfTbMl9KGGADCZ3fXGHj6XUD6nQtsYuE\\\\\\\/qFkBp2Vlu6Kqwmf6OOHnS4u30l0uX7EKJFvMKIQnwfIT16ve6\\\\\\\/JEafK\\\\\\\/1A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rd1wqXrLMywSig\\\\\\\/2zuha5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcbfcd8f221f5e4e5406168f0ea10edb4bb4e17a6cec82430a2f928f7950951a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BIE3tQ0pAbhWhUuPx0frtRjVfbwCnu6KSg2cGihxiMzWArQLKLAC0MmCKrsO2DltBnwhEXIbxPPIu3J45FJ4J36G\\\\\\\/q7FQIK3PxConR6c17\\\\\\\/JWqmx6+b7t\\\\\\\/Su4VYobWUdgZzpsk5iex1eNxg66qRhSvXB+GQq7OWUjqyV494xXKW3RZAIKH+xoPEsZwGyHcMffW1Xfoq3Am5CoAK1XhaM0KiQDfuH7jEfkZZLiUmK4nqvaEOiQ5F4ECHyPdeIhYa8IwGJyJMB9tdpeSMZ1lAJTlsqw6+hVDUVnr3qDE4iSUm9euSkOcmktTvKQ04zX\\\\\\\/eeElkl+CxF8zEXjia0QiuGMtkllNLqDN4Rd0My0RD7gE4PQRBaJ+LSV+2k\\\\\\\/\\\\\\\/GfUwGjXO26lI8FEqylAlBkwHzkRKR8qUqrtiSf\\\\\\\/Rsm7GHpAadrEGifLIhUo7JJaS+W5CgIx30OuBIfInUze\\\\\\\/6Xd9VEejBOsA\\\\\\\/e+oYTL34Z5BjXksJdlY99V\\\\\\\/5Le5cR5Au4CckRq9\\\\\\\/MDnelf2zJG4BFn10LGYcM7DAw8UwGMR7MepRP+ykZxOvk0AAGyV1u\\\\\\\/yvhDQttWZtpH+1UlkcmkiCLr+33z0Yh6BivCl7Tt0aq3ndDIXFpfCqID1o2KcMHxa5jP45NaEBdIpwPwDaazwJcPrONbm5lMawayTJFCwR8L\\\\\\\/5IKaZCzh\\\\\\\/B56Kwf+u27ueh6Vs7Laf\\\\\\\/VBIfuaZoFdXwp0L9Nn27+Y10z1mWVmnqsmyu41G50jYBOMwspEzckWGvwFmz5Vm\\\\\\\/qLACSNSvFRkwsBJxhnijG+NzFoRF9\\\\\\\/ygp6Q0B2ebqzxSYsGSRMpHBnA04d\\\\\\\/9VNqTsKDz2RIPseFM4UiUKo5JzvekujoJ9KNew4iVO+FwyTVE0XJWtLFZrbpYfBjqw+YrB\\\\\\\/yqvZ7\\\\\\\/fee+fo\\\\\\\/xIcWk28KABHKMOWAc85bDAWEv3XQTZ2HF\\\\\\\/xsC\\\\\\\/AmOg0i6ZrKnypXVj5SIWi4lj5gUjZiV2vRQcV4azRqdhP1GfPoEwylzR\\\\\\\/s+csC2HfdViNik788qm0EL1nVy0Us02uvGGXkYYdq97ohLJhvM2ZJ5ZTeriU259nRgsUkur6xeJNMRG+xqvXM9Act6PVz6GChhdxY7LAmC9ukH0Zsg+1DPgF4DO7WYNV8irHtDUSz6LYxKFkFhQ8D73AYWjtCxRakwQXMXXvnI1gqED92ILJjziP++n\\\\\\\/oTwqg6txHLh9yGdZzc5loHDoSbNyLQE9u7qzq5ChKu2Kzj9Cuek\\\\\\\/mPyOyjTXb5tpix5HuUUhkQWlypok95nqISU6n3IN2Ozs4bP2Z5biISWBhqhfa8FJEp6KtNUhLJr+ExCN4Y3yfeLt20MP\\\\\\\/treOpLJ2T4VpamchAl8NuLcbF5ak7wqq7qyty8mmA773QcGWKUUbjLSYWj8KppMYi75biVyEJdgOvU9B7BQEveWBbdQHpiVeFgh+YIMI41kjkR\\\\\\\/wqo5WQ3rRvT0Nm44fz9GPuk9hugZQbVLzdd3iOplbshEcuvR+el6pmwCOWOtr0l2n\\\\\\\/TJv3Ybnobn3K7wdABhJLDsbjrdANuyV7NfF9ngzuhDdzKxjXUWOkPymgiQYm8H5qgp84pRk4DW4MlWzJXnMDyuLqaKQJ2xLv2HNbRo4SfEvhqHbAsKDKBpCNoI9UDVwcgJzEou7HEqeNMxrQHHgGTz3ztiJMaRVwgdS8AwKplVB0qCDTzehLsYKbhgc2CorQ3o8uUg3uEnVjeYBw\\\\\\\/jxjPCqajfE76R16pcosgFcOzGOHJKCr3D62wFiyAnZeKls6kh0BzsD2xXQUFbwC\\\\\\\/KMVB98zB2J9ytWHDRzPhqrgHTHyyi0xFkZh4bl8ygn9Qnao1mOXDOS4mqy\\\\\\\/gQSDEKhuKQDbZ7oeJGJt87v2nsGVBlhDIZB0quce1UlshLDXYiGDdlPWgyb9sxGFYXOjgA+cBZuxVbGhXpGma+fDTMfEmUyXFX7rcV4CZGo2ATbhDMSAxSoIxJfNY1e6bMcBYsefJ+3yG9mTylFr\\\\\\\/wt6Mg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hxhEPKe9b\\\\\\\/mq0q5MLRDEEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022679e79ede2015eaa9cbd2c4673c5faef84875ce2cff3257f0f34b94a350d58c7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459397 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/1/119" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QUogRTZZ6xUBXfd5YIV8AXA_kn1UXbWW-a_PndxljQE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/1/119" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MlWvHZJ_vJjUt252njtHt4I-fh-LH8Xzm018YASQTyw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/1/109" title="Mapping Quantitative Trait Loci Affecting Arabidopsis thaliana Seed Morphology Features Extracted Computationally From Images" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/1/131" title="Compensatory Mutations in Predicted Metal Transporters Modulate Auxin Conjugate Responsiveness in Arabidopsis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459397" id="node459397" data-pisa="ggg;3/1/119" data-pisa-master="ggg;3/1/119" data-apath="/ggg/3/1/119.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systems-Level Analysis of Genome-Wide Association Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles R. Farber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">119-129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004788 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459397" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles R. Farber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Public Health Genomics, Departments of Medicine (Division of Cardiology) and Biochemistry and Molecular Genetics, University of Virginia, Charlottesville, Virginia 22908</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BR.%2BFarber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Farber%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BR.%2BFarber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:crf2s@virginia.edu" class="" data-icon-position="" data-hide-link-title="0">crf2s@virginia.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/1/119" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459397" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459397/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/1/119.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459397" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459397/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/1/119.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459397" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459397/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/1/119.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459397" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459397/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/1/119.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459397" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459397/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genome-wide association studies (GWAS) have emerged as the method of choice for identifying common variants affecting complex disease. In a GWAS, particular attention is placed, for obvious reasons, on single-nucleotide polymorphisms (SNPs) that exceed stringent genome-wide significance thresholds. However, it is expected that many SNPs with only nominal evidence of association (<em>e.g.</em>, <em>P</em> &lt; 0.05) truly influence disease. Efforts to extract additional biological information from entire GWAS datasets have primarily focused on pathway-enrichment analyses. However, these methods suffer from a number of limitations and typically fail to lead to testable hypotheses. To evaluate alternative approaches, we performed a systems-level analysis of GWAS data using weighted gene coexpression network analysis. A weighted gene coexpression network was generated for 1918 genes harboring SNPs that displayed nominal evidence of association (<em>P</em> ≤ 0.05) from a GWAS of bone mineral density (BMD) using microarray data on circulating monocytes isolated from individuals with extremely low or high BMD. Thirteen distinct gene modules were identified, each comprising coexpressed and highly interconnected GWAS genes. Through the characterization of module content and topology, we illustrate how network analysis can be used to discover disease-associated subnetworks and characterize novel interactions for genes with a known role in the regulation of BMD. In addition, we provide evidence that network metrics can be used as a prioritizing tool when selecting genes and SNPs for replication studies. Our results highlight the advantages of using systems-level strategies to add value to and inform GWAS.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-association-study-gwas" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study-gwas" rel="nofollow">genome-wide association study (GWAS)</a></li><li class="kwd"><a href="/keyword/systems-biology" class="hw-term hw-article-keyword hw-article-keyword-systems-biology" rel="nofollow">systems biology</a></li><li class="kwd"><a href="/keyword/coexpression-network" class="hw-term hw-article-keyword hw-article-keyword-coexpression-network" rel="nofollow">coexpression network</a></li><li class="kwd"><a href="/keyword/osteoporosis" class="hw-term hw-article-keyword hw-article-keyword-osteoporosis" rel="nofollow">osteoporosis</a></li></ul><p id="p-5">Genome-wide association studies (GWAS) have revolutionized complex disease genetics. In just the last few years, GWAS have been used to identify hundreds of variants affecting a diverse range of common diseases and disease associated quantitative traits (for a summary, see <a href="http://www.genome.gov/gwastudies/">http://www.genome.gov/gwastudies/</a>). Although GWAS have proven extremely effective at identifying common variants with relatively large effects, the first wave of data suggests that for many diseases, this class of variation accounts for only a small fraction of the genetic risk. For example, a large-scale, meta-analysis of ∼32,000 individuals identified 56 loci associated with bone mineral density (BMD), a strong predictor of osteoporotic fracture. However, in aggregate these single-nucleotide polymorphisms (SNPs) only explained 5.8% of the variance in femoral neck BMD (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Estrada <em>et al.</em> 2012</a>).</p><p id="p-6">It is possible that for most diseases, the missing heritability is attributable to a combination of many more common variants with increasingly smaller effect sizes and rare variants, both of which are difficult to detect with GWAS in its current form (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altshuler <em>et al.</em> 2008</a>). It has been suggested that additional genes and biological mechanisms underlying a disease process could be extracted from GWAS data by searching lists of genes harboring nominally significant (<em>e.g.</em>, <em>P</em> &lt; 0.05) associations. Most of the initial attempts to identify such pathways have used gene ontology (GO) and pathway-enrichment tools to compare the number of genes in a specific pathway harboring nominally significant SNPs to the number expected at random. This approach has been applied to several GWAS datasets with varying results (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Askland <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baranzini <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elbers <em>et al.</em> 2009a</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">O’Dushlaine <em>et al.</em> 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Peng <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ritchie 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Torkamani and Schork 2009</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Torkamani <em>et al.</em> 2008</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2007</a>).</p><p id="p-7">Several issues complicate pathway analysis. First, enrichment results can vary widely across software tools (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elbers <em>et al.</em> 2009b</a>). Second, enrichment analyses are biased toward what we already know concerning pathway membership, and most predefined gene categories are very general in nature, making it more difficult to develop testable hypotheses with the goal of investigating specific disease mechanisms. Third, these strategies fail to provide information on the relationships between associated genes. Such information is critical to understanding how networks of polymorphic genes work together to promote or provide protection against disease. Recently, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Baranzini <em>et al.</em> 2009</a> used protein−protein interaction data to address this latter point by identifying interacting partners that were nominally associated with multiple sclerosis. However, missing from this approach was the ability to incorporate network concepts with clinical information. The specific goal of this study was to address these issues.</p><p id="p-8">Weighted gene coexpression network analysis (WGCNA) is a widely used analytical method that identifies functional connections between genes using microarray gene expression data (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gargalovic <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ghazalpour <em>et al.</em> 2006</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Horvath <em>et al.</em> 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Oldham <em>et al.</em> 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">van Nas <em>et al.</em> 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Winden <em>et al.</em> 2009</a>). WGCNA groups genes into modules on the basis of their coexpression similarities across a population of samples. The resulting modules have been shown to be comprised of genes that share similar functions or are involved in the same pathway [as examples: (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Ghazalpour <em>et al.</em> 2006</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Horvath <em>et al.</em> 2006</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Oldham <em>et al.</em> 2008</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">van Nas <em>et al.</em> 2009</a>)]. The advantage of WGCNA is that connections between genes can be established in an unbiased manner using disease-relevant expression data.</p><p id="p-9">In the present work we used WGCNA to perform a systems-level analysis of GWAS data. The analysis was performed by combining SNP-level association data from a large BMD GWAS with microarray expression data from a disease-relevant cell type from subjects with known BMD status (low <em>vs.</em> high). Using WGCNA, we identified modules composed of genes that were highly interconnected with one another and displayed nominal evidence of association with BMD. Through the characterization of module content and topology, our approach identified biological mechanisms, modules, individual genes, and network concepts that likely play an important role in the regulation of BMD.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Converting SNP lists to gene lists using ProxyGeneLD</h3><p id="p-10">Several caveats complicate the conversion of a list of SNPs with association <em>P</em>-values to the assignment of gene-wide <em>P</em>-values using raw GWAS data. The primary confounders are linkage disequilibrium (LD) and biases due to gene size and the number of SNPs typed per gene. LD makes gene identification difficult because many nominally significant SNPs will be in LD with multiple genes. In addition, larger genes and genes with a greater density of SNPs typed have an increased probability of harboring nominally significant SNPs just by chance. Recently, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hong <em>et al.</em> 2009</a> developed an algorithm (referred to as ProxyGeneLD) that reduces biases by accounting for LD when annotating genes. ProxyGeneLD works by identifying clusters of GWAS SNPs (referred to as proxy clusters) in high LD (r2 ≤ 0.80) using HapMap data. It then assigns proxy clusters and singleton SNPs (that did not group within a proxy cluster) to the nearest gene. Unadjusted gene-wide <em>P</em>-values are then calculated as the minimum of any SNP, either as a singleton or member of a proxy cluster per gene. <em>P</em>-value adjustments are made by multiplying the unadjusted <em>P</em>-value by the number of SNPs assigned to that gene.</p><p id="p-11">We used precomputed <em>P</em>-values from a recently published GWAS performed by deCODE (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Styrkarsdottir <em>et al.</em> 2008</a>). These data are available for download from <a href="http://content.nejm.org/cgi/content/full/NEJMoa0801197/DC1">http://content.nejm.org/cgi/content/full/NEJMoa0801197/DC1</a> as individual text files. The GWAS consisted of 5,861 Icelandic subjects phenotyped for hip (HBMD) and spine (SBMD) BMD and genotyped at 301,019 SNPs (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Styrkarsdottir <em>et al.</em> 2008</a>). All SNPs for both traits were annotated using ProxyGeneLD. LD patterns were determined using CEU HapMap samples and genes were defined as the transcript plus a 1-kbp extension upstream to include promoter regions. <em>P</em>-values were assigned to a total of 16,878 genes. Genes with an adjusted <em>P</em> ≤ 0.05 for at least one of the two BMD traits were referred to as the nominally significant GWAS geneset (NSGG).</p></div><div id="sec-3" class="subsection"><h3>GO and pathway-enrichment analysis</h3><p id="p-12">We performed GO and pathway-enrichment analysis for the NSGG and network modules by using the Database for Annotation, Visualization and Integrated Discovery [DAVID (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dennis <em>et al.</em> 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huang da <em>et al.</em> 2009</a>)]. Each analysis was performed using the functional annotation charting and functional annotation clustering options. Functional annotation charting tests each individual GO or pathway term for enrichment. In contrast, functional annotation clustering combines single categories with a significant overlap in gene content and then assigns an enrichment score (ES; defined as the –log10 of the geometric mean of the <em>P</em>-values for each single term in the cluster) to each cluster, making interpretation of the results more straightforward. Functional annotation clustering cannot be performed for more than 3000 genes. Because the NSGG contained 3083 genes, we used to top 3000 ranked on adjusted <em>P</em>-value for the analysis. The search was limited to KEGG and Biocarta pathways, PFAM protein domains, and GO terms in the “Molecular Function,” “Biological Process,” and “Cellular Component” categories. Single categories were considered significantly enriched at a false discovery rate (FDR) ≤ 5%. To assess the significance of functional clusters, we created 10 sets of 3000 genes randomly selected from the aforementioned list of 16,878 genes with assigned <em>P</em>-values. Functional annotation clustering was performed for all 10 random gene sets. The max random ES was 2.75. Therefore, we used an ES cutoff of ≥3.0 as the threshold for significance in all analyses.</p></div><div id="sec-4" class="subsection"><h3>Gene expression data processing</h3><p id="p-13">To generate gene coexpression networks we used previous published microarray data from 26 healthy Chinese females ages 20−45 yr, with a mean age of 27.3 yr (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lei <em>et al.</em> 2009</a>). In this study expression profiles were generated from circulating monocytes that were isolated and purified from subjects with low (n = 12) and high (n = 14) BMD. We downloaded the Affymetrix CEL files from National Center for Biotechnology Information (NCBI)’s Gene Expression Omnibus (GSE7158; <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7158">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7158</a>). The raw data were imported and processed using the affy package (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gautier <em>et al.</em> 2004</a>) for the R Language and Environment for Statistical Computing (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ihaka and Gentleman 1996</a>). Robust multiarray algorithm was used to normalize and generate probe level expression data (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Irizarry <em>et al.</em> 2003</a>).</p></div><div id="sec-5" class="subsection"><h3>WGCNA</h3><p id="p-14">Network analysis was performed using the WGCNA R package (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Langfelder and Horvath 2008</a>). An extensive overview of WGCNA, including numerous tutorials, can be found at <a href="http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/">http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/</a>. To begin, we identified all probes assaying the expression of NSGG genes. To eliminate noise due to genes that were not expressed, we selected NSGG probes whose levels exceeded the median level of expression across the entire array. As part of our quality control, we performed a clustering and principal components analysis based on the expression of these probes. Two samples from the high BMD group, GSM172405 and GSM172418, were significant outliers and were removed from the analysis. A preliminary calculation of network connectivity was used to identify the most connected probe for each gene. A WGCNA network for the selected probes was generated exactly as described in (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Farber 2010</a>). GeneSignificance (GS) for the each network gene was defined as the absolute value of its Pearson correlation with BMD status. Module Membership (MM) was calculated as the Pearson correlation between each gene’s expression and its module eigengene, calculated using Singular Value Decomposition (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alter <em>et al.</em> 2000</a>). Network depictions were constructed using Cytoscape (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shannon <em>et al.</em> 2003</a>).</p></div><div id="sec-6" class="subsection"><h3><em>In silico</em> replication</h3><p id="p-15">To compare replication success rates in hubs and genes with the highest GWAS <em>P</em>-values, we used data from a second GWAS, the Framingham Osteoporosis Study [FOS (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kiel <em>et al.</em> 2007</a>)]. The FOS GWAS consisted of 1141 subjects genotyped at ∼100,000 SNPs. We downloaded the association data [in the form of SNPs and precomputed <em>P</em>-values generated using generalized estimating equation models (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kiel <em>et al.</em> 2007</a>)] for three BMD traits (femoral neck, lumbar spine, and trochanter) from the database of Genotype and Phenotype at NCBI (<a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gap">http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gap</a>). SNP lists for each of the three traits were converted to gene lists using ProxyGeneLD precisely as described previously. A gene was considered successfully replicated if it had an unadjusted <em>P</em> ≤ 0.05 for at least one of the three BMD traits. The percentage of successfully replicated genes was calculated in the blue, magenta, greenyellow, and brown modules for the top 20%, 10%, and 5% of genes based on intramodular connectivity (k.in). These rates were compared with those for the top 20%, 10%, and 5% of GWAS network genes selected based on adjusted <em>P</em>-value from the deCODE (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Styrkarsdottir <em>et al.</em> 2008</a>) GWAS or GS.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Identifying genes with nominally significant genome-wide associations</h3><p id="p-16">An overview of the systems-level analysis of GWAS data are presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The first step in the analysis was the identification of genes displaying evidence of association using data from a BMD GWAS [n = 5861 (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Styrkarsdottir <em>et al.</em> 2008</a>)]. We used the ProxyGeneLD algorithm (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hong <em>et al.</em> 2009</a>), which takes LD patterns into account when assigning SNPs to genes and adjusts for gene length and SNP density biases (see <em>Materials and Methods</em>), to generate gene-wide adjusted <em>P</em>-values for two osteoporosis-related traits, HBMD and SBMD. Gene-wide <em>P</em>-values were calculated for a total of 16,878 genes. Of these, 1777 and 1861 had gene-wide adjusted <em>P</em> ≤ 0.05 for HBMD and SBMD, respectively. By combining the two lists, 3083 unique genes were identified with adjusted <em>P</em> ≤ 0.05 for at least one of the BMD traits. We refer to these genes as NSGG.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/119/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the systems-level analysis of GWAS data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423719514" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the systems-level analysis of GWAS data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/119/F1.medium.gif" width="344" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/119/F1.medium.gif" width="344" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/119/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/119/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462285" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-17" class="first-child">Overview of the systems-level analysis of GWAS data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">To determine whether gene length and SNP density were potential confounders in the NSGG, we calculated the correlation between these two variables and HBMD unadjusted (defined as the minimum <em>P</em>-value for proxy clusters and single SNPs assigned to a particular gene) and adjusted <em>P</em>-values. As described previously, 1777 genes had adjusted <em>P</em> ≤ 0.05. In contrast, 5228 genes had unadjusted <em>P</em> ≤ 0.05. In the latter gene set, we observed a strong correlation between unadjusted <em>P</em> and gene length (r = 0.46, <em>P</em> = 0) and SNP density (r = 0.50, <em>P</em> = 0). However, this correlation was not observed after adjustment for gene length (r=-0.01, <em>P</em> = 0.88) or SNP density (r = −0.01, <em>P</em> = 0.74). Thus, our network analysis of GWAS genes should not be influenced by these systematic biases.</p></div><div id="sec-9" class="subsection"><h3>Conventional pathway enrichment fails to pinpoint specific biological mechanisms</h3><p id="p-19">We next determined whether the NSGG was enriched for “biological themes” using the conventional approach of GO and pathway enrichment analysis. DAVID (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dennis <em>et al.</em> 2003</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Huang da <em>et al.</em> 2009</a>) was used for this analysis, although we also used WebGestalt (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em> 2005</a>) and observed similar results. A total of 24 individual terms, all of which were GO categories, were significantly enriched in the NSGG at an FDR ≤ 5% (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1/FileS1.xls">File S1</a>). The most significant term was protein binding (GO:0005515; FDR = 1.7 × 10<sup>−10</sup>). Other significant categories included developmental process (GO:0032502; FDR = 9.5 × 10<sup>−5</sup>), cation binding (GO:0043169; FDR = 2.5 × 10<sup>−3</sup>), and cell differentiation (GO:0030154; FDR = 2.7 × 10<sup>−2</sup>).</p><p id="p-20">DAVID also generates category clusters by condensing sets of related terms (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Dennis <em>et al.</em> 2003</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Huang da <em>et al.</em> 2009</a>). This condenses redundant categories, identifies terms containing a smaller number of genes that on their own would require higher fold enrichments to reach statistical significance, and makes interpreting the results much easier. Each cluster receives an ES, which is defined as the geometric mean (on a –log10 scale) of the <em>P</em>-values for all single terms in the cluster. A total of 32 clusters had ESs &gt; 1.3 (equivalent to a nominal <em>P</em> ≤ 0.05); however, it was unclear whether this was an appropriate significance cutoff. To determine the distribution of ESs observed using a set of random genes we created 10 sets of 3000 genes randomly selected from the whole genome and ran each through DAVID. ESs for the random gene sets ranged from 1.36 to 2.75. Therefore, we selected an ES cutoff of ≥3.0. Using this threshold, a total of five significant clusters were identified in the NSGG (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1/FileS2.xls">File S2</a>). The top GO terms in each of the five clusters were “intracellular part,” “metal ion binding,” “developmental process,” “intracellular organelle part,” and “organelle inner membrane.” These data indicate that the NSGG is enriched for groups of genes sharing similar functionality; however, because the identified categories are very general in nature this analysis does little to pinpoint specific biological mechanisms underlying variation in BMD.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462341/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462341/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Gene category and pathway enrichment analysis of NSGG genes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Generation of a weighted gene coexpression network for NSGG genes</h3><p id="p-22">WGCNA reveals connections between genes using microarray expression data by grouping genes based on a topological overlap measure [TOM (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dong and Horvath 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhang and Horvath 2005</a>)]. Two genes have a high TOM if they are highly interconnected with the same set of genes (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dong and Horvath 2007</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zhang and Horvath 2005</a>). To evaluate the coexpression relationships between NSGG genes in a disease-relevant context we used microarray expression profiles of purified circulating monocytes isolated from individuals with discordant levels of BMD (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lei <em>et al.</em> 2009</a>). The dataset included 24 profiles from young (mean age = 27.3 years) Chinese females, 12 with low BMD (mean Z-score=-1.72) and 12 with high BMD (mean Z-score = 1.57). We choose to use this dataset because it represents the largest study performed to date with both expression profiles for a cell-type relevant to BMD [monocytes are precursors to bone-resorbing osteoclasts (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fujikawa <em>et al.</em> 1996</a>)] and clinical information on the subjects. After excluding non- and lowly expressed genes we identified probes representing 1918 (62%) of the 3083 NSGG genes and applied the WGCNA algorithm to generate a GWAS network. The resulting network was composed of 13 distinct gene modules (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Sixty-three of the genes failed to fit within a distinct group and were assigned to the “grey” module. The modules ranged in size from 40 (salmon module) to 356 genes (turquoise module). A complete list of module assignments and network metrics for all genes is included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1/FileS3.xls">File S3</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/119/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="WGCNA coexpression network composed of BMD GWAS genes. Shown is the hierarchical clustering dendogram for all 1918 genes used in the analysis. Each line is an individual gene. Genes were clustered based on a dissimilarity measure (1 &#x2212; TOM). The branches correspond to modules of highly interconnected groups of genes. The tips of the branches represent genes that are the least dissimilar and thus share the most similar network connections. Below the dendogram each gene is color coded to indicate its module assignment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423719514" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;WGCNA coexpression network composed of BMD GWAS genes. Shown is the hierarchical clustering dendogram for all 1918 genes used in the analysis. Each line is an individual gene. Genes were clustered based on a dissimilarity measure (1 &#x2212; TOM). The branches correspond to modules of highly interconnected groups of genes. The tips of the branches represent genes that are the least dissimilar and thus share the most similar network connections. Below the dendogram each gene is color coded to indicate its module assignment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/119/F2.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/119/F2.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/119/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/119/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462350" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-23" class="first-child">WGCNA coexpression network composed of BMD GWAS genes. Shown is the hierarchical clustering dendogram for all 1918 genes used in the analysis. Each line is an individual gene. Genes were clustered based on a dissimilarity measure (1 − TOM). The branches correspond to modules of highly interconnected groups of genes. The tips of the branches represent genes that are the least dissimilar and thus share the most similar network connections. Below the dendogram each gene is color coded to indicate its module assignment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The WGCNA approach has been used to generate robust networks in several diverse applications (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gargalovic <em>et al.</em> 2006</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Ghazalpour <em>et al.</em> 2006</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Horvath <em>et al.</em> 2006</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Oldham <em>et al.</em> 2008</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">van Nas <em>et al.</em> 2009</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Winden <em>et al.</em> 2009</a>), including experiments with a similar or smaller number of samples relative to this study (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gargalovic <em>et al.</em> 2006</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gong <em>et al.</em> 2007</a>). Most WGCNA analyses, however, use a series of preliminary filtering steps to select the most biologically meaningful genes for network construction (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Ghazalpour <em>et al.</em> 2006</a>). In such studies, the expression data exclusively determines which genes are used in the analysis. Because our network genes were not selected entirely based on expression profiles, we wanted to ensure that the resulting modules were cohesive and robust. To test cohesiveness, we calculated the mean MM for each module. MM is the correlation between each gene in a module and its module eigengene. Thus, it is a measure of how tightly a particular gene fits into its module. The greater the mean MM for a module, the more similar the coexpression relationships are across the module. The mean MM ± SEM ranged from 0.60 ± 0.01 (brown module) to 0.74 ± 0.01 (tan module), indicating that modules consisted of genes sharing highly similar expression patterns. We addressed robustness, as described previously (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Ghazalpour <em>et al.</em> 2006</a>), by randomly splitting the dataset in half 1000 times and calculating k.in in each half. The analysis was performed for the largest (turquoise) and smallest (salmon) modules. The mean correlation ± SEM between the real and random k.in values was 0.65 ± 0.05 and 0.52 ± 0.03 in the turquoise and salmon modules, respectively. Thus, the GWAS network modules are cohesive and robust to exclusion of half the data.</p></div><div id="sec-11" class="subsection"><h3>Characterization of module content reveals a key role for oxidative phosphorylation in the regulation of BMD</h3><p id="p-25">One way in which network analysis can inform GWAS is to expose pathway enrichments that were not observed in a large set of nominally significant genes, such as the NSGG. We expected that by parsing genes based on coexpression similarities, more refined functions would be condensed within modules, revealing enrichments for more specific processes. This would improve the process of converting a detectable enrichment into a testable hypothesis.</p><p id="p-26">To determine whether specific modules were enriched for novel gene categories or pathways we repeated the DAVID analysis for each module. Of the 13 modules, five had at least one cluster with an ES ≥ 3.0. Interestingly, the turquoise module stood out as displaying detailed enrichments that were not observed in the analysis of the entire NSGG (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1/FileS4.xls">File S4</a>). In the turquoise module, significant enrichments were observed for six clusters with the following top terms “cytoplasmic part” (ES = 8.1), “mitochondrion” (ES = 7.3), “electron carrier activity” (ES = 4.9), “electron carrier activity” (ES = 4.2), “hydro-lyase activity” (ES = 3.9), and “RNA splicing” (ES = 3.5). Within each cluster there were a number of terms that were not significant in the entire NSGG, suggesting that partitioning genes into coexpression can reveal hidden enrichments.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462346/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462346/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Network modules with significant DAVID enrichments</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">To investigate the enrichments in more detail, we focused on a single enriched term in cluster 2, the KEGG pathway “oxidative phosphorylation” (oxphos), because it represented one of the most specific enriched terms. This single term was not enriched in the NSGG (FDR = 99.8); however, its enrichment in the turquoise module was significant (FDR = 1.1 × 10<sup>−3</sup>). Of the 356 turquoise module genes, 16 (4.5%) were involved in oxphos (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). To determine whether this enrichment was specific to the GWAS network, we generated 100 random networks. Each network was created by selecting 3083 genes at random using the same gene filtering steps and network parameters used to construct the real network. A total of 114 of the 20,080 genes (0.6%) with unique gene identifiers on the array belonged to the KEGG oxphos pathway. As shown above 16 of the 356 turquoise (4.5%) module genes were involved in oxphos. Using a Fisher’s Exact test this enrichment is highly significant (4.5% <em>vs.</em> 0.6%; <em>P</em> = 1.8 × 10<sup>−9</sup>). We then performed this same test for each of 1709 modules belonging to the 100 random networks. None of the random module enrichment <em>P</em>-values exceeded the <em>P</em>-value for the real turquoise module, indicating that this enrichment is specific to the BMD GWAS network.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462364/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462364/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Members of the turquoise module involved in oxidative phosphorylation</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Oxphos genes were also among the most connected in both the turquoise module and the whole network (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In fact, the three most connected turquoise hubs were oxphos genes. In addition, of the 16 total genes, 15 were in the top 20% of genes when ranked on k.in (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Another observation was that the expression of all 15 highly connected oxphos genes was negatively correlated with BMD status (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Thus, by exploring the content of the turquoise module, we have identified an association between genetic variation in oxphos genes and BMD, determined that oxphos genes are module and network hubs, and determined that oxphos gene expression in monocytes was inversely correlated with BMD levels.</p></div><div id="sec-12" class="subsection"><h3>Discovery of a turquoise submodule highly correlated with BMD status</h3><p id="p-33">In addition to content, module topology (the unique distribution of edges among nodes) can also be evaluated in WGCNA networks. We investigated turquoise module topology by generating a network view showing all edges with a TOM ≥ 0.15 and their corresponding nodes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The network consisted of 88 nodes and 256 edges. An initial inspection indicated that most nodes were grouped into a central core (containing many of the oxphos genes identified previously in this article) with two small submodules radiating from <em>COX5B</em>, an oxphos gene and the second most connected node in the module. We then overlaid information regarding the correlation between each gene’s expression and BMD status in the monocyte expression study. We suggest that correlation is a meaningful measure of biological significance, especially when considering GWAS genes, because it is likely that the correlations reflect subtle genetically-regulated differences in expression that are associated with alterations in BMD. As shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> most of the genes were either not correlated (nodes shaded white) or slightly negatively correlated with BMD (nodes shaded light green). None of the genes were significantly positively correlated (max correlation in the turquoise module is 0.10). Interestingly, the genes in one of the submodules were among the most negatively correlated (shaded dark green) in the turquoise module and the entire network (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). One of the submodule genes, <em>IFI35</em>, was the second most negatively correlated (r = −0.58, <em>P</em> = 2.7 × 10<sup>−3</sup>) with BMD in the NSGG network and 4 of the 8 genes in the sub-module were in the top 50. The average correlation for this group was -0.42. To determine the probability of randomly observing a group of 8 genes this negatively correlated (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>) we created 10<sup>6</sup> sets of 8 genes selected at random from the turquoise module. Of the random gene sets none had an average correlation more extreme than this turquoise sub-module (most negative r = −0.36).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/119/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Network view of the turquoise module reveals a submodule of genes negatively correlated with BMD status. This network contains all turquoise module edges with TOM &#x2265; 0.15 and their corresponding nodes. Genes are shaded based on their correlation with BMD from white (no correlation) to dark green (strong negative correlation). Node sizes are proportional to each gene&#x2019;s &#x2013;log10 GWAS P (most significant unadjusted GWAS P-value for either HBMD or SBMD). The submodule of interest is on the right-hand side of the figure. Notice that this group of gene is highly interconnected and negatively correlated with BMD status." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423719514" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Network view of the turquoise module reveals a submodule of genes negatively correlated with BMD status. This network contains all turquoise module edges with TOM &#x2265; 0.15 and their corresponding nodes. Genes are shaded based on their correlation with BMD from white (no correlation) to dark green (strong negative correlation). Node sizes are proportional to each gene&#x2019;s &#x2013;log10 GWAS P (most significant unadjusted GWAS &lt;em&gt;P&lt;/em&gt;-value for either HBMD or SBMD). The submodule of interest is on the right-hand side of the figure. Notice that this group of gene is highly interconnected and negatively correlated with BMD status.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/119/F3.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/119/F3.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/119/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/119/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462242" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-34" class="first-child">Network view of the turquoise module reveals a submodule of genes negatively correlated with BMD status. This network contains all turquoise module edges with TOM ≥ 0.15 and their corresponding nodes. Genes are shaded based on their correlation with BMD from white (no correlation) to dark green (strong negative correlation). Node sizes are proportional to each gene’s –log10 GWAS P (most significant unadjusted GWAS <em>P</em>-value for either HBMD or SBMD). The submodule of interest is on the right-hand side of the figure. Notice that this group of gene is highly interconnected and negatively correlated with BMD status.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462253/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462253/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Genes comprising the turquoise sub-module</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">Using gene information and literature searches, we found no obvious functional connection between the genes that comprised this subnetwork. However, using expression data from a panel of mouse tissues [<a href="http://www.biogps.org">http://www.biogps.org</a> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lattin <em>et al.</em> 2008</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Su <em>et al.</em> 2002</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2004</a>)] we did observe that six of the genes are expressed in osteoclasts (<em>EPSTl1</em>, <em>IFI35</em>, <em>PARP12</em>, <em>CMPK2</em>, <em>ZCCHC2</em>, and <em>TAP1</em>) and the other two are expressed in osteoblasts (<em>LOC26010</em> and <em>LYSMD2</em>). The group of osteoclast genes is also the most negatively correlated with BMD (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Next, we determined whether any of the eight genes were located in close proximity to suggestive or significant GWAS loci (<em>P</em> &lt; 1.0 × 10<sup>−5</sup>) identified in a recent meta-analysis of BMD (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Estrada <em>et al.</em> 2012</a>). Interestingly of the eight, the transcription start site for four (<em>EPSTl1</em>, <em>IFI35</em>, <em>ZCCH2</em>, and <em>LYSMD2</em>) are less than 750 Kbp away from a GWAS association (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). Therefore, these genes represent a highly interconnected sub-module whose expression is negatively correlated with BMD. These data together suggest they play a role in the regulation of BMD. Again, as demonstrated above, the functional interconnections between genes in this sub-module, and its correlation with BMD, was only revealed by network analysis.</p></div><div id="sec-13" class="subsection"><h3>Identifying functional connections between known and novel genes</h3><p id="p-39">One of the advantages of our approach is the ability to identify connections between novel genes with evidence of association and those with a previously established role in disease. This information can be used in two ways. First, it can identify new pathways that a known gene may participate in and second, it can identify novel genes through “guilty by association.” To investigate the network connections for a known gene we focused on tumor necrosis factor (TNF), the most highly connected gene in the NSGG network with a known role in BMD. <em>TNF</em> was the 13th most connected gene in the entire network with a total network connectivity (k.total) of 29.0 (max k.total = 35.2). It was the 6th most connected gene in the blue module with a k.in = 27.6 (max blue module k.in = 30.8). <em>TNF</em> is known to play a prominent role in osteoclastogenesis (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lam <em>et al.</em> 2000</a>) and several studies have found associations between <em>TNF</em> polymorphisms and BMD (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fontova <em>et al.</em> 2002</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kim <em>et al.</em> 2009</a>). In the deCODE GWAS it was associated with HBMD and SBMD with unadjusted <em>P</em>-values of 1.2 × 10<sup>−3</sup> and 1.6 × 10<sup>−2</sup>, respectively. The fact that <em>TNF</em> is one of hubs of a monocyte network provides additional support for the biological relevance of the GWAS network.</p><p id="p-40">We created a <em>TNF</em> submodule by identifying all edges within the blue module involving <em>TNF</em> with a TOM ≥ 0.15. The submodule contained 99 genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Using DAVID we identified three significant clusters that were enriched in the sub-module with terms related to “nuclear proteins” (ES = 4.0), “gene expression” (ES = 3.6), and “regulation of transcription” (ES = 3.0) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1/FileS5.xls">File S5</a>). Of the 99 genes, 47 belonged to the GO cellular component category “nucleus” (FDR = 3.82 × 10<sup>−6</sup>, 1.9 fold enrichment), and 32 were in the GO molecular function category “transcription factor activity” (FDR = 1.8 × 10<sup>−4</sup>, 3.5 fold enrichment). In support of its disease relevance the submodule included several genes with known roles in bone metabolism, such as nuclear receptor subfamily 3, group C, member 1 (glucocorticoid receptor; <em>NR3C1</em>); protein tyrosine phosphatase, receptor type, E (PTPRE); <em>CD44</em> molecule (Indian blood group; <em>CD44</em>); NLR family, pyrin domain containing 3 (<em>NLRP3</em>); FBJ murine osteosarcoma viral oncogene homolog B (<em>FOSB</em>); and dual-specificity phosphatase 6 (<em>DUSP6</em>). Thus, our network analysis rediscovered <em>TNF</em> as key intracellular signaling “hub” gene important in bone metabolism. More importantly, this network can be mined in future studies to identify novel genes that interact with <em>TNF</em> in some way (<em>e.g.</em>, are downstream targets of <em>TNF</em> signaling, etc.) to affect bone mass.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/119/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterizing the coexpression relationships for a highly connected known BMD gene. This TNF centered network provides a view of all edges and their corresponding nodes connected to TNF with a TOM &#x2265; 0.15. Genes are color coded based on their correlation with BMD; white (&#x2212;0.20 &lt; r" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423719514" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Characterizing the coexpression relationships for a highly connected known BMD gene. This TNF centered network provides a view of all edges and their corresponding nodes connected to TNF with a TOM &#x2265; 0.15. Genes are color coded based on their correlation with BMD; white (&#x2212;0.20 &lt; r&lt;0.20), blue (r &#x2265; 0.20), and yellow (r&#x2264;-0.20). Node sizes are proportional to each gene&#x2019;s &#x2013;log10 GWAS P (most significant unadjusted GWAS &lt;em&gt;P&lt;/em&gt;-value for either HBMD or SBMD).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/119/F4.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/119/F4.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/119/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/119/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462272" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-41" class="first-child">Characterizing the coexpression relationships for a highly connected known BMD gene. This TNF centered network provides a view of all edges and their corresponding nodes connected to TNF with a TOM ≥ 0.15. Genes are color coded based on their correlation with BMD; white (−0.20 &lt; r&lt;0.20), blue (r ≥ 0.20), and yellow (r≤-0.20). Node sizes are proportional to each gene’s –log10 GWAS P (most significant unadjusted GWAS <em>P</em>-value for either HBMD or SBMD).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Relating network concepts to measures of biological relevance</h3><p id="p-42">Exploring GWAS genes in the context of an expression network also allows one to relate network concepts, such as MM, to a measure of biological relevance. If a network property, inherent to a specific module, is associated with disease this suggests that the module serves an important biological role. It may also be possible to use the property as a gene screening tool to select genes for downstream studies.</p><p id="p-43">We focused on the association between the network concept MM and GS, a measure of biological relevance. GS was defined as the absolute value of the correlation between a gene’s expression and BMD status. Of the 13 network modules, significant (<em>P</em> &lt; 0.003 after adjusting for number of modules) positive correlations were observed between MM and GS in the magenta (r = 0.44, <em>P</em> = 9.9 × 10<sup>−5</sup>), greenyellow (r = 0.66, <em>P</em> = 1.6 × 10<sup>−10</sup>), and brown (r = 0.36, <em>P</em> = 1.9 × 10<sup>−7</sup>) modules (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/119/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between MM and GS for each of the 13 distinct GWAS modules. MM (defined as the correlation between each gene&#x2019;s expression and its module eigengene) for each module is plotted against GS (defined as each gene&#x2019;s correlation with BMD status). MM in the blue, magenta, greenyellow and brown modules is significantly (P &lt; 0.003) correlated with GS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423719514" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlation between MM and GS for each of the 13 distinct GWAS modules. MM (defined as the correlation between each gene&#x2019;s expression and its module eigengene) for each module is plotted against GS (defined as each gene&#x2019;s correlation with BMD status). MM in the blue, magenta, greenyellow and brown modules is significantly (&lt;em&gt;P&lt;/em&gt; &lt; 0.003) correlated with GS.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/119/F5.medium.gif" width="426" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/119/F5.medium.gif" width="426" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/119/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/119/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462302" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-44" class="first-child">Correlation between MM and GS for each of the 13 distinct GWAS modules. MM (defined as the correlation between each gene’s expression and its module eigengene) for each module is plotted against GS (defined as each gene’s correlation with BMD status). MM in the blue, magenta, greenyellow and brown modules is significantly (<em>P</em> &lt; 0.003) correlated with GS.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">On the basis of the correlations between MM and GS, we hypothesized that hub genes from these three modules were the most biologically relevant and thus, the most likely to represent true positive associations with BMD. If true this suggests that selecting genes based on MM may result in greater replication success rates in subsequent studies compared with selecting genes using the traditional metric, GWAS <em>P</em>-value. To test this we performed an <em>in silico</em> replication study using data from a second BMD GWAS [FOS (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kiel <em>et al.</em> 2007</a>)]. Of the 1918 total network genes, 1264 were annotated in FOS using ProxyGeneLD. Genes were considered successfully replicated if their gene-wide associations were less than the significance thresholds defined below with any one of three BMD traits (femoral neck, lumbar spine, and trochanter BMD). From the 1264 network genes annotated in both studies, we compared the FOS replication rates for three groups of genes: (1) hub genes (based on k.in) from the magenta, greenyellow, and brown modules; (2) network genes ranked on GS; and (3) network genes ranked on <em>P</em>-value in the deCODE GWAS. The replication rates were compared for the top 20%, 10%, and 5% of genes within each group at three different significant levels, <em>P</em> ≤ 0.05, <em>P</em> ≤ 0.01, and <em>P</em> ≤ 0.001. As shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, selecting genes on K.in resulted in greater replication rates in all comparisons. The difference in replication rate between K.in and GWAS <em>P</em>-value increased as the definition of a hub gene became more stringent. For example, when comparing the top 5% of hubs <em>vs.</em> the top 5% of genes based on <em>P</em>-value, the difference in replication rate was twofold higher for hubs. Although validation studies will be needed, these data suggest that k.in may be a better metric than GWAS <em>P</em>-value to use to select genes for subsequent replication studies.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462262/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462262/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Replication rates of network genes selected using intramodular connectivity (k.in), gene significance (GS), or <em>P</em>-value</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-47">In this study, we have applied network theory to a list of genes with evidence of association with BMD using disease-relevant microarray gene expression data in subjects with known BMD status. We demonstrate that network analysis can group genes into modules that are enriched for specific biological processes. In some cases the enrichments were unique to modules and were more detailed and specific than those identified in the entire gene set. We also show that module topology can be used to identify groups of interconnected genes strongly associated with a clinical trait. Not only can this approach be used to reveal hidden enrichments, but it can also identify potentially important coexpression relationships for genes that exceed genome-wide significant thresholds or that have been previously associated with the disease. We also demonstrate that for three of the modules there was a significant correlation between MM and GS. We go on to provide evidence suggesting that hub genes replicate at a higher rate relative to genes selected using GWAS <em>P</em>-value or GS. This study provides a framework for combing network analysis and gene expression data to extract additional biological information from GWAS data.</p><p id="p-48">One of the limitations of GWAS is that it does not provide functional information for associated genes. Our systems-level approach does so by grouping genes using expression data from a cell type or tissue that is relevant to the disease in subjects with clinical data. Our discovery of the turquoise submodule of eight genes negatively correlated with BMD is a good example. Importantly, the interconnections between genes in this group could only have been identified by studying their relationships in a disease context. This information combined with the knowledge that they are expressed in mouse osteoclasts can be used to guide <em>in vitro</em> and <em>in vivo</em> experiments to validate their role in bone.</p><p id="p-49">The major bottleneck in any analysis using GWAS data are generating gene lists. Because of the nature of GWAS data, many SNPs with nominally significant <em>P</em>-values will be false-positives. This coupled with the difficulties in converting SNP-based to gene-based <em>P</em>-values leads to gene lists that contain a considerable level of noise. What is clear from this study and others (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hong <em>et al.</em> 2009</a>) is that potential biases have to be taken into consideration. In addition, our data suggest that functional grouping using coexpression similarities is an excellent approach to separate noise from real biological signal. We have proven this by identifying that the inherent network concept MM is correlated with GS in three of the 13 modules.</p><p id="p-50">The main purpose of any analysis designed to mine GWAS data are the generation of testable hypotheses. We believe a systems-level approach offers many advantages over other strategies for this purpose. For example, we demonstrate that parsing GWAS gene lists into functional groups identified a key role for oxidative phosphorylation, which can now be experimentally validated. Additionally, we identified novel genes based on their connection to known bone genes, membership in an enriched pathway or connectivity in one of the modules in which MM was correlated with BMD. Such genes can be tested to validate their associations and to investigate their biological role in functional genomics and replication studies.</p><p id="p-51">Oxidative stress is known to be increased in age-related diseases such as osteoporosis. It is also known that oxphos plays a direct and key role in bone metabolism (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bratic and Trifunovic 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kousteni 2011</a>). In bone modeling and remodeling, osteoclasts resorb mineral by acidifying the bone matrix (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blair 1998</a>). This process requires significant energetic resources, which are primarily generated through the oxidative phosphorylation of glucose (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Williams <em>et al.</em> 1997</a>). Recently, it has been demonstrated that increased oxidative phosphorylation occurs in osteoclast precursors as they differentiate into mature osteoclasts (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2007</a>). Importantly, our data suggest that genetic variation in multiple oxphos genes influence bone mass. Moreover, the expression of these genes in monocytes is inversely correlated with bone mass, suggesting that increased oxphos in monocytes/osteoclasts results in decreased bone mass.</p><p id="p-52">Our analysis focused on osteoporosis; however, it is likely applicable to any disease with GWAS data and the appropriate gene expression profiles. GWASs have been performed for a myriad of disease. As an example, our search of the Gene Expression Omnibus database at NCBI using the term “cancer” resulted in 344 datasets, suggesting that for many diseases relevant gene expression data that can be used for network analysis is already available.</p><p id="p-53">In conclusion, this study provides proof-of-principle that a systems-level analysis of GWAS data is capable of adding significant value to existing datasets and future studies. This analysis provides a straightforward approach to identify pathways, individual genes, gene modules and network concepts that play an important role in disease.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We thank Jake Lusis and Steve Horvath at UCLA for insightful comments. This work was supported in part by National Institutes of Health/National Institute of Arthritis and Musculoskeletal and Skin Diseases R01 AR057759.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004788/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Gene expression microarray data in this article have been submitted to the GEO database at NCBI as series GSE7158. Summarized (<em>P</em>-values) genome-wide association data are available at <a href="http://content.nejm.org/cgi/content/full/NEJMoa0801197/DC1">http://content.nejm.org/cgi/content/full/NEJMoa0801197/DC1</a>.</p></li><li class="fn" id="fn-12"><p id="p-55"><em>Communicating editor: O. Troyanskaya</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-10-24"><span class="received-label">Received </span>October 24, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-11-20"><span class="accepted-label">Accepted </span>November 20, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Farber</li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.1" data-doi="10.1073/pnas.97.18.10101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alter</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Singular value decomposition for genome-wide expression data processing and modeling</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">10101</span>–<span class="cit-lpage">10106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAlter%26rft.auinit1%253DO.%26rft.volume%253D97%26rft.issue%253D18%26rft.spage%253D10101%26rft.epage%253D10106%26rft.atitle%253DSingular%2Bvalue%2Bdecomposition%2Bfor%2Bgenome-wide%2Bexpression%2Bdata%2Bprocessing%2Band%2Bmodeling%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.18.10101%26rft_id%253Dinfo%253Apmid%252F10963673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMTgvMTAxMDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.2" data-doi="10.1126/science.1156409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic mapping in human disease</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D322%26rft.issue%253D5903%26rft.spage%253D881%26rft.epage%253D888%26rft.atitle%253DGenetic%2BMapping%2Bin%2BHuman%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1156409%26rft_id%253Dinfo%253Apmid%252F18988837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMy84ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.3" data-doi="10.1007/s00439-008-0600-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pathways-based analyses of whole-genome association study data in bipolar disorder reveal genes mediating ion channel activity and synaptic neurotransmission</span>. <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DAskland%26rft.auinit1%253DK.%26rft.volume%253D125%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D79%26rft.atitle%253DPathways-based%2Banalyses%2Bof%2Bwhole-genome%2Bassociation%2Bstudy%2Bdata%2Bin%2Bbipolar%2Bdisorder%2Breveal%2Bgenes%2Bmediating%2Bion%2Bchannel%2Bactivity%2Band%2Bsynaptic%2Bneurotransmission.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-008-0600-y%26rft_id%253Dinfo%253Apmid%252F19052778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-008-0600-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19052778&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262567400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.4" data-doi="10.1093/hmg/ddp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baranzini</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galwey</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khankhanian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pathway and network-based analysis of genome-wide association studies in multiple sclerosis</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp120%26rft_id%253Dinfo%253Apmid%252F19286671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8yMDc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.5" data-doi="10.1002/(SICI)1521-1878(199810)20:10&lt;837::AID-BIES9&gt;3.0.CO;2-D"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">H. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">How the osteoclast degrades bone</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DBlair%26rft.auinit1%253DH.%2BC.%26rft.volume%253D20%26rft.issue%253D10%26rft.spage%253D837%26rft.epage%253D846%26rft.atitle%253DHow%2Bthe%2Bosteoclast%2Bdegrades%2Bbone.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291521-1878%2528199810%252920%253A10%253C837%253A%253AAID-BIES9%253E3.0.CO%253B2-D%26rft_id%253Dinfo%253Apmid%252F9819571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1521-1878(199810)20:10&lt;837::AID-BIES9&gt;3.0.CO;2-D&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9819571&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076484400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.6" data-doi="10.1016/j.bbabio.2010.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bratic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trifunovic</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mitochondrial energy metabolism and ageing</span>. <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1797</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DBratic%26rft.auinit1%253DI.%26rft.volume%253D1797%26rft.issue%253D6-7%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DMitochondrial%2Benergy%2Bmetabolism%2Band%2Bageing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbabio.2010.01.004%26rft_id%253Dinfo%253Apmid%252F20064485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbabio.2010.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20064485&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.7" data-doi="10.1038/nature06757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lum</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Variations in DNA elucidate molecular networks that cause disease</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D452%26rft.issue%253D7186%26rft.spage%253D429%26rft.epage%253D435%26rft.atitle%253DVariations%2Bin%2BDNA%2Belucidate%2Bmolecular%2Bnetworks%2Bthat%2Bcause%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06757%26rft_id%253Dinfo%253Apmid%252F18344982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18344982&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254341300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">G.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosack</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DAVID: Database for Annotation, Visualization, and Integrated Discovery</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: 3.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.9" data-doi="10.1186/1752-0509-1-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Understanding network concepts in modules</span>. <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DDong%26rft.auinit1%253DJ.%26rft.volume%253D1%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DUnderstanding%2Bnetwork%2Bconcepts%2Bin%2Bmodules.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-1-24%26rft_id%253Dinfo%253Apmid%252F17547772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-1-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17547772&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.10" data-doi="10.1002/gepi.20395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elbers</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eijk</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Schouw</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Using genome-wide pathway analysis to unravel the etiology of complex diseases</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DElbers%26rft.auinit1%253DC.%2BC.%26rft.volume%253D33%26rft.issue%253D5%26rft.spage%253D419%26rft.epage%253D431%26rft.atitle%253DUsing%2Bgenome-wide%2Bpathway%2Banalysis%2Bto%2Bunravel%2Bthe%2Betiology%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20395%26rft_id%253Dinfo%253Apmid%252F19235186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19235186&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267605700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.11" data-doi="10.1002/gepi.20395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elbers</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eijk</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Schouw</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Using genome-wide pathway analysis to unravel the etiology of complex diseases</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DElbers%26rft.auinit1%253DC.%2BC.%26rft.volume%253D33%26rft.issue%253D5%26rft.spage%253D419%26rft.epage%253D431%26rft.atitle%253DUsing%2Bgenome-wide%2Bpathway%2Banalysis%2Bto%2Bunravel%2Bthe%2Betiology%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20395%26rft_id%253Dinfo%253Apmid%252F19235186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19235186&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267605700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.12" data-doi="10.1038/ng.2249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estrada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styrkarsdottir</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide meta-analysis identifies 56 bone mineral density loci and reveals 14 loci associated with risk of fracture</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DEstrada%26rft.auinit1%253DK.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D501%26rft.atitle%253DGenome-wide%2Bmeta-analysis%2Bidentifies%2B56%2Bbone%2Bmineral%2Bdensity%2Bloci%2Band%2Breveals%2B14%2Bloci%2Bassociated%2Bwith%2Brisk%2Bof%2Bfracture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2249%26rft_id%253Dinfo%253Apmid%252F22504420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22504420&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.13" data-doi="10.1002/jbmr.138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of a gene module associated with BMD through the integration of network analysis and genome-wide association data</span>. <abbr class="cit-jnl-abbrev">J. Bone Miner. Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2359</span>–<span class="cit-lpage">2367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbone%2Band%2Bmineral%2Bresearch%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BAmerican%2BSociety%2Bfor%2BBone%2Band%2BMineral%2BResearch%26rft.stitle%253DJ%2BBone%2BMiner%2BRes%26rft.aulast%253DFarber%26rft.auinit1%253DC.%2BR.%26rft.volume%253D25%26rft.issue%253D11%26rft.spage%253D2359%26rft.epage%253D2367%26rft.atitle%253DIdentification%2Bof%2Ba%2Bgene%2Bmodule%2Bassociated%2Bwith%2BBMD%2Bthrough%2Bthe%2Bintegration%2Bof%2Bnetwork%2Banalysis%2Band%2Bgenome-wide%2Bassociation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjbmr.138%26rft_id%253Dinfo%253Apmid%252F20499364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jbmr.138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20499364&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fontova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vendrell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vendrell</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bone mineral mass is associated with interleukin 1 receptor autoantigen and TNF-alpha gene polymorphisms in post-menopausal Mediterranean women</span>. <abbr class="cit-jnl-abbrev">J. Endocrinol. Invest.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">684</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bendocrinological%2Binvestigation%26rft.stitle%253DJ%2BEndocrinol%2BInvest%26rft.aulast%253DFontova%26rft.auinit1%253DR.%26rft.volume%253D25%26rft.issue%253D8%26rft.spage%253D684%26rft.epage%253D690%26rft.atitle%253DBone%2Bmineral%2Bmass%2Bis%2Bassociated%2Bwith%2Binterleukin%2B1%2Breceptor%2Bautoantigen%2Band%2BTNF-alpha%2Bgene%2Bpolymorphisms%2Bin%2Bpost-menopausal%2BMediterranean%2Bwomen.%26rft_id%253Dinfo%253Apmid%252F12240899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12240899&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.15" data-doi="10.1210/en.137.9.4058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujikawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabokbar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGee</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athanasou</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The human osteoclast precursor circulates in the monocyte fraction</span>. <abbr class="cit-jnl-abbrev">Endocrinology</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">4058</span>–<span class="cit-lpage">4060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEndocrinology%26rft.stitle%253DEndocrinology%26rft.aulast%253DFujikawa%26rft.auinit1%253DY%26rft.volume%253D137%26rft.issue%253D9%26rft.spage%253D4058%26rft.epage%253D4060%26rft.atitle%253DThe%2Bhuman%2Bosteoclast%2Bprecursor%2Bcirculates%2Bin%2Bthe%2Bmonocyte%2Bfraction%26rft_id%253Dinfo%253Adoi%252F10.1210%252Fen.137.9.4058%26rft_id%253Dinfo%253Apmid%252F8756585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1210/en.137.9.4058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8756585&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VD08600062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.16" data-doi="10.1073/pnas.0605457103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gargalovic</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharavi</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Identification of inflammatory gene modules based on variations of human endothelial cell responses to oxidized lipids</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">12741</span>–<span class="cit-lpage">12746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605457103%26rft_id%253Dinfo%253Apmid%252F16912112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM0LzEyNzQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.17" data-doi="10.1093/bioinformatics/btg405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cope</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolstad</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">affy–analysis of Affymetrix GeneChip data at the probe level</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DGautier%26rft.auinit1%253DL.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D307%26rft.epage%253D315%26rft.atitle%253Daffy--analysis%2Bof%2BAffymetrix%2BGeneChip%2Bdata%2Bat%2Bthe%2Bprobe%2Blevel%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg405%26rft_id%253Dinfo%253Apmid%252F14960456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMC8zLzMwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMS8xMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.18" data-doi="10.1371/journal.pgen.0020130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plaisier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Integrating genetic and network analysis to characterize genes related to mouse weight</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253De130%26rft.epage%253De130%26rft.atitle%253DIntegrating%2Bgenetic%2Band%2Bnetwork%2Banalysis%2Bto%2Bcharacterize%2Bgenes%2Brelated%2Bto%2Bmouse%2Bweight.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020130%26rft_id%253Dinfo%253Apmid%252F16934000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16934000&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.19" data-doi="10.1186/gb-2007-8-7-r149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barajas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Air-pollutant chemicals and oxidized lipids exhibit genome-wide synergistic effects on endothelial cells</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGong%26rft.auinit1%253DK.%2BW.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253DR149%26rft.epage%253DR149%26rft.atitle%253DAir-pollutant%2Bchemicals%2Band%2Boxidized%2Blipids%2Bexhibit%2Bgenome-wide%2Bsynergistic%2Beffects%2Bon%2Bendothelial%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-7-r149%26rft_id%253Dinfo%253Apmid%252F17655762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-7-r149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17655762&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.20" data-doi="10.1007/s00439-009-0676-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawitan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnusson</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Strategies and issues in the detection of pathway enrichment in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DHong%26rft.auinit1%253DM.%2BG.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DStrategies%2Band%2Bissues%2Bin%2Bthe%2Bdetection%2Bof%2Bpathway%2Benrichment%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-009-0676-z%26rft_id%253Dinfo%253Apmid%252F19408013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-009-0676-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19408013&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269056500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.21" data-doi="10.1073/pnas.0608396103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of oncogenic signaling networks in glioblastoma identifies ASPM as a molecular target</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">17402</span>–<span class="cit-lpage">17407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608396103%26rft_id%253Dinfo%253Apmid%252F17090670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ2LzE3NDAyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.22" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang da</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.23" data-doi="10.2307/1390807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ihaka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">R: a language for data analysis and graphics</span>. <abbr class="cit-jnl-abbrev">J. Comput. Graph. Statist.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BGraph.%2BStatist.%26rft.volume%253D5%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1390807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1390807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.24" data-doi="10.1093/biostatistics/4.2.249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobbs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beazer-Barclay</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonellis</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Exploration, normalization, and summaries of high density oligonucleotide array probe level data</span>. <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.stitle%253DBiostatistics%26rft.aulast%253DIrizarry%26rft.auinit1%253DR.%2BA.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D264%26rft.atitle%253DExploration%252C%2Bnormalization%252C%2Band%2Bsummaries%2Bof%2Bhigh%2Bdensity%2Boligonucleotide%2Barray%2Bprobe%2Blevel%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252F4.2.249%26rft_id%253Dinfo%253Apmid%252F12925520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjQvMi8yNDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.25" data-doi="10.1186/1471-2350-8-S1-S14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiel</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demissie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunetta</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murabito</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association with bone mass and geometry in the Framingham Heart Study</span>. <abbr class="cit-jnl-abbrev">BMC Med. Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">Suppl 1</span>): <span class="cit-fpage">S14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMed.%2BGenet.%26rft.volume%253D8%26rft.spage%253DS14%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2350-8-S1-S14%26rft_id%253Dinfo%253Apmid%252F17903296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2350-8-S1-S14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17903296&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.26" data-doi="10.1159/000110454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Osteoclast precursors display dynamic metabolic shifts toward accelerated glucose metabolism at an early stage of RANKL-stimulated osteoclast differentiation</span>. <abbr class="cit-jnl-abbrev">Cell. Physiol. Biochem.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Bphysiology%2Band%2Bbiochemistry%2B%253A%2B%2Binternational%2Bjournal%2Bof%2Bexperimental%2Bcellular%2Bphysiology%252C%2Bbiochemistry%252C%2Band%2Bpharmacology%26rft.stitle%253DCell%2BPhysiol%2BBiochem%26rft.aulast%253DKim%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D20%26rft.issue%253D6%26rft.spage%253D935%26rft.epage%253D946%26rft.atitle%253DOsteoclast%2Bprecursors%2Bdisplay%2Bdynamic%2Bmetabolic%2Bshifts%2Btoward%2Baccelerated%2Bglucose%2Bmetabolism%2Bat%2Ban%2Bearly%2Bstage%2Bof%2BRANKL-stimulated%2Bosteoclast%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000110454%26rft_id%253Dinfo%253Apmid%252F17982276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000110454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17982276&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.27" data-doi="10.1097/gme.0b013e3181a039c8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association between polymorphisms in tumor necrosis factor (TNF) and TNF receptor genes and circulating TNF, soluble TNF receptor levels, and bone mineral density in postmenopausal Korean women</span>. <abbr class="cit-jnl-abbrev">Menopause</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1014</span>–<span class="cit-lpage">1020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMenopause%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMenopause%26rft.aulast%253DKim%26rft.auinit1%253DH.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D1014%26rft.epage%253D1020%26rft.atitle%253DAssociation%2Bbetween%2Bpolymorphisms%2Bin%2Btumor%2Bnecrosis%2Bfactor%2B%2528TNF%2529%2Band%2BTNF%2Breceptor%2Bgenes%2Band%2Bcirculating%2BTNF%252C%2Bsoluble%2BTNF%2Breceptor%2Blevels%252C%2Band%2Bbone%2Bmineral%2Bdensity%2Bin%2Bpostmenopausal%2BKorean%2Bwomen.%26rft_id%253Dinfo%253Adoi%252F10.1097%252Fgme.0b013e3181a039c8%26rft_id%253Dinfo%253Apmid%252F19369902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/gme.0b013e3181a039c8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19369902&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.28" data-doi="10.1007/s11914-011-0054-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kousteni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FoxOs: Unifying links between oxidative stress and skeletal homeostasis</span>. <abbr class="cit-jnl-abbrev">Curr. Osteoporos. Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BOsteoporos%2BRep%26rft.aulast%253DKousteni%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D60%26rft.epage%253D66%26rft.atitle%253DFoxOs%253A%2BUnifying%2Blinks%2Bbetween%2Boxidative%2Bstress%2Band%2Bskeletal%2Bhomeostasis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11914-011-0054-3%26rft_id%253Dinfo%253Apmid%252F21404001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11914-011-0054-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21404001&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.29" data-doi="10.1172/JCI11176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeshita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanagawa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">TNF-alpha induces osteoclastogenesis by direct stimulation of macrophages exposed to permissive levels of RANK ligand</span>. <abbr class="cit-jnl-abbrev">J. Clin. Invest.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">1481</span>–<span class="cit-lpage">1488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253DLam%26rft.auinit1%253DJ.%26rft.volume%253D106%26rft.issue%253D12%26rft.spage%253D1481%26rft.epage%253D1488%26rft.atitle%253DTNF-%257B%257Balpha%257D%257D%2Binduces%2Bosteoclastogenesis%2Bby%2Bdirect%2Bstimulation%2Bof%2Bmacrophages%2Bexposed%2Bto%2Bpermissive%2Blevels%2Bof%2BRANK%2Bligand%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI11176%26rft_id%253Dinfo%253Apmid%252F11120755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI11176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11120755&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165908500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.30" data-doi="10.1186/1471-2105-9-559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">WGCNA: an R package for weighted correlation network analysis</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLangfelder%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.spage%253D559%26rft.epage%253D559%26rft.atitle%253DWGCNA%253A%2Ban%2BR%2Bpackage%2Bfor%2Bweighted%2Bcorrelation%2Bnetwork%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-559%26rft_id%253Dinfo%253Apmid%252F19114008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19114008&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.31" data-doi="10.1186/1745-7580-4-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lattin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Expression analysis of G protein-coupled receptors in mouse macrophages</span>. <abbr class="cit-jnl-abbrev">Immunome Res.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DImmunome%2BRes%26rft.aulast%253DLattin%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D4%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DExpression%2Banalysis%2Bof%2BG%2BProtein-Coupled%2BReceptors%2Bin%2Bmouse%2Bmacrophages.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-7580-4-5%26rft_id%253Dinfo%253Apmid%252F18442421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-7580-4-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18442421&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.32" data-doi="10.1016/j.bone.2008.05.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An in vivo genome wide gene expression study of circulating monocytes suggested GBP1, STAT1 and CXCL10 as novel risk genes for the differentiation of peak bone mass</span>. <abbr class="cit-jnl-abbrev">Bone</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1010</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBone%26rft.aulast%253DLei%26rft.auinit1%253DS.%2BF.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D1010%26rft.epage%253D1014%26rft.atitle%253DAn%2Bin%2Bvivo%2Bgenome%2Bwide%2Bgene%2Bexpression%2Bstudy%2Bof%2Bcirculating%2Bmonocytes%2Bsuggested%2BGBP1%252C%2BSTAT1%2Band%2BCXCL10%2Bas%2Bnovel%2Brisk%2Bgenes%2Bfor%2Bthe%2Bdifferentiation%2Bof%2Bpeak%2Bbone%2Bmass.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bone.2008.05.016%26rft_id%253Dinfo%253Apmid%252F19223260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bone.2008.05.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19223260&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.33" data-doi="10.1093/bioinformatics/btp448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Dushlaine</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heron</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segurado</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The SNP ratio test: pathway analysis of genome-wide association datasets</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2762</span>–<span class="cit-lpage">2763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp448%26rft_id%253Dinfo%253Apmid%252F19620097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjAvMjc2MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMS8xMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.34" data-doi="10.1038/nn.2207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oldham</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konopka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Functional organization of the transcriptome in human brain</span>. <abbr class="cit-jnl-abbrev">Nat. Neurosci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DOldham%26rft.auinit1%253DM.%2BC.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1271%26rft.epage%253D1282%26rft.atitle%253DFunctional%2Borganization%2Bof%2Bthe%2Btranscriptome%2Bin%2Bhuman%2Bbrain.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.2207%26rft_id%253Dinfo%253Apmid%252F18849986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn.2207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18849986&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260472700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.35" data-doi="10.1038/ejhg.2009.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene and pathway-based second-wave analysis of genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DPeng%26rft.auinit1%253DG.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DGene%2Band%2Bpathway-based%2Bsecond-wave%2Banalysis%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2009.115%26rft_id%253Dinfo%253Apmid%252F19584899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2009.115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584899&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272609900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.36" data-doi="10.1186/gm65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Using prior knowledge and genome-wide association to identify pathways involved in multiple sclerosis</span>. <abbr class="cit-jnl-abbrev">Genome Med</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenome%2BMed%26rft.aulast%253DRitchie%26rft.auinit1%253DM.%2BD.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253D65%26rft.epage%253D65%26rft.atitle%253DUsing%2Bprior%2Bknowledge%2Band%2Bgenome-wide%2Bassociation%2Bto%2Bidentify%2Bpathways%2Binvolved%2Bin%2Bmultiple%2Bsclerosis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm65%26rft_id%253Dinfo%253Apmid%252F19566917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19566917&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.37" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.38" data-doi="10.1056/NEJMoa0801197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Styrkarsdottir</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halldorsson</span>  <span class="cit-name-given-names">B. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gretarsdottir</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple genetic loci for bone mineral density and fractures</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">2355</span>–<span class="cit-lpage">2365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.volume%253D358%26rft.issue%253D22%26rft.spage%253D2355%26rft.epage%253D2365%26rft.atitle%253DMultiple%2Bgenetic%2Bloci%2Bfor%2Bbone%2Bmineral%2Bdensity%2Band%2Bfractures.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa0801197%26rft_id%253Dinfo%253Apmid%252F18445777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa0801197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18445777&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256306700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.39" data-doi="10.1073/pnas.012025199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ching</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakak</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Large-scale analysis of the human and mouse transcriptomes</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">4465</span>–<span class="cit-lpage">4470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.012025199%26rft_id%253Dinfo%253Apmid%252F11904358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS83LzQ0NjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.40" data-doi="10.1073/pnas.0400782101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batalov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapp</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ching</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A gene atlas of the mouse and human protein-encoding transcriptomes</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">6062</span>–<span class="cit-lpage">6067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0400782101%26rft_id%253Dinfo%253Apmid%252F15075390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE2LzYwNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.41" data-doi="10.1007/978-1-60761-175-2_16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torkamani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pathway and network analysis with high-density allelic association data</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">563</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DTorkamani%26rft.auinit1%253DA.%26rft.volume%253D563%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DPathway%2Band%2Bnetwork%2Banalysis%2Bwith%2Bhigh-density%2Ballelic%2Bassociation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-175-2_16%26rft_id%253Dinfo%253Apmid%252F19597792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60761-175-2_16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19597792&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.42" data-doi="10.1016/j.ygeno.2008.07.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torkamani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topol</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pathway analysis of seven common diseases assessed by genome-wide association</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DTorkamani%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D5%26rft.spage%253D265%26rft.epage%253D272%26rft.atitle%253DPathway%2Banalysis%2Bof%2Bseven%2Bcommon%2Bdiseases%2Bassessed%2Bby%2Bgenome-wide%2Bassociation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2008.07.011%26rft_id%253Dinfo%253Apmid%252F18722519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2008.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18722519&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260715400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.43" data-doi="10.1210/en.2008-0563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Nas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guhathakurta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehya</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Elucidating the role of gonadal hormones in sexually dimorphic gene coexpression networks</span>. <abbr class="cit-jnl-abbrev">Endocrinology</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEndocrinology%26rft_id%253Dinfo%253Adoi%252F10.1210%252Fen.2008-0563%26rft_id%253Dinfo%253Apmid%252F18974276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1210/en.2008-0563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18974276&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263613200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.44" data-doi="10.1086/522374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pathway-based approaches for analysis of genomewide association studies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1278</span>–<span class="cit-lpage">1283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D81%26rft.issue%253D6%26rft.spage%253D1278%26rft.epage%253D1283%26rft.atitle%253DPathway-based%2Bapproaches%2Bfor%2Banalysis%2Bof%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F522374%26rft_id%253Dinfo%253Apmid%252F17966091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/522374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17966091&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251098800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.45" data-doi="10.1006/bbrc.1997.6795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Regulation of osteoclastic bone resorption by glucose</span>. <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">235</span>: <span class="cit-fpage">646</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DWilliams%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D235%26rft.issue%253D3%26rft.spage%253D646%26rft.epage%253D651%26rft.atitle%253DRegulation%2Bof%2Bosteoclastic%2Bbone%2Bresorption%2Bby%2Bglucose.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fbbrc.1997.6795%26rft_id%253Dinfo%253Apmid%252F9207213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/bbrc.1997.6795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9207213&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.46" data-doi="10.1038/msb.2009.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winden</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oldham</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirnics</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swan</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The organization of the transcriptional network in specific neuronal classes</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DWinden%26rft.auinit1%253DK.%2BD.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D291%26rft.epage%253D291%26rft.atitle%253DThe%2Borganization%2Bof%2Bthe%2Btranscriptional%2Bnetwork%2Bin%2Bspecific%2Bneuronal%2Bclasses%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2009.46%26rft_id%253Dinfo%253Apmid%252F19638972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjUvMS8yOTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-other" id="cit-3.1.119.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A general framework for weighted gene co-expression network analysis</span>. <span class="cit-source">Stat. Appl. Genet. Mol. Biol.</span> <span class="cit-vol">4</span>: <span class="cit-comment">Article 17</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.119.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoddy</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">WebGestalt: an integrated system for exploring gene sets in various biological contexts</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W741–748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D33%26rft.spage%253DW741%2B748%26rft.atitle%253DWEBGESTALT%253A%2BAN%2BINTEGRATED%2BSYSTEM%2BFOR%2BEXPLORING%2BGENE%2BSETS%2BIN%2BVARIOUS%2BBIOLOGICAL%2BCONTEXTS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/1/119.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/1/109" title="Mapping Quantitative Trait Loci Affecting Arabidopsis thaliana Seed Morphology Features Extracted Computationally From Images" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/1/131" title="Compensatory Mutations in Predicted Metal Transporters Modulate Auxin Conjugate Responsiveness in Arabidopsis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/1"><span>Volume 3 Issue 1, January 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/1.cover-source.jpg?itok=IC8XGx84" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/1/119" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--K9ysLNxGdnLdWOFbW1eB8SYI85l0BFswXBpJ8gLNcA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/1/119" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/1/119" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/1/119" class="active" data-icon-position="" data-hide-link-title="0">Systems-Level Analysis of Genome-Wide Association Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459397" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-l0pZRvgjn3QXNP9MGRn4YnbiWWvO5tf_-Fcc0yeoyrI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/1/119.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/1/119" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CF-5SBG8F3vfyJpwTSJ_MyW3Z25RBNkukXmL7el9Re4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/1/119" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/1/119.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459397" id="node459397--2" data-pisa="ggg;3/1/119" data-pisa-master="ggg;3/1/119" data-apath="/ggg/3/1/119.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systems-Level Analysis of Genome-Wide Association Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles R. Farber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">119-129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004788 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459397--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles R. Farber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Public Health Genomics, Departments of Medicine (Division of Cardiology) and Biochemistry and Molecular Genetics, University of Virginia, Charlottesville, Virginia 22908</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BR.%2BFarber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Farber%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BR.%2BFarber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:crf2s@virginia.edu" class="" data-icon-position="" data-hide-link-title="0">crf2s@virginia.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/1/119" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/1/119&amp;title=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/1/119&amp;title=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/1/119&amp;title=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/1/119&amp;text=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/1/119&amp;title=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/1/119&amp;t=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/1/119&amp;title=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/1/119&amp;title=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459397/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459397" id="node459397--3" data-pisa="ggg;3/1/119" data-pisa-master="ggg;3/1/119" data-apath="/ggg/3/1/119.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systems-Level Analysis of Genome-Wide Association Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles R. Farber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">119-129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004788 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459397--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles R. Farber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Public Health Genomics, Departments of Medicine (Division of Cardiology) and Biochemistry and Molecular Genetics, University of Virginia, Charlottesville, Virginia 22908</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BR.%2BFarber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Farber%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BR.%2BFarber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:crf2s@virginia.edu" class="" data-icon-position="" data-hide-link-title="0">crf2s@virginia.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=119&amp;rft.epage=129&amp;rft.atitle=Systems-Level%20Analysis%20of%20Genome-Wide%20Association%20Data&amp;rft.volume=3&amp;rft.issue=1&amp;rft.date=2013-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Farber%2C+Charles+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459397/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459397/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459397/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459397/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459397/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459397/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459397/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459397/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459397/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459397/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459397/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459397/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/1/119" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/1/119.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/1/119.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/1/119.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/1/119.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
