<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/1/23" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/1/23.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/1/23.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/1/23.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/1/23.atom" />
<meta name="HW.pisa" content="ggg;3/1/23" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples" />
<meta name="DC.Identifier" content="10.1534/g3.112.004069" />
<meta name="DC.Date" content="2013-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Jiang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Most genome-wide association studies to date have been performed in populations of European descent, but there is increasing interest in expanding these studies to other populations. The performance of genotyping chips in Asian populations is not well established. Therefore, we sought to test the performance of widely used fixed-marker, genome-wide association studies chips in the Han Chinese population. Non-HapMap Chinese samples (n = 396) were genotyped using the Illumina OmniExpress and Affymetrix 6.0 platforms, whereas a subset also were genotyped using the Immunochip. Genotyped markers from the Affymetrix 6.0 and Illumina OmniExpress were used for full genome imputation based on the HapMap 2 JPT+CHB (Japanese from Tokyo, Japan and Chinese from Beijing, China) reference panel. The concordance between markers genotypes for the three platforms was very high whether directly genotyped or genotyped and imputed single nucleotide polymorphisms (SNPs; &gt;99.8% for directly genotyped and &gt;99.5% for genotyped and imputed SNPs, respectively) were compared. The OmniExpress chip data enabled more SNPs to be imputed, particularly SNPs with minor allele frequency &gt;5%. The OmniExpress chip achieved better coverage of HapMap SNPs than the Affymetrix 6.0 chip (73.6% vs. 65.9%, respectively, for minor allele frequency &gt;5%). The Affymetrix 6.0 and Illumina OmniExpress chip have similar genotyping accuracy and provide similar accuracy of imputed SNPs. The OmniExpress chip however provides better coverage of Asian HapMap SNPs, although its coverage of HapMap SNPs is moderate." />
<meta name="DC.Contributor" content="Lei Jiang" />
<meta name="DC.Contributor" content="Dana Willner" />
<meta name="DC.Contributor" content="Patrick Danoy" />
<meta name="DC.Contributor" content="Huji Xu" />
<meta name="DC.Contributor" content="Matthew A. Brown" />
<meta name="article:published_time" content="2013-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most genome-wide association studies to date have been performed in populations of European descent, but there is increasing interest in expanding these studies to other populations. The performance of genotyping chips in Asian populations is not well established. Therefore, we sought to test the performance of widely used fixed-marker, genome-wide association studies chips in the Han Chinese population. Non-HapMap Chinese samples (n = 396) were genotyped using the Illumina OmniExpress and Affymetrix 6.0 platforms, whereas a subset also were genotyped using the Immunochip. Genotyped markers from the Affymetrix 6.0 and Illumina OmniExpress were used for full genome imputation based on the HapMap 2 JPT+CHB (Japanese from Tokyo, Japan and Chinese from Beijing, China) reference panel. The concordance between markers genotypes for the three platforms was very high whether directly genotyped or genotyped and imputed single nucleotide polymorphisms (SNPs; &amp;gt;99.8% for directly genotyped and &amp;gt;99.5% for genotyped and imputed SNPs, respectively) were compared. The OmniExpress chip data enabled more SNPs to be imputed, particularly SNPs with minor allele frequency &amp;gt;5%. The OmniExpress chip achieved better coverage of HapMap SNPs than the Affymetrix 6.0 chip (73.6% &lt;i&gt;vs.&lt;/i&gt; 65.9%, respectively, for minor allele frequency &amp;gt;5%). The Affymetrix 6.0 and Illumina OmniExpress chip have similar genotyping accuracy and provide similar accuracy of imputed SNPs. The OmniExpress chip however provides better coverage of Asian HapMap SNPs, although its coverage of HapMap SNPs is moderate.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/01/01" />
<meta name="citation_mjid" content="ggg;3/1/23" />
<meta name="citation_id" content="3/1/23" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/1/23" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/1/23.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/1/23.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/1/23.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004069" />
<meta name="citation_pmid" content="23316436" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="23" />
<meta name="citation_lastpage" content="29" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lei Jiang" />
<meta name="citation_author_institution" content="Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China" />
<meta name="citation_author" content="Dana Willner" />
<meta name="citation_author_institution" content="The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and" />
<meta name="citation_author_institution" content="Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Brisbane, Australia 4072" />
<meta name="citation_author" content="Patrick Danoy" />
<meta name="citation_author_institution" content="The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and" />
<meta name="citation_author" content="Huji Xu" />
<meta name="citation_author_institution" content="Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China" />
<meta name="citation_author_email" content="xuhuji@smmu.edu.cn" />
<meta name="citation_author_email" content="matt.brown@uq.edu.au" />
<meta name="citation_author" content="Matthew A. Brown" />
<meta name="citation_author_institution" content="The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and" />
<meta name="citation_author_email" content="xuhuji@smmu.edu.cn" />
<meta name="citation_author_email" content="matt.brown@uq.edu.au" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. C. Barrett;citation_author=L. R. Cardon;citation_title=Evaluating coverage of genome-wide association studies.;citation_pages=659-662;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16715099;citation_doi=10.1038/ng1801" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=Q. Cai;citation_author=J. Long;citation_author=W. Lu;citation_author=S. Qu;citation_author=W. Wen;citation_title=Genome-wide association study identifies breast cancer risk variant at 10q21.2: results from the Asia Breast Cancer Consortium;citation_volume=20;citation_year=2011;citation_pmid=21908515;citation_doi=10.1093/hmg/ddr405" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=C. L. Carty;citation_author=N. A. Johnson;citation_author=C. M. Hutter;citation_author=A. P. Reiner;citation_author=U. Peters;citation_title=Genome-wide association study of body height in African-Americans: the Women’s Health Initiative SNP Health Association Resource (SHARe);citation_pages=711-720;citation_volume=21;citation_year=2011;citation_pmid=22021425" />
<meta name="citation_reference" content="citation_journal_title=Arthritis research &amp; therapy;citation_journal_abbrev=Arthritis Res Ther;citation_author=A. Cortes;citation_author=M. A. Brown;citation_title=Promise and pitfalls of the Immunochip.;citation_pages=101-101;citation_volume=13;citation_year=2011;citation_issue=1;citation_pmid=21345260;citation_doi=10.1186/ar3382" />
<meta name="citation_reference" content="citation_title=Single Nucleotide Polymorphisms;citation_title=High-throughput methods for SNP genotyping;citation_author=C. Ding;citation_author=S. Jin;citation_pages=245-254, 578;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Franceschini;citation_author=F. van Rooij;citation_author=B. Prins;citation_author=M. Feitosa;citation_author=M. Karakas;citation_title=Discovery and fine mapping of serum protein loci through transethnic meta-analysis.;citation_pages=744-753;citation_volume=91;citation_year=2012;citation_issue=4;citation_pmid=23022100;citation_doi=10.1016/j.ajhg.2012.08.021" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. Hao;citation_author=E. Schadt;citation_author=J. Story;citation_title=Calibrating the performance of SNP arrays for whole-genome association studies.;citation_pages=e1000109-e1000109;citation_volume=4;citation_year=2008;citation_issue=6;citation_pmid=18584036;citation_doi=10.1371/journal.pgen.1000109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Hindorff;citation_author=P. Sethupathy;citation_author=H. Junkins;citation_author=E. Ramos;citation_author=J. Mehta;citation_title=Potential etiologic and functional implications of genome-wide association loci for human diseases and traits;citation_volume=106;citation_year=2009;citation_pmid=19474294;citation_doi=10.1073/pnas.0903103106" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Huang;citation_author=Y. Li;citation_author=A. B. Singleton;citation_author=J. A. Hardy;citation_author=G. Abecasis;citation_title=Genotype-imputation accuracy across worldwide human populations.;citation_pages=235-250;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19215730;citation_doi=10.1016/j.ajhg.2009.01.013" />
<meta name="citation_reference" content="citation_journal_title=Arthritis and rheumatism;citation_journal_abbrev=Arthritis Rheum;citation_author=T. Hughes;citation_author=X. Kim-Howard;citation_author=J. A. Kelly;citation_author=K. M. Kaufman;citation_author=C. D. Langefeld;citation_title=Fine-mapping and transethnic genotyping establish IL2/IL21 genetic association with lupus and localize this genetic effect to IL21.;citation_pages=1689-1697;citation_volume=63;citation_year=2011;citation_issue=6;citation_pmid=21425124;citation_doi=10.1002/art.30320" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Jimenez-Sanchez;citation_author=B. Childs;citation_author=D. Valle;citation_title=Human disease genes.;citation_pages=853-855;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237009;citation_doi=10.1038/35057050" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=G.C. Kennedy;citation_author=H. Matsuzaki;citation_author=S. Dong;citation_author=W.M. Liu;citation_author=J. Huang;citation_title=Large-scale genotyping of complex DNA.;citation_pages=1233-1237;citation_volume=21;citation_year=2003;citation_issue=10;citation_pmid=12960966;citation_doi=10.1038/nbt869" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=M. Li;citation_author=C. Li;citation_author=W. Guan;citation_title=Evaluation of coverage variation of SNP chips for genome-wide association studies.;citation_pages=635-643;citation_volume=16;citation_year=2008;citation_issue=5;citation_pmid=18253166;citation_doi=10.1038/sj.ejhg.5202007" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.;citation_pages=816-834;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058334;citation_doi=10.1002/gepi.20533" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Mägi;citation_author=A. Pfeufer;citation_author=M. Nelis;citation_author=A. Montpetit;citation_author=A. Metspalu;citation_title=Evaluating the performance of commercial whole-genome marker sets for capturing common genetic variation.;citation_pages=159-159;citation_volume=8;citation_year=2007;citation_pmid=17562002;citation_doi=10.1186/1471-2164-8-159" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorff;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Noguchi;citation_author=H. Sakamoto;citation_author=T. Hirota;citation_author=K. Ochiai;citation_author=Y. Imoto;citation_title=Genome-wide association study identifies HLA-DP as a susceptibility gene for pediatric asthma in Asian populations.;citation_pages=e1002170-e1002170;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21814517;citation_doi=10.1371/journal.pgen.1002170" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=M. Nothnagel;citation_author=D. Ellinghaus;citation_author=S. Schreiber;citation_author=M. Krawczak;citation_author=A. Franke;citation_title=A comprehensive evaluation of SNP genotype imputation.;citation_pages=163-171;citation_volume=125;citation_year=2009;citation_issue=2;citation_pmid=19089453;citation_doi=10.1007/s00439-008-0606-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y.-F. Pei;citation_author=J. Li;citation_author=L. Zhang;citation_author=C. J. Papasian;citation_author=H.-W. Deng;citation_title=Analyses and comparison of accuracy of different genotype imputation methods.;citation_pages=e3551-e3551;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18958166;citation_doi=10.1371/journal.pone.0003551" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Price;citation_author=N. Patterson;citation_author=R. Plenge;citation_author=M. Weinblatt;citation_author=N. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_title=R: A Language and Envrionment for Statistical Computing;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=J. Ragoussis;citation_title=Genotyping technologies for genetic research.;citation_pages=117-133;citation_volume=10;citation_year=2009;citation_pmid=19453250;citation_doi=10.1146/annurev-genom-082908-150116" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=J. A. Sinnott;citation_author=P. Kraft;citation_title=Artifact due to differential error when cases and controls are imputed from different platforms.;citation_pages=111-119;citation_volume=131;citation_year=2012;citation_issue=1;citation_pmid=21735171;citation_doi=10.1007/s00439-011-1054-1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. G. Smith;citation_author=J. W. Magnani;citation_author=C. Palmer;citation_author=Y. A. Meng;citation_author=E. Z. Soliman;citation_title=Genome-wide association studies of the PR interval in African Americans.;citation_pages=e1001304-e1001304;citation_volume=7;citation_year=2011;citation_issue=2;citation_pmid=21347284;citation_doi=10.1371/journal.pgen.1001304" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=F. J. Steemers;citation_author=W. Chang;citation_author=G. Lee;citation_author=D. L. Barker;citation_author=R. Shen;citation_title=Whole-genome genotyping with the single-base extension assay.;citation_pages=31-33;citation_volume=3;citation_year=2006;citation_issue=1;citation_pmid=16369550;citation_doi=10.1038/nmeth842" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A haplotype map of the human genome.;citation_pages=1299-1320;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16255080;citation_doi=10.1038/nature04226" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet;citation_title=Efficiency of trans-ethnic genome-wide meta-analysis and fine-mapping;citation_author=R. Twee-Hee Ong;citation_author=X. Wang;citation_author=X. Liu;citation_author=Y. Teo;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Wang;citation_author=M. Li;citation_author=M. Bucan;citation_title=Pathway-based approaches for analysis of genomewide association studies.;citation_pages=1278-1283;citation_volume=81;citation_year=2007;citation_issue=6;citation_pmid=17966091;citation_doi=10.1086/522374" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Xu;citation_author=X. Yin;citation_author=S. Li;citation_author=W. Jin;citation_author=H. Lou;citation_title=Genomic dissection of population substructure of Han Chinese and its implication in association studies.;citation_pages=762-774;citation_volume=85;citation_year=2009;citation_issue=6;citation_pmid=19944404;citation_doi=10.1016/j.ajhg.2009.10.015" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=D. Yoon;citation_author=Y.-J. Kim;citation_author=W.-Y. Cui;citation_author=A. Van der Vaart;citation_author=Y. S. Cho;citation_title=Large-scale genome-wide association study of Asian population reveals genetic factors in FRMD4A and other loci influencing smoking initiation and nicotine dependence.;citation_pages=1009-1021;citation_volume=131;citation_year=2012;citation_issue=6;citation_pmid=22006218;citation_doi=10.1007/s00439-011-1102-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/1.cover-source.jpg" />
<meta name="twitter:description" content="Most genome-wide association studies to date have been performed in populations of European descent, but there is increasing interest in expanding these studies to other populations. The performance of genotyping chips in Asian populations is not well established. Therefore, we sought to test the performance of widely used fixed-marker, genome-wide association studies chips in the Han Chinese population. Non-HapMap Chinese samples (n = 396) were genotyped using the Illumina OmniExpress and Affymetrix 6.0 platforms, whereas a subset also were genotyped using the Immunochip. Genotyped markers from the Affymetrix 6.0 and Illumina OmniExpress were used for full genome imputation based on the HapMap 2 JPT+CHB (Japanese from Tokyo, Japan and Chinese from Beijing, China) reference panel. The concordance between markers genotypes for the three platforms was very high whether directly genotyped or genotyped and imputed single nucleotide polymorphisms (SNPs; &gt;99.8% for directly genotyped and &gt;99.5% for genotyped and imputed SNPs, respectively) were compared. The OmniExpress chip data enabled more SNPs to be imputed, particularly SNPs with minor allele frequency &gt;5%. The OmniExpress chip achieved better coverage of HapMap SNPs than the Affymetrix 6.0 chip (73.6% vs. 65.9%, respectively, for minor allele frequency &gt;5%). The Affymetrix 6.0 and Illumina OmniExpress chip have similar genotyping accuracy and provide similar accuracy of imputed SNPs. The OmniExpress chip however provides better coverage of Asian HapMap SNPs, although its coverage of HapMap SNPs is moderate." />
<meta name="og-title" property="og:title" content="Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/1/23" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Most genome-wide association studies to date have been performed in populations of European descent, but there is increasing interest in expanding these studies to other populations. The performance of genotyping chips in Asian populations is not well established. Therefore, we sought to test the performance of widely used fixed-marker, genome-wide association studies chips in the Han Chinese population. Non-HapMap Chinese samples (n = 396) were genotyped using the Illumina OmniExpress and Affymetrix 6.0 platforms, whereas a subset also were genotyped using the Immunochip. Genotyped markers from the Affymetrix 6.0 and Illumina OmniExpress were used for full genome imputation based on the HapMap 2 JPT+CHB (Japanese from Tokyo, Japan and Chinese from Beijing, China) reference panel. The concordance between markers genotypes for the three platforms was very high whether directly genotyped or genotyped and imputed single nucleotide polymorphisms (SNPs; &gt;99.8% for directly genotyped and &gt;99.5% for genotyped and imputed SNPs, respectively) were compared. The OmniExpress chip data enabled more SNPs to be imputed, particularly SNPs with minor allele frequency &gt;5%. The OmniExpress chip achieved better coverage of HapMap SNPs than the Affymetrix 6.0 chip (73.6% vs. 65.9%, respectively, for minor allele frequency &gt;5%). The Affymetrix 6.0 and Illumina OmniExpress chip have similar genotyping accuracy and provide similar accuracy of imputed SNPs. The OmniExpress chip however provides better coverage of Asian HapMap SNPs, although its coverage of HapMap SNPs is moderate." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/1.cover-source.jpg" />
    <title>Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"prm4VGTfRM0AxDs6jouI4dG77ugDQal_GPpbFJtWFA4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459388","apath":"\/ggg\/3\/1\/23.atom","pisa":"ggg;3\/1\/23","ac":{"\/ggg\/3\/1\/23.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/1\/23.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/1\/23"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/1\/23"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/1\/23.long":true,"\/content\/3\/1\/23":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222CnrU6WrqgAHJy3FM4JSngmk47ANENtb5Ex\\\\\\\/\\\\\\\/L+kGxKxEmWMWR+\\\\\\\/X3aONA7UBgmFsHpVTGMLgoAwQ30lU4GpQph2IpRhIDSO6jJy6jVUyAivOqG5i1LauO79Jbdzz5D6eRiEC2lj9s3Z2e2d8CI8cV2fiiTlbegNx2Cz6+ie+WVJGWvsjcwK0F9liOwAY72fmj+5aZPyJwQ13dxm1ALDQNd\\\\\\\/rEWKv85A0fevr+ZgH5yekKyqBqjoHkJdfou53qN\\\\\\\/OcU\\\\\\\/gaxLzGXEoNPE8c8UosogljbB2Q+CbhU6q9690e11FZ+nrvHvadNlOT8EEL6umEU3ShLb2zpyEnqsHxmicO5izfbs5wpUe5YZE60jem8vKfe0sWmUciA5zswKe89IDEGqXeFKL0OSXS7VdgJ+Thc7pQcX3q0pqjrxYKV\\\\\\\/cPRXz\\\\\\\/RyR3BcurdWeR2KBUxVgWYOYKWvVwMW7jMJFAaqtVhCGfqrAQnpR5wADKFZIr0a+f2RWTdkV1rYROHtpACxSn\\\\\\\/gwQLYLPsz3jazK1\\\\\\\/mo78xhhPbH5UwfetgiBabQJ04sLxJaP8ilfUb3hO1XgcS05LKSMTs7dL9z\\\\\\\/9IkEawwEWxb3Yaf5FSSHLa2G2iyx6nDztm47smNyg8aAK0eiREA6p6U++GfATQ0VXpThtYi7mOqUCabLHqx3bE8HTFfqFP8bc9pehEIHgdjR\\\\\\\/rPh9x7bz46xMnnwR5ZOdkYYrEXZgZI0vwNRnK2hIk0BUTAJ8LxPTpunjPFjxxqhBNzPOuoFhBSpJSCgcGL\\\\\\\/ro9xH4GmtB+0DLVJsI\\\\\\\/OPyRhy62RkzCTfriQU2aEgwNq0VHwrVCtjOFZUfOnMu1d91MnJNgbl3QZt0RCl2BJyesTkDJ2pj7LmFyLpvDHBLQwwjdXJwfg9um++NT5D02FUm8ifHP4s4flOWnKMzOgyUdPyxyQx1Os7yZKnj+RgZi\\\\\\\/AH2T0Nr2Y9TE+LQGN+esXcmMM8JrqZ8GZoU0+zQ9fk3M43Iz2lmwwodQ0aXjYoH36E6JOztxq\\\\\\\/AdKmvNQl85cIXQRmEYe4lD+hQaKPHTA9hS0gzOzIPOu9XrpKo2VevkNDu4+ZEqSawPY7UiEg6Z6DTn\\\\\\\/sSIBKFqo6o1+WKn84Gw8WqActpOtIU6\\\\\\\/8HY+80b+hJtgIT2EIagV\\\\\\\/UAr+HSC4dKd+sPPsD5Y2T67WbkFXmvMi6FUNajl2xCQ5qC2IkO\\\\\\\/Gi7q+vHxx9opWgEUGmJ93rEYTqXcy1LyMlhKykPaIFCgT5EOFzaZkAN0fCVSPsLnI3Yif\\\u0022,\\\u0022iv\\\u0022:\\\u0022diI\\\\\\\/6Sdw4On7kVqUtItnMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ee357d941b5396d6952990de46013960634cf7e6c291ea7f25eb1c513ade0646\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AJGFOViOACtchH9TVpd21yBrITPA3E\\\\\\\/WekB1+9SMl3QsYvp1gbEwUXxZ6dDjUqszzXAz1EHhpX33cHEO04Fq+fz0eBnzwhFi5R3S9aq1\\\\\\\/ImFxsmOuCljEB+o6KW3FQpfwC2gYAWBIArPf2uc6PIuI80XVA6MP3jmubLsmEvcysaIjW0BQkfgNdLejz+qCdzYcNcwojjYfDjLm0L+KRDTA9mO+VsmftbC3wrcuWr0O+ZnU8mNYI\\\\\\\/RLEQczCFVjM3QFXC7UYaGwX6u7jT6ZCYTlWrrS+0P9s+zP3Te+xJJKAsefeQsGSLS7HXQlPSUn4On7+SXhNjHaiBQCdR389z2ZhNcxSKGmKUqpQC1bod1qxACPJKlawrRvICflXXgd9IGv\\\\\\\/6GUBW8oNPpw0WY4BMrMnyu2NJ6rcSElqwz3YSjkoSikAQVoRcseYk6OisfHk0kH1n6gTXPMqGhKQO9WS0p6ZUdmnDUOVJndGo91sAJ9rhRQvMYCM7y3Kj85VMug1tsapeRwlKuz+TqiRBmeuo8Hkl\\\\\\\/b+FkSf9e26JVG3hR6UTeAh85WGCY0OHU56lvr3WHbBzYYLOr\\\\\\\/o9IFlZWM6m6jV47Z\\\\\\\/nDDWYRrxSRFplqr37YNyNydtnnofNyIIIbRlwGe6LCX2p68k6Q5c81im2LbERGE48FfrMJFTQk4jTewYeqNgDbyvFVQNI1uLBXQiSJVCSVdjVoPyvEUusbaGzJGR674WXQBMZKG9J9WBIfq5O6GoK2v1Va6TXHVVfzE8mzlGHnZCBRaRRbkbQiD0iQxUdoHzxVPzNWQDKR4wKH\\\\\\\/is3MOECt493eU2\\\\\\\/zq\\\\\\\/oQLv1qd+t9cKtgrgjI5sIEs2yZmMqGrIr6dV2mbpOy6jXWp7ublt3\\\\\\\/Nue1j8vbb6hCMxT2ai6aPIBeu\\\\\\\/1XRhXM9VQzBNbAnPZvpU0nNESA7LTBXlQ8vgleQI8EszmS3jxsKPibJkWdqQocBmeIP96I0\\\\\\\/EBbQrdparmfj1\\\\\\\/WvsfwtHIWocEOkShwXbQrxr7Mij44ar5eyYTPIkeZEM4j1w3b65SSiIF53gdkHaV0m2eOejR3W7cW9IoJZwcG66wXfbYdQbOymkeTBIq6ClPrSzQHib+8mqavenzkSOwU666T+fLltBh\\\\\\\/izq\\\\\\\/UdhgkPkj2fUN6ahgYZRk2RY5L2CB\\\\\\\/p7tR+hShJldtUXY0NMu+bTaeiOcRIj03tFQ6YZo\\\\\\\/Pybo5gOvtkPRxlrX8AGvsgqvNckoPtICg13U4We8MluVjVWtvfGvdRvGEk4cb4j6p6xbEltHa90f5ULuwLXrIC1QmJvGJFf6Jjq3y5IzjHVVwGYAS4ArbLl3CEeSQ2nz6RdtkmSIP35gcNPoy7vYzI8gk6HwIuJkLtDEb3zEfh8a5Y5ZXs0cRX9b2mb2qpA4ZolfNvXqqGIE7q1noJ3XiRdhh4LMo8YC5fO4sLpBFe2n\\\\\\\/xTg6Ov5AqN2uk851qhI9qBFG7elBie0yIyjXFcDzkye4ACM7YifT151WFKo4RL71gehFNncb80yia0fJeDMRnh\\\\\\\/IPTz34G5lB1Rf4JPDkoM5qVXQtfV6926VRjI6j3yymzf7u6iKSNYJG2gddGf3NZMS1pZr3nPRN9Yc46EMzUEHYMDR4iVLyCnzog4taDZxmGwurfXNqk1GivQkA+tDosjNEbFIFWrzjQBVsQIrADZEuDq1QSKKibuYPxIKvboVURm6uBAMaEQExyBnweK6Uud1psCJAHWHDOeaKWJYowW\\\\\\\/aTOr24\\\\\\\/u2FuccouiwLozOwWMkT0BI22J2S6rGNNXUwJWkXhNf6l+JysI9LK3wSauS62Z78s8kzCCcmReD5O7vHpONkbR51moWAEvuOpRi1pfvnWK\\\\\\\/zSxRB3QcL0YSxSGZwcB+E1MQSO8eyQV\\\\\\\/sqLXvevYjqzayJdZuAY1BJYgqtCUQ9TrTYrx6t83qMb2O+PbZ\\\\\\\/z\\\\\\\/tZUNXfwVa7f6jIbHdlWKkALguxCznLVdM9DV37HJSjqDoT3FF+rn7uEbSbYEUlRGbPu2ydTiWwR3jI+Z2bEob5iWevyq\\\\\\\/zZjANC3b\\\\\\\/QbkPrcpKwx4TY+c4h3dZb\\\\\\\/KmP7O2V6SHjTwN\\\\\\\/r0l+RALQQaIFhtOZiodMdXfT3jO\\\\\\\/lf8jlQjynj1lERSDVAGXJSYMw0LtCcorVgGc9M0nFpbKZjlz9QvhNvT5j08rs4Drwpi8z6Jhcd\\\\\\\/hqc7eCWHGk0rXlFRIklhZ7aCYtqBflFc\\\\\\\/NjxxOESNdBN4tBNU6poN2mxtSm9R\\\\\\\/noJJ4wJi8\\\\\\\/I5zGuOmbli52PScqCX4DcN3l7G\\\\\\\/ZMWiM+a\\\\\\\/A55n+LQsf7XKu3rnaSPjzrIn1PkYi43QxEM1lREq6+ADMfZsmpuTjllTJ2I2UI5bJm012bV+tFDrdjyKZTG\\\\\\\/inVRZE\\\\\\\/VdINDB6TevkbDh8U0RcMhnDpem20WbJxXZi1VJfZ2SlF0QrsHU\\\\\\\/OlBrxdWVBck9YJf+AmilhMdn1QJrlmZxLdD1ImsTsrvtVy7XvTTQpDf1C8VFBS1MfRW6kVA54LWzY2g0hjd9Gla5IevO4hP\\\\\\\/OR8eJJMJEITYfIuG0vBT9729IS9DjTg9UAkwVnkT8uP233\\\\\\\/t31iO62cUrZ1aNaOWt1bo\\\\\\\/ceUw9c0f2DLiEvPga5C8XE6+cEkUhJ3fPaULqhZr6fDjRCFVjuwISKXq2twrvaeXeF78Tx8OaWk0mXHkbqbADMc+ohJHYMv1AUOPnLsBzlp8Fuy9YVxguZ6Z1WCzx+qLVVjw8ZaHwca47zOczpDZ8FBk93oU5kUpfeUoc3vQagC37uCkVptifk9FBECXqAL2wasJjSjLS7YBV649Qn+2P\\\\\\\/+DlEUyuI+TJI6nnlk4m6Ls+0mVfuI4itWGecAOPOngxAphkpvPINYIPVj9EYrJI1fS0E+yfJiIcUjE\\\\\\\/t19CEDgl1Gt\\\\\\\/tC4WGhcykJz4xGrPvdKTunNTMSWsg5+m7IxbOcJYhb7XJjT5hIHFSrE8\\\\\\\/7aZaYwigop\\\\\\\/eAPIwubAKbQpNEd\\\\\\\/ZCWHL1xFFtWIc8MTnweKJWRig2qxSOZDfJtDpmge+nqPgzCTMprWDj8IcMFHOd5cL37wbbII5TAj6qik05SckUape\\\\\\\/W3OQtB4ZH6K3FH0W\\\\\\\/wCVV9JFpcAB\\\\\\\/KAlx1BWMWPQVmPjXBtovlV2VRKD+TBVot\\\\\\\/qkR28uSQetLA3g\\\\\\\/V7r+Y6Pzodfgm2fmXu7Ur0IUSo3NyZT+mUOmOMaTFHERLgbwEBABDOpwbivdbjitYi9MCRXqkJTgEhfVPyTsIW2axUwZQ5OTkZVNUJW2HH9KgPzYoKZsMd83m6NOMFqC5tJqTfSJi2icTw6nbhc8S\\\\\\\/mfvFHBOzrFiG16fHChDJnEyjaM8+3V8S8Se06S8Bv1itW1eeGTJrXzbUebQoo2WlyNfUWJyHXffNFJbkgoxTTmMrgzDDlXB8wdpllNSrs379E\\\\\\\/bnWpFgtU6hN\\\\\\\/HcTJcPM2OS4YeHqYXzhMAoU2qAkxafngAZb2dVyQlReGl7XK25PaOxiXaCWK7gYG04kZh4mE\\\\\\\/MF4QOn6jZgTkUPUXcEdMC6ECyYAUc946K2jPWke\\\\\\\/vv9twnJT7n4Q6AqME19WWE4iwrQRuIgDi+\\\\\\\/J9tuR4bOzeoz5tSnnJoDDdhXizAMYAKG8cdRmvEyqOKWjK+9ggB5BiWbvJfHElsQkWO7G9jroVh292um4yBx7ZJZ1Jq9Nc3PT4cHKPWFtghcogfNstZ127gv9RoTkQdoEaYunzsDCbC5wRHk4OEPtEY7i9sf7sDSNsUhheRfiSU30U5rY1rD3RIs2Xxl0RwafKJtROOZw0mpSrVrZ\\\\\\\/DrPjcI+N3c0opazfepCRNP5q9G0jLfZYWjgAsuLAb9jb1L5UrZQi3POd+UurVlX1dNCfn5MkQ2xcZSTMbDEvnsYW2QQw3Oy1ynPo0rKFKPjczZBfi1WARXbFI2eumr21yqbSmXyasPxyjQ05CDrtfMr7v+bjxpozJ3Rql4pBTnu7ebrj1uFIN+eXb6hqeMeC80UOCGJ7NjubaBMaFSSSBjtaoKfIH1h5KM+VdhEsJXIMhaPOW4YziuQyCiB6mPh7shelgaK9JRdaESB+daaoyrzceb5\\\\\\\/BOi74\\\\\\\/7y9oLyD41f8izoWzDpzxkfjOKCHBLKVavb9XclGTpaPNu4mJl3ffYxzSVjm6cnfWWH6VA7+uGBxjzcsPYW44t\\\\\\\/SOoAKfgm8j+hx5O0ESQKP9Cff2dhuh2iINzQ8sKyYtUjm5O57KNLAEOUrJIv3uCyY1YYTdX3k7upn0gVxIYRHT3\\\\\\\/hX959tcsdfjZJuSkUjdthHASvSDCTmg6Wn5aKZOXMaodeCRPAvgnzwZNmWIsGd56kwGFRQo\\\\\\\/A6XGT51JeadjWVNP3bCH9EGs0HhnDx1w1h+8evppAiAm7M5oZQ4jKD15d30TCzVnbXIqT33wE6aSIS57BL0OdeKFjFuVM+qEHt4DEkhH1TQGiUKNXZaSfLJSV1Efr82U8yw3cupXFb+bP8cA4sA9y+UgmS1qDGu\\\\\\\/jR7Z61xDh3N9dL3E9bhYg5zmKttFamLkdMtOUZ93KIrdxJNlSLypjf\\\\\\\/TCTm82xL5NpWyN3Sz4nf8M8TRZVjAymgyHBbJtKPSRah\\\\\\\/t8PNHgTPDUU4QiCOUAfH0EQNzS4w34TQ0Dy\\\\\\\/0XIFDJlt4xurJKUQuu9kJDTntIL0NFLhrmK7S27NRdIIF+WylD5esc+Fumr9uMIHPfasm+w4rxrFvTFfTV7xqiMUkdH8IUa7phTInqf7IWTX8DcHEc3v0qO4eeVBQTuwl8SXEYCK+3KOJ\\\\\\\/sckjrAAnXLMB3u\\\\\\\/NkjzTt\\\\\\\/2UB1RliJJXohMhKyxZzMkAZRRtPFlLqA1tA7P7RTyU5RCGZh1ixFXiueiP6d5WRS4efwihqp1Om7iQjejAJIBviwc7vyos6kjW44fpEZYqHiOOYwW12sDQG+5NiqI5MCygE7YwZP5BG5a4R10Kx27\\\\\\\/Bpj9aB7vp55QXLuH0\\\\\\\/P+QZlKUhcBI+UnMEuU9CA+C\\\\\\\/BfT26m9w9fuTJaU9YKESq6Z6\\\\\\\/c4jbNA6hmklgiP5syULcXrQVlI+mc0nD48wncWBciakVJgRShAJjiFXS83C8UFj24vwotFxDs88eo6RGzlwIbOXAfNjgFyAKy0F2glo5p9kbo+DLJOaxddo5dBfk+OKHD9ioo0tX7QrChiSxt3PB\\\\\\\/1LK8AGRhEHNrwUVJUx3aWxmmwzXOq0opcQjx6TnHv+CgjwlaBI5cSoOvUBeVboIqE\\\\\\\/w3umZeD2DwrtGvTVl8GZJDcKYwmTSrVpcEv54b8ymoq61vL3v2vAf2KhpW1jf5YUHQcJWKEcKSDo7cYIbBsLzKyT1FHqhdJuRiC6nIJklb2AR2oc5PM7RJd1q2k4ggwboPUe4aG2BJaupruvgMxAnp8600tlovMlpm8\\\\\\\/cZ+fpPosabwEzh64b7oMYCEQ2xoj8EbytqdyF22l0yvxMsmw+jtAyaHh7FBL8uD7YK5f4JwXGnmd5sNKf8f5qaC\\\\\\\/oRZaWOwLdXwqUW7NJPz5PWJpFIiAwU67RUF06lBY4lxzazdHYBtmsTbz9QF8SggbYzYMSzKVO\\\\\\\/2bmsZeofxMvW7CjaYAg+kJnNr2kdvL6al23B3b7KHGAXqRhs1OfkGiFBqIBANHGoKjyaOl\\\\\\\/iVHmMoN22kY1vd7gU4GSBsf2sGHxp9zKH4YqKZz0wGFKsr0+NfEA2sAyd\\\\\\\/IWZd+hhXQ2M8vwi99F3fV6nUYqyTp+JNUqk6mFkJrxMBy60nLBPjSKHSpF8bpjpWCOSthDuFETLaCwkZdgb2GrdmrnbgnZb78PcVIAw4LKIbNoCM9KBcYolekvSEiGejzzMjosRWzMm1Vts0VQZTL8QYLb3arnhozYWZ55\\\\\\\/SaZA1AhDWE+viVKpo0mXFo84b32Uua3Av+t8YagcVwsgu9rXrcv6wiKkgTbWp6Lw05+2ZybrxtkB6TpfsBPETRc8oiRosFnbEtIaGrB1VL4W0mn2a1RGlgmHBR9oPD4m1GIJO99UJYJqUXi01t94OMmEIRKIjflZg27D3dYdsXmSDbluKFkPhODP+HWKDuvs1SXU51tcTlbvdWP7oCH7zJBNK5M93Y7xtfxwJo8WksNPoUCBKSol6iavHdk9AlFT2M7NTI1L+PpA4dbpao5BZyT+xVSNQ\\\\\\\/Tdlr4Y2vCj76RORr4ccXQjMvoDs1qFkwKdGzxQcPW1yryjC53URkLwm2ivNeTj3Q82cFyvkvlL84445LTKhME+3RnRCgV4caXM9Y5DhjEQipz9GhKxRNvbsF0CD313VAS4yj5Y6KRKYbodrZ0Vyc7pkMOlSfLzy\\\\\\\/nr\\\\\\\/RaVvmC8h0fiydzO5mbTtC6BbSZ8odHV0U1FwvuAKyJdXyY27fRe9U\\\\\\\/+orurTgQwZvVLkE90gAH5kWJKw3PWMbzuA5ToaYGQNJ0JXlZP0DnYfx6cS46rylljesu61+5KvaqKeDfnWBYQzAl+tzDFL0+rEdFn8+kVTxOm8VM8tpyuP5bjYm3JtYuSjzSLRCJ4fsbuiFFv1RON+fYSbGNqhprvg7micm2YeJd1\\\\\\\/o8VkmNZ7tuIi5hlOPIzTKd94PevmnzC7HEERvlV8pQvLwmDPK0SrcxeUTFT0PQ\\\\\\\/Vi2WeMpZ932Bmy2DAwRTe4i+q8OJniG1p0aVoZWouDRfaEmgir41c\\\\\\\/wav1iS7L9q4P\\\\\\\/MSScd+4RnXHXL6+xcwW\\\\\\\/yfR1Yh5yPHQ0NlNXjEqbuy7rUOcDDTsTecasBwUD+WnWi24GxIR0H\\\\\\\/LUwnsgTdYvFXvMg8plGz5iBWaGF3Y4NKZO\\\\\\\/t+rOkuPFz62Wk6ye77WDlVt2qfvicppEn+9BPbKfhSsxqYheRfM87Qu3XkTWjo9hsAnsc7SETMECwv\\\\\\\/OsMdm3MdqBlFl2UFZkshp4\\\\\\\/mZ08OtQLUDkarwt5zKNNIWwrN+LWUNfj4JICrDdtChe+0Lgm5HxKgYgr0PY54vlTmWgoC1dCFThyGktGTWDdQX8tvyac6iiTdczk4xbtOh\\\\\\\/Om8QUcnkjS2OOHqlAM94tvoF0t4XLIctPz8brWgawoSnTzTCNmVl6xaTGNH3GRCdBVYuma3\\\\\\\/lDxAS77VHvkVHp2WHaELY85aRtvSPcSBurE58P8MYECANYfMQcoOynLPhF8wQNJ4Q\\\\\\\/bC7tHdGOvS+S78KIekrVQhMbk\\\\\\\/qqhRKDBPw\\\\\\\/X9TIXGBWg4SuT5huhfKCMnSUrD7CPtVhJiwsR02dl7iNEGy9YXU5IH\\\\\\\/Y5dXqp1T+CU4q6b+pSOgZWl2Nc8xZjow1I\\\\\\\/glD0SVlfKTx44Z75hPn5WiQs+ia2ixjy+Aqyo7IzYDzfHJY1PwYWshGWw+2W5AkFhRXwAoAeBRM0Lb0DPnG\\\\\\\/R7UzDA0sefp9eOgF4+04y4v9zorTX2yFz87uV6Gzerr60sfLPc4404NKgw8NVoHsdC+Y+UC1I0ZFccQKIqVhx1U2JL7AfQAzKrSU6N\\\\\\\/t2yA3VRgU5NupC\\\\\\\/ROwDFrZ0eyKxUlRelQYe7AUteyG2DJOqFyVD6t\\\\\\\/aeQIbBngPocSZ6Kc0xZUJTyawgQUSSy\\\\\\\/4jIYFPVwJV1YguZ1oJsOTwP9t2Q1I2Z7kU2L1zW+Z67vuobgRmcXJ4LCPvB71LSqJIZBgIDgKIdGh0qgkzGH8+VCWHFnDocHimoJFE07bQbYaw2gxHuRtiMMAT6rOZFZCIbDsGSTTY8B\\\\\\\/1dQ21ls7mCJOR4znyIgl2dX+W8NK28oZo\\\\\\\/D3mP7RaVfRcnutFPAjSlRSAUmKIda7eWckZK4qYgUJSu8t3qTf\\\\\\\/+k\\\\\\\/ndNjax21VL2RpNY+2JOlg4lR\\\\\\\/tjrIpfKk0ErVnBlfWM1QHtJkbKJVIwdM1necoH83Ia7bzl5xLD\\\\\\\/mHsGqBzYssYox4BOmwQJwtkJoIIZRxmCLp9ZcU0rHBhm6PYONs1CKHXAK1nBgtXAwO\\\\\\\/W6J7GMhYBCv3I9\\\\\\\/GvEL57\\\\\\\/2UgeWN\\\\\\\/WK8i6Fd4a6EfDlMumfsaALF0R9U0TgLnAGo6LpkxTUSZSuqBwwyhh9sr9CG5sogQFg+XjZBGcA\\\\\\\/q\\\\\\\/ikRnTRoQLsA9RoVrvGAX5r9qUzm0jHiZULo2LOVES8e9JDJ1JnnkURTHP0T+CX9+CIIcmuSGgikl54BrgYY2WUTymrFsKIdf8mFf07RqEifY1f+XRXmu4hcCxMVN6tXkWHagDezPUP9EXvXJ3OR+enbMBOutMx9zaq2l1GIkomNgEhrAQYkZA014uFJw0uu\\\\\\\/XD3PQEQN2lLBB1SkBcfQQvhOxgAUILpYuT4KH\\\\\\\/14Ar3ChgDYAp18kplR7vcEv2tVmg75Gi5cqojvAh1dTViJF5nmcwtPaB6tJ2xJGN\\\\\\\/A4zebU1YRFYbjOKrgTXgnHPwTnRmCwuMM5GDK+5gL\\\\\\\/T2+vZ0BUYWda\\\\\\\/lB5zxOw8lGZM+1T2D\\\\\\\/R5s5gtMp8vubOHAr7xr+wd9kVeKeq11Pjitvqi3eHgRzYArsaElcQFRtjgtDGBtz1iOoZ\\\\\\\/28CSmuy4qD5zNN8XSLRM4jw7kx9Y3h+xJdq0TpSrPbBCqFPT497xkHdcZXNG3ULViaU0IHRLlw0vq9Lqmrql7QKJvM4+TJcjoaxxGtj1LJa+Zv+cYnEJaFk\\\\\\\/ut+7UAZg24HCaOaLJ+hkXH67DoA1vs5+9P9DSJNFPe6PcRP55exnkDEgohfDEcTbFkYttEpBOtb+lK6A9nMo2UyhNnNNJLRtT4nAhtMBueFgq6e8GJg0+eIhWRTW0fWEzEinYyMDlZDsRbXIIsZV5GWCJY3NYaRyNtNftolE9pS5\\\\\\\/Zvmu6Y86qNYxlWFlgdN\\\\\\\/ZU1HNIqxkNE5zP+ZgLBHKRKMmJWAq5+051\\\\\\\/d2lyeUuDICqMm701ExP9CQlimUBRF6mpmpPwLy2TSka5uXtxY1sS4i5+REgVLReJ6+4M\\\\\\\/MNrw1ot3O\\\\\\\/t1IgQoYSNtCOwIyEe77sp7uyTe3ZonFqTRGzIQtrlSof3MGq0KV+D4A2upwdLasY9miWQVTb5ZEjvMJ\\\\\\\/l1L7oixGQe+uynJpz\\\\\\\/HFeCb1JJBQM4wm4N5VaxmjXwS36SNqjvNpPzF5iZ4XBMUwJX6FE1Y8L7nJnKguRQhsII3T\\\\\\\/GJs4wfUaOFOZXN\\\\\\\/k5LxCpvKdSXKmX\\\\\\\/64goas\\\\\\\/G9TMHXcl6PmpyRdFsqHE+3AjN6L9gfbylT80MNaql9EgZTb3z+fAY8GzPthUpPsARcBi9oE3xBW+7n9BvzkG\\\\\\\/BfdJCvvtHwvCqjAa+LMf4gDSNRfkdGjO97wQ8VzJdQFftChbDuLxS521QxCWOygbFXpXl9AqO1vHC3scCs8F9hGi2UtWKxJHNmI7kPFWfNwUn\\\\\\\/7STuYnslGsS4Iivif2AxPQxnfDqycylpHfxlGfGTSU81vyt+AOK1KPoCZhe3V43hf0oSh\\\\\\\/hBQyA2DQZ1l4HQECosI7rwSUSRy8s4nQsHWuGyHKRY+AeRUIBAiQr3LaRKySuig0xnjHLohV+hYavPZHFR8ZngNUbZEVv3Kz6KZs5dqxS2kf9MBeC5GzTqwcyBCcJbfvG6VMXggl4HETjfkltogDjXADUZR4dfmMXgmwNySraa+ekMEsVUI+B6i2AQW\\\\\\\/WZE4kvlvQWwyaYk3gHkxWV8BOX0J17mH80xGf3s0CxAw4KkZVmlOxlbYo35aHwUvd6Bo3waKVRI06YdoWtZRIqlSixMU\\\\\\\/Ut40MrjB1V5aTZt9QBCvsoYvSvgzgMUgQJboTGauLnOKzQWzU3JR49NBFBHt6yUsum09DU5pQNWICRgG+xPa6HJbblRWb9uRFPD2pqd3sOdgiMuQx6fUncwg7H6Ic2YaVLEKQlSQfUq4TP2CULRoVXKaZwRxQKUe6p5fAHK9ko9mgrTf58WZZ9OyNjfJ4nFcDF6GNDc7IEyctLRCrpI4WRWaYDED4Sd8O5o2wVzH4cdGz4yy2MWI\\\\\\\/XdBKVXlgzK5JSykTPyg1L7QMYrRw2JBxRpq6CXRgm1smqgbDWEfus+UBVoJOnrie058Zm328Pa7urYkz3e1V6kjsmdY9WqzScIgXDy6K7N5S9dQgt\\\\\\\/rG1nnT543T0Dj3k+u20tXcTmuLwSI6N8N803U+AMH5LdmKl8+SQZqauzayO7P+CKqgyj85Z+RWNOrW2TPTRJJ9k+84Q9zoyRklFz3zNPONOWLMjDJhdEk3mr1k\\\\\\\/qh4qjkuB6xhH\\\\\\\/n1ltT1d85tlCvkO31T7I6hew0Cj4aNwr5PKDurqgpZPByy6ycQ5d7VMgNRUWn17GcMlS3dyD1prJ0JxnORndesD3QbswS9gHHkVT4ypRU9YvVusx1jikfqlLiY5BRu8qbyMJUBRyAapGOE2RFP7gSLtgL+G9VDsinDpQHpkfYZ6FoqRJAnTaA6KDK6xZTObnILfkPnDXS8ZK88mvW6rzoaeo4fuPVGmW8KVPqfWyPyJhZFQmr3PGvYJaihLDQO3OJ5z7qq3QFXMgmDW5yUNqr\\\\\\\/7Xj+3pW\\\\\\\/W4hR6IyLK6GyJ1zl+DhKfDzA9J6RxL803\\\\\\\/79B7u6pYWMZS+aFDEm+vf8m93f6FfYsRaa9oB36+OuJmZz9iCeiHPjph0NRI+M2Om1dELYOgd0mFTYGmeBcDt4qlldNLX0HEKibcMTUfL6LE\\\\\\\/AOyQMJTCt60AAIuuxVVbsrtiLE0VL1EV\\\\\\\/GbtPYIazhOMkP6O\\\\\\\/tV60ClGNHMwHP\\\\\\\/zHR4IruzP9Pe1Y3OQQ5X5PhPDcq\\\\\\\/0h9XO+3+uPZa65y+O\\\\\\\/sU4waEYtwbq33dChOYGtMtc2d8yfoqT\\\\\\\/cFRu9PsXkyoM5Gyr\\\\\\\/dVk2mHRwL8hsxgQDlTJu2VL\\\\\\\/zNEqfCFNpsBFfJr4Kjx3rTz9vW+gL\\\\\\\/7vL5OyvjTVFBsCxWJgndBNKz9C8\\\\\\\/UcDtiDGHc9y7toU9h08cABgVQRIVO0XsNveR6oFS7igYKWX4I50FNskPb6GTmAb6IUEW57vSpeiP39CMjSEf+TCrugIUnADg\\\\\\\/\\\\\\\/4RJLQjK0pKX42yki1GVbNaVYWqmgXX1p0vayaNrQccxJ2LWbuBFpaUjl2TemOsS5afl4rEXFdVt4vYs654I4GWeFTtXOHAfVK5csjLvUQjAWVtHN26GO8RFxzQ0PEM23XtvYi1X3yxJLSSUsZwls4GKVRulPBYElKsaTAMbjY60FucEv9mkVSZ2fwsdw\\\\\\\/m\\\\\\\/ykcTtCkDSS7py2Ej3vc0vnQlqU9xpu099Wn3D7Dkq7llA4txiqehdeE6SttZrY8d2K4gzzD86d5RBsnTrxapJjoaGLImJl4j4NLQ1fDV1P8FtrP\\\\\\\/ejcsV7xu77UHGzoKI319cwLgdjAJqusjIpRDUSeqyFO8pUQ4hWcTePDl673JWkRGBk56mh+1F+\\\\\\\/5+HiyOwxrhKY2Jwp2wsOeAKLA1hoSkJNRiqfBB0ubAQh6LMTgw4e8TBIdll+e4ofKwjVxXSWJvVB8MZX6YyqBKhG3NC9bsfEjE4SytFuewDttJVjvAu7qWRDxSnaLg+h2TwgKW73fj9cVU+Hwl8Diuf7b5CWoif6SfOeTj9aFhEfm8\\\\\\\/30j0jkTWKQFY5R79invcVF4tyCrv8C66mM6jvcwicAGOsVr1R6Fa6JMeGB0RvRsksZMGDP7UEXoab++setVh8glH0IOYO7EZpGhFMHzB5hGmsepuj9RLQyxS7+DXBM8HLZLS18eEIpNhRJ\\\\\\\/xET6fdRrKRqMwRkUmIxfp8ZQcKBLCZ0hvkTfWIe8vQR3pjYF5DOXx6WGCf8fLnj+PJ41JQqDfgT0\\\\\\\/Zr4b+DY\\\\\\\/ldThcE7Ym5zSbydp6mnziiHJlvIcrRAshL1pQFbvJC9Hi8OT3FHwwcZj4DwgJuDiX9Vu3E7hocq3kYT5kLgLisF397G71BQI+0aX7xp6xG5IggUPir0aqXyZSge5I0K\\\\\\\/\\\\\\\/jKo4R23kXt9bg9eS4S1NlI7pl7\\\\\\\/ufCfmp\\\\\\\/8ST5PcdvSNOWSFgVBkYEXrb3S7Mgt+dOdUTyCOc1GyvAJ37HM1LlUg+IQ942jzWDJoHvsTp+pHlDbDZlkjtKNyzlrpylt2QyxR2FQxuj1xk8JWMkW6Dl5lP5TGnRYkXGfVsedDK\\\\\\\/6CL1sw11nZPU+2a\\\\\\\/zwtBqjOqJWyfQFtfB2jbbkaAC+pEVw3FxKfLYwgJF8sZ9zTvOU11Z8kwrzxejp+cX69oTj5bYkQ1XikQIrjUOMRKI5IZQOPVaTUnsf2dnV+wMaaoJULI3O8c1cKmddWHWPGUTjCQLrPbuWW2wQlYQCPvLTWIVoWB5HQHYqhTQAWQPQlmtLnc\\\\\\\/G9lVfr3dWOWHTff9sLnqrJYlKRQuGvvZ72jWdFTeTSvoIlm8si5r7od4VTpAAk+u1IDYWlbHw4GPjYrmwhHRElZuwpcoop17J\\\\\\\/qElg+QYTtRDxuiPH60xszjxIv85eVcjcPmFKLRVtoxtpLjkKNFy1jKkz9ueyzfpoqlxqZOtLZR0+hXF\\\\\\\/gjX0vPQ\\\\\\\/SQFrjDWuQqK2T9l9zER3Xgeidl4cC84cGK+p2SccQeBrAKFGYV8IedwbNXEBz1emO2+8LK0amNE8ibOE8+Uh4PxMCWfjzblQwnYxVd6DJciIBDQrcpHiCrMD2vL+j0XF3S2JxDYAlPrUUMqHx8lMnRzywOo0QXKFSAYvGVfpp2Rx8gzvlKfTT+StMuxK0BqmpMIQLRvyGZ4cFgyvn3EgQXJtbHRt4qCDfDFSHjfEOk6Ccwtr6GWtBMB4fMtY\\\\\\\/pen4y\\\\\\\/3UBI\\\\\\\/Mb51X4Y4Bn\\\\\\\/sPLaXdngwkb37okNaDCrGkioPyeYmZ0mE7XRnKNBREelTipqmrtqKu+g5SISQ6OLYbuNFXEl1GHMihOKwNgD9nkPyI3At1PDpwH3YC+GL3QL43o4jDy8RO6VKYtPsG6BH6qrxzrLMexfGT3FVYlTw5ZA68byPXOkyzfKtWVEjSHp2vIrs2gV\\\\\\\/iXqgjiJ8nQJcqZqXItwv\\\\\\\/p5HtFsrm1hqDLsf+azq0FT5ce5ULNn2cakBuUgVxb5wwHNJkXZUih9w1jucN2lI+vgIHefUlOrU0nYCFWrkL\\\\\\\/w3ufE75enAsV+tanGUvDCn+EhFUYFtC+zxerqsxNTM0XyKHWjAvRTMRZFL66jTZVMreP880ViDkFv8nxbhJ\\\\\\\/V7S6naY43uR1o4ms9oLW2xzZQVTKxm49Bv\\\\\\\/JdjK6K9jxyciz2BGza\\\\\\\/p4qAtj63bv\\\\\\\/kIgw3M9xvwHfQQ2TyYfm5oTE6NKIThMQIBUwJPVJiZ6zVhWMRCtqz6sytkHoGBGm43wnwi6NTrswJw8lbVvBpHvoWHM8gSISM5\\\\\\\/Jvmjk5zYxjbZWrS2HAGaetTlZhKJVfAc\\\\\\\/DyKCanIg7bscJr52Qy0\\\\\\\/CwjjNX+6XJy7WiMdfv7fIGdpmgH4c0HMwD0xUBIkFn9J+DkjayTOmBsR7ZdCLaItLIQ6uJ61dX3+azs70ntInkQqc3EeO2dkhjavDC2DRmuezsrt2KCy24rixeBE0FRLWDUeUymBXMeV1rZh2p4K7NfhCoFCAehZY4GXkLIM5NHHGRcWn9Ndb9r+72wp6ts0CaSXEUhmHZMUlEqc2dQQNCohsnFqhWOG1lhnTN+aqQDAt57UIp6ayX80tKqtBLvUXsWsegFWZBv1BtDO7aEJVZbBR7qsZY\\\\\\\/PqEgoeTFdxt\\\\\\\/2l6gA3dqfxXhFdbeWdt1RnGFZsMNXtUQxBAzQp36TvNWkVuOdp2eMuJxdaLTllojEdnaFULwuxWSn\\\\\\\/\\\\\\\/OngStnx1MFzBhiTbPmBg1z+1ctIiDy+vVKW1E7oYYqixkFbAvdEK6sZ9t7P\\\\\\\/YCegNetW1MGLOQjulMz3UsoCn4eA\\\\\\\/x26Q37yVOJa4fc5tWewhndMliRUdyT3P+WNnnKYO6Xk24hNFO2vaI1oBbZ06gf1X28zBPAVfZVuDotOhp6kv\\\\\\\/p3aoTPpQbLBAy0LM1QDisgPXR4TcwhvwvNktQ+M\\\\\\\/kIasC\\\\\\\/CZQHVtY31sZkrKwvuLA1hpkowvfzh8BgDpsi1rsBbpF0j4KII5clVUmB8mgM4xiAEA397rUkMCq7DUdJcyY2kWs9Frig0vBxBBjCVgUx5Gul8YH+W22HM7SkLUVlPUxNsfSGwT++w68p2xBsUMArXnNfil9vJSKPn+1tytS9T86ovee4nJF4MqeTzsYYLe735rVNGF+CrSr8qMHsB0hNMnFBtKixJPK9qGQUMsAEtJLh4otGYu\\\\\\\/qpQoFu9qwJ0IzJEsgXVsrhknadZMKZT4O0gIPFN3teuewgWiI9blJcmiRzyvsufQxA8yZ539RMB+Xov0yAtjaCZYK1PrItlfm\\\\\\\/GpcWguxFS1Z\\\\\\\/Iq9VHtSoQyaLsCfrm8Us8gw\\\\\\\/QDMgRE8CVKurNdsdgUsKvuikSVo7Q1BzsnyeEmbW7WPmeKhHBibyUZXSwZ1gDQzEiOoheNX1DHm31E7c19y+Pfwcp88IVdlvxsAQMWoyJlmKBIF49uR1\\\\\\\/XbZSK6SX9fpmTd\\\\\\\/udX6oUca1tU7ran3d4d2q5++9uZAphQPpHXWMdIaiQlhg1f+uomZJTpU216WYqGNMFarmR4mIGb8YDpPhrWLYE+c4JoY9Gn6+lrHxmn1papbze2KuVpnH4TonhZCx1UFZ0iQOULzLCuXoCdNc5wEqnEDpeDAUcX+PuFHLQNPZ5LmMel05vM1AXN7P\\\\\\\/Pt7ihvbgRSBqRE0aQJ+XjUWLNMApVceMqdQ+FmjfyVyDtZXBI2W1px5iZCMP2szuQi6MdxAp+mIPJI9SSm58tNPUt6\\\\\\\/WWMdG9q4rpOXiZfPoNHeT4Xjw+2nWhxPRzQSW99zvtG5mL6\\\\\\\/0tD1ii+EsrNXgOLCMVnrEOc9dStfYsmpAm+ynQMS4lk9LmNhag+BYgWtYsqJLgUkyPFayAwYRR8URN5KrtZ6lYuzuko1JmtRf2hrpRCvJqQe8VzsfPjzSF0LySVV3\\\\\\\/fp7Q0OiTLw281NVHX8vFw5TwF4c4J7PnCitCH0V6IBX8Rcl7HKKHQS3vOgaZWFT535GwoXfCSE+ypBZPqp95g3AP0\\\\\\\/AZfa7uFQxYgzS8+\\\\\\\/AnGdQOEndZlN4bzH7aCpyN6CueggQioUCT5aAG4nxbFxSrzZDxjeA2F5H3HRDkKW0tfF1H70DFnMnquvavmGgBDIYMPkJJLHmKnG\\\\\\\/5HfEsQN\\\\\\\/pNySclBIsUXKCBJCQiOaboIq3vCEeCE1EBPzQmLO8YPZNv1Adv9Zy7TikwZY6sC0G3vV0ZSKLAFTDAv73fCLU3OhRlXNYLiMiOIM+ZYpp7GYvtKwNFntB\\\\\\\/2PlDCK3F3LdSciYC\\\\\\\/UVgryi8RSFMEkQZwsE7hoU\\\\\\\/qOFo+uIHQ+6NP9JD36HU4sjZl1M8L87zJvBG10eeMbkqv+oj597dqtcyMxK8\\\\\\\/nI\\\\\\\/+x2x2yfrjgBNpKgLdSzIErGJVz\\\\\\\/67S65PwVnyvsnK3OM8WBkm6vEZIZmcm5CvscXfZTX6kUQieoEQ1pNfBKLGq0n85kKNZg1Km1MCf0WYBXjbN4oflMpkN5oOPkIrKdWd4aC+m4xmP6xMUbBzlIdnTWdtH2puS3xBlEqrSh3A2xk6rjgZ7\\\\\\\/8dqocSG9hdAzicne4qjc5OauHJbGHC+EDh53GRGJvgt0y+Ljmemqnrc+NlVhMXbvc8A5C7mYe0ddqKU6ZXe8ocYBJRqFqI3xUt7kxLaw9vx7sOAk9gR3wSJZyHV4valZiYkDTrklvsjn7\\\\\\\/GGf2zyNQIDfabKLuG+V95OUKGa6rCPhkRLOLHTx+BSTkn2vs4\\\\\\\/Jw1lokOuxi4dYm\\\\\\\/o9Z+hdM9wbKr2Bhf2rzUpk9WcnDIUz4k34jBnZD+95n0DYUkn\\\\\\\/JLheGbsxO6YdjYhXtR9bx72pJTVu7MEcrtgod0vOz4edQ41YZky7m3ZF5WEueOZIUGZc9hGMUqtEF6zJ5AtD8pz8brZ5w2xULeR03orGYwyZ9+aTE+PnzW0NI5XPsCjJkPHn57\\\\\\\/WPsiWbW2ZTGwlBSbijzu52bltFYWxMFFbeRVi766P5O6ukDlpZ1iK6ezuc+9V\\\\\\\/jkgtUZ6h9lDB5IWlLQacgrx5wYEO8N2lq5YZ22nNHRjiYN1BLLcpD6qmPs6YjnclLNEJB2zMlyVcTbX8WJU7RN6NxWCb+Ja0Bs6LrVeP2qmOCdzoEtj32M6MN6rxFNNehVuoPxnHbGcRvJCVHqNw+qX\\\\\\\/6WS80j1khRF2K3mVHmMSRWnktZxrUkJvWUjoSq1mKpfr\\\\\\\/y6Kwbc4hVdLDs2kvua0UrOu+qUqq3FzFDFD4THsx6Uc07HxgdgvmnAmy5em6UJE0qBGpQuS8m18ywO\\\\\\\/3Tn1UBHQsmOFN55r7k4bOHDOggIirvzQaFUBI88tH3jyYDroHAwz+0GcS9lM3U245h5h4I2IUIEeyunOPVgs\\\\\\\/C0Q3B4MA\\\\\\\/OWqs+OJQnHPOahH0Q7mpUbY9tZTNF3D6SFHjAgxJh1l416s+HFIkgUbJwfXHkVKxWO4ZY6z3FAGU8fZ7UKxPQaW6PGzy4r+AjE1yaFxHfmjF+\\\\\\\/LrNEzx1xN7r2L8cQQXrfwbSaoE2Z30SYUmJcwh+NBYuPjtVVYIrQdamgpv3fDOhSqRrCGyU\\\\\\\/76RJm\\\\\\\/EZw5Eba7VPZb1nn89Cs7Ga5dfBpaWBRqUXJjuiJUzq4lYrWNJ6UCqGjsRcr8RcPA1eQZJOneSsbgSfoQ7zq\\\\\\\/sUVyrLRdIfc90wurAvdXAIr7ekwr9ds4NSIzwg7TUNEZnOnLJVdiNte78t4E7+b3ELlnBXy8sOows256wHDFVXaE6LhAYqWlTo1MCmn1QNO3KaiZuEz2vmTY\\\\\\\/gz9eXZoR20MUYx4S3cyguT7fubDS9tTJ1\\\\\\\/eTALGvbP9eFpsQ7BrndRbEq36LoM+m8Yxv62YFyjvl5O1IfrwOcpEgFOv9CFXZMkifALHFyZsMB+MvNkHIKAN8Ts3sAre3o0xM2\\\\\\\/oGNCRMBiZg0+lG7TgOK\\\\\\\/Ev91BgE7qnvbeRPVpNBwHMMBgRbMLPBscFFJQq\\\\\\\/2fhOhGfy+Zxp0muZc\\\\\\\/\\\\\\\/rkV8q9pdCX6zklT9xyATBeHKJj+81jeFIGKeuX4UezoZkw9uK\\\\\\\/LYgg0xPMyOf8tMwaFB9KqabAlbMHfwNCKhnyMhDb5YURI6DalD4wrAeM+NITte2pz\\\\\\\/ZIGu1SMBxFoDVU\\\\\\\/GfxdUalR11D+pOrPl79tw2OGyuJ7cWkZSv\\\\\\\/AG8p5Kjeq\\\\\\\/5OLos7q8x+4FELWj6d4LjX3f1ONzPZaHWy62MI6BbHDEE+twb7160vZ72OkjMaHa2C+Gql47+wODtF6VKoMuMQdHGyahEZTCeGJLcm0REokp4fN94gAR\\\\\\\/1D3qEotHUUDnEZffuoFh\\\\\\\/uIQ7maiVTTmjR1mBb5XnRt5oF9ycAeQQg9BwRb61pqRLkhQqNKdRFXjirq8i7XyMpG4KZ9GzkqRYYqbPdvx77+S2BE00RBJtvPhI5hEIpTOyQxU8ljcDiK4BYQ8YYSC4+mAj5YcrkKBt4FAgfdNs\\\\\\\/G+8aVX09O4zP2+oXEulj73S85MBCSDdiNytS6L9bj7PGyy7buBjTmq4njSkqoH2FmklBrXg+R5sLNhYgxoUdGz3zKqGJvvfytDs4vXtjkBneqeMQobldrvJWeUAWTv1CUSzDEatqUHQ4XUaCiIsKOtKUK2iu3vMnyK+dJzyoQeqUlmap+CCq\\\\\\\/6FTeRVS2vyR9YM1EW9wbSaupReHOLtpI83K4Z6NRy3pwS12CUeSGFZhaVHRX8lXl17CQsFp5Oqg6ZozM8gwqIlJJhWxB8BEAe+o7fW1f5QpBZv\\\\\\\/5ot0tfCjb8WH9KL5YUEaEAvir+ePR8Wp2ZP5sszU3mpH1D4Ivvynbv0Toa0HrbFW51MtzouVK5tKRLAlEatn4T0X7dQCwOWddGqajzwqwrsO3HZ0pJEeggaWNBjFcdsB4TGmOSESF4KuT4bk2YsEmp+7KTMd2OAbGgjUzpwlGDmJpUKu0s8xZpb3Xi5tjrce9YKLdSYOGI4uffYq8TiE0sTU1suQyY\\\\\\\/XCi9biBivD8xsr4w94M271XxgnSH3GKXZUxI8UPqGgib5H0XP\\\\\\\/tXduegwBgVpIRiscG+6cgkFgWxDG8sXP\\\\\\\/iClTV31QkQ42QJC9IHLqZC7XKSXFCvZM2ww7Tv4JXw0k3ky6MBg5FPpwxIlpYmwADF4kwE0pruoLw1w5D+p5LmJT89XHxywgWpR6J5HeKVechnYDvNEjldnm2oTp6+NL1xpFk6kVaUwmHv65H9mP6R8QSD0gqdXWs5Fx6zvJxbGgzb4MZVBxBj89LHmEr\\\\\\\/sqCQb3mb3JfuX0+KGoLdx7zxSpwDhjj2QaojjzPjVe+zW7DZALTTGnhbfgfS7\\\\\\\/xeChr55QciEeoywTMvFhtPvUuw0D4l0v3Xk\\\\\\\/OgJ+vCjYvSDfYYi3I\\\\\\\/C\\\\\\\/TZmv3w8o\\\\\\\/ORhE14W7JRhT\\\\\\\/PDoYvmsNVAgQl5k+wfvK4ZJD4nWAXemef7CYoNZD+LfLIL7rJS80e9ewUuZetZejh7jtIt+LzhbixjPfJn2kLw9hgZ6opm3sF5EGTlnTmGjszoaEl7TLhGTKUEE4pqrhfSYoI8WXRCBZ2bZsO25GW5yDfuqcJ2JH1tETEghu2gl7vitV+ke\\\\\\\/cYwUJ0b+OKQmTJVil\\\\\\\/lw5O9mdqbwcWtg2xlfZGnhfgyaRMm9cLCUyyjnEJ32cOb1u3mRrNvld+6CdqptbscL1V1zYFFoT7octHstb3jit+RHYly3G8sZVf+8i71i1BWhXChhiY8MkubadT37n3NIhZ4ymOqO5eq7jNIraMbNTnY3h4QMcyi7C+IIqT\\\\\\\/OTeIcYhu6gbXOs6meZ7rg2oQnhXkSarvfl82Hh5hE\\\\\\\/jp00dGSCks8JusjqR9NSoKgBzrnY9K3AF71dnWrXhujZrM6Am7+AJvPlTlqeztP5Wb2UOKTczJ5KzeuVFfdFCIx0P4DxSQOgktN5ydQHxbRWEGACm+WYYy+1BC2XTeMecYCBXyddYuMt8jaIwJZ8lvTcHEkFej3poWAdeYBmC6N0UioxcSygqOWcrOb13dax4oNWC8\\\\\\\/B95q3sfQmbihwuPvnKd5utOJzJkXwGmmXib8D0PG4S2I5QLdqEDwPpVfRi6ho3FeExcP76GGNQWGDt2M3rIc9pbL7+uxCoeGvdwPPOU4JxF1rZ7JdmI5RG22pnQdTuMJPGwpQ7ir\\\\\\\/RvMpQoHlLwRSREo4h8X3XGvnaEHlFFr5YyZo2gZMNAExdIubJNRDSGdpvqelWD5QcnAnVdTZabABG90v6AI34R0J0eruYPsfd8L1VfN8OdqEKe6krseWIL6A1X8CjmF2TYUyFYgS36dmtAmdPqdvmyBpapUMzCOL66sFNk83ZhqZm50R1wYfuT6xNbpcsZKWAVLgCSQ6y85IYXP3Rm7CxAryrjw32Tc0vcIt9peUeN3v3s40y5fnm5Ng3u\\\\\\\/IamBb0C\\\\\\\/mzDzsknGyKAK9htIEJfT4MB2Jy6e2HEg7J4iv9HdHkfO1MQNuTA6SI85hWfE8oNO5SD+SN9CY7GYD0p4TsyV8GMnRYy\\\\\\\/zp\\\\\\\/ZQ8jpUaASplVeVJxZLwuducyQNs5hwXI3a2En+sEbLFTSy95SOzw\\\\\\\/9aG29+EhFFU2brTl65rjU6ZxlhIiLQbibjRn\\\\\\\/mTKoL7NPDbUObyzBKUQTKwBxTw3XJclStqycc0RrLd8Y+nIM8O6uuZ5MfpwLvKmcZG2yDhzvfzwlpZh8htepAS160d1tKVApp\\\\\\\/YXXItOOtqPL3VTOylCavysQBirSKKOZvs3WexHyun1Hazd+idJOltcXolAaBOplR3TLbAeT8daKISj7RLuFClnunaRivELDH6C2\\\\\\\/XILQazs\\\\\\\/R6wHgVofy6hVYBPm\\\\\\\/YjgSt+AS4dzAB+TCjVi6kPeMuWPgAa8T9fhOD0UzyKD4tWmJWKjJ07NrycRA8db1rcPzWpqoajJI5mA2yoMCv++mpjknaW2u8uqd6Hnt8MYkwj3WNQYDOUJjwAVc4c5WEAxZJpdGfcGRbm++DRxltue9QzYUuTMFOF0FvxVsAYlOAqdRoih5IcNxg62nXPx+SLXzOLHyREbGjiwPU28GxgQ6JLhQUbogTZbIYUjXIpSxwY4TCXCWuWckNDm3Cqb8DN227PK1fYVz1QhNqTVVBZwP6MCn5oy0f+zQsB5TyHUYqwR+sdCbfWe7nvro4bzz+S6bwhqV+Ew9+CfHtp5Vrui3+5B\\\\\\\/S20p92dXbPpvbu1V39WMsigvCoEOQv1vgVnRUzPiGXiVpL2nIvgzpiDdy3f1dDcaLgUxihlQ+PmAzMSk2EAnli9Gf1WmZIHH774Q3EeoDICMtKztLPXUIazuCx56MJW83XJnmgHRLJ3ULfla+6lRntSD4pyHDohNsHyJj4jcH3bOW\\\\\\\/0MPvJi6Jo2nXsBudb41spf1SpUyZf1W5IdEkGgXwsO0On3zMU2NlOqt7NmNqesze6+SsiRFpijWZTOXFgxm3tPgE30EuKRq2vGi7bMKFD6wAaUaxRxTj2HRRlyWZNHnQJzuQg+TypWwdiNvnzPhi6FBHpn9GtE0xeC51iCYyoXTGnoeORPK3V8flbcUnBTQ+BxHooye3QarEO3K\\\\\\\/GKIDlsH62KcN7z83I9BfAnsTl+\\\\\\\/IjXhj\\\\\\\/HkDyzvC8IESNVMosxRza7Ekw+5NbmhO5VmyQ0Og\\\\\\\/rDt49DasWVmYKHyYSYmKru3oLGf4nlU2xRpzkH2E+0eem9Kp1yop8bP6NdDEd\\\\\\\/uHOAmlyYKSc0N3GUYpcn2JNnesQW8EZYl8wThAyP3icnvKnqOTI7xCaYRhkBsofcJM2fFoPQgzVlM3Q26Z0cGkLw6whqh10HjANSvNAwYFWk0wwg\\\\\\\/m9AtG9VnV5fXB+qR4EfK4+W0bz+kiSUd\\\\\\\/VW7+1IMkNriRu5jTQz1QeUXfO2l8auNidYPDKyAUWSTiZWcgLmzMGH6D5XfK7YP+nEyKvf6LTNtppF+ymvTcyCdye6bN3b3k00EpZFJaPCUw4mKzDj1PrnWT2v4TUa1itowOWsFQZqJjnO72TLOhss3no58K771gAHjaWe6Uy9rAPHw9\\\\\\\/i8hu34SJcLGFTtOM6VPJH0ZJeGmuDUyfduOpie49d9daNqLC2y8Y7czagY2ET4A\\\\\\\/FrYGgbYcKehwPT0X0mkRdWspnyWgKQl7U6CGhMVIoLQ9se9914t8VBj\\\\\\\/\\\\\\\/MSk2AJoZBQxnkT+KO0dRheevJSpsNcbwYbp93w1Wck1E7M\\\\\\\/U1VHEVTWO3tjyKe0UkZACGSKTnhDljEFMXJWW7aBb9TmcA6RU6z5tHu+Ujd4RZ9XTTxi4D\\\\\\\/hAng7bKSZKu1Xz096lkq+CODrM0vq0dfr1VukHbfC9F8Bd0m8RIJMmAUFWybp9WK1vcbYDpgWCxUvu4l2NESrZJcOs5xM1pOeG5ByOe1riaPHuWjZdbc0CR4JB5mD7GoCmkvm+phKfTkoa+NkN\\\\\\\/0i122nLUZEO0MXMyGTyl9nmOkbOPoE0eXm9Fz4hyxHC7On5n2r8deed11a8pRGTsdf9+BJQWjfrw20zg+ig4FtPP2up7pTnGHlm\\\\\\\/1ofdATW\\\\\\\/wNOcahpAuYydWdxNZsyKfW7q1Z\\\\\\\/5JmMIZJc5Pz96MYxpkts7HE8lc93j0tZmEi\\\\\\\/0k2VqImj3AJNAPOqGTgUtn\\\\\\\/J5ddtSb9lJo7cBCFcERHM5O5PEz+hoijN3A1ZFxxmxmDyi\\\\\\\/KLrCsWNedNdJTEGfVml13K0LsZc6eLycmwRVPtxDG6L5l4G7iAImbgL8QT0Rl7ZdN+kmgpSbeP9hBKyot7sFPGOAWLLd4IqhBbWOautFobrWg9p5k29EiJpa2cUNyYQw7N5C1JY7DFK3aDuzHbPZMOfEH0cFzrC2V5Bo4O+pO58ui5m26i0YRBbgG\\\\\\\/zfOLW\\\\\\\/5tztp+hrRgyBZfzZf2nqdEykH2WX+F7ICMEMB50ezE\\\\\\\/RHNpwjWhOp8M92TxrqDWbMbipNeZA\\\\\\\/nAj6hYUJu9mqgB++\\\\\\\/jVkrwcEjOKjsbl4HC52oEQnn5nzjdDGa3t6xLfka+raJPhIjDoBIugQylyXE11hmhS8wAEkIudra\\\\\\\/s\\\\\\\/gFZshAiKHBcwdlFm76\\\\\\\/zFPMGgS+C4SmKNbrCJre1v3MBIy8aaXmfPv0cs\\\\\\\/h\\\\\\\/m3o7Y2MnLAnyOkMD0o6fJj8gizSF42fvwkw+6ZpxKmtSKfPO9yQsjp3prrX34ukHfPWTPN4uYsHXcLD1qWP0DuHXhrVjq6ghqcltrlueK9reeMOTF42GonRkLSXpULBV9BIPCFesMFxSfQMMudwFE0YCfccVYJD3acVc2wwSYvQ3t27VaPKgIaVgJ6kr5+X8Bwl2ojTZF+VES\\\\\\\/txvaNKsBTH+bAIQCtCnMCztjNRNcZHfujEPJ5k\\\\\\\/q55j834amvrFi8ueLeG9MQI8EvTvjrwY8Zqw2EEVwranBjKkZtg\\\\\\\/9Hs3hVJVL9Wyo\\\\\\\/KXBiO6iH09VRet78cparGEeCfkJFM+2+\\\\\\\/bB7StsWMtTrqyXL9jrj0y0AlSBr9LDd3laNhaDHgdLDSqmxASypqkxK5BQ7Toypz13Gk\\\\\\\/mhQAZP+G1KnvngyopKg97JS\\\\\\\/IubGRHtA1FW+A\\\\\\\/lVtfMS\\\\\\\/RTwVVTSlBpvT+LPXKGzaiCtkDzLodEJTwNbnnpq6Sjv1Mk51yqPLWWjCCrFW1QpG8AmWYnA+q9Wf03f1kfLZgWps27rLGJdREYszQt4mdWp2LsRkzAwGHEAa6HSWU8E9hi3hnL8yGZLXpFbR8KQMDqLMPnceuQOOQO084uQivpF9qsqeBIW\\\\\\\/VNO72O2pM8UP5oRefBcfRzT9SY387ko2p1sv6u6B6creMGdgcjqwaxOvNUBj3N0teJgozndkM1k9cJ31iLqOCFJHJACOQ8k4NCqXvZUKCPqYnlyax0TKGWUrRBEVVuPLhxR5qwU8SfHiyg2WXwlZ+qvTdfN2LZIhlOYfRuvx6wDfYhY4gApLcA4rzqakbjI0Os9HKAoA33zWoJdvRXoMl7IuQ5YFpfLcfoMhPAXFWz7iDKm\\\\\\\/HZnOue4qYzze9vwxpkfnXtuO0GWFmTZ\\\\\\\/23twEisFMJXWVCSFwjDdOtnjTIhcRpJoc3KwtaAK1MQotQIu2OC+7bawLHch+zZPfVrHXqdGKxIDP4fbqo73MpN1lt+el\\\\\\\/v0+TODTynPNVVuWQCuj\\\\\\\/YIMbE1SwdSNSOA1qHyNTtwEHjHYgCYmkofRKl1VVoRD4z+rXynnBHaynNDcRTB8dcJTOwbG2uM3NzTUqXnBXixfxspc0oJBE3qUhpOgoXZANCbqLBByFGPUfPmcGPnGvjua67paQAlqgIpx3K+QBqPg6W5AJ6+EPom6+YAjwKKSdQoGJsZfiDjwtL\\\\\\\/G9TzR4FR4WvfRIunHwDJGbEossYaQ1M9enxXn8OGjamUTY5uBcTmJNUvqL6uH9CuiheUGn+8PGgn6Df0sJ9LJuMGRxbFgN1p21wx9eKdbf9LevSfVzSGnUK79KtPzLuNniXrb7FIQ2rRCVLDRYqgtaWvDxCOp2HrEzWqBWwaY9FciA0vJrRf1aZ528QBOkJVX2mQVd4e7HhZ6bTPAhtY5NiSeHxhDT+dFpt5UuXWNmjlOnrwMethHB5NFm638ejzy+LuCyXunhcLh5c9cEsmgox4lshnnIf370KGg706+x3Nb5TOJbtuLArx\\\\\\\/Ztz10fzxjWmjCTqexOy3rVkATH+w2pB6hMk3fW5+FjoAhWZ8HOvRAPcPgXL2CkCDLilqpXcBFyEvwE7N1tfdRTnMHyztdIRugzkzwjtv9EcF3uT7e\\\\\\\/hGLwninkhP\\\\\\\/4xER\\\\\\\/IubYP48ZLzG67jP8kGe6mKeSr2tlhkFVtuvLD6Ke4vgPlRwdLNCXYmTAyWdgk7m5QGTpEAOSnchRTeSl8kOtpV4YGCbKFU78z96pp2ozu0VXhpsLxALYYjxN17P47\\\\\\\/4b9F4sVu+Q3gXPe1pQm9jvhj15lVf+0un6setZsR1wu+zL0Uongsxqo8GAS412+slFKl7QHqvPb13n6f5Uqp6ni7Z\\\\\\\/jOc0\\\\\\\/1fU2bnwNbGHUE2oM37VOE6h1n4Hu5pL6eKHi0LQy523PhZjNsjnTMxbTXMRbR7qF7X6mPH9ypyj1igf+JCGEaRBhtRJvKXnDmdgEbwYFS1WeeaNCAwHeRRhCSrEanezVQ9iqZdNhe+4kzPHPVHrAZ54RdHWO9E8wRauYf090U8LTgQfEfIaX8oXG0UtLDSDx\\\\\\\/RyYBO7cgIlI818kI4qqaet6uVEUHmjip5ja7ZTq9iXFDqBKwjkST8ijS6iFmshse+dxg8h4OMBtTe+ZFryKyF1fLMizDnFNYYNS3G4rq6iz7b9VfhgqnNgAituqRkiijBsIYU1WjTJzaQpow7rrgPC6O3xk1+v1ayARTq+EMzSLyEwmdU06ipUVADchBXArlSPNpDqMxb+54MSNJ+pxIOlkWlzZrTsqc47rKdQQgoZK15T3R8SrrOdwFnef+SnWF5QisqGkSp\\\\\\\/GuK32rwhYgZloh0Hb1V0vk+sO8wm+NyBHHeKHh6ouv6FN2Zcwafrwtp7e2OC647LAiXYTbJ46Bmh1rIJfm8ha5A7G4sSAz0IrT6AoczizZJkp8nZndU7W2XrIHR2Oup+8QDnQ0RSiodaelUC5KrgYltRTGUL10T1AOfdTiGwZRc7V5Pdz0qCHXDCTiD3PujG3M7sU6Xe2XonFEhNIg2fx3VZBSHUFHe1USlku8ijZ0lTg6w7\\\\\\\/6SMwyPM\\\\\\\/w70qYjmnHX\\\\\\\/JhLjTWWY6UbEm7XTo3eeswPwabwA9FftWRuF99W06xI7za+Z1fXjAqY1WzocyayJQAyuUeSmKId0wsFjo\\\\\\\/rndPUFuyK+w79JVc3v7Q7+JIoFVvl7UgTMULV7lOkJ+XMBxdL22FEBDIXKPi23\\\\\\\/JfE3fw7puMkiQaNS3fNPIaEIyuVUD7J\\\\\\\/6TD4fYFZVhe9WYl8QrStSv+vWW+51BjKi7rwl0x7zD712Ym2ThV4wI8zEh403owqZTMmDvtvQH27xvMJvH9dDih0tbVdiWE+XxPddLoUf8YjE\\\\\\\/GIdqfAvBGUdS6VC1WBY008q3OZNl7pzJkK+cp0w358I2+nf4Qs7gLg9nNDmTgDkBnNgeILebbaMjmBH9F0x31\\\\\\\/o9iTjvK0Tx3uQHOpG95xVrvkkDvR7rin3BLRBh2q7fMYvl7\\\\\\\/I+Q0bsT0AXAyod9+rgVOpIiqpw0D9dUMBnr69Rr2ZVnNCJiEVAMJX0uCrjzurykunID6Yh+qLQ2CYW9YMDcB3cB\\\\\\\/yykDMrZCbrYqOA0TDyi7XnDw+9ZM4kfYecBoliOu1JHOpMiC71aso3B1xsjkZrT\\\\\\\/z\\\\\\\/owLiVS\\\\\\\/5A5FnfiPDBhIEnwao5X0NKt2IBQFZJBYaDBIB53sd1uOS33xaAoYhWTQMa9IH8ak69Gc\\\\\\\/uRThlXzLSJHwZwxUiRlDYzZZY0AEPLbeqZFTXxa8dS26AeVJkkobcVw7fXl8oNckTLeE9Zp75yuDK7P2JU6+UuLJaPdStAbYhK6Ez2bzePzGcZCTWCDx6Fcgf0dxc5sExv7cZfvJXTK0u6SzjbVvUn00NR4tFvx7qRaswmg0U2cBAzmA1gwtZ3zdji3\\\\\\\/On5a4TnKbWYOinMcARZpm34Z0lHjqbYtCN6LLMlWV\\\\\\\/N6y2jNqpPtA9ZRliEfU1+vLz8tE+eLEgWE5gI\\\\\\\/BecWCSUKYZ0SlDI+fln1wFvc0x2C9MHw1hjEOcPQ7To9LygqvgUFZJVThsH4foqPgUie8EvgE7mFQlhqBMbllIF3npOa0bSDyfiTwmJXXJLQFtwAICqLYB2NBy3XVIDvZlFcrpSdtoByx4dAvzpJ8K9ePm2O7ZOm8eLU9R3saxgH\\\\\\\/KQhxblRh7KVzM8KGqrYKAuLSUdGid9f0CWSw4b3Q5woHoob7pMw6KPvg5YScYmGzUhUbah91FCsl44PnDTEvx3+L7DHY4NUF4qGsvtjC+nMrDBc6ZdKeVEE6N6hmzBg7nMrolHdd6A\\\\\\\/lJU7ceQ1iB6S0LSmLvqumBfoUwGU\\\\\\\/7dTtw\\\\\\\/KkKTjZUv6fUjvaWbjGQHWP65\\\\\\\/Lp3\\\\\\\/A8L8GSkDXgR8RJy9E3pI8tvILCU1orMsD+8fs2TXES3pgdrxDyjFSwM9M9qVZm+HUJhOcaX7FGNKf6u4anPpMXJyDx\\\\\\\/87po1fkLhfqWnrsu4s6wJU6DxzZtvx+p+1dElkXjkwVhquZwML5tT1EDUvoGizpCaPDPMV6yji1dc4ZwFoTKwlLidXrWGd7msvTGnORqWM04gGwvFhtRJrPMarIWDXLyVLSTS4fM1z2hEhlgYpkwHbnDgS2UtyWTZQ90V28nEG1zJfmD+amzCnpaKdiww2yx\\\\\\\/DEXebwKbk+fDkvX+yQ8fUCcQW9KuypqGsaDgeZcB7W+FcM9061xrm\\\\\\\/qQFgIEfpF9HH5AF2ONzmkGLjxdZh\\\\\\\/r2x57SZxvkMEV7ufxo1KkkZXJGnj2eLpRrXQt75+jFtu6UG9n+muqBcXK0MtTqmykQ6bDrNF6gxtbDE6n8BlVxqnxMlfbBQg86qrQVeuqWpsiAKUMePvosqcdBHxQ7g\\\\\\\/D0\\\\\\\/5D\\\\\\\/+z+puFOQnEDftWiQGnS5LWgTi8IEgG+rove9wGwjRWOE+Lzab4IWiEU9anJzl4lPF5SuZbE4HOfL7T8CiDXWl81A\\\\\\\/LVaWZhuQJVvyo+owqGgWSJMiCBdMd0\\\\\\\/yPkDNdifghV5ZeZ7RCkjkkVVM3oWvqHdGXScvgcUiLI9Hur\\\\\\\/3VJz\\\\\\\/z9EVckucDbQi+UeXiKl4ZCQpy5HeVXL2CXukUnmU2pbPIGrfXpIgL5AJgwYMxA4Az1Fp6j3DzW\\\\\\\/cCumGZagNDaoA\\\\\\\/Pahb2EqD\\\\\\\/mm7\\\\\\\/3I7Snb0+ypdMdNt3IcEmcuwii+2tB48CqFy4NXPo3aNqkjK4\\\\\\\/iEigKYr0g9Yo\\\\\\\/iNyoWoq7F5UpErLKz4Wg6iRcpUBiPEOyEbNApNcXeokYCQdmmesswXbd8ZVdYkektQl0aZfqtxEhGVq+EeipSl3yVmjUMKNN1nTJ55Qb8mkPENJrgescvRaPNBCAgAsPZMkplhPRk1YZyIYal5GiXazfT2Uur9fIHNzE7FYMHrEHtE2qGWeKdZKO+9eyzdjHiSLThSsM\\\\\\\/yA0vdQbYcak3L0inTPCmOMySSFoeAZ8drmPdI+Ry9mIKxuuKci54GcMyiQKTiGYsyx8DKsSjQlKUjgCQpeZRoLF1uF74a9R18k1OT3f6SXaUnPLI4MpvUzu2khVKrHKe0hgyYaazXNPsHjgbnuChhc7C5dwaY0jwmg4LGbNA5gn9pCNMofrjJI458MyrCBCp8luIH\\\\\\\/AUZwjZYbNpEf6ec+V9mOMK6vTt7dtvfk1Q0rcRwGqJs5lFqRZo7fkQTGdTs1sTOQDkgfYTolI79F7yIwtPWfk+\\\\\\\/2gF9uCJSFrifVuIM7w1KmUwsKe2dsA7eB+TYWcK1ArBxerLa48l42jFZ+a8Ww95n1VcVP32p9mFBE9qZrANbtn1SpsznHd9nIvxL3fa6jIqOgRgHZbx6VceqOjmVlaIUIIHbc0Z1qAdud+Z1+QoaEta2uhYzylHa3CK1h2XvvCFGZHIBX2rKfIM33WvBHAVHIy9M9txBrFXVdau2iET8siZ\\\\\\\/fXVFaCP3bkCTGFn1zXdhWHxuk1m52Tx67OYPB1UCdyrUE35Wc9e1ZuOztMdQR9NrkSmnZhclxzgZ4h6dkmQcWiYf+geaKNdfb3OICCXCes0kdbFwctvCS4+e6uY827KbYpYYTSSSSRgvmERXldukE\\\\\\\/K9\\\\\\\/UvbeXNCs2Wa5BRE9sgXnq4xdAH\\\\\\\/ZUN96rI3MAAyhcjENq062dp3bVXvZpflOQ2PRrw2W4OMrcwlLca9XLvqtTtkAwgTW1BHSGBcamWPfyhPNtIDFnCngzd0WONPF3PJH8fwf6LO0ggdEIrzNcNFmlW7SySsy2zzXdjGjxwIDWpzI\\\\\\\/xomH2rAWy5rcgHW\\\\\\\/8\\\\\\\/AYA7rBtT9lPX\\\\\\\/Jr5sZFCxjpmviPsBg4Z\\\\\\\/baHHz7ynQ5a\\\\\\\/jde9B2LX8NdPmWnJp+NQ7fyTk7d99Vhb4shiL7RP2ruOad4S988NoEXS+Q7dcjrab0E6zYyXOh8suqeuD8EiXFIC9qHoWxFLbIUkQYeizt9F8ldbIUq715pm+c1JNMtEg\\\\\\\/SdBHflvS7s2d3X0caXu7RbHCFvlQDIR0Qw2h8enNLT1Ptt9OcAikh64L57P+AHwTlewalmJe78R9+NzZs0Vz3pjrIcLzEQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vUn1EmqFzMvTkOvwBs0Yjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022917f856857251b9654069695df457dbf0c345291208e332aa7eb73c6c00f0476\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228rXLw+nKDk1YKhNKMRM\\\\\\\/+icZhzT6ZeT\\\\\\\/erM0yp4EKz7nLwkFJ+MNqo4AE672VSCkbA2uUQrONbGIRGs24mfvyYOjKiC\\\\\\\/A8nBgU1BRedn8BXOukQd4v0DwXwo\\\\\\\/JUr6KBVXWLYdl9cyHId7+KDq9ouMUhZFYCca8+rhpUc\\\\\\\/B8H3K2YuOSkLfG0eXrtOe5hTJ57X14sian1Ygs5\\\\\\\/UEtCt3ERxPYoi5CIhF+s3H7KLib57oDbjMY+1myW7MBjfnpiHr0qDqHvszzRwcHb\\\\\\\/t3Ewm1NlPjGNPxI\\\\\\\/19rpY58SZEQ0lZTJxwiNJwW5qfUF9XDi89M7YxM\\\\\\\/AufuTqlGeq97zjBu9llRDYnrbH78JKlPpCO8xwXIRbXp+1\\\\\\\/Y\\\\\\\/7L77Wg691MM1qZGHDQHE9cLWyEa251mXywXaSfnB5A3rgZLe7uqaQ\\\\\\\/KURyRc+iO\\\\\\\/yXHA19LVGvba+69\\\\\\\/PqQpAEZYEtK06kkNrx8pKyr134WL+4AYADCIKBCcPzZ7OCvL0wa3sM91CE73HRi4DH+JlBEtaaWoFngUCNEwjNUUQjSFTgevlSBsEP4SUin0zDxFBRuayQ+SEcMFU7C0suNXnrcxts2aFvly79daFdxs0+daZRpaBhWP7RG8MzzdcNG1BFW70TS3Gfz1k2Q89sT3Gb\\\\\\\/w32Kox5wlCctld9klg50aRi+8NRHkWVLAlgKVWDRgVyYEtG56RxoZKU2xtX9kAZiBJ0DXjpCADVKb6xNcqVigfXti+QSqd2lpQvBm6uzEK3R5N11C391ch4NeBujZiPPk+yHBP5ycDr0lbX8L8ioILeLHDHDtTPybTmv34r6BXpDtLr\\\\\\\/2r+SYf0u1MnLn84j3f\\\\\\\/dAaTep9a4whcAFYUHlKi\\\\\\\/fQqpdtwPwBt4Eh2EmAkdzdYXCTSkqI2jTp05zvg1F9VTJVZMewTNd+qSeUgrrWg1ewYTirXzVJc20Fc+aSW0KIjHV0xvVUd9M730Nk3r\\\\\\\/+AILWjY+1HvuCNJ7kdct6mLz3XJnN4+GQztliQ2ScnudUSJSTtYehMyhA3JDJtTrVmccbivQtK+3rXwyN5hJcvhjxVPfbTIkyvp6q4ppyleUpr9Hcnw\\\\\\\/z757QxQlflxWV3YWI3SpHbUIiTr2XMsxojEKKMalIErATixcdb1B2vjPaGTLyBvWVNfCliluD36GF8dmCDBC9yalzOLvcEkqnbIaz3xbaLBHtnLdnopT\\\\\\\/XStxZLmZeEBmO6tZP+Q4TR3JXs8xnCFdC8Ux3IhOsQ5zKfgRslFPtm2YNbzPRqx\\\\\\\/iDeFTRiG\\\\\\\/cH\\\\\\\/QobqmT\\\\\\\/bRhvUPXYsO+y4Mhy3x2I3fJrOvb53dFaHAbzUNfkRU7d6S3D9t+dLVQMa9tVUL8c421SxothYc9M3sz+EMkn9rJg5GbJ0sDOHbDPyFxlUgZlNCoE9tuQ1EOCBgZ4UAbpt1bjfiNmybeN2gytC0j9HjaaKkF2eM6mt0kXblgy8Gz+GeN2JKXyY5otfTPoRfjpZq4V7zsBjSiQhGHCvQtLQ\\\\\\\/3Cp8QQPbhdHgZ2fNE4JrYXqlBPqccIv6i3ipBayYd3z5nmR84BD77Oukge4IbyrYXpJOsyE0zBPNEmSwt0gi3ZB8I7znSGs8UiLQ\\\\\\\/KqTQN0DIoXP4ByLa7JU7hbKRNy02Ou3ua2NNpfHRTeg49WEsqcz8uTpd4EBhsRZ77HG9w+TSW8b7eT\\\\\\\/nHXcsxZLKnutfgLSPqWa6ASaDXxmUWLFS6YPAmKU5n7+4knSJQto3ymJ\\\\\\\/R7nRHTpLBsoudQk41BmWMhzZs2IKSfLFOsY1wVo2weiEs41yTwuYY\\\\\\\/xRHsc4wEp7fpS06yg+oCEgozDXOY6kXJIikzPdm79LYIT79hIAuxG7P6AqEjICs7tcstmiEFyoi77eN75wY1YfID+mYeY39m+8L64EkkVuzsoKASoV0FH62Kq4QVsVN6ZadQYXm8dfLO6PMoFSxucJAXTdAz+epHqwWeu42aEWvFiBFYKWeAJ7H\\\\\\\/L\\\\\\\/WfwAfRkRHfbHS8m9Cu7J7cbA==\\\u0022,\\\u0022iv\\\u0022:\\\u00220iD3fRFBYu35V2C3dMIMsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e3870f82a3e7781f72b64e114e12700cffdc455f7e33e3aa1a2029f43587bbaf\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459388 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/1/23" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Vv477kqG5VxEQzfyuKRsEaGvhHcRq6uuxoUWsNN8g98" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/1/23" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RjCHUF2EPS2WHNc-ZAatpDk-e3gfZmoxfo-z7HIJp7Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/1/9" title="Genetic Analysis of mlh3 Mutations Reveals Interactions Between Crossover Promoting Factors During Meiosis in Baker’s Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/1/31" title="Toward a Molecular Cytogenetic Map for Cultivated Sunflower (Helianthus annuus L.) by Landed BAC/BIBAC Clones" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459388" id="node459388" data-pisa="ggg;3/1/23" data-pisa-master="ggg;3/1/23" data-apath="/ggg/3/1/23.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lei Jiang</span>, <span class="highwire-citation-author" data-delta="1">Dana Willner</span>, <span class="highwire-citation-author" data-delta="2">Patrick Danoy</span>, <span class="highwire-citation-author" data-delta="3">Huji Xu</span> and <span class="highwire-citation-author" data-delta="4">Matthew A. Brown</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">23-29;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004069 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459388" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lei Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALei%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dana Willner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Brisbane, Australia 4072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BWillner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willner%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BWillner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick Danoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BDanoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Danoy%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BDanoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huji Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huji%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHuji%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xuhuji@smmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xuhuji@smmu.edu.cn</a>
<a href="mailto:matt.brown@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">matt.brown@uq.edu.au</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew A. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BA.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BA.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xuhuji@smmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xuhuji@smmu.edu.cn</a>
<a href="mailto:matt.brown@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">matt.brown@uq.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/1/23" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459388" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/1/23.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459388" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/1/23.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459388" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/1/23.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459388" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/1/23.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459388" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67417 first last"><a href="/content/3/4/781" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - April 01, 2013</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Most genome-wide association studies to date have been performed in populations of European descent, but there is increasing interest in expanding these studies to other populations. The performance of genotyping chips in Asian populations is not well established. Therefore, we sought to test the performance of widely used fixed-marker, genome-wide association studies chips in the Han Chinese population. Non-HapMap Chinese samples (n = 396) were genotyped using the Illumina OmniExpress and Affymetrix 6.0 platforms, whereas a subset also were genotyped using the Immunochip. Genotyped markers from the Affymetrix 6.0 and Illumina OmniExpress were used for full genome imputation based on the HapMap 2 JPT+CHB (Japanese from Tokyo, Japan and Chinese from Beijing, China) reference panel. The concordance between markers genotypes for the three platforms was very high whether directly genotyped or genotyped and imputed single nucleotide polymorphisms (SNPs; &gt;99.8% for directly genotyped and &gt;99.5% for genotyped and imputed SNPs, respectively) were compared. The OmniExpress chip data enabled more SNPs to be imputed, particularly SNPs with minor allele frequency &gt;5%. The OmniExpress chip achieved better coverage of HapMap SNPs than the Affymetrix 6.0 chip (73.6% <em>vs.</em> 65.9%, respectively, for minor allele frequency &gt;5%). The Affymetrix 6.0 and Illumina OmniExpress chip have similar genotyping accuracy and provide similar accuracy of imputed SNPs. The OmniExpress chip however provides better coverage of Asian HapMap SNPs, although its coverage of HapMap SNPs is moderate.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/affymetrix-60" class="hw-term hw-article-keyword hw-article-keyword-affymetrix-60" rel="nofollow">Affymetrix 6.0</a></li><li class="kwd"><a href="/keyword/illumina-omniexpress" class="hw-term hw-article-keyword hw-article-keyword-illumina-omniexpress" rel="nofollow">Illumina OmniExpress</a></li><li class="kwd"><a href="/keyword/immunochip" class="hw-term hw-article-keyword hw-article-keyword-immunochip" rel="nofollow">Immunochip</a></li><li class="kwd"><a href="/keyword/genetic-polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-genetic-polymorphisms" rel="nofollow">genetic polymorphisms</a></li></ul><p id="p-5">Genetic polymorphism is known to contribute to phenotypic variation, disease risks, and an individual’s response to pharmaceuticals and the environment. For the past 20 years, genetic linkage combined with positional cloning has achieved tremendous success for mapping the variations that underlie monogenic Mendelian diseases (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Jimenez-Sanchez <em>et al.</em> 2001</a>). However, it was only with the development of the genome-wide association study (GWAS) approach that significant progress was made mapping disease-associated loci in common complex human diseases (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2007</a>). Since the advent of the GWAS era with the publication of the Wellcome Trust Case Control Consortium studies in 2007 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">The Wellcome Trust Case Control Consortium 2007</a>), GWAS has become a routinely used tool to identify common, low-risk variants associated with or causative of a wide variety of human diseases. This has led to the robust identification of more than 500 loci associated with common human diseases, representing a huge leap in our understanding of the aetiopathogenesis of human disease (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Hindorff <em>et al.</em> 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Manolio <em>et al.</em> 2009</a>).</p><p id="p-6">The GWAS approach has matured considerably since its introduction. The key technical advances have come about through refinement of single-nucleotide polymorphism (SNP) selection on chips, marked increases in SNP density, the development of copy number variant tagging approaches, and improvements in chip throughput and reduction in per SNP genotyping costs. At the moment, Illumina (<a href="http://www.illumina.com">http://www.illumina.com</a>) and Affymetrix (<a href="http://www.affymetrix.com">http://www.affymetrix.com</a>) microarrays are the two most popular microarray platforms in the GWAS research area. The ability to identify associations depends heavily on the coverage of the genotyping chip used (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barrett and Cardon 2006</a>). This coverage varies significantly between major ethnic groups and can be assessed <em>in silico</em> using previously genotyped samples of known ethnicity, typically from the HapMap study (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Barrett and Cardon 2006</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">The International Hapmap Consortium 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Pei <em>et al.</em> 2008</a>). This approach does not take into account real-world factors, notably the genotyping success rate of the different platforms and chips.</p><p id="p-7">To date, most GWAS studies have been performed in populations of European descent. Interest in gene-mapping in Asian and African populations in particular has increased recently, driven at least partly by the fact that the so-called “low-hanging fruit” already have been identified for most common human diseases and the increasing evidence of specificity of a significant number of genetic associations to individual ethnic groups (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cai <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carty <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Noguchi <em>et al</em>. 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Smith <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Yoon <em>et al.</em> 2012</a>). This is particularly the case in East Asia, where large case collections have already been established, or where high-quality health services in large populations enable the establishment of suitable case collections for GWAS studies. Transethnic mapping, or comparing association findings in ethnically remote populations, can be highly informative about established loci in addition to identifying novel loci (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hughes <em>et al.</em> 2011</a>). When disease associations are shared (with the same polymorphisms) between ethnically remote groups, this implies that the same causal variant is in linkage disequilibrium with the genotyped variant in each population, indicating a common founder mutation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Franceschini <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Twee-Hee Ong <em>et al.</em> 2012</a>).</p><p id="p-8">Here, we test the coverage of two commonly used genotyping chips, the Affymetrix SNP Array 6.0 (abbreviated as Affymetrix 6.0) and Illumina Human OmniExpress Bead Chip (abbreviated as OmniExpress), in a Han Chinese population (n = 396). The Affymetrix 6.0 is designed to genotype more than 906,600 SNPs and 946,000 copy number variants. The OmniExpress is an Illumina Infinium HD BeadChip, which is designed to genotype 733,202 markers per sample. A subset of individuals also was genotyped using the Immunochip, an Illumina Infinium chip containing 196,524 polymorphisms for known immunogenetic loci (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cortes and Brown 2011</a>). Chip performance was assessed in relation to data quality and genomic coverage, both for directly genotyped and imputed SNPs. Genotype concordance was evaluated between the Affymetrix 6.0 and OmniExpress, as well as between these two platforms and a third genotyping platform, the Immunochip.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Study samples</h3><p id="p-9">This study was approved by the research ethics committee of the Second Military Medical University, China. Blood samples were obtained from patients attending outpatient clinics in Changzheng Hospital, Shanghai, China. All the study subjects provided signed informed consent. Subjects genotyped with the Affymetrix 6.0 platform were a subset of a larger cohort of non-HapMap Han Chinese individuals at Changzheng Hospital, which included rheumatoid arthritis cases (n = 51) and controls (n = 188). Only control subjects were genotyped further using OmniExpress or Immunochip. Genomic DNA was extracted from peripheral blood leukocytes using the AxyPrep Blood Genomic DNA Miniprep Kit (Axygen, Union City, CA) according to the manufacturer’s instructions. All genomic DNA was resuspended in TE Buffer for the following study.</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-10">The concentration of genomic DNA was measured using the Quant-iT PicoGreen dsDNA assay (Invitrogen, Carlsbad, CA), and concentrations were standardized to 50 ng/μL for genotyping. All samples were genotyped initially using the Illumina OmniExpress (Illumina Inc., San Diego, CA) and Affymetrix 6.0 (Affymetrix Inc., Santa Clara, CA) chips. Correlation of genotypes was then checked with a further Illumina Infinium chip, the Immunochip (Illumina Inc.), which contains a mixture of common and rare variants (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cortes and Brown 2011</a>). All platforms were operated according to the manufacturer’s instructions.</p></div><div id="sec-4" class="subsection"><h3>Genotype clustering and quality control</h3><p id="p-11">Illumina genotype clustering was performed using Illumina’s BeadStudio software (Illumina Inc.). SNPs were reclustered using the study samples, low quality subjects were removed, and a subset of clustering results were manually inspected and verified. All SNPs with GenCall scores less than 0.15 were excluded. For Affymetrix chips, clustering was performed using the program Birdseed version 2, as implemented using Affymetrix Genotyper Console v4.0 (Affymetrix Inc.), using the default quality control (QC) thresholds. Population stratification was assessed with Eigenstrat and SmartPCA (both available from <a href="http://genepath.med.harvard.edu/%E2%88%BCreich/Software.htm">http://genepath.med.harvard.edu/∼reich/Software.htm</a>) using the HapMap phase 3 Release 27 data (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Price <em>et al.</em> 2006</a>). This identified three ethnic outliers, which were removed before further analysis. Genotyped SNPs were evaluated using PLINK version 1.07 (<a href="http://pngu.mgh.harvard.edu/%E2%88%BCpurcell/plink/">pngu.mgh.harvard.edu/∼purcell/plink/</a>), and markers with call rates less than 95%, minor allele frequency (MAF) less than 1%, and Hardy-Weinberg Equilibrium <em>P</em>-values less than 10<sup>−6</sup> as were excluded from further analysis (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Purcell <em>et al.</em> 2007</a>). Genotyping data in PLINK format has been deposited at the European Genome Phenome Archive (EGA, Genome-Phenome <a href="http://www.ebi.ac.uk/ega/">http://www.ebi.ac.uk/ega/</a>) which is hosted at the European Bionformatics Institute, under accession number EGAS00000000131.</p></div><div id="sec-5" class="subsection"><h3>Imputation</h3><p id="p-12">Imputations were carried out using MACH (<a href="http://www.sph.umich.edu/csg/abecasis/MACH">www.sph.umich.edu/csg/abecasis/MACH</a>) with 50 rounds of model building (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li <em>et al.</em> 2010</a>). The phase 2 JPT+CHB (Japanese from Tokyo, Japan and Chinese from Beijing, China) HapMap Release 27 data were used as the reference panel. Imputation quality was assessed using the R<sup>2</sup> metric. Imputed SNPs with R<sup>2</sup> less than 0.3 were considered to be of poor quality and were excluded from analysis of imputation efficacy and genotyping concordance. MACH output files were converted to PLINK format using the free software GenGen [<a href="http://www.openbioinformatics.org/gengen/">www.openbioinformatics.org/gengen/</a> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2007</a>)]. Coverage was calculated as the number of genotyped plus imputed SNPs (r<sup>2</sup> &gt; 0.8) divided by the number of genotyped HapMap markers for the JPT+CHB reference panel at a given MAF. Corrected coverage was calculated using the formula presented in <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Barrett and Cardon (2006)</a>, which adjusts coverage estimates for the total number of polymorphisms in the genome.</p></div><div id="sec-6" class="subsection"><h3>Concordance analysis</h3><p id="p-13">Genotype concordance was analyzed using PLINK version 1.07 with the merge function in merge-mode 7, which compares concordance ignoring missing genotypes (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Purcell <em>et al.</em> 2007</a>). Prior to comparison SNPs genotyped on different platforms were adjusted for strand differences and allele coding to allow for accurate assessment of homozygote to homozygote miscalls as discussed in Results. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the number of individuals genotyped by each pair of platforms, and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the number of shared markers. Types of discordance, MAF, and R<sup>2</sup> values for each SNP were extracted from PLINK and GenGen files using Perl scripts. Correlations between concordance, MAF, R<sup>2</sup> were analyzed using the R function cor.test, and scatterplots were made using the R plot function (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">R Development Core Team 2005</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/23/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Study sample size. The total number of individuals genotyped by each platform is indicated under the platform name. Sample sizes in overlap regions represent the number of study subjects genotyped by two or more platforms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1078342866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Study sample size. The total number of individuals genotyped by each platform is indicated under the platform name. Sample sizes in overlap regions represent the number of study subjects genotyped by two or more platforms.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/23/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/23/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/23/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/23/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462260" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-14" class="first-child">Study sample size. The total number of individuals genotyped by each platform is indicated under the platform name. Sample sizes in overlap regions represent the number of study subjects genotyped by two or more platforms.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462271/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462271/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Concordance between genotyped SNPs</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Genotyping success and concordance across chips</h3><p id="p-16">A total of 396 subjects were genotyped, with subsets subjects genotyped on one, two, or all three platforms (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). For each array type, data were first subjected to QC to remove SNPs with low call rates and MAF less than 1%. For the Affymetrix 6.0 array, 582,284 SNPs (62.28% of total markers) passed QC and had MAF greater than 1% for each array. The number of SNPs that were successfully genotyped by the OmniExpress chip was 593,582 (81.19% of total markers), which was slightly greater than for Affymetrix 6.0. For the Immunochip, 82,084 markers had MAF less than 0.01, and an additional 2,388 failed QC, leaving 112,112 (57.04% of total markers) genotyped markers. Genotype concordance between chip types was very high for all pairs of platforms; dually genotyped SNPs were more than 99.8% concordant in all cases (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). No significant differences were noted between pairwise concordance, indicating that for genotypes that passed QC, genotyping accuracy did not vary substantially between the different platforms.</p></div><div id="sec-9" class="subsection"><h3>Imputation efficacy</h3><p id="p-17">Information on imputation efficacy for HapMap SNPs using genotyped Affymetrix 6.0 and OmniExpress markers is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004069/-/DC1/TableS1.pdf">Table S1</a>. The number of imputable SNPs refers to the number of SNPs in the HapMap II CHB+JPT data set for a particular chromosome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004069/-/DC1/TableS1.pdf">Table S1</a>A and B, second column). Imputation efficacy was initially evaluated using the MACH default value of r<sup>2</sup> =0.3 as in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Nothnagel <em>et al.</em> (2009)</a>. Overall, imputations were more efficacious using the OmniExpress chip, with 4.59% of SNPs failing QC <em>vs.</em> 7.10% for the Affymetrix 6.0. For both data sets, there was a high prevalence of SNPs with MAF less than 1%, but this was greater for the Affymetrix 6.0 chip (8.42%) than the OmniExpress (4.72%).</p></div><div id="sec-10" class="subsection"><h3>Coverage of HapMap SNPs</h3><p id="p-18">Imputed SNPs also were evaluated at r<sup>2</sup> = 0.80 to calculate coverage of HapMap CHB+JPT markers (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The HapMap Phase II CHB+JPT panel contains 2,133,507 markers with MAF greater than 5%. Using the OmniExpress platform, 25.3% of these were genotyped, whereas 52.9% were successfully imputed, giving a naïve coverage rate of 78.2%. Using Barrett and Cardon’s corrected formula for coverage with an estimate of 7.5 million SNPs in the genome, we calculated coverage as 73.0% (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Barrett and Cardon 2006</a>), which is much lower than the 91% coverage for markers with MAF greater than 5% reported by Illumina for JPT+CHB populations (<a href="http://www.illumina.com/products/human_omni_express.ilmn">www.illumina.com/products/human_omni_express.ilmn</a>). Corresponding coverage estimates for HapMap SNPs with MAF greater than 1% were 76.1% (uncorrected) and 71.1% (corrected; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Using the Affymetrix platform, we found that the empirical coverages for SNPs with MAF greater than 5% were 66.6% (uncorrected) and 59.2% (corrected), and for MAF greater than 1% were 66.4% (uncorrected) and 57.3% (corrected). This observed coverage was much lower than the theoretical coverage of 84% for JPT+CHB reported in <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li <em>et al.</em> (2008)</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462297/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462297/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Coverage of JPT+CHB HapMap 2 SNPs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Concordance of imputed SNPs</h3><p id="p-20">The number of imputable SNPs, <em>i.e.</em>, the number of SNPs for one platform that can be imputed using a second platform, was calculated as the number of unshared (not dually genotyped) markers with phasing information in the HapMap II CHB+JPT set (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Imputation efficacy between platforms corresponds to the percentage of imputable SNPs which passed imputation QC (r<sup>2</sup> &gt; 0.3, MAF &gt; 0.1, call rate &gt; 0.95; <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The OmniExpress platform was more successful at imputing Affymetrix 6.0 markers than vice versa; however, both platforms had similar efficacy for imputation of Immunochip SNPs.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462329/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462329/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Imputable SNPs for each array</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462292/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462292/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Concordance and types of discordance in dually genotyped or genotyped and imputed SNPs</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Concordance between imputed and genotyped SNPs for pairs of array platforms was very similar to concordance between pairs of genotyped SNPs (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Concordance rates exceeded 99.5% in all cases, and no differences were noted in the concordance rates for different pairs of genotyping arrays. This finding suggests that imputation is sufficiently robust to permit combining imputed and genotyped SNPs in studies using both Affymetrix 6.0 and OmniExpress arrays.</p><p id="p-24">Discordance between imputed and genotyped markers was classified into three categories: (1) a heterozygote mis-called as a homozygote, (2) a homozygote mis-called as a heterozygote, and (3) a homozygote called as the opposite homozygote. For all chip types, homozygote mis-calls as heterozygotes were more common than the converse (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Homozygote-to-homozygote mis-calls were uncommon with all chip types, comprising less than 5% of miscalls in all cases, and occurred slightly less frequently in the imputed OmniExpress data than the imputed Affymetrix 6.0 data (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). Interestingly, homozygote mis-calls were noted between the Illumina OmniExpress and Immunochip arrays, which use the same strand designation system (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>).</p></div><div id="sec-12" class="subsection"><h3>Correlation between concordance, MAF, and R<sup>2</sup></h3><p id="p-25">Concordance was analyzed on a per-marker basis to explore correlations with MAF and R<sup>2</sup>, as well as to determine the types of discordance present. It has been previously reported that imputation accuracy is positively correlated with MAF, which implies that concordance between genotyped and imputed markers also should increase with increasing MAF (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Pei <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Marchini and Howie 2010</a>). Here, we observed small but statistically significant positive correlations between MAF and concordance. A statistically significant but negligible correlation was observed between concordance and both MAF (Pearson’s r = 0.07, <em>P</em> &lt; 0.001) and R<sup>2</sup> (r = −0.09, <em>P</em> &lt; 0.001) were detected when comparing the imputed OmniExpress SNPs to the Affymetrix 6.0 genotyped SNPs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, top panels). Similarly, a negligible correlation was observed between concordance and MAF (r = −0.04, <em>P</em> &lt; 0.001) for the imputed Affymetrix 6.0 markers compared with OmniExpress genotypes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, bottom panels). The correlation between R<sup>2</sup> and concordance was greater (r = −0.12, <em>P</em> &lt; 0.001), but still quite low.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/1/23/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance vs. MAF and R2 for genotyping platforms. Each data point represents a genetic marker genotyped on one platform and imputed by the other. Concordance represents the proportion of subjects for which the directly obtained and imputed genotypes were the same, and MAF and R2 were calculated using PLINK." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1078342866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Concordance &lt;em&gt;vs.&lt;/em&gt; MAF and R&lt;sup&gt;2&lt;/sup&gt; for genotyping platforms. Each data point represents a genetic marker genotyped on one platform and imputed by the other. Concordance represents the proportion of subjects for which the directly obtained and imputed genotypes were the same, and MAF and R&lt;sup&gt;2&lt;/sup&gt; were calculated using PLINK.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/1/23/F2.medium.gif" width="440" height="378"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/1/23/F2.medium.gif" width="440" height="378"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/1/23/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/1/23/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462248" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-26" class="first-child">Concordance <em>vs.</em> MAF and R<sup>2</sup> for genotyping platforms. Each data point represents a genetic marker genotyped on one platform and imputed by the other. Concordance represents the proportion of subjects for which the directly obtained and imputed genotypes were the same, and MAF and R<sup>2</sup> were calculated using PLINK.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-27">In this study we have compared the real-world performance of two widely used genotyping platforms, Affymetrix 6.0 and Illumina OmniExpress, in a non-HapMap Han Chinese population. Concordance between genotyped markers across the Affymetrix and Illumina platforms, including the Immunochip, was high, with no significant differences noted between different pairs of chips. Nothnagel <em>et al.</em> showed similarly high rates of concordance between the Affymetrix 6.0 and Illumina 550k platforms for genotyping in a German population (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Nothnagel <em>et al.</em> 2009</a>). Concordance of imputed with directly genotyped SNPs was slightly lower than between genotyped SNPs alone, but the level of discordance was still quite low. Although this discordance is minor, care must be taken when combining information from different genotyping platforms for cases and controls because platform-specific differences can lead to spurious or inflated associations (Sinnott and Kraft 2011). Low MAF had little practical impact on the concordance or imputation quality as assessed by R<sup>2</sup>, suggesting that these GWAS chips, although not specifically designed to capture rare variants, may actually capture a significant fraction of genetic variation due to low frequency variants.</p><p id="p-28">Although discordance between both directly genotyped and imputed and genotyped SNPs was low, we noted differences in the type of discordance, with a greater frequency of homozygote genotypes being imputed as heterozygotes than the converse. A slightly greater error rate was observed when comparing imputed data from Affymetrix 6.0 chips with either the OmniExpress or Immunochip data, than with genotypes imputed from OmniExpress compared with directly genotyped Affymetrix 6.0 or Immunochip SNPs. This finding suggests that the imputation accuracy from Affymetrix SNPs is lower than from OmniExpress, although the difference was small. Imputation accuracy was lower on average for Affymetrix platforms as compared to Illumina platforms for CHB+JPT populations in a previous study by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Li <em>et al.</em> (2008)</a>.</p><p id="p-29">Significant differences were noted in the ability to impute from the OmniExpress and Affymetrix 6.0, and imputation efficacy using MACH was much lower in general than previously reported [16]. Many imputed SNPs were not considered for further analysis due to low genotyping rates, poor QC values, and MAF less than 1% in the target panel (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Xu <em>et al.</em> (2009)</a> demonstrated the presence of significant substructure in Han Chinese populations, with notable differences between Northern and Southern Han subpopulations. The HapMap CHB+JPT reference panel includes genotypes from Northern Han (Beijing) and Japanese individuals, while our study population was from Shanghai, which is in the Southern Han region. Some markers which are polymorphic in the JPT+CHB reference panel with MAF greater than 1% may be less polymorphic or monomorphic in the Han Chinese population studied here, contributing to the reduced imputation efficacy. Greater imputation efficacy might be achieved by the use of mixed reference panels as described in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Huang <em>et al.</em> (2009)</a>.</p><p id="p-30">Using HapMap data, Illumina reports that OmniExpress chips capture around 91% SNPs with MAF &gt; 5% and r<sup>2</sup> &gt; 0.8 in CEU (Utah residents with European ancestry from the Centre du Etude Polymorphisme Humain collection) populations, with similarly good performance also in CHB+JPT (91% in HapMap). In our study, coverage was much lower for the OmniExpress platform (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). This decreased coverage was largely attributable to imputation efficacy as described previously. Coverage estimates are known to be inflated when coverage is calculated using on the reference panel from which tagSNPs were selected during the chip design process (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Barrett and Cardon 2006</a>). This results in overfitting, which in combination with the small sample size of the HapMap data set can lead to exaggerated coverage estimates (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Hao <em>et al.</em> 2008</a>). Markers for the Illumina OmniExpress platform were selected from HapMap SNPs representative of common variants (MAF &gt; 5%) which may also resulted in overestimation of coverage (<a href="http://www.illumina.com/products/human_omni_express.ilmn">www.illumina.com/products/human_omni_express.ilmn</a>). The use of more specialized reference panels with larger sample sizes, and genotyping platforms tailored to East Asian populations could improve coverage in future studies.</p><p id="p-31">Coverage for the Affymetrix 6.0 chip was lower than for the OmniExpress because a greater proportion of HapMap SNPs could be imputed using OmniExpress. Lower coverage of CEU, JPT+CHB, and YRI populations by Affymetrix arrays as compared with Illumina arrays has previously been reported and may be related to differences in SNP selection between the two platforms (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mägi <em>et al.</em> 2007</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2008</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">2010</a>). The Affymetrix microarray technology relies on the differential hybridization of genomic DNA to 25-mer probes which match SNP alleles, while the Illumina Infinium technology uses hybridization followed by primer extension (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kennedy <em>et al.</em> 2003</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Steemers <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ragoussis 2009</a>). Although Illumina emphasizes tagging SNPs, SNP selection for the Affymetrix system is limited by technical constraints. For example, SNPs chosen for the Affymetrix 6.0 assay must be located within fragments generated by a restriction digest by the enzymes Nsp I and Sty I (<a href="http://www.affymetrix.com">http://www.affymetrix.com</a>), and are then required to ameliorate to universal hybridization conditions for adaptor ligation and subsequent inclusion on the array (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ding and Jin 2009</a>).</p><p id="p-32">This study demonstrates that, in a real-world setting, both the Affymetrix 6.0 and OmniExpress chips achieve good coverage of genetic variation in Japanese and Chinese as defined by HapMap. From our results, we can conclude that despite the different mechanisms in chemistry, the Affymetrix 6.0 and OmniExpress platforms both give good call rates and similar genotype accuracy, in comparison with the Illumina Infinium Immunochip genotypes. Further, the imputation accuracy comparing SNPs imputed on one platform with SNPs genotyped on another platform was high, indicating that, with appropriate quality control, it is valid to combine imputed and genotyped SNPs in studies where samples have been genotyped either on the Affymetrix 6.0 or OmniExpress chips.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">The authors would like to thank all participating rheumatoid arthritis cases and healthy individuals who provided the DNA and clinical information necessary for this study. H.X. was funded by National Natural Science Foundation of China grants (30872339 and 81020108029) and Science and Technology Commission of Shanghai Municipality grants (10JC1418400 and 08XD1400400). M.A.B. was funded by a National Health and Medical Research Council (Australia) Senior Principal Research Fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004069/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004069/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-34"><em>Communicating editor: J. E. Richards</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-08-08"><span class="received-label">Received </span>August 8, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-10-31"><span class="accepted-label">Accepted </span>October 31, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Jiang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.1" data-doi="10.1038/ng1801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluating coverage of genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D659%26rft.epage%253D662%26rft.atitle%253DEvaluating%2Bcoverage%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1801%26rft_id%253Dinfo%253Apmid%252F16715099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16715099&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.2" data-doi="10.1093/hmg/ddr405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study identifies breast cancer risk variant at 10q21.2: results from the Asia Breast Cancer Consortium</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4991</span>–<span class="cit-lpage">4999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddr405%26rft_id%253Dinfo%253Apmid%252F21908515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8yNC80OTkxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMy8xLzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carty</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of body height in African-Americans: the Women’s Health Initiative SNP Health Association Resource (SHARe)</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft.volume%253D21%26rft.spage%253D711%26rft_id%253Dinfo%253Apmid%252F22021425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22021425&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299351000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.4" data-doi="10.1186/ar3382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Promise and pitfalls of the Immunochip</span>. <abbr class="cit-jnl-abbrev">Arthritis Res. Ther.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArthritis%2Bresearch%2B%2526%2Btherapy%26rft.stitle%253DArthritis%2BRes%2BTher%26rft.aulast%253DCortes%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D101%26rft.epage%253D101%26rft.atitle%253DPromise%2Band%2Bpitfalls%2Bof%2Bthe%2BImmunochip.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Far3382%26rft_id%253Dinfo%253Apmid%252F21345260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/ar3382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21345260&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-3.1.23.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput methods for SNP genotyping</span>, pp. <span class="cit-fpage">245</span>–<span class="cit-lpage">254, 578</span> <span class="cit-comment">in</span> <span class="cit-source">Single Nucleotide Polymorphisms</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Komar</span>  <span class="cit-name-given-names">A. A.</span></span> <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">Totowa, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.6" data-doi="10.1016/j.ajhg.2012.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franceschini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Rooij</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feitosa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karakas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery and fine mapping of serum protein loci through transethnic meta-analysis</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">744</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DFranceschini%26rft.auinit1%253DN.%26rft.volume%253D91%26rft.issue%253D4%26rft.spage%253D744%26rft.epage%253D753%26rft.atitle%253DDiscovery%2Band%2Bfine%2Bmapping%2Bof%2Bserum%2Bprotein%2Bloci%2Bthrough%2Btransethnic%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.08.021%26rft_id%253Dinfo%253Apmid%252F23022100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23022100&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.7" data-doi="10.1371/journal.pgen.1000109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Story</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Calibrating the performance of SNP array for whole-genome association studies</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHao%26rft.auinit1%253DK.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De1000109%26rft.epage%253De1000109%26rft.atitle%253DCalibrating%2Bthe%2Bperformance%2Bof%2BSNP%2Barrays%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000109%26rft_id%253Dinfo%253Apmid%252F18584036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18584036&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.8" data-doi="10.1073/pnas.0903103106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sethupathy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junkins</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Potential etiologic and functional implications of genome-wide association loci for human diseases and traits</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9362</span>–<span class="cit-lpage">9367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0903103106%26rft_id%253Dinfo%253Apmid%252F19474294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIzLzkzNjIiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8zLzEvMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.9" data-doi="10.1016/j.ajhg.2009.01.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singleton</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genotype-imputation accuracy across worldwide human populations</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D235%26rft.epage%253D250%26rft.atitle%253DGenotype-imputation%2Baccuracy%2Bacross%2Bworldwide%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.013%26rft_id%253Dinfo%253Apmid%252F19215730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19215730&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.10" data-doi="10.1002/art.30320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim-Howard</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langefeld</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fine-mapping and transethnic genotyping establish IL2/IL21 genetic association with lupus and localize this genetic effect to IL21</span>. <abbr class="cit-jnl-abbrev">Arthritis Rheum.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">1689</span>–<span class="cit-lpage">1697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArthritis%2Band%2Brheumatism%26rft.stitle%253DArthritis%2BRheum%26rft.aulast%253DHughes%26rft.auinit1%253DT.%26rft.volume%253D63%26rft.issue%253D6%26rft.spage%253D1689%26rft.epage%253D1697%26rft.atitle%253DFine-mapping%2Band%2Btransethnic%2Bgenotyping%2Bestablish%2BIL2%252FIL21%2Bgenetic%2Bassociation%2Bwith%2Blupus%2Band%2Blocalize%2Bthis%2Bgenetic%2Beffect%2Bto%2BIL21.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fart.30320%26rft_id%253Dinfo%253Apmid%252F21425124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/art.30320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21425124&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291683200028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.11" data-doi="10.1038/35057050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jimenez-Sanchez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valle</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Human disease genes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJimenez-Sanchez%26rft.auinit1%253DG.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D853%26rft.epage%253D855%26rft.atitle%253DHuman%2Bdisease%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057050%26rft_id%253Dinfo%253Apmid%252F11237009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237009&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.12" data-doi="10.1038/nbt869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">G.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuzaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Large-scale genotyping of complex DNA</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKennedy%26rft.auinit1%253DG.%2BC.%26rft.volume%253D21%26rft.issue%253D10%26rft.spage%253D1233%26rft.epage%253D1237%26rft.atitle%253DLarge-scale%2Bgenotyping%2Bof%2Bcomplex%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt869%26rft_id%253Dinfo%253Apmid%252F12960966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12960966&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185647200037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.13" data-doi="10.1038/sj.ejhg.5202007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evaluation of coverage variation of SNP chips for genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D635%26rft.epage%253D643%26rft.atitle%253DEvaluation%2Bof%2Bcoverage%2Bvariation%2Bof%2BSNP%2Bchips%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.ejhg.5202007%26rft_id%253Dinfo%253Apmid%252F18253166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.ejhg.5202007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18253166&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255260500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.14" data-doi="10.1002/gepi.20533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D816%26rft.epage%253D834%26rft.atitle%253DMaCH%253A%2Busing%2Bsequence%2Band%2Bgenotype%2Bdata%2Bto%2Bestimate%2Bhaplotypes%2Band%2Bunobserved%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20533%26rft_id%253Dinfo%253Apmid%252F21058334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058334&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284719100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.15" data-doi="10.1186/1471-2164-8-159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mägi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeufer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montpetit</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metspalu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evaluating the performance of commercial whole-genome marker sets for capturing common genetic variation</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMagi%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.spage%253D159%26rft.epage%253D159%26rft.atitle%253DEvaluating%2Bthe%2Bperformance%2Bof%2Bcommercial%2Bwhole-genome%2Bmarker%2Bsets%2Bfor%2Bcapturing%2Bcommon%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-159%26rft_id%253Dinfo%253Apmid%252F17562002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17562002&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.16" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the missing heritability of complex diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.17" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.18" data-doi="10.1371/journal.pgen.1002170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirota</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochiai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study identifies HLA-DP as a susceptibility gene for pediatric asthma in Asian populations</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNoguchi%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002170%26rft.epage%253De1002170%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bidentifies%2BHLA-DP%2Bas%2Ba%2Bsusceptibility%2Bgene%2Bfor%2Bpediatric%2Basthma%2Bin%2BAsian%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002170%26rft_id%253Dinfo%253Apmid%252F21814517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814517&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.19" data-doi="10.1007/s00439-008-0606-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nothnagel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellinghaus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krawczak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comprehensive evaluation of SNP genotype imputation</span>. <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DNothnagel%26rft.auinit1%253DM.%26rft.volume%253D125%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D171%26rft.atitle%253DA%2Bcomprehensive%2Bevaluation%2Bof%2BSNP%2Bgenotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-008-0606-5%26rft_id%253Dinfo%253Apmid%252F19089453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-008-0606-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19089453&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263496900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.20" data-doi="10.1371/journal.pone.0003551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papasian</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analyses and comparison of accuracy of different genotype imputation methods</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPei%26rft.auinit1%253DY.%2BF.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3551%26rft.epage%253De3551%26rft.atitle%253DAnalyses%2Band%2Bcomparison%2Bof%2Baccuracy%2Bof%2Bdifferent%2Bgenotype%2Bimputation%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003551%26rft_id%253Dinfo%253Apmid%252F18958166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18958166&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.21" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.22" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-3.1.23.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">R: A Language and Envrionment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.24" data-doi="10.1146/annurev-genom-082908-150116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ragoussis</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genotyping technologies for genetic research</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DRagoussis%26rft.auinit1%253DJ.%26rft.volume%253D10%26rft.spage%253D117%26rft.epage%253D133%26rft.atitle%253DGenotyping%2Btechnologies%2Bfor%2Bgenetic%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150116%26rft_id%253Dinfo%253Apmid%252F19453250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19453250&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.1.23.25" data-doi="10.1007/s00439-011-1054-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinnott</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Artifact due to differential error when cases and controls are imputed from different platforms</span>. <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DSinnott%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D119%26rft.atitle%253DArtifact%2Bdue%2Bto%2Bdifferential%2Berror%2Bwhen%2Bcases%2Band%2Bcontrols%2Bare%2Bimputed%2Bfrom%2Bdifferent%2Bplatforms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-011-1054-1%26rft_id%253Dinfo%253Apmid%252F21735171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-011-1054-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21735171&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.26" data-doi="10.1371/journal.pgen.1001304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnani</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soliman</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association studies of the PR interval in African Americans</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001304%26rft.epage%253De1001304%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2Bthe%2BPR%2Binterval%2Bin%2BAfrican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001304%26rft_id%253Dinfo%253Apmid%252F21347284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21347284&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.27" data-doi="10.1038/nmeth842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steemers</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Whole-genome genotyping with the single-base extension assay</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSteemers%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D33%26rft.atitle%253DWhole-genome%2Bgenotyping%2Bwith%2Bthe%2Bsingle-base%2Bextension%2Bassay.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth842%26rft_id%253Dinfo%253Apmid%252F16369550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16369550&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234528000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.28" data-doi="10.1038/nature04226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A haplotype map of the human genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1299%26rft.epage%253D1320%26rft.atitle%253DA%2Bhaplotype%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04226%26rft_id%253Dinfo%253Apmid%252F16255080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16255080&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.29" data-doi="10.1038/nature05911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Wellcome Trust Case Control Consortium</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Twee-Hee Ong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficiency of trans-ethnic genome-wide meta-analysis and fine-mapping</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.31" data-doi="10.1086/522374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pathway-based approaches for analysis of genomewide association studies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1278</span>–<span class="cit-lpage">1283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D81%26rft.issue%253D6%26rft.spage%253D1278%26rft.epage%253D1283%26rft.atitle%253DPathway-based%2Bapproaches%2Bfor%2Banalysis%2Bof%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F522374%26rft_id%253Dinfo%253Apmid%252F17966091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/522374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17966091&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251098800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.32" data-doi="10.1016/j.ajhg.2009.10.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic dissection of population substructure of han chinese and its implication in association studies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D85%26rft.issue%253D6%26rft.spage%253D762%26rft.epage%253D774%26rft.atitle%253DGenomic%2Bdissection%2Bof%2Bpopulation%2Bsubstructure%2Bof%2BHan%2BChinese%2Band%2Bits%2Bimplication%2Bin%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.10.015%26rft_id%253Dinfo%253Apmid%252F19944404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.10.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19944404&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272797100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.1.23.33" data-doi="10.1007/s00439-011-1102-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">W.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Vaart</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Large-scale genome-wide association study of Asian population reveals genetic factors in FRMD4A and other loci influencing smoking initiation and nicotine dependence</span>. <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DYoon%26rft.auinit1%253DD.%26rft.volume%253D131%26rft.issue%253D6%26rft.spage%253D1009%26rft.epage%253D1021%26rft.atitle%253DLarge-scale%2Bgenome-wide%2Bassociation%2Bstudy%2Bof%2BAsian%2Bpopulation%2Breveals%2Bgenetic%2Bfactors%2Bin%2BFRMD4A%2Band%2Bother%2Bloci%2Binfluencing%2Bsmoking%2Binitiation%2Band%2Bnicotine%2Bdependence.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-011-1102-x%26rft_id%253Dinfo%253Apmid%252F22006218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-011-1102-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22006218&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/1/23.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/1/9" title="Genetic Analysis of mlh3 Mutations Reveals Interactions Between Crossover Promoting Factors During Meiosis in Baker’s Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/1/31" title="Toward a Molecular Cytogenetic Map for Cultivated Sunflower (Helianthus annuus L.) by Landed BAC/BIBAC Clones" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/1"><span>Volume 3 Issue 1, January 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/1.cover-source.jpg?itok=IC8XGx84" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/1/23.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9SxLqpE7hG_xlaVzUvOY3IHGcE1v2alr4tXhGQ-3mjU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/1/23" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/1/23" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/1/23" class="active" data-icon-position="" data-hide-link-title="0">Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459388" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mkE5gIAgQ7Jlq-l7pBKcHXdGzBkrRn20VQdZQkkQ8fI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/1/23.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/1/23" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lFmulxFgfeT4dWkEV0fIA2QHtZZt4sL4eT7OFKAumec" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/1/23" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/1/23.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459388" id="node459388--2" data-pisa="ggg;3/1/23" data-pisa-master="ggg;3/1/23" data-apath="/ggg/3/1/23.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lei Jiang</span>, <span class="highwire-citation-author" data-delta="1">Dana Willner</span>, <span class="highwire-citation-author" data-delta="2">Patrick Danoy</span>, <span class="highwire-citation-author" data-delta="3">Huji Xu</span> and <span class="highwire-citation-author" data-delta="4">Matthew A. Brown</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">23-29;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004069 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459388--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lei Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALei%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dana Willner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Brisbane, Australia 4072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BWillner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willner%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BWillner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick Danoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BDanoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Danoy%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BDanoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huji Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huji%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHuji%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xuhuji@smmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xuhuji@smmu.edu.cn</a>
<a href="mailto:matt.brown@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">matt.brown@uq.edu.au</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew A. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BA.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BA.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xuhuji@smmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xuhuji@smmu.edu.cn</a>
<a href="mailto:matt.brown@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">matt.brown@uq.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/1/23" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/1/23&amp;title=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/1/23&amp;title=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/1/23&amp;title=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/1/23&amp;text=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/1/23&amp;title=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/1/23&amp;t=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/1/23&amp;title=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/1/23&amp;title=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459388/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459388" id="node459388--3" data-pisa="ggg;3/1/23" data-pisa-master="ggg;3/1/23" data-apath="/ggg/3/1/23.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of the Performance of Two Commercial Genome-Wide Association Study Genotyping Platforms in Han Chinese Samples</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lei Jiang</span>, <span class="highwire-citation-author" data-delta="1">Dana Willner</span>, <span class="highwire-citation-author" data-delta="2">Patrick Danoy</span>, <span class="highwire-citation-author" data-delta="3">Huji Xu</span> and <span class="highwire-citation-author" data-delta="4">Matthew A. Brown</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">23-29;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004069 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459388--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lei Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALei%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dana Willner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Brisbane, Australia 4072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BWillner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willner%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BWillner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick Danoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BDanoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Danoy%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BDanoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huji Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Rheumatology, Shanghai Changzheng Hospital, The Second Military Medical University, 200003 Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huji%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHuji%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xuhuji@smmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xuhuji@smmu.edu.cn</a>
<a href="mailto:matt.brown@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">matt.brown@uq.edu.au</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew A. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland Diamantina Institute, University of Queensland, Princess Alexandra Hospital, Brisbane, Australia 4102, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BA.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BA.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xuhuji@smmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xuhuji@smmu.edu.cn</a>
<a href="mailto:matt.brown@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">matt.brown@uq.edu.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=23&amp;rft.epage=29&amp;rft.atitle=Comparison%20of%20the%20Performance%20of%20Two%20Commercial%20Genome-Wide%20Association%20Study%20Genotyping%20Platforms%20in%20Han%20Chinese%20Samples&amp;rft.volume=3&amp;rft.issue=1&amp;rft.date=2013-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jiang%2C+Lei&amp;rft.au=Willner%2C+Dana&amp;rft.au=Danoy%2C+Patrick&amp;rft.au=Xu%2C+Huji&amp;rft.au=Brown%2C+Matthew+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459388/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459388/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459388/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459388/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459388/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459388/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459388/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459388/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459388/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459388/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459388/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459388/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/1/23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/1/23.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/1/23.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/1/23.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/1/23.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
