<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1671" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1671.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1671.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1671.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1671.atom" />
<meta name="HW.pisa" content="ggg;4/9/1671" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" />
<meta name="DC.Identifier" content="10.1534/g3.114.010892" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Andersen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, Saccharomyces cerevisiae , biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Σ1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that [AIM1][1] , [ASG1][2] , [AVT1][3] , [DRN1][4] , [ELP4][5] , [FLO8][6] , [FMP10][7] , [HMT1][8] , [KAR5][9] , [MIT1][10] , [MRPL32][11] , [MSS11][12] , [NCP1][13] , [NPR1][14] , [PEP5][15] , [PEX25][16] , [RIM8][17] , [RIM101][18] , [RGT1][19] , [SNF8][20] , [SPC2][21] , [STB6][22] , [STP22][23] , [TEC1][24] , [VID24][25] , [VPS20][26] , [VTC3][27] , [YBL029W][28], [YBL029C-A][29], [YFL054C][30], [YGR161W-C][31], [YIL014C-A][32], [YIR024C][33], [YKL151C][34], [YNL200C][35], [YOR034C-A][36], and [YOR223W][37] controlled biofilm through [FLO11][38] induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform [Tpk3p][39] functioned specifically in biofilm and mat formation. In a [tpk3][39] mutant, transcription of [FLO11][38] was induced three-fold compared with wild-type, but biofilm development and cell–cell adhesion was absent, suggesting that [Tpk3p][39] regulates [FLO11][38] positive posttranscriptionally and negative transcriptionally.

The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of [FLO11][38] is more complex than previously anticipated.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000044
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001392
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003761
 [4]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003325
 [5]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006022
 [6]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911
 [7]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000984
 [8]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000238
 [9]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004669
 [10]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000733
 [11]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000596
 [12]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004774
 [13]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001084
 [14]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127
 [15]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004844
 [16]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006033
 [17]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003013
 [18]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019
 [19]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001521
 [20]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005923
 [21]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004519
 [22]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001555
 [23]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000514
 [24]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287
 [25]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000309
 [26]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004682
 [27]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005940
 [28]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000125
 [29]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007591
 [30]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001840
 [31]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029726
 [32]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003536
 [33]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001463
 [34]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001634
 [35]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005144
 [36]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028856
 [37]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005749
 [38]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458
 [39]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649" />
<meta name="DC.Contributor" content="Kaj Scherz Andersen" />
<meta name="DC.Contributor" content="Rasmus Bojsen" />
<meta name="DC.Contributor" content="Laura Gro Rejkjær Sørensen" />
<meta name="DC.Contributor" content="Martin Weiss Nielsen" />
<meta name="DC.Contributor" content="Michael Lisby" />
<meta name="DC.Contributor" content="Anders Folkesson" />
<meta name="DC.Contributor" content="Birgitte Regenberg" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Σ1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that &lt;i&gt;AIM1&lt;/i&gt;, &lt;i&gt;ASG1&lt;/i&gt;, &lt;i&gt;AVT1&lt;/i&gt;, &lt;i&gt;DRN1&lt;/i&gt;, &lt;i&gt;ELP4&lt;/i&gt;, &lt;i&gt;FLO8&lt;/i&gt;, &lt;i&gt;FMP10&lt;/i&gt;, &lt;i&gt;HMT1&lt;/i&gt;, &lt;i&gt;KAR5&lt;/i&gt;, &lt;i&gt;MIT1&lt;/i&gt;, &lt;i&gt;MRPL32&lt;/i&gt;, &lt;i&gt;MSS11&lt;/i&gt;, &lt;i&gt;NCP1&lt;/i&gt;, &lt;i&gt;NPR1&lt;/i&gt;, &lt;i&gt;PEP5&lt;/i&gt;, &lt;i&gt;PEX25&lt;/i&gt;, &lt;i&gt;RIM8&lt;/i&gt;, &lt;i&gt;RIM101&lt;/i&gt;, &lt;i&gt;RGT1&lt;/i&gt;, &lt;i&gt;SNF8&lt;/i&gt;, &lt;i&gt;SPC2&lt;/i&gt;, &lt;i&gt;STB6&lt;/i&gt;, &lt;i&gt;STP22&lt;/i&gt;, &lt;i&gt;TEC1&lt;/i&gt;, &lt;i&gt;VID24&lt;/i&gt;, &lt;i&gt;VPS20&lt;/i&gt;, &lt;i&gt;VTC3&lt;/i&gt;, YBL029W, YBL029C-A, YFL054C, YGR161W-C, YIL014C-A, YIR024C, YKL151C, YNL200C, YOR034C-A, and YOR223W controlled biofilm through &lt;i&gt;FLO11&lt;/i&gt; induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform Tpk3p functioned specifically in biofilm and mat formation. In a &lt;i&gt;tpk3&lt;/i&gt; mutant, transcription of &lt;i&gt;FLO11&lt;/i&gt; was induced three-fold compared with wild-type, but biofilm development and cell–cell adhesion was absent, suggesting that Tpk3p regulates &lt;i&gt;FLO11&lt;/i&gt; positive posttranscriptionally and negative transcriptionally.&lt;/p&gt;&lt;p&gt;The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of &lt;i&gt;FLO11&lt;/i&gt; is more complex than previously anticipated.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1671" />
<meta name="citation_id" content="4/9/1671" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1671" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1671.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1671.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1671.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010892" />
<meta name="citation_pmid" content="25009170" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1671" />
<meta name="citation_lastpage" content="1680" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kaj Scherz Andersen" />
<meta name="citation_author_institution" content="Department of Biology, University of Copenhagen, Copenhagen, Denmark" />
<meta name="citation_author" content="Rasmus Bojsen" />
<meta name="citation_author_institution" content="Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark" />
<meta name="citation_author" content="Laura Gro Rejkjær Sørensen" />
<meta name="citation_author_institution" content="Department of Biology, University of Copenhagen, Copenhagen, Denmark" />
<meta name="citation_author" content="Martin Weiss Nielsen" />
<meta name="citation_author_institution" content="Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark" />
<meta name="citation_author" content="Michael Lisby" />
<meta name="citation_author_institution" content="Department of Biology, University of Copenhagen, Copenhagen, Denmark" />
<meta name="citation_author" content="Anders Folkesson" />
<meta name="citation_author_institution" content="Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark" />
<meta name="citation_author" content="Birgitte Regenberg" />
<meta name="citation_author_institution" content="Department of Biology, University of Copenhagen, Copenhagen, Denmark" />
<meta name="citation_author_email" content="bregenberg@bio.ku.dk" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=M. P. Ashe;citation_author=S. K. De Long;citation_author=A. B. Sachs;citation_title=Glucose Depletion Rapidly Inhibits Translation Initiation in Yeast;citation_pages=833-848;citation_volume=11;citation_year=2000;citation_issue=3;citation_pmid=10712503;citation_doi=10.1091/mbc.11.3.833" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Barrales;citation_author=J. Jimenez;citation_author=J. I. Ibeas;citation_title=Identification of Novel Activation Mechanisms for FLO11 Regulation in Saccharomyces cerevisiae;citation_pages=145-156;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202364;citation_doi=10.1534/genetics.107.081315" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=J. C. Bayly;citation_author=L. M. Douglas;citation_author=I. S. Pretorius;citation_author=F. F. Bauer;citation_author=A. M. Dranginis;citation_title=Characteristics of Flo11-dependent flocculation in Saccharomyces cerevisiae.;citation_volume=5;citation_year=2005;citation_pmid=16043420;citation_doi=10.1016/j.femsyr.2005.05.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=G. H. Braus;citation_author=O. Grundmann;citation_author=S. Brückner;citation_author=H. U. Mösch;citation_title=Amino acid starvation and Gcn4p regulate adhesive growth and FLO11 gene expression in Saccharomyces cerevisiae.;citation_volume=14;citation_year=2003;citation_pmid=14517335;citation_doi=10.1091/mbc.E03-01-0042" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=S. Brückner;citation_author=H. U. Mösch;citation_title=Choosing the right lifestyle: adhesion and development in Saccharomyces cerevisiae.;citation_pages=25-58;citation_volume=36;citation_year=2011;citation_pmid=21521246" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Bumgarner;citation_author=R. D. Dowell;citation_author=P. Grisafi;citation_author=D. K. Gifford;citation_author=G. R. Fink;citation_title=Toggle involving cis-interfering noncoding RNAs controls variegated gene expression in yeast.;citation_volume=106;citation_year=2009;citation_pmid=19805129;citation_doi=10.1073/pnas.0909641106" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. L. Bumgarner;citation_author=G. Neuert;citation_author=B. F. Voight;citation_author=A. Symbor-Nagrabska;citation_author=P. Grisafi;citation_title=Single-cell analysis reveals that noncoding RNAs contribute to clonal heterogeneity by modulating transcription factor recruitment.;citation_pages=470-482;citation_volume=45;citation_year=2012;citation_issue=4;citation_pmid=22264825;citation_doi=10.1016/j.molcel.2011.11.029" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. W. Cain;citation_author=M. B. Lohse;citation_author=O. R. Homann;citation_author=A. Sil;citation_author=A. D. Johnson;citation_title=A conserved transcriptional regulator governs fungal morphology in widely diverged species.;citation_volume=190;citation_year=2012;citation_pmid=22095082;citation_doi=10.1534/genetics.111.134080" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=H. Chen;citation_author=G. R. Fink;citation_title=Feedback control of morphogenesis in fungi by aromatic alcohols.;citation_volume=20;citation_year=2006;citation_pmid=16618799;citation_doi=10.1101/gad.1411806" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. O. De Craene;citation_author=O. Soetens;citation_author=B. Andre;citation_title=The Npr1 kinase controls biosynthetic and endocytic sorting of the yeast Gap1 permease.;citation_volume=276;citation_year=2001;citation_pmid=11500493;citation_doi=10.1074/jbc.M102944200" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=L. Fichtner;citation_author=F. Schulze;citation_author=G. H. Braus;citation_title=Differential Flo8p-dependent regulation of FLO1 and FLO11 for cell-cell and cell-substrate adherence of S. cerevisiae S288c.;citation_pages=1276-1289;citation_volume=66;citation_year=2007;citation_issue=5;citation_pmid=18001350;citation_doi=10.1111/j.1365-2958.2007.06014.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. Gagiano;citation_author=M. Bester;citation_author=D. Van Dyk;citation_author=J. Franken;citation_author=F. F. Bauer;citation_title=Mss11p is a transcription factor regulating pseudohyphal differentiation, invasive growth and starch metabolism in Saccharomyces cerevisiae in response to nutrient availability.;citation_pages=119-134;citation_volume=47;citation_year=2003;citation_issue=1;citation_pmid=12492858;citation_doi=10.1046/j.1365-2958.2003.03247.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. J. Gimeno;citation_author=P. O. Ljungdahl;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Unipolar cell divisions in the yeast S. cerevisiae lead to filamentous growth: regulation by starvation and RAS.;citation_pages=1077-1090;citation_volume=68;citation_year=1992;citation_issue=6;citation_pmid=1547504;citation_doi=10.1016/0092-8674(92)90079-R" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Granek;citation_author=D. Murray;citation_author=O. Kayrkci;citation_author=P. M. Magwene;citation_title=The genetic architecture of biofilm formation in a clinical isolate of Saccharomyces cerevisiae.;citation_volume=193;citation_year=2013;citation_pmid=23172850;citation_doi=10.1534/genetics.112.142067" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Guo;citation_author=C. A. Styles;citation_author=Q. Feng;citation_author=G. R. Fink;citation_title=A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating.;citation_volume=97;citation_year=2000;citation_pmid=11027318;citation_doi=10.1073/pnas.220420397" />
<meta name="citation_reference" content="citation_title=Guide to Yeast Genetics and Molecular Biology. Methods Enzymol;citation_author=C. Guthrie;citation_author=G. R. Fink;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=L. Hall-Stoodley;citation_author=J. W. Costerton;citation_author=P. Stoodley;citation_title=Bacterial biofilms: from the natural environment to infectious diseases.;citation_pages=95-108;citation_volume=2;citation_year=2004;citation_issue=2;citation_pmid=15040259;citation_doi=10.1038/nrmicro821" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Halme;citation_author=S. Bumgarner;citation_author=C. Styles;citation_author=G. R. Fink;citation_title=Genetic and epigenetic regulation of the FLO gene family generates cell-surface variation in yeast.;citation_pages=405-415;citation_volume=116;citation_year=2004;citation_issue=3;citation_pmid=15016375;citation_doi=10.1016/S0092-8674(04)00118-7" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=S. P. Hawser;citation_author=L. J. Douglas;citation_title=Biofilm formation by Candida species on the surface of catheter materials in vitro.;citation_pages=915-921;citation_volume=62;citation_year=1994;citation_issue=3;citation_pmid=8112864" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Kuchin;citation_author=V. K. Vyas;citation_author=M. Carlson;citation_title=Snf1 Protein Kinase and the Repressors Nrg1 and Nrg2 Regulate FLO11, Haploid Invasive Growth, and Diploid Pseudohyphal Differentiation;citation_pages=3994-4000;citation_volume=22;citation_year=2002;citation_issue=12;citation_pmid=12024013;citation_doi=10.1128/MCB.22.12.3994-4000.2002" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. Kuthan;citation_author=F. Devaux;citation_author=B. Janderova;citation_author=I. Slaninova;citation_author=C. Jacq;citation_title=Domestication of wild Saccharomyces cerevisiae is accompanied by changes in gene expression and colony morphology.;citation_pages=745-754;citation_volume=47;citation_year=2003;citation_issue=3;citation_pmid=12535073;citation_doi=10.1046/j.1365-2958.2003.03332.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=T. Köhler;citation_author=S. Wesche;citation_author=N. Taheri;citation_author=G. H. Braus;citation_author=H. U. Mösch;citation_title=Dual Role of the Saccharomyces cerevisiae TEA/ATTS Family Transcription Factor Tec1p in Regulation of Gene Expression and Cellular Development;citation_pages=673-686;citation_volume=1;citation_year=2002;citation_issue=5;citation_pmid=12455687;citation_doi=10.1128/EC.1.5.673-686.2002" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. M. Lamb;citation_author=A. P. Mitchell;citation_title=The Transcription Factor Rim101p Governs Ion Tolerance and Cell Differentiation by Direct Repression of the Regulatory Genes NRG1 and SMP1 in Saccharomyces cerevisiae;citation_pages=677-686;citation_volume=23;citation_year=2003;citation_issue=2;citation_pmid=12509465;citation_doi=10.1128/MCB.23.2.677-686.2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Liu;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Saccharomyces cerevisiae S288C Has a Mutation in FL08, a Gene Required for Filamentous Growth;citation_pages=967-978;citation_volume=144;citation_year=1996;citation_issue=3;citation_pmid=8913742" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=W. S. Lo;citation_author=A. M. Dranginis;citation_title=FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin.;citation_pages=7144-7151;citation_volume=178;citation_year=1996;citation_issue=24;citation_pmid=8955395;citation_doi=10.1128/jb.178.24.7144-7151.1996" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=W. S. Lo;citation_author=A. M. Dranginis;citation_title=The Cell Surface Flocculin Flo11 Is Required for Pseudohyphae Formation and Invasion by Saccharomyces cerevisiae;citation_pages=161-171;citation_volume=9;citation_year=1998;citation_issue=1;citation_pmid=9436998;citation_doi=10.1091/mbc.9.1.161" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. C. Lorenz;citation_author=J. Heitman;citation_title=The MEP2 ammonium permease regulates pseudohyphal differentiation in Saccharomyces cerevisiae;citation_pages=1236-1247;citation_volume=17;citation_year=1998;citation_issue=5;citation_pmid=9482721;citation_doi=10.1093/emboj/17.5.1236" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=G. Lucchini;citation_author=A. G. Hinnebusch;citation_author=C. Chen;citation_author=G. R. Fink;citation_title=Positive regulatory interactions of the HIS4 gene of Saccharomyces cerevisiae.;citation_pages=1326-1333;citation_volume=4;citation_year=1984;citation_issue=7;citation_pmid=6095062" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=R. S. McIsaac;citation_author=S. J. Silverman;citation_author=L. Parsons;citation_author=P. Xu;citation_author=R. Briehof;citation_title=Visualization and analysis of mRNA molecules using fluorescence in situ hybridization in Saccharomyces cerevisiae.;citation_pages=e50382;citation_volume=76;citation_year=2013;citation_pmid=23793137" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Nguyen;citation_author=A. Joshi-Datar;citation_author=F. Lepine;citation_author=E. Bauerle;citation_author=O. Olakanmi;citation_title=Active Starvation Responses Mediate Antibiotic Tolerance in Biofilms and Nutrient-Limited Bacteria;citation_pages=982-986;citation_volume=334;citation_year=2011;citation_issue=6058;citation_pmid=22096200;citation_doi=10.1126/science.1211037" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. Palkova;citation_author=B. Janderova;citation_author=J. Gabriel;citation_author=B. Zikanova;citation_author=M. Pospisek;citation_title=Ammonia mediates communication between yeast colonies.;citation_pages=532-536;citation_volume=390;citation_year=1997;citation_issue=6659;citation_pmid=9394006;citation_doi=10.1038/37398" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. B. Reynolds;citation_author=G. R. Fink;citation_title=Bakers' Yeast, a Model for Fungal Biofilm Formation;citation_pages=878-881;citation_volume=291;citation_year=2001;citation_issue=5505;citation_pmid=11157168;citation_doi=10.1126/science.291.5505.878" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. L. Roberts;citation_author=G. R. Fink;citation_title=Elements of a single MAP kinase cascade in Saccharomyces cerevisiae mediate two developmental programs in the same cell type: mating and invasive growth.;citation_pages=2974-2985;citation_volume=8;citation_year=1994;citation_issue=24;citation_pmid=8001818;citation_doi=10.1101/gad.8.24.2974" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. S. Robertson;citation_author=G. R. Fink;citation_title=The three yeast A kinases have specific signaling functions in pseudohyphal growth;citation_pages=13783-13787;citation_volume=95;citation_year=1998;citation_issue=23;citation_pmid=9811878;citation_doi=10.1073/pnas.95.23.13783" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=S. Rupp;citation_author=E. Summers;citation_author=H. J. Lo;citation_author=H. Madhani;citation_author=G. Fink;citation_title=MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast FLO11 gene;citation_pages=1257-1269;citation_volume=18;citation_year=1999;citation_issue=5;citation_pmid=10064592;citation_doi=10.1093/emboj/18.5.1257" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=O. Ryan;citation_author=R. S. Shapiro;citation_author=C. F. Kurat;citation_author=D. Mayhew;citation_author=A. Baryshnikova;citation_title=Global Gene Deletion Analysis Exploring Yeast Filamentous Growth;citation_pages=1353-1356;citation_volume=337;citation_year=2012;citation_issue=6100;citation_pmid=22984072;citation_doi=10.1126/science.1224339" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=N. Sarode;citation_author=B. Miracle;citation_author=X. Peng;citation_author=O. Ryan;citation_author=T. B. Reynolds;citation_title=Vacuolar protein sorting genes regulate mat formation in Saccharomyces cerevisiae by Flo11p-dependent and -independent mechanisms.;citation_volume=10;citation_year=2011;citation_pmid=21908597;citation_doi=10.1128/EC.05078-11" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Smukalla;citation_author=M. Caldara;citation_author=N. Pochet;citation_author=A. Beauvais;citation_author=S. Guadagnini;citation_title=FLO1 is a variable green beard gene that drives biofilm-like cooperation in budding yeast.;citation_pages=726-737;citation_volume=135;citation_year=2008;citation_issue=4;citation_pmid=19013280;citation_doi=10.1016/j.cell.2008.09.037" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Torbensen;citation_author=H. D. Møller;citation_author=D. Gresham;citation_author=S. Alizadeh;citation_author=D. Ochmann;citation_title=Amino acid transporter genes are essential for FLO11-dependent and FLO11-independent biofilm formation and invasive growth in Saccharomyces cerevisiae.;citation_pages=e41272-e41272;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22844449;citation_doi=10.1371/journal.pone.0041272" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=L. Vachova;citation_author=V. Stovicek;citation_author=O. Hlavacek;citation_author=O. Chernyavskiy;citation_author=L. Stepanek;citation_title=Flo11p, drug efflux pumps, and the extracellular matrix cooperate to form biofilm yeast colonies.;citation_volume=194;citation_year=2011;citation_pmid=21875945;citation_doi=10.1083/jcb.201103129" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=S. Van De Velde;citation_author=J. M. Thevelein;citation_title=Cyclic AMP-protein kinase A and Snf1 signaling mechanisms underlie the superior potency of sucrose for induction of filamentation in Saccharomyces cerevisiae.;citation_volume=7;citation_year=2008;citation_pmid=17890371;citation_doi=10.1128/EC.00276-07" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=M. Vandenbol;citation_author=J. C. Jauniaux;citation_author=M. Grenson;citation_title=The Saccharomyces cerevisiae NPR1 gene required for the activity of ammonia-sensitive amino acid permeases encodes a protein kinase homologue.;citation_pages=393-399;citation_volume=222;citation_year=1990;citation_issue=2-3;citation_pmid=2125693;citation_doi=10.1007/BF00633845" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. Voordeckers;citation_author=D. De Maeyer;citation_author=E. Van Der Zande;citation_author=M. D. Vinces;citation_author=W. Meert;citation_title=Identification of a complex genetic network underlying Saccharomyces cerevisiae colony morphology.;citation_pages=225-239;citation_volume=86;citation_year=2012;citation_issue=1;citation_pmid=22882838;citation_doi=10.1111/j.1365-2958.2012.08192.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=W. Xu;citation_author=F. J. Smith;citation_author=R. Subaran;citation_author=A. P. Mitchell;citation_title=Multivesicular body-ESCRT components function in pH response regulation in Saccharomyces cerevisiae and Candida albicans.;citation_volume=15;citation_year=2004;citation_pmid=15371534;citation_doi=10.1091/mbc.E04-08-0666" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, Saccharomyces cerevisiae , biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Σ1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that [AIM1][1] , [ASG1][2] , [AVT1][3] , [DRN1][4] , [ELP4][5] , [FLO8][6] , [FMP10][7] , [HMT1][8] , [KAR5][9] , [MIT1][10] , [MRPL32][11] , [MSS11][12] , [NCP1][13] , [NPR1][14] , [PEP5][15] , [PEX25][16] , [RIM8][17] , [RIM101][18] , [RGT1][19] , [SNF8][20] , [SPC2][21] , [STB6][22] , [STP22][23] , [TEC1][24] , [VID24][25] , [VPS20][26] , [VTC3][27] , [YBL029W][28], [YBL029C-A][29], [YFL054C][30], [YGR161W-C][31], [YIL014C-A][32], [YIR024C][33], [YKL151C][34], [YNL200C][35], [YOR034C-A][36], and [YOR223W][37] controlled biofilm through [FLO11][38] induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform [Tpk3p][39] functioned specifically in biofilm and mat formation. In a [tpk3][39] mutant, transcription of [FLO11][38] was induced three-fold compared with wild-type, but biofilm development and cell–cell adhesion was absent, suggesting that [Tpk3p][39] regulates [FLO11][38] positive posttranscriptionally and negative transcriptionally.

The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of [FLO11][38] is more complex than previously anticipated.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000044
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001392
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003761
 [4]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003325
 [5]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006022
 [6]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911
 [7]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000984
 [8]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000238
 [9]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004669
 [10]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000733
 [11]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000596
 [12]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004774
 [13]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001084
 [14]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127
 [15]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004844
 [16]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006033
 [17]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003013
 [18]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019
 [19]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001521
 [20]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005923
 [21]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004519
 [22]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001555
 [23]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000514
 [24]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287
 [25]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000309
 [26]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004682
 [27]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005940
 [28]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000125
 [29]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007591
 [30]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001840
 [31]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029726
 [32]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003536
 [33]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001463
 [34]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001634
 [35]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005144
 [36]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028856
 [37]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005749
 [38]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458
 [39]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649" />
<meta name="og-title" property="og:title" content="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1671" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, Saccharomyces cerevisiae , biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Σ1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that [AIM1][1] , [ASG1][2] , [AVT1][3] , [DRN1][4] , [ELP4][5] , [FLO8][6] , [FMP10][7] , [HMT1][8] , [KAR5][9] , [MIT1][10] , [MRPL32][11] , [MSS11][12] , [NCP1][13] , [NPR1][14] , [PEP5][15] , [PEX25][16] , [RIM8][17] , [RIM101][18] , [RGT1][19] , [SNF8][20] , [SPC2][21] , [STB6][22] , [STP22][23] , [TEC1][24] , [VID24][25] , [VPS20][26] , [VTC3][27] , [YBL029W][28], [YBL029C-A][29], [YFL054C][30], [YGR161W-C][31], [YIL014C-A][32], [YIR024C][33], [YKL151C][34], [YNL200C][35], [YOR034C-A][36], and [YOR223W][37] controlled biofilm through [FLO11][38] induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform [Tpk3p][39] functioned specifically in biofilm and mat formation. In a [tpk3][39] mutant, transcription of [FLO11][38] was induced three-fold compared with wild-type, but biofilm development and cell–cell adhesion was absent, suggesting that [Tpk3p][39] regulates [FLO11][38] positive posttranscriptionally and negative transcriptionally.

The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of [FLO11][38] is more complex than previously anticipated.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000044
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001392
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003761
 [4]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003325
 [5]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006022
 [6]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911
 [7]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000984
 [8]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000238
 [9]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004669
 [10]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000733
 [11]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000596
 [12]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004774
 [13]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001084
 [14]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127
 [15]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004844
 [16]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006033
 [17]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003013
 [18]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019
 [19]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001521
 [20]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005923
 [21]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004519
 [22]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001555
 [23]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000514
 [24]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287
 [25]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000309
 [26]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004682
 [27]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005940
 [28]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000125
 [29]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007591
 [30]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001840
 [31]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029726
 [32]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003536
 [33]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001463
 [34]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001634
 [35]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005144
 [36]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028856
 [37]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005749
 [38]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458
 [39]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rwI2ZrVFwHC9gLb9GeL3N4TzD3iIIMshmmP_rI7wf0s","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459660","apath":"\/ggg\/4\/9\/1671.atom","pisa":"ggg;4\/9\/1671","ac":{"\/ggg\/4\/9\/1671.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1671.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1671"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1671"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1671":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228BIcoajKcerqLU2XJlmwtXTYONPAOBZ3G3qCQHCPUCTXMFJQAmQpHMz9ECJgmSPCK+CEzDbupWFAta21EUtekQRu4dZuLJ0R\\\\\\\/1IbmhqWzfEdeq7\\\\\\\/CDkGS9d71uBW0avGRvHzJKlSyYhan4THNLBeGIYq+PlV+0TIEDO+vljda4bxHHIUY8OaQOq+43+pdc29ZG9+L7bIEjoJHM9QeeZzxX1+UVcmgs1tmyvWCtVLbOk0Q+z5sfViWuKjOl4k+AWoIdUNRs6yuKfxKUMjDx1azu8KSKiIqS8uW+nirzlvOaDr\\\\\\\/r5ggYbfUh3C6mLwjYmTzP1wGLXJiefqvjM67buiRJ65QrmcjhgEq8keT+Iu4oGfc7cGdb0+biYp\\\\\\\/ETjHpW6aDrVzZWQEh2DJFvKNsle42u930vGH7TD\\\\\\\/ms61syyly2lTpW2\\\\\\\/Sfw2KS0FeZAshFVs1E6nUi00L6N4KlNM66LqfILKrMdkKQ5ziezGiL1nydEovW3AhLPkJ2yuhvLf7c3bI1Rcy8MYNWJhGeWYP5lkekVTMVANftDnrUPDMU9mAsSPnUuB7R5o9KK4ZNXubBrxBaU5ntQYj57zqCxfGO1dOm3zDks8ZVwBirlek27VJpl7dnJYrnUiDkToA80piXtG4D5qELgevPXwXtU67OwI+GTB0EvMfSQrwZb8uXODwGperdwY6b7Jrjm5SH29COakb+owDF4vNc68yhVBPs0PV7UEFa90Ting+kfobkWtKeksJlfRFjGG7amaWiAAiFFQIFNwt7heywM1vlKtXYaVXI3cNiox8sItiNrQ45Tyh\\\\\\\/TxSUC5NmwntTDlCdOzGDPHwBqSvN6vD1SvS0ftimbNeYnGvN\\\\\\\/t3haUUlD3Oqz\\\\\\\/AZyroKXeOw7xmUXuVNA3R8Khml9I9QF\\\\\\\/ZRz5a3qy7eX\\\\\\\/ptHEC94uIRJNx\\\\\\\/NND+2EMgpC4NwgW6nhDPiWgCADdXJo6\\\\\\\/pG4a1ha3PcfK9RrRHJeQ0+1Hjvt5Zj7mrCEkt+kurOWkGbb3Zoc7Kh\\\\\\\/4XeMkoGh+NJ9jKTedEUIDC5V\\\\\\\/2bWMuoGKpNJJAMNs\\\\\\\/ymzHLNX+utOMIVfqMitf8OSQeyAmUgUMs1+98rzBp7yC\\\\\\\/wd6fIkcdLsOTkDvAGH73+WLh24dg\\\\\\\/MBmQr0c9Km05MTfSTc17dwiDu8VBls4D9KAUf1nuLUZyqHesYOyzefsmgWEvvCzjrTIxv2MtW2+BINeGoN25IM8mL96uv8CY65Q7lDdhbhjpi0Q\\\\\\\/FucPbVQQJ8AOFDS4fuUAP6SWTyJSTgu6lD\\\u0022,\\\u0022iv\\\u0022:\\\u0022tKoZSAIdKzQ67O50NtIw+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002283af2c5c4081aab7e8e54e933c2cb1b8f2a07bc30972045aa76edf2d94bae378\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oFjQ9rFdr6RaZ3N63Fjs3+8xdUdOens50ClARRtzwZ\\\\\\\/\\\\\\\/P+MTIE8eH2iwSyAwUJeYwWhH4J+UvpSOWXH+vpuZuDZLFK45cZym+\\\\\\\/SNNvMmvHCwlmjA0xuU0ow7vuTplUSCfixPyZOBdi5i0j8aBHornVbaFEScgmwlU0PktdzKTc9lt8fBkaIBDAB5t9Sxls\\\\\\\/DoOUJXDBh3NtrEq14dj39Vo+S8K7OiUJbG9Gl4GosYxmADhZficCm0X2eyphtHmd2z6xyqaR9X9uDd5ZbeLl2einGCJjMmEueXTAMLAYSgO+Er4TnIcLNSqd1lDFy8Ax6cNVnO1j4B+VNbw3V0\\\\\\\/vCrlGzKWrPtFRuSHIdjy2a7vl3Wkx8XlwZz9M20Z7YssmPEvkB\\\\\\\/f5wKqyn+MLenRy7EPtyHxPrJUp30WUMfZAfHCrXOGP2BX2DZ\\\\\\\/\\\\\\\/4g02PGxb4Xjb2dFps36\\\\\\\/1x6JqR8JiwZSr4WGTc5KTayL8szlhbEU1qTRCadBDz2KnbRPLph3iAjQmAsqlj5X\\\\\\\/QdXTcyCIJZwuqTHrV52xaGtMYYrT2eYB2RiYwFOWnudzS+qgHc\\\\\\\/OQG25Rrtqyi8K1oPBRTLyYQxT7RkDR2qYxaGaJJQBgC8vUHysxHRj6hckGS6gx+zuvfuQ0Dbd5nVeV5YrXMVJzRxm9UlRFexi3kHq7YHYgT0ECl88s7D5AGFBgaK3sqA9SxhZ7ip\\\\\\\/FDnOAdInDMea0QTXVTPcNHh9THAX20laAgL+8J0zpSBaoAK7eyEAV9vXDSUOSlPeH\\\\\\\/NPZHl4Rsb8PdtEzfte1ySsa6tVaT76EXWB2bbPPAtxzqMFKqmMjJxVgxbcdA59m4\\\\\\\/eNHh2E0BEY\\\\\\\/\\\\\\\/uZOm+VUH4FfAxWzIbiYOFQjevOVQY4IKFs8DHsVQQFLkeoPgZ1TYWTH7Pq1HCrPYgFdD+ykdlNk3Vw3slXPrDWXjHVgd9Dkew2Nu8nyAUiFzQSCbUy9qr7s1Xq\\\\\\\/L\\\\\\\/gBzti8ak9xB\\\\\\\/hpNaW1QzvnbdtpCSqm\\\\\\\/uPvoG12eieJN7u7ort2rVNojbiuXavputydmZCIvM+oorkoXLKhVq2VUELmRCmOS4illHFbELAuA4abTgzM6dSehckLUaiMBe53Rw\\\\\\\/1TkNObInvFrUw0Us3fm7qYaYWsd1Vre5rNLe5c0ZYColQPLWjxFI32NoAs\\\\\\\/EPFACGcKQteAJgl9Qo0bpx2ZK55A8o3PnmVyrLB3gGuL2AtNOG4+oGZUIe7dYUzdbqgK4QI2ULlkALZZutXSYNJK2pI2dP9zbrVzQWi714SbLSUW3xLuxPVs74aRyplMavmPbuGGicL8793WOHNNe5AFxKlNzyH13+NlW3+U1cI378glv4QQpyQj8xgkh4kyFFqkkWkdknDfMSyNJJmMxm3\\\\\\\/7sZrBvwdbHeUuCxgrHBU2eMOspDmPsrmJD3eCubzTYjj5va5Y+h0LlIF9scGiIVrtqYqVrXfnZiLhGOOsA5xlC4+9GoTi0z4\\\\\\\/UjoNgS6nhjhojLtooEZCeDFcG6TdkRK6EY\\\\\\\/vIHnPqWcVr3nx0UyyGuEpFJeg6lnEhs7Qx1u5WPndacHySWEtWmLaTPwPyIMEJ\\\\\\\/RZoovoFd6IkAODX9iYTOmmuPby7oqD8VjdTQzSy6jpUFZ9Csl7cz+rvlRoUhcxGZ8oKmmy7o7b4lovQSjPz4Gkjiu4A+Bi7HLyhbJZYFb5OuILFiJfXrZtIkAH95qOcA5Wth\\\\\\\/GG+udI+nLo40FTca3DdZSvgptuMyPNAl8RxiZ\\\\\\\/onpwKGcLT15BsH2cGb\\\\\\\/+5z\\\\\\\/OriIcjrtNH3nMN\\\\\\\/T2K2h5TFatR8OES6Rk0GWhvyB7GMkUcwDcvIfoTo+h7r4iIAQIo\\\\\\\/C3TRw2XgT62bT4VkWkI10Vg9dK51VCYYsx08KcZ3Ls3B0Ki3bCMAuAXFyDC2Jf55vhmGijvQkkWDX1pYgWrtbsU8XxBlbi9x3d6Abd+FCc\\\\\\\/6omK9yDbD4T9c5LoJgDh\\\\\\\/ct9wVmboTSJ0WxznrY86UlvCxg\\\\\\\/Q4hhSlkq+CsZp+qUK5SPFRQmzJBFRGTVL2Bd9jdL\\\\\\\/0SzqveDTUcy\\\\\\\/Otnu7NtpG09yR+WrExrS3\\\\\\\/ZSLp3oeMrfcZu3nuSYAl7EeLh8tmuULqSNfEcy8zqyEwt3k6H23iqNRnIHoLgxzqSAG4+Zhrnpk7BWc7pLaEM47n6Xq8+WHFpqecIATKuSA5FDJgnlcop+Mndo63fGtJNeyHw0GiguzkYIgXpXOp3O1F+V0cdzV22rg5X07c6zsdkA4IMpLxvXOLcQwaJ96Uhv3j1posMfaoXwrqeX2rhd72pC58BKvikeDtPKrsWUb20Gspbr6cdwni3WCkcDwxCl8LbfivQLxopAoQGmhP3HKARJ\\\\\\\/Jlohd3crB4AR9P6Sw6usQe4mtQStKBH50NK7eOwhSl6fcW0r9uL3txETLh9zBZ4UZeXnWkTEiElDbl06Sak2tXRcYsKM7YhuMHH6oUTIKs0ZBPiba2mlAmr5TgNZWtmNezwBMtqYtzIicm7eKkayDCTUsFehncYcIxEE\\\\\\\/prnWT3yKP5KVSdo6jxWBeozby8jqcgcTaTVrU+oixuSxeOJBbjJz5E9n\\\\\\\/T75MnvCH4oWZJRKoV4N8b+79q69BfnjoXoid2dC5YOZxALb0dx3\\\\\\\/4StWlrMGXvKjmmjH3GfQUNjgzXOOvE9sU8jAF2HChaHV2he6dvCFHBiz6AzE0lIo7RsasWR3BoqqWSKY\\\\\\\/kh2EMpK2tTfvEER0VkdTmx1wrJenAuHQg7IUrBsRW\\\\\\\/hSZoLNCkMymp+TM1CzJlhe2tiEnElq1WoDVROsVEiWeLmpCqDnH0fymVv7\\\\\\\/B4OXOdkdZ9L4jEwgnjCd2JakN9CVzbX0Q4dDSVMOLpun6mvM6c78Oy22z0R\\\\\\\/c5pVY4xxYVpAQpoz842mLAcQEn9u1EHPNJkLqaAqQEbaHaG7WuawJ6m9xi\\\\\\\/B9U2Tj\\\\\\\/PENYVnkSmz7S2\\\\\\\/7tpfgP29ygvQGjRGsy1RNpvMs+cDLoOzWLRrGGR0LEZWGBBYfDLBZWURWRNZ39X57f8NgbRkWX+p6DNTgkcxQjJyVr+DVl76aVP9vAvoV6TRAIuIRjljQjDVabLJ3DtZihBuVOtFylBc5mGySXZQ1qdPjC+mqm0K0uXdgnssFi8LB5jd5zFUDpu9BysuZC+oKJI2IbZQr\\\\\\\/iRKFT5kphxOINLtg4CmgPvo0CYMHgqYg+2a+gkY59E\\\\\\\/1j7w1RnuO76KEyAPcxqUNpAxb6roPwPA\\\\\\\/Cc6G4ycy6xRypYjd+RhhVLu0tByWZu1fKSGjknR9EhWfi9GrDmVxgZRV\\\\\\\/DQTLO2NOWcqSXHo0akJvoeF8GtWTqVA2talic0hhgLYCYIeUyM093bHXCilwFEKlvOkPK2Mqu874ncSuznTBiA5j4EuBsVmTQU1WGbZWVcRLDx+lifrXMotFN1Wq7EkRu9DXbE0qSAkLhLG+peBnXTA6i6OnCOa9BSTk3\\\\\\\/duicRosoRRckLYcUYEATDh4sKKu\\\\\\\/7wC5w09F7AOb\\\\\\\/yev8p4RHgq\\\\\\\/EicGy02X8iNZYSizmJ4Jd7nvig8USz\\\\\\\/D8tc23CTapwIIgwcSKND2qmdIgD1lheDYCcP35ly1rFcJPt8dh+TyYD9NeUjqX\\\\\\\/tcudsyiZGEI+IZY6LBSgNn7tEl6VV\\\\\\\/QZdWNwoDAzYjTINAxDCMUX3YAAFulVNCjTB\\\\\\\/Xxm1ig1b1wnStW6ZekS5PIEvNOmPoQm2CCQTnc7jiIeLTgmvyOXV2h4d2KBSBK7O1ximlQoa4xaKdEaqY5hCEVDiXrM103kkf630IiY6\\\\\\\/mG+uDx1ApDfuLrFV0iaZXlGC0dVXlb+AApGXrC+CA6IhFABhhSGx1CTv2K0bRwhnG3DAr9lMnQwk3FyBAqWVcPAk1nIAuuxibdnfu8APQlyWex5k8RIrXNop69VaLNjMF42BSrZ76s50M6pgz+kasKyQnv9PDWMj\\\\\\\/gSgQMEPxtUd\\\\\\\/3YcWjGy+L6Eov+CKikFKqyF26MibCsUONiYaHbd9de+bwseNu5IuSy0xylxRAOp+Z8n2DwVy7\\\\\\\/KBrNnwScUs8z14qo7zdkBCbBpZSm6V6npnxM3Ma5vICLTl+vPigl3jybsAK5HnPgXtHQC0m8fvU+\\\\\\\/bHsmuKi6eajDcDlF\\\\\\\/Vd8oCjhN\\\\\\\/Ep0iutu82F\\\\\\\/fBih62O1Y18e+EN+JIIvHfcMwrBrtYYFgQPAXtrX5jq0vZ7UMnNYRjumRIFZLZ0wwNiC8Kgi3UdrnweExzGN7D2zON1VNZ0jlBB7po51UDQvVFuPNtxd8497DG1xSLMX9PRzuI45sVAAtLu5a\\\\\\\/F0ka8qkPAfksLyHMAFttTv2\\\\\\\/e8i48KKRuHzqbBNUqM\\\\\\\/qpVNcezliG1wnq1sJD\\\\\\\/8KLE1AWzkyCuuzWdmCp7x7mzoQGbXO3OHP+ut2r8prxzMURlJH+I\\\\\\\/xTKPTk5H9aPVRJOk6w\\\\\\\/Nb3qORCERzPtv0xLOXM8VMT31+VEaPNhWiDEEYNjzySTQ6gQsLYrsdYoiai0ge1l+eapR4EWnW6t+zQ7riJS5CsK1LDP71KQ\\\\\\\/diWAltTkByUWIfZQUAi5X9UGVeLGZHdWROALT8+VJ+Rih6aPUWm+1gAYV+8N94KMRbvTTaPOFRvl\\\\\\\/28\\\\\\\/T5OQOMDfmQ0c0rty2cbyKRrt1AChh9pTaxDoJkp4wfg+lV7g5YtK8kA5xcGuX3G0yr+sVzaf43nxKAdTsaabaR4z+uIYLFbQRXTV2Z\\\\\\\/K010b9yRwqOLocG11ClCVJGpiRImaJVuB69\\\\\\\/QQnWEQjh+m5nsqXJ2Pd728Ii+kPUhyEcgSoWLvBUE6tLJEU3fGq3KuHEHGQs2wpniFI1hep8fBqrFa6lERcyO3jvJfETspMijUiC0ilukiSo61er28UNjy6lO7Uhx0aWiGVQvWR9tWfgac34iaeTbgpa\\\\\\\/nH0bTeWx9MXJKbmXcINow5H7x+1HHjlLvD0CgXnD+hMW5vMTQ13\\\\\\\/F6VLS6I81n2gsK3gox9FXL7ilRKVJT4+P2XmEgpvFiRFiqQj6qb+R2qUUEiOVChDia3acZ0QaVYpgDz4jwmG\\\\\\\/kC76Hi7I6fJ4YO5UsC41\\\\\\\/OqFqORK0JKeUBjDCDPjA+xSZ8\\\\\\\/UnvQEfECERME5Wh79W\\\\\\\/rL+27bCrwiX6exCJTIx+wrN8dH0qLFviQBIuCF0NtbyW0DyCvxlt0tnv5DKayl+cQxnC1kHunKsg0lkF3jD+R6Dt0R\\\\\\\/z2qB+7vS9rU43l1eJlXcAUZjtbXnif67qK3eAeas76VpdJitWMRo9S3sL8H\\\\\\\/QATUs\\\\\\\/h3TXpK7Pfn7gwTyoeNXTHa1ezuc2ZuYCHkDpiVbLA1IKsDuywIp1ZAI9rRz0ecUF9NJDYQSk38RajwSIkg4l82iixn0RAXhFfmItq6pBl7RDrheZpS6mygY54eYEwnbFN+QyAmkifQoFyopew1MiH9Jn+2a9SwN9iIUWs9yHI+0o\\\\\\\/atcIVBwZ+JBBHdTQOblJML+cJFjL0RN7WvoCpTQfmp6ptDSXkA0qDT9qvpg1u0wtARi4G6q9fsg\\\\\\\/4gilqDG76ekmMyRfythafD3RAMGhwSy8VYsekMNWT2r72Uu4lmxrfm8pNWqBf8VLufRZclNtnGOLh8zd23L12FmE0KoL8B0jdBulpHrqKHQ2Lw2Vog7oa9HQpGQGmQUS01FJjhUN5HCePvjOY0Uppaf38eZh8qASQFNQnmxLFxmZyFYeEiol9PxerCLma28YuvwIQOoX7Y1Ew3oSyJyOql7jUU4mcRipdQWZTA++Dcov7vl8B+gUt1GDWvRJJyUGSW16HBNd7Z6X\\\\\\\/RH4L4uYPBewEIAx2CgkewSJvLD6EzTk1sZGPGB\\\\\\\/ETF53p6qKy866BiO3gq9c9bVZ58lhGV2b7Qgz2uPyrXWUwHLRL2p4inK6MoSH\\\\\\\/3iDRwwaMfGxZEBd95Z04h33WYby42kpHvyA\\\\\\\/gLhWcXvtiGxyeWsfaoccMyNhEQtBJLJXwf0BO2bKb4yvaLuRn6UjP22Fh1fkK9tCVjmApzfppTOh6WgwKUFBk9ldKSsVT\\\\\\\/WKHkQ3tqLqd2DbBnSbcBqJP25taJuMF6OVwoLsA6ZkNif80LUxyf6cBNxJtk68TevTlifURNt11kTY8kskpzGiKbSuMYwhG38zgsRpgwVV+rH42rmwAhmEcopNAA56tveQHfgBNiXFA6kCrD0wHUn74AngJSSej\\\\\\\/7zQveA\\\\\\\/BjzHZJ7HWtpjDCBp9YXpqD2rcPzL5DtGguc17C3OH77fhK8FkswoVS0d2+mNk2UdEqsB7T9Y+jyYlF9X97sp0EJtNgE1sn0K24Sy8TUqDfwlNZCLqAJmQbUaJl8GEMHriPcle4+FWkHJtp4OLRfvi2FzVlLcJzuh5X2VOgubwW9ECIOInmokGjsMENOwAbKPmOB6xqRSyqXQMz6xrTymbr0zcqRybLF+z1RG5avaU+jsg7dtALFV+zQgeKARvfabUy2vmFfAmewgAzVTfzLLzeO24TGNI0d5ZdL+s21Fj9k5L43\\\\\\\/6NNGlTpIQxyif7rvS+\\\\\\\/3lMdonmHte8QlOzCsNysCCQhPb+GLqF0Bw163SjbQXZou\\\\\\\/UlhwxJUP7XcnrMCukaaLRXm7U98FD0GM3YWDGR4Vxo6CJrpDWsgPHya\\\\\\\/MWzkQsSB0kRizHm7U6UKMOnheZsYI40LmHFkEmnMJCY\\\\\\\/JH93RedVqL+3QVPQCDrTSpwsBlmwLRMslPhTlGE9XoU+k4v6l5WqF37bo\\\\\\\/3cl\\\\\\\/aatPTJoRkXxdS18LiL9Pz6c6qq+1Hc9wLbLx4Zb06oqeDKdR8Zrvzp\\\\\\\/LGhdexL4UDjQwO0Z22u+PB0uOD+uirRnp7GvNxcz9UQfdLMsm0YyurhcEZom1LfVOCvuP\\\\\\\/Ce1mcAde6l9gz+7diO8F8uCTmAiGamm1LLtkyitF2GA8X4XRRMX2cV6RiNWEf0zx4uapqqXHYPS6nV24W5MUxFjUFGGF8vJpy8yLp8TkgqpSJUDTHsXMYqMPwFIIVIdQVkBLtX844QbZtHplNP2UL5ZNUrYaTWYcg\\\\\\\/qQqpVei4coFmuHCWjvTiJtlyBfHRJ3CKPBhVICt2aNG2WQg3QC9mIorjImyw88XQqITz2EqWfjZW77pWCrGnzjYIvv8lA1J5nhF5bL5QVa5AQY9ffEnRgb+nJgoUs8Er44hGfi6082L96sVU6IlJEElGcnPV0UE668AnR5L26vH6KXv3weoVkza7OJdr2Mh23llsMPjk8vURqvi\\\\\\\/dHcRC4XpN2B2MEDUWxPEJQ72afXnZ2F7WagARaVDXVmLnHCam4vvtR+awL7pOywb8aEztDzYsNHPsQqH+IbFOULVhYWpuoWZz3KkCxU8SZKeLngELkQYrX9TPpSkSqyGwjrfPCYWz5JCfl3TMXldI45GXh1xUEZvPVvIoA026XT8hMyR9rKwfHXI7Inm+4RTzyzGgHYbA5ra7rBVfUZfLEmiSYzRbmmyOAlP3rgzoJ3AecszNj6E2HLi3o+ZC8UbmVN312IfUEq6VsXMIhvK0BtwNRMKuiVG3GQ\\\\\\\/ECvzm6SGOsYGEu6pDeMCLLnf3AlFEUdkmOavx01u2sx5XoDxFGOkWmw5ueA1CkZQKg6Jig62ory+MpGeK5EXLtfn2W+INXC00tPGQ3r9MEOnJHmqhtuzzyd0jnHPkegHMBKU8lhJxfgJzJ6lfvIsj4RTgTuG+MC4M8HksaV3TdkxzTcnXrb1j8HO0LB1pS2hRYhC4eTH+BizmeqWxrSs7MUpjmqExyQROUHPheWWgjlzuWBrAFfMN8A8jr\\\\\\\/lAAJU8M6WBUBxViZxbYk+V3JCAfYgGn\\\\\\\/oNqq1b1PS6c6\\\\\\\/NAb4rdpTo+m4gmoF6dueb0WkNoPJn+Wd8ef0h6t9YRhdPxbb\\\\\\\/h3samJgKMiTSPnnGQiL302zWBPBsRBo9fQm7\\\\\\\/goSlpQQZ5XCCXqgvNRGfCbmfxQBO7+LviekJcWScYmO6YotvkPmI9LovonHzrPIRoeZB6A69A0gq6qpBhMqKPbNRZ2+sKzYoPHjierDD23eItxmolRhohu+uLteAfoVXkCsksB6St33gdpUL501TcF2+W8WG8OsKzGUBs5h9qN9472S2N2DxLt\\\\\\\/pnaiIHsoQQ6zar\\\\\\\/1Gyr7gXaM2\\\\\\\/n831MBXI0bqBYEmBOZyYnWcmDc1UWC6Pu2sNxl1kN3uYL0metCVqHMxO0oqup0BwV8Kpiv2wSQHYCtzKz75jgIvx84XBl2kjSTddOYlV+63Es9Iq9MvPcptLg9cO64kvc9bzOAgB3kpQi7Pm6GEz+mQGBCDIDd79Q2OtvQelufdNliIOa1ZsOsR9HbO2LDoldkwUyBFV+IDiGSZj2rPhi8f\\\\\\\/WPWtPw37eXI0LgBu49BPFdpo+113SxiJnJ6ghLbECFqTHpaBybN0I4pL0alGtX+cdm2qY5M+ItqgNg2EUVyoxRIbCqjoI+MAmXlowq\\\\\\\/UG00Bu1aq1KcZVfk259TejYu0YFemk1KFjyY9REEoFN\\\\\\\/Ank3MeIPZNu8RuOVLpgzM7XDBXYKftTEG\\\\\\\/OHDZZykG8\\\\\\\/G1xbZ++Q1Q\\\\\\\/foL7i0ZHHUx3ntPKNlD4sEjntHLZniMEeoGrnUbCphE9pt2\\\\\\\/sMkRnplOSu9rubFXcU29870\\\\\\\/hDIoRW6cPYG1SL8AsBodAWrD6XRq+X1DlzAXzTTexVknjLJF0ARYXOGe2CGbl6V5vfJ55SRHairOK\\\\\\\/mOFFH5UonbVejijb8NDeKOwfjrs4S62jRfYgWmdaGvWOa5tvrw7OTCIM\\\\\\\/6P24IXMFdKSl5Tst\\\\\\\/vGFl\\\\\\\/vvOFe6usQKCWj7UWhjv9abDkpjPQ4ZM\\\\\\\/9wCObg48Szy20yBbGDeX6NgLAIbpKdVfEyqCxB9M2ve5jNHqDWioAk9zNQYg\\\\\\\/FzDbKJ\\\\\\\/5deXy2SOXWC1ppxynRVwP3yG8ykj9RLpJheDIFhDNuehAUsOPbZn5rQNnHILLWWQlIL068irzUE4KIDvNB5ZyL0NNXtX2fENF3PgChS14GOdDSPF+MfRIx12AZHLXVoYaNodTWT6ukTt0vTq8TOHujKPJQ9YPE04TqLly+k5IZftE7DcW\\\\\\\/AnJaxDtSLlpsBCddxgtUFwPmgzOB30AYVJi7B5ne1NBDmQ2NM9ehHvhl99cXd+g0C+E8pi3umYa2kDhD0atZlC2uarVFZ\\\\\\\/yMdNGYXRsV+cWeQuuOCYyqF9f+E0AGeLsJCIRayEwg2GW0\\\\\\\/lg2wjYvg2UIrc9HYeAp6FARIyukJA4k39gc\\\\\\\/PLae+3\\\\\\\/oB7wSklOrAE+WFgTv6FV4VvmJEe1YozTtVbGeE0Rzi\\\\\\\/xjwpwaMcyO60aZZIzexHK+GPKuXkDgc8v9IdLZ81eqIEmLBJByQV4RMARWmyTPskbtrT7Zd8sD9MO6XCVqDQdqOc1sXoAr6ISbgUsF6bd6Ep0yTDeKPjqIU+\\\\\\\/bI\\\\\\\/IjKQAInqMGsegn5K+TFUOJAdXFKTA\\\\\\\/HOmoYKt+KxKujB0pAPOOMvG\\\\\\\/zWg0kgQHm3j40wd2HINZWlg0\\\\\\\/Lhs\\\\\\\/z8qAmo9dRzkptIMiQJ8rJFiyEJt9cITku2q9Q7pHPVEtX20XQzZM7wmgVMb7ApGGcV4j28amEipALq8gcBIxUejKYi9cpnGhy6ZhzIVFmSkg3DnYP0H4cCi25v2nWcsg3z44Vc1z+19e2p48tNOuK2OlaEYlx9AHzOZro1z2CENnpVNC\\\\\\\/O1uWS5If0ICOshuHw975OOKs14UvyKKtAE82SNRuAoYcl7NZV8pb+FbBdHbcjxF9nUGSlpySiybFdlPh7ImafCEkMx4Vb+8NBeBr62nZiMLgf\\\\\\\/QkVyHKWM9hnloWNZF+L5vWkhCUTKutFFFYnU7iB\\\\\\\/8Z+BWXgiW95z1bnt8P9pUhN9\\\\\\\/0gqBBFRRXqW\\\\\\\/E99wI+7JYHrsdU6yRUBmf7xOyjzzgtM5MW3gVK2r2apky3YiQZKLMXyPzb7fsAibbVQSVGM\\\\\\\/0ajrK27BGM83iHfzoUzSHIJZTC\\\\\\\/As9ejLjXlEf7SdJqo1H6T7MMuf6Ot1+T6n0Wkx77TX5ZarWmILRLHJACK7SPtAoQtHcY0X9L1HjXc\\\\\\\/Md4MD314ULIK+0FUeL\\\\\\\/vIKikYgpgfOltx2Ef\\\\\\\/iT0K5OsQUvaKNXCw5qFwjjqp\\\\\\\/0mQy2xi6ihkwo9DbihRRt\\\\\\\/BpnxzsCs8HHqhKwmPUiXu16cGygSe\\\\\\\/9yj1Lk8slcNtPz3jA92mYWzMnvcPjAn1RuNDY8ao57EeFMJnLlxpSwW3wcwWR9yEpg5igK9r9gp\\\\\\\/rc2hoJgNSg2PX5V6UCRNvEkD7Awml831C9jaJlLnYkqZKbWmEzJ5BFBt\\\\\\\/R5tfUhPtyzEdCPDkyWFffk2P\\\\\\\/nAiXYyDVkokaPm9m12NZV4JqB8NtiadswaPN9A++361h5dvsO+0jNQylYDGC9\\\\\\\/oA1eMRtY9MpAw10tGiKqbmhkYq73jgKBrGHFzhtfeHLE6i2ZPhx5Y9PDa9SCNEYd5vQqjcU\\\\\\\/ycUwgH9mhJxL9VUNDFRlg4VgXeTh\\\\\\\/yse78ibAN2hiG04NsWZNjxZ6ib01WBTcFfxLs7vZ01xzaa0Bf6Zyh\\\\\\\/wqIXSbgOOzM3hT5rvJafbxfvNWhCtjZMVJWpRrWatHW8SBmm7Re6UYMghL8i6GaGnHjQcKldGXcAdZZ20HEz6JpK5do6AvAB0XK9EQzSj\\\\\\\/F3oaIU1Te0QbWxdZku+eUPVuTiQYhdJJRkQrDsEBBuH5rBdeo1dnJDZyosNQQ12OUfB5r1gGRNF\\\\\\\/glDBSRcNhT0bALtKAb2lLmvtHUgrHMMApCqW08aWjYdQOwMfAYkkO4AzwloxukLfnJJ\\\\\\\/jF2KXxnMWv46TcCQUANaRNRgWs2LJldY5VzPOLJ4mh6xkBr3AVCbtnWM7AkuITgBEfqB6B7RF4ardaWpty+3rJFnn5apWxjF8FC6vBZKC6Ysp\\\\\\\/ii747YAaVtCTFc3XffdQ\\\\\\\/0Gz9SRhIrCN3R0pRsY\\\\\\\/q8nxs5rZiEnWGty2TGduEVDfgXFJar\\\\\\\/cMRvhigwZ\\\\\\\/C9YkOA8gszeajmNtGqFIX6BE5foUio0UNdFx7Zb\\\\\\\/EubbGaFOYnmiPBSwBs6qFUTnwzvxpEQQjBjyp4GuIYP\\\\\\\/BhvmBTX4KgRbUawAf1\\\\\\\/VNz0JLcZar93QO\\\\\\\/cOav+OuOaRxrUZDCLQ3uLx79fHAzwO0Q60t6W3OafaYAY+TPc1z\\\\\\\/Cc7oUAmeFF7LFyncP\\\\\\\/zEUHuuXXMVo0wXT5KbzgmexJqvwnFr0xQosCJiAfZ83+doHSMNHWR7bnOKT97s79CSy2Xsupu+p\\\\\\\/qH1gRNWJa3nmzdjA+CS7iVFXzI+QOwIjKVKLmNXMH8YEM781K+GzAKdERpmXUyI0TY1bRz9CErBzs0hAWk4fQNBebCTNwWKwUL61zKl1gj+W1I2uGVou6LzL1ejkAV8KZ7xvvuwH\\\\\\\/VS0y\\\\\\\/9xkihU4Zb4B2WvmAlWpYwKlifB6fYjOmSWqotYQDPQqWiJitYqfmBfX5OThbuZhG5es2xDqAAtU\\\\\\\/\\\\\\\/9GWACfR+x\\\\\\\/jRFZGpzAdl2L\\\\\\\/gv3+bOzNRcqS6NkAnha3QVLpmzN3YKYmqVyLBG5v2L3rFxEUtXWl6+vvfnmyNxG8xf+RfiUX5MuZygKJK9CLH+DeC4YSw2dYoNqULt2DtYqBoVkShVgIf53fK4QxH\\\\\\\/Kek7L6cSWIUPHdz9Izv48R\\\\\\\/lGHZPWCadkEA5G4sIZvqUh7Hi8iVwcgAUiktjxKZKG0ncJOrK3LdGjHk\\\\\\\/ZLueA7FkWNXxJ6BN2ZtIHJaIFk5BB2y\\\\\\\/bZzfiWTyPepHkiQB4LQOmF8A6pAspN2YjPzRFy99vPV6Cxt3Pfj6JpU3S6nMuJzPlbf1lf7tKJ+3qlTpedC+9IUP\\\\\\\/5sP434QzglkGvsw15aJyAUHhZVe+w74IDqLUko84TpEKum62PhwswO4pECQouNd\\\\\\\/EMsAJRap73PhqsNlFe+B74D7pFgp3XgoVxk7rI9q24zKgqMGuEvTyfIHFykIuxqJ7YebYp3s4kARm\\\\\\\/k6Min90NoxwuzhjSBbxrjmeN7ih6Q1jkgrOVnXKMFKvtK2+qtrpwINGz50vf0E3\\\\\\\/\\\\\\\/8hh3OHcF\\\\\\\/8YDvqIEDyhmA+sDbo1ozwxXf+E+ZL+qs8bhf+nWiAkcIrRkEDNMsXNt4Mjg2IWpHE5wdSOKm48RrRZFSU1gD0ZOpHvDrtD3derPlHEboXJotYJs6rXz\\\\\\\/UWDbe3VhUiezl17KGCoGaBeb+5IVxsmF0HVR1Ac6nMuxOLi4UmvaIzY320n0L4r2UgBR9O6T5yEYMm\\\\\\\/Bb7WFj6MuHnVPTAN6Cgpsjy81UTgP24+Yj7drdgnX\\\\\\\/Ctm3uvaPOlg4xiKVf0NfkTsDFgMCI8U6EzCCMFyi9CKkKs0+XpMWZ4NKFF0FSrwKmO98hS4ag2V1DcDGGpGv0q84NG8YnV4rYbm7Ll8Medik9YIxriv+3\\\\\\\/EIE67l6TxCpjt6bCqsCu\\\\\\\/r7urYViF562sWd2mIvWTWF78QoaZM8u8Pk5eaQI2BdjXI3VznfKBYO7yt3\\\\\\\/fW7HDOSwylBx4yF1y6rEAwiPN8\\\\\\\/BlPWUVLjWkdlXKV1pwaJcdxqU6HFHS9Y27WUU9jUTjat9rmbHj976DIm5HXpHVC0brmfoUOnsOsqmBeTC5wJbJkW9\\\\\\\/sILqglY9Y7y4YB8Q5onn67L7AvTuvR9tcDhCDI4HBqfzmSOSqyFRvGE07FbIRZCtaChz1XMpnJ2YUE0xXndsd8voemAE1Mg6IhLRi\\\\\\\/ghqHXZ2uLmr11siX\\\\\\\/lRrhr\\\\\\\/7+gPaVg7Zv9YYHVWv7lL\\\\\\\/g+99PaCi54GKoCiWB2cBCNjxYJPzeVm\\\\\\\/XW+TyO4oghLPktgvECWVztR6f3wO0rWGVycIsF6cLg2NVICdnIpIb7EQR\\\\\\\/p\\\\\\\/UM90n3dYnMbdZkHzf6D8DFxAy7G7ZHdo7rkBU5MM\\\\\\\/lGQrgpY7dYtDb0I5QCIgilZk9M3b16j9PRBhEbej8tHM6D4o\\\\\\\/VAHzgVJmMJVHVgj4wmpZ1jZATAxtBg40uDsM3Ss6pra+ScIU4ppFC8J1gddCiyniRvyEFSkAiL9nHx0r2Yx8SrKviWvtSHYxSL5WPymP60HwJJSFAsJy1yvAuMd0nt4FmYT2L1p8Lk3VGFwZoSZqZbVkylGVf8iJH6oXoTVl7ERFUdgeJ1hWl41rg1LRzDHTQQNbcw7\\\\\\\/Hqv+YlExm7IZ6M6DHOJZzY6QAo1s7FAeAw8NLPfo71QPDscYLC0zA\\\\\\\/HH9YloJtHjQh8OhIRBejZUynS2leZ7c\\\\\\\/bKnmiN5vW8cTYv\\\\\\\/dtl9s4Wiu2kDMS2Qed205C1vJkH0KeVGBhNRtSpIv+AVESVlYyHaS1VbqflHA0hJnbVFFDnbcefc3UVIucJcXRmpbQ\\\\\\\/TtZY9RdOsJ77k9h+E9Yae0ZHO1wQP4AWKEcxA3ME2GXXi5eTQ1Kxa9P331K5LyI8uwqRqvxANvHlMSYknYEr1OtWSqqFlCllBsJS83IGFChUYQlKqRFPCLLQBNrSEALYjhZmL9iv1hBNwRKvh3AolxOmzQp1F2MvgvOGhl+9DPo\\\\\\\/KzCoYsCjHzh\\\\\\\/tui7s3ShB98nFSn28hsP1PXQvXyWAMNG+7E7h6NoQWxLKVlziIAVVnMjnAgG0UnzywyvEZ+L62imDj4L2zhT3doObBK6iZt2QNmqiLWdjB2eL863mJdFKt24iSoD7KuO6mSDmOsD7MjeEcj0ULG4k1fF13H9PzuQ5MnsmS0IoJWMslDgyI8D3KPI7p6hWALJXnL5uvKJKPmKLmx3L7vENhThQewrSg0i2XGYMADE41reXSi0\\\\\\\/QWOqCG4gh1rsj8SJEHmzjzFMMQRMW4rLolcTX5UqyNiJNO+AKHg2Qi2eEYDwmTFYQodgn9v3FRVEAJoxnTPkD4A5jR\\\\\\\/HxLw3IzmpfxhHSoWMVM4Bj8WCTaHVN93yKUZcuz+YXBiqJkmFTfs6GDvlmwTEgJ50uRn4AxTvmIpk7CPIfxLk3EEUKti\\\\\\\/hx6DwBnMQoV\\\\\\\/7uo26PlU\\\\\\\/QnuLizQvu\\\\\\\/wPDRCCkmS0n00LZlYDa6cltzp8N27X\\\\\\\/fhVl7cdMcpRy4pmGses\\\\\\\/1DkScMg059Uay2jypdf9z2I7VD\\\\\\\/JvNJsm9Vpea5j1ZcmJ\\\\\\\/qN3qCvhBZN9pFa7G08fXA2xyISXD3LUFen55kafpR4mG4JjSbuKvR2U6P4zVJ\\\\\\\/OlM\\\\\\\/fXBdkEJMBmEJwvQzjk9blbaPdqROkQTGccHEjigcUUL7PAhPZWWvj1oDBc5h5kPRs7U5tIMF+6CIB1d+m4BGU3gYb0MSxSZkVeHDOOq23N5a\\\\\\\/\\\\\\\/hqp9Nd3y9o7VVpW3C\\\\\\\/qxFFVuuXySDs34J\\\\\\\/hfP6ElYNL9B2tKLv5Re2CUsj\\\\\\\/lKg3ozUvu\\\\\\\/eBQI3GPOCtyf+57RdVQGuoUdYo+D3JeQ8ilgonHs5Uv2YzvRw0M9LjErUMAsbDfOq\\\\\\\/sfbbTR4fFT477A3nyVlYz4vHvq0clvO8kSLYVfk7SXEs9EsRHcA+uv2KNRTPUl\\\\\\\/z2QY7FycHqU9IXtiFmfpCdpr8gtPR513dCULRvfIIFSxInmFL6nTyhm2Y1NS5WkfS1pk9h5JBeFDSvyUGHZWgyWalCMWTgqB35oWlwSIcNUWO178LyyZFMS2G8N+kaUxC1wQO6FJgK21qZEh56MjltsbqhrFKxUc1gqbNBz+Uvjl\\\\\\\/MIIIctWUxaH5Ap0NG3aiqmeQMIona3rc\\\\\\\/3BJHsXh6UbwD6pbC882GT5C53VE3lqZpMsCMWgl3tUpn1cqxiZ7NOukRxTZSL74PEq7l6mvLk1xOX0h8WfZW8dm2ou\\\\\\\/wyPVgsrP6TWER4P+2A0Ayo3D55ELb6UBSkvQ20q8RGauNqYn8mRb8x2kBMypsYX3Pzar2MUGADDVqIHZjW+xWq2U\\\\\\\/sYYxBX70ZhrcP8aCPoTHc0dCu9bV6DkAYd+k+NrvdJcfg\\\\\\\/2P795EAWA+dePTQTSKKX\\\\\\\/gDId59CLH2c29X6YAALOkKl+d+ZqHv5udPmDfp9R+BTIrl1FBIgdcQ4IGJ2NhASeNRgv1+b6OM84FgY7hgohZiF2KLqWQ9WVNANO1lRpIbQIC0bd3z323MQlrga9Devqe1jhAmsbEFqMuBxItE0pfdYHdQzJ+YzRQIPKGTzkmDd4Ks+8HPlcFoCc1VBM\\\\\\\/DtvliEmeWjEWnK8Ip0li0cq14CzTQxZB7nuoRrHKpniSEsV0rn3iGtVUm0xi2vBOscEN47LWhEO6c7A8q7\\\\\\\/Qh7F6bA12p3bD97WXlNlG6qaso6CVeyCDNJL8OaXirMClK6wuSCOcTFP8kzELYKCc2qahEMjnuBA6WCGGuek8eBLp\\\\\\\/dPxqNa+VN\\\\\\\/Qzo1hY+YKzimly+\\\\\\\/pEMOsUnJcB35MntKHMqq+64iEukrEUF17GQxPg93\\\\\\\/Gb7dwjMHYW3IrFXqwjI4MDfwsfLDJ7eSZjgoYGIkfqUossRJvGx5E1J8FR3GNEZtWo38T59FsPmRjYNhxCLKssVaADa4AhlNqfLN71G7u0fmEXztTxaikA8OAKXjpujRYeeqQcezMEZ\\\\\\\/jzO7\\\\\\\/m22sZc3Ab6EkUOsFpRnFC8TPRr\\\\\\\/kSfnliEQ5e3iwvsrQJDW8N725F9yNcBweCoKS9g976mylzyOMYHbO0uXPOe4NjNXvgFtArW\\\\\\\/dEDzgLac\\\\\\\/wDOW0mTxxO8iFum21cKsopfh+dJrbYOJ1UYDJoQlJCE2dLV8XBGKotC2LQpD9TZ2Dc5g5TcDxwyN5IH9ECfB0NPxg7mNbQu3D33O8jnvt5CRMmYXPO3D+1Ogelm42wnOfQPBXTTAqhgL5sHCAPIHX9E6E4UKEIvUlAvCfZqTaN3XPrlrkf9\\\\\\\/UxL1Vw2+\\\\\\\/xwcgHWIRNVg+uNr01zJ2Zhj7GsV0I7nqCzVz8xGjtl1IHUFVVvytgDR\\\\\\\/TKzL5\\\\\\\/9l\\\\\\\/e67SiXO19sajztVED+JvbJcnAObfhDcFAS6vn2WLVgOwX6RpwfbPaUAmSOisWZsMCfCNYBTIDGOS8rRGm+4PX0I6mI46pDWaLt6lCBo+KcaOxLKcUhzVwoU8v0FDTTjZjPDVQ0lW7NXwXOUhdqE41RHVG7OSQY\\\\\\\/GXbZO4xc+OieCJ2aWWi75aezqnyWiwXu3qefPmCmzvuJudl3TR99aiChCcEmvs38VDk4hyhFzIl8onvS6Lufm5rOcVE6CsZ4rSxhGKAMf+u0yMzmIHhziMI+\\\\\\\/LzzfDzmrsFJ6hcHFstomA7FcyzCSCRuWlVpsGnX1jKd3pqpxeS\\\\\\\/T7xkgjvanriEydxsAxh4jeDFiCyFiKyGp1ZZJqRpIXCMQoKXUGUzkPMsD0fTKVKPr3eZDXuQLwHvP4V5L9v4oKEeW+9zczZtUBlJfmftS7kijTVqRgkg+4swAjBQ\\\\\\\/Wv8vGHdBM\\\\\\\/+Ani3xf6k7UlUyeHUE15wPafMXNHG0mUpbvH6BAda91arOCPIDknHLtepg+YYPMjK+PIVvPqrW7wW6uLNujc9ePohRiaXX1vbk9UPEz1QbB98Fzwho4K\\\\\\\/cbAlIJMC4a4F6k4YOt8q0GYEk19WO6RfMvHgc3cLf4coX4mt6JQUWOT45JdFY1hNw1sANgAaJ8iWsZBJeL2m5dGGEmsVt2vj9y5eDFYpO29COor3vtVGow4wX2Vm+djNfpb7BKP6SPCAVMmNEr6xk0CxbhJjLIE68GpO7JzjN8F47DK4\\\\\\\/sefNzK53zSZsVo3iH2yrg8aOxIM0BbESyR21B4gBWhuGtjH8Ce++j1blooRrrOCspHZYNjiq\\\\\\\/nXbbnHMlFXCGP2k7YhXornDIEkT8R83GlEefg048\\\\\\\/fisc8uLrMDb4OM9rHJ6oBhOe9lhrIsSfHKUi0Z8NnGY+4\\\\\\\/4lXOEBjphmTrFOy+I0VKvGf9PdfWJCr7znsiLs5sCaiSSb9FFw13G0O7CbF+06gUvG+Np8bZxe3HQta1w1d7fJKrA2idngzcKp5xf0qyE0ELBMZwqGu1o0YAFatrbWwB9V1KgJJeqJQLGhSb7eaR+EL1Tbgpk5aiLxITBwDHFz5D5S3kiqlwZ+08w9Lhtemzfou1Byb2Qo4ZVBHtO+fQ2qbjLBheO6pqHA6vWBAZsMJ1Z3aaHIoIitbhbSb6uCfQ\\\\\\\/uN80EcVLO0mKWIyusRRlkVYrLO7tcdfK0FFOYfsDwIaqtqeA0DNTCAixCirl2fYgO08U35jfCqcGuyTJQcbrQ2LahOlbesYYH2n6+A\\\\\\\/HeMxrh1lzCSSjSOERkAUHv39Pub1gFm3bj8aZVeD8DxRmtU3to07Ngz3sva2rbzwi23z9mOUhY0csbHakxDlG2Iak35gYFdvpNBVP2q5n329MUI0pWwXn1eieKnBdwTbwm7ItoavmsEqaqYgrnxdWNzegcGeagtjJDBA403kIRfEP7miihqiP5p2dXsIz2df4OF2DlK2Rf2sK5xT+ahO9k0pjTWtPbbvFOj9u3APImstmOeGMlN4aSQYPvKN7idyjeGH\\\\\\\/2Y5lLeJsy0gVMFNQpwgip38rPvPl7bJTpClH4YPIZBYvSXX4QZv6t7RtLlja0U3PmqmN8wOizTdWUMTzwjhN8ZCZ\\\\\\\/vIVZWeaz1igbb+WkBW9w9Wvfhb9JLKAZRENHy7TutQdIjoQ2X9LQbZff5otbjb2UOiSXw8aTt3EfnOmXnFImYLw72whuVwAP3mmVe\\\\\\\/CvCrrjnwabGRzU1lVaZ9qO4VNa5oDGTJrA0v2VmwNWZQRY5rFzsjSphUMD+eeDOSwj0PsKaHjjMSc5SzEs1IoqS70DWDH3hx7nXJkKc96Bvs0dsk1YKldRIFDoECF303hahsCJzgMdhBVqNPJd0\\\\\\\/SjqCf7\\\\\\\/CJTQte472Cc\\\\\\\/cnEVtsbMk+c+lSxUIVweLgTNs8T0QIB4yRKNEur9g3L2C0m+nhJz6NYBLrAo3bYWYQcjnAce5Gqk4v33p365RtBPnMbvmTJfrGMSjm6PIargwrJZNSZv4qbIKlP3i0VipTBMrjczNrZNr47QgwFPZwnqyFD9v+8zdqGooJ+WnqLpSfQjh1HSs5eLMHdhj8uDcAkY1onkwiifaAnSVYHyEvPyJCPq6FIjCWYig79lFu16FqVtg+S3BvGg54nqSZsK36vzrZ\\\\\\\/6ZIBVamAQyhA5SEhrTJGzDZXuWr\\\\\\\/vyJRKAIbajL6civ4pTWPIMz7BLj3FUH3b4YMGzPBXRb95C+LloE1i5oKywwE11ta0ssZp3z6iRNLkjvQDAYyQTR5KgDww6kY1UoQ\\\\\\\/n0uonNE71eoxJYL8qOU+98UAGUcVD9qXcF+wHsqhIl4AkieJLHAxK+f5E4a7aedfPLrIjz8j2zB16QZFsAW90wJAMtCDr4fs+xENhIyZO2bbWGXMQhtz6Vl1N9Dz6luUATl7PJ1qK4iR+7g9MpUf+ssckOlFkAGe6Jf+cKM41pBkWVJdThd0ZQsfLxBVp6fR9vna4cfCyKt4mAI\\\\\\\/J09HmEc8NTcuHiHgtAx8auM9Zb91SuFrgKwLWz6UYxqr3688H5zb6da0NuBSm8rVRU1XgeLOHLdhCMF0Jzt0rphcz3yxhqCYSJkgYwHIResImFweMP9ue1yfUsmZfvzvU2MDeGk6F3GwEK7qq1ri006sihT8XIcoCR9DMH6GK8vHP938pA5W2cdZwUo3l9jR1LcHRSHeN\\\\\\\/+J8VtNwxy9dQa0P6hEnp88+yxrWw6bJnmQv7poNQ7tMHXSy8qmftOT+xU74Yt4p5sIrgXaBJgzStbxQZTf9FM97MXKyqhpXzEuPaefGbpVEbB4ZtWjLEkqfAJHi5GWAIn3PdC4Ve999\\\\\\\/SZOOGxRp\\\\\\\/NC5Vb9MQa+EDhiRIFztrrXReLYi7\\\\\\\/RGY+\\\\\\\/WW2J6MNxhHRWgaCUFEXY4D6S9OBYWLzm2MVwXzYoi4H6KXle02eQgnq56lfVD07H+9l\\\\\\\/Z81K59t+sV8AW+yF4Qcm8j1omcpUZ5Am8nx9McdOCqQuHcDtXtiLCvKqpn0YTTXdEc6+2dVa9O0JrVMGRftyL8iccrM3jnoPqxT60+qr763zJBHhMSc0dE9JOgqjTS8plLTFKaZpb4Ug\\\\\\\/FhJmn7GWCcBXSXoWjFBTVduGG\\\\\\\/BrlmjM3Z8Y5hucMbqPJeBcX1V4J5r25mEuFz3LlrImLpna1Y4p9Y\\\\\\\/Cpqx5kMNfNjFNu\\\\\\\/eWxqdcbO++du\\\\\\\/XFrlwy+TWpVpDH4RZYGF3m8sxWIwFF6Qt7hNGvcPiC+d\\\\\\\/d1Th+apXQzyb1iF50LuKHz3YHpm6VB+Sg\\\\\\\/bjaA4A5CIjGEyUp6uXcpYD\\\\\\\/MQQqvGq2dh2B0yS0D+nXm2bUpDX+2QgwQmKacpTm+cCM06b5XlSemdnq3oZe8nwybrs2YTwkUntN6qeWpXnZNFoRGk3FWbk4d\\\\\\\/mNGg9vlMb5sy16pXUZ\\\\\\\/ztyyjsi8KMu0\\\\\\\/HeZNqchWpPefEKetSbLszTrHiHU4vuco1eyhZBFD+vmE7KmmzAOwmw3PM1SGFlh4Pkbf2ixKcumlKuSQqg1dnHpENf8+MaRXb41af3hs8BfVSa5E5AdzXBi\\\\\\\/FmeVY6mfo5KI5clPpwX9nhdB407hWtbiKBih1bEFi31ZGKrhZ\\\\\\\/M3Ul0oGWJGhLGo+nhnIEwqI2LaTBhcHv6uTKT+SwCi6je+trdpPjNjDMM5OS\\\\\\\/ycUyuQ5cRtQceVArhbwoSQYUgXk3RJede5WYL6SvJOTMN3H0MKQ\\\\\\\/raocOa3Jw2IQqh8KpnQL7GoeeH3kNhZDz3JqEQqKRJq9GhMyAe7jN\\\\\\\/wx3UEtGHN6YM7WlGXCYZEZY8r08zRvxxQGrQiNcavxT8vJCMBQUOIHYelwVWEeXigXO0iiQ2GXX\\\\\\\/77GmUhGdAw2fm9rJTm7J63bhu+k3q9L05tKd82+fj6OvUgj7643LsGpnJW7CuiBUyaADPV72\\\\\\\/A7hPWYPrutoBnwtYS46cQ3d7kKoGvmaFooRmeLXU5C70Ld1Io2db7szzeAbMtTzcg\\\\\\\/lOMUGgaTkT0vPc1kAfG0WH7K8L4PaVn6sLzVhZl+gliYNp9AcZAMKvRTxAFDk3PKsQU0dcPJH+YEJFbpLvatUBqumnzxHR\\\\\\\/sJIisSzmQl9cy70ZmjIjpk2SDvqbvH5Xxs04x1qwRdFsSmaJhlFvasJ4JlqVx3Lnu384\\\\\\\/+kIqRLCRgzApLds8x\\\\\\\/CBFo8miNyJfBroWauJWzqxvFQlpRm0yAnN4\\\\\\\/t1sG+oLfA+YC97Y3VFz1AyLZCzxRxsGQ6uE2vXtBJR0oJuYDszZC3k9l3EXHR7huDR+hBqBLu2fTVD9fPGeudl1OyMq0lrBN7MVoOrzUuiD9t3I65kumQpvOVUtpYtjbWh4AFUZRTimjNR6QOqgA66FPOF1cLkaD1\\\\\\\/7W8eZGw0IMqRFRq\\\\\\\/6GpLtRWZmc6eI\\\\\\\/pbhVRECIxjG5qybsB31VT9QXNbwQtJXKHWrcoh1WmzEtwxGvhGBAgADDXtiiqiMVUShMlRq9+8hhzmGJ95BgJY\\\\\\\/RcxK6oswTWCIvWUYzU11zjSD5EmxnadO+z6bbn+Ek5Rxy5Ej08\\\\\\\/cK7brYTfMtlPuUHKIfsvpEWu2MvIkcXLyk4pA63aJSkLXrJp8FE4pOGxdC4NvS4k1J8ekfOboSvdJgNwIK7tElbxDWOTy3lD+5kQ6HiEx9O\\\\\\\/TF2jloCmsK+N\\\\\\\/c4hHijwQc4I2po\\\\\\\/g3RROAZ5zORoUdTYuiYJQkPK7rUBSIUhPpxg2NmtWMwAgD3gTOrkHcsl5MpPdzZzO2vcHE8xIIqU53N+Kzo9rkrbB9\\\\\\\/GhVHBsUJUW0xUh5T4aK7XhTtC0+rej\\\\\\\/9x5K\\\\\\\/SHoOFWKTQ1dZgWfHEH1aj145+KcGl3HEVrlkQIMf8txzgoAx\\\\\\\/oUMnLOrybKF0+w40C5FCxFel4U182WuBEPumD+Bqd6C6F01lXYG88+J\\\\\\\/3PXwO\\\\\\\/m7bEdh4YugdMgxvPPup+4N+wFI6SHY\\\\\\\/urYDnXe8enRhXg65dmaQ1oRTB9IxmAYFnBVG8SICkgGkZTaCHeSGBhmhMSCo97Q28x6LZZTxvef6QuDb4k9RwOI5k17gUCUtg4M2nMqrEkp0oPqbnlkLq6ZrtKrQzVXLqNav29dTCVaGP0JkTS2ZevU2L0\\\\\\\/O8lvnXFUb2ZYmtsuyGIJnfKKDG2We5A+kXAImCRY6vF0jGEqRDtKTrtZ8rhZl13j72AL90O7fhuSBcQwrJpBtO7gy0lkX\\\\\\\/I8B37sqwOJVcCn9D\\\\\\\/xG5btT+aeuXBv5QFUBki++kX9PZ\\\\\\\/1xA+dJhNPrUYxVa\\\\\\\/uEklGFj3dwtKmfAbu55Y78G8W1NArm5s1u12qTdNnuuFg0Kh4ksda3uOsp\\\\\\\/0beay52uOtQ\\\\\\\/2EgmeCTaVHS40hpfvre2xTnsYvlCF5EFPguMkvNK7TI2tTK0bTVrQh3e7wE6CTSyAEBKqAocKks\\\\\\\/dky2xAfSmR90KxvW\\\\\\\/14VkzzRq8Jq0RZ8U5UiH6dMG1BX8yexbToPZQC8VpPQdq6LPK5Ml5GbCyD0bC7EsE+nB4WnywDacMOP0Q7GN0jEBc13VL8LgzepHqJe6apQiDD+P72WS6K609g0I7V2irNzSnMtfoKp33PFVGhzT2jMMdoGANEYXD+h9rSKSi3Gq1TgKTezUTDs44GP6bjvtmcS1eEQjMu8hQF6kyYAOJnAW1ihWGo0PykrXsXNzRoON+9snwCZDGRQJH2UYA2L85jvasX3wDmD8kwFnuQlQ9lwjctmiQ4X0f5yaAM+Z6Nm+I2Uvb6KwJ8UP3NowYb5napHrIwWpTtxCSYcHq\\\\\\\/81ePXarWk+K+uU7dmBl3y8qufi9Xns9+2kNHE2062MNTbRreZ2Cz2CCehRVCqrcIZyZa\\\\\\\/cLO\\\\\\\/cyxK2j38kP9ORt7FZ0sKGU9Q6Kqfg1Q1yZzQubJEkGm9qeXKp0Ydr4sJ4yHpP+LTa3UmLd0AQ7HU+iTz63bhJdFzagLrq1wc+uRG0zqUtKar2KjBni0kr+4Ef7xTsbmS3zIuNWkvzw5ojHrgPxkaHdSd1j+W8vCN5nycrpZlWWdPaI4ttg407sobQlrOhkDpyyZddtm4W4pvKbt0C0bBf0GcOnU9rBm8LvdOGfZXbFZL3zrPsJfn9\\\\\\\/cVq2VzQcTOyMn9HQcnOirxglB8GMTK275KFZP8GOAAiraJpBW+a27mEAvV06+CpwlSR5GL9fidWW60IcVlJIMB0hyWTyUcjFY3vUW6tedbXoz9g3LJ1EsndYl2b98kU8oxinssG5EM1MnSi0re9RzsfbZ3jq1XJwwggu6WWCsYPXwHZMWcpxwmvRSq2JGu1sry2AeCNzGsEzLwsYmpp9IEZBA4DLWS2L7NRl+JYadKzmhBpZgKS3B\\\\\\\/ItKNxdQmCwTr7+05vbeHhJutG\\\\\\\/Yain2n9we52\\\\\\\/gtF+OOpo9i49lUZUl0CbtNMrJPR7VJIPurCV4eed+sZUSybiMH5YtJzT3netyhKYO9ko+\\\\\\\/qXJrey0zB6ozF9xCRcmSRoq3EtlZ3QrNpkbVmcuPwL1ZJJrAeWg2dS9ws23v5mHKejG8HnplL9Hj6+axdhw3jiBWQVhRqcgwH3vke\\\\\\\/yZazUDxbU4Y+Jx3DBnoj2gXaO+eP+QAqJbvxxNBzPk9ICPDewpubiUINSAqNS2\\\\\\\/pHVbLHspV7yecyDH+pfWfbbplfUdWw\\\\\\\/izctapzCF04dVNcpUqdgCujOs3aTBT8HQ1RUie2NxwVMSizctCBfUnn0IPNBpUVoby7pbopU9HloynC6KlPRCvkg1fchkkaHgo7jR11c44HYwc4SHcA5wvzcBHHFaFa1EzhkFaIaKJZ+B4\\\\\\\/uhnvdptj9GaXaObE6vMxcWtUqagKOGHgXVc1rOR1J2hDov1ocNbvrOQb4Z4expkcei5Q9x0OR+ga5uxgLTAd8qFjkMh5KO1riRq3pRWmtmbEiMLx4pyKG0JIhfeAlFSnHmHmRR7tfsi2bDEB4RH1IqMsEADoRvwrKQv4kGSmvLAMRf7BPC258xU2ZHX38vRxoVmLfv573igIKymIpV6IG61CtZdStbNvqMOmOzx55paiTH1O80vzbqpqGcG20I1UFHhx904FZpJ0BIIP7+nzKG6pgrFzNb18NOZDODmZk1eK1s6qD\\\\\\\/yaCsaU9fi4BxyPEMEdcecqKcAqtmqOivToOG8mNiZtYNqNXBMco6QdJmfevBVGS1yX8maRMv\\\\\\\/LGELu08SIg5tlSt3C\\\\\\\/HUj6jvTOW5YoVzKjklfOGiSD2ytjnu\\\\\\\/RnzDqNw1qZ0OtoE9z\\\\\\\/MCDb4ba6l1Dk8FfPIr4DHmm1SvBpgQ\\\\\\\/Ql2sRlwYATALaUemuG3arUrzD5nyj5qsKxZmPlgSm7znj3v71Q\\\\\\\/TZKJLfvNqHj9Hx+nB4FLWovBD7hWky1GXDYOZFNaNoBz7FVLBjTke5cv6kPr6zgEH99iWP9dIqLnUfNUu1xp6dZWy1uHORMX1SY4O7D5NASTeKu4v\\\\\\\/I7l8JFYqRS5Aa62E2vyFSLshaGxBlY4nlJ4J2N85N4kcdYHuSAxKMQkdMC52YwiItQnRDuSY93CfzH1NgmoXTMUa+K0fC3TK2Z+KV\\\\\\\/EOAiniV5tsHHS2Z9dTCYIEWpRw2rFV3W\\\\\\\/krt5vfGB2BQBJsFQiB7cz6CwWRg470EkWYtHmXU+0FD5c+5u3O5WQJxCg6rt7DDiKw99zR71PHAc+AD8u346WhsfOXv3xD78DS4N\\\\\\\/3fEnTZ6qinrBB4csj+0auiFeOTlrhFkvT9wnGbUGHCaPgLRA6faRIRNi8Ui\\\\\\\/hJ672\\\\\\\/6lYJ0pztDEitdVej7hsabMJxZHtU6S2mC98HKj8Xfwzz4NgM12AqXkTC4U+\\\\\\\/nim2dOORd\\\\\\\/lywOqWLFMp\\\\\\\/VK7Nkgl2C6z8bL+LF1g4XZwxYs+SLQZgEq6QzgF0pBXBIkTFb0w9LFFtQk3gJhz0MVZuG9ZOutJaEJxQeyEijMC6gxA+bXzWZ0bp1xMZjgqe8nGVvy9gXe7ByI20v\\\\\\\/G5x8vmLJY0AqDBmmjwd8zTLV5o+uXly8RPIWggB87DeGYvxM1u8oxl0J22hn2X4V9Uj0cpJr5TO\\\\\\\/cZgDXt4+Lrr+qqAtNMt0H2b9kCCIj3onmTSzVr3ihFJRTRzBJ+6cceL18pWqjLApejMYuU4retTzpwOfttFYBHBKeKde6txE1QrUwvaEKiexBXtmzdvOC5QSDi2Moh2O3Lt32FqJ8ev+oZzMlOG6u04Ur1ynOjCTOfYM2ILHFYmMoHnLaJzAUzIhqfaH6l9CHx0aAgYoGsXtfGeWB0g1D9v46oXQr4IEfmr6aTFmY1q5V0g4NMRgdAspZgbAWOmpyQ7W23Jzht5IfEutCV07lEeaMTKJ6494HRwtsXMfGQi4NTJvhm6dDJXBsZA3rfduAU+eaWeTNW6KIFv3ePX2TMJhIyoDDhP6D+6cQKnr09mYmehYbMsa55nKSykJz71LJ+YmRDB6tQGExt9UZKlAqYtdhMcFvPx7v1zkPwI0TdEc\\\\\\\/g9vRHoRbTY1uaH89G4YsZ6lvs2TIkyCC8gJTfyLnsrqi5sBWLXaMk\\\\\\\/rSxA1KbktLKHhy88tiYp\\\\\\\/et5abQWkmsxNSLvvVV0Q\\\\\\\/I3SJVsX3B0\\\\\\\/tN2jxVShIut+BPvnUEIBWiHyXoO3XbaQI6FKfqGYGRkvG29fJq+kFxXhDCRm870lpNHJTCihNHiFzLfZj30xBqgbS1\\\\\\\/NXr+2Fu34pnBUdC+tl8Z3ZoKg27j+T3rgKJIjXIW2Hdb6IUsx1kgiu5f01HiyE0ZjhV9fjXE9PRdYyAdT3HeYEKlRVBu0OBxAWOs3WJOL+3SX9ZUekgqQxT6T74pBmJggvQWaUNFP3SpWRhiffoF9HKzRQ+R0fxOwOsLRHGOpmA9ED10Pte1Zq7ilU2CyfgyuZMaMiUzHvq8GVWrYu9iRarTiop7b0+WuoAqm4IY8KlaBSAiOvTBKOz5OvLO\\\\\\\/sEvItpGTDDBsZiRCF5FweCNDdyXDhqRlTGu\\\\\\\/DOHImVsKsgLt\\\\\\\/uAToAoDHHpv6+nwaGFWjKgnI\\\\\\\/Ib+DpUXV6HPQ+\\\\\\\/+imke9HmCP6tiqFu1CjpXiw9+aEJg6+UmufApH\\\\\\\/FFYTo2K3MQ8+76TmQygWGCRjg\\\\\\\/Pv1uVO4DpkSjw6uuSfaT\\\\\\\/r2fO4G06W5qn7LaCHHkdXTSvjCuJFcLgvZsJl1mwHbbYHWMkXLs6NeUVmXTjkyPQQ3dTI1GKER22fgEO09ZdI\\\\\\\/OcV4nSlwqQ69bFbOIYvNklBA0y8gjTM8HwnVOa1YNfjFigKOrqkjF8XLfihvep27+eoLCvcuL8ZYu5V+vMXyv7+l4tmtUIDdZFTtipcInLLtaxrlETTxdvKGqjSWCyDnp27X5\\\\\\\/o9k8395NsVUto1t15D7MU9T+3IRM25CqnF+oFOrYOHM5U5Y1Wg57KqEqYlS1nCkm\\\\\\\/hRxAoUXyqoWXLZYf6FuyF25Y\\\\\\\/QtZsSh6VWeBI9loOAu5FVHKLWfoxZy7yIDKAZcmfFhSLvJQyyadQMO\\\\\\\/dI9ndkJHy72igpbvvicbvr\\\\\\\/19Nggo6kPsZj+\\\\\\\/kA9fqmf4Dru54y1A2LTskF2POwaDFJllf83TBfSFtvLqy+EB0YyDXNjPaTFg1lGh3JXC\\\\\\\/am40USzlFEoxBzfnMgVA7xGUMCejDidFocyQc1BtJSaw2I9JUJegTaPbPbU2jMosa7OszaEItxo8F6gSJc11C5qUS+szBF+unUBMGv\\\\\\\/XpZzcZgrkmAfWorLpWWKyfikBUMhinslEn\\\\\\\/IBDHDI86ydBeUmAylu4FlQVRv+PwZmJvAFg5+qbbYOsjoH5aN3GPeiAAwhQJmYW0zY5HYtYH2Ws3MR5V2J56DGOq+rt8xCyGNRHpIXYLSkXqd3gZSnXPlapH3+Xgcx6Nk1cxnXjXtUKSZiQZC6gbwOy\\\\\\\/AGxVMKsVQp4HN757NPoK3TDD0GpeUP+8iMJOvY9I+U2EYi6G2RenX\\\\\\\/nZNlSbjk33H0mv5tRGV+jjxK8fd9J0Dp\\\\\\\/50RRc1h1nKqoGFV8xtqmmQFG5OFe5EIjTd+ow8lJmIylsi7+IRukxMDSA+AvXKXkDSiTRHGa8AieUQn1OByrVsfcD0PLaDEtTSTSSokOuODftaDO1d0iFddz7xihnRqbwCVnz8GrEGupjKsrtdy6or2PPmH3DYnNFnoKl94QLpQru48Il6XNQoLJteUVc6bKsBoBZ1zrdoMck99O+h5+D\\\\\\\/j0TskFo0BiqutgC+tHe1bB7okXbmILSOMPgZWfPE9t+gtQG3reR3OokkXn+LKYzPrZ2ys2QzE\\\\\\\/duXKV\\\\\\\/oQonIUTv3ndg6QV0cQXyITwWm8Iy0BN3tbjAi42\\\\\\\/yEDZJK6yvz7Wj1lpq\\\\\\\/8KXMTzQ16ywRcz2yvklYL47xt2JCHptBNn5WHhBz9z1CJhjmU7CB07oVRJydvWsOLaQV26CKssQCpgTI25A6tEeuI46IKK24wpzk+svEspDZDLu81TxBJb6ngzdVfONwO4KIrZfYh8thfX4rTQba+sEChNlEl92Kj8w8\\\\\\\/ZO3rPjAqS7tq1W3XBuLkxdL61BCOOU92N21QAJSQzvKcF7CTMbYR5Nw+OEq0VBQGbx\\\\\\\/pYPkyvf\\\\\\\/DTATwoo2uNfbzz1mpp8TRmfWjf1NKcWLHp46Iae+A\\\\\\\/GZKFZvFPQtSxzqCh1rn4mE82YrTRicWVZ7sj\\\\\\\/sY5dQo1L8AiR\\\\\\\/6lGy2rAQIB3gXSN1+hBd0nG\\\\\\\/6trquFoOKT8\\\\\\\/zOrU1u5GLWRA6JnKL4L3Drs5NdDsEqHehr4ukXYP\\\\\\\/kqv\\\\\\\/MjJNWpO89WOu74wuPms1+rvyn6MXev4+7SPq2UyOvhTOshKg\\\\\\\/g30BjHhs+FUQq5QlwckSCzdPRkzDiNRhfbDFEqfYV9SGIdSDEusEwrFLLlY22DFeJugnJ5PZzKA7veGFz7AcdX2YpExTGnYMNIx6yXEep\\\\\\\/OzLns7jBBJPffqFUWO6YZju92R+XkZWlSAp20zfSGnGZms1vSGxwEc94TL81T1NIEexOkDfZ3khYjsyovHHNen8m1toZIiPS1i34Jz+EYE1oXC8POe0jKEkoofxmIJofGLQp2TtmR5z0FxcxDnc1\\\\\\\/d2jbPDnF\\\\\\\/j70xdp3fBFbOGLz\\\\\\\/0s7szQs+mxu9Odf\\\\\\\/W+yo6A7CgHFOHPeViULffxW4LH03X8BSTpjrH5pLTlyykHesHVqtgnnSOPjRb3rtDKJ+CaoF2fSuM3ZLVZEqSA72ioXF5C0obyWGbxL5f7bfC4+7\\\\\\\/syHEuo1lxXUlCvPLSQReZdfiQHyFDO6O9zrYGQUACGGpwIG3WeALD10YVwqKGyM9+KgimTf2X6Z9T8nSuVB4akvDOphpMKer3xQ+4YBKbLLq8B2BmvKAzQ23usATETRW3Z1iAAxTyfozYlpW7cBUOp9f38Dj4fI+K5o5neRObxhN7xbvecY+QhAyDDgaNx+OwBT8KaDR5eD2a9dBoeVQvBYF1bN4EJK8UXEyzJmDCKwHiHmUH8esF3bbQPaBG\\\\\\\/oXBNTfegdDYmqEAF0PCBHU7h7WIhU7qnd\\\\\\\/SydwCa2E6y3jaEU9L5HDxAQ6XtYyeAtENYcsHCweCK\\\\\\\/AN0huxTz1A+p6Hnr9vkWEChNpG6f3ToY5zUocaht6yekUfiNpOu4r9pS5EOd3M7oLehP2J\\\\\\\/LCOKzBFe9HH+LyyskKzkma8QjiWipwTrmq8kei8LO+JzZ2nMf49KBzb\\\\\\\/VQyMdPDDPWkMDIxBoKjuhcUPTZh2FuipaUqJU3z4+5CQe9hrHYn2wW4rseO7dvg6OubO6SOoSS977poaDAIpe1iQrH23XDqfRXvD+X1WoTRKK72Xte1Lu7EvrRUhB3awlWflrxi5yovY0AMEwtKPYgcFQG5zBYd6hdW8ajsSXbCgSCYmGfwyvhb+9PufKDR\\\\\\\/XC1jIub96Y5sNPACSgicxazFf+e\\\\\\\/AqDmyKD9DMQ\\\\\\\/DGcKcsKbVP77CBeGdMXdmXD6hCKaRmKqQmVv2t\\\\\\\/OnEj\\\\\\\/A\\\\\\\/Z\\\\\\\/Osv6JRV2Y+e0M5vDsWZcMglWJyRdIahsm3+cvjI6xBQuFKPvbZj0yNgxDnuutHaxuFgBd8ja0Y3LzLGtWvjfadROQQZVqWdVU\\\\\\\/UlGZ2\\\\\\\/9iigLCjAbKiOfKMATIm7aHchf3HRS3GBciyKhdgPJLWQMR4VD3l5QybuaQPcxwpKoUtv764RQqV9P5ZP05nqVl2t047F\\\\\\\/qT27Wwpwytwp+VWPpWWbYx5jQg+9pBe7jWVIhpWgGiX4waPDoJQFa1a6Zr6c43vyYOtOKa3Qrum+Fqis7JWZiRJOGP2aqXKRw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022blkQSr5TgRsBJG7X1OnaXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ddeeac7b4ee2f764f7eb61606495ebd9456186c168fd76fa8949ee614ebcf1d9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XeGqmvhXWJjK8\\\\\\\/hM\\\\\\\/nxffMYaEVBAgfM+aNOg7YrmwwAI4i69FTen0oAI7AwAaqe8lH+IkM\\\\\\\/tDucDHOMOfHciitmpRLAUezQIL4671dZL2JxqDGCBUg9bxhA2WkmO7p7S8R8bUPOwUXfANly7KU7QgNcRAt1ZsRSMugkQxb1fzyFbTnEX\\\\\\\/Y1sz3+zNmIrvySJxeyiy3eXQT3acdfLvQHPnWrGTZOR\\\\\\\/j5DjgGsJo4GpXMZstG7oBfuLif1V\\\\\\\/RLwvMJrA550Upi3qWjcyH5fleY9d78ifQKwNya30r3gTN47\\\\\\\/D2dHnxBGyQypWxQ70oOVHS3hoCTFqI6AaxAXzjWvmuBlIIu86AlLdciiyNlOPXlJ8TJy1Xbo127b\\\\\\\/MLs00WkrRQWxUhtNTFrZnIs2Pe8+Q1mb5ScB0Iynt4XPvr9nw3p855M0G2Vtgu8v\\\\\\\/kiYuX4G62E\\\\\\\/oEsbfcUqdrDx55\\\\\\\/1uP\\\\\\\/GmPzOls2C928nbIUQAiK9K2F2kpFIuXOEjeXUJUszeCyrGEr8XCrrH\\\\\\\/fz84Se0rQe8H1zHM8xdgoSu\\\\\\\/t4ghfdxdizvHiTJF2YtXGG43xo66KsptDWuMtQX8mALpCX3Xax6dEpTY5bFSarDdFzlPRY3Lb9\\\\\\\/Xvck6D85P21G8CoNBbr+9FJlruBQoit3hJnkgjICPEj44HKJ4giGMLS1ceJkLDUzss28CFOVAXj3cjI7Fl61VajD2PH9aqra4OpkhgYWTUrPhtseqDRWczQiKLz1TEkRqfa7uQEq9oX41xXtzaFuNx0NeOyqrh7fBYDznab7UvYqJGvTIjDzmwqnzLp\\\\\\\/7A+FWh0UdBwEd2OkOgK+xcnudYMVIbNB4VVs2Ii7OESZOf4n7TaNt3N6Pav55SXOPJ82DdhQT7bv56p1ETcUoqvtN0QwrG3Jk2n24fDCYuiyF1CcAL63YShQSqL+XW78ZHrIEf51Yo0rpVouQhDbQZC7y9WXe15SyMT6yuFhTZEMNZPyx1seDcj3nQsKHsS2rL9U3caJf8Znl0LZ6046CqmLpmaK75YGeP9JE0SvDNR6Lte3anWG49fKzCN129rlUpcHCopWI2Ve0xQNIdL6EqhHOX6\\\\\\\/npp7ZJ9RYnUXK499MK7NnlKY1gGMtGibZqpDXtwX66lt7NyVprWQJtYHa5kVt25i9qeJyEZOjQqirm9MGkd\\\\\\\/kwfofqtiniQRlrI\\\\\\\/E6kNY7T6vSdQBy+L5nKJfGK8M5Dy6moR6Hp1s97OL1FSQ10xV77l2jE0Qg5C+wMsPPifWf7wCDweDWcRoWYxv+JH7jholV\\\\\\\/Yh+x3EPMu7irmyaxKhZAUtE5yuthsBwHzkJ2RZvYqIoptxXEQz++gsVITvteeT2BZfIR\\\\\\\/Bqvh27yqSjsxfwDdf1TcK4CnISRtA53tPh1d48dWopH\\\\\\\/fW1Bd0kWDwBmLwNQYtqpP1A4NkbEdaqH49rJI9uyZsGf6Uhr1JyucTPzXK55613KT3pKC0AMa2gIRbXDtNDNRfg6rQoNrOA9iTSF7W1EnVdQI5blXSI9nDRkX+Zo+sy9ZDKzpXPsUbsqLbGUxY6GZqmqMsvWyvIhyZn6HDco8Jke5lj5jpbv12r6U7W8ZQhEHczzSN5Rb0U4BTBkgQxmXkm3zKAilZJnvVtTDcqv\\\\\\\/OHrayEg5e9OBhngecM5466Awkg1hQYCMU8bFqEqLDkjFCNJVheca3IdwKHU0CngyuWNSDtQsy06cs\\\\\\\/Dp9BVRYMkoISXcz3wC5MDdzNO0aO2UxLT3rSBp8kiT4ba9ydgVoHNdlehr\\\\\\\/miHOlCNisbWb9EMceEQjEKsO\\\\\\\/YiWcOLJzjwXuSHJJ6hD1q\\\\\\\/03bQwDTvz9FVzwK8CPdX1N\\\\\\\/K6FO0HFPF3gflFx\\\\\\\/bQIhSpUCUfIaMfyR+pIqEGO6ORKuwwAyMgDLi1oUiIvVqpoPoVj1LTsJn6NLbq9ByGLRgZFgVllTJ\\\\\\\/WjOt8uoHa8zt+6XeLMl+FQeCZB+uXXMLvioHnYOh76gA+1RYZZGKzhSZET1has3Is95g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ADvkRwuBmrQjwqlRbJ2X7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002207eda59675e651e731ef62226da95e33876a2460359d110f5239af517cd1cbe3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459660 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1671" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C351TN3xWZe53dnU2iwggxqF9G0w1cjVG-O2GRhq1cs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1671" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tjZzy4FViWtsxb1VXPFODC_i1gYMf4l3Ei_kLZQ8tIc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1667" title="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1681" title="Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459660" id="node459660" data-pisa="ggg;4/9/1671" data-pisa-master="ggg;g3.114.010892" data-apath="/ggg/4/9/1671.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Basis for <em>Saccharomyces cerevisiae</em> Biofilm in Liquid Medium</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kaj Scherz Andersen</span>, <span class="highwire-citation-author" data-delta="1">Rasmus Bojsen</span>, <span class="highwire-citation-author" data-delta="2">Laura Gro Rejkjær Sørensen</span>, <span class="highwire-citation-author" data-delta="3">Martin Weiss Nielsen</span>, <span class="highwire-citation-author" data-delta="4">Michael Lisby</span>, <span class="highwire-citation-author" data-delta="5">Anders Folkesson</span> and <span class="highwire-citation-author" data-delta="6">Birgitte Regenberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1671-1680;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010892 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459660" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kaj Scherz Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaj%2BScherz%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaj%2BScherz%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rasmus Bojsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rasmus%2BBojsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bojsen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARasmus%2BBojsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laura Gro Rejkjær Sørensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BGro%2BRejkj%C3%A6r%2BS%C3%B8rensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%B8rensen%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BGro%2BRejkj%25C3%25A6r%2BS%25C3%25B8rensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Weiss Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BWeiss%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BWeiss%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Lisby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLisby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lisby%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLisby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anders Folkesson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BFolkesson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Folkesson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnders%2BFolkesson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Birgitte Regenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Birgitte%2BRegenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Regenberg%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABirgitte%2BRegenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bregenberg@bio.ku.dk" class="" data-icon-position="" data-hide-link-title="0">bregenberg@bio.ku.dk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1671" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459660" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459660/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1671.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459660" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459660/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1671.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459660" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459660/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1671.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459660" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459660/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1671.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459660" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459660/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, <em>Saccharomyces cerevisiae</em>, biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Σ1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000044">AIM1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001392">ASG1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003761">AVT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003325">DRN1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006022">ELP4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000984">FMP10</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000238">HMT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004669">KAR5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000733">MIT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000596">MRPL32</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004774">MSS11</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001084">NCP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">NPR1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004844">PEP5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006033">PEX25</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003013">RIM8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019">RIM101</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001521">RGT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005923">SNF8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004519">SPC2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001555">STB6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000514">STP22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287">TEC1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000309">VID24</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004682">VPS20</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005940">VTC3</a></em>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000125">YBL029W</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007591">YBL029C-A</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001840">YFL054C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029726">YGR161W-C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003536">YIL014C-A</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001463">YIR024C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001634">YKL151C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005144">YNL200C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028856">YOR034C-A</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005749">YOR223W</a> controlled biofilm through <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> functioned specifically in biofilm and mat formation. In a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutant, transcription of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> was induced three-fold compared with wild-type, but biofilm development and cell–cell adhesion was absent, suggesting that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> regulates <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> positive posttranscriptionally and negative transcriptionally.</p><p id="p-5">The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> is more complex than previously anticipated.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/biofilm" class="hw-term hw-article-keyword hw-article-keyword-biofilm" rel="nofollow">biofilm</a></li><li class="kwd"><a href="/keyword/pka" class="hw-term hw-article-keyword hw-article-keyword-pka" rel="nofollow">PKA</a></li><li class="kwd"><a href="/keyword/adhesion" class="hw-term hw-article-keyword hw-article-keyword-adhesion" rel="nofollow">adhesion</a></li><li class="kwd"><a href="/keyword/genome-wide-screen" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-screen" rel="nofollow">genome-wide screen</a></li><li class="kwd"><a href="/keyword/multicellular" class="hw-term hw-article-keyword hw-article-keyword-multicellular" rel="nofollow">multicellular</a></li></ul><p id="p-6">Many microorganisms have the ability to form the multicellular, sessile, surface-bound communities known as biofilms. Biofilm formation has been described in prokaryotes such as the Gram-negative <em>Pseudomonas aeruginosa</em> and the eukaryotic yeasts <em>S. cerevisiae</em>, <em>Candida albicans</em>, and <em>Candida glabrata</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hawser and Douglas 1994</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Reynolds and Fink 2001</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hall-Stoodley <em>et al.</em> 2004</a>). Cells in biofilms are reported to have a higher degree of diversity, so they have more possible fates than cells in free-living planktonic form. One consequence of this diversity is the high number of antibiotic-persistent cells in <em>P. aeruginosa</em> biofilms, a result of the high frequency of slow-growing or dormant cells in mature biofilms (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nguyen <em>et al.</em> 2011</a>).</p><p id="p-7">Although the molecular basis for biofilm development and biofilm cell diversification has been studied extensively in bacteria, less is known about the genetic basis for biofilm formation and cell diversification in eukaryotic microbes such as yeasts. Laboratory <em>S</em>. <em>cerevisiae</em> strains have, in some cases, been selected to not form biofilms (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 1996</a>). The trait can reappear in suppressor mutants that derepress expression of the cell wall protein <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">Flo11p</a> or other members of the Flo protein family that induce cell–cell adhesion (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fichtner <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Torbensen <em>et al.</em> 2012</a>). The <em>S. cerevisiae</em> strain Σ1278b naturally forms biofilm in liquid medium on solid surfaces such as polystyrenes because it expresses <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">Flo11p</a> (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Reynolds and Fink 2001</a>). In addition to its importance for biofilm formation, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">Flo11p</a> is also essential for other morphotypes, including haploid-invasive growth on complex solid medium and diploid-pseudohyphal growth (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lo and Dranginis 1998</a>). A fourth Flo11p-dependent phenotype is a type of giant colony that develops on semisolid complex mediums at room temperature. The giant colonies have been denoted surface-spreading biofilm as well as mats in the literature (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Reynolds and Fink, 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> 2012</a>). Although biofilm in liquid medium, surface-spreading biofilm (mats), invasive, and pseudohyphal growth are dependent on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>, they cannot <em>a priory</em> be expected to be regulated in identical fashions because the growth conditions required for induction of these phenotypes are different.</p><p id="p-8">The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> gene is located in the middle of the right arm of chromosome IX (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lo and Dranginis 1996</a>) and has a 2.8-kb promoter (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rupp <em>et al.</em> 1999</a>). The relatively large promoter contains an extensive set of <em>cis</em>-acting elements that respond to multiple signaling pathways (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brückner and Mösch 2011</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> is regulated by a mitogen-activated protein kinase (MAPK) pathway via the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001126">Ste12p</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287">Tec1p</a> transcription factors (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Roberts and Fink 1994</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Köhler <em>et al.</em> 2002</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Rupp <em>et al.</em> 1999</a>). The pH-sensitive <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019">Rim101p</a> pathway regulates <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barrales <em>et al.</em> 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bayly <em>et al.</em> 2005</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lamb and Mitchell 2003</a>), and the response is believed to include components from the endosomal sorting complex required for transport (ESCRT), because ESCRT I, II, and III proteins are required for activation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019">Rim101p</a> and transcription of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sarode <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xu <em>et al.</em> 2004</a>). Nutrient levels regulate <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription through other pathways. Amino acid levels influence <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription via the general control nonderepressible (GCN) pathway (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Braus <em>et al.</em> 2003</a>), which induces transcription on amino acid starvation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lucchini <em>et al.</em> 1984</a>). The presence of amino acids induces the Ssy1p-Ptr3p-Ssy5p-sensor complex, which regulates <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription through amino acid permeases (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Torbensen <em>et al.</em> 2012</a>). Furthermore, glucose depletion induces <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> via the AMP kinase homolog <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002885">Snf1p</a> by inactivating the transcriptional repressors <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002450">Nrg1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000270">Nrg2p</a> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kuchin <em>et al.</em> 2002</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Van De Velde and Thevelein 2008</a>). Low glucose is also known to induces transcription of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> through G-protein-coupled glucose receptor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002193">Gpr1p</a>, cAMP (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Van De Velde and Thevelein 2008</a>), the protein kinase A (PKA) isoform <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a>, and the competing transcription regulators <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Rupp <em>et al.</em> 1999</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> is repressed when <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> is bound and a noncoding RNA is transcribed in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> promoter and in a transcriptionally permissive state when <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> is bound to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> promoter and the ncRNA gene is transcriptionally inactive (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bumgarner <em>et al.</em> 2009</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> activity is thought to facilitate the binding of other positive transcription factors such as <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287">Tec1p</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001126">Ste12p</a>, and Pol II that reinforce the active state of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> promoter (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bumgarner <em>et al.</em> 2012</a>). An interesting aspect of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulation is the toggle switch that results from competition between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> and leads to variegated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bumgarner <em>et al.</em> 2009</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2012</a>). Because of variable expression, only a subpopulation of cells expresses <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> and contributes to cell–cell adhesion. Variegated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression is seen in pseudohyphal and invasive growth and could play a role in development of biofilm in liquid medium.</p><p id="p-9">In contrast to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> induction through active <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a>, the PKA <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> is reported to repress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>. This has been shown with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutants that have three-fold higher levels of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA than wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">TPK3</a></em> cells and show more robust invasive and pseudohyphal growth (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>). Because <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression is repressed in a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">tpk2</a></em> double mutant, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> is thought to inhibit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> activity (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>).</p><p id="p-10"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulation has mainly been investigated under conditions in which cells grow invasively or form mats or pseudohyphae (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brückner and Mösch 2011</a>). Because the growth conditions essential for biofilm in liquid medium are very different from those favoring mats and pseudohyphal and invasive growth, it is unknown if the transcriptional program that regulates biofilm also regulates the other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent phenotypes. On abiotic surfaces, biofilms are formed in synthetic media with glucose as the carbon source (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Torbensen <em>et al.</em> 2012</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Reynolds and Fink 2001</a>), whereas surface-spreading biofilm formation and invasive growth both occur on complex solid medium, and pseudohyphal growth is formed by diploid cells on solid, synthetic, nitrogen-poor medium (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Reynolds and Fink 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gimeno <em>et al.</em> 1992</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Roberts and Fink 1994</a>). A recent screen for genes essential for mat formation and invasive and pseudohyphal growth found limited overlap between genes regulating the three phenotypes (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> 2012</a>), suggesting a dedicated transcriptional program for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent biofilm formation. In addition to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>, other <em>FLO</em> genes and conditions might influence biofilms, including genes regulating the extracellular matrix that strengthens the three-dimensional structure of biofilms (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kuthan <em>et al.</em> 2003</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vachova <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2000</a>), and quorum signaling that might coordinate the developmental program. Quorum signaling is reported for yeast, but its involvement in biofilm development is unknown (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen and Fink 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Smukalla <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Palkova <em>et al.</em> 1997</a>).</p><p id="p-11">In the current work, we investigated the molecular basis for Σ1278b biofilm development, including the extent to which biofilm development was dependent on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>. To do this, we screened a global collection of deletion mutants in the Σ1278b background for biofilm-forming ability on a solid abiotic surface. The extent to which the molecular program for biofilm formation overlapped with other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent phenotypes was investigated by testing biofilm-deficient mutants for ability to form mats and grow invasively. We found that a substantial fraction of Σ1278b cells were not part of the biofilm but existed as planktonic cells. Genes involved in the planktonic phenotype were identified by screening for Σ1278b mutants with more biofilm-phenotype cells in the total cell mass. Our study gives comprehensive insight into the molecular program controlling biofilm development in the genetic tractable yeast <em>S. cerevisiae</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-12"><em>S. cerevisiae</em> Σ1278b YS-11 (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>Δ</em>::<em>STE2p-spHIS5 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a>Δ</em>::<em>STE3p-LEU2 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>::<em>HisG <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em>) was used as a reference strain (Boone Lab, University of Toronto) (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> 2012</a>). The 4019 deletion mutants of Σ1278b YS-11 were from <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> (2012)</a> and have the same barcodes and deletions as the S288c collection (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giaever <em>et al.</em> 2002</a>).</p></div><div id="sec-3" class="subsection"><h3>Media</h3><p id="p-13">Synthetic complete (SC) media for biofilm formation on polystyrene were made as previously described (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guthrie and Fink 1991</a>), with the exception that amino acids and nucleotides were added in the following concentrations: adenine sulfate 20 mg/liter; uracil 38 mg/liter; <span class="sc">l</span>-histidine 38 mg/liter; <span class="sc">l</span>-arginine 38 mg/liter; <span class="sc">l</span>-tryptophan 38 mg/liter; <span class="sc">l</span>-methoinine 38 mg/liter; <span class="sc">l</span>-tyrosine 15 mg/liter; <span class="sc">l</span>-leucine 57 mg/liter; <span class="sc">l</span>-isoleucine 57 mg/liter; <span class="sc">l</span>-lysine 57 mg/liter; <span class="sc">l</span>-phenylalanine 48 mg/liter; <span class="sc">l</span>-valine 57 mg/liter; and <span class="sc">l</span>-threonine 57 mg/liter. Yeast extract peptone dextrose (YPD) complex medium was made as described (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Guthrie and Fink 1991</a>) using 20 g/liter agar for invasive growth and 3 g/liter for mats.</p></div><div id="sec-4" class="subsection"><h3>Assay for biofilm</h3><p id="p-14">Precultures were grown overnight at 30° in synthetic medium with 0.2% glucose and 100 mM NH<sub>4</sub><sup>+</sup>. Cells were subsequently inoculated into synthetic medium to OD<sub>600nm</sub> 0.1 for 2 hr before biofilm assays. Assays were in 200 µL in flat-well polystyrene microtiter plates (Frisenette). Cell density of the total population or of biofilm or planktonic subpopulations was recorded at OD<sub>450nm</sub> at indicated time points. Planktonic subpopulations were measured by removing nonadhering cells by pipetting and measuring cell density in new microtiter wells. Biofilm subpopulations were measured by addition of 200 µL fresh medium to adhering cells and measuring OD<sub>450nm</sub>. To visualize biofilms, crystal violet (HT901-8FOZ; Sigma-Aldrich) was added to wells for 24 hr at a final concentration of 0.05%. Planktonic cells and medium were removed and wells were washed four times with 200 µL H<sub>2</sub>O. Biofilms were dried and resuspended in 170 µL 96% ethanol for 1 hr. Biomass was determined at OD<sub>595nm</sub>.</p></div><div id="sec-5" class="subsection"><h3>Biofilm screens of the deletion strain collection</h3><p id="p-15">Deletion mutants were grown on solid YPD for 2 d. Cells were subsequently transferred to 96-well flat-bottom microtiter plates (Frisenette) containing 200 µL SC medium with 0.2% glucose. Cells were propagated for 46 hr or 96 hr at 30° and biomass was determined at OD<sub>600nm</sub> using a Synergy <sup>1</sup>H Hybrid Reader (Biotek). All experiments were performed in triplicate. Crystal violet staining was as mentioned except that washing and OD<sub>595nm</sub> measurements used a Biomek 2000 robot.</p></div><div id="sec-6" class="subsection"><h3>Data analysis</h3><p id="p-16">Crystal violet biofilm measurements for Σ1278b YS-11 mutants were normalized to the total biomass and log-transformed as ln(OD<sub>595nm</sub>/OD<sub>600nm</sub>). Normalized biofilm scores were used to determine the median biofilm score for each mutant for both 46 hr and 96 hr. Median values were used for all further analysis (Supporting Information, File S5 and File S6). Samples with a total biomass less than OD<sub>600nm</sub> &lt;0.01 were excluded from analysis. Replicate biofilm assays of the parental YS-11 strain (n = 288) showed normalized biofilm values that followed a Gaussian distribution at both 46 hr and 96 hr. The average normalized biofilm value ±2σ of the parental strain was used to determine mutants that had a significantly different biofilm score at 46 hr and 96 hr. Mutants with median-normalized biofilm scores less than 0.584 were considered to form significantly less biofilm than the parental strain and mutant with scores more than 1.972 were considered to form significantly more biofilm than the parental strain.</p></div><div id="sec-7" class="subsection"><h3>Synthetic genetic array</h3><p id="p-17">Selection of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a> geneX</em> double mutants in the parental (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>Δ</em>::<em>STE2p-spHIS5 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a>Δ</em>::<em>STE3p-LEU2 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>::<em>HisG <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em>) background was essentially conducted as described previously (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tong <em>et al.</em> 2001</a>).</p></div><div id="sec-8" class="subsection"><h3>Confocal laser scanning microscopy</h3><p id="p-18">Overnight cultures were diluted to OD<sub>600nm</sub> 0.1 in synthetic complete medium (0.2% glucose) and incubated on Rinzl plastic coverslips (Electron Mictoscopy Sciences) for 96 hr at 30°. Biofilms were washed twice in saline before 30 min of FUN-1 (Invitrogen) staining. Imaging used a plan-Neofluar 40×/1.3 oil microscopy objective with a Zeiss LSM510 microscope.</p></div><div id="sec-9" class="subsection"><h3>RNA FISH</h3><p id="p-19">RNA FISH was conducted essential as described previously (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McIsaac <em>et al.</em> 2013</a>) using Stellaris probes from Biosearch Technologies. Thirty-five 20mer Quasar 670 probes covering the stretch +9 to +708 of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> were used for detection of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA, whereas 35 20mer Quasar 570 probes covering the stretch +2 to +701 of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> were used as positive hybridization control. Precultures were grown overnight at 30° in synthetic medium. Cells were subsequently inoculated into synthetic medium to OD<sub>600nm</sub> 0.1 for 20 hr before fixation. Cells were fixated with 3% paraformaldehyde for 30 min at 30° followed by 4-hr incubation at 5°. Expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> was subsequently recorded with a Zeiss LSM780 microscope. Cells were counted as recordable if they were labeled with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> probe. All <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> positive cells were subsequently recorded for their expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA by counting cells with one or more red foci as positive for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA. A total of 1329 wild-type cells, 1167 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">sfl1</a></em> cells, and 542 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">flo11</a></em> cells were counted as blinded samples. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">flo11</a></em> cells served a negative control for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA expression. None of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">flo11</a></em> cells showed any signal for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA labeling.</p></div><div id="sec-10" class="subsection"><h3>Glucose concentration</h3><p id="p-20">Glucose concentration was determined enzymatically with a glucose assay kit (GAGO-20; Sigma-Aldrich) adjusted for use with microtiter plates.</p></div><div id="sec-11" class="subsection"><h3>Invasive growth</h3><p id="p-21">Invasive growth was assayed essentially as described previously (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Roberts and Fink 1994</a>) Cells were patched on solid 2.0% agar YPD using an inoculation loop and propagated for 3 d at 30°. Plates were covered with water for 30 min and shaken carefully, and nonadhering cells were removed. Complete colonies or colonies with portions remaining in or on the agar were categorized as invasive.</p></div><div id="sec-12" class="subsection"><h3>Mat formation</h3><p id="p-22">Cells were patched in the center of 25 ml YPD with 0.3% agar and incubated for 5 d at room temperature (22°–25°). Colonies with a structured hub and spokes were categorized as mat formers. Colonies that were completely smooth were categorized as having lost the ability to form mats.</p></div><div id="sec-13" class="subsection"><h3>Northern blot</h3><p id="p-23">RNA for northern dot blots was purified from mutants and the parental strain, YS-11, grown for 96 hr in square 120-mm × 120-mm Petri dishes (Frisenette). Total RNA was purified as previously described (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Torbensen <em>et al.</em> 2012</a>). All samples were treated with DNAse and tested for removal of DNA by PCR using <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> primers 5′-TGGATTCTGGTATGTTCTAGC-3′ and 5′-GAACGACGTGAGTAACACC-3′. Samples that still contained DNA were treated further with DNAse until no trace of DNA was detected by PCR. DNAse-treated RNA, 2 µg in 3 µl, was dropped onto Hydrobond-N<sup>+</sup> membranes (GE Healthcare) and dried. Membranes were wrapped in plastic wrap and RNA was cross-linked by UV light for 30 sec at 302 nm. One set of membranes was hybridized to a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> [<sup>32</sup>P]-labeled probe and another set was hybridized to a [<sup>32</sup>P]-labeled <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> probe. Hybridization and probes were as described (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Torbensen <em>et al.</em> 2012</a>). Hybridization to each dot was recorded with a Storm 840 phosphorimager (Bio-Rad), and Molecular Dynamics ImageQuant TL software was used for quantification. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcript levels were normalized to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> transcript levels and the average normalized <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcript level was calculated from three independent experiments for each mutant and parental strain.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3><em>S. cerevisiae</em> Σ1278b formation of mixed biofilm and planktonic populations is <em>FLO11</em>-dependent</h3><p id="p-24">In low glucose, haploid <em>S. cerevisiae</em> Σ1278b forms biofilm on polystyrene (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Reynolds and Fink 2001</a>). We tested this phenotype in liquid synthetic medium with different carbon sources and found that haploid Σ1278b formed a biofilm when cultured in 0.2% glucose, 1% maltose, 1% glycerol, or 1% ethanol, whereas biofilm formation was repressed in 2% glucose (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Further examination of biofilms formed in 0.2% glucose revealed that only 25% of cells were part of the biofilm (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Confocal laser scanning microscopy (CLSM) revealed that the proportion of biofilm-forming cells decreased substantially in a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">flo11</a></em> mutant, confirming that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> was responsible for biofilm formation (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, C and D</a>). Fluorescent <em>in situ</em> hybridization (FISH) revealed that only a minor fraction of reference cells expressed <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA (21%, n = 1329) in the synthetic 0.2% glucose medium (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1F</a>) corresponding to the fraction of cells participating in biofilm (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). The proportion of cells expressing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA increased to 73% (n = 1167) in a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">sfl1</a></em> mutant (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1H</a>), suggesting that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> took part in participation of cells in a planktonic and a biofilm-forming subpopulation (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>). This hypothesis was supported by CLSM of biofilm formed by <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">sfl1</a></em> and the reference strain (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, C and E</a>). Both strains formed microcolonies, and microcolonies formed by the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">sfl1</a></em> mutant appeared to be larger than those of the reference strain (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1, C and E</a> and Figure S1). Based on these initial experiments, we expected that other molecular factors involved in biofilm development and repression of biofilm could be found.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="S. cerevisiae &#x3A3;1278b forms mixed populations of biofilm and planktonic cells. (A) Wild-type cells grown at 30&#xB0; in polystyrene wells in synthetic complete (SC) media in indicated carbon and nitrogen sources (Gln = glutamine 100 mM; Pro = proline 100 mM; NH4+ = ammonium 100 mM) for 96 hr and stained with crystal violet. Dark staining indicates biofilm formation. (B) Biofilm formation on polystyrene by a wild-type (wt) population in SC 0.2% glucose and NH4+. Blue triangles = nonadhering planktonic cells; green circles = biofilm-forming cells. Planktonic cells were separated from biofilms by pipetting. Cell density was measured at OD450nm. Black circles = measured glucose concentration left in growth medium over time. (C, D, E) Biofilm was recorded with confocal laser scanning microscopy (CLSM) after 96 hr of growth in SC 0.2% glucose and NH4+. Nonadhering cells were removed by a single gentle pipetting and adhering cells were dyed with FUN-1. White bar = 50 &#xB5;m. (F, G, H) Single cell fluorescence in situ hybridization (FISH) of representative samples of cells grown in SC 0.2% glucose and NH4+ for 20 hr. Bars = 5 &#xB5;m. (C, F) Wild-type. (D, G) flo11::kanMX in the wild-type background. (E, H) sfl1::kanMX in the wild-type background; MATa can1&#x394;::STE2p-spHIS5 lyp1&#x394;::STE3p-LEU2 his3::HisG leu2&#x394; ura3&#x394;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-640566828" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;S. cerevisiae&lt;/em&gt; &#x3A3;1278b forms mixed populations of biofilm and planktonic cells. (A) Wild-type cells grown at 30&#xB0; in polystyrene wells in synthetic complete (SC) media in indicated carbon and nitrogen sources (Gln = glutamine 100 mM; Pro = proline 100 mM; NH&lt;sub&gt;4&lt;/sub&gt;&lt;sup&gt;+&lt;/sup&gt; = ammonium 100 mM) for 96 hr and stained with crystal violet. Dark staining indicates biofilm formation. (B) Biofilm formation on polystyrene by a wild-type (wt) population in SC 0.2% glucose and NH&lt;sub&gt;4&lt;/sub&gt;&lt;sup&gt;+&lt;/sup&gt;. Blue triangles = nonadhering planktonic cells; green circles = biofilm-forming cells. Planktonic cells were separated from biofilms by pipetting. Cell density was measured at OD&lt;sub&gt;450nm&lt;/sub&gt;. Black circles = measured glucose concentration left in growth medium over time. (C, D, E) Biofilm was recorded with confocal laser scanning microscopy (CLSM) after 96 hr of growth in SC 0.2% glucose and NH&lt;sub&gt;4&lt;/sub&gt;&lt;sup&gt;+&lt;/sup&gt;. Nonadhering cells were removed by a single gentle pipetting and adhering cells were dyed with FUN-1. White bar = 50 &#xB5;m. (F, G, H) Single cell fluorescence &lt;em&gt;in situ&lt;/em&gt; hybridization (FISH) of representative samples of cells grown in SC 0.2% glucose and NH&lt;sub&gt;4&lt;/sub&gt;&lt;sup&gt;+&lt;/sup&gt; for 20 hr. Bars = 5 &#xB5;m. (C, F) Wild-type. (D, G) &lt;em&gt;flo11&lt;/em&gt;::&lt;em&gt;kanMX&lt;/em&gt; in the wild-type background. (E, H) &lt;em&gt;sfl1&lt;/em&gt;::&lt;em&gt;kanMX&lt;/em&gt; in the wild-type background; &lt;em&gt;MAT&lt;/em&gt;&lt;strong&gt;a&lt;/strong&gt; &lt;em&gt;can1&#x394;&lt;/em&gt;::&lt;em&gt;STE2p-spHIS5 lyp1&#x394;&lt;/em&gt;::&lt;em&gt;STE3p-LEU2 his3&lt;/em&gt;::&lt;em&gt;HisG leu2&#x394; ura3&#x394;&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1671/F1.medium.gif" width="251" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1671/F1.medium.gif" width="251" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464960" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child"><em>S. cerevisiae</em> Σ1278b forms mixed populations of biofilm and planktonic cells. (A) Wild-type cells grown at 30° in polystyrene wells in synthetic complete (SC) media in indicated carbon and nitrogen sources (Gln = glutamine 100 mM; Pro = proline 100 mM; NH<sub>4</sub><sup>+</sup> = ammonium 100 mM) for 96 hr and stained with crystal violet. Dark staining indicates biofilm formation. (B) Biofilm formation on polystyrene by a wild-type (wt) population in SC 0.2% glucose and NH<sub>4</sub><sup>+</sup>. Blue triangles = nonadhering planktonic cells; green circles = biofilm-forming cells. Planktonic cells were separated from biofilms by pipetting. Cell density was measured at OD<sub>450nm</sub>. Black circles = measured glucose concentration left in growth medium over time. (C, D, E) Biofilm was recorded with confocal laser scanning microscopy (CLSM) after 96 hr of growth in SC 0.2% glucose and NH<sub>4</sub><sup>+</sup>. Nonadhering cells were removed by a single gentle pipetting and adhering cells were dyed with FUN-1. White bar = 50 µm. (F, G, H) Single cell fluorescence <em>in situ</em> hybridization (FISH) of representative samples of cells grown in SC 0.2% glucose and NH<sub>4</sub><sup>+</sup> for 20 hr. Bars = 5 µm. (C, F) Wild-type. (D, G) <em>flo11</em>::<em>kanMX</em> in the wild-type background. (E, H) <em>sfl1</em>::<em>kanMX</em> in the wild-type background; <em>MAT</em><strong>a</strong> <em>can1Δ</em>::<em>STE2p-spHIS5 lyp1Δ</em>::<em>STE3p-LEU2 his3</em>::<em>HisG leu2Δ ura3Δ</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Screening of a Σ1278b deletion collection for genes essential for biofilm development</h3><p id="p-26">To identify genes essential for biofilm in liquid medium in Σ1278b, as well as genes that regulate the proportion of biofilm-forming cells in a population, we screened a complete library of Σ1278b haploid mutants deleted in 4019 nonessential genes for biofilm-forming ability. The mutants were constructed by substituting each open reading frame (ORF) in Σ1278b (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>Δ</em>::<em>STE2p-spHIS5 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a>Δ</em>::<em>STE3p-LEU2 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>::<em>HisG <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em>) with a <em>kanMX</em> cassette derived from the S288c deletion collection (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> 2012</a>). Thus, mutant alleles in the Σ1278b mutant collection were identical to the mutant alleles in the S288c collection (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Giaever <em>et al.</em> 2002</a>).</p><p id="p-27">The biofilm screen was conducted by growing each of the 4019 mutants in liquid synthetic complete medium with 0.2% glucose and ammonium. Biofilm development was tested in triplicate after 46 hr and 96 hr by staining with crystal violet and removing planktonic cells. Mutants varied greatly in the amount of biomass formed (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). To compensate for differences in biomass, values from crystal violet staining were normalized to the total cell mass of planktonic and biofilm-forming cells (OD<sub>600nm</sub>) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Median normalized biofilm values ln(OD<sub>595nm</sub>/OD<sub>600nm</sub>) were calculated for each mutant at 46 hr and 96 hr (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C and D</a> and File S1, File S2). Biofilm formation by the parental strain (n = 288) followed a Gaussian distribution. Thus, the average normalized biofilm value ±2σ for the parental strain was used to identify mutants that formed significantly more or less biofilm than the parental strain (<em>P</em> &lt; 0.05). We found that 137 mutants formed significantly less biofilm after 46 hr and, of these, 71 mutants still formed significantly less biofilm after 96 hr (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, C and D</a>, blue bars, and File S3). A larger set of 427 mutants formed significantly more biofilm after 46 hr (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>, yellow bars). After 96 hr, 371 mutants formed significantly more biofilm than the parental strain (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>, yellow bars), with 100 mutants forming more biofilm at both time points (File S3). To obtain a conservative estimate of the genes involved in biofilm development and to avoid growth rate effects, we considered only mutants with significantly altered biofilm effects after both 46 hr and 96 hr. This resulted in 71 candidate genes essential for biofilm development and 100 genes that repressed biofilm development by maintaining a high proportion of planktonic cells (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, C and D</a>, blue and yellow bars, and File S3).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screening of a &#x3A3;1278b deletion mutant collection for genes essential in biofilm development. Mutants deleted for one of 4019 nonessential genes (indicated as geneX in MATa can1&#x394;::STE2p-spHIS5 lyp1&#x394;::STE3p-LEU2 his3::HisG leu2&#x394; ura3&#x394; geneX::kanMX) were tested for biofilm formation on polystyrene at 30&#xB0; in liquid SC with 0.2% glucose and NH4+ medium. (A) Total biomass (OD600nm) vs. biofilm formation measured at OD595nm after crystal violet staining. Black circles = all data points for each mutant; red line = correlation between cell density and crystal violet staining; dotted red line = &#xB1;2&#x3C3;. (B) Distribution of normalized biofilm from individual mutants as ln(crystal violet dyed biofilm/total biomass) compared with cell mass. Black circles = all data points for each mutant; red line = correlation between biomass and normalized biofilm; dotted red line = &#xB1;2&#x3C3;. (A and B) Values obtained after 96 hr of biofilm development. (C) Histograms representing median normalized biofilm values of the 4019 deletion mutants assayed in triplicate. Black line = normalized biofilm values for the parental wild-type &#x3A3;1278b strain (n = 288) showing Gaussian distribution after 48 hr of growth. Biofilm distribution of the parental strain after 48 hr was used to set cut-off to average &#xB1;2&#x3C3;. Blue bars = biofilms of mutants that formed significantly less biofilm than the parental strain; yellow bars = biofilm values of mutants that formed significantly more biofilm than the wild-type strain. (D) As in (C) after 96 hr of growth. Biofilm distribution of the parental strain after 96 hr of growth was used to set cut-offs to average &#xB1;2&#x3C3;. A complete list of mutants and median normalized biofilm values are listed File S1 and File S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-640566828" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Screening of a &#x3A3;1278b deletion mutant collection for genes essential in biofilm development. Mutants deleted for one of 4019 nonessential genes (indicated as &lt;em&gt;geneX&lt;/em&gt; in &lt;em&gt;MAT&lt;/em&gt;&lt;strong&gt;a&lt;/strong&gt; &lt;em&gt;can1&#x394;&lt;/em&gt;::&lt;em&gt;STE2p-spHIS5 lyp1&#x394;&lt;/em&gt;::&lt;em&gt;STE3p-LEU2 his3&lt;/em&gt;::&lt;em&gt;HisG leu2&#x394; ura3&#x394; geneX&lt;/em&gt;::&lt;em&gt;kanMX&lt;/em&gt;) were tested for biofilm formation on polystyrene at 30&#xB0; in liquid SC with 0.2% glucose and NH&lt;sub&gt;4&lt;/sub&gt;&lt;sup&gt;+&lt;/sup&gt; medium. (A) Total biomass (OD&lt;sub&gt;600nm&lt;/sub&gt;) &lt;em&gt;vs.&lt;/em&gt; biofilm formation measured at OD&lt;sub&gt;595nm&lt;/sub&gt; after crystal violet staining. Black circles = all data points for each mutant; red line = correlation between cell density and crystal violet staining; dotted red line = &#xB1;2&#x3C3;. (B) Distribution of normalized biofilm from individual mutants as ln(crystal violet dyed biofilm/total biomass) compared with cell mass. Black circles = all data points for each mutant; red line = correlation between biomass and normalized biofilm; dotted red line = &#xB1;2&#x3C3;. (A and B) Values obtained after 96 hr of biofilm development. (C) Histograms representing median normalized biofilm values of the 4019 deletion mutants assayed in triplicate. Black line = normalized biofilm values for the parental wild-type &#x3A3;1278b strain (n = 288) showing Gaussian distribution after 48 hr of growth. Biofilm distribution of the parental strain after 48 hr was used to set cut-off to average &#xB1;2&#x3C3;. Blue bars = biofilms of mutants that formed significantly less biofilm than the parental strain; yellow bars = biofilm values of mutants that formed significantly more biofilm than the wild-type strain. (D) As in (C) after 96 hr of growth. Biofilm distribution of the parental strain after 96 hr of growth was used to set cut-offs to average &#xB1;2&#x3C3;. A complete list of mutants and median normalized biofilm values are listed File S1 and File S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1671/F2.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/9/1671/F2.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464870" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Screening of a Σ1278b deletion mutant collection for genes essential in biofilm development. Mutants deleted for one of 4019 nonessential genes (indicated as <em>geneX</em> in <em>MAT</em><strong>a</strong> <em>can1Δ</em>::<em>STE2p-spHIS5 lyp1Δ</em>::<em>STE3p-LEU2 his3</em>::<em>HisG leu2Δ ura3Δ geneX</em>::<em>kanMX</em>) were tested for biofilm formation on polystyrene at 30° in liquid SC with 0.2% glucose and NH<sub>4</sub><sup>+</sup> medium. (A) Total biomass (OD<sub>600nm</sub>) <em>vs.</em> biofilm formation measured at OD<sub>595nm</sub> after crystal violet staining. Black circles = all data points for each mutant; red line = correlation between cell density and crystal violet staining; dotted red line = ±2σ. (B) Distribution of normalized biofilm from individual mutants as ln(crystal violet dyed biofilm/total biomass) compared with cell mass. Black circles = all data points for each mutant; red line = correlation between biomass and normalized biofilm; dotted red line = ±2σ. (A and B) Values obtained after 96 hr of biofilm development. (C) Histograms representing median normalized biofilm values of the 4019 deletion mutants assayed in triplicate. Black line = normalized biofilm values for the parental wild-type Σ1278b strain (n = 288) showing Gaussian distribution after 48 hr of growth. Biofilm distribution of the parental strain after 48 hr was used to set cut-off to average ±2σ. Blue bars = biofilms of mutants that formed significantly less biofilm than the parental strain; yellow bars = biofilm values of mutants that formed significantly more biofilm than the wild-type strain. (D) As in (C) after 96 hr of growth. Biofilm distribution of the parental strain after 96 hr of growth was used to set cut-offs to average ±2σ. A complete list of mutants and median normalized biofilm values are listed File S1 and File S2.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Biofilm genes form a large complex regulatory network</h3><p id="p-29">To obtain insight into the mechanisms that regulate biofilm development, all 171 biofilm-related genes were grouped into functional categories according to their gene onthology (GO) process (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Among the genes with a positive effect on biofilm when deleted were several functional groups of genes encoding proteins with mitochondrial function (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and File S4), and genes encoding respiratory chain components (<em>P</em> = 4.56e<sup>−3</sup>) and mitochondrial ribosomal proteins (<em>P</em> = 2.19e<sup>−16</sup>) were especially overrepresented. To analyze if the increased proportion of biofilm-forming cells in these mutants was dependent on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>, we made <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> northern blots of representative mutants with reduced mitochondrial function (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and File S5). No change in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression was observed in seven of the tested mutants representing reduced mitochondrial function, suggesting that posttranscriptional mechanisms acted on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> gene product, other <em>FLO</em> genes were responsible for the increase in biofilm, or mutants with impaired mitochondrial function affected a non-Flo-dependent biofilm mechanism.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464968/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464968/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Biofilm gene GO processes for 71 genes that were essential for biofilm development and 100 genes that induced biofilm when deleted (all gene names are given in Table S3)</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of FLO11 mRNA levels in mutants with reduced biofilm. FLO11 mRNA levels used for the heat map were measured by northern dot blots of mutants grown for 96 hr in 0.2% glucose synthetic medium. FLO11 mRNA levels were normalized to ACT1 mRNA levels and the wild-type ACT1:FLO11 mRNA ratio was set as 0. Mutants with significantly altered FLO11 mRNA levels compared with the parental strain are indicated with asterisks (***P &#x2264; 0.01, **0.01 &lt; P &lt; 0.05, *0.05 &lt; P &lt; 0.1; n = 3). (A) Transcript levels of FLO11 in deletion mutants with increased biofilm formation compared with the parental strain. (B) Transcript levels of FLO11 in deletion mutants that had lost the ability to form biofilm. Numerical values of FLO11 mRNA levels are listed in File S5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-640566828" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heat map of &lt;em&gt;FLO11&lt;/em&gt; mRNA levels in mutants with reduced biofilm. &lt;em&gt;FLO11&lt;/em&gt; mRNA levels used for the heat map were measured by northern dot blots of mutants grown for 96 hr in 0.2% glucose synthetic medium. &lt;em&gt;FLO11&lt;/em&gt; mRNA levels were normalized to &lt;em&gt;ACT1&lt;/em&gt; mRNA levels and the wild-type &lt;em&gt;ACT1&lt;/em&gt;:&lt;em&gt;FLO11&lt;/em&gt; mRNA ratio was set as 0. Mutants with significantly altered &lt;em&gt;FLO11&lt;/em&gt; mRNA levels compared with the parental strain are indicated with asterisks (***&lt;em&gt;P&lt;/em&gt; &#x2264; 0.01, **0.01 &lt; &lt;em&gt;P&lt;/em&gt; &lt; 0.05, *0.05 &lt; &lt;em&gt;P&lt;/em&gt; &lt; 0.1; n = 3). (A) Transcript levels of &lt;em&gt;FLO11&lt;/em&gt; in deletion mutants with increased biofilm formation compared with the parental strain. (B) Transcript levels of &lt;em&gt;FLO11&lt;/em&gt; in deletion mutants that had lost the ability to form biofilm. Numerical values of &lt;em&gt;FLO11&lt;/em&gt; mRNA levels are listed in File S5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1671/F3.medium.gif" width="256" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/9/1671/F3.medium.gif" width="256" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Heat map of <em>FLO11</em> mRNA levels in mutants with reduced biofilm. <em>FLO11</em> mRNA levels used for the heat map were measured by northern dot blots of mutants grown for 96 hr in 0.2% glucose synthetic medium. <em>FLO11</em> mRNA levels were normalized to <em>ACT1</em> mRNA levels and the wild-type <em>ACT1</em>:<em>FLO11</em> mRNA ratio was set as 0. Mutants with significantly altered <em>FLO11</em> mRNA levels compared with the parental strain are indicated with asterisks (***<em>P</em> ≤ 0.01, **0.01 &lt; <em>P</em> &lt; 0.05, *0.05 &lt; <em>P</em> &lt; 0.1; n = 3). (A) Transcript levels of <em>FLO11</em> in deletion mutants with increased biofilm formation compared with the parental strain. (B) Transcript levels of <em>FLO11</em> in deletion mutants that had lost the ability to form biofilm. Numerical values of <em>FLO11</em> mRNA levels are listed in File S5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Among the 71 genes that were essential for biofilm development were several that were essential for invasive and pseudohyphal growth (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). These included components of the PKA pathway (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005042">RAS2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">TPK3</a></em>), a MAP kinase pathway (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287">TEC1</a></em>), the Rim pathway (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019">RIM101</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003013">RIM8</a></em>), and the GCN pathway (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000735">GCN4</a></em>), as well as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription factors that have not been assigned to specific signaling pathways (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000733">MIT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002392">MFG1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004774">MSS11</a></em>) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gagiano <em>et al.</em> 2003</a>; <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cain <em>et al.</em> 2012</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Braus <em>et al.</em> 2003</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barrales <em>et al.</em> 2008</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Roberts and Fink 1994</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Rupp <em>et al.</em> 1999</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>). We also found genes encoding components of the ESCRT complexes that are essential for regulation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005923">SNF8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000514">STP22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004409">VPS36</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004682">VPS20</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003863">VPS25</a></em>) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sarode <em>et al.</em> 2011</a>). Additionally, the GO revealed a number of genes not previously associated with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression such as <em>AGT9</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005940">VTC3</a></em>, which are involved in membrane invagination, the vacuolar amino acid transporter <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003761">AVT1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">NPR1</a></em>, a regulator of amino acid transporter endocytosis. Identification of these genes suggested that vacuolar function or transport of one or more proteins to the vacuole was essential for biofilm development. In addition, there is also a group of genes involved in NADH repair such as NADHX epimerase (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005144">YNL200C</a>) and NADHX dehydratase (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001634">YKL151C</a>). In all, we identified 58 genes not previously associated with biofilm formation or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression (File S3).</p><p id="p-33">Because formation of biofilm in the parental strain was dependent on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1D</a>), we tested the level of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA in the mutants that did not form biofilm. Quantitative northern blots for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA revealed that 38 mutants had significantly less <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA than the parental strain (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and File S5). As expected, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">flo11</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">flo8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000733">mit1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004774">mss11</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003013">rim8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001019">rim101</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005923">snf8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000287">tec1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004682">vps20</a></em> had lower levels of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA. A large number of genes not previously implicated in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulation were also found to reduce <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription or mRNA stability when deleted (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000044">AIM1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001392">ASG1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003761">AVT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003325">DRN1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006022">ELP4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000984">FMP10</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000238">HMT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004669">KAR5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000596">MRPL32</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001084">NCP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">NPR1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004844">PEP5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006033">PEX25</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001521">RGT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004519">SPC2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001555">STB6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000514">STP22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000309">VID24</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005940">VTC3</a></em>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000125">YBL029W</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007591">YBL029C-A</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001840">YFL054C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029726">YGR161W-C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003536">YIL014C-A</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001463">YIR024C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001634">YKL151C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005144">YNL200C</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028856">YOR034C-A</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005749">YOR223W</a>).</p><p id="p-34">Several factors known to regulate <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression did not appear as regulators in the biofilm screen. Most notably, deletion of PKA isoform 2 did not significantly affect biofilm development (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and File S1, File S2). The involvement of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription and adhesive invasive phenotypes was previously determined by testing on rich complex medium (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>). We found that on liquid synthetic medium, a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">tpk2</a></em> mutant formed 33%–50% of the amount of biofilm of the wild-type strain (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> homolog <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a>, however, was essential for biofilm development (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). The positive effect of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> on biofilm formation was not at the transcriptional level, because the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutant had a 2.7-fold increase in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcript compared with wild-type (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a> and File S5). Hence, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> appeared to have two roles in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulation: partial repression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription but induction of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">Flo11p</a> expression at a posttranscriptional level. To investigate if <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">Flo11p</a> was expressed in the cell wall of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutant, we tested the mutant for cell–cell adhesion. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> cells grown in liquid synthetic biofilm medium did not adhere to each other as seen for the wild-type strain (data not shown). Hence, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutant did not appear to express functional <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">Flo11p</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="TPK3 is essential for biofilm development. (A) The percentage of cells forming a biofilm on polystyrene compared with planktonic cells was determined as described in the legend for Figure 1B using OD450nm to determine the biomass of the total population and the biofilm subpopulation. Wild-type = green circle; flo11 mutant = black triangles; tpk1 mutant = purple diamond; tpk2 mutant = blue cross; tpk3 mutant = turquoise square. (B) FLO11 mRNA levels in the wild-type and the tpk3 mutant. Relative FLO11 mRNA levels were found by normalizing to ACT1 mRNA levels and the average wild-type ACT1:FLO11 mRNA ratio was set as 1. Both experiments were performed in triplicate (File S5). (C) Model of Tpk3p transcriptional and posttranscriptional regulation of FLO11. Green arrows, positive regulation; red bars, negative regulation. Experimental evidence for the interaction between Tpk2p and Tpk3p is also available (Robertson and Fink 1998)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-640566828" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;TPK3&lt;/em&gt; is essential for biofilm development. (A) The percentage of cells forming a biofilm on polystyrene compared with planktonic cells was determined as described in the legend for Figure 1B using OD&lt;sub&gt;450nm&lt;/sub&gt; to determine the biomass of the total population and the biofilm subpopulation. Wild-type = green circle; &lt;em&gt;flo11&lt;/em&gt; mutant = black triangles; &lt;em&gt;tpk1&lt;/em&gt; mutant = purple diamond; &lt;em&gt;tpk2&lt;/em&gt; mutant = blue cross; &lt;em&gt;tpk3&lt;/em&gt; mutant = turquoise square. (B) &lt;em&gt;FLO11&lt;/em&gt; mRNA levels in the wild-type and the &lt;em&gt;tpk3&lt;/em&gt; mutant. Relative &lt;em&gt;FLO11&lt;/em&gt; mRNA levels were found by normalizing to &lt;em&gt;ACT1&lt;/em&gt; mRNA levels and the average wild-type &lt;em&gt;ACT1&lt;/em&gt;:&lt;em&gt;FLO11&lt;/em&gt; mRNA ratio was set as 1. Both experiments were performed in triplicate (File S5). (C) Model of Tpk3p transcriptional and posttranscriptional regulation of &lt;em&gt;FLO11&lt;/em&gt;. Green arrows, positive regulation; red bars, negative regulation. Experimental evidence for the interaction between Tpk2p and Tpk3p is also available (Robertson and Fink 1998).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1671/F4.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/9/1671/F4.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464962" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child"><em>TPK3</em> is essential for biofilm development. (A) The percentage of cells forming a biofilm on polystyrene compared with planktonic cells was determined as described in the legend for <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1B</a> using OD<sub>450nm</sub> to determine the biomass of the total population and the biofilm subpopulation. Wild-type = green circle; <em>flo11</em> mutant = black triangles; <em>tpk1</em> mutant = purple diamond; <em>tpk2</em> mutant = blue cross; <em>tpk3</em> mutant = turquoise square. (B) <em>FLO11</em> mRNA levels in the wild-type and the <em>tpk3</em> mutant. Relative <em>FLO11</em> mRNA levels were found by normalizing to <em>ACT1</em> mRNA levels and the average wild-type <em>ACT1</em>:<em>FLO11</em> mRNA ratio was set as 1. Both experiments were performed in triplicate (File S5). (C) Model of Tpk3p transcriptional and posttranscriptional regulation of <em>FLO11</em>. Green arrows, positive regulation; red bars, negative regulation. Experimental evidence for the interaction between Tpk2p and Tpk3p is also available (<a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">To identify genes involved in regulation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> at the posttranscriptional level, we screened a library of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a> geneX</em> double mutants for ability to suppress the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> phenotype and fully restore biofilm formation. Double mutants were made by crossing a biofilm-deficient <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em>::<em>natMX</em> mutant with the mutant collection using a synthetic genetic array (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Tong <em>et al.</em> 2001</a>). Diploids were sporulated and haploid double mutants were tested in three independent experiments for ability to form biofilm (Figure S2). The screen identified 35 mutant alleles that suppressed the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> biofilm phenotype (File S6). However, none of these genes encoded known components of the PKA pathway or components of translation or RNA processing that suggested a pathway or complex through which <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> acted on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>.</p></div><div id="sec-18" class="subsection"><h3>Genes essential for biofilm formation are also essential for mat formation on semisolid medium</h3><p id="p-37">Next, we tested biofilm-deficient mutants for other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent phenotypes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A–C</a>). Mat formation was tested on complex medium with 0.3% agar. Colonies that formed a structured hub and spokes were considered mats, whereas unstructured, smooth colonies were deemed to have lost the ability to form mats (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Of the 71 mutants that lost the ability to form a biofilm in liquid medium on polystyrene, 69 were also unable to form mats, revealing a nearly complete overlap between genes involved in biofilm in synthetic medium and mats formation on semisolid complex medium (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, D and E</a> and Figure S3). Next, invasive growth was tested on complex medium with 2% agar by growing cells in patches for 2 d and washing the resulting colonies with water (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). Colonies that left no macroscopic traces of cells after washing were considered noninvasive, whereas colonies that remained on plates after washing were considered invasive (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a> and Figure S4). Of the 69 mutants that did not form mats or biofilm in synthetic medium, 49 also did not show invasive growth (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5, D and E</a>), revealing a core group of 49 genes essential for the haploid phenotypes of mat and biofilm formation in synthetic medium and invasive growth.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Most genes essential for biofilm are also essential for invasive growth and surface-spreading biofilm formation. (A) Biofilms in microtiter plates stained with crystal violet as described in Figure 2A legend. Color indicates biofilm. Wt, wild-type. &#x3A3;1278b parental strain (MATa can1&#x394;::STE2p-spHIS5 lyp1&#x394;::STE3p-LEU2 his3::HisG leu2&#x394; ura3&#x394;); flo11&#x394;, flo11::kanMX in the wild-type background. (B) Mat formation of wild-type and lack of mat by flo11 on semisolid YPD (0.3% agar) after 5 d at room temperature (22&#xB0;&#x2013;25&#xB0;). (C) Invasive growth of wild-type and flo11 mutant. Invasive growth was tested after 3 d on solid YPD (2% agar) at 30&#xB0; by washing colonies gently with water to remove nonadhering cells. (D) Diagram of deletion mutants that lost the ability to form biofilm and mats and grow invasively. (E) List of phenotypes of mutants with gene deletion that eliminated biofilm formation. Images of mat formation and invasive growth of all mutants are in Figure S3 and Figure S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-640566828" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Most genes essential for biofilm are also essential for invasive growth and surface-spreading biofilm formation. (A) Biofilms in microtiter plates stained with crystal violet as described in Figure 2A legend. Color indicates biofilm. Wt, wild-type. &#x3A3;1278b parental strain (&lt;em&gt;MAT&lt;/em&gt;&lt;strong&gt;a&lt;/strong&gt; &lt;em&gt;can1&#x394;&lt;/em&gt;::&lt;em&gt;STE2p-spHIS5 lyp1&#x394;&lt;/em&gt;::&lt;em&gt;STE3p-LEU2 his3&lt;/em&gt;::&lt;em&gt;HisG leu2&#x394; ura3&#x394;&lt;/em&gt;); &lt;em&gt;flo11&#x394;&lt;/em&gt;, &lt;em&gt;flo11&lt;/em&gt;::&lt;em&gt;kanMX&lt;/em&gt; in the wild-type background. (B) Mat formation of wild-type and lack of mat by &lt;em&gt;flo11&lt;/em&gt; on semisolid YPD (0.3% agar) after 5 d at room temperature (22&#xB0;&#x2013;25&#xB0;). (C) Invasive growth of wild-type and &lt;em&gt;flo11&lt;/em&gt; mutant. Invasive growth was tested after 3 d on solid YPD (2% agar) at 30&#xB0; by washing colonies gently with water to remove nonadhering cells. (D) Diagram of deletion mutants that lost the ability to form biofilm and mats and grow invasively. (E) List of phenotypes of mutants with gene deletion that eliminated biofilm formation. Images of mat formation and invasive growth of all mutants are in Figure S3 and Figure S4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1671/F5.medium.gif" width="374" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/9/1671/F5.medium.gif" width="374" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1671/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464872" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Most genes essential for biofilm are also essential for invasive growth and surface-spreading biofilm formation. (A) Biofilms in microtiter plates stained with crystal violet as described in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a> legend. Color indicates biofilm. Wt, wild-type. Σ1278b parental strain (<em>MAT</em><strong>a</strong> <em>can1Δ</em>::<em>STE2p-spHIS5 lyp1Δ</em>::<em>STE3p-LEU2 his3</em>::<em>HisG leu2Δ ura3Δ</em>); <em>flo11Δ</em>, <em>flo11</em>::<em>kanMX</em> in the wild-type background. (B) Mat formation of wild-type and lack of mat by <em>flo11</em> on semisolid YPD (0.3% agar) after 5 d at room temperature (22°–25°). (C) Invasive growth of wild-type and <em>flo11</em> mutant. Invasive growth was tested after 3 d on solid YPD (2% agar) at 30° by washing colonies gently with water to remove nonadhering cells. (D) Diagram of deletion mutants that lost the ability to form biofilm and mats and grow invasively. (E) List of phenotypes of mutants with gene deletion that eliminated biofilm formation. Images of mat formation and invasive growth of all mutants are in Figure S3 and Figure S4.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-39">In this work, we found that the <em>S. cerevisiae</em> strain Σ1278b showed a stable, dimorphic growth pattern on polystyrene in liquid synthetic complete medium, with both planktonic and biofilm-forming cells. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> was essential for formation of the biofilm subpopulation. A possible explanation for the dimorphism is that biofilm subpopulations express <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> under control of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> while planktonic populations repress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> via <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a>. It was previously shown that cells in the Σ1278b background show variegated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression that is dependent on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Halme <em>et al.</em> 2004</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">Flo8p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> compete for regulation of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> promoter and concomitant transcription of either of two ncRNAs in the promoter determines whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> promoter is in a repressed state or a permissive form that allows other transcription factors to induce transcription of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bumgarner <em>et al.</em> 2009</a>). The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">Sfl1p</a> switch also appeared to be essential for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent dimorphism in biofilms, because only a small subpopulation of cells expressed <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA in the parental strain, whereas this subpopulation increased to 73% in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005666">sfl1</a></em> mutant (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1, F and H</a>).</p><p id="p-40">To investigate the molecular basis for <em>S. cerevisiae</em> biofilm formation, we screened a deletion mutant collection in the Σ1278b strain background for loss of biofilm-forming ability. We identified 71 genes with significantly lower biofilm formation than the reference strain (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2, C and D</a> and File S3). Of these, 38 genes resulted in a significant reduction in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA levels when deleted, suggesting that biofilm was primarily regulated through transcriptional induction of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). A majority of genes have not previously been associated with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulation and the regulatory network underlying <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> thus appears to be much more diverse than previously suggested.</p><p id="p-41">Some of the novel regulators are probably directly involved in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulation while others might have an indirect effect on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>. For example, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">Npr1p</a> might be an indirect factor. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">Npr1p</a> is essential for correct targeting of plasma membrane proteins such as the ammonium permease <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005086">Mep2p</a> and the general amino acid permease <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1p</a> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">De Craene <em>et al.</em> 2001</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Vandenbol <em>et al.</em> 1990</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lorenz and Heitman 1998</a>). Both <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005086">Mep2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1p</a> are involved in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression in some strain backgrounds (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lorenz and Heitman 1998</a>; <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Torbensen <em>et al.</em> 2012</a>). Therefore, the reduction of biofilm formation and expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> in an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">npr1</a></em> mutant is likely to be a consequence of intracellular retention of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005086">Mep2p</a> and/or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1p</a>. Although the biofilm assay uncovered several <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulators, a number of genes previously described as essential for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression were not found. This was partly the result of a stringent cutoff in the biofilm assay that led to a large number of false negatives. The absence of some known <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> regulators in the biofilm assay could also mean that certain gene products are only conditionally essential for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression. Hence, some genes might be essential for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent pseudohyphal growth but not <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-dependent biofilm development (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> 2012</a>). One example is the two PKAs, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a>. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> is essential for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression in pseudohyphal growth, whereas <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> partially represses <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription under these conditions (<a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Robertson and Fink 1998</a>). We found that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> was not essential for biofilm formation, although detailed analysis of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">tpk2</a></em> mutant revealed that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006124">Tpk2p</a> contributed substantially to the biofilm phenotype (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> appears to have two effects on biofilm formation. First, a 2.7-fold increase in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutant suggested that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> repressed <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription in biofilms (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>). Second, deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">TPK3</a></em> led to complete biofilm loss (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>), despite the presence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutants. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> thus must also affect posttranscriptional levels of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>, as indicated in the model in <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4C</a>. The positive role of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> in biofilm development is thus opposite to its role in pseudohyphal growth. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">Tpk3p</a> is reported to be essential for redistribution of polysomes on glucose starvation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashe <em>et al.</em> 2000</a>) and might also be involved in polysome maintenance of, for example, polysomes with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA. This would explain why the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001649">tpk3</a></em> mutant did not form a biofilm despite the high levels of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA.</p><p id="p-42">Although a number of genes previously described as regulating <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> were not found in our screen, we did find a strong correlation between genes essential for biofilm formation and genes essential for two other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>-related phenotypes (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5, D and E</a>). Of the 71 mutants that did not form a biofilm, 69 also lost the ability to form mats, even though the growth conditions for the two phenotypes are very different. Mats are formed at room temperature on semisolid complex medium and are characterized by large, flat colonies with cable-like structures and a central hub. In our experiments, biofilms were formed in liquid synthetic medium at 30°. A smaller subset of 49 genes involved in biofilm and mat formation was also essential for invasive growth on solid complex medium. Although both invasive growth and mat formation are assayed on complex agar medium, the overlap between genes essential for invasive growth and mat formation was smaller than the overlap of genes essential for mat and biofilm formation. This result suggested that a single genetic program mediated the three developmental phenotypes of biofilm and mat formation and invasive growth, whereas at least one other genetic program was specific for mat and biofilm formation (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5, D and E</a>). The common genetic program between the three phenotypes appeared to be almost entirely at the level of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription. Only a few genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004089">ICT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005065">TOM70</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003338">VOA1</a></em>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003498">YGR266W</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004116">YLR126C</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005458">YOL098C</a>, did not affect <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> mRNA levels significantly when deleted (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-43">Several of the genes we identified as involved in biofilm formation were recently reported to be involved in biofilm formation or biofilm-related phenotypes, although their function has not been linked to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression. <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> (2012)</a> found 655 genes essential for mat formation, whereas 211 genes were found to be essential for structured colony morphology (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Voordeckers <em>et al.</em> 2012</a>). Furthermore, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Granek <em>et al.</em> (2013)</a> reported that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001521">RGT1</a></em> was within a quantitative trait locus found using a biofilm-forming clinical isolate of <em>S. cerevisiae</em>. We compared genes essential for biofilm and mat (this study) and genes essential for mat/surface-spreading biofilm identified by <a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> (2012)</a> and found an overlap of 38 genes (File S7). Both studies are based on the use of the same deletion strain collection and the <em>a priori</em> assumption was an overlap of 69 genes. Some of the discrepancies in our findings might be ascribed to the method by which mutants were recorded as biofilm and mat formers. Although <a id="xref-ref-37-10" class="xref-bibr" href="#ref-37">Ryan <em>et al.</em> (2012)</a> applied colony size as a measure of mat formation, we used colony morphology to determine if mutants formed mats and the adhering proportion of a population to determine biofilm.</p></div><div class="section conclusions" id="sec-20"><h2 class="">Conclusion</h2><p id="p-44">A genome-wide screen of yeast mutants identified that 71 genes were essential for biofilm development. Half of the genes were required for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription, but only a small subset is previously described as regulators of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> transcription. These results revealed that the regulation of biofilm formation and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression is far more complex than previously anticipated. The results of this study will be beneficial for our research in yeast biofilm in general for identification of targets for antifungal drugs and new targets for studying biofilms, mats, and pseudohyphal and invasive growth. Biofilm formed by genetically identical eukaryotic cells might be considered as a primitive form of multicellularity. In this context, the current study therefore provides genetic data for understanding of the development programs for primitive multicellularity in a eukaryotic organism.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">The authors thank Charles Boone and Owen Ryan for the Σ1278b mutant collection and discussions about experimental design. Funding was provided by the Danish Agency for Science Technology and Innovation (FTP 10-084027).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010892/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010892/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-46"><em>Communicating editor: J. H. McCusker</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-05"><span class="received-label">Received </span>March 5, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-06-26"><span class="accepted-label">Accepted </span>June 26, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Andersen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.1" data-doi="10.1091/mbc.11.3.833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashe</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Long</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Glucose depletion rapidly inhibits translation initiation in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DAshe%26rft.auinit1%253DM.%2BP.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D833%26rft.epage%253D848%26rft.atitle%253DGlucose%2BDepletion%2BRapidly%2BInhibits%2BTranslation%2BInitiation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.3.833%26rft_id%253Dinfo%253Apmid%252F10712503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjExLzMvODMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.2" data-doi="10.1534/genetics.107.081315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrales</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimenez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibeas</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of novel activation mechanisms for <em>FLO11</em> regulation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBarrales%26rft.auinit1%253DR.%2BR.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D156%26rft.atitle%253DIdentification%2Bof%2BNovel%2BActivation%2BMechanisms%2Bfor%2BFLO11%2BRegulation%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081315%26rft_id%253Dinfo%253Apmid%252F18202364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvMTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.3" data-doi="10.1016/j.femsyr.2005.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bayly</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Characteristics of Flo11-dependent flocculation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.femsyr.2005.05.004%26rft_id%253Dinfo%253Apmid%252F16043420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjk6IjUvMTIvMTE1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.4" data-doi="10.1091/mbc.E03-01-0042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braus</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grundmann</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brückner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mösch</span>  <span class="cit-name-given-names">H. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Amino acid starvation and Gcn4p regulate adhesive growth and <em>FLO11</em> gene expression in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4272</span>–<span class="cit-lpage">4284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E03-01-0042%26rft_id%253Dinfo%253Apmid%252F14517335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMC80MjcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brückner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mösch</span>  <span class="cit-name-given-names">H. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Choosing the right lifestyle: adhesion and development in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft.volume%253D36%26rft.spage%253D25%26rft_id%253Dinfo%253Apmid%252F21521246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21521246&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.6" data-doi="10.1073/pnas.0909641106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bumgarner</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowell</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grisafi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Toggle involving cis-interfering noncoding RNAs controls variegated gene expression in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">18321</span>–<span class="cit-lpage">18326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909641106%26rft_id%253Dinfo%253Apmid%252F19805129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQzLzE4MzIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.7" data-doi="10.1016/j.molcel.2011.11.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bumgarner</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symbor-Nagrabska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grisafi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single-cell analysis reveals that noncoding RNAs contribute to clonal heterogeneity by modulating transcription factor recruitment.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBumgarner%26rft.auinit1%253DS.%2BL.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D470%26rft.epage%253D482%26rft.atitle%253DSingle-cell%2Banalysis%2Breveals%2Bthat%2Bnoncoding%2BRNAs%2Bcontribute%2Bto%2Bclonal%2Bheterogeneity%2Bby%2Bmodulating%2Btranscription%2Bfactor%2Brecruitment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.11.029%26rft_id%253Dinfo%253Apmid%252F22264825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.11.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22264825&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300753800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.8" data-doi="10.1534/genetics.111.134080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cain</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Homann</span>  <span class="cit-name-given-names">O. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A conserved transcriptional regulator governs fungal morphology in widely diverged species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134080%26rft_id%253Dinfo%253Apmid%252F22095082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNTExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.9" data-doi="10.1101/gad.1411806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Feedback control of morphogenesis in fungi by aromatic alcohols.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1150</span>–<span class="cit-lpage">1161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1411806%26rft_id%253Dinfo%253Apmid%252F16618799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiMjAvOS8xMTUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.10" data-doi="10.1074/jbc.M102944200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Craene</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soetens</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andre</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The Npr1 kinase controls biosynthetic and endocytic sorting of the yeast Gap1 permease.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">43939</span>–<span class="cit-lpage">43948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M102944200%26rft_id%253Dinfo%253Apmid%252F11500493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzYvNDcvNDM5MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.11" data-doi="10.1111/j.1365-2958.2007.06014.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fichtner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braus</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Differential Flo8p-dependent regulation of <em>FLO1</em> and <em>FLO11</em> for cell-cell and cell-substrate adherence of <em>S. cerevisiae</em> S288c.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1276</span>–<span class="cit-lpage">1289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DFichtner%26rft.auinit1%253DL.%26rft.volume%253D66%26rft.issue%253D5%26rft.spage%253D1276%26rft.epage%253D1289%26rft.atitle%253DDifferential%2BFlo8p-dependent%2Bregulation%2Bof%2BFLO1%2Band%2BFLO11%2Bfor%2Bcell-cell%2Band%2Bcell-substrate%2Badherence%2Bof%2BS.%2Bcerevisiae%2BS288c.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2007.06014.x%26rft_id%253Dinfo%253Apmid%252F18001350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2007.06014.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18001350&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.12" data-doi="10.1046/j.1365-2958.2003.03247.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gagiano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franken</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mss11p is a transcription factor regulating pseudohyphal differentiation, invasive growth and starch metabolism in <em>Saccharomyces cerevisiae</em> in response to nutrient availability.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DGagiano%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D134%26rft.atitle%253DMss11p%2Bis%2Ba%2Btranscription%2Bfactor%2Bregulating%2Bpseudohyphal%2Bdifferentiation%252C%2Binvasive%2Bgrowth%2Band%2Bstarch%2Bmetabolism%2Bin%2BSaccharomyces%2Bcerevisiae%2Bin%2Bresponse%2Bto%2Bnutrient%2Bavailability.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03247.x%26rft_id%253Dinfo%253Apmid%252F12492858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03247.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12492858&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.13" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.14" data-doi="10.1016/0092-8674(92)90079-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimeno</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Unipolar cell divisions in the yeast <em>S. cerevisiae</em> lead to filamentous growth: regulation by starvation and RAS.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGimeno%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D68%26rft.issue%253D6%26rft.spage%253D1077%26rft.epage%253D1090%26rft.atitle%253DUnipolar%2Bcell%2Bdivisions%2Bin%2Bthe%2Byeast%2BS.%2Bcerevisiae%2Blead%2Bto%2Bfilamentous%2Bgrowth%253A%2Bregulation%2Bby%2Bstarvation%2Band%2BRAS.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990079-R%26rft_id%253Dinfo%253Apmid%252F1547504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90079-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1547504&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HK67400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.15" data-doi="10.1534/genetics.112.142067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayrkci</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic architecture of biofilm formation in a clinical isolate of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142067%26rft_id%253Dinfo%253Apmid%252F23172850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNTg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.16" data-doi="10.1073/pnas.220420397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A <em>Saccharomyces</em> gene family involved in invasive growth, cell-cell adhesion, and mating.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12158</span>–<span class="cit-lpage">12163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.220420397%26rft_id%253Dinfo%253Apmid%252F11027318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjIvMTIxNTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1671.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source">Guide to Yeast Genetics and Molecular Biology. Methods Enzymol</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.18" data-doi="10.1038/nrmicro821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall-Stoodley</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costerton</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoodley</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bacterial biofilms: from the natural environment to infectious diseases.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DHall-Stoodley%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D95%26rft.epage%253D108%26rft.atitle%253DBacterial%2Bbiofilms%253A%2Bfrom%2Bthe%2Bnatural%2Benvironment%2Bto%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro821%26rft_id%253Dinfo%253Apmid%252F15040259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15040259&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220431700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.19" data-doi="10.1016/S0092-8674(04)00118-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halme</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bumgarner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic and epigenetic regulation of the <em>FLO</em> gene family generates cell-surface variation in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHalme%26rft.auinit1%253DA.%26rft.volume%253D116%26rft.issue%253D3%26rft.spage%253D405%26rft.epage%253D415%26rft.atitle%253DGenetic%2Band%2Bepigenetic%2Bregulation%2Bof%2Bthe%2BFLO%2Bgene%2Bfamily%2Bgenerates%2Bcell-surface%2Bvariation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900118-7%26rft_id%253Dinfo%253Apmid%252F15016375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00118-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15016375&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188825800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawser</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Biofilm formation by <em>Candida</em> species on the surface of catheter materials in vitro.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DHawser%26rft.auinit1%253DS%2BP%26rft.volume%253D62%26rft.issue%253D3%26rft.spage%253D915%26rft.epage%253D921%26rft.atitle%253DBiofilm%2Bformation%2Bby%2BCandida%2Bspecies%2Bon%2Bthe%2Bsurface%2Bof%2Bcatheter%2Bmaterials%2Bin%2Bvitro.%26rft_id%253Dinfo%253Apmid%252F8112864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjg6IjYyLzMvOTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.21" data-doi="10.1128/MCB.22.12.3994-4000.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuchin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Snf1 protein kinase and the repressors Nrg1 and Nrg2 regulate <em>FLO11</em>, haploid invasive growth, and diploid pseudohyphal differentiation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3994</span>–<span class="cit-lpage">4000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKuchin%26rft.auinit1%253DS.%26rft.volume%253D22%26rft.issue%253D12%26rft.spage%253D3994%26rft.epage%253D4000%26rft.atitle%253DSnf1%2BProtein%2BKinase%2Band%2Bthe%2BRepressors%2BNrg1%2Band%2BNrg2%2BRegulate%2BFLO11%252C%2BHaploid%2BInvasive%2BGrowth%252C%2Band%2BDiploid%2BPseudohyphal%2BDifferentiation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.12.3994-4000.2002%26rft_id%253Dinfo%253Apmid%252F12024013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8zOTk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.22" data-doi="10.1046/j.1365-2958.2003.03332.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuthan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devaux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janderova</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slaninova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacq</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Domestication of wild <em>Saccharomyces cerevisiae</em> is accompanied by changes in gene expression and colony morphology.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKuthan%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D745%26rft.epage%253D754%26rft.atitle%253DDomestication%2Bof%2Bwild%2BSaccharomyces%2Bcerevisiae%2Bis%2Baccompanied%2Bby%2Bchanges%2Bin%2Bgene%2Bexpression%2Band%2Bcolony%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03332.x%26rft_id%253Dinfo%253Apmid%252F12535073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03332.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12535073&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180776700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.23" data-doi="10.1128/EC.1.5.673-686.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Köhler</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wesche</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taheri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braus</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mösch</span>  <span class="cit-name-given-names">H. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dual role of the <em>Saccharomyces cerevisiae</em> TEA/ATTS family transcription factor Tec1p in regulation of gene expression and cellular development.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DKohler%26rft.auinit1%253DT.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D673%26rft.epage%253D686%26rft.atitle%253DDual%2BRole%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2BTEA%252FATTS%2BFamily%2BTranscription%2BFactor%2BTec1p%2Bin%2BRegulation%2Bof%2BGene%2BExpression%2Band%2BCellular%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.1.5.673-686.2002%26rft_id%253Dinfo%253Apmid%252F12455687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIxLzUvNjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.24" data-doi="10.1128/MCB.23.2.677-686.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The transcription factor Rim101p governs ion tolerance and cell differentiation by direct repression of the regulatory genes <em>NRG1</em> and <em>SMP1</em> in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLamb%26rft.auinit1%253DT.%2BM.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D677%26rft.epage%253D686%26rft.atitle%253DThe%2BTranscription%2BFactor%2BRim101p%2BGoverns%2BIon%2BTolerance%2Band%2BCell%2BDifferentiation%2Bby%2BDirect%2BRepression%2Bof%2Bthe%2BRegulatory%2BGenes%2BNRG1%2Band%2BSMP1%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.2.677-686.2003%26rft_id%253Dinfo%253Apmid%252F12509465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvNjc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> S288C has a mutation in <em>FLO8</em>, a gene required for filamentous growth.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D967%26rft.epage%253D978%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BS288C%2BHas%2Ba%2BMutation%2Bin%2BFL08%252C%2Ba%2BGene%2BRequired%2Bfor%2BFilamentous%2BGrowth%26rft_id%253Dinfo%253Apmid%252F8913742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzMvOTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.26" data-doi="10.1128/jb.178.24.7144-7151.1996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>FLO11</em>, a yeast gene related to the <em>STA</em> genes, encodes a novel cell surface flocculin.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">7144</span>–<span class="cit-lpage">7151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLo%26rft.auinit1%253DW.%2BS.%26rft.volume%253D178%26rft.issue%253D24%26rft.spage%253D7144%26rft.epage%253D7151%26rft.atitle%253DFLO11%252C%2Ba%2Byeast%2Bgene%2Brelated%2Bto%2Bthe%2BSTA%2Bgenes%252C%2Bencodes%2Ba%2Bnovel%2Bcell%2Bsurface%2Bflocculin.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.178.24.7144-7151.1996%26rft_id%253Dinfo%253Apmid%252F8955395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3OC8yNC83MTQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.27" data-doi="10.1091/mbc.9.1.161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The cell surface flocculin Flo11 is required for pseudohyphae formation and invasion by <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DLo%26rft.auinit1%253DW.-S.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D161%26rft.epage%253D171%26rft.atitle%253DThe%2BCell%2BSurface%2BFlocculin%2BFlo11%2BIs%2BRequired%2Bfor%2BPseudohyphae%2BFormation%2Band%2BInvasion%2Bby%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.1.161%26rft_id%253Dinfo%253Apmid%252F9436998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjkvMS8xNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.28" data-doi="10.1093/emboj/17.5.1236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The MEP2 ammonium permease regulates pseudohyphal differentiation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DLorenz%26rft.auinit1%253DM.%2BC.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1236%26rft.epage%253D1247%26rft.atitle%253DThe%2BMEP2%2Bammonium%2Bpermease%2Bregulates%2Bpseudohyphal%2Bdifferentiation%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.5.1236%26rft_id%253Dinfo%253Apmid%252F9482721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxNy81LzEyMzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinnebusch</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Positive regulatory interactions of the <em>HIS4</em> gene of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1326</span>–<span class="cit-lpage">1333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLucchini%26rft.auinit1%253DG%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D1326%26rft.epage%253D1333%26rft.atitle%253DPositive%2Bregulatory%2Binteractions%2Bof%2Bthe%2BHIS4%2Bgene%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F6095062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjQvNy8xMzI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIsaac</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briehof</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Visualization and analysis of mRNA molecules using fluorescence <em>in situ</em> hybridization in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">e50382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253D76%26rft.spage%253De50382%26rft_id%253Dinfo%253Apmid%252F23793137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23793137&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.31" data-doi="10.1126/science.1211037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi-Datar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepine</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauerle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olakanmi</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Active starvation responses mediate antibiotic tolerance in biofilms and nutrient-limited bacteria.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNguyen%26rft.auinit1%253DD.%26rft.volume%253D334%26rft.issue%253D6058%26rft.spage%253D982%26rft.epage%253D986%26rft.atitle%253DActive%2BStarvation%2BResponses%2BMediate%2BAntibiotic%2BTolerance%2Bin%2BBiofilms%2Band%2BNutrient-Limited%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1211037%26rft_id%253Dinfo%253Apmid%252F22096200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzQvNjA1OC85ODIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.32" data-doi="10.1038/37398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palkova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janderova</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zikanova</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pospisek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Ammonia mediates communication between yeast colonies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">390</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPalkov%25C3%25A1%26rft.auinit1%253DZ.%26rft.volume%253D390%26rft.issue%253D6659%26rft.spage%253D532%26rft.epage%253D536%26rft.atitle%253DAmmonia%2Bmediates%2Bcommunication%2Bbetween%2Byeast%2Bcolonies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F37398%26rft_id%253Dinfo%253Apmid%252F9394006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/37398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9394006&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YJ86500058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.33" data-doi="10.1126/science.291.5505.878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bakers’ yeast, a model for fungal biofilm formation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">878</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReynolds%26rft.auinit1%253DT.%2BB.%26rft.volume%253D291%26rft.issue%253D5505%26rft.spage%253D878%26rft.epage%253D881%26rft.atitle%253DBakers%2527%2BYeast%252C%2Ba%2BModel%2Bfor%2BFungal%2BBiofilm%2BFormation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.291.5505.878%26rft_id%253Dinfo%253Apmid%252F11157168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTEvNTUwNS84NzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.34" data-doi="10.1101/gad.8.24.2974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Elements of a single MAP kinase cascade in <em>Saccharomyces cerevisiae</em> mediate two developmental programs in the same cell type: mating and invasive growth.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2974</span>–<span class="cit-lpage">2985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRoberts%26rft.auinit1%253DR%2BL%26rft.volume%253D8%26rft.issue%253D24%26rft.spage%253D2974%26rft.epage%253D2985%26rft.atitle%253DElements%2Bof%2Ba%2Bsingle%2BMAP%2Bkinase%2Bcascade%2Bin%2BSaccharomyces%2Bcerevisiae%2Bmediate%2Btwo%2Bdevelopmental%2Bprograms%2Bin%2Bthe%2Bsame%2Bcell%2Btype%253A%2Bmating%2Band%2Binvasive%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.24.2974%26rft_id%253Dinfo%253Apmid%252F8001818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOC8yNC8yOTc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.35" data-doi="10.1073/pnas.95.23.13783"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The three yeast A kinases have specific signaling functions in pseudohyphal growth.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">13783</span>–<span class="cit-lpage">13787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRobertson%26rft.auinit1%253DL.%2BS.%26rft.volume%253D95%26rft.issue%253D23%26rft.spage%253D13783%26rft.epage%253D13787%26rft.atitle%253DThe%2Bthree%2Byeast%2BA%2Bkinases%2Bhave%2Bspecific%2Bsignaling%2Bfunctions%2Bin%2Bpseudohyphal%2Bgrowth%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.23.13783%26rft_id%253Dinfo%253Apmid%252F9811878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMjMvMTM3ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.36" data-doi="10.1093/emboj/18.5.1257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rupp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madhani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast <em>FLO11</em> gene.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DRupp%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D1257%26rft.epage%253D1269%26rft.atitle%253DMAP%2Bkinase%2Band%2BcAMP%2Bfilamentation%2Bsignaling%2Bpathways%2Bconverge%2Bon%2Bthe%2Bunusually%2Blarge%2Bpromoter%2Bof%2Bthe%2Byeast%2BFLO11%2Bgene%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F18.5.1257%26rft_id%253Dinfo%253Apmid%252F10064592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxOC81LzEyNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.37" data-doi="10.1126/science.1224339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurat</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayhew</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global gene deletion analysis exploring yeast filamentous growth.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRyan%26rft.auinit1%253DO.%26rft.volume%253D337%26rft.issue%253D6100%26rft.spage%253D1353%26rft.epage%253D1356%26rft.atitle%253DGlobal%2BGene%2BDeletion%2BAnalysis%2BExploring%2BYeast%2BFilamentous%2BGrowth%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1224339%26rft_id%253Dinfo%253Apmid%252F22984072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjEwMC8xMzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.38" data-doi="10.1128/EC.05078-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarode</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miracle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Vacuolar protein sorting genes regulate mat formation in <em>Saccharomyces cerevisiae</em> by Flo11p-dependent and -independent mechanisms.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1516</span>–<span class="cit-lpage">1526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.05078-11%26rft_id%253Dinfo%253Apmid%252F21908597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czoxMDoiMTAvMTEvMTUxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.39" data-doi="10.1016/j.cell.2008.09.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smukalla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pochet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beauvais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guadagnini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>FLO1</em> is a variable green beard gene that drives biofilm-like cooperation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">726</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSmukalla%26rft.auinit1%253DS.%26rft.volume%253D135%26rft.issue%253D4%26rft.spage%253D726%26rft.epage%253D737%26rft.atitle%253DFLO1%2Bis%2Ba%2Bvariable%2Bgreen%2Bbeard%2Bgene%2Bthat%2Bdrives%2Bbiofilm-like%2Bcooperation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.09.037%26rft_id%253Dinfo%253Apmid%252F19013280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.09.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19013280&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260886900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.40" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.41" data-doi="10.1371/journal.pone.0041272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torbensen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Møller</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alizadeh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Amino acid transporter genes are essential for <em>FLO11</em>-dependent and <em>FLO11</em>-independent biofilm formation and invasive growth in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTorbensen%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41272%26rft.epage%253De41272%26rft.atitle%253DAmino%2Bacid%2Btransporter%2Bgenes%2Bare%2Bessential%2Bfor%2BFLO11-dependent%2Band%2BFLO11-independent%2Bbiofilm%2Bformation%2Band%2Binvasive%2Bgrowth%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041272%26rft_id%253Dinfo%253Apmid%252F22844449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22844449&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.42" data-doi="10.1083/jcb.201103129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vachova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stovicek</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hlavacek</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernyavskiy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stepanek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Flo11p, drug efflux pumps, and the extracellular matrix cooperate to form biofilm yeast colonies.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201103129%26rft_id%253Dinfo%253Apmid%252F21875945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE5NC81LzY3OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.43" data-doi="10.1128/EC.00276-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van De Velde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thevelein</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cyclic AMP-protein kinase A and Snf1 signaling mechanisms underlie the superior potency of sucrose for induction of filamentation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00276-07%26rft_id%253Dinfo%253Apmid%252F17890371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI3LzIvMjg2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.44" data-doi="10.1007/BF00633845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vandenbol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jauniaux</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae NPR1</em> gene required for the activity of ammonia-sensitive amino acid permeases encodes a protein kinase homologue.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">222</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DVandenbol%26rft.auinit1%253DM.%26rft.volume%253D222%26rft.issue%253D2-3%26rft.spage%253D393%26rft.epage%253D399%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BNPR1%2Bgene%2Brequired%2Bfor%2Bthe%2Bactivity%2Bof%2Bammonia-sensitive%2Bamino%2Bacid%2Bpermeases%2Bencodes%2Ba%2Bprotein%2Bkinase%2Bhomologue.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00633845%26rft_id%253Dinfo%253Apmid%252F2125693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00633845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2125693&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DR65400032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.45" data-doi="10.1111/j.1365-2958.2012.08192.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voordeckers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Maeyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Zande</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinces</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meert</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of a complex genetic network underlying <em>Saccharomyces cerevisiae</em> colony morphology.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DVoordeckers%26rft.auinit1%253DK.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D239%26rft.atitle%253DIdentification%2Bof%2Ba%2Bcomplex%2Bgenetic%2Bnetwork%2Bunderlying%2BSaccharomyces%2Bcerevisiae%2Bcolony%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08192.x%26rft_id%253Dinfo%253Apmid%252F22882838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08192.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22882838&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1671.46" data-doi="10.1091/mbc.E04-08-0666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">F. J.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subaran</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Multivesicular body-ESCRT components function in pH response regulation in <em>Saccharomyces cerevisiae</em> and <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">5528</span>–<span class="cit-lpage">5537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E04-08-0666%26rft_id%253Dinfo%253Apmid%252F15371534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi81NTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1671.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1667" title="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1681" title="Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1671" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Zm0LGSK3bTTq8V8wle4fJ1cpAwpDbvS-72G-gVE3EG8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1671" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1671" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1671" class="active" data-icon-position="" data-hide-link-title="0">Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459660" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qOJ1JKGgp5pmJXQAzgoGvwpehSLmxVTZaM7SHfVMdwk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1671.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1671" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pH--iUGHgPD-TNO8aK0cm6kRA5079VnKXoURHOTwToU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1671" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1671.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459660" id="node459660--2" data-pisa="ggg;4/9/1671" data-pisa-master="ggg;g3.114.010892" data-apath="/ggg/4/9/1671.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Basis for <em>Saccharomyces cerevisiae</em> Biofilm in Liquid Medium</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kaj Scherz Andersen</span>, <span class="highwire-citation-author" data-delta="1">Rasmus Bojsen</span>, <span class="highwire-citation-author" data-delta="2">Laura Gro Rejkjær Sørensen</span>, <span class="highwire-citation-author" data-delta="3">Martin Weiss Nielsen</span>, <span class="highwire-citation-author" data-delta="4">Michael Lisby</span>, <span class="highwire-citation-author" data-delta="5">Anders Folkesson</span> and <span class="highwire-citation-author" data-delta="6">Birgitte Regenberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1671-1680;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010892 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459660--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kaj Scherz Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaj%2BScherz%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaj%2BScherz%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rasmus Bojsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rasmus%2BBojsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bojsen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARasmus%2BBojsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laura Gro Rejkjær Sørensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BGro%2BRejkj%C3%A6r%2BS%C3%B8rensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%B8rensen%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BGro%2BRejkj%25C3%25A6r%2BS%25C3%25B8rensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Weiss Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BWeiss%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BWeiss%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Lisby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLisby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lisby%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLisby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anders Folkesson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BFolkesson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Folkesson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnders%2BFolkesson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Birgitte Regenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Birgitte%2BRegenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Regenberg%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABirgitte%2BRegenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bregenberg@bio.ku.dk" class="" data-icon-position="" data-hide-link-title="0">bregenberg@bio.ku.dk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1671" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1671&amp;title=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1671&amp;title=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1671&amp;title=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1671&amp;text=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1671&amp;title=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1671&amp;t=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1671&amp;title=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1671&amp;title=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459660/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459660" id="node459660--3" data-pisa="ggg;4/9/1671" data-pisa-master="ggg;g3.114.010892" data-apath="/ggg/4/9/1671.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Basis for <em>Saccharomyces cerevisiae</em> Biofilm in Liquid Medium</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kaj Scherz Andersen</span>, <span class="highwire-citation-author" data-delta="1">Rasmus Bojsen</span>, <span class="highwire-citation-author" data-delta="2">Laura Gro Rejkjær Sørensen</span>, <span class="highwire-citation-author" data-delta="3">Martin Weiss Nielsen</span>, <span class="highwire-citation-author" data-delta="4">Michael Lisby</span>, <span class="highwire-citation-author" data-delta="5">Anders Folkesson</span> and <span class="highwire-citation-author" data-delta="6">Birgitte Regenberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1671-1680;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010892 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459660--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kaj Scherz Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaj%2BScherz%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaj%2BScherz%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rasmus Bojsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rasmus%2BBojsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bojsen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARasmus%2BBojsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laura Gro Rejkjær Sørensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BGro%2BRejkj%C3%A6r%2BS%C3%B8rensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%B8rensen%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BGro%2BRejkj%25C3%25A6r%2BS%25C3%25B8rensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Weiss Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BWeiss%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BWeiss%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Lisby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLisby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lisby%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLisby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anders Folkesson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BFolkesson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Folkesson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnders%2BFolkesson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Birgitte Regenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Copenhagen, Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Birgitte%2BRegenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Regenberg%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABirgitte%2BRegenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bregenberg@bio.ku.dk" class="" data-icon-position="" data-hide-link-title="0">bregenberg@bio.ku.dk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1671&amp;rft.epage=1680&amp;rft.atitle=Genetic%20Basis%20for%20Saccharomyces%20cerevisiae%20Biofilm%20in%20Liquid%20Medium&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Andersen%2C+Kaj+Scherz&amp;rft.au=Bojsen%2C+Rasmus&amp;rft.au=S%C3%B8rensen%2C+Laura+Gro+Rejkj%C3%A6r&amp;rft.au=Nielsen%2C+Martin+Weiss&amp;rft.au=Lisby%2C+Michael&amp;rft.au=Folkesson%2C+Anders&amp;rft.au=Regenberg%2C+Birgitte"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459660/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459660/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459660/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459660/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459660/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459660/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459660/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459660/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459660/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459660/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459660/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459660/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1671" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1671.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1671.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1671.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1671.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
