<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1681" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1681.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1681.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1681.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1681.atom" />
<meta name="HW.pisa" content="ggg;4/9/1681" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.114.011023" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Gardner et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Next-generation DNA sequencing (NGS) produces vast amounts of DNA sequence data, but it is not specifically designed to generate data suitable for genetic mapping. Recently developed DNA library preparation methods for NGS have helped solve this problem, however, by combining the use of reduced representation libraries with DNA sample barcoding to generate genome-wide genotype data from a common set of genetic markers across a large number of samples. Here we use such a method, called genotyping-by-sequencing (GBS), to produce a data set for genetic mapping in an F1 population of apples ( Malus × domestica ) segregating for skin color. We show that GBS produces a relatively large, but extremely sparse, genotype matrix: over 270,000 SNPs were discovered but most SNPs have too much missing data across samples to be useful for genetic mapping. After filtering for genotype quality and missing data, only 6% of the 85 million DNA sequence reads contributed to useful genotype calls. Despite this limitation, using existing software and a set of simple heuristics, we generated a final genotype matrix containing 3967 SNPs from 89 DNA samples from a single lane of Illumina HiSeq and used it to create a saturated genetic linkage map and to identify a known QTL underlying apple skin color. We therefore demonstrate that GBS is a cost-effective method for generating genome-wide SNP data suitable for genetic mapping in a highly diverse and heterozygous agricultural species. We anticipate future improvements to the GBS analysis pipeline presented here that will enhance the utility of next-generation DNA sequence data for the purposes of genetic mapping across diverse species." />
<meta name="DC.Contributor" content="Kyle M. Gardner" />
<meta name="DC.Contributor" content="Patrick Brown" />
<meta name="DC.Contributor" content="Thomas F. Cooke" />
<meta name="DC.Contributor" content="Scott Cann" />
<meta name="DC.Contributor" content="Fabrizio Costa" />
<meta name="DC.Contributor" content="Carlos Bustamante" />
<meta name="DC.Contributor" content="Riccardo Velasco" />
<meta name="DC.Contributor" content="Michela Troggio" />
<meta name="DC.Contributor" content="Sean Myles" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Next-generation DNA sequencing (NGS) produces vast amounts of DNA sequence data, but it is not specifically designed to generate data suitable for genetic mapping. Recently developed DNA library preparation methods for NGS have helped solve this problem, however, by combining the use of reduced representation libraries with DNA sample barcoding to generate genome-wide genotype data from a common set of genetic markers across a large number of samples. Here we use such a method, called genotyping-by-sequencing (GBS), to produce a data set for genetic mapping in an F1 population of apples (&lt;i&gt;Malus&lt;/i&gt; × &lt;i&gt;domestica&lt;/i&gt;) segregating for skin color. We show that GBS produces a relatively large, but extremely sparse, genotype matrix: over 270,000 SNPs were discovered but most SNPs have too much missing data across samples to be useful for genetic mapping. After filtering for genotype quality and missing data, only 6% of the 85 million DNA sequence reads contributed to useful genotype calls. Despite this limitation, using existing software and a set of simple heuristics, we generated a final genotype matrix containing 3967 SNPs from 89 DNA samples from a single lane of Illumina HiSeq and used it to create a saturated genetic linkage map and to identify a known QTL underlying apple skin color. We therefore demonstrate that GBS is a cost-effective method for generating genome-wide SNP data suitable for genetic mapping in a highly diverse and heterozygous agricultural species. We anticipate future improvements to the GBS analysis pipeline presented here that will enhance the utility of next-generation DNA sequence data for the purposes of genetic mapping across diverse species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1681" />
<meta name="citation_id" content="4/9/1681" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1681" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1681.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1681.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1681.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011023" />
<meta name="citation_pmid" content="25031181" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1681" />
<meta name="citation_lastpage" content="1687" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kyle M. Gardner" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada" />
<meta name="citation_author" content="Patrick Brown" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois, Urbana, Illinois" />
<meta name="citation_author" content="Thomas F. Cooke" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California" />
<meta name="citation_author" content="Scott Cann" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada" />
<meta name="citation_author" content="Fabrizio Costa" />
<meta name="citation_author_institution" content="Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy" />
<meta name="citation_author" content="Carlos Bustamante" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California" />
<meta name="citation_author" content="Riccardo Velasco" />
<meta name="citation_author_institution" content="Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy" />
<meta name="citation_author" content="Michela Troggio" />
<meta name="citation_author_institution" content="Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy" />
<meta name="citation_author" content="Sean Myles" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada" />
<meta name="citation_author_email" content="sean.myles@dal.ca" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Antanaviciute;citation_author=F. Fernandez-Fernandez;citation_author=J. Jansen;citation_author=E. Banchi;citation_author=K. Evans;citation_title=Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array.;citation_pages=203-203;citation_volume=13;citation_year=2012;citation_pmid=22631220;citation_doi=10.1186/1471-2164-13-203" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Barba;citation_author=L. Cadle-Davidson;citation_author=J. Harriman;citation_author=J. Glaubitz;citation_author=S. Brooks;citation_title=Grapevine powdery mildew resistance and susceptibility loci identified on a high-resolution SNP map.;citation_pages=73-84;citation_volume=127;citation_year=2014;citation_pmid=24072208;citation_doi=10.1007/s00122-013-2202-x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Acta Hortic.;citation_author=V. Bus;citation_author=C. Ranatunga;citation_author=S. Gardiner;citation_author=H. Basset;citation_author=E. Rikkerink;citation_title=Marker-assisted selection for pest and disease resistance in the New Zealand apple breeding program.;citation_pages=541-547;citation_volume=538;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. M. Celton;citation_author=D. S. Tustin;citation_author=D. Chagné;citation_author=S. E. Gardiner;citation_title=Construction of a dense genetic linkage map for apple rootstocks using SSRs developed from Malus ESTs and Pyrus genomic sequences.;citation_pages=93-107;citation_volume=5;citation_year=2009;citation_doi=10.1007/s11295-008-0171-z" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Chagne;citation_author=R. N. Crowhurst;citation_author=M. Troggio;citation_author=M. W. Davey;citation_author=B. Gilmore;citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.;citation_pages=e31745-e31745;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363718;citation_doi=10.1371/journal.pone.0031745" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Chagne;citation_author=C. Krieger;citation_author=M. Rassam;citation_author=M. Sullivan;citation_author=J. Fraser;citation_title=QTL and candidate gene mapping for polyphenolic composition in apple fruit.;citation_pages=12-12;citation_volume=12;citation_year=2012;citation_pmid=22269060;citation_doi=10.1186/1471-2229-12-12" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. Chagne;citation_author=K. Lin-Wang;citation_author=R. V. Espley;citation_author=R. K. Volz;citation_author=N. M. How;citation_title=An ancient duplication of apple MYB transcription factors is responsible for novel red fruit-flesh phenotypes.;citation_volume=161;citation_year=2013;citation_pmid=23096157;citation_doi=10.1104/pp.112.206771" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. I. B. G. S. Consortium;citation_title=A physical, genetic and functional sequence assembly of the barley genome.;citation_pages=711-716;citation_volume=491;citation_year=2012;citation_issue=7426;citation_pmid=23075845" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Grape Wine Res.;citation_author=G. di Gaspero;citation_author=F. Cattonaro;citation_title=Application of genomics to grapevine improvement.;citation_pages=122-130;citation_volume=16;citation_year=2010;citation_doi=10.1111/j.1755-0238.2009.00072.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Dirlewanger;citation_author=E. Graziano;citation_author=T. Joobeur;citation_author=F. Garriga-Calderé;citation_author=P. Cosson;citation_title=Comparative mapping and marker-assisted selection in Rosaceae fruit crops.;citation_volume=101;citation_year=2004;citation_pmid=15159547;citation_doi=10.1073/pnas.0307937101" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=R. V. Espley;citation_author=R. P. Hellens;citation_author=J. Putterill;citation_author=D. E. Stevenson;citation_author=S. Kutty-Amma;citation_title=Red colouration in apple fruit is due to the activity of the MYB transcription factor, MdMYB10.;citation_pages=414-427;citation_volume=49;citation_year=2007;citation_issue=3;citation_pmid=17181777;citation_doi=10.1111/j.1365-313X.2006.02964.x" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=F. Fernández-Fernández;citation_author=L. Antanaviciute;citation_author=M. M. Dyk;citation_author=K. R. Tobutt;citation_author=K. M. Evans;citation_title=A genetic linkage map of an apple rootstock progeny anchored to the Malus genome sequence.;citation_pages=991-1002;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-012-0478-7" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot. 62: 5117–5130;citation_title=Integration of physical and genetic maps in apple confirms whole-genome and segmental duplications in the apple genome.;citation_author=Y. Han;citation_author=D. Zheng;citation_author=S. Vimolmangkang;citation_author=M. A. Khan;citation_author=J. E. Beever;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Acta Hortic.;citation_author=M. Kellerhals;citation_author=L. Gianfranceschi;citation_author=N. Seglias;citation_author=C. Gessler;citation_title=Marker-assisted selection in apple breeding.;citation_pages=255-266;citation_volume=521;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. Kenis;citation_author=J. Keulemans;citation_title=Genetic linkage maps of two apple cultivars (Malus × domestica Borkh.) based on AFLP and microsatellite markers.;citation_pages=205-219;citation_volume=15;citation_year=2005;citation_doi=10.1007/s11032-004-5592-2" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. A. Khan;citation_author=Y. Han;citation_author=Y. F. Zhao;citation_author=M. Troggio;citation_author=S. S. Korban;citation_title=A Multi-Population Consensus Genetic Map Reveals Inconsistent Marker Order among Maps Likely Attributed to Structural Variations in the Apple Genome.;citation_pages=e47864-e47864;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23144832;citation_doi=10.1371/journal.pone.0047864" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Kumar;citation_author=D. Chagné;citation_author=M. C. A. M. Bink;citation_author=R. K. Volz;citation_author=C. Whitworth;citation_title=Genomic selection for fruit quality traits in apple (MalusÃdomestica Borkh.).;citation_pages=e36674-e36674;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22574211;citation_doi=10.1371/journal.pone.0036674" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Kumar;citation_author=D. Garrick;citation_author=M. Bink;citation_author=C. Whitworth;citation_author=D. Chagne;citation_title=Novel genomic approaches unravel genetic architecture of complex traits in apple.;citation_pages=393;citation_volume=14;citation_year=2013;citation_pmid=23758946;citation_doi=10.1186/1471-2164-14-393" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=R. Liebhard;citation_author=M. Kellerhals;citation_author=W. Pfammatter;citation_author=M. Jertmini;citation_author=C. Gessler;citation_title=Mapping quantitative physiological traits in apple (Malus x domestica Borkh.).;citation_pages=511-526;citation_volume=52;citation_year=2003;citation_issue=3;citation_pmid=12956523;citation_doi=10.1023/A:1024886500979" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=K. U. I. Lin-Wang;citation_author=D. Micheletti;citation_author=J. Palmer;citation_author=R. Volz;citation_author=L. Lozano;citation_title=High temperature reduces apple fruit colour via modulation of the anthocyanin regulatory complex.;citation_pages=1176-1190;citation_volume=34;citation_year=2011;citation_issue=7;citation_pmid=21410713;citation_doi=10.1111/j.1365-3040.2011.02316.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: Novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Mascher;citation_author=G. J. Muehlbauer;citation_author=D. S. Rokhsar;citation_author=J. Chapman;citation_author=J. Schmutz;citation_title=Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).;citation_pages=718-727;citation_volume=76;citation_year=2013;citation_pmid=23998490;citation_doi=10.1111/tpj.12319" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=M. Mascher;citation_author=S. Wu;citation_author=P. S. Amand;citation_author=N. Stein;citation_author=J. Poland;citation_title=Application of genotyping-by-sequencing on semiconductor sequencing platforms: A comparison of genetic and reference-based marker ordering in barley.;citation_pages=e76925;citation_volume=8;citation_year=2013;citation_pmid=24098570;citation_doi=10.1371/journal.pone.0076925" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Micheletti;citation_author=M. Troggio;citation_author=A. Zharkikh;citation_author=F. Costa;citation_author=M. Malnoy;citation_title=Genetic diversity of the genus Malus and implications for linkage mapping with SNPs.;citation_pages=857;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-011-0380-8" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_title=Vitis phylogenomics: hybridization intensities from a SNP array outperform genotype calls.;citation_author=A. Miller;citation_author=N. Matasci;citation_author=M. K. Aradhya;citation_author=B. H. Prins;citation_author=G. Y. Zhong;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=S. Myles;citation_title=Improving fruit and wine: what does genomics have to offer?;citation_pages=190-196;citation_volume=29;citation_year=2013;citation_pmid=23428114;citation_doi=10.1016/j.tig.2013.01.006" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_title=Genomic selection in wheat using genotyping-by-sequencing.;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Romay;citation_author=M. Millard;citation_author=J. Glaubitz;citation_author=J. Peiffer;citation_author=K. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. Tardivel;citation_author=G. Légaré;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Spindel;citation_author=M. Wright;citation_author=C. Chen;citation_author=J. Cobb;citation_author=J. Gage;citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.;citation_pages=2699-2716;citation_volume=126;citation_year=2013;citation_pmid=23918062;citation_doi=10.1007/s00122-013-2166-x" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linkage maps by means of a new computer package: JoinMap.;citation_pages=739-744;citation_volume=5;citation_year=1993;citation_doi=10.1046/j.1365-313X.1993.03050739.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. M. Takos;citation_author=F. W. Jaffé;citation_author=S. R. Jacob;citation_author=J. Bogs;citation_author=S. P. Robinson;citation_title=Light-induced expression of a MYB gene regulates anthocyanin biosynthesis in red apples.;citation_volume=142;citation_year=2006;citation_pmid=17012405;citation_doi=10.1104/pp.106.088104" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. Velasco;citation_author=A. Zharkikh;citation_author=J. Affourtit;citation_author=A. Dhingra;citation_author=A. Cestaro;citation_title=The genome of the domesticated apple (Malus Ã domestica Borkh.).;citation_pages=833-839;citation_volume=42;citation_year=2010;citation_issue=10;citation_pmid=20802477;citation_doi=10.1038/ng.654" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Ward;citation_author=J. Bhangoo;citation_author=F. Fernandez-Fernandez;citation_author=P. Moore;citation_author=J. Swanson;citation_title=Saturated linkage map construction in Rubus idaeus using genotyping by sequencing and genome-independent imputation.;citation_pages=2;citation_volume=14;citation_year=2013;citation_pmid=23324311;citation_doi=10.1186/1471-2164-14-2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Y. Zhu;citation_author=K. Evans;citation_author=C. Peace;citation_title=Utility testing of an apple skin color MdMYB1 marker in two progenies.;citation_pages=525-532;citation_volume=27;citation_year=2011;citation_doi=10.1007/s11032-010-9449-6" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="Next-generation DNA sequencing (NGS) produces vast amounts of DNA sequence data, but it is not specifically designed to generate data suitable for genetic mapping. Recently developed DNA library preparation methods for NGS have helped solve this problem, however, by combining the use of reduced representation libraries with DNA sample barcoding to generate genome-wide genotype data from a common set of genetic markers across a large number of samples. Here we use such a method, called genotyping-by-sequencing (GBS), to produce a data set for genetic mapping in an F1 population of apples ( Malus × domestica ) segregating for skin color. We show that GBS produces a relatively large, but extremely sparse, genotype matrix: over 270,000 SNPs were discovered but most SNPs have too much missing data across samples to be useful for genetic mapping. After filtering for genotype quality and missing data, only 6% of the 85 million DNA sequence reads contributed to useful genotype calls. Despite this limitation, using existing software and a set of simple heuristics, we generated a final genotype matrix containing 3967 SNPs from 89 DNA samples from a single lane of Illumina HiSeq and used it to create a saturated genetic linkage map and to identify a known QTL underlying apple skin color. We therefore demonstrate that GBS is a cost-effective method for generating genome-wide SNP data suitable for genetic mapping in a highly diverse and heterozygous agricultural species. We anticipate future improvements to the GBS analysis pipeline presented here that will enhance the utility of next-generation DNA sequence data for the purposes of genetic mapping across diverse species." />
<meta name="og-title" property="og:title" content="Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1681" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Next-generation DNA sequencing (NGS) produces vast amounts of DNA sequence data, but it is not specifically designed to generate data suitable for genetic mapping. Recently developed DNA library preparation methods for NGS have helped solve this problem, however, by combining the use of reduced representation libraries with DNA sample barcoding to generate genome-wide genotype data from a common set of genetic markers across a large number of samples. Here we use such a method, called genotyping-by-sequencing (GBS), to produce a data set for genetic mapping in an F1 population of apples ( Malus × domestica ) segregating for skin color. We show that GBS produces a relatively large, but extremely sparse, genotype matrix: over 270,000 SNPs were discovered but most SNPs have too much missing data across samples to be useful for genetic mapping. After filtering for genotype quality and missing data, only 6% of the 85 million DNA sequence reads contributed to useful genotype calls. Despite this limitation, using existing software and a set of simple heuristics, we generated a final genotype matrix containing 3967 SNPs from 89 DNA samples from a single lane of Illumina HiSeq and used it to create a saturated genetic linkage map and to identify a known QTL underlying apple skin color. We therefore demonstrate that GBS is a cost-effective method for generating genome-wide SNP data suitable for genetic mapping in a highly diverse and heterozygous agricultural species. We anticipate future improvements to the GBS analysis pipeline presented here that will enhance the utility of next-generation DNA sequence data for the purposes of genetic mapping across diverse species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WI9PP5BL18SuwgY6kpv0kBWUeKEi_GGUnO7dPYOKIOY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459664","apath":"\/ggg\/4\/9\/1681.atom","pisa":"ggg;4\/9\/1681","ac":{"\/ggg\/4\/9\/1681.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1681.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1681"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1681"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1681.full":true,"\/content\/4\/9\/1681":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DPey3Ril1UQ8DoU028cEpZl6qq4Ihy4mWluoerHOTF4yrRypJBvkPaFunANoLL5AlEtAwlYRP1oED2WkUsdERo17QzHtyyE023JkYBEgUGd5Xz+tfQ1M5v0RdSa0z7nHJe6zvINiusjupdQ86QEyXZXO1olMttihN1Ha2DmtE5+iP\\\\\\\/\\\\\\\/gEGnsSL8HeoQ4Q9AV5pIlaZ6bGgTosWAyEUmLPj1RDyJSmy9WYW55cJzwMf0dhLyMwXdKofR8X3San4SFnW2GZKYT5tBIvBFdYrErceP2ed+J8LDIoCeHKG0OR7sZMhBjwb2fpgUekNG45uEDKkPoNtAwT55RXNasBvQ9yKz+xKeaR7Tuv5dm2pRSn5pc4O3xpVGIENBz3b1phGThI8f0ZkM9SJU7W+9S00SW9sGoWlkklStsDw\\\\\\\/20+I+uwIShs47f+Mra\\\\\\\/Z9IshlZsuYh6\\\\\\\/4IM7EGyYWgXNp65XKeuJIg1\\\\\\\/muOoLlmSKY\\\\\\\/8YIdTQ7e1Kg8hSmbNjA3+0KEh7+OX++GG53oRqFkELg1hn20d1sP0dGBwxfl7LMzv14BA1whXyzl6Tzj9U4YQ2k6ieDVRq+bbwnss3zf88C7ZCCr4t1qrR2LWdCPhCV9d9ZS9TPGOnu3t5HsXDwQJV8lmy7mRlEVDn4kyS4KQ9lUkew8qObLm5NIvO0o7yjkPDbgA8PiPDSWgtYlwQ8QYD0d1CtSIArYd3KW5p+WkEuMJnPx9FDsDTD\\\\\\\/uHHClkbdjjykrjQhRgiaQxSEnvl35fPMsLFChlaC6+Jo9ZKaVoyk3yKDkSY0\\\\\\\/BbcZNA4gvBc5WfYsZQORTlYT+unbHO+u3irTCmtKppdddRhZkNBL8WTM2KY\\\\\\\/AtBRexjLC1q+k8JTo0OHGmZJznoYCQesKdEiQpqHhRPEl0FyBsF0bjk6v1z9ufmI2+JYZGCkuHD8\\\\\\\/1rlSxx1NJBoPVEfzl1xdMI0LJhgCkzMd6TRijeMYQ+luaTjF+T60xWrdAp92tVT14TIv65R8GYHG\\\\\\\/AmVMkhHl3YrE6N0e6AQRhqTkBW7QhCurf7f5LYRa+PsgviaN2FhYivroxmJgFT7RfcO712sZnOwcZZ8RsOgnUscWS6ju8wJuKpEJpPvLj+4MCR8dbAFCojqfBvnXPXQYRObCHrtcwms5Yfbv1swMtGDGAsPvf2pP9e1uQRIUQpXklzvNvmEVqXLcH3pmTaPigombL8SIN9cD0NXDUxbbV+jHmKWjqRO3hKmU3O6XZ9UjRVmWjTYjRgrMjiOh26hL9GMJ6G7rw\\\\\\\/kbKrJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022etEKd79aA304ej40HLVXTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222e5b5f2c22954ccb5338463bd9bf2aef0012aad5a6247f2ef974a76b1c93b59c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mhXtPbq1FCCWIeJqJadFTmuTWfDcOQErq8yq+5BVsAjSF1S0AiScDoXRd2yRXUPvUZyZ2DlPruEXKhoBmDkro6IFXUgQ6oSTH7+rRN1lle6Q\\\\\\\/AzIZlQTFDOkvlawIakA4SsWTn7YEWkv7Venq06nIxHilg9Ytd4L9I4sMi4viIavd2PyiB2qpLlyNgECYNyTVDE+8f8mw3qo9gKEak4sFEAfvwGl4gSSywHtjZqBg2CEN8de3a5\\\\\\\/XUO\\\\\\\/EH3c2zjKOQQjUUsLkO2U81EIG8B2A1mNTflvUw7V3jB5woIjW8Z3zpjQjOup+rNxQDRimVl8xlcR++YmM\\\\\\\/tSLDpBATjYxL6K6eT0CIn2oL3c+MaL96fQUcYmCWu7MXKcsqXr6DfRjOw4xd4BQ1oVzEAmPBwNwl2LZVs6KgQchQ0VXte2xA403eqIP6je5KTVXAvP+8\\\\\\\/QPXZtgbjNLm3yfN+IFGTtfwIg5lZR3RgG1CIlc7dEHj+FK7Uq5IKkRA\\\\\\\/+dK5X\\\\\\\/zhHz9TH6MoDZ\\\\\\\/TH4hC\\\\\\\/miSLb7dNhWg9igq0v7eD5jtZogOrMbKxIbSRwsTQehMZKhLwjB7333sUbJ\\\\\\\/hmOq1mAzfVTttUqRG9sNKhTPJDk9RaqJH9wJAtqENdl1fWSOEQ3yYXDvLP1oeU4T\\\\\\\/sjdILS\\\\\\\/k73U1RPdfs54BdEMKvY+veQI\\\\\\\/dn1kH8iVjpXHJdtsNGKhzhhuEjM\\\\\\\/rjOaYCJgrdk1zqdPj8vKot\\\\\\\/JJ2tQ1948PshJjxHXCBWCZT4ZinvkVTp\\\\\\\/+rcdyD8UC\\\\\\\/HVs+0XjcCt8BwPJq+repny13CumtCesTQMyHBK6Zr5X3P4bmD1hnhLkV\\\\\\\/vXgyPbxFzwNYsZLAs6SlHAIxIxAxJM0WGUebJEC3b9aVrmt\\\\\\\/2yf9PjcyQ1QvKZjhDj4e5hr9FMsVoCe148HDzfupCESWxf6tAqKfZ5uJF+SLCPPbJRnVhZTi+DYuz2QmypZ9l5yWeTZohzduyzuh+o\\\\\\\/zIAfpZIwoNojIZf5+psX8q0skMQoM+ANo1qAtoixR4nGxBjsP\\\\\\\/Sjj8bvCyTzlnR97UQJRTtPeNw1ZLEz8xnQjUGQF92Z6feitBnd6Dpf8d0FuCMGnkUb9lmYGpK7qc49GMfFmqc7kTbfyUFSPXJvJj9BOPUZrk7nR6YJQlD8MvNZ+k8O6nuaKobmLffgHoSSSUoako1WeVRTwjfJz1FNVN0OYvPTD9bY1AAosR3B3Qvhc6LKxXd9px0NXRIB0ftQ7Tj0ECoBIToWEvJZPdW8WpnztCSoyIauHQjQpAL6Z08t+3j2pxhxpLYJ5w4CP3R8cQTbfkTHRvErjzKwjMqAAjz8jo+GaaEbFM2Tr6uqyPZasLcBeyn35D\\\\\\\/ynODZvA537sKKFNEloCt0RViw8JToC676fteC7cIUwreqMTW+Pp8e4Mwmu9XapVABjzkHV6pSnNG2j9fuaAwht6LpdS520o3CSrZkXUOaAXcz1OMTE4szM4wHvPcFn8cp7U8IoOxt8eTrIgEXffh8MtuSoDYvpXMxFaxH9Ln7oAuzNb3PcZJoSH739rTazfgLncAdCIV51piPtEIHsgcJB7on9l8cL4BhgF0hKIa2Wif3SrGy\\\\\\\/rUSyZPCgT9xfTpYSFqHRZuhkT2wy8NMfnq2\\\\\\\/Z2\\\\\\\/zaXu2QV2K3\\\\\\\/gLuuLK8dzcRh9HGJHQ33qcgzirh1tGUd2oTx9O1PZxEGU0Em+M+QoNrU6pZsSwb6AyuDejyeMiJkMmnFoBWV2wlq91TNMaa2NCgtMNSy9bKQp81UIvGNpZWynAmamG74ZXgxgpO2V4j2TXiiQnATZBrvJUPAywRBtQ11mx4gqbpQVpb+5evtDGmBJGXG2W7hCdpCwFMvkzWzuFsGS4gDLt76KMXMB+jLWS\\\\\\\/Jvip8kvDg+Q\\\\\\\/ncWPWB0lSwpZeLhDJMCGArmc9kPHXhC4mOWQGajF18x93kvonL4hWlq5cdPrnxnUvKEhMdlqDMQsTtBLwDF\\\\\\\/onXNKRs0YJ+v5gaF6UyxOM+K2Bx11MnqrUcRustLJU8nSBa44DGEcGBPWyS\\\\\\\/jm1KrYfioy2izNWcOxL10LlrrUfTkS285rgMwEt4Uk41wUiDQyI5fZpT2oMfP9T+nh9y4QOBNRJascaWsKKFqB50EEk3ccclqGVciqMZCrtuZqB\\\\\\\/HJjyiR57aPXi0SYp9EfF7Ppelt1aqskHtSgpXrU3b8C1+v\\\\\\\/WLBKcE1QItJ65ShH8ann+YOLl6rTdnGrfXXfu7kyBae2jZzR41DSQCdOv4BMXDz5UHtYKXow8ind5eIf52G1IdYOlhJycTT+ALYR56DsJgZgOuCXJZFGbh1t4h5VtTJjlhvgpx\\\\\\\/Pw\\\\\\\/YGPlh3LyP3+HATMZoUwx9EXfJEdY5mKWAdOX20Y2+9niwEKxWWGZ8hLhH+H9OjOgtc6bE2mrliMlm66E46yLUP8O9doeahg4m0bugshBuUnZN3fHfVnixwW6dh1j5YZGZC2zUSgYT7SXvYVBzDRkx4V3riIGlKVNxI4D8nuV2yi3xn9\\\\\\\/KNjhvlkCEHRKgqmjEMoqgn+brwVs3aDdcfGEeUTMc220vOUJF9ra2aRkHzm8Cay0X9ElcAG3RuQmy8M91edEQJHKXI2Zwjx8F+ibfB0voYV3OGbCzlyKgTnzptvUzxhkv8G24K3ff1VRtHzTcHxa46PvDIcjYbxnEaCzI3nRp8PpBqsWypl2WcQtBkiFUrzKkelw2Cc1dCvFq66neMhJRbGzAwcCS4u5DzNIcqFPO5wvi6JlUMaBrCR26PFULeAvQxAWHhfi\\\\\\\/prRGTX7Q8prrUOcmQ3tzJNaVfvXDzkj85acgXCvCwjq8brL8bYxE\\\\\\\/5yTU4KOZkKRfyKB4Jpbsp9q+c7ttnL0kxLy7ekQaESsCbKBH+Q\\\\\\\/7A+8jSFw\\\\\\\/N++BZxSWbNJD9fWQx8XgBZxG8FCkny3tt3P5IgCodLKWFzRf1f6\\\\\\\/V2hGXN6R1ri+EBRR4Urzwa9Mmi0Xf9K7lUZIrPd77\\\\\\\/WjkSFL3UGuSSawfohn8Sv\\\\\\\/vZRGyFNTOMt7LjmHqNl5CzGb4Mfcvmoz9X7wpC5PUBE\\\\\\\/PHpQcxfoAAZGplXEME+9aQq\\\\\\\/ss+9JaDxUd1kYg0GOA\\\\\\\/J\\\\\\\/zdRgM6ZQRUiNQqBQDnbeqV6rXQbnXEuVl1BuIpaJaJwPDogwVAOFtleysyE6Tzh7zbf64Xt\\\\\\\/Fv0nAACFWa+3J\\\\\\\/n2MwHtxeWNSUreSjlWjMmlElqI\\\\\\\/8hEsg4CRpKCL3+yobFiLM\\\\\\\/qANRGvqUb2SdvSziUlG8dmXI+xILJvGduQHXdVUE5wBQVDpJN165RdL6ZJxAERt6OXLR9tuTFwg0WSoQdVOiOHk6tDyT3dMBYQKRVsOtL7nASYaUX8GzczmMPL+iki2Sr+bhVtKHlZMXHc7CbksT7M\\\\\\\/9gSqC541Y6YSSzHvQebmScJr+k7inMsNhZtvGTpFn87acJUap\\\\\\\/n0yR\\\\\\\/Uo5lB1spszhRaIzaisg9tqz73ca7UBHlp3klil6j959ReE0mKIp8LYUqKzjLt9cSTE9XCyxlWBH5MtDEJRTaa4InQB2nHng7EjgcLYwFHoS1n91+OusUe6VNpKXQX2PD+Ij70revDDlrB6fS9m4nvO75vsMg2WOWyAfM3SoU0sqqGQ4xJjCbDaRsz2fR2jeY2b0dmrNIqUvp0zdqSUelgYXMM5gH0\\\\\\\/SWNEjqSbiY8ulns+FTBF0+vafhYfIvUVvSuxUxYTOZq2z8X1CuBuBHp4fh9RIHIQs5P50A58hHh3bmqyImRaH9YHgaRuXi6ilUwvqC2TF\\\\\\\/\\\\\\\/DdtJ6v9gbjjVyJ\\\\\\\/CEWYxO9KJB2Wd92OonRGbVNTeXPaqVsROBG\\\\\\\/KRjf6YyanXBO1lDJ1r19UQJQhzjEstCRaPNjy4OfMQCBnR47N4XzoA+zaNPvIFw1uPbugBptm7X0E3H8EnSqmm66c80m+G9rpE6h5NNhxUuniJ9vk+1Qg9TqT\\\\\\\/tF5F5r\\\\\\\/6n1Otp\\\\\\\/+al0h2UHYWr6pGFFVkW14TJEcZ2Fd0vv946VsWLrCI4nCJEctiz+kOu9gifD08LMo7aivolj+vkQitIyVDEabfTtG8MxkY+QOa\\\\\\\/gFs487XGW7YJ49\\\\\\\/85iHPKvcCL63kpV\\\\\\\/vJvkT4OZT2vGWmOo8pZ8a3Us1dqr\\\\\\\/heSotgaRfAKoFb4AWdWvvNg1JSxyX9IMg5qrSF20yiCrp1w0U6ZXtdjQhnWG4BqZQJUdGdas\\\\\\\/xfDJhyna5H3b+rl30jXPp4+KoFtwBeonUbZAkp5a0tEDyjUwU8VQLhuCTiIMzaBZtA27wJLFtAOqWgdOf\\\\\\\/gb1cCPVEwdhzrxiTJF\\\\\\\/A2iqrNOtoZmfGOgXIORIDUj3Cstwx3Ji+3MVYqmczWjsvvltgvNR7QSJBF1MwwKDs9hSA+M3z9FFKx4CDpBMbv0luGBS9JLe7K754y2p0QJrn+ZVQf0AVPi05CNbjE58Ev8jCLDfAq1h8iMbNhZ58BkEmW6eGQqiwPLUP0x0+9bH9LOcZ850acdPXzL46nDutjedZGWc5\\\\\\\/URKOWNOyZEZVKMg3eS2eRAHM\\\\\\\/wzDLf3vhapFXn32I3v4w0EMMTdsRXz81LSGJ2ugJFUw8X68L1ELDqDshGDe8jKuavtB4HBqoTG6ME0MfFK7I5wnf9Ufll09xziJ+O22mAiXE5MSghzzr+e\\\\\\\/FCJjcv\\\\\\\/IE1QOFLVBMJ2AQayd\\\\\\\/uK2vWcJMPcjBwN68G7KnSCNnC0+rq2uRTZ5LQsAMyduanNWzrIFs8vGrOf74n4\\\\\\\/hgCHoECD+Mkqvg32Te+5J03367zpVvQqsqjh7w5LSKS9HZx6oxkINviuDB2fVe4MK7rXtBU0Ro3qqagOWb6F0E5opiszO0UDyzZ9tV6aosjB03CiEPwqqJK2PKtgn3JVO0vqVy3gE6hYkQIFBIqLQolQQ4oTUOqzRBVT8hlkRo7cob1owu1Q1iDNw1wSBZUJRohNLeP5VKQ8iiO3CZQMRjRkVZYU9mddERy0T+SpSc3Y2bdIae8GJRZc1HNkksTtQl\\\\\\\/9rRuF3ZSiSwSLVofVhjqsrvLBxsw4LMk+TT07ac5hnYe0Z3yJWSxpdsGs3wf14OqIKG9l393PFlKrg7eFRuEAAGdmG3mwFXaDAQFTNzxJDxAnRXNgOKJIr1Ow4OaDrLzq3u7Eg\\\\\\\/53yMODUAiP08Pt44tWduB5qoL1iJ9Zz75Cqhh6gdyEA2jSKzaUhojrw4pp9bSbAaI5eKLuxs245wELPWYw+7CtWt+6vKXh0bgZTMyWKjX1VeIb4JYRr5LdFMqhmW1Z965NgXoyd41RONt9jK7FrJnR894d9RkoC+nyfI9EovsnHHhDIFj75g0yLExdb5IrAdmK5QCTbDgHPSz+bsFrHsLKJpO3J6GLPwbr1mOiP\\\\\\\/0NC2+XSUl5umPtASNO\\\\\\\/drZduCATy8uEhfOoStoxjnT4sqBionVfEPPpcLDriRHoo9AotJME1wHmKA0ay5l\\\\\\\/Q+zK2wg\\\\\\\/NuUMKsf3cLFU\\\\\\\/UIbtWDavLkNviawUKOqDi4bmI0v0vgkI1jmoDHi9jv5ak283+sMGYa8CzuxBEbCVp4EzDwcjL+4i62figb\\\\\\\/7hs+k43IAHmuH54ZHIfoQHoMMq9tiz\\\\\\\/tN\\\\\\\/Qr9x2xU0r1JTGkBAeq+HK4VKIGhpWL+QLDZJVShEjlkc1utD7W652zPJmuRAnOkqOI0dlHH8W40Gn4bERzcyE+gtDJruUNHU7H0NVl9qUfJmFJMR6HTSIAdKwPGV4TYenE8pUYmCMLW5EiHCNb6qmuU\\\\\\\/p5VkU624fQfAc0hXTcLnZ5twVjcktKAPvMyZ4ACCLU3BI5ciQcpVPzOldvzpWkrQ9USoNdrlVHSBdN8F+JOUDUO136zuRzr8aqpZPlWgDrApHRS4m4WdbJ8Q9+c3nt4wj7pnKbJ9Tpn0nauzP8cfV1lPp9sid7dBLBLwJ6Q+iDyCEZdNPGBh5AHyRqrp0TRgvK1xeI7BdHVCc48RN1IQtpyIKp0zjLp7RhlFX\\\\\\\/ILjJtYo1XFl8t0nRMUpg2he8zrNR+Ghja9HW9Ygmo2OuGGYeR74A\\\\\\\/Pc8K7LXU3AveK3VnnR+6gvxP7IUewOMILgiiQIy3VDjC1NOefZcHySxjnGbzzeaffUDhhJIBmDcmMGvn8LQagypAp3sCvBlSMLWYJi8urskV\\\\\\\/M5PsdKV61iliCL0LBTpv9rEHOiXMnTehtPiN\\\\\\\/YGyLu5tx+PBFhjpyTwpkloi9htYC81H2\\\\\\\/U6Ns76K6qWYOOp8WZE6nhd3+u3iEET2KobfehEGBSfrrNK+dwuRrwKV7AiZsqGNwoXAkNj1u6qH8BiTmwH8NGU9p8L8QVTuHGcdtQFWaq7JJjNMs57uaE\\\\\\\/0Z00n+UbFfMqeGMFWvXTkJu5zXoBvD+h3QoHxU43ZUuIlL56A8h1C3L7gn0Hyq0sDi\\\\\\\/9hwKY0+sa3SpQZByP1PfKqwG+rTWJSThgNoVPJWOZ5M\\\\\\\/aNZxxSo+Vn0CD7eikdwTsqipil2eP2tlPGlXNvYcRGD3BfdofF43PFAS8Z8nRm4lZdOliZBlVFAHqyE27zvWeg5yCQ9iSdTL\\\\\\\/qSSHpWNwMcDcEtGH0ZZp5fZESqQSqx51jT5WoATjMe6Ah2Nyh8qNWaaHXD9F3MpbhqN0f3VOSif3mHzBQMdE6xvV2H+2p4WPqOpRmlycsJBFpS+mX5Dt7hGSCT86vCo75VuaUg2oUpobR0ib6aFaOZ2vYCFcTGlqX3NqlET9EqmaMdHP6dPWEfwsZ5xFUEx4xUY+98Zdd6VUa4cGSRHfERmRZ8crfe6CVRdwdoUJqpX3byG8TV1P2N3dSdfqGuZZQeIYfcSTTlIEFV08SOZLpALughI1ql9PqY7ef+EPVqGQ8zyVLFvovj7aUzxv9YlSYZmlRUd\\\\\\\/A5pvNI17cPzA7+bR5iCQjpIXri\\\\\\\/Hh01RgmY+YPihP8slRyLiB25bABYUh\\\\\\\/s8Mb4WL7ojQx1jjUsCdCgpyGVahRGQkLPxsTN5XS9fvuDBhUmhkt\\\\\\\/LIJNtro3ghAwsGKLDdvGLyYzhyTTI3G5qAI+7\\\\\\\/nqLHNg5dE48+EpRtDMJq67wfBRzyOBq\\\\\\\/7IBWI7a2GIYdH7uzu4+zO0fAzDYAD1G7f8tMqIHxolt7gTsrINQQHG2erwyMKC4pERUovk773D9BtqSwzElJyRIn\\\\\\\/9ivb9DnCBppzk4dFCrbf97iMLVywWhtpJHRy6Kxw4GSdsRLo6FvOeVwS5rxt+Wqf52Fp4JXz5Uwd1M5+6NZM\\\\\\\/9zk0xXzP6a208eIAYUq+sTjXoVLUvg\\\\\\\/Khe\\\\\\\/Q2qF5Su7bkdjwiQzlS4uPyc4pbu9SygNB9ZJtiurOJ2If8ILfhhxrwYl68t7m9JAyAwq0Tg1gt5s78Z25nWCXN1KWXRCYsAJidER0j1Ow9OO6gcPrSmEX+0npQzil+9YmvbnwiKvXpjgHarhgXtFDkkbV\\\\\\\/eQx\\\\\\\/oqJBEQgYe8sqQcag2WcR4dqlxMK9HAAf9t0je9fK8uL5JkQiBFQmk0nfr7QzuewvbbE\\\\\\\/UtVh9EaLPlPhcAs\\\\\\\/daixFLV95MbqMIi\\\\\\\/UiHb3LP7DD83T7ieRI6RMkwkSiiGnkn408LKH6hGIEKZ\\\\\\\/IZm8IcsvAYrME4rk0u28l9ERpHqq0sm2U08JHQ9JZTUVkQbZDJgYgrAVc2lfpWLvniPcPzwyJ9EUSELQVVS3M6cG06vTUo2t\\\\\\\/rFqdSzStbMH3aNM5SVPtfUGcrQwBtYnOzHK7UGSqJ+gakCMo5U4\\\\\\\/2+GwFuF2p1s4\\\\\\\/VO48BzHH3H7UBmfp4+TAAbgJSXPj5F9tYd+6UCKeiUquO7R5SZ+f7pdU2zK+3euAn4p99AbjTl2WcZChe2tAd7n3udG4D4ul7LZSxVizVdLh+xDqvAy2sXG1JfTY7eG2kj\\\\\\\/ZkoZM4qZFAIrkUudqCVRuOw5q1jVNZYhShNB6EDtJqeApmAdjijdsYwUf1oRkl5TVhG90k4iaPYWIp0XtO04JRnjDGiI5TCcUnFwdITbluod87OdQLprKeZpKoTReODaBJMnggMGtwUWADp8ug75qENc7AWCUW5ofz6Gl\\\\\\\/gzmje4hR4h5O7T9C35VKdqY\\\\\\\/PF6KpbIhtI7QD97o6h0tG02au7Zgsa5gYJA8jBtjzZLM9S71kJH+RIJHrf7F6SsJisWH3+BOKhyeRZOdgHP6VaJ8Y9hyu\\\\\\\/5xsuLvG1OedUwDfW13iPTW3qcZGqMtrAqJJ3\\\\\\\/IlJjnZD3+DMVWFfNYFiK5JNLsSc5E15eypVOehiQ\\\\\\\/Nsps2hL4wMykvHzGO5PMfQYCFsfysebbB23c7aqns0F4sIcwCNHcMLlPqwGkGRCQo5KP1Ehfa90K9wRdzr04W9+U8lYBAd7dKKrpCbPRbwGbiXpLTl+FYdky0\\\\\\\/Ty\\\\\\\/H3xnyzR0yzB8oMEUw6ekWz\\\\\\\/6KVNQf\\\\\\\/OwhMmtSTU3WpWsTG5vg9E7zKY8PXcji7flBa5qMJjgI4ihNsSh2wTZGflwxsWc+6PaqUhKiu\\\\\\\/7tneJ4HREoeRYhGydZP8\\\\\\\/7ywmfaBk4RLwhw+Mf87q+iMWoDWME\\\\\\\/uamRoStUB0dFquvCP3Ju1RXmMPK2o3AULjC4BFBe1qz\\\\\\\/vHFw\\\\\\\/0yssuTGN4oB6af2cVznqP2NIvXnOZKk2tRiOzIseAuH6oeKB9QI9+qi+zi37Jkbr07p72h7hnT7ueztiiNM2ZfsKewvy5QoOQanoQLgs0CCDPZtqsQh1FqI17b1oHZWFnOmvlf8++cPdHwc9K56pHW7srn7KTXAccncWPXxGR9\\\\\\\/qVYCJKdaljkU6RsEZb9m5C9k4APcFHR++EfX+Db6H7x0\\\\\\\/oePVMq6gIG4bWTd086\\\\\\\/Zq+FuXXpZrobQpboqYnJnisqE\\\\\\\/3R1CMqJrXvthVD8Tk8s6N\\\\\\\/YdFdBLr6hTypTLA+w98jtdyyGsg93Pv5cs3ZE4yiWiptDWWz7sfZvJ6rs6Btkh+lGp5pr92xd6lihCtcoPfdU9tAeUUhCJzrVDPw39UBB0HVAjDk0X6YqhxgbXhgUVlZz9rylXX6hToKh82H+kW+QX19JGqBbMh9jTjFgTh4T1R6OpbghkddoRarQRSLM\\\\\\\/Bk195kYqor7sIHUzGEdp80LqxCYS72lc11xJKVhfTNrn6ujmJVA+IY7wpN1VkFh6Ky8jyvWuv41yyZO1+L6U387E66v7ngi1HcBJC2XjKf5LH6mMrlVPYOkVY6ZojP8UcS3TOYQ7wLgMUJVSu3Ko8SKRUnqf0Q1IKZgQFqQuDwnRl5QUJjWIFtlEQRlk+EqVZ73JDzq5wK8zbPZVDfBsbp\\\\\\\/5MXBouhSqif8onlfVKBYQnn0A5bd6y1mTdJi9Vw5aJZ3sWgvuPK42w2F3ZKGs1s3v4CH4meNSDLf3ojP8v5HnWxTNWhEleZzOLDETPhoRXWhKpCfsJtXdpOLu+zGkjIXNDDwIkZL0CF2w7p9ilqDOmm6iBHycD1yDXvFjRZ47HRKF3Ju3MDPUtFBlJyIJDiPUyBubULkk\\\\\\\/WGgQTrdMdhLlt69DJkmHT5ouRXJvIcujeJKhe+B1Fy3NmIsvbD1oPNpfQ8WO7za0neZybmlw5GJrh3Etr7kHZPxtoLFBCGBh383wEssr8+OGIFhMP\\\\\\\/3l7DsN24jvy0z\\\\\\\/nP5aOF2wU+Wc9dnteDUdTrn8H9B+ADdOAjUMxzb+mtO3ZHTA5nBHyTPegkqWcw1i5yPqAg4n1K2DC7FQjimimlw7hG4GSDy06MpJo0+VS6hAH6FmZBUPEpyktP7n5vMnzEfyxei8LxIDzj4+uhv35xVqkY2cXQMuyoCrKWpk8m9UsfgCButG+6xBaVVdPWwzt8871hEAk\\\\\\\/aImUGJustB6F42Iy6ZwcXbJtq2c1xmeCSuaAhfMUpAjzgumrYaZ5vtQ1bBP1wNvFkl3Q9H6NSD6uQAQ1jd11ptG73aXSkYtoOe0Hmg4LO8YI25kVeg77TM6NpZoDIgVgsmKO7Y+Ri+M7fQsHZCTTqRJeuyllggOfzsi4U3J+47nOqUEKueB3WsXEFdvDd52IuFutHmjgnaDhjs0PyqougfEFTA12uUikEGDMJ\\\\\\\/g4hUwNS0yecGblucSA6\\\\\\\/atPMLvGF6rLkZnCrXNDTXhOlXfO9Gp0bwwa\\\\\\\/9IgaqE5Wz7p5Pyk831n+I6YJ7AW3VoQ5ZAr+qcRPZ7LNqRpA3b0HfVDI7kq5DmAuLmbnZntFWVQ5fJufnpNH75CPs\\\\\\\/YsCH5ct7FCg7KEipG5Hea2JBEYn\\\\\\\/TiSQ8AkOetPOa3ztXHpT85nNk3k36j8Toz4BWkrlDlwRkYMDuPP\\\\\\\/VQ\\\\\\\/xTY9orZG1weHmZG1t\\\\\\\/xpK0wE9UJKIHX+QBXKaogcjnamVEWkgCHElNGQNpt75IYw95\\\\\\\/lwEjS3OuKg1tPprfCfSWMQehNJnPQMLfQN8ASfNQbg3AHwEyYo0ToOO\\\\\\\/IOwUTuZ5G10O2zPpv2LFB7I17\\\\\\\/E1PRp0X5pzZY21Imh0hUbnOzZfxg8lMkMrrHP0AItutQ3KrWetiWhvtO4xDI3KCvj4JbbRv2mJmTBkqRkwQ9HRN95c+FtKyitU8bkOzpwmuhPOolAwA8c67SklNi06weAzmYOeGoY8G2ZNdXSPPKxkI\\\\\\\/Z4spmgb8ri89dYaHlT0R3wM2U8O2xriOdlA9dzJPzKiOrvVFGWkIamBH8JWcZ9UukPoHW6SqDxIBVvvyOxnvl3ZapEO8M2ZsbVVj8XvSgwS\\\\\\\/UwsNdekzhJ9lOrOlHBNS23\\\\\\\/lDU7MXdGWv1M7LcoWxKM7ReULEWrs43vuOhpTEzRqXUeJA+HObipY7FxX+IjUl12vbyqSygS3gRlKTDlQY7OV8xyG6EQXcx9PO4mWAi9NwXOLvKHD+Ns36\\\\\\\/P\\\\\\\/Yw4LfEkxBkHAa5WW5IKmpj7y3+c7J3v6hdpGCI+4oh1zjjEyXZUZ1TvCRFCrpZGBhTYAhp9+4CZvc\\\\\\\/i3pn\\\\\\\/1WbiVv32xnuFa2gGsbqlR6eaGfYHw\\\\\\\/Y3ufBgebfAmMYxa+HemyHtit02Qw\\\\\\\/+sH66r3FzL2aRuTmWqE6MnhBjca5fXTqhAydrorDL8ThSwqgbkS+Wxp5v3k3DZbV9RgBp\\\\\\\/z8ITMChHtEzacsdAsMy9drPnJUHEVD3QgcHJk8wBap9WB+F2CFsmT4mLRR2IayKDgiUMszFgGc9Onba+wALd3Bkl5fyVFClTFx4AUl+EUIKcpp3o9xUjIIYUB7G\\\\\\\/aKOZmfE86Z\\\\\\\/cLkEzt52DBTD+6mUlsKEkaqM0QrcR3GiJyLDxGFl3S427DcfHDKCJBDT7MNzZfCNj8q09jUYXTMFdrXrOYbKTuxrNVQPZK6QOMzNTrUgmoxCwEplQHuayFWtB0z2x\\\\\\\/meFPqSLwjfNf6PpLhxlCJ\\\\\\\/H\\\\\\\/Ni0cJwLNyYzWGJnH\\\\\\\/szcPFs4Inh+H4ygQvoFCvtFS+DkTtMOwVh6UJ6jmk1qBEk4e8vv3eq9QyExutbt\\\\\\\/S6ZpfHaY0VAE6MjvsH5jysQhxrMUyo\\\\\\\/1FAeyXBypD2ZdWj6k3xxmHbA2dhAeXf1ymp3+bGOP1VCXAJg7v4nZ3riuV81WKopYiE84uCeMkkvQHGpMpGc0dTCfyVE6Ro5\\\\\\\/shUyWo3SuWncZ49zPQA4iPaHPKTg0IKrmvHfXhOU8O8JibFziYsfpu\\\\\\\/Hd2RZ+nU+zU6QiT85sCMC5JFgmU8ulYw9oY+Q3QhwISqUPeNFtHTEEkW4pqUIGt\\\\\\\/UbW8uRGSRVN36Lc7169ZfzqFgm8kwfIihMA9WiYDmrFh1X1XQjHEVxWOSP5soNB0FLK2Cl+STVAATEjd1yTag+nOSoGRYIvKXcB11tDgvDvN3i99ChZRGuc9jHJnNrdYkxVK3wFh8Dq53VG6N7eQEj7gws8HnQWgexrmGQQLiP\\\\\\\/qvJUsRYGLhM2IixGFeFwx+hUUhlPEWAp7wFShgBrvsxU9WkIGr5\\\\\\\/6\\\\\\\/G4pbZZIgXtVlJLpz4ILketRG9fg4c43o6ERKVO0s82JJU\\\\\\\/LCthIiDUIrzwVaBTqASqmSjebZLNh+1la76fZ5mFUoQpWSKHplp3n27+k3nDikySRuN9NoQG88ivhAjPjnN58zNxACpIEri9dspyUdLgGgmqFjfXyC49WW96tDamltbT4JLq+86luJCSZtzAtpCbvapGFXTlSLngFMkFhqZ7mmykCkCOX8A+TMykkZ1bfZP6mQprPI\\\\\\\/3L8u301Z3G7DGHigfKgB3Sqlanw\\\\\\\/XVWklBHzR9HrMBzOWjGuT6s2BlKk4lRzO0QC8IVIxy9VNWkqe1sy+ne2\\\\\\\/\\\\\\\/WU83+yxXZNcTw2YLhyshkoMJoK72OGAC\\\\\\\/lgvup6DUunFbQ\\\\\\\/zYu4CLfaCOY9Fo4SwkB9ZyLwcg5AnIcVD9TDiN32OQXMPR+T1JU94Amr+8udArmvzXtFUQZ+BODd+eq0nvjRAlcslcMS+wG2dMDTRwYv2eEoy5wx4+dHfOC7M0xIBOTZ\\\\\\\/rB7SUII5pGnYplFemxhWla6GvSwTJEgDzaPsdfzCWrX2H2AvHO7fvWP9f8sovyDBbe4PeUVprkr7fLJ0JpfWi\\\\\\\/vkfalDsWlg06DvsCA8J6DdwM0vliY+FL\\\\\\\/GpMid8sOXCl+k+U6GzP4n+wImwA7eMrj3JiVizBVC2HIkWeKDP3ud7020feoVwPUG\\\\\\\/GI5NWsO4ebpUI6JfSZzrLja\\\\\\\/dUJEvC6Js8jrIQdmmDL0Qs\\\\\\\/LqqeyvwNp05n4SiAtQvB2K\\\\\\\/sK8pGSkWs7Up2OSrWlUArOtN5Adi3X20bBxtUn4BnW52R9BmTU0APSGvSoKqKgI2xxE7+NAoTdkfmqkJG7rMu0lzBYlfBO94XxGU\\\\\\\/qbj9GFTiVzk+s4X13\\\\\\\/I722VSS2KVzX2dzqUA+HY62rT0KYuAiek+7N+rRNdvuStRsMtZyUnBOmBs421RsexpKSioVQU+zQiQ7jlXx5G9JmnxyEgNI6TIbKl1FNnEmYg\\\\\\\/TZK0Sh+F4wfH55F0hcshNlgfXhLvOwY42OyIUvsPyUh46dXgLQFpFnXox0XBgpey8kzePe1impu0zFJ9OJUV9R5nfGNz9ekP6HX2zn8Sj2W55KdgWrILK0q1LfQbDGa\\\\\\\/5N3aRN3xyDZzxVfVMlTWtrXg4oxsJxpDn36FuhvpIx+SGkuli372OfsNH4wAr+7tdoPp7dn3\\\\\\\/UGSxn8BJvWtIR8gQfVC4TkxeC2Z86RGH0VawoK1SxZG0d29cQ0GovzKKJ2wfGnBdIyqE1BJ0uzqpUHz\\\\\\\/QAV1ihgwdC2nIXAVSY25SNTgUYlZjJiSQgr1YRZpPzbWAeQ0HPIhDUj0HRGvlF+IJg9I+TrqSLYOmzcTw93Pw9d2AwmLESrI+z1UIwXtzR4qjZ5qk1pG\\\\\\\/GFeexdxyoeDBZA2J0\\\\\\\/BAplU2ceGKFvdlwvt\\\\\\\/pkudVrOrqFrdYcMRHR\\\\\\\/k8RCVBfzPsRxlZHBtZFciXs1zmTi2HE05A2mmSzWdmszJZCGIMwrjoYHlSnd0EP+1AFJMHgoHljavioAzhcvABrUQrH6qZ+lh\\\\\\\/HC5AS1GSZNda8VCiT+6dHymssd7XmOSIWlCk+fDppd+xKwbV+RPV2Z1O6VuMfix6A3OJk1GAHZKr0\\\\\\\/lfKV4+2Hi2+BQhkDcI9XZ8eDUtQJzOFYm80Uz9FZLpYcmsGqbJKkNwp2gc3A8XC9DpLkSSZ3Qkm+zD8CJdN3fXhbYbi\\\\\\\/GdYjYnXoTDyFLDZuuSf1L6B9CvRFvu39sZ+7LXIvKGyKfEP3b5lE+FONm9PKl4AMFpUnaVnLaAw0GFdgE9KUSoMtMXJ49yBwXraeqIWL5yANgZLDwG1PlDS5VBYD8pA5AFoVa2JOshZJxW1jp1OIjagmchFg85LiT3w37J5jBA1n\\\\\\\/ZoEs4oBTDkcXkh8NFJ+Ahrh2WgMonAoKMf1SkPWbPq7N6SxY35yWPGNQrwC+6ra7MfAT\\\\\\\/u3weKDL7ZsTljYKmXmmqxM1yNF7WPaDT3+0tKOWm6lz5\\\\\\\/68r7co0tdrNgw4VOmIqkbW8xX2XaY7TGqhJCcnSM1VvDtPL906GDpFOHgz2MLkhfeFUgvv0fIbGdsGSwbeRicQu4aZYSgIr71Fa7I8YrMirl\\\\\\\/hC4LEZQjoPly0FCveEfDisw9phs6Y+Loj4pDIx15RmxBVkW8GtLX7E+Nns0w9H\\\\\\\/j8Zi8WUQWJ7+JLNKbmuExFmM\\\\\\\/0EWfqoe248JsZLBlv38mXv4u8tzdH3QhLmjqq0EjYXxFWbDmOOptFksqbAu4FoYKTECsd762reb7coNYW02PMeCiCmAHycLDnoN0rSP3in7NQar22iAyx3buD9rjdyWph2iLHHvM\\\\\\\/NH4\\\\\\\/r9d6mxOcXVYPTHTOiGoNiWT2G8C4kINKQRU8A3LL4TZsR1mKK1+2GJmRmtAhtFPtTrQfdyZlTqWSgREDZs1\\\\\\\/iOQOp3id+hnTE8f6DrJBFY9Xwnat0KNG0S2WyO3BbuAXMAjs\\\\\\\/BSOUkDkUzbfV9JY2beiWTx0MdNqmA0kxeAD4lgUbSu8Cjm4Xsh09BZ3VWxoSdHJo\\\\\\\/xM25aZldUbZpWz6G\\\\\\\/k\\\\\\\/uTTzYP9Cv8DPWpbjWWhRanWRS83\\\\\\\/Uea0mDCMfkqw+FeMvC3w+elJlnBxhjxJPDsiF1cJ+0N6pNzQP33sO7mY\\\\\\\/g9vHNoRFMBfc32PC1BiX6c4KzTa4HIQlcxm1\\\\\\\/XH8NRnrH+DVWpK0K2RCfqU2NLjwXPAuZKbO9QhopkTkmCFQhYrm1r4rWbBf60N4PKkwNOWKaRLc3B+0qDILZs4Yb1qXEbKZufV7UFQ3zxwX\\\\\\\/r7S4N3At2eqxELd7yHkd\\\\\\\/kBA7zZGXxCDfbC4BoLyc6wC2uXD202H78iEfCk5ncmN4hqhWY4bRAhs5f\\\\\\\/nbKt5MCezxZsomRjwSaqWyGulT4Rz4qWXDLRqqn8heu\\\\\\\/UjVZWtDaGOjkJyHX7Nh0c6jgAEeCiEd2MYUfX8Gqq7wtUKdUa8Jp683w2ZPClcDQiEt5B6+IYIx3r0+UQYLy1NdygAbTZiX7RXgrVjd2Eho3pZQiXThjktcqGmVYKXoAWVqDppsAr0GyZ5XhVm\\\\\\\/vTsPsoRxgTg89pofBt06K4xrYIGaDVwxrOvJxTWEQuVmjoHGjCke4e\\\\\\\/jeFms2JfuOMgg3G2v+W6lR+pAVFcPN5J4S5Ey2B3g4UM7wu8nXCxYkVPtzsAZ4jGvC1yl7U\\\\\\\/gwRL7NoYjeeYCHykdc9h68tYesTQP+Mf\\\\\\\/z1QjlO7dYDUbVy2apKZZGDq4vYg2JJB9wMFm3uuEBNHipjpdVUqe2aE1s5yJO0n6RTSgSGgcnM05ds0yvLNeb+MBq0veJxifNXO21WQTR1f9GzGjyH3Uv8z+O\\\\\\\/KPqV3GfCeE+map7D971J1BXQCEc+xFnEJu\\\\\\\/hamAfTdpej10YBs9\\\\\\\/WKaZAVdMAGnA\\\\\\\/2DLyYPhD3lyr\\\\\\\/75e\\\\\\\/WsewZI6Kc6DLd0ZDKqBDq4gLH6XxjJUHf8Q6rwKnfIoY79ZwFAWqOsn2UdlzrVFJIxaQGvfc8uc4+afKUkalO1B7DsBzdzr\\\\\\\/a44KsSRVw8lBZb7CuziuOdl9dZzfsO2hSM6ce3EQzrFRkfSZgQ0MWRh1tayBaAJq5e7OCXNpGHtStAaNL2Yx14Dp2ZzSI0wa3+umjAtTgulhGOVFoupF1VWhqyNL5fUrIV38J1cI+Z4GqcQm9WVcL12+VPZqcjfZ9hqdJkig\\\\\\\/vjkMvaVanVhY\\\\\\\/VwUgJFgZR6mNbYH7CJQTmmeyVGoTSSPbYGwbDKvfFUdFZN7PhG+tPHFnWqEsVGNIl3TiJitBR+5Nyx4B8PMvN5y9gRIKlXjaEmA6PYDmzNme7\\\\\\\/T5bK5z9sWztXjzw9JIWo15gz7T9wvbc\\\\\\\/dHVRvsOA4ouwV9UZHad0hDGifRjY1EkiCe9r4KN+Ocix94MHma9OAbiTjTyxUlIY7Nx65r2qKovKBO3SkuzRXoaTqRPiivTKnyysxN5waChVVIfFGLfQLl6l2NIH0qGkYxZqFE5m+6AD\\\\\\\/nFMeMkxDPvq5hSG3g\\\\\\\/WTZlL52Dz34duGiY0XtQbngiy33CXDJQ3+KEPSuofjNLpFI22QH\\\\\\\/YF\\\\\\\/L9asG68Bb23jv\\\\\\\/fsxxBcoLgwBBm\\\\\\\/4AUteGk2BwjTuEx2Yko8l6SIxWFPNoHbi8V+kc4eobVPCRlVJppNjSQiXwlFu4UG3w+1gAsSkjo3eCPTeYxFSzg\\\\\\\/PcPQKO+nFy9bUMAkMLWPAWHVHERvNG9B5v9E8T0hAURq1kk3FuQWGM9Vgyx2TWrhI74RwqV8rEDSwQAmoRNKpbA2xDB983jsBYFS27uv+BX2TVHQX2KFRYZF6J3pYGaVlfFzPwwqiUF15B7FKqSkOKmgT8XRmwsB8vbB31TVhjS8\\\\\\\/IrACbqZ\\\\\\\/ScpnA4FrRdfBDn9WCbEE4Pcu0ZpBPawCnyScYWYRjvaCcdSaEdt8p5HXKnlLcDiChs8ZgDhBXklfx6Hlji8Bt\\\\\\\/+CmmM8PmDaSCuSmUxrR1RKiymPaNYVipp5CWecAsmdXICpHkR3\\\\\\\/vQlVMPsAB6T+hA6esSr6kfJlPkF1qURFd5cuLWyptc3LSZ2B\\\\\\\/Sktmn3sxbiqdqiNDy9kbLzPV0R5qk1AU0cOl9aNmoXEQTEuGyYfMB+AtQJkH99zJJAfZCLbdiiyO2r\\\\\\\/Gxb1EYGwuDiqw3eaqrR4+2r8Ucd7vt79xSXM3UYMNX9zqoUI+MPH3eibAYkONC2OiE1t0AqtVPAWWKBCYYiDhvhzdJkXkFFc+BU+72Z2EG+3lLoor19jglDr3aHa2DVCLPo3FetBy0gOJQbEzKJlU9hnUcOrQOtf2aWZT3hjMlGUZIOyf7k9K6QlnLnhS6uEhvUFpzA0pMhTauQebzAiifVjBuusuMERNhxuBj1xzUoIPJzlC\\\\\\\/0G6qUCOBSyzLNqkFCENfsymFtjGaVuo5nxpu8gYq0xeeguJ\\\\\\\/JqhT74DtvlOHCdhw0GMebNXXi0mhOmpi8QaLMxh+PMwqLzEVhdxI6jgTwpwYlld\\\\\\\/DwiQMi6MSG5oqzS0GKyIEfraW6sdRwBaJ1HPB4S4orubFp46A0B+MOFPIX7gL8FxZ6XX\\\\\\\/NT6eaW0nptt\\\\\\\/0uY8LK8yAsJYXHTVyQsgXQHIyipzv87SQKUKxlgq8gyBbj5zuRGMGeYXm6ljeLGkLrCUCH3IDsIBJvDTfqY1IwD2exf78kNmgL6fnMgU6qAQZtIGIEdw0+Uo4W2w2Au\\\\\\\/0DS8joZ49VWsNhSxO3eJoa190cH7fIcpBKdgaBbKJn7xregreTOdyCQ0WNlwlboCwCFAPoOiDz9jI1u1bYdadM+7ltZ0R38QF\\\\\\\/wpD8PtvBeSWg\\\\\\\/lz8vjeUi83bOybHbnpcu43MCBQMeMv+hSmwcikrhzliOP+EREakoOFM5VL\\\\\\\/uGrVGsgg6QjhC3jCYcS6zKtvCzAD53Fdh\\\\\\\/N7NhqfdbqZ\\\\\\\/9Pv5ufWo0azoSX\\\\\\\/s1w59whMty1hnTTVAjD8guFLQhychZ1vBG1sgQt+fqC1En\\\\\\\/2rE6Xxa3OjEzP3lPcRaK3kQNG7ZbFvHVwcbruWP\\\\\\\/IOAvTWp1uGGXv3fG\\\\\\\/iIk786Xg38VVouK7bEYfyLoUQ85Jb30x2LRWCEGdXa8U1WTxg+4WyRczccYle5pGZblakxc+c52OWr4LofIYUkszsbzXZaK6hG1KhNN85iIOYY70WlYvqMBfwpue4zjQPl25apvUyxELxhLH4ZK2v1ft3eI15VyD9wuOi4h2LYNCCYdGFPukjlUZqY0Qe6oE0Iz8pdnh1eSw0FW\\\\\\\/IaAYB8dqyuVWViZNpn972P2qzJnsR\\\\\\\/TKgWeT1ldkW\\\\\\\/mEh5SsErnhttvbtykQJUiRe\\\\\\\/sETy9SAbmwfgbREjN7mGai4luP+nbPDVOjPVwGdY2mNhDHmyKEzmpay+71jaUrdxiDOsu6bTrg0KfbtaL199E99XUDBroGiPvmmceezZ8mwWD+i9Q99yZqU3fu\\\\\\\/z9v7oOZpUK3mHfCdatzZ32HgN1YOO7pIizKniakctZui35iM2FRvtqptuzrpCB9YcaQ8G\\\\\\\/hi4TUnBVjUkhFdMqWobOljVUG2mXYE5j7RosgoAQhbh5CpqTeyH9riwLSoCEG2zTrTjI2Kqibfrh7ID87wNJxhpUhcA3i+8qYh3ktHNU+o7SeWZ8eAOLktgM1M42ENqImwZDt5C0CLZGPyPCyzOEO1Vd\\\\\\\/7irDr7vSTsgje4fkKiqj7F64T51sJlkKecpYvRl3v3Rf+FoadieNuLAxEBmQFR5irIjVa1I74Qzv04ae08QCfzisnI6bXuRjDI\\\\\\\/V3Qneqid309wcyul+bAt8q4zkAl8zwOxqAPBAPFJcOh662gcqynxbogM9uYQwNp7tY7YPV8ZgOvHGwyXPQjPxJVWJzaMRZSdHqgW3gMgt5+DXbRA8KrVQeol\\\\\\\/z73Me4S7VtrBv4qb+nzAK\\\\\\\/s022jsvDSNjm4zr9a2NBHfz4gnhQmpiv5fQv+WV1++ip4TqLrsx7o+FHQU1EJsKnUurBsYqnycrTWQaW95WCzF+mPVimmNw0hvjbDdUv1rIerohaVmBzYpVC07fHrFY+ZU5EHlHaey9FTGeryzXOYpiwARsujP84LHR8v42J0W5Sp6BP8W7IMXovRRhLXE6mzJI9mr3fgmq9Ur9HU4jjN9xB0oNcAagCfStmm3+L\\\\\\\/gudXAmWEn3JHqSslfYqvhF8Zt1DCHZRKnHlhdlEuO\\\\\\\/XUvGgAqvDzYgYZPkn8W2pMoEtSp9Tyl9FBMTt1zW5xXSCcmPDmLSHRNgw3p\\\\\\\/FLxWdeyr2CRjZPRcbJMZfpUu+mZSIUk5Pscp\\\\\\\/AXPgF4lO4WWJT2ieF9IuRliygd27P5xfBzyzUI\\\\\\\/uzQADllG9GKA0jVQAnZE9MwNXjQYtfpWIajOqPLDRV4XzoM9LQmQV8DnBaMNc6XkxqiAH6YSyzV\\\\\\\/1qNOQTnAg++4ugU6xbFS\\\\\\\/U185NSJ9t01PuFKZv\\\\\\\/JeMdDwrgE8QqbIms4CjEr2YXUWarnpzxggYdpsDNqT2rnSNbW+YKSCdb20sh6ShfUcX0QTKaILZ185ZwItTFckZVf2xhmLK1D7UBTALXyghc3wvESM5XV1y\\\\\\\/NHOZLwft+64GuNkHuSgI62t0Uu+i0LVUn\\\\\\\/3LukiwF3yJh5ZXhF5xuOlFYOHgalSqZjeIUWaerb\\\\\\\/uHC3EgmVEvl\\\\\\\/FeQsn93WioJfYCWVIqxoMaQHGyDVVeHFdOKHo6JNnFssfYzALj+TY6ePjvPvFKvZmwlY\\\\\\\/EEjwEPLTi+Di3XW7OU5d9+GNBeI15QNPquRoomVgQuKdW5nRNyf3LXV5fwbzcOg1EDGjNPxlKbwjm1n5e2vVFoGejvkbE3TZMsotmQHY2zzcJicD5FUgovU2jVM8qhAZ+tFYJEad68MOY+fI2UHWAEH8mPvEDRi4f6KFo5qA4asJfNZwKGn9RJcrotKa6nmO\\\\\\\/FIY622EU4lmBV3Dnjpdk6vGSgkNVGZPNaQNO6u30yuoGy0wUIywbIzfRLXJPi6DRe71kXfIQC90XtVAHh8FWHpvmcI+oVh4hh6DgjNsGxsjqWJrg9Pn6FU+40WJ3ViwoRWlsqfBeSuPc\\\\\\\/WQZFeQtiYDfc+AXmbhqTbi4I6FlMaTLlep1GvM7XyhpvBcZ9GdEhUhGyLS1dMjsjeySuWdhOiPPr9Zjf\\\\\\\/MNjOZxCNmH8PTct\\\\\\\/221mujBFzBR3GOA2\\\\\\\/q8TicoyoXIP+GuoKOY6Qkvtd21iyD4EJdiwzHM61vwKuBljtGyhD\\\\\\\/OxT\\\\\\\/6ZudlqT55hADffT8tsSBF3B5xHX3sc9n5fNmv7UV\\\\\\\/o\\\\\\\/2b2mI7tOTN0rF9HXMbHofEM6BASV\\\\\\\/WzaErYVg6TFTqME\\\\\\\/MbdcJZjcCvd+m2Wx0GuZLM8FMRcSO63ozK788vAgnRA2uVJOlPnHctkSU7lmWLrjjuMOH22rH\\\\\\\/nrdYDJt6pjE5A+D\\\\\\\/E2aAcxVq8BFI0wjI2PidOf7S1ikMxZmjU1kjFxPioKg12HWNBqXv+PVxDgagsxIA1jklZ1Rv2HC+vz2bvdkqFKijEnXFPRWucYXWbNnwz1A9\\\\\\\/5lTckxwhPjdvYB0CZlQ3+SrQrrT66syeQcGIbeno+F4H8hBRewBb6jqBvSz1F+Ge9RFpUbyCWquMiAdtc2dqP66ltWWv970AAfXeYks+D6sE9FmEM6qj3RBgeZh7Wn9CT0wWMNVe3WDRKY26G2S3XHyIdjMbT4vio4yRywWHg2sEc6U+u4\\\\\\\/3ij2BdPeT9HVXY0AnIlzFhF+OQ9OmKTh3fu6J9nMXi6F2HvW8VEEC1\\\\\\\/MAdU+iBAGFxAALJsKEgYN+EBjI71OqC5mXSt8fZXZOxRVIf1+e+efxXtqwBO6I0zD+rLJpUS8k1v24FPCGpS4U9UhZ8JEdeNjehKUTb8EKi7PjbWa\\\\\\\/joi7qxKLusWN1ggYBG66QpD7P39CyPRHJGWs4B\\\\\\\/U1VKlbnbObXXhQQiZiH26Kd9RoyoEUlkrrh5d7gm93rPAgfAiTqD72xxUwmiTkY+P0Ls0xU7QVqzWe9F6F\\\\\\\/zNx2ZGKev56yJkk1GGapf37kN3zVAi1fWs1a49c5g8Xu4ELKohalY\\\\\\\/8iOy8yIFjRo\\\\\\\/cXJdQDKNEOr8Cy8yQhagiiszepTWqNLOQfzAcN9LKmzEd6tSRNfRlICugJy7UQe\\\\\\\/d+6p9Z7hJGzwL811iUDyfK10WKhdWfbaPceuTN6ALEF3k4t8UbYrOk5FjWE\\\\\\\/K75y60lADYF3K4favUH1n5VgDpbA9vxoqvvTf0dAVgNaqbs9qmPlCKiKVkWYydXWfqAOC07nP3DE8tJNj8wWCPsCiA1Rwx3e4yJnT9+R78NKg4tz0kh2MxJpvrlT1aFAP70rFbxuEzsybYHFBdeRqkv+trUPKGmzgyGyfqPNUf5NB53gr8t5fSfy\\\\\\\/gQSr0EPyTJEAI0cOPSsh50j5U+vrkbKd100RgMPG60b481j2MuIWqdh6YjEuWBwkjLaOBuRIKXQjGeJasPADops7FyjKV94LTGq\\\\\\\/vwHqd35Kk4f0gjy\\\\\\\/WSe6rdNCiMy64LWAYmzouBmjxgzkvIJJvR+jAIbwkM5sXkVyQZmGxZoKmXrmVZ7uL4MZHfoOuAi1Skja+cYvXNS4VBo5DM4iKQzp67\\\\\\\/19N3aCwpaTZYrOIeXw9DySluy3YYYJBFYJAucMF\\\\\\\/x3DyfcolY6RPjf2TUb5tZuFjxj9udAfYD6Wx0CTnpnBCrSw5aZ1fvETc0DyC8M6TFX81urYqzUX6vi9bsjk2U3\\\\\\\/+DtzkLCdk70PGOqzy5ZNdPYABJHhqmI+voQ6U52+THO7JZJJUNTuF5gAhP7wPBtXj2g7gwuWmfDIbclBlqfhFCFDAasjhECtYon8yY1UOW\\\\\\\/0izR1qvzW5hqPAnt4cIDXM29X1P66KFf89H+7eiyar7V4Fc97jJi0fXuaESzV0rGHIa4hIf7sRjygibD\\\\\\\/jb9qS7CY6bT46bNmt+ArkYKxzyfeEmi7uk0MGBt5dVrzczQlL2eyfLjQbnOl7dyjR0vVxyUC0ZBCto8DxOeZKu\\\\\\\/DcmC2QCpqvCF7GkzNXmrpWZuH9kyWLlDTJTkCzW5xO1dGovr9qRbrq7TMHGNbqwyiF7S8e4ju+b9fCdHgOmr1LLDopjDSop9GKlibZrh1gLBBGxuZYKmfp9F9zmqMpEv7BsHZTOvxZ3pTsQCXGDfVgr+frFgsLEn0tWy\\\\\\\/7I+ZoGc5aMyRVx1ILCoEpu4O6wzO9q9SLUNGea7FUqkp1fqJHZb8hTxIC5V5KOwIL18zvdDa1z4dFn30tT7x96p2np153bM1QMHUytdyff8gZVeh0HVHbU\\\\\\\/TgU3iXkjjFTysLbixQ4n8C7WTK1l\\\\\\\/JLRJ46qKN+XgSUoboA+PvMKffS3RSPqt0yjV837EygkdI+abuupQXEdH0+Q4rofGyufp90+6TrFL9nHfkXncua8N+SOn5bJIrG+OlPB4cwyU2ntRmf4XV6toQHv9ce8WO8lY9nNaYyeFH3tPJis+allmLareS4lo74JwJHB3e4DFvIkCOBoprsZaWt11VWTrJPJ9GC91OUkHGjFc4vbP16UUrfmT4Kxk61dPvW9FLMMC6XjEN57l\\\\\\\/W5HhqRZ58+3FHljAmACT9OvrjNQR\\\\\\\/fvb+nxAsraQYzoTxToA\\\\\\\/eJMiiFCMCcyDi2Zl6CgMeYcy\\\\\\\/Vpc+zyGjUOGAfQXk8RVUJB3uBYyrmozqhHld+LCMGM\\\\\\\/Pj2aH1hvCoD4EykbemwHh7fVWCgqbYDFgkTAz6bnHjrpvn322ltQcGw++feBKCAnE+21fPYaciVDfHGInBao9T+hJrciAiUHK4yA0rfYrAND4SvOawmfI1l+0GgDIsocr+jAj79I81OEEgflf7ywImtZ14gqF1no4tjUQp7l8IJNPDgdwut4Sm34Sgp2WuqOedK+IdgTzWah\\\\\\\/O1wenbPcQX5M6bYCRltqKy8e0h9vSdJ52xZu1vsAR2rV5TzIHDX+1avz6XQ+53Uz0Vat2aGwTE6okrGiEnCmOfc\\\\\\\/skV\\\\\\\/Vkwkx\\\\\\\/HL1ESrcPkwK2eOa3UIUqF2OaZoD+xduNnaTpXboGUw0Z\\\\\\\/a3clLoHOswSfFNckN9a5bZwu3cgh7OeXhPOsBYFaS8JPoheCkuLkZsQ6F2HWUeeHLH6g\\\\\\\/bwDZGm8OuywCyXaJLOMc3cfaFMS+DkBxthQsSIfqZgviw5xcS6e1R1otK2ZynJJzucuvMQgJh7YMtRXQoMsQiiQ1CjaSCz2MSHLp6TTjzqz27q2CxwFSQm4bFnTpOYXxu6BEJzOt+hnXOWCJfBca49rwwgAbcH6u6pcfKH\\\\\\\/hnxmS3j7NkFVSWK6\\\\\\\/f1ECqgwgc552B23UWbWATv+56QBeMCGVrY2sd+S+AYxRSAOwSbl1zpGu1mD9aq2g8Lp0p\\\\\\\/aBmaB1xY+qKx\\\\\\\/WkbTUFxXKG+ROR03nA7djYZcx+hULBLxgXekh2b24YNScG\\\\\\\/4EWf+yw8TpBseCm\\\\\\\/MnKREx8jlUHT\\\\\\\/cJicUOu1G1Ui517wr8JedKwNXP24BV3iT\\\\\\\/HXD1XgE7jhmX9Qj26XoPHWAmR5WFz0lygdbs9LBMDxQZhODP0ODsEzz+lUOKUCl8q4NdVK\\\\\\\/1K6cdV+OLaN5wie\\\\\\\/o50c\\\\\\\/bdPzrkyTJeqgVSBiEm2pg8A0jdGsChk38mOMZKlxIm9Ay0JR5kuZzWD5+kCUkwNYwiz\\\\\\\/XZmrZIxkVY4z+MHpRbblmhwedyIWiL84gD4sMj2PzWS4EpJZJm793qZL9SnQ5s8DliRcNNw9Wl3mawvCpkiE0Ul8lcbR0GSLs4Ue3oXS\\\\\\\/1\\\\\\\/jXX5WjVtxPA\\\\\\\/FzJBSrBWOn\\\\\\\/P96Q8I\\\\\\\/ZWsi8ArJs82BBJS7DsXTT4zM+0HUq4HxrRC4bH6d3APNqKz8\\\\\\\/Z9wNJqjWadFUxggwGzaP\\\\\\\/brMtOM2EL+siR6WQGJabk+0RYDktEUAYJbBo0prpeSp5Th4laxLgBa49vu2zQHyWJIWB0905vfAGTm8IZvmgeJjzrTj4fdy4H175bTD16PtPyNXtQ+Br0gwwCVsvjBrk\\\\\\\/9qgdfxxwf4FoKsUWgJXWHwcARTHuQlPVykdOj30Y2A3r73zPTSXLYJqyz8UWQVBv6CBeIzEaM1xk4AqOnyJCwyjryZHUfwxhLar+dmf4lgEJwooYeq6U+NWlbO9mALBqxNbzbskRmZk9j8uoYr6vGinhaVcPcbVTJ3L+Y1max7FigWzfGc48fiIXvoHCz1wAOHTti98i56qbBu9QlShnn33cEeKaKpJjSUDod1IcvJPqtY5VQzZUo4zoqHSp858QMkuQD9dwdUHAKyI4rMzauURioo4aKqy97gGIeyvAI+gm\\\\\\\/qdKqYGFvvNtpn7xxryjnxficwavre+SWvdTxOM3BAp3y+BKlYyyth93ZNRbHZHJE6PRShFUNDogJqFwA8FHGKQIBgxpqF4w0mv7dI+jFbWtcx\\\\\\\/oX58bMXsYlChfDHyxOwPxOEARj\\\\\\\/6Xk8CUyFdclNNGMJnegvh\\\\\\\/vRBGBWbI1U9OqYMRax6L\\\\\\\/Ns5hQtUWlADMQ+pITD\\\\\\\/dlwWLW2t53KAW0tSZzgH7CGSGC3lkyV7QkZXI8OG1DiCv0lRyphvrrhtGvMveN5tzVmUT+xkw+0Gq77wwz1FFsj1katPm6sYL7JuGqEXh\\\\\\\/IYL197p\\\\\\\/hhyCmiKhoDMhdH5Ve7Y4J3L1RsY\\\\\\\/qQrHm0QG1T43tCRQLqYlz7sVISy6Fi7zqMyhufk39oFCyT1ZzAFSOMXPrY2tpvIpCwFbFyQbj1w3fK+r\\\\\\\/Q6toawOzeQ2HFCUm47IKpJZGSFlOxYFpA5D0v9e7T17sNtkli9ymVOXoWn2V5sEQ2Drh0mBBKLoIVjR6\\\\\\\/3McdICB7O0e6VxR\\\\\\\/tka2OQ5\\\\\\\/xEVPPbT188O+n+XVZ5UfTbMczkYfZ7hhNHJ7onXdEDqn82hBdh5kg5OFXahNYxiSeOTqw2UHAsjaTuZ1FhQdSGdNx5tfL4WdPGcB5+NvKJlAiB51LnpJF8Jw7J0BNr\\\\\\\/WY9xGdOa6ltvJE2qMUL1VA19ZhewBQ3LJYYbgSGoUovV2jLPpmqKiFQ4fb1TZ85ntncCWawBzydSquIky2dI1G8io9dRW6bUQ+63Fo9iNmB\\\\\\\/PalgnV5234xbl4jXNvrjJfRTX7dQdEuSsEQMXvHPCRFnJFWuTojR4OHvuayIfYHJUn9lp2nHZueqNvTBgQ1ZHuezAmnxAdQ+PNivVlxVJtPtJcqTdTyvgtpB0aYhrXE2nkEPniIJAZVTm3nZKi8g6a3PufZ264bU5Teg54ZFb\\\\\\\/RCglubvkrd2eNom1QKh0NwOup\\\\\\\/5BlZLnr4+v\\\\\\\/zqMSLJBUSUVRto1pMbcuxcc0gL4BhI2CO9XKrZL+50b5uqSfXJSJImVgKHSlTn6XsP519Bzqt3KokcUWDe\\\\\\\/HvO4UAO0uzJKcW1rekHfZ++l\\\\\\\/kcKIVxR0RULRZeMwvLcr2fv\\\\\\\/MdscJzsAFMA4epQ\\\\\\\/\\\\\\\/3zCbxoAV\\\\\\\/RAARiV9D6PUcWNnRPp2tFeHDOy5FbBCf6kUcgNlkuYECeGNnsEQPRF9rqY5HEm2wdfHyA+j6k2nV6SuU764j0KIqsvg+0aT7McegXFLGrgmAlKlmxIOPkiQuwNxW9Xx2rwMtFBTXMESl30H5O45cpo5+hLd8IwJKLGHRgGtIz5eSCtWPzY3NeeB4ZcRFiHbKnBVU\\\\\\\/76jcm4YpRUkdfY3N8L+hchk60O5iYrRWf7cl9F1sgIeX+5o02SXXElsHFEg0WdwV1jAQ5xTiPQWP0M\\\\\\\/OzVa7RQGCYEn7vI5\\\\\\\/YgEMCipocPI3+gG\\\\\\\/+sT3QDvMGSWPJhVDL6IpumLE5QqqK\\\\\\\/16t3aL\\\\\\\/MxT5xAzM4uM2SA4qv9OggmBSmiS5uZmPaeLzuaOdxgbrnSlLbxEhar\\\\\\\/bO3jEl2nURXUfTG3nI7X3UmJOAh+pT7PqhAExqmyykntfRxKBJneaSE9Z1Rsha+muukKEmCZMSL+y8UPAer4FvK7quM0BhGHPm4RJE55jqHJ3mnH+MxhuMLUFWntsP8aPitL13pMhMg72KiDRoEXe36g3eowu9KrNoh9Joj5HMvKRERVdNpivbK7HZcF8RLdcHhPntk3m\\\\\\\/gXjpX+cDxcctRGzuQtzouhP7DzC3IlA5TbxTGSH+v5iPbLpHysjfZZdDVfM2Q1XEVrBWZQXe9TDZR93O1xO48eBphlON+UrrTpxcirKVgv9TtdEsjEdRZmR\\\\\\\/Trg\\\\\\\/JsREUAUOgqz24Oxvrqvw0e0gquKAyQZ6PaPQBPf0+nCzocLAxL4k86Sttw2ALcDNRjJa7oR\\\\\\\/qFSyAjpOy4ZJTKS7Xg10R\\\\\\\/jQqUQpPXLe12134zQNW6qVF9LRtQUNAX187dPstylSgBAndpdiD+1ye+8cic3k+gcaaFoLjNecql\\\\\\\/\\\\\\\/iep8ov5aLys9IpbVzbp+\\\\\\\/WrPmAuRHZedHD+R+zqLxOGbVGLCGa2QBRy29aliTJwUU71LK6gk7baqC9\\\\\\\/+\\\\\\\/soR1qpI215gRPqjE8vU27Bi0sC24NJTLFLsK0A+loYF4jhILCf5o8P8eHECU3ijBbvOXiT8KG1XFdkY0TQjLy25SSqVCHXm2ABrSF6XK0HyH9+WrTTJM15gNWctEEgNlqpKxVnd5lwaNlz0TNKLYOYnJHSNlbKNd+qt0HbCbxmULYvnltuHNs088KXz7C4DQJV1w+uImXYHXtLAd9WQnnnOqQPkXcvxXapuTTCV4pFLDrG3\\\\\\\/gCcp0b4OyZ7USUZVjup5KwOVdQIaBA8iylbwwA3JyWHUQWxFb\\\\\\\/Hjtv8IqQWONyhAteMGOPvrNBNMEXIBlJntKl16LyDCRbaYD0hiG50NWJQClOUcbh4xgrIshT3owuydd0Wse8LHVfrl6VYsMgeJQDyPE8hNmbvcQKGIZQsrk+zJJcV+pN9DFMSTNAHoO9TUaKZxkRP3L8UIJMFdLk5n5bwYcbdkDJXkuO0qLZkXjkrQ+Y5ok4oH3zH87HcZxa6Xb0L+eLzPmq6P5fpiFJQ5JTHfhhP4i3BS3g0Hds1IKT3UZ0NqkNOzxV1\\\\\\\/ILPk\\\\\\\/E6rGQ86wJUEtuXQCMF4W043VExhsjaBZ74NjV76tFmCXl\\\\\\\/IBt5ql+3lfuCz6YckRRfvLVQF4tkxYfpiD8h5CrB4sn0ZqmC3FHaBpbM4KkBlWi9Ns+lq+nn483\\\\\\\/Pwtt3WPf1pqa\\\\\\\/m1D3q0INh2bkJDvKKWUm4sVk+4XxqdTRmgHDKwtYAkcI3OiVkreDPE+PmQxKbpfIxo7iO8uTPnR1AdpQ03Bo+YsPkDwiYZoJvaKKTo5BE+71\\\\\\\/F89ltt0HFHWByR65dTmTh6uEs3DcidfdKOfs4uKtwilS0ZYWZECjJdwuMyKDtOmRHgiLKzCIjhfHbyXKaH1NRzdIFm2JnlSk\\\\\\\/uDwMlDHhumkoYerPVPvzUR3ncfdkTpqDWv9gQXBQRpnS\\\\\\\/2QTcparWgHGzACOrwN9c0l+OY3mh\\\\\\\/bP8WMoPvz9kmwMlAhkpjKp8qGOkpKUbX22mZwY\\\\\\\/\\\\\\\/FP+hBem8OGLU5B+bbTJL1Gsr6f454LuBMXQHpHUGWboOht8VhnFrMkDZIa6tkgi0ExQCLCQS0xI1NHrAh\\\\\\\/BMuxii5Twhdx04YZCCs0RFoSG3yc1uKk2L6oDMCrOd4\\\\\\\/z53F2oWfWLGfoPU2DL7+ocFOWFOPWbPOZaYGLM8Y3aXqCxeXYuVs+uoTS83peLNkW75WkyE2X51RLeMylEz0s3VLcgBW6sto+oapv0bR\\\\\\\/JB+lsNBLrTZ0Uubt3oZPAxhe2F1wTk\\\\\\\/4nQEfiJ1h85ApejnE8pR3\\\\\\\/jIo0dk8biTweGB5FU872NzlVOqy7RRTppRNuVIpl9mcDeFFnn2iu2DKtTRAwrRznAWbD7SIb+PRr8+CTqr0ZTHLuxkF6j2LNp4ftZik\\\\\\\/Y5WWGs9URxQVVHMOp+Oz1ROQVfa6EVHTLeoTDe7QNVx0V6STLPUHDvsCDuf\\\\\\\/wjarw1n5IPUzwMwmwPBTV\\\\\\\/q8ZfsVDyojHFNIflYkNDmRdwPgIwUWkTjHbj0fWX7WDXYtzeA5jl068g0Pbu9fpeM5\\\\\\\/OOU74XgkV+I+8sOENAVg8Va8T8owURGFiaE84tfrYdu0aM44dDMMB+4KMF2uDJgFZwiPmFvn2MMXBX+quiG4ah+RE0CjlJX6xutMyzMPkUdz1Op7rY7I8juPyBylDM2tj54+Wo9J8gSbIHIaD6id5IxvHUQl62qoZ5dyfy1gEuHJ49ucVu2iT7WVf7jqg1UviZjVUGrXRGMr3o6J7bas1Ipf\\\\\\\/UGFF6bvkd8ujh5zYPFl7CHw4Njc93ZImp2Ay08w8kYMyGnwLMtaowSV+8e\\\\\\\/7WYSCPXqDzyAxX5jCn1UkjadsDIRE2Vs6jKpoBDFG7eUAtqHfPi3dkqBjaYTcjjBbojO1hAcrty46lsG8Vd8A7yuhC+VZxF1WFHGyRLENv8xTC5+Gg0EKWE++aUB7F15hHI93+VQ2ehinJKzS\\\\\\\/66vcLDSniftxcPXZpA8HECsT5S0jRggU\\\\\\\/yxturAnRgEIjg+e+Y2JUXMMHD7zC2U4k7RJ8lyA0obS8UCH5vpMrCNNnSCBvSGYfEKsRacGrOKh5nzdNymQFDWz2J1q4r22LmtuUv9RWYeJlh3EuIFEgnsklVuzZDkL7zqalKazhQIohUjKofnp8TDRJnodTzdvfbg\\\\\\\/EmsJUoXofqLXeixmf6w\\\\\\\/0imvlhEJjLVzRZ2C\\\\\\\/+ak3IcjqBXa2WH\\\\\\\/1sWYkNRXAA0t8YRef+36hzNshVc0vhg\\\\\\\/J2v2Hln6SFR+gbYg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022H0fEJch2hhKgomPsyi03pw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a92e30df2b344e7b1c9df6558649b6a5d05695eaab62cfd3f9842d7536430609\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Fv9kap\\\\\\\/ay5uiYL5mPWvsxIgR4y9WmY5EDfWjlmNpFnQ+b24Mpp74WMNdB6MKulxRJgBb7O3LFEQt+kSjjYpDEN89X7P3zsy\\\\\\\/C9NRjn1RRHVxVH8\\\\\\\/TtioB\\\\\\\/bknE9k5OJUUnP14\\\\\\\/OGVD2oTfs1kYRKKhd3CM42gEftczyoJELfezxOFulWwaSX1UCudFFdLhx8oTgDd3vWeGXPDqMN0L\\\\\\\/IgKjQYG9MgSBQIv1oP0k8+1LYSpVaVORdp2NYNNI3hZeeanPHm5quUk\\\\\\\/FTya+0EIcQVQsdJbsEqBqY8nHIAK7h0j4mCSEQddOxs9ECISJe7\\\\\\\/6xfOJo0FL4cz\\\\\\\/3m7YZ0yUwzyx81UgfFViLuR1egMyq3aI2Bb1ZsUd9\\\\\\\/q0uYg10jLahGoFT6MBOwEZab2fKrUdWEaFL5MrYfqRNfymVyuTKFK3Ex4LG1gqF4hk+HKiU8pls4\\\\\\\/mzJWOypiJ21w2AwobC1kkNYwpzCeaHn22q8zxcZswfHTelAVf5GhfE9qLV2gjOeRJkPIQprxi6bg2SARYcVtL8tPrniKrUzHGC4IwllgY8yQodJq4mYncgVSYJHTeoi0C9NQE6ajL5+syuowTBInv7P5LbKXQir45IyLiIA4DgTTTXFUNyM8ot2xNJ\\\\\\\/o96TVtbrOwfZVDw07Rd+oQVEf7iri0sgNYF9CLMyO8scF2V9YpIBW\\\\\\\/Wei8bd+IWA6ybB9PPKjGgShTYTdrTyKhMYrZQDfhxsmpg7EFXaWJqohkU6Q5k5ukL1oaz90qRWQoCQLVYmbhKD0OEHm3wzL2g+5SvSJR+I29PLmuyXJs0UvNih3hLwCS0opnPPd+QnONN19jkCRnkMPhL4\\\\\\\/qB40qr42pKnUJzPPj0Duz3fDBurzXdsG2KgOyn4Zb3ArScOBQlKtvMzouIxZDL1LwdMeo21jISl1a7Hd4iBe5tj6QPnn6JkFn2Um73ovqsv4ewpq1h1V9b3DaxIZDRO+FrkCFqoozNNiAFp44nE4sl3eEzW9\\\\\\\/9WXxOudYUkUbrb1QrWSewsayUfTc5lx8tkWj+dEPI8vvcPVztsVIzknVZn5HNxyp1\\\\\\\/gLAisZXjol5NPqSnFauH0cPkn3DAWiJKBWfbSoNZIgmnKzh1Cb35WjOO8MoCcCE32vDVq3ccWHkTyAjXBXSb\\\\\\\/uyUmjsQewNSrvPUNZGTaMtXtPPDjxeXqk\\\\\\\/KLyDkwvw4GVGEG2l55xmuDxhhCvULERNQlen2bCjn4nqm479\\\\\\\/Z0UqOrdfX6WVeB+oHkEUnCzSTt2nl2LCWOWWnOGkFrLBNCxH6PrmV5mpLfa5yanalpPw2TJtl6sT1TFj0mircul9yJutPmhiMhoomvR61vv9cxJ+9TS0KzDJdxWvgdVkohFhrYeOlbREqSVDD0nmFTBaxypAL5VreOvRK9bs\\\\\\\/hQQQw8M0r8YxNbOz0oIkPR0YO3HybjYXvZS4OVo3wuoseONMtko07HGDDFOk71AAI1hlh9dEXs9M0ZNloXoCDG2DmdHkxiInSATSRuxAmTzJbGDPwJKRCRF6INiDqZzVGzBd3fp4M+wSNO09Mg+SHx9OFpaUdvo\\\\\\\/SEdjIh\\\\\\\/cHlTtvYS7RfZ9ccIdEjmt4rEFnpM8ObPlrk4cN8xFJK0t60EyeBFLe0g\\\\\\\/B4i8BKHyCWzDMsKiybuU4pRg9MHjSgFTNbX7iuO2pPm6nvDMgvebb+ZYdbH1TCu6Dnu9\\\\\\\/I93xk8gZ\\\\\\\/1w0w87FLjiJ3zckvr3PgkVqoQyvU9byFom0iKnHzDwAIlnB3U+ZFgAOBnmyexp4px4gMoOMQpFnLTTtaoJr+IJVfobE4tPFt2OhuPa34DBo1N2xDOC4SnTcZ1SsKa9lSg3DT5DIc8U+OLg5g7fWrzva2bSbywc1dsND\\\\\\\/duNt\\\\\\\/aEc4FAGwJEJd\\\\\\\/gKvqEJi+pbbqgt4eAHAKCjaf8pfo5IDFHr9mTatfWnew70jPu9Yv2Ef809jDHNu2kVpTciFbmpsWWXdKuj1zNlnFLpfvMdXSa8sBG2RFWlUPk7A8hGQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vzeLpEZVxar5yTgWdQZtXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002251948d7b5b5071d81930db5f54c087df041e290ea079cf96654e0ad642ce4499\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459664 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1681" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j4Z5snuwE3eo0owFq4jw2yryvYNsY6RB30rv96cdsjE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1681" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6ojQ2Es86TDEmsMNjoEBJIJLcmqRTad-gFE5QrFkHoU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1671" title="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1689" title="Whole-Genome Sequence of the C57L/J Mouse Inbred Strain" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459664" id="node459664" data-pisa="ggg;4/9/1681" data-pisa-master="ggg;g3.114.011023" data-apath="/ggg/4/9/1681.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kyle M. Gardner</span>, <span class="highwire-citation-author" data-delta="1">Patrick Brown</span>, <span class="highwire-citation-author" data-delta="2">Thomas F. Cooke</span>, <span class="highwire-citation-author" data-delta="3">Scott Cann</span>, <span class="highwire-citation-author" data-delta="4">Fabrizio Costa</span>, <span class="highwire-citation-author" data-delta="5">Carlos Bustamante</span>, <span class="highwire-citation-author" data-delta="6">Riccardo Velasco</span>, <span class="highwire-citation-author" data-delta="7">Michela Troggio</span> and <span class="highwire-citation-author" data-delta="8">Sean Myles</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1681-1687;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011023 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459664" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kyle M. Gardner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BM.%2BGardner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gardner%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BM.%2BGardner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois, Urbana, Illinois</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas F. Cooke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BF.%2BCooke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooke%20TF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BF.%2BCooke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Scott Cann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BCann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BCann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fabrizio Costa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabrizio%2BCosta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costa%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFabrizio%2BCosta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carlos Bustamante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BBustamante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bustamante%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BBustamante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Riccardo Velasco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riccardo%2BVelasco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Velasco%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiccardo%2BVelasco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michela Troggio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michela%2BTroggio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Troggio%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichela%2BTroggio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sean Myles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BMyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASean%2BMyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sean.myles@dal.ca" class="" data-icon-position="" data-hide-link-title="0">sean.myles@dal.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1681" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459664" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459664/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1681.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459664" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459664/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1681.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459664" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459664/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1681.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459664" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459664/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1681.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459664" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459664/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Next-generation DNA sequencing (NGS) produces vast amounts of DNA sequence data, but it is not specifically designed to generate data suitable for genetic mapping. Recently developed DNA library preparation methods for NGS have helped solve this problem, however, by combining the use of reduced representation libraries with DNA sample barcoding to generate genome-wide genotype data from a common set of genetic markers across a large number of samples. Here we use such a method, called genotyping-by-sequencing (GBS), to produce a data set for genetic mapping in an F1 population of apples (<em>Malus</em> × <em>domestica</em>) segregating for skin color. We show that GBS produces a relatively large, but extremely sparse, genotype matrix: over 270,000 SNPs were discovered but most SNPs have too much missing data across samples to be useful for genetic mapping. After filtering for genotype quality and missing data, only 6% of the 85 million DNA sequence reads contributed to useful genotype calls. Despite this limitation, using existing software and a set of simple heuristics, we generated a final genotype matrix containing 3967 SNPs from 89 DNA samples from a single lane of Illumina HiSeq and used it to create a saturated genetic linkage map and to identify a known QTL underlying apple skin color. We therefore demonstrate that GBS is a cost-effective method for generating genome-wide SNP data suitable for genetic mapping in a highly diverse and heterozygous agricultural species. We anticipate future improvements to the GBS analysis pipeline presented here that will enhance the utility of next-generation DNA sequence data for the purposes of genetic mapping across diverse species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/next-generation-dna-sequencing" class="hw-term hw-article-keyword hw-article-keyword-next-generation-dna-sequencing" rel="nofollow">next-generation DNA sequencing</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/apple" class="hw-term hw-article-keyword hw-article-keyword-apple" rel="nofollow">apple</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/malus" class="hw-term hw-article-keyword hw-article-keyword-malus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Malus</em></a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li></ul><p id="p-5">The introduction of new high-throughput DNA sequencing technologies has dramatically reduced sequencing costs and increased the pace of genomics research. One of the primary goals of genomics research is to establish relationships between genotypes and phenotypes. In agriculture, such genotype–phenotype associations form the basis of genomics-assisted breeding programs that aim to accelerate the breeding of improved varieties. Long-lived woody perennials are expensive to breed using traditional methods and therefore stand to benefit more from the genomics revolution than most other agricultural species. Offspring from breeding programs can be genetically screened and discarded at the seedling stage without incurring the enormous expense of growing them for years to fruit-bearing maturity for evaluation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bus <em>et al.</em> 2000</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kellerhals <em>et al.</em> 2000</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dirlewanger <em>et al.</em> 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">di Gaspero and Cattonaro 2010</a>).</p><p id="p-6">Most forms of genetic mapping require the collection of genome-wide polymorphism data, where genotypes from a set of common loci are obtained from a set of samples. Although next-generation DNA sequencing technologies produce vast amounts of DNA sequence data, they were not designed to generate these types of genotype data. Recently, however, next-generation sequencing technologies have been coupled with reduced representation libraries and DNA barcoding to simultaneously identify and genotype a common set of polymorphic loci across a set of samples in a single experiment. The two most common of these methods include genotyping-by-sequencing (GBS) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>) and RADseq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>), and the present study focuses on GBS. In addition to the low per-sample cost, there are several benefits to using sequence-based genotyping methods over microarray-based technologies (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Myles 2013</a>). For example, polymorphism discovery and genotyping are completed in a single step, which not only saves time but also reduces the ascertainment bias inherent in the process of developing genotyping microarrays. Moreover, as reference genomes, alignment methods, and genotype calling algorithms improve, raw sequence data collected today will become more valuable in the future because improved methods will enable more information to be extracted from the original raw files.</p><p id="p-7">Despite difficulties in experimental design, due to self incompatibility and high heterozygosity, there is a wide variety of apple genetic maps constructed from bi-parental crosses. Most of these linkage maps have been built with low-throughput genetic markers such as microsatellites (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Celton <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fernández-Fernández <em>et al.</em> 2012</a>) and AFLPs (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liebhard <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kenis and Keulemans 2005</a>), resulting in relatively low marker density across assembled linkage groups. Recently, there has been a shift toward the high throughput identification of single nucleotide polymorphisms (SNPs) in apple spurred on by decreasing DNA sequencing costs and the availability of an apple (<em>Malus</em> × <em>domestica</em>, cultivar "Golden Delicious") reference genome (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Velasco <em>et al.</em> 2010</a>). <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chagne <em>et al.</em> (2012a)</a> detail the creation of a SNP genotyping microarray that assays 8000 SNPs discovered from low coverage sequencing of 27 cultivars. To date, the apple 8K SNP array has been used to create saturated linkage maps in bi-parental cross populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Antanaviciute <em>et al.</em> 2012</a>) and to perform genomic selection (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2012</a>) and genome-wide association (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kumar <em>et al.</em> 2013</a>) in diverse breeding material. Although SNP arrays are widely used, the high levels of polymorphism in many agricultural species like apples often result in unreliable or useless genotype calls because of highly variable probe–sequence hybridization ( <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Miller <em>et al.</em> 2013</a>). In addition, the ascertainment bias inherent in the design of SNP genotyping microarrays results in only a small fraction of the queried loci being polymorphic in any given bi-parental cross (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Micheletti <em>et al.</em> 2011</a>). For example, only approximately one-third of the SNPs on the apple 8K SNP array were observed to be polymorphic in a "Royal Gala" × "Granny Smith" segregating population (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chagne <em>et al.</em> 2012a</a>).</p><p id="p-8">It is evident that GBS offers several advantages over competing technologies and is quickly becoming the genotyping method of choice in many agricultural systems (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Poland and Rife 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Myles 2013</a>). For example, GBS has been recently used for a variety of applications including saturating an existing genetic map in rice (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> 2013</a>), creating high-density genetic maps in wheat and barley (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Poland <em>et al.</em> 2012b</a>), performing genomic selection in wheat (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Poland <em>et al.</em> 2012a</a>), ordering of a draft genome sequence in barley (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Consortium 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mascher <em>et al.</em> 2013a</a>), and characterizing germplasm diversity in maize and switchgrass (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lu <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Romay <em>et al.</em> 2013</a>). Almost all GBS studies to date have focused on inbred lines, because genotype calling in highly heterozygous crops using next-generation DNA sequence data requires more data and is far more complicated. The present study addresses this issue by presenting a pipeline for GBS SNP calling in apples and follows recently published work on GBS workflows developed for other heterozygous crops like grape (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barba <em>et al.</em> 2014</a>) and raspberry (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ward <em>et al.</em> 2013</a>). Using a single lane of Illumina HiSeq data, we identified a robust set of SNPs and used them to generate a saturated genetic linkage map of the apple genome and map a major QTL for apple skin color in an F1 population.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Population description and phenotyping</h3><p id="p-9">The "Golden Delicious × Scarlet Spur" population investigated here is planted at the experimental orchard of the Foundation Edmund Mach (FEM) in San Michele all’Adige, Italy. Each individual progeny is represented by a single tree grafted on M9 rootstock and planted in 2003. The population has been grown and maintained following standard agronomical practice for fruit thinning, canopy pruning, chemical fertilization, and disease control. Due to the large variation in ripening time among progeny, phenotyping was repeated three times during the harvesting season. Skin color was scored as a binary trait: trees had apples with either yellow/green skin (like Golden Delicious) or red skin (like Scarlet Spur).</p></div><div id="sec-3" class="subsection"><h3>GBS library construction</h3><p id="p-10">GBS libraries were constructed using the two-enzyme modification of the original GBS protocol (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Poland <em>et al.</em> 2012b</a>). DNA was extracted using commercial extraction kits. Restriction/ligation reactions were performed in 96-well plates using 500 ng of DNA from each individual, digestion with <em>Hin</em>dIII-HF and <em>Msp</em>I (New England Biolabs, Ipswich, MA), and 0.1 μM and 10 μM of A1 and A2 adapters per well, respectively. Libraries were pooled, size-selected on a 1% agarose gel, column-cleaned using a PCR purification kit (Qiagen, Valencia, CA), and amplified for 12 cycles using Phusion DNA polymerase (NEB). Average fragment size was estimated on a Bioanalyzer 2100 (Agilent, Santa Clara, CA) using a DNA1000 chip following a second column-cleaning, and library quantification was performed using PicoGreen (Invitrogen, Carlsbad, CA). Pooled libraries were adjusted to 10 nmol and sequenced with 100-bp, single-end reads on the HiSeq2000 (Illumina, San Diego, CA).</p></div><div id="sec-4" class="subsection"><h3>SNP calling</h3><p id="p-11">We created a custom bioinformatics pipeline using custom Python scripts and existing software to process raw GBS sequence data from a single lane of an Illumina HiSeq sequencer into SNP genotype tables (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/011023SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/FigureS1.pdf">Figure S1</a>). DNA barcode deconvolution and basic sequence quality filtering was performed using a custom Python program (barcode_splitter.py). This program splits the raw Illumina fastq file into 96 separate fastq files based on the barcode sequences associated with each sample while filtering out reads containing any ambiguous bases in the barcodes or restriction site remnants immediately after the barcode sequence. All sequences successfully passing these basic filters were then scanned for the presence of an additional restriction site remnant and, if present, the read was trimmed accordingly. Reads were also trimmed if the sequence contained the common (A2) GBS adapter, indicating that the genomic fragment sequenced was less than 90 to 100 bp in size.</p><p id="p-12">The separate fastq files for each DNA sample were then aligned to version 1.0 of the apple reference genome (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Velasco <em>et al.</em> 2010</a>) using bwa (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>), allowing a maximum of 4% sequence mismatch. Alignments were converted to the SAM format, then merged and sorted into one master binary alignment file (BAM format) with SAMtools 0.1.18. (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>). SNP calling was performed using the genome analysis toolkit (GATK; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>) on the BAM file using a minimal set of filters that required a called SNP to have a locus quality score of at least 30 given a prior probability of heterozygosity of 0.01.</p></div><div id="sec-5" class="subsection"><h3>SNP filtering and segregation analysis</h3><p id="p-13">SNP calls were filtered for quality by restricting the marker set to biallelic SNPs, requiring genotype calls at each SNP to have a depth of coverage of six reads in each sample, implementing a minor allele frequency threshold of ≥0.2 and limiting missing genotype data to a maximum of 20% per SNP. All filtering was performed using vcftools 0.1.10 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Danecek <em>et al.</em> 2011</a>) and the final SNP genotype tables were output into PLINK format (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Purcell <em>et al.</em> 2007</a>). The final SNP table contained 3967 SNPs and all of these SNPs were used to map apple color using a simple chi-squared test (see below). The construction of the genetic map, however, required further filtering based on the segregation pattern of the genotypes in parents and offspring.</p><p id="p-14">Linkage mapping in apple F1 populations is referred to as a double pseudo-testcross and only three genotype combinations in the parental lines are informative for the construction of a genetic map: when one parent is heterozygous and the other is homozygous (<em>i.e.</em>, either AA × AB or AB × AA) and when both parents are heterozygous (<em>i.e.</em>, AB × AB). Only SNPs following these segregation patterns in the parents were retained. Each SNP was subsequently tested for the expected segregation ratios in the F1 progeny: heterozygous in Golden Delicious (1:1); heterozygous in Scarlet Spur (1:1); and heterozygous in both Golden Delicious and Scarlet Spur (1:2:1). SNPs deviating from these ratios according to a chi-squared test (<em>P</em> &lt; 0.01) were not included in map construction. Finally, progeny genotypes inconsistent with Mendelian inheritance from the parental genotypes were set to missing. After implementing these filters, 2436 SNPs remained for linkage map construction.</p></div><div id="sec-6" class="subsection"><h3>Linkage map construction</h3><p id="p-15">A single composite linkage map was constructed using JoinMap 4.0 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Stam 1993</a>) by combining the backcross type (Aa × aa) markers segregating within each parental background and intercross type (Aa × Aa) markers segregating within both parental backgrounds. Markers suspected of being incorrectly phased by JoinMap, due to high pairwise linkage LOD and spurious recombination fractions above 0.5, had their allele codes switched manually but were dropped from further analysis if phasing problems persisted. To increase mapping efficiency, pairs or groups of loci with identical genotypes (<em>i.e.</em>, complete linkage) were identified and a single marker was chosen to represent the group.</p><p id="p-16">Linkage groups were constructed and ordered with a linkage LOD of at least 6.0, a minimum recombination fraction of 0.35, and a jump threshold of 5. SNPs exhibiting “suspect linkage” to several loci within their assigned group (as determined by JoinMap), poorly fitting markers within an ordered group, and markers that greatly inflated the linkage group size were dropped from the final mapping. All map distances were calculated using the Kosambi mapping function. After applying these filters, 1994 SNPs remained within the genetic linkage map.</p></div><div id="sec-7" class="subsection"><h3>Mapping skin color</h3><p id="p-17">For association analysis based on the physical coordinates obtained from the reference genome, the full set of 3967 SNPs were tested for association with skin color measurements using the case/control single marker analysis in PLINK 1.04 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Purcell <em>et al.</em> 2007</a>), which uses a chi-squared test of independence of allele frequencies between cases and controls with one degree of freedom at each marker independently. For linkage map–based QTL analysis, R/qtl 1.26-14 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2003</a>) was used with the single binary QTL interval mapping model, scanning the linkage map at 1-cM steps for the presence of a significant QTL. A significance threshold of 3.05 was determined by permutation tests on 1000 randomizations of the trait data (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Churchill and Doerge 1994</a>). The PLINK genotype files, JoinMap input files, and phenotype data are available from the Dryad Digital Repository (<a href="http://doi.org/10.5061/dryad.55t54">http://doi.org/10.5061/dryad.55t54</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Alignment of GBS reads to the reference genome</h3><p id="p-18">GBS of a single plate of 96 DNA samples yielded 85,129,960 100-bp reads using Illumina HiSeq 2000 sequencing technology. The high proportion of reads beginning with a barcode sequence (98.2%) and containing a restriction site remnant (99%) indicated that the library preparation was effective and the data were of high quality. In addition, there were very low occurrences of chimeric reads (∼1.0%) and of reads containing downstream adapter sequences (∼1.2%).</p><p id="p-19">After aligning each of the 96 samples’ reads to the "Golden Delicious" v1.0 reference genome, seven of the samples were found to have a relatively low numbers of reads (&lt;150,000) that uniquely aligned to the genome and these samples were subsequently dropped from further analysis. In the remaining 89 samples (87 F1 progeny and 2 parents) there was an average of 973,896 (SD = 609,869) reads per sample and an average of 628,085 reads (SD = 393,153) uniquely aligned to the reference genome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Despite the wide range of read counts across samples, there was relative uniformity in the proportion of reads successfully mapped across samples with an average of 63.5% (SD = 2.5%).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of alignment of GBS reads to the apple reference genome. For each sample, the number of reads mapped and number of reads unmapped to the reference genome are shown. The read counts for the parents of the F1 mapping population, Golden Delicious and Scarlet Spur, are indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26991191" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of alignment of GBS reads to the apple reference genome. For each sample, the number of reads mapped and number of reads unmapped to the reference genome are shown. The read counts for the parents of the F1 mapping population, Golden Delicious and Scarlet Spur, are indicated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1681/F1.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1681/F1.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464875" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Results of alignment of GBS reads to the apple reference genome. For each sample, the number of reads mapped and number of reads unmapped to the reference genome are shown. The read counts for the parents of the F1 mapping population, Golden Delicious and Scarlet Spur, are indicated.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>SNP calling</h3><p id="p-21">Considering only the successfully mapped reads from 89 samples, SNPs were discovered and genotypes were called by analyzing the single master alignment file with GATK (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>). After using a minimal set of initial quality filters (see <em>Materials and Methods</em>), 273,835 SNPs were identified. However, the resulting genotype matrix was extremely sparse: more than 75% of the 273,835 SNPs contained &gt;50% missing genotypes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Restricting the analysis to SNPs with &lt;20% missing genotype data drastically reduced the number of SNPs to 30,393. It is likely that many false-negative genotype calls still exist in the resulting genotype table because confidently calling heterozygotes in a highly heterozygous diploid species like apple requires a relatively high depth of sequence coverage compared with genotype calling in inbreds. For the set of 30,393 SNPs, the number of genotype calls at various sequence depth thresholds is shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. Despite the observation that many genotype calls are supported by &gt;20 reads, the number of SNPs for downstream analysis declines rapidly as the minimum depth of coverage threshold is increased from 1 to 10 while implementing a missing data threshold of 20% (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). We further reduced the number of markers to 3967 SNPs by applying a minimum depth of coverage threshold of six sequence reads for a genotype call (<em>i.e.</em>, any genotype with fewer than six supporting sequence reads was set to missing) and then re-filtering for SNPs with &lt;20% missing genotypes. The choice of six reads per genotype as a threshold was arbitrary: it was chosen as a tradeoff between increased confidence in genotype calls and the number of SNPs retained for mapping. This set of 3967 SNPs was used to map skin color using a chi-squared test. To create the genetic map, a final set of 2436 SNPs was retained after filtering for Mendelian inconsistencies and segregation distortion (see <em>Materials and Methods</em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP and genotype counts from GBS data. (A) Cumulative count of SNPs identified across varying missing data thresholds. More than 200,000 SNPs are called with a very liberal missing data threshold of 90%, but only 30,393 SNPs remain if only SNPs with " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26991191" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SNP and genotype counts from GBS data. (A) Cumulative count of SNPs identified across varying missing data thresholds. More than 200,000 SNPs are called with a very liberal missing data threshold of 90%, but only 30,393 SNPs remain if only SNPs with &lt;20% missing data are retained. (B) The number of genotypes called at increasing levels of sequencing depth, after retaining only SNPs with &lt;20% missing data. (C) The number of SNPs retained at increasing minimum thresholds of sequence depth while retaining only SNPs with &lt;20% missing data. Here, we chose a minimum depth of coverage of six reads. Thus, only SNPs with at least six supporting reads and &lt;20% missing genotypes were retained, resulting in a set of 3967 SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1681/F2.medium.gif" width="440" height="132"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/9/1681/F2.medium.gif" width="440" height="132"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464995" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">SNP and genotype counts from GBS data. (A) Cumulative count of SNPs identified across varying missing data thresholds. More than 200,000 SNPs are called with a very liberal missing data threshold of 90%, but only 30,393 SNPs remain if only SNPs with &lt;20% missing data are retained. (B) The number of genotypes called at increasing levels of sequencing depth, after retaining only SNPs with &lt;20% missing data. (C) The number of SNPs retained at increasing minimum thresholds of sequence depth while retaining only SNPs with &lt;20% missing data. Here, we chose a minimum depth of coverage of six reads. Thus, only SNPs with at least six supporting reads and &lt;20% missing genotypes were retained, resulting in a set of 3967 SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genetic map construction</h3><p id="p-23">The final set of 2436 SNPs was separated into the three segregation types: 884 heterozygous in Golden Delicious, 1044 heterozygous in Scarlet Spur, and 508 heterozygous in both parental varieties. Once this set of SNPs was imported into JoinMap 4.0, 442 segregated identically to at least one other SNP (<em>i.e.</em>, no recombinants observed) and were dropped from further mapping. The remaining 1994 SNPs were successfully grouped into 17 linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/FigureS2.pdf">Figure S2</a>), as expected for the apple genome, at a conservative LOD threshold of six. Only 35 loci were found to be unlinked to any group, suggesting a high degree of saturation of the assembled mapping groups. Once ordered, the linkage groups spanned 1272 cM, with individual linkage groups ranging from 56 cM to 96 cM. Across linkage groups, the average marker density was high with a marker found every 0.68 cM (± 0.13 SD).</p></div><div id="sec-12" class="subsection"><h3>QTL mapping for apple skin color</h3><p id="p-24">Single marker analysis using the final set of 3967 SNPs identified 15 SNPs significantly associated with skin color after applying a conservative Bonferroni correction for multiple testing (<em>P</em> &lt; 1.26 × 10<sup>−5</sup>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). These SNPs cluster within a single interval on chromosome 9 (bounded by coordinates 29,305,493 and 33,701,563). There was also a cluster of SNPs on the distal portion of chromosome 13 that showed a suggestive association with skin color, but all fell short of the corrected significance threshold (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of a single marker association analysis for apple skin color. Each of the 3967 SNPs is plotted according to its physical position from the &quot;Golden Delicious&quot; reference genome and the &#x2212;log10 P value of the single marker association test. The horizontal dotted line represents the Bonferonni-corrected P value significance threshold. The vertical dotted line represents the location of the MYB transcription factor gene known to be responsible for skin color variation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26991191" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plot of a single marker association analysis for apple skin color. Each of the 3967 SNPs is plotted according to its physical position from the &quot;Golden Delicious&quot; reference genome and the &#x2212;log&lt;sub&gt;10&lt;/sub&gt; &lt;em&gt;P&lt;/em&gt; value of the single marker association test. The horizontal dotted line represents the Bonferonni-corrected &lt;em&gt;P&lt;/em&gt; value significance threshold. The vertical dotted line represents the location of the MYB transcription factor gene known to be responsible for skin color variation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1681/F3.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/9/1681/F3.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Manhattan plot of a single marker association analysis for apple skin color. Each of the 3967 SNPs is plotted according to its physical position from the "Golden Delicious" reference genome and the −log<sub>10</sub> <em>P</em> value of the single marker association test. The horizontal dotted line represents the Bonferonni-corrected <em>P</em> value significance threshold. The vertical dotted line represents the location of the MYB transcription factor gene known to be responsible for skin color variation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Interval mapping using 1994 SNPs detected one highly significant QTL for skin color on the distal portion of chromosome 9, at position 89.5 cM, near SNP 9_30303238 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The “2 LOD” confidence interval for this QTL extended to a 14-cM region around the point estimate of the QTL position.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Result of QTL analysis across the linkage group corresponding to chromosome 9 of the apple genome. The left panel indicates the genetic map positions in cM of each of the SNPs or groups of SNPs. Each SNP&#x2019;s ID indicates its physical position according to the reference genome, i.e., the physical coordinate it was assigned through alignment and SNP calling (e.g., SNP 9_ 449878 mapped to position 449878 on chromosome 9 of the &quot;Golden Delicious&quot; v1.0 reference genome). Note that many SNPs genetically mapping to the linkage group corresponding to chromosome 9 are assigned to other chromosomes according to the reference genome (e.g., SNP 13_30028140). The right panel displays the LOD scores from a QTL analysis for skin color for markers that segregate in the Scarlet Spur genetic background. The horizontal dashed line represents the significance threshold determined by permutation. LOD scores across all linkage groups are shown in Figure S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26991191" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Result of QTL analysis across the linkage group corresponding to chromosome 9 of the apple genome. The left panel indicates the genetic map positions in cM of each of the SNPs or groups of SNPs. Each SNP&#x2019;s ID indicates its physical position according to the reference genome, &lt;em&gt;i.e.&lt;/em&gt;, the physical coordinate it was assigned through alignment and SNP calling (&lt;em&gt;e.g.&lt;/em&gt;, SNP 9_ 449878 mapped to position 449878 on chromosome 9 of the &quot;Golden Delicious&quot; v1.0 reference genome). Note that many SNPs genetically mapping to the linkage group corresponding to chromosome 9 are assigned to other chromosomes according to the reference genome (&lt;em&gt;e.g.&lt;/em&gt;, SNP 13_30028140). The right panel displays the LOD scores from a QTL analysis for skin color for markers that segregate in the Scarlet Spur genetic background. The horizontal dashed line represents the significance threshold determined by permutation. LOD scores across all linkage groups are shown in Figure S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1681/F4.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/9/1681/F4.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1681/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464861" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Result of QTL analysis across the linkage group corresponding to chromosome 9 of the apple genome. The left panel indicates the genetic map positions in cM of each of the SNPs or groups of SNPs. Each SNP’s ID indicates its physical position according to the reference genome, <em>i.e.</em>, the physical coordinate it was assigned through alignment and SNP calling (<em>e.g.</em>, SNP 9_ 449878 mapped to position 449878 on chromosome 9 of the "Golden Delicious" v1.0 reference genome). Note that many SNPs genetically mapping to the linkage group corresponding to chromosome 9 are assigned to other chromosomes according to the reference genome (<em>e.g.</em>, SNP 13_30028140). The right panel displays the LOD scores from a QTL analysis for skin color for markers that segregate in the Scarlet Spur genetic background. The horizontal dashed line represents the significance threshold determined by permutation. LOD scores across all linkage groups are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/FigureS3.pdf">Figure S3</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-28">A major consideration when conducting GBS experiments is the choice of restriction enzyme used to generate the reduced representation library (RRL). This choice determines the tradeoff between the number of fragments in the library and the sequencing depth of the fragments. There are several GBS library preparation methods currently in use, including the original single enzyme <em>Ape</em>KI protocol described by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> (2011)</a>, a double digest using enzymes with differing restriction site lengths (Pst I/Msp I) (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Poland <em>et al.</em> 2012b</a>) and a multi-step procedure that combines the double restriction enzyme digest with selective PCR amplification (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sonah <em>et al.</em> 2013</a>). Although there was ultimately no explicit test of library fragment composition in the present study, using the <em>Hin</em>dIII-<em>Msp</em>I enzyme combination followed by size selection we observed a very small proportion of sequence reads that required adapter trimming (1.2%), suggesting that most fragments were larger than 100 bp and were thus suitable for GBS. However, the number of DNA sequence reads across individual samples varied by more than an order of magnitude (0.15 million–3 million) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), and we ultimately excluded the sequence data of seven samples from analyses due to low read counts. These observations highlight the sensitivity of GBS to DNA sample uniformity. Recent GBS library preparation methods have been shown to improve the uniformity in read counts across samples (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Sonah <em>et al.</em> 2013</a>), and further improvements are expected.</p><p id="p-29">Because of the apple’s high heterozygosity and ancestral polyploidy, there are major challenges in the assembly of its genome. It is estimated that the assembled portion of the current apple reference assembly represents only 71% of its ∼750-Mb genome (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Velasco <em>et al.</em> 2010</a>). Despite this constraint, we restricted our analysis to sequences aligning to this assembly to enable a comparison of our inferred linkage groups to the physical map. Approximately 40% of the sequence reads did not map to the assembled genome and were thus excluded from further analysis (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). These excluded reads are a combination of DNA sequences from ∼29% of the genome that is not represented in the current assembly, DNA sequences that mapped to multiple locations, and sequences that map to a unique position but with too many mismatches (&gt;4 mismatches per 100 bp). As improvements are made to the genome assembly and to DNA sequence length and quality, we anticipate significant improvements in the mapping step of the GBS analysis pipeline. It is worth noting that a potential alternative is to avoid the use of a reference genome altogether and to use a SNP calling pipeline that does not rely on a reference genome (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lu <em>et al.</em> 2013</a>).</p><p id="p-30">There is a larger number of possible paths one can take from a DNA sequence file to a SNP genotype table, and it is known that alignment and genotype calling parameters have a strong effect on the resulting quantity and quality of genome-wide SNP data (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mascher <em>et al.</em> 2013b</a>). Although established software does exist for SNP genotype calling from GBS data (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2007</a>), our goal here was to demonstrate that, with simple heuristics applied together with standard software packages, one can generate SNPs of sufficient quality and quantity to be of utility for genetic mapping. Regardless of what tools are used, it is evident that GBS generates a sparse genotype matrix due to uneven sequence coverage across samples and sites: many SNPs are discovered, but genotypes for these SNPs are most often generated from only a small proportion of the DNA samples. For example, more than 250,000 SNPs were discovered in the present study, but when SNPs with &gt;20% missing data are excluded, the number of SNPs remaining for analysis is reduced to ∼30,000 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). Moreover, the number of SNPs decreases as the depth of coverage filter is increased (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, B and C</a>). We chose arbitrarily to include only genotypes with six or more supporting reads as a trade-off between genotype quality and quantity. Using these thresholds, we obtain a set of 3967 SNPs derived from only 6% of the sequence reads. Thus, in the end, 94% of the sequence we collected in this experiment was ignored. Most GBS studies to date have focused on genotyping inbred lines, which requires far less data and is statistically far simpler than genotype calling in highly heterozygous species like apple, or in polyploids, which are common among species of agricultural importance. GBS SNP calling pipelines designed specifically for highly heterozygous and polyploid species that take haplotype phasing and imputation into account promise to significantly enhance the utility of GBS (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lu <em>et al.</em> 2013</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Ward <em>et al.</em> 2013</a>).</p><p id="p-31">Despite the high proportion of sequence reads discarded due to filtering, there remained a sufficient number of markers to perform genetic mapping of apple skin color with a modest sample size. By mapping skin color, we have intentionally focused on a trait with a simple genetic architecture, which deviates from most QTL studies that focus on more complex traits. Skin color is known to be controlled almost entirely by a single locus of large effect (<em>i.e.</em>, effectively Mendelian), and we leverage this knowledge here to verify the utility and power of GBS to identify this known locus. With 3967 SNPs genotyped in 89 samples, a simple chi-squared test for association revealed a single significant peak from position 29.3 Mb to 33.7 Mb on chromosome 9 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). This peak overlaps with the R2R3 MYB transcription factor gene at position 32.8 Mb on chromosome 9 known to regulate apple skin color (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Takos <em>et al.</em> 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Espley <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lin-Wang <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chagne <em>et al.</em> 2013</a>). In addition, a set of 1994 SNPs was used to produce a well-saturated linkage map spanning 1272 cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/FigureS2.pdf">Figure S2</a>). This map size is consistent with the sizes of apple genetic maps from previous studies, for example, 1538 cM (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Khan <em>et al.</em> 2012</a>), 1005 cM (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chagne <em>et al.</em> 2012b</a>), and 1143 cM (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Han <em>et al.</em> 2011</a>). On average, the genetic map has a marker every 0.68 cM, which is in line with the marker density achieved using the apple 8K SNP array (0.5 cM/marker in <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Antanaviciute <em>et al.</em> 2012</a>; 0.88 cM/marker in <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chagne <em>et al.</em> 2012a</a>). Using this genetic map, interval mapping revealed one highly significant QTL for skin color centered at position 89.5 cM (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/FigureS3.pdf">Figure S3</a>), in agreement with the results from the chi-squared test for association. Thus, with a set of simple heuristics for calling genotypes from GBS data, a saturated genetic map can be generated and QTL mapping can robustly identify genotype–phenotype associations.</p><p id="p-32">In the present study, the parents of the F1 population were included only once and had average sequence coverage (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). However, because inclusion of a SNP in the genetic map relied on accurate genotype calls from both parents, it may be advisable to sequence the parents of mapping populations to a higher depth, <em>i.e.</em>, include them multiple times in the plate of samples. Moreover, the genetic map presented here was constructed using only SNPs that mapped to the anchored portion of the reference genome to allow comparison between physical and genetic map positions. By including the unanchored portion of the genome sequence during the initial alignment stage, it is likely that additional SNPs could be identified and placed on the genetic map.</p><p id="p-33">For 364 SNPs (18.3% of all mapped SNPs), the linkage group assignments conflicted with the predicted chromosomal locations according to the reference genome. <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Antanaviciute <em>et al.</em> (2012)</a> reported a similar proportion (13.7%) using genotype data from the apple 8K SNP array. The most likely reasons for these conflicts between genetic and physical maps are the high frequency of paralagous genomic regions in the apple genome and the incorrect anchoring of sequences during the assembly of the reference genome. Such conflicts may obscure association signals and complicate the interpretation of genetic mapping results. For example, although below the conservative threshold for significance, we detected a signal of association on chromosome 13 when SNPs were positioned according to the physical map (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). However, this same block of SNPs that physically map to chromosome 13 was subsequently found to genetically map close to the QTL for skin color on linkage group 9 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). This demonstrates that caution is warranted when relying on the physical map coordinates of the current reference genome sequence.</p><p id="p-34">It is worth noting that our modest mapping population size of 87 F1 offspring likely often prevented the ordering algorithm from finding the correct order for SNPs that were closer than ∼2–5 cM. Over larger distances, however, the estimated map order of SNPs was generally in agreement with physical coordinates from the reference genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-35">The present study demonstrates that a Mendelian trait (skin color) can be mapped in an apple F1 population using GBS data from a single lane of Illumina next-generation sequencing. Considering the currents costs of acquiring SNP data from the apple 8K genotyping microarray, we estimate that a similar quantity and quality of apple SNP data can be achieved using GBS with a 10–100× decrease in cost. Because of the rapid improvements in DNA sequencing technology, we anticipate that genotyping efforts will increasingly favor the use of GBS or similar methods over the use of microarrays. The present study uses standard software tools and simple heuristics to generate biological insights from GBS data. However, it is clear that methodological developments required for analyzing GBS data lag far behind the technology developed to generate GBS data. To maximize the utility of next-generation DNA sequence in the future, there is a clear need for improved computational and statistical tools to extract as much information as possible from the raw data, and to phase, impute, order, and genetically map large sets of genetic markers.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">This article was written, in part, thanks to funding from the Canada Research Chairs program and the National Sciences and Engineering Research Council of Canada.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011023/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Data available via the Dryad Digital Repository: <a href="http://doi.org/10.5061/dryad.55t54">http://doi.org/10.5061/dryad.55t54</a></p></li><li class="fn" id="fn-3"><p id="p-37"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-13"><span class="received-label">Received </span>March 13, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-07-02"><span class="accepted-label">Accepted </span>July 2, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Gardner <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.1" data-doi="10.1186/1471-2164-13-203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antanaviciute</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Fernandez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAntanaviciute%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D203%26rft.epage%253D203%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bof%2Ban%2Bapple%2Brootstock%2Bprogeny%2Busing%2Bthe%2BMalus%2BInfinium%2Bwhole%2Bgenome%2Bgenotyping%2Barray.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-203%26rft_id%253Dinfo%253Apmid%252F22631220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22631220&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.3" data-doi="10.1007/s00122-013-2202-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barba</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cadle-Davidson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Grapevine powdery mildew resistance and susceptibility loci identified on a high-resolution SNP map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2202-x%26rft_id%253Dinfo%253Apmid%252F24072208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2202-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24072208&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.4" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.5" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bus</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranatunga</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basset</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rikkerink</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection for pest and disease resistance in the New Zealand apple breeding program.</span> <abbr class="cit-jnl-abbrev">Acta Hortic.</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BHortic.%26rft.volume%253D538%26rft.spage%253D541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.7" data-doi="10.1007/s11295-008-0171-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Celton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tustin</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Construction of a dense genetic linkage map for apple rootstocks using SSRs developed from Malus ESTs and Pyrus genomic sequences.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D5%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-008-0171-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-008-0171-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261265100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.8" data-doi="10.1371/journal.pone.0031745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowhurst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e31745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChagne%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De31745%26rft.epage%253De31745%26rft.atitle%253DGenome-wide%2BSNP%2Bdetection%252C%2Bvalidation%252C%2Band%2Bdevelopment%2Bof%2Ban%2B8K%2BSNP%2Barray%2Bfor%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0031745%26rft_id%253Dinfo%253Apmid%252F22363718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0031745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363718&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.9" data-doi="10.1186/1471-2229-12-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krieger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rassam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">QTL and candidate gene mapping for polyphenolic composition in apple fruit.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DChagne%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DQTL%2Band%2Bcandidate%2Bgene%2Bmapping%2Bfor%2Bpolyphenolic%2Bcomposition%2Bin%2Bapple%2Bfruit.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-12%26rft_id%253Dinfo%253Apmid%252F22269060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22269060&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.10" data-doi="10.1104/pp.112.206771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin-Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espley</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">How</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An ancient duplication of apple MYB transcription factors is responsible for novel red fruit-flesh phenotypes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.206771%26rft_id%253Dinfo%253Apmid%252F23096157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjEvMS8yMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Consortium</span>  <span class="cit-name-given-names">T. I. B. G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A physical, genetic and functional sequence assembly of the barley genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMayer%26rft.auinit1%253DK.%2BF.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D711%26rft.epage%253D716%26rft.atitle%253DA%2Bphysical%252C%2Bgenetic%2Band%2Bfunctional%2Bsequence%2Bassembly%2Bof%2Bthe%2Bbarley%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F23075845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23075845&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.13" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.14" data-doi="10.1111/j.1755-0238.2009.00072.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">di Gaspero</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattonaro</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Application of genomics to grapevine improvement.</span> <abbr class="cit-jnl-abbrev">Aust. J. Grape Wine Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BGrape%2BWine%2BRes.%26rft.volume%253D16%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0238.2009.00072.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0238.2009.00072.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.15" data-doi="10.1073/pnas.0307937101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dirlewanger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graziano</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joobeur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garriga-Calderé</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative mapping and marker-assisted selection in Rosaceae fruit crops.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">9891</span>–<span class="cit-lpage">9896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307937101%26rft_id%253Dinfo%253Apmid%252F15159547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzI2Lzk4OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.16" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.17" data-doi="10.1111/j.1365-313X.2006.02964.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Espley</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellens</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Putterill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevenson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutty-Amma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Red colouration in apple fruit is due to the activity of the MYB transcription factor, MdMYB10.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">414</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DEspley%26rft.auinit1%253DR.%2BV.%26rft.volume%253D49%26rft.issue%253D3%26rft.spage%253D414%26rft.epage%253D427%26rft.atitle%253DRed%2Bcolouration%2Bin%2Bapple%2Bfruit%2Bis%2Bdue%2Bto%2Bthe%2Bactivity%2Bof%2Bthe%2BMYB%2Btranscription%2Bfactor%252C%2BMdMYB10.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02964.x%26rft_id%253Dinfo%253Apmid%252F17181777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02964.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17181777&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243637900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.18" data-doi="10.1007/s11295-012-0478-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Fernández</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antanaviciute</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyk</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobutt</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A genetic linkage map of an apple rootstock progeny anchored to the Malus genome sequence.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">1002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D991%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-012-0478-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-012-0478-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vimolmangkang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beever</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integration of physical and genetic maps in apple confirms whole-genome and segmental duplications in the apple genome.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot. 62: 5117–5130</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellerhals</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianfranceschi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seglias</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gessler</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection in apple breeding.</span> <abbr class="cit-jnl-abbrev">Acta Hortic.</abbr> <span class="cit-vol">521</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BHortic.%26rft.volume%253D521%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.21" data-doi="10.1007/s11032-004-5592-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kenis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keulemans</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic linkage maps of two apple cultivars (Malus × domestica Borkh.) based on AFLP and microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D15%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-004-5592-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-004-5592-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.22" data-doi="10.1371/journal.pone.0047864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korban</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKhan%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47864%26rft.epage%253De47864%26rft.atitle%253DA%2BMulti-Population%2BConsensus%2BGenetic%2BMap%2BReveals%2BInconsistent%2BMarker%2BOrder%2Bamong%2BMaps%2BLikely%2BAttributed%2Bto%2BStructural%2BVariations%2Bin%2Bthe%2BApple%2BGenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047864%26rft_id%253Dinfo%253Apmid%252F23144832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23144832&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.23" data-doi="10.1371/journal.pone.0036674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection for fruit quality traits in apple (<em>Malus×domestica Borkh</em>.).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36674%26rft.epage%253De36674%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bfruit%2Bquality%2Btraits%2Bin%2Bapple%2B%2528Malus%25C3%2597domestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036674%26rft_id%253Dinfo%253Apmid%252F22574211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574211&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.24" data-doi="10.1186/1471-2164-14-393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Novel genomic approaches unravel genetic architecture of complex traits in apple.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-393%26rft_id%253Dinfo%253Apmid%252F23758946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23758946&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.27" data-doi="10.1023/A:1024886500979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liebhard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellerhals</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfammatter</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jertmini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gessler</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping quantitative physiological traits in apple (<em>Malus X domestica</em> Borkh.).</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DLiebhard%26rft.auinit1%253DR.%26rft.volume%253D52%26rft.issue%253D3%26rft.spage%253D511%26rft.epage%253D526%26rft.atitle%253DMapping%2Bquantitative%2Bphysiological%2Btraits%2Bin%2Bapple%2B%2528Malus%2Bx%2Bdomestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1024886500979%26rft_id%253Dinfo%253Apmid%252F12956523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1024886500979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12956523&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184278800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.28" data-doi="10.1111/j.1365-3040.2011.02316.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin-Wang</span>  <span class="cit-name-given-names">K. U. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Micheletti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High temperature reduces apple fruit colour via modulation of the anthocyanin regulatory complex.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1176</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DLin-Wang%26rft.auinit1%253DK.%26rft.volume%253D34%26rft.issue%253D7%26rft.spage%253D1176%26rft.epage%253D1190%26rft.atitle%253DHigh%2Btemperature%2Breduces%2Bapple%2Bfruit%2Bcolour%2Bvia%2Bmodulation%2Bof%2Bthe%2Banthocyanin%2Bregulatory%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2011.02316.x%26rft_id%253Dinfo%253Apmid%252F21410713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2011.02316.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21410713&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292543500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.29" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: Novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.30" data-doi="10.1111/tpj.12319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokhsar</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D76%26rft.spage%253D718%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12319%26rft_id%253Dinfo%253Apmid%252F23998490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23998490&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326138100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.31" data-doi="10.1371/journal.pone.0076925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amand</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Application of genotyping-by-sequencing on semiconductor sequencing platforms: A comparison of genetic and reference-based marker ordering in barley.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e76925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De76925%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0076925%26rft_id%253Dinfo%253Apmid%252F24098570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0076925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24098570&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.32" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.33" data-doi="10.1007/s11295-011-0380-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Micheletti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zharkikh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malnoy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic diversity of the genus <em>Malus</em> and implications for linkage mapping with SNPs.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0380-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0380-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matasci</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aradhya</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Vitis phylogenomics: hybridization intensities from a SNP array outperform genotype calls.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> (<span class="cit-comment">submitted</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.35" data-doi="10.1016/j.tig.2013.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improving fruit and wine: what does genomics have to offer?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">190</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D190%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.01.006%26rft_id%253Dinfo%253Apmid%252F23428114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23428114&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.36" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection in wheat using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.38" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.39" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.40" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.41" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Légaré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.42" data-doi="10.1007/s00122-013-2166-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2699</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2699%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2166-x%26rft_id%253Dinfo%253Apmid%252F23918062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2166-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918062&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.43" data-doi="10.1046/j.1365-313X.1993.03050739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linkage maps by means of a new computer package: JoinMap.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D5%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1993.03050739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1993.03050739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LP73600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.44" data-doi="10.1104/pp.106.088104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaffé</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Light-induced expression of a MYB gene regulates anthocyanin biosynthesis in red apples.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">1216</span>–<span class="cit-lpage">1232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.106.088104%26rft_id%253Dinfo%253Apmid%252F17012405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQyLzMvMTIxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.45" data-doi="10.1038/ng.654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zharkikh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhingra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome of the domesticated apple (Malus x domestica Borkh.).</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">833</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DVelasco%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D839%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bdomesticated%2Bapple%2B%2528Malus%2B%25C3%2597%2Bdomestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.654%26rft_id%253Dinfo%253Apmid%252F20802477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20802477&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282276600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.46" data-doi="10.1186/1471-2164-14-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhangoo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Fernandez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Saturated linkage map construction in Rubus idaeus using genotyping by sequencing and genome-independent imputation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-2%26rft_id%253Dinfo%253Apmid%252F23324311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324311&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1681.47" data-doi="10.1007/s11032-010-9449-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peace</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Utility testing of an apple skin color MdMYB1 marker in two progenies.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D27%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-010-9449-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-010-9449-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1681.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1671" title="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1689" title="Whole-Genome Sequence of the C57L/J Mouse Inbred Strain" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1681.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xmo9TxuUgYdqaSHXULFuRyy1Okv0hQcWE3FVLwJgFJ8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1681" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1681" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1681" class="active" data-icon-position="" data-hide-link-title="0">Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459664" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XDjxtXwmGVm6S9OtfU-R3FQPQ2rNcLLQToyTcJo6ypQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1681.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1681" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--Oil-wq88ss7mlAcGqCbx2wezBAb7Ha05J50Y_8_-Xk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1681" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1681.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459664" id="node459664--2" data-pisa="ggg;4/9/1681" data-pisa-master="ggg;g3.114.011023" data-apath="/ggg/4/9/1681.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kyle M. Gardner</span>, <span class="highwire-citation-author" data-delta="1">Patrick Brown</span>, <span class="highwire-citation-author" data-delta="2">Thomas F. Cooke</span>, <span class="highwire-citation-author" data-delta="3">Scott Cann</span>, <span class="highwire-citation-author" data-delta="4">Fabrizio Costa</span>, <span class="highwire-citation-author" data-delta="5">Carlos Bustamante</span>, <span class="highwire-citation-author" data-delta="6">Riccardo Velasco</span>, <span class="highwire-citation-author" data-delta="7">Michela Troggio</span> and <span class="highwire-citation-author" data-delta="8">Sean Myles</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1681-1687;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011023 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459664--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kyle M. Gardner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BM.%2BGardner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gardner%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BM.%2BGardner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois, Urbana, Illinois</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas F. Cooke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BF.%2BCooke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooke%20TF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BF.%2BCooke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Scott Cann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BCann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BCann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fabrizio Costa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabrizio%2BCosta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costa%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFabrizio%2BCosta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carlos Bustamante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BBustamante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bustamante%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BBustamante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Riccardo Velasco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riccardo%2BVelasco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Velasco%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiccardo%2BVelasco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michela Troggio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michela%2BTroggio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Troggio%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichela%2BTroggio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sean Myles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BMyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASean%2BMyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sean.myles@dal.ca" class="" data-icon-position="" data-hide-link-title="0">sean.myles@dal.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1681" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1681&amp;title=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1681&amp;title=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1681&amp;title=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1681&amp;text=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1681&amp;title=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1681&amp;t=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1681&amp;title=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1681&amp;title=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459664/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459664" id="node459664--3" data-pisa="ggg;4/9/1681" data-pisa-master="ggg;g3.114.011023" data-apath="/ggg/4/9/1681.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast and Cost-Effective Genetic Mapping in Apple Using Next-Generation Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kyle M. Gardner</span>, <span class="highwire-citation-author" data-delta="1">Patrick Brown</span>, <span class="highwire-citation-author" data-delta="2">Thomas F. Cooke</span>, <span class="highwire-citation-author" data-delta="3">Scott Cann</span>, <span class="highwire-citation-author" data-delta="4">Fabrizio Costa</span>, <span class="highwire-citation-author" data-delta="5">Carlos Bustamante</span>, <span class="highwire-citation-author" data-delta="6">Riccardo Velasco</span>, <span class="highwire-citation-author" data-delta="7">Michela Troggio</span> and <span class="highwire-citation-author" data-delta="8">Sean Myles</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1681-1687;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011023 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459664--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kyle M. Gardner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BM.%2BGardner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gardner%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BM.%2BGardner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois, Urbana, Illinois</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas F. Cooke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BF.%2BCooke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooke%20TF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BF.%2BCooke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Scott Cann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BCann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BCann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fabrizio Costa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabrizio%2BCosta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costa%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFabrizio%2BCosta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carlos Bustamante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford School of Medicine, Stanford University, Stanford, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BBustamante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bustamante%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BBustamante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Riccardo Velasco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riccardo%2BVelasco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Velasco%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiccardo%2BVelasco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michela Troggio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics and Molecular Biology Department, IASMA Research Center, San Michele all’Adige, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michela%2BTroggio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Troggio%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichela%2BTroggio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sean Myles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Nova Scotia, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BMyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASean%2BMyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sean.myles@dal.ca" class="" data-icon-position="" data-hide-link-title="0">sean.myles@dal.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1681&amp;rft.epage=1687&amp;rft.atitle=Fast%20and%20Cost-Effective%20Genetic%20Mapping%20in%20Apple%20Using%20Next-Generation%20Sequencing&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gardner%2C+Kyle+M.&amp;rft.au=Brown%2C+Patrick&amp;rft.au=Cooke%2C+Thomas+F.&amp;rft.au=Cann%2C+Scott&amp;rft.au=Costa%2C+Fabrizio&amp;rft.au=Bustamante%2C+Carlos&amp;rft.au=Velasco%2C+Riccardo&amp;rft.au=Troggio%2C+Michela&amp;rft.au=Myles%2C+Sean"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459664/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459664/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459664/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459664/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459664/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459664/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459664/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459664/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459664/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459664/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459664/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459664/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1681" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1681.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1681.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1681.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1681.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
