<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1773" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1773.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1773.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1773.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1773.atom" />
<meta name="HW.pisa" content="ggg;4/9/1773" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.114.013250" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Hope and Dunham. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The ability of yeast to form biofilms contributes to better survival under stressful conditions. We see the impact of yeast biofilms and “flocs” (clumps) in human health and industry, where forming clumps enables yeast to act as a natural filter in brewing and forming biofilms enables yeast to remain virulent in cases of fungal infection. Despite the importance of biofilms in yeast natural isolates, the majority of our knowledge about yeast biofilm genetics comes from work with a few tractable laboratory strains. A new collection of sequenced natural isolates from the Saccharomyces Genome Resequencing Project enabled us to examine the breadth of biofilm-related phenotypes in geographically, ecologically, and genetically diverse strains of Saccharomyces cerevisiae . We present a panel of 31 haploid and 24 diploid strains for which we have characterized six biofilm-related phenotypes: complex colony morphology, complex mat formation, flocculation, agar invasion, polystyrene adhesion, and psuedohyphal growth. Our results show that there is extensive phenotypic variation between and within strains, and that these six phenotypes are primarily uncorrelated or weakly correlated, with the notable exception of complex colony and complex mat formation. We also show that the phenotypic strength of these strains varies significantly depending on ploidy, and the diploid strains demonstrate both decreased and increased phenotypic strength with respect to their haploid counterparts. This is a more complex view of the impact of ploidy on biofilm-related phenotypes than previous work with laboratory strains has suggested, demonstrating the importance and enormous potential of working with natural isolates of yeast." />
<meta name="DC.Contributor" content="Elyse A. Hope" />
<meta name="DC.Contributor" content="Maitreya J. Dunham" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The ability of yeast to form biofilms contributes to better survival under stressful conditions. We see the impact of yeast biofilms and “flocs” (clumps) in human health and industry, where forming clumps enables yeast to act as a natural filter in brewing and forming biofilms enables yeast to remain virulent in cases of fungal infection. Despite the importance of biofilms in yeast natural isolates, the majority of our knowledge about yeast biofilm genetics comes from work with a few tractable laboratory strains. A new collection of sequenced natural isolates from the &lt;i&gt;Saccharomyces&lt;/i&gt; Genome Resequencing Project enabled us to examine the breadth of biofilm-related phenotypes in geographically, ecologically, and genetically diverse strains of &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. We present a panel of 31 haploid and 24 diploid strains for which we have characterized six biofilm-related phenotypes: complex colony morphology, complex mat formation, flocculation, agar invasion, polystyrene adhesion, and psuedohyphal growth. Our results show that there is extensive phenotypic variation between and within strains, and that these six phenotypes are primarily uncorrelated or weakly correlated, with the notable exception of complex colony and complex mat formation. We also show that the phenotypic strength of these strains varies significantly depending on ploidy, and the diploid strains demonstrate both decreased and increased phenotypic strength with respect to their haploid counterparts. This is a more complex view of the impact of ploidy on biofilm-related phenotypes than previous work with laboratory strains has suggested, demonstrating the importance and enormous potential of working with natural isolates of yeast.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1773" />
<meta name="citation_id" content="4/9/1773" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1773" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1773.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1773.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1773.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.013250" />
<meta name="citation_pmid" content="25060625" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1773" />
<meta name="citation_lastpage" content="1786" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elyse A. Hope" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Maitreya J. Dunham" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_email" content="maitreya@uw.edu" />
<meta name="citation_reference" content="citation_journal_title=Biophotonics International;citation_author=M. D. Abramoff;citation_author=P. J. Magalhaes;citation_author=S. J. Ram;citation_title=Image processing with ImageJ.;citation_pages=36-42;citation_volume=11;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=F. F. Bauer;citation_author=P. Govender;citation_author=M. C. Bester;citation_title=Yeast flocculation and its biotechnological relevance.;citation_pages=31-39;citation_volume=88;citation_year=2010;citation_issue=1;citation_pmid=20676629;citation_doi=10.1007/s00253-010-2783-0" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=J. C. Bayly;citation_author=L. M. Douglas;citation_author=I. S. Pretorius;citation_author=F. F. Bauer;citation_author=A. M. Dranginis;citation_title=Characteristics of Flo11-dependent flocculation in Saccharomyces cerevisiae.;citation_volume=5;citation_year=2005;citation_pmid=16043420;citation_doi=10.1016/j.femsyr.2005.05.004" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. Beauvais;citation_author=C. Loussert;citation_author=M. C. Prevost;citation_author=K. Verstrepen;citation_author=J. P. Latgé;citation_title=Characterization of a biofilm-like extracellular matrix in FLO1-expressing Saccharomyces cerevisiae cells.;citation_volume=9;citation_year=2009;citation_pmid=19207290;citation_doi=10.1111/j.1567-1364.2009.00482.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Bergström;citation_author=J. T. Simpson;citation_author=F. Salinas;citation_author=B. Barré;citation_author=L. Parts;citation_title=A high-definition view of functional genetic variation from natural yeast genomes.;citation_volume=31;citation_year=2014;citation_pmid=24425782;citation_doi=10.1093/molbev/msu037" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. Bony;citation_author=D. Thines-Sempoux;citation_author=P. Barre;citation_author=B. Blondin;citation_title=Localization and cell surface anchoring of the Saccharomyces cerevisiae flocculation protein Flo1p.;citation_pages=4929-4936;citation_volume=179;citation_year=1997;citation_issue=15;citation_pmid=9244284;citation_doi=10.1128/jb.179.15.4929-4936.1997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. J. Brauer;citation_author=C. M. Christianson;citation_author=D. A. Pai;citation_author=M. J. Dunham;citation_title=Mapping novel traits by array-assisted bulk segregant analysis in Saccharomyces cerevisiae.;citation_volume=173;citation_year=2006;citation_pmid=16624899;citation_doi=10.1534/genetics.106.057927" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=S. Bruckner;citation_author=H. U. Mosch;citation_title=Choosing the right lifestyle: adhesion and development in Saccharomyces cerevisiae.;citation_volume=36;citation_year=2012;citation_pmid=21521246;citation_doi=10.1111/j.1574-6976.2011.00275.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. A. Cubillos;citation_author=E. Billi;citation_author=E. Zorgo;citation_author=L. Parts;citation_author=P. Fargier;citation_title=Assessing the complex architecture of polygenic traits in diverged yeast populations.;citation_pages=1401-1413;citation_volume=20;citation_year=2011;citation_issue=7;citation_pmid=21261765;citation_doi=10.1111/j.1365-294X.2011.05005.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. A. Cubillos;citation_author=L. Parts;citation_author=F. Salinas;citation_author=A. Bergström;citation_author=E. Scovacricchi;citation_title=High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.;citation_volume=195;citation_year=2013;citation_pmid=24037264;citation_doi=10.1534/genetics.113.155515" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. J. Cullen;citation_author=G. F. Sprague;citation_title=Glucose depletion causes haploid invasive growth in yeast.;citation_volume=97;citation_year=2000;citation_pmid=11095711;citation_doi=10.1073/pnas.240345197" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. J. L. de Hoon;citation_author=S. Imoto;citation_author=J. Nolan;citation_author=S. Miyano;citation_title=Open Source Clustering Software.;citation_volume=20;citation_year=2004;citation_pmid=14871861;citation_doi=10.1093/bioinformatics/bth078" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. L. Drees;citation_author=V. Thorsson;citation_author=G. W. Carter;citation_author=A. W. Rives;citation_author=M. Z. Raymond;citation_title=Derivation of genetic interaction networks from quantitative phenotype data.;citation_pages=R38-R38;citation_volume=6;citation_year=2005;citation_issue=4;citation_pmid=15833125;citation_doi=10.1186/gb-2005-6-4-r38" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Galitski;citation_author=A. J. Saldanha;citation_author=C. A. Styles;citation_author=E. S. Lander;citation_author=G. R. Fink;citation_title=Ploidy Regulation of Gene Expression;citation_pages=251-254;citation_volume=285;citation_year=1999;citation_issue=5425;citation_pmid=10398601;citation_doi=10.1126/science.285.5425.251" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. J. Gimeno;citation_author=P. O. Ljungdahl;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Unipolar cell divisions in the yeast S. cerevisiae lead to filamentous growth: regulation by starvation and RAS.;citation_pages=1077-1090;citation_volume=68;citation_year=1992;citation_issue=6;citation_pmid=1547504;citation_doi=10.1016/0092-8674(92)90079-R" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. A. Granek;citation_author=P. M. Magwene;citation_title=Environmental and genetic determinants of colony morphology in yeast.;citation_pages=1-12;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Granek;citation_author=D. Murray;citation_author=O. Kayrkci;citation_author=P. M. Magwene;citation_title=The genetic architecture of biofilm formation in a clinical isolate of Saccharomyces cerevisiae.;citation_volume=193;citation_year=2013;citation_pmid=23172850;citation_doi=10.1534/genetics.112.142067" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Guo;citation_author=C. A. Styles;citation_author=Q. Feng;citation_author=G. R. Fink;citation_title=A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating.;citation_volume=97;citation_year=2000;citation_pmid=11027318;citation_doi=10.1073/pnas.220420397" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Halfmann;citation_author=D. F. Jarosz;citation_author=S. K. Jones;citation_author=A. Chang;citation_author=A. K. Lancaster;citation_title=Prions are a common mechanism for phenotypic inheritance in wild yeasts.;citation_pages=363-368;citation_volume=482;citation_year=2012;citation_issue=7385;citation_pmid=22337056;citation_doi=10.1038/nature10875" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. A. Hill;citation_author=R. Ammar;citation_author=D. Torti;citation_author=C. Nislow;citation_author=L. E. Cowen;citation_title=Genetic and genomic architecture of the evolution of resistance to antifungal drug combinations.;citation_pages=1-22;citation_volume=9;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=D. L. Holmes;citation_author=A. K. Lancaster;citation_author=S. Lindquist;citation_author=R. Halfmann;citation_title=Heritable remodeling of yeast multicellularity by an environmentally responsive prion.;citation_pages=153-165;citation_volume=153;citation_year=2013;citation_pmid=23540696;citation_doi=10.1016/j.cell.2013.02.026" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=M. Jara;citation_author=F. A. Cubillos;citation_author=V. García;citation_author=F. Salinas;citation_author=O. Aguilera;citation_title=Mapping genetic variants underlying differences in the central nitrogen metabolism in fermenter yeasts.;citation_pages=e86533;citation_volume=9;citation_year=2014;citation_pmid=24466135;citation_doi=10.1371/journal.pone.0086533" />
<meta name="citation_reference" content="citation_title=Genetic control of flocculation, pp. 205-224 in Yeast genetics: fundamental and applied aspects;citation_author=J. R. Johnston;citation_author=H. P. Reader;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=M. Kendall;citation_title=A NEW MEASURE OF RANK CORRELATION;citation_pages=81-93;citation_volume=30;citation_year=1938;citation_issue=1-2;citation_doi=10.1093/biomet/30.1-2.81" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Liu;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Saccharomyces cerevisiae S288C Has a Mutation in FL08, a Gene Required for Filamentous Growth;citation_pages=967-978;citation_volume=144;citation_year=1996;citation_issue=3;citation_pmid=8913742" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=W. S. Lo;citation_author=A. M. Dranginis;citation_title=FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin.;citation_pages=71447151;citation_volume=178;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=W. S. Lo;citation_author=A. M. Dranginis;citation_title=The Cell Surface Flocculin Flo11 Is Required for Pseudohyphae Formation and Invasion by Saccharomyces cerevisiae;citation_pages=161-171;citation_volume=9;citation_year=1998;citation_issue=1;citation_pmid=9436998;citation_doi=10.1091/mbc.9.1.161" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. C. Lorenz;citation_author=J. Heitman;citation_title=Yeast pseudohyphal growth is regulated by GPA2, a G protein {alpha} homolog;citation_pages=7008-7018;citation_volume=16;citation_year=1997;citation_issue=23;citation_pmid=9384580;citation_doi=10.1093/emboj/16.23.7008" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=B. L. Miki;citation_author=N. H. Poon;citation_author=V. L. Seligy;citation_title=Possible mechanism for flocculation interactions governed by gene FLO1 in Saccharomyces cerevisiae.;citation_pages=878889;citation_volume=150;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=G. A. O’Toole;citation_author=R. Kolter;citation_title=Initiation of biofilm formation in Pseudomonas fluorescens WCS365 proceeds via multiple, convergent signalling pathways: a genetic analysis.;citation_pages=449-461;citation_volume=28;citation_year=1998;citation_issue=3;citation_pmid=9632250;citation_doi=10.1046/j.1365-2958.1998.00797.x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=G. A. O’Toole;citation_author=H. B. Kaplan;citation_author=R. Kolter;citation_title=BIOFILM FORMATION AS MICROBIAL DEVELOPMENT;citation_pages=49-79;citation_volume=54;citation_year=2000;citation_issue=1;citation_pmid=11018124;citation_doi=10.1146/annurev.micro.54.1.49" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=S. P. Palecek;citation_author=A. S. Parikh;citation_author=S. J. Kron;citation_title=Sensing, signalling and integrating physical processes during Saccharomyces cerevisiae invasive and filamentous growth;citation_pages=893-907;citation_volume=148;citation_year=2002;citation_issue=4;citation_pmid=11932437;citation_doi=10.1099/00221287-148-4-893" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. B. Reynolds;citation_author=G. R. Fink;citation_title=Bakers' Yeast, a Model for Fungal Biofilm Formation;citation_pages=878-881;citation_volume=291;citation_year=2001;citation_issue=5505;citation_pmid=11157168;citation_doi=10.1126/science.291.5505.878" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. I. Roop;citation_author=R. B. Brem;citation_title=Rare variants in hypermutable genes underlie common morphology and growth traits in wild Saccharomyces paradoxus.;citation_volume=195;citation_year=2013;citation_pmid=23934881;citation_doi=10.1534/genetics.113.155341" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=P. Ruusuvuori;citation_author=J. Lin;citation_author=A. C. Scott;citation_author=Z. Tan;citation_author=S. Sorsa;citation_title=Quantitative analysis of colony morphology in yeast.;citation_pages=18-27;citation_volume=56;citation_year=2014;citation_pmid=24447135" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=O. Ryan;citation_author=R. S. Shapiro;citation_author=C. F. Kurat;citation_author=D. Mayhew;citation_author=A. Baryshnikova;citation_title=Global Gene Deletion Analysis Exploring Yeast Filamentous Growth;citation_pages=1353-1356;citation_volume=337;citation_year=2012;citation_issue=6100;citation_pmid=22984072;citation_doi=10.1126/science.1224339" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. J. Saldanha;citation_title=Java Treeview–extensible visualization of microarray data.;citation_volume=20;citation_year=2004;citation_pmid=15180930;citation_doi=10.1093/bioinformatics/bth349" />
<meta name="citation_reference" content="citation_journal_title=Eng. Life Sci.;citation_author=G. Sivakumar;citation_author=D. R. Vail;citation_author=J. Xu;citation_author=D. M. Burner;citation_author=J. O. Lay;citation_title=Bioethanol and biodiesel: Alternative liquid fuels for future generations.;citation_pages=8-18;citation_volume=10;citation_year=2010;citation_doi=10.1002/elsc.200900061" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Skelly;citation_author=G. E. Merrihew;citation_author=M. Riffle;citation_author=C. F. Connelly;citation_author=E. O. Kerr;citation_title=Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.;citation_volume=23;citation_year=2013;citation_pmid=23720455;citation_doi=10.1101/gr.155762.113" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Smukalla;citation_author=M. Caldara;citation_author=N. Pochet;citation_author=A. Beauvais;citation_author=S. Guadagnini;citation_title=FLO1 is a variable green beard gene that drives biofilm-like cooperation in budding yeast.;citation_pages=726-737;citation_volume=135;citation_year=2008;citation_issue=4;citation_pmid=19013280;citation_doi=10.1016/j.cell.2008.09.037" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=J. D. Storey;citation_title=A direct approach to false discovery rates.;citation_pages=479-498;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00346" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=V. Stovicek;citation_author=L. Vachova;citation_author=M. Kuthan;citation_author=Z. Palkova;citation_title=General factors important for the formation of structured biofilm-like yeast colonies.;citation_pages=1012-1022;citation_volume=47;citation_year=2010;citation_issue=12;citation_pmid=20728557;citation_doi=10.1016/j.fgb.2010.08.005" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=L. Vachova;citation_author=V. Stovicek;citation_author=O. Hlavacek;citation_author=O. Chernyavskiy;citation_author=L. Stepanek;citation_title=Flo11p, drug efflux pumps, and the extracellular matrix cooperate to form biofilm yeast colonies.;citation_volume=194;citation_year=2011;citation_pmid=21875945;citation_doi=10.1083/jcb.201103129" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. J. Verstrepen;citation_author=F. M. Klis;citation_title=Flocculation, adhesion and biofilm formation in yeasts.;citation_pages=5-15;citation_volume=60;citation_year=2006;citation_issue=1;citation_pmid=16556216;citation_doi=10.1111/j.1365-2958.2006.05072.x" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=K. J. Verstrepen;citation_author=G. Derdelinckx;citation_author=H. Verachtert;citation_author=F. R. Delvaux;citation_title=Yeast flocculation: what brewers should know.;citation_pages=197-205;citation_volume=61;citation_year=2003;citation_issue=3;citation_pmid=12698276;citation_doi=10.1007/s00253-002-1200-8" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=K. J. Verstrepen;citation_author=T. B. Reynolds;citation_author=G. R. Fink;citation_title=Origins of variation in the fungal cell surface.;citation_pages=533-540;citation_volume=2;citation_year=2004;citation_issue=7;citation_pmid=15197389;citation_doi=10.1038/nrmicro927" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. Voordeckers;citation_author=D. De Maeyer;citation_author=E. van der Zande;citation_author=M. D. Vinces;citation_author=W. Meert;citation_title=Identification of a complex genetic network underlying Saccharomyces cerevisiae colony morphology.;citation_pages=225-239;citation_volume=86;citation_year=2012;citation_issue=1;citation_pmid=22882838;citation_doi=10.1111/j.1365-2958.2012.08192.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zorgo;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Wilkening;citation_author=G. Lin;citation_author=E. S. Fritsch;citation_author=M. M. Tekkedil;citation_author=S. Anders;citation_title=An Evaluation of High-Throughput Approaches to QTL Mapping in Saccharomyces cerevisiae.;citation_volume=196;citation_year=2014;citation_pmid=24374355;citation_doi=10.1534/genetics.113.160291" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. Zorgo;citation_author=K. Chwialkowska;citation_author=A. B. Gjuvsland;citation_author=E. Garre;citation_author=P. Sunnerhagen;citation_title=Ancient evolutionary trade-offs between yeast ploidy states.;citation_pages=e1003388;citation_volume=9;citation_year=2013;citation_pmid=23555297;citation_doi=10.1371/journal.pgen.1003388" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="The ability of yeast to form biofilms contributes to better survival under stressful conditions. We see the impact of yeast biofilms and “flocs” (clumps) in human health and industry, where forming clumps enables yeast to act as a natural filter in brewing and forming biofilms enables yeast to remain virulent in cases of fungal infection. Despite the importance of biofilms in yeast natural isolates, the majority of our knowledge about yeast biofilm genetics comes from work with a few tractable laboratory strains. A new collection of sequenced natural isolates from the Saccharomyces Genome Resequencing Project enabled us to examine the breadth of biofilm-related phenotypes in geographically, ecologically, and genetically diverse strains of Saccharomyces cerevisiae . We present a panel of 31 haploid and 24 diploid strains for which we have characterized six biofilm-related phenotypes: complex colony morphology, complex mat formation, flocculation, agar invasion, polystyrene adhesion, and psuedohyphal growth. Our results show that there is extensive phenotypic variation between and within strains, and that these six phenotypes are primarily uncorrelated or weakly correlated, with the notable exception of complex colony and complex mat formation. We also show that the phenotypic strength of these strains varies significantly depending on ploidy, and the diploid strains demonstrate both decreased and increased phenotypic strength with respect to their haploid counterparts. This is a more complex view of the impact of ploidy on biofilm-related phenotypes than previous work with laboratory strains has suggested, demonstrating the importance and enormous potential of working with natural isolates of yeast." />
<meta name="og-title" property="og:title" content="Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1773" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The ability of yeast to form biofilms contributes to better survival under stressful conditions. We see the impact of yeast biofilms and “flocs” (clumps) in human health and industry, where forming clumps enables yeast to act as a natural filter in brewing and forming biofilms enables yeast to remain virulent in cases of fungal infection. Despite the importance of biofilms in yeast natural isolates, the majority of our knowledge about yeast biofilm genetics comes from work with a few tractable laboratory strains. A new collection of sequenced natural isolates from the Saccharomyces Genome Resequencing Project enabled us to examine the breadth of biofilm-related phenotypes in geographically, ecologically, and genetically diverse strains of Saccharomyces cerevisiae . We present a panel of 31 haploid and 24 diploid strains for which we have characterized six biofilm-related phenotypes: complex colony morphology, complex mat formation, flocculation, agar invasion, polystyrene adhesion, and psuedohyphal growth. Our results show that there is extensive phenotypic variation between and within strains, and that these six phenotypes are primarily uncorrelated or weakly correlated, with the notable exception of complex colony and complex mat formation. We also show that the phenotypic strength of these strains varies significantly depending on ploidy, and the diploid strains demonstrate both decreased and increased phenotypic strength with respect to their haploid counterparts. This is a more complex view of the impact of ploidy on biofilm-related phenotypes than previous work with laboratory strains has suggested, demonstrating the importance and enormous potential of working with natural isolates of yeast." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Q-OvFVorNSwfrAmEHSFVwvytEjFCD9iIvgXidU5GWVc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459667","apath":"\/ggg\/4\/9\/1773.atom","pisa":"ggg;4\/9\/1773","ac":{"\/ggg\/4\/9\/1773.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1773.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1773"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1773"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1773.figures-only":true,"\/content\/4\/9\/1773":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+\\\\\\\/yClnzTkdXWimOrr8gIlj5KNthSsW1FDteCuv9rPisxTizi3RtNNceSmBKBCvInaIR2AP7QsF059UHaTVCGdb4tC5rNbm2lqcQzFHeOv18X\\\\\\\/4QduMWk+MupGYOLp6KzIydj7O2KTIanYtwXKqjMKy\\\\\\\/YEHAnuYdbhaFqRkYsoO7uOTdAknZEmPdJGcnkjJp3vxKRr0VvjobkNpz3+995M04Xo3L\\\\\\\/rot1GDhGg0NfePz1FMdaKNP6QkLgfc3lt015fuLwc9G\\\\\\\/UN4Cmcz6KaTbqkLNUEC80vtG165rl4QFMwU68Hks9cPgPFvkZ2z+b1rp38vDhmLi+yEzOXctQwmCTIsKmt9kBfvxpj7IKZqtjYu\\\\\\\/Ui6hrqsxRUhs8q8pJjD3D6hf6RpjLBlD6Xl7FhLnCoYqVFAgY0bqqfIa4MsKLivjnv5eAUTM26aO9BY0poarMpTe9MNYgFrc5j7DHO7eIYQTrbln4KWYbCV8es+B1vG2E78I6lJgYASA9TzUqeDEldCIB+i5dH2tYMe3U1GGg7Q5mEar0RTqctLAkwwGTp5RAywB9hx\\\\\\\/Qs7DhLNHKnqWKqf\\\\\\\/MxPSwMMGbgiCr+Do68oCiXdsrbhxTq6oFrXalMJ8DJ\\\\\\\/ammMiikRlbGRqhX14kkr7TApRm2T9DfbyTg376TFyY649wUpkN7C4vNoT4wiVUWrtFij1TVtt+nboVD8smFVfA7kDVfkL+NVZfQ13obAr05dROvVLEr5IMS8BNy\\\\\\\/ZkfPaxHav728RVOlEXgmFU1V4M7TEdk4Qy4QMeCax8mbrBlpHNoWEoWjZ\\\\\\\/ZKKstPe3P9m5ik26p7yIyJVWuigrrDuQc0cYG0W44oH0sUAhytyttagdiYo1RYvnSCNl98HHPq3ttdS9Zp9cV7Uh3zOy5aPQsLCr3Q91zEqbU8hcAl0G\\\\\\\/nomWuUSoysrjWMSJUQU7dODKokVDEdwMN\\\\\\\/WRg3enso2HHdA7HoHwjD1oPCYnPGkohGQai7qO\\\\\\\/jx5729nrdsgf4orGKklPca+m3tdPA8vYI0l5qcDha2LehCUcogzeVVv778zK2Tq8r9H\\\\\\\/p238YhAZBzVLEUxG3EdDXDz0S0XFEO13XDlF6EbXvKVz2gV7ynFrd952fxSyMszsKQaxjaJx+R6HjSrFLEe0AAiqvq0G37Rymrhh2AtzifrWl3YQ0X02me8cG4Cmzcm1DxKKD0DUq9Uhv\\\\\\\/Y95K6zS5T5aCDZmA3XHwhoRwTGPwWGr6SgLR6ybO3ULuVpgFycnDoM4A3VMadmIuqTLBT8Ln1i+\\\u0022,\\\u0022iv\\\u0022:\\\u0022oVU9i4BpYPBBaEdlfyKAHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c90aea0ccdc4647ce233c561a79faf673416f1377f41caf15d16fd648a344cba\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002203eTGBIX6sumVxHHOyRbehQ7xzIAk4b4ZNv83YL0kKQtZLJw9qgYLKnjQERoqLeVCDKxnKJDx1pGoHQk3U70LPO6tPyohQAmf1lPF4tSBNcj+65HzGSYfGb7qeKrR4Eu+hjnVht6wb95BYqEA6vqfmbiLvbxthDJm6qFohjmSAT5bB2sBHu1\\\\\\\/WmrN0\\\\\\\/8zOYzS9PizDwFNq22qjV7XPXwKsDlHxYMkYa2sOe0z\\\\\\\/skMDxjeskKAawFzWpRmRSd0JypEMeEGPN2ghwcHpC+tIq\\\\\\\/iLcp54qsMjrICL73diZvQgoxr6DLP141SPB1av2UwQHaaZZpVs3fX0fgDWyqUYeeEAya+RFtZhYaRRTXP9JVIiLZ8k37kQ9MsbKsLCH2N0QimhGi37Jmbbj1pTap9\\\\\\\/72JVc6l8ySooAhSTBMMF08Xzcj6ot4cvNDHx4oZnqwhmce7db9HMMzCO5c4ZJb3lppfBr1VwhO+4\\\\\\\/Am\\\\\\\/Azf3N+OimppekIYJPREAXNV3OUpd6fkaCmkwYgAFDQawjYq88sQfaW1wuLmpzx+gRNvTdm5iBExgUqz2bHHSJ7tGo6eJZZRaDQAldg6rntTvdm\\\\\\\/IoZL09O\\\\\\\/\\\\\\\/4ePXeqkk9oWnhlBGQ4W8757WlLpQ9nWkxCxmyor2boqiq5bd\\\\\\\/zc1igT0R4n5rk3nA4zGvCdc4MqAam1psmAxpt\\\\\\\/Sccp+UKfeFLV+JSqL02iD8HjPyADzRuEQgDA2c2\\\\\\\/84St550Hamsky9KsmW8O2f8lQSzo6RCqc9E5OGtxHyYHwM9S04vgdVb9zM+o0L3oxNZiU6xnuPmk\\\\\\\/ONooCJlB+fmZaUrJdEjsVK2HH1ERs\\\\\\\/kLY6OTF2NKz+b3Efi725HYjc+DZxe7CLp5QJdGLdWJoo9dz1ehtZedhBYRunXIdJV9LOvFYyPNnWKZ80wSSCZndYGR0QKon0YQxZ\\\\\\\/rvnlXeIGFXZR2mKkYRYWiFzYDuBQsZmqDmL69LhgXO\\\\\\\/IdAs87tbfvysJYtLb7drtxDkVOvi+z+ZFQUs5Nj4vezvlyy+i4U4ZMlgrk7sdA2q8JJQZNhgA6NC3bwdJtNcdOojr2mA\\\\\\\/t75Hs0uApzbEgo0Y\\\\\\\/5FLXEVJ0QyEcNJGRMrq20e50\\\\\\\/1yk8I288+YQBQ43kfYFnN8zVNO6R7yGKnpHnJo\\\\\\\/M1hB8Pmt2\\\\\\\/IYVgJ\\\\\\\/XrrFJe2HRO3f2aTVGDFRKFscHy03I9qZqOK1JxCyUk+IMuID5gIlpVawRsnOuyM+WloWiKN8pyVtno6av1iYpktauC+HOUUm3+Ef4MBD7\\\\\\\/VWV4Iez\\\\\\\/NBOhzMvBMTQgdzA5tx1GXCGyetQd3iLxP\\\\\\\/yD4JYx2TF7W5\\\\\\\/eSckpgf+H\\\\\\\/EQalOQgTlDniI0REhvdvBmKKluR21nY0CVV4jNWFkkTduEVrnqDVCT4BJ9c\\\\\\\/hPxZJT9F7T1MGbTIgoWvFaGY8SgHEDH+JUekxs8ZMNY1JJ\\\\\\\/le3ZJllB9UkcLwaX+OAInuNYBoJIc1IDJ4CvR4iQ5JWFkmcI6D0F7sKAeyxddqZwPcIOjpOITDV1o\\\\\\\/8McRLiXKNqCfGn+LmbVDvzII\\\\\\\/RKn6kQInpMj0Cf69KgmzOqvOD+xL2QBezARK8sPmus3OhLg7PnEk0McuRK1DfdsX8XVBJV09Y8JeUoC0KyBe\\\\\\\/fZ6Fw8I6NxYPd9MKtPj9qxKsSiOHWydw1owT0zvqwn7Urlgs\\\\\\\/BO22A6zyUrANLeM0PV6\\\\\\\/9iOefNHVBqlJJjf7U8ZSJ4imTSaOpZEJ\\\\\\\/Ozw75Xv2r6TAwbWum1yy0TrzH2yyk3lbHOLOTYxsCizRiMkm0L6dRWu+ektMWw1hepyHlS8fgvOtNC3DFW+bCa69YFBIK+eizNH9ib9TNa4bVVKPu7fUEw3Sya392tRR4t7GCv08uD\\\\\\\/5qQTh3tyPlXUnE7aFMQTsb3un4uD0d2qluzmpC\\\\\\\/lXOQZOG9tslnjnYNFiHDe3Til0FApvft7PJ5526e3RgkZ89znBHivlJWVc9BGJbx6033IsroWNmuLIX8UN3IQXT\\\\\\\/mfu2qQ46sebm6JKK8Sbq7fGcBGm3315L57UgMuY\\\\\\\/\\\\\\\/F0PhywdPvb7+wpNM92zGjEGbzZtK3Sy9toS1eG0F+Y5Ad0j6rdr\\\\\\\/FMIhM7AAj8W9Dz4JIzrLk6NXJbxDDVR3i+7fmtkDR6il5R\\\\\\\/OCnaP+IFBpEeeioC3okjAdFcfSTcdHwB+870KBgqJ5jMoS7027XLzXY3M0Np3Cj8BW3FNPYVRWwl8OD5GbsgEIwmi\\\\\\\/862RsJuTTHaqE3TTS1m3rqtmCyGIci0CFvy9P8H1nPJDTuyYo3SAuFXuDAjpL+EdJqgffsCPPvXovyKjsjUPvb1xmBBQSC0cobuhEqbYR+iUt3eA3VpDw48lMoBolDzMXPFJN\\\\\\\/NLlyU\\\\\\\/iWVIFZK0sxxkk1VKs6bzsrzSjRskHTHOHq7dA9VbJIUuGN+ChnfBBWLpsWggdth5H+NaavqAnT7TEKOyVJTCZw2bgjWF4dm0qhPb9Vr5AuzIMVu9xgp0RCbZy6kCYCUJhu\\\\\\\/\\\\\\\/dtydnAxF2GtnVpvp34HBac5m2zMvekodptjRFlg2pK3JlVfz8mh1YgCaad4BIuAz5hzmn6YlPv7kgPKsNAXzLpL01EkFb5au3tJqlv6VChaeU5b0Q4fdsM8TygVN3oVNqkHOB3mQQWptdehqUnLnJ4t4TPhyj2Q3uTH7uqobPwzZfrmvBgGSKDO3floryYieqoo7Yuczv1f3dqKVZNZmHu5Tm0cW\\\\\\\/LayegIAwn7U0VnoPfBGy0r\\\\\\\/tYzpDL1Xri1g\\\\\\\/sG\\\\\\\/hmeozaAZB6ZfsI3NxTq+lhZt9GXUfqoVOE+5t+jQ5D\\\\\\\/5Zt0+Sq7EWqCt\\\\\\\/b4PRM76G2E81px3AFtm5YFUaJwLGyUYNaiZaZUFVzMezjc0yCUFyEcLctOeWmE97pymE5HMWZZ1IoE6DkJHpsXFmkPT59Zemv\\\\\\\/HXqtzRna51P2WIaLxRH80e3K+gJg1ipUr6W2xasEf\\\\\\\/hoHSnQrgFZfyvUGdHT7S7eYUDDKRJpZbpDOWMZVT2ZEjwHTxaxEUqfHV2u+Kp+GH0vtxm6BlI7KwzL9XW1666EzZpLfN0mQIjTkZGyJg6ioRGFb9NttrJ5BKAPpyP6xwADmYAZiwfjOrSDnADXlnFSyQ+ADRuRhTP1X0Irzb6k3qcb3Gc7jh5+mnLGXXflbXMXol28Et3zdZimK\\\\\\\/O2LfMn1hTl9qCRxcDrLXdKeNE4zo5aymcvaOrqRJQfeM9VkbK5wFcc06zSE9F1ZEQMW2WKXaqmtitb2PwZUxNOhOjEdttMihDvTXhA2RD6eH+42OYU0kg6DQUn5ILQcELW6ZRzNPv4d9Wa38nSDLQkJdgjFmaUO3OtUx6m0flQBieHYsDBPbufXnAhOI8dMPQG7LTfmRX5YG3ngAb3MbdYsZ5e00KPJOqpMdoBEWvk8529mYt0tezZK\\\\\\\/2yIVkzDpfky0S6deK5Mtb3n8lYA4R1Ps7WacidyM\\\\\\\/Px7lQYi+IQ1Yvd0p58OuptEq+miLIb++MdBsS9S9EqXxmlPd0em5HGGmVI6YPUqALMwdNa\\\\\\\/anvd5dB8GIvsB1fxFFvO1+gpTgrYZrJEXLxmanwC3IQzls7c6jtvipF\\\\\\\/ZiKbMoRwi6497k5H\\\\\\\/7nqM2uocE4a72RCV\\\\\\\/KG0\\\\\\\/VxqaB3Iz83v2t0v9MOKPu7nRyPS\\\\\\\/UL7RCbmueyiAtZTr4CC3HX9ZB8HPbIvRuTcydbnxHdLb1iJI43wE7uV7XxBJM6JL\\\\\\\/f0K2eW0Z9ZMoOubFs804uzXhrEMxSxsNeF3W6V6cQnZhB\\\\\\\/ReWFuJ5EjyYF2zQIy\\\\\\\/Zd2FTOyQ5dPy0IZo9O6bgARjB6nxGH984OkjrTeLzn1HNMHSFHqCGIi389vz4Ovt8wa27MjWu8ysiHBNmdkW3mBsrevW\\\\\\\/a1PLJAU+xz5NAa4nIg5MaRoAU4gEKscNexmeOSSgNeOvBCWZmoUfkeVRadlGTsfeHhTWmpsP6ckRzDuPmcj3iZjIY2O2R9JG2Xxm0yvV4DJgcMvO4rgxLCYg3BbiKqGtf9iXBD4uBIhWQbmJYUQezYfpJDy0U1Of1ZdgJunV6g+1zD03+YL6ryuWDhWgjfjnCiQEgnztmYm+Iu3hLn2mj44MdEEilFACCtoo4Lr7Pl53l6AxJtpvFcmVJXCrqsXvFnHOc+xIXf2ZWY0k6+YvGbl7HJh6f3VxJQMfgZUlt2+aDvzGalVXk8J\\\\\\\/e6IMidQYgMEQlnkVVBUb8jFwiTcsg7RxvpKw2rnjMAIa7pGpUO2sosh+mdpnq+FNVfqvA6vj6A9I6JwJb0r\\\\\\\/FBKyXa1oBDY6P5xbBRC1a5lJHZlEG4oAbyAY+bPHoS0897MyZG1yDkourKAzlZplwxB0RO+7yGju1y6PV0ly0PbYKbX1EOppWmrEhidrWHZm5o6WT8Tqg1zMePh9EhXeCggeCze5v3dG+h4PZ34sW\\\\\\\/URmRzrIhBgtUEFLr0KaeSq3qXjWeGawA7PiRDVn+hmkE5WR796VYFBLJpnAu8WdkXqPicdscTW7uIQ63f7qg+zAf2T9qN9Pgy8Ew3FivG0yuAi2HFwt2zaoPwHEueYx0gWH2Y7IZWk6Ehe0vmMCfMPAn9knmOP6h8FmH6pmETSGJa7jXbtHB9VTtUcf1akHuY10eb+gbW7EZQuSr8z9p+KhYjJUvPLqbwpb5hymDk9BdGQebchRiDnVaELiqLPiwimxJPNx3B2H8SdzoJ+Lxc77WC+uPIUHi3gqE4EvSBZYF4dSO+eR\\\\\\\/jyuzDEu+kFywELiFCatHWzDAzXGNmtruyJPu0USSYW70z1UUZkVNj54VAS\\\\\\\/qWf2htjpaYF7Mf+T7KLgzFy26WcrA8\\\\\\\/Jq07NhITGmOaOA9uPRMiDaztMbDYclyyIbCtKNqHP7V7DSRp18jC1M8QdK5NHXQCOFjyvwuJblI9KnIR8E6wfiBiEefqjZs1s8j\\\\\\\/f3LoRjIlGhErHAYADd27TKGNrn6G2l6ngE0nEgCpSqKaSfO2VpXl1iAGxvtYqLXtfSzjapixLN7Mf3XZZWb7Z0OPHofEMPyRHl0jafkIVlZ+h\\\\\\\/M7nvtIuK\\\\\\\/cVXaqJvzGgCE1eDeyrXK03+o0xzDY5KWoq94eYIXVkEKaTCBIs\\\\\\\/mUtbqytNwWmheKMyGU9l92svwAxZ5H3f6drroF9o2\\\\\\\/0wjw\\\\\\\/C87UzbTNE2Eg3seJocDI90ozAOwfViJsD7yaBQZDGKdmwEo00bjm9+WdDgwvTCISTEKt0zxMVuipsppy15yE6rXztWRipDOdFHDwkrgIgDAXFUC9qM\\\\\\\/NJgFq+gmc9jsV17gVIcacuayIN+8UzN+5DqxlXZsn+NTS7uWLWcMKuVGVkDXoF91GcsxZMjSnKWOUMYmrBT2swD2Lf6XraxHd1UzOuVHRjvoZsWokZSHD064AioDjAHg6gGSc5jCAw\\\\\\\/9TFnvP+M0lCMlxiu2mvfYHs98urcSczr1ISvBX3Eh9GYCLqKEvBU0pKd56ilw2cyw24pF9+c9N4NwKrCtR4QIuvEnh1Ok9sY2fSDiDAnAklIfWzZIP0Z93l9Ac8AHITKixeZSGmp1tf1wfCQcjRwQ3MtXeSceGf7U0ftPYQKbqgupjO0mpoSmdv+p7mUQTnkFZfyCsUvoJrpJN9iehqg+4OBaj25+f+bgcjEQFn26Zugh3pqWSLPJE0j0P0PzbgzmF3ardy8xUXGNffuOHCksMg4WVbQtNKHNMcztj7XOFyMFgyWc21Jd5f7z0fBgVrBmcQHwlsvOse4PTQ14zI7b8HtWtzVtjRAtG+QFgbIclLohjRXojwlbcR3Ns8UD+5pm+T45Sgnb024XU+UOktsTrLwJyO3s6rcIcRx+uNsQcL7JGiVkhk+I8iYXRlKBKfKwTNH62jrvj3z+u5BIUw4BU4HIjn5FrdrF0R8iRLwWC8M8wLXwV\\\\\\\/cc8TiD52mlZReGtvurXyD7pTKA+6KF9ZF8L4tr6\\\\\\\/pwGfP\\\\\\\/dDgQIJrOaYb7d+wMzqNnnZ41GDLvR8cdMoE1hQwO8cFlt9WvAW7bywEIHpQJkly\\\\\\\/dZECDTcxURRq+aGh0gsrLMdh9V7bv5iE+kiDYzsS0xyKsuaW19fsyWSz+fgFHINwRRBGGBOZxFZt34D7i5yKWAj9ZVBvTqhPDV7bGxeGHf9GHt0eFO7GWR5RvTrO8JijHsCZHIoAFAoowzv0TSDgl\\\\\\\/4G02RrQTd55lXxazvLNsDoxsf2CUWHIuc62dJy9AqSK4\\\\\\\/AcjL14CFR0Vp1y\\\\\\\/PZPp\\\\\\\/eplLL\\\\\\\/OPmny9tOt38JE2vAWILHXLYlS23nWateskOq9OaYxkXXYKxuvgsuj0d8BldT3jsVl7beF0lJ+ES8QxbKLQnVLJejVnRy7EWd3AMHpEV6vpjZgqJ4AK9Q933AVblA3vIGsODPZd6RBsZMxW4twtpf\\\\\\\/MxCtk+44u8qoYZH5r+yuuWx7D6b0IH6pRyQoMZDYAB6aE3cyQY2SwQfakvFjpQywmQC2ayRackzrTjBShV\\\\\\\/UdWHMblQRPOWMrnrDr3YJIpeS5EMIJwHwPq0amLRYfPForPUbw8HvcIYAN+DJ6SXpOSvDVPXRSh3OVGiq93DD5qwsl\\\\\\\/+ifGu88irOX\\\\\\\/IvAUrAds1is037ikMoY4zvYeD3QajO7DrIqIIkTWs4Q7AJ00AkQfxN4nGjg4Q99TABFAy6eyM6JECsHHXMy\\\\\\\/ESTMzXjXl0kljRgMHy1U7AMmXjdCBpYRM+aHSM6IFtdrpprj2EBQAs1u734eSrs3zGSmG6qBXzJpvwrN+GYGVmAwIoNgUZfpyUSQrWYo4sHGHb0VCqFW8JU7FXbTeU7Io3dMxpQWnyz7CsOePRyiJGoa9T5UlG3wGKB47RtSk\\\\\\\/LH6Bx8n\\\\\\\/somigwAnrqSbcJmqNfV\\\\\\\/qRCdrQCCX\\\\\\\/kToPDajtSYEPbUFJ84vrLGyoWLcMxh2nhdZiTVB8i9wavM1neJE\\\\\\\/Up3p91jh0kIVwVlnDrNeE\\\\\\\/ykfH0fPhTrQZFILqIfjkVxsMzpFQKfFR93rW7rpbgqXy6FVJa7nKWJxsbp4QWT+j3Tu8N0lU6Yd0gMdkGmeovyC0fMRISUwbGaPztcln0UKcetDN5C7hK+TX65IdN4xGKHiinC11D74pRKqxyhL2AXrZ2LD4eX6zHxbMZoKTu5fLSIxC1xd53g8mp4TusAkBd0bLcGIHy1YRfQGFd\\\\\\\/kmKbM8rtJKXP7uWeGGGFHSmdcAm+WFJN0ClEVODVHo8IOC9w7cgL+BdhDxbghRiN3JfMJpOpFCqSyfU\\\\\\\/65xm43qq8CLeU6mYhq8E46ZTEUB\\\\\\\/OBEuJCCBXrFFHYmEqG9xVnJnJf18LCbznW8cz+gRIMgMO9zZwIbC\\\\\\\/JHJ9kpwYe5D+obwy+n5FDHWuh5RZHAddWu4\\\\\\\/sovCjfW4LVaom6jvtmmegxX9+0FAl6ZCN0a0HZ6c6vMXzcs+dLoa9FzBT7JwmRcNAsWfmnkL9mGAXbRLlbFeVUDuQCccHfYQMcONL3iGVOOsg8A19CAklFMQmcd5LwkTAbb8POzSOHnGjlY+w+N0ejpd2FJwwLf92mgD2ZzYbIZyQhvgNpbuc8X0g2sVJ6MI\\\\\\\/3piggx2hkVF8XrqZh48IDT6+VsVc7LMMHEoP9ojfkYPVatH3qZL7BGht2\\\\\\\/hlDZLRFK\\\\\\\/ZwBYS6pxIdTJIDmlCAzDu3AB0nxR4EFf+JUAZ3DQYXgSKRqPgMhj8htBQSmbQOC92bOBrjtkZBmUcIyAHUU0neUQYJP48GcMsECcaf+5\\\\\\\/G49N8zUtZPqUumvAZjXVffFmjyfBIwuoEFRNiEbrDK4dPApMHfffpe7dqRqJ00ReisBldfEH7mVcdVHWz44VlysmxX3QxT1ETY2hxGa3KNzJgEK5pYU2GmE666x26p2c+OecUGNjnF7MksowZHAT5bDdp\\\\\\\/4fxX3fN2E5\\\\\\\/pZGpoCrLGB9Hso+uT5Oym+r48NbRIITjEHRFQTrBStMsormBkxmtt96wj7fw0w0ysHCcEJWFXWH5j3aJBLX8Zaglo6dR2hmtl30j0crGH9eWt7xnyQuZlwYlPdM88hxWyFdDhVB7M0RGGEzgdeqE7XQRublQG0ehr1\\\\\\\/J85taQLvKGCHJ3aQpQtCx4fzIL7brRFNL0Tko3rh8injX4ojRkcpgY5Ac0EVa9CEQIKvEg+sTlZGQQA7l9vfGk5\\\\\\\/3c\\\\\\\/gr5m9ATa3woaD44aKNkYo8SRCGD9FCtGgr2ovrET8GehmL89Z0wo5Zm4YGeYpn+oFwOLC7BO4hCHiAA29\\\\\\\/OtQRHnBt2jcBWb\\\\\\\/\\\\\\\/+3EhEooDEboV3DCpQCVkGK7RXE\\\\\\\/2ghQ2sjMY73c8KzM6dmPhynFat04fZaQOXuc7YpjpkdHZUsguTEIclcq6vMLeju0xf6zM5ZwCA6bkzxuiK2emuszW3iubNqww4Sprgb095hlfcgCa419Uxm7HKil6TdX\\\\\\\/GQx2cRQgcEy\\\\\\\/sEErrdQ6qzTErw41GCa4Fa2ZFc\\\\\\\/uJDwOGPIVZQfM2mS6\\\\\\\/mHsvOXd2VGzqOQoQ8CiZUdQ0j9oQH1Bh0yFDzYOhtCGItAahlGBTTa9ZNU3xuP4NCr6N3xXHQWk9+Y3mZhiSkFWK2TgNeHuCLFUeB8I\\\\\\\/eO9FpJFDCCeQuJsartiYmODV0jmL2DPcQOYu0\\\\\\\/f5yXWPV8Ex76CYf1oZ6XVGSLc1h69oDDxkLW3nreOMKPfXpbv0zdc\\\\\\\/Fghcz3BwQDX7Mb9E4AoSmg5xEL+UadqW7B4yzcQ07YMV57ubMUiX9zPkZvssCYYY7LpY+S26ZFExSnG7VcLU3PJP+LyO9nIs1ygLU260OKU6JsSGolwyFDCkTlaB3NEVx7MMzlbyYLsuv6cyCqFv6jkfCJfchyK\\\\\\\/8JLWEtJinmKbTlzcyBKpnNCoLgfVHR8LIPNzF8lzqwHF3Y9bX6kDiRz5MJEGTPtgobq03Z\\\\\\\/YeuUaqV70zn0YHodpH7zsro4VrVyDFOC2HwsIO2YpTNw1aQu3PJOVcYu3bhaG\\\\\\\/BrqS7k5QgC845Qvf59CiCNEOuvtqg3vWZRKCbXDTj5kUjoZjgahVnySsBTcoElKopmer+aa7vNAdA\\\\\\\/k6vFI+01JFDVGLmJXaMJH6holHqCUzjqdz+eKOxRzR0tkuAbe1I\\\\\\\/7XSL0qV43gPkD02fDJfHxlQZvb+jaljfJLrUTvO2OLFh75i5fzZyIR8p8ixvjTgjlCwgBu5RYUjTHA+2vRCixXR8PtjAaZ06IZCWMyZ9SBThUzrH4hmM7ZM3JRYy6RcFJKCUOr6Kim7YciPMYY8jSlr0N15ueGdGi\\\\\\\/xVr96v+wYbPEw7bqIwbQOyrgRweLjGxQA\\\\\\\/NykiuzEPIylmfQYfQoTerfiv6z15WAedz8ifzuvGhltcnpDL+418h2oyUWSM9W8boVKerL3KdOJE+HVESWdo1Bef90hvyKD167SQuGlCjraYjQh549depRcgYsbFcJxomSvDOEL58ivhVxBv1GaGU5yPZVa50Dzab+g0xxNeHPT9CJj4tTry4wSbRhFGmGubZ0cFzPeJTFFyoRzWE0cwuvuhV6ctUI2ipYj8eVqGwmrjZkiGtvWuKRwfr5ljJUQbNIutycPyfpzZauuaB8Ww5mSw8TOByt4fg4zNkrHt4LKmvUTyc0XN+2bct6PfLhJKOQsLR\\\\\\\/LtVeeNdlwKP8hi2eLl4+TELZfp8U65eU4puhYJ6BbjuniB7hPAGY8YYXqtls68e\\\\\\\/RkiQHPSzUBksw5IBW0rig7ztLvP7F\\\\\\\/X9udj2kCE8OZMV8x0fqFzck1D97LL9lwAdMz2KOWkEgMrF3Zmqcpt5b8LVPiDkL2VN4T9U5Wy3L24NSql8KQ\\\\\\\/EhbiGZjcASrfQ8aasM9o\\\\\\\/9PBHjog7rIao27ys5s759dgIAjEuNzQbGrQlt7Ue5aGigQ2vdbJozgsSDOQ\\\\\\\/VpUmgzZ1YpwEJUstkWiB4we4Skl4FupQN9iD2rAAh3fq+kGYjkiLLzT2zaTr3X99g1HjfTrIWSIrySlAR9qA1Z3NS+xNZ3pGQwXdWwp07cXOD2Hgqqb9u8cMd87P1StN9DKFfSPBQFkSmL8wNVCn0\\\\\\\/3g51FUBtu5crHGJesMXUhhfuqs61mS\\\\\\\/zveBNGFagLCaS0906Cwg8rnDL7GFZAQvP8Fc+Fb4bu8giogVZrDx3XELp0erVkTuByjBnoO1RbflyAhCFneZsC+A0s7J53LzLd\\\\\\\/fWU7JVjtqMIISqg3eYGJCI0JbB9xo1XMeq0OoMjfPPaUb7Bpw0Ow0WFggVPHaNe8yaB0NynNAeUG2MUAC6omj7Mmu3J6GWrLYJ4jFr5jKLMOZUzyRsskXWNERzpDBOxSRfBD1Ich+S1YYvcBXuoWlB7G7aEWgiv3ykdpAQSsNltTjlGr1fuQ26SqdreMs2A\\\\\\\/T+JraO7p4uDdqS0btaF9wHRjTd6QF+pxfKz8qt0dxOU1POIDljOpuuu2vuUltHqCMhsAYx6mxW9RD3D5lIc37CAuo0geMkmZKL+1fvQ05kFP\\\\\\\/4QE3agO6Tl706mWuye27naxIn6FcSlL0vA08uv2B4wNVbBho\\\\\\\/h14yLcFIStd0ZNvrdMQ9Nn1ol06gR\\\\\\\/9KSkWOLFAyPlTrRgM+eGjMKqLSIaE1IJ45daOLcg73myMPpxfUo3tUU2N9xS0\\\\\\\/r1jQiU1bueRpz3JxFLkKW+uNFp\\\\\\\/g1wMHDmNbjUDCGL6nB1PqnWqIpSiAyEohf3HJdXTiEmHsjS5v64N1XyMudJ8gDKu\\\\\\\/3p+QrTwR9WhPZqPw\\\\\\\/oZsDTcHq+e7U6zvtrX4R4TncOX0u4m98zDHjoaD8H\\\\\\\/KZd+IEM45TlQ0XxF+EReAPWWxUWVT6wDuKr0+jTo3FcwUdHAHluHIEIYQFg9VMlrNg3KxqhIi0HT4OCwszyMV02x16BdmKNHZoSyqdHqqlw2FN5CvM0pAJ6ioeH1fsO5cWaN9VdP41cRAnhH\\\\\\\/XAVMQ13CyZ3EJ5jGGeeNUtyj5mbk1IC7zUMSIm7ZwYJl2LjPIRulUvyAn+0SbaZzUeVXKLp8gBUeyJLbkHbo+pMYW1G+8sIHBXk1WZ38fHUsLxyEGE3r9aMCwDGLzkGqufTP1vfGuSONZHtQYCFtdBQcwJ24\\\\\\\/DgVRgw8ecOSa5yU4tvmfD+Y0UYUgpUcoXPUp7vVKXMCqUpCFuw8u99i7bgs08Tth89tc0lfbkyOW93Csl4JVxnnBohvTgpe7eklW0ExSeR8w8pYR4fi12w19lmegV6nOzIXh4Cq8Sy9iyJzA1wuNXCK\\\\\\\/40TZEdAhTo7I4Nswb1ah2\\\\\\\/i1kXML78l\\\\\\\/tRepHzbA+a5cpoPAqvBzSjlg80jDRvQ7hoH30DmI7PUmQk+Uaf4d6P+wMAXZ+1N1Dj7A5+BXIbMSwtkBVL3Nn68wDiMaQ\\\\\\\/pUMQiIQAh5fAUPSaY6I8eJoHXPRZ8s1mepZRQFOSF3eiXOtoaek\\\\\\\/c9X8yZxgrssxVnFgHv3qvNndyyaSfASa2ErFL1dPJUM9f\\\\\\\/alaWVco0L3pXGiQY5sj1XCOutLwXucdSTm1pKcGO7vQu6oI1ygnvSBQUpLYVM3wVcnieT9RWbcGd8zWCm6M3kdgmQVZS7VpvDjehWhHS3jkY9dY7FCgDEC1i+wRKSkWqSXZZ5XUUd27cK7Tn6UZib\\\\\\\/vMKCaJSuTYMOx7TwllgBGstkLuz5boOk8uMszwwmrRdXJ4xaeNNjVZYgTY2VAk0l88USAZgFgpFAjJb5fEtNNGLqQ98EJUBSp9gwdTcvJGz+lgw4wDTIe8z0YK0Vid0cfOQJSdZgulHzr9PdFCIZeevNeMjxDsyAwLbqmec\\\\\\\/8GE4MMErfDC9TR2J1FxXVe+eg06y4UgKj8ctPBYNVWTletoM6psKSzi7yvH7XFjp838afeCnz69zVBwgeADzSaaNFmjntwhd+T+06Q1lWc0oUaAcGDgS7GhPYK8G3l1VSpFemzw62EwNXrk\\\\\\\/\\\\\\\/P+1DNW9\\\\\\\/7N9FZTvrag7g8kGvcoAWk\\\\\\\/nEV+E4x5ZU\\\\\\\/b0HZ3ppXOsJfAuBXUAJXkEYFEvX8tL5rM0oC3r13dDducYCj7+eltN7d8nRr8u87bVa6Cy0axAoooN5wV+N9FTeuMeyhhLOFbvCDKKoSO84YKtbKaaVd278C5gBhAH\\\\\\\/2B\\\\\\\/723KPunUzIMhzEv1vBEj5++R6RFrbuXa+Z2v1e\\\\\\\/1sd9yuioKZGT\\\\\\\/ss65\\\\\\\/v+A7W9GJAHZA6KFHXIB65G9ebdIdqh9MVTfa078p7K27mz31hMadS7vGHBUwDOWRQtz1ALJPRKTrKh96tWlrraE4a1jZitgxOsYjNEU5\\\\\\\/Fdcg8Mwr9eMnTLzq1qB7iIuNY8IR0BdQ8RgObAaOtUXMzaz1ScdV2SNMmtQZ9WCYt8+9R9Dz6WvbJ9hWD8G0uvDIMcYjVf2VrvPfKAG8XuUoIdtJh2nUlrjoMY9iC+89VQnMzDyX8kQZ9mfS1wDw0T01q4p3Hjyae\\\\\\\/RiusOnt\\\\\\\/ebj6ACx95KvHzR0xCAeBg6BPFfuMbO5qMSspzWQ\\\\\\\/JQN8kSOMgm7nn6UVlwDGlG\\\\\\\/YNnF4AkLQ4Wvb5of5D\\\\\\\/napPH34D29zyL9P9BvlOr8nnwBkWgKSffHig7GlfkVaVsqw0jDl3e8nEe6AdpP7lFH3W7OLpZlIo+CZ1MRAvFACoPovo+NoNueGgXMkIDorRCFZFRi\\\\\\\/lKNhYp2nLhshSaH0oIcp+vHPVYvD7i7rEsOutuqAMW7+kvnomZVCH2J42vN+FpmiZrnIXH1cyU1rdYHnzLs1OQKgewHcsHuG9Y1e2jtYW+Uwjfrwu+mXaROehsGVnTvlhTfD7wstEqbaNaS2rsFyAQToxCY7bvjT0OjCtOtPnsLRpVFBqnXjiP2okgOZ6wwmQOUh\\\\\\\/1Aidef272KBizvuvfwSxiC90U1hAfvvcJMDEho18CNq6CyIpjOs\\\\\\\/HiVILTWQrHZqAKPcHbp5WhOuu93WBkNGm1bCysDGZzrdQ5ay0e3K6tlpVcqkvd5yhigJY\\\\\\\/FpPtpjYo9IzWfBdvbGxNqia23hX\\\\\\\/mlHzRbghMSJ\\\\\\\/ekBN20beQqlYP8sqOgx564ZaW2DON+5b0GmhmFjUlQQuRq8xcUp5FOPWkgvyJkQKjzcLpZDBRsMZ\\\\\\\/e\\\\\\\/B9+dyOMBy0BlyFMAWr4QM0pdwPDK+KPqnvFvJC2WrP+IE9dwO+N+Pg\\\\\\\/ogpPHP78SOj5jZAQIdAlj0H5ztu4BibEOEu2AbCqAOjEQydTFydgW6lNLTHaHbRIXM1l7oRoH99ormwrkYXt3\\\\\\\/eqOlufvnkNnYtBiu8a34fbqf5feFKPl79RK4CtDvl0aFSs7ha3YMZYQxX6mfymuGI2T3TDAk5pcuYz060JVHYFm2qhe+Qp7jPvHfUW83d4NmTmQFAwSM0AtE+zE7VFZAdiGpFNnWEvMOhz2+7CTtV0U+wTA5oTt4\\\\\\\/HjyyiynflZCjp+t5CraEvjRevl\\\\\\\/RxNwWwNBWyTZYQDZglpAHlUloVGxB8vT3+Z12Kgmoh64D+EemyzFDHI+kMymTchAsfa7\\\\\\\/DeiCpuIapXlIFj3ZRD3yfDi0I3P0SIrHiJaeagPPoCySINMcDfDHCo60Ip283ucLZvHQXYOpKEIag0uQCj6Og1TgdQRaQSOKIoNJ521qCkkLrrKNrd1oQTLlgD23kbiImXq1QP0LG9Um2T53MlV9yPfm9sFdZ9+nHiTF5mCFCqz1GGf9\\\\\\\/4ij\\\\\\\/7oSGATnoTKOcN3K9F1tnWNyLZCiyHEYe7\\\\\\\/CpmibIHL5SZ\\\\\\\/B2\\\\\\\/yMTePBs6uJfKFDnJ8XfBQ4XjwAAOuevl2p0b6WpneNuBYM4nwv415NqcbgX8B\\\\\\\/btN48TfnLflSxMKiuIUhjgXLNph1CvPLKJrFedDI0zRAOndAeVIPQZGT4BjXkDo3XsbMc8CD\\\\\\\/pIOzJjYl+y22Lf9GlKFq\\\\\\\/OlXWYiCrIdKHwAYWi2+Db6DXQXRl8Mwf9j\\\\\\\/IemTJLi1x6wVtwpxjg75eDavmj3WblZyRqu+MzsdY+ymIqjsZ9N5wp8h4YbtqXf44wj7WcDstS5cpdb7+KfUnRgj6Vp3zhPzsbOzz6nyOtrEnSLG4ILItmvxtszOvVnPzQesrw53eW8fiJe4sFzXA2S5nZG5380Oqx8Mv8vt\\\\\\\/TL3yaDdiNUul+bFL9wUY5cSLGaRawcgDxYUK5vR66ZFpZ51tmNcli0iEobOiE9SGxLEkFmzo4Bp6tK4sX\\\\\\\/d8brqDVs3vYog3IgOoh8HYSLoSNUNPH1B4LhSJxe3rFJfZzf+mxKV+bCVHxa5IifI5bnlfvM\\\\\\\/zZ0gHaZDc230HOzpc6InNMVSSApGk3G6As8ebWU7sz7ZiPPm+L4AxH1zQxi4pZGCFQ\\\\\\\/z2x8lTmx7lNX2KbqL+HQNl5EYw7pTyUE05wG78V6XgmFOsmXE3yrK\\\\\\\/dGOAnG2nj3JmGa0cMVZMOoa4+MSzrd0+4yGQZoYGA2EGWxUxaqivl2+6gw\\\\\\\/WC3\\\\\\\/fvIw0oQRzPpKKr7lU3E5CRoV8d4doLRRh\\\\\\\/F9DH\\\\\\\/03iQWCQkVfCtfLSWJUVYPdZvW\\\\\\\/njlQD3LKeT5EBNqvHiKP1oWrd1j5KnO1q40VQrsJvMgMcRy9ZonBkVHCVu8gYbMNQ+8ItBikWbvCPbRoSjXIcGbz751Q+wq3EabvEeaE05p+rktfThvw\\\\\\\/w15BQgf9jidnYZZwSB7jXqFgqHglSOSC452KlbdFXGjtz0lQ6sT\\\\\\\/TJjZcm5rr2MgAPCxGG\\\\\\\/kep6w1H0oJcU65KBPPLxSzHWW60mKxyNrWixoaUul9pHs6sOa+6CIwS67uMxrhZrMjV4mU1Kjj\\\\\\\/7F\\\\\\\/Q201xO2VAILKXpZt7SSldKO0PaBFcyCiCy+2uSoSIrb\\\\\\\/R9+Do8jeXe3aq\\\\\\\/AdUAW7u8MjQB7C3wLDIUXBVInHgRTFqNh9bfcT23bkrPI9IMoyBGKYdokaLcXxOWkwwfc6SnuenvrrOe4nwztLXCA+Mg4HzGBRvjH+5RTcQ+1oF6ntiWHokFW6oVar7GZEZGx1iHbqury9cyxg2DO\\\\\\\/0RtOJIm7w0T4OGHd5Joi9\\\\\\\/wy\\\\\\\/6pxJhjKBm7rJ\\\\\\\/asvdTgG5XmDuBWz6USR5zZdmh38rJeuRH7t8APjokyieYybnE1pebT\\\\\\\/0evWnSgpHtctcS6OHllW65OrOW0r1cQ95jPe3\\\\\\\/Of98xAUwEUp+C1ixVvae91Wm+EWSyKQ8l7PNP6mO1t8NOpMQtd0jCq5ABuVNIkNt6bB\\\\\\\/AsllEankeo6YgOXKGo93WcPOhtUky0\\\\\\\/WNXXGEctaU\\\\\\\/lE5w+tiYaWO\\\\\\\/uLYdnKQLNl1XFosujR+ZVvSxFe3+R\\\\\\\/cqMjFgSCm5\\\\\\\/7fPQNHaWCpvmo1P5piaV6+dJAtqXd5fCakwGAveAnFtEYhiSB6e0P\\\\\\\/cQcAnczXz\\\\\\\/nEycT73y0wekgShoYrOmhbnY5OEu4Z0klsLsxLmqhjDGP6cZkzhvubn7\\\\\\\/q2gsSPh0Is59qCtP6vP0bsdebNZBLP78V8f939LX8nuN2d0VkUyzpMaG\\\\\\\/i1Obp2EU4anPE8MzAwQZ4av5pkmWcEPOU7IwPbeXjzkUUDmq3YxD0FEFglEcdgD2+FGOSfaab73c2XZO7If79NTEljFYp2T8PBb+OAOKzj1Rp\\\\\\\/dRXRnk7jPFgjnolkOAWRrKrYSh5PPoxdkEM8mgA6wjBAyELxFqDreGgYLE6bIpBXkCBYAZYdliWoeTpkVqORat3GxRw5P4aEv8hTg7lnl3GU92vK4JONS3AFgsx4FwMc6C5v5xCVGat6neHsa9sHS0fRHZLN+LuUDcdICO2x\\\\\\\/rYFYkusMqQ976DobDYGGc9Fx\\\\\\\/N\\\\\\\/Y0kUd07fFymUfWOObyelydd2lmV4OTBsjf4NhuJJTs+q4gQys4fI+3M6lGjb+UzAOZAWNJ0EE8Izwo57jWLuqey+rNJLLHJ98tHJJKafbImj1YVHRWP6aEgpl8M1OoI\\\\\\\/KQLqjXXDEn+2K\\\\\\\/EO8NmiDgAA9yazOmlsguPGYtrO1YLgL33oJJb6Lfw8Q5RqLHmL5X3o4RMFLWdh3gOakvKi2smAD\\\\\\\/Z+GdJLwZZoB6D\\\\\\\/okubaHOEZzxxKTwX\\\\\\\/yQikfH0IlgpzVd+1ayAkWY2XVEhvw6EI+hd\\\\\\\/bAIZkECphet79ySDSBJRjBKrsUZXaOmVwZL2Za4Vq4viwGkWtR5QjaOyHPHViXEIL1cW8bw1bBlH5ZdTNQQ8wXGuDv+P4vsx75lQ1RuiQwhjVVvc+sBYVLtcSFmQyklsPNlZzIUb\\\\\\\/qh3H8R60QpXT2Ay3NzPfoYXOYv125pH0ZNcTgdX5e3pZc68j2yVnkNKAJYVpE6y+n8b\\\\\\\/DjMWvKVKC2FC1OTGgAKJgiM0DrE\\\\\\\/HzNPjd1jxM3xjQjpgo2itPAR2xgou+KqGfuZmKQypD92PsHDBLZE9qnD+3NrzHtgXNGgf4YgthhqMvLlfls0yNLOh4oTDaudI3NOzILoJmUPWMRykJvs6x9JX4XoIRkNcGNxReB7KJcIWyRXgX4EpRvdLqlFYAsDzs1zZpLFBxwe1FveX2mxgBoUeX56wXiY3BE+0zMJbjuF7r2J7PBHGSLpv3ili1gQIk5k1v9+GKyBYdNH8Zp\\\\\\\/LS6\\\\\\\/9cKcCKP\\\\\\\/q75duSuxd6755t0mufMcbRMrykQ6NDVJ\\\\\\\/DE35Mh5lvzQbliC6YHTUIAXHlhdXbnhpDu4kVDoMKWNp0v1pCyMICAaJjfdEboxZQLNgABSn9oIlk7vZsr1\\\\\\\/oQSf9f+zolYpoYXaWqo2fwhduyLc8pWZW9IOCjXev9sbFDekgsjlS8wX7KmQ2Xc1agXllryQBi7NFBVBTwQl3aD93YgLABuVXbtgUMAeO4FdyUQN31SPftFXVbM5QOD9ryx0TbXCIOqT8SUyedbq\\\\\\\/qqyFirhtNtw3+sRtUZbOdkLiAhjOFBrqf\\\\\\\/XfY3tak+6poUhM5\\\\\\\/pEg53Z0b\\\\\\\/4a\\\\\\\/CCySwZnlguUGridNzJ3GcpKGuYNGdmLlbph9aY9PLHGEB4OZqTKYPhwfrESHavyJqZwVhISjFSSBHnDHXxrxJKhV1D2uQAKEINMTANhmyzXUBADx7vkeS7boi6RdYDeTJFY7kNgDAQSakM+1tOVAjfgxNfAOnDFO9phwRfIntkCmxJ0N78jQJ2c6avhHvd3V4sIE220irmg\\\\\\\/hw5WLERmocPtVW8Fl3p6Us8VrkyF8qrbBPVs1H+Pq7qBMTfjnuLmS3TUwaFzMfhubwELiGaWMER2kFLSKL8eNaPzhwEYmi6ahLjwV3su5sydOzghMIqeA8Ct4xlQe7CwW+B9ArhRZg9Q9edPt\\\\\\\/4bI0e3PTvwbJh80XZ1PeVGC9RYHnmN5qo7M7cWfeUFzWWGR8eVHuJ+WIsy1cIAw+lMUq8VyHVg9UFdcBoPZuqz6dlBZHQdrtlDTC7J1PeYATrLoiEvGqRAxU4FI6HeRTO+1KkG9f8pn27wRzCN9PYFhSdWNg\\\\\\\/bKbSyrQ7EHdnYaGPwTFytdFrbnYa3+BL7+2lAyG2he14wa7l9KagaVFazrjRA4XIQL1DSNc85hrG\\\\\\\/qXlDtq6pzWKQcgdOqgGkS6V2zRwPmTO4kSrkj7\\\\\\\/Ans+OzTxrZJI4fx9\\\\\\\/QpIUHMB44NEVrV5iMd5X8X81eFb65UroWkveQmZpL7HFtebX1hVchp\\\\\\\/7xmQMWwScF3JXdO1RzCztbjznInIncN1UZbv7zrUZjncB\\\\\\\/cB3BezkI3jlgOcBTWzwBUNN8W6V2bnUMZ4LSRhYEOiPBi1pHWXVQwUGVaVFPKVcXcSF0RytLvQJEVuLRDK0Ytrs9AlRHMFkSUi0y+LeaUoTSlArcxWyINubigAJdrZTdbpaafzw6xl3zKzgCDTIIlOc5DiiDiqU+hEpUcnnZnxkbeTCaTb\\\\\\\/9lmZKKHOz6zp\\\\\\\/n4U79u5GkGd2MalkgZT6kEWZHnE5TcUUyb7dk4YEU+EzI4OC8iaYL8c10ag2Zp0o1u5lyxEdIRqLIDDtTxrkIBdN5SFI67NujfDOoAlP9uKQJaPWW4fq6hZFEQNFlGycY+XPNZ\\\\\\\/Qmd5SFjXBuqrw7EqGPB7SKxX8Tzc\\\\\\\/Poq5IOy6oY8lpEfcdsWC0aMly2Q2nixZN5HKX8DF9dYNijZx6mCi8acrkHRG7hxmQQoghdKjWnnV9fprZX4nS9XNQASEEE0XcFrCcdvKBgbqb+4ggijOFZ0SH4wAz3xlvUnya4V5Y+eNNltGMwvpb1w5SCe97mJNCYbDIvnEBJGy\\\\\\\/vtNpRCch20ol3YNmQwPjv29J77CUcqts4pLH5xNxhnoXKcJU3OnX5tqgnG+hY3QZfreTF93fbm7EEMyqldcslewC4BObGRhGHXyShAQ4jjgig5stQQp8bGPKHLChbafNZ651Dx+wXkofBKm66WxcEZCoidWNfxsMLEHVAYlDaIkJotl8e+NZKn01nf7j3CeUj15oiEQx8wtLqNEmjkE98ERoXsUeIfjhL7NPh9yLsRF+UUg+6A+AS9Tb7G9s4jus4ue4oDaK3N5NpWC8I2qVcXYD324fWtLH2myh0A7xcjT1i1EvYLXdZ36+vRfczTUwljDfVOSCHWuMbe\\\\\\\/xuM1mOLcxGw6Ke7u6yNd1dzOwL7OZJnwORfyPUC2mmXQNH9wlpyh3M5vry0fOJ+OELO\\\\\\\/4G1awb\\\\\\\/\\\\\\\/QST0rqRqtmwadZLh3FCmcx2v2RvZRFlTAy6jOPodFNdf4jOAPQaLAwLHJ25ltVMj4Qvg\\\\\\\/9FqGuvFGOrV\\\\\\\/8OHNUUn9Sq+l2VJTX5mIK0MIdfVV2n0src6Kidcij5X2Gobkc60O2XA2rS\\\\\\\/7LpR+o482mtIP3APG6s27ihOx4AOr+krC6806fhCqXWYqTxo0PKuK1pRkfAqRQehXlSjV5633kO6NoxbXuy7MAG4KDYB9uk4faPr6rwYFZpZx2HJ\\\\\\\/750DTKmfwZ8cyGeo6TKQZWiWEjljOYvFRqGKMyWxdxMDnswMKF7XoJKNeCWYS9Q9UJ3j56FYDRA0+wFnVi+gone4kwd5pqyIq6ltjq\\\\\\\/SXZfwkSURVs1u0CmFDen8senSAHUeQCfzwR1TozLngHw+2o11C6n+8W+xvokXRh1A2dis0lnV3zBTvBUrQwtr7Pkqcdj8fP4wr0mdxih9BROanJntZd2yZkg+Uiy8Is7uIWuldpp1WB80Ag+wSzcsITljRUscSsrvu5q+3eKSJMQOXfGmIUU8mIyyRD2y5UEOzBtBwiGvJuRfgw4gVmgfYXBQ3sofBNJrdzHSDdDwHJr5wNQywzwSqMuHv0Qu5ugXYH+sPb9d3yEmPFkwLvXYdXBBBSWD+IFILjHDE68DbStCEIDEalfQ6hPmZucu8PwROLMVaDlws+62bK0TU4dmzDZaxrlYcT27mwOcJYPBOPSbnrhN6reT9DjRNCA8T8YCPdyL+a4VUI6zek3dcVnyT6uU67z\\\\\\\/l6JKJBwbR2krYcM734\\\\\\\/cQTOtp2P4NOVIvw7PsI+kNPJ54gBAxC1lzaggkQYfM0mw\\\\\\\/T6uxTwCCMyyXiBWWhmfmkJbieC956n81Ra8gD3f8oP1hVvyGnl3LWqPdj7CGFty6GybCTkd3bApo5eaYn8DjivThUfGmk9dlJmb1gqhdhbLjRJQg1Efkdh2+8MjV\\\\\\\/Bx8EQ4heP0TmE7IQAdaG2b1HcEvO69rY2Pom709gEacSZkdXtGM45Jh\\\\\\\/kjAlL5hZvH3ACc4QJVvabGQQBfhqfPgm6ZAbKOojvFT0PJgMHAdPWiZvAZPuxAN04AaMXhJOClC7\\\\\\\/6AGvO\\\\\\\/cmaLdlDTRMObePK7KNax9Trzm5k9VTaoosFYyCUoa5BD8twp+oo+B99ufW77OO\\\\\\\/cAWmN3Vaz78zjqCvYqmFky5zfn6Mrxr6rB4OD7oUc1RAXn255SA4KbSRLbLiKvMxLC73vmQDKYCQefkZpAxeVMEQKdRDFXl8wZAV5ORFqT3vpyPJ39T2bBFXO6wuXh04gDRXtvIw873RuctJWQ3FCOnghPmFBDQqo4Ne+OKoKUrYht0fzot2TwL4nrMvZpBYGW7Ma9n8RYBkXXrYoe4bUdQKryqLM3DfhhN21IC3Y+A9GrqCJzM6A9N0scbwsOxcVnWuQmKXFGVhdWFv8Elik+oi25ozTIG0Kzeayz6Y862f1QVf\\\\\\\/+gaxzL415g9rRCkVdEoXE+2k08w+MH7h3S92s1XiK9bRISUV36d7OzdXMEa1KnTuEKiYWDURG4ta5rCOGhp4c21QVAKMeOhCbg5QGuZJB82BENYOZIw\\\\\\\/r6s\\\\\\\/jf3MeQXW9QsRj0PI5OyHgMQ9flc+j8d\\\\\\\/69hfrfjU9jADehZzxzpgleyuJ2VDXDMw+pXCTcNg1nQdYknaN1Z8\\\\\\\/T35l5pUfiToHRKBM1r1jJ1TIWT3TG8pG4OpHVZaQ5iofeC+B+Jb8ERKi6Pg8GIa6YBVS9PfR4zTFzJ\\\\\\\/+8fDbBK6pkpvWwiBMBXC3PS2A601OEcpYiasABXY8M5nnw00wfVHEo3vg3O6HKT9+BEvN6iUU+VOwLURY3MZ4fJ0CUTnjbbx5shqEBCtLD1tkSMm3FEdyipCZkOUQpqPqtxUzlN1\\\\\\\/wH0YTaiiMpEbl5HlkYA1S3IdIToch5RmG0p3kPYtFWlNorIpKvNEjOU3oN4uR86+F8k7RL79WN\\\\\\\/2ojzsrsTzwJqnRJIipmQFWtmgOm0Kd4aYFUBiwrY05olMQgM0gqZ4InkZvyAr7Ld+JhEOH3dDXt3uc3nR5KSdyqhgEVzknLHgz7Jzp90A0k21ze5vnPx1oBZtz+nVMSJ2YVhfTeHbz2kKnEFERxftoCpp1UkXwU9ApK6DPE5HHJT75rYjxZbnYIJ28sU5pndE90Q414wWE+845v9uvwmXjYDQoDX20alEXODsVO6qAE4+CITKXRvapSX8+SaNX1JhYfYxx1FTWBG6csNfHYjQzIhGfk6GBx2ET+X01cK1se1m+2pKdMJNvgQQMwIU2d2qj17Lyy7B1Re8fRgr5gD8iPh8DNfvUMFxNZz+CRNieE\\\\\\\/sJugIMdXbsM4Z7MH4XiTS+v3EZT5iSJAcjh+IE76f8ViBEDHaTDNjzqYOPHS2J8kM+2CurqC7nqWeiIBAphV\\\\\\\/OhSAb2jGSYs7p2NMgcjzaUKzS6zIhekclaHsrBjgFbzvSKq2uPWHm5m4f9iIXup+yR\\\\\\\/tHz6wvoj6D3XEwU30md31YGxio5eZlg9nsuxiXo89crJQ+u5mjyE7BYf71+30J2zgGTgDmJiIkcQTmF3slEFMnfrOrrrymCkt354kh3U5Up5hT\\\\\\\/FN1g6apLQBGhmPCRmQGrptEfN3maCOTtlw6ZDvR2BbZUeM51RajL0BDkQtQESheAr7\\\\\\\/QLpzY6Hpo++z5bDS44RlT2JLE+y\\\\\\\/\\\\\\\/Hqj25dZmCSrO0\\\\\\\/opsSOvVRsyFFO3+fgGdVoKvm9cjcIyPon4rFil0FGHdYTJ+1x+HPhe++kjZDEOEIyMs6N9iqcZUk67VY3YH5ktc+qhxf5bqnolKXtEnSKPZ8exU0w1iFsEvJ\\\\\\\/BAGseDH\\\\\\\/iCLpMaWdLhiAD2DUQSsrPNh9LH8+ctBRGQ62kQ\\\\\\\/6AwA36O5GNKBkfPtKlRWKlXXaTXpcNNeMSLwn34grqo0GX2fF0gzqkvd9lFFeRLWpwgKn6g3o35dUwk5Z\\\\\\\/QPVe6swOlou9mWE7S6VhAhN14esxEW4rxKn1LNdtnM0f3OjIIepWJJ1C\\\\\\\/i\\\\\\\/LboaNVnvDgYRvf\\\\\\\/CnReJ8cjYliKP3xnKaqHlKUyp2oUKXG7tJtBPZFq4K8YFGxdpsWuVlcffMNlrbiMqWXHunovHHYRm24m3E5Oj1drHr3UZGs8wXjrBKB\\\\\\\/VhArN6md\\\\\\\/eq4OxKRvaiuzVEVq\\\\\\\/E0g1wJKgG0DN+QoKDQZ8Jxn7RhBX4seRImbBDCU1P85\\\\\\\/VOkzjY3MHNG8pzpbQWcq2RiTKIas6HcRnsqo3s1JW\\\\\\\/GM\\\\\\\/DFd\\\\\\\/kgB+CFGHPrywKJ1JoEGJbzvRGrNP4DujBXT89TWRKbr3Vc035AAhfvTCGXMLH9X6KpJ0rPeTs\\\\\\\/H\\\\\\\/Z4YcpVVOFwqgtfjRLNXHP9Zlbq9B\\\\\\\/U7pNpvB1hFaUG2QNiMuQVwLhB96NFbRb9hK502vjdbuiK3Hhy5a9JZI0UOQwELuYakYOFs1jIQzzS59+AkzGwKWL2ni0PD4L5zG53RRSUVNRb2dejC3eq+G3zAiWfQ5StfftKl+yqZPsnr1flEJ3O7P5b\\\\\\\/ZNwUSRLfmqvnfmaPVUjmi98uKT4XE4vZiHOJ+wtkpXBGigKnx9UQuXIYv\\\\\\\/YLnj20p4ByAPkMPKoGTN28IaEwk8gLxUX\\\\\\\/AZ8HMd4NY+VJlKbUSKa8nHdE8vzuXpcU1Ya+4w8mbwPZ2tCBr2AK6yvdkRBv3hMUr5ZnojLH7uGFVURWcKSKZS77pRhziXtWW8pviO+GTdYypExAL+VkTIzvcuRC5JA9C+7GRJKw53gXmkZSmCGDXJnQLc2KYSVCLkphpnbMfBAvkYWUUXwcL4k6jTt5YXmx1ommlm8uDAAUL7eubOM0x9gkbzvj\\\\\\\/IxlTS+fgk9kRH45E8kyp2m5JFMHS0X4ptq6Q2jJKm4DQ7775\\\\\\\/rJh2gluCwumolE2eo8wocgBYVXFQSjC0Cs3U6dshrcYEzr\\\\\\\/ysiCTZKEwzHxxinenfghvtwptuzCaZbciYMKYanBVDeudi8mDqD\\\\\\\/pvJePPGTJ5nryjfXDJVEip7iWJzj6Uqy6Apf7IV\\\\\\\/VDjLCijBjMYi\\\\\\\/B41Ht19LzjOT1aY8Io70uJykZ2THvw0K+uL4W0P2mvUtaE+a989yVGJxky7C9o3bqfzl8aWZvwdPK6RvEoeRTqHZSlyqaBPXq+iTmF7M6x8IBoZW6OeELYWPQTdN4BUn1OwOPev3pBLmuxa3QsVusYpCH8ayZEAnWDI\\\\\\\/EMSaOL9WR1paYzllpUCV2INNYf4CNBotdXsWRsUt8oxZVXMNu4gT08hTwp5lWewk48XuKZbkjFFTTIsBYIpOsLiQpP1kKb8H9itHL0pj+d3P+BWhmkMbMK2jOhIEgfv6qZmj6hbqoVGAUVwlBbnbW4X29oIZ6UjepMS\\\\\\\/BHKR2vWaGYBhVeF9zF05P3fRIp9qLZmFNw\\\\\\\/sWmHgmWR3PI\\\\\\\/gE2ttHC74EELxzcl1u8dPlVSvIZdVlNGg9zDVHRxj9uLJBuxFElV81rOz2I8Q5IdfI7bChmuNpCoy6NDFzNF9oKzMXc65ngtYQJ+4QPHPbCw70J\\\\\\\/Q4KJbRU\\\\\\\/SG+CTy7DF0SFQ3DCuxKuF2egXDbhGW\\\\\\\/PbcTCK+aj6e7+cezpTzY6kMqInaijfZ36CsSq4QatsT\\\\\\\/vtLSNX3j8Wv5Fee\\\\\\\/9Gc2riR6PyztEgNcTSE5wqJ7ZPOHu0eZOsY+I88XL\\\\\\\/IkGKciIjp9s0APoTVssFvf8u\\\\\\\/yF1SlsGt3cZFSIAQbADg7\\\\\\\/5diKQl\\\\\\\/COE0IKsC+CPP3zg4JX66nghqq9aiTWyGojGMXBNTX6mKL6vU38xSqZ8vN3W1rwuN34hsrehH4\\\\\\\/oWqbq+kTZkdEZom671tjz4HSn2f8icdrbXRikTM4LMv\\\\\\\/g7KfmLouANcxJtKsRd1KlP9BOrwxu1GA8Iwb2xfFG+TsMCP5C51d3h21qGaQeEt7pjeMS2uh65dc0t5AD5x4\\\\\\\/nMVFbSjs7ufv1nNuD8mdlS9WArj6I9NUxs3LjQ4Lt6V+MeTe\\\\\\\/xje8NeNV2DRQRnRDfdp5WrgAR1nNpY+YHNqwYW2uMuKupNY29wZW0HE1BTMuJx\\\\\\\/xta8QaZoPMou1Tkdu4775Dhl7ly577UZD6jr+U2X8knRW3WUQcgqVkVxgsojVOFdAgNLfnS6Bpadx0dOrrxfeo14Y0rYooZZnkL9O\\\\\\\/r78i+bqfymCz1KZOglYCyDHqV2kMqTmAIU0BuVh5NKuSW\\\\\\\/leGLNxXOZMrWSrS5amLI92TmOgM7TICBwmdTSO7wHnG4EWTRnDhjg1hgjdGWQicRlQTiqqyL25UmLF2ifw3qfiV8F9KyWuiblAWRbOxLPetoYXoqtcLKbAAM4sVNxa5dRh0+Yv4IJudF3K046Q\\\\\\\/kVXKRGq5BR6Y0l5lCIfceAcNQ3tWhtgjleUeCm+1Lv8KRqYNY5aWOCQO0pt\\\\\\\/FsjoEt6GS1YB8fV4JIacmW12lAbRn1AHF64+H8A2gz4eu84sYmvym2O0doiDf6U+g5AsV6BEbw5nT8+hEt5+P941\\\\\\\/Vobr2wrAOwKyRfj5kAvNdwOvgA5smKCQALyw23fAm9lMcfGIDx9xTzczup5Oft35Kigv6OeMvoF5b+jdw3yxJ96jxElZhZ1YUi+OOr3JKbiOAmMInb\\\\\\\/dfOP5+5bsIPVCMH3qWkWVv88nzbmV35LPObE36LxVIm2hlK24TO8IH4yyc1lqu0F0wDzpzzmN0Ist7iUutrWbVzUvIdDa07q5aaKAb91K8+nj0+IXVkwQ53RBRak38gMU7Z3en3gUBmySVomj5R3ch81MY6XqvjZQ1nbY7pHeq42VY6DEnoeHWiy6CExxBUC8EFT1JR57BFs8stuk1dZneJfnzewFeMZNi6a984BY3+JSW0T7uFpPRhB9F8Z6iNeEk5YPCOWML+N\\\\\\\/RlBvqktoaAfK+68svthWF9hXFl2CiPoAIczYJODr0\\\\\\\/rqeoFOw2RYavd7nAGBwzzOQ6O4PMWVG9k7V95thwfaoHsSZ1c41vkSrcf\\\\\\\/CrGDamp4i49HQJaSJBG2spJ\\\\\\\/A2QwJCh3KdZgzP9DpGOrr4QRVqrLBwNcfDsEEyqDebpwnlGv42P0zGjnhL49AVUUIb0WGKgXUhsRneaXk4lhKiwwpCYh1G9dCML+zYtSqDzX\\\\\\\/FBmKU9cC2De3K8PR2fGHehOTxPL2y8K0U95M1LDuN3DlCxijFdRX45BB77Mo1GG830NpM1LDZ38dHJbTto86+2m+Imx0XjoYjVIibG1pClnDwnNuJ3Dd\\\\\\\/PA7nB8HqTH9BiuMtQjnkxuBEhGl9O6sqQWEzdAjDNMoa3vanhXPNa1GoiIdivDM6b5gtKF6I3d4I2SEKTOrx2TBACwM8CUFYEndFyzBa9e8IsnCEYD8HBuUbICe0SPVqNYnPvHZP3iGE1WwVLLRJkG2pCDU51HCS1DB8DWNqPQRnHndfB8vqRQfgC4dKDWZD2jL8SvUBwN\\\\\\\/q8BJpogiqNUhFLx1LZja7rqFJn1Hm8TYYv6tFcXgqSjxjO\\\\\\\/AOEigrsP9nnAYFlpgNTCn+dOHw0Uf+34tNdzi9NYPL+epgO01IoAE3Rh+1FPW+P0zyLAM8CHCcJch8WgWGelw75370CvxQdgQMukz7vhBeJdm\\\\\\\/Brv6OAHRM8PvZXnaF3Ote56wagbwtYgkxxH8UcBUL+DLa8YTHRQz2MvlbTOVhGXK1FtXMcoVrKN5wLf4b0l+XCp+j\\\\\\\/Yte6p8UversD4\\\\\\\/O0llaXj8+ALLE+wR3eA1KHwyX4N7xi6\\\\\\\/MOyJSf24v5WuXiyaxD4Ynn8IK8NW0Vl+u24fKrzlPEmA3aPxulfuyBa1ivdkFfdpdHJG+uNRNrAbOXbZRYBiZV7exO2fOHBq9TBFY7JsnNO\\\\\\\/qRQwIpKsfO8UwXj32SuCY5GOQWHaRkfs03+JNS16rsFPo5BD2y4Zj1+13n4\\\\\\\/RnjIs0MtMRdfIiN2wASHGZ2JmBV24P10KEDxVeE9NOtLzIDvhi2eLwtoG7rJ2pO\\\\\\\/k0UHYFWNPET4MD0MI4l78xtF1ntHjaDpe6hFW7yLHzf94gzYhe0i1lBP\\\\\\\/1JZleZaBxSiw88ZGDB2i0P+sS8PS\\\\\\\/aUOg+KULYDfFUT\\\\\\\/dIXpJ1npU90XL8GpP4mp7rAhm81oFbxbWoOrtJugeaFOMr1AxAEUu2mECGasllzYR0u9vM7L9lSkL21\\\\\\\/dmxr3hNBLghA+mDm5Sq4yoW4Zl4nqy5IyoYajCK44vdoLZP\\\\\\\/wFf9yoAc3c3DXUk1kDXNN4kcHo0wNfGdmcbd2C8TOeOTLhnWab5KUG6yDaLvp+rx173yBA6YJ6xqsLnssQDFUoKZpeJjfptsdo2Exq98\\\\\\\/t5neHrNFUAj0\\\\\\\/dNPJqxwzG42INy\\\\\\\/ln+10lMNcYnjzCfQn8K6sOEuxuRaK7hy6QbUeif4JMilq2PHBgjqiAQldMNHst0nLxYZUhOwWq0MuupqmyKn65mZPHdmiLDCIH5axJPjDXVVWtGV9t1BYSNRphvXCS23p1Xmu\\\\\\\/oyZitjhBWfcoGx7eRYkA2Ch3oUaCZKplcMnJ5npyCrLJ2HP6hVNI3hjFkZTQh+YcuTVHLWMTij75lN9AoyTiwGYsijYKkULa1ER0Ot189+NegbwBZqQJiCEECJekG3ZMBjx2uqnELG6kjxuaAlBtoHGLb2h4EvpAA95+5APubbLgfmy+j20r8f4r5nTMbYGlly6iR89aa\\\\\\\/iSgy2FzK6IvPeCtQ8UkYUn4hroismstIF4rTKFYyhKfJ6i\\\\\\\/9bcMvwOHToGDauppBdvmR\\\\\\\/9M18r5f9AhsU9yxh5A7KIjfPImsHNqmsqwzYi3SIOOECo\\\\\\\/25+EkWgz71BX72PPlKxYrf4gi7CWfrvDskCBn2Q2X\\\\\\\/jSru4H\\\\\\\/xs40fb8ZuUQAUdiI7E8mCOWedlKYlAgmGcIkdT3FCqhX7yqQ2qSml7vyQcp19rTdTmT7hW7qqJwz\\\\\\\/pyvD1kbpshImEWwsDrLFHxUmx5o1QGT51UHKCzfBItNpn2y7Mmxhyhb5O\\\\\\\/WUEywElyH1PIsjPksKMRPNL38VCNEN9JyuLwzWMg8m0cozLPiOPYKX6AcuA17tSMDV+97RTmYblEu3qWEv+0j1zrZVl0pjU6Olkjdfwi3hSYPOnZYpUz4\\\\\\\/ekF3Svya8lqVUY+MRzEGAKaJZgw3PT5a25CbZFSGxfkC5\\\\\\\/20t3Hzl9KzB9Aew\\\\\\\/\\\\\\\/DuKjK1cZ\\\\\\\/zTlvUecDAgfyzjd8jt1OzjhRBXOV9FeQB7Pz1UieuFBP0BKCleS6BfJYv2\\\\\\\/PYy1M\\\\\\\/7ashNgd5ENuQflTI3geGK7N5q3qbunx9N0L63VE9aLMAdZPri\\\\\\\/F8rp5Awx4DvTU61brnVVi00QLRbtoTaecOQbsmVeGACZRsF\\\\\\\/vX\\\\\\\/3iRxuWKny9vJjyvEN\\\\\\\/tBj9rOgSd0RGLaaYzYKd5Jlcb+X+oRAo6BtyXxUgLIbehd9+JIYBOo2tq7YH1PC3h0bCFujs9FFmDj\\\\\\\/PTUnn0Q52EKivf+L4djc0pc8Y\\\\\\\/KiRgiJxP2dn\\\\\\\/srTs0WUXpYI3AScMDYcdmv5Cc\\\\\\\/HbueQU3NxVGMvwPJXn58PCB969l52dcUMeevuncfZUHX1a2ksnZbWomuyayfJ9vxaoXGUNygqDzdH4q7mLakPeudt0sJkTuZBMz+PnyFxGOnVemh7llxlUX9To5Pga1E2g6eIfJzEEpxjKZshTDkC+YiQ1AFt6hLreyOqgdel3ooOaj+Nt4GWcIykavV6JR39qftzAytBfzZ7PFVrQUpY6ZF0x+EGarK\\\\\\\/YMhGQ84EjL9U4wOY++7+HM1OGApO9XegKJdH1LxMSXuxH5FIoE2VsFpqehN6\\\\\\\/Q+awUZoLjKRH5CC43Fw+vW8B4JWOXIFJGsG3vi9h2kkKLkJOCVyGygCYkiRgCx\\\\\\\/i4unRfC1byr63ZAqPlinJk\\\\\\\/DZ1l2YsHh9MWJF+K1T04OyIg2TJ+ai3r1gHeJixDs\\\\\\\/Wg5a4tP7m95ZCBdg+AwQ2k1C14xlUhFyTDOSNEXsmNzNorbqDJvuFJzSjsBVUSQK6Vlv4ou0RHm2FqKiLUHNu3A6ojTkug8LlasiL1jP1uyMpScZpgAQp+2YoztEmFYDKwNBxGrreQLhDRs0R0A6m5eE04P9LfBIWhyFPiE4YD9OdWZKj2G52\\\\\\\/ET50mboYeZLlbe8Q6dmTUxFiIy3ftxOLkR+hNqgG61+QOhFXb\\\\\\\/GVZMep2ihfJQnheJsqQ9EANqkGlOw2m7Kly5pZ+eUw70J\\\\\\\/4pr3zsca4MYEghjhdJ8WGoCBaag5OfWGu8ZqQiHjKbJUH6v6LsyCON6Cp2Y9bUdSQ\\\\\\\/k5EpUvjYjCKk+rmVKRGfzh4L8GVKfmIQJy54EtBu58wP9VcVUwovbdMqmMBmdvUIn+kaMr+nQJBuYGKQCdcM3sawBYURhE6lUHD++7ZresgR\\\\\\\/jJrx2DJpPindjcd6+pU8o0osmcLs\\\\\\\/LEwyjW+zsYbIt\\\\\\\/OhZExDSOXawpQ3Y7WbV+Z6fgTxtts+nkkOjTkPxzGzGG2MubGMIXS5o5oqU1BH69w7Mrqxx6FSaR9rtueK3I9zUhnq+pF1soILgGQ+SLHr\\\\\\\/wDktpNtvDejxbXx55rWRiAFoDRlnNvwo5tbnRx93PtmYbHSE\\\u0022,\\\u0022iv\\\u0022:\\\u002236XaxtEpQE\\\\\\\/654QVHsB\\\\\\\/KA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022de64fc09063382115d394db315aabc3f25e1699ad2f5233f76dccb6b7a25246b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vuu8k8Kqdnac0Cq9pBZJ0O7GQ\\\\\\\/Gvttpsgcm5s425jxCanP+ZwibADQsEc\\\\\\\/JnfpANGNEb7l8oOZh8hcpMBSvVgo3XR13LVT3JXQh37WItkRN3Uao1+2du+IdDsKKV2jCwB+Y4txCzMKDXGlmf\\\\\\\/Cpv3UE5w41cbvYEWuPeUyns0dAlG0rgJLT797qrycQQLYR3\\\\\\\/3yZZfJNs+ceD3CV4QosJ2FJtNlrexa\\\\\\\/puTRPn5rH0ZNTYlxnJErORxkf3He7jHMrYLKNZOuQZj6FotbKnnQEzoBel8SxEEGYFVQL8LTOQfchhZkDgnvp+nN7sMpk+lx0O6SaIdLcLD3puXU4szCSBOAGvZiD5m56vEMYJpO7Jf9j5rqWqp2WF8WTzxp8F9eJQG6jNoKrDvqIJEsZfpf9GSRiJu\\\\\\\/5KQ2TkPXF7tZwxlsn7SMwwpcj2Vm7psmbvy1kKlB0C8vOfhekw32MKeDg9ITRIKu4c9HgGUr5e\\\\\\\/djlfdaAEIwu89iJyuCdj6O81kFkO1oRsBY\\\\\\\/VUHbidcaYIQwpz5CyBzpreDJfmIG7fCVRZirwLVAVAMdhvVDw94n+QeC5MYiBuHWQA8Alxa+AQRy03CICE3EjsZp2qIN+gsB7VuA9qjpfx2aEzEqUCPAz\\\\\\\/1kF7EUjI7cDkxBNFkSUbILQPDJURVRvXLlArmURUKWhub+17eL9RzhFN52pEa9GyBVkKBP0yQ0tPteBi9QTSCBsCtdWnEyIiQfEv+g69hNGGkGvu+HmSBG1\\\\\\\/3vI+qNoo36aiIhCpZq+7bJHYLVonRvcrN+vgrzQfVoeAl\\\\\\\/oejie3wtPpr8nlpD4u9Ngsv\\\\\\\/vABrYAgwuBk6xkbAkbVmOaFseS\\\\\\\/kuWOb9tNAKne+EHNl5u2lOKirRlFq6u3VaKrf8YSYElKh3NdPoPmvSBLYFaup8Hq\\\\\\\/oja4LCRKOjm88JBcWtLNGchTcgaUFpt9znV8MGOO47h5qVdmEwPKY0c2d7Pn0nG2W\\\\\\\/bgn3o23YjPRxWtmrH7XivPUyfXIiwWmubxR6HSU8FwnqMcIZaNuzVEuHHK+nbriGS5Tlc7c3aoPuoft4YCKB9RVnqnV1XcZ\\\\\\\/bCmh5+DTVIoydzYsWBzjxXv1iy2Uc4TnwUGjDoIJaksLuB+wJ45gpJTjT2NxXWqPR4w4taI6hooHwFWUeDMkNNgyYJgM4AfcCLiyqLZtcNaust15yTPrV0OtrhpBmdRuTo\\\\\\\/yFTyluugxjvW5A++4LK5ZBe+e5M8QKSlcaR8JCtoxCFWdweJB9kiumlQN\\\\\\\/vNrIG\\\\\\\/vqVf+ksLQKpeeKaZZxk\\\\\\\/ujtg66aGyrObX5sirNOwAGnEgff1dulHLJx95lYfqIaDL1sDlPrjho4nBqhwE55n2uHnuUfMnKCM4vV5EW0UagXcSoWJQ1B5cuCFeeFrOVcFxTVSc3KNa5Q0cJ\\\\\\\/nX5b8JchMIs9zDm8wzfO5u0B5R0Hlij+WD6mpiNcVI+3Lx1Ijo+h+B5\\\\\\\/V+piDg9+Tx4lQPCN2K1ZM4MeM911JSlRlvuAGi2tkA1jlBjvGZfJq9JVuRm\\\\\\\/2UPV73NQ7oJVhbj23K+dhV7mKwzLuZFfPrm5AYCVJk+xfVNuwDpeulVey7Pva+JJyTOUk3UitkrAH+wrNilCnj93xn7oSXF9d3aagNEFm2IiscKF7hQj7YsaWeHYjZObHFnNKUe3QjOtjoM23boGVQnIMzxPJ49Tonmos0tSYt63mIUG7lLogMQZTlSFbKWwY51t48sI+xkW+d6SZnTQYOX95XlSdF9sj7WFJQhF2nLB1TguQL3axb52bOZIqRa\\\\\\\/YGj+dvZLhwOXSQJaOD51hRLzDDnUMaCr7MiRzuMftHuCWyR0fjIGxKRGkQw4Geqk6wPxCJaMcVJolUTDGAMpu9dVxRdeMLPEuPk7fxZwQ6furcl6TW6GGCg6Lr9RkqMFcgSWBR2KHjklbQUrzx9SUXcDfgkieatNLlRrYaNxMQMOoEpHRFP\\\\\\\/Ugdd7oj5MUwEDbNfzzsgUvC4KoHoeHmMEG8CjoEx9Euwp3UymxsZzXdRJxWfJHqspcmC3Lm9V1\\\u0022,\\\u0022iv\\\u0022:\\\u0022vb5V8capo77PEFKs\\\\\\\/oX2Tg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022050560280c71aae05cab43d69b60ff316380bad059a494cab379510312dd6769\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459667 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1773" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yi4arkEm3ZLR4SiGu-2TKjejXKpVwv92maftzd41Iek" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1773" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U7mScsiF1bQQeL0en-oGcLRRfYQmNkoMh1VhsiBYets" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1765" title="A New Dataset of Spermatogenic vs. Oogenic Transcriptomes in the Nematode Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1787" title="Gene Expression in the Skin of Dogs Sensitized to the House Dust Mite Dermatophagoides farinae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459667" id="node459667" data-pisa="ggg;4/9/1773" data-pisa-master="ggg;g3.114.013250" data-apath="/ggg/4/9/1773.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elyse A. Hope</span> and <span class="highwire-citation-author" data-delta="1">Maitreya J. Dunham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1773-1786;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013250 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459667" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elyse A. Hope </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elyse%2BA.%2BHope%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hope%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElyse%2BA.%2BHope%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1773" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459667" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1773.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459667" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1773.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459667" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1773.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459667" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1773.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459667" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The ability of yeast to form biofilms contributes to better survival under stressful conditions. We see the impact of yeast biofilms and “flocs” (clumps) in human health and industry, where forming clumps enables yeast to act as a natural filter in brewing and forming biofilms enables yeast to remain virulent in cases of fungal infection. Despite the importance of biofilms in yeast natural isolates, the majority of our knowledge about yeast biofilm genetics comes from work with a few tractable laboratory strains. A new collection of sequenced natural isolates from the <em>Saccharomyces</em> Genome Resequencing Project enabled us to examine the breadth of biofilm-related phenotypes in geographically, ecologically, and genetically diverse strains of <em>Saccharomyces cerevisiae</em>. We present a panel of 31 haploid and 24 diploid strains for which we have characterized six biofilm-related phenotypes: complex colony morphology, complex mat formation, flocculation, agar invasion, polystyrene adhesion, and psuedohyphal growth. Our results show that there is extensive phenotypic variation between and within strains, and that these six phenotypes are primarily uncorrelated or weakly correlated, with the notable exception of complex colony and complex mat formation. We also show that the phenotypic strength of these strains varies significantly depending on ploidy, and the diploid strains demonstrate both decreased and increased phenotypic strength with respect to their haploid counterparts. This is a more complex view of the impact of ploidy on biofilm-related phenotypes than previous work with laboratory strains has suggested, demonstrating the importance and enormous potential of working with natural isolates of yeast.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/biofilm" class="hw-term hw-article-keyword hw-article-keyword-biofilm" rel="nofollow">biofilm</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li><li class="kwd"><a href="/keyword/natural-isolates" class="hw-term hw-article-keyword hw-article-keyword-natural-isolates" rel="nofollow">natural isolates</a></li><li class="kwd"><a href="/keyword/flocculation" class="hw-term hw-article-keyword hw-article-keyword-flocculation" rel="nofollow">flocculation</a></li></ul><p id="p-4">Wild-type budding yeast selectively bind to each other and to substrates in cohesive groups called yeast “flocs” (or clumps) and yeast biofilms. The ability of yeast to form biofilms contributes to better survival under stress and affects myriad processes important to human health and industry. In brewing, yeast flocs facilitate easy removal of yeast from the final product, but yeast biofilms can complicate biofuel production and help pathogenic yeast remain virulent on hospital surfaces and during infection (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Verstrepen <em>et al.</em> 2003</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">2004</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bauer <em>et al.</em> 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sivakumar <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hill <em>et al.</em> 2013</a>). In the laboratory, yeast biofilms create significant challenges for many experiments. As a result, biofilm formation has been selected against in most laboratory strains and our knowledge of the genetic basis of yeast biofilm formation remains incomplete.</p><p id="p-5">Yeast flocs and biofilms are two distinct but related phenotypes regulated by environmental changes that cause complex signaling and gene expression responses. A true biofilm requires that yeast both form an extracellular protein matrix and adhere to a surface (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stovicek <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Vachova <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bruckner and Mosch 2012</a>). The matrix is made up of secreted proteins and the biofilm is anchored to surfaces by hydrophobic interactions between yeast cell surface proteins and the surface (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Voordeckers <em>et al.</em> 2012</a>). Biofilms can be observed phenotypically by several assays including examination of a strain’s ability to adhere to polystyrene surfaces, visualized by staining, a strain’s ability to form a complex colony with an extracellular matrix, and its ability to form a complex mat on low agar media (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">O’Toole and Kolter 1998</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">O’Toole <em>et al.</em> 2000</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Reynolds and Fink 2001</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Granek and Magwene 2010</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Stovicek <em>et al.</em> 2010</a>). Flocs, in contrast to true biofilms, are clumps of yeast adhering to each other by a protein–carbohydrate bond between cell surface proteins and cell wall sugars (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bony <em>et al.</em> 1997</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2000</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Verstrepen <em>et al.</em> 2003</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bruckner and Mosch 2012</a>). This phenotype is most commonly observed as clumping and settling in liquid media (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Johnston and Reader 1983</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Smukalla <em>et al.</em> 2008</a>). Due to the intercellular bonds, flocculation is different from the type of clumping that results from mother–daughter cell separation defects. Flocculent clumps have been shown to render the participating yeast cells more resistant to chemical stresses (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Smukalla <em>et al.</em> 2008</a>). There is also evidence that flocs contain an extracellular matrix, a biofilm-related trait, reflecting the close relationship between biofilm and floc phenotypes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beauvais <em>et al.</em> 2009</a>). The formation of biofilms and flocs is thought to be regulated by similar genes. The most well-studied of these genes are the <em>FLO</em> genes, including the important genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>, involved primarily in cell–cell and cell–surface adhesion, respectively, and transcription factor <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em>, which regulates expression of the other <em>FLO</em> genes (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Verstrepen and Klis 2006</a>).</p><p id="p-6">Biofilm-related and flocculation-related phenotypes are predicted to differ depending on ploidy. Another common phenotype associated with <em>FLO</em> gene activity is invasiveness into solid agar media, which in haploids occurs on rich media and in diploids occurs under limited nitrogen supply as filamentous growth (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cullen and Sprague 2000</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2000</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ryan <em>et al.</em> 2012</a>). Diploid strains under a limited nitrogen supply are able to undergo a dimorphic switch from a yeast form, which generates colonies on budding, and the filamentous form, which generates long chains of interconnected cells called “pseudohyphae” that can be seen as filamentous extensions from a colony (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gimeno <em>et al.</em> 1992</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lo and Dranginis 1998</a>). Increases in ploidy have also previously been shown to reduce complex colony morphology, complex mat formation, and invasion; this decreased phenotypic strength is thought to result from a cellular response to increased gene dosage or DNA content, or differential expression of cell–surface proteins (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Galitski <em>et al.</em> 1999</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2000</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Reynolds and Fink 2001</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Granek and Magwene 2010</a>). The possible phenotypic differences between haploid and diploid versions of the same strain and the filamentous growth phenotype specific to diploid strains make a compelling case for considering diploid as well as haploid versions of a strain in any effort to generate a complete picture of its biofilm-forming capabilities.</p><p id="p-7">Natural isolates provide a unique testing ground for understanding the spectrum of biofilm-related phenotypes achieved by <em>S. cerevisiae</em> outside of laboratory strains.</p><p id="p-8">The existence of a recent collection of natural yeast isolates from the <em>Saccharomyces</em> Genome Resequencing Project (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liti <em>et al.</em> 2009</a>) makes it possible to examine important phenotypes that exist among natural yeasts but are not prevalent in laboratory strains. It also provides an opportunity to explore the connection between genotype and biofilm-related phenotypes on an unprecedented scale. The SGRP collection of natural isolates continues to grow, providing an extended resource for examining questions of phenotypic variation in natural isolates and now, with high-quality sequence information available, questions of genotype to phenotype connections (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Skelly <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bergström <em>et al.</em> 2014</a>).</p><p id="p-9">Other recent studies have used the SGRP collection and other similar natural isolate collections [<em>e.g.</em>, Phaff Yeast Culture Collection, ARS (NRRL) Culture Collaboration] to examine questions of lineage and sequence similarity across geographical niches (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cromie <em>et al.</em> 2013</a>). The SGRP collection also has associated “phenome” data, for which each strain’s morphology, metabolite, gene expression, and protein traits were recorded, as well as the strains’ responses to environmental perturbations, including toxins and nutrient limitations (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Skelly <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zorgo <em>et al.</em> 2013</a>). With this extensive profile available on the sequence, gene expression, and phenotypic levels, the SGRP collection is a natural and informative starting point for examining genotype/phenotype relationships, and strains from the collection have already been used effectively in a few quantitative trait mapping studies (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cubillos <em>et al.</em> 2011</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Granek <em>et al.</em> 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jara <em>et al.</em> 2014</a>).</p><p id="p-10">Our objective in this study was to assess the diversity of biofilm-related phenotypes in the natural isolate collection and to determine the extent to which any or all of those biofilm-related phenotypes are related. We present a panel of biofilm-related phenotypic assays for 31 haploid and 24 diploid strains in the SGRP natural isolate collection. We show that there is strong qualitative and quantitative variation across six primary phenotypes within the collection: complex colony morphology; complex mat formation; flocculation; agar invasion; polystyrene adhesion; and filamentous growth. The strength of each phenotype is only weakly correlated with the others, with some notable exceptions, demonstrating the potential of these assays to evaluate different aspects of biofilm-related morphology. We also show that the diploid phenotypes are significantly different from their haploid counterparts, in several cases showing increased complexity with respect to the haploid strains contrary to expectations based on the literature about laboratory strains. These findings clearly demonstrate the utility of examining natural isolates to better understand the correlations between these phenotypes and their relationship to ploidy.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media used in this study</h3><p id="p-11">SGRP strains used in this study (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Liti <em>et al.</em> 2009</a>) were purchased from the National Collection of Yeast Cultures (<a href="https://catalogue.ncyc.co.uk">https://catalogue.ncyc.co.uk</a>) and are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/013250SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS1.pdf">Table S1</a>. Strain identity and ploidy was verified for a subset of diploid strains by sequencing, sporulation, and cytometry (data not shown). Standard growth medium for plates and liquid overnight cultures was yeast extract peptone dextrose (YPD) media, with 2% glucose and 2% agar for plates. Growth in the polystyrene adherence assay was performed in synthetic complete media with 2% glucose or 0.1% glucose. Standard media were prepared according to <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sherman (1998)</a>. Diploid filamentous growth was assayed on synthetic low-ammonia histidine dextrose (SLAHD) media prepared according to a modified recipe (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gimeno <em>et al.</em> 1992</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lorenz and Heitman 1997</a>) with 50 μM ammonium sulfate, 2% glucose, 2% bacto-agar, 1.7 g YNB without ammonium sulfate or amino acids, and 0.2 mM histidine hydrochloride per liter. Prion curing was conducted on 3-mM guanidine hydrochloride plates prepared according to <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holmes <em>et al.</em> (2013)</a>.</p></div><div id="sec-3" class="subsection"><h3>Standardized phenotypic assay protocols</h3><div id="sec-4" class="subsection"><h4>Measuring complex colony morphology:</h4><p id="p-12">Two biological replicates of strains were inoculated from colonies into 200 μL YPD liquid overnight cultures in 96-well plates. They were diluted to concentrations of 10<sup>−5</sup>, 10<sup>−6</sup>, and 10<sup>−7</sup> for haploid strains, with the second biological replicate at concentration 10<sup>−6</sup> only; 200 μL of all four dilutions of culture was plated using glass beads on 2% agar, 2% glucose YPD plates. The colony plates were incubated at 30° for 3 d and at 25° for an additional 10 d. The colonies were photographed and scored for complexity on days 5, 9, and 13. All imaging in this and other assays was performed using a Canon Powershot SD1200 IS digital camera. This procedure was maintained for all collections (haploid SGRP strains, diploid SGRP strains, and prion-cured strains) with modifications only in the dilutions plated (10<sup>−6</sup> only for diploid and prion-cured strains). Colony complexity was scored from 0 (smooth colony) to 5 (very strong complex colony morphology) according to the metric described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS2.pdf">Table S2</a>, adapted from <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Granek and Magwene (2010)</a>.</p></div><div id="sec-5" class="subsection"><h4>Measuring complex mat formation:</h4><p id="p-13">Strains were inoculated by toothpick from colonies onto the center of a 0.3% agar, 2% glucose YPD plate left to dry for 1 d. The mat formation plates were sealed with parafilm and incubated upright at 25° for 13 d (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Reynolds and Fink 2001</a>). On the final day, the complex mats were photographed and scored for complexity according to the metric described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS2.pdf">Table S2</a>. This assay was performed once across all strains in the diploid and prion-cured collections and for two biological replicates in the haploid collection. A second biological replicate was performed for eight of the nine diploid strains with phenotypes that differed from the haploid strains. One of these replicates with a weak phenotype was inconclusive by visual scoring.</p></div><div id="sec-6" class="subsection"><h4>Measuring flocculation as settling in liquid culture:</h4><p id="p-14">The settling assay measures clumping and settling of cells to the bottom of 2% glucose YPD liquid culture (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Johnston and Reader 1983</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Smukalla <em>et al.</em> 2008</a>). Strains were inoculated from colonies into 5 mL YPD liquid cultures and grown for 20 hr at 30°. The strains were then vortexed vigorously in the culture tubes and allowed to settle. For each strain, four photographic time points were taken at 0, 15, 30, and 60 min, with the culture tubes undisturbed between time points. This assay was performed once across all strains in the diploid and prion-cured collections and for two biological replicates in the haploid collection.</p></div><div id="sec-7" class="subsection"><h4>Quantitative flocculation assay:</h4><p id="p-15">Quantitative flocculation analysis was conducted on the images from the settled 60-min time point for each strain across two biological replicates of the haploid collection, the diploid collection, and the prion-cured collection. The images were converted to black and white in image processing software Picasa version 3.9.16.37 and analyzed in image processing software ImageJ version 1.47 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abramoff <em>et al.</em> 2004</a>). A line was drawn from the meniscus to the bottom of the culture tube in each image and used to measure the maximum gray intensity and the plot profile of intensity along the line. From the plot profile data, the x-coordinate along the line at which half of the maximum gray was reached was calculated and used to calculate a ratio: x coordinate of half of maximum gray intensity / length of the line. Three of these measurements were taken per image and the average ratio across plot profiles was calculated to determine, for each strain, the percentage of the tube cleared at 60 min.</p></div><div id="sec-8" class="subsection"><h4>Measuring agar invasion:</h4><p id="p-16">The agar invasion assay measures a strain’s ability to invade solid agar media, quantified by the density of cells remaining in the media when colonies are washed off the surface. Strains were inoculated from colonies into 5 mL YPD liquid overnight cultures. They were grown to saturation and standardized to an OD between 0.5 and 0.8. The 2% agar, 2% glucose YPD invasion plates were spotted with all strains from a collection (haploid SGRP strains, diploid SGRP strains, or prion-cured strains) and the FY4 (s288C) reference strain with a 1-μL spot per strain. Four invasion plates were inoculated per collection to provide technical replicates. Invasion plates were grown at 30° for 5 d, after which time the colonies were washed off the surface of the plates under a stream of distilled water using a gloved finger (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Roop and Brem 2013</a>). The remaining invaded spots were photographed for each spot on each of the four plates. The washed plates were incubated again at 30° for 24 hr, then washed and photographed again to observe additional growth from cells trapped in the agar (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Drees <em>et al.</em> 2005</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ryan <em>et al.</em> 2012</a>). This assay was performed once across all strains in the diploid and prion-cured collections and for two biological replicates in the haploid collection. Eight diploid strains had a statistically significant difference in invasive ability compared with the haploid strains and these strains were selected for a second biological replicate.</p></div><div id="sec-9" class="subsection"><h4>Quantitative invasion calculations:</h4><p id="p-17">From the four technical replicates of each strain, the three best images were selected for quantitative analysis. The images were converted to black and white in image processing software Picasa version 3.9.16.37 and analyzed in image processing software ImageJ version 1.47 (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abramoff <em>et al.</em> 2004</a>). To quantify the invasion for each spot, the mean gray intensity of the spot and surrounding background were calculated in ImageJ. The mean gray intensity of the entire image was also calculated in ImageJ and used to subtract an average background value from the invaded spot. The formula used for background correction calculated the background-corrected intensity value as [(A<sub>t</sub>*I<sub>t</sub>)−(A<sub>s</sub>*I<sub>s</sub>))/(A<sub>t</sub>−A<sub>s</sub>)], where A<sub>t</sub> is the area of the total image, A<sub>s</sub> is the area of the invaded spot, I<sub>t</sub> is the mean gray intensity of the total image, and I<sub>s</sub> is the mean gray intensity of the invaded spot. For any strains for which background correction yielded a negative invasion value, the value was corrected to zero. This analysis was performed for the full haploid, diploid, and prion-cured collections, and for an additional biological replicate of the haploid collection.</p></div><div id="sec-10" class="subsection"><h4>Polystyrene adhesion with crystal violet assay:</h4><p id="p-18">Polystyrene adhesion is measured by growing biofilms in flat-bottom polystyrene 96-well plates and staining for remaining cells using crystal violet dye. Strains were inoculated into SC cultures with 2% glucose and grown to an OD between 0.5 and 1.5 according to <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Reynolds and Fink (2001)</a>. OD600 was measured on a BioTek Synergy <sup>1</sup>H hybrid microplate reader. Cells were spun down and washed with distilled water, then resuspended in SC media with 0.1% glucose to a standard OD of 1. Then, 100 μL of culture was inoculated in two technical replicates, three for strains with enough culture volume after resuspension, into a 96-well Costar untreated flat-bottom polystyrene plate. The flat-bottom plate was incubated for 6 hr at 30°. The cultures were then fixed and stained with 100 μL per well of crystal violet dye solution (Fisher Scientific). Crystal violet dye was prepared as a 1% crystal violet solution in 100% ethanol and filter sterilized through a 0.2-μm filter. The crystal violet was incubated in the wells for 15 min at room temperature, then decanted onto absorbent bench pads. The wells were washed three times with 300 μL sterile water, with decanting to rinse. This assay was performed once across all strains in the diploid and prion-cured collections and for two biological replicates in the haploid collection.</p></div><div id="sec-11" class="subsection"><h4>Quantitative polystyrene adhesion with crystal violet assay:</h4><p id="p-19">Stained biofilms were analyzed according to <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Reynolds and Fink (2001)</a>. Biofilms were solubilized in 100 μL 10% SDS per well with a 30-min incubation at room temperature; 100 μL sterile water was then added and 100 μL of solubilized dye was transferred from each well to a new flat-bottom 96-well plate. Absorbance was measured at 570 nm, the absorbance of crystal violet, on a BioTek Synergy <sup>1</sup>H hybrid microplate reader.</p></div><div id="sec-12" class="subsection"><h4>Diploid filamentous growth assay:</h4><p id="p-20">Strains were plated following the complex colony morphology assay protocol described above onto synthetic low-ammonia histidine dextrose (SLAHD) plates to encourage filamentous growth in a nitrogen-poor environment (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gimeno <em>et al.</em> 1992</a>; <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lorenz and Heitman 1997</a>). The colony plates were incubated at 30° for 3 d and at 25° for an additional 2 d. The colonies were photographed and scored for filamentous growth under a brightfield microscope at between 20× and 50× magnification. The colonies grew at 25° for an additional 4 d and were photographed and scored for additional filamentous growth on the ninth day of growth. Colonies were scored from 0 to 5 for filamentous growth according to the metric described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS2.pdf">Table S2</a>.</p></div><div id="sec-13" class="subsection"><h4>Curing strains of prions:</h4><p id="p-21">To eliminate any prions in the natural isolates, all haploid strains were passaged four times on 3 mM guanidine hydrochloride YPD plates, bottlenecking through a single colony each time (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Holmes <em>et al.</em> 2013</a>). Clones were selected from the cured strains and passaged on 1.5% agar yeast-extract peptone glycerol media to check for respiration competence. Strains that were respiration competent were stored as prion-negative strains in glycerol stocks; strains that were not respiration-competent were excluded from further experiments. Prion-cured strains were phenotyped according to the same assay protocols described above. For seven strains that showed any potential phenotypic differences after curing and two that did not, additional cured replicates were generated to verify the phenotypes and observe the effects of potential additional mutations resulting from the curing process.</p></div></div><div id="sec-14" class="subsection"><h3>Correlations between quantitative phenotypes</h3><p id="p-22">To determine whether any of the biofilm-related phenotypic traits were correlated, mean quantitative data across each phenotype were compared using a paired two-tailed <em>t</em>-test assuming unequal variance. For colony morphology and mat formation, the two qualitative phenotypes scored from 0 to 5, a Kendall’s tau rank correlation test (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kendall 1938</a>) was performed in R on the scores for each strain, comparing each phenotype pairwise.</p></div><div id="sec-15" class="subsection"><h3>Hierarchical clustering of phenotypes by niche</h3><p id="p-23">To determine whether any phenotypes clustered by niche of origin, the quantitative phenotypic values for each strain were linearly normalized to a scale of 0 to 5 and used as input into Cluster version 3.0 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">De Hoon <em>et al.</em> 2004</a>). The strains were clustered hierarchically based on Euclidean distance between the quantitative phenotype vectors and visualized in Java Treeview (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Saldanha 2004</a>).</p></div><div id="sec-16" class="subsection"><h3>Statistical comparison of haploid and diploid strains</h3><p id="p-24">Quantitative values for haploid and diploid strains across the flocculation, invasion, and crystal violet assays were compared using a paired two-tailed <em>t</em>-test assuming unequal variance across all technical replicates for each strain. <em>P</em>-values were evaluated for significance using the <em>q</em>-values package from Storey with a false discovery rate of 0.05 and the Benjamini-Hochberg method (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Storey 2002</a>).</p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>Natural isolates exhibit extensive diversity across biofilm-related phenotypes</h3><p id="p-25">To determine the diversity of phenotypes present in the <em>Saccharomyces</em> Genome Resequencing Project collection (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Liti <em>et al.</em> 2009</a>), we examined five different biofilm-related phenotypes across 30 MATa haploid strains from the collection and a laboratory strain control (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS1.pdf">Table S1</a>). The five phenotypes (complex colony formation, complex mat formation, flocculation/settling, agar invasion, and polystyrene adhesion) are shown for a representative set of strains in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The full dataset for two biological replicates is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS1.pdf">File S1</a>. These six representative strains not only originate from varied ecological and geographical niches but also demonstrate the full range of phenotypes we observed in this collection from grade 0 (no phenotype) to grade 5 (very strong phenotype), as outlined in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS2.pdf">Table S2</a>, with score assignments shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS3.pdf">Table S3</a>. We observed that many strains differed in the strength of each trait. For example, the Hawaiian cactus strain UWOPS87-242.1 exhibits strong flocculation and polystyrene adhesion phenotypes and weak phenotypes across the other assays (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Within a single phenotypic grading there was also significant morphological variation making each strain unique. In the complex mat formation assay, for example, strains YPS128, UWOPS05-227.2, and 322134S formed highly complex mats that received qualitative scores of 4 and 5, but all three mats exhibit distinct morphologies (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). By examining the strains using a range of phenotypic assays, we were able to generate what is essentially a unique profile of biofilm-related phenotypes for each of the 31 strains, showcasing the incredible diversity accessible in this collection of natural isolates.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Natural isolates exhibit extensive diversity across biofilm-related phenotypes. Full phenotypic panel for six representative strains and reference strain FY4. Strains are listed with their formal name and origin and are shown across five different phenotypes. Complex colony morphology is shown at day 5 and day 13 of growth on 2% agar YPD plates. Complex mat formation is shown at day 13 of growth on 0.3% agar YPD plates, with the plate included for scale. Settling photo time points were taken at 0, 15, 30, and 60 min. Three technical replicates are shown for the invasion assay and photographed after 24 hr of growth after washing on day 5. Two technical replicates are shown for the polystyrene adhesion assay. Pictured biofilms are fixed and stained with a 1% w/v crystal violet solution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Natural isolates exhibit extensive diversity across biofilm-related phenotypes. Full phenotypic panel for six representative strains and reference strain FY4. Strains are listed with their formal name and origin and are shown across five different phenotypes. Complex colony morphology is shown at day 5 and day 13 of growth on 2% agar YPD plates. Complex mat formation is shown at day 13 of growth on 0.3% agar YPD plates, with the plate included for scale. Settling photo time points were taken at 0, 15, 30, and 60 min. Three technical replicates are shown for the invasion assay and photographed after 24 hr of growth after washing on day 5. Two technical replicates are shown for the polystyrene adhesion assay. Pictured biofilms are fixed and stained with a 1% w/v crystal violet solution.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F1.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1773/F1.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464887" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Natural isolates exhibit extensive diversity across biofilm-related phenotypes. Full phenotypic panel for six representative strains and reference strain FY4. Strains are listed with their formal name and origin and are shown across five different phenotypes. Complex colony morphology is shown at day 5 and day 13 of growth on 2% agar YPD plates. Complex mat formation is shown at day 13 of growth on 0.3% agar YPD plates, with the plate included for scale. Settling photo time points were taken at 0, 15, 30, and 60 min. Three technical replicates are shown for the invasion assay and photographed after 24 hr of growth after washing on day 5. Two technical replicates are shown for the polystyrene adhesion assay. Pictured biofilms are fixed and stained with a 1% w/v crystal violet solution.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Quantitative measures of biofilm-related phenotypes agree with qualitative scoring</h3><p id="p-27">In addition to the qualitative scores assigned to each phenotype, the flocculation, invasion, and polystyrene adhesion assays were also amenable to quantitation, as has been described in previous studies (see <em>Materials and Methods</em>). Previous literature attempted to assign quantitative metrics to mat formation as well (<a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Reynolds and Fink 2001</a>). Although major mat features were retained across biological replicates in our dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS1.pdf">Figure S1</a>D), quantifiable features like spoke number and lobe number were not consistent across replicates. In addition, many strains in this collection formed filigreed mats without a spoke/lobe structure (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS1.pdf">File S1</a>) and would not be quantifiable by those metrics.</p><p id="p-28">The quantitative values across the flocculation, invasion, and polystyrene adhesion assays shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> contrasted with the qualitative image data. Across all three quantitative assays, the strains with the most visually striking phenotypes (UWOPS87-242.1 for settling, 322134S for invasion, and SK1 and 322134S for adhesion) all correspond to the highest recorded quantitative measurements. Distinctions between strains with weak phenotypes are less clearly represented by the quantitative data, but trends of increasing phenotypic strength are consistent across all three quantitative assays. The results of the quantitative assays compared across biological replicates also demonstrate that these assays are a consistent, repeatable method for examining these phenotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative measures of biofilm-related phenotypes accurately represent qualitative data. Haploid strains on the x-axis are ranked by increasing phenotypic strength. For procedural details of each method of quantification, see Materials and Methods. (A) Qualitative colony classifications at day 13 of growth, from (0) weak to (5) very strong. (B) Quantitative flocculation measured as the percent of culture settled at 60 min. Average of three measurements per image. (C) Quantitative invasion data measured as the mean gray intensity of the invaded spot. Average of three technical replicates. (D) Quantitative crystal violet data measured as OD570 absorbance of crystal violet&#x2013;stained biofilms. Average of two technical replicates; exceptions are DBVPG6040, YS4 with one measurement only for crystal violet." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative measures of biofilm-related phenotypes accurately represent qualitative data. Haploid strains on the x-axis are ranked by increasing phenotypic strength. For procedural details of each method of quantification, see &lt;em&gt;Materials and Methods&lt;/em&gt;. (A) Qualitative colony classifications at day 13 of growth, from (0) weak to (5) very strong. (B) Quantitative flocculation measured as the percent of culture settled at 60 min. Average of three measurements per image. (C) Quantitative invasion data measured as the mean gray intensity of the invaded spot. Average of three technical replicates. (D) Quantitative crystal violet data measured as OD570 absorbance of crystal violet&#x2013;stained biofilms. Average of two technical replicates; exceptions are DBVPG6040, YS4 with one measurement only for crystal violet.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F2.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/9/1773/F2.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464840" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Quantitative measures of biofilm-related phenotypes accurately represent qualitative data. Haploid strains on the x-axis are ranked by increasing phenotypic strength. For procedural details of each method of quantification, see <em>Materials and Methods</em>. (A) Qualitative colony classifications at day 13 of growth, from (0) weak to (5) very strong. (B) Quantitative flocculation measured as the percent of culture settled at 60 min. Average of three measurements per image. (C) Quantitative invasion data measured as the mean gray intensity of the invaded spot. Average of three technical replicates. (D) Quantitative crystal violet data measured as OD570 absorbance of crystal violet–stained biofilms. Average of two technical replicates; exceptions are DBVPG6040, YS4 with one measurement only for crystal violet.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Biofilm-related phenotypes show complex correlations</h3><p id="p-30">Our results demonstrate that the phenotypic strength of a strain in one assay is not necessarily predictive of its strength across another. Of the six SGRP strains shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, only two (UWOPS05-227.2 and 322134S) have strong phenotypes across all assays; the rest vary. We were interested in determining if any of these phenotypes show correlations with each other and to what extent each additional assay is providing new information about the behavior of the strain.</p><p id="p-31">For the two assays for which only qualitative data were available (complex colony morphology and complex mat formation), we performed a Kendall’s tau rank correlation test on the qualitative scores across all strains for a single biological replicate in a pairwise fashion. At a significance threshold of <em>P</em> &lt; 0.05, we found that these two phenotypes were highly correlated, with a <em>P</em>-value of 1.03×10<sup>−6</sup>. For assays with quantitative data, we binned the data across the associated qualitative colony scores (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A–C</a>) to look for trends associated with increasing colony morphology complexity. Flocculation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) showed no correlation with colony morphology by this method, but there was a trend associating increasing invasive ability (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>) and polystyrene adhesion (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>) with increasing complexity in colony morphology (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). For the three assays with quantitative data available, we correlated the average quantitative values across technical replicates between each of the phenotypes. Despite the shared genetic networks implicated in regulation of these traits, the majority of phenotypic correlations we observed were weak. We found no correlation between haploid flocculation and invasion phenotypes (R<sup>2</sup> = 0.04) (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>) and a weak correlation between haploid invasion and polystyrene adhesion (R<sup>2</sup> = 0.35) (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3F</a>). We found a stronger correlation between haploid flocculation and polystyrene adhesion (R<sup>2</sup> = 0.42) (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3E</a>), although this correlation is still weaker than those between biological replicates (R<sup>2</sup> = 0.83 and R<sup>2</sup> = 0.68 from flocculation and adhesion biological replicates, respectively) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS1.pdf">Figure S1</a>, B and D).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Some biofilm-related phenotypes are correlated. All data are for 31 haploid strains. Qualitative data shown are for a single haploid replicate; quantitative data are averaged across two biological replicates. Mean quantitative data for (A) flocculation, (B) invasion, and (C) adhesion are shown on the y-axis binned according to haploid colony complexity scores (x-axis). Mean quantitative data are plotted for correlation, with (D) flocculation (x) against invasion (y), (E) flocculation (x) against adhesion (y), and (F) invasion (x) against adhesion (y)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Some biofilm-related phenotypes are correlated. All data are for 31 haploid strains. Qualitative data shown are for a single haploid replicate; quantitative data are averaged across two biological replicates. Mean quantitative data for (A) flocculation, (B) invasion, and (C) adhesion are shown on the y-axis binned according to haploid colony complexity scores (x-axis). Mean quantitative data are plotted for correlation, with (D) flocculation (x) against invasion (y), (E) flocculation (x) against adhesion (y), and (F) invasion (x) against adhesion (y).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F3.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/9/1773/F3.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Some biofilm-related phenotypes are correlated. All data are for 31 haploid strains. Qualitative data shown are for a single haploid replicate; quantitative data are averaged across two biological replicates. Mean quantitative data for (A) flocculation, (B) invasion, and (C) adhesion are shown on the y-axis binned according to haploid colony complexity scores (x-axis). Mean quantitative data are plotted for correlation, with (D) flocculation (x) against invasion (y), (E) flocculation (x) against adhesion (y), and (F) invasion (x) against adhesion (y).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">It is noteworthy that these statistical and pairwise comparisons of phenotypes fail to include any phylogenetic information about these strains, and it is possible that some of the weak correlations observed between phenotypes are not representative of true correlations but rather of genetic similarity between the strains being compared. To begin to address this possibility, we examined the distribution of phenotypes across the phylogeny for these strains (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) generated in the Yeast Resource Center browser (<a href="http://www.yeastrc.org/g2p/">http://www.yeastrc.org/g2p/</a>). Filled bubbles on each tree represent the phenotypic scores, with a color scale based on the binned quantitative values of the phenotypes. There are examples of genetically similar strains with matching phenotypes (<em>e.g.</em>, YPS606 and YPS128, UWOPS05-217.3, and UWOPS05-227.2) but also genetically similar pairs with very different phenotypic signatures (<em>e.g.</em>, DBVPG6765 and L-1374).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic similarity is not the only predictor of phenotype. (A) 22 SGRP strains clustered by genome sequence in the YRC phenome browser (http://www.yeastrc.org/g2p/). Qualitative and quantitative phenotypic data are shown above each branch, with increasing color intensity corresponding to increasing score. Qualitative colony data are scored and colored from intensity 0 to 5. Quantitative data were binned into 5 corresponding color intensities. (B) Tree generated from FLO1 gene sequence. Quantitative flocculation data are shown above each branch. (C) Tree generated from FLO11 gene sequence. Qualitative colony morphology and quantitative invasion data are shown above each branch." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic similarity is not the only predictor of phenotype. (A) 22 SGRP strains clustered by genome sequence in the YRC phenome browser (http://www.yeastrc.org/g2p/). Qualitative and quantitative phenotypic data are shown above each branch, with increasing color intensity corresponding to increasing score. Qualitative colony data are scored and colored from intensity 0 to 5. Quantitative data were binned into 5 corresponding color intensities. (B) Tree generated from &lt;em&gt;FLO1&lt;/em&gt; gene sequence. Quantitative flocculation data are shown above each branch. (C) Tree generated from &lt;em&gt;FLO11&lt;/em&gt; gene sequence. Qualitative colony morphology and quantitative invasion data are shown above each branch.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F4.medium.gif" width="194" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/9/1773/F4.medium.gif" width="194" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Genetic similarity is not the only predictor of phenotype. (A) 22 SGRP strains clustered by genome sequence in the YRC phenome browser (<a href="http://www.yeastrc.org/g2p/">http://www.yeastrc.org/g2p/</a>). Qualitative and quantitative phenotypic data are shown above each branch, with increasing color intensity corresponding to increasing score. Qualitative colony data are scored and colored from intensity 0 to 5. Quantitative data were binned into 5 corresponding color intensities. (B) Tree generated from <em>FLO1</em> gene sequence. Quantitative flocculation data are shown above each branch. (C) Tree generated from <em>FLO11</em> gene sequence. Qualitative colony morphology and quantitative invasion data are shown above each branch.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Because the genome-wide phylogeny of these strains fails to account for the different evolutionary histories of individual genome regions, we also examined these traits at the level of single genes of interest. There are dozens of candidate genes that have been identified for their involvement in biofilm-related phenotypes in QTL studies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brauer <em>et al.</em> 2006</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wilkening <em>et al.</em> 2014</a>) and deletion collection studies (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Granek and Magwene 2010</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Ryan <em>et al.</em> 2012</a>). From these identified candidate genes we focused on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em>, because they have also been extensively shown experimentally to be involved in flocculation (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Miki <em>et al.</em> 1982</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Smukalla <em>et al.</em> 2008</a>) and cell–surface interactions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lo and Dranginis 1996</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">1998</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Vachova <em>et al.</em> 2011</a>), respectively. The gene trees were generated in the Yeast Resource Center phenome project browser using ORF sequences and represent 22 resequenced strains from the collection. In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>, six of the most flocculent strains are shown juxtaposed on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> gene tree; four cluster together while two strains, YJM981 and SK1, are genetically divergent at that locus. This indicates a possible role for certain very similar alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> in the flocculation phenotype, with YJM981 and SK1 as possible points for comparison. YJM981 also has a different phenotype from its most genetically similar strain at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> locus, YJM978. The relationship between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> genetic variation and phenotype is stronger (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). Two phenotypes are shown here, complex colony morphology and invasion, both of which are known to be associated with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> expression (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Galitski <em>et al.</em> 1999</a>; <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Granek and Magwene 2010</a>). There are several instances of genetically similar pairs with identical or similar phenotypes (YS9 and Y12, YPS606 and YPS128, UWOPS05-217.3 and UWOPS05-227.2, NCYC361 and YJM975, 376804X and BC187), and instances of strains similarly related genetically at this locus with different phenotypes show weak differences only (SK1 and Y55, YJM978 and YJM981). To effectively understand the role of genetic variation in these biofilm-related phenotypes, a more targeted approach might be successful.</p></div><div id="sec-21" class="subsection"><h3>Phenotypic pattern does not cluster according to environmental or geographical niche</h3><p id="p-36">To determine if any of the phenotypic fingerprints of these strains clustered by their niche of origin, we examined the hierarchical clustering of the qualitative and quantitative scores by Euclidean distance using the simplified designations for each strain provided by <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Liti <em>et al.</em> (2009)</a>. In <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, two primary clusters emerge with strong phenotypes, with the cluster including strains DBVPG6044 through YJM978 driven by strong colony morphology and mat complexity, and the cluster including strains SK1 through YJM981 with strong phenotypes across all assays. No strong clusters emerge for the rest of the strains with weaker phenotypes. Some strains with similar origin niches do cluster together, including Pennsylvania strains YPS128 and YPS606 in the top cluster and Malaysian strains UWOPS05-217.3 and UWOPS05-227.2 in the second cluster. Both of these pairs also have a high degree of sequence similarity, shown through their relatedness at both the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> loci and genome-wide in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>. However, there are also several exceptions where strains with very similar niches do not cluster together. Examples include the five clinical strains, cactus strains UWOPS87-242.1 and UWOPS83-787.3, and Chilean must strains L-1374 and L-1528, which do not cluster together phenotypically despite sharing the same ecological and geographical niches.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Few strains cluster phenotypically according to niche of origin. Hierarchical clustering of qualitative phenotypes (scores 0 to 5) and quantitative phenotypes (linear normalization to scale 0 to 5) for 31 haploid strains with simplified niche labels according to Liti et al. (2009). Cluster 3.0 (De Hoon et al. 2004) generated dendrogram based on Euclidean distance between strains; clustering visualized in JavaTreeView (Saldanha 2004). No phenotype (0) registers as white; very strong phenotype (5) registers as black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Few strains cluster phenotypically according to niche of origin. Hierarchical clustering of qualitative phenotypes (scores 0 to 5) and quantitative phenotypes (linear normalization to scale 0 to 5) for 31 haploid strains with simplified niche labels according to Liti &lt;em&gt;et al.&lt;/em&gt; (2009). Cluster 3.0 (De Hoon &lt;em&gt;et al.&lt;/em&gt; 2004) generated dendrogram based on Euclidean distance between strains; clustering visualized in JavaTreeView (Saldanha 2004). No phenotype (0) registers as white; very strong phenotype (5) registers as black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F5.medium.gif" width="333" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/9/1773/F5.medium.gif" width="333" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464971" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Few strains cluster phenotypically according to niche of origin. Hierarchical clustering of qualitative phenotypes (scores 0 to 5) and quantitative phenotypes (linear normalization to scale 0 to 5) for 31 haploid strains with simplified niche labels according to <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Liti <em>et al.</em> (2009)</a>. Cluster 3.0 (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">De Hoon <em>et al.</em> 2004</a>) generated dendrogram based on Euclidean distance between strains; clustering visualized in JavaTreeView (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Saldanha 2004</a>). No phenotype (0) registers as white; very strong phenotype (5) registers as black.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Colony morphology and agar invasion phenotypes in some natural isolates might be driven by prion content</h3><p id="p-38">We hypothesized that nongenetic causes may affect some of the correlations and lack thereof that we observed. Previous work on natural isolates has shown that up to 8% of natural isolates may harbor prions (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Halfmann <em>et al.</em> 2012</a>). Some of the phenotypic changes in natural isolates cured for prions include differences in colony morphology, invasion, and flocculation phenotypes (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Holmes <em>et al.</em> 2013</a>). To determine if the biofilm-related phenotypes in these natural isolates were driven by prions, we cured the strains of prions according to the procedure outlined by <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Holmes <em>et al.</em> (2013)</a> (see <em>Materials and Methods</em>). We tested strains after passaging for respiration competence on glycerol plates and six strains repeatedly failed to be respiration competent after curing. For the remaining 25 cured strains, we completed a full phenotypic panel across four assays (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS2.pdf">File S2</a>), excluding complex mat formation because of its strong correlation with complex colony morphology. For nine of those strains, we also generated an additional, separately cured replicate to confirm the cured phenotype; these replicates are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS2.pdf">File S2</a> as well. We observed no significant phenotypic differences from the haploid strains in our cured strains across all phenotypes except colony morphology and agar invasion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS2.pdf">Figure S2</a>). Three strains (273614N, YJM978, and YPS128) showed a reduction or variation in complex colony morphology after curing. Two additional cured strains (DBVPG1853 and W303) exhibited changes in colony morphology that were not carried through all cured replicates. These five strains had little shared history, spanning four ecological and five geographical niches. The agar invasion phenotype has both a characteristic intensity and a characteristic invasion pattern for each strain that is highly reproducible across biological replicates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS1.pdf">File S1</a>). For strains YPS128, DBVPG1106, Y12, YS2, YS4, and YS9, the prion-cured strains exhibited changes in both invasion intensity and pattern; Y12 also demonstrated differences between cured replicates. These differences are reflected in the correlation plot for haploid <em>vs.</em> prion-cured strains across the invasion phenotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS2.pdf">Figure S2</a>B) with an R<sup>2</sup> value of 0.83 lower than the haploid biological replicates (R<sup>2</sup> = 0.95). The six strains with altered invasion phenotypes in their prion-cured versions fall across four ecological niches and five geographical niches; all of the baker strains (YS2, YS4, YS9) and all of the Australian strains (DBVPG1106, YS2) in the collection potentially harbor prions.</p></div><div id="sec-23" class="subsection"><h3>Diploid strains show changes in correlations between biofilm-related phenotypes</h3><p id="p-39">To determine how ploidy may affect these traits, we completed a full phenotypic panel of all six assays, including complex mat formation, on the homothallic diploid SGRP collection. The complete panel is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS4.pdf">File S4</a>. We also examined an additional phenotype, filamentous growth, which is specific to the diploid strains (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Gimeno <em>et al.</em> 1992</a>; <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 1996</a>). We binned the quantitative invasive growth scores into five groups according to the same metric used in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and compared those scores with the qualitative filamentous growth scores with a Kendall’s tau rank correlation test. These two phenotypes are significantly correlated, with a <em>P</em>-value of 0.04. We repeated the low-nitrogen filamentous growth assay for the haploid collection as a control to ensure that none of the haploid strains demonstrated a filamentous growth phenotype. We observed no filamentous growth for any haploid strains (data not shown).</p><p id="p-40">We repeated the correlation analyses we performed on the diploid collection to observe how the relationships between traits change in the diploids. We compared the quantitative diploid data across the three quantitative traits (flocculation, invasion, adhesion) with the complex colony morphology scores and observed a weak trend of increasing colony morphology associated with increased invasion (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>), similar to the haploids. We also performed a Kendall’s tau rank correlation test on the diploid qualitative data; we found that the relationship between the colony morphology and mat formation assays changed with the change in ploidy, with the previous correlation lost (new <em>P</em> = 0.08, above the significance threshold <em>P</em> &lt; 0.05). Several other correlations are lost in the diploid data, as shown in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>. There is no correlation between any of the quantitative phenotypes (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, D–F</a>) with the correlation between flocculation and adhesion we observed in the haploids specifically lost in the diploids (R<sup>2</sup> = 0.005) (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6E</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diploid strains show weaker correlations between biofilm-related phenotypes. All data are for 24 diploid strains and averaged across technical replicates (see Materials and Methods). Mean quantitative data for (A) flocculation, (B) invasion, and (C) adhesion are shown on the y-axis binned according to diploid colony complexity scores (x-axis). Mean quantitative data are plotted for correlation, with (D) flocculation (x) against invasion (y), (E) flocculation (x) against adhesion (y), and (F) invasion (x) against adhesion (y)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diploid strains show weaker correlations between biofilm-related phenotypes. All data are for 24 diploid strains and averaged across technical replicates (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Mean quantitative data for (A) flocculation, (B) invasion, and (C) adhesion are shown on the y-axis binned according to diploid colony complexity scores (x-axis). Mean quantitative data are plotted for correlation, with (D) flocculation (x) against invasion (y), (E) flocculation (x) against adhesion (y), and (F) invasion (x) against adhesion (y).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F6.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/9/1773/F6.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464852" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Diploid strains show weaker correlations between biofilm-related phenotypes. All data are for 24 diploid strains and averaged across technical replicates (see <em>Materials and Methods</em>). Mean quantitative data for (A) flocculation, (B) invasion, and (C) adhesion are shown on the y-axis binned according to diploid colony complexity scores (x-axis). Mean quantitative data are plotted for correlation, with (D) flocculation (x) against invasion (y), (E) flocculation (x) against adhesion (y), and (F) invasion (x) against adhesion (y).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h3>Diploid strains do not uniformly have reduced phenotypic strength with respect to haploid strains</h3><p id="p-42">The strength and complexity of several of these biofilm-related phenotypes have been reported to be influenced by ploidy. <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Galitski <em>et al.</em> (1999)</a> and <a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Reynolds and Fink (2001)</a> observed that an increase in ploidy in laboratory strain Σ1278B results in both decreased invasive ability on agar as well as decreased mat complexity. A correlation between increased ploidy and decreased complex colony morphology across natural isolates has also been reported (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Granek and Magwene 2010</a>).</p><p id="p-43">Across the five primary phenotypic assays, we observed that the strength of the phenotypes differed with respect to the haploid strains and were generally attenuated across all phenotypes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). No diploid strains achieved a complex colony morphology score of 5 (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6, A–C</a>), a complex mat score of 5, or a flocculation score of more than 0.3, where the highest haploid flocculation score was 0.8 (strain UWOPS83-787.3) and 16 strains achieved a score of 0.3 or higher. These trends are reflected in the haploid/diploid correlation plots in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>. In <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a> and <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>, the low slopes (m = −0.05 and m = 0.31, respectively) and correlation coefficients (R<sup>2</sup> = 0.04 and R<sup>2</sup> = 0.27) demonstrate that in the flocculation and polystyrene adhesion assays, diploid strains have very different phenotypes from their haploid counterparts and these phenotypes are weaker than the haploid phenotypes. These numbers are particularly striking compared with the haploid/prion-cured differences, which should be negligible if the phenotypes are not strongly influenced by prions. In fact, we do find that to be the case with 20 out of 26, 20 out of 23, and 23 out of 25 prion-cured strains showing no significant change in flocculation, invasion, and adhesion, respectively, compared with the haploids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS4.pdf">Table S4</a>). These values confirm that the magnitude of the changes in the diploid strains is well outside the range of biological noise in the assay. The invasion assay correlation, shown in <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>, shows a different pattern. The high R<sup>2</sup> value of 0.74 is slightly lower than the biological replicate noise level (R<sup>2</sup> of 0.95 for invasion) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS1.pdf">Figure S1</a>C) and the slope m = 0.77 close to 1 indicates that the haploid and diploid quantitative invasion values are very similar. Contrary to predictions from literature, we observe two cases of increases in invasive ability in the diploid strains (shown in <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This increase in invasive ability was statistically significant in a <em>q</em>-value test between average haploid and diploid values for strains UWOPS87-242.1 and UWOPS83-787.3. This increase in invasive ability is also visible in the invasion images (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diploid quantitative phenotypes are weaker than haploid quantitative phenotypes except for invasion. (A) Mean quantitative haploid flocculation data for two biological replicates (x-axis) are plotted against quantitative diploid flocculation data (y-axis) for correlation. (B) Mean quantitative haploid invasion data for two biological replicates (x-axis) are plotted against quantitative diploid invasion data (y-axis) for correlation. (C) Mean quantitative haploid polystyrene adhesion data for two biological replicates (x-axis) are plotted against quantitative diploid polystyrene adhesion data (y-axis) for correlation. All quantitative haploid and diploid data are available in File S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diploid quantitative phenotypes are weaker than haploid quantitative phenotypes except for invasion. (A) Mean quantitative haploid flocculation data for two biological replicates (x-axis) are plotted against quantitative diploid flocculation data (y-axis) for correlation. (B) Mean quantitative haploid invasion data for two biological replicates (x-axis) are plotted against quantitative diploid invasion data (y-axis) for correlation. (C) Mean quantitative haploid polystyrene adhesion data for two biological replicates (x-axis) are plotted against quantitative diploid polystyrene adhesion data (y-axis) for correlation. All quantitative haploid and diploid data are available in File S3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F7.medium.gif" width="195" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/9/1773/F7.medium.gif" width="195" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-44" class="first-child">Diploid quantitative phenotypes are weaker than haploid quantitative phenotypes except for invasion. (A) Mean quantitative haploid flocculation data for two biological replicates (x-axis) are plotted against quantitative diploid flocculation data (y-axis) for correlation. (B) Mean quantitative haploid invasion data for two biological replicates (x-axis) are plotted against quantitative diploid invasion data (y-axis) for correlation. (C) Mean quantitative haploid polystyrene adhesion data for two biological replicates (x-axis) are plotted against quantitative diploid polystyrene adhesion data (y-axis) for correlation. All quantitative haploid and diploid data are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS3.txt">File S3</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464889/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464889/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Quantitative changes in diploid strains <em>vs.</em> haploid strains</span><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of phenotypic differences depending on ploidy. Three representative strains are listed with their formal name and origin and are shown across five different phenotypes, with ploidy designated &#x201C;1n&#x201D; or &#x201C;2n&#x201D; for haploid and diploid strains. Red boxes indicate notable changes in phenotype between haploid and diploid versions of each representative strain. Image details and experimental methods are the same as Figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1562023974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples of phenotypic differences depending on ploidy. Three representative strains are listed with their formal name and origin and are shown across five different phenotypes, with ploidy designated &#x201C;1n&#x201D; or &#x201C;2n&#x201D; for haploid and diploid strains. Red boxes indicate notable changes in phenotype between haploid and diploid versions of each representative strain. Image details and experimental methods are the same as Figure 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1773/F8.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/9/1773/F8.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1773/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464834" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-50" class="first-child">Examples of phenotypic differences depending on ploidy. Three representative strains are listed with their formal name and origin and are shown across five different phenotypes, with ploidy designated “1n” or “2n” for haploid and diploid strains. Red boxes indicate notable changes in phenotype between haploid and diploid versions of each representative strain. Image details and experimental methods are the same as <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Although we saw examples of diploid strains with weakened phenotypes across all metrics, there were several notable counterexamples. In <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>, weakened phenotypes are shown for strains DBVPG1853 and UWOPS87-242.1, which lose their polystyrene adhesive ability as diploids, and strains UWOPS83-787.3 and UWOPS87-242.1, which both lose their strong flocculation ability as diploids. However, some exceptions emerge: strain DBVPG1853 gains a complex mat as a diploid, and invasive ability improves in the diploid versions of UWOPS83-787.3 and UWOPS87-242.1, especially for UWOPS83-787.3. Across qualitative scores for a single haploid biological replicate, nine strains showed a more complex mat as diploids than as haploids, although in five of these cases the qualitative scores show a weak gain of complexity only (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/TableS3.pdf">Table S3</a>). The exceptions in the quantitative assays are also highlighted in a statistical comparison of the haploid and diploid data shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, which shows the number of diploid strains that showed a significant increase or decrease across each phenotypic category with respect to the haploid strains (see <em>Materials and Methods</em>). With those conditions applied, 22 of 23 tested strains had a significant decrease in flocculation as diploids, but only six strains had a significant decrease in invasive ability. The most common scenario was to see no significant difference between haploids and diploids (15 out of 23 strains for invasion, 13 out of 24 strains for adhesion). Instances of no significant difference were not dominated by strains with weak phenotypes; of the 15 strains with no significant difference in invasive ability, only six strains had diploid mean gray intensity values less than 10.</p></div></div><div class="section discussion" id="sec-25"><h2 class="">Discussion</h2><p id="p-52">Understanding connections between genotype and phenotype is one of the primary driving motivations behind biology and genomics today, yet many of the tools we are using to answer this question are constrained by lack of phenotypic diversity. Biofilm-related phenotypes in yeast, for example, are limited in laboratory strains and only now with natural yeast isolates are we able to examine the full diversity of phenotypes and their possible genetic origins. In this study we have demonstrated the incredible amount of phenotypic diversity present across biofilm-related phenotypes in just 30 natural isolates, a limited panel. With this phenotypic panel, we have shown that despite the predicted involvement of similar genetic pathways, many of these biofilm-related phenotypes are uncorrelated or weakly correlated with each other and are potentially controlled by different genetic networks. We have also demonstrated that the majority of phenotypes we observe are not due to prions and are variable depending on ploidy, but not in predictable ways. General conclusions about the effects of ploidy on biofilm-related phenotypes have been drawn in the literature based on only a few strain backgrounds and, although these trends are generally observed in our data, there are also notable exceptions. These findings demonstrate the importance of using natural isolates to understand genotype–phenotype relationships because many predictions made in laboratory strains do not hold true for natural isolates.</p><p id="p-53">In comparing complex colony morphology, complex mat formation, flocculation/settling, agar invasion, and polystyrene adhesion/crystal violet phenotypes in haploid natural isolates, we found two strongly correlated relationships (complex colony morphology and complex mat formation; flocculation and polystyrene adhesion) and two weakly correlated relationships (complex colony morphology and invasion; invasion and polystyrene adhesion). Given that most of the relationships we observed in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a> were weakly correlated or uncorrelated, this suggests complex genetic interactions underlying each phenotype, although some of the stronger correlations could be explained by shared genetic drivers. All of these traits have been shown in previous work to be dependent on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> activity (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lo and Dranginis 1998</a>; <a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">Reynolds and Fink 2001</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Granek <em>et al.</em> 2013</a>), including flocculation under specific conditions (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bayly <em>et al.</em> 2005</a>), although flocculation is primarily considered to be a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em>-mediated trait (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Smukalla <em>et al.</em> 2008</a>). This difference in genetic control is potentially responsible for the lack of correlation we observed between quantitative flocculation and invasion values (R<sup>2</sup> = 0.03), which is contrary to the correlation between these phenotypes observed in <em>S. paradoxus</em> (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Roop and Brem 2013</a>). It would have been equally unsurprising to see a lack of correlation between the flocculation and polystyrene adhesion assays, but it was not what we observed in the quantitative data (R<sup>2</sup> = 0.47); in fact, it was the strongest correlation we observed. The caveat to this relationship is that we observed that clumpy strains are less likely to form a consistent monolayer of cells in the polystyrene adhesion assay, instead retaining some of their clumpiness in the biofilm. These clumps absorb more of the crystal violet stain, potentially skewing the adhesion results in the direction of particularly flocculent strains. This could potentially be addressed in the future by treating the cultures with a deflocculation buffer prior to either biofilm formation or staining. More interesting is the weak correlation between the invasion phenotype and the polystyrene sticking phenotype (R<sup>2</sup> = 0.29), as previous studies have shown that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> is involved in the manifestation of both phenotypes. This suggests that though these phenotypes have a connection to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001458">FLO11</a></em> activity, they are likely regulated by more complex genetic interactions.</p><p id="p-54">Several of these phenotypes in the <em>S. cerevisiae</em> Σ1278B knockout collection were recently examined (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Ryan <em>et al.</em> 2012</a>) and it was found that, with a few exceptions, different “growth programs” like filamentous growth and mat formation were controlled by many different genes in an only semi-overlapping fashion. Given that we observed only one very strong correlation between biofilm-related phenotypes (complex colony morphology and complex mat formation), this compartmentalized management of growth programs is what our data suggest as well. The practical outcome of these findings is that to consider the biofilm-related phenotype of one of these natural isolate strains fully, many different assays are needed to accurately describe the characteristics of the strain. The independence of these phenotypes further provides future opportunities to differentiate and classify the genes and pathways affecting each biofilm-related phenotype.</p><p id="p-55">We hypothesized that diploid phenotypes would be weaker in general than the haploid phenotypes for the corresponding strains, according to the work of <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Galitski <em>et al.</em> (1999)</a>, <a id="xref-ref-35-9" class="xref-bibr" href="#ref-35">Reynolds and Fink (2001)</a>, and <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Granek and Magwene (2010)</a>, who each showed that strength of invasion, strength of complex mat formation, and strength of complex colony morphology decrease with ploidy. We observed this same general trend across all phenotypes with some interesting exceptions; across natural isolates, the relationship between the haploid and diploid phenotypes varied according to strain and phenotype examined. Within our representative set of strains in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>, there are examples of phenotypes both stronger and weaker in the diploid strains, which is a characteristic of the collection as a whole (see haploid strains in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS1.pdf">File S1</a> and diploid strains in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FileS4.pdf">File S4</a>). Most importantly, we have recorded examples in this dataset of significantly increased phenotypic strength in the diploid strains across the complex mat formation and invasion assays; both assays had an attenuated diploid phenotype that was predicted based on experiments in laboratory strains. Our finding that predictions about the effect of ploidy on biofilm-related phenotypes are not necessarily borne out in natural isolates is consistent with previous findings that were discovered on examining ploidy–environment interactions for SGRP haploid and diploid natural isolates, and countless exceptions to predictions of how ploidy would impact a strain’s response to environmental perturbations were also discovered (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Zorgo <em>et al.</em> 2013</a>). Other studies with this collection suggest laboratory strain S288C is a phenotypic outlier with respect to natural isolates (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Warringer <em>et al.</em> 2011</a>), a further reminder that the phenotypes recorded from laboratory strains of <em>S. cerevisiae</em> are not necessarily generalizable to natural isolates. The SGRP collection and other collections of natural isolates will allow us to test predictions generated from analyzing laboratory strains and to see under what conditions those predictions hold true.</p><p id="p-56">Given a larger subset of strains, the conclusions we draw about the correlations between these phenotypes would undoubtedly be refined. With a sample set of more than 30 strains, the data generated about phenotypic relationships and especially niche relationships would particularly benefit; for some of the niches in which we observed clustering (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), there were only a few representatives from a given niche (<em>e.g.</em>, only three African strains, only three baking strains, only two Malaysian strains). As a result, the conclusions we draw come with a significant caveat. Although our clustering analysis is likely underpowered, the lack of correlation we saw between strength of biofilm-related traits and any particular niche is consistent with observations from <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Warringer <em>et al.</em> (2011)</a>, who demonstrated that trait variation across an enormous panel of phenotypes was not driven primarily by the niche of origin.</p><p id="p-57">With a larger collection it would also be critical to develop high-throughput and potentially automated versions of the standard phenotypic assays described in this work. We could potentially refine the picture further by expanding the phenotypic assays to examine natural isolates’ behavior under nutrient-limited conditions, which have been shown to alter biofilm-related phenotypes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Palecek <em>et al.</em> 2002</a>; <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Granek and Magwene 2010</a>). There is also room to improve the scope of the quantitative assays. Right now, the assays only capture one primary dimension of each phenotype, but there is additional variation that stratifies the phenotypes even further. For example, in the flocculation assay plot profiles as shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1/FigureS4.pdf">Figure S4</a>, there is additional information about the way a strain settles in the slope and integral of the plot profile line that is not captured by taking a single distance measurement. Similarly, in the invasion assay as shown ranked in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>, strains with very similar quantitative invasion scores have clearly different invasion structures, with the majority of the intensity driven by a strong edge (<em>e.g.</em>, L-1528 and YS9), a punctate pattern (<em>e.g.</em>, 378604X), or a completely invaded patch (<em>e.g.</em>, Y12 and K11); this fine-scale information is lost in the current invasion quantification metric. There is also room to generate improved quantitative data across the two assays that remain qualitative in this study, particularly the complex colony morphology assay. Quantitative methods for measuring complex colony morphology are becoming available using time-course imaging and specialized image analysis software (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ruusuvuori <em>et al.</em> 2014</a>). In future work with natural isolates, it would be useful to assign quantitative scores to the complex colonies in this natural isolate collection as well.</p><p id="p-58">We anticipate that the phenotypic data generated in this study will be a valuable platform for examining genotype–phenotype questions in a targeted way. Now that we appreciate the complexity of biofilm-related phenotypes across these strains, we plan to examine the contributions of variation in individual alleles to each phenotype in the context of different genetic backgrounds. We also hope that the phenotypic data generated in this study will be a useful community resource for the many groups now working with strains from this collection. Ultimately, understanding the contributions of variation in known biofilm-related genes to each of these phenotypes could also contribute to effective engineering of strains with very specific biofilm-related traits.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">We thank Aimée Dudley for helpful comments on the manuscript; Celia Payen for editing and clustering advice; Rachel Gittelman, Maxwell Libbrecht, and Matthew Snyder for statistics consultations; Joshua Akey and Joseph Felsenstein for phylogeny advice; and Caitlin Connelly for information on the SGRP collection. This project was supported by grants R01 GM101091 and P41 GM103533 from the National Institute of General Medical Sciences from the National Institutes of Health. This material is based in part on work supported by the National Science Foundation under Cooperative Agreement No. DBI-0939454. E.H. was supported by T32 HG00035. M.D. is a Rita Allen Foundation Scholar and a Fellow in the Genetic Networks program at the Canadian Institute for Advanced Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013250/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-60"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-05-30"><span class="received-label">Received </span>May 30, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-07-17"><span class="accepted-label">Accepted </span>July 17, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Hope and Dunham</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abramoff</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magalhaes</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Image processing with ImageJ.</span> <abbr class="cit-jnl-abbrev">Biophotonics International</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophotonics%2BInternational%26rft.volume%253D11%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.2" data-doi="10.1007/s00253-010-2783-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govender</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bester</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Yeast flocculation and its biotechnological relevance.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DBauer%26rft.auinit1%253DF.%2BF.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D39%26rft.atitle%253DYeast%2Bflocculation%2Band%2Bits%2Bbiotechnological%2Brelevance.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-010-2783-0%26rft_id%253Dinfo%253Apmid%252F20676629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-010-2783-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20676629&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280914900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.3" data-doi="10.1016/j.femsyr.2005.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bayly</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Characteristics of Flo11-dependent flocculation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.femsyr.2005.05.004%26rft_id%253Dinfo%253Apmid%252F16043420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjk6IjUvMTIvMTE1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.4" data-doi="10.1111/j.1567-1364.2009.00482.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beauvais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loussert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prevost</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latgé</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of a biofilm-like extracellular matrix in <em>FLO1</em>-expressing <em>Saccharomyces cerevisiae</em> cells.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00482.x%26rft_id%253Dinfo%253Apmid%252F19207290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjkvMy80MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.5" data-doi="10.1093/molbev/msu037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barré</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-definition view of functional genetic variation from natural yeast genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu037%26rft_id%253Dinfo%253Apmid%252F24425782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzQvODcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.6" data-doi="10.1128/jb.179.15.4929-4936.1997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bony</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thines-Sempoux</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Localization and cell surface anchoring of the <em>Saccharomyces cerevisiae</em> flocculation protein Flo1p.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">4929</span>–<span class="cit-lpage">4936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DBony%26rft.auinit1%253DM.%26rft.volume%253D179%26rft.issue%253D15%26rft.spage%253D4929%26rft.epage%253D4936%26rft.atitle%253DLocalization%2Band%2Bcell%2Bsurface%2Banchoring%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bflocculation%2Bprotein%2BFlo1p.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.179.15.4929-4936.1997%26rft_id%253Dinfo%253Apmid%252F9244284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3OS8xNS80OTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.7" data-doi="10.1534/genetics.106.057927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christianson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping novel traits by array-assisted bulk segregant analysis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057927%26rft_id%253Dinfo%253Apmid%252F16624899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE4MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.8" data-doi="10.1111/j.1574-6976.2011.00275.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruckner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosch</span>  <span class="cit-name-given-names">H. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Choosing the right lifestyle: adhesion and development in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2011.00275.x%26rft_id%253Dinfo%253Apmid%252F21521246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjc6IjM2LzEvMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-4.9.1773.9"><div class="cit-metadata unstructured">Cromie, G. A., K. E. Hyma, C. L. Ludlow, C. Garmendia-Torres, T. L. Gilbert <em>et al.</em>, 2013 Genomic sequence diversity and population structure of <em>Saccharomyces cerevisiae</em> assessed by RAD-seq. G3 3<strong>:</strong> 2163–2171.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.10" data-doi="10.1111/j.1365-294X.2011.05005.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fargier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing the complex architecture of polygenic traits in diverged yeast populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCubillos%26rft.auinit1%253DF.%2BA.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D1401%26rft.epage%253D1413%26rft.atitle%253DAssessing%2Bthe%2Bcomplex%2Barchitecture%2Bof%2Bpolygenic%2Btraits%2Bin%2Bdiverged%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05005.x%26rft_id%253Dinfo%253Apmid%252F21261765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05005.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21261765&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288705300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.11" data-doi="10.1534/genetics.113.155515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scovacricchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155515%26rft_id%253Dinfo%253Apmid%252F24037264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.12" data-doi="10.1073/pnas.240345197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullen</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sprague</span>  <span class="cit-name-given-names">G. F.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2000</span> <span class="cit-article-title">Glucose depletion causes haploid invasive growth in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">13619</span>–<span class="cit-lpage">13624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.240345197%26rft_id%253Dinfo%253Apmid%252F11095711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjUvMTM2MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.13" data-doi="10.1093/bioinformatics/bth078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Hoon</span>  <span class="cit-name-given-names">M. J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyano</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Open Source Clustering Software.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1453</span>–<span class="cit-lpage">1454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth078%26rft_id%253Dinfo%253Apmid%252F14871861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMC85LzE0NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.14" data-doi="10.1186/gb-2005-6-4-r38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drees</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorsson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rives</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Derivation of genetic interaction networks from quantitative phenotype data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DDrees%26rft.auinit1%253DB.%2BL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253DR38%26rft.epage%253DR38%26rft.atitle%253DDerivation%2Bof%2Bgenetic%2Binteraction%2Bnetworks%2Bfrom%2Bquantitative%2Bphenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-4-r38%26rft_id%253Dinfo%253Apmid%252F15833125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-4-r38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15833125&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.15" data-doi="10.1126/science.285.5425.251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galitski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saldanha</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Ploidy Regulation of Gene Expression.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGalitski%26rft.auinit1%253DT.%26rft.volume%253D285%26rft.issue%253D5425%26rft.spage%253D251%26rft.epage%253D254%26rft.atitle%253DPloidy%2BRegulation%2Bof%2BGene%2BExpression%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5425.251%26rft_id%253Dinfo%253Apmid%252F10398601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyNS8yNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.16" data-doi="10.1016/0092-8674(92)90079-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimeno</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Unipolar cell divisions in the yeast <em>S. cerevisiae</em> lead to filamentous growth regulation by starvation and RAS.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGimeno%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D68%26rft.issue%253D6%26rft.spage%253D1077%26rft.epage%253D1090%26rft.atitle%253DUnipolar%2Bcell%2Bdivisions%2Bin%2Bthe%2Byeast%2BS.%2Bcerevisiae%2Blead%2Bto%2Bfilamentous%2Bgrowth%253A%2Bregulation%2Bby%2Bstarvation%2Band%2BRAS.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990079-R%26rft_id%253Dinfo%253Apmid%252F1547504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90079-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1547504&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HK67400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Environmental and genetic determinants of colony morphology in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.18" data-doi="10.1534/genetics.112.142067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayrkci</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic architecture of biofilm formation in a clinical isolate of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142067%26rft_id%253Dinfo%253Apmid%252F23172850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNTg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.19" data-doi="10.1073/pnas.220420397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A <em>Saccharomyces</em> gene family involved in invasive growth, cell-cell adhesion, and mating.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12158</span>–<span class="cit-lpage">12163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.220420397%26rft_id%253Dinfo%253Apmid%252F11027318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjIvMTIxNTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.20" data-doi="10.1038/nature10875"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halfmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarosz</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prions are a common mechanism for phenotypic inheritance in wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHalfmann%26rft.auinit1%253DR.%26rft.volume%253D482%26rft.issue%253D7385%26rft.spage%253D363%26rft.epage%253D368%26rft.atitle%253DPrions%2Bare%2Ba%2Bcommon%2Bmechanism%2Bfor%2Bphenotypic%2Binheritance%2Bin%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10875%26rft_id%253Dinfo%253Apmid%252F22337056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10875&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22337056&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300287100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ammar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nislow</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowen</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic and genomic architecture of the evolution of resistance to antifungal drug combinations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.22" data-doi="10.1016/j.cell.2013.02.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halfmann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable remodeling of yeast multicellularity by an environmentally responsive prion.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.02.026%26rft_id%253Dinfo%253Apmid%252F23540696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.02.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23540696&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316853700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.23" data-doi="10.1371/journal.pone.0086533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping genetic variants underlying differences in the central nitrogen metabolism in fermenter yeasts.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e86533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De86533%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0086533%26rft_id%253Dinfo%253Apmid%252F24466135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0086533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24466135&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.9.1773.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reader</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Genetic control of flocculation, pp. 205-224 in Yeast genetics: fundamental and applied aspects</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">J. F. T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">D. M. </span></span>, <span class="cit-ed"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. R. W.</span></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.25" data-doi="10.1093/biomet/30.1-2.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">A new measure of rank correlation.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DKENDALL%26rft.auinit1%253DM.%2BG.%26rft.volume%253D30%26rft.issue%253D1-2%26rft.spage%253D81%26rft.epage%253D93%26rft.atitle%253DA%2BNEW%2BMEASURE%2BOF%2BRANK%2BCORRELATION%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F30.1-2.81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJiaW9tZXQiO3M6NToicmVzaWQiO3M6OToiMzAvMS0yLzgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.26" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> S288C has a mutation in <em>FLO8</em>, a gene required for filamentous growth.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D967%26rft.epage%253D978%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BS288C%2BHas%2Ba%2BMutation%2Bin%2BFL08%252C%2Ba%2BGene%2BRequired%2Bfor%2BFilamentous%2BGrowth%26rft_id%253Dinfo%253Apmid%252F8913742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzMvOTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>FLO11</em>, a yeast gene related to the STA genes, encodes a novel cell surface flocculin.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">7144–7151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft.volume%253D178%26rft.spage%253D71447151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.29" data-doi="10.1091/mbc.9.1.161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The cell surface flocculin Flo11 is required for pseudohyphae formation and invasion by <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DLo%26rft.auinit1%253DW.-S.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D161%26rft.epage%253D171%26rft.atitle%253DThe%2BCell%2BSurface%2BFlocculin%2BFlo11%2BIs%2BRequired%2Bfor%2BPseudohyphae%2BFormation%2Band%2BInvasion%2Bby%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.1.161%26rft_id%253Dinfo%253Apmid%252F9436998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjkvMS8xNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.30" data-doi="10.1093/emboj/16.23.7008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Yeast pseudohyphal growth is regulated by <em>GPA2</em>, a G protein α homolog.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">7008</span>–<span class="cit-lpage">7018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DLorenz%26rft.auinit1%253DM.%2BC.%26rft.volume%253D16%26rft.issue%253D23%26rft.spage%253D7008%26rft.epage%253D7018%26rft.atitle%253DYeast%2Bpseudohyphal%2Bgrowth%2Bis%2Bregulated%2Bby%2BGPA2%252C%2Ba%2BG%2Bprotein%2B%257Balpha%257D%2Bhomolog%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.23.7008%26rft_id%253Dinfo%253Apmid%252F9384580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMjMvNzAwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miki</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seligy</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Possible mechanism for flocculation interactions governed by gene <em>FLO1</em> in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">878–889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft.volume%253D150%26rft.spage%253D878889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.32" data-doi="10.1046/j.1365-2958.1998.00797.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Toole</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolter</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Initiation of biofilm formation in <em>Pseudomonas fluorescens</em> WCS365 proceeds via multiple, convergent signalling pathways: a genetic analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DO%2527Toole%26rft.auinit1%253DG.%2BA.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D461%26rft.atitle%253DInitiation%2Bof%2Bbiofilm%2Bformation%2Bin%2BPseudomonas%2Bfluorescens%2BWCS365%2Bproceeds%2Bvia%2Bmultiple%252C%2Bconvergent%2Bsignalling%2Bpathways%253A%2Ba%2Bgenetic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1998.00797.x%26rft_id%253Dinfo%253Apmid%252F9632250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1998.00797.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9632250&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073797900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.33" data-doi="10.1146/annurev.micro.54.1.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Toole</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolter</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Biofilm formation as microbial development.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DO%2527Toole%26rft.auinit1%253DG.%26rft.volume%253D54%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D79%26rft.atitle%253DBIOFILM%2BFORMATION%2BAS%2BMICROBIAL%2BDEVELOPMENT%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.54.1.49%26rft_id%253Dinfo%253Apmid%252F11018124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.54.1.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11018124&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165272300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.34" data-doi="10.1099/00221287-148-4-893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palecek</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parikh</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kron</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Sensing, signalling and integrating physical processes during <em>Saccharomyces cerevisiae</em> invasive and filamentous growth.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DPalecek%26rft.auinit1%253DS.%2BP.%26rft.volume%253D148%26rft.issue%253D4%26rft.spage%253D893%26rft.epage%253D907%26rft.atitle%253DSensing%252C%2Bsignalling%2Band%2Bintegrating%2Bphysical%2Bprocesses%2Bduring%2BSaccharomyces%2Bcerevisiae%2Binvasive%2Band%2Bfilamentous%2Bgrowth%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-148-4-893%26rft_id%253Dinfo%253Apmid%252F11932437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-148-4-893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11932437&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175084400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.35" data-doi="10.1126/science.291.5505.878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bakers’ yeast, a model for fungal biofilm formation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">878</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReynolds%26rft.auinit1%253DT.%2BB.%26rft.volume%253D291%26rft.issue%253D5505%26rft.spage%253D878%26rft.epage%253D881%26rft.atitle%253DBakers%2527%2BYeast%252C%2Ba%2BModel%2Bfor%2BFungal%2BBiofilm%2BFormation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.291.5505.878%26rft_id%253Dinfo%253Apmid%252F11157168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTEvNTUwNS84NzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTc3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.36" data-doi="10.1534/genetics.113.155341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roop</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rare variants in hypermutable genes underlie common morphology and growth traits in wild <em>Saccharomyces paradoxus</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155341%26rft_id%253Dinfo%253Apmid%252F23934881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruusuvuori</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorsa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative analysis of colony morphology in yeast.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D56%26rft.spage%253D18%26rft_id%253Dinfo%253Apmid%252F24447135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24447135&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.38" data-doi="10.1126/science.1224339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurat</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayhew</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global gene deletion analysis exploring yeast filamentous growth.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRyan%26rft.auinit1%253DO.%26rft.volume%253D337%26rft.issue%253D6100%26rft.spage%253D1353%26rft.epage%253D1356%26rft.atitle%253DGlobal%2BGene%2BDeletion%2BAnalysis%2BExploring%2BYeast%2BFilamentous%2BGrowth%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1224339%26rft_id%253Dinfo%253Apmid%252F22984072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjEwMC8xMzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.39" data-doi="10.1093/bioinformatics/bth349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saldanha</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Java Treeview–extensible visualization of microarray data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3246</span>–<span class="cit-lpage">3248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth349%26rft_id%253Dinfo%253Apmid%252F15180930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTcvMzI0NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1773.40"><div class="cit-metadata unstructured">Sherman, F., 1998 An introduction to the genetics and molecular biology of the yeast <em>Saccharomyces cerevisiae</em>. Encyclopedia Mol. Biol. Mol. Med. 6: 302–325.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.41" data-doi="10.1002/elsc.200900061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sivakumar</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vail</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lay</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bioethanol and biodiesel: Alternative liquid fuels for future generations.</span> <abbr class="cit-jnl-abbrev">Eng. Life Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEng.%2BLife%2BSci.%26rft.volume%253D10%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felsc.200900061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elsc.200900061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.42" data-doi="10.1101/gr.155762.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merrihew</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1496</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.155762.113%26rft_id%253Dinfo%253Apmid%252F23720455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTQ5NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.43" data-doi="10.1016/j.cell.2008.09.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smukalla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pochet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beauvais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guadagnini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>FLO1</em> is a variable green beard gene that drives biofilm-like cooperation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">726</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSmukalla%26rft.auinit1%253DS.%26rft.volume%253D135%26rft.issue%253D4%26rft.spage%253D726%26rft.epage%253D737%26rft.atitle%253DFLO1%2Bis%2Ba%2Bvariable%2Bgreen%2Bbeard%2Bgene%2Bthat%2Bdrives%2Bbiofilm-like%2Bcooperation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.09.037%26rft_id%253Dinfo%253Apmid%252F19013280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.09.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19013280&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260886900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.44" data-doi="10.1111/1467-9868.00346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A direct approach to false discovery rates.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D64%26rft.spage%253D479%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.45" data-doi="10.1016/j.fgb.2010.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stovicek</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vachova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuthan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palkova</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">General factors important for the formation of structured biofilm-like yeast colonies.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1012</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DStovicek%26rft.auinit1%253DV.%26rft.volume%253D47%26rft.issue%253D12%26rft.spage%253D1012%26rft.epage%253D1022%26rft.atitle%253DGeneral%2Bfactors%2Bimportant%2Bfor%2Bthe%2Bformation%2Bof%2Bstructured%2Bbiofilm-like%2Byeast%2Bcolonies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2010.08.005%26rft_id%253Dinfo%253Apmid%252F20728557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2010.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20728557&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.46" data-doi="10.1083/jcb.201103129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vachova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stovicek</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hlavacek</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernyavskiy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stepanek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Flo11p, drug efflux pumps, and the extracellular matrix cooperate to form biofilm yeast colonies.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201103129%26rft_id%253Dinfo%253Apmid%252F21875945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE5NC81LzY3OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.47" data-doi="10.1111/j.1365-2958.2006.05072.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klis</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Flocculation, adhesion and biofilm formation in yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DVerstrepen%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D15%26rft.atitle%253DFlocculation%252C%2Badhesion%2Band%2Bbiofilm%2Bformation%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05072.x%26rft_id%253Dinfo%253Apmid%252F16556216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05072.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16556216&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.48" data-doi="10.1007/s00253-002-1200-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derdelinckx</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verachtert</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delvaux</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Yeast flocculation: what brewers should know.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DVerstrepen%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D197%26rft.epage%253D205%26rft.atitle%253DYeast%2Bflocculation%253A%2Bwhat%2Bbrewers%2Bshould%2Bknow.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-002-1200-8%26rft_id%253Dinfo%253Apmid%252F12698276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-002-1200-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12698276&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183198900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.49" data-doi="10.1038/nrmicro927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Origins of variation in the fungal cell surface.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DVerstrepen%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D533%26rft.epage%253D540%26rft.atitle%253DOrigins%2Bof%2Bvariation%2Bin%2Bthe%2Bfungal%2Bcell%2Bsurface.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro927%26rft_id%253Dinfo%253Apmid%252F15197389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro927&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15197389&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222460900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.50" data-doi="10.1111/j.1365-2958.2012.08192.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voordeckers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Maeyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Zande</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinces</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meert</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of a complex genetic network underlying <em>Saccharomyces cerevisiae</em> colony morphology.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DVoordeckers%26rft.auinit1%253DK.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D239%26rft.atitle%253DIdentification%2Bof%2Ba%2Bcomplex%2Bgenetic%2Bnetwork%2Bunderlying%2BSaccharomyces%2Bcerevisiae%2Bcolony%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08192.x%26rft_id%253Dinfo%253Apmid%252F22882838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08192.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22882838&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.51" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.52" data-doi="10.1534/genetics.113.160291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkening</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekkedil</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An Evaluation of High-Throughput Approaches to QTL Mapping in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160291%26rft_id%253Dinfo%253Apmid%252F24374355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1773.53" data-doi="10.1371/journal.pgen.1003388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chwialkowska</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garre</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunnerhagen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ancient evolutionary trade-offs between yeast ploidy states.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003388%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003388%26rft_id%253Dinfo%253Apmid%252F23555297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555297&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1773.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1773.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1765" title="A New Dataset of Spermatogenic vs. Oogenic Transcriptomes in the Nematode Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1787" title="Gene Expression in the Skin of Dogs Sensitized to the House Dust Mite Dermatophagoides farinae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1773.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PybcZ-6n9FGB7TPz6uQe3mt-5V2Vx20xZaYNI7FkDG4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1773" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1773" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1773" class="active" data-icon-position="" data-hide-link-title="0">Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459667" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Uc8uXxnxaoeXFDiDVLhxUBVfereUFede0qAHOIeQmmI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1773.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1773" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZIb2VF-ew5vM2ND8HJ9HSF-2cKbzYqDVFUx0UIkiRR4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1773" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1773.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459667" id="node459667--2" data-pisa="ggg;4/9/1773" data-pisa-master="ggg;g3.114.013250" data-apath="/ggg/4/9/1773.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elyse A. Hope</span> and <span class="highwire-citation-author" data-delta="1">Maitreya J. Dunham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1773-1786;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013250 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459667--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elyse A. Hope </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elyse%2BA.%2BHope%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hope%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElyse%2BA.%2BHope%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1773" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1773&amp;title=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1773&amp;title=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1773&amp;title=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1773&amp;text=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1773&amp;title=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1773&amp;t=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1773&amp;title=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1773&amp;title=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459667/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459667" id="node459667--3" data-pisa="ggg;4/9/1773" data-pisa-master="ggg;g3.114.013250" data-apath="/ggg/4/9/1773.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ploidy-Regulated Variation in Biofilm-Related Phenotypes in Natural Isolates of <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elyse A. Hope</span> and <span class="highwire-citation-author" data-delta="1">Maitreya J. Dunham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1773-1786;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013250 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459667--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elyse A. Hope </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elyse%2BA.%2BHope%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hope%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElyse%2BA.%2BHope%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1773&amp;rft.epage=1786&amp;rft.atitle=Ploidy-Regulated%20Variation%20in%20Biofilm-Related%20Phenotypes%20in%20Natural%20Isolates%20of%20Saccharomyces%20cerevisiae&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hope%2C+Elyse+A.&amp;rft.au=Dunham%2C+Maitreya+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459667/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459667/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459667/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459667/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459667/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459667/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459667/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459667/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459667/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459667/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459667/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459667/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1773" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-25" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1773.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1773.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1773.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1773.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
