<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1717" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1717.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1717.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1717.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1717.atom" />
<meta name="HW.pisa" content="ggg;4/9/1717" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event" />
<meta name="DC.Identifier" content="10.1534/g3.114.012294" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Kodama et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Whole genome duplication has been implicated in evolutionary innovation and rapid diversification. In salmonid fishes, however, whole genome duplication significantly pre-dates major transitions across the family, and re-diploidization has been a gradual process between genomes that have remained essentially collinear. Nevertheless, pairs of duplicated chromosome arms have diverged at different rates from each other, suggesting that the retention of duplicated regions through occasional pairing between homeologous chromosomes may have played an evolutionary role across species pairs. Extensive chromosomal arm rearrangements have been a key mechanism involved in re-dipliodization of the salmonid genome; therefore, we investigated their influence on degree of differentiation between homeologs across salmon species. We derived a linkage map for coho salmon and performed comparative mapping across syntenic arms within the genus Oncorhynchus , and with the genus Salmo , to determine the phylogenetic relationship between chromosome arrangements and the retention of undifferentiated duplicated regions. A 6596.7 cM female coho salmon map, comprising 30 linkage groups with 7415 and 1266 nonduplicated and duplicated loci, respectively, revealed uneven distribution of duplicated loci along and between chromosome arms. These duplicated regions were conserved across syntenic arms across Oncorhynchus species and were identified in metacentric chromosomes likely formed ancestrally to the divergence of Oncorhynchus from Salmo . These findings support previous studies in which observed pairings involved at least one metacentric chromosome. Re-diploidization in salmon may have been prevented or retarded by the formation of metacentric chromosomes after the whole genome duplication event and may explain lineage-specific innovations in salmon species if functional genes are found in these regions." />
<meta name="DC.Contributor" content="Miyako Kodama" />
<meta name="DC.Contributor" content="Marine S. O. Brieuc" />
<meta name="DC.Contributor" content="Robert H. Devlin" />
<meta name="DC.Contributor" content="Jeffrey J. Hard" />
<meta name="DC.Contributor" content="Kerry A. Naish" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Whole genome duplication has been implicated in evolutionary innovation and rapid diversification. In salmonid fishes, however, whole genome duplication significantly pre-dates major transitions across the family, and re-diploidization has been a gradual process between genomes that have remained essentially collinear. Nevertheless, pairs of duplicated chromosome arms have diverged at different rates from each other, suggesting that the retention of duplicated regions through occasional pairing between homeologous chromosomes may have played an evolutionary role across species pairs. Extensive chromosomal arm rearrangements have been a key mechanism involved in re-dipliodization of the salmonid genome; therefore, we investigated their influence on degree of differentiation between homeologs across salmon species. We derived a linkage map for coho salmon and performed comparative mapping across syntenic arms within the genus &lt;i&gt;Oncorhynchus&lt;/i&gt;, and with the genus &lt;i&gt;Salmo&lt;/i&gt;, to determine the phylogenetic relationship between chromosome arrangements and the retention of undifferentiated duplicated regions. A 6596.7 cM female coho salmon map, comprising 30 linkage groups with 7415 and 1266 nonduplicated and duplicated loci, respectively, revealed uneven distribution of duplicated loci along and between chromosome arms. These duplicated regions were conserved across syntenic arms across &lt;i&gt;Oncorhynchus&lt;/i&gt; species and were identified in metacentric chromosomes likely formed ancestrally to the divergence of &lt;i&gt;Oncorhynchus&lt;/i&gt; from &lt;i&gt;Salmo&lt;/i&gt;. These findings support previous studies in which observed pairings involved at least one metacentric chromosome. Re-diploidization in salmon may have been prevented or retarded by the formation of metacentric chromosomes after the whole genome duplication event and may explain lineage-specific innovations in salmon species if functional genes are found in these regions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1717" />
<meta name="citation_id" content="4/9/1717" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1717" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1717.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1717.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1717.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012294" />
<meta name="citation_pmid" content="25053705" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1717" />
<meta name="citation_lastpage" content="1730" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Miyako Kodama" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105" />
<meta name="citation_author_email" content="mkodama@uw.edu" />
<meta name="citation_author_email" content="knaish@uw.edu" />
<meta name="citation_author" content="Marine S. O. Brieuc" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105" />
<meta name="citation_author" content="Robert H. Devlin" />
<meta name="citation_author_institution" content="Fisheries and Oceans Canada, West Vancouver, British Columbia, V7K 1N6, Canada" />
<meta name="citation_author" content="Jeffrey J. Hard" />
<meta name="citation_author_institution" content="National Marine Fisheries Service, Northwest Fisheries Science Center, Seattle, Washington 98112" />
<meta name="citation_author" content="Kerry A. Naish" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105" />
<meta name="citation_author_email" content="mkodama@uw.edu" />
<meta name="citation_author_email" content="knaish@uw.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3275-8778" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=M. A. Alexandrou;citation_author=B. A. Swartz;citation_author=N. J. Matzke;citation_author=T. H. Oakley;citation_title=Genome duplication and multiple evolutionary origins of complex migratory behavior in Salmonidae.;citation_pages=514-523;citation_volume=69;citation_year=2013;citation_pmid=23933489;citation_doi=10.1016/j.ympev.2013.07.026" />
<meta name="citation_reference" content="citation_title=Evolutionary Genetics of Fishes;citation_title=Tetraploidy and the evolution of salmonid fishes;citation_author=F. W. Allendorf;citation_author=G. H. Thorgaard;citation_pages=1-27;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Allendorf;citation_author=R. G. Danzmann;citation_title=Secondary Tetrasomic Segregation of MDH-B and Preferential Pairing of Homeologues in Rainbow Trout;citation_pages=1083-1092;citation_volume=145;citation_year=1997;citation_issue=4;citation_pmid=9093860" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. R. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Berthelot;citation_author=F. Brunet;citation_author=D. Chalopin;citation_author=A. Juanchich;citation_author=M. Bernard;citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.;citation_pages=3657;citation_volume=5;citation_year=2014;citation_pmid=24755649;citation_doi=10.1038/ncomms4657" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=V. Bourret;citation_author=M. P. Kent;citation_author=C. R. Primmer;citation_author=A. Vasemagi;citation_author=S. Karlsson;citation_title=SNP-array reveals genome-wide patterns of geographical and potential adaptive divergence across the natural range of Atlantic salmon (Salmo salar).;citation_pages=532-551;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12003" />
<meta name="citation_reference" content="citation_author=M. S. O. Brieuc;citation_author=C. D. Waters;citation_author=J. E. Seeb;citation_author=K. A. Naish;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cell Dev. Biol.;citation_author=C. Cañestro;citation_author=R. Albalat;citation_author=M. Irimia;citation_author=J. Garcia-Fernandez;citation_title=Impact of gene gains, losses and duplication modes on the origin and diversification of vertebrates.;citation_pages=83-94;citation_volume=24;citation_year=2013;citation_pmid=23291262;citation_doi=10.1016/j.semcdb.2012.12.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=STACKS: building and genotyping loci de novo from short-read sequences. G3: Genes, Genomes;citation_pages=171-182;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=STACKS: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Crête-Lafrenière;citation_author=L. K. Weir;citation_author=L. Bernatchez;citation_title=Framing the salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.;citation_pages=e46662-e46662;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071608;citation_doi=10.1371/journal.pone.0046662" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. D. Crow;citation_author=G. P. Wagner;citation_title=What is the role of genome duplication in the evolution of complexity and diversity?;citation_volume=23;citation_year=2006;citation_pmid=16368775;citation_doi=10.1093/molbev/msj083" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=R. G. Danzmann;citation_author=M. Cairney;citation_author=W. S. Davidson;citation_author=M. M. Ferguson;citation_author=K. Gharbi;citation_title=A comparative analysis of the rainbow trout genome with 2 other species of fish (Arctic charr and Atlantic salmon) within the tetraploid derivative Salmonidae family (subfamily: Salmoninae).;citation_pages=1037-1051;citation_volume=48;citation_year=2005;citation_issue=6;citation_pmid=16391673;citation_doi=10.1139/g05-067" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Danzmann;citation_author=E. Davidson;citation_author=M. Ferguson;citation_author=K. Gharbi;citation_author=B. Koop;citation_title=Distribution of ancestral proto-Actinopterygian chromosome arms within the genomes of 4R-derivative salmonid fishes (Rainbow trout and Atlantic salmon).;citation_pages=557-557;citation_volume=9;citation_year=2008;citation_pmid=19032764;citation_doi=10.1186/1471-2164-9-557" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=W. S. Davidson;citation_author=B. F. Koop;citation_author=S. J. M. Jones;citation_author=P. Iturra;citation_author=R. Vidal;citation_title=Sequencing the genome of the Atlantic salmon (Salmo salar).;citation_pages=403-403;citation_volume=11;citation_year=2010;citation_issue=9;citation_pmid=20887641;citation_doi=10.1186/gb-2010-11-9-403" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=R. H. Devlin;citation_author=C. A. Biagi;citation_author=D. E. Smailus;citation_title=Genetic mapping of Y-chromosomal DNA markers in Pacific salmon.;citation_pages=43-58;citation_volume=111;citation_year=2001;citation_issue=1-3;citation_pmid=11841186;citation_doi=10.1023/A:1013759802604" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=W. S. Eisbrenner;citation_author=N. Botwright;citation_author=M. Cook;citation_author=E. A. Davidson;citation_author=S. Dominik;citation_title=Evidence for multiple sex-determining loci in Tasmanian Atlantic salmon (Salmo salar).;citation_pages=86-92;citation_volume=113;citation_year=2014;citation_pmid=23759729;citation_doi=10.1038/hdy.2013.55" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=P. P. Edger;citation_author=J. C. Pires;citation_title=Gene and genome duplications: the impact of dosage-sensitivity on the fate of nuclear genes.;citation_pages=699-717;citation_volume=17;citation_year=2009;citation_issue=5;citation_pmid=19802709;citation_doi=10.1007/s10577-009-9055-9" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. V. Everett;citation_author=M. R. Miller;citation_author=J. E. Seeb;citation_title=Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.;citation_pages=521-521;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23031582;citation_doi=10.1186/1471-2164-13-521" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=M. Feldman;citation_author=A. A. Levy;citation_author=T. Fahima;citation_author=A. Korol;citation_title=Genomic asymmetry in allopolyploid plants: wheat as a model.;citation_volume=63;citation_year=2012;citation_pmid=22859676;citation_doi=10.1093/jxb/ers192" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Guyomard;citation_author=M. Boussaha;citation_author=F. Krieg;citation_author=C. Hervet;citation_author=E. Quillet;citation_title=A synthetic rainbow trout linkage map provides new insights into the salmonid whole genome duplication and the conservation of synteny among teleosts.;citation_pages=15-15;citation_volume=13;citation_year=2012;citation_pmid=22424132;citation_doi=10.1186/1471-2156-13-15" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=A. L. Hufton;citation_author=G. Panopoulou;citation_title=Polyploidy and genome restructuring: a variety of outcomes.;citation_pages=600-606;citation_volume=19;citation_year=2009;citation_issue=6;citation_pmid=19900800;citation_doi=10.1016/j.gde.2009.10.005" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=I. A. Hurley;citation_author=R. L. Mueller;citation_author=K. A. Dunn;citation_author=E. J. Schmidt;citation_author=M. Friedman;citation_title=A new time-scale for ray-finned fish evolution;citation_pages=489-498;citation_volume=274;citation_year=2007;citation_issue=1609;citation_pmid=17476768;citation_doi=10.1098/rspb.2006.3749;citation_georef=2013069924" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=O. Jaillon;citation_author=J. M. Aury;citation_author=F. Brunet;citation_author=J. L. Petit;citation_author=N. Stange-Thomann;citation_title=Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype.;citation_pages=946-957;citation_volume=431;citation_year=2004;citation_issue=7011;citation_pmid=15496914;citation_doi=10.1038/nature03025" />
<meta name="citation_reference" content="citation_journal_title=Comptes rendus biologies;citation_journal_abbrev=C R Biol;citation_author=O. Jaillon;citation_author=J. M. Aury;citation_author=P. Wincker;citation_title=&quot;Changing by doubling&quot;, the impact of Whole Genome Duplications in the evolution of eukaryotes.;citation_pages=241-253;citation_volume=332;citation_year=2009;citation_issue=2-3;citation_pmid=19281955;citation_doi=10.1016/j.crvi.2008.07.007" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. Jorgenson;citation_author=H. Tang;citation_author=M. Gadde;citation_author=M. Province;citation_author=M. Leppert;citation_title=Ethnicity and human genetic linkage maps.;citation_pages=276-290;citation_volume=76;citation_year=2005;citation_issue=2;citation_pmid=15627237;citation_doi=10.1086/427926" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Evolutionary Applications;citation_author=W. A. Larson;citation_author=L. W. Seeb;citation_author=M. V. Everett;citation_author=R. K. Waples;citation_author=W. D. Templin;citation_title=Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon (Oncorhynchus tshawytscha).;citation_pages=355-369;citation_volume=7;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=T. Lenormand;citation_author=J. Dutheil;citation_title=Recombination difference between sexes: A role for haploid selection.;citation_pages=396-403;citation_volume=3;citation_year=2005;citation_doi=10.1371/journal.pbio.0030396" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_title=Statistical Genomics: Linkage, Mapping, and QTL Analysis;citation_author=B. H. Liu;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=K. P. Lubieniecki;citation_author=S. L. Jones;citation_author=E. A. Davidson;citation_author=J. Park;citation_author=B. F. Koop;citation_title=Comparative genomic analysis of Atlantic salmon, Salmo salar, from Europe and North America.;citation_pages=105-105;citation_volume=11;citation_year=2010;citation_pmid=21092310" />
<meta name="citation_reference" content="citation_title=The Origins of Genome Architecture;citation_author=M. Lynch;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=G. R. A. Margarido;citation_author=A. P. Souza;citation_author=A. A. F. Garcia;citation_title=OneMap: software for genetic mapping in outcrossing species.;citation_pages=78-79;citation_volume=144;citation_year=2007;citation_issue=3;citation_pmid=17663699;citation_doi=10.1111/j.2007.0018-0661.02000.x" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cell Dev. Biol.;citation_author=D. Mayfield-Jones;citation_author=J. D. Washburn;citation_author=T. Arias;citation_author=P. P. Edger;citation_author=J. C. Pires;citation_title=Watching the grin fade: Tracing the effects of polyploidy on different evolutionary time scales.;citation_pages=320-331;citation_volume=24;citation_year=2013;citation_pmid=23466286;citation_doi=10.1016/j.semcdb.2013.02.002" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=I. Mayrose;citation_author=S. H. Zhan;citation_author=C. J. Rothfels;citation_author=K. Magnuson-Ford;citation_author=M. S. Barker;citation_title=Recently formed polyploid plants diversify at lower rates.;citation_volume=333;citation_year=2011;citation_pmid=21852456;citation_doi=10.1126/science.1207205" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=E. K. McClelland;citation_author=K. A. Naish;citation_title=A genetic linkage map for coho salmon (Oncorhynchus kisutch).;citation_pages=169-179;citation_volume=39;citation_year=2008;citation_issue=2;citation_pmid=18318791;citation_doi=10.1111/j.1365-2052.2008.01699.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=E. K. McClelland;citation_author=K. Naish;citation_title=Quantitative trait locus analysis of hatch timing, weight, length and growth rate in coho salmon, Oncorhynchus kisutch.;citation_pages=562-573;citation_volume=105;citation_year=2010;citation_issue=6;citation_pmid=20234386;citation_doi=10.1038/hdy.2010.22" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. R. Miller;citation_author=J. P. Brunelli;citation_author=P. A. Wheeler;citation_author=S. X. Liu;citation_author=C. E. Rexroad;citation_title=A conserved haplotype controls parallel adaptation in geographically distant salmonid populations.;citation_pages=237-249;citation_volume=21;citation_year=2012;citation_issue=2;citation_pmid=21988725;citation_doi=10.1111/j.1365-294X.2011.05305.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Moen;citation_author=B. Hayes;citation_author=M. Baranski;citation_author=P. Berg;citation_author=S. Kjoglum;citation_title=A linkage map of the Atlantic salmon (Salmo salar) based on EST-derived SNP markers.;citation_pages=223-223;citation_volume=9;citation_year=2008;citation_pmid=18482444;citation_doi=10.1186/1471-2164-9-223" />
<meta name="citation_reference" content="citation_journal_title=Fish Fish.;citation_author=K. A. Naish;citation_author=J. J. Hard;citation_title=Bridging the gap between the phenotype and the genotype: linking genetic variation, selection, and adaptation in fishes.;citation_pages=396-422;citation_volume=9;citation_year=2008;citation_doi=10.1111/j.1467-2979.2008.00302.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. J. Near;citation_author=R. I. Eytan;citation_author=A. Dornburg;citation_author=K. L. Kuhn;citation_author=J. A. Moore;citation_title=Resolution of ray-finned fish phylogeny and timing of diversification.;citation_volume=109;citation_year=2012;citation_pmid=22869754;citation_doi=10.1073/pnas.1206625109" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=K. M. Nichols;citation_author=W. P. Young;citation_author=R. G. Danzmann;citation_author=B. D. Robison;citation_author=C. Rexroad;citation_title=A consolidated linkage map for rainbow trout (Oncorhynchus mykiss).;citation_pages=102-115;citation_volume=34;citation_year=2003;citation_issue=2;citation_pmid=12648093;citation_doi=10.1046/j.1365-2052.2003.00957.x" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication;citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=S. Ohno;citation_title=Gene duplication and the uniqueness of vertebrate genomes circa 1970-1999.;citation_pages=517-522;citation_volume=10;citation_year=1999;citation_issue=5;citation_pmid=10597635;citation_doi=10.1006/scdb.1999.0332" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Ostberg;citation_author=L. Hauser;citation_author=V. Pritchard;citation_author=J. Garza;citation_author=K. Naish;citation_title=Chromosome rearrangements, recombination suppression, and limited segregation distortion in hybrids between Yellowstone cutthroat trout (Oncorhynchus clarkii bouvieri) and rainbow trout (O. mykiss).;citation_pages=570;citation_volume=14;citation_year=2013;citation_pmid=23968234;citation_doi=10.1186/1471-2164-14-570" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. P. Otto;citation_author=J. Whitton;citation_title=POLYPLOID INCIDENCE AND EVOLUTION;citation_pages=401-437;citation_volume=34;citation_year=2000;citation_issue=1;citation_pmid=11092833;citation_doi=10.1146/annurev.genet.34.1.401" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. B. Phillips;citation_author=P. Ráb;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2001;citation_issue=1;citation_pmid=11325050;citation_doi=10.1017/S1464793100005613" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=R. B. Phillips;citation_author=N. R. Konkol;citation_author=K. M. Reed;citation_author=J. D. Stein;citation_title=Chromosome painting supports lack of homology among sex chromosomes in Oncorhynchus, Salmo, and Salvelinus (Salmonidae).;citation_pages=119-123;citation_volume=111;citation_year=2001;citation_issue=1-3;citation_pmid=11841160;citation_doi=10.1023/A:1013743431738" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=R. B. Phillips;citation_author=M. R. Morasch;citation_author=L. K. Park;citation_author=K. A. Naish;citation_author=R. H. Devlin;citation_title=Identification of the sex chromosome pair in coho salmon (Oncorhynchus kisutch): lack of conservation of the sex linkage group with chinook salmon (Oncorhynchus tshawytscha).;citation_pages=166-170;citation_volume=111;citation_year=2005;citation_issue=2;citation_pmid=16103659;citation_doi=10.1159/000086387" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. B. Phillips;citation_author=K. M. Nichols;citation_author=J. J. DeKoning;citation_author=M. R. Morasch;citation_author=K. A. Keadey;citation_title=Assignment of rainbow trout linkage groups to specific chromosomes.;citation_volume=174;citation_year=2006;citation_pmid=16951085;citation_doi=10.1534/genetics.105.055269" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. B. Phillips;citation_author=K. A. Keatley;citation_author=M. R. Morasch;citation_author=A. B. Ventura;citation_author=K. P. Lubieniecki;citation_title=Assignment of Atlantic salmon (Salmo salar) linkage groups to specific chromosomes: conservation of large syntenic blocks corresponding to whole chromosome arms in rainbow trout (Oncorhynchus mykiss).;citation_pages=46-46;citation_volume=10;citation_year=2009;citation_pmid=19689812;citation_doi=10.1186/1471-2156-10-46" />
<meta name="citation_reference" content="citation_author=R. B. Phillips;citation_author=L. K. Park;citation_author=K. A. Naish;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Journal of experimental zoology. Part B. Molecular and developmental evolution;citation_journal_abbrev=J Exp Zoolog B Mol Dev Evol;citation_author=J. H. Postlethwait;citation_title=The zebrafish genome in context: ohnologs gone missing.;citation_pages=563-577;citation_volume=308B;citation_year=2007;citation_issue=5;citation_pmid=17068775" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. E. Rexroad;citation_author=Y. Palti;citation_author=S. A. Gahr;citation_author=R. L. Vallejo;citation_title=A second generation genetic map for rainbow trout (Oncorhynchus mykiss).;citation_pages=74-74;citation_volume=9;citation_year=2008;citation_pmid=19019240;citation_doi=10.1186/1471-2156-9-74" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Sakamoto;citation_author=R. G. Danzmann;citation_author=K. Gharbi;citation_author=P. Howard;citation_author=A. Ozaki;citation_title=A Microsatellite Linkage Map of Rainbow Trout (Oncorhynchus mykiss) Characterized by Large Sex-Specific Differences in Recombination Rates;citation_pages=1331-1345;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880492" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=F. Santini;citation_author=L. J. Harmon;citation_author=G. Carnevale;citation_author=M. E. Alfaro;citation_title=Did genome duplication drive the origin of teleosts? A comparative study of diversification in ray-finned fishes.;citation_pages=194-194;citation_volume=9;citation_year=2009;citation_pmid=19664233;citation_doi=10.1186/1471-2148-9-194" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=I. Schubert;citation_author=M. A. Lysak;citation_title=Interpretation of karyotype evolution should consider chromosome structural constraints.;citation_pages=207-216;citation_volume=27;citation_year=2011;citation_issue=6;citation_pmid=21592609;citation_doi=10.1016/j.tig.2011.03.004" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. S. Soltis;citation_author=D. E. Soltis;citation_title=The role of genetic and genomic attributes in the success of polyploids;citation_pages=7051-7057;citation_volume=97;citation_year=2000;citation_issue=13;citation_pmid=10860970;citation_doi=10.1073/pnas.97.13.7051" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=J. F. Storz;citation_author=J. C. Opazo;citation_author=F. G. Hoffmann;citation_title=Gene duplication, genome duplication, and the functional diversification of vertebrate globins.;citation_pages=469-478;citation_volume=66;citation_year=2013;citation_pmid=22846683;citation_doi=10.1016/j.ympev.2012.07.013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. H. Thorgaard;citation_author=F. W. Allendorf;citation_author=K. L. Knudsen;citation_title=GENE-CENTROMERE MAPPING IN RAINBOW TROUT: HIGH INTERFERENCE OVER LONG MAP DISTANCES;citation_pages=771-783;citation_volume=103;citation_year=1983;citation_issue=4;citation_pmid=17246124" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=E. R. Timusk;citation_author=M. M. Ferguson;citation_author=H. K. Moghadam;citation_author=J. D. Norman;citation_author=C. C. Wilson;citation_title=Genome evolution in the fish family salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout.;citation_pages=68-68;citation_volume=12;citation_year=2011;citation_pmid=21798024" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Y. Van de Peer;citation_author=S. Maere;citation_author=A. Meyer;citation_title=The evolutionary significance of ancient genome duplications.;citation_pages=725-732;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19652647;citation_doi=10.1038/nrg2600" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. H. Wolfe;citation_title=Yesterday's polyploids and the mystery of diploidization.;citation_pages=333-341;citation_volume=2;citation_year=2001;citation_issue=5;citation_pmid=11331899;citation_doi=10.1038/35072009" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. A. Woram;citation_author=K. Gharbi;citation_author=T. Sakamoto;citation_author=B. Hoyheim;citation_author=L. E. Holm;citation_title=Comparative Genome Analysis of the Primary Sex-Determining Locus in Salmonid Fishes;citation_pages=272-280;citation_volume=13;citation_year=2003;citation_issue=2;citation_pmid=12566405;citation_doi=10.1101/gr.578503" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=J. E. Wright;citation_author=B. May;citation_author=M. Stoneking;citation_author=G. M. Lee;citation_title=Pseudolinkage of the duplicate loci for supernatant aspartate aminotransferase in brook trout, Salvelinus fontinalis;citation_pages=223-228;citation_volume=71;citation_year=1980;citation_issue=4;citation_pmid=6774006" />
<meta name="citation_reference" content="citation_journal_title=Isozymes;citation_journal_abbrev=Isozymes Curr Top Biol Med Res;citation_author=J. E. Wright;citation_author=K. Johnson;citation_author=A. Hollister;citation_author=B. May;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes.;citation_pages=239-260;citation_volume=10;citation_year=1983;citation_pmid=6354984" />
<meta name="citation_reference" content="citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinformatics;citation_author=Y. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Accurate construction of consensus genetic maps via integer linear programming.;citation_pages=381-394;citation_volume=8;citation_year=2011;citation_doi=10.1109/TCBB.2010.35" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=A. Yano;citation_author=R. Guyomard;citation_author=B. Nicol;citation_author=E. Jouanno;citation_author=E. Quillet;citation_title=Semaphorin and Eph receptor signaling guide a series of cell movements for ventral enclosure in C. elegans.;citation_pages=1-11;citation_volume=22;citation_year=2012;citation_issue=1;citation_pmid=22197242;citation_doi=10.1016/j.cub.2011.12.009" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=J. Zhang;citation_title=Evolution by gene duplication: an update.;citation_pages=292-298;citation_volume=18;citation_year=2003;citation_doi=10.1016/S0169-5347(03)00033-8" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="Whole genome duplication has been implicated in evolutionary innovation and rapid diversification. In salmonid fishes, however, whole genome duplication significantly pre-dates major transitions across the family, and re-diploidization has been a gradual process between genomes that have remained essentially collinear. Nevertheless, pairs of duplicated chromosome arms have diverged at different rates from each other, suggesting that the retention of duplicated regions through occasional pairing between homeologous chromosomes may have played an evolutionary role across species pairs. Extensive chromosomal arm rearrangements have been a key mechanism involved in re-dipliodization of the salmonid genome; therefore, we investigated their influence on degree of differentiation between homeologs across salmon species. We derived a linkage map for coho salmon and performed comparative mapping across syntenic arms within the genus Oncorhynchus , and with the genus Salmo , to determine the phylogenetic relationship between chromosome arrangements and the retention of undifferentiated duplicated regions. A 6596.7 cM female coho salmon map, comprising 30 linkage groups with 7415 and 1266 nonduplicated and duplicated loci, respectively, revealed uneven distribution of duplicated loci along and between chromosome arms. These duplicated regions were conserved across syntenic arms across Oncorhynchus species and were identified in metacentric chromosomes likely formed ancestrally to the divergence of Oncorhynchus from Salmo . These findings support previous studies in which observed pairings involved at least one metacentric chromosome. Re-diploidization in salmon may have been prevented or retarded by the formation of metacentric chromosomes after the whole genome duplication event and may explain lineage-specific innovations in salmon species if functional genes are found in these regions." />
<meta name="og-title" property="og:title" content="Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1717" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Whole genome duplication has been implicated in evolutionary innovation and rapid diversification. In salmonid fishes, however, whole genome duplication significantly pre-dates major transitions across the family, and re-diploidization has been a gradual process between genomes that have remained essentially collinear. Nevertheless, pairs of duplicated chromosome arms have diverged at different rates from each other, suggesting that the retention of duplicated regions through occasional pairing between homeologous chromosomes may have played an evolutionary role across species pairs. Extensive chromosomal arm rearrangements have been a key mechanism involved in re-dipliodization of the salmonid genome; therefore, we investigated their influence on degree of differentiation between homeologs across salmon species. We derived a linkage map for coho salmon and performed comparative mapping across syntenic arms within the genus Oncorhynchus , and with the genus Salmo , to determine the phylogenetic relationship between chromosome arrangements and the retention of undifferentiated duplicated regions. A 6596.7 cM female coho salmon map, comprising 30 linkage groups with 7415 and 1266 nonduplicated and duplicated loci, respectively, revealed uneven distribution of duplicated loci along and between chromosome arms. These duplicated regions were conserved across syntenic arms across Oncorhynchus species and were identified in metacentric chromosomes likely formed ancestrally to the divergence of Oncorhynchus from Salmo . These findings support previous studies in which observed pairings involved at least one metacentric chromosome. Re-diploidization in salmon may have been prevented or retarded by the formation of metacentric chromosomes after the whole genome duplication event and may explain lineage-specific innovations in salmon species if functional genes are found in these regions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_a20DcTKCr5xYofUAwJcUJXls42il5tWRwNWvTgv1wo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459661","apath":"\/ggg\/4\/9\/1717.atom","pisa":"ggg;4\/9\/1717","ac":{"\/ggg\/4\/9\/1717.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1717.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1717"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1717"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1717.long":true,"\/content\/4\/9\/1717":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WZK9nhHNDivkABsoQcZfX0fzWB5G1iDg9oRPwN2hYVboh01SrycFfGjvuxka6+e4doIiZWWHMsMHXnDKrWRjak9dgDFQTm6zZO+iwfbGqDxvIzKcuUZfa\\\\\\\/x9FCqaABOXuOUUrE4O5hxlMqK0mmYoykBKcaTHx+DO+tcsC4FfpqLf\\\\\\\/1wBk5NbG0dqF4KFWTJU4rmDdhKAzje7l5YkeDLC0yS0E3dVrq\\\\\\\/QTv9XasMbHIvAOJ61+ewjRHhg7Byb55bVcvyI6TRXhN9Fa35BgijBpPy8lCMP9aFftYLdrT5rHoX4w01ZbdvBDANVsVVO0rAky3unA9sc\\\\\\\/TTxN+P3w7YHf\\\\\\\/YGKwOJwg5tHWkJMGpm408z9K\\\\\\\/IAzO9HGabKSqgzpCZzt0rLqLRQvWL9bPOo9e75uTAmk\\\\\\\/qm09Ms24ORuGTjWkt\\\\\\\/DNQ\\\\\\\/fS9vJBJD\\\\\\\/XQpLiMOEOjRQjHspSLYvGX\\\\\\\/e5QjUNtxx95tfvqCqyzZFohNLS+z5lJymH74ipYhWe7gj1gXbBPlBSBaniVv1\\\\\\\/NEETQNRXFZvyLq1dT3Vh1slaqi4f6M\\\\\\\/Rypktqgum2tx\\\\\\\/uFAlME1IhRjTaHd7Se\\\\\\\/lhoykrmiq49xicbFrVTexFZqRgZL2iQHA98WPXNcyiZZz6KQXnvLoI+eLrnhFWLtmsRjCncpVYvrn7Pcb16A+4x9f1Z70SoLfITJy+YSGmVMvthw0I2AZZStwSMgVAvgRoBr9ofW83p3VpyJvEPLgqrz4wiguKfaxQnCa8Dr84ndNa9diE1gqhp31mk++FsTyjPIgopGw4cWogkQzOe+dHCTzrEF5ui+TaENxRH1Gqvmea31Hi60C4yGUMm10ZiKWqp\\\\\\\/VV5O4t5yVg5uBAkN\\\\\\\/WJT5biqxe\\\\\\\/XtRwe1HS5aBeaBJM6uKwq7l0JjaCYIjAn2PdiQS6vzFzk14MxKfalZEdXSWRxpumAWIlojC9tF6\\\\\\\/fyOBwKAPW0v5CJNIXYeu1vKNWru9Pxx1fKpCrbp58p7XlKJbcOz6bPKujwmGQrI4n2iLiNyLaFW\\\\\\\/oACNK1paxkjD9hST7mZU2bE+\\\\\\\/d25Dkm+d9AHqkphONn6WlM6sFLZxMqe7big0SKCm9WiB5Ic8OlpXWugEKroRoN+pVmtfX5GSqzK5mEE9cUS4amo3xAXn3xmSr2YCOaVpk5+03tw2O5bKsVz4hTjxQbTP5vRLmrxB2\\\\\\\/scheyl1biry6+zVkVFwHmGMI3XytqufpImsjwq+vZQSwAGkb\\\\\\\/dU3heWjXWNTVw54jY1i78thQbssQLj9vVtRq\\\\\\\/fl\\\u0022,\\\u0022iv\\\u0022:\\\u002202WBeJ4zRM+ycX2YDPqvMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022675f6b20c336f30bd8c1457ce2a88aea71f9cfacd8a41a0855c5072feb331262\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DuDe0NYO4GcViNzH7rLHk+OIkpR1x3YZ8LCiX5auOBnD43fzbHZKEsSrbI287RlPq7yyZTec4b4kCNTRBDI4Nz2e+Mf7rN4Qe\\\\\\\/DJUltciCg0k0uH0ZOCFASnhnoYbM1xmYBBcAgxIgpweK\\\\\\\/MMKXLSj3m988RryucwFeuBBOQmJ8MNvW0DqMZMoJNKJJl8CKjxdiLDNuAAkSX7+PbSHYL8QFm6X4w9Z\\\\\\\/ORRY8ClyJyXP5FEHGPKs5nPC4vhFSmC1hEX\\\\\\\/593Ijx+L4SJAP1msh1TzxNq5TNJp2twuG5i616\\\\\\\/Fufic5q+Or4o+MVvyNzdz6InVw2ENDCywovRt0IMel7NTTNFQFkefjr4McOb91c5d9JwsLwRp5kQjBPoHJ0I6MMeXuvAO8fy7cSE0c\\\\\\\/mP7RlbUZ6UukCPhF8GN0jrHHLKm\\\\\\\/oi3oGwMfESWwa7eVHQU6dvQfa7w\\\\\\\/EZ7CJ4nwQqwvcED5PIFkzSzKHIRs6Vq6OfHHAT0PPKMmQnsai59hmGlgU+my\\\\\\\/5BfzG8kuhQBH24BCoPrqukIEmQT0tvrfYgNYfTBJVdDLDIH2TEyf4by0O0RtYsJVg6OPB9FV6ytKilSAI6nCpoGSsHI0p8n4MrQ5OXGtyGw5Cx\\\\\\\/tEQO0\\\\\\\/dhQDAiuy1txELVRL86SPKpzmQ2pT0KHu3sXzf9fK9YQmMYqwLqnptuYr\\\\\\\/CY1NlbU5JsNC+lwALCZ\\\\\\\/XuBHTSRrRdRphEidx8EXgiPMjM9o8Za4Hoiufq090h1sNeF6qmyrQWEoHFw6Li+eQDR63\\\\\\\/6X5KPgPP9itE7pBUr8OGtcB+OvztvY87awiq6XpUoOM2RoC+YNrIUtA0IWwbejiZQ6nrl5jcGhSTl09pjD+QKQ87q1hS6lFAUo28H3K94s97wlfqb9SSy\\\\\\\/EddUCsCFKI+ojIFxhM6VNc74zauQqkEsW8PEQFJfBZXbIgNjjnl1jZvFK\\\\\\\/hCn0royvJMdqrxi\\\\\\\/xqza4FPGupBW\\\\\\\/HxEgBLQqR\\\\\\\/194UrHf2b\\\\\\\/75YKm04PW23iAs+EjlyuV8YlYpOq5VXeqM9bQaqCGA0qf8Vv3aULjSwC5ISJRmrG4ppzYRhyD6k79h0S04vDv0Yrh+BG7tD7w2eWuK++jt7v3hesGyzvgQ9vJcA3N30N9NQQqDoDB8A2ibvpd+U4jJFkpjYBe65Pw1pyPTE6gjZ9OiTtyh2NXTOxBDOUao7r4EaqlDs6Ax0XIA3Pvb4iZU6syWaTCZR+WE7HSsMuGAAWf8XLHwr2v7XZ5rWgj0kSOSpNWnFQFRKkjoZNcFq7YoEEM+sX\\\\\\\/JocirWU\\\\\\\/XQGHCluV8mNKdAIYWi6PNJcImmVM7wvfAVLoubO6o7XpURsWQLhERxbaeO6D5Z18Rr92gQzTJ1aUiVMqg3qO0kwRdQaOvXdCgwBiafqALjz9SvlrW51CGK\\\\\\\/EF\\\\\\\/yZ0OivCd48t+5wxIsDpGtOWmBatLZVPXYi3KqXhQ6YoY\\\\\\\/amup98bO0cUhbKQj+cnUPpdc3Q\\\\\\\/HKH5B3D7G4OuSRMZ1mk04ZlW8K2KU9Mnf6a95jrfTKLWdPBDI7Y1dOi1qBDU6SMLJxqiepq\\\\\\\/djjaAMOoutORVpTWQZt6M+0AfVNPQA1BtyATUDHovh8DPjYl6OZFz+40nlopS59b16HIuMrG\\\\\\\/+MfNuYN40pM1GBwY4MpbsffIIj0ohCn7Fc7FK1trDebzfZbAX6tJletk03NaHQ4qOouPgKlYyHsbCbO25hKz0sEhwuxzOTPgGTBqggX47pn5IS2RscpRMMVkwLOENrTzPmTJZruukCBzdKXskXjuFHKXhwz70Y0b9gqGeoFgdiZodOn19Krt4e7VVGLhsEpZESxYPKORuVTPWPFrEtLM57U6awo3+7LZbv2xdSEQKTdF0BWreBpsck7o4zKi0dyha4cTpLtacJ3IMw3lA082v3wSTQ3fUTXsPkRUR3ugSwcBoA9QanE9YUBXzIgrgb+HJ4UjFbxsFjXHAwsqMXumLBm9I3py0wJoIsFzM\\\\\\\/U2HNcaWhF3JsyUeAVBYubJEa6CGj+8nBZQyfh3lErXGYitncUoIFKyks3vGU9Fv8UC6ycXxK8q9HyhkxJHKXQLNZHWj1FbJj40SmE+\\\\\\\/JcAnQUlC014mV5bvSLQmAbex64nqxbuXB3QXjKRUQXBbPVX+QmFHVWedr7FnFsAtJJraoTfQDMiqsKoNVfHoLUGr02tg9FecqeYRs+b3b6sC6rcQdHx83zTUb5Hs\\\\\\\/U9u9yIihecntO0iytyh8ThxZwo6QsyJnBW+TsPbrfNeOYV4tfKfHDW2eD9TsOzwcDRhiBwWRcxai4kb9HQktHXiwmkHfdfm5\\\\\\\/fFzcAl4HXtBY1ZeCy+BJRiCiKXGKPxrP66AaV9jnpygM8D6EjCLlVKnpT86vomBJTc16LvSy0H3SAITVOhI35dK2udGiTlhNPYYah5FgfqPQvKHuYIoHAEKcj6RTqd67ErRGt5bn0oLIsOg2eJBlb1GI2WFzGaeGfrwfQYivE75279g4h+\\\\\\\/qq1ajf24xqGGNHwrdOYeRMt8F1eU+C5gcBTfDC1Nb+30S9BxHA2vYM92MlkaahoZfsJPpfPoEAIL+4WFUzlfKCvzra5Qtv+fkVdQwjauHztO7cGLAj12vuYzsKT+m+WkSUIBGRrAwMwB3rSrOGszrLpCmmyJbZm+iNeof85P7FPQ+WnPsvjpuHlifK456x1VYjkIpivFPNQwvcZxZEzus4NzMwZwIDnPd5N9nUK0MtQkYrLUPhJW4aL7yUfwFIaPM8Z9e7lVkXyGiDfF+xVwL8jGQoCfJxx1TOCgUDCmvPYVr8TFjYp8XZjLN1MSt1\\\\\\\/smRwjFMCfVJwLi1Cgi\\\\\\\/u8yr+j5RgDS0ywUj7JPKZqTt3j4aV+kHhBsXxfJgQ0hRJJws3zkt4LldOXhnZRL0ssBhmfOdAa\\\\\\\/g+vCQAQL0IBjmeum9CmCNzf+ZOvzKnPS8rMbGGtdV5h5vcd6BaFBbn+K68iLwAlpsNqqa+Px8LpZDbnk+AYX26prdRqEye+Cmxg+yop0giphs6B1yKxlYhDLxFTaJl7XCI0CBTsXBVw08IeRHk0CqNCZI5PUlA9X8j3Ck5uFgCOZ\\\\\\\/8eMm2OVXDJ\\\\\\\/E3KrbNa140esmchiBCNm\\\\\\\/ri1e08FyTSP8Y93jnfKZksjDy1QxyQN5ZlvEC\\\\\\\/djOD7c6CP6YLlYkUb3e2vGqAPj+3fAlLpG1Z2ItXBtMQ9dFbFc+tGNPupfCeiqmmfkl8mfv4j2l8vQYUSjP7koAHRClimbp3PdoTJnRDSiFi0nhJ7dvlSgrS93SApoGeGGygqMzHn6V8Jaz\\\\\\\/y6nulRgM+D94VHgUa\\\\\\\/ROt5cPtcRgpV4zFHEJTvCBeffAJV61+oB1Kgvp8Eu6fMGf4\\\\\\\/hvuNerSu4\\\\\\\/GIizLIjwlLpu2omsdYFVx6IY6G\\\\\\\/Ma3oie2dYNd5RzVcDJIKQuZmVNizzH\\\\\\\/CR6z+tiltGt6wCTtjDT4pKukSJIyC65nTaJEj18ZOoh\\\\\\\/xqPAj9qRLhxRc6M9aB2PAN5BDiEVSuGXQFixTMuZkmmXwlzM7pa++7ew6pUCL82zRpxbI4n6sL\\\\\\\/Yu7CHhviaslLZa8gbw7TnPUlWAmCYP2eoM6OwYSdiqGjiDD9Qn5nYFUBa2igNzq0amL90f0IG6gDJaBQj\\\\\\\/0DTkA0uHt5ukAdW2b+NCdhvZUe+VZldFZBIDOOVyYTwFQ0fbh6XJYQx+R1NTSV0PMil8P9CWd3rmRvzUBtJNeuWdY8OSkf0IAtVHqMol8R3FpisK4N6\\\\\\\/MNwOa5Gijl6Ou4RnM3WyGL7RRMv\\\\\\\/lbep7QWfMS91C8TJGiENHNZJ7v4u5aVLkIPeN1iFs27VIQRfFTt1DrZndUg2Zya10VrEoOJQledCh0jpF01nrN\\\\\\\/UZ8SnVjMpEsaSXAz2RgaDuKxguM8izd2475qkwsTcQGbT7k3FzJiUnAFX9lvpDvjBXb2CQj+3gkz7g6juTl\\\\\\\/3eH7GPNsGM4ZgpCrm8KKugLPpd9w1XsgZBdBPz6bt3H1FTjfh2I0PaLckBzY3dGPjRv3OdkK+h0U5AC7VUfobZ0slPzUGVpf5OqQeiIjNoZM6O0hMKo1FRAz5\\\\\\\/dG1G3Wl8zu+OiljkelXFHbFfLAliKTDBYawHxmKzsZCx\\\\\\\/mZNjgN2\\\\\\\/7C6O6+Rr+VVDiwpyIuvCtSyrZXd4V\\\\\\\/Kv2SePX7YbYMIcd3PyHUYgkJMUrhDCHvg5FDvDWlNoNrA6rvSbpbgl3nmCyTOnxr1rqfAprDnmIKDH8vM7jlYa\\\\\\\/VDsjl8qwhAz76wWhTeUg0aZUjzxlYAN9EPViQ12lgo7hLNEOhaIM99D7maiSncyIyqUIq5XBNJ9BLMTlCyewLurvIJGuXMAeMYxezJbry4eZ5mSPWZiuJLFR+CS4x5PvV5wc9Dgb6BqMQ1Xy7\\\\\\\/b+c2dS3MQitrWQAJQnF80y6fADF0FrZFN+yMbPq5VpdV5UxZav94iRlwDWDcnyBLeWe2bvQ1uYpMnXhOHzJXDgdfaJOWNyskt8+HH+KR0nf3o9B3oN2AyWCO7euev1PqglMrvJukpBZTUbnkll9rWiK\\\\\\\/q2wGfQ3pq2u6\\\\\\\/RzZHarSSe5MuFbwgdTKe2Yy4A6SoElwcxFTMvJ027XBX2Vf\\\\\\\/C30tmAHD4wStB5K2O4mCnvbxLqiVzzcuOiPvo\\\\\\\/ZmwXPdGG3PH6luZNsewYQqk4uUdxB2qXZ4Ff0OzbPRZoUPNwuRALXjMHyqSyn7iBsxnR2YloJUyeJoYf9JFlSrx0NOWy3SYL8FaCW+sJ53XOppDBwm2+zsTU5EFg5pCZ6cshLscSLmv1k3qJd7dkxnBosQ5NEVEZFCthD3lu9ZF1R+fNKJfaMhQW4ZXgEBxKK51y0w3KSN49LI6FqhxGAoqngNsZ+Y6o2QHCUiLTtmumt6DUq1NLPzeM6yDWdehID5ljCj9QmQZ6\\\\\\\/X3N872wL+r+uDLp18KmuUjnsHJxwIVH8DFETwdocwpNk\\\\\\\/\\\\\\\/z\\\\\\\/mnFgYOCdUcl\\\\\\\/WF8JPjOUiCdjOoRZAE7zTYTHElhDBHxTJ47HY2mM1Lmf2X+v1GxVgYnXXAgJeq+xQgvRM703L49CcKpg5MOZDjks7Y0PnxRnaMHRoyv5dX9OjujRkMHl89UnxqWjv\\\\\\\/QkGbsf4xyUh3zUt+RgVYVXbEYyRaZjp74S0XqxmoE6IAJtFc7r7orUKND823Yx4JibgigmjBoecapJNlDL6REYnCoMobnYaGQnIIj7b+OgMsugZ023WOiv5aSw1RDPdKq8+JbV5f9YT79T31gc2Mgwv45eWDGEptS6XxTxiWaQJGIX55H1TUrYavbi+8XTfAKCJWjbbon7ZjLpg0FE040qZtHtGhwe9vZ899HNV2CMKxh+xsahsxHlTqQ\\\\\\\/GeJvCLq2kirmsGiVk6me1AjRx2dqUKYgyawqEP5C1O65+b2xxwUm0cVW4L35lMXM5t5xtHXanORJ5Xn04y8Jk60iWFbEDP7mZ8TJBTfprgjybYXUnMvUT2yM03MqiYtGkp5vw++3wqglQAxxDSW\\\\\\\/mwbT9knu\\\\\\\/WayKYDXqY8E0dMbuGU6mfjgRNjM3ttgp8i8NDlSEJjzFjJS6cT3tLB8kNdH9XuVxSECh\\\\\\\/w7CWCAMDwG9krnk7aCsg9tVtTaCqUsfQAwUJGSJI8glh\\\\\\\/wADgzwMjuBmNbXFqb6MjIqVnrZloLFl\\\\\\\/Hggbns9OrgWyGgYmfEn5plbQFhyzuIOcTP24tCGske5AjhYcIQ+c3yyf8a9t1Q6oH0uIbJWwzVLyCkRxfLzvQCb1ADFwVW6HK86weUz9JcUU0P\\\\\\\/9a3\\\\\\\/88NkbhF1Flb+9mAf1sh4wh4YmtmGkpODPiQO9BtPMQrMm6mr42BksPZKp\\\\\\\/7nCOjpXxzEVKt2TfpTuHwOvnUGOh2XPMNjU6eS1rXd5aVRes\\\\\\\/s14C1ATR7fxbjj0KiQSjrF2jfyP8lTBTCK5lP0G2viQRo76qjXBqd\\\\\\\/7xnDV63cRhN2aOgovNcqtU8qc2Xklte+M8o98APsK8znCQ38mFStb0f+Lg\\\\\\\/OZqiQIPNH5gHHvg3XmskMzVrb\\\\\\\/IIJUIppEia5nLyW+8GmB0uxbIzt3rqE1oKYf+sdNZ3uRgZ8Z51MjiVSPm64VjwJz6p7ZjtI\\\\\\\/WmH59mB0PYBkP4oFXt+tx+\\\\\\\/KpHs\\\\\\\/ywX9uRXmJx7vAN3A50sKdv2abqUnYmHROUzgn9ZJYru\\\\\\\/Tccd9MexrFt+RgQRDe1biliokiA4uS7V2PSaZSl6BKcFkd6sRuHYFRmOa93\\\\\\\/DM14wE74l1qil8wT98lnaZwOSJ+9chBk3KI85ki1izvVoyghxvK8GOIEDYHURzzU8vMELnuytZmW2NgQUCpR+Wb4aN1sCstwAU8JZ13KNE4g9GHHETZkNmKalQk+uM0Km7AGBvwXDwrUbFZqjGdjjpLGN9s+ChPW5D7SvVgbc6pKKnmyMbIaqJh9895caWDNm5GcaS2Bczz+G7mGQZdakq199OBkrUuWj678ZKgQPqYUhksSBC+3lSzTACUg9Vsdw6zIE8gGNuxlNXhKxNt+m15kc78X6S9tvjlCeriK9tfBS+EtMg4Iuzh5i5KRPCofNsPt2ba+7hYDNVXR9ym34uW04su3y6qQWfM3jHGQ7w2DIj9mzSCqqvCpo+ZLUChG0Sa9eJEDQDyngUFqvjQIrmVND6rz8XeCXH0N1rK8mou\\\\\\\/q8krcUruCrfOENzignZfU6VMu+jvoaxNdVs7aGRKUXmATtvmEZqgYb\\\\\\\/LkNh9jSQAcgmJDCD8MKpNrA2gE1id0ayp8z20Ap4HTZUWj\\\\\\\/pKTkJYrX1mSdFBXNEAhmzRFzyIwtuED4OtaShhc5sgSn6Fq6Rt2F37dU\\\\\\\/PAZlRe6Bn86WTAyhBhQ1THr8uorcM2K6Jmd1GrdTZJU1BIaXd7tvwj28xfZEQI2veGBQfGawufuwSftlSmuT\\\\\\\/bh2mInT6uBk3SMZLVbUXgxbe\\\\\\\/1YVjT\\\\\\\/Z3jYP9QWsmrQSi85gp3py6wQPqtCFqPr0yw6RDHCkIVopPQMIxWASrp36ItlQsXYXhUtY5lWC+w+itJXuzuO7wMFWqw66BoJyNbNxB\\\\\\\/\\\\\\\/sB6RppkLxfJ4\\\\\\\/gU8Gvd9m0j2x6ncS6LiNPKNu7LsPjVtcYOcKLt5iNsgvV5JUZbdccSiVBppa0zIAzSvQi5Lu5fUCpXd79Gdgq9zTPugdCgIBh5JhOkMySghIiDQj1mBwE05qX0CUu9uJ6LomgxFMbXskh28L7LE0rbYVFNlwgYik5yvaD16MhBi2wNP+i6OzSfKpjUzzDm30MLkUPHVKcda0NzDWj7lVBmtpclVBhLqO8w6cN6uR5tRBxm2JtQitxFPLERDZ4oH0Dupyvzae3aMR4HgKrY4PBRmjyJ9c5eL1AJzYkDMf+FO+Q8D2jJywN1\\\\\\\/v81nnBZjf\\\\\\\/\\\\\\\/4nR1upg9h7w7Uo6wHThblNvw11C6+eOik3nEGc6xHqEyIX0LgsqMmn6zci+JhJVaTy7YRDlYgi0BZxCKbumrClJWU3aEuvQ0twfj1BfRvdGC+sXSYI882\\\\\\\/0jXFNb+7lDVsmv\\\\\\\/GSOEd90HSLJm0RjF42HG4hQlsh6sI+gM59D6VIpcx3Bt9x6F5422t8BEgvcpWayq3GJcnsU701nNQPVGLf3ar9NCVvVEVzLPElxEwHfVucG0MtXLCDN6oEkWMo1wbZOVsaHd4LEtS2JZV0nnpwHU8brlrSaSSCCsEVg3AKHrd24NXPdHiPVkLuqJrxKyNOwtTKWk+1BfqNVbzKJakklWl9LOkK7r57qrGtDgW5dVLZnofeVNSYL1s9pmO7CkMqlQqv6SWwjz85nhX+lf3lkjMoGTnvHodEzpVkSVPKHUUDWSLVr34qdNDaRTfgO8CeQ4Ki5KWQS+C+tW6exYFAH9IYkh8SuUd789qsZ5NoB550GCWZOQzbk5mRKbdSY7A0ePt0yX6nIOKpomRs0E372CSVVjgKYjhlHjE\\\\\\\/iJ0ZPlk6RxwO8onhb4+iwtK4G07gNzRd8jf1U3I1i8915I5iPDYQNn3dbCMeh6MeaFgkQKmDJ6OyAMt+nxSuvHjJNNMFoHMWvYNtm9p7RXbOSP4jnZ15tJfyW4gA+xpQgxUiGlZBTXvmVp9+tf2YtVTRr2QTuU6adyhHI3HFh8Ix\\\\\\\/zWpPkSjXhr1Q+WZ9ef\\\\\\\/WY3gQlis9LLMLKElA9cx6lyNQYeFZL2aIs+eOhx34RTYrAIWUKe9XvIx6EkGkiuWRl5BKPt9pb94OjhrJhCWKwZitncUYp+Y9aaA\\\\\\\/yEjYaX6RKrB9ht0zGz32yzvjefak0n95+DZMmubAnjiMDcklAFNUwr8beIZePnvMgsBSY1L21mavey0WRuednSAyP2sMsNJWB8v22MOhHSLzKK3cmsbL8AYOKbvEnZ4XdlpVLZ4eTD5MXB\\\\\\\/+xOEljgcAeHXlHVby+6PnGQl2tBOW1nT0LLiSzk3z\\\\\\\/G7v4rDdFU\\\\\\\/kXWW4RATK3ejmTKyZoHJZ+cDrdbwvoEYzQ5QB\\\\\\\/5nhftvDOmJ5ZCwiJAn9epmOi0blRrXDZNae8eJlVyVjUqvJ6wt4bk0q2w006yg76BAo26m3+X7+\\\\\\\/4T23v05RWkDAWHp26F7ejdqP\\\\\\\/aTxVihHNGpVDhcJlgr+bcEHWL8uxM0u7UHHlFU403tSHLWUiDOgULLdIZJWQ4cX18et05qOazk3vwj3+SxQZicd0q2WJzj+659vNIIDLkSNsA2RN+Xa6Lhyke6YWCjfeg\\\\\\\/HwTBy9AVqU\\\\\\\/IQ7RtsQFNDdyWtXVKqb0lGfXDlMD454kil0J4Hl9fMCM7WxUCFxoq9U2t\\\\\\\/o0vE01r+Z031GecHPuRow3p7IAMHa\\\\\\\/k1U5VzdcqyNEvOF9esEakSsVcXPSI2UkRslHEQTEvPPT2YiWINg5jVHYgSh6meycXBorzu2Slx\\\\\\\/4ifOG1z3pZEu87YubL2VIalg2FLMGeoOFYgMr42L8Im4AM3ZX\\\\\\\/0CgYPs1ErJL\\\\\\\/xKwsWlT6DUJdXoBL1LdOupvsSn+Rf0ItuhncISDE0LG1lveN0voeAaWuSTM1WJ7KEyhXEsYAuUXLC01Hzw9UIaVNRhbU8+iQFjhgstp6Z8vUMvvLPEmUWhi1HUvYFdR7M7ay7GcSBT7lA3dJIG\\\\\\\/R1BATMlYIkn+rNtAYhCYxo4VMTNmiNLbI37GgdAPO+S8H9lprJ6ZC22whT+KbSJFE5JHh7CzkKgSTZVy7EdPiiIJ7UdvMkgl9Of7Ip0ctBZRkSDbXblK3bwvxff6Ocd7izHddsvMbwjJSego8ythQUnJZpErDThGo0diNGn8GgI8FUyQcuAxuQsF0L572CL+Y7xTmpXuPhx11dTrroQ72Tc0o9qHsCex7GQao1KssadvKdX5C14\\\\\\\/JVJ9en64aA73ao9eKa0voUP7gZvBm5ZSX9u+Aj5Ayp1ZwIFrSIn0dv\\\\\\\/LDz1Abv5ubZ7e\\\\\\\/v4tqbIdjzBakFIbM+3vIFYVTJdvHZ+UQ+CFvb8n8nHc\\\\\\\/HaFK8ud27JkZQQ6\\\\\\\/PWYLoPM2RbZsU+pOAJjm5VF+yqby67rgbCIGFo1cZxPMNCiAU3qZzCkhML8mFHi\\\\\\\/x6twPVIpoOUu4okPgu5tzXM3ZfqAgvr2RdEHAPmCQnF7Xv9pWG50t+uQH2hDUJuPeQ8vUgWr6p9oNmetEr+2QGodsHFl7esDLDkFZ3vfWmJWpNn6G0VDSDqIGjiwdO8PFHogori0WfC3CI8Ikbt\\\\\\\/iJFaPYryej5Tq6D+lu6Kuwh\\\\\\\/nk1ewxIipE1gG8\\\\\\\/r98c9+z3aCyy1pbHQNhEMwyqsopBVW75NhvQYWwcalP7MWjlQxtI4oo7ltE+CrDD1zeMOCw+IOl29qJop4xNX\\\\\\\/K4TGliEKbHIiz2EXIZGxaKiE4eRRVbNCy77UXyB4ztQamCAb6sA5moX14Hl\\\\\\\/YKa4UK5Jartza8cyPgD+3Hk7t\\\\\\\/E9eW+O+Z+1CR50sFqeDoTiBA1WKjNGEG+9rPfKfiOiRuCfF7+ovYogPwhQgUZESu4MABxWuYsYxNb9yhlfYY1xxaJaYbHNOAOopB4BYa7WaLDs1O2K9XVSKG4vibQf0N6Zd5yBZfV5c25Nt+9tU1AUtOVmM3cxyAzN7S1XRmJuzFk42tpDQovlxE0I1A9fZqbISbe1efruEda5hWuVA8ZPJZw0UEZWRUE8IYUwAnSJSPIIdQCCRmWozbGDVQ6WJgFndSrroS5j3JJ4J3x2\\\\\\\/SMEGBHGxEe0DbIEZEpvzAIvyZBKP7YLn\\\\\\\/LYbuiGfubfy0JSVs29kyct5nmCVZ7HARj\\\\\\\/V4fNRmso64BmNPiTemStXSAJhbq26CIgglaraKzeWLsTtNxhB2Z1is6rvTBnWXJ8tG8Fq8seuoI+MZh1Mr7cl28J6Q0IiwwKVjZ8lqvJFnfrYPvzYPptVnTvnoHBxv5moDlSg9JTWsWmdfR\\\\\\\/VwgYdyfCPBRKlACcPfoVMkSAcaSiDw3j+cFcK9wwVMM44Ae8tr\\\\\\\/1nCFMdl+wuOj6wy2RGTQ9WdWuC1Uu2RoT1ZFGgbUQCX2ChPbbfSHvH+YhL5HRZk2Gs\\\\\\\/D\\\\\\\/EFiVprSYLaNNhCXPtmLHnlJBA\\\\\\\/GxDGzRlbY7RfVWGoNoCIFoMEj0qL8mcJcg+7LkZlRFj+XgS2HAQZWAhpfZ\\\\\\\/0DeFJOeLYi4Y1QcdwO5\\\\\\\/03b9ZxgZQ6O1eUFcMtGL9WzJZ7YppnfAsTrKcqgA4yVWYnaS1VloHprdCgMUCBvm6i\\\\\\\/mztFNwceBfuwGqzza7BrlPbY8zE3+BjNwFLHPDem\\\\\\\/AebCwYbJvQgWRWDB2Yb3e5ibsEeGDZt+xBeBFyXwm+cN+H1Fh38ANia4y3JR+NgdyRRxh6Ao3oRvxiqN0kTKlA8EW2PFPHQbQ6G3fA9gxdXDsaVUCUal9wXAl0dRsbLlBKTOpYQ6OsIajwaxfaUrZNDs19PsCPsMbFyR6i1oIuv+PuV4ypFmM+FqbvNGb7BthcZopJjysNXMd\\\\\\\/F0JPn+ayM47cUvWvIRT1TvFPAUktuY3nDvliZxcftmnRGGaDuykGAQ4vOycaqJtkVzcrrBaLt6ghqahxyGABOK+4M6M6lxjaeiLJ6+oXqPCm6i64jRAVQSXhEF1qO0rc68+MTzI97LjE4zS0HNhIg8sv03nbqZljq\\\\\\\/nhk5RvDIe+8ml70CLyvUcFXlk\\\\\\\/hqvQa5hDMKJOVcMV9jHLhQJARTKEzH+eB14lZXbYPz4rYOM3Hst\\\\\\\/gsSI+qLP83whIyIBEg0vt+iMzU5BrMYqS0y\\\\\\\/YK6wlYBLlDz8yPGBG7LyR7rtvp3gaamwjkMedAX+ZrMNg8KdROJm0rZaloHJxTmAWOe6L6j4xRe\\\\\\\/QRu9qYZsajYQK\\\\\\\/IcFecSnD5adSQlQUxSyAUYwIG82hFO2SYHXctT61zC6cYYGQlgqcx9RM+SgkfigBXQSeHw2zO3\\\\\\\/fzyMlbSjU6Zo1BdCjl6F+inTiSeLYTsTtrKPy8Fr1wdib3uV4BmhqX6PQAq+\\\\\\\/jZWUYJkV1Sjsr6748bR+G049uVASTyeSMq4QZukYRJ9mcXZXnNJ3T7K\\\\\\\/P2qWw7AczlAzqzOunrRJw7oEnEUGeAmqypSzmoMOd1tmgHSFin\\\\\\\/k4EKEdOmsSTHIleJuKoinBwKe1PZTHcEdqB3fdZLtZBS4kwtfKw+njCeJl7yS1OjHXmtTIzlyCZsnqqL6hMDu41GL8c6mdD6S1whhndsZqIIza3Wu7Wy9Axz4HqESFhN5QJSIiDFl4oucB0usz7ymdaCPNvUPpAsX\\\\\\\/SxrFAxVAyoc+BfSNDN35QSp15GpsXhPH8dwmRZHr436vnyj4Ou5W3IRW3Nr4+bdNPToHSsmhRS6QQ6M4USEWZiB55Vzt1eQejmVHaEJ+WG444RuDTnfFcYlsdJJYULpHktFMNjXln70URWoOXUq\\\\\\\/uekk1IS7mv\\\\\\\/+5rLh9ts71a9rvBF4q3VNyMSxiP30Lo0CmIKTitQs6yLEPhQ4cW3hwmtBj2D7WVDe1KF3kJ575YL448k8K3hxXRvnlgV3QdOcLQ9qGqbjJsDwmP89jGwrsWLKmtjvfxXd+Cbbwe5MYGSmtPX79KouOg1MUTmRbU1BxBgPNeWgw7YIgFJZ5pagHWpTSEaf2JmS1jHPZzFMTJ9ofXSal7djHQdkr\\\\\\\/a2f5OyRIESgq6Ad12qGdLNVYngOGO7H9R8Jx4oGBS12m6NQlWYw+j+XDtUviKFTu7QcL61GYGDCaeXpvV9GR+sH\\\\\\\/SNqqoJKoxgx4ZjOFM+YEEg79n7va4VA2XYn7kfPGhjrWHYmXf0PiQ6rEahgX64alUUM6tZME2aUbOK7TZSpKDI69Q1xb4DgpJ2vyUjls2OnYZfdJNwC6VpjJAmVymwBuM92kyYJdQFYb23g7geu9tpJWVBw8jco6V\\\\\\\/BDomp1VZJEdDhXCLqV5v9t+CuoZ7i2tJos0WEX94IxqqOCqZ9U82TjAnt\\\\\\\/4sJ3snVGjgZSxZPShqxiq7ds2KBo\\\\\\\/UFwu\\\\\\\/mq2RSPFKIL4tTJNBer9S\\\\\\\/Z6Qo2yR\\\\\\\/LophEYkFRJ1usHQVgNRL9ulk+f0Zcsg0bSdRfSUXgFi\\\\\\\/c7kovSt\\\\\\\/4YULmJgS2fQdzJEQa4nh6haHQA3X5rHGjUQcxWIOvHvqQyYvXYk8Z1ZQck5uFk6lPsUHM8gOXOlkXwTBGVWRLiYLYU088kiOEu9M7ZFTBXbX\\\\\\\/NuNGbeWFvAlnLwEXWz8piVplhxb7rpJTwsnI9KPy8DZV4z62K0+D9UZziKCnx8\\\\\\\/m+7pHtEjLgdiverkfeKfDl9QfKqKGFRosGXaklmlnqhuA1goQd2fyedoYi4tgiytrQg2JqaFU8hVVWZd4Sz9vwjSXYHkn18oBcF5Ee1y5QmliOSsOp0PzhSIOutEfKWl6z8ohKl+ZL21dquS2AYn2nBXhD5xBYDhDtaTL0RmQyUhwqI9jSaVbDkAxGmX4lA59oUx0aU5x0p1kHkVAqJdTmahuYG2uFcYN0Ky8GG1zJm6KPHGHPTRehyajxx\\\\\\\/J2GtLcIQo51VcXfwfmgkU+1aVfJVyO\\\\\\\/akcurzPNHkTlz4ibKQ+\\\\\\\/a2Dep7xLRhwX0aZFAaAXNrGVDw9apcJFOWBdvVKHb0f1l6WzV38\\\\\\\/0bpA1SMHN+x5SS6w1LTjcPjGbp\\\\\\\/ApYTrzc0jg9AOjwuULvj77geI8k0FpthedUkcdJ7H3dyJCpFnKCURau\\\\\\\/TiDWL04VHrugkpSdr+ac4p5fG3rH77O+krxX1BUp5Gdyg7+0bHeSfNMh1RFweoXmr1zJVrSMy6tWUOZN54CCr3aeKH9\\\\\\\/PnO8wVMelCdE2qDRLJ6hfjxRRGfRRc+6\\\\\\\/e2oYYYDqTrGUJ3\\\\\\\/jFS85yeF5PDwWSeNKVxFN82IF32hjf75dHdGVXlMaunr9twYDtJL43Rjm6GAHOXymMQsvFm0OsEPsn1rRGbKDsQKCPb8zkWgDiPB7bDKX\\\\\\\/XayOpMGKAWO0EEZxK4ZD2Xk\\\\\\\/2FU6TLKSleHbxdINQtEo830VdnnFU7ydfCCIlvfJkTL1zbiQ3TaVl2UjHSoVy6vOvSv\\\\\\\/4byMWk6yr7OIF2bOcQgoYkEjpGdmbxU5PEOYUVVjC9Nt7GqvijDl4tmvEQ\\\\\\\/6HI99dB7\\\\\\\/SsN4fFqR0yRt+GZkpb8Gass+3TCEXVu236SlAU7nDm0Dsi7gKglgPwuM9T6SgSYICRuVz5ZWEMFcZjVoo\\\\\\\/RXnsPYqJbTkzpsxxQGZE32BsnbjBztu1u949QAgPK+DKaiOM2AfHkXifyL5hzTYFaYx37tpYztwcnMNWxyRsxHyqWXRR1fKmDE7FAmx\\\\\\\/fyv9k\\\\\\\/998PCa3cITnlmhRGxZ42w6ov6CsCYtWAJFnpPHZJ76l0KJYaLUc7Y+P2KP4GyemJFXK9I+eEXj7QBksIvNwpeAyYbloomgtbe9\\\\\\\/bnQdbqyA8CZ8mTHcWWuZlK2mu33ErRwEmn6NE54tdGCbMxdQ6zX4fh1XI\\\\\\\/rlGwe2nsHv9p+9NFqSVXWbLiWH\\\\\\\/enKRGiCCVroEefbbwAxCI9QAW5xuN91pJTumaeSRWO+sGTvpb+gJlDXavPB0vx7XejbO24VDbz9Al1k5W7lP+3AxKyM8vihDlvyHWGyfZxbQsnXRmNBJHw6g3b4nqARColtYVz9TpZmGSMK\\\\\\\/vvJvACRlPsYkVhXPu9MSV6XchOew+PSQXsUMRPpDFyMO6DnO9qkMz3wo9Y7pkQHdUexHjvB1ssn3uAPlG1rc8gP0hgxjY6vd\\\\\\\/ystJPlufFrhDeWb1RHSd4cYhYHgUkrMEHBVGNAzOnWEqBwAzdHRvwH25I1RunvP93l6q0xKfZXzY8Sv7Dhzdq491sbss4QpBVfcE1gXPkks43A4nSFRyKLMr2Ac\\\\\\\/0R9MJeehrIsOT0ohf2sIBBzuI6MsGAZawc0jiUFQYwPNzBRwOJnF2UFvEqwDEEGhzmylgLhQHN0Ej4GSrdRXOQspcqstUUe4ZD5XpoZeHCc\\\\\\\/HZXBqx0AEHQFNYBXCSzG0GPY7xyQTn38NmcUbMlmIu+oZhNMHaBik2wQL+w1cwGS\\\\\\\/HJs+TJCoK8+9p7FvA7xvwk05TR5h1wovBwnv1J42SlzC2n5kj+W0XjXl7uUe70v3Mhw9QST1E6j615nM3T+wOp1jSUQ2Z91xRCUUABqqVsYXI4QVIzua4m2L+SnUzUmB7\\\\\\\/tmIRLCu3guJP3BrhxQ1eiQza\\\\\\\/o0YCoLX6SD1VE37\\\\\\\/nUisMRIVve\\\\\\\/TdG0Fsbg8U7D5\\\\\\\/oKw2kQlKFJJy5tjYht8sx7NxPOZ01pc14zJj9Uxx7Rab47T9KTsB2kn8baoj4fZpuP9RlR5mEHK6UOiZ0WHC3jOVJ+W+Pa7hoRpEGknPb5+kbDsodOSGD4M1WCZ28AFFhQL98ug\\\\\\\/V8PcnDusrYpRaCR+AMMnOZGW9OKEX+nkC1z3HRLzMwchVB58yUjk7s89NoWluy9GGLd3r9z1gtZsQOkD5Q4TSbYEbrtjWFzk3jwm5E24gToR1\\\\\\\/0hflyjz9mokFfE4lMTeWqjDbwXOsOkyKegKxd4LD5CngsxKuG0yXWwtu8IND6uUBtA4+GQQc2IXYjjMmF+v15orINUPsAjlwCXuqosdHTefnmgCcPJkpPVpJsV04Aw4bRQ8d\\\\\\\/ZmiY+og6wCdmgZEFVOgxrcakhsxKmUSFaPM\\\\\\\/dxFtwUPBxkrc1jWFUNv0AZs+QNulvuaPqtiBgXqjGvuFER98iTTco+ptbiJZEtGKUDzMpq1sFfjfFUHaRSApmnJ4+MZujJ29reHjBU5zBE3eQArQuRoQ1gPUxxrjcOSJAF19LMEGlxOj+8\\\\\\\/kEfVnSI3sZ7dvhT0\\\\\\\/kdtAKUAiDrCSbmgHMXV7O\\\\\\\/BR9Uq3HUgdxL8BNRCmTGeuwnfL0dU68KnLpcd8oNxIXriAB+5spN2mdFuge8JHe9NmYEOAewUqt8hb8DH0uSUp5ttqZ0IyQsFRqXdlEjoQJUn8ZIwIWLFvtfks5gKEyDyRKDWmZFPHbVrDV2pVPF\\\\\\\/MHZWmo9fkklCgtSFwmvZ2vLADvakPNl0WTKu89uZwcWijj+r\\\\\\\/qb\\\\\\\/vyNpmGLOQ1qjIvPX3vqEimdGvgIgtp3sImq37GEfAHYFdLsd2zSCH1bYBkoHouAckFVk2vfV4vpyLZkj4GxHB+IBBGRC7fna9kxy+z6+DopBG\\\\\\\/pO8hQgXOqli7RQpoL\\\\\\\/kJ\\\\\\\/qr5EtYuwxza9qidh3gDE\\\\\\\/+xIH5bBHpiC+SciQRTj45WBL2NjA64VYEg9j75KoVe68wUBm4I0CIpdTyorUd72R+SDBS67nD70AgBqc2KBTJAXUwAxnZXUM+yuXauc1KXPJfW9HjsIU1yeLseHjzoz85Ul7UKcIC3jqf9h4koUgQZNqlXI4RPmy2TF\\\\\\\/Yx6hFP54L4GUJg9RZmdxz0k3nv6hob+350Qg0gq1uMXQwIepvNud6O67ZBiHWqaNYd3JxdvfADNHga9MxDA1qn0NCx0cte1t2biYUbbQl9hxGCHd0MOtlIpJYpNCK1Us\\\\\\\/EN42rlZcXsuO2rseMU5q0xaKl5iHzqpRvdGXOhb3DfSfk1FBDV+OSB+HlMTSgJaU36lkzs1ZToyWkEimqbuQgHnwihGn1ZcVF3rq\\\\\\\/qJgqsN51rR9zmia+2YjxWLgNSoS8ztIMk+p4Lycbs3TkFSB1KKf7QJMiqY9fv624goPUo2DOvQPkaz\\\\\\\/H33reVtYPD+V2HYo7b1FmC8mMNpyx6h5RZqGzZFAV0TPVutgX56lgMWsfGO\\\\\\\/tGzNyJHEpiuqOZTXj1WWFQ3bDzeANeXFDVbvfB9NFWmLRNGP7XGOAnCxBnU\\\\\\\/bYIHk0d5F8CpJPGHuERhQRfca5ddQd3GSGxRLG9LGJbVXQvClkhK2j5EJBTwA9DvK9fkre\\\\\\\/VeE4yreAv3Sv+PICqIdANMvtIJn0ScKMUhFL7jS2Kl58ZoURLdOO9or7hy1Sbp1TmDpwhTzWQsT4TOQiGr1cx\\\\\\\/V4GMYh9mOhCxywgIyPTfkiOMqBoV43V4\\\\\\\/Ov2avZdWbHnt6hme0y5Lwe9PaRAB5pRoYqAmZsjXw\\\\\\\/6+TYvsep15hWhHcfMeGJJLGV\\\\\\\/tkDHZqYB9XN7SJhrbSLiDoRb9Si2AAMJfMMXzXmdK7V4eap53Yh7UHF54Ecd9dBhB7x9E7szCROHaIo0AU\\\\\\\/F1GdjTnx1RJtX+EIWDzTQHY7hiXfqzuKoox\\\\\\\/\\\\\\\/jqveBY1gWRVdReQn\\\\\\\/172jMSAXupKwTxIZhHuYGCh0FMeb2aeWiX6+wKMv4ETEwCGFlHfE87I0TumRUIP1DJYfBoarsrRax6BK2P+PqJrCINLe0kKNY\\\\\\\/6JZt9brqv\\\\\\\/KgpCYiMrQY1\\\\\\\/aPYyFKqew9wn5ioHTPkmXYeTVluIQE+DGBYfHK+3I6euuzMEpGIRp57yCFxQIRea8ERC7mrBCTlk3irACKd1TA4t26vNuWXHD+u6DVvBx1v+2amasa3t0p\\\\\\\/VDZf6LRVUJ8RygbsVKksDctvGxMfE0H7wptIhLQbWtseLTXie+OdcuFpJMF54bR7Z2542B9vIc\\\\\\\/y7QTl1FA5PfLgAiRPdGHsZTQkh21PxaMzElDn9cYnDf7niKZmhEWYxNpYVEKD4c7dR6XL+UsvcmyrwJ7U+FbM8UzWgZCPV5eRoly\\\\\\\/6N8oO\\\\\\\/8m5bEAKhuXWisGESMTOhyNE0biVoGMcSLN7Kb7dkk7u4NmqqktRIE8\\\\\\\/uLOdWUW3kt4CZYh05pFdZMreyk71pjAV\\\\\\\/KXhcsu9+s+JTmFmxt+fIPT9uyd8tfvi6Bh39O8pscw1k51xYRht4lxA9EXn21HQXu2wBdBLJRGEQM6NAjHn2onBBS4hkcdJJl47LKYU9KXd55dR9uiOjBNZxcb1GjdH5HAOSzBN1+R53Ub7t1wrtt6AXCs83BvuMhT9Marfk5W91bLULyhZik1zIM\\\\\\\/FZhXHF\\\\\\\/TKEazan\\\\\\\/BfloMD92E8hVfrGaDse\\\\\\\/9Ykn2wPaDc3vBx0uvQFiIVELYe2+p8nsREXR3MFgb05kU2O0g4CU8vfUkStGZlxZow57GLB5vmahqnPF3phxulvP6cMbI8azPzQ10Tj3Riiln4+hpDPpLozRABpFnnJRrP\\\\\\\/cwYwpF8+ydK+SfJ8BjjvWs2i8gB4k1ZmqbsSvkGPEfOeqf92qM72F2Q3kfH6hhS21hdnXsxsexRFmWrKe6VeD3Rrh7OeD6NhS\\\\\\\/VgHdgZgU0MAfdBLa7mOKywn43E12kleIUBHN\\\\\\\/CKOTWwDsPDxyzwishmZfO+ukPKaysHTik3ODlNjNMldu2r+or6+swhOas8J3Lj1bX6nyGqA+J+07aOMs31kRSEQk9ObSDyDVXvgyZ6l+RtAkcY0Ad3bLz0+7ErjbRatk4fsT181DAMp99gQh4GFs8z4Ej4VdwFbTF1byQ+xgnfwVliYCx0FI46zWs0fgzFFAr\\\\\\\/11G9P1V\\\\\\\/z4KkJsnHv7iHSnKDbg6jNgOLhgAM6aSNkpxQK1lVOQ7KEd\\\\\\\/wsGCYvEDts\\\\\\\/R1US1JoSZZ3He1DopkyhI3hf+Tt+e6LWzUwr20KaqPHiS4uYl\\\\\\\/gLvQklEneiLvR5Yqz3QW1a4KY0EImPOP+DTNT8Y6y1j\\\\\\\/xll15oSRAhiL\\\\\\\/WE1izmIT6uIksPYCNgvYjS5eCU0yRn8Jhx6Un6D0QK9ytd4NaSZrwr2M8TcDX6D+oeZ+nuickaibfFaSAL5bBJBpiDFGa+VhI8BDkJP0z4okx8uV1EOzweZ7X+zAm8B+NTBVCfQ5gqm84Sesb5Izjjh11A3cW7d9hrNGC1aEfoT5GdbfcjKte4\\\\\\\/9SC7qIi12LgB8TCbExXnnukhLvtuR2m1JE06RUOs\\\\\\\/q+TFeqVAYX3yDaxR0N6M6J8mNynWROFhG6\\\\\\\/RxpjBDIk+iI8noNyKmc1jzuGpxuPr60yq0s0piKLOtNrcDOKIoCfuoN7qXALFfnEaxDuM9QfcWYJGgus\\\\\\\/J5PRG+uoVXgrZ2+55b7ovOXyiFyrgm1pMFj5vp+DLpvrDVMEwlhyYoQD3TdJYmg2nkLPB+KJHg0mvgSm6OcRnAYyEV532mqLqb5SModEb5sb6dlOqsj3d+mH9+4nYKMuvX8e0IGU8DQ9hJgxvS7LcfuTRK2yPNWKIn\\\\\\\/v4oz+483xFDGCGBOfp3kfOQJv46WxUX8oEf1XMsAM\\\\\\\/OoWc28enK\\\\\\\/YfwfIUPQWpz97OBEc\\\\\\\/6xGk9aODvyxhoP2zyflY\\\\\\\/hN\\\\\\\/u6Bp8Va56klTcjGbKc0EgLnixfUL1vm9iYGPAid4S1I471xWadYEthCl7vESdZtWFx1fMwFco56Bmg4aeNhjdY063iR6LhTwvMmNNZtg218IymeEi+57WLbNROZ6\\\\\\\/CGLfjXvErPGbsqzX3VReZ\\\\\\\/OZ7VNWRbQ7N2\\\\\\\/6m4yydWP7b5enTtpHA2J8YZj5MQ\\\\\\\/IcnN6jkYhuMwSjEbrWsxvuOAHeaRYlefwBy3WSWCsklqe4pTZXPaZpQGNpS2psi+1\\\\\\\/9vPUkLO9BwgM+6MU+EX03MdS+k8uhVZM6U2PJ3DwWjLp+kQnE0+v9zFvrf5kLH59ozTmxhtaXnFS6Uv344cFpL6uZXcRXbG4glpmaflz8uh3jYbm2PyrRrGg\\\\\\\/s+oMEeH11mgdIIhJGgIGLeNhAQxS35h89dAY16YEzOnoz2ter1klygtiGBLM0g1kx2qoTkTywYWGfc\\\\\\\/mL+W42Bn7kkJX1DBHWXCSLhf2f4qyAWSkancpC2J29YrZxTUyBFt+4u+mEvbjVmlopmHxmp7i7uWeyNMloA46DLuP26wyx+kklHFxe2+dq2Cs5TSI9\\\\\\\/Vs5b6nnWan84riH2cJmV+Kf\\\\\\\/RRDgF8zH27sccz4J9MvDPGsqepEFM70sxS1EapqIgC+YPQctzgKTXiml7axRx8xA0HaOsBxGUetIdw\\\\\\\/aTMUxosWTKCMeSlkMUXVlKqBGDjAVfGhOeyPeXT1q3ZLjAZt+NABTUbh8JMbO\\\\\\\/\\\\\\\/bqHOALFw85EoEHb1ZS03n6fBv4hoJwi05B0Lf59aglk+I7wyc7nGMiVl300oRf4CWNuEq2m8Bo4ZLr3F3wRQBBWm1PxQ0pONjmTQokB1\\\\\\\/xeO5w\\\\\\\/nJFkPZuiYmUcWueukfMZXn05o38MxPkRnfpbJcZDumR6cKEirLHlmbffD85MFjRArE5TRzmk7cfwB6RO1CgoUScGLypl96FJofcC10D0R7rl2\\\\\\\/WziBynBOwyFeLq8Zb9kY9CBM0qSOLX0PSK7QhE\\\\\\\/Wm6\\\\\\\/i1S7+52sWkqbFhwku9TQuOXtO6Xmx2n8qgUrZVBS8xo8DuqfbsLlaGb4s8n4ZD0ycixDNZamBUvIJesDq2h6Vbd7m29ldTe2ZBH6KEul7QzzPN3nhF52LrVlAyXaX6ngScq3l9j\\\\\\\/V2Q+hB4r\\\\\\\/KbGU15ckvw6wYFtAXvWm1DffFaJ7tM09zbckb4sFspC8ikNImNp+ZZelckUCqbVvzf\\\\\\\/Qc\\\\\\\/XAe5zWy8wKD9Jb8feTFaQNtQqqOeUAntyWf4x7DFf332Ljk0513Scm6p2BZnSOa7pv7GR\\\\\\\/PhQqEx4sXLkl7UZ54xz+XSH47L755jZWyuEyQhDEH7fTdaiVPp3f7H12O4BH1A2ehwH\\\\\\\/TYrOMBpr2GLF6biyOCnuBEbf5qJMoY8Na0fM8KLPfZG+\\\\\\\/R\\\\\\\/vOEac27orQMl32Ml7N85209kFlIuAU3bJpeQpEfihzlrnG5MEBW1ItObvjN\\\\\\\/URaQcYUfGe6KBWgYMRG1TkY39XYqqY75pYa0R+9YXh8UA6HLHRadLXyHPFYeF0TMgnGO08E3HC8fAUw9k3nuHm\\\\\\\/GkBAzRpUIuRst7sosSiZtqoq8cu+SWbvfBl5zPA+tHjv6lOVlKxcW4xyAx6\\\\\\\/8Mcb4zO7eOHGW7pRqkYaqFpoa2oF\\\\\\\/BV93AshLgJ11QUZ16R97UF5ypPmRH6lXCqO7\\\\\\\/9UY9yFvXVrfrOtbmNgw3CnozdESjfxzwzhVAuYp1WEf3RRp3UXHb7lPEaoi8D9oP9QIAHVQH8+w+9ga1plQ4MPJ1fDPPzOfNqPcyuDP2H2hkwTyQ3qMZqzXHz8nyADG8CGMigJMSSGVqwAkI5BYue5a75Q0MV8Bq9TChjlf0eg+tuNmKOuuh2cWedXEz5RZLcnjH\\\\\\\/LT+OEQkC162hi7YrQoHvLMdTHa+pxqmqeLmKXOLCXHvKyFULUK2uXmsPSsI4fy7VVy8GZbjVthx39+U0Gt9COTFXl4EpxwRa0+xi+xFJIQDwjkhyX2fuaAJN7JlCAwQwM4Zn2oUTCZqkcomQaoDDSQGfywHOKKcJW9Csj\\\\\\\/YQ11QWsg14nWKiBbNlu7nHcdvdGRiw3TAmGC9Ju8T7px9EpdODd+8txVk\\\\\\\/UD2YP1\\\\\\\/LzE0FWJ3EEhbMGiz7j2tNGKZF9bEmmD2wWd5wE+\\\\\\\/TVVlWsSNLpS+dXLU\\\\\\\/61eXca2E1pp\\\\\\\/i2pF72JuJ531hUVz8WLi3Z8gzQXERrdsHnvEC7C8g4NXhxCAUWhLJN5\\\\\\\/iV3hAMVVjJw0CbUo0JQzmPq3SUrRL+UA59V8F7br\\\\\\\/ITT41w\\\\\\\/jOKM0ZYboipGQ7AkRL0eyPk0grWgdQjk5jXxIeS6Qq\\\\\\\/kJ0mgMLsb3M\\\\\\\/gN8jMUwVvg\\\\\\\/fTdWg9q9puepaisTLE7x5IfySetuJL6PjG9W9kozm4C+e4I5\\\\\\\/ae4jKtky6cuJxm2jE7F\\\\\\\/cH1tVZGEQSuYzwuUUZULqZNShntNh95vH1EzFJYYlkgWDnTKolq+cpvZusoQYV8a601n0Yw6x1PgC96N2U26xbqio04njAOF2F6HZ281wtc\\\\\\\/tnWV0EmvDzGAUfauIaywxC1t1CmLlbWPpqDbdYGcLJ2KVL1ZOEDfwzJJGC2woP5Fi8jOJfMHh+TLO7zxfEK65vGwDcjOCUWp6jS97xv4s95WS8wHHWij1TOXetti9eGarb9d5+kvCytiQxu2tK6ahzfWq0rdhoL93KOlHANLCmNiTokU4ba66T8wJ2PKkfTz6bAQ8golOr6L6KC4xQ0mbyjVeaFTd4eLCu\\\\\\\/34KgfaLFeaRM\\\\\\\/tIIDYJJE5LaWvzzXQpjeX\\\\\\\/ECBXpFBD1mbSYlEeHsO3xabvBJyx8\\\\\\\/qX4DCwjWiyixgCCD02vww5YB8NKiPeoadfzYoYy95y5tjWkbChA95Qp+wn0Ts+Z+oXtcAZ7E3CBHHHcnBqe7I\\\\\\\/G5XbZPj9ZvK9KIxnsDtMv5RZ8iSyqcFqcMcbC2BtVbCyXPPnUYMAAAI0Gy5P2kL2jfTv8mDTdJo1AfGHZv1glN0+LJpGjZzW3qmaWXkxuI8YA96mw6n3Izn38RApHBD4JZDkL0FgxNno07EMFDMOGVbss68+QQyeW\\\\\\\/u\\\\\\\/cMRKGi8kHOCthnJB0xq1cVbt+URdzHepY0IK\\\\\\\/V+cZjdbmpa9DSQiC3PYLQHsrx0DoIpHKbJbsUFQeCx05Iu\\\\\\\/gmRxdqpSLOunElQWCNCLMZr5Sc0jaTN3FPYqmQxzHVgDhTpSvWPQSfn59jRKbvnP\\\\\\\/oTruehJj\\\\\\\/vCJkOpCykJRg1OKPo0j7ONoRzZ3LJGG8C9VyTOiW4kr5MPuIDjXymX0GhFkCCJ7D3ypFZB7Hv2BIOXlGeWpyKtgZ0hBOz3IfhuGIKoUA3HiRkh1MdWOb4nrQ393up46LVH1u299XVQA7mDu31U8SPJXh4nRD\\\\\\\/YBaQPZduOGzTPEzxtYt5Vsf7e9w97ITGL9HyKX6QotTujmjZ9qZH0fiFD95o4SBddsH4JM8RkEKqROUiL+nVtQUClOWu\\\\\\\/4AEbbuia33JEfMb79hFVgy8Eh+wXinsTjFLC7KQeTDnyH0GH3Oz1EvRlf6lelMbyEi5q5IafxyarvXWinF6Ss0O0+wrKLpT4\\\\\\\/TM2UFmjbFeneDmkuD0spki7PGYP8kfFLTKaGUqCemcOrpmQ261koSaabsMLh6OqQwI78FSf37lAgVnC\\\\\\\/FtAqaks6yyCp5dReMNGylp5qo84vQBM8SIdR0s+24TQP\\\\\\\/WC+WjViY73OzTF5Z4U13P2vjCAvzNyQ3NTuagXzq8kYrE\\\\\\\/KBqLNCbpMlTmYeCxh8o+tMgdQmMd0ORpSIaB5McT6h61RwQdhmAUeCa44mrNg0+xrd3T60gxgoQGHKGfH1iKK18yj4s6JIdGBlWfrRYmYHOn\\\\\\\/kwG0gilvIBPMjaITb1UZp8XWctzB34FH8dpah4qgD8GvgCAkK642NCjbqt9fnWLD\\\\\\\/o37Tkv35DbYQyJ2a89Ed3lW6ap4KpveNGSxG6ho8O95k18ql\\\\\\\/v9ZRmqO24XUSZMVHmRr4ISrC3ep18pZrloMlRtjYYHh+uTfbIpmdA+9Njv1nG1QPJxByiHdViN4RNKckLmY+M8H3xY3mtEgYD12uEAQ3rVG+cuOnNc3cMaS0Ptybk67VV8rviqFvsx38w5hfcZTaD5\\\\\\\/k6FzEH1vCK9aZ7NcMqeI3t+WrgWnaypclyHRnfcIhsHHLLMx0PbGNFMQyU7zN\\\\\\\/wAKiupsCQ3aY37CBMO4MvQZimLjHr0OuNG6nShxdKK6zbKbTmvQ5EXDjnZh9Dk2vmxEkXgLM0j5JuuFmEsENpQanhfeSpHjT27gPR3JajW8iqDk0M72Qspr7\\\\\\\/W6g1y\\\\\\\/mbBzIFx0b1JbL0sIwKQ3VyCnG+\\\\\\\/T9mozq4Wr+qGyAyF0uiC+T0YBkpxMnJgqLubBl8zv300OJegBlu\\\\\\\/+h3q0xibVXrLrFEHSd9pmJ6AVs8QMUif4CzpBR1saM+d9VofkP1aDuQFEZLLUUuqGLmLCZtdKEC6WUgp8kWXE+5va3TIkCmOpOKM\\\\\\\/ShUotyEx\\\\\\\/hLUCEsIQ9j6hodOnVSz7LkmGVLK3jXVk0d9BBnhwlMBHOaHUlH3i5seA15bXFUNp8gViPNXPY2fDU+NHHccN3bmtynZoncLgtib5+NCGNW7iu++Syn4V8OJ2vOBM6ahRIJ5FifjpBXoUeqq\\\\\\\/iuiGTbEClK+jnHO+rOX7ubIGNMOOkGQ2q7q32FQFRxK8Kr35vwxz4giOJqA8Pp4mW\\\\\\\/exeRiBN6fBoByv8bF2H2uJxnT5gO5ChoxrOMntfbkykOo8\\\\\\\/A0Vr300a1Z0637mrrOc56i9x8fEMtz7LzOB1hrex\\\\\\\/WN5xmEsO5ZcALYtAuj2yYLRVct5GcZr9AZjZlq4+SvZi48QMZOewDFuig0TPhZwOYJpFjaXmWh9IC82JOoqkGS8qJsAQMMl8HNAJWMF6sq3Ytmq0IjjJnvjRezJJ8fT9IeGXG+fYfxa9wlyqmQeVGl2Z\\\\\\\/Vr0d\\\\\\\/kP1m3mtrK03oa\\\\\\\/7hY1HAxzAH5YlHSOyL+CvvcYSGKBCalDAm3J067D1gAYshytVzenKYhgDcENDTxVfJdYA3LqkdCxlkr6RdFRi8zBlwzvEIGNDLcEEQ9X4e9GEASKFAnggDJmTTtxT0kZo04xCv4bg\\\\\\\/qpODGJFqcvNntcA9pEiINU4wzFGOyYA+V\\\\\\\/1lybn+G+2fjeqUPBskLwdkvAAKssO+OQnna\\\\\\\/Hzq\\\\\\\/5JUur3cPIfgmXWaqpVYKP9tbc2Rb\\\\\\\/mihP1FRvGySPwtp9KsBPTp2sqKKYmxFIYdC0+V+MRcr5j5dwCUhq6QaMP1QGEIEWbW0tvTLksvtGgh7V6DhGnrbdqr2M6kSgA8gpuN0fifDOXepbYZo7\\\\\\\/lHmCqLifMo1XM9HNVrvyVLnqjh0gVdEu+PF\\\\\\\/CbsQsmZhFRUEa0mfWCYvZQzdWYGp\\\\\\\/mjKu83SHdl73+FRs6JXgHzKUoQTh2vtW8Kp\\\\\\\/opPxKDJvApq27HGhgyMG8Rq2JWew5uRKaC1vCx33gv\\\\\\\/x9I1B2ErMUveBDwGNN0zAeQ++9l8MOKZLA1bMb5zxJ+1ErTr5pcAZpcrP+3WaHF1qwgyWrzP7XF9Y1SSiIR\\\\\\\/xdnmmavV7DG0RJLdF7Q3QVg1NbLFIZ1qdQoQAZl0Vku61H7SfZ9GKdYNv3cVnzRwJbXQ6zAJV7j+rJagXDbW6nK+U4rftms\\\\\\\/fqMyqPpheBIQEYZA8WRw61HbALjTMwggzzPzh4MoFE8c7Br5KZuyyAg6rAFmaqw+n\\\\\\\/oVh+9LttcX8l5ThkHV5jAdYWB+nl1\\\\\\\/2f7LjccMA7hZRaKf9WzACm+k2r4rpjBv7KFLHhqJN6nx3CNZk4dAa8+8cttHg6GvFK+8iRtzn9Xsr5MUi1W5R7Bo3CNez1syUC6eYPkPuw3TB++FBb0HEs\\\\\\\/PaytB5z50v\\\\\\\/CcNY2tAT1y5h7XnollMuDD8bejpsS\\\\\\\/neIHslEZKva1jvj5HBE4a2ASibqC\\\\\\\/KvSkdSFSlV\\\\\\\/6dp3Pa9GaCNWuBozZlkBR6jS0EM71WFJaH8MqyLH05gRChQ2bxwazGVa58\\\\\\\/BxQBL1zM2\\\\\\\/H\\\\\\\/euTS++4ZzBQUmG+NFxk\\\\\\\/uuFG+mKN6Jp1cfmbm+TIPmnmmW1RU70jpsjA2UQqEK9CiIFstPRdly4NCr4TSCuQDP0RXlcPZdKEecx+19N52AqN\\\\\\\/5vRJZKw8LXVz4bsBAkUTFV+FXJb3y3f842LjBqFczP\\\\\\\/nzpxHTX6saoyE7rC9WiwFcW69x3ByMLA0bsmwvjdNsHilrNGDCmnR0j2NkAYf56eoqeF5bPzmP9uK+BFGx11iUwcGew9vyM\\\\\\\/av7sCEsI2tt7rytT1p\\\\\\\/9XtEaE\\\\\\\/oZcUK+9BlwiiBSqyrwsloKbo9j93Mjx8FTsqSuA\\\\\\\/cvCVsh1Q5yUZx2oHZ1HoaXBjynVnpDAWGxJ8j1Ww7YWMG6ybYN\\\\\\\/4FIiApg\\\\\\\/WNLsGpOTXSl9rIc1Y00d42740WgCrfRlbkcetyUVoN9y7YCyJHNEtQrMC16uhkkESmkqezoCpY8pWnCHyGwXH6OZrYkgg36egkULuI22vNt8wpa2C15eYcl79VGWLPLPTUiZS6KYqaFiyIOa1HiAe\\\\\\\/3G2cd4CWkVwgpjhHQJGiE+V517suY02CNkRDxTonlleIlfOH721xX\\\\\\\/Z2QeO4fA8wesS5pSd8QhAXRzodt1yqE9GCxamAnxoHnJFleGNjkTP7FVOjpNmwEvWvs1QefOTEYA057MKXsgKDFHwzHvw+B6+uZ4PNRG5OurbKFWua2kEwkBUXwLX7DdC6lQQg1aAFQJ3zgGHxAK5L142z5wbaeIhvAFzVEHEAn3HOYFKLr9nQVwJGdgEWUunNw1Dx2EZLb6\\\\\\\/Ss5Uq1Ebq72xjeIscAJD4CNuYtukQ30LWSr8zsohwh01jYKZ0TmU+HkrltMWU\\\\\\\/P27WSM9AOG5hc1\\\\\\\/3tZnXl89gBzYQ4f+LSvWhonBtSvYKfHdDDRC5129xp59JxyywMf+Za7echm+hYx6H+W+2\\\\\\\/sdTdDq6uJCO7fPXDlddKPjiMspe4hJcP6IL0Jep0B93DaUKEG9s1KXSQy1xIOrRp8xPwypiRVH3BilmhO+LTUZRcaMS+lVkBBwAqvdSmlCMwiIGNV4WnqTPtcAQ7F+DF1ZObv8VZ7ZGsKqjZDcCeuN+WUzropf3e6ONxZ+o2SwtVaMmQTzi6mAI9gqQwsqpXCMeC4m8nrYbtOEuZYbwM18+d6ok7eq7uupVdK5w20NW7v+W2FaIXNZPYfVWjXCgHGGKoIWftygycrVGIJVkMYNduKi69wf+n+VJo05JU4SxYEyNRW5M3Art9TcYgWF7Jd+no0KfLhV+72snCaoXU0d\\\\\\\/XC9CNhQkvWpg2TAOpkBRqwW455yIJ2K1Og3Aic2QKgXjvPHtGwyOeKmJvVBk43\\\\\\\/MqPkf9cUaR9\\\\\\\/IiDyQMENQpBemfXRHPZ31AwJVLpJH8e1xhP4hkEBTuW+5xljV1x0nOlxYLa+kdAcYTlSWnOQsEBUXkQ3DDm8GzDId8A5cMasZZMoL9kE2IOVVO2zaqyBSAzVsB5PtOgkYxqJuMLiamKDQiJkRYP6rDmfUWiicmqhlg3WPu8LEtgWSE\\\\\\\/gQimWONmEFe7EU7\\\\\\\/iuNh2wh\\\\\\\/NLmrVGIIgaesy3+DB9gjp3KX0XpCo7KgfWXZeznZXm5M7rVpOy2mSHZQ9GqpcizAS3e\\\\\\\/lp5IMVshZkk1H2EBQFH\\\\\\\/EXdF46YbwSJx2M7pHlOeXnifuoLyuD0GIVFfLNLr+MKNJnAjBnSAF8WVS2AjdDnbi+wy42vIJerxpx3Qp2w42DcZAwSFUlowI4jgXbj3u0S7tk\\\\\\\/78Ic8pvVj9EKtpdJI7jsd78hQbl14nwVwIzS1kUHwodzVmMM4w4mRYMNA9Cgk5t+1vbWEIW6Ddn1xmEF0PVECLlnbw0J3YzYZiHsit0cDDR+Dy0bYybGdFczhvMX0v5LRaZUF+uO2ybLfIQq8poTUUmT60zWO8JWOXweWfSaIpmXcT9h7LGVd2IvL\\\\\\\/+KQorS+lj0tXQuWLvSDS5syqQPtoRQucv9J+mnzP8AFJDfYvKP6pidv0hMyAVay+eOUlWq4JvvayYTPQFNcVXI3QKqHAFkmw6+8d40aWIqK+VsVlnj8uZX7mOj3M\\\\\\\/lxlvDHL\\\\\\\/Wf7K4DitccUwZ3+9LQO+VVi10BBxlooP0dVPU3dJKsth7wOj4QwYAiBytoTAeMw7\\\\\\\/bQ0H2x2JJsWD\\\\\\\/hviw+jaDa4btG6qYU27zOrvsBdHf3LtGqXosYVYWVTA2HrBQiBqE8OLxXkiGmPs4Xunf9YkUVqmQiHsRHojlg+NjUPCG9Zh6lC3n24LZ9qflN4wQs5UYVJYafiXblA8L4ESOjbHUkxBiVG6k910VoGQ5CBS8SV35YnTh9dtuWLcFr9dogZgCBkl4RufP94CFzGF+I6jhqRuVwV1YwoD\\\\\\\/nVzfwTE95l6iJcICNUh+9j9RYTvrr9Rl0FJHI8ChhafML1BG7nkX5OTDPlSQyxBGlYeFvTYotk5krGTgSMUtBxu1WVsR5mx+RMQalt5dM3dgcb3HzFZgupAfFBBeamTERDb\\\\\\\/k3yjRCTezvguwsz1ytlCDy88ul8yGtXp9yyPWJjA\\\\\\\/xpzwqmQg2V7AaA5LdASTAsZXorLeWsLQcmrRD0wWiPAArSrQ\\\\\\\/bfTyx1v4v8lzAZrOL12Yt7d6Nfuj+L66V\\\\\\\/WtEqF8I9eklxtAsThaXlRRXo3HljMNaVsDHAxwkzBfBwyU6GcMhVsKW5wSJ7TmOnMrLoZJ0NIhXZ2b2G5p+BCXpcV54Yxlc5Ja+H83Cptii4cU+GgvQ3TZNQ8vKX41bGIqNtzOcf\\\\\\\/WT1\\\\\\\/Df73bAyi7vZMgXE0EIvTSmb9916mYUq2EvdBaQdeKd0iqs52XtCA9anE006tUq0rIL8UbB87jlaCJqMwWhb00NtCfACtzeJrWKzAUheJEASoQ78XIHXfaKACt14j+po9KFPQCayfgew+StLdAFyuPATLnrifaxsq\\\\\\\/bthi33nTTsQZv372iVsG7TQb04weo0g4w7KuLQorDOch9kSIh0TnvB6Dx1BbWLtFeKZ5QAbWwc7jzVtoGkygY2S0jynkxY+Ge3p\\\\\\\/OJygK1uaRZdV2GQG+3b0MHH3WBkumpsn1f4cKsTrz+eNx6fad1Pj7lViVKW0r56MebJ0KKm8ZcvX2D7kqCXSJ8tyKoUb+UY86UjL4pxkQzirHnT1U5J20zhN5D3PQbKn9gm7DVD\\\\\\\/tc\\\\\\\/gA+EJWu0lrsMIaUrQnFKyUUEFt0ywpyf0L0k3\\\\\\\/gSuQJByP1uc6MgrB6yaHJrGqgS\\\\\\\/q03P\\\\\\\/7IkYsUhWbleGj92D4JBv6RAnizHfIoTZB2YLfiaO2aKCOEDf\\\\\\\/eeQFg4c48TprWGyJTi9xgR0txXBCoWXERQvSrNatCTwc8jLJh8\\\\\\\/JX\\\\\\\/XVi3Jkv0X7bXACUyfLT0erEo4I8PgPgks8O0SemKSSGhMbQR0PW0JBM5mjDrpEI3DJCSK8A==\\\u0022,\\\u0022iv\\\u0022:\\\u00222PD0aiNNk6j0El9QIohW7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022919b87871e6d9ff0f54479b5daf8ff9f581b5707e27830ed37da06f21f67e795\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ptWrt1WjDD+zCeLWFJQrNzAFUIvn7l7\\\\\\\/CiAaAjygZEsj\\\\\\\/E3l6xkGRNErgDhDUm2uUWWjhuf4lz2a0ASDqcQCw3Ftehmwlk8yvWv\\\\\\\/xNfbT0KAqV5KXXZBlVScAGJ+uieW9HXfBQMXtk+JoT4UxnrF+6Lq\\\\\\\/d4DiHHxMK0+5\\\\\\\/WYSnKlEpQ3IX3xHcb03HqZRXLIkeY8Iy+w9HAK7UovEcnKe1gPFY735CRFecmV+yKFlZNlJonMDwN2eXlot+sEOIiMC3wTeOYsf3kBs5q4yI80XRLnXK+wu9S3OPKsDRFuNHhaRxXLtuc5xhI1ZBfrOCEmsq36ZkxmfC2uPVe40YMMPKKxPqR7m9J2H2kCu9TZOd\\\\\\\/Sg+zx+XVekEK6w6xwM9CTm07HvX1jyXmtknh\\\\\\\/qAXaj2+FKzwuP161L58FWMVBnkUS+gjel6egYdwG1zt0fpEUMu7QCKs5PkHpByItplsVUGdV\\\\\\\/CAIGo9HXQYrvsxdRxCJxtG2uddmj6NRrpQx3yXP3SnGOwtU9PTaMDRV7Ni\\\\\\\/zRrmyA1oDhTwMOoAch3aece99hwwB0M6upGTboDmkuuniHUP1CyjBUCOt8f6dOaZ0mttQ7YdMo97BtRKQUTJvBc49zVWcplYPN0qLhVu6XEPuZLPvS9PwOl8QCQK6ADhHi+zttfVYAfg7v0xDEyuKvighO2lxhdftanPQpg09gaQXzEnvgWct0SXEUcrvXEC6L0oTyMiVMb1dQgMWWDdxip2yVt0X2JDtsgQuPxG+vVw8dttKU9OGuL984o+iLmWhAMrt8UcKyNBIFvOWYehKCWHqIEbRarzz1EOR2KXd8p5AQLh0WPbewN0y0hmHB7JtwN7+Fxysn9q2a80R6Jpv5uPExmwEmCvG\\\\\\\/f+ukHBkPdiPM\\\\\\\/FRzz1iN79Kd3j1cD33YMHaNIDqEoRpMIfQu8fG9joCgr8dicvyWMeoAbInB08RduID291XgyzmmQXrWdGBQNy06H2sQjzuSGuZ\\\\\\\/rke+KxBg1uXQEXfr\\\\\\\/ZIhCE71zFAoWnUMueu5OmsuPcVIA+qjBVo9p4QfkhwWQVT8N6\\\\\\\/48MNNUSPs7nYFQ+aDvpUqDt7DCvX24DAHZB2YDgvypOKWIwhn8UitVHavP9JYvW4p1p0w7emjDbl44se6nqpQZJriiElvrvoLWE0uAxuZ01tWteXIzCeP+11Qvg+9OQ17jQ3iHAOW+jW9osZ8lbt7TUGK4YnGIKB778RYqlopPDVXpY23zN26ND1ep\\\\\\\/mYPREbg1Tbu1Km1Sr9uFwEWeojg5bqE8+8WkDoP\\\\\\\/M96NtCwLSs7Oui0bUFQkjlTNqMZqkeQelTJXuImvT+5o3iqIMsiAiuMrcvVfNjGpmTFg3Qvdl44ii0k7x15T5O55I5E4LnPibNAZcs6R31h8E0tO0uoX5WNyw+kXXQREJopsrEGmyeHaklghyEzFxm3rMClWB7SjF1HwZSsQC\\\\\\\/qjKYzhDOc83XlrBpz+WG278bB84d43JFNJNIdACkpdGrQEsExUdLDGgwH7\\\\\\\/zDb7W9X+3ve\\\\\\\/YfjsocELfUo1LKV1be7Jb7VETZd14JMOg3HNRt4XyIzEf5GCy1RhTQhpLK5lO0tcbzAenXzQje3eD\\\\\\\/geSsrQ6Rqbvau63yugKfbtRNNAtxxNHJ\\\\\\\/KWrGM+kz5o7yGE57bOqoTf6NEp36nigOStmlF8+0JiplRN0HhunQAsxc5Rzs0MB2Gx2yAaYubs0x2o4\\\\\\\/QkRqsnW1DY3m2+sbHnlOR5fHXCceFmNCQzA\\\\\\\/U4+sZF2rXqzi1oHYer+sFbTX6B+mSs3+oLkXabxmT4ffMeJ4+tncli\\\\\\\/wBXzjpXDUvhpRoEIqFb+h5XPny2ctTGfwdI56LZ9fQisSmV2lyVsT7jvOdHw+aecJhuk2y1+Ckn7e7+vdQr7wrs\\\\\\\/rpqTvin75XyuN7iR1jbRVeJ+evTzY5Uj6VC+ldkNHYoWp9wQzVxAksQIn3pfEzTpNI7acmRyF6TbPLndKjzBaXiH7CDNlaN8RStIPCw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022uNnHX5umoqd+\\\\\\\/gL42fENBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c92f78dc95f05f7305b6db0b048fa95d60b2f4919f4f5ef3984bf89bce93a37e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459661 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1717" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HkrnpXPGg0yRQ8vwEACiKTiUTt--hnIySrIqCeRwIWc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1717" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YOgsDiSGoigjPqP2xoUsAdScG5MHZ2vfPMySsF8SclQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1709" title="Position Effect Variegation and Viability Are Both Sensitive to Dosage of Constitutive Heterochromatin in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1731" title="Genome-Wide Characterization of Light-Regulated Genes in Neurospora crassa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459661" id="node459661" data-pisa="ggg;4/9/1717" data-pisa-master="ggg;g3.114.012294" data-apath="/ggg/4/9/1717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Mapping Between Coho Salmon (<em>Oncorhynchus kisutch</em>) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miyako Kodama</span>, <span class="highwire-citation-author" data-delta="1">Marine S. O. Brieuc</span>, <span class="highwire-citation-author" data-delta="2">Robert H. Devlin</span>, <span class="highwire-citation-author" data-delta="3">Jeffrey J. Hard</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3275-8778" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kerry A. Naish</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1717-1730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012294 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459661" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miyako Kodama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miyako%2BKodama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kodama%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiyako%2BKodama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkodama@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mkodama@uw.edu</a>
<a href="mailto:knaish@uw.edu" class="" data-icon-position="" data-hide-link-title="0">knaish@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marine S. O. Brieuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BS.%2BO.%2BBrieuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brieuc%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarine%2BS.%2BO.%2BBrieuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert H. Devlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Fisheries and Oceans Canada, West Vancouver, British Columbia, V7K 1N6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH.%2BDevlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devlin%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH.%2BDevlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey J. Hard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Marine Fisheries Service, Northwest Fisheries Science Center, Seattle, Washington 98112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BJ.%2BHard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hard%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BJ.%2BHard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kerry A. Naish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerry%2BA.%2BNaish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naish%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKerry%2BA.%2BNaish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3275-8778" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kerry A. Naish</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkodama@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mkodama@uw.edu</a>
<a href="mailto:knaish@uw.edu" class="" data-icon-position="" data-hide-link-title="0">knaish@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1717" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459661" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459661/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1717.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459661" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459661/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1717.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459661" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459661/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1717.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459661" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459661/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1717.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459661" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459661/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Whole genome duplication has been implicated in evolutionary innovation and rapid diversification. In salmonid fishes, however, whole genome duplication significantly pre-dates major transitions across the family, and re-diploidization has been a gradual process between genomes that have remained essentially collinear. Nevertheless, pairs of duplicated chromosome arms have diverged at different rates from each other, suggesting that the retention of duplicated regions through occasional pairing between homeologous chromosomes may have played an evolutionary role across species pairs. Extensive chromosomal arm rearrangements have been a key mechanism involved in re-dipliodization of the salmonid genome; therefore, we investigated their influence on degree of differentiation between homeologs across salmon species. We derived a linkage map for coho salmon and performed comparative mapping across syntenic arms within the genus <em>Oncorhynchus</em>, and with the genus <em>Salmo</em>, to determine the phylogenetic relationship between chromosome arrangements and the retention of undifferentiated duplicated regions. A 6596.7 cM female coho salmon map, comprising 30 linkage groups with 7415 and 1266 nonduplicated and duplicated loci, respectively, revealed uneven distribution of duplicated loci along and between chromosome arms. These duplicated regions were conserved across syntenic arms across <em>Oncorhynchus</em> species and were identified in metacentric chromosomes likely formed ancestrally to the divergence of <em>Oncorhynchus</em> from <em>Salmo</em>. These findings support previous studies in which observed pairings involved at least one metacentric chromosome. Re-diploidization in salmon may have been prevented or retarded by the formation of metacentric chromosomes after the whole genome duplication event and may explain lineage-specific innovations in salmon species if functional genes are found in these regions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromsome-rearrangements" class="hw-term hw-article-keyword hw-article-keyword-chromsome-rearrangements" rel="nofollow">chromsome rearrangements</a></li><li class="kwd"><a href="/keyword/comparative-genome-mapping" class="hw-term hw-article-keyword hw-article-keyword-comparative-genome-mapping" rel="nofollow">comparative genome mapping</a></li><li class="kwd"><a href="/keyword/rad-sequencing" class="hw-term hw-article-keyword hw-article-keyword-rad-sequencing" rel="nofollow">RAD sequencing</a></li><li class="kwd"><a href="/keyword/salmon" class="hw-term hw-article-keyword hw-article-keyword-salmon" rel="nofollow">salmon</a></li><li class="kwd"><a href="/keyword/whole-genome-duplication-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-duplication" rel="nofollow">whole genome duplication</a></li></ul><p id="p-4">Whole genome duplication (WGD) is a mutational mechanism that can serve as a primary driver of evolutionary novelty (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ohno 1970</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zhang 2003</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crow and Wagner 2006</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lynch 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Edger and Pires 2009</a>). Changes in ploidy levels after WGD can lead to dramatic alterations at the cellular and phenotypic level (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mayfield-Jones <em>et al.</em> 2013</a>) and provide additional genetic variation for mutation, drift, and selection to act on. These evolutionary processes can result in new adaptations and species diversification (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Van De Peer <em>et al.</em> 2009</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Storz <em>et al.</em> 2013</a>). Genome sequencing projects are increasingly revealing that WGD is widespread in many key lineages, such as flowering plants and vertebrates, and represents an ongoing phenomenon in many species (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Otto and Whitton 2000</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Van De Peer <em>et al.</em> 2009</a>). Understanding the processes governing the return to a diploid mode—diploidization—by comparing the genomes of species descended from a WGD event can provide insights into the event’s role in evolutionary innovation and persistence of duplicated regions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jaillon <em>et al.</em> 2009</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Mayfield-Jones <em>et al.</em> 2013</a>).</p><p id="p-5">The stabilization of the duplicated genome through diploidization can be achieved by rearrangements (such as translocations, fissions, fusions, and transpositions), gene loss, and sequence deletion and divergence (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hufton and Panopoulou 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schubert and Lysak 2011</a>). These processes tend to reduce the similarity of the duplicated ohnologs (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wolfe 2001</a>), and the homeologous chromosomes resulting from WGD, but the exact mechanisms vary across lineages (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hufton and Panopoulou 2009</a>). Whole genome duplication has been frequently implicated in evolutionary innovation in eukaryotic genomes of paleopolyploids (ancient polyploids) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ohno 1999</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lynch and Conery 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jaillon <em>et al.</em> 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cañestro <em>et al.</em> 2013</a>), but evidence in plants suggests that the rate of diversification and extinction of neopolyploids can be lower than that of related diploid lineages (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mayrose <em>et al.</em> 2011</a>). Increasing the number of studies on mesopolyploids—organisms in the intermediate process of diploidization (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Mayfield-Jones <em>et al.</em> 2013</a>)—will provide a clearer understanding of contribution of WGD events to evolutionary innovation.</p><p id="p-6">Salmonid fishes are descended from a whole genome duplication event in an autotetraploid ancestor (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>), distinct from the second round of duplication (2R) that occurred basal to the vertebrate tree and the third round (3R) early in the evolution of the teleosts 225 to 333 million years ago (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hurley <em>et al.</em> 2007</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Postlethwait 2007</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Santini <em>et al.</em> 2009</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Near <em>et al.</em> 2012</a>). This fourth round (4R) of duplication was recently estimated as occurring 88–103 million years ago (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Macqueen and Johnston 2014</a>; see also <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crête-Lafrenière <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexandrou <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berthelot <em>et al.</em> 2014</a>). Although the genomes of these species are returning to a stable diploid state through chromosomal rearrangements and divergence of homeologous chromosomes, evidence of tetrasomic inheritance in males and extensive rearrangements among chromosomes has shown that restoration of diploidy is not yet complete (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allendorf and Danzmann 1997</a>). Comparative genome sequencing between ohnologs in rainbow trout has revealed extensive collinearity between the duplicated chromosomes, characterized by loss of approximately half the protein-coding regions through pseudogenization but retention of most of the duplicated miRNA genes (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Berthelot <em>et al.</em> 2014</a>).</p><p id="p-7">The role of the WGD event in salmonid trait innovation and diversification is unclear. Recent evidence based on molecular clock estimates suggest that duplication is unlinked to a major transition in life history, anadromy (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexandrou <em>et al.</em> 2013</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Macqueen and Johnston 2014</a>), and preceded rapid species diversification by several million years (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Macqueen and Johnston 2014</a>). Rather, both transitions appear to correspond with climate cooling (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Macqueen and Johnston 2014</a>). Re-diploidization has been subsequently characterized as a gradual process unlinked to significant genome rearrangements (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Berthelot <em>et al.</em> 2014</a>). However, it has also been argued that the duplication event might have provided the raw material for evolution to act on, and that differential divergence of duplicated regions might have promoted speciation at varying time points (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Macqueen and Johnston 2014</a>). Large-scale genome characterization in the salmonids is increasingly revealing the location of genes or regions that may have played a role in adaptation and diversification (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bourret <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Larson <em>et al.</em> 2013</a>). Therefore, it is important to combine these studies with an understanding of the mechanism and timing of divergence between homeologous chromosome arms across salmon lineages so that it will be possible to understand how duplication played a role in evolution of salmon, and whether key genomic regions might explain innovation across a subset of species.</p><p id="p-8">It has been known for some time that one of the key mechanisms for diploidization within the subfamilies Coregoninae and Salmoninae (which includes <em>Salmo</em>, <em>Salvelinu</em>s, and <em>Oncorhynchus</em>) has occurred through Robertsonian rearrangements of whole chromosome arms (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ohno 1999</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Phillips and Ráb 2001</a>). Most chromosome arms are syntenic between Salmoninae species, and the combined efforts of genome mapping and karyotyping have permitted alignment of chromosome arms among several species within this subfamily (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Phillips <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lubieniecki <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Timusk <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guyomard <em>et al.</em> 2012</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Phillips <em>et al.</em> 2013</a>). Chromosome arm number is largely conserved (NF = approximately 100) but the numbers of chromosomes vary substantially across species as a result of the Robertsonian rearrangements (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Phillips and Ráb 2001</a>). The exception is Atlantic salmon, with reduced chromosome arm number compared with the other species (NF = 72–74). However, large syntentic blocks within the arms of this species correspond to whole arms in other members of the Salmoninae, making comparative studies feasible across this subfamily as a whole.</p><p id="p-9">Comparative mapping between Chinook salmon (<em>O. tshawytscha</em>) and rainbow trout (<em>O. mykiss</em>) has revealed evidence for the retention of at least eight metacentric chromosomes and four acrocentric chromosomes that are ancestral to species divergence within the genus <em>Oncorhynchus</em> (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Phillips <em>et al.</em> 2013</a>). One of the eight metacentric chromosomes and one of the acrocentric chromosomes are also ancestral to the divergence between <em>Salmo</em> and <em>Oncorhynchus</em>. There is also further evidence for another ancestral metacentric and an ancestral acrocentric chromosome, but these have undergone subsequent rearrangements within at least one descendant species (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ostberg <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). High-density linkage maps have revealed that recently diverged or undifferentiated duplicated loci are not uniformly distributed among chromosomes (Atlantic salmon, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>; Chinook salmon, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>), and the biased distribution of duplicated loci along chromosome arms provides evidence that pairs of homeologous arms have diverged at different rates from each other (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). This finding confirms observations from previous studies conducted with lower marker densities (for example, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Guyomard <em>et al.</em> 2012</a>). Intriguingly, homeologous pairings have been observed to include at least one metacentric chromosome (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Wright <em>et al.</em> 1983</a>), and duplicated markers map to such chromosomes (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>), supporting the view that metacentric chromosomes play an important role in homeologous pairing (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Phillips <em>et al.</em> 2009</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). These observations raise the interesting possibility that the evolutionary timing of metacentric chromosome formation during re-diploidization after the WGD event might influence the retention, and hence the evolutionary role, of duplicated regions across species. Therefore, by comparing chromosomal arrangements and distribution of duplicated regions across salmon species, we aim to provide a contextual framework for the further investigation of loci involved in species diversification.</p><p id="p-10">The development of a high-density linkage map for a less-described salmon species will contribute further information regarding chromosome rearrangements that have already been defined in several salmon species and enhance our understanding of the timing of these arrangements in a phylogenetic context. Examining the distribution of duplicated regions across individual chromosome arms in a second species—beyond Chinook salmon (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>)—will also facilitate an understanding of the relationship between timing of metacentric chromosome formation and diversification between homeologs. Coho salmon (<em>Oncorhynchus kisutch</em>) is a species whose genome has not been extensively described to date. A low-density linkage map of coho salmon has been constructed using microsatellites (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McClelland and Naish 2008</a>), but this map is not sufficiently resolved to study the consequences of WGD because there is a low number of duplicated loci mapped. A high-density map in this species is feasible, given recently emerged sequencing technologies (for example, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baird <em>et al.</em> 2008</a>). Coho and Chinook salmon are sister species (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Crête-Lafrenière <em>et al.</em> 2012</a>); therefore, comparative mapping across coho and Chinook salmon, as well as more divergent species in the genus <em>Oncorhynchus</em> and <em>Salmo</em>, will help validate the hypotheses and provide more robust evidence on the process of chromosomal evolution following WGD.</p><p id="p-11">The aim of our research is to determine the relationship between chromosome arrangements and the retention of recently diverged or undifferentiated duplicated regions by deriving a linkage map for coho salmon and comparing this map with those of Chinook salmon, rainbow trout, and Atlantic salmon. We therefore constructed high-density linkage maps for coho salmon using restriction site–associated DNA (RAD) sequencing (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Baird <em>et al.</em> 2008</a>). By achieving this objective, we also produced a reference database of RAD markers that can be used for alignment of sequences generated in future work, and described in detail the properties of the coho linkage map. Coho salmon chromosome arms were identified by comparative mapping with Chinook salmon using markers in common between the species, and whole chromosome arm homologies were described across species to improve our current understanding of chromosome arm rearrangement within the genera <em>Oncorhynchus</em> and <em>Salmo</em>. Linkage groups representing homeologous chromosome arms in coho salmon were discovered using duplicated markers, and regions of duplicated markers were compared across species to determine the extent to which these regions were conserved across lineages. By identifying genomic regions that are in the process of diploidization and linking these regions to chromosomal rearrangements, we aim to provide the basis for determining the role of duplication in maintaining ongoing polymorphisms and explaining processes of diversification across Salmoninae species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Justification and description of sample collection and experimental crosses</h3><p id="p-12">A two-step approach was used to develop genomic resources and construct linkage maps. First, RAD sequences from individuals sampled from multiple populations were used to construct a reference database for aligning loci across mapping families. This reference database was screened for errors, duplicated loci, and repeat regions following approaches described by <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> (2014)</a>, and loci were subsequently named to ensure consistency across mapping families. Second, specific cross types were used to perform the mapping: gynogenetic haploid crosses were used to map both duplicated and nonduplicated loci, and diploid crosses were used to construct sex-specific maps.</p><p id="p-13">The reference database of RAD loci was constructed using sequences from 583 individuals representing four populations in the Pacific Northwest of the United States and Canada: (1) the Washington Department of Fish and Wildlife’s (WDFW) Wallace River Hatchery (47°87′N, 121°71′W); (2) the Domsea broodstock population, which originated in 1973 and 1974 from Wallace River Hatchery; (3) Bingham Creek (47°15′N, 123°40′W), a tributary to the Satsop River in the Southwest Washington Coast/Lower Columbia ESU; and (4) Chehalis River located in British Columbia, Canada (49°29′N, 121°94′W).</p><p id="p-14">An initial framework map was constructed using two haploid crosses (haploid family 1 and 2) comprising 64 and 62 individuals, respectively. These types of crosses have the advantage of identifying duplicated loci, because these loci will appear as heterozygotes in the offspring if they are polymorphic, while nonduplicated loci will be homozygous. Haploid families were created at the University of Washington hatchery facility (47°65′N, 122°31′W), following the protocol of <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Thorgaard <em>et al.</em> (1983)</a>. Embryos were collected before hatching and preserved in 100% ethanol.</p><p id="p-15">Sex-specific maps were created using two F3 outbred diploid crosses and one outbred diploid cross. Specifically, F3 diploid crosses were created from a cultured line originally derived from an outbred cross between two populations in Washington State (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McClelland and Naish 2010</a>). F0 males were collected from Bingham Creek in Southwest Washington (47°15′N, 123°40′W). F0 females were obtained from the Domsea broodstock farm. Two F3 crosses were established in December 2010 by mating two F2 full-sibs to create one family, and two F2 half-sibs to create the other. The two families comprised 55 and 67 offspring, respectively, (diploid family 1 and 2). An additional diploid outbred cross was created from an aquaculture population using coho salmon derived from the Chehalis River located in British Columbia, Canada (49°29′N, 121°94′W). Specifically, cultured individuals were repeatedly backcrossed with wild individuals from the Chehalis River for six generations, and diploid crosses were created in January 2011. One diploid family from these crosses comprising 99 individuals was used for further analyses (diploid family 3).</p></div><div id="sec-3" class="subsection"><h3>DNA extraction, sequencing, and amplification of sex-linked markers</h3><p id="p-16">Genomic DNA from the sampled individuals was extracted using the DNeasy extraction kit (QIAGEN, Valencia, CA) following the manufacturer’s procedures. The DNA was digested with <em>Sbf</em>I, and a 6-nucleotide barcode was added to each sample for individual identification following protocols described by <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Baird <em>et al.</em> (2008)</a>. Between 24 and 36 individuals were pooled in a single library and sequenced with 100-bp single-read lengths using the Illumina HiSequation 2000 sequencer. The sequences were separated by individual using PROCESS_RADTAGS implemented in STACKS (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2013</a>). Because the quality score of sequences decreased beyond 74 nucleotides, sequences were trimmed to 74 nucleotides to remove low-quality sequences. A locus was defined as a 74-nucleotide RAD sequence for the purpose of this study.</p><p id="p-17">Genetic sex was determined in the two diploid families (diploid family 1 and 2) using a Y-linked growth hormone pseudogene (GH5 and GH6) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Devlin <em>et al.</em> 2001</a>) and sex-determining gene, sdY (sdY E2S1 and sdY E2AS4) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Yano <em>et al.</em> 2012</a>). Polymerase chain reactions were performed for each set of primers using a QIAGEN Multiplex PCR kit. Specifically, reaction mixtures consisted of 10–200 ng genomic DNA, 1× QIAGEN Multiplex PCR Master Mix, 0.25 μM of GH5 and GH6, or 0.4 μM of sdY E2S1 and sdY E2AS4, comprising a total volume of 10 μl. Cycling conditions consisted of a 15-min initial activation step at 95°, 30 cycles of 30-s denaturing step at 94°, 90-s annealing step at 60°, a 60-s extension step at 72°, and a 10-min final extension step at 72°.</p></div><div id="sec-4" class="subsection"><h3>Reference database of RAD loci</h3><p id="p-18">RAD loci that are found within repeat regions, and loci containing repeat units, can confound the identification of unique loci. Therefore, a reference sequence database comprising a set of pre-screened RAD loci was first created from the survey of four populations following bioinformatic procedures fully described by <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> (2014)</a>. This database served as a resource for aligning loci across studies. In brief, sequences from all 583 individuals sampled across the four populations described previously were extracted using STACKS 0.9995 (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em> 2011</a>). Both monomorphic and polymorphic loci that were sequenced with a depth greater than 5× in more than 496 individuals (85%) were retained in a temporary database and used for further screening.</p><p id="p-19">Loci in the temporary database that corresponded to repeat regions and loci containing repeat units were removed using two alignment-based strategies following the protocol of <a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> (2014)</a>. First, loci in the temporary database were aligned against themselves using BOWTIE (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Langmead <em>et al.</em> 2009</a>) by allowing a maximum of three nucleotide mismatches per locus. A locus that aligned to several loci, or a locus that did not align to itself, was removed from the temporary database. Then, a BLAST search (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Altschul <em>et al.</em> 1990</a>) of the temporary database was conducted against itself. Loci that did not return a match, or loci where the best matches were not themselves, were removed from the temporary database.</p><p id="p-20">Using the updated temporary database of RAD loci, polymorphic duplicated loci were identified based on two haploid families. First, sequences from these haploid families were aligned to the temporary database using BOWTIE, allowing a maximum of three nucleotide mismatches per locus. Sequences from the haploid individuals that aligned to more than one locus in the database could not be confidently relied on in further analyses; they were thus identified as “black-listed” loci and removed from the temporary database. Subsequently, polymorphic loci sequenced with a depth greater than 10× per haploid individual were identified using STACKS and retained for further screening. Among these polymorphic loci, a locus was identified as being putatively duplicated when more than one haploid offspring in a family was heterozygous at this particular locus (<a id="xref-ref-8-10" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). A final reference database comprising named duplicated and nonduplicated loci, as well as loci removed from the alignment-based screening steps and “black-listed” loci, was created.</p></div><div id="sec-5" class="subsection"><h3>Genotyping of individuals in map crosses</h3><p id="p-21">Haploid individuals were genotyped at both nonduplicated and duplicated loci. Sequences from all haploid individuals were aligned to the nonduplicated and duplicated loci from the final reference database using BOWTIE by allowing a maximum of three nucleotide mismatches per locus. In haploids, we have shown reliable identification of single loci that have up to three SNPs; we have confirmed this result with genome mapping. To remain consistent, we used up to three mismatches so that we could differentiate between nonduplicated loci and duplicated loci. Both this study and a previous one (<a id="xref-ref-8-11" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>) have shown that very reliable linkage results can be obtained in haploids using these criteria. Polymorphic loci sequenced with a depth greater than 10× per individual were identified using STACKS. Both nonduplicated and duplicated markers in the haploid families were used for mapping, described below and following the protocol of <a id="xref-ref-8-12" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> (2014)</a>. Polymorphic duplicated loci were mapped when one of the paralogs was polymorphic (OPP: one paralog polymorphic, parental genotypes aa and ab or aa and bc) or when both paralogs were polymorphic (BPP: parental genotypes ab and ac or ab and cd; see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> in <a id="xref-ref-8-13" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464888/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464888/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of the Coho salmon consensus linkage map constructed with haploid and diploid female parents and comparison with chromosome arms of Chinook salmon, rainbow trout, and Atlantic salmon</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">Diploid individuals were only genotyped at nonduplicated loci. Sequences from all diploid individuals were aligned to the nonduplicated loci identified in the final reference database using BOWTIE by allowing a maximum of three nucleotide mismatches per locus. Subsequently, polymorphic loci were identified in each diploid family using STACKS, and genotypes at these loci were determined when alleles were sequenced with a depth greater than 10× per individual.</p><p id="p-29">STACKS uses a maximum likelihood statistical model to identify sequence polymorphisms and determine individual genotypes (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">2013</a>). This approach can be biased toward heterozygous genotypes when sequence depths differ between the two alleles. To correct this bias against heterozygous genotypes, genotypes were corrected after running STACKS with the Python script developed by <a id="xref-ref-8-16" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> (2014)</a>. Specifically, individuals were determined as heterozygotes at a locus if both alleles had a depth of more than two and the total read depth was 10× or greater.</p></div><div id="sec-6" class="subsection"><h3>Linkage mapping</h3><p id="p-30">Linkage maps in all haploid and diploid families were constructed using software for genetic mapping, ONEMAP 2.0-3 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Margarido <em>et al.</em> 2007</a>), implemented in R version 3.0.2 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">R Development Core Team 2013</a>). Because coho salmon have 30 chromosome pairs (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Phillips and Ráb 2001</a>), each mapping family was expected to have at least 30 linkage groups. Linkage groups were named “Co,” following the convention used in mapping studies in salmonids; this practice uses abbreviated common names for groups that are not yet anchored to chromosomes (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>). RAD loci with 20% or less missing values among individuals within a family were used for linkage analyses, and these loci were assigned to linkage groups in each family separately using a minimum log of odd ratio (LOD) score of 4.0 and a maximum recombination fraction of 0.25. The LOD score was subsequently increased by 1.0 until the number of linkage groups reached 30 or higher. An integrated haploid map was first constructed from the two haploid families using MERGEMAP (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wu <em>et al.</em> 2011</a>) because genotypes at duplicated loci were only determined in these families. This integrated haploid map was later used to examine the distribution of duplicated loci across all linkage groups and identify linkage groups involved in recent or ongoing homeologous pairing.</p><p id="p-31">Recombination rates in male salmonids tend to be lower than those observed for females (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Ostberg <em>et al.</em> 2013</a>), but these differences tend to decrease with high marker density and genome coverage (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rexroad <em>et al.</em> 2008</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>). We used the female meiosis from the three diploid families to estimate marker order in these crosses. The data from all haploid and diploid female parents were then combined to calculate an integrated female haploid/diploid map using MERGEMAP.</p><p id="p-32">Ordering markers in the diploid male map was computationally difficult, potentially due to reduced recombination and occasional tetrasomic inheritance in males (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Allendorf and Danzmann 1997</a>). Therefore, information from the integrated female map constructed with haploid and diploid mothers was used to infer the order and map the male meiosis in the three diploid families (diploid family 1, 2, 3). Polymorphic loci in common between the male parents and the integrated female map, as well as the Y-linked growth hormone pseudogene and sex-determining gene, sdY, were grouped using a log of odd ratio (LOD) score of 4.0 and a maximum recombination fraction of 0.25 using ONEMAP. The LOD score was subsequently increased by 1.0 until the number of linkage groups reached 30 or higher. Grouped loci were then ordered based on the known order on the integrated female map using the <em>make.seq</em> and <em>map</em> functions implemented in ONEMAP. The position of a Y-linked growth hormone pseudogene and sex-determining gene, sdY, on the male map was estimated in the two diploid families (diploid family 1 and 2) using the <em>try.seq</em> and <em>map</em> functions implemented in ONEMAP. The data from all diploid male parents were then combined to calculate an integrated male map using MERGEMAP.</p></div><div id="sec-7" class="subsection"><h3>Comparative mapping with Chinook salmon and comparison with other salmonid species</h3><p id="p-33">The reference database for coho salmon containing duplicated and nonduplicated RAD loci was aligned to the 54,937 filtered RAD loci identified in Chinook salmon (<a id="xref-ref-8-17" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>) using BOWTIE, allowing no more than three nucleotide mismatches per locus. Homologies between Chinook and coho salmon were determined by examining the chromosomal arm locations of shared loci between the two species. Putative centromere positions on coho linkage groups were estimated based on markers mapped in the gynogenetic diploid families in Chinook salmon (<a id="xref-ref-8-18" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). The order of mapped loci between the Chinook and coho salmon map was compared to determine if marker orders for chromosomes or chromosomal arms between the species were conserved. Finally, homologies identified between Chinook and coho salmon were used to infer homologies across coho salmon, rainbow trout, and Atlantic salmon using molecular markers in common between published maps (<a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Phillips <em>et al.</em> 2009</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Miller <em>et al.</em> 2012</a>; <a id="xref-ref-8-19" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>).</p></div><div id="sec-8" class="subsection"><h3>Homeologous relationships and the distribution of duplicated loci across genomes</h3><p id="p-34">As we pointed out, two categories of duplicated loci were identified in this study, where one of the paralogs was polymorphic (OPP) or both paralogs were polymorphic (BPP). Duplicated loci with both paralogs polymorphic (BPP) were used to infer homeologous linkage groups because both paralogs could be mapped. The positions of duplicated loci were subsequently examined on the integrated haploid map to determine whether there was a bias in the distribution of these loci across linkage groups. A kernel smoothing approach using a sliding window of 2 cM was used to determine whether there was a regional bias in distribution of these loci for each linkage group following methods described by <a id="xref-ref-8-20" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> (2014)</a>. Homeologous relationships detected in coho salmon were also compared with those identified in Chinook and Atlantic salmon (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-8-21" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Reference database of RAD loci</h3><p id="p-35">A reference database comprising a unique set of RAD loci was created for the purpose of sequence alignment and identification of polymorphisms across individuals. A total of 70,037 loci were sequenced with a depth greater than 5× per individual in at least 496 individuals. These loci formed the temporary reference database of RAD loci, and they were retained for further screening. Sequence alignment using BOWTIE showed that 4075 loci did not align uniquely to themselves and likely corresponded with repeat regions; therefore, these loci were removed from the temporary database. After performing the BLAST search of the temporary reference database against itself, 2085 loci did not return a match or the best match score was not the locus itself. It was possible that these loci contained repeat sequences; therefore, these loci were also removed from the temporary reference database. Sequences from the haploid individuals were aligned to the reference database using BOWTIE; 3706 loci from the haploid individuals aligned to several other loci, and these loci were thus black-listed and removed from the reference database. Additionally, 7235 loci were identified as polymorphic duplicated loci in the haploid families. The final reference database comprising 52,936 nonduplicated loci and 7235 duplicated loci, as well as the 9866 loci that were removed by screening, are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/012294SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-11" class="subsection"><h3>Linkage mapping</h3><p id="p-36">An initial framework map was constructed with two haploid families. Haploid family 1 and 2 had 3976 and 4048 biallelic polymorphic RAD loci, respectively, comprising a total of 6652 unique RAD loci. Among these loci, a mixture of duplicated and nonduplicated loci (3897 loci in haploid family 1; 3996 loci in haploid family 2) were successfully assigned to 30 linkage groups with a LOD score of 5.0 to 7.0. The total map length for the haploid family 1 and 2 was 3040.1 cM and 3185.5 cM, respectively. The integrated haploid map had 5377 nonduplicated markers and 1266 duplicated markers with a total map length of 3602.6 cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS2.zip">File S2</a>).</p><p id="p-37">Linkage analyses were conducted in the diploid families following the construction of the integrated haploid map. Diploid family 1, 2, and 3 had 1360, 1176, and 1931 biallelic nonduplicated loci that were polymorphic in each female parent, respectively. Among these loci, a set of loci (1214 loci in diploid family 1; 1138 loci in diploid family 2; 1765 loci in diploid family 3) were successfully assigned to 30 linkage groups with a LOD score of 4.0 to 8.0. The total map length for the diploid family 1, 2, and 3 was 3714.1 cM, 3068.9 cM, and 5047.2 cM, respectively. Although the diploid family 3 had the largest total map length, it also had the highest number of markers mapped. Because more recombination events are captured with more markers (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Liu 1998</a>), it is not surprising that the diploid family 3 had the largest total map length. Finally, data from the haploid parents and diploid female parents were combined; an integrated haploid/diploid female map measured 6596.7 cM, and it comprised 7415 nonduplicated markers and 1266 duplicated markers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS2.zip">File S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of 30 consensus linkage groups in haploid and diploid female coho salmon. Co01 to Co20 are metacentric, and Co21 to Co30 are acrocentric, inferred from comparative mapping. The size of linkage groups ranges from 127.18 to 295.21 cM (Kosambi), and each line corresponds to the location of one or more markers. The putative location of the centromere, estimated by comparative mapping with Chinook salmon, is represented in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628395614" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Graphical representation of 30 consensus linkage groups in haploid and diploid female coho salmon. Co01 to Co20 are metacentric, and Co21 to Co30 are acrocentric, inferred from comparative mapping. The size of linkage groups ranges from 127.18 to 295.21 cM (Kosambi), and each line corresponds to the location of one or more markers. The putative location of the centromere, estimated by comparative mapping with Chinook salmon, is represented in red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1717/F1.medium.gif" width="440" height="167"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1717/F1.medium.gif" width="440" height="167"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464862" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-38" class="first-child">Graphical representation of 30 consensus linkage groups in haploid and diploid female coho salmon. Co01 to Co20 are metacentric, and Co21 to Co30 are acrocentric, inferred from comparative mapping. The size of linkage groups ranges from 127.18 to 295.21 cM (Kosambi), and each line corresponds to the location of one or more markers. The putative location of the centromere, estimated by comparative mapping with Chinook salmon, is represented in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The male meioses were mapped using linkage analyses in the diploid families. Among the 8681 loci placed on the integrated haploid/diploid female map, diploid family 1, 2, and 3 had 846, 814, and 879 polymorphic loci in common for each male parent. Among these loci, a set of loci (792 loci in diploid family 1; 790 loci in diploid family 2; 851 loci in diploid family 3), as well as the Y-linked growth hormone pseudogene and sex-determining gene, sdY, were successfully assigned to 30 linkage groups with a LOD score of 4.0 to 7.0. Both the growth hormone pseudogene and sdY mapped to the telomeric region of the linkage group, Co30. All linkage groups were successfully merged, except for Co22, which was split into two linkage groups (Co22_1 and Co22_2) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS2.zip">File S2</a>). The number of markers in common between the integrated male and female maps varied for each linkage group, ranging from 25 to 106 markers per linkage group (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS3.xlsx">File S3</a>). The male map had a total map length of 4141.76 cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS3.xlsx">File S3</a>).</p><p id="p-40">The comparison between the male and female linkage groups reflected different recombination patterns between the sexes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS4.pdf">File S4</a>). Although telomeres were not mapped in males due to a lack of duplicated markers, many male linkage groups were expanded in size toward the terminal regions relative to the female, as seen by the increased distance in these regions reflecting more recombination events. Such patterns were particularly prominent for several linkage groups (Co02, Co04, Co05, Co08–Co10, Co13–Co15, Co17–Co19, Co21–Co29). Although qualitative, there was also evidence of suppressed recombination around the region containing the centromere in the male map compared with female integrated map for all linkage groups. The male map had reduced distance in these regions compared to the female map.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of map distances between common markers mapped in male and female coho salmon. Linkage groups Co04 (A), Co17 (B), Co18 (C), and Co26 (D) are given as examples. The putative region containing the centromere is represented by the cross-hatched area. All linkage groups are presented in File S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628395614" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of map distances between common markers mapped in male and female coho salmon. Linkage groups Co04 (A), Co17 (B), Co18 (C), and Co26 (D) are given as examples. The putative region containing the centromere is represented by the cross-hatched area. All linkage groups are presented in File S4.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1717/F2.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/9/1717/F2.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464983" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Comparison of map distances between common markers mapped in male and female coho salmon. Linkage groups Co04 (A), Co17 (B), Co18 (C), and Co26 (D) are given as examples. The putative region containing the centromere is represented by the cross-hatched area. All linkage groups are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS4.pdf">File S4</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Comparative mapping with Chinook salmon and comparisons with other salmonid species</h3><p id="p-42">We mapped 664 RAD loci in coho salmon that had been previously placed on the Chinook salmon map, which permitted the identification of homologous chromosomal arms between the two species (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). On the basis of this comparison, we also identified the putative locations of centromeres within coho salmon linkage groups (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Two homologous relationships between the species were inferred. An arm of the linkage group Co07a had three markers that mapped to Ots34 in Chinook salmon. However, Co12b had six markers that mapped to Ots34 and three markers that mapped to Ots11p. Ots11p and Ots34 are likely involved in recent or ongoing homeologous pairing in Chinook salmon (<a id="xref-ref-8-22" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>); therefore, it was not surprising that loci on Ots34 mapped to both homologous arms in coho salmon. In this case, we assumed that Co07a was homologous to Ots11p, and Co12b was homologous to Ots34 for reasons given in the <em>Discussion</em>.</p><p id="p-43">Comparative mapping with Chinook salmon permitted inference of the structure of coho linkage groups. Twenty linkage groups in coho salmon corresponded to putative bi-armed metacentric chromosomes, and 10 linkage groups corresponded to putative uni-armed acrocentric chromosomes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). These inferred structures are in agreement with the known chromosome structures in coho salmon (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Phillips and Ráb 2001</a>). The short (p) arm of an acrocentric chromosome is usually uncharacterized in mapping studies because there are often insufficient markers describing this region (<a id="xref-ref-8-23" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). In this study, we identified the small arm for two putative acrocentric chromosomes (Co22, Co29) through comparative mapping with Chinook salmon.</p><p id="p-44">Comparative mapping between the Chinook and coho salmon maps also provided information on chromosomal arrangements that are shared between the two species. Eighteen chromosomes are conserved between the species (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>); specifically, nine metacentric chromosomes and nine acrocentric chromosomes were conserved between the species. The remaining chromosome structures likely support independent Robertsonian rearrangements that occurred after descent from a common ancestor.</p><p id="p-45">Five metacentric linkage groups in coho salmon (Co10–Co14) consist of one acrocentric chromosome and one arm from a metacentric chromosome in Chinook salmon. Four metacentric linkage groups in coho salmon (Co15–Co18) comprise arms that are found in two separate metacentric chromosome pairs in Chinook salmon. Two metacentric linkage groups (Co19, Co20) comprise two acrocentric chromosome pairs in Chinook salmon. Finally, one acrocentric linkage group (Co30) corresponds to an arm that is a part of a metacentric chromosome pair in Chinook salmon.</p><p id="p-46">The orders of the RAD loci on the Chinook and coho salmon maps were compared across each linkage group or for each chromosome arm to determine whether any chromosomal inversions occurred after divergence between the species. There was a strong linear relationship among mapped loci for all the linkage groups or arms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS5.pdf">File S5</a>), suggesting that marker orders were conserved for all chromosomes or chromosomal arms. Such analyses provide additional evidence for the occurrence of centrometric inversion in Omy20 in rainbow trout after divergence between rainbow trout and Chinook/coho salmon, and this chromosomal inversion may be exclusive to rainbow trout (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Ostberg <em>et al.</em> 2013</a>; <a id="xref-ref-8-24" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>).</p><p id="p-47">The homologies we observed between Chinook and coho salmon permitted alignment of coho linkage groups to those of rainbow trout and the Atlantic salmon, and the results are summarized in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>. Three acrocentric and eight metacentric chromosomes were conserved among coho salmon, Chinook salmon, and rainbow trout. Comparison between the <em>Oncorhynchus</em> species and the Atlantic salmon revealed that one metacentric chromosome and one acrocentric chromosome were conserved across all compared species.</p></div><div id="sec-13" class="subsection"><h3>Homeologous relationships and the distribution of duplicated loci across linkage groups</h3><p id="p-48">The identification of linkage groups involved in homeologous pairing, as well as the localization of duplicated loci across individual linkage groups, was examined using the integrated haploid female map. A total of 1266 duplicated loci (1066 OPP and 200 BPP) were placed on this map. These loci were not distributed evenly among the linkage groups (χ<sup>2</sup> test for uniform distribution across linkage groups, after correction for the number of markers per linkage group: p-value ∼0, <em>df</em> = 29); 87.0% of the duplicated loci were located on 16 linkage groups (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). There was a bias in distribution of these loci along the 16 linkage groups; duplicated loci were mostly found in the distal regions of all 16 linkage groups (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Homeologies were identified between putative chromosome arms using marker pairs in which both paralogs were polymorphic (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). All eight homeologous arm pairs with a high retention of duplicated loci detected in coho salmon were also observed in Chinook salmon, and four homeologous arm pairs were conserved in Atlantic salmon (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). All of these chromosome arms, likely involved in recent or ongoing homeologous pairing, involved at least one metacentric chromosome ancestral to the divergence between Pacific salmon species.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of markers and distribution of duplicated markers across each coho salmon linkage group. Nonduplicated loci are represented in light gray. Duplicated loci are represented in dark gray (loci with only one paralog polymorphic) or black (both paralogs polymorphic)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628395614" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of markers and distribution of duplicated markers across each coho salmon linkage group. Nonduplicated loci are represented in light gray. Duplicated loci are represented in dark gray (loci with only one paralog polymorphic) or black (both paralogs polymorphic).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1717/F3.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/9/1717/F3.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464857" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-49" class="first-child">Number of markers and distribution of duplicated markers across each coho salmon linkage group. Nonduplicated loci are represented in light gray. Duplicated loci are represented in dark gray (loci with only one paralog polymorphic) or black (both paralogs polymorphic).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of duplicated and nonduplicated loci along the 16 linkage groups with a high proportion of duplicated loci. Nonduplicated loci are represented in light gray. Duplicated loci are represented in dark gray (loci with one paralog polymorphic) or in black (loci with both paralogs polymorphic). The putative region containing the centromere is represented by the cross-hatched area." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628395614" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of duplicated and nonduplicated loci along the 16 linkage groups with a high proportion of duplicated loci. Nonduplicated loci are represented in light gray. Duplicated loci are represented in dark gray (loci with one paralog polymorphic) or in black (loci with both paralogs polymorphic). The putative region containing the centromere is represented by the cross-hatched area.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1717/F4.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/9/1717/F4.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464842" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-50" class="first-child">Distribution of duplicated and nonduplicated loci along the 16 linkage groups with a high proportion of duplicated loci. Nonduplicated loci are represented in light gray. Duplicated loci are represented in dark gray (loci with one paralog polymorphic) or in black (loci with both paralogs polymorphic). The putative region containing the centromere is represented by the cross-hatched area.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464988/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464988/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Homeologous chromosome arm pairs identified in coho salmon and the number of marker pairs supporting the homeologous relationship</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-55">Here, we aimed to examine the relationship between chromosomal evolution and retention of duplicated regions within the genus <em>Oncorhynchus</em>, and between this genus and <em>Salmo</em>, by deriving a linkage map for coho salmon and comparing this map with that of Chinook salmon, rainbow trout, and Atlantic salmon. Thirty linkage groups including 20 putative metacentric and 10 putative acrocentric chromosomes were described across two haploid and three diploid families. Chromosomal rearrangements were identified by comparing homologous arms between coho salmon, Chinook salmon, rainbow trout, and Atlantic salmon. Results confirmed the conservation of at least one metacentric chromosome between <em>Oncorhynchus</em> and <em>Salmo</em> (Co09) and seven metacentric chromosomes across the genus <em>Oncorhynchus</em> (Co01–Co07) (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Phillips <em>et al.</em> 2013</a>; <a id="xref-ref-8-26" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>), and detected a polymorphism in another across coho and Chinook salmon and rainbow trout (Co14 and Co15). Another metacentric chromosome was detected as ancestral to coho and Chinook salmon only (Co08). The placement of 1266 duplicated loci on the consensus haploid map of 6643 markers revealed that these loci were not evenly distributed across all linkage groups, supporting a previous finding in Chinook salmon (<a id="xref-ref-8-27" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>), namely, that homeologous pairs diverged from each other at different rates following the whole genome duplication event. Regions of the genome with polymorphic duplicated markers were found on the same eight pairs of homologous chromosome arms (16 arms in total) across coho and Chinook salmon. Each of the eight pairs of chromosomes likely involved in ongoing or recent homeologous pairing included at least one of the ancestral metacentric chromosomes that are conserved between the two species. The other chromosome arm involved in the pairing may be part of either an acrocentric chromosome or a metacentric chromosome. The data suggest that Robertsonian rearrangements that result in metacentric chromosome formation prior to the diversification of homeologous pairs might partly explain the uneven retention of duplicated regions across the genome, at least within Pacific salmon.</p><p id="p-56">The consensus male map, constructed with three diploid families, was significantly smaller (4141.7 cM) than the consensus female map (6596.7 cM) constructed with two haploid families and three diploid families. There are three main reasons that might explain the reduced map size in the male compared with the female map. First, the difference could simply be a function of more markers being placed on the female map (8681 for the female and 2043 for the male), because map size tends to increase when more markers are added (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Liu 1998</a>). Second, some male linkage groups only represented a portion of those in females; for example, in three metacentric chromosomes (Co07, Co09, Co13) only one arm and a region containing the centromere were mapped, the duplicated markers were not mapped in males. Third, recombination in males is suppressed relative to females, and male maps in salmonids often tend to be smaller. Comparisons between the consensus male map and the consensus (haploid and diploid) female map indicated that recombination in the males was suppressed around the region containing the centromere in some linkage groups, whereas recombination in females seemed suppressed toward telomeric regions.</p><p id="p-57">Accurate identification of chromosome structure in coho salmon relied on aligning homologous chromosome arms with Chinook salmon. In addition, regions containing the centromere were inferred through comparative mapping between coho and Chinook salmon as no gynogenetic diploid families were used for this study to identify the exact location of the centromere. Homology of Co07a and Co12b with Chinook salmon chromosome arms Ots11p and Ots34 was not completely resolved. These arms are homeologous to each other within both species, and markers on Ots34 mapped to both Co07a and Co12b in coho salmon. We inferred that Co07a was homologous to Ots11p, as the arm Co07a is part of a metacentric chromosome that is conserved in Chinook salmon (Ots11) and rainbow trout (Omy19) (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). In fact, <a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> (2013)</a> speculate that these are conserved across the genus because Chinook salmon and rainbow trout are distantly related. In contrast, the metacentric chromosome Co12 is not conserved in Chinook salmon and rainbow trout. We assumed these structures in the subsequent discussion.</p><div id="sec-15" class="subsection"><h3>Coho salmon map coverage, size, and differences in sex-specific recombination</h3><p id="p-58">The coho salmon linkage map constructed in this study has 8681 markers, spanning all predicted 30 linkage groups. This coverage is comparable with recently published maps across a number of salmon species (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Everett <em>et al.</em> 2012</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Miller <em>et al.</em> 2012</a>; <a id="xref-ref-8-28" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gonen <em>et al.</em> 2014</a>). We observed different map sizes in the consensus female maps; the map constructed with combined haploid and diploid families had a size of 6596.7 cM, which is significantly larger than the coho map created with haploids alone (3602.6 cM). There are several reasons that might explain the differences such as nonrandom missing values (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jorgenson <em>et al.</em> 2005</a>), genotyping errors (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hackett and Broadfoot 2003</a>), and numbers of markers mapped. Potential bias against heterozygotes in RAD sequencing (<a id="xref-ref-8-29" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>) may also partly explain the inflated map distances in the haploid/diploid map, especially because the size of the map created with only haploid families in this study was much smaller.</p><p id="p-59">Male salmonids are the heterogametic sex (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>). In this study, both the Y-linked growth hormone pseudogene and sex-determining gene, sdY, mapped to the telomeric region on the acrocentric chromosome, Co30. This finding is in agreement with previous findings (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Phillips <em>et al.</em> 2005</a>). Mapping has shown that the sex chromosome is not conserved across the species, and that a small male-specific region including the sex determining gene has been repeatedly transposed to different chromosomes in different salmon species (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Phillips <em>et al.</em> 2001</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Woram <em>et al.</em> 2003</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Yano <em>et al.</em> 2012</a>) and can be polymorphic within a species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Eisbrenner <em>et al.</em> 2014</a>).</p><p id="p-60">Our results showed that suppressed recombination around the region containing the centromere in males was widely apparent, whereas higher recombination was observed in telomeric regions for some male linkage groups relative to female. The results are in agreement with a number of studies performed on other salmonid species (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Moen <em>et al.</em> 2008</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Rexroad <em>et al.</em> 2008</a>; <a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>). Male recombination rate in telomeric regions of a subset of rainbow trout linkage groups has been shown to be higher than that of females, but lower in centromeric regions (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Sakamoto <em>et al.</em> 2000</a>). Such different recombination patterns might be partly explained by occasional multivalent formation during male meiosis, in which crossovers between homeologous chromosomes are increased in the telomeric regions while crossovers between homologous chromosomes are hindered in the centromeric regions through structural constraints (<a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Sakamoto <em>et al.</em> 2000</a>). In our study, however, suppressed recombination around the centromere and increased recombination in telomeric regions were apparent for most male linkage groups, including the ones not likely involved in homeologous pairing. Some studies have also found notable clustering of markers in centromeric regions for many male linkage groups (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nichols <em>et al.</em> 2003</a>; <a id="xref-ref-34-10" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Miller <em>et al.</em> 2012</a>). In Atlantic salmon, high marker densities were also involved in regions close to the centromere for male linkage groups with a lower frequency of duplicated markers (<a id="xref-ref-34-11" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>). Recombination rates are known to differ between the sexes in a wide range of species (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lenormand and Dutheil 2005</a>); although homeologous chromosome pairing during male meiosis could certainly account for some of the differences, the origin of the sex differences observed in this study still remains unclear.</p></div><div id="sec-16" class="subsection"><h3>Comparative genome mapping</h3><p id="p-61">Comparative mapping provided insights into the process of chromosomal evolution occurring after the whole genome duplication event, and this is the first study that characterizes chromosomal evolution between coho and Chinook salmon. Nine metacentric and nine acrocentric chromosomes appear to be conserved between these two species. Among these conserved chromosomes, one metacentric (Co08 in coho and Ots12 in Chinook) and five acrocentric chromosomes are unique to coho and Chinook salmon, suggesting that the structure of these linkage groups is ancestral to the divergence of these species relative to rainbow trout and Atlantic salmon. The remaining linkage groups are not conserved, reflecting extensive chromosomal rearrangements since coho and Chinook salmon diverged.</p><p id="p-62">Syntenic relationships between the Chinook salmon and rainbow trout maps permitted comparisons across the genus <em>Oncorhynchus</em> (<a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Phillips <em>et al.</em> 2013</a>). There are four acrocentric chromosomes that are conserved across all three <em>Oncorhynchus</em> species (Co26–Co29). Similarly, eight metacentric chromosomes are conserved among all three species (Co01–Co07, Co09). The results support the hypothesis of <a id="xref-ref-48-9" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> (2013)</a> that the arm rearrangements that resulted in these metacentric chromosomes are ancestral to the divergence of the species and could be conserved across the genus <em>Oncorhynchus</em>. There is one interesting extension to these earlier observations. One metacentric chromosome in Chinook salmon, Ots08, sometimes occurs as a metacentric chromosome (Omy25p and q) or as two acrocentric arms (Omy25 and Omy 29) in rainbow trout (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). Here, we found that the homologous arms in coho salmon occur in two separate unrelated metacentric linkage groups (Co14a and Co15a) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). In Atlantic salmon, these two arms are fused together along with a third arm to form a large acrocentric chromosome (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). Robertsonian fusions are common and can also form acrocentric chromosomes; this outcome is likely more frequent in Atlantic salmon than in Pacific salmon (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Phillips and Ráb 2001</a>). Taken together, the configurations of these particular chromosomes suggest they may have undergone recurrent fusions and fissions across species.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree showing the orientation of homologous arms Ssa09qab, Omy25/29, Co14a, Co15a, and Ots08 in Atlantic salmon, rainbow trout, coho salmon, and Chinook salmon, respectively. Chromosomal rearrangements and homeologous relationships conserved across species at phylogenetic nodes A, B, C are summarized in Table 1 and Table 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628395614" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic tree showing the orientation of homologous arms Ssa09qab, Omy25/29, Co14a, Co15a, and Ots08 in Atlantic salmon, rainbow trout, coho salmon, and Chinook salmon, respectively. Chromosomal rearrangements and homeologous relationships conserved across species at phylogenetic nodes A, B, C are summarized in Table 1 and Table 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1717/F5.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/9/1717/F5.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1717/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464924" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-63" class="first-child">Phylogenetic tree showing the orientation of homologous arms Ssa09qab, Omy25/29, Co14a, Co15a, and Ots08 in Atlantic salmon, rainbow trout, coho salmon, and Chinook salmon, respectively. Chromosomal rearrangements and homeologous relationships conserved across species at phylogenetic nodes A, B, C are summarized in <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">One metacentric and one acrocentric chromosome, Co09 and Co26, respectively, appear to be conserved across coho salmon, Chinook salmon, rainbow trout, and Atlantic salmon (<a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Phillips <em>et al.</em> 2009</a>; <a id="xref-ref-8-30" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). Our study supports the previous findings that this metacentric chromosome is likely ancestral to the divergence of the two genera, <em>Salmo</em> and <em>Oncorhynchus</em>. It also appears that this chromosome is conserved in Arctic charr and Brook charr within the genus <em>Salvelinus</em> (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Timusk <em>et al.</em> 2011</a>), both of which share a more recent common ancestor with <em>Oncorhynchus</em>. In addition, previous results provided evidence for the occurrence of centrometric inversion in Omy20 in rainbow trout following divergence between rainbow trout and Chinook/coho salmon, and this chromosomal inversion may be exclusive to rainbow trout (<a id="xref-ref-48-10" class="xref-bibr" href="#ref-48">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Ostberg <em>et al.</em> 2013</a>; <a id="xref-ref-8-31" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). In the current study, marker orders are fully conserved between coho and Chinook salmon chromosome arms, Co25 and Ots25, respectively, further supporting this previous observation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1/FileS5.pdf">File S5</a>).</p></div><div id="sec-17" class="subsection"><h3>Conservation of reduced divergence between homeologous chromosomes across species</h3><p id="p-65">Although our results confirm that the divergence rates of homeologs following the whole genome duplication event have not been uniform (<a id="xref-ref-8-32" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>), the key finding of this study is that the ancestral metacentric chromosomes retain recently diverged duplicates and are the ones likely involved in recent or ongoing homeologous pairing (Co01–Co07, conserved among all three <em>Oncorhynchus</em> species; Co09, conserved among all four species). Such findings suggest that homeologies may be preferentially retained between larger metacentric chromosomes (<a id="xref-ref-59-7" class="xref-bibr" href="#ref-59">Phillips <em>et al.</em> 2009</a>), and the involvement of at least one metacentric chromosome provides the stability required for the formation of multivalents (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wright <em>et al.</em> 1980</a>, <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">1983</a>; <a id="xref-ref-8-33" class="xref-bibr" href="#ref-8">Brieuc <em>et al.</em> 2014</a>). These results support the hypothesis of <a id="xref-ref-59-8" class="xref-bibr" href="#ref-59">Phillips <em>et al.</em> (2009)</a>, which suggested that diploidization of chromosomes not involved in homeologous parings may have occurred in the ancestral salmonid before the divergence between <em>Salmo</em> and <em>Oncorhynchus</em>. We speculate that this process also differed to some extent following the divergence of the two genera. Although the exact distribution of duplicated markers along chromosome arms in Atlantic salmon has not yet been described (<a id="xref-ref-34-12" class="xref-bibr" href="#ref-34">Lien <em>et al.</em> 2011</a>), only four out of eight homeologous pairings appear to share polymorphic duplicated loci between the Atlantic salmon and Chinook and coho salmon grouping (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-66">The implications of our findings for species divergence within the subfamily Salmoninae will become clearer once we gain a greater understanding of the role of duplicated regions in evolution. If the duplicated regions we detected have genes that permit greater flexibility for adaptation by providing the opportunity to acquire additional or novel functions (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Soltis and Soltis 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Koop and Davidson 2008</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Van De Peer <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Feldman <em>et al.</em> 2012</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alexandrou <em>et al.</em> 2013</a>; <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Macqueen and Johnston 2014</a>), then retention of particular duplicated regions within certain lineages may explain their subsequent innovation and diversification. However, the physical formation of metacentric chromosomes may inhibit diploidization because of ongoing recombination; such chromosomes may continue to exhibit tetrasomic inheritance, thus becoming “evolutionary dead ends.” Whole genome sequencing of duplicated chromosome arms in rainbow trout points toward the fact that duplicated protein coding loci have simply become lost through gradual change (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Berthelot <em>et al.</em> 2014</a>); the conserved metacentric chromosomes may continue exhibiting tetrasomic inheritance and prevent functional divergence of protein coding regions. In this study, we provide preliminary evidence that the evolutionary timing of metacentric chromosome formation varied, which might have impacted the rate of diploidization across different lineages. As comparative genome sequencing of salmon species continues, comparing the rates of differentiation along certain chromosome arms between species and identifying the location of genes involved in diversification will provide insights into the role of WGD in salmon evolution. Here, we have identified chromosome arms of interest for further efforts addressing such questions.</p></div></div><div class="section conclusions" id="sec-18"><h2 class="">Conclusion</h2><p id="p-67">Here, we developed an extensive set of genomic resources for coho salmon: a reference database of unique RAD loci, two types of consensus female linkage maps, and a consensus male linkage map. A dense female map constructed in this study permitted alignment of linkage groups in this species with that of Chinook salmon, enabling interspecies comparisons with related salmon species. Syntenic relationships across multiple salmonid species identified in this study provided strong evidence for chromosomal rearrangements and conservation of metacentric and acrocentric chromosomes following the divergence between <em>Salmo</em> and <em>Oncorhynchus</em>. We have also identified linkage groups that recently have been or may be involved in ongoing homeologous pairing in coho salmon. Such pairings were conserved with related Pacific salmon species. Ancestral metacentric chromosomes appear to retain recently diverged duplicated regions and may be involved in ongoing homeologous pairings; such results indicate that diploidization may have been prevented or retarded in these ancestral metacentric chromosomes following the whole genome duplication event. The resources developed here will facilitate genome-wide studies in coho salmon, such as genome scans, QTL mapping, and genome-wide association studies (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Naish and Hard 2008</a>), as well as provide resources for studies concerning ecology and evolution in related salmon species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We thank Isadora Jimenez-Hidalgo for assistance with genomic data processing. Dan Drinan provided thoughtful discussions and helpful suggestions. We also thank Linda Park and Jim Myers for help in setting up the initial crosses and David Rose for maintaining the lines. Finally, we are grateful to two anonymous referees and the Associate Editor for their very helpful comments regarding the first draft of the manuscript. Initial experimental lines were obtained from SweetSpring Salmon/Aquaseed Corporation, suppliers of the Domsea broodstock and we thank Per Heggelund, Greg Hudson and Patty Munsell for providing these lines. Funding for this study was provided by NOAA Fisheries/Federal Columbia River Power System (FCRPS) Biological Opinion Remand Funds (to K.A.N. and J.J.H.), School of Aquatic and Fishery Sciences, and Graduate Opportunities and Minority Achievement Program (GO-MAP, University of Washington) award (to M.K.). The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012294/-/DC1</a></p></li><li class="fn" id="fn-12"><p id="p-69"><em>Communicating editor: D.-J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-05-22"><span class="received-label">Received </span>May 22, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-07-13"><span class="accepted-label">Accepted </span>July 13, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Kodama <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.1" data-doi="10.1016/j.ympev.2013.07.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexandrou</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swartz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oakley</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome duplication and multiple evolutionary origins of complex migratory behavior in Salmonidae.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D69%26rft.spage%253D514%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2013.07.026%26rft_id%253Dinfo%253Apmid%252F23933489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2013.07.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23933489&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1717.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Tetraploidy and the evolution of salmonid fishes</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">27</span> in <span class="cit-source">Evolutionary Genetics of Fishes</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. J.</span></span> <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Secondary tetrasomic segregation of MDH-B and preferential pairing of homeologues in rainbow trout.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1083%26rft.epage%253D1092%26rft.atitle%253DSecondary%2BTetrasomic%2BSegregation%2Bof%2BMDH-B%2Band%2BPreferential%2BPairing%2Bof%2BHomeologues%2Bin%2BRainbow%2BTrout%26rft_id%253Dinfo%253Apmid%252F9093860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NS80LzEwODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.4" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.5" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.6" data-doi="10.1038/ncomms4657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalopin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juanchich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3657%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4657%26rft_id%253Dinfo%253Apmid%252F24755649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755649&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.7" data-doi="10.1111/mec.12003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourret</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Primmer</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasemagi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP-array reveals genome-wide patterns of geographical and potential adaptive divergence across the natural range of Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313726300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> A dense linkage map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) reveals variable chromosomal divergence after an ancestral whole genome duplication event. G3 (Bethesda) 4<strong>:</strong> 447–460.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.9" data-doi="10.1016/j.semcdb.2012.12.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cañestro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albalat</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irimia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Fernandez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Impact of gene gains, losses and duplication modes on the origin and diversification of vertebrates.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCell%2BDev.%2BBiol.%26rft.volume%253D24%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2012.12.008%26rft_id%253Dinfo%253Apmid%252F23291262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2012.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23291262&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">STACKS: building and genotyping loci de novo from short-read sequences. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D1%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.11" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STACKS: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.12" data-doi="10.1371/journal.pone.0046662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crête-Lafrenière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Framing the Salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrete-Lafreniere%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46662%26rft.epage%253De46662%26rft.atitle%253DFraming%2Bthe%2Bsalmonidae%2Bfamily%2Bphylogenetic%2Bportrait%253A%2Ba%2Bmore%2Bcomplete%2Bpicture%2Bfrom%2Bincreased%2Btaxon%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046662%26rft_id%253Dinfo%253Apmid%252F23071608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071608&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.13" data-doi="10.1093/molbev/msj083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">What is the role of genome duplication in the evolution of complexity and diversity?</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj083%26rft_id%253Dinfo%253Apmid%252F16368775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIzLzUvODg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.14" data-doi="10.1139/g05-067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A comparative analysis of the rainbow trout genome with 2 other species of fish (Arctic charr and Atlantic salmon) within the tetraploid derivative Salmonidae family (subfamily: Salmoninae).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DDanzmann%26rft.auinit1%253DR.%2BG.%26rft.volume%253D48%26rft.issue%253D6%26rft.spage%253D1037%26rft.epage%253D1051%26rft.atitle%253DA%2Bcomparative%2Banalysis%2Bof%2Bthe%2Brainbow%2Btrout%2Bgenome%2Bwith%2B2%2Bother%2Bspecies%2Bof%2Bfish%2B%2528Arctic%2Bcharr%2Band%2BAtlantic%2Bsalmon%2529%2Bwithin%2Bthe%2Btetraploid%2Bderivative%2BSalmonidae%2Bfamily%2B%2528subfamily%253A%2BSalmoninae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg05-067%26rft_id%253Dinfo%253Apmid%252F16391673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g05-067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16391673&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.9.1717.15" data-doi="10.1186/1471-2164-9-557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Distribution of ancestral proto-Actinopterygian chromosome arms within the genomes of 4R-derivative salmonid fishes (Rainbow trout and Atlantic salmon).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDanzmann%26rft.auinit1%253DR.%2BG.%26rft.volume%253D9%26rft.spage%253D557%26rft.epage%253D557%26rft.atitle%253DDistribution%2Bof%2Bancestral%2Bproto-Actinopterygian%2Bchromosome%2Barms%2Bwithin%2Bthe%2Bgenomes%2Bof%2B4R-derivative%2Bsalmonid%2Bfishes%2B%2528Rainbow%2Btrout%2Band%2BAtlantic%2Bsalmon%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-557%26rft_id%253Dinfo%253Apmid%252F19032764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19032764&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.16" data-doi="10.1186/gb-2010-11-9-403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iturra</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing the genome of the Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DDavidson%26rft.auinit1%253DW.%2BS.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D403%26rft.epage%253D403%26rft.atitle%253DSequencing%2Bthe%2Bgenome%2Bof%2Bthe%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-9-403%26rft_id%253Dinfo%253Apmid%252F20887641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-9-403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20887641&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.17" data-doi="10.1023/A:1013759802604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biagi</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smailus</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic mapping of Y-chromosomal DNA markers in Pacific salmon.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DDevlin%26rft.auinit1%253DR.%2BH.%26rft.volume%253D111%26rft.issue%253D1-3%26rft.spage%253D43%26rft.epage%253D58%26rft.atitle%253DGenetic%2Bmapping%2Bof%2BY-chromosomal%2BDNA%2Bmarkers%2Bin%2BPacific%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1013759802604%26rft_id%253Dinfo%253Apmid%252F11841186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1013759802604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11841186&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173270000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.18" data-doi="10.1038/hdy.2013.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisbrenner</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botwright</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evidence for multiple sex-determining loci in Tasmanian Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D113%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.55%26rft_id%253Dinfo%253Apmid%252F23759729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759729&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.19" data-doi="10.1007/s10577-009-9055-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene and genome duplications: the impact of dosage-sensitivity on the fate of nuclear genes.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DEdger%26rft.auinit1%253DP.%2BP.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D699%26rft.epage%253D717%26rft.atitle%253DGene%2Band%2Bgenome%2Bduplications%253A%2Bthe%2Bimpact%2Bof%2Bdosage-sensitivity%2Bon%2Bthe%2Bfate%2Bof%2Bnuclear%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-009-9055-9%26rft_id%253Dinfo%253Apmid%252F19802709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-009-9055-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19802709&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270448800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.20" data-doi="10.1186/1471-2164-13-521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DEverett%26rft.auinit1%253DM.%2BV.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D521%26rft.epage%253D521%26rft.atitle%253DMeiotic%2Bmaps%2Bof%2Bsockeye%2Bsalmon%2Bderived%2Bfrom%2Bmassively%2Bparallel%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-521%26rft_id%253Dinfo%253Apmid%252F23031582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23031582&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.21" data-doi="10.1093/jxb/ers192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic asymmetry in allopolyploid plants: wheat as a model.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">5045</span>–<span class="cit-lpage">5059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fers192%26rft_id%253Dinfo%253Apmid%252F22859676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2My8xNC81MDQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.22" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>) genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.23" data-doi="10.1186/1471-2156-13-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guyomard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boussaha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krieg</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hervet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A synthetic rainbow trout linkage map provides new insights into the salmonid whole genome duplication and the conservation of synteny among teleosts.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGuyomard%26rft.auinit1%253DR.%26rft.volume%253D13%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DA%2Bsynthetic%2Brainbow%2Btrout%2Blinkage%2Bmap%2Bprovides%2Bnew%2Binsights%2Binto%2Bthe%2Bsalmonid%2Bwhole%2Bgenome%2Bduplication%2Band%2Bthe%2Bconservation%2Bof%2Bsynteny%2Bamong%2Bteleosts.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-13-15%26rft_id%253Dinfo%253Apmid%252F22424132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-13-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22424132&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.24" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.25" data-doi="10.1016/j.gde.2009.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panopoulou</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Polyploidy and genome restructuring: a variety of outcomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DHufton%26rft.auinit1%253DA.%2BL.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D600%26rft.epage%253D606%26rft.atitle%253DPolyploidy%2Band%2Bgenome%2Brestructuring%253A%2Ba%2Bvariety%2Bof%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2009.10.005%26rft_id%253Dinfo%253Apmid%252F19900800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2009.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19900800&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273117900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.26" data-doi="10.1098/rspb.2006.3749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurley</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new time-scale for ray-finned fish evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DHurley%26rft.auinit1%253DI.%2BA%26rft.volume%253D274%26rft.issue%253D1609%26rft.spage%253D489%26rft.epage%253D498%26rft.atitle%253DA%2Bnew%2Btime-scale%2Bfor%2Bray-finned%2Bfish%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2006.3749%26rft_id%253Dinfo%253Apmid%252F17476768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc0LzE2MDkvNDg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.27" data-doi="10.1038/nature03025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange-Thomann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJaillon%26rft.auinit1%253DO.%26rft.volume%253D431%26rft.issue%253D7011%26rft.spage%253D946%26rft.epage%253D957%26rft.atitle%253DGenome%2Bduplication%2Bin%2Bthe%2Bteleost%2Bfish%2BTetraodon%2Bnigroviridis%2Breveals%2Bthe%2Bearly%2Bvertebrate%2Bproto-karyotype.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03025%26rft_id%253Dinfo%253Apmid%252F15496914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15496914&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224585600035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.28" data-doi="10.1016/j.crvi.2008.07.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wincker</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">“Changing by doubling”, the impact of whole genome duplications in the evolution of eukaryotes.</span> <abbr class="cit-jnl-abbrev">C. R. Biol.</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2Brendus%2Bbiologies%26rft.stitle%253DC%2BR%2BBiol%26rft.aulast%253DJaillon%26rft.auinit1%253DO.%26rft.volume%253D332%26rft.issue%253D2-3%26rft.spage%253D241%26rft.epage%253D253%26rft.atitle%253D%2522Changing%2Bby%2Bdoubling%2522%252C%2Bthe%2Bimpact%2Bof%2BWhole%2BGenome%2BDuplications%2Bin%2Bthe%2Bevolution%2Bof%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.crvi.2008.07.007%26rft_id%253Dinfo%253Apmid%252F19281955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.crvi.2008.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19281955&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264581600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.29" data-doi="10.1086/427926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorgenson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Province</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leppert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ethnicity and human genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DJorgenson%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D276%26rft.epage%253D290%26rft.atitle%253DEthnicity%2Band%2Bhuman%2Bgenetic%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F427926%26rft_id%253Dinfo%253Apmid%252F15627237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/427926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15627237&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226215100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.9.1717.30"><div class="cit-metadata unstructured">Koop, B. F., and W. S. Davidson, 2008 Genomics and the genome duplication in salmonids, pp. 77–86 in <em>Fisheries for Global Welfare and Environment</em>, <em>5th World Fisheries Congress</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.31" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D7%26rft.spage%253D355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.33" data-doi="10.1371/journal.pbio.0030396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination difference between sexes: A role for haploid selection.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D3%26rft.spage%253D396%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227984000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.34" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1717.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source"><em>Statistical Genomics: Linkage</em>, <em>Mapping</em>, <em>and QTL Analysis</em></span>, <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lubieniecki</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative genomic analysis of Atlantic salmon, <em>Salmo salar</em>, from Europe and North America.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DLubieniecki%26rft.auinit1%253DK.%2BP.%26rft.volume%253D11%26rft.spage%253D105%26rft.epage%253D105%26rft.atitle%253DComparative%2Bgenomic%2Banalysis%2Bof%2BAtlantic%2Bsalmon%252C%2BSalmo%2Bsalar%252C%2Bfrom%2BEurope%2Band%2BNorth%2BAmerica.%26rft_id%253Dinfo%253Apmid%252F21092310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21092310&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1717.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">The Origins of Genome Architecture</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.38" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.9.1717.39"><div class="cit-metadata unstructured">Macqueen, D. J., and I. A. Johnston, 2014 A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification. Proceedings of the Royal Society B: Biological Sciences 281.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.40" data-doi="10.1111/j.2007.0018-0661.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">OneMap: software for genetic mapping in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMargarido%26rft.auinit1%253DG.%2BR.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D78%26rft.epage%253D79%26rft.atitle%253DOneMap%253A%2Bsoftware%2Bfor%2Bgenetic%2Bmapping%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0018-0661.02000.x%26rft_id%253Dinfo%253Apmid%252F17663699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0018-0661.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17663699&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248257800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.41" data-doi="10.1016/j.semcdb.2013.02.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayfield-Jones</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Washburn</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Watching the grin fade: Tracing the effects of polyploidy on different evolutionary time scales.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCell%2BDev.%2BBiol.%26rft.volume%253D24%26rft.spage%253D320%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2013.02.002%26rft_id%253Dinfo%253Apmid%252F23466286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2013.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23466286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.42" data-doi="10.1126/science.1207205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayrose</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothfels</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnuson-Ford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recently formed polyploid plants diversify at lower rates.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">1257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1207205%26rft_id%253Dinfo%253Apmid%252F21852456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0Ny8xMjU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.43" data-doi="10.1111/j.1365-2052.2008.01699.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClelland</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genetic linkage map for coho salmon (<em>Oncorhynchus kisutch</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DMcClelland%26rft.auinit1%253DE.%2BK.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D169%26rft.epage%253D179%26rft.atitle%253DA%2Bgenetic%2Blinkage%2Bmap%2Bfor%2Bcoho%2Bsalmon%2B%2528Oncorhynchus%2Bkisutch%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2008.01699.x%26rft_id%253Dinfo%253Apmid%252F18318791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2008.01699.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18318791&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254381100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.44" data-doi="10.1038/hdy.2010.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClelland</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative trait locus analysis of hatch timing, weight, length and growth rate in coho salmon, <em>Oncorhynchus kisutch</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMcClelland%26rft.auinit1%253DE.%2BK.%26rft.volume%253D105%26rft.issue%253D6%26rft.spage%253D562%26rft.epage%253D573%26rft.atitle%253DQuantitative%2Btrait%2Blocus%2Banalysis%2Bof%2Bhatch%2Btiming%252C%2Bweight%252C%2Blength%2Band%2Bgrowth%2Brate%2Bin%2Bcoho%2Bsalmon%252C%2BOncorhynchus%2Bkisutch.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.22%26rft_id%253Dinfo%253Apmid%252F20234386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20234386&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284307700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.45" data-doi="10.1111/j.1365-294X.2011.05305.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunelli</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A conserved haplotype controls parallel adaptation in geographically distant salmonid populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMiller%26rft.auinit1%253DM.%2BR.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D249%26rft.atitle%253DA%2Bconserved%2Bhaplotype%2Bcontrols%2Bparallel%2Badaptation%2Bin%2Bgeographically%2Bdistant%2Bsalmonid%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05305.x%26rft_id%253Dinfo%253Apmid%252F21988725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05305.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988725&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298845600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.46" data-doi="10.1186/1471-2164-9-223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjoglum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A linkage map of the Atlantic salmon (<em>Salmo salar)</em> based on EST-derived SNP markers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMoen%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.spage%253D223%26rft.epage%253D223%26rft.atitle%253DA%2Blinkage%2Bmap%2Bof%2Bthe%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Bbased%2Bon%2BEST-derived%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-223%26rft_id%253Dinfo%253Apmid%252F18482444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18482444&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.47" data-doi="10.1111/j.1467-2979.2008.00302.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hard</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bridging the gap between the phenotype and the genotype: linking genetic variation, selection, and adaptation in fishes.</span> <abbr class="cit-jnl-abbrev">Fish Fish.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish%2BFish.%26rft.volume%253D9%26rft.spage%253D396%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-2979.2008.00302.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-2979.2008.00302.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-other" id="cit-4.9.1717.48"><div class="cit-metadata unstructured">Naish, K. A., R. B. Phillips, M. S. O. Brieuc, L. R. Newton, A. E. Elz <em>et al.</em>, 2013 Comparative genome mapping between Chinook salmon (<em>Oncorhynchus tshawytscha</em>) and rainbow trout (<em>O. mykiss</em>) based on homologous microsatellite loci. G3 (Bethesda) 3<strong>:</strong> 2281–2288.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.49" data-doi="10.1073/pnas.1206625109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Near</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eytan</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dornburg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Resolution of ray-finned fish phylogeny and timing of diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13698</span>–<span class="cit-lpage">13703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206625109%26rft_id%253Dinfo%253Apmid%252F22869754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM0LzEzNjk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.50" data-doi="10.1046/j.1365-2052.2003.00957.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robison</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A consolidated linkage map for rainbow trout (<em>Oncorhynchus mykiss</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DNichols%26rft.auinit1%253DK.%2BM.%26rft.volume%253D34%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D115%26rft.atitle%253DA%2Bconsolidated%2Blinkage%2Bmap%2Bfor%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2052.2003.00957.x%26rft_id%253Dinfo%253Apmid%252F12648093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2052.2003.00957.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12648093&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181734700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1717.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">Evolution by Gene Duplication</span>, <span class="cit-publ-name">George Allen and Unwin</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.52" data-doi="10.1006/scdb.1999.0332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Gene duplication and the uniqueness of vertebrate genomes circa 1970–1999.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DOhno%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D517%26rft.epage%253D522%26rft.atitle%253DGene%2Bduplication%2Band%2Bthe%2Buniqueness%2Bof%2Bvertebrate%2Bgenomes%2Bcirca%2B1970-1999.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fscdb.1999.0332%26rft_id%253Dinfo%253Apmid%252F10597635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/scdb.1999.0332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10597635&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083733400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.53" data-doi="10.1186/1471-2164-14-570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ostberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garza</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromosome rearrangements, recombination suppression, and limited segregation distortion in hybrids between Yellowstone cutthroat trout (<em>Oncorhynchus clarkii bouvieri</em>) and rainbow trout (<em>O. mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D570%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-570%26rft_id%253Dinfo%253Apmid%252F23968234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23968234&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.54" data-doi="10.1146/annurev.genet.34.1.401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitton</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Polyploid incidence and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D401%26rft.epage%253D437%26rft.atitle%253DPOLYPLOID%2BINCIDENCE%2BAND%2BEVOLUTION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.34.1.401%26rft_id%253Dinfo%253Apmid%252F11092833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.34.1.401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11092833&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166299400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.55" data-doi="10.1017/S1464793100005613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ráb</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D25%26rft.atitle%253DChromosome%2Bevolution%2Bin%2Bthe%2BSalmonidae%2B%2528Pisces%2529%253A%2Ban%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793100005613%26rft_id%253Dinfo%253Apmid%252F11325050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793100005613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11325050&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.56" data-doi="10.1023/A:1013743431738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konkol</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome painting supports lack of homology among sex chromosomes in <em>Oncorhynchus</em>, <em>Salmo</em>, and <em>Salvelinus</em> (Salmonidae).</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DPhillip%26rft.auinit1%253DR.%2BB.%26rft.volume%253D111%26rft.issue%253D1-3%26rft.spage%253D119%26rft.epage%253D123%26rft.atitle%253DChromosome%2Bpainting%2Bsupports%2Black%2Bof%2Bhomology%2Bamong%2Bsex%2Bchromosomes%2Bin%2BOncorhynchus%252C%2BSalmo%252C%2Band%2BSalvelinus%2B%2528Salmonidae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1013743431738%26rft_id%253Dinfo%253Apmid%252F11841160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1013743431738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11841160&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.57" data-doi="10.1159/000086387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morasch</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification of the sex chromosome pair in coho salmon (<em>Oncorhynchus kisutch</em>): lack of conservation of the sex linkage group with chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%2BB.%26rft.volume%253D111%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D170%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bsex%2Bchromosome%2Bpair%2Bin%2Bcoho%2Bsalmon%2B%2528Oncorhynchus%2Bkisutch%2529%253A%2Black%2Bof%2Bconservation%2Bof%2Bthe%2Bsex%2Blinkage%2Bgroup%2Bwith%2Bchinook%2Bsalmon%2B%2528Oncorhynchus%2Btshawytscha%2529.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000086387%26rft_id%253Dinfo%253Apmid%252F16103659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000086387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103659&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231728500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.9.1717.58" data-doi="10.1534/genetics.105.055269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeKoning</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morasch</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keadey</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assignment of rainbow trout linkage groups to specific chromosomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.055269%26rft_id%253Dinfo%253Apmid%252F16951085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE2NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.59" data-doi="10.1186/1471-2156-10-46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keatley</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morasch</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubieniecki</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Assignment of Atlantic salmon (<em>Salmo salar</em>) linkage groups to specific chromosomes: Conservation of large syntenic blocks corresponding to whole chromosome arms in rainbow trout (<em>Oncorhynchus mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%2BB.%26rft.volume%253D10%26rft.spage%253D46%26rft.epage%253D46%26rft.atitle%253DAssignment%2Bof%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Blinkage%2Bgroups%2Bto%2Bspecific%2Bchromosomes%253A%2Bconservation%2Bof%2Blarge%2Bsyntenic%2Bblocks%2Bcorresponding%2Bto%2Bwhole%2Bchromosome%2Barms%2Bin%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-10-46%26rft_id%253Dinfo%253Apmid%252F19689812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-10-46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19689812&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Assignment of Chinook salmon (<em>Oncorhynchus tshawytscha</em>) linkage groups to specific chromosomes reveals a karyotype with multiple rearrangements of the chromosome arms of rainbow trout (<em>Oncorhynchus mykiss</em>). G3 (Bethesda) 3: 2289–2295.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The zebrafish genome in context: Ohnologs gone missing.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zoolog. B Mol. Dev. Evol.</abbr> <span class="cit-vol">308B</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bexperimental%2Bzoology.%2BPart%2BB.%2BMolecular%2Band%2Bdevelopmental%2Bevolution%26rft.stitle%253DJ%2BExp%2BZoolog%2BB%2BMol%2BDev%2BEvol%26rft.aulast%253DPostlethwait%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D308%26rft.issue%253D5%26rft.spage%253D563%26rft.epage%253D577%26rft.atitle%253DThe%2Bzebrafish%2Bgenome%2Bin%2Bcontext%253A%2Bohnologs%2Bgone%2Bmissing.%26rft_id%253Dinfo%253Apmid%252F17068775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17068775&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-4.9.1717.62"><div class="cit-metadata unstructured">R Development Core Team, 2013 R: A Language and Environment for Statistical Computing, pp. R Foundation for Statistical Computing. <a href="http://www.R-project.org">http://www.R-project.org</a>, Vienna, Austria.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.63" data-doi="10.1186/1471-2156-9-74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gahr</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A second generation genetic map for rainbow trout (<em>Oncorhynchus mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRexroad%26rft.auinit1%253DC.%2BE.%26rft.volume%253D9%26rft.spage%253D74%26rft.epage%253D74%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bgenetic%2Bmap%2Bfor%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-9-74%26rft_id%253Dinfo%253Apmid%252F19019240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-9-74&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19019240&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozaki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A microsatellite linkage map of rainbow trout (<em>Oncorhynchus mykiss</em>) characterised by large sex-specific differences in recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSakamoto%26rft.auinit1%253DT.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1331%26rft.epage%253D1345%26rft.atitle%253DA%2BMicrosatellite%2BLinkage%2BMap%2Bof%2BRainbow%2BTrout%2B%2528Oncorhynchus%2Bmykiss%2529%2BCharacterized%2Bby%2BLarge%2BSex-Specific%2BDifferences%2Bin%2BRecombination%2BRates%26rft_id%253Dinfo%253Apmid%252F10880492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzEzMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.65" data-doi="10.1186/1471-2148-9-194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harmon</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carnevale</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alfaro</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Did genome duplication drive the origin of teleosts? A comparative study of diversification in ray-finned fishes.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DSantini%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.spage%253D194%26rft.epage%253D194%26rft.atitle%253DDid%2Bgenome%2Bduplication%2Bdrive%2Bthe%2Borigin%2Bof%2Bteleosts%253F%2BA%2Bcomparative%2Bstudy%2Bof%2Bdiversification%2Bin%2Bray-finned%2Bfishes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-9-194%26rft_id%253Dinfo%253Apmid%252F19664233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-9-194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19664233&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.66" data-doi="10.1016/j.tig.2011.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interpretation of karyotype evolution should consider chromosome structural constraints.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DSchubert%26rft.auinit1%253DI.%26rft.volume%253D27%26rft.issue%253D6%26rft.spage%253D207%26rft.epage%253D216%26rft.atitle%253DInterpretation%2Bof%2Bkaryotype%2Bevolution%2Bshould%2Bconsider%2Bchromosome%2Bstructural%2Bconstraints.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.03.004%26rft_id%253Dinfo%253Apmid%252F21592609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21592609&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291567400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.67" data-doi="10.1073/pnas.97.13.7051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The role of genetic and genomic attributes in the success of polyploids.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7051</span>–<span class="cit-lpage">7057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSoltis%26rft.auinit1%253DP.%2BS.%26rft.volume%253D97%26rft.issue%253D13%26rft.spage%253D7051%26rft.epage%253D7057%26rft.atitle%253DThe%2Brole%2Bof%2Bgenetic%2Band%2Bgenomic%2Battributes%2Bin%2Bthe%2Bsuccess%2Bof%2Bpolyploids%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.13.7051%26rft_id%253Dinfo%253Apmid%252F10860970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTMvNzA1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.68" data-doi="10.1016/j.ympev.2012.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opazo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">F. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene duplication, genome duplication, and the functional diversification of vertebrate globins.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D66%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.07.013%26rft_id%253Dinfo%253Apmid%252F22846683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22846683&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Gene-centromere mapping in rainbow trout: High interference over long map distances.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThorgaard%26rft.auinit1%253DG.%2BH.%26rft.volume%253D103%26rft.issue%253D4%26rft.spage%253D771%26rft.epage%253D783%26rft.atitle%253DGENE-CENTROMERE%2BMAPPING%2BIN%2BRAINBOW%2BTROUT%253A%2BHIGH%2BINTERFERENCE%2BOVER%2BLONG%2BMAP%2BDISTANCES%26rft_id%253Dinfo%253Apmid%252F17246124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTAzLzQvNzcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timusk</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghadam</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution in the fish family salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DTimusk%26rft.auinit1%253DE.%2BR.%26rft.volume%253D12%26rft.spage%253D68%26rft.epage%253D68%26rft.atitle%253DGenome%2Bevolution%2Bin%2Bthe%2Bfish%2Bfamily%2Bsalmonidae%253A%2Bgeneration%2Bof%2Ba%2Bbrook%2Bcharr%2Bgenetic%2Bmap%2Band%2Bcomparisons%2Bamong%2Bcharrs%2B%2528Arctic%2Bcharr%2Band%2Bbrook%2Bcharr%2529%2Bwith%2Brainbow%2Btrout.%26rft_id%253Dinfo%253Apmid%252F21798024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21798024&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.71" data-doi="10.1038/nrg2600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The evolutionary significance of ancient genome duplications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DVan%2Bde%2BPeer%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D725%26rft.epage%253D732%26rft.atitle%253DThe%2Bevolutionary%2Bsignificance%2Bof%2Bancient%2Bgenome%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2600%26rft_id%253Dinfo%253Apmid%252F19652647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19652647&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269965100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.72" data-doi="10.1038/35072009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Yesterday’s polyploids and the mystery of diploidization.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D333%26rft.epage%253D341%26rft.atitle%253DYesterday%2527s%2Bpolyploids%2Band%2Bthe%2Bmystery%2Bof%2Bdiploidization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35072009%26rft_id%253Dinfo%253Apmid%252F11331899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35072009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11331899&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168467800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.73" data-doi="10.1101/gr.578503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woram</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyheim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparative genome analysis of the primary sex determining locus in five species of salmonid fish.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWoram%26rft.auinit1%253DR.%2BA.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D272%26rft.epage%253D280%26rft.atitle%253DComparative%2BGenome%2BAnalysis%2Bof%2Bthe%2BPrimary%2BSex-Determining%2BLocus%2Bin%2BSalmonid%2BFishes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.578503%26rft_id%253Dinfo%253Apmid%252F12566405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvMjcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Pseudolinkage of the duplicate loci for supernatant aspartate aminotransfease in brook trout, <em>Salvelinus fontinalis</em>.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DWRIGHT%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D223%26rft.epage%253D228%26rft.atitle%253DPseudolinkage%2Bof%2Bthe%2Bduplicate%2Bloci%2Bfor%2Bsupernatant%2Baspartate%2Baminotransferase%2Bin%2Bbrook%2Btrout%252C%2BSalvelinus%2Bfontinalis%26rft_id%253Dinfo%253Apmid%252F6774006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6ODoiNzEvNC8yMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes.</span> <abbr class="cit-jnl-abbrev">Isozymes Curr. Top. Biol. Med. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIsozymes%26rft.stitle%253DIsozymes%2BCurr%2BTop%2BBiol%2BMed%2BRes%26rft.aulast%253DWright%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D10%26rft.spage%253D239%26rft.epage%253D260%26rft.atitle%253DMeiotic%2Bmodels%2Bto%2Bexplain%2Bclassical%2Blinkage%252C%2Bpseudolinkage%252C%2Band%2Bchromosome%2Bpairing%2Bin%2Btetraploid%2Bderivative%2Bsalmonid%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F6354984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6354984&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RL29500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.76" data-doi="10.1109/TCBB.2010.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accurate construction of consensus genetic maps via integer linear programming.</span> <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinformatics%26rft.volume%253D8%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2010.35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2010.35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.77" data-doi="10.1016/j.cub.2011.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guyomard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouanno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An immune-related gene evolved into the master sex-determining gene in rainbow trout, <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DIkegami%26rft.auinit1%253DR.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DSemaphorin%2Band%2BEph%2Breceptor%2Bsignaling%2Bguide%2Ba%2Bseries%2Bof%2Bcell%2Bmovements%2Bfor%2Bventral%2Benclosure%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.12.009%26rft_id%253Dinfo%253Apmid%252F22197242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22197242&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1717.78" data-doi="10.1016/S0169-5347(03)00033-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evolution by gene duplication: an update.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">292</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D18%26rft.spage%253D292%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900033-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00033-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183516000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1717.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1709" title="Position Effect Variegation and Viability Are Both Sensitive to Dosage of Constitutive Heterochromatin in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1731" title="Genome-Wide Characterization of Light-Regulated Genes in Neurospora crassa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1717.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VRKYDYuAnwv1mtWsczd0c9JnDKiGkqB97-2ixJ6uMIg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1717" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1717" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1717" class="active" data-icon-position="" data-hide-link-title="0">Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459661" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6QUCkzLMdZYu_bCSGMSDUpsnhTQh_i9ni97mSBiP2HQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1717.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1717" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xlMH8x1e8Mm9MxQE5K5n8f21B85LA6zNmMpISrmOPOU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1717" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1717.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459661" id="node459661--2" data-pisa="ggg;4/9/1717" data-pisa-master="ggg;g3.114.012294" data-apath="/ggg/4/9/1717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Mapping Between Coho Salmon (<em>Oncorhynchus kisutch</em>) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miyako Kodama</span>, <span class="highwire-citation-author" data-delta="1">Marine S. O. Brieuc</span>, <span class="highwire-citation-author" data-delta="2">Robert H. Devlin</span>, <span class="highwire-citation-author" data-delta="3">Jeffrey J. Hard</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3275-8778" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kerry A. Naish</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1717-1730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012294 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459661--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miyako Kodama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miyako%2BKodama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kodama%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiyako%2BKodama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkodama@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mkodama@uw.edu</a>
<a href="mailto:knaish@uw.edu" class="" data-icon-position="" data-hide-link-title="0">knaish@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marine S. O. Brieuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BS.%2BO.%2BBrieuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brieuc%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarine%2BS.%2BO.%2BBrieuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert H. Devlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Fisheries and Oceans Canada, West Vancouver, British Columbia, V7K 1N6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH.%2BDevlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devlin%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH.%2BDevlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey J. Hard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Marine Fisheries Service, Northwest Fisheries Science Center, Seattle, Washington 98112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BJ.%2BHard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hard%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BJ.%2BHard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kerry A. Naish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerry%2BA.%2BNaish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naish%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKerry%2BA.%2BNaish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3275-8778" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kerry A. Naish</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkodama@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mkodama@uw.edu</a>
<a href="mailto:knaish@uw.edu" class="" data-icon-position="" data-hide-link-title="0">knaish@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1717" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1717&amp;title=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1717&amp;title=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1717&amp;title=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1717&amp;text=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1717&amp;title=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1717&amp;t=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1717&amp;title=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1717&amp;title=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459661/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459661" id="node459661--3" data-pisa="ggg;4/9/1717" data-pisa-master="ggg;g3.114.012294" data-apath="/ggg/4/9/1717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Mapping Between Coho Salmon (<em>Oncorhynchus kisutch</em>) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miyako Kodama</span>, <span class="highwire-citation-author" data-delta="1">Marine S. O. Brieuc</span>, <span class="highwire-citation-author" data-delta="2">Robert H. Devlin</span>, <span class="highwire-citation-author" data-delta="3">Jeffrey J. Hard</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3275-8778" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kerry A. Naish</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1717-1730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012294 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459661--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miyako Kodama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miyako%2BKodama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kodama%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiyako%2BKodama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkodama@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mkodama@uw.edu</a>
<a href="mailto:knaish@uw.edu" class="" data-icon-position="" data-hide-link-title="0">knaish@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marine S. O. Brieuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BS.%2BO.%2BBrieuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brieuc%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarine%2BS.%2BO.%2BBrieuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert H. Devlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Fisheries and Oceans Canada, West Vancouver, British Columbia, V7K 1N6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH.%2BDevlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devlin%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH.%2BDevlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey J. Hard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Marine Fisheries Service, Northwest Fisheries Science Center, Seattle, Washington 98112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BJ.%2BHard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hard%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BJ.%2BHard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kerry A. Naish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerry%2BA.%2BNaish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naish%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKerry%2BA.%2BNaish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3275-8778" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kerry A. Naish</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkodama@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mkodama@uw.edu</a>
<a href="mailto:knaish@uw.edu" class="" data-icon-position="" data-hide-link-title="0">knaish@uw.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1717&amp;rft.epage=1730&amp;rft.atitle=Comparative%20Mapping%20Between%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20and%20Three%20Other%20Salmonids%20Suggests%20a%20Role%20for%20Chromosomal%20Rearrangements%20in%20the%20Retention%20of%20Duplicated%20Regions%20Following%20a%20Whole%20Genome%20Duplication%20Event&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kodama%2C+Miyako&amp;rft.au=Brieuc%2C+Marine+S.+O.&amp;rft.au=Devlin%2C+Robert+H.&amp;rft.au=Hard%2C+Jeffrey+J.&amp;rft.au=Naish%2C+Kerry+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459661/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459661/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459661/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459661/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459661/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459661/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459661/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459661/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459661/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459661/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459661/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459661/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1717" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1717.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1717.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1717.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1717.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
