<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1657" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1657.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1657.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1657.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1657.atom" />
<meta name="HW.pisa" content="ggg;4/9/1657" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1534/g3.114.012914" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Chandler. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Experimental evolution studies, coupled with new advances in DNA sequencing technology, have become a powerful tool for exploring how populations respond to selection at the genomic level. Recent experiments in microbes typically have found evidence for multiple novel mutations, which are usually fixed. In contrast, in animal model systems, evolutionary responses seem to involve more modest changes in the frequencies of pre-existing alleles, probably because these populations outcross and are usually initialized with greater levels of standing variation. In this experiment, I used whole-genome resequencing to estimate allele frequencies and look for novel substitutions in experimentally evolved populations of Caenorhabditis elegans . These populations were founded with a fixed pair of deleterious mutations introgressed into multiple wild genetic backgrounds and allowed to evolve for 50 generations with a mixed mating system. There is evidence for some recombination between ancestral haplotypes, but selective sweeps seem to have resulted in the fixation of large chromosomal segments throughout most of the genome. In addition, a few new mutations were detected. Simulations suggest that strong selection and low outcrossing rates are likely explanations for the observed outcomes, consistent with earlier work showing large fitness increases in these populations over 50 generations. These results also show clear parallels to population genetic patterns in C. elegans in nature: recent selective sweeps, high linkage disequilibrium, and low effective recombination rates. Thus, the genomic consequences of selection depend heavily on the biology of the organism in question, including its mating system and levels of genetic variation." />
<meta name="DC.Contributor" content="Christopher H. Chandler" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Experimental evolution studies, coupled with new advances in DNA sequencing technology, have become a powerful tool for exploring how populations respond to selection at the genomic level. Recent experiments in microbes typically have found evidence for multiple novel mutations, which are usually fixed. In contrast, in animal model systems, evolutionary responses seem to involve more modest changes in the frequencies of pre-existing alleles, probably because these populations outcross and are usually initialized with greater levels of standing variation. In this experiment, I used whole-genome resequencing to estimate allele frequencies and look for novel substitutions in experimentally evolved populations of &lt;i&gt;Caenorhabditis elegans&lt;/i&gt;. These populations were founded with a fixed pair of deleterious mutations introgressed into multiple wild genetic backgrounds and allowed to evolve for 50 generations with a mixed mating system. There is evidence for some recombination between ancestral haplotypes, but selective sweeps seem to have resulted in the fixation of large chromosomal segments throughout most of the genome. In addition, a few new mutations were detected. Simulations suggest that strong selection and low outcrossing rates are likely explanations for the observed outcomes, consistent with earlier work showing large fitness increases in these populations over 50 generations. These results also show clear parallels to population genetic patterns in &lt;i&gt;C. elegans&lt;/i&gt; in nature: recent selective sweeps, high linkage disequilibrium, and low effective recombination rates. Thus, the genomic consequences of selection depend heavily on the biology of the organism in question, including its mating system and levels of genetic variation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1657" />
<meta name="citation_id" content="4/9/1657" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1657" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1657.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1657.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1657.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012914" />
<meta name="citation_pmid" content="24989146" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1657" />
<meta name="citation_lastpage" content="1665" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christopher H. Chandler" />
<meta name="citation_author_institution" content="Department of Biological Sciences, SUNY Oswego, Oswego, New York 13126" />
<meta name="citation_author_email" content="christopher.chandler@oswego.edu" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. C. Andersen;citation_author=J. P. Gerke;citation_author=J. A. Shapiro;citation_author=J. R. Crissman;citation_author=R. Ghosh;citation_title=Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity.;citation_pages=285-290;citation_volume=44;citation_year=2012;citation_issue=3;citation_pmid=22286215;citation_doi=10.1038/ng.1050" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. L. Anderson;citation_author=L. T. Morran;citation_author=P. C. Phillips;citation_title=Outcrossing and the maintenance of males within C. elegans populations.;citation_volume=101;citation_year=2010;citation_pmid=20212008;citation_doi=10.1093/jhered/esq003" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. L. Araya;citation_author=C. Payen;citation_author=M. J. Dunham;citation_author=S. Fields;citation_title=Whole-genome sequencing of a laboratory-evolved yeast strain.;citation_pages=88-88;citation_volume=11;citation_year=2010;citation_pmid=20128923;citation_doi=10.1186/1471-2164-11-88" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=A. Barrière;citation_author=M.-A. Félix;citation_title=High local genetic diversity and low outcrossing rate in Caenorhabditis elegans natural populations.;citation_pages=1176-1184;citation_volume=15;citation_year=2005;citation_issue=13;citation_pmid=16005289;citation_doi=10.1016/j.cub.2005.06.022" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Barrière;citation_author=M.-A. Félix;citation_title=Temporal dynamics and linkage disequilibrium in natural Caenorhabditis elegans populations.;citation_volume=176;citation_year=2007;citation_pmid=17409084;citation_doi=10.1534/genetics.106.067223" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. K. Burke;citation_author=J. P. Dunham;citation_author=P. Shahrestani;citation_author=K. R. Thornton;citation_author=M. R. Rose;citation_title=Genome-wide analysis of a long-term evolution experiment with Drosophila.;citation_pages=587-590;citation_volume=467;citation_year=2010;citation_issue=7315;citation_pmid=20844486;citation_doi=10.1038/nature09352" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. H. Chandler;citation_title=Cryptic intraspecific variation in sex determination in Caenorhabditis elegans revealed by mutations.;citation_pages=473-482;citation_volume=105;citation_year=2010;citation_issue=5;citation_pmid=20502478;citation_doi=10.1038/hdy.2010.62" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=C. H. Chandler;citation_author=P. C. Phillips;citation_author=F. J. Janzen;citation_title=The evolution of sex-determining mechanisms: lessons from temperature-sensitive mutations in sex determination genes in Caenorhabditis elegans.;citation_pages=192-200;citation_volume=22;citation_year=2009;citation_issue=1;citation_pmid=19120819;citation_doi=10.1111/j.1420-9101.2008.01639.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=C. H. Chandler;citation_author=G. E. Chadderdon;citation_author=P. C. Phillips;citation_author=I. Dworkin;citation_author=F. J. Janzen;citation_title=Experimental evolution of the Caenorhabditis elegans sex determination pathway.;citation_pages=82-93;citation_volume=66;citation_year=2012;citation_issue=1;citation_pmid=22220866;citation_doi=10.1111/j.1558-5646.2011.01420.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. Charusanti;citation_author=T. M. Conrad;citation_author=E. M. Knight;citation_author=K. Venkataraman;citation_author=N. L. Fong;citation_title=Genetic basis of growth adaptation of Escherichia coli after deletion of pgi, a major metabolic gene.;citation_pages=e1001186-e1001186;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21079674;citation_doi=10.1371/journal.pgen.1001186" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=I. M. Chelo;citation_author=H. Teotonio;citation_title=The opportunity for balancing selection in experimental populations of Caenorhabditis elegans.;citation_pages=142-156;citation_volume=67;citation_year=2013;citation_pmid=23289568;citation_doi=10.1111/j.1558-5646.2012.01744.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=I. Comas;citation_author=S. Borrell;citation_author=A. Roetzer;citation_author=G. Rose;citation_author=B. Malla;citation_title=Whole-genome sequencing of rifampicin-resistant Mycobacterium tuberculosis strains identifies compensatory mutations in RNA polymerase genes.;citation_pages=106-110;citation_volume=44;citation_year=2011;citation_pmid=22179134;citation_doi=10.1038/ng.1038" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. B. Corbett-Detig;citation_author=J. Zhou;citation_author=A. G. Clark;citation_author=D. L. Hartl;citation_author=J. F. Ayroles;citation_title=Genetic incompatibilities are widespread within species.;citation_pages=135-137;citation_volume=504;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Denver;citation_author=D. K. Howe;citation_author=L. J. Wilhelm;citation_author=C. A. Palmer;citation_author=J. L. Anderson;citation_title=Selective sweeps and parallel mutation in the adaptive recovery from deleterious mutation in Caenorhabditis elegans.;citation_volume=20;citation_year=2010;citation_pmid=21036923;citation_doi=10.1101/gr.108191.110" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. R. Dettman;citation_author=N. Rodrigue;citation_author=A. H. Melnyk;citation_author=A. Wong;citation_author=S. F. Bailey;citation_title=Evolutionary insight from whole-genome sequencing of experimentally evolved microbes.;citation_pages=2058-2077;citation_volume=21;citation_year=2012;citation_issue=9;citation_pmid=22332770;citation_doi=10.1111/j.1365-294X.2012.05484.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. M. Dodd;citation_title=Reproductive isolation as a consequence of adaptive divergence in Drosophila pseudoobscura.;citation_pages=1308-1311;citation_volume=43;citation_year=1989;citation_doi=10.2307/2409365" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=E. S. Dolgin;citation_author=B. Charlesworth;citation_author=S. E. Baird;citation_author=A. D. Cutter;citation_title=Inbreeding and outbreeding depression in Caenorhabditis nematodes.;citation_pages=1339-1352;citation_volume=61;citation_year=2007;citation_issue=6;citation_pmid=17542844;citation_doi=10.1111/j.1558-5646.2007.00118.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Estes;citation_author=M. Lynch;citation_title=Rapid fitness recovery in mutationally degraded lines of Caenorhabditis elegans.;citation_pages=1022-1030;citation_volume=57;citation_year=2003;citation_issue=5;citation_pmid=12836820;citation_doi=10.1554/0014-3820(2003)057[1022:RFRIMD]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=J. C. Gray;citation_author=A. D. Cutter;citation_title=Mainstreaming Caenorhabditis elegans in experimental evolution.;citation_volume=281;citation_year=2014;citation_pmid=24430852;citation_doi=10.1098/rspb.2013.3055" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. M. Hammell;citation_author=I. Lubin;citation_author=P. R. Boag;citation_author=T. K. Blackwell;citation_author=V. Ambros;citation_title=nhl-2 Modulates microRNA activity in Caenorhabditis elegans.;citation_pages=926-938;citation_volume=136;citation_year=2009;citation_issue=5;citation_pmid=19269369;citation_doi=10.1016/j.cell.2009.01.053" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=W. R. Harcombe;citation_author=R. Springman;citation_author=J. J. Bull;citation_title=Compensatory evolution for a gene deletion is not limited to its immediate functional network.;citation_pages=106-106;citation_volume=9;citation_year=2009;citation_pmid=19445716;citation_doi=10.1186/1471-2148-9-106" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. J. Hillers;citation_author=A. M. Villeneuve;citation_title=Chromosome-wide control of meiotic crossing over in C. elegans.;citation_pages=1641-1647;citation_volume=13;citation_year=2003;citation_issue=18;citation_pmid=13678597;citation_doi=10.1016/j.cub.2003.08.026" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=T. J. Kawecki;citation_author=R. E. Lenski;citation_author=D. Ebert;citation_author=B. Hollis;citation_author=I. Olivieri;citation_title=Experimental evolution.;citation_pages=547-560;citation_volume=27;citation_year=2012;citation_issue=10;citation_pmid=22819306;citation_doi=10.1016/j.tree.2012.06.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. J. Kulathinal;citation_author=B. R. Bettencourt;citation_author=D. L. Hartl;citation_title=Compensated deleterious mutations in insect genomes.;citation_volume=306;citation_year=2004;citation_pmid=15498973;citation_doi=10.1126/science.1100522" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=R. E. Lenski;citation_author=M. R. Rose;citation_author=S. C. Simpson;citation_author=S. C. Tadler;citation_title=Long-term experimental evolution in Escherichia coli. I. Adaptation and divergence during 2,000 generations.;citation_pages=1315-1341;citation_volume=138;citation_year=1991;citation_doi=10.1086/285289" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. L. Malmberg;citation_title=THE EVOLUTION OF EPISTASIS AND THE ADVANTAGE OF RECOMBINATION IN POPULATIONS OF BACTERIOPHAGE T4;citation_pages=607-621;citation_volume=86;citation_year=1977;citation_issue=3;citation_pmid=892424" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=F. B. G. Moore;citation_author=D. E. Rozen;citation_author=R. E. Lenski;citation_title=Pervasive compensatory adaptation in Escherichia coli;citation_pages=515-522;citation_volume=267;citation_year=2000;citation_issue=1442;citation_pmid=10737410;citation_doi=10.1098/rspb.2000.1030" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. T. Morran;citation_author=M. D. Parmenter;citation_author=P. C. Phillips;citation_title=Mutation load and rapid adaptation favour outcrossing over self-fertilization.;citation_pages=350-352;citation_volume=462;citation_year=2009;citation_issue=7271;citation_pmid=19847164;citation_doi=10.1038/nature08496" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. T. Morran;citation_author=O. G. Schmidt;citation_author=I. A. Gelarden;citation_author=R. C. Parrish;citation_author=C. M. Lively;citation_title=Running with the Red Queen: Host-Parasite Coevolution Selects for Biparental Sex;citation_pages=216-218;citation_volume=333;citation_year=2011;citation_issue=6039;citation_pmid=21737739;citation_doi=10.1126/science.1206360" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Orozco-terWengel;citation_author=M. Kapun;citation_author=V. Nolte;citation_author=R. Kofler;citation_author=T. Flatt;citation_title=Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.;citation_pages=4931-4941;citation_volume=21;citation_year=2012;citation_issue=20;citation_pmid=22726122;citation_doi=10.1111/j.1365-294X.2012.05673.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=A. Pischedda;citation_author=A. K. Chippindale;citation_title=Sex, mutation and fitness: asymmetric costs and routes to recovery through compensatory evolution.;citation_pages=1115-1122;citation_volume=18;citation_year=2005;citation_issue=4;citation_pmid=16033585;citation_doi=10.1111/j.1420-9101.2005.00915.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. C. Remolina;citation_author=P. L. Chang;citation_author=J. Leips;citation_author=S. V. Nuzhdin;citation_author=K. A. Hughes;citation_title=GENOMIC BASIS OF AGING AND LIFE-HISTORY EVOLUTION IN DROSOPHILA MELANOGASTER.;citation_pages=3390-3403;citation_volume=66;citation_year=2012;citation_issue=11;citation_pmid=23106705;citation_doi=10.1111/j.1558-5646.2012.01710.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Rice;citation_title=Sexually antagonistic genes: experimental evidence;citation_pages=1436-1439;citation_volume=256;citation_year=1992;citation_issue=5062;citation_pmid=1604317;citation_doi=10.1126/science.1604317" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Recombinational landscape and population genomics of Caenorhabditis elegans.;citation_pages=e1000419-e1000419;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19283065;citation_doi=10.1371/journal.pgen.1000419" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. R. Rose;citation_title=Laboratory evolution of postponed senescence in Drosophila melanogaster.;citation_pages=1004-1010;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408434" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. M. Stoebel;citation_author=K. Hokamp;citation_author=M. S. Last;citation_author=C. J. Dorman;citation_title=Compensatory evolution of gene regulation in response to stress by Escherichia coli lacking RpoS.;citation_pages=e1000671-e1000671;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19798444;citation_doi=10.1371/journal.pgen.1000671" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Teotonio;citation_author=S. Carvalho;citation_author=D. Manoel;citation_author=M. Roque;citation_author=I. M. Chelo;citation_title=Evolution of outcrossing in experimental populations of Caenorhabditis elegans.;citation_pages=e35811-e35811;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22540006;citation_doi=10.1371/journal.pone.0035811" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Tobler;citation_author=S. U. Franssen;citation_author=R. Kofler;citation_author=P. Orozco-terWengel;citation_author=V. Nolte;citation_title=Massive habitat-specific genomic response in D. melanogaster populations during experimental evolution in hot and cold environments.;citation_volume=31;citation_year=2014;citation_pmid=24150039;citation_doi=10.1093/molbev/mst205" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. L. Turner;citation_author=P. M. Miller;citation_title=Investigating natural variation in Drosophila courtship song by the evolve and resequence approach.;citation_volume=191;citation_year=2012;citation_pmid=22466043;citation_doi=10.1534/genetics.112.139337" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. L. Turner;citation_author=A. D. Stewart;citation_author=A. T. Fields;citation_author=W. R. Rice;citation_author=A. M. Tarone;citation_title=Population-based resequencing of experimentally evolved populations reveals the genetic basis of body size variation in Drosophila melanogaster.;citation_pages=e1001336-e1001336;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437274;citation_doi=10.1371/journal.pgen.1001336" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Zhou;citation_author=N. Udpa;citation_author=M. Gersten;citation_author=D. W. Visk;citation_author=A. Bashir;citation_title=Experimental selection of hypoxia-tolerant Drosophila melanogaster.;citation_volume=108;citation_year=2011;citation_pmid=21262834;citation_doi=10.1073/pnas.1010643108" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="Experimental evolution studies, coupled with new advances in DNA sequencing technology, have become a powerful tool for exploring how populations respond to selection at the genomic level. Recent experiments in microbes typically have found evidence for multiple novel mutations, which are usually fixed. In contrast, in animal model systems, evolutionary responses seem to involve more modest changes in the frequencies of pre-existing alleles, probably because these populations outcross and are usually initialized with greater levels of standing variation. In this experiment, I used whole-genome resequencing to estimate allele frequencies and look for novel substitutions in experimentally evolved populations of Caenorhabditis elegans . These populations were founded with a fixed pair of deleterious mutations introgressed into multiple wild genetic backgrounds and allowed to evolve for 50 generations with a mixed mating system. There is evidence for some recombination between ancestral haplotypes, but selective sweeps seem to have resulted in the fixation of large chromosomal segments throughout most of the genome. In addition, a few new mutations were detected. Simulations suggest that strong selection and low outcrossing rates are likely explanations for the observed outcomes, consistent with earlier work showing large fitness increases in these populations over 50 generations. These results also show clear parallels to population genetic patterns in C. elegans in nature: recent selective sweeps, high linkage disequilibrium, and low effective recombination rates. Thus, the genomic consequences of selection depend heavily on the biology of the organism in question, including its mating system and levels of genetic variation." />
<meta name="og-title" property="og:title" content="Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1657" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Experimental evolution studies, coupled with new advances in DNA sequencing technology, have become a powerful tool for exploring how populations respond to selection at the genomic level. Recent experiments in microbes typically have found evidence for multiple novel mutations, which are usually fixed. In contrast, in animal model systems, evolutionary responses seem to involve more modest changes in the frequencies of pre-existing alleles, probably because these populations outcross and are usually initialized with greater levels of standing variation. In this experiment, I used whole-genome resequencing to estimate allele frequencies and look for novel substitutions in experimentally evolved populations of Caenorhabditis elegans . These populations were founded with a fixed pair of deleterious mutations introgressed into multiple wild genetic backgrounds and allowed to evolve for 50 generations with a mixed mating system. There is evidence for some recombination between ancestral haplotypes, but selective sweeps seem to have resulted in the fixation of large chromosomal segments throughout most of the genome. In addition, a few new mutations were detected. Simulations suggest that strong selection and low outcrossing rates are likely explanations for the observed outcomes, consistent with earlier work showing large fitness increases in these populations over 50 generations. These results also show clear parallels to population genetic patterns in C. elegans in nature: recent selective sweeps, high linkage disequilibrium, and low effective recombination rates. Thus, the genomic consequences of selection depend heavily on the biology of the organism in question, including its mating system and levels of genetic variation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2gHn_aIn7W1rMID9946p8nrQGDrW6K0tH_od6FXJ0y8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459662","apath":"\/ggg\/4\/9\/1657.atom","pisa":"ggg;4\/9\/1657","ac":{"\/ggg\/4\/9\/1657.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1657.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1657"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1657"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1657":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224pi\\\\\\\/HHWmIHTbI7Fxz79K\\\\\\\/2wm6IgKTIqalw9aJDy+IyebEqyWAEUf3kXgN7REypikYmki7IqzlRSNWPaopYy3eTCiWl0JMPU8UN4UIsbgVBLMVmv6JjggMJojXAZR\\\\\\\/OVRVDAQgRB8F5gJsc3hA9xbP5c4qnG9CodxxpBFXVR9hp11lm0eTRsawdl9L3l4AlGrqpJAI\\\\\\\/+TosilHCr47amutFdz0jmy8Zy3zIQUCEXi7N+Og8cCgAe4gkh9VG13CXKrPo6OIO8OoJGVop8zhmIDlyuZIxUimOJhiAesDgeEGp93SVzVUB3Qb4cOFS\\\\\\\/RD0R9fHFI5mMXsd\\\\\\\/Jr25DiayF6LyhtNsDMy5CCloReGFYK\\\\\\\/N7m35jDMKsZH8MFVltOrf6l8t+JuK8N+8Y3OHTwFO5KlmuBQwDDwxZStdKlGvx69MZ0m\\\\\\\/C10zNQfeAVGYRi9fMDVZxzSdJ+k6jI3fEa+lZ1XCFqE\\\\\\\/T3e4n2wh\\\\\\\/RrPHak\\\\\\\/HKEMzfmm9t\\\\\\\/chYO0XZTnPhUgVp5B5HCPQTsi48fhxa5LzfQkJ6M56Wg5dgGgd9TVSEUp+mM3W0TAMpaUnYbacaednS\\\\\\\/OECQ8WIHECdd4iwWishsG5hWR\\\\\\\/WrFJTZMHM0FPtxcyZNF0vwXuIiM41mbDK2bDMp2QzkHUKy2\\\\\\\/\\\\\\\/It2HJ\\\\\\\/vGuWoCmh6IR9WrUfnirIJfRzyBTkOZ04KcR4VGNd5jL+wf+MV9z0u807Vn4dSLP7d0wiapRtwUqkcSwW3ut61SUcnTVenYcDGrsQZXLLh2xbqCfUZDa5OD8gGFBozaIyqTeDBY3GwrQXOnoYY07P0EBsl8+v0VSM7KkshmQFKFi6+B+sEiUybVDcFpvUHXkBeZLbgEPHf5vHn67hzs6CIswZcaeEOaT3zrW4XBTORW9VyXNbVihqqKo0Dk43DkU\\\\\\\/Qa3sxmxYUUru8901PMU5n1cIQEZE6RJdYb0wI\\\\\\\/4wKu39\\\\\\\/hvEHGJ9ZXiyUrscm1VSB6Br2FLSyjhhzys2faxgG\\\\\\\/wJBV1d4kSjHjsjM0o1n4JiCYSDj4TZcb86LEQXcwD6LMy8znh4+kDneRoJVB6Pnj4yQ6R6zWPT9xtXQj72SE76ZZiXIWSWcZRq+ucekLEzZAALu+G\\\\\\\/6LPkxKHfFGdhjbaVc+O65LEZ\\\\\\\/qUL6AO5u+wPg5TbDKNQkGcfyd9Mg4i\\\\\\\/YCQIFxb4ydKqN6zLu5TvG\\\\\\\/DFEQYYDjZ+jAZR3snLb\\\\\\\/P0B9pqDZHFmJeRKhzNWRU0ElsU8l3HYYjNRREL1LPnRGMgfBa83u0ZaK03Kic57\\\u0022,\\\u0022iv\\\u0022:\\\u0022y3JYQ\\\\\\\/bm2KqXPFN1\\\\\\\/Tb2TA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cd6e863b9ef40ad32c18c15a6a8b2e8d8187a79e530129d69bbb7e640a0de0f7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022idd+C3Qbttk1Ujr1KgJhcQ2GmQhHaQpWeRUcjZk8iXS6ejGMURiiHWxChx5s9iP\\\\\\\/MXAWWQ0N72ds+ozrtzW4AgRbS5RJ7Re3DrfwRbSer185QEg5i1RoODuR1eiI\\\\\\\/IWX2VZEyQzFkwAeIJ12FABFRQ3oebfdmvtqGAhqJqvZhk8eDgBf6mTsK+igJfDK\\\\\\\/JNln+ZsiGDw7JBjQgjeSsCt\\\\\\\/GXkTNOnRQmBXczQnoKvGIMVyiCGL2DcAL6lDiQaQ3fztW+id18AQzHPAYucBcXODVEWyx0849B5gMbBs1Tys+dQDEN0SehM4W7VX0Uchnn\\\\\\\/vbCTe7RjsR3Wj6rjA1Bpv597Tpd1LDYoBdT6RP+d2AZkPTFDDljuoBS7OxJaqmUYXXNmZ0C+E03kECaG8Jax5UCdE8z1A3nrHoKN731+FOb39aphLrW01c4Hw+G14HS2JTAqj1YOtjPdAFeMTpkWKFdjdIaZAEiJtF0v8WtwW4SP8Q+OC4Q2LNBfs8hN9Ccm\\\\\\\/TjlTWSu7aP+F3QEaLsPKhFefPTE6xOzmVmP5luaLTFxY46IRi7kYL4riG9txmnzExCcjd53RRjoSLIx0vlu1nLewsIbY9Fk26XgqPaTJrgXwggkc0m0hbN4OOWr5DiVac4pHFQTPH1zEdZhNsovU3vUiDACM45DErqOsiILjxb1W1Kr53GysuI\\\\\\\/5PuWW52wqN9wFzfuSACoS6Tr2uKACb4Sxv0eODnPC4p9PZzUoc8wl8hSEii0MoFQfaGWMcG6\\\\\\\/Ejb+BIUzd++DRu5Er5LgySyEygVs1UWuvbNkByeu1szxe5KmItlil+HRfMHqQr8WicjhTExUixDkDH7Lv5Vu9Pp57Saq9Dm0XmMgYFtjdlyKqK9Gb4xcG\\\\\\\/b9daazmp4Lumln3lNg3maMZZia\\\\\\\/eB8aCfPKSyYbdkkHUEtJVbJn1v0oBnV52o\\\\\\\/i6iXQi\\\\\\\/fY2hYRrEelusWMdcE9MbZebG1VH1iYrf2E8dShfsOd5p\\\\\\\/ayMY8u50rs6Fu4CpkVCrwUjVqtofTCz69Ed8XYi4qb2pCh\\\\\\\/HMs6bgHrJ97IaqybFDqWK6aTrsfyAzftrhqf9XgsGVRUj08mhxjRVLdwUIoqdd2k+kNX2jWzAnchtqytbkWiTzi6GMX60aXPElA6anKeB+S3xb9dfKbowPBxUDyC0Rrczaz0CfjS6HR2M50OrmVZNPqIy4TKAI+gxG0C\\\\\\\/VOI+hR9kcnhA1zhTlTK36pF8e46LzWJ4UwelTpEdBh8vxdTAqmmtRyjhnNTuLO2DHxnIZLtBD7YaOckK6B5gUkRD6Z5z2slfGB5Fy1Rkxa7Td52vmyPerDopTjp079klENWf5RFZCRTwZNKXpSu7K3uZfaYocEkP5uf3FbC1bW3UTjBm4tRTwlB26u8BWGx3AjQIjm5rgnbkBb8nC1J7OBYdBLHAGv8u4aL+m9d9JhCKEN6dDnyJYjpI5+cwS4ezUaGhqiWjiTxud5dYsU4fDbl+fgMohRTo9++g2IuUjKQnqKiEfCrITNnDzGqNjBrcv1+XkKTK3ASi\\\\\\\/HTZmm1FJ\\\\\\\/iYpxngfcLjGhle7TgPcb2U0FpY7wIWNoHzlPgR\\\\\\\/6QzfvlN1Eh1rtCG3EiUurzM6JrXZ1PH+Zj7f64VCwayhxgTuWMhts\\\\\\\/a8jZywcXXkPWQ3eJHuxl4TMu9uO8v0S0dbV5MH+VvyIrGGCPWDXLbW9WKnIdwvjxUKYP\\\\\\\/5ZCADfcwb2XeVxx\\\\\\\/yo7mus0Jh0X8d2v3pYlj2cRFMP6BCXBkYaz1IbZShrTzBxFiMNeNfisdsq3ZQGsfPi\\\\\\\/0jMYv2fVqTWo+UL3Qw7KLdtqSBaLNuTh+uTCp9AFTar5wiqwXQ5ZS3mPDdHg7TKXoO9gZnIdaVkY8cVx37ZpQA3guz8Wrftt0i+2OdJP3jDPHDc4U1wpeAf7T0VwQDFdPZso8cglpL+ybxezZol7uXP5YBED8c8PPjep+VnwtO3gsEXIrXJcowsb2hSi08DKmhRv\\\\\\\/YCzIt9PcUi3rBSAVS4SqmpmFbDn\\\\\\\/hShGz+IPquh5+0uPQVSh6lt7Ldtg4sWZfW0uHfMLRGVirQ2lQYNWFwcY17pVuveHxPTG2eFKCPDDcPMRWBIqnpiP9dZ9USm3KAP90ussR7qb6TAmUiIW53SUm5AmkcNKTVGbHiNbiNZ\\\\\\\/BRrGhfqhMDlSXz8Gq24NnzsTDc1Vic9J3iEPpjLhHe5zbcoWSXaAigpR4xbt0iLdtOB\\\\\\\/QRlGjtF8\\\\\\\/aNIdu553ldBC1LaueQ2BJ7PH9ci6c778bEHU57UPvu9AVrobvp4fpDlpszUW5INuLcXBYbZV+ZQagmO4VN+OTOroZtMNuPmJ\\\\\\\/mFpOeP6AgtI4m5\\\\\\\/eL2owhjZDaDhFJrzp10iFVSjllWqzhysBbIO0V\\\\\\\/9yUef5HJmUv2+nZOn47f6mSlYUh2ninj5yn8Z\\\\\\\/J4mfNxQlI13sI8DqwiztX3T+yb+T+zpEKwS8DZE7+2oXY+lhYw4bDcjPHHuzkBPP0qVwkWWa49pfIS6NJvVVSRGz+bWutjeBOrFaIYaEKLoOoW2a\\\\\\\/5reObC4c+oB3R9vDAzbmxNq7ad7VgO6awDj2yV3n2+LN+5SrfLt8cXVh3b1O\\\\\\\/JVYSAHlp0+kBDKPKcXVAKyPXtEbguy\\\\\\\/I3TIKXFfpSB8Z2jjTfRn6xH8Z0OYLwWV2ONJyRBDK0jT\\\\\\\/nETRttQ+O+qzgjObBO8iWDMnB4iwTCRS4YObuUv5cUD3uPFtXXz8YJHS0GENetJo0qR0qoBRJOdA3h5+mfQDVeBFW4o24fHPLDpIpAfgemn6RJUMXT3Ksqx+8ARCI3L1N+hnE21rx70BDEtMQ2oPuYguzehEnYIO8bVAldjOHtt5i9evIbifAOgO6XgDNj+dliI8JjoKWpwu\\\\\\\/MUI+ejrNREhbuMDyfu6jfmEPsSF\\\\\\\/DO0gotVINkzbZL+b8UcsEYrXgLutCE6kBcZ7++Z8Ev2XSe3lyMlgiDQ5EkEkiB3A33CAZmb4q44zYE2QHWhWwNAqX3N1MajmR2wIrV+cIQRnJmSDqX476\\\\\\\/NQZ9E7h069Tx8Z+tDrDFSuWOaoS+ig4RGkeqbI1D9Bmx27Nc1FQQID8S2gx3y0TO3CnysqZHbTV\\\\\\\/pYx9H6QBglq+24tuo2wHzHO5GbcWtSJ71EjxOcvWA6ViHmBrypx8b9gL0z5x7RUQrzahbGe2\\\\\\\/r55mh5gVLzfN0n42dfekFmZN2Vm\\\\\\\/apJsMpwna0eHg6QfF3Nk4x4WPNT6RAz5l87r532O52CbMhMljpddkkyJ\\\\\\\/YhB2h4jqFlIwhwvuHDdUj3rGpQgjmwf+0xFq3kG\\\\\\\/b8jimaRXkcQhnUDmduNkv67vu4ipeoJwjJLazOtvT1VyJEnTn6aH8WI64Y\\\\\\\/Zhq0KXIkTAoinDZoWQqe31x6vHZ+\\\\\\\/9Zh6nwY\\\\\\\/u8JquS\\\\\\\/aNa0iGLPc4QiSF9U0OO7uGH6DuZn+qtbUmgpHZqczS96+PW+AIGUFH8vEPDp6axJVZ3moRR+oRP5gpt7Y4MdArDX8IhsrwRXtqf1FXWxU8TzNZDSNICp2hELuL4qcgKimohWW+fUpXzzBM8syfEwq9vN3ddkHs6\\\\\\\/CJr9YpJiReIudhEoxUwzyhHJPu0tT8ltA1VBuX0gAR9QRHEhvr0Gt8ZdwI0Fs2MLn3tT1D6L3RzVrQho7YQBnmyg4D6DcVjjFa195ZGwa\\\\\\\/CAUhZvELcKm5mxxdEFM8cLSbnhAiffjmeXJBZji76oubDLl1NJFLeXY\\\\\\\/UmnmvMKTrHF+XymXwEbpS5+mlpIKf\\\\\\\/iTBj8ec2h0NxbL16uRWR2I3GXA3CtWEBpkwefA+U\\\\\\\/89SSiKU\\\\\\\/VoEx2dT8dsTFtQNr9DxoeJ4VoCZpexyCBA7euEdt0qiXywQc9EIViKz2js6YYQsDLMWMN\\\\\\\/BVrqMUa+eYAXasFWiE3PAi0T3eGjhTp+RCPTrhH704fBUYJlsuWs3vshkFEzBxOfjnAa2\\\\\\\/BOo2d4+QG6\\\\\\\/qIEpBe2xEe0smP\\\\\\\/Y6qRSsSbutjaa6r9dD02HqF2O0KG4sF6FxID440A\\\\\\\/Vw+7Wy+d5OghIxrzAiXPb6JeBYnblbxXZYfFRdAswrKJhhfgIVLECZMP2bODHvwPHc1PsSksmHeqncKmQ5BkC02h7+ivoBQOgOEj2sZ0QZPJM2NDV9M\\\\\\\/KV0OEzaBWPyxbYtUKX4Si6lO1rIKGSVp7tgM79LcEBkiWLzx6PcwNp3e7w7ddQtA0YyyUGEG5CUL38QM5aLr2yhUkGWxC2PhyXGS94HIw+Gnj3Vy4H22rPTT1VaIggLF4QA4T7Vf7r8\\\\\\\/fckjlzEbbOMRhONEJQDv1e6AC7tcXYOGxptDmF0SxXCgZnlg26SoH1M7jb1FXAzNrFWqIlgTHcQjAXETcSJbSiI1AEpS81oo3UjNY4nTpVgg3TNmiplsR3ySTTYiy9e2Bl3VGs1SN7NTxrSiggfcFXlSpTllHNPJPbuaT1dsalyGxj3MaKdmQqK\\\\\\\/QWBI8XKbwnA66GeQKOVlvIwDMJakMBb7x5aEy1UurSeKCCeMlr8uRHx6VRejcnNIKrAJU3ENtK2kLHbmx7+eAkXwTDx67mW0uZyXthFbIAleVZvb1dMSYnCpfkNUzXq8fgG2ZQ6Tn5veoNXkpzqybaDP1YDbvWBIsm\\\\\\\/SHwLv5xuG6xRVijXtEx\\\\\\\/FmT2jY1qR4+aL29A+nanWwEH10MA4ym7swBnDvr4dApMIw7dqSg3hwwYKZiXE3Q346eueNTM3Jm\\\\\\\/9W\\\\\\\/f8qK7jA0Y+uKGE3qFR2NDFCGcE4mLPCNeraZPbWi0jnaerWq91rCDXL+Mudy8lGH5I4cBCRnkI05OmQlrGotmr2WSx\\\\\\\/1pULWPgnzB8LkXY4Ho\\\\\\\/4V4t04z5DTMBP10PxoTUt1AI8Hl4ULKpHSryZEbR+s25WYa3JJxIHF4of3dlLe+EsRhNnXV7PU5HCkkllivYJ0X7thm0C6njxII+Pt\\\\\\\/qX+Gou+AKzN7nZdQ+v2gB6baqThmLAEipVmXuWc8C6uWRXowH9ylw8YZZezL8ek6pRfK1hyjtB2eP3QAVXLu706u3ydCRBIIKsxc32411\\\\\\\/Aq603y4+irACr2+m5ptJfIDRX43WovJ1M5O8EKN3t4DxAGGPRsia\\\\\\\/pO9kTaOT9+W\\\\\\\/QZfR50ETW9kDgRcU9Afp1ZSqInvScfIlE\\\\\\\/b4Vj2a7AgBuNTSjCTCmwvrKydx6Q2Iz7DzX80DDoJ0l2lz0JDquECUhr0WOPaWZwEFvDfHdKe2dUWf5FKnnGTWAmCbPrue1UzMfnG+HCTvyCpQKwHy5PHd6ArxEEyirLv+M0rJFZFVa8ghBb7bFt0ZAVRH+GAv4+\\\\\\\/jRapjKGRQ0viBYacvQDL2zl2jCzLSyEZbDvlF452b4rAgq\\\\\\\/6zPovGtL4quhWHzflNx3Ri5ZhK8HqW9ELxLcH26FMCVqb4CE60Ll8PkiGBRQ8Ud8ghZlh9fPrqJlZHdUYF8SC23Uk+92Bm4LlqJ5Ga0n1u7pQmtwi1ucUEJeDlIJsoI+ExWBjGwDfFUBX16u\\\\\\\/tmXGGEbc0KqVInlcl\\\\\\\/kL09RbRvx97k8zhficHm7ek9P6OsP4xMnuIEEA4YouYa2SZJNCVUU9xoT2mhjci6Mb2g3x5qh2qS8\\\\\\\/vJ68\\\\\\\/PYQ1FX+gy8xqnAfKND1P\\\\\\\/uhF1q8v0RJ1neLyT4oE8v62E9YiS9kHuV+vyggwXxQVPZ2Q0zKzC9CjjY0G86q2z25QHRPun2FA+4YJ9lFPLvHhKGn9aEnVTuWEJvIDgcxlU\\\\\\\/gQbbP2LG6V4EQQacHI\\\\\\\/FCuYAak\\\\\\\/uV2H2uCByjVALbN8B6CRvcAgwuiF18VNVK7w0tMggQM+4sYANTic7AUJoWYbACD\\\\\\\/5dC5HuQcgoVA2\\\\\\\/CvfBkzKaFmioP9Nrf9q3cQFebm8dJ0Fas9IfNSu5Ar3K0TZxZZuW+ClUgl3sRu+J\\\\\\\/HC09C57BnMc0xGQaz+joq8awKD\\\\\\\/47soeapIJulEm4OiJ7Odhs6M1uAk\\\\\\\/fjWhonAJ2L1Zx\\\\\\\/OzhQ6SkoSNAjRwIxk6RTuPC55iBN+17GoihWbh08aiCBtAdUrG+WXem4mTQ2ZTfn9X3lU+NwHSibjr7RgTA\\\\\\\/1SM49S94AJ\\\\\\\/07FVJxFNmCq3ezFsq3UsY7wJrhl88jfKbzRrzSa8XTTeN0PVAr1y0bEZhKJZfzDQah4Ipq59cfpgsccNEmCrIzXUM0VKaUXF1QMTexKIo5gus4YrAgr3HwWGvXDe2U2uamu8Uh3I08bDNKvQzVQZJDARDG8YJr1A9Z5JstPQkiULcanrHYR2tFei+RDmxbNTIyrT4wLdf4bJ6ABeM2SJZMiKnn82CUtXjHwd3yhBsdjPoqA77aAAmUR9tJUf6lFTO4fsfnV8amOEXDsfN4k9tP6YYZGpz4+U3jvlGeS0uvJ0wVpOh7jk93Y9he225lznQcuyNYgUWPJCU3ai7bu6iOZI52SgKIFxZwg2v+zYbiuePPUj7KDDTcGL2xlGCBpegN3SCkseuCdC\\\\\\\/RjF\\\\\\\/CG3E+MvI+wmT7RqgiXI9Gos6DDqacMGpjEhRy6aLw8F25jwrkWjg3MuReYgAU9ccGWvVKE80X1NMjQrgA+bOkQtdqHiofDVb1bSaTZzyHBVXZijwxMiLthhZ2E4eMlSbW07o1V4j6HNSRdt5OWnpGCAMj5UYaesI4nI42VbW\\\\\\\/333HHgXiaV7lz4nu2ASvF+xbJijcEqchu78G338xzFQHr9qicjeFNVbU\\\\\\\/aG+Gw4j+D\\\\\\\/MHtTAzsSnI4E5yZvugldl7bNoJCg6aVR3\\\\\\\/9peGZ293J6gmxrJRerCxARXQSs9O4FM7lse2Q36OaknpahSTwAnJEfKT7feV83NwNt5Yy2q\\\\\\\/aCH4TROm7k\\\\\\\/8EVQnGB+ICE7hkmaPcxyb00mUGxLmBNLvNA+JF4wnHV5IoPSoVK2yCjZkqgpYnRb62Rs3UsTbnAn\\\\\\\/GDMJfr2YEv5EXBgluFNJCc4RNDghcuk1GI79lfR2GdBhfJUQu0LcHOFrH4Okk07DViAhX4+bA+eOg1dtBFW4XjOSQMllEip+HCJPQ52Iz8GqElJPVjklqxO2GXIqChV3r3u52DRa2bM11AAhSC+oLHqnOCldcdXGR25I04dYc3XfrXymyNvBUpNNjOSVuFtKbBQDsA\\\\\\\/SVg1qUm\\\\\\\/F9i6qQpB50dvcvX3vMLxXhKZ0GS3ZoU8DO4rRiIPTwlB119XL+ttXdSj93YOLLBaH+FNoNxzr69VA9fxylIFsxO7Uj3tmVF1LJ\\\\\\\/b4Js0U+3JgqO3M\\\\\\\/T5JKNc9wk4dyOlebdC4\\\\\\\/hUNEsTcIN3NOgczbhYWSKS8qOYb\\\\\\\/P0htS4Whtxe6C7Zz\\\\\\\/LzJIcXHlfy86Hk\\\\\\\/n0YdkQceRupq6mbZ4s0AdzTYe+WQH+iQFJQAEmJhtm3Q26Tt7UgK9vsp0thdYMV3nu9VezfWEECssvrJcu4LcWRxIXVVTHl1eTm6FDbas0YcoUHPzbKp1qhNfq1d5As6F\\\\\\\/VJPEZeX\\\\\\\/CN7X4BdIqqWMsKaErGD\\\\\\\/ZIfJa6xna5JcmDHV76ZcMFntx68HJpp4vwXoWkhLiR02WVrcf+I9pu0RaMG1F6MIRXth3F0CJ0p5udx4z5226rQK6y04A8YXUf9hP1WDFV9uHVRPOStZyPD1hyc4p6pT3b5cDI\\\\\\\/zAokqtz4AwfOvsfYIsndKrbxPTurrYb0\\\\\\\/vkTB5XlOWRWvo1AzN4fshoGRrPP+BydfeEy5BfafRT22Jedsywn2uwP39LBIR86b2Lb7nhOn63HXLphQU2gqFZI40lRfzsCPh+fjMiBglPaGGroyAnkxmSTz+Uj6BSJM0Bqq3cjFOhnP6wycXQ7N+6GrNRt+dzr0rh7Z+tr\\\\\\\/FFKBn8T2IGZMX4AEijnleXdIvDQMs0iS7Bis13Qc5umZfK2k6sIeJNaUVgXcRDSRrrfUqVDkyb1yBvaBg8PL3kw+xBc8VxA2IeXPGid96GeC2ZUMfp+otT3bqFGnBRmiktASaSsxTQAgDsM2MR9gpUKIj1dLQhKXyrAcNJ0kKd0hIaGk3\\\\\\\/QOjhQwjF+NYxlNKkAhxWj6bgzrvw2NfopPYHHkyeQV0VGxbJkI\\\\\\\/VCU3Ampqf0GYU7NZJVuiKwAk0J88sx7WqLWNJyqxzsC2HXlMaowixDV8YalVc7zIQQ4rlHFjt9Z+Zo2EuhQa5S\\\\\\\/NclEBsB1R4QRx2VYn7R2laph5UfaTuV1PHp2bp0aWnce+JDFtXIPcrRtkq3Bwxiz79D01AHTjUYzXA8sUT1jYNFYYQ6f97Zous3nC\\\\\\\/2MSyHulHw2a1z3deT5\\\\\\\/jnEIUsRRPnOY+ssQ36ZZAi2\\\\\\\/Cm5PS4HGIZ5vuP4dNeizCRBn9gTEi0DVKdGJ40HH+6hcSwlq501Q8imEDk7C3gEzg5J8RNDrkv3AQNB1\\\\\\\/xBXZIBBwKh5rjGnfd40OASx8cgVdSAUWeZxbb0KzkqkOqJXb3PUGgfPYddAuiOjvj4ANPScRH0co1cMTDcLUACMIqwqKdFSZh0OevKSU5wJVQmT0DvsY2+QW1EhKX\\\\\\\/mwIfqP+f5s96D4BRPlx8H6vtiPxKZU3OCkCkO3z+gHnvqu6eF759xc3B+L7IfpPcXagSzSIOYlPne3pYDJ43NOPNN8EONSUqcCbEVU8RCm37CjJ8imNA5GEv+\\\\\\\/4JwkZcdQvBP3WEko6QDj9XoS7wFZ8vb3lFqKhOPGR5IUYP7KY+2SwMdC6B\\\\\\\/Qp\\\\\\\/o8NIJ6rUY744VVjLY\\\\\\\/d3Rd7T83QtRWFi0Sgdz1IxJ06l4I4mgUqOdZPJwv6E\\\\\\\/mIY4C6EPoYaS\\\\\\\/ugikNRsyFa53gqq0OfKrWYXaB92LF9MREq\\\\\\\/gNO6xgeSmoYfDg3w4IUG\\\\\\\/PxaQqsY5Dqg2qPWJWZv4t1R\\\\\\\/QIjQxvskiBFcYyPWmyD\\\\\\\/f8Uy1zlrTturdGUXGyaEDrjGcHeCJif0SuS56KLFifuuoDQ2uYRZqNvMbQbzCPd0mkSgJkZZO9jKYgR5zVGYqrTH+b4CjargzKWgnjdqKoHYmTClqjCcVVsudgCZYjJOKlRkxfP\\\\\\\/xqlKTL3Jx3AzEmeApkghUQyPgaLXyzfArrSisJfR6TIlAUW+oEcMlt\\\\\\\/A93x+p+kk9qvT7P88YQBuFnUkzsrCp\\\\\\\/nKedMl3DNkASkLc40Q29JbpvfSO0DOeB7m1PZgDfqV5rZY+hEg6UenQKOLS8S15eylyzRP3Tar+QfD6zFXxRg3seYZi+2o1ZUz8h2g\\\\\\\/fylxSZisYd16RZcUxCSU1k2vEUrcn9wEicAwlzdNguwpvBMmhIDn4PiY5e4Ma\\\\\\\/RBpbfY7fSQskZaX+JqY3Q8OBnrZtwVf9biDGleCkp+sSVPovu3qvd28XIitG3+M4VfNv+oQAkgj4RaJMnfuBh4pNEkWNPBxdQtYNQzINwkWDFFhnE+7f2mSaYxdXPSirDWL+PUV+K8\\\\\\\/9ZojT53VxQOHQFUZaO4kkYzsOlJVvcm5mNVxotDHBkMxv\\\\\\\/\\\\\\\/LC3Gu0JJpJEbXGPlTJXCGUmelRrppaW4ssMglrdP6Ec7O17BwGepqyfrasa12moYLThZvNlpjW17F88EJpphJ5uM6qr\\\\\\\/mO3NXH1Fw4y9k8UcXCaCIMroT9m1MZh7ZewHsvxgqhZ4p6xxGBYlqEtPg7ApeUBACmjOcEo6dkwy1gh+oUEyBeFn3st0Xwb05k+uO2RiLXkPC31pPvkTsbttLEgTXfFZax63GPS+EeAmwR0C0dQBGWTf9XDtrQo5QaysLB7y9jtcHEyTprLuHGBjGRxAa995MnaXuqnSQEQEPxtsGlTHkXsJPNSr896kmHM\\\\\\\/czsxGXWp58X8rHd7oficFFJ9P+aXsG2XZmXl1cahJ0PFwu3a3HL2UUZMzQu2vyzMsxDczVzMv\\\\\\\/UT0HL1H6nMKzuy5zKqJsfKZTO7RovRuv\\\\\\\/GCdgAm6Qots40pGhzUx4MWShqRIaZ4MFau\\\\\\\/JZLpwmLN+yYuGntNk5veGRE9raTDFrawsMo23JUzYx+jMXAy8HSJz6t57kw\\\\\\\/7NDPbPWM89hJfORgxx1eLPH+iJ4m6yUNk6mttvKTJZNHj2CSVqkJWo48AzqXOwLsY\\\\\\\/eD6bUmFkfBmiP+wkyyhn2GZhK4H\\\\\\\/ggVQz1RvRcgAav7HVOAUMyno8\\\\\\\/llyoSAfP2FPuGb+y1LAZN8b4MYJqqJLt0RbOx75AIfPIPphxZ8hAF9uL+yRX8BDnSXLm2gGa1grAtITcB0+Vsin933BZKBWUlvc1gxGQQIOaxg3AbCJsnF4FjgcgSo33kd8hh00yVkuEhp7sWqLr7QmkIj1UFLMGiE63SoKT6uam0ynhMM3SxKI+J3TYjySYuSHx6EFvtdHIRKbVliEZhIgcKxkFW\\\\\\\/FU5Oa748\\\\\\\/cwO5IP3\\\\\\\/y71AWv1av+8UWjoue3EcNFK6jqzCc3\\\\\\\/CCmIo5bIinD5sJp9i\\\\\\\/LVcBWkF15Dz90DKx7v2Cx5dsVNX2GIJfxwYzaPgJew9eG+JaRkzz3er3iFpj70\\\\\\\/ayzkTKLAZMH4TaoAh44aGp7FZx2tOLJemaweS9IYb4UcdY+zStmAvOMUcjvVfq1O+SQrrnRVZsiGJ+pGVXo2A3cWLACCfCp\\\\\\\/zgDe9YFjrBYyphCI1VEtR\\\\\\\/FEKt7QQdWChuYqmJpPcsLlzbwrBHWcN+yu7fn9CGfbGneX\\\\\\\/b6dTcI+TnuEyRvSkN5ycdg+4+Gcntv83TD5l7tpHyKLfdts5QOa7+E6Tjgg876H7qYH5SjbYJL9OOvgtXxzLYb6ORVidqGcRq0EBKZw0IO66stK55jK2fuNXhOlBKBKBgJMWqUWGG8siuG4MYrES6IGhmoutO\\\\\\\/92f0AIPj5+JDR0rINDVoOg99Pe0\\\\\\\/BStwpKUwpcSZO6t+xWt2VrsrnyvT5+9ZI4JmAIjbwiSHzgg1JLasr1cxEugTsdS4ZM8soXLQ8KI1KJtZh5Lfim\\\\\\\/Z4ItEb5aec0Fda69VmmclZUmkucYqaWwJmMmxQw7BPGa33raN8Ale54z10c17ugeSv+H\\\\\\\/Xb5c7NjlayZOdEg7Ew+Ozk11Gy4ymRSM9I1mTwOu\\\\\\\/5bC5R+hQuCAtEcPjavFqSovF9XO8\\\\\\\/EKP\\\\\\\/DLSmAXWa1SlSSaCoMgJc95zMPO4sDWL63dxU9HSzKpgKQz62CgP76i8y8OY607cdk9DyJmaoNM\\\\\\\/6p8kN2ZHwV5rf8WJ7IhmZ8phbJTEbLAFRk78IUF746Zkz2KX8gARW\\\\\\\/HeP+ZAEbDhSy2UJkF8eTA7qvJ6V\\\\\\\/FZ3AcUOfJefM8yAGlaSPAOnuAOYOpGSZ3DD5HpMm5tvgNWxfepNcWVsEEDAzMigoQoksbltY\\\\\\\/9JPYnGJLGI67r15qxsScUe6ajFsWwcBXwWG5x5dBk7O8AC8CGnhAkPK3ccL\\\\\\\/xe5U47uoV55SfUSz\\\\\\\/SXlz0Rtl06aJuGrm0zYdf1zjq12OgaB+radYRb\\\\\\\/HqgdidNNMH2cqQYOnOl+UAftSF3Cx6\\\\\\\/gi68+p9NNC7YCfE6i8IXvH1vfRsTcKXujZF4VtkFmMlnAwUgp1DuMuv10iL6upY7gXCEM6i5ZaUF\\\\\\\/BcmMNNW\\\\\\\/W7ugPphVNKSACwqUUhp3aEnm3Hkb2tF047CgTIZUJSR3CIegCv+5FyPyYAnc+X8wp6MFol551fln2GQKCTkzeNDxidGTlLZlLZSXMk2EQ1aRdalk7l6sx+VdkLK8MGJ9ZoGzCIjgOPDXMf1TWifDPFP8e2WiiQYzEepeIlx73N2cwLDQovky9IZk6pNAuc4cfYfi3CjwVHf7F63oychtGQ4xgNYFgY7mYYU\\\\\\\/Vfz2d8GQOJs3AxOjf54wOmRguMJ8imgOzJhPaFi9KyVR46NLk9wsSCMn+CUJZnM0OhdAycPcz2YdcaVp7nUa3iMFzB07ZfCF5HZieCMdlyTN0xB5OytY+lqJ8QscrNNpRFVDgu3VuIQ2qtA2qf9A6kt4G0HGkgcZcWtdz76whthD3WxGqatlpncQ0\\\\\\\/J1xQ\\\\\\\/xBjqOkjr5CUBmF9Q3nUyAuzkldjekAh3t3BjpHXgJFJsO09gHBRD9d\\\\\\\/G0HLc7DcqW4W5aExiX0jCwBz\\\\\\\/UsO3\\\\\\\/mVAn9b1hL0MMr5uhQMi77mmfGoh5e0V3dTaMar8cdci8EBb7rb6rAqpZsRxmeUY4um5yb7YBNoOtPXA5mMgiIWyMsT3R0\\\\\\\/CYHdXWqsvFIGLUvoAmd0Nf4TH6DWhhSOVctKZdMnZPpvAB\\\\\\\/NoZ1D1r94ZuxsXvwkwuCw\\\\\\\/Cz5UO1kd0joFv95vPb3HvYxShDsvvIW2L9mtH3MAUDMK1sV+DT1ZN+6icrvrFTUIBRV6KWlCg9h0LDPz4bHaYu5FKsOvRoS7TChMlFR3BwUbcXLh3O\\\\\\\/ivqXvMwv7DmJlI3Nh8K2iMKsNkrhf\\\\\\\/vJYOu7f9cUsdpam5ELJd4LA1NHAXAdxOaIT79rgbzVtJhPB4WVV\\\\\\\/Gu\\\\\\\/y5rcLyKDs3u3xuqhM9u3NTO5OP4vlAd8WvsezunXWHxrxz4Aza+ivx5oQsNSvqSqvzEiz9o3in2Z1vJp9hcQ8Li2kgrocv1PGdHWu1Gh0y9lS+JhyjMQPmYcVxUbwY3AY5XdSp6mNn+l0Fm+s8kG0GQZJu6TiJ\\\\\\\/IwiDiLMCjuSfDB1G98UVyG86C6J1TEDZvJlVdxGLOykvUuXunxIi5brWfPmtgydM7h0b52GVJwOtx9HnhX5RGDmCs5eCExIf9758koNWKkmFeKWaeiWo7VS2Oid3pmDXXB0NwqzeNk2yZv25qvJ4u107MOiNAnQ2FWiFcCwSAmoCJc5CMt9ejrKgOT87zn8v1BY0vZmCEAr32m90x5iwpX6AA8WMG7Wb1Z5EnJKzL1o009rPsdJP0OfsVo0IjkYxHNAC0Az3JrvRCM0cEYn59cTPSeQ8WvbKw6xcnHBvmDQY3RB+9wtXNufydDVEBpcwN7fDWfkmtXyE1SjFSbsQa2rbyfzjgkSugcZA3ip9EAr65qRJR6PIKjc8zqY0Ie9T3\\\\\\\/3vBK2coTrCis5mwzWSYIILsjjg2M9wu0sirVJa+b497n3YymIhMPTWe0z1R5lgB\\\\\\\/c\\\\\\\/mQ3H\\\\\\\/Eomqcd5G9k\\\\\\\/4AMr2qsXFwY3xu428ahnKUDjkxdVI+FE6DxFXdSDm7yBeiIfQ6qACOv+VwVtdSGWmxfuNfIXE4v9yw+xetjo5XHYRk0V7dZEiy0w7fq7Br\\\\\\\/GWZ1uABU4Qg5JO0i37NOaIEGJvUzZPjbGAPCD5e5DxoYbgwy4fHYK\\\\\\\/nVa1okwiKYistvtSn\\\\\\\/3AWrS9saiLA7tTnnOIxeeRSTSmJbU9y1qiB9CIaw5fAN0sQyIXOWj3FEr7WiCMFxxkBU1vIpduJgWu6BawK6WAJc5zX7RGrJ80BcaSJafe8GsSn46qFVEQotcuMznlDIxS2Vd1lWzvZgKlxnr9G1gxMNF8CEDwreFj5iR+Wsw+queV8N7\\\\\\\/ACgoT4mGyeb\\\\\\\/WoN4rVj4aehycRPsf7vrrRpSi2rdJkJbDtj9CpGAjQu5EbDPBt3H33oU6wRGPlBibuFpEWNuzX+bQP3eEGwh7lol+yUJnC5HcPtoV4g3jE3Xy6Ub1r0QjvbtmFUAsqJnTMcockTMPME3b0RYPOgZr9ea5FPLznsOm3EjCVXY6gKfHAhpsIjiD6PHbwtkEk2X55sO7wNYsY9B9tpsF3q0yKvA1tUCvQZpdOyLTajKIBbykd21uyfj6BvwSqH2mia1s1jrmC50Uu6WyzPQAh\\\\\\\/\\\\\\\/KwJhDP4I3ABR9Lx8JFkq2\\\\\\\/4e0K1N0nkTVXb5gF2Qnnk3qS+e8DQRNclH6iZ1C0Al41+1zyxhlP5UMCw2Wh59QzQdI7v1rjGcRj2zlKbn9\\\\\\\/zVMj7PJVOGUGdDrV8ElRg81eEVQJPaXy2dBI5NFc8MSQs2gm2ZlCrrI1J8HS4L5fyg4y62VjrLETzvk0SCNhXaGFwt9qylWla5TpBGmLHwYvD7YV5iV15ulw+peyBgltYRfLwAM0y5dQVSyQolVzKA5gSnedKbTTqiKzsxRc7KQ2XR7mctxp9hfEvXsqCDXbO2Rhd8UMahqnSMl9lq6nDF018SYagwTQWleJVlnNYNwnQ11JJQ7ZO0OA1HUud6n\\\\\\\/ptMdLWE1UvI+pI9EnMMF4vvLbP9Ze\\\\\\\/xttSHvxcx6YPbe1C3KG+k03clX4FHlbmI0zFZsDvIg8BtghDnJHTfGEMnsTYoQkqQYg8PeIF0RzRltw391qvNlR2B2Nu6dak4K8RyGdtCZKp2WUKdVoilLlNHob69MO+ED5RMBtFtyg69zD6KhkVq6KpL3QM1wMusy3b6PxMd+WGYyuG0VIQvhOgMsYahSOawX6Sj+69WLoPv2RzpfDCLYCYoIQZkcTG5cneMwyFxs+jCnUHxGS7uU12NuL6XNFdveQivNu+219dhTRjFDayHMekqe+8yNJPTiQw5DHmAVHsh9DH3iFkfN\\\\\\\/d1Fu1lQL+YzM9H\\\\\\\/yclxJmEqtIKWpqFzwERgw7JdoneAzQYGX1d4XxJjlDJPFxINyWC7IGYFvQl5PvPuHOiYiwepHf46MbxSw51JVy5poW9DfTxJDQS9OmN0p24KFV\\\\\\\/wczOA8vibDBAnN44dIHikApVkvym2J3C0iLhUJ\\\\\\\/WDBGbOLUAerLlUWNI0KAq41CwJCCSNA5Kit+zg8glxJD4bD0xnVW5Cle1te\\\\\\\/+rfYw1AczXp4qoAS8KLqBZmT1DF6uIzh8bFT2CmxJ++6chFF0QPSkBZ+aEGDhqexTZQnNMyUptdSu\\\\\\\/yZoLSPbjMael5TLH+9+U+YqakQTgw5GLSwU0LO8QYDAncIXp8F6NHwiAQtDgnClW0QWqiTq7Wi\\\\\\\/9fh6+lsPJ50MUckvN+uvx1yyncnCB\\\\\\\/X\\\\\\\/5U2xPCXIfH+OQSY28s+w2JPLu7Nd1fiyc1oh7K4nX5UEmmEriw8G20ZLhlL7opIEWRjybKEi+K6UAco0LkpG7klhE\\\\\\\/aToXk5jvfkXwyYu7P\\\\\\\/ZrwKraFh0uMeseHPvtRMqCvQiBMRk+TKukQn2ZueR\\\\\\\/QDS9J1gdy4qmTgGXZEMsTKobro8aW1AKUlC8lijscEQkjVDnMj14eTplSNQ0cwwkWYTgv8aG9Ngtp1SdCoatetrc8TjNEEZ5lCyU6M6fTjfJ+RN\\\\\\\/i24hACPxdKqNsHo1JDGy3tSbtS4j6yRWhA3Ei1NP9AsEMC\\\\\\\/ghudNCsI2jsmLcgraaR\\\\\\\/0h0bAcNHWD9+Wh0iAFdriRXTCDvJLNMMyMhASJEGFHyJvDDEDSLfGFVHvKdomS6CpHAcgV8vV4PK1f8bSyNtbXrdHQpHVAR\\\\\\\/URc6FhW+CU9tXm8dGbb5AP\\\\\\\/C+ApLslKW62Vuyb2JSFMrohALUTx53MWIwDrpgZBqh5doNoQw7A3hcwgj\\\\\\\/KoQd5wiUgay3PhSEOOpOcakjyWxN0i5XKlhKFe8xKIDTk8sm+P38R1a4Av7w5x3b5wh7xC80r0NP+wh1UlqZ4US3+91qbFEv7H2r6Kwf8xwe0AAFLZMd8MjwbnsSmABuYZyBNyWUQQU0W7Fx2ckw0tNMJH+nNmc8rDXFEciYYKd010hgHMggt+fzqMohxGJPG1H9Rg5p2CQ54LXrBEuMFcrb7KcuvyIdzZ2mF1+xV2fKXcQ+XOtlSmQ7wkAeZVwS8t6+uotrxaA70OhjkbhXFcsi1a7gh8JOdM+yko0\\\\\\\/lYx3JhwTfHJhxnvdCh7vTO6oVgy6h4wp2gIpkooc04IretVucq1yxBsLtdaxaK0fXuL3VjSTF550oH52qCtuTKh4ujon3gVb+8SwZkNP8xvc+HeslKmHCWGukkhULYnKszBJAneWtqR942PcuDxmZiGM1CWcRxsQdD8nCDoG\\\\\\\/Y6I\\\\\\\/s2itamNJVfc4N04g5gyfXEdXnG+WpL\\\\\\\/MyO0GPzfsuoBBM5aXpucrMCCN3Kb\\\\\\\/XUzoKKeKq9lIXiDdezDHw8d2e6s4fROAGzPzdqt+Vv7jegQIeAkFQ4LTZyhGhs4ErWwddbWLbuWn5iY2+rw\\\\\\\/uyPplS1YdpatOtKNIfmMst5GV\\\\\\\/aTDtXxst3miWb\\\\\\\/kuxIUiPbRiPh6kZ5XrdYEO4FDacXchsZtwc9oDGCsPTf0E9qArdV01+B0XM0ahKNAJblmyxEhYpzscltEvs1hSg1oBWBo7DzvDA1PK1WgNfZv2g48AJqXhxleeI8MYGg3im0Zb\\\\\\\/w2gHBJe2175bTPEDwYTzWSNljpUaTFBIOde6Ifhy79wzQDDtS8WZkMnS7Bx\\\\\\\/p3sYTyfUt01TXHAx1RU4kFTHmkalh14EJa4rEeTyoqJanvS2ibufD7Cd1iE1CND5uav+X18F6MW2Xvd5eA9rWJ7c1YDKMVWaXCiKCT+WYHJ9OtrRgco8Ik0UtXbUbQr0GZM6qo2QlL7MbNf8yO1gAPlU6YN0\\\\\\\/xscwcXbWoxTB+IsokaIsdDtE8WGY3bCfGb9F7ifwr2NNXWot9BJ7zLnymWtBz0Do93Z7n\\\\\\\/ju+q+FzRRTYyJ6vWcIVJrCSw\\\\\\\/0rxtgu9Jb9vtlR0Nx2YWkYVHWlmR++iKfkC+vxVGa1DZaDI359oKTcoVNC7y77os66KRi+KZrsEVWNqulxfv\\\\\\\/7\\\\\\\/iB1\\\\\\\/ipmI+6lfCBeb9sSkj6ZyL1hCbqQboveaz+sQa2jvNNRrRGUqPNBEqLlKZsyu15wBRSWURKRxBU5S60XWMmACCfY9t6dCpumKefmnGerK3xU\\\\\\\/F8GDE9ISD0w5v8KTl531i6OZvtmTu9sunoQesaVgHs7s8jfWjxBsuZSORay4LRP8s3KsXHgsx+YKj61qySUqju8HMCbL1aIKXq5jcX1iZ\\\\\\\/Xv0GFfdBsiVd5harRMWjVRrxlAgJQ1PZuQOGVImRxFjryisACTc9P8nAh\\\\\\\/erDmYvWSN8qgatg6taxFNuQIlgXENIcSXhyxaVLEiQUqs3iDBK4B\\\\\\\/krxNnx2dWlOXwzdMpvcyBAC93XxASLuY\\\\\\\/jR7jl4ZWv73\\\\\\\/zcmRngtXyAbLD2EFPGgB7IenORB4mdrhUgzxhpq8y7ATuOP2j8cKnR+H7EkZxnbffHvPFDIKSeoXAOMV2hEwuzrTXKJqxQMjUP099npF7ZDDObfI+hvDANMROz3s6peswdKh5A0m+JYkBLs7IM+gYr8J3160FLXR9JmunxhJiQiU4CevEqI1cJqbAkZ683wfwY3PH+wRXe0bEQypH4wSmVpmF5jZmS8PkiznHWYSsO9EHqNzBYrbdZmjRZRNAkwyGMwUz8VzbIwOrHYqjWjNmSK4w8ghddmpZVBZaSZEF5EXK99H6BD32uyt\\\\\\\/4rov0kevvptzK8vw9zkQ2GNMnGLdGOAkx9V2DFseTTCL+YRg1vaRnBJtlrDK9yHQjszvkKJ4dDMrNGzaa+6zD4YV3kid5SaQ6KVuqTVX7oPnvPxfb7Zwoo1j4\\\\\\\/mjwwCto4swaH7xX0ogZL5RkpXXfhw9Ebd2pRGj95+hHewWaIuhE+cZ5Hvt\\\\\\\/OfD0A4Pce9U2OiNT5cl11nv1PSnIjkvOHw61CRsAUt1\\\\\\\/5pfCtgdwEREAr8CJoqGMvYP1xWA0KtBd+owN0\\\\\\\/JWlx5\\\\\\\/cnJIU6DKE1feih+VMMRSosLkkvmC4B66\\\\\\\/qQpPbNK6Zi21pS33a9ZmMauxFdOhpyMbxazw9Gr9UwTBALEGWHfvafaWspLPZqVeDs1jz7wQFIuIAqPXxgU8tB7XhTQGBUQy7z89DkTP97ZAU\\\\\\\/sBqNarNIvj86vuRbXkl42HxPAD4jQ67hwBxJnHvHP8JI9nyo+9C4fBoSunhIR6IRoPi3OqUmax4DOD8dzT38SNLic\\\\\\\/FdYLWmUYpr780XNSPGbl3HDOKdxhIVsRL+958AbS6YEKD3XXuigZaE5i5kZ+cNcc6zmKaq0Xna8VmX\\\\\\\/MUqpSVcltQIpu1SxHavkLlTa3m3uO9hd81rPUJpe5tLv0cr\\\\\\\/zkFQoP\\\\\\\/6TKRiWG402GqHywvACXd9VRFHLaKPhPSfn46bpEJKPswtVfztHbUDKx9dU7+BzghieEQJQjV5wvouWy4kKqrJwDGHPbrorO\\\\\\\/\\\\\\\/IlSoTZN2sOXVsTApqDsm6xnx24walCPxjqwYcarmMDGIHoeNq6AkFQ2aYHdj\\\\\\\/DSCLjdwI5sn4fw3UHzVjM1m9eK99h3Y7yFRDTZyPGCAqJ3DUH4rmQ6IuGi4yHPGTYFWeTq+DLfKlMt3odw51fLP6msPh3bp07azqlPe\\\\\\\/btFenpaseock4clWD+bIIEYWdEtWNW6lj4Nq76bsm4HRN487kMqcARXcKPl4cabK5Y8j\\\\\\\/MIa9rTqwHoRVNusLvmOqXrK321Xixr97V1MvhE6D6n+a7MM2GDEL1JoMqlQuN\\\\\\\/P9wWiHHaDiexVmd5m4enbMvHqQ0Z4sSdHaaFuZAaVOMEez6CZOn+I3svVW32kFMtVc19+cQOOzNpNCq5HwO8PGEhMOtGzYmEpgVKWvjjAfyiQnxxoqrU5TnrNlpFhDhTp81QDSnOfF22HEj8yQ2qtXdrXcGrkYgj37\\\\\\\/0ByIM1lQlpLS2QIshrsKJ+tdf0DuCS3ilmrfAmLVy3hFXwAGNk54OBg9Sl3sFgqXUW5k8ROmNI5SE2sT1KpZWYHKffqDtUfiCRnHqlgsr9AFryW051bvQ7m9KXImOdP3gakKD+rsMNSJG4dfKoMzXaMFVNeXAjeyYJdZYomYZKsFCjdADq3hiUOjC5lEBwx64Ei+vj1\\\\\\\/z1HYJhBYHKJrQ8qQRn\\\\\\\/kx1NheXDx\\\\\\\/Rdqzu7P1nAFUJOZMViM6QuuVUiUP569TH29ca23KzlRT4GOrcyPzqLBbbUPITEL7Nt5lSXACyWeCCEz5pf\\\\\\\/IO6EqOXMXojlMF0xT88JIL5uTqNFawzbsK7eMeg8FiaZApskPfqH9BTme8\\\\\\\/6qMWb5Br5ov8iTg9mDS1ziPf7zdZ18nbk\\\\\\\/ygeSFiXvBf23JKCABvG9m3xX6zNlZLCGGEFKnnflQICl0epPzfxPBGzALsvnOGqM4nx\\\\\\\/5hZ2Q\\\\\\\/Vg+fq21AQedAbmQiowB38MpZ1AhCFcaALLqtsssIUFySgOKP0c2bmDh0bTGO5uE+UbPXfrTEI1YBJZmGmLsnwmYcsAgeZdSELgft5T4EIz9r35fQAfr+4WB4SuPFFZuF9wpokNpgA\\\\\\\/xpeoZuSKtjew4xQUCZbYlgGiK9CgAeR6li5071yBrFA5aZN0kfGPIleyKu5fbm\\\\\\\/LDTMQjqwc1ZsQr01HY8hnClO3ah5eicHqFS\\\\\\\/EP7L8BUtonwHi+4kSxqg3X49RxbP8yRlymH9Yfqx03gDapbdv1ngenrxTOJ5p7llKwevL9C\\\\\\\/yobbRbX66liQUkG9Maqoz9qebyL4v6deyTwGuHfiZ+A1Fo+lL1qzULlawaTnk7UKpBkhoetgjSBCpVGRcDk+\\\\\\\/NQghrqGAWMNJqnqVcy3votLLiFMXxGMJ\\\\\\\/Rwo6ees64XD0f6gFgwrGzM2IVPRQT1tjsBcPxX1CkcE2OZsYl7Egmcc2MjVFuD\\\\\\\/C79Nd7x0sLCFS8KwbcXJm+ImO0CvX3xSgyDHNresYjJoGjBQWvD+gW\\\\\\\/lsmuAubpouV8Y8JE0Lvrsj3c0HLsWcIAq3Unn2H1W92ww6RnD+KArtk0Ck9j9dvrCdOWkRUtzWKreCpXidIJhssZDyHVu6Ktz1z81x4wFZ6Crakov6XjF+wSYqa1iKEaDYFXrwpufMsfwwba30NehHFPNlAvwe0pclHVO5Zn9VyuILDtORObvMfD766HR5F85gNf\\\\\\\/L\\\\\\\/cQQgnxJ0fVgrZfmOt3+8C1rPLujN5OMSwcGfKT1uCzmuhDmF\\\\\\\/h2iNbj04k7Jev5vkAXzEmuE22cVacvdQRga8vcJAcDZ+7rwTc7zPf3Gh3zZfjh+NeLuNd\\\\\\\/ZE+K9pn1nS9JUWtqkfp4b6+LMQG59B2ThcoyjT2llnYNckCdHTZ9YO9Vmr2sHCFqCz92B7Ngs1MOaxNm75DymZBCYD0zORaE2O1RxhSNLTRG93e4OvZv\\\\\\\/c3DaWNRNCA9PvICQxuFqe5bkWzrE5Z\\\\\\\/bt\\\\\\\/Mw\\\\\\\/h0PWgYwGVsDeobx2TeLnx3dDrAv9LKxdphwb2H0mwIlM\\\\\\\/ky5D9Vg8ZGvnsJtbCCs2inTDCf03Iz+sVM5R60HB6MZ9KAJfHQZ+guPxLIYzSN4B7S3bQ5NmUvVeqg3KrUiVYr+2dHgMq6a+7EiX0zU4DUmkJr+1ydStR\\\\\\\/utV0WkcRWAlpSFfTGgCoteXNSjMkU6bUecJcnmE5+60IFEkFWuoS8r32uqHNf2iFXywcfOlX2pVB6J1riwdss7DkpLoH+43vGLRrluhDR6fKlVbEvv6vWI5QztOMtRNr4sy6+ke\\\\\\\/NT6tW5r+kpkt2WmlPcGVumL8G0jR90ek1JkQP3bIjzbf7N9OqLBI9f10f2r2aIYJk1zKcVkR\\\\\\\/oGneN6dw\\\\\\\/0vVwd9IL0O5yRfGskZIY1hzrTFPiUah84HLoD+q\\\\\\\/ZuClBV3o0oh25O9xXbXUY1CQXd2SxgStVGmrMvPsa9j2xDLxvnnugcXRdZAg23qclV6GDe5pSSOvwTKrBlbsybi7SDnd\\\\\\\/p76yA\\\\\\\/dnyksjtXlqwRZWMCI\\\\\\\/VELoeLI5zZh3CKCEvGezLni2XKrPNZfrAwUCYpudMORbURiWjwWuFCJz0xMxzKTFbmMGNs9RJrNNtr24Gg0AUKR95V\\\\\\\/lV15F2l94kJaFBkeqtepfOAi1xJN2oxeaWRn0SeheVvTw7o7XagHC8bodrXsCuMyNpU77qK0TxSkA2cPKi+QSSUsnvaoETm41P8tYIRVZbLXPJ+TpZvWL8ke28uXSzNyRDYPjnPq\\\\\\\/UqcfXHxuCuECYLWv0nqUVjj168wVi9dhIadawpWWkoB868zLxtlofvQpiuymv7XegB4I8fvn5ldRyJOB4sgNI6FvU9rNhwtGqjI6E9HuLxw+l4VAi24Mm\\\\\\\/xbNXUXv35EjAKezqcOQn234cLwCz9eh7GUcfoRYsR5zcUXCUqpkc47nl+eqmtH3DqmQIS9JftI5T\\\\\\\/mB9TVIOK+xguoN1xjvn94yK5ip4WhwVeTg7yHBaInW69M6LNVrHsNCp\\\\\\\/y+3uAelJxE\\\\\\\/wUiPhUQrtWtQ6Xw4Xz1WeOJok3qTeDSQRcFY+0QLaBGw\\\\\\\/yqvQkpwuq\\\\\\\/j+sr\\\\\\\/YxASrLElGIEaswUncXvmJWauFiLOux+wUpkfGwlNA5eCEdY5gO9iDLflDAieAtoPnKo3gfE2om0Wyze+oStMxHkEVmyfuClhYxBxozoN5kO3kivqqM5Ih7UtQHDybyMiZ++UKdjBSYtd0L7GQwszn4ZeEPsDMkxzm9BWJE0dMWDP0b1FN89BIOiEVV84q8u+YApEZRZhAhms9HI1sVk4IADSYps3Vaa\\\\\\\/ojfO8Y8\\\\\\\/LeCxPrP8N70KhGHdwda0bnOliDtouM6Wk2qwx3TVrw48cGO4B5mXAc2DxSWF2dzyYoSW63ZGIIaZU0+ow1JD34J+niiAJu5hL9\\\\\\\/DPmJW4QUloOMsxaQRiqlEF0y\\\\\\\/7w9w\\\\\\\/HpnFTCtTMP91qcoxXF5mbN2VhwVY9BZ4Ww0W7rSCPTEAP39DlC0T8T8Aa17i9kbOY6Quqv4\\\\\\\/fswgCkR57HAjKz9\\\\\\\/E0JCYzFjQMNaHihz\\\\\\\/g5QN59pKvmg1D8uOhE2STYl\\\\\\\/L5P6S9oQFBVZwcQGJobxLDl1CZUic0\\\\\\\/DfcdUEpGYnBmjykLiomndw+\\\\\\\/RsoH0KEM5tLLTikUNB\\\\\\\/HbYhKevIFvTLqQaM2c0IvMYOmP+Kt9M2tbkwa6UJKc+zXAHKMKp+eDHAD7NSvhHk2xkpNs6i1eCEhi9Nq8qOIHJQAypQNT0i5cudR81rIxkU2AYwCEpzB7uX2fhWUd1vhg4DbJUbsoqIlBlCGKcmln\\\\\\\/IkKF5xIhTJXNdIALIK0EY3s0g5iGTxmRoRwa5RpYXYGIWZJZWwYjBgZ9dXjTtx40EkYunTAhEXzgsMw95XImlRWzWjMaJEZUtM\\\\\\\/0j9WUVXhqXV8VuWpGMna5M9JVIfFmNj2ymr2I1rL9mEQGv+ThqOjxyrtJ+jAhqxW6Ih81kCUV0sc7romAxn8Wq8IVtBOovl7Tftp3+8n79FiUK1qnc9Te3gwoLGtmkT+swQhDIaXqYXO7Q7N5B+NHZ1OLSvmgtS2WyxSpVuncPg\\\\\\\/UU5A87uG4DFSIQ7T39V8g6g7WGojPJdWjbFb0koHIWCO4utGy6CPECP2gGi2koeQrHUCaevwlHaRXubewcSxvxeHNOVu+jMEkrZRlAVK3XrrwKN46RcIUHeNkvG7J7b4WUciS7rSc71\\\\\\\/bNHc\\\\\\\/fQ\\\\\\\/2dwp0fp0iR0FHAqZk9BKUF5s19GkNtAnLc1m0\\\\\\\/tcxl4whFaEEElDIm2Q3VaJPhX3pIuZ+C\\\\\\\/Dmczk4aLyHkgkgAnWqAVciVXp+PIn7loUaKorrk90TMzsWeuOCm6QUY8lPU6ofFq8W4nqBY6Rb51W7HiEC8nBCKms\\\\\\\/URlB20rxhAZHX387T8rDM61xDGp5nPEo9gBearTgsYMHOENr\\\\\\\/T6f0XQTAJsK1VOH4ZJQ4cdJqUxEC8yo5Rrubl+YUEqH1\\\\\\\/\\\\\\\/rzpS5DoEM5q8+PibNfhFV+FFbKVHgDoZRuiQuh+TLLk6K8NnjMmfy3OXTMN9ktycaj+GwGOeb7HmYvESi6zL7AmSL9JzfLQUbYNRdAok9krCPZXnMZIjKpm1LanVkrkTaPkMoJ9c8nHOY7b+H58X0GnKi4LSvRGBQKD+QhM0I8Gt9IvbAVr\\\\\\\/EKdXGCcsGu+\\\\\\\/KXWJiRxPxp1l\\\\\\\/SHvGeNIvUxLlyF5EejVsToDQYJoveigtFeH5YHpjBNf2HXegBYBGEeluYgggcKZtmjcEOyjjNcplnhFX39+sXAjxOwnP3UH9v5XRd+pxfDvQxj4NmZPIjI5aTF2piWMZ39s5Qp5BL8fzRlv3cShXSdquuMg+f0t9wZImmu9QR689SKRekA8nuFOBqzxKtNL8bXN0lNd87nANDJK9R0wAXkiR\\\\\\\/NMfwLgW4B94L1UzlSuYNCdUa5Hq7MPMVHt\\\\\\\/kk+nIYKfsfu\\\\\\\/GAGZYEMU70+JfnZA37k2\\\\\\\/nICLKG7X\\\\\\\/3O8\\\\\\\/bPDAomlYG9bSQUtHTgob7hNoKCDZC2m6AbTVUBJ2WkSC7UxxQa9Trg13j++bh9i7lkPdYB97E+lKrFSuQICCL4JxpYR+Q6dQ2REa3HumyaIYJEtQwtxpgkbtlz1LvYtJyQKSF0pAu0liNIbj0PL1JGZ6L1j+7OpEV3pLdL3j1mnCsSNrnnZ909sEthyEpRjnDkxqSSRkXIa9I2HyOU7wKT3WYs4j1uOXM5++0fbQ1ItpKexh7xbHB5NKvV24UrtbE4KBpgWZbcQy+azL6g5K0b7W3jfiOa7mVjlAIY7\\\\\\\/Z3KTTI+p+H+At73Nh3MoWgARH0riQdGVBEivwJ9Y1R1e\\\\\\\/G+v3dt8FqiVsKBbjkxb69h9CBjjwHkSdH5urld4N4YCBYDPCkT\\\\\\\/p7\\\\\\\/xSCCpOOmaMb6QJBU2fdPo1ZM2pXiapD+U6jQfr2cEGp9CnCdBDhQPvImZrfkCbZOrz5ps89wXGoAzZmPiGyHV4Fp2heoVvhazH1TL8TrzmNM7QDvjVa3dQLtVmqW+JT30ZGkBz5R\\\\\\\/GeVHOjy2XEOwEDBpn4kO03FoxBAiIeXqtZsh7WQOw4zZb3alp9Q9dQ\\\\\\\/gG8LfGVE72w+oVVKxfQWR5hjHrBI3zyT2ygI40Dtw8cGAKtwJhFddlnSiIn0dfZjqENiVYkw+JKjVej3bBsRlX+DFO8Vn8lIdYs1jFnR6LErpHApeJua5ByaBpU4YHyxC8z61Ub7clXQNcaJ\\\\\\\/AK50pYNwE+5FNbc+O8\\\\\\\/9MKfrML1jUUr\\\\\\\/AZjpt05zxHouMGW01wJnlyAdUeYhBhZxgKVuiJ8YZNkliGs5WVJVDmKKxKohxoh1XdDwiv6JtF0cHyQWTRJ0EBFq1zmdguoALz7EvFOHqZt\\\\\\\/kw8p2mVY\\\\\\\/sISslfQtLkKee7Bg\\\\\\\/ve3NjHJWTcGODhji1C8Z56pBzQaZ5rccAiJZOMHucZMAn\\\\\\\/R9U2R5HTfRDc7grLFF\\\\\\\/aGr6MGGk0rF1zD9rQl8LVZrcBoP\\\\\\\/CZFsuuT42wgUpH27tQ8WIC712gANJYXBjSmnQgEnKmzKPj6bbF7HeZldZheiQ2ckztRDazSuWM8dV8f83b8672ppCFeTuFTX3E2mTHMbpkbFHfluGeHYn+ZXnys+zkWu8zqGFA7SMTndF+Lh0i+q9fkd2yi39cOm3SWTGDIexgkRA+wwsOVzpXr2+h9Pd0AA+HNmmEvjkMIjLtDHvC8SzUQ81\\\\\\\/asD5YAcha2Cd6IR\\\\\\\/JRi3x2ACIL1SXThuICharDWOI74Uyi989TsYsc8eWirr7SpbzkqgwjNxJfreSNXiT5TAmtkuilbX74pYaO\\\\\\\/NpLmkOWeFFbjHqSDReuWnczihi2ma5uPz5bfoFy8A7PzWEYtqihb7v+lPEtJI4bg2\\\\\\\/yup5C3sQZYzkQsjnYuiwIRlyFVX4aa677zbG7MZ3S5\\\\\\\/t65+zr0O88fD2hfV66lOM0wBVjqLQElzVAUycBfgFE5nfFRyhUBXfm+IRkaq\\\\\\\/gr3dpydCPQvnJJRRvSKDiPJqSXBSr0uBELjFqiDwIKMTCxEksiuX0qD6b869VrkoD9AJuQrr6KOoeDlAClSLDXI+bQLwT7DC1JxvgYuIl6HevW5OFit74nvlYwnNz5EEljfjIBNG0eKBCWCM2OWLxGLb9ZZe9EMJNOt6G\\\\\\\/2OzoMmyPcI8rVqu2hIpDPuTGcDmlSvybUO7khJnEZ\\\\\\\/06vBO\\\\\\\/Vu9SLB4uSlE9cnDdCIgNGCLouzFpZsCzST\\\\\\\/E2Kh3VtPq7aE+KJ8paZ1ToIaGLtqkIlEx7w\\\\\\\/ds4TqwFr8poy+DIhLg+JZBawspfk81yo+gNCpt7bELzgNLVUCsynXObsVW+QaZPxuMaBiFrzR9GEjtqZc189MdSbPtXAg6w\\\\\\\/+ydUAyWN9KIKs8ZiZdq9SGn9UKzUkyKw3AFMDTLxKC7Jc+kpkUUoy9+mjTPPTArJfpMvgzOinCMONztUaZi11RH\\\\\\\/bZBmKpzPYHy9f+8i0azqlynoJn1VNruHQgGI4EAzjmbNQDjxCsKO3fPl1Htjvbzjbzr6cmGfRmEICTCBjhG0ooMRU2MzmBDV7UVqeue+g\\\\\\\/jtszantiH89gRuplYgCuyyP37M2QUtaK8tiKhBjsSHnF64k7\\\\\\\/DLVZGZGLDMFRrXlXsS5KA6Hkp8HjLDfLWbZG2\\\\\\\/XUClF+N2JcQ7o\\\\\\\/01doW6kmZLUscjEAxzI0o7GPmuR5C10+gaEDdOlAi8pea3krO6yus0I2IEnvkAxlDA+a5ds1FRnungN0QEXdRrY30MF\\\\\\\/qwx6rnzrlr0OAbDYAGEgN+sBLC7qlRWWLGORIeCqIyQeC3B+sn0rw1kaO9s\\\\\\\/VYfSUfStAe6AlCVz04Z2haDL9nw6A3TvXclOBfBeYVA6pqmF7ATv+tcYMS61e4U7SLJqZ+wFylk2fSPtd+Xzrw5ae\\\\\\\/em6h1YInjSDBcrGLVD4WFZ4qyQWWfH4rh1OCKm5mT9He2NsZ6tm6sn1nyjuO3lvlQGXWjOBp4SSy\\\\\\\/lYifMy5umwqJDqx7HwZhhRhRzdgRQvYuG6eLyNDq5hEPpi\\\\\\\/wYIYl+7\\\\\\\/EtF6398qx+K7eqsdsGykpLlOWEygEM6EFxIOsiJmPPOQfj5\\\\\\\/4uIOn8QnY4I7vhZ+ygcgp8P02xjfyAkn7gGE+PsXIrt8NRDijlmeoGoVBw0PVgujSbF3w5heDE8Kbhqr7yD9XCOCX8ctIp7FnU4jy6ahRKWaYEG758d3Jp4NzJod6BJbjaFO7nr67DgHV2GVB633zu3IFdnFriKICuu8m4rwjn\\\\\\\/RX8pjBhyr\\\\\\\/hGDRi5UcSXh4RM2RSzWOT0egvvoJ3I4oI7TTTuNdyerBJxs2BgckVU0Y6YpiPue\\\\\\\/yDfsDGbDE6sJUFLEkQDrVu9tJgfylvLlRGpVJx3YJn68YoGl+ZIl1RzCxSepDylPNFGSc9KqckfWtQg\\\\\\\/xAxrgoki0iLu9RPu0tkL5BSxmcroiqyJm1Borrsq2mw53yLD6OgNhbOeeOP3mkRwAeGDQNr6eWsLblDlmNEOtreHke0GriyQZYSX2REip2qRv6fZS2Z\\\\\\\/glLUUiqBjun1cMxpWjQqr4dvVsNSM54wjpTusSMoXyc4a7XVQoS1CGv+LFFeLWKDKByGsc6cA4Ednm85bwcB7zBb1XycHcKAmzjgnEXzg30U\\\\\\\/bdXs4hIjsopAKQ\\\\\\\/hH53W37A6NMMN7P0PfyiwSgicut\\\\\\\/V0c7iTnXZ91m4NFRVg5u1fTj8X1hcvW8lANWR\\\\\\\/dNJfOINY15DsezBRooCvoDyMMgMrgC1FD4TdL1edBy3XWCHAOECM8iGFC+anOdPclc7bhwyuBEvQYvy0k8XpGNp0W\\\\\\\/8X4\\\\\\\/d5IXDjs+i+LaeVSOYVo55J4wQz9dXyiECaBUFFSRAPAfxGJTuh0QN2RDZv+BbXdBwArzHg2lr4c0KRqsxZ+e\\\\\\\/N7SDGwUSF78v6BC\\\\\\\/pXU7WNbHboPKDyjhVj3Np98P3RBMFQn66k3IjZ1C2LoZ6a0WzLg1HDkbJtDOmEv640QyX42ew\\\\\\\/Io4vk7Q+tkzPB2N+np7Xxtz9xuMq0BB4EnCSh4LyEW4tIsc84NK422YWaaPNkZJFiRRtYdkaWNuEtLh3ozYpSOUEO4j47iZLnao\\\\\\\/mcTdpbVI3\\\\\\\/O1A09r8ayMAlmBJPJRoHOv5FD039illV+CAgtwWJ8zT\\\\\\\/dY3AD5DBTjeuyQ3d7bRjyKSz9X6FGIBN89tjboGwFjexO5nzpWzhB\\\\\\\/IlrQkhDSCQ2ZcXAZMIK6PKiXktuNZOf\\\\\\\/1ojzwwohg07RO8zBvt\\\\\\\/IMYKjnmoRnOU0kPmfanq84OCNLQoVpP5UjlIYJVY18Kl53Aj6v2z\\\\\\\/eiZ8tXjLt\\\\\\\/Srr0M+5Lp7Lve4bwBxev3rXKYq7\\\\\\\/CsaFT6CZ9mMWFtKmS3ObxiqsQihh2xGWReuGLBg1ghP3u0e5UdXL7t9YZLMC3Cj\\\\\\\/1MieXDzfAQmndCe\\\\\\\/pMm1BLT+O0wzLQyT2PrLme50MLZFuS3dKK2dkvvdX4kISdsS0j892SOd5nMPLFw6lL+QKkchnlwGDvUV2aqhFKIo4XIf0QGaVlhSjkWBgY641LQRQLYG5x4JSezLnS22lZII9+A8gGDs3V0hqthtYTfSJLfJnDnnRA0KUaYznPzSj56GHj8+F\\\\\\\/kFvT4xU975jSLekv4wjxlzlZvLJM3F6ZBNOYxlHRm1DQy6x5b0UYz5pRASV7IraizfstjCxEeDXoQRWSvl0lzOWeC37o9hBpZXjLA2H7V\\\\\\\/BIt3YCx+6VbB74yUXphq5zDwDfLJ5WvRBXIXMtCzCOvgCqW+fOjX1toHsJTWvWWpmO6IGjsBksP6VtzrUWNP71F2e4ORCPs5jY6WEoV4bbgBEntz1B354SEN+yL8p\\\\\\\/zB0FojdklM\\\\\\\/SRBLPsyVpozI+\\\\\\\/4CJVWqKxEGJ2COI+MPlsE9jLQHlJnCSNDuDU4ok4HT9XKELkjAFiATE5P6qLZE+\\\\\\\/SNEns4NmHB6amJ3+O5BLYgntqlVABh6sZUi4fp8HxcfMmk2AxhomUPNg+qsHmE9ORhUM9giPb+peM2pY3vcWiM0iMVYwz4tUcqRTcx7RE5NDcTnDqZrwAnm5l4avdtlQaktMgKcKiNrec\\\\\\\/K2KrRtPkLTXOKWUOVx1WZXiClRblIKW\\\\\\\/2gZsTvp3GKI0x8ZidJJX\\\\\\\/9bkt2uOJ\\\\\\\/8OLogGehhzJdoPgRq3k\\\\\\\/lmqlqmRpcsVtxoMYAunugeotOHOXYaGDksHSEiVQ4E32R1+vjYzXjpvwlPEcowI6gKe3APY4u2a6OxfXew\\\\\\\/du8EN84nrJrWkhHI9VxDzBYTf0kkYq+0ZhHEK5+HJRbpT5xvA5DIeYWZVFsYPp3nl5i11sIi\\\\\\\/m3o4P6y5YzEWjx7wkDu1a\\\\\\\/3SwOA558d0\\\\\\\/A97Bne9ZERPD3sm5lhV2aIhMEVwxwHMTuI59B\\\\\\\/pc4xRh98wHVUHIZUcdkUyIpce4eSP+XBt03FgQ5b99VMvVI4zwMl4Dcdg\\\\\\\/8b3VyWYY6vGEb6tH94dx7o4+FTxYRx\\\\\\\/dZMYCDAVIRJ20si7pEpeltwGbC45rjR6gYHtgE14foGzIEaSt3GYTl4CDIsTPegBVSA8pB4sjkvO69cbM0ky2lroOnsxUrg43LOJxoPTnwMVEPSXFp8r1CKf01IxpIISkr7LO4XmzJXHvWh4m9FBa5h5yxMwMXKdw981Uk4wXgRZq8W+a5muYe9i4Oh1HIuCBzw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WGMc5Gp+YPJoFqUx0TN1Ag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2b05abbb3bf487eedbcce6a46dbf2920204d0591e503fd3b5e34639b534d248\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JOvpqJDpB+XY8s1vnlc\\\\\\\/B6SIYNxWhXLMwuVY8qo6kCNev+EgeHWcTKa6ABxR+\\\\\\\/fLydC2WUHVYf94v1fw4OdDIRBHLibRbsDYuv7GmMVCRgwOv0PRmPfGPY1QDFcBGDSsHnWBxelnNrBLcXChPUFWW0UMVq\\\\\\\/FzVLUrPiZCu7btcOT2EwtCPpkfEVV6Eq3eq4cImxyHas8DD022JNWE\\\\\\\/KpBX0OAnZpNtiChnjRx4BwWfOoeH63qxS8hqiLR+zfyo+2qy3Py04lPe8t7WutI8\\\\\\\/98TSk4XyRKTHWK62tjGeHRk1bULVYwShh7LeIdUG+ArzEg8WGQaHIEadcEQTxapnOBC8oaKAww8OTcacz0r8gXwjlxrIENuyzeJf\\\\\\\/rulSb0AFeMB53NAIwGseLbgFj7sC2zkxe+wUSE2xKNsu153QGzC0uGSQQqf0UR1cDKdkht6ePRaUnLCbKliM8pC5fTgAUlhFDA77AoJu7R5DqqfTcCXfyXWlAVN7sbcwonvdJM+zxy7TlQFK6AVwqqCWGKTKZ19C17jVXY1vnuFF7r04Mg3yvolvFI5lWdRNeFOqZK2sLj6IBINo5OMqoJTdgZoLY1\\\\\\\/F7wT5HmDz6TeAITn7P5jFsOuS8tFptE6z5k866j2sNR2dUen\\\\\\\/djn1hU8pnJ\\\\\\\/n+9q2HWOs+RPsWs8AeinDsPYuIyf\\\\\\\/GU3RG2enVLYpD0z8WGzZ0oVGuHTJMll4QBPO02BWe08tYPMREJU3uVJdH0Qd32zuf8w1gqxuhCuI5tSrUa46sG\\\\\\\/MhqItzzN4EaeCOgYCPgkEvG3HFWKppVmR6JPQVaDvR5J0qvtxHgeNVOZS9+HcgdbZ40D6tJcYm+xsPvwEoueqOr7QQZJqNufiIF7zN2WBOyoH\\\\\\\/2wodslBknOWhaVlYs5PM5QzNL64OBrl8ijYVFbnhqfTSQVZed\\\\\\\/l9MaBPFBlxY0GqTzHxItH2Jn5atdCjGuzwPnX30P2z8tWvj2Dhnuki8t5Z1as7klVpLUeivn6r0THbJj3x6xPA2SzosxRITQKVl2qKbaztzMkryes9mZMWTfososCxJHt5C0kjeMlu0hHe4MjNC+eAUwEpFKSC\\\\\\\/vRX7DycAk0aMar3\\\\\\\/PvRffsfzqgUUSIM3+Mo3\\\\\\\/IqmpNkTgUItpRx30OOcvZz9rDu4cEnScFEVGnq1b\\\\\\\/3MDbmqZiTdPO42pM3dtWls+wH5TXyxvWylF2PAk0fCcDKTf5b11DhWM1pmC2oSHtlR4vLBrWhkbPCJU2hqjk5aLncIcxBJ6Ywj8L7NKDIWPmK2+upLmjxK\\\\\\\/NDGnBc5DuEKaNjZWfVYX5XGrgf7HpFegaMQI3W8k83iMnMigD73pdxx1spJPwkylxkSE5fIxyAu+HkOmv30hqcy4aYZ+WdpPjMfuys13XNZasTEWyt7QSLrj62SYWV2VZvogjLZylsfM0v8nGOBRmN4291cB7KIeq9YQoaS07mVNh34BGU3Y2Iz9tjp5p8PWorZwPsRcrzp7mz6TdTy2oB2EMP0yv7Bbv\\\\\\\/c26TxFxTFNTku4PvipWuXgIXBJWDNn0e6CBODzhACiLUIlczRp+yvbpO92b8+jhXHg2nBFUtCmA1vEvy\\\\\\\/yyzkRnqUJ1NDbd9w1TTB2noLQWm1ENlaD2BNfjgT8WmsfLoSJ6ZAAakEYy9QbslflP4OJj9+Pl\\\\\\\/TQWTe5MB3VTZFq7ymZEdqbbBpwiFRx54QdCc8EcTklFlcxOLgegTA7gDle74sq9XjvVUEBY5\\\\\\\/OaVEGt9kOyhZULZNMwjMgvWa3lgpFU+00mh5uNqXxygwXMevRunzewxSqBU69a\\\\\\\/JVsRmgsWNResEnFU4VSnkq7rlX0v4QSEmvpxjJox7YT7IlFondLJOw544fqILzwlbaz5zKRV2DLrWlCIFEWjd22iHLS6ydZfApdZO3cCEKdOcVauK0c2HcmeKEpJoiX7K7rIhwXOc9gWy\\\\\\\/H7ZerpX\\\\\\\/cNLC8iwlePREIyCfVonLirFtLO8pWlroACidTqLNvNXKVnQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00224YuPinYhNPWB+vGpQL0cbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227d1a9fdbc7f3763179800fba8890fd9f0eda960a4220f1e8b93aceda202f3ec1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459662 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1657" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JocO0JymKybdoEIFTBxEA5PPFpbkn9RH15WeAYAZtwc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1657" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8lfREyuPtcInFutIf0mtTvUsyidREXqnaCADbT9-9GA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1647" title="The Functional Significance of Common Polymorphisms in Zinc Finger Transcription Factors" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1667" title="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459662" id="node459662" data-pisa="ggg;4/9/1657" data-pisa-master="ggg;g3.114.012914" data-apath="/ggg/4/9/1657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher H. Chandler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1657-1665;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012914 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459662" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher H. Chandler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, SUNY Oswego, Oswego, New York 13126</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BH.%2BChandler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chandler%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BH.%2BChandler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christopher.chandler@oswego.edu" class="" data-icon-position="" data-hide-link-title="0">christopher.chandler@oswego.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1657" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459662" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459662/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1657.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459662" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459662/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1657.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459662" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459662/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1657.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459662" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459662/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1657.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459662" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459662/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Experimental evolution studies, coupled with new advances in DNA sequencing technology, have become a powerful tool for exploring how populations respond to selection at the genomic level. Recent experiments in microbes typically have found evidence for multiple novel mutations, which are usually fixed. In contrast, in animal model systems, evolutionary responses seem to involve more modest changes in the frequencies of pre-existing alleles, probably because these populations outcross and are usually initialized with greater levels of standing variation. In this experiment, I used whole-genome resequencing to estimate allele frequencies and look for novel substitutions in experimentally evolved populations of <em>Caenorhabditis elegans</em>. These populations were founded with a fixed pair of deleterious mutations introgressed into multiple wild genetic backgrounds and allowed to evolve for 50 generations with a mixed mating system. There is evidence for some recombination between ancestral haplotypes, but selective sweeps seem to have resulted in the fixation of large chromosomal segments throughout most of the genome. In addition, a few new mutations were detected. Simulations suggest that strong selection and low outcrossing rates are likely explanations for the observed outcomes, consistent with earlier work showing large fitness increases in these populations over 50 generations. These results also show clear parallels to population genetic patterns in <em>C. elegans</em> in nature: recent selective sweeps, high linkage disequilibrium, and low effective recombination rates. Thus, the genomic consequences of selection depend heavily on the biology of the organism in question, including its mating system and levels of genetic variation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/experimental-evolution" class="hw-term hw-article-keyword hw-article-keyword-experimental-evolution" rel="nofollow">experimental evolution</a></li><li class="kwd"><a href="/keyword/selective-sweep" class="hw-term hw-article-keyword hw-article-keyword-selective-sweep" rel="nofollow">selective sweep</a></li><li class="kwd"><a href="/keyword/compensatory-adaptation" class="hw-term hw-article-keyword hw-article-keyword-compensatory-adaptation" rel="nofollow">compensatory adaptation</a></li><li class="kwd"><a href="/keyword/sex-determination" class="hw-term hw-article-keyword hw-article-keyword-sex-determination" rel="nofollow">sex determination</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/tra-2" class="hw-term hw-article-keyword hw-article-keyword-tra-2" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">tra-2</em></a></li></ul><p id="p-4">Experimental evolution has long been a powerful tool for biologists (<em>e.g.</em>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Malmberg 1977</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rose 1984</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dodd 1989</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lenski <em>et al.</em> 1991</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rice 1992</a>). Only recently, however, has the widespread adoption of high-throughput DNA sequencing technology made “evolve-and-resequence” a feasible experimental strategy (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Turner <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Remolina <em>et al.</em> 2012</a>). In this approach, populations of a model organism are reared in the laboratory for a number of generations, usually using an experimentally manipulated selective agent. After experimental evolution, whole-genome resequencing is used to identify genetic loci responding to selection. This experimental design has the potential to shed light on a number of important evolutionary questions, including the repeatability of the genetic basis of evolutionary change and how selection affects variation at linked sites (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kawecki <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gray and Cutter 2014</a>).</p><p id="p-5">To date, evolve-and-resequence studies have largely fallen along two ends of a continuum. On one end, populations are initiated with a single isogenic progenitor, relying entirely on <em>de novo</em> mutations to drive adaptation, and typically lacking genetic recombination. This design is typical of microbial experiments (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charusanti <em>et al.</em> 2010</a>). Such studies, in general, have found a handful of novel substitutions in adapted populations, with mutations in similar genes or pathways often occurring convergently in independent experimental replicates (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Charusanti <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comas <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dettman <em>et al.</em> 2012</a>).</p><p id="p-6">On the other end of this continuum are studies initialized with a considerable degree of standing genetic variation and with obligately outcrossing populations, typically using animal systems (to date, mostly <em>Drosophila melanogaster</em>). These organisms typically also have larger genomes, potentially offering more mutational targets for adaptation, although the population mutation rate (N<sub>e</sub>μ) often is smaller in absolute magnitude because of the population sizes used. Although the repeatability of genetic changes is difficult to assess in these studies because they often identify candidate loci by looking for shared patterns across replicates, there usually are multiple loci showing a common signal (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Turner <em>et al.</em> 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhou <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Orozco-Terwengel <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Remolina <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Turner and Miller 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tobler <em>et al.</em> 2014</a>), suggesting that some degree of repeatability is common. However, in contrast to microbial studies, animal studies typically find polygenic responses involving ancestral alleles rather than novel mutations and fewer fixed sites. One notable exception among these animal studies involved the nematode worm <em>Caenorhabditis elegans</em>, in which a handful of novel substitutions became fixed, likely due to selective sweeps (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Estes and Lynch 2003</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Denver <em>et al.</em> 2010</a>). However, this study adopted a setup similar to microbial experiments, involving isogenic, self-fertilizing populations. Thus, the differences in outcomes between microbial and animal studies are likely explained by both their experimental designs and the biology of the study organisms.</p><p id="p-7">Although the two endpoints of this continuum may provide reasonable laboratory models for many organisms, there are of course others that fall somewhere in between. <em>C. elegans</em>, for instance, is androdioecious, with primarily self-fertilizing hermaphrodites and outcrossing males. In natural populations, male frequencies and outcrossing rates are low (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrière and Félix 2005</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2007</a>). Genetic diversity in this species is also low, and there are large blocks of strong linkage disequilibrium, even extending across separate chromosomes (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rockman and Kruglyak 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>). Patterns of genetic variation in natural populations provide evidence for recent and rapid selective sweeps (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>). In addition, effective population outcrossing rates estimated from population genetic data are orders of magnitude lower than those estimated directly from the occurrence of males and heterozygotes in nature, suggesting outbreeding depression or selection against recombinant genotypes, probably due to epistasis (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barrière and Félix 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dolgin <em>et al.</em> 2007</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Rockman and Kruglyak 2009</a>). (However, some laboratory studies suggest that outcrossing may facilitate adaptation to novel or changing environments, at least in some situations; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Morran <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Teotonio <em>et al.</em> 2012</a>). Based on these observations, selective sweeps extending across large chromosomal segments also might be expected to occur in androdioecious experimental populations of <em>C. elegans</em>, even in the presence of standing variation. If these sweeps involve ancestral variants rather than novel mutations, then repeatable outcomes are also predicted.</p><p id="p-8">In this work, I follow up on an earlier experimental evolution study in <em>C. elegans</em> to test these predictions. These populations were fixed for a pair of mutations rendering sex determination temperature-sensitive but introgressed into multiple wild genetic backgrounds, so they were founded with standing genetic variation (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). They also exhibited a mixed mating system, with some degree of facultative outcrossing (at least initially), allowing for some recombination between divergent haplotypes. Ten populations were evolved at each of two intermediate temperatures for 50 generations. In these environments, the temperature-sensitive mutant genotype exhibits dramatically deleterious effects: fecundity is reduced by up to 75%, with as many as half the worms in the founder populations being infertile or sterile (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chandler <em>et al.</em> 2009</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">2012</a>). Ancestral worms at these temperatures also exhibit obvious intersexuality in their tail morphology, which is sexually dimorphic in this species. After 50 generations of experimental evolution, however, fertility rates and tail morphology have recovered in some populations to near wild-type levels, suggesting that selection has favored compensatory alleles suppressing these deleterious mutant phenotypes. However, transcript levels of five key sex-determining genes cannot explain this fitness recovery: there are clear expression differences between wild-type and ancestral mutant worms, but expression patterns in evolved populations are overall much more similar to ancestral mutant worms than to wild-type worms (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). This finding suggests that other changes underlie these apparently compensatory adaptations. Genetic variants segregating among wild isolates are known to modulate the phenotypic effects of these mutations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chandler 2010</a>), and some of these modifier alleles may have become fixed; new mutations in these populations might contribute as well.</p><p id="p-9">Here, I use population-level, whole-genome resequencing in these populations to address the following questions: (i) to what degree have ancestral alleles become fixed, and how much recombination has occurred among these ancestral haplotypes? (ii) How similar are genomic patterns across experimental replicates? (iii) Is there evidence of any novel mutations? (iv) What selective scenarios best explain these outcomes? The results suggest that selection played an important role in generating the observed genomic outcomes, which here highly convergent across replicates and show similarities to evolve-and-resequence studies of isogenic microbial populations as well as outcrossing, genetically variable animal model systems. Moreover, the results also show clear parallels to patterns of genetic variation in natural populations of <em>C. elegans</em>—namely, strong selective sweeps with limited recombination among ancestral haplotypes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genome resequencing and analysis</h3><p id="p-10">A detailed description of the experimental evolution populations has been previously published (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). In summary, the <em>C. elegans</em> strain <a href="http://www.wormbase.org/db/get?name=CB5362;class=Strain">CB5362</a> <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>II; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em>X carries two mutations (in an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genetic background) that together result in an entirely XX population. These worms are hermaphrodites at cooler temperatures (&lt;13°) and males at warmer temperatures (&gt;24°). At the intermediate temperatures used in this experiment (16° and 18°), ancestral populations are a mix of males and hermaphrodites but with a high frequency of intersex phenotypes and strongly reduced fitness (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chandler <em>et al.</em> 2009</a>, <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">2012</a>). These mutations were introgressed into four additional wild-type genetic backgrounds (<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, <a href="http://www.wormbase.org/db/get?name=AB1;class=Strain">AB1</a>, <a href="http://www.wormbase.org/db/get?name=MY2;class=Strain">MY2</a>, and <a href="http://www.wormbase.org/db/get?name=JU258;class=Strain">JU258</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chandler 2010</a>; <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). Populations were founded with 10 hermaphrodites and 12 males with each ancestral genetic background; 10 replicate populations were evolved at 16° and another 10 at 18°. At generation 50 (g50), evolved populations displayed substantial phenotypic recovery (<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). Three populations showing strong evolutionary responses (18EE1, 18EE2, and 16EE6) were chosen for sequencing. In addition, I sequenced an ancestor sample consisting of DNA from pooled g1 worms of all three replicates.</p><p id="p-11">Frozen samples of each population were thawed and grown at a permissive temperature (13°) for three to four generations before DNA isolation. For each population, hundreds of worms were washed <em>en masse</em> from Petri dishes in sterile water. DNA was extracted from each sample using a QIAGEN DNeasy Blood &amp; Tissue Kit (QIAGEN, Valencia, CA) following the manufacturer’s instructions. The pooled ancestor sample and three evolved samples were submitted to the SUNY University at Buffalo Next-Generation Sequencing and Expression Analysis Core for sequencing on an Illumina HiSequation 2000 with single-end, 51-bp reads (1×51). In addition, I downloaded sequence data for wild-type strains <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, <a href="http://www.wormbase.org/db/get?name=AB1;class=Strain">AB1</a>, <a href="http://www.wormbase.org/db/get?name=MY2;class=Strain">MY2</a>, and <a href="http://www.wormbase.org/db/get?name=JU258;class=Strain">JU258</a> from the NCBI Sequence Read Archive (accession numbers SRX219150, SRX128707, SRX218993, SRX218998, and SRX218971); these were paired-end reads (2x77 and 2x101). I mapped sequence reads to the reference <em>C. elegans</em> genome (WormBase release WS235) using BWA v0.7.5a (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Durbin 2009</a>), allowing up to two mismatches for the 1×51 datasets, and using the default parameters with the BWA-MEM mapping algorithm for the other datasets. I called single-nucleotide polymorphisms (SNPs) using SAMTools v0.1.18 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>), excluding sites with coverage greater than 100x to avoid calling false SNPs in paralogous repeats and restricted further analyses to SNPs with a minimum quality score of 30.</p><p id="p-12">First, I wished to assess, for each position in the genome, which (if any) of the ancestral genetic backgrounds had become fixed. To do so, I identified SNP alleles unique to each of the five founder genetic backgrounds at nucleotide sites with at least 10× coverage in every background by using a custom R script. I then divided the genome into nonoverlapping 200-kb windows and calculated the average frequency of the alleles specific to each ancestral genetic background in each window, in each population.</p><p id="p-13">Next, I looked for evidence of fixed <em>de novo</em> mutations, by identifying nucleotide substitutions that were nearly fixed (&gt;90% frequency) in at least one evolved line but absent from the pooled ancestor and all the founder wild-type genetic backgrounds (and supported by at least 10× coverage in all cases).</p></div><div id="sec-3" class="subsection"><h3>Simulation analyses</h3><p id="p-14">I used simulations to assess whether the observed outcomes were consistent with various levels of selection, numbers of fitness quantitative trait loci (QTL), and outcrossing rates. The source code for the simulation is archived on Dryad (DOI: 10.5061/dryad.603d2).</p><p id="p-15">Simulations were written to reflect the design of the evolution experiment. In each run of the simulation, 20 populations were evolved for 50 generations each. The three populations with the greatest average fitness were selected and used to compute two test statistics: (i) <em>s</em>, the proportion of genomic positions in which the same allele was “dominant” (had the highest frequency) in all three populations; and (ii) <em>f</em>, the average proportion of sites fixed (in which the dominant allele has a frequency of at least 0.9) in each population. The expected distribution and 95% confidence intervals for each of these two statistics were computed using 200 replicate simulations for each set of parameter values (number of QTL, outcrossing rate, etc.). The observed values of these statistics (generated from the sequence data) were then compared with the expected distributions to generate a p-value for each scenario. A small p-value (&lt; 0.05) for a given scenario would indicate a poor fit for the observed data, whereas a larger p-value for both statistics would indicate that the parameter values are consistent with the experimental results.</p><p id="p-16">Each simulated population was initialized with 10 hermaphrodite and 12 male worms of each founder genetic background (<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>, <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, <a href="http://www.wormbase.org/db/get?name=AB1;class=Strain">AB1</a>, <a href="http://www.wormbase.org/db/get?name=MY2;class=Strain">MY2</a>, and <a href="http://www.wormbase.org/db/get?name=JU258;class=Strain">JU258</a>), and subsequently expanded to a population size of 1000, with nonoverlapping generations. Each worm in the simulation carried a diploid XX genome, composed of six chromosome pairs of 69−105 “windows” each (corresponding to the 200-kb windows used in the analysis of the actual evolved genomes). Alternative population sizes of 200, 500, and 2000 also were tested and gave similar results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/012914SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/TableS2.pdf">Table S2</a>).</p><p id="p-17">For each generation <em>n</em>, the relative fitness of each simulated worm in the population was first calculated based on its genotype. A value of 1.0 was assigned as the default fitness. This fitness value was multiplied by the fitness assigned to each of the QTL alleles carried by the worm (see below for details on modeling QTL). The worm’s phenotypic sex was then chosen randomly based on the user-specified outcrossing rate.</p><p id="p-18">Next, offspring genotypes for generation <em>n</em>+1 were generated one-by-one from the worm genotypes in generation <em>n</em>. To generate each offspring genotype, the first parent was selected randomly, weighted by relative fitness. If that parent was a hermaphrodite, the offspring genotype was generated by self-fertilization; if it was a male, a hermaphrodite mother was selected randomly, again weighted by relative fitness. Gamete genotypes were then generated by simulating meiosis. Each tetrad in the simulation underwent a single crossover during meiosis, mirroring the single crossover per chromosome pair observed in <em>C. elegans</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hillers and Villeneuve 2003</a>). After simulating crossing over, full gamete genotypes were obtained by randomly selecting a single chromatid for each of the six chromosome pairs making up the parental genotype. The complete offspring genotype was finally constructed by uniting the two gamete genotypes generated from the parent worms. This process was repeated until enough offspring genotypes had been generated to populate generation <em>n</em>+1. Again, in each replicate run of the simulation, this entire cycle was repeated for 50 generations in each of 20 populations, and the test statistics were computed for each of 200 replicates.</p><p id="p-19">I ran the simulation with various sets of parameter values to assess which could best explain the observed results. Although the simulation was implemented in C++ for speed, the number of variables precluded an exhaustive search of parameter space for an optimal model fit. Therefore, I searched for parameter values that could produce a reasonable fit as measured by the two test statistics, focusing on three key factors: outcrossing rate, maximum relative fitness, and number and distribution of fitness QTL. I tested five outcrossing rate scenarios: constant rates of 0.01, 0.05, and 0.10, and two variable rates (initially 0.05 or 0.10, dropping to 0.01 at generation 25).</p><p id="p-20">QTL were modeled by first selecting a value for the maximum relative fitness; five possible values for maximum relative fitness were tested: 1.1, 1.5, 2, 4, and 10. Then, for each maximum relative fitness value, various scenarios for the number and distribution of QTL were tested, based on the observed genotypes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/FileS1.pdf">File S1</a> for details on how models describing the locations of QTL were generated based on observed outcomes). Finally, fitness values for QTL alleles were assigned such that the optimal genotype (<em>i.e.</em>, a simulated worm homozygous for all high-fitness alleles) would have the maximum relative fitness assigned earlier. For example, if the maximum relative fitness was 2.0, with two diploid QTL with equal effects, then the fitness value for each QTL would be 2<sup>1/4</sup> ≈ 1.189; the fitness of the optimal genotype (say, <em>AABB</em> if <em>A</em> and <em>B</em> represent the high-fitness alleles at each locus) would therefore be 1.189<sup>4</sup> ≈ 2. I also tested a null model with no fitness QTL (<em>i.e.</em>, neutral evolution).</p><p id="p-21">Finally, in addition to the test statistics described previously, the simulation provided genome-wide allelic distributions for simulated evolved populations, so that the simulated outcomes could be qualitatively compared to the observed outcomes.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><div id="sec-5" class="subsection"><h3>Genomic patterns in experimental populations</h3><p id="p-22">I obtained a total of ~140 million single-end, 51-bp reads across the ancestral and evolved samples, resulting in at least ~9x genome-wide coverage for each population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/TableS1.pdf">Table S1</a>). By using the same analysis pipeline with the publicly available sequence data for strains <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, <a href="http://www.wormbase.org/db/get?name=AB1;class=Strain">AB1</a>, <a href="http://www.wormbase.org/db/get?name=MY2;class=Strain">MY2</a>, and <a href="http://www.wormbase.org/db/get?name=JU258;class=Strain">JU258</a>, I obtained ~30x coverage for each of the wild-type ancestral genetic backgrounds. Raw sequence files generated in this study are available from the NCBI Sequence Read Archive under accession numbers SRX658438, SRX658572, SRX658573, and SRX658574. Processed data and analysis scripts are archived in Dryad (DOI: 10.5061/dryad.603d2).</p><p id="p-23">Approximately 558,000 putative SNPs were detected, for an average density of one SNP every 180 bp. Of these, approximately 130,000 were specific to a single genetic background, for an average density of one diagnostic SNP per 750 bp, and 330 diagnostic SNPs per 200-kb window. Thus, power to detect heterozygosity within a given genomic window is fairly good because of the large number of diagnostic SNPs in each window, even though detecting heterozygosity at any individual SNP is low, given the modest sequencing depth. Indeed, substantial polymorphism was seen in the ancestor sample and on most of chromosome I in 16EE6. On the other hand, fixation or near-fixation of a single ancestral genetic background occurred throughout most of the genome in all the evolved lines; moreover, patterns of fixation were qualitatively similar in all three evolved lines (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The large segments of contiguous ancestral backgrounds fixed on each chromosome in the evolved lines (most spanning several megabases) suggest that the genomic window size of 200 kb used here was small enough to capture the recombination events that have occurred in these populations (<em>i.e</em>., a single window is extremely unlikely to span more than one recombination breakpoint).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1657/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequencies of alleles specific to each founder genetic background in (left) a pooled ancestral sample and in the three sequenced evolved populations, and in (right) simulated populations for various parameter values. Each point represents a 200-kb genomic window. See text and Table 2 for details on simulation models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-490173448" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequencies of alleles specific to each founder genetic background in (left) a pooled ancestral sample and in the three sequenced evolved populations, and in (right) simulated populations for various parameter values. Each point represents a 200-kb genomic window. See text and Table 2 for details on simulation models.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1657/F1.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1657/F1.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1657/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1657/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464880" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Frequencies of alleles specific to each founder genetic background in (left) a pooled ancestral sample and in the three sequenced evolved populations, and in (right) simulated populations for various parameter values. Each point represents a 200-kb genomic window. See text and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for details on simulation models.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">I found evidence for three fixed novel mutations in total: one coding mutation in line 16EE6 (a missense mutation in <em><a href="http://www.wormbase.org/db/get?name=nhl-2;class=Gene">nhl-2</a></em>), and one noncoding mutation each in lines 18EE1 and 18EE2 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The use of Bowtie2 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Langmead and Salzberg 2012</a>) as an alternative read aligner yielded nearly identical results for overall genomic composition and recovered the same three novel substitutions. Furthermore, all three mutations were confirmed using direct Sanger sequencing of PCR products (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/FileS2.pdf">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/TableS3.pdf">Table S3</a> for details). For the ancestral population, chromatograms only showed evidence of the reference allele, while only the mutant alleles were present in the respective evolved populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/FigureS2.pdf">Figure S2</a>). These results suggest that the mutant alleles are indeed absent from the ancestral population and fixed in the evolved populations, or at least that the frequency of the minor allele is below the detection limit in each case.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464967/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464967/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Candidate novel mutations in evolved lines</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Simulation results</h3><p id="p-27">Simulation analyses strongly rejected a null model of neutral evolution both quantitatively (<em>P</em> &lt; 0.01; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) and qualitatively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Models incorporating selection and low levels of outcrossing (1%), on the other hand, were sufficient to explain the shared patterns of fixation across the three replicate lines (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), and gave genomic patterns qualitatively similar to those observed (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/FigureS1.pdf">Figure S1</a>). Results from simulations with alternative population sizes (200, 500, and 2000) were consistent overall (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1/TableS2.pdf">Table S2</a>) in rejecting neutral evolution and supporting modest to large fitness gains for evolved worms, although greater outcrossing rates (5%) were also consistent with observed results in these cases.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464876/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464876/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Results of simulations testing whether various selective scenarios are consistent with genomic outcomes observed in evolved lines</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-7"><h2 class="">Discussion</h2><div id="sec-8" class="subsection"><h3>Patterns of fixation</h3><p id="p-29">Earlier work has shown strong phenotypic changes in these <em>C. elegans</em> populations over the course of 50 generations of experimental evolution. In particular, although the <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em> genotype these populations were founded with results in high rates of infertility and intersexuality at the rearing temperatures used in this experiment, both of these mutant phenotypes are largely suppressed in the evolved lines (<a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). The whole-genome resequencing data generated on this study shed light on the genomic changes that have accompanied this evolutionary response.</p><p id="p-30">The high degree of fixation and similar allelic compositions of these evolved lines supports an important role for selection in generating the observed genomic outcomes. Indeed, simulations strongly reject neutral explanations for the observed results, instead supporting a role for selection (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). These simulations also suggest that the outcrossing rate in these populations was probably low (1–5%) and that the overall relative fitness has at least doubled in these populations, consistent with the strongly deleterious effects of the <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em> genotype and the large fitness recovery observed in the actual populations after 50 generations (<a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). However, the simulations are not particularly informative with respect to the number of QTL; all four sets of QTL models produced similar outcomes for a given outcrossing rate and maximum fitness. Nevertheless, multiple loci are probably involved, as different genetic backgrounds became fixed in different parts of the genome in a repeatable manner (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-31">In some ways, these results are similar to evolve-and-resequence studies in other animal models (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Turner <em>et al.</em> 2011</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Zhou <em>et al.</em> 2011</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Remolina <em>et al.</em> 2012</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Turner and Miller 2012</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tobler <em>et al.</em> 2014</a>). For example, ancestral alleles, rather than new mutations, seem to have played an important role in the adaptive responses seen in these populations, based on the similarity of patterns of fixation across replicate populations (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). Indeed, evidence of just a single novel substitution was found in each population (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), fewer than most studies involving microbes and one other study in <em>C. elegans</em> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Denver <em>et al.</em> 2010</a>), although the sequencing approach used here could have missed some, such as substitutions in low-coverage regions or structural mutations. This paucity of new mutations may be explained by the presence of standing variation in the ancestral populations, which likely provided a rich pool of alleles for selection to act upon immediately. Indeed, a QTL mapping study showed that these genetic backgrounds harbor multiple polymorphisms modulating the effects of the <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em> genotype used in this experiment (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chandler 2010</a>). However, these populations also evolved for fewer generations than most microbial experimental evolution studies (<em>e.g.</em>, <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Charusanti <em>et al.</em> 2010</a>), which may have limited the opportunity for new mutations to accumulate.</p><p id="p-32">In other aspects, on the other hand, these results resemble those of microbial studies. For instance, near-complete fixation, rather than moderate changes in allele frequencies, occurred throughout almost the entire genome in these populations (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). In addition, although some recombination among ancestral haplotypes did occur, entire chromosomal segments became fixed, suggesting that outcrossing rates in these populations were low, which was supported by simulations (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), although segregating inversions could potentially explain this result too (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Tobler <em>et al.</em> 2014</a>).</p><p id="p-33">These populations also display interesting differences from other studies looking at genetic diversity in experimentally evolved <em>C. elegans</em> populations. For instance, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chelo and Teotonio (2013)</a> found much greater levels of diversity after 100 generations of experimental evolution in a novel environment. These differences can likely be explained by several factors. First, outcrossing rates in those populations were much higher (~20%; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Teotonio <em>et al.</em> 2012</a>). The lower outcrossing rates in my populations were likely caused by the mutant <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em> genotype, resulting in low-fitness males at these temperatures, and evolved populations are almost exclusively hermaphrodites (<a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">Chandler <em>et al.</em> 2012</a>). In addition, the populations used by <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chelo and Teotonio (2013)</a> were initialized with variation from 16 wild genetic isolates using a more involved funnel cross, whereas my populations were established simply by placing several worms of each of five wild genetic backgrounds together in a single cross. The more controlled crossing scheme in their study therefore probably generated a much larger number of recombinant haplotypes at the very start of the experiment. Finally, the populations of <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chelo and Teotonio (2013)</a> also probably experienced weaker selection (selection coefficients around 2%, whereas fitness has probably at least doubled in this study). The stronger selection in this study is probably attributable to the deleterious effects of the mutant genotype fixed in these populations.</p><p id="p-34">On the other hand, there are intriguing similarities to patterns of diversity seen in natural populations of <em>C. elegans</em>. Recent studies have found small numbers of haplotypes and strong linkage disequilibrium in wild isolates of <em>C. elegans</em> (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Rockman and Kruglyak 2009</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>). These observations provide evidence for recent global selective sweeps, that outcrossing rates are low (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Barrière and Félix 2005</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">2007</a>), and that selection against recombinant haplotypes drives effective population outcrossing rates even lower (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Barrière and Félix 2007</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Rockman and Kruglyak 2009</a>). Similarly, the populations in this study showed evidence of low recombination and a selective sweep leading to the nearly complete fixation of a single haplotype in each population within 50 generations. Although the loss of diversity in these experimental populations is even more extreme than that seen in nature, the difference may be partially explained by the lack of metapopulation dynamics thought to characterize wild <em>C. elegans</em> populations (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Barrière and Félix 2007</a>); migration could reintroduce diversity into local subpopulations. Finally, the evolution of highly similar recombinant haplotypes in independent and completely isolated populations in this study suggests that shared haplotypes observed in distant localities in nature may not necessarily have arisen through migration, but perhaps instead through convergence.</p><p id="p-35">In addition to the widespread fixation of ancestral alleles, one fixed novel mutation was detected in each population. The functional significance of these mutations is unclear. One is a missense mutation in <em><a href="http://www.wormbase.org/db/get?name=nhl-2;class=Gene">nhl-2</a></em>, which is involved in miRNA regulation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hammell <em>et al.</em> 2009</a>). A single intergenic substitution occurred in each of the other two lines, but these are not particularly close to any genes with obvious functions in sex determination or fertility, so whether they may influence fitness through effects on gene regulation remains to be tested. Alternatively, these mutations may be neutral, having simply hitchhiked on the recombinant chromosome that became fixed in each population.</p></div><div id="sec-9" class="subsection"><h3>Selective pressures</h3><p id="p-36">Simulations strongly suggest that selection led to the observed genomic patterns, but they do not reveal the precise nature of those selective pressures. One possibility is that the multilocus genotypes observed in the evolved lines are the result of selection against deleterious combinations of epistatically interacting alleles segregating among the founder genetic backgrounds. Indeed, such incompatibilities are probably common (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Corbett-Detig <em>et al.</em> 2014</a>), and at least one good example is known in <em>C. elegans</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Seidel <em>et al.</em> 2008</a>). Moreover, there is some evidence for selection against outcrossing in <em>C. elegans</em> in nature (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Barrière and Félix 2007</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Rockman and Kruglyak 2009</a>). Although this scenario may explain some of the loci that became fixed in the evolved lines, I suspect that at least some of the fixed alleles in these populations were selected to compensate for the <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em> mutant genotype fixed in these populations. First, these mutations are highly deleterious and were fixed, so I expect selection to suppress their effects to be quite strong. Indeed, the nature of the phenotypic recovery seen in these lines—<em>e.g.</em>, the suppression of intersexed phenotypes—suggests that some of the increase in fitness in these populations is attributable to compensatory adaptation. Such compensatory evolution is probably common, as evidenced by a number of studies of both laboratory and natural systems (<em>e.g.</em>, (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moore <em>et al.</em> 2000</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Estes and Lynch 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kulathinal <em>et al.</em> 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pischedda and Chippindale 2005</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Harcombe <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stoebel <em>et al.</em> 2009</a>).</p><p id="p-37">In addition, the “ancestor” sequence data provide some suggestive clues. This pooled “ancestor” consisted of g1 worms that had undergone one generation of outcrossing before being frozen, thawed, and subsequently recovering for three to four generations. Thus, there was some, albeit limited, opportunity for selection to act before DNA isolation. During this process, worms were cultured at a permissive temperature (13°), at which the deleterious effects of these mutations are largely absent, so selection to compensate for their effects should have been weak, providing something of a negative control. Consistent with the idea that selection for compensatory adaptation explains allele frequencies in the evolved lines, genomic patterns in the evolved lines are distinct from those of the “ancestral” sample. These differences should be interpreted cautiously because of the small number of generations “ancestral” worms were reared at the permissive temperature and because of the potential for genetic drift in those populations after thawing. Nevertheless, they provide a possible hint that selection acts differently when the <em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene">tra-2</a>(<a href="http://www.wormbase.org/db/get?name=ar221;class=Variation">ar221</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene">xol-1</a>(<a href="http://www.wormbase.org/db/get?name=y9;class=Variation">y9</a>)</em> genotype’s deleterious effects are expressed, suggesting compensatory adaptation.</p><p id="p-38">Unfortunately, generating a true negative control for this experiment would be difficult. For instance, populations seeded with the same wild genetic backgrounds but lacking the mutations would exhibit very different outcrossing rates, as males are typically maintained at relatively high frequencies in genetically mixed populations of <em>C. elegans</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2010</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Teotonio <em>et al.</em> 2012</a>). Thus, comparisons of genomic patterns between such wild-type “controls” and these experimental populations would be difficult to interpret.</p></div><div id="sec-10" class="subsection"><h3>Fitness recovery in remaining populations and conclusions</h3><p id="p-39">The three evolved lines investigated here were chosen from a larger set of 20 because of their especially strong phenotypic recovery. But if the fitness increase in these lines was driven largely by pre-existing genetic variants, why was the evolutionary response in the other seventeen lines noticeably weaker? First, it is possible that the few detected novel substitutions did actually contribute to the strong response seen in these three lines, or that this strong recovery is caused by additional mutations that were undetected due to low sequencing coverage. An alternative explanation is that the stochastic loss of beneficial ancestral alleles early on in the experiment may have prevented most of the evolved lines from attaining such a robust recovery. Deeper sequencing of all the evolved lines, especially using paired-end sequence reads to help identify structural mutations, would help resolve this issue.</p><p id="p-40">Most other evolve-and-resequence studies have focused on populations sitting at two ends of a continuum: either asexual, initially isogenic populations, or genetically variable, obligately outcrossing systems. In contrast, the populations in this study were seeded with a modest amount of genetic variation and exhibited a mixed mating system, similar to wild populations of <em>C. elegans</em>. As one might expect, genomic outcomes also were intermediate: although there was some recombination, ancestral variants became fixed in large genomic blocks, and there was evidence for fewer novel substitutions than in most microbial studies. This result is consistent with the low outcrossing rates and strong selective sweeps seen in natural populations of <em>C. elegans</em>. These results thus demonstrate the utility of the evolve-and-resequence approach in exploring how patterns of genomic diversity are shaped by factors such as selection, drift, mutation, and mating system.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">I am grateful to J. Hodgkin for generously providing worm strains. Some strains were provided by the Caenorhabditis Genetics Center, which is funded by National Institutes of Health Office of Research Infrastructure Programs (P40 OD010440). G. Chadderdon, F. Janzen, and P. Phillips provided valuable support in generating the experimental evolution lines. I thank A. Sethuraman for thawing and shipping populations to SUNY Oswego and I. Dworkin and the High Performance Computing Center at Michigan State University for computational support. I also appreciate helpful comments provided by A. Sethuraman, I. Dworkin., D. Denver, and three anonymous reviewers on earlier drafts of this manuscript. The University at Buffalo Next-Generation Sequencing and Expression Analysis Core provided DNA sequencing services. This work was supported by a SUNY Oswego Scholarly and Creative Activity Committee grant.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012914/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-42"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-04-10"><span class="received-label">Received </span>April 10, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-06-26"><span class="accepted-label">Accepted </span>June 26, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Chandler</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.1" data-doi="10.1038/ng.1050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crissman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.2" data-doi="10.1093/jhered/esq003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morran</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Outcrossing and the maintenance of males within <em>C. elegans</em> populations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">101</span>(<span class="cit-supplement">Suppl 1</span>): <span class="cit-fpage">S62</span>–<span class="cit-lpage">S74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesq003%26rft_id%253Dinfo%253Apmid%252F20212008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjE1OiIxMDEvc3VwcGxfMS9TNjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.3" data-doi="10.1186/1471-2164-11-88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araya</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing of a laboratory-evolved yeast strain.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAraya%26rft.auinit1%253DC.%2BL.%26rft.volume%253D11%26rft.spage%253D88%26rft.epage%253D88%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Ba%2Blaboratory-evolved%2Byeast%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-88%26rft_id%253Dinfo%253Apmid%252F20128923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20128923&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.4" data-doi="10.1016/j.cub.2005.06.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Félix</span>  <span class="cit-name-given-names">M.-A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">High local genetic diversity and low outcrossing rate in <em>Caenorhabditis elegans</em> natural populations.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1176</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBarri%25C3%25A8re%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D13%26rft.spage%253D1176%26rft.epage%253D1184%26rft.atitle%253DHigh%2Blocal%2Bgenetic%2Bdiversity%2Band%2Blow%2Boutcrossing%2Brate%2Bin%2BCaenorhabditis%2Belegans%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.06.022%26rft_id%253Dinfo%253Apmid%252F16005289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.06.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16005289&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230662400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.5" data-doi="10.1534/genetics.106.067223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Félix</span>  <span class="cit-name-given-names">M.-A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Temporal dynamics and linkage disequilibrium in natural <em>Caenorhabditis elegans</em> populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067223%26rft_id%253Dinfo%253Apmid%252F17409084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzIvOTk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.6" data-doi="10.1038/nature09352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahrestani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of a long-term evolution experiment with <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurke%26rft.auinit1%253DM.%2BK.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Ba%2Blong-term%2Bevolution%2Bexperiment%2Bwith%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09352%26rft_id%253Dinfo%253Apmid%252F20844486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844486&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.7" data-doi="10.1038/hdy.2010.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cryptic intraspecific variation in sex determination in <em>Caenorhabditis elegans</em> revealed by mutations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DChandler%26rft.auinit1%253DC.%2BH.%26rft.volume%253D105%26rft.issue%253D5%26rft.spage%253D473%26rft.epage%253D482%26rft.atitle%253DCryptic%2Bintraspecific%2Bvariation%2Bin%2Bsex%2Bdetermination%2Bin%2BCaenorhabditis%2Belegans%2Brevealed%2Bby%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.62%26rft_id%253Dinfo%253Apmid%252F20502478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20502478&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283168600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.8" data-doi="10.1111/j.1420-9101.2008.01639.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janzen</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The evolution of sex-determining mechanisms: lessons from temperature-sensitive mutations in sex determination genes in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DChandler%26rft.auinit1%253DC.%2BH.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D192%26rft.epage%253D200%26rft.atitle%253DThe%2Bevolution%2Bof%2Bsex-determining%2Bmechanisms%253A%2Blessons%2Bfrom%2Btemperature-sensitive%2Bmutations%2Bin%2Bsex%2Bdetermination%2Bgenes%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2008.01639.x%26rft_id%253Dinfo%253Apmid%252F19120819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2008.01639.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19120819&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261837300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.9" data-doi="10.1111/j.1558-5646.2011.01420.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chadderdon</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janzen</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Experimental evolution of the <em>Caenorhabditis elegans</em> sex determination pathway.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DChandler%26rft.auinit1%253DC.%2BH.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D93%26rft.atitle%253DExperimental%2Bevolution%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bsex%2Bdetermination%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01420.x%26rft_id%253Dinfo%253Apmid%252F22220866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01420.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22220866&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298739100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.10" data-doi="10.1371/journal.pgen.1001186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charusanti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkataraman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fong</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic basis of growth adaptation of <em>Escherichia coli</em> after deletion of <em>pgi</em>, a major metabolic gene.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCharusanti%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001186%26rft.epage%253De1001186%26rft.atitle%253DGenetic%2Bbasis%2Bof%2Bgrowth%2Badaptation%2Bof%2BEscherichia%2Bcoli%2Bafter%2Bdeletion%2Bof%2Bpgi%252C%2Ba%2Bmajor%2Bmetabolic%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001186%26rft_id%253Dinfo%253Apmid%252F21079674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079674&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.11" data-doi="10.1111/j.1558-5646.2012.01744.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chelo</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teotonio</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The opportunity for balancing selection in experimental populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01744.x%26rft_id%253Dinfo%253Apmid%252F23289568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01744.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289568&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313364500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.12" data-doi="10.1038/ng.1038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comas</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roetzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malla</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of rifampicin-resistant <em>Mycobacterium tuberculosis</em> strains identifies compensatory mutations in RNA polymerase genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D44%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1038%26rft_id%253Dinfo%253Apmid%252F22179134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22179134&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic incompatibilities are widespread within species.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">504</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D504%26rft.spage%253D135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.14" data-doi="10.1101/gr.108191.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Selective sweeps and parallel mutation in the adaptive recovery from deleterious mutation in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1663</span>–<span class="cit-lpage">1671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.108191.110%26rft_id%253Dinfo%253Apmid%252F21036923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8xNjYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.15" data-doi="10.1111/j.1365-294X.2012.05484.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dettman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigue</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnyk</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary insight from whole-genome sequencing of experimentally evolved microbes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2058</span>–<span class="cit-lpage">2077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDettman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253D2058%26rft.epage%253D2077%26rft.atitle%253DEvolutionary%2Binsight%2Bfrom%2Bwhole-genome%2Bsequencing%2Bof%2Bexperimentally%2Bevolved%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05484.x%26rft_id%253Dinfo%253Apmid%252F22332770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05484.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22332770&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302932500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.16" data-doi="10.2307/2409365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodd</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Reproductive isolation as a consequence of adaptive divergence in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1308</span>–<span class="cit-lpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D43%26rft.spage%253D1308%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409365&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989AN91500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.17" data-doi="10.1111/j.1558-5646.2007.00118.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolgin</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inbreeding and outbreeding depression in <em>Caenorhabditis nematodes</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">1339</span>–<span class="cit-lpage">1352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DDolgin%26rft.auinit1%253DE.%2BS.%26rft.volume%253D61%26rft.issue%253D6%26rft.spage%253D1339%26rft.epage%253D1352%26rft.atitle%253DInbreeding%2Band%2Boutbreeding%2Bdepression%2Bin%2BCaenorhabditis%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00118.x%26rft_id%253Dinfo%253Apmid%252F17542844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00118.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17542844&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247125100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.18" data-doi="10.1554/0014-3820(2003)057[1022:RFRIMD]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Rapid fitness recovery in mutationally degraded lines of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1022</span>–<span class="cit-lpage">1030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DEstes%26rft.auinit1%253DS.%26rft.volume%253D57%26rft.issue%253D5%26rft.spage%253D1022%26rft.epage%253D1030%26rft.atitle%253DRapid%2Bfitness%2Brecovery%2Bin%2Bmutationally%2Bdegraded%2Blines%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282003%2529057%255B1022%253ARFRIMD%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12836820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2003)057[1022:RFRIMD]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12836820&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183500900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.19" data-doi="10.1098/rspb.2013.3055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mainstreaming <em>Caenorhabditis elegans</em> in experimental evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20133055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.3055%26rft_id%253Dinfo%253Apmid%252F24430852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3NzgvMjAxMzMwNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.20" data-doi="10.1016/j.cell.2009.01.053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammell</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boag</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackwell</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambros</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">nhl-2 modulates microRNA activity in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">926</span>–<span class="cit-lpage">938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHammell%26rft.auinit1%253DC.%2BM.%26rft.volume%253D136%26rft.issue%253D5%26rft.spage%253D926%26rft.epage%253D938%26rft.atitle%253Dnhl-2%2BModulates%2BmicroRNA%2Bactivity%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.01.053%26rft_id%253Dinfo%253Apmid%252F19269369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.01.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19269369&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263930900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.21" data-doi="10.1186/1471-2148-9-106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harcombe</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Compensatory evolution for a gene deletion is not limited to its immediate functional network.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DHarcombe%26rft.auinit1%253DW.%2BR.%26rft.volume%253D9%26rft.spage%253D106%26rft.epage%253D106%26rft.atitle%253DCompensatory%2Bevolution%2Bfor%2Ba%2Bgene%2Bdeletion%2Bis%2Bnot%2Blimited%2Bto%2Bits%2Bimmediate%2Bfunctional%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-9-106%26rft_id%253Dinfo%253Apmid%252F19445716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-9-106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19445716&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.22" data-doi="10.1016/j.cub.2003.08.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chromosome-wide control of meiotic crossing over in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1641</span>–<span class="cit-lpage">1647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHillers%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D13%26rft.issue%253D18%26rft.spage%253D1641%26rft.epage%253D1647%26rft.atitle%253DChromosome-wide%2Bcontrol%2Bof%2Bmeiotic%2Bcrossing%2Bover%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.08.026%26rft_id%253Dinfo%253Apmid%252F13678597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.08.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13678597&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185404800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.23" data-doi="10.1016/j.tree.2012.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawecki</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivieri</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Experimental evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DKawecki%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D547%26rft.epage%253D560%26rft.atitle%253DExperimental%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2012.06.001%26rft_id%253Dinfo%253Apmid%252F22819306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2012.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22819306&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312055500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.24" data-doi="10.1126/science.1100522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bettencourt</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Compensated deleterious mutations in insect genomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">1553</span>–<span class="cit-lpage">1554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1100522%26rft_id%253Dinfo%253Apmid%252F15498973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwMS8xNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.25" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.26" data-doi="10.1086/285289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tadler</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Long-term experimental evolution in <em>Escherichia coli</em>. I. Adaptation and divergence during 2,000 generations.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1315</span>–<span class="cit-lpage">1341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D138%26rft.spage%253D1315%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991HB55000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.27" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth cit-collab">1000 Genome Project Data Processing Subgroup</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malmberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">The evolution of epistasis and the advantage of recombination in populations of bacteriophage T4.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMalmberg%26rft.auinit1%253DR.%2BL.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D607%26rft.epage%253D621%26rft.atitle%253DTHE%2BEVOLUTION%2BOF%2BEPISTASIS%2BAND%2BTHE%2BADVANTAGE%2BOF%2BRECOMBINATION%2BIN%2BPOPULATIONS%2BOF%2BBACTERIOPHAGE%2BT4%26rft_id%253Dinfo%253Apmid%252F892424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODYvMy82MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.30" data-doi="10.1098/rspb.2000.1030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">F. B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Pervasive compensatory adaptation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DMoore%26rft.auinit1%253DF.%2BB.-%2BG.%26rft.volume%253D267%26rft.issue%253D1442%26rft.spage%253D515%26rft.epage%253D522%26rft.atitle%253DPervasive%2Bcompensatory%2Badaptation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2000.1030%26rft_id%253Dinfo%253Apmid%252F10737410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjY3LzE0NDIvNTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.31" data-doi="10.1038/nature08496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morran</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmenter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutation load and rapid adaptation favour outcrossing over self-fertilization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">350</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMorran%26rft.auinit1%253DL.%2BT.%26rft.volume%253D462%26rft.issue%253D7271%26rft.spage%253D350%26rft.epage%253D352%26rft.atitle%253DMutation%2Bload%2Band%2Brapid%2Badaptation%2Bfavour%2Boutcrossing%2Bover%2Bself-fertilization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08496%26rft_id%253Dinfo%253Apmid%252F19847164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08496&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19847164&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271899300044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.32" data-doi="10.1126/science.1206360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morran</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelarden</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Running with the Red Queen: host-parasite coevolution selects for biparental sex.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMorran%26rft.auinit1%253DL.%2BT.%26rft.volume%253D333%26rft.issue%253D6039%26rft.spage%253D216%26rft.epage%253D218%26rft.atitle%253DRunning%2Bwith%2Bthe%2BRed%2BQueen%253A%2BHost-Parasite%2BCoevolution%2BSelects%2Bfor%2BBiparental%2BSex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1206360%26rft_id%253Dinfo%253Apmid%252F21737739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzMvNjAzOS8yMTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.33" data-doi="10.1111/j.1365-294X.2012.05673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flatt</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Adaptation of <em>Drosophila</em> to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4931</span>–<span class="cit-lpage">4941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOrozco-Terwengel%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D20%26rft.spage%253D4931%26rft.epage%253D4941%26rft.atitle%253DAdaptation%2Bof%2BDrosophila%2Bto%2Ba%2Bnovel%2Blaboratory%2Benvironment%2Breveals%2Btemporally%2Bheterogeneous%2Btrajectories%2Bof%2Bselected%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05673.x%26rft_id%253Dinfo%253Apmid%252F22726122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726122&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309888200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.34" data-doi="10.1111/j.1420-9101.2005.00915.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pischedda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chippindale</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sex, mutation and fitness: asymmetric costs and routes to recovery through compensatory evolution.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DPischedda%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D1115%26rft.epage%253D1122%26rft.atitle%253DSex%252C%2Bmutation%2Band%2Bfitness%253A%2Basymmetric%2Bcosts%2Band%2Broutes%2Bto%2Brecovery%2Bthrough%2Bcompensatory%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2005.00915.x%26rft_id%253Dinfo%253Apmid%252F16033585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2005.00915.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16033585&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.35" data-doi="10.1111/j.1558-5646.2012.01710.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remolina</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leips</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic basis of aging and life-history evolution in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">3390</span>–<span class="cit-lpage">3403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DRemolina%26rft.auinit1%253DS.%2BC.%26rft.volume%253D66%26rft.issue%253D11%26rft.spage%253D3390%26rft.epage%253D3403%26rft.atitle%253DGENOMIC%2BBASIS%2BOF%2BAGING%2BAND%2BLIFE-HISTORY%2BEVOLUTION%2BIN%2BDROSOPHILA%2BMELANOGASTER.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01710.x%26rft_id%253Dinfo%253Apmid%252F23106705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01710.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23106705&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310477600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.36" data-doi="10.1126/science.1604317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Sexually antagonistic genes: experimental evidence.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">1436</span>–<span class="cit-lpage">1439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRice%26rft.auinit1%253DW.%26rft.volume%253D256%26rft.issue%253D5062%26rft.spage%253D1436%26rft.epage%253D1439%26rft.atitle%253DSexually%2Bantagonistic%2Bgenes%253A%2Bexperimental%2Bevidence%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1604317%26rft_id%253Dinfo%253Apmid%252F1604317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNTYvNTA2Mi8xNDM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.37" data-doi="10.1371/journal.pgen.1000419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombinational landscape and population genomics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000419%26rft.epage%253De1000419%26rft.atitle%253DRecombinational%2Blandscape%2Band%2Bpopulation%2Bgenomics%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000419%26rft_id%253Dinfo%253Apmid%252F19283065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283065&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.38" data-doi="10.2307/2408434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Laboratory evolution of postponed senescence in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1004%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TR39600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.39" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.40" data-doi="10.1371/journal.pgen.1000671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoebel</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hokamp</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Last</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Compensatory evolution of gene regulation in response to stress by <em>Escherichia coli</em> lacking <em>RpoS</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStoebel%26rft.auinit1%253DD.%2BM.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000671%26rft.epage%253De1000671%26rft.atitle%253DCompensatory%2Bevolution%2Bof%2Bgene%2Bregulation%2Bin%2Bresponse%2Bto%2Bstress%2Bby%2BEscherichia%2Bcoli%2Blacking%2BRpoS.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000671%26rft_id%253Dinfo%253Apmid%252F19798444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19798444&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.41" data-doi="10.1371/journal.pone.0035811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teotonio</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chelo</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of outcrossing in experimental populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e35811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTeotonio%26rft.auinit1%253DH.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De35811%26rft.epage%253De35811%26rft.atitle%253DEvolution%2Bof%2Boutcrossing%2Bin%2Bexperimental%2Bpopulations%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035811%26rft_id%253Dinfo%253Apmid%252F22540006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22540006&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.42" data-doi="10.1093/molbev/mst205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franssen</span>  <span class="cit-name-given-names">S. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Massive habitat-specific genomic response in <em>D. melanogaster</em> populations during experimental evolution in hot and cold environments.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst205%26rft_id%253Dinfo%253Apmid%252F24150039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzIvMzY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.43" data-doi="10.1534/genetics.112.139337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Investigating natural variation in <em>Drosophila</em> courtship song by the evolve and resequence approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139337%26rft_id%253Dinfo%253Apmid%252F22466043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzIvNjMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.44" data-doi="10.1371/journal.pgen.1001336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarone</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population-based resequencing of experimentally evolved populations reveals the genetic basis of body size variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001336%26rft.epage%253De1001336%26rft.atitle%253DPopulation-based%2Bresequencing%2Bof%2Bexperimentally%2Bevolved%2Bpopulations%2Breveals%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bbody%2Bsize%2Bvariation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001336%26rft_id%253Dinfo%253Apmid%252F21437274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437274&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1657.45" data-doi="10.1073/pnas.1010643108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udpa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gersten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visk</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bashir</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Experimental selection of hypoxia-tolerant <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">2349</span>–<span class="cit-lpage">2354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010643108%26rft_id%253Dinfo%253Apmid%252F21262834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzYvMjM0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1657.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1647" title="The Functional Significance of Common Polymorphisms in Zinc Finger Transcription Factors" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1667" title="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1657" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j5kb6zlpPIfasJ8fHiAYqQrRn23B9OcbHL9A7GoXsUI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1657" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1657" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1657" class="active" data-icon-position="" data-hide-link-title="0">Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459662" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UNqbldwrcnFPLB7LjeTlowzDTJ9IfqfSF_edzPIJo_s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1657.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1657" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BtQ7X71SZQxR94aMuX1NoeqJZqu84HKIcmlBcHeRb_Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1657" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1657.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459662" id="node459662--2" data-pisa="ggg;4/9/1657" data-pisa-master="ggg;g3.114.012914" data-apath="/ggg/4/9/1657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher H. Chandler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1657-1665;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012914 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459662--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher H. Chandler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, SUNY Oswego, Oswego, New York 13126</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BH.%2BChandler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chandler%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BH.%2BChandler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christopher.chandler@oswego.edu" class="" data-icon-position="" data-hide-link-title="0">christopher.chandler@oswego.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1657" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1657&amp;title=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1657&amp;title=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1657&amp;title=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1657&amp;text=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1657&amp;title=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1657&amp;t=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1657&amp;title=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1657&amp;title=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459662/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459662" id="node459662--3" data-pisa="ggg;4/9/1657" data-pisa-master="ggg;g3.114.012914" data-apath="/ggg/4/9/1657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher H. Chandler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1657-1665;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012914 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459662--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher H. Chandler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, SUNY Oswego, Oswego, New York 13126</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BH.%2BChandler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chandler%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BH.%2BChandler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christopher.chandler@oswego.edu" class="" data-icon-position="" data-hide-link-title="0">christopher.chandler@oswego.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1657&amp;rft.epage=1665&amp;rft.atitle=Parallel%20Genome-Wide%20Fixation%20of%20Ancestral%20Alleles%20in%20Partially%20Outcrossing%20Experimental%20Populations%20of%20Caenorhabditis%20elegans&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chandler%2C+Christopher+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459662/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459662/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459662/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459662/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459662/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459662/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459662/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459662/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459662/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459662/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459662/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459662/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1657" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1657.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1657.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1657.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1657.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
