<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1569.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1569.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/9/1569/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1569.atom" />
<meta name="HW.pisa" content="ggg;4/9/1569" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC" />
<meta name="DC.Identifier" content="10.1534/g3.114.012971" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Verbyla et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multiparent Advanced Generation Inter-Cross (MAGIC) populations are now being utilized to more accurately identify the underlying genetic basis of quantitative traits through quantitative trait loci (QTL) analyses and subsequent gene discovery. The expanded genetic diversity present in such populations and the amplified number of recombination events mean that QTL can be identified at a higher resolution. Most QTL analyses are conducted separately for each trait within a single environment. Separate analysis does not take advantage of the underlying correlation structure found in multienvironment or multitrait data. By using this information in a joint analysis—be it multienvironment or multitrait — it is possible to gain a greater understanding of genotype- or QTL-by-environment interactions or of pleiotropic effects across traits. Furthermore, this can result in improvements in accuracy for a range of traits or in a specific target environment and can influence selection decisions. Data derived from MAGIC populations allow for founder probabilities of all founder alleles to be calculated for each individual within the population. This presents an additional layer of complexity and information that can be utilized to identify QTL. A whole-genome approach is proposed for multienvironment and multitrait QTL analysis in MAGIC. The whole-genome approach simultaneously incorporates all founder probabilities at each marker for all individuals in the analysis, rather than using a genome scan. A dimension reduction technique is implemented, which allows for high-dimensional genetic data. For each QTL identified, sizes of effects for each founder allele, the percentage of genetic variance explained, and a score to reflect the strength of the QTL are found. The approach was demonstrated to perform well in a small simulation study and for two experiments, using a wheat MAGIC population." />
<meta name="DC.Contributor" content="Arūnas P. Verbyla" />
<meta name="DC.Contributor" content="Colin R. Cavanagh" />
<meta name="DC.Contributor" content="Klara L. Verbyla" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multiparent Advanced Generation Inter-Cross (MAGIC) populations are now being utilized to more accurately identify the underlying genetic basis of quantitative traits through quantitative trait loci (QTL) analyses and subsequent gene discovery. The expanded genetic diversity present in such populations and the amplified number of recombination events mean that QTL can be identified at a higher resolution. Most QTL analyses are conducted separately for each trait within a single environment. Separate analysis does not take advantage of the underlying correlation structure found in multienvironment or multitrait data. By using this information in a joint analysis—be it multienvironment or multitrait — it is possible to gain a greater understanding of genotype- or QTL-by-environment interactions or of pleiotropic effects across traits. Furthermore, this can result in improvements in accuracy for a range of traits or in a specific target environment and can influence selection decisions. Data derived from MAGIC populations allow for founder probabilities of all founder alleles to be calculated for each individual within the population. This presents an additional layer of complexity and information that can be utilized to identify QTL. A whole-genome approach is proposed for multienvironment and multitrait QTL analysis in MAGIC. The whole-genome approach simultaneously incorporates all founder probabilities at each marker for all individuals in the analysis, rather than using a genome scan. A dimension reduction technique is implemented, which allows for high-dimensional genetic data. For each QTL identified, sizes of effects for each founder allele, the percentage of genetic variance explained, and a score to reflect the strength of the QTL are found. The approach was demonstrated to perform well in a small simulation study and for two experiments, using a wheat MAGIC population.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1569" />
<meta name="citation_id" content="4/9/1569" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1569" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1569.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1569.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1569.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012971" />
<meta name="citation_pmid" content="25237109" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1569" />
<meta name="citation_lastpage" content="1584" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Arūnas P. Verbyla" />
<meta name="citation_author_institution" content="Computational Informatics and Food Futures National Research Flagship, CSIRO, Atherton, QLD 4883, Australia" />
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, The University of Adelaide, Adelaide, SA 5005, Australia" />
<meta name="citation_author_email" content="Ari.Verbyla@csiro.au" />
<meta name="citation_author" content="Colin R. Cavanagh" />
<meta name="citation_author_institution" content="Plant Industry and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia" />
<meta name="citation_author" content="Klara L. Verbyla" />
<meta name="citation_author_institution" content="Computational Informatics and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia" />
<meta name="citation_reference" content="citation_journal_title=Rice;citation_author=N. Bandillo;citation_author=C. Raghavan;citation_author=P. A. Muyco;citation_author=M. A. L. Sevilla;citation_author=I. T. Lobina;citation_title=Multi-parent advanced generation inter-cross (MAGIC) populations in rice: progress and potential for genetics research and breeding.;citation_pages=11;citation_volume=6;citation_year=2013;citation_pmid=24280183;citation_doi=10.1186/1939-8433-6-11" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Beales;citation_title=A pseudo-response regulator is misexpressed in the photoperiod insensitive Ppd-D1a mutant of wheat (Triticum aestivum L.).;citation_pages=721-733;citation_volume=115;citation_year=2007;citation_issue=5;citation_pmid=17634915;citation_doi=10.1007/s00122-007-0603-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Boer;citation_author=D. Wright;citation_author=L. Feng;citation_author=D. W. Podlich;citation_author=L. Luo;citation_title=A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize.;citation_volume=177;citation_year=2007;citation_pmid=17947443;citation_doi=10.1534/genetics.107.071068" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Cavanagh;citation_author=M. Morell;citation_author=I. Mackay;citation_author=W. Powell;citation_title=From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.;citation_pages=215-221;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18295532;citation_doi=10.1016/j.pbi.2008.01.002" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Danyluk;citation_title=TaVRT-1, a putative transcription factor associated with vegetative to reproductive transition in cereals.;citation_pages=1849-1860;citation_volume=132;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=V. Gegas;citation_author=A. Nazari;citation_author=S. Griffiths;citation_author=J. Simmonds;citation_author=S. Fish;citation_title=A genetic framework for grain size and shape variation in wheat.;citation_volume=22;citation_year=2010;citation_pmid=20363770;citation_doi=10.1105/tpc.110.074153" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=H. Gilbert;citation_author=P. Le Roy;citation_title=Comparison of three multitrait methods for QTL detection.;citation_pages=281-304;citation_volume=35;citation_year=2003;citation_issue=3;citation_pmid=12729550;citation_doi=10.1051/gse:2003009" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=A. R. Gilmour;citation_author=B. R. Cullis;citation_author=A. P. Verbyla;citation_title=Accounting for natural and extraneous variation in the analysis of field experiments.;citation_pages=269-293;citation_volume=2;citation_year=1997;citation_doi=10.2307/1400446" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=C. A. Hackett;citation_author=R. C. Meyer;citation_author=W. T. B. Thomas;citation_title=Multi-trait QTL mapping in barley using multivariate regression.;citation_pages=95-106;citation_volume=77;citation_year=2001;citation_issue=1;citation_pmid=11279835;citation_doi=10.1017/S0016672300004869" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. E. Huang;citation_author=A. W. George;citation_title=R/mpmap: a computational platform for the genetic analysis of multi-parent recombinant inbred lines.;citation_volume=27;citation_year=2011;citation_pmid=21217121;citation_doi=10.1093/bioinformatics/btq719" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=B. E. Huang;citation_author=A. W. George;citation_author=K. L. Forrest;citation_author=A. Kilian;citation_author=M. Hayden;citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat.;citation_pages=826-839;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22594629;citation_doi=10.1111/j.1467-7652.2012.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Jiang;citation_author=Z.-B. Zeng;citation_title=Multiple trait analysis of genetic mapping for quantitative trait loci.;citation_pages=1111-1127;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672582" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the drosophila synthetic population resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Knott;citation_author=C. S. Haley;citation_title=Multitrait Least Squares for Quantitative Trait Loci Detection;citation_pages=899-911;citation_volume=156;citation_year=2000;citation_issue=2;citation_pmid=11014835" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Korol;citation_author=Y. Ronin;citation_author=V. Kirzhner;citation_title=Interval mapping of quantitative trait loci employing correlated trait complexes.;citation_pages=1137-1147;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672584" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=A. B. Korol;citation_author=Y. I. Ronin;citation_author=E. Nevo;citation_author=P. M. Hayes;citation_title=Multi-interval mapping of correlated trait complexes.;citation_pages=273-284;citation_volume=80;citation_year=1998;citation_doi=10.1038/sj.hdy.6882530" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lund;citation_author=P. Sorensen;citation_author=B. Guldbrandtsen;citation_author=D. Sorensen;citation_title=Multitrait Fine Mapping of Quantitative Trait Loci Using Combined Linkage Disequilibria and Linkage Analysis;citation_pages=405-410;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586725" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Malosetti;citation_author=J. M. Ribaut;citation_author=M. Vargas;citation_author=J. Crossa;citation_author=F. A. van Eeuwijk;citation_title=A multi-trait multi-environment QTL mixed model with an application to drought and nitrogen stress trials in maize (Zea mays l.).;citation_pages=241-257;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9594-0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Malosetti;citation_author=F. A. van Eewijk;citation_author=M. P. Boer;citation_author=M. Casas;citation_title=Gene and QTL detection in a three-way barley cross under selection by a mixed model with kinship information using SNPs.;citation_pages=1605-1616;citation_volume=122;citation_year=2011;citation_issue=8;citation_pmid=21373796;citation_doi=10.1007/s00122-011-1558-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mott;citation_author=C. J. Talbot;citation_author=M. G. Turri;citation_author=A. C. Collins;citation_author=J. Flint;citation_title=A method for fine mapping quantitative trait loci in outbred animal stocks.;citation_volume=97;citation_year=2000;citation_pmid=11050180;citation_doi=10.1073/pnas.230304397" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H.-P. Piepho;citation_title=A Mixed-Model Approach to Mapping Quantitative Trait Loci in Barley on the Basis of Multiple Environment Data;citation_pages=2043-2050;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102394" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. B. Smith;citation_author=P. Lim;citation_author=B. R. Cullis;citation_title=The design and analysis of multi-phase quality trait experiments.;citation_pages=393-409;citation_volume=144;citation_year=2006;citation_doi=10.1017/S0021859606006319" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=D. O. Stram;citation_author=J. W. Lee;citation_title=Variance components testing in the longitudinal mixed effects model.;citation_pages=1171-1177;citation_volume=50;citation_year=1994;citation_issue=4;citation_pmid=7786999;citation_doi=10.2307/2533455" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. D. Taylor;citation_author=A. P. Verbyla;citation_title=Using the R Package crlmm for Genotyping and Copy Number Estimation.;citation_pages=1-32;citation_volume=40;citation_year=2011;citation_issue=12;citation_pmid=22523482;citation_doi=10.18637/jss.v040.i01" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Trevaskis;citation_author=D. J. Bagnall;citation_author=M. H. Ellis;citation_author=W. J. Peacock;citation_author=E. S. Dennis;citation_title=MADS box genes control vernalization-induced flowering in cereals.;citation_volume=100;citation_year=2003;citation_pmid=14557548;citation_doi=10.1073/pnas.1635053100" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=F. A. van Eeuwijk;citation_author=M. C. A. M. Bink;citation_author=K. Chenu;citation_author=S. C. Chapman;citation_title=Detection and use of QTL for complex traits in multiple environments.;citation_pages=193-205;citation_volume=13;citation_year=2010;citation_issue=2;citation_pmid=20137999;citation_doi=10.1016/j.pbi.2010.01.001" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Vargas;citation_author=F. A. van Eeuwijk;citation_author=J. Crossa;citation_author=J.-M. Ribaut;citation_title=Mapping QTLs and QTL x environment interaction for CIMMYT maize drought stress program using factorial regression and partial least squares methods.;citation_pages=1009-1023;citation_volume=112;citation_year=2006;citation_issue=6;citation_pmid=16538513;citation_doi=10.1007/s00122-005-0204-z" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Stat.;citation_author=A. P. Verbyla;citation_title=A conditional derivation of residual maximum likelihood.;citation_pages=227-230;citation_volume=32;citation_year=1990;citation_doi=10.1111/j.1467-842X.1990.tb01015.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. P. Verbyla;citation_author=B. R. Cullis;citation_title=Multivariate whole genome average interval mapping: QTL analysis for multiple traits and/or environments.;citation_pages=933-953;citation_volume=125;citation_year=2012;citation_issue=5;citation_pmid=22692445;citation_doi=10.1007/s00122-012-1884-9" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=A. P. Verbyla;citation_author=P. J. Eckermann;citation_author=R. Thompson;citation_author=B. R. Cullis;citation_title=The analysis of quantitative trait loci in multi-environment trials using a multiplicative mixed model.;citation_pages=1395-1408;citation_volume=54;citation_year=2003;citation_doi=10.1071/AR02239" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. P. Verbyla;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_title=The analysis of QTL by simultaneous use of the full linkage map.;citation_pages=95-111;citation_volume=116;citation_year=2007;citation_issue=1;citation_pmid=17952402;citation_doi=10.1007/s00122-007-0650-x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=A. P. Verbyla;citation_author=J. D. Taylor;citation_author=K. L. Verbyla;citation_title=RWGAIM: an efficient high dimensional random whole genome average (QTL) interval mapping approach.;citation_pages=291-306;citation_volume=94;citation_year=2012;citation_doi=10.1017/S0016672312000493" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. P. Verbyla;citation_author=A. W. George;citation_author=C. C. Cavanagh;citation_author=K. L. Verbyla;citation_title=Whole genome QTL analysis for MAGIC.;citation_pages=1753-1770;citation_volume=127;citation_year=2014;citation_pmid=24927820;citation_doi=10.1007/s00122-014-2337-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. L. Wang;citation_author=J. Zhu;citation_author=Z. K. Li;citation_author=A. H. Paterson;citation_title=Mapping QTLs with epistatic effects and QTL x environment interactions by mixed linear model approaches.;citation_pages=1255-1264;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051331" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=K. Williams;citation_author=M. E. Sorrells;citation_title=Three-dimensional seed size and shape QTL in hexaploid wheat (triticum aestivum l.) populations.;citation_pages=98-110;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2012.10.0609" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. J. Worland;citation_title=The influence of flowering time genes on environmental adaptability in European wheat.;citation_pages=49-57;citation_volume=89;citation_year=1996;citation_doi=10.1007/BF00015718" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=S. Xu;citation_title=Mapping quantitative trait loci using four-way crosses.;citation_pages=175-181;citation_volume=68;citation_year=1996;citation_doi=10.1017/S0016672300034066" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Yan;citation_title=Positional cloning of wheat vernalization gene VRN1.;citation_volume=100;citation_year=2003;citation_pmid=12730378;citation_doi=10.1073/pnas.0937399100" />
<meta name="citation_reference" content="citation_journal_title=Weed Res.;citation_author=J. C. Zadok;citation_author=T. T. Chang;citation_author=C. F. Konzak;citation_title=A decimal code for the growth stages of cereals.;citation_pages=415-421;citation_volume=14;citation_year=1974;citation_doi=10.1111/j.1365-3180.1974.tb01084.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Z.-B. Zeng;citation_author=C.-H. Kao;citation_author=C. J. Basten;citation_title=Estimating the genetic architecture of quantitative traits.;citation_pages=279-289;citation_volume=74;citation_year=1999;citation_issue=3;citation_pmid=10689805;citation_doi=10.1017/S0016672399004255" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=L. Zhang;citation_author=D. C. Liu;citation_author=X. L. Guo;citation_author=W. L. Yang;citation_author=J. Z. Sun;citation_title=Genomic distribution of quantitative trait loci for yield and yield-related traits in common wheat.;citation_pages=996-1007;citation_volume=62;citation_year=2010" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="Multiparent Advanced Generation Inter-Cross (MAGIC) populations are now being utilized to more accurately identify the underlying genetic basis of quantitative traits through quantitative trait loci (QTL) analyses and subsequent gene discovery. The expanded genetic diversity present in such populations and the amplified number of recombination events mean that QTL can be identified at a higher resolution. Most QTL analyses are conducted separately for each trait within a single environment. Separate analysis does not take advantage of the underlying correlation structure found in multienvironment or multitrait data. By using this information in a joint analysis—be it multienvironment or multitrait — it is possible to gain a greater understanding of genotype- or QTL-by-environment interactions or of pleiotropic effects across traits. Furthermore, this can result in improvements in accuracy for a range of traits or in a specific target environment and can influence selection decisions. Data derived from MAGIC populations allow for founder probabilities of all founder alleles to be calculated for each individual within the population. This presents an additional layer of complexity and information that can be utilized to identify QTL. A whole-genome approach is proposed for multienvironment and multitrait QTL analysis in MAGIC. The whole-genome approach simultaneously incorporates all founder probabilities at each marker for all individuals in the analysis, rather than using a genome scan. A dimension reduction technique is implemented, which allows for high-dimensional genetic data. For each QTL identified, sizes of effects for each founder allele, the percentage of genetic variance explained, and a score to reflect the strength of the QTL are found. The approach was demonstrated to perform well in a small simulation study and for two experiments, using a wheat MAGIC population." />
<meta name="og-title" property="og:title" content="Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1569" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multiparent Advanced Generation Inter-Cross (MAGIC) populations are now being utilized to more accurately identify the underlying genetic basis of quantitative traits through quantitative trait loci (QTL) analyses and subsequent gene discovery. The expanded genetic diversity present in such populations and the amplified number of recombination events mean that QTL can be identified at a higher resolution. Most QTL analyses are conducted separately for each trait within a single environment. Separate analysis does not take advantage of the underlying correlation structure found in multienvironment or multitrait data. By using this information in a joint analysis—be it multienvironment or multitrait — it is possible to gain a greater understanding of genotype- or QTL-by-environment interactions or of pleiotropic effects across traits. Furthermore, this can result in improvements in accuracy for a range of traits or in a specific target environment and can influence selection decisions. Data derived from MAGIC populations allow for founder probabilities of all founder alleles to be calculated for each individual within the population. This presents an additional layer of complexity and information that can be utilized to identify QTL. A whole-genome approach is proposed for multienvironment and multitrait QTL analysis in MAGIC. The whole-genome approach simultaneously incorporates all founder probabilities at each marker for all individuals in the analysis, rather than using a genome scan. A dimension reduction technique is implemented, which allows for high-dimensional genetic data. For each QTL identified, sizes of effects for each founder allele, the percentage of genetic variance explained, and a score to reflect the strength of the QTL are found. The approach was demonstrated to perform well in a small simulation study and for two experiments, using a wheat MAGIC population." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1569" />
    <title>Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rg09yztKR2hA3J-LGWeHnLH33Ds8KtXy1QGcQsTj6Ac","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459663","apath":"\/ggg\/4\/9\/1569.atom","pisa":"ggg;4\/9\/1569","ac":{"\/ggg\/4\/9\/1569.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1569.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1569"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1569"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1569.supplemental":true,"\/content\/4\/9\/1569":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JPOmVQCYmW8ERn03cKNYODYn0HiA0v8lOdg4W1U3zUm1MevlWzVOkYYnFH0nN20S9Gb0QyqpXaNmEihzX9ybXhS0EuFlpXDcJht\\\\\\\/Yx0dWVPnt2bX1WphZqokC0UO8ithiYpPxdr+2xDfKgjjtwXIUQZ0Ze5MGVDKsHtJ9UCnuAKwIQLuRL5m\\\\\\\/7NnTeKHfrN4NMBNhmdXhXNu1VK6WYXRf8ZycVFk4mZ9w+i1Hv3lv5TFefBR53hFxjAV5nLhPQR+\\\\\\\/AhJ+ay02plStmuH\\\\\\\/N\\\\\\\/Wzb\\\\\\\/xEUsdyD0GZfjMxMDS6ASG21Bt7BeRi+eF3P6d0bKQ+6tZDag\\\\\\\/zi53Ty2IOO\\\\\\\/Lo1Cj49fgPymnSB1Nd9tLrmX2uA0Z94+Sj3OGtHGyIBnEagX7HDVbaH7B62ZqozwuYjUZ2YKDrmaSkt947\\\\\\\/FHHPWernsDdO+XESjv\\\\\\\/\\\\\\\/z8+zmKES5wkIMRBa5D8foQ3EhCv15AHlWJZnhvHZygHBdssl0944sQ05RFyMJvJlLiBHyPS9eYqKjclMGzlbBZ1G1xqyLx2F2U17U5R9zNxsgtp91R10A8CDBla8DEbOkzC+h9nu7fb3JFh+S8qm1hbmrDYL65JjmSEa3eAHXhb2JcA7hVeKs4OFPFDGUpkLeJslm42Wk65RdHpeoIat+7YrweLTfdC7i90rBwzbNqeuaxgMXEuBxNVA65wL+HmXFpllatRwpLEPRnSsHNucT7GE\\\\\\\/n6DSzh9QmTtVrGQ3XjX8jv7CdZQtBP4LCvF\\\\\\\/SCReW1SzIWcXPyTb1aRIE74RNjd\\\\\\\/15SjV3yTwS5MQR9kznpYBz14l1xWLAsEXWxYENalLKGTQ7qfxW\\\\\\\/epY9GHo+EBtZ5gD3mk6hBv9QM0d5iVquCbbxCiZps2CsMbbt\\\\\\\/1eokHh3UDRV7JNqfRkvHLiff8qfrigDvf8as+Ft2OxbUHSpOHyeVsHGtfAzt\\\\\\\/sMsQSlvbh6sK0+hETanRL06UVpn26+6dCC0SKkhV5Lhe6WzAWtSWP5HiolbZ+WNDWrba08wE+weGiYcJHfb2ufWSDrfSx224QM5HD5KOHrOE+EKmGUxR3bcjjLV4osu+hebxNOH219XT+d1iB51oPwWKT1m0qtD\\\\\\\/s49J3bYYM4XKMqtN50hP6xfvclmp1vndauVqAYxOY6\\\\\\\/jfDCGs7qKu84QGjvFeDohnB4HfHHaTjAt+dCFMGtE4KnnwiJ2a253D4SpJqZNUXYL8rOt1d4UCuJTQvIgtpiu53vZYzqaDGKTOVdLLV3EIyWNYBm93NeXPvLDgVrN96hh\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZbrqPqYpbnMwiArzZjgf7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002288fc8385ae41707eec044593b5bd44cf46841c523d93c2a295342d96c0631556\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nfiS2C75a9NoHKWMLec9KcLKY0Wp+7hyR4DpeZ54W8N2fpIiv9EbFLqYmpTSqRefUZ38WeJjTmHXh9Y22o6agjOanC3J8SLi\\\\\\\/W1dyEdUnjYU7cRBY\\\\\\\/UQiok5+qjtn18fuH416HunQ2HeqlyrKWdiDbud959ghtM92NwrP6x\\\\\\\/Q1bhCs55LZuIqjxceDFepPvqHJnBO0ZwUcjL\\\\\\\/SWPnLmdgMTclcunuaNwXceH2J6gQ2iqwkgvBQja5oBS90plNeFqMXe+DIyF0Qd9AwQTDArneC6LPSnfYsEG6VYL9Ww1d6HXDOxqVU6goTa3yxZNAc\\\\\\\/vwLL1uAcyVGlmuZNwMUwK4g2z2r7vZDiTx1dtk6LZTp0ctpfEvAq4t4LZwwu38DTaIdmW48nKIVUNtf8ppojQreUECGvat5a0mj6R9qijDkZOZPgtsCALW9OQWdEWewNV4n03p9P3EnjBh3isc1pokgBrCmaupK45e4OffSmSTP4D9dezOyn0Q2x2v9kHGSWFeaBrg6fnEZMOc8N8pYJ7vN4c4kEtW6zssEpv3n4JpHD0OZ+N3tnJG5DW8JqvL7889uzSPR+RP0UXNKsNvfsUgsY9ezfXaKadDEwbvOtROzmrWLgGzptuZOCFum7RcqhIWxnDuFQ4APkNV\\\\\\\/+EjcHZ\\\\\\\/NNSVwwz1Cob9thO8MCIos5EnQ7dSPpYz6+dmZ82Huoz15uAncJ0rq55NyqWdV22YHc4GBuwbo+cGwV0uh+6REuP4mAfVdjZNlH2MBUMiRTyqx7rJjFSpT9uMYlfNjdPOQ03FdLw2NoAlG1TcFbKU9TMLhdVhgXDUyakAx+\\\\\\\/imcGdT1aC9oElr7Ascl+TYvqHZ22p6I8FM+bYcNbd2ZL+dUroFl9\\\\\\\/ZcauKheA83BYlWyE7MpN6Ywnngn4SDL3ENXc1FUREFVZnWsaB0ZXCPR07KWs6R0HHBJAz4efeK7NUanwdJQ9WlH2J1TljuHHT4R4nD9qnwQu\\\\\\\/XbhgRrwy3Gp+yPGMRlyNB+bNiXmS7jGmSGKtihouPkhne0bDVgXkWV7ziehXwjZTHtPSCSTblrtSmNBAy648axOkWs\\\\\\\/LRbw81TAOcyyNycnsgoqWGhWxQZNd1Prg8LESUp4XorAGsMkfJDIcT4wF3+iMjHkwrBxmwHq0TuR6qTiAyJp2jQDDcALsFyHB23HUQDEI1LVeoJgVnAV\\\\\\\/CUJQuoKM\\\\\\\/XOQ+DCtEmypAEqYWUIAEHRyFEWIbWjhURpf98Yxq60hGiLxufTa2f1S\\\\\\\/RJ5IKeSsErlio\\\\\\\/bbZd5XkG6ixiGAq\\\\\\\/QgO2dSQBDxqqI7QRHgrhLec5Ore7eO6yZXoa5pSithM2rTIBpVN4tfv02JIIdLBSzMeGYmZS+6RCgS3HOAJtc1Y2zDfGa5iwrlsBBhL3ewcs2wbBDGtjYW6IazhAzZYBTODIll9wjGYeEGTb0oQbT14f7hxTL8V7TDBXoXaMBPy2A0M8HYow33D2\\\\\\\/PPjV1NMcWwV24iYjaGqON0gzPGumaz32PN7Yc9KJ8kDudvB2guHfFu9qTz0qSou6lhCxZS+sG4RpNj8l4jsutpRYN\\\\\\\/2ImUaVCXRZqf5npm\\\\\\\/191\\\\\\\/JE3TBRI4a0Xm74ENpFwxBx29br4oZ5RtmC0waS6rCQOAvVAht\\\\\\\/WVUNBiCLJJDV5aZan9UqAkjF6JuWoQYp32NEbExuWjhokwi1NgosXK7NwnBjlpHLlCMzha\\\\\\\/zZroQ0v100PRVio\\\\\\\/SWwcBA63SVuI+TY04hYbDFf1qAlyxG1l3jjMb7Z+UPP1eBt8+dsduDw3ZWTqu4LsOXoScMqw2gXXQCtHPi\\\\\\\/VMjcpPco1mSw6cELCpUE3xje4Qjz6SDY5JAbI+9Y+sXL6ZhVF9owPt7aJdTSOJDGM+zg67q97yAIPA3lf9oDA+g++B23vqbh6icLgmvSb4wASHzdawhC95AE1rATSUCrp\\\\\\\/NduA4REF24IEXex\\\\\\\/tpENcF6eziZORRaxSncFNrKEbOJJNg3j4KFlzzbeMGWIAQkwv+ewFKpZfpCiFToycqUL\\\\\\\/zVqChqlYFXHQ6OJuxToT3bfontEU3R\\\\\\\/6+2duHRVcL0TCoOc2uPkaUZMsMfMXXtK+Yos1YKuU87bvrrnwnl3DJXfJrdI4xRRk3QS0E1hev4ishmHiEwM\\\\\\\/SEiHlPNkbnzaZngf5u4nW\\\\\\\/r2mzUjKi9JDzOupj63iHUyFd\\\\\\\/+9+zhYX+SA5aaBqFWWiC5uvrPUQ0w5IRoTqnz9HTLy124iScufKQ6UXdO+XDT+TXq3sbjUvrJG0OI9fYpcjw+m7IPHYUweAxKxudhKjqdaT3KcPxKWYWt6X9hij\\\\\\\/2HdkpBqDdIBf8ZDNCMApbIPjDEa9VGgWT6Qv0LZLhpMV8shRKvCLAYB4mh\\\\\\\/IB9d6dCbalnblnTLR45GiqOCTZ+O6wQUkLCI6\\\\\\\/FsQRbq2Y1c0CDaGsdArWV2jlwPeffKf5lS\\\\\\\/JyEacUldDddSyh2BzVIArEQjviuyW9KUGxc\\\\\\\/cm5UlWcrJeIzUsQ+7qehK988g\\\\\\\/pPbxzASMtZYGor7trz6Wgb6211jXrG3w\\\\\\\/SPHigiyL\\\\\\\/X6ZZ+UTHLFpZLWpsEiplKIn\\\\\\\/bu6dsNNdTOM8NLJFoLQrSECPdjEJoAzDWkbzcHouvUPNCVw7gP14Uvki1D0bk1osnb78NRLRHM7xPek3JOitNi81Jyc14v7zA4u+mr1Xs9aDO6am8f4jfEBCbTdXRZ7tDr0fny2OkxCGBOFUJmXb60Uu4JO+y4XJ1QUiMOKu9zrfFNWxSEqJAL6WfzZIKJwjp3yTcZWI+tOOn3w4Sr0G32lEdQNRxNe9Oo\\\\\\\/yVEL0QdnVpIaDRSuvMHCtlkeZiN56qSvX+JcVL5hivlGfRqrUj3PmX5QQN5WaqweskA0uUg9O2A\\\\\\\/pzz9HzM4N6\\\\\\\/0+XA2TZbdG\\\\\\\/SOytfaydCawooW1T8MC8W7xNv9MHS0isD7GdWuwW8gLGaWjV2C3Fr0vneVABr2gsbENQbtbXWCq0HEpiC5OhRdePEBi\\\\\\\/WmJRJlVvX2BTaWdsLsU9YVBtMWzFHYEHUNI4RN1BgKIpbvUkpv7vY9K5\\\\\\\/02gOpDI5NekiRa16hRrMIod2yTL9G21GMbdue2RwJHLmqv\\\\\\\/\\\\\\\/navKMUq4GCEu+iSqC9fb3QHeydW7kAvGhA1LXgJXgjIqXyFTFlK8IB3GZzvHpHKgUmAZygTU41AUt+KV9ebLX36a+aUMIP8OYG\\\\\\\/vIaRX7Lby\\\\\\\/Da1hGxkbkkVgj8Uz7ztuCP71Cyqnu\\\\\\\/prPjGSSR3K4Xj0HkAwKl1QPBwIaRyIhIrfImnLaDj6njKCZ8UOUWi6TjaMk0P1y608jmu8588CoFEpe1Boj3UIq248gp6C08pOAe6dnwY\\\\\\\/4QZx20Qjmz4ydOsqvgmdjvYqf2rBT1rwUeLYkhLk0qVWXNi+jwvIWf2LqoKwMuUTYkKZn+tPpCa5xUzsJMAu0vcX3DAzgBm1s\\\\\\\/sl\\\\\\\/d1RN4yXOHfWkoTdHv3uOQdzOcwREarLDmsQqa93eUPlucw6wiLeRI1AbnLWse+AJv2NY25AxAwidBY\\\\\\\/QnWLSxYIM30GARgng9F2mm5JScHpiVzsAIU4hsPTsdocUTy2CZYeXh56cFgCX7nq3LwXajozPzfey\\\\\\\/bEJ4sRy6nxO0U2Fpsq\\\\\\\/JgR3LNoQJ4JOnx4jsNNY7b857zR2p6azp+GIfUn0JjkKpaE9mb0hZJ7OLIwnDpSTM0WPbEaqP\\\\\\\/KeQXVl1BxClGRfmDMJfbp9f\\\\\\\/Epd5QF0C\\\\\\\/Mw6YMUkSftfwd0\\\\\\\/EsF8mg\\\\\\\/AoByEaCITFpdf7a6oer1VNNroiD2Ej9nlAojINpWmamJwZRIimBSz+nUOZfGvvvbvOiOzlaB2bCEaYOaBPohCEagE3cWZrGjD\\\\\\\/PDk5yfUBN6Zv8tBxEwz4Bzqx7iFNYLhBlNUM3HYKfZ+xXP79XmWRTOtKDPo98az77IzoXlvtZqalbYn08V4pMwiFof5Vqc6qukK\\\\\\\/zkD2wq1ZGsaaX1oYVlnsW3VhXvjj4KqCKYRcdq+dVKW8w7BGp\\\\\\\/ATG\\\\\\\/CrknXRdUZT\\\\\\\/7K+Qkyqq1qKw2ifqTOos\\\\\\\/jhp5GE4fbjsOYIo64sPfUJoJpwl7Ri7GyHb2auekTPK7yMP2d0U9YNW2mK1k0us4GjV5Awo90LTnKtSDMtlbedD7Un2jqK+SzxhbtjA3+xWkJf8LefQGEb9ceufGvfFry\\\\\\\/zrocLBo2NAaKcTqxnPqLM73neFKw\\\\\\\/TcP27+DJjSIyI5FrjBCqh45yMnKlBxkbHKDlfrv9g75nYjaLlF5TsiQHFhPz33M7UHGaXyCSNMWFpEFBu2XFg5wLTJHcZZe44aXFWRzZBxiCum+rHDv6KTGeqZfkt7Rl0Gu+PKyeRy7b7bH6vftQ2HkQfpyurJeUFI6WLNB\\\\\\\/BmqFqct+f7Tk7Bn\\\\\\\/UOy4CoP1mEv+5T1wguIwP9TfdkrEAtCKKrVto\\\\\\\/tABPvwVXbocT\\\\\\\/ULD9AjLrbwupepyPCc8iMPDa9CRo5yGmpO0oXmd64oaDYTrNbqnpzttLBQO\\\\\\\/pyXuJPsyZpPwYtvao+zpyZs7l35i\\\\\\\/PBEHgE5tY0RCwfyNTvWx9MtE3pDM0sqZAneKGJ8nIj72aTEIWwik\\\\\\\/Xa+juKNaFPrDuaml2+EJ7ayl8B8SlWLxmUfGqTBYYqGewQ\\\\\\\/bajdB2NLmnfCL5943nB7VZ\\\\\\\/rAAh6leYtvFeq5IZgBEzZY6Nc2qA91dP2P5kQzqAomk484k3RWvTfjEqf9b3iAJlRX1R\\\\\\\/+yZbtpWVQWE2jYtp2daVQA81r9+kjJB7dCnlhRdS2rnfhBsckimUWsJfYc4wzvvk7JqU7yWNf8EoPG\\\\\\\/UXRW65BWpD3wT9G5yHzYm1TqcOHQ+ywlTqjPAimK0\\\\\\\/nWsKCkeeRRBmElmrC4qR9P6jCyPAuclNDambFQwN6IOx6eP6SQ6rVms7aUW4qiqU7uR05bnzqMNQn7G9fR6NMcgRpukAGxCsYqLaM5xt5AOcF41zX4qZUgDw4Lamr4oATdNvsn6dFYrH2yKNB0diLSnzq4l\\\\\\\/IRPfw\\\\\\\/KzwJ3UDdVcJbrDYsIfAWz7fUiEi9VcXiH33V\\\\\\\/wp3iKgcE0lVQl8bUZ0IYe2cnTJB6bMaW0eP14TbKv5ID62Da9ip2t75aSt8\\\\\\\/epOcBu7KWX+qPb6mE9hp\\\\\\\/\\\\\\\/rygMKvIfhjw1wcIlFX5XdEHbZ51gMj3e9Ax\\\\\\\/ypWoVdAP8eliLw6jnl8VHzzDu6KqXeHNIW6DL0LKOFnZ3zt9z0Yv\\\\\\\/lcar9YqPVEkC5s9EDu\\\\\\\/G7AeShLfX1BlhqjcSDElg13pIIh3TML+E5+FLrKo6sx5k1jnr1Cf+FGgU\\\\\\\/m0J56xEnvdTHwXoIThuABCkAXXv+AtS\\\\\\\/huGmLGRFR04MOwbNw28cAclZbkboe\\\\\\\/c2Es2rGTdg1vzKNivcLy\\\\\\\/HvePn7HpuXsnpb7nPVTQs0l02UyiEM+QgKB3JBIF08sszGltqJ4npV4ZqJiDVIx6YHTK5O2FsNwsPugmxYervtRJsh5XdRnw8p7EOdRF1x5uaAUjtPlMnqNQe7v+DXRTtACh6BrDEd1p2IWEcIq2UEMMO8Kjc50fyURgjYrLoonUmmK7Sg42JvOqJrEnhzbqXLZpTqTjKWhRM6nGvEfLzDC7o87bgku7IBiGIIF7L3doRJZKOypE7+Lp4jO5mwvQDRLNZxakOHjrDOThQKTOCq\\\\\\\/tafk0yu2FGSHGkUgVttjtcHwCn7BPcpYv6R4R3JA39itj0Av9mh85rMY51yNsJ3lkCspNTDvH+viCfFeafRwFSVhx0saNNTM2mgVWjnJNiaue5070MySxfyjbmrXHRI\\\\\\\/6Z\\\\\\\/KuxQsbV6KMkHGI1R0gVN25yfANP8xEq+CBbmD9rnxQ4kSyEVofPdQADSZB\\\\\\\/qoQa8O4cp2mTCY\\\\\\\/VETVNjcJKcJ51Q7orTc6xXgUtg0Iip3h4IcZuh3vqXu8KNxILy40SUhV6WVMQdYDezIM71WvFHqbPT18gnD+pOwb\\\\\\\/DgSd4mky+gfsPGJkPVligqUgDgTc2xNlcn46NkXEF5rIlgMEA5p8lyqkTV7FONHjyZULZuHJCQ2ugw4Sz9v3dvSLbbFFOL2bOBqda1To+AdRTeTnu7DyESn6wPmxmGF\\\\\\\/EPYQY7rtl0eY89FyQWQpG00EfhGgzgUeBHXuIkZVrnf\\\\\\\/3moDKFOEZaJYAd1byT6KcWnWyCKYfmuEnCToSKkI20laJAD+PT81mPZWNhfu8etnepQk9sco1soPjdqV4CTTgIJWAVktXO4nac+PA0AakvtDS12LW4r8STP3Q7L3qL4cVgVEShJ7W\\\\\\\/fBHsj86rMkbgK6CpyA2juizYAirA2KrOrxPTzuTEoyZCbiGS6ssxmjH3y8cCDtPM5hp4vTrKSKkAWThtVoRFrq5Sw6uHA+eqc\\\\\\\/LJkNkzj3R+4dUbK\\\\\\\/yec8a8wqW42zv2jslVgo3xMFIG+kiVAFVfS5JvD2l0Qedu5BVWh5sqWvo9VbOMBkPqbGM1T8dlZlLsKfNsR8eHYJe1P9g\\\\\\\/9qA5HMSfoW0E8NjJbDq1KuezazQx02CpBp6d4lOa1D+Uy3MopOmDzSbV0YkOAQX36JtUsl+RDg+nfW7j0cpYfqpx\\\\\\\/FZferQ3LRSBuEtP46NpmGhGwEUfDGdf\\\\\\\/lPc9nO5dkDasKgUap2d6PndnG9Kk4IbkyXdSCK0tqe9bfgoUrup3vvvUaYVr4Mx7vpvPXWYbyjSGTs5THYC0sivNs+yEssodrD2BLKL4XXgYlS0PP74HGYoxhNGBGdzmpQ4GzRt03dPEo1zf49cDaNscGpWMZhm\\\\\\\/oIQw5Xqwh9Zs5pTr82ujQsdzM2yx45H8IOZOr5\\\\\\\/SQEjkYrPA4h1epXyuDQE7MG0UYdbl1U2J7YfYingSfNcguhhSFXUV6pgjG77718OVIbwXqpW2faF7TeO9CuEQSqojRitgOeyF+eW5dMiyduEe9mjRHEG1+QdYTS+kqTH57si6\\\\\\\/bXrnJzAuXRuea\\\\\\\/fQrioupBqpALtREQZp0RwWt5J0Otcc\\\\\\\/2AzD31AnF\\\\\\\/baXoKQtXkPD2db8rP8Fl9A5yLx7kZCMJO+4kFrVIMoamBnPSPYFHIyJO3vm\\\\\\\/TH5MWfRpJ+n1iWOfQf+ZEEduk\\\\\\\/5RfBBdBTbuyHhiUzBdvkouVigz7zg68KkpewfT1ZtDFCWDrC9QV4lWFI7+s3UnHaPTdpWvjI2k9JYfLVcBcOnku7Zl4xVz+vNU+4MOvlSkhsAH\\\\\\\/mfEW4Xak3BVhgehSdlkWn65xrDpcm58CNkbjMetkpRDLNv0\\\\\\\/Fgo9bM3dNKy3EgRSqoZgEkywWVJlxEt\\\\\\\/1kQtfOMcpYvnCK9yKCJABCs2B0k+B+nlMrHtaiXIwe8khjptf70uLsN1UOZLiNmo5D\\\\\\\/C4IEAq1LigHTllc1dlYkSi24l8HRHUJU9XM9IdxCJIs\\\\\\\/0ZmNv+WsJg+FQa9FVcj0vGVteRBGH9wcKFTdZWGmg\\\\\\\/rDR7AAxvtvj+AAVRNJ+\\\\\\\/AcRCMDHjChTzurNT+yWre+8TgFBJQciZnOFWJMqwUjjCCalrhur2L2\\\\\\\/zHxx8hE0vmOKQqOC53JbgRl9kPMMh3xdxt7Rq09QNUqNIv8cBsSAitsB83R57bHax48dBAKrp9khnVaq9zqmnPvf4X8fdyoXMudcF\\\\\\\/17KHOYNkkTubOKdT1eXaGeZ87NTz+Tqv3yESNvAKdmmZz+jy0+ieUUoidXCJgrFA0Q0e9PQvV6ddyl3FkXbmNJgNQSIPhdxlwFXE6UG1uHCshRqma3IU3FDYDovvR3ikJg1GrEUylvdlQ3QaH4nt0LIcoXj+I49hARD4PCkNXyYbFCocMrFzsSfYRFh2JcJQ9qXJj2TnE8Mwo0SNc3ezjXkGAxePifP79pqNdJRV78Ow+qtakwPYweXKP1dewEE8CmLDct\\\\\\\/AWYVGUgvi+Kh+yqwWvR7v+hxpecbmjEM\\\\\\\/Oj1JpYtpDFPp+hDboos5CDmN53RgP\\\\\\\/CXB1Yp2HhcvKz1c8HBp59EHWGhKJv++YzldnBZMM7CybEbFuFnLijlk8sOJtpfSpzQGfYVI0\\\\\\\/Ymu+bmwvQ31Vp94hhdpr2kgaTnh2JBhT5OIpMQOLgV6UjX0s8\\\\\\\/q0Fz5WnpW+XkuvxY08NnZieSBShZ79nop5CkBTYwDGiJyzywJnRk3f6qhAdscOFKNRWKSRAHb3lGgLd4AG2Zd0J\\\\\\\/it1wXYKXTBR9EhnPd3vwANJAQMg3p+Ldt5j6uY7cHHSm2LQnZmpV5juj3fXq\\\\\\\/G3Ge8oi+XzL3vYSxcy2hxL7r8EKnWlkKAgwFO0ZmKiW2E4FfOm+CRgwhUD\\\\\\\/5SiXlAyL5rTpoIbV13Dsm\\\\\\\/tOMRX2dyzXlaCaY0q0bIOMZGBL1G1zRRJ9Lpdw9p6r4sdQcc6TN2O9BubwXLj0XdBj97nWa0bSOvEWuvPSzcbIGxYoVRWVzdOES4OGa2EOs3+ppT6gfiVilXRYSxmPh\\\\\\\/BLMAYKWjgl1rLPw+ijbkPivbX9Ikfy8yN23u32NB0eWDKBYWdzz0gN0OxVw2Z0m5t6SWc0Pbd9S3osu3D8zJSvXkwqtTySqrFimb3zk+58jUQRvIuHdpW7Y1ksW7wFLRqPEL985ixXsaRW4hpdvtzlmwudDKptSHtUFUVir5AUHP7xudOBqmTJjVuY87am75kLo586mz83WpCee0rEaWPW8Bn9nyjz8kPYwoqd0EnjKaqsWmSX9jZheK6Ehw6MtNHF5ME97J8sSVCiF4qiWNyUSbuPoVlkqUZXBpRLiUsj0wV2uvn\\\\\\\/oE1gS79OKIcdPp3epJMImogQr60j\\\\\\\/sZpGb8WIv6T1iSnuhNdNKTyEFtudkeL2Di8lvBMP8cby2GEzvv8QXiO7GGatJTjGAbtzXV2PHRzWVW0Fl27DATUxKII7LVpNReqZKhfAdFu9KlFcj67vnmSLbpiB9qo0HTYkLdKCCUKsxDbwxlxLslbcrQJnwIYpCpKCLkT9DbME72Y18mPx4QR1rqF5QQZ71Lj+U2XCWng4UGN\\\\\\\/n3zEmRv+s1R5oNkW6amEOtzxZ+zwlJ5ojPKcthTn4cyXLt31BJfxWB7Hrkuf19y0YvMDBeJye4ElIreAHuheAoImkrbgumU1QpKvS2CsPs1nQZdEoEpA1ASbYJYkMZwJyp69JF6DKM6XNKVVT+OECEc7bGiISntvKCYKAu6yM0ggXMqvfZeAmR54lE83CUJSVmTfRUUjC6x66JLOtnlg7scqVfDm5idQ0xWkxhoH259c2VCYZkJunkJ+\\\\\\\/q\\\\\\\/8\\\\\\\/1K3vVrrFt\\\\\\\/RA2H4tGfEPg6ZGYZ40k4PNbhT7F3F+ockDZqoemZPA2J+6vPfM3d\\\\\\\/cx9Z0d7wLspN6023OyaCSas3bHvSAjuJq2aiREWDhKbDzjp7oPDutAY0DdnLj00c7IQy+wn8izRnvtTCkCVEhnjP+qmmORiWJ3TxJjPRfSq1opVKV\\\\\\\/lFJuLf4crWVaFsOPnBYPuF0lLw7RADbV6sSLzfd0Ifrcpuk1zWln9a\\\\\\\/FDIeTLXDupDmD+kBsW6bwbVeLbj4QixcT11pVjlifdulCVHlYDrxDy9gxS2zSyxLXrWJctquVHdW1yimg96tPNWRSKXlfa+D2lZifxjpTldNPdFXJq3cWqf\\\\\\\/gO5+KqB0LHAWoke\\\\\\\/NVxSY4VNiPih9oOaR+SAMFkJyM9ruN9U0az+L9M1A+ISfeqjbzaJbd0NsQlwTjss9JE3IkP5CIQJu7vs9PKXQMGgXa4y\\\\\\\/TIvahy1YH86O1Ec66VGd5DEKbiVF3zgFY5+neO7exmJTI76I7j7H7FVzpkTNYk2smAV3HTuOSP3iaKa2ClrCQ29VYJb3Yz5gMIiqhxOctJzwTU9mbmMLZsBRFHJQjfaAJGDkhxQLc8jBvDhkCcQPc\\\\\\\/FOdOwFyRn1NMqwmKEhbhxiJUSSu6RaeClT9fqDYND23zFy7LMjsd0pHdFII8wp+axVOu8Dbds\\\\\\\/recRIP8Lno9YasnANE22wth9qUxQYN6uvyaKN8V4RWPt4rvwVzXSpFRohrpMMKYDi7\\\\\\\/Myq2TeaR0rtDHeZUpWONxVSY+2\\\\\\\/ynVEOKgdYPj06a2kZFejSnIxb+QflRHED9Y7e46kYmlsaSGH13w0HbeAQAC8kH0u5CIA5PakOf\\\\\\\/XGv9aNZ\\\\\\\/20pQQyCGq+epsM6wTOEJK6qrggeA0TeiNGIrh2R9EFxAb9UFjx\\\\\\\/+wsTwL+4J5HtPXTsY9RqmIh1HtHQyxoX9g1TMdU+Z6Nd8qOy37ikPImoZcJqzWkqMhqU+Dc71A\\\\\\\/AF4ZK5uU6B+6bdYtFQXxQsDBn3kS2PB5DTpHM7HQOKARf6uwH9mARmdmmW\\\\\\\/RrvBvUo3dMn3\\\\\\\/i+51n39fAsYTBRHIg1Rhewd1X15AaefkGu8fflZlCs2unjyi3AUODoNkMy+Xoa14wiZW77\\\\\\\/50X0k+0VdVDS+BTq+X5Kmfs\\\\\\\/WLoSwHZkpBwWGEiOroDW8UACXdP827Ea9zjCzi86ve1XcLHrzh1S1YcfpwP9SDA3OCgFZqE6f7CmGuqUT87ImxRtR95tLXR1YbemPVIWuRf+DKFpsKOcF42ggqdT8\\\\\\\/zA1irKKX0j115+\\\\\\\/lrWn5Gwan0nr9G5IhFOENGPnU9VywMQvgu5Ii8PmDUZa9kYe0yqVdPCG731OosCW\\\\\\\/ME2ewvrdH+6FxLRP5PBn1DmZD7vNsaldq5295lRIscT7kv0zsJPM1L2clzQaDnaHWhuDzgu+i3MwOX1ArplK\\\\\\\/gZ5mP87U7Q0RuGLfjDK1gMZmKgOnslhT+VBXlnEC6GQC8O+aAYt4xPleBBKZsiMKzt\\\\\\\/2oKhsTf2kxBE06By8pSHlCGc9HMsw6cs5QdyOq0xNT\\\\\\\/62jc13HSQaxszL8epTdc344\\\\\\\/8BpoGELY6bnjcgkO+VVT1hbmg56NjcEp2NEEfAw7k+7UBdz4oIkXKSzPJpqJoiE2x36EjdQrIAp\\\\\\\/eMPn82kx8Q3oOWIY9UQkNgpmOUdcH+xQq4urkUSP+CnLpm4Xod6jC29Xq7zvnB2YWo3C0vOCEGH8W5W3F4D\\\\\\\/V7JeTc6suU\\\\\\\/EF3NSCnJK7j8vuXTZhcdeKNXEUi+PsZ8RpuX0H1Uef++c2sZrSyB0RLxVg5BqXZOaB3t8Qter3PY6HXUAnQEtYdBF8O6RNEniSlZWDCVQr85SZnn4O9+JbyYr6Tv1qbE+J6uvHuGHaJ+z0sVyBn56PhAUPgnNXLKSnOw0jvqyBB+Vm7NkcnMsjTEi5cW0mTJ9bBviWgtwkaI\\\\\\\/aLBit5SFznNhyMdFJS67vIvURzk9EetNeF7\\\\\\\/YoX3jL79uIeHZ3UFOJuh0gG3LBfE4ETpmRCMjsAzA15EVn5T4znAVb\\\\\\\/k4O2o49pGtqTKOfp3h8ZOq4Rg8Uw\\\\\\\/sKjTr+j4xdL9XkVLH1NPLS0E8EFsqP90gy+jM2hPFRKUJvOydkh4ueA2bct\\\\\\\/OXp5kSgOiuIhr\\\\\\\/iSgN65RSBdXxGAcQWxD51gQ5dKtVc8cFiNdlBGQgHo+w0iWL1jQ7EXyV05\\\\\\\/iOrZMeC25uDSD0aUUB1YRKHHxCJmFWG5jNd91t0XRAgFjXxsmuvCCcaR\\\\\\\/HnCpw6oFMB4F+28omPkoVRr2Ie8pQQ3o+ybIDg4jDUClF0G5e7tVNe7xmUNt4OIOIkTjh4QF6vf5J5aBXZPQQg\\\\\\\/LTzWp22Quy18sOrIgK+rCQwc6NSvRy4mxNlN9Wk4VPW+Rku4XY4Jmw2ueOxLUzc+8mvO8+oftosPaGUUX8O+rPb1M44b6\\\\\\\/8HeSZNDli316aB2KY3nr61v8GOPgEHbcWjmHEcZMGRTgLt5DdjXDEr6NzbS2vnAnvDTTImuSmCuGZ5zPwZDRXBYmFQKA7IN8rpwzHbo\\\\\\\/\\\\\\\/xlG\\\\\\\/54SZsQo0X1DArqIOryzEygmSgo1kDymGYUn\\\\\\\/LnyGu2C2hlJUCkjfsRlwklXQDuMMfopAgRSARjiI\\\\\\\/Y3B1A3HxyvOUoLYjPotDckcP0XZcwWqVeYVcXhO6GjuLInTJs9s8r1Of6uB3TNA8Tees1FXuQz8lpsJO9U1t11J7\\\\\\\/FZ78Dngod7ruhycmEWtJMYPFCTJ7PkDYoLd0lwGnG7wM3ZGnR3jBguDJvZ415OWRMW1IRZg\\\\\\\/QpQQXTGgDyEB\\\\\\\/G0LwsFqbOJ6l0JrtZhuQZrfIVSkogGh7TmvWnDwnEMxsDHDCfR1TA+vOIuUWCXajNRrT+cAwCFvCphxFpaABldwZ9omsX9bz\\\\\\\/Z2DKJ+\\\\\\\/qCgXtItXuMjOUul5bSiJ4wJfkKUgYrJLQ3suFFE9Apx9FvyjkCaEl\\\\\\\/BDeRdDqMG\\\\\\\/yGkujdnNEE8+RV8UGbUEeTrNov+rsmqg22jBlRtiR55otA+z8CpQCTT2Jpnckd4eIILIEQ2HqzK57TQO0Eujno8zQqC1jKpxNkhyRRnwH+9\\\\\\\/e2xo7k7CualXNen8uI3pW1YKXtdJCnfqJl\\\\\\\/2Xl2o3bMtV3QeJ8KEZnaA+zM\\\\\\\/jFdYPUF5iMdASSisS\\\\\\\/efSPV6WkLd4zCtBxoV3RNKrk14pKg2SIfNeW+Oiwb4LeOWrHbVe1\\\\\\\/eawAEiYhA9x4TBQywOc8jI4XcDUB0b7w\\\\\\\/\\\\\\\/evc+CuD5vDCyvpmdI6kGzyfq54fITW2cpaJNRCY7mnpnzRCnVFZXXPlSWsiae5VvI+Q3nSHGBmZo8bIfSmjk7MHIiQdupYXIWv0guoBm1whR45P1eCXnJdMAEd0yCT8LbTjDovZoM5qZ+GjABdXyUy+K9bS7Z0ASeARaxhrYnb6X7wbmRut5iFn9m1m9fXXxcvmMBlgEBdGAezvLQtFHoRHFodZGuhhCzBtn2qrq1TnAEL\\\\\\\/z2OtKNS1mc7Ki4ni6IFBbRHmJr3UbWZlPEZxiSOxrbVaUlzR+G0\\\\\\\/Dq8gywm3+mbkJJprxpL6DLs9NkSakuEwqbpl8RaqmwNY7w3srR1KRAx9VofAN8Azn7l3QqlZzGTAvcCTvPS82FJ28zchPB6rmaxMIr7VYnBU5mgLCsCkgHdHyXx29tApds0Ff06MrdZQSTlzrHm5Mp4NQcI7BxEABxUgDF10DHqC1e2zvbGx2ZoEE5aSaycjIomtouSBovH9OLga5C77TWOBn8myPoF1ueoZCoPutn1PL0mlynpxb7JZxBFdhcBmxCK\\\\\\\/l1mqfTo0gHcGOQiI4aV2HpQ0fz8d7gDEOawonsQS8opqh4WPRbKgSrYbVZKOqyn8TP1wneab5JOZIxJvG50NvpOIMbxYUe+iWCFefqF9tBPtq5KojBe88fO8w022redNNvqs8JoMlEgcjWRnwpD2nXPsqDkSBsxTL+VLfLsbtwo91uvE1mtEfyUNggTeSpIg\\\\\\\/4\\\\\\\/3mgyGRAutdE0r03wQ0U9WFeT\\\\\\\/I7Z9jPHsBap9Y6VkrV3u8+y9D+VeIkodjVxdn69AHYsCRZ4fyBKjISY4qqnIsN\\\\\\\/i7tMdG6vfLhOi1fqQ4snbQCzLHu1YiexXSjpY9eD\\\\\\\/LsSG5P+B8lmNo1ZrdSnbDBbktY7lFBV2cYZZEV4EnVxz7E04ru09utAxLmk9RPMJBU5rdpUVFBRBZkC0BYue+TmQAO09mRM7jnqN36dMudtnSU57QyhRGucTqT5KZxD0ASlUe5APQyi4KabAhJDRz\\\\\\\/fKgjFmphN\\\\\\\/LskLTrGam2qquUGg3Y+n+EAD\\\\\\\/RGl8d0O9dRuiiZhQ38\\\\\\\/wkx\\\\\\\/SIkWL9WgCbIgvl3NtNipBk1M6vz80wPcHjQEReG7DgOXgns\\\\\\\/dbGeQr4NGV6JqaiW1w3rUt2siFSVsdTQ7xP7B7Ymh2bR94MDmaNNTxA3GuEhagVoF9JwxxRRzQSyE90FLFf6RdW72HVpBdRbKYAiEE7nkwak8ygXPnIndluCxqzpIxASqvQ\\\\\\\/D7yMzEys44liW7pHxzO7PelH8VFh1hPse3MDU3sT79JdkKypsNczAYgYKlojB7Mfqg6SG6ji7MoSQLphKfThcmokDBzy5WrUd4JydHii43m7hE\\\\\\\/19pdN+0J6EfWQdCysX707f+hSahxa9P3JYhhNawPFtCP59tilnOgSy0YV+gikHDyIk0qADPdCriE6Y3BXTm4e884TbTdcIG2rNHQswxhKAYe3uErLtpyc3DyzQyu86cNGAPiInYyGeeuA+j4GN36XT3vi9a4p8BPmOeRqRAFC1E7CnfUV9tVgnjwD\\\\\\\/Wvyb4VB+p14zgLXOamwekcywqOtFgkTFfxor8CEmolvXj5ch859pc+Q68BkaGcNSXMwDBa0R\\\\\\\/ObYYk1\\\\\\\/dRleiz6t+U6qmWcY6u\\\\\\\/26GjGJKTv+EVOaLfFXUdEPvMOUgR51Bf2wwjnHN6XJ1\\\\\\\/zxoPstb2vBfzLUJZee0bToeozNQeUSNNDCwToEa\\\\\\\/522imG5QwPVpyY7qeG46fmxoyRbQZNVv33aT9nB1AAQ4r1KDgI8dp4BUeCL+OR7mpDWd4j2HIRj4vut6wezW2rHJzj2HT2KMVq7kR0cy2+NqO8e0ZpZetM6Pqan15ZDQd31SwX7EBwb6zI5TZzvGRDNpoOGGrmvvEluI+OTDJcj9WeQGHehP+aEVVJ62EfoWyUSwitQ0mEVEjx5nL8lN01MuGepKoLWe+NO0o+JnahyuyNoTvDJWcgj4QOuCB\\\\\\\/su0o3YZQYPKANy6mut0BtgtJenOBMpBdkfvWOvXHTXWo4SNxdtfV+NQBtAwtvd9ue\\\\\\\/LVfF6TBOAH15xLV0F+\\\\\\\/qb959ecd2fnLBOl33qsJK6p14CjKVpn1ZrFwnHlzM3o57avqI9kjTEdLDBSRCYs5EKF1Hhl4C8DlFJXV5\\\\\\\/UdwCskRWrtDtGoVb92v4EhNT6kmCe4CJcHzImVaDT4f5o05Vbo\\\\\\\/v8mMZRqenY1eyCOlQhyLuzK8pbPIYHimUEfCeYC8ObUs759FARECpYR2cJ3B4FV8J0X3ZOMEjG7pseXKsX\\\\\\\/zGECZwzblhJvRqwIFGBQKwZ\\\\\\\/hQWf5S3XNec1L8itepTe\\\\\\\/nUcjcVoEIiseToVGJPI0eFJ\\\\\\\/KTJAFCFcZvNMN03VDGdVAT8W338vBdS0qZuKe+PDabxLMRgSl0UPQkq09\\\\\\\/6QcA4jE\\\\\\\/NQlH+ucvFMEu5eoIPA+X1Wg2kyigr1kfyj\\\\\\\/nLjZJ78gltboLnr+nkV8j4zhQESRDDongehFoDn2u7RBnEMpMDeaBhxNWoARXCnMZZoaFA7rfj4NUXXw+EVHle6Plw\\\\\\\/FGGcx8ttu0t\\\\\\\/cCzwE\\\\\\\/Nw+8mAJkIoDHthxFwv8OKUjb+d1Kf806IkiC9OfVtYYEqxIXZyj0CwMMi5T00yQkRWpaaBsyj3IpnsxUCKaXqyt+pX34\\\\\\\/91ns7pARksdo3vvZ9WL+TJ6jKHddQ\\\\\\\/+q6SInGic4CCinOyVmezsvL0pKcACqVuornXJtWpkQN4OQgVW\\\\\\\/rBHueqVirg8JygT+kiSvTjxdhn9F\\\\\\\/QqaBEfdTntvNX6RofIRUZfoDT\\\\\\\/QysIpRk+dymXxISlooKI4rqNTCOGjzKxxx2swOXd4adtsz+OlMWYMXsPwXRoETRK5Xpc4sBM2mxwX2TfnabSVbh6tpbGVJudeLXloZgKzqGmBbasZRNqSDCk6p+TYhbZiTD1sT2kw6jYr\\\\\\\/PnX0lAXu37nopk7EBEtELhKg1bzfgHNw+7ua2xlBNOvtLKMytCfFPf31mpn33qWyKs1trhLRPRWgWjfmZlDtjcbKD\\\\\\\/D4rDORk0XyxCMB2hqUPchgXzH+Ud1iBR+2E7TnFpod2fNFF1SD1eVVx3DBYXhD9qUdQwEaJFWtTWuViAQTCNPgthlLQhUcdMvf2Bh4yyoX22NqzW0jcwUXIkVS9JnsmNGB9d42hNRYd2Xpe4CguqB1fRZvbTOF4Z4OqpVZ\\\\\\\/5LyU5HyxD9yWvEjt06\\\\\\\/fwv14peM++oLDLCCNOWgR5Udw8UWB7aXtiITO952nwfrHZdH5Lu2p1F9up1E6mO7AOu\\\\\\\/q1D8BKvAsH\\\\\\\/4Hm6AFAJqr+3gzL07dvVAI3qoadS8LnqGzrrEg5WTqOUnpZrNUKWGmQ0UF1Spozr0NjWL6voDsPCWzsnDr8iw\\\\\\\/Vh\\\\\\\/XRyKp\\\\\\\/nWKmqFyPZeMY6rvZXg7HbEjU8r1UPuP2t2JnUdjiSPY4x4749tgHRYunccUuRg1N\\\\\\\/70y1RTC6JZie44WthiHo0iGobuXEWJR2RZPLhHRgvW31a36I90QDtFljefjFS0RRE39T94MZhEVG31prcSUl3Q1JJzLwtHkbNqzHR1dAdfNzlqYdyCt6X4RJnyqscYJmVI4uWVEyWBLzWJIfLUJ\\\\\\\/d7FTV7foE4K+mlZ5vSSBV1QC42zH6F\\\\\\\/ulG\\\\\\\/c\\\\\\\/LUGz06LgIoF3t6rmM\\\\\\\/52imlKDKN0C+JZ7u45HGZk2b0SJbuNLrUiFyMZQftIo8dsowW0t8IFGHpHCBmMlUF71\\\\\\\/KYVv4WIga8NoxxjN6sXWiCrquAmTJHsDqN3EnHCbZCWZNzPE\\\\\\\/adTYQ6wdlDQkdXDgIIybS6qiqmRmfd3Eg9Id047sR7gOljB5mBZ5tRWh6Jb0fg8KYVOqEbV19e43i8Ns2KYvFzTYuCs6\\\\\\\/fl0BhzL5h0VqjUcbt6SZNCCgyp\\\\\\\/Cju66UpydYU9ywPExrDM0zqcrZhA7YtK4p6lE3nHGuGj\\\\\\\/t7pxcSnWXk445149+IzL2C9rsShv8zF9J4s4\\\\\\\/ChYSMuxGmdPZteDuUo57g1eniqW4xQQYVfg9YrIwOH\\\\\\\/3q1c6mN0PoQyoo9OfBNkYwWl3HErGIVgiMjhqObe9ZM3AHXFKox3CDxV0SIY9iBdpSj\\\\\\\/A4sOe3bZYvth5rbdXBERG4vtWzl3FTFcxPWkFkTX49x4Km8eLT7ZfCso5M4jC2\\\\\\\/3cMJhQ4XvaUZOBdgPZKcxDqTcgduB9t85U8ZvOhVWEY4yyscMv+VbHxTNWqT8tp4DjNA5C\\\\\\\/c2NSMMP+JcrGSWM9pPyiSgKVpi+AJylqgxVsFE08bg8hyEBa5pSsCB77Mv4RwMw5gPxAiVFnYLUsLeTkieQNeX9aFpHI37eIB9tvplzQuIWR4+Ztf0aOs8IR+um\\\\\\\/EJ8q48TcWlyRA4goJ+V59O8VtM0NhSrKhJPf+G8SUurYwd6LF1xdy\\\\\\\/eSMBrsDbXCG3NQD8jBTWkxWyLbfhJTcwtbVqvKgEf0Mu3tJpqXdyv3pFEwwQyDidoBRpnQ4TAh12F2rcG\\\\\\\/LN12E7vmeLcsI8OCHSvadWesHYi9iWRIDHQaGHohKdPUYIcr\\\\\\\/8penaDTSQjBAB2NfXzoI1Hmh2jITSRgAm8XT\\\\\\\/eOJC3RzB0tdiiFGNlU5lVCrGpQCQK2PlRH6GBTtu5fzmAxYanrNa5tJd+BYHQfA+pyICOiZKzDH5hngZQ22XHE899tiM\\\\\\\/\\\\\\\/z4A\\\\\\\/gIM5t4IxZ5kSIBi9QB0mSiXa\\\\\\\/IqnbWYEtxALBKr+hkpCSWC7INH2cVfhbm1HkvKIYqZShJyzKTDE\\\\\\\/f\\\\\\\/VtilLkoXfeVJU9Mkeoc8Kv1Hf2ajYmySK+XrrZKtpk4LYe5arcsbTRXBW11pPhzK25gUl54sK4lDyPdAcbRXuun7BaUDw9afs\\\\\\\/go4v4Dewrvtj9+LLliza+n8i3ZfSCn2CXjJHHYKcNPd2xIAlhivHOdLJoJ9Gac5Iwjn87GrSCUMG5u7OWYIya1cMTD4tU3di4Pu\\\\\\\/+kKE5ZHjNzsBTT1fKLo+YnF7qvMaa2rDnIROA\\\\\\\/B6g9xdF8VaGmw\\\\\\\/MdXJAPZoTDkEqOfmSc3vzsxfdgo8rPjnOdN13DXiKmQd2HE6KDdyxodS3ua44w1R+267QxK112JE2QbP\\\\\\\/hfB1Lq2VWFnw+UqzmHqaIT+n3Q+KOFOqypJLm+BrSfUsO5jmO5IZJPPYXHLd8kEBCFGS0uWPBrMOnfVo56RTdGa4aR+\\\\\\\/8KQdf4ZCg9oeXRe\\\\\\\/cXvD+Gjuvn\\\\\\\/S6tHc4W29Egl4h4zgOgUwWqdBI8bvv1S+9brBawPFACnXtaUmX+vul0vFoex\\\\\\\/onWyf7eGqhHz+Ox\\\\\\\/w1HXUUmAgb5bj4B6r0h3w+jvyhHdDUnLp6GBc7Ol3rsPUnFbUC0Yen4jkAObhSns7Azccs2GaJZU9SVSpRy7q8ittDSX9btpVTBOoO1aWTjTD4ABbU6n2XizaPrjlArciZrnuRsLFfRLjIo7BlM7zxvTFvbWQivDaTSi4p1rXAsfn\\\\\\\/yKuKYPHGnhkBsrSQD265KlGWE\\\\\\\/jmPFOiPo3m+IBxZ4tfvkINdsUnyKuSAhTDSa830dlbrJBZWTX892p0JMr4jZX5EEaLfTItE8+B\\\\\\\/0GPthS0ce3YLq3v7lTCvTyytP656hEq4slEJYLKQDM4siG1IREXhrguTCHaHHioAEB7HD6jaY5t8J5jo3WI\\\\\\\/Yu2OwOLezjq5Q1UHkP07VQ0zzQaGrSrp5JJT1i+4KmWza6iBeAI83wF8NGD0z1DotRG8xLf2pWAaiit6nYTa+tu0Qf7zu7GzNuLsIq04hRIlLBg3Rmja5yrFB8mbbrUeFahu7Zo6rjun\\\\\\\/BaPSUxmsbqBalmCx2axpmgp2hkHxwgTD+7fTnQEKbDVAitgMqhwEES4UuxdgRWMEtg8Vr20UAB3gCDiJOVpKBCwba83nY+dehobi4ZQUgdraogbZcA3a6eloLE1ec1SayJlABduG4wuNNfwViFl0Zvlk5NGUkAsr6S\\\\\\\/9L36cEG5QzK+BeVwe4\\\\\\\/WuclpuI4vo29tLCKjuDYl7vnNblh\\\\\\\/BSVGDIzxsey8zZe+zy3f6csf+uzILNJx5YFwmWc9PHt6g+rRTqfGnCgIw9u6uhWo0uSTNkcofvIQAIgFWZEi4MYXhRvoV5VL2j0kHeWauOD+KQinlvSd2wNjclkuahlBek2wi\\\\\\\/7CJHAgLSDBTobqJZ91TaPda9csVZi+1rAxoh07DW9BwUUNs7CeWqFBpgA4g8sy+9HXfX4H4eoSy9CKOBBV43hvrysBdzvVHLWzRr8HZlLgnOqVS4tqiMsVnm1uxXRb5iEn4keuyOG2XfywvEhg2xCcqvPyJGbY0JWxdVKtiP+Kxs8l3w4V98Jjz5MiObC65dzOn0DM98mOUbnjsDi25AheWs73OyI9A2CQxY3DZgrB81AIlg2uMtsaX5SNN7v4tpNVwAC5U0RIvU+nLUtj0RFn4z210cIF30qtUhlqNTwObRV\\\\\\\/sFkwWQQoSZa9cqJYRmIJpcy\\\\\\\/CXNAGLOpGRadfsEa+xfBOXxx5m3qn\\\\\\\/SalGiGkNN2gzZSb3Re9xg+WD7s5ztrGvIcHil6qZEfbr4CGngPbjRIQsTXuULrenIMG0sP\\\\\\\/2sxGKWJ4HGcwY8+b8TN1mgI7e2ArkRbEhsW4VCv0r7VnEqXV813dpvqRfERW\\\\\\\/mUDHePW05ULUTLuAdwn1pAVRlUtZTe3LmC6JjsWVRpsf3KnJp7tTv4OrsCrPMsSFB6J61roW\\\\\\\/0asIWENjz6UATbrxNHru3O7dG7c8sUcmRuM1d5rH38XXpV7yeQDVG\\\\\\\/lq0l4nqxJSnfKhH2y8cKlfroyL\\\\\\\/8nTl4uAmBGu9fKZfvjILXQhnCsIte6ATQlo57AHJKAx9TFxB3idDOV+jpnLS+Wew\\\\\\\/QFQoeEbsxwzv5WP5vegEKU4T7lJiRjwRNp9O0STCsFfNURAuJTilZrSMG94hSdasyFjjObnSSJymNZrNINOMPcrLMKuf9DZwBRvrHlny3MEed4YpTGhgsBD5+8Xt33IejcnOYuliZJkbn0IsTOCU+WBA9NzwvEFQMUZmI4DFHDiOm3\\\\\\\/cZPS0R+0OsdKFL8tFP4eQLrKadCrX+LgWzFNSoYySyY1WUzOjNfb8VT7e7ycHaLAsqUZ\\\\\\\/\\\\\\\/sb33cd9M9j3AdOJEom08tRiX55dQli8szqtdmdodgC59zGf9EZIHOp0Y1j18nyKSJxTPfoV3lXPd6rBVk5N0EnnfHU+a2ille1KVIHpokDNcfWT0RTzuFu42zNVbViXhQAqLSK5I5LLcSGMUveA0FUT+ixBL1Oq0tCmCvqeGEj0zQDv4+bMxuHquoK\\\\\\\/X4m4dthulJxduKXIEbFL92PLwld2IGrImdSGixnpRYoHcPG\\\\\\\/43F2x484Fs5nbUZ9RhIGXwtnZfUNYrQwdk7AeyFyZt2+qEEvUWJYUwJbVCjB6Kr+xzUZ4OtjYSZEdyY+Wf5ziAp4Vn1S0KPGvnNoL8tcMS3kFh34XyPpB3TjkWkeAEBD1\\\\\\\/yBd3ynNb9rFCUh6Vg+o8U7qADM0lG6bcRHymWcqrqSDiPaNxVWQpSa4mWlHJZFxUku5qn2MLORUMjBwJ1Lhq\\\\\\\/dZlTDGKJDyElj7l4+hXg5EUGrCTzQp5htHLbm4M8oeTz+QFq6uXCnHRwLBkpTKjShwaT5VWdb0iIZ9kmKhxOn5RgtiisDz23mqiuGjMooj4ydW2pxkesR31tm8ZXHD15v9Yl8KmChcuVhzhIWsvM4nZKsFiLQsxBD8ET5OPtSugM0ydNp7yE7BPVPee1B\\\\\\\/QxPLMEm3mHJl43gOOLmosWG4u8Knpdoo0g5ashxyDdPhPoQ++DQPdLsEa7dir3n6+Xxr4566ztCibdslz4iyQNjt6tSVM59bx0PK4A1P5iXQ+amtm0m5lhr3gQ4YbbdWyz\\\\\\\/C3RyWply1DqnHIaYrbVMoB9Eg9R7rxyzeCwunsqqG7RWp\\\\\\\/xF83n0CPL2hNF9zLiWg+94MOtwTDlT5\\\\\\\/bIRi0UQSPQHiqpdv2gID0AAgW7OZLqx8NjQ4mbXUPQZBE8HrdhJLvJ9Yi2ikGtPn\\\\\\\/Q9EQNSZwrsRoErBqSfXbvmtjSsPYsxVJTVPsqpg+PzrwEJr4QltqflJiBEtUxisH+7xXC3Z2RGwbBroc4TQZ3nBYWK5i\\\\\\\/aGV1h8IDbr7\\\\\\\/eUkYznTyNSeDs2iltqzIM\\\\\\\/7kFQ8SXNx2NiaHBCO5OuZcGusr348xDiD6k6zBXzhbGRUHD3lyW0wSpHKsyMxtcuLMUZNYeKEGpfH9nizZ6PsVkQdOaAFrAgPB6LxRoLTKn8RremSaJYVMyPkG+yfQT4NqwJwpN05uAxSzvFFCwfvlNJol1FJHKxZT7QMVIU7+CGYnNuXEwWdpruVxoqPLzOFM19aOxCjU9gUmVEKaTFItcEre0tipphuNCdU5DSUJ7U\\\\\\\/euAQi4yL2xljxLf6NrwLxTtirYWRN75D8IFCPgzstSPFa0vsSf7wu95oaj4RanH2UommP4E3y6GwcYdL5qnpTMg44oOGMgC8Prze9GgdZNhdbWG8YL9g5U931bsRPQjvMgQwAxLf0dHlClLdbTJ7ku3XZcHN3qYu93DqbR44oJR0v7rD7Rx4ioS+mkGa21+hVFL8E15Yqn0V\\\\\\\/F0QSgOCBsCmAS\\\\\\\/A5HB19t9aVxNRa\\\\\\\/7h8kKK8g1MkCMezKpdczXEfTqFynESJ899tdEpQbuc1bl5gWiRBeeINuyORw7uqgySG9g0\\\\\\\/poiWm\\\\\\\/z4hJt0fF\\\\\\\/xhAdINJcRd7vMie5kl+N248YJrCEpuKL39QpiZd4ZNHp3+HtpU4zMYzFQ1fhR6\\\\\\\/7+TwC5Po9YzZe1NiBpfeWMSFuZyp6ZSidao63eplaTZeYW2EWqRK0BxTo4Cn0AKg+GI2yrrVSlZcoigXiZa5ZV0gQC4Dg80u04cnVFIQdCjuGGjN7zb1A0U5aMkXlJ1BdF8DD9hk2mbGhkvTgp3Yn\\\\\\\/sZ5l99d8Dx6Wf3JT67BLYVKiSAI6iSS6M0Ty28orBI9bUkbl4tMIBPJ4xMSZUS9fIXv0YnRZ5bjOB6i6RPzHjTJ4TElVMYM\\\\\\\/qmmgVTdxHXRJ+1foMHpNJNULWf42fGSvp\\\\\\\/Gnvu3pqi67ZAUlEJgEoXEugGZ9VXS1U9ofl0sr1M\\\\\\\/wU7CyyGriU4dgQT2Q9ZcSgJE+C5Zrf4BzXa\\\\\\\/xhkIjNbGLbFLYQnb10eoimdCjxWgYXEcjTzBY7O2N+XCsdWfCwG8ShWWy3\\\\\\\/TVAw6oYyKlWZsP+Aya7mn8f3Ib5R30vrfSXcUDqaJsxVfJau2fEPnJXzfs05SI0u8SlpXubfzJnJYtv9y\\\\\\\/ebMu9Mf+8gcxVbazhSST3H6lq16vrSFGN68Xozg6PJfGSZ+52GgOzQ5lRTEjH7jCZKhi7DIiQdRDEoLuoA5Sf1XctsNq2Dz5ubtXW78xWXySP+G4vWAEraumFTMfDOFrPG7UlDndv8c\\\\\\\/qtAknMi1ZthduMF3uib\\\\\\\/oAvORpMMkFm68QT\\\\\\\/n\\\\\\\/BNhMIezbbvTjEIK3WWOlsFYL3DNykCztNHUT6wmalfdW+qaF1k7SRW\\\\\\\/XMfqNpmjVrXadJbDHJFtTAcZmihiHtJSifuDIymPmsXs7+MpWz8T1ic5YlICJal7MzH\\\\\\\/iyTrXXt\\\\\\\/8QDQrHfaBpUiVunLEGZN+NeZSMF50t3DhngJdbL8caKbEzxvUBJIxuoXoV\\\\\\\/E3VBKLwvbM+ICU1fWLnEjTT\\\\\\\/pB8rtvDuCuN3eku+ArzNRPZyvNutLVM5FRhmTRvyZOE8WEqyYYw0AZ12PfCRcls1JEmdqy8ohzcfIxy6aSxJxcuSQBzr7aR+DmPWa+SnDvMkRkMLfyia32N4mjV3OqoeBmewlHHSIUCf6\\\\\\\/jhDdFjZBvzzg+kv0IRKilicN3cibQhKxZryL04i\\\\\\\/K1NNZ\\\\\\\/Rb7YuXSH5tj0vcuOPNM2g3BY1KJCrYx78VUwgeJqaKW7Y27SlFCEH1UxtMfhJjob65o4nrgc04lIQSIOWiSwNtMTpluk6Ekro\\\\\\\/Yb+tPBWJdmQhqtqxGyFN8GX0gQgJJGqdYIlGlnOTsFpztkTu8CXmL6MqVrMfYnfzpWfgwlB7EvJJbO8zQ4ZT0\\\\\\\/IEHIbXQYqNusZavayNyl4fzHpZAKwJgLzQKTTDJw7FDdddcYG\\\\\\\/Hyj4uj41ywiQpXjhOLdWjm7wLpC02wWBXGxxi0CFriL8DW1zdqt009KY5NKH2MQj5zo7il7m+Jg7s5sm7Lj9DlbMiCkK1u3ff7MnaEEa+n+J1At92SRBhWi+ZoRrH\\\\\\\/ukb+MkrM7JfHDMn9L2Lk8LBDc5QrNN9LlZWsvU9kVD66JWq6p8ActtR7reyu3MVtwKPAQRByUcFC+MPMWf4DuwTFXp4XRK2y7tKreuS6wseK63+KYwF+8\\\\\\\/fyv8C\\\\\\\/done5sise1HQ+jTaWBpN3B6SundSlyUmbJgv2gE+kdvZHEQBcl60jRvhM7wYz92XGP\\\\\\\/JNk4tpevsG8UQrYfOH3a72Uxu2P+2Mp1pkCcRLT\\\\\\\/FRYaBH4Z+kaLpWDspdfQyjm3e6Zajby2\\\\\\\/Mhh6mtNhKZKMhvakGQTuY9bYCkgNjjImQ1NTGGICbtXRuQ0aHfn4VhEgdtOrQcQd99mZg91FB2scddwMhNwGt5ooEPqOhh00ZRXmCPNeJ9nijmh6\\\\\\\/w8bKbCrLd2GpnDzL8HVbDmALWsOaP\\\\\\\/do91Nx5jhAmyvXPcdrDesrB68kYuWsT0+1WP4O17s1y2IkQWSFHRRKK0yhZHMZpo8RFfoTN6PIXE\\\\\\\/p75CfAIh5dtoGt5PysU+EmNglNLVNfJPP5PZ1QgwdfyYEUbCJm9riAIAembE8lv82zHDXUQ3L0LqIyRe2MU5AgYFSZu70c9GvOFjYBjUbl9Wk2vsAuy3bDA3P0ZYpP3KJh2r0+qYbwoKTmAAWpQ4rJk2yXsp9xJak2qnCoRB0tFb\\\\\\\/+aXQQffGbjaMoU9UYciuZJzIPfxOvAB2r927akWf+bji0eouEi+m1yXegrwRW60lyjf4if48BLXk\\\\\\\/yDINAQP4BRHM7I3bB7hiKw9cGz8Xi+BlJ8RP8T5ZMEZNiMrmn9otQyFVAVF5\\\\\\\/5UvQhWzF+a3qejZJdRbVONP8HcIqCIajhqX8vXl1fnVTkuDn7eGGdwwLOyCvFxFqlYTzUxWB2ufTLzxzaRbMc+xNipeLc4lTGSvSyBoyifVufOOQMmGXl97RK\\\\\\\/Oyn0ZHHrE2Li5fISMnFjiqNzyGykzmKc8wLTU8eClhx91vR\\\\\\\/ZqGLaoqePypPn2j2m2phGEjKb6GEKVTNP2IOhOT1icuFtYHJ9LSKzgeiTK2zsKLxSohHk2t2C2lvKK2yHoo8ukoYnxoNVNEFtCGHUR\\\\\\\/RZ6FabhtzEjYX0\\\\\\\/K+dNbAhgBl+fxHbsnygF5xS2gw2LSzeW1gT+Y9FE1j3runsbgyk9gL8QU2xQD1PBPM9J5G\\\\\\\/30Sf5huIlAL2EYf8UcY5rA8718u2sd1UTPTAesEJP2CMis33c673PPS4IIWI0tvC75eSoObSo04KJRkyTbyh01yXhs3H8MfqwhjlKEkU4lA3E6cLAwnZD78nwX45RYudbmVGK65kz6yTb+ZXrYcwzwVHG5J1nE9Hs1kg08BU+wI4r+idfizfKUXyN01gMSSvpThGvq3IjY6GCPlZACKSMevysgBv2UaEt8bMFqiy4xUOCuNYMsrH0x6w5cZuENTf9ywBwCG1rMWqPW+Zct2mcUqhnts83usiNnIEum+ytAKg1PWh8OD7AdDg2s7x4fJ2NE3RrrtjUfHKfZTvHEWwBAm4KdKCEWnaPbh+0Mp0ouN6\\\\\\\/JjXuvpEzPUP7HLkQzm2EJ09cCNxqE7+3kHozfqBrHDoWq1d6GBI8rJVHAclchFPqQe2l0kA08FfvlPSXYD+x3o5TTXnAugziFa5PC\\\\\\\/\\\\\\\/\\\\\\\/K\\\\\\\/D2O0\\\\\\\/AOxhsVLRsp8TV7R9E5IhSTV2g1+UHPG\\\\\\\/G49Fu8pZxTwVCH8TszLZhWTLR4anvLcPAcO+3VijPHZR9io6bTZ5MxR2HM7lM8azUjAGVT7YCwVE1pBq1K5KFGG3ViBiQI4bGdvyN8Ys3rAKs7FROUZ2fb1+bGnbFdW0lGUFBT15gl88r\\\\\\\/PKdoIyc+3bTH7Xi\\\\\\\/Ueias7g25dHkLFmG9oLXYXtcSqMF0TjHPZ3pnc7EHv0ic557YKo+BNrD\\\\\\\/0HEPVNUtT\\\\\\\/Q\\\\\\\/PYHG5Jt2n7fnGjpRmRbs3Vr3OnQ9NCG5HUSpsBNOA7YXAU4H94PA3p7kqTz1V+\\\\\\\/ChIg1V\\\\\\\/nk7b5NR6\\\\\\\/uV9JlkBHM1rqVKKGsaTBu9P4j8G1q1CHwt+XQUA5FOtcfgbDBxxiaDzV8p2F9ycItDb9TsY9EnqxsEMYMpdynrhjVE09BMwR1lXbXbWZ6C1neIC1WfX0hPARKJufJjkp9JGhzxHLTG9D4D7rrrZ8Two6O7i7lanzIInrZOLAchW3vImG28vRoB1tvEp0juGztC1OTfBwNWe2fJY2eHpqdW01Y0pI39J3fRnOzSQbxm56RGdcw109iDrrItWu2w7KCKE7BN3bOB3U0k8l9eLb3UmOotPLHhzCOCm5mzJakzTYkGnCEOfOkpdDMODLOrmca8UJ3pjzX+KqL2ARmBkNtS4SJqoZyuori9CmAZ94utBfLhHnhokQAkZz6TPIwuCQUi3DdPmcTLSPpQQqkBuTGt+wxx6X6wsDCs2xMIbEs4\\\\\\\/sPYeVnPzeEDB3tx9KHT0tTvx97wu26TylQvSj2opH5TALAoUAeWnaekSeA3PLkI6BuU18rtqwgsQAia4qCZfYgfLGLlzjhZ+NHNMkaevIXbO5S+bw4qzmdump98qWFaWbldsacrVOOPykGNS7d4NymCFFfbElXul9joQdmEvksM2nPOvIBD1i5oDht5WRPIhfoaekE2dwRfDHJiAZxcuZ7EUQ\\\\\\\/\\\\\\\/oRuTPhRjNYKkjKT+SBczkv+t4YoGxRo5aIFvZVFHzD8eIlVXSejxz73zRGaSCv7WXsuPbo+UdsZz6AmCbzfSPlRWsIOKokDWf+UP7V9wwFNCOkWwV0umVM1NKnbeSckAV0gWl1YDvoTaUrn63\\\\\\\/Do\\\\\\\/9f2\\\\\\\/zmz2Crp3KXARNnEsYtSitiIzmNRSM8VLFZBt2OrhzVP9lTw\\\\\\\/+6B9cnYk\\\\\\\/vFTd3b8oO9F6DBDrmlZ35unuy3+XP6x+0aJWg6PMD\\\\\\\/TwyfmYTsVilvLJAl714j7lN70dgkLY2Xy7XJDimFNkf6UqULp7ltO4U2eGUdFm8FEVLL0cb83kEsc7aIbAKYSEcNFG6Kyi2Ox2Go1tp8Mlh+yVcKpn2YSGAivPS0J2buUm+gH0NSBrdiD0H9ZcrRhfgGUPgahKhtZmatukMtLxavpIUZuJDxsycntjwtvqxzZOkpLi4r7BUwNO46zaJRTa1Ot1siIPd75e+Fvkpep89GVAoi7nfz5kgTDqXQv5Lgv1+YM0aJ8sFAVqyC7dm+l50Id\\\\\\\/pLl75muhao1hOZ3CGAlH1Vi3WshPrS6vL+7\\\\\\\/x4cjAi4I0ptRNr9TgG5tI0v3MgKJFAMMeMe5kQhrEaanqvj6X6Q2VAZZQptwxchI7MANMueCnM\\\\\\\/RxHWufIZpUeyJ\\\\\\\/5h7EurFC9\\\\\\\/ZfcUa6eG3nEY2RZ80D+2cxFFQA698MktHCBpH4Lii\\\\\\\/XB2a\\\\\\\/C22GVskZnGuoUGkt3vDa3YqUIP9rsYBu76YyjctwywrC4eta+hapffKBg3wwpavQmcqnFwYxkt\\\\\\\/CGgE3qItPcAmZOsWGdpkiMW0Ys9fz0WruLEqA6v9+aYMfxpLQYqnwiC0AfWWwLQdHztzIvC5FQhxDKN+E2BEz9Xud2YWpZ9Ah76Tyu4bsv9CzL77GRbhwFhVAnY97UKDYPSxLFMDOxeWC6KKAWLwfcWkmPzBvzyu9RnzkaSfNIU\\\\\\\/aUCWNTa80862msuFglX5zb4AjZ\\\\\\\/PXbZxTBupoQRaCv\\\\\\\/PK9lS2z\\\\\\\/J1izw1slz6v5I6r5JCkMyr98XTXw3+woUjouy+LN+jfzW9BAZ33p2SlV2tbvBpdysHcu1vQkke1OPZmkUWjODRJjSVl98XSFSJ\\\\\\\/gVeGd3GXksy2KSvvxVUIicxNk5xZ10RqPUfYmg8GWay3TJDMP\\\\\\\/NkBnbXefmQuN9oxrFm18oikCVly2sI3GlMqGHr3O6K20Ez\\\\\\\/ImN6jDT\\\\\\\/eVZNUthRsisic05utGYxLbDnXd5u8Vxesvh5No6wrIMrWYthaDoaObhpql2xWUxsjPfAcBo\\\\\\\/Fh3j2udzWUFX8Ym210C5FQS+rIZCc3e9TaFBmgcOLvY5N50djtVVUAym4jivztlO9Au+zjj50JUQSayDMAVzWvWN4DV1mJt0tdhNN7j\\\\\\\/u700bmnmQaqDYiug\\\\\\\/PpqPgxge\\\\\\\/jVUOYAoGkmtEgT2Btm14qf6kvl3oSEFyhVFOt84LXOtGu42O7nlwp3KNac5yVOTq5S2DTlQrjNM9fE6JaPz78keX0wc8nLgCYHjtA6z9Ef5QNWkf\\\\\\\/IWILauexx3DNhnM\\\\\\\/N\\\\\\\/PWlXreh+seFDWN2dhvJk3l77FaNnW8KPooP4UjmpH0lYUXglT2ocm8to35a5JMRPqKvHIHGfEjkyRXEExo0SGXEmBvHzP2p\\\\\\\/lIi5+XfjCzY7cthP0sF9FTCGIEbwJ89am4lfXh5e9KlOxzMcv0s0tPFA07AbbtwK9atYXC1OEbUTliZHX6aRBpvtsTb253pwWMAW2FjYh4PB59NMnPXxx\\\\\\\/QVeLq2HKXf40dDX8rZSgKgqVAHf8CKa8tbGTbpL7yrYliTCE2VUjke5wCpFHnZUsICvHa1MKKiriomNsF4n3G1E9sejIx4qmF7a0BsfQO8IWI0\\\\\\\/vgoBAONu2wbz3O9lh8FPbcqx+8Wq\\\\\\\/8wBLd3xy+lW9HuIRegF9oDaULxoWpxSy4qVhEbmlo+2Q3oQviVZx7isEqWl+gRHEfM6EFBZfLRyVoe7mHM9qNh73h51NOr9qV6Y6wJXcFeAlVkeHxhLPv22Dehp1mX1eaPiHNJuzB\\\\\\\/k4J9wNB0fDGk2S+wclcMPcb3sV2eb+sUZx7+W9Z6GxZTcWf1\\\\\\\/Q8W92EnPakPe6lhoogYmfSYcFbahkvY97qubbI2QeCimDk68PP1ah57Voys9+yOhPDH26xew9frKsBcbQjG0OyJhVusP4kRQaxIxnafFymzVulzV+kYYDtOo5J2zsciEmcSolStQxM5Qi4bxWhRFB6fQmv4nrY56uvHCEzAIWi7cRExtck4q+0ROjnZuW6DJQi1k0norNviWd\\\\\\\/lbIJhp2O9AAY4jbGxZOk515FUHkYwdsPGaF6Rb8InhrVk9te2wNs\\\\\\\/5E3vxjB3YaM1YPiAq+aK0hd+WXcA37p9d371Mk8tW\\\\\\\/oIGX+yiFS\\\\\\\/Zfm\\\\\\\/lkaPysJZjL\\\\\\\/j7ovLkB5vT+dOxr4RGmz5SridNq7MniPBipxPqnT\\\\\\\/GC6V6Uiyfx6ICKbc96ssVH7FkdrTKT4fG3TSkcZ0dYEmRFLrtX32AAipf+N2GfeORGG8Zj0SET8fFAd6rWqCGlCGLqJC\\\\\\\/yJwH\\\\\\\/OR\\\\\\\/Z3JOAt0jW6WOiuHMW9qPVxubnlwGJL8f\\\\\\\/M146nA4wNL5eV2g\\\\\\\/K8QlaKrYig6ZR5HgzaUwZnb7Ii3umX9vM0D9Lpz\\\\\\\/fVGN7sTdkr6w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022b5IP7Prfhh6AZv0ZGuI5bA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022129f9bc4befeed16955352f819812e1f1f7967843a26b5721a94eb09d250a0de\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BSOxtmQGphNaiO5D3ZJjPzeSkeALrDfZzHUKaq33ws3hEBvI7BYcdKIUJYNAjVkxsL5LliqRoRprRhOiGZWPtwejovAyJJ0DfLm+n8hN29Pr5IAV\\\\\\\/xLimfxttykt3WqTpbQut+UIuYL1IHLQqYRThh9f9N4JFvMMcpNkQFriSInpZSLK0RiBADMy4wqaHZ\\\\\\\/t58W66QXynuFU9qKKxhnKDTA7jB8IwgVZNXlS9vv357g6eAySEjnXhBpPWwbX2gC7xCbuO+Aig3wfCU53XoHdMIz6VnO72ano6D82dBaQMHbGeouF4G2nJPPUr5EpFnlp1A4lEgdUPyphCqo0nINfxNk1aEp3z4QZpkAhGtEVarNNC+a3d\\\\\\\/4CcyygRAxetzwzVb2+M+1BlvzZ8BKgAZCEN61XmLCuGhADsRjk+zD1WdyM0Ug5FUhrIa+yD2cEnqbPKXJg3RPuHeYpVKjqkT1I6l\\\\\\\/GT\\\\\\\/k6qxCT\\\\\\\/vKKhcU1OZc9WdaiKbpssRgcNQj+WSujavaMriR\\\\\\\/z62oGPz71fcfehfg3w2iFXkVDVnrNvBDXrcvVtT6RtmIeTY4rnL1DooHnmEYsEkam4nxd6vHEp4FWTIEbhQSdwa++3B1qf+2J88UcHT\\\\\\\/J1Up84MQynjoDNiqetVpgonuLJ\\\\\\\/CvKxcDSm82T0wrFkZeAUqbuf26SbCUTLt\\\\\\\/\\\\\\\/iEBVQHzpGrI13qRLzUk2BrvE\\\\\\\/Hn13ILNJQu9++ElCD2IvTXF0Ft5zDD2tKRw7bsahPwm9tqgLb2H53GibRsUNnpuTsCBFrH8hd\\\\\\\/LqAHkzQqLwpCV3DS0vwvQ97x5qaEeIkJTqNMhYjdWwnVZVWQedTQy+wTBwINTrh1xmFvWomQktS3flzJnginNtpNpHh\\\\\\\/VhKsLgvh0kN1w8dIKE5QecKHRcqvd4XAvzbHpLv5OHtdaJf0pMVdUtl4I12FjC9Y6\\\\\\\/oN\\\\\\\/LVU\\\\\\\/4Tq33RBRx+uj7vqwTxv2jOnFKdKc\\\\\\\/Op7BywRg0oH73c+QJOtsH1Swov04OCEFSABL42OdReMPLGhxOQzsgVKJ6aL\\\\\\\/mKs2hyMd9YoSXcGmXS+LYFbaHegABD\\\\\\\/iTTNEncMQS3QVr84K24pUjn4KPyjHpHznwHuePQPM1DyzMgHMGWUt\\\\\\\/+D+GadqNi2dQHaT1MkspLaL\\\\\\\/YjLwB8LD7smCLXpiQkdGdZxGvLUNSvmTF7yiWnnK0dyCAXbykeYjG6KcpjhVzUn6RrdUp5bePiOelZvK4En2WtcAcT4lFApLtw7lcO5bWHCSAe9AgySHFEPYiMBcF0OK7CfRMLPoGxEug8a7bafoG1vDk6lDGarOqwL7AAPOZ5K\\\\\\\/2mDE72koq8w\\\\\\\/fODLMv0O8\\\\\\\/VZkcTt4hbDx7DWO9CnWk3wHySjlVmwz9cTwVLbt6BCR\\\\\\\/5ZJuzI5qsEnrb5mvLVqibknsabWQ3H1FAeYebSfyEvvS59t1c8QMLkmNEsJGeBuUjCUetLbn0OW+O+iaFudzI\\\\\\\/w4im6Tn+VIbiDwBE1i+ChBlugPGeRmcc34jvj7wCbY009ewxXSXHhlUmvGagDs3OGsFAtRgCsQDy+yCsU9JllC0y2TdSItDUKw12OLW0oQKMacMmcXdpwzg67Yy27lBWAS9JXH1BrrMwO0Gt94CcDtfv042FBBu0ClAumb98OoPRJaBhx\\\\\\\/mbqAeHRgRsG0xdz3BMicswB6UWzgBpR6JUK42xrZDkdNiIJLxw0q40ZidaMol1vb5zcIwmH0LRofL1bAmyYRFmNihr6hGo\\\\\\\/WmCwL6ZWzWUZxO\\\\\\\/OyZjwDK0GBhAW0QVCc85CfI1uHNKQUwPOnTiWmkZzLtKmBeBueIaU5cLZViLRZ70pDnuDbstQC2ZSUM0PcWkxSzzlICmVV9HXQdfi33OvzDFQBGqZDbBroCFojcnR1fNmI0mfnNrKaIRmJnjQO2HdmuFVi24XMRY8Uw2MNoJFSy\\\\\\\/9Ynn+eku+Im2HFum4rnSrDEEK7yoXeCbGFGYhhXcyVWGYnON\\\\\\\/4uUIFV0hvuLjgA45SExVCntFg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rZKQr7bViYytOunmoRoAvA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022828aa09dc8a4837a875c827a2b9f7e92e607aa67e03cbb99dd7f695ea0ab279c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459663 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1569" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jaaHqBbR-PqL64RGXBFIMaRhPu6MLHy5bIV83kUrluY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1569" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--glWY1aDubxSGD8RmaxfEvtOeUsNArUKOCDHkyPcnTg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1567" title="GENETICS and G3: Community-Driven Science, Community-Driven Journals" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1585" title="Adult Plant Development in Triticale (× Triticosecale Wittmack) Is Controlled by Dynamic Genetic Patterns of Regulation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459663" id="node459663" data-pisa="ggg;4/9/1569" data-pisa-master="ggg;g3.114.012971" data-apath="/ggg/4/9/1569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Arūnas P. Verbyla</span>, <span class="highwire-citation-author" data-delta="1">Colin R. Cavanagh</span> and <span class="highwire-citation-author" data-delta="2">Klara L. Verbyla</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1569-1584;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012971 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459663" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arūnas P. Verbyla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Informatics and Food Futures National Research Flagship, CSIRO, Atherton, QLD 4883, Australia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Agriculture, Food and Wine, The University of Adelaide, Adelaide, SA 5005, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ar%C5%ABnas%2BP.%2BVerbyla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verbyla%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAr%25C5%25ABnas%2BP.%2BVerbyla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Ari.Verbyla@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Ari.Verbyla@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Colin R. Cavanagh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Industry and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Colin%2BR.%2BCavanagh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavanagh%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AColin%2BR.%2BCavanagh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Klara L. Verbyla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Computational Informatics and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klara%2BL.%2BVerbyla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verbyla%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKlara%2BL.%2BVerbyla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1569" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459663" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459663/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1569.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459663" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459663/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1569.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459663" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459663/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1569.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459663" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459663/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1569.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459663" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459663/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Multiparent Advanced Generation Inter-Cross (MAGIC) populations are now being utilized to more accurately identify the underlying genetic basis of quantitative traits through quantitative trait loci (QTL) analyses and subsequent gene discovery. The expanded genetic diversity present in such populations and the amplified number of recombination events mean that QTL can be identified at a higher resolution. Most QTL analyses are conducted separately for each trait within a single environment. Separate analysis does not take advantage of the underlying correlation structure found in multienvironment or multitrait data. By using this information in a joint analysis—be it multienvironment or multitrait — it is possible to gain a greater understanding of genotype- or QTL-by-environment interactions or of pleiotropic effects across traits. Furthermore, this can result in improvements in accuracy for a range of traits or in a specific target environment and can influence selection decisions. Data derived from MAGIC populations allow for founder probabilities of all founder alleles to be calculated for each individual within the population. This presents an additional layer of complexity and information that can be utilized to identify QTL. A whole-genome approach is proposed for multienvironment and multitrait QTL analysis in MAGIC. The whole-genome approach simultaneously incorporates all founder probabilities at each marker for all individuals in the analysis, rather than using a genome scan. A dimension reduction technique is implemented, which allows for high-dimensional genetic data. For each QTL identified, sizes of effects for each founder allele, the percentage of genetic variance explained, and a score to reflect the strength of the QTL are found. The approach was demonstrated to perform well in a small simulation study and for two experiments, using a wheat MAGIC population.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mixed-models" class="hw-term hw-article-keyword hw-article-keyword-mixed-models" rel="nofollow">mixed models</a></li><li class="kwd"><a href="/keyword/multienvironment" class="hw-term hw-article-keyword hw-article-keyword-multienvironment" rel="nofollow">multienvironment</a></li><li class="kwd"><a href="/keyword/multitrait" class="hw-term hw-article-keyword hw-article-keyword-multitrait" rel="nofollow">multitrait</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/wgaim" class="hw-term hw-article-keyword hw-article-keyword-wgaim" rel="nofollow">WGAIM</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-4">DISCOVERING the underlying genes affecting important traits such as yield, quality, disease resistance, and climate adaptability is of paramount importance to increase the agricultural productivity needed to feed the world’s growing population. The first stage in identifying these genes is QTL analyses. Traditionally in plants, biparental crosses are used to create experimental populations on which QTL analyses were carried out. Recently the advantages of a different type of experimental cross, the Multiparent Advanced Generation Inter-Cross (MAGIC), have been explored. One of the most compelling reasons for developing MAGIC populations is the ability to conduct research on a broad range of traits in genetically diversity populations. The genetic diversity present in MAGIC populations is maximized through the selection of multiple genetically diverse founders. The founder lines in a classic plant MAGIC population are intercrossed until all founders have an equal probability of contributing to the genetic makeup of a line (more complex intercrossing patterns can be used). In plants this initial intercross is followed by multiple generations of selfing to create recombinant inbred lines (RILs). Such a structure leads to an amplified number of recombination events, which means that any QTL can be mapped at a higher resolution. Consequently, these populations are now being utilized to more accurately identify the underlying genetic basis of quantitative traits through quantitative trait loci (QTL) analyses (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mott <em>et al.</em> 2000</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cavanagh <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Trebbi <em>et al.</em> 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bandillo <em>et al.</em> 2013</a>).</p><p id="p-5">One point of difference between the QTL approaches that have been used is whether they use marker scores or founder probabilities. The structure of MAGIC populations means that the probabilities that an allele has been inherited from each founder can be calculated. These probabilities can contain additional information as the marker scores may not be fully informative in a population with multiple founders. <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xu (1996)</a> described the first QTL analyses for a MAGIC population, using marker scores with interval mapping to analyze a four-way cross. Contrastingly, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mott <em>et al.</em> (2000)</a> found that using marker scores failed and described the first use of founder probabilities in QTL analyses for MAGIC. Other methods have used probabilities rather than marker scores (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">King <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> 2014</a>).</p><p id="p-6">Regardless of whether marker scores or probabilities are used, most studies have employed QTL approaches that use genome scans to test each marker or interval separately for association or linkage with the trait of interest (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Xu 1996</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Mott <em>et al.</em> 2000</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Malosetti <em>et al.</em> 2011</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">King <em>et al.</em> 2012</a>). An alternative is to use all information simultaneously in a single model, overcoming the need for genome scans. Whole-genome average interval mapping (WGAIM) was proposed by <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Verbyla <em>et al.</em> (2007)</a> for biparental populations and modified by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Verbyla <em>et al.</em> (2012)</a>. WGAIM was shown to outperform composite-interval mapping (CIM). The approach allows for population structure to be modeled and for any nongenetic effects, such as experimental design terms, to be easily included. A likelihood-ratio test of significance is conducted to decide whether selection of a putative QTL is warranted. Forward selection of putative QTL continues until the likelihood-ratio test is nonsignificant. WGAIM was extended for use in MAGIC populations by <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a>, utilizing the probabilities of inheriting founder alleles for each individual at each locus.</p><p id="p-7">To date, most QTL analyses in MAGIC are conducted separately for each trait within a single environment. However, separate QTL analyses (for any population) do not take advantage of the underlying correlation structure found in multienvironment or multitrait data. Joint analyses provide the opportunity for a greater understanding of genotype- or QTL-by-environment interactions or of pleiotropic effects across traits. This can result in a greater understanding of traits or environments and the relationship between them. This information could lead to improvements in accuracy for a range of traits or in a specific target environment and can influence selection decisions.</p><p id="p-8">In biparental populations, the advantages presented by multivariate approaches have led to the development of a range of approaches for the analysis of multienvironment trials for a single trait (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jiang and Zeng 1995</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tinker and Mather 1995</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 1999</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Piepho 2000</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Verbyla <em>et al.</em> 2003</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Vargas <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boer <em>et al.</em> 2007</a>). In addition, multitrait analysis has also been considered by many authors (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Korol <em>et al.</em> 1995</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">1998</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zeng <em>et al.</em> 1999</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Knott and Haley 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gilbert and Le Roy 2003</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lund <em>et al.</em> 2003</a>). <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hackett <em>et al.</em> (2001)</a> present a review and an interval-mapping method based on multivariate regression. A more recent review of methods for multienvironment QTL analysis is presented by <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Van Eeuwijk <em>et al.</em> (2010)</a>. <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Malosetti <em>et al.</em> (2008)</a> investigate multitrait, multienvironment analysis. An approach for multivariate QTL analysis in biparental populations based on WGAIM was presented by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a>.</p><p id="p-9">In this article, an approach for multivariate QTL analysis is proposed for MAGIC populations, building upon <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a> and <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a>; the approach is called multivariate multiparent (MVMP)WGAIM. Multivariate QTL effects are included in the model for all intervals (or markers) on the linkage map simultaneously. Probabilities of inheriting founder alleles are an integral part of the model and allow estimation of QTL sizes for all founder alleles. These multivariate genetic QTL sizes are modeled as random effects with an associated variance–covariance matrix, be they traits or environments. A likelihood-ratio test is presented for testing the significance of the QTL variance–covariance matrix. If the test is significant, a multivariate outlier detection method is used to select the most likely interval for a QTL. Multivariate QTL are chosen in a forward selection process. These QTL are also included in the random effects. For multienvironment QTL analysis it is possible that the QTL effects are the same over all environments. A test for the interaction of QTL by environment can be carried out by including a main effect QTL. The final summary of QTL effects includes a level of significance, a score that indicates the strength of the QTL, and the percentage of genetic variance accounted for by each QTL. A small simulation study examines type I error rates and power of the approach, complemented by multitrait and multienvironment examples using wheat MAGIC data, presented for illustration and interpretation.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-10">The approach presented generalizes that presented for the univariate (single trait or environment) situation (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> 2014</a>). The methods presented are largely self-contained, however, even though the development matches the univariate case as given by <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a>.</p><div id="sec-2" class="subsection"><h3>Base model</h3><p id="p-11">In the plant sciences, genetic studies are usually based on designed experiments, often multiphase in nature (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Smith <em>et al.</em> 2006</a>). To provide a statistical analysis for these experiments, a linear mixed model is often used. This model has the form<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <strong>y</strong> is the vector of responses, which may comprise multiple traits or a single trait scored in multiple environments. The fixed effects <strong>Xτ</strong> and random effects <strong>Z</strong><sub>o</sub><strong>u</strong><sub>o</sub> reflect the experimental design and nongenetic effects while the residual vector <strong>e</strong> allows, for example, for possible spatial trends within environments or relationships between traits. Note that it is assumed <strong>u</strong><sub>o</sub> ∼ <em>N</em>(<strong>0</strong>, <strong>G</strong><sub>o</sub>) and <strong>e</strong> ∼ <em>N</em>(<strong>0</strong>, <strong>R</strong>) and that they are uncorrelated. The genetic effects <strong>Z</strong><sub>g</sub><strong>u</strong><sub>g</sub> are discussed in the following section.</p></div><div id="sec-3" class="subsection"><h3>Genetic model</h3><p id="p-12">Suppose there are <em>n</em><sub>g</sub> lines or varieties and <em>t</em> traits or environments in the study. The <em>n</em><sub>g</sub><em>t</em> × 1 vector of genetic effects is given by <strong>u</strong><sub>g</sub> in (1) and the matrix <strong>Z</strong><sub>g</sub> assigns the appropriate line to each observation in <strong>y</strong>. Without marker data, models for <strong>u</strong><sub>g</sub> are typically based on the infinitesimal or polygenic model. We consider the simplest polygenic model<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <strong>u</strong><sub>p</sub> is the polygenic effect assumed, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-3.gif"/></noscript></span></span>, where <strong>G</strong><sub>p</sub> is the <em>t</em> × <em>t</em> genetic variance–covariance matrix between environments or traits. This model can be extended to include a relationship matrix based on pedigree information.</p><p id="p-13">The aim of this article is to use marker information in the determination of QTL for a MAGIC population. Suppose there are <em>n</em><sub>f</sub> founders in the MAGIC population and that we have <em>c</em> linkage groups (or chromosomes) and <em>r<sub>k</sub></em> markers on linkage group <em>k</em>, <em>k</em> = 1, 2, … , <em>c</em>. We allow for a QTL in every interval or at each marker if analysis is based on markers; the development assumes intervals rather than markers but the marker-based analog simply replaces <em>r<sub>k</sub></em> − 1 by <em>r<sub>k</sub></em> where appropriate. Our model for the genetic effect for line <em>i</em> for trait or environment <em>s</em>, <em>u<sub>gis</sub></em>, is given by<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-4.gif"/></noscript></span></span>where <strong>q</strong><em><sub>ikj</sub></em> is the <em>n</em><sub>f</sub> × 1 vector that indicates the founder allele for line <em>i</em> for a potential QTL in interval <em>j</em> on linkage group <em>k</em>; thus one element of <strong>q</strong><em><sub>ikj</sub></em> is 1 and the rest are zero. Note that <strong>a</strong><em><sub>kjs</sub></em> is an <em>n</em><sub>f</sub> × 1 vector of sizes of effects for each potential QTL and <em>u<sub>pis</sub></em> is a polygenic effect.</p><p id="p-14">If we place the effects for all environments or traits in a single vector (as a row vector) for line <em>i</em>,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-5.gif"/></noscript></span></span>where <strong>A</strong><em><sub>kj</sub></em> is a <em>n</em><sub>f</sub> × <em>t</em> matrix of sizes of effects for <em>n</em><sub>f</sub> founder alleles and <em>t</em> environments or traits and <strong>u</strong><em><sub>pi</sub></em> is a vector of polygenic effects for line <em>i</em> for all environments or traits. Placing the total genetic effects <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-6.gif"/></noscript></span></span> for all lines as the rows in an <em>n</em><sub>g</sub> × <em>t</em> matrix <strong>U</strong><sub>g</sub>, the model becomes<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <strong>Q</strong><em><sub>kj</sub></em> is an <em>n</em><sub>g</sub> × <em>n</em><sub>f</sub> matrix with <em>i</em>th row <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-8.gif"/></noscript></span></span></p><p id="p-15">Each vector <strong>q</strong><em><sub>ikj</sub></em> has a multinomial distribution with sample size 1 and a vector consisting of the probability of inheriting each founder allele for line <em>i</em> in interval <em>j</em> on linkage group <em>k</em>; we denote the vector of probabilities by <strong>p</strong><em><sub>ikj</sub></em>. Then<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-9.gif"/></noscript></span></span>The regression approach for QTL mapping is used (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Haley and Knott 1992</a>) and so (3) is replaced by<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-10.gif"/></noscript></span></span>where <strong>P</strong><em><sub>kj</sub></em> is the matrix of probabilities with <em>i</em>th row <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-11.gif"/></noscript></span></span> If we form the vector of genetic line effects by stacking the columns of <strong>U</strong><sub>g</sub>, we find<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-12.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where if <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-13.gif"/></noscript></span></span> <strong>P</strong> is a <em>n</em><sub>g</sub> × (<em>r</em> − <em>c</em>)<em>n</em><sub>f</sub> matrix of probabilities and <strong>a</strong> is the vector of potential QTL sizes ordered as founders in intervals (or markers). Our working model is given by <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-14.gif"/></noscript></span></span> where <strong>G</strong><em><sub>a</sub></em> specifies a model for the genetic variances for environments or traits and covariances between environments and traits for the sizes of potential QTL effects.</p><p id="p-16">Determination of <strong>P</strong> is discussed in <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a> for single-environment or -trait analysis. The use of three-point probabilities and probabilities based on a hidden Markov model (HMM) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman 2006</a>) was discussed and examined in that article. An averaging over each interval was used to determine probabilities for interval-based analysis. It was found in a simulation study that the probabilities found using HMM led to reduced false positives while maintaining power of detection of QTL. HMM-based probabilities were used in the extension to multivariate situations discussed in this article.</p><p id="p-17">The actual selection of putative QTL proceeds by forward selection, one QTL at a time. This requires a suitable test at each stage of potential selection.</p></div><div id="sec-4" class="subsection"><h3>Threshold for QTL selection</h3><p id="p-18">A multienvironment or multitrait QTL exists if <strong>G</strong><em><sub>a</sub></em> ≠ <strong>0</strong>. Thus we test the hypothesis H<sub>0</sub>: <strong>G</strong><em><sub>a</sub></em> = <strong>0</strong> to establish whether a QTL exists. If the test is rejected, there is evidence that at least one putative QTL exists and a process (described below) is used to select the most likely interval for the putative QTL. If the test is retained, the selection process concludes.</p><p id="p-19">The test of H<sub>0</sub>: <strong>G</strong><em><sub>a</sub></em> = <strong>0</strong> is nonstandard and is discussed in <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a>. The process involves fitting two models, which have diagonal matrices for the genetic effects, potential QTL, and polygenic effects. The test then is equivalent to H<sub>0</sub>: tr(<strong>G</strong><em><sub>a</sub></em>) = 0. Thus if <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-15.gif"/></noscript></span></span> is the maximized residual log-likelihood including the diagonal variance model for putative QTL sizes and the polygenic effects, and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-16.gif"/></noscript></span></span> is the maximized residual log-likelihood omitting the diagonal variance model for the QTL effects, the likelihood-ratio test statistic is found by<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-17.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>The statistic (5) has an approximate distribution under the null hypothesis (zero diagonal variance matrix) that is a mixture of chi-square distributions (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Stram and Lee 1994</a>); namely<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-18.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-19.gif"/></noscript></span></span> represents a chi-square distribution on <em>k</em> d.f. Thus a test of size <em>α</em> of the hypothesis that the diagonal <strong>G</strong><em><sub>a</sub></em> is zero is rejected if <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-20.gif"/></noscript></span></span> where the critical value <em>c</em><sub>1−</sub><em><sub>α</sub></em> is determined using (6). This establishes the presence of variation that is necessary for a QTL to exist.</p></div><div id="sec-5" class="subsection"><h3>Outlier statistic for QTL selection</h3><p id="p-20">If the test of H<sub>0</sub>: <strong>G</strong><em><sub>a</sub></em> = <strong>0</strong> is rejected, a model with correlated genetic effects (both putative QTL and polygenic) across traits or environments is fitted and based on that fitted model, an outlier statistic is used to select the putative QTL.</p><p id="p-21">The process for selecting a QTL in the biparental situation is presented by <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a>. The same argument leads to the statistic<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-21.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>so that for MAGIC it is necessary to sum over the founders; thus <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-22.gif"/></noscript></span></span> is the best linear unbiased predictor of the vector of sizes for all the environments or traits for the <em>j</em>th interval on linkage group <em>k</em> for founder <em>f</em> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-23.gif"/></noscript></span></span> is the generalized inverse of <strong>G</strong><em><sub>a</sub></em>. The interval (or marker) with the largest (outlier) statistic (7) is selected as the putative QTL and is added to the model as a random effect(s).</p></div><div id="sec-6" class="subsection"><h3>Revised models</h3><p id="p-22">If a putative QTL is selected, the models (2) and (4) are revised by adding the QTL to the models as a random effect(s). The procedure above is then repeated to examine whether a further QTL can be added, until the test of H<sub>0</sub>: <strong>G</strong><em><sub>a</sub></em> = <strong>0</strong> is not rejected.</p><p id="p-23">The revised models depend on whether the analysis is multienvironment or multitrait in nature. The reason for the difference is because for multienvironment analysis it is sensible to examine whether the QTL has the same expression at all sites or there is QTL-by-environment interaction. Thus the multienvironment model contains a main effect that is common across environments and environment-specific effects that allow for departures from the common effect. These latter effects can be tested for significance to establish whether the putative QTL sizes are common across environments or are different across environments.</p><p id="p-24">Let <strong>a</strong><sub>−1</sub> be the vector of possible sizes omitting the trait- or environment-by-founder effects for the first QTL; the corresponding matrix of remaining probabilities is given by <strong>P</strong><sub>−1</sub>. If <strong>P</strong><sub>1</sub> is the <em>n</em><sub>g</sub> × <em>n</em><sub>f</sub> matrix of probabilities corresponding to the first QTL chosen, for a multitrait analysis the model (2) becomes<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-24.gif"/></noscript></span></span>and (4) becomes<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-25.gif"/></noscript></span></span>where <strong>a</strong><sub>1</sub><em><sub>t</sub></em> is the vector of trait-by-founder effect sizes for the first QTL. Note that it is assumed that <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-26.gif"/></noscript></span></span> so that the putative QTL effects are modeled as random effects with their own variance matrix with each trait having its own variance.</p><p id="p-25">For a multienvironment analysis (2) is updated to<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-27.gif"/></noscript></span></span>and (4) is updated to<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-28.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-29.gif"/></noscript></span></span> is the vector of sizes for each founder for the QTL chosen; this is the same for all environments and hence the <strong>1</strong><em><sub>t</sub></em> in the design matrix for the term containing <strong>a</strong><sub>1</sub>. For the multienvironment situation, it is also assumed <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-30.gif"/></noscript></span></span> which differs from the multitrait situation. The two components allow for a simple QTL-by-environment model in which the common effect allows for correlation between environments, much in the same manner as the simplest variance component mode for genotype-by-environment modeling.</p><p id="p-26">The process is now repeated until the test for possible QTL is not rejected. If the number of putative QTL selected is <em>l</em>, <strong>P</strong><em><sub>j</sub></em> is the vector of probabilities for all lines that correspond to QTL <em>j</em>, and <strong>P</strong><em><sub>−l</sub></em> is the matrix of probabilities omitting all <strong>P</strong><em><sub>j</sub></em>, the final model for a multitrait analysis is given by<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-31.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-32.gif"/></noscript></span></span> or for a multienvironment analysis<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-33.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-35.gif"/></noscript></span></span></p><p id="p-27">The multitrait model specifies a diagonal form for the trait-by-QTL combinations. This is a very simplistic model. It would be preferable to include correlations between traits. The main reason correlations have not been included is that they cannot be fitted due to limited information; the variances and correlations in essence depend on the vector of sizes. All attempts to fit models with correlation in simulations or real studies failed. In addition, if an estimated variance for a trait or environment is zero (or very close to zero), correlations between that trait and other traits are not defined. This makes fitting of models very difficult. Thus while the diagonal matrix is not ideal, it represents a solution that seems to work in practice.</p><p id="p-28">In the multienvironment situation, the environments are correlated but in a very simple way. It would be preferable to model both QTL variances and correlations more generally, but the same computational difficulties exist as for multitrait analyses.</p></div><div id="sec-7" class="subsection"><h3>Test of QTL-by-environment interaction</h3><p id="p-29">For the multienvironment situation, it is of interest to test for QTL-by-environment interaction. Thus we wish to test <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-36.gif"/></noscript></span></span> which is nonstandard, just like the test for possible selection of a QTL. The same form of test is used to examine this interaction for every QTL, using a residual log-likelihood-ratio statistic for the final model (8) and a null hypothesis model with the appropriate term in <strong>a</strong><em><sub>jt</sub></em> removed. The null distribution is a mixture of a point probability of 0.5 at zero and one-half a chi-square distribution with 1 d.f. (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Stram and Lee 1994</a>). Each QTL is examined in turn, allowing for all other effects, and model reduction is carried out according to these tests.</p></div><div id="sec-8" class="subsection"><h3>Significance, LOGP scores, and percentage of variance</h3><p id="p-30">The significance, calculation of a measure of strength of a putative QTL, and percentage of variance of each QTL are all assessed in a manner similar to that presented in <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a>, for each trait or environment.</p><p id="p-31">We begin with a measure of significance. If the analysis is multitrait, the vector of QTL sizes for trait <em>s</em> is <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-37.gif"/></noscript></span></span> while in the multienvironment case it is <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-38.gif"/></noscript></span></span> if QTL-by-environment interaction is present or <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-39.gif"/></noscript></span></span> if QTL-by-environment interaction is not present. Then under the normality assumptions for a linear mixed model,<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-40.gif"/></noscript></span></span>where <strong>y</strong><sub>2</sub> is the component of the data free of fixed effects (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Verbyla 1990</a>). The mean of this conditional distribution is the best linear unbiased prediction of <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-41.gif"/></noscript></span></span> that is, the estimated size of the QTL <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-42.gif"/></noscript></span></span> and <strong>V</strong><em><sub>js</sub></em> is the prediction error variance matrix (PEV) of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-43.gif"/></noscript></span></span> If <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-44.gif"/></noscript></span></span> is a generalized inverse of <strong>V</strong><em><sub>js</sub></em>, the distance measure<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-45.gif"/></noscript></span></span>has a chi-square distribution on <em>n</em><sub>f</sub> − 1 d.f. If<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-46.gif"/></noscript></span></span>a measure of the strength of the putative QTL is given by<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-47.gif"/></noscript></span></span>This probability can be calculated for the QTL as a whole, that is, for all founders together, and also for individual founders, enabling “significance” of QTL effects both at the overall and at the founder level to be reported.</p><p id="p-32">To measure the strength of a putative QTL, the probability <em>p<sub>js</sub></em> is transformed using<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-48.gif"/></noscript></span></span>and this measure, LOGP, is similar to a LOD score.</p><p id="p-33">Finally, the (approximate) percentage of genetic variance explained by each QTL can be found as follows. Consider the genetic effect for line <em>i</em> for trait or environment <em>s</em> in terms of the indicator variable <strong>q</strong><em><sub>ij</sub></em> for QTL <em>j</em> for line <em>i</em>,<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-49.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-50.gif"/></noscript></span></span> is the <em>i</em>th row of <strong>P</strong><sub>−</sub><em><sub>l</sub></em>. Then the variance of <em>u<sub>gis</sub></em> is approximately given by<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-51.gif"/></noscript></span></span>To evaluate var(<strong>q</strong><em><sub>ij</sub></em>), we proceed as in <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a> and define an “average” line and hence an average QTL indicator <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-52.gif"/></noscript></span></span> so that an overall approximate variance can be found. The average founder probabilities, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-53.gif"/></noscript></span></span>, found by averaging those probabilities over the lines are used and the multinomial nature of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-54.gif"/></noscript></span></span> means that<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-55.gif"/></noscript></span></span>For the term involving the intervals (or markers) not selected, we simply average the founder probabilities over all the lines for the nonselected intervals (or markers), <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-56.gif"/></noscript></span></span> say. Then the total variance of an average line effect, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-57.gif"/></noscript></span></span> is defined as<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-58.gif"/></noscript></span></span>The percentage of genetic variance attributed to the <em>j</em>th QTL is then<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-59.gif"/></noscript></span></span>In practice the unknown sizes <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-60.gif"/></noscript></span></span> and variance components <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-62.gif"/></noscript></span></span> are replaced by their estimates.</p></div><div id="sec-9" class="subsection"><h3>Dimension reduction</h3><p id="p-34">The dimension reduction discussed by <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Verbyla <em>et al.</em> (2012)</a> and used in univariate MAGIC QTL analysis (<a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> 2014</a>) can be utilized in the multivariate situations discussed in this article. Thus a model that is equivalent to (4) is<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-63.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span>where <strong>a</strong>∗ is an <em>n</em><sub>g</sub> × 1 vector with assumed distribution <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-64.gif"/></noscript></span></span> The model (9) then generates the same variance model as (4) and the predicted random effects for the original model can be recovered from those found using (9) as<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-65.gif"/></noscript></span></span>with variance matrix<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-66.gif"/></noscript></span></span>and only diagonal elements of this matrix are required in computing the outlier statistics (7). Thus the computations for model fitting have a dimension of the number of lines rather than the number of intervals or markers.</p></div><div id="sec-10" class="subsection"><h3>Computation</h3><p id="p-35">The computations were carried out in R (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">R Development Core Team 2013</a>), using packages asreml (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Butler <em>et al.</em> 2011</a>) and components of wgaim (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Taylor and Verbyla 2011</a>). The required functions, including a likelihood-ratio routine for testing QTL-by-environment interaction and summary methods for displaying results of an analysis, are in the mpwgaim package in R available from the authors. Note also that a worked example is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/012971SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS1.zip">File S1</a>.</p></div></div><div class="section materials" id="sec-11"><h2 class="">Materials</h2><div id="sec-12" class="subsection"><h3>Simulation study</h3><div id="sec-13" class="subsection"><h4>Genetic data for simulation studies:</h4><p id="p-36">The genetic data were generated for the simulations, using the mpMap package (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Huang and George 2011</a>) in R (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">R Development Core Team 2013</a>).</p><p id="p-37">A classic four-way population of 500 individuals was simulated, so that there were two crosses between two pairs of founder lines (F<sub>1</sub> × F<sub>2</sub> and F<sub>3</sub> × F<sub>4</sub>). A line from each cross was crossed and then 500 lines were generated from that cross. These lines were selfed for six generations.</p><p id="p-38">The linkage map was simulated to have seven chromosomes, each of length 300 cM and each with 201 markers equally spaced (1.5-cM spacing). The marker data were simulated for both founders and the 500 selfed lines, using mpMap.</p></div><div id="sec-14" class="subsection"><h4>Null simulation study:</h4><p id="p-39">Two hundred simulations were conducted for the case where no QTL were present, to assess the type I error rate and the average number of QTL detected per simulation. Phenotypic data for each simulation were generated for the simulated 500 MAGIC lines with two replicates and three traits in a MAGIC population, using the model (<em>i</em> = 1, 2, … , 500; <em>j</em> = 1, 2, 3; <em>k</em> = 1, 2)<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-67.gif"/></noscript></span></span>where <em>μ<sub>j</sub></em> were 9, 10, and 12; the errors <em>e<sub>ijk</sub></em> were independent standard normal; and the polygenic effects <em>u<sub>pij</sub></em> were simulated having zero mean and covariance matrix<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-68.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>for the three environments for each line.</p></div><div id="sec-15" class="subsection"><h4>Power simulation study:</h4><p id="p-40">A simple simulation to examine the power of the MVMPWGAIM approach was conducted. The 500 simulated MAGIC lines (with two replicates) and the linkage map used in the null simulation study were also used in the power study. Data were again simulated for three traits, now with four QTL. The generating model for the phenotype was (<em>i</em> = 1, 2, … , 500; <em>j</em> = 1, 2, 3; <em>k</em> = 1, 2)<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-69.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>where the means <em>μ<sub>j</sub></em> were as for the null simulation, the polygenic effects <em>u<sub>pij</sub></em> were defined as for the null distribution simulation, including (10), and <em>e<sub>ijk</sub></em> were assumed independent and identically distributed as standard normal variables.</p><p id="p-41">The vector <strong>q</strong><em><sub>il</sub></em> in (11) is the vector of founder indicators and for each line <em>i</em>, one element is one and the rest are zero. The element that is 1 indicates which founder provides the QTL allele. The vector of QTL sizes is <strong>a</strong><em><sub>jl</sub></em>, of length 4 (one for each founder), and there are 3 × 4 such vectors of sizes corresponding to the three traits and four QTL. The sizes of QTL effects used in the simulations are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The percentage of genetic variance explained by each QTL for each trait was calculated using the given sizes and the multinomial nature of those sizes; the calculation mirrored the development presented in the <em>Methods</em> section. It was assumed that all founder alleles were equally likely to be inherited, that is, 0.25 for the four founders.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464831/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464831/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Specification for the power simulation</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">The simulation is an attempt to examine some simple scenarios to see how successful is the detection of QTL using MVMPWGAIM. The first QTL has the same pattern of sizes across all traits, a pleiotropic QTL. The second QTL has the same vector of sizes for traits 1 and 3 but opposite signs for trait 2. The third QTL is absent for traits 1 and 2. Finally, trait 2 does not express for the fourth QTL.</p><p id="p-44">In the power study, a QTL was declared as detected if the interval selected was within 5 cM either side of the true QTL position. For the scenario presented, 200 simulations were carried out.</p></div></div><div id="sec-16" class="subsection"><h3>Examples</h3><div id="sec-17" class="subsection"><h4>Linkage map:</h4><p id="p-45">A linkage map for a four-way cross in wheat developed in the Commonwealth Scientific and Industrial Research Organisation (CSIRO), Australia, was described in <a id="xref-ref-42-10" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> (2014)</a>. There were 5763 markers on the map, and most were SNPs with 755 DArT markers (Diversity Arrays Technology) and 39 multiallelic microsatellites, grouped into 21 chromosomes of wheat together with three additional linkage groups. The total map length was 5788 cM.</p><p id="p-46">Many of the markers were colocated at the same position on the map and so markers were removed prior to QTL analysis to ensure nonzero recombination fractions between the remaining markers. The final map for QTL analysis consisted of 3230 markers (including 620 DArTs and 37 microsatellites).</p><p id="p-47">The following files are related to the genetic data: The linkage map is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS2.zip">File S2</a>; the pedigree information for the MAGIC RIL lines and the founders is given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS3.zip">File S3</a>, while the correspondence between the pedigree identifiers and the identifiers in the phenotypic data is given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS4.zip">File S4</a>; and the marker data for the founders and RILs are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS5.zip">File S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS6.zip">File S6</a>, respectively.</p></div><div id="sec-18" class="subsection"><h4>Seed size in wheat:</h4><p id="p-48">Two important measures of seed size in wheat are hectoliter weight (HW) and the weight of 1000 kernels (TKW) due to their impact on milling quality and yield. These two variables were measured on the four-way wheat MAGIC population developed by CSIRO in Australia in a field trial at Yanco, New South Wales in 2009. The trial was designed as a partially replicated experiment with 53% of 1063 four-way lines replicated and the rest unreplicated. The layout was 81 rows by 20 columns, in three blocks of 27 rows. The data are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS7.zip">File S7</a>.</p><p id="p-49">Individual trait analyses were conducted for the two traits, using the methods of <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gilmour <em>et al.</em> (1997)</a>. The blocking factor in the design was included in each model. The two traits exhibited some spatial variation, similar for both traits, that was included in the model. The only other extraneous or global effect that was found in the data was a random effect for rows that was required for the 1000-kernel weight analysis. The heritability for hectoliter weight was 77% and for 1000-kernel weight it was 85%.</p><p id="p-50">The bivariate models that are required for the multitrait QTL analysis were of the (symbolic) form<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-70.gif"/></noscript></span></span>where <em>y</em> is the response variable, composed of HW and TKW arranged suitably in a vector, <em>Trait</em> denotes the factor for the two traits that indicates the corresponding trait for each value in <em>y</em>, <em>Genotype</em> is the indicator for the line for each <em>y</em>, <em>Block</em> denotes the blocking factor, and <em>error</em> is the residual error variable. The factor <em>Type</em> separates out the four-way lines of interest from other lines that were planted in the trial; these other lines consisted of the four-way founders and other standard commercial wheat varieties. The replication of these additional lines varied but in general these lines had several replicates. The meaning of “.” depends on what other terms are in the model, but they generally are interactions or simply combinations of variables. In the symbolic model above, the <em>Trait.Genotype</em> term is nested in <em>Trait.Type</em>.</p><p id="p-51">In the model, the term <em>Trait.Type</em> is a fixed-effects term, that is, corresponds to <strong>Xτ</strong> in (1), that allows a different mean for each trait and type combination. The other effects were taken as random. The term <em>Trait.Block</em>, which allows for a different block effect for each trait, and the term <em>at</em>(<em>Trait</em>, <em>2</em>).<em>Row</em>, which allows for a random row effect for the second trait, here 1000-kernel weight, are components of <strong>Z</strong><sub>o</sub><strong>u</strong><sub>o</sub> in (1).</p><p id="p-52"><em>Trait.Genotype</em> allows for variation of genotypes for each trait [<strong>Z</strong><sub>g</sub><strong>u</strong><sub>g</sub> in (1)], and the form of this variation depends on the structure imposed on the <em>Trait</em> factor. For example, effects associated with this factor might be random with zero mean vector (2 × 1) and a diagonal variance matrix with separate variance for each trait. Alternatively, we would in general allow for a correlation between the traits; this is the genetic correlation between traits. These model are also appropriate for <em>Trait.Block</em>.</p><p id="p-53">The <em>error</em> term also has a model, namely a separable structure that corresponds to a three-way combination of factors, namely <em>Trait.Column.Row</em>. The <em>Trait</em> component is treated in the same manner as the terms preceding it, while we impose an autoregressive process of order 1 for both the column and the row components. This generates a three-way separable variance matrix.</p><p id="p-54">The bivariate models required for QTL analysis have first diagonal structures for the <em>Trait</em> component in each term to establish whether a putative QTL should be selected and, for QTL selection, a 2 × 2 variance matrix that allows for different variances for the two traits and a correlation between traits.</p><p id="p-55">The analysis with correlation between the traits resulted in a genetic correlation of 0.28 and genetic variances of 2.6 and 14.1 for HW and TKW, respectively. The genetic correlation is not large but is present so we expect some pleiotropic QTL. The correlation between the traits at the residual level was 0.26 and residual variances were 1.1 and 3.2 for HW and TKW, respectively. The spatial correlation in the column direction was close to zero while it was 0.16 in the row direction.</p></div><div id="sec-19" class="subsection"><h4>Multienvironment trials for flowering time:</h4><p id="p-56">Three trials were conducted using the MAGIC four-way wheat population of CSIRO, Australia. One of the aims was to investigate flowering time, using zadok scores (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zadok <em>et al.</em> 1974</a>). Plant maturity or flowering time is an important adaptation trait (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Worland 1996</a>) that breeders either directly or indirectly select for in breeding. The development from the vegetative phase to the reproductive phase can be divided into three components, vernalization requirement (VR), photoperiod sensitivity (PS), and earliness <em>per se</em> (eps). Each stage is controlled by different genes and influences the overall flowering time.</p><p id="p-57">The trials were conducted in Yanco, in 2009 (the same trial as the seed size experiment above), and at Leeton and Temora in 2010; all sites are in New South Wales. The numbers of four-way lines used in the trials were 1063, 1026, and 1025 for Yanco, Leeton, and Temora, respectively. The trials were all partially replicated designs (53%, 44%, and 56% replication, respectively). The layouts as rows by columns were 81 × 20, 40 × 46, and 90 × 16, respectively. The blocking structure varied across trials: Yanco had three blocks of 27 rows; Leeton had two blocks of 20 rows; and Temora was blocked in two directions, three blocks of 30 rows and two blocks of 8 columns. The data are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/FileS8.zip">File S8</a>.</p><p id="p-58">The analyses for the individual trials resulted in models that included the blocking structures in the experimental design for each trial and a separable autoregressive of order 1 for rows and columns (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gilmour <em>et al.</em> 1997</a>) at the error level, thereby modeling the spatial correlation. No other terms were necessary. The heritabilities for zadok score for the three trials were 82%, 82%, and 92%, respectively, which are very high.</p><p id="p-59">The multienvironment model was of the (symbolic) form<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-71.gif"/></noscript></span></span>The terms are similar to those explained for the seed size analysis. <em>Site</em> is the factor of three levels for the trials, <em>Cblock</em> is the blocking in the column direction for Temora [hence the term <em>at</em>(<em>Site</em>, <em>2</em>)], and <em>Site.Type</em> is a fixed effect to ensure correct mean effects are obtained for four-way and non-four-way lines. The other terms in the model are random effects. The <em>Site</em> variable is again modeled using a diagonal variance matrix or a 3 × 3 variance matrix that includes correlations between the sites. The <em>error</em> represents <em>at</em>(<em>Site</em>)<em>.Column.Row</em>, which allows for separate spatial models for each site, consisting of separable autoregressive processes of order 1 for rows and columns. The two models required for QTL analysis provide for the <em>Site</em> variance matrix to be diagonal or a general 3 × 3 structure for the <em>Site.Genotype</em> term.</p><p id="p-60">The estimated genotype-by-environment (<em>Site.Genotype</em>) covariance/correlation matrix for the data was (correlations above the diagonal, variances down the diagonal, and covariances below the diagonal)<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-72.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>so that the three environments are highly genetically correlated.</p></div></div></div><div class="section results" id="sec-20"><h2 class="">Results</h2><div id="sec-21" class="subsection"><h3>Simulation study</h3><div id="sec-22" class="subsection"><h4>Null simulation:</h4><p id="p-61">For the null QTL simulation, the type I error rate was 0.035 with an average number of false positive QTL per simulation of 0.045. Thus the testing procedure is conservative when compared to the nominal level of 0.05 that was used.</p></div><div id="sec-23" class="subsection"><h4>Power simulation:</h4><p id="p-62">The rate of detection of each QTL (and the overall total across QTL) is presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. On average three of four of the QTL were detected. This is probably due in part to the small size of the simulated MAGIC population, but also because the QTL that express in a subset of the traits (QTL on chromosomes 3 and 4) are more difficult to detect. This is similar to the results found in <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a>. Perhaps more surprising is the lower rate of detection of the QTL on chromosome 1 compared to that on chromosome 2, although the rates for both are quite high.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464884/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464884/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Proportion of correct determination (within 5 cM of the true QTL position) in 200 simulations for each of the four QTL, together with a total out of four</span><div class="sb-div caption-clear"></div></div></div><p id="p-63">The number of false positives was very low and is given in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The proportion per chromosome per simulation is ∼0.02. Overall the proportion per simulation was 0.125.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464915/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464915/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Proportion of false positives over 200 simulations for each QTL chromosome and chromosomes 5–7 that contained no QTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-64">The estimated mean QTL sizes and the standard error of the mean are given in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. In general the sizes tend to be underestimated and it is conjectured that this might in part be due to the small size of the simulated MAGIC population. <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a> show this type of bias reduces for multivariate QTL analysis in biparental populations as the population size increases.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464958/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464958/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Means and standard errors of the mean for each QTL and every founder size for the three traits in the 200 simulations</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-24" class="subsection"><h3>Examples</h3><div id="sec-25" class="subsection"><h4>Seed size:</h4><p id="p-65">The putative QTL found in the bivariate analysis are given in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. There were 29 putative QTL found in the analysis. In total, the QTL explain 48% and 56% of the genetic variance for HW and TKW, respectively. The major contribution to HW comes from a QTL on 7B, while that for TKW comes from 2D.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465000/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465000/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Results for bivariate QTL analysis (using intervals) of hectoliter weight, labeled HW, and 1000-kernel weight, labeled TKW</span><div class="sb-div caption-clear"></div></div></div><p id="p-67">After the QTL analysis, the polygenic variances for HW and TKW were 1.70 and 8.22 with a genetic correlation of 0.25. Thus the QTL have resulted in a reduction in polygenic variance of 42% in both traits. The polygenic correlation remains much the same as before the QTL analysis.</p><p id="p-68">Univariate analyses were also conducted (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/TableS2.pdf">Table S2</a>) to compare results with the bivariate analyses. There were 18–20 QTL identified for TKW and HW (respectively), and of these, 7 and 8 for HW and TKW (respectively) were in common across univariate and bivariate analyses when considering the position as the same as if they were within 5 cM. However, the main difference was that the position identified on each of the chromosomes differed.</p></div><div id="sec-26" class="subsection"><h4>Multienvironment trial for flowering time:</h4><p id="p-69">The multienvironment QTL analysis resulted in 16 QTL being found, with 11 being significant as judged by the probability measure presented in the <em>Methods</em> section. The putative QTL are listed in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. The percentage of genetic variance explained by the QTL overall in each environment was 82.1%, 81.6%, and 82.4% for Leeton, Temora, and Yanco, respectively. Six of the putative QTL expressed in the same way across all environments; that is, common sizes for each QTL were appropriate for all environments and this can be seen in <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>. These putative QTL were on 2D (in the interval adjacent to the PPD-D1 gene also found in the analysis), which contributes ∼7–9% of the genetic variance across the environments, and also on 4B, 6B, 7A, and 7B.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464894/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464894/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Results for multienvironment QTL analysis (using intervals) for Yanco, Temora, and Leeton</span><div class="sb-div caption-clear"></div></div></div><p id="p-71">The estimated polygenic or <em>G</em> × <em>E</em> covariance/correlation matrix after QTL analysis was<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1569/embed/mml-math-73.gif"/></noscript></span></span>and compared to (12) we see that the genetic variances have decreased substantially (44%, 41%, and 46% for Leeton, Temora, and Yanco, respectively) after determination of putative QTL.</p><p id="p-72">As a comparison univariate QTL analyses were carried out using MPWGAIM (<a id="xref-ref-42-11" class="xref-bibr" href="#ref-42">Verbyla <em>et al.</em> 2014</a>); see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/TableS4.pdf">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1/TableS5.pdf">Table S5</a>. There were 11, 9, and 14 QTL detected at Leeton, Temora, and Yanco, respectively. Of the QTL detected, 4 appeared to be common to three sites and 6 were common to two of the three sites. The 4 common QTL were on 2D (2 QTL), 6B, and 7A. The univariate analyses clearly identified PPD-D1 as the major QTL influencing plant maturity across all sites. Note, however, that the size of the estimated effects is larger for the more powerful multienvironment analysis and the percentage of variance explained increases accordingly. The other QTL identified across all sites were on 2D, 6B, and 7A; the latter two turn out to be common effects across all environments in the trivariate analysis. The QTL on 2D, however, was not detected in the trivariate analysis, which is surprising, but we conjecture it is due to the forward selection of putative QTL. Interestingly 3 of the 11 QTL identified in the trivariate analysis were not identified in the univariate analyses, supporting the fact that correlation across sites offers greater power.</p></div></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><div id="sec-28" class="subsection"><h3>Seed size</h3><p id="p-73">Of 29 QTL identified, the largest QTL for TKW was identified on chromosome 2D, consistent with a locus near PPD-D1 as reported by <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Williams and Sorrells (2014)</a>. The largest QTL for HW was located on 7B and appears novel. Of particular note was the number of QTL identified for both traits at the same location. Of the 29 QTL, 21 QTL were identified for both traits. However, of these 21 only 4 had the superior allele donated by the same founder for both traits. In addition, each trait had 4 independent QTL. These results, which may not be that surprising, confirm the challenges in selecting for seed morphology traits and reflect the complex genetic structure that underpins seed size and volume. The locations of the QTL identified are consistent with those in previous studies (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gegas <em>et al.</em> 2010</a>); in particular the QTL on 2B, 2D, 3A, and 5A are well supported in the literature; interestingly, these are the same chromosomes in this study where the founder contributing the favorable allele was the same for both traits. Conversely, the QTL on 7B for hectoliter weight explaining 14.2% of the genetic variation has not been reported previously. In addition, as in previous studies, there are a large number of genomic regions contributing significant effects for these traits, reinforcing the complex genetics underpinning these traits.</p></div><div id="sec-29" class="subsection"><h3>Multienvironment trial for flowering time</h3><p id="p-74">The analysis detected 11 significant QTL, located on nine linkage groups. The largest QTL identified was for the QTL on 2D at the position of the PPD-D1 gene marker. This gene is one of the major genes involved in the photoperiod insensitivity to long days in wheat (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beales 2007</a>). All sites experienced conditions that would satisfy the vernalization requirement. The major genes controlling the vernalization response in wheat are VRN1 on the group 5 chromosomes, VRN2 on 5AL, and VRN3 on 7B (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dubcovsky and Yan 2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Trevaskis <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yan <em>et al.</em> 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Danyluk <em>et al.</em> 2007</a>). The analysis detected QTL near/at these positions on 5B and 5D. As with the seed morphology data the QTL detected are well supported in the literature, in particular those on 2B, 2D, 4B, 5B, 5D, and the group 7 chromosomes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hanocq <em>et al.</em> 2007</a>).</p></div><div id="sec-30" class="subsection"><h3>Multivariate multiparent WGAIM</h3><p id="p-75">The approach presented for multienvironment or multitrait analysis of MAGIC populations (MVMPWGAIM) utilizes probabilities of inheriting founder alleles for a marker or putative QTL in a whole-genome QTL analysis. A forward selection approach is used based on a likelihood-ratio test that determines whether a putative QTL should be selected and an outlier statistic is used to select the location of the putative QTL. This QTL is added to the model as a random effect for multiple traits or two random effects for multiple environments; a main effect is added in the multienvironment situation to allow for a possible common set of QTL sizes. In the latter case it is possible to test for environment-by-QTL interaction and modify the model if the hypothesis of no interaction is retained. The method allows QTL effect sizes to be determined for all founder originating alleles for each QTL and measures of strength to be specified in terms of percentage of genetic variance and a log-probability.</p><p id="p-76">As with all approaches, there are advantages and disadvantages in using MVMPWGAIM. One positive was highlighted in the simulation study. The type I error rate was demonstrated to be controlled and is conservative. The power of the method is also shown to be very good. In the examples many QTL were found so effects of small QTL can be detected. The analyses of the examples took 30 and 65 hr to complete, both running on machines with 16 GB of memory. Of course these are a function of the number of QTL detected, which was large, but the biggest time factor was fitting so-called dense models. The mixed-model framework implemented accommodates trials with complex experimental designs (including spatial variation), includes derived marker/interval variables, and can include both genetic and nongenetic effects. In addition, the models used involved multiple traits or environments, which is generally computationally demanding regardless of the complexity of the model. The computational demands of fitting such a model are an issue with the asreml software used in the MVMPWAIM package or indeed with any mixed-models software that might be used. Other potential methods of analysis (for example using some kind of genome scan) may in some circumstances be less computationally and time demanding; however, they are likely to require the use of a simple linear model. This may be possible through the use of a two-stage analysis. Depending on the experimental design and model this may affect the statistical efficiency and consequently may not be ideal. The other aspect is that multivariate models are inherently difficult to fit and this is particularly true when trying to automate the process. Problems arise in the fitting of these models that sometimes require manual intervention. This means that software can be fragile and in particular very difficult to use by nonexperts.</p><p id="p-77">In the power simulations, most of the QTL were found successfully. The simulation study did highlight that some QTL might be difficult to detect using multivariate methods. QTL that are expressed for a subset of traits or environments might be more difficult to detect; this is similar to the findings of <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Verbyla and Cullis (2012)</a> for biparental multivariate QTL analysis and was confirmed in the power study. The effect sizes for each trait tended to be underestimated in the simulations, probably due to the small population size. Note that the number of possible scenarios for founder effect patterns and effect sizes across traits were too numerous to test. However, the simulations demonstrated that the method was able to identify QTL present only in a subset of traits. In addition, the QTL found in the two examples were large in number. As previously discussed, some of the putative QTL have been previously identified, while some QTL are novel. Overall the method performed very well across all data sets.</p><p id="p-78">Multivariate methods are by their nature complex and difficult. The method presented in this article is a powerful approach for multivariate QTL analysis for MAGIC populations. The available software will allow such analysis in situations for a moderate number of traits or environments. When the number of traits/environments becomes large in number or the models are complex or have issues with convergence, it may be necessary to carry out the analyses manually or on a supercomputer. Research is underway to allow for manual intervention during the analyses, as well as further investigation of the ability of the method to identify linked QTL <em>vs.</em> pleiotropic QTL.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012971/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-79"><em>Communicating editor: S. F. Chenoweth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-02-15"><span class="received-label">Received </span>February 15, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-05-25"><span class="accepted-label">Accepted </span>May 25, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Verbyla <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.1" data-doi="10.1186/1939-8433-6-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muyco</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sevilla</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobina</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multi-parent advanced generation inter-cross (MAGIC) populations in rice: progress and potential for genetics research and breeding.</span> <abbr class="cit-jnl-abbrev">Rice</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%26rft.volume%253D6%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1939-8433-6-11%26rft_id%253Dinfo%253Apmid%252F24280183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1939-8433-6-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24280183&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.2" data-doi="10.1007/s00122-007-0603-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beales</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A pseudo-response regulator is misexpressed in the photoperiod insensitive Ppd-D1a mutant of wheat (<em>Triticum aestivum L</em>.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBeales%26rft.auinit1%253DJ.%26rft.volume%253D115%26rft.issue%253D5%26rft.spage%253D721%26rft.epage%253D733%26rft.atitle%253DA%2Bpseudo-response%2Bregulator%2Bis%2Bmisexpressed%2Bin%2Bthe%2Bphotoperiod%2Binsensitive%2BPpd-D1a%2Bmutant%2Bof%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0603-4%26rft_id%253Dinfo%253Apmid%252F17634915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0603-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17634915&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249019700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.3" data-doi="10.1534/genetics.107.071068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071068%26rft_id%253Dinfo%253Apmid%252F17947443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE4MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-4.9.1569.4"><div class="cit-metadata unstructured">Broman, K. W., 2006 Use of Hidden Markov Models for QTL mapping. Technical report, working paper 125. Department of Biostatistics, John Hopkins University, Baltimore.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1569.5"><div class="cit-metadata"><cite>Butler, D. G., B. R. Cullis, A. R. Gilmour, and B. J. Gogel, 2011 Mixed models for S language environments: ASReml-R reference manual. Technical report. Queensland Department of Primary Industries. <span class="cit-publ-loc">Toowoomba, Queensland, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.6" data-doi="10.1016/j.pbi.2008.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DFrom%2Bmutations%2Bto%2BMAGIC%253A%2Bresources%2Bfor%2Bgene%2Bdiscovery%252C%2Bvalidation%2Band%2Bdelivery%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.01.002%26rft_id%253Dinfo%253Apmid%252F18295532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18295532&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danyluk</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, N. A. Kane, G. Breton, A. E. Limin, D. B. Fowler <em>et al</em>., <span class="cit-pub-date">2007</span> <span class="cit-article-title">TaVRT-1, a putative transcription factor associated with vegetative to reproductive transition in cereals.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1849</span>–<span class="cit-lpage">1860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft.volume%253D132%26rft.spage%253D1849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.9.1569.8"><div class="cit-metadata"><cite>Dubcovsky, J., and L. Yan, 2003 Allelic variation in the promoter of Ap1, the candidate gene for Vrn-1, pp. 243–246 in <em>Proceedings of the 10th International Wheat Genetics Symposium</em>, Vol. 1, edited by N. E. Pogna, and R. A. McIntosh. Istituto Sperimentale per la Cerealicoltura, <span class="cit-publ-loc">Paestum, Italy</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.9" data-doi="10.1105/tpc.110.074153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gegas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nazari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fish</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite> <em>et al</em>., <span class="cit-pub-date">2010</span> <span class="cit-article-title">A genetic framework for grain size and shape variation in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1046</span>–<span class="cit-lpage">1056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.074153%26rft_id%253Dinfo%253Apmid%252F20363770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzQvMTA0NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.10" data-doi="10.1051/gse:2003009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Roy</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparison of three multitrait methods for QTL detection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGilbert%26rft.auinit1%253DH.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D281%26rft.epage%253D304%26rft.atitle%253DComparison%2Bof%2Bthree%2Bmultitrait%2Bmethods%2Bfor%2BQTL%2Bdetection.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2003009%26rft_id%253Dinfo%253Apmid%252F12729550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2003009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12729550&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182656900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.11" data-doi="10.2307/1400446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Accounting for natural and extraneous variation in the analysis of field experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D2%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1400446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1400446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.12" data-doi="10.1017/S0016672300004869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multi-trait QTL mapping in barley using multivariate regression.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D95%26rft.epage%253D106%26rft.atitle%253DMulti-trait%2BQTL%2Bmapping%2Bin%2Bbarley%2Busing%2Bmultivariate%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004869%26rft_id%253Dinfo%253Apmid%252F11279835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279835&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168340200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.13" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.9.1569.14"><div class="cit-metadata unstructured">Hanocq, E., O. Laperche, A. Jaminon, A. L. Lainé, and J. Le Gouis, 2007 Most significant genome regions involved in the control of earliness traits in bread wheat, as revealed by QTL meta-analysis. Theor. Appl. Genet. 114: 569–584.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.15" data-doi="10.1093/bioinformatics/btq719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">R/mpmap: a computational platform for the genetic analysis of multi-parent recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq719%26rft_id%253Dinfo%253Apmid%252F21217121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy81LzcyNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.16" data-doi="10.1111/j.1467-7652.2012.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22594629&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307051700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Multiple trait analysis of genetic mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJiang%26rft.auinit1%253DC.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D1111%26rft.epage%253D1127%26rft.atitle%253DMultiple%2Btrait%2Banalysis%2Bof%2Bgenetic%2Bmapping%2Bfor%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F7672582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC8zLzExMTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.18" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic dissection of a model complex trait using the drosophila synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Multitrait least squares for quantitative trait loci detection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">899</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BA.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D899%26rft.epage%253D911%26rft.atitle%253DMultitrait%2BLeast%2BSquares%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%26rft_id%253Dinfo%253Apmid%252F11014835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzIvODk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE1NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirzhner</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Interval mapping of quantitative trait loci employing correlated trait complexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1137</span>–<span class="cit-lpage">1147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKorol%26rft.auinit1%253DA.%2BB.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D1137%26rft.epage%253D1147%26rft.atitle%253DInterval%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bemploying%2Bcorrelated%2Btrait%2Bcomplexes.%26rft_id%253Dinfo%253Apmid%252F7672584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC8zLzExMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.21" data-doi="10.1038/sj.hdy.6882530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Multi-interval mapping of correlated trait complexes.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D80%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6882530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6882530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000072783500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.22" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multitrait fine mapping of quantitative trait loci using combined linkage disequilibria and linkage analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLund%26rft.auinit1%253DM.%2BS.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D405%26rft.epage%253D410%26rft.atitle%253DMultitrait%2BFine%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BUsing%2BCombined%2BLinkage%2BDisequilibria%2Band%2BLinkage%2BAnalysis%26rft_id%253Dinfo%253Apmid%252F12586725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvNDA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE1NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.24" data-doi="10.1007/s10681-007-9594-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multi-trait multi-environment QTL mixed model with an application to drought and nitrogen stress trials in maize (<em>Zea mays</em> l.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9594-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9594-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.25" data-doi="10.1007/s00122-011-1558-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eewijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, A<span class="cit-auth"><span class="cit-name-surname">Casas</span>  <span class="cit-name-given-names">M.</span></span>, M. Elia <em>et al</em>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gene and QTL detection in a three-way barley cross under selection by a mixed model with kinship information using SNPs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">1605</span>–<span class="cit-lpage">1616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMalosetti%26rft.auinit1%253DM.%26rft.volume%253D122%26rft.issue%253D8%26rft.spage%253D1605%26rft.epage%253D1616%26rft.atitle%253DGene%2Band%2BQTL%2Bdetection%2Bin%2Ba%2Bthree-way%2Bbarley%2Bcross%2Bunder%2Bselection%2Bby%2Ba%2Bmixed%2Bmodel%2Bwith%2Bkinship%2Binformation%2Busing%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1558-z%26rft_id%253Dinfo%253Apmid%252F21373796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1558-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21373796&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.26" data-doi="10.1073/pnas.230304397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turri</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A method for fine mapping quantitative trait loci in outbred animal stocks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12649</span>–<span class="cit-lpage">12654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.230304397%26rft_id%253Dinfo%253Apmid%252F11050180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjMvMTI2NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A mixed-model approach to mapping quantitative trait loci in barley on the basis of multiple environment data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">2043</span>–<span class="cit-lpage">2050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPiepho%26rft.auinit1%253DH.-P.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D2043%26rft.epage%253D2050%26rft.atitle%253DA%2BMixed-Model%2BApproach%2Bto%2BMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2BBarley%2Bon%2Bthe%2BBasis%2Bof%2BMultiple%2BEnvironment%2BData%26rft_id%253Dinfo%253Apmid%252F11102394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzIwNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1569.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.29" data-doi="10.1017/S0021859606006319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The design and analysis of multi-phase quality trait experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D144%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859606006319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859606006319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.30" data-doi="10.2307/2533455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stram</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Variance components testing in the longitudinal mixed effects model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DStram%26rft.auinit1%253DD.%2BO.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1171%26rft.epage%253D1177%26rft.atitle%253DVariance%2Bcomponents%2Btesting%2Bin%2Bthe%2Blongitudinal%2Bmixed%2Beffects%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533455%26rft_id%253Dinfo%253Apmid%252F7786999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7786999&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QP05000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.31" data-doi="10.18637/jss.v040.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">R package wgaim: QTL analysis using complex linear mixed models.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DJara%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1%26rft.epage%253D32%26rft.atitle%253DUsing%2Bthe%2BR%2BPackage%2Bcrlmm%2Bfor%2BGenotyping%2Band%2BCopy%2BNumber%2BEstimation.%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v040.i01%26rft_id%253Dinfo%253Apmid%252F22523482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v040.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523482&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-4.9.1569.32"><div class="cit-metadata unstructured">Tinker, N. A., and D. E. Mather, 1995 Methods for QTL analysis with progeny replicated in multiple environments. J. Quant. Trait Loci 1. Available at: <a href="http://probe.nalusda.gov:8000/otherdocs/jqtl/index.html">http://probe.nalusda.gov:8000/otherdocs/jqtl/index.html</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-4.9.1569.33"><div class="cit-metadata unstructured">Trebbi, D., M. Maccaferri, S. Giuliani, A. Sorensen, M. C. Sanquineti <em>et al.</em>, 2008 Development of a multi-parental (four-way cross) mapping population for multi-allelic QTL analysis in durum wheat, pp. 231–232. in <em>The 11th International Wheat Genetics Symposium Proceedings</em>, edited by R. Appels, R. Eastwood, E. Lagudah, P. Langridge, M. Mackay <em>et al.</em> University of Sydney, Sydney, NSW, Australia.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.34" data-doi="10.1073/pnas.1635053100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trevaskis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bagnall</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peacock</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">MADS box genes control vernalization-induced flowering in cereals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">13099</span>–<span class="cit-lpage">13104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1635053100%26rft_id%253Dinfo%253Apmid%252F14557548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIyLzEzMDk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE1NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.35" data-doi="10.1016/j.pbi.2010.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection and use of QTL for complex traits in multiple environments.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253Dvan%2BEeuwijk%26rft.auinit1%253DF.%2BA.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D193%26rft.epage%253D205%26rft.atitle%253DDetection%2Band%2Buse%2Bof%2BQTL%2Bfor%2Bcomplex%2Btraits%2Bin%2Bmultiple%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2010.01.001%26rft_id%253Dinfo%253Apmid%252F20137999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2010.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20137999&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.36" data-doi="10.1007/s00122-005-0204-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping QTLs and QTL × environment interaction for CYMMYT maize drought stress program using factorial regression and partial least squares methods.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVargas%26rft.auinit1%253DM.%26rft.volume%253D112%26rft.issue%253D6%26rft.spage%253D1009%26rft.epage%253D1023%26rft.atitle%253DMapping%2BQTLs%2Band%2BQTL%2Bx%2Benvironment%2Binteraction%2Bfor%2BCIMMYT%2Bmaize%2Bdrought%2Bstress%2Bprogram%2Busing%2Bfactorial%2Bregression%2Band%2Bpartial%2Bleast%2Bsquares%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0204-z%26rft_id%253Dinfo%253Apmid%252F16538513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0204-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16538513&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236486600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.37" data-doi="10.1111/j.1467-842X.1990.tb01015.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">A conditional derivation of residual maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Aust. J. Stat.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BStat.%26rft.volume%253D32%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-842X.1990.tb01015.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-842X.1990.tb01015.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.38" data-doi="10.1007/s00122-012-1884-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multivariate whole genome average interval mapping: QTL analysis for multiple traits and/or environments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVerbyla%26rft.auinit1%253DA.%2BP.%26rft.volume%253D125%26rft.issue%253D5%26rft.spage%253D933%26rft.epage%253D953%26rft.atitle%253DMultivariate%2Bwhole%2Bgenome%2Baverage%2Binterval%2Bmapping%253A%2BQTL%2Banalysis%2Bfor%2Bmultiple%2Btraits%2Band%252For%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1884-9%26rft_id%253Dinfo%253Apmid%252F22692445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1884-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22692445&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.39" data-doi="10.1071/AR02239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckermann</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The analysis of quantitative trait loci in multi-environment trials using a multiplicative mixed model.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D54%26rft.spage%253D1395%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR02239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR02239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.40" data-doi="10.1007/s00122-007-0650-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The analysis of QTL by simultaneous use of the full linkage map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVerbyla%26rft.auinit1%253DA.%2BP.%26rft.volume%253D116%26rft.issue%253D1%26rft.spage%253D95%26rft.epage%253D111%26rft.atitle%253DThe%2Banalysis%2Bof%2BQTL%2Bby%2Bsimultaneous%2Buse%2Bof%2Bthe%2Bfull%2Blinkage%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0650-x%26rft_id%253Dinfo%253Apmid%252F17952402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0650-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17952402&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.41" data-doi="10.1017/S0016672312000493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RWGAIM: an efficient high dimensional random whole genome average (QTL) interval mapping approach.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D94%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672312000493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672312000493&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.42" data-doi="10.1007/s00122-014-2337-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome QTL analysis for MAGIC.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1753%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2337-4%26rft_id%253Dinfo%253Apmid%252F24927820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2337-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24927820&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.43" data-doi="10.1007/s001220051331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Mapping QTLs with epistatic effects and QTL x environment interactions by mixed linear model approaches.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D99%26rft.spage%253D1255%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084014900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.44" data-doi="10.2135/cropsci2012.10.0609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Three-dimensional seed size and shape QTL in hexaploid wheat (<em>triticum aestivum l</em>.) populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.10.0609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.10.0609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.45" data-doi="10.1007/BF00015718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Worland</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The influence of flowering time genes on environmental adaptability in European wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D89%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00015718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00015718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UK82100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.46" data-doi="10.1017/S0016672300034066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mapping quantitative trait loci using four-way crosses.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D68%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300034066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300034066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VU51800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.47" data-doi="10.1073/pnas.0937399100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, A. Loukoianov, G. Tranquilli, M. Helguera, T. Fahima <em>et al</em>., <span class="cit-pub-date">2003</span> <span class="cit-article-title">Positional cloning of wheat vernalization gene VRN1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">6263</span>–<span class="cit-lpage">6268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0937399100%26rft_id%253Dinfo%253Apmid%252F12730378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEwLzYyNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.48" data-doi="10.1111/j.1365-3180.1974.tb01084.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zadok</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konzak</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A decimal code for the growth stages of cereals.</span> <abbr class="cit-jnl-abbrev">Weed Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWeed%2BRes.%26rft.volume%253D14%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3180.1974.tb01084.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3180.1974.tb01084.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974V546200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.49" data-doi="10.1017/S0016672399004255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Estimating the genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D279%26rft.epage%253D289%26rft.atitle%253DEstimating%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004255%26rft_id%253Dinfo%253Apmid%252F10689805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10689805&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085417000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1569.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic distribution of quantitative trait loci for yield and yield-related traits in common wheat.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D62%26rft.spage%253D996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1569.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1567" title="GENETICS and G3: Community-Driven Science, Community-Driven Journals" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1585" title="Adult Plant Development in Triticale (× Triticosecale Wittmack) Is Controlled by Dynamic Genetic Patterns of Regulation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1569.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FP1QaAd8uA_3x-4zoc8x_zlIEYYEh9QPpZt2L9OFmvg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1569" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1569" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1569" class="active" data-icon-position="" data-hide-link-title="0">Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459663" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zkGtWe-QmewfolKC5cn894QQ3XXY96c6AbxzmlbGxiE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1569.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1569" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Bd-b689kDxPck624diU8oCmfZII5c-v2e4AvGWeiGOU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1569" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1569.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459663" id="node459663--2" data-pisa="ggg;4/9/1569" data-pisa-master="ggg;g3.114.012971" data-apath="/ggg/4/9/1569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Arūnas P. Verbyla</span>, <span class="highwire-citation-author" data-delta="1">Colin R. Cavanagh</span> and <span class="highwire-citation-author" data-delta="2">Klara L. Verbyla</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1569-1584;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012971 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459663--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arūnas P. Verbyla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Informatics and Food Futures National Research Flagship, CSIRO, Atherton, QLD 4883, Australia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Agriculture, Food and Wine, The University of Adelaide, Adelaide, SA 5005, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ar%C5%ABnas%2BP.%2BVerbyla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verbyla%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAr%25C5%25ABnas%2BP.%2BVerbyla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Ari.Verbyla@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Ari.Verbyla@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Colin R. Cavanagh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Industry and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Colin%2BR.%2BCavanagh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavanagh%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AColin%2BR.%2BCavanagh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Klara L. Verbyla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Computational Informatics and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klara%2BL.%2BVerbyla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verbyla%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKlara%2BL.%2BVerbyla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1569" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1569&amp;title=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1569&amp;title=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1569&amp;title=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1569&amp;text=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1569&amp;title=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1569&amp;t=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1569&amp;title=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1569&amp;title=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459663/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459663" id="node459663--3" data-pisa="ggg;4/9/1569" data-pisa-master="ggg;g3.114.012971" data-apath="/ggg/4/9/1569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Arūnas P. Verbyla</span>, <span class="highwire-citation-author" data-delta="1">Colin R. Cavanagh</span> and <span class="highwire-citation-author" data-delta="2">Klara L. Verbyla</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1569-1584;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012971 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459663--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arūnas P. Verbyla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Informatics and Food Futures National Research Flagship, CSIRO, Atherton, QLD 4883, Australia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Agriculture, Food and Wine, The University of Adelaide, Adelaide, SA 5005, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ar%C5%ABnas%2BP.%2BVerbyla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verbyla%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAr%25C5%25ABnas%2BP.%2BVerbyla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Ari.Verbyla@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Ari.Verbyla@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Colin R. Cavanagh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Industry and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Colin%2BR.%2BCavanagh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavanagh%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AColin%2BR.%2BCavanagh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Klara L. Verbyla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Computational Informatics and Food Futures National Research Flagship, CSIRO, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Klara%2BL.%2BVerbyla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verbyla%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKlara%2BL.%2BVerbyla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1569&amp;rft.epage=1584&amp;rft.atitle=Whole-Genome%20Analysis%20of%20Multienvironment%20or%20Multitrait%20QTL%20in%20MAGIC&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Verbyla%2C+Ar%C5%ABnas+P.&amp;rft.au=Cavanagh%2C+Colin+R.&amp;rft.au=Verbyla%2C+Klara+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459663/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459663/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459663/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459663/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459663/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459663/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459663/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459663/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459663/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459663/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459663/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459663/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496049" id="node496049" data-pisa="ggg;g3.119.400658v1" data-pisa-master="ggg;g3.119.400658" data-apath="/ggg/early/2020/03/26/g3.119.400658.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.119.400658" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Dissecting Adaptive Traits with Nested Association Mapping: Genetic Architecture of Inflorescence Morphology in Sorghum</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1569" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1569.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1569.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1569.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1569.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
