<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1623" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1623.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1623.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1623.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/9/1623/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1623.atom" />
<meta name="HW.pisa" content="ggg;4/9/1623" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice" />
<meta name="DC.Identifier" content="10.1534/g3.114.013748" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2014 Gatti et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic mapping studies in the mouse and other model organisms are used to search for genes underlying complex phenotypes. Traditional genetic mapping studies that employ single-generation crosses have poor mapping resolution and limit discovery to loci that are polymorphic between the two parental strains. Multiparent outbreeding populations address these shortcomings by increasing the density of recombination events and introducing allelic variants from multiple founder strains. However, multiparent crosses present new analytical challenges and require specialized software to take full advantage of these benefits. Each animal in an outbreeding population is genetically unique and must be genotyped using a high-density marker set; regression models for mapping must accommodate multiple founder alleles, and complex breeding designs give rise to polygenic covariance among related animals that must be accounted for in mapping analysis. The Diversity Outbred (DO) mice combine the genetic diversity of eight founder strains in a multigenerational breeding design that has been maintained for &gt;16 generations. The large population size and randomized mating ensure the long-term genetic stability of this population. We present a complete analytical pipeline for genetic mapping in DO mice, including algorithms for probabilistic reconstruction of founder haplotypes from genotyping array intensity data, and mapping methods that accommodate multiple founder haplotypes and account for relatedness among animals. Power analysis suggests that studies with as few as 200 DO mice can detect loci with large effects, but loci that account for &lt;5% of trait variance may require a sample size of up to 1000 animals. The methods described here are implemented in the freely available R package DOQTL." />
<meta name="DC.Contributor" content="Daniel M. Gatti" />
<meta name="DC.Contributor" content="Karen L. Svenson" />
<meta name="DC.Contributor" content="Andrey Shabalin" />
<meta name="DC.Contributor" content="Long-Yang Wu" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="DC.Contributor" content="Petr Simecek" />
<meta name="DC.Contributor" content="Neal Goodwin" />
<meta name="DC.Contributor" content="Riyan Cheng" />
<meta name="DC.Contributor" content="Daniel Pomp" />
<meta name="DC.Contributor" content="Abraham Palmer" />
<meta name="DC.Contributor" content="Elissa J. Chesler" />
<meta name="DC.Contributor" content="Karl W. Broman" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic mapping studies in the mouse and other model organisms are used to search for genes underlying complex phenotypes. Traditional genetic mapping studies that employ single-generation crosses have poor mapping resolution and limit discovery to loci that are polymorphic between the two parental strains. Multiparent outbreeding populations address these shortcomings by increasing the density of recombination events and introducing allelic variants from multiple founder strains. However, multiparent crosses present new analytical challenges and require specialized software to take full advantage of these benefits. Each animal in an outbreeding population is genetically unique and must be genotyped using a high-density marker set; regression models for mapping must accommodate multiple founder alleles, and complex breeding designs give rise to polygenic covariance among related animals that must be accounted for in mapping analysis. The Diversity Outbred (DO) mice combine the genetic diversity of eight founder strains in a multigenerational breeding design that has been maintained for &amp;gt;16 generations. The large population size and randomized mating ensure the long-term genetic stability of this population. We present a complete analytical pipeline for genetic mapping in DO mice, including algorithms for probabilistic reconstruction of founder haplotypes from genotyping array intensity data, and mapping methods that accommodate multiple founder haplotypes and account for relatedness among animals. Power analysis suggests that studies with as few as 200 DO mice can detect loci with large effects, but loci that account for &amp;lt;5% of trait variance may require a sample size of up to 1000 animals. The methods described here are implemented in the freely available R package DOQTL.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1623" />
<meta name="citation_id" content="4/9/1623" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1623" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1623.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1623.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1623.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.013748" />
<meta name="citation_pmid" content="25237114" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1623" />
<meta name="citation_lastpage" content="1633" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel M. Gatti" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Karen L. Svenson" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Andrey Shabalin" />
<meta name="citation_author_institution" content="Medical College of Virginia of Virginia Commonwealth University, Richmond, Virginia 23298" />
<meta name="citation_author" content="Long-Yang Wu" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Petr Simecek" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Neal Goodwin" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Riyan Cheng" />
<meta name="citation_author_institution" content="Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, Australian Capital Territory 0200, Australia" />
<meta name="citation_author" content="Daniel Pomp" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Abraham Palmer" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, Chicago, Illinois 60637" />
<meta name="citation_author" content="Elissa J. Chesler" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Karl W. Broman" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_email" content="gary.churchill@jax.org" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Baud;citation_author=R. Hermsen;citation_author=V. Guryev;citation_author=P. Stridh;citation_author=D. Graham;citation_title=Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.;citation_pages=767-775;citation_volume=45;citation_year=2013;citation_pmid=23708188;citation_doi=10.1038/ng.2644" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=L. E. Baum;citation_title=A maximization technique occuring in the statistical analysis of probabilistic functions of Markov chains.;citation_pages=164-171;citation_volume=41;citation_year=1970;citation_doi=10.1214/aoms/1177697196" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=T. Billings;citation_author=E. E. Sargent;citation_author=J. P. Szatkiewicz;citation_author=N. Leahy;citation_author=I. Y. Kwak;citation_title=Patterns of recombination activity on mouse chromosome 11 revealed by high resolution mapping.;citation_pages=e15340-e15340;citation_volume=5;citation_year=2010;citation_issue=12;citation_pmid=21170346;citation_doi=10.1371/journal.pone.0015340" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_author=S. Sen;citation_author=S. E. Owens;citation_author=A. Manichaikul;citation_author=E. M. Southard-Smith;citation_title=The X chromosome in quantitative trait locus mapping.;citation_volume=174;citation_year=2006;citation_pmid=17028340;citation_doi=10.1534/genetics.106.061176" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=W. S. Bush;citation_author=J. H. Moore;citation_title=Chapter 11: genome-wide association studies.;citation_pages=e1002822;citation_volume=8;citation_year=2012;citation_pmid=23300413;citation_doi=10.1371/journal.pcbi.1002822" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=A. A. Palmer;citation_title=A simulation study of permutation, bootstrap, and gene dropping for assessing statistical significance in the case of unequal relatedness.;citation_volume=193;citation_year=2013;citation_pmid=23267053;citation_doi=10.1534/genetics.112.146332" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=J. E. Lim;citation_author=K. E. Samocha;citation_author=G. Sokoloff;citation_author=M. Abney;citation_title=Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.;citation_volume=185;citation_year=2010;citation_pmid=20439773;citation_doi=10.1534/genetics.110.116863" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Cheng;citation_author=M. Abney;citation_author=A. A. Palmer;citation_author=A. D. Skol;citation_title=QTLRel: an R package for genome-wide association studies in which relatedness is a concern.;citation_pages=66-66;citation_volume=12;citation_year=2011;citation_pmid=21794153;citation_doi=10.1186/1471-2156-12-66" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=R. Cheng;citation_author=C. C. Parker;citation_author=M. Abney;citation_author=A. A. Palmer;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bulletin of mathematical biology;citation_journal_abbrev=Bull Math Biol;citation_author=G. A. Churchill;citation_title=Stochastic models for heterogeneous DNA sequences.;citation_pages=79-94;citation_volume=51;citation_year=1989;citation_issue=1;citation_pmid=2706403;citation_doi=10.1007/BF02458837" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Naive Application of Permutation Testing Leads to Inflated Type I Error Rates;citation_pages=609-610;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202402;citation_doi=10.1534/genetics.107.074609" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_title=R News;citation_title=The regress function;citation_author=D. Clifford;citation_author=P. McCullagh;citation_pages=6;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Cox;citation_author=C. L. Ackert-Bicknell;citation_author=B. L. Dumont;citation_author=Y. Ding;citation_author=J. T. Bell;citation_title=A new standard genetic map for the laboratory mouse.;citation_volume=182;citation_year=2009;citation_pmid=19535546;citation_doi=10.1534/genetics.109.105486" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Didion;citation_author=H. Yang;citation_author=K. Sheppard;citation_author=C. P. Fu;citation_author=L. McMillan;citation_title=Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.;citation_pages=34-34;citation_volume=13;citation_year=2012;citation_pmid=22260749;citation_doi=10.1186/1471-2164-13-34" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=K. J. Eash;citation_author=J. M. Means;citation_author=D. W. White;citation_author=D. C. Link;citation_title=CXCR4 is a key regulator of neutrophil release from the bone marrow under basal and stress granulopoiesis conditions.;citation_volume=113;citation_year=2009;citation_pmid=19264920;citation_doi=10.1182/blood-2008-09-177287" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=W. Valdar;citation_author=S. Shifman;citation_author=R. Mott;citation_title=Strategies for mapping and cloning quantitative trait genes in rodents.;citation_pages=271-286;citation_volume=6;citation_year=2005;citation_issue=4;citation_pmid=15803197;citation_doi=10.1038/nrg1576" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. C. Gentleman;citation_author=V. J. Carey;citation_author=D. M. Bates;citation_author=B. Bolstad;citation_author=M. Dettling;citation_title=Bioconductor: open software development for computational biology and bioinformatics.;citation_pages=R80-R80;citation_volume=5;citation_year=2004;citation_issue=10;citation_pmid=15461798;citation_doi=10.1186/gb-2004-5-10-r80" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Journal of Pharmacology and Experimental Therapeutics;citation_journal_abbrev=J. Pharmacol. Exp. Ther.;citation_author=B. Hitzemann;citation_author=K. Dains;citation_author=S. Kanes;citation_author=R. Hitzemann;citation_title=Further studies on the relationship between dopamine cell density and haloperidol-induced catalepsy;citation_pages=969-976;citation_volume=271;citation_year=1994;citation_issue=2;citation_pmid=7965819" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=O. D. Iancu;citation_author=P. Darakjian;citation_author=N. A. Walter;citation_author=B. Malmanger;citation_author=D. Oberbeck;citation_title=Genetic diversity and striatal gene networks: focus on the heterogeneous stock-collaborative cross (HS-CC) mouse.;citation_pages=585-585;citation_volume=11;citation_year=2010;citation_pmid=20959017;citation_doi=10.1186/1471-2164-11-585" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila synthetic population resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Alcoholism, clinical and experimental research;citation_journal_abbrev=Alcohol Clin Exp Res;citation_author=T. K. Li;citation_author=L. Lumeng;citation_title=Alcohol preference and voluntary alcohol intakes of inbred rat strains and the National Institutes of Health heterogeneous stock of rats.;citation_pages=485-486;citation_volume=8;citation_year=1984;citation_issue=5;citation_pmid=6391261;citation_doi=10.1111/j.1530-0277.1984.tb05708.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Y. Liu;citation_author=A. P. Morgan;citation_author=E. J. Chesler;citation_author=W. Wang;citation_author=G. A. Churchill;citation_title=High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.;citation_volume=197;citation_year=2014;citation_pmid=24578350;citation_doi=10.1534/genetics.114.161653" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mott;citation_author=C. J. Talbot;citation_author=M. G. Turri;citation_author=A. C. Collins;citation_author=J. Flint;citation_title=A method for fine mapping quantitative trait loci in outbred animal stocks.;citation_volume=97;citation_year=2000;citation_pmid=11050180;citation_doi=10.1073/pnas.230304397" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. D. Parvanov;citation_author=P. M. Petkov;citation_author=K. Paigen;citation_title=Prdm9 controls activation of mammalian recombination hotspots.;citation_volume=327;citation_year=2010;citation_pmid=20044538;citation_doi=10.1126/science.1181495" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=R. Pyke;citation_title=Spacings.;citation_pages=395-449;citation_volume=27;citation_year=1965" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=L. R. Rabiner;citation_title=A tutorial on hidden Markov models and selected applications in speech recognition.;citation_pages=257-286;citation_volume=77;citation_year=1989;citation_doi=10.1109/5.18626" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=A Statistical Framework for Quantitative Trait Mapping;citation_pages=371-387;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560912" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. A. Shabalin;citation_title=Matrix eQTL: ultra fast eQTL analysis via large matrix operations.;citation_volume=28;citation_year=2012;citation_pmid=22492648;citation_doi=10.1093/bioinformatics/bts163" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-Resolution Genetic Mapping Using the Mouse Diversity Outbred Population;citation_pages=437-447;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=A. M. Van der Bliek;citation_author=F. Baas;citation_author=T. Van der Velde-Koerts;citation_author=J. L. Biedler;citation_author=M. B. Meyers;citation_title=Genes Amplified and Overexpressed in Human Multidrug-resistant Cell Lines;citation_pages=5927-5932;citation_volume=48;citation_year=1988;citation_issue=21;citation_pmid=2901906" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Hill;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Inf. Theory;citation_author=A. J. Viterbi;citation_title=Error bounds for convolutional codes and an asymptotically optimum decoding algorithm.;citation_pages=260-269;citation_volume=13;citation_year=1967;citation_doi=10.1109/TIT.1967.1054010" />
<meta name="citation_reference" content="citation_journal_title=Mammalian Genome;citation_author=C. E. Welsh;citation_author=D. R. Miller;citation_author=K. F. Manly;citation_author=J. Wang;citation_author=L. McMillan;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=N. A. Zaitlen;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=A. L. Price;citation_title=Advantages and pitfalls in the application of mixed-model association methods.;citation_pages=100-106;citation_volume=46;citation_year=2014;citation_pmid=24473328;citation_doi=10.1038/ng.2876" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=Bayesian modeling of haplotype effects in multiparent populations.;citation_author=Z. Zhaojun;citation_author=W. Wang;citation_author=W. Valdar;citation_pages=139-156" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="Genetic mapping studies in the mouse and other model organisms are used to search for genes underlying complex phenotypes. Traditional genetic mapping studies that employ single-generation crosses have poor mapping resolution and limit discovery to loci that are polymorphic between the two parental strains. Multiparent outbreeding populations address these shortcomings by increasing the density of recombination events and introducing allelic variants from multiple founder strains. However, multiparent crosses present new analytical challenges and require specialized software to take full advantage of these benefits. Each animal in an outbreeding population is genetically unique and must be genotyped using a high-density marker set; regression models for mapping must accommodate multiple founder alleles, and complex breeding designs give rise to polygenic covariance among related animals that must be accounted for in mapping analysis. The Diversity Outbred (DO) mice combine the genetic diversity of eight founder strains in a multigenerational breeding design that has been maintained for &gt;16 generations. The large population size and randomized mating ensure the long-term genetic stability of this population. We present a complete analytical pipeline for genetic mapping in DO mice, including algorithms for probabilistic reconstruction of founder haplotypes from genotyping array intensity data, and mapping methods that accommodate multiple founder haplotypes and account for relatedness among animals. Power analysis suggests that studies with as few as 200 DO mice can detect loci with large effects, but loci that account for &lt;5% of trait variance may require a sample size of up to 1000 animals. The methods described here are implemented in the freely available R package DOQTL." />
<meta name="og-title" property="og:title" content="Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1623" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic mapping studies in the mouse and other model organisms are used to search for genes underlying complex phenotypes. Traditional genetic mapping studies that employ single-generation crosses have poor mapping resolution and limit discovery to loci that are polymorphic between the two parental strains. Multiparent outbreeding populations address these shortcomings by increasing the density of recombination events and introducing allelic variants from multiple founder strains. However, multiparent crosses present new analytical challenges and require specialized software to take full advantage of these benefits. Each animal in an outbreeding population is genetically unique and must be genotyped using a high-density marker set; regression models for mapping must accommodate multiple founder alleles, and complex breeding designs give rise to polygenic covariance among related animals that must be accounted for in mapping analysis. The Diversity Outbred (DO) mice combine the genetic diversity of eight founder strains in a multigenerational breeding design that has been maintained for &gt;16 generations. The large population size and randomized mating ensure the long-term genetic stability of this population. We present a complete analytical pipeline for genetic mapping in DO mice, including algorithms for probabilistic reconstruction of founder haplotypes from genotyping array intensity data, and mapping methods that accommodate multiple founder haplotypes and account for relatedness among animals. Power analysis suggests that studies with as few as 200 DO mice can detect loci with large effects, but loci that account for &lt;5% of trait variance may require a sample size of up to 1000 animals. The methods described here are implemented in the freely available R package DOQTL." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"IMwfGL1Y-VAV_Pd6uAMHmemrgNzwPQj2a3X6vzIpiak","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459655","apath":"\/ggg\/4\/9\/1623.atom","pisa":"ggg;4\/9\/1623","ac":{"\/ggg\/4\/9\/1623.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1623.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1623"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1623"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1623":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Nr3+Q4gbf\\\\\\\/raf5ylgqVp9bd6aE4lo2qQxk3PWpd07lM6ya+HOCYPWuw9JbvFSFW8htc+7ldLe\\\\\\\/9CT1LUfvFFGqhZhrmCsecC+RWXoloxrwWpsHw6KWvfFuYot+Fs8MvsesGiieDqjRjlF4iqF2vr9AIsh2iVjHNt7\\\\\\\/yPrjBvx4wJWtlup76xpfRgl6GDuVJ6Oc3mu\\\\\\\/JVuu7kE7f1H2HOThiSJvR6pttWs1WCU\\\\\\\/4S+GlDdfUPpZmr2ZhY4axaODe2zBJYPkRIByZVoaWPbOCF68TCKimPykRoPFYJ1uykX4JzF68TBsfTMdK3PWSuM4HDCtp7iMloX2abdAe6UlPv5fWqW65QcL8KyzJnMQyOolg\\\\\\\/Ej9kAju1PHNp23fH1WZrQz8mC\\\\\\\/QM92xHS8pP3VMzmpF8aEzAnzIQ1eBr8odYc7ywaV8anjtB3jzYxekbteaHHHCxOG\\\\\\\/uBLUUVbJ+WogTVM6Ag87RlflJzQHYMlDatVO5RiP0KXq5A9CJ8Qq6U3a25FdV3SNadOHP0i+rKuBfFxjtAo0BxNB6Tnbu77bhMyvVUqEj3pZP9dZAH9AgWh9d7t4dHM\\\\\\\/O35irln1+XtBaHUepy1wZWcwOZsAOMZuECdc5KWlfVPt2c28bXJdWTnQS1aLEXyorns5DwlFTWruy+ySMPBnOma5PfCrfRjWfpWl27auaV5i4Ght\\\\\\\/NPUOTBuBJG+aoTXFXAkzOXroTemhmXmxVUFjR\\\\\\\/wE9tSBnOpQrpbtaQXdwbX23tCaT3UNuRSU+VkJaXom36WOTHHEJVPPiodqypJwwSWmtZce9THlxPb0w2annJZL6K\\\\\\\/Fg19FhBDJHahSwRGsM+TAPrwd1m\\\\\\\/LJHM6IV8muMPR9RHqjUjYlui7WbGq48OrZT7W4GBcntTFIeob3FFbREsHgWDXh9B5lGqZo9Ew5b9RGW3H3hzZIEl13f22wWiKsmlulrHzUX1rj3A8wF9aRmB4Wf5YUFDAR1Zt0exoysIKpG829SyiSzR84ezYwh5YnCx7Vet3Z3jI070\\\\\\\/jYD\\\\\\\/NFO243RmUUhSon3QGjhieoRIVlhEF+MH+eBXZAR3LHKGgwxP+OOOs3ognuEykW7BAyvV9AdyZzNeo1RAeeBWW2PegOqACLt\\\\\\\/\\\\\\\/YZxz1AXSN8RTBrDwxguPRrUVST1ouYAysG0H1T5nbel6zsATm1AcYsOCJCfdX6AD6OkymR5XlUUhRkGsl0HtLOZyXF9Qdu\\\\\\\/rrESiPGFTobEYEEMSwNiIn90ljVAYxRjBuRLusySqZgONOB0WCgug\\\\\\\/i7\\\u0022,\\\u0022iv\\\u0022:\\\u0022TZfI17ih0dM7ix4qKBv4JQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ed03b724255493b428e954a563ca58ae850881efa6cb6704c89aef0658d7075e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GmCWJfxz5hd+ZUI0Ba8AM3Q5wPvda1gJ5UQfz1+nGDIatpvD5iy8gQmhenREKkbRPkKUejLinqytUGqfl8beUCQU3JioSkmwTQ52kkncobGZQrz1yrKGunPi+UJ0DUimmVrXyEzlPxwIE6oytQFsGllfx4OwsHpQoVjsPdrtv+dOrCGL0kk7dy4dyIIH\\\\\\\/+BLUvJZFTdKCLjWrULee258jjG7g92xuKCvtZb3SBi312Z95TOFJMHuKkZW9g\\\\\\\/P517KG6vpfT2mjzfr2hMkffhioRpmo2PeSfEG\\\\\\\/gpIcH\\\\\\\/HZqxysQr3mzb7qabAUxMLTs8AndYX4a0Eb2OfeFjCHXEvjL8QK3dk76XEAlc0wPgPu3mT2+lrMOjcw8GiPIhE3Dw73uAbOjNAQ\\\\\\\/RJUFvlgIkXAvX03Uu61PVYz2DvMuJ+WS+VvQkj+s4gN6SH6iDPMWrfcK0PKfjKH370cbh+MnWzcXarQ0YDLan8m2jMiH2JNwmTD8G70L5eLL5t\\\\\\\/ZIjHUwLI6JOnC5wWLRelMRROsb9xGA8dfUuRjOkqGsqlC7O\\\\\\\/fYFBHnp4jkCX1IbQuk4DljuFfMKgNKP+YwVGt1azslR2cRJX4gasjQMIpGwvOjbNLpUjg8qysH2CsWQtft208NNfRiojqReyhPnv45MJpqQYYJGESOHr+RTnqYUonU3BSCDBDyLgD6dCiFlxCR3uotQjDzRwOrifw9ePm32pZFw8vvdn+8TsxuEHhIYyZXRZVln8VYOVJJbw6iXV1JJ9\\\\\\\/7M8Voz37V7cQcByOQGRJ7HiXFWfYcOzHTnNfmRZ6\\\\\\\/nFWSxFwNTVipA5bgOEWxIVJ7zJDlb5NnqPA1MArj2746RKGDa1gBTAWQ6LLy8u4P3jdU7XoboqDhnYazK2YkIzdHM+\\\\\\\/CHyVeUV5AOx04hyzXZFVmftVn99zcIuIzcb6Ko+otiOynHY6BCy2tMxOp1tvE5u3+u7A36M55inzI6rBKiiBn1MtotFpZqhMvrk\\\\\\\/fktC+6F4PMSozLaTynquzc7oiAm2PylcedIIBvBExLnussI1XwwyScgiBKK0V67h1wiaZFmsdP0I0N7eWKCn1mWCT\\\\\\\/z1neVnAYtXpO68rW3tW\\\\\\\/QC5R5Z0BaswW3OBfdveOwLIdI\\\\\\\/aPDUuZt3Q7zb7vhdQ8+BxBuSPkuh3WzSj4eh3SXgwAkuVZg5i\\\\\\\/pPh7vJE\\\\\\\/PnCsYSc9Ev93Rxldnt0vCLESMh\\\\\\\/eVYhJTxu3v76F\\\\\\\/8opkqGhGF4QoT8KxxGDz4LaQ3GOSMswSbWtyhd6r2xw9qFPzUPolaVQo8viGd8xfOomLRGxQgZBpH90FykW3J9Z2MlHTqlttU3FWEJv4ScG4oz2GmnUP4xEVGzui8bm3Aky81PQp8XoyrloUZACK7Rw5G7hVygNTyYLeru0O1zEquZLEI4glrhOKnZbmwdrqKP0vq4wpEJmC4vpmLeU+aCdbEwBmQg8iTMptVLH1NW7tI\\\\\\\/jAa8OYmKGai6iXlNgu26eTDayZQw6VsV+ww3RzznfVTDi+b5yoD7R++INeac85U1K+FjF6RaBS4Y9lEBtuYgMzchLasocV3NtoWxAu6IljN0Z13xzeQaOtNtPgTzRSzQn0q0akDduMNTOdL5grZkpgsD\\\\\\\/CNsjVkbc1u09tKVZSAOnDHav\\\\\\\/jqbO3ym36qafwrAyxBpw4vblGxFI5yo9z24TUxC4h7RspuQg9Hc1Gjomp8qJrQJYHOi3rFeg20ZydLTv6H6aGsHpmPsaegqHwhwkd4vz3KmBEkK7baTOIXgco15ZMRQXHHJJpBu5E\\\\\\\/UFk+BE\\\\\\\/c66kvY7RlBDQomXKDxHeUc+Fp8Xvmud4eoX7xWacLqp8njtkK0ZkBFXNNpnvw7NSS8kTQpY4Q6rkhBd1\\\\\\\/A7q+zfURlHc3gGSOjc54CqaRE+YSi1VE7CvhMJieuK0hH2KQY4qaj5K8thWp2fAqu0\\\\\\\/Qj3fugU74E4mZnCHTW0bcEUpUr7rQMQH\\\\\\\/NPwMQ6y41LQFe9NpXTB0WYqrv9GqLrcmA3nJy1\\\\\\\/D9tWud8AXoTHKd90+aSWeS5OF5k2DVLBuH6CJ1u2k397lSAZSl+bV475vAW62lUNRmklsYcsW11boaq1AJ4cE9+VDrsxUXb6a1FvGof4XFDU7eEUMETAm6E1fzSsfKeikVk1zGNE7vYxKidLRCgzYO6pHuyH0fiuRHSDFuIolm6TDIZiv5BvQPQvolYcniilwq0Mklv2Ws6KyEWFCXmcUL5HtKdms37wn6N5Ps+Q0afZ0O6wAWsOE7lxJACS9KlS75xSl661ALDpJKduyvai49LFP+zOF6gPaVdV30jCRWt8xSVef2P20ET1\\\\\\\/met10QBE3VK1XeD0PDFANNynhAZ432tzu+rhHMHsbBXrA0x+nJCWq651YseV0mPd\\\\\\\/UN9Fz\\\\\\\/9neM2L5bAOXOz6SSreF+6fN7n2d2pDKKy0xEQKD87JXY4YPeiWyd0XkyPiZ3dsDsRP+3AHlRMi9JRFCfaMnx5P0GfZYRwXF+FUgVsG9NEv856p0v1iQXHNiWmkTMkbB66T7a7W9QrBydQkD6eCZBXRehARK3HgJpMicaTCHlO4iIDFqJnY7RXQHu\\\\\\\/Q6EQhIN\\\\\\\/ISy46PxfMdH19Je6o5Rzv9v0Afhk8OPZMguY21\\\\\\\/eO\\\\\\\/0axfLirZcT\\\\\\\/B+QSnecDS\\\\\\\/LHpBo8okN0VmbvUZ7Btnnk3VjGzAhZJ5HIPUPB44FMKWFV9RekKMIju7ztX\\\\\\\/nM0jHD4H0GYh622N90rnLUA7jrbLaST4NiyzoUOc0rkZ7JWs2uGAHDSK1Ynyp5c+lQu6qYqO188rzctCakUDS0VpcvEfemC7ZCFC1tR8HXRverD0SKp5Tg8da8mw3RqKeuu7YuXa0bdHpDTxhyrziGQ0Z6gd+EbaURIuYGu\\\\\\\/wS05TRIdAFyPnhtKTTxPEsfjCKz5IKceU5sPfrbnmbKyrHhzPy8yozyooDyqJs3ml98O1W4jUXBTmzXj2iI2VvmKQnIrG3PDT4K7GEbbuxZhcjXxgc+RWsef36zMwrRt464s16+IpuF5qQeYrwBe8AacSjtz5Nq1lBQpkEyMDZf1WA\\\\\\\/rrknUkra1HGN4Dpiic9cxBfVWa7eNn7wCzmsJQU2B69lQ0WU\\\\\\\/RwFRwvASwM2j+luLqr284XOCT33l+rutgmUt7JMgl8AP\\\\\\\/O1EB1gRFUWVoCzDpXSzEWB0MKej1837r5eHmG0G2OiwMr1EF\\\\\\\/U\\\\\\\/PMfmB\\\\\\\/qIJ1jNHNWM\\\\\\\/6Lk8uOXkSGyDhURiZJD\\\\\\\/9ruByhFhB3oABV\\\\\\\/wASDPsYhOT2lIONCXpC4K+NyUuf+G9aFBskerfS+eWgzCL95i1YuY66+LLTrXGRUGva2Io7omRN9o\\\\\\\/j1g9ykAY5VQEJpGSQ09A5gJl4clFgZ0NThuDz8NQp0FpbyQDN+f1Av69iisvYVOsEjh3XHvMKF7gDN7aVxio9+il6f8IWXXGcbtVQXPY5wVXhSf9xrc+aePX76\\\\\\\/HAGz92jBvWsDVUSxury2WUiGPi0D45FAr8Ap6oKGT4lBrfq0PINkjjJ+k4IEp5NDQ9jsUmCvzCaM3fVMNz2UYnBGQzZV6QMbfMsp6yc0ze6RjdkCBJZXqqtbHhHBvhH9DnL0XMLZDp1158A9Vumf1D3LrRgSlXyncKP8Wf9mxEfwJ4b5IkyP1XYOJvoZB8LVdIJWh4\\\\\\\/VLh3IUTAz763xBF\\\\\\\/mgchg9ltwM1j7RHk1F7vozg0pmItiliRH199Q2uo6ClR6ONvFAQJaWeqn78jDMFUMVzohy8dqgDLsWsAHS066babM3+jpSxbFO2JEZHargfn+s5JeH3Vl+OAoqIOMtRgyUuisK64zM+oD1a+B73tW8L0ebLmQIcmL1gLjhYUfWlHfGGKw7OfKoqejcBVFIaXkrS3OzWWCNbd2FPwK7g14B5G37beYieFW1Mc5V16kBSx\\\\\\\/8Xl0gulnhVYAyw8ktQHrjvy3KPJZmY74kvoxuhKIcyuPOz3hAydGcHU\\\\\\\/IEB6ZsITKns5vF3pPFJAw62Scqg5EPEJxYFDdaR7VLBs02vvYccQUDz+vdzSRzKKU5BXj37sIlHdNjbxrc\\\\\\\/MdQgrtfku\\\\\\\/LIcl0ldpmFHFWmc5KFVflj57YECfT0r07YisL+Dz5aSsGcFQTReZBINvkoxmhL3gAcq86+chucGWomySTlzQVYrMK\\\\\\\/UkoI5cj+k+GhAo8OGeBwrg6+sMzGcSx0Ug8sSS4JgvUMTGMUT\\\\\\\/FQ5yP7BT8whtJALvFsyaT1xYOr\\\\\\\/mYnQFgbPM0wigyxNVHTf3rsXNzUp0hk0HRKKI+dsncu5IcNYHrtOEZOwBF+bUARvIMyyfAXxLYo+16ZOvOdh4i0hGlLNlkJiHUsEmLuVdozhMPxm1Pu6\\\\\\\/q0J4Gj\\\\\\\/yR1PD73L4sasFG09H5wBkRW4vgJ5C81+VMyP58Lo+Gu99M4PSxfKx84wO8pscSUMz6DJFM1yYsbv5Hx76c0DGCteB1zHFUH0dsBJXkfZTusUUsfzCKx7Qxy0g1QSIrXMp0FUd6fmFdhsy7wGnnzCceMmIVqkCfZaGGAs0W+mCsc5xGR9uShCfXMxi6jUG0upX7f6P9DSYhiZTXtOU2qkJtzIo8oLT20XGUIwzaoJVz7YjbK+VNwG0vC6K6X7NsCKwffupVGVtVc4MjC2XKQVyz32EZWzigh8vIjs\\\\\\\/upuLdF7wFQBJ4YPDkT08oflW0dm+ei88Som3J9KiUg9ZueAjzOO1QSL8vFq1WqUDsHCPF1qPksMAGQ16H5z0VtXvvaBc21XhwZB4Q6ajgU14DJJrSCy\\\\\\\/QILeX5hPQGjVo0jhLFjFqjtEwOqYZR8rp+k5wS3nM1x1qvsOiqjaJVwl9Xj+CyiR8pBQC5qi6+a\\\\\\\/+Xzx4o0qoP1vMVNBXaaWLNi8kmGVD3QwsAG5ct5PS4pR94zHogg4j+l48D1YK1jc0oTfwsqzWmJsNuz0FfI488Ka3i9TAaBw+J8BfGKGc2n\\\\\\\/6fiIDyll7TFoirvQqi5sGkg+VU1PJ\\\\\\\/qQsguFPYO+goA\\\\\\\/NoJwgXkq4ky6AMaJ4Zea4+7JslmkTf+QcWhIJwwBNHA1pyhD5QZMBXrYYAbTyBvbNSNkhEuSGW\\\\\\\/obRkuarMih+pZwU3PCIphxaWsQye+OBmSD0W5ZT3T5\\\\\\\/XSo9tm2tSH6JtLFmOFVlnf+2DMOngLvBVJcMf6E701I7VLluD6DNMTrpvOw4P2t5kNhbmLZN8IcD9k\\\\\\\/vSCpXdJXwffJxApW8CrACR0g0BFGbNAeeWWgld0T\\\\\\\/9R1ZItZqIfXxVKbZFPaGT7khXTysmzs+AFXgSu5mJrMspjR77lgz3XEV\\\\\\\/MuHQnkSqYd8RhVKmo+rlaBYy41UumRgeZEqNyt+Ly2b9rdWhCLaflNYd2VTSeNfA\\\\\\\/wQMdNRUA6ho0o8W4i9\\\\\\\/kb0E+vDVIXOu8YohBANBZj62rUGTJgyMpeyzojbLR0zm6lXl+feUnCkT2q8jDvduWvHF0lRW2jfRG7T8c0OC1jwkqrbttCXrwQeqqHKMMYficMtAyFa\\\\\\\/7wrSXOcslv7aQV8uD2XQnrGXmPHNRo3NSpWT0NkUkDPWJG1MT4pC\\\\\\\/rst+gggXieTM2WoKbT\\\\\\\/eyOxU458VWAVL8qdpR3JCwhHYOXPDUh+ymmkHioRl\\\\\\\/XUK4GQ7rTXZiJhWm0nP0Ko7O1P4L6vZ20I4SX0MZKdP5030mJALNqrIFMfQuIJWrTv6n6\\\\\\\/fEYREXUVqqxozDf7yZTt4gsEbnnxac0gh34J70pk3qlmUgIXcq6\\\\\\\/5y9NmBMukolSWp6hJpo0tPCYULtTFbisRfvVxONBOW9L9baDhIZ3+uO+m5g2RsM7i8tHzUhPi6kdY4EvHAw34qdbUUUUbY6SO\\\\\\\/EKe63l4ZbuY6Nl0PZekpj4ifqCJcW+GZmfLiWEdGF\\\\\\\/F88pGo7RRWO\\\\\\\/Wm+KoZ6kJkh8a8T1uH5UgRSau5P8MRJXph5\\\\\\\/cPQZB4mPxd0jJ0uw4cSz\\\\\\\/1GP8a3abQ8w4nb0lgg71YyREgGPwZVzJ6NTLjk8Y7KIyifG11YALggfCt2PqhUWEzeZV+2gyGxXlU365QdxTWbSIrewsErQyio05yA\\\\\\\/lRJd8Gsn0b9TUIQaOEWB8388xppCnCXzDJbZ7x1m7xQmVyVlP9Vn3KtEnkO38b2PUfoVSrWEn1m0ynQ5vQYkgi1rxFXJeMaay4oglgq+ktNtG\\\\\\\/FigrjaZxDdcWJT1FeP\\\\\\\/OdqQOrVqAEtx6BcsYgSLk28+nU\\\\\\\/Gif7cbUz+vfWnoCGVMghro6ZY48lrFbK9RfAkARnDzvnrn6CVEZeRhYnnQV5NZf67+Af0TSRxMczvXnu9w3RQh+4K0YjMECMIwij\\\\\\\/VA27Y5hNu7Xrcg2rKCsvYQ8R3HS9G83hkLxpAPWgR0rOlq93mveHi2RsvlMrur2qGvPfrpff5u+JIuBKxwvbKZAJ5\\\\\\\/irMHoAmUoIU8YP9uH8\\\\\\\/hYKYBoSeFcEjKTsldnTyuZ2qkU\\\\\\\/rGfhcdzcIhA9iyXuCUw3AgSnzTqfR1dxvget0yNS+8ESowPIV0sRxNo5c7SQdj3OfHd8KIjGbempD2W16kVP2Gp0cNBei+IT9ByDLbzAhrkhzHQDzi1NZMA78cFBiksQiK6mh+jmvgTZ3Nv09y988+RjM62GERwNDkKgIONlsyGNTN5ql2X8xkLUbrHF8cWX+Or6LlpOXvls6Xc\\\\\\\/m+rhHlbD4baRA9eLi8pJolVp9uWbXxAX1CzgonLzI8oHIvNEjVGKSv\\\\\\\/90kG6VJqFxpf7Rn4fsYVn9KXguuMJXeTyU5gymNDTQuh\\\\\\\/i\\\\\\\/0UYqA\\\\\\\/UpvoxhagTcqIPcx8z4ZVJx8usM+x+zF4ZxB6P6F7loBcAylPXBGS1IirmEt4Rg3CnfOmfj1mOS4pJc0vvbBRopGOWjv0mL7KH+7FtbZEa7tk3IS52hxjy4Pz1Qhoe6mxQFLIrknSPuAzWX\\\\\\\/mqxBeFwoQ3JJroZSWDoy9Q9z5ruqC0ijo5fRDGRUlJL6c783lbcEG7nAiexkt4JPaz4kL4fGOFqa0p6X43oFJZ5\\\\\\\/SrMFYIrUms3rKWXacvc9TX3PEzbevbE2hfjvdLRkg38QYMZMXgMDl9Ic\\\\\\\/IydPBEZjoIMrMteEx3vmTiO6WzjoAohoupmOMnphTzcBJKikZW84Ag6FD7W4cjDxAw4P91qgsBBa79yx5Cq8PwWAPAmNakWCpL1QR3C+AaLgG8Z07QPqqU+puphLrMWOhSzfrKbB\\\\\\\/UCUv\\\\\\\/pgfGHglYla6moa7+cQ3D7wZEBhG2qQ88rucJF1WUA\\\\\\\/Lf4gOEAWdhty91c1BRp5ueU\\\\\\\/EPAE7ZYiUHHLjqST8Ng4v8pOMdrSivKMq2S\\\\\\\/4\\\\\\\/mJk\\\\\\\/Lp4+StP7R40TOpTz+Sgr\\\\\\\/AwsetJ5LVx9mUfJ5hkoHhjpXCJB\\\\\\\/XNMRSX\\\\\\\/11bFe5MZ4yJXaLKqmGV+ianj87w3cYUaJdY0h9jCXmoGO\\\\\\\/uiiUEE\\\\\\\/DqNWGKoGX234tIMcSN5Zb7JSVSoufCveFbUZsGWOUnmyIeO1wDo3+VCr6MfTQ+64vs19s8m12LCbrp5sGjihZAgfxN\\\\\\\/t+z+Pj5otMwMNzXNL\\\\\\\/XH1u15naPEXeH1qSkBzKdHt4vZzrvXe\\\\\\\/6uY57uW1jVmke83\\\\\\\/HLy8OqJ2h5FYZPjxi4VvQOHzqog8AhnRTg4IZuLYPgo\\\\\\\/IJA9kx9bcPYZj9UzILt35FLaBgyB49wFpChtnNlR1AEToDQBiaah8W9HPXrk8MIZttVjETQ3Yr\\\\\\\/jUh3RXl3Nex7UWhDsf581ZhQ+YYj85UWgPJsl+tSQ3Q5+AfawHu4jfawh12\\\\\\\/LLuLzbE3\\\\\\\/RQEnYIYhfivryw42By+7XdgpuJ3dYvj5HS5PU3N3Xm3+uhNEca2NjxCoLfHv8CUNntLpY\\\\\\\/GDVS6Y6BfPwNPtqdfPexSrGAotpAfAGmpXvKSJemid2\\\\\\\/Ey7K9My8MShNomUgklSin6hya7LwbByt3IVS7hIQQLhSma1\\\\\\\/HenrSbTNTEvu20+48E2tOEskMSaFbC06OeOWZ0Yif6Lkmti476mVodyjQJ2TmVLvgYCUAEbHVEd6inagxXBQVYUSQbPuS92VgBQcTfkg70CTm3BjybRp5sREwxh9QA9zKw5Ao0ekvRqss5MfQEiO9OYxJTfjy6+9br5oIh4FmlGVil1DaW4FB4cSGTnq33E7XK6Ow2m\\\\\\\/GbU0P2rDHeJZ9TCevCFFbdQ17ZBqc5hCQSs+nKsUjGbpH+zhHp3XKZj2975hww8A8KtyLM1xxgU3yYu9cnv1bcp05D6hb0aSvI0ojMPFcZOQxovOlUCUSUBESohSOSTwaU1L0\\\\\\\/da1T3t8OOpwCyOt1P8EUsgiAhxsRH4REtwezKtVN+ojxQmEGyhDZbpqCa9KRi2ZzWWpyND+HjHyDp\\\\\\\/bO8HymMcYVbBmlbEJW1KRMzXb2QMQDmCkU4Ka5xoYnxMatfUGzpE06YfC5Q2r\\\\\\\/mnVf\\\\\\\/YdhIiJQXeQAX6onUrV+u\\\\\\\/tlzs4zLg0aok\\\\\\\/kvdI0+tGd6ufytyvNMg2CeEe84p2\\\\\\\/edEAFdt2t0FAbftJNYNyq+IYVnSaYBdfQ1sUFhRHfjHyNlHxzPriXEndwu4SQVRc3v6tGXfzQ\\\\\\\/wikaYEf\\\\\\\/wFTD8N9\\\\\\\/yWEbqkCwLVdykNuzlSYE61Mt\\\\\\\/HahgrVf7CTiEWBmQQG2gA6IVjfVKU3s51E9mt4TOs8nWOQzsYa6\\\\\\\/yEbq4ZmRjgWjtgiZAPKTzroJIZppdUUZD2JU34CqKv67hAlBeMLHDdOAH29ZaONBvjPPhcAYKQWMUhVsSbtnva+fH4YDI1WxNHqwWdLkdUdketkKl\\\\\\\/MN\\\\\\\/yRtvJP5S7KvMpQM9Y5OLuMT4CKrxpEInXpESW91+Yz3WLTUOnVBMslx4xSQCKYI0OemkGeHDb+HznHFmovJmKf72\\\\\\\/u+mFrhXjkpIa694i6LNEmhn2xT0rPgjiBCW5G2xTRUcov0qqaPpN4ubSDnyVO3LLHbDrnzfUj9JGj5r0ki3mXSfUw1\\\\\\\/x5Il3Ab8LyxyJMnruUZmG2i79JlgyAL70cxSQJQrXXFxtteSU3utRw4js+PnHzmt9z3hiMB6J09Gbe9\\\\\\\/dDC7kjmCxoI4nelk3ITpBcBpitf6mzb\\\\\\\/HfEbMxM4+x3P77JbEBo0KwtoBOOpug2U\\\\\\\/DnuLDo9r7tTmECbWSzTw0S5FKGLYfesmVcMGNCrQnwR614AoYvyxMRBv5OVLhLVZcS6nDhqTu+gTEhK\\\\\\\/Ts42YUifu15cL7lYQDbe32saZAnqofnpbtdqbCTtm7xfEqwyOe6PJ6jaVFdeSXkH2DCp3pP+Js3td5bMdy3hDOj8oNE1wlZ+zC8BRgtPWE1uxvpmx71nFm9aouq2+iowFMHJYJx32de4\\\\\\\/AFc6abbIp42sGFArGRpenm0smAuWl+C6GTQK+gIFb2LHfOUOlloqSx5vKYk0GQHqXw\\\\\\\/oLvxcTZ8N2A9JwXzjBHyGtHb4Rwn1BjSSAED4ZTwJD8Iik2cLRAIOaPc9Y3LNdAeAwY8metUSeeYWfo+8UZg6uARWrR\\\\\\\/AxsFnbH1eEpwt+cZqSApVegtKi7vjEH5zxzEHBsphHwqYRID5BoD+DMcQM40Ugx22KhSG6P\\\\\\\/bMMw02aDD90cwYMzvbhqAb+GggFyYhwaj4I6zE+2F1DlIG5b6yUZsMpNxwCDpJMfEzy7Bi7t3NSV2M+5tmsAskX0R8c8u4RAWQtVmoGLKzkqUxXBixGQEGxWZMivTozcpt\\\\\\\/sbXhF7Jcf1CZkPJzpN7OmNAeiyvj+nNfRJEJoQ0yavBCMZK2e3lHDIiXG6YnmvyA\\\\\\\/TGGXZwQBhBi2FTY0nvdZklz+2PacyJDfbug\\\\\\\/kbgIIr7FyrxvxKwa6fwkzk2JU9aKG\\\\\\\/+bM3DQTzf6S6oVNZG7jIiBhyZ2pC0nNSt8Yp9MChgzRKHJmAL2cv\\\\\\\/Xmixt8yceDjj6NVKnB\\\\\\\/\\\\\\\/nRucAkgRVVWFZy0nYto+flRGPLLnjXYx0eS0Wnonl74zlpDLTkVJ4BqxE91BAUpzre36IQSVTbzCaZGly8hBql4ipfXy3dfwok2mAe0\\\\\\\/5CdAkFpEPtKYaZ5H9srpqIxovLbB11GSOdM6MINxlkJizQCvuPeG9dBoSkshEtSal3pF5NwG\\\\\\\/j5cgWGBbcSKI02OD3efOkfYeHos3nyTsHgYi7wi9hrrFxnyHASJLGQ+GdXjF77dZAasRB\\\\\\\/kLoSnMZbKS4dwky4JTZIMQcTeU0SssvyiDL22UB5N+\\\\\\\/XW5ZJOFDWNtvGQ4namLrnE2sggk5I8ZnuHa8VI9C5TUvpF+T8jvnzzN\\\\\\\/3s4hBTl3HVaFHC4y+TQX6lh3532yex3QRR+2gA+tPGsNztyyWmPBegdeeZM+4TEzEcEyOdzCcVqB1sGDyI2klczvRdA8p9n9bYVU6arswgg3xku9oEIvW+BTqZh1FiYkweVlWAdZM8H8Zz1iDkmD6KY0MK2JJHsQemCcF0Vr2LTGFgpJac7D31BP7W6iF9Wpv8ZMPWoCnfa\\\\\\\/A4Q+3S9XFehbd2x2C2sPNs3RTXubosjqHgmMItuLbDuiRDMjyJkedabuH8\\\\\\\/2sLKOqBXjVDeEry7CpDuwV2kowCtYyfjwr5sJ3WPsOTS7s5pNwyBdDypyOIiyLDaFj\\\\\\\/De\\\\\\\/uJBxuHwa+VF1a\\\\\\\/O70hQm\\\\\\\/q8UP8QsXAD34uRY7u95BURbAxE0qfcohfpDy6u0FiVqtgWTOLzpkG2dEldmdQVBlp2xGk1pfLwGVtxFS30C6ApGkf+7irli1baygWcPx0o1f3OHHSE2HDEAudAeeUCBuiRw2vE2gMhQB2KNWJLUj2D02eX4wkphzGEz0vnr7aF6wrlJGdmGhRHY4ZI4tyuGuCzruYOkWNfuhyahdV+kUbLA76xdYoxo5Wy7pkTv7vrTE1sraxCrw+ZvMGRe55UbEi7TLnT8XNiR51niREmqjV1xhWkMjIl97mDR\\\\\\\/h0NGjI7FlS8\\\\\\\/cT+PuaN7Ob8wdjrp83JYohU27iJE+SoL6hYGtw5QC1xP7\\\\\\\/jxt0Ex1B0BLnbkzfjnOjme8fuMMd3pIjiJGN68gOZSNumYE33\\\\\\\/PLjP\\\\\\\/a5W3lkk0JdSNrbQMnNEa+sz4C5MlPpjoch1\\\\\\\/6qkKsyhJX\\\\\\\/9YnbbU24SaQfJQNFDPYFqUAbwVJNHzSDIfRfFZWXwDsr0YRejxxbfRLNgjtndSTsDvcDEDKOgMNPa+Y\\\\\\\/\\\\\\\/LPspxDkw\\\\\\\/WsZWzx9twgWKyalJEyNf\\\\\\\/5X47BaTIF98JTiRA1pUzbH2xyq9L83OqW5Sj+HQ6MmkMdZVSyC7MvHawEHah30nL0M9ePBtLqRL+Hr6kzfC3kLhCQuCrITp420gu1tPl8ZFElCk8RpZfqiEJjgvlU\\\\\\\/TrYJaM\\\\\\\/CH+FKC3IIK576xO26cJNfYMWxaP21lMDCpceLy+v9yXvlSaM6v0NnIn52tNT0ksi80WveS\\\\\\\/qC6jUKMGJmnQfvhbKnSfZlAq7rwhJ0KMTSb7TGN1Cbhmo\\\\\\\/5w69EFEBPgMYRVQgf5wsMBumgmCcvILf7FWlWVjDxigBfiVG0PjkKDk2UJb3b6Sw9hbICkIhIv6XuafSJihmakqICON6fSC3ENwN2+2hh+mzocRB+oUK2u3E9\\\\\\\/dvADvJeS2QVOPxJ9xFHP\\\\\\\/qZJbEMZupgBXYjYh13e2f9Y4mJI+IJXfnhgOIVrUBrCk0r6rOCnN7fPEwvn6oZY2Y1\\\\\\\/gRe3z2FRCf30Y8lCagrReuwSwgrH1iayj7yVYSWIHr3rwToaB5ZFHedNY2r9HMEsWDejBAp43Znkq5LrZVWbdbksbvRMkWxCyNXefJIk5KlBBAYGDKAihgNkpjKyYAjq34Bf9PNXMJWIWyVFnVHMjd25LuM55Y+pZkbctsvjDlN8mXC27kzYUEaKJlzgvtIKqnJXYs4SBiMDeNOfpS+T3Lt2CC2Mg9SkDmykvP9YuwfKdUQRf2N7mvMnWwjD1oUvZPy46y\\\\\\\/ilr\\\\\\\/vcFg7b\\\\\\\/RiuSFVPiIQiKp+xSId5O37l1oG2BIuaopCJTXI+as8NkovjIb6LgYdGd6lfTZTS5z4ZG\\\\\\\/QB6aQw7GOCz6DC+cCvmiEls\\\\\\\/QIUMpiHyQLzj0fXBrE6jKgBmJjqwH1lgRs7t5ON6qZT9flMIJmJtDHhbK057U3fugXX0vCbD9Ehq91Qc0ugrEVYq\\\\\\\/q7qv7qQhMZvfujaF+ZH1rFZKmWY6LxxMbu2IYxEWiSqVVhbKZpg5lzRDxEw7P22xbRCsWurFd3V4Xu183npRoNxl5uotv+B4tmqtZw9TFt800ItCaaKgKcd72ZojCx9CX6FXHK3\\\\\\\/z8be8IyTYQe91s01acGSBHBRlc\\\\\\\/dlxJZxTrP\\\\\\\/WY5W3fRdSxHAwLVdGIgJuP7Qs6I2D06jGWZtT38cuNI6C13DFaWTGZ5ONegyNmL7I1A7LvlT4Q+XFVq+aKDts0h0YDcYO9M8QnEf6tkSnIR\\\\\\\/M\\\\\\\/KyCplayIvlQDwAzNNlmiKAcXsIToljjiyYhdfIfDy8luUIU\\\\\\\/as+3c1S6y10vmncp1kuI8qKw\\\\\\\/Z5jp93pA+WTvTlYXRsfNuI8F8nSMT0DJ8w7q58smXxznMBL37SGdsl7uqjwKiuCAkg2YvoDQVQHQ3jrk8Fj910nTMpSfeK9v8WE1vBng8sA7xgb9a+2YyqzZsxQ0YNjK83wzlESJNiRDjtugUCCDD8\\\\\\\/pJfdRuw1F3dV\\\\\\\/jJahONYNCg3wda8sM5R5f2fR7foJOcASQmeSWgomG\\\\\\\/Ec8znWs+v\\\\\\\/6lumhZ3eSgQIAAFSdfk0WcXwxm9X0ajPtgV+OCJqrVjgqaQpbEjAN+qJdbZzlwi63x\\\\\\\/heD\\\\\\\/3fWWMNd0P\\\\\\\/GqharVGPNO2nTuvrmUz6ZVNA8WNXzHOVORndWnzJuTz8WovnLO648QT36CLeteZjK04kAQcDdQTZ8DD+6baTkvuZIEXLVvnFKOPYjUSZz\\\\\\\/PEKswFVg2s+scfQdUf8YLN1ERxTWSbJdIY5X6LnP7Tx7iiuyI0kvMzrbVCV7ncYpwN6P3PPoIkXjUEiDY\\\\\\\/sCGIaGcB4973NVXlapcuEWD7k8qML5rmSC28DQKwyNUiMnF0V9be8CcivUmBmZKAHzZ7SB7DJMIJvCxDwqs8T6IAr9QECTgB\\\\\\\/zwTcdhJfPI8EHsQ2vihtXFEQrvx\\\\\\\/sZI3aFptVUG0qvwVyJDz+U7eALzx9sdn1d8D5oKgFl+JxmxhV+XJqigWgkTdDPOaRyUAL6EvtViKSCx+3BVDff\\\\\\\/XsW2V6mNJSHoVqgMtV3v1xBELfvmqrCyX23mvDuJSpkuKwItK0K5x8dogktWCFZKuhlhygJydyqj9gMNI0aWrtGSqB8OEr8kHwlErw\\\\\\\/feMm2f\\\\\\\/1lWBjThlb0yBmdcSqm3q0MVUT2BKhZvtR9SQwH4cggzoJ2AR2Gj+wOFHS8a+uas007KM4OIIihPu3GnPk4ZTaAJs4MsPBwruNEEWS+TKwGdyNQdCdL0gKAG3cT7PPo\\\\\\\/cESx92GXu45eaSskQtWMf9jYTSrbSb3h5D+mNUXzvmMa3ZEcrva\\\\\\\/QCt5IpgwwPo+X8qplmyC+v1RVb+jdnGPKRHsNDi6fM5YImzEmfhi32x4C\\\\\\\/ijhx\\\\\\\/AyXSvpp0caXliguSqf8w2higKBtxdnsKZcaktJugFK\\\\\\\/qDK6z2t3BJZk6mf5dcb6XBoau3Ygad+EB3Aa97RLZURokxDKKiuhX5fIBrzwHfEf4s16CtrENciutqT\\\\\\\/deFT5e5OxRNbz5jw2FZscILpUuFI1TaqgYta6l+DaRf7+0f9Auqs2wTYzGORSDk8voVLl7ylAzG+LAZnwksMciZcGM25skQZw0ubiePsFD\\\\\\\/QWriyAx0upQNHsyRCyQgn1NUkf4AQ3UV6fCq5ArIhx7B4cEv0oXlSWc+lXHaBzKAVgOHD4HLur+HGF1KPTGc6F48cVfyPKae6CNErho3QIrZaPXNr6Eikcu176ZMuXymiMhw5LPkmJqXbKqR8\\\\\\\/6wEroR0Frcx3LQjykFVz2vP+6T2zk2u70X5s9rPIMFbCPugR5BvLIk17ddprTrUAu8jPVxo9MCWLPX4HnShbPYIy0ouYEYlqun71o0EduaX6uCv5rzaEOL7RooliW7RuJjEeABQvCskt++jUh6Hu4Ajd2ESC5+K0vZuhesKNPRgmnqAg5j5A4mirjaiHM43B\\\\\\\/GDbhggyXrjGX7CcqBAJv+T0jWEt34IQpTiLy+CvtWLF64klCn1KU7xDX\\\\\\\/8lzevB6K02ba6qRMV8ME8IYUoZP95sohyQd4h\\\\\\\/98vgWReBpYJZ0sQwR9c1elv9vkvdLzzaCAbdL84nMYOM+H81XOv77qQt7i3Nc00XLCBKOWAUadzQV4\\\\\\\/ePa54PuXIgfnqs5uMW5PA50iswWkVLWAvJCt2AjTXYEqaMFrDrjLfrO2rMQnplva0hnqtEi59RdpvS1NbGN\\\\\\\/L8IOEBvIWfL\\\\\\\/CLrsxn7P7dpbgAJkuAPdD3JjWF3AlM5FADz88t6M3FMClkmNTsOqACD+m1OE0T7w+vzRJWA8lnTbivafuKh3igS\\\\\\\/kuLP2U6hoJLllvHMhecMoLHUHbPygQpEuenxVpY6lLS7IihSKAkFjWvFOWKvrsp3yHApWY\\\\\\\/rQuEqLkgtj3HdNskzVq7WGyQsQvmlMXR65gxkWu2\\\\\\\/jPxAmmJOyQ07MMznp5o9PicqFVW+tCadguG2Jty+sAeWf6ruANopPuCefGk40qlwHGmZ9DoYCFbpmRvo0i45lQCHn\\\\\\\/3+msvJFFBnG+vRU+ApfhzY7uxiohTRidrqT0KJ5gQpriRK7CCdfxrSSneNWkaVWtTqm8oWf86VWshWwFvE+\\\\\\\/LnEjQfwMC08LQa6aws8U+ts6lqv+r0k\\\\\\\/EI2X0E9VZdoOwE1eRZIlN4oPKeleOiz6dT2cj4k0paTDVYXojpN2MwsS3iIigkMZNNZPekffN1rm1N+mXvFEhSBk97LbWfKIhGTTRGPXck3oKy0xlW20hJw\\\\\\\/D\\\\\\\/0Edcgkyd5Nuc+jG358ls2LC3ZuI+uKy5eMjpzKvciexQEXjaCL1FpE3HVYpZuHAb9CK9qh4Si5\\\\\\\/OmmWS\\\\\\\/6tTrgMYB33hMoNq6H7gpvwKNp4uyxzLepKeAmVyHTqXmTHx0GrZs6QVL0KyB5JVv+Kf5RvRyGepB79GjLUHFU0SFmicn4Glp8tvIT2Vq6qAFLyXzrIqRVAamdFubuCee87k9QEawXJ6+NwoU5sKccx+pfK5+S+NFZURM4ufs4210FFZFmIJkBJcHEJH1G2U4wNUQBjS1ChpVq1xE59pTsGTFwsdit7AxIMp9iOU7PwmPO4ZH+Pgr5KR6M7G46QeCedP4IVXoCar9yQ6ZTWP7uk+MgJZC3tpJHHthnlPovCh8+haqwQrlsLvKEJs49xHyQDIN30TykbSFkEaow4zuwF+0ejiH9EpSVRQCiRe1jiKWUNU7w4EJ7YvH0CwHX5y4SZTEyFkFJyYtUaNPDn4jdgjKNUaSrKXaryN9f9OoycoOYJHhtTGRDl\\\\\\\/YgfiTp+JmxtH6rsR0OIaSyyg3aIDtyq1fXWSw9Gv4t6exMJGV5Dh88ckso6iantVUQSRcBwDNeIc2Qykth97ZvcW0wtt7qiwXTvSBVTD5j0+qY3Z5CrZhzg0J9pTB\\\\\\\/xQUn9g+SDqLFcZ5+4gWiumFrI2P5x3lpZr7azU56g3EhdYuvApk0bVO8lZxqqlNpufzKT+peu6vQz\\\\\\\/BGmZqslfkGKpWg+bmbBS60no8oGeUlWdsttIlDVs85PnraNJK5KnVVV97ecco5JiAZl9y7VyEHJYlO2mkBdVun75rSd5Fny\\\\\\\/efkBgVTWarkCTUtZ8UTAx6mHVnieWw9fcCigeIScWGvHiLCtKSF8YyMamWr54Im9Aq5vpmQNSvbcp6Vo8V1XB\\\\\\\/1F2WwwL5hVSfTZWJNkRVWCLRL8pVfrcHQQVCXeu7iy\\\\\\\/xV2axENsQU1L01FVwL6GUKnoS3imN\\\\\\\/LVVb\\\\\\\/7ihaKD8FnzKQRWm0WGKTI3uvXYVslbgOlf9DIP7FkxQG+Ap\\\\\\\/3bOy6XCz+O3Khkki2AhsFOqIMrfgtOUFFUs8nxpFR9\\\\\\\/8aBqnWAVUI9RoaLWiEauTG5iuHo5Xcol1owyOuOPpiCuFsPX4bZmxapTPc35u1MKaqcyCSpFMDf7EWvw\\\\\\\/7SkmKehu7WQLKu1F53xD4R9KDKIBJ+veSUPTBj\\\\\\\/MD8p0eiN6wd3IF\\\\\\\/51gWJKq89jjZfFEDp4UPD\\\\\\\/2l\\\\\\\/voswP0\\\\\\\/Py+oitdzXuJhIf\\\\\\\/nd3ufuKRBe88ZdQHA7TwA7JUlvyT\\\\\\\/AttgMHeEuPs\\\\\\\/xy9WrTjMYxxOis4x8cL8cw065MY1MffJTzG+cqidQKzB0R93OhGvqOK4N+l\\\\\\\/lhinE\\\\\\\/qYRIutR7DB7R5vNSJLI3EaW8xv5VFcW+SkPLQ7G2xhGdeOa0Y7U5NzMXJR+JkQjG1nnxYg6pvlXXb52JGukDisMnRqW1nkU7mJ828WJjzO6ReKFKSmtt+J4ztY906mglduGj9\\\\\\\/Q\\\\\\\/dQyDtIa876DHwCtVbh5uLls9orHLlWr30pYpeY5SQQjuerpfnlsLVTEpCrWoVYPdtC7rySUypdyE2+ZtcDmsTX52vg2XLTE5TLY\\\\\\\/wAqs8KyWdveAgciV\\\\\\\/JXO4ll78cElEZfmf0mImokfWd9QXMZPoa3pa18zmjKZi2Bns6ZfJgPgKh0VidtjO61jLQBWVZx0ucPxj5KrXyvx+CiK3IhrGpCBXqZQe1Q\\\\\\\/FblXC+izVPYZyyjgvnez+QCo8x4JdAak8t1zN31aM6cPJdcjdklMl\\\\\\\/BEU3FG4wThE6Y8l76+tzjOlVsA6cqWIpv7wHEllrP9dZ\\\\\\\/\\\\\\\/lh3\\\\\\\/SEjmtUFoux7lH0YZlVETzPOSKyhMq24In+QL8AIls6cTuokQKK6PTSKy0l+9oH9JvhadFjNppgx2y0C7VWLYYxYKjTeRvgDnfQhkCdXJG8\\\\\\\/ax8IGeGyttxc1FA3Oa\\\\\\\/LP1AmyI4D0rKNPhoVCfMAtpxPIH0fwOcUym+NgPJ\\\\\\\/mhUOiM6oP4CiIafqg2V5NLXN4IbAW+joo7TNkI2YXgXvr5UVl08oFdY1xNdtxsQ0+CDuQsz\\\\\\\/imLW7ZUn66DCNFf8pMfM7QwlskfrAFnTxjv9ojsfsxyBd\\\\\\\/FN\\\\\\\/od4RSXf52Dd3f0\\\\\\\/xg\\\\\\\/mVZOJlaruaxXRTCcmIbqm2yQP38Aht1FhNKJTCXuEr+tajOHE0NhtXhpkxib7Ad4WGjSvIT0cTVUE5ftyBYoVNZd1v9QZWSjOThXD5TJNmjEhPmTj9tZX846mQU83t4Oha0HbuCxfEKihwkWolnz7rla4zrydGwiwd3AO7uTqV6pCYWh8aCQti9f3wl9SLX4mMhc\\\\\\\/Ocvi4hIG3H0AUFOJmbNy\\\\\\\/ktH77RGMc4M7fszf4zGhKfxNsAGbguEmq4PVRrMIHJPE6jYmEX3FrYU30qIJMn1UJK9Lf5k1dzh\\\\\\\/FAC4OJE6jMjO92MEI3M6\\\\\\\/EHdgqIVap\\\\\\\/F1iA3RugvL7oehS3wK2FFhxcUAgaBWrEBRuMhcwiZF9gFODo\\\\\\\/rqIPSdTKdrYEiWVFvlwPM0mjbGatkTdDCEqpw60TSzc1XLlqdoJhQ8hPE\\\\\\\/1EHxCJacgsB2lcN6mMTCQgYmXtpDVxoQZqZC\\\\\\\/TkUUOKxeuey9L3WalEwIMD+W4kPguFtRjpNW0c0P5uXoqQi745BeOS\\\\\\\/BEMSeSLhHaqWFFS+TYi3v6xwUEXGB\\\\\\\/PVrMj2P4DKeW1j1s6eypmXbWIIwS7rTqjcW+OUvkpJGiplZ5GIJUTW1ALJoexLbTd1H2JS6s2JgBTj47MTDQSDnZPhvM5GCwUnfxREVuZ1HN65E1yjB1PnKbe0tqVGg0jHofZznq\\\\\\\/nWOEsegtSG7U1b9UF1Uk119VMMu\\\\\\\/gvcWOFnLdExmHkSd9lx6SUNj0r9+Go3DxrDErOK1XMot\\\\\\\/H5ytcnIhubqjsLrvsczeqgXDtCb7KhuNKrlOju5Gdm\\\\\\\/+IGGIBn6UtBea1o100kKee12SFluCZyP2do7f+7VJFGfuiytmwUJM+3IJEazQ\\\\\\\/kET5UDUiaJTDltJIpAE\\\\\\\/7\\\\\\\/AeCLE4twQBq8BcpdXltVVsuIB1Z\\\\\\\/swy5Z8+oguuEsm1mDpCPMQP3hyTmaI0Gb8mVY+w6u2SHwsk0aAaDkDDqGXTE\\\\\\\/j83vmRMsSBkMY6nm6Ub4mMmKKtnMszTR5IOWwK20LjkwhxnJEzDA6n\\\\\\\/Seu+RLy\\\\\\\/AoT0rfzd7M7oyIDwdk5kIJhqDw7bYUbTf9vQ2R98WPngzGZ4LYpBNw2QtJBX2OaDz2Jayxmhe5XhZwbRoWkrs8rGxM7MQFGQycvsYWXpry0olcSfgD9AdUv+7UJKa+KlSFxVQp+t+NDzJqUQVW57czwgixAYx2Js9zi7WsL3rxeIVVTYI\\\\\\\/bMfKdic1m1Fv1U1era6Heyqus2JWrzRkv\\\\\\\/wIZJTrrwMm8NYj9w4ticDmABlYEfhQC\\\\\\\/Og2SA8fUJ9q8fam0SUCtqoe3ntmVXU7cop4CocwTAvaYZnaRu9qNRezcKkhlDqE6DdXJWtS\\\\\\\/J6d7IYzu1lAyVhyIXTOpke6Vm0uPgOoJ0jw\\\\\\\/xkMnfSn\\\\\\\/rzkCsTbC0rwzgdg0vGoqW5aADm4JSwig5ur3LhEctCwu5mEOmgbsmK575yMJkY7gP8jC0e98stv7WPnLTTTM7tw4du+1H3XmnkxFBs2Z39lQSc0ziTVLBIKQ\\\\\\\/dzHoaDRdhXWe2l\\\\\\\/w4WUCc1SxvWJvYTNgkmU7uDSLwh6ThrD\\\\\\\/iaebOcdDdmUvq2\\\\\\\/bWny3Q5fsxGSrHTWtKrnOUzERiqiRnSjwG9pEf\\\\\\\/YDAmQf2SPXqft5W7QYVhcOmy00P4N+rmWWyylOs3tR9lt2LZgWo1g2i4B7Ey7ZugBRgNfsi5e3fcRI9H0ZvAMNQl9ooN6ZfA1mhMab7fohvXwU6O\\\\\\\/ii9khfs2yvoFGiw6xiy3W0qymjbuEmR3ld+GTXVd66OFxW0yZSJSbq6JxgKeupHL0uObwXOfARvTq0+oJP+bkWqFudi0sJZnQdCGO56y2OwPD7rOXfbsddzgQmmcXeWYsA2A9\\\\\\\/3WEOOQRK6BxZy3b5ooi6tFmqVcsYMhyQb1go5o2+sj9VwwGoQuuZGGvRYFbJb6G3J8k3e6OxRFIEMdEvaqkUGi6dA+HMP0PSSsFD6TYfCEhCD6vWtz+PNOVAqoHHhyFNYdm8RpiL99EiqmH2yUj2WY1vvbEyhtbH1MhnYeIg7ikr\\\\\\\/QTk21nWNAf2+qiYFuG1NkP8S61Ydfvp7\\\\\\\/hjn+BJHJ78y\\\\\\\/O5kOXXBgcRmQXUoqg1rkj9WUwT+VeS6Uyaz7B4UGagqLzqqCloLsxb1Bac0O1846Yqk66LuzFlmC0n7v5+2Zt5UHYMzpxyAJz6B0VEgYuG+0l1Hg6X\\\\\\\/JDxI6XJIhg7epjTtW+Iz36IBz6Hz42825FPMrctifbQPdiLxF3\\\\\\\/+O34lfII3P84nO1Z3Gs5T9i5nKhLIUWpr2YmB3IB+W5JCS1V2yvZA0oJekq6f1VDUx+bwvinV50yza3tMR2WrBfUeJ9psDm28r5DbFX0QjX\\\\\\\/cyIOUP0G0VXCBqZSNaN\\\\\\\/9htRVEl+Xktz3ZTEnBQgR2JscfUVBUgGfJdqTK6yE\\\\\\\/x98F50cJk29Lgmqd7S1ZkyAyFHPozTuUZCvO99GEYDGlpVXZMuObfJdl0Mk7KfF9GZtoW6a1eugf2u+Gbr3zzZ22o2naHv49JBnEoVNKg5tOLTZvjMAleBG8NWQ4zBCT737aX5PbeD9\\\\\\\/gBJ7SoLTgf1OB82X6awnlKKyMkPtQhaWZSwChUE5d66MA5mieMnoh8pYRIXoy6gJDNWMaa8XkczGQ5LwNQ2V4qY5rm0U5FOlvheGecuC+KSK0ZhAtGA\\\\\\\/iqINUcBJgXwoXocpU+HTbEI7Qq6uzDLVU5v\\\\\\\/dFYyHzq8I0oer\\\\\\\/SpiiF0to25x9YJolFEt1mM2izheoRg0rnSUYzZAvUgQ65RXbjy1tg9030JhRQDe\\\\\\\/JdfeRZd+bL\\\\\\\/jFBWUnKh5R7zeGsnB\\\\\\\/rcSzGy8+r9veolSvLmNldjhb1d1HPJF4GjEBzRaZLieJs8LFRAZleBqwRfDD0UaNZYwrDyeIhWN5LthaUTnrpCVrn4uLcNLVvq\\\\\\\/P4GtZrjWhZ3u8DwJ+tI8ZePo8tygJZ1QlGGYq0wgfrDdZ78UMeqIC0A0KSlWg+noOByf64RoB7Q9hvE7OR5LDob9OAfupXGyJqxltMLHCKRGLaOEHEjKtSunNjs6gco5F6DnccWyxU7FkDJpLwGio\\\\\\\/MT7zr8\\\\\\\/mwdAt2TBMFpCoc8VNFDAvEqAFqNEmro6gNWBYZ7W\\\\\\\/UlnPCq3NcPjxKLLDYRxcxXeg2ccIVHRBTWBK8wz7ASSPZtO6RaoXLTk8AS87yfHlo52906wFF94LxnrqRwyHXa7PMmcWDWVZQkPhhWXJCmHW1JS2izMlz0\\\\\\\/pjtBSLn7hFlGdOZW2vJeaQrREZRTC91dpC8emkJ95msmYIBV5gCO6no+MIkKj8QfKmrya3BHMhWTf+Rl50XkZG1B64HOWK65UVtCehEiQ3Lapz\\\\\\\/SAxqcSK67UhXDpeKFZzOB97QQerjJv6vfbnGTABjYMP\\\\\\\/YsnH9n3segKc\\\\\\\/4fwvA6anwEIQHzsp\\\\\\\/HY5KMNMXFHHtNfehD1MXVuFeT6tcYw90UeAyNV6ZkwRSav9HdcoyHg4bjX1AQk0hjpD9javJ2AWNbbrMgYt1x5L9x6neANPAe8wB7qFimfwkPMOYU3oksOnno\\\\\\\/6Eg+hXedmC1r+Y3\\\\\\\/hXVkRKrCeAKkFhX0ePeN5n0dNE3Z5nD7rflQTwEjqk4cfaJLMB9BC2\\\\\\\/urSl8azGLykB6y\\\\\\\/Kx\\\\\\\/60vjRbB+I1OIFanDR1My7myAfSTwafGsgz2ISetpgK7jPNJoQIMoi9orqGIp+OsBaIdMQvUyZThbWjWzaJ2PYiIfX7X5apgEK8JVT7iJGsa4sjl9PzCug\\\\\\\/bTTuwv\\\\\\\/0BsXnG2vmlmwFQPb01\\\\\\\/AAtW4mU7LsDK4g2nxoyhkjBvf\\\\\\\/I+X8OGg7AI0RJgi23XBcqol9eoy22QuLwmPF9v63vljRozbzUtmzunD5PZg2NFAzQQxwh3hwwK1SAd4yNkXPnWgHqxlyW1ogq3K6hQD2VuvpQz6qPkhXNJ03Fv4IxbWWoAR\\\\\\\/Z\\\\\\\/9P3Gg8vryMrwZ22fjwsZ9As0XUkMvvQztDQBPfsn8KkNlG\\\\\\\/LDH6gQIXRqCjdpB\\\\\\\/61HAsWqEJQmpm\\\\\\\/gUUP\\\\\\\/fUk\\\\\\\/1vZ6Cnq5EoV\\\\\\\/hwgvSR3tfkQKFi0oVJ2vicIWRAn92r6vFUdph9lV72sADF5iFIe5psqOIqySLXVKlPdGHqNOq7NxVvf8rJ+1K4ss\\\\\\\/SV8ao1bF7TXa6oen8hzTyyHePOtb3jZBrRyu6GCZAegqIQPvlyuWVXpAQSvQ98sYecpop+FEUaSJAvZGPj0cH0Yhs1prmdkZLyrNjOOlfL1uDEUlKOoEkhCKBJK3ZM0ok+Xmkj1an\\\\\\\/cvXoy5ZsMAgJbfafld\\\\\\\/RZudVV\\\\\\\/oivdrNkQvBIdz7bgSwUpXZimp6arO6Bl4jlYE4HLV8zvi0Z1HNEyWfNBfNcjrkhAK6hv0dkOICSS3q7UnA5C8MJ26QnsRrEWzb4tDIcWd13M1xTojrJ7HvHllWq1YMKWEWobNyoQiPUlHFQtde1aiTubcyzl1i6LKkwxKo5fYWF6D8FTPuDevgXOCathu33EdMTeJHcL0iuIfwIXRdnrhn2L\\\\\\\/Jh417hZtvIzhCcNMAV0oh49HdjN5UVqrYLK57c6QnDHZLJo61NgmmlN8ESHq5KKmn4lyUZ2TMD1fbc+EgAZrO4KxNtGmwFV+MrEg4Xt3Pr3xdK9SJUjMwo8hPsr4f5OIsBQVDK4qKJR6FA7GZEIo9euXHS8Xvgn72K\\\\\\\/FFpwMDrsyYkatdvTrC7bTzybe4uDtB0KU1\\\\\\\/OQViYq2fKlEhznlNzCGZuaRJNt99\\\\\\\/pv3\\\\\\\/8o\\\\\\\/yzoyyDyPQOOlROYtZ4umVvVC2k0uH7vwaB1jM0AeEK+9+oXyCzJI7aj8t883k9YIMGfAFejPjzac+JXc6SBiK54Ji04v5oQefALO7fCGhFL3YfFFs7ulNzb+e\\\\\\\/z83Z1Guv46o6G7u9F3yb13yql9VLIjKzfm99dVf8AwGPJQWRfz0KbJzP0FjNoueqANxznMl6QBylD75AnQNtKTseiTeLLeC87bPu96NSr5h9VsrB5oTLdR+8lfwViuUC6R8hJ2G7LunwWFApGK2SPvWbvogwBEt\\\\\\\/qC0\\\\\\\/5xsZn9qVpYMs5oVOO3IKGKVe069c0NnN1AZNfh5XQJUwMDk21igxtctPMbFQJZ0HQecDpElRttkpuFaWlyOSpFLX3ldhjsb3JeGId3jMvgfp4CQg3mP1B+xV4VQOw+U0g0ypdMmyygUjyVtIbfYMOnSJGH4YkHSXZUatns6q2\\\\\\\/OIrrKwe4Cb7Lnbx7J3JcWe0yKCXw3S0zYlXwwqkdFaw5DeMa6K4\\\\\\\/lRyOtL3bGMF+HI7Sx9ca5RAbsRswnXW1kip9DlE\\\\\\\/V844qzTeC+EpfIxw8kfS6FM0XgHvaf2+0sgbr8RzhYXQVSmpm4hKHoTrfRVTKOPv4EpYDuYZdfj4uI8jcnxBGXaOfAdsS1y1jG3KImnXXFmbOpfIk1GrOtQDRdXdD4Wm+G1SS0kJ+i7zA0tlnHpURN9vXGksomg5pLB9F0lKvb5oTpd2wh8RLqLEXJJ5U\\\\\\\/X+9rO4\\\\\\\/BlrxPlblqyH2yErxe\\\\\\\/qaEToVwz+ZGRItRPflM8p4TZUA\\\\\\\/RZLgFatvP5N3fQCQGpOlsTHcD8NsyQLRrxfLHneUUq\\\\\\\/YHyCtJVyQCx0mjWYGalgOjJijdgdBIDi32lp80rszQK\\\\\\\/dxTGfGjdWRgCybgppzUUvh5RUhlEE0c3llyqA2h79HcGQ+NSBBf1ca4AAiSMycX6qLoK1wwx3nqGEKRkNUQUvlscxD8WPDXgOFUKstNhU30gq+F4WN4hGa7nRzbfbaqaqbS1kjHqmFmbzfLWZOYGlJaHCLoRR49PSOtBkwaGgejLJ6PVJcAYzhn1R0UwYnpngonaWgORlZg0c6rT1PDl5+7eb4Bi4YAA9DNmJs24yh9btO80w\\\\\\\/XCHUvsuMCgu+eiRS5pb9uTA79XhuPp6TI3lCBkWzgm0lGjU1RuKlhEDFod7zgN7PDnYzlCqYqZVPYouAyCj9ZA\\\\\\\/OpSUIjdqLZGP3TOTLR1hV7eexHcbx9uq4Ga2f\\\\\\\/aQSUnB99lCgvpz0oK8htTYu7BgivttTghNBzAaCMFumA8AZEB7OsyP0J9LQqNgZIvptv+mRO\\\\\\\/+2kv08BXkdRUwtK57qyQaPPVsCLJ8NDZo\\\\\\\/Pyzzqx0qIrHZQsTiSaVgT3lHc2676R4lIZbmb0RbLpTGPwLmZbB3S\\\\\\\/vYOu\\\\\\\/F5k21ECNAzkl1cTMugwvrFq9AV5MKRMlrtHYaVv15oFEWyWJK+JyHV0HDi93bPsqtCWCJydbu\\\\\\\/YC9iS7FI3hURVg8nceQTuerAfqLSF\\\\\\\/3J3TehMKBQTNdDpt2N+Jx5Ah1V5TsCx8EHdCAqBmmEUjXRh2Mnae4SLxcOE7RX\\\\\\\/vaORmyMIUsjAe6QDX+8oMzbkpVqNoWvDXzLDgU+BjSAz4Mr7uf7hsK8MwCcF7rpQDlPK6F6mHlcCUU9\\\\\\\/FsfzChNvjQc\\\\\\\/KUnEaZOqzQt8FxUnTxqxz0dCoGdX9FIyDvP5Zf588rk0IoL1gGB046IPOvQS0p6aHJlRKRP0DDLPYuweYP5qGgi92CJ1jDk3ZHEOJ5\\\\\\\/WRctafIuTBH4M68yPeHskqT1KW\\\\\\\/xyTAexStlb01EEvygLR9wmsHStUdMZA5K80v\\\\\\\/zZWsGH4V\\\\\\\/9u\\\\\\\/SMPXRERqr14282JgFpN9yZNjLSVPcNFTc+rcowfarEYP\\\\\\\/vrTXjgpNwYw4th0VVXSuNHnNJTb\\\\\\\/JaKfY64WMkVnM6+OaoEPWlnUJ+S+3DQ3U7eUHYCr7ioThjp4rpmHAMQlNuHoCg4DqvLYRwvs5V0QHzqTC8yd035IgLxSiHYdbvk6uYqTvL58MXjc8F\\\\\\\/QArd51VMiLdrlSYX3fc0ts2YPLCzm0w5ZuNOMtJxkR8LjfOsR1XEue3fCaGifzyD8EQCvF0ReEo5K5b8nKheGaoQYP2IIr3G1\\\\\\\/Owey1Gh3LxCkCzCu+QOevjBb\\\\\\\/fNRrnfcctUSsReZMrfJyFVHS8aAzRDgkHUUCuFqcNBzO52KRAsn\\\\\\\/CU5hlS68vWFA7Zn3kSZpKAjh4uWyccJ\\\\\\\/78wuExmEPuYB9wjw8nKsPnlE6N7LOCWTXD0fyisKDDE9D7FL6UdqXh\\\\\\\/6YBhkG7zYignguY+QYtCdHLK9cp\\\\\\\/V\\\\\\\/P27f4mVQoim4wHnF06lexuHAwoD6upvHfWGevsuh9+jY\\\\\\\/uOhLWKBO92CP\\\\\\\/kj4CSGmslqjF+eZoQ75Gw\\\\\\\/rGZfGv1UutlfEyLhVqJ722i\\\\\\\/IJc0JlGrWHLwgc7Uw4wT96tUyaiwRpyQXQPZgzgzkbZm\\\\\\\/ZglVe0M6Pr\\\\\\\/Ie6LLJv9t6PvfP\\\\\\\/\\\\\\\/ayjDIGtMIVVhpiaCHjth9FJ3M4BN3nb1e7Mj1cqCeD6sePkLtJergmwq60YgWt7AxxGsNf7XTfWEwDhYU9Hps47ZYmAzSf0VzOChwDD9lBKHLd\\\\\\\/uuP13w7FVY4IGXa\\\\\\\/+GhDaEtlQLee+C4KNAXqw46ex8TM5MpYl94C9566nAx54PluHiF5XCBsJB85G\\\\\\\/dYHShUOIZ6rYsjXf7l00ETUMfeTLxP2tWGu4rMvNLGBNF42l2nJ+IsEffmec6HTUPMVQqnDtyfCcL58GQoHPyc\\\\\\\/UwXo6uuY\\\\\\\/cLEeFfUXKENNCsUOG8kmy0ntqMWgykjt\\\\\\\/bxuZ8CJGWS8ETldBAT4ji100rg7uvgsE0GTPlECdgDHgYZcyUAEhOvwB0bS1Mp+alyxfoj8krrfijSfwR\\\\\\\/YNPGzqgJDuhPsOoVKPjdqzcw0R3EGhouey7n8\\\\\\\/MfpnLkoMT40Hyv3+5FnJEljbJt08sMnCp+JqOP6Akx0HIOLPl0Ohw1c9iTtGCPkrhHRi2hwCZpjWG5kicguxZVT2SAuvOzZqXsUGZ6vis2OMimmDfj6p1cVTAotDmlTtg72Qu1yyhcRGfBuv8YDLCeCWghbjcPUHBgL5L4KVbrd2X39R+MGzTbV\\\\\\\/+jkqqg3G\\\\\\\/GZkAtfjjMQynbT3tnEPVSTLNwevVN5HbJQEWT0fdzJdqaeZWWDyUDqoO3TKhMC56y4Z4PtzVhy+7rvfKit5KUUImCumvfmKpaqt9R+JB2jxrE\\\\\\\/55zmd7oJEfvslq0QpvJvo3b1Jc4pQQD9ptOr\\\\\\\/IrDo3+\\\\\\\/nQwZQuyGFOeO74kmoWYeb7Uom9I0iVJQ5nI9S\\\\\\\/n8H0N3Drn\\\\\\\/kv+gz6OKRFtmrJN1AsCgdJbr+4MkGA60uaK72PWqt5b1zFOqRQJOHZnKJRBymOAdN7+XkbbD4YoYy5x8JTPLEdbaA9tS59mVEH6EEM11A9Cz9y4cn898lP7xArIsxa8g4XLdpIY\\\\\\\/D\\\\\\\/JQ5yHWofMfUnV9OtuDPRRz9Xu4jmKaQbCBOTxlcOied9LOCX3QrKMiER8abiFeAI\\\\\\\/\\\\\\\/tA+ZEg+V\\\\\\\/vLtn5UmWvprtAmq6vZeo9X4mDfLRKgpRlml\\\\\\\/bxmYMgpr0PmXG0MeSdJY+dqxBM3eafboMC6M5kqgsPpXZgBIfSCWg\\\\\\\/nodvxZ6GZZlfGIu2lHcfSZF52A71h06zrP5IIPvz\\\\\\\/wz1wblbOIqcqwV2d7\\\\\\\/AWYAqVhT6TqsZp4cm5CfjRw2llPXrG+iPvf2xBa6e81IeNMsRUa3DAn5RPuRFlb8aABPz5Pq8LGMTvJradv1qhuYU1Nr0pJrGxNnLU0d32F3ZgLLKfRlJt+N1dmk9TNXBDuLDPR1uxvEYH9sS9wo28aIUqIy6Qcde8EHPE9xtAPzy6Be5S\\\\\\\/SFwNKWqcTT5ge9D8ou+AVeX6UPhFA3eM87j9A3KTSItJaNF3N0lCbs+1h4inpkwSUEyfnWCh27JazlRstTIfwtCUsfDEjUn4nTQRj3bRvItpZqXbNSa38yCCCjU2p+ozShN7XaL8NWLlGCxd5hgazQVflqc9ka\\\\\\\/+k38Qta4+hPxZzNqeQyx9nYwwygcDYCvVTWxO8pZESMruLvM8xWvCSVLYMZPjsbg07DJN8QZX+AxiebcaDKgc7jxbzyOLOh97kJhdF1FXSAkoO0go2culWo\\\\\\\/vAuv\\\\\\\/zQqfB+Jg5a4ytPmkEHkfB7ChIAVXo90OjI0sX6Zd5APdsS\\\\\\\/UQmbBL4E4gRGtfu\\\\\\\/OxAAdFPkMOUffYywRliDPuws3OopxJldEU3HCJU9+banY+78riaAEEjfxy2V+1ZkBw1TfcXYcO\\\\\\\/kalFBO5vWqhDO0a0Ii\\\\\\\/oXMVZnog+R5QmdWITi0jrT86oTX7xqIdKG6Ma3PiuOQRqMYHTNysmMzqDVr8GYoJBvyPIfFDFlcEHzvXZKXhBtxoPWMhTesBldT\\\\\\\/uE\\\\\\\/GEmTM4idGJFM1Utuy+KEqmv7q3KCZXmsxGZ6HRWihdkdc3MEYds2Ns8+zKIMRBVAdeB3E9ag+OKPDQvEEoS10n5fGZk5tQ3BRpf4zWNwW99OQc5F9PpGcvD\\\\\\\/14L2KkX8sdOiK15rmrQ6f1hmqHBOjCe90CIUduAAocGLiIhrVlFoYw+\\\\\\\/EokgYKUKijWPknzvm+BnyBSzEmZ4GPjWCVvlV0Z0i094ZIkTcFe+rvSiAJ1LTZoKfXyqW6bgPVwAx31q4QFCO+FFQmW1QvinlLMg3OUuKFQMorxw6n3gFraK8lTYnt8\\\\\\\/OmRllNNu8VEDphXP\\\\\\\/SVDrLjdQr0+de3dimNDeAXWcwdb8MtWkpdPMQHKUF32pUsiLWqvZfsBh9QSUKzH0RLM6j+T+jIcf6wQENv8Id+mBn39sMsl3z7wRWYHr2YVOeWvK5wNPbo2wzxBSLzH+nB6Tj314FQvCPTmZjj1vA4emuSPuwpLxUEvrZumHSYJfXhvgAL+hXGM1n9Ww6XlsHAO2NTFJLzKULmt7X9KIS0xJWTuPuHq5EPwAugx+IbNBLcWv9aHzM1ibYgplsKJczTqaJwRkaPl9RCcooHghJT8quWud9NNAQRkiYpna0ua4O1h4nkRLcqMIEPObpv2sHVXBDVVeWEs7lQhG2gxVqeM35oy86i1F2w3hNzI7RHtZtgbGH+0TB2YJFIFaRJKJYH9v\\\\\\\/V3H7LBzIOJh2apSDRpG0czK0t9z+MHa6UVO2OK3pB8X8c5XlrS3cWgGob0WS\\\\\\\/2Vy\\\\\\\/07gDruHgJZqTCx0wKwF45veL8C0KhsLEzdv9u1vVWS9wUyCReC0L7g1Ac6KCjT7FJQ59mz0JC4nsvdRmRHByyt4y6kPYhgQNbo\\\\\\\/OgW73tKKAqI+MOoGcntAVSwo2joAGmRyRkBYCk3EDMLnAHFAU\\\\\\\/D+m0OBLX5f5jQUHEc+G01D2NNMtep51KNBJL01emcgOfByat7ld73s01vZlkKHQ4wopojGXcQcHyDbYfvITUXXvZ5\\\\\\\/C3AJHRdDMYvlZMdENg\\\\\\\/+S1uFNzsNKRk3HDJpIpJJnGEBJvRFC0fmggWPcNw2C6UiGigkhrJgWpV3qQfLbDQH6DzBRoO0U9RstyIMgO5hjgK+ZTQejoWtuySYPSr4EjK+6oGjOOoStoJ0dQUMHEQ9no7IOGmrrZU9txumf2M\\\\\\\/Ps86h1epQd2fW5oJMOQuqPIUoFVR+QfV6QFuFMDo0W6ZXuiPZs8adFSOD9u+ShHDMYVor65qQOrBXiihAYcZOFmBfAdgqskocD9R6bDkwq3+zRnrPiQImVqYGaho7i+gGxOFZLW7jc467i95R00dnbXmD\\\\\\\/XZYWWDSRX5RdQ6A\\\\\\\/2SqUrRW9CGM4WZNNw090eGopAIcC099DhVhhlGiw071I0HjFiat7UOVrCIxpd5Bnx403W6IbTtH97sIocB6mCIzqkyFCXMJBt7lPmdUGQvgv4j8WKsneLRqkEftxMSLOV2oxivj4\\\\\\\/sj+JABkz\\\\\\\/nHj6xopvNdSogrS1mGUnwLqLnGIHKg3fEfAe5FThhjdmCh\\\\\\\/7bmoYv3BN+gNUsDlXm0vtF+UT9v\\\\\\\/IdODvclLlLhb\\\\\\\/RgSOJ61y+ua70NthVXQj\\\\\\\/fQ+nowPIX+vcBEg3Pclm\\\\\\\/baRlbrgkg1tHzmQ6RyB9at\\\\\\\/jbQDOeq5TCeipsH\\\\\\\/uGtAlCLSBcH9hPFkuj83UphA\\\\\\\/2qN44RoIBvllK0Ru\\\\\\\/ad\\\\\\\/qL8Vl\\\\\\\/dbEeG5FEYakg2Rr3VBzI4tAfVVxQuql9ujbp7J9EnRrotd7oFmoWMlgsS3FZVMrrnw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wSrBNyGmRHLfheJoFIz5Bg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e6fa1471ff2bfa30d3a73ea643f455fe35a8642971fb5cc06c3e8d1ec9d45f2a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mFephCeH0bbC5OEr+myr0ihC\\\\\\\/RMQ5Wl6lxekaK0AxxJ4DIQMCWKCW6ftgw+YcQq7UjMXjuAYQxlkMM5II7eJ0kyM71gHEPnaR3xWadsQkIEx+gHg5qb2GqYTNIktfdJ\\\\\\\/QvhSQx1x6x\\\\\\\/1qpQOXvAovcC8zGAEL32LkAP1a0Ij4wQw3HtEMkKe9QrIeJc90VcYo0Hq2KIyJUPnK1moP41GjeyE4E4Y3eAJ7Q\\\\\\\/Sy\\\\\\\/WphKKDoaW+r1LKK27qpiuF8rM6fLKIGdUF5XN9ctqDxhD8gRygE+fyK7vL4IUW\\\\\\\/PtjaDTp00MQD\\\\\\\/d++icc+putKhp47lpNF7DwYZAH6Dk0BIKOfBhRUvLBHTgvK5NYFvhfgoMe3PliCUzIIkQie5oqHz7COXOrO7TNZZsH+vWWFtPMRUDnUg0RGiJL14T8k1j00FD\\\\\\\/wVaW0s3gPe4iSD3ILeV2cboIHv+ms7wNg6mVs6oRKcYyLH7F7wiZ1uWa3Wy4fAdKozF+mxD3FBQGhvH3vqrj+MlEtiXW8rJpiHywmGC55u29on94DgUvmRWgycT7faVWHVfAO+QeMa5n7s3Cha9NtOUoagsD0Cdu8nDR7gEzrtTKePY8B2jMWo103uBe\\\\\\\/5\\\\\\\/EG\\\\\\\/9h41Y0n2qjqVL1Jk65Z2mC4tMiDvyRSp6coyPQ6ijnr4N4AlE4u\\\\\\\/9iphCZ7qw+j4KfGSnIWBtoRHdw5K53VfO5\\\\\\\/nuDNTMwjNcMXTEibGSwhFhvEqHVEKF9z21Zpcja5mZftWKqjhmG1Sa+iMyR3Ium5+MvpsirLALTwh16amWcag4Ya9+wBqgo+ehIdyP0inb7VQd5OJLIn3gM06sE66+LtGAsUeNNUgtl+MLOdOm\\\\\\\/WAdHW2NHXqPU586w67MEW\\\\\\\/THMsddPwfUVywmWrXXYtxwblKP1SQCZBqGOY1ncTaWDEgPsC6vNozVKezIh1YEl6\\\\\\\/hsVkPLOtCBw0qpwuQN9GPjETbk+aPmLwDdxt2qvX5oPgBtUntQNTxFwe2HyFxBdIsXpmNK22mXOIRxMTeaI1S3AXRHfi7ttBndzAo5rU0sqWmRLxZjEU\\\\\\\/K3rgaQWl1+Q1ei31GOxsMZgHBnbxzzjfTHjeqZ3yhw\\\\\\\/nr5Aa3LDtGNiYNucwbo0TyNSVv5fpHksbHgAsnAWHQhyklh86Tiy2YMAMWGOMiRZfn5rpcxTsXOLnngpT8xPueoa6eHIV5arZkccF1PgxtF4fADgRWPOwnKkCteNKcIMliwkYVVFTuM+L9HKLT1ogTj5bjvhvqtDHil3pTpgcujEzHNnkHdRx5JdV0Dl\\\\\\\/zD3iOz4OAfwWWtoiulpBHeJAKZjUZDvEc3b2TZVulQjDCqS1ey7lJWrpSIFGbCBhwat\\\\\\\/1UVp\\\\\\\/k4vxppZHRN63t3t28dFMOZS8+ibWZEV5HlmoAwNXO3cF2OG9F4EFNeQ55lt4o7EHL7E1TCgCwrTALdOZyhV\\\\\\\/fYUDWfo3XTt3DQlG1\\\\\\\/SXGjwIbRZPm+NhORxicW1jNLoc\\\\\\\/1Al385gxXzCedS4MqRMPzINE+Cq7fefcIshC7t9r8CXOfP3o58VC0hfxZnE6zAVMoRMYZE2gwLM\\\\\\\/dpQpQQQW0qgbqlEZFU8Dupt9+2OXgt0F04cjVS+X4YrE2Qav17nq7tK+J6tVFrNwtgL0+J5KDqsY0073J2CfrtGtSFdjwhpXmJ6+qDxZl1s8Tnz6bRN33YMx7Pl2zarS\\\\\\\/D4ob2qMaq6\\\\\\\/aN+F0fbn9VanpVvef6Bhy8wRmaapRwiX5\\\\\\\/dq56IihwH4W6p41V9K56BauMT+WA0L3dsdK00al3YAPCT5\\\\\\\/m5bV3oSaEVEB4Nfm23Jr7lSLJFsy2hfnQAXYI4kfEqy7H8ZUNkMY7a+mH7LRrWsi45vMKnC9fGZns16oGKYRE5YDqDpRm90CmyFAnv\\\\\\\/4L1EGkWhJkV66EUzD1FKpcKnJXC5tIsscxIu8XpQ0B0voXBj9UGIGYwctlsKpE7RTZGg2NEmZ8wTr1V2OsOaRQbhjfqbqaBp0DIAC6IRmsYBkzXQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TVELaXHzICHYatHHWcVVDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222431a383952d315e3d182be948b58d4494477905b83e6843c29d3b2cb80fb0dc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459655 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1623" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5RoAkrUx4plCySqt8GQCD7KqNO5y4bhC_biqmoCrob4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4jeXGPCqf8vTU7FvXcXaN6rJPSzOWJDnXJT99z7eHr0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1611" title="The Genetic Architecture of Maize (Zea mays L.) Kernel Weight Determination" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1635" title="Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459655" id="node459655" data-pisa="ggg;4/9/1623" data-pisa-master="ggg;g3.114.013748" data-apath="/ggg/4/9/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel M. Gatti</span>, <span class="highwire-citation-author" data-delta="1">Karen L. Svenson</span>, <span class="highwire-citation-author" data-delta="2">Andrey Shabalin</span>, <span class="highwire-citation-author" data-delta="3">Long-Yang Wu</span>, <span class="highwire-citation-author" data-delta="4">William Valdar</span>, <span class="highwire-citation-author" data-delta="5">Petr Simecek</span>, <span class="highwire-citation-author" data-delta="6">Neal Goodwin</span>, <span class="highwire-citation-author" data-delta="7">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="8">Daniel Pomp</span>, <span class="highwire-citation-author" data-delta="9">Abraham Palmer</span>, <span class="highwire-citation-author" data-delta="10">Elissa J. Chesler</span>, <span class="highwire-citation-author" data-delta="11">Karl W. Broman</span> and <span class="highwire-citation-author" data-delta="12">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1633;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013748 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459655" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Karen L. Svenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BL.%2BSvenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Svenson%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BL.%2BSvenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrey Shabalin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Medical College of Virginia of Virginia Commonwealth University, Richmond, Virginia 23298</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BShabalin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shabalin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BShabalin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Long-Yang Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Long-Yang%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALong-Yang%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Neal Goodwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neal%2BGoodwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodwin%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeal%2BGoodwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, Australian Capital Territory 0200, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Daniel Pomp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BPomp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pomp%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BPomp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Abraham Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â â </span>Department of Human Genetics, University of Chicago, Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbraham%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡â¡</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1623" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459655" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459655/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1623.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459655" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459655/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1623.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459655" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459655/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1623.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459655" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459655/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1623.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459655" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459655/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genetic mapping studies in the mouse and other model organisms are used to search for genes underlying complex phenotypes. Traditional genetic mapping studies that employ single-generation crosses have poor mapping resolution and limit discovery to loci that are polymorphic between the two parental strains. Multiparent outbreeding populations address these shortcomings by increasing the density of recombination events and introducing allelic variants from multiple founder strains. However, multiparent crosses present new analytical challenges and require specialized software to take full advantage of these benefits. Each animal in an outbreeding population is genetically unique and must be genotyped using a high-density marker set; regression models for mapping must accommodate multiple founder alleles, and complex breeding designs give rise to polygenic covariance among related animals that must be accounted for in mapping analysis. The Diversity Outbred (DO) mice combine the genetic diversity of eight founder strains in a multigenerational breeding design that has been maintained for &gt;16 generations. The large population size and randomized mating ensure the long-term genetic stability of this population. We present a complete analytical pipeline for genetic mapping in DO mice, including algorithms for probabilistic reconstruction of founder haplotypes from genotyping array intensity data, and mapping methods that accommodate multiple founder haplotypes and account for relatedness among animals. Power analysis suggests that studies with as few as 200 DO mice can detect loci with large effects, but loci that account for &lt;5% of trait variance may require a sample size of up to 1000 animals. The methods described here are implemented in the freely available R package DOQTL.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/diversity-outbred" class="hw-term hw-article-keyword hw-article-keyword-diversity-outbred" rel="nofollow">diversity outbred</a></li><li class="kwd"><a href="/keyword/haplotype-reconstruction" class="hw-term hw-article-keyword hw-article-keyword-haplotype-reconstruction" rel="nofollow">haplotype reconstruction</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus-mapping" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus-mapping" rel="nofollow">quantitative trait locus mapping</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-4">QUANTITATIVE trait locus (QTL) mapping is an approach widely used for detecting and localizing genetic variants responsible for phenotypic variation. Traditionally, QTL intervals are defined by the statistical association of phenotypes with genetic markers in backcross or intercross progeny derived by crossing two inbred strains (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lander and Botstein 1989</a>). This experimental design relies on a single generation of meiotic recombination to break up and randomize the parental genomes; as a result QTL intervals are often too large to identify causal variants (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Flint <em>et al.</em> 2005</a>). Furthermore, the potential for discovery of genetic associations is limited to loci that are polymorphic between the two parental inbred strains. Experimental mapping populations with increased mapping precision and greater genetic diversity are desirable to improve the performance of QTL mapping studies.</p><p id="p-5">Heterogeneous stocks (HS) derived from multiple inbred founder strains and maintained as outbreeding populations for many generations can increase mapping precision and introduce additional genetic diversity (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Lumeng 1984</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hitzemann <em>et al.</em> 1994</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Iancu <em>et al.</em> 2010</a>). The Diversity Outbred (DO) mice are a heterogeneous stock derived from eight inbred founder strains and maintained by randomized breeding among 175 mating pairs (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Svenson <em>et al.</em> 2012</a>). The founder strains A/J, C57BL6/J, 129S1/SvImJ, NOD/ShiLtJ, NZO/HlLtJ, CAST/EiJ, PWK/PhJ, and WSB/EiJ (denoted below as AâH, respectively) are the same strains that were used to establish the Collaborative Cross (CC) recombinant inbred lines (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Churchill <em>et al.</em> 2004</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Collaborative Cross Consortium 2012</a>). The DO population captures at least 37.8 million single nucleotide polymorphisms (SNPs) plus 6.9 million insertions, deletions, and structural variants that are uniformly distributed across the genome (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Keane <em>et al.</em> 2011</a>). Each DO mouse is heterozygous by ancestry over an average of 7/8th of its genome, and genetic variants have an expected minimum minor allele frequency (MAF) of 1/8. A related heterogeneous stock (CC-HS) was derived from the same founder strains using a circular breeding design (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Iancu <em>et al.</em> 2010</a>). The CC, DO, and CC-HS populations segregate the same genetic variants in different population structures; they provide a complementary set of resources for high-resolution QTL mapping and validation.</p><p id="p-6">QTL mapping analysis in HS populations presents a number of analytical challenges. The genome of each animal is a unique mosaic of founder haplotypes. Accumulation of meiotic recombination events over many generations of breeding results in small haplotype blocks that provide high mapping resolution but also require high-density genotyping (&gt;50K SNPs) to reconstruct the haplotypes. We demonstrate the necessity of high-density genotyping to obtain accurate haplotype reconstructions and show that accuracy is improved by directly utilizing probe intensity data from genotyping arrays rather than genotype calls. We produce probabilistic estimates of founder haplotypes and these provide a basis for genetic mapping and genome-wide SNP imputation. QTL mapping regression models can be formulated in a variety of ways using founder haplotypes or SNP allele calls and we implement three different regression models and discuss their relative merits. Unlike simple cross designs, HS mice are not all related to the same degree and covariance among related individuals necessitates the use of a mixed-model linear regression with a kinship term (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kang <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheng and Palmer 2013</a>). Methods for determining genome-wide significance levels and support intervals for QTL location also require special attention in the context of HS populations. We evaluate the performance of a simple permutation test for HS populations and demonstrate that it has the correct size despite the presence of population structure. Simulations provide estimates of power and sample size required for successful QTL mapping. We illustrate novel features of the DOQTL software using an example of mapping hematological traits in DO mice.</p><p id="p-7">We have implemented specialized QTL mapping software (DOQTL) for use with DO mice and other HS mapping populations. DOQTL implements a complete analytical pipeline for haplotype reconstruction and QTL mapping in the DO. It includes alternative modes of analysis for some steps in the pipeline. The modular design of the software will enable development and evaluation of new methods and extension to other HS populations with relatively few modifications.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mice and genotyping</h3><p id="p-8">We isolated DNA using the Promega (Madison, WI) Maxwell 16 and Tissue DNA purification kit according to the manufacturerâs instructions. GeneSeek (Lincoln, NE) carried out genotyping assays on a total of 4542 DO mice. Genotype callsâA, B, H, or Nâwere generated using Illuminaâs BeadStudio algorithm. Here A represents homozygosity for the reference allele, B is homozygous for the alternative allele, H is a heterozygous genotype, and N is âno call.â</p><p id="p-9">We genotyped 3029 of the 4542 DO mice from outbreeding generations G4 through G9 using the Mouse Universal Genotyping Array (MUGA) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Welsh <em>et al.</em> 2012</a>). The MUGA assays 7854 SNPs spanning the 19 autosomes and X chromosome of the mouse with a mean spacing of 325 kb. We removed 190 probe pairs from the MUGA data that did not perform well. Of these DNA samples, 34 were run in duplicate in different batches of MUGA genotyping.</p><p id="p-10">We genotyped 1513 of the 4542 DO mice from outbreeding generations G8 through G11 using the higher-density MegaMUGA platform (GeneSeek, 2013). The MegaMUGA assays 77,818 SNPs across the mouse genome with a mean spacing of 33 kb. We used a set of 57,977 SNP assays that distinguish among the genotypes of the eight founder strains of the DO and their heterozygous F1 offspring.</p><p id="p-11">We obtained MUGA and MegaMUGA data for the eight founder strains and most of their F1 hybrid progeny. We adjusted the founder and DO sample intensities for batch differences by quantile normalization across samples at each marker and calculated the intensity quantiles at each marker for the DO samples and the founders/F1âs separately. We then normalized the intensities of the founder and F1 samples to match the quantiles of the DO samples.</p><p id="p-12">To obtain reliable and robust reference points for interpreting the intensity data, we assayed multiple DNA samples from animals of each sex, each founder strain, and most of the F1 hybrids, on both MUGA and MegaMUGA platforms. At each marker, we applied a polar coordinate transformation to the <em>X</em> and <em>Y</em> intensity values on combined data from founder, F1 and DO DNA samples (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>). We estimated the mean and variance of the angle <em>Î¸</em> and radius <em>Ï</em> for each cluster using mclust software (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fraley 2012</a>). Mclust fits a bivariate Gaussian mixture model at each marker and maximizes the Bayesian information criterion to determine the number of clusters. Probe intensities are expected to fall into three distinct clusters corresponding to SNP genotypes A, H, and B. However, 16% of MUGA probes and 6% of MegaMUGA probes consistently yielded more than three distinct clusters due to variant(s) that are present near or within the constant portion of a probeâs target sequence. Intensity data from these nonconforming probes can provide more information than genotype calls leading to improved accuracy in haplotype reconstructions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fu 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="X and Y probe intensities on the MUGA contain information about founder haplotypes that is not captured by the four genotype allele calls. Each point represents the X and Y intensity values for one sample at marker UNC190139327, chromosome 19, 5.083353 Mb (rs46230775). Locations of the founder strains, including multiple independent samples of founder and F1 hybrids animals, are shown as colored circles. Founder strains are represented as colored circles and F1 hybrids are represented by a circle in one founder color and an internal triangle in the other founder color. For example, the eight red points in cluster 3 represent replicates of the PWK/PhJ founder. (A) Cluster 1 represents homozygotes and heterozygotes between A/J, CAST/EiJ, NOD/ShiLtJ, and WSB/EiJ. Cluster 2 represents heterozygotes between the founders in cluster 1 and PWK/PhJ. Cluster 3 represents PWK/PhJ homozygotes. Cluster 4 contains heterozygotes between the founders in clusters 1 and 6. Cluster 5 represents homozygotes and heterozygotes between PWK/PhJ and the strains in cluster 6. Cluster 6 consists of homozygotes and heterozygotes between 129S1/SvImJ, C57BL/6J, and NZO/HlLtJ. (B) The X and Y intensities from A have been transformed to &#x3C1; and &#x3B8; coordinates and the cluster axes are aligned vertically. (C) Initial 36 diplotype state cluster locations that are estimated using mclust from the sample and founder data. Many diplotype states have the same cluster center. (D) Final diplotype state locations after the genotyping algorithm has completed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1145617912" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;X and Y&lt;/em&gt; probe intensities on the MUGA contain information about founder haplotypes that is not captured by the four genotype allele calls. Each point represents the &lt;em&gt;X&lt;/em&gt; and &lt;em&gt;Y&lt;/em&gt; intensity values for one sample at marker UNC190139327, chromosome 19, 5.083353 Mb (rs46230775). Locations of the founder strains, including multiple independent samples of founder and F1 hybrids animals, are shown as colored circles. Founder strains are represented as colored circles and F1 hybrids are represented by a circle in one founder color and an internal triangle in the other founder color. For example, the eight red points in cluster 3 represent replicates of the PWK/PhJ founder. (A) Cluster 1 represents homozygotes and heterozygotes between A/J, CAST/EiJ, NOD/ShiLtJ, and WSB/EiJ. Cluster 2 represents heterozygotes between the founders in cluster 1 and PWK/PhJ. Cluster 3 represents PWK/PhJ homozygotes. Cluster 4 contains heterozygotes between the founders in clusters 1 and 6. Cluster 5 represents homozygotes and heterozygotes between PWK/PhJ and the strains in cluster 6. Cluster 6 consists of homozygotes and heterozygotes between 129S1/SvImJ, C57BL/6J, and NZO/HlLtJ. (B) The &lt;em&gt;X&lt;/em&gt; and &lt;em&gt;Y&lt;/em&gt; intensities from A have been transformed to &lt;em&gt;&#x3C1;&lt;/em&gt; and &lt;em&gt;&#x3B8;&lt;/em&gt; coordinates and the cluster axes are aligned vertically. (C) Initial 36 diplotype state cluster locations that are estimated using mclust from the sample and founder data. Many diplotype states have the same cluster center. (D) Final diplotype state locations after the genotyping algorithm has completed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1623/F1.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1623/F1.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464830" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child"><em>X and Y</em> probe intensities on the MUGA contain information about founder haplotypes that is not captured by the four genotype allele calls. Each point represents the <em>X</em> and <em>Y</em> intensity values for one sample at marker UNC190139327, chromosome 19, 5.083353 Mb (rs46230775). Locations of the founder strains, including multiple independent samples of founder and F1 hybrids animals, are shown as colored circles. Founder strains are represented as colored circles and F1 hybrids are represented by a circle in one founder color and an internal triangle in the other founder color. For example, the eight red points in cluster 3 represent replicates of the PWK/PhJ founder. (A) Cluster 1 represents homozygotes and heterozygotes between A/J, CAST/EiJ, NOD/ShiLtJ, and WSB/EiJ. Cluster 2 represents heterozygotes between the founders in cluster 1 and PWK/PhJ. Cluster 3 represents PWK/PhJ homozygotes. Cluster 4 contains heterozygotes between the founders in clusters 1 and 6. Cluster 5 represents homozygotes and heterozygotes between PWK/PhJ and the strains in cluster 6. Cluster 6 consists of homozygotes and heterozygotes between 129S1/SvImJ, C57BL/6J, and NZO/HlLtJ. (B) The <em>X</em> and <em>Y</em> intensities from A have been transformed to <em>Ï</em> and <em>Î¸</em> coordinates and the cluster axes are aligned vertically. (C) Initial 36 diplotype state cluster locations that are estimated using mclust from the sample and founder data. Many diplotype states have the same cluster center. (D) Final diplotype state locations after the genotyping algorithm has completed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Haplotype reconstruction</h3><p id="p-14">Each chromosome pair of a DO animal is composed of a unique mosaic of founder haplotypes. We refer to the pair of founder haplotypes at a given locus as the founder <em>diplotype</em>. There are 36 possible diplotypesâ8 homozygous and 28 heterozygous diplotypes. We developed a hidden Markov model (HMM) to reconstruct the diplotypes from the genotype array data by generating a probabilistic estimate of the diplotype state (<em>s</em>) at each marker locus (<em>j</em>) in each DO animal (<em>i</em>). We denote these <em>diplotype probabilities</em> as <em>p<sub>ij</sub>(s)</em>.</p><p id="p-15">The HMM is composed of a <em>transition model</em> and an <em>emission model</em>. The transition model is a Markov chain with 36 states that defines the probability of transitioning from one diplotype state to another between adjacent marker loci by meiotic recombination. We calculated transition probabilities between markers based on genetic map positions of markers (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cox <em>et al.</em> 2009</a>) and the outbreeding generation (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman 2012</a>). The emission model defines the conditional probability distribution of observed data given the underlying diplotype state. We implemented two different emission models, a discrete model for genotype call data, and a continuous model for intensity data. Emission model parameters are estimated using an expectation-maximization (EM) algorithm. We describe each of the emission models here.</p><div id="sec-4" class="subsection"><h4>Genotype model:</h4><p id="p-16">The emission model for genotype calls is a multinomial distribution over the four outcomes A, H, B, and N. The expected diallelic genotypes corresponding to the diplotype states at a given marker were obtained from the genomic sequences of the founder strains (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Keane <em>et al.</em> 2011</a>). We initially set the expected genotype to occur with high probability (0.97) and distributed the remaining probability evenly over the other outcomes (0.01 each) to allow for genotype calling errors and no calls. We treat no call N as a separate, informative outcome because it contains information about the underlying diplotype state. A no call can occur at different rates for different true diplotype states and therefore contains information about the diplotype state. For example, when the diplotype is FF (CAST homozygous), the genotype call may be N 90% of the time, whereas for a BB diplotype (C57BL/6J homozygous), N calls may occur only 1% of the time.</p></div><div id="sec-5" class="subsection"><h4>Intensity model:</h4><p id="p-17">The emission model for intensity data are a bivariate Gaussian distribution. For each diplotype state at a given marker, the distribution of intensities in polar coordinates is defined as the product of two univariate normal distributions. We also investigated models with a covariance term and models of <em>xây</em> intensity but these are not considered further here. There are four parameters for each diplotype at each marker, two means and two variances. It is important to obtain good initial estimates for these parameters. At a given marker, we assigned each of the 36 diplotype states to a cluster (described above) using the founder and F1 samples as reference points. Initial parameter estimates for each diplotype state were obtained from these clusters (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a>). However, we allowed each diplotype state to follow its own distribution to correct errors in the initial cluster assignments and to extract additional information from the intensity data (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>).</p></div><div id="sec-6" class="subsection"><h4>The EM algorithm:</h4><p id="p-18">EM algorithms for parameter estimation in discrete or continuous outcome HMMs have been described in detail elsewhere (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baum 1970</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Churchill 1989</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rabiner 1989</a>). In our implementation, transition model parameters were held fixed over iterations; only emission model parameters were estimated from the data. In principle, the transmission parameters could also be estimated but we found that the resulting diplotype reconstructions tended to transition too frequently between diplotype states.</p><p id="p-19">The E-step uses current estimates of model parameters to execute a forwardâbackward HMM algorithm and generates diplotype probabilities <em>p<sub>ij</sub></em>(<em>s</em>). The E-step is computationally intensive but it can be applied to one individual and one chromosome at a time. The M-step uses the diplotype probabilities to compute new estimates of the (emission) model parameters. The M-step can be computed one marker at a time; it combines information across individuals. These steps are iterated until a convergence criterion is satisfied. We stopped iterations when the log-likelihood of the model changed by &lt;1/1000 of the initial log-likelihood in successive iterations. Best results are obtained with large collections of individuals, at least several hundred.</p><p id="p-20">A <em>marginal reconstruction</em> of DO haplotypes can be obtained by assigning each locus to the diplotype state that has maximum probability. The diplotypes are phased to obtain a pair of haplotypes by minimizing the number of recombination events. Phasing is not critical for QTL mapping but we use phased chromosomes for graphical representation and to count recombination events. Our phasing algorithm could result in undercounting of recombination events but it is simple and appears to work well. We place transitions at the midpoint between flanking marker loci. SNP genotypes across the entire genome can be imputed from the marginal reconstruction by reference to the founder genome sequences to obtain a diploid genome sequence for each DO animal. Probabilistic imputation of SNP genotypes can also be obtained as discussed below.</p></div></div><div id="sec-7" class="subsection"><h3>Regression modeling</h3><p id="p-21">We computed genome scans by regression on diplotype probabilities (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Haley and Knott 1992</a>). We fit a linear mixed model with sex and batch as covariates. A random-effect term captures the polygenic covariance due to kinship among the animals. The likelihood ratio comparing this model to a null model without the diplotype term is converted to a LOD score or to âlog10(p-value). We consider three different ways to include the diplotype states in the regression model, which we refer to as the <em>full model</em>, the <em>additive haplotype model</em>, and the <em>additive SNP model</em>. The full and additive haplotype models can be computed at the marker loci or, if desired, at points between marker loci using interpolation to approximate the diplotype probabilities. The additive SNP model is computed at imputed SNP loci.</p><div id="sec-8" class="subsection"><h4>Full model:</h4><p id="p-22">The full model is an unconstrained regression on the 36 diplotype states. We express the full model at marker locus <em>j</em> as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>y<sub>i</sub></em> is the phenotype for animal <em>i</em>, <em>x<sub>ik</sub></em> is an indicator for covariate <em>k</em> and animal <em>i</em>, <em>a<sub>k</sub></em> is the effect of covariate <em>k</em>, <em>p<sub>ij</sub>(s)</em> is the diplotype probability at a marker locus <em>j</em> for animal <em>i</em> generated by the HMM, <em>Î³<sub>i</sub></em> is an adjustment for kinship, and <em>Îµ<sub>i</sub></em> is the residual error. The model includes 36 regression coefficients, <em>Î²<sub>s</sub></em>, one for each diplotype state <em>s</em>. The full model is general and allows for arbitrary patterns of additive and dominance effects among the founder alleles. However, the likelihood-ratio test with 35 degrees of freedom is expected to perform poorly unless sample sizes are very large.</p></div><div id="sec-9" class="subsection"><h4>Additive haplotype model:</h4><p id="p-23">The additive model assumes that the effects of founder alleles combine additively; there are no dominance effects in this model. To implement the additive model, we compute the <em>allelic dosage</em> of founder <em>h</em> at each marker at marker locus <em>j</em> as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>N<sub>h</sub>(s)</em> is the number of alleles coming from founder <em>h</em> in diplotype <em>s</em>. For example, <em>N<sub>c</sub></em>(CC) = 2, <em>N<sub>c</sub></em>(BC) = 1, and <em>N<sub>c</sub></em>(BB) = 0.</p><p id="p-24">We express the regression equation as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <em>y<sub>i</sub></em>, <em>x<sub>ik</sub></em>, <em>a<sub>k</sub></em>, <em>Î³<sub>i</sub></em>, and <em>Îµ<sub>i</sub></em> are the same as in Equation 1, and <em>d<sub>ij</sub></em>(<em>h</em>) is the allelic dosage of founder <em>h</em> at marker locus <em>j</em> from Equation 2. The model includes eight coefficients <em>Î²<sub>h</sub></em> representing the additive genetic effects of each founder haplotype. Thus the likelihood-ratio test has 7 degrees of freedom. We expect a significant gain in power and more precise estimation of the regression coefficients compared to the full model. However, we risk missing dominance interactions among founder haplotypes.</p></div><div id="sec-10" class="subsection"><h4>Additive SNP model:</h4><p id="p-25">To implement the additive SNP model we compute a probabilistic imputation of the genotype at every known SNP locus genome-wide (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baud <em>et al.</em> 2013</a>). The expected allelic dosage is computed as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where <em>G<sub>j</sub>(s)</em> is the number of occurrences of the reference allele at marker locus <em>j</em> for an animal with allelic dosage of founder <em>h</em> at that locus. An advantage of mapping at the two-state SNP level is the potential to introduce dominance effects with one additional degree of freedom. We focus on the SNP-based additive model, which is widely used in human association mapping (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bush and Moore 2012</a>). To gain computational efficiency we assign the diplotype state probability between each adjacent pair of genotyped markers to the average of the flanking diplotype state probabilities. Any sets of SNPs in the interval with identical strain distribution patterns among the eight founders are assigned identical values and we compute the regression once for each set of identical SNPs in an interval.</p><p id="p-26">We express the regression equation at SNP locus <em>j</em> as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where <em>y<sub>i</sub></em>, <em>x<sub>ik</sub></em>, <em>a<sub>k</sub></em>, <em>Î³<sub>i</sub></em>, and <em>Îµ<sub>i</sub></em> are the same as in Equation 1, <em>g<sub>ij</sub></em> is generated by Equation 4, and <em>Î²<sub>g</sub></em> is the additive effect of allelic substitution.</p></div><div id="sec-11" class="subsection"><h4>Kinship:</h4><p id="p-27">For each pair of DO animals, we computed the expected allele sharing of founder haplotypes based on the diplootype probabilities from Equation 2. Specifically, the kinship between sample <em>i</em> and <em>e</em> is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1623/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>where <em>M</em> is the number of markers and <em>d<sub>ij</sub>(h)</em> is the allelic dosage of founder <em>h</em> for sample <em>i</em> at marker locus <em>j</em> (Equation 2). We have implemented genome scans using the leave-one-chromosome-out method (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cheng and Palmer 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2014</a>).</p></div><div id="sec-12" class="subsection"><h4>Implementation:</h4><p id="p-28">We extended the QTLRel software (v. 0.2-14) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2011</a>) to fit any of the regression models described above. QTLRel estimates the variance scaling factor for the kinship correction for each phenotype. To provide an alternative method for fitting the mixed model regression we adapted the R package MatrixEQTL (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shabalin 2012</a>). MatrixEQTL accepts any user-supplied matrix as the error covariance matrix and applies the same covariance structure to all phenotypes. This provides substantially faster computation for multiple phenotypes but the use of the same error covariance matrix for all phenotypes is an approximation.</p></div><div id="sec-13" class="subsection"><h4>Significance thresholds:</h4><p id="p-29">A Bonferroni correction can be used to obtain genome-wide thresholds. For MUGA the 0.05 Bonferroni corrected threshold is âlog(p) &gt; 5.2, and for MegaMUGA it is âlog(p) &gt; 6.0. The equivalent LOD thresholds are dependent on the sample size and choice of regression model. Bonferroni corrections are often too conservative and it is desirable to use a stringent but less conservative threshold based on permutation analysis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Churchill and Doerge 1994</a>). However, the kinship structure of the DO population violates the usual exchangeability conditions (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Churchill and Doerge 2008</a>). Recent reports have suggested that by fitting the regression model without the kinship adjustment term to each permuted data set, one can obtain conservative thresholds for the mixed-model genome scan (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cheng and Palmer 2013</a>). We validate this procedure using simulations to show that it provides expected control of the type I error rates in the DO. We used a significance threshold of p â¤ 0.05 to select significant mapping associations.</p><p id="p-30">Support intervals for QTL localization were determined using a 95% Bayesian credible interval (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sen and Churchill 2001</a>). The LOD curve is transformed by raising it to the power of 10 and a region covering 95% of the area under the transformed curve defines the support interval. The area under the curve is numerically approximated using trapezoids between the marker loci.</p></div></div><div id="sec-14" class="subsection"><h3>Power simulations</h3><p id="p-31">We randomly sampled genomes from a set of marginal reconstructions of 1129 DO animals at outbreeding generation G8. We simulated phenotypes using sample sizes of 200, 400, 600, 800, and 1000 animals, with MAF of 1, 2, 3, and 4 and QTL effect sizes of 0.125, 0.25, 0.375, 0.5, 0.625, 0.75, and 1.0 in standardized units. The MAF refers to the number of founders that contribute the causative allele. Each combination of these parameters was simulated 1000 times. Phenotypes were simulated as the sum of a QTL effect plus a polygenic background component and an independent normal error component each with unit variance. Thus the polygenic and independent error variance components contribute equally to the total background variance. For each simulation, the QTL location was selected at random on the autosomal genome, minor allele status was assigned to the appropriate number of founder strains, and the genotype was determined by the diplotype of the marginal reconstruction. The QTL effect size was added to one homozygous class and subtracted from the other. We ran genome scans using the additive haplotype model and declared detection of the target QTL if the LOD score exceeded a genome-wide adjusted p â¤ 0.05 threshold. LOD thresholds were determined once for each set of simulations at a given sample size based on the maximal LOD score obtained on each of 1000 genome scans of phenotypes drawn independently from a normal (0,1) distribution. We tabulated power as the detection of significant QTL within 5 Mb of the simulated QTL location. Note that while the type I error rate is controlled genome-wide, power is calculated at a fixed locus and thus can fall below the nominal 0.05 level.</p></div><div id="sec-15" class="subsection"><h3>Type I error simulations</h3><p id="p-32">We used the same set of DO genomes as in the power simulations above and mapped simulated phenotypes on the 19 autosomes. We selected a sample size <em>n</em> (200, 400, 600, 800, or 1000) and drew <em>n</em> DO genomes at random from our G8 animals. We simulated a null phenotype by sampling <em>n</em> values from a Gaussian distribution. We mapped the phenotype using an additive haplotype model with a kinship adjustment. We repeated this process 1000 times for each sample size and report the proportion of times in which a QTL was detected on any autosome at the p â¤ 0.05 significance thresholds. Significance thresholds for the X chromosome require special consideration (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2006</a>) and have not yet been implemented in DOQTL.</p></div><div id="sec-16" class="subsection"><h3>Phenotypes</h3><p id="p-33">DO mice (742 total, 410 females, 332 males) from outbreeding generations G4 and G5 were obtained from the Jackson Laboratory. Mice were housed in polycarbonate cages (50 in.<sup>2</sup>, 4â5 mice per cage) and fed NIH-31 6% fat mouse diet <em>ad libitum</em>. Acidified (pH 2.8â3.2) or chlorinated (10â15 ppm residual Cl) water was provided in water bottles. Mice were maintained on a 12 hr light/12 hr dark cycle. At ages ranging from 8 to 10 weeks, 50 ml of blood was collected via retro-orbital bleed and stored at 2â8Â° in K<sub>2</sub>EDTA. Neutrophil counts and other whole blood parameters were measured within 24 hr of collection on the Drew Scientific HemaVet 850 FS analyzer (Dallas, TX). Samples were processed in 15 batches of â¼50 animals each and some failed samples were removed; 5 mm of tail were collected for genotyping. These mice were a subset of the 3029 mice genotyped on the MUGA array. All procedures were reviewed and approved by the Jackson Laboratoryâs Institutional Animal Care and Use committee (Protocol JW10001/JW202).</p></div><div id="sec-17" class="subsection"><h3>Heritability</h3><p id="p-34">The narrow sense heritability of neutrophil counts in the 742 DO mice was estimated by fitting a mixed model with sex and phenotyping batch as fixed effects and the additive genetic matrix as a random effect (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Clifford and Mccullagh 2006</a>). The ratio of the additive genetic variance over the total phenotypic variance was used to calculate <em>h</em><sup>2</sup> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Visscher <em>et al.</em> 2008</a>).</p></div><div id="sec-18" class="subsection"><h3>Data and software availability</h3><p id="p-35">All data associated with this manuscript can be obtained at <a href="http://do.jax.org/">http://do.jax.org/</a>. The DOQTL R package is available from Bioconductor (<a href="http://bioconductor.org/">http://bioconductor.org/</a>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gentleman <em>et al.</em> 2004</a>). Genotyping data on the MUGA and MegaMUGA for the DO founders and F1s is available at <a href="ftp://ftp.jax.org/MUGA">ftp://ftp.jax.org/MUGA</a>.</p></div></div><div class="section results" id="sec-19"><h2 class="">Results</h2><div id="sec-20" class="subsection"><h3>Genome reconstructions</h3><p id="p-36">We computed diplotype state probabilities and marginal haplotype reconstructions for 4542 DO mice using allele-call and intensity-based methods for MUGA or MegaMUGA data. For each of 34 DO animals, samples from the same DNA isolation were run on the MUGA in separate batches on different dates. The mean Pearson correlation between the allelic dosages of replicate samples was 0.97 with a standard deviation of 0.05. The mean correlation between unrelated samples was 0.02 with a standard deviation of 0.05. Marginal genotype reconstructions were highly concordantâthe vast majority of SNPs shared the same maximum probability diplotype state between repeated samples (minimum 63%, median 97%, and maximum 99%). Discordant diplotype states occurred most often in regions where one or more of the founder strains are identical by state and as a result, two or more diplotype states have very similar probabilities.</p><p id="p-37">We used marginal haplotype reconstructions to compute the proportion of each founder summed across markers for each DO animal (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2, A and B</a>). The median founder contribution ranged from 10.6% (PWK/PhJ) to 13.2% (C57BL/6J)âconsistent with an expectation of 12.5%. Individual DO animals displayed a wide range of founder contributions with a minimum of 3.4% and a maximum of 25.1%. The proportions of the 36 diplotypes are also consistent with expectationsâeach heterozygous diplotype should occur at a frequency of 1/32 and each homozygous diplotype should occur at a frequency of 1/64. The only notable deviation is an underrepresentation of homozygous PWK/PhJ genotype, which occurs at a median frequency of 0.011%, which is consistent with the lower overall contribution of PWK/PhJ haplotypes. We also computed the contribution of each founder strain at individual SNPs summed across all of the DO mice (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2, C and D</a>). A notable deviation from expected frequencies occurs on chromosome 2 where a previously reported excess of WSB/EiJ founder allele is observed (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Svenson <em>et al.</em> 2012</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Founder allele proportions across samples and markers. Horizontal axis shows the proportion of alleles from each founder across all samples. Boxes show median and interquartile range. Dotted line is 1/8. (B) Diplotype state proportions across all samples. Diplotypes on vertical axis are given in two-letter codes shown in legend. Dotted lines at 1/32 and 1/64. (C) Founder allele proportions across markers. Dashed line is a 1/8. (D) Diplotype state across markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1145617912" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Founder allele proportions across samples and markers. Horizontal axis shows the proportion of alleles from each founder across all samples. Boxes show median and interquartile range. Dotted line is 1/8. (B) Diplotype state proportions across all samples. Diplotypes on vertical axis are given in two-letter codes shown in legend. Dotted lines at 1/32 and 1/64. (C) Founder allele proportions across markers. Dashed line is a 1/8. (D) Diplotype state across markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1623/F2.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/9/1623/F2.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">(A) Founder allele proportions across samples and markers. Horizontal axis shows the proportion of alleles from each founder across all samples. Boxes show median and interquartile range. Dotted line is 1/8. (B) Diplotype state proportions across all samples. Diplotypes on vertical axis are given in two-letter codes shown in legend. Dotted lines at 1/32 and 1/64. (C) Founder allele proportions across markers. Dashed line is a 1/8. (D) Diplotype state across markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The average number of autosomal recombination events in DO mice is expected to increase linearly at a rate of 23.9 events per generation (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Broman 2012</a>). We compared this rate between the intensity-based and genotype-call methods of haplotype reconstruction on each of the MUGA and MegaMUGA genotyping platforms. For MUGA samples spanning generations G4 through G9 we observed a rate of increase of 14.2 events per generation using the intensity-based method compared to 6.9 events per generation for the allele-call method (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). For MegaMUGA samples spanning generations G8 through G11 we observed a rate of increase of 21.3 events per generation using the intensity-based and 17.2 events per generation for the allele-call method (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The intensity-based MegaMUGA data give estimates that are close to the theoretical rate, suggesting that recombination events are accumulating as expected and that the other methods are not capturing all of the events. For both platforms, the intensity-based method captures more recombination events compared to the allele call but the difference is greater for the MUGA platform. The average marker spacing of the MUGA (325 kb) is large relative to the expected minimum spacing of â¼40 kb at generation G8 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pyke 1965</a>), whereas MegaMUGA, with an average marker spacing of 33.2 kb, should capture most of the recombination events in current DO generations. We note that the size distribution of haplotype segments should approximate an exponential distribution. However, histograms of interval sizes on the MUGA array (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>) show a drop in the numbers of small segments relative to their expected distribution at sizes below the mean marker spacing, indicating that small haplotype blocks may be lost in the reconstructions. The closer marker spacing on the MegaMUGA helps to increase the rate of detecting small recombination blocks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Estimated number of autosomal recombinations per sample run on the MUGA plotted by generation using the marker allele calls (&#x25A1;) or the genotyping array intensities (&#x25CB;). Solid lines are the least-squares fit for each method. Dashed line is the expected theoretical number of recombinations. Numbers are the slopes of each line. Boxes show median plus interquartile range. (B) The same plot as in A, but with samples run on the MegaMUGA. Top and bottom numbers are the slope of the intensity- and allele-call-based reconstructions, respectively. (C) Histogram of recombination block size using the MUGA (C) and MegaMUGA (D). The inset shows the region from 0 to 10 Mb. Red curves are the maximum-likelihood fit of the data to an exponential distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1145617912" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Estimated number of autosomal recombinations per sample run on the MUGA plotted by generation using the marker allele calls (&#x25A1;) or the genotyping array intensities (&#x25CB;). Solid lines are the least-squares fit for each method. Dashed line is the expected theoretical number of recombinations. Numbers are the slopes of each line. Boxes show median plus interquartile range. (B) The same plot as in A, but with samples run on the MegaMUGA. Top and bottom numbers are the slope of the intensity- and allele-call-based reconstructions, respectively. (C) Histogram of recombination block size using the MUGA (C) and MegaMUGA (D). The inset shows the region from 0 to 10 Mb. Red curves are the maximum-likelihood fit of the data to an exponential distribution.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1623/F3.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/9/1623/F3.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">(A) Estimated number of autosomal recombinations per sample run on the MUGA plotted by generation using the marker allele calls (â¡) or the genotyping array intensities (â). Solid lines are the least-squares fit for each method. Dashed line is the expected theoretical number of recombinations. Numbers are the slopes of each line. Boxes show median plus interquartile range. (B) The same plot as in A, but with samples run on the MegaMUGA. Top and bottom numbers are the slope of the intensity- and allele-call-based reconstructions, respectively. (C) Histogram of recombination block size using the MUGA (C) and MegaMUGA (D). The inset shows the region from 0 to 10 Mb. Red curves are the maximum-likelihood fit of the data to an exponential distribution.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Simulations</h3><p id="p-41">Null model simulations established that type I error rates are correctly controlled for permutation-based thresholds (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Cheng and Palmer 2013</a>). We simulated 1000 phenotypes by sampling from a multivariate normal distribution with a covariance structure equal to the kinship matrix between samples. We permuted the phenotype values 1000 times and retained the maximum LOD score across the 19 autosomes to obtain the empirical distribution of maximum LOD scores. Quantile plots of the type I error indicate that all three regression models described in <em>Materials and Methods</em> control the type I error at or below the desired threshold (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/genetics.114.161844/-/DC1/161844SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/genetics.114.161844/-/DC1/FigureS1.pdf">Figure S1</a>). In particular, the type I error at <em>Î±</em> = 0.05 is properly controlled. The computational time required to fit a linear mixed-model genome scan repeated for each permuted data sample can be substantial. Thus our ability to obtain valid thresholds with a simple linear model fit has practical importance.</p><p id="p-42">Power simulations provide guidelines for selecting an appropriate sample size for experiments. The power to detect a QTL depends on its effect size, the residual and genetic background variances, and the frequency of the causative allele. These factors are difficult to determine in advance but simulations can help to establish expectations for a study of a given size. We simulated QTL over a wide range of effect sizes with minor allele frequencies that range from 1/8 to 1/2. We fixed the ratio of genetic to residual variation to be oneâequal contributions of eachâand scaled effect sizes relative to the residual standard deviation. We show results for the additive haplotype model (Equation 3) and find, as expected, that increasing MAF, effect size, or the sample size increases power (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/genetics.114.161844/-/DC1/TableS1.pdf">Table S1</a>). The percentaage variance explained provides an effective summary of the combined effects of these factors on power (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). With a sample size of 200 mice, we can expect to detect QTL that explain &gt;20% of the phenotypic variance with 90% power. With a sample of 1000 mice, we can expect to detect a QTL that explain 5% of the variance with 90% power.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power simulations demonstrate the relationship between power, sample size, and percentage variance explained. The percentage variance explained by the simulated QTL is plotted vs. the power to detect the simulated QTL for five different sample sizes. Points are the mean value from 1000 simulations and curves are logistic regression models fit to the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1145617912" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Power simulations demonstrate the relationship between power, sample size, and percentage variance explained. The percentage variance explained by the simulated QTL is plotted &lt;em&gt;vs.&lt;/em&gt; the power to detect the simulated QTL for five different sample sizes. Points are the mean value from 1000 simulations and curves are logistic regression models fit to the data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1623/F4.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/9/1623/F4.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464859" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Power simulations demonstrate the relationship between power, sample size, and percentage variance explained. The percentage variance explained by the simulated QTL is plotted <em>vs.</em> the power to detect the simulated QTL for five different sample sizes. Points are the mean value from 1000 simulations and curves are logistic regression models fit to the data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">We examined power of the full model and of the additive SNP model using simulations with a sample size of 600, a MAF of 2, and an effect size of 0.5 standard deviations from the mean. A QTL with these characteristics will account for 9% of the phenotypic variance. We found the power of the full model to be 0.003 and the type I error at <em>Î±</em> = 0.05 to be 0.046. For the additive haplotype model, power was 0.94 and the type I error was 0.051. For the additive SNP model, power was 0.91 and the type I error was 0.04. These limited simulations indicate that the size of the test is controlled at the nominal genome-wide <em>P</em> â¤ 0.05 level. There is a dramatic loss of power using the full model, whereas, for data simulated under the additive SNP model, the additive haplotype and additive SNP models have similar power. This similarity in power for the additive SNP model represents a best-case scenario because we simulated a biallelic, additive QTL. When allelic heterogeneity is present under the QTL, the power of the additive SNP model may be lower.</p><p id="p-45">Larger sample sizes and larger effect sizes also provide narrower QTL intervals (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/genetics.114.161844/-/DC1/TableS2.pdf">Table S2</a>). Support intervals can range (on average) from 5 Mb for effects that explain 5% of the variance down to &lt;1 Mb for effects that explain 30% of the variance in mapping population of 1000 mice.</p></div><div id="sec-22" class="subsection"><h3>Example: QTL mapping of neutrophil counts</h3><p id="p-46">Here we illustrate the unique features of QTL mapping in the DO with an example: mapping neutrophil counts in whole blood using 742 DO mice (410 female, 332 male). Males [median, 1004; interquartile range,IQR, (698, 1288)] had higher median neutrophil counts than females [median, 729; IQR, (546, 1064))]. The trait has an estimated heritability of 0.132.</p><p id="p-47">We regressed log neutrophil counts on founder allele dosages at each marker using a kinship correction with sex and log white blood cell counts as covariates. We established genome-wide significance thresholds as described in <em>Materials and Methods</em>. We fit the full model (Equation 1) by regressing the log of neutrophil counts on the 36 diplotype probabilities at each marker and found that no LOD peaks exceeded the significance threshold (adjusted p â¤ 0.05) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). We fit the additive haplotype model (Equation 3) and found that one locus on chromosome 1 (LOD, 9.3 at 129.58 Mb GRC Build 38) achieved genome-wide significance and spanned a 5.7-Mb support interval (126.7â132.4 Mb, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). We fit an additive SNP model (Equation 5) by regressing the log of neutrophil counts on the imputed DO genotypes and found one locus with LOD above the p â¤ 0.05 threshold on chromosome 1 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). In the additive SNP model, the peak on chromosome 1 is still the largest peak. The additive haplotype model produces estimates of the eight founder allele effects (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>). The founder allele effects separate into two groups and the peak may be due to a single diallelic polymorphism, which would be consistent with the additive SNP model. In this case, DO mice containing the C57BL/6J, CAST/EiJ, or PWK/PhJ alleles at the QTL on chromosome 1 have lower neutrophil counts than mice containing the other five founder alleles.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping of constitutive neutrophil counts. (A) Genome scan of constitutive neutrophil counts using the full model does not show any significant peaks. The horizontal axis shows the mouse genome. The vertical axis plots the LOD score at each locus. Red line is p &#x2264; 0.05 significance threshold. (B) Linkage mapping of neutrophil counts using the additive haplotype model reveals a large peak on chromosome 1. (C) Association mapping of neutrophil counts using the additive SNP model produces a plot similar to linkage mapping. Vertical axis shows LOD score. Red line is p &#x2264; 0.05 significance threshold. (D) Founder coefficients from the linkage model on chromosome 1 show the effects of each founder allele. The founder coefficients are centered around zero. Bottom shows the LOD score with support interval shaded light blue. (E) Zoomed view of QTL support interval highlights two genes (GRC Build 38 coordinates). Top shows the LOD score for the additive SNP model with the red line denoting the P &#x2264; 0.05 significance threshold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1145617912" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping of constitutive neutrophil counts. (A) Genome scan of constitutive neutrophil counts using the full model does not show any significant peaks. The horizontal axis shows the mouse genome. The vertical axis plots the LOD score at each locus. Red line is p &#x2264; 0.05 significance threshold. (B) Linkage mapping of neutrophil counts using the additive haplotype model reveals a large peak on chromosome 1. (C) Association mapping of neutrophil counts using the additive SNP model produces a plot similar to linkage mapping. Vertical axis shows LOD score. Red line is p &#x2264; 0.05 significance threshold. (D) Founder coefficients from the linkage model on chromosome 1 show the effects of each founder allele. The founder coefficients are centered around zero. Bottom shows the LOD score with support interval shaded light blue. (E) Zoomed view of QTL support interval highlights two genes (GRC Build 38 coordinates). Top shows the LOD score for the additive SNP model with the red line denoting the &lt;em&gt;P&lt;/em&gt; &#x2264; 0.05 significance threshold.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1623/F5.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/9/1623/F5.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1623/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-48" class="first-child">Mapping of constitutive neutrophil counts. (A) Genome scan of constitutive neutrophil counts using the full model does not show any significant peaks. The horizontal axis shows the mouse genome. The vertical axis plots the LOD score at each locus. Red line is p â¤ 0.05 significance threshold. (B) Linkage mapping of neutrophil counts using the additive haplotype model reveals a large peak on chromosome 1. (C) Association mapping of neutrophil counts using the additive SNP model produces a plot similar to linkage mapping. Vertical axis shows LOD score. Red line is p â¤ 0.05 significance threshold. (D) Founder coefficients from the linkage model on chromosome 1 show the effects of each founder allele. The founder coefficients are centered around zero. Bottom shows the LOD score with support interval shaded light blue. (E) Zoomed view of QTL support interval highlights two genes (GRC Build 38 coordinates). Top shows the LOD score for the additive SNP model with the red line denoting the <em>P</em> â¤ 0.05 significance threshold.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">We examined the mapping results from the additive SNP model in the QTL support interval on chromosome 1 and found that two types of SNPs had LOD greater than the p â¤ 0.05 significance threshold (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>). Together these SNPs span an interval from 128.519324 to 128.801978 Mb and overlap two annotated features: <em>Gm25384</em> and chemokine receptor 4 (<em>Cxcr4</em>). <em>Gm25384</em> is a noncoding RNA with no polymorphisms in the DO. <em>Cxcr4</em> has three SNPs in the coding region of its transcript, two synonymous SNPs (no rs#, rs8256193), and one nonsynonymous SNP (rs8256191, 128.589277 Mb, C â T). The minor allele at each of these three SNPs distinguishes strains C57BL/6J, CAST/EiJ, and PWK/PhJ from the remaining founders, consistent with the estimated effects from the additive haplotype model in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>. <em>Cxcr4</em> has two alternative transcripts and the nonsynonymous SNP is in an exon shared by both isoforms. One of the synonymous variants (no rs#) at 128.589098 Mb occurs in one of the isoforms. The other synonymous variant (rs8256193) occurs in both isoforms. In addition, the founder strains C57BL/6J and PWK/PhJ share a segment of common ancestry (130.414213â130.698555 Mb) from the <em>Mus mus musculus</em> progenitor (<a href="http://csbio.unc.edu/ccv/">http://csbio.unc.edu/ccv/</a>) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yang <em>et al.</em> 2011</a>). <em>Cxcr4</em> has been shown to play a role in regulation of neutrophil release from the bone marrow (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eash <em>et al.</em> 2009</a>), making it a strong functional candidate. On the basis of this evidence we suggest that <em>Cxcr4</em>, specifically the valine to isoleucine substitution, is a causal factor contributing to variation in neutrophil levels.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-50">Haplotype reconstruction in progeny from multiparent crosses requires specialized methods and software. These methods have evolved significantly over the past decade and we expect to see a continued increase in the overlap between heterozygous model systems and human genetic mapping. HAPPY, one of the first software packages developed for haplotype reconstruction in heterogeneous stocks, uses genotype calls and a dynamic programming algorithm for haplotype reconstruction (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mott <em>et al.</em> 2000</a>). Our HMM implementation is more complex than dynamic programming but it provides probabilistic estimates of haplotypes (and diplotypes) that we can use to account for genotype uncertainty in genome scans. The current implementation is based on the marginal diplotype state probabilities. Implementation of a globally optimal reconstruction (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Van der Bliek <em>et al.</em> 1988</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Viterbi 1967</a>) is planned for a future release of DOQTL.</p><p id="p-51">We have implemented an EM algorithm to estimate HMM parameters directly from genotype or intensity data. The direct use of probe intensities from genotyping arrays can improve genome reconstructions (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fu 2012</a>). Probes that are affected by off-target variants and might otherwise be removed by genotyping quality filtering can provide additional information about founder haplotypes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Didion <em>et al.</em> 2012</a>). Our emission model for intensities is heavily parameterized as a bivariate normal mixture model over 36 diplotype states at each marker locus. It is likely that performance could be improved by adding constraints to this model, for example, by leveraging haplotype identity among founder strains to reduce the number of states required at each SNP or by modeling heterozygous genotypes in terms of the homozygous state intensities. Our state transition model is not estimated; it is derived from theoretical expectations as a function of the outcrossing generation in the DO. In more complex breeding designs, this may not be feasible and an empirical method to estimate the transition probabilities would be desirable. One approach might be to use a continuous âtimeâ Markov model with transition probabilities p â¤ <em>e<sup>Qt</sup></em>, where <em>Q</em> is a common 36 Ã 36 rate matrix and <em>t</em> is in an interval-specific measure of recombination. As with the emission model, there is a bias <em>vs.</em> variance tradeoff to consider in the specification of these highly parameterized models. Models that capture the main features of the data with the fewest parameters are best.</p><p id="p-52">Our estimates of the number of recombination events per mouse are close to the theoretical expectation when we used the higher density MegaMUGA array and intensity based reconstruction. The MegaMUGA includes &gt;77,000 informative markers with average spacing &lt;33 kb. Still it is clear that smaller haplotype blocks are lost in the reconstructions. The economics of using even higher density arrays platform <em>vs.</em> low coverage whole-genome sequencing is starting to tip. New analytical methods will be required for genotypes derived from sequencing data. The DO is currently at outbreeding generation G17 and the accumulation of recombination events appears to be linear as expected through generation G11. Hotspots of recombination of 1â2 kb in size can have recombination rates as high as 2â3 cM (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Billings <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Parvanov <em>et al.</em> 2010</a>). Recurrent recombination events at hotspots could impose a limit on the ultimate mapping resolution of the DO. In addition, cold regions of &gt;500 kb with strongly suppressed recombination are present in the DO (data not shown) and are largely concordant with recent reports of cold regions in the founder strains (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2014</a>).</p><p id="p-53">Availability of whole-genome sequences of the founder strains adds substantially to the power and utility of multiparent populations (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Baud <em>et al.</em> 2013</a>). With dense genotypes and haplotype reconstructions we can effectively impute the entire genome of each individual and track all of the possible causal variants associated with any phenotype. SNPs can be imputed using either marginal or probabilistic genotypes. The same is true for small indels but larger structural and copy-number variations present some unsolved challenges. Structural variants could potentially be changing rapidly in multiparent populations, providing a unique opportunity to study their short-term evolution.</p><p id="p-54">We considered three different regression models with which to perform mapping in the DO: a full model, an additive haplotype model, and an additive SNP model. When mapping using the full model, we observed loss of power that we attribute to the additional degrees of freedom. The additive haplotype and SNP models can produce similar results. A number of open questions about optimal strategies to model QTL effects on phenotypes in multiparent crosses remain. Our work looks at the extremes of a spectrum of possible models. The challenge is to develop models that are flexible and yet retain sufficient power to detect QTL. Random effects or hierarchical Bayes models look promising in this regard (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhaojun <em>et al.</em> 2014</a>).</p><p id="p-55">One advantage of linkage mapping in multiparent populations such as the DO is that we obtain estimates of the founder allele effects at each locus. The pattern of allele effects and sequencing data from the eight founders were critical in narrowing the neutrophil QTL to a single gene. For some QTL the allele effects split into two clearly delineated groups, <em>e.g.</em>, C57BL/6J, CAST/EiJ, and PWK/PhJ <em>vs.</em> the remaining five founders for the neutrophil QTL on chromosome 1. This is expected if the QTL is caused by a single variant. We have also observed cases where the allele effects split into multiple classes or where the classification of effects into discrete clusters is not clear. These patterns likely result from multiple closely linked causal variants. Allele-effect patterns can help to establish whether comapping of multiple phenotypes is due to the same variant(s) or linked but independent variants (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">King <em>et al.</em> 2012</a>).</p><p id="p-56">DO mice, after multiple generations of outcrossing, will have complex kinship relationships, including siblings, cousins, and extended multigenerational relationships of unknown degree. These relationships must be accounted for in the mapping analysis or increases in false positives may be observed (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cheng <em>et al.</em> 2013</a>). Methods for kinship correction remain an active area of research (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhou and Stephens 2012</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2014</a>).</p><p id="p-57">We recommend using a larger number of animals for a DO mapping study than is typical for mapping studies in the mouse. In our experience loci of major effect have been precisely mapped with &lt;200 animals. It would be prudent to plan on large studies when using the DO to investigate complex traits. The DO provides substantial advantages, as illustrated in the above QTL mapping example.</p><p id="p-58">The DOQTL software implements the methods described in this article from reconstruction of haplotypes through QTL mapping and graphical summaries and reports. With a few modifications, DOQTL can be used to analyze other multiparent populations (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kover <em>et al.</em> 2009</a>) and heterogeneous stock populations (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Baud <em>et al.</em> 2013</a>). We encourage investigators who wish to apply our software to analyze their populations to contact us. Tutorials and additional supporting materials are available at <a href="http://do.jax.org">http://do.jax.org</a>. This site will serve as a repository for DO related data and we encourage groups using the DO to contribute their published data.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">We thank Carol Bult, John E. French, Alison Harrill, Luanne Peters, Peter Turnbaugh, and Peter Vedell for sharing their DO genotype data. We acknowledge the efforts of Kunjie Hua, Liyang Hayworth, Kuo-Chen Jung, and Tangi Smallwood in collecting the University of North Carolina (UNC) DNA tissue/DNA samples. This work was supported in part by National Institutes of Health grants P50 GM076468, P30 GM070683, P30 AG038070, P30 AG038070 and R01 GM070683 (to G.A.C.), R01 GM074244 (to K.W.B.), and RC1DK087346, DK076050, DK056350 (to DP.). D.M.G., K.L.S., L.W., P.S., E.J.C., and G.A.C. are employed by The Jackson Laboratory, which distributes DO mice. Daniel Pomp consults for the Neogen Corporation (GeneSeek is a wholly owned subsidiary of Neogen).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.genetics.org/lookup/suppl/doi:10.1534/genetics.114.161844/-/DC1">http://www.genetics.org/lookup/suppl/doi:10.1534/genetics.114.161844/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-60"><em>Communicating editor: F. van Eeuwijk</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-02-07"><span class="received-label">Received </span>February 7, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-07-28"><span class="accepted-label">Accepted </span>July 28, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2014 Gatti <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.1" data-doi="10.1038/ng.2644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baud</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guryev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stridh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">767</span>â<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D767%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2644%26rft_id%253Dinfo%253Apmid%252F23708188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23708188&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.2" data-doi="10.1214/aoms/1177697196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baum</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, Petrie, T. , Soules, G. , and Weiss, N., <span class="cit-pub-date">1970</span>â<span class="cit-article-title">A maximization technique occuring in the statistical analysis of probabilistic functions of Markov chains.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">164</span>â<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D41%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177697196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177697196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.3" data-doi="10.1371/journal.pone.0015340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Billings</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leahy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwak</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Patterns of recombination activity on mouse chromosome 11 revealed by high resolution mapping.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBillings%26rft.auinit1%253DT.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De15340%26rft.epage%253De15340%26rft.atitle%253DPatterns%2Bof%2Brecombination%2Bactivity%2Bon%2Bmouse%2Bchromosome%2B11%2Brevealed%2Bby%2Bhigh%2Bresolution%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015340%26rft_id%253Dinfo%253Apmid%252F21170346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21170346&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-other" id="cit-4.9.1623.4"><div class="cit-metadata unstructured">Broman, K. W., 2012âHaplotype probabilities in advanced intercross populations. G3 Genes Genomes Genet. <strong>2:</strong> 199â202.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.5" data-doi="10.1534/genetics.106.061176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southard-Smith</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The X chromosome in quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">2151</span>â<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061176%26rft_id%253Dinfo%253Apmid%252F17028340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzIxNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.6" data-doi="10.1371/journal.pcbi.1002822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Chapter 11: genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D8%26rft.spage%253De1002822%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002822%26rft_id%253Dinfo%253Apmid%252F23300413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.7" data-doi="10.1534/genetics.112.146332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">A simulation study of permutation, bootstrap, and gene dropping for assessing statistical significance in the case of unequal relatedness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1015</span>â<span class="cit-lpage">1018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146332%26rft_id%253Dinfo%253Apmid%252F23267053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My8zLzEwMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.8" data-doi="10.1534/genetics.110.116863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokoloff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1033</span>â<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116863%26rft_id%253Dinfo%253Apmid%252F20439773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.9" data-doi="10.1186/1471-2156-12-66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skol</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">QTLRel: an R package for genome-wide association studies in which relatedness is a concern.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DCheng%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DQTLRel%253A%2Ban%2BR%2Bpackage%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bwhich%2Brelatedness%2Bis%2Ba%2Bconcern.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-66%26rft_id%253Dinfo%253Apmid%252F21794153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21794153&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>âPractical considerations regarding the use of genotype and pedigree data to model relatedness in the context of genome-wide association studies. <abbr class="cit-jnl-abbrev">G3</abbr> 3: 1861â1867.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.11" data-doi="10.1007/BF02458837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Stochastic models for heterogeneous DNA sequences.</span> <abbr class="cit-jnl-abbrev">Bull. Math. Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">79</span>â<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DBull%2BMath%2BBiol%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D51%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D94%26rft.atitle%253DStochastic%2Bmodels%2Bfor%2Bheterogeneous%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02458837%26rft_id%253Dinfo%253Apmid%252F2706403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02458837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2706403&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989R957000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>â<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.13" data-doi="10.1534/genetics.107.074609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Naive application of permutation testing leads to inflated type I error rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">609</span>â<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D609%26rft.epage%253D610%26rft.atitle%253DNaive%2BApplication%2Bof%2BPermutation%2BTesting%2BLeads%2Bto%2BInflated%2BType%2BI%2BError%2BRates%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074609%26rft_id%253Dinfo%253Apmid%252F18202402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNjA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.14" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">The Collaborative Cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>â<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-book" id="cit-4.9.1623.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The regress function</span>. <span class="cit-source">R News</span>, <span class="cit-fpage">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.16" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>â<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.17" data-doi="10.1534/genetics.109.105486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackert-Bicknell</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A new standard genetic map for the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1335</span>â<span class="cit-lpage">1344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105486%26rft_id%253Dinfo%253Apmid%252F19535546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.18" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.19" data-doi="10.1182/blood-2008-09-177287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eash</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Means</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Link</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">CXCR4 is a key regulator of neutrophil release from the bone marrow under basal and stress granulopoiesis conditions.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">4711</span>â<span class="cit-lpage">4719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2008-09-177287%26rft_id%253Dinfo%253Apmid%252F19264920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE5LzQ3MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.20" data-doi="10.1038/nrg1576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Strategies for mapping and cloning quantitative trait genes in rodents.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">271</span>â<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D271%26rft.epage%253D286%26rft.atitle%253DStrategies%2Bfor%2Bmapping%2Band%2Bcloning%2Bquantitative%2Btrait%2Bgenes%2Bin%2Brodents.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1576%26rft_id%253Dinfo%253Apmid%252F15803197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15803197&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228093000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-book" id="cit-4.9.1623.21"><div class="cit-metadata unstructured">Fraley, C., A. E. Raferty, T. B. Murphy, and L. Scrucca, 2012â<em>mclust</em>, <em>version 4 for R: Normal Mixture Modeling for Model-Based Clustering</em>, <em>Classification</em>, <em>and Density Estimation</em>. University of Washington, Seattle.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-book" id="cit-4.9.1623.22"><div class="cit-metadata unstructured">Fu, C., C. E. Welsh, F. Pardo-Manuel de Villena, and L. McMillan, 2012âInferring ancestry in admixed populations using microaray probe intensities, pp. in <em>ACM-BCB 2012</em>. Association for Computing Machinery, Orlando, FL.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.23" data-doi="10.1186/gb-2004-5-10-r80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolstad</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dettling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Bioconductor: open software development for computational biology and bioinformatics.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGentleman%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253DR80%26rft.epage%253DR80%26rft.atitle%253DBioconductor%253A%2Bopen%2Bsoftware%2Bdevelopment%2Bfor%2Bcomputational%2Bbiology%2Band%2Bbioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-10-r80%26rft_id%253Dinfo%253Apmid%252F15461798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-10-r80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15461798&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.24" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>â<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hitzemann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dains</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hitzemann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Further studies on the relationship between dopamine cell density and haloperidol-induced catalepsy.</span> <abbr class="cit-jnl-abbrev">J. Pharmacol. Exp. Ther.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">969</span>â<span class="cit-lpage">976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPharmacology%2Band%2BExperimental%2BTherapeutics%26rft.stitle%253DJ.%2BPharmacol.%2BExp.%2BTher.%26rft.aulast%253DHitzemann%26rft.auinit1%253DB.%26rft.volume%253D271%26rft.issue%253D2%26rft.spage%253D969%26rft.epage%253D976%26rft.atitle%253DFurther%2Bstudies%2Bon%2Bthe%2Brelationship%2Bbetween%2Bdopamine%2Bcell%2Bdensity%2Band%2Bhaloperidol-induced%2Bcatalepsy%26rft_id%253Dinfo%253Apmid%252F7965819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoianBldCI7czo1OiJyZXNpZCI7czo5OiIyNzEvMi85NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.26" data-doi="10.1186/1471-2164-11-585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iancu</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darakjian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malmanger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oberbeck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genetic diversity and striatal gene networks: focus on the heterogeneous stock-collaborative cross (HS-CC) mouse.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DIancu%26rft.auinit1%253DO.%2BD.%26rft.volume%253D11%26rft.spage%253D585%26rft.epage%253D585%26rft.atitle%253DGenetic%2Bdiversity%2Band%2Bstriatal%2Bgene%2Bnetworks%253A%2Bfocus%2Bon%2Bthe%2Bheterogeneous%2Bstock-collaborative%2Bcross%2B%2528HS-CC%2529%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-585%26rft_id%253Dinfo%253Apmid%252F20959017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20959017&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.27" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>â<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.28" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>â<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.29" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genetic dissection of a model complex trait using the Drosophila synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>â<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.30" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>â<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.32" data-doi="10.1111/j.1530-0277.1984.tb05708.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lumeng</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Alcohol preference and voluntary alcohol intakes of inbred rat strains and the National Institutes of Health heterogeneous stock of rats.</span> <abbr class="cit-jnl-abbrev">Alcohol. Clin. Exp. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">485</span>â<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlcoholism%252C%2Bclinical%2Band%2Bexperimental%2Bresearch%26rft.stitle%253DAlcohol%2BClin%2BExp%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DT.%2BK.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D485%26rft.epage%253D486%26rft.atitle%253DAlcohol%2Bpreference%2Band%2Bvoluntary%2Balcohol%2Bintakes%2Bof%2Binbred%2Brat%2Bstrains%2Band%2Bthe%2BNational%2BInstitutes%2Bof%2BHealth%2Bheterogeneous%2Bstock%2Bof%2Brats.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1530-0277.1984.tb05708.x%26rft_id%253Dinfo%253Apmid%252F6391261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1530-0277.1984.tb05708.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6391261&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TP25900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.33" data-doi="10.1534/genetics.114.161653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">91</span>â<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161653%26rft_id%253Dinfo%253Apmid%252F24578350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk3LzEvOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.34" data-doi="10.1073/pnas.230304397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turri</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">A method for fine mapping quantitative trait loci in outbred animal stocks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12649</span>â<span class="cit-lpage">12654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.230304397%26rft_id%253Dinfo%253Apmid%252F11050180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjMvMTI2NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.35" data-doi="10.1126/science.1181495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parvanov</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petkov</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paigen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Prdm9 controls activation of mammalian recombination hotspots.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1181495%26rft_id%253Dinfo%253Apmid%252F20044538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyke</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span>â<span class="cit-article-title">Spacings.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">395</span>â<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D27%26rft.spage%253D395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.37" data-doi="10.1109/5.18626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">A tutorial on hidden Markov models and selected applications in speech recognition.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">257</span>â<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D77%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.18626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.18626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989U374600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">A statistical framework for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">371</span>â<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D371%26rft.epage%253D387%26rft.atitle%253DA%2BStatistical%2BFramework%2Bfor%2BQuantitative%2BTrait%2BMapping%26rft_id%253Dinfo%253Apmid%252F11560912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.39" data-doi="10.1093/bioinformatics/bts163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Matrix eQTL: ultra fast eQTL analysis via large matrix operations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1353</span>â<span class="cit-lpage">1358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts163%26rft_id%253Dinfo%253Apmid%252F22492648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTAvMTM1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.40" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">High-resolution genetic mapping using the mouse diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSvenson%26rft.auinit1%253DK.%2BL.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D447%26rft.atitle%253DHigh-Resolution%2BGenetic%2BMapping%2BUsing%2Bthe%2BMouse%2BDiversity%2BOutbred%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Bliek</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Velde-Koerts</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biedler</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">Genes amplified and overexpressed in human multidrug-resistant cell lines.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">5927</span>â<span class="cit-lpage">5932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DVan%2Bder%2BBliek%26rft.auinit1%253DA.%2BM.%26rft.volume%253D48%26rft.issue%253D21%26rft.spage%253D5927%26rft.epage%253D5932%26rft.atitle%253DGenes%2BAmplified%2Band%2BOverexpressed%2Bin%2BHuman%2BMultidrug-resistant%2BCell%2BLines%26rft_id%253Dinfo%253Apmid%252F2901906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI0OC8yMS81OTI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.42" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Heritability in the genomics eraâconcepts and misconceptions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>â<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.43" data-doi="10.1109/TIT.1967.1054010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viterbi</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span>â<span class="cit-article-title">Error bounds for convolutional codes and an asymptotically optimum decoding algorithm.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Inf. Theory</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">260</span>â<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BInf.%2BTheory%26rft.volume%253D13%26rft.spage%253D260%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTIT.1967.1054010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TIT.1967.1054010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19679497100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>âStatus and access to the Collaborative Cross population. <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr> 23: 706â712.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.45" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>â<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.46" data-doi="10.1038/ng.2876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Advantages and pitfalls in the application of mixed-model association methods.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">100</span>â<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2876%26rft_id%253Dinfo%253Apmid%252F24473328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24473328&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhaojun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, 2014â<span class="cit-article-title">Bayesian modeling of haplotype effects in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-fpage">139</span>â<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.9.1623.48" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">821</span>â<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1623.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1611" title="The Genetic Architecture of Maize (Zea mays L.) Kernel Weight Determination" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1635" title="Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CZqcvptqfNSyLK2Ai603VO-9ICh50hRTzFXpPV6eXUY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1623" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1623" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1623" class="active" data-icon-position="" data-hide-link-title="0">Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459655" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-f3LnNFZ9--wUcp5IfCsB-cXSm4OEJymrRJLpvtf8hRQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1623.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1623" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IUcUN36wzFYbqzheJWxT0AIpn-5W-j3bjC9s9Kz0i-U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1623" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1623.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459655" id="node459655--2" data-pisa="ggg;4/9/1623" data-pisa-master="ggg;g3.114.013748" data-apath="/ggg/4/9/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel M. Gatti</span>, <span class="highwire-citation-author" data-delta="1">Karen L. Svenson</span>, <span class="highwire-citation-author" data-delta="2">Andrey Shabalin</span>, <span class="highwire-citation-author" data-delta="3">Long-Yang Wu</span>, <span class="highwire-citation-author" data-delta="4">William Valdar</span>, <span class="highwire-citation-author" data-delta="5">Petr Simecek</span>, <span class="highwire-citation-author" data-delta="6">Neal Goodwin</span>, <span class="highwire-citation-author" data-delta="7">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="8">Daniel Pomp</span>, <span class="highwire-citation-author" data-delta="9">Abraham Palmer</span>, <span class="highwire-citation-author" data-delta="10">Elissa J. Chesler</span>, <span class="highwire-citation-author" data-delta="11">Karl W. Broman</span> and <span class="highwire-citation-author" data-delta="12">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1633;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013748 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459655--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Karen L. Svenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BL.%2BSvenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Svenson%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BL.%2BSvenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrey Shabalin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Medical College of Virginia of Virginia Commonwealth University, Richmond, Virginia 23298</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BShabalin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shabalin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BShabalin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Long-Yang Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Long-Yang%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALong-Yang%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Neal Goodwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neal%2BGoodwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodwin%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeal%2BGoodwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, Australian Capital Territory 0200, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Daniel Pomp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BPomp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pomp%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BPomp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Abraham Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â â </span>Department of Human Genetics, University of Chicago, Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbraham%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡â¡</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1623" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1623&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1623&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1623&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1623&amp;text=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1623&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1623&amp;t=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1623&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1623&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459655/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459655" id="node459655--3" data-pisa="ggg;4/9/1623" data-pisa-master="ggg;g3.114.013748" data-apath="/ggg/4/9/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel M. Gatti</span>, <span class="highwire-citation-author" data-delta="1">Karen L. Svenson</span>, <span class="highwire-citation-author" data-delta="2">Andrey Shabalin</span>, <span class="highwire-citation-author" data-delta="3">Long-Yang Wu</span>, <span class="highwire-citation-author" data-delta="4">William Valdar</span>, <span class="highwire-citation-author" data-delta="5">Petr Simecek</span>, <span class="highwire-citation-author" data-delta="6">Neal Goodwin</span>, <span class="highwire-citation-author" data-delta="7">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="8">Daniel Pomp</span>, <span class="highwire-citation-author" data-delta="9">Abraham Palmer</span>, <span class="highwire-citation-author" data-delta="10">Elissa J. Chesler</span>, <span class="highwire-citation-author" data-delta="11">Karl W. Broman</span> and <span class="highwire-citation-author" data-delta="12">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1633;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013748 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459655--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Karen L. Svenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BL.%2BSvenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Svenson%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BL.%2BSvenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrey Shabalin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Medical College of Virginia of Virginia Commonwealth University, Richmond, Virginia 23298</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BShabalin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shabalin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BShabalin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Long-Yang Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Long-Yang%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALong-Yang%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Neal Goodwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neal%2BGoodwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodwin%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeal%2BGoodwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, Australian Capital Territory 0200, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Daniel Pomp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BPomp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pomp%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BPomp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Abraham Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â â </span>Department of Human Genetics, University of Chicago, Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbraham%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡â¡</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1623&amp;rft.epage=1633&amp;rft.atitle=Quantitative%20Trait%20Locus%20Mapping%20Methods%20for%20Diversity%20Outbred%20Mice&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gatti%2C+Daniel+M.&amp;rft.au=Svenson%2C+Karen+L.&amp;rft.au=Shabalin%2C+Andrey&amp;rft.au=Wu%2C+Long-Yang&amp;rft.au=Valdar%2C+William&amp;rft.au=Simecek%2C+Petr&amp;rft.au=Goodwin%2C+Neal&amp;rft.au=Cheng%2C+Riyan&amp;rft.au=Pomp%2C+Daniel&amp;rft.au=Palmer%2C+Abraham&amp;rft.au=Chesler%2C+Elissa+J.&amp;rft.au=Broman%2C+Karl+W.&amp;rft.au=Churchill%2C+Gary+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459655/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459655/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459655/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459655/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459655/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459655/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459655/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459655/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459655/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459655/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459655/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459655/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1623" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1623.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1623.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1623.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1623.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
