<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1635" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1635.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1635.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1635.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1635.atom" />
<meta name="HW.pisa" content="ggg;4/9/1635" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster" />
<meta name="DC.Identifier" content="10.1534/g3.114.012757" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Wilches et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="There is a growing interest in investigating the relationship between genes with signatures of natural selection and genes identified in QTL mapping studies using combined population and quantitative genetics approaches. We dissected an X-linked interval of 6.2 Mb, which contains two QTL underlying variation in chill coma recovery time (CCRT) in Drosophila melanogaster from temperate (European) and tropical (African) regions. This resulted in two relatively small regions of 131 kb and 124 kb. The latter one co-localizes with a very strong selective sweep in the European population. We examined the genes within and near the sweep region individually using gene expression analysis and P -element insertion lines. Of the genes overlapping with the sweep, none appears to be related to CCRT. However, we have identified a new candidate gene of CCRT, [brinker][1] , which is located just outside the sweep region and is inducible by cold stress. We discuss these results in light of recent population genetics theories on quantitative traits.

 [1]: http://flybase.org/reports/FBgn0024250.html" />
<meta name="DC.Contributor" content="Ricardo Wilches" />
<meta name="DC.Contributor" content="Susanne Voigt" />
<meta name="DC.Contributor" content="Pablo Duchen" />
<meta name="DC.Contributor" content="Stefan Laurent" />
<meta name="DC.Contributor" content="Wolfgang Stephan" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;There is a growing interest in investigating the relationship between genes with signatures of natural selection and genes identified in QTL mapping studies using combined population and quantitative genetics approaches. We dissected an X-linked interval of 6.2 Mb, which contains two QTL underlying variation in chill coma recovery time (CCRT) in &lt;i&gt;Drosophila melanogaster&lt;/i&gt; from temperate (European) and tropical (African) regions. This resulted in two relatively small regions of 131 kb and 124 kb. The latter one co-localizes with a very strong selective sweep in the European population. We examined the genes within and near the sweep region individually using gene expression analysis and &lt;i&gt;P&lt;/i&gt;-element insertion lines. Of the genes overlapping with the sweep, none appears to be related to CCRT. However, we have identified a new candidate gene of CCRT, &lt;i&gt;brinker&lt;/i&gt;, which is located just outside the sweep region and is inducible by cold stress. We discuss these results in light of recent population genetics theories on quantitative traits.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1635" />
<meta name="citation_id" content="4/9/1635" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1635" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1635.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1635.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1635.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012757" />
<meta name="citation_pmid" content="24970882" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1635" />
<meta name="citation_lastpage" content="1645" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ricardo Wilches" />
<meta name="citation_author_institution" content="Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany" />
<meta name="citation_author_email" content="wilches@bio.lmu.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0002-7862-1507" />
<meta name="citation_author" content="Susanne Voigt" />
<meta name="citation_author_institution" content="Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany" />
<meta name="citation_author" content="Pablo Duchen" />
<meta name="citation_author_institution" content="Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany" />
<meta name="citation_author" content="Stefan Laurent" />
<meta name="citation_author_institution" content="Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany" />
<meta name="citation_author" content="Wolfgang Stephan" />
<meta name="citation_author_institution" content="Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Axelsson;citation_author=A. Ratnakumar;citation_author=M.-L. Arendt;citation_author=K. Maqbool;citation_author=M. T. Webster;citation_title=The genomic signature of dog domestication reveals adaptation to a starch-rich diet.;citation_pages=360-364;citation_volume=495;citation_year=2013;citation_pmid=23354050;citation_doi=10.1038/nature11837" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=N. H. Barton;citation_author=P. D. Keightley;citation_title=Understanding quantitative genetic variation.;citation_pages=11-21;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823787;citation_doi=10.1038/nrg700" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Boitard;citation_author=C. Schlötterer;citation_author=V. Nolte;citation_author=R. V. Pandey;citation_author=A. Futschik;citation_title=Detecting selective sweeps from pooled next-generation sequencing samples.;citation_volume=29;citation_year=2012;citation_pmid=22411855;citation_doi=10.1093/molbev/mss090" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. M. Cano;citation_author=C. Matsuba;citation_author=H. Mäkinen;citation_author=J. Merilä;citation_title=The utility of QTL-Linked markers to detect selective sweeps in natural populations--a case study of the EDA gene and a linked marker in threespine stickleback.;citation_pages=4613-4621;citation_volume=15;citation_year=2006;citation_issue=14;citation_pmid=17107487;citation_doi=10.1111/j.1365-294X.2006.03099.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L.-M. Chevin;citation_author=F. Hospital;citation_title=Selective sweep at a quantitative trait locus in the presence of background genetic variation.;citation_volume=180;citation_year=2008;citation_pmid=18832353;citation_doi=10.1534/genetics.108.093351" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. De Luca;citation_author=N. V. Roshina;citation_author=G. L. Geiger-Thornsberry;citation_author=R. F. Lyman;citation_author=E. G. Pasyukova;citation_title=Dopa decarboxylase (Ddc) affects variation in Drosophila longevity.;citation_pages=429-433;citation_volume=34;citation_year=2003;citation_issue=4;citation_pmid=12881721;citation_doi=10.1038/ng1218" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Duchen;citation_author=D. Živković;citation_author=S. Hutter;citation_author=W. Stephan;citation_author=S. Laurent;citation_title=Demographic inference reveals African and European admixture in the North American Drosophila melanogaster population.;citation_volume=193;citation_year=2013;citation_pmid=23150605;citation_doi=10.1534/genetics.112.145912" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Excoffier;citation_author=M. Foll;citation_title=fastsimcoal: a continuous-time coalescent simulator of genomic diversity under arbitrarily complex evolutionary scenarios.;citation_volume=27;citation_year=2011;citation_pmid=21398675;citation_doi=10.1093/bioinformatics/btr124" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. J. Fanara;citation_author=K. O. Robinson;citation_author=S. M. Rollmann;citation_author=R. R. H. Anholt;citation_author=T. F. C. Mackay;citation_title=Vanaso Is a Candidate Quantitative Trait Gene for Drosophila Olfactory Behavior;citation_pages=1321-1328;citation_volume=162;citation_year=2002;citation_issue=3;citation_pmid=12454076" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Foll;citation_author=O. Gaggiotti;citation_title=A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective.;citation_volume=180;citation_year=2008;citation_pmid=18780740;citation_doi=10.1534/genetics.108.092221" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of ageing and development;citation_journal_abbrev=Mech Ageing Dev;citation_author=G. L. Geiger-Thornsberry;citation_author=T. F. C. Mackay;citation_title=Quantitative trait loci affecting natural variation in Drosophila longevity.;citation_pages=179-189;citation_volume=125;citation_year=2004;citation_issue=3;citation_pmid=15013662;citation_doi=10.1016/j.mad.2003.12.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Glinka;citation_author=L. Ometto;citation_author=S. Mousset;citation_author=W. Stephan;citation_author=D. De Lorenzo;citation_title=Demography and Natural Selection Have Shaped Genetic Variation in Drosophila melanogaster: A Multi-locus Approach;citation_pages=1269-1278;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668381" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Glinka;citation_author=D. De Lorenzo;citation_author=W. Stephan;citation_title=Evidence of gene conversion associated with a selective sweep in Drosophila melanogaster.;citation_volume=23;citation_year=2006;citation_pmid=16868022;citation_doi=10.1093/molbev/msl069" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster.;citation_pages=473-479;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179090;citation_doi=10.1038/nature09715" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=A. M. Hancock;citation_author=G. Alkorta-Aranburu;citation_author=D. B. Witonsky;citation_author=A. Di Rienzo;citation_title=Adaptations to new environments in humans: the role of subtle allele frequency shifts;citation_pages=2459-2468;citation_volume=365;citation_year=2010;citation_issue=1552;citation_pmid=20643735;citation_doi=10.1098/rstb.2010.0032" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. T. Harbison;citation_author=A. H. Yamamoto;citation_author=J. J. Fanara;citation_author=K. K. Norga;citation_author=T. F. C. Mackay;citation_title=Quantitative Trait Loci Affecting Starvation Resistance in Drosophila melanogaster;citation_pages=1807-1823;citation_volume=166;citation_year=2004;citation_issue=4;citation_pmid=15126400;citation_doi=10.1534/genetics.166.4.1807" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Hellemans;citation_author=G. Mortier;citation_author=A. De Paepe;citation_author=F. Speleman;citation_author=J. Vandesompele;citation_title=qBase relative quantification framework and software for management and automated analysis of real-time quantitative PCR data.;citation_pages=R19-R19;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17291332;citation_doi=10.1186/gb-2007-8-2-r19" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=N. Herold;citation_author=C. L. Will;citation_author=E. Wolf;citation_author=B. Kastner;citation_author=H. Urlaub;citation_title=Conservation of the protein composition and electron microscopy structure of Drosophila melanogaster and human spliceosomal complexes.;citation_volume=29;citation_year=2009;citation_pmid=18981222;citation_doi=10.1128/MCB.01415-08" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Lett.;citation_author=A. A. Hoffmann;citation_author=A. Anderson;citation_author=R. Hallas;citation_title=Opposing clines for high and low temperature resistance in Drosophila melanogaster.;citation_pages=614-618;citation_volume=5;citation_year=2002;citation_doi=10.1046/j.1461-0248.2002.00367.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. T. Hu;citation_author=M. B. Eisen;citation_author=K. R. Thornton;citation_author=P. Andolfatto;citation_title=A second-generation assembly of the Drosophila simulans genome provides new insights into patterns of lineage-specific divergence.;citation_volume=23;citation_year=2013;citation_pmid=22936249;citation_doi=10.1101/gr.141689.112" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Hutter;citation_author=S. Saminadin-Peter;citation_author=W. Stephan;citation_author=J. Parsch;citation_title=Gene expression variation in African and European populations of Drosophila melanogaster.;citation_pages=R12-R12;citation_volume=9;citation_year=2008;citation_issue=1;citation_pmid=18208589;citation_doi=10.1186/gb-2008-9-1-r12" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Innan;citation_author=W. Stephan;citation_title=Selection Intensity Against Deleterious Mutations in RNA Secondary Structures and Rate of Compensatory Nucleotide Substitutions;citation_pages=389-399;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560913" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. C. Kane;citation_author=L. H. Rieseberg;citation_title=Selective sweeps reveal candidate genes for adaptation to drought and salt tolerance in common sunflower, Helianthus annuus.;citation_volume=175;citation_year=2007;citation_pmid=17237516;citation_doi=10.1534/genetics.106.067728" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=L. A. Kelley;citation_author=M. J. E. Sternberg;citation_title=Protein structure prediction on the Web: a case study using the Phyre server.;citation_pages=363-371;citation_volume=4;citation_year=2009;citation_issue=3;citation_pmid=19247286;citation_doi=10.1038/nprot.2009.2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Kim;citation_author=W. Stephan;citation_title=Detecting a Local Signature of Genetic Hitchhiking Along a Recombining Chromosome;citation_pages=765-777;citation_volume=160;citation_year=2002;citation_issue=2;citation_pmid=11861577" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=M. Kimura;citation_title=The role of compensatory neutral mutations in molecular evolution.;citation_pages=7-19;citation_volume=64;citation_year=1985;citation_doi=10.1007/BF02923549" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. Lande;citation_title=The response to selection on major and minor mutations affecting a metrical trait.;citation_pages=47-65;citation_volume=50;citation_year=1983;citation_doi=10.1038/hdy.1983.6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=K. Stevens;citation_author=C. Cardeno;citation_author=Y. C. G. Lee;citation_author=D. R. Schrider;citation_title=Genomic variation in natural populations of Drosophila melanogaster.;citation_volume=192;citation_year=2012;citation_pmid=22673804;citation_doi=10.1534/genetics.112.142018" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. J. Y. Laurent;citation_author=A. Werzner;citation_author=L. Excoffier;citation_author=W. Stephan;citation_title=Approximate Bayesian analysis of Drosophila melanogaster polymorphism data reveals a recent colonization of Southeast Asia.;citation_volume=28;citation_year=2011;citation_pmid=21300986;citation_doi=10.1093/molbev/msr031" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=P. O. Lewis;citation_title=A likelihood approach to estimating phylogeny from discrete morphological character data.;citation_volume=50;citation_year=2001;citation_pmid=12116640;citation_doi=10.1080/106351501753462876" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Li;citation_author=W. Stephan;citation_title=Inferring the demographic history and rate of adaptive substitution in Drosophila.;citation_pages=e166-e166;citation_volume=2;citation_year=2006;citation_issue=10;citation_pmid=17040129;citation_doi=10.1371/journal.pgen.0020166" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. R. Linnen;citation_author=Y.-P. Poh;citation_author=B. K. Peterson;citation_author=R. D. H. Barrett;citation_author=J. G. Larson;citation_title=Adaptive Evolution of Multiple Traits Through Multiple Mutations at a Single Gene;citation_pages=1312-1316;citation_volume=339;citation_year=2013;citation_issue=6125;citation_pmid=23493712;citation_doi=10.1126/science.1233213" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=T. J. Morgan;citation_author=T. F. C. Mackay;citation_title=Quantitative trait loci for thermotolerance phenotypes in Drosophila melanogaster.;citation_pages=232-242;citation_volume=96;citation_year=2006;citation_issue=3;citation_pmid=16404413;citation_doi=10.1038/sj.hdy.6800786" />
<meta name="citation_reference" content="citation_title=Molecular Evolutionary Genetics;citation_author=M. Nei;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Nei;citation_author=W. H. Li;citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases.;citation_pages=5269-5273;citation_volume=76;citation_year=1979;citation_issue=10;citation_pmid=291943;citation_doi=10.1073/pnas.76.10.5269" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Nielsen;citation_author=S. Williamson;citation_author=Y. Kim;citation_author=M. J. Hubisz;citation_author=A. G. Clark;citation_title=Genomic scans for selective sweeps using SNP data;citation_pages=1566-1575;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251466;citation_doi=10.1101/gr.4252305" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. M. Norry;citation_author=A. C. Scannapieco;citation_author=P. Sambucetti;citation_author=C. I. Bertoli;citation_author=V. Loeschcke;citation_title=QTL for the thermotolerance effect of heat hardening, knockdown resistance to heat and chill-coma recovery in an intercontinental set of recombinant inbred lines of Drosophila melanogaster.;citation_pages=4570-4581;citation_volume=17;citation_year=2008;citation_issue=20;citation_pmid=18986501;citation_doi=10.1111/j.1365-294X.2008.03945.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. G. Pasyukova;citation_author=C. Vieira;citation_author=T. F. C. Mackay;citation_title=Deficiency Mapping of Quantitative Trait Loci Affecting Longevity in Drosophila melanogaster;citation_pages=1129-1146;citation_volume=156;citation_year=2000;citation_issue=3;citation_pmid=11063689" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Pavlidis;citation_author=S. Hutter;citation_author=W. Stephan;citation_title=A population genomic approach to map recent positive selection in model species.;citation_pages=3585-3598;citation_volume=17;citation_year=2008;citation_issue=16;citation_pmid=18627454" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pavlidis;citation_author=J. D. Jensen;citation_author=W. Stephan;citation_title=Searching for footprints of positive selection in whole-genome SNP data from nonequilibrium populations.;citation_volume=185;citation_year=2010;citation_pmid=20407129;citation_doi=10.1534/genetics.110.116459" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pavlidis;citation_author=D. Metzler;citation_author=W. Stephan;citation_title=Selective sweeps in multilocus models of quantitative traits.;citation_volume=192;citation_year=2012;citation_pmid=22714406;citation_doi=10.1534/genetics.112.142547" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Pavlidis;citation_author=D. Živković;citation_author=A. Stamatakis;citation_author=N. Alachiotis;citation_title=SweeD: likelihood-based detection of selective sweeps in thousands of genomes.;citation_volume=30;citation_year=2013;citation_pmid=23777627;citation_doi=10.1093/molbev/mst112" />
<meta name="citation_reference" content="citation_journal_title=European Journal of Chemical Biology;citation_author=D. Perl;citation_author=F. X. Schmid;citation_title=Some like it hot: the molecular determinants of protein thermostability.;citation_pages=39-44;citation_volume=3;citation_year=2002;citation_pmid=17590951" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. E. Pool;citation_author=C. F. Aquadro;citation_title=The genetic basis of adaptive pigmentation variation in Drosophila melanogaster.;citation_pages=2844-2851;citation_volume=16;citation_year=2007;citation_issue=14;citation_pmid=17614900;citation_doi=10.1111/j.1365-294X.2007.03324.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population genomics of Sub-Saharan Drosophila melanogaster: African diversity and non-African admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. K. Pritchard;citation_author=A. Di Rienzo;citation_title=Adaptation - not by sweeps alone.;citation_pages=665-667;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20838407;citation_doi=10.1038/nrg2880" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Qanbari;citation_author=H. Pausch;citation_author=S. Jansen;citation_author=M. Somel;citation_author=T. M. Strom;citation_title=Classic selective sweeps revealed by massive sequencing in cattle.;citation_pages=e1004148;citation_volume=10;citation_year=2014;citation_pmid=24586189;citation_doi=10.1371/journal.pgen.1004148" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Rebeiz;citation_author=J. E. Pool;citation_author=V. A. Kassner;citation_author=C. F. Aquadro;citation_author=S. B. Carroll;citation_title=Stepwise Modification of a Modular Enhancer Underlies Adaptation in a Drosophila Population;citation_pages=1663-1667;citation_volume=326;citation_year=2009;citation_issue=5960;citation_pmid=20019281;citation_doi=10.1126/science.1178357" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Riebler;citation_author=L. Held;citation_author=W. Stephan;citation_title=Bayesian variable selection for detecting adaptive genomic differences among populations.;citation_volume=178;citation_year=2008;citation_pmid=18245358;citation_doi=10.1534/genetics.107.081281" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C.-J. Rubin;citation_author=M. C. Zody;citation_author=J. Eriksson;citation_author=J. R. S. Meadows;citation_author=E. Sherwood;citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication.;citation_pages=587-591;citation_volume=464;citation_year=2010;citation_issue=7288;citation_pmid=20220755;citation_doi=10.1038/nature08832" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C.-J. Rubin;citation_author=H.-J. Megens;citation_author=A. Martinez Barrio;citation_author=K. Maqbool;citation_author=S. Sayyab;citation_title=Strong signatures of selection in the domestic pig genome.;citation_volume=109;citation_year=2012;citation_pmid=23151514;citation_doi=10.1073/pnas.1217149109" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=P. S. Schmidt;citation_author=L. Matzkin;citation_author=M. Ippolito;citation_author=W. F. Eanes;citation_title=Geographic variation in diapause incidence, life-history traits, and climatic adaptation in Drosophila melanogaster.;citation_pages=1721-1732;citation_volume=59;citation_year=2005;citation_issue=8;citation_pmid=16331839;citation_doi=10.1554/05-115.1" />
<meta name="citation_reference" content="citation_journal_title=Science's SAGE KE;citation_journal_abbrev=SAGE KE;citation_author=P. M. Service;citation_title=How Good Are Quantitative Complementation Tests?;citation_pages=pe13-pe13;citation_volume=2004;citation_year=2004;citation_issue=12;citation_pmid=15044758;citation_doi=10.1126/sageke.2004.12.pe13" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=W. Stephan;citation_title=Genetic hitchhiking versus background selection: the controversy and its implications;citation_pages=1245-1253;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308100;citation_doi=10.1098/rstb.2009.0278" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=W. Stephan;citation_author=H. Li;citation_title=The recent demographic and adaptive history of Drosophila melanogaster.;citation_pages=65-68;citation_volume=98;citation_year=2007;citation_issue=2;citation_pmid=17006533;citation_doi=10.1038/sj.hdy.6800901" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=N. Svetec;citation_author=A. Werzner;citation_author=R. Wilches;citation_author=P. Pavlidis;citation_author=J. M. Álvarez-Castro;citation_title=Identification of X-linked quantitative trait loci affecting cold tolerance in Drosophila melanogaster and fine mapping by selective sweep analysis.;citation_pages=530-544;citation_volume=20;citation_year=2011;citation_issue=3;citation_pmid=21199023;citation_doi=10.1111/j.1365-294X.2010.04951.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=K. R. Thornton;citation_author=J. D. Jensen;citation_author=C. Becquet;citation_author=P. Andolfatto;citation_title=Progress and prospects in mapping recent selection in the genome.;citation_pages=340-348;citation_volume=98;citation_year=2007;citation_issue=6;citation_pmid=17473869" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="There is a growing interest in investigating the relationship between genes with signatures of natural selection and genes identified in QTL mapping studies using combined population and quantitative genetics approaches. We dissected an X-linked interval of 6.2 Mb, which contains two QTL underlying variation in chill coma recovery time (CCRT) in Drosophila melanogaster from temperate (European) and tropical (African) regions. This resulted in two relatively small regions of 131 kb and 124 kb. The latter one co-localizes with a very strong selective sweep in the European population. We examined the genes within and near the sweep region individually using gene expression analysis and P -element insertion lines. Of the genes overlapping with the sweep, none appears to be related to CCRT. However, we have identified a new candidate gene of CCRT, [brinker][1] , which is located just outside the sweep region and is inducible by cold stress. We discuss these results in light of recent population genetics theories on quantitative traits.

 [1]: http://flybase.org/reports/FBgn0024250.html" />
<meta name="og-title" property="og:title" content="Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1635" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="There is a growing interest in investigating the relationship between genes with signatures of natural selection and genes identified in QTL mapping studies using combined population and quantitative genetics approaches. We dissected an X-linked interval of 6.2 Mb, which contains two QTL underlying variation in chill coma recovery time (CCRT) in Drosophila melanogaster from temperate (European) and tropical (African) regions. This resulted in two relatively small regions of 131 kb and 124 kb. The latter one co-localizes with a very strong selective sweep in the European population. We examined the genes within and near the sweep region individually using gene expression analysis and P -element insertion lines. Of the genes overlapping with the sweep, none appears to be related to CCRT. However, we have identified a new candidate gene of CCRT, [brinker][1] , which is located just outside the sweep region and is inducible by cold stress. We discuss these results in light of recent population genetics theories on quantitative traits.

 [1]: http://flybase.org/reports/FBgn0024250.html" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Z6PUbTBaGjbovtg2P9azCnuikSFKRfSfInqeN08AfXI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459657","apath":"\/ggg\/4\/9\/1635.atom","pisa":"ggg;4\/9\/1635","ac":{"\/ggg\/4\/9\/1635.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1635.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1635"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1635"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1635":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227vdJR4SfShMpfBQxr9i\\\\\\\/VHJIuoT9\\\\\\\/K8YORNeIxIm1he7UOxVJDWUMQhL0cQ4y10THjmIpFObL+VZKkHf9k7J8eLo6gVPkBhC4LirIzJDkGp5u7iLg+qGBM\\\\\\\/rPCIeLKXNBF705AgvDXI0YnjYTrNKDwduvd1sWKtCeqdwy3HmTDq1V5rAzDUc2vf9aObN7ClQ7Ai3wetx+1edX08ROSzPAgpiT5b\\\\\\\/5HzNuo8xk3YlZQo1KYaNZDdnFbVneceohs\\\\\\\/81tiqRhE4aRZIKQPKC1UbKvA4zdw4rP0OHT\\\\\\\/QwGHHJJuHwzw\\\\\\\/f9D+Z3VupJ0ADogzUA5\\\\\\\/TXF\\\\\\\/8+yjEXTYHBRp2Vra33\\\\\\\/jPgku0hXvpm+2adfFn9ryAq5ohtkwcGYD6oK1H10JLJc71cAIOQFUl7CPwdlvVFUBZIMLJBNOadHGhZ\\\\\\\/BCOPbMVOSFBITg9zyr04I8mJRhIdf1hnDvyt\\\\\\\/Qt4DH1RIURIHyXPwljsPXMOH0WPWmcYEjjHTPAgA8VJYVuniwmJef28VxLXygA4JBJNVz+UX64QqpNv9gRCprS0pJmnUWxueYzSFw67pb+Zxn83dLjuIbj9ODgC0ikg1C5YoLPLkxRHLOBC8g8jQL6ONbWClhPhmjjzgv5lXDUxKB7NjJbZ0eUQbpBo+Re2GJHY\\\\\\\/JybWHzAVKMlaEjKrVfDbofNbOavz2JPU1r377IXNxqAH3Z+R9N3U8bg7OukZIMoaFx64PoGBKC+pWiOL3M7UrsEg3qdzkDO8Rtr+XFLx1Mkvx0aBioEwlmkEVoACa1EPI1vz8w4k1X\\\\\\\/Q\\\\\\\/Iec5M1R5Lb4hlc3sNRSnY\\\\\\\/5+Aq8mAA3Eix3AjbkUwyrDLTeZkrmfRehotKYG2XSRi6uqcDhFGCyzbepd39Rn0AzXF0UO3IBr5U5MWMJOP2aIlQjOT0s2+vsdDR9iy7PzKmytqpLz5+\\\\\\\/NyGRG988OlYW9aypt4K+kQwUzA9nZ4aT7mKEYTK7DhEdhh8r2tvdiFNvMXRDMbodAOKdEqZhitFisdWn3CupwY\\\\\\\/mrJxAYykCkqHMhQmkFsYHerVJBPfcZt4qppZ0Is1yiRYj0XkDvV\\\\\\\/PBpMt+NK8WzBbfjiEAZhOLw0mDMUIQ1W\\\\\\\/3gS4I9GnZen4Sry81d+7+98Yrkbh5tNOjKyWwVlmfuB7lt\\\\\\\/ZqB3LBykdCXsaPZzyracBe2r4pZR1UHwwyU3czaPJ\\\\\\\/1g4EEGyoMqRqVKeRx7xtXkU2r6wBODafJ2EaBN8jCZl7O90prgJOt0blszz6taZM+ECrIChRDsYHUNE\\\u0022,\\\u0022iv\\\u0022:\\\u0022MdTkJsTlWQG24Kvk6VOfEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022358fb39a979b92d14c067d5cb1c3597f974de0bfda5ac65c77e81673f4620117\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AL32cn\\\\\\\/aiS8IHigKxtxwEwNN997CiN0f3ZPBnYP+68fdVqzlimZEu4FnlHz80IxICPmRhnnzP9bhswJjNH7mBzq5R6FPf7SH7clKclD2gBNZpbwWDSR+nSFSCYBSsyqPTbhYyQoxD2G3eEKLq47PAaWZmu+NMx7qKfeAz21Vk9Ky96kCa66xVHTHH9\\\\\\\/5DIZGd9SG2ug8sSP3BMigyE\\\\\\\/ZM+ucDNa5YdEqq3liuDCcqfEy4vxP3VNT9a4HCZESbHyGYRFrmbVqHcCBWju+yaxbAKYh0lygc728YP+h0e\\\\\\\/Y7kpvVdQtEvXRsA+lODVHzmXfeREM4OWI3FYgHMfnCcvWghH4Bpq4hhFu6abK0hDgldZFvIt+2\\\\\\\/r\\\\\\\/H1cQfT0Z5b01TDM2h1c6NoxbDHlJlPj+rUGpjfktySUgB1AKDMmdZI\\\\\\\/mCyd64Ysa24BHe5E8v\\\\\\\/sLQb9qgJJRNg1OCk\\\\\\\/PB\\\\\\\/v4XgZza2of18Udim\\\\\\\/8H1J82FEmUP2mtsJmbX6fZnEL3356K9dCdRK\\\\\\\/Jduoi0VKUzvA6MINAk0mrT7OWDBjrETXMpYiSTmg1gPlTv\\\\\\\/Zftac3T4cdw49WPeMRIhQtXKmF43ycPEAn9jASjckV5y8mzpg9skQ5bPl\\\\\\\/LN2tF1tExTIqSAwcpbHILIBPIfECTUURDlaTfarrFv4XWD3t9hrno6eJYhZMTj1\\\\\\\/ITassw4YTcAKPpK5oNndriVCDgCLhXQkcjsJkQmDaq0PFoItrWQlA8wQVvJ78ikx7uHqiAMi9avVntcImbQfeTlpm0tANasoqkN87drEDzhmfXKl1VJ5PTc+MZDK7+QmhqAvGm5NQ78Cnlt7w1kk4\\\\\\\/b8WjTrR7sBPh29T0CMFkqLPJ4kTnamYAeQ\\\\\\\/PovvzrloUOF5S5giKpn4acjUbiu6xD8MptCLTlpHkXfDGfA8dmLoW4kW9gdg7LqvGT+IIU17pUjpCXwr5+otUYy6Al5nLIv+uf586Ef7zw+16FALFA4vQ2wxE5R99j02DSgcj87oarO6XYlZNIq4w7uct\\\\\\\/9jKidTnVjD0YiRmbeyqiekt8nDpV6SJPqdSFyxh8dcMIHWj1\\\\\\\/gDlZ2NN3p5FwcEqMrSjyLChPueqxdKx30IWMqmfV7QK18DbKLCK\\\\\\\/U4qnd4BIISr4eflNCaDXAuWX\\\\\\\/FB+xVy\\\\\\\/Qioxvx3HGjB7v4pb5sQ1eJAfLiYFdwL1Hz80DR8btr7E3FW3VL3+cLCEqxcxHMasTChKJ9E6fhQNrZiH079w7w2m\\\\\\\/CLZmHPqhDqs5sp3enp8mMQ9HqBZePoyqAl8UvkAiP+nOWI56uu0KGYihqxAm1YQo1oy6laFttSuJvsI4cnpxUDHKoIZwK2N+HpFYaUnJzoK\\\\\\\/pcAHGOt\\\\\\\/pDxDWXt7HOabRQfBLC8WUrZcTtwyAzR+tcENAoE4a5ZMeOCG6j409Xzd4iwH1mzeSuuNy8+s+VjKLEik0XNamnsakdJ8tR8PbIFQenE1gQN4zhtoGg6MY79W\\\\\\\/5D0NTShTZNbE8bm9egPjsrGwvFH4I95HjAwSMHdhHICOzTFaSPS2FyCSrxh64\\\\\\\/kYTFJLGnCk5oGVfiNb\\\\\\\/3tBQ+XMsEOb1UF+F\\\\\\\/lonqLrDqfoxG7BMGBgELgxCIw0SVzE07DSqhR8cUTsDjowiDfmpaCeMFTBXCId6IsHQvKU4Djz+o8TWzFSIcBq\\\\\\\/QNswFAjbY7vRyA0jQLnUpSJN5mOIazKeS3iRPpJAzsH0\\\\\\\/Suvu3KAzDQR+anV47ThBZIbhCPtzCbQKElYRVRs5GWywPI6HIRhzmbP6\\\\\\\/8l4CrE3q5cYWHMwQNRq8xH0ToBPdFvwOEPYkiUiDH1CbgRrGOo00SXT2iKWFjIflL6st7\\\\\\\/PYMk9G4fN\\\\\\\/UVW3axckwEr\\\\\\\/TaeOZSmJD\\\\\\\/SNGzMQwrPShpF7jvHdT7MrXTSvJ9U1go+jmxR4GvDSD37npe\\\\\\\/qUcswBG5iJEjau+MvdCPGWqg57dsSTVnPECI4MWV73MRicSpcjPFzuCzsSAQ8cVJcA++17DN+TfHRDzuhd2aOJ0WRx0P2RnFQ48RTj98yeVboMmS2VaHUZSQL5wCftu6LuAiXf6CjEjK\\\\\\\/fmMmXhYtVKYxHAXAiXKLaJEFVrYLn6bK0pU0tCLvM40k41dnUe8eweJo8ZBVQvSBULjGcz3EeDn3StcCU9dXqu2ErHYrw0PwcwRoOB8DthUBWAcgOPGT0o3CX1a1w671+f0sKomnMK72nIDBfZRTtITDIxDZnmVzJJRQFWGOcbMLgM1FjbUlcaAtKBr6tEO7eX+4ONdhF+VAe0v\\\\\\\/wN6cNfaxrQbhSNXF552NDlIQLGp6xPsV6l760Inzq+B4WKZFkIji7OHAu2RtI5mQliAmw1zIGX77Uw6sctFgH7dOY4ri4F91kTkAifdi4c7cXcfB2QLws9jtAX971AcCJxTF87HBo70IWGoNbLz\\\\\\\/gdGfqrRSWCB80KkPZiAoHBiLZobjW1mDmDWQu263h2vF45tpBygRbqMfRwLG\\\\\\\/JG9RvjSGaId4FDDRHPpEwD2SU7RmlMasPtpSqd5j0ycmCKbueGOgn7m7bEFF0CCPYXGbzeBV9ImyisiQ0eyy4KzUSTdnAuSXbTMg\\\\\\\/bTmUoOoN+EYg8Ecs4Ad18jGPur50nogDUm9qzGNTBHLISnmIuX41I4sNGlY3ru3XBe5BwXi4maLRVZ2JTc4lIvFrs8gphyiegNgvA56yDrOCA2jelb6ZQVjoVu76h8J\\\\\\\/vP5lKFlfPJdonHVP73g+KoN2uwUBK8VbnUueoKWZnIQRse5UDhoIJR+NrbKqCMYghzrC5U5C6gXmbJPI2J1u9Bf0qNEb\\\\\\\/OFpNGFlec2txea+N7EVFa2Ksb42UGJ030qVp4QAErOvyFzT0durhGG+jkMfmMybaeqQXA6JiyOpyl\\\\\\\/lCshkPnapfynenDHzSyjVI8L\\\\\\\/8hq5bTHY1A6WXrjJYbDxjfuPmHMuvKqylEegCUMrLsWZ\\\\\\\/9jsGn4u2VRElh5QwNzWXD\\\\\\\/eGAscqIaQfRRgHhPax\\\\\\\/+eUnhNt\\\\\\\/VPC3GMGVujILQhH0ZnrU6EkquZ+g1bMDbccaubGdGWYVpT49ciSboSZZxjKer8OEYoaOpbz9v\\\\\\\/DG7LDeBj0W0JjE66HzT8oBzKV26I2PxZM8RUeFk8smNWhqOU2ilsxVsibrEVCWo+nsyQoPT8ztklmbUS\\\\\\\/yhvn2dL\\\\\\\/+bgTkhZePqJyl06DBoLhGrbcslgk3JYf8AilPe46tjHbLL+RzBoPOniL8g14aYYesiD7ZTuDDE7rMANsX2bg\\\\\\\/Ao87lXkPxe4FGKM\\\\\\\/uBItzd897z62QKWCnyO18O0YqTLSHzFAPA7elcadMQbjpN6v39SBB9kB+H6mWpw+XSYOd+mUPf\\\\\\\/W5xtnPZSp3e\\\\\\\/zpVdRk0F97FbNJFxuZZxcmOVQX8BnlJdIxyer4ruzxuxvghPe6518PT2otn9MI4fUcoPggSpzKAQ8oGQFJB4QpSKbji7k2j8IEVDgZpfsTRfwjrA4IhoCXqGLUyL8TnPFlkAMelbus+3B2d20A3TacLiNcvWhBKJ9aAEyQ\\\\\\\/SWe+no5bLj\\\\\\\/xIYVFg7ckG70ABv5hQ8zGJHIE8rgLbC2SohQWKiP3Ya8aHxNfK2v1TW8KuF4diWK+u10qo9l5nZ9U9TWt3zdHAeaV9Uf4y6LIeWJJIZjS\\\\\\\/VTGDrM1uW42x3cKH2SM3\\\\\\\/NQ8z8TVK\\\\\\\/BeoTmzQ+OQLs\\\\\\\/DVssvghP+INk\\\\\\\/ypNm0ZCeBq0jAe7UREnZpaxRThl63bvqYrAkNLWiZLv8oXa27oy8Bq24I5\\\\\\\/mneFOWOpZQQkBbLs1qEpOBh1xF+rC2FHQcnJsXhGqaNuApCvnFXChjy8Unb4IwFXw1e21EMsFNKjHGQhrP8byBUdViKSoYjbLRRLbijG\\\\\\\/SjzOx22rzPrfernxL96jaFu3MxjLXANCG\\\\\\\/iScb2kLiUom47hF3NE9exglUZiXWM4tGw9hYYWmZ3hHUq3gACAXBIaDxJ0mcSYR8khpT7reULu6hFNI2HwAyPBBRv9ofGMq01FWwVdPzZkkH5ye02k9VNcpaGBeO8qt\\\\\\\/pEX9pcJerbplpl6qn409Q9daiQmVeA77\\\\\\\/\\\\\\\/+LEOuY3K9V2qSAml6rtd0wHsF6WST\\\\\\\/yA56JNMOxsjFkrEMHpobo3YI48S1H\\\\\\\/oWD9dV\\\\\\\/Q\\\\\\\/BHlc76zg0t+CJBLnS\\\\\\\/5nNXZShxti3OLfasvT11\\\\\\\/2pNUkiXlt3qmrQXSG5trSQz1uDMxh9P5UZKw1GqJOYnWlO505AqRmSZMEbpMjo6Z9OjnY+s8h4kFWXvw672EkSDRZsMY8\\\\\\\/R1oRb66n\\\\\\\/KqxlG0Py7OFo29DhzAfvT4eWdXKGbIMQcAJD\\\\\\\/UNIqBE6\\\\\\\/Ih4mxss2l07rJiVVOy4pvjdAGV8UjhjCgfvWryzOY1vp5pVg6rmbc8GOEeYYczBay9kPv4e\\\\\\\/Bo5yST9FgaIaqImIsBXkSGwKMMTHZTeSFGr+lkVtkY4MUdDRGqO1PdnGaX66pfHJ767UXmUfeAcrntPMsqKDoC9WHsgk4DrFS6Qn0o0GvE8r9nq1g4wCzEqJghMKeCtht90auaJxbllvO9kQYRImw6XTJWoD55\\\\\\\/MPoi\\\\\\\/Gu+H02LN1GSgieHIFvIydcewJuQ35NRe9p+FBTpZndMPsN1viAADXVJWbHh5H\\\\\\\/T9uzDdTGiG\\\\\\\/vCM7E1JTuI4s14n5dY6Sc\\\\\\\/0oRkCABhFopmhlo\\\\\\\/qGGkq7weqgBPvKZh4NCEyuV5YWvMhSO2UN6Yt7e7dIcAu3rNai8j\\\\\\\/\\\\\\\/hnkA7Yfucg8Sgqcm6+taqeHtozLDiCkPlDzbjdwASkTChGLST6uuhI7X4t58v6bjW28a\\\\\\\/uhjB8sOV3caVpOUWmh3ZWOzszx9ZK7Hs15YTe74s5pofFq2Z4L2r6F5mXvlXMoVWeDrt36GcX\\\\\\\/r8Pj4MWh9TFFHsC47\\\\\\\/vH4+Yfo2NfKOlK5s1D5V4MLNtDkH7V6s0ibiuI\\\\\\\/GeaN3HuvZPwYpHottORo3vMuaPobWC+N\\\\\\\/UId3AyESVryR6OmdpZgQuhuNtOyJftfYkyIVZS3K5lgOBaF3IpyW2rAW+ftuaMPxOxc3FvwZaEAH7NL5ZtpK3LwADIYZFu9cYs3zlDdzO93bYZcbgFPS3w0qqp2W00cUn2\\\\\\\/hRlK2ZMuLXZA3v0euUNMT1fzS4YYRawBQEoUhSmpy9yL4qB8Ti\\\\\\\/4gL\\\\\\\/I2nAhQ70qOa1DZSo6stzUpzFTcFAedRN8\\\\\\\/wEdL4G7vzUt1OBO+yGH0rYSya7PTRhVWsm0PLnLeqpXDrsZ52TBPBXtGr\\\\\\\/G95+3Iw1ySTmR5FjTwJPcHNQ4nTZ\\\\\\\/3Kk5mINTbazAQm4tQ2j\\\\\\\/LVMJkgDAwAOj2Lvj+OfUVJozsDxhbfsffDuy5KEjEX9fgJ7FLK+TsQPfGGV0M3Rw+K0OxwkJC9JYI4qPiT5dIWGmYoKgtAdbWaH8WPfrWaUjGvCukyPtNaxBKYCo40BAzJMhQn2EEu6ZiD0d+eJrPV7juH\\\\\\\/PthcdEjfkaKOlceEdTO6eRL+ehUlM3NxhuYlzjrUBPyksRP1xYxKs3wPtDW4xh2d2reLbBDUGGzYw2Lbjvah0BwR+5qhCn3j3lcLeaz3rNxr8ab3dSD1Hlg\\\\\\\/CNYxK1gPLP1PVAJuNg7Eu+m\\\\\\\/qwut2fgGJFw5xYSwG0gZ9EDhSs24Ch+7GDNSvjVZJgxBA4cL5682\\\\\\\/mSvnUukmV5onB7XWky3Xm2U70tU6Y0jb+mRYL7syqLFVGDB2w4ZogBoQii93fytc+wi2wzLp0Vl5zGX7tYckWTw9XApNbfpOF7F6U0Xndelj6b1XV1rCCUOptTEtNGHSzS35g6X6CY4G9hBAhekNa05wMmQ0kVbtAi3GouMrjAzM8p71XeU2x+jlM\\\\\\\/YC2hEcm7yyFk8CYtC6CuFw8dw3TT2FiIIuZYuC38QiSJCj\\\\\\\/sEDhdC7yl7\\\\\\\/mYi0M3kOoAT+AWPKW8I2s0LhJZP6uLVZ09yVonp9NP\\\\\\\/zPp9o0u+HhAWbH08TMG8YEO+QKmnvLBLTDAKcp9TJgMJgb+4No1xH3ZcVu1GDESiT6c46GMPv4oic8F9nPkNz8UUglaQKBMeG81vEaXYYH5kQGq5H7RSfF6NZ1qbr3rfVL\\\\\\\/i2zs0PA\\\\\\\/STSK7xvEOjA18WVY63aV+BdjWCJSNZRkm6lHBH4\\\\\\\/EZ1EpstHluFBiu+wRtrMkYHUozY7xKd0kVx2CA38CWq2fEScpB4I7NgGXr\\\\\\\/JFWWs378Vm3N76cPiig7OuSGVpzs2QXtfIRuptV+wbhfpi9oHHHOTS7ULPm\\\\\\\/kIehKr7ezKrJPg2mHIh9r5sAN8Pi+3U0A8wXn0bcJqQZpCAIIMQ3Rl2jKEdMjorIMtWvPtxLg3IQ77fCObQkc6Hqw93RTwRE8Bh6YYEaSSUViKTBHtqdl7T4GYLjmzHXEHxqpl7wJ7ybYB58mRnZYisLMwUL5XYU95HO70yaOyWqNKFxoa2XcXvY0vo7Q3ylfaq+6gpx1ZzroDWBUXdOvoIX0Wrld8z5X7k0baR+ymLgxqfNbBKW7tF012PhsyJ3KxgQddJw3rkywAsGj5dKUxeM5LkN5n9x27FkIFvbvz9nkfFozX5IBAH3S5I4fVyUrqzrsyWgfiHaTXgeT640GyZXGKSmimaQTpeZzKUVkcLYsEUhgw2ahUktUb8fZIqr\\\\\\\/31QeGfK6aUjAa5QZ+Avj5AeOljLJMxZdEziYsBU+IS1KTonSsvgWIR8IBHR3HqmaG9TUO8N05UN\\\\\\\/X\\\\\\\/1lVKpBngbg1ca5xaeJGotCGbqFGTayzDxfKcG5538Ro3LtmXfix8Cj24c7+ual+ECCdITggGuZ2lwxpNRBQBo6SjK0QiR7RiLua6DG1ir2zvQ30TSx+wfF2WqxdpbNKbJ6TBvErujWoI\\\\\\\/L0qdUV+TZxcKL35Uje03GOKEmlLTizYJaJK2uM4\\\\\\\/jzOd+yj05p32Ipb3p73nI\\\\\\\/4g5a+g92visOJUo9PAQSf0UA+lU1IBdBWieMKHhKRHaW\\\\\\\/yYQBiVcFVTADj1pEeaKbo\\\\\\\/nHESrbDS9SHU1Kky+KM8Bl025P0H\\\\\\\/ufsGiwB8\\\\\\\/rU71zjQJ29gEPrrPJsGM0JtWfi+nl5MV+\\\\\\\/tJN9Kpui7IxHgGFyoOzqKgfRSsXQka8Z5gN6f+bFAXen5MOtN9iXhtd4q6L5P6X4aARa5AebpFordmP+UyAe4d1lrkEgp4lFH6JNLIckvZ6g7\\\\\\\/LiRu0rsndo+DhxZgQXw6gN5fgwxLNIsG7c7XZvM4IlTKWLsHGGLJOIC+9IDj3ep5UWVK8OOAqfcBGmlFwuvuw3rI3eeNcPEZ7EguPqeHk+uR7Z2r7bs8WBKXFeDdUqFzExudBCu8qXLQj4opox208I6RtclZDRq6bAdIPDWd3RzM035ZAnA6Vf2zZMCAeNl78npQ2vI+8vE\\\\\\\/M005ioXKMeYeB7aYsNCcmUcgKSZPD1X0UKz0R3nZZY5mFZn2uGJVAeNlPrn\\\\\\\/jQf4P6rcW+C07AAYha4E36g2LCOqKVVa5Iv+mXzbC0l79WoBUadEyfBz3qWwEBgt9m2BmMKaGGMY4mYtOuB9D4t1JU5WkHLWuSV6n6B\\\\\\\/JwwksWBBNokK64D3jTTGGb9uhRo4ditSpZYWuz6nRgo+FXcC4WkOOp\\\\\\\/HDVWe8Evwi3UDekV2bfHhRm\\\\\\\/m2P0u+fPvzEZwcN23OqLzByQAUPDKy3jYJiyng1zngN06QfdjYVHwEzl8K2P\\\\\\\/e4nCKfyIFfs6Q286Wlt7bEZwoKt336i9AHzPGPjSLVTQrZ+Hqe+0jpm4sQ0nP4P1rQePA0kBfmtBL3JCMSqefr37u1aKcyzr\\\\\\\/Is4KILMWqQe7tx1MhTOi55e+SFkGzgg1RZAugf2d0sMy2wfgAm+bv2yCExaBy37dUItv9IICzZZNV13fu8Y0tKia0jEP0317EaunoDMBz5JnCUyVb\\\\\\\/fgu7bUTqNS6XRE1h9qKEzinvjjLmFBXQUkWmIYcg8vK4\\\\\\\/t4WFN4pGJfdgD3tk8uy+tzke60EcD6JB5mLU6eHrPRLAu1f\\\\\\\/WWyF2ryyw\\\\\\\/yji5nsvqxl4DTMuVCDCfs+4eEMhxkYBaCOowrtEVgcukzgdZK66n2kWxDmPAYf32aIFZPhfSiE2gIXKS6NY4xOkzxtHg1+Ou+tHm3TjBuIjNOZUIi5V3+YMHTKSfY441PV6AaAJZibz3pFrEjzIwZk1N3h5Ne3ROHD9\\\\\\\/X0OfaRsswYpryJIBgflHc2POX2gD68+kva6O29JzapcnG+nyPbJU5J7vTfon7QpIvFDmAG\\\\\\\/LUO9YAdVfL\\\\\\\/gA97LIeBJuPP3qrPwjubUld3nXjtSr26R9Ez3OK7KdyiSoM\\\\\\\/D4r0bdriUfgeowhlc7QA1JCFKtafburGDb\\\\\\\/c24CdswY6O58e2rrw2ViLHuOHTdwBuVQSiT+XS05gg6Fntmyv+FG\\\\\\\/GJdcBcI5pVGBWasXvall2uv0eYMkruETyCh4co\\\\\\\/a8EOKfXQ24KqQAPH0NHuXa5rSvYJnerCEpHzJKIuRvjADWeqTGbePXVoKjYal9Ky1hrOmYvZ8s7URPiHFIvGR7IPkJPig1enOwFNUnGspwlDMLMK3\\\\\\\/+QNaHoT+GIzfkjRvYDWsBDVUcipLGOoQQjdfxKDZ3yQpTIRD8t3zl2OjO2PGGU4z4tyvIcaJmPhQ+g1REUrJDAzQOLqfBEJnE8wcV7a4bf7984vUS8xHuZm\\\\\\\/GAfBlaCCuS90k34qXcqFsWMw\\\\\\\/kv+Cu5MpaffpFzUF7\\\\\\\/m2\\\\\\\/7KVCZrpYEaN52nCja9puErEEIXJOvbGMjTdhUMGRBxLZs0c5ob0NeaiAZp7x\\\\\\\/WX63r+CoEGumEv8bs76KciJX6573a1\\\\\\\/eOl5nCOwgG0ZoPpeSS0S6ZT2EZAYsDEdE\\\\\\\/6PCS0uKFY7oUQjAObAQbaQB7ZKqmRvpFU1eWnA7O5K5xZBQvgFtVxsP7CEtiVID\\\\\\\/sfF0TFbnFF4DLE3ycVj6JobsIhnsWLqQzV+\\\\\\\/3LthmKUXqpjXbJ3+r5fkEkh+6nkwhhBxtQPSL3LRA0qPej5gU8QyVO1InFf30k\\\\\\\/H9bCDcmsNNEoDbxDrvfsasDZMyok64UFfS8TKreuo32w0Q+zn70D5Jx81Z2OvACxzPQxj375asNZNzlPSih9iG6UoJ4YjACKw7rVL98uC44CYx2oN1Sy9WmmSlHYFh0rYJ8ZTUQ1qkd3EqtV9wHbCdBP+1lewnnJxYVIhQzURVBjOX6RYqIULVUc8vlAcV4nMf526UycWuRp6mAMh4MfGkkoebLkNKxPDeNgGEe\\\\\\\/414jCclEiUZyD3HRK21j5iI1pa3TgfxIWxBckiSb4GlJtqqev3Ts9KHrGRqmxWHctXasrGqTWlf0JwrGLoMSXW8ZcJFW+OtE5IWKI65CjSO+7H6C7VQkkkwrH1YjcxVJVRBFdQY7VsvlxcwKllpnCrpYZQASmHzN82S4l0BmJVOCrqTuiDjptBV6i68inIU3b8qoP27EVw4h7WILffEoY1ALPKyT8xE2Sj8Lz46M6tr\\\\\\\/qaCO7dK7QtCzMNjL7FBnSsLsH1s+jX8ws2TD9vcX3MJJ2r6WMlA\\\\\\\/ymryu9FbuFnp39Up2Dyn4nRZDcSiiLyK2ceWXDPBWiXUUk1UoklgJ3sGgkwLmcbaFibUIqsSKitgVK2bYI3S0wmPF9Y35SSS2ZPr9OGu7adbZhdmuOF6m2YwdO9Y0bJfaSvd7vsksCNBv2eReamfL51TVG0QDq6SNMiAYCLQHjdcEaLboTFpEGoxVr3hh\\\\\\\/MABzUULLK4KVyx803vEdFGIH215ZZf5fim4jj26+QG2MUdtDzHInxPIgM347wGqo12GywaIskX\\\\\\\/GO4aF9LKbA3w35lDRK0Mn4npIpD1ZHfr3Is540f5pwZuVxGPF4utP4vHy9ZNlpJe6TG9XMEEASkPEKEwRpW55vZDKYR3SCG86RXjJgvQIedIaojB0A2e+FLfPrRjD8pIgwTtu4oTINDUeFOm9axtjEDiAxZ7imXfzRYNwVSDeZsJ9yQiRceaL57EiZonGGOL3jh73b+Lu4UKvfAkF2yx5nZiW9BYodHvBOoymqU68HOCbBEfItNcDx\\\\\\\/S8c7V9A\\\\\\\/RbA5Xwmw5lxAnsqmUu\\\\\\\/1cgQp46QvTNiPdVcvLEC705g5WIPQm8V2fXZkqJTEJ0oIIz+duqgl5J\\\\\\\/zaYkhKZ7Yz8rQRLYVQGvqgDDOnHBkCbe+Lxi\\\\\\\/86mrTkGu+9H5ktQPKolDfyyssWEeBibc0dNowPpOY0qD4Qcefl2xDMcnH5wI6J3ohF5gc+O6ftdcAsPZmU9tpYx2DIr8o5qmZXEZO7lP4Su96iQTudwndb3Vmh644RvA2kXRp1X6RCEaN30xJNBDdlWpzpZuX5IP0xK9kLiEA0zbkAlaTWMeTQXLDtmd9hTDKEQPnqDEbe4MpLRguOP2zxCYkW4LexCJqXPLFCJMPdyuVjFTbGV8LWcvHKHEG7Uvi6tLGRhlfsmylb61+V2Dist6M3qsMjNKaxrX1Egg\\\\\\\/GEsLF70dWwcxnCXvoUrhVoLPvChFWZNLokg9awP9Sc8Ebv\\\\\\\/bxHlucY2kmPxFwBks2BOWE1dVR+sJqCcQgw7fJ60b7+8JQQxjJCgsR4iW58hEXdErJLTRcqXJ0oNUNTITckP5PnSaaQbLTnVce0sFjgDpvSzLuTeLAYD+7ut8XCm\\\\\\\/UWk9koUhTqyqxpoALpXBbaN0YEdQha1R7fHesrGgsgcZoSNshGRmdh0Ca72chIeROlaQ1b9yjT89EHfpUWWr0cSCGkh3LTdznAVScs8ARGPf3mmBGURmMqDXesMGRcbCvxBvEc2WmuYMNOwFesRzX5wkK2aLPFEm68DSze6ez97yx16uQccEaQShwLTEm24vQ3w+eFR9WFUHrinWwl\\\\\\\/U0czVBig57dgWOLNPm9EIJAbIsDCU7+AxxCm8hL9hmWBehl\\\\\\\/dzYRrDSljjMmgvRs7PFfIg9E3DWXu3aBnQEsNcI9TrssUuzSp+Z3+irPWvCIqEajX\\\\\\\/ectyXnLtE9jt2LglRg9wJRxBwAVMQIoc7yB7W2hZshqHbNcY36xchePxBBhu1553G4QYTupb2CAe6czJ8huXSmencqr\\\\\\\/Nr\\\\\\\/yiOi7pBCLwy2udcB7gpivjB2Rya5amURimDZX+lmdw9AZtGpgoxqvPLJpKI9Nc+KArjCBBQoPtpbmauXubqeR5TKk1tUKIkNNB4Gq4Iesjm7VToPyOxkH1NEocQhUEUOnIAzx7Y1Y9CsdW8xIAa8mPa9Rj5sCmmM3zQV4YlGTLCEtgCESucl1p0BjO2cbXWOWkce0t47Di\\\\\\\/rWbAPJPphVvk\\\\\\\/p2U1SeIkXDSReeZREx+dSsM3ypwdx3T7KeJ8fxQba8IVnB2mIO1OiapSRK7juj15E5Rq8dSV3799iURC8qOO\\\\\\\/x65GNUpTzT64J2ImOn\\\\\\\/NXujdDnfykap6ppfGF2bA9uKlJKshaD5g8v+P0Dc8Jrp1EwPsFzfW+rRon+AsK1HA0XIW+9rVtQtyBoZk3Wxpg9+156IblP1iKHS9\\\\\\\/TwqssRM0dMaT+5SivZ7KMqTM6ywVhDC4xpI6suS2ISCRjtbege5zBhMo5H5opzVyT5JKHnMbP\\\\\\\/hUG7D3JEgnDY8Y0cFWEFbSWp5EApqNdq3maBRGcfIp588Alf7+O1K2LAJRtYBRIqpuofP6+P7Q2aq\\\\\\\/pOyoiHbI7vIe0ngS0JnUG8Vt4RpgVe8AkEB8TDAFKyER4OzIQsIhcj1xNAXxL+BuVoHpz+xf1HKBRMwFV8+DLza82yKLd6F8hZPNbve6CTV8nRrLi75NRM1iXoJo5wLWxtSTR8wdXa57zfHmZA61R2mc7vJ6\\\\\\\/ABMtRXrEArK2N\\\\\\\/bqDO47oPCgnsNBlspQnu+m4sfiwX9uNHAic\\\\\\\/oaWtpBJiC8Bd21HtrCJAPo36SCrh51hp3lA3Qv8plrSu2KlvSe\\\\\\\/1FDupQ7KjPnE\\\\\\\/CnxpvfSA+7O\\\\\\\/e2cls1\\\\\\\/MR1U9zpympQRmXqHB\\\\\\\/eVF+kF8UY84aQoAeJwBOyqzSuM3OiC6Yh\\\\\\\/Y+Ks4vm14cDMtM0FpuJdAQvZrZNyyz6ZZBzE6uZD4Qej8QCgSOjTg0xf3R0MajLCb91HqQ4YjcQCZv83XnHE32JbxtAs6eAyv1A1ZQO\\\\\\\/jtlU\\\\\\\/idT\\\\\\\/Y55+znFvJtE4amiip6c7nR46PW+RP4+n7LFKGWkd4rDORg0YOylJtdSCN5Q8CquDqlM5igZzpr4jwvLUHOm1mZTvGajAR9KjDvdFs3n9T9jca6uBK5lmFL5GEifFYBKOv6XwAokcv5vRUSHG5fT6krEjQzAM6cdpZGXvyuq8jY2sEsmtYlSsaSwIsuVyXI+bFrDajJqtYDvB+fiJfrcQH\\\\\\\/CGUZEimi0ZC2ecyWRGe4Hi+VjBXcRpp6iUlD\\\\\\\/j0eoRdZ3vR1Tt0DdK4fuBqMubNtfuMWnKyCA\\\\\\\/A6C15ukRevR1kEtzyOCu1tqS1DDFrl8pTZP61csclHIzHowfhsFiaHRnBRrkrBT87CCWR2mj+OPZZuDp7+KW5BF289U7iqJlNJpeNCHIMnucIyXZSIYqZrssVGg04u9NooTsz1HwMeWuc8hghSKyuD4IQNHrRQYJmYx30CbfMuiDH+W0r21nu01xCAytrbMSpoHzAYSK+h\\\\\\\/OBKuM6mwJZYT+BSlY0GyCZ1E\\\\\\\/Ie43wNzW4eGSSnOVW6oDvydgIiLzn1rE6BIC4nNX168gJlIoWA\\\\\\\/OUSLOE2uUpMQL77UgURXU\\\\\\\/ySrQB89SjFt0hyrPJKJACxeEq+FgHLmLBhBUIF5erld\\\\\\\/JRq9+vzXmjTlvLKnKcT35Eisfp6ogTsph0rzTo+Iuxw0MHkPHe2Mlws\\\\\\\/95b2xvmt1g+KwPnCixClmUfLHpUXL1oAHYx2E9AKHZD0kHLQFNHjsxUXd6qhERhDnnpDAVdYp68UmqVE9fU6MxJkz0fBNbqOtBsbsjVIG34FANrT+QSE1pAzYrMA\\\\\\\/e9I8ecqDhuoMKFlzPZ5y2BpL\\\\\\\/vD\\\\\\\/WU9Vcfe5ARvGREpEDcZxMCsQt1Mq8nNssA9FknOiSk5til4fKhLl8JQjvrIjpk0DMNVVliH6JSa+g6r7PXrBRtF18wADyLMVK9iKkFNaB0g7XaLdoiqisUnOWbMHBa3MY4hZPQO2k4joCr1d97scWL\\\\\\\/sEH57aEB5PS03fJ5DZbLGbuyge\\\\\\\/BHUqXDuMloKBUnpGmGOizlRYtrXnDEMBr329q\\\\\\\/o4mdVKwKWrn8NvIqKvo9\\\\\\\/wSU0sFY6pFZsGrIilTmE7A+pM12\\\\\\\/ECjsH0oNQmyFfTEOvTy9LBmxh+u0NvmsdMsSCLcE6xNHPDDxIwwfzkbgVVkJaGRC71ROOHonBHanEDKS6JetMb7zKXA0Bwtc2rXQ34QpFDFfh61q4AKqi09+uhPM5poNt\\\\\\\/MfIMxQcq1DapJyK61LaM2MFblswqCBx0FfCUswTUWszXVJXhH2\\\\\\\/XRIu10CSHTyxRlHiYYgoM6EosLet6\\\\\\\/ots3N2M5LJ3bmAWjnm1xfxkjKZretkTGwJG8NlEb4OBiFpggJcc53WOn2mNrY637cv38tdarGYnBT0NJqMTYTkNHYRGLitcIVbb6\\\\\\\/wb4Ugmce\\\\\\\/tmCd93rnY41euxKNvDfErucaD1sJxpKuS8g4FfDpoP9tmQZUmCvuob27jOnLwSjct35CREcafdS3E4xOWnxThmtkQTBZV\\\\\\\/F\\\\\\\/hUjDiwB+sJVQ80Z+I1DCcBY2y9Yf\\\\\\\/BEZu+x5xWQlVkt1g1fgErgjn6zIe6fwItdQM+\\\\\\\/NXGw930KgVlJHcJ5OnJtprNFhMbWOCnTybWnej58BYSbeuOScRbmVww0729nX5VfjDqqpzR8AtBf3VnVxZMOW+jRPADEHq5kueJEEWRdtfGud46WP4Q8F37l+Jh6imrrAZnyBDRJQQmIet+btWKcQ1dJsfx53ru7\\\\\\\/jw45GadGRe83Oa24zxZylYkYpWP4T7ZkpALFwEw6kUHKdfDOcXah3jrXPvw0VmCq\\\\\\\/lx2BwK5xu3uaKE5gcFJaY5VgHQfZA02QImlKIQLcQ1Dlpuigdlhl8GP257yDUJ04N5C5X4av9MS5rps6lqFc+tZ0LefEWMaN0fiogDISsmYmf4ec38zkKK1Up2kM0CbkiFNFHvx9aBb\\\\\\\/IHTXMiFc941vS+1TXDAynrXmSIPEoQcBTdTdY61stk8AphA3JyXe2H\\\\\\\/kg77o03iYBAwiFNsGbbRekqZF7aGvL6Vy3Sy\\\\\\\/T6Rct97XK42DiwEG0KWnMyQWu4wV3XXofAlFi0JHHpRH0u5itxmC4NqT9XqNV81Pn1snFtOTQ+r8K2ZFgAcgNU4t4vQCZVenZQY09tCpEYAEVF9vt5ge6lxZErbRdtCXaQmwb6431ARcRe8YuoMffznqg89DBUaS4UbCskxT0KuPW+kptQeZypoVsN8y1scHj+peHrQxtHHGF\\\\\\\/IPvH70K2LtepRYBNQlGkh0iMBgljNaqE8C1UEFlSbwsGOa2CRxyf1F8\\\\\\\/fLeL0\\\\\\\/95lTMV5W5O\\\\\\\/GzN0BuZx\\\\\\\/VkyYRvMm+hlCkbIvRA7II5J4pXGjpY3dNR5pL8y1W10E2Je5vXH2VpHs2Nkh5qFw8lu2iTA904jPeT7eCDZBThctwpoERYKoi+1tOP9dpxAnvMeOD1iLEwaph8mX6pbANLCjVfyC3ytmSx8mhhtPAzqKxPqecsSmMkSK8SE6yZ6idVattY\\\\\\\/AKOn9x3z7jrYCF3aNGmZBUzQAvwmsLtRMC6Tt9e+N0Lfoj0LAVGXXRQKDHxXv8RtTgYC4a1H3EGTHhv6OEf91B6xJ7MHbpjLLK6ynrp+sJ+53sTw0f2MEYtcU3Y5UmOiH2sQsXzf9i+f1rXNkw2d\\\\\\\/0OULcpE9tbPGAzObZH7LBDP9cBfT3YwqMismTNR8mWmE\\\\\\\/zpxJU6aJKLlU6IaeZr+yiHbnNzKMZpf1XhDp0\\\\\\\/0PxCR7LLgP3yEatgSbXVsiJakeq0\\\\\\\/O3bBdZu\\\\\\\/EdjEy4sp66BgEDn1M24l+0XInsWAJIu+OMZ1IPqRTdSNAeFo1G9cIL+DDzfIW2DLHs5xYAHCOsdjyJn5+CGObn+sPsgqjUL\\\\\\\/ZUq584KhKJtC9EOJOHhNQxPlgP29rkMGZmxipl7Tl\\\\\\\/ywJZK4vUNb1owrh8T73UT8ULwRVY9fzorvmbCi9K9ZUR9Sb0YS+M7cD5pZergq15rf3eJBjonwdmdhyGW9SeQt4Ouo8xPGzjdHSvt4dwUrPWApyJGwARtcegTnf5NMcfowFc1LimxCdm715IYX3ZMS7i1aKsAsch603Dgb6bJZwylDO\\\\\\\/8IvQHHvyr6AoxBsgbLraJ5f6pc5wulxXrT4MJMEtAzjUVN0QLZE1jtce7Pfd8FZzLCHSTdMwipxDcc94IOq1588YxR0iQfHFNsNH7dVCB7tmqSAtpgmUnznWhOozJAN56xsxsWlV7XPAJ9Gpmp0nHruc5IehDMU0QF\\\\\\\/kT369PfItQuQg\\\\\\\/vnVNBKGFUwzGmVGVsjsTF0souOomirVambQVPDuPHTgbN3qPER1RjmZwnevt4BngSICFpWWUUm1ZOjaThhpZ0wNKClvzyGLRKktZumrjPHX0MNc\\\\\\\/k9n5ItTCeNH8OfdM6vv9LnWblZrLIufuQdA7qyOMczEIeYgHPjmBtnenswGvcfTesHzS2YOm0vV1tdZW8\\\\\\\/4yHobk5A3ATv5LB+DYxXra7+ZAP3y41HQ+DLErrmzFa0Qyug2aEFKbA77lEV24UHcyjYw0fWZcjkvMYXcqAOIh25haOmKNgZNzcS\\\\\\\/KE9t6ib2ZbChsv974\\\\\\\/Et6M\\\\\\\/1FgCP7ctlVGjV+TxMTpkGnQ28qbZXNVbQkgJhojE6lRBYqcTLl8eUooEr52QZf5QsaBOYPEiqPXm+bdsu\\\\\\\/YczQp1ibUxmnnXPvzINxhDYQ9eF9B7I6fbBMmsyRjSLEznmXkYuH7267DfIxyCSkT7wfFFJMlaTs3eC6Wl+7EkZUS\\\\\\\/lqVlrKmH7W3\\\\\\\/wHm9MBesK\\\\\\\/JQEOVYU8EFLyvsOsVHdmaV7+xbXEkJI\\\\\\\/VDlVlLh07Xa\\\\\\\/jnRVZieD7cnUtXWkxzxSEwtkVpzaurKnxw43vtuIjlTbSqLWH4+iHSKraSVfVABQONNYmLLrxielKoGNl+tYQZgZCJBZOJL+j\\\\\\\/t4Py7Impd\\\\\\\/QhosgJ\\\\\\\/oIeghFeAqUkUNxfo+9M4a\\\\\\\/OrKbKCvNJh1YBlXgbaDZKFHs5ffc3+YU0BZvvxOOBpsdqYWci8Uln8+QJt1oYoYl55Uc5TKsp4kw2\\\\\\\/YfwPmSPJ2PVfWa9fyN7Js92fhxO0357Ev3x+GuWq43glQ8YkRAi\\\\\\\/OaQ2yONJkoaEJN866CfZNi0pBIBGneUl9tvIRu2mmIZEur6r0PS2Q08Hx0uxiiDK\\\\\\\/XkscHtoBYI5d+7w8fPY+X6Dw8eYbB+kyqXpfDty4stHCK5kbJA2orHm2Eaxh2G9dAT05AptBAIZ4cwbu3jbxA1Ze8fTXHJOMRyfEESpwn0akRC4y6psZu+Xq+mfJ9+Bn\\\\\\\/iJ9siOvhywWv9Ol5sOvIyW1EPcEv2f9EdFRJs3PnPwYNK30s2w+xqT9nvp2yoob6UKXNqIAoApJctGHSbDuFY9VW2yzUMltsbzVAWHqOBuanvs4I1E51CFWE6qQhH\\\\\\\/MgIMxeR\\\\\\\/oE5cIFRB8MlmiCTuWEQUav\\\\\\\/UrSQUA0YzPFjPGlOfHGXFFDX2kYygFRjvXYgbKyk2fSOr4guKuawByP2\\\\\\\/3N09QUbN\\\\\\\/lxMXzAPlPw5TtfaDMWiDfvLCZXxxJCVmBQ2wXZMeKe0N9K+eChCVhjpFDq9C04sWmcUtm\\\\\\\/zH7\\\\\\\/4CI5z7Af2uh7TywRijQJwOKVSc\\\\\\\/Re0vrb56o6v1toU6DsJ27TWixDjpBat\\\\\\\/NxkJfiqpE34z1BQ4kS56pUJu9Ey0VweIGnemPOBwaqhQ33\\\\\\\/77bYvnLwS9+Eu6cf7mSmWacHck\\\\\\\/gG1PhtYeEEOdTIxMnyMZWEEPvmGSRem3rZinfkBq7KCjxly\\\\\\\/4lBlDVndEIyaDeC9ZLEMbPis4CquAJnQF0vARmivb59RaCYRnrr9jMAbIdnCziPb1UhkvkTOhqy8lTmNbJD8OFmCVVgWlTTnTkFG\\\\\\\/8925tpUGbcn63qPgesuEPB8HaTA3JPveG9wvVuY\\\\\\\/HyTnSfsKtwZm2MVFXC4PSeuZ4xaWlnZWrWE1+DbVqgSx1H1v07nozueTgxc0\\\\\\\/4plxr4EgoOllHYs9AeE2ol2dtedQs\\\\\\\/apqf6n5yV2jAm80w50j2gS5cqFWHDnfb60r1SNkQIqUrWL3mT6K2JmFJ1z0c\\\\\\\/lbIv9qg4tYuRG+fjDqTSVY3F4k9AXTOXIfmId2C9QVA4KvA3oqRjDdIbyUX6xpTULEmpBTo3i+nw2Ue5u7JQRc75Pm2ybJJo\\\\\\\/kkYWkbpGaLFvLSdlOTmSTdPVWU0PSyEKeIb1dOqI7gvohw+hbZGGhUi22g148jUivxLwcDsKQN+s7yB4x3EKzMozK8NRcX6ygf27l6EHtMEIaIczL8WW+YlGsSfCa0Q3cclHKHFeQE5y8l8lH0PUdvBWYIFGhw3QYriI8z0v18mIpNxS8WbOjlx7QGoOuNO+wU4R09KeEI8Zz53bJKeBxTpXSLTroElkvl8UAtXDz4FpX9+9fHv0xFFMldzrGH3zSuy7m7UgeNUzrJFR8TgXkWIwvjQX5R24Od+uVcs10OCw\\\\\\\/xB1qxih09S3IfB6DBsI0YtkgU0LReiW99n5ZtPIJkt+EhTMLIL3wB82iLgfFIxR8QN1WgzwYDso735xI7QF\\\\\\\/usEQYKFKm6wKz03z1q3wkHksMnn9D+qdsHUgMrI5sGSVr285CasPDCbnSCbXwNtEQSqxFVMoLr7\\\\\\\/UthYqgRI\\\\\\\/HNfbU2eGl\\\\\\\/IBF4wsRi7HkXVpt38yc+lt3HNUpQw3X6SLpEVeemwawMB1oZeo7pH28ZLpuEEWFdir9jBoWDl+EOQrIA5pNszKVImi1Wv\\\\\\\/5tnDqzkc3wgRPRulIUJvwZ41No+LeQusKq4ocaq+TgYIzTkRsk0HhkdQND48TgaH0LfIvlREf4JpuTHHGNjOc6WRR5tsz9ptu9\\\\\\\/51Ya1sQyW+UbqCWlTQsNj\\\\\\\/Qh4q4\\\\\\\/Q5VGpXHr6sOHKdGiMnPZIh4hdoTS3K6d\\\\\\\/uaffkRujspwSWdmF78ptTm6RXQQMtkNgpUW7VS1sl3XvOAvSlkDrYVMNQNNijFdnlXTbLcyj\\\\\\\/4VY1trt+FkG5eTmfbJqyT\\\\\\\/Z9oRpp6Sg3Kp3m0MF6Im1htiu504csf0hXdDILUiarJQ92e+qD23okjhOTGuUCNiYVR3e0aSR2ybx3Ka15bgiKaTdCsorsBGYajCLTwWbT82WxIEt2pJEvMuZ\\\\\\\/PqUTIdAuORqVAxtvIkjvpCLUtc4QU5W8THsDWyWBQTBw5lrrl8IzMGn+l1UhgweWytoKoa8\\\\\\\/ajtY4FUi4GR1s2mowHkU6oUJASLCInLNEk6K9QOkU29iGxTLS282e9iwdt9fQIJf6nDXhMPkRebPyfIJQqF3MjYNzlPNWroDBHpoOnN8O9vtCsxFm1KKEsOR+ZDYAPpD5RRMqBp4eOXkD8o\\\\\\\/fI4OI6RGoztqLcq7wM4s3jQ5W38Smq6jJvOecM5EXLt2T5zZZO72Zvz\\\\\\\/v74Btro7ZBqpYLIiF095eZQ1jw3H7SF5mOAn8Kho51CKUXOWe5Ln1VfvRB\\\\\\\/Z195Yvp4\\\\\\\/FZLNSFyYjr95ycLlxmfUNLpgvxd3nQgD4li6zjgxpYt1w6wLqvugMdajJ\\\\\\\/BEwiHW5qwaq3BmTOE7RoTitTPECYXsQMiBjEithxGR5lmioOqyPMabfN1kfluBG\\\\\\\/OhLd7Zrdotmdrm5cwU1gy99JGTG9f5dhjmzjQoeae6UsxnppqssfeWaY9ruxYzn3NnQw6W8cxjbpdZ0J6Au9WfCDUvkCkCBBcDAtFMu2a1Zxhf4akkh8cuKTaQjoCTFCrdfHIAUhMj3XCrcHvIqliH9y26Vq\\\\\\\/sNADFfx9stESCoMiCUmjA2Y8z10IiCA2lKWhsqZYKmq7PHJ6wm3nJc9XKNjMYlM\\\\\\\/6Pgv5O4qrGCK3Xg7gGEAZu\\\\\\\/SCiLxIyKeOyOnk98PBC4YsMeJw2YTv7sUq7Zwo7rJtXCQWA2AyX2GfU+X+DkkMjE562LcwtQdu2LBy+OJ+kjCyyKrkIPk75ob1j3qq\\\\\\\/7lvAKXWu2Iiz2ohQ36EW77AqqPb6yn+uhCScwT3VwzxgbtLT\\\\\\\/ee4aUHleaHtz4jv1dPI3ViVLlhiHJDzvAOfeVpyvj9TB3Hw1WBJJirYgGeDz7YVb22daOUYlbaiysmEK72U5gJ0UQ66OtCvj04QEgZ6WlWoTfRtL0ONItEYt7He\\\\\\\/6Divdrxi772ToQPW0WlMEzBBg0gj0MzayuZ5hSQBo84O4xEZOXgDxVmMCwnB\\\\\\\/LiBo2jW9lALTAWodd3fCND0MQzaWKOpWFHZ8WPg6E6hKjiKtpXT7YEMEwTnzFkYnsxadMlXVlCVl\\\\\\\/83uyW9727j+Z4hjPs\\\\\\\/widGaB\\\\\\\/wSijjfRPyjcvKVjImO8+FA6qUtyBmFy04Ws3AfP2pQsV1QySJvMn6V7b0+\\\\\\\/aBTC5rUK5g7c5NqX4umtrxKvpgzmDvQF5xWW5voBUhBgS+vY\\\\\\\/1F4yi+r4AYRDArTDI3T9GvTTJsKOGMl+JKIHCGWdSYLA60bcyZAirk3Oxr7xuYUaOj6McsB0ElvhWtySlcygeYCgU9YfnvYxy\\\\\\\/sk3N2wQ+nM9\\\\\\\/mGA9EeE0LIdqjD45vf0RR4j8WKF48+vcBUeSzYbUNsjhPBjY1QoTa+aQnbhrVq54+Jn0prxL+9tjEUE\\\\\\\/crJLT64IO+O1GtkqflbQNfm\\\\\\\/ODGdmVZwk7Bbzr2r7qnavfV\\\\\\\/gt6FuYeRJRfOlWnkrFuA9HEA4szWoCT+tVAoXqCx4EW6Sppzrrrro9uI4edFgLE9ggWBwxENFajoohzs2z1hoKBQS5ZYnS21f7xyunPE14PVYwwPQd8krFMEwjRPe9uxSex1TRz8UI+kZh\\\\\\\/jTR1XU4LiaYJc2cAaSHI0UnEf6ml8K64lxHUp5J9bsttt9cEtvJD1h5f1rIbUmrSbLNrcbYxpFBOOAzuT3hluMGqhyzd6La4TAC4YUO9\\\\\\\/rObROvDXKtVx5HKYsBK4EwYzGkhV0R63LtVfMBwFNEx2olIAQtG9bHA7WHYP4+7P591dHcGmLDipyO6nYmIAq56aJY+sEifukNRx9Vum567LbuCOlyX5FVipffpWuez6Z9oRPbQkI+2Fe08JS2utFpdAecfnsviC7lzkplU0B16G8xSXIhdBL3SW0UJ9xiciBy\\\\\\\/ilDkP5ulf75vDO9xgi8MrQ8YhZLaZD\\\\\\\/HfY5q44S46Zw3Rgop1pKD5+1Y\\\\\\\/0du\\\\\\\/NZd2pWvZLsEkocDKW5Zdeb58YF2Cx743Fu7tX9vZNGr6RIXk8eoX\\\\\\\/ZoDx+tfbLIRHBnIehPRLl3hmUaRIL1SgtCTHSeiykgpOXJZew0xnIUJUXkS8fD\\\\\\\/fWDqQ6VCRPcECMGRia\\\\\\\/6ZR6pQSJK1rvQ+vBT\\\\\\\/chMruMvG2C45ol4D86eTBCUOnLwkmDxteJf13pOja8qL5aLQVY2Ti9LvsFNsf+Qm481OoYxl2ybpwGdhvlMIylHmsLpjzqDgWbeGl1ArTVdnf2lbcmxjEFn+eT1Pn2tPJZjpw5duTRrJ8p76uUQW+xOSTF7\\\\\\\/OVOJXVmcWTlKUDAuL+\\\\\\\/SNfLh4VJeTT\\\\\\\/pBkmyjax\\\\\\\/r5AazOKrXlDUOcAv1pCBrlJhCbMt8ULLbsyyvuQe2Zq4fe\\\\\\\/iU9cxYrhDhbIgJR0vNP61czAm8nj8Ut6gVxZ\\\\\\\/lssCmuGLI5+IbzIgZY2N+Y7lVOk2KCvoVf066qT1zLSWAl2dK+H7bnv4ge\\\\\\\/rs1WjYjUnr3Sr+bHuIMZPvrZ9TAhz\\\\\\\/vPAAjj4xAXN61zG2lMHqhzys19onOUhhXJpSLxVuxUU9aI0aqBNwWJiHOU9KBhzv54wraXigiSmqG+3LmvkYaWgx82GSNOUUh150z\\\\\\\/dLKLyA89ojckCBGbohvnbmAmTuqIqKV6Zk9Jq72yfK+ZFSMDIH4cyJ6o82DoqLYmzzs+woldrktb1EWFet5k27WxmgXvRxCNAj5gwQKwfrX4dlMVW7mEmD2lBG2h4V5jvggo8h9e6XTBpvObNEt7CxXAq2otojsFNb8RXDk\\\\\\\/we4d3404Hr48+96Y2+4ssVtGhscn3xsBSVg369tM9b4mlr7qoSbakNTeKlPDnH76x+NOthvxG+9\\\\\\\/TmNOIGfdNB48X7XaQRWm43R5NRjBt8YhR3XkaoCKp8v84vdFIg4laJVEu7+\\\\\\\/n0SRklPnEYTEQ0DvAsgyFrDZYHSHfYPT5lt+P0cCm23WHjEv1nIx+hIH6L09ylzF32fHws0Oo292Y9g\\\\\\\/UE3iyKOIBIIKaYwsZwH8fX8c00S6xpjtBkDqfhzj9Gb\\\\\\\/QTady942cN8CJUog40jllEb6QohPQSSFYpQ+Mqh6wXJvkspr0lkgiRJ4w9mIly\\\\\\\/LK8xR\\\\\\\/B+2YBJC36Zvy6Y55jvYWIc2c0pcdV6G2f6bP00K4NDr7FGWdloH6Ld78OawMoTxDT3WXeAy7WaUfGV1pArPd8FNjJnrt0os5ATE9ZLwNztSWc5p7LaFJO6c6eX8UkFukuHM2NUy89T9cMpiBj0E0EjPCt6hpQ8eL9P+pRdyJwjjh57TskijeaTP553rR\\\\\\\/BAAU7qxa\\\\\\\/qE0aJ54eak3BO7ny4lGczkBkjSQ9St+jQSJBjHrh\\\\\\\/pqpfwk5T\\\\\\\/ugvs59BEb0mykwj+0VEL7ldewBst+KqDUR70VadU6GP7\\\\\\\/I3bJ80valWS7Iq7XCw98\\\\\\\/G5IJCrgUEq83z4AV09b\\\\\\\/OfmdoNf9Ou3WEiz0keAV8qAvaAkhU6B+pBgBfPo\\\\\\\/Ki+Sdf9Bf4CPlEqU2eYZNTV56dYSi3aYlmxG7jDqwrks7TEQQ06gcmKdLUgzEwKiP\\\\\\\/sWkDRhuodguw7rKf9uCP5+azcZ+0dfZDbBe4BpQl48LZuYBDqmqDNsHzWlZJdBk4VU0tvZpAwRBQw1ijQqyE\\\\\\\/bbmrme95HChdqmiT4EzZPre0XLEuo5WdSzqranxSm5RxGRPnKoYSG+zUVm8pZDh2TpacJLj0u8HO+dzQhLUhctbzRcXsvdcTYsUYduoDaVXdzyuvzPl8FuJBI+aG8LBH1wp3gzbKfENfzuoUH2v7+9+wI0oBrG5m6S0Fv2\\\\\\\/bXqAUlm\\\\\\\/BaqM118IiYLWmbQ3Eq7CVnUEYpIublNqZHiZIL\\\\\\\/fiSIfxEDMe9UV0OtKi1t3SeLPpsKRSrzTA61s2eS0mYhK0eOHAGuYzprJcwv+Nb08uEy7x8\\\\\\\/fUpIGlfz8jYxdIhBhg2jMghzjTG7iapi9pmAJmZw795K2WI8rhi\\\\\\\/O+FTV0PHGGgp6lvERoXCU4xmNwVqSe9XIM24VMyKc1hhWuypBVOHmUolTN\\\\\\\/VGv7b1gGOZBagjOdTXM03A946vbPCtKPzpjDH9+Vh6g3YuqKWZlqW0zjSDWAjWDojS2y9VL1rGTJY\\\\\\\/rWVvB4mdrU90mxNRyMLSPB4aF8gWjVYOBWIrUQk4N2LnJNJeGcFbURwOqcF8Q1wSu5emA5qLgO5xIc7PIHc35ifWSZv45ndurfzntWPYxHOx8s82nOl\\\\\\\/gtHHnzQZLea9EjK+U3KcZVMdqE+SSaxkL2SZJg\\\\\\\/rm9b8Ftna0OTAkwx0nY4kp\\\\\\\/GK4ehGifrp6YN6X0ozOHaal8KAtZ7N\\\\\\\/KuNnCKtWiXOtDggw+95i6a5hHZ7L2BT1EYTKuInaosnNp7cQl9tgSVkgUJJy0DCdgJqV1ou8pKnVnBqFCNBCDtBmz\\\\\\\/qXBBN7cdk0AwKE+O5vlxyhIJGL60BimUhmmMSz6JhdCuTBIklsZ9POOqXUwJYyfEoTg1HgHdyHxsad9gxz0U82hGQs1HGtMkVjxAfiYIj3TJxVCenzfjlP7izvmGVFi1MTbmbgO+Iu8B9TSEGVwLaLJDyj4wIvNeSGQfNyXpl1EiHMQJxHnN0m7qNHXIKdRjJwiQXKBLLdFJqykmXRDY8lIKN070Yc5SHoP3lMGrCzQGatQ9DTEgP2QyI3O63rTbY6Gu9xDPABw2qwF47uPFdnVaGJLa3oBTUNojLMn1iLfpHT8sR3rYGlpYibDOVVoC\\\\\\\/tfETN6MjRqHsnUmQzDgMp5U5qTnAWVkGy9q3y+MN4cS\\\\\\\/uI\\\\\\\/Rs4m07o9\\\\\\\/+Tv3ixRwdGFNWaM0fjk4wAaEY8SyIqY\\\\\\\/hBI1QGPs4oA77xXU2S5ATzTDF4PK7EAb4wfxAFN15lfCfvpKEm\\\\\\\/fiE1\\\\\\\/G11ZjVIteSOAQZcMb+gCeCodmAH+0sT4LejVxiTZCkSykeQfQYwFhK\\\\\\\/iIUJpbh2Yt+yCw\\\\\\\/lLjgUrJzpUvz4\\\\\\\/xbPEWi7pp57thUMvhvUL7EFhFGzy9PjOMEOsAIKfv6Cu84VSArum\\\\\\\/Lg8KE1rvChXoeIoeMERyniqXtnvgZPD2NhHQQElpOWvF6cUjMi3Nkc\\\\\\\/G3R3u4XgIBbxuP46wV+EbDXKzfQOJme7\\\\\\\/lg6IOr6eYI4KV7RjZydtUzWKzxhRhuLUL72PzSB+oTLa17UIZrmK4IGq0VQDsqNlk6rHFHMiY3AVjZUsKjRL5QfZ7hXNcV6+WFSNYuKbQTl92jqIa0qZRG6VAc1Kc7oQr9j7KCITVrJJelz0MIG8hxF2KZDEfzV+6JroriVu7O\\\\\\\/7oVU3NEyk2558dYEW2mqe3VVzHG4qto09Mz7IlBnKJOtI4uHL+hfeBc7Dv9ztVC\\\\\\\/vasiOXaMiCF\\\\\\\/BPck+DpLX6WiljsAPo4MgNKxdXC\\\\\\\/zweQq\\\\\\\/ycpV0ZOK6vnuQePC3Q0qpp8Hx\\\\\\\/T\\\\\\\/NxpkUhwn3lZgtOs9seIfskztUetYyHDO5VecILl\\\\\\\/DTtS80XtHk6JQUEs3pGKN5VV06V9TTiNSgTGez3yFcTZd2blcVD1je56lsaixvF2F4W27HCTTngnv6krC9dveMlSyg0HpuEt+iiQqM7EUZRJjO4OxUH2WWdBqnUkScHYF96Mv5GuZyV+IPJMLpUVyuPzBWRD5XNSXYJt5tDTX5UxlpBNWByWZJ\\\\\\\/xYBDR2bD8elNolXsFUx5zmI20dSxU53L2YaI+7vlgy\\\\\\\/hDgvRAzfB9bbdJr2Nvm6IzYfvAet+jSr8Dri4TyMbZp\\\\\\\/IvyHLhb6jrp4451bbJQNZai292kkb27O34OkrG4UrcqKJbwKk6w+WYfVRFsWJlxIOab13rFN1KtdTLU1KMJAuKkcs7cy3Mw9\\\\\\\/vXIhWDNd1iwIjpyFFqHv5kt7b4Gu63ZUZB9GMyhrLu3u+zwUGeRCBmXFsvNcFajPgN5HcsHaDxqcQI5sfCOzMlJSePReoOyu63xPy4kdD8sjNV7+ZHg9hCTGsjVd93Rq1LTMhH1BoNOL2WnzKGPUJWcax66QoD9fOHG3BYexOkGdB50ROb7z0Zd04ZhBjKeyB19\\\\\\\/Ek505yEq99c+ScBrd7J7NlLPHyhF0w2LcQKuFIPkYN\\\\\\\/jRixYybNm1PjnMaRpuWLdIwwPq3Tw9TrzZ7KVxefw5316ckDp+DI7lJ6oavxdEoU355EWLtblM1MXK4yeqPwsmY3f8F1P1289FhqW18a5FRLBrYCE3xFuQcaCXr1VDiauLwLkcIqmBN+kwcegTgCk0ULN1HK5fazLni3I17dvb6cSbjgHJKmejpNNdGYLjriFls73z1mYBomMm92QRxjZHPP\\\\\\\/F+QlO\\\\\\\/DIHVjL2oMzXlIQtIoOMP9\\\\\\\/Yp5KyHOim2xoy0zdmkenJREFRTSdAu89TLOxNmJgrWlyOVHykgiC3dVWJpLEzkvMdTdIsyxts2F6fNgp9D5boXn5iYUMmI6AbxS5xWr2Z4A2VGxsM2UZxdoxF2MZmtR2IJeHvBBJnHFJGdDF5yh6LT7kt\\\\\\\/ESQssOhq2aynMtrPf8nT1ta44ZLwFR1FWYqTUtGfPAzdD7AfAQhRKVD2jh4be\\\\\\\/P2PUZJ6pY\\\\\\\/Kqfp1fuWRsMpd2aLjKIwTTyC4OiGWgdtfG8ufQqKX+Ga\\\\\\\/a\\\\\\\/OHNnUEiG6Kw34BBGirUb8UVyCCP97MFmUKqcum7pzCO5abLwjHmneaVTQqs+WfGBlgs0yNY4VT59hDYbaBBVmY+phvfDMlNpZmCGuE1HdDjJQohHwjS+uCIRKacnooJsTUHAaPeUouEguZ7Q5MZ2wOOzEkMNtiXSBww4lpcavEms7tx+cOrjjZASvmbvijVUQG4BVZHyPNaD\\\\\\\/5cCaozG+rY\\\\\\\/BeE9TF7PeO2YNKwUG4zys0w0ZC3Q0bxHta6yafcc3mcY93RZ6\\\\\\\/qoO5Ezqhw9oFknYzoG3WfSkUYZMCkSJ6yC7rB5mz\\\\\\\/Jukh0vkpfKbmsmv5\\\\\\\/BbiJAfx9NMdijkViGFKcqneSqMhIw55SRsHP7Om3VtRuK6XwBE4aWCrGyg1NavcYnXvXZ+JHyDD1DvbDXYxOLkzNen4B4wLCpnCkYdNpuLGFIBZgpWsqSJUwIXn8GWWd3Bk7ussM6tavIla+VM2\\\\\\\/S2InBSwpmdPI8bSH+FO1YiC6GEwOJ0nisqqy3xuKMeO\\\\\\\/GX2kHbXLJvfXurriVfj67x3NbBC1hiPvkqve55K6BaHefJqjwb6lgpfvd776CQZsXVm3l02xdY6U6sOrLBDxcEDWqbamgY7gEI7iH4G\\\\\\\/MEYTjlTGOTcWWfcct2z3nJOg+JWulkdkVNkG5tYeBLp6gp7XXTFU1D\\\\\\\/NMJrk004qE\\\\\\\/9bh3cYH3LghlofpY85wXCdDMXkNXAcUfxuj8teyPyj6GajPQemKqNC6Cpna3Itl7MkwbK06baXn\\\\\\\/PpMLEXeG\\\\\\\/6eALywgbiPBW641t5U7T8NzNOPhdZX5Xd1XquVbDaJSzO541\\\\\\\/exH4bE8l9mxbzEQnrEpCbvy\\\\\\\/TMHrioQ1VSq7f5PZ0LM2XbhY4a4AFg3PcQJlqYGOKQef9Gc3Wwz5Xv9q8Lvj4iwaPBRFFXqaVHkcrcq+4n11Z4eZkXa3aRc92Qw\\\\\\\/pZ1akXuc9lztZUQsA+EitZ5L58OsR1i217P2eHZZ+Xe+UhhoUKylmg1TyKj9cOIuCvpaQLVZsN3kjfwGtMK\\\\\\\/xpJG+nf\\\\\\\/eHaqmEApZDrGo08Vrplg\\\\\\\/K0cPkRQcYHLWW+q+KCZZCtvIC6n6sUOHoST0uLKSmr5VkZLGIDzhliOvLv4vzYVk4K3f8A3peN88y+gX68JJ0JctHnvPsE4XbpP1j5mt92KajqoA67aXVRSA3gHLKLXwuwBg2EbxqrweMP+\\\\\\\/TMyxlv29p7oOu7v83FRfy\\\\\\\/Zg1ppxGMUkW6dFXUEjLYK+Ubmm2mU1ULFMAFIuKN6lDcgbCkMf2LTBrgc6TrIhIMezjDEIIEf9SNHZtEtbsad6DNuh4wwDL6181qKS8f9WNBYwQliYKtIxkWkRCD1PCIkn7tA+Wc1zQ\\\\\\\/Pu6ZnmjaLi6NrQjY9WACHK3PNfRtCcafs8vIOk6E6IowIs8xn3\\\\\\\/C4MtcAjYzArK6EFz\\\\\\\/g8RY5I5WAN87rd6hPtnWvADvtnvvZ7JpI2Atu2mNKjCQPs7qijmezU8bA0KCskqcifIT7\\\\\\\/FIQRowkORKIh8e2uwVKgh0tG6jk71LcDvqmpi1Cesz1VbMvC6zx2\\\\\\\/Dvl4pY7AUY5beE\\\\\\\/cOeTLK2sZeO\\\\\\\/uOIFfVJRNOIdhsn9Szr4RFv1Hw+Z0WkUwmZd+C8APl38jFMshbFG+5fSRh8XjYnEZ7YypfPjujuZ8aiITDNCcbOWiwZZBF6r8ztcvg2p9enrSML+q7WA4R1irJrUdHxOehtMeRSlWnIOueVAZLRG6\\\\\\\/EFoWXziI++5z3Yuirtme6XGIPG1yt+Okl7KF4hVFyvx5UnnXcJ7e0f8F2XWFfRHAKZltoXbyjB92CMs4E6n8SstJ\\\\\\\/4geTVToswKNhFWegU5vLNUDtRxfW4dtb4mguypNs60wperDbTgYsPbDNg8vDDgN56L\\\\\\\/6rlgYW8W8g\\\\\\\/1k0zNj3EAzTD1swnXCLKsDMYqTbZqzasRMlaTsPVsQh7UnX+uT06f\\\\\\\/C6HTnl4suQ9B1eOsNWC9J\\\\\\\/0m4Khi09cLvIxpQ3\\\\\\\/1pNxf6WofWct2az1iLj7hMbPBoZfu22zb7cbZAFMGrVaHpgFqhOCFHyMlfaRFRGu\\\\\\\/BSo5Ynlx2nc8EkMciVKQEX34v1N\\\\\\\/lEUP5fg\\\\\\\/HHoWAE4MzrZy6QrEHAUGaXFOSCRIFQfTBgKNYwjO7WYq1nHe\\\\\\\/0VO9K16iTYe9N1AtSyA7\\\\\\\/facednp9VuBKbA8l8KACKR44xT+U6rVzII6yjjzJiwA1WR+qydk0zd9TPTtL0Rfvu\\\\\\\/K4U1TUMEtHzvejAUv8vFYBBf++EoGcrT9Trbnm5wer2EMxX+x50i+W5xREoDvucsZOVJBE\\\\\\\/92yZ130C+lk6OFp17bETUIb3kG0iCNU+nzWn83cuVgJylDDYCvH0B41K94EykUpfXN37OOZC0aAqAbIWbO3ITsVSCwHOjsRY2yakFYCIIwQS8jhRpIzIs0BM\\\\\\\/ux+P1jlKKOGCB+Y9HKRWXallopAjZkbdcWoyzGg6Y\\\\\\\/ACVP0RsKnErOl4QjEd4gH9k53vyO7jz2lFleSJz+dXn6EosrRhZHfLm0JI\\\\\\\/AA3lXNvJeJzaTYFMOzfNzGOQo6B4dnhPLFye4fbXNW2YCyg+rQ8Z4Ye46UPOmOIYjL8Q7qf+poYrNL5XynTXxZz5DV9pRLePsMyB\\\\\\\/FHkRXss3B6C2UK+ZXAE2jM3v6W6bkBL8Ce5\\\\\\\/\\\\\\\/bVQ3YXQOcOCe7uGqfVx+ACdgrBLsj1TycYGRtyhoaFAf8zy5NZqxlLmNsBb\\\\\\\/XBkxVcuzVspP+nkdDCpp1Jdhla5inyuKIoS8SYsBY5+wVcHnLDntCuK8+vr5X78nFL8WhulkrDsIUIKWTzJyVqAdP7PuoXHdnYi0r9cbnRArT0QZdYJm6cF4dLPLEvYsKsG6RSoEzXjQuKXe4uAv+G14hURC5bXa\\\\\\\/JIkLrpp1Vs3s5KH7mFQYdCArm1x9j0t+7Qylu5fvJIZrjTQjGTnMwVB8\\\\\\\/1HVJREezpZUm0zPVmNHzhLwhWJ+SIUu+N7PLnbZhcchrJsK\\\\\\\/gXjeWS4rGbkNT7Mx2grC4XiA7ynA\\\\\\\/JTbkwnhjfUaxkKWtX6Dc6h4FCxVDXD4iZOKsaJp6+xSmypEW4M0OH\\\\\\\/gyX36P0Q\\\\\\\/f7+MvJSi2eguShdlr1VwEJ4Gd3Ez0jD\\\\\\\/7POrCmcSCxJ\\\\\\\/iEdB9J5vSOq2cVG82a7Yec6e4+tTN2URbH2v31igNbAFiSQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00223vigQUoKn51CISz9AzoR9g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f4524e24c1e1fa943f3f7a9e2f1421b928231ec3948e619a8adda992ad970d77\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Xh714ofABDcm4bUEHbwQ7WLmy6kfQqQpTssvRPWyqRzBY3b5asgiBkRefDo71D3KKybrBIKmdA68V8aXjsnFGCZPV5LrxIQTYuUM6EQY7adqqsYZRnN5NXuwOYPNpQK8Zpspd0vZn4rEFwfG0kZAZkG9qR2ijWa68krLd8qwUKn2OXbxH5aztZAhAY7s99dCMyAGnyidmIzLDS0j\\\\\\\/bF5zSYHosZ+\\\\\\\/N+VHybfmsgDeP3gcLUJAH7sNnQt1SR0r0GMrkdxobv+W1WNh5aIb4voCL4nCVbAnoZdecynb1ROjsGb48lPc\\\\\\\/XVAmmvTbp5GjXT3935pBOquRXAxdxv\\\\\\\/RLhKCQR5+cGJKJpGjzAySsT7SVJoNS33DkbWOJGsihheCw5yV3gL3Iksfbfd2TbK08YfrJ8GW2MgEGuuT16sgiXO+MAKbflUWLJlvIwmONI\\\\\\\/rykmqNHao3cV1paBhR5ZqjSFf8uwHQNKSFH6gJwAXTygbL6CXtn1YvEDaq1I+CK10Q8rQ29mYOt30uFMLsBF9mJn0XnuD7DlKGyDXhu6oOTcH4w+IOwrCwy\\\\\\\/CEwEittISpm7HrH3yzGtDJOqphzdG9keJplJs3IWBBZwMUdlOaDY4zyTXcam77sQUtOFjAQfgoirewoEC8U8KCWFGg1SyBKD+7+Ib0ddo98uogdq894YP8cbYw\\\\\\\/lsul8UMr+SgTuzFkElOfSGlQ3WLCxiUCfgwxf4B6GPhATeOOrEy0WY4In0a\\\\\\\/5TCVeKfFRui8+JkTCS8knbW5MNfL5KiUAaF6YgreABj4L9S022t9Xr4H0I79OcjlynX0Q\\\\\\\/FOnMGnWNuUEi0yyEdkkeC3hLr6xdmKj7r9F\\\\\\\/ev9t4E7QX1FhFOtQ23MLgV5JPo2uazZctJPoU+ryWShNibJXZOD3jnFo63p8epVP4AwdapSbq1G3Ekp6vLmgrPC1N0g5m4ScIiB5Wl7mIKmqc9t2DsSridO5KxCcNgm0m0KbfxEdfMLi58WKHV733TaEm4LuRag\\\\\\\/Te9B5fAPF+Qq3yiSzq6sMtzI\\\\\\\/X0FqvF\\\\\\\/cAPZm97zW1hqUiGgzTSOKQ5ruMYsee+H0bxB7KZAImFqY\\\\\\\/AzMmSmHo4QG4fYITVyauXpOicCzIfP0tyR2fq\\\\\\\/xsjpfiX5aydEsz+9dCQAYCUL3bHt1Kr7O6Ciqiuo78df2L7DkNvb4YYxhfrISeD9yEqchfiyIGAYvclutIPMxaqD1w84dqnO3hR1ZuBPMG8n\\\\\\\/DwNocH7W5u+sLa6GX0ThsK72x\\\\\\\/8GsOMR\\\\\\\/W\\\\\\\/TjnQ63paVR4TcxyVP08Pqkyt\\\\\\\/Zr18BCWDSUQzfS8b9y1I3OimqV+bCi1HOGk94k2KPvOliAaP+fR4kINVVOztB7lgJS7iZzlkpJwrJa6sXseL0kkRC\\\\\\\/IN6ykCvU+7VExbIiYPwiFIkszlDtIjZaDBRfE6lsls7EZ9YBo9oI0j+IhlGY2WL9hIgf+\\\\\\\/WVzqs8ZtbzIx+CunzrfwC2mQmLAx0mOPZ2PmQYLD5kIca9NXuRbuuxird0r3RTKWU3c9AJDOjTH3AkSqhaXBrxfkhP1bG+Qwog0XTH75JI7vr6ojb1n2DR93NaBG7EvaCF8R7VxfUlsSPRj\\\\\\\/nSrdnBstglC2GHnFFnhUfUlwShrW\\\\\\\/QNXpQAusrcQBCrPEycLG4Ta2I46hDjXkF1V4BtFc1hqTAhdpVYx5tJtR5ODO\\\\\\\/KlMrl3weMDXEOJozmJNPXxOm\\\\\\\/TLWhY+NcjVhwYJ4l7xSvxtg9ca3\\\\\\\/9uXp1fIPMrxKPECw8alJJ1nE3rvBnDfTRlI2o\\\\\\\/lvUVbD7Y7S3ZUqBbnv8NfPGVYYAZyRPtxJ7umNT6FYpvqjRQna10Wndb+o4Zridg1HCzCin2ir\\\\\\\/+t8AwFUdGqz6ZNf8EZOymZxVYsPzR+tm4FtG9qfj3xU\\\\\\\/eIs3XJQ0AXpccPiytZZna47kYwe+eLL\\\\\\\/X+vu4feCMqA2ABvoCWvRiW6Rj0sYFDMwWF3\\\\\\\/WOm\\\\\\\/SyLUs2zbXt0ahxVLu\\\\\\\/fHX7knVElIiBA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kXjnQrEM4L3ndjP8G7Ix0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e1c28805ffe42cbd8555aa1086857d1743cb23c837c13bed081d6ddfe4cefdd5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459657 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1635" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LS7muuc4yjhhDf3qCmxFF2g6qzUmgUWfbLDJszMO2kA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1635" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fRW6h7gvTaMtiZhfmAZFVMHEdhGJpiAvpEJFzCykzGw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1623" title="Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1647" title="The Functional Significance of Common Polymorphisms in Zinc Finger Transcription Factors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459657" id="node459657" data-pisa="ggg;4/9/1635" data-pisa-master="ggg;g3.114.012757" data-apath="/ggg/4/9/1635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0002-7862-1507" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ricardo Wilches</span>, <span class="highwire-citation-author" data-delta="1">Susanne Voigt</span>, <span class="highwire-citation-author" data-delta="2">Pablo Duchen</span>, <span class="highwire-citation-author" data-delta="3">Stefan Laurent</span> and <span class="highwire-citation-author" data-delta="4">Wolfgang Stephan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1635-1645;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012757 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459657" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ricardo Wilches </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ricardo%2BWilches%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilches%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARicardo%2BWilches%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0002-7862-1507" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ricardo Wilches</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wilches@bio.lmu.de" class="" data-icon-position="" data-hide-link-title="0">wilches@bio.lmu.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Susanne Voigt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BVoigt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voigt%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BVoigt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pablo Duchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BDuchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duchen%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BDuchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stefan Laurent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BLaurent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurent%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BLaurent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wolfgang Stephan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BStephan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BStephan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1635" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459657" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459657/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1635.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459657" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459657/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1635.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459657" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459657/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1635.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459657" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459657/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1635.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459657" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459657/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">There is a growing interest in investigating the relationship between genes with signatures of natural selection and genes identified in QTL mapping studies using combined population and quantitative genetics approaches. We dissected an X-linked interval of 6.2 Mb, which contains two QTL underlying variation in chill coma recovery time (CCRT) in <em>Drosophila melanogaster</em> from temperate (European) and tropical (African) regions. This resulted in two relatively small regions of 131 kb and 124 kb. The latter one co-localizes with a very strong selective sweep in the European population. We examined the genes within and near the sweep region individually using gene expression analysis and <em>P</em>-element insertion lines. Of the genes overlapping with the sweep, none appears to be related to CCRT. However, we have identified a new candidate gene of CCRT, <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em>, which is located just outside the sweep region and is inducible by cold stress. We discuss these results in light of recent population genetics theories on quantitative traits.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cold-tolerance" class="hw-term hw-article-keyword hw-article-keyword-cold-tolerance" rel="nofollow">cold tolerance</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/fine-mapping" class="hw-term hw-article-keyword hw-article-keyword-fine-mapping" rel="nofollow">fine-mapping</a></li><li class="kwd"><a href="/keyword/selective-sweep" class="hw-term hw-article-keyword hw-article-keyword-selective-sweep" rel="nofollow">selective sweep</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/brinker" class="hw-term hw-article-keyword hw-article-keyword-brinker" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">brinker</em></a></li></ul><p id="p-4">Quantitative genetics assumes that selection on an adaptive trait involves simultaneous selection at multiple loci contributing to this trait. This causes small to moderate allele frequency shifts at these loci (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barton and Keightley 2002</a>). Therefore, adaptation does not require new mutations in the short-term. Instead, selection may use alleles that are found in the standing genetic variation (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pritchard and Di Rienzo 2010</a>). Genome-wide data suggest that this quantitative genetic view is relevant (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>). In particular, association studies confirm that quantitative traits are typically polygenic.</p><p id="p-5">However, there is the view that the molecular population genetics scenario of selective sweeps is also important in describing selection on quantitative traits. These sweeps may be caused by new mutations or low-frequency alleles from the standing variation. Empirical evidence for the occurrence of sweeps at QTL has been reported in studies of artificial selection, including the domestication of chickens (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rubin <em>et al.</em> 2010</a>), dogs (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Axelsson <em>et al.</em> 2013</a>), pigs (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rubin <em>et al.</em> 2012</a>), and cattle (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Qanbari <em>et al.</em> 2014</a>). Furthermore, there is growing evidence of sweeps associated with positive directional selection on quantitative traits in natural populations. <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Linnen <em>et al.</em> (2013)</a> studied coat color adaptation in deer mice controlled by a single large gene that shows multiple signatures of sweeps. Incomplete sweeps in the enhancer region of the gene <em><a href="http://flybase.org/reports/FBgn0000527.html">ebony</a></em> have contributed to the darker color of the abdominal segments of high-altitude <em>Drosophila melanogaster</em> from Uganda (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pool and Aquadro 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rebeiz <em>et al.</em> 2009</a>). Sweeps have also been observed at the EDA locus in three-spine sticklebacks associated with the reduction of lateral plate armor in fresh water environments (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cano <em>et al.</em> 2006</a>). In the common sunflower, selective sweeps have revealed candidate genes for adaptation to drought and salt tolerance (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kane and Rieseberg 2007</a>).</p><p id="p-6">Sophisticated methods have been developed to detect selective sweeps in a genome (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Thornton <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pavlidis <em>et al.</em> 2008</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stephan 2010</a>). In this study, we utilized these methods in combination with quantitative genetics tools to dissect a QTL interval for cold tolerance in <em>D. melanogaster</em>. Cold tolerance has been shown to be driven by environmental selection (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hoffmann <em>et al.</em> 2002</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Schmidt <em>et al.</em> 2005</a>) and to have a highly polygenic basis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Morgan and Mackay 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Norry <em>et al.</em> 2008</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> 2011</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>). In a previous analysis, using strains from African (tropical) and European (temperate) regions, we have identified X-linked QTL controlling chill coma recovery time (CCRT), a proxy for cold tolerance (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> 2011</a>). Here, we chose one interval of size 6.2 Mb, which contains two QTL shared between both sexes, and fine-mapped it using quantitative complementation tests. This resulted in a 131-kb region and a 124-kb region. In the European population, a strong selective sweep co-localized with the 124-kb region. We then analyzed this region in detail using population genetics and gene expression analyses. We found that the genes within the selective sweep region are probably not related to the trait, but a gene (<em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em>) just outside the sweep is induced by cold stress.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly lines</h3><p id="p-7">To conduct quantitative complementation tests on chromosomal deletions, a set of available deficiency lines were ordered at the Bloomington stock center (<a href="http://flystocks.bio.indiana.edu">http://flystocks.bio.indiana.edu</a>). Although the QTL interval was defined by <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> (2011)</a> to be 6.2 Mb long (6C to 11D), the set of available deletions spans 5.8 Mb of its total length between coordinates 6,642,419 and 12,461,494 that correspond to cytological bands 6C to 11B (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This 5.8-Mb-long interval is covered by a total of 24 overlapping deletions with known breakpoints at the sequence level in 92% of the cases. Additional deficiencies were tested if necessary.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of tested deletions within the QTL interval undergoing study. All deletions, represented by green or blue bars, span a 5.8-Mb fraction of the 6.2-Mb-long interval of interest on the X-chromosome. Deletion breakpoints at the base pair level are known for all deletions except Df(1)HA32 and Df(1)C128, for which only cytological bands are reported. Both physical and cytological coordinates are provided. The 24 deletions represented in green represent the minimum available set spanning the 5.8-Mb QTL interval; deletions in blue were tested on failure to complement of one of the overlapping deletions in green. Fractions of the QTL interval with light gray shading indicate regions of interest under deletions that show failure to complement. Red borders of this gray background indicate highly significant failure to complement (P &lt; 0.01)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1991845272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Map of tested deletions within the QTL interval undergoing study. All deletions, represented by green or blue bars, span a 5.8-Mb fraction of the 6.2-Mb-long interval of interest on the X-chromosome. Deletion breakpoints at the base pair level are known for all deletions except &lt;em&gt;Df(1)HA32&lt;/em&gt; and &lt;em&gt;Df(1)C128&lt;/em&gt;, for which only cytological bands are reported. Both physical and cytological coordinates are provided. The 24 deletions represented in green represent the minimum available set spanning the 5.8-Mb QTL interval; deletions in blue were tested on failure to complement of one of the overlapping deletions in green. Fractions of the QTL interval with light gray shading indicate regions of interest under deletions that show failure to complement. Red borders of this gray background indicate highly significant failure to complement (&lt;em&gt;P &lt; &lt;/em&gt;0.01).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1635/F1.medium.gif" width="440" height="155"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1635/F1.medium.gif" width="440" height="155"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Map of tested deletions within the QTL interval undergoing study. All deletions, represented by green or blue bars, span a 5.8-Mb fraction of the 6.2-Mb-long interval of interest on the X-chromosome. Deletion breakpoints at the base pair level are known for all deletions except <em>Df(1)HA32</em> and <em>Df(1)C128</em>, for which only cytological bands are reported. Both physical and cytological coordinates are provided. The 24 deletions represented in green represent the minimum available set spanning the 5.8-Mb QTL interval; deletions in blue were tested on failure to complement of one of the overlapping deletions in green. Fractions of the QTL interval with light gray shading indicate regions of interest under deletions that show failure to complement. Red borders of this gray background indicate highly significant failure to complement (<em>P &lt; </em>0.01).</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">The African and European versions of the X-chromosome used in the complementation tests are contained in fly lines A* and E*, created by introgressing one X-chromosome from a population of Zimbabwe and one from the Netherlands into a common laboratory strain (<a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> 2011</a>). Hence, these two lines bear different X-linked alleles while the rest of the major nuclear chromosomes and mitochondrial DNA are the same. These two lines are the parents of an X-recombinant inbred population used to localize the QTL interval that concerns us in this project.</p><p id="p-10">Prior to CCRT scoring experiments, virgin female flies bearing the deficiency chromosome and the respective balancer were mated with males of the A* and E* lines, respectively. Eggs were allowed to develop in the same medium in which they were laid at 23°. Female F1 were sorted on hatching by phenotype as balancer or deletion bearer. Because all balancer types used to maintain the deletions have a dominant mutation for eye shape at the <em><a href="http://flybase.org/reports/FBgn0000154.html">Bar</a></em> (<em><a href="http://flybase.org/reports/FBal0000817.html">B<sup>1</sup></a></em>) locus, F1 flies exhibiting the <em><a href="http://flybase.org/reports/FBal0000817.html">B<sup>1</sup></a></em> mutant phenotype were considered as balancer bearers, whereas wild-type appearance was indicative of bearing the deletion. Sorted flies were kept at room temperature until CCRT scoring on their fourth to sixth day of life.</p><p id="p-11">Assessments of expression levels of candidate genes were conducted using 4- to 6-day-old female flies belonging to the Netherlands population (isofemale lines: NL01, NL12, NL14, NL15, NL16, NL18, NL19, NL20) and the Zimbabwean population (isofemale lines: ZK84, ZK131, ZK145, ZK157, ZK186, ZK229, ZK377, ZK398). Flies were reared at 23° and subjected to cold stress in the same manner as reported for CCRT scoring. Three flies per line were used as controls (not exposed to cold). Three flies of each line were snap-frozen in liquid nitrogen at 10 min after being brought to room temperature, whereas three remaining flies per line were scored for their CCRT and frozen 15 min after the minute in which they were reported as recovered. Control flies, which remained at 23° in glass vials during the 7 hr of treatment, were also snap-frozen at the end of this time period. Frozen material was stored at −80° until RNA extractions were performed. Population pools per line/treatment were made prior to RNA extraction. Each population pool per treatment consisted of eight flies of the same population. Three population pools per treatment were made for both the Netherlands and Zimbabwe.</p></div><div id="sec-3" class="subsection"><h3>CCRT scoring</h3><p id="p-12">Once flies reached 4 to 6 days of age, they were scored for CCRT following the protocol of <a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> (2011)</a>. Briefly, flies were transferred to glass vials without anesthesia and placed in an ice-water bath of 0° for 7 hr. At the end of this time period, flies were brought back to room temperature (23°) and observed in time intervals of 1 min. The minute in which a fly was standing on its feet was recorded as its CCRT.</p></div><div id="sec-4" class="subsection"><h3>Quantitative complementation tests on deficiencies</h3><p id="p-13">On average, 35 female flies per each of the four resulting genotypes <em>E*/def</em>, <em>A*/def</em>, <em>E*/bal</em>, and <em>A*/bal</em> were scored. For ANOVA analysis, log-transformed CCRT scores per genotype, line (<em>L</em>), and genomic background (<em>G</em>) were kept as fixed effects. We focused on the significance of the interactions of these two factors (<em>L</em> × <em>G</em>) as well as on the following two conditions to call the procedure quantitative failure to complement: the differences in CCRT for the genotypes bearing the balancer had to be small compared with that of the genotypes bearing the deletion; and in the latter case, the <em>E*/def</em> flies should show reduced CCRT with respect to the <em>A*/def</em> genotypes. It is expected that if these conditions are satisfied, failures to complement due to the presence of other QTL outside the region in question can be ruled out. However, a failure to complement detected with a given deficiency either can be caused by its interaction with QTL alleles in the region uncovered by a deficiency (allelism) or arise by interaction between this deficiency and QTL alleles elsewhere in the genome (epistasis) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pasyukova <em>et al.</em> 2000</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Service 2004</a>). Because we are interested in the allelic cases of failure to complement by using the E* and A* lines (as well as its inbred wild-type progenitor lines) in the tests, we control for most of the epistatic effects that can be caused by loci residing on chromosomes 2 and 3. Bonferroni correction was applied to control for multiple testing.</p></div><div id="sec-5" class="subsection"><h3>RNA extraction and cDNA synthesis</h3><p id="p-14">RNA was extracted from pools using the MasterPure RNA Purification Kit (Epicentre Biotechnologies, Madison, WI), followed by DNase treatment. Purified RNA was quantified with a nanodrop apparatus and tested for genomic DNA contamination based on a PCR (Phusion) protocol using a primer pair binding in nontranscribed regions of the X-chromosome (Primer code: X-1435; sequence available on request). Samples tested positive for genomic DNA were excluded from downstream protocols. cDNA synthesis was performed with SuperScript III Reverse Transcriptase (Invitrogen, Carlsbad, CA) on 1400 ng of RNA per reaction.</p></div><div id="sec-6" class="subsection"><h3>RT-qPCR assays</h3><p id="p-15">RT-qPCR assays for candidate genes <em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em>, <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em>, <em><a href="http://flybase.org/reports/FBgn0029942.html">CG2059</a></em>, <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em>, <em><a href="http://flybase.org/reports/FBgn0024250.html">brk</a></em>, and <em><a href="http://flybase.org/reports/FBgn0029943.html">Atg5</a></em> were performed with primers designed using the online tool QuantPrime (<a href="http://www.quantprime.de">www.quantprime.de</a>) to match all possible transcript types per candidate gene. The ribosomal genes <em><a href="http://flybase.org/reports/FBgn0019936.html">RpS20</a></em> and <em><a href="http://flybase.org/reports/FBgn0002626.html">RpL32</a></em> were taken as reference genes, against which relative gene expression levels of our genes of interest were normalized. RT-qPCR assays consisted of a total of three biological replicates each run in triplicate and were conducted on a Real-Time thermal cycler CFX96 platform (BioRad, Hercules, CA). Each reaction was taken to a final volume of 10 µl using iQ SYBR Green Supermix (BioRad, Hercules, CA). Further details of the experimental setup, such as amplification efficiency assessments with dilution series, can be provided on request.</p><p id="p-16">Obtained Cq values per replicate within line (or pool) and treatment were transformed to calibrated normalized relative quantities (CNRQ) following <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hellemans <em>et al.</em> (2007)</a>. Log-transformed CNRQs were then used to test the hypothesis of expression differences between pairs of lines (or pool) within and between treatments. For this purpose, Welch two-sample <em>t</em>-tests were performed on comparisons with fold differences above a threshold (defined by the variance between technical replicates). The <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Benjamini and Hochberg (1995)</a> <em>P</em>-value correction was applied to control for false positives, due to the high number of simultaneous tests performed.</p></div><div id="sec-7" class="subsection"><h3>Basic population genetics analysis</h3><p id="p-17">Molecular variation was characterized in the genomic fragment uncovered by the deletion <em>Df(1)ED9606</em>; <em>i.e.</em>, a total of 124 kb between coordinates 7,089,000 and 7,212,999. Publically available whole-genome sequences generated by Illumina NGS technology for four <em>D. melanogaster</em> populations were retrieved from the Drosophila Population Genomics Project (DPGP) at <a href="http://www.dpgp.org">http://www.dpgp.org</a>. The populations include the Netherlands (NL) with 11 lines (NL01, NL02, NL11, NL12, NL13, NL14, NL15, NL16, NL17, NL18, NL19), France (FR) with 8 lines (FR14, FR151, FR180, FR207, FR217, FR229, FR310, FR361), Rwanda (RG) with 25 lines (RG2, RG3, RG4N, RG5, RG6N, RG7, RG8, RG9, RG10, RG11N, RG13N, RG15, RG18N, RG19, RG21N, RG22, RG24, RG25, RG28, RG2, RG32N, RG33, RG36, RG37N, RG38N), and Southeast Africa consisting of a pool of 18 lines from Zambia (ZI91, ZI261, ZI268, ZI468, ZO12, ZO65), Zimbabwe (ZK84, ZK131, ZK186, ZS5, ZS11, ZS56), and Malawi (MW6, MW11, MW28, MW38, MW46, MW63). The corresponding <em>D. simulans</em> sequence (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hu <em>et al.</em> 2013</a>) was used as outgroup.</p><p id="p-18">The following quality-control steps during the initial handling of the sequence data were used. First, nucleotides with a PHRED score lower than 21 were set to N. Unless otherwise stated, this quality criterion was applied to all analyses in which DPGP sequence data were used. Second, if a given polymorphic site in the alignment showed a frequency of N higher than 10%, then it was excluded from the analysis. The following summary statistics were then computed on 2-kb-long nonoverlapping windows: θ<sub>π</sub> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tajima 1983</a>), θ<sub>W</sub> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Watterson 1975</a>), and divergence (<em>D<sub>xy</sub></em>) to the outgroup (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nei 1987</a>). In addition, pairwise <em>F</em><sub>ST</sub> was calculated as normalized distance of Nei (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nei and Li 1979</a>). Neutrality tests based on the site frequency spectrum (SFS) using the <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tajima (1989)</a> <em>D</em> statistic were also calculated.</p></div><div id="sec-8" class="subsection"><h3>Composite likelihood ratio test of positive selection</h3><p id="p-19">To investigate whether the observed SFS in the region of interest is compatible with the one expected after a selective sweep, we calculated the composite likelihood ratio (CLR) statistic (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kim and Stephan 2002</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nielsen <em>et al.</em> 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pavlidis <em>et al.</em> 2010</a>) as it is implemented in the software SweeD (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pavlidis <em>et al.</em> 2013</a>). This likelihood ratio test statistic compares a selective sweep model and a neutral model that is calibrated with the genomic background frequency spectrum. We used the parallel version of the software (SweeD-P) to calculate the CLR statistic along the X-chromosome in our European sample (19 lines from the Netherlands and France). In addition to the classes of the SFS (<em>i.e.</em>, 1 to <em>n</em>-1, where <em>n</em> is the sample size), we considered two additional site classes consisting of sites that are monomorphic in the European sample and polymorphic in the Rwandan sample. Extending the SFS in this way was shown to improve the power of the method to detect selective sweeps (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Nielsen <em>et al.</em> 2005</a>). SweeD was run on a 16-core CPU using the command line option “- -monomorphic” with 500,000 grid points. The background SFS was taken from the complete X-chromosome. However, following <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pool <em>et al.</em> (2012)</a>, we excluded from the analysis telomere and centromere regions of the X-chromosome due to their very low recombination rate. The coordinates of the excluded regions range from the origin until position 2,222,391 for the telomere and from position 20,054,556 to the end for the centromere region. Finally we compared the CLR profile of our region of interest to the profile calculated for the complete chromosome.</p><p id="p-20">The significance level of the CLR-test statistic was calculated by simulating large genomic regions with the coalescent simulator fastsimcoal2 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Excoffier and Foll 2011</a>) under a neutral model that takes into account our current knowledge of the demography of European populations of <em>D. melanogaster</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Laurent <em>et al.</em> 2011</a>). For every one of the simulated datasets, we computed the CLR-test statistic in the same way as we did for the observed dataset and recorded the maximum CLR value. We used the 95<sup>th</sup> quantile of the distribution of top CLR values as our significance threshold. Because this analysis becomes computationally intensive as the size of the simulated genomic region increases, we investigated the relation between the threshold value and the size of the simulated region. We simulated batches of 100 datasets of increasing size from 50 to 5000 kb in length and took the asymptotic value reached as the chromosomal threshold.</p></div><div id="sec-9" class="subsection"><h3><em>F</em><sub>ST</sub>-based scan for positive selection</h3><p id="p-21">For the set of F<sub>ST</sub> analyses performed with BayeScan (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Foll and Gaggiotti 2008</a>) (<a href="http://cmpg.unibe.ch/software/bayescan/">http://cmpg.unibe.ch/software/bayescan/</a>), input files were prepared following the authors’ instructions. The different runs were performed using default parameters with the sequence data of the same DPGP samples from the Netherlands, France, Rwanda, and Southeast Africa. In addition, lines from the following African populations were included: Ethiopia (ED2, ED3, ED5N, ED6N, ED10N, EZ2, EZ5N, EZ9N, EZ25), Cameroon (CO1, CO2, CO4N, CO8N, CO9N, CO10N, CO13N, CO14, CO15N, CO16), and South Africa (SP80, SP173, SP188, SP221, SP235, SP241, SP254). SNP exclusion criteria were as follows: positions showing more than two segregating alleles as well as sites with less than 50% base calls in one population were excluded from the analysis.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-22">We first describe the results of the deficiency mapping and then those of the population genetics analysis and the gene expression studies.</p><div id="sec-11" class="subsection"><h3>Quantitative deficiency mapping</h3><p id="p-23">To dissect the QTL interval reported by <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> (2011)</a> corresponding to the interval at 6C-11D of approximately 6.2 Mb in length, we conducted quantitative complementation tests for 24 overlapping deletions spanning 94% of this interval. The chromosome fractions comprising the remaining 6% of the interval were left untested due to lack of suitable deletions. With the set of 24 deficiencies (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), we could potentially uncover the effect of line-specific alleles (line-specific refers to the type of X-chromosome involved in the test, which is either African or European) at 588 (95%) of the 622 annotated and computationally predicted genes within the interval. Overall, 14 of 24 of the tested deficiencies showed significant line effects at the 5% level, whereas 9 of 24 showed a significant effect of the genomic background on CCRT scores (with the term “genomic background” we refer to the involved deletion and balancer chromosomes; see <em>Materials and Methods</em>). We observed failure to complement in 4 of the 24 tested deletions (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Failure to complement implied both a significant effect of line (<em>L</em>) and a significant line by genomic background interaction (<em>L</em> × <em>G</em>) as long as the differences in CCRT followed the expected direction. That is, there were shorter CCRT times for flies bearing the E* X-chromosome in the presence of the deletion compared with the corresponding flies bearing the A* X-chromosome in the presence of the same deletion, but no difference was shown between the CCRT of the flies bearing the E* and A* X-chromosomes in the presence of the balancer chromosome.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464866/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464866/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Deficiency analysis of X-linked QTL affecting CCRT in female flies</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">Deletions <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> and <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em> were the only ones that revealed a highly significant failure to complement. In the case of <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em>, the difference between the means of the CCRT scores for the flies bearing this deletion is 9.18 min, whereas that of the flies harboring the balancer chromosome is 1.82 min (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This means that deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> has potentially uncovered E* X-chromosome alleles and/or alleles with the opposite effect residing on the A* X-chromosome. A similar interpretation can be given for the results obtained for <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em>, <em><a href="http://flybase.org/reports/FBab0045541.html">Df(1)BSC592</a></em> and <em><a href="http://flybase.org/reports/FBab0045429.html">Df(1)BSC537</a></em>, which also failed to complement, as evidenced by the significant <em>L</em> and <em>L</em> × <em>G</em> effects and by the higher CCRT differences in the presence of the deficiency than in the presence of the balancer. However, for the two last deletions, these effects were only marginally significant (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Thus, they are not considered for further study.</p><p id="p-26">Although our results with <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> and <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em> meet the requirements to be considered allelic failures to complement, these can also be interpreted as an epistatic failure to complement due to interactions of these deficiencies with other loci that affect CCRT residing elsewhere on the X or in the other two major chromosomes. We are aware of the limitation of quantitative deficiency mapping to tell these two causes apart (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Pasyukova <em>et al.</em> 2000</a>). This is also a problem in similar studies (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fanara <em>et al.</em> 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Luca <em>et al.</em> 2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Geiger-Thornsberry and Mackay 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Harbison <em>et al.</em> 2004</a>). However, because we used the E* and A* lines that share the same genetic background for their respective wild-type–derived X-chromosomes and not the wild-type inbred lines NL14 and ZK157, we can exclude interactions with factors located outside the X-chromosome.</p><p id="p-27">The fact that we used a set of overlapping deficiencies allowed us to better define the stretch that revealed quantitative failure to complement. With respect to the 210-kb-long deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em>, the 67.15 kb overlapping with deletion <em><a href="http://flybase.org/reports/FBab0045428.html">Df(1)BSC536</a></em> were subtracted from the stretch of interest (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Furthermore, the results of the complementation tests with yet another overlapping deficiency at the same end (<em><a href="http://flybase.org/reports/FBab0045780.html">Df(1)BSC711</a></em>) allowed us to subtract an additional 19.64 kb from the 210 kb encompassing <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). At the other end of deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em>, its overlap with deletion <em><a href="http://flybase.org/reports/FBab0000413.html">Df(1)HA32</a></em> is not known at the base pair level. Thus, the redefined fraction of interest under deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> encompasses 124 kb between coordinates 7.09 and 7.21 Mb. Similarly, for the other highly significant deletion <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em>, the redefined region of interest has a of length 131 kb (between coordinates 7.85 and 7.98 Mb).</p><p id="p-28">This quantitative complementation mapping approach based on overlapping deletions has allowed us to reduce the number of initial candidate genes within the QTL undergoing study from 622 to a subset of 21. A total of 7 genes are located within the 124 kb uncovered by deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em>, and 14 genes were uncovered by deletion <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em>. This is remarkable given the substantial fraction of uncharacterized genes in the 6.2 Mb of the QTL defined by <a id="xref-ref-58-7" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> (2011)</a> and the absence of known <em>a priori</em> candidate genes for CCRT in this chromosomal region.</p><p id="p-29">In the next section we show that of the two regions identified by the complementation tests, the 124-kb region uncovered by deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> overlaps with a selective sweep identified in several previous analyses (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Glinka <em>et al.</em> 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boitard <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>), whereas we did not detect evidence for positive directional selection in deletion <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em> (see below). Therefore, in the following section we focus on deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em>.</p></div><div id="sec-12" class="subsection"><h3>Molecular population genetics analysis</h3><p id="p-30">We characterized molecular variation in the genomic region of 124 kb uncovered by deletion <em>Df(1)ED9606</em> in natural populations of <em>D. melanogaster</em>. First, we calculated a set of summary statistics on a 2-kb nonoverlapping window basis using next-generation sequence data from two European (the Netherlands and France) and two African (Rwanda and Southeast Africa) populations. The Netherlands population and a set of Southeast African lines represent the gene pools from which the E* and A* lines were derived. The additional two populations consisted of French and Rwanda sequence data from the DPGP (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Pool <em>et al.</em> 2012</a>). These four populations allowed us to draw conclusions on patterns of variation in temperate and tropical populations.</p><p id="p-31">For each population we obtained nucleotide diversity estimates measured by the average number of pairwise differences (θ<sub>π</sub>) and Watterson’s estimator (θ<sub>W</sub>). The European populations showed a three-fold to four-fold reduction in nucleotide diversity when compared with the African populations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/012757SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS1.pdf">Figure S1</a> depicts the values of the 2-kb windows along the entire region of 124 kb in the four populations. The θ<sub>π</sub> values in the Netherlands and French populations are along a 40-kb fragment (positions 65,000 to 105,000) 1 SD lower than the average over the 124-kb region. This pattern is in contrast to that observed in the two African populations for the same coordinates, for which nucleotide diversity values tend to be above their respective population averages. <em>F</em><sub>ST</sub> estimates along the 124-kb region between each of the European populations and the Southeast African pool parallel the diversity estimates such that <em>F</em><sub>ST</sub> vales are higher where diversity is low in the two European populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS1.pdf">Figure S1</a>). Regarding divergence of each population to <em>D. simulans</em>, the averages of the two European populations are approximately 10% whereas those of the African populations have values between 12% and 13% (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464854/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464854/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary statistics average for the QTL undergoing study in four <em>D. melanogaster</em> populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">We also analyzed possible deviations from the standard neutral SFS using Tajima’s <em>D</em> statistic (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS2.pdf">Figure S2</a>). The African samples from Rwanda and Southeast Africa show generally negative <em>D</em> values that are typical for these populations (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Glinka <em>et al.</em> 2003</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Pool <em>et al.</em> 2012</a>). The European samples have average <em>D</em> values near zero and a larger variance, as was also found previously (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Glinka <em>et al.</em> 2003</a>). Most interestingly, however, is the 40-kb window from position 65,000 to 105,000, in which Tajima’s <em>D</em> has lower than average values (except for a peak around coordinate 87,000).</p><p id="p-33">The patterns of polymorphism observed in the region of interest in the European and African populations revealed a conspicuous reduction of genetic variability and a negative Tajima’s <em>D</em> in both European populations that extends for approximately 40 kb between relative position 65,000 and 105,000 in the 124-kb region. This reduction already has been identified as a footprint of positive selection in non-African populations (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Glinka <em>et al.</em> 2006</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Boitard <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>). In this work, motivated by the link to the QTL, we revisited these analyses.</p><p id="p-34">Because demographic scenarios (particularly bottlenecks) can create similar genomic patterns as positive directional selection (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stephan and Li 2007</a>), we subjected the data of the available European samples (pooling the Netherlands and French lines) to the most advanced composite likelihood ratio test (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Pavlidis <em>et al.</em> 2013</a>). This likelihood ratio was computed between a selective sweep model and a neutral model that is calibrated with the genomic background frequency spectrum. The background SFS was obtained from 20 Mb of the X-chromosome, excluding the telomere and centromere regions (see <em>Materials and Methods</em>). In our region of interest, the fragment between relative positions 65,000 and 105,000 exhibits a SFS that is in contrast to that of the genomic background and is better described by a selective sweep model (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The CLR values obtained for this interval (Λ<sub>CLR</sub> &gt;300) are within the top 1% of CLR values along the entire region of the X-chromosome analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS3.pdf">Figure S3</a>) and are above the significance threshold of 72 that corresponds to the 95<sup>th</sup> quantile of the top CLR values of 100 simulated subgenomic regions of 5 Mb. This value did not increase when larger genomic regions were simulated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS4.pdf">Figure S4</a>). Simulations were based on our current understanding of the demographic history of European populations (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Laurent <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Duchen <em>et al.</em> 2013</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evidence of positive selection and candidate SNPs in the 124-kb region under deletion Df(1)ED6906. (A) Likelihood (CLR) profile along the 124-kb on the X-chromosome using SNP data of two pooled European D. melanogaster from the Netherlands and France. Two significance thresholds are depicted. The solid line corresponds to the average of the top 1% CLR values for the X-chromosome in Europe and the dashed red line represents the significance threshold from simulations of equivalent subgenomic regions. (B) Model-based FST values for 7316 SNPs from a dataset including two European and five African samples (see Materials and Methods). Top SNPs (above the false discovery rate of 5%) are marked with position and a thin continuous line across panels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1991845272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evidence of positive selection and candidate SNPs in the 124-kb region under deletion &lt;em&gt;Df(1)ED6906&lt;/em&gt;. (A) Likelihood (CLR) profile along the 124-kb on the X-chromosome using SNP data of two pooled European &lt;em&gt;D. melanogaster&lt;/em&gt; from the Netherlands and France. Two significance thresholds are depicted. The solid line corresponds to the average of the top 1% CLR values for the X-chromosome in Europe and the dashed red line represents the significance threshold from simulations of equivalent subgenomic regions. (B) Model-based &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; values for 7316 SNPs from a dataset including two European and five African samples (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Top SNPs (above the false discovery rate of 5%) are marked with position and a thin continuous line across panels.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1635/F2.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/9/1635/F2.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464911" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Evidence of positive selection and candidate SNPs in the 124-kb region under deletion <em>Df(1)ED6906</em>. (A) Likelihood (CLR) profile along the 124-kb on the X-chromosome using SNP data of two pooled European <em>D. melanogaster</em> from the Netherlands and France. Two significance thresholds are depicted. The solid line corresponds to the average of the top 1% CLR values for the X-chromosome in Europe and the dashed red line represents the significance threshold from simulations of equivalent subgenomic regions. (B) Model-based <em>F</em><sub>ST</sub> values for 7316 SNPs from a dataset including two European and five African samples (see <em>Materials and Methods</em>). Top SNPs (above the false discovery rate of 5%) are marked with position and a thin continuous line across panels.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Using the same CRL test, we also analyzed in detail <em><a href="http://flybase.org/reports/FBab0000349.html">Df(1)C128</a></em>, the second highly significant deletion uncovered by the quantitative complementation test (between coordinates 7.85 and 7.98 Mb). However, we could not find evidence for selective sweeps (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-37">Because a large fraction of the region of low variation in Europe (particularly the coding regions of genes <em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em>, <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em>, <em><a href="http://flybase.org/reports/FBgn0037007.html">CG5059</a></em>, and <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em>; see the gene model below) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) contains no or very little variation, the CLR tests cannot be used to identify the targets of selection. Instead, we utilized genetic differentiation between African and European populations to obtain model-based <em>F</em><sub>ST</sub> coefficients (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Foll and Gaggiotti 2008</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Riebler <em>et al.</em> 2008</a>) for each SNP within the 124-kb region of interest (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). We considered SNP data from seven populations along a gradient across Africa and Europe: South Africa, Southeast Africa, Rwanda, Cameroon, Ethiopia, France, and the Netherlands. Using BayeScan (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Foll and Gaggiotti 2008</a>), we obtained a pattern of <em>F</em><sub>ST</sub> values from a dataset of 7316 SNPs with an average <em>F</em><sub>ST</sub> of 0.2621 and revealed four outlier SNPs that show the highest differentiation across populations at an FDR of 5% (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). These four SNPs are located within the 40-kb-long fragment enriched for SNPs showing significant CLR values between positions 65,000 and 105,000 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). The 65-kb-long and 19-kb-long flanking regions to the left and to the right of the 40-kb fragment, respectively, are enriched for SNPs showing below-average <em>F</em><sub>ST</sub> values (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>). However, none of these SNPs with low differentiation across populations is significant at the 5% FDR.</p><p id="p-38">The exclusion of European populations from the analysis did not change the pattern of high-differentiation outlier SNPs (results not shown). This suggests that allele frequency differentiation at outlier SNPs had already started within the African continent. Furthermore, because the European populations probably have experienced more severe bottlenecks than the African populations (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Stephan 2006</a>; <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Pool <em>et al.</em> 2012</a>), we may conclude that the BayeScan results are relatively robust against demographic changes and are not due to the strong bottleneck in the European population.</p><p id="p-39">Among the outlier SNPs that show high differentiation across the entire intercontinental dataset, the top ones are <em>86,661C /T</em> (<em>F</em><sub>ST</sub>=0.4697, α=2.02, q-value=0.0024) and <em>86,670T/C</em> (<em>F</em><sub>ST</sub>=0.4654, α=1.98, q-value=0.0042). These two nonsynonymous SNPs are located in exon 5 of the computationally predicted gene <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> and show alleles in perfect linkage disequilibrium (LD) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The TT haplotype (<em>86,661T–86,670T)</em> is in high frequency in the Southeast African samples and is intermediate in Rwanda; its frequency decreases with increasing latitude to be replaced in the European populations by the CC haplotype. Both SNPs predict changes in the amino acid sequence of the protein. The common Southeast African form of the protein codes for a threonine (Thr) and an asparagine (Asn) at residues 936 and 939, respectively, whereas the cosmopolitan form has an alanine (Ala) and aspartic acid (Asp) at these two positions. The third highly significant SNP is <em>80,089A/G</em> (<em>F</em><sub>ST</sub>=0.4146, α=1.54, q-value=0.0313) located between genes <em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em> and <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). Its allele frequency distribution across populations is also shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. Finally, SNP <em>101,154C/G</em> (<em>F</em><sub>ST</sub> = 0.4068, α=1.4804, q-value = 0.0481) is located 5 kb upstream of gene <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em> within the large intron of gene <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele frequency change at highly differentiated SNPs at the QTL of interest. (A) Allele frequencies of the top four highly differentiated SNPs across seven different populations along a latitudinal gradient. Populations are as follows: the Netherlands and France (EUR), Ethiopia (ED), Cameroon (CO), Rwanda (RG), Southeast Africa (SEA), and South Africa (SP) (see Materials and Methods). (B) European and Southeast African D. melanogaster haplotypes for the two nonsynonymous SNPs (86,661&#x2013;86,670) in intron 5 of gene CG1677. These two SNPs correspond to amino acid positions 939 and 936." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1991845272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Allele frequency change at highly differentiated SNPs at the QTL of interest. (A) Allele frequencies of the top four highly differentiated SNPs across seven different populations along a latitudinal gradient. Populations are as follows: the Netherlands and France (EUR), Ethiopia (ED), Cameroon (CO), Rwanda (RG), Southeast Africa (SEA), and South Africa (SP) (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (B) European and Southeast African &lt;em&gt;D. melanogaster&lt;/em&gt; haplotypes for the two nonsynonymous SNPs (86,661&#x2013;86,670) in intron 5 of gene &lt;em&gt;CG1677&lt;/em&gt;. These two SNPs correspond to amino acid positions 939 and 936.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1635/F3.medium.gif" width="440" height="289"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/9/1635/F3.medium.gif" width="440" height="289"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464973" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Allele frequency change at highly differentiated SNPs at the QTL of interest. (A) Allele frequencies of the top four highly differentiated SNPs across seven different populations along a latitudinal gradient. Populations are as follows: the Netherlands and France (EUR), Ethiopia (ED), Cameroon (CO), Rwanda (RG), Southeast Africa (SEA), and South Africa (SP) (see <em>Materials and Methods</em>). (B) European and Southeast African <em>D. melanogaster</em> haplotypes for the two nonsynonymous SNPs (86,661–86,670) in intron 5 of gene <em>CG1677</em>. These two SNPs correspond to amino acid positions 939 and 936.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Candidate gene expression analyses and complementation tests with <em>P</em>-element insertion lines</h3><p id="p-41">We observed that the CLR profile of the selective sweep does not overlap with <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> and <em><a href="http://flybase.org/reports/FBgn0029943.html">Atg5</a></em>, but spans four of the seven candidate genes in the 124-kb region (see gene model below) (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>). To analyze whether these four genes in the sweep region are related to cold tolerance, we conducted expression analyses; <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> and <em><a href="http://flybase.org/reports/FBgn0029943.html">Atg5</a></em> were also included (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). qPCR assays were performed on cDNA prepared from pools of female flies from the Netherlands and Zimbabwe (see <em>Materials and Methods</em>). Expression of candidate genes was measured at two time points after cold stress exposure as well as under control conditions. The two post-cold stress time points were 10 min after the end of cold stress and 15 min after flies recovered from chill coma. Controls consisted of flies of the same lines that were not subjected to cold stress.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of genes located in the region under deletion Df(1)ED6906 that was affected by positive selection (see Figure 2A). Relative expression was measured under two cold stress and control conditions in pools of flies from a temperate population [the Netherlands (NL)] and a tropical population [Zimbabwe (ZK)]. Expression levels of these candidate genes were normalized with respect to that of the ribosomal genes RpS20 and RpL32. The height of the bars represents the mean of three calibrated normalized relative quantities (CNRQ) per pool per gene rescaled to that of the corresponding ZK control. Error bars also represent rescaled confidence intervals. Levels of significance for tests of differences in expression levels among treatments within and between populations are indicated with asterisks: *P &lt; 0.05; **P &lt; 0.01; and ***P &lt; 0.001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1991845272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of genes located in the region under deletion &lt;em&gt;Df(1)ED6906&lt;/em&gt; that was affected by positive selection (see Figure 2A). Relative expression was measured under two cold stress and control conditions in pools of flies from a temperate population [the Netherlands (NL)] and a tropical population [Zimbabwe (ZK)]. Expression levels of these candidate genes were normalized with respect to that of the ribosomal genes &lt;em&gt;RpS20&lt;/em&gt; and &lt;em&gt;RpL32&lt;/em&gt;. The height of the bars represents the mean of three calibrated normalized relative quantities (CNRQ) per pool per gene rescaled to that of the corresponding ZK control. Error bars also represent rescaled confidence intervals. Levels of significance for tests of differences in expression levels among treatments within and between populations are indicated with asterisks: *&lt;em&gt;P&lt;/em&gt; &lt; 0.05; **&lt;em&gt;P&lt;/em&gt; &lt; 0.01; and ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1635/F4.medium.gif" width="440" height="134"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/9/1635/F4.medium.gif" width="440" height="134"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1635/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464951" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Expression of genes located in the region under deletion <em>Df(1)ED6906</em> that was affected by positive selection (see <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a>). Relative expression was measured under two cold stress and control conditions in pools of flies from a temperate population [the Netherlands (NL)] and a tropical population [Zimbabwe (ZK)]. Expression levels of these candidate genes were normalized with respect to that of the ribosomal genes <em>RpS20</em> and <em>RpL32</em>. The height of the bars represents the mean of three calibrated normalized relative quantities (CNRQ) per pool per gene rescaled to that of the corresponding ZK control. Error bars also represent rescaled confidence intervals. Levels of significance for tests of differences in expression levels among treatments within and between populations are indicated with asterisks: *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; and ***<em>P</em> &lt; 0.001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Of the six genes, <em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em> and <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> showed significant differences in constitutive expression levels between the Netherlands pool and the Zimbabwean pool (<em>P &lt;</em> 0.01) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This difference in expression levels between these populations also has been previously observed (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hutter <em>et al.</em> 2008</a>). Furthermore, average expression level appeared to be unaffected by cold stress within pools at 10 min during recovery from chill coma for five of the genes. At this time point, the only highly significant difference between pools was observed for <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> (<em>P &lt;</em> 0.001). Expression levels measured at 15 min after recovery from chill coma revealed one significant difference within pools: <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> was significantly overexpressed with respect to controls in the Netherlands pool (<em>P &lt;</em> 0.05). Between-pool contrasts at 15 min after recovery from chill coma revealed only a significant difference for <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> (<em>P &lt;</em> 0.01). This suggests that—of the six candidate genes—only <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> is induced by cold stress and may contribute to CCRT variation between temperate and tropical populations.</p><p id="p-44">We also investigated <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> induction on the A* and E* lines individually. Again, we found that <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> is significantly induced 15 min after recovery from chill coma only in the E* line (<em>P</em> &lt; 0.01). Furthermore, we noted that the constitutive expression differences between A* and E* disappeared (in contrast to the experiment with the African and the Netherlands pools described above), presumably because the A* and E* lines share the same autosomal background (data not shown).</p><p id="p-45">Finally, we performed quantitative complementation tests on two of the four genes under the sweeps (<em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> and <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em>) for which lines with <em>P</em>-element insertions were available. None of these tests (performed in the same way as with the deletions) revealed quantitative failure to complement (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). This further supports our conclusion that it is unlikely that the genes under the sweep (at least <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> and <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em>) affect cold tolerance.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464807/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464807/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">P-element analysis of X-linked candidate genes affecting CCRT in female flies</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>Overview</h3><p id="p-47">First, we dissected a QTL interval for CCRT (a proxy for cold tolerance) using quantitative complementation tests. This approach revealed two deletions that failed to complement. Second, we used population genetic methods to narrow the number of genes in these two deletions. This approach led to the precise demarcation of a strong selective sweep in deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em>. Third, we investigated the genes within and near the sweep region by gene expression analysis. We found no evidence that the four genes within the sweep region (<em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em>, <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em>, <em><a href="http://flybase.org/reports/FBgn0029942.html">CG2059</a></em>, and <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em>) are related to cold tolerance. However, this analysis also revealed a new candidate gene related to CCRT: <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em>, a gene located just outside the sweep region that was induced by cold stress. In the following, we discuss these results, including the methods used.</p></div><div id="sec-16" class="subsection"><h3>Quantitative complementation tests on deficiencies and gene expression assays</h3><p id="p-48">Using a set of deficiencies in the framework of the quantitative complementation test allowed us to narrow the QTL interval to two highly significant deficiencies, one of which contains a selective sweep. Furthermore, the list of genes under the QTL (encompassing the sweep) could be reduced to seven candidate genes. However, there is a caveat, because for both deletions significant <em>L</em> × <em>G</em> interactions were found in the presence of significant <em>L</em> effects. This is not uncommon in <em>D. melanogaster</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fanara <em>et al.</em> 2002</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">De Luca <em>et al.</em> 2003</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Geiger-Thornsberry and Mackay 2004</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Harbison <em>et al.</em> 2004</a>), but this means that it is difficult to distinguish between allelic failure to complement at the deficiency and an epistatic interaction between the deficiency and variation elsewhere. However, for the context of this article, this is not important because we have not relied exclusively on quantitative complementation tests to show relatedness to CCRT. In the case of <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em>, our evidence of an association with CCRT is confirmed by gene expression analysis.</p></div><div id="sec-17" class="subsection"><h3>The genes under the selective sweep and the putative targets of positive selection</h3><p id="p-49">Although we have not estimated the selection coefficient, the large value of the CLR statistic indicates that the selective sweep at deletion <em><a href="http://flybase.org/reports/FBab0030519.html">Df(1)ED6906</a></em> in the European population is very strong, the strongest on the entire X-chromosome analyzed (see <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS3.pdf">Figure S3</a>). It encompasses approximately 40 kb (with boundaries that are sharply defined). Based on the demarcation of the sweep, we observe that <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em>, the only candidate gene that was induced by cold stress, is located outside the sweep. The four genes under the sweep are not induced in both African and European populations. <em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em> is differentially expressed at the constitutive level (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), and it has been reported that <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> expression is increased relative to constitutive levels during sustained cold stress (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Graveley <em>et al.</em> 2011</a>). Therefore, the expression of these genes may be temperature-dependent. However, it seems unlikely that regulatory elements determining their expression are the target of selection related to cold tolerance. Furthermore, we have performed quantitative complementation tests on two of the four genes under the sweep (<em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> and <em><a href="http://flybase.org/reports/FBgn0025549.html">unc-119</a></em>) for which lines with <em>P</em>-element insertions were available. None of these tests (performed in the same way as with the deletions) revealed quantitative failure to complement (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). This supports our conclusion that it is unlikely that the genes under the sweep affect cold tolerance.</p><p id="p-50">This leaves us to search for fixed differences in coding sequences on which selection for cold tolerance may have operated. To identify strong selective fixations (leading to sweeps) in coding regions, we need to analyze the sweep profile in more detail. Yet, because variation is almost completely depleted in this genomic region, we cannot use the CLR approach even if we include LD (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Pavlidis <em>et al.</em> 2010</a>). Instead, we used an <em>F</em><sub>ST</sub>-based method (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Foll and Gaggiotti 2008</a>) to identify the target(s) of positive selection. The results are shown in <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2B</a>.</p><p id="p-51">We found four significantly differentiated polymorphisms under the selective sweep. The two SNPs that code for amino acid differences in the gene <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> are most interesting. In the Southeast African sample, both combinations, Thr-Asn and Ala-Asp, are present at positions 936 and 939, where the former is more common. No other combinations exist. In Europe, however, Ala-Asp is fixed (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Subjecting the primary protein sequence encoded by this gene to a structure prediction program (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kelley and Sternberg 2009</a>) reveals that both amino acid positions are part of the α-helix, <em>i.e.</em>, they are located on neighboring helix turns and can therefore interact. Interestingly, Thr and Asn can form one hydrogen bond between their side-chains more than Ala-Asp. The combination Thr-Asn may therefore make the protein more heat-stable than Ala-Asp (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Perl and Schmid 2002</a>), which appears to be an advantage in tropical Africa, given that ambient temperature is an important variant affecting life history traits in fruit flies. Conversely, the combination Ala-Asp may lead to a less rigid structure and thus possibly a more efficient protein, which may be an advantage in the temperate climate of Europe. Ancestral state reconstruction (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lewis 2001</a>) shows that the Thr-Asn combination represents the ancestral state with high probability and that Ala-Asp arose through two point mutations. Because the intermediate states are not observed in the European and African population samples, the transition from Thr-Asn to Ala-Asp probably follows a compensatory evolution model (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kimura 1985</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Innan and Stephan 2001</a>) in which the intermediates are assumed to be strongly deleterious.</p><p id="p-52">Do these adaptive fixations have anything to do with cold adaptation? The protein encoded by <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> is part of the spliceosome (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Herold <em>et al.</em> 2009</a>) whose function may depend on temperature. However, there is no evidence known to us that splicing has a specific function in the protection of flies against cold. The other two significantly differentiated SNPs occur in noncoding regions between genes <em><a href="http://flybase.org/reports/FBgn0029940.html">CG1958</a></em> and <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> and within the huge intron of <em><a href="http://flybase.org/reports/FBgn0029941.html">CG1677</a></em> (see gene model below) (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2B</a>). There is no evidence that they are involved in the regulation of cold tolerance. This leads us to conclude that strong positive selection causing the observed sweep has probably operated on traits (or molecular variants) other than cold tolerance.</p></div><div id="sec-18" class="subsection"><h3>Comparison with population genetics theory</h3><p id="p-53">One of our salient observations is that the genes within the selective sweep region do not affect CCRT, whereas <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> located just outside the sweep is related to this trait. A similar observation was made previously for another QTL of cold tolerance in <em>D. melanogaster</em> (<a id="xref-ref-58-8" class="xref-bibr" href="#ref-58">Svetec <em>et al.</em> 2011</a>). Recent theoretical work has addressed the question of whether we should expect to find selective sweeps at genes controlling a quantitative trait. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chevin and Hospital (2008)</a> presented a model for the footprint of selection at an adaptive QTL in the presence of background variation due to other loci. This analysis is based on the <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lande (1983)</a> model that consists of a locus of major effect on the trait and treats the remaining loci of minor effects as genetic background (such that background variation is maintained at a constant amount). This model predicts that adaptive traits that are under stabilizing selection and show the molecular signature of a selective sweep are only a very small subclass of quantitative traits. <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pavlidis <em>et al.</em> (2012)</a> analyzed a model with <em>n</em> loci controlling a trait under stabilizing selection. In their model, sweeps are more common than in the scenario presented by <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chevin and Hospital (2008)</a>. They find that a multi-locus response to selection may in some cases prevent selective sweeps from being completed, but that conditions causing this to happen strongly depend on the genetic architecture of the trait. For instance, the probability of fixation of selected mutations decreases with the number <em>n</em> of loci involved and also depends on their effect sizes. Fixations are more common when the effects are approximately equal (in absolute size). This raises the question of to what extent CCRT is under stabilizing selection and to what extent CCRT is under directional selection. Although there is evidence that cold tolerance may have experienced positive directional selection from one optimum in Africa to another optimum in Europe, it is currently unclear whether this optimum shift is sufficiently large to overcome stabilizing selection that is expected to be widespread (<em>e.g.</em>, in the form of apparent stabilizing selection due to pleiotropic deleterious effects of mutations).</p></div><div id="sec-19" class="subsection"><h3><em>brinker</em>—a new candidate gene of CCRT</h3><p id="p-54">Based on our gene expression study (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>), <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> is a candidate gene affecting variation in cold tolerance. However, it is important to note that <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> is located outside the large selective sweep described above and thus is not affected by the strong selection generating this sweep. This is consistent with current theory that sweeps at genes controlling phenotypic traits under stabilizing selection are expected to be rare (see above).</p><p id="p-55">Theoretical models of weak selection (particularly for highly polygenic traits) predict the occurrence of allele frequency shifts between populations as a hallmark of polygenic selection (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hancock <em>et al.</em> 2010</a>). For this reason, we searched the region upstream of <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> and found one conspicuous indel polymorphism (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS5.pdf">Figure S5</a>A) at relative positions 109,442 to 109,976, <em>i.e.</em>, approximately 3 kb upstream of <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> and thus also outside the sweep region. This indel is located close to a polymorphic marker (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS5.pdf">Figure S5</a>A) that is significantly associated with CCRT in a Raleigh population (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>). Using an extended sample of populations from the DGPG project (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Pool <em>et al.</em> 2012</a>), we investigated the frequencies of this indel polymorphism in these populations. We classified the indel polymorphism into nondeletion haplotypes and three classes of deletions (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1/FigureS5.pdf">Figure S5</a>B). Based on linear regression analysis of the frequencies of the nondeletion haplotypes, we detected two antiparallel latitudinal clines where one spans from the populations near the equator (Rwanda, Gabon, Cameroon, Ethiopia, and Nigeria) to the north (France and the Netherlands) and another one from the equator to the south (Southeast Africa and South Africa) (<em>P</em> &lt; 0.05 in both cases). This agrees with models of weak selection on highly polygenic traits. However, to what extent the observed frequency differences from the equator to the north and to the south explain the expression differences of <em><a href="http://flybase.org/reports/FBgn0024250.html">brinker</a></em> between tropical and temperate populations (see <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>) is currently an open question and beyond the scope of this article.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We acknowledge A. Steincke, H. Lainer, and S. Lange for providing technical assistance, and F. Fadel and M. Arias for fly work and CCRT scoring. Furthermore, we thank the editors Steve Chenoweth and Mark Johnston and two anonymous reviewers for their comments that improved the presentation of this paper. This work was supported by grant I/848313 from the Volkswagen Foundation (to R.W. and W.S.), and by the DFG Research Unit grant STE 325/12-2 (to W.S.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012757/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-25"><span class="received-label">Received </span>March 25, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-06-20"><span class="accepted-label">Accepted </span>June 20, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Wilches <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.1" data-doi="10.1038/nature11837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnakumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arendt</span>  <span class="cit-name-given-names">M.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic signature of dog domestication reveals adaptation to a starch-rich diet.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D360%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11837%26rft_id%253Dinfo%253Apmid%252F23354050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23354050&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316650500041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.2" data-doi="10.1038/nrg700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Understanding quantitative genetic variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D21%26rft.atitle%253DUnderstanding%2Bquantitative%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg700%26rft_id%253Dinfo%253Apmid%252F11823787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823787&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.4" data-doi="10.1093/molbev/mss090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting selective sweeps from pooled next-generation sequencing samples.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2177</span>–<span class="cit-lpage">2186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss090%26rft_id%253Dinfo%253Apmid%252F22411855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMjE3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.5" data-doi="10.1111/j.1365-294X.2006.03099.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuba</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mäkinen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The utility of QTL-linked markers to detect selective sweeps in natural populations—a case study of the EDA gene and a linked marker in threespine stickleback.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">4613</span>–<span class="cit-lpage">4621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCano%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D4613%26rft.epage%253D4621%26rft.atitle%253DThe%2Butility%2Bof%2BQTL-Linked%2Bmarkers%2Bto%2Bdetect%2Bselective%2Bsweeps%2Bin%2Bnatural%2Bpopulations--a%2Bcase%2Bstudy%2Bof%2Bthe%2BEDA%2Bgene%2Band%2Ba%2Blinked%2Bmarker%2Bin%2Bthreespine%2Bstickleback.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.03099.x%26rft_id%253Dinfo%253Apmid%252F17107487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.03099.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17107487&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242042800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.6" data-doi="10.1534/genetics.108.093351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevin</span>  <span class="cit-name-given-names">L.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selective sweep at a quantitative trait locus in the presence of background genetic variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1645</span>–<span class="cit-lpage">1660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.093351%26rft_id%253Dinfo%253Apmid%252F18832353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8zLzE2NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.7" data-doi="10.1038/ng1218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Luca</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roshina</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiger-Thornsberry</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasyukova</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dopa decarboxylase (<em>Ddc</em>) affects variation in <em>Drosophila</em> longevity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDe%2BLuca%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D433%26rft.atitle%253DDopa%2Bdecarboxylase%2B%2528Ddc%2529%2Baffects%2Bvariation%2Bin%2BDrosophila%2Blongevity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1218%26rft_id%253Dinfo%253Apmid%252F12881721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12881721&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184470500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.8" data-doi="10.1534/genetics.112.145912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duchen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Živković</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Demographic inference reveals African and European admixture in the North American <em>Drosophila melanogaster</em> population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145912%26rft_id%253Dinfo%253Apmid%252F23150605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzEvMjkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.9" data-doi="10.1093/bioinformatics/btr124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">fastsimcoal: a continuous-time coalescent simulator of genomic diversity under arbitrarily complex evolutionary scenarios.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1332</span>–<span class="cit-lpage">1334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr124%26rft_id%253Dinfo%253Apmid%252F21398675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNy85LzEzMzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fanara</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">K. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rollmann</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>Vanaso</em> is a candidate quantitative trait gene for <em>Drosophila</em> olfactory behavior.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1321</span>–<span class="cit-lpage">1328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFanara%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D162%26rft.issue%253D3%26rft.spage%253D1321%26rft.epage%253D1328%26rft.atitle%253DVanaso%2BIs%2Ba%2BCandidate%2BQuantitative%2BTrait%2BGene%2Bfor%2BDrosophila%2BOlfactory%2BBehavior%26rft_id%253Dinfo%253Apmid%252F12454076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi8zLzEzMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.11" data-doi="10.1534/genetics.108.092221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092221%26rft_id%253Dinfo%253Apmid%252F18780740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzIvOTc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.12" data-doi="10.1016/j.mad.2003.12.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geiger-Thornsberry</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait loci affecting natural variation in <em>Drosophila</em> longevity.</span> <abbr class="cit-jnl-abbrev">Mech. Ageing Dev.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bageing%2Band%2Bdevelopment%26rft.stitle%253DMech%2BAgeing%2BDev%26rft.aulast%253DGeiger-Thornsberry%26rft.auinit1%253DG.%2BL.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D179%26rft.epage%253D189%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Baffecting%2Bnatural%2Bvariation%2Bin%2BDrosophila%2Blongevity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mad.2003.12.008%26rft_id%253Dinfo%253Apmid%252F15013662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mad.2003.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15013662&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220469000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glinka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ometto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mousset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Lorenzo</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Demography and natural selection have shaped genetic variation in <em>Drosophila melanogaster</em>: a multi-locus approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGlinka%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1269%26rft.epage%253D1278%26rft.atitle%253DDemography%2Band%2BNatural%2BSelection%2BHave%2BShaped%2BGenetic%2BVariation%2Bin%2BDrosophila%2Bmelanogaster%253A%2BA%2BMulti-locus%2BApproach%26rft_id%253Dinfo%253Apmid%252F14668381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzEyNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.14" data-doi="10.1093/molbev/msl069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glinka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Lorenzo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evidence of gene conversion associated with a selective sweep in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1869</span>–<span class="cit-lpage">1878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl069%26rft_id%253Dinfo%253Apmid%252F16868022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xODY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.15" data-doi="10.1038/nature09715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The developmental transcriptome of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGraveley%26rft.auinit1%253DB.%2BR.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D473%26rft.epage%253D479%26rft.atitle%253DThe%2Bdevelopmental%2Btranscriptome%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09715%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.16" data-doi="10.1098/rstb.2010.0032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkorta-Aranburu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witonsky</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptations to new environments in humans: the role of subtle allele frequency shifts.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">2459</span>–<span class="cit-lpage">2468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D365%26rft.issue%253D1552%26rft.spage%253D2459%26rft.epage%253D2468%26rft.atitle%253DAdaptations%2Bto%2Bnew%2Benvironments%2Bin%2Bhumans%253A%2Bthe%2Brole%2Bof%2Bsubtle%2Ballele%2Bfrequency%2Bshifts%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2010.0032%26rft_id%253Dinfo%253Apmid%252F20643735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU1Mi8yNDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.17" data-doi="10.1534/genetics.166.4.1807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harbison</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fanara</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norga</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait loci affecting starvation resistance in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1807</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarbison%26rft.auinit1%253DS.%2BT.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D1807%26rft.epage%253D1823%26rft.atitle%253DQuantitative%2BTrait%2BLoci%2BAffecting%2BStarvation%2BResistance%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.4.1807%26rft_id%253Dinfo%253Apmid%252F15126400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni80LzE4MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.18" data-doi="10.1186/gb-2007-8-2-r19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellemans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortier</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Paepe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speleman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandesompele</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">qBase relative quantification framework and software for management and automated analysis of real-time quantitative PCR data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHellemans%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253DR19%26rft.epage%253DR19%26rft.atitle%253DqBase%2Brelative%2Bquantification%2Bframework%2Band%2Bsoftware%2Bfor%2Bmanagement%2Band%2Bautomated%2Banalysis%2Bof%2Breal-time%2Bquantitative%2BPCR%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-2-r19%26rft_id%253Dinfo%253Apmid%252F17291332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-2-r19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17291332&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.19" data-doi="10.1128/MCB.01415-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herold</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kastner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urlaub</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Conservation of the protein composition and electron microscopy structure of <em>Drosophila melanogaster</em> and human spliceosomal complexes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01415-08%26rft_id%253Dinfo%253Apmid%252F18981222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI5LzEvMjgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.20" data-doi="10.1046/j.1461-0248.2002.00367.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallas</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Opposing clines for high and low temperature resistance in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D5%26rft.spage%253D614%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1461-0248.2002.00367.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1461-0248.2002.00367.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000177964900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.21" data-doi="10.1101/gr.141689.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A second-generation assembly of the <em>Drosophila simulans</em> genome provides new insights into patterns of lineage-specific divergence.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.141689.112%26rft_id%253Dinfo%253Apmid%252F22936249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjIzLzEvODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.22" data-doi="10.1186/gb-2008-9-1-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saminadin-Peter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene expression variation in African and European populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHutter%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DGene%2Bexpression%2Bvariation%2Bin%2BAfrican%2Band%2BEuropean%2Bpopulations%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-1-r12%26rft_id%253Dinfo%253Apmid%252F18208589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-1-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18208589&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Selection intensity against deleterious mutations in RNA secondary structures and rate of compensatory nucleotide substitutions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D399%26rft.atitle%253DSelection%2BIntensity%2BAgainst%2BDeleterious%2BMutations%2Bin%2BRNA%2BSecondary%2BStructures%2Band%2BRate%2Bof%2BCompensatory%2BNucleotide%2BSubstitutions%26rft_id%253Dinfo%253Apmid%252F11560913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.24" data-doi="10.1534/genetics.106.067728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Selective sweeps reveal candidate genes for adaptation to drought and salt tolerance in common sunflower, <em>Helianthus annuus</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1823</span>–<span class="cit-lpage">1834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067728%26rft_id%253Dinfo%253Apmid%252F17237516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE4MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.25" data-doi="10.1038/nprot.2009.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">M. J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Protein structure prediction on the Web: a case study using the Phyre server.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKelley%26rft.auinit1%253DL.%2BA.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D371%26rft.atitle%253DProtein%2Bstructure%2Bprediction%2Bon%2Bthe%2BWeb%253A%2Ba%2Bcase%2Bstudy%2Busing%2Bthe%2BPhyre%2Bserver.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.2%26rft_id%253Dinfo%253Apmid%252F19247286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19247286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265782000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting a local signature of genetic hitchhiking along a recombining chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D160%26rft.issue%253D2%26rft.spage%253D765%26rft.epage%253D777%26rft.atitle%253DDetecting%2Ba%2BLocal%2BSignature%2Bof%2BGenetic%2BHitchhiking%2BAlong%2Ba%2BRecombining%2BChromosome%26rft_id%253Dinfo%253Apmid%252F11861577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzIvNzY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.27" data-doi="10.1007/BF02923549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">The role of compensatory neutral mutations in molecular evolution.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D64%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02923549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02923549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.28" data-doi="10.1038/hdy.1983.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The response to selection on major and minor mutations affecting a metrical trait.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D50%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1983.6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1983.6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983QH33200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.29" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation in natural populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.30" data-doi="10.1093/molbev/msr031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">S. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werzner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Approximate Bayesian analysis of <em>Drosophila melanogaster</em> polymorphism data reveals a recent colonization of Southeast Asia.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2041</span>–<span class="cit-lpage">2051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr031%26rft_id%253Dinfo%253Apmid%252F21300986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzcvMjA0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.31" data-doi="10.1080/106351501753462876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A likelihood approach to estimating phylogeny from discrete morphological character data.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F106351501753462876%26rft_id%253Dinfo%253Apmid%252F12116640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUwLzYvOTEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.32" data-doi="10.1371/journal.pgen.0020166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inferring the demographic history and rate of adaptive substitution in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De166%26rft.epage%253De166%26rft.atitle%253DInferring%2Bthe%2Bdemographic%2Bhistory%2Band%2Brate%2Bof%2Badaptive%2Bsubstitution%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020166%26rft_id%253Dinfo%253Apmid%252F17040129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17040129&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.33" data-doi="10.1126/science.1233213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linnen</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">R. D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptive evolution of multiple traits through multiple mutations at a single gene.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLinnen%26rft.auinit1%253DC.%2BR.%26rft.volume%253D339%26rft.issue%253D6125%26rft.spage%253D1312%26rft.epage%253D1316%26rft.atitle%253DAdaptive%2BEvolution%2Bof%2BMultiple%2BTraits%2BThrough%2BMultiple%2BMutations%2Bat%2Ba%2BSingle%2BGene%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1233213%26rft_id%253Dinfo%253Apmid%252F23493712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNS8xMzEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.34" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.35" data-doi="10.1038/sj.hdy.6800786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Quantitative trait loci for thermotolerance phenotypes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMorgan%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D96%26rft.issue%253D3%26rft.spage%253D232%26rft.epage%253D242%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bfor%2Bthermotolerance%2Bphenotypes%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800786%26rft_id%253Dinfo%253Apmid%252F16404413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16404413&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235499000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-4.9.1635.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Molecular Evolutionary Genetics</span>, <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.37" data-doi="10.1073/pnas.76.10.5269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Mathematical model for studying genetic variation in terms of restriction endonucleases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">5269</span>–<span class="cit-lpage">5273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D76%26rft.issue%253D10%26rft.spage%253D5269%26rft.epage%253D5273%26rft.atitle%253DMathematical%2Bmodel%2Bfor%2Bstudying%2Bgenetic%2Bvariation%2Bin%2Bterms%2Bof%2Brestriction%2Bendonucleases.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.10.5269%26rft_id%253Dinfo%253Apmid%252F291943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzYvMTAvNTI2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.38" data-doi="10.1101/gr.4252305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic scans for selective sweeps using SNP data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1566</span>–<span class="cit-lpage">1575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1566%26rft.epage%253D1575%26rft.atitle%253DGenomic%2Bscans%2Bfor%2Bselective%2Bsweeps%2Busing%2BSNP%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4252305%26rft_id%253Dinfo%253Apmid%252F16251466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.39" data-doi="10.1111/j.1365-294X.2008.03945.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norry</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scannapieco</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sambucetti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertoli</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeschcke</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">QTL for the thermotolerance effect of heat hardening, knockdown resistance to heat and chill-coma recovery in an intercontinental set of recombinant inbred lines of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4570</span>–<span class="cit-lpage">4581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNorry%26rft.auinit1%253DF.%2BM.%26rft.volume%253D17%26rft.issue%253D20%26rft.spage%253D4570%26rft.epage%253D4581%26rft.atitle%253DQTL%2Bfor%2Bthe%2Bthermotolerance%2Beffect%2Bof%2Bheat%2Bhardening%252C%2Bknockdown%2Bresistance%2Bto%2Bheat%2Band%2Bchill-coma%2Brecovery%2Bin%2Ban%2Bintercontinental%2Bset%2Bof%2Brecombinant%2Binbred%2Blines%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03945.x%26rft_id%253Dinfo%253Apmid%252F18986501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03945.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18986501&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasyukova</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Deficiency mapping of quantitative trait loci affecting longevity in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1129</span>–<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPasyukova%26rft.auinit1%253DE.%2BG.%26rft.volume%253D156%26rft.issue%253D3%26rft.spage%253D1129%26rft.epage%253D1146%26rft.atitle%253DDeficiency%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BAffecting%2BLongevity%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F11063689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni8zLzExMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A population genomic approach to map recent positive selection in model species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">3585</span>–<span class="cit-lpage">3598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPavlidis%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D16%26rft.spage%253D3585%26rft.epage%253D3598%26rft.atitle%253DA%2Bpopulation%2Bgenomic%2Bapproach%2Bto%2Bmap%2Brecent%2Bpositive%2Bselection%2Bin%2Bmodel%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F18627454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18627454&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258220500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.42" data-doi="10.1534/genetics.110.116459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Searching for footprints of positive selection in whole-genome SNP data from nonequilibrium populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">907</span>–<span class="cit-lpage">922</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116459%26rft_id%253Dinfo%253Apmid%252F20407129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.43" data-doi="10.1534/genetics.112.142547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Selective sweeps in multilocus models of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142547%26rft_id%253Dinfo%253Apmid%252F22714406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMjI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.44" data-doi="10.1093/molbev/mst112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Živković</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alachiotis</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SweeD: likelihood-based detection of selective sweeps in thousands of genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2224</span>–<span class="cit-lpage">2234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst112%26rft_id%253Dinfo%253Apmid%252F23777627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzkvMjIyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOS8xNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perl</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">F. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Some like it hot: the molecular determinants of protein thermostability.</span> <abbr class="cit-jnl-abbrev">European Journal of Chemical Biology</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2BJournal%2Bof%2BChemical%2BBiology%26rft.volume%253D3%26rft.spage%253D39%26rft_id%253Dinfo%253Apmid%252F17590951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17590951&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.46" data-doi="10.1111/j.1365-294X.2007.03324.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The genetic basis of adaptive pigmentation variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2844</span>–<span class="cit-lpage">2851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPool%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D16%26rft.issue%253D14%26rft.spage%253D2844%26rft.epage%253D2851%26rft.atitle%253DThe%2Bgenetic%2Bbasis%2Bof%2Badaptive%2Bpigmentation%2Bvariation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03324.x%26rft_id%253Dinfo%253Apmid%252F17614900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03324.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17614900&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.47" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of Sub-Saharan <em>Drosophila melanogaster</em>: African diversity and non-African admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.48" data-doi="10.1038/nrg2880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation—not by sweeps alone.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D665%26rft.epage%253D667%26rft.atitle%253DAdaptation%2B-%2Bnot%2Bby%2Bsweeps%2Balone.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2880%26rft_id%253Dinfo%253Apmid%252F20838407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838407&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281911300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.49" data-doi="10.1371/journal.pgen.1004148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pausch</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strom</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Classic selective sweeps revealed by massive sequencing in cattle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004148%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004148%26rft_id%253Dinfo%253Apmid%252F24586189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586189&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.50" data-doi="10.1126/science.1178357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebeiz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassner</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Stepwise modification of a modular enhancer underlies adaptation in a <em>Drosophila</em> population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1663</span>–<span class="cit-lpage">1667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRebeiz%26rft.auinit1%253DM.%26rft.volume%253D326%26rft.issue%253D5960%26rft.spage%253D1663%26rft.epage%253D1667%26rft.atitle%253DStepwise%2BModification%2Bof%2Ba%2BModular%2BEnhancer%2BUnderlies%2BAdaptation%2Bin%2Ba%2BDrosophila%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178357%26rft_id%253Dinfo%253Apmid%252F20019281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk2MC8xNjYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.51" data-doi="10.1534/genetics.107.081281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riebler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian variable selection for detecting adaptive genomic differences among populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081281%26rft_id%253Dinfo%253Apmid%252F18245358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE4MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.52" data-doi="10.1038/nature08832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwood</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome resequencing reveals loci under selection during chicken domestication.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRubin%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D464%26rft.issue%253D7288%26rft.spage%253D587%26rft.epage%253D591%26rft.atitle%253DWhole-genome%2Bresequencing%2Breveals%2Bloci%2Bunder%2Bselection%2Bduring%2Bchicken%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08832%26rft_id%253Dinfo%253Apmid%252F20220755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220755&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275974200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.53" data-doi="10.1073/pnas.1217149109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez Barrio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayyab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Strong signatures of selection in the domestic pig genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">19529</span>–<span class="cit-lpage">19536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217149109%26rft_id%253Dinfo%253Apmid%252F23151514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ4LzE5NTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.54" data-doi="10.1554/05-115.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matzkin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ippolito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eanes</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Geographic variation in diapause incidence, life-history traits, and climatic adaptation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">1721</span>–<span class="cit-lpage">1732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DSchmidt%26rft.auinit1%253DP.%2BS.%26rft.volume%253D59%26rft.issue%253D8%26rft.spage%253D1721%26rft.epage%253D1732%26rft.atitle%253DGeographic%2Bvariation%2Bin%2Bdiapause%2Bincidence%252C%2Blife-history%2Btraits%252C%2Band%2Bclimatic%2Badaptation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F05-115.1%26rft_id%253Dinfo%253Apmid%252F16331839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/05-115.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16331839&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231658900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.55" data-doi="10.1126/sageke.2004.12.pe13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">How good are quantitative complementation tests?</span> <abbr class="cit-jnl-abbrev">Sci. SAGE KE</abbr> <span class="cit-vol">2004</span>: <span class="cit-fpage">pe13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2527s%2BSAGE%2BKE%26rft.stitle%253DSAGE%2BKE%26rft.aulast%253DService%26rft.auinit1%253DP.%2BM.%26rft.volume%253D2004%26rft.issue%253D12%26rft.spage%253Dpe13%26rft.epage%253Dpe13%26rft.atitle%253DHow%2BGood%2BAre%2BQuantitative%2BComplementation%2BTests%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsageke.2004.12.pe13%26rft_id%253Dinfo%253Apmid%252F15044758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic2FnZWtlIjtzOjU6InJlc2lkIjtzOjEyOiIyMDA0LzEyL3BlMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.56" data-doi="10.1098/rstb.2009.0278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic hitchhiking <em>vs.</em> background selection: the controversy and its implications.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1245</span>–<span class="cit-lpage">1253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DStephan%26rft.auinit1%253DW.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1245%26rft.epage%253D1253%26rft.atitle%253DGenetic%2Bhitchhiking%2Bversus%2Bbackground%2Bselection%253A%2Bthe%2Bcontroversy%2Band%2Bits%2Bimplications%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0278%26rft_id%253Dinfo%253Apmid%252F20308100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.57" data-doi="10.1038/sj.hdy.6800901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The recent demographic and adaptive history of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DStephan%26rft.auinit1%253DW.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D65%26rft.epage%253D68%26rft.atitle%253DThe%2Brecent%2Bdemographic%2Band%2Badaptive%2Bhistory%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800901%26rft_id%253Dinfo%253Apmid%252F17006533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17006533&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243730100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.58" data-doi="10.1111/j.1365-294X.2010.04951.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svetec</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werzner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilches</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Álvarez-Castro</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of X-linked quantitative trait loci affecting cold tolerance in <em>Drosophila melanogaster</em> and fine mapping by selective sweep analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">530</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSvetec%26rft.auinit1%253DN.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D530%26rft.epage%253D544%26rft.atitle%253DIdentification%2Bof%2BX-linked%2Bquantitative%2Btrait%2Bloci%2Baffecting%2Bcold%2Btolerance%2Bin%2BDrosophila%2Bmelanogaster%2Band%2Bfine%2Bmapping%2Bby%2Bselective%2Bsweep%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04951.x%26rft_id%253Dinfo%253Apmid%252F21199023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04951.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21199023&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286254700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evolutionary relationship of DNA sequences in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becquet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Progress and prospects in mapping recent selection in the genome.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DThornton%26rft.auinit1%253DK.%2BR.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D340%26rft.epage%253D348%26rft.atitle%253DProgress%2Band%2Bprospects%2Bin%2Bmapping%2Brecent%2Bselection%2Bin%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F17473869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17473869&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246721700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1635.62" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1635.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1623" title="Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1647" title="The Functional Significance of Common Polymorphisms in Zinc Finger Transcription Factors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1635" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4zMSDZnj5f0cpYzWqBtls6XQ1tVUB2YUZuVWdi6u7kQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1635" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1635" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1635" class="active" data-icon-position="" data-hide-link-title="0">Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in Drosophila melanogaster</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459657" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AFCkGcctHL1UbxiJ3un_1m2TaK9ZwOuNNgmcJPi3vUE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1635.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1635" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sQl6KcjCzzjpGMupGVK7VPKqaMQY2_x09GHcLz2TFoY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1635" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1635.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459657" id="node459657--2" data-pisa="ggg;4/9/1635" data-pisa-master="ggg;g3.114.012757" data-apath="/ggg/4/9/1635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0002-7862-1507" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ricardo Wilches</span>, <span class="highwire-citation-author" data-delta="1">Susanne Voigt</span>, <span class="highwire-citation-author" data-delta="2">Pablo Duchen</span>, <span class="highwire-citation-author" data-delta="3">Stefan Laurent</span> and <span class="highwire-citation-author" data-delta="4">Wolfgang Stephan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1635-1645;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012757 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459657--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ricardo Wilches </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ricardo%2BWilches%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilches%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARicardo%2BWilches%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0002-7862-1507" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ricardo Wilches</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wilches@bio.lmu.de" class="" data-icon-position="" data-hide-link-title="0">wilches@bio.lmu.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Susanne Voigt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BVoigt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voigt%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BVoigt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pablo Duchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BDuchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duchen%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BDuchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stefan Laurent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BLaurent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurent%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BLaurent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wolfgang Stephan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BStephan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BStephan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1635" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1635&amp;title=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1635&amp;title=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1635&amp;title=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1635&amp;text=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1635&amp;title=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1635&amp;t=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1635&amp;title=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1635&amp;title=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459657/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459657" id="node459657--3" data-pisa="ggg;4/9/1635" data-pisa-master="ggg;g3.114.012757" data-apath="/ggg/4/9/1635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Mapping and Selective Sweep Analysis of QTL for Cold Tolerance in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0002-7862-1507" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ricardo Wilches</span>, <span class="highwire-citation-author" data-delta="1">Susanne Voigt</span>, <span class="highwire-citation-author" data-delta="2">Pablo Duchen</span>, <span class="highwire-citation-author" data-delta="3">Stefan Laurent</span> and <span class="highwire-citation-author" data-delta="4">Wolfgang Stephan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1635-1645;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012757 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459657--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ricardo Wilches </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ricardo%2BWilches%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilches%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARicardo%2BWilches%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0002-7862-1507" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ricardo Wilches</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wilches@bio.lmu.de" class="" data-icon-position="" data-hide-link-title="0">wilches@bio.lmu.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Susanne Voigt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BVoigt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voigt%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BVoigt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pablo Duchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BDuchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duchen%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BDuchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stefan Laurent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BLaurent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurent%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BLaurent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wolfgang Stephan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Section of Evolutionary Biology, Department of Biology II, Ludwig-Maximilian University of Munich, 82152 Planegg-Martinsried, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BStephan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BStephan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1635&amp;rft.epage=1645&amp;rft.atitle=Fine-Mapping%20and%20Selective%20Sweep%20Analysis%20of%20QTL%20for%20Cold%20Tolerance%20in%20Drosophila%20melanogaster&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wilches%2C+Ricardo&amp;rft.au=Voigt%2C+Susanne&amp;rft.au=Duchen%2C+Pablo&amp;rft.au=Laurent%2C+Stefan&amp;rft.au=Stephan%2C+Wolfgang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459657/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459657/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459657/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459657/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459657/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459657/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459657/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459657/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459657/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459657/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459657/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459657/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1635" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1635.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1635.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/9/1635.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/9/1635.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
