<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/9/1667" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/9/1667.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/9/1667.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/9/1667.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/9/1667/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/9/1667.atom" />
<meta name="HW.pisa" content="ggg;4/9/1667" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" />
<meta name="DC.Identifier" content="10.1534/g3.114.012849" />
<meta name="DC.Date" content="2014-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Eyre-Walker and Eyre-Walker. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="It has been claimed recently that it may be possible to predict the rate of de novo mutation of each site in the human genome with a high degree of accuracy [[Michaelson et al. (2012)][1], Cell 151: 1431−1442]. We show that this claim is unwarranted. By considering the correlation between the rate of de novo mutation and the predictions from the model of Michaelson et al. , we show there could be substantial unexplained variance in the mutation rate. We investigate whether the model of Michaelson et al. captures variation at the single nucleotide level that is not due to simple context. We show that the model captures a substantial fraction of this variation at CpG dinucleotides but fails to explain much of the variation at non-CpG sites.

 [1]: #ref-16" />
<meta name="DC.Contributor" content="Adam Eyre-Walker" />
<meta name="DC.Contributor" content="Ying Chen Eyre-Walker" />
<meta name="article:published_time" content="2014-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;It has been claimed recently that it may be possible to predict the rate of &lt;i&gt;de novo&lt;/i&gt; mutation of each site in the human genome with a high degree of accuracy [Michaelson &lt;i&gt;et al.&lt;/i&gt; (2012), Cell 151: 1431−1442]. We show that this claim is unwarranted. By considering the correlation between the rate of &lt;i&gt;de novo&lt;/i&gt; mutation and the predictions from the model of Michaelson &lt;i&gt;et al.&lt;/i&gt;, we show there could be substantial unexplained variance in the mutation rate. We investigate whether the model of Michaelson &lt;i&gt;et al.&lt;/i&gt; captures variation at the single nucleotide level that is not due to simple context. We show that the model captures a substantial fraction of this variation at CpG dinucleotides but fails to explain much of the variation at non-CpG sites.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/09/01" />
<meta name="citation_mjid" content="ggg;4/9/1667" />
<meta name="citation_id" content="4/9/1667" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/9/1667" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/9/1667.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/9/1667.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/9/1667.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012849" />
<meta name="citation_pmid" content="24996580" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="4" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1667" />
<meta name="citation_lastpage" content="1670" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Adam Eyre-Walker" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom" />
<meta name="citation_author_email" content="a.c.eyre-walker@sussex.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5527-8729" />
<meta name="citation_author" content="Ying Chen Eyre-Walker" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Awadalla;citation_author=J. Gauthier;citation_author=R. A. Myers;citation_author=F. Casals;citation_author=F. F. Hamdan;citation_title=Direct measure of the de novo mutation rate in autism and schizophrenia cohorts.;citation_pages=316-324;citation_volume=87;citation_year=2010;citation_issue=3;citation_pmid=20797689;citation_doi=10.1016/j.ajhg.2010.07.019" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Commun.;citation_author=N. C. Beaulieu;citation_author=A. A. Abu-Daya;citation_author=P. J. McLane;citation_title=Estimating the distribution of a sum of independent lognormal random variables.;citation_pages=2869-2873;citation_volume=43;citation_year=1995;citation_doi=10.1109/26.477480" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome.;citation_pages=69-87;citation_volume=437;citation_year=2005;citation_issue=7055;citation_pmid=16136131;citation_doi=10.1038/nature04072" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Conrad;citation_author=J. E. Keebler;citation_author=M. A. Depristo;citation_author=S. J. Lindsay;citation_author=Y. Zhang;citation_title=Variation in genome-wide mutation rates within and between human families.;citation_pages=712-714;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21666693;citation_doi=10.1038/ng.862" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. Eory;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Distributions of selectively constrained sites and deleterious mutation rates in the hominid and murid genomes.;citation_volume=27;citation_year=2010;citation_pmid=19759235;citation_doi=10.1093/molbev/msp219" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Commun. Syst.;citation_author=L. Fenton;citation_title=The sum of lognormal probability distributions in scatter transmission systems.;citation_pages=57-67;citation_volume=CS-8;citation_year=1960;citation_doi=10.1109/TCOM.1960.1097606" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Hodgkinson;citation_author=A. Eyre-Walker;citation_title=Variation in the mutation rate across mammalian genomes.;citation_pages=756-766;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=21969038;citation_doi=10.1038/nrg3098" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=A. Hodgkinson;citation_author=E. Ladoukakis;citation_author=A. Eyre-Walker;citation_title=Cryptic variation in the human mutation rate.;citation_pages=e27;citation_volume=7;citation_year=2009;citation_doi=10.1371/journal.pbio.1000027" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. G. Hwang;citation_author=P. Green;citation_title=Bayesian Markov chain Monte Carlo sequence analysis reveals varying neutral substitution patterns in mammalian evolution.;citation_volume=101;citation_year=2004;citation_pmid=15292512;citation_doi=10.1073/pnas.0404142101" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=I. Iossifov;citation_author=M. Ronemus;citation_author=D. Levy;citation_author=Z. Wang;citation_author=I. Hakker;citation_title=De novo gene disruptions in children on the autistic spectrum.;citation_pages=285-299;citation_volume=74;citation_year=2012;citation_pmid=22542183;citation_doi=10.1016/j.neuron.2012.04.009" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. L. F. Johnson;citation_author=I. Hellmann;citation_title=Mutation rate distribution inferred from coincident SNPs and coincident substitutions.;citation_volume=3;citation_year=2011;citation_pmid=21572094;citation_doi=10.1093/gbe/evr044" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=M. L. Frigge;citation_author=G. Masson;citation_author=S. Besenbacher;citation_author=P. Sulem;citation_title=Rate of de novo mutations and the importance of father's age to disease risk.;citation_pages=471-475;citation_volume=488;citation_year=2012;citation_issue=7412;citation_pmid=22914163;citation_doi=10.1038/nature11396" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. M. Leffler;citation_author=Z. Gao;citation_author=S. Pfeifer;citation_author=L. Segurel;citation_author=A. Auton;citation_title=Multiple instances of ancient balancing selection shared between humans and chimpanzees.;citation_volume=339;citation_year=2013;citation_pmid=23413192;citation_doi=10.1126/science.1234070;citation_georef=2013044317" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=G. Lunter;citation_author=C. P. Ponting;citation_author=J. Hein;citation_title=Genome-wide identification of human functional DNA using a neutral indel model.;citation_pages=e5-e5;citation_volume=2;citation_year=2006;citation_issue=1;citation_pmid=16410828;citation_doi=10.1371/journal.pcbi.0020005" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=J. J. Michaelson;citation_author=Y. Shi;citation_author=M. Gujral;citation_author=H. Zheng;citation_author=D. Malhotra;citation_title=Whole-genome sequencing in autism identifies hot spots for de novo germline mutation.;citation_pages=1431-1442;citation_volume=151;citation_year=2012;citation_pmid=23260136;citation_doi=10.1016/j.cell.2012.11.019" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. M. Neale;citation_author=Y. Kou;citation_author=L. Liu;citation_author=A. Ma’ayan;citation_author=K. E. Samocha;citation_title=Patterns and rates of exonic de novo mutations in autism spectrum disorders.;citation_pages=242-245;citation_volume=485;citation_year=2012;citation_issue=7397;citation_pmid=22495311;citation_doi=10.1038/nature11011" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. J. O’Roak;citation_author=P. Deriziotis;citation_author=C. Lee;citation_author=L. Vives;citation_author=J. J. Schwartz;citation_title=Exome sequencing in sporadic autism spectrum disorders identifies severe de novo mutations.;citation_pages=585-589;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21572417;citation_doi=10.1038/ng.835" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. J. Sanders;citation_author=M. T. Murtha;citation_author=A. R. Gupta;citation_author=J. D. Murdoch;citation_author=M. J. Raubeson;citation_title=De novo mutations revealed by whole-exome sequencing are strongly associated with autism.;citation_pages=237-241;citation_volume=485;citation_year=2012;citation_issue=7397;citation_pmid=22495306;citation_doi=10.1038/nature10945" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
<meta name="twitter:description" content="It has been claimed recently that it may be possible to predict the rate of de novo mutation of each site in the human genome with a high degree of accuracy [[Michaelson et al. (2012)][1], Cell 151: 1431−1442]. We show that this claim is unwarranted. By considering the correlation between the rate of de novo mutation and the predictions from the model of Michaelson et al. , we show there could be substantial unexplained variance in the mutation rate. We investigate whether the model of Michaelson et al. captures variation at the single nucleotide level that is not due to simple context. We show that the model captures a substantial fraction of this variation at CpG dinucleotides but fails to explain much of the variation at non-CpG sites.

 [1]: #ref-16" />
<meta name="og-title" property="og:title" content="How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/9/1667" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="It has been claimed recently that it may be possible to predict the rate of de novo mutation of each site in the human genome with a high degree of accuracy [[Michaelson et al. (2012)][1], Cell 151: 1431−1442]. We show that this claim is unwarranted. By considering the correlation between the rate of de novo mutation and the predictions from the model of Michaelson et al. , we show there could be substantial unexplained variance in the mutation rate. We investigate whether the model of Michaelson et al. captures variation at the single nucleotide level that is not due to simple context. We show that the model captures a substantial fraction of this variation at CpG dinucleotides but fails to explain much of the variation at non-CpG sites.

 [1]: #ref-16" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9.cover-source.jpg" />
    <title>How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained? | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"P6B7i9OPXvcIlnxRnqTs37458gN6IbFq0KnLIgaDyGU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459674","apath":"\/ggg\/4\/9\/1667.atom","pisa":"ggg;4\/9\/1667","ac":{"\/ggg\/4\/9\/1667.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/9\/1667.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1667"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/9\/1667"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/9\/1667":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ioOAEjeujF4TG2t3EV58rRJgBYikU8R2rxN\\\\\\\/I3dQPwss2Rcne32p0r2RB+fCjRzRbyJad7Dr1GR\\\\\\\/aZ+n2RPcRP\\\\\\\/bHekIdWG\\\\\\\/DcRrbwYTCwVNzT1p\\\\\\\/0jH\\\\\\\/GuVIKLJTZAAqnP43ewCQOy72tihSdwkvfOmbIWMIgEeRzFkaTLveRbPkf1PWoMem5BJXyhgAWZK\\\\\\\/2Ixhnp7n9E\\\\\\\/DNw8c3la7VKnpwULLxij+A0kAkGiMZGee1rdSZxeYlK6Or181AqdTefYthvqc4qg+8D6zoXhfaaeuatqCpsxrDRpIFrcWtuwa2\\\\\\\/oyCxT8fxkXM3lO9U8UHmQSYpYLzaf0wHy0yb8C8+lOhkS1DOC22BYiqnDl93\\\\\\\/7qwe4HIp0BF66gkTJ8hLqnjSAwOxXSGnlSb\\\\\\\/JrM\\\\\\\/ySzjw2zviXK6NDmhTINuvgcqJgmqjD3TcCzAxglrwgiiVN6kygB67Chds1w36GxU2zcgFszOc\\\\\\\/lSAciMYQZA\\\\\\\/lKze3y6kcDPviXl+q4MMnMVJ25\\\\\\\/xmld8T4lQBKc9\\\\\\\/GrabXQ8brDM9nKGBflBO\\\\\\\/0OCrOsKM0I\\\\\\\/e\\\\\\\/OUj4KJ2yBTRs4cL6fAryXMRj77IEsF9bGaGAz4JXJt1tg0QCmK3hn1TYzAZgK0\\\\\\\/VzApNb5KtZsPzpQanfyomh5G6vJPdzOtpfQO+7rxd\\\\\\\/\\\\\\\/m8Xp9xwp+c0zqvh2Pn7bRNI1QHwg9HJ54TcxGbfFItxY9nrhBahu7jE09s06K2rohwkXiGQC0NkTYHpfaEeYOYPXGpy3YbK3leoxAWEVQcenILNuK4bJd0Zg35+CaD8KaHrQh\\\\\\\/ywYuDW00pcXjqSZlI9+LbeyfWd1wfLTR0HSDepLZ\\\\\\\/9AsgHOnHDklG5FoiY9D5Oog9Q+MkcfU1yeK4dvMj08kIKCXsoCXJLh4vukAsuckPIrnL\\\\\\\/TYpvaPk+RGMCCzhZ0ZQAStb9Vv0BI\\\\\\\/lEfZ\\\\\\\/yX0Fy\\\\\\\/oTlvkXK4fmJexVtCsbcU1O2q4fnQQPL+SCQ9ubEZ3WcJaLSdcmOMWllGz9Wya2rVrUs6QODNHZo+AHPTKy2XhTZbn1HIixuOS9r96K9CxeaXX3dxa715XH4pOsOq9ZFamIS6\\\\\\\/+Rp2dZx0CT\\\\\\\/fWWSXF2XLFlU2V5MBkj2x+0+\\\\\\\/GLuyPde1G5A3kqHMxes+e9lFSQxeBzMB7yagjYzMrVB720oYp5T\\\\\\\/A5r6Gy\\\\\\\/PjPErQvcnHItP22uk+8qDofs8yfC57av4NEgFJLrds+LbKxNXYkftVaKYjeRKX80yza4b2l9yge0nhqFzVHkR8bqaUjnC6Foa\\\u0022,\\\u0022iv\\\u0022:\\\u0022+sRaGZQRgIRcz3Gk2Hb+sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a39cf6a58b21020e8f91c00a0d1724a9c959c6c4c306107dbc349533331d6b9d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aMLeGNEcBoadRVBiClxhtrD9AZnMLqta9TWMgVdEzoO7GgirVA8IRFWISM6OffmSfvh8noOPQhvAiYDteOS8koDq+IcYwMk+zkYDUbhFRYOGGUCGf79ATtcg+tNEZfdB8Op2lOuh8dhiIm7NdPWM+WZZmhcQeilFLJ6\\\\\\\/VcFAZBmQzNWGKOoyU2HnWC3OEOEbxFRd9Hit22rR3\\\\\\\/bkYL9JyB+PgViBf6hz1auc6jjPHYTIkLmwhdw9FWg4TkipyaOweK\\\\\\\/XWg49xI\\\\\\\/CVQX2pBv1auS7rxKctUZVspgDM9fC3TMtqSVIUET2mIKRAuwCgTizN9fQAM+M4tzfU5TV6ClhIBkdSKOBaWEvqOyZH1N2e\\\\\\\/4BsFi3tkM1bkW0th9ThZd5SAKXpw8PcaN1XCDQLuBWDDV\\\\\\\/004hVU\\\\\\\/7qD2Ve1i7xPTi8sWahhCVb4bTMM9NwaUWfAQJmL9RNHnG9fMoIsjo2La0J\\\\\\\/vEjJMvIiPxaLidi3\\\\\\\/hnimePV1rTJqXRyslQKVne2VQ1IneISSILPqrdR3obxC4OKnkJZ66+yl9QE8yRl5o6VMBfLZEnXCaW5hm3VcHMWttV73imgNXEuSXWBtv0Lr6YAWsYV3mepOIuv+CvZZ8AecbL+BVwh6yHeoMDLrwc6djyVBPmIxjE3D98M+T5HSuGJ5d2vE8r+\\\\\\\/wahXOC7sQEmzCrzNLkfEAaRJ0bx7D4QXuweKGff\\\\\\\/KHgrDu6\\\\\\\/CdTIe6xp0hB99ZPl6Gch0PCPgbMwIM0jj3SrBaMkFd5qrVAtbfEa86P6evBvUdzJnZB6kTu+2GfU3+FHqoYFjv3ESUUaLuypIQTTN0pfyuYXxCcpsE5UwMr0qN8IyjleNHCS7fGdk4PJ51wyE9kY3xjMM7sU8GJf0W1Un4dCuwJsC1QY4h7AY+uOSztoCJ8pbcME4X74A1HQyF3a2ByK3+XoH0Ic6ewFdRhNtE2ns6N4HBw8ww5yvQvzGSnPif4uMSY0MzKWfIc1uFjZZFxunia5eUGlsoOFotkkMsja61\\\\\\\/RG8wQUPIiKj03ZZsqUItqLZVsXCblslylDFWJ6Hk6VEbLTg7A+2Qfw0qiE\\\\\\\/Azpvq92Oc7Yz+FmnbZ1bS+ddTxBxjmo9FiMtxi9UzJtQwEzoAnIPz63UiXlIf9d1onUFUvs4AULp1wErTArMwGbZy1t1M9pRf6T23gLBq3iy505V9TrzZbyQbh9zprsSLeNNftpe8hM\\\\\\\/VKadQeQRvo7N6+dm5otqPWAS3j7DWJtmTZsyGK6BQEy6l5f54BeMoE6fH+nDI6mO5uF81wru8CVddMZpuhEg4R1eOZJSAjuMMkMq3lV\\\\\\\/yj31spIqQ8UTY1fUQ7pEtc\\\\\\\/XxNHiA39u+V57cljlHBxhxvgqpN+a3UpDIij0YYrB9H4LQDDDNxTPWHT1+pH4BZPoziJDsoYUoUsNvw2qDBSOpvuK7kcbUrKjTzV8ojMPnm3Iia3CASDUqzNJiWGhup86OnMWnRakcpP+Z9ci3pojr7VV8FYhAYMIGOTW+v2sgzrqyD5Nh0LnCovFWNM8X2berAo6Zn6M8LTzwNwG76FMnwuPt2vMOZ15p7HnG9LCnLouHPHp5PwJNETXzRfFW3aAabrHTs73S7PDX2E2NYZ9+KJfkEIdH+ZC5IdHQu\\\\\\\/uGumMSitsrLJ7tyOD3hA\\\\\\\/BclDPPtHYFEIT+JikdOd7nClaYL6r+hjkS8Y9NSIpNoPHrZmsDsbPaDFqOpqLHBIfTjtHzmw7n3VxDQuFgaqv79JY1j\\\\\\\/VsYSvQuR5Pu+2R0kcGj+OaEXNm91+u1v7hIj5zYoMxO8lz0QwvgtahKMuxcqD+c8u8vNh6JG6orVK\\\\\\\/UJ+3A172X9n\\\\\\\/QHsAlNvetnfLU+wvKQf6syUB8Sr2Hiv7ECtef48TVyR6EDP84v74Sx7zFOhniiD8DPu0ByFZ4TR70r9NrXr4pg239IW69PZsBZpsWH4TMsA9VU5EzFnwOmW\\\\\\\/kpSYaSLxOL3hMlt4\\\\\\\/L0ob0LcZXcySFeRU5h6HUPJxXygYl7oz2onenm03oqybQIUs5ruA6nHKWT14Rsf6Z4iECOmQqkwFk3O3Tu2tSfu0vGiIUv\\\\\\\/PeCOu+UhXrVlC+2Gn9mpT18iUpXYOEQ9MjlTalvwvHndN17idfZojh3lpAjuHEQ7Qi14kARj0I+y1LYEDPVJlkRCOtcjHm8PxXxclbkCwXUgPIbr6sDpc00AmSt0X0GZsf+YFKsLw8gQa6NfMTbgdBltLTVW44PuYOpkujogOct28KjEUxw9Nqq9ws0CqZfIy1QQjBnUa9n9k2XV+z+oP7zKcAD+fKCwYUt03DVmxAaWXjyuKWj\\\\\\\/sPwuvVDrR2ziefC+zaAeLTcGQFNCRPqdxiuJLeq2msLDWeFfECgQ+R88L\\\\\\\/rJAKQCBX13CYR74xN7dzOcOztqbWh12GYEuLZXawsAWXK31t0wEHfEwlWADu79iGNRtMOYM+mIf78nO8Inrlb1v8MfW+DljDDGk328eZoo+e3EhIlj55DF54RiU72\\\\\\\/V77AoITUG1zkUf8toR1BJe0Yg+TuNizQX01uAFDeVDCkbzU4bN4M9ZgxmbC6poOX5kfhsBzRx2aztubrtjWw6nHRiGT1SVidsQfXO3ovNf0xcLPhOBsy\\\\\\\/Kxr5190l++Z4Mxyc+0DQ+ZHksQJJsrsxU+2e90g4E0I+aVCpNI6ncKgFwhYnSV+MNl6ZSUwmCBaDSEwKSfe9Sa4igOXDRnWzqkDuPvlqOON4oNVgTPctuKaWXVIiPJNajL+tBwji+nj\\\\\\\/t0epgTnNP1bfZsELWreLVxBlS5Rn8hgU1fd+dDftymH71MOv2Oj6MRTWBn2ioygtNLBbKvCWtxPWf9PuX9FJ2ArkCKi3IM8Bem9yvvJm75FHdkpj0VvEHp7eKGzrf2MCe\\\\\\\/aQZYWVONY3DhBv\\\\\\\/s\\\\\\\/Syp5xPDFi+ox2rrvyklVsT0T6mnK1IvALvhizVBwWaXwodsLWX17iyd3ge5C+4YOQSr318eiA3HCUNxcMEOYOtgbSh7yhujLcS3\\\\\\\/8dRpnpvaPup3mp74+jd7oF57nDZhHSD4M74Icn2W6EFH6LdnJZEtmxfvYUFcT9JdkYU0Vh4ZYbmmFBJme1BJM3UCYBrqcF+BwdCFcKIp1bOQ5hHEgFV7nDo39dgLeS66kbnwLQM8cvHP7Q\\\\\\\/j1LzguhUGdaUzU09dx9fT4fXVMNQkJoFrA8sw\\\\\\\/1NVV+VKo8uVXYUWLzSc\\\\\\\/Si7sJuENyPAntKCcnJDQpkcp0insrAhjjhEXg3upDPAt7EEhR3TkWJLnThj0iu5SMCE505+rE0xn2Nt6lHz+sUw+zaIafmVb6emVH\\\\\\\/4HRJcGd5WlGxm1wlFbI7AvHYQROYMvy2Pcj0LbLaglCiFKAD78ECT0xP86gemXEUdd1OOXOi+YuFzCGOTe9xrwk2KV7A2DGv8iByV0krc+oVXGL5sWBKUZm7EQuxFubqxzfOojwV6thy+ENG1MQm03sGtVZD00Qc6Cypu8iUcWzqKZgh3FKm9lZjOunENwxPfckzUWWyjAneCReDUQOESw2rMlXKQoJ3IL5f7W165XbpdgAeeObwwKn6jJ2KavkkSaKgITo7QkbweTsVtRUwiikqN7vbuzP2dqsCwZp5fi8BV8jnSe75rIvJtTg5dv2wIBht7FElAivZNoGJ+AL6AF30odNmUaydrkYRXC\\\\\\\/n5g0rcZpKZAnKRfzXzpwuoOKHYCTFtrtvM2XhRAzf49ZM9nKVq\\\\\\\/jiE93QWnXUTIzXc0bi6s1BFJwXaKDGzIRpDZ3c5g\\\\\\\/c5z9A1dyvYKUwWKQwFXezhmz0FJaktHhB\\\\\\\/l2fdQnG7agDXWEBqDb0fwznoWheiiai\\\\\\\/dz+vAy4CfUrH8PKI3eH0TIRq7oTHrKub2m9tW6dJQKkKhflP5FKYe37F07mvovkMQYT1Dg7EzlMm6nIL+RmbqBFN8zlhUUCHpzuevrJfeVttPBH3HMh6CWHb+C4eQ2dM6nlDDBHFMFU1KbU4fjwvdCcg52YAXS0Tje07wexmIyy1buzzMtqq3e98pqENe3zxJynaiHHtbZnV5z00Di4g+iO57SauVOWwhO3JNq6B9qgTkR2SSSUev1f2OFxN8ei0JDYsdYKkt8bCmJ9\\\\\\\/xfrz1wL9hN5uOen6PIzE\\\\\\\/zc8iN6uWlK3xpUc\\\\\\\/fhdlvxsScdo7lV5d+bKdJUQprGuiTdi4NbIlwEMJTcI5JwcprQS5SGbS8pgY5j2ZEiGL8xOwpgLY+vdvNG1CVzm2MrURuMPwpx9rYaNsvvZfIuweDMW1SDEyaTbprhkCSG\\\\\\\/KA3ytKHqRd6HiF+vFZuIYruomvEZGb4STFRAuHrV21tGSv7KkC8hkUmQbOnBNgB3pwaLftziZ01kYexSvYO4FDCzKJvfIeLtXsuMWL\\\\\\\/r++jxVyn+fRXckKYi6zdVfcnVTnMN1\\\\\\\/Sqo6FtLtZK0\\\\\\\/vpASdKGxAM1985mZxg6eynesV3SuLYQ73PaehHnI77AH9FxCI3C+bsfpU51C6TsotPJ7oL8QZwS63SX17\\\\\\\/yX1eELlXXaHqXOClzrACsEk1QaLtL+kTvoi82+oJBloFUlv12zz3MmJ+P\\\\\\\/f523ldFT2wzvohbgyI\\\\\\\/WdY8ggvdBv8fatJISHgm9jAZZdcKO89lI\\\\\\\/9Zeqd1wh1L0ME5dMYqaGMxLGIc555Ty0jbpqKPFgqznnwmqfir0nMoZDOxf7sGjmq9LPevmnhApYnhssNsRaHzdmdOUsl6bW9Sxtnj6Mi2QjY5XSxGgvDypBPxottOkBFk8n+ugWOuAq1Jn\\\\\\\/T7qeNaHu0fQAhIEFPFjhGG7T2nOZNww\\\\\\\/iqE6NVd64rIdsa5C7WGcHQitV6gpQgeeAV7zbTWSmMXOX3P\\\\\\\/813LXaI\\\\\\\/bHwkC\\\\\\\/rwWfXlXQa+yUTBnHskNn3h710DJ2AxoQLcuKQogHk0gH3chE3Ag+MdybKwclCyOfaz8O4S8fmPXZg3RTKrpgjrOo5rZyjfWXtwyPVqOkB3jAX9K4rl1PRoMWpN8vHMfY5WvGjfe5EQyXOCxp3ig4qirAINSulbPJYOdTivyYdo1yGDrhCAsCe+gylA7lbgPtAhVpqPWf3LJ9rin8vYDyHHBlz659Yp9xqYx5cwB+mRSENx9UN3DPGUs\\\\\\\/m00uUNnmgg63PgKbGXXu6FLe7AXloH2gH21iUlhVLqkgFge4rLYR0pq5SAjzzVdrR99jaW+dPxhjfJs1MSX4yFUyOvh3pUGo2bwMq7PqrotXqPzcILZsdxGF1ZRgqk0225EIsu4mtxd46cehiOJW\\\\\\\/Qx8NY64XhYEd6Rp6ZUPlNmT1MvuajYp96y26ruCWqmSWlcjGFNY3+K8EjjoWpvtEE\\\\\\\/hEQgzCdXZWjRurqaOhwGhkD5u\\\\\\\/qxO4a8eWH7LFE5NBTWzxSWDqC3KhAbJKQalqxIXAIFhxd01Gob+GfPcRFlXo2BzWZrWOz1zR+b72rcYz\\\\\\\/SrY33NlsHlZallb3cNxcGVa\\\\\\\/VElw0XBw9MW3uallUUDLwJB12g3grWngUsv4GMvGW1TKpR7wM6BB1XZ8+coqRwEijjAdWgURfHuvSFUQze6oGT5Ck5KGYW9WOgkZXbsi8EwvERdbi8k8Gl35QsVfb+oz++ilRP\\\\\\\/eHSnXTPm3Nz6x3GAHKdzOKy4fr3X8m051UVogftC3IzQ3VMipX94ugMemC9OG70Zjv5wM5Omrjl2OFQuKtbiQ5HvPZYL5bTl8HxdW6hW6rbgaHJ9zn5G1M0QR012q5GEuZDEn95VoTVUe8UyKsMfIu2du1sTkoOPCAsZSefsoRBO7ERxyNDSj2AD0OiemeRFbfMb3RgBvLYdGxv60q+0Be7EaAcyZd0ZEy9iDU0wih49Ikh2EaTRVbaZG23ocVJlibFj8XI8wGP\\\\\\\/K+miWz31+M2U6yW8EQnIk0eZYM9N\\\\\\\/luOiCbTJg2RYINrr9sw7hxscaKXGpUW\\\\\\\/XpOnzKGwssz86f72KsfZAnPXBO8asNokM1EPS2bSXFksyGXM1UdgX6RlBG8agCvYyBCxA+2xvnF1tsRdD25DnL4wEOPs2DzZb9HCvEGTQTM0yOAAJvanoRPJbxpIO7a4lNhYrcp+dYq5zZat1NhnEWKpsefjUoT6tb30gfs2640KXPSlojL02eBt6mByL3cpwtrDCtvfOQn5GMy6iWOL7JB7PTVMkRcDj86yiJIT91\\\\\\\/ozgL2MCUkpMwM42KCQw\\\\\\\/Q2wkBHBFngwYAvu1tRdhOPHGLbB\\\\\\\/jJoiOLLjWZ6Xnm\\\\\\\/Tq7WPlYkPciI2wvKbU5TG3S2yl7261ClMfO7jnZS5NaEBCGEqDLCjsPnPQT\\\\\\\/s8nslgI5Dp9Aa61mgGzGVUge\\\\\\\/TENn\\\\\\\/ilN9iXES6utESW9ubpvPd6oG\\\\\\\/baoMEwDCTe0C2\\\\\\\/BcQlQkY5TmpBTrLr83E5u4V5CeYXWPgAPDmIsFXoZs+cnW+khWEogPv1Qmj4uKd+cogBJt4KP67YQQTfA\\\\\\\/ZTxsVM2g\\\\\\\/38egfqQv7kWSDdTHkhdE7g5mZ+pKdO+Dpq0KliXQK5t5nJ\\\\\\\/8MUOqtGgPgrmurVJezKgCaKHVQOxcWLFxYkmjYzkHCHDcIhm1wUvM9O1zMuDk3Hh58tWAhaJde3DlqJe0Lg10743AuqbG0AyKJIkI9PhqWyGC+MTgOZarzXjGJqogdnf+KND99KXfcLgJlXYXMql6pYzWeomVTygF\\\\\\\/5bRm+JoH1HDAvZLGnvEOhb6FKQUu\\\\\\\/jKJ5fxnEXw6iylSJ9lBwSCH+rEqB0SQRkerH5w5K54DEsOCL8sUQssCluXCghRJpI3koq\\\\\\\/Qrx\\\\\\\/mmKnO6rcY+f22h4YOgwbxeRfOKDWXufpiNtzhXcFCAKhV11iXvGW9sYsT+f4g4ht8JNB9nHnvVQKX4vQaqAZr3MOFjNMBGkBef+xp1aMU+VBRVtf8o4iYXNOG2ECq0R83k9ZeQzdhLQlL6IRzNviyk2xohelbRScBnh56yd0q84RDFDIx6\\\\\\\/1hboYXJDzaGtnmSa59qF5Qf\\\\\\\/YRUJRqO8r7yr\\\\\\\/NsYtVpuuujOnYiULR4+aF77x+DSgEh1J5SxbztYGc88WDJcL+fcWslp+wLLjNjcFpS6WLfEcAH3\\\\\\\/NlLqd\\\\\\\/eE6NTnKKauSVXXb7PojFzk3LVzcCaJ3xIgXPAWgQK\\\\\\\/RNj1VRyfRLRWiadSw8yvmRtSxrQN0YrVm3fenVC9ZxVhcNnHfjYfg\\\\\\\/vW+lvcz1JIELARUSHCOxRh8Ptb7Ja+isClmSRCrVhXwEJCaka9jSxR4xSlWY8J4muvuJaAuiJWUtsa5sgHGilQxNKwty0M1Lr8J+XEx2FpJeb9M6IAgdVDMrc3sH8JV\\\\\\\/mZbss8xZ3p4vo3DlihFzzk\\\\\\\/fE29xl7lY+cPbBxq61apWiP2HBU\\\\\\\/qLMcHCUpChED17nRBzQDiytYrru9CoVptvV3ZOxdCEfdTazzKI66\\\\\\\/PyfNswuY0ruT1DwPB6UpeJuyPdH5vrBCdfWxLDaf9r3bTfEIPh2eYT4q1aQG0XR3+grulfNF\\\\\\\/woyfUnCGnP9VDj0J+zt02Q01vawp\\\\\\\/4x4NmHsTVO6AGGpMstR8C0lk8BWz6OqyEndPZPIwSjRX+DLfmcgae3uGgEYZRRcx+Fv83pZG7OqSwgQxRQIgfVtwivsTyBizWys605em5wOsXrVqyWOyq9aUyjjhOUbOtW0xGO9evRzLqx2K15IPz10QCyG6eRXX7DKgw+HBrHSRLVtRqQ0byNOs0wHl84JfIPc4g0d54cEwQngxf0YD91Gk1tgPuqj9HeMpHZroOkRh29vUdguMMILnPHsmBYFATiyzfdcdbNV12\\\\\\\/HIU1Ma1+MiP20r1ainYAmpRPg3b6z42iTHvyddh9yXBE9c3oApSEMePXh4Mu17dDFUbe+1kTQPvBkyzkpQFjDiVhmfklqKeHhJIvzVNaEuWiUdSo0WnTmjVkwVfAn91hh6b1\\\\\\\/bvoyqmN019oCTmNVRJ8kZiTf3B1O5JJVIsenmqKc6o4vk\\\\\\\/NM\\\\\\\/EBt2wQy56mrD0lYiLvECNIU+qyrJ0z7EbBKMLk+R3xDdSsx5YivOcUFj5pHknVjzoINvVgh8KdgJe5a8mleEGk76iNR9g3GuYRwHTdbDfyHGHPU89Qqp3UoFkmaDnk1vrfKP6mi97Mr3SHlyscyRRREjFdFJcnh2A+tS2kBC3Cn84MGrWEd7tgN8ccOy1qhDly9mw2ZrG6ExIU0YfxsJPNsxsUYi\\\\\\\/dyeoq6+N3aKRcJ27cnq7p5311TTbDStMy\\\\\\\/F04SpkrjVm6XWwoE0ck4su4OdUjsCGqLnHJMycVy5gdCJ4PqTpM\\\\\\\/3k92cPwrXCqSkRdU7WdyEr\\\\\\\/+527Upssal7k5En9SN0Y71R0PSE1RhGW\\\\\\\/p\\\\\\\/0LbECUsgRTlsFfoNIL0kAj67nOo5PUoxmxngwUJmJI7z+pWzBWXHKC8uY8MOz3mExiv+VsrnkZRrgXr+YSY173rk+2oI7gobOfVYdf6aPxs6EMvMs9eiYGl5i6jbdMqDZPBrt+IkpvNrvFkRco\\\\\\\/X9KWWAhsadigmze\\\\\\\/guqLQZM\\\\\\\/jyeqqokDHKXMZudc8TBhV2YPDT4EaNbROlpc+1mtyRGZivaQcAsXz18F7qGRTLYdTxBVyr4PHrDso0QqzJt5meDde1xcMBocH\\\\\\\/YR0HgOxMDX7rfinmJ6Mz44H+M9tX41RJPG0CZkf7bvmD2K0PEPTq323dWxNH57C1wtvlsBgKWspRMdmCHLvSMqSXNAr+ASXUqUZXdiMVSVAi9+2eJSWtYP+gmilqlIt41cgOogW9kjDL5bJtf70m9FjkOiqpWMC+4vNeEbuVOp1bYCfSeLC9KXHgO5IIOWmCIW32iHxE9lNJe25+a2121t6vhocQ7lytss8cKtQxSM1OJCggoSIwYhp8qdzmSafVYwRPglTI3nLpDpWl1Iaq6F70bwls7HHbp\\\\\\\/XfoPEE1EPxx3SG7Z41HYbMz4IB+0k3W3kgVXmM4Qrk\\\\\\\/vOu+7J7fNHHQtGkFPKEuCCAE0eIKsMNMayEY+9GPurvf0WiAckK540WNMIahNsZ+OTaFI8Fe1D8clSME61DmW2rlCvKRtw2emLbSVLWb\\\\\\\/NbypkCCYsmBaX7sylYMFMyLEpvYnXIFTHDIqNPhwG07WWEPXukGcQ3QqK9tjtRtePbgOZWPqhsZI53oXl+6eevf2cZ5HM4SSgNpPQAVvbnabKpqygfXX9UAmp3z1ziszgFI8BC7MJT7jxUoeQgeqCF2LGV8S97pz+ZKahW77nf7rvUs39SaKWI7n\\\\\\\/sh4xJ0S\\\\\\\/KBBzVYhOA2Cpurb9\\\\\\\/xMWEApMLgX\\\\\\\/UKAXtZgxuUIlkEiKbJPNSXWrQWNy3L4MUkosi9rfrrgRhgXn3+JlSBAA0uXZuH9+O5e77kmpwldp0p10Ngk0x\\\\\\\/qXDSioBhGfFHDnATqOkrDfNk3ZswCUaDlbNyqddTKYpXL+1VpJKkmzqdO7Oj1Hw1QKzwwEuwSWvwq0mHOt++qZnLQjLN1Iu1Q6qtCRiI2nqDXyM0C1edQrxB4+VfhnmEYfruAV73A7XMazDwQDyUCE0NNdzX2scWL17Po7JLOo0jBCRHQoVvnLFyZlgAkfsgqHWb\\\\\\\/yTG4KFEWrVbtST5gf2dcW7WvGtfpMn\\\\\\\/K3eltqlpGNpDGy0m\\\\\\\/uz54+eezHDUNvoYucIUoNjzqvrT85dzmFE3qXv8Acq+8eBVAKqD3MQpr3o\\\\\\\/yd1el30KD8ADiLkLt3GueUhKIL9IPpgRVpmr2mJJ6DBVdmUW07AkDJgpoNkzYmUJUErRogltrRMAHRBtdhptNZlB7C34a0no+XjyhjClh3HC9t2f6DlPiq2wJmqwt95c5blZZJqYd8LHvOIgjZIvUrN6pDnKsceiKSqaeKAA4qGR7JO6PM2+2k3TFHjhtQ3tj3lMBUMKAeaMFR6YLHHrynkiDki67O+OrKtOJvlNrdYTV3CcfUb4wCUNeW6hPvS1+Tbo\\\\\\\/dwN+rN\\\\\\\/Wn++z8YWs4mHUuxbB64iAL2QfnVrkpqXuxxKg1GiMDPXTRE5KgIcNO9XLs0O8ssqApsaC5QH0WM4B83HpytqZD0PLYpvHWVV3EGP+lUB94N5rnHTXeI42+RBSyiEVEPJq0sLq\\\\\\\/iEHPI3bO7fZQNuVe1LVPOgaSGH9U9VhVRphzMRgbMKhf3BOQdYidOLacLRmQ1qxFVYzSrtjsHD5s0iFa3lOSk3tLN\\\\\\\/Hf7Bg0FiLd10SLPLga41ZpItCJjB1HF0W1RUHZ9o4maBzahwZ2I\\\\\\\/RnlDwUBIkDkEP6XOoGuZSs5Wi8p5Om8RbhV\\\\\\\/rmSZ0EmrLp4V7XyTMRFJ2SA7BsEiiI1b9Hw\\\\\\\/MO0g5t85CT5FGYdXbSZwamTu\\\\\\\/EsCRPWXt3MBQVs2HI2othr5TLOlpEm\\\\\\\/CtW2aYCvaKY6jFycWTLuv\\\\\\\/2CoUFUa9DtePH1tq1d7SC8\\\\\\\/XPBnxz6DBiyEkXyBkCKSncEEK0XK8I3DU7YSTxPvGuPOMN8g05Vmao5lw629rFj3WC05LKW9tdP1NTeEfX5ZEmNtX9rTQz3fiqRUwc4j7NRBN3VOue6EHva6R21sEekZstTzP4bikVSGXnK1DyJh+vtrVM5k8hW2YFi74NW0Gl8+Xe0nvyr6RMAU4Bnif3mWBettEhZ3zZyEUAG8wbFpc672an0teKIxTj6CAWetTM0XMBpMdDbRiVSRidfVKQZ+qcyCReo\\\\\\\/CO\\\\\\\/Ntaq2PmgpdBCAkxVzWWxiuTBiuAvN282sfiwvlZhRV2M8hh496m+l7BVfh3MwzL8eFQvTK3nFc6JR+esegFYentUY\\\\\\\/vymAtCTPhn7Hmv\\\\\\\/2bO1nHViM9Qwy82HOAvCsv30ix1N\\\\\\\/Yr7LBnLON6AuSslVKh9pUULDgQGeLPJSiPkRVAqDBhvC5p4TY8cIcNsM82Kzip3fftBJlUhdGDzMiWLgwn9\\\\\\\/1sP\\\\\\\/NgFEGcNqumjWHbDTweYU5iFr3RJ9YBMnDaxhGCR\\\\\\\/4g2gWjhUtmn+OEz31SsGGapHuaCo0UAJV6uCGi4T\\\\\\\/B4Z5peLYgS5t3OdFoq1VG3GRVAVkZme\\\\\\\/JCcXOMPUbI+lUaN3PtpPfDKOxWnVEczx7Abhs68hqsV2nbKuOiPwExS3HBDVaVSExU\\\\\\\/mI6mW+B7oK0LA9MYTBtcfzdldDGSIzu0fZWsBu1bxQUivoNhtZkXs6HQfTNTYUUCdT5WBtYNeMxXEpIUOaH4nOCy+5Xz+8DnrNSVEmXc4gh1Lr70aSiKyjFebuj7qpV1qaoF0aqgkU4rVeEMzfWMeWt6P5B2fr5p5Qaf4ffSF\\\\\\\/OZHxAaAqslBXrO9vm+tnBUqMZw+McsSk3NdSBCHEhaGxbIZQU34ErSHpkoexrYAJWba6j9TJrUTlY9XSqVaU0Yh3AfbP5MNwG0fP7DZ37tDjk2qEXSISEA7HoRky6I8xGtSq4OVXfPtApy3DTqTPU8uGop5PAvEzQl2eMfTg9WdkBuBCr++k2A2auzBRTwjwZif26jzHZ9f7BMKeQImcfRoBZO0PXhG\\\\\\\/6wX2gahtQl995JA+nmzDvaOM+KwoBiopbgyQ81jiFGFO4X12TJ++ReBSFfVeXgUGMgnITCKBBFUISn44kIccj9e7lBAO2IzRsagfKNkwLvhu9CkfN6c8ujx8PjcVcybN+9Zo1TLoelyCamGoHy0iP+F\\\\\\\/eRjmTye3rpnt+JYRSzBz1s0QhQa6IUB3gxpWHiz9w+mmcHumZlPaRMWtjRRSzdDn1V\\\\\\\/9hDCCc5TzOJL3XEl1NtrGN6zBLENQd73mXs\\\\\\\/nfpRNvAcWod0gnCtwod\\\\\\\/iG9IawPUHmZSqXAG5PnRZ4syHpUnaNQdpheqYLzZVQFV0cTgLy1jF0na\\\\\\\/fiPZDzwcT8FWtYbjbyZd8XX07\\\\\\\/C+9rrhiaeLaUfEjhTskB0lz3ulLYtQJbf6nosu4bKQVpRLz5iaMg42etPj+yKc6u\\\\\\\/d4fUCoU6MkaCNzAW0n0i3KfoJrJ\\\\\\\/LX0zE2StELCnYiqKEFniSMNF7w2o1Mw41r7zlYrIvhAJflwWCG+sCVwTJxy6EKCTVbXWQIYAOaQILaKiVmT2EqRcgk\\\\\\\/VjhJ+dufIxpWM\\\\\\\/mgZ9mZkSxWtDv16ADZYKU8g3nz2vQD1gE19GGncP3VkLEwFXa2WVbtRsAg1+mGnaD3iDKUWwcmWUIj8m9m2piLuNwHCTCGim7jGs9DjQdvRKi56\\\\\\\/c1yIELxs92fjQDg9fVaXT+Y\\\\\\\/M7l7j\\\\\\\/7QN+md6sammeAQ+CXH7X8XmnvJQKuvF\\\\\\\/BfQrxcDe74Kf+d2Q6bw13PrOxce5p6ySF4CtGDDfvGIEHF23bxX+Jw1HA1A+rp6uyfns1ZLpApsbGmm+IOBzfSdEK8tQFThmOD8EeN2R1PdK1RUTL5JLwRmBOhf6VpgRWJagVvcayQ\\\\\\\/ASzN4H7PoAtQNV0oSEuBJCnhg8UNpp1FaZkmDW+aUeB45nJwyMdgiJcQa1VRZoUe8gZTD5uDNKgAivEDZTg1xta18+o8MSI7q+bw3AmIGITW8K28jaz8eS2igf+DY3CZiEjTmlfhyqCHPvPnL1nZPI5veNv9wOSciZzwjAu5Wv5Nvokkjvwj1r5mVh1EsPiw\\\\\\\/qijrYCa9gOwsF4I\\\\\\\/2WMJwR\\\\\\\/Wlmbi58Yg6u7EdJPXWcXHqz0R5vGl6ltCHV1Ob0odt1PNHlUQ4A8GS4SVGSSmHPwkDWLj7TIORgPuHZIB4S1vqoM5xyMWVlH0GrOsKR5PFIX5ukt+Pj7\\\\\\\/Yo9l3Wog3O\\\\\\\/7mDt+l00MRNMNxhe7gBGWbGakVdjM1Z0q3F4eZ96fIjh6K4kGVIHnuhrKCcocniHs6cZNndMQ35JHQCV+irixz7kB1RUpPZqzl197jvF5P3XKtLCIKqKNMbBPRMKusz7kXqyQwrF5m5VaXu6\\\\\\\/ZNPVYt+uPNFml8UJ5FxMto6bpzZezG6GZofzuwk9j5gY1VFu1GpxWQrT6KIVLQQFv0GN0GZXaAK1uIHx5qW3BmcMltRuylgJajtu91aZFxEThn2PF9pGhqvHVLAIAt0mo8ap6enkxxEU1UYY6RSzqVdQ88fZfc19BJW0P57568PFF1FFPqZd31fux\\\\\\\/RATkdLavB8aNHb+o5\\\\\\\/pvTWvlpMnJgoXISSFMY+GOdNOzkCQPsyr+pjUy3oD4O5ozsxLwF5jrBoggsnFiR6Nn6flxIN8pwkC2HFFNBTl+z81UAJd9apE3F42j40EDSocNJlbnhGvhU0wL9uuCaIN8iqLTUGPrzXFBoIrr9L9y8kgCoj432hKiaFYlf80jJTPTIOoarS1x\\\\\\\/rF3R3EOa9m+\\\\\\\/MmkYAVyFnlLV6vwVQyZgHuDQyWFcm1fyYJVmyCAIN6Iq6BphKEH7AzOaqv8fVvFKnnnL\\\\\\\/2ARzyme07IMWcN9WgWjnZIxDAhvPbeO9\\\\\\\/MfmlWz8Wcso7QpLdLsKbo+Oo2XtypD0uJTtlrKcm\\\\\\\/dgB7uwSxuARGQAoZU5FCKH5imzyiJ77pl+kYjMgRZu\\\\\\\/YdbdOcw5rQ8hhjsTAzkrIGIzcwMhvkEGr2NnxGlNa0zzcbI7bOomiIFxraHLu2oU2UCd7jhBzHvAufI7djqUc3UG8xkvf18rweEpD4g1Vy63rtJrxMeJ0r4388V7O7BFfrBIrc\\\\\\\/wfg\\\\\\\/LwbQPFeg3NZ4pxGDf2ZSjzLdX0y6bBjksmoYXBqcxXeJmk75gip4hUzCsRT5+4tQ7fDwtJ7uhsOKDhj4ZDHSekud1vi7NDtdbn4CqduT6e8fu3BK9aHjSfvl73eKmo3gq+pQqz1\\\\\\\/YxXuE3HQs+s7Gu0fkv9kI8yomH9tFgC3pweugSEDOAfrxYFqFD3FiPGa7ijRqXrlyC5quqW6H5hExRyWh27uqrFTFZvuaOE4vW+mfanBuyK0yH2E0af+sQTT1ccwsNcvAkHpEpWNheVvP1bISOUkpr4rvOOIBFl8g\\\\\\\/OGTjQCzH3DdKhjFE661hc+ucic3d2KOAgKlJ8U9NqMD8wzkjyQylb7mWERP\\\\\\\/wmqYF77xRj3VPxbajxX97nBrNbMb261WzCVPwFKs41l\\\\\\\/EuBtshLbH+Xz6j+RiYTeglVwbdQE7RjcoC6eSKPnqiJ9jTJ8svGIrMIFK4YHh+jj6\\\\\\\/CwxzDv2vpGrP8sh10NgQrebnYl77JwMh8G78mYa0h1uJmFScnkvhfXZjb+gTpFthNJD+mZ5HHwl4M0Z9S6H6v8Ya9qEL\\\\\\\/ozQ\\\\\\\/24NG8c9hatE5\\\\\\\/KwHCT+g5vdah\\\\\\\/81TEoEyk0ohAMbS6S+\\\\\\\/Q5xZq+Vh12qz0gACPuNh4sw\\\\\\\/iIxGE1zxBZX8zjntbtBxfouXmxGA6BqKk5EoXvkTpnqejMTWKT\\\\\\\/RKKjXGPzRmFMfpa3rNqUQVA8rHMImdfRd4bJwP\\\\\\\/AoD5JjP+H9rMovfKvCpaK7VgrYZ9IjcWDocGsi9ZC4kDHbCSkeL4x3aoGJ8tQGXNKpbqbeD6V9IxpYE\\\\\\\/Nc4BZ3on8jMHty2CmiMs6SWAr6m+EQTf4KD0s4oZC3z58\\\\\\\/93pnyh+8R1j6YOLQA5dvegczssPFNVcFxROnWBUelbPIftuE3isQuDfv8fivfS9z0TZp5mQcRx4FMZEkQr7kdMXyDNorv9G\\\\\\\/bpYeaQt8NfRyJXMCkpMuTLN2vSfklxwrioNyTHf6BsxjASOpCT8vrrWpWuI3uHg+5wEZRguMaa9e30\\\\\\\/R2YoxHF\\\\\\\/ap0BuPDJYieSayBJmCYz8\\\\\\\/KQq8a2MKtSc2qVINv3iWA7zas1pzpAJ2H5b0Ryy4UX2wUiQKmTRyfykvspb7TRohb4VYYNi5ZIOw61YddWNUn7\\\\\\\/xAYGZUgGCaBz6dkcjcgN0QcjlueOq6q3ZkOqMeNpoQFiUv7lGa00YpzdzqvY8xCNbVsxp8YcqZ+z6aLsEY0FdTJZDKGutd\\\\\\\/TpEmxS7cWxCpMIwqLJsbid4\\\\\\\/BQ4aiSMe5CcZMTEDANL\\\\\\\/VA0FftV8Czw8Z\\\\\\\/SAHvO9sPqq+2dCAVzu0x0EQ8gLnjy\\\\\\\/N8DQxIfhLqek8DbTCzFytgwKO59pK\\\\\\\/hFyzNdtbRhrbCXoLwBVU4FEGZQVNMfBqB00hKd\\\\\\\/FhuqghOERNoa0yc8Gu6dDQFiYjFhytZzVECAJ1A9AbaKs6sMowpJSC\\\\\\\/z0R4stLdoWsO6yOHChJ2ixm4W9YbbEOCiNEFvE4ntjGO2ZjOtCz83vYQfLK14JoF\\\\\\\/rWhaBNR6imSPG3MAQygLKearmdpEpMAJuQo6spvUQTRyBUQSBE4mM3iPEzXWwLU3VLQb8qrILzcq4SItn\\\\\\\/dJ5XZP5V7WZ32hjFlJnyxBSRwmOM2olTBAQQSa3cWcejNBDiVP+cwICxlh5hAVUR2GLFKUQCpguTZpVZpylqHkDGy7pZJEuHaY9qrD9MtDWqSOI\\\\\\\/rhCA316lh3MH\\\\\\\/fjYTFpMGPLg06GJFKY9060K7ttZc1Zm5x5KixloxEmsBD9zFt6u\\\\\\\/Y\\\\\\\/gq4apyCJeqAFVm1E10Z5w1gOKKvSkzkOD2F4ctu1ZMahC31xhjeHxEEjV79d9MwzkiQNLhNu01kZvvTZ9jyKfkQPUNQuxo7Q56VPAeSO\\\\\\\/X2ys6Ka31tX6XVMDWeMn5yuNxEzXk6yDe7YwiwlorYFE90UD4fDz5UQmCQeApz\\\\\\\/CUSc3BXD3QIojlYyQtEkds34L23m5eNWx1hvdo94dwL4aVtzmDgdIjPdmylSAlulrV4C0vRJz\\\\\\\/GQ2BpnyRr5HC6WkCo7bcHCJrXHtoKlUmy6+ctNuwNhfgVgY9QAgnPPOc866JlhsTbL8\\\\\\\/CGvup6Uu+jTJE4o3N43S2v0\\\\\\\/vGQ16C59rsaS0O6b18vCpdMtY\\\\\\\/sCuqvd3reufnsv8FFVSW53YqjHV+FioPmVXCg2V7yEvl5nN\\\\\\\/ZyYG+H69NFoDILaciZPEh45PyahVl3j6FnGeVus369cit5t99pbAiYP1CncrMfwOgVzGb0x+cSA2t5OW288G0uHXhsSAiBiSs8RPGDy0mwZCRCOSyfaou7nB2T0TLS8OjXGDDK++eRx0A0sUrxMj+F1EDpAhY2ohh60R05v23sKO5NxgT6nsicrrvr9vsfjuRfAvM67CenWrAWEyY5P5EzJOoBr0BR8kjPBFrtpnnR6876E\\\\\\\/fVPlYdfCu4V12ZI7eBqHIeAg+PWtrmWbA35ZQTm9oeUDbpk+NVTMJciZfA0YUjtix5MowtA0e4kKMyfB4DDWH0mbdGeeX6iwHoHm65U0Lz\\\\\\\/i98ir9FdCOswU\\\\\\\/jzJkdTAkQFHWhc26yeOMho8tUddOx7C5f3pUPoH7JTkhnXCAzLFfCG9QQtaCxGpVAaBluVxNwk2Ls2hmFSojKCXbs8HxTdg7wkQN3mq4d9T\\\\\\\/m4TIaR+c1IJSb1aFZJKywrCigTm5Hg9\\\\\\\/mZ5DxZqwOm2HRephq3rXtGgiQBASKtF5oEPZpdFOA7sT3p+9eQxBHdlNXCk8Ec729Xo2UI\\\\\\\/kgV\\\\\\\/gOiJNw+eOYL1N76YI9y6RurZdyxQeuLDy42\\\\\\\/bgY3xBo8tx6u3jImoxOLP2921UCHdGQfThdIGQaRsUiF\\\\\\\/ECqGTzfnQqaIF8U9GJaQuuIE6G8CWhF8dj5ATWvo6\\\\\\\/amTF5H8yHsfkVAwmcGxp8AOWXFGbbu3paSXjq1exiTlzE1gDnmy8fBTs1OV66OgwHujIOu4eIkMLEsfdQF\\\\\\\/HLpTqk4S4pEgamRipqn6p81BqL5Fd4u7mJP26JpHcQYlEopWJKLjW+BGXs4pXz8+TG6yjiewayLJrE5rFSOt2Lb2ldhtPnWW0yMkNAIjGH1tBnZ+xHCpEUcZzXCOBuxe2owTK8ApDZr9LQ9SJYIlT25gzeHUMnW\\\\\\\/HQT1QoPpVX5zzR82lH2hJjjBhtwWZYoPk0r83QOMb6UjW9DRePNZxx8z1hNUsKFZJwHRtIvniViaoXSbTTRWXOEOiOF0NrjKP\\\\\\\/\\\\\\\/8RHDAlw1deNsJcM7D\\\\\\\/jygM0gJdDXg6NjP0Zi\\\\\\\/6UW9MH5vVF\\\\\\\/Mk+e+x5x0bfO6WCvhl\\\\\\\/EAIhnPkkLGHhO4G57\\\\\\\/3PGjK6NoJrbl\\\\\\\/vcc5+xnE\\\\\\\/dhvrn3qgweh6Ttuo12z1ML0UxG76ayKJt1hLAGlNIikAI+jW9XqelZqSAojqqNtf+DwQdAuoJ\\\\\\\/AJJpZCgnKuGRGzcNb8C0EwA042UdM+NHLRkIzcViQy0AfPu9UJCkPZWfrywaQ+YhLhZJ\\\\\\\/w0H2ibbpQNchq3n\\\\\\\/C8bGLbqJgO68IrTER\\\\\\\/rX2RhaBV9xLmxn+iqCmcpQE0NVjiaoq9VzHchqjc7D3mZnxmJCLv9gkg73xptksOtsFtqpWxxTMiy8714W3yp6+Hj2h75rMr8vbmVi9vnqM+dWLfKAoGVUhOukcFxAIvm72nq3ad\\\\\\\/XBYgaC8ryB5Ox3n2WLUKL\\\\\\\/yOboDxdroH7SZANarxS26bTeSnXj5uvqjSoDQoUEWmDvv3CH2AtFBcBaxOan4jnb5hiyIgqknmCOw8k2PnROBKoK3fzQ6HdXkJMm8FzWBgRzWNdFeQYNUDK\\\\\\\/lIU72KMzjZKrcGsn7aMXUpNsnp6EVfxnyi1XTgsqXTXr4w\\\\\\\/+Xk3t7zvCcaHv89QvroodCbCHEAd0wLkyADq9cnKoTn7OSX7NiZv129omX44SMovKvzPJsAoxvOqsEr\\\\\\\/4OP5LZyX+sR25aSTLq\\\\\\\/PG2kjz7+eQ2dtD03QsigsqT5XiyQvL6kh3ckGDj+VW13dIpDyzNEIwhWIE7PfM+3Vwxn+hb2BnRWoOo5TKqOTF+z26rheV2n6JSqQj4nrkqfCUcWe4E8hZBp+yMojS1H2Qo4PkfL8UMRwYwwXbBleC\\\\\\\/EJQ0FfUKyg2Mo18SZbfL2\\\\\\\/NVthSoKTxYQ9omt52IxEip1aIeEyOZQBn+hPJsGY9LoksfgjAay0TErZImhEhBLbs7Q7IX+NCDVXwcsUUwzVKL\\\\\\\/yGcZNJU8u7UFik3L2Zx7VhLtQvJ6k\\\\\\\/zoQFUsACyyimGhW\\\\\\\/eHrnq7l+YkXpql96H9DZiY2n7ESXuAj6+jnrj9CCHy4tzJ54gE5u0L5cFtgMS1pPFuj+pxXwZF+u7H1pj7AqkhpsfQJRHG7GAB7Wz0AM\\\\\\\/zG7xsfp1VDaS2OVMDV0dGhWB8QL14GKHh03l68062wuX9URSo52S2pkcaCrg6tdJASkdK2yW8qLkUGiIN1AacuCuylKbQpyRt1I4u2W6zg2tzc1+ifUpKK45aghFZ+yV2u7K43Qo1xQKEgpAgqTjQQ\\\\\\\/7BdfO7VwUkcXGofAPpDgzcHyo8HC27Wrxwm7btjm\\\\\\\/tD0YVvsBCLuitF4LBAbHgnvQpyAhsxkkiby9UF6hAzCJDsRIcItRPtRKRxG1axRnsb9QXCdEDua+95ecknnuhTiKCJ9LS7az+eNp66VHGegVQccPp5nxLoYIgOWV8iuIiM6TBkk+D61K+UT+v9Jq+zl\\\\\\\/VRluF12odKtKWi05kWeuq0c2EpsRMRF1+IjTeL0S8RnY8NZhTWEHSy1RdkSBkTVYk7\\\\\\\/h2NZqMLxyX3OLqLQbMKYJYLVyQHkwMqg8xeetFZrQ2K+vKc2\\\\\\\/db8RdxnFmAjfa3eFQ1qgMzytNDeTdJhACcVUgGC5VdMMtoERTC8IjViksZlMkcQjavaRyw7blDrSe2+e7iQWHhNI2uC2TP1bdsOYQ7uTykaWiai50Gaxbj\\\\\\\/zFCLrlPXQx+XCj3+ZLIZ7wsCP0Dx1rxTwOBqjlbkc\\\\\\\/7m1fuaIMxlR2kqNNyYY\\\\\\\/7rtl74fV+4FCLqSqf+5i\\\\\\\/BBbgVeAEkWDtuBaZ\\\\\\\/w5g9C2VrGcCDG+TQAqUcJJhjjou8GDOaSp9Q0583Zgpi3g9x9G0lfW6QrNlPYvaIKe4MUUJqr\\\\\\\/v8T07h6xu5UyNAMDkT4ZhtdLzpiiqAg+y7YcrW1vTnjkj1wwVLRCA76IOmynEI0HhCC\\\\\\\/qmw6gkvBukjHLwuPGeXAzqX+o8866iJYanAYN3LefJv0FDyedD0TmS0NwZHjaH0Af35rCQyZ1bUlcAaSKU6+avt98jkl7U6WaBS6J4IH4JLmQ0GQctO0nItlJQOLoXSUYI\\\\\\\/TMJ39iZfsce8hZOjTo\\\\\\\/G78M5glGKDolVbAPSex6PPK\\\\\\\/tv+3GHQaNrGssb3R+8TzcnxNdjOadg7Tp3dVjb2vfxz0E0vwGqBBS6M51NZcyDa0DDiRJeHXsDvSxbD12IfrSUKHP3PnaJlSI8z60pCCl6mzAFz6FYdV8bz9SfduQXphcBQHRYImSDRhkxhkppQtw4cWvdvJinziyy093m+E+n\\\\\\\/P22ProwcVhV9YaVneG0bbi3fPnurxvNA819XcONFWV1wZ+PWgL\\\\\\\/fRI0V7vhKAnWSKpkxch1QttkgeMZ0E5RMMC0u1TzCmf2MW\\\\\\\/VLrqaiG4ONvPxYBge8l\\\\\\\/kiHcnHK31w77ZBY3hlp4ytRuSGhGRidB14BDD\\\\\\\/wAqsUNK2Ntifa6CTVFjzFXDFP1Lg\\\\\\\/nxtn\\\\\\\/pYASDYhBwi\\\\\\\/e4UvSIqx2gpy1tb9UXu\\\\\\\/Gnn6hAwGkrejaZy12d3r+qw5RKu2EbeXcAx3oqPO+KD30SnwLRCVjUlHWxVB7HM9WcN6nAGrTpYQ2vW2ij9xpbkoDL7gGDNfE8fif\\\\\\\/6TzR3j8RcR0rjEigF4MBsSBM7yxUbJ0CgnRbPw\\\\\\\/Wf7GnMDh+C3gVJt1Hv76EtIHfFipNj94RkSaE8MoGAkF2XAKxOl5Xqh+T6VgTAp9M5ehV69TsNWhoyoFz92vZYaKcmH5opm5swbynfE\\\\\\\/P6oMZts57nTxVYrQYXUAFofN4DoFUU4iE7ordRp0XfuEPttIeN4rFTTE5AcNL8Jl\\\\\\\/b8kmgtzmIf1L6IfkBx22GuCdIA4NM34eH7W6P1pLGKn\\\\\\\/hSsZdScXsAMMwRbN5j7NSygxDuKBhAyUVYQG4wdFBWtqK6RjaHI7jkJMKbvfZXfV7diZWLHJEF0yrImr6TOaQp8d1zhogeOOwCERCHWXCPVG5yzXv\\\\\\\/2HpT6uNimFT7DKIrOQW3bpyD0rQo8tsTcsM1XMQveq6da\\\\\\\/E10JR+xXapR8pmrYQzCne9r3weuTL9QmhUaVAVssi8sFsLUfhwkK5pj+G+1ZcEO4XS7xde7iVxWBPvomn2JrqkXBEtJ4w5Ix9bY+b7kGZ73FABBs6y\\\\\\\/BRm6pe\\\\\\\/f\\\\\\\/B3clJND5OEALuI1ETdUfiAPe9svSHGg4op1YC+sF94fnmiuuUkOwpoSRvQClq49j1WDX1Egfer06fKrkL9MXdOS1k8f2sD249wxpaFGHYbm+vwwwhlI9KSkppKavI4xXqmDDrF55oZCaxODui7I5kllR8c5n0z2JPUEW+uvfPIfIaotZkvTwdGU0A8f6AvuVBov48M2ngfmJooHz9PlzKfqixP0L0G3hmfgKMdc\\\\\\\/PdArLgagDWzxoQIfa39ZFvDT6lKW7ncqt503+8Dj5zEw7OZZPdXDNAou38q07su2l204tMTlJLAVZ+Ukoe4sl85oWA2rdyGMU0jzJNfSzLWhlxgLpGi0p29SCsaLkGESyXnw9Bq1H\\\\\\\/6CDEZVTURKBxcvaSDOh8XIIUeUui6P2P\\\\\\\/7JbmhFEPlmMzUC92q9t4TUPaMTmGB5B\\\\\\\/CqjvnTwNA+7dAkJl4VTlqmMXuX8HLMGkNmvCUDZ7gHV4nVK16NM9YGg2F2HRjUp0y2uifiXsZh\\\\\\\/2SvgW31xiUXkYd\\\\\\\/zjyNY+EbHh8AQrU63GXeuhmeCN2PBoSqOglJCTYTTwlIUz8c9C26pavJAwpqLSCRR4DMqOsvXm0xKJLfbY5Cq3tMLM87G7GAoIoiRfuKaQKClrOqVopmv2h2BcT+CgrGYfczQEtCXH9l7xnSrxRVy7atQrGsFCUIe9cyVD8Im1KN2h4IW1cSPHlLmBWadcwSpJVq1Y7vbbqxoQonL5VP3VZpEYC64RdW\\\\\\\/rEudFYtEZgMGmh7v7dihQYO5qCrR4tFcB6QN7QA7BEbOEc4FoqS9lOWS3iGgt4i0iAIIBFjV4Nd+z9imqOcEPYqDf84d\\\\\\\/BxIEh9t7t4ybb7zqTfsV\\\\\\\/SEGX3jI2njApSiby6DuihO\\\\\\\/QxeQlep+FFVBfPuKZDKkIHBEwMkX8Hk\\\\\\\/LsrdkCLQsxek\\\\\\\/zvP\\\\\\\/J+X3VVL78YliFwn5qeRLtk3Rzi+kQmpXBROK2GdmXBZbKO6q5UNOdz9wafxibPuLu+UiMQNq1ZEu+hXWQ71MeQXeRzqVKhiWhLcTNYcNBGPwbzIqP\\\\\\\/ktjfYtp5LYHhLLH0kIvbD\\\\\\\/MhCCxijmh6F4mE+HUfaiKpBtHookw0MeSTr2mBE3WlEbuZqrEg1nw+LI7FQGuOoidfcSEmbamXRzzp0w+AkBc1ie\\\\\\\/6s7XuJVA9C5vILt7NKODtfS3tBs\\\\\\\/zGm\\\\\\\/WZx2\\\\\\\/iz553i1rnQm9gM4SDTvkWbV3XfNd40tev2Rq0GIrxoQeo2HwmrWGtCIGgeECiMFlbwYET9Pv1L7HiDgN1eXPDHZC91TzjUdG1BZmDtXptnsWoNTjbTj6sIsaEb6+y29Bxt7J5scAUDQ+b3XgDZoxTHvUu1U+LPykViuNSb7D\\\\\\\/reuY+HkeGuOv+I\\\\\\\/XDX8InCD6e9l7MQNsFB+d\\\\\\\/nSCy5zGE7rsUQ\\\\\\\/meJBbGGqgwlkanG2ly18uKX3StaQzcv5p9a4TjUZEqwJHvyINPraeQj8pJOrJAShXSeAvHdC5qH6yeDqvf8QqobmeNSDkOsToC\\\\\\\/atZaPXawz8d8cfoRrAloH+G7uJ2soMxGD\\\\\\\/MQz0zsdtSV0Pf5wx1y77YyQqHgjSe8Sy7YcXR\\\\\\\/vkaAhchtbPNCsg+S2aqwW\\\\\\\/lZrYcASeqd9roI6qxNTbtfFdGbPde3ClHI+ZELpMSAqcwnKgESsaf2aav7+8X7KNEJbIinbOJw3ANC4MNfQleyDxERniiv7FVZfje2e7qtglWjNenpB2qrqme7c+IR9ZvCMjhBeVa8Oflt9Mt\\\\\\\/r4RuBmVMzmJUr6IgOCjI3SfYlwh47nHlH347XWstHI8d0ZcX53Y0grvH0WR3lu2wM\\\\\\\/rSV98VLGmkZbqGJ4Cy0HhpBLAIKp4PRyFpnca6XofpGcfn6Xd\\\\\\\/hdp0f7ifmSAVhm3wNQBt+kqJNyUYVESiqxdZH0ILBv8HZV4njF3rkd4nM+n1FdWDup0SMTp2sI1PkeRWJfAc8ljgeomvDm8XBYyZcXeIHoQ1bK81C1Kso6eDwMDLvwmldcy+uOF0lqIyGTV7CnW1UoCr8mELmY2WToiCQmcxogijCaPnLu\\\\\\\/Vl1fZJChhuN3wUEIFVaZI6P0pQnunSgFIXZsm0jXK32cZ9+1G0M7ldxlognrsrfAuTZ09BGEr9GO4zcrJz7qnleqc1BZ637mZdf\\\\\\\/efN1MMdc2eN+qgLBRvIIXUikyMFXdES1wzsnCOe3AYNGcqww4Slt9hLI874BO9WYyjAboDsEqhpc2tdY2sg9sDXk6oo0q98r8xef2mRQDD1RhIRJqkBQlG0U7Dylw3dPmav+70uXTpgvfhTdOsmTv81WbW+luNkI93ofBiucpMWi3NkzuN9SyVsWRMF9o1EAzWTNbyOy4KZcMfggqHMHoEquHS\\\\\\\/BFIXm7sCnW3GXZeFHAWsG9HCuO0KTgiukOz6ESl8yfjB33GPA\\\\\\\/P\\\\\\\/OAunyGdqq5L1QF0dgYxfLQBUhJg5lXdFoluYrIwCeSFva6K35fKdYk8N2rY6kUo63yJ2L0tZsEJhhxda36qZZX1wjJLWrCCgJry63ENBqR4lyl2Hj0v\\\\\\\/UWljE+ZOCXUezwHF9H552NYqbEGWRmCGSOARnH\\\\\\\/fengNaNyfD3k7TvlSZZOb63uH5qNjocmzlA8ED4t\\\\\\\/NUW5MQZOrDjXw+2D0NIItamJvRaFCrr4lGGrvhsl+iv7QsN60O1gemDt2KyGXosN7XEA4FKv506\\\\\\\/\\\\\\\/ZguRTsGvxcSmLqXzysG1fqJEh6jaxrbtevcEtiqJiLxPNgcH0bnu3cTcviyWl2SBf721AsjItfumr843MwxWIdw8KGGYte52oF2KUHbWtcyo07IJciwjhi9CDN0fVhrDdtbbqk0hA8cD1cAo8BkIx\\\\\\\/oJR9abhgJZDi3GkIu0XaB6G0V\\\\\\\/8DOD5++fKlJ8\\\\\\\/VmFMIifLe7ot2ZrwJRJQO2FzwhghHcKJ8sR4xMAEmuUw5QdPd+wuOtsWnxiVaNa8DXImmkuO8SnIPBfUNX92Btj1UcK1zhxZMnPQgoDd7FOpcV5HcwHeKLTcLftSOs6pk0\\\\\\\/0v0i1iU9hxYFNtRbsZJQsty1ZVvA6ugMaWwMnlXwDUqr1okDjXv1lY\\\\\\\/MrdkDq4GDXlScDWiwL8lNXgSDOzBHjb8B8nbA0HnldzMOOI+xp7alQlng900vKwMq90iTfZapc6shrQW3Lv6V8UA4TfwLMgLKPqFx5V1bmakdC7s99Oy8sWg6U4jlHpvV4CaaQMHDlT\\\\\\\/OtoDGqgIKBEmiXNWAsFR8soHkRcevZQnUobHXGZdJUg66rN6j+n+ibRFeWtkvgKFBuNZEf+HGT2pzyN0w1gL7gu3FoPt8KwyeF4sgdBXSj4KJJnWHdyAM1TCy2dN+SkU\\\\\\\/Fq7oVkSfegjrM0\\\\\\\/H4abHp+reTxJATYkArvjsvV81ELE4Mrp9DeKKusA4+I9JfU95jLjE6L+EqHf0eVkiRIwxlTRTaNi0E5vn3KVJFytX+4UIKX6tvEJ5RJjfguLo2d9vHJM+5\\\\\\\/WpqJnlhJYZrObdk+CWuD3maxWdQDyTrNcTaq96YJ2cJDr7IZgEZUjtU7tsu7Cxba54m9B3aT4fBwlu0BvuI1hC30MFZ4h+2jp0E9l3Lb38afWEcLzuXS+OyycdyCKauB+UL1KKD9Y2Td+kIir1zWlL\\\\\\\/sdsUPj+C8lCtm7D33sfeGMwW4zxFlOH0E6Na2R7U6Gl2+6VYV8c13+KM2zsNuzH37DYMzFUNqIZ\\\\\\\/dHNIWaEP3SmHD\\\\\\\/SXhys22Mnxd8OBEFLWwj+DlXoDMmpSZ5j0KVoMzO2Ii6aOF6ynZ+3O7pOQAdRMjTcvP0daPE75uTO32dSQ3r5vDAHi6N6Nm2sc\\\\\\\/Kn+unjbAkxq4f7P66fkRHipiEKMbcQiSukjVOTElolDDEC9b6QkvMl3GTcG+2TNYrp8Vc75KncCDT+qKyV3yjZe5\\\\\\\/lI9vQPFYaUBwh4ElTVfVz\\\\\\\/VxAdbi9UMwlZ9EsMjy6V+L4TFdbegC6pk3\\\\\\\/vu3wndFVIfbJHM81hgQWadrm0FUtQtfwJZYS7yKb3a\\\\\\\/7YBPnbp7T2Ff4a\\\\\\\/R7RESyDEwXfOtyta1So1I8DPN3WUjBWNT3tZEgxW6olj2jscUfs5q0ZNuwLMaLX4nvNJungetnZG7x576O77gcz49d9QsRfXTFEc21qhZ6svdBOR3NbZ68r1y19gn+g1DXE4YsgCCfl0wdyDBgToyGwd4l\\\\\\\/AlWj+MSLDLDf\\\\\\\/KKeoiaS76guBDDILOsbmy4fiarvMvKg3Ijr\\\\\\\/Ct8Ru3PsvKL+M6j9XD3MeewDRYBojuqHksNGrdOLfoNaywLZ8R+s1Z4H3h2ZYrgZoFE5sDWlLOemIRmnn0u0FGTbb+VJUUkCGoZGnFSXplIXN0wFsIiUR7v+M61lE6i8fO6MVCBNMBWxg4WazQilndHmg+dZZto8WP7X31miMnrgaPs99cUBQuKqP+uhbwBbvIw8+61iVHdtq6P65F0dq2kOabh4sT7\\\\\\\/feV2KCBuII8KlNwG3lqbF5LLyxPzDh7JpqEk+t3Cq\\\\\\\/ztRlS1LkXj0jnSfmf\\\\\\\/dLTEDSjwFCbg5fTA9to0zGx\\\\\\\/SX14d\\\\\\\/L1v3Ooc7Z2YvNM6Unbf2ZiBAz4CGFHbb2ls97SP+9JQ5zk2QUd6Mmk0gJa8fw44O3OR1Te6BkUBipqwRT9hiuH+wkypsJtEMMiCTvfVXefO21RWrxJSQifflvbCqVXo1Zj1TZJg\\\\\\\/Q\\\\\\\/quNaDAu\\\\\\\/1m2bns+f9Y45MbSebhw1L8VbKEJjtmpAEcaXHXNfEgdJOWbiKhjtEVZsV0xizMrP2kTruNL5heeT\\\\\\\/BcV5NeAsSou3QIZ8T1J5f299+uE4fpqFmC+JIDIPM1wahShcfs0GaPSU9lKlqZ7yccF\\\\\\\/DkaTmG+NK6LbnX3OKk+p57JIkc5xbrz5Qluo+V8Au4JLFn7XKNIw1kk71oZxousifR4leaGIASRtm17MQphAMSnCUtg5tEjXv1ol\\\\\\\/\\\\\\\/+UcvdMQZf8gFOQxy0uGGQLcQ44PaI7irH7HGFAl+lg5JlWXCa9Vs4dATxMggiZKFBtuNIBAqyrH6TmH3Cpjx\\\\\\\/5Q7x1qsUeKDQH0bGUb1Bzot3gslm2y8iPGRYYNGHtfYQKFeBZsHMLOr8b1v6EVhNdyEMR8ke2ZKAeESsoeagjOLtqw3DtN5FnOdt9n98eDdKVzujFXNe8Gp0TK+PRBkI9PWmz56fOWgMcSkHKBO17\\\\\\\/Q6t6M0nhiTPxk\\\\\\\/ioZOMHEqZ7KzCE897GZCPUgBJZFSr9\\\\\\\/g2znbbE6PIMh7wR113c\\\\\\\/Qd6d9sh2+ETGgLiEkYYS6Y0+rRafW0mJafG+DB+xi8JzD1sNX+twc8T\\\\\\\/eIecoxKBXrgjRveRpugyq4tIVUXLX0+cudXFRkrZ821MM2DW7036\\\\\\\/CwyU0\\\\\\\/qZ4qVkxaVsbzXnRgVv2AAoFoT6uFdO4Kx8TBL6IgVTdgQxKh5nH\\\\\\\/MgjMkm0uHNQCK1yc1nOuN0QMTUR8k5aSHOpSqSmJcsVl6IPouoY2YgK52I7kmiv48yfmXSzAsK14PyMHuYKOsSvBP9yHPMOaBMIVPTvtMZu4huQ8Uni+BHxr+T7iNJ0OA5v4JduDo2tH3IIZzcvDw+EohjTHo7myaXR9kaN9G9ZT0kIUfheC4WuO093Efn7zdd0Awkn1IiAj5g0SqEnTdEUsyW5QcTakSIa8OSISNlfGnf58cZOmqBmrG9r1lCpIwuiWfXQpXcxfAQisi36\\\\\\\/CRbZD5y70P1FBEup\\\\\\\/KtMk\\\\\\\/xudCrnHl+ksiH3J5fiyuMy+B21lkyUAf+cv1fuDdsvleebSiHCsHzN6FYmI2OUeyk1rT\\\\\\\/gDKdXTM3V4iONLQZnsNxuJQldpYQUEci6D+POljebwIOBn30XjWx3DHSxZkKlEHOB0K33ftbB4KejAq7nCWgcI+PAYL\\\\\\\/QpuDKpXmqw1qN7r41p2q6HfWGngIPcg4xxeQNYfUsXB2qh5VEn+c5L+cP+RLJg5EhpqM0ltz9twY3eFr4OQnX65Htn1e6ptvXxS3jZU8eFlA2FBXUb3x9IISrtUuu2wsASnRD5JbiluoaYh89qyTa2U7NgHcNpvnwvni8QFo8qRPu0huZ4k2u6vU57K5a8QthiOJcsFHNKU1k6i+smq8WrbiHRPDeivouSDRdERd9yGaXvjIf55y9Ajfc2UNfTphu7fixhwg5l+6Ge2v0pcPTJEkGyQOxtHvBbeB04ryXTz+30m\\\\\\\/3O\\\\\\\/oEVKGGtHSi9DxUQgmofTcrSDP\\\\\\\/XHUJLTgtiif6NRDn3p3u0YQRxFge4LFOtAKDphg3RwmA5xgyy6W0xlhGL9yeq15xmINpmrRwdScfF8n4wlaJorcCC8T5A3HzgReTQX42GsMCjZ2w0miO9XRF6RGifrOVLwPfKDthEchZmxK7k2mVB98\\\\\\\/ofwsO0MMDH4kckFB9mzGaBWKa\\\\\\\/PP4ztHPZYYmNrj1TJSDRD9MVCubxXxHuMC0NEv0+cVDtitGtblTMiEwX8M2zE3k1NMLRW\\\\\\\/YX749MMFaq1bv2oCSxdiMIb2dVknEZkjNtKTLxm8omoU7+xzXH+zvK4XBOqf9FSy+etZYnOZKfZlpP2J+3PF9auriewV\\\\\\\/Lv26QY8fzIbY8FOT33zzgFDsIBQDWWm3jJsvD\\\\\\\/HTf1kXuftrW7ZrUkNedrXmlTnZVYuSYyuQzVJk3L+SQLJctwmW4pg39nDNd5M9UfmiBiyQOELx392FrZiBn5xsrJVb9sFSWneZq3o+ydCpJrrrdf9rlDLZa1Kd8kMmh\\\\\\\/EE+PwNqnETuhjkEM0Bo0IyBIynHueJACLLEHyMcINE8q9uGrXPGg0ppeSnetLp7BnrMjRAdaI0ovuK+7k8hGdkct9VITT\\\\\\\/uIhhHrVcjSHPpE6beamu7gqbNScW\\\\\\\/uwEYwuEkk5nl+yBgfYAQvIN0\\\\\\\/9RrPwxeeKFksONOgWBzNWIUZuyhgaPefa1nIwzjdKrR\\\\\\\/tVL4ETgfn\\\\\\\/0B\\\\\\\/GUyLAaMFglG4FXsOzzTWOgigyUgBKUUA8GmvFTtqpeHPo22lli0IvwFT0h+VMu3\\\\\\\/BPN+rxjjo004IkwxLEtFUg08bFU+XtG+C1RMC7txxcsS7X8MMUwp3aKATT29gT3OXrgwboCuC5qm0ZSZ5w6CvpJeowtQSz9NlzkFeP3a7xJZXoNlY8w\\\\\\\/n8GviDHOOC2Zky5mBMdLvJAQZxMs4FSdy1R3by6l5e9tlf0vFt6NVgEAxdVkjeaQwjaCAJQ+2d1Yyd6S3GFUcHe7xSUm3dUDvx2XhkVooL9ovmIHmcnhqi9d4T3+OIyx1OCicgcgn4H3DHEk68SAWPJdGewtuQlwSc1dVebQ2cd1zMLyNEXczE0Rk2cbFovw6rnyPqqoCBThmfU\\\\\\\/cjsuoeOXag9jHExaFlRIJPv\\\\\\\/EoQvkyBXMS5PeWaDOXUJxWLkXo67alkF+EepT7sV0SXEmxSbpHWlOIhUJq7YS4vDwYc9UpSLWgRMFnLuD4fflNvyfyOYjpRluYwNmG2KYHNjEitWuONRvL7WNpkQ40tEhJBtlfYW0lxZQ\\\\\\\/F7C3V0Cypg3qZjXQ0Gohr74mAG+7qZdbILJYiLNOKW6jpsWDVIGfH8Nd9dVNF+FhBx0jkJxk9pX5ZgU6opuh1zeK+IBrfOrmL9ETLNbRF0NiQgqrgCw\\\\\\\/uUPgDNfpRiWDoQq3NCqDiM957+zRxIv\\\\\\\/tOPrdEIG3NBJDP8pGa7oBXU4Lspg91R7rkcEEIBt5SizOTo3T1pj3142TRIM3tCGPAK4Hga\\\\\\\/Tg13jLn2mby3rbBd3ERftt+5BnofkmG26eNseMbzBrsFzxzbnwTHCt3P6Mx2POzQ5ETUsOjb8B2CGIy2n\\\\\\\/4wfGUcPxEf7cjAn8XrKM19NVaT2TCYnisFHjr6\\\\\\\/42\\\\\\\/YdbCMHT6Jqs7UHEGTGiOPOckhVchdfgD6j1ccojnRjEhOZVrY8uXWQdplgiMrNG9Z+YXToGjDQILYy91J50LR6fcb\\\\\\\/MG0\\\\\\\/hmTJEXVH9mxp5HXZ0GDnZGkjkZ91iTfOKVAU\\\\\\\/ftdRZzfSQrS2zPT5TwWebL0TIVGvAn9vFtnkV8WWCeykLIljfa3yGxO8cCnyYHNs6S\\\\\\\/sz5fwCMbIX04gG7uuanT8P2pfLUpILxAq2uw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Nr\\\\\\\/94QBOfZWlKYOHhK5FAQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002297329a9323a029f44a8e64f61c20b13f24cdbd231ef70debceca1b56586db442\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dCtY+rmU9\\\\\\\/nojs2vx7KZMkE4bw\\\\\\\/GXt2He2GsqrLbkaEU308uvAh3Rhm3PjPc9W3EsUJFcHs3Kt0drCy5hbdblp3CGqcwtNuBSTaXZ0TUIw658Q73pJG0N3EYoZfsJ4iiXuIFQozzrL1dODOZ5KNXkOyNzlf\\\\\\\/M0VD4+RaudamMSBe1aQwTQScHpW4ysAuhRcjRTVhNqKq32F+zERbLl4qXecO2+J2f4SjuGHZoWle+QjthOuJHXv6oBhN+isK0BqvlxiWgh4RtnHuYK3WPa4SXbXSpJ2aSCnq51HvMu9C0gdseT++V8n2Os4T0ml0bmQJYxs5VNnGmQImPfx5sClZWds9A+Vim0cErs99GqVvCXwg0W\\\\\\\/qxVYN4E22GueHPFRqA7KmH+8WzvWScGmRnk+vZybgozNUe9j5w15uQN2oUz4HX5Q15Y04A9SuuXKQYt5pYprSR5yaRb465UjvRKvSrhvbtocvVpkJSXoONTYvT4Vs8+3pyM5D1LXOPqGP5iBxKiucgYwCttJFVe8a9qfollv9xOVntFXP39dLnLjf4OIXLlcldDDbfVFwjtDcrk3+Fqfj43fo2Ltg+ZIGkTk1wWGFuneAi0WrsIYCz71CmX9t2rYSMdap06KqUvoKsmnxAnj50ej8R6wxnWDraw7tNaBvxeF8QNOyHDQ\\\\\\\/cBF2OhUGfSx8dOu2xl05CQ6mYPIOMkUj5C\\\\\\\/0rh6vMuGxYI++sCuCb1zKyCmiADjkTXsuXyvapkNI9OsSXYz\\\\\\\/Fhi\\\\\\\/gZRYD3LwyblSGGb7ClaS+3uYaeEEmpdpTZ0o2A\\\\\\\/\\\\\\\/2+VejnrpuDjn5Sv9B+XCcSILuIVvrEA24Ne4aW5+FtYyLOxOyJSyhyEJ4w9CPUi0TRycVdymDPJVKY7bPB8g2qee1ItKZ7iML81k115FqXeIW9TAg6qftrH75crHEMLZGdo0lFMwNuA11CVGiVoB5EsqNWktlANA32ne+6+T161\\\\\\\/NLuJ4gy3ZmQbPJE86oBR9kDivXXn0uZ\\\\\\\/MhHwGNy\\\\\\\/tFNKDNXMRCDJYHddYUJN+2\\\\\\\/3IzQEY6TJkKzv+Ar2\\\\\\\/YXJHAHNlxWwCFZ+5+nynaigHcTX\\\\\\\/v\\\\\\\/axcTxPcjRNVIrGJaN1bItm0aO8\\\\\\\/XiFN\\\\\\\/WIOa88FRXwT0CSvnuqfRos+HY9Nbk31Ol2NQL0rAX1fyXzmlInJoVFy2Wpr3Z1FIrlfB1YtFsyTnv6m1hlwhuWbILgUsmYwUALRm7KlgBf+6ciH\\\\\\\/4XyzJ\\\\\\\/it4XnWCFz5NMtduAnhiDdctJTjBZmbuQDn0jF4PC8YRswrM8rokeJhBDhBH8rXuiRJ5ZeTVj6qclUekJaQi9glyNR5OVcieUngbjS+toEre4\\\\\\\/ty4KjiohpWvrvztyXGVV763q4AEL4VCfJfcKSArzyy\\\\\\\/btTaDdCyhoJbtMUC0iKDBiQP8qHpYO1Dq48\\\\\\\/En3fSI2u1u09z0x3hCbS7MKSftL+B6H5kKcBTmFG\\\\\\\/yc8whs+iQNkispxJOFzrtcV6ScYRRofTb1VVjeVACJx0kUGnRHIdHW4+HpIicUPLaGspJOYQEDCX6svLIepK2rO3MVNZp2I2Vq16unzwm7kX8lq8P2R7MwOUA7K2R8Znggi6LjiYArPcetokJH7KoKL8VaYaXgg87L\\\\\\\/+\\\\\\\/U3yFNO0gMIgcdqk81lhib3xIyI4K520hBkGfX1+gNmkATo1AOiM6oigCHrOn\\\\\\\/b9h+ATrxchcwt7FYgBw9YjS7anf+Q1gCm8zqJVsRCk04ewFE9OGCnsCyl5rB40Epvw5uwVXljwF3AjsBD\\\\\\\/k4E9T49qixx3IMKIIdr8BEUidUSLMHVh1YwjVtyhe0L1PYIdgncOWC6SnBPXTo+LXkYQUxU8bH0gNzK6GnEX8jkNwhfLHweX9tdScypeBveF\\\\\\\/J8uPDOEx9hPIgx4Kc5RMEIUrRhKwsfnGW6VqeHzkBaLgKsKg3FQqHsvQ0uUbcmr2X9mvNhsFBKJnUfsT4odBjAlnMUJSx+0lVOfJpm8e5WBVnxx7fTIx3b\\\\\\\/zQ7w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AKN+H5nnYosGOYCHf3jl4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c69ac8bc72948b6d8755d826af6568a516177f7c5df0f52bcf7ded40b7e7fab1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459674 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1667" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LR5_7xC4gSlgkcQVuL5C63i5BCHjtCoKqkKShcj8PgA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1667" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XqUCqhdiGkxOA3oNZ6vDlebKTMfES6HxiR_6YYTNnQ8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1657" title="Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1671" title="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459674" id="node459674" data-pisa="ggg;4/9/1667" data-pisa-master="ggg;g3.114.012849" data-apath="/ggg/4/9/1667.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5527-8729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adam Eyre-Walker</span> and <span class="highwire-citation-author" data-delta="1">Ying Chen Eyre-Walker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1667-1670;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012849 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459674" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adam Eyre-Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BEyre-Walker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eyre-Walker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BEyre-Walker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5527-8729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adam Eyre-Walker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:a.c.eyre-walker@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.c.eyre-walker@sussex.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ying Chen Eyre-Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BChen%2BEyre-Walker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eyre-Walker%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BChen%2BEyre-Walker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/9/1667" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459674" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459674/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1667.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459674" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459674/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/9/1667.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459674" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459674/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/9/1667.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459674" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459674/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">It has been claimed recently that it may be possible to predict the rate of <em>de novo</em> mutation of each site in the human genome with a high degree of accuracy [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a>, Cell 151: 1431−1442]. We show that this claim is unwarranted. By considering the correlation between the rate of <em>de novo</em> mutation and the predictions from the model of Michaelson <em>et al.</em>, we show there could be substantial unexplained variance in the mutation rate. We investigate whether the model of Michaelson <em>et al.</em> captures variation at the single nucleotide level that is not due to simple context. We show that the model captures a substantial fraction of this variation at CpG dinucleotides but fails to explain much of the variation at non-CpG sites.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/de-novo-mutation" class="hw-term hw-article-keyword hw-article-keyword-de-novo-mutation" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">de novo</em> mutation</a></li><li class="kwd"><a href="/keyword/human-mutation" class="hw-term hw-article-keyword hw-article-keyword-human-mutation" rel="nofollow">human mutation</a></li><li class="kwd"><a href="/keyword/cryptic-variation" class="hw-term hw-article-keyword hw-article-keyword-cryptic-variation" rel="nofollow">cryptic variation</a></li><li class="kwd"><a href="/keyword/mutation-rate-variation" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate-variation" rel="nofollow">mutation rate variation</a></li><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li></ul><p id="p-3">It has been known for some time, from comparative studies, that the mutation rate varies at a number of different scales along the human genome, from variation between individual nucleotides, to differences between whole chromosomes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Hodgkinson and Eyre-Walker 2011</a>). Much of this variation has remained unexplained (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Hodgkinson and Eyre-Walker 2011</a>). However, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> recently have claimed that the rate of mutation at each site is highly predictable. They use principle component logistic regression fitted to a dataset of 653 <em>de novo</em> mutations (DNMs) to estimate a model from which they can predict the mutability index (MI), a measure of the mutation rate, of each site in the human genome. To assess the fit of the model, they count the number of sites in the genome with a particular MI (<em>n</em>) and the number of DNMs at those sites (<em>d</em>). They therefore have a prediction of the mutation rate from their model, the MI, and the observed rate of mutation, <em>z</em>=<em>d</em>/<em>n</em>. They find a very strong correlation between the logarithm of <em>z</em> and MI and infer that<q id="disp-quote-1">MI was highly predictive of site-specific (1 bp resolution) mutation rates … and could explain 90% of the variability in mutation rates at sites across the genome…..We conclude that our statistical model of mutability can explain a majority of the variance in site-specific mutation rates.</q></p><p id="p-5">However, for each MI value they have thousands to millions of sites. As a consequence, any variation that their model does not explain will be averaged out when they consider the observed number of mutations. This can be illustrated as follows. Consider sites with an MI such that their mutation rate is 10<sup>−8</sup>, approximately the mean mutation rate in humans (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Awadalla <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Conrad <em>et al.</em> 2011</a>). If the model of <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> explains all the variation in the mutation rate then all sites with this MI will have a mutation rate of 10<sup>−8</sup>. However, if there is unexplained variance, the mutation rate of each site will deviate from this value. Let us assume that equal numbers of sites with this MI actually have mutation rates of 0.1 × 10<sup>−8</sup> and 1.9 × 10<sup>−8</sup>; the mean mutation rate for sites with this MI is still 10<sup>−8</sup>. It is clear that if we only sample a few sites, then the observed mean mutation rate often will deviate substantially from the average value of 10<sup>−8</sup>, and the correlation between the log of the observed number of DNMs and the MI will be correspondingly weak if we have similar levels of variation at sites with other MI values. However, as we sample more and more sites, the mean value will approach the expected value of 10<sup>−8</sup> and the correlation between the log of the number of DNMs per site and MI will become better [assuming that the model of <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> explains at least some of the variance]. Because there are typically thousands if not millions of sites for each MI value, any unexplained variance will be averaged away.</p><p id="p-6">Here we examine how much variation there could be that is unexplained by the model of <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a>. We also test whether their model explains a source of variation in the mutation rate that is both strong and likely to be difficult to explain – cryptic variation in the mutation rate. This is variation that exists at the single nucleotide level but which appears to be independent of the context of the adjacent nucleotides. Cryptic variation was first discovered in mammalian nuclear DNA by noting that there is an excess of single-nucleotide polymorphisms (SNPs) that occur at the same site in humans and chimpanzees, even when the influence of the adjacent nucleotides on the mutation rate is taken into account (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hodgkinson <em>et al.</em> 2009</a>; see also <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Leffler <em>et al.</em> 2013</a>). A similar excess has been noted for single-nucleotide substitutions in pairs of independent primate species (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Johnson and Hellmann 2011</a>). This excess of SNPs does not appear to be due to ancestral polymorphism, sequence assembly errors, or balanced polymorphism because the allele frequencies at sites with SNPs in both humans and chimpanzees are no different to other polymorphic sites (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Johnson and Hellmann 2011</a>; see also additional discussion in <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Johnson and Hellmann 2011</a> and at <a href="http://www.plosbiology.org/annotation/listThread.action?root=21243">http://www.plosbiology.org/annotation/listThread.action?root=21243</a>). Estimates suggest there might be as much variation in the mutation rate that is not associated with context as there is associated with context (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Hodgkinson <em>et al.</em> 2009</a>). However, predicting whether a site is cryptically hyper- or hypomutable is likely to be difficult, because the increase or decrease in the mutation rate is extremely precise—it affects one nucleotide and not the adjoining sites—and it does not depend on the adjacent nucleotides.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-7">We downloaded details of DNMs from the supplementary information of several publications (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Conrad <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Iossifov <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kong <em>et al.</em> 2012</a>; <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Neale <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">O’Roak <em>et al.</em> 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Sanders <em>et al.</em> 2012</a>). If necessary these were lifted over to hg18, the human genome assembly used by <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a>.</p></div><div id="sec-3" class="subsection"><h3>Simulating data</h3><p id="p-8">We simulated data under the model of <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> as follows. First, for a dataset of DNMs we regressed, using weighted regression, the log of the observed number of DNMs per site, <em>z</em>, against MI, to yield the relationship between the mutation rate and MI under the Michaelson model. Because there are a limited number of DNMs for some MI values, we binned the MI values into groups of 10 and removed those bins that had 5 or fewer DNMs. Using the regression equation, and the number of sites, we predicted the expected number of mutations at sites with an MI of <em>x</em>, Z(<em>x</em>). To generate data under the assumption that the Michaelson model explains all the variance in the mutation rate, we sampled from a Poisson distribution with expected values <em>Z</em>(<em>x</em>). To investigate the effect of variance unexplained by the Michaelson model, we added an additional step to the simulation. Having used the regression model (of log(DNMs per site) <em>vs.</em> MI) to predict the expected number of mutations for a site with an MI of <em>x</em>, <em>Z(x)</em>, we multiplied this by a random variate drawn from a lognormal distribution with variance = <em>ϒ/n</em>, where <em>n</em> is the number of sites and ϒ is the variance in the mutation rate at each site unexplained by the Michaelson model. This yielded the expected number of mutations for sites with an MI of <em>x</em>; to generate the observed number we sampled from a Poisson distribution. The logic is as follows; the mean mutation rate for sites with an MI of <em>x</em> is <em>Z(x)</em>, but the rate of a particular site is <em>Z(x)α</em> where α is a random variate that is lognormally distributed. Because the mean of <em>n</em> lognormally distributed variates, each with a variance <em>ϒ</em>, is itself approximately lognormal with a variance equal to <em>ϒ/n</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> 1995</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fenton 1960</a>), we can simulate the effect of unexplained variation among sites with an MI of <em>x</em> by multiplying the expected mutation rate by a random lognormal variate with variance <em>ϒ/n</em>. We generated 1000 simulated datasets and calculated the correlation between MI and the log of the simulated number of mutations per site. Occasionally the simulation would generate no DNMs for an MI value; we removed these datasets. We then compared the correlation between the log of the observed number of DNMs and MI against the correlation between the log of the simulated number of DNMs and MI. To take into account the uncertainty in the relationship between the log of the observed mutation rate and MI, we bootstrapped the data before performing the regression by resampling the data points from the regression.</p></div><div id="sec-4" class="subsection"><h3>Coincident SNP analysis</h3><p id="p-9">We investigated the difference in the mutation rate between sites with and without a coincident SNP (a site with a SNP in both humans and chimpanzees) as follows. We assume that the distribution of mutation rates is a gamma distribution arbitrarily scaled such that the mean of the distribution is one; it is therefore characterized solely by its shape parameter. We also assume that hypermutable sites destroy themselves when they mutate. This assumption makes little difference to the non-CpG analysis but reduces the level of variation needed to explain the coincident SNPs in the CpG analysis. <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Hodgkinson <em>et al.</em> (2009)</a> have shown that under this model the probability of observing a coincident SNP at a site is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1667/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1667/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>u<sub>h</sub></em> and <em>u<sub>c</sub></em> are the density of SNPs in the two species being considered, <em>v</em> is the average divergence between the species and <em>D(γ)</em> is the distribution of the rates. Therefore the average mutation rate of sites with coincident SNPs, relative to the average mutation rate (arbitrarily set to one) is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1667/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/9/1667/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>In our calculations, we assume that the divergence at non-CpG sites between human and chimpanzee sites is 0.0092 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chimpanzee-Sequencing-and-Analysis-Consortium 2005</a>) with the divergence at CpG sites 10x higher at 0.092 (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chimpanzee-Sequencing-and-Analysis-Consortium 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hwang and Green 2004</a>).</p><p id="p-10">We used the coincident SNPs found in humans and chimpanzees discovered by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Leffler <em>et al.</em> (2013)</a>. We did not exclude coincident SNPs that they had inferred to be subject to balancing selection because there were &lt;300 of these of a total of 34,000 coincident SNPs</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><p id="p-11"><a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> assessed the fit of their model by comparing the mutation rate predicted by their model to the observed number of DNMs. In doing so, they averaged the mutation rate across thousands if not millions of sites, potentially hiding unexplained variance in the mutation rate. To investigate how much unexplained variance there might be, we simulated data under their model (henceforth referred to as the Michaelson model) with and without additional variance. In the simulation we estimated the relationship between MI (<em>i.e.</em>, the Michaelson model) and the rate of mutation using sets of DNMs. We then used this relationship to predict the expected number of mutations at a site and then simulated data based on these expectations with and without additional variation in the mutation rate (details in the section <em>Materials and Methods</em>). We performed the analysis for three sets of DNMs: (i) the 652 DNMs reported by <a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> and used to build the model upon which the MI values are based (referred to as the Michaelson data), (ii) 1380 DNMs reported by various other studies (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Conrad <em>et al.</em> 2011</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Iossifov <em>et al.</em> 2012</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Neale <em>et al.</em> 2012</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">O’Roak <em>et al.</em> 2011</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Sanders <em>et al.</em> 2012</a>) (Other data), and (iii) 4933 DNMs reported by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kong <em>et al.</em> (2012)</a> (Kong data; note that only DNMs with an MI value were included).</p><p id="p-12">As previously shown by <a id="xref-ref-16-11" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a>, the correlation between the log of the number of DNMs per site and the MI value is very strong for the Michaelson data (r = 0.98, <em>P</em> &lt; 0.001; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>); this finding is perhaps not surprising, given that these were the data used to construct the Michaelson model and the model is relatively parameter rich. However, as <a id="xref-ref-16-12" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> showed, their model also fits the data from other studies well (r = 0.97, <em>P</em> &lt; 0.001; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), although there is a clear nonlinearity in the relationship (a quadratic term in a nonlinear regression is significant <em>P</em> = 0.010). However, the fit of the Michaelson model to the Kong data, which <a id="xref-ref-16-13" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> did not study, is relatively poor (r = 0.94, <em>P</em> &lt; 0.001; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). The problem would seem to lie with the Kong data, since the model fits the other two datasets well. The slope of the regression line from the Kong data [0.0047 (0.0006)] is significantly less than that observed for the <a id="xref-ref-16-14" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> [0.010 (0.0007)] and Other datasets [0.0084 (0.0007)], suggesting that there has been systematic underreporting of DNMs from the more mutable areas of the genome in the <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Kong <em>et al.</em> (2012)</a> dataset (or alternatively, that there are large numbers of false positives in the less mutable parts).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/9/1667/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The log of the number of de novo mutations (DNMs) per site vs. the mutability index (MI). The mutability index for the (A) Michaelson, (B) Other, and (C) Kong datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1722304771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The log of the number of &lt;em&gt;de novo&lt;/em&gt; mutations (DNMs) per site &lt;em&gt;vs.&lt;/em&gt; the mutability index (MI). The mutability index for the (A) Michaelson, (B) Other, and (C) Kong datasets.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/9/1667/F1.medium.gif" width="180" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/9/1667/F1.medium.gif" width="180" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/9/1667/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/9/1667/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">The log of the number of <em>de novo</em> mutations (DNMs) per site <em>vs.</em> the mutability index (MI). The mutability index for the (A) Michaelson, (B) Other, and (C) Kong datasets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">If we assume that the Michaelson model explains all the variation in the mutation rate, we find that simulated datasets have similar levels of correlation between the log of the number of DNMs per site and MI to that observed in the real data for the Michaelson and other datasets, <em>i.e.</em>, any variance not explained by the Michaelson model can be attributed to sampling error due to the fact that there are limited number of DNMs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). All the simulated correlations are stronger than the observed correlation in the Kong data, but this is probably because the Michaelson model fits these data poorly.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464879/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464879/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The proportion of simulated datasets with a greater correlation between the log of the number of DNMs per site and MI than observed in the actual data</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">However, despite the good fit between model and data for two of the datasets, we find that there could be very substantial levels of unexplained variance and the correlations would remain almost unaffected (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Only when the variance associated with the unexplained variance approaches 10<sup>5</sup> do we see the correlations being affected and approaching the values seen in the real data. This level of variance dwarfs that explained by the Michaelson model; the coefficient of variation in the mutation rate explained by the Michaelson model is 1.10, the coefficient of variation for the unexplained variation is 300 if the variance is 10<sup>5</sup>. This analysis therefore shows that there could be a substantial amount of unexplained variance that would never be detected, assessing model fit as <a id="xref-ref-16-16" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> have done. However, it does not show that this unexplained variance exists.</p><p id="p-17">Assessing model fit is not easy with these datasets; there are very few DNMs spread across millions of sites. We therefore sought to test one component of mutation rate variation that is both substantial and likely to be difficult to predict, the so-called cryptic variation in the mutation rate (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Hodgkinson <em>et al.</em> 2009</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Johnson and Hellmann 2011</a>). This is variation at the single-nucleotide level that is independent of adjacent nucleotides.</p><p id="p-18">To investigate whether the model of <a id="xref-ref-16-17" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a> captures cryptic variation in the mutation rate we proceeded as follows. Sites with SNPs in both humans and chimpanzees are inferred to have greater mutation rates than sites without coincident SNPs. Using some simple theory, we can estimate how much more mutable these sites need to be on average to yield the observed excess of coincident SNPs, and we can then compare this with the difference in mutation rate predicted by the Michaelson model for sites with and without coincident SNPs. If the Michaelson model captures cryptic variation in the mutation rate then it should correctly predict the level of variation needed to explain the observed excess of coincident SNPs.</p><p id="p-19">Although coincident SNPs and DNMs are measuring variation in the mutation rate over very different time scales, it is expected that any source of variation identifiable through coincident SNPs should also be present in DNMs. In fact if variation in the mutation rate evolves through time, we would expect the variation associated with coincident SNPs to be less than that present in DNMs.</p><p id="p-20"><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Leffler <em>et al.</em> (2013)</a> have shown, using a carefully curated dataset of human and chimpanzee SNPs, that there is a 16% excess of coincident SNPs at CpG sites (95% confidence intervals of 14% and 17%) and a 83% (80%, 86%) excess at non-CpG sites between human and chimpanzee, when correcting for the nucleotides at the adjacent sites. Assuming that rates are gamma distributed across sites, we estimate, using the theory set out by <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Hodgkinson <em>et al.</em> (2009)</a>, that sites with coincident SNPs are 1.40-fold (95% confidence intervals of 1.35, 1.43) and 2.71-fold (2.65, 2.77) more mutable on average than the genomic average for CpG and non-CpG sites respectively. How do these values compare to those under the Michaelson model? Under the Michaelson model, we find that sites with coincident SNPs have significantly greater MI values at both CpG (mean MI for coincident sites = 91.6, noncoincident sites = 81.4; <em>P</em> &lt; 0.001) and non-CpG sites (coincident sites = −7.77, noncoincident sites = −16.0, <em>P</em> &lt; 0.001). However, the differences in MI are small and equate to minor differences in the mutation rate predicted using the regression model from the data of <a id="xref-ref-16-18" class="xref-bibr" href="#ref-16">Michaelson <em>et al.</em> (2012)</a>; coincident SNPs are predicted to be 1.21-fold (1.21, 1.21) more mutable at CpG and 1.17-fold (1.17, 1.17) more mutable at non-CpG sites. Thus, our analysis suggests that the Michaelson model may capture a substantial fraction, although by no means all, of the variation at CpG sites; the level of variation required to explain the excess of coincident SNPs at CpG sites is such that we would expect sites with coincident SNPs to be 1.40-fold (1.35, 1.43) more mutable than noncoincident sites and the Michaelson model predicts them to 1.21-fold (1.21, 1.21) more mutable; these estimates are significantly different (<em>P</em> &lt; 0.001). However, the Michaelson model seems to fail to capture most of the variation at non-CpG sites; sites with coincident SNPs are expected to be 2.71-fold (2.65, 2.77) more mutable than noncoincident sites, but the Michaelson model predicts them to be only 1.17-fold (1.17, 1.17) more mutable. These estimates are significantly different (<em>P</em> &lt; 0.001).</p><p id="p-21">Although the Michaelson model appears to explain a substantial fraction of the variation at CpG sites this might be deceptive because the level of cryptic variation may have been underestimated by analyzing coincident SNPs. Cryptic variation is, by its definition, variation that does not depend upon the context of the immediately adjacent nucleotides. However, it is possible, in fact likely, that cryptic variation depends upon context in some manner, may be through a dispersed context (for example, if the second nucleotide upstream is an A and fifth nucleotide downstream is a C, etc). If this is the case then as the context evolves so the mutation rate at the focal site will change. This will lead to a reduction in the correlation in the mutation rate between sites in humans and chimpanzees and hence to a lower level of variation inferred from coincident SNPs. However, one might argue that the level of variation is overestimated by a consideration of coincident SNPs, because SNPs can be subject to natural selection, which will tend to increase the number of coincident SNPs by excluding SNPs from certain parts of the genome. This seems unlikely because less than 10% of the human genome is inferred to be subject to selection (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Eory <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lunter <em>et al.</em> 2006</a>).</p><p id="p-22">Currently we do not have a robust estimate of how much of the total variance in the mutation rate is contributed by cryptic variation. Initial rough estimates suggest that there is approximately as much variation associated with cryptic variation as there is with simple context (<em>i.e.</em>, CpG type effects) (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Hodgkinson <em>et al.</em> 2009</a>). Because variation at the single-nucleotide level appears to contribute the most to variation in the mutation rate (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Hodgkinson and Eyre-Walker 2011</a>) it would seem that the Michaelson model is missing a major component of the variation, particularly at non-CpG sites.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-23">The authors are grateful to Jake Michaelson, Jonathan Sebat, Chris Ponting, and Peter Keightley for helpful discussion.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-24"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-25"><span class="received-label">Received </span>March 25, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-06-26"><span class="accepted-label">Accepted </span>June 26, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Eyre-Walker and Eyre-Walker</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.1" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.2" data-doi="10.1016/j.ajhg.2010.07.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casals</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamdan</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Direct measure of the de novo mutation rate in autism and schizophrenia cohorts.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAwadalla%26rft.auinit1%253DP.%26rft.volume%253D87%26rft.issue%253D3%26rft.spage%253D316%26rft.epage%253D324%26rft.atitle%253DDirect%2Bmeasure%2Bof%2Bthe%2Bde%2Bnovo%2Bmutation%2Brate%2Bin%2Bautism%2Band%2Bschizophrenia%2Bcohorts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.07.019%26rft_id%253Dinfo%253Apmid%252F20797689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.07.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20797689&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281923500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.3" data-doi="10.1109/26.477480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abu-Daya</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLane</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Estimating the distribution of a sum of independent lognormal random variables.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Commun.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">2869</span>–<span class="cit-lpage">2873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BCommun.%26rft.volume%253D43%26rft.spage%253D2869%26rft_id%253Dinfo%253Adoi%252F10.1109%252F26.477480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/26.477480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.4" data-doi="10.1038/nature04072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Chimpanzee-Sequencing-and-Analysis-Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Initial sequence of the chimpanzee genome and comparison with the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D437%26rft.issue%253D7055%26rft.spage%253D69%26rft.epage%253D87%26rft.atitle%253DInitial%2Bsequence%2Bof%2Bthe%2Bchimpanzee%2Bgenome%2Band%2Bcomparison%2Bwith%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04072%26rft_id%253Dinfo%253Apmid%252F16136131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16136131&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231560400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.5" data-doi="10.1038/ng.862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keebler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Depristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in genome-wide mutation rates within and between human families.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">712</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D712%26rft.epage%253D714%26rft.atitle%253DVariation%2Bin%2Bgenome-wide%2Bmutation%2Brates%2Bwithin%2Band%2Bbetween%2Bhuman%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.862%26rft_id%253Dinfo%253Apmid%252F21666693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21666693&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.6" data-doi="10.1093/molbev/msp219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eory</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Distributions of selectively constrained sites and deleterious mutation rates in the hominid and murid genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp219%26rft_id%253Dinfo%253Apmid%252F19759235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzEvMTc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.7" data-doi="10.1109/TCOM.1960.1097606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fenton</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">The sum of lognormal probability distributions in scatter transmission systems.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Commun. Syst.</abbr> <span class="cit-vol">CS-8</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BCommun.%2BSyst.%26rft.volume%253D8%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCOM.1960.1097606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCOM.1960.1097606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.8" data-doi="10.1038/nrg3098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkinson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in the mutation rate across mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHodgkinson%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D756%26rft.epage%253D766%26rft.atitle%253DVariation%2Bin%2Bthe%2Bmutation%2Brate%2Bacross%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3098%26rft_id%253Dinfo%253Apmid%252F21969038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21969038&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.9" data-doi="10.1371/journal.pbio.1000027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkinson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ladoukakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cryptic variation in the human mutation rate.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D7%26rft.spage%253De27%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.10" data-doi="10.1073/pnas.0404142101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bayesian Markov chain Monte Carlo sequence analysis reveals varying neutral substitution patterns in mammalian evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">13994</span>–<span class="cit-lpage">14001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404142101%26rft_id%253Dinfo%253Apmid%252F15292512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM5LzEzOTk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.11" data-doi="10.1016/j.neuron.2012.04.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iossifov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronemus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakker</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">De novo gene disruptions in children on the autistic spectrum.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D74%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2012.04.009%26rft_id%253Dinfo%253Apmid%252F22542183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2012.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22542183&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303361800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.12" data-doi="10.1093/gbe/evr044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">P. L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation rate distribution inferred from coincident SNPs and coincident substitutions.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">842</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr044%26rft_id%253Dinfo%253Apmid%252F21572094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC84NDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzkvMTY2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.13" data-doi="10.1038/nature11396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frigge</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besenbacher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulem</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rate of de novo mutations and the importance of father’s age to disease risk.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D488%26rft.issue%253D7412%26rft.spage%253D471%26rft.epage%253D475%26rft.atitle%253DRate%2Bof%2Bde%2Bnovo%2Bmutations%2Band%2Bthe%2Bimportance%2Bof%2Bfather%2527s%2Bage%2Bto%2Bdisease%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11396%26rft_id%253Dinfo%253Apmid%252F22914163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22914163&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307761600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.14" data-doi="10.1126/science.1234070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segurel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple instances of ancient balancing selection shared between humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">1578</span>–<span class="cit-lpage">1582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1234070%26rft_id%253Dinfo%253Apmid%252F23413192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNy8xNTc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC85LzE2NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.15" data-doi="10.1371/journal.pcbi.0020005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide identification of human functional DNA using a neutral indel model.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DLunter%26rft.auinit1%253DG.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253De5%26rft.epage%253De5%26rft.atitle%253DGenome-wide%2Bidentification%2Bof%2Bhuman%2Bfunctional%2BDNA%2Busing%2Ba%2Bneutral%2Bindel%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020005%26rft_id%253Dinfo%253Apmid%252F16410828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16410828&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.16" data-doi="10.1016/j.cell.2012.11.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaelson</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gujral</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malhotra</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole-genome sequencing in autism identifies hot spots for de novo germline mutation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D151%26rft.spage%253D1431%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.11.019%26rft_id%253Dinfo%253Apmid%252F23260136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23260136&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312890300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.17" data-doi="10.1038/nature11011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma’ayan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Patterns and rates of exonic de novo mutations in autism spectrum disorders.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNeale%26rft.auinit1%253DB.%2BM.%26rft.volume%253D485%26rft.issue%253D7397%26rft.spage%253D242%26rft.epage%253D245%26rft.atitle%253DPatterns%2Band%2Brates%2Bof%2Bexonic%2Bde%2Bnovo%2Bmutations%2Bin%2Bautism%2Bspectrum%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11011%26rft_id%253Dinfo%253Apmid%252F22495311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495311&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303799800042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.18" data-doi="10.1038/ng.835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deriziotis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vives</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exome sequencing in sporadic autism spectrum disorders identifies severe de novo mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DO%2527Roak%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D585%26rft.epage%253D589%26rft.atitle%253DExome%2Bsequencing%2Bin%2Bsporadic%2Bautism%2Bspectrum%2Bdisorders%2Bidentifies%2Bsevere%2Bde%2Bnovo%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.835%26rft_id%253Dinfo%253Apmid%252F21572417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572417&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.9.1667.19" data-doi="10.1038/nature10945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murtha</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murdoch</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raubeson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">De novo mutations revealed by whole-exome sequencing are strongly associated with autism.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSanders%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D485%26rft.issue%253D7397%26rft.spage%253D237%26rft.epage%253D241%26rft.atitle%253DDe%2Bnovo%2Bmutations%2Brevealed%2Bby%2Bwhole-exome%2Bsequencing%2Bare%2Bstrongly%2Bassociated%2Bwith%2Bautism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10945%26rft_id%253Dinfo%253Apmid%252F22495306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495306&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F9%2F1667.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303799800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/9/1667.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/9/1657" title="Parallel Genome-Wide Fixation of Ancestral Alleles in Partially Outcrossing Experimental Populations of Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/9/1671" title="Genetic Basis for Saccharomyces cerevisiae Biofilm in Liquid Medium" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/9"><span>Volume 4 Issue 9, September 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/9.cover-source.jpg?itok=ECLwglsV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/9/1667" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vN2gbwj_OMqEq1tnBdhcLnd4YZb8lUMqjYGZqgE02ko" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/9/1667" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/9/1667" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/9/1667" class="active" data-icon-position="" data-hide-link-title="0">How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459674" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pt805X4ofxt8uljPzvUFfj3p8mr1VKd1CVpqZxpG61I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1667.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/9/1667" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Lt0Zc89k6QapwvGBCUZV29TceRKMlqqBs5y7Zs-xsWI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/9/1667" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/9/1667.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459674" id="node459674--2" data-pisa="ggg;4/9/1667" data-pisa-master="ggg;g3.114.012849" data-apath="/ggg/4/9/1667.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5527-8729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adam Eyre-Walker</span> and <span class="highwire-citation-author" data-delta="1">Ying Chen Eyre-Walker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1667-1670;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012849 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459674--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adam Eyre-Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BEyre-Walker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eyre-Walker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BEyre-Walker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5527-8729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adam Eyre-Walker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:a.c.eyre-walker@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.c.eyre-walker@sussex.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ying Chen Eyre-Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BChen%2BEyre-Walker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eyre-Walker%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BChen%2BEyre-Walker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/9/1667" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/9/1667&amp;title=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/9/1667&amp;title=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/9/1667&amp;title=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/9/1667&amp;text=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/9/1667&amp;title=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/9/1667&amp;t=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/9/1667&amp;title=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/9/1667&amp;title=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459674/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459674" id="node459674--3" data-pisa="ggg;4/9/1667" data-pisa-master="ggg;g3.114.012849" data-apath="/ggg/4/9/1667.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Much of the Variation in the Mutation Rate Along the Human Genome Can Be Explained?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5527-8729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adam Eyre-Walker</span> and <span class="highwire-citation-author" data-delta="1">Ying Chen Eyre-Walker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1667-1670;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012849 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459674--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adam Eyre-Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BEyre-Walker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eyre-Walker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BEyre-Walker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5527-8729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adam Eyre-Walker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:a.c.eyre-walker@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.c.eyre-walker@sussex.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ying Chen Eyre-Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Life Sciences, University of Sussex, Brighton, BN1 9QG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BChen%2BEyre-Walker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eyre-Walker%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BChen%2BEyre-Walker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1667&amp;rft.epage=1670&amp;rft.atitle=How%20Much%20of%20the%20Variation%20in%20the%20Mutation%20Rate%20Along%20the%20Human%20Genome%20Can%20Be%20Explained%3F&amp;rft.volume=4&amp;rft.issue=9&amp;rft.date=2014-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Eyre-Walker%2C+Adam&amp;rft.au=Eyre-Walker%2C+Ying+Chen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459674/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459674/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459674/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459674/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459674/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459674/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459674/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459674/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459674/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459674/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459674/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459674/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/9/1667" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/9/1667.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/9/1667.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/4/9/1667.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
