<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/7/1205" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/7/1205.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/7/1205.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/7/1205.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/7/1205/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/7/1205.atom" />
<meta name="HW.pisa" content="ggg;4/7/1205" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" />
<meta name="DC.Identifier" content="10.1534/g3.114.011783" />
<meta name="DC.Date" content="2014-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Duveau et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic variants identified by mapping are biased toward large phenotypic effects because of methodologic challenges for detecting genetic variants with small phenotypic effects. Recently, bulk segregant analysis combined with next-generation sequencing (BSA-seq) was shown to be a powerful and cost-effective way to map small effect variants in natural populations. Here, we examine the power of BSA-seq for efficiently mapping small effect mutations isolated from a mutagenesis screen. Specifically, we determined the impact of segregant population size, intensity of phenotypic selection to collect segregants, number of mitotic generations between meiosis and sequencing, and average sequencing depth on power for mapping mutations with a range of effects on the phenotypic mean and standard deviation as well as relative fitness. We then used BSA-seq to map the mutations responsible for three ethyl methanesulfonate−induced mutant phenotypes in Saccharomyces cerevisiae . These mutants display small quantitative variation in the mean expression of a fluorescent reporter gene (−3%, +7%, and +10%). Using a genetic background with increased meiosis rate, a reliable mating type marker, and fluorescence-activated cell sorting to efficiently score large segregating populations and isolate cells with extreme phenotypes, we successfully mapped and functionally confirmed a single point mutation responsible for the mutant phenotype in all three cases. Our simulations and experimental data show that the effects of a causative site not only on the mean phenotype, but also on its standard deviation and relative fitness should be considered when mapping genetic variants in microorganisms such as yeast that require population growth steps for BSA-seq." />
<meta name="DC.Contributor" content="Fabien Duveau" />
<meta name="DC.Contributor" content="Brian P. H. Metzger" />
<meta name="DC.Contributor" content="Jonathan D. Gruber" />
<meta name="DC.Contributor" content="Katya Mack" />
<meta name="DC.Contributor" content="Natasha Sood" />
<meta name="DC.Contributor" content="Tiffany E. Brooks" />
<meta name="DC.Contributor" content="Patricia J. Wittkopp" />
<meta name="article:published_time" content="2014-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic variants identified by mapping are biased toward large phenotypic effects because of methodologic challenges for detecting genetic variants with small phenotypic effects. Recently, bulk segregant analysis combined with next-generation sequencing (BSA-seq) was shown to be a powerful and cost-effective way to map small effect variants in natural populations. Here, we examine the power of BSA-seq for efficiently mapping small effect mutations isolated from a mutagenesis screen. Specifically, we determined the impact of segregant population size, intensity of phenotypic selection to collect segregants, number of mitotic generations between meiosis and sequencing, and average sequencing depth on power for mapping mutations with a range of effects on the phenotypic mean and standard deviation as well as relative fitness. We then used BSA-seq to map the mutations responsible for three ethyl methanesulfonate−induced mutant phenotypes in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. These mutants display small quantitative variation in the mean expression of a fluorescent reporter gene (−3%, +7%, and +10%). Using a genetic background with increased meiosis rate, a reliable mating type marker, and fluorescence-activated cell sorting to efficiently score large segregating populations and isolate cells with extreme phenotypes, we successfully mapped and functionally confirmed a single point mutation responsible for the mutant phenotype in all three cases. Our simulations and experimental data show that the effects of a causative site not only on the mean phenotype, but also on its standard deviation and relative fitness should be considered when mapping genetic variants in microorganisms such as yeast that require population growth steps for BSA-seq.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/07/01" />
<meta name="citation_mjid" content="ggg;4/7/1205" />
<meta name="citation_id" content="4/7/1205" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/7/1205" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/7/1205.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/7/1205.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/7/1205.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011783" />
<meta name="citation_pmid" content="24789747" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1205" />
<meta name="citation_lastpage" content="1216" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Fabien Duveau" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author_email" content="fduveau@umich.edu" />
<meta name="citation_author_email" content="wittkopp@umich.edu" />
<meta name="citation_author" content="Brian P. H. Metzger" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author" content="Jonathan D. Gruber" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author" content="Katya Mack" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author" content="Natasha Sood" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author" content="Tiffany E. Brooks" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author" content="Patricia J. Wittkopp" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, &amp; Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109-1048" />
<meta name="citation_author_institution" content="Center for Computational Medicine and Bioinformatics, University of Michigan Medical School, Ann Arbor, Michigan 48109-2218" />
<meta name="citation_author_email" content="fduveau@umich.edu" />
<meta name="citation_author_email" content="wittkopp@umich.edu" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. W. Albert;citation_author=S. Treusch;citation_author=A. H. Shockley;citation_author=J. S. Bloom;citation_author=L. Kruglyak;citation_title=Genetics of single-cell protein abundance variation in large yeast populations.;citation_pages=494-497;citation_volume=506;citation_year=2014;citation_pmid=24402228;citation_doi=10.1038/nature12904" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. Bastide;citation_author=A. Betancourt;citation_author=V. Nolte;citation_author=R. Tobler;citation_author=P. Stöbe;citation_title=A genome-wide, fine-scale map of natural pigmentation variation in Drosophila melanogaster.;citation_pages=e1003534;citation_volume=9;citation_year=2013;citation_pmid=23754958;citation_doi=10.1371/journal.pgen.1003534" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. R. Birkeland;citation_author=N. Jin;citation_author=A. C. Ozdemir;citation_author=R. H. Lyons;citation_author=L. S. Weisman;citation_title=Discovery of mutations in Saccharomyces cerevisiae by pooled linkage analysis and whole-genome sequencing.;citation_volume=186;citation_year=2010;citation_pmid=20923977;citation_doi=10.1534/genetics.110.123232" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. J. Brauer;citation_author=C. M. Christianson;citation_author=D. A. Pai;citation_author=M. J. Dunham;citation_title=Mapping novel traits by array-assisted bulk segregant analysis in Saccharomyces cerevisiae.;citation_volume=173;citation_year=2006;citation_pmid=16624899;citation_doi=10.1534/genetics.106.057927" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. K. Breslow;citation_author=D. M. Cameron;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=J. Stewart-Ornstein;citation_title=A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.;citation_pages=711-718;citation_volume=5;citation_year=2008;citation_issue=8;citation_pmid=18622397;citation_doi=10.1038/nmeth.1234" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=B. L. Chin;citation_author=M. a. Frizzell;citation_author=W. E. Timberlake;citation_author=G. R. Fink;citation_title=FASTER MT: Isolation of Pure Populations of a and {alpha} Ascospores from Saccharomyces cerevisiae;citation_pages=449-452;citation_volume=2;citation_year=2012;citation_issue=4;citation_pmid=22540036;citation_doi=10.1534/g3.111.001826" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. M. Deutschbauer;citation_author=R. W. Davis;citation_title=Quantitative trait loci mapped to single-nucleotide resolution in yeast.;citation_pages=1333-1340;citation_volume=37;citation_year=2005;citation_issue=12;citation_pmid=16273108;citation_doi=10.1038/ng1674" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Deutschbauer;citation_author=D. F. Jaramillo;citation_author=M. Proctor;citation_author=J. Kumm;citation_author=M. E. Hillenmeyer;citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.;citation_volume=169;citation_year=2005;citation_pmid=15716499;citation_doi=10.1534/genetics.104.036871" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. D. Edwards;citation_author=D. K. Gifford;citation_title=High-resolution genetic mapping with pooled sequencing.;citation_pages=S8;citation_volume=13;citation_year=2012;citation_pmid=23282012;citation_doi=10.1186/1471-2105-13-S14-S8" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=I. M. Ehrenreich;citation_author=N. Torabi;citation_author=Y. Jia;citation_author=J. Kent;citation_author=S. Martis;citation_title=Dissection of genetically complex traits with extremely large pools of yeast segregants.;citation_pages=1039-1042;citation_volume=464;citation_year=2010;citation_issue=7291;citation_pmid=20393561;citation_doi=10.1038/nature08923" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=I. M. Ehrenreich;citation_author=J. Bloom;citation_author=N. Torabi;citation_author=X. Wang;citation_author=Y. Jia;citation_title=Genetic architecture of highly complex chemical resistance traits across four yeast strains.;citation_pages=e1002570-e1002570;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22438822;citation_doi=10.1371/journal.pgen.1002570" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Eyre-Walker;citation_author=P. D. Keightley;citation_title=The distribution of fitness effects of new mutations.;citation_pages=610-618;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17637733;citation_doi=10.1038/nrg2146" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=I. Fellows;citation_title=Deducer: a data analysis GUI for R.;citation_pages=1-15;citation_volume=49;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Gerke;citation_author=K. Lorenz;citation_author=B. Cohen;citation_title=Genetic interactions between transcription factors cause natural variation in yeast.;citation_pages=2007-2010;citation_volume=323;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. C. Gerstein;citation_author=S. P. Otto;citation_title=Cryptic fitness advantage: diploids invade haploid populations despite lacking any apparent advantage as measured by standard fitness assays.;citation_pages=e26599-e26599;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174734;citation_doi=10.1371/journal.pone.0026599" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Granek;citation_author=D. Murray;citation_author=O. Kayikçi;citation_author=P. M. Magwene;citation_title=The genetic architecture of biofilm formation in a clinical isolate of Saccharomyces cerevisiae.;citation_pages=587-600;citation_volume=193;citation_year=2012;citation_pmid=23172850" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. D. Gruber;citation_author=K. Vogel;citation_author=G. Kalay;citation_author=P. J. Wittkopp;citation_title=Contrasting properties of gene-specific regulatory, coding, and copy number mutations in Saccharomyces cerevisiae: frequency, effects, and dominance.;citation_pages=e1002497-e1002497;citation_volume=8;citation_year=2012;citation_issue=2;citation_pmid=22346762;citation_doi=10.1371/journal.pgen.1002497" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. E. Hanlon;citation_author=J. M. Rizzo;citation_author=D. C. Tatomer;citation_author=J. D. Lieb;citation_author=M. J. Buck;citation_title=The stress response factors Yap6, Cin5, Phd1, and Skn7 direct targeting of the conserved co-repressor Tup1-Ssn6 in S. cerevisiae.;citation_pages=e19060-e19060;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21552514;citation_doi=10.1371/journal.pone.0019060" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. T. Harbison;citation_author=D. B. Gordon;citation_author=T. I. Lee;citation_author=N. J. Rinaldi;citation_author=K. D. Macisaac;citation_title=Transcriptional regulatory code of a eukaryotic genome.;citation_pages=99-104;citation_volume=431;citation_year=2004;citation_issue=7004;citation_pmid=15343339;citation_doi=10.1038/nature02800" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. C. Koboldt;citation_author=K. Chen;citation_author=T. Wylie;citation_author=D. E. Larson;citation_author=M. D. McLellan;citation_title=VarScan: variant detection in massively parallel sequencing of individual and pooled samples.;citation_volume=25;citation_year=2009;citation_pmid=19542151;citation_doi=10.1093/bioinformatics/btp373" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Kofler;citation_author=R. V. Pandey;citation_author=C. Schlötterer;citation_title=PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).;citation_volume=27;citation_year=2011;citation_pmid=22025480;citation_doi=10.1093/bioinformatics/btr589" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=P. Kumar;citation_author=S. Henikoff;citation_author=P. C. Ng;citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.;citation_pages=1073-1081;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19561590;citation_doi=10.1038/nprot.2009.86" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Larochelle;citation_author=S. Drouin;citation_author=F. Robert;citation_author=B. Turcotte;citation_title=Oxidative Stress-Activated Zinc Cluster Protein Stb5 Has Dual Activator/Repressor Functions Required for Pentose Phosphate Pathway Regulation and NADPH Production;citation_pages=6690-6701;citation_volume=26;citation_year=2006;citation_issue=17;citation_pmid=16914749;citation_doi=10.1128/MCB.02450-05" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Liti;citation_author=E. J. Louis;citation_title=Advances in quantitative trait analysis in yeast.;citation_pages=e1002912-e1002912;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22916041;citation_doi=10.1371/journal.pgen.1002912" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=P. M. Magwene;citation_author=J. H. Willis;citation_author=J. K. Kelly;citation_title=The statistics of bulk segregant analysis using next generation sequencing.;citation_pages=e1002255-e1002255;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072954;citation_doi=10.1371/journal.pcbi.1002255" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=T. A. Mennella;citation_author=L. G. Klinkenberg;citation_author=R. S. Zitomer;citation_title=Recruitment of Tup1-Ssn6 by Yeast Hypoxic Genes and Chromatin-Independent Exclusion of TATA Binding Protein;citation_pages=1288-1303;citation_volume=2;citation_year=2003;citation_issue=6;citation_pmid=14665463;citation_doi=10.1128/EC.2.6.1288-1303.2003" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Michelmore;citation_author=I. Paran;citation_author=R. V. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=X. Pan;citation_author=D. S. Yuan;citation_author=D. Xiang;citation_author=X. Wang;citation_author=S. Sookhai-Mahadeo;citation_title=A robust toolkit for functional profiling of the yeast genome.;citation_pages=487-496;citation_volume=16;citation_year=2004;citation_issue=3;citation_pmid=15525520;citation_doi=10.1016/j.molcel.2004.09.035" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Parts;citation_author=F. A. Cubillos;citation_author=J. Warringer;citation_author=K. Jain;citation_author=F. Salinas;citation_title=Revealing the genetic structure of a trait by sequencing a population under selection.;citation_volume=21;citation_year=2011;citation_pmid=21422276;citation_doi=10.1101/gr.116731.110" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=K. R. Pomraning;citation_author=K. M. Smith;citation_author=M. Freitag;citation_title=Bulk segregant analysis followed by high-throughput sequencing reveals the Neurospora cell cycle gene, ndc-1, to be allelic with the gene for ornithine decarboxylase, spe-1.;citation_volume=10;citation_year=2011;citation_pmid=21515825;citation_doi=10.1128/EC.00016-11" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Rohland;citation_author=D. Reich;citation_title=Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture.;citation_volume=22;citation_year=2012;citation_pmid=22267522;citation_doi=10.1101/gr.128124.111" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=W. Song;citation_author=I. Treich;citation_author=N. Qian;citation_author=S. Kuchin;citation_author=M. Carlson;citation_title=SSN genes that affect transcriptional repression in Saccharomyces cerevisiae encode SIN4, ROX3, and SRB proteins associated with RNA polymerase II;citation_pages=115-120;citation_volume=16;citation_year=1996;citation_issue=1;citation_pmid=8524287;citation_doi=10.1128/MCB.16.1.115" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=F. Storici;citation_author=M. A. Resnick;citation_title=The delitto perfetto approach to in vivo site-directed mutagenesis and chromosome rearrangements with synthetic oligonucleotides in yeast.;citation_pages=329-345;citation_volume=409;citation_year=2006;citation_pmid=16793410;citation_doi=10.1016/j.ymeth.2006.06.011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Swinnen;citation_author=K. Schaerlaekens;citation_author=T. Pais;citation_author=J. Claesen;citation_author=G. Hubmann;citation_title=Identification of novel causative genes determining the complex trait of high ethanol tolerance in yeast using pooled-segregant whole-genome sequence analysis.;citation_volume=22;citation_year=2012;citation_pmid=22399573;citation_doi=10.1101/gr.131698.111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Van Leeuwen;citation_author=P. Demaeght;citation_author=E. J. Osborne;citation_author=W. Dermauw;citation_author=S. Gohlke;citation_title=Population bulk segregant mapping uncovers resistance mutations and the mode of action of a chitin synthesis inhibitor in arthropods.;citation_volume=109;citation_year=2012;citation_pmid=22393009;citation_doi=10.1073/pnas.1200068109" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. W. Wenger;citation_author=K. Schwartz;citation_author=G. Sherlock;citation_title=Bulk segregant analysis by high-throughput sequencing reveals a novel xylose utilization gene from Saccharomyces cerevisiae.;citation_pages=e1000942-e1000942;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20485559;citation_doi=10.1371/journal.pgen.1000942" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. R. Wicks;citation_author=R. T. Yeh;citation_author=W. R. Gish;citation_author=R. H. Waterston;citation_author=R. H. Plasterk;citation_title=Rapid gene mapping in Caenorhabditis elegans using a high density polymorphism map.;citation_pages=160-164;citation_volume=28;citation_year=2001;citation_issue=2;citation_pmid=11381264;citation_doi=10.1038/88878" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Wilkening;citation_author=G. Lin;citation_author=E. S. Fritsch;citation_author=M. M. Tekkedil;citation_author=S. Anders;citation_title=An evaluation of high-throughput approaches to QTL mapping in Saccharomyces cerevisiae.;citation_volume=196;citation_year=2014;citation_pmid=24374355;citation_doi=10.1534/genetics.113.160291" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. M. Wloch;citation_author=K. Szafraniec;citation_author=R. H. Borts;citation_author=R. Korona;citation_title=Direct Estimate of the Mutation Rate and the Distribution of Fitness Effects in the Yeast Saccharomyces cerevisiae;citation_pages=441-452;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606524" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Xia;citation_author=S. Won;citation_author=X. Du;citation_author=P. Lin;citation_author=C. Ross;citation_title=Bulk segregation mapping of mutations in closely related strains of mice.;citation_volume=186;citation_year=2010;citation_pmid=20923982;citation_doi=10.1534/genetics.110.121160" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Yang;citation_author=M. R. Foulquié-Moreno;citation_author=L. Clement;citation_author=E. Erdei;citation_author=A. Tanghe;citation_title=QTL analysis of high thermotolerance with superior and downgraded parental yeast strains reveals new minor QTLs and converges on novel causative alleles involved in RNA processing.;citation_pages=e1003693;citation_volume=9;citation_year=2013;citation_pmid=23966873;citation_doi=10.1371/journal.pgen.1003693" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="twitter:description" content="Genetic variants identified by mapping are biased toward large phenotypic effects because of methodologic challenges for detecting genetic variants with small phenotypic effects. Recently, bulk segregant analysis combined with next-generation sequencing (BSA-seq) was shown to be a powerful and cost-effective way to map small effect variants in natural populations. Here, we examine the power of BSA-seq for efficiently mapping small effect mutations isolated from a mutagenesis screen. Specifically, we determined the impact of segregant population size, intensity of phenotypic selection to collect segregants, number of mitotic generations between meiosis and sequencing, and average sequencing depth on power for mapping mutations with a range of effects on the phenotypic mean and standard deviation as well as relative fitness. We then used BSA-seq to map the mutations responsible for three ethyl methanesulfonate−induced mutant phenotypes in Saccharomyces cerevisiae . These mutants display small quantitative variation in the mean expression of a fluorescent reporter gene (−3%, +7%, and +10%). Using a genetic background with increased meiosis rate, a reliable mating type marker, and fluorescence-activated cell sorting to efficiently score large segregating populations and isolate cells with extreme phenotypes, we successfully mapped and functionally confirmed a single point mutation responsible for the mutant phenotype in all three cases. Our simulations and experimental data show that the effects of a causative site not only on the mean phenotype, but also on its standard deviation and relative fitness should be considered when mapping genetic variants in microorganisms such as yeast that require population growth steps for BSA-seq." />
<meta name="og-title" property="og:title" content="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/7/1205" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic variants identified by mapping are biased toward large phenotypic effects because of methodologic challenges for detecting genetic variants with small phenotypic effects. Recently, bulk segregant analysis combined with next-generation sequencing (BSA-seq) was shown to be a powerful and cost-effective way to map small effect variants in natural populations. Here, we examine the power of BSA-seq for efficiently mapping small effect mutations isolated from a mutagenesis screen. Specifically, we determined the impact of segregant population size, intensity of phenotypic selection to collect segregants, number of mitotic generations between meiosis and sequencing, and average sequencing depth on power for mapping mutations with a range of effects on the phenotypic mean and standard deviation as well as relative fitness. We then used BSA-seq to map the mutations responsible for three ethyl methanesulfonate−induced mutant phenotypes in Saccharomyces cerevisiae . These mutants display small quantitative variation in the mean expression of a fluorescent reporter gene (−3%, +7%, and +10%). Using a genetic background with increased meiosis rate, a reliable mating type marker, and fluorescence-activated cell sorting to efficiently score large segregating populations and isolate cells with extreme phenotypes, we successfully mapped and functionally confirmed a single point mutation responsible for the mutant phenotype in all three cases. Our simulations and experimental data show that the effects of a causative site not only on the mean phenotype, but also on its standard deviation and relative fitness should be considered when mapping genetic variants in microorganisms such as yeast that require population growth steps for BSA-seq." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
    <title>Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"syAVObidtJO0lmE8PIHObJmDsFY0ALT3uwF2JKBVjok","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459929","apath":"\/ggg\/4\/7\/1205.atom","pisa":"ggg;4\/7\/1205","ac":{"\/ggg\/4\/7\/1205.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/7\/1205.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1205"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1205"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/7\/1205":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wD0ygEHkkLB8REOw8Dozlit6Q1DnyI20mPZaLusEkuagOlYRJ8eLY8tNNmf1E53BnMG\\\\\\\/s2gH\\\\\\\/t5zLqmb0uskmCGLrU9HDJj9hQ8oh66w8Q0oQdj9uMNPc1rNaBQBIywLO5mJaFQ5Ninn5d7S1bausEEqTvR2BR6RF5SIsd\\\\\\\/ERD\\\\\\\/tcc0s8ZtRdLpjaGIrlzovEP3cLNElmrm8GtuayxMBKY66aU\\\\\\\/an908yZjBGiSo42Z1CwfPh0LLPMFhU3384Y7VXMdsL7m50aYbYQZ+\\\\\\\/ocIZFtqsC2dtvzoa2\\\\\\\/4mTsiOE6YMQQCIEZ48w7g9CFInbx8FpmI0JMStwrdrRZKgE00Db2OLgG6z71K7pYKNkGx+\\\\\\\/PTd6TpBivBENe6Ctr6dvVgsyTCmE\\\\\\\/THmZge8p3v4qVLSz4kX8j6X\\\\\\\/bvh1+FudklkZ4Yz5015zy1WZFMEGyRBzJIGOR2l+oWIF140LCXBQA6XhRzD0nk\\\\\\\/P1SuNQ05m0kg8BHnjqpsUflGrSpg3frDjyrf4IZgHiVHEGKsiUHgt5kNph+qiNgS7FbvqyNhfYpXGHlWK6j7RfJhZfN+0HjTpUWZXVOo+8gKDLkiqjmIrQ8qhQmxs\\\\\\\/XgaTUODgX+Mwjv15g1gGMWtBdH0lgCKty+obgD7TGDDSYRpgRXFAyKCTlms6H0pZdfoCSK94O381eM9I5c0GwjULpYvPMmqX8y9sU6L7LyYvQlxEhy4ogPWz2Ak2f3N+kYJSzXS5DmxVbu8523k2UyZYL6jleynnk7sDOEme5wVuFCW5LlAf2dHFVhMnieLrISj6ROTQvQlGudCi66iYgQo8SITgZHbvdUO+ngBxcnAomjoDyKela1\\\\\\\/oKKPOnyaMMIyEPQxjvcTOishrMPEHXnD7nwi1O9ljtIdvv26fXL3UHZ82IdKWUvZQX3cjFYrTlYBwqA0hVsxs3otymAQZoK1dYz8XX4vjpQeuzkA77HdLwAgOyb8fqd4g7vVsBtl1LAuHH8V3cOLjtbTPc+fXAcPB0D6Plrf+suQOTmwQjvM5Krex9kBlBIuirwz6lXd5EZRneSAPYGBOAN9Uiq90YPAMF+ez+GZ59gqBWSnkWI1vI4ayJInLwdnqtPDkin4+Y2eeUgAgrtt08AIs6iXezIBKLS4W1+779Ctb\\\\\\\/o0EsyLxYHyCYrHBRRybXqfO+RddAxysDOb+f6gONrq4bZtNN7f8kTMnlHnhoCA+5mGyAffEczj3O\\\\\\\/KQHUBtV9lZnV+wLQg2iN9oI6ZCYZ+VyVQe2PwN8b4TmhxJ\\\\\\\/MEH\\\u0022,\\\u0022iv\\\u0022:\\\u0022qXfIOfW6C1+VscgYYvdSNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a931cbdb36498793776cd49ba718ed61bca96cd7d8be2e0bf1a671501aa9f259\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DphwCkAygCU0l9unVidhGURcQpNMVJkuXGNc+9IWey9iqeGS4HzS+pOzx4ULz1BOkYMidyu2glupZZCFsJWHEP2t98DNyRhd9ziRqSvL9NGVFUEihMls9C+eqB1CIts0dYryjzgjnh8+da9dT8k7l3sFW\\\\\\\/vt0hoVzTZ89ID5+KunNHiTBith0InvbhHM\\\\\\\/TsVxylQhIAjjvEfJ7h1\\\\\\\/YDzIli7qE3FSZs4GN\\\\\\\/OeAPRfgZ4Utoyec6FyaN9\\\\\\\/OH0eHdmqGuCBXDtOKlTvhJr5xjDDdzqDNJ+rBk\\\\\\\/uX9N9s0SQYk7jUMCDzlNqEb08Yof9B9wYStNP0IFsJVScPxNUXRsBAjRhOuSsdT9wCCM8LoCIl8Oj\\\\\\\/i1ANCq8I6xbiYA5S3Bl0DDA+QuaKpu8yd86Rpqcl\\\\\\\/tqkagmaonx025EJzg4+Ce8\\\\\\\/yh+5pxPuS\\\\\\\/GTNsjHJnbqJRumlnnRpEpiNcq0Cx0oonaNEfOPK+AnGuOowVxRntPiYTE4i5fDX7olBBHxalO88Q1Vh4wldaKkIIU0qXYo\\\\\\\/kwlYtQjhSv4J1esdxSiI4xj0txfpwsUklQ2zZUSI0bZRLX8giuLPVq6izFxh+0R9AyTCXbIEIzr66YfJ4dX592PQ8Ga8pSF+Zp\\\\\\\/0RLqvZmSVSMf5YyBlWDaSDzbxBL9JkiNpCzvBD6H96\\\\\\\/zn4CK\\\\\\\/j2Mfr\\\\\\\/x\\\\\\\/5UwjU9TPHIpKr\\\\\\\/nzmpROAO6ddd6++\\\\\\\/0hrWMo0I7LVkEAwaVBm6msg0VLpFvCIxnwtbgaHc8JWQLoejO2fBwAvN0zrQEWFj\\\\\\\/8RbUCJD7n\\\\\\\/ZLtsA2DtPDp7TBnJkytYFjEduzqE2NatyF\\\\\\\/DmCXXLUicfvywGD62oBpCt8qPMt+JGeZXZUCBrrjHAL2PBKhb9nzlcp2WEzTVAlcxNFNivfdifpUuOt6JPDtAcV+uMe9LLLAW2g66W1wIEwLog+a619HTwRRS+X6EmlplMC2y+cDqYuqb\\\\\\\/GyEtBVboQz4UYOo48ktzK9jfkkXsLdyJWknA242yUQ5GSpO8K6dA\\\\\\\/QW4CrLGzbrCOrb+CUNukQ6ybAXDwWCT3Bzq8pDjCNoyF4wSNeylb2hJvbFEuVTo5a9f\\\\\\\/4kqmQ\\\\\\\/c1EbhB0r2OZEAf4w7bV\\\\\\\/7UDjj78lfB2tuHTT4vlFb1Vr61CBihL+0C88JNSZnR\\\\\\\/SLEp9boJoKS1R4fiHlJ2GBqAPvclCwBNhbW6XcUgO+GhfA9TpCaCB0Nw+v5Lo2G5jWFjgBNTmpB0TJe0b9o98F+FO9vpXsgVJJGLRdIXuUmf4d+YA6XgR9YRg+2mz1rTLE7zD0C5Fu5qBK+XnlPMriKooHd0T58+8Nqz2lpgv+R6ipSZ0XQBqH5YI0VTzAdT\\\\\\\/\\\\\\\/+9wRiN\\\\\\\/Y\\\\\\\/iLOZ5gA3eLExSfBtFPKDGXf1NqkiH\\\\\\\/KvHc8uWcwJk8VXlIaQ5giE1pXur0DKdzCVEhA8eoiKTEJR2Is6Tv5qmvLP3JgStCY+6uu\\\\\\\/8D0i4Jtfqp5yhNeZOuIwByc1LCxOEzYQ2j5XcQhb9kxPrYCOcjrQiFrNMQFin9mqpe7oVsAUQEGilKYFn6duAGVXd5O90bVTe9ZqPuopIiA2MMJw5RHWcaPkEvKPEZrKYzwwQ18Ej716fJa6WSZSBjnx1FYnAIjVSu23X8n+WCfzswOFKt\\\\\\\/J+d94us5X8+HSboVsOCnvKYSAuxs+ziWeIwDSx7eG2WIhX+6+pDk8xQQmwInIitfXPwWX0wQPJsVqIE9t7PAg9CdISlFv3GBl6M1o40rU7ML+nK7i2Wx\\\\\\\/sJLw\\\\\\\/WuS+i+HhChJWOYhTUM14AWV+22bZ1oeVhmE9moG+1XEq8fdBZVdTqCJH3OKj1szwFv+8fX\\\\\\\/oazpjKEwwF\\\\\\\/9UxsAsuxew58a9t7yvApAPJvhsr1Qirk0EE9EUDH\\\\\\\/XtHCZE+RaiDKUqoq+SUd35qU7iRLb6xa9cPrcurKZJpAxa066mtEiiM4ADNpc9+noS8hbzaKP2bbF8\\\\\\\/vguPGQRJbsvZ2WlVotcmLtbXy0H9yBOrg2mTlYhZfEG252lqkT+E9iQ1sT2Is689to\\\\\\\/SLedcZvXXDwUAA\\\\\\\/2nhJceGAHzSZEX+nNRSVp\\\\\\\/M93X0cmtxvYROmSy0ioJXUN347mMrQ4ZbHa+6ZyNAwHtk5b2K4jSHijmAZzRIw33ArUYflYVEpSQZkGn8qgZdW36D6wLvoK9cVVJoZWDdvpRP2e5k9yn8XMq97+gww\\\\\\\/ssVhVc+Y\\\\\\\/nvJXsBv10G+dnqVdbAqnXtXOeCUuISaaFc6UJL8wQmSG6bKnKduxWE\\\\\\\/pAO6S4clPWwVlKaGo\\\\\\\/xWz5qfE6bXdexDuaQQEFSbYwvl+PXZVmbSc+E7mfOJ0ZDUKzwzT1gSAQB\\\\\\\/1mr6ZWfEK24qXLdJqjxrXbjszSKif\\\\\\\/SWkrY7qaTQI7cHgUE5MU3ZaV5Q1CjMqxqTa6Jk5OAQjdG7xSYNJn+O8Uip3FnTJJ33wERQQq5oTPHyNn\\\\\\\/ixkoxQ4z3\\\\\\\/uKOKmyBSIOWB4QI7r85\\\\\\\/QKA6Ihva\\\\\\\/ZRPjGn+QKExzPNnGYqUxo5Ymou+kzWqD8JGaEJxY1JW\\\\\\\/xGZwnxYz5fLzk5IHm9\\\\\\\/qvZw+4xRHadVvBpsCtZqYvDM2hhsWyMRwo2um0jTFA79m6kRFs1l2ZCZHCctBOv8gIgVJE+WdGAOSiZ1uC46gnqO6TZVy1pSnJ14OsAhURy1arohcMoucSqomNsjlE6WmUZuWjrjOvDSBYEqlnSzxt9jgkwawmqvm\\\\\\\/WtW5FZTcEYKd3T+z2CSCNDOIfZlzgN47y9Ap7pNkgdVOeiz2gIq5qwoWRw1UiFeCj4Bl1WhbhqtgHNtX6Iafw4hnsNeG9rNGioXCtgu6gUURO+fTb\\\\\\\/gpBLhB8yKQ5d7olyU3JQyj7R36\\\\\\\/KF5mgGIbS7zhcQflATaxEdAX6QznztAg0\\\\\\\/uy8fsAhl6Gbg61T7bS1SXUF9PCcY4DkUPk68wKIL\\\\\\\/R3ufMswYCcSU60Nms3Xar8bpwcuH0Pcxaxv2RDUDHuHfYvVRrwP\\\\\\\/t0DwrPSb+BqfXc9q1uwoU3D3a8LZJIHEisc22h5Y06uNUZHFdykd5OBzqqgx8MKsIwVOAGJf8r9Aki28OQOE3aoSRzVnoX8ZdaHJRbyJc\\\\\\\/LMLkPctKgHRe+aRXNfaSTRWIQwYXi9gfh2qw7X3uAw2l\\\\\\\/Sny1LEkl7yPbB7v7te0T1IlVg1Bec+fkwm7WeIMbUF2q9WKWolQBeEyDPW6em0L2fc\\\\\\\/DiWJUcPzORqeItUaT7cZRof2Zr6SQ7h2B8wVi2HzoK1Ws213qG2OobNrrC6IDUZOptOMwP2ijHOxe\\\\\\\/BGp3HS3hu6kGsZCk7v5I5VsQAVrZxkyZT9bu5hVQaHoArAiQ6ozlFPwa9N19lEP+XLnHqZIc8EZ1ihwCX1Agrw2GK8QB5yYhRTqov+HKxS+KyYymWbAv9NdHcjduwJjeMBqPDh0SA4nNppwLdNoQDvwHXPeT2AQt\\\\\\\/xnUOjlhpFQHsNB6hOeu\\\\\\\/oLhhmDe0Tq+\\\\\\\/1aQ5QQW9jV0VacMAvnL3HruGnp80C9TeKpP5blZ3QsLXHpWFZxc6AK6pgrehgXoWZLJM8o3DxBz15buGvrsPJtB5cSxqKCTf4dz5DBGj6QHM4vsSuBx3zknePPxVX7G9LOrneF3+u9mcmdflJ5\\\\\\\/b7i0Ncs2AunmL04ZikKJc8cR1TzntBWNmY\\\\\\\/J+mJcdyqmwj5b6K9V85JKMTW8t8a20hvgI4lj3hFFMc6WbS7lXdoD4q86PFZVz0bAgROsXC37ZA\\\\\\\/KOnrsuzHJuJPrVoy152BOB7w7SFIjieShKZYrU2xlkrREIDLO5TqSjNMgx9ItvFInBSc029K+Ml760TaqkJ6cJWRrcZ6cYCoF8EY+oMm85KVUsL+MdW9d2U1ElLk+VydabzGrov2YObiOZ0UFr7JvA34xFufNehq4ef2KM0L5ROm6Dc7xVOy1p6bmM49xC4OKwE6G6BJ9V+evDexGYDvaMCGQU6G4tBNxcQjWXdIUEfrN9mPwBZ0Nv\\\\\\\/+WvYmLWSGcxLS3kowbEBMRSJK+lB0gc4h9PuKDFP96XVjMqQTP2vQgACTbdfVJROm3L1yrZPGkLgIvfszV51GpuT6wUCJKc3AnSZYvqYUyuNi6ZOExnhdvh83ntg1HoN\\\\\\\/TQmOh2pSW6wUDuDDNWxba4EbzZEYrd33CgLBS903LF9HVWi++oZyLRfcFIWUnRVZZ8w4oZrmgIdedJ2aOwe+TjtYuaffgnj\\\\\\\/wCNveihsMeoR3xJLkilxtntQ+KG2XWXK0jNfP4m6ZC8K9DpOA86CRIzvf9vXxzwAdSbB1qnsTpp8x5cftGNT7Jw7KvQxGATD84wy+L2kr55wFGaaHrdfIYmfLjkDKE+XnjzGQoIKim+yQUPKvGOpgYJwCD9HCiNdyZJGGR8IgaU7GhqFaiBWETS3Rb8c05fMZhbOjvE1mO9kd1yf8\\\\\\\/8RQY3Q1EEzdmAS8H0tujKDhY9U8bGKknhps+LOWGq83XsmDG+\\\\\\\/sl+pu\\\\\\\/6lYz7yyozN\\\\\\\/s1+hbJLl3FjOqGyajkjEbaDxkgpA\\\\\\\/GM+So4eJA1t8wbCWTH3h6XF3yrpwYeTlu0IoaKZlqsHozjvLHeEpBOPSFareVtd\\\\\\\/UhVqO7MYEzVcur527G641Tr+UGmVnH8sQttBRaRaLSqJ7BoKZ3wnEl2TEyZoSNZya77Bzj2GChL7PIhITphRHdKF2783HZglUUJ8tiuEKBcrVjVIctlzU24VPOHMdTI4lhT29ehUl8m0xOLmfcIsD0RTuK908vuRDZKxptq9Mvsj9MRLbvTFUM+z89pZvzDFPXxbHcUU\\\\\\\/Wcx2srJu302Go+SnntT4zvdMNnrSzLUbO4FWpXkPykEk3dvKxO9FGAo4UWpRHz6kVr4cIiw1F7\\\\\\\/8wD2nbsf7m2n9l7n9ltvSPg0sEdrT9kPqi\\\\\\\/r+SXi0MTIyUCgQhKM6dhGyFZLqvhCwJYgFFyZ\\\\\\\/JkLuhh6\\\\\\\/gnbTmcv007JGSCdcD6Sp8WpftSUlI2Hw7qxaw7pfzp7nvIZuKNKTauoRISZ3Hn0CdbaeCwlj\\\\\\\/jfXaO65E\\\\\\\/LZ+uhBahuBP5dH71\\\\\\\/8Q9C0OgkeprarNf8MJNXBdT4IA9z24RHgOhpd0DpOF78vXQt+07YkvEGVVXe70RFCpzePVDbJLw9iwCRALHXmKYK\\\\\\\/C1IbCYlyosE18E2hDGYGvovtDdI30Dc8fmSig+TVhWbjxXWSN4ZAxvJe0yasDoDYLge3cJOaneDGhTHA+znun4uLCaY2Y+pos6hX6LbJQ45768jDTQA03H3EwTHu6q0QK2v95CNJCRGA3cL4z4EBhw47hDcsk1KWSHTyH3Wt3y+YSvLSPmABIYVhHlKMLWn4BsR8qkXMpjYwHVT6FBt0GvtPV9foySrJMNkl++tNuLMnaDr2xYwFDUpfth9drLIydj902QdVZUlmH4\\\\\\\/iu8RHn0Nc1FNVGJlYTqnNFUrt24ZOGz\\\\\\\/8LxlUrKdjuPtEZwpX0QqTk8mR97O+rARoR2rXyasb3x6VDC1YpdYHMj4625Hafh8muBOiejiHTBeSgfQ4K7wdGr\\\\\\\/mxmZyF15SYTkTa7J8uc5J8mpIdwt5bmjTEn96UdybRUlyKDVbGrbhVAF4iCz75VCOkiP5wE1DfwAPppMUwK+4chEA1BHcMsyYaSZxIqRmQ3YOpKi41Iqs3bUeFqkSq8T4qCHbKpkflUXs0rGnBhm4DTOrNHTVVbkezRE0JZBB+oZVZnuhcGYgbOTw9fYNefGv1dnFHiU943TMGiv+qE7BYTtjNCX4APjPvIbsCVw+\\\\\\\/aWnU4J8NLzDJSVkQK+d2k6FBqjWpxasJVQL4uvnIKQn+noTAAVvE1Hcgap6tuAS6U6rbhCEO8bXbt80q4biEOh9fd4yx2JcYqr6WOmXlkd58NcET1w8HfXu8T\\\\\\\/t5qM5K81HrceAuQmBVwpLttgmMsNXN9oB0\\\\\\\/YE8M4a+uaDJIgRgy1gq7NMY1Rt3BWx3I3Kw8fGgU96NI6MD+fJCixyK9VLQ\\\\\\\/sP5h6ybgYdxAIZHUo7+fpQfMM2Rkg1R0IXpT2f+d6sIll14YgZfsuI1A0eMWcuxQp1xoSeQ1J1P9uhayfdww\\\\\\\/czTsCVwAsM3qWpcxIlJKxp2UDxTyn3NMwz4pPSrM5SKNUwTARXUW9E8MO3NXnoOZKVeFivOeFaXeesMtxSOsMK8we0U\\\\\\\/nczuFmGp8hXMTR9DHISHmUYnhd7rTdvlV68jvHhetxdWgTAWbB3NPr7lRfYMcgFP8gNCN\\\\\\\/Hhw1GZYXsy0tETq1xrKeIasRXlrbnPMIGajv4CWCjmjCeQE1vdN451OLH6+Dr2AbwPt\\\\\\\/St1QaBjtLDRvV4PIGq3c\\\\\\\/ctnbocmz38uSC2jTQ8DRqfNXd18L1VFDyXOjwu2lGSIpeg8fowz+wD6z22uqreURI9ZAEZvGi\\\\\\\/4QvhxfceL+RNQZNYe6cYxR9hmV0+zb22zg8ZBZdDWAFx2h1FEVjR6slDcjvdyBKWT4\\\\\\\/lkcC+LXLeiO9wwdPRucF0AVT0cG4jmMjrMu2RMIGc849aH21TuTOub7HiEOkaAXUVBG9Ul0FRUGjDwXK3v\\\\\\\/f6NL+CXe5Mhq7rA9Zsbgg5mR+Go0ctBxs7lOoObKhXHZ+4UYBH5vS8D4++8zO+vvM6weiWD9xP7ufT9\\\\\\\/mHav21cPTlRZZoQmkSSnJ8o6vXRFAu6Mnz2LHVQo4snkAMitjhcusMt1Ga21OawNwFpy4zQ2WETThGWY9YD0hzgKndzQ3Ey09bJla3\\\\\\\/koFb5BwPehSQEgl0s46r8jNve1MEUIhxYyOdYxW71h2CUasVLeGJHc+L2po+rTu4ENg0DDMEiFMB7qlUZCFABCEnbKpbz3\\\\\\\/hplLjP2KBxeAtdlx6JMmjPfVGckPt3PfP06qMR+PvOGejs9sAW4K+jkWU4gOXTepnJ4VwsuIgWr9ni808bOHZ\\\\\\\/3XIXXjXkUTPy9zbpX6LS246nCzHCxhWV1ZjFgmYcruT9p68nMUmo3kDGNuPYt\\\\\\\/8ZrSRs5VGrMqRYFwhKvmI0Z65u2EV1awveAWmXNVFzuYCN1iHup5ocSpImfDU6qDMsu+e0YHTBmDCwt3DrjDm9lJLsCjpef7rzIsJlpeKYKgEQBJW9d5RAFDbGpuUn61vle\\\\\\\/3fNjRnSpPPtBgfdg+J2JjQRKdunxg4nrgWNdnsoTEdKRIF0WxEg1vfx1wdAZ51omt5SB5MI94LQIoUSJEPmgliU5XOKjYlxhbBvqPlCwmSrm2gfIlw2KIAiJQ2gpzWTqVrYbcQREJO76Z9UpKwBJ376c7hIREo27gUpyT9VzP+BDT6kPbd2uy9of3enjKZpAyHNpRU9+07GmoR+SgURjuRFFVvCXvAnqElN\\\\\\\/e6+QowQRs\\\\\\\/YcpmNbcM\\\\\\\/u9RHuCat5JArF5ksmbxN8RM7xVmAsbr\\\\\\\/UprrAdAaDaivea4XTCc9fiey9J7Gfm0Dplobfb0VxfPvJDAzbxVC97EMBn84FWs+31j2YgEP\\\\\\\/YDXxxKyvlU65Gnz2EE390s7JKvQ7Gnwcc085o8I39pqKHfkyOpKpef0Mao21j1e9kxpC7ijZ5yGoSioPLjveM3AvMys15l2LuD2tyqOmxi8tvrF7U6v8cg\\\\\\\/p5RGaVa5NgDrYqh4Z3kA6jp0RV4M27xgYMUpgirsgfuHqCXSy7QAvaW\\\\\\\/ePtjVY1ezeRNflNVLWHGXiBsNRI9K2oR0SSIwJJJlRvDS3oaIhLYUwUMMpGNZQL2M5h2yMDkRFHjVr\\\\\\\/Xh\\\\\\\/qCbOJUab\\\\\\\/eVNFAiGitJ8E\\\\\\\/FhWl+0+LOQx4kCVb0uTuETWdELclc9BS7\\\\\\\/4x0Wec7sK\\\\\\\/RW1yRMParf9XUIP9TJoLtW6ZlUfMYwiggTY7V22MO6l6DcW5tqi27euvRZaFdM19NsS2jMJptF8JaEzbS6IF6XrxKVhbPfEFOtXAPJdIwzUagGIMExfgXnvYWMxoQBwVF3R8qnSnHZ3x6cy7aq18Bqkx\\\\\\\/TUsO+HM8Jor9szb8ayClbIFIwYHvR397dkQtxleE0pCtcZO37K7jQpnoZ5Dg\\\\\\\/FJGdGzaOv6k\\\\\\\/Vag6qfv3XWnPs5f70ccAOTUri6QMreGyTF+FHZr6POSnBBX4VOyI6cOI7qll3vyryMT2AbWOaa7jtyKpJ+234HuEhIgam7mnAuS2GGE2dzXGbWNPdAJc0b0JLEf7DH5H0r4MjX6pRDph8Hb5GenVlzfUHDDi7T0qy6Mk1o7HYJ+alFqs0yMwejzXl0oil0dgGb7dJpXFcJFKPj\\\\\\\/9NZL0606R1ytvwTVOPbT0T4d+0cMw4hBLs0iP2aybHy8GIyW4VsdeR1dzggv7IytWpCIiCJ2Wvtxc9RVdzKViPXcR8WBZcorrNU41Wt0u7yZ88Hyn+hHxi3KAnvjjjs7iEsfX5BY6H0rse7w1Bd+TbdMzHDIijzuTmEfkdHrS0dziXuEgBjJEhEuLc2GvEIvKE0dJcgUQuWMijUpDa8YWWgpGO3DOq9DW2F1ocmmZ9s3L30QmNk+\\\\\\\/dEWbqiDqajRMSgp9FNS+9s5JxYz+hkKAUsHTt4DpyyqAp9maSgg+UrBO4vmc6KE42Q3w75STx2\\\\\\\/Gkyuy6dXqzJedqHiGB5kyk0oZ9vjfuFAIHjMudKO4DcBk8RuJFqRy8hSUYjwJH0nTekUQPeb6DBqtkyKBQxsLpht6\\\\\\\/A6ZLj3cZzUv2Gu\\\\\\\/TZCLeW4QIP4f20w3\\\\\\\/V15T6tAvHvUfUQ7eOPgekD55eulFXVI0\\\\\\\/O3h2Tqtk8OQMW3AzImrekjix0LHwgUIQgLUwMo09l138+kCRMvbwfzp0NKeio19Den2PPvnXI9f2yzbcA9okvLMFdNoo1CNt8ePXqv7jN\\\\\\\/G36B2QNXboJJ7igxXhOzX8VRt27hdJkVTWKykfe3if9yUimMSfFaPKDoF9eeC2XUuYfUuAOo5wAhT6w01W1VYciJ1139v++fWBujLehWxDR8S0\\\\\\\/2jyN8ZzeQ+OfuxqN2nNE2ImivYebU\\\\\\\/cM0TWktBMAtIuXaxSi7KJvacx9HrfNZLSEVigu8vMwC7rJRAb8sT8jeZJAXACJBusY2KIxhR0lqaXuqHDYtgJsX1pGShVbNCOv6M54Gl8Y\\\\\\\/tR\\\\\\\/qzb4RhRiXChCeV3OzZAPTgIQCbG3Bow+bgKWKSE010OxwcLnjqYRFPynZ2TRylJYNbXcIuKPyz9KuYFydZeOhmW2NnU7\\\\\\\/7hXl\\\\\\\/QPoZRKtgfYXz8CT1ZEw8lUwBAWfc08GvSjEiRsl71Xis6a0cLTzDE1J30BlGvUAPzOOs5Rcx1bMQ+B1AFAbMU9HwO2wTV4+Bjncp6p2rrDM+\\\\\\\/1c\\\\\\\/XcvAul+qVOUK6Y2AkIXjY2abhMlGM6Pb+OaMPippkhqMqmgEvGLGPXOjpU8btxJSkXpBrTTqH2cSVcsR6VumtfsgvRIdlhVhYAjjfftzd5a04Q\\\\\\\/JcRUVWKZZ3TaI23ZJGt0z9u9NU7QXbvBjPFda57JFEBMVJl97zxUnZgqo9CAOy12Sg5qJp+uwXapWYEg5Br2hhcIKcGqmc7ecqnuhQ6+ySNU2LB1YEUvn84d9JFN7M7IR9p9a6uvVPRLAWFQhhW5O3qiPoBanxw03rCW8SJMrJds\\\\\\\/OPnik6V87bSEWCG\\\\\\\/8aMRKEHQHB8acx5AkP1kNAOrIQzlAumr\\\\\\\/scFODy6QT+fZPJL0gRKgYbYEbV8jbOLUQW7BBBXZlnqsvGgmisZCLkk6lEUmnFZMJLMjKgjVmPr4LkgAp1KsilhNAe5F4i\\\\\\\/zizc\\\\\\\/dvKVniZncKrmB5y\\\\\\\/27QPZpzxJgPhOvsV3YNdPn3wbdF\\\\\\\/pUs0NqJzj9PdYVaW0Q\\\\\\\/Q30uqtFph40ZNourXKmp0cpFAe51WNxr4v9DCQc3zauavS9WiALPQku6yR9u+IuE4rOvEqA709svCwBAn7EUyL8kqrw4wRfr130hgf639hqjWgCgt6wJ3GskVEHNsrtsKDJSZ9iotkN2xhXPHbmmGS0dTrlzLbGotgFZ2rbvmnkJTeOvf3i8hBmPd91nA+ur7ya5zXuMd2UtXjJzAgeY74kHDQYXzkks7EI0EX4YFnZTHot9qztT2n5923goQGhXDdVR644wGWgzECmhuwlM7bx0iO4OEcNPBgYrm6j7qe4Iblwfomy\\\\\\\/OlikIrbs3wAGiq7tGpe9zeLF9+GTjbFpJ+qsY8cevSRDBA1GxZjgnNxhhp27PmB+m8DIc7A7J7kpX5sxz1IV5eUCUePpib4HPq4w5DB3VCixM7IjNIR1N6je+8FoLHHCr\\\\\\\/oCjs0\\\\\\\/L3C9nrfS01+1S5YDoJ19ar7l1859hKerzsbRPxsYLYQiGyBeb+DhpaRTkia9pojB1G+LBRGRqzT1FRkGJfh\\\\\\\/Tht99e0Cvkn8BqNeQ0d8LJ+yWZxhGEtHT6C70GIEXmiZJWzEz\\\\\\\/GXM2Au98TJAPPl4sm74AQFsobSI8swcW4Lo2v5+lGCz84zp4Z8O01jdv4FRy9GoVn6hE195tJG16EhEhWmXx6OX3EoKB7MkrUTAzTDGnh04YtZgCnHlufuYNt2OsbS9EloHen2T1TVw1hHHtvuYOi8ZAOr+\\\\\\\/TKB2NocMWCvvzeRs808g7PDmu3Pvc3l\\\\\\\/1Yn0EzvxBCNMLayVgxpYwcPn\\\\\\\/J5RYuXSgXMAtdkWPbLWBGOkYIwrQJOJ+CgaNibxmO1y04eTYFcgOICMEbbMWdtRZXCxNsFGq7yXiV5nyvUK63n84z00VweNKqrSOKPDlIbgG4y6MYw9CGwuf4PTWLQhMNRxMmCc0KeBfwZQnbAgPFcIW76wBAkOtj\\\\\\\/Kpp8XDuSM1TnQtVsZ4aogNxxC5Jd+1zxrYPS5zqf4fNjDcy9vVTy74lXD6T9IjuHtVIMV+MvrUsl+kSv75fbOEgkf3iAySG1p1b68vE5k2YX1sU3vfYZJTpKup1SzyNRw+pQ5J8+ILbzRGX8yBtkxkNYGBcQRscmX5koJ71ivqErRA2xFc4N1\\\\\\\/6QA+4\\\\\\\/8tVzaUz04aX9sRbO6XxNUP3SpD8L0bu71ZgM9nqasc1zDfeqRjNpl93js0tfGP2PFZgVWfbNL++LynE13J07C20jw8jHQ6qh9u4h62vfINqsrrrQZWpJM8382RXvt1UsQ4IHTtGX2Jt2qKIeeLxhSNGrwRlxcsdNLrYCDXI5T2aEsjaFFRgdf\\\\\\\/s3pOflCtjKcMbAYP5FHtbqBSuoIAThRBUveVLyXAznJ7cNxR4r5aul2bqhkgGL6SCJ0QJYnXb0a4J564tGY09uCyIyWXca7AzZagqDxul+qYECdLavEKTHQ11CRmCH4oTo37GD17mrnxH4qjszxVVCDXA4nUAjOjExlJv1ik21eqSUINTxJvQ8E5XFsEkO0VMejhUo99OPKflgI3B5MRHq+F0uNjImo1yBarqFZ1QhE6ZrgbaY8fiGFEHItsHWZWf5DGxdlwlHwnFc\\\\\\\/GC2vvyLOajDxuLx1w+bNjIIIvVyKDhKbu5npFnotn3AODKdTNjhrp6a88jjOBx2\\\\\\\/HeIWN779GvclNyKSH49BiB68Fg3iyJ+lh5zVNRT2LiC5I17KG67qELnQR\\\\\\\/dagUtgO0mcecW3+VATiQV36hwJYgB5lTvRVujg4l70uuEsKyYtX1dd1DMWLk1SjJGkldQ0zh+QSy52R6u2t5adZ4Z8E9mAvE\\\\\\\/b84z1CJbUgwTOoarPoPUPP2+LCxP2QNwaeoWdyNj2NivV20zwdIzYaEkMJu4E6NGKTs35223nPy1QbRFyLgkW+yYPn6QeE8Obx578xy2L8OwcTagbcYEA560iwJ2WdRPhX9PinYTaN0NU9EO4RXETiMt\\\\\\\/Jz76sJi2pxkQbFy1hhBGlTizL3AYvS5N4rvXdg8DihNhNs8WvLOTyIeBuTQ0VhK1A4ZzoFVzCdDVAyTXNkKUVsniVTbTCGgRT6yv\\\\\\\/8ODvtvu+KpYXDZNd28R4Y9\\\\\\\/nTE0\\\\\\\/wbk9V7VtHDBBHU4WCUcAOZpDhs2Vyn8M4QzId2wezze0eU6B8yWvDaxxRAvI0Ji7VndxyWeuyzatBpKvFYSfuoSHyyy4\\\\\\\/wmDzkHaEq018Tr4xsfJLqwNvPtvP6z6rPA4++uDAPlKns\\\\\\\/QKLiCx4KG6ucrFgdjGlYzS9zwoDlQjBBRLXrk+5kCwRLjYRBZuasUJd09R4+NIgvN0Q6rhoHLxBjby8vo3fZ2l6U19p90lWpbfbml1n58gt4SYgtv2CoWOd8kRAtX9U7E6U16ZgZVIOfBNJaKAQBWo6SLdfFmdIOXTms57UEeDTgE6OuZkjfY1N+rS91swK1ZhGkNVVyhORFAJPrh4TDwCIBj26y7\\\\\\\/Uls96tA7bZqOhD5riEKwHC1YFo+1Cy0u1qC0DmTGGo49nFxDlqYpZ3AJV8TZbl5uhG\\\\\\\/62N0DIZzWumyOdGHtg270uz+ta5AXX7zppd4mnxb75h89FdI5ycPRhNzd3ZXyKkAeCw3TQUncGAxHkkPrWy3MIfqAOvv4T6SqVgL8Rc8gDpMsSr+rNVsr\\\\\\\/hr5Q\\\\\\\/X0NNP3BGYeRnlpVNcR3mX4FL7RERvly0lQYh9m6NJDOfq2rVvRmxyNxsLgsoxT\\\\\\\/R6TYfXRL5LY3yQAx0w\\\\\\\/ccKRLo6ZSlWGUzpvjb1UQm1a3PUKr2EGeWf+fXnz7Yjm3x0AbTxsguzkSH+xY5Ur\\\\\\\/PcV0SVWVqn9zHtxH56sv4yc6Vq8whm4pOJ31VpIgKrPUHb9ZkIOlLGHLWNc0CWXdx1OiUiEucpBAxvDchgkiAOpqCyO40PTAyIxmsrhw+2KeSyj+rjjYYvHOm+5YmDw\\\\\\\/BIgM3dTImZ3K8X\\\\\\\/9lP99\\\\\\\/WBoxs4ORinCrgldtR5PABnKl5B+ElNrS7aSHk6Nc1Bh9YPv5xUF4yGo4mSoCOwyWuX9NLLFztfeRCzBgyvZkyUGSN2LuloQuTZhKbEIecpTxdTuWi52ojsE11LJ3dLhzRohdob4904lWxfmtK8xMNNHrp09WPHwPBQnrIb7Go1ES3EDFRzAwg6SJdmN6bH44wzc9HXcrAZMXK3kVzayWxLhCSz35uD8cVxHBJ9\\\\\\\/icKulAqnN91UWf3OH5404nFfkDwdeZbGguAKYk5s1pS79SbtoR9iOkUIah1N0Fm0V7weaQoRsBZjrhE1EKABJls3C8u6padiKo8ZIjs8swxdTyqPDNedVFnkT3kBRmz1woFeada3sGudpRNMhWKqzPEXi809bs2o9ecH878OlQJbzezLJEG1UOm4C8KsMLD0Wvdlvnoj4JWF5gK9q7J9l1yokijFq7Y2fYiClYvqgrHQ3OtQSHPBuj4g1vRTz+SQTMrc9I0pqABRoxgXSq2aUKFZhV7nrf+SgbZfkwhyY6NB7tfEQZHHp7aSBdArpW15Aa\\\\\\\/iO7ZgGoV2VqXGFLNnUlvqKkLOIACuBUoMP\\\\\\\/UFscz4Q4Otxgqb5QA44QCpXeevWRYIDT8Q+BxlANJrSId2P0CqGQKx1ZNtJl7V+RXJsc29tFvYmoBMyOHsEP\\\\\\\/L\\\\\\\/cAwWpoAMheujdTFucdCsWLHN7KLn2axQSTq9NC8sp2vCbrrQgCReSFJE+MBDBLCwVax5sRK2dlHmDm7X+PdPhEeAIi7Eew4QYnCw\\\\\\\/CVAp0C48ntlKduk4qnMykwt7kppmbD9pt5Yfbaw8AIK+BS0n\\\\\\\/6uix8xi\\\\\\\/c672iYgb9EsGwlct4nykmTcvNxsb4jY39NHUWsAM7jSnOG9jdF4o3fEiETs7fY2t0+z7cFLQtmxh5C3iUE99B7zgYpcK\\\\\\\/wgyjHlhdAI6ezQvYa8y4pv4rBnrZtEm5wDF732m8MG143ToTfTQNU7oewxHd\\\\\\\/n871NnJme\\\\\\\/aJ\\\\\\\/+r6uGhmrnyNagY8aWfa+rwLztnzUUoKrK9C5vUZ1aS0rMk9\\\\\\\/Evsys0KB+lpvq5DbAlN5nRrpXjmG+c2BZ7txrk7hEgGBcXiWMSDRvYAONuL602kP4lkU7w5yTHRov6T1xCMnBEYmqLFpINFPKhs2Rj3tdYA8D00GV0EljtXSOIR4paHtmbvNCDlPY6XYVrMwXN74eZM6y++RBH8j9Rj+XbFMm33qphhYAVzYpr7HiJ9TY9VCLc3wKaUn6Oli6EympQm+vK0gjFlHirwiu8Z+FybzeLvo6nNulo+rEFCdZpYtvARFvPDKUfrtTZWbR0Df7D7522RWZaAYCoOgO\\\\\\\/whO5vXXeO3O4sySSI2JamE\\\\\\\/mztcTQ+5cED115sP92j9nHqyOaQndMNA26XNc9mxewe0O+t0+es6h64emo7TmR4ImKjMzaYwzJef5rJH3dBrBI1gI56B8cespyBw3XYO33oG2o0HyNhwhpmUpqujJDiJH4LSNxUzbDOXBTXY7KSleek2dN\\\\\\\/Ei4eboxRAXgCnp4e8YVNbPTqPESJZcEjkJhZW3Q847JoiinPBRfGnHW1qwxwF4OmDCLcC+UUpcdgludnHLHxcBl5xiqAE5q9NsnEhSX4+\\\\\\\/OBG4Gn2brE9ClUG1oRQLtX9MNecS0Ihw7u9NEaDQXtv1pMq7Doz7rLn1PwFNghxdEAh7wI1w4bRImapxbHfX9huHk+V3BydYecgCtFvlH8OhH8PweUMcVEzAvGHnAOQkrwlgkMZjPIueqDRq5sbzV5d0ibwFNBzz\\\\\\\/h4EDRpKXIjE9j\\\\\\\/+tYisldR+6ftaij1HteAjAVbF5f\\\\\\\/j\\\\\\\/2SAgJ8Ze55pRicDtnM1Ef04O3pFA4yTLa6xGmU6lwk7XX052YOcV\\\\\\\/CLT\\\\\\\/+fYY5skZwtwJN56xnGjwPErVwFN7XhRSjPPs\\\\\\\/T9ltWgcz3jJbh+kkyTCOAS2e+kB8oFuVexur4QDCTwzPkUfLBvb82aTe4A01Mz3nYnmOABsWVOBYS4VFSXmSEWGeF8zXMwtKGkr0YETSumvdyI9zP10IPASy+PZAQvj+vi\\\\\\\/ENRCrfuNdMNJjTy6EJ0gbidrQQh39BC65O13CMFeY1pbmrXOJCn8w47bqrIPgU8OoOdehHBJ+Y12cD6vZ7Ro4FVWy+inHSwMovZ4wRdl7mfcr12yKUv8BRtjJnqQNgYwGeWoasMJtu5fAJHkoy+G1veMaELrVmCSPcjaF9oVdhMMxjinlyqd62mw+Qdy6Do6ZGnrXR9CWqXVtTALBmA4vxTiVR106jQm2QlKMTbanHMiDxOrQt0lvJlMXM7dVm\\\\\\\/s0XvEpTQeR+fCKdPP6wqSw9CNeYoiJlKk0bYn0c3LVv1G5c5dZ8smZR9KJExBr4P\\\\\\\/aqHqbTiFhugeHM+5hW+IRS05kPrJxOvKqw9vZGEeunonnkBGx\\\\\\\/R5AiDl2fyR4vXI0q1ouuSSB1b\\\\\\\/jS+uR+j1UeGeF13LY2F\\\\\\\/JNmKleuhMb1kYD12Ssj65SHBUMCPsIKtUGn5QG0Pzyer4sbVWG5lGevU+nMqrdSmrn2BTZ4mG\\\\\\\/s8BebAFyuN\\\\\\\/J3Et0LYiVr57+Z0TJUBiMzBpLgLu7bE09sP0A2BL8kEOsxuYGmnZ2YjPNHfrFvufwlk8pUyLJWbMT3RFOicTEBRYAdKWb26DNgpzfNIf9hfq7UZn\\\\\\\/qXCyaxnlWbtMV5pHleztexUt8ZhEeIcNNc2Wk4nrukrUWt1SFazY1mtgaZYrfF8\\\\\\\/ylaGBEsh7toJeiK2SUbPOTcX57aikrTzZKDA3mYA\\\\\\\/mmUICkfCEUYwcc+Pojzb9IP8MnogUIvL2NcikTjy4ahMhR2c8C8ebQxaiAdElT2eoC2l9KptFRjmggn09gdItkOsdZH+9VF9pa8RWK6g4vrCEh0jQPbMHWeJJkVDZV4JV9KfY2BzGsOBfMno26SlezYfxh\\\\\\\/E49c6WzSKvCNhuKPhLXAsTvD7oRSg3DSWgrhD6URbsvLxmj4fLrINmjyDchJZMyNIWnoeFmB\\\\\\\/f+IPVRvnl9Wx9jBMkPWA2jFoGm9ZpcRslV3UlRTbEEI9XQ0M9QdkkVLDDaWyQGl8iedz2ADEIZXmzUlJ4\\\\\\\/vuLVQXxZR9ANbixV640aYMhCUGUumXDtk0et7vixbhJvwy2IsRSqaVjyVc2nLUdUYgaE+WYo7\\\\\\\/C8I\\\\\\\/GM0RJDB1\\\\\\\/pgvnIbRlURTSc2fgtwxKXl9Og71cCxWAK6qw\\\\\\\/SQCAEGvCWW98gEiE3u6Yg8c0eEixcOFy6KkNl22ZS7+9vJ\\\\\\\/\\\\\\\/69CN57m9EbfXgN\\\\\\\/BVLbLueTlVhNgS26Ry7nnl8H8\\\\\\\/u+u0jcx92d52vS7foUaihh00GPJYyvSHCQZb\\\\\\\/RZ7Dfq\\\\\\\/KYpCAdA6PLMOTQweJB16KsYBmWQqzInRQWEkJiVssPG9rCAZ\\\\\\\/YW\\\\\\\/C5uNnYNF76QxrhdJM167C32Mw2HlqE\\\\\\\/WDBn7WXYbHU0EXRDxmWHLFOqmcaou+leIUA\\\\\\\/hGsaeywmclTYM30ajJ5y+cwKaMExwknr8RFi4WgbCyWIQF\\\\\\\/T9e04ZiBQLSKRW7IAQGyb5LFrcu4up6BPfo\\\\\\\/GK3Cndq2MwoYBeHUa6Ko5Byc\\\\\\\/v9uC1etmLhVjTJztrR9M+8vzuQqJVsp0\\\\\\\/Yy7yeqZwQA2x2UDZY9jj4PDekwj3GTgq16jnopxm6ceBb0SHdpszHPG+iy8B2UsWOto8VogPWTQHLEGVkt7QTxaPoXQ\\\\\\\/Jt0MGQ5tbne3PlszTJ2s3XaKJZYcSOzEyJssXflTjnaJ0\\\\\\\/ZrLsqXFvPpdCaIs4foKASt1h8+k\\\\\\\/m2RnUayADO3e8TtrF40r61dqecbdaMQsucD7waq7jHlKUzSkqUHvtaRItNxeQqw2JGV9bzG1pYRAiUMQSDPw\\\\\\\/h34Te9d9XLrY2e1RFP5KiX9yWIqibWquP3brb67zeIMB3AZ209IbPoPCRhw77p7CXhvoWZgVVIhkpQS1Q2kYeEAkkRtVg6+P7QSjxRMznIHkzoFaAMje18eORVpbv28pYlx0I5nlJNvfkR1vm1xR8P+Wt6I0jDfGT3gxEl5qtcm6K2VjrVXWPRL0TOU\\\\\\\/55NXohW5rYoVfv24eUqw+\\\\\\\/Lphb\\\\\\\/uezMRbQEaMCWSOgmMekqxO7qD5jLZf\\\\\\\/LJwsdUQp0rqqiAR7z4J27duxkQjvStWgD64XtWYZPWXfVsBWdajRiti9L01f6ux80oygxkbJE5Pm5h7ppgdy7MAVDFlN32VnYYTYgg1oq87iv98HE48m+wAFy699G9xg2LqkB+W6MqTAHSqh1EpC3oc6qlGyBO4Vue65P87dOABgYE8FdTgDtWgxXgsWJmHhMwyuY4z8pic8mQuYHZAOyiAwhNtlAeTIIJnMskOay3SMOG6yxxqkaYbDKKN5KF1QHGyDCPC0cJ1xv\\\\\\\/n2a0XgwwYNVRWbAmg2b\\\\\\\/pQVPhvVhhnOKp8yCiKNmwHnyu9fT3j6wCvH4KxshAdrVGV8zMH4N+nz1vyWyWbb7dThAN66bzSYLmQySC7OXSER9I06cuIqgCeB6qj2TV+W09bxXqCQS++L19RRemZEbXPW+UUCxgVnQqqQK2b11wFKP8j790iIVxVPEvpKjxOLpbUDzGwbFBeaDb\\\\\\\/gEZVxpAwIev6YLuKUVtxga2A1S8tbL0WrQAIQ0AW+qv5Jnao0sJ98Dp8zTXZfXMWP\\\\\\\/hWRB8DhjCCKEwpSkeaGZ67RbfDepwbOqwDCIuHGpkF7rXH2KWPchscMtz1QG5cMShhGplUBn\\\\\\\/Bd0DzU+RcMlzv39eXVvYAShJICX98+pcLm6JTGMBRO5hEz7\\\\\\\/vxgO2FoNsZeDXLX1pOA3t3khiyUXl4P\\\\\\\/v3D2FacUhvfZorNKA18WXQPWyjIagaXEEFnOJOAyN3FuJTUe0XOD37XPBLTrBhy4dVoC6vgIq4lzlJY7fvjrN7yfWfzlQtwP3hU3myertdm\\\\\\\/zXF95+XTizs9b4xRtqeYtvCn+OLsyOpT5XY3Qfb2P9lXllM27RTrw+iBCtNTYOl1Cs9roUZk9TbxHV+fORnY3D35O\\\\\\\/Y9lpswxRQuKNmSj5caK4LNdEtd79DiG3uqtKN\\\\\\\/9d71CI3Exz9NiYUrBYo8UpMadqWCdLr+GH8tkcZeFdTVz\\\\\\\/Y299Wx6AgyX68alVOg0B60LKKHmD6xlgxT4qe\\\\\\\/5BphG3vnx\\\\\\\/Sg81lMdutW1lCY\\\\\\\/ZU\\\\\\\/0RL7WP1mpjvYbL3GD2Xdy3l6LVAAWZypuYQY\\\\\\\/yZ+ZS\\\\\\\/ljS+rY0M0oxeR9mbNEeEN3WZ+fgKt99Paac1hXuUVz7JBq1Ut1TvPC1OQmAhm0gYh6wZVbw4pagrlMLI8gUpruiacqdCKBB5Z\\\\\\\/pfnLkNhIBK5GADsh0ElLEnwJZUQK9Nmc+GW3RUVGNqYnkFB3+XUXogcEN0c7ZGfBNyeJBhzRA3k\\\\\\\/QTqSZAcVfVZpdtXa3fmkSZ\\\\\\\/JWNiUNEi5AcThPZZlc28ix5MOrOaYMftzFo+f7J9OJ\\\\\\\/SE2lue1\\\\\\\/egItVKD84abfKiBSvK8TcWZnJ2X0W4iMdJTmdc03mgc4I\\\\\\\/S0dhE\\\\\\\/C8SKJK8tHDtWu1kMd7o6mDakIOhSqgPwN2CMO3pFMS\\\\\\\/SM+6a2U3KX9rEXQkoTloBxCSQRrdsQW2v\\\\\\\/zeQ5jrmWG9Kgr2NNeggM7TRYCmE3UnCMagX06mZyOv0mktg7TAjW5cf5HwIZvOxYrgVWPI+nc6q8BThdBBH35Wjx9EIXNTVywGKdB6De1Ve4m6PrSrQazW\\\\\\\/PTxgKHHximCRjbhqA+5Y6r7V\\\\\\\/LXO\\\\\\\/ZrMjjOSWYxgvbACFn4xsc9xPmDvOzvkz2yafUecGykSqOjyTr361z\\\\\\\/znXxoRxRBEGLo6aZBrRD0zONpgC5MwA6H5UGY0Uz2T5dPoQJHEHX8p7OOBVQ\\\\\\\/6OP3cwl0n7SlQ2MjSZ2L1HeduEp2hOc7m61wRsALBzlnA73U4K0oyKb3tiH6BmTzzWHXvGim+WtdV4LN+aFzlj8XLMzJQwV0D+pRm5mTLC6FkkVDHH+B4nyGYAH1JuRQHIxsjdQK4nPWAXKPVkWfWXxW\\\\\\\/FmALrqSE1Y44rL75I0VgagkFYE3aKvddxO6DAXZyp+rr4GJl+bvSPdgXRGnF6qEfrJiWQkqroyLbf68pS0VOmrxHqKe\\\\\\\/LzMmoA\\\\\\\/KSLBLvQxK7wa6PhG4ykbvbT4zLLalEfsH\\\\\\\/FE53WNBT4pHwCE4SCSQYZ\\\\\\\/y1O0\\\\\\\/ECggLIDZ9ju6vNKEf0n3P1q\\\\\\\/QQTHdWWiZuPMEKh07qvc0mKQuq9QpKqqBIyIG38t2r3aut6rwd0kTbmC+KsUQ\\\\\\\/9pw\\\\\\\/7bBpLqgvr3tHE3gTtP6YU7BD\\\\\\\/E3nBRE5wThCAiO+dMr0ftihsVf2puVEkE02xtlYVMDUnJU0oRMcSkiymtAfjpOjh2T39u+XZiQfVHqEct10ZpStZ0k7aA9ZP2ZgRfgAJc+jrJGkGhNAeXrC6zZcdAjN6lH6HkLF4UIXRmE1rvrYAe6CjEqBRc3Iapdtjzx8tZrkzpEwwPLisy29INL+qz6j4LseEq+z3DGayFNKDlAShkNDrzw8Ja8KaQaeQQgU5zTxC9FZ0sQYnmWPdsELgja+eAD6O+\\\\\\\/\\\\\\\/AhYXPlUu1jMDSL2rTNUcx82q1rFKhCooHNfI93WenmVBkZdnivLnMKqZgN0n9KI9GQBMDPuytZa1Y0LYZNAG833rKC\\\\\\\/r6ZPfNYpFp6KTA9S5uyfrvuat6ScncKDSqM83fslbBYp5Z6u+EiFZHc0n2WAlVcRls21Q7Fyf\\\\\\\/SMDnrUYo79ol9KF9TT4Vrpu\\\\\\\/XvGtqEJLv51Pv9J4ZhbBc9tbOgzb\\\\\\\/SOvjWH4WVezEcLN4Is0MGC\\\\\\\/oSbPpNwo8nqqz0hp9VnQPONK03FLQ2ll2Tezo3PE3UwFcvyZ7FW\\\\\\\/1goyycV1D2bK4wLpF5jVhoid+\\\\\\\/T\\\\\\\/i326FO8F\\\\\\\/fsqeT\\\\\\\/n76SZi\\\\\\\/lyiPpCwydSBsPz4mkyQwIzPneDDPSG\\\\\\\/cfsC0OlbLhykGlf1gZigpIO8mAyXrFpPZ2SYcU108y1pIzBeqjXCozemYto0+pwXoCMT570LFHAVgZwoz2Qpdp0SsedkPkbZAK8O2e4ItCTRzy+v0RPfxEbaiQ8A8EjbzmBD1sbb09PDMo01SWo00Mt0cKGSfnw2km8eeWxBOHtDA03wqsqUn6gchewpkitUfoXgY6\\\\\\\/88C6A+8kD45PlVstBi4gykNoQ1U4jqjiNvBRT\\\\\\\/qGi\\\\\\\/hOaO3e1Y6\\\\\\\/FBgIEdHl4VTvDv+Rp4NL+nUBlVrVI5BFTGXFMwNhLhBT7Q+4FavzXw2m2jJmwVso88e\\\\\\\/5JOhuvfz0a4X4vmxs+DtrVj0X73+r3TzlP5hpcCFyGqZmXKIY4sW7Kea7DlvMu0JdMw1IpVSRfpdh\\\\\\\/xUxpXip4xB3k6qfi+napGOsTRPCafuvJveIAG4Q98wBJJLIROuIJ6BgIj6+233Rr7Rc9QNLjHzvOXZM3MWpkCCm4kGmjp8vdccT94YlIQIZjflZZ9u1HLA2m4whxEDngQyzghnEnG\\\\\\\/4QqTj0i6DNyN7s82gX1Hho0iGKqNP7Cd+090cE1vgufaUlDJ03l+NNrgiggFQ\\\\\\\/LDAuR30rBn1HNyGHG4jT+8KnSxCd0kiVXXZ5c5zbu4HGduLSvRuQgsCwmAorpPTlSggS2FxiLMPlMzbBCiWWmI+HeeCg73s3xGdw\\\\\\\/vjGwscVWLHj2HZ8WVlufbyHXDIU31+lwjtmLbhIjRZKxzJQFaCgF3bEsNhmHqHGID9R8bCuf\\\\\\\/z6muVhWLNegJWy7sAI9esa6+fbr03Z5Xmik6G0G1KT\\\\\\\/Oc8erF1aTTpaqa8XHVGYDdEKV\\\\\\\/B7+6BZsJWLKN3ybzX3wP1UrvJoYL1wQ2wwWXZNh0PMIJUj3r7knSJ1tLHRVSSbVJS0+f7pDxsmhsyenSTKAvsLhkXklhOtG+RR5CTHvFQvAqg1b4GlISGdPzPSgYG2eQxOzBPQBJ37b7PlIhOwpWJUagw\\\\\\\/5DLHoVErZ4G2Sjv4DTMYOmHnuxbDtIfxjxnveZdL9aI0c3hHPcKvSDKx9njmgMOTPF3fWVFXT6E6ISZH8NO1iZZ6IhJU3gJrHbqFgcFT2pqQviRQaaZrz8FsGfu9lY3cTnIvs1ES1tZnmcw8ITc3XigxsupjmwOkXIptPxexKWbs3Trj77fkPQ8oNsRwrR5H6HnAAOgavVkKm6jNO6gp29\\\\\\\/8vdxpVHl6p3gAqv3H211wuQ+KNUn0EWOa8vZoHE8DYBLWkW4W79Luzboj9DRr2y9o4JYcXtsNEI9w699xKRKUUqxwlRRkoGMnxS8GlJ9vQM9OHnnZJafqnt+jUkZmoDpoSQW+K4VKHghFBgiNsoXv8gzvZO8y4Tm6IYM48ZgRrNDjw4g8OYz4vkOQmSa28c+osjIOw470rJJLuG5ty\\\\\\\/eHyNvtqUvTYd2Rxtx9ip6uEHEPqHEzte5blZva\\\\\\\/rxukkUcqRNnBo+Zk28QTC0RDETCjvqHvNuLyFRBfHqRpwTF83Jmc0okFtAzEP1H8MLx7ooyMatAOJri+i1tgvhhNcf70Kj1oH75cSzajdaFvuhcMwY40mF9b9BtcSBH6ZRhGfSX0ClMdQME24hwigVWppTIuwU0jvdAeYiSaVM1en5VAbgXVsgu4ACX2b4xzJkeJFP5XYp2VCLDh9kLgy9ORc7k1VZCe873qSVOpuCvKKa5z71w8N3zltJ0Z9jdXKfBr1X073UPzTFUCPNSKFOjxIVomEeVN413ovaKtszi2b84f7NxtkmR5srCjJvp8v52KG5ega0pz4AEaC72ov+GHn4i968QVTKlz4vyoE+\\\\\\\/F0iIcSHvRQOaBRyjW7ZHYDwa+lUOMMs+zL6VNU0bQ40kVOG\\\\\\\/qdkou0nw0nA6ehKBSEcFShp8XAJwEW10k56AV8HWeSlHqPIaqa6thgrGY6LKw+ndbLUngFQahqP2fuNjvPm\\\\\\\/6y4\\\\\\\/rAbK1pd7S55uPi8dUF\\\\\\\/L+AwQoG+tbac5XYONujJ0nkaL4nO7oae85yIdsWeSAb7KiVK\\\\\\\/LEiGpOYdp+IdlM2GPNjSzW4e3MHIl9Wa6R0Xy7WdQodX\\\\\\\/s3zpy9F54kesEgJGPRYXxgEArD8QbrFLvxOPA3rfVTGRU17z+eyThd0gDdNgZ7TCpLgz\\\\\\\/o3rFRiIxCFApUVLzI4nLm9yRu0o\\\\\\\/SQqB796L8dbHf9ZyKznEuNmx8LFinVSNycw9Gjpq8iSdYzPtE3gG\\\\\\\/HpYUPedwx3tDUZDY0m9Xv+BAh+mh7kGbw\\\\\\\/SZev6Y+ifKoYXAaLcjnAGhaf799WlgLuRZ7Bj6gQs\\\\\\\/Y2VxPFYNie8QYls3wIBfyC+AZgGqY6Thy3i6IZH+GmpeG2G7dckRNUzc1UYy3HUJ\\\\\\\/45Yi1E3V+P2vVjunCcj1CIk3u9mKPhEvj5M8zHkLhvtWxNE0hKXBoioAGDuDE1EjuAjSrMQCBbobKFPJCm+dzIi69Igy4S2qbfchI4eXqIcgt8m9qW183XHTBuNUpNa3sy+VYmUHPftZ2edMVoayBH3n+HeYaMJsjQsv3y1ozZlgvJeaDXNNdulN1bmuJ8VB4pWY9yDGovi2ud6tdxK4oYo1Jy8zWTeAaHAUyIa32fWJdCgOIB\\\\\\\/OMaH3CyEl\\\\\\\/\\\\\\\/1P3QWFoOY0tZdyy76a+Y0TZdeW7DIoFjnS2bYkcxM6xdwmSOPKzdTUQz+AmEXtMust6zvCMpnWAWWQsU2Ib2NCqOSp90iaa1AFx5XudnxkiNVFERxX4smdRfdGHKiP+eOOaJ\\\\\\\/ziW7Lr683hquO\\\\\\\/c0eHPvFYfRW7MIWFcWjp84PfcAAp9KBlX5vO2ZNBg\\\\\\\/L7r0qk\\\\\\\/yBmv5Nv+qgWhE3zZ7hBD9Ow7FgdYsaBPg2dXJe7lrtBevwiabRJLww0b+dtxPqnkGCMCONL8vaI05v13OxSPsfgxkTC3XWTTiHrhmbSItl8CZOhNqrsM9hwXFjX18U0+e8LQB94IsXU5xc7m3d0zTcN9KuRr+VRSkc2IU4tB4EGvt2zOkBQz2MARdaqYG5bawoUaMZ8AZ86T3gLRgcVl1eCMEwTCOHqS8MZ6ecoNNEkbidgkblgYGdD\\\\\\\/jEXEMCfaBveYpHVBUJ6Uo\\\\\\\/TnjHmPjGqWr67xn\\\\\\\/oX2fxjgwLFHTMjzIx4jko99gSrlGeLVTHDy\\\\\\\/GgimZJECrtKFKh5WBZibmIis3\\\\\\\/ukYl4qK8o9Z1DSxdLS8SQY7N+IPa3aec\\\\\\\/tEwRkn4fyfLHWz9LD91lQntySuvmVmreaEoYgyuS7dQ6W1oqGoaJnXGp6wCmRgtO8zKtg5vgLKDRLw0ipW6gPV09nf5pcWubKfCz26l1kafyMZG5ZTK9ufzKOUa8jqy3Cth4TCKlG+P0XNsVPYKJhl6BqZz\\\\\\\/\\\\\\\/z\\\\\\\/2\\\\\\\/G94TNhBBS7zsSsL2AgzEe+WHsMB+rJ6CYUfub5UbNmRwd063rFRc+R6TldH6ibq7kXFaUUzGyswK3u8TrCODyes7gbDCDZYSpJNQBZOGRafY9ex2hzsvUh3ZIfBUbennGFaCGmk0pZaJmwOWD9j3bkIWBqgoxx1TLAe7HyjgLafgH4ppSpR8jEl6L5Q9e9+blW3xhcO5URAOxH8dcjvffsU\\\\\\\/UKy8kTugnygzFei63qmzjOAZKQFXsmiyPN9ZW+fVri6kjohNc3prtc4KBMmv+UTV\\\\\\\/qTq+fVRDiM0PNd92jVeE1TPJqIR7fXL6W24Q0FmgbTqlMG1CZi8B\\\\\\\/3uWJDdmvWzVL3ee968rHfFflHuG1TAZ5Iqy7tmqw1+GoHWR+bDhJMxyVKqkey8a36wtCVOEo2ky1PrEVSELBB3kNqq\\\\\\\/VmooG3QPagWTMfX8ijH6If1vT1y4wr1GoHOQGrjtR9K3YJfHmrOFbiffUnNjcrjXtWAje+qJRB3i2h10NYmfTjwMyWMSbFmTir12i0DlM2opFaPNvOGQaXaM67wHkBtgG9XP1WIqkf\\\\\\\/\\\\\\\/6VOfqDTfSxBRULB4NHponyWQGaxkn8IMaqk5yG+bshDZbUDxzWPKGnTQVjbcYymjGvUtgKbHM+xc\\\\\\\/XBYBaBuT78A8IkJYBw4uWI6BUwBIIwnsL\\\\\\\/Dg505Ng1wvNPoivdJJ844f96RfkGboHmgpJslI6P2TpHWDCqXoX+I3pG0TYKE\\\\\\\/mtBij1DgV6nNsMlLi8ugeN6nQ7iwpYE4ktO8eNTMKIJWu9DItta6CJbuRnrZfIgTK+BiK6jy3pWYDwFGrkplSHDdB+41d+NQBlNZa6dcAj+0e6UAUG7mGhQmppW0XrQIOC7\\\\\\\/P0dxKyJUBa7gIM\\\\\\\/sStzuX01uKDrHm8aoe0gB3rR\\\\\\\/gKXFoTX4GbVID9wAA6+HvzpH5vT1SGEuUH2Ik56W5uSQGKKwIg\\\\\\\/Iu2A0k2AvsQFOQP3jBBPX\\\\\\\/jUB4fUvFoI1CGYeUMx\\\\\\\/MrsNmFXLp0opcL0eyumshe\\\\\\\/N4vZJFcNegAANV0C7sEMo6\\\\\\\/H\\\\\\\/jCF0SnOsjC2PhnTqf3uhNiytPHAIgvI5K4X9hOflrQRzDEznOg\\\\\\\/hxlcrH7DsB3BMmFJrJBogoYRahMO9RvBSkh3aCyKATr4dccIZfDxiJ8P9Wc5YONIR6E\\\\\\\/pS3YuHZTdqbR7AEwl66s+BtRKELf7PauRbOMaxlZm9gr\\\\\\\/JToMy7TuByjfaEK22OfdUVilHolNDYbFglY6H0sCEFXbZaSWrP2R4Wil\\\\\\\/0zu7njslM3dn1wMcr1POyZznSE6T1uuq9LGD4Th6ujeula7r5o6nbwdj641pzQh8ufDcf8QVN0YhOOxTUT99MNmxrKFH7hQQalmb8jHhdul1mV+ohcruoSTI3w3gxl8CjBBdcD0FM34aiFBqON5q7II1JHpyhCpj0mSUxXc20PFdLtBtNhn64GKd33ngCsCsC+Z5wFB9iUOTxhZVeKGmPofAKmDE5Ewh6lsWqJLxasHnQfeaVbZaHqSK5GbYC\\\\\\\/gHzkbW5qbj5lnxyymd4uq5KS+7QvEZT3cmgJaSTmgPox4ATDooYKmy2b\\\\\\\/pMSPiywPiLFk92LVvBJ9fo5\\\\\\\/oQawGVLLvZaz2GgXC3\\\\\\\/O+IJ8DWZ+ojqZYLz1JaVTYBlDjNY55QaJQiEj910o79g5M3CQJK1N\\\\\\\/0VuvZuNLa6qN7agNa53OZfFgssd6MsYL377u0FrtPwen18U\\\\\\\/R4BhYd4lB+oZD8Rqy\\\\\\\/NKelm32upsMEi+ddaRxMO6pdOZ93VyJO0BDm3yV1AlEzBYXQaV4yHBfcB14v24lgHqCcYTyh4eA+BTMgyUhIFx+YDUmKVm77r4yYn3LZAeQy717CWnKlypq7AARzueO9XM4l9jUT0SFtT825jISPTpGVvp2kFMC3YtUe3qRvW3JtlfVMBKjsGLkYGikSDx+z9Wkxejt46J9\\\\\\\/3lzR7iVwsTwbE0zi0giuoXyFH6mR3DLX\\\\\\\/eVki\\\\\\\/S3QI67J06ijP9lXlKilx2NH0Wxcnjwy9jukPovt7h9GA2UkM1VniOcRw\\\\\\\/nmKluQJIRHJ07\\\\\\\/bssXnl3xJ4tydZo0xgKWQ9ShYjUG6ddxaUEnIkrJ\\\\\\\/M4qQj+JClEMsPt1zwoW9TqCN79+7SqUCYAZfxthPelOwIPl9wg\\\\\\\/Udyu90AvPlxW3ucxFLCJiguLy81ans6Lw1U0kw+2GuJrgPukcxKhYJ58FvcLwAJ\\\\\\\/HH8VnDjYTLKtx3s4OWeLyk77vxqXUDl9PQzYRdjWPyCC4V5Y5Y03gHMA4T6C7loHl77FeLdV0mFPDM3720CEmO\\\\\\\/dZ4VH6pdqcdsTwsxNWWne2woMQqoF0Yn9M1\\\\\\\/vrMhhpRziHleUD3vpW2f9q4LRAUcKRaVaPBHGTeqfU3FLWqQ+vaFlRVM1rn92638BjgG\\\\\\\/+PgeTuRgaDJgVD3R30YP7RYNjCFsbteGRUsKrl+p+dmEjpEX7EaJrMXEQP8nQP2mp4AObjbBYVelhcoucMF26aUaeP+MbTx6tJZWvYx+ndZcHMD9svQm9CCtOJMrvd7PKbpsnspA4T8P5ANptg5ymU5lL2v\\\\\\\/e98Hl6U+VMMLCtE2R27QeBEuxofDQrfp4VqjTXuBTxGhxSORi+H7ll1\\\\\\\/hiyjvyWwBGSROdD+ObiEWlLQ8n0sEIoZ\\\\\\\/UwZI5ls4pzNSErdloPvw7ygplmIX3NCyKjX03JAOsJEqc+ZOmbAmNOKiuaWlmIFAb16Om8bQHjh3fUwEEiLzSeXUmSdQD7IYhGTPtyL35mxu0zeEt5Y0eOOthbPwKtJBEHKcFV3Tg4wHnAMl5SngjrwzRXgF+eW\\\\\\\/R2IE2R8rYCLl9RvZEfCOtJqm4eL9ti3TDIU+09hbnlFy4hSncS7MuyjYDyTqkwZPjVOGgge0wmvRlU4us2Zit4\\\\\\\/EJgA+XXy4m61Y1yJ23imMf3wQUjh47qn5pF6lLH\\\\\\\/9Zh2lNjICLIMRC7c+ZJlKTLd4ir7hi9yAJzGXjd13bfkyJ+XFiAkGtlus3tC71k9W+zZE2Cg59lOjC+F+mdDtQsLbrCexkVSFYkoymEUsHjDLNxy6Qmjz789kZ6R1YyuqOSr3GLckmTLj6pTxBjfOKTwDoz\\\\\\\/IdJN6LfmqOjf2FSgipkNk7iIG9+IiZJdJOZcFKdZB4sACocCHiHbRBjXVymVobUykks9Su\\\\\\\/qFRg53Av6L879b\\\\\\\/VhlAlvbmRL6o7GQSIHzpZt6qaHewBArkMTW6FFg+QQBCEUmPSleGi0BfvCIkksk3RuJkB+JjddPt6FWI6u6Uwfc8yXlR7V4d1UMTN7dJ6KrQ9xdltcrTaBhSyFILOarqbDJH3DFoYp2GDZBt7tzVMl0WF8FjogXREBwm6k5KWLaCuqaX914L3\\\\\\\/eaRBkJvrwlszUDWxBdzEzgQlrHS6ZOyVM+VeEyCXiBhStTYL4kfur\\\\\\\/tQMsTjeR+VTBGoGocKWw8IGNSeoBo3aHdvyjAyvZtv6yLyoZt7eyZPMU71fuQvp1SUWFV8\\\\\\\/scfpQMwzTDZjv5GI7eK7U0GtAmAhb\\\\\\\/4br9TZvtxuchQxAAOJYrqdkCu6FVxfCF41ZoptZC7yl0qUHHgB\\\\\\\/EM9If0rP3P4Gkg7qUjXTVN87X9pr\\\\\\\/SRt5YuzZRBnz9kgW7FCvbdt4F6UTjGt7zV8EsKGcPnIjTuf+pNJ0a9SKgxvoF7mpKQ7yCYNlJ9gtszOA+i+jgXZ+SAhRyWq\\\\\\\/N1REiKjj4UCT8ZlO+Zc5viXwDipksCfWLi8VZ7BQlGxg16YpwnATbz+P+haaQqjH7YsjN3hum20WVuB81YVtLdcRYw5cBOeHhmxclvPcrzI0dUk7JWvKm56GX8pbvGZzNmvhUKH1iZOvon2nRTr0y5qgnfUhQ2u+1HsqlRTYkWvic49CBljZeYGD70tyy8Ks1yO2adBvnJgnqY0Ad2qEJnLZ6LXKqVnXvMjZMDOI0XuTURk\\\\\\\/u5hb83iauwZbGpGQciCSp9jG8hfrWz5Tv8OQoX0XavcZM9g1tpjgyxnvCR1ZCvp0mBaBJnzVb\\\\\\\/za\\\\\\\/PssclIPAzpkCMnhQepwAlfl277Ea8YhhCvJI62WIv56WDKwYN65qcRskPhkdsNPxftnaY5rg\\\\\\\/Qs157Z2podEwipJWH7IbE9WlSm5QnD7AmJOvlu2j8bv8eKNtaduW8l7aekEO8RB58qX5\\\\\\\/om6NjjKZwJ0Zzzfu7bRMULxNQrJeFymM9HtHt8oXTkU7q+xcsI8hTBIHRAjo6k5RvAdmPuudq8CcqyyVtnR4fd9uHnXLthadWi7NVkQ1bdvhiYbkO1aXfeTeLVw7ipPxttcuj1BXGb2O+fNorSa03rm4cYa9ng+ioE+Pwdrh1hOrcRvQRwV0VSMvbUwF3npC7a0lGwqBMcGeGllJDvLdiLPpywSoqqI6OXsnDCiXAdZX2Qcmk9En+et4SDzvqFxxllXnmtHvICn1nL5fSShrlDRvcVnL\\\\\\\/w7NqL02Trnwx42brmd3+Etw0y4g2u4PggouHJ\\\\\\\/2EsPxbBCoPq97v4CWkThcIuTPsJCdZijz\\\\\\\/Eq9sgh4IwI5W1Alyysg5ONN\\\\\\\/3ZAFlTnmwYRUPCcf0He2VCPmhL2kHrGiOmIJ\\\\\\\/XujfnyXZJNbC3X66vUHzTfiK8YjyzAEM1ZZqdcfOI4lInvE1EHcqSS2Nd8DwpjfBWuV0Fe3gTntPUpWcaBx1np85f\\\\\\\/sdxuE6LXQvONl3IltFFngziyX1ilMX7hrgSkTqo\\\\\\\/1lda8NdDTZS1GlsGKGhlZtuFrBxHS3wkUfS8gnPNBglG+yD8pQkkAqmr0j1\\\\\\\/fIVtsegcoT9cBfumKJIe1wuV3la24q9IO6pv+e19UhFUTHbwwQTEMu4XmcOHEsy\\\\\\\/++tFCJZQUXtkuhh+JpfT5GiRTwm8McXJQngnxyro80d9Ieagl\\\\\\\/6P98yfoMDqDxnldh0WKAvQKTqraSEgPKja1l3UdUxT6EgzbE2jgPeSGN3MITEu4JFOFBthQOyk79haWr4w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kqXF8EsOGfjCyxw0IRexMA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220b4f740ef2f5035a3a1378baeb74f9a1e0c353eba6ebba45c2aa5df7aa565de2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ylAdK0yk3CYF\\\\\\\/RF1byrsdxUQr7xXir+WHuOdyvMqK5uuoUD8jE6FmSRCjjDhzPOfZu39vBeUQHWVm8++PtD\\\\\\\/4Yx2Qo+B\\\\\\\/1urRQEqn7Cxjr3\\\\\\\/nMuyBO6sZJNNT5\\\\\\\/KFcU\\\\\\\/60Pj3hSdV7i1UXfQfnJODU6qPgGQLlNT8HygEp8pqMP2FmA5E90HPPvy06ruqmRSNqL+eWhYvxhqYFxk0jTRLt6qb\\\\\\\/o\\\\\\\/FadgqGJepYiLgKHox5ZZpEq222H8XXeLycSEHdIjPWsM4ZKEOxGs6TiG2KKDqf0ZygsvgNuyu3CqqjHJSWdkv0jiBavfhYp3uRjlIqWjCNTebW4fixdVDCitSayCoKXHDijXAixHPc7GCNPn6sWh6YRPm+qs16DsddXE5fcl7w9yKnedDKMXLXCvW45Bgr2eZW7wjUBlhohwm5H\\\\\\\/GB0FQKVKfphRSj\\\\\\\/B\\\\\\\/5As080BrTN5nyGnqk\\\\\\\/xITP50+GfgaiFy6BlP8I91joCcKpnkVKpY7bAjxX9jMTWBz4Rgt4iUn3vQoYqmoSE7+y1WbrGFHDXuI9HEHE6zKqOD+3H7ihB27XJ0jrT+4i2+8pkxGjJEXsFAgLvsZMneV8vFyVjg4yHB9e5kJMrpaQKS1gIVmr1BGgwwvDhqXqQPbReSPmTMVrwyGFBKN7EAXeeverHQGUO7usDCr8VS3zEl2oYc44Vue2\\\\\\\/1GG5m+BE8XvfDP1RBKhOwtDLDK\\\\\\\/dkChJjD1Ebpx1cXzf75ZuvGQD4h0Bsqf\\\\\\\/NthXecXURfU1ZMJ67J6yZqBp5qEu\\\\\\\/eunqcSbbq2S8iQjgUtbBJyHyVET540P3\\\\\\\/vo9ZxP\\\\\\\/djOdItUx8xkygnORMnLuF2TC6TX25W4gnFwERCz8+EchO8MLXnw+qCT0H6bL+GL8LlNjGdXKMLzOj6QtLlx6r7d9SIrn4dxNg+s1uEby2si1NY7KoVnKkq3xc3E7Vqf7PT5ZFTqLyUANGMS+TBbDwMVUTSDxZ+JzAJdbgiK2VN\\\\\\\/8eW1sYC84+BhCEu4UFgecoD2tLxCmW\\\\\\\/6Dvihuao3irz6Vvu8\\\\\\\/cMsVLhfDlw7sdZVSgIUUpLZ4e6bhMyF11oxdrISfvKr4NeVNYUo9nQVgrnyXZwuN26npN56oo88OjuaAZ2mSnUyOK400qg+5x9QPuDZ5DFMijhEE415NyjvP9K8a3Tyqg3I8MuI8m4EBkd8ZKIRSQwGQ3satoRBf6s0mFK\\\\\\\/K6h\\\\\\\/5aTGE5l+9hYI9IRY45rvQ8Ey1YxdGPFso3vhQ5TW\\\\\\\/G8kdpxwuVOf4Nmp9Jg\\\\\\\/FAOWR\\\\\\\/YRlZBZxvuoTa7pIAkoOfEFsFPqtwZ9uQNwg5rfQbBAx4X6rgq31OJnkLekB7HLS2Jb\\\\\\\/O79K9TU5\\\\\\\/GnC5WzMAVvCvZ9fJ78ThCh5xa1Rmjn0mUy7IboX\\\\\\\/hZZpzRFfzCcmE0OqrMAI5vW9m4Tf+r07IcqXS1FO\\\\\\\/ZBFQ4mco1YlS8nSJOiBobHAGd8JvYx2YNNXhHlz6WrS0FiqD8ddgwqRUxt7NhIMA13JbfS+ORqTwh08sE6z0\\\\\\\/RkWLzJ3RMyxe9\\\\\\\/iyDJNDMAbyFf65Cuvo4E9sbqKzI8OfGD9\\\\\\\/0201l3yZHDaZE7uG6K0fizQ1yqG36RDPUl5oBPjK5gqMEChmWYSyCtUi\\\\\\\/EFdarJhc9bTknihxdp7RsoXuQ\\\\\\\/fJiEVUkQ481tF9S4Oq8QJMEO+xCm0rJK3usBgv79K3j9pZ313ybOvSOPZI1Hz9yyN09bMkhCLKdYiWwTGXnD3IsgS9aJzxc2Ketyv4UVPcAdQI3fVLlRajsimcIQzMJL7DWbk6dkKwnCokt+S0v39WV\\\\\\\/BSbM9ufSwiWIDwUidctWLtcc2tKgSEsK6hZX+7h8LDzrNJYUY51w\\\\\\\/oCaINE1CDtd\\\\\\\/SUGwlvIPXpgtZC35b3T2N0kb\\\\\\\/rsx7XC8SPIZ6nXxlA3sLu+ryLssL2Tf\\\\\\\/mQc+M0Iu2J\\\\\\\/cA7hUpb\\\\\\\/BqcoVfuHYccvH\\\\\\\/6rUFCKfQpsiC+zH6jK1ftwMmQVf\\\\\\\/ZKqLDxFUCp23KG\\\\\\\/d5sxw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mjaLzwG1p4nwGqyh7dECYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e5c40b4df372a9a764753582a50b1a0d99298f9d526107f44b3cd57ae5b13018\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459929 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1205" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LcNyWGl4IvPq1UTzl5sB1xKMxpATSiHvybVz8f1aeDA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1205" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YOv0LXP5TiuIJx6XkASng5lvSCRzihlANJ8NWGDLFSc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1193" title="Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1217" title="Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459929" id="node459929" data-pisa="ggg;4/7/1205" data-pisa-master="ggg;g3.114.011783" data-apath="/ggg/4/7/1205.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Small Effect Mutations in <em>Saccharomyces cerevisiae</em>: Impacts of Experimental Design and Mutational Properties</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fabien Duveau</span>, <span class="highwire-citation-author" data-delta="1">Brian P. H. Metzger</span>, <span class="highwire-citation-author" data-delta="2">Jonathan D. Gruber</span>, <span class="highwire-citation-author" data-delta="3">Katya Mack</span>, <span class="highwire-citation-author" data-delta="4">Natasha Sood</span>, <span class="highwire-citation-author" data-delta="5">Tiffany E. Brooks</span> and <span class="highwire-citation-author" data-delta="6">Patricia J. Wittkopp</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1205-1216;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011783 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459929" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fabien Duveau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabien%2BDuveau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duveau%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabien%2BDuveau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fduveau@umich.edu" class="" data-icon-position="" data-hide-link-title="0">fduveau@umich.edu</a>
<a href="mailto:wittkopp@umich.edu" class="" data-icon-position="" data-hide-link-title="0">wittkopp@umich.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian P. H. Metzger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BH.%2BMetzger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metzger%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BP.%2BH.%2BMetzger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan D. Gruber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BD.%2BGruber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gruber%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BD.%2BGruber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katya Mack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katya%2BMack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mack%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatya%2BMack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Natasha Sood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natasha%2BSood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sood%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatasha%2BSood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tiffany E. Brooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiffany%2BE.%2BBrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brooks%20TE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiffany%2BE.%2BBrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia J. Wittkopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, & Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Computational Medicine and Bioinformatics, University of Michigan Medical School, Ann Arbor, Michigan 48109-2218</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BJ.%2BWittkopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittkopp%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BJ.%2BWittkopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fduveau@umich.edu" class="" data-icon-position="" data-hide-link-title="0">fduveau@umich.edu</a>
<a href="mailto:wittkopp@umich.edu" class="" data-icon-position="" data-hide-link-title="0">wittkopp@umich.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/7/1205" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459929" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459929/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1205.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459929" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459929/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1205.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459929" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459929/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1205.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459929" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459929/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/7/1205.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459929" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459929/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genetic variants identified by mapping are biased toward large phenotypic effects because of methodologic challenges for detecting genetic variants with small phenotypic effects. Recently, bulk segregant analysis combined with next-generation sequencing (BSA-seq) was shown to be a powerful and cost-effective way to map small effect variants in natural populations. Here, we examine the power of BSA-seq for efficiently mapping small effect mutations isolated from a mutagenesis screen. Specifically, we determined the impact of segregant population size, intensity of phenotypic selection to collect segregants, number of mitotic generations between meiosis and sequencing, and average sequencing depth on power for mapping mutations with a range of effects on the phenotypic mean and standard deviation as well as relative fitness. We then used BSA-seq to map the mutations responsible for three ethyl methanesulfonate−induced mutant phenotypes in <em>Saccharomyces cerevisiae</em>. These mutants display small quantitative variation in the mean expression of a fluorescent reporter gene (−3%, +7%, and +10%). Using a genetic background with increased meiosis rate, a reliable mating type marker, and fluorescence-activated cell sorting to efficiently score large segregating populations and isolate cells with extreme phenotypes, we successfully mapped and functionally confirmed a single point mutation responsible for the mutant phenotype in all three cases. Our simulations and experimental data show that the effects of a causative site not only on the mean phenotype, but also on its standard deviation and relative fitness should be considered when mapping genetic variants in microorganisms such as yeast that require population growth steps for BSA-seq.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bulk-segregant-analysis" class="hw-term hw-article-keyword hw-article-keyword-bulk-segregant-analysis" rel="nofollow">Bulk Segregant Analysis</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">Next Generation Sequencing</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/tdh3" class="hw-term hw-article-keyword hw-article-keyword-tdh3" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">TDH3</em></a></li><li class="kwd"><a href="/keyword/mutagenesis-screen" class="hw-term hw-article-keyword hw-article-keyword-mutagenesis-screen" rel="nofollow">Mutagenesis screen</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/faster-mt" class="hw-term hw-article-keyword hw-article-keyword-faster-mt" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">FASTER MT</em></a></li></ul><p id="p-4">Characterizing the causal relationships between genotypes and phenotypes is a major goal of modern genetics. Bulk segregant analysis (BSA), in which two phenotypically distinct subpopulations (bulks) of recombinant progeny (segregants) are isolated from a genetic cross and genotyped, is one way to achieve this goal (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Michelmore <em>et al.</em> 1991</a>). With this method, regions of the genome contributing to the phenotypic difference between the two pools of segregants are identified because causative alleles (and linked loci) occur at different frequencies in the two bulks. BSA is a cost-effective approach to mapping because genotypes are determined only for the two bulk samples rather than each of the individual recombinants. The recent development of high-throughput sequencing, which can be used to determine allele frequencies for nearly all sites in the genome in each phenotypic pool simultaneously, has made BSA particularly effective for mapping polymorphisms in organisms with small genomes such as yeast (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ehrenreich <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pomraning <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liti and Louis 2012</a>; Wilkening <em>et al.</em> 2013). Even small differences in allele frequency between bulks can be detected with this genotyping-by-sequencing approach (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Parts <em>et al.</em> 2011</a>), allowing detection of small effect variants. Because BSA requires sorting large numbers of individuals based on their phenotype, it is particularly well suited to the analysis of traits that can easily be selected or scored in the laboratory, such as growth in different environments (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wenger <em>et al.</em> 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ehrenreich <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Swinnen <em>et al.</em> 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2013</a>) or expression of a fluorescent reporter gene (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2014</a>).</p><p id="p-5">BSA can be used to identify sites contributing to natural variation (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Parts <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Granek <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Van Leeuwen <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bastide <em>et al.</em> 2013</a>) or mutant phenotypes isolated from genetic screens (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wicks <em>et al.</em> 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brauer <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Xia <em>et al.</em> 2010</a>). Experimental design and statistical properties of BSA coupled with high-throughput sequencing (BSA-seq) for mapping quantitative trait loci (QTL) have been examined in detail (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Edwards and Gifford 2012</a>); however, methods for mapping mutations using BSA-seq after a mutagenesis screen have received less theoretical attention (but see <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Birkeland <em>et al.</em> 2010</a>). Compared with natural variation, the density of polymorphic sites is usually much lower after a mutagenesis screen, and the mutations are more likely to have effects on fitness. As a result, optimal experimental design and statistical power are expected to be different for BSA-seq when analyzing natural variation and mutant genotypes created by random mutagenesis in the laboratory. For example, sequencing information from linked segregating sites can be combined when mapping natural variation to increase the power of detection (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Edwards and Gifford 2012</a>), but this is usually not possible with the lower genetic diversity present after mutagenesis. In such cases, sequencing coverage sufficient for statistical analysis must be recovered from the causative site itself.</p><p id="p-6">Here, we examine the influence of experimental design and mutational properties on the mapping success of BSA-seq when the density of segregating sites is low, with the goal of providing a general framework for large-scale mapping of small effect mutations after a mutagenesis screen. We describe the effect on mapping sensitivity of four experimental parameters (population size, intensity of phenotypic selection, number of mitotic generations between meiosis and sequencing, and sequencing depth) as well as three mutation properties (effect on mean phenotype, effect on standard deviation of the phenotype, and effect on fitness) that can potentially bias genotype frequencies in the segregant bulks. Previous studies modeling BSA-seq for QTL mapping primarily considered the effects of a genetic variant on the mean phenotype for the trait of interest (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Parts <em>et al.</em> 2011</a>).</p><p id="p-7">We used the results from this computational modeling to design a bulk segregant mapping experiment suitable for identifying mutations in yeast causing small changes in expression of a yellow fluorescent protein (YFP) reporter controlled by the <em>Saccharomyces cerevisiae</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> promoter. These mutations were previously isolated from a low-dose mutagenesis screen in which each haploid mutant recovered was predicted to have, on average, 47 new mutations with only one affecting fluorescence of the reporter gene (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> 2012</a>). Our simulations indicated that isolating very large pools of haploid segregants (&gt;10<sup>5</sup> cells) with extreme fluorescence phenotypes was essential for mapping success given the biological properties of the mutant strains. To achieve this, we developed an experimental system for efficiently collecting phenotypically divergent cells from a population of haploid segregants that uses (i) a genetic background with a greater meiosis rate than the typical laboratory strain (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Deutschbauer and Davis 2005</a>), (ii) a robust and tractable mating type marker to efficiently isolate stable haploid bulks (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2012</a>), and (iii) fluorescence-activated cell sorting (FACS) for high-throughput phenotyping and selection of individuals with extreme fluorescence levels. Genetic variants responsible for changes in mean YFP expression as small as 3% relative to the wild-type genotype were then successfully mapped despite their significant impact on fitness and confirmed using allele replacement, showing that BSA-seq is a powerful method for identifying small effect mutations after a mutagenesis screen.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Power analyses</h3><p id="p-8">To identify parameters that influenced and maximized power for BSA-seq, we modeled the effects of sequencing depth, phenotypic selection cutoff for choosing bulks, total population size, and generations of growth after meiosis as functions of a causal mutation’s effect on mean expression, standard deviation of expression, and fitness. Because of the low density of mutations expected in mutants isolated from mutagenesis screens, we assumed only one causal mutation influenced the phenotype of interest in each mutant. We also assumed that noncausal mutations were in linkage equilibrium with the causal site. Finally, for simplicity, we assumed that noncausal mutations did not affect fitness. Violating this final assumption should not affect allele frequencies for the causal site as long as it is not linked to these noncausal mutations.</p><p id="p-9">Power analyses were performed in two steps. First, a deterministic model was used to calculate the expected mutant and reference allele frequencies for the causal site in both phenotypically high and phenotypically low bulks prior to sequencing (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/011783SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS1.pdf">Figure S1</a>). Then, using these expected frequencies, sampling was used to account for variation introduced by library preparation, sequencing depth, and allele frequency from sequencing. For each set of parameters, we simulated 1000 sets of reference and mutant allele read counts for both the high and low bulks. These modeling and simulation steps were all performed in R (v 2.14.1; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Development Core Team 2013</a>) and are described fully in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FileS1.pdf">File S1</a> with R code provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FileS2.R">File S2</a>.</p></div><div id="sec-3" class="subsection"><h3>Strains used for mapping</h3><p id="p-10">Haploid mutant strains from <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> (2012)</a> with <em>trans</em>-regulatory effects on expression of a fluorescent reporter gene in <em>S. cerevisiae</em> were used in this study. These mutants were isolated from a low-dose ethyl methanesulfonate (EMS) mutagenesis of a BY4724 (<em>MAT</em><strong>a</strong> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319"><em>lys2</em></a><em>Δ0</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747"><em>ura3</em></a><em>Δ0</em>) derivative called YPW1 with a <em>P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP</em> reporter gene inserted on chromosome I at position 199,270 (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> 2012</a>). Based on Canavanine resistance assays, each strain was estimated to contain 47 ± 17 (99% confidence interval) EMS-induced point mutations, with exactly one mutation expected to affect YFP expression in 98.7% of the strains (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> 2012</a>). A red fluorescent protein (RFP) marker was inserted at the <em>MAT</em><strong>a</strong> locus in each of these mutant strains before crossing to the mapping strain described below to avoid diploid contamination when sorting haploid segregant progeny (FASTER MT approach; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2012</a>). The genetic basis of altered fluorescence was mapped for the YPW89, YPW94, and YPW102 mutants from <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> (2012)</a>, which showed +10%, +7%, and −3% changes in mean fluorescence relative to the nonmutagenized reference strain, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>A). These mutants also reduced the standard deviation of fluorescence phenotypes for each strain (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>B). The mapping strain (<em>MATα <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met17</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0 P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003276">RME1</a>(ins-308A)</em>) that each of these mutants was crossed to was obtained from a series of crosses involving YPW1 (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0 P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP</em>), BY4722 (<em>MATα <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0</em>), BY4730 (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met17</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0</em>), and a YAD373 derivative (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003276"><em>RME1</em></a><em>(ins-308A)</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001391"><em>TAO3</em></a><em>(E1493Q)</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005029"><em>MKT1</em></a><em>(D30G)</em>) from <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Deutschbauer and Davis (2005)</a>. The dominant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003276"><em>RME1</em></a><em>(ins-308A)</em> allele increased sporulation frequency of heterozygous diploids relative to the starting strain, which facilitated isolating large numbers of spores.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464621/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464621/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Fluorescence phenotypes and selection coefficients for the three mutants analyzed</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Obtaining bulk segregant populations</h3><p id="p-15">For each mutant, the segregating pools were obtained as follows. First, the mutant strain was crossed to the mapping strain on YPD plate and diploid colonies were isolated on SC-Lys-Met medium. A single diploid colony was then inoculated to 2 mL of GNA (5% D-glucose, 3% Difco nutrient broth, 1% yeast extract) and grown to saturation at 30°. Then, 0.2 mL of this culture was diluted into 1.8 mL of GNA and grown 4 hr to log-phase. Next, cells were washed twice in 1 mL of H<sub>2</sub>O, resuspended in 30 μL of H<sub>2</sub>O and spotted on KAc plate to induce sporulation. Sporulation plates were incubated at room temperature without Parafilm sealing to allow oxygenation. Sufficient sporulation (&gt;5%) was usually observed after 4 d, at which point random spores were isolated.</p><p id="p-16">For each strain, the whole yeast spot (about 5 × 10<sup>7</sup> cells and tetrads) was resuspended in 1 mL of H<sub>2</sub>O in a microcentrifuge tube, washed once in 1 mL of H<sub>2</sub>O and incubated in 200 μL of zymolyase 20T (1 mg/mL) for 50 min on a rotor at room temperature. Once ascus walls were digested, samples were washed in 1 mL of H<sub>2</sub>O and resuspended in 100 μL of H<sub>2</sub>O. To enrich for spores relative to vegetative cells, each tube was vortexed vigorously for 2 min, which resulted in spores visibly sticking to the tube wall; diploid cells remained in suspension. The supernatant was then carefully aspirated off and the spores were washed once more with 1 mL of H<sub>2</sub>O. To release spores from tube walls, 1 mL of Triton-X (0.02%) was added to the empty tubes on ice and samples were briefly sonicated at low power (10 sec at power 3.5 using a Sonic Dismembrator Model 100 from Fisher Scientific). Spore suspensions with nonaggregated cells were observed under a microscope after this step, with less than 5% vegetative contaminants (diploid carryover). Spores suspension needed to be grown to log-phase to express the fluorescence phenotype. To avoid mating during growth, approximately 3 × 10<sup>5</sup> <em>MATα</em> spores were sorted using a BD FACSAria II based on the absence of RFP expression. These cells were then centrifuged, resuspended in 2 mL of YPD (to clear off traces of Triton-X) and incubated at 30° for 14 hr.</p><p id="p-17">Next, log-phase cultures (≈2 × 10<sup>7</sup> cells/mL) were washed in water and resuspended in 3 mL of SC-Arg (media that has lower autofluorescence than YPD). Cells were acclimated to their medium for 3 hr at 30° before sorting through the FACSAria II instrument. Cytometric gating was set up using FACSDiva software to sort 2 × 10<sup>5</sup> cells from both tails (&lt;2−5 and &gt;95−98 percentiles) of the YFP distribution in two separate tubes. Only events of intermediate size were sorted based on FSC.A, the area of forward scatter signal (10% tails of FSC.A distribution were discarded) and special care was made to keep the FSC.A median (a proxy for median cell size) the same in the two bulks of cells. Finally, low-fluorescent and high-fluorescent bulks were resuspended in 3 mL of YPD, grown to saturation at 30° and genomic DNA was extracted using a Gentra Puregene Yeast/Bact. Kit from QIAGEN. From this, genomic DNA libraries were prepared using a modified version of a previously described approach (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rohland and Reich 2012</a>) as explained in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FileS1.pdf">File S1</a>. Then, 100-bp paired-end sequencing was performed on Illumina HiSeq2000 platform at the University of Michigan Sequencing Core Facility.</p></div><div id="sec-5" class="subsection"><h3>Analysis of Illumina sequencing data</h3><p id="p-18">For each sample, FASTQ files containing all paired-end reads data were generated with CASAVA v1.8.2 software. Before alignment, low-quality ends were trimmed from reads using <em>sickle</em> v1.2 (<a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>) with default settings (−q 20 –l 20). Trimmed reads were then aligned to the S288c reference genome (<a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>, R64 release from 03-Feb-2011) with <em>P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP</em> inserted on chromosome I using Bowtie2 v2.1.0 (bowtie2 −p 2 −x ref.fasta −1 SampleX.R1.fastq −2 SampleX.R2.fastq −I 0 -X 900 −S SampleX.sam –t; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Langmead and Salzberg 2012</a>). Next, bamUtil v1.0.9 was used to clip overlaps between mate reads that could bias our estimation of allele frequencies (<em>bam clipOverlap</em>–in SampleX.sam–out SampleX.clipped.sam–stats–readName). MPILEUP files containing base calls from overlapping reads at each genomic position were generated with SAMtools v0.1.19 (<em>samtools view</em> –q 10 –bS; <em>samtools sort</em>; <em>samtools mpileup</em> –BD –f; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>).</p><p id="p-19">For each mutant, two different MPILEUP files were generated: one was used to call a set of high confidence single-nucleotide polymorphisms (SNPs) using VarScan v2.3.6 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Koboldt <em>et al.</em> 2009</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2012</a>; <a href="http://varscan.sourceforge.net">http://varscan.sourceforge.net</a>), and the other was used to estimate allele frequencies for a broader set of genomic positions using Popoolation2 v1.201 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kofler <em>et al.</em> 2011</a>). The first MPILEUP file was obtained from a BAM file containing sequencing data for the mapping strain and another BAM file merging reads for the low and high fluorescence bulks. SNPs were called using VarScan <em>somatic</em> and <em>somaticFilter</em> commands with the mapping strain considered as “normal” and the merged F<sub>1</sub> segregant bulks as “tumor” (<em>somatic</em>–min-coverage 5–min-var-frequation 0.01; <em>somaticFilter</em>–min-coverage 5–min-reads2 3–min-strands2 2–min-var-frequation 0.05). Filtering out sites with strong strand bias was critical to remove false positive variants. In parallel, the second MPILEUP file was generated from two separate BAM files containing reads data for the low and high bulks. Allele frequencies at variable sites were computed using Popoolation2 (<em>mpileup2sync.jar</em>–min-qual 20–threads 2; s<em>np-frequency-diff.pl</em>–min-count 10–min-coverage 10–max-coverage 500).</p><p id="p-20">Finally, G tests were computed for each variable sites using <em>likelihood.test()</em> function from the R package Deducer (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fellows 2012</a>). A large fraction of variable sites identified through Popoolation2 were absent from the set of high confidence SNPs obtained with VarScan and were removed before plotting <em>P</em>-values of G-tests. Mutant alleles at these sites were considered as mapping errors as they usually occurred at low frequency, at the end of reads or only in one strand direction.</p></div><div id="sec-6" class="subsection"><h3>Single-site mutagenesis</h3><p id="p-21">Targeted mutagenesis was performed using the <em>delitto perfetto</em> approach (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Storici and Resnick 2006</a>) to introduce each candidate mutation into the genetic background of the mutant ancestor (BY4724 with <em>P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP</em> inserted on chromosome I). The <em>CORE-UK</em> cassette (<em>CO</em>unterselectable <em>RE</em>porter <em>KlURA3</em> and <em>kanMX4</em>) was first inserted at the candidate mutation position in each target gene (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a>, or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a>) by homologous recombination. Then, coding sequence harboring the candidate mutation was amplified by polymerase chain reaction from the mutant strain (YPW89, YPW94, or YPW102) and introduced in place of the <em>CORE-UK</em> cassette. Sanger sequencing of the target gene confirmed allele replacement.</p></div><div id="sec-7" class="subsection"><h3>Expression level of fluorescent reporter for <em>P<sub>TDH3</sub></em> activity</h3><p id="p-22">Expression level of the YFP fluorescent reporter protein was quantified in the wild-type, EMS-induced mutants, and single-site mutants using flow cytometry. Eight replicates of each strain and the nonfluorescent BY4724 were arrayed at random positions in a 96-well format on YPG agar rectangular plates (OmniTrays). In addition, 20 replicates of the wild-type fluorescent strain were arrayed at specific positions to control for plate position effects. After growth on YPG, arrayed strains were transferred into 0.5 mL of YPD in a 96 deep-well plate using a V&amp;P Scientific Pin Tool and grown for 20 hr at 30° to saturation. Cells were maintained in suspension by the addition of a 3-mm glass bead in each well and constant shaking at 220 rpm. Immediately before flow cytometry, 20 μL of each culture was diluted into 0.5 mL of SC-Arg in another 96-well plate. Fluorescence was then quantified for an average of 10<sup>4</sup> events per sample using a HyperCyt Autosampler (IntelliCyt Corp) coupled to a BD Accuri C6 Flow Cytometer (533/30nm optical filter used for YFP acquisition).</p><p id="p-23">Flow cytometry data were analyzed with custom R scripts. First, a set of cytometric events considered as single fluorescing cells was filtered for each sample using Bioconductor <em>flowCore</em> and <em>flowClust</em> packages. An average of 5 × 10<sup>3</sup> events per sample were retained after this step. Next, a fluorescence phenotype was calculated for each single event corresponding to log(FL1.A)<sup>2</sup>/log(FSC.A)<sup>3</sup>, which corrected for the correlation between fluorescence level and cell size. FL1.A and FSC.A are the area of the YFP fluorescence signal and forward scatter signal (proxy for cell size), respectively. The phenotype of a given sample corresponds to the median phenotype of all filtered events. Finally, we tested for plate position effects by fitting a linear model to the fluorescence data obtained from the 20 control samples. We included in this model the effects of each plate, row, column and half-plate (plates were scored one half at a time). A stepwise approach based on Akaike information criterion for model selection was conducted using the <em>step()</em> function in R and showed that a simple model including only the half-plate effect explained 74.9% of the fluorescence variation across the 20 control samples. Therefore, the effect of each half-plate was extracted from the linear model and subtracted from all samples occurring on the same half-plate. To calculate mean expression relative to wild type, the fluorescence phenotype of the reference strain YPW1 was subtracted from the fluorescence phenotype of each tested strain and this was divided by the difference in fluorescence phenotype between YPW1 and the non-fluorescent control BY4724.</p></div><div id="sec-8" class="subsection"><h3>Fitness assay</h3><p id="p-24">Competition experiments were performed to estimate fitness of the parental strain YPW1; the EMS mutants YPW89, YPW94, and YPW102; and the three single-site mutants with mutation in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a>. Fitness was also measured for five mutants randomly chosen from the 179 <em>trans</em>-regulatory mutants described in <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> (2012)</a> to estimate the range of selection coefficients for this set of mutants (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>C). In all cases, fitness was measured relative to a common reference strain expressing GFP (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0 P<sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub>-GFP</em>) as follows. Each strain was grown for 24 hr to saturation in 5 mL of YPD medium, and then cultures were diluted to 6 × 10<sup>6</sup> cells/mL based on optical density measurement. A total of 500 μL of each YFP yeast culture was mixed thoroughly with 500 μL of GFP culture and 9 mL of YPD. Next, 500-μL samples of each mix were randomly arrayed in 8 wells of a first 96 deep-well plate and 10-μL samples were diluted to 10<sup>4</sup> cells/mL into the same eight random wells of a second 96 deep-well plate containing 490 μL of YPD per well. The first plate was used to estimate the proportion of YFP and GFP cells at the beginning of the competition assay (T<sub>0</sub>) using flow cytometry. The second plate was grown for 24 hr at 30° with constant shaking (220 rpm with glass beads to keep cells in suspension). Then, 20 μL of each culture was diluted in 500 μL of YPD in a clean 96-well plate, and the proportion of YFP and GFP cells was estimated at the end of the competition assay (T<sub>1</sub>) using flow cytometry. Fluorescence was recorded for at least 2 × 10<sup>4</sup> events per sample using a HyperCyt Autosampler (IntelliCyt Corp) coupled to a BD Accuri C6 Flow Cytometer (585/40 nm optical filter used for YFP acquisition and 533/30 nm optical filter used for GFP acquisition). Despite considerable overlap of the YFP and GFP signal detected through the 533/30 nm filter, control experiments showed that cells expressing YFP or GFP could be distinguished using this filter combination. Custom R scripts were used to filter out spurious events from flow data and to compute the proportion of YFP and GFP cells for each sample. A selection coefficient was then calculated for each replicate using the following formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-1.gif"/></noscript></span></span>where <em>YFP</em><sub>0</sub> and <em>GFP</em><sub>0</sub> are the observed number of cells expressing YFP and GFP at time T<sub>0</sub> and <em>YFP</em><sub>1</sub> and <em>GFP</em><sub>1</sub> are analogous numbers of cells at time T1 for each sample. The experiment was started from an average density of 10<sup>4</sup> cells/mL and stopped at 6 × 10<sup>7</sup> cells/mL, yielding an approximate number of generations <em>g</em> = 12.55. The selection coefficient obtained by competing mutants expressing YFP against a GFP reference strain can be explained by the mutant genetic background or by YFP expression itself. The fitness effect of the YFP marker was quantified by competing the parental strain YPW1 to the GFP reference strain. From this, the selection coefficient associated with the mutant background was calculated as follows, assuming the effect of EMS-induced mutations and YFP on fitness were additive:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-2.gif"/></noscript></span>
</span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-3.gif"/></noscript></span></span> is the selection coefficient for the mutant background, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7/1205/embed/mml-math-4.gif"/></noscript></span></span> is the selection coefficient for the YPW1 reference strain expressing YFP measured in competition to the GFP strain and <em>s</em> is the selection coefficient for the mutant strain expressing YFP measured in competition with the same GFP reference strain.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Optimizing experimental design for mapping mutations of small effect</h3><p id="p-25">To determine how power for detecting different types of mutations varies with mutational properties and experimental parameters, we developed a flexible simulation that models mapping via BSA-seq computationally. This power analysis was parameterized for mapping mutations affecting fluorescence in <em>S. cerevisiae</em> that can be efficiently scored in large populations of recombinant cells using FACS but can be adapted to other biological systems with different genome sizes, mutation effects, or attainable population sizes. Variant discovery and allele-frequency estimation were modeled assuming whole genome sequencing of DNA extracted from bulks of recombinants with high- and low-fluorescence phenotypes. We included two phases of cell growth during which competition among genotypes can affect allele frequencies. The first growth phase between meiosis and bulk selection was required to express the fluorescence phenotype and the second between bulk selection and DNA extraction to increase the amount of genomic DNA for sequencing (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Experimentally controllable parameters included in our simulation were population size, intensity of phenotypic selection, sequencing depth, and number of generations between spore isolation and DNA extraction. Innate biological properties of a mutation included were the effects on the mean and standard deviation of the fluorescence phenotype as well as the effect on fitness, which changes the frequency of the mutation during growth. We used the range of biological properties observed for <em>trans</em>-regulatory mutants affecting fluorescence of a reporter gene isolated in <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> 2012</a> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>) to identify experimental parameters that provide a high probability of detecting a causal mutation for a minimal cost.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimentally controllable parameters affect statistical power for detecting a significant difference in the frequency of a causal mutation between bulks. (A) An overview of the modeled BSA-seq experiment is shown, with the four experimental parameters we allowed to vary (population size, generations of growth, cutoff for bulk selection, and average coverage of sequencing) indicated. Power is shown for various population sizes (B&#x2212;D), generations of growth (E&#x2212;G), bulk selection cutoffs (H&#x2212;J), and average sequencing coverages (K&#x2212;M), for a range of effects of the causal mutation on mean expression (B, E, H, K), standard deviation of expression (C, F, I, L), and fitness (measured in terms of the selection coefficient) (D, G, J, M). In all plots, the dashed line indicates 90% power. Gray shaded regions represent 90% confidence intervals of the mean effect and standard deviation of the fluorescence phenotypes observed in a recent set of trans-regulatory mutants (Gruber et al. 2012, see Figure S2, A and B). The 90% confidence interval for selection coefficients was inferred from fitness assays performed on 8 mutants (see Materials and Methods and Figure S2C). In all analyses, only the indicated parameters were allowed to vary; all other experimentally controllable parameters were fixed at values ultimately used in our mapping experiment (sequencing depth = 100, population size = 107, cutoff percent = 5%, generations = 20), and mutational parameters were fixed at values representative of the mutants used for mapping (mean effect = 5%, standard deviation = 100%, selection coefficient = 0.03)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886662508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Experimentally controllable parameters affect statistical power for detecting a significant difference in the frequency of a causal mutation between bulks. (A) An overview of the modeled BSA-seq experiment is shown, with the four experimental parameters we allowed to vary (population size, generations of growth, cutoff for bulk selection, and average coverage of sequencing) indicated. Power is shown for various population sizes (B&#x2212;D), generations of growth (E&#x2212;G), bulk selection cutoffs (H&#x2212;J), and average sequencing coverages (K&#x2212;M), for a range of effects of the causal mutation on mean expression (B, E, H, K), standard deviation of expression (C, F, I, L), and fitness (measured in terms of the selection coefficient) (D, G, J, M). In all plots, the dashed line indicates 90% power. Gray shaded regions represent 90% confidence intervals of the mean effect and standard deviation of the fluorescence phenotypes observed in a recent set of &lt;em&gt;trans&lt;/em&gt;-regulatory mutants (Gruber &lt;em&gt;et al.&lt;/em&gt; 2012, see Figure S2, A and B). The 90% confidence interval for selection coefficients was inferred from fitness assays performed on 8 mutants (see &lt;em&gt;Materials and Methods&lt;/em&gt; and Figure S2C). In all analyses, only the indicated parameters were allowed to vary; all other experimentally controllable parameters were fixed at values ultimately used in our mapping experiment (sequencing depth = 100, population size = 10&lt;sup&gt;7&lt;/sup&gt;, cutoff percent = 5%, generations = 20), and mutational parameters were fixed at values representative of the mutants used for mapping (mean effect = 5%, standard deviation = 100%, selection coefficient = 0.03).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1205/F1.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/7/1205/F1.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464491" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Experimentally controllable parameters affect statistical power for detecting a significant difference in the frequency of a causal mutation between bulks. (A) An overview of the modeled BSA-seq experiment is shown, with the four experimental parameters we allowed to vary (population size, generations of growth, cutoff for bulk selection, and average coverage of sequencing) indicated. Power is shown for various population sizes (B−D), generations of growth (E−G), bulk selection cutoffs (H−J), and average sequencing coverages (K−M), for a range of effects of the causal mutation on mean expression (B, E, H, K), standard deviation of expression (C, F, I, L), and fitness (measured in terms of the selection coefficient) (D, G, J, M). In all plots, the dashed line indicates 90% power. Gray shaded regions represent 90% confidence intervals of the mean effect and standard deviation of the fluorescence phenotypes observed in a recent set of <em>trans</em>-regulatory mutants (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> 2012</a>, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>, A and B). The 90% confidence interval for selection coefficients was inferred from fitness assays performed on 8 mutants (see <em>Materials and Methods</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>C). In all analyses, only the indicated parameters were allowed to vary; all other experimentally controllable parameters were fixed at values ultimately used in our mapping experiment (sequencing depth = 100, population size = 10<sup>7</sup>, cutoff percent = 5%, generations = 20), and mutational parameters were fixed at values representative of the mutants used for mapping (mean effect = 5%, standard deviation = 100%, selection coefficient = 0.03).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We found that the size of the population from which the bulks were selected did not have a large impact on power as long as the population size was at least an order of magnitude higher than sequencing depth (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B</a>−D). When segregant pools were smaller than sequencing depth, a high rate of false-positive results was observed (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS3.pdf">Figure S3</a>). For the range of biological parameters observed among mutants isolated in <a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> (2012)</a> (shaded areas in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B</a>−M), we found that 20 generations or less of mitotic growth provided sufficient power to detect most causal mutations (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, E</a>−G). Increasing this number to 50 generations allowed competition among genotypes to strongly bias allele frequencies, causing a loss of power to detect mutations with selection coefficients greater than 0.05 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1G</a>). Our analyses also suggested that selecting cells from the 10% or smaller tails of the fluorescence distribution is sufficient to achieve high detection power under most conditions. Generally, sampling cells from more extreme tails by decreasing the fluorescence cutoff for selection increased power (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, H</a>−J); however, mutations causing very large increase or decrease in the standard deviation of the fluorescence phenotype were found to require a particular range of cutoff percentages to maximize power (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1I</a>).</p><p id="p-28">Unlike in our simulations, population size, generations of growth, and the fluorescence cutoff for bulk selection are inter-dependent in a real experiment. Starting with a larger population size, for instance, decreases the number of generations of growth required to obtain sufficient cells for analysis, which in turn decreases the impact of the selection coefficient of the mutation on mapping success. However, increasing the population size requires more time and money spent phenotyping individuals before bulk selection. Considering the ease of creating large populations of yeast and the high-throughput phenotyping possible when using FACS to measure fluorescence, we decided to fix the population size at 10<sup>7</sup> cells and the cutoff for bulk selection at 2% for the rest of this study, resulting in 2 × 10<sup>5</sup> cells in each bulk. This allowed us to keep the total number of generations to 20, with 10 generations of growth between spore isolation and bulk selection, and 10 generations between bulk selection and DNA extraction.</p><p id="p-29">Using these conditions, we investigated the impact of sequencing depth on mapping power. As expected, increasing the average genome coverage always improved the power to detect causal variants (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, K</a>−M), but this requires increased cost. Selecting the ideal sequencing coverage therefore depends on the properties of the mutation(s) that the researcher seeks to identify. For mutations similar to those isolated in <a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> (2012)</a>, we deemed 100× genome coverage the best compromise between power and cost. We note, however, that mutations with a mean effect of 5% or larger can be reliably detected with sequencing coverage as low as 25× (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1K</a>) as long as the mutation does not have a large impact on the phenotypic standard deviation (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1L</a>) or relative fitness (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1M</a>). Mutations with mean phenotypic effects smaller than 2% or with selection coefficients larger than 0.15 would likely require &gt;100× sequencing coverage in each bulk to be mapped.</p><p id="p-30">With the population size, generations of growth, intensity of phenotypic selection, and sequencing coverage fixed as described previously, we used our simulation to further investigate the relationship between power to detect a causative mutation and that mutation’s effects on mean expression (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), standard deviation of expression (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2E</a>), and relative fitness (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2I</a>). We found that mutations that change the mean phenotype at least 3%, cause a phenotypic standard deviation ranging from 75 to 150% of the wild type, and have a selection coefficient less than 0.1 should be detected with a power greater than 90% with this experimental design (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). This combination of effects on mean and standard deviation includes &gt;90% of the 179 <em>trans</em>-acting mutants isolated by Gruber <em>et al.</em> 2012 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>, A and B). It also includes five of the eight mutants for which we measured relative fitness (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>C). Fitness measurements for mutants isolated from a mutagenesis screen are expected to overestimate the fitness effects of a causative mutation, however, because mutations that do not affect the phenotype of interest can also affect fitness. Therefore, using the relative fitness of a mutant strain to determine the best experimental design is expected to underestimate the true power for mapping the causative mutation in that strain. Increasing the mean effect of a mutation always improved detection power (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, B, C, D, and G</a>), but a more complex relationship was observed between the effects of a mutation on the phenotypic standard deviation and selection coefficient. Specifically, when a mutation had a large effect on the standard deviation, more deleterious mutations could have greater detection power than less deleterious mutations under some conditions (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2, F and H</a>). This is because these parameters bias mutation frequency in the two bulks in opposite directions: increasing the selection coefficient lowers the mutation frequency in both bulks, while increasing the standard deviation raises it.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inherent properties of mutations affect statistical power to detect a difference in the frequency of a causal mutation between bulks. Power is shown for various mutation effects on mean (B, C), standard deviation (D, F), and relative fitness (G, H). Comparisons of hypothetical wild-type (red) and mutant (blue) populations with effects of a mutation on mean expression (A), standard deviation of expression (E), and relative fitness (I) are also shown. In all plots, the dashed line indicates 90% power. Gray shaded regions represent values of the mean effect, standard deviation, or selection coefficient of causal mutations observed in a recent set of expression mutants (see Figure S2). In all analyses, only the indicated parameters were allowed to vary; all others were fixed. These fixed values were: mean effect = 5%, standard deviation = 100%, selection coefficient = 0.03, sequencing depth = 100, population size = 107, cutoff percent = 5%, generations = 20." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886662508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inherent properties of mutations affect statistical power to detect a difference in the frequency of a causal mutation between bulks. Power is shown for various mutation effects on mean (B, C), standard deviation (D, F), and relative fitness (G, H). Comparisons of hypothetical wild-type (red) and mutant (blue) populations with effects of a mutation on mean expression (A), standard deviation of expression (E), and relative fitness (I) are also shown. In all plots, the dashed line indicates 90% power. Gray shaded regions represent values of the mean effect, standard deviation, or selection coefficient of causal mutations observed in a recent set of expression mutants (see Figure S2). In all analyses, only the indicated parameters were allowed to vary; all others were fixed. These fixed values were: mean effect = 5%, standard deviation = 100%, selection coefficient = 0.03, sequencing depth = 100, population size = 107, cutoff percent = 5%, generations = 20.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1205/F2.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/7/1205/F2.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464516" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Inherent properties of mutations affect statistical power to detect a difference in the frequency of a causal mutation between bulks. Power is shown for various mutation effects on mean (B, C), standard deviation (D, F), and relative fitness (G, H). Comparisons of hypothetical wild-type (red) and mutant (blue) populations with effects of a mutation on mean expression (A), standard deviation of expression (E), and relative fitness (I) are also shown. In all plots, the dashed line indicates 90% power. Gray shaded regions represent values of the mean effect, standard deviation, or selection coefficient of causal mutations observed in a recent set of expression mutants (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS2.pdf">Figure S2</a>). In all analyses, only the indicated parameters were allowed to vary; all others were fixed. These fixed values were: mean effect = 5%, standard deviation = 100%, selection coefficient = 0.03, sequencing depth = 100, population size = 107, cutoff percent = 5%, generations = 20.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Identifying single candidate mutations in <em>trans</em>-acting mutants</h3><p id="p-32">To empirically evaluate the BSA-seq approach using parameters selected based on the simulations described previously, we attempted to map mutations responsible for altered fluorescence in three <em>trans</em>-regulatory mutants carrying a YFP reporter protein under the control of the <em>S. cerevisiae</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> promoter. Assuming that a single causative mutation explains the phenotypic effects observed in each mutant (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), a mapping power &gt;97% is expected for bulks consisting of 2 × 10<sup>5</sup> cells sorted from the 2% tails of the fluorescence distribution, with 20 generations of growth and an average sequencing coverage of at least 75× (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-33">To efficiently obtain such large and stringently selected bulks of pure haploid segregants, we followed the protocol shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Each haploid mutant strain was mated to a common mapping strain and sporulation (meiosis) was induced in the resulting diploids. Including the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003276">RME1</a>(ins-308A)</em> allele (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Deutschbauer and Davis 2005</a>) in the mapping strain increased sporulation frequency from 2 to 20%, making it easier to isolate a large population of F<sub>1</sub> haploid segregants. To prevent mating between <em>MAT</em><strong>a</strong> and <em>MATα</em> haploids in the population of segregants, we sorted ~3 × 10<sup>5</sup> cells lacking expression of a RFP reporter gene that we had inserted at the <em>Mata2</em> locus in each <em>MAT</em><strong>a</strong> parent. More than 99.6% of cells lacking expression of this <em>FASTER MT</em> marker (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2012</a>) were confirmed to be <em>MATα</em> haploids. After 10 generations of growth to allow these <em>MATα</em> cells to robustly express their YFP fluorescence phenotype, high- and low-fluorescence bulks of ~2 × 10<sup>5</sup> cells each were isolated via FACS. Attention was paid during cell sorting to avoid introducing other phenotypic variation between the two bulks. For instance, no more than 1% variation in median cell size (FSC) was allowed between bulks (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS5.pdf">Figure S5</a>). After an additional 10 generations of growth, genomic DNA from the low- and high-fluorescing bulks was sequenced to at least 75× coverage (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) using 100-bp paired-end Illumina sequencing.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of experimental design for mapping small effect mutations affecting the expression of a fluorescent reporter in EMS-induced mutants. This approach is based on the isolation of a large number of random F1 segregant haploid cells, followed by high-throughput phenotypic selection using FACS, and estimation of allele frequencies genome-wide using next generation sequencing. Note the selection of haploid MAT&#x3B1; cells using expression of the RFP reporter linked to MAT&#x3B1; locus that is indicated with a red dot. Quantitative differences in the level of YFP expression are indicated by differences in the intensity of yellow background." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886662508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of experimental design for mapping small effect mutations affecting the expression of a fluorescent reporter in EMS-induced mutants. This approach is based on the isolation of a large number of random F&lt;sub&gt;1&lt;/sub&gt; segregant haploid cells, followed by high-throughput phenotypic selection using FACS, and estimation of allele frequencies genome-wide using next generation sequencing. Note the selection of haploid &lt;em&gt;MAT&#x3B1;&lt;/em&gt; cells using expression of the RFP reporter linked to &lt;em&gt;MAT&#x3B1;&lt;/em&gt; locus that is indicated with a red dot. Quantitative differences in the level of YFP expression are indicated by differences in the intensity of yellow background.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1205/F3.medium.gif" width="197" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/7/1205/F3.medium.gif" width="197" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464617" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Overview of experimental design for mapping small effect mutations affecting the expression of a fluorescent reporter in EMS-induced mutants. This approach is based on the isolation of a large number of random F<sub>1</sub> segregant haploid cells, followed by high-throughput phenotypic selection using FACS, and estimation of allele frequencies genome-wide using next generation sequencing. Note the selection of haploid <em>MATα</em> cells using expression of the RFP reporter linked to <em>MATα</em> locus that is indicated with a red dot. Quantitative differences in the level of YFP expression are indicated by differences in the intensity of yellow background.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464693/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464693/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average genome coverage and number of mutations detected from Illumina sequencing of the two segregant bulks for each of the three mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Sequencing data were analyzed using a pipeline with two main parallel steps (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). First, a set of high confidence variants was called for each mutant with VarScan (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Koboldt <em>et al.</em> 2012</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">2009</a>) based on genome sequencing data from the mapping strain and genomes of the segregant bulks, with reads from both bulks merged. The number of mutations identified in each mutant ranged from 33 to 77 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), which closely matches the number of mutations (30−64) predicted using Canavanine-resistance mutation rates in <a id="xref-ref-17-11" class="xref-bibr" href="#ref-17">Gruber <em>et al.</em> (2012)</a>. Most of these mutations (85–94%) were G:C to A:T transitions, as expected for EMS-induced mutations (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Allele frequencies were then estimated at every variable site for each bulk using Popoolation2 (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kofler <em>et al.</em> 2011</a>). These allele frequencies were strongly correlated (r = 0.983) with independent estimates determined by pyrosequencing (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FileS3.xls">File S3</a>). The number of reads containing reference and mutant alleles at high confidence polymorphic sites was compared between low and high fluorescence bulks using a two-sided G-test.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of Illumina sequencing data. A set of high confidence variants was called using the somatic command in VarScan (dark gray), with reads from the mapping strain treated as &#x201C;normal&#x201D; data and reads from merged bulks treated as &#x201C;tumor&#x201D; data. Allele frequencies were then estimated for these sites in the low fluorescence and high fluorescence bulks with Popoolation2 (light gray). Differences between these two bulks were assessed using G-tests." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886662508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of Illumina sequencing data. A set of high confidence variants was called using the &lt;em&gt;somatic&lt;/em&gt; command in VarScan (dark gray), with reads from the mapping strain treated as &#x201C;normal&#x201D; data and reads from merged bulks treated as &#x201C;tumor&#x201D; data. Allele frequencies were then estimated for these sites in the low fluorescence and high fluorescence bulks with Popoolation2 (light gray). Differences between these two bulks were assessed using G-tests.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1205/F4.medium.gif" width="440" height="361"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/7/1205/F4.medium.gif" width="440" height="361"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464501" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Analysis of Illumina sequencing data. A set of high confidence variants was called using the <em>somatic</em> command in VarScan (dark gray), with reads from the mapping strain treated as “normal” data and reads from merged bulks treated as “tumor” data. Allele frequencies were then estimated for these sites in the low fluorescence and high fluorescence bulks with Popoolation2 (light gray). Differences between these two bulks were assessed using G-tests.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">For each mutant, we observed a single, highly significant (<em>P</em> &lt; 0.001) association with YFP fluorescence level (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>); physically linked sites also showed significant associations with comparatively higher <em>P</em>-values (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). To determine the likelihood that similar associations would have been detected for other mutants, we examined the distribution of aligned sequence reads in more detail (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FileS4.pdf">File S4</a>). We found that ~3% of the genome had little-to-no sequencing coverage in each mutant (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS9.pdf">Figure S9</a>) and that this was due to difficulty obtaining and/or aligning sequence reads from these regions rather than stochastic fluctuations in coverage due to sampling (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS10.pdf">Figure S10</a>, A and B). We also found that reducing sequencing depth would have caused us to miss the significantly associated site in YPW89—the mutant with the largest effect on mean fluorescence of the <em>P<sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424">TDH3</a></sub>-YFP</em> reporter gene—because of its strong deleterious effect on fitness (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS10.pdf">Figure S10</a>, C and D).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BSA-seq clearly identified a single causative site in three trans-regulatory mutants affecting fluorescence of a reporter gene. Significance of the difference in allele frequency between low fluorescence and high fluorescence bulks is shown as the negative of logarithm of P-value from G-test for mutants YPW89 (A), YPW94 (B), and YPW102 (C). Each bar shows significance for an individual EMS-induced mutation with its genomic position represented on x-axis. Roman numerals indicate each of the 16 S. cerevisiae chromosomes. Insets in (A) and (B) are magnifications of chromosomes harboring causative sites and show linked mutations with significant effects. Horizontal dotted lines represent a significance threshold of &#x3B1; = 0.001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886662508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;BSA-seq clearly identified a single causative site in three &lt;em&gt;trans&lt;/em&gt;-regulatory mutants affecting fluorescence of a reporter gene. Significance of the difference in allele frequency between low fluorescence and high fluorescence bulks is shown as the negative of logarithm of &lt;em&gt;P&lt;/em&gt;-value from G-test for mutants YPW89 (A), YPW94 (B), and YPW102 (C). Each bar shows significance for an individual EMS-induced mutation with its genomic position represented on &lt;em&gt;x&lt;/em&gt;-axis. Roman numerals indicate each of the 16 &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosomes. Insets in (A) and (B) are magnifications of chromosomes harboring causative sites and show linked mutations with significant effects. Horizontal dotted lines represent a significance threshold of &#x3B1; = 0.001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1205/F5.medium.gif" width="247" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/7/1205/F5.medium.gif" width="247" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464503" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">BSA-seq clearly identified a single causative site in three <em>trans</em>-regulatory mutants affecting fluorescence of a reporter gene. Significance of the difference in allele frequency between low fluorescence and high fluorescence bulks is shown as the negative of logarithm of <em>P</em>-value from G-test for mutants YPW89 (A), YPW94 (B), and YPW102 (C). Each bar shows significance for an individual EMS-induced mutation with its genomic position represented on <em>x</em>-axis. Roman numerals indicate each of the 16 <em>S. cerevisiae</em> chromosomes. Insets in (A) and (B) are magnifications of chromosomes harboring causative sites and show linked mutations with significant effects. Horizontal dotted lines represent a significance threshold of α = 0.001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">The sites with the strongest statistical associations in YPW89 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>), YPW94 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), and YPW102 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>) were nonsynonymous substitutions affecting the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a> genes located on chromosomes IV, III, and XVI, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). These three mutations are all coding substitutions, which is not surprising given that open reading frames constitute 73% of <em>S</em><em>. cerevisiae</em> genome (<em>Saccharomyces</em> Genome Database). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a><em>(G696D)</em> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a><em>(R12K)</em> are both missense mutations that change one amino acid, whereas <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a><em>(Q971*)</em> introduces an early stop codon truncating 450 amino acids of the protein. All three mutations affect amino acids that are highly conserved across <em>Saccharomyces</em> species, with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a><em>(G696D)</em> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a><em>(R12K)</em> substitutions predicted to be deleterious using SIFT (SIFT score = 0 for both; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2009</a>). The fitness estimates of each substitution are similar to the fitness consequences reported for deletion alleles of these genes (W = 0.859, 0.954, and 0.983 for single-site substitutions in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a>; W = 0.896, 0.921, and 0.971 for deletions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a> [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Breslow <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>]), suggesting that the mutations observed severely impair the function of the corresponding proteins. Interestingly, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a> appears to be a direct regulator of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> expression: <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a> protein is a general transcriptional repressor that was shown to directly bind <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> promoter in ChIP-chip experiments (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hanlon <em>et al.</em> 2011</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a> is a repressor of hypoxic genes that might indirectly affect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> expression through the regulation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002981">Pdr1</a> transcription factor (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Harbison <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Larochelle <em>et al.</em> 2006</a>). The Tup1-Ssn6 complex is also a well-established regulator of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a> expression (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mennella <em>et al.</em> 2003</a>), suggesting that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680">Tup1</a> acts at multiple levels of the regulatory network controlling <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> expression. Finally, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a> encodes a facultative subunit of the RNA polymerase II holoenzyme (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Song <em>et al.</em> 1996</a>), which could potentially act on several components of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a> regulatory network.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464597/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464597/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Properties of the three causative sites identified by BSA-seq and confirmed by single site mutagenesis</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Validating bulk segregant mapping results</h3><p id="p-47">In parallel to the method described previously, we used a more traditional mapping approach involving tetrad dissection (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Birkeland <em>et al.</em> 2010</a>) to analyze the YPW89, YPW94, and YPW102 mutants. Each mutant was crossed to a common mapping strain (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FileS1.pdf">File S1</a>); the resulting diploids were sporulated, and a dozen tetrads were dissected. For each tetrad, the fluorescence phenotype of each spore was determined by measuring mean fluorescence of their haploid mitotic progeny using flow cytometry, and the two colonies most likely to carry the causative mutation were identified manually (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS7.pdf">Figure S7</a>). Segregant progeny deemed to show the mutant phenotype were pooled together, genomic DNA from this pool was extracted and subjected to Illumina sequencing, and allele frequencies were estimated for each variable site. G-tests were used to compare the observed mutation frequency at variable sites to a null model with a frequency of 0.5.</p><p id="p-48">For YPW89, YPW94, and YPW102, the best candidate mutations identified with the BSA-seq approach were also highly significant with the tetrad dissection method (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS8.pdf">Figure S8</a>). Compared with the mass sporulation and BSA, however, tetrad dissection was tedious and less amenable to the analysis of a large number of mutants. The tetrad approach was also very sensitive to errors in phenotype assignment, which can be caused by environmental variation or stochastic noise. Indeed, several mutants failed to yield a significant candidate site when using this approach (data not shown). For such mutants, some tetrads showed a clear 2:2 segregation of the fluorescence phenotype (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS7.pdf">Figure S7</a>A), whereas others were harder to characterize (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1/FigureS7.pdf">Figure S7</a>B). This might be explained by the fact that each of the four sister spores was grown in a separate vial prior to phenotyping with the tetrad dissection approach, but all spores were grown together in the same vial for BSA-seq, minimizing the influence of micro-environmental factors.</p><p id="p-49">We also tested directly whether the variants identified by BSA-seq in YPW89, YPW94, and YPW102 were responsible for their mutant phenotypes by using site-directed mutagenesis to introduce each candidate mutation individually into the genetic background carrying the <em>P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP</em> transgene that was originally used for the EMS mutagenesis screen. In all three cases, the single site mutation completely recapitulated the fluorescence phenotype of the EMS-mutant from which it was identified (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). This result, combined with the absence of any other significant mutation unlinked to these causative sites (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>), shows that each original mutant carried exactly one causative mutation and that this mutation could be unambiguously identified using BSA-seq despite its small phenotypic effect.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Single mutations identified by BSA-seq completely explain the mutant phenotypes. Mean expression level (measured as YFP fluorescence) for 8 replicates each of the wild-type genotype (WT); the YPW89, YPW94, and YPW102 mutants; and the three allele-replacement strains (&#x201C;Single site&#x201D;) for each mapped mutation (SSN2(Q971Stop), TUP1(G696D), and ROX1(R12K)) are shown. For each replicate, the median level of fluorescence of at least 5000 cells was quantified and expressed relative to mean fluorescence in the WT reference strain. Mutant genotypes and allele-replacement strains were compared to the WT strain using t-tests (***P &lt; 0.001). In all three cases, the single site mutant was found to phenocopy the EMS mutant strain (P = 0.58 for SSN2, P = 0.23 for TUP1, and P = 0.44 for ROX1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886662508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Single mutations identified by BSA-seq completely explain the mutant phenotypes. Mean expression level (measured as YFP fluorescence) for 8 replicates each of the wild-type genotype (WT); the YPW89, YPW94, and YPW102 mutants; and the three allele-replacement strains (&#x201C;Single site&#x201D;) for each mapped mutation (&lt;em&gt;SSN2(Q971Stop)&lt;/em&gt;, &lt;em&gt;TUP1(G696D)&lt;/em&gt;, and &lt;em&gt;ROX1(R12K)&lt;/em&gt;) are shown. For each replicate, the median level of fluorescence of at least 5000 cells was quantified and expressed relative to mean fluorescence in the WT reference strain. Mutant genotypes and allele-replacement strains were compared to the WT strain using &lt;em&gt;t&lt;/em&gt;-tests (***&lt;em&gt;P&lt;/em&gt; &lt; 0.001). In all three cases, the single site mutant was found to phenocopy the EMS mutant strain (&lt;em&gt;P&lt;/em&gt; = 0.58 for &lt;em&gt;SSN2&lt;/em&gt;, &lt;em&gt;P&lt;/em&gt; = 0.23 for &lt;em&gt;TUP1&lt;/em&gt;, and &lt;em&gt;P&lt;/em&gt; = 0.44 for &lt;em&gt;ROX1&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1205/F6.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/7/1205/F6.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1205/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-50" class="first-child">Single mutations identified by BSA-seq completely explain the mutant phenotypes. Mean expression level (measured as YFP fluorescence) for 8 replicates each of the wild-type genotype (WT); the YPW89, YPW94, and YPW102 mutants; and the three allele-replacement strains (“Single site”) for each mapped mutation (<em>SSN2(Q971Stop)</em>, <em>TUP1(G696D)</em>, and <em>ROX1(R12K)</em>) are shown. For each replicate, the median level of fluorescence of at least 5000 cells was quantified and expressed relative to mean fluorescence in the WT reference strain. Mutant genotypes and allele-replacement strains were compared to the WT strain using <em>t</em>-tests (***<em>P</em> &lt; 0.001). In all three cases, the single site mutant was found to phenocopy the EMS mutant strain (<em>P</em> = 0.58 for <em>SSN2</em>, <em>P</em> = 0.23 for <em>TUP1</em>, and <em>P</em> = 0.44 for <em>ROX1</em>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-51">Using both simulated and empirical data, we describe the impact of innate properties of genetic variants and controllable experimental factors on the success of mapping single nucleotide variants using a BSA with high-throughput sequencing after a mutagenesis screen. We show how mapping success is affected by a mutation’s effect on the mean phenotype as well as its effects on phenotypic variance and fitness. By using simulations to determine optimal experimental conditions and new genetic tools to efficiently isolate large pools of informative segregants, we demonstrated the efficiency of the approach by identifying mutations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680"><em>TUP1</em></a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269"><em>ROX1</em></a> with small effects on the expression of the <em>P</em><sub><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003424"><em>TDH3</em></a></sub><em>-YFP</em> reporter gene in <em>S. cerevisiae</em>. In the sections to follow, we discuss (1) how the effects of a mutation on fitness affect mapping using BSA-seq; (2) how our findings can be applied to mapping other traits in other organisms; and (3) how our conclusions and methods can be used to study QTL underlying natural variation.</p><div id="sec-14" class="subsection"><h3>Impact of fitness on mapping success</h3><p id="p-52">Previous statistical models of BSA-seq focused on the effect of segregating sites on the mean phenotype (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Edwards and Gifford 2012</a>), but their effects on the standard deviation and relative fitness can also impact mapping success when using pooled segregant approaches in <em>S. cerevisiae</em> (Wilkening <em>et al.</em> 2013). Fitness effects could be an especially important source of discovery bias in BSA-seq data given the high proportion of random mutations showing detrimental effects on growth (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eyre-Walker and Keightley 2007</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wloch <em>et al.</em> 2001</a>). We examined this issue computationally and found that starting with a large population of spores and selecting large pools of segregants (2 × 10<sup>5</sup>) was essential for achieving high mapping power when a mutation affected both the phenotype of interest and relative fitness. For example, our model predicted that selection of only 10<sup>3</sup> segregants would be sufficient to map a mutation with a 3% effect on the mean and no effect on fitness, yet reducing the size of the segregant bulk in this study from 2 × 10<sup>5</sup> to 10<sup>3</sup> would have caused the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002851"><em>SSN2</em></a>(Q971*) mutation with a &gt;10% effect on the mean to remain undetected because of its deleterious fitness effects. Because deleterious alleles tend to be purged from both bulks during growth, the power to detect a significant difference in allele frequencies between the two bulks is decreased. One solution to reduce the impact of fitness on mapping success would be to decrease the generations of growth after meiosis, but this is not always possible. For example, in our case, growth was needed for the cells to express the fluorescence phenotype as well as to increase the amount of genomic DNA available for sequencing. This latter growth phase could be shortened by using a protocol for preparing DNA libraries that requires less genomic DNA, but this usually increases noise and cost.</p><p id="p-53">In the absence of fitness effects and competitive growth, alternative alleles for a site not affecting the trait of interest should be found in 50% of segregants in each bulk. Comparing the allele frequency in a single segregant bulk after phenotypic selection to a null frequency of 0.5 to detect causative mutations should be avoided, however, because any effects of a genetic variant on fitness can cause allele frequencies to deviate from this null model, increasing false positives. Rather, allele frequencies should be compared between bulks from the extremities of the phenotypic distribution. If only one tail of the distribution is amenable to phenotypic selection, for instance when selecting for drug resistance, cells that have not been subjected to phenotypic sorting but have otherwise undergone the same experimental steps as the segregant bulks should be used to define the null model (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Ehrenreich <em>et al.</em> 2010</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Parts <em>et al.</em> 2011</a>).</p></div><div id="sec-15" class="subsection"><h3>Applications for other traits and organisms</h3><p id="p-54">BSA is a powerful approach to mapping for species and traits in which large numbers of recombinant offspring can be analyzed and individuals with extreme phenotypes can be efficiently isolated. Selected bulks can now be genotyped <em>en masse</em> by high-throughput sequencing whenever a reference genome is available or can be obtained. When these conditions are met, BSA-seq can quickly identify mutations causing a mutant phenotype, even when the phenotypic effect of a mutation is very small. Our data show that the experimental design needed to most reliably and cost-effectively identify such mutations is different in each case. We encourage researchers to use the simulations and statistical models described in this study to identify experimental parameters that will maximize their own mapping success by tuning the parameters in the model to their specific system. These parameters include not only a mutation’s effect on the mean phenotype and fitness, but also its effect on the standard deviation of the phenotype. For example, we found that when the mutant phenotype has a standard deviation much larger or much smaller than the wild-type phenotype, mapping power decreases quickly with a wide range of BSA-seq experimental designs. Under these conditions, sequencing individuals from the two symmetric tails of the phenotypic distribution is not recommended and an alternative approach should be considered, such as selection of asymmetric bulks. Analysis of larger bulks should also help increase power in these cases.</p><p id="p-55">When extrapolating our findings to mapping other species and traits, it is important to consider that we modeled a BSA-seq experiment in yeast including population growth between meiosis and phenotyping as well as between phenotyping and DNA sequencing. This competitive growth is not necessary when using BSA-seq in multicellular organisms such as fruit flies or nematodes. Therefore, mapping power should be much less affected by mutations that impact reproductive fitness, allowing smaller population sizes to be used. Still, the minimum effect size that can be mapped in these types of organisms will usually be larger than the minimum effect size that can be mapped in yeast, both because of the increased genome size and because of the smaller attainable bulk size. For example, if a single causative mutation is segregating in an F<sub>2</sub> population, our model predicts that the power of BSA-seq to detect a mutation changing the phenotype by 5% relative to wild type is greater than 0.9 for a total population size of 10<sup>4</sup>, a 5% cutoff for phenotypic selection, and an average sequencing coverage of 25×. Although this is not a simple task, these parameter values can be achieved in <em>Drosophila melanogaster</em> and <em>Caenorhabditis elegans</em>, respectively, using tools such as fly cages to raise large populations of flies or worm sorters to automate phenotypic scoring and selection.</p></div><div id="sec-16" class="subsection"><h3>Mapping phenotypic variation in natural populations</h3><p id="p-56">BSA-seq has been shown to be a powerful approach for mapping small effect QTL underlying natural variation in <em>S. cerevisiae</em> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2014</a>). Compared with the mutants characterized in our study, strains used for QTL mapping typically have more segregating sites and more causative loci. The large number of sites segregating in these strains leads to many linked polymorphisms, which reduces mapping resolution, but can improve the power to detect small effect QTL (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Edwards and Gifford 2012</a>). However, the presence of multiple QTL acting in the same direction can decrease the power to detect a polymorphism of small effect compared with the case in which it segregates alone (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2013</a>). Our work suggests that the effects of QTL on phenotypic noise and/or fitness should also be considered in future statistical models of QTL mapping via BSA-seq to avoid discovery biases.</p><p id="p-57">The three genetic tools we used to increase the sensitivity of BSA-seq for finding novel mutations can also be used to study natural variation in yeast. Specifically, the dominant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003276">RME1</a>(ins-308A)</em> allele that we inserted into our mapping strain to increase meiosis rate (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Deutschbauer and Davis 2005</a>) can also be incorporated into other strains, allowing for the efficient recovery of large numbers of segregants. This is important because many strains of <em>S. cerevisiae</em>, including the commonly used S288c lab strain and its derivatives as well as some wild isolates, have low sporulation rates that limit the efficiency of BSA-seq (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gerke <em>et al.</em> 2009</a>). The <em>FASTER MT</em> cassette (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2012</a>) can also be inserted into other genotypes to allow for robust and efficient recovery of <em>MATα</em> spores, preventing mating among F<sub>1</sub> segregants. Compared with the Yeast Magic Marker (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tong <em>et al.</em> 2001</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pan <em>et al.</em> 2004</a>) used for a similar purpose in previous BSA studies (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2014</a>; Wilkening <em>et al.</em> 2013; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Ehrenreich <em>et al.</em> 2010</a>), <em>FASTER MT</em> requires less genetic manipulation of the mapping strain(s), reduces biases caused by diploids in the sorted haploid cultures (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gerstein and Otto 2011</a>; Wilkening <em>et al.</em> 2013), and limits the impact of competitive growth on mapping power by allowing <em>MATα</em> cells to be sorted immediately after spore isolation. Finally, if the phenotype of interest can be coupled to a fluorescent reporter gene, FACS can be used for high-throughput phenotyping and selection. Other easy-to-score phenotypes should also be well suited for genetic mapping using BSA-seq.</p><p id="p-58">In conclusion, this study provides a methodological framework for efficiently mapping genetic variants with small effects, illustrates the importance of considering the fitness effects of causative variants when using BSA-seq in microorganisms such as yeast, and describes the use of experimental tools that can reduce the bias against detection of variants with small effects by allowing very large populations of phenotypically divergent individuals to be collected and analyzed.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">We thank the University of Michigan Flow Cytometry Core, as well as the Center for Chemical Genomics and High Throughput Screening for the access to flow cytometry and cell sorting platforms, and the University of Michigan Sequencing Core for Next Generation Sequencing. We also thank Brian Chin and William Timberlake for sharing the <em>FASTER MT</em> plasmid; Adam Deutschbauer and Barry Williams for sharing yeast strains; and Joseph Coolon, Alisha John, Rich Lusk, Calum Maclean, and Kraig Stevenson for helpful discussions and comments on the manuscript. Financial support for this work was provided to P.J.W. by grants from the University of Michigan, National Science Foundation (MCB-1021398), National Institutes of Health (1 R01 GM108826), and March of Dimes (Basil O’Connor Starter Scholar Research Award 5- FY07-181). F.D. was funded by a Long-Term Fellowship from the European Molecular Biology Organization (EMBO ALTF 1114-2012); B.P.H.M. was supported by the University of Michigan Genome Sciences Training Program (T32 HG000040) and by a University of Michigan Rackham Merit Fellowship; J.D.G. was supported by a National Institutes of Health NRSA (1F32GM083513-0); N.S. was supported by the University of Michigan Undergraduate Research Opportunities Program; and T.E.B. was supported by the National Science Foundation Research Experiences for Undergraduate Program at University of Michigan (ED-QUE<sup>2</sup>ST).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011783/-/DC1</a></p></li><li class="fn" id="fn-13"><p id="p-60"><em>Communicating editor: A. Gasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-20"><span class="received-label">Received </span>March 20, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-04-24"><span class="accepted-label">Accepted </span>April 24, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Duveau <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.1" data-doi="10.1038/nature12904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treusch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of single-cell protein abundance variation in large yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">506</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D506%26rft.spage%253D494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12904%26rft_id%253Dinfo%253Apmid%252F24402228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24402228&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332165100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.2" data-doi="10.1371/journal.pgen.1003534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bastide</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stöbe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide, fine-scale map of natural pigmentation variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003534%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003534%26rft_id%253Dinfo%253Apmid%252F23754958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23754958&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.3" data-doi="10.1534/genetics.110.123232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birkeland</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozdemir</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisman</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discovery of mutations in <em>Saccharomyces cerevisiae</em> by pooled linkage analysis and whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1127</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123232%26rft_id%253Dinfo%253Apmid%252F20923977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzExMjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.4" data-doi="10.1534/genetics.106.057927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christianson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping novel traits by array-assisted bulk segregant analysis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057927%26rft_id%253Dinfo%253Apmid%252F16624899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE4MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.5" data-doi="10.1038/nmeth.1234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart-Ornstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBreslow%26rft.auinit1%253DD.%2BK.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D711%26rft.epage%253D718%26rft.atitle%253DA%2Bcomprehensive%2Bstrategy%2Benabling%2Bhigh-resolution%2Bfunctional%2Banalysis%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1234%26rft_id%253Dinfo%253Apmid%252F18622397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18622397&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258077400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-other" id="cit-4.7.1205.6" data-doi="10.1534/g3.111.001826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frizzell</span>  <span class="cit-name-given-names">M. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timberlake</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FASTER MT: isolation of pure populations of a and α ascospores from <em>Saccharomyces cerevisiae</em>.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DChin%26rft.auinit1%253DB.%2BL.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D449%26rft.epage%253D452%26rft.atitle%253DFASTER%2BMT%253A%2BIsolation%2Bof%2BPure%2BPopulations%2Bof%2Ba%2Band%2B%257Balpha%257D%2BAscospores%2Bfrom%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001826%26rft_id%253Dinfo%253Apmid%252F22540036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNC80NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.7" data-doi="10.1038/ng1674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait loci mapped to single-nucleotide resolution in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1333</span>–<span class="cit-lpage">1340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDeutschbauer%26rft.auinit1%253DA.%2BM.%26rft.volume%253D37%26rft.issue%253D12%26rft.spage%253D1333%26rft.epage%253D1340%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bmapped%2Bto%2Bsingle-nucleotide%2Bresolution%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1674%26rft_id%253Dinfo%253Apmid%252F16273108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16273108&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233657300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.8" data-doi="10.1534/genetics.104.036871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036871%26rft_id%253Dinfo%253Apmid%252F15716499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.9" data-doi="10.1186/1471-2105-13-S14-S8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping with pooled sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>(<span class="cit-supplement">Suppl 6</span>): <span class="cit-fpage">S8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-S14-S8%26rft_id%253Dinfo%253Apmid%252F23282012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-S14-S8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23282012&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.10" data-doi="10.1038/nature08923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torabi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dissection of genetically complex traits with extremely large pools of yeast segregants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">1039</span>–<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEhrenreich%26rft.auinit1%253DI.%2BM.%26rft.volume%253D464%26rft.issue%253D7291%26rft.spage%253D1039%26rft.epage%253D1042%26rft.atitle%253DDissection%2Bof%2Bgenetically%2Bcomplex%2Btraits%2Bwith%2Bextremely%2Blarge%2Bpools%2Bof%2Byeast%2Bsegregants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08923%26rft_id%253Dinfo%253Apmid%252F20393561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20393561&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276635000038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.11" data-doi="10.1371/journal.pgen.1002570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torabi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of highly complex chemical resistance traits across four yeast strains.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DEhrenreich%26rft.auinit1%253DI.%2BM.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002570%26rft.epage%253De1002570%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bhighly%2Bcomplex%2Bchemical%2Bresistance%2Btraits%2Bacross%2Bfour%2Byeast%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002570%26rft_id%253Dinfo%253Apmid%252F22438822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22438822&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.12" data-doi="10.1038/nrg2146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The distribution of fitness effects of new mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D610%26rft.epage%253D618%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bfitness%2Beffects%2Bof%2Bnew%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2146%26rft_id%253Dinfo%253Apmid%252F17637733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637733&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248170700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fellows</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Deducer: a data analysis GUI for R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D49%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic interactions between transcription factors cause natural variation in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">2007</span>–<span class="cit-lpage">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D323%26rft.spage%253D2007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.15" data-doi="10.1371/journal.pone.0026599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cryptic fitness advantage: diploids invade haploid populations despite lacking any apparent advantage as measured by standard fitness assays.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e26599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGerstein%26rft.auinit1%253DA.%2BC.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De26599%26rft.epage%253De26599%26rft.atitle%253DCryptic%2Bfitness%2Badvantage%253A%2Bdiploids%2Binvade%2Bhaploid%2Bpopulations%2Bdespite%2Blacking%2Bany%2Bapparent%2Badvantage%2Bas%2Bmeasured%2Bby%2Bstandard%2Bfitness%2Bassays.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0026599%26rft_id%253Dinfo%253Apmid%252F22174734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0026599&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174734&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayikçi</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genetic architecture of biofilm formation in a clinical isolate of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D193%26rft.spage%253D587%26rft_id%253Dinfo%253Apmid%252F23172850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23172850&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.17" data-doi="10.1371/journal.pgen.1002497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Contrasting properties of gene-specific regulatory, coding, and copy number mutations in <em>Saccharomyces cerevisiae</em>: frequency, effects, and dominance.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGruber%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1002497%26rft.epage%253De1002497%26rft.atitle%253DContrasting%2Bproperties%2Bof%2Bgene-specific%2Bregulatory%252C%2Bcoding%252C%2Band%2Bcopy%2Bnumber%2Bmutations%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bfrequency%252C%2Beffects%252C%2Band%2Bdominance.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002497%26rft_id%253Dinfo%253Apmid%252F22346762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22346762&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.18" data-doi="10.1371/journal.pone.0019060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanlon</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzo</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatomer</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The stress response factors Yap6, Cin5, Phd1, and Skn7 direct targeting of the conserved co-repressor Tup1-Ssn6 in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHanlon%26rft.auinit1%253DS.%2BE.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De19060%26rft.epage%253De19060%26rft.atitle%253DThe%2Bstress%2Bresponse%2Bfactors%2BYap6%252C%2BCin5%252C%2BPhd1%252C%2Band%2BSkn7%2Bdirect%2Btargeting%2Bof%2Bthe%2Bconserved%2Bco-repressor%2BTup1-Ssn6%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019060%26rft_id%253Dinfo%253Apmid%252F21552514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552514&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.19" data-doi="10.1038/nature02800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harbison</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macisaac</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transcriptional regulatory code of a eukaryotic genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHarbison%26rft.auinit1%253DC.%2BT.%26rft.volume%253D431%26rft.issue%253D7004%26rft.spage%253D99%26rft.epage%253D104%26rft.atitle%253DTranscriptional%2Bregulatory%2Bcode%2Bof%2Ba%2Beukaryotic%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02800%26rft_id%253Dinfo%253Apmid%252F15343339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15343339&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223641500050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.20" data-doi="10.1093/bioinformatics/btp373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wylie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">VarScan: variant detection in massively parallel sequencing of individual and pooled samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp373%26rft_id%253Dinfo%253Apmid%252F19542151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTcvMjI4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.21" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.22" data-doi="10.1093/bioinformatics/btr589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3435</span>–<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr589%26rft_id%253Dinfo%253Apmid%252F22025480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjQvMzQzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.23" data-doi="10.1038/nprot.2009.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKumar%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D1073%26rft.epage%253D1081%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bcoding%2Bnon-synonymous%2Bvariants%2Bon%2Bprotein%2Bfunction%2Busing%2Bthe%2BSIFT%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.86%26rft_id%253Dinfo%253Apmid%252F19561590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561590&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.24" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.25" data-doi="10.1128/MCB.02450-05"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larochelle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turcotte</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Oxidative stress-activated zinc cluster protein Stb5 has dual activator/repressor functions required for pentose phosphate pathway regulation and NADPH production.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">6690</span>–<span class="cit-lpage">6701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLarochelle%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D17%26rft.spage%253D6690%26rft.epage%253D6701%26rft.atitle%253DOxidative%2BStress-Activated%2BZinc%2BCluster%2BProtein%2BStb5%2BHas%2BDual%2BActivator%252FRepressor%2BFunctions%2BRequired%2Bfor%2BPentose%2BPhosphate%2BPathway%2BRegulation%2Band%2BNADPH%2BProduction%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.02450-05%26rft_id%253Dinfo%253Apmid%252F16914749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xNy82NjkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.27" data-doi="10.1371/journal.pgen.1002912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Advances in quantitative trait analysis in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002912%26rft.epage%253De1002912%26rft.atitle%253DAdvances%2Bin%2Bquantitative%2Btrait%2Banalysis%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002912%26rft_id%253Dinfo%253Apmid%252F22916041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916041&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.28" data-doi="10.1371/journal.pcbi.1002255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The statistics of bulk segregant analysis using next generation sequencing.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMagwene%26rft.auinit1%253DP.%2BM.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002255%26rft.epage%253De1002255%26rft.atitle%253DThe%2Bstatistics%2Bof%2Bbulk%2Bsegregant%2Banalysis%2Busing%2Bnext%2Bgeneration%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002255%26rft_id%253Dinfo%253Apmid%252F22072954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072954&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.29" data-doi="10.1128/EC.2.6.1288-1303.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mennella</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klinkenberg</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zitomer</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Recruitment of Tup1-Ssn6 by yeast hypoxic genes and chromatin-independent exclusion of TATA binding protein.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1288</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DMennella%26rft.auinit1%253DT.%2BA.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253D1288%26rft.epage%253D1303%26rft.atitle%253DRecruitment%2Bof%2BTup1-Ssn6%2Bby%2BYeast%2BHypoxic%2BGenes%2Band%2BChromatin-Independent%2BExclusion%2Bof%2BTATA%2BBinding%2BProtein%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.2.6.1288-1303.2003%26rft_id%253Dinfo%253Apmid%252F14665463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIyLzYvMTI4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.30" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.31" data-doi="10.1016/j.molcel.2004.09.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sookhai-Mahadeo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A robust toolkit for functional profiling of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPan%26rft.auinit1%253DX.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D496%26rft.atitle%253DA%2Brobust%2Btoolkit%2Bfor%2Bfunctional%2Bprofiling%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.09.035%26rft_id%253Dinfo%253Apmid%252F15525520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.09.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15525520&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225004500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.32" data-doi="10.1101/gr.116731.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revealing the genetic structure of a trait by sequencing a population under selection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.116731.110%26rft_id%253Dinfo%253Apmid%252F21422276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEzMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.33" data-doi="10.1128/EC.00016-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pomraning</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bulk segregant analysis followed by high-throughput sequencing reveals the <em>Neurospora</em> cell cycle gene, <em>ndc-1</em>, to be allelic with the gene for ornithine decarboxylase, <em>spe-1</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00016-11%26rft_id%253Dinfo%253Apmid%252F21515825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMC82LzcyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-other" id="cit-4.7.1205.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.35" data-doi="10.1101/gr.128124.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.128124.111%26rft_id%253Dinfo%253Apmid%252F22267522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvOTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.36" data-doi="10.1128/MCB.16.1.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuchin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>SSN</em> genes that affect transcriptional repression in <em>Saccharomyces cerevisiae</em> encode SIN4, ROX3, and SRB proteins associated with RNA polymerase II.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSong%26rft.auinit1%253DW.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D120%26rft.atitle%253DSSN%2Bgenes%2Bthat%2Baffect%2Btranscriptional%2Brepression%2Bin%2BSaccharomyces%2Bcerevisiae%2Bencode%2BSIN4%252C%2BROX3%252C%2Band%2BSRB%2Bproteins%2Bassociated%2Bwith%2BRNA%2Bpolymerase%2BII%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.1.115%26rft_id%253Dinfo%253Apmid%252F8524287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvMTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.37" data-doi="10.1016/j.ymeth.2006.06.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storici</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The <em>delitto perfetto</em> approach to <em>in vivo</em> site-directed mutagenesis and chromosome rearrangements with synthetic oligonucleotides in yeast.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DStorici%26rft.auinit1%253DF.%26rft.volume%253D409%26rft.spage%253D329%26rft.epage%253D345%26rft.atitle%253DThe%2Bdelitto%2Bperfetto%2Bapproach%2Bto%2Bin%2Bvivo%2Bsite-directed%2Bmutagenesis%2Band%2Bchromosome%2Brearrangements%2Bwith%2Bsynthetic%2Boligonucleotides%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2006.06.011%26rft_id%253Dinfo%253Apmid%252F16793410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2006.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16793410&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238354800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.38" data-doi="10.1101/gr.131698.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swinnen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaerlaekens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pais</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claesen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubmann</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of novel causative genes determining the complex trait of high ethanol tolerance in yeast using pooled-segregant whole-genome sequence analysis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131698.111%26rft_id%253Dinfo%253Apmid%252F22399573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvOTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.39" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.40" data-doi="10.1073/pnas.1200068109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Leeuwen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demaeght</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborne</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dermauw</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gohlke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population bulk segregant mapping uncovers resistance mutations and the mode of action of a chitin synthesis inhibitor in arthropods.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">4407</span>–<span class="cit-lpage">4412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1200068109%26rft_id%253Dinfo%253Apmid%252F22393009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzEyLzQ0MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.41" data-doi="10.1371/journal.pgen.1000942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bulk segregant analysis by high-throughput sequencing reveals a novel xylose utilization gene from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWenger%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000942%26rft.epage%253De1000942%26rft.atitle%253DBulk%2Bsegregant%2Banalysis%2Bby%2Bhigh-throughput%2Bsequencing%2Breveals%2Ba%2Bnovel%2Bxylose%2Butilization%2Bgene%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000942%26rft_id%253Dinfo%253Apmid%252F20485559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20485559&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.42" data-doi="10.1038/88878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicks</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Rapid gene mapping in <em>Caenorhabditis elegans</em> using a high density polymorphism map.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWicks%26rft.auinit1%253DS.%2BR.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D164%26rft.atitle%253DRapid%2Bgene%2Bmapping%2Bin%2BCaenorhabditis%2Belegans%2Busing%2Ba%2Bhigh%2Bdensity%2Bpolymorphism%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F88878%26rft_id%253Dinfo%253Apmid%252F11381264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/88878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11381264&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169011800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.7.1205.43" data-doi="10.1534/genetics.113.160291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkening</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekkedil</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An evaluation of high-throughput approaches to QTL mapping in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160291%26rft_id%253Dinfo%253Apmid%252F24374355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wloch</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szafraniec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korona</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Direct estimate of the mutation rate and the distribution of fitness effects in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWloch%26rft.auinit1%253DD.%2BM.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D441%26rft.epage%253D452%26rft.atitle%253DDirect%2BEstimate%2Bof%2Bthe%2BMutation%2BRate%2Band%2Bthe%2BDistribution%2Bof%2BFitness%2BEffects%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11606524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNDQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.45" data-doi="10.1534/genetics.110.121160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Won</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bulk segregation mapping of mutations in closely related strains of mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1139</span>–<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121160%26rft_id%253Dinfo%253Apmid%252F20923982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzExMzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1205.46" data-doi="10.1371/journal.pgen.1003693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulquié-Moreno</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erdei</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanghe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QTL analysis of high thermotolerance with superior and downgraded parental yeast strains reveals new minor QTLs and converges on novel causative alleles involved in RNA processing.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003693%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003693%26rft_id%253Dinfo%253Apmid%252F23966873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23966873&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/7/1205.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1193" title="Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1217" title="Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/7"><span>Volume 4 Issue 7, July 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/7.cover-source.jpg?itok=MrsYFroL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1205" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--cqMy2tlEVE5RC_2tfLSXV2UqJX1on7FfFqwYumGfgk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/7/1205" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/7/1205" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/7/1205" class="active" data-icon-position="" data-hide-link-title="0">Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459929" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-VvNXQgPME1h-06x71CBKPHv0XN9hpQWFGlZq7F6HnSM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1205.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/7/1205" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CvesJB3qzc17wx7gLfEFIKolHo65Ztbux6i4fvdP5ww" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/7/1205" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1205.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459929" id="node459929--2" data-pisa="ggg;4/7/1205" data-pisa-master="ggg;g3.114.011783" data-apath="/ggg/4/7/1205.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Small Effect Mutations in <em>Saccharomyces cerevisiae</em>: Impacts of Experimental Design and Mutational Properties</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fabien Duveau</span>, <span class="highwire-citation-author" data-delta="1">Brian P. H. Metzger</span>, <span class="highwire-citation-author" data-delta="2">Jonathan D. Gruber</span>, <span class="highwire-citation-author" data-delta="3">Katya Mack</span>, <span class="highwire-citation-author" data-delta="4">Natasha Sood</span>, <span class="highwire-citation-author" data-delta="5">Tiffany E. Brooks</span> and <span class="highwire-citation-author" data-delta="6">Patricia J. Wittkopp</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1205-1216;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011783 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459929--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fabien Duveau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabien%2BDuveau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duveau%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabien%2BDuveau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fduveau@umich.edu" class="" data-icon-position="" data-hide-link-title="0">fduveau@umich.edu</a>
<a href="mailto:wittkopp@umich.edu" class="" data-icon-position="" data-hide-link-title="0">wittkopp@umich.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian P. H. Metzger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BH.%2BMetzger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metzger%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BP.%2BH.%2BMetzger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan D. Gruber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BD.%2BGruber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gruber%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BD.%2BGruber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katya Mack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katya%2BMack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mack%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatya%2BMack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Natasha Sood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natasha%2BSood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sood%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatasha%2BSood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tiffany E. Brooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiffany%2BE.%2BBrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brooks%20TE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiffany%2BE.%2BBrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia J. Wittkopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, & Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Computational Medicine and Bioinformatics, University of Michigan Medical School, Ann Arbor, Michigan 48109-2218</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BJ.%2BWittkopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittkopp%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BJ.%2BWittkopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fduveau@umich.edu" class="" data-icon-position="" data-hide-link-title="0">fduveau@umich.edu</a>
<a href="mailto:wittkopp@umich.edu" class="" data-icon-position="" data-hide-link-title="0">wittkopp@umich.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/7/1205" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/7/1205&amp;title=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/7/1205&amp;title=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/7/1205&amp;title=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/7/1205&amp;text=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/7/1205&amp;title=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/7/1205&amp;t=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/7/1205&amp;title=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/7/1205&amp;title=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459929/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459929" id="node459929--3" data-pisa="ggg;4/7/1205" data-pisa-master="ggg;g3.114.011783" data-apath="/ggg/4/7/1205.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Small Effect Mutations in <em>Saccharomyces cerevisiae</em>: Impacts of Experimental Design and Mutational Properties</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fabien Duveau</span>, <span class="highwire-citation-author" data-delta="1">Brian P. H. Metzger</span>, <span class="highwire-citation-author" data-delta="2">Jonathan D. Gruber</span>, <span class="highwire-citation-author" data-delta="3">Katya Mack</span>, <span class="highwire-citation-author" data-delta="4">Natasha Sood</span>, <span class="highwire-citation-author" data-delta="5">Tiffany E. Brooks</span> and <span class="highwire-citation-author" data-delta="6">Patricia J. Wittkopp</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1205-1216;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011783 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459929--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fabien Duveau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabien%2BDuveau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duveau%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabien%2BDuveau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fduveau@umich.edu" class="" data-icon-position="" data-hide-link-title="0">fduveau@umich.edu</a>
<a href="mailto:wittkopp@umich.edu" class="" data-icon-position="" data-hide-link-title="0">wittkopp@umich.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian P. H. Metzger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BH.%2BMetzger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metzger%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BP.%2BH.%2BMetzger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan D. Gruber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BD.%2BGruber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gruber%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BD.%2BGruber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katya Mack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katya%2BMack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mack%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatya%2BMack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Natasha Sood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natasha%2BSood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sood%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatasha%2BSood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tiffany E. Brooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiffany%2BE.%2BBrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brooks%20TE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiffany%2BE.%2BBrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia J. Wittkopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, & Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109-1048</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Computational Medicine and Bioinformatics, University of Michigan Medical School, Ann Arbor, Michigan 48109-2218</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BJ.%2BWittkopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittkopp%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BJ.%2BWittkopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fduveau@umich.edu" class="" data-icon-position="" data-hide-link-title="0">fduveau@umich.edu</a>
<a href="mailto:wittkopp@umich.edu" class="" data-icon-position="" data-hide-link-title="0">wittkopp@umich.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1205&amp;rft.epage=1216&amp;rft.atitle=Mapping%20Small%20Effect%20Mutations%20in%20Saccharomyces%20cerevisiae%3A%20Impacts%20of%20Experimental%20Design%20and%20Mutational%20Properties&amp;rft.volume=4&amp;rft.issue=7&amp;rft.date=2014-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Duveau%2C+Fabien&amp;rft.au=Metzger%2C+Brian+P.+H.&amp;rft.au=Gruber%2C+Jonathan+D.&amp;rft.au=Mack%2C+Katya&amp;rft.au=Sood%2C+Natasha&amp;rft.au=Brooks%2C+Tiffany+E.&amp;rft.au=Wittkopp%2C+Patricia+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459929/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459929/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459929/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459929/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459929/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459929/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459929/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459929/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459929/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459929/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459929/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459929/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/7/1205" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/7/1205.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/7/1205.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/7/1205.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/7/1205.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
